<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-02 06:30:49 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>6</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2f585a6d644d3577541d684c98016ab2b165e02" target='_blank'>
              Sensing and guiding cell-state transitions by using genetically encoded endoribonuclease-mediated microRNA sensors.
              </a>
            </td>
          <td>
            Lei Wang, Wenlong Xu, Shun Zhang, Gregory C Gundberg, Christine R Zheng, Zhengpeng Wan, Kamila Mustafina, Fabio Caliendo, Hayden Sandt, Roger D. Kamm, Ron Weiss
          </td>
          <td>2024-07-09</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Therapeutic resistance is the leading cause of treatment failure and death from cancer. While resistance can be driven by genetic mutations, mounting evidence also points to an epigenetic basis of resistance. Much of this epigenetic, or non-genetic, resistance has been attributed to drug-resistant transcriptional cell states that are either induced by drug treatment or pre-exist in a fraction of cells selected by treatment. However, the extent to which long-term resistance is manifested in the early inherent cellular response to drugs is poorly understood, and which aspects, if any, of this early response to drug-induced transcriptional response are evolutionarily conserved properties of cells. To address these questions, we integrate datasets of long-term drug resistance and early drug response data across multiple cell lines with drug response and resistance data from bacteria and yeast. Our findings suggest that cancer cell states in both the drug-naive populations as well as in populations shortly after treatment share transcriptional properties with fully established resistant cell populations, and CRISPR-cas9 knockout of transcription factors predicted to regulate the resistant transcriptional programs result in inceased drug sensitivity. Furthermore, the resistance states manifested as early drug response are evolutionarily conserved. Finally, we show that early resistant states discriminate responders from non-responders across multiple human cancer trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad4a4d49f9d38cdc5b6bcd770ac201bba19fb9f" target='_blank'>
              Transcriptional Characterization of Resistance in Early Drug Response
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Michael M. Gottesman, Sridhar Hannenhalli
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Summary Phenotypic adaptation, the ability of cells to change phenotype in response to external pressures, has been identified as a driver of drug resistance in cancer. To quantify phenotypic adaptation in BRAFV600E-mutant melanoma, we develop a theoretical model that emerges from data analysis of WM239A-BRAFV600E growth rates in response to drug challenge with the BRAF-inhibitor encorafenib. Our model constitutes a cell population model in which each cell is individually described by one of multiple discrete and plastic phenotype states that are directly linked to drug-dependent net growth rates and, by extension, drug resistance. Data-matched simulations reveal that phenotypic adaptation in the cells is directed towards states of high net growth rates, which enables evasion of drug-effects. The model subsequently provides an explanation for when and why intermittent treatments outperform continuous treatments in vitro, and demonstrates the benefits of not only targeting, but also leveraging, phenotypic adaptation in treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db72a236ec0fc5bd479b42430df6525df1d343c7" target='_blank'>
              Growth rate-driven modelling reveals how phenotypic adaptation drives drug resistance in BRAFV600E-mutant melanoma
              </a>
            </td>
          <td>
            Sara Hamis, A. Browning, A. Jenner, Chiara Villa, Philip Maini, Tyler Cassidy
          </td>
          <td>2024-08-18</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Prostate cancer (PC) depicts a major health challenge all over the globe due to its complexities in the treatment and diverse clinical trajectories. Even in the advances in the modern treatment strategies, the spectrum of resistance to the therapies continues to be a significant challenge. This review comprehensively examines the underlying mechanisms of the therapy resistance occurred in PC, focusing on both the tumor microenvironment and the signaling pathways implicated in the resistance. Tumor microenvironment comprises of stromal and epithelial cells, which influences tumor growth, response to therapy and progression. Mechanisms such as microenvironmental epithelial-mesenchymal transition (EMT), anoikis suppression and stimulation of angiogenesis results in therapy resistance. Moreover, dysregulation of signaling pathways including androgen receptor (AR), mammalian target of rapamycin/phosphoinositide 3 kinase/AKT (mTOR/PI3K/AKT), DNA damage repair and Janus kinase/signal transducers and activators of transcription (JAK/STAT) pathways drive therapy resistance by promoting tumor survival and proliferation. Understanding these molecular pathways is important for developing targeted therapeutic interventions which overcomes resistance. In conclusion, a complete grasp of mechanisms and pathways underlying medication resistance in PC is important for the development of individualized treatment plans and enhancements of clinical outcomes. By studying and understanding the complex mechanisms of signaling pathways and microenvironmental factors contributing to therapy resistance, this study focuses and aims to guide the development of innovative therapeutic approaches to effectively overcome the PC progression and improve the survival rate of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27b4a025bb1904ed23371786225a1e4cfcd5778f" target='_blank'>
              Therapy resistance in prostate cancer: mechanism, signaling and reversal strategies
              </a>
            </td>
          <td>
            Neha Thakur, Pallavi Singh, Aditi Bagri, Saumya Srivastava, Vinay Dwivedi, Asha Singh, Sunil Kumar Jaiswal, S. Dholpuria
          </td>
          <td>2024-08-29</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer resistance refers to the ability of cancer cells to evade the effects of treatments, such as chemotherapy and radiation, making it challenging to eliminate tumors. Cancer cells develop resistance through genetic modifications, alterations in cellular pathways, or microenvironmental adaptations that allow cancer cells to persist, proliferate, and even thrive despite the application of therapies designed to eradicate them. Despite efforts to create more potent target-based drugs, the flexibility of acquired resistance necessitates advanced options. Combination therapy and precision immunotherapy has revolutionized treatment but is limited by patient specificity and requires further development. New approaches like small molecules, peptides, and nanotherapeutics aim to overcome resistance by enhancing site-specific delivery and increasing drug concentration inside cancer cells. This review provides a comprehensive overview of the molecular mechanisms underlying cancer drug resistance, including genetic mutations, epigenetic alterations, efflux pump activity, and the influence of the tumor microenvironment. Additionally, the roles of microRNAs, long noncoding RNAs, and cellular processes such as autophagy and hypoxia in mediating resistance are examined. This review seeks to improve patient care and help to the development of more effective cancer medicines by explaining these complicated systems and investigating novel therapeutic alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c262ab9bdd31ff19f10e8cd7a734094b9985bc60" target='_blank'>
              An overview of molecular mechanisms in cancer drug resistance and therapeutic strategies
              </a>
            </td>
          <td>
            Navin Kumar, Ranjeeta Bansala
          </td>
          <td>2024-07-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Drug resistance is the major cause of therapeutic failure in high-grade serous ovarian cancer (HGSOC). Yet, the mechanisms by which tumors evolve to drug resistant states remains largely unknown. To address this, we aimed to exploit clone-specific genomic structural variations by combining scaled single-cell whole genome sequencing with longitudinally collected cell-free DNA (cfDNA), enabling clonal tracking before, during and after treatment. We developed a cfDNA hybrid capture, deep sequencing approach based on leveraging clone-specific structural variants as endogenous barcodes, with orders of magnitude lower error rates than single nucleotide variants in ctDNA (circulating tumor DNA) detection, demonstrated on 19 patients at baseline. We then applied this to monitor and model clonal evolution over several years in ten HGSOC patients treated with systemic therapy from diagnosis through recurrence. We found drug resistance to be polyclonal in most cases, but frequently dominated by a single high-fitness and expanding clone, reducing clonal diversity in the relapsed disease state in most patients. Drug-resistant clones frequently displayed notable genomic features, including high-level amplifications of oncogenes such as CCNE1, RAB25, NOTCH3, and ERBB2. Using a population genetics Wright-Fisher model, we found evolutionary trajectories of these features were consistent with drug-induced positive selection. In select cases, these alterations impacted selection of secondary lines of therapy with positive patient outcomes. For cases with matched single-cell RNA sequencing data, pre-existing and genomically encoded phenotypic states such as upregulation of EMT and VEGF were linked to drug resistance. Together, our findings indicate that drug resistant states in HGSOC pre-exist at diagnosis and lead to dramatic clonal expansions that alter clonal composition at the time of relapse. We suggest that combining tumor single cell sequencing with cfDNA enables clonal tracking in patients and harbors potential for evolution-informed adaptive treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1441270a54104f75e7713e2bb6b7323866959" target='_blank'>
              Tracking clonal evolution of drug resistance in ovarian cancer patients by exploiting structural variants in cfDNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Gryte Satas, Hannah J. Lees, Jake June-Koo Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, N. Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitriy Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Epithelial-to-mesenchymal transition (EMT) is a major axis of phenotypic plasticity not only in diseased conditions such as cancer metastasis and fibrosis but also during normal development and wound healing. Yet-another important axis of plasticity with metastatic implications includes the cancer stem cell (CSCs) and non-CSC transitions. However, in both processes, epithelial (E) and mesenchymal (M) phenotypes are not merely binary states. Cancer cells acquire a spectrum of phenotypes with traits, properties, and markers of both E and M phenotypes, giving rise to intermediary hybrid (E/M) phenotypes. E/M cells play an important role in tumor initiation, metastasis, and disease progression in multiple cancers. Furthermore, the hybrid phenotypes also play a major role in causing therapeutic resistance in cancer. Here, we discuss how a systems biology perspective on the problem, which is implicit in the ‘Team Medicine’ approach outlined in the theme of this Special Issue of The Journal of Clinical Medicine and includes an interdisciplinary team of experts, is more likely to shed new light on EMT in cancer and help us to identify novel therapeutics and strategies to target phenotypic plasticity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6867e0a77d67eee55a122d53d387d619f33219b" target='_blank'>
              Phenotypic Plasticity and Cancer: A System Biology Perspective
              </a>
            </td>
          <td>
            A. R. Subbalakshmi, S. Ramisetty, Atish Mohanty, Siddhika Pareek, Dana Do, Sagun Shrestha, Ajaz Khan, Neel Talwar, Tingting Tan, Priya Vishnubhotla, Sharad S Singhal, Ravi Salgia, Prakash Kulkarni
          </td>
          <td>2024-07-23</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Outcomes for women with breast cancer have improved dramatically in recent decades. However, many patients present with intrinsic drug resistance and others are initially sensitive to anti-cancer drugs but acquire resistance during the course of their treatment, leading to recurrence and/or metastasis. Drug therapy-induced senescence (TIS) is a form of drug resistance characterised by the induction of cell cycle arrest and the emergence of a senescence-associated secretory phenotype (SASP) that can develop in response to chemo- and targeted- therapies. A wide range of anticancer interventions can lead to cell cycle arrest and SASP induction, by inducing genotoxic stress, hyperactivation of signalling pathways or oxidative stress. TIS can be anti-tumorigenic in the short-term, but pro-tumorigenic in the long-term by creating a pro-inflammatory and immunosuppressive microenvironment. Moreover, the SASP can promote angiogenesis and epithelial-mesenchymal transition in neighbouring cells. In this review, we will describe the characteristics of TIS in breast cancer and detail the changes in phenotype that accompany its induction. We also discuss strategies for targeting senescent cancer cells in order to prevent or delay tumour recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1902ff7d1c0379c707774d89c56e002ae6793dc1" target='_blank'>
              Therapy-induced senescence in breast cancer: an overview
              </a>
            </td>
          <td>
            Suraj Narayanan Chembukavu, Andrew J Lindsay
          </td>
          <td>2024-07-25</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is the second leading cause of global mortality and claims approximately 10 million lives annually. Despite advances in treatments such as surgery, chemotherapy, and immunotherapy, resistance to these methods has emerged. Multidrug resistance (MDR), where cancer cells resist diverse treatments, undermines therapy effectiveness, escalating mortality rates. MDR mechanisms include, among others, drug inactivation, reduced drug uptake, enhanced DNA repair, and activation of survival pathways such as autophagy. Moreover, MDR mechanisms can confer resistance to other therapies like radiotherapy. Understanding these mechanisms is crucial for improving treatment efficacy and identifying new therapeutic targets. Extracellular vesicles (EVs) have gathered attention for their role in cancer progression, including MDR development through protein transfer and genetic reprogramming. Autophagy, a process balancing cellular resources, also influences MDR. The intersection of EVs and autophagy further complicates the understanding of MDR. Both extracellular (exosomes, microvesicles) and intracellular (autophagic) vesicles contribute to therapy resistance by regulating the tumor microenvironment, facilitating cell communication, and modulating drug processing. While much is known about these pathways, there is still a need to explore their potential for predicting treatment responses and understanding tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5249ba1d82fab28ebb5cc330d8e07a3024195f" target='_blank'>
              The Role of Intracellular and Extracellular Vesicles in the Development of Therapy Resistance in Cancer.
              </a>
            </td>
          <td>
            Magdalena Wilczak, Magdalena Surman, M. Przybyło
          </td>
          <td>2024-08-07</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="All routine clinical treatments for colorectal cancer include 5-fluorouracil (5-FU), which cannot counteract recurrence and metastases formation. As the pyrimidine analog 5-FU can impact multiple pathways including both DNA and RNA metabolism, studying its mode of actions could lead to improved therapies. Using a dedicated reporter system for lineage-tracing and deep translatome profiling we demonstrate that 5-FU causes some colorectal cancer cells to tolerate the drug, due to a durable translational reprogramming that sustains cell plasticity. This period of drug tolerance coincides with specific translational activation of genes coding for proteins with major pro-tumoral functions. We unravel a major unexpected translational overexpression of the pro-inflammatory and pro-tumoral IL-8 cytokine, alongside other anti-apoptotic, senescence-associated secretory phenotype and cancer-related senescence phenotype genes. Given the adverse prognostic implications of elevated IL-8 levels across various cancers, our findings suggest IL-8 targeting could counteract 5-FU resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3f76dde38afaccb213072b4b26b46e5ff73bd2b" target='_blank'>
              Translational control of cell plasticity drives 5-FU tolerance
              </a>
            </td>
          <td>
            Mounira Chalabi-Dchar, Olivia Villeronce, J. Ripoll, A. Vincent, T. Fenouil, Rita Khoueiry, Jérôme Kucharczak, Laura Jentschel, Frédéric Catez, Arnaud M Vigneron, Julie Tréguier, Céline Mandier, Céline Bouclier, Jihane Vitre, Louise Lagerqvist, Armelle Choquet, Z. Herceg, C. Machon, Jérôme Guitton, Alexandre David, Eric Solary, David Bernard, Nadine Martin, Eric Rivals, N. D. Venezia, Julie Pannequin, J- J. Diaz
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Cellular responses to environmental stimuli are typically thought to be governed by genetically encoded programs. We demonstrate that melanoma cells can form and maintain cellular memories during the acquisition of therapy resistance that exhibit characteristics of cellular learning and are dependent on the transcription factor AP-1. We show that cells exposed to a low dose of therapy adapt to become resistant to a high dose, demonstrating that resistance was not purely selective. The application of therapy itself results in the encoding of transient gene expression into cellular memory and that this encoding occurs for both transiently induced and probabilistically arising expression. Chromatin accessibility showed concomitant persistence. A two-color AP-1 reporter system showed that these memories are encoded in cis, constituting an example of activating cis epigenetics. Our findings establish the formation and maintenance of cellular memories as a critical aspect of gene regulation during the development of therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a15dd395d0e2a350d250357de4baa259bd2f20" target='_blank'>
              AP-1 Mediates Cellular Adaptation and Memory Formation During Therapy Resistance
              </a>
            </td>
          <td>
            Jingxin Li, Pavithran T. Ravindran, Aoife O’Farrell, Gianna T. Busch, Ryan H. Boe, Zijian Niu, Sean Woo, Margaret C. Dunagin, Naveen Jain, Yogesh Goyal, K. Sarma, M. Herlyn, Arjun Raj
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>113</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single-nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosome 20q or 1q, both known for conferring an in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings showed higher loads of both CNVs and SNVs at later passages, which were due to the cumulative acquisition of mutations over a longer time in culture, and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs followed the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid of cancer-associated mutations. This suggests that recurrent chromosomal abnormalities are potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36cc208bebf01a60a96b6bcb671a8cdcf9d8bbf6" target='_blank'>
              De Novo Cancer Mutations Frequently Associate with Recurrent Chromosomal Abnormalities during Long-Term Human Pluripotent Stem Cell Culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huyghebaert, M. Regin, Mai Chi Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Connecting genetic variation to phenotypes and understanding the underlying biological mechanisms has been a fundamental goal of biological genetics. Here, we used the association analysis to identify a Vip3Aa resistance-associated genomic region in a strain of fall armyworm, JC-R, which exhibits >5000-fold resistance to the Bt toxin Vip3Aa. However, through various analytical approaches and fine-scale mapping across different populations, we demonstrated that this genomic region exhibits strong genetic linkage. The chromosome-level genome of JC-R and its parent strain JC-S were assembled, and extensive structural variations in the linkage regions were identified, which could be responsible for maintaining the linkage. To identify the causal variation within this linked region, a chromosome fragment stepwise knockout strategy based on CRISPR/Cas9 was developed. By crossing with the resistant strain and phenotyping segregating offspring on Vip3Aa-containing diet, we identified a chromosomal segment, KO8, containing the resistant gene. Subsequently, we conducted a comprehensive analysis of the variations in the KO8 region using multi-omics approaches, including genomic data, RNA-seq, proteomic, PacBio long read Iso-seq, and phosphoproteomic data. This analysis identified multiple variations in the chitin synthase gene CHS2, including amino acid substitution, alternative splicing, and changes in phosphorylation sites. After knocking out the CHS2, larvae exhibited over 6777-fold resistance to Vip3Aa. These results demonstrate that the chromosome fragment stepwise knockout strategy is a viable approach for studying complex genomic regions, and highlight the value of comprehensive analysis of genetic variations using multi-omics data. The identified candidate gene could potentially advance monitoring and management of pest resistance to Vip3Aa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca50d6fcbaf49cb004ae82c9568f261b121333e" target='_blank'>
              CRISPR-mediated chromosome deletion facilitates genetic mapping of Vip3Aa resistance gene within complex genomic region in an invasive global pest
              </a>
            </td>
          <td>
            M. Jin, Yinxue Shan, Yan Peng, Senlin Chen, Xuanhe Zhou, Kaiyu Liu, Yutao Xiao
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eaccde1d6ccb6dacff234a0f6ddffc263c341a3" target='_blank'>
              Targeting oncogene-induced cellular plasticity for tumor therapy
              </a>
            </td>
          <td>
            Bin Li, Lingling Zheng, Jianhua Yang, Lianghu Qu
          </td>
          <td>2024-07-15</td>
          <td>Advanced Biotechnology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Directed evolution is an efficient strategy to steer protein function to either understand specific biological properties or develop new biotechnology tools. Currently available methods for targeted mutagenesis in human cells rely on deaminases which can only modify specific bases, limiting the region of sequence space explored during evolution. By leveraging CRISPR-Cas9 coupled with an error-prone variant of human DNA polymerase λ, here we developed CRISPR-λ, an unbiased mutagenesis tool for directed evolution in human cells. We evaluated CRISPR-λ by reverting the fluorescence of a mutated EGFP and characterized it using ultra-deep sequencing. The mutagenic activity of CRISPR-λ spans 36-46 nucleotides from the target site, with a mutation frequency as high as 1.4e-4 substitutions per base and with no bias for specific nucleotide substitutions. The versatility of CRISPR-λ extends beyond base substitution, enabling modifications of the target gene through insertions and deletions, thereby broadening its potential for genetic diversification. We validated the efficacy of CRISPR-λ in directed evolution approaches by functionally reverting a mutated blasticidin resistance gene. Furthermore, we demonstrated the sequence diversification power of CRISPR-λ by steering the syncytia formation activity of the SARS-CoV-2 Spike envelope protein in cultured cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d747466dd7f2edf4242403524fc8d70fe775ed9" target='_blank'>
              DNA polymerase λ-driven targeted mutagenesis for directed evolution in human cells
              </a>
            </td>
          <td>
            Davide Aiello, Matteo Ciciani, Federica Marelli, Marta Stancampiano, Veronica De Sanctis, R. Bertorelli, E. Kheir, G. Maule, A. Cereseto, Daniele Arosio
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ae5740675a47153265ae470439ce458fde35c02" target='_blank'>
              Single-cell sensor analyses reveal signaling programs enabling Ras-G12C drug resistance.
              </a>
            </td>
          <td>
            Jason Z Zhang, Shao-En Ong, D. Baker, D. Maly
          </td>
          <td>2024-08-05</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23cf92719cd49496400fba609df94fa13b009008" target='_blank'>
              Precision medicine in AML: overcoming resistance.
              </a>
            </td>
          <td>
            Samuel Urrutia, Koichi Takahashi
          </td>
          <td>2024-08-01</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e387009a4ed97bf637b9dd6a3591f151035e10" target='_blank'>
              Insights into gemcitabine resistance in pancreatic cancer: association with metabolic reprogramming and TP53 pathogenicity in patient derived xenografts
              </a>
            </td>
          <td>
            Mariam M. Konaté, Julia Krushkal, Ming-Chung Li, Li Chen, Yuri Kotliarov, Alida Palmisano, Rini Pauly, Qian Xie, P. M. Williams, Lisa M. McShane, Yingdong Zhao
          </td>
          <td>2024-08-05</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In this review, we revisit the pivotal role of fibroblast growth factor receptor 3 (FGFR3) in bladder cancer (BLCA), underscoring its prevalence in both non-muscle-invasive and muscle-invasive forms of the disease. FGFR3 mutations in up to half of BLCAs play a well-established role in tumorigenesis, shaping distinct tumor initiation patterns and impacting the tumor microenvironment (TME). Emphasizing the importance of considering epithelial-mesenchymal transition profile and TME status, we revisit their relevance in predicting responses to immune checkpoint inhibitors in FGFR3-mutated BLCAs. This writing highlights the initially promising yet transient efficacy of the FGFR inhibitor Erdafitinib on FGFR3-mutated BLCA, stressing the pressing need to unravel resistance mechanisms and identify co-targets for future combinatorial studies. A thorough analysis of recent preclinical and clinical evidence reveals resistance mechanisms, including secondary mutations, epigenetic alterations in pathway effectors, phenotypic heterogeneity, and population-specific variations within FGFR3 mutational status. Lastly, we discuss the potential of combinatorial treatments and concepts like synthetic lethality for discovering more effective targeted therapies against FGFR3-mutated BLCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d32b54c415deb1782db39b2832fb5448db0b31" target='_blank'>
              FGFR3 alterations in bladder cancer: Sensitivity and resistance to targeted therapies.
              </a>
            </td>
          <td>
            Maxim Noeraparast, Katarina Krajina, Renate Pichler, Dora Niedersüß-Beke, S. Shariat, Viktor Grünwald, S. Ahyai, Martin Pichler
          </td>
          <td>2024-08-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Next-generation sequencing (NGS) technology has revolutionized cancer research and treatment by enabling comprehensive analysis of genetic mutations, alterations, and expression profiles. It allows for the identification of cancer-driving mutations, helping in the development of targeted therapies. NGS provides detailed insights into tumor heterogeneity, resistance mechanisms, and clonal evolution. Its high-throughput capacity facilitates large-scale studies, improving our understanding of cancer genomics. By enabling personalized treatment plans based on individual genetic profiles, NGS holds promise for more effective and tailored cancer therapies. Early reviews on cancer genomics often lacked comprehensive coverage of emerging technologies. They missed in-depth analysis of NGS advancements, their impact on cancer research, and clinical applications. The review addresses this gap by reviving a thorough examination of NGS methods, their role in identifying genetic mutations, and their potential in personalized cancer treatment, thus providing essential insights into the evolving landscape of cancer genomics. The article covers the advancements in technology and bioinformatic approaches for NGS data analysis. It delves into NGS applications in research and diagnostics, particularly for solid cancer diagnosis. The review highlights specific cancer types, including hereditary breast cancer, melanoma, prostate cancer, thyroid cancer, lung cancer, and colorectal cancer. It explores NGS contribution in understanding the genetic basis of these cancers and its potential for enhancing personalized diagnosis and treatment strategies. This review rectifies early lacunas by providing a comprehensive and updated examination of NGS technology, addressing gaps in previous analyses and emphasizes bioinformatic approaches for NGS data analysis, crucial for interpreting vast genomic data accurately. The review meets the current need for a thorough understanding of NGS’s role in personalized cancer treatment and research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fa1bb9c8e82b88cec555956d178d8b56b6683c" target='_blank'>
              Next-generation sequencing technology in cancer
              </a>
            </td>
          <td>
            P. Krubaa, Dr. Anand Mohan Jha, Prof Dr.Ammar A.Razzak Mahmood, Dr. Anil Kumar, D. Abraham
          </td>
          <td>2024-07-18</td>
          <td>International Journal of Trends in OncoScience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to sustain rapid proliferation, resist cell death, and adapt to varying microenvironmental conditions. This review elucidates the key mechanisms underlying metabolic reprogramming in cancer cells, including alterations in glucose metabolism, glutamine addiction, and lipid biosynthesis. A central feature of cancer metabolism is the Warburg effect, where cancer cells preferentially utilize aerobic glycolysis over oxidative phosphorylation, even in the presence of oxygen. This metabolic shift is driven by oncogenes and tumor suppressor genes, such as MYC and TP53, which modulate the expression and activity of enzymes involved in glycolysis and mitochondrial function. Additionally, cancer cells exhibit increased glutaminolysis, relying on glutamine as a carbon and nitrogen source to support anabolic processes and redox balance. Lipid metabolism is also reprogrammed, with enhanced de novo lipogenesis supplying membrane components and signaling molecules critical for tumor growth and survival. Understanding these metabolic alterations provides a basis for developing targeted therapies. Several therapeutic strategies have emerged, including inhibitors of key metabolic enzymes, such as hexokinase 2 (HK2), pyruvate kinase M2 (PKM2), and glutaminase. Metabolic interventions can disrupt the metabolic flexibility of cancer cells, sensitizing them to conventional therapies and overcoming resistance mechanisms. This review highlights the potential of metabolic targets in cancer treatment and emphasizes the need for personalized approaches considering tumor-specific metabolic profiles. Future research should focus on identifying biomarkers of metabolic vulnerabilities and developing combination therapies that exploit the intricate metabolic dependencies of cancer cells. Ultimately, targeting metabolic reprogramming holds promise for improving cancer prognosis and achieving more effective and durable therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d015236b22d21b366aae600c9f9e8d4b15a3c83" target='_blank'>
              A review of metabolic reprogramming in cancer cells: Mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Kelly Osayi Otakhor, Elizabeth O. Soladoye
          </td>
          <td>2024-07-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="New and improved drugs are required for the treatment and ultimate eradication of malaria. The efficacy of front-line therapies is now threatened by emerging drug resistance; thus, new tools to support the development of drugs with a lower propensity for resistance are needed. Here, we describe the development of a RESistance Mapping And Profiling (ResMAP) platform for the identification of resistance-conferring mutations in Plasmodium drug targets. Proof-of-concept studies focused on interrogating the antimalarial drug target, Plasmodium falciparum lysyl tRNA synthetase (PfKRS). Saturation mutagenesis was used to construct a plasmid library encoding all conceivable mutations within a 20-residue span at the base of the PfKRS ATP-binding pocket. The superior transfection efficiency of Plasmodium knowlesi was exploited to generate a high coverage parasite library expressing PfKRS bearing all possible amino acid changes within this region of the enzyme. The selection of the library with PfKRS inhibitors, cladosporin and DDD01510706, successfully identified multiple resistance-conferring substitutions. Genetic validation of a subset of these mutations confirmed their direct role in resistance, with computational modeling used to dissect the structural basis of resistance. The application of ResMAP to inform the development of resistance-resilient antimalarials of the future is discussed.


IMPORTANCE
An increase in treatment failures for malaria highlights an urgent need for new tools to understand and minimize the spread of drug resistance. We describe the development of a RESistance Mapping And Profiling (ResMAP) platform for the identification of resistance-conferring mutations in Plasmodium spp, the causative agent of malaria. Saturation mutagenesis was used to generate a mutation library containing all conceivable mutations for a region of the antimalarial-binding site of a promising drug target, Plasmodium falciparum lysyl tRNA synthetase (PfKRS). Screening of this high-coverage library with characterized PfKRS inhibitors revealed multiple resistance-conferring substitutions including several clinically relevant mutations. Genetic validation of these mutations confirmed resistance of up to 100-fold and computational modeling dissected their role in drug resistance. We discuss potential applications of this data including the potential to design compounds that can bypass the most serious resistance mutations and future resistance surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d7dc04ef86759e728eca0280100918ebb17c149" target='_blank'>
              ResMAP-a saturation mutagenesis platform enabling parallel profiling of target-specific resistance-conferring mutations in Plasmodium.
              </a>
            </td>
          <td>
            Richard J. Wall, Stuart A MacGowan, Irene Hallyburton, Aisha J Syed, Sowmya Ajay Castro, Gourav Dey, Rachel Milne, Stephen Patterson, J. Phelan, Natalie Wiedemar, Susan Wyllie
          </td>
          <td>2024-08-27</td>
          <td>mBio</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A. Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M. As Sobeai
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e223c600817ad1cd6259f50a38399720abf1f7d" target='_blank'>
              Drug tolerant persister cell plasticity in cancer: A revolutionary strategy for more effective anticancer therapies
              </a>
            </td>
          <td>
            Jun He, Zejing Qiu, Jingjing Fan, Xiaohong Xie, Qinsong Sheng, Xinbing Sui
          </td>
          <td>2024-08-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Lung cancer patients treated with targeted therapies frequently respond well but invariably relapse due to the development of drug resistance. Drug resistance is in part mediated by a subset of cancer cells termed “drug-tolerant persisters” (DTPs), which enter a dormant, slow-cycling state that enables them to survive drug exposure. DTPs also exhibit stem cell-like characteristics, broad epigenetic reprogramming, altered metabolism, and a mutagenic phenotype mediated by adaptive mutability. While several studies have characterised the transcriptional changes that lead to the altered phenotypes exhibited in DTPs, these studies have focused predominantly on protein coding changes. As long non-coding RNAs (lncRNAs) are also implicated in the phenotypes altered in DTPs, it is likely that they play a role in the biology of drug tolerance. In this review, we outline how lncRNAs may contribute to the key characteristics of DTPs, their potential roles in tolerance to targeted therapies, and the emergence of genetic resistance in lung adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83141f0410de3ec7e33ac04641da48080cc097e5" target='_blank'>
              The Potential Links between lncRNAs and Drug Tolerance in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            William J. H. Davis, Catherine J. Drummond, Sarah D. Diermeier, Glen Reid
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="INTRODUCTION
Colon cancer, ranked as the fourth leading global cause of cancer death, exhibits a complex progression marked by genetic variations. Over the past decade, the utilization of diverse CRISPR systems has propelled accelerated research into colorectal cancer (CRC) treatment.


AREAS COVERED
CRISPR/Cas9, a key player in this research, identifies new oncogenes, tumor suppressor genes (TSGs), and drug-resistance genes. Additionally, it facilitates the construction of experimental models, conducts genome-wide library screening, and develops new therapeutic targets, especially for targeted knockout in vivo or molecular targeted drug delivery, contributing to personalized treatments and significantly enhancing the care of colon cancer patients. In this review, we provide insights into the mechanism of the CRISPR/Cas9 system, offering a comprehensive exploration of its applications in CRC, spanning screening, modeling, gene functions, diagnosis, and gene therapy. While acknowledging its transformative potential, the article  highlights the challenges and limitations of CRISPR systems.


EXPERT OPINION
The application of CRISPR/Cas9 in CRC research provides a promising avenue for personalized treatments. Its potential for identifying key genes and enabling experimental models and genome-wide screening enhances patient care. This review underscores the significance of CRISPR-Cas9 gene editing technology across basic research, diagnosis, and the treatment landscape of colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e832452c0de49427c039c9545ac3618de43c5948" target='_blank'>
              Recent advances in CRISPR-Cas systems for colorectal cancer research and therapeutics.
              </a>
            </td>
          <td>
            Saeideh Khorshid Sokhangouy, Farzaneh Alizadeh, Malihe Lotfi, Samaneh Sharif, Atefeh Ashouri, Yasamin Yoosefi, Saeed Bozorg Qomi, M. Abbaszadegan
          </td>
          <td>2024-08-02</td>
          <td>Expert review of molecular diagnostics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A. Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Using five complementary short- and long-read sequencing technologies, we phased and assembled >95% of each diploid human genome in a four-generation, 28-member family (CEPH 1463) allowing us to systematically assess de novo mutations (DNMs) and recombination. From this family, we estimate an average of 192 DNMs per generation, including 75.5 de novo single-nucleotide variants (SNVs), 7.4 non-tandem repeat indels, 79.6 de novo indels or structural variants (SVs) originating from tandem repeats, 7.7 centromeric de novo SVs and SNVs, and 12.4 de novo Y chromosome events per generation. STRs and VNTRs are the most mutable with 32 loci exhibiting recurrent mutation through the generations. We accurately assemble 288 centromeres and six Y chromosomes across the generations, documenting de novo SVs, and demonstrate that the DNM rate varies by an order of magnitude depending on repeat content, length, and sequence identity. We show a strong paternal bias (75-81%) for all forms of germline DNM, yet we estimate that 17% of de novo SNVs are postzygotic in origin with no paternal bias. We place all this variation in the context of a high-resolution recombination map (∼3.5 kbp breakpoint resolution). We observe a strong maternal recombination bias (1.36 maternal:paternal ratio) with a consistent reduction in the number of crossovers with increasing paternal (r=0.85) and maternal (r=0.65) age. However, we observe no correlation between meiotic crossover locations and de novo SVs, arguing against non-allelic homologous recombination as a predominant mechanism. The use of multiple orthogonal technologies, near-telomere-to-telomere phased genome assemblies, and a multi-generation family to assess transmission has created the most comprehensive, publicly available “truth set” of all classes of genomic variants. The resource can be used to test and benchmark new algorithms and technologies to understand the most fundamental processes underlying human genetic variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcb8df79fd9bfc629f05070b595811f041bfe33" target='_blank'>
              A familial, telomere-to-telomere reference for human de novo mutation and recombination from a four-generation pedigree
              </a>
            </td>
          <td>
            David Porubsky, Harriet Dashnow, T. Sasani, Glennis A. Logsdon, P. Hallast, Michelle D. Noyes, Z. Kronenberg, T. Mokveld, Nidhi Koundinya, Cillian Nolan, C. Steely, A. Guarracino, E. Dolzhenko, William T. Harvey, W. Rowell, K. Grigorev, T. J. Nicholas, Keisuke K. Oshima, Jiadong Lin, Peter Ebert, W. Watkins, Tiffany Y. Leung, Vincent C. T. Hanlon, Sean Mcgee, Brent S. Pedersen, Michael E. Goldberg, Hannah C. Happ, Hyeonsoo Jeong, Katherine M. Munson, Kendra Hoekzema, Daniel D. Chan, Yanni Wang, Jordan Knuth, Gage H. Garcia, Cairbre Fanslow, Christine Lambert, Charles Lee, Josh Smith, Shawn Levy, C. Mason, Erik Garrison, Peter Lansdorp, D. Neklason, L. Jorde, Aaron R. Quinlan, M. Eberle, Evan E. Eichler
          </td>
          <td>2024-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="A multitude of exogenous and endogenous processes have the potential to result in DNA damage. While the repair mechanisms are typically capable of correcting this damage, errors in the repair process can result in mutations. The findings of research conducted in 2012 indicate that mutations do not occur randomly but rather follow specific patterns that can be attributed to known or inferred mutational processes. The process of mutational signature analysis allows for the inference of the predominant mutational process for a given cancer sample, with significant potential for clinical applications. A deeper comprehension of these mutational signatures in CRC could facilitate enhanced prevention strategies, facilitate the comprehension of genotoxic drug activity, predict responses to personalized treatments, and, in the future, inform the development of targeted therapies in the context of precision oncology. The efforts of numerous researchers have led to the identification of several mutational signatures, which can be categorized into different mutational signature references. In CRC, distinct mutational signatures are identified as correlating with mismatch repair deficiency, polymerase mutations, and chemotherapy treatment. In this context, a mutational signature analysis offers considerable potential for enhancing minimal residual disease (MRD) tests in stage II (high-risk) and stage III CRC post-surgery, stratifying CRC based on the impacts of genetic and epigenetic alterations for precision oncology, identifying potential therapeutic vulnerabilities, and evaluating drug efficacy and guiding therapy, as illustrated in a proof-of-concept clinical trial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9f4817ac8c40910eefa1198ffb3e184664ea3d" target='_blank'>
              Mutational Signatures in Colorectal Cancer: Translational Insights, Clinical Applications, and Limitations
              </a>
            </td>
          <td>
            G. Crisafulli
          </td>
          <td>2024-08-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8868db406f4d297083b3336bdacf7e70f944e3" target='_blank'>
              Characterize molecular signatures and establish a prognostic signature of gastric cancer by integrating single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Zhiwei Wang, Zhiyan Weng, Luping Lin, Xianyi Wu, Wenju Liu, Yong Zhuang, J. Jian, Changhua Zhuo
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Ovarian cancer ecosystems are exceedingly complex, consisting of a high heterogeneity of cancer cells. Development of drugs such as poly ADP‐ribose polymerase (PARP) inhibitors, targeted therapies and immunotherapies offer more options for sequential or combined treatments. Nevertheless, mortality in metastatic ovarian cancer patients remains high because cancer cells consistently develop resistance to single and combination therapies, urging a need for treatment designs that target the evolvability of cancer cells. The evolutionary dynamics that lead to resistance emerge from the complex tumour microenvironment, the heterogeneous populations, and the individual cancer cell's plasticity. We propose that successful management of ovarian cancer requires consideration of the ecological and evolutionary dynamics of the disease. Here, we review current options and challenges in ovarian cancer treatment and discuss principles of tumour evolution. We conclude by proposing evolutionarily designed strategies for ovarian cancer, with the goal of integrating such principles with longitudinal, quantitative data to improve the treatment design and management of drug resistance. Key points/Highlights Tumours are ecosystems in which cancer and non‐cancer cells interact and evolve in complex and dynamic ways. Conventional therapies for ovarian cancer inevitably lead to the development of resistance because they fail to consider tumours’ heterogeneity and cellular plasticity. Eco‐evolutionarily designed therapies should consider cancer cell plasticity and patient‐specific characteristics to improve clinical outcome and prevent relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f584cd6ae71db959a312c8365ae2a9ada15cf5" target='_blank'>
              Ecological and evolutionary dynamics to design and improve ovarian cancer treatment
              </a>
            </td>
          <td>
            , Monica Alexander, Julia Gattozzi, Marilyn Day, Elayna Kirsch, Narges Tafreshi, Raafat Chalar, Soraya Rahni, Gabrielle Gossner, William Burke, Mehdi Damaghi
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/699986230662d4afba97aea6b14e6128a02adbf7" target='_blank'>
              Molecular mechanisms of DNA lesion and repair during antibody somatic hypermutation.
              </a>
            </td>
          <td>
            Qian Hao, Jinfeng Li, Leng-Siew Yeap
          </td>
          <td>2024-07-23</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Transcription and splicing errors in cancer cells generate erroneous transcripts. Since erroneous transcripts are degraded by the nonsense-mediated mRNA decay (NMD) pathway, whether they are junk or could be functional has been overlooked and understudied. We addressed this question by first performing a pan-cancer analysis and identified substantial erroneous noncoding transcripts (ENT) in cancers. Given that RNA/DNA binding domains (DBD) were predicted in ENTs, we deleted predicted DBDs in multiple ENTs in multiple cell lines, with RNA-sequencing and cell experiments before and after DBD deletion. DBD deletion caused significantly changed expression of ENTs’ target genes (whose promoter regions contain ENTs’ DNA binding sites, DBS) and changed cell migration and proliferation ability, indicating that many ENTs can transcriptionally regulate genes. Tightly coupled data analysis and experiments reveal that ENTs’ functions are highly cancer- and cellular-context specific, making ENTs a new class of safe and specific targets for noncoding RNA-based cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e45af8d1c765a560ff555893aa2ed200e47f3c" target='_blank'>
              Many erroneous noncoding transcripts in cancer cells can highly specifically regulate cancer-related genes and pathways
              </a>
            </td>
          <td>
            Sha He, Wei Xiong, Jianping Huo, Jie Lin, Jianmin Li, Hao Zhu
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Single nucleotide variants (SNVs) contribute to cancer by altering the coding and the non-coding regions of the genome. Connecting SNVs to transcriptomic and epigenetic changes at the single-cell level remains challenging. To enable studies of rare cell populations harboring specific point mutations, we developed STAR-FACS, Specific-To-Allele PCR-FACS, to sort cells based on genomic allele alterations. We show that STAR-FACS can separate cells based on TERT promoter mutation status and is compatible with bulk and single-cell transcriptomic and epigenetic profiling. We demonstrate that glioblastoma cell lines derived from the same tumor but harboring distinct TERT promoter SNVs have different transcriptional programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f2ef09f49e055964d96a3768d156e4cf968931" target='_blank'>
              Cell sorting based on single nucleotide variation enables characterization of mutation-dependent transcriptome and chromatin states
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, Oszkar Szentirmai, M. Janiszewska
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Methods to engineer the genomes of human cells for therapeutic intervention continue to advance at a remarkable pace. Chimeric antigen receptor-engineered T lymphocytes have pioneered the way for these therapies, initially beginning with insertions of chimeric antigen receptor transgenes into T-cell genomes using classical gene therapy vectors. The broad use of clustered regularly interspaced short palindromic repeats (CRISPR)-based technologies to edit endogenous genes has now opened the door to a new era of precision medicine. To add complexity, many engineered cellular therapies under development integrate gene therapy with genome editing to introduce novel biological functions and enhance therapeutic efficacy. Here, we review the current state of scientific, translational, and regulatory oversight of gene-edited cell products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2d1ab3d9838377779d6ccf7a7d2e82b715e992" target='_blank'>
              Regulatory Considerations for Genome-Edited T-cell Therapies.
              </a>
            </td>
          <td>
            J. Jadlowsky, Ju-Fang Chang, DH Spencer, John M Warrington, Bruce L. Levine, Carl H. June, J. Fraietta, Nathan Singh
          </td>
          <td>2024-07-17</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The intrinsic oncogenic mechanisms and properties of the tumor microenvironment (TME) have been extensively investigated. Primary features of the TME include metabolic reprogramming, hypoxia, chronic inflammation, and tumor immunosuppression. Previous studies suggest that senescence-associated secretory phenotypes that mediate intercellular information exchange play a role in the dynamic evolution of the TME. Specifically, hypoxic adaptation, metabolic dysregulation, and phenotypic shifts in immune cells regulated by cellular senescence synergistically contribute to the development of an immunosuppressive microenvironment and chronic inflammation, thereby promoting the progression of tumor events. This review provides a comprehensive summary of the processes by which cellular senescence regulates the dynamic evolution of the tumor-adapted TME, with focus on the complex mechanisms underlying the relationship between senescence and changes in the biological functions of tumor cells. The available findings suggest that components of the TME collectively contribute to the progression of tumor events. The potential applications and challenges of targeted cellular senescence-based and combination therapies in clinical settings are further discussed within the context of advancing cellular senescence-related research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f69d28c4893741d69a2e81f7df756fd68f976626" target='_blank'>
              Cellular senescence and metabolic reprogramming: Unraveling the intricate crosstalk in the immunosuppressive tumor microenvironment.
              </a>
            </td>
          <td>
            Fusheng Zhang, Junchen Guo, Shengmiao Yu, Youwei Zheng, Meiqi Duan, Liang Zhao, Yihan Wang, Zhi Yang, Xiaofeng Jiang
          </td>
          <td>2024-07-12</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome flexibility is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and flexibility determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin a more prone to form SVs than those located in euchromatin, highlighting that genome stability and flexibility differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA Damage Response (DDR), ATM and ATR, suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure UV-B, UV-C and protons induced the Microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a78aa5b539dcd98a3297c9111dc41b5d43d56b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The DNA modifications, 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), represent powerful epigenetic regulators of temporal and spatial gene expression. Yet, how the cooperation of these genome-wide, epigenetic marks determine unique transcriptional signatures across different brain cell populations is unclear. Here we applied Nanopore sequencing of native DNA to obtain a complete, genome-wide, single-base resolution atlas of 5mC and 5hmC modifications in neurons, astrocytes and microglia in the mouse cortex (99% genome coverage, 40 million CpG sites). In tandem with RNA sequencing, analysis of 5mC and 5hmC patterns across cell types reveals astrocytes drive uniquely high brain 5hmC levels and support two decades of research regarding methylation patterns, gene expression and alternative splicing, benchmarking this resource. As such, we provide the most comprehensive DNA methylation data in mouse brain as an interactive, online tool (NAM-Me, https://olsenlab.shinyapps.io/NAMME/) to serve as a resource dataset for those interested in the methylome landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60076a55629be60f22fe1afccd00156f5bc051e7" target='_blank'>
              Decoding the Epigenetic Landscape: Insights into 5mC and 5hmC Patterns in Mouse Cortical Cell Types
              </a>
            </td>
          <td>
            Xiaoran Wei, Jiangtao Li, Zuolin Cheng, Songtao Wei, Guoqiang Yu, Michelle L. Olsen
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d65bf85005c8a5a2fb214830aeec58d6f7820d7" target='_blank'>
              Key genes and molecular mechanisms related to Paclitaxel Resistance
              </a>
            </td>
          <td>
            Adel I Alalawy
          </td>
          <td>2024-07-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions.We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters.Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations.Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, Weisen Fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af8a5e84090feb0d13451624ef0b08cb6aa245a3" target='_blank'>
              Epigenetic regulation of DNA repair gene program by Hippo/YAP1-TET1 axis mediates sorafenib resistance in HCC
              </a>
            </td>
          <td>
            Chunli Mo, Weixin You, Yipeng Rao, Zhenping Lin, Shuai Wang, Ting He, Huanming Shen, Xun Li, Rui Zhang, Boan Li
          </td>
          <td>2024-07-05</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/435232aea775847959662a12da4da5a9a69c73c7" target='_blank'>
              A whole-genome CRISPR screen identifies the spindle accessory checkpoint as a locus of nab-paclitaxel resistance in a pancreatic cancer cell line
              </a>
            </td>
          <td>
            Priya Mondal, George Alyateem, Allison V. Mitchell, Michael M. Gottesman
          </td>
          <td>2024-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acquired chemoresistance remains a significant challenge in the clinics as most of the treated cancers eventually emerge as hard-to-treat phenotypes. Therefore, identifying chemoresistance targets is highly warranted to manage the disease better. In this study, we employed a label-free LC-MS/MS-based quantitative proteomics analysis to identify potential targets and signaling pathways underlying acquired chemoresistance in a sub-cell line (A549DR) derived from the parental lung adenocarcinoma cells (A549) treated with gradually increasing doses of doxorubicin (DOX). Our proteomics analysis identified 146 upregulated and 129 downregulated targets in A549DR cells. The KEGG pathway and Gene ontology (GO) analysis of differentially expressed upregulated and downregulated proteins showed that most abundant upregulated pathways were related to metabolic pathways, cellular senescence, cell cycle, and p53 signaling. Meanwhile, the downregulated pathways were related to spliceosome, nucleotide metabolism, DNA replication, nucleotide excision repair, and nuclear-cytoplasmic transport. Further, STRING analysis of upregulated biological processes showed a protein-protein interaction (PPI) between CDK1, AKT2, SRC, STAT1, HDAC1, FDXR, FDX1, NPC1, ALDH2, GPx1, CDK4, and B2M, proteins. The identified proteins in this study might be the potential therapeutic targets for mitigating DOX resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/974ec76bceeb820f708d654bf77904bfd435f4c2" target='_blank'>
              Identification of chemoresistance targets in doxorubicin-resistant lung adenocarcinoma cells using LC-MS/MS-based proteomics.
              </a>
            </td>
          <td>
            Nuzhat Bano, K. Kainat, Mohammad Imran Ansari, Anjali Pal, Sana Sarkar, Pradeep Kumar Sharma
          </td>
          <td>2024-08-05</td>
          <td>Journal of chemotherapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Understanding the evolutionary potential of mutations in gene regulatory networks is essential to furthering the study of evolution and development. However, in multicellular systems, genetic manipulation of regulatory networks in a targeted and high-throughput way remains challenging. In this study, we designed TF-High-Evolutionary (HighEvo), a transcription factor (TF) fused with a base editor (activation-induced deaminase), to continuously induce germline mutations at TF-binding sites across regulatory networks in Drosophila. Populations of flies expressing TF-HighEvo in their germlines accumulated mutations at rates an order of magnitude higher than natural populations. Importantly, these mutations accumulated around the targeted TF-binding sites across the genome, leading to distinct morphological phenotypes consistent with the developmental roles of the tagged TFs. As such, this TF-HighEvo method allows the interrogation of the mutational space of gene regulatory networks at scale and can serve as a powerful reagent for experimental evolution and genetic screens focused on the regulatory genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2f5224e39a793a9098b6100dcfef52f5acedf2" target='_blank'>
              TF-High-Evolutionary: In Vivo Mutagenesis of Gene Regulatory Networks for the Study of the Genetics and Evolution of the Drosophila Regulatory Genome
              </a>
            </td>
          <td>
            Xueying C. Li, Vani Srinivasan, Ian Laiker, Natalia Misunou, Nicolás Frankel, Luisa F. Pallares, Justin Crocker
          </td>
          <td>2024-08-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Currently, cancer is still a leading cause of human death globally. Tumor deterioration comprises multiple events including metastasis, therapeutic resistance and immune evasion, all of which are tightly related to the phenotypic plasticity especially epithelial–mesenchymal plasticity (EMP). Tumor cells with EMP are manifest in three states as epithelial–mesenchymal transition (EMT), partial EMT, and mesenchymal–epithelial transition, which orchestrate the phenotypic switch and heterogeneity of tumor cells via transcriptional regulation and a series of signaling pathways, including transforming growth factor‐β, Wnt/β‐catenin, and Notch. However, due to the complicated nature of EMP, the diverse process of EMP is still not fully understood. In this review, we systematically conclude the biological background, regulating mechanisms of EMP as well as the role of EMP in therapy response. We also summarize a range of small molecule inhibitors, immune‐related therapeutic approaches, and combination therapies that have been developed to target EMP for the outstanding role of EMP‐driven tumor deterioration. Additionally, we explore the potential technique for EMP‐based tumor mechanistic investigation and therapeutic research, which may burst vigorous prospects. Overall, we elucidate the multifaceted aspects of EMP in tumor progression and suggest a promising direction of cancer treatment based on targeting EMP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32fcd891c5d88b251baf8fc511031710e0fc1629" target='_blank'>
              Epithelial–mesenchymal plasticity in cancer: signaling pathways and therapeutic targets
              </a>
            </td>
          <td>
            Xiangpeng Wang, Xiao-yong Xue, Mingshi Pang, Liuchunyang Yu, Jinxiu Qian, Xiaoyu Li, Meng Tian, Aiping Lyu, Cheng Lu, Yuanyan Liu
          </td>
          <td>2024-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Single-cell CRISPR screens link genetic perturbations to transcriptional states, but high-throughput methods connecting these induced changes to their regulatory foundations are limited. Here we introduce Multiome Perturb-seq, extending single-cell CRISPR screens to simultaneously measure perturbation-induced changes in gene expression and chromatin accessibility. We apply Multiome Perturb-seq in a CRISPRi screen of 13 chromatin remodelers in human RPE-1 cells, achieving efficient assignment of sgRNA identities to single nuclei via an improved method for capturing barcode transcripts from nuclear RNA. We organize expression and accessibility measurements into coherent programs describing the integrated effects of perturbations on cell state, finding that ARID1A and SUZ12 knockdowns induce programs enriched for developmental features. Pseudotime analysis of perturbations connects accessibility changes to changes in gene expression, highlighting the value of multimodal profiling. Overall, our method provides a scalable and simply implemented system to dissect the regulatory logic underpinning cell state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6424220cb11bff7a646c5a12b16458018bda8e6" target='_blank'>
              Multiome Perturb-seq unlocks scalable discovery of integrated perturbation effects on the transcriptome and epigenome
              </a>
            </td>
          <td>
            Eli Metzner, Kaden M. Southard, Thomas M. Norman
          </td>
          <td>2024-07-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Melanoma being one of the most common and deadliest skin cancers, has been rising since the past decade. Patients at advanced stages of the disease have very poor prognoses, as opposed to at the earlier stages. Nowadays the standard-of-care of advanced melanoma is resection followed by immune checkpoint inhibition based immunotherapy. However, a substantial proportion of patients either do not respond or develop resistances. This underscores a need for novel approaches and therapeutic targets as well as a better understanding of the mechanisms of melanoma pathogenesis. Long non-coding RNAs (lncRNAs) comprise a poorly characterized class of functional players and promising targets in promoting malignancy. Certain lncRNAs have been identified to play integral roles in melanoma progression and drug resistances, however systematic screens to uncover novel functional lncRNAs are scarce. Here, we profile differentially expressed lncRNAs in patient derived short-term metastatic cultures and BRAF-MEK-inhibition resistant cells. We conduct a focused growth-related CRISPR-inhibition screen of overexpressed lncRNAs, validate and functionally characterize lncRNA hits with respect to cellular growth, invasive capacities and apoptosis in vitro as well as the transcriptomic impact of our lead candidate the novel lncRNA XLOC_030781. In sum, we extend the current knowledge of ncRNAs and their potential relevance on melanoma. Significance Previously considered as transcriptional noise, lncRNAs have emerged as novel players in regulating many cellular aspects in health and disease including melanoma. However, the number and as well as the extent of functional significance of most lncRNAs remains elusive. We provide a comprehensive strategy to identify functionally relevant lncRNAs in melanoma by combining expression profiling with CRISPR-inhibition growths screens lowering the experimental effort. We also provide a larger resource of differentially expressed lncRNAs with potential implications in melanoma growth and invasion. Our results broaden the characterized of lncRNAs as potential targets for future therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768f16ebbf2c3ec424c154b1baf19ad0e931615" target='_blank'>
              CRISPR-inhibition screen for lncRNAs linked to melanoma growth and metastasis
              </a>
            </td>
          <td>
            Stavroula Petroulia, K. Hockemeyer, Shashank Tiwari, Pietro Berico, Sama Shamloo, Seyedeh Elnaz Banijamali, Eleazar Vega-Saenz de Miera, Yixiao Gong, Palaniraja Thandapani, Eric Wang, Michael Schulz, A. Tsirigos, Iman Osman, Ioannis D. Aifantis, Jochen Imig
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Cancer treatment has long been fraught with challenges, including drug resistance, metastasis, and recurrence, making it one of the most difficult diseases to treat effectively. Traditional therapeutic approaches often fall short due to their inability to target cancer stem cells and the complex genetic and epigenetic landscape of tumors. In recent years, cancer immunotherapy has revolutionized the field, offering new hope and viable alternatives to conventional treatments. A particularly promising area of research focuses on non-coding RNAs (ncRNAs), especially long non-coding RNAs (lncRNAs), and their role in cancer resistance and the modulation of signaling pathways. To address these challenges, we performed a comprehensive review of recent studies on lncRNAs and their impact on cancer immunotherapy. Our review highlights the crucial roles that lncRNAs play in affecting both innate and adaptive immunity, thereby influencing the outcomes of cancer treatments. Key observations from our review indicate that lncRNAs can modify the tumor immune microenvironment, enhance immune cell infiltration, and regulate cytokine production, all of which contribute to tumor growth and resistance to therapies. These insights suggest that lncRNAs could serve as potential targets for precision medicine, opening up new avenues for developing more effective cancer immunotherapies. By compiling recent research on lncRNAs across various cancers, this review aims to shed light on their mechanisms within the tumor immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e39ed55d88fbecad6fec7ba9004faa9355e61712" target='_blank'>
              Exploring potential roles of long non-coding RNAs in cancer immunotherapy: a comprehensive review
              </a>
            </td>
          <td>
            A. Arshi, E. Mahmoudi, Farzaneh Raeisi, Masoud Dehghan Tezerjani, Elham Bahramian, Yeasin Ahmed, Chun Peng
          </td>
          <td>2024-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="A central challenge in the quest for precise gene regulation within mammalian cells is the development of regulatory networks that can achieve perfect adaptation-where outputs consistently return to a set baseline post-stimulus. Here, we present such a system that leverages the CRISPR activation (CRISPRa) and anti-CRISPR proteins as two antithetic elements to establish perfect adaptation in mammalian cells and dynamically regulate gene expression. We demonstrate that this system can maintain stable expression levels of target genes in the face of external perturbations, thus providing a robust platform for biological applications. The versatility of our system is further showcased through its integration with endogenous regulatory mechanisms in T cells, such as the NF-κB-mediated immune response, and its ability to program apoptosis responses for precise spatial and temporal control of cellular growth and death. This study not only advances our understanding of gene regulation in mammalian cells but also opens new avenues for therapeutic intervention, particularly in diseases characterized by dysregulated gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9bdcf363536b5c7b6585a226adb59f423a716ec" target='_blank'>
              CRISPR perfect adaptation for robust control of cellular immune and apoptotic responses.
              </a>
            </td>
          <td>
            Yichi Zhang, Shuyi Zhang
          </td>
          <td>2024-08-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Kirsten Rat Sarcoma (KRAS) is the most commonly mutated oncogene in colorectal carcinoma (CRC). We have previously reported the interactions between microsatellite instability (MSI), DNA promoter methylation, and gene expression. In this study, we looked for associations between KRAS mutation, gene expression, and methylation that may help with precision medicine. Genome-wide gene expression and DNA methylation were done in paired CRC tumor and surrounding healthy tissues. The results suggested that (a) the magnitude of dysregulation of many major gene pathways in CRC was significantly greater in patients with the KRAS mutation, (b) the up- and down-regulation of these dysregulated gene pathways could be correlated with the corresponding hypo- and hyper-methylation, and (c) the up-regulation of CDKN2A was more pronounced in tumors with the KRAS mutation. A recent cell line study showed that there were higher CDKN2A levels in 5-FU-resistant CRC cells and that these could be down-regulated by Villosol. Our findings suggest the possibility of a better response to anti-CDKN2A therapy with Villosol in KRAS-mutant CRC. Also, the more marked up-regulation of genes in the proteasome pathway in CRC tissue, especially with the KRAS mutation and MSI, may suggest a potential role of a proteasome inhibitor (bortezomib, carfilzomib, or ixazomib) in selected CRC patients if necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40b3e427194ea41aedea0dfcb0cf1b6c9f955b47" target='_blank'>
              Association of KRAS Mutation and Gene Pathways in Colorectal Carcinoma: A Transcriptome- and Methylome-Wide Study and Potential Implications for Therapy
              </a>
            </td>
          <td>
            F. Jasmine, Armando Almazan, Yuliia Khamkevych, Marc Bissonnette, Habibul Ahsan, M. Kibriya
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Antibiotic resistance is the leading cause of death globally, with a higher possibility of the emergence of highly resistant pathogens, leading to epidemics. Several antibiotic resistance mechanisms have been discovered, such as enhanced efflux of antibiotics, reduced influx of antibiotics, alteration of antibiotics or their targets, and adaptation to antibiotics. However, this mechanism cannot fully explain the development of antibiotic resistance because the genes associated with this mechanism have been elucidated. However, the factors governing their regulation are not yet fully understood. Recent studies have highlighted the epigenetic and epitranscriptomic roles of antibiotic resistance development-associated genes. Epigenetic modification is associated with DNA modification, whereas epitranscriptomic modification is associated with RNA modification to control gene expression by regulating various biological phenomena such as splicing, translation, and stability. Therefore, this review will focus on the discovery of epigenetic modifications, particularly by DNA methyltransferases, such as restriction-modification (R-M) systems associated with methyltransferases, orphan DNA methyltransferases, and nucleoid-associated proteins that contribute to the development of antibiotic resistance. This scrutinization further expands to epitranscriptomic modification of non-coding RNA, which has a role in the regulation of antibiotic resistance. Epitranscriptomic modification of ribosomal RNA (rRNA), which is a major target of antibiotics, has been well explored. while non-coding RNA such as cis and trans small non coding RNA, and riboswitches are poorly explored. This epigenetic and epitranscriptomic modification will help to understand the regulation of antibiotic resistance-associated genes, which will help to identify key regulators of antibiotic resistance, paving the way for new antibiotic discovery, leading to decreased antibiotic mortality globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b16c6a0ffde83445547b9ffde6ff971186ffa3" target='_blank'>
              A comprehensive review on epigenetic and epitranscriptomic-mediated regulation of antibiotic resistance
              </a>
            </td>
          <td>
            P. K. Giri, Shahil Alam, Madhav Dhakal
          </td>
          <td>2024-07-24</td>
          <td>F1000Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background. Lung adenocarcinoma (LUAD) is the most common subtype of non-small cell lung cancer (NSCLC) and frequently affects non-smokers, especially women. It is characterized by a complex genetic profile and interactions with its microenvironment, which contribute to its aggressive and adaptable nature. Early symptoms are often subtle, leading to late diagnoses. Treatment approaches have advanced with targeted therapies and immunotherapy supplementing traditional chemotherapy and radiation. Despite these advancements, the prognosis remains variable, highlighting the need for continued research into new treatment strategies to improve outcomes. Method. In this study, we employed Single-cell RNA Sequencing (scRNA-seq) to comprehensively analyze the impact of chemotherapy on lung adenocarcinoma at the individual cell level. By comparing before and after treatment samples, we assessed the differential expression of genes and pathways, revealing insights into how different cell types within the tumour respond to chemotherapy. This approach enabled us to pinpoint specific mechanisms of drug resistance and highlight potential therapeutic targets for overcoming these challenges. Results. Our analysis uncovered substantial changes in gene expression between primary tumour cells and metastatic cells following chemotherapy. Notably, we observed that 45 pathways were shared between the top 50 upregulated pathways in the primary tumour and the top 50 downregulated pathways in the metastatic tumour post-chemotherapy. Conversely, there was no overlap between the top 50 downregulated pathways in the primary tumour and the top 50 upregulated pathways in the metastatic tumour after chemotherapy. This suggests that chemotherapy effectively downregulated the major upregulated pathways but did not upregulate the key downregulated pathways in metastatic tumours. Conclusions. Integrating single-cell transcriptomics into LUAD research offers detailed insights into the tumour's response to chemotherapy and its interaction with the immune system. This approach enhances our understanding of LUAD and aids in developing targeted and effective treatments. Based on our analysis, we hypothesize that combining chemotherapy with drugs designed to upregulate the downregulated pathways in primary tumour cells could significantly enhance treatment efficacy and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9748c84cdcd0c997de55e9e674c83813950dc2" target='_blank'>
              Unveiling Chemotherapy's Impact on Lung Cancer through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            MD Saed Sayad, PhD Mark Hiatt, Mba MS Hazem Mustafa Md
          </td>
          <td>2024-07-10</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Therapy-induced acquired resistance limits the clinical effectiveness of mutation-specific RAS inhibitors in colorectal cancer. It is unknown whether broad-spectrum active-state RAS inhibitors meet similar limitations. Here, we identify and categorize mechanisms of resistance to the broad-spectrum active-state RAS inhibitor RMC-7977 in colorectal cancer cell lines. We found that KRAS-mutant colorectal cancer cell lines are universally sensitive to RMC-7977, inhibiting the RAS-RAF-MEK-ERK axis, halting proliferation and in some cases inducing apoptosis. To monitor KRAS downstream effector pathway activity, we developed a compartment-specific dual-color ERK activity reporter. RMC-7977 treatment reduced reporter activity. However, long-term dose escalation with RMC-7977 revealed multiple patterns of reporter reactivation in emerging resistant cell populations that correlated with phosphorylation states of compartment-specific ERK targets. Cells sorted for high, low, or cytoplasmic reporter activity exhibited distinct patterns of genomic mutations, phospho-protein, and transcriptional activities. Notably, all resistant subpopulations showed dynamic ERK regulation in the presence of the RAS inhibitor, unlike the parental sensitive cell lines. High levels of RAS downstream activities were observed in cells characterized by a KRAS Y71H resistance mutation. In contrast, RAS inhibitor-resistant populations with low, or cytoplasmic ERK reporter reactivation displayed different genetic alterations, among them RAF1 S257L and S259P mutations. Colorectal cancer cells resistant to RMC-7977 and harboring the RAF1 mutation specifically exhibited synergistic sensitivity to concurrent RAS and RAF inhibition. Our findings endorse reporter-assisted screening together with single-cell analyses as a powerful approach for dissecting the complex landscape of therapy resistance. The strategy offers opportunities to develop clinically relevant combinatorial treatments to counteract emergence of resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58f2dfe945929a08a6260606b0df62f40daad49b" target='_blank'>
              Reporter-based screening identifies RAS-RAF mutations as drivers of resistance to active-state RAS inhibition in colorectal cancer
              </a>
            </td>
          <td>
            Oleksandra Aust, Eric Blanc, Mareen Lüthen, Viola Hollek, Rosario Astaburuaga-García, B. Klinger, Alexandra Trinks, D. Beule, Björn Papke, David Horst, Nils Blüthgen, Christine Sers, Channing Der, M. Morkel
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma (PDAC) is one of the most aggressive tumors with poor prognosis and inadequate response to treatment, such as gemcitabine (Gem), the first-line chemotherapeutic drug. Understanding the molecular determinants that control drug resistance to Gem is critical to predict potentially responsive patients and improve the benefits of Gem therapy. Emerging evidence suggests that certain developmental pathways, such as Hippo signaling, are aberrated and play important roles in Gem resistance in cancers. Although Hippo signaling has been reported to play a role in chemoresistance in cancers, it has not been clarified which specific target gene(s) functionally mediates the effect. In the present study, we found that YAP serves as a potent barrier for the cellular sensitivity of PDAC cells to Gem. We then identified and characterized laminin subunit beta 3 (LAMB3) as a bona fide target of YAP-TEAD4 to amplify YAP signaling via a feedback loop. Such a YAP-LAMB3 axis is critical to induce epithelial-mesenchymal transition and mediate Gem resistance. Taken together, we uncovered that YAP-LAMB3 axis is an important regulator of Gem, thus providing potential therapeutic targets for overcoming Gem resistance in PDAC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8fe5c02b7e92cb644c9f2e0ec5d3fe826601d80" target='_blank'>
              YAP-LAMB3 axis dictates cellular resistance of pancreatic ductal adenocarcinoma cells to gemcitabine.
              </a>
            </td>
          <td>
            Yecheng Li, Xiaolong Wang, Hongpei Yu, Jinming Cao, Jia-ming Xie, Jinhong Zhou, Zhenyu Feng, Wei Chen
          </td>
          <td>2024-07-17</td>
          <td>Molecular carcinogenesis</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Meiotic recombination is a fundamental process that generates genetic diversity by creating new combinations of existing alleles. Although human crossovers have been studied at the pedigree, population and single-cell level, the more frequent non-crossover events that lead to gene conversion are harder to study, particularly at the individual level. Here we show that single high-fidelity long sequencing reads from sperm can capture both crossovers and non-crossovers, allowing effectively arbitrary sample sizes for analysis from one male. Using fifteen sperm samples from thirteen donors we demonstrate variation between and within donors for the rates of different types of recombination. Intriguingly, we observe a tendency for non-crossover gene conversions to occur upstream of nearby PRDM9 binding sites, whereas crossover locations have a slight downstream bias. We further provide evidence for two distinct non-crossover processes. One gives rise to the vast majority of non-crossovers with mean conversion tract length under 50bp, which we suggest is an outcome of standard PRDM9-induced meiotic recombination. In contrast ∼2% of non-crossovers have much longer mean tract length, and potentially originate from the same process as complex events with more than two haplotype switches, which is not associated with PRDM9 binding sites and is also seen in somatic cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7a7057073d62495af1df781d99095c76dca90d" target='_blank'>
              Insights into non-crossover recombination from long-read sperm sequencing
              </a>
            </td>
          <td>
            Regev Schweiger, Sangjin Lee, Chenxi Zhou, Tsun-Po Yang, Katie Smith, Stacy Li, R. Sanghvi, Matthew Neville, Emily Mitchell, Ayrun Nessa, Sam Wadge, K. Small, Peter J Campbell, Peter H. Sudmant, R. Rahbari, R. Durbin
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554fe13daafca3f151510eb83bbcfb8701b7284c" target='_blank'>
              Genome-scale CRISPR-Cas9 screening in stem cells: theories, applications and challenges
              </a>
            </td>
          <td>
            Heng Zhou, Peng Ye, W. Xiong, Xingxiang Duan, Shuili Jing, Yan He, Zhi Zeng, Yen Wei, Qingsong Ye
          </td>
          <td>2024-07-19</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A major goal in biology is to uncover the relationship between genotype and phenotype. However, identifying gene function is often hampered by genetic redundancy. For example, under standard laboratory conditions, three-quarters of the genes in the human pathogen Streptococcus pneumoniae are non-essential. A powerful approach to unravel genetic redundancy is by identifying gene-gene interactions. To uncover genetic interactions (GIs) in S. pneumoniae on a genome-wide scale, a generally applicable dual CRISPRi-Seq method and associated analysis pipeline was developed. Specifically, we created a library of 869 dual sgRNAs targeting high-confidence operons that encode essential and non-essential genes, covering over 70% of the genetic elements in the pneumococcal genome. Testing these 378,015 unique combinations, 4,026 significant GIs were identified, including 1,935 negative and 2,091 positive interactions. Besides known GIs, we found and confirmed previously unknown interactions involving genes responsible for fundamental cellular processes such as cell division, cell shape maintenance, and chromosome segregation. The presented methods and bioinformatic approaches can serve as a roadmap for genome-wide gene interaction studies in other organisms. Lastly, all interactions are available for exploration via the Pneumococcal Genetic Interaction Network (PneumoGIN) at https://veeninglab.shinyapps.io/PneumoGIN, which can serve as a starting point for new biological discoveries and translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dd78fdd99cdc75fb8cdbb9e56ea5f0c3b73930" target='_blank'>
              Dual CRISPRi-Seq for genome-wide genetic interaction studies identifies key genes involved in the pneumococcal cell cycle
              </a>
            </td>
          <td>
            Julien Dénéréaz, Elise Eray, Bimal Jana, Vincent de Bakker, Horia Todor, Tim van Opijnen, Xue Liu, J. Veening
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/553f4560eb657fca2fe86385e460c71427df430c" target='_blank'>
              GLI1 confers resistance to PARP inhibitors by activating the DNA damage repair pathway.
              </a>
            </td>
          <td>
            Hiroshi Ikeuchi, Yusuke Matsuno, Rika Kusumoto-Matsuo, S. Kojima, Toshihide Ueno, Masachika Ikegami, Rina Kitada, Hitomi Sumiyoshi-Okuma, Yuki Kojima, K. Yonemori, Y. Yatabe, K. Takamochi, Kenji Suzuki, Ken‐ichi Yoshioka, H. Mano, S. Kohsaka
          </td>
          <td>2024-08-03</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="This report presents the largest collection of gamma-delta T cell receptor (γδ TCR) reads in human cancer to date, analyzing about 11,000 patient tumor samples across 33 cancer types using the TRUST4 algorithm. Despite γδ T cells being a small fraction of the T cell population, they play a key role in both innate and adaptive immunity. Our comprehensive analysis reveals their significant presence across all cancer types, specifically highlighting the diverse spectrum and clonality patterns of their γδ receptors. This research highlights the complex roles of γδ T cells in tumor tissues and their potential as prognostic biomarkers. We also demonstrate the utility of T cell receptor gamma (TRG) and delta (TRD) gene expression values from standard RNA-seq data. Ultimately, our work establishes a fundamental resource for future tumor-infiltrating γδ T cell research and may facilitate the development of novel γδ-T-cell-based therapeutic strategies. Together, we demonstrate the strong diversity and prognostic potential of γδ T cells in multiple cancer types. Highlights Comprehensive analysis of γδ TCRs from 11,473 tumor samples Significant variability and overall consistency in γδ gene expression and clonotype γδ TCR expression and diversity as prognostic biomarkers across multiple cancers Centralized γδ TCR repertoire database for future therapeutic discovery">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f81b0341b0c9b28947c09231986dc63cf346223" target='_blank'>
              A pan-cancer gamma delta T cell repertoire
              </a>
            </td>
          <td>
            Xiaoqing Yu, Li Song, Ling Cen, Biwei Cao, Ranran Tao, Yuanyuan Shen, Daniel Abate- Daga, Paulo C. Rodriguez, Jose R Conejo-Garcia, Xuefeng Wang
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2fd505ae2ea1de627590bede77cfbae2e3ab46" target='_blank'>
              CRISPR screen of venetoclax response-associated genes identifies transcription factor ZNF740 as a key functional regulator
              </a>
            </td>
          <td>
            Lixia Zhang, Xinyue Zhou, Sajesan Aryal, Virginia Veasey, Pengcheng Zhang, Fu Jun Li, Yu Luan, Ravi Bhatia, Yang Zhou, Rui Lu
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Alterations in the FMS-like tyrosine kinase 3 (FLT3) gene are the most frequent driver mutations in acute myeloid leukaemia (AML), linked to a high risk of relapse in patients with internal tandem duplications (FLT3-ITD). Tyrosine kinase inhibitors (TKIs) targeting the FLT3 protein are approved for clinical use, yet resistance often emerges. This resistance is mainly seen following the acquisition of additional point mutations in the tyrosine kinase domain (TKD), resulting in a double mutant FLT3-ITD/TKD, which sustains cell signalling and survival despite the presence of FLT3 inhibitors. Here, we developed a FLT3-mutant AML model with adaptive resistance to type II TKIs, sorafenib, and quizartinib by in vitro drug selection. Through global multiomic profiling, we identified upregulation of proteins involved in reactive oxygen species (ROS) production, particularly NADPH-oxidases, driving cellular ’ROS-addiction’, with resistant cells relying on ROS for survival, and genome fidelity preserved by ATM-driven DNA repair. Transcriptomic analysis of adult and paediatric AML (pAML) patients identified high ATM expression as a biomarker for shorter median overall survival in both the de novo and relapsed settings. Inhibition of ATM with clinically relevant therapy WSD-0628 effectively killed TKI- and chemotherapy-resistant AML cells in vitro and significantly extended the survival of mice with sorafenib- and quizartinib-resistant FLT3-ITD AML in vivo. We propose a new treatment strategy to improve survival of patients who develop resistance to sorafenib and quizartinib, as well as relapsed and refractory pAML, exploiting resistance mechanisms to precision therapies and cell-intrinsic features of high-risk cases, highlighting a clinically relevant salvage strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0bcd4e64b15b98143cfc995f3b3d32e9c623b66" target='_blank'>
              Adaptive resistance to FLT3 inhibitors is potentiated by ROS-driven DNA repair signalling
              </a>
            </td>
          <td>
            Dilana E. Staudt, Zacary P. Germon, Abdul Mannan, Tabitha Mclachlan, Heather C Murray, Ryan J. Duchatel, Bryce C. Thomas, Tyrone S Beitaki, Holly P. McEwen, Mika L Persson, Leah Calvert, Izac J. Findlay, Evangeline R. Jackson, Nathan D. Smith, D. Skerrett‐Byrne, David Mossman, Brett Nixon, Geoffry De Iullis, Alicia M. Douglas, A. Enjeti, Jonathan R. Sillar, Janis Chamberlain, Frank Alvaro, Andrew H. Wei, Patrick Connerty, N. Verrills, Matthew D. Dun
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Hepatocellular carcinoma (HCC) is a common and lethal liver cancer characterized by complex aetiology and limited treatment options. FAM210B, implicated in various cancers, is noteworthy for its potential role in the progression and treatment response of HCC. Yet, its expression patterns, functional impacts and correlations with patient outcomes and resistance to therapy are not well understood. We employed a comprehensive methodology to explore the role of FAM210B in HCC, analysing its expression across cancers, subcellular localization and impacts on cell proliferation, invasion, migration, biological enrichment and the immune microenvironment. Additionally, we investigated its expression in single cells, drug sensitivity and relationships with genomic instability, immunotherapy efficacy and key immune checkpoints. While FAM210B expression varied across cancers, there was no notable difference between HCC and normal tissues. Elevated levels of FAM210B were associated with improved survival outcomes. Subcellular analysis located FAM210B in the plasma membrane and cytosol. FAM210B was generally downregulated in HCC, and its suppression significantly enhanced cell proliferation, invasion and migration. Biological enrichment analysis linked FAM210B to metabolic and immune response pathways. Moreover, its expression modified the immune microenvironment of HCC, affecting drug responsiveness and immunotherapy outcomes. High expression levels of FAM202B correlated with increased resistance to sunitinib and enhanced responsiveness to immunotherapy, as evidenced by associations with tumour mutation burden, PDCD1, CTLA4 and TIDE scores. FAM210B exerts a complex influence on HCC, affecting tumour cell behaviour, metabolic pathways, the immune microenvironment and responses to therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c718e7a7e3f48499c53df736551b09864a4287d7" target='_blank'>
              Multifaceted role of FAM210B in hepatocellular carcinoma: Implications for tumour progression, microenvironment modulation and therapeutic selection
              </a>
            </td>
          <td>
            Xianzhu Pan, Jun Xu, Yuanqin Zhou
          </td>
          <td>2024-08-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2256152e45d11bd82f68a7e3810e940b41cb02" target='_blank'>
              Essential gene screening identifies the bromodomain-containing protein BRPF1 as a new actionable target for endocrine therapy-resistant breast cancers
              </a>
            </td>
          <td>
            A. Salvati, G. Giurato, J. Lamberti, Ilaria Terenzi, Laura Crescenzo, Viola Melone, L. Palo, A. Giordano, Francesco Sabbatino, G. Roscigno, C. Quintavalle, G. Condorelli, Francesca Rizzo, Roberta Tarallo, G. Nassa, Alessandro Weisz
          </td>
          <td>2024-08-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b62bbc42b475a59548fce65b7db966d5dba12012" target='_blank'>
              CRISPRi-TnSeq maps genome-wide interactions between essential and non-essential genes in bacteria.
              </a>
            </td>
          <td>
            Bimal Jana, Xue Liu, Julien Dénéréaz, Hongshik Park, Dmitry Leshchiner, Bruce Liu, Clément Gallay, Junhao Zhu, J. Veening, Tim van Opijnen
          </td>
          <td>2024-07-19</td>
          <td>Nature microbiology</td>
          <td>2</td>
          <td>47</td>
        </tr>

        <tr id="Drug resistance is currently one of the biggest challenges in cancer treatment. With the deepening understanding of drug resistance, various mechanisms have been revealed, including metabolic reprogramming and alterations of redox balance. Notably, metabolic reprogramming mediates the survival of tumor cells in harsh environments, thereby promoting the development of drug resistance. In addition, the changes during metabolic pattern shift trigger reactive oxygen species (ROS) production, which in turn regulates cellular metabolism, DNA repair, cell death, and drug metabolism in direct or indirect ways to influence the sensitivity of tumors to therapies. Therefore, the intersection of metabolism and ROS profoundly affects tumor drug resistance, and clarifying the entangled mechanisms may be beneficial for developing drugs and treatment methods to thwart drug resistance. In this review, we will summarize the regulatory mechanism of redox and metabolism on tumor drug resistance and highlight recent therapeutic strategies targeting metabolic–redox circuits, including dietary interventions, novel chemosynthetic drugs, drug combination regimens, and novel drug delivery systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cc6698aebc91736fc846603c2b274976902d8b" target='_blank'>
              Targeting Metabolic–Redox Nexus to Regulate Drug Resistance: From Mechanism to Tumor Therapy
              </a>
            </td>
          <td>
            Yuke Wang, Jingqiu He, Shan Lian, Yan Zeng, Sheng He, Jue Xu, Li Luo, Wenyong Yang, Jingwen Jiang
          </td>
          <td>2024-07-01</td>
          <td>Antioxidants</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
Primary liver cancer is a major health problem being the sixth most frequent cancer in the world and the third cause of cancer-related death in the world. The most common histological type of liver cancer is hepatocellular carcinoma (HCC, 75-80%).


AREAS COVERED
Based on primary literature, this review provides an updated analysis of studies of genetic characterization of HCC at the level of gene mutation profiling, copy number alterations, and gene expression, with the definition of molecular subgroups and the identification of some molecular biomarkers and therapeutic targets. Recent therapeutic developments are also highlighted.


EXPERT OPINION
Deepening the understanding of the molecular complexity of HCC is progressively paving the way for the development of more personalized treatment approaches. Two important strategies involve the definition and validation of molecularly defined therapeutic targets in a subset of HCC patients and the identification of suitable biomarkers for approved systematic therapies (multikinase inhibitors and immunotherapies). The extensive molecular characterization of patients at the genomic and transcriptomic levels and the inclusion of detailed and relevant translational studies in clinical trials will represent a fundamental tool for improving the benefit of systemic therapies in HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e469821ee56a2f4f32d5dc57fb86ab06b3278ba1" target='_blank'>
              Recent developments in molecular targeted therapies for hepatocellular carcinoma in the genomic era.
              </a>
            </td>
          <td>
            Ugo Testa
          </td>
          <td>2024-08-28</td>
          <td>Expert review of molecular diagnostics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Identifying host genetic factors modulating immune checkpoint inhibitor (ICI) efficacy has been experimentally challenging because of variations in both host and tumor genomes, differences in the microbiome, and patient life exposures. Utilizing the Collaborative Cross (CC) multi-parent mouse genetic resource population, we developed an approach that fixes the tumor genomic configuration while varying host genetics. With this approach, we discovered that response to anti-PD-1 (aPD1) immunotherapy was significantly heritable in four distinct murine tumor models (H2 between 0.18-0.40). For the MC38 colorectal carcinoma system (H2 = 0.40), we mapped four significant ICI response quantitative trait loci (QTL) localized to mouse chromosomes (mChr) 5, 9, 15 and 17, and identified significant epistatic interactions between specific QTL pairs. Differentially expressed genes within these QTL were highly enriched for immune genes and pathways mediating allograft rejection and graft vs host disease. Using a cross species analytical approach, we found a core network of 48 genes within the four QTLs that showed significant prognostic value for overall survival in aPD1 treated human cohorts that outperformed all other existing validated immunotherapy biomarkers, especially in human tumors of the previously defined immune subtype 4. Functional blockade of two top candidate immune targets within the 48 gene network, GM-CSF and high affinity IL-2/IL-15 signaling, completely abrogated the MC38 tumor transcriptional response to aPD1 therapy in vivo. Thus, we have established a powerful cross species in vivo platform capable of uncovering host genetic factors that establish the tumor immune microenvironment configuration propitious for ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7a70d713da2ee90a6af227b41627e893515c66" target='_blank'>
              Mapping the genetic landscape establishing a tumor immune microenvironment favorable for anti-PD-1 response in mice and humans
              </a>
            </td>
          <td>
            Daniel A. Skelly, John P Graham, Mingshan Cheng, Mayuko Furuta, Andrew Walter, Thomas A. Stoklasek, Hongyuan Yang, Timothy M. Stearns, Olivier Poirion, Ji-Gang Zhang, J. Grassmann, D. Luo, William F. Flynn, Elise T. Courtois, Chih-Hao Chang, D. Serreze, F. Menghi, L. Reinholdt, Edison T. Liu
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66407cd2989bbfac769e0f4b6906811d3ffd112d" target='_blank'>
              Epigenetic modification of ferroptosis by non-coding RNAs in cancer drug resistance
              </a>
            </td>
          <td>
            Hongquan Wang, Joshua S Fleishman, Sihang Cheng, Weixue Wang, Fan Wu, Yumin Wang, Yu Wang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML), one of the most common types of leukemia, is characterized by its high malignancy and rapid progression with a 5-year survival rate of less than 30%. The incidence and mortality rates of AML are increasing with age. Over the past few decades, progress in AML treatment has been relatively slow. While traditional approaches such as chemotherapy and hematopoietic stem cell transplantation have significant limitations including treatment toxicity and chemotherapy resistance, recent advancements in the in-depth study of AML mechanisms have made targeted therapy a new option for AML treatment. Metabolic reprogramming is one of the key features of cancer, and mitochondrial dysfunction has been widely studied in various cancers. Mitochondrial dysfunction is prevalent in AML cells and closely associated with the development of AML. The AML cells exhibit significant differences from normal hematopoietic cells in energy metabolism, autophagy, apoptosis, and other aspects. Given that mitochondria are at the core of cellular energy metabolism, inhibiting pathways related to mitochondrial function holds significant potential for AML treatment. This review aims to explore recent advances on the role of mitochondrial dysfunction in AML cell survival, potential therapeutic targets in mitochondria, and related targeted drugs, aiming to provide ideas for the development of targeted therapies for AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58e38d332730c1931f871826d11fa781445ed55e" target='_blank'>
              [Recent advances in the study of potential mitochondria-related therapeutic targets in acute myeloid leukemia].
              </a>
            </td>
          <td>
            Lu Han, Li Liu, Jing Liu
          </td>
          <td>2024-08-25</td>
          <td>Sheng li xue bao : [Acta physiologica Sinica]</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) has a poor survival rate for both pediatric and adult patients due to its frequent relapse. To elucidate the bioenergetic principle underlying AML relapse, we investigated the transcriptional regulation of mitochondrial–nuclear dual genomes responsible for metabolic plasticity in treatment-resistant blasts. Both the gain and loss of function results demonstrated that NFκB2, a noncanonical transcription factor (TF) of the NFκB (nuclear factor kappa-light-chain-enhancer of activated B cells) family, can control the expression of TFAM (mitochondrial transcription factor A), which is known to be essential for metabolic biogenesis. Furthermore, genetic tracking and promoter assays revealed that NFκB2 is in the mitochondria and can bind the specific “TTGGGGGGTG” region of the regulatory D-loop domain to activate the light-strand promoter (LSP) and heavy-strand promoter 1 (HSP1), promoters of the mitochondrial genome. Based on our discovery of NFκB2′s novel function of regulating mitochondrial–nuclear dual genomes, we explored a novel triplet therapy including inhibitors of NFκB2, tyrosine kinase, and mitochondrial ATP synthase that effectively eliminated primary AML blasts with mutations of the FMS-related receptor tyrosine kinase 3 (FLT3) and displayed minimum toxicity to control cells ex vivo. As such, effective treatments for AML must include strong inhibitory actions on the dual genomes mediating metabolic plasticity to improve leukemia prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd91cb6b195c34903e27bb3147d1ba2db984023c" target='_blank'>
              Discovery of NFκB2-Coordinated Dual Regulation of Mitochondrial and Nuclear Genomes Leads to an Effective Therapy for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Yi Xu, David J. Baylink, Jeffrey Xiao, Lily Tran, Vinh Nguyen, Brandon Park, Ismael Valladares, Scott Lee, Kevin Codorniz, Laren D Tan, Chien-Shing Chen, Hisham Abdel-Azim, Mark E. Reeves, Hamid Mirshahidi, Guido Marcucci, H. Cao
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Fungi can cause devastating invasive infections, typically in immunocompromised patients. Treatment is complicated both by the evolutionary similarity between humans and fungi and by the frequent emergence of drug resistance. Studies in fungal pathogens have long been slowed by a lack of high-throughput tools and community resources that are common in model organisms. Here we demonstrate a high-throughput transposon mutagenesis and sequencing (TN-seq) system in Cryptococcus neoformans that enables genome-wide determination of gene essentiality. We employed a random forest machine learning approach to classify the Cryptococcus neoformans genome as essential or nonessential, predicting 1,465 essential genes, including 302 that lack human orthologs. These genes are ideal targets for new antifungal drug development. TN-seq also enables genome-wide measurement of the fitness contribution of genes to phenotypes of interest. As proof of principle, we demonstrate the genome-wide contribution of genes to growth in fluconazole, a clinically used antifungal. We show a novel role for the well-studied RIM101 pathway in fluconazole susceptibility. We also show that 5’ insertions of transposons can drive sensitization of essential genes, enabling screenlike assays of both essential and nonessential components of the genome. Using this approach, we demonstrate a role for mitochondrial function in fluconazole sensitivity, such that tuning down many essential mitochondrial genes via 5’ insertions can drive resistance to fluconazole. Our assay system will be valuable in future studies of C. neoformans, particularly in examining the consequences of genotypic diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126840639a843aec8469a2a2562239beea433d1f" target='_blank'>
              Saturation transposon mutagenesis enables genome-wide identification of genes required for growth and fluconazole resistance in the human fungal pathogen Cryptococcus neoformans
              </a>
            </td>
          <td>
            R. B. Billmyre, Caroline J. Craig, Joshua Lyon, Claire Reichardt, Michael T. Eickbush, Sarah E. Zanders
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, Andre Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Combined CDK4/6 inhibitor (CDK4/6i) and endocrine therapy significantly improves outcome for patients with estrogen receptor-positive (ER+) metastatic breast cancer, but drug resistance and thus disease progression inevitably occur. Herein, we aimed to identify genomic alterations associated with combined CDK4/6i and endocrine therapy resistance, and follow the levels of specific mutations in longitudinal circulating tumor DNA (ctDNA) for early detection of progression. From a cohort of 86 patients with ER+ metastatic breast cancer we performed whole exome sequencing or targeted sequencing of paired tumor (N = 8) or blood samples (N = 5) obtained before initiation of combined CDK4/6i and endocrine therapy and at disease progression. Mutations in oncogenic genes at progression were rare, while amplifications of growth-regulating genes were more frequent. The most frequently acquired alterations observed were PIK3CA and TP53 mutations and PDK1 amplification. Longitudinal ctDNA dynamics of mutant PIK3CA or private mutations revealed increased mutation levels at progression in 8 of 10 patients (80%). Impressively, rising levels of PIK3CA-mutated ctDNA were detected 4-17 months before imaging. Our data add to the growing evidence supporting longitudinal ctDNA analysis for real-time monitoring of CDK4/6i response and early detection of progression in advanced breast cancer. Further, our analysis suggests that amplification of growth-related genes may contribute to combined CDK4/6i and endocrine therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4f8f900b692158e74287adf0a693a5a4d08296" target='_blank'>
              Genomic alterations associated with resistance and circulating tumor DNA dynamics for early detection of progression on CDK4/6 inhibitor in advanced breast cancer.
              </a>
            </td>
          <td>
            Charlotte K Kindt, Carla L. Alves, S. Ehmsen, Amalie Kragh, T. Reinert, Marianne Vogsen, A. R. Kodahl, Jeanette D Rønlev, Dilan Ardik, Anna L Sørensen, Kirstine Evald, Mia L Clemmensen, J. Staaf, Henrik J Ditzel
          </td>
          <td>2024-08-11</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, A. Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cancer drug resistance poses a significant obstacle to successful chemotherapy, primarily driven by the activity of ATP-binding cassette (ABC) transporters, which actively efflux chemotherapeutic agents from cancer cells, reducing their intracellular concentrations and therapeutic efficacy. Recent studies have highlighted the pivotal role of long noncoding RNAs (lncRNAs) in regulating this resistance, positioning them as crucial modulators of ABC transporter function. lncRNAs, once considered transcriptional noise, are now recognized for their complex regulatory capabilities at various cellular levels, including chromatin modification, transcription, and post-transcriptional processing. This review synthesizes current research demonstrating how lncRNAs influence cancer drug resistance by modulating the expression and activity of ABC transporters. lncRNAs can act as molecular sponges, sequestering microRNAs that would otherwise downregulate ABC transporter genes. Additionally, they can alter the epigenetic landscape of these genes, affecting their transcriptional activity. Mechanistic insights reveal that lncRNAs contribute to the activity of ABC transporters, thereby altering the efflux of chemotherapeutic drugs and promoting drug resistance. Understanding these interactions provides a new perspective on the molecular basis of chemoresistance, emphasizing the regulatory network of lncRNAs and ABC transporters. This knowledge not only deepens our understanding of the biological mechanisms underlying drug resistance but also suggests novel therapeutic strategies. In conclusion, the intricate interplay between lncRNAs and ABC transporters is crucial for developing innovative solutions to combat cancer drug resistance, underscoring the importance of continued research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/964e0709fe20c21443ecd0c53a8a7ab6cf10b8c7" target='_blank'>
              lncRNAs: New players of cancer drug resistance via targeting ABC transporters.
              </a>
            </td>
          <td>
            Mohammad Ebrahimnezhad, Sanaz Hassanzadeh Asl, Maede Rezaie, Mehran Molavand, Bahman Yousefi, M. Majidinia
          </td>
          <td>2024-08-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c05cd3b7eb71af823cd51dee708a69c95084a142" target='_blank'>
              The role of lncRNA NEAT1 in human cancer chemoresistance
              </a>
            </td>
          <td>
            Feng Long, Xue Li, Jingyu Pan, Hailin Ye, Cuixia Di, Yong Huang, Jiawei Li, Xuan Zhou, Huiyi Yi, Qiaozhen Huang, Jing Si
          </td>
          <td>2024-07-05</td>
          <td>Cancer Cell International</td>
          <td>2</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W. Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers.
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a848be54c4ea82e1661c7913223b39ce6de8b05" target='_blank'>
              Small RNA activation of CDH13 expression overcome BCR-ABL1-independent imatinib-resistance and their signaling pathway studies in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Rui Su, Ziqi Wen, Xingri Zhan, Yiling Long, Xiuyuan Wang, Chuting Li, Yubin Su, Jia Fei
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Linking disease phenotypes with molecular targets is key to the rational design of treatment interventions. Resistance to the chemotherapeutic cisplatin is one of the major factors limiting the clinical utility of this therapy, which is central to the treatment of a variety of solid malignancies. In this study, we couple the upregulation of a chemoresistant factor, the glutamate-cystine antiporter SLC7A11, with the addiction of cisplatin-resistant cancer cells to extracellular glutamine. In doing so, we thus provide a putative biomarker for this acquired metabolic dependency of chemoresistance. Subsequently, we evaluate various therapeutic strategies to selectively kill SLC7A11high cisplatin-resistant cancer cells, identifying cross-resistance to ferroptosis-inducing compounds and hypersensitivity to glutaminase inhibitor CB-839. We identify enzymatic depletion of extracellular glutamine using the long-standing anti-leukemic therapy asparaginase (ASNase), which possesses glutaminase activity, as a potential approach, and show this can be successfully combined with cisplatin in cell models. In summary, this study mechanistically links an acquired metabolic dependency of chemoresistant cancer cells with a putative biomarker and provides a potentially actionable strategy to target these drug resistant cells warranting further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c836fb51ec4a12788e118211188bb2a1ae2ab5" target='_blank'>
              Glutamine-addiction in cisplatin resistant cancer cells is mediated by SLC7A11 and can be targeted with asparaginase therapy
              </a>
            </td>
          <td>
            Jiantao Wang, Robert Strauss, Jiri Bartek, S. G. Rudd
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Drug resistance is a major challenge in the treatment of advanced cholangiocarcinoma (CCA). Understanding the mechanisms of drug resistance can aid in identifying novel prognostic biomarkers and therapeutic targets to improve treatment efficacy. This study established 5-fluorouracil- (5-FU) and gemcitabine-resistant CCA cell lines, KKU-213FR and KKU-213GR, and utilized comparative proteomics to identify differentially expressed proteins in drug-resistant cells compared to parental cells. Additionally, bioinformatics analyses were conducted to explore the biological and clinical significance of key proteins. The drug-resistant phenotypes of KKU-213FR and KKU-213GR cell lines were confirmed. In addition, these cells demonstrated increased migration and invasion abilities. Proteomics analysis identified 81 differentially expressed proteins in drug-resistant cells, primarily related to binding functions, biological regulation, and metabolic processes. Protein–protein interaction analysis revealed a highly interconnected network involving MET, LAMB1, ITGA3, NOTCH2, CDH2, and NDRG1. siRNA-mediated knockdown of these genes in drug-resistant cell lines attenuated cell migration and cell invasion abilities and increased sensitivity to 5-FU and gemcitabine. The mRNA expression of these genes is upregulated in CCA patient samples and is associated with poor prognosis in gastrointestinal cancers. Furthermore, the functions of these proteins are closely related to the epithelial–mesenchymal transition (EMT) pathway. These findings elucidate the potential molecular mechanisms underlying drug resistance and tumor progression in CCA, providing insights into potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dfaa4af4b64a761bf16d4bce1645ee438c64a04" target='_blank'>
              Proteomics and Bioinformatics Identify Drug-Resistant-Related Genes with Prognostic Potential in Cholangiocarcinoma
              </a>
            </td>
          <td>
            Kankamol Kerdkumthong, S. Roytrakul, Kawinnath Songsurin, Kandawasri Pratummanee, Phanthipha Runsaeng, S. Obchoei
          </td>
          <td>2024-08-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Adoptive cell therapy is becoming a cornerstone of tumour immunotherapy. It relies on the relatively long-term (> 2 week) ex vivo expansion of T cells either in the form of tumour-infiltrating cells, or bulk cells modified with the expression of heterologous signalling proteins, e.g., chimeric antigen receptors. However, relatively little is known about the developmental trajectories of T cells under these conditions at the system level, or whether the pathways governing these trajectories could be manipulated for clinical advantage. Using bulk RNA-seq analysis of T cells expanded and rested over a 17-day period, we produce a resource revealing how gene expression changes as cells transition through distinct cellular states over the course of activation and ex vivo expansion. By integrating this resource with published single-cell RNA-seq data, we identify a member of the AP1 transcription factor (TF) family, FOSL1, that primes CD8+ T-cells towards an effector/killing phenotype. Remarkably, FOSL1 over-expression during T-cell expansion produced ‘super engager-like’ T-cells, evidenced by their gene-expression signatures and enhanced cancer-cell killing capacity. This establishes proof-of-principle for the rational engineering of T cells via TF modification during ex vivo expansion, offering a route to improving adoptive T-cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c6308d2b13138e23c30c7f12340afe1d6a3ca9" target='_blank'>
              Cell state and transcription factor modulation during extended ex vivo CD8+ T-cell expansion
              </a>
            </td>
          <td>
            Yuan Lui, Edward Jenkins, Emily Zhi Qing Ng, M. Kotowski, Sydney J Mullin, Joseph Clarke, Simon J. Davis, Ana Mafalda Santos, Sumana Sharma
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Essential gene products carry out fundamental cellular activities in interaction with other components. However, the lack of essential gene mutants and appropriate methodologies to link essential gene functions with their partners poses significant challenges. Here, we have generated deletion mutants in 32 genes previously identified as essential, with 23 mutants showing extremely slow growth in the SK36 strain of Streptococcus sanguinis. Whole-genome sequencing of 243 independently evolved populations of these mutants has identified >1000 spontaneous suppressor mutations in experimental evolution, many of which define new gene and pathway relationships, such as F1Fo-ATPase/V1Vo-ATPase/TrkA1-H1. Patterns of spontaneous mutations occurring in essential gene mutants differed from those found in wildtype. While gene duplications occurred rarely and appeared most often at later stages of evolution, substitutions, deletions, and insertions were prevalent in evolved populations. These essential gene deletion mutants and spontaneous mutations fixed in the mutant populations during evolution establish a foundation for understanding gene essentiality and the interaction of essential genes in networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b8085c4071c7f49004500689bcbfd19d2769747" target='_blank'>
              Experimental evolution of gene essentiality in bacteria
              </a>
            </td>
          <td>
            Liang Bao, Zan Zhu, Ahmed Ismail, Bin Zhu, Vysakh Anandan, Marvin Whiteley, T. Kitten, Ping Xu
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The DNA damage repair (DDR) pathway is a complex signaling cascade that can sense DNA damage and trigger cellular responses to DNA damage to maintain genome stability and integrity. A typical hallmark of cancer is genomic instability or nonintegrity, which is closely related to the accumulation of DNA damage within cancer cells. The treatment principles of radiotherapy and chemotherapy for cancer are based on their cytotoxic effects on DNA damage, which are accompanied by severe and unnecessary side effects on normal tissues, including dysregulation of the DDR and induced therapeutic tolerance. As a driving factor for oncogenes or tumor suppressor genes, noncoding RNA (ncRNA) have been shown to play an important role in cancer cell resistance to radiotherapy and chemotherapy. Recently, it has been found that ncRNA can regulate tumor treatment tolerance by altering the DDR induced by radiotherapy or chemotherapy in cancer cells, indicating that ncRNA are potential regulatory factors targeting the DDR to reverse tumor treatment tolerance. This review provides an overview of the basic information and functions of the DDR and ncRNAs in the tolerance or sensitivity of tumors to chemotherapy and radiation therapy. We focused on the impact of ncRNA (mainly microRNA [miRNA], long noncoding RNA [lncRNA], and circular RNA [circRNA]) on cancer treatment by regulating the DDR and the underlying molecular mechanisms of their effects. These findings provide a theoretical basis and new insights for tumor-targeted therapy and the development of novel drugs targeting the DDR or ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63af0c8bea30d510199d8c53e8bb1fd81c1002b8" target='_blank'>
              DNA damage response-related ncRNAs as regulators of therapy resistance in cancer
              </a>
            </td>
          <td>
            Ziru Gao, Xinchi Luan, Xuezhe Wang, Tianyue Han, Xiaoyuan Li, Zeyang Li, Peifeng Li, Zhixia Zhou
          </td>
          <td>2024-08-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer drug resistance constitutes a severe limitation for the satisfactory outcome of these patients. This is a complex problem due to the co-existence in cancer cells of multiple and synergistic mechanisms of chemoresistance (MOC). These mechanisms are accounted for by the expression of a set of genes included in the so-called resistome, whose effectiveness often leads to a lack of response to pharmacological treatment. Additionally, genetic variants affecting these genes further increase the complexity of the question. This review focuses on a set of genes encoding members of the transportome involved in drug uptake, which have been classified into the MOC-1A subgroup of the resistome. These proteins belong to the solute carrier (SLC) superfamily. More precisely, we have considered here several members of families SLC2, SLC7, SLC19, SLC22, SLCO, SLC28, SLC29, SLC31, SLC46, and SLC47 due to the impact of their expression and genetic variants in anticancer drug uptake by tumor cells or, in some cases, general bioavailability. Changes in their expression levels and the appearance of genetic variants can contribute to the Darwinian selection of more resistant clones and, hence, to the development of a more malignant phenotype. Accordingly, to address this issue in future personalized medicine, it is necessary to characterize both changes in resistome genes that can affect their function. It is also essential to consider the time-dependent dimension of these features, as the genetic expression and the appearance of genetic variants can change during tumor progression and in response to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96b3489c776763d0625940057b010e5e5524219f" target='_blank'>
              Impact of genetic variants in the solute carrier (SLC) genes encoding drug uptake transporters on the response to anticancer chemotherapy
              </a>
            </td>
          <td>
            Jose J. G. Marin, Maria A. Serrano, E. Herráez, E. Lozano, Sara Ortiz-Rivero, Laura Perez-Silva, María Reviejo, O. Briz
          </td>
          <td>2024-07-18</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66eb86e9bf90c107f44800e55996f2d2a686c181" target='_blank'>
              Integrative transcriptome analysis reveals the molecular events underlying impaired T-cell responses in EGFR-mutant lung cancer
              </a>
            </td>
          <td>
            Yu Zhao, Gu Tang, Jun Li, Xiaonan Bian, Xiaorong Zhou, Jian Feng
          </td>
          <td>2024-08-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/354d0cada1a73a94d156b70d215246eda0c2a524" target='_blank'>
              T cell dysfunction and therapeutic intervention in cancer.
              </a>
            </td>
          <td>
            Caitlin C. Zebley, D. Zehn, Stephen Gottshalk, Hongbo Chi
          </td>
          <td>2024-07-18</td>
          <td>Nature immunology</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="Cells must adapt to environmental changes to maintain homeostasis. One of the most striking environmental adaptations is entry into hibernation during which core body temperature can decrease from 37°C to as low at 4°C. How mammalian cells, which evolved to optimally function within a narrow range of temperatures, adapt to this profound decrease in temperature remains poorly understood. In this study, we conducted the first genome-scale CRISPR-Cas9 screen in cells derived from Syrian hamster, a facultative hibernator, as well as human cells to investigate the genetic basis of cold tolerance in a hibernator and a non-hibernator in an unbiased manner. Both screens independently revealed glutathione peroxidase 4 (GPX4), a selenium-containing enzyme, and associated proteins as critical for cold tolerance. We utilized genetic and pharmacological approaches to demonstrate that GPX4 is active in the cold and its catalytic activity is required for cold tolerance. Furthermore, we show that the role of GPX4 as a suppressor of cold-induced cell death extends across hibernating species, including 13-lined ground squirrels and greater horseshoe bats, highlighting the evolutionary conservation of this mechanism of cold tolerance. This study identifies GPX4 as a central modulator of mammalian cold tolerance and advances our understanding of the evolved mechanisms by which cells mitigate cold-associated damage—one of the most common challenges faced by cells and organisms in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181b39f570d649216102dc87d265ba81bbf8c6ba" target='_blank'>
              Multi-species genome-wide CRISPR screens identify GPX4 as a conserved suppressor of cold-induced cell death
              </a>
            </td>
          <td>
            Breanna Lam, Kathrin M. Kajderowicz, Heather R. Keys, Julian M. Roessler, Evgeni M. Frenkel, Adina Kirkland, Punam Bisht, Mohamed A. El-Brolosy, Rudolf Jaenisch, George W. Bell, Jonathan S Weissman, Eric C. Griffith, Siniša Hrvatin
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d47d5363206f0a4fb9e459a4bf61f772ff9c57" target='_blank'>
              Overexpression of REC8 induces aberrant gamete meiotic division and contributes to AML pathogenesis - a multiplexed microarray analysis and mendelian randomization study.
              </a>
            </td>
          <td>
            Wenxi Hua, J. Qi, Meng Zhou, Shiyu Han, Xiao-Yun Xu, Jinwen Su, Tingting Pan, Depei Wu, Yue Han
          </td>
          <td>2024-07-16</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="High risk - neuroblastoma (HR-NB) is a pediatric solid tumor with high lethality. Half of HR-NB are driven by MYCN gene amplification (MNA). These HR-NBs require high dosage chemotherapy and often relapse. Moreover, current therapies can cause severe long-term side effects and new therapies are urgently needed. This study investigates a novel therapeutic approach targeting the metabolic vulnerabilities of MNA NB cells. We discovered that Diphenyleneiodonium chloride (DPI), an inhibitor of flavoprotein enzymes and mitochondrial complex I, synergizes with mitoquinone mesylate (MitoQ), a mitochondria-targeted antioxidant in 2D and 3D in vitro models of NB. Similarly to DPI, MitoQ affects MNA cells in a MYCN-dependent way, being more toxic when MYCN levels are high. Furthermore, low nanomolar concentrations of MitoQ significantly decrease MYCN protein expression and induce differentiation of MNA cells. The DPI and MitoQ combination further synergizes with vincristine, a chemotherapeutic agent used in NB treatment. Phosphoproteomics and proteomics analysis suggests that the drug combination induces MNA NB cell death by arresting the cell cycle and inhibiting oxidative phosphorylation (OXPHOS) in the mitochondria. Thus, interference with mitochondrial metabolism may represent an effective strategy to enhance the activity of chemotherapeutic drugs in MNA-NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84bd27bba09b6f802fb233fccdb2001e1425eb1" target='_blank'>
              OXPHOS TARGETING OF MYCN-AMPLIFIED NEUROBLASTOMA
              </a>
            </td>
          <td>
            Soraya Epp, Donagh Egan, E. Poon, Amirah Adlina Abdul Aziz, Kieran Wynne, Melinda Halasz, Walter Kolch
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary With advanced bioinformatic and sequencing technologies, many non-coding RNAs have been found to harbor non-canonical open reading frames (ncORFs). Some studies suggest that many ncORF-encoded microproteins or micropeptides possess important roles in cancer biology, yet the functional roles of these ncORF-encoded peptides are mostly unknown. The purpose of this review is to provide a comprehensive overview of existing knowledge on ncORF-encoded microproteins in cancer biology and highlight their roles as potential prognostic and therapeutic targets in cancer. Abstract Cancer develops from abnormal cell growth in the body, causing significant mortalities every year. To date, potent therapeutic approaches have been developed to eradicate tumor cells, but intolerable toxicity and drug resistance can occur in treated patients, limiting the efficiency of existing treatment strategies. Therefore, searching for novel genes critical for cancer progression and therapeutic response is urgently needed for successful cancer therapy. Recent advances in bioinformatics and proteomic techniques have allowed the identification of a novel category of peptides encoded by non-canonical open reading frames (ncORFs) from historically non-coding genomic regions. Surprisingly, many ncORFs express functional microproteins that play a vital role in human cancers. In this review, we provide a comprehensive description of different ncORF types with coding capacity and technological methods in discovering ncORFs among human genomes. We also summarize the carcinogenic role of ncORFs such as pTINCR and HOXB-AS3 in regulating hallmarks of cancer, as well as the roles of ncORFs such as HOXB-AS3 and CIP2A-BP in cancer diagnosis and prognosis. We also discuss how ncORFs such as AKT-174aa and DDUP are involved in anti-cancer drug response and the underestimated potential of ncORFs as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d95f26adc077c4e04bfda2c2e30184b8f21906d" target='_blank'>
              Exploring the Dark Matter of Human Proteome: The Emerging Role of Non-Canonical Open Reading Frame (ncORF) in Cancer Diagnosis, Biology, and Therapy
              </a>
            </td>
          <td>
            Anni Ge, Curtis Chan, Xiaolong Yang
          </td>
          <td>2024-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pyrimidine nucleotide biosynthesis is a druggable metabolic dependency of cancer cells, and chemotherapy agents targeting pyrimidine metabolism are the backbone of treatment for many cancers. Dihydroorotate dehydrogenase (DHODH) is an essential enzyme in the de novo pyrimidine biosynthesis pathway that can be targeted by clinically approved inhibitors. However, despite robust preclinical anticancer efficacy, DHODH inhibitors have shown limited single-agent activity in phase 1 and 2 clinical trials. Therefore, novel combination therapy strategies are necessary to realize the potential of these drugs. To search for therapeutic vulnerabilities induced by DHODH inhibition, we examined gene expression changes in cancer cells treated with the potent and selective DHODH inhibitor brequinar (BQ). This revealed that BQ treatment causes upregulation of antigen presentation pathway genes and cell surface MHC class I expression. Mechanistic studies showed that this effect is (1) strictly dependent on pyrimidine nucleotide depletion, (2) independent of canonical antigen presentation pathway transcriptional regulators, and (3) mediated by RNA polymerase II elongation control by positive transcription elongation factor B (P-TEFb). Furthermore, BQ showed impressive single-agent efficacy in the immunocompetent B16F10 melanoma model, and combination treatment with BQ and dual immune checkpoint blockade (anti-CTLA-4 plus anti-PD-1) significantly prolonged mouse survival compared to either therapy alone. Our results have important implications for the clinical development of DHODH inhibitors and provide a rationale for combination therapy with BQ and immune checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b0c0126387b9def9714026a2af50898ea415781" target='_blank'>
              DHODH inhibition enhances the efficacy of immune checkpoint blockade by increasing cancer cell antigen presentation
              </a>
            </td>
          <td>
            Nick Mullen, Surendra K. Shukla, Ravi Thakur, S. S. Kollala, Dezhen Wang, Nina V. Chaika, Juan F Santana, William R Miklavcic, Drew A. LaBreck, J. Mallareddy, David H. Price, A. Natarajan, Kamiya Mehla, David B. Sykes, Michael A. Hollingsworth, Pankaj K Singh
          </td>
          <td>2024-07-08</td>
          <td>eLife</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is the most lethal type of lung cancer. Paradoxically, this tumor displays an initial exquisite response to chemotherapy; however, at relapse, the tumor is highly resistant to subsequent available therapies. Here, we report that the expression of three prime repair exonuclease 1 (TREX1) is strongly induced in chemoresistant SCLCs. ATAC-seq and ChIP-seq revealed a significant increase in chromatin accessibility and transcriptional activity of TREX1 gene locus in chemoresistant SCLC. Analyses of human SCLC tumors and patient-derived xenografts (PDXs) also showed increase in TREX1 expression in post-chemotherapy samples. TREX1 depletion caused the activation of cyclic GMP-AMP synthase (cGAS)-stimulator of interferon genes (STING) pathway due to cytoplasmic accumulation of damage-associated double-stranded DNA, inducing immunogenicity and enhancing the sensitivity of drug resistant cells to chemotherapy. These findings suggest TREX1 upregulation may partially contribute to the survival of resistant cells, and its inhibition may represent a promising therapeutic strategy to enhance antitumor immunity and potentiate the efficacy of chemotherapy and/or immunotherapy in chemoresistant SCLCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920774aa4db00913c78963d621212ab8e7a2e837" target='_blank'>
              Targeting TREX1 induces innate immune response in drug-resistant Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Takahiko Murayama, N. Mahadevan, Catherine B Meador, Elena V. Ivanova, Yuqiao Pan, Erik H. Knelson, Tetsuo Tani, Jun Nakayama, Xueying Ma, Tran C Thai, Yin P Hung, William Kim, Hideo Watanabe, Kathy Q Cai, Aaron N Hata, C. Paweletz, D. Barbie, Israel Cañadas
          </td>
          <td>2024-08-23</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="DNA methylation is traditionally associated with gene silencing, but its causal relationship and role in shaping cell fate decisions still need to be fully elucidated. Here, we conducted a genome-wide analysis to investigate the relationship between DNA methylation and gene expression at gene regulatory regions in human immune cells. By utilizing CRISPR-dCas9 DNA methylation editing tools, we successfully established a cause-and-effect relationship between the methylation levels of the promoter of the Interleukin1-receptor antagonist (IL1RN) gene and its expression. Notably, we observed that modifying the DNA methylation status of the IL1RN promoter is sufficient to alter the acquisition of the human myeloid cell fate and change the cellular response to inflammatory stimuli, resulting in abnormal cytokine release and distinctive capacity to support cancer growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0466661c0180cda161896839ec2c88a8975d7da9" target='_blank'>
              Modulating immune cell fate and inflammation through CRISPR-mediated DNA methylation editing
              </a>
            </td>
          <td>
            Gemma Valcárcel, Anna V. López-Rubio, Aleksey Lazarenkov, Clara Berenguer, Josep Calafell, J. Rodríguez-Ubreva, E. Ballestar, J. L. Sardina
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain cancer characterized by significant molecular and cellular heterogeneity, which complicates treatment efforts. Current standard therapies, including surgical resection, radiation, and temozolomide (TMZ) chemotherapy, often fail to achieve long-term remission due to tumor recurrence and resistance. A pro-oxidant environment is involved in glioma progression, with oxidative stress contributing to the genetic instability that leads to gliomagenesis. Evaluating pro-oxidant therapies in brain tumors is crucial due to their potential to selectively target and eradicate cancer cells by exploiting the elevated oxidative stress levels inherent in these malignant cells, thereby offering a novel and effective strategy for overcoming resistance to conventional therapies. This study investigates the therapeutic potential of doxorubicin (DOX) and photodynamic therapy (PDT) with Me-ALA, focusing on their effects on redox homeostasis. Basal ROS levels and antioxidant gene expression (NFE2L2, CAT, GSR) were quantitatively assessed across GBM cell lines, revealing significant variability probably linked to genetic differences. DOX and PDT treatments, both individually and in combination, were analyzed for their efficacy in inducing oxidative stress and cytotoxicity. An in silico analysis further explored the relationship between gene mutations and oxidative stress in GBM patients, providing insights into the molecular mechanisms underlying treatment responses. Our findings suggest that pro-oxidant therapies, such as DOX and PDT in combination, could selectively target GBM cells, highlighting a promising avenue for improving therapeutic outcomes in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a4c8f4dd03777d2e91e0e7df47c1185f6a5a011" target='_blank'>
              Enhancing Therapeutic Approaches in Glioblastoma with Pro-Oxidant Treatments and Synergistic Combinations: In Vitro Experience of Doxorubicin and Photodynamic Therapy
              </a>
            </td>
          <td>
            Bruno Agustín Cesca, M. Caverzán, M. Lamberti, L. E. Ibarra
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="RNAs undergo a complex choreography of metabolic processes in human cells that are regulated by thousands of RNA-associated proteins. While the effects of individual RNA-associated proteins on RNA metabolism have been extensively characterized, the full complement of regulators for most RNA metabolic events remain unknown. Here we present a massively parallel RNA-linked CRISPR (ReLiC) screening approach to measure the responses of diverse RNA metabolic events to knockout of 2,092 human genes encoding all known RNA-associated proteins. ReLiC screens highlight modular interactions between gene networks regulating splicing, translation, and decay of mRNAs. When combined with biochemical fractionation of polysomes, ReLiC reveals striking pathway-specific coupling between growth fitness and mRNA translation. Perturbing different components of the translation and proteostasis machineries have distinct effects on ribosome occupancy, while perturbing mRNA transcription leaves ribosome occupancy largely intact. Isoform-selective ReLiC screens capture differential regulation of intron retention and exon skipping by SF3b complex subunits. Chemogenomic screens using ReLiC decipher translational regulators upstream of mRNA decay and uncover a role for the ribosome collision sensor GCN1 during treatment with the anti-leukemic drug homoharringtonine. Our work demonstrates ReLiC as a versatile platform for discovering and dissecting regulatory principles of human RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d61d936925a83dd1c8b255f5e6a3f00dce8861" target='_blank'>
              Decoding RNA Metabolism by RNA-linked CRISPR Screening in Human Cells
              </a>
            </td>
          <td>
            Patrick J. Nugent, Heungwon Park, Cynthia L. Wladyka, Katharine Y. Chen, Christine Bynum, Grace Quarterman, Andrew C. Hsieh, A. Subramaniam
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Epigenetics are the changes due to DNA methylation, histone modifications, and noncoding RNA regulation, which play a crucial part in the expression of genes. Although these changes do not alter the sequence of the DNA, they are significant in determining the phenotype and inheritance of the genes. The impact of epigenetics on inherited diseases like Fragile X syndrome, Angelman syndrome, and Prader-Willi syndrome emphasizes the significance of studying epigenetic dysregulation in disease pathology. The current review aims to investigate the role of epigenetic mechanisms in inherited diseases and to evaluate the utility of molecular diagnostic tools for epigenetic analysis in disease diagnosis. Case studies of diseases such as Duchenne muscular dystrophy and Prader-Willi syndrome highlight the clinical relevance of epigenetic analysis in disease diagnosis and management. This research is conducted through an extensive analysis of recent literature to explore the epigenetic mechanisms underlying inherited diseases. Additionally, advanced molecular diagnostic techniques like droplet digital PCR (ddPCR), transposase-based bisulfite tagging, and data mining coupled with conventional experimental procedures. However, the intricate relationship between genetic and epigenetic factors and challenges related to inclined data need to be addressed. Future research endeavors to elucidate the role of epigenetic modifications in disease pathogenesis and develop strategies for personalized medicinal treatment and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f24f0960efd210e8ee51cbad54070948fecc2ef" target='_blank'>
              A Review on Epigenetics of Human Inherited Diseases: Molecular Diagnosis
              </a>
            </td>
          <td>
            Bibi Shazia Sabir
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Innovative Science and Research Technology (IJISRT)</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Recurrent mutations in serine / arginine-rich splicing factor 2 (SRSF2), particularly the proline-to-histidine substitution at position 95 (P95H), have been proposed to drive neoplastic diseases. SRSF2 plays pivotal roles in pre-mRNA processing and gene transcription. However, the precise impact of these diverse functions of SRSF2 on cancer cell behaviour remains unclear. Here, we show that deletion or homozygous P95H mutation of SRSF2 both cause extensive DNA damage leading to cell cycle arrest in vitro and in vivo, regardless of cell genotype. We mechanistically demonstrate that SRSF2 is required for efficient bi-directional transcription of DNA replication and repair genes, independent of its function in splicing. In contrast, SRSF2 haploinsufficiency induces DNA damage without halting the cell cycle in cancer cells. Inducing the heterozygous Srsf2 P95H mutation leads to clonal expansion of epidermal cells in mouse skin, but tumor formation is inhibited following exposure to carcinogens. To survive carcinogen treatment, cells containing the Srsf2 P95H mutation undergo substantial transcription rewiring that restores bi-directional gene expression. Thus, our study reveals crucial roles of SRSF2 beyond splicing and underscores its importance in regulating transcription to orchestrate the cell cycle along with the DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32907ead7270689c743bba801c421a14be049a66" target='_blank'>
              SRSF2 transcriptional function maintains genome integrity during cell division
              </a>
            </td>
          <td>
            Rebecca E. Wagner, Leonie Arnetzl, Thiago Britto-Borges, Anke Heit-Mondrzyk, Ali Bakr, Etienne Sollier, Jasper Panten, Sylvain Delaunay, Daniela Sohn, P. Schmezer, D. T. Odom, Karin Müller-Decker, C. Plass, Christoph Dieterich, P. Lutsik, Susanne Bornelöv, Michaela Frye
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c785685c384edfe64958536f2b96ef40bece9a9d" target='_blank'>
              Let’s make it personal: CRISPR tools in manipulating cell death pathways for cancer treatment
              </a>
            </td>
          <td>
            Mobina Bayat, J. S. Nahand
          </td>
          <td>2024-07-29</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a dreadful disease, marked by the uncontrolled proliferation of clonal plasma cells (PCs) within the bone marrow (BM). MM is characterized by a highly heterogeneous clinical and molecular background, supported by severe genomic alterations. Important deregulation of long non-coding RNAs (lncRNAs) expression has been reported in MM patients, influencing progression and therapy resistance. NEAT1 is a lncRNA essential for nuclear paraspeckles and involved in gene expression regulation. We showed that NEAT1 supports MM proliferation making this lncRNA an attractive therapeutic candidate. Here, we used a combinatorial strategy integrating transcriptomic and computational approaches with functional high-throughput drug screening, to identify compounds that synergize with NEAT1 inhibition in restraining MM cells growth. AUKA inhibitors were identified as top-scoring drugs in these analyses. We showed that the combination of NEAT1 silencing and AURKA inhibitors in MM profoundly impairs microtubule organization and mitotic spindle assembly, finally leading to cell death. Analysis of the large publicly CoMMpass dataset showed that in MM patients AURKA expression is strongly associated with reduced progression-free (p < 0.0001) and overall survival probability (p < 0.0001) and patients displaying high expression levels of both NEAT1 and AURKA have a worse clinical outcome. Finally, using RNA-sequencing data from NEAT1 knockdown (KD) MM cells, we identified the AURKA allosteric regulator TPX2 as a new NEAT1 target in MM and as a mediator of the interplay between AURKA and NEAT1, therefore providing a possible explanation of the synergistic activity observed upon their combinatorial inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a62f32e75aca50b8f306a5cfaa25789b960cc17d" target='_blank'>
              Combinatorial strategies targeting NEAT1 and AURKA as new potential therapeutic options for multiple myeloma.
              </a>
            </td>
          <td>
            N. Puccio, G. Manzotti, Elisabetta Mereu, Federica Torricelli, D. Ronchetti, Michela Cumerlato, Ilaria Craparotta, L. Di Rito, Marco Bolis, Valentina Traini, V. Manicardi, V. Fragliasso, Yvan Torrente, N. Amodio, N. Bolli, E. Taiana, Alessia Ciarrochi, Roberto Piva, A. Neri
          </td>
          <td>2024-07-11</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Abstract Strict control over hematopoietic stem cell decision making is essential for healthy life‐long blood production and underpins the origins of hematopoietic diseases. Acute myeloid leukemia (AML) in particular is a devastating hematopoietic malignancy that arises from the clonal evolution of disease‐initiating primitive cells which acquire compounding genetic changes over time and culminate in the generation of leukemic stem cells (LSCs). Understanding the molecular underpinnings of these driver cells throughout their development will be instrumental in the interception of leukemia, the enabling of effective treatment of pre‐leukemic conditions, as well as the development of strategies to target frank AML disease. To this point, a number of precancerous myeloid disorders and age‐related alterations are proving as instructive models to gain insights into the initiation of LSCs. Here, we explore this myeloid dysregulation at the level of post–transcriptional control, where RNA‐binding proteins (RBPs) function as core effectors. Through regulating the interplay of a myriad of RNA metabolic processes, RBPs orchestrate transcript fates to govern gene expression in health and disease. We describe the expanding appreciation of the role of RBPs and their post–transcriptional networks in sustaining healthy hematopoiesis and their dysregulation in the pathogenesis of clonal myeloid disorders and AML, with a particular emphasis on findings described in human stem cells. Lastly, we discuss key breakthroughs that highlight RBPs and post–transcriptional control as actionable targets for precision therapy of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6dddad312b3774cf0fac0198aa7e076e582021" target='_blank'>
              RNA binding protein‐directed control of leukemic stem cell evolution and function
              </a>
            </td>
          <td>
            Pratik Joshi, Ava Keyvani Chahi, Lina Liu, Steven Moreira, Ana Vujovic, Kristin J Hope
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Structural variants (SVs) are important contributors to human disease. Their characterization remains however difficult due to their size and association with repetitive regions. Long-read sequencing (LRS) and optical genome mapping (OGM) can aid as their molecules span multiple kilobases and capture SVs in full. In this study, we selected six individuals who presented with unresolved SVs. We applied LRS onto all individuals and OGM to a subset of three complex cases. LRS detected and fully resolved the interrogated SV in all samples. This enabled a precise molecular diagnosis in two individuals. Overall, LRS identified 100% of the junctions at single-basepair level, providing valuable insights into their formation mechanisms without need for additional data sources. Application of OGM added straightforward variant phasing, aiding in the unravelment of complex rearrangements. These results highlight the potential of LRS and OGM as follow-up molecular tests for complete SV characterization. We show that they can assess clinically relevant structural variation at unprecedented resolution. Additionally, they detect (complex) cryptic rearrangements missed by conventional methods. This ultimately leads to an increased diagnostic yield, emphasizing their added benefit in a diagnostic setting. To aid their rapid adoption, we provide detailed laboratory and bioinformatics workflows in this manuscript.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d491356850d1dca650c1719115928a885707ff0" target='_blank'>
              Full characterization of unresolved structural variation through long-read sequencing and optical genome mapping
              </a>
            </td>
          <td>
            Griet De Clercq, Lies Vantomme, Barbara Dewaele, Bert Callewaert, Olivier Vanakker, Sandra Janssens, Bart Loeys, Mojca Strazisar, W. D. Coster, J. Vermeesch, A. Dheedene, B. Menten
          </td>
          <td>2024-07-19</td>
          <td>None</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8e1be4efe22c9fbd98a98ceb3e0939364442527" target='_blank'>
              Genome-wide CRISPR screening identifies tyrosylprotein sulfotransferase-2 as a target for augmenting anti-PD1 efficacy
              </a>
            </td>
          <td>
            Yumi Oh, Sujeong Kim, Yunjae Kim, Hyun Kim, Dongjun Jang, Seungjae Shin, Soo Jin Lee, Jiwon Kim, Sang Eun Lee, Jaeik Oh, Yoojin Yang, Dohee Kim, Hae-Rim Jung, Sangjin Kim, Jihui Kim, K. Min, Beomki Cho, Hoseok Seo, Dohyun Han, Hansoo Park, Sung-Yup Cho
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Biallelic pathogenic variants in the essential DNA repair gene BRCA2 causes Fanconi anemia, complementation group FA-D1. Patients in this group are highly prone to develop embryonal tumors, most commonly medulloblastoma arising from the cerebellar granule cell progenitors (GCPs). GCPs undergo high proliferation in the postnatal cerebellum under SHH activation, but the type of DNA lesions that require the function of the BRCA2 to prevent tumorigenesis remains unknown. To identify such lesions, we assessed both GCP neurodevelopment and tumor formation using a mouse model with deletion of exons three and four of Brca2 in the central nervous system, coupled with global Trp53 loss. Brca2Δex3-4;Trp53-/- animals developed SHH subgroup medulloblastomas with complete penetrance. Whole-genome sequencing of the tumors identified structural variants with breakpoints enriched in areas overlapping G-quadruplexes (G4s). Brca2-deficient GCPs exhibited decreased replication speed in the presence of the G4-stabilizer pyridostatin. Pif1 helicase, which resolves G4s during replication, was highly upregulated in tumors, and Pif1 knockout in primary MB tumor cells resulted in increased genome instability upon pyridostatin treatment. These data suggest that G4s may represent sites prone to replication stalling in highly proliferative GCPs and without BRCA2, G4s become a source of genome instability. Tumor cells upregulate G4-resolving helicases to facilitate rapid proliferation through G4s highlighting PIF1 helicase as a potential therapeutic target for treatment of BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d543ffa68779aadc1c04b11411614442736e5f08" target='_blank'>
              G-quadruplexes are a source of vulnerability in BRCA2 deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, Carolina Sandoval-Garcia, Margaret L. MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite the better understanding of the molecular mechanisms contributing to the pathogenesis of acute myeloid leukemia (AML) and improved patient survival in recent years, AML therapy still remains a clinical challenge. For this reason, it is important to search for new therapies that will enable the achievement of remission. Recently, the Food and Drug Administration approved three mutant IDH (mIDH) inhibitors for the treatment of AML. However, the use of mIDH inhibitors in monotherapy usually leads to the development of resistance and the subsequent recurrence of the cancer, despite the initial effectiveness of the therapy. A complete understanding of the mechanisms by which IDH mutations influence the development of leukemia, as well as the processes that enable resistance to mIDH inhibitors, may significantly improve the efficacy of this therapy through the use of an appropriate synergistic approach. The aim of this literature review is to present the role of IDH1/IDH2 mutations in the pathogenesis of AML and the results of clinical trials using mIDH1/IDH2 inhibitors in AML and to discuss the challenges related to the use of mIDH1/IDH2 inhibitors in practice and future prospects related to the potential methods of overcoming resistance to these agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/956158b91569ef3848247f4315ca0b54ae5a5903" target='_blank'>
              Clinical Implications of Isocitrate Dehydrogenase Mutations and Targeted Treatment of Acute Myeloid Leukemia with Mutant Isocitrate Dehydrogenase Inhibitors—Recent Advances, Challenges and Future Prospects
              </a>
            </td>
          <td>
            Adrian Kowalczyk, Julia Zarychta, Monika Lejman, Eryk Latoch, Joanna Zawitkowska
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="This journal article delves into the transformative impact of genomic technology in the realm of cancer diagnosis and treatment, specifically focusing on early detection and personalized therapeutic interventions. Genomic advancements have revolutionized our understanding of cancer at the molecular level, offering unprecedented insights into the genetic basis of the disease. This study explores the application of genomic technology to enable early diagnosis by identifying specific genetic markers associated with different cancer types. 
The research also investigates the integration of genomic information into personalized treatment plans, tailoring interventions based on the unique genetic profile of individual patients. Precision medicine, guided by genomic data, has shown promising results in improving treatment outcomes and minimizing adverse effects. The article reviews key genomic technologies, such as next-generation sequencing and gene expression profiling, highlighting their role in elucidating the intricate genetic landscape of cancer. 
Furthermore, ethical considerations and challenges associated with the implementation of genomic technology in cancer care are discussed. Issues about patient privacy, data security, and the need for informed consent are addressed to ensure responsible and patient-centered genomic practices. 
The findings underscore the potential of genomic technology to revolutionize cancer care, ushering in an era of early detection and personalized treatment strategies. As we move towards an era of precision oncology, understanding the implications and nuances of genomic applications becomes crucial for healthcare practitioners, researchers, and policymakers alike.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4805c1db7204ab7b7390c9cf915417ad43d94375" target='_blank'>
              Application of Genomic Technology in Early Diagnosis and Personalized Treatment for Cancer Patients
              </a>
            </td>
          <td>
            Hatijar Hatijar, Novycha Auliafendri, Robertus Surjoseto, Ni Ketut Sujati, Muhammad Nazarudin
          </td>
          <td>2024-07-25</td>
          <td>Global International Journal of Innovative Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2bf796c8f34094028dae8362be2961a526dfd4" target='_blank'>
              Understanding the function of Pax5 in development of docetaxel-resistant neuroendocrine-like prostate cancers
              </a>
            </td>
          <td>
            Sreyashi Bhattacharya, Hanna Harris, Ridwan Islam, Sanika Bodas, N. Polavaram, Juhi Mishra, Dipanwita Das, P. Seshacharyulu, Achyuth Kalluchi, Anirban Pal, Manish Kohli, Subodh M. Lele, Michael Muders, Surinder K. Batra, Paramita M Ghosh, K. Datta, M. Rowley, S. Dutta
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a significant public health challenge, with 5-fluorouracil (5-FU) resistance being a major obstacle to effective treatment. Despite advancements, resistance to 5-FU remains formidable due to complex mechanisms such as alterations in drug transport, evasion of apoptosis, dysregulation of cell cycle dynamics, tumor microenvironment (TME) interactions, and extracellular vesicle (EV)-mediated resistance pathways. Traditional chemotherapy often results in high toxicity, highlighting the need for alternative approaches with better efficacy and safety. Phytochemicals (PCs) and EVs offer promising CRC therapeutic strategies. PCs, derived from natural sources, often exhibit lower toxicity and can target multiple pathways involved in cancer progression and drug resistance. EVs can facilitate targeted drug delivery, modulate the immune response, and interact with the TME to sensitize cancer cells to treatment. However, the potential of PCs and engineered EVs in overcoming 5-FU resistance and reshaping the immunosuppressive TME in CRC remains underexplored. Addressing this gap is crucial for identifying innovative therapies with enhanced efficacy and reduced toxicities. This review explores the multifaceted mechanisms of 5-FU resistance in CRC and evaluates the synergistic effects of combining PCs with 5-FU to improve treatment efficacy while minimizing adverse effects. Additionally, it investigates engineered EVs in overcoming 5-FU resistance by serving as drug delivery vehicles and modulating the TME. By synthesizing the current knowledge and addressing research gaps, this review enhances the academic understanding of 5-FU resistance in CRC, highlighting the potential of interdisciplinary approaches involving PCs and EVs for revolutionizing CRC therapy. Further research and clinical validation are essential for translating these findings into improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b42d2539e636dd666b34c979ca54ba15797bab1" target='_blank'>
              Innovative Strategies to Combat 5-Fluorouracil Resistance in Colorectal Cancer: The Role of Phytochemicals and Extracellular Vesicles
              </a>
            </td>
          <td>
            M. Barathan, A. K. Zulpa, Sook-Luan Ng, Y. Lokanathan, Min Hwei Ng, J. Law
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>2</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4af8490fe3598124f1ff1f6ff633491cbc08e2" target='_blank'>
              Engineered targeting OIP5 sensitizes bladder cancer to chemotherapy resistance via TRIP12-PPP1CB-YBX1 axis.
              </a>
            </td>
          <td>
            Xianteng Wang, Ting Guo, Liman Niu, Binbin Zheng, Wei Huang, Hai－sheng Xu, Weiren Huang
          </td>
          <td>2024-08-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Advances in melanoma research have unveiled critical insights into its genetic and molecular landscape, leading to significant therapeutic innovations. This review explores the intricate interplay between genetic alterations, such as mutations in BRAF, NRAS, and KIT, and melanoma pathogenesis. The MAPK and PI3K/Akt/mTOR signaling pathways are highlighted for their roles in tumor growth and resistance mechanisms. Additionally, this review delves into the impact of epigenetic modifications, including DNA methylation and histone changes, on melanoma progression. The tumor microenvironment, characterized by immune cells, stromal cells, and soluble factors, plays a pivotal role in modulating tumor behavior and treatment responses. Emerging technologies like single-cell sequencing, CRISPR-Cas9, and AI-driven diagnostics are transforming melanoma research, offering precise and personalized approaches to treatment. Immunotherapy, particularly immune checkpoint inhibitors and personalized mRNA vaccines, has revolutionized melanoma therapy by enhancing the body’s immune response. Despite these advances, resistance mechanisms remain a challenge, underscoring the need for combined therapies and ongoing research to achieve durable therapeutic responses. This comprehensive overview aims to highlight the current state of melanoma research and the transformative impacts of these advancements on clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32ea057809622a436f9ea97ccea29c8077c2cd13" target='_blank'>
              Advances in Melanoma: From Genetic Insights to Therapeutic Innovations
              </a>
            </td>
          <td>
            Fernando Valdez-Salazar, Luis A. Jiménez-Del Rio, J. Padilla-Gutiérrez, Y. Valle, J. Muñóz-Valle, Emmanuel Valdés-Alvarado
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Background: Acute myeloid leukemia (AML) is a type of blood cancer with diverse genetic pathogenesis. The identification of potential molecular biomarkers could improve the AML diagnosis and targeted therapy. Recently, competing endogenous RNAs (ceRNAs) became an active area in cancer research to determine molecular mechanisms underlying tumor development. Objectives: The aim of the present study was to investigate the key molecular biomarkers that are closely related to AML through bioinformatics analysis. Methods: In this research, the RNA-seq data of 151 AML patients and 151 corresponding health samples were retrieved from The Cancer Genome Atlas (TCGA) database and GTEx Portal (genotype-tissue expression), respectively. After that we screened the differentially expressed long non-coding RNAs (lncRNAs), microRNAs (miRNAs), and mRNAs by “limma” package in R to construct the co-expression network ceRNA (miRNA-lncRNA-mRNA) with weighted gene co-expression network analysis (WGCNA) package in R and Cytoscape (version 3.7.2) software, respectively. Then the relevant modules were identified and functional enrichment analysis was used to uncover the modules that are biologically related to AML cancer. Results: Based on our bioinformatics studies, we constructed a significant module which contain 333 genes. Among them, five up-regulated miRNAs with the highest GS (gene significant), include hsa-miR-374B, hsa-miR-553, hsa-miR-3679, hsa-miR-548L, hsa-miR-597 and five down-regulated miRNAs with the lowest GS including hsa-miR-3934, hsa-miR-4746, hsa-miR-466, hsa-miR-6722, hsa-miR-4490. For protein-coding genes (PCGs), the top-five up-regulated PCGs with the highest GS were AMIGO3, H2AC17, SPACA5, GPR21, and OR13C3. The top-five down-regulated PCGs with the lowest GS were XBP1, TOMM6, TREX1, TNFRSF6B, and BOLA2. Among the lncRNAs, the five up-regulated lncRNAs with the highest level of GS were GREP1, ZBTB20-AS2, LINC01596, LINC00345, and RMRP. The five down-regulated lncRNAs with the lowest GS were LINC01609, LINC01707, LINC02270, LINC02309, and LINC02243. These lncRNAs can serve as potential biomarkers to distinguish between AML and normal samples. Conclusions: Finally, considering the role of ceRNA network in various biological processes, examining the role of ceRNAs in AML may provide a deeper insight into molecular mechanisms underlying the pathogenesis of cancer. This understanding may propose potential biomarkers for diagnosis and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b4efc24040b8239ec857871d4330cd62e238941" target='_blank'>
              Construction of CeRNA Regulatory Network Based on WGCNA to Reveal Potential Prognostic Biomarkers for AML Cancer
              </a>
            </td>
          <td>
            Fateme Larki Ard Darabi, Mahsa Dehdar, Ghasem Azizi-Tabesh
          </td>
          <td>2024-08-18</td>
          <td>International Journal of Cancer Management</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="INTRODUCTION
Advances in understanding of tumor biology shed light on hallmarks of cancer development and progression that include dysregulated DNA damage repair (DDR) machinery. Leveraging underlying tumor genomic instability and tumor specific defects in DDR, Poly (ADP-ribose) polymerases (PARP) inhibitors (PARPi) induced DNA damage emerges as a novel non-chemotherapy therapeutic opportunity. PARPis are currently approved in multiple tumor types, with the largest benefit seen in tumors with homologous recombination repair (HRR) deficiency, including germline and somatic mutations in BRCA1/2 genes (BRCA) and other pathway members such as PALB2 and Rad51c.


AREAS COVERED
This review article summarizes the current approval landscape and known and proposed mechanisms of resistance to PARPi. Further, therapeutic strategies to overcome PARPi resistance are discussed, including ongoing clinical trials.


EXPERT OPINION
PARPi have proven to be a safe and effective therapy and represent a cornerstone treatment across multiple solid tumor types. Elucidating innate and acquired mechanisms of resistance, coupled with the emergence of novel therapeutic options to capitalize on the activity of PARPi and prevent or reverse the acquisition of resistance, provides an opportunity to further expand the role of PARPi in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec7393511b29f158c45a3ee7ca1c8ffe71c659b" target='_blank'>
              Strategies for the prevention or reversal of PARP inhibitor resistance.
              </a>
            </td>
          <td>
            Zahi I. Mitri, Shaun M Goodyear, Gordon B. Mills
          </td>
          <td>2024-08-15</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e7e1fc16e696eaffc0fa02c1362c0f5c07aa1" target='_blank'>
              The complex interplay of tumor-infiltrating cells in driving therapeutic resistance pathways
              </a>
            </td>
          <td>
            Dengxiong Li, Fanglin Shao, Qingxin Yu, Rui-cheng Wu, Zhou-ting Tuo, Jie Wang, Luxia Ye, Yiqing Guo, Koo Han Yoo, Mang Ke, U. Okoli, Chaipanichkul Premkamon, Yubo Yang, Wuran Wei, Susan Heavey, William C. Cho, Dechao Feng
          </td>
          <td>2024-08-19</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cytogenomic characterization is crucial for the classification and risk stratification of acute myeloid leukemia (AML), thereby facilitating therapeutic decision-making. We examined the clinical utility of optical genome mapping (OGM) in 159 AML patients (103 newly diagnosed and 56 refractory/relapsed), all of whom also underwent chromosomal banding analysis (CBA), fluorescence in situ hybridization, and targeted next-generation sequencing. OGM detected nearly all clinically relevant cytogenetic abnormalities that SCG identified with >99% sensitivity, provided the clonal burden was above 20%. OGM identified additional cytogenomic aberrations and/or provided information on fusion genes in 77 (48%) patients, including eight patients with normal karyotypes and four with failed karyotyping. The most common additional alterations identified by OGM included chromoanagenesis (n = 23), KMT2A partial tandem duplication (n = 11), rearrangements involving MECOM (n = 7), NUP98 (n = 2), KMT2A (n = 2), JAK2 (n = 2), and other gene fusions in 17 patients, with 10 showing novel fusion gene partners. OGM also pinpointed fusion genes in 17 (11%) patients where chromosomal rearrangements were concurrently detected by OGM and CBA. Overall, 24 (15%) aberrations were identified exclusively by OGM and had the potential to alter AML classification, risk stratification, and/or clinical trial eligibility. OGM emerges as a powerful tool for identifying fusion genes and detecting subtle or cryptic cytogenomic aberrations that may otherwise remain undetectable by CBA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d116f1be018b74ce05fc5532f2fa6f900955563" target='_blank'>
              Optical genome mapping improves the accuracy of classification, risk stratification, and personalized treatment strategies for patients with acute myeloid leukemia.
              </a>
            </td>
          <td>
            S. Loghavi, Qing Wei, F. Ravandi, Andres E. Quesada, M. Routbort, Shimin Hu, G. Toruner, Sa A Wang, Wei Wang, Roberto N. Miranda, Shaoying Li, Jie Xu, Courtney D. DiNardo, Naval G Daver, T. Kadia, G. Issa, H. Kantarjian, L. J. Medeiros, G. Tang
          </td>
          <td>2024-07-17</td>
          <td>American journal of hematology</td>
          <td>1</td>
          <td>224</td>
        </tr>

        <tr id="Long-read sequencing technology enables highly accurate detection of allele-specific RNA expression, providing insights into the effects of genetic variation on splicing and RNA abundance. Furthermore, the ability to directly sequence RNA promises the detection of RNA modifications in tandem with ascertaining the allelic origin of each molecule. Here, we leverage these advantages to determine allele-biased patterns of N6-methyladenosine (m6A) modifications in native mRNA. We utilized human and mouse cells with known genetic variants to assign allelic origin of each mRNA molecule combined with a supervised machine learning model to detect read-level m6A modification ratios. Our analyses revealed the importance of sequences adjacent to the DRACH-motif in determining m6A deposition, in addition to allelic differences that directly alter the motif. Moreover, we discovered allele-specific m6A modification (ASM) events with no genetic variants in close proximity to the differentially modified nucleotide, demonstrating the unique advantage of using long reads and surpassing the capabilities of antibody-based short-read approaches. This technological advancement promises to advance our understanding of the role of genetics in determining mRNA modifications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc670683629b81daed9b31ffe9b2b01d01648b9e" target='_blank'>
              Long-read RNA sequencing reveals allele-specific N6-methyladenosine modifications
              </a>
            </td>
          <td>
            Dayea Park, Can Cenik
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The endonuclease activity of Pms1 directs mismatch repair by generating a nick in the newly replicated DNA strand. Inactivating Pms2, the human homologue of yeast Pms1, increases the chances of colorectal and uterine cancers. Here we use whole genome sequencing to show that loss of this endonuclease activity, via the pms1-DE variant, results in strong mutator effects throughout the Saccharomyces cerevisiae genome. Mutation rates are strongly increased for mutations resulting from all types of single-base substitutions and for a wide variety of single- and multi-base indel mutations. Rates for these events are further increased in strains combining pms1-DE with mutator variants of each of the three major leading and lagging strand replicases. In all cases, mutation rates, spectra, biases, and context preferences are statistically indistinguishable from strains with equivalent polymerases but lacking initial mismatch recognition due to deletion of MSH2. This implies that, across the nuclear genome, strand discrimination via the Pms1 endonuclease is as important for MMR as is initial mismatch recognition by Msh2 heterodimers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c81863bf8b4bcd0533ef4d09eddc5a93c0da4b" target='_blank'>
              Instability throughout the Saccharomyces cerevisiae genome resulting from Pms1 endonuclease deficiency.
              </a>
            </td>
          <td>
            S. Lujan, Marta A. Garbacz, S. E. Liberti, Adam B Burkholder, Thomas A. Kunkel
          </td>
          <td>2024-07-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="With age, hematopoietic stem cells can acquire somatic mutations in leukemogenic genes that confer a proliferative advantage in a phenomenon termed “clonal hematopoiesis of indeterminate potential” (CHIP). How these mutations confer a proliferative advantage and result in increased risk for numerous age-related diseases remains poorly understood. We conducted a multiracial meta-analysis of epigenome-wide association studies (EWAS) of CHIP and its subtypes in four cohorts (N=8196) to elucidate the molecular mechanisms underlying CHIP and illuminate how these changes influence cardiovascular disease risk. The EWAS findings were functionally validated using human hematopoietic stem cell (HSC) models of CHIP. A total of 9615 CpGs were associated with any CHIP, 5990 with DNMT3A CHIP, 5633 with TET2 CHIP, and 6078 with ASXL1 CHIP (P <1×10−7). CpGs associated with CHIP subtypes overlapped moderately, and the genome-wide DNA methylation directions of effect were opposite for TET2 and DNMT3A CHIP, consistent with their opposing effects on global DNA methylation. There was high directional concordance between the CpGs shared from the meta-EWAS and human edited CHIP HSCs. Expression quantitative trait methylation analysis further identified transcriptomic changes associated with CHIP-associated CpGs. Causal inference analyses revealed 261 CHIP-associated CpGs associated with cardiovascular traits and all-cause mortality (FDR adjusted p-value <0.05). Taken together, our study sheds light on the epigenetic changes impacted by CHIP and their associations with age-related disease outcomes. The novel genes and pathways linked to the epigenetic features of CHIP may serve as therapeutic targets for preventing or treating CHIP-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb071f9470db1b0d43e08fc0664a4d81582bdd97" target='_blank'>
              Epigenome-wide DNA Methylation Association Study of CHIP Provides Insight into Perturbed Gene Regulation
              </a>
            </td>
          <td>
            Daniel Levy, Sara Kirmani, T. Huan, Joseph Van Amburg, R. Joehanes, M. Uddin, N. Nguyen, Bing Yu, Jennifer A. Brody, M. Fornage, Jan Bressler, N. Sotoodehnia, David Ong, Fabio Puddu, James Floyd, Christie M. Ballantyne, B. Psaty, L. Raffield, Pradeep Natarajan, Karen Conneely, April P. Carson, Leslie Lange, Kendra Ferrier, N. HEARD-COSTA, Joanne M Murabito, A. Bick
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Understanding bacterial gene function remains a major biological challenge. Double-mutant genetic interaction (GI) analysis addresses this challenge by uncovering the functional partners of targeted genes, allowing us to associate genes of unknown function with novel pathways and unravel connections between well-studied pathways, but is difficult to implement at the genome-scale. Here, we develop and use double-CRISPRi to systematically quantify genetic interactions at scale in the Bacillus subtilis envelope, including essential genes. We discover > 1000 known and novel genetic interactions. Our analysis pipeline and experimental follow-ups reveal the distinct roles of paralogous genes such as the mreB and mbl actin homologs, and identify new genes involved in the well-studied process of cell division. Overall, our study provides valuable insights into gene function and demonstrates the utility of double-CRISPRi for high-throughput dissection of bacterial gene networks, providing a blueprint for future studies in diverse bacterial species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed882175f2414bae232e1b044a85ff77b9c005c" target='_blank'>
              Comprehensive double-mutant analysis of the Bacillus subtilis envelope using double-CRISPRi
              </a>
            </td>
          <td>
            B. Koo, Horia Todor, Jiawei Sun, Jordi van Gestel, John S. Hawkins, Cameron C. Hearne, Amy B. Banta, K. C. Huang, Jason M. Peters, Carol A Gross
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfda792d36cb0c85bc9dd50884926690c096c37e" target='_blank'>
              RNA polymerase stalling-derived genome instability underlies ribosomal antibiotic efficacy and resistance evolution
              </a>
            </td>
          <td>
            Yayun Zheng, Ruochen Chai, Tian Wang, Zeqi Xu, Yihui He, P. Shen, Jintao Liu
          </td>
          <td>2024-08-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="What Is the Issue? 
The first therapeutics based on clustered regularly interspaced short palindromic repeats (CRISPR) technologies are entering the market. These gene editing technologies have the potential to change treatment paradigms and may be used to treat conditions that cannot be treated or cured with current methods. This report aims to provide an overview of the technologies and their current and potential roles in health care. 
What Are the Technologies? 
CRISPR is a part of bacterial immune systems that can cut DNA strands and is used as a gene editing tool. A guide ribonucleic acid (RNA) sequence leads the CRISPR-associated nuclease to the target DNA sequence where the cut is made. These edits change the function of the gene, making genes nonfunctional or replacing the coding sequence for 1 gene with another. CRISPR can also be used to increase or decrease the expression of specific genes. 
What Is the Potential Impact? 
CRISPR-based technologies have a variety of potential applications in health care, including: 

treating genetic diseases 
understanding the genetic mechanisms of diseases and investigating the relevance of potential drug treatments 
managing infectious diseases through detection, treatment, and elimination. 

What Else Do We Need to Know? 
Ethical issues pertinent to the use of CRISPR include the ability to obtain adequately informed consent, the potential future consequences of gene editing and its potential unintended effects, and the impact gene editing could have on future generations. The long-term effects of CRISPR-based therapies are currently unknown. Further research into emerging applications is required. Long-term follow-up of the patients who have received the first CRISPR-based therapeutics will help inform understanding of the safety and effectiveness of these treatments. While the first of these therapies have been granted regulatory authorization, the next viable CRISPR-based therapies are still in the early phases of development, with the pivotal clinical trials not expected to be completed until at least 2027.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0814cd12a7b72cf57aa46663d5db2b368ed9efd" target='_blank'>
              CRISPR Technologies for In Vivo and Ex Vivo Gene Editing
              </a>
            </td>
          <td>
            Cadth Horizon, Scan
          </td>
          <td>2024-07-19</td>
          <td>Canadian Journal of Health Technologies</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8867a66b53ced02c166da3c33e0be6474929e7d1" target='_blank'>
              Somatic CpG hypermutation is associated with mismatch repair deficiency in cancer.
              </a>
            </td>
          <td>
            Aidan Flynn, Sebastian M. Waszak, Joachim Weischenfeldt
          </td>
          <td>2024-07-18</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A. DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="microRNAs (miRNAs), tiny, non-coding RNA molecules, fine-tune the expression of target genes through interacting with mRNAs. These miRNAs are involved in a wide range of biological processes, encompassing cell division, death, blood cell production, and tumor development. When these miRNAs become dysfunctional, they can promote the invasion and spread of cancer cells in various human malignancies, including leukemia. Acute lymphoblastic leukemia (ALL), the preeminent malignancy affecting children, is a blood cancer marked by the uncontrollable growth of immature lymphoid cells that displace healthy blood precursors in the bone marrow. Despite a decline in ALL mortality rates over the past two decades, a significant proportion of deaths still results from a lack of effective diagnostic and prognostic markers that can guide treatment decisions and overcome drug resistance. The analysis of miRNA expression patterns in ALL could lead to more precise disease classification, earlier diagnosis, and better prognostic outcomes in the near future. The connection between miRNA dysfunction and the biology of ALL suggests that these molecules could represent promising therapeutic targets. Therefore, this review delves into the regulatory mechanisms of miRNAs in pediatric ALL, exploring how miRNA-based diagnostic, prognostic, and therapeutic strategies offer unique advantages and hold promise for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b51d3730adcae846d16d92db76f51f153c5c1d0" target='_blank'>
              Advancements in the Regulatory Role of microRNAs in Childhood Acute Lymphoblastic Leukemia: Mechanisms and Clinical Implications
              </a>
            </td>
          <td>
            Wei Deng
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd083194686b61ccc53ca0d4c394b67971b4cdb7" target='_blank'>
              Targeted gene therapy for cancer: the impact of microRNA multipotentiality
              </a>
            </td>
          <td>
            Nourhan A. Abou Madawi, Zeinab E Darwish, Enas M Omar
          </td>
          <td>2024-08-01</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daaaeb917f1640a6de2396b29c1a5a0f942bd758" target='_blank'>
              Enhanced cancer cell proliferation and aggressive phenotype counterbalance in breast cancer with high BRCA1 gene expression.
              </a>
            </td>
          <td>
            Kohei Chida, M. Oshi, A. M. Roy, Takumi Sato, Maya Penelope Takabe, Li Yan, I. Endo, Kenichi Hakamada, K. Takabe
          </td>
          <td>2024-07-07</td>
          <td>Breast cancer research and treatment</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Leukemia is a malignant tumor with high heterogeneity and a complex evolutionary process. It is difficult to resolve the heterogeneity and clonal evolution of leukemia cells by applying traditional bulk sequencing techniques, thus preventing a deep understanding of the mechanisms of leukemia development and the identification of potential therapeutic targets. However, with the development and application of single-cell sequencing technology, it is now possible to investigate the gene expression profile, mutations, and epigenetic features of leukemia at the single-cell level, thus providing a new perspective for leukemia research. In this article, we review the recent applications and advances of single-cell sequencing technology in leukemia research, discuss its potential for enhancing our understanding of the mechanisms of leukemia development, discovering therapeutic targets and personalized treatment, and provide reference guidelines for the significance of this technology in clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059e4fa52cbb0003a82e3f417ae0958def767ace" target='_blank'>
              Application and research progress of single cell sequencing technology in leukemia
              </a>
            </td>
          <td>
            Dan Xie, Bangquan An, Mingyue Yang, Lei Wang, Min Guo, Heng Luo, Shengwen Huang, Fa Sun
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Therapy-induced senescence (TIS) represents a major cellular response to anticancer treatments. Both malignant and non-malignant cells in the tumor microenvironment undergo TIS and may be harmful for cancer patients since TIS cells develop a senescence-associated secretory phenotype (SASP) that can sustain tumor growth. The SASP also modulates anti-tumor immunity, although the immune populations involved and the final results appear to be context-dependent. In addition, senescent cancer cells are able to evade senescence growth arrest and to resume proliferation, likely contributing to relapse. So, research data suggest that TIS induction negatively affects therapy outcomes in cancer patients. In line with this, new interventions aimed at the removal of senescent cells or the reprogramming of their SASP, called senotherapy, have become attractive therapeutic options. To date, the lack of reliable, cost-effective, and easy-to-use TIS biomarkers hinders the application of recent anti-senescence therapeutic approaches in the clinic. Hence, the identification of biomarkers for the detection of TIS tumor cells and TIS non-neoplastic cells is a high priority in cancer research. In this review article, we describe the current knowledge about TIS, outline critical gaps in our knowledge, and address recent advances and novel approaches for the discovery of TIS biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80b8874610a137e048d67c39fb5ecaa332e59e90" target='_blank'>
              Therapy-Induced Senescence: Novel Approaches for Markers Identification
              </a>
            </td>
          <td>
            F. Pacifico, Fulvio Magni, Antonio Leonardi, E. Crescenzi
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="One of the main causes of cancer-related mortality for women worldwide is breast cancer (BC). The XRCC2 gene, essential for DNA repair, has been implicated in cancer susceptibility. This study aims to evaluate the association between XRCC2 and BC risk. The study was conducted at Zheen International Hospital in Erbil, Iraq, between 2021 and 2024 with a total of 88 samples, including 44 paired normal and cancer tissue samples. Mutation analysis was performed using Next-Generation Sequencing, coupled with in silico tools for variant impact prediction. Expression levels were assessed through RT-PCR, and methylation status was determined using methylation-sensitive restriction enzyme digestion PCR. The study identified seven inherited germline variants in the XRCC2 gene, with five of these mutations being Uncertain Significance, one being Likely Pathogenic, and one being Likely benign. RNA purity was found high with mean A260/280 ratios of 1.986 ± 0.097 in normal (N) and 1.963 ± 0.092 in tumor (T) samples. Tumor samples exhibited a higher RNA concentration (78.56 ± 40.87 ng/µL) than normal samples (71.44 ± 40.79 ng/µL). XRCC2 gene expression was significantly upregulated in tumor tissue, with marked increases in patients aged 40-55 and >56 years and in higher cancer grades (II and III) and invasive ductal carcinoma (p-values ranging from <0.0001 to 0.0392). DNA methylation rates in tumor tissues were low (7%), suggesting limited regulation by methylation. The study suggests that XRCC2 can be classified as an oncogene and that its structural investigation by targeted NGS and expression evaluation can be used as a potential biomarker in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81c449dfd85b137aba37b45f3107a8eae5435a35" target='_blank'>
              Association of XRCC2 with breast cancer, a multi-omics analysis at genomic, transcriptomic, and epigenomic level.
              </a>
            </td>
          <td>
            N. Gilani, Mehmet Ozaslan
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Due to the significant morbidity and mortality of hemoglobinopathies, curative options have long been pursued. The overall goal of gene therapy is to modify a patient's own hematopoietic stem cells to overcome the deleterious effects of the underlying genetic defect by gene addition, gene editing, or gene silencing. Gene addition incorporates genes with superior function than the abnormal gene; gene editing takes advantage of molecular tools such as zinc finger proteins, Transcription Activator-Like Effector Nucleases and Clustered Regularly Interspaced Short Palindromic Repeats coupled with Cas9 proteins (CRISPR-Cas9) which allow for sequence-specific breaks in DNA that disrupt gene function; and gene silencing suppresses gene expression by interference with mRNA transcription/protein translation or epigenetic modification. The majority of gene therapy strategies for hemoglobinopathies have targeted erythroid-specific BCL11A, a major regulator of fetal hemoglobin repression at the gamma-globin locus, in the normal fetal-to-adult hemoglobin switch that occurs shortly after birth. Other goals have involved the incorporation of anti-sickling globins, such as βT87Q or βAS3. Landmark clinical trials of gene therapy in transfusion-dependent thalassemia and sickle cell disease have shown remarkable efficacy and acceptable safety and culminated in recent regulatory approvals of gene therapy for both diseases in Europe and the United States.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f812c9e54de47feca0c8615f3dda20f25f85b41" target='_blank'>
              A Review of Gene Therapies for Hemoglobinopathies.
              </a>
            </td>
          <td>
            Boubini Jones-Wonni, Amar H Kelkar, Maureen O Achebe
          </td>
          <td>2024-08-15</td>
          <td>Hemoglobin</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Current treatments for KRAS-mutant colorectal cancers (CRCs) are often limited by cellular plasticity and rewiring responses. Here we describe a promising therapeutic strategy that simultaneously targets epigenetic and oncogenic signals. Specifically, we show that inhibitors of the histone methyltransferase, EZH2, synergize with various RAS pathway inhibitors and promote dramatic tumor regression in vivo. Together these agents cooperatively suppress WNT-driven transcription and drive CRCs into a more differentiated cell state by inducing the Groucho/TLE corepressor, TLE4, along with a network of WNT pathway inhibitors and intestinal differentiation proteins. However, these agents also induce the pro-apoptotic protein BMF, which subsequently kills these more differentiated cells. Accordingly, cell death can be prevented by activating β-catenin, blocking differentiation, or by ablating BMF expression. Collectively, these studies reveal a new therapeutic approach for treating KRAS-mutant CRCs and illustrate a critical convergence of EZH2 and RAS on oncogenic WNT signals, intestinal differentiation, and apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47a75ff581332b9384e613ce0d1c80e31add042e" target='_blank'>
              Epigenetic and oncogenic inhibitors cooperatively drive differentiation and kill KRAS-mutant colorectal cancers.
              </a>
            </td>
          <td>
            P. Loi, Amy E. Schade, Carrie L Rodriguez, Anjana Krishnan, N. Perurena, Van T M Nguyen, Yilin Xu, Marina Watanabe, Rachel A Davis, Alycia Gardner, Natalie F Pilla, Kaia Mattioli, Olesja Popow, Nuray Gunduz, T. Lannagan, Samantha Fitzgerald, Ewa T Sicinska, Jia-Ren Lin, William Tan, L. Brais, Kevin Haigis, M. Giannakis, K. Ng, Sandro Santagata, Kristian Helin, Owen J. Sansom, K. Cichowski
          </td>
          <td>2024-08-12</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Aberrant gene expression patterns in acute myeloid leukemia (AML) with balanced chromosomal translocations are often associated with dysregulation of epigenetic modifiers. The AML1/ETO (RUNX1/MTG8) fusion protein, caused by the translocation (8;21)(q22;q22), leads to the epigenetic repression of its target genes. We aimed in this work to identify critical epigenetic modifiers, on which AML1/ETO-positive AML cells depend on for proliferation and survival using shRNA library screens and global transcriptomics approaches. Using shRNA library screens, we identified 41 commonly depleted genes in two AML1/ETO-positive cell lines Kasumi-1 and SKNO-1. We validated, genetically and pharmacologically, DNMT1 and ATR using several AML1/ETO-positive and negative cell lines. We also demonstrated in vivo differentiation of myeloblasts after treatment with the DNMT1 inhibitor decitabine in a patient with an AML1/ETO-positive AML. Bioinformatic analysis of global transcriptomics after AML1/ETO induction in 9/14/18-U937 cells identified 973 differentially expressed genes (DEGs). Three genes (PARP2, PRKCD, and SMARCA4) were both downregulated after AML1/ETO induction, and identified in shRNA screens. In conclusion, using unbiased shRNA library screens and global transcriptomics, we have identified several driver epigenetic regulators for proliferation in AML1/ETO-positive AML. DNMT1 and ATR were validated and are susceptible to pharmacological inhibition by small molecules showing promising preclinical and clinical efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4848d5afaf1e1f3f5c753596a2ed8d1e491ca9f0" target='_blank'>
              Identification of epigenetic modifiers essential for growth and survival of AML1/ETO-positive leukemia.
              </a>
            </td>
          <td>
            J. Duque-Afonso, P. Veratti, Usama-Ur Rehman, Heike Herzog, J. Mitschke, G. Greve, Julian Eble, Bettina Berberich, Johanna Thomas, Milena Pantic, Miguel Waterhouse, Gaia Gentile, Olaf Heidenreich, Cornelius Miething, Michael Lübbert
          </td>
          <td>2024-08-15</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="DNA repair genomic aberrations in the Homologous Recombination pathway are identifiable in up to 25% of patients with advanced prostate cancer, making them more likely to benefit from treatment with poly (ADP-ribose) polymerase inhibitors (PARPi) alone or in combination with other therapies, particularly when BRCA driver genomic aberrations are documented. Although several clinical trials have demonstrated the efficacy of this approach, the validation of reliable biomarkers predictive of response still needs further improvement to refine patient selection. In this setting, the characterization of resistance mechanisms and the validation of novel biomarkers are critical to maximize clinical benefit and to develop novel treatment combinations to improve outcomes. In this review, we summarize the development of PARPi in prostate cancer as single agent as well as the efficacy of their combination with other drugs, and the future directions for their implementation in the management of advanced prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332cf66eb0d0e0669852e95223ac366cb113738f" target='_blank'>
              PARP inhibitors alone or in combination for prostate cancer
              </a>
            </td>
          <td>
            M. D. Fenor de la Maza, J. L. Pérez Gracia, B. Miñana, Elena Castro
          </td>
          <td>2024-01-01</td>
          <td>Therapeutic Advances in Urology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/761ad1cbab42eef53a71e8966883fae3c426f6a1" target='_blank'>
              Novel therapeutic agents in clinical trials: emerging approaches in cancer therapy
              </a>
            </td>
          <td>
            Deepak Chandra Joshi, Anurag Sharma, Sonima Prasad, Karishma Singh, Mayank Kumar, Kajal Sherawat, H. Tuli, Madhu Gupta
          </td>
          <td>2024-08-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR-T) cells that are obtained from the specific patient, modified genetically ex vivo, and possess the remarkable capability to identify and eradicate targeted cancer cells. These modified cells are subsequently reintroduced into the patient, effectively treating blood cancer. CAR-T therapy is approved to be applied in leukemia due to its great clinical therapeutic effect on B cell hematological malignancies. However, solid tumors are more resistant to this therapy for many reasons. The abnormal vascular structure of solid tumors hampers CAR-T cell trafficking. Various kinds of immunosuppressive cells and chemicals in the tumor microenvironment (TME) accelerates CAR-T cell exhaustion, showing poor persistence in vivo. More and more researches have demonstrated that T cell fate is strongly associated with epigenetic regulation. Epigenetic modification is not a direct addition or deletion of DNA, but a reversible method including modifications on DNA and histones, and non-coding RNA (ncRNA)-mediated regulations. The change of epigenetic landscape in CAR-T cells largely determines the therapeutic performance in vivo. This research outlines three major barriers in CAR-T therapy, including T cell exhaustion, differentiation and infiltration. Additionally, the research elucidates several promising epigenetic reprogramming strategies to reduce CAR-T cells exhaustion, modulate the cell differentiation process, and enhance their infiltration into solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4165f9901246426b44e74fa80485927b8d025d28" target='_blank'>
              Epigenetic Strategies to Optimize CAR-T Therapy
              </a>
            </td>
          <td>
            Danyang Song
          </td>
          <td>2024-07-11</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The XmnI Gg -158 C/T polymorphism has been widely associated with fetal hemoglobin (HbF) levels, the severity of disease, and the response to the drug hydroxyurea (HU) in both β-thalassemia (β-thal) and sickle cell disease (SCD) patients. However, the functional significance of this single nucleotide polymorphism (SNP) remains unclear. To gain insight, green fluorescence protein (GFP) cassettes harboring the XmnI C or T alleles in their left homology arms (i.e. Gg promoters) were knocked into the Gg gene(s) of K562 cells via CRISPR/Cas9. Subsequently, the GFP fluorescence levels were compared in the ensuing cell populations and isolated clones. In both instances, median fluorescence intensities (MFI) of the knockin cells having the inserted XmnI T allele were higher than those having the XmnI C allele. Our results suggest that the XmnI T allele can increase Gg expression in K562 cells. The possible functional significance of the XmnI Gg -158 C/T polymorphism provides a rationale for the aforementioned associations. Furthermore, the XmnI polymorphism as a functional SNP substantiates its importance as a prognostic marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bedc9fe1b28e8e9a54e59969786c8981bcb8830d" target='_blank'>
              Genome editing in K562 cells suggests a functional role for the XmnI Gg polymorphism: a widely used genetic marker in β-thalassemia and sickle cell disease patients.
              </a>
            </td>
          <td>
            A. Ahmadifard, Nahal Maroofi, Maryam Maleki Tehrani, Tahere Dabestani, Masoumeh Sadat Mousavi Maleki, Sepideh Bayrami, M. Banan
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Multidrug resistance (MDR) frequently occurs during cancer therapy and remains a major obstacle for the cure of most cancers(1, 2). Drug resistance could exist intrinsically or be acquired by drug treatment(3–5), yet factors that regulate the resistance remain elusive. Here, we show that most anticancer drugs damage neosynthesized proteins prior to reaching their canonic targets and elicit profound cytotoxicity, which is largely compensated by protein damage response (PDR). We demonstrate that the PDR includes damage recognition and clearance that are mainly mediated by ubiquitin and proteasome systems, although some other factors, including cellular ATP levels and proliferation status, are also involved. We show that cancer stem cells (CSCs), which have lower protein synthesis, and drug resistance acquired cells (DRAC), which have higher proteasome activity, are more resistant than other cells. We further demonstrate that ATP promotes protein synthesis and suppresses proteasome activity, thus, increasing mitochondrial ATP production by PDK1 inhibition and using proteasome inhibitor to block protein damage clearance render CSCs and DRACs more vulnerable to anticancer drugs. Thus, patients with drug-resistant cancers and treatment-naïve patients with low ATP levels and/or high proteasome activity can be identified and subtyped, and therapies containing PDK1-I and/or proteasome-I may be effective options for these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e9626795c580f97c2519d1342441e9145cbcfc" target='_blank'>
              A critical role of protein damage response in mediating cancer drug resistance
              </a>
            </td>
          <td>
            Fangyuan Shao, Jie Li, Hao Xiao, Ling Li, Bo Li, YuJun Chen, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Yuzhong Peng, Yujian Huang, Lijian Wang, Yanxia Shi, Nan Shao, Kai Miao, Changhua Zhang, Ying Lin, Jun Yan, Kin Yip Tam, Xiaoling Xu, Chu-Xia Deng
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Homologous recombination rearranges genetic information during meiosis to generate new combinations of variants. Recombination also causes new mutations, affects the GC content of the genome and reduces selective interference. Here, we use HiFi long-read sequencing to directly detect crossover and gene conversion events from switches between the two haplotypes along single HiFi-reads from testis tissue of humans, chimpanzees and gorillas as well as human sperm samples. Furthermore, based on DNA methylation calls, we classify the cellular origin of reads to either somatic or germline cells in the testis tissue. We identify 1692 crossovers and 1032 gene conversions in nine samples and investigate their chromosomal distribution. Crossovers are more telomeric and correlate better with recombination maps than gene conversions. We show a strong concordance between a human double-strand break map and the human samples, but not for the other species, supporting different PRDM9-programmed double-strand break loci. We estimate the average gene conversion tract lengths to be similar and very short in all three species (means 40-100 bp, fitted well by a geometric distribution) and that 95-98% of non-crossover events do not involve tracts intersecting with polymorphism and are therefore not detectable. Finally, we detect a GC bias in the gene conversion of both single and multiple SNVs and show that the GC-biased gene conversion affects SNVs flanking crossover events. This implies that gene conversion events associated with crossover events are much longer (estimated above 500 bp) than those associated with non-crossover events. Highly accurate long-read sequencing combined with the classification of reads to specific cell types provides a new, powerful way to make individual, detailed maps of gene conversion and crossovers for any species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e343940af051e78f3a88674b4f23d5e3391f5fb" target='_blank'>
              Insights into gene conversion and crossing-over processes from long-read sequencing of human, chimpanzee and gorilla testes and sperm
              </a>
            </td>
          <td>
            Peter Soerud Porsborg, A. P. Charmouh, Vinod Kumar Singh, S. B. Winge, C. Hvilsom, Marta Pelizzola, S. Laurentino, N. Neuhaus, A. Hobolth, Thomas Bataillon, Kristian Almstrup, Søren Besenbacher, M. Schierup
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Bacteriophages are the most abundant entities on earth and exhibit vast genetic and phenotypic diversity. Exploitation of this largely unexplored molecular space requires identification and functional characterisation of genes that act at the phage-host interface. To date, this is restricted to few model phage-host systems that are amenable to genetic manipulation. To overcome this limitation, we introduce a non-genetic mRNA targeting approach using exogenous delivery of programmable antisense oligomers (ASOs) to silence genes of both DNA and RNA phages. A systematic knockdown screen of core and accessory genes of the nucleus-forming jumbo phage ΦKZ, coupled to RNA-sequencing and microscopy analyses, reveals previously unrecognised proteins that are essential for phage replication and that, upon silencing, elicit distinct phenotypes at the level of the phage and host response. One of these factors is a ΦKZ-encoded nuclease that acts at a major decision point during infection, linking the formation of the protective phage nucleus to phage genome amplification. The non-genetic ASO-based gene silencing used here promises to be a versatile tool for molecular discovery in phage biology, will help elucidate defence and anti-defence mechanisms in non-model phage-host pairs, and offers potential for optimising phage therapy and biotechnological procedures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d6a132616204d6fa19229a092ac9afe5732d73" target='_blank'>
              Non-genetic messenger RNA silencing reveals essential genes in phage-host interplay
              </a>
            </td>
          <td>
            Milan Gerovac, Svetlana Ðurica-Mitić, Leandro Buhlmann, Valentin Rech, Yan Zhu, Samuel Carien, Linda Popella, Jörg Vogel
          </td>
          <td>2024-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Simple Summary The classification of breast cancer has been a complex subject, with diverse origins of normal breast epithelial cells and evolutionary pathways contributing to its extensive heterogeneity, which further complicates the treatment process. We developed a dynamic network model to simulate the emergence of different cellular states and explore the underlying biological mechanisms. This model revealed cellular states that align with four recognized molecular subtypes of breast cancer. It also identified feedback loops that either reinforce or facilitate phenotypic transitions, and allowed us to trace these transitions through the model’s topological structure. This approach provides new insights into the mechanisms driving breast cancer heterogeneity and may benefit the development of more targeted therapeutic strategies. Abstract Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii Der, Ida Chen, C. Charles, Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>None</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="Abstract CRISPR/Cas9 genome editing is a pervasive research tool due to its relative ease of use. However, some systems are not amenable to generating edited clones due to genomic complexity and/or difficulty in establishing clonal lines. For example, Drosophila Schneider 2 (S2) cells possess a segmental aneuploid genome and are challenging to single-cell select. Here, we describe a streamlined CRISPR/Cas9 methodology for knock-in and knock-out experiments in S2 cells, whereby an antibiotic resistance gene is inserted in-frame with the coding region of a gene-of-interest. By using selectable markers, we have improved the ease and efficiency for the positive selection of null cells using antibiotic selection in feeder layers followed by cell expansion to generate clonal lines. Using this method, we generated the first acentrosomal S2 cell lines by knocking-out centriole genes Polo-like Kinase 4/Plk4 or Ana2 as proof of concept. These strategies for generating gene-edited clonal lines will add to the collection of CRISPR tools available for cultured Drosophila cells by making CRISPR more practical and therefore improving gene function studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d9bec46a5fefb60dc99b1e4c11b9d69666ffa32" target='_blank'>
              Generating CRISPR-edited clonal lines of cultured Drosophila S2 cells
              </a>
            </td>
          <td>
            John M. Ryniawec, Anastasia Amoiroglou, Gregory C. Rogers
          </td>
          <td>2024-08-17</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Precision medicine approaches such as gene editing via clustered regularly interspaced short palindromic repeat (CRISPR) and their associated protein (Cas-9) are revolutionizing treatment strategies for complex diseases such as Alzheimer and cancer. In this review, we explore the application of CRISPR-Cas9 in advancing therapies for these conditions, as well as its potential in targeting senescent cells. Since the risk of Alzheimer’s disease is strongly associated with genetic mutations and variations, the use of gene editing technologies to rectify these genetic errors by modifying disease-associated proteins becomes relevant. For cancer, to enhance immunotherapy approaches, modification of immune cells have been utilized to improve their anti-tumor efficacy. Additionally, the review also investigates the role of CRISPR-Cas9 in targeting senescent cells, which are implicated in both aging-related disorders and cancer progression. While challenges remain in introducing delivery methods and specificity, CRISPR-Cas9 represents a significant advancement in developing targeted, personalized treatments for these challenging health issues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0022d5a1dcbe7db7f0bd9b012fe358d45973badd" target='_blank'>
              Precision Gene Editing Strategies with CRISPR-Cas9 for Advancing Cancer Immunotherapy and Alzheimer's Disease
              </a>
            </td>
          <td>
            Priyanka Gupte, Karan Dhingra, Saloni
          </td>
          <td>2024-07-28</td>
          <td>Journal of Knowledge Learning and Science Technology ISSN: 2959-6386 (online)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c24f0b42670eaa7d910971701aacf3a8d5213b" target='_blank'>
              Targeting RBM39 through indisulam induced mis-splicing of mRNA to exert anti-cancer effects in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tongting Ji, Yang Yang, Juanjuan Yu, Hongli Yin, Xinran Chu, Pengju Yang, Ling Xu, Xiaodong Wang, Shaoyan Hu, Yizhen Li, Xiaochen Wu, Wengyuan Liu, Bi Zhou, Wenjuan Wang, Shuqi Zhang, Wei Cheng, Yanling Chen, Lei Shi, Zhiheng Li, Ran Zhuo, Yongping Zhang, Yanfang Tao, Di Wu, Xiao-Lu Li, Zimu Zhang, Jun-Jie Fan, Jian Pan, Jun Lu
          </td>
          <td>2024-07-24</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Single-cell whole-genome sequencing (scWGS) detects cell heterogeneity at the aspect of genomic variations, which are inheritable and play an important role in life processes such as aging and cancer progression. The recent explosive development of high-throughput single-cell sequencing methods has enabled high-performance heterogeneity detection through a vast number of novel strategies. Despite the limitation on total cost, technical advances in high-throughput single-cell whole-genome sequencing methods are made for higher genome coverage, parallel throughput, and level of integration. This review highlights the technical advancements in high-throughput scWGS in the aspects of strategies design, data efficiency, parallel handling platforms, and their applications on human genome. The experimental innovations, remaining challenges, and perspectives are summarized and discussed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8eab54ef3e4fd4f2f8644136b03c33f6ec8dc590" target='_blank'>
              Recent Innovations and Technical Advances in High-Throughput Parallel Single-Cell Whole-Genome Sequencing Methods.
              </a>
            </td>
          <td>
            Y. Qiao, Tianguang Cheng, Zikun Miao, Yue Cui, Jing Tu
          </td>
          <td>2024-07-09</td>
          <td>Small methods</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Precise gene regulation and programmable RNA editing are vital RNA-level regulatory mechanisms. Gene repression tools grounded in small non-coding RNAs, microRNAs, and CRISPR-dCas proteins, along with RNA editing tools anchored in Adenosine Deaminases acting on RNA (ADARs), have found extensive application in molecular biology and cellular engineering. Here, we introduced a novel approach wherein we developed an EcCas6e mediated crRNA–mRNA annealing system for gene repression in Escherichia coli and RNA editing in Saccharomyces cerevisiae. We found that EcCas6e possesses inherent RNA annealing ability attributed to a secondary positively charged cleft, enhancing crRNA–mRNA hybridization and stability. Based on this, we demonstrated that EcCas6e, along with its cognate crRNA repeat containing a complementary region to the ribosome binding site of a target mRNA, effectively represses gene expression up to 25-fold. Furthermore, we demonstrated that multiple crRNAs can be easily assembled and can simultaneously target up to 13 genes. Lastly, the EcCas6e–crRNA system was developed as an RNA editing tool by fusing it with the ADAR2 deaminase domain. The EcCas6e–crRNA mediated gene repression and RNA editing tools hold broad applications for research and biotechnology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba46fb7fdc74a6727da0c82bac09a25eb2952ee2" target='_blank'>
              EcCas6e-based antisense crRNA for gene repression and RNA editing in microorganisms
              </a>
            </td>
          <td>
            Mutong Li, Zhaohui Cai, Shucheng Song, Xinmin Yue, Wenyu Lu, Shuquan Rao, Chuanbo Zhang, Chaoyou Xue
          </td>
          <td>2024-07-12</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genomic structural variants (SVs) play a pivotal role in driving the evolution of hematologic malignancies, particularly in leukemia, in which genetic abnormalities are crucial features. Detecting SVs is essential for achieving precise diagnosis and prognosis in these cases. Karyotyping, often complemented by fluorescence in situ hybridization and/or chromosomal microarray analysis, provides standard diagnostic outcomes for various types of SVs in front-line testing for leukemia. Recently, optical genome mapping (OGM) has emerged as a promising technique due to its ability to detect all SVs identified by other cytogenetic methods within one single assay. Furthermore, OGM has revealed additional clinically significant SVs in various clinical laboratories, underscoring its considerable potential for enhancing front-line testing in cases of leukemia. This review aims to elucidate the principles of conventional cytogenetic techniques and OGM, with a focus on the technical performance of OGM and its applications in diagnosing and prognosticating myelodysplastic syndromes, acute myeloid leukemia, acute lymphoblastic leukemia, and chronic lymphocytic leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7daafa70669857dc79dda86074368e05463db4df" target='_blank'>
              Genomic structural variants analysis in leukemia by a novel cytogenetic technique: Optical genome mapping.
              </a>
            </td>
          <td>
            Song Lu, Kefu Liu, Di Wang, Yuan Ye, Zhiping Jiang, Yunhua Gao
          </td>
          <td>2024-08-24</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G. Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Juvenile myelomonocytic leukemia (JMML), a clonal hematologic malignancy, originates from mutated hematopoietic stem cells (HSCs). The mechanism sustaining the persistence of mutant stem cells, leading to leukemia development, remains elusive. In this study, we conducted comprehensive examination of gene expression profiles, transcriptional factor regulons, and cell compositions/interactions throughout various stages of tumor cell development in Ptpn11 mutation-associated JMML. Our analyses revealed that leukemia-initiating Ptpn11E76K/+ mutant stem cells exhibited de novo activation of the myeloid transcriptional program and aberrant developmental trajectories. These mutant stem cells displayed significantly elevated expression of innate immunity-associated anti-microbial peptides and pro-inflammatory proteins, particularly S100a9 and S100a8. Biological experiments confirmed that S100a9/S100a8 conferred a selective advantage to the leukemia-initiating cells through autocrine effects and facilitated immune evasion by recruiting and promoting immune suppressive myeloid-derived suppressor cells (MDSCs) in the microenvironment. Importantly, pharmacological inhibition of S100a9/S100a8 signaling effectively impeded leukemia development from Ptpn11E76K/+ mutant stem cells. These findings collectively suggest that JMML tumor-initiating cells exploit evolutionarily conserved innate immune and inflammatory mechanisms to establish clonal dominance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14ee0ff31f4868484243a76f8cf02c1e5b54c52" target='_blank'>
              Prototypical innate immune mechanism hijacked by leukemia-initiating mutant stem cells for selective advantage and immune evasion in Ptpn11-associated juvenile myelomonocytic leukemia
              </a>
            </td>
          <td>
            Cheng-Kui Qu, Hong Zheng, Peng Zhao, Zhenya Tan, Wen‐mei Yu, Juwita Werner, Elliot Stieglitz, Christopher C Porter, S. Chandrakasan, Daniel Wechsler, Simon Mendez-Ferrer
          </td>
          <td>2024-08-02</td>
          <td>Research Square</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Elucidating the genetic architecture of DNA methylation (DNAm) is crucial for decoding the etiology of complex diseases. However, current epigenomic studies often suffer from incomplete coverage of methylation sites and the use of tissues containing heterogeneous cell populations. To address these challenges, we present a comprehensive human methylome atlas based on deep whole-genome bisulfite sequencing (WGBS) and whole-genome sequencing (WGS) of purified monocytes from 298 European Americans (EA) and 160 African Americans (AA) in the Louisiana Osteoporosis Study. Our atlas enables the analysis of over 25 million DNAm sites. We identified 1,383,250 and 1,721,167 methylation quantitative trait loci (meQTLs) in cis-regions for EA and AA populations, respectively, with 880,108 sites shared between ancestries. While cis-meQTLs exhibited population-specific patterns, primarily due to differences in minor allele frequencies, shared cis-meQTLs showed high concordance across ancestries. Notably, cis-heritability estimates revealed significantly higher mean values in the AA population (0.09) compared to the EA population (0.04). Furthermore, we developed population-specific DNAm imputation models using Elastic Net, enabling methylome-wide association studies (MWAS) for 1,976,046 and 2,657,581 methylation sites in EA and AA, respectively. The performance of our MWAS models was validated through a systematic multi-ancestry analysis of 41 complex traits from the Million Veteran Program. Our findings bridge the gap between genomics and the monocyte methylome, uncovering novel methylation-phenotype associations and their transferability across diverse ancestries. The identified meQTLs, MWAS models, and data resources are freely available at www.gcbhub.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02561d4c021777b32cc89ad698157b2f333e71dc" target='_blank'>
              An atlas of genetic effects on the monocyte methylome across European and African populations
              </a>
            </td>
          <td>
            W. Zhang, X. Zhang, C. Qiu, Z. Zhang, K.-J. Su, Z. Luo, Q. Tian, M. Liu, B. Zhao, L. Wu, H. Shen, C. Wu, H.-W. Deng
          </td>
          <td>2024-08-13</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="CIZ1 is part of the large RNA-dependent supramolecular assemblies that form around the inactive X-chromosome (Xi) in female cells, and smaller assemblies throughout the nucleus in males and females. Here we show that CIZ1 C-terminal anchor domain (AD) is elevated in primary human breast tumour transcriptomes, even at stage I. Elevation of AD correlates with deprotection of chromatin, and up-regulation of lncRNA-containing gene clusters in approximately 10Mb regions that are enriched in cancer-associated genes. We modelled the effect of ectopic AD on endogenous CIZ1-Xi assemblies and observed dominant-negative interference with their re-formation after mitosis, leading to abnormal Xi assemblies similar to those in breast cancer cells, and depletion of histone modifications H2AK119ub1 and H3K27me3. Within days consistent alterations in gene expression were evident across the genome, showing that disruption of CIZ1-RNA assemblies has a destabilizing effect, likely by unscheduled exposure of underlying chromatin to modifying enzymes. Together the data argue for a dominant, potent and rapid effect of CIZ1 AD, that can deprogram established patterns of gene expression and which may predispose incipient tumours to epigenetic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44d2a58fd6870a0dd51c6ac1e82db8148da5b9e2" target='_blank'>
              Epigenetic deprogramming driven by disruption of CIZ1-RNA nuclear assemblies in early-stage breast cancers
              </a>
            </td>
          <td>
            Gabrielle L. Turvey, Ernesto López de Alba, Emma R Stewart, Heather Cook, Ahmad Alalti, Richard T. Gawne, J. Ainscough, Andrew S. Mason, Dawn Coverley
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Introduction Patients with breast cancer, especially triple-negative breast cancer, have a poor prognosis. There is still no effective treatment for this disease. Due to resistance to traditional treatments such as chemotherapy and radiation therapy, there is a need to discover novel treatment strategies to treat this disease. Ribociclib is a selective CDK4/6 inhibitor. Approximately 20% of patients with HR+ breast cancer developed primary resistance to CDK4/6 inhibitors, and more than 30% experienced secondary resistance. Since most patients experience resistance during CDK4/6 inhibitor treatment, managing this disease is becoming more challenging. Many malignant tumors abnormally express microRNA (miR)-141, which participates in several cellular processes, including drug resistance, proliferation, epithelial–mesenchymal transition, migration, and invasion. Materials and methods In the present study, we cultured MDA-MB-231 and MCF-7 cells in DMEM-F12 medium. By performing MTT assay we determined the cytotoxic effects of ribociclib on breast cancer cells, as well as determining the IC50 of it. Then, we treated the cells with ribociclib at two time points: 24 h and 72 h. After that, RNA was isolated and reverse transcribed to cDNA. Finally, we performed qRT‒PCR to evaluate how ribociclib affects the expression level of desired genes. Results and conclusion We found that ribociclib can inhibit cell growth in a dose- and time-dependent manner. We examined the mRNA expression of 4 genes. After ribociclib treatment, the mRNA expression of CDK6 and MYH10 decreased (p < 0.01, p < 0.05). The mRNA expression of CDON increased (p<0.05), but no significant changes were observed in ZEB1 mRNA expression. Furthermore, the qRT‒PCR results for miR-141 showed that the expression of miR-141 increased (p<0.01) after 72 h of treatment with ribociclib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af2742d79589071ef6d86e863e12d574976bef5" target='_blank'>
              The effect of ribociclib on the expression levels of miR-141 and CDK4/6-USP51 signaling pathway genes in MCF-7 and MDA-MB-231 cells
              </a>
            </td>
          <td>
            Shayeste Sadat Baghermanesh, Mahmoud Barati, Arshad Hosseini
          </td>
          <td>2024-08-28</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c866905f4876931f400daffcfcbf112bc7a7f651" target='_blank'>
              Single-nucleus chromatin accessibility and transcriptomic map of breast tissues of women of diverse genetic ancestry.
              </a>
            </td>
          <td>
            P. Bhat-Nakshatri, Hongyu Gao, Aditi S. Khatpe, Adedeji K. Adebayo, Patrick C McGuire, Cihat Erdogan, Duojiao Chen, Guanglong Jiang, Felicia New, Rana German, Lydia Emmert, George E Sandusky, A. Storniolo, Yunlong Liu, H. Nakshatri
          </td>
          <td>2024-08-09</td>
          <td>Nature medicine</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="B cell progenitor acute lymphoblastic leukemia (BCP-ALL) is the most common childhood malignancy, driven by multiple genetic alterations that cause maturation arrest and accumulation of abnormal progenitor B cells. Current treatment protocols with chemotherapy have led to favorable outcomes but are associated with significant toxicity and risk of side effects, highlighting the necessity for highly effective, less toxic, targeted drugs, even in subtypes with a favorable outcome. Here, we used multimodal single-cell sequencing to delineate the transcriptional, epigenetic, and immunophenotypic characteristics of 23 childhood BCP-ALLs, belonging to the BCR::ABL1-positive, ETV6::RUNX1-positive, high hyperdiploid, and recently discovered DUX4-rearranged (DUX4-r) subtypes. Projection of the ALL cells along the normal hematopoietic differentiation axis revealed a diversity in the maturation pattern between the different BCP-ALL subtypes. Whereas the BCR::ABL1-, ETV6::RUNX1-positive, and high hyperdiploidy cells mainly showed similarities to normal pro-B cells, the DUX4-r ALL cells also displayed transcriptional signatures resembling mature B cells. Focusing on the DUX4-r subtype, we found that the blast population displayed multilineage priming toward non-hematopoietic cells, myeloid, and T cell lineages, but also an activation of PI3K/AKT signaling that sensitized the cells to PI3K inhibition in vivo. Given the multilineage priming of the DUX4-r blasts with aberrant expression of the myeloid marker CD371 (CLL-1), we generated chimeric antigen receptor T cells, which effectively eliminated DUX4-r ALL cells in vivo. These results provide a detailed characterization of BCP-ALL at the single-cell level and reveal therapeutic vulnerabilities in the DUX4-r subtype with implications for the understanding of ALL biology and new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1d7a915d43d3c2ff8b079040d662a79ad558986" target='_blank'>
              Single-cell genomics details the maturation block in BCP-ALL and identifies therapeutic vulnerabilities in DUX4-r cases.
              </a>
            </td>
          <td>
            Hanna Thorsson, Rasmus Henningsson, Noelia Puente-Moncada, Pablo Peña-Martínez, Ludvig Sjöström, H. Ågerstam, Carl Sandén, M. Rissler, A. Castor, H. Marquart, S. Modvig, K. Paulsson, C. Pronk, Kjeld Schmiegelow, Axel Hyrenius-Wittsten, Christina Orsmark-Pietras, H. Lilljebjörn, T. Fioretos
          </td>
          <td>2024-07-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The tumor suppressor DAB2IP, a RasGAP and cytoplasmic adaptor protein, modulates signal transduction in response to several extracellular stimuli, negatively regulating multiple oncogenic pathways. Accordingly, the loss of DAB2IP in tumor cells fosters metastasis and enhances chemo- and radio-resistance. DAB2IP is rarely mutated in cancer but is frequently downregulated or inactivated by multiple mechanisms. Solid experimental evidence show that DAB2IP reactivation can reduce cancer aggressiveness in tumors driven by multiple different oncogenic mutations, making this protein an interesting target for anti-cancer therapy. Based on these premises, we screened a library of FDA-approved drugs to search for molecules that can increase DAB2IP protein levels. We exploited CRISPR/Cas9 gene editing to generate two prostate cancer cell models in which endogenous DAB2IP is fused to HiBiT, a peptide tag that enables luminescence-based detection of protein levels in a sensitive and quantitative manner. Using this approach, we identified drugs able to increase DAB2IP levels. We focus our attention on thiostrepton, a natural cyclic oligopeptide antibiotic that has been reported to inhibit survival of various cancer cell lines. Functional experiments revealed that the cancer inhibitory effect of thiostrepton is reduced in the absence of DAB2IP, suggesting that the observed upregulation contributes to its action. These findings encourage the further development of thiostrepton for the treatment of solid cancers, and unveil a novel molecular mechanism underlying its anti-tumoral action.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08c2cc71cd0d80258c78032230a6c729e04c6e9e" target='_blank'>
              Drug-repurposing screen identifies thiostrepton as a novel regulator of the tumor suppressor DAB2IP
              </a>
            </td>
          <td>
            Rossella De Florian Fania, S. Maiocchi, Raffaella Klima, Valeria Pellegrini, Sabrina Ghetti, Davide Selvestrel, Luca L. Fava, Luca Braga, Licio Collavin
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Sickle cell anaemia is one of the haemoglobin abnormalities resulting from a genetic mutation— it is caused by inheriting two faulty genes that result in an abnormal substitution of glutamate for valine on the beta chain of haemoglobin, which causes haemoglobin molecules to stick together. According to a World Health Organization (WHO) report, 20 out of every 1,000 births suffer from sickle-cell anaemia, and 24% of Nigerians are carriers of this mutant gene. Scientists have suggested several solutions, including stem cell transplantation and gene therapies, but these have faced opposition due to ethical beliefs, high cost, and the ensuing immune issues. Research is now centered on advancing genome editing techniques for gene therapy. Ongoing studies have proven that genetic differences can be corrected methodically by modifying the genome at specific sites instead of introducing a new copy of the affected gene into the cells; due to the effectiveness of this method, scientists are testing its applications in manipulating genes in various systems. This review correlates a few studies that used the recently developed technique—CRISPR-Cas9—as a novel approach to gene therapy, dissecting the different clinical studies about sickle cell origin to point out many of its ethical and medical limitations, the consequences of these limitations, and the advancements this technology has made possible.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/004bc1251a2e642ec5d00bbc9ab43dda709a6a45" target='_blank'>
              CRISPR-Cas9 Gene Editing Therapy, a Curative Hope for Sickle Cell in Nigeria, West Africa
              </a>
            </td>
          <td>
            Babatunde Olowu, Ahmed Olaide, Oluwaloni Tinubu
          </td>
          <td>2024-07-15</td>
          <td>International Journal of Genetics and Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is an incurable malignancy characterised by altered expression of coding and non-coding genes promoting tumour growth and drug resistance. Although the crucial role of long non-coding RNAs (lncRNAs) in MM is clearly established, the function of the non-coding RNAome, which might allow the design of novel therapeutics, is largely unknown. We performed an unbiased CRISPR-Cas9 loss-of-function screen of 671 lncRNAs in MM cells and their Bortezomib (BZB)-resistant derivative. To rank functionally and clinically relevant candidates, we designed and used a bioinformatic prioritisation pipeline combining functional data from cellular screens with prognostic and transcriptional data from MM patients. With this approach, we unveiled and prioritised 8 onco-lncRNAs essential for MM cell fitness, associated with high expression and poor prognosis in MM patients. The previously uncharacterised RP11-350G8.5 emerged as the most promising target, irrespective of BZB resistance. We i) demonstrated the anti-tumoral effect obtained by RP11-350G8.5 inhibition in vitro and in vivo; ii) highlighted a modulation of the unfolded protein response and the induction of immunogenic cell death triggered by the RP11-350G8.5 knock-out, via RNA-sequencing and molecular studies; iii) characterised its cytoplasmic homing through RNA-FISH; iv) predicted its 2D structure and identified 2 G-quadruplex and 3 hairpin-forming regions by biophysical assays, including Thioflavin T, 1H-NMR and circular dichroism to pave the way to the development of novel targeted therapeutics. Overall we provided innovative insights about unexplored lncRNAs in MM and identified RP11-350G8.5 as an oncogenic target for treatment-naïve and BZB-resistant MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b3038ab25be0410e0d1aec773f7b5564cf954f8" target='_blank'>
              An unbiased lncRNAs dropout CRISPR-Cas9 screen reveals RP11-350G8.5 as a novel therapeutic target for Multiple Myeloma.
              </a>
            </td>
          <td>
            K. Grillone, S. Ascrizzi, Paolo Cremaschi, Jussara Amato, N. Polerà, Ottavio Croci, R. Rocca, C. Riillo, Francesco Conforti, Raffaele Graziano, Diego Brancaccio, Daniele Caracciolo, Stefano Alcaro, Bruno Pagano, Antonio Randazzo, P. Tagliaferri, F. Iorio, P. Tassone
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c25293ccda82c6d001fe5bae96c805b5e1fa7f9a" target='_blank'>
              The role of DNA polymerase I in tolerating single-strand breaks generated at clustered DNA damage in Escherichia coli
              </a>
            </td>
          <td>
            N. Shikazono, Ken Akamatsu
          </td>
          <td>2024-08-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="For several decades, induction therapy with nucleoside analogs, in particular cytarabine (Ara-C) and, to a lesser extent, fludarabine, has been the standard of care for patients diagnosed with acute myeloid leukemia (AML). Still, the anti-tumor efficacy of nucleoside analogs is often limited by intrinsic and acquired drug resistance, thereby leading to poor therapeutic response and suboptimal clinical outcomes. Here, we used genome-wide CRISPR-based pharmacogenomic screening to map the genetic factors that modulate the response to nucleoside analogs in AML and identified the E3 ubiquitin ligase Herc1 as a key modulator of Ara-C response in the MLL/AF9 (MA) and the HOXA9/MEIS1 (HM) murine AML models both in vitro and in vivo. Loss of HERC1 enhanced nucleoside analog-induced cell death in both murine and human AML cell lines by compromising cell cycle progression. In-depth proteomic analysis and subsequent validation identified deoxycytidine kinase Dck as a novel target of Herc1 in MA and HM murine cells. We observed that HERC1 is overexpressed in AML compared to other cancer types and higher HERC1 expression is associated with shorter overall survival of patients with AML in the TCGA and BEAT-AML cohorts. Collectively, this study highlights the importance of HERC1 in the response of AML cells to nucleoside analogs, thereby establishing this E3 ubiquitin ligase as a novel predictive biomarker and potential therapeutic target for the treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5245f1077766a809fae0247565289dd67c6fcad" target='_blank'>
              The E3 ubiquitin ligase Herc1 modulates the response to nucleoside analogs in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Maja Jankovic, William Wl Poon, Cristobal Gonzales-Losada, Gabriela Galicia Vazquez, Bahram Sharif-Askari, Yi Ding, Constance Craplet-Desombre, Alexandru Ilie, Jiantao Shi, Yongie Wang, A. Jayavelu, Alexandre Orthwein, F. E. Mercier
          </td>
          <td>2024-08-02</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Ovarian cancer (OC) poses a significant health risk to women worldwide, with late diagnoses and chemotherapy resistance leading to high mortality rates. Despite several histological subtypes, the primary challenge remains the subtle nature of its symptoms, resulting in advanced-stage diagnosis and reduced treatment success rates. With platinum-based therapies showing relative efficacy but limited survival enhancements, the emergence of chemotherapy resistance during recurrence remains a critical obstacle. Precision medicine development has aimed to address these challenges in the context of the molecular diversity of OC. The present review explored the landscape of microRNA (miRNA)-mediated approaches in OC treatment. miRNAs have emerged as regulators of gene expression, serving as both oncogenes and tumor suppressors in OC. Dysregulated miRNAs are associated with disease progression and chemotherapy resistance, underscoring their significance in diagnosis and tailored treatment strategies. The present review extracted 295 publications from the PUBMED database. Out of the 73 eligible studies, 55 miRNAs were assessed. A total of three of these miRNAs were not associated with any disease or cancer, whilst eight were associated with OC, albeit also associated with other diseases. The present review encompassed three dimensions: i) The role of miRNAs in treatment efficacy; ii) the use of miRNAs to enhance therapy outcomes; and iii) adjunctive strategies for improved treatment results. Furthermore, it offered insights into potential avenues for improving OC treatment using miRNA-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e04bb3c4e63f1fb22f64e7d076de1f329b943b90" target='_blank'>
              MicroRNA‑mediated approaches in ovarian cancer therapy: A comprehensive systematic review
              </a>
            </td>
          <td>
            Henny Meitri Andrie Rachmasari Putri, Putri Wikie Novianti, Heru Pradjatmo, S. Haryana
          </td>
          <td>2024-08-12</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background: Integrating immune checkpoint inhibitors with multi-target tyrosine kinase inhibitors presents an innovative and hopeful strategy in liver cancer treatment. Nonetheless, a degree of resistance to this treatment is noticeable in certain patients. Alternative splicing (AS) represents a common biological process that controls the variety of life functions via isoforms. Purpose: Investigating how gene AS affects the effectiveness of combined immunotherapy in treating hepatocellular carcinoma (HCC). Methods: Our retrospective examination focused on AS's effect on immune therapy effectiveness, utilizing accessible tissue sequencing and clinical records for HCC. For corroborating our results, we gathered samples of drug-resistant HCC tissue, nearby tissues, HCC tissue with high drug responsiveness, and healthy liver tissue from clinical studies. Results: The study revealed a link between the frequency of AS occurrences, the expression levels of programmed cell death 1 ligand 1, and the resistance to tumor medications. Our study detailed the AS occurrences in HCC, leading to the creation of a risk-assessment function and a predictive model using AS data. The results of our study revealed that the risk score effectively distinguished between various immune subtypes and the effectiveness of immune therapy. Additional examination of the chosen AS occurrences uncovered their effects on both the immune microenvironment and cellular immunity. Our investigation also delved into the regulatory framework of AS, uncovering the role of stringently controlled splicing factors in the emergence of tumors and the modulation of the body's immune response. Conclusions: Increased AS in HCC diminishes the efficacy of immunotherapy; conversely, more AS in peritumoral tissue elevates the likelihood of tumor immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc39d2a7e8075c89a1f4a7ca29b083b1fc5166ab" target='_blank'>
              Prediction Model for Immunotherapy Efficacy in Hepatocellular Carcinoma Based on Alternative Splicing Sequencing Data
              </a>
            </td>
          <td>
            Huizhong Shi, Yang Liu, Zhao Liu, Xinlan Ge, Jushan Wu, Haowen Tang, Yi Zhang, Shi-yong Lu
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background The methyltransferase SETD2 was originally identified as a tumor suppressor in clear cell renal cell carcinoma (ccRCC), and ccRCC with SETD2 mutations have increased metastatic potential and poor progression free survival. A known epigenetic regulator, SETD2 interacts with actively transcribing RNA polymerase II (RNAPII) and tri-methylates Histone 3 at Lysine 36 (H3K36me3), a modification that correlates with active gene transcription and is recognized by mRNA methylation and splicing factors. Despite these known roles in mRNA transcription, loss of SETD2 does not alter genic transcription, raising the question of whether or how SETD2 regulates RNA biology. Methods Given that SETD2 influences RNA processing, we sought to identify RNA-protein complexes that are regulated by SETD2. To do this in an unbiased and high-throughput manner, we used orthogonal organic phase separation (OOPS) coupled with mass spectrometry to identify changes in RNA binding proteins and pathways between control and SETD2-knockout cells. Results Corroborating previous findings, we identified that RNA binding proteins governing mRNA splicing and cellular metabolism were dysregulated in SETD2 mutant cells. However, the most robust changes in SETD2 knockout cells occurred in pathways regulated ribosome biogenesis (See Figure, orange highlights), resulting in a significant decrease in binding of ribosomal RNA (rRNA) processing factors to RNA. We confirm these alterations lead to defects in rRNA transcription, processing, and ribosome biogenesis. We identify that loss of SETD2 catalytic activity phenocopies these rRNA processing defects, suggesting H3K36me3 is essential for rRNA processing. Disruptions occur to both the large (60S) and small (40S) ribosome subunits, modifying protein translation. Lastly, we identify synthetic lethality between SETD2 deletion or inhibition and small molecules that interfere with rRNA transcription. Conclusions While the majority of research emphasizes the role SETD2 in the regulation of mRNA, our data identify that SETD2 and its catalytic activity play a critical role in the transcription and processing of rRNA. Importantly, rRNA comprises 80-90% of cellular RNA yet is typically overlooked and understudied. Next-generation RNA sequencing or transcriptional assays investigate mRNA or genic transcription while ignoring rRNA, demonstrating why SETD2-mediated rRNA processing has been unexplored. This may also have important implications for prospective clinical trials that utilized RNA-sequencing to identify optimal therapeutic regimens. Our work also identified that SETD2-mutated cells are more sensitive to compounds that inhibit rRNA transcription, potentially uncovering a new therapeutic vulnerability of ccRCC with mono- and bi-allelic loss of SETD2. Alterations in ribosome biogenesis leads to epithelial-to-mesenchymal transition and metastasis in many tumor types, and our future goal will be to investigate the role of ribosomes and rRNA transcription in transformation and metastasis in ccRCC as well as the molecular mechanisms connecting SETD2 to ribosome fidelity. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30cb284d919d6bd0de496d90d21e1c0abae77b34" target='_blank'>
              51 Epigenetic regulation of ribosomal RNA transcription and processing by the tumor suppressor SETD2
              </a>
            </td>
          <td>
            Logan Vlach, Scott Haake, K. Rathmell, Frank M. Mason
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 The size and composition of the intracellular DNA precursor pool is integral to the maintenance of genome stability, and this relationship is fundamental to our understanding of cancer. Key aspects of carcinogenesis, including elevated mutation rates and induction of certain types of DNA damage in cancer cells, can be linked to disturbances in deoxynucleoside triphosphate (dNTP) pools. Furthermore, our approaches to treat cancer heavily exploit the metabolic interplay between the DNA and the dNTP pool, with a long-standing example being the use of antimetabolite-based cancer therapies, and this strategy continues to show promise with the development of new targeted therapies. In this Review, we compile the current knowledge on both the causes and consequences of dNTP pool perturbations in cancer cells, together with their impact on genome stability. We outline several outstanding questions remaining in the field, such as the role of dNTP catabolism in genome stability and the consequences of dNTP pool expansion. Importantly, we detail how our mechanistic understanding of these processes can be utilised with the aim of providing better informed treatment options to patients with cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489114e77099687246712fffa583dd2206547a4f" target='_blank'>
              Understanding the interplay between dNTP metabolism and genome stability in cancer
              </a>
            </td>
          <td>
            Miriam Yagüe-Capilla, S. G. Rudd
          </td>
          <td>2024-08-01</td>
          <td>Disease Models &amp; Mechanisms</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Human induced pluripotent stem cells (hiPSCs) represent a powerful tool to investigate neuropathological disorders in which the cells of interest are inaccessible, such as in the Charcot–Marie–Tooth disease (CMT), the most common inherited peripheral neuropathy. Developing appropriate cellular models becomes crucial in order to both study the disease’s pathophysiology and test new therapeutic approaches. The generation of hiPS cellular models for disorders caused by a single nucleotide variation has been significantly improved following the development of CRISPR-based editing tools. In this study, we efficiently and quickly generated, by CRISPR editing, the two first hiPSCs cellular models carrying alterations involved in CMT4C, also called AR-CMTde-SH3TC2. This subtype of CMT is associated with alterations in the SH3TC2 gene and represents the most prevalent form of autosomal recessive demyelinating CMT. We aimed to develop models for two different SH3TC2 nonsense variants, c.211C>T, p.Gln71* and the most common AR-CMTde-SH3TC2 alteration, c.2860C>T, p.Arg954*. First, in order to determine the best CRISPR strategy to adopt on hiPSCs, we first tested a variety of sgRNAs combined with a selection of recent base editors using the conveniently cultivable and transfectable HEK-293T cell line. The chosen CRISPR base-editing strategy was then applied to hiPSCs derived from healthy individuals to generate isogenic CMT disease models with up to 93% editing efficiency. For point mutation generation, we first recommend to test your strategies on alternative cell line such as HEK-293T before hiPSCs to evaluate a variety of sgRNA-BE combinations, thus boosting the chance of achieving edited cellular clones with the hard-to-culture and to transfect hiPSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe956c657405832b052a7c5be2f405359785783c" target='_blank'>
              CRISPR Base Editing to Create Potential Charcot–Marie–Tooth Disease Models with High Editing Efficiency: Human Induced Pluripotent Stem Cell Harboring SH3TC2 Variants
              </a>
            </td>
          <td>
            Camille Loret, Amandine Pauset, Pierre-Antoine Faye, Valérie Prouzet-Mauleon, I. Pyromali, A. Nizou, Federica Miressi, Franck Sturtz, Frédéric Favreau, Béatrice Turcq, A. Lia
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Menin inhibitors that disrupt Menin-MLL interaction hold promise for treating specific acute myeloid leukemia subtypes, including KMT2A rearrangements (KMT2A-r), yet resistance remains a challenge. Here, through systematic chromatin-focused CRISPR screens, along with genetic, epigenetic, and pharmacologic studies in a variety of human and mouse KMT2A-r AML models, we uncover a potential resistance mechanism independent of canonical Menin-MLL targets. We show that a group of non-canonical Menin targets, which are bivalently co-occupied by active Menin and repressive H2AK119ub marks, are typically downregulated following Menin inhibition. The loss of Polycomb Repressive Complex 1.1 (PRC1.1) subunits, such as PCGF1 or BCOR, leads to Menin inhibitor resistance by epigenetic reactivation of these non-canonical targets, including MYC. Genetic and pharmacological inhibition of MYC can resensitize PRC1.1-deficent leukemia cells to Menin inhibition. Moreover, we demonstrate that leukemia cells with the loss of PRC1.1 subunits exhibit reduced monocytic gene signatures and are susceptible to the BCL2 inhibition, and combinational treatment of venetoclax overcomes the resistance to Menin inhibition in PRC1.1-deficient leukemia cells. These findings highlight the important roles of PRC1.1 and its regulated non-canonical Menin targets in modulating Menin inhibitor response and provide potential strategies to treat leukemias with compromised PRC1.1 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fba94eb1d6d5cb6e0a8de6969fbdc894d28222" target='_blank'>
              Epigenetic Regulation of Non-canonical Menin Targets Modulates Menin Inhibitor Response in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Xinyue Zhou, Lixia Zhang, Sajesan Aryal, Virginia Veasey, Amanda Tajik, Cecilia Restelli, Steven Moreira, Pengcheng Zhang, Yanfeng Zhang, Kristin J Hope, Yang Zhou, Changde Cheng, Ravi Bhatia, Rui Lu
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Carcinomas are common in humans but rare among closely related “great apes.” Plausible explanations, including human-specific genomic alterations affecting the biology of sialic acids, are proposed, but causality remains unproven. Here, an integrated evolutionary genetics-phenome-transcriptome approach studied the role of SIGLEC12 gene (encoding Siglec-XII) in epithelial transformation and cancer. Exogenous expression of the protein in cell lines and genetically engineered mice recapitulated approximately 30% of the human population in whom the protein is expressed in a form that cannot bind ligand because of a fixed, homozygous, human-universal missense mutation. Siglec-XII–null cells/mice recapitulated the remaining approximately 70% of the human population in whom an additional polymorphic frameshift mutation eliminates the entire protein. Siglec-XII expression drove several pro-oncogenic phenotypes in cell lines and increased tumor burden in mice challenged with chemical carcinogen and inflammation. Transcriptomic studies yielded a 29-gene signature of Siglec-XII–positive disease and when used as a computational tool for navigating human data sets, pinpointed with surprising precision that SIGLEC12 expression (model) recapitulates a very specific type of colorectal carcinomas (disease) that is associated with mismatch-repair defects and inflammation, disproportionately affects European Americans, and carries a favorable prognosis. They revealed a hitherto-unknown evolutionary genetic mechanism for an ethnic/environmental predisposition of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3a026fde004925d82bafb73d8c39f7bda40aeeb" target='_blank'>
              Human-specific elimination of epithelial Siglec-XII suppresses the risk of inflammation-driven colorectal cancers
              </a>
            </td>
          <td>
            Hector A Cuello, Saptarshi Sinha, Andrea L Verhagen, N. Varki, A. Varki, Pradipta Ghosh
          </td>
          <td>2024-07-11</td>
          <td>JCI Insight</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40fcdfbbc2df4270401ef8beed0b78736d658d46" target='_blank'>
              DNA methylation in mammalian development and disease.
              </a>
            </td>
          <td>
            Zachary D. Smith, Sara Hetzel, Alexander Meissner
          </td>
          <td>2024-08-12</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The combination of CDK4/6 and MEK inhibition as a therapeutic strategy has shown promise in various cancer models, particularly in those harboring RAS mutations. An initial high-throughput drug screen identified a high synergy between the CDK4/6 inhibitor palbociclib and the MEK inhibitor trametinib when used in combination in soft tissue sarcomas. In RAS mutant models, combination treatment with palbociclib and trametinib induced significant G1 cell cycle arrest, resulting in a marked reduction in cell proliferation and growth. CRISPR-mediated RB1 depletion resulted in a decreased response to CDK4/6 and MEK inhibition, which was validated in both cell culture and xenograft models. Beyond its cell cycle inhibitory effects, pathway enrichment analysis revealed the robust activation of interferon pathways upon CDK4/6 and MEK inhibition. This induction of gene expression was associated with the upregulation of retroviral elements. The TBK1(TANK-binding kinase 1) inhibitor GSK8612 selectively blocked the induction of interferon-related genes induced by palbociclib and trametinib treatment, and highlighted the separable epigenetic responses elicited by combined CDK4/6 and MEK inhibition. Together, these findings provide key mechanistic insights into the therapeutic potential of CDK4/6 and MEK inhibition in soft tissue sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/721445995cd4f6ad7aae623c32a235fcb7645dbc" target='_blank'>
              Separable cell cycle arrest and immune response elicited through pharmacological CDK4/6 and MEK inhibition in RASmut disease models.
              </a>
            </td>
          <td>
            Jin Wu, Jianxin Wang, Thomas N O'Connor, Stephanie L. Tzetzo, Katerina V Gurova, Erik S. Knudsen, Agnieszka K Witkiewicz
          </td>
          <td>2024-08-16</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Nonsense-mediated mRNA decay (NMD) is a highly conserved post-transcriptional gene expression regulatory mechanism in eukaryotic cells. NMD eliminates aberrant mRNAs with premature termination codons to surveil transcriptome integrity. Furthermore, NMD fine-tunes gene expression by destabilizing RNAs with specific NMD features. Thus, by controlling the quality and quantity of the transcriptome, NMD plays a vital role in mammalian development, stress response, and tumorigenesis. Deficiencies of NMD factors result in early embryonic lethality, while the underlying mechanisms are poorly understood. SMG5 is a key NMD factor. In this study, we generated an Smg5 conditional knockout mouse model and found that Smg5-null results in early embryonic lethality before E13.5. Furthermore, we produced multiple lines of Smg5 knockout mouse embryonic stem cells (mESCs) and found that the deletion of Smg5 in mESCs does not compromise cell viability. Smg5-null delays differentiation of mESCs. Mechanistically, our study reveals that the c-MYC protein, but not c-Myc mRNA, is upregulated in SMG5-deficient mESCs. The overproduction of c-MYC protein could be caused by enhanced protein synthesis upon SMG5 loss. Furthermore, SMG5-null results in dysregulation of alternative splicing on multiple stem cell differentiation regulators. Overall, our findings underscore the importance of SMG5-NMD in regulating mESC cell-state transition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1939acc39765be34373760ea0bb8f36b6d953921" target='_blank'>
              RNA Surveillance Factor SMG5 Is Essential for Mouse Embryonic Stem Cell Differentiation
              </a>
            </td>
          <td>
            Chengyan Chen, Yanling Wei, Xiaoning Jiang, Tangliang Li
          </td>
          <td>2024-08-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The main epidemiological and clinical data on colorectal cancer, as well as the features of molecular pathology, are discussed in the literature review. Efforts are being putto identify promising targets, particularly small non-coding nucleotide sequences, which can lead to new treatments for this disease. The discovery of significant mutations that contribute to the development of colorectal tumors is a major step in the advancement of molecular oncology, as these mutations give rise to heterogeneous tumors that differ in their origin. These mutations play a significant role in the progression of the disease and are now being targeted for treatment. The prognosis for a disease is influenced by the patient's sensitivity to antitumor therapy. However, new approaches to finding effective targets for antitumor treatments face new fundamental challenges due to clinical issues. These issues include the epigenetic regulation of markers of oncogenesis, which allows for the development of new therapeutic strategies. RNA interference, in particular, has been linked to non-copying RNA sequences such as microRNAs. These microRNAs are associated with certain processes that can influence all aspects of oncogenesis. The diversity of microRNAs allows for a differentiated approach when treating tumors in various locations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4796e80e11216dc287a04bf6cbff831c3d542a28" target='_blank'>
              New Perspectives in Colorectal Cancers Treatment, the Role of MicroRNAs.
              </a>
            </td>
          <td>
            V. A. Belova, Liudmila V Spirina, A. Avgustinovich, S. G. Afanasiev, Maxim Yu Volkov M, D. I. Azovsky, Alexander M Volkov, T. S. Klyushina
          </td>
          <td>2024-07-24</td>
          <td>Current drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92d22ec19c45e84786f29e00379dcc92c97dbcbf" target='_blank'>
              Identifying key genes and functionally enriched pathways in acute myeloid leukemia by weighted gene co-expression network analysis.
              </a>
            </td>
          <td>
            Jimo Jian, Chenglu Yuan, Hongyuan Hao
          </td>
          <td>2024-07-09</td>
          <td>Journal of applied genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Prostate cancer (PC), which is a disease driven by the activity of the androgen receptor (AR), is the most commonly diagnosed malignancy and despite advances in diagnostic and treatment strategies, PC is the second most common cause of cancer mortality in men (Bray et al., 2018). Taxane-based chemotherapy is the only chemotherapy that prolongs survival in metastatic PC patients (Petrylak et al., 2004; Tannock et al., 2004). At the cellular level, taxanes bind to and stabilize microtubules (MTs) inhibiting all MT-dependent intracellular pathways. MTs are highly dynamic polymers that stochastically switch between phases of growth, shrinkage, and pause (Jordan and Wilson, 2004). Altered MT dynamics endow cancer cells with both survival and migratory advantages (Mitchison, 2012). Taxanes inhibit MT dynamics and alter the spatial organization of the MT network, thereby inhibiting intracellular trafficking of molecular cargo critical for tumor survival. In PC specifically, taxanes inhibit transcriptional activity downstream of MT stabilization (Thadani-Mulero et al., 2012) and AR nuclear accumulation (Darshan et al., 2011; Zhu et al., 2010). Different tubulin inhibitors, even from within the same structural class as the taxanes, affect distinct parameters of MT dynamics (Jordan and Wilson, 2004), yet the selection of taxane for chemotherapy is not based on the particular patterns of dynamic behavior of the MT cytoskeleton in individual patients. We envisage that systematic characterization using quantitative analysis of MT dynamics in PC patient cells expressing clinically relevant protein isoforms (Matov et al., 2024; Thoma et al., 2010), before and after treatment with each of the taxanes, will allow us to identify criteria for the selection of the most suitable drug combination at the onset of treatment. We link MT dynamics in the presence of AR variants and sensitivity/resistance to taxanes and connect fundamental research with clinically relevant concepts to elucidate cellular mechanisms of clinical response to taxanes and, thus, advance the customization of therapy. Our computational live-cell analysis addresses questions in the context of the inherent differences in MT homeostasis as a function of AR content in PC cells, the specific parameters of MT dynamics each of the taxanes affects, and how can this information be used to match endogenous patterns of MT dynamics with drug-modulated MT behavior. We investigate whether the sensitivity to taxanes, evaluated by computational analysis of MTs, can be linked to gene expression driven by AR and its variants, and whether the resistance to taxanes be linked to the presence of a specific AR splice variant, and can we identify which of the taxanes will be most effective based on the endogenous patterns of MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e90d5e5c268c6138c1b50b9fd253e67162113bd" target='_blank'>
              Computational Analysis of Treatment Resistant Cancer Cells
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-08-31</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6f3b91bf405135dcfe9e5dfb76a9822cb899a07" target='_blank'>
              Fueling CARs: metabolic strategies to enhance CAR T-cell therapy
              </a>
            </td>
          <td>
            A. Van der Vreken, K. Vanderkerken, E. De Bruyne, K. De Veirman, Karine Breckpot, E. Menu
          </td>
          <td>2024-07-10</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Pathogenic variations in the fused in sarcoma (FUS) gene are associated with rare and aggressive forms of amyotrophic lateral sclerosis (ALS). As FUS-ALS is a dominant disease, a targeted, allele-selective approach to FUS knockdown is most suitable. Antisense oligonucleotides (AOs) are a promising therapeutic platform for treating such diseases. In this study, we have explored the potential for allele-selective knockdown of FUS. Gapmer-type AOs targeted to two common neutral polymorphisms in FUS were designed and evaluated in human fibroblasts. AOs had either methoxyethyl (MOE) or thiomorpholino (TMO) modifications. We found that the TMO modification improved allele selectivity and efficacy for the lead sequences when compared to the MOE counterparts. After TMO-modified gapmer knockdown of the target allele, up to 93% of FUS transcripts detected were from the non-target allele. Compared to MOE-modified AOs, the TMO-modified AOs also demonstrated reduced formation of structured nuclear inclusions and SFPQ aggregation that can be triggered by phosphorothioate-containing AOs. How overall length and gap length of the TMO-modified AOs affected allele selectivity, efficiency and off-target gene knockdown was also evaluated. We have shown that allele-selective knockdown of FUS may be a viable therapeutic strategy for treating FUS-ALS and demonstrated the benefits of the TMO modification for allele-selective applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40bef67258a89242d3fafd4ece2697b7aa372f10" target='_blank'>
              Allele-Selective Thiomorpholino Antisense Oligonucleotides as a Therapeutic Approach for Fused-in-Sarcoma Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            R. Mejzini, Marvin H. Caruthers, Balazs Schafer, Ondrej Kostov, Kavitha Sudheendran, Marija Ciba, Mathias Danielsen, Steve D. Wilton, P. A. Akkari, L. L. Flynn
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="ABSTRACT Changes in gene dosage can have tremendous evolutionary potential (e.g. whole-genome duplications), but without compensatory mechanisms, they can also lead to gene dysregulation and pathologies. Sex chromosomes are a paradigmatic example of naturally occurring gene dosage differences and their compensation. In species with chromosome-based sex determination, individuals within the same population necessarily show ‘natural’ differences in gene dosage for the sex chromosomes. In this Review, we focus on the mammalian X chromosome and discuss recent new insights into the dosage-compensation mechanisms that evolved along with the emergence of sex chromosomes, namely X-inactivation and X-upregulation. We also discuss the evolution of the genetic loci and molecular players involved, as well as the regulatory diversity and potentially different requirements for dosage compensation across mammalian species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016ee497a5dab2410dc213e6a7a7d311ff9fabfc" target='_blank'>
              Compensation of gene dosage on the mammalian X
              </a>
            </td>
          <td>
            Daniela Cecalev, Beatriz Viçoso, Rafael Galupa
          </td>
          <td>2024-08-01</td>
          <td>Development (Cambridge, England)</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b903232278019fc20d595adb95c47c4a0e3a7532" target='_blank'>
              Deletion of IRC19 Causes Defects in DNA Double-Strand Break Repair Pathways in Saccharomyces cerevisiae.
              </a>
            </td>
          <td>
            Ju-Hee Choi, Oyungoo Bayarmagnai, Sung-Ho Bae
          </td>
          <td>2024-07-12</td>
          <td>Journal of microbiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) is associated with malignant tumors in women worldwide with persistently high incidence and mortality rates. The traditional therapies including surgery, chemotherapy, radiotherapy and targeted therapy have certain therapeutic effects on BC patients, but acquired drug resistance can lead to tumor recurrence and metastasis. This remains a clinical challenge that is difficult to solve during treatment. Therefore, continued research is needed to identify effective targets and treatment methods, to ultimately implement personalized treatment strategies. Several studies have implicated that the long non-coding RNA LINC00511 is closely linked to the occurrence, development and drug resistance of BC. Here we will review the structure and the mechanisms of action of lnc RNA LINC00511 in various cancers, and then explore its expression and its related regulatory mechanisms during BC. In addition, we will discuss the biological functions and the potential clinical applications of LINC00511 in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13c5511a8f64badfc82cdc1ea0b1f93b17cf90d2" target='_blank'>
              The emerging roles of LINC00511 in breast cancer development and therapy
              </a>
            </td>
          <td>
            Lifeng Zhao, Sangita Biswas, Ye Li, S. Sooranna
          </td>
          <td>2024-08-14</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e16f71ff68478a0f30f71105d86b66a3bf8d6b91" target='_blank'>
              Single-Cell Sequencing: High-Resolution Analysis of Cellular Heterogeneity in Autoimmune Diseases.
              </a>
            </td>
          <td>
            Xuening Tang, Yudi Zhang, Hao Zhang, Nan Zhang, Ziyu Dai, Q. Cheng, Yongzhen Li
          </td>
          <td>2024-08-26</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Despite the availability of life-extending treatments for B cell leukemias and lymphomas, many of these cancers remain incurable. Thus, the development of new molecular targets and therapeutics is needed to expand treatment options. To identify new molecular targets, we used a forward genetic screen in mice to identify genes required for development or survival of lymphocytes. Here, we describe Zfp574, an essential gene encoding a zinc finger protein necessary for normal and malignant lymphocyte survival. We show that ZFP574 interacts with zinc finger protein THAP12 and promotes the G1-to-S-phase transition during cell cycle progression. Mutation of ZFP574 impairs nuclear localization of the ZFP574-THAP12 complex. ZFP574 or THAP12 deficiency results in cell cycle arrest and impaired lymphoproliferation. Germline mutation, acute gene deletion, or targeted degradation of ZFP574 suppressed Myc-driven B cell leukemia in mice, but normal B cells were largely spared, permitting long-term survival, whereas complete lethality was observed in control animals. Our findings support the identification of drugs targeting ZFP574-THAP12 as a unique strategy to treat B cell malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0480486c2217b6dc5e3cddbe10c5e1e4dd9761e0" target='_blank'>
              Disruption of the ZFP574-THAP12 complex suppresses B cell malignancies in mice.
              </a>
            </td>
          <td>
            Xue Zhong, James J Moresco, J. SoRelle, Ran Song, Yiao Jiang, Mylinh T Nguyen, Jianhui Wang, C. Bu, E. Moresco, Bruce Beutler, J. Choi
          </td>
          <td>2024-07-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Recombination hotspot-activating DNA sites (e.g., M26, CCAAT, Oligo-C) and their binding proteins (e.g., Atf1-Pcr1 heterodimer; Php2-Php3-Php5 complex, Rst2, Prdm9) regulate the distribution of Spo11 (Rec12)-initiated meiotic recombination. We sought to create 14 different candidate regulatory DNA sites via bp substitutions in the ade6 gene of Schizosaccharomyces pombe. We used a fission yeast-optimized CRISPR-Cas9 system (SpEDIT) and 196 bp-long dsDNA templates with centrally located bp substitutions designed to ablate the genomic PAM site, create specific 15 bp-long DNA sequences, and introduce a stop codon. After co-transformation with a plasmid that encoded both the guide RNA and Cas9 enzyme, about one-third of colonies had a phenotype diagnostic for DNA sequence changes at ade6. PCR diagnostics and DNA sequencing revealed a diverse collection of alterations at the target locus, including: (A) complete or (B) partial template-directed substitutions; (C) non-homologous end joinings; (D) duplications; (E) bp mutations, and (F) insertions of ectopic DNA. We concluded that SpEDIT can be used successfully to generate a diverse collection of DNA sequence elements within a reporter gene of interest. However, its utility is complicated by low efficiency, incomplete template-directed repair events, and undesired alterations to the target locus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b22c8181855df22c439384335df284718d37e654" target='_blank'>
              Creating Meiotic Recombination-Regulating DNA Sites by SpEDIT in Fission Yeast Reveals Inefficiencies, Target-Site Duplications, and Ectopic Insertions
              </a>
            </td>
          <td>
            R. U. Protacio, Seth Dixon, M. Davidson, W. P. Wahls
          </td>
          <td>2024-08-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6545f55fcc52a50f9511c504a46a4bc2ad014fe" target='_blank'>
              DNA Methylation Profiling in Genetically Selected Clarias magur (Hamilton, 1822) Provides Insights into the Epigenetic Regulation of Growth and Development.
              </a>
            </td>
          <td>
            K. S. Risha, K. Rasal, D. Reang, M. Iquebal, Arvind Sonwane, Manoj Brahmane, Aparna S. Chaudhari, N. Nagpure
          </td>
          <td>2024-07-22</td>
          <td>Marine biotechnology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Background Synthetic lethality offers a promising strategy for cancer treatment by targeting genetic vulnerabilities unique to tumor cells, leading to selective tumor cell death. However, single-gene knockout screens often miss functional redundancy due to paralog genes. Multiplex CRISPR systems, including various Cas9 and Cas12a platforms, have been developed to assay genetic interactions, yet no systematic comparison of method to identify synthetic lethality from CRISPR screens has been conducted. Results We evaluated data from four in4mer CRISPR/Cas12a screens in cancer cell lines, using three bioinformatic approaches to identify synthetic lethal interactions: delta log fold change (dLFC), Z-transformed dLFC (ZdLFC), and rescaled dLFC (RdLFC). Both ZdLFC and RdLFC provided more consistent identification of synthetic lethal pairs across cell lines compared to the unscaled dLFC method. Conclusions The ZdLFC method offers a robust framework for scoring synthetic lethal interactions from paralog screens, providing consistent results across different cell lines without requiring a training set of known positive interactors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c5934ec3c9f26ccd5d51f46039a111f65c5837e" target='_blank'>
              Optimal methods for analyzing targeted pairwise knockout screens
              </a>
            </td>
          <td>
            Juihsuan Chou, N. Anvar, Reem Elghaish, Junjie Chen, Traver Hart
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer (mCRPC) is a lethal disease that resists therapy targeting androgen signaling, the primary driver of prostate cancer. mCRPC resists androgen receptor (AR) inhibitors by amplifying AR signaling or by evolving into therapy-resistant subtypes that do not depend on AR. Elucidation of the epigenetic underpinnings of these subtypes could provide important insights into the drivers of therapy resistance. In this study, we produced chromatin accessibility maps linked to the binding of lineage-specific transcription factors (TF) by performing ATAC sequencing on 70 mCRPC tissue biopsies integrated with transcriptome and whole genome sequencing. mCRPC had a distinct global chromatin accessibility profile linked to AR function. Analysis of TF occupancy across accessible chromatin revealed 203 TFs associated with mCRPC subtypes. Notably, ZNF263 was identified as a putative prostate cancer TF with a significant impact on gene activity in the double-negative (AR- neuroendocrine-) subtype, potentially activating MYC targets. Overall, this analysis of chromatin accessibility in mCRPC provides valuable insights into epigenetic changes that occur during progression to mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac71bf183535cdb84fd367ef22969e3b56650628" target='_blank'>
              An Atlas of Accessible Chromatin in Advanced Prostate Cancer Reveals the Epigenetic Evolution during Tumor Progression.
              </a>
            </td>
          <td>
            Raunak Shrestha, L. Chesner, Meng Zhang, Stanley Zhou, A. Foye, A. Lundberg, Alana S Weinstein, M. Sjöström, Xiaolin Zhu, Thaidy Moreno-Rodriguez, Haolong Li, Su C/Pcf West Coast Prostate Cancer Dream Team, J. Alumkal, Rahul Aggarwal, Eric J Small, Mathieu Lupien, David A Quigley, Felix Y Feng
          </td>
          <td>2024-07-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer.
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Human telomerase reverse transcriptase (hTERT) may have noncanonical functions in transcriptional regulation and metabolic reprogramming in cancer cells, but it is a challenging target. We thus developed small-molecule ligands targeting hTERT promoter G-quadruplex DNA structures (hTERT G4) to downregulate hTERT expression. Ligand 5 showed high affinity toward hTERT G4 (Kd = 1.1 μM) and potent activity against triple-negative breast cancer cells (MDA-MB-231, IC50 = 1 μM). In cell-based assays, 5 not only exerts markedly inhibitory activity on classical telomere functions including decreased telomerase activity, shortened telomere length, and cellular senescence but also induces DNA damage, acute cellular senescence, and apoptosis. This study reveals that hTERT G4-targeting ligand may cause mitochondrial dysfunction, disrupt iron metabolism and activate ferroptosis in cancer cells. The in vivo antitumor efficacy of 5 was also evaluated in an MDA-MB-231 xenograft mouse model and approximately 78.7% tumor weight reduction was achieved. No observable toxicity against the major organs was observed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1d3d47d8b1822b8c2901a320f1107b7691744c1" target='_blank'>
              Targeting hTERT Promoter G-Quadruplex DNA Structures with Small-Molecule Ligand to Downregulate hTERT Expression for Triple-Negative Breast Cancer Therapy.
              </a>
            </td>
          <td>
            W. Long, Yao-Xun Zeng, Bo-Xin Zheng, Yu-Bo Li, Ya-Kun Wang, K. Chan, Meng-Ting She, Yu-Jing Lu, Chunyang Cao, Wing-Leung Wong
          </td>
          <td>2024-07-10</td>
          <td>Journal of medicinal chemistry</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3ca92a03051caca152f854fcb5c2f6b12d56f76" target='_blank'>
              Integrated genomic/epigenomic analysis stratifies subtypes of clear cell ovarian carcinoma, highlighting their cellular origin
              </a>
            </td>
          <td>
            Akira Nishijima, K. Oda, Kosei Hasegawa, Takahiro Koso, K. Asada, Yuji Ikeda, Ayumi Taguchi, Daichi Maeda, G. Nagae, S. Tsuji, K. Tatsuno, Y. Uehara, A. Kurosaki, S. Sato, M. Tanikawa, K. Sone, M. Mori, M. Ikemura, Keiichi Fujiwara, Tetsuo Ushiku, Yutaka Osuga, Hiroyuki Aburatani
          </td>
          <td>2024-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a75dd48ccfcd6591fcd252d1a4a9a583e49480b" target='_blank'>
              Cockayne Syndrome Linked to Elevated R-Loops Induced by Stalled RNA Polymerase II during Transcription Elongation
              </a>
            </td>
          <td>
            Xuan Zhang, Jun Xu, Jing Hu, Sitao Zhang, Yajing Hao, Dongyang Zhang, Hao Qian, Dong Wang, Xiang-Dong Fu
          </td>
          <td>2024-07-17</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Thymomas and thymic carcinomas (together referred to as thymic epithelial tumors, TETs) are a heterogeneous group of rare tumors of the anterior mediastinum. Most treatment regimens for advanced TETs are empirical and patients have not benefited from developments in personalized medicine. We believe that one of the major unresolved problems in TET research is the lack of suitable cellular or animal models that represent the full heterogeneity of TETs, help us to study the driving mechanisms behind TET biology, and explore vulnerabilities. In this narrative review, we discuss the current state of 2D and 3D TET cell culture models and protocols. To move beyond the current state, innovative high-throughput techniques are mandatory for both systematic optimization of cell culture conditions and high-throughput drug screening. Abstract Cell culture model systems are fundamental tools for studying cancer biology and identifying therapeutic vulnerabilities in a controlled environment. TET cells are notoriously difficult to culture, with only a few permanent cell lines available. The optimal conditions and requirements for the ex vivo establishment and permanent expansion of TET cells have not been systematically studied, and it is currently unknown whether different TET subtypes require different culture conditions or specific supplements. The few permanent cell lines available represent only type AB thymomas and thymic carcinomas, while attempts to propagate tumor cells derived from type B thymomas so far have been frustrated. It is conceivable that epithelial cells in type B thymomas are critically dependent on their interaction with immature T cells or their three-dimensional scaffold. Extensive studies leading to validated cell culture protocols would be highly desirable and a major advance in the field. Alternative methods such as tumor cell organoid models, patient-derived xenografts, or tissue slices have been sporadically used in TETs, but their specific contributions and advantages remain to be shown.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89ce3ee9aaf655d07e4e9a399628eb007a235cdd" target='_blank'>
              Cell Culture Models for Translational Research on Thymomas and Thymic Carcinomas: Current Status and Future Perspectives
              </a>
            </td>
          <td>
            Denise Müller, Jürgen Loskutov, Stefan Küffer, Alex Marx, C. Regenbrecht, Philipp Ströbel, Manuela J. Regenbrecht
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27056894f9dfbd876e99de631ac52147696ee60c" target='_blank'>
              Mitochondrial DNA Copy Number as a Biomarker for Guiding Adjuvant Chemotherapy in Stages II and III Colorectal Cancer Patients with Mismatch Repair Deficiency: Seeking Benefits and Avoiding Harms
              </a>
            </td>
          <td>
            Mian Chen, Shenghe Deng, Yinghao Cao, Jun Wang, Falong Zou, Junnan Gu, Fuwei Mao, Y-F Xue, Zhenxing Jiang, Denglong Cheng, Ning Huang, Liang Huang, Kailin Cai
          </td>
          <td>2024-07-10</td>
          <td>Annals of Surgical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The CRISPR (Cluster of regularly interspersed short palindromic repeats) technology is the nowadays trend of genetic engineering. The simplicity of the system has made genetic engineering more applicable and convenient for bioengineers, in comparison to ZFNs and TALENs. It is now used in a wide range of aspect, by inserting, deleting, silencing, mutating the target gene. However, this technology still has drawbacks. Off-target affect, one of the most common shortcoming has made this technology often considered to be too risky to be put in actual treatment. CRISPR technology holds a great promising prospect in cancer treatment and immunotherapy through enhancing the direct or indirect inhibition of cancer growth. This article will provide an overview of the CRISPR system, its application in studying and screening of cancer cells, improving the efficacy and specificity of cancer immunotherapy, and possible future direction of cancer treatment, for example editing or modifying oncogene and drug resistant genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d917a93741bd28e997daf97db2b258b41c8e3620" target='_blank'>
              Progress of CRISPR Technology in Cancer Treatment
              </a>
            </td>
          <td>
            Junxin Zhang
          </td>
          <td>2024-07-24</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Each piece of cell-free DNA (cfDNA) has a length determined by the exact metabolic conditions in the cell it belonged to at the time of cell death. The changes in cellular regulation leading to a variety of patterns, which are based on the different number of fragments with lengths up to several hundred base pairs at each of the almost three billion genomic positions, allow for the detection of disease and also the precise identification of the tissue of their origin. A Kullback-Leibler (KL) divergence computation identifies different fragment lengths and areas of the human genome, depending on the stage, for which disease samples, starting from pre-clinical disease stages, diverge from healthy donor samples. We provide examples of genes related to colorectal cancer (CRC), which our algorithm detected to belong to divergent genomic bins. The staging of CRC can be viewed as a Markov Chain and that provides a framework for studying disease progression and the types of epigenetic changes occurring longitudinally at each stage, which might aid the correct classification of a new hospital sample. In a new look to treat such data as grayscale value images, pattern recognition using artificial intelligence (AI) could be one approach to classification. In CRC, Stage I disease does not, for the most part, shed any tumor circulation, making detection difficult for established machine learning (ML) methods. This leads to the deduction that early detection, where we can only rely on changes in the metabolic patterns, can be accomplished when the information is considered in its entirety, for example by applying computer vision methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769c868029bf18fe644fcae9a2141c21f2b5a763" target='_blank'>
              Channel Capacity of Genome-Wide Cell-Free DNA Fragment Length Distribution in Colorectal Cancer
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-07-18</td>
          <td>None</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="The seven human APOBEC3 (hA3) genes encode polynucleotide cytidine deaminases that play vital roles in restricting replication of viruses and retrotransposons. However, off-target A3 deamination of the cellular genome is a major source of somatic mutations in human cancer. The ability to study A3 biology in vivo is hindered by the fact that the solitary murine Apobec3 gene (mA3) encodes a cytoplasmic enzyme, with no apparent mutagenic activity. Transgenic expression of individual hA3 genes in mice has helped to confirm their oncogenic potential but important questions including which hA3 genes are active in different tissue contexts and how they function in concert when under control of their cognate promoters cannot be addressed using these models. Here we describe humanization of the mouse mA3 locus by integration of a modified BAC clone encompassing the entire 7-gene hA3 locus from human chromosome 22 replacing mA3 on mouse chromosome 15. APOBEC3 mice are viable and fertile and hA3 gene expression in cells and tissues correlates strongly with expression in corresponding human cells and tissues, indicating human-like regulation of hA3 gene expression in the mice. Splenocytes from this line display a functional human A3 response to Friend Murine Leukaemia Virus (F-MLV) infection. We propose that the Hs-APOBEC3 mouse will uniquely model the function of the complete hA3 locus in a living organism and that it will serve as a useful background upon which to model human cancer, as well as assisting drug discovery efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8034f8597ff29dbab41cdb895f637bb191838ad2" target='_blank'>
              Human-like APOBEC3 gene expression and anti-viral responses following replacement of mouse Apobec3 with the 7-gene human APOBEC3 locus
              </a>
            </td>
          <td>
            Nerissa K. Kirkwood, H. Prosser, Melvyn W Yap, Jane Gibson, Ross Cook, Ian Reddin, Ane Stranger, Emma Howes, Nur Zainal, M. Periyasamy, S. Conticello, Gareth J. Thomas, James Scott, Kate N. Bishop, Simak Ali, Allan Bradley, Tim R Fenton
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Persister cells (PS) selected for anticancer therapy have been recognized as a significant contributor to the development of treatment-resistant malignancies. It is found that imposing glutamine restriction induces the generation of PS, which paradoxically bestows heightened resistance to glutamine restriction treatment by activating the integrated stress response and initiating the general control nonderepressible 2-activating transcription factor 4-alanine, serine, cysteine-preferring transporter 2 (GCN2-ATF4-ASCT2) axis. Central to this phenomenon is the stress-induced ATF4 translational reprogramming. Unfortunately, directly targeting ATF4 protein has proven to be a formidable challenge because of its flat surface. Nonetheless, a G-quadruplex structure located within the promoter region of ATF4 (ATF4-G4) is uncovered and resolved, which functions as a transcriptional regulator and can be targeted by small molecules. The investigation identifies the natural compound coptisine (COP) as a potent binder that interacts with and stabilizes ATF4-G4. For the first time, the high-resolution structure of the COP-ATF4-G4 complex is determined. The formation of this stable complex disrupts the interaction between transcription factor AP-2 alpha (TFAP2A) and ATF4-G4, resulting in a substantial reduction in intracellular ATF4 levels and the eventual death of cancer cells. These seminal findings underscore the potential of targeting the ATF4-G4 structure to yield significant therapeutic advantages within the realm of persister cancer cells induced by glutamine-restricted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e9a3aeeaaba15f0673e425713df633d5da2043" target='_blank'>
              Overcoming Cancer Persister Cells by Stabilizing the ATF4 Promoter G-quadruplex.
              </a>
            </td>
          <td>
            Chengmei Xiao, Yipu Li, Yushuang Liu, Ruifang Dong, Xiaoyu He, Qing Lin, Xin Zang, Kaibo Wang, Yuanzheng Xia, Lingyi Kong
          </td>
          <td>2024-07-12</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd17af76ab151f83c8653ba7afa98d9d3324c71" target='_blank'>
              Simultaneous multi-site editing of individual genomes using retron arrays.
              </a>
            </td>
          <td>
            A. González-Delgado, Santiago C. Lopez, Matías Rojas-Montero, C. Fishman, Seth L. Shipman
          </td>
          <td>2024-07-09</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, Bert van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba856bced2e48ff7a94743b3236b182aa83cc934" target='_blank'>
              Epigenome editing technologies for discovery and medicine.
              </a>
            </td>
          <td>
            Sean R. McCutcheon, Dahlia Rohm, Nahid Iglesias, C. Gersbach
          </td>
          <td>2024-07-29</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) stands as one of the most prevalent and aggressive forms of renal malignancies, accounting for a substantial portion of kidney cancer-related mortality worldwide. The heterogeneity within ccRCC poses a significant challenge to understanding its molecular mechanisms and devising effective therapeutic strategies. [1] Previous studies utilizing single nucleus RNA sequencing (snRNA-seq) have revealed transcriptional contributions to cell-type specificity in both mature human and mouse kidneys. Moreover, recent advancements have expanded this approach to single-cell profiling of chromatin accessibility. Single-nuclei assay for transposase-accessible chromatin using sequencing (snATAC-seq) has enabled the measurement of chromatin accessibility in thousands of individual cells, providing insights into the dynamic process that drives nephron development and differentiation. Integration and analysis of multimodal single-cell datasets, such as snRNA-seq and snATAC-seq, allows for the prediction of cell-type-specific cis-regulatory DNA interactions and transcription factor activity, complementing the transcriptional information obtained by snRNA-seq.[2] [3] [1] Yang, Wang, and Yang, “Treatment Strategies for Clear Cell Renal Cell Carcinoma.” [2] Muto et al., “Single Cell Transcriptional and Chromatin Accessibility Profiling Redefine Cellular Heterogeneity in the Adult Human Kidney.” [3] Monteagudo-Sánchez, Noordermeer, and Greenberg, “The Impact of DNA Methylation on CTCF-Mediated 3D Genome Organization.” Methods This study leveraged paired snRNA/ATAC-seq in tumor and normal-adjacent samples from the Renal Tumor Biobank at Dartmouth (n= 63) to explore the ccRCC landscape at single-cell resolution using 10x multiome technology. The snRNA-seq dataset was processed with Seurat v5.0 to remove low-quality nuclei. Doublets were removed with DoubletFinder v2.02, and ambient DNA was removed using decontX from celda. Normalization was conducted via SCTransform, and missing data was imputed with ALRA. Clustering was performed by constructing a KNN graph and running the Louvain algorithm. Cell type assignments were predicted using Azimuth with the human kidney reference dataset. The snATAC-seq data was processed with ArchR v1.0.1, which performed dimensionality reduction and clustering. Cluster identities defined in the Seurat workflow were cross-annotated to the ATAT-seq data, and open peaks were identified within these cell-type clusters using MACS3 v3.0.1. Identification of marker peaks, motif enrichment, and motif enrichment deviations were inferred utilizing ArchR’s getmarkerFeatures, peakAnnoEnrichment, and addMotifAnnotations functions, respectively. Results Motif enrichment analysis revealed canonical cancer-associated transcription factor (TF) dysregulation in tumor cells (nephron loop-like), including enrichment of HNF1B, SMARCC1, FOSL1, and JUND. Interestingly, HNF and SMARCC were also enriched in the lymphoid compartment, indicating possible crosstalk between these two cell types. Previous work has investigated the involvement of the HNF1 in ccRCC and in chromophobe RCC for prognosis,and we plan on investigating the relationship of these enrichments to tumor grade, stage and outcome.[1] Additional TF enrichments included BCL families and ELF within the myeloid compartment, and vascularization related TF families in endothelial cell types including enrichment of various ETS TFs. Investigation of cell-cell interactions between immune and endothelial cell populations as well as between tumor and normal adjacent tissues will be performed to further disentangle the role differentially accessable chromatin plays in ccRCC. [1] Buchner et al., “Downregulation of HNF-1B in Renal Cell Carcinoma Is Associated With Tumor Progression and Poor Prognosis.” Conclusions Our multi-omic approach has several advantages. First, it enables the characterization of cellular heterogeneity within the tumor microenvironment, identifying cell populations and their contributions to disease progression. Secondly, by integrating transcriptomic and epigenomic data, we gain insights into the dynamic interplay between gene expression and chromatin accessibility, uncovering potential therapeutic vulnerabilities and biomarkers for personalized treatment strategies. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e88d9a75048a69eb2afe5150d509757fff83b4" target='_blank'>
              60 Dissecting the Cellular and Epigenetic Landscape of Clear Cell Renal Cell Carcinoma Through Paired snRNA/ATAC-Seq Analysis
              </a>
            </td>
          <td>
            Ian Gingerich
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA methylation is an important epigenetic regulatory mechanism which plays a crucial role in cell differentiation and development. Its function is closely related to DNA methyltransferase 3 alpha (DNMT3A), which can affect gene expression and stem cell differentiation. The mutation rate of the DNMT3A gene is relatively high in Acute myeloid leukemia (AML), but its type and pathogenic mechanism are not yet clear. Further research on DNMT3A may help to identify its pathogenic targets and provide a basis for precise treatment of AML. This article has provided a review for the research progress on the expression of the DNMT3A gene in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ec69c604a676713e1a7e86038752ff556999862" target='_blank'>
              [Research progress on DNMT3A gene expression in Acute myeloid leukemia].
              </a>
            </td>
          <td>
            Jiawei Zhou, Tao Wu, Wenhui Liu
          </td>
          <td>2024-08-10</td>
          <td>Zhonghua yi xue yi chuan xue za zhi = Zhonghua yixue yichuanxue zazhi = Chinese journal of medical genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Proteasome inhibitors (PIs), bortezomib, carfilzomib, and ixazomib, are the first-line treatment for multiple myeloma (MM). They inhibit cytosolic protein degradation in cells, which leads to the accumulation of misfolded and malfunctioned proteins in the cytosol and endoplasmic reticulum, resulting in cell death. Despite being a breakthrough in MM therapy, malignant cells develop resistance to PIs via different mechanisms. Understanding these mechanisms drives research toward new anticancer agents to overcome PI resistance. In this review, we summarize the mechanism of action of PIs and how MM cells adapt to these drugs to develop resistance. Finally, we explore these mechanisms to present strategies to interfere with PI resistance. The strategies include new inhibitors of the ubiquitin–proteasome system, drug efflux inhibitors, autophagy disruption, targeting stress response mechanisms, affecting survival and cell cycle regulators, bone marrow microenvironment modulation, and immunotherapy. We list potential pharmacological targets examined in in vitro, in vivo, and clinical studies. Some of these strategies have already provided clinicians with new anti-MM medications, such as panobinostat and selinexor. We hope that further exploration of the subject will broaden the range of therapeutic options and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3182993bd2a4adb29f0241a9e69655965af9da42" target='_blank'>
              Different Strategies to Overcome Resistance to Proteasome Inhibitors—A Summary 20 Years after Their Introduction
              </a>
            </td>
          <td>
            Paweł Tyrna, Grzegorz Procyk, Łukasz Szeleszczuk, I. Młynarczuk-Biały
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) are the remnants of retroviral germline infections and are highly abundant in the genomes of vertebrates. At one time considered to be nothing more than inert ‘junk’ within genomes, ERVs have been tolerated within host genomes over vast timescales, and their study continues to reveal complex co-evolutionary histories within their respective host species. For example, multiple instances have been characterized of ERVs having been ‘borrowed’ for normal physiology, from single copies to ones involved in various regulatory networks such as innate immunity and during early development. Within the cell, the accessibility of ERVs is normally tightly controlled by epigenetic mechanisms such as DNA methylation or histone modifications. However, these silencing mechanisms of ERVs are reversible, and epigenetic alterations to the chromatin landscape can thus lead to their aberrant expression, as is observed in abnormal cellular environments such as in tumors. In this review, we focus on ERV transcriptional control and draw parallels and distinctions concerning the loss of regulation in disease, as well as their precise regulation in early development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88fcb4c839dae363d4185cc295fc1a0003652e42" target='_blank'>
              Transcription of Endogenous Retroviruses: Broad and Precise Mechanisms of Control
              </a>
            </td>
          <td>
            Abigail S. Jarosz, Julia V. Halo
          </td>
          <td>2024-08-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA base lesions, such as incorporation of uracil into DNA or base mismatches, can be mutagenic and toxic to replicating cells. To discover factors in repair of genomic uracil, we performed a CRISPR knockout screen in the presence of floxuridine, a chemotherapeutic agent that incorporates uracil and fluoro-uracil into DNA. We identified known factors, such as uracil DNA N-glycosylase (UNG), but also unknown factors, such as the N6-adenosine methyltransferase, METTL3, as required to overcome floxuridine-driven cytotoxicity. Visualized with immunofluorescence, the product of METTL3 activity, N6-methyladenosine, formed nuclear foci in cells treated with floxuridine. The observed N6-methyladenosine was embedded in DNA, called 6mA, which was confirmed using mass spectrometry. METTL3 and 6mA were required for repair of lesions driven by additional base damaging agents, including raltitrexed, gemcitabine, and hydroxyurea. Our results establish a role for METTL3 and 6mA to promote genome stability in mammalian cells, specially in response to base damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db989576812beb4e3ed7a27aef984122d9c71b2" target='_blank'>
              N6-methyladenosine in DNA promotes genome stability
              </a>
            </td>
          <td>
            Brooke A. Conti, Leo Novikov, Deyan Tong, Qing Xiang, Savon Vigil, Thomas J. McLellan, Chuong Nguyen, Nancy De La Cruz, Reshma T. Veettil, Prashant Pradhan, Parag Sahasrabudhe, Jason D. Arroyo, Lei Shang, Benjamin R. Sabari, David J. Shields, Mariano Oppikofer
          </td>
          <td>2024-07-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb85026de0a1065b160b9eed08ef28f424a5cc3" target='_blank'>
              The AKT inhibitor AZD5363 elicits synthetic lethality in ARID1A-deficient gastric cancer cells via induction of pyroptosis.
              </a>
            </td>
          <td>
            Menghan Fang, Youfen Lin, Chaorong Xue, Kaiqin Sheng, Zegeng Guo, Yuting Han, Hanbin Lin, Yuecheng Wu, Yuchao Sang, Xintan Chen, Stephen B. Howell, Xu Lin, Xinjian Lin
          </td>
          <td>2024-07-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Progression from pre-cancers like ductal carcinoma in situ (DCIS) to invasive disease (cancer) is driven by somatic evolution and is altered by clinical interventions. We hypothesized that genetic and/or phenotypic intra-tumor heterogeneity would predict clinical outcomes for DCIS since it serves as the substrate for natural selection among cells. We profiled two samples from two geographically distinct foci from each DCIS in both cross-sectional (N = 119) and longitudinal cohorts (N = 224), with whole exome sequencing, low-pass whole genome sequencing, and a panel of immunohistochemical markers. In the longitudinal cohorts, the only statistically significant predictors of time to non-invasive DCIS recurrence were the combination of treatment (lumpectomy only vs mastectomy or lumpectomy with radiation, HR = 12.13, p = 0.003, Wald test with FDR correction), ER status (HR = 0.16 for ER+ compared to ER-, p = 0.0045), and divergence in SNVs between the two samples (HR = 1.33 per 10% divergence, p = 0.018). SNV divergence also distinguished between pure DCIS and DCIS synchronous with invasive disease in the cross-sectional cohort. In contrast, the only statistically significant predictors of time to progression to invasive disease were the combination of the width of the surgical margin (HR = 0.67 per mm, p = 0.043) and the number of mutations that were detectable at high allele frequencies (HR = 1.30 per 10 SNVs, p = 0.02). These results imply that recurrence with DCIS is a clinical and biological process different from invasive progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2027c796343907ccaa630775f7381619210caedc" target='_blank'>
              Evolutionary Measures Show that Recurrence of DCIS is Distinct from Progression to Breast Cancer
              </a>
            </td>
          <td>
            Angelo Fortunato, D. Mallo, Luis Cisneros, Lorraine M. King, Aziz Khan, Christina Curtis, M. Ryser, Joseph Y. Lo, Allison Hall, Jeffrey R. Marks, E. S. Hwang, Carlo C. Maley
          </td>
          <td>2024-08-16</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f22523498a31f883e04ffefa6645c4ecc8fbfb" target='_blank'>
              Systematic optimization of prime editing for the efficient functional correction of CFTR F508del in human airway epithelial cells.
              </a>
            </td>
          <td>
            Alexander A Sousa, Colin Hemez, Lei Lei, Soumba Traore, Katarina Kulhankova, Gregory A. Newby, Jordan L. Doman, Keyede Oye, Smriti Pandey, Philip H Karp, P. B. Mccray, David R. Liu
          </td>
          <td>2024-07-10</td>
          <td>Nature biomedical engineering</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="Impaired control of the G1/S checkpoint allows initiation of DNA replication under non-permissive conditions. Unscheduled S-phase entry is associated with DNA replication stress, demanding for other checkpoints or cellular pathways to maintain proliferation. Here, we uncovered a requirement for ADARp150 to sustain proliferation of G1/S-checkpoint-defective cells under growth-restricting conditions. Besides its well-established mRNA editing function in inversely oriented short interspersed nuclear elements (SINEs), we found ADARp150 to exert a critical function in mitosis. ADARp150 depletion resulted in tetraploidization, impeding cell proliferation in mitogen-deprived conditions. Mechanistically we show that ADAR1 depletion induced aberrant expression of Cyclin B3, which was causative for mitotic failure and whole-genome duplication. Finally, we find that also in vivo ADAR1-depletion-provoked tetraploidization hampers tumor outgrowth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d2bb8be1edc7659c2bc78816fc95a0c749bf23" target='_blank'>
              ADARp150 counteracts whole genome duplication.
              </a>
            </td>
          <td>
            Frank van Gemert, Alexandra Drakaki, Isabel Morales Lozano, Daniel de Groot, Maud Schoot Uiterkamp, N. Proost, C. Lieftink, M. van de Ven, R. Beijersbergen, Heinz Jacobs, H. te Riele
          </td>
          <td>2024-08-27</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
The genetic material within cells plays a pivotal role in shaping the structure and function of living organisms. Manipulating an organism's genome to correct inherited abnormalities or introduce new traits holds great promise. Genetic engineering techniques offers promising pathways for precisely altering cellular genetics. Among these methodologies, clustered regularly interspaced short palindromic repeat (CRISPR), honored with the 2020 Nobel Prize in Chemistry, has garnered significant attention for its precision in editing genomes. However, the CRISPR system faces challenges when applied in vivo, including low delivery efficiency, off‐target effects, and instability. To address these challenges, innovative technologies for targeted and precise delivery of CRISPR have emerged. Engineered carrier platforms represent a substantial advancement, improving stability, precision, and reducing the side effects associated with genome editing. These platforms facilitate efficient local and systemic genome engineering of various tissues and cells, including immune cells. This review explores recent advances, benefits, and challenges of CRISPR‐based genome editing delivery. It examines various carriers including nanocarriers (polymeric, lipid‐derived, metallic, and bionanoparticles), viral particles, virus‐like particles, and exosomes, providing insights into their clinical utility and future prospects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f48664bd4bcab7f27ff34a25f2e1c64fa38e6d" target='_blank'>
              Exploring Advanced CRISPR Delivery Technologies for Therapeutic Genome Editing
              </a>
            </td>
          <td>
            N. Rostami, M. M. Gomari, Edris Choupani, Shadi Abkhiz, Mahmood Fadaie, Seyed Sadegh Eslami, Zahra Mahmoudi, Yapei Zhang, Madhu Puri, Fatemeh Nafe Monfared, Elena Demireva, V. Uversky, B. R. Smith, S. Bencherif
          </td>
          <td>2024-07-25</td>
          <td>Small Science</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Triplex DNA formation is a useful genomic targeting tool that is expected to have a wide range of applications, including the antigene method; however, there are fundamental limitations in its forming sequence. We recently extended the triplex DNA-forming sequence to methylated DNA sequences containing 5mCG base pairs by developing guanidino-dN, which is capable of recognizing a 5mCG base pair with high affinity. We herein investigated the effect of triplex DNA formation using TFOs with guanidino-dN on methylated DNA sequences at the promoter of the RASSF1A gene, whose expression is epigenetically suppressed by DNA methylation in MCF-7 cells, on gene expression. Interestingly, triplex DNA formation increased the expression of the RASSF1A gene at the transcript and protein levels. Furthermore, RASSF1A-activated MCF-7 cells exhibited cell growth suppressing activity. Changes in the expression of various genes associated with the promotion of apoptosis and breast cancer survival accompanied the activation of RASSF1A in cells exhibited antiproliferative activity. These results suggest the potential of increases in gene expression through triplex DNA formation as a new genomic targeting tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/367d9640afcf3f528a05a1512247caf159507678" target='_blank'>
              Novel strategy for activating gene expression through triplex DNA formation targeting epigenetically suppressed genes
              </a>
            </td>
          <td>
            Ryotaro Notomi, Shigeki Sasaki, Yosuke Taniguchi
          </td>
          <td>2024-07-31</td>
          <td>RSC Chemical Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer mortality worldwide. KRAS oncogenes are responsible for at least a quarter of lung adenocarcinomas, the main subtype of lung cancer. After four decades of intense research, selective inhibitors of KRAS oncoproteins are finally reaching the clinic. Yet, their effect on overall survival is limited due to the rapid appearance of drug resistance, a likely consequence of the high intratumoral heterogeneity characteristic of these tumors. In this study, we have attempted to identify those functional alterations that result from KRAS oncoprotein expression during the earliest stages of tumor development. Such functional changes are likely to be maintained during the entire process of tumor progression regardless of additional co-occurring mutations. Single-cell RNA sequencing analysis of murine alveolar type 2 cells expressing a resident Kras oncogene revealed impairment of the type I interferon pathway, a feature maintained throughout tumor progression. This alteration was also present in advanced murine and human tumors harboring additional mutations in the p53 or LKB1 tumor suppressors. Restoration of type I interferon (IFN) signaling by IFN-β or constitutive active stimulator of interferon genes (STING) expression had a profound influence on the tumor microenvironment, switching them from immunologically "cold" to immunologically "hot" tumors. Therefore, enhancement of the type I IFN pathway predisposes KRAS mutant lung tumors to immunotherapy treatments, regardless of co-occurring mutations in p53 or LKB1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4ab14ca9fcef735ebd7c09bd8738b90b51f2b7" target='_blank'>
              Type I interferon signaling pathway enhances immune-checkpoint inhibition in KRAS mutant lung tumors.
              </a>
            </td>
          <td>
            Fernando Fernández-García, Ana Fernández-Rodríguez, C. Fustero-Torre, Elena Piñeiro-Yáñez, Haiyun Wang, C. Lechuga, S. Callejas, Rebeca Álvarez, Alejandra López-García, Laura Esteban-Burgos, Marina Salmón, Marta San Román, Carmen Guerra, Chiara Ambrogio, M. Drosten, David Santamaría, Fátima Al-Shahrour, Ana Dopazo, Mariano Barbacid, M. Musteanu
          </td>
          <td>2024-08-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Radiotherapy is one of the main cancer treatments being used for ~50% of all cancer patients. Conventional radiotherapy typically utilises X-rays (photons); however, there is increasing use of particle beam therapy (PBT), such as protons and carbon ions. This is because PBT elicits significant benefits through more precise dose delivery to the cancer than X-rays, but also due to the increases in linear energy transfer (LET) that lead to more enhanced biological effectiveness. Despite the radiotherapy type, the introduction of DNA damage ultimately drives the therapeutic response through stimulating cancer cell death. To combat this, cells harbour cell cycle checkpoints that enables time for efficient DNA damage repair. Interestingly, cancer cells frequently have mutations in key genes such as TP53 and ATM that drive the G1/S checkpoint, whereas the G2/M checkpoint driven through ATR, Chk1 and Wee1 remains intact. Therefore, targeting the G2/M checkpoint through specific inhibitors is considered an important strategy for enhancing the efficacy of radiotherapy. In this review, we focus on inhibitors of Chk1 and Wee1 kinases and present the current biological evidence supporting their utility as radiosensitisers with different radiotherapy modalities, as well as clinical trials that have and are investigating their potential for cancer patient benefit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c16232402c6f8af47e8e32325c40b75dba212a77" target='_blank'>
              The Potential for Targeting G2/M Cell Cycle Checkpoint Kinases in Enhancing the Efficacy of Radiotherapy
              </a>
            </td>
          <td>
            Emma Melia, Jason L. Parsons
          </td>
          <td>2024-08-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Intratumoral hypoxia is associated with tumor progression, aggressiveness, and therapeutic resistance in several cancers. Hypoxia causes cancer cells to experience replication stress, thereby activating DNA damage and repair pathways. MutT homologue-1 (MTH1, also known as NUDT1), a member of the Nudix family, maintains the genomic integrity and viability of tumor cells in the hypoxic tumor microenvironment. Although hypoxia is associated with poor prognosis and can cause therapeutic resistance by regulating the microenvironment, it has not been considered a treatable target in cancer. This study aimed to investigate whether hypoxia-induced MTH1 is a useful target for immunotherapy and whether hypoxic conditions influence the antitumor activity of immune cells. Our results showed that MTH1 expression was elevated under hypoxic conditions in head and neck cancer cell lines. Furthermore, we identified a novel MTH1-targeting epitope peptide that can activate peptide-specific CD4+ helper T cells with cytotoxic activity. The proliferation and cytotoxic activity of T cells were maintained under hypoxic conditions, and PD-1 blockade further augmented the cytotoxicity. These results indicate that MTH1-targeted immunotherapy combined with checkpoint blockade can be an effective strategy for the treatment of hypoxic tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02fd79a48171b74f56ec3547f207f7b41e323150" target='_blank'>
              Hypoxia-Targeted Immunotherapy with PD-1 Blockade in Head and Neck Cancer
              </a>
            </td>
          <td>
            Risa Wakisaka, Hidekiyo Yamaki, Michihisa Kono, Takahiro Inoue, Ryosuke Sato, H. Komatsuda, K. Ohara, A. Kosaka, T. Ohkuri, T. Nagato, K. Kishibe, Koh Nakayama, Hiroya Kobayashi, T. Kumai, Miki Takahara
          </td>
          <td>2024-08-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Retrons are promising gene editing tools because they can produce multi-copy single-stranded DNA in cells via self-primed reverse transcription. However, their potential for inserting genetic cargos in eukaryotes remains largely unexplored. Here we report the discovery and engineering of highly efficient retron-based gene editors for mammalian cells and vertebrates. Through bioinformatic analysis of metagenomic data and functional screening, we identified novel retron reverse transcriptases (RTs) that are highly active in mammalian cells. Rational design further improved the editing efficiency to levels comparable with conventional single-stranded oligodeoxynucleotide donors but from a genetically encoded cassette. Small molecule inhibitors of non-homologous end joining factors and Cas9-DNA repair protein fusions further increase homology-directed repair. Retron editors also exhibited robust activity with Cas12a nuclease and Cas9 nickase, expanding the genomic target scope and bypassing the need for a DNA double-stranded break. Using a rationally engineered retron editor, we incorporate a split GFP epitope tag for live cell imaging. Finally, we develop an all-RNA delivery strategy to enable DNA-free gene editing in cells and vertebrate embryos. This work establishes retron editors as a versatile and efficient tool for precise gene editing, offering new opportunities for biotechnology and biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8f2c620f5c7bf4a518c60425b0c63267672964" target='_blank'>
              Discovery and Engineering of Retrons for Precise Genome Editing
              </a>
            </td>
          <td>
            Jesse D. Buffington, Hung-Che Kuo, Kuang Hu, You-Chiun Chang, Kamyab Javanmardi, Brittney Voigt, Yi-Ru Li, Mary E. Little, Sravan K Devanathan, Blerta Xhemalçe, R. Gray, Ilya J. Finkelstein
          </td>
          <td>2024-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="CRISPR/Cas9 gene editing in the model green alga Chlamydomonas reinhardtii relies on the use of selective marker genes to enrich for non-selectable target mutations. This becomes challenging when many sequential modifications are required in a single cell line, as useful markers are limited. Here, we demonstrate a cyclical selection process which only requires a single marker gene to identify an almost infinite sequential series of CRISPR-based target gene modifications. We used the NIA1 (Nit1, NR; nitrate reductase) gene as the selectable marker in this study. In the forward stage of the cycle, a stop codon was engineered into the NIA1 gene at the CRISPR target location. Cells retaining the wild-type NIA1 gene were killed by chlorate, while NIA1 knockout mutants survived. In the reverse phase of the cycle, the stop codon engineered into the NIA1 gene during the forward phase was edited back to the wild-type sequence. Using nitrate as the sole nitrogen source, only the reverted wild-type cells survived. By using CRISPR to specifically deactivate and reactivate the NIA1 gene, a marker system was established that flipped back and forth between chlorate- and auxotrophic (nitrate)-based selection. This provided a scarless cyclical marker system that enabled an indefinite series of CRISPR edits in other, non-selectable genes. We demonstrate that this 'Sequential CRISPR via Recycling Endogenous Auxotrophic Markers (SCREAM)' technology enables an essentially limitless series of genetic modifications to be introduced into a single cell lineage of C. reinhardtii in a fast and efficient manner to complete complex genetic engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f75908314649d1188956bda26ea29417db966e6a" target='_blank'>
              A cyclical marker system enables indefinite series of oligonucleotide-directed gene editing in Chlamydomonas reinhardtii.
              </a>
            </td>
          <td>
            Ian L. Ross, Hong Phuong Le, Sabar Budiman, Dake Xiong, Fritz Hemker, Elizabeth A. Millen, Melanie Oey, Ben Hankamer
          </td>
          <td>2024-08-23</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequent and mortality-causing neoplasia, with various distributions between populations. Strong hereditary predispositions are the causatives of a small percentage of CRC, and most cases have no transparent genetic background. This is a vast arena for exploring cancer low-susceptibility genetic variants. Nonetheless, the research that has been conducted to date has failed to deliver consistent conclusions and often features conflicting messages, causing chaos in this field. Therefore, we decided to organize the existing knowledge on this topic. We screened the PubMed and Google Scholar databases. We drew up markers by gene locus gathered by hallmark: oncogenes, tumor suppressor genes, genes involved in DNA damage repair, genes involved in metabolic pathways, genes involved in methylation, genes that modify the colonic microenvironment, and genes involved in the immune response. Low-penetration genetic variants increasing the risk of cancer are often population-specific, hence the urgent need for large-scale testing. Such endeavors can be successful only when financial decision-makers are united with social educators, medical specialists, genetic consultants, and the scientific community. Countries’ policies should prioritize research on this subject regardless of cost because it is the best investment. In this review, we listed potential low-penetrance CRC susceptibility alleles whose role remains to be established.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0541d34727973134a35fb48d39ef83a36856b18" target='_blank'>
              Low-Penetrance Susceptibility Variants in Colorectal Cancer—Current Outlook in the Field
              </a>
            </td>
          <td>
            Marcin Szuman, M. Kaczmarek-Ryś, S. Hryhorowicz, Alicja Kryszczyńska, Natalia Grot, Andrzej Pławski
          </td>
          <td>2024-07-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="By reviewing the literature and summarizing and refining it, this research investigated the various ways in which CRISPR gene editing technology can treat the leukemia pathology, and this research will look forward to the future application of CRISPR-based gene editing technology. At present, there are two main ways to apply CRISPR technology to influence leukemia treatment. The first is to directly knock out a known disease-causing gene to inhibit the proliferation of harmful cells, thus delaying the disease and realizing the purpose of leukemia treatment. The other is to indirectly inhibit leukemia by inserting a certain gene through the CRISPR gene editing technology to achieve the inhibition of the expression of disease-causing genes. However, the current direction of research by using CRISPR technology to explore the pathogenesis of leukemia and disease-causing genes is relatively small. In the future, more research and experiments can be conducted to explore the pathogenesis and causative genes of leukemia by applying CRISPR technology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a6c6fc8190aafb211ac3d5b505a7af035cdf19" target='_blank'>
              Leukemia Treatment Based on CRISPR Genome-Editing Technology
              </a>
            </td>
          <td>
            Yiting Liu
          </td>
          <td>2024-07-11</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a2685872da8abf8d89c45e2e00bd648c1daaeb" target='_blank'>
              DNA methylation and gene expression as determinants of genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Michaël Noë, Dimitrios Mathios, A. Annapragada, Shashikant Koul, Zachariah H. Foda, J. Medina, S. Cristiano, Christopher Cherry, D. C. Bruhm, N. Niknafs, V. Adleff, Leonardo Ferreira, Hari Easwaran, Stephen Baylin, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="Since the identification of DNA as the hereditary material, therapeutic advancements have focused on gene therapy (GT) and allogeneic hematopoietic cell transplants for inherited diseases. Despite successes in treating conditions like β-thalassemia and sickle cell disease, challenges such as insertional oncogenesis and immunogenicity persist. Future efforts must prioritize improving vector safety, refining manufacturing processes, and expanding access to GT to fully realize its potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9198c613dac012277fe80dd191ba49946328eeff" target='_blank'>
              Audacity of Gene Therapy.
              </a>
            </td>
          <td>
            Marketa Dimitrov, C. Ebens, Jakub Tolar
          </td>
          <td>2024-08-20</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53766cf525e4b93ccab4c09464ebfa98a84e6734" target='_blank'>
              Transcriptome- and DNA methylation-based cell-type deconvolutions produce similar estimates of differential gene expression and differential methylation
              </a>
            </td>
          <td>
            Emily R. Hannon, Carmen J. Marsit, Arlene E. Dent, P. Embury, Sidney Ogolla, D. Midem, Scott M. Williams, James W. Kazura
          </td>
          <td>2024-07-11</td>
          <td>BioData Mining</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Immune checkpoint inhibitor (ICI) therapies for tumors of different systems have attained significant achievements and have changed the current situation of tumor treatment due to their therapeutic characteristics of high specificity and low side effects. The immune checkpoint Programmed death 1/Programmed cell death-Ligand 1 (PD-1/PD-L1) axis exerts a vital role in the immune escape of tumor cells. As a result, it has become a key target for tumor immunotherapy. Therefore, to perfect research into potential regulatory factors for the PD-1/PD-L1 axis, in order to understand and illustrate tumor ICI therapy mechanisms, is a significant goal. Moreover, ncRNA has been verified to regulate the PD-1/PD-L1 axis in the tumor immune microenvironment to regulate tumor genesis and development. ncRNAs can improve or decrease the efficacy of ICI therapy by modulating PD-L1 expression. This review aimed to investigate the mechanisms of action of ncRNA in regulating the PD-1/PD-L1 axis in ICI therapy, to provide more efficient immunotherapy for tumors of different systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a649160555a3a8ac9ed7bee54f17e40bc70e1bd5" target='_blank'>
              Mechanism research of non-coding RNA in immune checkpoint inhibitors therapy.
              </a>
            </td>
          <td>
            Jie Bian, Rui Shao, Juan Li, Jing-Feng Zhu, Ai-Zhong Shao, Chao Liu, L. V. Lu, Hui-Wen Pan, Yi-Jun Shi, Na Fang
          </td>
          <td>2024-08-13</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cellular senescence has been increasingly recognized as a hallmark of cancer, reflecting its association with aging and inflammation, its role as a response to deregulated proliferation and oncogenic stress, and its induction by cancer therapies. While therapy-induced senescence (TIS) has been linked to resistance, recurrence, metastasis, and normal tissue toxicity, TIS also has the potential to enhance therapy response and stimulate anti-tumor immunity. In this review, we examine the Jekyll and Hyde nature of senescent cells (SnCs), focusing on how their persistence while expressing the senescence-associated secretory phenotype (SASP) modulates the tumor microenvironment through autocrine and paracrine mechanisms. Through the SASP, SnCs can mediate both resistance and response to cancer therapies. To fulfill the unmet potential of cancer immunotherapy, we consider how SnCs may influence tumor inflammation and serve as an antigen source to potentiate anti-tumor immune response. This new perspective suggests treatment approaches based on TIS to enhance immune checkpoint blockade. Finally, we describe strategies for mitigating the detrimental effects of senescence, such as modulating the SASP or targeting SnC persistence, which may enhance the overall benefits of cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52dbb8efb9179d8a3a7870f442fb59ca7f4eb573" target='_blank'>
              Therapy-Induced Cellular Senescence: Potentiating Tumor Elimination or Driving Cancer Resistance and Recurrence?
              </a>
            </td>
          <td>
            Yue Liu, Isabelle Lomeli, Stephen J. Kron
          </td>
          <td>2024-07-30</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Identifying biomarkers to predict immune checkpoint inhibitor (ICI) efficacy is warranted. Considering that somatic mutation-derived neoantigens induce strong immune responses, patients with a high tumor mutational burden reportedly tend to respond to ICIs. Therefore, the original function of neoantigenic mutations and their impact on the tumor microenvironment (TME) require attention. RNF43 is a type of RING E3 ubiquitin ligase, and long-term survivors in most cancers had conserved patterns of mutations of RNF43. Also, high microsatellite instability patients had a higher RNF43 mutation rate compared with microsatellite stability tumor patients, who were more sensitive to ICI treatment. Therefore, RNF43 has become a promising biomarker of immunotherapy in a wide range of cancers. This review focuses on the up-to-date knowledge of RNF43 mutation in cancer. We summarize the cancer hallmarks involving activities regulated by RNF43 and highlight its extremely sophisticated regulation of WNT signaling and tumor microenvironment. The key genes interacting with RNF43 have also been summarized and discussed. Additionally, we highlight and propose new strategies of targeting RNF43 and RNF43-based combinations with established immunotherapy and combination therapy. These efforts may provide new perspectives for RNF43-based target therapy in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c419259554c9a116f2d42d506d7f1a73316659d7" target='_blank'>
              RNF43 in cancer: Molecular understanding and clinical significance in immunotherapy.
              </a>
            </td>
          <td>
            Xingfa Huo, Weizhong Han, Zhen Yang, Yongzhi Lu, Ning Liu, H. Hou
          </td>
          <td>2024-08-01</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Simple Summary Individualized medicine means understanding how each tumor is different from normal cells and how each tumor is different from other tumors, including profiling mutations, non-mutational epigenetic changes, and differences in gene expression. This allows the discovery of key processes each tumor absolutely depends on for survival and growth, which are intrinsic weaknesses. This profiling means selecting treatments to specifically target each tumor’s survival-dependent pathways, killing them. Tip60 is a master controller of processes that maintain genome stability and signaling regulating gene expression. While disrupted in many cancers, Tip60 is essential for cell survival, and inhibiting Tip60 kills tumors. While we understand some key aspects of the molecular roles Tip60 plays, much more remains to be discovered. A more complete understanding of the diverse roles and functions of Tip60 in cancer, and how targeting Tip60 kills cancer cells, will lead to better treatments for patients and increased survival. Abstract Precision (individualized) medicine relies on the molecular profiling of tumors’ dysregulated characteristics (genomic, epigenetic, transcriptomic) to identify the reliance on key pathways (including genome stability and epigenetic gene regulation) for viability or growth, and then utilises targeted therapeutics to disrupt these survival-dependent pathways. Non-mutational epigenetic changes alter cells’ transcriptional profile and are a key feature found in many tumors. In contrast to genetic mutations, epigenetic changes are reversable, and restoring a normal epigenetic profile can inhibit tumor growth and progression. Lysine acetyltransferases (KATs or HATs) protect genome stability and integrity, and Tip60 is an essential acetyltransferase due to its roles as an epigenetic and transcriptional regulator, and as master regulator of the DNA double-strand break response. Tip60 is commonly downregulated and mislocalized in many cancers, and the roles that mislocalized Tip60 plays in cancer are not well understood. Here we categorize and discuss Tip60-regulated genes, evaluate Tip60-interacting proteins based on cellular localization, and explore the therapeutic potential of Tip60-targeting compounds as epigenetic inhibitors. Understanding the multiple roles Tip60 plays in tumorigenesis will improve our understanding of tumor progression and will inform therapeutic options, including informing potential combinatorial regimes with current chemotherapeutics, leading to improvements in patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0558cedd75f278d6feed6575b276aad74d694cf" target='_blank'>
              Evaluating the Cellular Roles of the Lysine Acetyltransferase Tip60 in Cancer: A Multi-Action Molecular Target for Precision Oncology
              </a>
            </td>
          <td>
            Nazanin Zohourian, Erin Coll, Muiread Dever, Anna Sheahan, Petra Burns-Lane, James A. L. Brown
          </td>
          <td>2024-07-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Disruption of the epigenome is a hallmark of human disease including liver cirrhosis and hepatocellular carcinoma (HCC). While genetic heterogeneity is an established effector of pathologic phenotypes, epigenetic heterogeneity is less well understood. Environmental exposures alter the liver-specific DNA methylation landscape and influence the onset of liver cancer. Given that currently available treatments are unable to target frequently mutated genes in HCC, there is an unmet need for novel therapeutics to prevent or reverse liver damage leading to hepatic tumorigenesis, which the epigenome may provide. We performed genome-wide profiling of DNA methylation, copy number, and gene expression from multiple liver regions from 31 patients with liver disease to examine their crosstalk and define the individual and combinatorial contribution of these processes to liver disease progression. We identify epigenetic heterogeneity hotspots that are conserved across patients. Elevated epigenetic heterogeneity associates with increased gene expression heterogeneity. Cirrhotic regions comprise two distinct cohorts, one exclusively epigenetic, and a second where epigenetic and copy number variations collaborate. Epigenetic heterogeneity hotspots are enriched for genes central to liver function (e.g., HNF1A) and known tumor suppressors (e.g., RASSF1A). These hotspots encompass genes including ACSL1, ACSL5, MAT1A, and ELFN1, which have phenotypic effects in functional screens, supporting their relevance to hepatocarcinogenesis. Moreover, epigenetic heterogeneity hotspots are linked to clinical measures of outcome.


CONCLUSION
Substantial epigenetic heterogeneity arises early in liver disease development, targeting key pathways in the progression and initiation of both cirrhosis and HCC. Integration of epigenetic and transcriptional heterogeneity unveils putative epigenetic regulators of hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46b475bb15d42990ae95e5c459fd1ea63a3f93cd" target='_blank'>
              Epigenetic heterogeneity hotspots in human liver disease progression.
              </a>
            </td>
          <td>
            R. Hlady, Xia Zhao, Louis El Khoury, Ryan T. Wagner, Aesis M. Luna, Kien Pham, Nikolaos Pyrosopoulos, Dhanpat Jain, Liguo Wang, Chen Liu, Keith D. Robertson
          </td>
          <td>2024-07-19</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Wilms' tumor is a malignant neoplasm where current medical advancements have significantly improved survival rates; however, challenges persist such as the resistance of the tumor to chemotherapy drugs like doxorubicin. This necessitates higher dosages, leading to decreased sensitivity. However, using high doses of doxorubicin can have late effects on the heart. Unc-13 homolog B (UNC13B) may be involved in the drug resistance in several tumors, yet its role in modulating drug sensitivity in Wilms' tumor remains unexplored. UNC13B levels were quantified using reverse transcription-qPCR and Western blotting. The half-maximal inhibitory concentration for doxorubicin, vincristine, and actinomycin-D was determined using CCK-8 assays. Cell cycle and apoptosis were analyzed using flow cytometry, and lysosomal changes were observed using Lyso-Tracker staining. The present study initially evaluated UNC13B expression levels in the Wilms' tumor 17.94 cell line. Additionally, through short hairpin RNA-mediated knockdown, changes in doxorubicin sensitivity in 17.94 Wilms' tumor cells were assessed. Concurrently, preliminary investigations into the role of UNC13B in regulating lysosomes was performed, revealing a significant positive association between UNC13B levels and lysosome formation in the 17.94 cell line. Lysosomes likely serve a role in the sensitivity of Wilms' tumor cell lines to drugs. Elevated UNC13B expression was observed in the 17.94 Wilms' tumor cell line compared to normal kidney cells. UNC13B knockdown also resulted in increased apoptosis levels upon doxorubicin treatment. Immunofluorescence revealed UNC13B localization within cellular vesicles, and its knockdown significantly decreased lysosome levels. Overall, the findings of the present study demonstrate that UNC13B regulates the sensitivity of the Wilms' tumor 17.94 cell line to doxorubicin by modulating lysosome formation within cells. The results suggest that UNC13B is likely an enriched target involved in lysosomal regulation in certain tumors, offering a new approach for optimizing chemotherapy in Wilms' tumor and other cancers with high UNC13B expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe89ea39861be17508761078e7b43d988f8eb8e" target='_blank'>
              UNC13B regulates the sensitivity of Wilms' tumor cells to doxorubicin by modulating lysosomes
              </a>
            </td>
          <td>
            Xi Chen, Ying-ying Bao, Ge Sun, Xiaobo Wang, Jiajun Zhu
          </td>
          <td>2024-07-22</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b0e84c70be39b55c5ab372a108ddda6ab970765" target='_blank'>
              Polθ: emerging synthetic lethal partner in homologous recombination-deficient tumors.
              </a>
            </td>
          <td>
            T. B. Bazan Russo, Clarissa Mujacic, Emilia Di Giovanni, Maria Concetta Vitale, Carla Ferrante Bannera, U. Randazzo, Silvia Contino, M. Bono, V. Gristina, A. Galvano, A. Perez, G. Badalamenti, A. Russo, V. Bazan, L. Incorvaia
          </td>
          <td>2024-08-09</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>28</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [20],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>