<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-05 13:10:33 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>1</td>
          <td>40</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Epithelial-to-mesenchymal transition (EMT) is a major axis of phenotypic plasticity not only in diseased conditions such as cancer metastasis and fibrosis but also during normal development and wound healing. Yet-another important axis of plasticity with metastatic implications includes the cancer stem cell (CSCs) and non-CSC transitions. However, in both processes, epithelial (E) and mesenchymal (M) phenotypes are not merely binary states. Cancer cells acquire a spectrum of phenotypes with traits, properties, and markers of both E and M phenotypes, giving rise to intermediary hybrid (E/M) phenotypes. E/M cells play an important role in tumor initiation, metastasis, and disease progression in multiple cancers. Furthermore, the hybrid phenotypes also play a major role in causing therapeutic resistance in cancer. Here, we discuss how a systems biology perspective on the problem, which is implicit in the ‘Team Medicine’ approach outlined in the theme of this Special Issue of The Journal of Clinical Medicine and includes an interdisciplinary team of experts, is more likely to shed new light on EMT in cancer and help us to identify novel therapeutics and strategies to target phenotypic plasticity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6867e0a77d67eee55a122d53d387d619f33219b" target='_blank'>
              Phenotypic Plasticity and Cancer: A System Biology Perspective
              </a>
            </td>
          <td>
            A. R. Subbalakshmi, S. Ramisetty, Atish Mohanty, Siddhika Pareek, Dana Do, Sagun Shrestha, Ajaz Khan, Neel Talwar, Tingting Tan, Priya Vishnubhotla, Sharad S Singhal, Ravi Salgia, Prakash Kulkarni
          </td>
          <td>2024-07-23</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Therapeutic resistance is the leading cause of treatment failure and death from cancer. While resistance can be driven by genetic mutations, mounting evidence also points to an epigenetic basis of resistance. Much of this epigenetic, or non-genetic, resistance has been attributed to drug-resistant transcriptional cell states that are either induced by drug treatment or pre-exist in a fraction of cells selected by treatment. However, the extent to which long-term resistance is manifested in the early inherent cellular response to drugs is poorly understood, and which aspects, if any, of this early response to drug-induced transcriptional response are evolutionarily conserved properties of cells. To address these questions, we integrate datasets of long-term drug resistance and early drug response data across multiple cell lines with drug response and resistance data from bacteria and yeast. Our findings suggest that cancer cell states in both the drug-naive populations as well as in populations shortly after treatment share transcriptional properties with fully established resistant cell populations, and CRISPR-cas9 knockout of transcription factors predicted to regulate the resistant transcriptional programs result in inceased drug sensitivity. Furthermore, the resistance states manifested as early drug response are evolutionarily conserved. Finally, we show that early resistant states discriminate responders from non-responders across multiple human cancer trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad4a4d49f9d38cdc5b6bcd770ac201bba19fb9f" target='_blank'>
              Transcriptional Characterization of Resistance in Early Drug Response
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Michael M. Gottesman, Sridhar Hannenhalli
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2f585a6d644d3577541d684c98016ab2b165e02" target='_blank'>
              Sensing and guiding cell-state transitions by using genetically encoded endoribonuclease-mediated microRNA sensors.
              </a>
            </td>
          <td>
            Lei Wang, Wenlong Xu, Shun Zhang, Gregory C Gundberg, Christine R Zheng, Zhengpeng Wan, Kamila Mustafina, Fabio Caliendo, Hayden Sandt, Roger Kamm, Ron Weiss
          </td>
          <td>2024-07-09</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfd5cecdc67c5ea14eee4cb838269e77eddb6f01" target='_blank'>
              Farnesyltransferase inhibition overcomes oncogene-addicted non-small cell lung cancer adaptive resistance to targeted therapies
              </a>
            </td>
          <td>
            S. Figarol, C. Delahaye, R. Gence, A. Doussine, J. Cerapio, Mathylda Brachais, Claudine Tardy, N. Béry, R. Asslan, J. Colinge, J. Villemin, Antonio Maraver, Irene Ferrer, Luis Paz-Ares, Linda Kessler, Francis Burrows, Isabelle Lajoie-Mazenc, Vincent Dongay, Clara Morin, Amélie Florent, S. Pagano, E. Taranchon-Clermont, A. Casanova, A. Pradines, Julien Mazieres, Gilles Favre, O. Calvayrac
          </td>
          <td>2024-06-27</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Leukemia is characterized by oncogenic lesions that result in a block of differentiation while phenotypic plasticity is retained. A better understanding of how these two phenomena arise during leukemogenesis in humans could help inform diagnosis and treatment strategies. Here, we leveraged the well-defined differentiation states during T-cell development to pinpoint the initiation of T-cell acute lymphoblastic leukemia (T-ALL), an aggressive form of childhood leukemia, and study the emergence of phenotypic plasticity. Single-cell whole genome sequencing of leukemic blasts was combined with multiparameter flow cytometry to couple cell identity and clonal lineages. Irrespective of genetic events, leukemia-initiating cells altered their phenotypes by differentiation and de-differentiation. Construction of phylogenies of individual leukemias using somatic mutations revealed that phenotypic diversity is reflected by the clonal structure of the cancer. The analysis also indicated that the acquired phenotypes are heritable and stable. Together, these results demonstrate a transient period of plasticity during leukemia initiation where phenotypic switches appear unidirectional.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0eb1ac967dc33753add0f41321531d1ce4cb054a" target='_blank'>
              Transient Differentiation-State Plasticity Occurs During Acute Lymphoblastic Leukemia Initiation.
              </a>
            </td>
          <td>
            Vera M. Poort, R. Hagelaar, Markus J van Roosmalen, L. Trabut, J. Buijs-Gladdines, Bram van Wijk, Jules P P Meijerink, R. van Boxtel
          </td>
          <td>2024-06-17</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Deficiencies in DNA mismatch repair (MMRd) leave characteristic footprints of microsatellite instability (MSI) in cancer genomes. We used data from the Cancer Genome Atlas and International Cancer Genome Consortium to conduct a comprehensive analysis of MSI-associated cancers, focusing on indel mutational signatures. We classified MSI-high genomes into two subtypes based on their indel profiles: deletion-dominant (MMRd-del) and insertion-dominant (MMRd-ins). Compared with MMRd-del genomes, MMRd-ins genomes exhibit distinct mutational and transcriptomic features, including a higher prevalence of T>C substitutions and related mutation signatures. Short insertions and deletions in MMRd-ins and MMRd-del genomes target different sets of genes, resulting in distinct indel profiles between the two subtypes. In addition, indels in the MMRd-ins genomes are enriched with subclonal alterations that provide clues about a distinct evolutionary relationship between the MMRd-ins and MMRd-del genomes. Notably, the transcriptome analysis indicated that MMRd-ins cancers upregulate immune-related genes, show a high level of immune cell infiltration, and display an elevated neoantigen burden. The genomic and transcriptomic distinctions between the two types of MMRd genomes highlight the heterogeneity of genetic mechanisms and resulting genomic footprints and transcriptomic changes in cancers, which has potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b737e04b95df7e27390442dd66d22a5ba373a014" target='_blank'>
              A Subset of Microsatellite Unstable Cancer Genomes Prone to Short Insertions over Deletions Is Associated with Elevated Anticancer Immunity
              </a>
            </td>
          <td>
            Sunmin Kim, Dong-Jin Han, Seo-Young Lee, Youngbeen Moon, Su Jung Kang, Tae-Min Kim
          </td>
          <td>2024-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer resistance refers to the ability of cancer cells to evade the effects of treatments, such as chemotherapy and radiation, making it challenging to eliminate tumors. Cancer cells develop resistance through genetic modifications, alterations in cellular pathways, or microenvironmental adaptations that allow cancer cells to persist, proliferate, and even thrive despite the application of therapies designed to eradicate them. Despite efforts to create more potent target-based drugs, the flexibility of acquired resistance necessitates advanced options. Combination therapy and precision immunotherapy has revolutionized treatment but is limited by patient specificity and requires further development. New approaches like small molecules, peptides, and nanotherapeutics aim to overcome resistance by enhancing site-specific delivery and increasing drug concentration inside cancer cells. This review provides a comprehensive overview of the molecular mechanisms underlying cancer drug resistance, including genetic mutations, epigenetic alterations, efflux pump activity, and the influence of the tumor microenvironment. Additionally, the roles of microRNAs, long noncoding RNAs, and cellular processes such as autophagy and hypoxia in mediating resistance are examined. This review seeks to improve patient care and help to the development of more effective cancer medicines by explaining these complicated systems and investigating novel therapeutic alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c262ab9bdd31ff19f10e8cd7a734094b9985bc60" target='_blank'>
              An overview of molecular mechanisms in cancer drug resistance and therapeutic strategies
              </a>
            </td>
          <td>
            Navin Kumar, Ranjeeta Bansala
          </td>
          <td>2024-07-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cellular responses to environmental stimuli are typically thought to be governed by genetically encoded programs. We demonstrate that melanoma cells can form and maintain cellular memories during the acquisition of therapy resistance that exhibit characteristics of cellular learning and are dependent on the transcription factor AP-1. We show that cells exposed to a low dose of therapy adapt to become resistant to a high dose, demonstrating that resistance was not purely selective. The application of therapy itself results in the encoding of transient gene expression into cellular memory and that this encoding occurs for both transiently induced and probabilistically arising expression. Chromatin accessibility showed concomitant persistence. A two-color AP-1 reporter system showed that these memories are encoded in cis, constituting an example of activating cis epigenetics. Our findings establish the formation and maintenance of cellular memories as a critical aspect of gene regulation during the development of therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a15dd395d0e2a350d250357de4baa259bd2f20" target='_blank'>
              AP-1 Mediates Cellular Adaptation and Memory Formation During Therapy Resistance
              </a>
            </td>
          <td>
            Jingxin Li, Pavithran T. Ravindran, Aoife O’Farrell, Gianna T. Busch, Ryan H. Boe, Zijian Niu, Sean Woo, Margaret C. Dunagin, Naveen Jain, Yogesh Goyal, K. Sarma, M. Herlyn, Arjun Raj
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Outcomes for women with breast cancer have improved dramatically in recent decades. However, many patients present with intrinsic drug resistance and others are initially sensitive to anti-cancer drugs but acquire resistance during the course of their treatment, leading to recurrence and/or metastasis. Drug therapy-induced senescence (TIS) is a form of drug resistance characterised by the induction of cell cycle arrest and the emergence of a senescence-associated secretory phenotype (SASP) that can develop in response to chemo- and targeted- therapies. A wide range of anticancer interventions can lead to cell cycle arrest and SASP induction, by inducing genotoxic stress, hyperactivation of signalling pathways or oxidative stress. TIS can be anti-tumorigenic in the short-term, but pro-tumorigenic in the long-term by creating a pro-inflammatory and immunosuppressive microenvironment. Moreover, the SASP can promote angiogenesis and epithelial-mesenchymal transition in neighbouring cells. In this review, we will describe the characteristics of TIS in breast cancer and detail the changes in phenotype that accompany its induction. We also discuss strategies for targeting senescent cancer cells in order to prevent or delay tumour recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1902ff7d1c0379c707774d89c56e002ae6793dc1" target='_blank'>
              Therapy-induced senescence in breast cancer: an overview
              </a>
            </td>
          <td>
            Suraj Narayanan Chembukavu, Andrew J Lindsay
          </td>
          <td>2024-07-25</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a particularly aggressive mammary neoplasia with a high fatality rate, mainly because of the development of resistance to administered chemotherapy, the standard treatment for this disease. In this study, we employ both bulk RNA-sequencing and single-cell RNA-sequencing (scRNA-seq) to investigate the transcriptional landscape of TNBC cells cultured in two-dimensional monolayers or three-dimensional spheroids, before and after developing resistance to the chemotherapeutic agents paclitaxel and doxorubicin. Our findings reveal significant transcriptional heterogeneity within the TNBC cell populations, with the scRNA-seq identifying rare subsets of cells that express resistance-associated genes not detected by the bulk RNA-seq. Furthermore, we observe a partial shift towards a highly mesenchymal phenotype in chemoresistant cells, suggesting the epithelial-to-mesenchymal transition (EMT) as a prevalent mechanism of resistance in subgroups of these cells. These insights highlight potential therapeutic targets, such as the PDGF signaling pathway mediating EMT, which could be exploited in this setting. Our study underscores the importance of single-cell approaches in understanding tumor heterogeneity and developing more effective, personalized treatment strategies to overcome chemoresistance in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33e0aae4bc7f6edfa5e8ffddd5c3bb7acfad4a59" target='_blank'>
              Identification of New Chemoresistance-Associated Genes in Triple-Negative Breast Cancer by Single-Cell Transcriptomic Analysis
              </a>
            </td>
          <td>
            S. Foutadakis, Dimitrios Kordias, G. Vatsellas, A. Magklara
          </td>
          <td>2024-06-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="All routine clinical treatments for colorectal cancer include 5-fluorouracil (5-FU), which cannot counteract recurrence and metastases formation. As the pyrimidine analog 5-FU can impact multiple pathways including both DNA and RNA metabolism, studying its mode of actions could lead to improved therapies. Using a dedicated reporter system for lineage-tracing and deep translatome profiling we demonstrate that 5-FU causes some colorectal cancer cells to tolerate the drug, due to a durable translational reprogramming that sustains cell plasticity. This period of drug tolerance coincides with specific translational activation of genes coding for proteins with major pro-tumoral functions. We unravel a major unexpected translational overexpression of the pro-inflammatory and pro-tumoral IL-8 cytokine, alongside other anti-apoptotic, senescence-associated secretory phenotype and cancer-related senescence phenotype genes. Given the adverse prognostic implications of elevated IL-8 levels across various cancers, our findings suggest IL-8 targeting could counteract 5-FU resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3f76dde38afaccb213072b4b26b46e5ff73bd2b" target='_blank'>
              Translational control of cell plasticity drives 5-FU tolerance
              </a>
            </td>
          <td>
            Mounira Chalabi-Dchar, Olivia Villeronce, J. Ripoll, A. Vincent, T. Fenouil, Rita Khoueiry, Jérôme Kucharczak, Laura Jentschel, Frédéric Catez, Arnaud M Vigneron, Julie Tréguier, Céline Mandier, Céline Bouclier, Jihane Vitre, Louise Lagerqvist, Armelle Choquet, Z. Herceg, C. Machon, Jérôme Guitton, Alexandre David, Eric Solary, David Bernard, Nadine Martin, Eric Rivals, N. D. Venezia, Julie Pannequin, J- J. Diaz
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Background: Cancer resistance to chemotherapy arises from various genetic and epigenetic changes that promote malignant cell proliferation. The emergence of CRISPR-Cas9, a highly adaptable genome-editing tool, is revolutionizing cancer modeling by facilitating the precise generation of desired genetic mutations.
Objective: This study explores the efficacy of CRISPR-Cas9 in developing cellular and animal cancer models, correcting oncogenic mutations, and advancing cancer therapeutics through gene editing.
Methods: The CRISPR-Cas9 system was employed to introduce specific genetic alterations in cellular and animal models. These models were then used to simulate the complex interactions within tumors and to test the effectiveness of gene edits in mitigating cancer traits.
Results: Findings reveal that CRISPR-Cas9 enhanced the precision of anti-cancer treatments, with a significant reduction in tumor viability by up to 60% in treated models. Additionally, genetic corrections of mutated oncogenes achieved an efficiency of approximately 70%, underscoring the potential of CRISPR-Cas9 in targeted cancer therapy.
Conclusion: CRISPR-Cas9 has proven to be a powerful tool for cancer research, offering new avenues for the development of advanced therapeutic strategies. Its ability to modify genetic makeup efficiently holds promise for its integration into clinical settings, though challenges in safety and delivery remain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b99c0c1d043c0661f840f21beb535ad9c76b1a39" target='_blank'>
              Towards Personalized Cancer Care: A Report of CRISPR-Cas9 Applications in Targeted Therapies and Precision Medicine
              </a>
            </td>
          <td>
            A. Noor, Asif Bilal, Umer Ali
          </td>
          <td>2024-06-15</td>
          <td>Journal of Health and Rehabilitation Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Copy number alterations (CNAs), resulting from the gain or loss of genetic material from as little as 50 base pairs or as big as entire chromosome(s), have been associated with many congenital diseases, de novo syndromes and cancer. It is established that CNAs disturb the dosage of genomic regions including enhancers/promoters, long non-coding RNA and gene(s) among others, ultimately leading to an altered balance of key cellular functions. In cancer, CNAs have been associated with almost all steps of the disease: predisposition, initiation, development, maintenance, response to treatment, resistance, and relapse. Therefore, understanding how specific CNAs contribute to tumourigenesis may provide prognostic insight and ultimately lead to the development of new therapeutic approaches to improve patient outcomes. In this review, we provide a snapshot of what is currently known about CNAs and cancer, incorporating topics regarding their detection, clinical impact, origin, and nature, and discuss the integration of innovative genetic engineering strategies, to highlight the potential for targeting CNAs using novel, dosage-sensitive and less toxic therapies for CNA-driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b107cf5f7a3a29e8e0fe916d4102734572eefc3a" target='_blank'>
              Insights into the Clinical, Biological and Therapeutic Impact of Copy Number Alteration in Cancer
              </a>
            </td>
          <td>
            Shannon L. Carey-Smith, R. Kotecha, L. C. Cheung, Sébastien Malinge
          </td>
          <td>2024-06-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to sustain rapid proliferation, resist cell death, and adapt to varying microenvironmental conditions. This review elucidates the key mechanisms underlying metabolic reprogramming in cancer cells, including alterations in glucose metabolism, glutamine addiction, and lipid biosynthesis. A central feature of cancer metabolism is the Warburg effect, where cancer cells preferentially utilize aerobic glycolysis over oxidative phosphorylation, even in the presence of oxygen. This metabolic shift is driven by oncogenes and tumor suppressor genes, such as MYC and TP53, which modulate the expression and activity of enzymes involved in glycolysis and mitochondrial function. Additionally, cancer cells exhibit increased glutaminolysis, relying on glutamine as a carbon and nitrogen source to support anabolic processes and redox balance. Lipid metabolism is also reprogrammed, with enhanced de novo lipogenesis supplying membrane components and signaling molecules critical for tumor growth and survival. Understanding these metabolic alterations provides a basis for developing targeted therapies. Several therapeutic strategies have emerged, including inhibitors of key metabolic enzymes, such as hexokinase 2 (HK2), pyruvate kinase M2 (PKM2), and glutaminase. Metabolic interventions can disrupt the metabolic flexibility of cancer cells, sensitizing them to conventional therapies and overcoming resistance mechanisms. This review highlights the potential of metabolic targets in cancer treatment and emphasizes the need for personalized approaches considering tumor-specific metabolic profiles. Future research should focus on identifying biomarkers of metabolic vulnerabilities and developing combination therapies that exploit the intricate metabolic dependencies of cancer cells. Ultimately, targeting metabolic reprogramming holds promise for improving cancer prognosis and achieving more effective and durable therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d015236b22d21b366aae600c9f9e8d4b15a3c83" target='_blank'>
              A review of metabolic reprogramming in cancer cells: Mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Kelly Osayi Otakhor, Elizabeth O. Soladoye
          </td>
          <td>2024-07-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821cee27b336dd922ca1d195350feabe9168efcf" target='_blank'>
              CRISPR-Cas9 potential for identifying novel therapeutic targets in muscle-invasive bladder cancer.
              </a>
            </td>
          <td>
            Danielle J Smith, S. Lunj, Antony D Adamson, Sankari Nagarajan, Tim A D Smith, Kimberley J Reeves, P. Hoskin, Ananya Choudhury
          </td>
          <td>2024-07-01</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Epigenetic evolution is a common mechanism used by cancer cells to evade the therapeutic effects of drug treatment. In ovarian cancers, epigenetically-driven resistance may be responsible for a large number of late-stage patient deaths. Here, we describe the first investigation into the role of G-quadruplex (G4) DNA secondary structures in mediating epigenetic regulation in drug-resistant ovarian cancer cells. Through genome-wide mapping of G4s in paired drug-sensitive and drug-resistant cell lines, we find that increased G4 formation is associated with significant increase in gene expression, with high enrichment in signalling pathways previously established to promote drug-resistant states. However, in contrast to previous studies, the expression-enhancing effects of G4s were not found at gene promoters, but intergenic and intronic regions, indicating that G4s promote long-range transcriptional regulation in drug-resistant cells. Furthermore, we discovered that clusters of G4s (super-G4s) are associated with particularly high levels of transcriptional enhancement that surpass the effects of super-enhancers, which act as well established regulatory sites in many cancers. Finally, we demonstrate that targeting G4s with small molecules results in significant down-regulation of pathways associated with drug-resistance, which results in resensitisation of resistant cells to chemotherapy agents. These findings indicate that G4 structures are critical for the epigenetic regulatory networks of drug-resistant cells and may represent a promising target to treat drug-tolerant ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef77f99d4d1172d6a547e45d37053b1f52fb2ad" target='_blank'>
              G-quadruplex structures regulate long-range transcriptional reprogramming to promote drug resistance in ovarian cancer
              </a>
            </td>
          <td>
            Jenna Robinson, Gem Flint, I. Garner, Silvia Galli, Thomas E. Maher, Marina K. Kuimova, Ramon Vilar, I. McNeish, Robert Brown, Hector Keun, Marco Di Antonio
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="
 Aneuploidy, an imbalanced number of chromosomes or chromosome arms, is the most common genetic aberration in cancer and is associated with a worse prognosis and advanced disease. However, in non-transformed cells, aneuploidy has a substantial fitness cost, leading to cell cycle arrest and cell death. This fitness cost is the result of several cellular stresses associated with aneuploidy, including proteotoxic, metabolic, mitotic and replication stress. Therefore, for aneuploidy to become beneficial, tolerance mechanisms that enable cancer cells to cope with aneuploidy-induced cellular stresses must come into play. Such mechanisms may uncover novel synthetic lethalities of aneuploid cancer cells. To this end, we systematically dissected the cellular mechanisms that facilitate aneuploidy tolerance. We combined large-scale analyses of gene expression, as well as genetic and pharmacological dependency datasets from human tumors, cancer cell lines and non-transformed cells, to identify genes and pathways associated with aneuploidy tolerance. We identified 55 genes associated (with high confidence) with aneuploidy tolerance. Gene set enrichment analysis (GSEA) revealed an enrichment for ribosome biogenesis and translation, DNA repair, replication stress and the p53 pathway. Interestingly, these pathways were all previously implicated with the aforementioned aneuploidy-induced cellular stresses. We plan to conduct a dependency mini-screen to validate the top ‘hits’ from our comprehensive bioinformatic analysis, and to explore the role of validated candidates in conferring aneuploidy tolerance. One of the top ‘hits’ of our exploratory analysis was a novel gene with an unclear cellular function. We experimentally confirmed that this gene was over-expressed in aneuploid cells, across several isogenic systems of matched diploid-aneuploid cell lines. Aneuploidy induction using MPS1 inhibitors led to its significant upregulation, demonstrating that its increased expression is a general feature of aneuploid cells. Mechanistically, we found that the novel gene had a role in regulating chromosome segregation, as its knockdown increased the rate of mitotic aberrations – as well as the sensitivity to mitotic checkpoint inhibition – resulting in increased chromosomal instability. We now study the molecular basis for the effect of this novel gene on aneuploidy tolerance. Specifically, we knocked it down in aneuploid cells, and over-expressed it in diploid cells, and we currently explore how these manipulations affect the above-mentioned aneuploidy tolerance pathways, namely replication stress, ribosome biogenesis, DNA damage repair and tp53 activation. In summary, we revealed a novel gene that is a key regulator of aneuploidy tolerance in cancer cells, likely due to its direct involvement in pathways required for the cellular coping with aneuploidy-induced stresses.
 Citation Format: Yonatan Eliezer, Adi V Tarrab, Tal Ben-Yishay, Hajime Okada, Tom Winkler, Uri Ben-David. Identification of novel genes that regulate aneuploidy tolerance by attenuating aneuploidy-induced stresses [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82fed258fc2885195d90b257738b8f12490644d6" target='_blank'>
              Abstract B003: Identification of novel genes that regulate aneuploidy tolerance by attenuating aneuploidy-induced stresses
              </a>
            </td>
          <td>
            Yonatan Eliezer, Adi V Tarrab, Tal Ben-Yishay, Hajime Okada, Tom Winkler, Uri Ben-David
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Multiple myeloma (MM) is a genetically heterogeneous disease and the management of relapses is one of the biggest clinical challenges. TP53 alterations are established high‐risk markers and are included in the current disease staging criteria. KRAS is the most frequently mutated gene affecting around 20% of MM patients. Applying Clonal Competition Assays (CCA) by co‐culturing color‐labeled genetically modified cell models, we recently showed that mono‐ and biallelic alterations in TP53 transmit a fitness advantage to the cells. Here, we report a similar dynamic for two mutations in KRAS (G12A and A146T), providing a biological rationale for the high frequency of KRAS and TP53 alterations at MM relapse. Resistance mutations, on the other hand, did not endow MM cells with a general fitness advantage but rather presented a disadvantage compared to the wild‐type. CUL4B KO and IKZF1 A152T transmit resistance against immunomodulatory agents, PSMB5 A20T to proteasome inhibition. However, MM cells harboring such lesions only outcompete the culture in the presence of the respective drug. To better prevent the selection of clones with the potential of inducing relapse, these results argue in favor of treatment‐free breaks or a switch of the drug class given as maintenance therapy. In summary, the fitness benefit of TP53 and KRAS mutations was not treatment‐related, unlike patient‐derived drug resistance alterations that may only induce an advantage under treatment. CCAs are suitable models for the study of clonal evolution and competitive (dis)advantages conveyed by a specific genetic lesion of interest, and their dependence on external factors such as the treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736cb2bf2240ae0e1d6133faf0f979a9afbbf92" target='_blank'>
              Clonal competition assays identify fitness signatures in cancer progression and resistance in multiple myeloma
              </a>
            </td>
          <td>
            L. Haertle, U. Munawar, Hipólito N. C. Hernández, Andrés Arroyo-Barea, T. Heckel, I. Cuenca, Lucia Martin, Carlotta Höschle, Nicole Müller, C. Vogt, Thorsten Bischler, Paula L. del Campo, Seungbin Han, Natalia Buenache, Xiang Zhou, Florian Bassermann, J. Waldschmidt, Torsten Steinbrunn, L. Rasche, T. Stühmer, J. Martínez-López, K. Martin Kortüm, S. Barrio
          </td>
          <td>2024-07-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Simple Summary Lung adenocarcinoma, the most prevalent type of lung cancer, presents a significant treatment challenge owing to drug resistance. This study aimed to investigate the role of long non-coding RNAs (lncRNAs) in promoting intrinsic resistance in three lung adenocarcinoma cell lines from the onset of treatment. Drug-tolerant persister (DTP) cells, a subset of cancer cells with the ability to survive and proliferate after exposure to therapeutic drugs, were generated. RNA sequencing was used to investigate the differential expression of lncRNAs, and the clinical relevance of lncRNAs was assessed in a cohort of lung adenocarcinoma patients from The Cancer Genome Atlas. Knockdown of these lncRNAs increases the sensitivity of the analyzed cell lines to therapeutic drugs. This study provides an opportunity to investigate the role of lncRNAs in the genetic and epigenetic mechanisms underlying intrinsic resistance. Abstract Lung adenocarcinoma is the most prevalent form of lung cancer, and drug resistance poses a significant obstacle in its treatment. This study aimed to investigate the overexpression of long non-coding RNAs (lncRNAs) as a mechanism that promotes intrinsic resistance in tumor cells from the onset of treatment. Drug-tolerant persister (DTP) cells are a subset of cancer cells that survive and proliferate after exposure to therapeutic drugs, making them an essential object of study in cancer treatment. The molecular mechanisms underlying DTP cell survival are not fully understood; however, long non-coding RNAs (lncRNAs) have been proposed to play a crucial role. DTP cells from lung adenocarcinoma cell lines were obtained after single exposure to tyrosine kinase inhibitors (TKIs; erlotinib or osimertinib). After establishing DTP cells, RNA sequencing was performed to investigate the differential expression of the lncRNAs. Some lncRNAs and one mRNA were overexpressed in DTP cells. The clinical relevance of lncRNAs was evaluated in a cohort of patients with lung adenocarcinoma from The Cancer Genome Atlas (TCGA). RT–qPCR validated the overexpression of lncRNAs and mRNA in the residual DTP cells and LUAD biopsies. Knockdown of these lncRNAs increases the sensitivity of DTP cells to therapeutic drugs. This study provides an opportunity to investigate the involvement of lncRNAs in the genetic and epigenetic mechanisms that underlie intrinsic resistance. The identified lncRNAs and CD74 mRNA may serve as potential prognostic markers or therapeutic targets to improve the overall survival (OS) of patients with lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47f1b958113e3cfa109f6bab13ad3bc4cd8c921" target='_blank'>
              Transcriptomic Analysis Reveals Early Alterations Associated with Intrinsic Resistance to Targeted Therapy in Lung Adenocarcinoma Cell Lines
              </a>
            </td>
          <td>
            Mario Perez-Medina, J. Lopez-Gonzalez, Jesus J. Benito-Lopez, Santiago Ávila-Ríos, Maribel Soto-Nava, M. Matías-Florentino, A. Méndez-Tenorio, M. Galicia-Velasco, R. Chavez-Dominguez, S. Meza-Toledo, D. Aguilar-Cázares
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 The Werner syndrome helicase (WRN) has emerged as a promising synthetic lethal target in cancers with microsatellite instability (MSI). This discovery has spurred numerous efforts developing therapeutics targeting WRN and the advancement of at least two WRN programs into Phase I clinical trials. The development of acquired resistance remains a significant obstacle for the durable efficacy of targeted therapies in oncology and this challenge may be particularly pronounced in the setting of mismatch repair deficient (dMMR) tumors, which inherently possess large reservoirs of mutational burden. Despite the potential of WRN-targeted treatments, there remains a notable gap in the literature concerning the mechanisms of resistance to these drugs. To explore potential acquired resistance mechanisms of WRN-directed therapies, we examined three helicase inhibitors—HRO761 and two proprietary compounds which all possess a similar mechanism of action (MOA). Continuous treatment of HCT116 and SW48 cell lines with these inhibitors rapidly lead to the emergence of phenotypically resistant cell populations. Subsequent sequencing of several independently derived resistant cultures identified 5 emergent point mutations within the helicase domain of WRN that are potentially responsible for the observed inhibitor resistance. Parallel in vivo studies with HRO761 in SW48 xenograft tumors also demonstrated the acquisition of resistance after an initial period of deep response. Whole exome sequencing of these tumors revealed 2 of the previously observed, as well as 4 novel WRN helicase mutations. Structural modeling of WRN with putative resistance mutations suggested their capacity to either directly impede inhibitor binding or disfavor the adoption of the compound binding confirmation. Surprisingly, despite the general similarity in MOA and ligand binding sites, cross-resistance analyses demonstrated that some mutations conferred preferential resistance to HRO761 and were only minimally detrimental to the activity of our proprietary compounds. We conclude that WRN inhibitor monotherapy leads to drug resistance by acquisition of on-target mutations that disrupt inhibitor binding through direct or indirect mechanisms. This process may be extremely rapid in the dMMR background that represents the intent to treat population for WRN-directed therapeutics and is potentially further exacerbated by the complex allosteric MOA of WRN inhibitors currently in the clinic. Notably, some WRN inhibitor-resistant cell lines were not resistant to all WRN inhibitors tested, suggesting the potential for employing different WRN inhibitors at different stages of treatment to overcome resistance. This study underscores the importance of understanding and addressing resistance mechanisms to enhance the effectiveness of WRN-targeted therapies in MSI cancers.
 Citation Format: Faith C. Fowler, Aileen Kelly, Cindy Jeffries, Jessica Gajda, Jonathan Hickson, Fei Han, Nate Elsen, Mariam George, Danli Towne, Xiangdong Xu, Nate Gesmundo, Wei Qiu, Henry Tang, Charlie Hutchins, Yunsong Tong, Mick Dart, Ari Firestone. Understanding mechanisms of resistance to WRN small molecule inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr A002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a564fb5b1cc273702b30cede79e0ced9fe555ba" target='_blank'>
              Abstract A002: Understanding mechanisms of resistance to WRN small molecule inhibitors
              </a>
            </td>
          <td>
            Faith C. Fowler, Aileen Kelly, Cindy Jeffries, Jessica Gajda, Jonathan Hickson, Fei Han, Nate Elsen, Mariam D George, D. Towne, Xiangdong Xu, Nate Gesmundo, Wei Qiu, Henry Tang, Charlie Hutchins, Yunsong Tong, Mick Dart, Ari Firestone
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/795e6a16077e23dab48be945dcd0a59b697bb23e" target='_blank'>
              Mechanism of ERBB2 gene overexpression by the formation of super-enhancer with genomic structural abnormalities in lung adenocarcinoma without clinically actionable genetic alterations
              </a>
            </td>
          <td>
            S. Kaneko, Ken Takasawa, Ken Asada, K. Shiraishi, Noriko Ikawa, Hidenori Machino, Norio Shinkai, M. Matsuda, Mari Masuda, Shungo Adachi, Satoshi Takahashi, Kazuma Kobayashi, Nobuji Kouno, Amina Bolatkan, M. Komatsu, M. Yamada, M. Miyake, Hirokazu Watanabe, A. Tateishi, T. Mizuno, Y. Okubo, Masami Mukai, Tatsuya Yoshida, Yukihiro Yoshida, H. Horinouchi, Shun-ichi Watanabe, Y. Ohe, Yasushi Yatabe, V. Saloura, Takashi Kohno, R. Hamamoto
          </td>
          <td>2024-06-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eaccde1d6ccb6dacff234a0f6ddffc263c341a3" target='_blank'>
              Targeting oncogene-induced cellular plasticity for tumor therapy
              </a>
            </td>
          <td>
            Bin Li, Lingling Zheng, Jianhua Yang, Lianghu Qu
          </td>
          <td>2024-07-15</td>
          <td>Advanced Biotechnology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Abnormal cell proliferation and growth leading to cancer primarily result from cumulative genome mutations. Single gene mutations alone do not fully explain cancer onset and progression; instead, clustered mutations—simultaneous occurrences of multiple mutations—are considered to be pivotal in cancer development and advancement. These mutations can affect different genes and pathways, resulting in cells undergoing malignant transformation with multiple functional abnormalities. Clustered mutations influence cancer growth rates, metastatic potential, and drug treatment sensitivity. This summary highlights the various types and characteristics of clustered mutations to understand their associations with carcinogenesis and discusses their potential clinical significance in cancer. As a unique mutation type, clustered mutations may involve genomic instability, DNA repair mechanism defects, and environmental exposures, potentially correlating with responsiveness to immunotherapy. Understanding the characteristics and underlying processes of clustered mutations enhances our comprehension of carcinogenesis and cancer progression, providing new diagnostic and therapeutic approaches for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d085d5b52cda0d463d3c07016cf9320b287c441e" target='_blank'>
              Exploring the Role of Clustered Mutations in Carcinogenesis and Their Potential Clinical Implications in Cancer
              </a>
            </td>
          <td>
            Yi Li, Rui Zhu, Jiaming Jin, Haochuan Guo, Jiaxi Zhang, Zhiheng He, Tingming Liang, Li Guo
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Diffuse large B-cell lymphoma is the most common type of lymphoma. Despite initial treatment, up to 40% of patients do not achieve a cure and need second-line therapy. For those experiencing late relapse or lacking access to CAR-T cell therapy, platinum-based chemotherapy followed by stem cell transplantation remains the standard of care. In this study, we used genomewide CRISPR/Cas9 screens and single-gene knockout experiments to identify the genes associated with responses to platinum-based drugs. We provide a comprehensive list of genes involved in the response to cisplatin in DLBCL. Our functional experiments highlight the critical roles of the DNA damage response genes XPA and ERCC6, in addition to BTK, in the response to platinum-based drugs. Additionally, we show that inhibition of BTK at lower concentrations sensitizes DLBCL cells to platinum-based drugs. Abstract The recurrence of diffuse large B-cell lymphoma (DLBCL) has been observed in 40% of cases. The standard of care for refractory/relapsed DLBCL (RR-DLBCL) is platinum-based treatment prior to autologous stem cell transplantation; however, the prognosis for RR-DLBCL patients remains poor. Thus, to identify genes affecting the cisplatin response in DLBCL, cisplatin-based whole-genome CRISPR-Cas9 knockout screens were performed in this study. We discovered DNA damage response (DDR) pathways as enriched among identified sensitizing CRISPR-mediated gene knockouts. In line, the knockout of the nucleotide excision repair genes XPA and ERCC6 sensitized DLBCL cells to platinum drugs irrespective of proliferation rate, thus documenting DDR as essential for cisplatin sensitivity in DLBCL. Functional analysis revealed that the loss of XPA and ERCC6 increased DNA damage levels and altered cell cycle distribution. Interestingly, we also identified BTK, which is involved in B-cell receptor signaling, to affect cisplatin response. The knockout of BTK increased cisplatin sensitivity in DLBCL cells, and combinatory drug screens revealed a synergistic effect of the BTK inhibitor, ibrutinib, with platinum drugs at low concentrations. Applying local and external DLBCL cohorts, we addressed the clinical relevance of the genes identified in the CRISPR screens. BTK was among the most frequently mutated genes with a frequency of 3–5%, and XPA and ERCC6 were also mutated, albeit at lower frequencies. Furthermore, 27–54% of diagnostic DLBCL samples had mutations in pathways that can sensitize cells to cisplatin. In conclusion, this study shows that XPA and ERCC6, in addition to BTK, are essential for the response to platinum-based drugs in DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2794e056513b2d174f98b17ca50338733fe72c8" target='_blank'>
              CRISPR-Cas9 Knockout Screens Identify DNA Damage Response Pathways and BTK as Essential for Cisplatin Response in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            I. I. Issa, Hanne Due, R. Brøndum, Vidthdyan Veeravakaran, Hulda Haraldsdóttir, Cathrine Sylvester, Asta Brogaard, Soniya Dhanjal, Bernhard Schmierer, K. Dybkær
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23cf92719cd49496400fba609df94fa13b009008" target='_blank'>
              Precision medicine in AML: overcoming resistance.
              </a>
            </td>
          <td>
            Samuel Urrutia, Koichi Takahashi
          </td>
          <td>2024-08-01</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fba17e91584024423ac0c6e3cc0e42afd48ee9aa" target='_blank'>
              Emerging advances in defining the molecular and therapeutic landscape of small-cell lung cancer.
              </a>
            </td>
          <td>
            Triparna Sen, N. Takahashi, Subhamoy Chakraborty, Naoko Takebe, Amin H. Nassar, Nagla A Karim, S. Puri, Abdul Rafeh Naqash
          </td>
          <td>2024-07-04</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0807e64db184649a4b0df3bd08502c4d34be0a53" target='_blank'>
              Programming tumor evolution with selection gene drives to proactively combat drug resistance.
              </a>
            </td>
          <td>
            Scott M. Leighow, Josh Reynolds, Ivan Sokirniy, Shun Yao, Zeyu Yang, Haider Inam, Dominik Wodarz, Marco Archetti, Justin Pritchard
          </td>
          <td>2024-07-04</td>
          <td>Nature biotechnology</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/589af1da53c29b65656184828eec590acfbf3b30" target='_blank'>
              Genetic and epigenetic instability as an underlying driver of progression and aggressive behavior in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            Timothy E Richardson, Jamie M. Walker, Dolores Hambardzumyan, Steven Brem, K. Hatanpaa, Mariano S Viapiano, Balagopal Pai, M. Umphlett, Oren J. Becher, M. Snuderl, S. McBrayer, Kalil G. Abdullah, N. Tsankova
          </td>
          <td>2024-07-16</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4297de6ca1f2c34995dc4df4f4fa137e566d405" target='_blank'>
              Potent therapeutic strategy in gastric cancer with microsatellite instability-high and/or deficient mismatch repair.
              </a>
            </td>
          <td>
            A. Ooki, H. Osumi, K. Yoshino, Kensei Yamaguchi
          </td>
          <td>2024-06-26</td>
          <td>Gastric cancer : official journal of the International Gastric Cancer Association and the Japanese Gastric Cancer Association</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="PURPOSE
In multiple myeloma (MM), therapy-induced clonal evolution is associated with treatment resistance and is one of the most important hindrances toward a cure for MM. To further understand the molecular mechanisms controlling the clonal evolution of MM, we applied single-cell RNA-sequencing (scRNA-seq) to paired diagnostic and post-treatment bone marrow (BM) samples.


EXPERIMENTAL DESIGN
scRNA-seq was performed on 38 BM samples from patients with monoclonal gammopathy of undetermined significance (MGUS) (n = 1), MM patients at diagnosis (n = 19), MM post-treatment (n = 17), and one healthy donor. The single-cell transcriptome data of malignant plasma cells and the surrounding immune microenvironment were analyzed.


RESULTS
Profiling by scRNA-seq data revealed three primary trajectories of transcriptional evolution after treatment: clonal elimination in patients with undetectable minimal residual disease (MRD-), as well as clonal stabilization and clonal selection in detectable MRD (MRD+) patients. We noted a metabolic shift towards fatty acid oxidation in cycling-resistant plasma cells (PCs), while selective PCs favored the NF-κB pathway. Intriguingly, when comparing the genetic and transcriptional dynamics, we found a significant correlation between genetic and non-genetic factors in driving the clonal evolution. Furthermore, we identified variations in cellular interactions between malignant plasma cells and the tumor microenvironment (TME). Selective PCs showed the most robust cellular interactions with the TME.


CONCLUSIONS
These data suggest that MM cells could rapidly adapt to induction treatment through transcriptional adaptation, metabolic adaptation, and specialized immune evasion. Targeting therapy-induced resistance mechanisms may help to avert refractory disease in multiple myeloma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4967fd77a9456d257d33b503e573eb358b554d" target='_blank'>
              Identification of therapy-induced clonal evolution and resistance pathways in minimal residual clones in multiple myeloma through single-cell sequencing.
              </a>
            </td>
          <td>
            Jian Cui, Xiaoyun Li, S. Deng, C. Du, H. Fan, W. Yan, Jingyu Xu, Xiaoqing Li, Tengteng Yu, Shuaishuai Zhang, R. Lv, W. Sui, Mu Hao, Xin Du, Yan Xu, S. Yi, D. Zou, Tao Cheng, Lugui Qiu, Xin Gao, Gang An
          </td>
          <td>2024-06-20</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Tumors often show an initial response to chemotherapy, but then develop resistance, leading to relapse and poor prognosis. We hypothesized that a genomic comparison of mutations in pre-treated versus treatment-naive tumors would serve to identify genes that confer resistance. A challenge in such an analysis is that therapy alters mutation burdens and signatures, confounding association studies and complicating identifying causal, selected mutations. We developed DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes. Crucially, DiffInvex draws on a mutation rate baseline that accounts for these shifts in neutral mutagenesis during cancer evolution. We applied DiffInvex to 9,953 cancer whole-genomes from 29 cancer types from 8 studies, containing both WGS of treatment-naive tumors and tumors pre-treated by various drugs, identifying genes where point mutations are under conditional positive or negative selection for a certain chemotherapeutic, suggesting resistance mechanisms occurring via point mutation. DiffInvex confirmed well-known chemoresistance-driver mutations in EGFR, ESR1, KIT and AR genes as being under conditional positive selection, with additional cancer types identified for EGFR and KIT. Additionally, DiffInvex identified 11 genes with treatment-associated selection for different classes of therapeutics. In most cases, these genes were common cancer genes including PIK3CA, APC, MAP2K4 and MAP3K1. This suggests that tumor resistance to therapy via mutation often occurs via selective advantages conferred by known driver genes, rather than via mutations in specialized resistance genes. Various gene-chemotherapy associations were further supported in tests for functional impact of mutations, again implemented in a conditional selection setting, as well as replicating in independent panel or exome sequencing data. In addition to nominating drug resistance genes that could be targeted by future therapeutics, DiffInvex can also be applied to diverse analysis in cancer evolution, such as comparing normal and tumoral tissues, or analyzing subclonal evolution, identifying changes in selection over time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b71e5df0edb673b45bd36f564fa0da7eae40ed" target='_blank'>
              The DiffInvex evolutionary model for conditional somatic selection identifies chemotherapy resistance genes in 10,000 cancer genomes
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2024-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8428e89731398de2122358b2387bfd15c652d8d" target='_blank'>
              Homopolymer switches mediate adaptive mutability in mismatch repair-deficient colorectal cancer
              </a>
            </td>
          <td>
            H. Kayhanian, William Cross, Suzanne E M van der Horst, Panagiotis Barmpoutis, E. Lakatos, G. Caravagna, L. Zapata, Arne van Hoeck, Sjors Middelkamp, Kevin Litchfield, Christopher Steele, W. Waddingham, Dominic Patel, Salvatore Milite, Chen Jin, A. Baker, Daniel C Alexander, Khurum Khan, Daniel Hochhauser, Marco Novelli, Benjamin Werner, R. van Boxtel, Joris H. Hageman, Julian R Buissant des Amorie, Josep Linares, M. Ligtenberg, I. Nagtegaal, M. Laclé, Leon M. G. Moons, L. Brosens, N. Pillay, A. Sottoriva, Trevor A Graham, M. Rodriguez-Justo, K. Shiu, Hugo J G Snippert, Marnix Jansen
          </td>
          <td>2024-07-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2239c71d70f51fc31d667365569c847abafe0d7a" target='_blank'>
              Novel drug resistance mechanisms and drug targets in BRAF-mutated peritoneal metastasis from colorectal cancer
              </a>
            </td>
          <td>
            Christin Lund-Andersen, A. Torgunrud, Chakravarthi Kanduri, V. Dagenborg, I. S. Frøysnes, Mette M. Larsen, Ben Davidson, Stein G Larsen, K. Flatmark
          </td>
          <td>2024-07-09</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution.
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Breast cancer (BCa) is a major global health challenge, characterized by chromosomal instability (CIN) and subsequent acquisition of extensive somatic copy number alterations (CNAs). CNAs including amplifications and deletions, significantly influence intra-tumor heterogeneity and the tumor microenvironment (TME). Among these, the loss of chromosome 13q14.2 emerges as a considerable factor in BCa pathogenesis and treatment responses. We provide evidence that this genomic alteration is under positive selective pressure, correlating with poorer patient survival. Furthermore, through multi-omic and in vitro analyses, we uncover a dual role of 13q14.2 loss: it confers a survival advantage to tumor cells and modulate the cell cycle and pro-apoptotic pathways in cancer cells, affecting macrophages population in the TME, while paradoxically increasing tumor susceptibility to BCL2 inhibitors. These findings suggest that targeting 13q14.2 as a biomarker in BCa could enhance the efficacy of existing treatments and offer a new avenue for improving clinical outcomes in BCa. Figure 1. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e388fe8f21125bbefce3cee2961f295d519ae77" target='_blank'>
              Loss of chromosome cytoband 13q14.2 orchestrates breast cancer pathogenesis and drug response
              </a>
            </td>
          <td>
            Parastoo Shahrouzi, Youness Azimzade, Wioletta Brankiewicz, Sugandha Bhatia, David Kunke, Derek Richard, Xavier Tekpli, Vessela N Kristensen, Pascal H.G. Duijf
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>103</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca139f6422b464298938108db348c2845dba664c" target='_blank'>
              Valine aminoacyl-tRNA synthetase promotes therapy resistance in melanoma
              </a>
            </td>
          <td>
            Najla El-Hachem, Marine Leclercq, Miguel Susaeta Ruiz, Raphael Vanleyssem, K. Shostak, Pierré-René Körner, Coralie Capron, Lorena Martin-Morales, Patrick Roncarati, Arnaud Lavergne, Arnaud Blomme, Silvia Turchetto, Eric Goffin, Palaniraja Thandapani, Ivan Tarassov, L. Nguyen, Bernard Pirotte, A. Chariot, J. Marine, Michael Herfs, F. Rapino, R. Agami, Pierre Close
          </td>
          <td>2024-06-07</td>
          <td>Nature Cell Biology</td>
          <td>1</td>
          <td>64</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Background Neoadjuvant chemoradiotherapy (nCRT) is the standard treatment for locally advanced rectal cancer, but only 20–40% of patients completely respond to this treatment. Methods To define the molecular features that are associated with response to nCRT, we generated and collected genomic and transcriptomic data from 712 cancers prior to treatment from our own data and from publicly available data. Results We found that patients with a complete response have decreased risk of both local recurrence and future metastasis. We identified multiple differences in DNA mutations and transcripts between complete and incomplete responders. Complete responder tumors have a higher tumor mutation burden and more significant co-occurring mutations than the incomplete responder tumors. In addition, mutations in DNA repair genes (across multiple mechanisms of repair) were enriched in complete responders and they also had lower expression of these genes indicating that defective DNA repair is associated with complete response to nCRT. Using logistic regression, we identified three significant predictors of complete response: tumor size, mutations within specific network genes, and the existence of three or more specific co-occurrent mutations. In incompletely responder tumors, abnormal cell-cell interaction and increased cancer associated fibroblasts were associated with recurrence. Additionally, gene expression analysis identified a subset of immune hot tumors with worse outcomes and upregulated of immune checkpoint proteins. Conclusions Overall, our study provides a comprehensive understanding of the molecular features associated with response to nCRT and the molecular differences in non-responder tumors that later reoccur. This knowledge may provide critical insight for the development of precision therapy for rectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b58c04acad0e1548042e7825a154a041443bca6e" target='_blank'>
              Molecular differentiation between complete and incomplete responders to neoadjuvant therapy in rectal cancer
              </a>
            </td>
          <td>
            Zechen Chong, Fengyuan Huang, M. McLeod, Regina K. Irwin, Mary Smithson, Zongliang Yue, Min Gao, Karin M. Hardiman
          </td>
          <td>2024-07-01</td>
          <td>Research Square</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Our research aimed to identify new therapeutic targets for Lung adenocarcinoma (LUAD), a major subtype of non-small cell lung cancer known for its low 5-year survival rate of 22%. By employing a comprehensive methodological approach, we analyzed bulk RNA sequencing data from 513 LUAD and 59 non-tumorous tissues, identifying 2,688 differentially expressed genes. Using Mendelian randomization (MR), we identified 74 genes with strong evidence for a causal effect on risk of LUAD. Survival analysis on these genes revealed significant differences in survival rates for 13 of them. Our pathway enrichment analysis highlighted their roles in immune response and cell communication, deepening our understanding. We also utilized single-cell RNA sequencing (scRNA-seq) to uncover cell type-specific gene expression patterns within LUAD, emphasizing the tumor microenvironment’s heterogeneity. Pseudotime analysis further assisted in assessing the heterogeneity of tumor cell populations. Additionally, protein-protein interaction (PPI) network analysis was conducted to evaluate the potential druggability of these identified genes. The culmination of our efforts led to the identification of five genes (tier 1) with the most compelling evidence, including SECISBP2L, PRCD, SMAD9, C2orf91, and HSD17B13, and eight genes (tier 2) with convincing evidence for their potential as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e174b3e000455b97e2cc57b71e1d6819743b30f" target='_blank'>
              Identifying potential therapeutic targets in lung adenocarcinoma: a multi-omics approach integrating bulk and single-cell RNA sequencing with Mendelian randomization
              </a>
            </td>
          <td>
            Youpeng Chen, Enzhong Li, Zhenglin Chang, Tingting Zhang, Zhenfeng Song, Haojie Wu, Zhangkai J. Cheng, Baoqing Sun
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Tumor genomes often harbor a complex spectrum of single nucleotide mutations, small indels, and large chromosome rearrangements that can perturb coding and non-coding regions of the genome in ways that remain poorly understood. The mutational processes responsible for the genesis of these events are also not static; instead, they continue to operate throughout disease evolution and further diversify the genetic and phenotypic landscape of cancer cells. Mounting evidence suggests that tumor genotype can be an important determinant of disease progression and therapy response, such as by modulating the sensitivity or resistance of cancer cells to mutant-specific drugs. These types of observations have motivated efforts to treat cancers with genome-informed therapies, highlighting the need to understand how different genetic variants precisely affect gene function and overall tumor phenotypes. Variant-function maps that provide a mechanistic understanding of the biology driven by cancer-associated mutations are needed to design these types of treatment strategies. Precision genome editing technologies like base and prime editing are uniquely suited to tackle this problem. Nevertheless, deploying these methods for systematic variant-function studies and disease modeling in vivo has not been straightforward due to lack of robust and scalable platforms capable of assessing editing efficiency and precision, particularly at endogenous loci. With this goal in mind, we previously developed and applied high-throughput base editing ‘sensor’ approaches that link endogenous genome editing outcomes with synthetic DNA-based readouts and cellular fitness measurements. Using these approaches, we showed that several uncharacterized mutant p53 alleles drive cancer cell proliferation and in vivo tumor development. Building upon this work, we recently developed new prime editing guide RNA design tools and sensor-based approaches that similarly couple quantitative editing outcomes to cellular fitness, allowing us to significantly expand the breadth of cancer-associated mutations that can be interrogated using these precision genome editing technologies. In this talk, I will describe ongoing work using base and prime editing sensor libraries to probe the biological impact of thousands of functionally-distinct genetic variants in diverse types of protein-coding genes and families to learn how these influence various cancer phenotypes. I will also discuss how the implementation of these modular technologies will allow researchers to functionally disentangle the impact of endogenous and exogenous mutational processes on functional selection and tumor evolution with close to single base pair resolution. This generalizable precision genome editing framework will facilitate the functional interrogation of genetic variants across diverse biological contexts, providing much needed insight into cancer variant-function relationships that could be leveraged to develop more precise cancer treatment paradigms, including synthetic lethal therapies that exploit tumor genotype.
 Citation Format: Francisco J. Sánchez-Rivera, Samuel I. Gould, Alexandra N. Wuest, Kexin Dong, Grace A. Johnson, Alvin Hsu, Varun K. Narendra, Ondine Atwa, Stuart S. Levine, David R. Liu. Functional studies of genetic variation using precision genome editing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA020.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31918f227a5efe4ffc9748fec8993c778a740c9" target='_blank'>
              Abstract IA020: Functional studies of genetic variation using precision genome editing
              </a>
            </td>
          <td>
            Francisco J. Sánchez-Rivera, Samuel I. Gould, Alexandra N. Wuest, Kexin Dong, Grace A. Johnson, Alvin Hsu, Varun K. Narendra, Ondine Atwa, Stuart S. Levine, David R. Liu
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Objective To identify nivolumab resistance-related genes in patients with head and neck squamous cell carcinoma (HNSCC) using single-cell and bulk RNA-sequencing data. Methods The single-cell and bulk RNA-sequencing data downloaded from the Gene Expression Omnibus database were analyzed to screen out differentially expressed genes (DEGs) between the nivolumab resistant and nivolumab sensitive patients using R software. The Least Absolute Shrinkage Selection Operator (LASSO) regression and Recursive Feature Elimination (RFE) algorithm were performed to identify key genes associated with nivolumab resistance. Functional enrichment of DEGs was analyzed with Gene Ontology and Kyoto Encyclopedia of Genes and Genomes analyses. The relationships of key genes with immune cell infiltration, differentation trajectory, dynamic gene expression profiles, and ligand-receptor interaction were explored. Results We found 83 DEGs. They were mainly enriched in T-cell differentiation, PD-1 and PD-L1 checkpoint pathways, and T-cell receptor pathways. In six key genes identified using machine learning algorithms, only PPP1R14A gene was differentially expressed between the nivolumab resistant and nivolumab sensitive groups both before and after immunotherapy (P < 0.05). The high PPP1R14A gene expression group had lower immune score (P < 0.01), higher expression of immunosuppressive factors (such as PDCD1, CTLA4, and PDCD1LG2) (r > 0, P < 0.05), lower differentiation of infiltrated immune cells (P < 0.05), and a higher degree of interaction between HLA and CD4 (P < 0.05). Conclusions PPP1R14A gene is closely associated with resistance to nivolumab in HNSCC patients. Therefore, PPP1R14A may be a target to ameliorate nivolumab resistance of HNSCC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e5e0e948e87319aaf4421e185ddbf3b11ef5c14" target='_blank'>
              PPP1R14A is Associated with Immunotherapy Resistance in Head and Neck Squamous Cell Carcinoma Identified by Single-Cell and Bulk RNA-Sequencing.
              </a>
            </td>
          <td>
            Junjie Ma, Lei Zhang, Jin Lu, Hao-Xuan Zhang
          </td>
          <td>2024-06-18</td>
          <td>Chinese medical sciences journal = Chung-kuo i hsueh k'o hsueh tsa chih</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recombinant adeno-associated virus (rAAV) is a widely used viral vector for gene therapy. Despite its clinical efficacy, the manufacturing of rAAV faces challenges in productivity and quality, leading to limited availability. To address the growing demand, next-generation process development should be informed by a mechanistic understanding of the cellular response to rAAV. In this study, we performed transcriptomic analysis of 5 cell lines with variable capacities for rAAV production. Using an intersectional approach, we assessed the transcriptional response to rAAV production and compared transcriptional profiles between high and baseline producers to identify possible targets for enhancing production. Modulation of cell cycle and nucleosome components suggested a reduction of proliferative capacity and a shift toward DNA replication to support rAAV production. During rAAV production, we observed upregulation of several core functions including transcription, stress response, and Golgi and endoplasmic reticulum organization. Conversely, inhibitors of DNA-binding proteins and mitochondrial components were consistently downregulated during rAAV production. We next performed a drug connectivity analysis of these results and identified 5 classes of drugs predicted to enhance rAAV production. Validation studies confirmed the efficacy of HDAC and microtubule inhibitors. Our data uncover novel and previously identified pathways that may enhance rAAV productivity, potentially enabling a path to engineer improved processes and cell lines for higher yields and better quality rAAV production. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b270da94a458df57bb4725ecb5dd57f8a71f23d" target='_blank'>
              Transcriptomics-informed pharmacology identifies epigenetic and cell cycle regulators as enhancers of AAV production
              </a>
            </td>
          <td>
            Joshua Tworig, Francis Grafton, Markus Hörer, Christopher A. Reid, M. Mandegar
          </td>
          <td>2024-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c9abec0157ccbe7ffcea6b2ec6d1e2342588925" target='_blank'>
              Sustained inactivation of the Polycomb PRC1 complex induces DNA repair defects and genomic instability in epigenetic tumors
              </a>
            </td>
          <td>
            C. Rawal, V. Loubiere, N. Butova, Juliette Gracia, V. Parreno, C. Merigliano, Anne-Marie Martinez, G. Cavalli, I. Chiolo
          </td>
          <td>2024-06-18</td>
          <td>Histochemistry and Cell Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Dr. S. Shruthi, Prof. Dr. K. Bhasker Shenoy. Cisplatin (CDDP) is the first metal‐based drug and has been a popular drug in the treatment of various types of cancer for decades. Cancer cell death is promoted by this platinum‐based drug by generating DNA adducts and activating apoptotic signaling pathways. The development of drug resistance is the foremost clinical concern that compromises its efficacy. The mechanism of CDDP resistance encompasses changes in drug influx and efflux, drug accumulation reduction, enhanced drug detoxification, and repair which may affect its therapeutic value. Researchers have paid significant attention to overcoming CDDP resistance, which provides an opportunity to enhance clinical outcomes. Combinational therapy, use of CDDP analogs, phytochemicals, nanoparticle‐mediated drug delivery, gene targeting and regulation of drug induced epigenetic changes are the specific regimens to overcome CDDP resistance and treatment burden. In this review, we summarize the mechanisms involved in CDDP resistance and strategies considered to overcome these hurdles concerning preclinical and clinical studies. This may provide insights into the further development of novel approaches to uproot CDDP resistance, to ascertain a safer and more effective cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ccb45b66ff19feee321d314dfb60f5bf30f15361" target='_blank'>
              Cisplatin Resistance in Cancer Therapy: Causes and Overcoming Strategies
              </a>
            </td>
          <td>
            S. Shruthi, K. Bhasker Shenoy
          </td>
          <td>2024-06-26</td>
          <td>ChemistrySelect</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The genetic integrity of pluripotent stem cells (PSC) is integral to their applications in research and therapy, but it is compromised by frequent development of copy number variations. Little is known about the basis of the variable genomic integrity among different PSC lines. Here we identify aneuploidies using RNA-seq and proteomics data from a panel of mouse embryonic stem cell (mESC) lines derived from 170 Diversity Outbred mice. We identified 62 lines with detectable aneuploid subpopulations and a subset of originally XX lines that lost one Chromosome X (XO). Strikingly, a much lower proportion of XX lines were aneuploid, compared to XY or XO lines. Two single-cell RNA-seq data sets demonstrated that aneuploid XY DO mESC also show lower Chromosome X gene expression and that XY mESC accumulate higher aneuploid proportions in culture than isogenic XX lines. Possible mechanisms for this protective effect of X chromosome dosage include our discovery that the lines with two active X Chromosomes have a higher expression of 2-cell-like state genes, and differential expression of X-linked tumor suppressor genes associated with DNA damage response. Highlights First genetic analysis of predisposition to aneuploidy in pluripotent stem cell cultures Chromosomal regions duplicated in aneuploid mouse embryonic stem cells are syntenic with regions overrepresented in human pluripotent stem cell lines bearing recurrent genetic abnormalities X-Chromosome dosage strongly influences susceptibility to aneuploidy in mouse embryonic stem cells and to a lesser degree in human pluripotent stem cells XX mouse embryonic stem cell lines show a higher proportion of cells in 2 cell-like state and higher expression of tumor suppressor genes associated with DNA damage response">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f853ae6e95a31cb9ad3712a21b0838a6601fe33" target='_blank'>
              Chromosome X Dosage Modulates Development of Aneuploidy in Genetically Diverse Mouse Embryonic Stem Cells
              </a>
            </td>
          <td>
            Alexander Stanton, Selcan Aydin, Daniel A. Skelly, Dylan Stavish, Kim Leonhard, Seth M. Taapken, Erik M. McIntire, Matthew Pankratz, A. Czechanski, T. Ludwig, Ted Choi, Steve P. Gygi, Ivana Barbaric, Steven C. Munger, L. Reinholdt, Martin F. Pera
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Single nucleotide variants (SNVs) contribute to cancer by altering the coding and the non-coding regions of the genome. Connecting SNVs to transcriptomic and epigenetic changes at the single-cell level remains challenging. To enable studies of rare cell populations harboring specific point mutations, we developed STAR-FACS, Specific-To-Allele PCR-FACS, to sort cells based on genomic allele alterations. We show that STAR-FACS can separate cells based on TERT promoter mutation status and is compatible with bulk and single-cell transcriptomic and epigenetic profiling. We demonstrate that glioblastoma cell lines derived from the same tumor but harboring distinct TERT promoter SNVs have different transcriptional programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f2ef09f49e055964d96a3768d156e4cf968931" target='_blank'>
              Cell sorting based on single nucleotide variation enables characterization of mutation-dependent transcriptome and chromatin states
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, Oszkar Szentirmai, M. Janiszewska
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Replication repair deficiency (RRD) is a pan-cancer mechanism characterized by abnormalities in the DNA mismatch repair (MMR) system due to pathogenic variants in the PMS2, MSH6, MSH2 or MLH1 genes, and/ or in the polymerase-proofreading genes, POLE and POLD1. RRD predisposition syndromes [constitutional MMR deficiency (CMMRD), Lynch, polymerase-proofreading associated polyposis (PPAP)] share overlapping phenotypic and biological characteristics. Moreover, cancers stemming from germline defects of one mechanism can acquire somatic defects in another, leading to complete RRD. Here we describe the recent advances in the diagnostics, surveillance, and clinical management for children with RRD syndromes. For patients with CMMRD, new data combining clinical insights and cancer genomics have revealed genotype-phenotype associations, helped in the development of novel functional assays, diagnostic guidelines, and surveillance recommendations. Recognition of non-gastrointestinal/ genitourinary malignancies, particularly aggressive brain tumors, in select children with Lynch and PPAP syndromes harbouring an RRD biology have led to new management considerations. Additionally, universal hypermutation and microsatellite instability have allowed immunotherapy to be a paradigm shift in the treatment of RRD cancers independent of their germline etiology. These advances have also stimulated a need for expert recommendations regarding genetic counselling for these patients and their families. Future collaborative work will focus on newer technologies such as quantitative measurement of circulating tumor DNA and functional genomics to tailor surveillance and clinical care, improving immune surveillance, develop prevention strategies, and deliver these novel discoveries to resource-limited settings to maximize benefits for patients globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f07f0c1be296ec56b40417a06c4f28b0fe8a9c" target='_blank'>
              Clinical updates and surveillance recommendations for DNA replication-repair deficiency syndromes in children and young adults.
              </a>
            </td>
          <td>
            Anirban Das, S. MacFarland, Julia Meade, Jordan R Hansford, Kami Wolfe Schneider, Roland P Kuiper, M. Jongmans, Harry Lesmana, Kris Ann P Schultz, Kim E Nichols, C. Durno, Kristin Zelley, Christopher C Porter, Lisa J States, S. Ben-Shachar, Sharon A Savage, J. Kalish, M. Walsh, Hamish S. Scott, Sharon E. Plon, U. Tabori
          </td>
          <td>2024-06-11</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Lung cancer patients treated with targeted therapies frequently respond well but invariably relapse due to the development of drug resistance. Drug resistance is in part mediated by a subset of cancer cells termed “drug-tolerant persisters” (DTPs), which enter a dormant, slow-cycling state that enables them to survive drug exposure. DTPs also exhibit stem cell-like characteristics, broad epigenetic reprogramming, altered metabolism, and a mutagenic phenotype mediated by adaptive mutability. While several studies have characterised the transcriptional changes that lead to the altered phenotypes exhibited in DTPs, these studies have focused predominantly on protein coding changes. As long non-coding RNAs (lncRNAs) are also implicated in the phenotypes altered in DTPs, it is likely that they play a role in the biology of drug tolerance. In this review, we outline how lncRNAs may contribute to the key characteristics of DTPs, their potential roles in tolerance to targeted therapies, and the emergence of genetic resistance in lung adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83141f0410de3ec7e33ac04641da48080cc097e5" target='_blank'>
              The Potential Links between lncRNAs and Drug Tolerance in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            William J. H. Davis, Catherine J. Drummond, Sarah D. Diermeier, Glen Reid
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a genetically and clinically diverse hematological cancer affecting middle-aged and elderly individuals. Novel targeted therapy options are needed for patients who relapse following initial responses or who are intrinsically resistant to current treatments. There is a growing body of investigation currently underway on MDM2 inhibitors in clinical trials, reflecting the increasing interest in including these drugs in cancer treatment regimens. One of the developed compounds, idasanutlin (RG7388), has shown promise in early-stage clinical trials. It is a second-generation MDM2–p53-binding antagonist with enhanced potency, selectivity, and bioavailability. In addition to the TP53 status, which is an important determinant of the response, we have shown in our previous studies that the SF3B1 mutational status is also an independent predictive biomarker of the ex vivo CLL patient sample treatment response to RG7388. The objective of this study was to identify novel biomarkers associated with resistance to RG7388. Gene set enrichment analysis of differentially expressed genes (DEGs) between RG7388-sensitive and -resistant CLL samples showed that the increased p53 activity led to upregulation of pro-apoptosis pathway genes while DNA damage response pathway genes were additionally upregulated in resistant samples. Furthermore, differential expression of certain genes was detected, which could serve as the backbone for novel combination treatment approaches. This research provides preclinical data to guide the exploration of drug combination strategies with MDM2 inhibitors, leading to future clinical trials and associated biomarkers that may improve outcomes for CLL patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e336986b17d0b66d36dfa25a57ef235e819da3" target='_blank'>
              RNA Sequencing Reveals Candidate Genes and Pathways Associated with Resistance to MDM2 Antagonist Idasanutlin in TP53 Wild-Type Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            E. Aptullahoglu, S. Nakjang, J. Wallis, Helen Marr, Scott Marshall, E. Willmore, John Lunec
          </td>
          <td>2024-06-22</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="BACKGROUND
Metastatic colorectal cancer (mCRC) is increasingly characterized by myriad genomic alterations beyond the well-known factors such as RAS, BRAF, and microsatellite instability (MSI). Novel genomic changes, including ERBB2 amplifications, mutations, and gene fusions, are now recognized as potential targets for precision therapy. This study aims to explore the genomic landscape of a Chinese cohort with mCRC to identify potentially targetable genetic alterations for personalized treatment strategies.


METHODS
A total of 500 mCRC patients in China were enrolled, based on which genomic profiling was performed using capture-based targeted sequencing across a panel of 520 genes on tumor tissues to identify prevalent genomic alterations. The mutations were analyzed by optimized proprietary algorithms. MSI and mismatch repair deficiency status were analyzed using the read-count-distribution approach. Besides, the overall survival (OS) related to these molecular changes was estimated.


RESULTS
The cohort's genomic profiling revealed TP53 mutations in 78%, APC in 60%, and KRAS in 47% of the patients. MSI-High status was confirmed in 5.8% of cases via a next-generation sequencing (NGS)-based algorithm. ERBB2/HER2 amplifications were found in 12% (60/500) of patients, with potential therapeutic implications for those without concurrent KRAS mutations. A subset of patients (1.2%; 6/500) showed fusions and DNA damage response (DDR) gene mutations (except TP53) that could be targeted therapeutically. The KRAS (G12C) variant was detected in 14 patients (2.8%), and 61 (12.2%) had a BRAF V600E mutation. Notably, survival analysis showed no significant differences in OS between KRAS mutant loci and NRAS mutations (p = 0.436). However, BRAF V600E mutations were associated with a poorer prognosis than BRAF wild-type and non-V600E mutations (16.3 months vs. 29.5 and 31.1 months, respectively; p < 0.001).


CONCLUSIONS
This study validates the feasibility of using NGS to detect prognostic and therapeutically actionable genetic variants in Chinese mCRC patients, contributing to understanding the genomic variation within this population and highlighting the potential for personalized medicine in managing mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d75b3be37f6d3fb50ba2d4fcbf63055e7d0a5a8b" target='_blank'>
              Genomic Alterations and Clinical Characterization in Chinese Patients with Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            Xu-hui Zhang, Jie-Qiong Zhou, Qing Wei, Jian Li, Ting Xu, Chun-mei Bai, Jian-Feng Zhou, Xi-cheng Wang
          </td>
          <td>2024-07-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc88953f3695ac0c90c5077d3442247bd93ef70" target='_blank'>
              Mixed responses to targeted therapy driven by chromosomal instability through p53 dysfunction and genome doubling
              </a>
            </td>
          <td>
            S. Hobor, M. Al Bakir, C. Hiley, Marcin Skrzypski, A. Frankell, Bjorn Bakker, T. Watkins, A. Markovets, J. Dry, Andrew P. Brown, Jasper van der Aart, H. van den Bos, D. Spierings, D. Oukrif, Marco Novelli, T. Chakrabarti, Adam H. Rabinowitz, Laila Ait Hassou, Saskia Litière, D. Kerr, L. Tan, Gavin P. Kelly, David A Moore, M. Renshaw, Subramanian Venkatesan, William Hill, A. Huebner, C. Martínez-Ruiz, James R. M. Black, Wei Wu, Mihaela Angelova, N. Mcgranahan, Julian Downward, J. Chmielecki, Carl Barrett, K. Litchfield, S. Chew, Collin M Blakely, Elza C de Bruin, F. Foijer, Karen H. Vousden, T. Bivona, , Jason Lester, A. Bajaj, A. Nakas, Azmina Sodha-Ramdeen, M. Tufail, M. Scotland, R. Boyles, S. Rathinam, C. Wilson, Domenic Marrone, S. Dulloo, D. Fennell, Gurdeep Matharu, J. Shaw, E. Boleti, Heather Cheyne, Mohammed Khalil, S. Richardson, Tracey Cruickshank, Gillian Price, K. M. Kerr, Sarah Benafif, Jack French, Kayleigh Gilbert, B. Naidu, Akshay J. Patel, A. Osman, Carol Enstone, G. Langman, Helen Shackleford, M. Djearaman, S. Kadiri, Gary Middleton, Angela E. Leek, Jack Davies Hodgkinson, Nicola Totton, A. Montero, E. Smith, E. Fontaine, F. Granato, A. Paiva-Correia, Juliette Novasio, K. Rammohan, L. Joseph, P. Bishop, Rajesh Shah, Stuart Moss, Vijay Joshi, Philip A J Crosbie, Katherine D. Brown, Mathew Carter, Anshuman Chaturvedi, Pedro Oliveira, Colin R. Lindsay, Fiona H. Blackhall, Matthew G. Krebs, Yvonne Summers, A. Clipson, J. Tugwood, Alastair Kerr, D. Rothwell, C. Dive, H. Aerts, R. Schwarz, Tom L. Kaufmann, G. Wilson, R. Rosenthal, P. Van Loo, Nicolai J. Birkbak, Zoltán Szállási, J. Kisistók, M. Sokač, R. Salgado, M. Dióssy, J. Demeulemeester, Abigail Bunkum, Angela Dwornik, Alastair Magness, Andrew J. Rowan, Angeliki Karamani, A. Toncheva, B. Chain, C. Castignani, Chris Bailey, C. Abbosh, C. Puttick, C. Weeden, Claudia Lee, Corentin Richard, C. Naceur-Lombardelli, D. Pearce, D. Karagianni, D. Biswas, D. Levi, E. Larose Cadieux, E. Lim, Emma C Colliver, E. Nye, F. Gálvez-Cancino, F. Gimeno-Valiente, G. Kassiotis, Georgia Stavrou, Gerasimos Mastrokalos, Helen L Lowe, Ignacio Matos, I. Noorani, J. Goldman, J. Reading, J. Rane, J. Nicod, John A. Hartley, K. Peggs, Katey S. S. Enfield, Kayalvizhi Selvaraju, K. Thol, Kevin W. Ng, Kezhong Chen, K. Dijkstra, Kristiana Grigoriadis, Krupa Thakkar, L. Ensell, Mansi Shah, Maria Litovchenko, M. Jamal-Hanjani, Mariana Werner Sunderland, M. Huska, M. Hill, M. Dietzen, Michelle Leung, M. Escudero, M. Tanić, Monica Sivakumar, O. Chervova, Olivia Lucas, O. Pich, Othman Al-Sawaf, Paulina Prymas, Philip S Hobson, Piotr Pawlik, R. Stone, R. Bentham, Roberto Vendramin, S. Saghafinia, Samuel Gamble, S. Veeriah, S. Ung, Sergio A Quezada, Sharon P. Vanloo, S. Hessey, S. Ward, Sian Harries, S. Boeing, Stephan Beck, S. Bola, Takahiro Karasaki, Tamara Denner, T. Marafioti, Thomas P. Jones, V. Spanswick, Vittorio Barbè, Wei-Ting Lu, Wing Kin Liu, Yin Wu, Yutaka Naito, Zoe Ramsden, C. Veiga, Gary Royle, C. Collins-Fekete, Francesco Fraioli, Paul Ashford, Martin D. Forster, Siow-Ming Lee, E. Borg, M. Falzon, D. Papadatos-Pastos, James Wilson, T. Ahmad, A. Procter, Asia Ahmed, Magali N. Taylor, Arjun Nair, David S. Lawrence, D. Patrini, Neal Navani, R. Thakrar, Sam M. Janes, Emilie Martinoni Hoogenboom, Fleur Monk, James W. Holding, Junaid Choudhary, Kunal Bhakhri, M. Scarci, Pat Gorman, Reena Khiroya, Robert C. M. Stephens, Y. Wong, Zoltán Káplár, S. Bandula, A. Hackshaw, A. Hacker, A. Sharp, Sean Smith, Harjot Kaur Dhanda, Camilla Pilotti, R. Leslie, Anca-Ioana Grapa, Hanyun Zhang, K. AbdulJabbar, Xiaoxi Pan, Yinyin Yuan, David Chuter, M. Mackenzie, S. Chee, A. Alzetani, Judith Cave, J. Richards, Eric Lim, P. De Sousa, Simon Jordan, Alex Rice, H. Raubenheimer, H. Bhayani, L. Ambrose, A. Devaraj, Hema Chavan, S. Begum, Silviu I. Buderi, Daniel Kaniu, Mpho Malima, S. Booth, Andrew G. Nicholson, Nádia Fernandes, Pratibha Shah, C. Proli, M. Hewish, S. Danson, M. Shackcloth, Lily Robinson, P. Russell, K. Blyth, A. Kidd, C. Dick, J. Le Quesne, Alan Kirk, Mohd. Asif, Rocco Bilancia, N. Kostoulas, Mathew Thomas, R. Hynds, N. Kanu, S. Zaccaria, E. Grönroos, C. Swanton
          </td>
          <td>2024-06-13</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>81</td>
        </tr>

        <tr id="Poor prognosis and drug resistance in glioblastoma (GBM) can result from cellular heterogeneity and treatment-induced shifts in phenotypic states of tumor cells, including dedifferentiation into glioma stem-like cells (GSCs). This rare tumorigenic cell subpopulation resists temozolomide, undergoes proneural-to-mesenchymal transition (PMT) to evade therapy, and drives recurrence. Through inference of transcriptional regulatory networks (TRNs) of patient-derived GSCs (PD-GSCs) at single-cell resolution, we demonstrate how the topology of transcription factor interaction networks drives distinct trajectories of cell-state transitions in PD-GSCs resistant or susceptible to cytotoxic drug treatment. By experimentally testing predictions based on TRN simulations, we show that drug treatment drives surviving PD-GSCs along a trajectory of intermediate states, exposing vulnerability to potentiated killing by siRNA or a second drug targeting treatment-induced transcriptional programs governing nongenetic cell plasticity. Our findings demonstrate an approach to uncover TRN topology and use it to rationally predict combinatorial treatments that disrupt acquired resistance in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ad1f578ef16e0b9da3d9d4afd31f2cea877a074" target='_blank'>
              Gene regulatory network topology governs resistance and treatment escape in glioma stem-like cells
              </a>
            </td>
          <td>
            James H Park, P. Hothi, Adrián López García de Lomana, Min Pan, Rachel Calder, S. Turkarslan, Wei-Ju Wu, Hwahyung Lee, Anoop P. Patel, C. Cobbs, Sui Huang, N. S. Baliga
          </td>
          <td>2024-06-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary Drug resistance is still an imminent issue for melanoma patients even after the use of BRAFi/MEKi combination therapy in clinic. Tumor heterogeneity and heterogeneity in treatment responses make it difficult to find consensus genes and pathways in resistance to therapy. This study used an objective method to analyze published gene expression data from pre-treated tumors and drug-resistant tumors, and identified possible targets and markers for resistant tumors, which centered on the PLXNC1 gene, which promotes a pro-inflammatory tumor microenvironment. Abstract Melanoma tumors exhibit a wide range of heterogeneity in genomics even with shared mutations in the MAPK pathway, including BRAF mutations. Consistently, adaptive drug resistance to BRAF inhibitors and/or BRAF plus MEK inhibitors also exhibits a wide range of heterogeneous responses, which poses an obstacle for discovering common genes and pathways that can be used in clinic for overcoming drug resistance. This study objectively analyzed two sets of previously published tumor genomics data comparing pre-treated melanoma tumors and BRAFi- and/or MEKi-resistant tumors. Heterogeneity in response to BRAFi and BRAFi/MEKi was evident because the pre-treated tumors and resistant tumors did not exhibit a tendency of clustering together. Differentially expressed gene (DEG) analysis revealed eight genes and two related enriched signature gene sets (matrisome and matrisome-associated signature gene sets) shared by both sets of data. The matrisome was closely related to the tumor microenvironment and immune response, and five out of the eight shared genes were also related to immune response. The PLXNC1 gene links the shared gene set and the enriched signature gene sets as it presented in all analysis results. As the PLXNC1 gene was up-regulated in the resistant tumors, we validated the up-regulation of this gene in a laboratory using vemurafenib-resistant cell lines. Given its role in promoting inflammation, this study suggests that resistant tumors exhibit an inflammatory tumor microenvironment. The involvement of the matrisome and the specific set of immune genes identified in this study may provide new opportunities for developing future therapeutic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fabe8a302d27af7d60acbb39b872cfcbb898453" target='_blank'>
              Unsupervised Analysis Reveals the Involvement of Key Immune Response Genes and the Matrisome in Resistance to BRAF and MEK Inhibitors in Melanoma
              </a>
            </td>
          <td>
            F. Liu-Smith, Jianjian Lin
          </td>
          <td>2024-06-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background: Patients with advanced-stage epithelial ovarian cancer (EOC) receive treatment with a poly-ADP ribose-polymerase (PARP) inhibitor (PARPi) as maintenance therapy after surgery and chemotherapy. Unfortunately, many patients experience disease progression because of acquired therapy resistance. This study aims to characterize epigenetic and genomic changes in cell-free DNA (cfDNA) associated with PARPi resistance. Materials and Methods: Blood was taken from 31 EOC patients receiving PARPi therapy before treatment and at disease progression during/after treatment. Resistance was defined as disease progression within 6 months after starting PARPi and was seen in fifteen patients, while sixteen patients responded for 6 to 42 months. Blood cfDNA was evaluated via Modified Fast Aneuploidy Screening Test-Sequencing System (mFast-SeqS to detect aneuploidy, via Methylated DNA Sequencing (MeD-seq) to find differentially methylated regions (DMRs), and via shallow whole-genome and -exome sequencing (shWGS, exome-seq) to define tumor fractions and mutational signatures. Results: Aneuploid cfDNA was undetectable pre-treatment but observed in six patients post-treatment, in five resistant and one responding patient. Post-treatment ichorCNA analyses demonstrated in shWGS and exome-seq higher median tumor fractions in resistant (7% and 9%) than in sensitive patients (7% and 5%). SigMiner analyses detected predominantly mutational signatures linked to mismatch repair and chemotherapy. DeSeq2 analyses of MeD-seq data revealed three methylation signatures and more tumor-specific DMRs in resistant than in responding patients in both pre- and post-treatment samples (274 vs. 30 DMRs, 190 vs. 57 DMRs, Χ2-test p < 0.001). Conclusion: Our genome-wide Next-Generation Sequencing (NGS) analyses in PARPi-resistant patients identified epigenetic differences in blood before treatment, whereas genomic alterations were more frequently observed after progression. The epigenetic differences at baseline are especially interesting for further exploration as putative predictive biomarkers for PARPi resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1b0806ed22cf2dfb5abff66ed3446b3d58e282" target='_blank'>
              Epigenetic and Genomic Hallmarks of PARP-Inhibitor Resistance in Ovarian Cancer Patients
              </a>
            </td>
          <td>
            Tugce Senturk Kirmizitas, Caroline van den Berg, R. Boers, J. Helmijr, S. Makrodimitris, H. H. Dag, Marijn Kerkhofs, C. Beaufort, J. Kraan, W. V. van Ijcken, J. Gribnau, Pakriti Garkhail, G. N. Boer, Eva-Maria Roes, Helene J van Beekhuizen, T. Gunel, S. Wilting, John W M Martens, M. Jansen, Ingrid Boere
          </td>
          <td>2024-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8868db406f4d297083b3336bdacf7e70f944e3" target='_blank'>
              Characterize molecular signatures and establish a prognostic signature of gastric cancer by integrating single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Zhiwei Wang, Zhiyan Weng, Luping Lin, Xianyi Wu, Wenju Liu, Yong Zhuang, J. Jian, Changhua Zhuo
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135134a62bac76cb46dae2f04090c849f2c01172" target='_blank'>
              Targeting chromosomal instability in patients with cancer.
              </a>
            </td>
          <td>
            Duaa H Al-Rawi, Emanuele Lettera, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-11</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Osteosarcoma is a rare primary bone tumor for which no significant therapeutic advancement has been made since the late 1980s despite ongoing efforts. Overall, the five-year survival rate remains about 65%, and is much lower in patients with tumors unresponsive to methotrexate, doxorubicin, and cisplatin therapy. Genetic studies have not revealed actionable drug targets, but our group, and others, have reported that epigenomic biomarkers, including regulatory RNAs, may be useful prognostic tools for osteosarcoma. We tested if microRNA (miRNA) transcriptional patterns mark the transition from a chemotherapy sensitive to resistant tumor phenotype. Small RNA sequencing was performed using 14 patient matched pre-chemotherapy biopsy and post-chemotherapy resection high-grade osteosarcoma frozen tumor samples. Independently, small RNA sequencing was performed using 14 patient matched biopsy and resection samples from untreated tumors. Separately, miRNA specific Illumina DASL arrays were used to assay an independent cohort of 65 pre-chemotherapy biopsy and 26 patient matched post-chemotherapy resection formalin fixed paraffin embedded (FFPE) tumor samples. mRNA specific Illumina DASL arrays were used to profile 37 pre-chemotherapy biopsy and five post-chemotherapy resection FFPE samples, all of which were also used for Illumina DASL miRNA profiling. The National Cancer Institute Therapeutically Applicable Research to Generate Effective Treatments dataset, including PCR based miRNA profiling and RNA-seq data for 86 and 93 pre-chemotherapy tumor samples, respectively, was also used. Paired differential expression testing revealed a profile of 17 miRNAs with significantly different transcriptional levels following chemotherapy. Genes targeted by the miRNAs were differentially expressed following chemotherapy, suggesting the miRNAs may regulate transcriptional networks. Finally, an in vitro pharmacogenomic screen using miRNAs and their target transcripts predicted response to a set of candidate small molecule therapeutics which potentially reverse the chemotherapy resistance phenotype and synergize with chemotherapy in otherwise treatment resistant tumors. Importantly, these novel therapeutic targets are distinct from targets identified by a similar pharmacogenomic analysis of previously published prognostic miRNA profiles from pre chemotherapy biopsy specimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/030c30436809f8513c1b8d90a7748f211e900709" target='_blank'>
              A dynamic microRNA profile that tracks a chemotherapy resistance phenotype in osteosarcoma. Implications for novel therapeutics
              </a>
            </td>
          <td>
            Christopher E Lietz, Erik T Newman, Andrew D Kelly, David H Xiang, Ziying Zhang, Nikhil Ramavenkat, Joshua J Bowers, Santiago A Lozano-Calderón, David H Ebb, Kevin A. Raskin, Gregory M Cote, Edwin Choy, G. P. Nielsen, Ioannis S. Vlachos, Benjamin Haibe-Kains, D. Spentzos
          </td>
          <td>2024-06-20</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Resistance to cancer therapy is driven by both cell-intrinsic and microenvironmental factors. Previous work has revealed that multiple resistant cell fates emerge in melanoma following treatment with targeted therapy and that, in vitro, these resistant fates are determined by the transcriptional state of individual cells prior to exposure to treatment. What remains unclear is whether these resistant fates are shared across different genetic backgrounds and how, if at all, these resistant fates interact with the tumor microenvironment. Through spatial transcriptomics and single-cell RNA sequencing, we uncovered distinct resistance programs in melanoma cells shaped by both intrinsic cellular states and the tumor microenvironment. Consensus non-negative matrix factorization revealed shared intrinsic resistance programs across different cell lines, highlighting the presence of universal and unique resistance pathways. In patient samples, we demonstrated that these resistance programs coexist within individual tumors and associate with diverse immune signatures, suggesting that the tumor microenvironment and distribution of resistant fates are closely connected. Single-cell resolution spatial transcriptomics in xenograft models revealed both intrinsically determined and extrinsically influenced resistant fates. Overall, this work demonstrates that each therapy resistant fate coexists with a distinct immune microenvironment in tumors and that, in vivo, tissue features, such as regions of necrosis, can influence which resistant fate is adopted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5979abc00b349ddf0161079fda7fee453bd0ce0a" target='_blank'>
              Spatial transcriptomics reveals influence of microenvironment on intrinsic fates in melanoma therapy resistance
              </a>
            </td>
          <td>
            Ryan H. Boe, Catherine G. Triandafillou, R. Lazcano, J. Wargo, Arjun Raj
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="Introduction Glioblastomas utilize malignant gene expression pathways to drive growth. Many of these gene pathways are not directly accessible with molecularly targeted pharmacological agents. Chromatin-modifying compounds can alter gene expression and target glioblastoma growth pathways. In this study, we utilize a systematic screen of chromatin-modifying compounds on a panel of patient-derived glioblastoma lines to identify promising compounds and their associated gene targets. Methods Five glioblastoma cell lines were subjected to a drug screen of 106 chromatin-modifying compounds representing 36 unique drug classes to determine the twelve most promising drug classes and the best candidate inhibitors in each class. These twelve drugs were then tested with a panel of twelve patient-derived gliomasphere lines to identify growth inhibition and corresponding gene expression patterns. Overlap analysis and weighted co-expression network analysis (WCGNA) were utilized to determine potential target genes and gene pathways. Results The initial drug screen identified twelve candidate pharmacologic agents for further testing. Drug sensitivity testing indicated an overall high degree of variability between gliomasphere lines. However, CPI203 was the most consistently effective compound, and the BET inhibitor class was the most consistently effective class of compounds across the gliomasphere panel. Correspondingly, most of the compounds tested had highly variable effects on gene expression between gliomasphere lines. CPI203 stood out as the only compound to induce a consistent effect on gene expression across different gliomasphere lines, specifically down-regulation of DNA-synthesis genes. Amongst the twelve tested cell lines, high expression of CDKN2A and CDKN2B distinguished more drug sensitive from more drug resistant lines. WCGNA identified two oncogenic gene modules (FBXO5 and MELK) that were effectively downregulated by CPI203 (FBXO5) and ML228 (FBXO5 and MELK). Conclusions The bromodomain inhibitor CPI203 induced relatively consistent effects on gene expression and growth across a variety of glioblastoma lines, specifically down-regulating genes associated with DNA replication. We propose that clinically effective BET inhibitors have the potential to induce consistent beneficial effects across a spectrum of glioblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1561fa27def3a17181508712df5ccd39c21a9f63" target='_blank'>
              CPI203, a BET inhibitor, down-regulates a consistent set of DNA synthesis genes across a wide array of glioblastoma lines
              </a>
            </td>
          <td>
            Matthew C. Garrett, Troy P Carnwath, Rebecca R Albano, Yonghua Zhuang, Catherine A Behrmann, Merissa Pemberton, Farah Barakat, Robert Lober, Mark Hoeprich, Anthony Paravati, Marilyn Reed, Hailey Spry, Daniel Woo, Eric O’Brien, Brett VanCauwenbergh, John Perentesis, Rani Nasser, Mario Medvedovic, D. Plas
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Venetoclax is the first example of personalized medicine for multiple myeloma (MM), with meaningful clinical activity as a monotherapy and in combination in myeloma patients harboring the t(11:14) translocation. However, despite the high response rates and prolonged PFS, a significant proportion of patients eventually relapse. Here, we aimed to study adaptive molecular responses after the acquisition of venetoclax resistance in sensitive t(11:14) MM cell models. We therefore generated single-cell venetoclax-resistant t(11:14) MM cell lines and investigated the mechanisms contributing to resistance as well as the cells' sensitivity to other treatments. Our data suggests that acquired resistance to venetoclax is characterized by reduced mitochondrial priming and changes in BCL-2 family proteins' expression in MM cells, conferring broad resistance to standard-of-care anti-myeloma drugs. However, our results show that the resistant cells are still sensitive to immunotherapeutic treatments, highlighting the need to consider appropriate sequencing of these treatments following venetoclax-based regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc29857b46abaf1a033e0c4b6c0e2ae376a185cb" target='_blank'>
              Venetoclax resistance leads to broad resistance to standard-of-care anti-MM agents, but not to immunotherapies.
              </a>
            </td>
          <td>
            Shuhui Deng, Sanika Derebail, Vera Joy Weiler, J. Fong Ng, Elena Maroto-Martin, Madhumouli Chatterjee, Giulia Giorgetti, C. Chakraborty, Poonam Kalhotra, Ting Du, Yao Yao, R. Prabhala, M. Shammas, Annamaria Gullà, Anil Aktas Samur, M. Samur, Lugui Qiu, Kenneth C. Anderson, M. Fulciniti, Nikhil C Munshi
          </td>
          <td>2024-06-11</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="
 The success of precision oncology depends on our ability to translate accumulating genomic data into actionable treatment options in a personalized manner. The first step requires the identification of a specific genomic signature, then matching this signature with the most effective therapy. Unfortunately, more often than not, we do not have a specific cancer drug that is effective for the mutation found in a specific tumor. We desperately need to identify new targets so that we can develop new drugs to deliver on the promise of precision cancer medicine. In addition, we need to develop a rational way to combine drugs that does not depend on trial and error. Combinatorial genetic screening CRISPR-Cas9 now allows us to identify synthetic lethal interactions in which simultaneous perturbation of two genes leads to cell death. However, multiplex gene editing to reveal these synergies between genes is a major challenge. For model systems like yeast, high-throughput methods and technologies have made it possible to create genetic networks with around 23 million double mutants (6000 genes x 6000 genes) and has resulted in the most detailed genetic interaction network to date consisting of ∼900,000 genetic interactions. This landscape of genetic interaction network of yeast has taken the scientific research community 15 years. In human cells, there are ∼20,000 genes and 400 million combinations, making this very complex network impossible to screen and test all these 400M combinations. To systematically interrogate such genetic interactions, we designed a dual CRISPR-Cas9 perturbation library targeting the top 1000 genes upregulated in KRAS mutant cancers (lung, pancreas and colon cancers). We performed these combinatorial double knockout screens on 100K (100x1000) unique gene pairs and identified 27 pairs whose co-disruption results in a loss of cellular fitness. We next validated those top synthetic lethal pairs of genes by performing secondary screens using CRISPR-Cas12a on more KRAS mutant and KRAS WT cell lines. Overall, our findings will provide insight into potential combinational targets in KRAS mutant tumors and will highlight the synthetic lethality effects that occurs among the novel targets, and other proliferation, metastasis, immune and metabolism modules.
 Citation Format: Rand Arafeh, Laura Chang, Lydia Sawyer, Helen Wang, James McFarland, Joshua Dempster, Peter DeWeirdt, John Doench, William C. Hahn. Combinatorial genetic screens to map synthetic lethal interactions and identify new cancer drug targets in KRAS mutant cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bcf5d6eda0b47ecdd6a82ff8b401bed98f9b3d57" target='_blank'>
              Abstract PR014: Combinatorial genetic screens to map synthetic lethal interactions and identify new cancer drug targets in KRAS mutant cancers
              </a>
            </td>
          <td>
            Rand Arafeh, Laura Chang, Lydia Sawyer, Helen Wang, James McFarland, Joshua Dempster, Peter C. DeWeirdt, John G Doench, William C Hahn
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca87f16129e49df59c97d9b9e83e846766aff915" target='_blank'>
              iPS cell generation-associated point mutations include many C > T substitutions via different cytosine modification mechanisms
              </a>
            </td>
          <td>
            R. Araki, T. Suga, Y. Hoki, Kaori Imadome, Misato Sunayama, Satoshi Kamimura, Mayumi Fujita, M. Abe
          </td>
          <td>2024-06-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="T cell acute lymphoblastic leukemia (T-ALL) is an aggressive leukemia predominantly affecting adolescents and young adults. T-ALL relapses are characterized by chemotherapy resistance, cellular heterogeneity and dismal outcome. To gain a deeper understanding of the cellular heterogeneity and mechanisms driving relapse, we conducted single-cell full-length RNA sequencing of 13 matched pediatric T-ALL patient-derived xenografts (PDX) samples obtained at initial diagnosis and relapse, generating the to date most comprehensive longitudinal single cell study in paired T-ALL samples, along with 5 non-relapsing PDX samples collected at initial diagnosis. This dataset identifies considerable transcriptomic diversity among individual T-ALL cell populations. Notably however, 11 of the 18 patients exhibit a small T-ALL cell subpopulation with a shared set of gene regulatory networks characterized by a common set of active regulons, expression patterns and splice isoforms. This profile involves the upregulation of a stem-like cell signature with enrichment of cell adhesion, and inhibition of the cell cycle and metabolic activity. Comprehensive investigations of these networks identify transcripts enforcing drug resistance through NF-κB expression and TGF-β signaling and of anti-apoptotic T cell signaling. Longitudinal monitoring of these stem-like cells demonstrates this subpopulation to account for only a small proportion of leukemia cells initially with a substantial expansion at relapse suggesting resistance to first line therapy. Chemotherapy resistance is functionally corroborated through in vitro and in vivo drug testing. We thus report the discovery of inherently treatment-resistant stem-like T-ALL cell populations underscoring the potential for devising future therapeutic strategies aimed at targeting stemness pathways in pediatric T-ALL. Key Points Single-cell full-length RNA sequencing reveals dormant, treatment-resistant cells expanding upon T cell acute lymphoblastic leukemia relapse. Stem-like T-ALL cells exhibit a shared gene regulatory network conferring chemotherapy resistance and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88dbfdc32eaada9aa463b5fbbffd8e6808956ec" target='_blank'>
              Role of Stem-Like Cells in Chemotherapy Resistance and Relapse in pediatric T Cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Julia Costea, Kerstin K. Rauwolf, Pietro Zafferani, T. Rausch, A. Mathioudaki, Judith B. Zaugg, M. Schrappe, C. Eckert, G. Escherich, Jean-Pierre Bourquin, B. Bornhauser, Andreas E. Kulozik, J. Korbel
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/152652d8b9732cb4ce54ecaade267ae36f26fe12" target='_blank'>
              Analysis of 10,478 cancer genomes identifies candidate driver genes and opportunities for precision oncology.
              </a>
            </td>
          <td>
            B. Kinnersley, Amit Sud, Andrew Everall, A. Cornish, D. Chubb, R. Culliford, Andreas J. Gruber, Adrian Lärkeryd, C. Mitsopoulos, David Wedge, R. Houlston
          </td>
          <td>2024-06-18</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>24</td>
        </tr>

        <tr id="Melanoma being one of the most common and deadliest skin cancers, has been rising since the past decade. Patients at advanced stages of the disease have very poor prognoses, as opposed to at the earlier stages. Nowadays the standard-of-care of advanced melanoma is resection followed by immune checkpoint inhibition based immunotherapy. However, a substantial proportion of patients either do not respond or develop resistances. This underscores a need for novel approaches and therapeutic targets as well as a better understanding of the mechanisms of melanoma pathogenesis. Long non-coding RNAs (lncRNAs) comprise a poorly characterized class of functional players and promising targets in promoting malignancy. Certain lncRNAs have been identified to play integral roles in melanoma progression and drug resistances, however systematic screens to uncover novel functional lncRNAs are scarce. Here, we profile differentially expressed lncRNAs in patient derived short-term metastatic cultures and BRAF-MEK-inhibition resistant cells. We conduct a focused growth-related CRISPR-inhibition screen of overexpressed lncRNAs, validate and functionally characterize lncRNA hits with respect to cellular growth, invasive capacities and apoptosis in vitro as well as the transcriptomic impact of our lead candidate the novel lncRNA XLOC_030781. In sum, we extend the current knowledge of ncRNAs and their potential relevance on melanoma. Significance Previously considered as transcriptional noise, lncRNAs have emerged as novel players in regulating many cellular aspects in health and disease including melanoma. However, the number and as well as the extent of functional significance of most lncRNAs remains elusive. We provide a comprehensive strategy to identify functionally relevant lncRNAs in melanoma by combining expression profiling with CRISPR-inhibition growths screens lowering the experimental effort. We also provide a larger resource of differentially expressed lncRNAs with potential implications in melanoma growth and invasion. Our results broaden the characterized of lncRNAs as potential targets for future therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768f16ebbf2c3ec424c154b1baf19ad0e931615" target='_blank'>
              CRISPR-inhibition screen for lncRNAs linked to melanoma growth and metastasis
              </a>
            </td>
          <td>
            Stavroula Petroulia, K. Hockemeyer, Shashank Tiwari, Pietro Berico, Sama Shamloo, Seyedeh Elnaz Banijamali, Eleazar Vega-Saenz de Miera, Yixiao Gong, Palaniraja Thandapani, Eric Wang, Michael Schulz, A. Tsirigos, Iman Osman, Ioannis D. Aifantis, Jochen Imig
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Efforts to cure BCR::ABL1 B cell acute lymphoblastic leukemia (Ph+ ALL) solely through inhibition of ABL1 kinase activity have thus far been insufficient despite the availability of tyrosine kinase inhibitors (TKIs) with broad activity against resistance mutants. The mechanisms that drive persistence within minimal residual disease (MRD) remain poorly understood and therefore untargeted. Utilizing 13 patient-derived xenograft (PDX) models and clinical trial specimens of Ph+ ALL, we examined how genetic and transcriptional features co-evolve to drive progression during prolonged TKI response. Our work reveals a landscape of cooperative mutational and transcriptional escape mechanisms that differ from those causing resistance to first generation TKIs. By analyzing MRD during remission, we show that the same resistance mutation can either increase or decrease cellular fitness depending on transcriptional state. We further demonstrate that directly targeting transcriptional state-associated vulnerabilities at MRD can overcome BCR::ABL1 independence, suggesting a new paradigm for rationally eradicating MRD prior to relapse. Finally, we illustrate how cell mass measurements of leukemia cells can be used to rapidly monitor dominant transcriptional features of Ph+ ALL to help rationally guide therapeutic selection from low-input samples. HIGHLIGHTS Relapse after remission on TKI can harbor mutations in ABL1, RAS, or neither Mutations and development-like cell state dictate fitness in residual disease Co-targeting cell state and ABL1 markedly reduces MRD Biophysical measurements provide an integrative, rapid measurement of cell state">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf85e0b581ae34776dbc829809704e7e0de66ff9" target='_blank'>
              Mutation and cell state compatibility is required and targetable in Ph+ acute lymphoblastic leukemia minimal residual disease
              </a>
            </td>
          <td>
            Peter S. Winter, Michelle L. Ramseier, Andrew W. Navia, Sachit D. Saksena, Haley Strouf, Nezha Senhaji, Alan DenAdel, Mahnoor Mirza, Hyun Hwan An, Laura L. Bilal, Peter Dennis, Catharine S. Leahy, Kay Shigemori, Jennyfer Galves-Reyes, Ye Zhang, Foster Powers, Nolawit Mulugeta, Alejandro J. Gupta, Nicholas L. Calistri, Alexandria van Scoyk, Kristen Jones, Huiyun Liu, Kristen Stevenson, Siyang Ren, M. Luskin, Charles P. Couturier, Ava P. Amini, Srivatsan Raghavan, Robert J. Kimmerling, Mark M. Stevens, Lorin Crawford, David M. Weinstock, S. Manalis, Alex K. Shalek, Mark A. Murakami
          </td>
          <td>2024-06-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="
 Lung cancer is the leading cause of cancer-related mortality. Detection of drug actionable genetic alterations has made the standard systemic therapy to a more personalized one including targeted therapy and immune checkpoint blockade. Despite the initial response, majority of patients develop resistance to treatment. Therefore, understanding the mechanisms of resistance as well as discovering novel cancer targets is necessary. Resistance to therapies can be associated with an existing clone of tumor cells with genetic alteration, or the emergence of cells with epigenetic changes overcoming the treatment-induced stress. Therefore, investigating the epigenetic aberrations that occur in tumor cells may reveal novel targetable markers that can be of great clinical significance. Methods: Organoid models established from non-small cell lung cancer (NSCLC) patient surgical resections and pleural effusion were dissociated into single cells and plated in matrigel-coated 384-well plate. A panel of 32 NSCLC organoid models were treated with a library of 41 epigenetic probes (1uM) and a DMSO control over a period of 8 days. Cell titre glo assay was performed to measure cell survival. Results: Arginine methyl transferase protein 5 (PRMT5) inhibited the cell growth of large proportion of the NSCLC organoid models. LLY-283 showed the most significant effect, with 50% of all organoid models (16/32) treated with LLY283 showing ≥50% reduction in cell viability. Further testing with multiple concentrations of several PRMT5 inhibitors revealed models with differential responses to PRMT5 inhibition. To investigate the differential gene expression changes induced by PRMT5 inhibition, we performed RNA-sequencing on three PRMT5 inhibitor sensitive and three resistant organoid models. Ongoing analyses aim at identifying differentially expressed genes in sensitive and resistant models in response to PRMT5 inhibition, and pathway enrichment analysis to determine the differential mechanisms of response to PRMT5 inhibition. Conclusion: Epigenetic drug screening on a large cohort of NSCLC organoid models reveals PRMT5 as potential novel therapeutic target in NSCLC. Further correlative analysis with transcriptomic changes induced by the PRMT5 inhibitors may reveal biomarkers of response.
 Citation Format: Khadija Jafarova, Quan Li, Nikolina Radulovich, Geoffrey Liu, Ming S Tsao. Screening non-small cell lung cancer patient-derived organoids with epigenetic probes reveals PRMT5 as an oncogenic target [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B026.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfe4ba74577b78f566cb31441d0d727f3eb4bf31" target='_blank'>
              Abstract B026: Screening non-small cell lung cancer patient-derived organoids with epigenetic probes reveals PRMT5 as an oncogenic target
              </a>
            </td>
          <td>
            Khadija Jafarova, Q. Li, N. Radulovich, Geoffrey Liu, Ming-Sound Tsao
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Tumor heterogeneity poses a significant challenge in combating treatment resistance. Despite Polo-like kinase 1 (PLK1) being universally overexpressed in cancers and contributing to chromosomal instability (CIN), direct PLK1 inhibition hasn’t yielded clinical progress. To address this, we utilized the synthetic dosage lethality (SDL) approach, targeting PLK1’s genetic interactions for selective killing of overexpressed tumor cells while mitigating heterogeneity-associated challenges. Employing computational methods, we conducted a genome-wide shRNA screen, identifying 105 SDL candidates. Further in vivo CRISPR screening in a breast cancer xenograft model and in vitro CRISPR analysis validated these candidates. Employing Perturb-seq revealed IGF2BP2/IMP2 as a key SDL hit eliminating PLK1-overexpressing cells. Suppression of IGF2BP2, genetically or pharmacologically, downregulated PLK1 and limited tumor growth. Our findings strongly propose targeting PLK1’s genetic interactions as a promising therapeutic approach, holding broad implications across multiple cancers where PLK1 is overexpressed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a751c2b790bc8a1c41eae9708c62d181b7514129" target='_blank'>
              Identification of targetable vulnerabilities of PLK1-overexpressing cancers by synthetic dosage lethality
              </a>
            </td>
          <td>
            Chelsea E. Cunningham, Frederick S. Vizeacoumar, Yue Zhang, Liliia Kyrylenko, Peng Gao, Vincent Maranda, He Dong, Jared DW Price, Ashtalakshmi Ganapathysamy, Rithik Hari, Connor Denomy, Simon Both, Konrad Wagner, Yingwen Wu, Faizaan Khan, Shayla R. Mosley, Angie Chen, Tetiana Katrii, Ben G. E. Zoller, Karthic Rajamanickam, Prachi Walke, Lihui Gong, Hardikkumar Patel, Mary Lazell-Wright, Alain Morejon Morales, Kalpana K Bhanumathy, Hussain Elhasasna, Renuka Dahiya, Omar Abuhussein, Anton Dmitriev, T. Freywald, Erika Prando Munhoz, Anand Krishnan, E. Ruppin, Joo Sang Lee, K. Rox, Behzad M Toosi, M. Koebel, Mary Kinloch, Laura Hopkins, Cheng-Han Lee, R. Datla, Sunil Yadav, Yuliang Wu, Kristi Baker, M. Empting, A. Kiemer, A. Freywald, Franco J. Vizeacoumar
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e606cab213cb81e6c3f7ec232b67c83ab5b219c" target='_blank'>
              Unveiling novel genetic variants in 370 challenging medically relevant genes using the long read sequencing data of 41 samples from 19 global populations.
              </a>
            </td>
          <td>
            Yanfeng Ji, Junfan Zhao, J. Gong, F. Sedlazeck, Shaohua Fan
          </td>
          <td>2024-07-07</td>
          <td>Molecular genetics and genomics : MGG</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Abstract Androgen receptor- (AR-) indifference is a mechanism of resistance to hormonal therapy in prostate cancer (PC). Here we demonstrate that ONECUT2 (OC2) activates resistance through multiple drivers associated with adenocarcinoma, stem-like and neuroendocrine (NE) variants. Direct OC2 gene targets include the glucocorticoid receptor (GR; NR3C1) and the NE splicing factor SRRM4, which are key drivers of lineage plasticity. Thus, OC2, despite its previously described NEPC driver function, can indirectly activate a portion of the AR cistrome through epigenetic activation of GR. Mechanisms by which OC2 regulates gene expression include promoter binding, enhancement of genome-wide chromatin accessibility, and super-enhancer reprogramming. Pharmacologic inhibition of OC2 suppresses lineage plasticity reprogramming induced by the AR signaling inhibitor enzalutamide. These results demonstrate that OC2 activation promotes a range of drug resistance mechanisms associated with treatment-emergent lineage variation in PC and support enhanced efforts to therapeutically target OC2 as a means of suppressing treatment-resistant disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afa57725f4c34a58c8bfb81365718e7515db8a48" target='_blank'>
              ONECUT2 acts as a lineage plasticity driver in adenocarcinoma as well as neuroendocrine variants of prostate cancer
              </a>
            </td>
          <td>
            Chen Qian, Qian Yang, Mirja Rotinen, Rongrong Huang, Hyoyoung Kim, Brad Gallent, Yiwu Yan, Radu M. Cadaneanu, Baohui Zhang, S. Kaochar, Stephen J. Freedland, Edwin M Posadas, Leigh Ellis, Dolores Di Vizio, C. Morrissey, Peter S. Nelson, Lauren Brady, Ramachandran Murali, Moray J. Campbell, Wei Yang, Beatrice S Knudsen, E. Mostaghel, Huihui Ye, I. Garraway, Sungyong You, Michael R Freeman
          </td>
          <td>2024-06-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Therapy-induced acquired resistance limits the clinical effectiveness of mutation-specific RAS inhibitors in colorectal cancer. It is unknown whether broad-spectrum active-state RAS inhibitors meet similar limitations. Here, we identify and categorize mechanisms of resistance to the broad-spectrum active-state RAS inhibitor RMC-7977 in colorectal cancer cell lines. We found that KRAS-mutant colorectal cancer cell lines are universally sensitive to RMC-7977, inhibiting the RAS-RAF-MEK-ERK axis, halting proliferation and in some cases inducing apoptosis. To monitor KRAS downstream effector pathway activity, we developed a compartment-specific dual-color ERK activity reporter. RMC-7977 treatment reduced reporter activity. However, long-term dose escalation with RMC-7977 revealed multiple patterns of reporter reactivation in emerging resistant cell populations that correlated with phosphorylation states of compartment-specific ERK targets. Cells sorted for high, low, or cytoplasmic reporter activity exhibited distinct patterns of genomic mutations, phospho-protein, and transcriptional activities. Notably, all resistant subpopulations showed dynamic ERK regulation in the presence of the RAS inhibitor, unlike the parental sensitive cell lines. High levels of RAS downstream activities were observed in cells characterized by a KRAS Y71H resistance mutation. In contrast, RAS inhibitor-resistant populations with low, or cytoplasmic ERK reporter reactivation displayed different genetic alterations, among them RAF1 S257L and S259P mutations. Colorectal cancer cells resistant to RMC-7977 and harboring the RAF1 mutation specifically exhibited synergistic sensitivity to concurrent RAS and RAF inhibition. Our findings endorse reporter-assisted screening together with single-cell analyses as a powerful approach for dissecting the complex landscape of therapy resistance. The strategy offers opportunities to develop clinically relevant combinatorial treatments to counteract emergence of resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58f2dfe945929a08a6260606b0df62f40daad49b" target='_blank'>
              Reporter-based screening identifies RAS-RAF mutations as drivers of resistance to active-state RAS inhibition in colorectal cancer
              </a>
            </td>
          <td>
            Oleksandra Aust, Eric Blanc, Mareen Lüthen, Viola Hollek, Rosario Astaburuaga-García, B. Klinger, Alexandra Trinks, D. Beule, Björn Papke, David Horst, Nils Blüthgen, Christine Sers, Channing Der, M. Morkel
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db5f5f80053a63fc1633772174de85b7485c0712" target='_blank'>
              Saturation genome editing of BAP1 functionally classifies somatic and germline variants
              </a>
            </td>
          <td>
            Andrew J Waters, Timothy Brendler-Spaeth, Danielle Smith, Victoria Offord, Hong Kee Tan, Yajie Zhao, Sofia Obolenski, Maartje Nielsen, R. van Doorn, Jo-Ellen Murphy, Prashant Gupta, C. F. Rowlands, H. Hanson, E. Delage, Mark Thomas, Elizabeth J Radford, S. Gerety, Clare Turnbull, J. R. Perry, M. Hurles, David J. Adams
          </td>
          <td>2024-07-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="ABSTRACT Aneuploidy, the presence of an aberrant number of chromosomes, has been associated with tumorigenesis for over a century. More recently, advances in karyotyping techniques have revealed its high prevalence in cancer: About 90% of solid tumors and 50–70% of hematopoietic cancers exhibit chromosome gains or losses. When analyzed at the level of specific chromosomes, there are strong patterns that are observed in cancer karyotypes both pan-cancer and for specific cancer types. These specific aneuploidy patterns correlate strongly with outcomes for tumor initiation, progression, metastasis formation, immune evasion and resistance to therapeutic treatment. Despite their prominence, understanding the basis underlying aneuploidy patterns in cancer has been challenging. Advances in genetic engineering and bioinformatic analyses now offer insights into the genetic determinants of aneuploidy pattern selection. Overall, there is substantial evidence that expression changes of particular genes can act as the positive selective forces for adaptation through aneuploidy. Recent findings suggest that multiple genes contribute to the selection of specific aneuploid chromosomes in cancer; however, further research is necessary to identify the most impactful driver genes. Determining the genetic basis and accompanying vulnerabilities of specific aneuploidy patterns is an essential step in selectively targeting these hallmarks of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0123dd290f56b3193001c1283aea1071e6f411ed" target='_blank'>
              Selection forces underlying aneuploidy patterns in cancer
              </a>
            </td>
          <td>
            Tamara C. Klockner, Christopher S. Campbell
          </td>
          <td>2024-06-24</td>
          <td>Molecular & Cellular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cell-cell interactions, which allow cells to communicate with each other through molecules in their microenvironment, are critical for the growth, health, and functions of cells. Previous studies show that drug-resistant cells can interact with drug-sensitive cells to elevate their drug resistance level, which is partially responsible for cancer recurrence. Studying protein targets and pathways involved in cell-cell communication provides essential information for fundamental cell biology studies and therapeutics of human diseases. In the current studies, we performed direct coculture and indirect coculture of drug-resistant and drug-sensitive cell lines, aiming to investigate intracellular proteins responsible for cell communication. Comparative studies were carried out using monoculture cells. Shotgun bottom-up proteomics results indicate that the P53 signaling pathway has a strong association with drug resistance mechanisms, and multiple TP53-related proteins were upregulated in both direct and indirect coculture systems. In addition, cell-cell communication pathways, including the phagosome and the HIF-signaling pathway, contribute to both direct and indirect coculture systems. Consequently, AK3 and H3-3A proteins were identified as potential targets for cell-cell interactions that are relevant to drug resistance mechanisms. We propose that the P53 signaling pathway, in which mitochondrial proteins play an important role, is responsible for inducing drug resistance through communication between drug-resistant and drug-sensitive cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cc9726c157261dbc51a02e3ad073aa8b2f6d16b" target='_blank'>
              Proteomics Analysis of Interactions between Drug-Resistant and Drug-Sensitive Cancer Cells: Comparative Studies of Monoculture and Coculture Cell Systems.
              </a>
            </td>
          <td>
            Zongkai Peng, Nagib Ahsan, Zhibo Yang
          </td>
          <td>2024-06-22</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer develops from abnormal cell growth in the body, causing significant mortalities every year. To date, potent therapeutic approaches have been developed to eradicate tumor cells, but intolerable toxicity and drug resistance can occur in treated patients, limiting the efficiency of existing treatment strategies. Therefore, searching for novel genes critical for cancer progression and therapeutic response is urgently needed for successful cancer therapy. Recent advances in bioinformatics and proteomic techniques have allowed the identification of a novel category of peptides encoded by non-canonical open reading frames (ncORFs) from historically non-coding genomic regions. Surprisingly, many ncORFs express functional microproteins that play a vital role in human cancers. In this review, we provide a comprehensive description of different ncORF types with coding capacity and technological methods in discovering ncORFs among human genomes. We also summarize the carcinogenic role of ncORFs such as pTINCR and HOXB-AS3 in regulating hallmarks of cancer, as well as the roles of ncORFs such as HOXB-AS3 and CIP2A-BP in cancer diagnosis and prognosis. We also discuss how ncORFs such as AKT-174aa and DDUP are involved in anti-cancer drug response and the underestimated potential of ncORFs as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d95f26adc077c4e04bfda2c2e30184b8f21906d" target='_blank'>
              Exploring the Dark Matter of Human Proteome: The Emerging Role of Non-Canonical Open Reading Frame (ncORF) in Cancer Diagnosis, Biology, and Therapy
              </a>
            </td>
          <td>
            Anni Ge, Curtis Chan, Xiaolong Yang
          </td>
          <td>2024-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Prostate cancer (PCa) poses significant challenges in treatment, particularly when it progresses to a metastatic, castrate-resistant state. Conventional therapies, including chemotherapy, radiotherapy, and hormonal treatments, often fail due to toxicities, off-target effects, and acquired resistance. This research perspective defines an alternative therapeutic strategy focusing on the metabolic vulnerabilities of PCa cells, specifically their reliance on non-essential amino acids such as cysteine. Using an engineered enzyme cyst(e)inase to deplete the cysteine/cystine can induce oxidative stress and DNA damage in cancer cells. This depletion elevates reactive oxygen species (ROS) levels, disrupts glutathione synthesis, and enhances DNA damage, leading to cancer cell death. The combinatorial use of cyst(e)inase with agents targeting antioxidant defenses, such as thioredoxins, further amplifies ROS accumulation and cytotoxicity in PCa cells. Overall, in this perspective provides a compressive overview of the previous work on manipulating amino acid metabolism and redox balance modulate the efficacy of DNA repair-targeted and immune checkpoint blockade therapies in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbf096fee4a52be93875a3cc7398db3ac71384ef" target='_blank'>
              Starving cancer cells to enhance DNA damage and immunotherapy response
              </a>
            </td>
          <td>
            Aashirwad Shahi, Dawit Kidane
          </td>
          <td>2024-06-20</td>
          <td>Oncotarget</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSC) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosomes 20q or 1q, both known for conferring in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings show higher loads of both CNVs and SNVs at later passages which are due to the cumulative acquisition of mutations over a longer time in culture and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs follow the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid cancer-associated mutations. This suggests that the recurrent chromosomal abnormalities are the potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f552b60bd87e180c4d6a5dd522428893e4b81b" target='_blank'>
              De novo cancer mutations frequently associate with recurrent chromosomal abnormalities during long-term human pluripotent stem cell culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huygebaert, M. Regin, Chi Mai Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 Lung cancer is the leading cause of cancer-related death in the United States, and worldwide, with the 5-year survival rate of non-small cell lung cancer at 24%. Treatment options are limited, and even in cancers that harbor druggable biomarkers, resistance is near-universal. Identifying novel treatment strategies is essential to improve the outcomes of patients with lung cancer and provide secondary strategies for those who acquire resistance to first-line therapeutics. Identifying targeted therapies based on the genotype or expression pattern of cancers is an ongoing endeavor, and has been fruitful in the discovery of identifying cancer vulnerabilities. In lung cancer, treatment of patients that harbor activating mutations in EGFR with specific EGFR inhibitors has improved progression-free survival rates and extended the lifespan of patients diagnosed with this aggressive cancer, demonstrating the utility of targeted therapies based genetic-based vulnerabilities, or synthetic lethal interactions. A rich source of synthetic lethal genetic interactions are paralogous genes. Paralogs result from gene duplication events through evolutionary history, and many paralogs share both sequence homology and functional redundancy. Co-targeting of some paralogous genes induces a fitness defect that is synergistic compared to individual paralog loss alone. Nearly 70% of genes in the human genome are considered paralogs, and until recent efforts from the Berger lab and others, this buffering effect has caused them to be under-represented in lethality studies. Identifying frequently-inactivated paralog genes in lung cancer will provide a platform to identify new vulnerabilities in lung cancer and other cancer types that share mutational backgrounds. To identify patterns of mutation and expression of paralog genes in lung cancer, we have accessed publicly available data sets for both cell line and patient tumor samples. Using TCGA and CCLE data sources, and a set of 1030 paralog pairs identified previously by the Berger lab as potential drug targets, we have identified paralog genes that harbor high mutational frequencies in lung cancer. Using these early candidates, we next asked whether genetic inactivation or transcriptional repression of one paralog influences the essentiality of its paired gene using CERES scores generated by the Cancer Dependency Map. Preliminary results from this work has identified several paralog pairs with distinct dependency profiles in cell lines for which expression of one paralog is lost, including paralog pairs MAGOH/MAGOHB and TLK1/TLK2. Further work to characterize these dependencies in relevant cell line models is ongoing. Additionally, the druggability of these targets will be tested using targeted degradation using the dTAG system. This work will uncover a new set of paralog synthetic lethality targets, and further validate the utility of targeting known synthetic lethal paralogs for the treatment of highly lethal lung cancer.
 Citation Format: Alice H. Berger, Siobhan O'Brien. Computational discovery of paralog dependencies drives target identification in lung cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/793b68ebb3cc975ae8e379cb0fd0799900ac2059" target='_blank'>
              Abstract PR004: Computational discovery of paralog dependencies drives target identification in lung cancer
              </a>
            </td>
          <td>
            Alice H. Berger, Siobhan O’Brien
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d65bf85005c8a5a2fb214830aeec58d6f7820d7" target='_blank'>
              Key genes and molecular mechanisms related to Paclitaxel Resistance
              </a>
            </td>
          <td>
            Adel I Alalawy
          </td>
          <td>2024-07-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite sustained clinical responses to immune-checkpoint blockade (ICB) therapies in non-small-cell lung cancer (NSCLC), the majority of patients derive no clinical benefits, and the cellular and molecular underpinnings of such resistance remain incompletely understood. To identify cell types that may influence immunotherapy responses, we first integrated newly generated and previously published single-cell RNA sequencing data from 110 treatment-naïve patients with NSCLC. Among tumor-resident cell types, we identified MMP1+ cancer-associated fibroblasts (CAFs), which were inversely correlated with the level of tumor-reactive T cells—a key determinant of response to ICB. Further single-cell analysis for newly collected 21 tumor samples from NSCLC patients treated with anti-PD-1/PD-L1 agents revealed that MMP1+ fibroblasts were indeed enriched in treatment-refractory patients, and this observation was also validated in an independent dataset of bulk RNA sequencing from 344 NSCLC patients treated with PD-L1 agents. Examination of the spatial architecture showed that MMP1+ fibroblasts were located at the tumor-stroma boundary, forming a single-cell layer that encircled the cancer cell aggregates, and we hence defined MMP1+ fibroblasts as tumor-stroma boundary (tsb)CAFs. Such tsbCAFs likely promote resistance to ICB by functioning as a physical barrier that prevents tumor-reactive T cells from recognizing and killing cancer cells. Our study provides a new framework to identify cellular underpinnings of resistance to ICB and suggests new strategies to overcome ICB resistance. Highlights ◊ Identification and characterization of MMP1+ fibroblasts in lung cancer. ◊ Single-cell meta-analysis reveals cell populations impeding the accumulation of tumor-reactive T cells. ◊ MMP1+ fibroblasts correlate with the low infiltration of tumor-reactive T cells and the resistance to anti-PD-1/PD-L1 treatment. ◊ MMP1+ fibroblasts appear to form a space barrier between malignant and T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba2d872e95a0ff58467c4e7529f81e90a6eb1891" target='_blank'>
              Defining cellular determinants of resistance to PD-1 pathway blockade in non-small-cell lung cancer
              </a>
            </td>
          <td>
            Baolin Liu, K. Feng, Kezhuo Yu, Ranran Gao, Xueda Hu, Boyu Qin, Jinliang Wang, Zhiqiang Xue, Weidong Han, Zemin Zhang
          </td>
          <td>2024-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/553f4560eb657fca2fe86385e460c71427df430c" target='_blank'>
              GLI1 confers resistance to PARP inhibitors by activating the DNA damage repair pathway.
              </a>
            </td>
          <td>
            Hiroshi Ikeuchi, Yusuke Matsuno, Rika Kusumoto-Matsuo, S. Kojima, Toshihide Ueno, Masachika Ikegami, Rina Kitada, Hitomi Sumiyoshi-Okuma, Yuki Kojima, K. Yonemori, Y. Yatabe, K. Takamochi, Kenji Suzuki, Ken‐ichi Yoshioka, H. Mano, S. Kohsaka
          </td>
          <td>2024-08-03</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Introduction: Chemotherapy resistance remains a significant challenge in the treatment of pancreatic adenocarcinoma (PDAC), particularly in relation to gemcitabine (Gem), a commonly used chemotherapeutic agent. MicroRNAs (miRNAs) are known to influence cancer progression and chemoresistance. This study investigates the association between miRNA expression profiles and gemcitabine resistance in PDAC. Methods: The miRNA expression profiles of a gemcitabine-sensitive (GS) PDAC cell line, MIA PaCa-2, and its gemcitabine-resistant (GR) progeny, MIA PaCa-2 GR, were analyzed. miRNA sequencing (miRNA-seq) was employed to identify miRNAs expressed in these cell lines. Differential expression analysis was performed, and Ingenuity Pathway Analysis (IPA) was utilized to elucidate the biological functions of the differentially expressed miRNAs. Results: A total of 1867 miRNAs were detected across both cell lines. Among these, 97 (5.2%) miRNAs showed significant differential expression between the GR and GS cell lines, with 65 (3.5%) miRNAs upregulated and 32 (1.7%) miRNAs downregulated in the GR line. The most notably altered miRNAs were implicated in key biological processes such as cell proliferation, migration, invasion, chemosensitization, alternative splicing, apoptosis, and angiogenesis. A subset of these miRNAs was further analyzed in patient samples to identify potential markers for recurrent tumors. Discussion: The differential miRNA expression profiles identified in this study highlight the complex regulatory roles of miRNAs in gemcitabine resistance in PDAC. These findings suggest potential targets for improving prognosis and tailoring treatment strategies in PDAC patients, particularly those showing resistance to gemcitabine. Future research should focus on validating these miRNAs as biomarkers for resistance and exploring their therapeutic potential in overcoming chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd403db221cf4ad4261322e87837b262e08166a3" target='_blank'>
              miRNA signatures underlie chemoresistance in the gemcitabine-resistant pancreatic ductal adenocarcinoma cell line MIA PaCa-2 GR
              </a>
            </td>
          <td>
            Ryan N Fuller, P. Vallejos, Janviere Kabagwira, Tiantian Liu, Charles Wang, Nathan R Wall
          </td>
          <td>2024-06-11</td>
          <td>Frontiers in Genetics</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="Liver cancer is a heterogeneous disease and is one of the leading causes of cancer deaths worldwide. Hepatocellular carcinoma, comprising approximately 90% of cases, presents a formidable challenge with a less than 20% 5-year survival rate despite recent treatment advancements. The impediments of drug resistance and off-target effects underscore the critical need for innovative and efficacious therapies. Harnessing the growing understanding of RNA function offers a promising avenue to address previously "undruggable" proteins, transcripts, and genes. Various RNAs demonstrate the potential to selectively act on these targets, expanding the scope of therapeutic interventions. With diverse regulatory roles in cancer pathways, RNAs emerge as valuable targets and tools for anticancer therapy development. This article provides an in-depth exploration of current RNA-based therapies, elucidates their mechanisms of action, and discusses their combinations with chemo-/immunotherapies in clinical trials for hepatocellular carcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16116ad053310e9b38070cc9f4f72620da429655" target='_blank'>
              RNA-based therapies in hepatocellular carcinoma: state of the art and clinical perspectives
              </a>
            </td>
          <td>
            Le Xing, Zhen-Kai Wang, Dongli Li, Jing Li, Ming Liu
          </td>
          <td>2024-06-20</td>
          <td>Hepatoma Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b2c5c7d1cda920e03f84144f36e08d5e8bdd9f" target='_blank'>
              Genomic barcoding for clonal diversity monitoring and control in cell-based complex antibody production
              </a>
            </td>
          <td>
            Niels Bauer, Christoph Oberist, Michaela Poth, Julian Stingele, Oliver Popp, Simon Ausländer
          </td>
          <td>2024-06-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The DNA modifications, 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), represent powerful epigenetic regulators of temporal and spatial gene expression. Yet, how the cooperation of these genome-wide, epigenetic marks determine unique transcriptional signatures across different brain cell populations is unclear. Here we applied Nanopore sequencing of native DNA to obtain a complete, genome-wide, single-base resolution atlas of 5mC and 5hmC modifications in neurons, astrocytes and microglia in the mouse cortex (99% genome coverage, 40 million CpG sites). In tandem with RNA sequencing, analysis of 5mC and 5hmC patterns across cell types reveals astrocytes drive uniquely high brain 5hmC levels and support two decades of research regarding methylation patterns, gene expression and alternative splicing, benchmarking this resource. As such, we provide the most comprehensive DNA methylation data in mouse brain as an interactive, online tool (NAM-Me, https://olsenlab.shinyapps.io/NAMME/) to serve as a resource dataset for those interested in the methylome landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60076a55629be60f22fe1afccd00156f5bc051e7" target='_blank'>
              Decoding the Epigenetic Landscape: Insights into 5mC and 5hmC Patterns in Mouse Cortical Cell Types
              </a>
            </td>
          <td>
            Xiaoran Wei, Jiangtao Li, Zuolin Cheng, Songtao Wei, Guoqiang Yu, Michelle L. Olsen
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The genome of a cell is continuously battered by a plethora of exogenous and endogenous processes that can lead to damaged DNA. Repair mechanisms correct this damage most of the time, but failure to do so leaves mutations. Mutations do not occur in random manner, but rather typically follow a more or less specific pattern due to known or imputed mutational processes. Mutational signature analysis is the process by which the predominant mutational process can be inferred for a cancer and can be used in several contexts to study both the genesis of cancer and its response to therapy. Recent pan-cancer genomic efforts such as "The Cancer Genome Atlas" have identified numerous mutational signatures that can be categorized into single base substitutions, doublet base substitutions, or small insertions/deletions. Understanding these mutational signatures as they occur in non-small lung cancer could improve efforts at prevention, predict treatment response to personalized treatments, and guide the development of therapies targeting tumor evolution. For non-small cell lung cancer, several mutational signatures have been identified that correlate with exposures such as tobacco smoking and radon and can also reflect endogenous processes such as aging, APOBEC activity, and loss of mismatch repair. Herein, we provide an overview of the current knowledge of mutational signatures in non-small lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37777d22dfc5a2bd99dc42f300aad4d0a01e922f" target='_blank'>
              Topography of mutational signatures in non-small cell lung cancer: emerging concepts, clinical applications, and limitations.
              </a>
            </td>
          <td>
            Ritujith Jayakrishnan, David J. Kwiatkowski, Michal G Rose, Amin H. Nassar
          </td>
          <td>2024-06-22</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/435232aea775847959662a12da4da5a9a69c73c7" target='_blank'>
              A whole-genome CRISPR screen identifies the spindle accessory checkpoint as a locus of nab-paclitaxel resistance in a pancreatic cancer cell line
              </a>
            </td>
          <td>
            Priya Mondal, George Alyateem, Allison V. Mitchell, Michael M. Gottesman
          </td>
          <td>2024-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Alterations in the FMS-like tyrosine kinase 3 (FLT3) gene are the most frequent driver mutations in acute myeloid leukaemia (AML), linked to a high risk of relapse in patients with internal tandem duplications (FLT3-ITD). Tyrosine kinase inhibitors (TKIs) targeting the FLT3 protein are approved for clinical use, yet resistance often emerges. This resistance is mainly seen following the acquisition of additional point mutations in the tyrosine kinase domain (TKD), resulting in a double mutant FLT3-ITD/TKD, which sustains cell signalling and survival despite the presence of FLT3 inhibitors. Here, we developed a FLT3-mutant AML model with adaptive resistance to type II TKIs, sorafenib, and quizartinib by in vitro drug selection. Through global multiomic profiling, we identified upregulation of proteins involved in reactive oxygen species (ROS) production, particularly NADPH-oxidases, driving cellular ’ROS-addiction’, with resistant cells relying on ROS for survival, and genome fidelity preserved by ATM-driven DNA repair. Transcriptomic analysis of adult and paediatric AML (pAML) patients identified high ATM expression as a biomarker for shorter median overall survival in both the de novo and relapsed settings. Inhibition of ATM with clinically relevant therapy WSD-0628 effectively killed TKI- and chemotherapy-resistant AML cells in vitro and significantly extended the survival of mice with sorafenib- and quizartinib-resistant FLT3-ITD AML in vivo. We propose a new treatment strategy to improve survival of patients who develop resistance to sorafenib and quizartinib, as well as relapsed and refractory pAML, exploiting resistance mechanisms to precision therapies and cell-intrinsic features of high-risk cases, highlighting a clinically relevant salvage strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0bcd4e64b15b98143cfc995f3b3d32e9c623b66" target='_blank'>
              Adaptive resistance to FLT3 inhibitors is potentiated by ROS-driven DNA repair signalling
              </a>
            </td>
          <td>
            Dilana E. Staudt, Zacary P. Germon, Abdul Mannan, Tabitha Mclachlan, Heather C Murray, Ryan J. Duchatel, Bryce C. Thomas, Tyrone S Beitaki, Holly P. McEwen, Mika L Persson, Leah Calvert, Izac J. Findlay, Evangeline R. Jackson, Nathan D. Smith, D. Skerrett‐Byrne, David Mossman, Brett Nixon, Geoffry De Iullis, Alicia M. Douglas, A. Enjeti, Jonathan R. Sillar, Janis Chamberlain, Frank Alvaro, Andrew H. Wei, Patrick Connerty, N. Verrills, Matthew D. Dun
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="CRISPR gene editing strategies are shaping cell therapies through precise and tunable control over gene expression. However, achieving reliable therapeutic effects with improved safety and efficacy requires informed target gene selection. This depends on a thorough understanding of the involvement of target genes in gene regulatory networks (GRNs) that regulate cell phenotype and function. Machine learning models have been previously used for GRN reconstruction using RNA- seq data, but current techniques are limited to single cell types and focus mainly on transcription factors. This restriction overlooks many potential CRISPR target genes, such as those encoding extracellular matrix components, growth factors, and signaling molecules, thus limiting the applicability of these models for CRISPR strategies. To address these limitations, we have developed CRISPR-GEM, a multi-layer perceptron (MLP)-based synthetic GRN constructed to accurately predict the downstream effects of CRISPR gene editing. First, input and output nodes are identified as differentially expressed genes between defined experimental and target cell/tissue types respectively. Then, MLP training learns regulatory relationships in a black-box approach allowing accurate prediction of output gene expression using only input gene expression. Finally, CRISPR-mimetic perturbations are made to each input gene individually and the resulting model predictions are compared to those for the target group to score and assess each input gene as a CRISPR candidate. The top scoring genes provided by CRISPR-GEM therefore best modulate experimental group GRNs to motivate transcriptomic shifts towards a target group phenotype. This machine learning model is the first of its kind for predicting optimal CRISPR target genes and serves as a powerful tool for enhanced CRISPR strategies across a range of cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/486423f8c90770082bb7f4dc4e4e7334da919173" target='_blank'>
              CRISPR-GEM: A Novel Machine Learning Model for CRISPR Genetic Target Discovery and Evaluation
              </a>
            </td>
          <td>
            Josh P. Graham, Yu Zhang, Lifang He, Tomas Gonzalez-Fernandez
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11be0db0863a56dda25d8c62273567d68174a817" target='_blank'>
              Therapy-induced secretion of spliceosomal components mediates pro-survival crosstalk between ovarian cancer cells
              </a>
            </td>
          <td>
            V. Shender, Ksenia S. Anufrieva, P. Shnaider, G. Arapidi, M. Pavlyukov, Olga M. Ivanova, I. K. Malyants, G. Stepanov, E. Zhuravlev, R. Ziganshin, I. Butenko, Olga Bukato, Ksenia M Klimina, V. Veselovsky, T.V. Grigorieva, Sergey Y Malanin, O. I. Aleshikova, A. Slonov, N. A. Babaeva, L. Ashrafyan, Elena Khomyakova, E. Evtushenko, M. Lukina, Zixiang Wang, A. S. Silantiev, A. Nushtaeva, Daria D. Kharlampieva, Vassili N. Lazarev, Arseniy I Lashkin, Lorine K Arzumanyan, I. Petrushanko, Alexander A Makarov, O. Lebedeva, A. Bogomazova, M. A. Lagarkova, V. M. Govorun
          </td>
          <td>2024-06-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33af20a7d0aac05ac83adef2c51ac9ffcee3995c" target='_blank'>
              Enhancer engagement sustains oncogenic transformation and progression of B-cell precursor acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            G. Corleone, C. Sorino, M. Caforio, S. Di Giovenale, F. de Nicola, F. Goeman, V. Bertaina, A. Pitisci, C. Cortile, Franco Locatelli, V. Folgiero, Maurizio Fanciulli
          </td>
          <td>2024-06-27</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 The goal of this study was to identify cancer types with synthetic vulnerability to RNA polymerase I (Pol I) inhibitors and to resolve mechanistic underpinnings of this vulnerability. Specific and selective Pol I inhibitors, pioneered by our group, block the transcription of rRNA, which is the rate-limiting step in ribosome biogenesis and protein synthesis. We conducted unbiased large-scale cancer cell line screens with the Pol I inhibitors and performed correlative analyses using DepMap and Achilles databases to identify cancer types and features that sensitize cancer cells to these inhibitors. The most sensitive lineages were colorectal, endometrial, esophageal, leukemia, myeloma, and ovarian. Microsatellite instable (MSI) cancer cell lines were highly sensitive and significantly more sensitive than microsatellite stable lines. Comparison of the cancer cell line responses to the Achilles gene essentiality database showed high correlation of the sensitivity to genetic knock-out of three Pol I subunits. This unbiased finding strongly supports identification of Pol I as the target. Correlative analyses for vulnerability identified two ribosomal proteins, RPL22 and RPL22L1, and MDM4, a p53 transcriptional repressor as the top candidates. Notably, RPL22K15Rfs hotspot mutation, the top-ranking genetic correlate, is present in up 70% MSI cancer cell lines and tumors. RPL22 and RPL22L1 are a synthetic lethal paralog pair. RPL22 is a negative regulator of RPL22L1 and consequently, haploinsufficiency or loss of RPL22 induces RPL22L1. RPL22L1 was the top-ranking expression correlate. Lastly, MDM4 was the top protein marker and is highly expressed in MSI cancers. These findings were validated using genetic and mechanistic analyses. We show that either genetic or chemical inhibition of the Pol I enzyme decreases the expression of RPL22L1 and MDM4 and alters their splicing. RPL22L1 is spliced to an alternative 3’ splice site in intron 2 leading to an early truncation. MDM4 exon 6 skipping converts the full-length p53 repressive form to a short NMD-targeted form. Mechanistically, by using GoldCLIP-seq, rMATS and splicing analyses, we show that RPL22 is a master regulator of splicing by binding to intronic sequences and splice sites of RPL22L1 and MDM4. Finally, we show the efficacy and tolerance of the Pol I inhibitors in xenograft and patient-derived MSI models expressing these markers. In summary, this synthetic vulnerability screen using new Pol I inhibitors uncovered a connection between Pol I transcription and cellular splicing programs and identified perturbation of this connection in MSI cancers. We conclude that RPL22 mutation could serve as tumor-agnostic biomarker for targeting cancers by Pol I inhibitors. Furthermore, this work identifies a new Pol I transcription-dependent ribosomal protein-governed pathway that converges on the regulation of p53.
 Citation Format: Marikki Laiho, Wenjun Fan, Hester Liu, Gregory C. Stachelek, Asma Begum, Catherine E. Davis, Tony Dorado, Pablo de Leon, Rajeshkumar N.V., James C. Barrow. Mechanisms of therapeutic vulnerability of mismatch repair defective cancers to RNA polymerase I inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cefc87042a8cb7740c0f2cbbcff23d295a38413" target='_blank'>
              Abstract PR003: Mechanisms of therapeutic vulnerability of mismatch repair defective cancers to RNA polymerase I inhibitors
              </a>
            </td>
          <td>
            M. Laiho, Wenjun Fan, Hester Liu, Gregory C. Stachelek, A. Begum, Catherine E. Davis, Tony Dorado, Pablo de León, N.V. Rajeshkumar, James C Barrow
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Linking disease phenotypes with molecular targets is key to the rational design of treatment interventions. Resistance to the chemotherapeutic cisplatin is one of the major factors limiting the clinical utility of this therapy, which is central to the treatment of a variety of solid malignancies. In this study, we couple the upregulation of a chemoresistant factor, the glutamate-cystine antiporter SLC7A11, with the addiction of cisplatin-resistant cancer cells to extracellular glutamine. In doing so, we thus provide a putative biomarker for this acquired metabolic dependency of chemoresistance. Subsequently, we evaluate various therapeutic strategies to selectively kill SLC7A11high cisplatin-resistant cancer cells, identifying cross-resistance to ferroptosis-inducing compounds and hypersensitivity to glutaminase inhibitor CB-839. We identify enzymatic depletion of extracellular glutamine using the long-standing anti-leukemic therapy asparaginase (ASNase), which possesses glutaminase activity, as a potential approach, and show this can be successfully combined with cisplatin in cell models. In summary, this study mechanistically links an acquired metabolic dependency of chemoresistant cancer cells with a putative biomarker and provides a potentially actionable strategy to target these drug resistant cells warranting further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c836fb51ec4a12788e118211188bb2a1ae2ab5" target='_blank'>
              Glutamine-addiction in cisplatin resistant cancer cells is mediated by SLC7A11 and can be targeted with asparaginase therapy
              </a>
            </td>
          <td>
            Jiantao Wang, Robert Strauss, Jiri Bartek, S. G. Rudd
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer chimeric, or fusion, transcripts are thought to most frequently appear due to chromosomal aberrations that combine moieties of unrelated normal genes. When being expressed, this results in chimeric RNAs having upstream and downstream parts relatively to the breakpoint position for the 5’- and 3’-fusion components, respectively. As many other types of cancer mutations, fusion genes can be of either driver or passenger type. The driver fusions may have pivotal roles in malignisation by regulating survival, growth, and proliferation of tumor cells, whereas the passenger fusions most likely have no specific function in cancer. The majority of research on fusion gene formation events is concentrated on identifying fusion proteins through chimeric transcripts. However, contemporary studies evidence that fusion events involving non-coding RNA (ncRNA) genes may also have strong oncogenic potential. In this review we highlight most frequent classes of ncRNAs fusions and summarize current understanding of their functional roles. In many cases, cancer ncRNA fusion can result in altered concentration of the non-coding RNA itself, or it can promote protein expression from the protein-coding fusion moiety. Differential splicing, in turn, can enrich the repertoire of cancer chimeric transcripts, e.g. as observed for the fusions of circular RNAs and long non-coding RNAs. These and other ncRNA fusions are being increasingly recognized as cancer biomarkers and even potential therapeutic targets. Finally, we discuss the use of ncRNA fusion genes in the context of cancer detection and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec4b199299758974fdcbbc48ab668c481680ff0" target='_blank'>
              Cancer fusion transcripts with human non-coding RNAs
              </a>
            </td>
          <td>
            Tharaa Mohammad, Marianna A. Zolotovskaia, Maria V. Suntsova, Anton Buzdin
          </td>
          <td>2024-06-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The fitness effects of new mutations determine key properties of evolutionary processes. Beneficial mutations drive evolution, yet selection is also shaped by the frequency of small-effect deleterious mutations, whose combined effect can burden otherwise adaptive lineages and alter evolutionary trajectories and outcomes in clonally evolving organisms such as viruses, microbes, and tumors. The small effect sizes of these important mutations have made accurate measurements of their rates difficult. In microbes, assessing the effect of mutations on growth can be especially instructive, as this complex phenotype is closely linked to fitness in clonally evolving organisms. Here, we perform high-throughput time-lapse microscopy on cells from mutation-accumulation strains to precisely infer the distribution of mutational effects on growth rate in the budding yeast, Saccharomyces cerevisiae. We show that mutational effects on growth rate are overwhelmingly negative, highly skewed towards very small effect sizes, and frequent enough to suggest that deleterious hitchhikers may impose a significant burden on evolving lineages. By using lines that accumulated mutations in either wild-type or slippage repair-defective backgrounds, we further disentangle the effects of 2 common types of mutations, single-nucleotide substitutions and simple sequence repeat indels, and show that they have distinct effects on yeast growth rate. Although the average effect of a simple sequence repeat mutation is very small (approximately 0.3%), many do alter growth rate, implying that this class of frequent mutations has an important evolutionary impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e2d4d1c86097c9432759848b7286f55d338b931" target='_blank'>
              Spontaneous single-nucleotide substitutions and microsatellite mutations have distinct distributions of fitness effects
              </a>
            </td>
          <td>
            Y. Plavskin, M. S. de Biase, Naomi Ziv, Libuše Janská, Yuan O. Zhu, David W. Hall, Roland F Schwarz, Daniel Tranchina, M. Siegal
          </td>
          <td>2024-07-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Introduction Field cancerization is suggested to arise from imbalanced differentiation in individual basal progenitor cells leading to clonal expansion of mutant cells that eventually replace the epithelium, although without evidence. Methods We performed deep sequencing analyses to characterize the genomic and transcriptomic landscapes of field change in two patients with synchronous aerodigestive tract tumors. Results Our data support the emergence of numerous genetic alterations in cancer-associated genes but refutes the hypothesis that founder mutation(s) underpin this phenomenon. Mutational signature analysis identified defective homologous recombination as a common underlying mutational process unique to synchronous tumors. Discussion Our analyses suggest a common etiologic factor defined by mutational signatures and/or transcriptomic convergence, which could provide a therapeutic opportunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae797c96e298d92f976b952e9d1ccd0f24f2a970" target='_blank'>
              Transcriptomic convergence despite genomic divergence drive field cancerization in synchronous squamous tumors
              </a>
            </td>
          <td>
            Qiu Xuan Tan, Nicholas B. Shannon, Weng Khong Lim, J. X. Teo, Daniel R. Y. Yap, Sze Min Lek, Joey W. S. Tan, Shih Jia, Josephine Hendrikson, Ying Liu, Gillian Ng, Clara Y. L. Chong, Wanyu Guo, Kelvin K. N. Koh, C. Ng, Vikneswari Rajasegaran, Jolene S.M. Wong, Chin Jin Seo, C. Ong, Tony K.H. Lim, Bin Tean Teh, Oi Lian Kon, C. Chia, Khee Chee, N. G. Iyer, C. Ong, M. Shammas, Sarah Mazzilli, Prajish Iyer
          </td>
          <td>2024-06-13</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671f339111b7b478e20a7b9554c095e50b64b22b" target='_blank'>
              Cellular and molecular events in colorectal cancer: biological mechanisms, cell death pathways, drug resistance and signalling network interactions
              </a>
            </td>
          <td>
            Lei Yan, Jia Shi, Jiazuo Zhu
          </td>
          <td>2024-07-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The intrinsic oncogenic mechanisms and properties of the tumor microenvironment (TME) have been extensively investigated. Primary features of the TME include metabolic reprogramming, hypoxia, chronic inflammation, and tumor immunosuppression. Previous studies suggest that senescence-associated secretory phenotypes that mediate intercellular information exchange play a role in the dynamic evolution of the TME. Specifically, hypoxic adaptation, metabolic dysregulation, and phenotypic shifts in immune cells regulated by cellular senescence synergistically contribute to the development of an immunosuppressive microenvironment and chronic inflammation, thereby promoting the progression of tumor events. This review provides a comprehensive summary of the processes by which cellular senescence regulates the dynamic evolution of the tumor-adapted TME, with focus on the complex mechanisms underlying the relationship between senescence and changes in the biological functions of tumor cells. The available findings suggest that components of the TME collectively contribute to the progression of tumor events. The potential applications and challenges of targeted cellular senescence-based and combination therapies in clinical settings are further discussed within the context of advancing cellular senescence-related research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f69d28c4893741d69a2e81f7df756fd68f976626" target='_blank'>
              Cellular senescence and metabolic reprogramming: Unraveling the intricate crosstalk in the immunosuppressive tumor microenvironment.
              </a>
            </td>
          <td>
            Fusheng Zhang, Junchen Guo, Shengmiao Yu, Youwei Zheng, Meiqi Duan, Liang Zhao, Yihan Wang, Zhi Yang, Xiaofeng Jiang
          </td>
          <td>2024-07-12</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41728da5eec57442f344cb37688d002e66c10d7e" target='_blank'>
              Tumor initiation and early tumorigenesis: molecular mechanisms and interventional targets
              </a>
            </td>
          <td>
            Shaosen Zhang, Xinyi Xiao, Yonglin Yi, Xinyu Wang, Lingxuan Zhu, Yanrong Shen, Dongxin Lin, Chen Wu
          </td>
          <td>2024-06-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Ovarian cancer is the most lethal gynecological malignancy and is often associated with both DNA repair deficiency and extensive metabolic reprogramming. While still emerging, the interplay between these pathways can affect ovarian cancer phenotypes, including therapeutic resistance to the DNA damaging agents that are standard-of-care for this tumor type. In this review, we will discuss what is currently known about cellular metabolic rewiring in ovarian cancer that may impact DNA damage and repair in addition to highlighting how specific DNA repair proteins also promote metabolic changes. We will also discuss relevant data from other cancers that could be used to inform ovarian cancer therapeutic strategies. Changes in the choice of DNA repair mechanism adopted by ovarian cancer are a major factor in promoting therapeutic resistance. Therefore, the impact of metabolic reprogramming on DNA repair mechanisms in ovarian cancer has major clinical implications for targeted combination therapies for the treatment of this devastating disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d7a4f6e6a58cd9429369d6c2946e615d6a364e5" target='_blank'>
              Interplay between altered metabolism and DNA damage and repair in ovarian cancer.
              </a>
            </td>
          <td>
            Apoorva Uboveja, K. Aird
          </td>
          <td>2024-06-14</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Simple Summary Cancer cells often become resistant to treatment, making it harder to fight the disease effectively. This resistance can happen due to genetic changes, proteins that pump out drugs, or the environment around the tumor. Crizotinib is a drug that helps overcome this resistance in certain lung cancers and lymphomas by targeting specific proteins. It can also break down a harmful protein found in some cancers. Researchers have found that combining crizotinib with other drugs can improve its effectiveness, especially for cancers with specific genetic mutations. Abstract Chemoresistance is a major obstacle in cancer treatment, often leading to disease progression and poor outcomes. It arises through various mechanisms such as genetic mutations, drug efflux pumps, enhanced DNA repair, and changes in the tumor microenvironment. These processes allow cancer cells to survive despite chemotherapy, underscoring the need for new strategies to overcome resistance and improve treatment efficacy. Crizotinib, a first-generation multi-target kinase inhibitor, is approved by the FDA for the treatment of ALK-positive or ROS1-positive non-small cell lung cancer (NSCLC), refractory inflammatory (ALK)-positive myofibroblastic tumors (IMTs) and relapsed/refractory ALK-positive anaplastic large cell lymphoma (ALCL). Crizotinib exists in two enantiomeric forms: (R)-crizotinib and its mirror image, (S)-crizotinib. It is assumed that the R-isomer is responsible for the carrying out various processes reviewed here The S-isomer, on the other hand, shows a strong inhibition of MTH1, an enzyme important for DNA repair mechanisms. Studies have shown that crizotinib is an effective multi-kinase inhibitor targeting various kinases such as c-Met, native/T315I Bcr/Abl, and JAK2. Its mechanism of action involves the competitive inhibition of ATP binding and allosteric inhibition, particularly at Bcr/Abl. Crizotinib showed synergistic effects when combined with the poly ADP ribose polymerase inhibitor (PARP), especially in ovarian cancer harboring BRCA gene mutations. In addition, crizotinib targets a critical vulnerability in many p53-mutated cancers. Unlike its wild-type counterpart, the p53 mutant promotes cancer cell survival. Crizotinib can cause the degradation of the p53 mutant, sensitizing these cancer cells to DNA-damaging substances and triggering apoptosis. Interestingly, other reports demonstrated that crizotinib exhibits anti-bacterial activity, targeting Gram-positive bacteria. Also, it is active against drug-resistant strains. In summary, crizotinib exerts anti-tumor effects through several mechanisms, including the inhibition of kinases and the restoration of drug sensitivity. The potential of crizotinib in combination therapies is emphasized, particularly in cancers with a high prevalence of the p53 mutant, such as triple-negative breast cancer (TNBC) and high-grade serous ovarian cancer (HGSOC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c55a5da7409fe84d8ab6b88da7dafb5e0032e27" target='_blank'>
              Overcoming Chemoresistance in Cancer: The Promise of Crizotinib
              </a>
            </td>
          <td>
            Sanaa Musa, Noor Amara, Adan Selawi, Junbiao Wang, Cristina Marchini, A. Agbarya, J. Mahajna
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Breast cancer is the second most diagnosed cancer, as well as the primary cause of cancer death in women worldwide. Of the different breast cancer subtypes, triple-negative breast cancer (TNBC) is particularly aggressive and is associated with poor prognosis. Black women are two to three times more likely to be diagnosed with TNBCs than white women. Recent experimental evidence suggests that basal-like TNBCs may derive from luminal cells which acquire basal characteristics through phenotypic plasticity, a newly recognized hallmark of cancer. Whether chemical exposures can promote phenotypic plasticity in breast cells is poorly understood. Methods To investigate further, we developed a high-content immunocytochemistry assay using normal human breast cells to test whether chemical exposures can impact luminal/basal plasticity by unbiased quantification of keratin 14 (KRT14), a basal-myoepithelial marker; keratin 8 (KRT8), a luminal-epithelial marker; and Hoechst 33342, a DNA marker. Six cell lines established from healthy tissue from donors to the Susan G. Komen Normal Tissue Bank were exposed for 48 hours to three different concentrations (0.1μM, 1μM, and 10μM) of eight ubiquitous chemicals (arsenic, BPA, BPS, cadmium, copper, DDE, lead, and PFNA), with documented exposure disparities in US Black women, in triplicate. Automated fluorescence image quantification was performed using Cell Profiler software, and a random-forest classifier was trained to classify individual cells as KRT8 positive, KRT14 positive, or hybrid (both KRT8 and KRT14 positive) using Cell Profiler Analyst. Results and discussion Results demonstrated significant concentration-dependent increases in hybrid populations in response to BPA, BPS, DDE, and PFNA. The increase in hybrid populations expressing both KRT14 and KRT8 is indicative of a phenotypically plastic progenitor-like population in line with known theories of carcinogenesis. Furthermore, BPA, BPS, DDE, and copper produced significant increases in cell proliferation, which could be indicative of a more malignant phenotype. These results further elucidate the relationship between chemical exposure and breast phenotypic plasticity and highlight potential environmental factors that may impact TNBC risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49e98af388cf82a8a70746f82327ab464041f396" target='_blank'>
              Investigating phenotypic plasticity due to toxicants with exposure disparities in primary human breast cells in vitro
              </a>
            </td>
          <td>
            Jade Schroeder, Katelyn M. Polemi, Anagha Tapaswi, Laurie K. Svoboda, Jonathan Z. Sexton, Justin A. Colacino
          </td>
          <td>2024-06-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer is a complex and heterogeneous disease, in which a number of genetic and epigenetic changes occur in tumor onset and progression. Recent studies indicate that changes at the RNA level are also involved in tumorigenesis, such as adenosine-to-inosine (A-to-I) RNA editing. Here, we systematically investigate transcriptome-wide A-to-I editing events in a large number of samples from Non-Hodgkin lymphomas (NHLs). Using a computational pipeline that determines significant differences in editing level between NHL and normal samples at known A-to-I editing sites, we identify a number of differentially edited editing sites between NHL subtypes and normal samples. Most of the differentially edited sites are located in non-coding regions, and many such sites show a strong correlation between gene expression level and editing efficiency, indicating that RNA editing might have direct consequences for the cancer cell’s aberrant gene regulation status in these cases. Moreover, we establish a strong link between RNA editing and NHL by demonstrating that NHL and normal samples and even NHL subtypes can be distinguished based on genome-wide RNA editing profiles alone. Our study establishes a strong link between RNA editing, cancer and aberrant gene regulation in NHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3709fa38f5f1119b1fcb4db3f2e4cd77244ecc08" target='_blank'>
              A-to-I Editing Is Subtype-Specific in Non-Hodgkin Lymphomas
              </a>
            </td>
          <td>
            Cai Chen, Ralf Bundschuh
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32d3f3f60bb04e4c09508175df4f6c54ba8a2e35" target='_blank'>
              Proteomic profiling reveals ACSS2 facilitating metabolic support in acute myeloid leukemia.
              </a>
            </td>
          <td>
            L. Mochmann, D. Treue, M. Bockmayr, Patricia Silva, Christin Zasada, G. Mastrobuoni, Șafak Bayram, M. Forbes, Philipp Jurmeister, Sven Liebig, Olga Blau, Konstanze Schleich, Bianca Splettstoesser, Thierry M Nordmann, E. K. von der Heide, Konstandina Isaakidis, Veronika Schulze, Caroline Busch, Hafsa Siddiq, C. Schlee, Svenja Hester, Lars Fransecky, Martin Neumann, Stefan Kempa, Frederick Klauschen, C. Baldus
          </td>
          <td>2024-06-08</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic regulatory mechanism that plays a critical role in a variety of cellular processes, including the regulation of cell fate during development, maintenance of cell identity, and genome stability. DNA methylation is tightly regulated by enzymatic reactions and its deregulation plays an important role in the development of cancer. Specific DNA methylation alterations have been found in pediatric solid tumors, providing new insights into the development of these tumors. In addition, DNA methylation profiles have greatly contributed to tune the diagnosis of pediatric solid tumors and to define subgroups of patients with different risks of progression, leading to the reduction in unwanted toxicity and the improvement of treatment efficacy. This review highlights the dysregulated DNA methylome in pediatric solid tumors and how this information provides promising targets for epigenetic therapies, particularly inhibitors of DNMT enzymes (DNMTis). Opportunities and limitations are considered, including the ability of DNMTis to induce viral mimicry and immune signaling by tumors. Besides intrinsic action against cancer cells, DNMTis have the potential to sensitize immune-cold tumors to immunotherapies and may represent a remarkable option to improve the treatment of challenging pediatric solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1eac857dfcc5a5e3b672ddb17d6097e48bc70f" target='_blank'>
              Targeting DNA Methylation Machinery in Pediatric Solid Tumors
              </a>
            </td>
          <td>
            C. Cristalli, Katia Scotlandi
          </td>
          <td>2024-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554fe13daafca3f151510eb83bbcfb8701b7284c" target='_blank'>
              Genome-scale CRISPR-Cas9 screening in stem cells: theories, applications and challenges
              </a>
            </td>
          <td>
            Heng Zhou, Peng Ye, W. Xiong, Xingxiang Duan, Shuili Jing, Yan He, Zhi Zeng, Yen Wei, Qingsong Ye
          </td>
          <td>2024-07-19</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 K-Ras inhibitor resistance in PDAC is associated with metabolic reprogramming towards a high OXPHOS state. Existing inhibitors of components of the electron transport chain have on-target toxicities that limit their use in the clinic, highlighting an opportunity to develop mechanistically orthogonal approaches. In this talk, I discuss our development of molecular glue degraders that target metabolic vulnerabilities in K-Ras inhibitor resistant PDAC.
 Citation Format: Fleur M. Ferguson. Overcoming the K-Ras inhibitor resistant cell state in PDAC with molecular glues [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94ebaacceb0116a4b24f94be81be9bcc2afa15e2" target='_blank'>
              Abstract IA002: Overcoming the K-Ras inhibitor resistant cell state in PDAC with molecular glues.
              </a>
            </td>
          <td>
            F. Ferguson
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is a highly prevalent and incurable form of cancer that arises from malignant plasma cells, with over 35,000 new cases diagnosed annually in the United States. While there are a growing number of approved therapies, MM remains incurable and nearly all patients will relapse and exhaust all available treatment options. Mechanisms for disease progression are unclear and in particular, little is known regarding the role of long non-coding RNAs (lncRNA) in mediating disease progression and response to treatment. In this study, we used transcriptome sequencing to compare newly diagnosed MM patients who had short progression- free survival (PFS) to standard first-line treatment (PFS < 24 months) to patients who had prolonged PFS (PFS > 24 months). We identified 157 differentially upregulated lncRNAs with short PFS and focused our efforts on characterizing the most upregulated lncRNA, LINC01432. We investigated LINC01432 overexpression and CRISPR/Cas9 knockdown in MM cell lines to show that LINC01432 overexpression significantly increases cell viability and reduces apoptosis, while knockdown significantly reduces viability and increases apoptosis, supporting the clinical relevance of this lncRNA. Next, we used individual-nucleotide resolution cross-linking immunoprecipitation with RT-qPCR to show that LINC01432 directly interacts with the RNA binding protein, CELF2. Lastly, we showed that LINC01432-targeted locked nucleic acid antisense oligonucleotides reduce viability and increases apoptosis. In summary, this fundamental study identified lncRNAs associated with short PFS to standard NDMM treatment and further characterized LINC01432, which inhibits apoptosis. Key points: lncRNA expression was found to be dysregulated in patients with short PFS to standard multiple myeloma therapy. LINC01432-bound CELF2 inhibits apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847cba66ef27ec89828bb55803b54ea81c4c449e" target='_blank'>
              LINC01432 binds to CELF2 in newly diagnosed multiple myeloma promoting short progression-free survival to standard therapy
              </a>
            </td>
          <td>
            Richa Mishra, Prasanth Thunuguntla, Alani Perkin, Dhanusha Duraiyan, Katelyn Bagwill, Savannah Gonzales, Vanessa Brizuela, Steve Daly, Yoon Jae Chang, Mahdote Abebe, Yash Rajana, Kelly Wichmann, Catheryn Bolick, Jaiyana King, M. Fiala, J. Fortier, R. Jayasinghe, Mark Schroeder, Li Ding, Ravi Vij, Jessica M. Silva-Fisher
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background. Lung adenocarcinoma (LUAD) is the most common subtype of non-small cell lung cancer (NSCLC) and frequently affects non-smokers, especially women. It is characterized by a complex genetic profile and interactions with its microenvironment, which contribute to its aggressive and adaptable nature. Early symptoms are often subtle, leading to late diagnoses. Treatment approaches have advanced with targeted therapies and immunotherapy supplementing traditional chemotherapy and radiation. Despite these advancements, the prognosis remains variable, highlighting the need for continued research into new treatment strategies to improve outcomes. Method. In this study, we employed Single-cell RNA Sequencing (scRNA-seq) to comprehensively analyze the impact of chemotherapy on lung adenocarcinoma at the individual cell level. By comparing before and after treatment samples, we assessed the differential expression of genes and pathways, revealing insights into how different cell types within the tumour respond to chemotherapy. This approach enabled us to pinpoint specific mechanisms of drug resistance and highlight potential therapeutic targets for overcoming these challenges. Results. Our analysis uncovered substantial changes in gene expression between primary tumour cells and metastatic cells following chemotherapy. Notably, we observed that 45 pathways were shared between the top 50 upregulated pathways in the primary tumour and the top 50 downregulated pathways in the metastatic tumour post-chemotherapy. Conversely, there was no overlap between the top 50 downregulated pathways in the primary tumour and the top 50 upregulated pathways in the metastatic tumour after chemotherapy. This suggests that chemotherapy effectively downregulated the major upregulated pathways but did not upregulate the key downregulated pathways in metastatic tumours. Conclusions. Integrating single-cell transcriptomics into LUAD research offers detailed insights into the tumour's response to chemotherapy and its interaction with the immune system. This approach enhances our understanding of LUAD and aids in developing targeted and effective treatments. Based on our analysis, we hypothesize that combining chemotherapy with drugs designed to upregulate the downregulated pathways in primary tumour cells could significantly enhance treatment efficacy and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9748c84cdcd0c997de55e9e674c83813950dc2" target='_blank'>
              Unveiling Chemotherapy's Impact on Lung Cancer through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            MD Saed Sayad, PhD Mark Hiatt, Mba MS Hazem Mustafa Md
          </td>
          <td>2024-07-10</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recurrent mutations in serine / arginine-rich splicing factor 2 (SRSF2), particularly the proline-to-histidine substitution at position 95 (P95H), have been proposed to drive neoplastic diseases. SRSF2 plays pivotal roles in pre-mRNA processing and gene transcription. However, the precise impact of these diverse functions of SRSF2 on cancer cell behaviour remains unclear. Here, we show that deletion or homozygous P95H mutation of SRSF2 both cause extensive DNA damage leading to cell cycle arrest in vitro and in vivo, regardless of cell genotype. We mechanistically demonstrate that SRSF2 is required for efficient bi-directional transcription of DNA replication and repair genes, independent of its function in splicing. In contrast, SRSF2 haploinsufficiency induces DNA damage without halting the cell cycle in cancer cells. Inducing the heterozygous Srsf2 P95H mutation leads to clonal expansion of epidermal cells in mouse skin, but tumor formation is inhibited following exposure to carcinogens. To survive carcinogen treatment, cells containing the Srsf2 P95H mutation undergo substantial transcription rewiring that restores bi-directional gene expression. Thus, our study reveals crucial roles of SRSF2 beyond splicing and underscores its importance in regulating transcription to orchestrate the cell cycle along with the DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32907ead7270689c743bba801c421a14be049a66" target='_blank'>
              SRSF2 transcriptional function maintains genome integrity during cell division
              </a>
            </td>
          <td>
            Rebecca E. Wagner, Leonie Arnetzl, Thiago Britto-Borges, Anke Heit-Mondrzyk, Ali Bakr, Etienne Sollier, Jasper Panten, Sylvain Delaunay, Daniela Sohn, P. Schmezer, D. T. Odom, Karin Müller-Decker, C. Plass, Christoph Dieterich, P. Lutsik, Susanne Bornelöv, Michaela Frye
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Drug resistance is currently one of the biggest challenges in cancer treatment. With the deepening understanding of drug resistance, various mechanisms have been revealed, including metabolic reprogramming and alterations of redox balance. Notably, metabolic reprogramming mediates the survival of tumor cells in harsh environments, thereby promoting the development of drug resistance. In addition, the changes during metabolic pattern shift trigger reactive oxygen species (ROS) production, which in turn regulates cellular metabolism, DNA repair, cell death, and drug metabolism in direct or indirect ways to influence the sensitivity of tumors to therapies. Therefore, the intersection of metabolism and ROS profoundly affects tumor drug resistance, and clarifying the entangled mechanisms may be beneficial for developing drugs and treatment methods to thwart drug resistance. In this review, we will summarize the regulatory mechanism of redox and metabolism on tumor drug resistance and highlight recent therapeutic strategies targeting metabolic–redox circuits, including dietary interventions, novel chemosynthetic drugs, drug combination regimens, and novel drug delivery systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cc6698aebc91736fc846603c2b274976902d8b" target='_blank'>
              Targeting Metabolic–Redox Nexus to Regulate Drug Resistance: From Mechanism to Tumor Therapy
              </a>
            </td>
          <td>
            Yuke Wang, Jingqiu He, Shan Lian, Yan Zeng, Sheng He, Jue Xu, Li Luo, Wenyong Yang, Jingwen Jiang
          </td>
          <td>2024-07-01</td>
          <td>Antioxidants</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Leukemia represents a diverse group of hematopoietic neoplasms that can be classified into different subtypes based on the molecular aberration in the affected cell population. Identification of these molecular classification is required to identify specific targeted therapeutic approaches for each leukemic subtype. In general, targeted therapy approaches achieve good responses in some leukemia subgroups, however, resistance against these targeted therapies is common. In this review, we summarize molecular drug resistance biomarkers in targeted therapies in BCR::ABL1-driven chronic myeloid leukemia (CML) and JAK2-driven myeloproliferative neoplasms (MPNs). While acquisition of secondary mutations in the BCR::ABL1 kinase domain is the a common mechanism associated with TKI resistance in CML, in JAK2-driven MPNs secondary mutations in JAK2 are rare. Due to high prevalence and lack of specific therapy approaches in MPNs compared to CML, identification of crucial pathways leading to inhibitor persistence in MPN model is utterly important. In this review, we focus on different alternative signaling pathways activated in both, BCR::ABL1-mediated CML and JAK2-mediated MPNs, by combining data from in vitro and in vivo-studies that could be used as potential biomarkers of drug resistance. In a nutshell, some common similarities, especially activation of PDGFR, Ras, PI3K/Akt signaling pathways, have been demonstrated in both leukemias. In addition, induction of the nucleoprotein YBX1 was shown to be involved in TKI-resistant JAK2-mediated MPN, as well as TKI-resistant CML highlighting deubiquitinating enzymes as potential biomarkers of TKI resistance. Taken together, whole exome sequencing of cell-based or patients-derived samples are highly beneficial to define specific resistance markers. Additionally, this might be helpful for the development of novel diagnostic tools, e.g., liquid biopsy, and novel therapeutic agents, which could be used to overcome TKI resistance in molecularly distinct leukemia subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b47e0df361f50995af94c5ef4667538d96cf6be" target='_blank'>
              Molecular biomarkers of leukemia: convergence-based drug resistance mechanisms in chronic myeloid leukemia and myeloproliferative neoplasms
              </a>
            </td>
          <td>
            Meike Kaehler, N. von Bubnoff, Ingolf Cascorbi, S. P. Gorantla
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 Monosomy 7 is among the most frequent cytogenetic abnormalities in hematological disorders and is associated with short survival and drug resistance. Despite its high prevalence and detrimental impact, the therapeutic vulnerabilities underlying monosomy 7-associated blood disorders remain largely elusive, impeding progress toward improved patient care. The homeostatic cellular requirement for a normal dosage of essential genes creates an opportunity to target vulnerabilities that arise due to reduced levels of proteins encoded by a haploinsufficient essential gene. Briefly, the loss of one copy of a dosage-sensitive essential gene (gene X), in combination with the inhibition of itself or a related gene (gene Y), or an associated pathway results in lethal consequences for cells. The remarkable and selective clinical efficacy of lenalidomide for the treatment of del(5q) MDS has demonstrated how allelic haploinsufficiency underlies the sensitivity to this drug by synthetic lethality. Differential expression analysis of gene and protein expression in primary AML samples with monosomy 7 revealed significant downregulation of multiple proteasome pathway members at the protein level, but not at the RNA level. Primary AML samples with -7/del(7q) exhibited increased sensitivity (low IC50) to the proteasome inhibitor bortezomib, as evidenced by two independent ex vivo drug screening cohorts (the Beat AML and the FIMM study). Chromosome 7 harbors four proteasome subunits, PSMA2, PSMC2, PSMG3, and SEM1. We performed gene expression, protein expression, copy number analysis, and individual gene knockout experiments. The results have revealed PSMA2 to be a haploinsufficient essential gene on chromosome 7. PSMA2 knockout confers leukemia a growth disadvantage for multiple AML cell clines in both TP53 wild-type and knockout backgrounds. We generated PSMA2 isogenic hemizygous deletion and diploid single-cell clones. PSMA2 hemizygous deletion cells exhibited approximately half the protein expression compared to diploid controls, confirming that PSMA2 is a haploinsufficient gene. PSMA2hemizygous deletion single-cell clones showed significantly enhanced sensitivity to all three evaluated proteasome inhibitors (bortezomib, ixazomib, and carfilzomib), aligning with the sensitivity observed in primary -7/del(7q) leukemia samples. PSMA2 hemizygous deletion cell clones displayed increased p38 and decreased pERK levels upon treatment with proteasome inhibitors, potentially contributing to their increased sensitivity to proteasome inhibitors. Proteomics analysis and in vivo validation is ongoing. As such, we have identified haploinsufficient essential gene PSMA2 mediated proteasome pathway vulnerability in monosomy 7 associated leukemia and further showed that proteasome inhibitors as promising therapeutic approaches for treating hematological disorders associated with monosomy 7.
 Citation Format: Haijiao Zhang, Basil Allen, Daniel Bottomly, Peter Ryabinin, Schannon K. Targeting proteasome vulnerabilities for the treatment of monosomy 7 associated blood disorders [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr A012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc1af4eb1e6787f18ac5d5b25044d1f348db5af2" target='_blank'>
              Abstract A012: Targeting proteasome vulnerabilities for the treatment of monosomy 7 associated blood disorders
              </a>
            </td>
          <td>
            Haijiao Zhang, Basil Allen, D. Bottomly, P. Ryabinin, Schannon K Mcweeney
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="
 Triple-negative breast cancer (TNBC) accounts for 15% of breast cancers and is the most aggressive subtype lacking precision oncology therapeutic strategies. Standard-of-care is predominantly chemotherapy in the neoadjuvant setting (NACT). Despite good responses, ∼40% of TNBC patients develop resistance and present residual disease at surgery. Overcoming resistance and implementing better therapeutic options is critical. Our objectives are to develop targeting opportunities for drug tolerant persister (DTP) cells which underlie residual disease. Using our unique biobank of patient derived xenografts from treatment naive as well as NACT resistant TNBC, we developed longitudinal in vivo models of residual and relapse tumors to standard-of-care therapy. Transcriptomic analysis revealed transient metabolic changes such as oxidative phosphorylation, starvation and autophagy, associated with hallmarks of senescence at residual disease. We demonstrated that therapy-induced senescent cells in vitro can escape cell cycle arrest and resume proliferation through autophagy. Interfering with autophagy impairs redox balance, promotes ferroptosis and delays tumor relapse. These results support that autophagy is a promising targetable vulnerability in TNBC residual disease.
 Citation Format: Anne-Marie Fortier, Jason Topolski, Gabriel Alzial, Anie Monast, Hellen Kuasne, Dongmei Zuo, Alain Pacis, Genevieve Deblois, Morag Park. Drug tolerant persister cancer cells escape therapy-induced senescence [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b6f648d19433ea733a51e501d785e6719459732" target='_blank'>
              Abstract B010: Drug tolerant persister cancer cells escape therapy-induced senescence
              </a>
            </td>
          <td>
            Anne-Marie Fortier, Jason Topolski, Gabriel Alzial, A. Monast, H. Kuasne, D. Zuo, Alain Pacis, Genevieve Deblois, Morag Park
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 c-MYC (MYC) is a central regulatory protein that is dysregulated in >50% of all human cancers and is linked to aggressive disease. Developing MYC inhibitors would revolutionize cancer treatment; however, developing small molecules that directly target MYC is challenging. An alternative approach is to identify and inhibit critical MYC partner proteins to inactivate MYC and trigger cancer cell death. Inhibiting these targets therapeutically can result in synthetic lethality (MYC-SL), which can be exploited in MYC-dysregulated cancers. To identify MYC-SL targets, we performed a genome-wide CRISPR knock-out screen using an isogenic pair of normal and MYC-driven breast cancer cells. In contrast to other screens, this model is dependent on MYC and recapitulates human disease at both pathological and molecular levels in vivo. We identified high-priority hits to validate from the screen using two independent approaches: 1) a traditional gene-set enrichment analysis to identify highly represented biological pathways; and 2) analyzing the Cancer Dependency Map (DEPMAP) to select hits that are likely to be robust beyond the context of our screening conditions. Using a traditional gene-set enrichment analysis approach, we identified topoisomerase 1 (TOP1) as an actionable vulnerability that can be targeted with clinically approved inhibitors. Genetic and pharmacological inhibition of TOP1 resulted in MYC-driven cell death compared to that in control cells. Finally, drug response to TOP1 inhibitors significantly correlated with MYC levels and activity across panels of breast cancer cell lines and patient-derived organoids, highlighting TOP1 as a promising target for MYC-driven cancers. As a secondary approach to interpreting our CRISPR screen hits, we analyzed DEPMAP to identify MYC-SLs that are differentially essential in MYC-dependent cancer cells. Specifically, data from RNA interference screens in hundreds of cancer cell lines were used to describe the response of these cells to MYC knockdown. These data were used to define MYC-dependent and MYC-independent cell lines within the context of this analysis. These two groups were then investigated for their in silico response to the knockdown of each of our MYC-SL hits. MYC-SLs, which were also differentially essential in MYC-dependent cancer cells from DEPMAP, were prioritized for further investigation. Critical MYC cofactors that have been validated by us and others (e.g., CDK9) were identified, providing confidence in this approach, and rationalizing ongoing investigations. Together, this work features two successful strategies to prioritize hits from hundreds of synthetic-lethal genome-wide CRISPR screens to identify novel MYC-driven vulnerabilities in cancer.
 Citation Format: Peter Lin, Linda Penn. An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fe21fa230f85037dcec9224411cd263d57af781" target='_blank'>
              Abstract B004: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Linda Penn
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc05d0e416d1c5836cac6aabe3bd3a577d1440ba" target='_blank'>
              Enhanced eMAGE applied to identify genetic factors of nuclear hormone receptor dysfunction via combinatorial gene editing
              </a>
            </td>
          <td>
            Peter N Ciaccia, Zhuobin Liang, Anabel Y. Schweitzer, Eli Metzner, Farren J. Isaacs
          </td>
          <td>2024-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af8a5e84090feb0d13451624ef0b08cb6aa245a3" target='_blank'>
              Epigenetic regulation of DNA repair gene program by Hippo/YAP1-TET1 axis mediates sorafenib resistance in HCC
              </a>
            </td>
          <td>
            Chunli Mo, Weixin You, Yipeng Rao, Zhenping Lin, Shuai Wang, Ting He, Huanming Shen, Xun Li, Rui Zhang, Boan Li
          </td>
          <td>2024-07-05</td>
          <td>Cellular and Molecular Life Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Triple‐negative breast cancer (TNBC) is often considered one of the most aggressive subtypes of breast cancer, characterized by a high recurrence rate and low overall survival (OS). It is notorious for posing challenges related to drug resistance. While there has been progress in TNBC research, the mechanisms underlying chemotherapy resistance in TNBC remain largely elusive. We collect single‐cell RNA sequencing (scRNA‐seq) data from five TNBC patients susceptible to chemotherapy and five resistant cases. Comprehensive analyses involving copy number variation (CNV), pseudotime trajectory, cell–cell interactions, pseudospace analysis, as well as transcription factor and functional enrichment are conducted specifically on macrophages and malignant cells. Furthermore, we performed validation experiments on clinical samples using multiplex immunofluorescence. We identified a subset of SPP1+ macrophages that secrete SPP1 signals interacting with CD44 on malignant cell surfaces, potentially activating the PDE3B pathway within malignant cells via the integrin pathway, leading to chemotherapy resistance. The abnormally enhanced SPP1 signal between macrophages and malignant cells may serve as a factor promoting chemotherapy resistance in TNBC patients. Therefore, SPP1+ macrophages could potentially serve as a therapeutic target to reduce chemotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8af0989421bdfb2401a61fa8d2dab5197b650cb7" target='_blank'>
              Single‐cell transcriptome sequencing reveals SPP1‐CD44‐mediated macrophage–tumor cell interactions drive chemoresistance in TNBC
              </a>
            </td>
          <td>
            Fuzhong Liu, Junfeng Zhang, Xiaowei Gu, Qiuyang Guo, Wenjia Guo
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer.
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Antibiotic resistance is the leading cause of death globally, with a higher possibility of the emergence of highly resistant pathogens, leading to epidemics. Several antibiotic resistance mechanisms have been discovered, such as enhanced efflux of antibiotics, reduced influx of antibiotics, alteration of antibiotics or their targets, and adaptation to antibiotics. However, this mechanism cannot fully explain the development of antibiotic resistance because the genes associated with this mechanism have been elucidated. However, the factors governing their regulation are not yet fully understood. Recent studies have highlighted the epigenetic and epitranscriptomic roles of antibiotic resistance development-associated genes. Epigenetic modification is associated with DNA modification, whereas epitranscriptomic modification is associated with RNA modification to control gene expression by regulating various biological phenomena such as splicing, translation, and stability. Therefore, this review will focus on the discovery of epigenetic modifications, particularly by DNA methyltransferases, such as restriction-modification (R-M) systems associated with methyltransferases, orphan DNA methyltransferases, and nucleoid-associated proteins that contribute to the development of antibiotic resistance. This scrutinization further expands to epitranscriptomic modification of non-coding RNA, which has a role in the regulation of antibiotic resistance. Epitranscriptomic modification of ribosomal RNA (rRNA), which is a major target of antibiotics, has been well explored. while non-coding RNA such as cis and trans small non coding RNA, and riboswitches are poorly explored. This epigenetic and epitranscriptomic modification will help to understand the regulation of antibiotic resistance-associated genes, which will help to identify key regulators of antibiotic resistance, paving the way for new antibiotic discovery, leading to decreased antibiotic mortality globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b16c6a0ffde83445547b9ffde6ff971186ffa3" target='_blank'>
              A comprehensive review on epigenetic and epitranscriptomic-mediated regulation of antibiotic resistance
              </a>
            </td>
          <td>
            P. K. Giri, Shahil Alam, Madhav Dhakal
          </td>
          <td>2024-07-24</td>
          <td>F1000Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Currently, cancer is still a leading cause of human death globally. Tumor deterioration comprises multiple events including metastasis, therapeutic resistance and immune evasion, all of which are tightly related to the phenotypic plasticity especially epithelial–mesenchymal plasticity (EMP). Tumor cells with EMP are manifest in three states as epithelial–mesenchymal transition (EMT), partial EMT, and mesenchymal–epithelial transition, which orchestrate the phenotypic switch and heterogeneity of tumor cells via transcriptional regulation and a series of signaling pathways, including transforming growth factor‐β, Wnt/β‐catenin, and Notch. However, due to the complicated nature of EMP, the diverse process of EMP is still not fully understood. In this review, we systematically conclude the biological background, regulating mechanisms of EMP as well as the role of EMP in therapy response. We also summarize a range of small molecule inhibitors, immune‐related therapeutic approaches, and combination therapies that have been developed to target EMP for the outstanding role of EMP‐driven tumor deterioration. Additionally, we explore the potential technique for EMP‐based tumor mechanistic investigation and therapeutic research, which may burst vigorous prospects. Overall, we elucidate the multifaceted aspects of EMP in tumor progression and suggest a promising direction of cancer treatment based on targeting EMP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32fcd891c5d88b251baf8fc511031710e0fc1629" target='_blank'>
              Epithelial–mesenchymal plasticity in cancer: signaling pathways and therapeutic targets
              </a>
            </td>
          <td>
            Xiangpeng Wang, Xiao-yong Xue, Mingshi Pang, Liuchunyang Yu, Jinxiu Qian, Xiaoyu Li, Meng Tian, Aiping Lyu, Cheng Lu, Yuanyan Liu
          </td>
          <td>2024-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Epigenomic mechanisms are critically involved in mediation of genetic and environmental factors that underlie cancer development. Histone modifications represent highly informative epigenomic marks that reveal activation and repression of gene activities and dysregulation of transcriptional control due to tumorigenesis. Here, we present a comprehensive epigenomic and transcriptomic mapping of 18 tumor and 20 non-neoplastic tissues from non-small cell lung adenocarcinoma patients. Our profiling covers 5 histone marks including activating (H3K4me3, H3K4me1, and H3K27ac) and repressive (H3K27me3 and H3K9me3) marks and the transcriptome using only 20 mg of tissue per sample, enabled by low-input omic technologies. Using advanced integrative bioinformatic analysis, we uncovered cancer-driving signaling cascade networks, changes in 3D genome modularity, and differential expression and functionalities of transcription factors and noncoding RNAs. Many of these identified genes and regulatory molecules showed no significant change in their expression or a single epigenomic modality, emphasizing the power of integrative multimodal and multiomic analysis using patient samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1c31ae6b0dfa593dc1e04630030084d32008118" target='_blank'>
              Comprehensive multimodal and multiomic profiling reveals epigenetic and transcriptional reprogramming in lung tumors
              </a>
            </td>
          <td>
            Peiyao Wu, Zhengzhi Liu, Lina Zheng, Zirui Zhou, Wei Wang, Chang Lu
          </td>
          <td>2024-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Evolutionary processes in growing, well-mixed populations have been thoroughly investigated both experimentally and mathematically. More recently, insights have been gained into mutant evolution in expanding, spatially structured populations, using agent-based models and related simulation approaches. These become computationally challenging at high population sizes, leaving gaps in our understanding of evolutionary processes in microbes and cancer. To address this issue, we derive laws of mutant growth in two- and three-dimensional, expanding cell populations, including multi-step mutant accumulation. These laws, which include both neutral and advantageous mutations, can be used to readily predict mutant abundance in large populations under various assumptions. We apply this approach to ask under what conditions intermediate cell types with an elevated mutation rate can substantially contribute to the accumulation of point mutations. In particular, we focus on gene duplications/amplifications that can increase mutation rate due to increased copy number. We find that cells with elevated mutation rates contribute most in a 2D setting, with substantial contributions also occurring during 3D growth. In contrast to that, for well-mixed, exponentially growing populations, intermediate types cannot accelerate mutant production under biologically realistic assumptions. These results can reconcile contradicting experimental evolution studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66a74589a5a54778d1213b1773fc0c1fa947fd22" target='_blank'>
              Mutant scaling laws reveal that accelerated mutant evolution via gene amplifications requires spatially structured population growth
              </a>
            </td>
          <td>
            N. L. Komarova, Justin Pritchard, Dominik Wodarz
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Excess gene dosage from chromosome 21 (chr21) causes Down syndrome (DS), spanning developmental and acute phenotypes in terminal cell types. Which phenotypes remain amenable to intervention after development is unknown. To address this question in a model of DS neurogenesis, we derived trisomy 21 (T21) human induced pluripotent stem cells (iPSCs) alongside, otherwise, isogenic euploid controls from mosaic DS fibroblasts and equipped one chr21 copy with an inducible XIST transgene. Monoallelic chr21 silencing by XIST is near-complete and irreversible in iPSCs. Differential expression reveals that T21 neural lineages and iPSCs share suppressed translation and mitochondrial pathways and activate cellular stress responses. When XIST is induced before the neural progenitor stage, T21 dosage correction suppresses a pronounced skew toward astrogenesis in neural differentiation. Because our transgene remains inducible in postmitotic T21 neurons and astrocytes, we demonstrate that XIST efficiently represses genes even after terminal differentiation, which will empower exploration of cell type–specific T21 phenotypes that remain responsive to chr21 dosage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edc6c951e3aae8ef7952356533bbb9ae58e6f994" target='_blank'>
              A dynamic in vitro model of Down syndrome neurogenesis with trisomy 21 gene dosage correction
              </a>
            </td>
          <td>
            Prakhar Bansal, Erin C Banda, Heather R. Glatt-Deeley, Christopher E Stoddard, Jeremy W. Linsley, Neha Arora, Cécile Deleschaux, Darcy T. Ahern, Yuvabharath Kondaveeti, Rachael E. Massey, Michael Nicouleau, Shijie Wang, Miguel Sabariego-Navarro, Mara Dierssen, Steve Finkbeiner, Stefan F. Pinter
          </td>
          <td>2024-06-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 DNA damage response (DDR) genes are frequently altered in cancer, which may lead to protein loss-of-function (LoF) and result in particular DDR defects that can be therapeutically targeted via the concept of synthetic lethality (SL). There is a high prevalence of DDR alterations in endometrial cancer, a gynecologic malignancy with increased incidence and mortality in recent decades, and an unmet clinical need for a wider range of treatment options to reduce adverse effects and combat therapy resistance. In this study, a large-scale, agnostic bioinformatic screening approach was employed to discover novel potential SL targets specific to DDR LoF biomarkers in endometrial cancer for subsequent experimental validation in vitro. SL analysis was performed with the in-house Target / Biomarker / Lineage (TBL) bioinformatic pipeline (Division of Cancer Therapeutics, The Institute of Cancer Research) using datasets obtained from the Cancer Dependency Map (DepMap). The TBL pipeline statistically analyzed cancer cell line target gene dependency probabilities, inferred from Project Achilles and Project Score genome-wide CRISPR-Cas9 dropout screens, and integrated this with mutation and copy number variation genomic datasets. Potential interactions were filtered for target-biomarker mutual exclusivity using patient data from The Cancer Genome Atlas (TCGA). The ability of the TBL pipeline to identify known SL interactions was demonstrated through in silico identification of ARID1B/ARID1A in ovarian cancer and SMARCA2/SMARCA4 in lung cancer as significant hits. In addition, the identification of WRN/ARID1A as a significant hit in uterine cancer, given that ARID1A loss has been significantly associated with microsatellite instability (MSI) in endometrial cancer, is consistent with previous findings of WRN helicase as a SL partner with MSI in endometrial cancer. Analysis and downstream refinement of endometrial cancer significant hits led to the identification of seven final candidate target-biomarker gene pairs. These putative SL partners comprise novel potential endometrial cancer targets, with various functions ranging from nucleotide biosynthesis to epigenetic remodeling, and include novel potential and established LoF biomarkers relevant to endometrial cancer biology, namely DNA mismatch repair. In vitro validation of the final candidates by genetic or pharmacologic approaches to assess target gene dependency in endometrial cancer cell lines with altered or wildtype biomarker status will be discussed. These experiments will provide key experimental data to validate our bioinformatic approach to SL target-biomarker discovery, and may identify novel SL partners involving DDR LoF alterations in endometrial cancer that could lead to the development of new molecular targeted therapies and improved treatment options for this patient subpopulation.
 Citation Format: Julie Zhou, Konstantinos Mitsopoulos, Olivia Rossanese. Hunting for synthetic lethal partners in DNA damage response-altered endometrial cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr A021.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/083c615048ec61f99132382bde5b7a0011a16ab4" target='_blank'>
              Abstract A021: Hunting for synthetic lethal partners in DNA damage response-altered endometrial cancer
              </a>
            </td>
          <td>
            Julie Zhou, Konstantinos Mitsopoulos, Olivia Rossanese
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.
 Citation Format: Lixing Yang. Data-driven discoveries of molecular mechanism and therapeutic vulnerabilities of CDK12 mutant tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88abb1d2de21d0d5c6fc7861d116fbf694712a58" target='_blank'>
              Abstract B006: Data-driven discoveries of molecular mechanism and therapeutic vulnerabilities of CDK12 mutant tumors
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA damage-sensing (DDS) and DNA damage-repair (DDR) mechanisms are essential for the fidelity of genetic information transmission. Failure to accomplish an effective DDS/DDR mechanism can lead to cell death or otherwise to cell transformation and cancer development. microRNAs (miRs) are short noncoding RNAs that primarily function as micromanagers of gene expression. Herein, we aimed to investigate the links between miRs and the translation of specific mRNAs encoding proteins involved in genomic DDS and DDR and to screen drugs that have high binding affinity to the selected miRs, which may serve as cancer therapeutics. To accomplish these aims, we used a variety of computational methods spanning data analysis, molecular modeling, and simulation tools (i.e., PyRx, Biopython, ViennaRNA, RNAComposer, AutoDock Vina, OpenBabel, PyMOL, Discovery Studio, MarvinSketch). The genes and miRNAs involved in the DDS and DDR mechanisms were retrieved from either the literature or various online databases (e.g., miRDB). miR data were further cleaned and prepared using scripts, and various libraries were used to obtain their 3-D structures. Genes interacting with miRs were enriched based on multiple database annotations using Enrichr KG. Then, we used docking analyses to virtually screen compounds to serve as ligands for the miRs. Finally, we generated gene-disease-miR-drug networks to study the linkages between the compounds and miR molecules under investigation. For the first time, we were able to identify five compounds that could be repurposed for downregulating miRs that are linked to inhibition of translation of mRNA involved in the DDS and DDR processes. The gathered candidate drugs can be useful for preventing cell transformation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04982a355813b15d80368bb235c815920aa1fe9d" target='_blank'>
              Drugs targeting microRNAs that regulate DNA damage sensing and repair mechanisms: A computational drug discovery approach
              </a>
            </td>
          <td>
            S. Ravindranath, Ramirez, Gomez Mejiba S.E, S. Gomez Mejiba, ●. Ravindranath, ●. Ramirez, ●. G. Mejiba
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 It is well appreciated that the integrity of both pluripotent and terminally differentiated cellular genomes is under constant pressure from damage provoked by cell-intrinsic and environmental factors. As such, defects in the cellular machinery preserving genomic integrity lead to the accumulation of variants that can promote cancer, but they also confer tumor-cell specific vulnerabilities that present synthetic-lethal therapeutic opportunities. The deepening mechanistic appreciation of tumor-selective pressure on DNA replication-fork fidelity, namely oncogene-induced replication stress, has provoked great interest in the prosecution of drug targets within the cellular machinery that otherwise supports replication fork stability. Poly (ADP-Ribose) glycohydrolase (PARG) is an enzyme with a unique role in the resolution of DNA damage repair and DNA replication fork restart through hydrolysis of Poly (ADP-ribose) (PAR) chains. IDE161 is a potent, selective small molecule inhibitor of PARG that is being developed as an anti-cancer therapeutic for patients with advanced or metastatic cancer harboring defects in homologous recombination repair. In culture, the accumulation of PAR chains upon PARG inhibition causes delayed repair of DNA breaks, replication stress and mitotic catastrophe in settings with collateral liabilities in DNA damage repair and/or replication fork stability. Genome-wide CRISPR-mediated fitness screens across multiple genetic backgrounds revealed strong synthetic lethal interactions of IDE161 with otherwise non-essential components of cellular machinery supporting base excision repair, homologous recombination repair and replication fork stability. In addition, evaluation of IDE161 antiproliferative activity across a panel of 350 molecularly characterized cancer cell lines returned a response profile that was only partially overlapping with that observed with PARPi and was enriched for models harboring vulnerabilities associated with replication stress and/or defects in DNA repair. Notably, many homologous recombination deficient (HRD) cell models display differential dependency on PARG activity vs PARP1/2 activity, suggesting PARG inhibition as a potential therapeutic strategy distinct from PARP inhibitors. In vivo assessment of IDE161 anti-tumor activity in CDX and PDX models mirrored these mechanistic relationships. A number of HRD PDX models displayed a superior response to IDE161 versus PARPi, including robust regressions selectively observed with IDE161 versus niraparib in BRCA2 altered ER+, HER2-breast cancers. Thus, IDE161 has potential as novel targeted therapy that exploits the synthetic lethal relationship between PARG and genomic instability, leading to selective anti-proliferative activity in tumors harboring defects in DNA repair and replication fidelity.
 Citation Format: Reeja Maskey, Diana Munoz, Megan Conway, Arjun Rao, Vidhya Nagarajan, Ivan Shabalin, Kelly Trego, Jonathan Ryan, Peter Teriete, Christian Frey, Josh Taygerly, Paul Barsanti, Claire Neilan, Jasgit Sachdev, Michael White. Exploiting tumor selective vulnerabilities with structure based drug design [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/074f2143d9828ee7ba6111cebbb042ffcd4b5f81" target='_blank'>
              Abstract IA008: Exploiting tumor selective vulnerabilities with structure based drug design
              </a>
            </td>
          <td>
            Reeja Maskey, Diana Munoz, Megan Conway, Arjun A. Rao, Vidhya Nagarajan, Ivan Shabalin, Kelly Trego, Jonathan Ryan, Peter Teriete, Christian Frey, Josh Taygerly, Paul Barsanti, Claire Neilan, Jasgit Sachdev, Michael A. White
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a heterogeneous malignancy driven by diverse cytogenetic mutations. While risk stratification improved by identifying cytogenetic lesions, prognostication remains inadequate with 30% of standard-risk patients experiencing relapse within 5 years. Single-cell RNA sequencing (scRNAseq) enabled the interrogation of malignant cell heterogeneity in pAML and characterization of the immune microenvironment. Herein we report the largest pAML scRNAseq analysis to date with 708,285 cells from 164 bone marrow biopsies of 95 patients and 11 healthy controls. We uncovered treatment-resistant (TR) subtypes of pAML specific to RUNX1-RUNX1T1, FLT3-ITD, and CBFB-MYH11 patients. The enrichment of TR subtype gene signatures on the TARGET pAML data supported an association with significantly poor outcomes. Intriguingly, in addition to leukemic stem cells, we identified mast cell-like pAML associated with treatment resistance and poor outcomes. Together, immature and mature pAML subtypes are promising biomarkers for identifying patients at increased risk of relapse within cytogenetic categories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70fa48d362939fe5d28efc31302329cc356df67f" target='_blank'>
              Single-cell analysis of pediatric acute myeloid leukemia samples uncovers treatment-resistant stem and mast cells
              </a>
            </td>
          <td>
            Denis Ohlstrom, Mojtaba Bakhtia, Hope L. Mumme, Marina E. Michaud, Franklin Chien, William Pilcher, Sarthak Satpathy, Sean C Jordan, Swati S. Bhasin, Manoj K. Bhasin
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="A central challenge in the quest for precise gene regulation within mammalian cells is the development of regulatory networks that can achieve perfect adaptation-where outputs consistently return to a set baseline post-stimulus. Here, we present such a system that leverages the CRISPR activation (CRISPRa) and anti-CRISPR proteins as two antithetic elements to establish perfect adaptation in mammalian cells and dynamically regulate gene expression. We demonstrate that this system can maintain stable expression levels of target genes in the face of external perturbations, thus providing a robust platform for biological applications. The versatility of our system is further showcased through its integration with endogenous regulatory mechanisms in T cells, such as the NF-κB-mediated immune response, and its ability to program apoptosis responses for precise spatial and temporal control of cellular growth and death. This study not only advances our understanding of gene regulation in mammalian cells but also opens new avenues for therapeutic intervention, particularly in diseases characterized by dysregulated gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9bdcf363536b5c7b6585a226adb59f423a716ec" target='_blank'>
              CRISPR perfect adaptation for robust control of cellular immune and apoptotic responses.
              </a>
            </td>
          <td>
            Yichi Zhang, Shuyi Zhang
          </td>
          <td>2024-08-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34aec63ae603a17ab57ab7a733726a9aff63c36f" target='_blank'>
              Alterations in pharmacogenetic genes and their implications for imatinib resistance in Chronic Myeloid Leukemia patients from an admixed population.
              </a>
            </td>
          <td>
            Karla Beatriz Cardias Cereja-Pantoja, Tereza Cristina de Brito Azevedo, L. Vinagre, Francisco Cezar Aquino de Moraes, Giovanna Gilioli da Costa Nunes, Natasha Monte, Angélica Leite de Alcântara, A. Cohen-Paes, M. R. Fernandes, Sidney Emanuel Batista Dos Santos, P. P. de Assumpção, Andrea Kely Campos Ribeiro dos Santos, R. Burbano, Raquel Cruz Guerrero, Ángel Carracedo, Ney Pereira Carneiro Dos Santos
          </td>
          <td>2024-06-18</td>
          <td>Cancer chemotherapy and pharmacology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Pyrimidine nucleotide biosynthesis is a druggable metabolic dependency of cancer cells, and chemotherapy agents targeting pyrimidine metabolism are the backbone of treatment for many cancers. Dihydroorotate dehydrogenase (DHODH) is an essential enzyme in the de novo pyrimidine biosynthesis pathway that can be targeted by clinically approved inhibitors. However, despite robust preclinical anticancer efficacy, DHODH inhibitors have shown limited single-agent activity in phase 1 and 2 clinical trials. Therefore, novel combination therapy strategies are necessary to realize the potential of these drugs. To search for therapeutic vulnerabilities induced by DHODH inhibition, we examined gene expression changes in cancer cells treated with the potent and selective DHODH inhibitor brequinar (BQ). This revealed that BQ treatment causes upregulation of antigen presentation pathway genes and cell surface MHC class I expression. Mechanistic studies showed that this effect is (1) strictly dependent on pyrimidine nucleotide depletion, (2) independent of canonical antigen presentation pathway transcriptional regulators, and (3) mediated by RNA polymerase II elongation control by positive transcription elongation factor B (P-TEFb). Furthermore, BQ showed impressive single-agent efficacy in the immunocompetent B16F10 melanoma model, and combination treatment with BQ and dual immune checkpoint blockade (anti-CTLA-4 plus anti-PD-1) significantly prolonged mouse survival compared to either therapy alone. Our results have important implications for the clinical development of DHODH inhibitors and provide a rationale for combination therapy with BQ and immune checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b0c0126387b9def9714026a2af50898ea415781" target='_blank'>
              DHODH inhibition enhances the efficacy of immune checkpoint blockade by increasing cancer cell antigen presentation
              </a>
            </td>
          <td>
            Nick Mullen, Surendra K. Shukla, Ravi Thakur, S. S. Kollala, Dezhen Wang, Nina V. Chaika, Juan F Santana, William R Miklavcic, Drew A. LaBreck, J. Mallareddy, David H. Price, A. Natarajan, Kamiya Mehla, David B. Sykes, Michael A. Hollingsworth, Pankaj K Singh
          </td>
          <td>2024-07-08</td>
          <td>eLife</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 Application of molecular targeted therapies for esophageal adenocarcinoma (EAC) has been limited by a lack of druggable oncogenic drivers. We propose that synthetic lethal interactions may provide new opportunities for targeted therapies in EAC. We have taken an integrated multi-omics approach incorporating Perturb-Seq (CRISPR editing combined with single cell RNA sequencing) and in vivo tumorigenesis assays to perform high-throughput characterisation of >70 high-confidence EAC driver genes, and genome-wide CRISPR-Cas9 knockout screens in isogenic models of EAC tumorigenesis to identify disease relevant synthetic lethal genetic interactions. MS-based proteomics, reverse phase protein arrays, polysome profiling and bulk RNA-sequencing of isogenic models were utilised to interrogate the biology of bona fide EAC drivers and associated driver specific gene dependencies. The overall goals were to (i) enhance our understanding of EAC tumorigenesis, (ii) identify potential opportunities for therapeutic interventions targeting EAC drivers via synthetic lethal-like approaches, and (iii) reduce the complexity of genetic heterogeneity by categorising EAC drivers with similar phenotypic outcomes. Through our approach we have identified complex crosstalk between the tumor suppressor SMAD4 and regulation of mTOR signaling, with specific downstream effects on translational reprogramming in EAC. Mutation or loss of SMAD4 occurs in up to 20% of EAC, but not pre-malignant tissue (Barrett’s esophagus), and is sufficient to promote transformation of pre-malignant cells in our in vivo tumorigenesis model. In this model, xenotransplanted SMAD4-deficient (via CRISPR-Cas9 knockout or shRNA knockdown) Barrett’s metaplasia cells formed invasive, metastatic tumors after a period of latency. SMAD4 deficient cells had downregulated expression of 4E-BP1, which inhibits EIF4E, the cap-dependent translation initiation factor. This was accompanied by increased mTOR activity, including phosphorylation and inactivation of 4EBP1. Moreover, we found that SMAD4-deficient cells preferentially upregulate cap-dependent translation at the expense of IRES mediated translation. Furthermore, perturbation of additional negative regulators of mTOR signaling in combination with SMAD4 knockout exacerbated these effects and accelerated tumorigenesis in vivo. We have extended these findings to a model of Barrett’s esophagus patient-derived organoids (PDOs) and observed increased proliferative potential of our genetically modified PDOs. Finally, analysing gene ontologies for differentially expressed genes from Perturb-seq revealed that driver-dependent transcriptional changes can be categorized into a smaller number of functional pathways allowing us to potentially consider groups of drivers as functional units. This work advances our understanding of EAC tumorigenesis, provides new mechanistic insights into SMAD4-driven transformation as well as novel potential therapeutic avenues for SMAD4-deficient EAC.
 Citation Format: Julia V. Milne, Ebtihal Mustafa, Kenji Fujihara, Eric Kusnadi, Anna Trigos, Niko Thio, Maree Pechlivanis, Carlos Cabalag, Twishi Gulati, Kaylene Simpson, Cuong Duong, Luc Furic, Wayne Phillips, Nicholas Clemons. Delineating functional drivers of esophageal adenocarcinoma to identify synthetic lethal interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95f3d989c9a3fa8592c661128bc239a7c646772" target='_blank'>
              Abstract PR007: Delineating functional drivers of esophageal adenocarcinoma to identify synthetic lethal interactions
              </a>
            </td>
          <td>
            Julia V. Milne, Ebtihal Mustafa, Kenji Fujihara, Eric P. Kusnadi, A. Trigos, N. Thio, Maree Pechlivanis, C. Cabalag, Twishi Gulati, Kaylene Simpson, C. Duong, Luc Furic, Wayne Phillips, Nicholas Clemons
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The drug floxuridine (5-fluorodeoxyuridine, FUdR) is an active metabolite of 5-Fluorouracil (5-FU). It converts to 5-fluorodeoxyuridine monophosphate (FdUMP) and 5-fluorodeoxyuridine triphosphate (FdUTP), which on incorporation into the genome inhibits DNA replication. Additionally, it inhibits thymidylate synthase, causing dTMP shortage while increasing dUMP availability, which induces uracil incorporation into the genome. However, the mechanisms underlying cellular tolerance to FUdR are yet to be fully elucidated. In this study, we explored the mechanisms underlying cellular resistance to FUdR by screening for FUdR hypersensitive mutants from a collection of DT40 mutants deficient in each genomic maintenance system. We identified REV3, which is involved in translesion DNA synthesis (TLS), to be a critical factor in FUdR tolerance. Replication using a FUdR-damaged template was attenuated in REV3-/- cells, indicating that the TLS function of REV3 is required to maintain replication on the FUdR-damaged template. Notably, FUdR-exposed REV3-/- cells exhibited defective cell cycle arrest in the early S phase, suggesting that REV3 is involved in intra-S checkpoint activation. Furthermore, REV3-/- cells showed defects in Chk1 phosphorylation, which is required for checkpoint activation, but the survival of FUdR-exposed REV3-/- cells was further reduced by the inhibition of Chk1 or ATR. These data indicate that REV3 mediates DNA checkpoint activation at least through Chk1 phosphorylation, but this signal acts in parallel with ATR-Chk1 DNA damage checkpoint pathway. Collectively, we reveal a previously unappreciated role of REV3 in FUdR tolerance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfdf5fbed75636278b15a31931e1734c5df819c5" target='_blank'>
              REV3 promotes cellular tolerance to 5-fluorodeoxyuridine by activating translesion DNA synthesis and intra-S checkpoint
              </a>
            </td>
          <td>
            Mubasshir Washif, Ryotaro Kawasumi, Kouji Hirota
          </td>
          <td>2024-07-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3169b366c8dfc0a14136eb7bf3ab2054f7cc2f88" target='_blank'>
              CRISPR-Cas9 screens reveal common essential miRNAs in human cancer cell lines
              </a>
            </td>
          <td>
            D. Merk, L. Paul, F. Tsiami, H. Hohenthanner, Ghazal Mohseni Kouchesfahani, L. Haeusser, B. Walter, A. Brown, N. Persky, D. Root, G. Tabatabai
          </td>
          <td>2024-06-17</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Acute myeloid leukemia (AML) is a complex and genetically diverse with an overall survival rate of less than 32%. Despite the remarkable success of targeted therapy in mediating remission, the emergence of acquired resistance remains a major clinical challenge to overcome. The prevailing understanding of acquired resistance identifies successive genetic changes as the primary cause.TP53 mutations are found in 70-80% of acute myeloid leukemia (AML) patients with complex karyotypes and associated with resistance towards both conventional chemotherapy and newly approved venetoclax plus azacytidine (VEN/AZA) combination. By using CRISPR-Cas9-edited isogenic AML cells harboring, mutation (6 missense mutations) or deletion (KO) of TP53, we found that TP53 mutant/KO cells are less sensitive to etoposide or VEN-AZA induced apoptosis compared to WT without defect in G1 arrest. Surprisingly, we found that TP53-mutant and TP53-wild-type (WT) isogenic AML cells and primary tumors (n=40) had comparable mitochondrial outer membrane permeabilization (MOMP) at baseline, despite the key role of TP53 in transcriptionally activating proapoptotic regulators of MOMP (such as BAX, PUMA, and NOXA). Based on these findings, we hypothesize that the targets downstream of mitochondrial permeabilization drive resistance to HMA/VEN combinations in TP53 mutant disease. By leveraging unbiased bulk RNA-seq and proteomics, and whole genome CRISPR-cas9 screen we identified IAPs as functional vulnerability. Collectively we reveal novel chemoresistance mechanisms in TP53 mutant/KO downstream of MOMP and provide a targeting strategy to improve existing therapy by targeting non-transcriptional function of TP53 in overcoming therapy resistance.
 Citation Format: Shruti Bhatt. Vulnerabilities of TP53-mutated AML and therapeutic implications [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f055e36b2a6b8fc2fa6c5e95146fecd24a5555aa" target='_blank'>
              Abstract IA018: Vulnerabilities of TP53-mutated AML and therapeutic implications
              </a>
            </td>
          <td>
            Shruti Bhatt
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive brain cancer characterized by significant molecular and cellular heterogeneity, which complicates treatment efforts. Current standard therapies, including surgical resection, radiation, and temozolomide (TMZ) chemotherapy, often fail to achieve long-term remission due to tumor recurrence and resistance. A pro-oxidant environment is involved in glioma progression, with oxidative stress contributing to the genetic instability that leads to gliomagenesis. Evaluating pro-oxidant therapies in brain tumors is crucial due to their potential to selectively target and eradicate cancer cells by exploiting the elevated oxidative stress levels inherent in these malignant cells, thereby offering a novel and effective strategy for overcoming resistance to conventional therapies. This study investigates the therapeutic potential of doxorubicin (DOX) and photodynamic therapy (PDT) with Me-ALA, focusing on their effects on redox homeostasis. Basal ROS levels and antioxidant gene expression (NFE2L2, CAT, GSR) were quantitatively assessed across GBM cell lines, revealing significant variability probably linked to genetic differences. DOX and PDT treatments, both individually and in combination, were analyzed for their efficacy in inducing oxidative stress and cytotoxicity. An in silico analysis further explored the relationship between gene mutations and oxidative stress in GBM patients, providing insights into the molecular mechanisms underlying treatment responses. Our findings suggest that pro-oxidant therapies, such as DOX and PDT in combination, could selectively target GBM cells, highlighting a promising avenue for improving therapeutic outcomes in GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a4c8f4dd03777d2e91e0e7df47c1185f6a5a011" target='_blank'>
              Enhancing Therapeutic Approaches in Glioblastoma with Pro-Oxidant Treatments and Synergistic Combinations: In Vitro Experience of Doxorubicin and Photodynamic Therapy
              </a>
            </td>
          <td>
            Bruno Agustín Cesca, M. Caverzán, M. Lamberti, L. E. Ibarra
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The expression of eukaryotic genes relies on the precise 3’-terminal cleavage and polyadenylation of newly synthesized pre-mRNA transcripts. Defects in these processes have been associated with various diseases, including cancer. While cancer-focused sequencing studies have identified numerous driver mutations in protein-coding sequences, noncoding drivers – particularly those affecting the cis-elements required for pre-mRNA cleavage and polyadenylation – have received less attention. Here, we systematically analysed cancer somatic mutations affecting 3’UTR polyadenylation signals using the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset. We found a striking enrichment of cancer-specific somatic mutations that disrupt strong and evolutionarily conserved cleavage and polyadenylation signals within tumour suppressor genes. Further bioinformatics and experimental analyses conducted as a part of our study suggest that these mutations have a profound capacity to downregulate the expression of tumour suppressor genes. Thus, this work uncovers a novel class of noncoding somatic mutations with significant potential to drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4b3938c7f84b24956b4223cbd287669eff1caf" target='_blank'>
              Recurrent disruption of tumour suppressor genes in cancer by somatic mutations in cleavage and polyadenylation signals
              </a>
            </td>
          <td>
            Yaroslav A. Kainov, F. Hamid, E. Makeyev
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Chromatin-associated multi-protein complexes, which often comprise proteins involved in epigenetic regulation, play an important role in various cancers, including leukemia. Thus, pharmacologic inhibition of proteins involved in these complexes can lead to new anti-cancer therapies. We have focused on therapeutic targeting of various chromatin-bound multi-protein complexes, including Mixed Lineage Leukemia 1 (MLL1) and Polycomb Repressive Complex 1 (PRC1) complexes, as a novel treatment for leukemia. These efforts led to the development of small molecule inhibitors targeting the menin-MLL1 interaction, which were translated to leukemia patients, demonstrating promising efficacy in Acute Myeloid Leukemia (AML) with NPM1 mutations or MLL1 translocations. However, clinical studies revealed resistance to the single agent menin inhibitor in a sub-set of AML patients, suggesting that combinatorial treatments or new generations of menin inhibitors might be required to overcome resistance. Our recent efforts to address resistance to menin inhibitors will be presented, including combinatorial studies with other targeted agents (e.g. kinases inhibitors), which resulted in synergistic effects. Furthermore, discovery of a new generation of menin inhibitors with promising activity against menin patient mutations will also be presented.
 Citation Format: Jolanta Grembecka. Targeting epigenetic complexes in leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA023.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf3e3f47e83690e2e75e5925154673a9592d6fbd" target='_blank'>
              Abstract IA023: Targeting epigenetic complexes in leukemia
              </a>
            </td>
          <td>
            J. Grembecka
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Precision (individualized) medicine relies on the molecular profiling of tumors’ dysregulated characteristics (genomic, epigenetic, transcriptomic) to identify the reliance on key pathways (including genome stability and epigenetic gene regulation) for viability or growth, and then utilises targeted therapeutics to disrupt these survival-dependent pathways. Non-mutational epigenetic changes alter cells’ transcriptional profile and are a key feature found in many tumors. In contrast to genetic mutations, epigenetic changes are reversable, and restoring a normal epigenetic profile can inhibit tumor growth and progression. Lysine acetyltransferases (KATs or HATs) protect genome stability and integrity, and Tip60 is an essential acetyltransferase due to its roles as an epigenetic and transcriptional regulator, and as master regulator of the DNA double-strand break response. Tip60 is commonly downregulated and mislocalized in many cancers, and the roles that mislocalized Tip60 plays in cancer are not well understood. Here we categorize and discuss Tip60-regulated genes, evaluate Tip60-interacting proteins based on cellular localization, and explore the therapeutic potential of Tip60-targeting compounds as epigenetic inhibitors. Understanding the multiple roles Tip60 plays in tumorigenesis will improve our understanding of tumor progression and will inform therapeutic options, including informing potential combinatorial regimes with current chemotherapeutics, leading to improvements in patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0558cedd75f278d6feed6575b276aad74d694cf" target='_blank'>
              Evaluating the Cellular Roles of the Lysine Acetyltransferase Tip60 in Cancer: A Multi-Action Molecular Target for Precision Oncology
              </a>
            </td>
          <td>
            Nazanin Zohourian, Erin Coll, Muiread Dever, Anna Sheahan, Petra Burns-Lane, James A. L. Brown
          </td>
          <td>2024-07-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce12d4cb261e95e6f4ee6a3b370aa98326787b3" target='_blank'>
              Investigating druggable kinases for targeted therapy in retinoblastoma.
              </a>
            </td>
          <td>
            Kumar Jeyaprakash, M. Kumaran, U. Kim, Radhakrishnan Santhi, V. Muthukkaruppan, B. Devarajan, A. Vanniarajan
          </td>
          <td>2024-07-01</td>
          <td>Journal of human genetics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29e5e59bd894b6791e1caba61c558e842d1e8aab" target='_blank'>
              Mitoferrin2 is a synthetic lethal target for chromosome 8p deleted cancers
              </a>
            </td>
          <td>
            Stephan Krieg, Thomas Rohde, Tobias Rausch, Luise Butthof, Lena Wendler-Link, Christoph Eckert, Kai Breuhahn, Bruno Galy, J. Korbel, Maximilian Billmann, Marco Breinig, Darjus F. Tschaharganeh
          </td>
          <td>2024-06-17</td>
          <td>Genome Medicine</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="The Ten-Eleven Translocation (TET) family genes are implicated in a wide array of biological functions across various human cancers. Nonetheless, there is a scarcity of studies that comprehensively analyze the correlation between TET family members and the molecular phenotypes and clinical characteristics of different cancers. Leveraging updated public databases and employing several bioinformatics analysis methods, we assessed the expression levels, somatic variations, methylation levels, and prognostic values of TET family genes. Additionally, we explored the association between the expression of TET family genes and pathway activity, tumor microenvironment (TME), stemness score, immune subtype, clinical staging, and drug sensitivity in pan-cancer. Molecular biology and cytology experiments were conducted to validate the potential role of TET3 in tumor progression. Each TET family gene displayed distinct expression patterns across at least ten detected tumors. The frequency of Single Nucleotide Variant (SNV) in TET genes was found to be 91.24%, primarily comprising missense mutation types, with the main types of copy number variant (CNV) being heterozygous amplifications and deletions. TET1 gene exhibited high methylation levels, whereas TET2 and TET3 genes displayed hypomethylation in most cancers, which correlated closely with patient prognosis. Pathway activity analysis revealed the involvement of TET family genes in multiple signaling pathways, including cell cycle, apoptosis, DNA damage response, hormone AR, PI3K/AKT, and RTK. Furthermore, the expression levels of TET family genes were shown to impact the clinical staging of tumor patients, modulate the sensitivity of chemotherapy drugs, and thereby influence patient prognosis by participating in the regulation of the tumor microenvironment, cellular stemness potential, and immune subtype. Notably, TET3 was identified to promote cancer progression across various tumors, and its silencing was found to inhibit tumor malignancy and enhance chemotherapy sensitivity. These findings shed light on the role of TET family genes in cancer progression and offer insights for further research on TET3 as a potential therapeutic target for pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4df11d8b0544aeb2264d4da3fba3e8f78e1111" target='_blank'>
              Pan-cancer analyses reveal the molecular and clinical characteristics of TET family members and suggests that TET3 maybe a potential therapeutic target
              </a>
            </td>
          <td>
            Chunyan Zhang, Jie Zheng, Jin Liu, Yanxia Li, G. Xing, Shupeng Zhang, Hekai Chen, Jian Wang, Z. Shao, Yongyuan Li, Zhongmin Jiang, Yingzi Pan, Xiaozhi Liu, Ping Xu, Wenhan Wu
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b517632791bed91e0fed3fb023edfd9ccfe184b7" target='_blank'>
              Deciphering the Role of Exosomal Non-Coding RNA (ncRNA) in Drug Resistance of Gastrointestinal Tumors; an Updated Review.
              </a>
            </td>
          <td>
            Nafiseh Tashakori, Maryam Armanfar, Anahita Mashhadi, Adil Tawfeeq Mohammed, Manal Morad Karim, Abbas Hameed Abdul Hussein, Mohaned Adil, Sajad Ataei Azimi, Fatemeh Abedini
          </td>
          <td>2024-06-15</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA methylation is traditionally associated with gene silencing, but its causal relationship and role in shaping cell fate decisions still need to be fully elucidated. Here, we conducted a genome-wide analysis to investigate the relationship between DNA methylation and gene expression at gene regulatory regions in human immune cells. By utilizing CRISPR-dCas9 DNA methylation editing tools, we successfully established a cause-and-effect relationship between the methylation levels of the promoter of the Interleukin1-receptor antagonist (IL1RN) gene and its expression. Notably, we observed that modifying the DNA methylation status of the IL1RN promoter is sufficient to alter the acquisition of the human myeloid cell fate and change the cellular response to inflammatory stimuli, resulting in abnormal cytokine release and distinctive capacity to support cancer growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0466661c0180cda161896839ec2c88a8975d7da9" target='_blank'>
              Modulating immune cell fate and inflammation through CRISPR-mediated DNA methylation editing
              </a>
            </td>
          <td>
            Gemma Valcárcel, Anna V. López-Rubio, Aleksey Lazarenkov, Clara Berenguer, Josep Calafell, J. Rodríguez-Ubreva, E. Ballestar, J. L. Sardina
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bccd8d9ff2cb22d908b272a8863cb4fce2351ba" target='_blank'>
              Monitoring and quantifying replication fork dynamics with high-throughput methods
              </a>
            </td>
          <td>
            Nora Fajri, N. Petryk
          </td>
          <td>2024-06-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Over 100 modifications to RNA are known to occur in human cells, where they play critical roles in many aspects of normal cellular physiology, such as cell fate decisions and terminal differentiation, through effects on RNA biology such as protein and nucleic acid interactions. Our survey of human RNA-modifying enzymes suggests many are cancer essential enzymes with striking synthetic lethal profiles, including the XRN1 nuclease and DHX9 helicase. XRN1 degrades single stranded mRNA from the 5′→3′ direction and plays a key role in endogenous cellular mRNA turnover. XRN1 can also degrade double-stranded RNA (dsRNA), and as such plays a role in innate immunity. We identified XRN1 as a selective vulnerability in tumor cells with intrinsic elevation of a Type I Interferon Stimulated Gene (TISG) signature through analysis of publicly available CRISPR data across 483 tumor cell lines. XRN1 KO via CRISPR leads to robust anti-proliferative effects in TISG high cells, but not in TISG low cells, as well as upregulation of Interferon-β mRNA and induction of phosphorylated PKR. Exogenous treatment with Interferon-β to induce elevated Type I Interferon expression sensitizes TISG low cells to XRN1 KO, confirming this synthetic lethal relationship. Enzymatic and biophysical assays for XRN1 were developed and used to identify and optimize XRN1 inhibitors. We identified Compound 1, a substrate-competitive inhibitor with single-digit micromolar affinity for XRN1 which does not bind or inhibit XRN2. High-resolution crystal structures indicate that Compound 1 binds in an allosteric binding site, confirming the mechanism of inhibition and selectivity profile determined by in vitro assays. This data suggests that XRN1, through its regulation of dsRNA burden, is a compelling and druggable oncology target for TISG high tumors. DHX9 is a multifunctional DExH-box RNA helicase which can unwind regions of double-stranded DNA and RNA helices but has a greater propensity for secondary structures such as DNA/RNA hybrids (R-loops) and DNA/RNA G-quadruplexes. DHX9 interacts with and regulates many proteins, including key members of DNA damage repair pathways. We have found that DHX9 knockdown is synthetic lethal in microsatellite high (MSI-H) or defective mismatch repair (dMMR) tumor models. A suite of assays was developed to identify and optimize potent and selective inhibitors of the DHX9 helicase, which recapitulate our findings with genetic tools. Profiling of DHX9 inhibitors across a broad panel of cancer cell lines reveals that tumor cells with mutations in the DNA damage repair genes BRCA1 and/or BRCA2 are also responsive to DHX9 inhibitor treatment in vitro and in vivo. DHX9 inhibition leads to increased RNA/DNA secondary structures such as R-loops and G-quadruplexes, resulting in subsequent DNA damage and increased replication stress, leading to cell cycle arrest and apoptosis. These results suggest that DHX9 inhibitors are a novel treatment modality for patients with defective DNA damage repair pathways such as dMMR and/or BRCA mutations.
 Citation Format: Serena J Silver, Maureen M. Lynes, Brian A. Sparling, Jennifer Castro, Matthew H. Daniels, Sunaina Pai, Sophie A. Shen, David Brennan, Hyelee Lee, Gordon J. Lockbaum, Kevin Knockenhauer, Deepali Gotur, Simina Grigoriu, Shihua Yao, Shane Buker, Monique Laidlaw, Jie Wu, Stephen J. Blakemore, P. Ann Boriack-Sjodin, Kenneth W. Duncan, Jason A. Sager, Robert A. Copeland. RNA-modifying enzyme inhibitors as synthetic lethal cancer therapeutics [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30abe68091edce25118bc01153b25b2a691e0c6b" target='_blank'>
              Abstract IA024: RNA-modifying enzyme inhibitors as synthetic lethal cancer therapeutics
              </a>
            </td>
          <td>
            Serena J. Silver, Maureen M Lynes, Brian A. Sparling, Jennifer Castro, Matthew H. Daniels, Sunaina Pai, Sophie A. Shen, David Brennan, Hyelee Lee, Gordon J. Lockbaum, Kevin Knockenhauer, D. Gotur, Simina Grigoriu, Shihua Yao, Shane M. Buker, Monique Laidlaw, Jie Wu, Stephen J. Blakemore, P. Boriack-Sjodin, K. Duncan, Jason A. Sager, Robert A. Copeland
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Adenocarcinoma of the lung (LUAD) is a common and highly lethal disease. Clinical grading of disease strongly predicts recurrence and survival after surgery and is determined by morphological assessment of histological growth patterns in resected tumours. The molecular basis of growth pattern is poorly understood at present, as are the mechanisms linking growth pattern to recurrence and death. Interestingly, the two archetypal lethal morphologies, solid and micropapillary patterns, are characterised by their biphasic appearance. Both have an epithelial fraction which is in direct stromal contact, and a fraction which is not. This morphological variance seems likely to represent plasticity, and to be causally linked to mechanisms of virulence. To investigate the gene expression changes related to growth pattern both intra- and intertumoral, we applied spatial transcriptomics (Nanostring GeoMx DSP) to tissue microarray specimens of primary resected human lung adenocarcinoma. Using a variety of region-of-interest (ROI) selection strategies, we sampled 160 pure epithelial ROIs across 7 distinct morphological features of LUAD from 51 patients. Analyses of gene expression reveal fundamental trajectories connecting growth patterns, and crucial modes of plasticity which underly high-risk morphologies. These modes suggest mechanisms for the origins of growth pattern and mechanisms of virulence. Our work highlights dramatic divergence in gene expression programmes between highly lethal but morphologically diverse modes of tumour growth. Furthermore, it provides an explanation for how microscopically localised hypoxia in the primary tumour helps to establish and maintain survival strategies which ultimately determine morphology-specific mechanisms of tumour metastasis, suggesting new therapeutic vulnerabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a08117cd59bda92ca8c6af39ca3279ca1ec800" target='_blank'>
              Spatial resolution of transcriptomic plasticity states underpinning lethal morphologies in lung adenocarcinoma
              </a>
            </td>
          <td>
            Hannah L Williams, Nicolas Poulain, Ian Powley, Silvia Martinelli, R. Bielik, H. Leslie, Colin Nixon, C. Wilson, M. Sereno, Zhangyi He, Leah Officer-Jones, Fiona Ballantyne, Rachel Pennie, C. Wood, David Y Lewis, Nigel B Jamieson, J. Le Quesne
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="
 Aneuploidy – loss or gain of whole chromosomes or chromosome arms, is rare and poorly tolerated in normal cells but occurs in ∼90% of solid tumors; however, the mechanisms through which specific aneuploidies affect cancer development are unclear. Additionally, generating mammalian models of specific chromosome arm alterations is technically difficult, limiting further study. Cancers have tumor, cell, and tissue type-specific patterns of chromosome arm copy-number alterations that influence tumor evolution and sensitivity to anti-cancer therapies. Squamous cell carcinomas (SCCs) affecting lung, head and neck, esophageal, and cervical tissues have been shown to feature conserved early losses and gains of arms 3p and 3q, respectively, and are associated with relatively few oncogenic drivers and treatment options. These aneuploidies are particularly well established in lung SCC and have been shown to promote tumorigenesis, metastasis, and poor prognosis. To study cancer vulnerabilities associated with chromosome 3 arm aneuploidies, we used CRISPR-Cas9 to delete a 3p copy in human immortalized lung epithelial cell lines—the lung SCC cell-of-origin. A subset of clones duplicated a wild type chromosome 3 copy, transitioning to 3q gain and providing us with independent models for chromosome 3 wild type, 3p loss, and 3q gain. Next, we performed a CRISPRi screen and found that 3q gain cells are more sensitive to knockdown of sterol regulatory element-binding factor 1 (SREBF1)—a master regulator of cholesterol and fatty acid biosynthesis, its co-factor SREBP cleavage activating protein (SCAP), and downstream target HMG-CoA reductase (HMGCR)—the rate-limiting enzyme for cholesterol synthesis. 3q gain cells also showed up-regulated fatty acid metabolites, ceramides, triglycerides, and phosphatidylinositols. After performing a chemical screen, 3q gain cells showed reduced proliferation, increased apoptosis, and DNA damage in response to statins (HMGCR inhibitors). Statins have previously been shown to induce tumor-specific apoptosis and reduced cancer risk, although these studies have associated with variable efficacy and underlying mechanisms. We hypothesize that 3q gain is a biomarker of statin response and sensitivity in squamous cancers. We aim to study the mechanism through which aneuploidy alters lipid metabolism and contributes to cancer. Investigating aneuploidy-induced vulnerabilities will help us understand how cancers exploit aneuploidies to promote tumorigenesis and to elucidate novel and specific therapeutic targets for cancers with shared copy-number profiles.
 Citation Format: Nadja Zhakula-Kostadinova, Sejal Jain, Laura Byron, Matthew L. Meyerson, Alison M. Taylor. Investigating vulnerabilities associated with chromosome arm aneuploidy in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca81f5571b9f264386412ab8dcfed6cabd96466" target='_blank'>
              Abstract PR016: Investigating vulnerabilities associated with chromosome arm aneuploidy in cancer
              </a>
            </td>
          <td>
            Nadja Zhakula-Kostadinova, Sejal Jain, Laura Byron, Matthew L. Meyerson, Alison M. Taylor
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gastric cancer, the fifth most prevalent cancer worldwide, is often diagnosed in advanced stages with limited treatment options. Examining the tumor microenvironment (TME) and its metabolic reprogramming can provide insights for better diagnosis and treatment. This study investigates the link between TME factors and metabolic activity in gastric cancer using bulk and single-cell RNA-sequencing data. We identified two molecular subtypes in gastric cancer by analyzing the distinct expression patterns of 81 prognostic genes related to the TME and metabolism, which exhibited significant protein-level interactions. The high-risk subtype had increased stromal content, fibroblast and M2 macrophage infiltration, elevated glycosaminoglycans/glycosphingolipids biosynthesis, and fat metabolism, along with advanced clinicopathological features. It also exhibited low mutation rates and microsatellite instability, associating it with the mesenchymal phenotype. In contrast, the low-risk group showed higher tumor content and upregulated protein and sugar metabolism. We identified a 15-gene prognostic signature representing these characteristics, including CPVL, KYNU, CD36, and GPX3, strongly correlated with M2 macrophages, validated through single-cell analysis and an internal cohort. Despite resistance to immunotherapy, the high-risk group showed sensitivity to molecular targeted agents directed at IGF-1R (BMS-754807) and the PI3K-mTOR pathways (AZD8186, AZD8055). We experimentally validated these promising drugs for their inhibitory effects on MKN45 and MKN28 gastric cells. This study unveils the intricate interplay between TME and metabolic pathways in gastric cancer, offering potential for enhanced diagnosis, patient stratification, and personalized treatment. Understanding molecular features in each subtype enriches our comprehension of gastric cancer heterogeneity and potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b9428220090a4179bb1f023968e0100b63f3ef4" target='_blank'>
              Dissecting gastric cancer heterogeneity and exploring therapeutic strategies using bulk and single-cell transcriptomic analysis and experimental validation of tumor microenvironment and metabolic interplay
              </a>
            </td>
          <td>
            Xiantao Lin, Ping Yang, MingKun Wang, Xiuting Huang, Baiyao Wang, Chengcong Chen, Anan Xu, Jiazuo Cai, Muhammad Khan, Sha Liu, Jie Lin
          </td>
          <td>2024-06-19</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Hormone therapy resistance and the ensuing aggressive progression of tumors present a significant clinical challenge. However, the mechanisms underlying the induction of tumor malignancy by hormone inhibition remain poorly understood. Here we show that Drosophila malignant epithelial tumors exhibit a comparable decrease in ecdysone signaling, the primary steroid hormone pathway in Drosophila. Furthermore, we find that ectopic expression of the nuclear receptor E75 partially mimics ecdysone signaling inhibition, specifically promoting the malignant transformation of benign tumors by integrating the Hippo and Notch signaling pathways. Genome-wide DNA binding profiles and biochemistry data reveal that E75 not only physically interacts with the transcription factors of both Hippo and Notch pathways, but also exhibits widespread co-binding to their target genes, thus contributing to tumor malignancy. Moreover, we validate these findings by demonstrating that depletion of NR1D2, the mammalian equivalent of E75, inhibits the activation of Hippo and Notch target genes, impeding glioblastoma progression in vivo. In summary, our study unveils a previously unrecognized mechanism by which hormone inhibition promotes tumor malignant transformation, while providing a conserved mechanistic understanding of how Hippo and Notch pathways are integrated by the oncogene E75/NR1D2 during tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e08a2b2600760e6a58cf025d5240cb408b11aae5" target='_blank'>
              Nuclear receptor E75/NR1D2 drives tumor malignant transformation by integrating Hippo and Notch pathways
              </a>
            </td>
          <td>
            Xianping Wang, Yifan Guo, Peng Lin, Min Yu, Sha Song, Wenyan Xu, D. Kong, Yin Wang, Yanxiao Zhang, Fei Lu, Qi Xie, Xianjue Ma
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Deficiency of homologous recombination (HR)-mediated DNA repair occurs through genetic or epigenetic inactivation of the BRCA1 and BRCA2 (BRCA1/2) genes. HR-deficiency also provides unique opportunities for targeted therapy, through the mechanism of synthetic lethality, as exemplified by the hypersensitivity of BRCA1/2-mutated tumors to PARP inhibitors (PARPi). To date, several PARP inhibitors have been approved for clinical use. The promising clinical response of patients with germline BRCA1/2-mutations prompted the use of PARPi for patients with somatic BRCA1/2 mutations as well. In addition, these results extended the use of PARPi for various types of ovarian, breast, pancreatic, and prostate tumors with HR defects. PARPi resistance is emerging as the major obstacle to clinical effectiveness in patients with HR-deficient tumors. PARPi resistance results from several independent mechanisms, leading to the restoration of Homologous Recombination and/or Replication Fork stabilization. The absence of alternative options for patients with tumors with innate or acquired resistance underlines the urgency to develop additional therapeutics. More recent studies have identified new synthetic lethal opportunities for BRCA1/2 deficient tumors. Knockout of the genes for POLQ (DNA polymerase theta) or for USP1 (Ubiquitin Specific Protease 1) results in killing of these tumors, and inhibitors of these enzymes are emerging as promising agents for HR-deficient tumors. POLQ and USP1 expression is particularly high in subtypes of breast and ovarian tumors with defects in HR. As a result, POLQi or USP1i in HR-deficient tumors induces cell death. POLQi or USP1i can synergize with PARPi in killing HR-deficient tumors or PARPi-resistant tumors. BRCA1/2 deficient tumors have an increase in single strand DNA gaps (ssGAPs) near replication forks. Treatment with PARPi results in an increase in the size and number for these ssGAPs, ultimately leading to DSBs and cell death. Interestingly, PARPi resistance results, at least in part, from the ability of resistant cells to close the ssGAPs through enhanced POLQ and USP1 activity, thereby providing a rationale for the combination of PARP1, POLQi, and USP1 in HR-deficient cancers. USP1 inhibition was synergistic with PARP and POLQ inhibition in BRCA1-mutant cells, with enhanced ssDNA gap accumulation. Moreover, this synergy was observed in a set of patient-derived ovarian cancer organoids (PDOs), thus confirming the sensitivity of BRCA1-deficient cells to inhibition by these agents. The accumulation of ssDNA gaps after treatment with a USP1i, PARP, or POLQi correlated with the sensitivity to these drugs in all models tested. Ovarian cancer PDOs provide a powerful tool for detecting drug synergy and for rapid in vitro sensitivity testing. The detection of ssDNA gap accumulation may be a useful predictive biomarker for response to DNA Repair inhibitors as monotherapy or in combination in ongoing clinical trials.
 Citation Format: Alan D. D'Andrea. Single strand DNA GAP accumulation as a functional biomarker for DNA Repair Inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c9361e88294fef127d64e67bfd18dda9e6b654" target='_blank'>
              Abstract IA009: Single strand DNA GAP accumulation as a functional biomarker for DNA Repair Inhibitors
              </a>
            </td>
          <td>
            Alan D. D'Andrea
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Non-heritable, phenotypic drug resistance toward antibiotics challenges antibiotic therapies. Characteristics of such phenotypic resistance have implications for the evolution of heritable resistance. Diverse forms of phenotypic resistance have been described, but phenotypic resistance characteristics remain less explored than genetic resistance. Here, we add novel combinations of single-cell characteristics of phenotypic resistant E. coli cells and compare those to characteristics of susceptible cells of the parental population by exposure to different levels of recurrent ampicillin antibiotic. Contrasting expectations, we did not find commonly described characteristics of phenotypic resistant cells that arrest growth or near growth. We find that under ampicillin exposure, phenotypic resistant cells reduced their growth rate by about 50% compared to growth rates prior to antibiotic exposure. The growth reduction is a delayed alteration to antibiotic exposure, suggesting an induced response and not a stochastic switch or caused by a predetermined state as frequently described. Phenotypic resistant cells exhibiting constant slowed growth survived best under ampicillin exposure and, contrary to expectations, not only fast-growing cells suffered high mortality triggered by ampicillin but also growth-arrested cells. Our findings support diverse modes of phenotypic resistance, and we revealed resistant cell characteristics that have been associated with enhanced genetically fixed resistance evolution, which supports claims of an underappreciated role of phenotypic resistant cells toward genetic resistance evolution. A better understanding of phenotypic resistance will benefit combatting genetic resistance by developing and engulfing effective anti-phenotypic resistance strategies. IMPORTANCE Antibiotic resistance is a major challenge for modern medicine. Aside from genetic resistance to antibiotics, phenotypic resistance that is not heritable might play a crucial role for the evolution of antibiotic resistance. Using a highly controlled microfluidic system, we characterize single cells under recurrent exposure to antibiotics. Fluctuating antibiotic exposure is likely experienced under common antibiotic therapies. These phenotypic resistant cell characteristics differ from previously described phenotypic resistance, highlighting the diversity of modes of resistance. The phenotypic characteristics of resistant cells we identify also imply that such cells might provide a stepping stone toward genetic resistance, thereby causing treatment failure. Antibiotic resistance is a major challenge for modern medicine. Aside from genetic resistance to antibiotics, phenotypic resistance that is not heritable might play a crucial role for the evolution of antibiotic resistance. Using a highly controlled microfluidic system, we characterize single cells under recurrent exposure to antibiotics. Fluctuating antibiotic exposure is likely experienced under common antibiotic therapies. These phenotypic resistant cell characteristics differ from previously described phenotypic resistance, highlighting the diversity of modes of resistance. The phenotypic characteristics of resistant cells we identify also imply that such cells might provide a stepping stone toward genetic resistance, thereby causing treatment failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aacb6bdfb65cb5e5f9765ec92a6f4bb1379df93" target='_blank'>
              Phenotypic resistant single-cell characteristics under recurring ampicillin antibiotic exposure in Escherichia coli
              </a>
            </td>
          <td>
            Silvia Kollerová, Lionel Jouvet, Julia Smelková, Sara Zunk-Parras, A. Rodríguez-Rojas, Ulrich K. Steiner
          </td>
          <td>2024-06-26</td>
          <td>mSystems</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="RNAs undergo a complex choreography of metabolic processes in human cells that are regulated by thousands of RNA-associated proteins. While the effects of individual RNA-associated proteins on RNA metabolism have been extensively characterized, the full complement of regulators for most RNA metabolic events remain unknown. Here we present a massively parallel RNA-linked CRISPR (ReLiC) screening approach to measure the responses of diverse RNA metabolic events to knockout of 2,092 human genes encoding all known RNA-associated proteins. ReLiC screens highlight modular interactions between gene networks regulating splicing, translation, and decay of mRNAs. When combined with biochemical fractionation of polysomes, ReLiC reveals striking pathway-specific coupling between growth fitness and mRNA translation. Perturbing different components of the translation and proteostasis machineries have distinct effects on ribosome occupancy, while perturbing mRNA transcription leaves ribosome occupancy largely intact. Isoform-selective ReLiC screens capture differential regulation of intron retention and exon skipping by SF3b complex subunits. Chemogenomic screens using ReLiC decipher translational regulators upstream of mRNA decay and uncover a role for the ribosome collision sensor GCN1 during treatment with the anti-leukemic drug homoharringtonine. Our work demonstrates ReLiC as a versatile platform for discovering and dissecting regulatory principles of human RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d61d936925a83dd1c8b255f5e6a3f00dce8861" target='_blank'>
              Decoding RNA Metabolism by RNA-linked CRISPR Screening in Human Cells
              </a>
            </td>
          <td>
            Patrick J. Nugent, Heungwon Park, Cynthia L. Wladyka, Katharine Y. Chen, Christine Bynum, Grace Quarterman, Andrew C. Hsieh, A. Subramaniam
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Aberrant epigenetic regulation is a hallmark of Diffuse Midline Glioma (DMG), an incurable pediatric brain tumor. The H3K27M driver histone mutation leads to transcriptional dysregulation, indicating that targeting the epigenome and transcription may be key therapeutic strategies against this highly aggressive cancer. One such target is the Facilitates Chromatin Transcription (FACT) histone chaperone. We found FACT to be enriched at developmental gene promoters, coinciding with regions of open chromatin and binding motifs of core DMG regulatory transcription factors. Furthermore, FACT interacted and co-localized with the Bromodomain and Extra-Terminal Domain (BET) protein BRD4 at promoters and enhancers, suggesting functional cooperation between FACT and BRD4 in DMG. In vitro, a combinatorial therapeutic approach using the FACT inhibitor CBL0137, coupled with BET inhibition revealed potent and synergistic cytotoxicity across a range of DMG cultures, with H3K27M-mutant cells demonstrating heightened sensitivity. These results were recapitulated in vivo, significantly extending survival in three independent orthotopic PDX models of DMG. Mechanistically, we show that CBL0137 treatment decreased chromatin accessibility, synergizing with BET inhibition to disrupt transcription, silencing several key oncogenes including MYC, PDGFRA and MDM4, as well as causing alterations to the splicing landscape. Combined, these data highlight the therapeutic promise of simultaneously targeting FACT and BRD4 in DMG, proposing a novel strategy for combating this devastating pediatric brain tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6273ba39a45e304340c6436c0631345d003e75f3" target='_blank'>
              Chromatin remodeling with combined FACT and BET inhibition disrupts oncogenic transcription in Diffuse Midline Glioma
              </a>
            </td>
          <td>
            Holly Holliday, Aaminah Khan, A. Ehteda, Samuel E. Ross, Nisitha Jayatilleke, A. Gopalakrishnan, Xinyi Guo, Jie Liu, Hieu Nguyen, Alvin Lee, Yolanda Colino Sanguino, Marcel E. Dinger, C. Mayoh, Benjamin Rayner, M. Tsoli, David S. Ziegler
          </td>
          <td>2024-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Upregulation of the Wilms' tumour 1 (WT1) gene is common in acute myeloid leukaemia (AML) and is associated with poor prognosis. WT1 generates 12 primary transcripts through different translation initiation sites and alternative splicing. The short WT1 transcripts express abundantly in primary leukaemia samples. We observed that overexpression of short WT1 transcripts lacking exon 5 with and without the KTS motif (sWT1+/- and sWT1-/-) led to reduced cell growth. However, only sWT1+/- overexpression resulted in decreased CD71 expression, G1 arrest, and cytarabine resistance. Primary AML patient cells with low CD71 expression exhibit resistance to cytarabine, suggesting that CD71 may serve as a potential biomarker for chemotherapy. RNAseq differential expressed gene analysis identified two transcription factors, HOXA3 and GATA2, that are specifically upregulated in sWT1+/- cells, whereas CDKN1A is upregulated in sWT1-/- cells. Overexpression of either HOXA3 or GATA2 reproduced the effects of sWT1+/-, including decreased cell growth, G1 arrest, reduced CD71 expression and cytarabine resistance. HOXA3 expression correlates with chemotherapy response and overall survival in NPM1 mutation-negative leukaemia specimens. Overexpression of HOXA3 leads to drug resistance against a broad spectrum of chemotherapeutic agents. Our results suggest that WT1 regulates cell proliferation and drug sensitivity in an isoform-specific manner.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/747999aa3c419c4737033b97641dd45130ab9db4" target='_blank'>
              Upregulation of HOXA3 by isoform-specific Wilms tumour 1 drives chemotherapy resistance in acute myeloid leukaemia.
              </a>
            </td>
          <td>
            Basil Allen, Lindsey Savoy, P. Ryabinin, D. Bottomly, Reid Chen, Bonnie Goff, Anthony Wang, Shannon K McWeeney, Haijiao Zhang
          </td>
          <td>2024-06-12</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Due to the heterogeneity of cancer, precision medicine has been a major challenge for cancer treatment. Determining medication regimens based on patient genotypes has become a research hotspot in cancer genomics. In this study, we aim to identify key biomarkers for targeted therapies based on single nucleotide variants (SNVs) and copy number variants (CNVs) of genes. The experiment is carried out on 7 cancers on the Encyclopedia of Cancer Cell Lines (CCLE) dataset. Considering the high mutability of driver genes which result in abundant mutated samples, the effect of data sparsity can be eliminated to a large extent. Therefore, we focus on discovering the relationship between driver mutation patterns and three measures of drug response, namely area under the curve (AUC), half maximal effective concentration (EC50), and log2-fold change (LFC). First, multiple statistical methods are applied to assess the significance of difference in drug response between sample groups. Next, for each driver gene, we analyze the extent to which its mutations can affect drug response. Based on the results of multiple hypothesis tests and correlation analyses, our main findings include the validation of several known drug response biomarkers such as BRAF, NRAS, MAP2K1, MAP2K2, and CDKN2A, as well as genes with huge potential to infer drug responses. It is worth emphasizing that we identify a list of genes including SALL4, B2M, BAP1, CCDC6, ERBB4, FOXA1, GRIN2A, and PTPRT, whose impact on drug response spans multiple cancers and should be prioritized as key biomarkers for targeted therapies. Furthermore, based on the statistical p-values and correlation coefficients, we construct gene-drug sensitivity maps for cancer drug recommendation. In this work, we show that driver mutation patterns could be used to tailor therapeutics for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa993499162c59f7b76c5c6ffab6d2e07409c25b" target='_blank'>
              Single nucleotide and copy number variants of cancer driver genes inform drug response in multiple cancers
              </a>
            </td>
          <td>
            Zeyuan Wang, Hong Gu, Pan Qin, Jia Wang
          </td>
          <td>2024-07-31</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Studies on somatic mutations in cloned animals have revealed slight genetic variances between clones and their originals, but have yet to identify the precise effects of these differences within the organism. Somatic mutations contribute to aging and are implicated in tumor development and other age-related diseases. Thus, we compared whole genome sequencing data from an original dog with that of cloned dogs, identifying candidate somatic mutations that were disproportionately located within genes previously implicated in aging. The substitutional signature of cloning-specific somatic mutations mirrored the uniform distribution characteristic of the signature associated with human aging. Further analysis of genes revealed significant enrichment of traits associated with body size as well as the molecular mechanisms underlying neuronal function and tumorigenesis. Overall, the somatic mutations found in cloned dogs may indicate a conserved mechanism driving aging across species and a broad spectrum of pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f3d14c854be2c4eda75c0ca77517be9684eaeca" target='_blank'>
              Genome-Wide Characterization of Somatic Mutation Patterns in Cloned Dogs Reveals Implications for Neuronal Function, Tumorigenesis, and Aging
              </a>
            </td>
          <td>
            Seung-Wan Woo, Miju Kim, Dayeon Kang, Y. Choe, Seong-Ju Oh, Are-Sun You, S. Lee, Jaemin Kim
          </td>
          <td>2024-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b139e901b81d232b7592efe9dd25a1948b27ddfe" target='_blank'>
              IκBα kinase inhibitor BAY 11-7082 promotes anti-tumor effect in RAS-driven cancers
              </a>
            </td>
          <td>
            P. Guruvaiah, Romi Gupta
          </td>
          <td>2024-07-09</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Tumor cell heterogeneity in neuroblastoma, a pediatric cancer arising from neural crest-derived progenitor cells, poses a significant clinical challenge. In particular, unlike adrenergic (ADRN) neuroblastoma cells, mesenchymal (MES) cells are resistant to chemotherapy and retinoid therapy and thereby significantly contribute to relapses and treatment failures. Previous research suggested that overexpression or activation of miR-124, a neurogenic microRNA with tumor suppressor activity, can induce the differentiation of retinoic acid-resistant neuroblastoma cells. Leveraging our established screen for miRNA-modulatory small molecules, we validated PP121, a dual inhibitor of tyrosine and phosphoinositide kinases, as a robust inducer of miR-124. A combination of PP121 and BDNF-activating bufalin synergistically arrests proliferation, induces differentiation, and maintains the differentiated state of MES SK-N-AS cells for 8 weeks. RNA-seq and deconvolution analyses revealed a collapse of the ADRN core regulatory circuitry (CRC) and the emergence of novel CRCs associated with chromaffin cells and Schwann cell precursors. Using a similar protocol, we differentiated and maintained MES neuroblastoma GI-ME-N and SH-EP cell lines, as well as glioblastoma LN-229 and U-251 cell lines, for over 16 weeks. In conclusion, our novel protocol suggests a promising treatment for therapy-resistant cancers of the nervous system. Moreover, these long-lived, differentiated cells provide valuable models for studying mechanisms underlying differentiation, maturation, and senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff00c391d2a03bfab2bb7b850915ae4155feb81f" target='_blank'>
              Novel miRNA-inducing drugs enable differentiation of retinoic acid-resistant neuroblastoma cells
              </a>
            </td>
          <td>
            Lien D. Nguyen, Satyaki Sengupta, Kevin Cho, Alexander E Floru, Rani E. George, Anna M. Krichevsky
          </td>
          <td>2024-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79dbe68de75bb03fc187c37800cd2699920eb720" target='_blank'>
              Genomic and immune determinants of resistance to daratumumab-based therapy in relapsed refractory multiple myeloma
              </a>
            </td>
          <td>
            B. Ziccheddu, C. Giannotta, M. D'Agostino, G. Bertuglia, E. Saraci, S. Oliva, E. Genuardi, M. Papadimitriou, B. Diamond, P. Corradini, D. Coffey, O. Landgren, N. Bolli, B. Bruno, M. Boccadoro, Massimo Massaia, Francesco Maura, A. Larocca
          </td>
          <td>2024-07-19</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/354d0cada1a73a94d156b70d215246eda0c2a524" target='_blank'>
              T cell dysfunction and therapeutic intervention in cancer.
              </a>
            </td>
          <td>
            Caitlin C. Zebley, D. Zehn, Stephen Gottshalk, Hongbo Chi
          </td>
          <td>2024-07-18</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) possess a significant ability to renew themselves, which gives them a strong capacity to form tumors and expand to encompass additional body areas. In addition, they possess inherent resistance to chemotherapy and radiation therapies used to treat many forms of cancer. Scientists have focused on investigating the signaling pathways that are highly linked to the ability of CSCs to renew themselves and maintain their stem cell properties. The pathways encompassed are Notch, Wnt/β-catenin, hedgehog, STAT3, NF-κB, PI-3K/Akt/mTOR, sirtuin, ALDH, MDM2, and ROS. Recent studies indicate that directing efforts towards CSC cells is essential in eradicating the overall cancer cell population and reducing the likelihood of tumor metastasis. As our comprehension of the mechanisms that stimulate CSC activity, growth, and resistance to chemotherapy advances, the discovery of therapeutic drugs specifically targeting CSCs, such as small-molecule compounds, holds the potential to revolutionize cancer therapy. This review article examines and analyzes the novel anti-CSC compounds that have demonstrated effective and selective targeting of pathways associated with the renewal and stemness of CSCs. We also discussed their special drug metabolism and absorption mechanisms. CSCs have been the subject of much study in cancer biology. As a possible treatment for malignancies, small-molecule drugs that target CSCs are gaining more and more attention. This article provides a comprehensive review of the current state of key small-molecule compounds, summarizes their recent developments, and anticipates the future discovery of even more potent and targeted compounds, opening up new avenues for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c2ec47520bdf5746027b1d46298064477df151" target='_blank'>
              Novel Anti-Cancer Stem Cell Compounds: A Comprehensive Review
              </a>
            </td>
          <td>
            Shanchun Guo, Shilong Zheng, Mingli Liu, Guangdi Wang
          </td>
          <td>2024-08-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c785685c384edfe64958536f2b96ef40bece9a9d" target='_blank'>
              Let’s make it personal: CRISPR tools in manipulating cell death pathways for cancer treatment
              </a>
            </td>
          <td>
            Mobina Bayat, J. S. Nahand
          </td>
          <td>2024-07-29</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) represents the most heterogeneous and aggressive subtype of breast carcinomas, characterized by the absence of clinical biomarkers (ERα, PR, and HER2) and the lack of targeted therapies. In this regard, several clinical trials have consistently failed to effectively stratify patients and identify specific treatments that elicit substantial responses. This study aims to pinpoint biomarkers expressed exclusively in the basal mammary epithelial compartment, facilitating a refined subclassification of this breast cancer subtype. Using computational analyses of single-cell RNA sequencing data, we have identified a list of genes associated with basal identity (BC-markers). Histological validation in 137 human samples has enabled us to categorize TNBC patients into BC-positive and BC-negative TNBC subtypes. Significantly, the presence of these markers correlates with a poorer prognosis in TNBC patients. Functional analyses have revealed a pivotal role for TAGLN in cell migration, likely influencing tumor aggressiveness. Further, we discovered that BC-marker expression is associated with the mesenchymal phenotype and increased sensitivity to the tyrosine kinase inhibitor dasatinib, particularly in BC-positive TNBC, suggesting novel therapeutic avenues. In our study, TAGLN emerged as a potential predictive biomarker for dasatinib responsiveness, offering new directions for personalized therapy for TNBC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37a72d0f9fe14dc4dcdef733872b863c9bfc2b5a" target='_blank'>
              Discovery and evaluation of novel biomarkers reveal dasatinib as a potential treatment for a specific subtype of Triple-Negative Breast Cancer
              </a>
            </td>
          <td>
            D. Ortega-Álvarez, D. Tébar-García, M. Casado-Peláez, E. Castillo-Agea, J. Balibrea-Rull, D. Olivares-Osuna, D. Pérez-Parra, C. Guardia, E. Musulén, E. Vinuesa-Pitarch, E. Sánchez-López, F. Postigo-Corrales, E. Ballana, A. Martínez-Cardús, M. Margeli, M. Esteller, PL Fernández-Ruiz, G. Luengo-Gil, E. Mereu, EM Galán-Moya, V. Rodilla
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The use of tyrosine kinase inhibitors, such as imatinib, against the chronic myeloid leukemia (CML)-causing kinase BCR::ABL1 has become the model for successful targeted therapy. Nevertheless, drug resistance remains a clinical problem. Analysis of genome-wide expression and genetic aberrations of an in vitro imatinib-resistant CML cell line revealed downregulation of Bruton's tyrosine kinase (BTK), predominantly associated with B cell malignancies, and a novel BTK kinase domain variant in imatinib resistance. This raised the question of the role of BTK in imatinib-resistant CML. In the present study, BTK downregulation and the presence of the BTK variant c.1699_1700delinsAG p.(Glu567Arg) were confirmed in imatinib resistance in vitro. Similarly, BTK inhibition or small interfering RNA-mediated BTK knockdown reduced imatinib susceptibility by 84 and 71%, respectively. BTK overexpression was detrimental to CML cells, as proliferation was significantly reduced by 20.5% under imatinib treatment. In addition, BTK rescue in imatinib-resistant cells restored imatinib sensitivity. The presence of the BTK p.(Glu567Arg) variant increased cell numbers (57%) and proliferation (37%) under imatinib exposure. These data demonstrate that BTK is important for the development of imatinib resistance in CML: Its presence increased drug response, while its absence promotes imatinib resistance. Moreover, the BTK p.(Glu567Arg) variant abrogates imatinib sensitivity. These findings demonstrate a context-dependent role for BTK as an oncogene in B cell malignancies, but as a tumor suppressor in other neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/788e8517fbc0ee9685d2742cf678b90e5864d99a" target='_blank'>
              BTK acts as a modulator of the response to imatinib in chronic myeloid leukemia.
              </a>
            </td>
          <td>
            Lena Schmidlechner, Inga Nagel, I. Vater, Ingolf Cascorbi, Meike Kaehler
          </td>
          <td>2024-07-04</td>
          <td>Oncology letters</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Multidrug resistance (MDR) frequently occurs during cancer therapy and remains a major obstacle for the cure of most cancers(1, 2). Drug resistance could exist intrinsically or be acquired by drug treatment(3–5), yet factors that regulate the resistance remain elusive. Here, we show that most anticancer drugs damage neosynthesized proteins prior to reaching their canonic targets and elicit profound cytotoxicity, which is largely compensated by protein damage response (PDR). We demonstrate that the PDR includes damage recognition and clearance that are mainly mediated by ubiquitin and proteasome systems, although some other factors, including cellular ATP levels and proliferation status, are also involved. We show that cancer stem cells (CSCs), which have lower protein synthesis, and drug resistance acquired cells (DRAC), which have higher proteasome activity, are more resistant than other cells. We further demonstrate that ATP promotes protein synthesis and suppresses proteasome activity, thus, increasing mitochondrial ATP production by PDK1 inhibition and using proteasome inhibitor to block protein damage clearance render CSCs and DRACs more vulnerable to anticancer drugs. Thus, patients with drug-resistant cancers and treatment-naïve patients with low ATP levels and/or high proteasome activity can be identified and subtyped, and therapies containing PDK1-I and/or proteasome-I may be effective options for these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e9626795c580f97c2519d1342441e9145cbcfc" target='_blank'>
              A critical role of protein damage response in mediating cancer drug resistance
              </a>
            </td>
          <td>
            Fangyuan Shao, Jie Li, Hao Xiao, Ling Li, Bo Li, YuJun Chen, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Yuzhong Peng, Yujian Huang, Lijian Wang, Yanxia Shi, Nan Shao, Kai Miao, Changhua Zhang, Ying Lin, Jun Yan, Kin Yip Tam, Xiaoling Xu, Chu-Xia Deng
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Mutations in polymerases Pold1 and Pole exonuclease domains in humans are associated with increased cancer incidence, elevated tumor mutation burden (TMB) and response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond. Here we generated Pold1 and Pole proofreading mutator mice and show that ICB treatment of mice with high TMB tumors did not improve survival as only a subset of tumors responded. Similarly, introducing the mutator alleles into mice with Kras/p53 lung cancer did not improve survival, however, passaging mutator tumor cells in vitro without immune editing caused rejection in immune-competent hosts, demonstrating the efficiency by which cells with antigenic mutations are eliminated. Finally, ICB treatment of mutator mice earlier, before observable tumors delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the use of ICB in individuals at high risk for cancer prevention. Highlights Germline somatic and conditional Pold1 and Pole exonuclease domain mutations in mice produce a mutator phenotype. Spontaneous cancers arise in mutator mice that have genomic features comparable to human tumors with these mutations. ICB treatment of mutator mice with tumors did not improve survival as only a subset of tumors respond. Introduction of the mutator alleles into an autochthonous mouse lung cancer model also did not produce immunogenic tumors, whereas passaging mutator tumor cells in vitro caused immune rejection indicating efficient selection against antigenic mutations in vivo. Prophylactic ICB treatment delayed cancer onset, improved survival, and selected for tumors with no aneuploidy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e6ba5a717c7dd3412aa2d3806c5dc666fb03f9" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer and Extends Survival in Murine DNA Polymerase Mutator Syndromes
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. C. Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="PLK1 inhibitors are emerging anti-cancer agents being tested in monotherapy and combination therapies for various cancers. Although PLK1 inhibition in experimental models shows potent antitumor effects, translation to the clinic has been hampered by low antitumor activity and tumor relapse. Here, we report the identification of mitochondrial protein signatures that determine sensitivity to approaches targeting PLK1 in human melanoma cell lines. In response to PLK1 inhibition or gene silencing, resistant cells adopt a pro-inflammatory and dedifferentiated phenotype, while sensitive cells engage apoptosis. Mitochondrial DNA depletion and silencing of the ABCD1 transporter sensitize cells to PLK1 inhibition and attenuate the associated pro-inflammatory response. We also found that non-selective inhibitors of the p90 ribosomal S6 kinase (RSK) exert their anti-proliferative and pro-inflammatory effects via PLK1 inhibition. This work reveals overlooked impacts of PLK1 on phenotype switching and suggests that mitochondrial precision medicine can help improve response to targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be11cedd7b84b462b11820d3c974a0336e8915ba" target='_blank'>
              Mitochondrial Signatures Shape Phenotype Switching and Apoptosis in Response to PLK1 and RSK Inhibitors in Melanoma
              </a>
            </td>
          <td>
            Émilie Lavallée, Maëline Roulet-Matton, Viviane Giang, Roxana Cardona Hurtado, Dominic Chaput, Simon-Pierre Gravel
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Aneuploidy is an abnormal chromosome composition and a general hallmark of human cancer. Aneuploidy causes detrimental cellular stresses, but cancer cells evolve to cope with these stresses. Consequently, targeting such mitigation mechanisms is a promising potential therapeutic strategy. As an abnormal dosage of gene products from altered chromosomes can cause RNA and proteotoxic stress, dosage compensation (DC) of imbalanced gene products was reported to mitigate these stresses in aneuploid cells. However, the mechanisms that regulate DC remain elusive. To address these mechanisms, we focused on the role(s) of stress granules (SGs) and RNA binding proteins (RBPs) in aneuploid cancer cells. Our recent study revealed that aneuploid cancer cells preferentially depend on RNA and protein metabolism, and need to attenuate translation in order to cope with proteotoxic stress (Ippolito & Zerbib et al. bioRxiv 2023). In yeast, it has been previously reported that Ssd1, one of the RBPs localized in P-bodies, is indispensable to tolerating aneuploidy stress. Based on these findings, we set out to explore SGs as potential regulators of gene expression in aneuploid cancer cells. As expected, we revealed that SGs emerged in cells under acute aneuploidy stress induced by chromosome mis-segregation. These results imply that SGs are formed in cells during and right after aneuploidization, potentially as a way to mediate DC and ameliorate aneuploidy-induced cellular stress. To gain further insight into RNA regulation in aneuploid cells, we analyzed the essentiality of RBPs across cancer cell lines. We found that the RNA-editing enzyme ADAR, a suppressor of SG formation, is preferentially essential in aneuploid cancer cells. Remarkably, ADAR disruption synergized with acute aneuploidy stress to induce SGs, and ADAR exhibited increased RNA editing activity following chromosome mis-segregation. Our results suggest that SGs and their suppressor ADAR might contribute to ameliorating acute aneuploidy-induced stress. We are currently investigating the target RNAs regulated by SGs and ADAR and examining their functional role(s) in DC, with the goal of uncovering potential synthetic lethalities associated with aneuploidy in cancer cells.
 Citation Format: Hajime Okada, Eran Sdeor, Miriam Karmon, Erez Levanon, Uri Ben-David. Targeting mechanisms of dosage compensation to selectively kill aneuploid cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d65eea6dda42eca3bae9912c1560d589405d46e4" target='_blank'>
              Abstract PR008: Targeting mechanisms of dosage compensation to selectively kill aneuploid cancer cells
              </a>
            </td>
          <td>
            Hajime Okada, Eran Sdeor, Miriam Karmon, E. Levanon, Uri Ben-David
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Methods for personalizing medical treatment are the focal point of contemporary biomedical research. In cancer care, we can analyze the effects of therapies at the level of individual cells. Quantitative characterization of treatment efficacy and evaluation of why some individuals respond to specific regimens, whereas others do not, requires additional approaches to genetic sequencing at single time points. Methods for the continuous analysis of changes in phenotype, such as in vivo and ex vivo morphology and motion tracking of cellular proteins and organelles, over time-frames spanning the minute-hour scales, can provide important insights into patient treatment options. Despite improvements in the diagnosis and therapy of many types of breast cancer (BC), many aggressive forms, such as receptor triple-negative cancers, are associated with the worst patient outcomes; though initially effective in reducing tumor burden for some patients, acquired resistance to cytotoxic chemotherapy is almost universal, and there is no rationale for identifying intrinsically drug-resistant and drug-sensitive patient populations before initiating therapy. During cell division, the receptor triple-negative MDA-MB-231 mitotic spindles are the largest in comparison to other BC cell lines. Many of the MDA-MB-231 spindles exhibit rapid lateral twisting during metaphase, which remains unaffected by knockdown of the oncogene Myc and treatment with inhibitors of the serine/threonine-protein kinase B-Raf and the epidermal growth factor receptor (EGFR), alone or in any combination. In this manuscript, we outline a strategy for the selection of the most optimal tubulin inhibitor based on the ability to affect MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d90f15c7bb309b3d3ce06b26fa0c307848870d30" target='_blank'>
              Mitosis, Cytoskeleton Regulation, and Drug Resistance in Receptor Triple Negative Breast Cancer
              </a>
            </td>
          <td>
            Alexandre Matov
          </td>
          <td>2024-07-26</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>1</td>
        </tr>

        <tr id="Introduction. It has been established that the presence of homologous recombination deficiency in a breast tumor is associated with the effectiveness of treatment. But despite the high chemosensitivity of the tumor to DNA-damaging agents, complete pathological responses to treatment are very rare. And this process may be based on a change in the somatic status of BRCA1, that is, a reversion and return of the wild-type allele occurs and the DNA repair function is restored.Aim. To evaluate changes in the presence of chromosomal aberrations and the expression profile of the main genes of homologous recombination in cell models of breast cancer under the influence of cisplatin and docetaxel.Materials and methods. The study was conducted on breast cancer tumor cell cultures: MCF-7, MDA-MB-231 and MDA-MB-468. A cell model of drug resistance was obtained for two drugs: cisplatin and docetaxel. RNA and DNA were isolated from cell suspension using the RNeasy Plus Mini Kit and QIAamp DNA Mini Kit (Qiagen, Germany), respectively. The expression level of homologous recombination genes was assessed using reverse transcription polymerase chain reaction. To assess the presence of chromosomal aberrations, microarray analysis was performed on DNA chips.Results. Restoration of normal copy number for the BRCA1, CDK12, CHEK1 and RAD51D genes in MCF-7 under the influence of cisplatin was shown. For BRCA2 and PALB2, amplifications were detected. A statistically significant increase in the expression of the BRCA1 (p = 0.04), BRCA2 (p = 0.02), PALB2 (p = 0.01) and RAD51D (p = 0.05) genes was also shown. MDAMB-231 shows that all identified loci with deletions, where the BRCA2, BARD1, CHEK2, PALB2 and RAD54L genes are localized, are restored to normal copy number by cisplatin. The appearance of amplifications was registered for BRCA1, BRIP1, FANCL, RAD51B, PARP1. A similar result was shown for docetaxel. An increase in the expression level is typical for the genes BRCA1 (p = 0.02), BRCA2 (p = 0.02), CHEK2 (p = 0.05), FANCL (p = 0.04), PALB2 (p = 0.05), RAD51C (p = 0.02), PARP1 (p = 0.02), which corresponds to the appearance of amplifications. In the MDA-MB-468 cell culture, an increase in the copy number of only the BRCA1 gene is observed. The effect of docetaxel has no effect on this cell culture. The level of BRCA1 expression increases in direct proportion to the duration of drug action.Conclusion. Thus, the study showed that under the influence of cisplatin, reversion of not only homologous recombination gene mutations, but also other disorders can occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/767d30042e565f8e61dafca9671915ee6d03ecf7" target='_blank'>
              Modulation of homologous recombination gene activity in breast tumor cells in an in vitro model
              </a>
            </td>
          <td>
            M. Tsyganov, A. Frolova, E. A. Kravtsova, I. A. Tsydenova, M. Ibragimova
          </td>
          <td>2024-06-28</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="While the triggers for the metastatic transformation of breast cancer (BC) cells remain unknown, recent evidence suggests that intrinsic cellular metabolism could be a crucial driver of migratory disposition and chemoresistance. Aiming to decode the molecular mechanisms involved in BC cell metabolic maneuvering, we study how a ketomimetic (ketone body-rich, low glucose) medium affects Doxorubicin (DOX) susceptibility and invasive disposition of BC cells. We quantified glycocalyx sialylation and found an inverse correlation with DOX-induced cytotoxicity and DOX internalization. These measurements were coupled with single-cell metabolic imaging, bulk migration studies, along with transcriptomic and metabolomic analyses. Our findings revealed that a ketomimetic medium enhances chemoresistance and invasive disposition of BC cells via two main oncogenic pathways: hypersialylation and lipid synthesis. We propose that the crosstalk between these pathways, juxtaposed at the synthesis of the glycan precursor UDP-GlcNAc, furthers advancement of a metastatic phenotype in BC cells under ketomimetic conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fa6f2b5143747f6467f4d23175994fac910eade" target='_blank'>
              Ketomimetic Nutrients Trigger a Dual Metabolic Defense in Breast Cancer Cells
              </a>
            </td>
          <td>
            Mohini Kamra, Yuan-I Chen, Paula C. Delgado, Erin H. Seeley, S. Seidlits, Hsin-Chin Yeh, Amy Brock, S. Parekh
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4597b2c9168e5b3eb619c4010a4d29dad0a9c28b" target='_blank'>
              mTORC2-driven chromatin cGAS mediates chemoresistance through epigenetic reprogramming in colorectal cancer.
              </a>
            </td>
          <td>
            Guoqing Lv, Qian Wang, Lin Lin, Qiao Ye, Xi Li, Qian Zhou, Xiangzhen Kong, Hongxia Deng, Fuping You, Hebing Chen, Song Wu, Lin Yuan
          </td>
          <td>2024-07-30</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs), defined as RNA molecules exceeding 200 nucleotides in length, have been implicated in the regulation of various biological processes and the progression of tumors. Among them, LINC00518, a recently identified lncRNA encoded by a gene located on chromosome 6p24.3, consists of three exons and is predicted to positively regulate the expression of specific genes. LINC00518 has emerged as a key oncogenic lncRNA in multiple cancer types. It exerts its tumor-promoting effects by modulating the expression of several target genes, primarily through acting as a sponge for microRNAs (miRNAs). Additionally, LINC00518 influences critical signaling pathways, including the Wnt/β-catenin, JAK/STAT, and integrin β3/FAK pathways. Elevated levels of LINC00518 in tumor tissues are associated with increased tumor size, advanced clinical stage, metastasis, and poor survival prognosis. This review provides a comprehensive summary of the genetic characteristics, expression patterns, biological functions, and underlying mechanisms of LINC00518 in human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d93f220783bc0f4834eae797d7fdcb653821aa1b" target='_blank'>
              LINC00518: a key player in tumor progression and clinical outcomes
              </a>
            </td>
          <td>
            Qiang Yi, Gangfeng Zhu, Weijian Zhu, Jiaqi Wang, Xinting Ouyang, Kuan Yang, Jinghua Zhong
          </td>
          <td>2024-07-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Splicing modulation is a promising treatment strategy pursued to date only in splicing-factor mutant cancers; however, its therapeutic potential is poorly understood outside of this context. Like splicing factors, genes encoding components of the cohesin complex are frequently mutated in myeloid malignancies, including 15-20% of myelodysplastic syndromes (MDS) and secondary acute myeloid leukemia (AML), where they are associated with poor outcomes. I will discuss our recent findings identifying cohesin mutations as biomarkers of sensitivity to drugs targeting splicing-factor SF3B1 (H3B-8800 and E-7107) and describe the mechanism by which drug-induced alterations in splicing of DNA repair genes, such as BRCA1 and BRCA2, underlie this sensitivity. We have demonstrated that treatment of cohesin-mutant cells with SF3B1 modulators results in impaired DNA damage response, accumulation of DNA damage, and increased sensitivity to PARP inhibitors and a panel of chemotherapeutic agents in vitro and in vivo, using AML cell lines and patient-derived xenograft models. Furthermore, we identified RAD51 foci formation as a predictive biomarker of sensitivity to SF3B1 splicing modulation alone or followed by sequential treatment with PARP inhibition and chemotherapy, and have identified additional non-cohesin mutant subtypes of MDS/AML and ovarian and breast cancer which are sensitive to this therapeutic approach. Our findings expand the potential therapeutic benefits of SF3B1 splicing modulators to include cohesin-mutant MDS/AML and we propose this as broader strategy for therapeutic targeting of other DNA damage-repair deficient cancers.
 Citation Format: Zuzana Tothova. Therapeutic vulnerabilities of cohesin-mutant myeloid malignancies [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA005.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb2ff30f0b7a6f8251086d6baa4bac84ceb2be63" target='_blank'>
              Abstract IA005: Therapeutic vulnerabilities of cohesin-mutant myeloid malignancies
              </a>
            </td>
          <td>
            Zuzana Tothova
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Membrane-associated tyrosine- and threonine-specific Cdc2-inhibitory kinase (PKMYT1) is a cell cycle regulatory kinase that inhibits CDK1/CyclinB activity, delaying entry into mitosis in tumor cells experiencing replication stress (RS). RS is frequently induced by genetic alterations that drive premature transition from G1 to S phase, promoting genome instability and creating a synthetic lethal (SL) relationship between these specific alterations and PKMYT1 inhibition. This relationship has been demonstrated preclinically and clinically with CCNE1 amplification as well as FBXW7 and PPP2R1A deleterious mutations. These genetic alterations are common across a range of tumors including ovarian, endometrial and colorectal cancers. On this basis, we developed lunresertib (RP-6306), a first-in-class, potent and selective oral PKMYT1 inhibitor currently being investigated in phase I clinical trials (NCT04855656, NCT05147272, NCT05147350). Here we discuss the translation of pre-clinical data to the clinical setting as both monotherapy and mechanistic combinations approaches.
 Citation Format: Michael Zinda. Targeting genome instability in cancer: Translating synthetic lethal biology into the clinic [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2dab79a77a92e7c288ec66eea236365dc8d8a1" target='_blank'>
              Abstract IA010: Targeting genome instability in cancer: Translating synthetic lethal biology into the clinic
              </a>
            </td>
          <td>
            M. Zinda
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cellular senescence has been increasingly recognized as a hallmark of cancer, reflecting its association with aging and inflammation, its role as a response to deregulated proliferation and oncogenic stress, and its induction by cancer therapies. While therapy-induced senescence (TIS) has been linked to resistance, recurrence, metastasis, and normal tissue toxicity, TIS also has the potential to enhance therapy response and stimulate anti-tumor immunity. In this review, we examine the Jekyll and Hyde nature of senescent cells (SnCs), focusing on how their persistence while expressing the senescence-associated secretory phenotype (SASP) modulates the tumor microenvironment through autocrine and paracrine mechanisms. Through the SASP, SnCs can mediate both resistance and response to cancer therapies. To fulfill the unmet potential of cancer immunotherapy, we consider how SnCs may influence tumor inflammation and serve as an antigen source to potentiate anti-tumor immune response. This new perspective suggests treatment approaches based on TIS to enhance immune checkpoint blockade. Finally, we describe strategies for mitigating the detrimental effects of senescence, such as modulating the SASP or targeting SnC persistence, which may enhance the overall benefits of cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52dbb8efb9179d8a3a7870f442fb59ca7f4eb573" target='_blank'>
              Therapy-Induced Cellular Senescence: Potentiating Tumor Elimination or Driving Cancer Resistance and Recurrence?
              </a>
            </td>
          <td>
            Yue Liu, Isabelle Lomeli, Stephen J. Kron
          </td>
          <td>2024-07-30</td>
          <td>Cells</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4781c7d248ebd4c7b1d88e8451d7cdc1606f06e6" target='_blank'>
              Genome-wide enhancer RNA profiling adds molecular links between genetic variation and human cancers
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Bin Li, Jinyu Huang, Ming Zhang, Can Chen, Linyun Fan, Qian-Ying Ma, Chunyi He, Shuo Chen, Yuan Jiang, Yan-Min Li, Caibo Ning, Fu-Wei Zhang, Wenzhuo Wang, Yizhuo Liu, Heng Zhang, Meng Jin, Xiao-Yang Wang, Jinxin Han, Zhen Xiong, Ming Cai, Chaoqun Huang, Xiaojun Yang, Xu Zhu, Ying Zhu, Xiaoping Miao, Shao-Kai Zhang, Yongchang Wei, Jianbo Tian
          </td>
          <td>2024-06-11</td>
          <td>Military Medical Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c24f0b42670eaa7d910971701aacf3a8d5213b" target='_blank'>
              Targeting RBM39 through indisulam induced mis-splicing of mRNA to exert anti-cancer effects in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tongting Ji, Yang Yang, Juanjuan Yu, Hongli Yin, Xinran Chu, Pengju Yang, Ling Xu, Xiaodong Wang, Shaoyan Hu, Yizhen Li, Xiaochen Wu, Wengyuan Liu, Bi Zhou, Wenjuan Wang, Shuqi Zhang, Wei Cheng, Yanling Chen, Lei Shi, Zhiheng Li, Ran Zhuo, Yongping Zhang, Yanfang Tao, Di Wu, Xiao-Lu Li, Zimu Zhang, Jun-Jie Fan, Jian Pan, Jun Lu
          </td>
          <td>2024-07-24</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8e1be4efe22c9fbd98a98ceb3e0939364442527" target='_blank'>
              Genome-wide CRISPR screening identifies tyrosylprotein sulfotransferase-2 as a target for augmenting anti-PD1 efficacy
              </a>
            </td>
          <td>
            Yumi Oh, Sujeong Kim, Yunjae Kim, Hyun Kim, Dongjun Jang, Seungjae Shin, , Jiwon Kim, Sang Eun Lee, Jaeik Oh, Yoojin Yang, Dohee Kim, Hae-Rim Jung, Sangjin Kim, Jihui Kim, K. Min, Beomki Cho, Hoseok Seo, Dohyun Han, Hansoo Park, 
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Cancer is a serious public health issue and cases are rising at a high rate around the world. Altered production of reactive oxygen species (ROS) causes oxidative stress (OS) which plays a vital role in cancer development by disrupting signaling pathways and genomic integrity in the cellular microenvironment. In this study, we reviewed the regulation of noncoding RNAs, histone modifications, and DNA methylation which OS is involved in. These mechanisms promote cancer growth, metastasis, and resistance to chemotherapeutic agents. There is significant potential to improve patient outcomes through the development of customized medications and interventions that precisely address the role of OS in the onset and progression of cancer. Redox-modulating drugs, antioxidant-based therapies, and measures to restore regular cellular activity and OS-modulated signaling pathways are some examples of these strategies. One other hypothesis rationalizes the cancer-suppressing effect of OS, which acts as a two-edged condition that warns against the use of antioxidants for cancer treatment and management. The present study was executed to review the impact of OS on epigenetic machinery, the evolution of metastatic cancer, and how OS mediates cellular signaling. Along with, insights into the potential of targeting OS-mediated mechanisms for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ccd892c72bdf3ad1fab1c5313362d6af4eefd52" target='_blank'>
              Oxidative stress-mediated epigenetic remodeling, metastatic progression and cell signaling in cancer
              </a>
            </td>
          <td>
            A. Phull, Sadia Qamar Arain, Abdul Majid, Humaira Fatima, Madiha Ahmed, Song-Ja Kim
          </td>
          <td>2024-07-15</td>
          <td>Oncologie</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="


 Neddylation (NAE) inhibition, affecting posttranslational protein function and turnover, is a promising therapeutic approach to cancer. We report cytotoxic vulnerability to NAE inhibitors in a subset of glioblastoma (GBM) preclinical models and identify genetic alterations and biological processes underlying differential response.



 GBM DNA sequencing and transcriptomic data were queried for genes associated with response to NAE inhibition; candidates were validated by molecular techniques. Multi-omics and functional assays revealed processes implicated in NAE inhibition response.



 Transcriptomics and shotgun proteomics depict PTEN signaling, DNA replication, and DNA repair pathways as significant differentiators between sensitive and resistant models. Vulnerability to MLN4924, a NAE inhibitor, is associated with elevated S-phase populations, DNA re-replication, and DNA damage. In a panel of GBM models, loss of WT PTEN is associated with resistance to different NAE inhibitors. A NAE Inhibition Response Gene set could segregate the GBM cell lines that are most resistant to MLN4924.



 Loss of WT PTEN is associated with non-sensitivity to three different compounds that inhibit NAE in GBM. A NAE Inhibition Response Gene Set largely consisting of DNA replication genes could segregate GBM cell lines most resistant to NAEi and may be the basis for future development of NAE inhibition signatures of vulnerability and clinical trial enrollment within a precision medicine paradigm.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd754dfee4f9007deb8452cb425bc2a88cc4d9ce" target='_blank'>
              Glioblastoma vulnerability to neddylation inhibition is dependent on PTEN status, and dysregulation of the cell cycle and DNA replication
              </a>
            </td>
          <td>
            B. Taylor, N. Tang, Yue Hao, Matthew Lee, S. Peng, R. Bybee, L. Hartman, K. Garcia-Mansfield, Ritin Sharma, P. Pirrotte, Jianhui Ma, Alison D. Parisian, F. Furnari, Harshil Dhruv, Michael E Berens
          </td>
          <td>2024-06-20</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64dd346087b1d64ab0e5ccbf0c92c3994e5f755a" target='_blank'>
              Characterization and regulation of cell cycle-independent noncanonical gene targeting
              </a>
            </td>
          <td>
            Shinta Saito, N. Adachi
          </td>
          <td>2024-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a significant public health challenge, with 5-fluorouracil (5-FU) resistance being a major obstacle to effective treatment. Despite advancements, resistance to 5-FU remains formidable due to complex mechanisms such as alterations in drug transport, evasion of apoptosis, dysregulation of cell cycle dynamics, tumor microenvironment (TME) interactions, and extracellular vesicle (EV)-mediated resistance pathways. Traditional chemotherapy often results in high toxicity, highlighting the need for alternative approaches with better efficacy and safety. Phytochemicals (PCs) and EVs offer promising CRC therapeutic strategies. PCs, derived from natural sources, often exhibit lower toxicity and can target multiple pathways involved in cancer progression and drug resistance. EVs can facilitate targeted drug delivery, modulate the immune response, and interact with the TME to sensitize cancer cells to treatment. However, the potential of PCs and engineered EVs in overcoming 5-FU resistance and reshaping the immunosuppressive TME in CRC remains underexplored. Addressing this gap is crucial for identifying innovative therapies with enhanced efficacy and reduced toxicities. This review explores the multifaceted mechanisms of 5-FU resistance in CRC and evaluates the synergistic effects of combining PCs with 5-FU to improve treatment efficacy while minimizing adverse effects. Additionally, it investigates engineered EVs in overcoming 5-FU resistance by serving as drug delivery vehicles and modulating the TME. By synthesizing the current knowledge and addressing research gaps, this review enhances the academic understanding of 5-FU resistance in CRC, highlighting the potential of interdisciplinary approaches involving PCs and EVs for revolutionizing CRC therapy. Further research and clinical validation are essential for translating these findings into improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b42d2539e636dd666b34c979ca54ba15797bab1" target='_blank'>
              Innovative Strategies to Combat 5-Fluorouracil Resistance in Colorectal Cancer: The Role of Phytochemicals and Extracellular Vesicles
              </a>
            </td>
          <td>
            M. Barathan, A. K. Zulpa, Sook-Luan Ng, Y. Lokanathan, Min Hwei Ng, J. Law
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cancer, a multifactorial disease characterized by uncontrolled cellular proliferation, remains a global health challenge with significant morbidity and mortality. Genomic and molecular aberrations, coupled with environmental factors, contribute to its heterogeneity and complexity. Chemotherapeutic agents like doxorubicin (Dox) have shown efficacy against various cancers but are hindered by dose-dependent cytotoxicity, particularly on vital organs like the heart and brain. Autophagy, a cellular process involved in self-degradation and recycling, emerges as a promising therapeutic target in cancer therapy and neurodegenerative diseases. Dysregulation of autophagy contributes to cancer progression and drug resistance, while its modulation holds the potential to enhance treatment outcomes and mitigate adverse effects. Additionally, emerging evidence suggests a potential link between autophagy, DNA damage, and caretaker breast cancer genes BRCA1/2, highlighting the interplay between DNA repair mechanisms and cellular homeostasis. This review explores the intricate relationship between cancer, Dox-induced cytotoxicity, autophagy modulation, and the potential implications of autophagy in DNA damage repair pathways, particularly in the context of BRCA1/2 mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f00613c7c992a22afe5ec78107b8d19206649c8" target='_blank'>
              Complex Interplay between DNA Damage and Autophagy in Disease and Therapy
              </a>
            </td>
          <td>
            Aman Singh, Naresh Ravendranathan, Jefferson C. Frisbee, Krishna K. Singh
          </td>
          <td>2024-07-29</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Annotation of the cis-regulatory elements that drive transcriptional dysregulation in cancer cells is critical to improving our understanding of tumor biology. Herein, we present a compendium of matched chromatin accessibility (scATAC-seq) and transcriptome (scRNA-seq) profiles at single-cell resolution from human breast tumors and healthy mammary tissues processed immediately following surgical resection. We identify the most likely cell-of-origin for luminal breast tumors and basal breast tumors and then introduce a novel methodology that implements linear mixed-effects models to systematically quantify associations between regions of chromatin accessibility (i.e. regulatory elements) and gene expression in malignant cells versus normal mammary epithelial cells. These data unveil regulatory elements with that switch from silencers of gene expression in normal cells to enhancers of gene expression in cancer cells, leading to the upregulation of clinically relevant oncogenes. To translate the utility of this dataset into tractable models, we generated matched scATAC-seq and scRNA-seq profiles for breast cancer cell lines, revealing, for each subtype, a conserved oncogenic gene expression program between in vitro and in vivo cells. Together, this work highlights the importance of non-coding regulatory mechanisms that underlie oncogenic processes and the ability of single-cell multi-omics to define the regulatory logic of BC cells at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08bf4929898d9a3caa32f9bca65e96287e05306d" target='_blank'>
              Defining the Regulatory Logic of Breast Cancer Using Single-Cell Epigenetic and Transcriptome Profiling
              </a>
            </td>
          <td>
            Matthew J. Regner, Susana Garcia-Recio, A. Thennavan, K. Wiśniewska, Raul Mendez-Giraldez, Brooke Felsheim, Philip M. Spanheimer, Joel S. Parker, C. Perou, Hector L. Franco
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>159</td>
        </tr>

        <tr id="
 Aneuploidy, an imbalanced number of chromosomes or chromosome arms, is a genetic hallmark of cancer cells, yet aneuploidy remains a biological enigma and a missed opportunity for cancer therapy. Aneuploid cells must cope with several types of cellular stresses, potentially creating synthetic lethalities that can be used to target aneuploid cancer cells. Here, I will describe our efforts to identify synthetic lethalities of the aneuploid state (in contrast to synthetic lethalities of specific recurrent aneuploidies. Specifically, I will focus on three recent unpublished studies, where we report that: (1) Aneuploid cells are preferentially sensitive to perturbation of the spindle assembly checkpoint (SAC) and its regulator KIF18A; the expression and activity of CDC20 determine the sensitivity to SAC inhibition. (2) Aneuploid cells depend on MAPK signaling for overcoming aneuploidy-induced DNA damage; targeting MAPK signaling can sensitize aneuploid cells to DNA damage inducing agents and to PARP inhibitors. (3) Aneuploid cells depend on RNA and protein degradation mechanisms to attenuate the cellular consequence of extra chromosomes; this renders aneuploid cells more sensitive to inhibition of nonsense-mediate decay, miRNA-mediated gene silencing, and the proteasome complex.
 Citation Format: Uri Ben-David. Identifying and targeting synthetic lethalities of aneuploid (cancer) cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA015.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac71f1802eee0a6fdce6ada7502686abe3e34f7b" target='_blank'>
              Abstract IA015: Identifying and targeting synthetic lethalities of aneuploid (cancer) cells
              </a>
            </td>
          <td>
            Uri Ben-David
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cancer genomic studies have identified frequent alterations in components of the SWI/SNF (SWItch/Sucrose Non- Fermenting) chromatin remodeling complex including SMARCA4 and ARID1A. Importantly, clinical reports indicate that SMARCA4-mutant lung cancers respond poorly to immunotherapy and have dismal prognosis. However, the mechanistic basis of immunotherapy resistance is unknown. Here, we corroborated the clinical findings by using immune-humanized, syngeneic, and genetically engineered mouse models of lung cancer harboring SMARCA4 deficiency. Specifically, we show that SMARCA4 loss caused decreased response to anti-PD1 immunotherapy associated with significantly reduced infiltration of dendritic cells (DCs) and CD4+ T cells into the tumor microenvironment (TME). Mechanistically, we show that SMARCA4 loss in tumor cells led to profound downregulation of STING, IL1β and other components of the innate immune system as well as inflammatory cytokines that are required for efficient recruitment and activity of immune cells. We establish that this deregulation of gene expression is caused by cancer cell-intrinsic reprogramming of the enhancer landscape with marked loss of chromatin accessibility at enhancers of genes involved in innate immune response such as STING, IL1β, type I IFN and inflammatory cytokines. Interestingly, we observed that transcription factor NF-κB binding motif was highly enriched in enhancers that lose accessibility upon SMARCA4 deficiency. Finally, we confirmed that SMARCA4 and NF-κB co-occupy the same genomic loci on enhancers associated with STING and IL1β, indicating a functional interplay between SMARCA4 and NF-κB. Taken together, our findings provide the mechanistic basis for the poor response of SMARCA4-mutant tumors to anti-PD1 immunotherapy and establish a functional link between SMARCA4 and NF-κB on innate immune and inflammatory gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3fdeb7ebdd844e1757256a995b264456319e406" target='_blank'>
              SMARCA4 mutation induces tumor cell-intrinsic defects in enhancer landscape and resistance to immunotherapy
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Yonathan Lissanu
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d245ffa44bff0964d12c3d794cd68f26007361e4" target='_blank'>
              Venetoclax resistance in acute lymphoblastic leukemia is characterized by increased mitochondrial activity and can be overcome by co-targeting oxidative phosphorylation
              </a>
            </td>
          <td>
            S. Enzenmüller, A. Niedermayer, F. Seyfried, V. Muench, D. Tews, Ulrich Rupp, E. Tausch, Alexander Groß, Pamela Fischer-Posovszky, Paul Walther, S. Stilgenbauer, Hans A. Kestler, K. Debatin, L. Meyer
          </td>
          <td>2024-07-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="
 The translocation t(14;18) activates BCL2 and is considered the initiating genetic lesion in over ninety percent of follicular lymphomas (FL). Surprisingly, FL patients respond poorly to clinically approved inhibitor, venetoclax. The mechanisms underlying BCL2 independence in FL are poorly understood, and it has been speculated that epigenetic rewiring of cell death programs may play a role. We performed separate genome-wide CRISPR/Cas9 screens to explore the mechanisms of resistance to Venetoclax and an experimental MCL-1 inhibitor (S63845) and unveiled the histone lysine methyltransferase SETD1B (KMT2G) as an unexpected mechanism of resistance to both inhibitors in lymphoma. We show that mutations and deletions affecting SETD1B occur in 7% of FLs and 16% of diffuse large B cell lymphomas (DLBCL). In vivo, SETD1B acts as a tumor suppressor gene and cooperates with the related histone-lysine N-methyltransferase 2D (KMT2D) gene leading to a more aggressive disease presentation. This oncogenic cooperation is reflected in the genomes of human lymphoma, where SETD1B mutations typically co-occur with KMT2D mutations. Mechanistically, SETD1B is required to express pro-apoptotic BCL2 family proteins like BIM and BIK. Inhibitors of the KDM5 histone H3K4 de-methylase restore the expression of BIM and BIK, venetoclax sensitivity and lead to synergistic drug effects in SETD1B deficient lymphoma xenografts. Our results highlight a role for SETD1B in the epigenetic regulation of cell death proteins in B cell lymphomas, connecting the loss of SETD1B to lymphoma development and reduced sensitivity to venetoclax.
 Citation Format: Ana Portelinha, Shenqiu Wang, Sara Parsa, Man Jiang, Sagarajit Mohanty, Soumya Sharma, Neeraj Arya, Omid Tavana, Jiayu Wen, Jude Fitzgibbon, Ahmet Dogan, Ana Younes, Ari M. Melnick, Hans-Guido Wendel. SETD1B mutations confer apoptosis resistance and BCL2 independence in B-cell lymphoma [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR07.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b983f7fc86d8f88e6732308cf04ff08d3dfbd4f4" target='_blank'>
              Abstract PR07: SETD1B mutations confer apoptosis resistance and BCL2 independence in B-cell lymphoma
              </a>
            </td>
          <td>
            Ana Portelinha, Shenqiu Wang, Sara Parsa, Man Jiang, Sagarajit Mohanty, Soumya Sharma, Neeraj Arya, Omid Tavana, Jiayu Wen, Jude Fitzgibbon, Ahmet Dogan, Ana Younes, A. Melnick, H. Wendel
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6169884636c48723540d43c960017df0f71986" target='_blank'>
              Hidden secrets of the cancer genome: unlocking the impact of non-coding mutations in gene regulatory elements
              </a>
            </td>
          <td>
            Sandra Íñiguez-Muñoz, P. Llinàs-Arias, Miquel Ensenyat-Mendez, Andrés F. Bedoya-López, Javier I. J. Orozco, Javier Cortés, Ananya Roy, Karin Forsberg-Nilsson, M. DiNome, D. Marzese
          </td>
          <td>2024-06-20</td>
          <td>Cellular and Molecular Life Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Peripheral T-cell lymphomas (PTCL) are heterogeneous and highly aggressive hematologic malignancies with dismal outcomes, highlighting the need for novel targeted therapies. PTCLs are characterized by complex tumor-microenvironment (TME) ecosystems with low tumor cell content, admixed with rich immune infiltrates and stromal elements correlated with clinical outcomes. Direct targeting of the TME is an attractive strategy for PTCLs; however, the specific mechanisms that regulate the interdependence between lymphoma and TME remain poorly understood, hampering the development of TME-directed therapies. Our previous work based on driver genetic alterations in PTCL patients and genetic mouse models identified an important role of these driver mutations in remodeling the TME (Cortes et al., 2022). To address the specific role and mechanisms of tumor cell-TME interactions in the pathogenesis of PTCL, we developed a novel experimental platform and cutting-edge computational methodologies using single-nucleus RNAseq analysis to comprehensively profile lymphoma cells and their microenvironment from 27 PTCL patient samples of two major nodal pathologic subtypes, including 18 Nodal T follicular helper cell lymphomas (TFHcL) and 9 PTCL, not otherwise specified (PTCL, NOS). Our approach allowed the identification and in-depth characterization of tumor cells, lymphoid and myeloid TME, and vascular endothelium and non-endothelial stromal cells, which were challenging to capture with conventional single-cell RNAseq. The malignant T-cells in PTCL, NOS were grouped into patient-specific clusters demonstrating significant inter-tumor heterogeneity associated with diverse and frequently complex chromosomal abnormalities identified by copy number variation (CNV) analysis. In contrast, malignant TFHcL cells show a low degree to no apparent chromosomal abnormalities and divide into two major transcriptional subclusters, with tumor cells from the same patient falling into one of the two subclusters. Additionally, genomic profiles of TFHcL tumors revealed that the inter-tumor transcriptional heterogeneity of TFHcL tumor cells was strongly associated with the driver RHOA G17V mutation status. To profile oncogenic pathways critical for PTCL growth, we compared PTCL tumor cells to normal CD4 T-cells from reactive lymphoid tissue. PTCL, NOS and TFHcL tumor cells were equally enriched in the TCR signaling pathway, mitosis, and cell cycle related genes. However, exploitation of KRAS and cytokine-driven inflammatory signals was prevalent in TFHcL but not PTCL, NOS tumor cells. Analysis of the immune TME identified an increase in T regulatory cells, CD8 T-cells, immunoblasts, and macrophages in TFHcL. Furthermore, while dysregulated B-cells with heightened cytokine-driven inflammatory signals were unique to TFHcL, inflammatory macrophages and classic dendritic cells (cDC) were prominent in the TME of both subtypes. Our results uncover shared and subtype-specific pathogenic features in PTCL and pave the way for designing novel precision therapies for PTCL patients.
 Citation Format: Wen-Hsuan Wendy Lin, Anqi W Wang, Bobby B Shih, Jean-Baptiste F Reynier, Laura Quevedo Palacio, Craig Soderquist, Ryan D Najac, Cindy Ma, Govind Bhagat, Adolfo A Ferrando, Raul Rabadan, Teresa Palomero. Single-cell transcriptomics reveals shared and subtype-specific vulnerabilities of the tumor-microenvironment ecosystems in peripheral T-cell lymphomas [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d17bf46cac83bb1af31141fb58102ffe6ac218af" target='_blank'>
              Abstract PR05: Single-cell transcriptomics reveals shared and subtype-specific vulnerabilities of the tumor-microenvironment ecosystems in peripheral T-cell lymphomas
              </a>
            </td>
          <td>
            Wen-Hsuan Wendy Lin, Anqi W Wang, Bobby B. Shih, Jean-Baptiste F Reynier, Laura Quevedo Palacio, Craig R. Soderquist, Ryan Najac, Cindy Ma, Govind Bhagat, Adolfo A Ferrando, Raul Rabadan, Teresa Palomero
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="EPAS1HIF2α is the primary gene implicated in systemic hypoxia adaptation. Conversely, aberrantly activated EPAS1HIF2α acts as a tumor driver against which anti-tumor therapeutics are proven effective. We elucidated connections between adaptation to systemic hypoxia in high-altitude populations, such as Tibetans and Sherpas, and human tumors. Similar to the accelerated adaptability observed in high-altitude populations via genetic introgression, tumors from patients with hypoxia since birth exhibited impaired DNA repair and increased mutation burden. As in high-altitude dwellers, EPAS1HIF2α genetic variants were positively selected within sympathetic tumors developed under hypoxia, with a consistently high frequency of 90%. Bulk and single-cell RNA sequencing followed by in vitro studies have shown that hypoxia and EPAS1HIF2α gain-of-function tumor mutations induce COX4i2 expression and impair mitochondrial respiration, indicating that decreased cellular oxygen consumption may confer a proliferative advantage in hypoxia. Analyzing medical data from a patient cohort with hypoxia since birth who developed/did not develop tumors revealed tissue-specific and time-dependent tumorigenic effects of systemic hypoxia, which is limited to oxygen-sensitive and responsive cells, particularly during the postnatal period. This study supports connections between the EPAS1HIF2α genetic adaptation in human tumors developed under systemic hypoxia to populations living in high altitudes. The genetic adaptations in populations to different stressors can be explored further to understand tumorigenesis and tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb7c10df0e202cb735340e16f27a752b3b7b5d8" target='_blank'>
              Convergent genetic adaptation in human tumors developed under systemic hypoxia and in populations living at high altitudes
              </a>
            </td>
          <td>
            Carlota Arenillas, José Ruiz-Cantador, Lucía Celada, Bruna Calsina, Eduardo García-Galea, Debayan Datta, Roberta Fasani, Ana Belén Moreno-Cárdenas, J. J. Alba-Linares, Berta Miranda, Á. Martínez-Montes, Cristina Álvarez-Escolá, Beatriz Lecumberri, Elvira Ana González García, Shahida K. Flores, Emmanuel Esquivel, Yanli Ding, Rita Maria Regojo Zapata, J. Pozo-Kreilinger, Carmela Iglesias, Trisha Dwight, Christopher Muir, A. O. Alday, M. E. Garrido-Lestache Rodríguez-Monte, M. D. del Cerro, Isaac Martínez-Bendayán, D. M. Lourenço, Maria Adelaide A. Pereira, N. Burnichon, A. Buffet, Craig Broberg, P. Dickson, M. F. Fraga, J. L. Llorente Pendás, Joaquín Rueda Soriano, Francisco Buendía Fuentes, Sergio P. A. Toledo, R. Clifton-Bligh, R. Dienstmann, Jaume Capdevila, A. Gimenez-Roqueplo, J. Favier, Donate Weghorn, P. Nuciforo, William F Young, A. Opotowsky, Anand Vaidya, Irina Bancos, Mercedes Robledo, A. Casterás, L. Dos-Subirà, María Dolores Chiara, Igor Adameyko, P. Dahia, R. Toledo
          </td>
          <td>2024-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd083194686b61ccc53ca0d4c394b67971b4cdb7" target='_blank'>
              Targeted gene therapy for cancer: the impact of microRNA multipotentiality
              </a>
            </td>
          <td>
            Nourhan A. Abou Madawi, Zeinab E Darwish, Enas M Omar
          </td>
          <td>2024-08-01</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Emerging immunotherapies such as immune checkpoint blockade (ICB) and chimeric antigen receptor T-cell (CAR-T) therapy have revolutionized cancer treatment and have improved the survival of patients with multiple cancer types. Despite this success many patients are unresponsive to these treatments or relapse following treatment. CRISPR activation and knockout (KO) screens have been used to identify novel single gene targets that can enhance effector T cell function and promote immune cell targeting and eradication of tumors. However, cancer cells often employ multiple genes to promote an immunosuppressive pathway and thus modulating individual genes often has a limited effect. Paralogs are genes that originate from common ancestors and retain similar functions. They often have complex effects on a particular phenotype depending on factors like gene family similarity, each individual gene’s expression and the physiological or pathological context. Some paralogs exhibit synthetic lethal interactions in cancer cell survival; however, a thorough investigation of paralog pairs that could enhance the efficacy of cancer immunotherapy is lacking. Here we introduce a sensitive computational approach that uses sgRNA sets enrichment analysis to identify cancer-intrinsic paralog pairs which have the potential to synergistically enhance T cell-mediated tumor destruction. We have further developed an ensemble learning model that uses an XGBoost classifier and incorporates features such as gene characteristics, sequence and structural similarities, protein-protein interaction (PPI) networks, and gene coevolution data to predict paralog pairs that are likely to enhance immunotherapy efficacy. We experimentally validated the functional significance of these predicted paralog pairs using double knockout (DKO) of identified paralog gene pairs as compared to single gene knockouts (SKOs). These data and analyses collectively provide a sensitive approach to identify previously undetected paralog pairs that can enhance cancer immunotherapy even when individual genes within the pair has a limited effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9398918df5088c04d21b0123727e4ddeb607a57" target='_blank'>
              Sensitive detection of synthetic response to cancer immunotherapy driven by gene paralog pairs
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Emily He, Ping Ren, Nipun Verma, Xinxin Zhu, Di Feng, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5073eeabc50bea9fdc09e2d2eeb42d91221fda8" target='_blank'>
              Integrative genomics identifies SHPRH as a tumor suppressor gene in lung adenocarcinoma that regulates DNA damage response.
              </a>
            </td>
          <td>
            A. Nagelberg, T. Sihota, Yu-Chi Chuang, Rocky Shi, Justine L M Chow, John English, C. MacAulay, Stephen Lam, Wan L Lam, William W Lockwood
          </td>
          <td>2024-06-18</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9895762dc4559c029ea063221ae034c05dcf58ce" target='_blank'>
              DNA replication in early mammalian embryos is patterned, predisposing lamina-associated regions to fragility
              </a>
            </td>
          <td>
            Shuangyi Xu, Ning Wang, M. Zuccaro, Jeannine Gerhardt, R. Iyyappan, Giovanna Nascimento Scatolin, Zongliang Jiang, T. Baslan, A. Koren, Dieter Egli
          </td>
          <td>2024-06-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="
 In recent years, helicases have moved to the forefront of research as novel drug targets for a variety of human diseases, including cancer. As critical components of the cell cycle and DNA repair, these enzymes offer an attractive point of therapeutic intervention. Recent bioinformatic endeavours such as the Cancer DepMap have further highlighted the essentiality of these proteins in the progression of numerous cancers. Inhibitors of Werner helicase (WRN) entered Phase I evaluation in 2023, after studies highlighted its role in tumours with microsatellite instability (MSI) and impaired mismatch repair (MMR). Building on these advances, helicases now represent a significant drug target class and are the subject of intense research. However, these targets present significant complexities in their prosecution. Hit finding and robust validation of these putative active compounds remains challenging, with high attrition rates and significant sensitivity to false positives. Resultant hits require thorough biological characterisation via both biochemical and biophysical methods, alongside determination of selectivity against other closely related family members, to increase confidence in their provenance before commencing a detailed drug discovery project. To exemplify these approaches, we have profiled three commercially available WRN inhibitors from two chemical series through a selection of biochemical and biophysical assays. The three compounds exhibit excellent selectivity against 6 distinct helicase/translocase targets. In Michaelis-Menten mechanism of Inhibition studies, we defined contrasting mechanisms of inhibition for the two series and were also able to identify time dependent inhibition with one compound series. Analysing binding interactions of the compounds against protein and protein:substrate complexes, via a fluorescent thermal shift assay (FTSA) and microscale thermophoresis (MST), enabled us to further understand and interrogate the mechanism of inhibition. Herein, we describe some of our experiences, learnings and best practices when prosecuting these compelling, but challenging, targets.
 Citation Format: Yael Mamane, Zoe Caple, Katie Chapman, Ian Henderson, Allan Jordan, Vanessa Lyne, Cynthia Okoye, Laurent Rigoreau, Ganesh Kadamur, Stuart Thomson, Chris Tomlinson, Jana Wolf. Unwinding the complexities of helicases as compelling drug targets in oncology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr A003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/693e492bec5c1538ece3834db932648191e6db01" target='_blank'>
              Abstract A003: Unwinding the complexities of helicases as compelling drug targets in oncology
              </a>
            </td>
          <td>
            Yael Mamane, Zoe Caple, Katie Chapman, Ian Henderson, Allan Jordan, Vanessa Lyne, Cynthia Okoye, Laurent Rigoreau, Ganesh Kadamur, Stuart Thomson, Chris Tomlinson, Jana Wolf
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Radiotherapy is a key treatment option for a wide variety of human tumors, employed either alone or alongside with other therapeutic interventions. Radiotherapy uses high-energy particles to destroy tumor cells, blocking their ability to divide and proliferate. The effectiveness of radiotherapy is due to genetic and epigenetic factors that determine how tumor cells respond to ionizing radiation. These factors contribute to the establishment of resistance to radiotherapy, which increases the risk of poor clinical prognosis of patients. Although the mechanisms by which tumor cells induce radioresistance are unclear, evidence points out several contributing factors including the overexpression of DNA repair systems, increased levels of reactive oxygen species, alterations in the tumor microenvironment, and enrichment of cancer stem cell populations. In this context, dysregulation of microRNAs or miRNAs, critical regulators of gene expression, may influence how tumors respond to radiation. There is increasing evidence that miRNAs may act as sensitizers or enhancers of radioresistance, regulating key processes such as the DNA damage response and the cell death signaling pathway. Furthermore, expression and activity of miRNAs have shown informative value in overcoming radiotherapy and long-term radiotoxicity, revealing their potential as biomarkers. In this review, we will discuss the molecular mechanisms associated with the response to radiotherapy and highlight the central role of miRNAs in regulating the molecular mechanisms responsible for cellular radioresistance. We will also review radio-miRs, radiotherapy-related miRNAs, either as sensitizers or enhancers of radioresistance that hold promise as biomarkers or pharmacological targets to sensitize radioresistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f11e14f281c48ffc0ef7d72af9b361e4a40e0c9" target='_blank'>
              Radio-miRs: a comprehensive view of radioresistance-related microRNAs.
              </a>
            </td>
          <td>
            Abraham Pedroza-Torres, Sandra L Romero-Córdoba, S. Montaño, O. Peralta-Zaragoza, Dora Emma Vélez-Uriza, Cristian Arriaga-Canon, Xiadani Guajardo-Barreto, Diana Bautista-Sánchez, Rodrigo Sosa-León, Olivia Hernández-González, J. Díaz-Chávez, R. M. Álvarez-Gómez, Luis A Herrera
          </td>
          <td>2024-07-04</td>
          <td>Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8452ec11b7bc9cb4359d295a3fc50d7b8cbfbae" target='_blank'>
              HOXB2 promotes cisplatin resistance by upregulating lncRNA DANCR in ovarian cancer
              </a>
            </td>
          <td>
            Xiao Li, Zhen Zheng, Wanzhen Zhou, Huixian Huang, Yang Zhou, Qinyang Xu, Xiaolu Zhu, Yincheng Teng
          </td>
          <td>2024-06-08</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e81b6b5c5ed75ea0aa53f16f320c4ca6faed6d1d" target='_blank'>
              Intricate effects of post-translational modifications in liver cancer: mechanisms to clinical applications
              </a>
            </td>
          <td>
            Yu Zhang, Weihao Xu, Chuanhui Peng, Shenli Ren, Chengpeng Zhang
          </td>
          <td>2024-07-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite rapid advances in the field of immunotherapy, including the success of immune checkpoint inhibition in treating multiple cancer types, clinical response in high-grade gliomas (HGGs) has been disappointing. This has been in part attributed to the low tumor mutational burden (TMB) of the majority of HGGs. Hypermutation is a recently characterized glioma signature that occurs in a small subset of cases, which may open an avenue to immunotherapy. The substantially elevated TMB of these tumors most commonly results from alterations in the DNA mismatch repair pathway in the setting of extensive exposure to temozolomide or, less frequently, from inherited cancer predisposition syndromes. In this review, we discuss the genetics and etiology of hypermutation in HGGs, with an emphasis on the resulting genomic signatures, and the state and future directions of immuno-oncology research in these patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2f2c2e78fe8478861279397510414edbdfa255" target='_blank'>
              Pathways to hypermutation in high-grade gliomas: Mechanisms, syndromes, and opportunities for immunotherapy.
              </a>
            </td>
          <td>
            Tuesday Haynes, Mark R Gilbert, Kevin Breen, Chunzhang Yang
          </td>
          <td>2024-06-24</td>
          <td>Neuro-oncology advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21a2a05c7ef392ef903b88cba17ae6f2c9311cf5" target='_blank'>
              Mechanistic patterns and clinical implications of oncogenic tyrosine kinase fusions in human cancers
              </a>
            </td>
          <td>
            T. Cheong, Ahram Jang, Qi Wang, Giulia Leonardi, B. Ricciuti, J. Alessi, Alessandro Di Federico, Mark M Awad, Maria K Lehtinen, Marian H Harris, Roberto Chiarle
          </td>
          <td>2024-06-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Introduction: The Euchromatic Histone Methyl Transferase Protein 2 (EHMT2), also known as G9a, deposits transcriptionally repressive chromatin marks that play pivotal roles in the maturation and homeostasis of multiple organs. Recently, we have shown that Ehmt2 inactivation in the mouse pancreas alters growth and immune gene expression networks, antagonizing Kras-mediated pancreatic cancer initiation and promotion. Here, we elucidate the essential role of Ehmt2 in maintaining a transcriptional landscape that protects organs from inflammation. Methods: Comparative RNA-seq studies between normal postnatal and young adult pancreatic tissue from Ehmt2 conditional knockout animals (Ehmt2 fl/fl ) targeted to the exocrine pancreatic epithelial cells (Pdx1-Cre and P48 Cre/+ ), reveal alterations in gene expression networks in the whole organ related to injury-inflammation-repair, suggesting an increased predisposition to damage. Thus, we induced an inflammation repair response in the Ehmt2 fl/fl pancreas and used a data science-based approach to integrate RNA-seq-derived pathways and networks, deconvolution digital cytology, and spatial transcriptomics. We also analyzed the tissue response to damage at the morphological, biochemical, and molecular pathology levels. Results and discussion: The Ehmt2 fl/fl pancreas displays an enhanced injury-inflammation-repair response, offering insights into fundamental molecular and cellular mechanisms involved in this process. More importantly, these data show that conditional Ehmt2 inactivation in exocrine cells reprograms the local environment to recruit mesenchymal and immunological cells needed to mount an increased inflammatory response. Mechanistically, this response is an enhanced injury-inflammation-repair reaction with a small contribution of specific Ehmt2-regulated transcripts. Thus, this new knowledge extends the mechanisms underlying the role of the Ehmt2-mediated pathway in suppressing pancreatic cancer initiation and modulating inflammatory pancreatic diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6875f8969140d78af8699723af4ec6fb4942849a" target='_blank'>
              Ehmt2 inactivation in pancreatic epithelial cells shapes the transcriptional landscape and inflammation response of the whole pancreas
              </a>
            </td>
          <td>
            Gareth Pollin, A. Mathison, T. D. de Assunção, Anju Thomas, Atefeh Zeighami, Ann Salmonson, Hongfei Liu, Guillermo Urrutia, Pallavi Vankayala, Stephen J. Pandol, Johnny C. Hong, Michael T. Zimmermann, J. Iovanna, Victor X. Jin, Raul Urrutia, G. Lomberk
          </td>
          <td>2024-06-14</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e4d778cae3d7e3203dfad059efb7f4865240dc4" target='_blank'>
              Insights into treatment-specific prognostic somatic mutations in NSCLC from the AACR NSCLC GENIE BPC cohort analysis
              </a>
            </td>
          <td>
            Yi Liu, Sindhu Yalavarthi, Fan Yang, Yusif Abdul-Rashid, Shenkun Tang, Zihe Long, Yongkai Qin, Kerui Wu, Zhifei Wang
          </td>
          <td>2024-07-02</td>
          <td>BMC Pulmonary Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary This study investigates the impact of the BRAFV600E mutation, present in half of melanoma cases, on immune microenvironment and therapeutic response. Analyzing the Cancer Genome Atlas data, we uncover that the mutation correlates with reduced tumor mutation burden, indicating a lower generation of immune-stimulating neopeptides. Examination of immune subtypes reveal heightened immunosuppression in BRAFV600E-mutated tumors. Using melanoma cell lines harboring different genomic profiles, we observed an enhanced response to immunogenic cell death (ICD)—a crucial process eliciting anti-tumor immune responses—mediated by photodynamic therapy (PDT). Transcriptomic analysis highlights upregulation of IFNAR1, IFNAR2, and CXCL10 genes in BRAFV600E-mutated cells, suggesting their role in ICD. Our results describe the intricate relationship between the BRAFV600E mutation and immune responses, hinting at a potential link between this mutation and responsiveness to ICD-inducing therapies, particularly PDT, putatively linked to increased IFN-1 pathway activation. Abstract The BRAFV600E mutation, found in approximately 50% of melanoma cases, plays a crucial role in the activation of the MAPK/ERK signaling pathway, which promotes tumor cell proliferation. This study aimed to evaluate its impact on the melanoma immune microenvironment and therapeutic responses, particularly focusing on immunogenic cell death (ICD), a pivotal cytotoxic process triggering anti-tumor immune responses. Through comprehensive in silico analysis of the Cancer Genome Atlas data, we explored the association between the BRAFV600E mutation, immune subtype dynamics, and tumor mutation burden (TMB). Our findings revealed that the mutation correlated with a lower TMB, indicating a reduced generation of immunogenic neoantigens. Investigation into immune subtypes reveals an exacerbation of immunosuppression mechanisms in BRAFV600E-mutated tumors. To assess the response to ICD inducers, including doxorubicin and Me-ALA-based photodynamic therapy (PDT), compared to the non-ICD inducer cisplatin, we used distinct melanoma cell lines with wild-type BRAF (SK-MEL-2) and BRAFV600E mutation (SK-MEL-28, A375). We demonstrated a differential response to PDT between the WT and BRAFV600E cell lines. Further transcriptomic analysis revealed upregulation of IFNAR1, IFNAR2, and CXCL10 genes associated with the BRAFV600E mutation, suggesting their involvement in ICD. Using a gene reporter assay, we showed that PDT robustly activated the IFN-1 pathway through cGAS-STING signaling. Collectively, our results underscore the complex interplay between the BRAFV600E mutation and immune responses, suggesting a putative correlation between tumors carrying the mutation and their responsiveness to therapies inducing the IFN-1 pathway, such as the ICD inducer PDT, possibly mediated by the elevated expression of IFNAR1/2 receptors">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ed26388b396064bd1f63bcbdf14dd7a6adc97c" target='_blank'>
              Impact of Genomic Mutation on Melanoma Immune Microenvironment and IFN-1 Pathway-Driven Therapeutic Responses
              </a>
            </td>
          <td>
            F. Mentucci, Elisa Ayelén Romero Nuñez, Agustina Ercole, Valentina Silvetti, J. Dal Col, M. Lamberti
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Targeting activated oncogenes is an effective treatment strategy across many cancers now including therapeutics targeting KRAS. Genetic events including DNA damage deficiencies and tumor suppressor mutations require alternative strategies and the concept of synthetic lethality has been applied to these alterations. PARPi inhibitors were a founding member of this class of therapeutics demonstrating enhanced activity in the setting of BRCA-deficiency. By applying large-scale loss-of-function approaches1 we and other discovered the vulnerability to PRMT5 and WRN inhibition imposed by co-deletions of MTAP and CDKN2A, and the MSI+ genotype respectively2,3. Such inhibitors are now in clinical trials. To explore the potential for paralogous genes to act as synthetic lethal gene pairs we enacted dual knockout screens. These early efforts led to the discovery of DUSP4/6 paralog dependence in the setting of BRAF and NRAS mutations in melanoma4. Intriguingly the loss of DUSP4/6 impaired cancer cell viability through the increased activation of ERK highlighting the susceptibility of cancers to pathway activation in addition to the more common sensitivity to pathway inhibition. This inappropriate activation of the ERK signaling pathway, the conflict between EGFR and KRAS activation, the synthetic lethality enacted by TRIM8 knockouts, and the effects of AR agonists on prostate cancer viability, points to a wider than expected vulnerability of cancer to inappropriate gene activation. To systematically identify context-specific gene activation induced lethalities in cancer, we developed methods for enacting gain-of-function perturbations across ∼500 barcoded cancer cell lines. With this approach, we queried the pan-cancer vulnerability landscape upon activating 10 key cancer pathway revealing activation dependencies in MAPK and PI3K pathways. Notably, we discovered novel pathway hyperactivation dependencies in subsets of APC-mutant colorectal cancers where further activation of the WNT pathway by APC knockdown or direct β-catenin overexpression led to robust antitumor effects in xenograft and patient-derived organoid models. These latter discoveries paradoxically point to the residual activity of the APC ubiquitin-ligase complex as a target in APC-mutant CRC5. 1. McDonald, E. R., 3rd et al. Project DRIVE: A Compendium of Cancer Dependencies and Synthetic Lethal Relationships Uncovered by Large-Scale, Deep RNAi Screening. Cell 170, 577-592.e10 (2017). 2. Mavrakis, K. J. et al. Disordered methionine metabolism in MTAP/CDKN2A-deleted cancers leads to dependence on PRMT5. Science 351, 1208–1213 (2016). 3. Chan, E. M. et al. WRN helicase is a synthetic lethal target in microsatellite unstable cancers. Nature 568, 551–556 (2019). 4. Ito, T. et al. Paralog knockout profiling identifies DUSP4 and DUSP6 as a digenic dependence in MAPK pathway-driven cancers. Nat. Genet. 53, 1664–1672 (2021). 5. Chang, L. et al. Systematic profiling of conditional pathway activation identifies contextdependent synthetic lethalities. Nat. Genet. 55, 1709–1720 (2023).
 Citation Format: William R. Sellers. Exploiting pathway activation as a new form of synthetic lethality [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/008d68827fda5309c476845738e9fa9b3ca9c530" target='_blank'>
              Abstract IA001: Exploiting pathway activation as a new form of synthetic lethality
              </a>
            </td>
          <td>
            William R. Sellers
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb51667b0ded5712e35c88a3f70d8769cf6b0d56" target='_blank'>
              PR-SET7 epigenetically restrains uterine interferon response and cell death governing proper postnatal stromal development
              </a>
            </td>
          <td>
            H. Bao, Yang Sun, Na Deng, Leilei Zhang, Yuanyuan Jia, Gaizhen Li, Yun Gao, Xinyi Li, Yedong Tang, Han Cai, Jinhua Lu, Haibin Wang, Wenbo Deng, S. Kong
          </td>
          <td>2024-06-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="
 Synthetic lethality, the combination of two mutated genes which results in cell death, has been greatly investigated due to its therapeutic potential in cancer. Despite the common practice of experimentally detecting synthetic lethality, here we extend a computational framework that employs co-occurrence of gene mutations to infer synthetic lethality by incorporating pathways. Pathways aggregate the mutated genes into a functional unit and give power to uncover relationships containing genes of low mutation frequency e.g. in pediatric cancer, which otherwise would not be observed by testing individual gene pairs. Our proposed framework is an alternative avenue to a more focused synthetic lethality search by exploiting mutation data & pathway knowledge. We infer potentially synthetic lethal relationships based on mutated gene co-occurrence. A less often than expected co-occurrence, or so-called mutual exclusivity (ME), indicates a potentially synthetic lethal relationship whereas one that occurs more often than expected (CO) indicates a potentially advantageous relationship. In this framework we use a selected set of biological pathways from Reactome to aggregate gene mutations per pathway for each individual tumor (SNVs and indels with high/moderate impact) within two public pediatric cancer datasets, TARGET and DKFZ. This results in mutated-pathway profiles per tumor which are tested for co-occurrence using rediscover. The test outputs significant co-occurring and mutually exclusive pathways from which we extract the underlying mutated gene pairs. The co-occurring and mutually exclusive gene pairs are then annotated for pathway epistasis, protein complexes and their presence in BioGRID and SynLethDB. Finally, the validity of the proposed gene pairs is examined in literature. The test detected 439 (ME: 369, CO:70) significant pathway pairs in TARGET and 49 (ME: 31, CO:18) in DKFZ across several cancer types. Out of these, 3185 gene pairs were extracted in TARGET (ME: 2671, CO: 514) and 331 (ME: 16, CO: 315) in DKFZ. These relationships are significantly more than the ones found when testing solely for gene pairs. Pathways aid in partially decreasing findings due to subtype or pathway epistasis. For example, FLT3-KRAS found in TARGET-B-ALL by the gene test is not detected by the pathway test. The test was, also, able to uncover mutually exclusive (ME) gene pairs in smaller datasets. The initially found ME genes TP53-DROSHA in Wilms tumors and KIT-NRAS in AML tumors in TARGET by the gene test, were detected in the corresponding cancer types’ DKFZ datasets. In addition to TP53-DROSHA, TP53-DGCR8 were found ME in Wilms tumors of DKFZ, indicating the microprocessor complex mutations ME to TP53 mutations. To sum up, we present a pathway-informed synthetic lethality inference framework for pediatric cancer to explore synthetic lethal relationships and other complex functional relationships between mutated genes by exploiting already existing data and database information as an alternative avenue for follow-up experimentation.
 Citation Format: Anastasia Spinou, Richard Gremmen, Puck Veen, Jarno Drost, Patrick Kemmeren. A pathway-informed framework to infer synthetic lethal relationships in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B027.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d48ab4a56548086e8fdb609b8128beaca8d033c" target='_blank'>
              Abstract B027: A pathway-informed framework to infer synthetic lethal relationships in pediatric cancer
              </a>
            </td>
          <td>
            Anastasia Spinou, Richard Gremmen, Puck Veen, J. Drost, P. Kemmeren
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="
 I will introduce the Cancer Dependency Map (DepMap) project. This project aims to build a resource to identify the landscape of cancer vulnerabilities by linking molecular alterations of tumors to gene dependencies or drug sensitivities. This resource is being used by scientists around the world and has made significant contributions to basic and translational cancer research. I will describe how we are building DepMap and the multiple components it requires. Additionally, I will discuss ongoing projects to expand DepMap, including screening next-generation models. I will also present how to identify potential therapeutic targets using DepMap data and the emerging types of targets. Lastly, I will highlight a recently identified synthetic lethal target in the ribosome rescue pathway in molecularly defined patient populations, demonstrating the power of DepMap in identifying such targets.
 Citation Format: Francisca Vazquez. Towards mapping the landscape of cancer vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA011.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14b02eb727e279f5bbf20bcd44dd4aeb3802c434" target='_blank'>
              Abstract IA011: Towards mapping the landscape of cancer vulnerabilities
              </a>
            </td>
          <td>
            Francisca Vazquez
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/081cebd6c13932575bbecec80f6933eed0eb38d1" target='_blank'>
              Genetic drivers and cellular selection of female mosaic X chromosome loss.
              </a>
            </td>
          <td>
            Aoxing Liu, G. Genovese, Yajie Zhao, M. Pirinen, S. Zekavat, K. Kentistou, Zhiyu Yang, K. Yu, C. Vlasschaert, Xiaoxi Liu, Derek W. Brown, G. Hudjashov, Bryan R. Gorman, J. Dennis, Weiyin Zhou, Y. Momozawa, S. Pyarajan, Valdislav Tuzov, Fanny-Dhelia Pajuste, M. Aavikko, Timo P. Sipilä, A. Ghazal, Wenbin Huang, Neal D. Freedman, L. Song, Eugene J. Gardner, V. Sankaran, A. Palotie, Hanna M. Ollila, T. Tukiainen, S. Chanock, R. Mägi, Pradeep Natarajan, M. Daly, A. Bick, S. Mccarroll, Chikashi Terao, Po-Ru Loh, A. Ganna, J. R. Perry, M. Machiela
          </td>
          <td>2024-06-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>138</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f86a06efc22905e428c8b555e87519dcf521de0" target='_blank'>
              Simultaneous screening of overexpressed genes in breast cancer for oncogenic drivers and tumor dependencies
              </a>
            </td>
          <td>
            Ada Mofunanya, Eleanor R Cameron, Christian J Braun, Frank Celeste, Xiaoyu Zhao, M. Hemann, Kenneth L Scott, Jinyu Li, Scott Powers
          </td>
          <td>2024-06-09</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdb874383a628cf292c0913e4a06655cd292ef13" target='_blank'>
              Frequency-dependent selection of neoantigens fosters tumor immune escape and predicts immunotherapy response
              </a>
            </td>
          <td>
            Shaoqing Chen, Duo Xie, Zan Li, Jiguang Wang, Zheng Hu, Da Zhou
          </td>
          <td>2024-06-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="INTRODUCTION
Defining the chromosomal and molecular changes associated with myeloid neoplasms (MNs) optimizes clinical care through improved diagnosis, prognosis, treatment planning, and patient monitoring. This review will concisely describe the techniques used to profile MNs clinically today, with descriptions of challenges and emerging approaches that may soon become standard-of-care.


AREAS COVERED
In this review, the authors discuss molecular assessment of MNs using non-sequencing techniques, including conventional cytogenetic analysis, fluorescence in situ hybridization, chromosomal genomic microarray testing; as well as DNA- or RNA-based next-generation sequencing (NGS) assays; and sequential monitoring via digital PCR or measurable residual disease assays. The authors explain why distinguishing somatic from germline alleles is critical for optimal management. Finally, they introduce emerging technologies, such as long-read, whole exome/genome, and single-cell sequencing, which are reserved for research purposes currently but will become clinical tests soon.


EXPERT OPINION
The authors describe challenges to the adoption of comprehensive genomic tests for those in resource-constrained environments and for inclusion into clinical trials. In the future, all aspects of patient care will likely be influenced by the adaptation of artificial intelligence and mathematical modeling, fueled by rapid advances in telecommunications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce5821258a38d2a15a1cce4cdbe2844d4e8f9b60" target='_blank'>
              The impact of next-generation sequencing for diagnosis and disease understanding of myeloid malignancies.
              </a>
            </td>
          <td>
            Erica Vormittag-Nocito, M. Sukhanova, Lucy A. Godley
          </td>
          <td>2024-07-25</td>
          <td>Expert review of molecular diagnostics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cell immortalization, a hallmark of cancer development, is a process that cells can undergo on their path to carcinogenesis. Spontaneously immortalized mouse embryonic fibroblasts (MEFs) have been used for decades; however, changes in the global transcriptome during this process have been poorly described. In our research, we characterized the poly-A RNA transcriptome changes after spontaneous immortalization. To this end, differentially expressed genes (DEGs) were screened using DESeq2 and characterized by gene ontology enrichment analysis and protein–protein interaction (PPI) network analysis to identify the potential hub genes. In our study, we identified changes in the expression of genes involved in proliferation regulation, cell adhesion, immune response and transcriptional regulation in immortalized MEFs. In addition, we performed a comparative analysis with previously reported MEF immortalization data, where we propose a predicted gene regulatory network model in immortalized MEFs based on the altered expression of Mapk11, Cdh1, Chl1, Zic1, Hoxd10 and the novel hub genes Il6 and Itgb2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edae843236a102c56d19f5e2f20e7de0a5418029" target='_blank'>
              Transcriptome Profiling of Mouse Embryonic Fibroblast Spontaneous Immortalization: A Comparative Analysis
              </a>
            </td>
          <td>
            Jocshan Loaiza-Moss, U. Braun, Michael Leitges
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Lung cancer is a leading cause of cancer-related death worldwide in both men and women, however mortality in the US and EU are recently declining in parallel with the gradual cut of smoking prevalence. Consequently, the relative frequency of adenocarcinoma increased while that of squamous and small cell carcinomas declined. During the last two decades a plethora of targeted drug therapies have appeared for the treatment of metastasizing non-small cell lung carcinomas (NSCLC). Personalized oncology aims to precisely match patients to treatments with the highest potential of success. Extensive research is done to introduce biomarkers which can predict the effectiveness of a specific targeted therapeutic approach. The EGFR signaling pathway includes several sufficient targets for the treatment of human cancers including NSCLC. Lung adenocarcinoma may harbor both activating and resistance mutations of the EGFR gene, and further, mutations of KRAS and BRAF oncogenes. Less frequent but targetable genetic alterations include ALK, ROS1, RET gene rearrangements, and various alterations of MET proto-oncogene. In addition, the importance of anti-tumor immunity and of tumor microenvironment has become evident recently. Accumulation of mutations generally trigger tumor specific immune defense, but immune protection may be upregulated as an aggressive feature. The blockade of immune checkpoints results in potential reactivation of tumor cell killing and induces significant tumor regression in various tumor types, such as lung carcinoma. Therapeutic responses to anti PD1-PD-L1 treatment may correlate with the expression of PD-L1 by tumor cells. Due to the wide range of diagnostic and predictive features in lung cancer a plenty of tests are required from a single small biopsy or cytology specimen, which is challenged by major issues of sample quantity and quality. Thus, the efficacy of biomarker testing should be warranted by standardized policy and optimal material usage. In this review we aim to discuss major targeted therapy-related biomarkers in NSCLC and testing possibilities comprehensively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/009f3a9ab76c5408a3ffa568b375af58c7a758c0" target='_blank'>
              The rapidly changing field of predictive biomarkers of non-small cell lung cancer
              </a>
            </td>
          <td>
            László József Tóth, A. Mokánszki, Gábor Méhes
          </td>
          <td>2024-06-17</td>
          <td>Pathology and Oncology Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cells must adapt to environmental changes to maintain homeostasis. One of the most striking environmental adaptations is entry into hibernation during which core body temperature can decrease from 37°C to as low at 4°C. How mammalian cells, which evolved to optimally function within a narrow range of temperatures, adapt to this profound decrease in temperature remains poorly understood. In this study, we conducted the first genome-scale CRISPR-Cas9 screen in cells derived from Syrian hamster, a facultative hibernator, as well as human cells to investigate the genetic basis of cold tolerance in a hibernator and a non-hibernator in an unbiased manner. Both screens independently revealed glutathione peroxidase 4 (GPX4), a selenium-containing enzyme, and associated proteins as critical for cold tolerance. We utilized genetic and pharmacological approaches to demonstrate that GPX4 is active in the cold and its catalytic activity is required for cold tolerance. Furthermore, we show that the role of GPX4 as a suppressor of cold-induced cell death extends across hibernating species, including 13-lined ground squirrels and greater horseshoe bats, highlighting the evolutionary conservation of this mechanism of cold tolerance. This study identifies GPX4 as a central modulator of mammalian cold tolerance and advances our understanding of the evolved mechanisms by which cells mitigate cold-associated damage—one of the most common challenges faced by cells and organisms in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181b39f570d649216102dc87d265ba81bbf8c6ba" target='_blank'>
              Multi-species genome-wide CRISPR screens identify GPX4 as a conserved suppressor of cold-induced cell death
              </a>
            </td>
          <td>
            Breanna Lam, Kathrin M. Kajderowicz, Heather R. Keys, Julian M. Roessler, Evgeni M. Frenkel, Adina Kirkland, Punam Bisht, Mohamed A. El-Brolosy, Rudolf Jaenisch, George W. Bell, Jonathan S Weissman, Eric C. Griffith, Siniša Hrvatin
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR-T) cells that are obtained from the specific patient, modified genetically ex vivo, and possess the remarkable capability to identify and eradicate targeted cancer cells. These modified cells are subsequently reintroduced into the patient, effectively treating blood cancer. CAR-T therapy is approved to be applied in leukemia due to its great clinical therapeutic effect on B cell hematological malignancies. However, solid tumors are more resistant to this therapy for many reasons. The abnormal vascular structure of solid tumors hampers CAR-T cell trafficking. Various kinds of immunosuppressive cells and chemicals in the tumor microenvironment (TME) accelerates CAR-T cell exhaustion, showing poor persistence in vivo. More and more researches have demonstrated that T cell fate is strongly associated with epigenetic regulation. Epigenetic modification is not a direct addition or deletion of DNA, but a reversible method including modifications on DNA and histones, and non-coding RNA (ncRNA)-mediated regulations. The change of epigenetic landscape in CAR-T cells largely determines the therapeutic performance in vivo. This research outlines three major barriers in CAR-T therapy, including T cell exhaustion, differentiation and infiltration. Additionally, the research elucidates several promising epigenetic reprogramming strategies to reduce CAR-T cells exhaustion, modulate the cell differentiation process, and enhance their infiltration into solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4165f9901246426b44e74fa80485927b8d025d28" target='_blank'>
              Epigenetic Strategies to Optimize CAR-T Therapy
              </a>
            </td>
          <td>
            Danyang Song
          </td>
          <td>2024-07-11</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ee3ef93f75d18cca805d005d653d62f6f76b88e" target='_blank'>
              Comprehensive pan-cancer analysis reveals that C5orf34 regulates the proliferation and mortality of lung cancer.
              </a>
            </td>
          <td>
            Mengmeng Yang, Yuhan Deng, Yu Ma, Chunli Song, Zhenhua Wu, Xiayimaierdan Yibulayin, Xiaohong Sun, Yunquan Guo, Dan He
          </td>
          <td>2024-06-29</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="OBJECTIVE
Ionizing radiation generates genomic instability by promoting the accumulation of chromosomal rearrangements. The oncogenic translocation RET/PTC1 is present in more than 70% of radiation-induced thyroid cancers. Both RET and CCDC6, the genes implicated in RET/PTC1, are found within common fragile sites - chromosomal regions prone to DNA breakage during slight replication stress. Given that irradiated cells become more susceptible to genomic destabilization due to the accumulation of replication-stress-related double-strand breaks (DSBs), we explored whether RET and CCDC6 exhibit DNA breakage under replicative stress several days post-irradiation of thyroid cells.


METHODS
We analyzed the dynamic of DNA replication in human thyroid epithelial cells (HThy-ori-3.1) 4 days post a 5-Gy exposure using molecular DNA combing. The DNA replication schedule was evaluated through replication-timing experiments. We implemented a ChIP-qPCR assay to determine whether the RET and CCDC6 genes break following irradiation.


RESULTS
Our study indicates that replicative stress, occurring several days post-irradiation in thyroid cells, primarily causes DSBs in the RET gene. We discovered that both the RET and CCDC6 genes undergo late replication in thyroid cells. However, only RET's replication rate is notably delayed after irradiation.


CONCLUSION
The findings suggest that post-irradiation in the RET gene causes a breakage in the replication fork, which could potentially invade another genomic area, including CCDC6. As a result, this could greatly contribute to the high prevalence of chromosomal RET/PTC rearrangements seen in patients exposed to external radiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eea49490c4342b6b738c402ce79c72ea23e5f552" target='_blank'>
              A post-irradiation-induced replication stress promotes RET proto-oncogene breakage.
              </a>
            </td>
          <td>
            Fabio Hecht, Laura Valerio, Carlos Frederico Lima Gonçalves, Marylin Harinquet, Rabii Ameziane El Hassani, Denise P. Carvalho, Stephane Koundrioukoff, Jean-Charles Cadoret, Corinne Dupuy
          </td>
          <td>2024-07-01</td>
          <td>European thyroid journal</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Ras genes are important oncogenes that are frequently mutated in cancer. Human oncogenic variants exhibit functional distinctions in terms of their representation in different cancer types, impact on cellular targets, and sensitivity to pharmacological treatments. However, how these distinct variants influence and respond to the cellular networks in which they are embedded is poorly understood. To identify novel participants in the complex interplay between Ras genotype and cell interaction networks in vivo, we have developed and tested an experimental framework using a simple vulva development assay in the nematode C. elegans. Using this system, we evaluate a set of Ras oncogenic substitution changes at G12, G13 and Q61. We find that these variants fall into distinct groups based on phenotypic differences, sensitivity to gene dose and inhibition of the downstream kinase MEK, and their response to genetic modulators that influence Ras activity in a non-autonomous manner. Together, our results demonstrate oncogenic C. elegans Ras variants exhibit clear distinctions in how they interface with the vulva development network, and show that extracellular modulators yield variant-restricted effects in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/675fab343bc71b75d1eb1f6e179c6e3a20afa2a0" target='_blank'>
              Functional distinction in oncogenic Ras variant activity in Caenorhabditis elegans.
              </a>
            </td>
          <td>
            Haimeng Lyu, H. Chamberlin
          </td>
          <td>2024-07-01</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="B cell progenitor acute lymphoblastic leukemia (BCP-ALL) is the most common childhood malignancy, driven by multiple genetic alterations that cause maturation arrest and accumulation of abnormal progenitor B cells. Current treatment protocols with chemotherapy have led to favorable outcomes but are associated with significant toxicity and risk of side effects, highlighting the necessity for highly effective, less toxic, targeted drugs, even in subtypes with a favorable outcome. Here, we used multimodal single-cell sequencing to delineate the transcriptional, epigenetic, and immunophenotypic characteristics of 23 childhood BCP-ALLs, belonging to the BCR::ABL1-positive, ETV6::RUNX1-positive, high hyperdiploid, and recently discovered DUX4-rearranged (DUX4-r) subtypes. Projection of the ALL cells along the normal hematopoietic differentiation axis revealed a diversity in the maturation pattern between the different BCP-ALL subtypes. Whereas the BCR::ABL1-, ETV6::RUNX1-positive, and high hyperdiploidy cells mainly showed similarities to normal pro-B cells, the DUX4-r ALL cells also displayed transcriptional signatures resembling mature B cells. Focusing on the DUX4-r subtype, we found that the blast population displayed multilineage priming toward non-hematopoietic cells, myeloid, and T cell lineages, but also an activation of PI3K/AKT signaling that sensitized the cells to PI3K inhibition in vivo. Given the multilineage priming of the DUX4-r blasts with aberrant expression of the myeloid marker CD371 (CLL-1), we generated chimeric antigen receptor T cells, which effectively eliminated DUX4-r ALL cells in vivo. These results provide a detailed characterization of BCP-ALL at the single-cell level and reveal therapeutic vulnerabilities in the DUX4-r subtype with implications for the understanding of ALL biology and new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1d7a915d43d3c2ff8b079040d662a79ad558986" target='_blank'>
              Single-cell genomics details the maturation block in BCP-ALL and identifies therapeutic vulnerabilities in DUX4-r cases.
              </a>
            </td>
          <td>
            Hanna Thorsson, Rasmus Henningsson, Noelia Puente-Moncada, Pablo Peña-Martínez, Ludvig Sjöström, H. Ågerstam, Carl Sandén, M. Rissler, A. Castor, H. Marquart, S. Modvig, K. Paulsson, C. Pronk, Kjeld Schmiegelow, Axel Hyrenius-Wittsten, Christina Orsmark-Pietras, H. Lilljebjörn, T. Fioretos
          </td>
          <td>2024-07-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2782891e3829481e71a01e8d6826ef2175cb2c49" target='_blank'>
              BUB1 regulates non-homologous end joining pathway to mediate radioresistance in triple-negative breast cancer
              </a>
            </td>
          <td>
            Sushmitha Sriramulu, Shivani Thoidingjam, Wei-Min Chen, Oudai Hassan, Farzan Siddiqui, Stephen L. Brown, Benjamin Movsas, Michael D Green, Anthony J. Davis, C. Speers, Eleanor Walker, S. Nyati
          </td>
          <td>2024-06-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract The comprehensive genomic analysis of the head and neck squamous cell carcinoma (HNSCC) oncogenome revealed the frequent loss of p16INK4A (CDKN2A) and amplification of cyclin D1 genes in most human papillomavirus–negative HNSCC lesions. However, cyclin-dependent kinase 4 and 6 (CDK4/6) inhibitors have shown modest effects in the clinic. The aberrant activation of the PI3K/mTOR pathway is highly prevalent in HNSCC, and recent clinical trials have shown promising clinical efficacy of mTOR inhibitors (mTORi) in the neoadjuvant and adjuvant settings but not in patients with advanced HNSCC. By implementing a kinome-wide CRISPR/Cas9 screen, we identified cell-cycle inhibition as a synthetic lethal target for mTORis. A combination of mTORi and palbociclib, a CDK4/6-specific inhibitor, showed strong synergism in HNSCC-derived cells in vitro and in vivo. Remarkably, we found that an adaptive increase in cyclin E1 (CCNE1) expression upon palbociclib treatment underlies the rapid acquired resistance to this CDK4/6 inhibitor. Mechanistically, mTORi inhibits the formation of eIF4G–CCNE1 mRNA complexes, with the consequent reduction in mRNA translation and CCNE1 protein expression. Our findings suggest that mTORi reverts the adaptive resistance to palbociclib. This provides a multimodal therapeutic option for HNSCC by cotargeting mTOR and CDK4/6, which in turn may halt the emergence of palbociclib resistance. Significance: A kinome-wide CRISPR/Cas9 screen identified cell-cycle inhibition as a synthetic lethal target of mTORis. A combination of mTORi and palbociclib, a CDK4/6-specific inhibitor, showed strong synergistic effects in HNSCC. Mechanistically, mTORis inhibited palbociclib-induced increase in CCNE1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d8015b7ff294926111ccbf929e88b6d333407e" target='_blank'>
              A Kinome-Wide Synthetic Lethal CRISPR/Cas9 Screen Reveals That mTOR Inhibition Prevents Adaptive Resistance to CDK4/CDK6 Blockade in HNSCC
              </a>
            </td>
          <td>
            Yusuke Goto, Keiichi Koshizuka, Toshinori Ando, Hiroki Izumi, Xingyu Wu, Kuniaki Sato, Tomohiko Ishikawa, Kyle Ford, Xiaodong Feng, Zhiyong Wang, Nadia Arang, M. Allevato, Ayush Kishore, Prashant Mali, J. Gutkind
          </td>
          <td>2024-07-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5096f65eca5658128881222e362e03cc7a2beb2" target='_blank'>
              H3K27ac acts as a molecular switch for doxorubicin-induced activation of cardiotoxic genes
              </a>
            </td>
          <td>
            Yu Hong, Xinlan Li, Jia Li, Qiuyi He, Manbing Huang, Yubo Tang, Xiao Chen, Jie Chen, Ke-Jing Tang, Chao Wei
          </td>
          <td>2024-07-16</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321126b0cb5f073f8273850a71ea2ffc1df84ce8" target='_blank'>
              Deciphering the role of post-translational modifications in fanconi anemia proteins and their influence on tumorigenesis.
              </a>
            </td>
          <td>
            Rui Ma, Xinlin Xu
          </td>
          <td>2024-06-15</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/029a0047ce9f458f1a9aeda0ae896bff711f7acb" target='_blank'>
              Saturation genome editing maps the functional spectrum of pathogenic VHL alleles
              </a>
            </td>
          <td>
            Megan Buckley, Chloé Terwagne, A. Ganner, L. Cubitt, Reid A. Brewer, Dong-Kyu Kim, Christina M. Kajba, Nicole Forrester, Phoebe Dace, Joachim De Jonghe, Scott T C Shepherd, Chelsea Sawyer, Mairead McEwen, Sven Diederichs, E. Neumann-Haefelin, S. Turajlic, E. Ivakine, Gregory M. Findlay
          </td>
          <td>2024-07-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The amplification of MET is a major cause of acquired resistance to targeted therapy in EGFR-mutant non-small-cell lung cancer (NSCLC), only to be temporarily restrained by the partial efficacy of MET inhibitors. This study reveals that the MET inhibitor has unexpectedly limited efficacy due to amplified MET triggering a strong positive feedback loop in the Wnt/β-catenin signaling pathway, allowing optimal functionality even when the MET pathway is suppressed again. To test this conjecture and specifically target the Wnt/β-catenin pathway, a cleverly designed Wnt condensative pro drug called WntSI is developed using reversible supramolecular self-assembly driven by liquidliquid phase separation (LLPS). This process involves a MET/pH-responsive peptide (Tyr-Pep) and a potent Wnt inhibitor known as CA. Upon recognition and phosphorylation of Tyr-Pep by over expressed MET in cells, it disrupts LLPS propensity and facilitates the disintegration of WntSI. Consequently,this enables it to suppress the carcinogenic effect mediated by β-catenin,effectively overcoming acquired resistance to EGFR-TKIs caused by MET amplification in both cell line-derived and patient-derived tumor xenograft (PDX) mouse models while maintaining exceptional biosecurity. This effective strategy not only suppresses the Wnt/β-catenin signaling pathway selectively, but also serves as an innovative example for pro-drug development through biologically responsive LLPS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7920ce89e1700b0b27a46645e0f64ce45c19988" target='_blank'>
              Reversing MET-Mediated Resistance in Oncogene-Driven NSCLC by MET-Activated Wnt Condensative Prodrug.
              </a>
            </td>
          <td>
            Na Liu, Xiaoqiang Zheng, Jin Yan, Aimin Jiang, Yu Yao, Wangxiao He
          </td>
          <td>2024-06-13</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3ef3973d270e32f48aaa19a00bef4e18bca5be" target='_blank'>
              Research Progress of Long Non-coding RNA-ZFAS1 in Malignant Tumors.
              </a>
            </td>
          <td>
            Xin Liu, Zhong Ma, Xianxu Zhang, Shicheng Li, Jiangdong An, Zhiqiang Luo
          </td>
          <td>2024-07-26</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Meiotic recombination is not only a key mechanism for sexual adaptation in eukaryotes but crucial for the accumulation of beneficial alleles in breeding populations. The effective manipulation of recombination requires, however, a better understanding of the mechanisms regulating the rate and distribution of recombination events in genomes. Here, we identified the genomic features that best explain the recombination variation among a diverse set of segregating populations of barley at a resolution of 1 Mbp and investigated how methylation and structural variants determine recombination hotspots and coldspots at a high-resolution of 10 kb. Hotspots were found to be in proximity to genes and the genetic effects not assigned to methylation were found to be the most important factor explaining differences in recombination rates among populations along with the methylation and the parental sequence divergence. Interestingly, the inheritance of a highly-methylated genomic fragment from one parent only was enough to generate a coldspot, but both parents must be equally low methylated at a genomic segment to allow a hotspot. The parental sequence divergence was shown to have a sigmoidal correlation with recombination indicating an upper limit of mismatch among homologous chromosomes for CO formation. Structural variants (SVs) were shown to suppress COs, and their type and size were not found to influence that effect. Methylation and SVs act jointly determining the location of coldspots in barley and the weight of their relative effect depends on the genomic region. Our findings suggest that recombination in barley is highly predictable, occurring mostly in multiple short sections located in the proximity to genes and being modulated by local levels of methylation and SV load.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff6d3ab3ee7ea3ec56157a8f8804849a64467540" target='_blank'>
              The role of methylation and structural variants in shaping the recombination landscape of barley
              </a>
            </td>
          <td>
            F. Casale, Christopher Arlt, M. Kühl, Jinquan Li, J. Engelhorn, Thomas Hartwig, Benjamin Stich
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d29a3f65594cd49446c09c17e43f47afc0e70ea3" target='_blank'>
              Effects of super-enhancers in cancer metastasis: mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Shenglan Liu, Wei Dai, Bei Jin, Feng Jiang, Hao Huang, Wen Hou, Jinxia Lan, Yanli Jin, Weijie Peng, Jingxuan Pan
          </td>
          <td>2024-06-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b0e09e628f8fcf9bb1ad3d19ff2d5112c171387" target='_blank'>
              Unmutated RRAS2 emerges as a key oncogene in post-partum-associated triple negative breast cancer
              </a>
            </td>
          <td>
            C. Cifuentes, Clara L. Oeste, Isabel Fernández-Pisonero, Alejandro Hortal, C. García-Macías, Jeanne Hochart, Regina Rubira, L. Horndler, Carlos Horndler, Xosé R. Bustelo, Balbino Alarcón
          </td>
          <td>2024-07-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/546925c2a469a030a3be13217deb97f4e2ca31ca" target='_blank'>
              Suppression of CTC1 inhibits hepatocellular carcinoma cell growth and enhances RHPS4 cytotoxicity.
              </a>
            </td>
          <td>
            Arda Kıpçak, Sıla Sezan, Ozum Karpat, Ezgi Kaya, S. Baylan, Ece Sarıyar, Cihangir Yandım, Z. F. Karagonlar
          </td>
          <td>2024-07-13</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a22775068c6dc76ab2de6304533dc82e709d532" target='_blank'>
              Germline variation contributes to false negatives in CRISPR-based experiments with varying burden across ancestries
              </a>
            </td>
          <td>
            Sean Misek, Aaron Fultineer, Jérémie Kalfon, J. Noorbakhsh, Isabella Boyle, Priyanka Roy, Joshua M. Dempster, Lia Petronio, Katherine H. Huang, Alham Saadat, Thomas M Green, A. Brown, John G Doench, D. Root, James M. McFarland, R. Beroukhim, Jesse S Boehm
          </td>
          <td>2024-06-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Premature aging is a hallmark of Down syndrome, caused by trisomy of human chromosome 21, but the reason is unclear and difficult to study in humans. We used an aneuploid model in wild yeast to show that chromosome amplification disrupts nutrient-induced cell-cycle arrest, quiescence entry, and healthy aging, across genetic backgrounds and amplified chromosomes. We discovered that these defects are due in part to aneuploidy-induced dysfunction in Ribosome Quality Control (RQC). Compared to euploids, aneuploids entering quiescence display aberrant ribosome profiles, accumulate RQC intermediates, and harbor an increased load of protein aggregates. Although they have normal proteasome capacity, aneuploids show signs of ubiquitin dysregulation, which impacts cyclin abundance to disrupt arrest. Remarkably, inducing ribosome stalling in euploids produces similar aberrations, while up-regulating limiting RQC subunits or proteins in ubiquitin metabolism alleviates many of the aneuploid defects. Our results provide implications for other aneuploidy disorders including Down syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c6d2e4dd1921e05fad92cf19e5a9d35d322983a" target='_blank'>
              Premature aging in aneuploid yeast is caused in part by aneuploidy-induced defects in Ribosome Quality Control
              </a>
            </td>
          <td>
            Leah E. Escalante, James Hose, Hollis Howe, Norah Paulsen, Michael Place, A. Gasch
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The cell cycle is tightly regulated to ensure controlled cell proliferation. Dysregulation of the cell cycle machinery is a hallmark of cancer that leads to unchecked growth. This review comprehensively analyzes key molecular regulators of the cell cycle and how they contribute to carcinogenesis when mutated or overexpressed. It focuses on cyclins, cyclin‐dependent kinases (CDKs), CDK inhibitors, checkpoint kinases, and mitotic regulators as therapeutic targets. Promising strategies include CDK4/6 inhibitors like palbociclib, ribociclib, and abemaciclib for breast cancer treatment. Other possible targets include the anaphase‐promoting complex/cyclosome (APC/C), Skp2, p21, and aurora kinase inhibitors. However, challenges with resistance have limited clinical successes so far. Future efforts should focus on combinatorial therapies, next‐generation inhibitors, and biomarkers for patient selection. Targeting the cell cycle holds promise but further optimization is necessary to fully exploit it as an anti‐cancer strategy across diverse malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90d0705d9bfabb467e3923d19e44652272f28955" target='_blank'>
              Cell cycle machinery in oncology: A comprehensive review of therapeutic targets
              </a>
            </td>
          <td>
            S. Cavalu, Amir Mohamed Abdelhamid, Sameh Saber, Elsayed A Elmorsy, R. S. Hamad, M. A. Abdel-Reheim, G. Yahya, Mohamed M Salama
          </td>
          <td>2024-06-07</td>
          <td>The FASEB Journal</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Cancer is the second leading cause of death worldwide, and although there have been advances in treatments, including immunotherapies, these often require biopsies which can be costly and invasive to obtain. Due to lack of pre-emptive cancer detection methods, many cases of cancer are detected at a late stage when the definitive symptoms appear. Plasma samples are relatively easy to obtain, and they can be used to monitor the molecular signatures of ongoing processes in the body. Profiling cell-free DNA is a popular method for monitoring cancer, but only a few studies have explored the use of cell-free RNA (cfRNA), which shows the recent footprint of systemic transcription. Here we developed FastNeo, a computational method for detecting known neoepitopes in human cfRNA. We show that neoepitopes and other biomarkers detected in cfRNA can discern Hepatocellular carcinoma (HCC) patients from the healthy patients with a sensitivity of 0.84 and a specificity of 0.79. For colorectal cancer we achieve a sensitivity of 0.87 and a specificity of 0.8. An important advantage of our cfRNA based approach is that it also reports putative neoepitopes which are important for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39049eaeea832dff014994be3a821e1580cf6286" target='_blank'>
              Detecting known neoepitopes, gene fusions, transposable elements, and circular RNAs in cell-free RNA
              </a>
            </td>
          <td>
            Mayank Mahajan, Martin Hemberg
          </td>
          <td>2024-06-09</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Retrons are promising gene editing tools because they can produce multi-copy single-stranded DNA in cells via self-primed reverse transcription. However, their potential for inserting genetic cargos in eukaryotes remains largely unexplored. Here we report the discovery and engineering of highly efficient retron-based gene editors for mammalian cells and vertebrates. Through bioinformatic analysis of metagenomic data and functional screening, we identified novel retron reverse transcriptases (RTs) that are highly active in mammalian cells. Rational design further improved the editing efficiency to levels comparable with conventional single-stranded oligodeoxynucleotide donors but from a genetically encoded cassette. Small molecule inhibitors of non-homologous end joining factors and Cas9-DNA repair protein fusions further increase homology-directed repair. Retron editors also exhibited robust activity with Cas12a nuclease and Cas9 nickase, expanding the genomic target scope and bypassing the need for a DNA double-stranded break. Using a rationally engineered retron editor, we incorporate a split GFP epitope tag for live cell imaging. Finally, we develop an all-RNA delivery strategy to enable DNA-free gene editing in cells and vertebrate embryos. This work establishes retron editors as a versatile and efficient tool for precise gene editing, offering new opportunities for biotechnology and biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8f2c620f5c7bf4a518c60425b0c63267672964" target='_blank'>
              Discovery and Engineering of Retrons for Precise Genome Editing
              </a>
            </td>
          <td>
            Jesse D. Buffington, Hung-Che Kuo, Kuang Hu, You-Chiun Chang, Kamyab Javanmardi, Brittney Voigt, Yi-Ru Li, Mary E. Little, Sravan K Devanathan, Blerta Xhemalçe, R. Gray, Ilya J. Finkelstein
          </td>
          <td>2024-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a plasma cell malignancy considered incurable despite the recent therapeutic advances. Effective targeted therapies are therefore needed. Our previous studies proved that inhibiting CDK7 impairs the cell cycle and metabolic programs by disrupting E2F1 and MYC transcriptional activities, making it an appealing therapeutic target for MM. Given that CDK7 and BRD4 operate in two distinct regulatory axes in MM, we hypothesized that targeting these two complementary pathways simultaneously would lead to a deeper and more durable response. Indeed, combination therapy had superior activity against MM cell growth and viability, and induced apoptosis to a greater extent than single-agent therapy in both cell lines and patient cells. This synergistic activity was also observed in Waldenström's Macroglobulinemia (WM) cells and with other inhibitors of E2F1 activity. Dual inhibition effectively impaired the MYC and E2F transcriptional programs and MM tumor growth and progression in xenograft animal models, providing evidence for combination therapy's potential as a therapeutic strategy in MM and WM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00bc75c793e43f9ea9231956f2203e28b541aa53" target='_blank'>
              Unlocking the therapeutic potential of selective CDK7 and BRD4 Inhibition against multiple myeloma cell growth.
              </a>
            </td>
          <td>
            Yao Yao, Shuhui Deng, Jessica Fong Ng, Mei Yuan, C. Chakraborty, Vera JoyWeiler, Nikhil Munshi, M. Fulciniti
          </td>
          <td>2024-07-25</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Cycling cells replicate their DNA during the S phase through a defined temporal program known as replication timing. Mutation frequencies, epigenetic chromatin states, and transcriptional activities are different for genomic regions that are replicated early and late in the S phase. Here, we found from ChIP-Seq analysis that DNA polymerase (Pol) κ is enriched in early-replicating genomic regions in HEK293T cells. In addition, by feeding cells with N 2-heptynyl-2′-deoxyguanosine followed by click chemistry–based enrichment and high-throughput sequencing, we observed elevated Pol κ activities in genomic regions that are replicated early in the S phase. On the basis of the established functions of Pol κ in accurate and efficient nucleotide insertion opposite endogenously induced N 2-modified dG lesions, our work suggests that active engagement of Pol κ may contribute to diminished mutation rates observed in early-replicating regions of the human genome, including cancer genomes. Together, our work expands the functions of Pol κ and offered a plausible mechanism underlying replication timing–dependent mutation accrual in the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417d907a79911389824959d3e561a3c8846017ac" target='_blank'>
              DNA polymerase κ participates in early S-phase DNA replication in human cells
              </a>
            </td>
          <td>
            Feng Tang, Yinan Wang, Ting Zhao, Jun Yuan, Andrew H Kellum, Yinsheng Wang
          </td>
          <td>2024-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Introduction: Ewing sarcoma is an aggressive type of pediatric bone cancer, characterized by a fusion transcription factor (EWSR1-FLI1 in 85% of cases1,2). Recurrent secondary mutations are limited to alterations in TP53, CDKN2A and most frequently STAG2 (15-22%3-6). STAG2-loss-of-function (LOF) mutations correlate with poor prognosis, metastasis and relapse3,5,6, and are shown to induce a metastatic phenotype in preclinical studies7,8. Despite intensive multimodal treatment (surgery, radiation and chemotherapy), the estimated 5-year survival is a mere 65-75% for patients with localized disease, and below 30% for those presenting with metastasis or relapse9. Targeted treatment is currently not available. Our current work aims at identifying synthetic lethal (SL) interactions with STAG2-LOF mutations in the context of Ewing sarcoma, which could represent potential avenues for targeted therapy. Methods: In order to identify STAG2-SL interactions, we have performed genome-wide and subsequent targeted CRISPR knockout (KO) screening in Ewing sarcoma cell lines and their isogenic CRISPR-engineered STAG2-KO clones. To capture Ewing sarcoma inter- and intra-tumoral heterogeneity, we engineered additional bulk STAG2-KO populations across a wide range of cell lines and primary samples, using a lentiviral approach. These isogenic (STAG2 +/-) models are used for high throughput drug screening, validation of screening results and investigation of underlying SL mechanisms. Results: The quality of our genetic screening is validated by the recurrent emergence of (untargetable) paralog STAG1 as the top STAG2-SL hit. Additional preliminary results include a targetable candidate that emerged from both CRISPR- and drug-based screens in most of our tested models. Validation experiments across a broader panel of isogenic cell lines and primary models are currently ongoing. Conclusion: In this work, we employ genetic and drug-based screening approaches across a panel of isogenic (STAG2+/-) Ewing sarcoma cell lines, aiming to identify STAG2-SL interactions. These approaches have yielded potential hits that are currently being investigated across a broader panel of Ewing cell lines and primary models. Ultimately, this work aims to identify novel targeted therapeutic strategies for this aggressive subtype of Ewing sarcoma. References: 1. Grünewald, T. et al. Nat. Rev. Dis. Prim. (2018). 2. Delattre, O. et al. Nature (1992). 3. Crompton, B. D. et al. Cancer Discov. (2014). 4. Brohl, A. S. et al. PLoS Genet. (2014). 5. Tirode, F. et al. Cancer Discov. (2014). 6. Shulman, D. S. et al. Br. J. Cancer (2022). 7. Surdez, D. et al. Cancer Cell (2021). 8. Adane, B. et al. Cancer Cell (2021). 9. Gaspar, N. et al. J. Clin. Oncol. (2015).
 Citation Format: Lieke Mous, Michelle Hofer, Dorita Zabėlė, Ingrid Bechtold, Sakina Zaidi, Camille Fouassier, Pierre Gestraud, Aurélien Boré, Raphaël Margueron, Marco Wachtel, Beat Schäfer, Olivier Delattre, Didier Surdez. Synthetic lethality in the context of STAG2-mutant Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69fa9a572c0a248e45db6876f4744f09311a3fb3" target='_blank'>
              Abstract A009: Synthetic lethality in the context of STAG2-mutant Ewing sarcoma
              </a>
            </td>
          <td>
            Lieke Mous, Michelle Hofer, Dorita Zabėlė, Ingrid Bechtold, Sakina Zaidi, Camille Fouassier, P. Gestraud, Aurélien Bore, Raphaël Margueron, Marco Wachtel, Beat W Schäfer, O. Delattre, Didier Surdez
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The rapid advancement of sequencing technologies has led to the identification of numerous mutations in cancer genomes, many of which are variants of unknown significance (VUS). Computational models are increasingly being used to predict the functional impact of these mutations, in both coding and noncoding regions. Integration of these models with emerging genomic datasets will refine our understanding of mutation effects and guide clinical decision making. Future advancements in modeling protein interactions and transcriptional regulation will further enhance our ability to interpret VUS. Periodic incorporation of these developments into VUS reclassification practice has the potential to significantly improve personalized cancer care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eedd26cdc00bf6f5a4bd765cde8d7a62e6722b3d" target='_blank'>
              Understanding variants of unknown significance: the computational frontier.
              </a>
            </td>
          <td>
            Xi Fu, Raúl Rabadán
          </td>
          <td>2024-06-07</td>
          <td>The oncologist</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dca29cb8a3526737bad31e766fb17c818ee6f4c" target='_blank'>
              Significance of targeting DNMT3A mutations in AML.
              </a>
            </td>
          <td>
            Guiqin Huang, Xiaoya Cai, Dengju Li
          </td>
          <td>2024-07-30</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/694fb0c278abd1957ad9c10855875c5ac27794f3" target='_blank'>
              PIM1 is a potential therapeutic target for the leukemogenic effects mediated by JAK/STAT pathway mutations in T-ALL/LBL
              </a>
            </td>
          <td>
            Antonio Lahera, Laura Vela-Martín, Pablo Fernández-Navarro, Pilar Llamas, J. López-Lorenzo, Javier Cornago, Javier Santos, J. Fernández-Piqueras, M. Villa-Morales
          </td>
          <td>2024-07-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="We describe exciting recent advances in fusion-driven sarcoma etiology, from an epigenetics perspective. By exploring the current state of the field, we identify and describe the central mechanisms that determine sarcomagenesis. Further, we discuss seminal studies in translational genomics, which enabled epigenetic characterization of fusion-driven sarcomas. Important context for epigenetic mechanisms include, but are not limited to, cell cycle and metabolism, core regulatory circuitry, 3-dimensional chromatin architectural dysregulation, integration with ATP-dependent chromatin remodeling, and translational animal modeling. Paradoxically, while the genetic requirements for oncogenic transformation are highly specific for the fusion partners, the epigenetic mechanisms we as a community have uncovered are categorically very broad. This dichotomy prompts the question of whether the investigation of rare disease epigenomics should prioritize studying individual cell populations, thereby examining whether the mechanisms of chromatin dysregulation are specific to a particular tumor. We review recent advances focusing on rhabdomyosarcoma, synovial sarcoma, alveolar soft part sarcoma, clear cell sarcoma, undifferentiated round cell sarcoma, Ewing sarcoma, myxoid/round liposarcoma, epithelioid hemangioendothelioma and desmoplastic round cell tumor. The growing number of groundbreaking discoveries in the field, motivated us to anticipate further exciting advances in the area of mechanistic epigenomics and direct targeting of fusion transcription factors in the years ahead.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01bc563cf3c1cf1b5a64e47d7f0fdef6c20adb4b" target='_blank'>
              Epigenetic determinants of fusion-driven sarcomas: paradigms and challenges
              </a>
            </td>
          <td>
            Benjamin Z. Stanton, S. Pomella
          </td>
          <td>2024-06-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Ewing sarcoma (EwS) is a rare and aggressive disease that typically affects the bone of children and young adults. Although there are moderate 5-year survival rates for primary disease, there is a high rate of recurrence and a lack of targeted therapies in this setting. Furthermore, pediatric exposure to chemotherapies leads to an increased risk of long-term complications, including secondary cancers, cardiomyopathy, and infertility. Therefore, there is a desperate need for targeted therapies which improve therapeutic outcomes and reduce exposure to chemotherapy in these patients. Like many pediatric cancers, EwS has a low mutational burden. Instead, it is driven by chromosomal translocation between a FET family member, EWSR1, and an ETS family member, usually FLI1. The EWSR1-FLI1 fusion protein acts as an aberrant and oncogenic transcription factor, disrupting cell cycle control, cell migration, and proliferation. ETS family members in these fusions retain their ability to bind to the SWI/SNF chromatin remodeling complex, further dysregulating DNA methylation and gene expression. Although targeting of the EWSR1 fusion has not yet been successful, these elevated levels of replication stress (RS) present a unique element of vulnerability within the cell. To this end, our lab has previously identified replication stress response (RSR) inhibitors, DDKi and WEE1i, which can be utilized to exploit the characteristic ability of EwS to maintain high levels of replication stress (RS) and push EwS cells into mitotic catastrophe in vitro. Here, we evaluate the efficacy of the replication stress response inhibitors (RSRi) TAK931 (DDKi) and MK1775 (WEE1i) with chemotherapy regimens commonly utilized in the relapsed/refractory setting in vivo. Using cell line derived xenografts in NSG mice, we show that DDKi and WEE1i with irinotecan limits tumor growth significantly more than existing therapy (temozolomide + irinotecan). Dosing schedules are optimized through dosing de-escalation and alternative-day dosing strategies. We further demonstrate the ability of our RSRi combination to effectively limit tumor growth over time in vivo even when irinotecan dosage is decreased. We go on to investigate the RSRi combination in the presence of ultra-low dose irinotecan. Finally, we utilize reduced representation bisulfite sequencing to examine the epigenetic effects of these treatment regimens. Overall, the exploitation of high endogenous replication stress in EwS by targeted RSRi presents a promising potential therapeutic option for this pediatric patient population.
 Citation Format: Emily Isenhart, Ajay Gupta, Bryan Gillard, Kristopher Attwood, Joyce Ohm. Exploiting endogenous replication stress with a novel targeted therapy in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr A015.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bd5187068239c0d44736d9520ada5b7a3335e9b" target='_blank'>
              Abstract A015: Exploiting endogenous replication stress with a novel targeted therapy in Ewing sarcoma
              </a>
            </td>
          <td>
            E. Isenhart, Ajay Gupta, Bryan M. Gillard, Kristopher Attwood, Joyce Ohm
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6265bb5f389f186810f9e194c6ec1e0b80e3ec45" target='_blank'>
              Cell surface CD55 traffics to the nucleus leading to cisplatin resistance and stemness by inducing PRC2 and H3K27 trimethylation on chromatin in ovarian cancer
              </a>
            </td>
          <td>
            Rashmi Bharti, G. Dey, Debjit Khan, Alex Myers, Olivia G Huffman, C. Saygin, C. Braley, Elliott Richards, Naseer Sangwan, Belinda Willard, Justin D Lathia, Paul L Fox, Feng Lin, B. Jha, J. M. Brown, Jennifer S Yu, Mohammed Dwidar, Amy Joehlin-Price, Roberto Vargas, Chad M Michener, M. Longworth, O. Reizes
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Wilms' tumor is a malignant neoplasm where current medical advancements have significantly improved survival rates; however, challenges persist such as the resistance of the tumor to chemotherapy drugs like doxorubicin. This necessitates higher dosages, leading to decreased sensitivity. However, using high doses of doxorubicin can have late effects on the heart. Unc-13 homolog B (UNC13B) may be involved in the drug resistance in several tumors, yet its role in modulating drug sensitivity in Wilms' tumor remains unexplored. UNC13B levels were quantified using reverse transcription-qPCR and Western blotting. The half-maximal inhibitory concentration for doxorubicin, vincristine, and actinomycin-D was determined using CCK-8 assays. Cell cycle and apoptosis were analyzed using flow cytometry, and lysosomal changes were observed using Lyso-Tracker staining. The present study initially evaluated UNC13B expression levels in the Wilms' tumor 17.94 cell line. Additionally, through short hairpin RNA-mediated knockdown, changes in doxorubicin sensitivity in 17.94 Wilms' tumor cells were assessed. Concurrently, preliminary investigations into the role of UNC13B in regulating lysosomes was performed, revealing a significant positive association between UNC13B levels and lysosome formation in the 17.94 cell line. Lysosomes likely serve a role in the sensitivity of Wilms' tumor cell lines to drugs. Elevated UNC13B expression was observed in the 17.94 Wilms' tumor cell line compared to normal kidney cells. UNC13B knockdown also resulted in increased apoptosis levels upon doxorubicin treatment. Immunofluorescence revealed UNC13B localization within cellular vesicles, and its knockdown significantly decreased lysosome levels. Overall, the findings of the present study demonstrate that UNC13B regulates the sensitivity of the Wilms' tumor 17.94 cell line to doxorubicin by modulating lysosome formation within cells. The results suggest that UNC13B is likely an enriched target involved in lysosomal regulation in certain tumors, offering a new approach for optimizing chemotherapy in Wilms' tumor and other cancers with high UNC13B expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe89ea39861be17508761078e7b43d988f8eb8e" target='_blank'>
              UNC13B regulates the sensitivity of Wilms' tumor cells to doxorubicin by modulating lysosomes
              </a>
            </td>
          <td>
            Xi Chen, Ying-ying Bao, Ge Sun, Xiaobo Wang, Jiajun Zhu
          </td>
          <td>2024-07-22</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="G-quadruplex (G4) sequences, which can fold into higher-order G4 structures, are abundant in the human genome and are over-represented in the promoter regions of many genes involved in human cancer initiation, progression, and metastasis. They are plausible targets for G4-binding small molecules, which would, in the case of promoter G4s, result in the transcriptional downregulation of these genes. However, structural information is currently available on only a very small number of G4s and their ligand complexes. This limitation, coupled with the currently restricted information on the G4-containing genes involved in most complex human cancers, has led to the development of a phenotypic-led approach to G4 ligand drug discovery. This approach was illustrated by the discovery of several generations of tri- and tetra-substituted naphthalene diimide (ND) ligands that were found to show potent growth inhibition in pancreatic cancer cell lines and are active in in vivo models for this hard-to-treat disease. The cycles of discovery have culminated in a highly potent tetra-substituted ND derivative, QN-302, which is currently being evaluated in a Phase 1 clinical trial. The major genes whose expression has been down-regulated by QN-302 are presented here: all contain G4 propensity and have been found to be up-regulated in human pancreatic cancer. Some of these genes are also upregulated in other human cancers, supporting the hypothesis that QN-302 is a pan-G4 drug of potential utility beyond pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f12d7617e98441c66bfb53bc671f9147eed0b1fe" target='_blank'>
              A Phenotypic Approach to the Discovery of Potent G-Quadruplex Targeted Drugs
              </a>
            </td>
          <td>
            Stephen Neidle
          </td>
          <td>2024-08-01</td>
          <td>Molecules</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
Several biomarkers are currently available to address targeted treatments in cancer patients, with lung malignancies representing one of the best examples.


CASE DESCRIPTION
We report the case of a patient affected by advanced non-small cell lung cancer with an uncommon histology and a complex biology. The use of a large next-generation sequencing (NGS) NGS panel allowed us to identify an extremely rare BRAF mutation (V600Q), a MET amplification, a high tumor mutational burden, a germline pathogenetic BRCA1 mutation and a homologous recombination deficiency through RAD51 assay. The treatment decision was driven by the abundance of molecular information.


CONCLUSIONS
This case highlights that an attentive and critical evaluation of molecular reports is key for the tailoring of treatment algorithms at the patient-level scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3210c6a7f6bdd7275b1dd451dca4a9b663d889f7" target='_blank'>
              Multiple targets, germline BRCA1 mutation and HRD in a lung cancer patient: Molecular considerations and treatment decision-making.
              </a>
            </td>
          <td>
            F. Perrone, Francesco Facchinetti, Benedetta Pellegrino, R. Minari, L. Gnetti, C. Azzoni, L. Bottarelli, N. Campanini, Juan Francisco Grau-Bejar, Anna Mingozzi, V. Cognigni, Marcello Tiseo
          </td>
          <td>2024-06-13</td>
          <td>Tumori</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Transcriptional enhancers can regulate individual or multiple genes through long-range three-dimensional (3D) genome interactions, and these interactions are commonly altered in cancer. Yet, the functional relationship between changes in 3D interactions associated with regulatory regions and differential gene expression appears context-dependent. In this study, we used HiChiP to capture changes in 3D genome interactions between active regulatory regions of endometrial cancer cells in response to estrogen treatment and uncovered significant differential long-range interactions that are strongly enriched for estrogen receptor α (ER) bound sites (ERBS). The ERBS anchoring differential loops with either a gene’s promoter or distal regions were correlated with larger transcriptional responses to estrogen compared to ERBS not involved in differential interactions. To functionally test this observation, CRISPR-based Enhancer-i was used to deactivate specific ERBS, which revealed a wide range of effects on the transcriptional response to estrogen. However, these effects are only subtly and not significantly stronger for ERBS in differential loops. In addition, we observed an enrichment of 3D interactions between the promoters of estrogen up-regulated genes and found that looped promoters can work together cooperatively. Overall, our work suggests that changes in 3D genome structure upon estrogen treatment identify some functionally important regulatory regions; however, these changes aren’t required for a transcriptional response to E2 in endometrial cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d0d32b1b8568f5b7b8778716b0bae7e2d53785" target='_blank'>
              Estrogen-induced chromatin looping changes identify a subset of functional regulatory elements
              </a>
            </td>
          <td>
            Hosiana Abewe, Alexandra Richey, Jeffery M. Vahrenkamp, Matthew Ginley-Hidinger, Craig M Rush, Noel Kitchen, Xiaoyang Zhang, Jason Gertz
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The main epidemiological and clinical data on colorectal cancer, as well as the features of molecular pathology, are discussed in the literature review. Efforts are being putto identify promising targets, particularly small non-coding nucleotide sequences, which can lead to new treatments for this disease. The discovery of significant mutations that contribute to the development of colorectal tumors is a major step in the advancement of molecular oncology, as these mutations give rise to heterogeneous tumors that differ in their origin. These mutations play a significant role in the progression of the disease and are now being targeted for treatment. The prognosis for a disease is influenced by the patient's sensitivity to antitumor therapy. However, new approaches to finding effective targets for antitumor treatments face new fundamental challenges due to clinical issues. These issues include the epigenetic regulation of markers of oncogenesis, which allows for the development of new therapeutic strategies. RNA interference, in particular, has been linked to non-copying RNA sequences such as microRNAs. These microRNAs are associated with certain processes that can influence all aspects of oncogenesis. The diversity of microRNAs allows for a differentiated approach when treating tumors in various locations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4796e80e11216dc287a04bf6cbff831c3d542a28" target='_blank'>
              New Perspectives in Colorectal Cancers Treatment, the Role of MicroRNAs.
              </a>
            </td>
          <td>
            V. A. Belova, Liudmila V Spirina, A. Avgustinovich, S. G. Afanasiev, Maxim Yu Volkov M, D. I. Azovsky, Alexander M Volkov, T. S. Klyushina
          </td>
          <td>2024-07-24</td>
          <td>Current drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Simple Summary The atavistic theory of cancer suggests that cancer development proceeds through cells reverting to ancient survival mechanisms. The Serial Atavism Model (SAM) expands on this, proposing that cancer progresses through multiple stages of reversion to earlier evolutionary forms with cells losing modern traits and regaining primitive ones. One example is the Warburg effect, where cancer cells prefer a type of energy production used by ancient cells before Earth’s atmosphere had oxygen. However, this review argues that cancer metabolism is too complex to be fully explained by this theory. Cancer cells exhibit a wide range of metabolic behaviors that do not fit neatly into a pattern of reverting to an ancient state, indicating that the SAM may not provide a complete understanding of cancer. Abstract The atavistic theory of cancer posits that cancer emerges and progresses through the reversion of cellular phenotypes to more ancestral types with genomic and epigenetic changes deactivating recently evolved genetic modules and activating ancient survival mechanisms. This theory aims at explaining the known cancer hallmarks and the paradox of cancer’s predictable progression despite the randomness of genetic mutations. Lineweaver and colleagues recently proposed the Serial Atavism Model (SAM), an enhanced version of the atavistic theory, which suggests that cancer progression involves multiple atavistic reversions where cells regress through evolutionary stages, losing recently evolved traits first and reactivating primitive ones later. The Warburg effect, where cancer cells upregulate glycolysis and lactate production in the presence of oxygen instead of using oxidative phosphorylation, is one of the key feature of the SAM. It is associated with the metabolism of ancient cells living on Earth before the oxygenation of the atmosphere. This review addresses the question of whether cancer metabolism can be considered as an atavistic reversion. By analyzing several known characteristics of cancer metabolism, we reach the conclusion that this version of the atavistic theory does not provide an adequate conceptual frame for cancer research. Cancer metabolism spans a whole spectrum of metabolic states which cannot be fully explained by a sequential reversion to an ancient state. Moreover, we interrogate the nature of cancer metabolism and discuss its characteristics within the framework of the SAM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5f41e098ff498aa32258ffc44af055a9872bf7" target='_blank'>
              Is Cancer Metabolism an Atavism?
              </a>
            </td>
          <td>
            Eric Fanchon, A. Stéphanou
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fca6bd99087dd947a662cc7f699c6b9db7da8b4" target='_blank'>
              Severe Combined Immunodeficiency from a Homozygous DNA Ligase 1 Mutant with Reduced Catalytic Activity but Increased Ligation Fidelity
              </a>
            </td>
          <td>
            Huda Alajlan, Vlad-Stefan Raducanu, Yossef Lopez de Los Santos, M. Tehseen, Hibah Alruwaili, Amer Al-Mazrou, Reem Mohammad, M. Al-Alwan, A. De Biasio, J. Merzaban, H. Al-Mousa, Samir M. Hamdan, A. Alazami
          </td>
          <td>2024-06-19</td>
          <td>Journal of Clinical Immunology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background: Actionable driver mutations account for 40–50% of NSCLC cases, and their identification clearly affects treatment choices and outcomes. Conversely, non-actionable mutations are genetic alterations that do not currently have established treatment implications. Among co-occurring alterations, the identification of concurrent actionable genomic alterations is a rare event, potentially impacting prognosis and treatment outcomes. Methods: We retrospectively evaluated the prevalence and patterns of concurrent driver genomic alterations in a large series of NSCLCs to investigate their association with clinicopathological characteristics, to assess the prognosis of patients whose tumor harbors concurrent alterations in the genes of interest and to explore their potential therapeutic implications. Results: Co-occurring driver alterations were identified in 26 out of 1520 patients with at least one gene alteration (1.7%). Within these cases, the incidence of concurrent actionable gene alterations was 39% (0.7% of the overall cohort). Among compound actionable gene mutations, EGFR was the most frequently involved gene (70%). The most frequent association was EGFR mutations with ROS1 rearrangement. Front-line targeted treatments were the preferred approach in patients with compound actionable mutations, with dismal median PFS observed (6 months). Conclusions: Advances in genomic profiling technologies are facilitating the identification of concurrent mutations. In patients with concurrent actionable gene alterations, integrated molecular and clinical data should be used to guide treatment decisions, always considering rebiopsy at the moment of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59022475d568d630ac2b88c69a13d9f08f1373da" target='_blank'>
              Co-Occurring Driver Genomic Alterations in Advanced Non-Small-Cell Lung Cancer (NSCLC): A Retrospective Analysis
              </a>
            </td>
          <td>
            I. Attili, Riccardo Asnaghi, D. Vacirca, Riccardo Adorisio, A. Rappa, Alberto Ranghiero, Mariano Lombardi, C. Corvaja, V. Fuorivia, Ambra Carnevale Schianca, P. Trillo Aliaga, G. Spitaleri, E. del Signore, J. Guarize, Lorenzo Spaggiari, E. Guerini-Rocco, Nicola Fusco, F. de Marinis, A. Passaro
          </td>
          <td>2024-07-31</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="
 KIF18A is a plus-end directed kinesin known to play a role in mitosis by facilitating chromosome alignment and spindle microtubule dynamics. Knockdown or knockout of KIF18A has been shown to inhibit proliferation in cells with whole genome doubling or aneuploidy, cellular states characterized by chromosomal rearrangements. These alterations render cells particularly vulnerable to disrupted mitosis upon KIF18A loss. As such, KIF18A is a compelling synthetic lethality target in a subset of tumors with chromosomal instability (CIN), which have ongoing chromosomal segregation defects. We have identified potent and selective small molecule inhibitors of KIF18A. A proprietary tool compound, ATX020, inhibits KIF18A ATPase activity with an IC50 of 0.014 µM; KIF18A is selectively inhibited over other kinesins including CENPE (IC50 > 10 µM) and EG5 (IC50 5.87 µM). ATX020 exhibits robust cellular activity, with anti-proliferative activity observed in chromosomally instable ovarian cancer cell lines such as OVCAR-3 (IC50 0.053 µM) and OVCAR-8 (IC50 0.54 µM). Upon treatment with ATX020, Annexin V is robustly induced in sensitive cell lines, demonstrating that KIF18A-dependent cell lines undergo apoptosis upon KIF18A inhibition. In KIF18Ai sensitive cells, a dose-dependent upregulation of phospho-Histone H3 and gamma-H2AX is observed upon ATX020 treatment in OVCAR-3 and OVCAR-8 cells, consistent with the induction of mitotic arrest and DNA damage. In these sensitive cell lines, ATX020 induces a dose-dependent cell cycle arrest in G2/M, consistent with the role of KIF18A in facilitating mitosis. OVCAR-3 cells treated with ATX020 exhibit fragmented nuclei and malformed mitotic spindles. In contrast, CIN negative Ovarian cancer cells that are insensitive to KIF18A loss, such as A2780 and OVK18, proliferate normally upon ATX020 treatment, cell division is unaffected, and no induction of Annexin V, phospho-Histone H3, and gamma-H2AX is observed. This selective dependency is also observed in vivo, where administration of ATX020 leads to dose-dependent tumor growth inhibition in an OVCAR-3 cancer cell line-derived xenograft model, with significant tumor regression observed at 100 mpk/day. In contrast, OVK18 tumors are unaffected by ATX020 administration, as expected based on the lack of cell cycle and proliferation effects in that cell line. Together, these data demonstrate a critical role for KIF18A in facilitating mitosis in a subset of chromosomally instable cells. These effects are selective, with KIF18A-independent cell lines proceeding through the cell cycle normally in the presence of inhibitors. Small molecule inhibition of KIF18A is therefore expected to have robust efficacy in solid tumors with high levels of chromosomal instability such as Ovarian cancer.
 Citation Format: Maureen S. Lynes, Laura Ghisolfi, Sanjoy Khan, Taylor Hotz, Brian A. Sparling, Mary-Margaret Zablocki, Deepali Gotur, P. Ann Boriack-Sjodin, Kenneth W. Duncan, Stephen J. Blakemore, Serena J. Silver, Robert A. Copeland. KIF18A inhibition, via ATX020, leads to mitotic arrest and robust anti-tumor activity through a synthetic lethal interaction with chromosome instability [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c93d87266fb08c0f53f51496b94330637b88929" target='_blank'>
              Abstract PR001: KIF18A inhibition, via ATX020, leads to mitotic arrest and robust anti-tumor activity through a synthetic lethal interaction with chromosome instability
              </a>
            </td>
          <td>
            Maureen S. Lynes*, Laura Ghisolfi†, Sanjoy Khan, Taylor Hotz, Brian A. Sparling, M. Zablocki, D. Gotur, P. Boriack-Sjodin, K. Duncan, Stephen J. Blakemore, Serena J. Silver, Robert A. Copeland
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Motivation Eukaryotic cells contain organelles called mitochondria that have their own genome. Most cells contain thousands of mitochondria which replicate, even in nondividing cells, by means of a relatively error-prone process resulting in somatic mutations in their genome. Because of the higher mutation rate compared to the nuclear genome, mitochondrial mutations have been used to track cellular lineage, particularly using single-cell sequencing that measures mitochondrial mutations in individual cells. However, existing methods to infer the cell lineage tree from mitochondrial mutations do not model “heteroplasmy,” which is the presence of multiple mitochondrial clones with distinct sets of mutations in an individual cell. Single-cell sequencing data thus provide a mixture of the mitochondrial clones in individual cells, with the ancestral relationships between these clones described by a mitochondrial clone tree. While deconvolution of somatic mutations from a mixture of evolutionarily related genomes has been extensively studied in the context of bulk sequencing of cancer tumor samples, the problem of mitochondrial deconvolution has the additional constraint that the mitochondrial clone tree must be concordant with the cell lineage tree. Results We formalize the problem of inferring a concordant pair of a mitochondrial clone tree and a cell lineage tree from single-cell sequencing data as the Nested Perfect Phylogeny Mixture (NPPM) problem. We derive a combinatorial characterization of the solutions to the NPPM problem, and formulate an algorithm, MERLIN, to solve this problem exactly using a mixed integer linear program. We show on simulated data that MERLIN outperforms existing methods that do not model mitochondrial heteroplasmy nor the concordance between the mitochondrial clone tree and the cell lineage tree. We use MERLIN to analyze single-cell whole-genome sequencing data of 5220 cells of a gastric cancer cell line and show that MERLIN infers a more biologically plausible cell lineage tree and mitochondrial clone tree compared to existing methods. Availability and implementation https://github.com/raphael-group/MERLIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4e8eaf7dda8383dae6ef320720e71a06bb73607" target='_blank'>
              Joint inference of cell lineage and mitochondrial evolution from single-cell sequencing data
              </a>
            </td>
          <td>
            P. Sashittal, Viola Chen, Amey P. Pasarkar, Benjamin J. Raphael
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The emergence of mutational signatures in cancer research represents a paradigm shift, offering profound insights into tumor biology, therapeutic strategies, and risk assessment. A recent article published in The Journal of Pathology delves into the significance of mutational signatures, assessing their role as molecular fingerprints that illuminate the etiology of individual cancer cases. By deciphering the diverse patterns of DNA alterations, in the context of urachal and nonurachal bladder adenocarcinomas, researchers can unravel intrinsic and environmental risk factors shaping cancer development. The study highlighted a predominance of environmental risk factors in Chinese populations affected by bladder adenocarcinomas. In this context, in this commentary, we highlight some of the potential applications of molecular signatures in differential diagnosis and therapy prediction. As the field continues to evolve, a deeper understanding of mutational signatures promises to revolutionize precision oncology by offering personalized approaches informed by comprehensive mutational patterns of tumors. © 2024 The Authors. The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8befd4af10300f9c01154cd2d4d75cd58d605346" target='_blank'>
              Beyond driver mutations: exploring the landscape of mutational signatures in adenocarcinomas of the bladder†
              </a>
            </td>
          <td>
            Tibor Szarvas, Henning Reis
          </td>
          <td>2024-06-07</td>
          <td>The Journal of Pathology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genome editing technologies have seen remarkable progress in recent years, enabling precise regulation of exogenous and endogenous genes. These advances have been extensively applied to the engineering of human T lymphocytes, leading to the development of practice changing therapies for patients with cancer and the promise of synthetic immune cell therapies for a variety of non-malignant diseases. Many distinct conceptual and technical approaches have been used to edit T-cell genomes, however targeted assessments of which techniques are most effective for manufacturing, gene editing and transgene expression are rarely reported. Through extensive comparative evaluation, we identified methods that most effectively enhance engineering of research-scale and pre-clinical T-cell products at critical stages of manufacturing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8cdbd85fcdf596eac3e46738e8defa5a7f38e6" target='_blank'>
              Identification of core techniques that enhance genome editing of human T cells expressing synthetic antigen receptors.
              </a>
            </td>
          <td>
            Ju-Fang Chang, Nils Wellhausen, Nils W Engel, Jack H Landmann, Caitlin R. Hopkins, January Salas-McKee, Adham S Bear, M. E. Selli, S. Agarwal, J. Jadlowsky, Gerald P. Linette, Saar Gill, Carl H. June, J. Fraietta, Nathan Singh
          </td>
          <td>2024-06-13</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The identification of therapeutic target genes that are functionally involved in stemness is crucial to effectively cure patients with metastatic carcinoma. We have previously reported that inhibition of ribosomal protein L9 (RPL9) expression suppresses the growth of colorectal cancer (CRC) cells by inactivating the inhibitor of DNA-binding 1 (ID-1) signaling axis, which is functionally associated with cancer cell survival. In addition to cell proliferation, ID-1 is also involved in the maintenance of cancer stemness. Thus, we aimed in this study to investigate whether the function of RPL9 could correlate with CRC stem cell-like properties. Here, we demonstrated that siRNA silencing of RPL9 reduced the invasiveness and migrative capabilities of HT29 and HCT116 parental cell populations and the capacity for sphere formation in the HT29 parental cell population. CD133+ cancer stem cells (CSCs) were then separated from CD133- cancer cells of the HT29 parental cell culture and treated with RPL9-specific siRNAs to verify the effects of RPL9 targeting on stemness. As a result, knockdown of RPL9 significantly suppressed the proliferative potential of CD133+ colorectal CSCs, accompanied by a reduction in CD133, ID-1, and p-IκBα levels. In line with these molecular alterations, targeting RPL9 inhibited the invasion, migration, and sphere-forming capacity of CD133+ HT29 CSCs. Taken together, these findings suggest that RPL9 promotes CRC stemness via ID-1 and that RPL9 could be a potential therapeutic target for both primary CRC treatment and the prevention of metastasis and/or recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c06d9c01b79a11ea146cbbd1ee6450fd736a819" target='_blank'>
              Ribosomal Protein L9 Maintains Stemness of Colorectal Cancer via an ID-1 Dependent Mechanism
              </a>
            </td>
          <td>
            Eun-Hye Jeon, So-Young Park, Keon Uk Park, Yun-Han Lee
          </td>
          <td>2024-06-30</td>
          <td>Journal of Cancer Prevention</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Down syndrome is a genetic-based disorder that results from the triplication of chromosome 21, leading to an overexpression of many triplicated genes, including the gene encoding Dual-Specificity Tyrosine Phosphorylation-Regulated Kinase 1A (DYRK1A). This protein has been observed to regulate numerous cellular processes, including cell proliferation, cell functioning, differentiation, and apoptosis. Consequently, an overexpression of DYRK1A has been reported to result in cognitive impairment, a key phenotype of individuals with Down syndrome. Therefore, downregulating DYRK1A has been explored as a potential therapeutic strategy for Down syndrome, with promising results observed from in vivo mouse models and human clinical trials that administered epigallocatechin gallate. Current DYRK1A inhibitors target the protein function directly, which tends to exhibit low specificity and selectivity, making them unfeasible for clinical or research purposes. On the other hand, antisense oligonucleotides (ASOs) offer a more selective therapeutic strategy to downregulate DYRK1A expression at the gene transcript level. Advances in ASO research have led to the discovery of numerous chemical modifications that increase ASO potency, specificity, and stability. Recently, several ASOs have been approved by the U.S. Food and Drug Administration to address neuromuscular and neurological conditions, laying the foundation for future ASO therapeutics. The limitations of ASOs, including their high production cost and difficulty delivering to target tissues can be overcome by further advances in ASO design. DYRK1A targeted ASOs could be a viable therapeutic approach to improve the quality of life for individuals with Down syndrome and their families.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6325ca7451f96825cdc01a440a3d532d3e9fb0b3" target='_blank'>
              Down syndrome and DYRK1A overexpression: relationships and future therapeutic directions
              </a>
            </td>
          <td>
            Aidan J. Murphy, Steve D. Wilton, M. Aung-Htut, Craig S McIntosh
          </td>
          <td>2024-07-24</td>
          <td>Frontiers in Molecular Neuroscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Nonsense mutations are genetic mutations that create premature termination codons (PTCs), leading to truncated, defective proteins in diseases such as cystic fibrosis, neurofibromatosis type 1, Dravet syndrome, Hurler syndrome, Beta thalassemia, inherited bone marrow failure syndromes, Duchenne muscular dystrophy, and even cancer. These mutations can also trigger a cellular surveillance mechanism known as nonsense-mediated mRNA decay (NMD) that degrades the PTC-containing mRNA. The activation of NMD can attenuate the consequences of truncated, defective, and potentially toxic proteins in the cell. Since approximately 20% of all single-point mutations are disease-causing nonsense mutations, it is not surprising that this field has received significant attention, resulting in a remarkable advancement in recent years. In fact, since our last review on this topic, new examples of nonsense suppression approaches have been reported, namely new ways of promoting the translational readthrough of PTCs or inhibiting the NMD pathway. With this review, we update the state-of-the-art technologies in nonsense suppression, focusing on novel modalities with therapeutic potential, such as small molecules (readthrough agents, NMD inhibitors, and molecular glue degraders); antisense oligonucleotides; tRNA suppressors; ADAR-mediated RNA editing; targeted pseudouridylation; and gene/base editing. While these various modalities have significantly advanced in their development stage since our last review, each has advantages (e.g., ease of delivery and specificity) and disadvantages (manufacturing complexity and off-target effect potential), which we discuss here.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/622123aafe7c9d07d73b7b284c889caac3669566" target='_blank'>
              Therapeutic Nonsense Suppression Modalities: From Small Molecules to Nucleic Acid-Based Approaches
              </a>
            </td>
          <td>
            Pedro Morais, Rui Zhang, Yi-Tao Yu
          </td>
          <td>2024-06-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Functional copy-number alterations (fCNAs) are DNA copy-number changes with concordant differential gene expression. These are less likely to be bystander genetic lesions and could serve as robust and reproducible tumor biomarkers. To identify candidate fCNAs in neuroendocrine tumors (NETs), we integrated chromosomal microarray (CMA) and RNA-seq differential gene-expression data from 31 pancreatic (pNETs) and 33 small-bowel neuroendocrine tumors (sbNETs). Tumors were resected from 47 early-disease-progression (<24 months) and 17 late-disease-progression (>24 months) patients. Candidate fCNAs that accurately differentiated these groups in this discovery cohort were then replicated using fluorescence in situ hybridization (FISH) on formalin-fixed, paraffin-embedded (FFPE) tissues in a larger validation cohort of 60 pNETs and 82 sbNETs (52 early- and 65 late-disease-progression samples). Logistic regression analysis revealed the predictive ability of these biomarkers, as well as the assay-performance metrics of sensitivity, specificity, and area under the curve. Our results indicate that copy-number changes at chromosomal loci 4p16.3, 7q31.2, 9p21.3, 17q12, 18q21.2, and 19q12 may be used as diagnostic and prognostic NET biomarkers. This involves a rapid, cost-effective approach to determine the primary tumor site for patients with metastatic liver NETs and to guide risk-stratified therapeutic decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/976250a22dce56b41eaaf1761c3d5be047660920" target='_blank'>
              Functional Copy-Number Alterations as Diagnostic and Prognostic Biomarkers in Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Hayley Vaughn, Heather Major, Evangeline Kadera, Kendall Keck, Timothy Dunham, Qining Qian, Bartley Brown, Aaron Scott, Andrew M. Bellizzi, Terry A. Braun, Patrick Breheny, Dawn E. Quelle, James R. Howe, Benjamin W Darbro
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Acquired resistance poses a significant challenge to the long-term efficacy of RAS inhibitors in clinical settings (Awad et al, 2021). Genome-wide CRISPR screening efforts, conducted by us and others in the field, have identified key components of the Hippo pathway as major mediators of resistance to RAS inhibitors (Mukhopadhyay et al., 2023, Edwards et al., 2023). Specifically, the oncogenic transcriptional co-regulators YAP and TAZ, which require binding to the TEAD transcription factors to elicit their activity, have emerged as pivotal players in resistance mechanisms to MAPK pathway targeted therapies. In this work, we demonstrate that YAP overexpression is a driver of resistance to RAS(ON) multi-selective and RAS(ON) G12C-selective inhibitors. The YAP transcriptional signature is elevated in cells with acquired resistance to RAS(ON) inhibitors, and combination with TEAD inhibitors enhances their sensitivity to RAS(ON) inhibition. Notably, phosphoproteomics and imaging studies revealed that the YAP cytoplasmic retention mark is reduced upon treatment of cancer cells with RAS(ON) multi-selective inhibitor, leading to the translocation of YAP to the nucleus and its subsequent activation. Furthermore, a pan-TEAD inhibitor exhibits a high degree of synergy with RAS(ON) inhibitors in a panel of KRAS mutant cell lines in vitro. We are currently investigating the combination of RAS(ON) and TEAD inhibitors in vivo. Our studies support further investigation of combination strategies using RAS(ON) and TEAD inhibitors to mitigate the emergence of persister cells and overcome resistance to monotherapy.
 Citation Format: Vidyasiri Vemulapalli, Phuong Dinh, Mariela Moreno Ayala, Zheng Zhang, Mark Labrecque, Ida Aronchik, Jingjing Jiang, Mallika Singh, Elsa Quintana, Caroline Weller, David Wildes. TEAD inhibition overcomes YAP/TAZ-driven resistance to RAS(ON) inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f118b609145daaebbd410b706ad9c33d10c87e8c" target='_blank'>
              Abstract B022: TEAD inhibition overcomes YAP/TAZ-driven resistance to RAS(ON) inhibitors
              </a>
            </td>
          <td>
            Vidyasiri Vemulapalli, Phuong Dinh, Mariela Moreno Ayala, Zheng Zhang, Mark Labrecque, Ida Aronchik, Jingjing Jiang, Mallika Singh, Elsa Quintana, Caroline Weller, D. Wildes
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>14</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [16],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>