<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-16 06:31:20 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>7</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Summary Phenotypic adaptation, the ability of cells to change phenotype in response to external pressures, has been identified as a driver of drug resistance in cancer. To quantify phenotypic adaptation in BRAFV600E-mutant melanoma, we develop a theoretical model that emerges from data analysis of WM239A-BRAFV600E growth rates in response to drug challenge with the BRAF-inhibitor encorafenib. Our model constitutes a cell population model in which each cell is individually described by one of multiple discrete and plastic phenotype states that are directly linked to drug-dependent net growth rates and, by extension, drug resistance. Data-matched simulations reveal that phenotypic adaptation in the cells is directed towards states of high net growth rates, which enables evasion of drug-effects. The model subsequently provides an explanation for when and why intermittent treatments outperform continuous treatments in vitro, and demonstrates the benefits of not only targeting, but also leveraging, phenotypic adaptation in treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db72a236ec0fc5bd479b42430df6525df1d343c7" target='_blank'>
              Growth rate-driven modelling reveals how phenotypic adaptation drives drug resistance in BRAFV600E-mutant melanoma
              </a>
            </td>
          <td>
            Sara Hamis, A. Browning, A. Jenner, Chiara Villa, Philip Maini, Tyler Cassidy
          </td>
          <td>2024-08-18</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Drug resistance is the major cause of therapeutic failure in high-grade serous ovarian cancer (HGSOC). Yet, the mechanisms by which tumors evolve to drug resistant states remains largely unknown. To address this, we aimed to exploit clone-specific genomic structural variations by combining scaled single-cell whole genome sequencing with longitudinally collected cell-free DNA (cfDNA), enabling clonal tracking before, during and after treatment. We developed a cfDNA hybrid capture, deep sequencing approach based on leveraging clone-specific structural variants as endogenous barcodes, with orders of magnitude lower error rates than single nucleotide variants in ctDNA (circulating tumor DNA) detection, demonstrated on 19 patients at baseline. We then applied this to monitor and model clonal evolution over several years in ten HGSOC patients treated with systemic therapy from diagnosis through recurrence. We found drug resistance to be polyclonal in most cases, but frequently dominated by a single high-fitness and expanding clone, reducing clonal diversity in the relapsed disease state in most patients. Drug-resistant clones frequently displayed notable genomic features, including high-level amplifications of oncogenes such as CCNE1, RAB25, NOTCH3, and ERBB2. Using a population genetics Wright-Fisher model, we found evolutionary trajectories of these features were consistent with drug-induced positive selection. In select cases, these alterations impacted selection of secondary lines of therapy with positive patient outcomes. For cases with matched single-cell RNA sequencing data, pre-existing and genomically encoded phenotypic states such as upregulation of EMT and VEGF were linked to drug resistance. Together, our findings indicate that drug resistant states in HGSOC pre-exist at diagnosis and lead to dramatic clonal expansions that alter clonal composition at the time of relapse. We suggest that combining tumor single cell sequencing with cfDNA enables clonal tracking in patients and harbors potential for evolution-informed adaptive treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1441270a54104f75e7713e2bb6b7323866959" target='_blank'>
              Tracking clonal evolution of drug resistance in ovarian cancer patients by exploiting structural variants in cfDNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Gryte Satas, Hannah J. Lees, Jake June-Koo Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, N. Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitriy Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Phenotypic plasticity is a recognized mechanism driving therapeutic resistance in patients with prostate cancer. Although underlying molecular causations driving phenotypic plasticity have been identified, therapeutic success is yet to be achieved. To identify putative master regulator transcription factors (MR-TF) driving phenotypic plasticity in prostate cancer, this work utilized a multiomic approach using genetically engineered mouse models of prostate cancer combined with patient data to identify MYB proto-oncogene like 2 (MYBL2) as a significantly enriched transcription factor in prostate cancer exhibiting phenotypic plasticity. Genetic inhibition of Mybl2 using independent murine prostate cancer cell lines representing phenotypic plasticity demonstrated Mybl2 loss significantly decreased in vivo growth as well as cell fitness and repressed gene expression signatures involved in pluripotency and stemness. Because MYBL2 is currently not druggable, a MYBL2 gene signature was employed to identify cyclin-dependent kinase-2 (CDK2) as a potential therapeutic target. CDK2 inhibition phenocopied genetic loss of Mybl2 and significantly decreased in vivo tumor growth associated with enrichment of DNA damage. Together, this work demonstrates MYBL2 as an important MR-TF driving phenotypic plasticity in prostate cancer. Furthermore, high MYBL2 activity identifies prostate cancer that would be responsive to CDK2 inhibition. Significance: Prostate cancers that escape therapy targeting the androgen receptor signaling pathways via phenotypic plasticity are currently untreatable. Our study identifies MYBL2 as a MR-TF in phenotypic plastic prostate cancer and implicates CDK2 inhibition as a novel therapeutic target for this most lethal subtype of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d7169da5236ce487e4af9bc41a5465e1a1c863" target='_blank'>
              MYBL2 Drives Prostate Cancer Plasticity: Inhibiting Its Transcriptional Target CDK2 for RB1-Deficient Neuroendocrine Prostate Cancer
              </a>
            </td>
          <td>
            Beatriz German, S. A. Alaiwi, Kun-Lin Ho, Jagpreet S Nanda, Marcos A S Fonseca, Deborah L. Burkhart, Anjali V. Sheahan, Hannah E Bergom, Katherine L. Morel, H. Beltran, Justin H. Hwang, M. Freedman, K. Lawrenson, Leigh Ellis
          </td>
          <td>2024-08-08</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single-nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosome 20q or 1q, both known for conferring an in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings showed higher loads of both CNVs and SNVs at later passages, which were due to the cumulative acquisition of mutations over a longer time in culture, and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs followed the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid of cancer-associated mutations. This suggests that recurrent chromosomal abnormalities are potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36cc208bebf01a60a96b6bcb671a8cdcf9d8bbf6" target='_blank'>
              De Novo Cancer Mutations Frequently Associate with Recurrent Chromosomal Abnormalities during Long-Term Human Pluripotent Stem Cell Culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huyghebaert, M. Regin, Mai Chi Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Connecting genetic variation to phenotypes and understanding the underlying biological mechanisms has been a fundamental goal of biological genetics. Here, we used the association analysis to identify a Vip3Aa resistance-associated genomic region in a strain of fall armyworm, JC-R, which exhibits >5000-fold resistance to the Bt toxin Vip3Aa. However, through various analytical approaches and fine-scale mapping across different populations, we demonstrated that this genomic region exhibits strong genetic linkage. The chromosome-level genome of JC-R and its parent strain JC-S were assembled, and extensive structural variations in the linkage regions were identified, which could be responsible for maintaining the linkage. To identify the causal variation within this linked region, a chromosome fragment stepwise knockout strategy based on CRISPR/Cas9 was developed. By crossing with the resistant strain and phenotyping segregating offspring on Vip3Aa-containing diet, we identified a chromosomal segment, KO8, containing the resistant gene. Subsequently, we conducted a comprehensive analysis of the variations in the KO8 region using multi-omics approaches, including genomic data, RNA-seq, proteomic, PacBio long read Iso-seq, and phosphoproteomic data. This analysis identified multiple variations in the chitin synthase gene CHS2, including amino acid substitution, alternative splicing, and changes in phosphorylation sites. After knocking out the CHS2, larvae exhibited over 6777-fold resistance to Vip3Aa. These results demonstrate that the chromosome fragment stepwise knockout strategy is a viable approach for studying complex genomic regions, and highlight the value of comprehensive analysis of genetic variations using multi-omics data. The identified candidate gene could potentially advance monitoring and management of pest resistance to Vip3Aa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca50d6fcbaf49cb004ae82c9568f261b121333e" target='_blank'>
              CRISPR-mediated chromosome deletion facilitates genetic mapping of Vip3Aa resistance gene within complex genomic region in an invasive global pest
              </a>
            </td>
          <td>
            M. Jin, Yinxue Shan, Yan Peng, Senlin Chen, Xuanhe Zhou, Kaiyu Liu, Yutao Xiao
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Directed evolution is an efficient strategy to steer protein function to either understand specific biological properties or develop new biotechnology tools. Currently available methods for targeted mutagenesis in human cells rely on deaminases which can only modify specific bases, limiting the region of sequence space explored during evolution. By leveraging CRISPR-Cas9 coupled with an error-prone variant of human DNA polymerase λ, here we developed CRISPR-λ, an unbiased mutagenesis tool for directed evolution in human cells. We evaluated CRISPR-λ by reverting the fluorescence of a mutated EGFP and characterized it using ultra-deep sequencing. The mutagenic activity of CRISPR-λ spans 36-46 nucleotides from the target site, with a mutation frequency as high as 1.4e-4 substitutions per base and with no bias for specific nucleotide substitutions. The versatility of CRISPR-λ extends beyond base substitution, enabling modifications of the target gene through insertions and deletions, thereby broadening its potential for genetic diversification. We validated the efficacy of CRISPR-λ in directed evolution approaches by functionally reverting a mutated blasticidin resistance gene. Furthermore, we demonstrated the sequence diversification power of CRISPR-λ by steering the syncytia formation activity of the SARS-CoV-2 Spike envelope protein in cultured cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d747466dd7f2edf4242403524fc8d70fe775ed9" target='_blank'>
              DNA polymerase λ-driven targeted mutagenesis for directed evolution in human cells
              </a>
            </td>
          <td>
            Davide Aiello, Matteo Ciciani, Federica Marelli, Marta Stancampiano, Veronica De Sanctis, R. Bertorelli, E. Kheir, G. Maule, A. Cereseto, Daniele Arosio
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Ionizing radiation (IR) in high doses is generally lethal to most organisms. Investigating mechanisms of radiation resistance is crucial for gaining insights into the underlying cellular responses and understanding the damaging effects of IR. In this study, we conducted a comprehensive analysis of sequencing data from an evolutionary experiment aimed at understanding the genetic adaptations to ionizing radiation in Escherichia coli. By including previously neglected synonymous mutations, we identified the rpsH c.294T > G variant, which emerged in all 17 examined isolates across four subpopulations. The identified variant is a synonymous mutation affecting the 30S ribosomal protein S8, and consistently exhibited high detection and low allele frequencies in all subpopulations. This variant, along with two additional rpsH variants, potentially influences translational control of the ribosomal spc operon. The early emergence and stability of these variants suggest their role in adapting to environmental stress, possibly contributing to radiation resistance. Our findings shed light on the dynamics of ribosomal variants during the evolutionary process and their potential role in stress adaptation, providing valuable implications for understanding clinical radiation sensitivity and improving radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10a94830c53f3f47e59574a18dce14e20ea55a06" target='_blank'>
              Synonymous rpsH variants: the common denominator in Escherichia coli adapting to ionizing radiation
              </a>
            </td>
          <td>
            Katharina Stemwedel, N. Haase, Simon Christ, Natalia V Bogdanova, Sophia Rudorf
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer, a heterogeneous and intricate disease, ranks among the leading causes of mortality in women. Restricted therapeutic choices, drug resistance, recurrence, and metastasis are the predominant conditions that lead to mortality. Accumulating evidence has shown breast cancer initiation and progression happen through a multifaceted and intricate process that involves numerous genetic and epigenetic alterations. The modulation of gene expression through epigenetic modifications, encompassing DNA methylation, histone alterations, and non-coding RNA regulation, has emerged as a fascinating field that represents a new avenue for breast cancer therapy. This review emphasizes various aberrant epigenetic regulations implicated in the onset and advancement of breast cancer. The critical epigenetic modifications closely associated with estrogen signaling, epithelial-to-mesenchymal transition (EMT), cancer stemness, and drug resistance have been discussed extensively. Moreover, it highlights current epi-drugs, including DNA modifying agents, histone acetyltransferase inhibitors, histone deacetylase inhibitors, histone methyltransferase inhibitors, and histone demethyltransferase inhibitors used for breast cancer treatment. Nonetheless, we described current investigations pertaining to combination therapy employing epi-drugs and future challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f756e86bcaa3afefdda7ee5d7e501dcc5e41b11" target='_blank'>
              Epigenetic Modulations in Breast Cancer: An Emerging Paradigm in Therapeutic Implications.
              </a>
            </td>
          <td>
            Shilpi Sarkar, Dheepika Venkatesh, Thirukumaran Kandasamy, S. S. Ghosh
          </td>
          <td>2024-08-19</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M As Sobeai
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Despite development of several effective therapies for multiple myeloma (MM), the prognosis of patients with partial deletion of chromosome 17 (del(17p)) and TP53 aberrations remains poor. By applying comprehensive multi-omics profiling analyses (whole exome and transcriptome sequencing plus proteomics) and functional ex vivo drug screening to samples from 167 patients with MM, we uncovered novel therapeutic vulnerabilities specific to TP53 mutated MM. Our findings revealed a distinct sensitivity profile to a range of inhibitors (mitotic, topoisomerase, HDAC, HSP90, IGF1R and PI3K/AKT/mTOR inhibitors) irrespective of 17p deletion status. Conversely, no increase in sensitivity was observed for monoallelic TP53 (del(17p) with WT TP53) when compared to other samples, highlighting the remaining unmet clinical need. Notably, plicamycin, an RNA synthesis inhibitor linked to modulation of chromatin structure and increased transcription, emerged as particularly efficacious for TP53 mutated MM. The increased sensitivity correlated with higher protein expression of the drug targets: HDAC2, HSP90AA1 and multiple ribosomal subunits. Additionally, we observed increased RNA expression of G2M checkpoint, E2F targets and mTORC1 signaling in our cohort and the MMRF-CoMMpass (NCT01454297) study in TP53 mutated MM. Harmonization of multi-omics data with ex vivo drug screening results revealed that TP53 mutated MM is functionally distinct from MM with monoallelic TP53, and demonstrate that MM with mutated TP53, with and without del(17p), may be targetable by approved drugs, and further indicates the need for regular monitoring by sequencing to identify these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a52d0d33d267850e10246d4013cb5921df1165" target='_blank'>
              Multi-omic and functional screening reveal targetable vulnerabilities in TP53 mutated multiple myeloma
              </a>
            </td>
          <td>
            Dimitrios Tsallos, Nemo Ikonen, Juho J. Mienen, M. M. Majumder, S. Eldfors, Imre Västrik, A. Parsons, M. Suvela, Katie Dunphy, Paul Dowling, D. Bazou, Peter O’Gorman, J. Lievonen, R. Silvennoinen, Pekka Anla, C. Heckman
          </td>
          <td>2024-08-23</td>
          <td>None</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="INTRODUCTION
Colon cancer, ranked as the fourth leading global cause of cancer death, exhibits a complex progression marked by genetic variations. Over the past decade, the utilization of diverse CRISPR systems has propelled accelerated research into colorectal cancer (CRC) treatment.


AREAS COVERED
CRISPR/Cas9, a key player in this research, identifies new oncogenes, tumor suppressor genes (TSGs), and drug-resistance genes. Additionally, it facilitates the construction of experimental models, conducts genome-wide library screening, and develops new therapeutic targets, especially for targeted knockout in vivo or molecular targeted drug delivery, contributing to personalized treatments and significantly enhancing the care of colon cancer patients. In this review, we provide insights into the mechanism of the CRISPR/Cas9 system, offering a comprehensive exploration of its applications in CRC, spanning screening, modeling, gene functions, diagnosis, and gene therapy. While acknowledging its transformative potential, the article  highlights the challenges and limitations of CRISPR systems.


EXPERT OPINION
The application of CRISPR/Cas9 in CRC research provides a promising avenue for personalized treatments. Its potential for identifying key genes and enabling experimental models and genome-wide screening enhances patient care. This review underscores the significance of CRISPR-Cas9 gene editing technology across basic research, diagnosis, and the treatment landscape of colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e832452c0de49427c039c9545ac3618de43c5948" target='_blank'>
              Recent advances in CRISPR-Cas systems for colorectal cancer research and therapeutics.
              </a>
            </td>
          <td>
            Saeideh Khorshid Sokhangouy, Farzaneh Alizadeh, Malihe Lotfi, Samaneh Sharif, Atefeh Ashouri, Yasamin Yoosefi, Saeed Bozorg Qomi, M. Abbaszadegan
          </td>
          <td>2024-08-02</td>
          <td>Expert review of molecular diagnostics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Using five complementary short- and long-read sequencing technologies, we phased and assembled >95% of each diploid human genome in a four-generation, 28-member family (CEPH 1463) allowing us to systematically assess de novo mutations (DNMs) and recombination. From this family, we estimate an average of 192 DNMs per generation, including 75.5 de novo single-nucleotide variants (SNVs), 7.4 non-tandem repeat indels, 79.6 de novo indels or structural variants (SVs) originating from tandem repeats, 7.7 centromeric de novo SVs and SNVs, and 12.4 de novo Y chromosome events per generation. STRs and VNTRs are the most mutable with 32 loci exhibiting recurrent mutation through the generations. We accurately assemble 288 centromeres and six Y chromosomes across the generations, documenting de novo SVs, and demonstrate that the DNM rate varies by an order of magnitude depending on repeat content, length, and sequence identity. We show a strong paternal bias (75-81%) for all forms of germline DNM, yet we estimate that 17% of de novo SNVs are postzygotic in origin with no paternal bias. We place all this variation in the context of a high-resolution recombination map (∼3.5 kbp breakpoint resolution). We observe a strong maternal recombination bias (1.36 maternal:paternal ratio) with a consistent reduction in the number of crossovers with increasing paternal (r=0.85) and maternal (r=0.65) age. However, we observe no correlation between meiotic crossover locations and de novo SVs, arguing against non-allelic homologous recombination as a predominant mechanism. The use of multiple orthogonal technologies, near-telomere-to-telomere phased genome assemblies, and a multi-generation family to assess transmission has created the most comprehensive, publicly available “truth set” of all classes of genomic variants. The resource can be used to test and benchmark new algorithms and technologies to understand the most fundamental processes underlying human genetic variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcb8df79fd9bfc629f05070b595811f041bfe33" target='_blank'>
              A familial, telomere-to-telomere reference for human de novo mutation and recombination from a four-generation pedigree
              </a>
            </td>
          <td>
            David Porubsky, Harriet Dashnow, T. Sasani, Glennis A. Logsdon, P. Hallast, Michelle D. Noyes, Z. Kronenberg, T. Mokveld, Nidhi Koundinya, Cillian Nolan, C. Steely, A. Guarracino, E. Dolzhenko, William T. Harvey, W. Rowell, K. Grigorev, T. J. Nicholas, Keisuke K. Oshima, Jiadong Lin, Peter Ebert, W. Watkins, Tiffany Y. Leung, Vincent C. T. Hanlon, Sean Mcgee, Brent S. Pedersen, Michael E. Goldberg, Hannah C. Happ, Hyeonsoo Jeong, Katherine M. Munson, Kendra Hoekzema, Daniel D. Chan, Yanni Wang, Jordan Knuth, Gage H. Garcia, Cairbre Fanslow, Christine Lambert, Charles Lee, Josh Smith, Shawn Levy, C. Mason, Erik Garrison, Peter Lansdorp, D. Neklason, L. Jorde, Aaron R. Quinlan, M. Eberle, Evan E. Eichler
          </td>
          <td>2024-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Thalassemia is the most common autosomal genetic disorder in humans. The pathogenesis of thalassemia is principally due to the deletion or mutation of globin genes that then leads to disorders in globin-chain synthesis, and its predominant clinical manifestations include chronic forms of hemolytic anemia. However, research on the epigenetics and underlying pathogenesis of thalassemia is in its nascency and not yet been systematically realized. In this study, we compared the results of RNA-seq and the whole-genome bisulfite sequencing (WGBS) on 22 peripheral blood samples from 14 thalassemic patients and eight healthy individuals revealed a genome-wide methylation landscape of differentially methylated regions (DMRs). And functional-enrichment analysis revealed the enriched biological pathways with respect to the differentially expressed genes (DEGs) and differentially methylated genes (DMGs) to include hematopoietic lineage, glucose metabolism, and ribosome. To further analyze the interaction between the transcriptome and methylome, we implemented a comprehensive analysis of overlaps between DEGs and DMGs, and observed that biological processes significantly enriched the immune-related genes (i.e., our hypermethylated and down-regulated gene group). Hypermethylated and hypomethylated regions of thalassemia-related genes exhibited different distribution patterns. We thus, further identified and validated thalassemia-associated DMGs and DEGs by multi-omics integrative analyses of DNA methylation and transcriptomics data, and provided a comprehensive genomic map of thalassemia that will facilitate the exploration of the epigenetics mechanisms and pathogenesis underlying thalassemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea246ebd01aaa606cd77160e92c7c083bd7557bd" target='_blank'>
              Genome-wide methylation and gene-expression analyses in thalassemia
              </a>
            </td>
          <td>
            Wei Zhang, Xiaokang Li, Uet Yu, Xin Huang, Hongmei Wang, Yilin Lu, Sixi Liu, Jian Zhang
          </td>
          <td>2024-08-09</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f891979797e3688f35366887366668ec188a50ae" target='_blank'>
              Microsatellite instability at U2AF-binding polypyrimidic tract sites perturbs alternative splicing during colorectal cancer initiation
              </a>
            </td>
          <td>
            V. Jonchère, Hugo Montémont, Enora Le Scanf, A. Siret, Q. Letourneur, Emmanuel Tubacher, Christophe Battail, Assane Fall, Karim Labreche, Victor Renault, T. Ratovomanana, O. Buhard, A. Jolly, Philippe Le Rouzic, Cody Feys, Emmanuelle Despras, Habib Zouali, R. Nicolle, Pascale Cervera, M. Svrcek, P. Bourgoin, Hélène Blanché, A. Boland, Jérémie Lefèvre, Y. Parc, M. Touat, F. Bielle, D. Arzur, Gwennina Cueff, C. Le Jossic-Corcos, Gaël Quéré, Gwendal Dujardin, Marc Blondel, Cédric Le Maréchal, R. Cohen, Thierry André, Florence Coulet, P. de la Grange, A. de Reyniès, Jean-françois Flejou, Florence Renaud, A. Alentorn, Laurent Corcos, J. Deleuze, A. Collura, A. Duval
          </td>
          <td>2024-08-06</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d317e0704a11a531980e863a514ed7e3ab276870" target='_blank'>
              Advances in pathogenesis research and challenges in treatment development for acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hiroki Yamaguchi
          </td>
          <td>2024-09-03</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/699986230662d4afba97aea6b14e6128a02adbf7" target='_blank'>
              Molecular mechanisms of DNA lesion and repair during antibody somatic hypermutation.
              </a>
            </td>
          <td>
            Qian Hao, Jinfeng Li, Leng-Siew Yeap
          </td>
          <td>2024-07-23</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Microsatellite unstable colorectal cancer (MSI-CRC) can arise through germline mutations in mismatch repair (MMR) genes in individuals with Lynch syndrome (LS), or sporadically through promoter methylation of the MMR gene MLH1. Despite the different origins of hereditary and sporadic MSI tumours, their genomic features have not been extensively compared. A prominent feature of MMR-deficient genomes is the occurrence of many indels in short repeat sequences, an understudied mutation type due to the technical challenges of variant calling in these regions. In this study, we performed whole genome sequencing and RNA-sequencing on 29 sporadic and 14 hereditary MSI-CRCs. We compared the tumour groups by analysing genome-wide mutation densities, microsatellite repeat indels, recurrent protein-coding variants, signatures of single base, doublet base, and indel mutations, and changes in gene expression. We show that the mutational landscapes of hereditary and sporadic MSI-CRCs, including mutational signatures and mutation densities genome-wide and in microsatellites, are highly similar. Only a low number of differentially expressed genes were found, enriched to interferon-γ regulated immune response pathways. Analysis of the variance in allelic fractions of somatic variants in each tumour group revealed higher clonal heterogeneity in sporadic MSI-CRCs. Our results suggest that the differing molecular origins of MMR deficiency in hereditary and sporadic MSI-CRCs do not result in substantial differences in the mutational landscapes of these tumours. The divergent patterns of clonal evolution between the tumour groups may have clinical implications, as high clonal heterogeneity has been associated with decreased tumour immunosurveillance and reduced responsiveness to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43c6128f5465fd71306e14170da497710701fc42" target='_blank'>
              Lynch syndrome-associated and sporadic microsatellite unstable colorectal cancers: different patterns of clonal evolution yield highly similar tumours.
              </a>
            </td>
          <td>
            Samantha Martin, Riku Katainen, Aurora Taira, Niko Välimäki, Ari Ristimäki, T. Seppälä, L. Renkonen-Sinisalo, A. Lepistö, Kyösti Tahkola, Anne Mattila, S. Koskensalo, J. Mecklin, K. Rajamäki, Kimmo Palin, L. Aaltonen
          </td>
          <td>2024-08-24</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU- 023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention. Author Summary Bile duct cancer has the highest incidence in northeastern Thailand. In this study, we explored the mtDNA of CCA cell lines, as mitochondria play a key role in cellular energy production. We aimed to identify mutations that could serve as therapeutic targets or biomarkers. Our analysis revealed that each CCA cell line has unique mtDNA profiles, including INDELs, SVs, and SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. yet they also shared some common mutations. Future research is needed to understand how these mutations affect cell function, interact with nuclear DNA, and their potential for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eafb79f26f32f1aeaf6d648b336c327747f051f" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapan, A. Jareanrat, V. Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Simple Summary This review provides a comprehensive overview of the current knowledge on mutational signatures in colorectal cancer (CRC), highlighting their potential clinical applications. It discusses the challenges and limitations in translating these analyses into clinical practice and proposes strategies to overcome these obstacles. Additionally, it provides insights into future directions and emerging proof-of-concept studies that highlight the translational potential of mutational signature analysis in improving patient care and outcomes in CRC. Abstract A multitude of exogenous and endogenous processes have the potential to result in DNA damage. While the repair mechanisms are typically capable of correcting this damage, errors in the repair process can result in mutations. The findings of research conducted in 2012 indicate that mutations do not occur randomly but rather follow specific patterns that can be attributed to known or inferred mutational processes. The process of mutational signature analysis allows for the inference of the predominant mutational process for a given cancer sample, with significant potential for clinical applications. A deeper comprehension of these mutational signatures in CRC could facilitate enhanced prevention strategies, facilitate the comprehension of genotoxic drug activity, predict responses to personalized treatments, and, in the future, inform the development of targeted therapies in the context of precision oncology. The efforts of numerous researchers have led to the identification of several mutational signatures, which can be categorized into different mutational signature references. In CRC, distinct mutational signatures are identified as correlating with mismatch repair deficiency, polymerase mutations, and chemotherapy treatment. In this context, a mutational signature analysis offers considerable potential for enhancing minimal residual disease (MRD) tests in stage II (high-risk) and stage III CRC post-surgery, stratifying CRC based on the impacts of genetic and epigenetic alterations for precision oncology, identifying potential therapeutic vulnerabilities, and evaluating drug efficacy and guiding therapy, as illustrated in a proof-of-concept clinical trial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9f4817ac8c40910eefa1198ffb3e184664ea3d" target='_blank'>
              Mutational Signatures in Colorectal Cancer: Translational Insights, Clinical Applications, and Limitations
              </a>
            </td>
          <td>
            G. Crisafulli
          </td>
          <td>2024-08-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Objective The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions. Methods We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters. Results Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations. Conclusion Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, Weisen Fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Genomic rearrangements are chromosomal abnormalities that alter the arrangement of genes and are important for diagnosing multiple cancer types and guiding therapy selection. However, current diagnostic tests may not detect all genomic rearrangements. Hi-C sequencing is a promising new method for detecting genomic rearrangements. First, we examined whether Hi-C can detect known genomic rearrangements in pediatric leukemia and sarcoma specimens. Next, we evaluated whether Hi-C can detect genomic rearrangements that were not found using standard diagnostic testing. Hi-C showed complete agreement with other diagnostic methods in identifying known rearrangements. Hi-C also detected previously unknown genomic rearrangements in 5/11 pediatric leukemia cases that could impact diagnosis, prognosis, or treatment choices. These data suggest Hi-C could be beneficial for medical diagnostic testing of pediatric cancers, but more extensive clinical validation is needed. Abstract Hi-C sequencing is a DNA-based next-generation sequencing method that preserves the 3D genome conformation and has shown promise in detecting genomic rearrangements in translational research studies. To evaluate Hi-C as a potential clinical diagnostic platform, analytical concordance with routine laboratory testing was assessed using primary pediatric leukemia and sarcoma specimens. Archived viable and non-viable frozen leukemic cells and formalin-fixed paraffin-embedded (FFPE) tumor specimens were analyzed. Pediatric acute myeloid leukemia (AML) and alveolar rhabdomyosarcoma (A-RMS) specimens with known genomic rearrangements were subjected to Hi-C to assess analytical concordance. Subsequently, a discovery cohort consisting of AML and acute lymphoblastic leukemia (ALL) cases without known genomic rearrangements based on prior clinical diagnostic testing was evaluated to determine whether Hi-C could detect rearrangements. Using a standard sequencing depth of 50 million raw read-pairs per sample, or approximately 5X raw genomic coverage, we observed 100% concordance between Hi-C and previous clinical cytogenetic and molecular testing. In the discovery cohort, a clinically relevant gene fusion was detected in 45% of leukemia cases (5/11). This study provides an institutional proof of principle evaluation of Hi-C sequencing to medical diagnostic testing as it identified several clinically relevant rearrangements, including those that were missed by current clinical testing workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af941e63f57e2e2bb71ed7543d231768d840bed5" target='_blank'>
              Evaluation of Hi-C Sequencing for Detection of Gene Fusions in Hematologic and Solid Tumor Pediatric Cancer Samples
              </a>
            </td>
          <td>
            Anthony D. Schmitt, Kristin Sikkink, Atif A. Ahmed, Shadi Melnyk, Derek Reid, Logan Van Meter, Erin M. Guest, Lisa A. Lansdon, Tomi Pastinen, Irina Pushel, B. Yoo, M. Farooqi
          </td>
          <td>2024-08-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome flexibility is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and flexibility determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin a more prone to form SVs than those located in euchromatin, highlighting that genome stability and flexibility differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA Damage Response (DDR), ATM and ATR, suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure UV-B, UV-C and protons induced the Microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a78aa5b539dcd98a3297c9111dc41b5d43d56b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a diverse malignancy originating from myeloid progenitor cells, with significant genetic and clinical variability. Modern classification systems like those from the World Health Organization (WHO) and European LeukemiaNet use immunophenotyping, molecular genetics, and clinical features to categorize AML subtypes. This classification highlights crucial genetic markers such as FLT3, NPM1 mutations, and MLL-AF9 fusion, which are essential for prognosis and directing targeted therapies. The MLL-AF9 fusion protein is often linked with therapy-resistant AML, highlighting the risk of relapse due to standard chemotherapeutic regimes. In this sense, factors like the ZEB, SNAI, and TWIST gene families, known for their roles in epithelial–mesenchymal transition (EMT) and cancer metastasis, also regulate hematopoiesis and may serve as effective therapeutic targets in AML. These genes contribute to cell proliferation, differentiation, and extramedullary hematopoiesis, suggesting new possibilities for treatment. Advancing our understanding of the molecular mechanisms that promote AML, especially how the bone marrow microenvironment affects invasion and drug resistance, is crucial. This comprehensive insight into the molecular and environmental interactions in AML emphasizes the need for ongoing research and more effective treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b827ac4bebbcac825ffc1c60412a291161ab8775" target='_blank'>
              The Role of Epithelial-to-Mesenchymal Transition Transcription Factors (EMT-TFs) in Acute Myeloid Leukemia Progression
              </a>
            </td>
          <td>
            Diego Cuevas, Roberto Amigo, Adolfo Agurto, Adan Andreu Heredia, , , , Teresa Caprile, Jody J Haigh, Carlos Farkas
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Cancer is a multifaceted disease influenced by both intrinsic cellular traits and extrinsic factors, with the tumor microenvironment (TME) being crucial in its progression. To satisfy their high proliferation and aggressiveness, cancer cell always plunders large amounts of nutrition and releases various signals to the surrounding, forming a dynamic TME with special metabolic, immune, microbial and physical characteristics. Due to the neglect of interactions between tumor cell and TME, traditional cancer therapies often struggle with challenges such as drug resistance, low efficacy, and recurrence. Importantly, with the development of gene editing technologies, particularly the CRISPR-Cas system, offers promising new strategies for cancer treatment. Combined with nanomaterials strategies, CRISPR-Cas technology exhibits precision, affordability, and user-friendliness with reduced side effects, which holds great promise for profoundly altering the TME at a genetic level, potentially leading to lasting anticancer outcomes. This review will delve into how CRISPR-Cas can be leveraged to manipulate the TME, examining its potential as a transformative anticancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b4994cb649ea8f94220d3676ac8b2eeb79b1144" target='_blank'>
              Harnessing Gene Editing Technology for Tumor Microenvironment Modulation: An Emerging Anticancer Strategy.
              </a>
            </td>
          <td>
            Shujun Feng, Yu Zhang, Yanyi Wang, Yanfeng Gao, Yujun Song
          </td>
          <td>2024-09-03</td>
          <td>Chemistry</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) remains one of the most lethal cancers globally. Patients with advanced HCC tend to have poor prognoses and shortened survival. Recently, data from bulk RNA sequencing have been employed to discover prognostic markers for various cancers. However, they fall short in precisely identifying core molecular and cellular activities within tumor cells. In our present study, we combined bulk-RNA sequencing (bulk RNA-seq) data with single-cell RNA sequencing (scRNA-seq) to develop a prognostic model for HCC. The goal of our research is to uncover new biomarkers and enhance the accuracy of HCC prognosis prediction. Methods Integrating single-cell sequencing data with transcriptomics were used to identify epithelial-mesenchymal transition (EMT)-related genes (ERGs) implicated in HCC progression and their clinical significance was elucidated. Utilizing marker genes derived from core cells and ERGs, we constructed a prognostic model using univariate Cox analysis, exploring a multitude of algorithmic combinations, and further refining it through multivariate Cox analysis. Additionally, we conducted an in-depth investigation into the disparities in clinicopathological features, immune microenvironment composition, immune checkpoint expression, and chemotherapeutic drug sensitivity profiles between high- and low-risk patient cohorts. Results We developed a prognostic model predicated on the expression profiles of eight signature genes, namely HSP90AA1, CIRBP, CCR7, S100A9, ADAM17, ENG, PGF, and INPP4B, aiming at predicting overall survival (OS) outcomes. Notably, patients classified with high-risk scores exhibited a propensity towards diminished OS rates, heightened frequencies of stage III–IV disease, increased tumor mutational burden (TMB), augmented immune cell infiltration, and diminished responsiveness to immunotherapeutic interventions. Conclusions This study presented a novel prognostic model for predicting the survival of HCC patients by integrating scRNA-seq and bulk RNA-seq data. The risk score emerges as a promising independent prognostic factor, showing a correlation with the immune microenvironment and clinicopathological features. It provided new clinical tools for predicting prognosis and aided future research into the pathogenesis of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b96c5e3686dc07fb1b7548e3f485ea65717660f" target='_blank'>
              Identifying epithelial-mesenchymal transition-related genes as prognostic biomarkers and therapeutic targets of hepatocellular carcinoma by integrated analysis of single-cell and bulk-RNA sequencing data
              </a>
            </td>
          <td>
            Chen Chen, Shunyi Wang, Yuhong Tang, Huanxiang Liu, Dao-yuan Tu, Bing-bing Su, Rui Peng, Shengjie Jin, Guoqing Jiang, Jun Cao, Chi Zhang, Dousheng Bai
          </td>
          <td>2024-08-01</td>
          <td>Translational Cancer Research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Kirsten Rat Sarcoma (KRAS) is the most commonly mutated oncogene in colorectal carcinoma (CRC). We have previously reported the interactions between microsatellite instability (MSI), DNA promoter methylation, and gene expression. In this study, we looked for associations between KRAS mutation, gene expression, and methylation that may help with precision medicine. Genome-wide gene expression and DNA methylation were done in paired CRC tumor and surrounding healthy tissues. The results suggested that (a) the magnitude of dysregulation of many major gene pathways in CRC was significantly greater in patients with the KRAS mutation, (b) the up- and down-regulation of these dysregulated gene pathways could be correlated with the corresponding hypo- and hyper-methylation, and (c) the up-regulation of CDKN2A was more pronounced in tumors with the KRAS mutation. A recent cell line study showed that there were higher CDKN2A levels in 5-FU-resistant CRC cells and that these could be down-regulated by Villosol. Our findings suggest the possibility of a better response to anti-CDKN2A therapy with Villosol in KRAS-mutant CRC. Also, the more marked up-regulation of genes in the proteasome pathway in CRC tissue, especially with the KRAS mutation and MSI, may suggest a potential role of a proteasome inhibitor (bortezomib, carfilzomib, or ixazomib) in selected CRC patients if necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40b3e427194ea41aedea0dfcb0cf1b6c9f955b47" target='_blank'>
              Association of KRAS Mutation and Gene Pathways in Colorectal Carcinoma: A Transcriptome- and Methylome-Wide Study and Potential Implications for Therapy
              </a>
            </td>
          <td>
            F. Jasmine, Armando Almazan, Yuliia Khamkevych, Marc Bissonnette, Habibul Ahsan, M. Kibriya
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8e662896b1a823a5ffd8b3a87cc65bf3c320680" target='_blank'>
              A panel sequencing dataset of peripheral blood gene variations in pan-cancer
              </a>
            </td>
          <td>
            Yanxia Liu, Jie Liu, Shouwei Zhang, Jinyue Wang, Zhihong Sun, Huaibo Sun, Ying Yang, Guangmin Zheng, Yu S Huang, Meng Li, Zhaojun Zhang, Jingfa Xiao, Changqing Zeng, Chengming Sun, H. Qu, Xiangdong Fang
          </td>
          <td>2024-07-20</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Chronic Lymphocytic Leukemia (CLL) is the most common B-cell malignancy in the Western world, characterized by frequent relapses despite temporary remissions. Our study integrated publicly available proteomic, transcriptomic, and patient survival datasets to identify key differences between healthy and CLL samples. We exposed approximately 1000 proteins that differentiate healthy from cancerous cells, with 608 upregulated and 415 downregulated in CLL cases. Notable upregulated proteins include YEATS2 (an epigenetic regulator), PIGR (Polymeric immunoglobulin receptor), and SNRPA (a splicing factor), which may serve as prognostic biomarkers for this disease. Key pathways implicated in CLL progression involve RNA processing, stress resistance, and immune response deficits. Furthermore, we identified three existing drugs—Bosutinib, Vorinostat, and Panobinostat—for potential further investigation in drug repurposing in CLL. We also found limited correlation between transcriptomic and proteomic data, emphasizing the importance of proteomics in understanding gene expression regulation mechanisms. This generally known disparity highlights once again that mRNA levels do not accurately predict protein abundance due to many regulatory factors, such as protein degradation, post-transcriptional modifications, and differing rates of translation. These results demonstrate the value of integrating omics data to uncover deregulated proteins and pathways in cancer and suggest new therapeutic avenues for CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524d87cfc3fcaee95b1a168a590bc69c95feb91a" target='_blank'>
              Integrative Analysis of Multi-Omics Data to Identify Deregulated Molecular Pathways and Druggable Targets in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            D. Mavridou, K. Psatha, M. Aivaliotis
          </td>
          <td>2024-08-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554fe13daafca3f151510eb83bbcfb8701b7284c" target='_blank'>
              Genome-scale CRISPR-Cas9 screening in stem cells: theories, applications and challenges
              </a>
            </td>
          <td>
            Heng Zhou, Peng Ye, W. Xiong, Xingxiang Duan, Shuili Jing, Yan He, Zhi Zeng, Yen Wei, Qingsong Ye
          </td>
          <td>2024-07-19</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A major goal in biology is to uncover the relationship between genotype and phenotype. However, identifying gene function is often hampered by genetic redundancy. For example, under standard laboratory conditions, three-quarters of the genes in the human pathogen Streptococcus pneumoniae are non-essential. A powerful approach to unravel genetic redundancy is by identifying gene-gene interactions. To uncover genetic interactions (GIs) in S. pneumoniae on a genome-wide scale, a generally applicable dual CRISPRi-Seq method and associated analysis pipeline was developed. Specifically, we created a library of 869 dual sgRNAs targeting high-confidence operons that encode essential and non-essential genes, covering over 70% of the genetic elements in the pneumococcal genome. Testing these 378,015 unique combinations, 4,026 significant GIs were identified, including 1,935 negative and 2,091 positive interactions. Besides known GIs, we found and confirmed previously unknown interactions involving genes responsible for fundamental cellular processes such as cell division, cell shape maintenance, and chromosome segregation. The presented methods and bioinformatic approaches can serve as a roadmap for genome-wide gene interaction studies in other organisms. Lastly, all interactions are available for exploration via the Pneumococcal Genetic Interaction Network (PneumoGIN) at https://veeninglab.shinyapps.io/PneumoGIN, which can serve as a starting point for new biological discoveries and translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dd78fdd99cdc75fb8cdbb9e56ea5f0c3b73930" target='_blank'>
              Dual CRISPRi-Seq for genome-wide genetic interaction studies identifies key genes involved in the pneumococcal cell cycle
              </a>
            </td>
          <td>
            Julien Dénéréaz, Elise Eray, Bimal Jana, Vincent de Bakker, Horia Todor, Tim van Opijnen, Xue Liu, J. Veening
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/553f4560eb657fca2fe86385e460c71427df430c" target='_blank'>
              GLI1 confers resistance to PARP inhibitors by activating the DNA damage repair pathway.
              </a>
            </td>
          <td>
            Hiroshi Ikeuchi, Yusuke Matsuno, Rika Kusumoto-Matsuo, S. Kojima, Toshihide Ueno, Masachika Ikegami, Rina Kitada, Hitomi Sumiyoshi-Okuma, Yuki Kojima, K. Yonemori, Y. Yatabe, K. Takamochi, Kenji Suzuki, Ken‐ichi Yoshioka, H. Mano, S. Kohsaka
          </td>
          <td>2024-08-03</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="This report presents the largest collection of gamma-delta T cell receptor (γδ TCR) reads in human cancer to date, analyzing about 11,000 patient tumor samples across 33 cancer types using the TRUST4 algorithm. Despite γδ T cells being a small fraction of the T cell population, they play a key role in both innate and adaptive immunity. Our comprehensive analysis reveals their significant presence across all cancer types, specifically highlighting the diverse spectrum and clonality patterns of their γδ receptors. This research highlights the complex roles of γδ T cells in tumor tissues and their potential as prognostic biomarkers. We also demonstrate the utility of T cell receptor gamma (TRG) and delta (TRD) gene expression values from standard RNA-seq data. Ultimately, our work establishes a fundamental resource for future tumor-infiltrating γδ T cell research and may facilitate the development of novel γδ-T-cell-based therapeutic strategies. Together, we demonstrate the strong diversity and prognostic potential of γδ T cells in multiple cancer types. Highlights Comprehensive analysis of γδ TCRs from 11,473 tumor samples Significant variability and overall consistency in γδ gene expression and clonotype γδ TCR expression and diversity as prognostic biomarkers across multiple cancers Centralized γδ TCR repertoire database for future therapeutic discovery">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f81b0341b0c9b28947c09231986dc63cf346223" target='_blank'>
              A pan-cancer gamma delta T cell repertoire
              </a>
            </td>
          <td>
            Xiaoqing Yu, Li Song, Ling Cen, Biwei Cao, Ranran Tao, Yuanyuan Shen, Daniel Abate- Daga, Paulo C. Rodriguez, Jose R Conejo-Garcia, Xuefeng Wang
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/244d976555710935c004ec15262f6387c6d24c28" target='_blank'>
              Identifying prognostic biomarkers in oral squamous cell carcinoma: an integrated single-cell and bulk RNA sequencing study on mitophagy-related genes
              </a>
            </td>
          <td>
            Minsi Li, Yi Wei, Wenhua Huang, Cen Wang, Shixi He, Shuwen Bi, Shuangyu Hu, Ling You, Xuan-ping Huang
          </td>
          <td>2024-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2fd505ae2ea1de627590bede77cfbae2e3ab46" target='_blank'>
              CRISPR screen of venetoclax response-associated genes identifies transcription factor ZNF740 as a key functional regulator
              </a>
            </td>
          <td>
            Lixia Zhang, Xinyue Zhou, Sajesan Aryal, Virginia Veasey, Pengcheng Zhang, Fu Jun Li, Yu Luan, Ravi Bhatia, Yang Zhou, Rui Lu
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5ea42f9ac2f6d502e99cce5e348de337e22eda" target='_blank'>
              Effects of HOX family regulator-mediated modification patterns and immunity characteristics on tumor-associated cell type in endometrial cancer
              </a>
            </td>
          <td>
            Jiaolin Yang, Jinpeng Li, Sufen Li, Yutong Yang, Huancheng Su, Hongrui Guo, Jing Lei, YaLin Wang, KaiTing Wen, Xia Li, Sanyuan Zhang, Zhe Wang
          </td>
          <td>2024-08-14</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b62bbc42b475a59548fce65b7db966d5dba12012" target='_blank'>
              CRISPRi–TnSeq maps genome-wide interactions between essential and non-essential genes in bacteria
              </a>
            </td>
          <td>
            Bimal Jana, Xue Liu, Julien Dénéréaz, Hongshik Park, Dmitry Leshchiner, Bruce Liu, Clément Gallay, Junhao Zhu, J. Veening, Tim van Opijnen
          </td>
          <td>2024-07-19</td>
          <td>Nature Microbiology</td>
          <td>2</td>
          <td>46</td>
        </tr>

        <tr id="It has been observed that men have a shorter lifespan and higher incidence of certain cancers than women. We postulate here that this phenomenon may be attributed to the “toxic Y chromosome”, owing to the expression of normally silent HSATII repeats in cancer, abundantly present on chromosome Y. Investigating the contribution of all repetitive elements to chromatin interactions using a genome-wide HiC set derived from a colon cancer cell line (HCT116), we found several anomalies regarding satellite HSATII elements when compared to other genomic repeat types: (i) HSATII repeats tend to form HiC pairs mostly with their own kind, and depleted in most heterologous repeat pairs, whereas other types of repeats readily form heterologous HiC pairs; (ii) when treated with 5aza, a chemotherapeutic agent, the number of long-distance HSATII-overlapping HiC pairs significantly decreased when compared to the untreated samples; (iii) and most importantly, in the treated samples interchromosomal HiC read pairs involving a chromosome Y-located repeat and an HSATII repeat on either end almost completely disappear. (iv) We also found that HSATII repeats lack overlap with other types of repeats. To account for these anomalies we queried non-B DNA structure forming such as G-quadruplexes and transcription-related triplex forming. We found that Y chromosome-coded HSATII repeats form significantly stronger triplexes than HSATII repeats on other chromosomes, their strength significantly correlating with the number of HiC reads they overlap with. We surmise that the above phenomena may cause the higher incidence of cancer in men and may lead to the disappearance of the Y chromosome observed in cancerous cells in men.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53990a3a3a681b314979b859a2d2941c439f782" target='_blank'>
              Y chromosome-coded HSATII repeats may contribute to higher incidence of cancer in men
              </a>
            </td>
          <td>
            Hedi Hegyi, Matej Lexa
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac67a3ed7d66d72ed2d1b0afc516f030fad809a" target='_blank'>
              Immunological hide-and-seek: epigenetically reprogrammed cancer cells and the dynamics of CD8+ T cells.
              </a>
            </td>
          <td>
            Jie-Ting Low, M. W. Chan, Cheng-Huang Shen, Kuo-Liang Wei
          </td>
          <td>2024-09-04</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bbe642fe1a5320aff996d83dd54d13a05e4037" target='_blank'>
              The intersection of homologous recombination (HR) and mismatch repair (MMR) pathways in DNA repair-defective tumors
              </a>
            </td>
          <td>
            L. Incorvaia, T. B. Bazan Russo, V. Gristina, A. Perez, C. Brando, Clarissa Mujacic, Emilia Di Giovanni, M. Bono, Silvia Contino, Carla Ferrante Bannera, Maria Concetta Vitale, Andrea Gottardo, M. Peri, A. Galvano, D. Fanale, G. Badalamenti, Antonio Russo, V. Bazan
          </td>
          <td>2024-09-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Mutations commonly found in AML such as DNMT3A, TET2 and ASXL1 can be found in the peripheral blood of otherwise healthy adults a phenomenon referred to as clonal hematopoiesis (CH). These mutations are thought to represent the earliest genetic events in the evolution of AML. Genomic studies on samples acquired at diagnosis, remission, and at relapse have demonstrated significant stability of CH mutations following induction chemotherapy. Meanwhile, later mutations in genes such as NPM1 and FLT3, have been shown to contract at remission and in the case of FLT3 often are absent at relapse. We sought to understand how early CH mutations influence subsequent evolutionary trajectories throughout remission and relapse in response to induction chemotherapy. Here, we assembled a retrospective cohort of patients diagnosed with de novo AML at our institution that underwent genomic sequencing at diagnosis as well as at the time of remission and/or relapse (total n = 182 patients). Corroborating prior studies, FLT3 and NPM1 mutations were generally eliminated at the time of cytologic complete remission but subsequently reemerged upon relapse, whereas DNMT3A, TET2 and ASXL1 mutations often persisted through remission. Early CH-related mutations exhibited distinct constellations of co-occurring genetic alterations, with NPM1 and FLT3 mutations enriched in DNMT3Amut AML, while CBL and SRSF2 mutations were enriched in TET2mut and ASXL1mut AML, respectively. In the case of NPM1 and FLT3 mutations, these differences vanished at the time of complete remission yet readily reemerged upon relapse, indicating the reproducible nature of these genetic interactions. Thus, early CH-associated mutations that precede malignant transformation subsequently shape the evolutionary trajectories of AML through diagnosis, therapy, and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f35657f001405231a2b4d996ed0ac3f0415703" target='_blank'>
              Early drivers of clonal hematopoiesis shape the evolutionary trajectories of de novo acute myeloid leukemia
              </a>
            </td>
          <td>
            Ryan D. Chow, Priya Velu, Safoora Deihimi, Jonathan Belman, Angela Youn, Nisargbhai Shah, Selina M. Luger, Martin P. Carroll, Jennifer J D Morrissette, Robert L. Bowman
          </td>
          <td>2024-09-01</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background and Objective Lung cancer remains a leading cause of cancer-related mortality globally, with drug resistance posing a significant challenge to effective treatment. The advent of clustered regularly interspaced short palindromic repeats (CRISPR) and CRISPR-associated protein 9 (CRISPR-Cas9) technology offers a novel and precise gene-editing technology for targeting and negating drug resistance mechanisms in lung cancer. This review summarizes the research progress in the use of CRISPR-Cas9 technology for investigating and managing drug resistance in lung cancer treatment. Methods A literature search was conducted using the Web of Science and PubMed databases, with the following keywords: [CRISPR-Cas9], [lung cancer], [drug resistance], [gene editing], and [gene therapy]. The search was limited to articles published in English from 2002 to September 2023. From the search results, studies that utilized CRISPR-Cas9 technology in the context of lung cancer drug resistance were selected for further analysis and summarize. Key Content and Findings CRISPR-Cas9 technology enables precise DNA-sequence editing, allowing for the targeted addition, deletion, or modification of genes. It has been applied to investigate drug resistance in lung cancer by focusing on key genes such as epidermal growth factor receptor (EGFR), Kirsten rat sarcoma viral oncogene homolog (KRAS), tumor protein 53 (TP53), and B-cell lymphoma/leukemia-2 (BCL2), among others. The technology has shown potential in inhibiting tumor growth, repairing mutations, and enhancing the sensitivity of cancer cells to chemotherapy. Additionally, CRISPR-Cas9 has been used to identify novel key genes and molecular mechanisms contributing to drug resistance, offering new avenues for therapeutic intervention. The review also highlights the use of CRISPR-Cas9 in targeting immune escape mechanisms and the development of strategies to improve drug sensitivity. Conclusions The CRISPR-Cas9 technology holds great promise for advancing lung cancer treatment, particularly in addressing drug resistance. The ability to precisely target and edit genes involved in resistance pathways offers a powerful tool for developing more effective and personalized therapies. While challenges remain in terms of delivery, safety, and ethical considerations, ongoing research and technological refinements are expected to further enhance the role of CRISPR-Cas9 in improving patient outcomes in lung cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b68d65ad3458bc7689288a272d9e02c7b75d6a77" target='_blank'>
              Research into overcoming drug resistance in lung cancer treatment using CRISPR-Cas9 technology: a narrative review
              </a>
            </td>
          <td>
            Bin Liu, Ziyu Wang, Meng Gu, Jinghui Wang, Jinjing Tan
          </td>
          <td>2024-08-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) has a poor survival rate for both pediatric and adult patients due to its frequent relapse. To elucidate the bioenergetic principle underlying AML relapse, we investigated the transcriptional regulation of mitochondrial–nuclear dual genomes responsible for metabolic plasticity in treatment-resistant blasts. Both the gain and loss of function results demonstrated that NFκB2, a noncanonical transcription factor (TF) of the NFκB (nuclear factor kappa-light-chain-enhancer of activated B cells) family, can control the expression of TFAM (mitochondrial transcription factor A), which is known to be essential for metabolic biogenesis. Furthermore, genetic tracking and promoter assays revealed that NFκB2 is in the mitochondria and can bind the specific “TTGGGGGGTG” region of the regulatory D-loop domain to activate the light-strand promoter (LSP) and heavy-strand promoter 1 (HSP1), promoters of the mitochondrial genome. Based on our discovery of NFκB2′s novel function of regulating mitochondrial–nuclear dual genomes, we explored a novel triplet therapy including inhibitors of NFκB2, tyrosine kinase, and mitochondrial ATP synthase that effectively eliminated primary AML blasts with mutations of the FMS-related receptor tyrosine kinase 3 (FLT3) and displayed minimum toxicity to control cells ex vivo. As such, effective treatments for AML must include strong inhibitory actions on the dual genomes mediating metabolic plasticity to improve leukemia prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd91cb6b195c34903e27bb3147d1ba2db984023c" target='_blank'>
              Discovery of NFκB2-Coordinated Dual Regulation of Mitochondrial and Nuclear Genomes Leads to an Effective Therapy for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Yi Xu, David J. Baylink, Jeffrey Xiao, Lily Tran, Vinh Nguyen, Brandon Park, Ismael Valladares, Scott Lee, Kevin Codorniz, Laren D Tan, Chien-Shing Chen, Hisham Abdel-Azim, Mark E. Reeves, Hamid Mirshahidi, Guido Marcucci, H. Cao
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ec02f56417ae90246c83ab25ae98a1c2c7e30" target='_blank'>
              Targeting treatment resistance: unveiling the potential of RNA methylation regulators and TG-101,209 in pan-cancer neoadjuvant therapy
              </a>
            </td>
          <td>
            Yaoyao Zhou, Ziyun Liu, Cheng Gong, Jie Zhang, Jing Zhao, Xia Zhang, Xiangyu Liu, Bin Li, Rui Li, Zhenyu Shi, Yongjie Xie, Li Bao
          </td>
          <td>2024-08-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Ewing sarcoma is a highly aggressive pediatric cancer driven by the EWS::FLI1 (EF) fusion oncogene. Evidence suggests that EF expression levels dictate tumor cell plasticity contributing to treatment resistance and relapse. Our study aims to elucidate the transcriptional programs and phenotypes associated with distinct EF thresholds. To dynamically fine-tune EF levels, we engineered Ewing sarcoma cell line models by C-terminally tagging endogenous EF with a fluorescent (mNG) and a degron tag (dTAG). Using mNG as a quantitative proxy for EF levels, we confirmed that increasing concentrations of the dTAG ligand led to a gradual loss of the fusion protein. Already subtle modulation of EF thresholds significantly enhanced tumor cell migration. Furthermore, a 25% EF reduction was sufficient to significantly lower anchorage-independent growth in soft agar, which was completely abolished with further EF depletion. Supported by zebrafish xenograft experiments, these findings suggest that minor EF fluctuations enhance migratory properties while still retaining significant in vitro tumor-forming potential. To characterize acute transcriptional responses to variations in EF thresholds, we performed SLAM-sequencing after 3 hours of treatment with increasing dTAG concentrations. While minor reduction of the fusion protein to 76% led to immediate downregulation of EF-activated genes enriched in GGAA microsatellites, most EF-repressed targets were upregulated only when EF levels were reduced by 58% to 96%. These de-repressed targets were significantly enriched for epithelial-to-mesenchymal transition (EMT)-related pathways. Testing EF chromatin occupancy by CUT&RUN followed by motif enrichment analysis, identified several transcription factor binding motifs linked to EF-repressed genes. Promising candidates include SP1, EGR1, and TFAP2A, all of which have previously been implicated in the regulation of metastasis-related pathways. Long-term dTAG treatment (1 to 3 weeks), followed by EF rescue through dTAG ligand washout for 7 days revealed a set of metastasis-related genes that notably remained upregulated despite a full recovery of EF levels already within 24 hours of ligand washout. This suggests that transient oncogene fluctuations may cause a sustained increase in metastatic potential of Ewing sarcoma cells. Importantly, about thirty of these genes negatively correlated with overall survival in a cohort of 196 Ewing sarcoma patients and were anti-correlated with the GGAA microsatellite-driven transcriptional repressor BCL11B, which was highly sensitive to minor changes in EF thresholds. Collectively, our results highlight the critical role of EF oncogene fluctuations in Ewing sarcoma cell plasticity and patient prognosis providing new insights into the gene regulatory mechanisms involved. Further, our findings have important implications for therapeutic approaches aiming at directly targeting EF. Understanding EF dynamics in Ewing sarcoma serves a paradigmatic approach to the identification of therapeutic vulnerabilities in fusion oncogene-driven cancers.
 Citation Format: Veveeyan Suresh, Christoph Hafemeister, Sarah Grissenberger, Caterina Sturtzel, Florencia Cidre Aranaz, Thomas Grünewald, Ana Sastre, Javier Alonso, Martin Distel, Florian Halbritter, Heinrich Kovar, Valerie Fock. Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A017.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/533254be9587cdde4641f2edde320cdf2280f903" target='_blank'>
              Abstract A017 Dynamic modelling of EWS::FLI1 fluctuations reveals molecular determinants of phenotypic tumor plasticity and prognosis in Ewing sarcoma
              </a>
            </td>
          <td>
            Veveeyan Suresh, Christoph Hafemeister, S. Grissenberger, C. Sturtzel, F. C. Aranaz, Thomas Grünewald, A. Sastre, Javier Alonso, Martin Distel, F. Halbritter, Heinrich Kovar, Valerie Fock
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The fusion expression of DNA replication-related proteins with nucleotide deaminase enzymes promotes random mutations in bacterial genomes, thereby increasing genetic diversity among population. Most previous studies have focused on cytosine deaminase, which produces only C→T mutations, significantly limiting the variety of mutation types. In this study, we developed a fusion expression system by combining DnaG (RNA primase) with adenine deaminase TadA-8e (DnaG-TadA) in Escherichia coli, which is capable of rapidly introducing A→G mutations into the E. coli genome, resulting in a 664-fold increase in terms of mutation rate. Additionally, we engineered a dual-functional TadA variant, TadAD, and then fused it with DnaG. This construct introduced both C→T and A→G mutations into the E. coli genome, with the mutation rate further increased by 370-fold upon co-expression with an uracil glycosylase inhibitor (DnaG-TadAD-UGI). We applied DnaG-TadA and DnaG-TadAD-UGI systems to the adaptive laboratory evolution for Cd2+ and kanamycin resistance, achieving an 8.0 mM Cd2+ and 200 μg/mL kanamycin tolerance within just 17 days and 132 hours, respectively. Compared to conventional evolution methods, the final tolerance levels were increased by 320% and 266%, respectively. Our work offers a novel strategy for random mutagenesis in E. coli and potentially other prokaryotic species. For TOC only">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8dc3e641872269a76809a2a9f58812a1c2451c2" target='_blank'>
              Genome-wide A→G and C→T Mutations Induced by Functional TadA Variants in Escherichia coli
              </a>
            </td>
          <td>
            Hao Wang, Zhengxin Dong, Jingyi Shi, Lei Chen, Tao Sun, Weiwen Zhang
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f217995bbe1b34f9aeac0d831228d08545af9e" target='_blank'>
              Decoding single-cell molecular mechanisms in astrocyte-to-iN reprogramming via Ngn2- and Pax6-mediated direct lineage switching
              </a>
            </td>
          <td>
            Rongxing Qin, Yingdan Zhang, Yue Yang, Jiafeng Chen, Lijuan Huang, Wei Xu, Qingchun Qin, Xiaojun Liang, Xinyu Lai, Xiaoying Huang, Minshan Xie, Li Chen
          </td>
          <td>2024-07-27</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Acute lymphoblastic leukemia (ALL) is an aggressive hematological malignancy that affects thousands of pediatric and adult patients each year. A major hurdle to developing more effective treatments is a limited understanding of the genes and pathways that drive ALL malignancy, and which are required for continued tumor growth. ALL is often diagnosed with the onset of advanced symptoms, making it difficult to study the early drivers of oncogenic initiation. Identifying novel genetic interactions and pathways capable of driving ALL will help to develop new diagnostic criteria and identify actionable drug targets for clinical treatments. In addition, genetic interactions between co-expressed oncogenes and their impact on initiation, progression, and therapy resistance is difficult to study and requires the costly use of genetically engineered animal models, usually mice. Here, we report a novel screening method using transgenic zebrafish to unbiasedly identify collaborating oncogenic drivers that when co-expressed lead to T- and B-ALL. We screened a transgene pool of 68 putative oncogenes identified from human ALL for synergies in driving the initiation of leukemia using a large-scale F0 transgenic gain-of-function screen in zebrafish. We recovered zebrafish with tumors that were predominantly T- and B-ALL, but also identified several fish with thymic T cell lymphomas, muscle sarcomas, and liposarcomas, demonstrating the robust capability of our approach to discover novel oncogenic collaborations. Comparative transcriptome analysis using bulk RNA sequencing between zebrafish tumors (>150 tumors) revealed candidate transgene combinations that likely act as drivers of tumor formation in T- and B-ALL. When microinjected for validation studies, two transgene combinations: 1) rag2:SET + rag2:intracellular notch1a and 2) rag2:SET + rag2:mutationally-activated IL7R lead to robust ALL formation in zebrafish, validating the oncogenic role of these factors in leukemia initiation. SET is a multifunctional protein involved in transcriptional regulation, histone binding, and cell death. While SET has been well characterized in other cancers, we found that SET is capable of collaborating with known drivers to initiate T-ALL in vivo and highly expressed in a vast majority of human T-ALL. Our SET models also revealed an enrichment of key pathways that drive proliferation in both zebrafish and human disease. We are currently performing molecular characterization of SET in human ALL cell lines to investigate SET driven ALL growth and maintenance. In addition, our panel of unique zebrafish tumors will allow help establish new models of hematologic malignancies, launch future investigations into leukemia pathology, and identify novel targets of therapy resistance.
 Citation Format: James R. Allen, Luis Antonio Corchete Sanchez, Mohamed N. Bakr, Miriam Fernandez-Lajarin, Alexandra Hazelwood, Nathan Ford, Alexandra Veloso, Victoriano Mulero, Maria L. Cayuela, Esther Rheinbay, David M. Langenau. A transgenic zebrafish screen identifies new collaborating oncogenic drivers in acute lymphoblastic leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091a253fa76c63ad6deaa84d3f6f708b6e59ab16" target='_blank'>
              Abstract A002 A transgenic zebrafish screen identifies new collaborating oncogenic drivers in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            J. R. Allen, Luis Antonio Corchete Sanchez, Mohamed N. Bakr, Miriam Fernández-Lajarín, Alexandra Hazelwood, Nathan Ford, Alexandra Veloso, Victoriano Mulero, M. Cayuela, Esther Rheinbay, D. Langenau
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="A fundamental challenge in biomedicine is understanding the mechanisms predisposing individuals to disease. While previous research has suggested that switch-like gene expression is crucial in driving biological variation and disease susceptibility, a systematic analysis across multiple tissues is still lacking. By analyzing transcriptomes from 943 individuals across 27 tissues, we identified 1,013 switch-like genes. We found that only 31 (3.1%) of these genes exhibit switch-like behavior across all tissues. These universally switch-like genes appear to be genetically driven, with large exonic genomic structural variants explaining five (∼18%) of them. The remaining switch-like genes exhibit tissue-specific expression patterns. Notably, tissue-specific switch-like genes tend to be switched on or off in unison within individuals, likely under the influence of tissue-specific master regulators, including hormonal signals. Among our most significant findings, we identified hundreds of concordantly switched-off genes in the stomach and vagina that are linked to gastric cancer (41-fold, p<10-4) and vaginal atrophy (44-fold, p<10-4), respectively. Experimental analysis of vaginal tissues revealed that low systemic levels of estrogen lead to a significant reduction in both the epithelial thickness and the expression of the switch-like gene ALOX12. We propose a model wherein the switching off of driver genes in basal and parabasal epithelium suppresses cell proliferation therein, leading to epithelial thinning and, therefore, vaginal atrophy. Our findings underscore the significant biomedical implications of switch-like gene expression and lay the groundwork for potential diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d158a86f2a151fa830972300750ba84c1a11dbfc" target='_blank'>
              Switch-like Gene Expression Modulates Disease Susceptibility
              </a>
            </td>
          <td>
            Alber Aqil, Yanyan Li, Zhiliang Wang, Saiful Islam, Madison Russell, T. Kunovac Kallak, Marie Saitou, Omer Gokcumen, N. Masuda
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Mismatch repair proficient (MMRp) tumors of colorectal origin are one of the prevalent yet unpredictable clinical challenges. Despite earnest efforts, optimal treatment modalities have yet to emerge for this class. The poor prognosis and limited actionability of MMRp are ascribed to a low neoantigen burden and a desert-like microenvironment. This review focuses on the critical roadblocks orchestrated by an immune evasive mechanistic milieu in the context of MMRp. The low density of effector immune cells, their weak spatiotemporal underpinnings, and the high-handedness of the IL-17-TGF-β signaling are intertwined and present formidable challenges for the existing therapies. Microbiome niche decorated by Fusobacterium nucleatum alters the metabolic program to maintain an immunosuppressive state. We also highlight the evolving strategies to repolarize and reinvigorate this microenvironment. Reconstruction of anti-tumor chemokine signaling, rational drug combinations eliciting T cell activation, and reprograming the maladapted microbiome are exciting developments in this direction. Alternative vulnerability of other DNA damage repair pathways is gaining momentum. Integration of liquid biopsy and ex vivo functional platforms provide precision oncology insights. We illustrated the perspectives and changing landscape of MMRp-CRC. The emerging opportunities discussed in this review can turn the tide in favor of fighting the treatment dilemma for this elusive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823c184571d6413de5f857c9ff7de13bd6953768" target='_blank'>
              Mismatch repair-proficient tumor footprints in the sands of immune desert: mechanistic constraints and precision platforms
              </a>
            </td>
          <td>
            Biswanath Majumder, N. Nataraj, Leela Maitreyi, Santanu Datta
          </td>
          <td>2024-07-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Gastrointestinal tumors, the second leading cause of human mortality, are characterized by their association with inflammation. Currently, progress in the early diagnosis and effective treatment of gastrointestinal tumors is limited. Recent whole‐genome analyses have underscored their profound heterogeneity and extensive genetic and epigenetic reprogramming. Epigenetic reprogramming pertains to dynamic and hereditable alterations in epigenetic patterns, devoid of concurrent modifications in the underlying DNA sequence. Common epigenetic modifications encompass DNA methylation, histone modifications, noncoding RNA, RNA modifications, and chromatin remodeling. These modifications possess the potential to invoke or suppress a multitude of genes associated with cancer, thereby governing the establishment of chromatin configurations characterized by diverse levels of accessibility. This intricate interplay assumes a pivotal and indispensable role in governing the commencement and advancement of gastrointestinal cancer. This article focuses on the impact of epigenetic reprogramming in the initiation and progression of gastric cancer, esophageal cancer, and colorectal cancer, as well as other uncommon gastrointestinal tumors. We elucidate the epigenetic landscape of gastrointestinal tumors, encompassing DNA methylation, histone modifications, chromatin remodeling, and their interrelationships. Besides, this review summarizes the potential diagnostic, therapeutic, and prognostic targets in epigenetic reprogramming, with the aim of assisting clinical treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa46852e2f3c4f502e85931e028a5b5bde3a9141" target='_blank'>
              Epigenetic reprogramming in gastrointestinal cancer: biology and translational perspectives
              </a>
            </td>
          <td>
            Yingjie Wang, Hongyu Liu, Mengsha Zhang, Jing Xu, Liuxian Zheng, Pengpeng Liu, Jingyao Chen, Hongyu Liu, Chong Chen
          </td>
          <td>2024-08-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The DNA damage repair (DDR) pathway is a complex signaling cascade that can sense DNA damage and trigger cellular responses to DNA damage to maintain genome stability and integrity. A typical hallmark of cancer is genomic instability or nonintegrity, which is closely related to the accumulation of DNA damage within cancer cells. The treatment principles of radiotherapy and chemotherapy for cancer are based on their cytotoxic effects on DNA damage, which are accompanied by severe and unnecessary side effects on normal tissues, including dysregulation of the DDR and induced therapeutic tolerance. As a driving factor for oncogenes or tumor suppressor genes, noncoding RNA (ncRNA) have been shown to play an important role in cancer cell resistance to radiotherapy and chemotherapy. Recently, it has been found that ncRNA can regulate tumor treatment tolerance by altering the DDR induced by radiotherapy or chemotherapy in cancer cells, indicating that ncRNA are potential regulatory factors targeting the DDR to reverse tumor treatment tolerance. This review provides an overview of the basic information and functions of the DDR and ncRNAs in the tolerance or sensitivity of tumors to chemotherapy and radiation therapy. We focused on the impact of ncRNA (mainly microRNA [miRNA], long noncoding RNA [lncRNA], and circular RNA [circRNA]) on cancer treatment by regulating the DDR and the underlying molecular mechanisms of their effects. These findings provide a theoretical basis and new insights for tumor-targeted therapy and the development of novel drugs targeting the DDR or ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63af0c8bea30d510199d8c53e8bb1fd81c1002b8" target='_blank'>
              DNA damage response-related ncRNAs as regulators of therapy resistance in cancer
              </a>
            </td>
          <td>
            Ziru Gao, Xinchi Luan, Xuezhe Wang, Tianyue Han, Xiaoyuan Li, Zeyang Li, Peifeng Li, Zhixia Zhou
          </td>
          <td>2024-08-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="PURPOSE
High-grade complex karyotype sarcomas are a heterogeneous group of tumors with a uniformly poor prognosis. Within complex karyotype sarcomas, there are innumerable genetic changes but identifying those that are clinically relevant has been challenging.


EXPERIMENTAL DESIGN
To address this, we utilized a pooled genetic screening approach, informed by TCGA data, to identify key drivers and modifiers of sarcoma development that were validated in vivo.


RESULTS
YAP1 and wildtype KRAS were validated as drivers and transformed human mesenchymal stem cells into two distinct sarcoma subtypes, undifferentiated pleomorphic sarcoma (UPS) and myxofibrosarcoma (MFS), respectively. A subset of tumors driven by CDK4 and PIK3CA reflected leiomyosarcoma (LMS) and osteosarcoma (OS) demonstrating the plasticity of this approach and the potential to investigate sarcoma subtype heterogeneity. All generated tumors histologically reflected human sarcomas and had increased aneuploidy as compared to simple karyotype sarcomas. Comparing differential gene expression of TCGA samples to model data identified increased oxidative phosphorylation signaling in YAP1 tumors. Treatment of a panel of soft tissue sarcomas with a combination of YAP1 and oxidative phosphorylation inhibitors led to significantly decreased viability.


CONCLUSIONS
Transcriptional co-analysis of TCGA patient samples to YAP1 and KRAS model tumors support that these sarcoma subtypes lie along a spectrum of disease and adds guidance for further transcriptome-based refinement of sarcoma subtyping. This approach can be used to begin to understand pathways and mechanisms driving human sarcoma development, the relationship between sarcoma subtypes and to identify and validate new therapeutic vulnerabilities for this aggressive and heterogeneous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdaada75e0a2b742920f7cff889cd677616ee1d5" target='_blank'>
              Genetic Screen in a Pre-Clinical Model of Sarcoma Development Defines Drivers and Therapeutic Vulnerabilities.
              </a>
            </td>
          <td>
            Jack Freeland, Maria Muñoz, Edmond O'Donnell, J. Langerman, Morgan Darrow, Jessica Bergonio, J. Suarez-Navarro, Steven Thorpe, Robert Canter, R. L. Randall, Kathrin Plath, Kermit L Carraway, Owen N. Witte, Thomas G. Graeber, Janai R. Carr-Ascher
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35393ed7857060735a43bccb7ec1795bc1e251a9" target='_blank'>
              Comprehensive molecular characterization of TFE3-rearranged renal cell carcinoma
              </a>
            </td>
          <td>
            Cho-Rong Lee, J. Suh, Dongjun Jang, Bo-Yeong Jin, Jaeso Cho, Moses Lee, Hyungtai Sim, Minyong Kang, Jueun Lee, Ju Hyun Park, Kyoung-Hwa Lee, Geum-Sook Hwang, Kyung Chul Moon, Cheryn Song, J. Ku, C. Kwak, Hyeon-Hoe Kim, Sung-Yup Cho, Murim Choi, Chang Wook Jeong
          </td>
          <td>2024-08-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Simple Summary This research aims to explore the role of the TSGA10 protein in cancer development, specifically in how it might influence the growth and spread of cancer cells. Scientists are particularly interested in TSGA10 because it is found in both normal reproductive tissues and cancer cells, yet seems to slow down cancer progression. The key question is why cancer cells would produce a protein that could hinder their own survival. To investigate this, the authors propose several hypotheses about how TSGA10 might be involved in carcinogenesis. They will analyze both published and unpublished studies and data to understand how TSGA10 functions at different stages of cancer. By uncovering these mechanisms, this research could lead to new targeted therapies that use TSGA10 to combat cancer more effectively, offering fresh insights and potential breakthroughs in cancer treatment. Abstract Cancer-specific antigens have been a significant area of focus in cancer treatment since their discovery in the mid-twentieth century. Cancer germline antigens are a class of antigens specifically overexpressed in germline tissues and cancer cells. Among these, TSGA10 (testis-specific gene antigen 10) is of great interest because of its crucial impact on cancer progression. Early studies explored TSGA10 expression in a variety of cancer types. More recent studies revealed that TSGA10 can suppress tumor progression by blocking cancer cell metabolism, angiogenesis, and metastasis. An open question regarding the TSGA10 is why cancer cells must express a protein that prevents their progression. To answer this question, we conducted a comprehensive review to engage the TSGA10 in the context of the current understanding of “malignant transformation”. This review demonstrated that TSGA10 expression level in cancer cells depends on the cancer stage across malignant transformation. In addition, we evaluated how TSGA10 expression can prevent the “cancer hallmarks”. Given this information, TSGA10 can be of great interest in developing effective targeted anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fbb75a98c933791c688ebdc1263dceb33c21d30" target='_blank'>
              Exploring TSGA10 Function: A Crosstalk or Controlling Mechanism in the Signaling Pathway of Carcinogenesis?
              </a>
            </td>
          <td>
            F. Taghizadeh-Hesary, Mobina Ghadyani, F. Kashanchi, Babak Behnam
          </td>
          <td>2024-08-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="CRISPR/Cas9 gene editing technology is a promising tool for correcting pathogenic variants for autologous cell therapies for Inborn Errors of Immunity (IEI). The present IEI correction strategies mainly focus on the knock-in of therapeutic cDNAs, or knockout of the disease-causing gene when feasible. These strategies address many single-gene defects but may disrupt gene expression and require significant optimization for each newly discovered IEI-causing gene, highlighting the need for complementary platforms that can precisely correct diverse pathogenic variants. Here, we present a safe and efficient T cell single nucleotide variant (SNV) correction pipeline based on homology-directed repair (HDR), suitable for diverse monogenic mutations. By using founder mutations of Deficiency of ADA2 (DADA2), Autoimmune polyendocrinopathy-candidiasis-ectodermal dystrophy (APECED) and Cartilage Hair Hypoplasia (CHH) as IEI models, we show that our pipeline can achieve up to 80% bi-allelic editing, with resultant functional correction of the disease phenotype in patient T cells. We do not find detectable pre-malignant off-target effects or karyotypic, transcriptomic or proteomic aberrations upon profiling patient T cells with GUIDE-seq, single cell RNA sequencing, PacBio based long-read whole genome sequencing, and high-throughput proteomics. This study demonstrates that HDR-based SNV editing is a safe and effective option for IEI T cell correction and that it could be developed to an autologous T cell therapy, as the presented protocol is scalable for a GMP-compatible workflow. This study is a step towards the development of gene correction platform that targets a broad number of monogenic mutations. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3514ea764ac815dfdaa1d9a93597841e9c3e8944" target='_blank'>
              T cell correction pipeline for Inborn Errors of Immunity
              </a>
            </td>
          <td>
            Katariina Mamia, Sólrún Kolbeinsdóttir, Zhuokun Li, Kornel Labun, Anna Komisarczuk, S. Keskitalo, Ganna Reint, Frida Høsøien Haugen, Britt Olaug Lindestad, T. Gjerdingen, Antti Tuhkala, Carolina Wieczorek Ervik, Pavel Kopcil, Nail Fatkhutdinov, Monika Szymańska, Eero Tölö, V. Glumoff, J. Saarela, T. Michelsen, Camilla Schalin-Jäntti, Johanna Olweus, Eira Leinonen, M. Varjosalo, Eivind Valen, Timo Hautala, Martin Enge, T. Martelius, Shiva Dahal-Koirala, E. Haapaniemi
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Genetic editing of the germline using CRISPR/Cas9 technology has made it possible to alter livestock traits, including the creation of resistance to viral diseases. However, virus adaptability could present a major obstacle in this effort. Recently, chickens resistant to avian leukosis virus subgroup J (ALV-J) were developed by deleting a single amino acid, W38, within the ALV-J receptor NHE1 using CRISPR/Cas9 genome editing. This resistance was confirmed both in vitro and in vivo. In vitro resistance of W38-/- chicken embryonic fibroblasts to all tested ALV-J strains was shown. To investigate the capacity of ALV-J for further adaptation, we used a retrovirus reporter-based assay to select adapted ALV-J variants. We assumed that adaptive mutations overcoming the cellular resistance would occur within the envelope protein. In accordance with this assumption, we isolated and sequenced numerous adapted virus variants and found within their envelope genes eight independent single nucleotide substitutions. To confirm the adaptive capacity of these substitutions, we introduced them into the original retrovirus reporter. All eight variants replicated effectively in W38-/- chicken embryonic fibroblasts in vitro while in vivo, W38-/- chickens were sensitive to tumor induction by two of the variants. Importantly, receptor alleles with more extensive modifications have remained resistant to the virus. These results demonstrate an important strategy in livestock genome engineering towards antivirus resistance and illustrate that cellular resistance induced by minor receptor modifications can be overcome by adapted virus variants. We conclude that more complex editing will be necessary to attain robust resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8a5c8f973522317dce2248fd8edd93f9386c51" target='_blank'>
              Rapid adaptive evolution of avian leukosis virus subgroup J in response to biotechnologically induced host resistance
              </a>
            </td>
          <td>
            M. Matoušková, Jiří Plachý, Dana Kučerová, Ľ. Pecnová, M. Reinišová, J. Geryk, Vít Karafiát, T. Hron, J. Hejnar
          </td>
          <td>2024-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Tracing cell lineages has become a valuable tool for studying biological processes. Among the available tools for human data, mitochondria DNA (mtDNA) has a high potential due to its ability to be used in conjunction with single-cell chromatin accessibility data, giving access to the cell phenotype. Nonetheless, the existing mutation calling tools are ill-equipped to deal with the polyploid nature of the mtDNA and lack a robust statistical framework. Here we introduce scMitoMut, an innovative R package that leverages statistical methodologies to accurately identify mitochondrial lineage related mutations at the single-cell level. scMitoMut assigns a mutation quality q-value based on beta-binomial distribution to each mutation at each locus within individual cells, ensuring higher sensitivity and precision of lineage related mutation calling in comparison to current methodologies. We tested scMitoMut using single-cell DNA sequencing, scATAC sequencing and 10× Genomics single cell multiome datasets. Using a single-cell DNA sequencing dataset from a mixed population of cell lines, scMitoMut demonstrated superior sensitivity in identifying small proportion of cancer cell lines compared to existing methods. In a human colorectal cancer scATAC dataset, scMitoMut identified more mutations than state-of-the-art methods. Applied to 10× Genomics multiome datasets, scMitoMut effectively measured the lineage distance in cells from blood or brain tissues. Thus, the scMitoMut is a free available (https://www.bioconductor.org/packages/devel/bioc/html/scMitoMut.html.), well-engineered toolkit for mtDNA mutation calling with high memory and CPU efficiency. Consequently, it will significantly advance the application of single-cell sequencing, facilitating the precise delineation of mitochondrial mutations for lineage tracing purposes in development, tumor and stem cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a057cdbd6d7399eed5dec23b92e83e85fe7d0c9" target='_blank'>
              scMitoMut for calling mitochondrial lineage–related mutations in single cells
              </a>
            </td>
          <td>
            Wenjie Sun, Daphne van Ginneken, Leïla Perié
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e335090c3e9a1ddfa4a9c87a808106af39d43ca" target='_blank'>
              Gene regulatory networks reveal sex difference in lung adenocarcinoma
              </a>
            </td>
          <td>
            Enakshi Saha, Marouen Ben Guebila, Viola Fanfani, Jonas Fischer, K.H. Shutta, Panagiotis Mandros, Dawn L. DeMeo, John Quackenbush, C. Lopes-Ramos
          </td>
          <td>2024-08-06</td>
          <td>Biology of Sex Differences</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive and lethal form of brain cancer with few effective treatments. In this context, Zika virus has emerged as a promising therapeutic agent due to its ability to selectively infect and kill GBM cells. To elucidate these mechanisms and expand the landscape of oncolytic virotherapy, we pursued a transcriptomic meta-analysis comparing the molecular signatures of Zika infection in GBM and neuroblastoma (NBM). Over-representation analysis of dysregulated coding genes showed significant enrichment of tumor necrosis factor (TNF), NF-κB, and p53 signaling pathways. A refined list of long non-coding RNAs consistently dysregulated in Zika-infected GBMs was also developed. Functional review of these candidates revealed their potential regulatory role in Zika-mediated oncolysis. We performed validation of the less-researched targets in adult and pediatric GBM cell lines and found significant differential regulation, as predicted. Altogether, our results provide novel insights into the molecular mechanisms underlying the effect of Zika on GBM. We highlight potential therapeutic targets that could be further interrogated to improve the efficacy of tumor cell death and the utility of Zika as an adjuvant virotherapy for GBM and other related cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5fb2f82f19cb82c54f2e078dc5fd330417c5a7" target='_blank'>
              Transcriptomic Meta-analysis Identifies Long Non-Coding RNAs Mediating Zika’s Oncolytic Impact in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Youssef A. Kousa, Shriya Singh, Allison Horvath, Federica Tomasso, Javad Nazarian, Lisa Henderson, Tamer A. Mansour
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Objectives: Neuroblastoma (NB), a pediatric malignancy of the peripheral nervous system, is characterized by epigenetic and transcriptional (EP-TF) anomalies. This study aimed to develop an EP-TF clinical prognostic model for NB using CRISPR-Cas9 knockout screening. Results: An integrative analysis was conducted using CRISPR-Cas9 screening in vitro and in vivo with public NB datasets to identify 35 EP-TF genes that exhibited the highest expression in NB and were highly dependent on cancer viability. After univariate analysis, 27 of these 35 genes were included in the least absolute shrinkage and selection operator screen. We established and biologically validated a prognostic EP-TF model encompassing RUVBL1, LARP7, GTF3C4, THAP10, SUPT16H, TIGD1, SUV39H2, TAF1A, SMAD9, and FEM1B across diverse NB cohorts. MYCN serves a potential upstream regulator of EP-TF genes. The high-risk subtype exhibited traits associated with the malignant cell cycle, MYCN-linked signaling and chromatin remodeling, all of which are correlated with poor prognosis and immunosuppression. MEK inhibitors have emerged as promising therapeutic agents for targeting most EP-TF risk genes in NB. Conclusion: Our novel prognostic model shows significant potential for predicting and evaluating the overall survival of NB patients, offering insights into therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec96366563af269a504d1c7dc22ef594a4897d48" target='_blank'>
              CRISPR-Cas9 screening develops an epigenetic and transcriptional gene signature for risk stratification and target prediction in neuroblastoma
              </a>
            </td>
          <td>
            Liaoran Zhang, Jialin Mo, Hao Shi, Jing Xiong, Yeerfan Aierken, Feng Chen, Yujie Tang, Kewen Zhao, Zhibao Lv, Kezhe Tan
          </td>
          <td>2024-08-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="High risk - neuroblastoma (HR-NB) is a pediatric solid tumor with high lethality. Half of HR-NB are driven by MYCN gene amplification (MNA). These HR-NBs require high dosage chemotherapy and often relapse. Moreover, current therapies can cause severe long-term side effects and new therapies are urgently needed. This study investigates a novel therapeutic approach targeting the metabolic vulnerabilities of MNA NB cells. We discovered that Diphenyleneiodonium chloride (DPI), an inhibitor of flavoprotein enzymes and mitochondrial complex I, synergizes with mitoquinone mesylate (MitoQ), a mitochondria-targeted antioxidant in 2D and 3D in vitro models of NB. Similarly to DPI, MitoQ affects MNA cells in a MYCN-dependent way, being more toxic when MYCN levels are high. Furthermore, low nanomolar concentrations of MitoQ significantly decrease MYCN protein expression and induce differentiation of MNA cells. The DPI and MitoQ combination further synergizes with vincristine, a chemotherapeutic agent used in NB treatment. Phosphoproteomics and proteomics analysis suggests that the drug combination induces MNA NB cell death by arresting the cell cycle and inhibiting oxidative phosphorylation (OXPHOS) in the mitochondria. Thus, interference with mitochondrial metabolism may represent an effective strategy to enhance the activity of chemotherapeutic drugs in MNA-NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84bd27bba09b6f802fb233fccdb2001e1425eb1" target='_blank'>
              OXPHOS TARGETING OF MYCN-AMPLIFIED NEUROBLASTOMA
              </a>
            </td>
          <td>
            Soraya Epp, Donagh Egan, E. Poon, Amirah Adlina Abdul Aziz, Kieran Wynne, Melinda Halasz, Walter Kolch
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Breast cancer is a heterogeneous disease that arises as a multi-stage process involving multiple cell types. Patients diagnosed with the same clinical stage and pathological classification may have different prognoses and therapeutic responses due to alterations in molecular genetics. As an essential marker for the molecular subtyping of breast cancer, long non-coding RNAs (lncRNAs) play a crucial role in gene expression regulation, cell differentiation, and the maintenance of genomic stability. Here, we developed a modular framework for lncRNA identification and applied it to a breast cancer cohort to identify novel lncRNAs not previously annotated. To investigate the potential biological function, regulatory mechanisms, and clinical relevance of the novel lncRNAs, we elucidated the genomic and chromatin features of these lncRNAs, along with the associated protein-coding genes and putative enhancers involved in the breast cancer regulatory networks. Furthermore, we uncovered that the expression patterns of novel and annotated lncRNAs identified in breast cancer were related to the hormone response in the PAM50 subtyping criterion, as well as the immune response and progression states of breast cancer across different immune cells and immune checkpoint genes. Collectively, the comprehensive identification and functional analysis of lncRNAs revealed that these lncRNAs play an essential role in breast cancer by altering gene expression and participating in the regulatory networks, contributing to a better insight into breast cancer heterogeneity and potential avenues for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b6bf72d9207f000ec46937ff3a3ba19091f1e49" target='_blank'>
              Comprehensive Transcriptome Analysis Expands lncRNA Functional Profiles in Breast Cancer
              </a>
            </td>
          <td>
            Wenyong Zhu, Hao Huang, Zixi Hu, Yu Gu, Rong Zhang, Huiling Shu, Hongjia Liu, Xiao Sun
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction Biological processes such as aging, carcinogenesis, and immune response rely on the ability to maintain or rapidly expand cell populations. The fitness of the involved cells is constrained by their replicative potential, which is reflected in the cellular telomere content. Method We apply TelomereHunter to scATAC-seq data to determine telomere content on single-cell level, in a hematopoietic dataset consisting of 35,139 cells from samples of basal cell carcinoma patients receiving programmed cell death protein 1 (PD1) blockade treatment. Integrating information from open-chromatin-based signatures to assess cell identity, we characterize the heterogeneity of telomere length for individual cell populations pre- and post-immunotherapy. Results The extracted telomeric reads reflect the expected telomereome-to-genome fraction. Telomere content distributions differ significantly between cell populations, and the median telomere content in intermediate and terminal exhausted CD8+ T-cells pre-treatment is significantly correlated to response to PD-1 checkpoint blockade. Likewise, telomere content correlates with post-treatment cell proliferation in terminally exhausted and T follicular helper cells from responding patients. Conclusion Telomere content measurement from scATAC-seq data has a sufficiently high signal-to-noise ratio to detect significant differences between cell types. Furthermore, the telomere content of CD8+ exhausted T-cells pre-treatment is a putative biomarker for successful PD-1-based immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c0b43ae3c8ae5883f7e563f1131356d52bd5e2" target='_blank'>
              Quantifying immune cell telomere content at single-cell resolution in context of PD-1 checkpoint immunotherapy
              </a>
            </td>
          <td>
            Niklas L. Engel, Lea Herzel, Julie Surmely, Hanna Frieß, Malte Simon, B. Brors, Charles D. Imbusch, L. Feuerbach
          </td>
          <td>2024-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Allele-specific expression (ASE) is the differential abundance in levels of mRNAs that originated from the paternal and maternal copies of a gene. Such allelic imbalances can contribute to phenotypic variation and influence disease traits, including cancer. There is common ASE in tumors that results from somatic copy-number alterations (CNAs) at the DNA level, but there also exist other causes of ASE: cis-acting genetic or epigenetic variation that can lead to differential expression between the two alleles. However, the latter, non-CNA mechanisms of ASE remain understudied in cancer, as well as their role in tumor evolution and impact on clinical outcomes. By integrating a wide variety of genomic and transcriptomic pan-cancer data from the TCGA project, we show that ASE favoring the preferential expression of the mutant allele in some driver genes is subject to positive selection, and that these events are associated with worse overall survival across all cancer types. We found that the impact of ASE triggered by non-CNA causes is substantial, and we propose that some instances of cis-ASE are explained by the epigenetic changes affecting alleles differently. Furthermore, as a second mechanism, we find that splicing-altering mutations are selected in various cancer genes and result in ASE. We anticipate that the study and understanding of the role of mutant allele imbalances at the mRNA level can help understand epigenetic changes during cancer evolution, as well as identify new prognostic markers and therapeutic approaches that target altered allelic expression in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a918fe3f2472f99c1403cdc4bbe583961852532" target='_blank'>
              Allele-specific expression is selected in tumorigenesis, results from epigenetic changes and has prognostic relevance
              </a>
            </td>
          <td>
            Guillermo Palou-Márquez, Pere Pericot-Masdevall, F. Supek
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) poses significant challenges due to its complex biological characteristics and high recurrence rate. The high recurrence rate of LUAD is closely associated with cellular dormancy, which enhances resistance to chemotherapy and evasion of immune cell destruction. Using single-cell RNA sequencing (scRNA-seq) data from LUAD patients, we categorized the cells into two subclusters: dormant and active cells. Utilizing high-density Weighted Gene Co-expression Network Analysis (hdWGCNA) and pseudo-time cell trajectory, aberrant expression of genes involved in protein O-glycosylation was detected in dormant cells, suggesting a crucial role for O-glycosylation in maintaining the dormant state. Intercellular communication analysis highlighted the interaction between fibroblasts and dormant cells, where the Insulin-like Growth Factor (IGF) signaling pathway regulated by O-glycosylation was crucial. By employing Gene Set Variation Analysis (GSVA) and machine learning, a risk score model was developed using hub genes, which showed high accuracy in determining LUAD prognosis. The model also demonstrated robust performance on the training dataset and excellent predictive capability, providing a reliable basis for predicting patient clinical outcomes. The group with a higher risk score exhibited a propensity for adverse outcomes in the tumor microenvironment (TME) and tumor mutational burden (TMB). Additionally, the 50% inhibitory concentration (IC50) values for chemotherapy exhibited significant variations among the different risk groups. In vitro experiments demonstrated that EFNB2, PTTG1IP, and TNFRSF11A were upregulated in dormant tumor cells, which also contributed greatly to the diagnosis of LUAD. In conclusion, this study highlighted the crucial role of O-glycosylation in the dormancy state of LUAD tumors and developed a predictive model for the prognosis of LUAD patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bdea9ceef7f2c70dc3e70ad8f0a54632f63dceb" target='_blank'>
              Deciphering Dormant Cells of Lung Adenocarcinoma: Prognostic Insights from O-glycosylation-Related Tumor Dormancy Genes Using Machine Learning
              </a>
            </td>
          <td>
            Chenfei Dong, Yang Liu, Suli Chong, Jiayue Zeng, Ziming Bian, Xiaoming Chen, Sairong Fan
          </td>
          <td>2024-08-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The relevant role of LINE-1 (L1) retrotransposition in cancer has been recurrently demonstrated in recent years. However, their repetitive nature hampers their identification and detection, hence remaining inaccessible for clinical practice. Also, its clinical relevance for cancer patients is still limited. Here, we develop a new method to quantify L1 activation, called RetroTest, based on targeted sequencing and a sophisticated bioinformatic pipeline, allowing its application in tumor biopsies. First, we performed the benchmarking of the method and confirmed its high specificity and reliability. Then, we unravel the L1 activation in HNSCC according to a more extensive cohort including all the HNSCC tumor stages. Our results confirm that RetroTest is remarkably efficient for L1 detection in tumor biopsies, reaching a high sensitivity and specificity. In addition, L1 retrotransposition estimation reveals a surprisingly early activation in HNSCC progression, contrary to its classical association with advanced tumor stages. This early activation together with the genomic mutational profiling of normal adjacent tissues supports field cancerization process in this tumor. These results underline the importance of estimating L1 retrotransposition in clinical practice towards an earlier and more efficient diagnosis in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ae21a6395bf2fa53698eb0f53f5dc6c7132bfe" target='_blank'>
              RetroTest unravels LINE-1 retrotransposition in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, Ana Oitaben, Sonia Zumalave, Bernardo Rodriguez-4 Martin, María Gallardo-Gómez, Martín Santamarina, Ana Pequeño-Valtierra, L. Juaneda-Magdalena, Ramón García-Escudero, J. López-Cedrún, Máximo Fraga, J. Tubío, Mónica Martínez-Fernández, Monica Martínez
          </td>
          <td>2024-07-24</td>
          <td>None</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Background Genomic variations contribute to the phenotypic diversity of individuals. A number of polymorphisms in protein-coding regions that alter drug efficacy or lead to adverse reactions have been characterized; however, noncoding regions that affect drug responses are largely overlooked, except for a limited number of well-studied enhancers. Results We conducted a quantitative assessment of cis-regulatory elements (CREs) based on transcription initiation profiling of mRNAs and noncoding RNAs, including enhancer RNAs, by using CAGE (Cap Analysis of Gene Expression). Candidate CREs identified in a hepatocellular carcinoma HepG2 cell line with stable expression of drug-responsive transcription factor pregnane X receptor (PXR) were further narrowed down by integrating data of PXR-binding sites in human primary hepatocytes and genome-wide association studies. We found more than 100-fold enrichments of the candidates to genetically associated loci with circulating levels of bilirubin and vitamin D, which implicated a link to adverse reactions of PXR ligands. We uncovered novel enhancers of UGT1A1 and TSKU through CRISPR/Cas9 knockout experiments. We identified alleles altering regulatory activities of UGT1A1 and CYP24A1enhancers by using luciferase reporter assay. Furthermore, our siRNA experiments revealed an unexpected impact of TSKU on the expression of vitamin D-metabolizing enzymes. Conclusions Our transcriptome-based assessment of CREs expanded the list of drug-inducible and PXR-mediated enhancers and super-enhancers. We identified regulatory alleles that alter drug-induced gene expressions, and discovered a novel molecular cascade associated with an adverse reaction. Our results contribute a precise understanding of the noncoding elements of the human genome underlying drug responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f12494b9c7c48c33bdacd730ac5e83409d9983a" target='_blank'>
              Drug-induced cis-regulatory elements in human hepatocytes affect molecular phenotypes associated with adverse reactions
              </a>
            </td>
          <td>
            Saki Gotoh-Saito, Ryoko Wada, Hideya Kawaji
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epigenetics are the changes due to DNA methylation, histone modifications, and noncoding RNA regulation, which play a crucial part in the expression of genes. Although these changes do not alter the sequence of the DNA, they are significant in determining the phenotype and inheritance of the genes. The impact of epigenetics on inherited diseases like Fragile X syndrome, Angelman syndrome, and Prader-Willi syndrome emphasizes the significance of studying epigenetic dysregulation in disease pathology. The current review aims to investigate the role of epigenetic mechanisms in inherited diseases and to evaluate the utility of molecular diagnostic tools for epigenetic analysis in disease diagnosis. Case studies of diseases such as Duchenne muscular dystrophy and Prader-Willi syndrome highlight the clinical relevance of epigenetic analysis in disease diagnosis and management. This research is conducted through an extensive analysis of recent literature to explore the epigenetic mechanisms underlying inherited diseases. Additionally, advanced molecular diagnostic techniques like droplet digital PCR (ddPCR), transposase-based bisulfite tagging, and data mining coupled with conventional experimental procedures. However, the intricate relationship between genetic and epigenetic factors and challenges related to inclined data need to be addressed. Future research endeavors to elucidate the role of epigenetic modifications in disease pathogenesis and develop strategies for personalized medicinal treatment and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f24f0960efd210e8ee51cbad54070948fecc2ef" target='_blank'>
              A Review on Epigenetics of Human Inherited Diseases: Molecular Diagnosis
              </a>
            </td>
          <td>
            Bibi Shazia Sabir
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Innovative Science and Research Technology (IJISRT)</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Precision oncology promises individually tailored drugs and clinical care for patients with cancer: That is, "the right drug, for the right patient, at the right dose, and at the right time." Although stratification of the risk for treatment resistance and toxicity is key to precision oncology, there are multiple ways in which such stratification can be achieved, for example, genetic, functional pathway based, among others. Moving toward precision oncology is sorely needed in the case of acute lymphoblastic leukemia (ALL) wherein adult patients display survival rates ranging from 30% to 70%. The present study reports on the pathway activity signature of adult B-ALL, with an eye to precision oncology. Transcriptome profiles from three different expression datasets, comprising 346 patients who were adolescents or adults with B-ALL, were harnessed to determine the activity of signaling pathways commonly disrupted in B-ALL. Pathway activity analyses revealed that Ph-like ALL closely resembles Ph-positive ALL. Although this was the case at the average pathway activity level, the pathway activity patterns in B-ALL differ from genetic subtypes. Importantly, clustering analysis revealed that five distinct clusters exist in B-ALL patients based on pathway activity, with each cluster displaying a unique pattern of pathway activation. Identifying pathway-based subtypes thus appears to be crucial, considering the inherent heterogeneity among patients with the same genetic subtype. In conclusion, a pathway-based stratification of the B-ALL could potentially allow for simultaneously targeting highly active pathways within each ALL subtype, and thus might open up new avenues of innovation for personalized/precision medicine in this cancer that continues to have poor prognosis in adult patients compared with the children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72b528a52f9a1c25407f3a5bc1baf86dc4c7b9e9" target='_blank'>
              Expanding Beyond Genetic Subtypes in B-Cell Acute Lymphoblastic Leukemia: A Pathway-Based Stratification of Patients for Precision Oncology.
              </a>
            </td>
          <td>
            Ozlem Ulucan
          </td>
          <td>2024-08-19</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The fundamental transcription factor p53 regulates cellular processes and integrates signals of cellular stress, triggering a coordinated response to ensure survival of cells restored to healthy function and programmed death of those that couldn’t be repaired. Unsurprisingly, this is one of the most mutated genes in human cancers, with most changes occurring in the DNA-binding domain of the protein. In this work, we take a genome-wide approach and use available resources to identify high confidence p53-target genes, that we examine in three breast cancer cell lines with different p53 status, wild type (MCF-7) and different mutations in the DNA-binding domain (MDA-MB231, T47D). Comparison of p53-targets expression in response to DNA damage by RNAseq and cellular assays reveals that MDA-MB231 have a severely impaired p53-dependent pathway functionality while T47D are much less affected. MDA-MB231 are more resistant to DNA damage yet unable to repair and able to override cell cycle arrest leading to survival while T47D are sensitive only to high dose and exposure to genotoxic agents. This data shows the variability of effects of different p53 mutations and highlight the importance of understanding the mechanisms of p53 in the context of genotoxicity-based treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6785633c891564b2ca32ca3538063162d5c2bd" target='_blank'>
              Differential gene expression in cells with different p53 mutations identifies genome-wide p53 targets and shows distinct modulation of cellular pathways in response to DNA damage
              </a>
            </td>
          <td>
            Patricia Eror Barnes, Maria Jose de la Concha, Kioko Mwikali, B. L. Ng, Hannes Ponstingl, Alena Pance
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b7a1b60f27d8f207e63fcabad98fcf75b1aff6" target='_blank'>
              Deciphering the impact of genomic variation on function.
              </a>
            </td>
          <td>
            J. Engreitz, Heather A. Lawson, Harinder Singh, Lea M. Starita, Gary C. Hon, Hannah Carter, Nidhi Sahni, Timothy E. Reddy, Xihong Lin, Yun Li, Nikhil V. Munshi, Maria H. Chahrour, Alan P. Boyle, B. Hitz, Ali Mortazavi, Mark Craven, K. Mohlke, Luca Pinello, Ting Wang, A. Kundaje, Feng Yue, Sarah Cody, Nina P. Farrell, Michael I. Love, Lara A Muffley, M. Pazin, Fairlie Reese, Eric Van Buren, Kushal K. Dey, Martin Kircher, Jian Ma, P. Radivojac, Brunilda Balliu, Brian A. Williams, Danwei Huangfu, Chong Y. Park, Thomas Quertermous, Jishnu Das, M. Calderwood, Douglas M. Fowler, Marc Vidal, Lucas Ferreira, Sean D. Mooney, V. Pejaver, Jingjing Zhao, Steven Gazal, Evan Koch, Steven K. Reilly, S. Sunyaev, Anne E. Carpenter, Jason D. Buenrostro, Christina S. Leslie, Rachel E. Savage, Stefanija Giric, Chongyuan Luo, Kathrin Plath, Alejandro Barrera, Max Schubach, A. Gschwind, Jill E. Moore, N. Ahituv, S. Yi, Ingileif B. Hallgrímsdóttir, K. Gaulton, S. Sakaue, Sina Booeshaghi, Eugenio Mattei, Surag Nair, L. Pachter, Austin T. Wang, J. Shendure, Vikram Agarwal, Andrew Blair, Theofilos Chalkiadakis, Florence M. Chardon, P. Dash, Chengyu Deng, Nobuhiko Hamazaki, Pia Keukeleire, Connor A Kubo, Jean-Benoît Lalanne, Thorben Maass, Beth Martin, Troy A. McDiarmid, Mai Nobuhara, Nicholas F. Page, Sam Regalado, Jasmine Sims, A. Ushiki, Sabrina M. Best, Gabe Boyle, Nathan Camp, Silvia Casadei, Estelle Y. Da, Moez Dawood, Samantha C. Dawson, Shawn Fayer, Audrey Hamm, Richard G. James, Gail P. Jarvik, Abbye E. McEwen, Nick Moore, Sriram Pendyala, Nicholas A Popp, Mason Post, Alan F. Rubin, Nahum T. Smith, J. Stone, Malvika Tejura, Ziyu R. Wang, Melinda K. Wheelock, Ivan Woo, Brendan D. Zapp, Dulguun Amgalan, Aradhana Aradhana, Sophia M. Arana, M. Bassik, Julia R. Bauman, Asmita Bhattacharya, Xiangmeng S. Cai, Ziwei Chen, Stephanie Conley, Salil Deshpande, Benjamin R. Doughty, Peter P. Du, James A. Galante, Casey Gifford, W.J. Greenleaf, Katherine Guo, Revant Gupta, Sarasa Isobe, Evelyn Jagoda, Nimit Jain, Hank Jones, Helen Kang, Samuel H. Kim, YeEun Kim, Sandy Klemm, Ramen Kundu, Soumya Kundu, Mauro Lago-Docampo, Yannick C. Lee-Yow, Roni Levin-Konigsberg, Daniel Y. Li, Dominik Lindenhofer, X. R. Ma, Georgi K. Marinov, Gabriella E. Martyn, Chloe V. McCreery, Eyal Metzl-Raz, João P. Monteiro, Michael T. Montgomery, Kristy S. Mualim, Chad J. Munger, Glen Munson, Tri C. Nguyen, Tri C. Nguyen, Brian T. Palmisano, Anusri Pampari, Marlene Rabinovitch, Markus Ramste, Judhajeet Ray, Kevin R. Roy, Oriane M. Rubio, Julia M. Schaepe, Gavin Schnitzler, Jacob M Schreiber, Disha Sharma, Maya U. Sheth, Huitong Shi, Vasundhara Singh, Riya Sinha, Lars M Steinmetz, Jason Tan, Anthony Tan, Josh Tycko, Raeline Valbuena, Valeh Valiollah Pour Amiri, Mariëlle J. F. M. van Kooten, Alun Vaughan-Jackson, Anthony Venida, Chad S. Weldy, Matthew D. Worssam, Fan Xia, David Yao, Tony Zeng, Quanyi Zhao, Ronghao Zhou, Zitong Sam Chen, B. Cimini, Georges Coppin, Atina G. Coté, Marzieh Haghighi, Tong Hao, David E. Hill, Jessica Lacoste, Florent Laval, Chloe Reno, Fritz Roth, Shantanu Singh, Kerstin Spirohn-Fitzgerald, Mikko Taipale, Tanisha Teelucksingh, Maxime Tixhon, Anupama Yadav, Zhipeng Yang, W. L. Kraus, Daniel A. Armendariz, Ashley E. Dederich, Ashlesha Gogate, Lauretta El Hayek, S. Goetsch, Kiran Kaur, Hyung Bum Kim, Melissa K. McCoy, Mpathi Z. Nzima, C. Pinzon-Arteaga, Bruce A. Posner, Daniel A. Schmitz, Sushama Sivakumar, Anjana Sundarrajan, Lei Wang, Yihan Wang, Jun Wu, Lin Xu, Jian Xu, Leqian Yu, Yanfeng Zhang, Huan Zhao, Qinbo Zhou, Hyejung Won, Jessica L Bell, K. A. Broadaway, Katherine N. Degner, A. Etheridge, Beverly H. Koller, Won Mah, Wancen Mu, Kimberly D. Ritola, Jonathan D. Rosen, Sarah A. Schoenrock, Rachel A. Sharp, Daniel Bauer, G. Lettre, Richard Sherwood, Basheer Becerra, Logan J. Blaine, Eric Che, M. Francoeur, Ellie N. Gibbs, Nahye Kim, Emily M. King, B. Kleinstiver, Estelle Lecluze, Zhijian Li, Zain M. Patel, Quang Vinh Phan, Jayoung Ryu, Marlena L. Starr, Ting Wu, C. Gersbach, Gregory E. Crawford, Andrew S. Allen, William H. Majoros, Nahid Iglesias, Ruhi Rai, Revathy Venukuttan, Boxun Li, Taylor Anglen, Lexi R. Bounds, Marisa C. Hamilton, Siyan Liu, Sean R. McCutcheon, Christian McRoberts Amador, Samuel J. Reisman, Maria A. ter Weele, Josephine C. Bodle, Helen L Streff, Keith Siklenka, Kari Strouse, Bradley E. Bernstein, Juliana Babu, Guillermo Barreto Corona, Kevin Dong, Fabiana M. Duarte, Neva C. Durand, Charles B. Epstein, K. Fan, Elizabeth A. Gaskell, Amelia W. Hall, Alexandra M. Ham, Mei K. Knudson, N. Shoresh, Siddarth Wekhande, Cassandra M. White, Wang Xi, Ansuman T. Satpathy, M. Corces, Serena H. Chang, Iris M. Chin, James M Gardner, Zachary A. Gardell, Jacob C. Gutierrez, Alia W. Johnson, Lucas Kampman, Maya Kasowski, Caleb A. Lareau, Vincent Liu, Leif S. Ludwig, Christopher S. McGinnis, Shreya Menon, Anita Qualls, Katalin D. Sandor, Adam W. Turner, C. J. Ye, Yajie Yin, Wenxi Zhang, Barbara J. Wold, Maria Carilli, Dayeon Cheong, Ghassan Filibam, Kim Green, S. Kawauchi, Charlene Kim, Heidi Liang, Rebekah Loving, L. Luebbert, Grant MacGregor, Angel G. Merchan, Elisabeth Rebboah, Narges Rezaie, Jasmine Sakr, Delaney K. Sullivan, Nikki Swarna, Diane Trout, S. Upchurch, Ryan Weber, Christopher P. Castro, Elysia Chou, Fan Feng, Andre Guerra, Yuanhao Huang, Linghua Jiang, Jie Liu, Ryan E. Mills, Weizhou Qian, Tingting Qin, Maureen A. Sartor, Rintsen N. Sherpa, Jinhao Wang, Yiqun Wang, Joshua D. Welch, Zhenhao Zhang, Nanxiang Zhao, Sayan Mukherjee, C. D. Page, Shannon Clarke, Richard W. Doty, Yuncheng Duan, Raluca Gordân, Kuei-Yueh Ko, Shengyu Li, Boyao Li, Alexander Thomson, Soumya Raychaudhuri, Alkes Price, Thahmina A. Ali, Arun Durvasula, M. Kellis, L. Iakoucheva, Tulika Kakati, Yile Chen, Mariam Benazouz, Shantanu Jain, Daniel Zeiberg, M. C. De Paolis Kaluza, Michelle Velyunskiy, Audrey Gasch, Kunling Huang, Yiyang Jin, Qiongshi Lu, Jiacheng Miao, Michael Ohtake, Eduardo Scopel, Robert D. Steiner, Yuriy Sverchkov, Zhiping Weng, Manuel Garber, Yu Fu, Natalie Haas, Xihao Li, Nishigandha Phalke, Shuo C. Shan, Nicole Shedd, Tianxiong Yu, Yi Zhang, Hufeng Zhou, Alexis Battle, Livnat Jerby, Eran Kotler, Andrew R. Marderstein, Stephen B. Montgomery, AkshatKumar Nigam, Evin M. Padhi, Aman Patel, Jonathan Pritchard, Ivy Raine, Vivekanandan Ramalingam, Kameron B. Rodrigues, Jacob M Schreiber, Arpita Singhal, Marisa Abundis, Deepa Bisht, Trirupa Chakraborty, Jingyu Fan, David R. Hall, Zarifeh H. Rarani, Abhinav K. Jain, Babita Kaundal, Swapnil Keshari, Daniel McGrail, Nicholas A. Pease, Vivian F. Yi, Hao Wu, Sreeram Kannan, Hongjun Song, Jingli Cai, Ziyue Gao, Ronni Kurzion, Julia I. Leu, Fan Li, Dongming Liang, Guo-Li Ming, K. Musunuru, Qi Qiu, Junwei Shi, Yijing Su, Sarah Tishkoff, Ning Xie, Qian Yang, Wenli Yang, Hongjie Zhang, Zhijian Zhang, Michael A Beer, A. Hadjantonakis, Sharon Adeniyi, Hyein Cho, Ronald Cutler, Rachel A. Glenn, David Godovich, Nan Hu, Svetlana Jovanic, Renhe Luo, Jin Woo Oh, Milad Razavi-Mohseni, Dustin Shigaki, Simone Sidoli, Thomas Vierbuchen, Xianming Wang, Breanna Williams, Jielin Yan, Dapeng Yang, Yunxiao Yang, Maike Sander, Bing Ren, W. Bartosik, Hannah S Indralingam, A. Klie, Hannah M Mummey, Mei-Lin Okino, Gaowei Wang, Nathan R. Zemke, Kai Zhang, Han Zhu, Noah Zaitlen, Jason Ernst, J. Langerman, Terence Li, Yu Sun, Alexander Y. Rudensky, P. Periyakoil, Vianne R. Gao, Melanie H. Smith, Norman M. Thomas, L. Donlin, Amit Lakhanpal, Kaden M. Southard, Rico C. Ardy, J. M. Cherry, Mark B. Gerstein, Kalina Andreeva, Pedro Assis, Beatrice Borsari, Eric Douglass, Shengcheng Dong, I. Gabdank, Keenan Graham, Otto Jolanki, Jennifer Jou, Meenakshi S. Kagda
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>1</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c785685c384edfe64958536f2b96ef40bece9a9d" target='_blank'>
              Let’s make it personal: CRISPR tools in manipulating cell death pathways for cancer treatment
              </a>
            </td>
          <td>
            Mobina Bayat, J. S. Nahand
          </td>
          <td>2024-07-29</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Small-cell lung cancer (SCLC) is the most lethal type of lung cancer. Paradoxically, this tumor displays an initial exquisite response to chemotherapy; however, at relapse, the tumor is highly resistant to subsequent available therapies. Here, we report that the expression of three prime repair exonuclease 1 (TREX1) is strongly induced in chemoresistant SCLCs. Assay for transposase-accessible chromatin using sequencing and chromatin immunoprecipitation sequencing revealed a significant increase in chromatin accessibility and transcriptional activity of TREX1 gene locus in chemoresistant SCLCs. Analyses of human SCLC tumors and patient-derived xenografts (PDX) also showed an increase in TREX1 expression in postchemotherapy samples. TREX1 depletion caused the activation of cyclic GMP-AMP synthase stimulator of interferon gene pathway due to cytoplasmic accumulation of damage-associated double-stranded DNA, inducing immunogenicity and enhancing the sensitivity of drug-resistant cells to chemotherapy. These findings suggest TREX1 upregulation may partially contribute to the survival of resistant cells, and its inhibition may represent a promising therapeutic strategy to enhance antitumor immunity and potentiate the efficacy of chemotherapy and/or immunotherapy in chemoresistant SCLCs. Significance: In this study, we show that targeting TREX1 induces an innate immune response and resensitizes SCLC cells to chemotherapy, representing a promising novel target for “immunologically” cold tumors, such as SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920774aa4db00913c78963d621212ab8e7a2e837" target='_blank'>
              Targeting TREX1 Induces Innate Immune Response in Drug-Resistant Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Takahiko Murayama, N. Mahadevan, Catherine B Meador, Elena V. Ivanova, Yuqiao Pan, Erik H. Knelson, Tetsuo Tani, Jun Nakayama, Xueying Ma, Tran C Thai, Yin P Hung, William Kim, Hideo Watanabe, Kathy Q Cai, Aaron N Hata, C. Paweletz, D. Barbie, Israel Cañadas
          </td>
          <td>2024-08-23</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c02145e3ec2ea1a6d60bd9b435d738b941f9a4fa" target='_blank'>
              CRISPR-based genetic screens advance cancer immunology.
              </a>
            </td>
          <td>
            Yuanfang Cao, Xueting Li, Yumu Pan, Huahe Wang, Siyu Yang, Lingjuan Hong, Lupeng Ye
          </td>
          <td>2024-07-23</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dermatofibrosarcoma protuberans (DFSP) is a rare cutaneous sarcoma characterized by the COL1A1-PDGFB fusion gene. This study utilized single-cell RNA sequencing to dissect the cellular and molecular landscape of primary DFSP. Distinct DFSP cell clusters, exhibiting fibroblast-like traits, revealed variations in pathways associated with proliferation, inflammation and metabolism. Differential gene expression analysis during the differentiation from tumour stem cells to DFSP cells unveiled SMOC2, DCN and TGFBR3 as potential regulators of tumour invasion and immune infiltration through VEGF/TGF-β signalling modulation. Cellular communication analysis highlighted interactions within DFSP cell clusters and with endothelial cells, implicating molecules such as NAMPT, ANGPT2 and PTN in pathogenesis and treatment resistance. These findings offer insights into DFSP intratumour heterogeneity, elucidate molecular mechanisms underlying tumour behaviour, and suggest potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32236414fe55f4c60e9436a4ccb2514917fc2935" target='_blank'>
              Cellular and molecular landscape of primary dermatofibrosarcoma protuberans: Insights from single-cell RNA sequencing analysis.
              </a>
            </td>
          <td>
            Rui Peng, Yingyi Li, Yumei Gao, Dianfu Chen, Zhenghui Li, Yi Zhao
          </td>
          <td>2024-07-30</td>
          <td>Experimental dermatology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Uveal melanoma (UM) is a highly metastatic cancer with resistance to immunotherapy. The present study aimed to identify novel feature genes and molecular mechanisms in UM through analysis of single-cell sequencing data. For this purpose, data were downloaded from The Cancer Genome Atlas and National Center for Biotechnology Information Gene Expression Omnibus public databases. The statistical analysis function of the CellPhoneDB software package was used to analyze the ligand-receptor relationships of the feature genes. The Metascape database was used to perform the functional annotation of notable gene sets. The randomForestSRC package and random survival forest algorithm were applied to screen feature genes. The CIBERSORT algorithm was used to analyze the RNA-sequencing data and infer the relative proportions of the 22 immune-infiltrating cell types. In vitro, small interfering RNAs were used to knockdown the expression of target genes in C918 cells. The migration capability and viability of these cells were then assessed by gap closure and Cell Counting Kit-8 assays. In total, 13 single-cell sample subtypes were clustered by t-distributed Stochastic Neighbor Embedding and annotated by the R package, SingleR, into 7 cell categories: Tissue stem cells, epithelial cells, fibroblasts, macrophages, natural killer cells, neurons and endothelial cells. The interactions in NK cells|Endothelial cells, Neurons|Endothelial cells, CD74_APP, and SPP1_PTGER4 were more significant than those in the other subsets. T-Box transcription factor 2, tropomyosin 4, plexin D1 (PLXND1), G protein subunit α I2 (GNAI2) and SEC14-like lipid binding 1 were identified as the feature genes in UM. These marker genes were found to be significantly enriched in pathways such as vasculature development, focal adhesion and cell adhesion molecule binding. Significant correlations were observed between key genes and immune cells as well as immune factors. Relationships were also observed between the expression levels of the key genes and multiple disease-related genes. Knockdown of PLXND1 and GNAI2 expression led to significantly lower viability and gap closure rates of C918 cells. Therefore, the results of the present study uncovered cell communication between endothelial cells and other cell types, identified innovative key genes and provided potential targets of gene therapy in UM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7253fa248cf594e003c05b34bcbb26032ddcd1f" target='_blank'>
              Identification of feature genes and molecular mechanisms involved in cell communication in uveal melanoma through analysis of single‑cell sequencing data
              </a>
            </td>
          <td>
            Ning Lyu, Jiawen Wu, Yiqin Dai, Yidan Fan, Zhaoyuan Lyu, Jiayu Gu, Jingyi Cheng, Jianjiang Xu
          </td>
          <td>2024-08-20</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Strict control over hematopoietic stem cell decision making is essential for healthy life‐long blood production and underpins the origins of hematopoietic diseases. Acute myeloid leukemia (AML) in particular is a devastating hematopoietic malignancy that arises from the clonal evolution of disease‐initiating primitive cells which acquire compounding genetic changes over time and culminate in the generation of leukemic stem cells (LSCs). Understanding the molecular underpinnings of these driver cells throughout their development will be instrumental in the interception of leukemia, the enabling of effective treatment of pre‐leukemic conditions, as well as the development of strategies to target frank AML disease. To this point, a number of precancerous myeloid disorders and age‐related alterations are proving as instructive models to gain insights into the initiation of LSCs. Here, we explore this myeloid dysregulation at the level of post–transcriptional control, where RNA‐binding proteins (RBPs) function as core effectors. Through regulating the interplay of a myriad of RNA metabolic processes, RBPs orchestrate transcript fates to govern gene expression in health and disease. We describe the expanding appreciation of the role of RBPs and their post–transcriptional networks in sustaining healthy hematopoiesis and their dysregulation in the pathogenesis of clonal myeloid disorders and AML, with a particular emphasis on findings described in human stem cells. Lastly, we discuss key breakthroughs that highlight RBPs and post–transcriptional control as actionable targets for precision therapy of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6dddad312b3774cf0fac0198aa7e076e582021" target='_blank'>
              RNA binding protein‐directed control of leukemic stem cell evolution and function
              </a>
            </td>
          <td>
            Pratik Joshi, Ava Keyvani Chahi, Lina Liu, Steven Moreira, Ana Vujovic, Kristin J Hope
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Secondary acute myeloid leukemia (sAML) presents as a complex and multifaceted ensemble of disorders, positioning itself as both a challenge and an intriguing frontier within hematologic oncology. Its origins are diverse, stemming from antecedent hematologic conditions, germline predisposing mutations, or the sequelae of cytotoxic therapies, and its development is driven by intricate genetic and epigenetic modifications. This complexity necessitates a diverse array of therapeutic strategies, each meticulously tailored to address the distinctive challenges sAML introduces. Such strategies require a personalized approach, considering the variegated clinical backgrounds of patients and the inherent intricacies of the disease. Allogeneic stem cell transplantation stands as a cornerstone, offering the potential for curative outcomes. This is complemented by the emergence of innovative treatments such as CPX-351, venetoclax, and glasdegib, which have demonstrated promising results in enhancing prognosis. The evolving landscape of sAML treatment underscores the importance of continued research and innovation in the field, aiming not only to improve patient outcomes but also to deepen our understanding of the disease’s biological underpinnings, thereby illuminating pathways toward more effective and individualized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e411e905c73882bdcd7c22537e611482f6836fd2" target='_blank'>
              Novel insights and therapeutic approaches in secondary AML
              </a>
            </td>
          <td>
            Giovanni Marconi, Michela Rondoni, B. Zannetti, Irene Zacheo, Davide Nappi, Agnese Mattei, S. Rocchi, Francesco Lanza
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a major cause of cancer-related deaths in developed nations. Factors such as unhealthy lifestyle choices, particularly smoking, contribute to the development of this disease. Epigenetic abnormalities greatly affect gene expression and disrupt important cellular signaling pathways that are responsible for the proper growth, regulation, and functioning of cells. In the case of lung cancer, specifically non-small cell lung cancer (NSCLC), CBX3 acts as an epigenetic oncoprotein, promoting the growth and progression of tumors. Numerous studies have shown that CBX3 is overexpressed in NSCLC and is associated with a poor prognosis. It interacts with key oncogenic pathways leading to increased proliferation, reduced apoptosis, and enhanced resistance to therapy. Further research on the mechanisms and functions of CBX3 holds the potential to reveal new insights into the development of the disease and uncover novel therapeutic opportunities. Abstract Epigenetic changes regulate gene expression through histone modifications, chromatin remodeling, and protein translation of these modifications. The PRC1 and PRC2 complexes shape gene repression via histone modifications. Specifically, the CBX protein family aids PRC1 recruitment to chromatin, impacting the progressive multistep process driving chromatin silencing. Among family members, CBX3 is a complex protein involved in aberrant epigenetic mechanisms that drive lung cancer progression. CBX3 promotes lung tumorigenesis by interacting with key pathways such as PI3K/AKT, Ras/KRAS, Wnt/β-catenin, MAPK, Notch, and p53, leading to increased proliferation, inhibition of apoptosis, and enhanced resistance to therapy. Given our current lack of knowledge, additional research is required to uncover the intricate mechanisms underlying CBX3 activity, as well as its involvement in molecular pathways and its potential biomarker evaluation. Specifically, the dissimilar roles of CBX3 could be reexamined to gain a greater insight into lung cancer pathogenesis. This review aims to provide a clear overview of the context-related molecular profile of CBX3, which could be useful for addressing clinical challenges and developing novel targeted therapies based on personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b8cecc66992d19bc95a90f28c7e4981384afb5" target='_blank'>
              Exploring the Role of CBX3 as a Potential Therapeutic Target in Lung Cancer
              </a>
            </td>
          <td>
            Muhammad Aamir Wahab, Nunzio Del Gaudio, Biagio Gargiulo, V. Quagliariello, N. Maurea, A. Nebbioso, L. Altucci, Mariarosaria Conte
          </td>
          <td>2024-08-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Due to the heterogeneity of cancer, precision medicine has been a major challenge for cancer treatment. Determining medication regimens based on patient genotypes has become a research hotspot in cancer genomics. In this study, we aim to identify key biomarkers for targeted therapies based on single nucleotide variants (SNVs) and copy number variants (CNVs) of genes. The experiment is carried out on 7 cancers on the Encyclopedia of Cancer Cell Lines (CCLE) dataset. Considering the high mutability of driver genes which result in abundant mutated samples, the effect of data sparsity can be eliminated to a large extent. Therefore, we focus on discovering the relationship between driver mutation patterns and three measures of drug response, namely area under the curve (AUC), half maximal effective concentration (EC50), and log2-fold change (LFC). First, multiple statistical methods are applied to assess the significance of difference in drug response between sample groups. Next, for each driver gene, we analyze the extent to which its mutations can affect drug response. Based on the results of multiple hypothesis tests and correlation analyses, our main findings include the validation of several known drug response biomarkers such as BRAF, NRAS, MAP2K1, MAP2K2, and CDKN2A, as well as genes with huge potential to infer drug responses. It is worth emphasizing that we identify a list of genes including SALL4, B2M, BAP1, CCDC6, ERBB4, FOXA1, GRIN2A, and PTPRT, whose impact on drug response spans multiple cancers and should be prioritized as key biomarkers for targeted therapies. Furthermore, based on the statistical p-values and correlation coefficients, we construct gene-drug sensitivity maps for cancer drug recommendation. In this work, we show that driver mutation patterns could be used to tailor therapeutics for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa993499162c59f7b76c5c6ffab6d2e07409c25b" target='_blank'>
              Single nucleotide and copy number variants of cancer driver genes inform drug response in multiple cancers
              </a>
            </td>
          <td>
            Zeyuan Wang, Hong Gu, Pan Qin, Jia Wang
          </td>
          <td>2024-07-31</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Fragile X Syndrome (FXS) is a genetic neurodevelopmental disorder closely associated with intellectual disability and autism spectrum disorders. The core of the disease lies in the abnormal expansion of the CGG trinucleotide repeat sequence at the 5′end of the FMR1 gene. When the repetition exceeds 200 times, it causes the silencing of the FMR1 gene, leading to the absence of the encoded Fragile X mental retardation protein 1 (FMRP). Although the detailed mechanism by which the CGG repeat expansion triggers gene silencing is yet to be fully elucidated, it is known that this process does not alter the promoter region or the coding sequence of the FMR1 gene. This discovery provides a scientific basis for the potential reversal of FMR1 gene silencing through interventional approaches, thereby improving the symptoms of FXS. Epigenetics, a mechanism of genetic regulation that does not depend on changes in the DNA sequence, has become a new focus in FXS research by modulating gene expression in a reversible manner. The latest progress in molecular genetics has revealed that epigenetics plays a key role in the pathogenesis and pathophysiological processes of FXS. This article compiles the existing research findings on the role of epigenetics in Fragile X Syndrome (FXS) with the aim of deepening the understanding of the pathogenesis of FXS to identify potential targets for new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64f2a0a0d20e60f40e1927dda3be0710c0759d2d" target='_blank'>
              Epigenetic insights into Fragile X Syndrome
              </a>
            </td>
          <td>
            Liangqun Xie, Huiying Li, MengLiang Xiao, Ningjing Chen, Xiaoxiao Zang, Yingying Liu, Hong Ye, Chaogang Tang
          </td>
          <td>2024-08-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Understanding cancer evolution is fundamental to predicting cancer progression. However, the evolution of paediatric cancers is still under-researched. Large cohorts of patients are required to determine consistent evolutionary trajectories that shed light on key steps in cancer development and reveal underlying biology, especially in rare cancers. Additionally, well annotated clinical data is necessary for determining if evolutionary biomarkers are predictive of patient outcome. Methods We performed detailed evolutionary analysis of 64 paediatric kidney cancers, including 60 Wilms tumours (WT), using DNA microarrays and, in a subset of 30 patients, a WT-specific targeted sequencing assay, to detect copy number alterations (CNA) and mutations, respectively. By analysing multiple tissue samples in the majority of cases we could detect mutation heterogeneity in each tumour. We reconstructed clones across the cohort and described their phylogenetic histories, in addition to detecting mirrored subclonal allelic imbalance. Results Our results highlight pervasive evidence of parallel evolution in WTs affecting CNAs, and CTNNB1 and TP53 mutations. Furthermore, we demonstrate that stromal-type WTs often evolve from a consistent series of events (WT1 mutation, 11p uniparental disomy and CTNNB1 mutation) and we suggest that 19q uniparental disomy is an important ancestral event in both epithelial and diffuse anaplastic WTs. Finally, we propose the total number of evolutionary CNA events as a prognostic biomarker in WTs for event-free survival, particularly in high-risk WT. Conclusions Overall, this study sheds light on the evolution of the most common paediatric kidney cancer and links evolutionary analysis to fundamental clinical and biological questions in a large cohort of WTs. We conclude that histological subtypes of WT are often defined by consistent evolutionary sequences. We present evidence that a key marker of evolvability, namely CNA diversity measured phylogenetically across multiple tumour sites, is prognostic of patient outcome and should be considered for clinical use to detect the most aggressive blastemal and diffuse anaplastic type WTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24a5fbaf8b2a5b212183cb8a0bf89d97d1b5f3b" target='_blank'>
              Subtype-specific Patterns of Evolution and Clinically Relevant Genomic Instability in Wilms Tumour
              </a>
            </td>
          <td>
            George D. Cresswell, T. Chagtai, Reem Al-Saadi, T. Treger, G. Madhan, Borbala Mifsud, Gordan M. Vujanic, R. Williams, N. Luscombe, Kathy Pritchard-Jones, William Mifsud
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="New high-quality human genome assemblies derived from lymphoblastoid cell lines (LCLs) provide reference genomes and pangenomes for genomics studies. However, the characteristics of LCLs pose technical challenges to profiling immunoglobulin (IG) genes. IG loci in LCLs contain a mixture of germline and somatically recombined haplotypes, making them difficult to genotype or assemble accurately. To address these challenges, we introduce IGLoo, a software tool that implements novel methods for analyzing sequence data and genome assemblies derived from LCLs. IGLoo characterizes somatic V(D)J recombination events in the sequence data and identifies the breakpoints and missing IG genes in the LCL-based assemblies. Furthermore, IGLoo implements a novel reassembly framework to improve germline assembly quality by integrating information about somatic events and population structural variantions in the IG loci. We applied IGLoo to study the assemblies from the Human Pangenome Reference Consortium, providing new insights into the mechanisms, gene usage, and patterns of V(D)J recombination, causes of assembly fragmentation in the IG heavy chain (IGH) locus, and improved representation of the IGH assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8f48cf5eb22832827a1cb138fb7f409be2963b7" target='_blank'>
              IGLoo: Profiling the Immunoglobulin Heavy chain locus in Lymphoblastoid Cell Lines with PacBio High-Fidelity Sequencing reads
              </a>
            </td>
          <td>
            Mao-Jan Lin, Ben Langmead, Yana Safonova
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Cancer cell dormancy (CCD) in colorectal cancer (CRC) poses a significant challenge to effective treatment. In CRC, CCD contributes to tumour recurrence, drug resistance, and amplifying the disease's burden. The molecular mechanisms governing CCD and strategies for eliminating dormant cancer cells remain largely unexplored. Therefore, understanding the molecular mechanisms governing dormancy is crucial for improving patient outcomes and developing targeted therapies. This editorial highlights the complex interplay of signalling pathways and factors involved in colorectal CCD, emphasizing the roles of Hippo/YAP, pluripotent transcription factors such as NANOG, HIF-1α signalling, and Notch signalling pathways. Additionally, ERK/p38α/β/MAPK pathways, AKT signalling pathway, and Extracellular Matrix Metalloproteinase Inducer, along with some potential less explored pathways such as STAT/p53 switch and canonical and non-canonical Wnt and SMAD signalling, are also involved in promoting colorectal CCD. Highlighting their clinical significance, these findings may offer the potential for identifying key dormancy regulator pathways, improving treatment strategies, surmounting drug resistance, and advancing personalized medicine approaches. Moreover, insights into dormancy mechanisms could lead to the development of predictive biomarkers for identifying patients at risk of recurrence and the tailoring of targeted therapies based on individual dormancy profiles. It is essential to conduct further research into these pathways and their modulation to fully comprehend CRC dormancy mechanisms and enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5a1fd8a1c24adda33f9d39ee28ef2dc82663eff" target='_blank'>
              Colorectal cancer cell dormancy: An insight into pathways
              </a>
            </td>
          <td>
            Anil Kumar, L. Saha
          </td>
          <td>2024-09-07</td>
          <td>World Journal of Gastroenterology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND/AIM
There are no established biomarkers for immune checkpoint inhibitors (ICI) in colorectal cancer (CRC) with microsatellite stability (MSS) or proficient mismatch repair (pMMR). Therefore, this study aimed to identify biomarkers for ICI benefit in patients with pMMR by analyzing the down-regulated DNA repair-related genes involved in highly immunogenic and immune responses, and comparing their expression levels and clinical features.


MATERIALS AND METHODS
Mismatch repair (MMR), tumor-infiltrating lymphocytes (TIL), and tumor mutation burden (TMB) were evaluated in 13 CRC cases and mRNA expression levels of 95 DNA repair-related genes were measured. DNA repair-related genes with reduced mRNA expression in the high immunogenicity and high immune response groups were identified. Then, the mRNA expression levels of the identified DNA repair-related genes were measured in 135 patients with CRC. Hierarchical cluster analysis was performed using the mRNA expression levels to compare the clinicopathological characteristics of each cluster.


RESULTS
ATR, LIG4, and RAD52 mRNA levels were significantly down-regulated in the high immunogenicity group. GADD45B, SMUG1, and XRCC6 mRNA levels were significantly down-regulated in the high immune response group. Cases in the cluster with reduced mRNA expression of the six genes were pMMR cases. CD8 mRNA expression level was higher in this cluster than in the other clusters.


CONCLUSION
Decreased mRNA expression levels of ATR, LIG4, RAD52, GADD45B, SMUG1, and XRCC6 genes were associated with high cytotoxic T cell and TMB levels, suggesting that these genes could serve as biomarkers for ICI efficacy in pMMR cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb38d024ab94655fe817bc6cd4f0bb4ad2d969e2" target='_blank'>
              Identification of DNA Repair-related Genes as Biomarkers Reflecting MSI, TMB, and TIL in Colorectal Cancer.
              </a>
            </td>
          <td>
            Junya Fukuda, Tomoya Sudo, Maako Kikuchi, Akihiko Kawahara, Hirona Shigyo, Yusuke Kawamoto, Satoshi Shimamura, Fumiki Koga, Takasi Noguchi, Hiroyuki Nakane, Takahiro Shigaki, Kenji Fujiyoshi, T. Yomoda, Naohiro Yoshida, Kenichi Koushi, Takefumi Yoshida, Jun Akiba, Fumihiko Fujita
          </td>
          <td>2024-07-26</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Biallelic pathogenic variants in the essential DNA repair gene BRCA2 causes Fanconi anemia, complementation group FA-D1. Patients in this group are highly prone to develop embryonal tumors, most commonly medulloblastoma arising from the cerebellar granule cell progenitors (GCPs). GCPs undergo high proliferation in the postnatal cerebellum under SHH activation, but the type of DNA lesions that require the function of the BRCA2 to prevent tumorigenesis remains unknown. To identify such lesions, we assessed both GCP neurodevelopment and tumor formation using a mouse model with deletion of exons three and four of Brca2 in the central nervous system, coupled with global Trp53 loss. Brca2Δex3-4;Trp53-/- animals developed SHH subgroup medulloblastomas with complete penetrance. Whole-genome sequencing of the tumors identified structural variants with breakpoints enriched in areas overlapping G-quadruplexes (G4s). Brca2-deficient GCPs exhibited decreased replication speed in the presence of the G4-stabilizer pyridostatin. Pif1 helicase, which resolves G4s during replication, was highly upregulated in tumors, and Pif1 knockout in primary MB tumor cells resulted in increased genome instability upon pyridostatin treatment. These data suggest that G4s may represent sites prone to replication stalling in highly proliferative GCPs and without BRCA2, G4s become a source of genome instability. Tumor cells upregulate G4-resolving helicases to facilitate rapid proliferation through G4s highlighting PIF1 helicase as a potential therapeutic target for treatment of BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d543ffa68779aadc1c04b11411614442736e5f08" target='_blank'>
              G-quadruplexes are a source of vulnerability in BRCA2 deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, Carolina Sandoval-Garcia, Margaret L. MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1fa9923fb71606ea55e83b6b8cbb7a9e56b764" target='_blank'>
              Competition shapes the landscape of X-chromosome-linked genetic diversity
              </a>
            </td>
          <td>
            Teresa Buenaventura, Hakan Bagci, Ilinca Patrascan, Joshua J Graham, Kelsey D Hipwell, Roel Oldenkamp, James W D King, Jesus Urtasun, G. Young, Daniel Mouzo, D. Gómez-Cabrero, B. Rowland, Daniel Panne, Amanda G Fisher, Matthias Merkenschlager
          </td>
          <td>2024-07-26</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML), an aggressive malignancy of hematopoietic stem cells, is characterized by the blockade of cell differentiation, uncontrolled proliferation, and cell expansion that impairs healthy hematopoiesis and results in pancytopenia and susceptibility to infections. Several genetic and chromosomal aberrations play a role in AML and influence patient outcomes. TP53 is a key tumor suppressor gene involved in a variety of cell features, such as cell-cycle regulation, genome stability, proliferation, differentiation, stem-cell homeostasis, apoptosis, metabolism, senescence, and the repair of DNA damage in response to cellular stress. In AML, TP53 alterations occur in 5%–12% of de novo AML cases. These mutations form an important molecular subgroup, and patients with these mutations have the worst prognosis and shortest overall survival among patients with AML, even when treated with aggressive chemotherapy and allogeneic stem cell transplant. The frequency of TP53-mutations increases in relapsed and recurrent AML and is associated with chemoresistance. Progress in AML genetics and biology has brought the novel therapies, however, the clinical benefit of these agents for patients whose disease is driven by TP53 mutations remains largely unexplored. This review focuses on the molecular characteristics of TP53-mutated disease; the impact of TP53 on selected hallmarks of leukemia, particularly metabolic rewiring and immune evasion, the clinical importance of TP53 mutations; and the current progress in the development of preclinical and clinical therapeutic strategies to treat TP53-mutated disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/442da7a2ffb734ac40ab31ab13428c2a4af47cbd" target='_blank'>
              Impact of p53-associated acute myeloid leukemia hallmarks on metabolism and the immune environment
              </a>
            </td>
          <td>
            Monika Chomczyk, Luca Gazzola, Shubhankar Dash, Patryk Firmanty, Binsah S. George, V. Mohanty, Hussein A. Abbas, Natalia Baran
          </td>
          <td>2024-08-05</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Global healthcare systems have a great challenge in the form of inherited hematological diseases, which necessitates the development of new remedial strategies. By precisely targeting inherited abnormalities, Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-mediated ex vivo gene editing has surfaced as a promising approach to treat these diseases. This review offers a comprehensive examination of the advancements, challenges, and clinical eventuality of CRISPR-Cas9-intermediated ex vivo gene editing for inherited hematological diseases. With advancements in CRISPR-Cas9 technology, the eventuality to correct inheritable mutations responsible for inherited hematological diseases is within reach. However, challenges such as off-target effects, immune responses, and ethical considerations need to be addressed for the safe and effective perpetration of this technology. A promising understanding of how CRISPR-Cas9-intermediated gene editing functions in practice is handed by ongoing clinical studies, giving rise to the possibility of advanced remedial approaches and bettered patient issues. By addressing these complications in a human-readable format, this review attempts to provide greater understanding and appreciation for the eventuality of CRISPR-Cas9 technology in revolutionizing the treatment landscape for these challenging disorders and contribute to the ongoing discussion in the field and facilitate further exploration towards effective treatments for these challenging disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4d496cce2c3ddc8c2b97b6be801d229f168c15" target='_blank'>
              CRISPR-CAS9-MEDIATED EX VIVO GENE EDITING FOR INHERITED HEMATOLOGICAL DISORDERS: ADVANCEMENTS, CHALLENGES, AND CLINICAL POTENTIAL
              </a>
            </td>
          <td>
            Atasi Ranjan Panda, Shreeya Das
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Pharmacy and Pharmaceutical Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Alveolar rhabdomyosarcoma (ARMS) patients harboring PAX3-FOXO1 and PAX7-FOXO1 fusion proteins exhibit a greater incidence of tumor relapse, metastasis, and poor survival outcome, thereby underscoring the urgent need to develop effective therapies to treat this subtype of childhood cancer. To uncover mechanisms that contribute to tumor initiation, we developed a novel muscle progenitor model and used epigenomic approaches to unravel genome re-wiring events mediated by PAX3/7 fusion proteins. Importantly, these regulatory mechanisms are conserved across established ARMS cell lines, primary tumors, and orthotopic-patient derived xenografts. Among the key targets of PAX3- and PAX7-fusion proteins, we identified a cohort of oncogenes, FGF receptors, and genes essential for mitochondrial metabolism and protein translation, which we successfully targeted in preclinical trials. Our data suggest an explanation for the relative paucity of recurring mutations in this tumor, provide a compelling list of actionable targets, and suggest promising new strategies to treat this tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f840f071bc2696a99f7c83f33f72c0a963e3308" target='_blank'>
              PAX fusion proteins deregulate gene networks controlling mitochondrial translation in pediatric rhabdomyosarcoma
              </a>
            </td>
          <td>
            Bhargab Kalita, Gerard Martinez-Cebrian, Justina McEvoy, Melody Allensworth, Michelle Knight, Alessandro Magli, Rita C. R. Perlingeiro, Michael A. Dyer, Elizabeth Stewart, B. Dynlacht
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Advances in melanoma research have unveiled critical insights into its genetic and molecular landscape, leading to significant therapeutic innovations. This review explores the intricate interplay between genetic alterations, such as mutations in BRAF, NRAS, and KIT, and melanoma pathogenesis. The MAPK and PI3K/Akt/mTOR signaling pathways are highlighted for their roles in tumor growth and resistance mechanisms. Additionally, this review delves into the impact of epigenetic modifications, including DNA methylation and histone changes, on melanoma progression. The tumor microenvironment, characterized by immune cells, stromal cells, and soluble factors, plays a pivotal role in modulating tumor behavior and treatment responses. Emerging technologies like single-cell sequencing, CRISPR-Cas9, and AI-driven diagnostics are transforming melanoma research, offering precise and personalized approaches to treatment. Immunotherapy, particularly immune checkpoint inhibitors and personalized mRNA vaccines, has revolutionized melanoma therapy by enhancing the body’s immune response. Despite these advances, resistance mechanisms remain a challenge, underscoring the need for combined therapies and ongoing research to achieve durable therapeutic responses. This comprehensive overview aims to highlight the current state of melanoma research and the transformative impacts of these advancements on clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32ea057809622a436f9ea97ccea29c8077c2cd13" target='_blank'>
              Advances in Melanoma: From Genetic Insights to Therapeutic Innovations
              </a>
            </td>
          <td>
            Fernando Valdez-Salazar, Luis A. Jiménez-Del Rio, J. Padilla-Gutiérrez, Y. Valle, J. Muñóz-Valle, Emmanuel Valdés-Alvarado
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="INTRODUCTION
Advances in understanding of tumor biology shed light on hallmarks of cancer development and progression that include dysregulated DNA damage repair (DDR) machinery. Leveraging underlying tumor genomic instability and tumor specific defects in DDR, Poly (ADP-ribose) polymerases (PARP) inhibitors (PARPi) induced DNA damage emerges as a novel non-chemotherapy therapeutic opportunity. PARPis are currently approved in multiple tumor types, with the largest benefit seen in tumors with homologous recombination repair (HRR) deficiency, including germline and somatic mutations in BRCA1/2 genes (BRCA) and other pathway members such as PALB2 and Rad51c.


AREAS COVERED
This review article summarizes the current approval landscape and known and proposed mechanisms of resistance to PARPi. Further, therapeutic strategies to overcome PARPi resistance are discussed, including ongoing clinical trials.


EXPERT OPINION
PARPi have proven to be a safe and effective therapy and represent a cornerstone treatment across multiple solid tumor types. Elucidating innate and acquired mechanisms of resistance, coupled with the emergence of novel therapeutic options to capitalize on the activity of PARPi and prevent or reverse the acquisition of resistance, provides an opportunity to further expand the role of PARPi in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec7393511b29f158c45a3ee7ca1c8ffe71c659b" target='_blank'>
              Strategies for the prevention or reversal of PARP inhibitor resistance.
              </a>
            </td>
          <td>
            Zahi I. Mitri, Shaun M Goodyear, Gordon B. Mills
          </td>
          <td>2024-08-15</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Richter’s transformation (RT) is a progression of chronic lymphocytic leukemia (CLL) to aggressive lymphoma. MGA (Max gene associated), a functional MYC suppressor, is mutated at 3% in CLL and 36% in RT. However, genetic models and molecular mechanisms of MGA deletion that drive CLL to RT remain elusive. We established an RT mouse model by knockout of Mga in the Sf3b1/Mdr CLL model using CRISPR-Cas9 to determine the role of Mga in RT. Murine RT cells exhibited mitochondrial aberrations with elevated oxidative phosphorylation (OXPHOS). Through RNA sequencing and functional characterization, we identified Nme1 (nucleoside diphosphate kinase) as an Mga target, which drives RT by modulating OXPHOS. Given that NME1 is also a known MYC target without targetable compounds, we found that concurrent inhibition of MYC and electron transport chain complex II substantially prolongs the survival of RT mice in vivo. Our results suggest that the Mga-Nme1 axis drives murine CLL-to-RT transition via modulating OXPHOS, highlighting a potential therapeutic avenue for RT. A MGA/NME1 regulatory axis can promote CLL-to-RT transition by modulating OXPHOS, highlighting this pathway as a target for RT treatment. Editor’s summary Richter’s transformation (RT) involves the progression of chronic lymphocytic leukemia (CLL) to a more aggressive lymphoma with poor outcomes. Understanding the transition into RT is key to more targeted therapeutic strategies. Iyer et al. examined the role of the MYC suppressor MGA, which is differentially expressed in RT, in an in vivo CLL model with MGA dysfunction. MGA loss resulted in increased mitochondrial function and reactive oxygen species and led to aberrant cell morphology and up-regulated MYC as commonly observed in patients. Combined inhibition of MYC and mitochondrial respiration prolonged in vivo survival in this model, highlighting potential therapeutic strategies for RT with this genetic profile. —Catherine Charneski">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75d1c8fe0890ca3dad978b633b25c5e7b5178f59" target='_blank'>
              MGA deletion leads to Richter’s transformation by modulating mitochondrial OXPHOS
              </a>
            </td>
          <td>
            Prajish Iyer, Bo Zhang, Tingting Liu, Meiling Jin, Kevyn L Hart, Jibin Zhang, Viola Siegert, Marianne Remke, Xuesong Wang, Lei Yu, Joo Y Song, Girish Venkataraman, Wing C. Chan, Zhenyu Jia, M. Buchner, Tanya Siddiqi, Steven T. Rosen, Alexey Danilov, Lili Wang
          </td>
          <td>2024-07-31</td>
          <td>Science Translational Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Carcinogenesis is an evolutionary process, and mutations can fix the selected phenotypes in selective microenvironments. Both normal and neoplastic cells are robust to the mutational stressors in the microenvironment to the extent that secure their fitness. To test the robustness of genes under a range of mutagens, we developed a sequential mutation simulator, Sinabro, to simulate single base substitution under a given mutational process. Then, we developed a pipeline to measure the robustness of genes and cells under those mutagenesis processes. We discovered significant human genome robustness to the APOBEC mutational signature SBS2, which is associated with viral defense mechanisms and is implicated in cancer. Robustness evaluations across over 70,000 sequences against 41 signatures showed higher resilience under signatures predominantly causing C-to-T (G-to-A) mutations. Principal component analysis indicates the GC content at the codon’s wobble position significantly influences robustness, with increased resilience noted under transition mutations compared to transversions. Then, we tested our results in bats at extremes of the lifespan-to-mass relationship and found the long-lived bat is more robust to APOBEC than the short-lived one. By revealing APOBEC as the prime driver of robustness in the human (and other mammalian) genome, this work bolsters the key potential role of APOBECs in carcinogenesis, as well as evolved countermeasures to this innate mutagenic process. It also provides the baseline of the human and bat genome robustness under mutational processes associated with cancer. Highlights Sinabro, the sequential mutation simulator, facilitates measuring the robustness of human protein-coding sequences under all COSMIC mutational signatures. Robustness under APOBEC mutational signatures showed the largest mean and standard deviation in the human genome. Robustness to mutational signatures analysis reveals the role of APOBECs is complementary to cancer in the evolvability of cancer cells in later stages. Principal component analysis indicates that the GC content at the codon’s wobble position significantly influences robustness. A long-lived bat (Myotis myotis) has higher robustness to APOBECs than a short-lived one (Molossus molossus) than humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/011df1029d192af53d074e293c75b5f6a3b83940" target='_blank'>
              Human and bats genome robustness under COSMIC mutational signatures
              </a>
            </td>
          <td>
            Joon-Hyun Song, Ying Zeng, Liliana M. Dávalos, Thomas MacCarthy, Mani Larijani, Mehdi Damaghi
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/311bc754350204df7366f1cc87322a6aee4bc5a6" target='_blank'>
              Assessment of SWI/SNF chromatin remodeling complex related genes as potential biomarkers and therapeutic targets in pan-cancer
              </a>
            </td>
          <td>
            Kai Zhuang, Lishan Wang, Chengyu Lu, Zhiping Liu, Dongli Yang, Hao Zhong, Jiami Zou, Aamir Fahira, Jiaojiao Wang, Zunnan Huang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Pediatric cancers are frequently driven by genomic alterations that result in aberrant transcription factor activity. While direct targeting of transcription factors is difficult, one method to circumvent this problem is to target co-factors and chromatin complexes that are necessary for their oncogenic function. To evaluate protein complex dependencies enriched in MYCN-amplified neuroblastoma, a disease of dysregulated development driven by aberrant expression of an oncogenic expression program, we curated a list of protein complexes using the CORUM database and mined the Dependency Map using single sample gene set enrichment analysis. This analysis identified the KAT module of the transcriptional coactivator Spt-Ada-Gcn5-acetyltransferase (SAGA) complex as a selective dependency in MYCN-amplified neuroblastoma. Loss of SAGA complex activity in neuroblastoma through both genetic knockout and induced protein degradation of the lysine acetyltransferase (KAT) module subunit TADA2B resulted in a global reduction of histone lysine acetylation and impaired neuroblastoma cell growth. Genetic knockout of TADA2B did not impair the growth of pediatric sarcoma cell lines, suggesting that SAGA loss-of-function is not broadly deleterious. Furthermore, SAGA loss-of-function reduced MYCN binding to chromatin, resulting in suppression of the MYCN-driven gene expression program and impaired cell cycle progression. Importantly, the SAGA complex is pharmacologically targetable in vitro and in vivo with GSK-699, a PROTAC that induces proteolysis of both KAT2A and KAT2B paralogs. As observed with genetic knockout studies, loss of SAGA activity through KAT2A/B degradation reduced the MYCN-driven transcriptional signature and impaired cell cycle progression. Our findings expand our understanding of the chromatin complexes that maintain the oncogenic transcriptional state in MYCN-amplified neuroblastoma and suggest therapeutic potential for inhibitors of SAGA KAT activity in MYCN-amplified neuroblastoma.
 Citation Format: Nathaniel W. Mabe, Clare F. Malone, Alexandra B. Forman, Gabriela Alexe, Kathleen L. Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar, Bin Liu, Sharon YR Dent, Kimberly Stegmaier. Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be30c933e33904854e0410865fe3ee874fab512e" target='_blank'>
              Abstract A062 Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program
              </a>
            </td>
          <td>
            Nathaniel W. Mabe, Clare F Malone, Alexandra B Forman, G. Alexe, Kathleen Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar, Bin Liu, S. Y. Dent, K. Stegmaier
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The killer-cell immunoglobulin-like receptor (KIR) gene complex, a highly polymorphic region of the human genome that encodes proteins involved in immune responses, poses strong challenges in genotyping owing to its remarkable genetic diversity and structural intricacy. Accurate analysis of KIR alleles, including their structural variations, is crucial for understanding their roles in various immune responses. Leveraging the high-quality genome assemblies from the Human Pangenome Reference Consortium (HPRC), we present a novel bioinformatic tool, the structural KIR annoTator (SKIRT), to investigate gene diversity and facilitate precise KIR allele analysis. In 47 HPRC-phased assemblies, SKIRT identifies a recurrent novel KIR2DS4/3DL1 fusion gene in the paternal haplotype of HG02630 and maternal haplotype of NA19240. Additionally, SKIRT accurately identifies eight structural variants and 15 novel nonsynonymous alleles, all of which are independently validated using short-read data or quantitative polymerase chain reaction. Our study has discovered a total of 570 novel alleles, among which eight haplotypes harbor at least one KIR gene duplication, six haplotypes have lost at least one framework gene, and 75 out of 94 haplotypes (79.8%) carry at least five novel alleles, thus confirming KIR genetic diversity. These findings are pivotal in providing insights into KIR gene diversity and serve as a solid foundation for understanding the functional consequences of KIR structural variations. High-resolution genome assemblies offer unprecedented opportunities to explore polymorphic regions that are challenging to investigate using short-read sequencing methods. The SKIRT pipeline emerges as a highly efficient tool, enabling the comprehensive detection of the complete spectrum of KIR alleles within human genome assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c02692297be90da469598251c3027cba4d2d27ff" target='_blank'>
              Genetic complexity of killer-cell immunoglobulin-like receptor genes in human pangenome assemblies.
              </a>
            </td>
          <td>
            Tsung-Kai Hung, Wan-Chi Liu, Sheng-Kai Lai, Hui-Wen Chuang, Yi-Che Lee, Hong-Ye Lin, Chia-Lang Hsu, Chien-Yu Chen, Ya-Chien Yang, Jacob Shu-Jui Hsu, Pei-Lung Chen
          </td>
          <td>2024-09-09</td>
          <td>Genome research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfda792d36cb0c85bc9dd50884926690c096c37e" target='_blank'>
              RNA polymerase stalling-derived genome instability underlies ribosomal antibiotic efficacy and resistance evolution
              </a>
            </td>
          <td>
            Yayun Zheng, Ruochen Chai, Tian Wang, Zeqi Xu, Yihui He, P. Shen, Jintao Liu
          </td>
          <td>2024-08-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eddab72a32f5e67003082a5b17959fbc71f348a1" target='_blank'>
              Gene therapy for polygenic or complex diseases
              </a>
            </td>
          <td>
            Tingting Wu, Yujun Hu, 
          </td>
          <td>2024-09-04</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7a9a7a504580b14d647edcfb85c539f3ac48" target='_blank'>
              Enhanced depletion of MLL-fusion proteins in acute leukemia: potential for improved therapeutic outcomes
              </a>
            </td>
          <td>
            Noelia Che, S. Cantilena, R. Looi-Somoye, Danesh Sundar, Kent Fung, J. de Boer, Owen Williams
          </td>
          <td>2024-08-16</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved DNA repair system, in which aberrations can lead to the accumulation of DNA damage and genomic scars known as homologous recombination deficiency (HRD). The identification of mutations in key genes (i.e., BRCA1, and BRCA2 (BRCA)) and the quantification of large-scale structural variants (e.g., loss of heterozygosity) are indicators of the HRD phenotype. HRD is a stable biomarker and remains unchanged during recurrence, but fails to reveal the molecular profile of tumor progression. Moreover, interpretation of the current HRD score lacks comprehensiveness, especially for the HR-proficient group. Poly (ADP-ribose) polymerase (PARP) enzymes play an important role in the repair of DNA single-strand breaks, the blockage of which using PARP inhibitors (PARPi) can generate synthetic lethality in cancer cells with HRD. Although numerous studies have demonstrated that the benefit of PARPi is substantial in ovarian cancer (OC) patients, the efficacy is limited by the development of resistance, and seems to be irrespective of HR and/or BRCA mutation status. Moreover, in addition to improving progression-free survival, long-term benefit as overall survival brought by PARPi for advanced, recurrent and refractory OC patients remains unclear. Therefore, further investigations are needed to uncover the role of HR genes beyond BRCA and their interactions with other oncogenic pathways, to determine the value of HRD in the recurrent setting, and to identify alternative strategies for the precise management of advanced, refractory OC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fe4c9f0bc408e017d25339a0121e8f087ab765" target='_blank'>
              Limitations of homologous recombination status testing and poly (ADP-ribose) polymerase inhibitor treatment in the current management of ovarian cancer
              </a>
            </td>
          <td>
            Qianying Zhao, Liping Bai, Yu Tan, Mingrong Qie
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="What Is the Issue? 
The first therapeutics based on clustered regularly interspaced short palindromic repeats (CRISPR) technologies are entering the market. These gene editing technologies have the potential to change treatment paradigms and may be used to treat conditions that cannot be treated or cured with current methods. This report aims to provide an overview of the technologies and their current and potential roles in health care. 
What Are the Technologies? 
CRISPR is a part of bacterial immune systems that can cut DNA strands and is used as a gene editing tool. A guide ribonucleic acid (RNA) sequence leads the CRISPR-associated nuclease to the target DNA sequence where the cut is made. These edits change the function of the gene, making genes nonfunctional or replacing the coding sequence for 1 gene with another. CRISPR can also be used to increase or decrease the expression of specific genes. 
What Is the Potential Impact? 
CRISPR-based technologies have a variety of potential applications in health care, including: 

treating genetic diseases 
understanding the genetic mechanisms of diseases and investigating the relevance of potential drug treatments 
managing infectious diseases through detection, treatment, and elimination. 

What Else Do We Need to Know? 
Ethical issues pertinent to the use of CRISPR include the ability to obtain adequately informed consent, the potential future consequences of gene editing and its potential unintended effects, and the impact gene editing could have on future generations. The long-term effects of CRISPR-based therapies are currently unknown. Further research into emerging applications is required. Long-term follow-up of the patients who have received the first CRISPR-based therapeutics will help inform understanding of the safety and effectiveness of these treatments. While the first of these therapies have been granted regulatory authorization, the next viable CRISPR-based therapies are still in the early phases of development, with the pivotal clinical trials not expected to be completed until at least 2027.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0814cd12a7b72cf57aa46663d5db2b368ed9efd" target='_blank'>
              CRISPR Technologies for In Vivo and Ex Vivo Gene Editing
              </a>
            </td>
          <td>
            Cadth Horizon, Scan
          </td>
          <td>2024-07-19</td>
          <td>Canadian Journal of Health Technologies</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="137 Background: Resistance to RT remains a clinical challenge, given our limited understanding of the molecular pathways underpinning radioresistance. Here, we investigated for mechanisms linked to RR using paired experimental and clinico-molecular HNC and PCa datasets for discovery and validation of novel pathways that may be enriched in RR cancers. Methods: For experimental data, we generated RR HNC (FaDu, HK1) and PCa (22Rv1, DU145) cell lines following high-dose X-irradiation (90 Gy in 45 fr). Genotypic characterization of RR and wildtype (WT) cells was performed by genomic and transcriptomic sequencing using WES (100X) and RNAseq (Illumina, CA), respectively, followed by functional characterization by western blot (WB) and immunofluorescence (IF). For clinico-molecular data, we utilized two prospectively recruited RT cohorts, consisting of 311 patients with HNC (n=158) and PCa (n=153). Tumor transcriptomes were profiled using RNAseq and Affymetrix ST array (ThermoFisher, CA), respectively. Comparative analyses of molecular profiles were first performed between RR and WT in vitro models to identify dysregulated pathways that were linked to radioresistance, and subsequently tested for association with disease-free survival (DFS) in the clinical cohorts. Results: Comparative genomic analyses between RR and WT HNC and PCa models revealed an abundance of acquired mutations in the RR compared with WT models, with a higher mutation count observed in PCa than HNC cells (SNV counts: 2,158 [DU145-RR] and 1,387 [22Rv1-RR] vs 396 [FaDu-RR] and 25 [HK1-RR], P<0.0001). Mutational signature analyses indicated a common enrichment of DNA mismatch repair-related mutational signatures (SBS3, SBS14, and SBS21) across the RR HNC and PCa models. Transcriptomic profiling revealed an upregulation of the BAHD1 gene, which is involved in heterochromatin formation, across the 4 RR cell lines, corroborated by an enrichment of heterochromatin-related genesets. These results were consistent with functional characterization of RR versus WT cells by WB and IF indicating increased DNA repair capacity and heterochromatin response post-4 Gy irradiation. We further confirmed the dependency of the RR phenotype on the heterochromatin response with BAHD1 knockdown, resulting in reversal of these cellular responses. Finally, our results were supported by survival analyses indicating an inferior DFS post-RT in patients with HNC and PCa harboring a higher expression of heterochromatin-related geneset (HNC: HRhigh vs low 1.51 ; PCa: HRhigh vs low 1.37 ). Conclusions: Herein, by leveraging on paired experimental and clinico-molecular datasets, we have uncovered a novel BAHD1-dependent heterochromatin response that underpins resistance to RT in HNC and PCa, which may be amendable to systemic agents targeting chromatin remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea1270e10cc26b39aabc2ffd1632147257d9664" target='_blank'>
              Analysis of heterochromatin remodeling as a mechanism of radiotherapy (RT) resistance through molecular profiling of radioresistant (RR) head and neck (HNC) and prostate cancers (PCa).
              </a>
            </td>
          <td>
            Evelyn Tan, Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, Dennis J J Poon, Dewi Susanti, E. Yeo, Pek L Chu, Melvin L K Chua
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="microRNAs (miRNAs), tiny, non-coding RNA molecules, fine-tune the expression of target genes through interacting with mRNAs. These miRNAs are involved in a wide range of biological processes, encompassing cell division, death, blood cell production, and tumor development. When these miRNAs become dysfunctional, they can promote the invasion and spread of cancer cells in various human malignancies, including leukemia. Acute lymphoblastic leukemia (ALL), the preeminent malignancy affecting children, is a blood cancer marked by the uncontrollable growth of immature lymphoid cells that displace healthy blood precursors in the bone marrow. Despite a decline in ALL mortality rates over the past two decades, a significant proportion of deaths still results from a lack of effective diagnostic and prognostic markers that can guide treatment decisions and overcome drug resistance. The analysis of miRNA expression patterns in ALL could lead to more precise disease classification, earlier diagnosis, and better prognostic outcomes in the near future. The connection between miRNA dysfunction and the biology of ALL suggests that these molecules could represent promising therapeutic targets. Therefore, this review delves into the regulatory mechanisms of miRNAs in pediatric ALL, exploring how miRNA-based diagnostic, prognostic, and therapeutic strategies offer unique advantages and hold promise for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b51d3730adcae846d16d92db76f51f153c5c1d0" target='_blank'>
              Advancements in the Regulatory Role of microRNAs in Childhood Acute Lymphoblastic Leukemia: Mechanisms and Clinical Implications
              </a>
            </td>
          <td>
            Wei Deng
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd083194686b61ccc53ca0d4c394b67971b4cdb7" target='_blank'>
              Targeted gene therapy for cancer: the impact of microRNA multipotentiality
              </a>
            </td>
          <td>
            Nourhan A. Abou Madawi, Zeinab E Darwish, Enas M Omar
          </td>
          <td>2024-08-01</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ed5339b18c8cb661ce34053233bc7343f92ee3a" target='_blank'>
              1q jumping translocation as a biomarker in myeloid malignancy: frequently mutated genes associated with bad prognosis and low survival
              </a>
            </td>
          <td>
            Eitan Halper-Stromberg, V. Stinnett, L. Morsberger, Aparna Pallavajjala, M. Levis, A. DeZern, Michelle Lei, Brian Phan, Rena R. Xian, Christopher D. Gocke, Guilin Tang, Ying S Zou
          </td>
          <td>2024-08-01</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8eb55432a81ae9f11fc653e3722684f5e8a7ed1" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The detection of mitochondrial DNA (mtDNA) mutations in single cells holds considerable potential to define clonal relationships coupled with information on cell state in humans. Previous methods focused on higher heteroplasmy mutations that are limited in number and can be influenced by functional selection, introducing biases for lineage tracing. Although more challenging to detect, intermediate to low heteroplasmy mtDNA mutations are valuable due to their high diversity, abundance, and lower propensity to selection. To enhance mtDNA mutation detection and facilitate fine-scale lineage tracing, we developed the single-cell Regulatory multi-omics with Deep Mitochondrial mutation profiling (ReDeeM) approach, an integrated experimental and computational framework. Recently, some concerns have been raised about the analytical workflow in the ReDeeM framework. Specifically, it was noted that the mutations detected in a single molecule per cell are enriched on edges of mtDNA molecules, suggesting they resemble artifacts reported in other sequencing approaches. It was then proposed that all mutations found in one molecule per cell should be removed. We detail our error correction method, demonstrating that the observed edge mutations are distinct from previously reported sequencing artifacts. We further show that the proposed removal leads to massive elimination of bona fide and informative mutations. Indeed, mutations accumulating on edges impact a minority of all mutation calls (for example, in hematopoietic stem cells, the excess mutations on the edge account for only 4.3%-7.6% of the total). Recognizing the value of addressing edge mutations even after applying consensus correction, we provide an additional filtering option in the ReDeeM-R package. This approach effectively eliminates the position biases, leads to a mutational signature indistinguishable from bona fide mitochondrial mutations, and removes excess low molecule high connectedness mutations. Importantly, this option preserves the large majority of unique mutations identified by ReDeeM, maintaining the ability of ReDeeM to provide a more than 10-fold increase in variant detection compared to previous methods. Additionally, the cells remain well-connected. While there is room for further refinement in mutation calling strategies, the significant advances and biological insights provided by the ReDeeM framework are unique and remain intact. We hope that this detailed discussion and analysis enables the community to employ this approach and contribute to its further development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94914bd4d265668ea63fab9b4a1aa10d1231c91c" target='_blank'>
              Robustness and reliability of single-cell regulatory multi-omics with deep mitochondrial mutation profiling
              </a>
            </td>
          <td>
            Chen Weng, Jonathan S. Weissman, V. Sankaran
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Ganciclovir (GCV) is a clinically important drug as it is used to treat viral infections. GCV is incorporated into the DNA during replication, where it interferes with subsequent replication on GCV-incorporated templates. However, the effects of GCV on the host genome and the mechanisms underlying cellular tolerance to GCV remain unclear. In this study, we explored these mechanisms using a collection of mutant DT40 cells. We identified RAD17/-, BRCA1-/-, and RAD18-/- cells as highly GCV-sensitive. RAD17, a component of the alternative checkpoint-clamp loader RAD17-RFC, was required for the activation of the intra-S checkpoint following GCV treatment. BRCA1, a critical factor for promoting homologous recombination (HR), was required for suppressing DNA double-strand breaks (DSBs). Moreover, RAD18, an E3-ligase involved in DNA repair, was critical in suppressing the aberrant ligation of broken chromosomes caused by GCV. We found that BRCA1 suppresses DSBs through HR-mediated repair and template switching (TS)-mediated damage bypass. Moreover, the strong GCV sensitivity of BRCA1-/- cells was rescued by the loss of 53BP1, despite the only partial restoration in the sister chromatid exchange events which are hallmarks of HR. These results indicate that BRCA1 promotes cellular tolerance to GCV through two mechanisms, TS and HR-mediated repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb90975f7c5cd2ebcb118dd3d319f550cae4f8b1" target='_blank'>
              RAD18- and BRCA1-dependent pathways promote cellular tolerance to the nucleoside analog ganciclovir.
              </a>
            </td>
          <td>
            Tasnim Ahmad, Ryotaro Kawasumi, Kouji Hirota
          </td>
          <td>2024-08-22</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="One of the main causes of cancer-related mortality for women worldwide is breast cancer (BC). The XRCC2 gene, essential for DNA repair, has been implicated in cancer susceptibility. This study aims to evaluate the association between XRCC2 and BC risk. The study was conducted at Zheen International Hospital in Erbil, Iraq, between 2021 and 2024 with a total of 88 samples, including 44 paired normal and cancer tissue samples. Mutation analysis was performed using Next-Generation Sequencing, coupled with in silico tools for variant impact prediction. Expression levels were assessed through RT-PCR, and methylation status was determined using methylation-sensitive restriction enzyme digestion PCR. The study identified seven inherited germline variants in the XRCC2 gene, with five of these mutations being Uncertain Significance, one being Likely Pathogenic, and one being Likely benign. RNA purity was found high with mean A260/280 ratios of 1.986 ± 0.097 in normal (N) and 1.963 ± 0.092 in tumor (T) samples. Tumor samples exhibited a higher RNA concentration (78.56 ± 40.87 ng/µL) than normal samples (71.44 ± 40.79 ng/µL). XRCC2 gene expression was significantly upregulated in tumor tissue, with marked increases in patients aged 40-55 and >56 years and in higher cancer grades (II and III) and invasive ductal carcinoma (p-values ranging from <0.0001 to 0.0392). DNA methylation rates in tumor tissues were low (7%), suggesting limited regulation by methylation. The study suggests that XRCC2 can be classified as an oncogene and that its structural investigation by targeted NGS and expression evaluation can be used as a potential biomarker in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81c449dfd85b137aba37b45f3107a8eae5435a35" target='_blank'>
              Association of XRCC2 with breast cancer, a multi-omics analysis at genomic, transcriptomic, and epigenomic level.
              </a>
            </td>
          <td>
            N. Gilani, Mehmet Ozaslan
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Due to the significant morbidity and mortality of hemoglobinopathies, curative options have long been pursued. The overall goal of gene therapy is to modify a patient's own hematopoietic stem cells to overcome the deleterious effects of the underlying genetic defect by gene addition, gene editing, or gene silencing. Gene addition incorporates genes with superior function than the abnormal gene; gene editing takes advantage of molecular tools such as zinc finger proteins, Transcription Activator-Like Effector Nucleases and Clustered Regularly Interspaced Short Palindromic Repeats coupled with Cas9 proteins (CRISPR-Cas9) which allow for sequence-specific breaks in DNA that disrupt gene function; and gene silencing suppresses gene expression by interference with mRNA transcription/protein translation or epigenetic modification. The majority of gene therapy strategies for hemoglobinopathies have targeted erythroid-specific BCL11A, a major regulator of fetal hemoglobin repression at the gamma-globin locus, in the normal fetal-to-adult hemoglobin switch that occurs shortly after birth. Other goals have involved the incorporation of anti-sickling globins, such as βT87Q or βAS3. Landmark clinical trials of gene therapy in transfusion-dependent thalassemia and sickle cell disease have shown remarkable efficacy and acceptable safety and culminated in recent regulatory approvals of gene therapy for both diseases in Europe and the United States.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f812c9e54de47feca0c8615f3dda20f25f85b41" target='_blank'>
              A Review of Gene Therapies for Hemoglobinopathies.
              </a>
            </td>
          <td>
            Boubini Jones-Wonni, Amar H Kelkar, Maureen O Achebe
          </td>
          <td>2024-08-15</td>
          <td>Hemoglobin</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/080990519991d213ae9a360af95e0d99dd86aa83" target='_blank'>
              T4 DNA polymerase prevents deleterious on-target DNA damage and enhances precise CRISPR editing
              </a>
            </td>
          <td>
            Qiaoyan Yang, Jonathan S Abebe, Michelle Mai, Gabriella Rudy, Sang Y Kim, O. Devinsky, Chengzu Long
          </td>
          <td>2024-07-22</td>
          <td>The EMBO Journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aberrant gene expression patterns in acute myeloid leukemia (AML) with balanced chromosomal translocations are often associated with dysregulation of epigenetic modifiers. The AML1/ETO (RUNX1/MTG8) fusion protein, caused by the translocation (8;21)(q22;q22), leads to the epigenetic repression of its target genes. We aimed in this work to identify critical epigenetic modifiers, on which AML1/ETO-positive AML cells depend on for proliferation and survival using shRNA library screens and global transcriptomics approaches. Using shRNA library screens, we identified 41 commonly depleted genes in two AML1/ETO-positive cell lines Kasumi-1 and SKNO-1. We validated, genetically and pharmacologically, DNMT1 and ATR using several AML1/ETO-positive and negative cell lines. We also demonstrated in vivo differentiation of myeloblasts after treatment with the DNMT1 inhibitor decitabine in a patient with an AML1/ETO-positive AML. Bioinformatic analysis of global transcriptomics after AML1/ETO induction in 9/14/18-U937 cells identified 973 differentially expressed genes (DEGs). Three genes (PARP2, PRKCD, and SMARCA4) were both downregulated after AML1/ETO induction, and identified in shRNA screens. In conclusion, using unbiased shRNA library screens and global transcriptomics, we have identified several driver epigenetic regulators for proliferation in AML1/ETO-positive AML. DNMT1 and ATR were validated and are susceptible to pharmacological inhibition by small molecules showing promising preclinical and clinical efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4848d5afaf1e1f3f5c753596a2ed8d1e491ca9f0" target='_blank'>
              Identification of epigenetic modifiers essential for growth and survival of AML1/ETO-positive leukemia.
              </a>
            </td>
          <td>
            J. Duque-Afonso, P. Veratti, Usama-Ur Rehman, Heike Herzog, J. Mitschke, G. Greve, Julian Eble, Bettina Berberich, Johanna Thomas, Milena Pantic, Miguel Waterhouse, Gaia Gentile, Olaf Heidenreich, Cornelius Miething, Michael Lübbert
          </td>
          <td>2024-08-15</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ABSTRACT The human T-cell lymphotropic virus type 1 (HTLV-1) is a single-stranded positive-sense RNA virus that belongs to the Retroviridae family, genus Deltaretro, and infects approximately five to 10 million people worldwide. Although a significant number of individuals living with HTLV-1 remain asymptomatic throughout their lives, some develop one or more severe clinical conditions, such as HTLV-1-associated myelopathy/tropical spastic paraparesis (HAM/TSP), a progressive and debilitating disease, and/or a subtype of non-Hodgkin’s lymphoma with a more threatening course known as adult T-cell leukemia/lymphoma (ATLL). Moreover, current therapeutic options are limited and focus primarily on treating symptoms and controlling viral latency. CRISPR-Cas9 gene editing is proposed as a promising tool to address the intricate links associated with HTLV-1. By targeting or silencing key genes during initial infection and dysregulating immune signaling pathways, CRISPR-Cas9 offers potential intervention opportunities. In this review, we address the therapeutic potential of CRISPR-Cas9 gene editing, as well as examine the primary mechanisms involved in editing potential target genes and discuss the existing evidence in the current scientific literature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa8304d3f00cdbae60023f6fc20b04a24438fa0" target='_blank'>
              Novel approaches for HTLV-1 therapy: innovative applications of CRISPR-Cas9
              </a>
            </td>
          <td>
            Wilson Domingues, V. A. Folgosi, S. Sanabani, Pedro Domingos Leite Junior, Tatiane Assone, J. Casseb
          </td>
          <td>2024-08-26</td>
          <td>Revista do Instituto de Medicina Tropical de São Paulo</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Li-Fraumeni syndrome (LFS) is a hereditary cancer predisposition syndrome associated with a highly penetrant cancer spectrum characterized by germline TP53 mutations. We characterized the first LFS zebrafish hotspot mutants, tp53 R217H and R242H (human R248H and R273H), and found these mutants exhibit partial-to-no activation of p53 target genes, have a defective G1 cell-cycle checkpoint, and are resistant to apoptosis. Spontaneous tumor development histologically resembling human sarcomas developed as early as 6 months. tp53 R242H mutants had a higher lifetime tumor incidence compared to tp53 null and R217H mutants, suggesting it is a more aggressive mutation. We observed mutation-specific tumor phenotypes across tp53 mutants with associated diverse transcriptomic and DNA methylome profiles, impacting metabolism, cell signalling, and biomacromolecule synthesis and degradation. These tp53 zebrafish mutants demonstrate fidelity to their human counterparts and provide new insights into underlying tumorigenesis mechanisms and kinetics, which will guide targeted therapeutics for LFS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe84da849b9bf9707014c32841afe469ca3c5e99" target='_blank'>
              tp53 R217H and R242H Mutant Zebrafish Exhibit Dysfunctional p53 Hallmarks and Recapitulate Li-Fraumeni Syndrome Phenotypes
              </a>
            </td>
          <td>
            Kim Kobar, Lissandra Tuzi, Erin Burnley, Jennifer A. Fiene, Kristianne J. C. Galpin, C. Midgen, B. Laverty, Vallijah Subasri, Timmy T. Wen, Martin Hirst, M. Moksa, A. Carles, Qi Cao, A. Shlien, David Malkin, Sergey V. Prykhozhij, Jason N Berman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Long non‐coding RNAs (lncRNAs) can crucially regulate activation and transformation of cancer‐associated fibroblasts (CAFs) but have not been systematically investigated at single cell resolution. Here, by utilizing integrated single‐cell sequencing datasets, we screened the aberrantly expressed lncRNAs in CAFs, which are the major component of tumor microenvironment. Our findings revealed a consistent CAF‐specific downregulation of Maternally Expressed Gene 3 (MEG3) expression and increased MEG3+ proportion at the pan‐cancer level, which may be attributed to m6A‐related post‐transcriptional modifications. Through activation trajectory analysis of the major CAF subtypes, it was determined that elevated MEG3 expression in CAFs leads to an increase in PDGFRA expression. This, in turn, promotes CAF activation and transformation into an MEG3+ adipogenic CAF (MACAF) subtype, which is more sensitive to Dasatinib. MACAF‐related cell–cell interactions highlighted that MACAF could enhance the epithelial‐mesenchymal transition process in tumor cells via the TGF‐β pathway, promoting tumor cell migration and possibly contributing to tumor progression and invasiveness. Notably, patients with higher MACAF scores experience unfavorable prognoses and poor response rates to checkpoint inhibitor‐based immunotherapy, suggesting a correlation between MACAF and immunosuppressive microenvironment shaping. Our findings provide novel insights of the MEG3 in CAF activation and highlight the potential value of the MACAF score for therapeutic strategies design involving Dasatinib and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60513de4f9e6ad73bba4418e0a42f0b64c44877e" target='_blank'>
              The role of long non‐coding RNA Maternally Expressed Gene 3 in cancer‐associated fibroblasts at single cell pan‐cancer level
              </a>
            </td>
          <td>
            Tao Zhou, Huayun Yan, Yiqi Deng, Yunfeng Zhu, Xuyang Xia, Wanchun Wu, Weihan Zhang, Hai-Ning Chen, Jian‐Kun Hu, Zong‐Guang Zhou, Yang Shu, Yuan Li, Heng Xu
          </td>
          <td>2024-09-09</td>
          <td>Interdisciplinary Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The XmnI Gg -158 C/T polymorphism has been widely associated with fetal hemoglobin (HbF) levels, the severity of disease, and the response to the drug hydroxyurea (HU) in both β-thalassemia (β-thal) and sickle cell disease (SCD) patients. However, the functional significance of this single nucleotide polymorphism (SNP) remains unclear. To gain insight, green fluorescence protein (GFP) cassettes harboring the XmnI C or T alleles in their left homology arms (i.e. Gg promoters) were knocked into the Gg gene(s) of K562 cells via CRISPR/Cas9. Subsequently, the GFP fluorescence levels were compared in the ensuing cell populations and isolated clones. In both instances, median fluorescence intensities (MFI) of the knockin cells having the inserted XmnI T allele were higher than those having the XmnI C allele. Our results suggest that the XmnI T allele can increase Gg expression in K562 cells. The possible functional significance of the XmnI Gg -158 C/T polymorphism provides a rationale for the aforementioned associations. Furthermore, the XmnI polymorphism as a functional SNP substantiates its importance as a prognostic marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bedc9fe1b28e8e9a54e59969786c8981bcb8830d" target='_blank'>
              Genome editing in K562 cells suggests a functional role for the XmnI Gg polymorphism: a widely used genetic marker in β-thalassemia and sickle cell disease patients.
              </a>
            </td>
          <td>
            A. Ahmadifard, Nahal Maroofi, Maryam Maleki Tehrani, Tahere Dabestani, Masoumeh Sadat Mousavi Maleki, Sepideh Bayrami, M. Banan
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Genome editing has demonstrated its utility in generating isogenic cell-based disease models, enabling the precise introduction of genetic alterations into wild-type cells to mimic disease phenotypes and explore underlying mechanisms. However, its application in liver-related diseases has been limited by challenges in genetic modification of mature hepatocytes in a dish. Here, we conducted a systematic comparison of various methods for primary hepatocyte culture and gene delivery to achieve robust genome editing of hepatocytes ex vivo. Our efforts yielded editing efficiencies of up to 80% in primary murine hepatocytes cultured in monolayer and 20% in organoids. To model human hepatic tumorigenesis, we utilized hepatocytes differentiated from human pluripotent stem cells (hPSCs) as an alternative human hepatocyte source. We developed a series of cellular models by introducing various single or combined oncogenic alterations into hPSC-derived hepatocytes. Our findings demonstrated that distinct mutational patterns led to phenotypic variances, affecting both overgrowth and transcriptional profiles. Notably, we discovered that the PI3KCA E542K mutant, whether alone or in combination with exogenous c-MYC, significantly impaired hepatocyte functions and facilitated cancer metabolic reprogramming, highlighting the critical roles of these frequently mutated genes in driving liver neoplasia. In conclusion, our study demonstrates genome-engineered hepatocytes as valuable cellular models of hepatocarcinoma, providing insights into early tumorigenesis mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0098e14922960e96572df20d519a8a83fa8956" target='_blank'>
              Genome Engineering of Primary and Pluripotent Stem Cell-Derived Hepatocytes for Modeling Liver Tumor Formation
              </a>
            </td>
          <td>
            Lulu Zhang, Xunting Wang, Xuelian Yang, Yijia Chi, Yihang Chu, Yi Zhang, Yufan Gong, Fei Wang, Qian Zhao, Dongxin Zhao
          </td>
          <td>2024-09-02</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a62302568e13d5d4ea358c501360fb449540886f" target='_blank'>
              Epigenetic regulation of HERVs: Implications for cancer immunotherapy.
              </a>
            </td>
          <td>
            Joo Mi Yi
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09486473d5ae00655afa329978bc6a48b65d161f" target='_blank'>
              The response to influenza vaccination is associated with DNA methylation-driven regulation of T cell innate antiviral pathways
              </a>
            </td>
          <td>
            Hongxiang Fu, H. Pickering, L. Rubbi, Ted M. Ross, Wanding Zhou, Elaine F. Reed, Matteo Pellegrini
          </td>
          <td>2024-08-21</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Down syndrome is a genetic-based disorder that results from the triplication of chromosome 21, leading to an overexpression of many triplicated genes, including the gene encoding Dual-Specificity Tyrosine Phosphorylation-Regulated Kinase 1A (DYRK1A). This protein has been observed to regulate numerous cellular processes, including cell proliferation, cell functioning, differentiation, and apoptosis. Consequently, an overexpression of DYRK1A has been reported to result in cognitive impairment, a key phenotype of individuals with Down syndrome. Therefore, downregulating DYRK1A has been explored as a potential therapeutic strategy for Down syndrome, with promising results observed from in vivo mouse models and human clinical trials that administered epigallocatechin gallate. Current DYRK1A inhibitors target the protein function directly, which tends to exhibit low specificity and selectivity, making them unfeasible for clinical or research purposes. On the other hand, antisense oligonucleotides (ASOs) offer a more selective therapeutic strategy to downregulate DYRK1A expression at the gene transcript level. Advances in ASO research have led to the discovery of numerous chemical modifications that increase ASO potency, specificity, and stability. Recently, several ASOs have been approved by the U.S. Food and Drug Administration to address neuromuscular and neurological conditions, laying the foundation for future ASO therapeutics. The limitations of ASOs, including their high production cost and difficulty delivering to target tissues can be overcome by further advances in ASO design. DYRK1A targeted ASOs could be a viable therapeutic approach to improve the quality of life for individuals with Down syndrome and their families.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6325ca7451f96825cdc01a440a3d532d3e9fb0b3" target='_blank'>
              Down syndrome and DYRK1A overexpression: relationships and future therapeutic directions
              </a>
            </td>
          <td>
            Aidan J. Murphy, Steve D. Wilton, M. Aung-Htut, Craig S McIntosh
          </td>
          <td>2024-07-24</td>
          <td>Frontiers in Molecular Neuroscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Epigenetics mechanisms play a significant role in human diseases by altering DNA methylation status, chromatin structure, and/or modifying histone proteins. By modulating the epigenetic status, the expression of genes can be regulated without any change in the DNA sequence itself. Epigenetic drugs exhibit promising therapeutic efficacy against several epigenetically originated diseases including several cancers, neurodegenerative diseases, metabolic disorders, cardiovascular disorders, and so forth. Currently, a considerable amount of research is focused on discovering new drug molecules to combat the existing research gap in epigenetic drug therapy. A novel and efficient delivery system can be established as a promising approach to overcome the drawbacks associated with the current epigenetic modulators. Therefore, formulating the existing epigenetic drugs with distinct encapsulation strategies in nanocarriers, including solid lipid nanoparticles, nanogels, bio‐engineered nanocarriers, liposomes, surface modified nanoparticles, and polymer–drug conjugates have been examined for therapeutic efficacy. Nonetheless, several epigenetic modulators are untouched for their therapeutic potential through different delivery strategies. This review provides a comprehensive up to date discussion on the research findings of various epigenetics mechanism, epigenetic modulators, and delivery strategies utilized to improve their therapeutic outcome. Furthermore, this review also highlights the recently emerged CRISPR tool for epigenome editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a8af65ee273c5b2755ef2a82edb72c808ec6577" target='_blank'>
              Targeting the epigenome with advanced delivery strategies for epigenetic modulators
              </a>
            </td>
          <td>
            Sonia Guha, Yogeswaran Jagadeesan, M. M. Pandey, A. Mittal, Deepak Chitkara
          </td>
          <td>2024-08-17</td>
          <td>Bioengineering &amp; Translational Medicine</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Mitochondria are essential cellular organelles that play critical roles in hematological disorders. Recurrent mutations in mitochondrial DNA (mtDNA) have been identified in patients with myelodysplastic syndromes (MDS) and serve as significant prognostic indicators for their outcomes following allogeneic hematopoietic stem-cell transplantation (allo-HCT). However, the biological mechanisms of mtDNA mutations remain unclear. The current study utilizes computational structural genomics to improve our understanding of pathogenic variants in mitochondria-encoded genes. This emerging genomics discipline employs structural models, molecular mechanic calculations, and accelerated molecular dynamic simulations to analyze gene products, focusing on their structures and motions that determine their function. We applied this methodology to perform deep variant phenotyping of entire mitochondria-encoded protein complexes associated with the pathobiology of MDS and their prognosis after HCT. Our results demonstrate that this approach significantly outperforms conventional analytical methods, providing enhanced and more accurate information to support the potential pathogenicity of these variants and better infer their dysfunctional mechanisms. We conclude that the adoption and further expansion of computational structural genomics approaches, as applied to the mitochondrial genome, have the potential to significantly increase our understanding of molecular mechanisms underlying the disease. Our study lays a foundation for translating mitochondrial biology into clinical applications, which is of significant mechanistic and biomedical relevance and should be considered in modern biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc43713cc3714fe35f1cd36d22abdad08d31f712" target='_blank'>
              Advancing Variant Phenotyping in Myelodysplastic Syndromes via Computational Genomics of Mitochondrial Enzyme Complexes
              </a>
            </td>
          <td>
            Jing Dong, Michael T. Zimmermann, Neshatul Haque, Shahram Arsang-Jang, Wael Saber, Raul Urrutia
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Alexander Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii-DerIda Chen, C. Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="Abstract CRISPR/Cas9 genome editing is a pervasive research tool due to its relative ease of use. However, some systems are not amenable to generating edited clones due to genomic complexity and/or difficulty in establishing clonal lines. For example, Drosophila Schneider 2 (S2) cells possess a segmental aneuploid genome and are challenging to single-cell select. Here, we describe a streamlined CRISPR/Cas9 methodology for knock-in and knock-out experiments in S2 cells, whereby an antibiotic resistance gene is inserted in-frame with the coding region of a gene-of-interest. By using selectable markers, we have improved the ease and efficiency for the positive selection of null cells using antibiotic selection in feeder layers followed by cell expansion to generate clonal lines. Using this method, we generated the first acentrosomal S2 cell lines by knocking-out centriole genes Polo-like Kinase 4/Plk4 or Ana2 as proof of concept. These strategies for generating gene-edited clonal lines will add to the collection of CRISPR tools available for cultured Drosophila cells by making CRISPR more practical and therefore improving gene function studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d9bec46a5fefb60dc99b1e4c11b9d69666ffa32" target='_blank'>
              Generating CRISPR-edited clonal lines of cultured Drosophila S2 cells
              </a>
            </td>
          <td>
            John M. Ryniawec, Anastasia Amoiroglou, Gregory C. Rogers
          </td>
          <td>2024-08-17</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c24f0b42670eaa7d910971701aacf3a8d5213b" target='_blank'>
              Targeting RBM39 through indisulam induced mis-splicing of mRNA to exert anti-cancer effects in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tongting Ji, Yang Yang, Juanjuan Yu, Hongli Yin, Xinran Chu, Pengju Yang, Ling Xu, Xiaodong Wang, Shaoyan Hu, Yizhen Li, Xiaochen Wu, Wengyuan Liu, Bi Zhou, Wenjuan Wang, Shuqi Zhang, Wei Cheng, Yanling Chen, Lei Shi, Zhiheng Li, Ran Zhuo, Yongping Zhang, Yanfang Tao, Di Wu, Xiao-Lu Li, Zimu Zhang, Jun-Jie Fan, Jian Pan, Jun Lu
          </td>
          <td>2024-07-24</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background: Acute myeloid leukemia (AML) is a type of blood cancer with diverse genetic pathogenesis. The identification of potential molecular biomarkers could improve the AML diagnosis and targeted therapy. Recently, competing endogenous RNAs (ceRNAs) became an active area in cancer research to determine molecular mechanisms underlying tumor development. Objectives: The aim of the present study was to investigate the key molecular biomarkers that are closely related to AML through bioinformatics analysis. Methods: In this research, the RNA-seq data of 151 AML patients and 151 corresponding health samples were retrieved from The Cancer Genome Atlas (TCGA) database and GTEx Portal (genotype-tissue expression), respectively. After that we screened the differentially expressed long non-coding RNAs (lncRNAs), microRNAs (miRNAs), and mRNAs by “limma” package in R to construct the co-expression network ceRNA (miRNA-lncRNA-mRNA) with weighted gene co-expression network analysis (WGCNA) package in R and Cytoscape (version 3.7.2) software, respectively. Then the relevant modules were identified and functional enrichment analysis was used to uncover the modules that are biologically related to AML cancer. Results: Based on our bioinformatics studies, we constructed a significant module which contain 333 genes. Among them, five up-regulated miRNAs with the highest GS (gene significant), include hsa-miR-374B, hsa-miR-553, hsa-miR-3679, hsa-miR-548L, hsa-miR-597 and five down-regulated miRNAs with the lowest GS including hsa-miR-3934, hsa-miR-4746, hsa-miR-466, hsa-miR-6722, hsa-miR-4490. For protein-coding genes (PCGs), the top-five up-regulated PCGs with the highest GS were AMIGO3, H2AC17, SPACA5, GPR21, and OR13C3. The top-five down-regulated PCGs with the lowest GS were XBP1, TOMM6, TREX1, TNFRSF6B, and BOLA2. Among the lncRNAs, the five up-regulated lncRNAs with the highest level of GS were GREP1, ZBTB20-AS2, LINC01596, LINC00345, and RMRP. The five down-regulated lncRNAs with the lowest GS were LINC01609, LINC01707, LINC02270, LINC02309, and LINC02243. These lncRNAs can serve as potential biomarkers to distinguish between AML and normal samples. Conclusions: Finally, considering the role of ceRNA network in various biological processes, examining the role of ceRNAs in AML may provide a deeper insight into molecular mechanisms underlying the pathogenesis of cancer. This understanding may propose potential biomarkers for diagnosis and therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b4efc24040b8239ec857871d4330cd62e238941" target='_blank'>
              Construction of CeRNA Regulatory Network Based on WGCNA to Reveal Potential Prognostic Biomarkers for AML Cancer
              </a>
            </td>
          <td>
            Fateme Larki Ard Darabi, Mahsa Dehdar, Ghasem Azizi-Tabesh
          </td>
          <td>2024-08-18</td>
          <td>International Journal of Cancer Management</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metabolic dysfunction-associated Fatty Liver Disease (MAFLD) has emerged as one of the leading cardiometabolic diseases. Friend of GATA2 (FOG2) is a transcriptional co-regulator that has been shown to regulate hepatic lipid metabolism and accumulation. Using meta-analysis from several different biobank datasets, we identified a coding variant of FOG2 (rs28374544, A1969G, S657G) predominantly found in individuals of African ancestry (minor allele frequency~20%), which is associated with liver failure/cirrhosis phenotype and liver injury. To gain insight into potential pathways associated with this variant, we interrogated a previously published genomics dataset of 38 human induced pluripotent stem cell (iPSCs) lines differentiated into hepatocytes (iHeps). Using Differential Gene Expression Analysis and Gene Set Enrichment Analysis, we identified the mTORC1 pathway as differentially regulated between iHeps from individuals with and without the variant. Transient lipid-based transfections were performed on the human hepatoma cell line (Huh7) using wild-type FOG2 and FOG2S657G and demonstrated that FOG2S657G increased mTORC1 signaling, de novo lipogenesis, and cellular triglyceride synthesis and mass. In addition, we observed a significant downregulation of oxidative phosphorylation in FOG2S657G cells in fatty acid-loaded cells but not untreated cells, suggesting that FOG2S657G may also reduce fatty acid to promote lipid accumulation. Taken together, our multi-pronged approach suggests a model whereby the FOG2S657G may promote MAFLD through mTORC1 activation, increased de novo lipogenesis, and lipid accumulation. Our results provide insights into the molecular mechanisms by which FOG2S657G may affect the complex molecular landscape underlying MAFLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2945851076c2f51a928c3f7a7537ad4ec5a83e4e" target='_blank'>
              Genotype-First Approach Identifies an Association between rs28374544/FOG2S657G and Liver Disease through Alterations in mTORC1 Signaling
              </a>
            </td>
          <td>
            Donna Conlon, Siri Kanakala, Tess A. Cherlin, Yi-An Ko, Cecilia Vitali, Sharavana Gurunathan, Rasika Venkatesh, Jakob Woerner, Lindsay Guare, Penn Medicine Biobank, Anurag Verma, S. Verma, Marie A. Guerraty
          </td>
          <td>2024-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2dde0164af0025d5f19bf691294f177add4f7b" target='_blank'>
              Low-pass whole genome sequencing of circulating tumor cells to evaluate chromosomal instability in triple-negative breast cancer
              </a>
            </td>
          <td>
            S. Di Cosimo, M. Silvestri, C. De Marco, Alessia Calzoni, M. C. De Santis, Maria Grazia Carnevale, C. Reduzzi, Massimo Cristofanilli, Vera Cappelletti
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="CRISPR-Associated Transposases (CASTs) hold tremendous potential for microbial genome editing due to their ability to integrate large DNA cargos in a programmable and site-specific manner. However, the widespread application of CASTs has been hindered by their low efficiency in diverse, non-model bacteria. In an effort to address this shortcoming, we conducted the first genome-wide screen for host factors impacting Vibrio cholerae CAST (VchCAST) activity and used the findings to increase VchCAST editing efficiency. A genome-wide loss-of-function mutant library in E. coli was screened to identify 15 genes that impact type VchCAST transposition. Of these, seven factors were validated to improve VchCAST activity and two were found to be inhibitory. Informed by homologous recombination involved effectors, RecD and RecA, we tested the λ-Red recombineering system in our VchCAST editing vectors, which increased its insertion meditated-editing efficiency by 25.7-fold in E. coli while maintaining high target specificity and similar insertion arrangements. Furthermore, λ-Red-enhanced VchCAST achieved increased editing efficiency in the industrially important bacteria Pseudomonas putida and the emerging pathogen Klebsiella michiganensis. This study improves understanding of factors impacting VchCAST activity and enhances its efficiency as a bacterial genome editor. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2483d77579dd4ab8c5f977694f46b22d04f0a6" target='_blank'>
              Identification of Proteins Influencing CRISPR-Associated Transposases for Enhanced Genome Editing
              </a>
            </td>
          <td>
            , Amanda T.P. Alker, Agnès Oromí-Bosch, Sophia E. Swartz, Jonathan N.V. Martinson, Jigyasa Arora, Abby M. Wang, Rachel Rovinsky, Sara J. Smith, Emily C. Pierce, A. Deutschbauer, Jennifer A. Doudna, Brady F. Cress, Benjamin E. Rubin
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Abstract Somatic structural variations (SVs) in cancer can shuffle DNA content in the genome, relocate regulatory elements, and alter genome organization. Enhancer hijacking occurs when SVs relocate distal enhancers to activate proto-oncogenes. However, most enhancer hijacking studies have only focused on protein-coding genes. Here, we develop a computational algorithm ‘HYENA’ to identify candidate oncogenes (both protein-coding and non-coding) activated by enhancer hijacking based on tumor whole-genome and transcriptome sequencing data. HYENA detects genes whose elevated expression is associated with somatic SVs by using a rank-based regression model. We systematically analyze 1146 tumors across 25 types of adult tumors and identify a total of 108 candidate oncogenes including many non-coding genes. A long non-coding RNA TOB1-AS1 is activated by various types of SVs in 10% of pancreatic cancers through altered 3-dimensional genome structure. We find that high expression of TOB1-AS1 can promote cell invasion and metastasis. Our study highlights the contribution of genetic alterations in non-coding regions to tumorigenesis and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fb0cf8831bea2891dbca4714836cde381f2ca0" target='_blank'>
              HYENA detects oncogenes activated by distal enhancers in cancer
              </a>
            </td>
          <td>
            Anqi Yu, Ali E. Yesilkanal, Ashish Thakur, Fan Wang, Yang Yang, William Phillips, Xiaoyang Wu, Alexander Muir, Xin He, Francois Spitz, Lixing Yang
          </td>
          <td>2024-07-25</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Juvenile myelomonocytic leukemia (JMML), a clonal hematologic malignancy, originates from mutated hematopoietic stem cells (HSCs). The mechanism sustaining the persistence of mutant stem cells, leading to leukemia development, remains elusive. In this study, we conducted comprehensive examination of gene expression profiles, transcriptional factor regulons, and cell compositions/interactions throughout various stages of tumor cell development in Ptpn11 mutation-associated JMML. Our analyses revealed that leukemia-initiating Ptpn11E76K/+ mutant stem cells exhibited de novo activation of the myeloid transcriptional program and aberrant developmental trajectories. These mutant stem cells displayed significantly elevated expression of innate immunity-associated anti-microbial peptides and pro-inflammatory proteins, particularly S100a9 and S100a8. Biological experiments confirmed that S100a9/S100a8 conferred a selective advantage to the leukemia-initiating cells through autocrine effects and facilitated immune evasion by recruiting and promoting immune suppressive myeloid-derived suppressor cells (MDSCs) in the microenvironment. Importantly, pharmacological inhibition of S100a9/S100a8 signaling effectively impeded leukemia development from Ptpn11E76K/+ mutant stem cells. These findings collectively suggest that JMML tumor-initiating cells exploit evolutionarily conserved innate immune and inflammatory mechanisms to establish clonal dominance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14ee0ff31f4868484243a76f8cf02c1e5b54c52" target='_blank'>
              Prototypical innate immune mechanism hijacked by leukemia-initiating mutant stem cells for selective advantage and immune evasion in Ptpn11-associated juvenile myelomonocytic leukemia
              </a>
            </td>
          <td>
            Cheng-Kui Qu, Hong Zheng, Peng Zhao, Zhenya Tan, Wen‐mei Yu, Juwita Werner, Elliot Stieglitz, Christopher C Porter, S. Chandrakasan, Daniel Wechsler, Simon Mendez-Ferrer
          </td>
          <td>2024-08-02</td>
          <td>Research Square</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Non-coding regulatory sequences play essential roles in adjusting gene output to cellular needs and are thus critical to animal development and health. Numerous such sequences have been identified in mammalian genomes ranging from transcription factors binding motifs to recognition sites for RNA-binding proteins and non-coding RNAs. The advent of CRISPR has raised the possibility of assigning functionality to individual endogenous regulatory sites by facilitating the generation of isogenic cell lines that differ by a defined set of genetic modifications. Here we investigate the usefulness of this approach to assign function to individual miRNA binding sites. We find that the process of generating isogenic pairs of mammalian cell lines with CRISPR-mediated mutations introduces extensive molecular and phenotypic variability between biological replicates making any attempt of assigning function to the binding site essentially impossible. Our work highlights an important consideration when employing CRISPR editing to characterize non-coding regulatory sequences in cell lines and calls for the development and adoption of alternative strategies to address this question in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1d3a055f759b974342766678fe87bc5f92b153" target='_blank'>
              CRISPR-based dissection of miRNA binding sites using isogenic cell lines is hampered by pervasive noise
              </a>
            </td>
          <td>
            Mahendra Prajapat, Joana A. Vidigal
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Elucidating the genetic architecture of DNA methylation (DNAm) is crucial for decoding the etiology of complex diseases. However, current epigenomic studies often suffer from incomplete coverage of methylation sites and the use of tissues containing heterogeneous cell populations. To address these challenges, we present a comprehensive human methylome atlas based on deep whole-genome bisulfite sequencing (WGBS) and whole-genome sequencing (WGS) of purified monocytes from 298 European Americans (EA) and 160 African Americans (AA) in the Louisiana Osteoporosis Study. Our atlas enables the analysis of over 25 million DNAm sites. We identified 1,383,250 and 1,721,167 methylation quantitative trait loci (meQTLs) in cis-regions for EA and AA populations, respectively, with 880,108 sites shared between ancestries. While cis-meQTLs exhibited population-specific patterns, primarily due to differences in minor allele frequencies, shared cis-meQTLs showed high concordance across ancestries. Notably, cis-heritability estimates revealed significantly higher mean values in the AA population (0.09) compared to the EA population (0.04). Furthermore, we developed population-specific DNAm imputation models using Elastic Net, enabling methylome-wide association studies (MWAS) for 1,976,046 and 2,657,581 methylation sites in EA and AA, respectively. The performance of our MWAS models was validated through a systematic multi-ancestry analysis of 41 complex traits from the Million Veteran Program. Our findings bridge the gap between genomics and the monocyte methylome, uncovering novel methylation-phenotype associations and their transferability across diverse ancestries. The identified meQTLs, MWAS models, and data resources are freely available at www.gcbhub.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02561d4c021777b32cc89ad698157b2f333e71dc" target='_blank'>
              An atlas of genetic effects on the monocyte methylome across European and African populations
              </a>
            </td>
          <td>
            Wanheng Zhang, Xiao Zhang, Chuan Qiu, Zichen Zhang, Kuan-Jui Su, Zhe Luo, Minghui Liu, Bingxin Zhao, Lang Wu, Qing Tian, Hui Shen, Chong Wu, Hong-Wen Deng
          </td>
          <td>2024-08-13</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Human inborn errors of immunity (IEI) represent a diverse group of genetic disorders affecting the innate and/or adaptive immune system. Some IEI entities comprise defects in DNA repair factors, resulting in (severe) combined immunodeficiencies, bone marrow failure, predisposition to malignancies, and potentially result in radiosensitivity (RS). While other IEI subcategories such as common variable immunodeficiency (CVID) and immune dysregulation disorders also associate with lymphoproliferative and malignant complications, the occurrence of RS phenotypes in the broader IEI population is not well characterized. Nonetheless, identifying RS in IEI patients through functional testing is crucial to reconsider radiation-related therapeutic protocols and to improve overall patient management. This study aimed to investigate chromosomal RS in a diverse cohort of 107 IEI patients using the G0 cytokinesis-block micronucleus (MN) assay. Our findings indicate significant variability in RS across specific genetic and phenotypical subgroups. Severe RS was detected in all ataxia-telangiectasia (AT) patients, a FANCI deficient and ERCC6L2 deficient patient, but not in any other IEI patient included in this cohort. Age emerged as the single influencing factor for both spontaneous and radiation-induced MN yields, while the manifestation of additional clinical features, including infection susceptibility, immune dysregulation, or malignancies did not associate with increased MN levels. Our extensive analysis of RS in the IEI population underscores the clinical importance of RS assessment in AT patients and supports RS testing in all IEI patients suspected of having a DNA repair disorder associated with radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b4f50595c5c29676228d24464dde5631290eb13" target='_blank'>
              Investigating chromosomal radiosensitivity in inborn errors of immunity: insights from DNA repair disorders and beyond.
              </a>
            </td>
          <td>
            E. Beyls, Evi Duthoo, Lynn Backers, Karlien Claes, Marieke De Bruyne RAPID clinicians, L. Pottie, Victoria Bordon, C. Bonroy, S. Tavernier, Kathleen BM Claes, A. Vral, A. Baeyens, F. Haerynck, T. Kerre, B. Lambrecht, L. Hoste, B. Meertens, T. V. Genechten, A. Ferster, K. V. Driessche, C. D. Vriendt, L. Dedeken, I. Moors, C. Dhooge, L. Kornreich, C. Heijmans, M. L. Roux, M. Colard, A. V. Damme, K. V. Schil, B. D. Wilde
          </td>
          <td>2024-08-20</td>
          <td>None</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Simple Summary p53 mutations are prevalent across a variety of human cancers and are regarded as a major obstacle in cancer therapy. These mutations can confer resistance to apoptosis and cell cycle arrest, contributing to multidrug resistance in tumors. Recent studies have uncovered important immunomodulatory functions of p53, but these functions are still underappreciated compared to its other well-known roles. This review aims to summarize the latest literature on immune evasion in p53-mutant tumors and explore the potential of targeting p53 to enhance anti-tumor immunity. Abstract Mutation in p53 is the most frequent event in cancer development and a leading cause of cancer therapy resistance due to evasion of the apoptosis cascade. Beyond chemotherapies and radiation therapies, growing evidence indicates that p53-mutant tumors are resistant to a broad range of immune-based therapies, such as immune checkpoint inhibitors, chimeric antigen receptor (CAR) T, and hematopoietic stem cell transplantation (HSCT). This highlights the role of p53 mutations in driving immune evasion of tumor cells. In this review, we first summarize recent studies revealing mechanisms by which p53-mutant tumors evade immune surveillance from T cells, natural killer (NK) cells, and macrophages. We then review how these mutant tumor cells reshape the tumor microenvironment (TME), modulating bystander cells such as macrophages, neutrophils, and regulatory T (Treg) cells to foster immunosuppression. Additionally, we review clinical observations indicative of immune evasion associated with p53 loss or mutations. Finally, we discuss therapeutic strategies to enhance immune response in p53 wild-type (WT) or mutant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ac96dfaadf6e3a60d7f3146c1f00c2f180c35b" target='_blank'>
              TP53 Mutation-Mediated Immune Evasion in Cancer: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            Chuqi Wang, Jordan Yong Ming Tan, Nishtha Chitkara, Shruti Bhatt
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Objective: Colon cancer is associated with multiple levels of molecular heterogeneity. RNA processing converts primary transcriptional RNA to mature RNA, which drives tumourigenesis and its maintenance. The characterisation of RNA processing genes in colon cancer urgently needs to be elucidated. Methods: In this study, we obtained 1033 relevant samples from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases to explore the heterogeneity of RNA processing phenotypes in colon cancer. Firstly, Unsupervised hierarchical cluster analysis detected 4 subtypes with specific clinical outcomes and biological features via analysis of 485 RNA processing genes. Next, we adopted the least absolute shrinkage and selection operator (LASSO) as well as Cox regression model with penalty to characterise RNA processing-related prognostic features. Results: An RNA processing-related prognostic risk model based on 10 genes including FXR1, MFAP1, RBM17, SAGE1, SNRPA1, SRRM4, ADAD1, DDX52, ERI1, and EXOSC7 was identified finally. A composite prognostic nomogram was constructed by combining this feature with the remaining clinical variables including TNM, age, sex, and stage. Genetic variation, pathway activation, and immune heterogeneity with risk signatures were also analysed via bioinformatics methods. The outcomes indicated that the high-risk subgroup was associated with higher genomic instability, increased proliferative and cycle characteristics, decreased tumour killer CD8+ T cells and poorer clinical prognosis than the low-risk group. Conclusion: This prognostic classifier based on RNA-edited genes facilitates stratification of colon cancer into specific subgroups according to TNM and clinical outcomes, genetic variation, pathway activation, and immune heterogeneity. It can be used for diagnosis, classification and targeted treatment strategies comparable to current standards in precision medicine. It provides a rationale for elucidation of the role of RNA editing genes and their clinical significance in colon cancer as prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/120a9cc9c01f30852bfe106a395264eb722362ff" target='_blank'>
              Characterization of RNA Processing Genes in Colon Cancer for Predicting Clinical Outcomes
              </a>
            </td>
          <td>
            Jianwen Hu, Yingze Ning, Yongchen Ma, Lie Sun, Guowei Chen
          </td>
          <td>2024-01-01</td>
          <td>Biomarker Insights</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BACKGROUND
Lung cancer is a prevalent and severe form of malignant tumors worldwide. tRF-Leu-CAG, a recently discovered non-coding single-stranded small RNA derived from transfer RNA, has sparked interest in exploring its biological functions and potential molecular mechanisms in lung cancer.


METHODS
The abundance of tRF-Leu-CAG was measured via quantitative real-time polymerase chain reaction (qRT-PCR) in 96 sets of lung cancer tissue samples obtained from clinical patients. Subsequently, both in vivo and in vitro experiments were conducted to validate the biological functions of tRF-Leu-CAG in lung cancer. Furthermore, an exploration of the potential target genes of tRF-Leu-CAG and its association with autophagy and drug resistance in lung cancer was undertaken.


RESULTS
Our analysis revealed a significant upregulation of tRF-Leu-CAG in non-small cell lung cancer (NSCLC) tissues. Additionally, we observed that heightened expression of tRF-Leu-CAG significantly augmented the proliferation and migration of NSCLC cells, facilitated cell cycle progression, and suppressed apoptosis. Furthermore, we identified transcription elongation factor A3 (TCEA3) as a direct target gene of tRF-Leu-CAG. TCEA3 inhibited the proliferation and migration of NSCLC, and tRF-Leu-CAG promoted the proliferation and migration of NSCLC by mediating the silencing of TCEA3. Moreover, we demonstrated that the augmentation of paclitaxel resistance by tRF-Leu-CAG was contingent on autophagy. Finally, tRF-Leu-CAG notably accelerated tumor growth and promoted the process of epithelial-mesenchymal transition (EMT) in vivo.


CONCLUSIONS
tRF-Leu-CAG promotes NSCLC tumor growth and metastasis by targeting TCEA3 and promotes paclitaxel resistance by enhancing cellular autophagy. These results provide potentially effective targets and therapeutic options for the clinical treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a8cbdbaa838f4eb2da1ce3af02d5ba168916d9" target='_blank'>
              Oncogenic tRNA-derived fragment tRF-Leu-CAG promotes tumorigenesis of lung cancer via targeting TCEA3 and increasing autophagy.
              </a>
            </td>
          <td>
            Fan Wu, Binshu Chai, Pengfei Qi, Yaqi Han, Z. Gu, Weikang Pan, Hui Zhang, Xianyi Wang, Xiaomin Liu, Heng Zou, Chen Liang, Yanli Li, Wen-xu Fang, Zhongliang Ma
          </td>
          <td>2024-08-28</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Abstract Disruptions in normal development and the emergence of health conditions often result from the malfunction of vital genes in the human body. Decades of scientific research have focused on techniques to modify or substitute defective genes with healthy alternatives, marking a new era in disease treatment, prevention and cure. Recent strides in science and technology have reshaped our understanding of disorders, medication development and treatment recommendations, with human gene and cell therapy at the forefront of this transformative shift. Its primary objective is the modification of genes or adjustment of cell behaviour for therapeutic purposes. In this review, we focus on the latest advances in gene and cell therapy for treating human genetic diseases, with a particular emphasis on FDA and EMA‐approved therapies and the evolving landscape of genome editing. We examine the current state of innovative gene editing technologies, particularly the CRISPR‐Cas systems. As we explore the progress, ethical considerations and prospects of these innovations, we gain insight into their potential to revolutionize the treatment of genetic diseases, along with a discussion of the challenges associated with their regulatory pathways. This review traces the origins and evolution of these therapies, from conceptual ideas to practical clinical applications, marking a significant milestone in the field of medical science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/605617113766e025c90c18b7b7452f93c13a0842" target='_blank'>
              Gene and cell therapy of human genetic diseases: Recent advances and future directions
              </a>
            </td>
          <td>
            Busra Cetin, Fulya Erendor, Y. E. Eksi, A. Şanlioğlu, S. Sanlioglu
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9323b395426f00928696c5f63d0f88e7dffceb5" target='_blank'>
              The evolving genetic landscape of telomere biology disorder dyskeratosis congenita.
              </a>
            </td>
          <td>
            H. Tummala, A. Walne, M. Badat, Manthan Patel, Abigail M Walne, Jenna Alnajar, Chi Ching Chow, Ibtehal Albursan, J. Frost, David Ballard, Sally Killick, Peter Szitányi, Anne M Kelly, Manoj Raghavan, Corrina Powell, Reinier Raymakers, Tony Todd, Elpis Mantadakis, Sophia Polychronopoulou, Nikolas Pontikos, Tianyi Liao, Pradeepa M. Madapura, Upal Hossain, T. Vulliamy, I. Dokal
          </td>
          <td>2024-08-28</td>
          <td>EMBO molecular medicine</td>
          <td>1</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab9291d16699439a2d6b53e165a3605cd9a24fb" target='_blank'>
              Unraveling the spatial organization and development of human thymocytes through integration of spatial transcriptomics and single-cell multi-omics profiling
              </a>
            </td>
          <td>
            Yanchuan Li, Huamei Li, Cheng Peng, Ge Meng, Yijun Lu, Honglin Liu, Li Cui, Huan Zhou, Zhu Xu, Lingyun Sun, Lihong Liu, Qing Xiong, Beicheng Sun, Shiping Jiao
          </td>
          <td>2024-09-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Personalized medicine programs, including the Zero Childhood Cancer Program (ZERO), perform molecular analysis of individual patient tumours to identify molecular targets for therapy. The comprehensive and unbiased sequencing approaches used by ZERO, including whole genome sequencing (WGS) and RNA sequencing (RNAseq), enables detection of novel structural variants (SVs) that may be cryptic to standard cytogenetic techniques or panel sequencing approaches. Identification of Receptor Tyrosine Kinase (RTK) activating variants in paediatric cancers, for example NTRK2 gene fusions, is a high priority given the availability and remarkable clinical success of RTK inhibitors. Here we describe a novel internal tandem duplication (ITD) in NTRK2, encoding TRKB, in a ZERO patient with CNS neuroblastoma. This SV was detected by WGS and was notably missed by RNAseq-based gene fusion and SV detection. The ITD spans exons 10-13 of NTRK2, is in-frame, and includes the juxtamembrane and transmembrane protein domains. We hypothesized that this ITD would result in constitutive activation of TRKB and would be sensitive to TRK-targeted therapies. We overexpressed NTRK2 ITD, wild-type NTRK2, and SPECC1L-NTRK2 (an established TRK-activating fusion), in the interleukin-3 (IL-3)-dependent cell line, Ba/F3, and the neuroblastoma cell line, SH-SY5Y. In these models, we showed that like wild-type NTRK2, NTRK2 ITD was expressed on the cell surface. NTRK2 ITD was sufficient to transform Ba/F3 cells to IL-3 independence through constitutive activation of TRKB and downstream signaling pathways PI3K/AKT and MEK/ERK. NTRK2 ITD-expressing cells were sensitive to TRK inhibitors, including larotrectinib, at similar doses to SPECC1L-NTRK2-expressing cells. Interestingly, we observed that cells expressing NTRK2 ITD were specifically sensitive to MEK inhibition, which was not observed in SPECC1L-NTRK2-expressing cells, suggesting the mechanisms by which these variants drive transformation is different. Indeed, in silico structural analysis showed that the duplicated region of NTRK2 ITD contains two internalized tyrosine residues whose phosphorylation could both drive autophosphorylation of the receptor and act as docking sites for adaptor proteins. Mutation of these tyrosine residues delayed transformation of Ba/F3 cells. This study functionally characterizes a novel NTRK2 ITD and shows that this variant is transforming and sensitive to TRK inhibition. While ITDs have been described in other RTK genes, notably FGFR1 and FLT3, structurally these have involved the kinase domain or the juxtamembrane domain. This is the first report of an ITD that spans the transmembrane domain of an RTK, characterizing an additional mechanism by which RTKs can be activated in cancer. This study highlights the value of unbiased WGS approaches to identify novel SVs and identifies another class of patients that may benefit from RTK inhibitor therapy.
 Citation Format: Lauren M. Brown, Gabor Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah Jalud, Teresa Sadras, Pamela Ajuyah, Paulette Barahona, M. Emmy M. Dolman, Marie Wong, Chelsea Mayoh, Mark J. Cowley, Paul G. Ekert. A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ae609e3759161918ab8221b777b148ec5605447" target='_blank'>
              Abstract B002: A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation
              </a>
            </td>
          <td>
            Lauren M. Brown, G. Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah B Jalud, Teresa Sadras, P. Ajuyah, P. Barahona, M. E. M. Dolman, Marie Wong, C. Mayoh, Mark J. Cowley, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="
 Children born with Down syndrome (DS), a genetic condition caused by the presence of an additional chromosome 21 (trisomy 21 or T21), have a substantially higher risk of developing childhood leukaemia. Approximately 30% of DS infants develop a preleukemic condition called transient abnormal myelopoiesis (TAM). TAM is strictly associated with GATA1 truncating mutations on a T21 background. Most TAM cases spontaneously resolve within the first few months of life. However, by the age of five, approximately 10% of cases progress to megakaryoblastic/erythroid leukaemia, known as myeloid leukaemia of DS (ML-DS). This final step of transformation is often driven by specific third-hit somatic mutations, most of which affects genes encoding JAK kinases, cohesin complexes, or epigenetic regulators. These well-defined stepwise genetic aberrations associated with ML-DS pathogenesis provide a unique and highly tractable system to elucidate the molecular changes underlying leukaemogenesis. Using single-cell mRNA sequencing of primary human samples, we directly interrogate the transcriptional changes underpinning the multi-step pathogenesis of ML-DS. We observe an expansion of megakaryocyte-erythrocyte progenitors (MEPs) specifically in T21 foetal liver compared to diploid or other trisomies. However, it is the GATA1 mutation that accounts for the majority of the transcriptomic changes towards that of the leukaemic blasts. Furthermore, investigation of an aggressive TAM case reveals an enrichment in expression of the leukaemogenic gene module associated with the full-blown leukaemia state ML-DS. We also demonstrate that TAM and ML-DS blasts still retain transcriptional signatures of normal haematopoiesis, showing heterogeneous features of differentiation towards erythrocytes, megakaryocytes, and mast cells. Finally, by integrating genomic and transcriptomic data, we directly defined the genomic evolution, and its corresponding transcriptional consequences, that underpins a treatment-refractory case of ML-DS. Overall, we have generated the first comprehensive single-cell mRNA atlas of TAM / ML-DS, enabling unbiased quantitative characterisation of the transcriptional transformation along ML-DS pathogenesis. Our study explores the molecular signature associated with progression of TAM to ML-DS, as well as the mechanism underlying differences in treatment response. These insights offer the possibility of more meaningful clinical interpretations and potential to explore novel therapeutic targets for these conditions.
 Citation Format: Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, Angus Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, Jan-Henning Klusmann, Jack Bartram, Sam Behjati. Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f4dde5c53348eeefdb9808ec633733f445cffc" target='_blank'>
              Abstract A018 Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome
              </a>
            </td>
          <td>
            Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, A. Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, J. Klusmann, Jack Bartram, Sam Behjati
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8245d23a6d26f49afe8896fd7ee25b1dbf95a8d" target='_blank'>
              Tailored antisense oligonucleotides designed to correct aberrant splicing reveal actionable groups of mutations for rare genetic disorders
              </a>
            </td>
          <td>
            H. Wai, Eliska Svobodova, Natalia Romero Herrera, Andrew G L Douglas, John W Holloway, F. Baralle, Marco Baralle, D. Baralle
          </td>
          <td>2024-08-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04846b6dbb15a0b4465791645d0fdc5393431ffd" target='_blank'>
              Epigenetics and immunotherapy in colorectal cancer: progress and promise
              </a>
            </td>
          <td>
            Tianjiao Dang, Xin Guan, Luying Cui, Yuli Ruan, Zhuo Chen, H. Zou, Ya Lan, Chao Liu, Yanqiao Zhang
          </td>
          <td>2024-09-10</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction: The role of genetic variants in response to chemotherapy has been investigated in several studies. This study aimed to investigate genetic variants associated with response to chemotherapy in breast cancer (BC) patients. Methods: Significant variants (p < 5 × 10-8) associated with response to chemotherapy were obtained from GWA studies. Candidate variants were identified by haplotype analysis (r2 ≥ 0.9, D'≥0.9) using 1000Genome LD data. To determine the effects of the variants on gene expression, expression quantitative trait loci (eQTL) were evaluated. To compare the expression of the identified genes in tumor samples, expression levels were compared between TCGA tumor types and adjacent normal tissues. Results: Six rs3820706, rs147451859, rs4784750, rs17587029, rs16830728, and rs16972207 variants were significantly associated with response to chemotherapy in BC patients (p < 5 × 10-8). Seven novel haplotypic structures were identified to be associated with adverse response to chemotherapy in BC patients. These haplotypes formed two genetic structures associated with neutropenia, leukopenia, chemotherapy-induced cytotoxicity (GAG-TTAT), and chemotherapy-induced alopecia (CC-CAACTCCCGTTGCGG). These variants are located on PPCDC, NLRC5, STAM2, and TNFSF13B genes, and the expression of these genes significantly changed in BC tissues than normal tissues (P ≤ 0.05), also showing gene-gene correlation (P ≤ 0.05). Conclusions: These genetic variants and their associated novel haplotypic structures can predict adverse response to chemotherapy in BC patients and could potentially form BC-associated genetic panel for adverse response to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f6f813884867edf6f2b79ca7de2ac3b63826948" target='_blank'>
              Novel genetic structures associated with adverse response to chemotherapy in breast cancer.
              </a>
            </td>
          <td>
            Morteza Gholami, Mohsen Asouri, A. Ahmadi, Mehrab Nasirikenari
          </td>
          <td>2024-08-28</td>
          <td>Journal of oncology pharmacy practice : official publication of the International Society of Oncology Pharmacy Practitioners</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c25293ccda82c6d001fe5bae96c805b5e1fa7f9a" target='_blank'>
              The role of DNA polymerase I in tolerating single-strand breaks generated at clustered DNA damage in Escherichia coli
              </a>
            </td>
          <td>
            N. Shikazono, Ken Akamatsu
          </td>
          <td>2024-08-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a hereditary blood disorder characterized by the presence of abnormal hemoglobin molecules and thus distortion (sickling) of the red blood cells. SCD causes chronic pain and organ damage and shortens life expectancy. Gene therapy emerges as a potentially curative approach for people with SCD who lack a matched sibling donor for hematopoietic stem cell transplantation. Here, we review recent progress in gene therapy for SCD and focus on innovative technologies that target the genetic roots of the disease. We also review the challenges associated with gene therapy, including oncogenic risks, and the need for refined delivery methods. Despite these hurdles, the rapidly evolving landscape of gene therapy for SCD raises hope for a paradigm shift in the treatment of this debilitating disease. As research progresses, a deeper understanding of the molecular mechanisms involved and continuous improvements in gene-editing technologies promise to bring gene therapy for SCD closer to mainstream clinical application, offering a transformative, curative option for patients with this genetic disorder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6e785d7cbf41248336e372c5a5fb5eb1bf4e0c" target='_blank'>
              Treating Sickle Cell Disease: Gene Therapy Approaches.
              </a>
            </td>
          <td>
            M. Cavazzana, Alice Corsia, Mégane Brusson, A. Miccio, Michaela Semeraro
          </td>
          <td>2024-09-11</td>
          <td>Annual review of pharmacology and toxicology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Near-haploidization, that is, loss of one copy of most chromosomes, is a relatively rare phenomenon in most tumors, but is enriched among certain soft tissue sarcomas, including undifferentiated pleomorphic sarcoma (UPS). Presumably, near-haploidization can arise through many mechanisms. This study aimed to identify gene rearrangements that could cause near-haploidization. We here present two UPS in which near-haploidization was an early event, identified through single nucleotide polymorphism (SNP) array analysis. One of the cases was studied further using whole genome and transcriptome sequencing, as well as cytogenetic and molecular cytogenetic methods. Both tumors had chromosomal rearrangements in the form of copy number shifts/structural variants affecting the SMC1A gene. These findings suggest that cohesin defects could contribute to mitotic errors resulting in massive loss of chromosomes. SMC1A encodes one of the components of the cohesin multiprotein complex, which is critical for proper alignment of the sister chromatids during S-phase and separation to opposite spindle poles. Further studies should explore the role of cohesin defects in near-haploidization in other sarcomas and to clarify its role in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac46389944e17adf0a38d1c92f41322a72ad7ff" target='_blank'>
              Structural Variants in the SMC1A Gene Associated With Near-Haploidy in Undifferentiated Pleomorphic Sarcomas.
              </a>
            </td>
          <td>
            Sebastian Ibstedt, Paul Piccinelli, Saskia Sydow, Jan Köster, Fredrik Mertens
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dca29cb8a3526737bad31e766fb17c818ee6f4c" target='_blank'>
              Significance of targeting DNMT3A mutations in AML.
              </a>
            </td>
          <td>
            Guiqin Huang, Xiaoya Cai, Dengju Li
          </td>
          <td>2024-07-30</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e7718e7c250fe926c8cdf48ae45f72605c99d8" target='_blank'>
              REXO5 promotes genomic integrity through regulating R-loop using its exonuclease activity.
              </a>
            </td>
          <td>
            Ye Jin Lee, Seo Yun Lee, Soomi Kim, Soomi Kim, Soo Hyeon Lee, Sungho Park, Jae Jin Kim, Dong-Wook Kim, Hongtae Kim
          </td>
          <td>2024-07-30</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/694fb0c278abd1957ad9c10855875c5ac27794f3" target='_blank'>
              PIM1 is a potential therapeutic target for the leukemogenic effects mediated by JAK/STAT pathway mutations in T-ALL/LBL
              </a>
            </td>
          <td>
            Antonio Lahera, Laura Vela-Martín, Pablo Fernández-Navarro, Pilar Llamas, J. López-Lorenzo, Javier Cornago, Javier Santos, J. Fernández-Piqueras, M. Villa-Morales
          </td>
          <td>2024-07-20</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01413a6e42d02eb690fa59aa79101fc898cc9619" target='_blank'>
              RNA sequestration in P-bodies sustains myeloid leukaemia.
              </a>
            </td>
          <td>
            Srikanth Kodali, Ludovica Proietti, Gemma Valcárcel, Anna V. López-Rubio, Patrizia Pessina, Thomas Eder, Junchao Shi, Annie Jen, Núria Lupión-Garcia, Anne C. Starner, Mason Bartels, Yingzhi Cui, Caroline M Sands, Ainoa Planas-Riverola, Alba Martínez, T. Velasco-Hernández, Laureano Tomás-Daza, Bernhard Alber, G. Manhart, I. Mayer, K. Kollmann, A. Fatica, Pablo Menendez, Evgenia Shishkova, Rachel E Rau, B. Javierre, Joshua J. Coon, Qi Chen, Eric L. Van Nostrand, J. L. Sardina, F. Grebien, Bruno Di Stefano
          </td>
          <td>2024-08-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/315c73079e1d95c257190420cacdda2569b17936" target='_blank'>
              RNA Editing by ADAR Adenosine Deaminases in the Cell Models of CAG Repeat Expansion Diseases: Significant Effect of Differentiation from Stem Cells into Brain Organoids in the Absence of Substantial Influence of CAG Repeats on the Level of Editing.
              </a>
            </td>
          <td>
            Viacheslav V. Kudriavskii, A. O. Goncharov, A. V. Eremeev, Evgenii S. Ruchko, V. Veselovsky, Ksenia M Klimina, A. Bogomazova, M. Lagarkova, Sergei A. Moshkovskii, A. A. Kliuchnikova
          </td>
          <td>2024-08-01</td>
          <td>Biochemistry. Biokhimiia</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mutations in the male germline are a driving force behind rare genetic diseases. Driver mutations enjoying a selective advantage expand to mutant clusters within the aged testis and are thus overrepresented in sperm with age. Other kinds of driver mutations, occurring prepubescently, have been the focus of recent attention given their high occurrence rate independent of age. Here, we investigated the ErbB2 gene via error-corrected sequencing and detected a high percentage of missense mutations, including recurrent mutations, which were observed mainly in the tyrosine kinase domain with likely functional consequences, as we verified for a subset via biophysical methods. While these mutations increased with age, we found no evidence that they originated from mutational clusters in the aged testis, and young donors also showed an accumulation of driver mutations, suggesting that mutational enrichment is not exclusive to the sexually mature germline but can occur earlier during germline development, and are likely evenly distributed in the testis and stable in size.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa9f975fbf6b3e62de966f4ae5746c898526f4b3" target='_blank'>
              Mutations in ErbB2 accumulating in the male germline measured by error-corrected sequencing
              </a>
            </td>
          <td>
            Atena Yasari, Monika Heinzl, Theresa Mair, Tina Karimian, Shehab Moukbel Ali Aldawla, Ingrid Hartl, Andrea J. Betancourt, Peter Lanzerstorfer, Irene Tiemann-Boege
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Gastric cancer (GC) remains a significant public health concern because of its lethality, underscoring the need for deeper insights into its molecular mechanisms. Recent studies have increasingly highlighted the role of epigenetic modifications as critical players in cancer progression. Despite their importance, research specifically addressing epigenetic factors in GC is relatively scarce. This paper seeks to bridge that gap by examining recent literature that elucidates the epigenetic landscape associated with GC. The investigation of long noncoding RNAs (lncRNAs) has revealed their substantial involvement in gene dysregulation and epigenetic alterations within GC tumors. Notably, lncRNAs such as LINC00853 and LINC01266 have been identified as significant contributors to the epigenetic modulation of gene expression. Furthermore, the overexpression of KAT5 and GPX4 has been shown to mitigate the antiproliferative effects resulting from the depletion of circRHOT1, suggesting a complex interplay between these molecules in GC pathophysiology. Another pivotal aspect of epigenetic regulation in GC involves modifications in N6-methyladenosine (m6A), which play crucial roles in mRNA maturation processes such as splicing, export, degradation, and translation. m6A modifications are known for their influence on various cancer-related pathways, thus presenting a potential avenue for targeted interventions. Our findings indicate that the most pronounced instances of epigenetic dysregulation in GC can be traced back to the effects of long lncRNAs and alterations in m6A modification patterns. This underscores the urgent need for comprehensive investigations into these epigenetic factors, as a deeper understanding could lead to enhanced diagnostic markers and innovative therapeutic strategies. The integration of genetic and epigenetic considerations is essential for advancing the field of GC research. This synthesis of recent findings concerning epigenetic regulation offers valuable insights that could inform future studies and therapeutic developments. There is a critical need for ongoing research to elucidate the complexities of epigenetic modifications in GC, ultimately improving patient outcomes through tailored interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a765b9c07208b042a592cb41a6d3a0677c91a431" target='_blank'>
              Epigenetic Regulation of DNA Methylation and RNA Interference in Gastric Cancer: A 2024 Update
              </a>
            </td>
          <td>
            I. Lupan, Vasile Bintintan, D. Deleanu, G. Samașca
          </td>
          <td>2024-09-03</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, Jens Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Genome sequencing has revealed frequent mutations in Ras homolog family member A (RHOA) among various cancers with unique aberrant profiles and pathogenic effects, especially in peripheral T-cell lymphoma (PTCL). The discrete positional distribution and types of RHOA amino acid substitutions vary according to the tumor type, thereby leading to different functional and biological properties, which provide new insight into the molecular pathogenesis and potential targeted therapies for various tumors. However, the similarities and discrepancies in characteristics of RHOA mutations among various histologic subtypes of PTCL have not been fully elucidated. Herein we highlight the inconsistencies and complexities of the type and location of RHOA mutations and demonstrate the contribution of RHOA variants to the pathogenesis of PTCL by combining epigenetic abnormalities and activating multiple downstream pathways. The promising potential of targeting RHOA as a therapeutic modality is also outlined. This review provides new insight in the field of personalized medicine to improve the clinical outcomes for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd71cf13763be0cc23a4dc23d32cf828f50620b5" target='_blank'>
              Mutations in Ras homolog family member A in patients with peripheral T-cell lymphoma and implications for personalized medicine.
              </a>
            </td>
          <td>
            Lina Hu, Xuanye Zhang, Shengbing Zang
          </td>
          <td>2024-08-09</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56053b18f2967a281eeeeebbf2213c5fdde7cd2a" target='_blank'>
              Embryonic genome instability upon DNA replication timing program emergence.
              </a>
            </td>
          <td>
            Saori Takahashi, Hirohisa Kyogoku, Takuya Hayakawa, Hisashi Miura, Asami Oji, Yoshiko Kondo, S. Takebayashi, Tomoya S. Kitajima, I. Hiratani
          </td>
          <td>2024-08-28</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Menin inhibitors that disrupt Menin-MLL interaction hold promise for treating specific acute myeloid leukemia subtypes, including KMT2A rearrangements (KMT2A-r), yet resistance remains a challenge. Here, through systematic chromatin-focused CRISPR screens, along with genetic, epigenetic, and pharmacologic studies in a variety of human and mouse KMT2A-r AML models, we uncover a potential resistance mechanism independent of canonical Menin-MLL targets. We show that a group of non-canonical Menin targets, which are bivalently co-occupied by active Menin and repressive H2AK119ub marks, are typically downregulated following Menin inhibition. The loss of Polycomb Repressive Complex 1.1 (PRC1.1) subunits, such as PCGF1 or BCOR, leads to Menin inhibitor resistance by epigenetic reactivation of these non-canonical targets, including MYC. Genetic and pharmacological inhibition of MYC can resensitize PRC1.1-deficent leukemia cells to Menin inhibition. Moreover, we demonstrate that leukemia cells with the loss of PRC1.1 subunits exhibit reduced monocytic gene signatures and are susceptible to the BCL2 inhibition, and combinational treatment of venetoclax overcomes the resistance to Menin inhibition in PRC1.1-deficient leukemia cells. These findings highlight the important roles of PRC1.1 and its regulated non-canonical Menin targets in modulating Menin inhibitor response and provide potential strategies to treat leukemias with compromised PRC1.1 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fba94eb1d6d5cb6e0a8de6969fbdc894d28222" target='_blank'>
              Epigenetic Regulation of Non-canonical Menin Targets Modulates Menin Inhibitor Response in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Xinyue Zhou, Lixia Zhang, Sajesan Aryal, Virginia Veasey, Amanda Tajik, Cecilia Restelli, Steven Moreira, Pengcheng Zhang, Yanfeng Zhang, Kristin J Hope, Yang Zhou, Changde Cheng, Ravi Bhatia, Rui Lu
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40fcdfbbc2df4270401ef8beed0b78736d658d46" target='_blank'>
              DNA methylation in mammalian development and disease.
              </a>
            </td>
          <td>
            Zachary D. Smith, Sara Hetzel, Alexander Meissner
          </td>
          <td>2024-08-12</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="SF3B1 is the most recurrently mutated RNA splicing gene in cancer; However, the study of its pathogenic role has been hindered by a lack of disease-relevant cell line models. Here, we compared four genome engineering platforms to establish SF3B1 mutant cell lines: CRISPR-Cas9 editing, AAV HDR editing, base editing (ABEmax, ABE8e), and prime editing (PE2, PE3, PE5Max). We showed that prime editing via PE5max achieved the most efficient SF3B1 K700E editing across a wide range of cell lines. We further refined our approach by coupling prime editing with a fluorescent reporter that leverages a SF3B1 mutation-responsive synthetic intron to mark successfully edited cells. By applying this approach, called prime editing coupled intron-assisted selection (PRECIS), we introduced the K700E hotspot mutation into two chronic lymphocytic leukemia (CLL) cell lines, HG-3 and MEC-1. We demonstrated that our PRECIS-engineered cells faithfully recapitulate known mutant SF3B1 phenotypes including altered splicing, copy number variations, and cell growth defect. Moreover, we uncovered that SF3B1 mutation can cause the loss of Y chromosome in CLL. Our results showcase PRECIS as an efficient and generalizable method for engineering genetically faithful SF3B1 mutant models. Our approach provides new insights on the role of SF3B1 mutation in cancer and enables the generation of SF3B1 mutant cell lines in relevant cellular context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca0b3ac1820b19171ca9e6d9370b57efca6b03" target='_blank'>
              Engineering oncogenic hotspot mutations on SF3B1 via CRISPR-directed PRECIS mutagenesis.
              </a>
            </td>
          <td>
            Mikel Fernandez, Lei Yu, Qiong Jia, Xuesong Wang, Kevyn L Hart, Zhenyu Jia, Ren-Jang Lin, Lili Wang
          </td>
          <td>2024-08-28</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent advances in ''omics'' technologies allow for the identification of an increasing number of individuals with diseases caused by nano-rare mutations. These difficult-to-diagnose individuals are uniquely disadvantaged and pose significant challenges to healthcare systems and society. Despite having diseases caused by actionable single gene mutations, in many cases, there is no commercial path for treatments for such small patient populations. Since antisense oligonucleotide (ASO) technology has proven to be suited to address the needs of a portion of these patients, the n-Lorem Foundation is establishing an industrialized approach that couples detailed genotypic and phenotypic data to the immediate potential for ASO therapy. In this manuscript we have leveraged our experience in assessing the causality of nano-rare genetic variants and associated proximal molecular pathological events to attempt a correlation between detailed genetic data with patient specific phenotypic observations in 173 nano-rare individuals from diverse age groups evaluated for experimental ASO therapy. We found that the time required to achieve a molecular diagnosis varies from 1 month to 36 years, with the mean and median times from symptom onset to diagnosis estimated to be 4.32 years and 2 years, respectively. Amongst submitted cases there is a significant bias toward neurological diseases, with diverse genes and functional families involved and a marked preponderance of mutations in ion channel genes. The variability in phenotypic expression associated with nano-rare variants in genes such as GNAO1, H3F3A, GBE1, UBTF, or PACS1 clearly supports previous observations that phenotypes associated with same variants in the same gene can vary. We also observe that different, but functionally equivalent variants can result in both similar (e.g., TARDBP) and different phenotypes (e.g., GNAO1). Despite the relatively small size of the patient population investigated, this first compilation of its kind allows a variety of insights into the genotype and phenotype relationships in nano-rare conditions. Moreover, we show that our unique patient population presents a remarkable opportunity to apply ''modern omics'' approaches to begin to understand the various homeostatic, compensatory, and secondary effects of these genetic variants on the networks that result in expression of their unique phenotypes. To provide a more detailed description of the processes involved to provide a personalized antisense medicine, we have included nonclinical and clincal data on four exemplary patients who display disease in three different organs, the CNS, the eye and the kidney and are treated with ASOs of different designs. In contrast to traditional drug development, each patient presents unique genomic, ASO design, clinical treatment and management and evaluation challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d6386d30baeff2ca96c279e928ec4153f9291f" target='_blank'>
              Genotypic and phenotypic analysis of 173 patients with extremely rare pathogenic mutations who applied for experimental antisense oligonucleotide treatment
              </a>
            </td>
          <td>
            S. Crooke, T. A. Cole, J. B. Carroll, J. G. Gleeson, L. Mignon, J. Douville, W. Chung, J. Bain, E. M. Berry-Kravis, N. Leung, S. Demarest, E. McCourt, A. Watt, B. Powers, C. Ngongang
          </td>
          <td>2024-08-06</td>
          <td>None</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="ABSTRACT The size and composition of the intracellular DNA precursor pool is integral to the maintenance of genome stability, and this relationship is fundamental to our understanding of cancer. Key aspects of carcinogenesis, including elevated mutation rates and induction of certain types of DNA damage in cancer cells, can be linked to disturbances in deoxynucleoside triphosphate (dNTP) pools. Furthermore, our approaches to treat cancer heavily exploit the metabolic interplay between the DNA and the dNTP pool, with a long-standing example being the use of antimetabolite-based cancer therapies, and this strategy continues to show promise with the development of new targeted therapies. In this Review, we compile the current knowledge on both the causes and consequences of dNTP pool perturbations in cancer cells, together with their impact on genome stability. We outline several outstanding questions remaining in the field, such as the role of dNTP catabolism in genome stability and the consequences of dNTP pool expansion. Importantly, we detail how our mechanistic understanding of these processes can be utilised with the aim of providing better informed treatment options to patients with cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489114e77099687246712fffa583dd2206547a4f" target='_blank'>
              Understanding the interplay between dNTP metabolism and genome stability in cancer
              </a>
            </td>
          <td>
            Miriam Yagüe-Capilla, S. G. Rudd
          </td>
          <td>2024-08-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The liver is an essential organ of the body that performs several vital functions, including the metabolism of biomolecules, foreign substances, and toxins, and the production of plasma proteins, such as coagulation factors. There are hundreds of genetic disorders affecting liver functions and, for many of them, the only curative option is orthotopic liver transplantation, which nevertheless entails many risks and long-term complications. Some peculiar features of the liver, such as its large blood flow supply and the tolerogenic immune environment, make it an attractive target for in vivo gene therapy approaches. In recent years, several genome-editing tools mainly based on the clustered regularly interspaced short palindromic repeats associated protein 9 (CRISPR-Cas9) system have been successfully exploited in the context of liver-directed preclinical or clinical therapeutic applications. These include gene knock-out, knock-in, activation, interference, or base and prime editing approaches. Despite many achievements, important challenges still need to be addressed to broaden clinical applications, such as the optimization of the delivery methods, the improvement of the editing efficiency, and the risk of on-target or off-target unwanted effects and chromosomal rearrangements. In this review, we highlight the latest progress in the development of in vivo liver-targeted genome editing approaches for the treatment of genetic disorders. We describe the technological advancements that are currently under investigation, the challenges to overcome for clinical applicability, and the future perspectives of this technology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b703470ab6abddf03d9c85c71a3b2d8e4d148e9" target='_blank'>
              In vivo liver targeted genome editing as therapeutic approach: progresses and challenges
              </a>
            </td>
          <td>
            Chiara Simoni, Elena Barbon, Andrés F. Muro, Alessio Cantore
          </td>
          <td>2024-08-23</td>
          <td>Frontiers in Genome Editing</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8787bff41d07bad7870367afd2e6ff19a0d0ec2d" target='_blank'>
              CDC27 gene expression patterns as a potential biomarker in Acute Leukemia.
              </a>
            </td>
          <td>
            Yasaman Pouriafar, S. Rostami, Nasrin Alizadghandforoush, Mahmood Barati, Ali Amini, Majid Safa
          </td>
          <td>2024-07-29</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Rhabdoid tumors (RTs) are rare, highly aggressive pediatric malignancies which occur in kidney, soft-parts and brain. They are characterized by a complete inactivation of the SMARCB1 tumor suppressor gene encoding a core subunit of the chromatin remodeling SWI/SNF complex. Prognosis for children with RTs is poor as, in many instances, these tumors are resistant to conventional type chemotherapy. Pharmacological inhibition of EZH2 is a promising strategy to treat some tumors with loss of function in SMARCB1. However, the clinical response rate remains low, and both primary and secondary resistance have been reported. Understanding the mechanisms of resistance to EZH2 inhibition may allow new therapeutic hypothesis. For this purpose, we realized a genome-wide CRISPR-Cas9 knockout screening on two rhabdoid cell lines treated with EZH2 inhibitor. We identified TP53 loss as the sole consistent gene knock-out able to confer some resistance to EZH2 inhibition in both cell lines. Conversely, we demonstrated that MDM2/MDM4 inhibition, in a TP53-dependant manner, strongly synergized with EZH2 inhibition to control cell lines viability in vitro. We further treated 4 rhabdoid patient-derived xenografts with UNC1999, Idasanutlin and the combination of both and observed a potent tumor growth control upon combined EZH2 and MDM2/MDM4 inhibitions. To conclude, our results strongly encourage to assess the actual efficacy of combined MDM2/MDM4 and EZH2 inhibitors in the treatment of patients with SMARCB1-deficient rhabdoid tumors.
 Citation Format: Céline Chauvin, Tiphaine Hery, Rachida Bouarich, Fariba Nemati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, Sergio Roman-Roman, Didier Decaudin, Raphaël Margueron, Franck Bourdeaut. Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A051.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85bf34da504cd580597c4be8039c1663b824b215" target='_blank'>
              Abstract A051 Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors
              </a>
            </td>
          <td>
            C. Chauvin, Tiphaine Héry, Rachida Bouarich, F. Némati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, , Sakina Zaidi, Didier Surdez, S. Roman-Roman, Didier Decaudin, Raphaël Margueron, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/955437c70eeda854f01b1ed4043fef915e6b9a78" target='_blank'>
              Context-aware single-cell multiomics approach identifies cell-type-specific lung cancer susceptibility genes
              </a>
            </td>
          <td>
            Erping Long, Jinhu Yin, Ju Hye Shin, Yuyan Li, Bolun Li, Alexander Kane, Harsh Patel, Xinti Sun, Cong Wang, Thong Luong, Jun Xia, Younghun Han, J. Byun, Tongwu Zhang, Wei Zhao, M. T. Landi, Nathaniel Rothman, Qi Lan, Yoon Soo Chang, Fulong Yu, C. Amos, Jianxin Shi, Jin Gu Lee, Eun Young Kim, Jiyeon Choi
          </td>
          <td>2024-09-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="In recent decades, the field of genetics has revolutionized our understanding of biology. In particular, we know about how genetic information encodes the blueprint for life and how variations and mutations in these sequences contribute to disease or health. What is unique is that the genetic trait changes do not always caused by any changes in genetic sequences. 
Epigenetics refers to the study of how environmental factors and lifestyle choices can affect the gene activity. Changes in the heritable traits of organisms can be caused by modification of gene expression rather than alteration to the underlying DNA sequence. Therefore, this change does not involve any DNA mutation. 
DNA methylation is a biological process of adding a methyl group to the DNA molecule. The activity of DNA segments can change without changing the sequence by the process of methylation. This process plays an important role in normal development and aberrant DNA methylation patterns are associated with various diseases. Histones are highly basic proteins around which DNA is wound. The modifications in histone protein plays a fundamental role in most biological processes and can alter the accessibility of the DNA for transcription, as well as influencing gene expression. There are non-coding RNA molecules that do not encode proteins but play vital role in regulating the gene expression by interacting with other molecules and influencing chromatin structure. 
Over the past few decades, mechanisms of epigenetics have emerged as a crucial factor in governing every biological process in the human body. Throughout human life, these mechanisms have been impacted various physiological and pathological conditions. For example, cancer is often characterized by epigenetics abnormalities, such as hypermethylation of tumor suppressor genes or hypomethylation of oncogenes. Epigenetic modifications are also implicated in complex diseases like cardiovascular disease, autoimmune conditions and neurodegenerative disorder. Beyond these diseases, epigenetics also offers insights into health and development. Furthermore, understanding the epigenetic mechanisms influence human health, leading to new preventive strategies and treatment of various diseases. Early life experiences, including nutrition and stress, can lead to epigenetic changes that affect long-term health outcomes of the individual.   
Advances in this research can pinpoint any epigenetic changes that affect long-term health outcomes. The advances of epigenetic research can lead to early diagnosis and preventive measures. Identifying the epigenetic markers associated with the diseases enable early detection of the diseases and more effective preventative strategies. Some epigenetic therapies like; drugs that modify histone acetylation or DNA methylation, are already in development and provide future treatments for disease. To understand how life style factors, influence epigenetic regulation can lead to personalized lifestyle recommendations that influence the public health strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6963fc644883e01d78b4f2a627be037841fc025" target='_blank'>
              The Role of Epigenetics in Disease and Health: Beyond DNA Sequences
              </a>
            </td>
          <td>
            .. Diki
          </td>
          <td>2024-08-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Desmoplastic small round cell tumor (DSRCT) is a highly aggressive cancer predominantly occurring in male adolescents and young adults. The lack of a comprehensive understanding on the biology of the disease is paralleled by its dismal survival rates (5–20%). To overcome this challenge, we first identified and prioritized urgently needed resources for clinicians and researchers. Thus, we established genome-wide single-cell RNA-sequencing and bulk proteomic data of in vitro and in vivo-generated knockdown models of the pathognomonic DSRCT fusion oncoprotein (EWSR1::WT1) and combined them with an original systems-biology-based pipeline including patient data and the largest histology collection of DSRCTs and morphological mimics available to date. These novel tools were enriched with curated public datasets including patient- and cell line-derived ChIP-seq, bulk and single-cell RNA-seq studies resulting in a multi-model and multi-omic toolbox for discovery analyses. As a proof of concept, our approach revealed the alpha-2/delta subunit of the voltage-dependent calcium channel complex, CACNA2D2, as a highly overexpressed, super-enhancer driven, direct target of EWSR1::WT1. Single-cell and bulk-level analyses of patient samples and xenografted cell lines highlighted CACNA2D2 as a critical component of our newly established EWSR1::WT1 oncogenic signature, that can be employed to robustly identify DSRCT in reference sets. Finally, we show that CACNA2D2 is a highly sensitive and specific single biomarker for fast, simple, and cost-efficient diagnosis of DSRCT. Collectively, we establish a large-scale multi-omics dataset for this devastating disease and provide a blueprint of how such toolbox can be used to identify new and clinically relevant diagnostic markers, which may significantly reduce misdiagnoses, and thus improve patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1adb9c7dca4062ab1492b77bcd56eb7a98c207ea" target='_blank'>
              Super-enhancer-driven CACNA2D2 is an EWSR1::WT1 signature gene encoding a diagnostic marker for desmoplastic small round cell tumor (DSRCT)
              </a>
            </td>
          <td>
            Florian H. Geyer, Alina Ritter, Seneca S Kinn-Gurzo, Tobias Faehling, Jing Li, Armin Jarosch, Carine Ngo, Endrit Vinca, Karim Aljakouch, Azhar Orynbek, S. Ohmura, Thomas Kirchner, R. Imle, L. Romero-Pérez, Stefanie Bertram, Enrique de Álava, Sophie Postel-Vilnay, A. Banito, M. Sill, Y. Versleijen-Jonkers, Benjamin F B Mayer, Martin Ebinger, M. Sparber-Sauer, Sabine Stegmaier, D. Baumhoer, W. Hartmann, Jeroen Krijgsveld, David Horst, Olivier Delattre, P. Grohar, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2024-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Pathogenic variations in the fused in sarcoma (FUS) gene are associated with rare and aggressive forms of amyotrophic lateral sclerosis (ALS). As FUS-ALS is a dominant disease, a targeted, allele-selective approach to FUS knockdown is most suitable. Antisense oligonucleotides (AOs) are a promising therapeutic platform for treating such diseases. In this study, we have explored the potential for allele-selective knockdown of FUS. Gapmer-type AOs targeted to two common neutral polymorphisms in FUS were designed and evaluated in human fibroblasts. AOs had either methoxyethyl (MOE) or thiomorpholino (TMO) modifications. We found that the TMO modification improved allele selectivity and efficacy for the lead sequences when compared to the MOE counterparts. After TMO-modified gapmer knockdown of the target allele, up to 93% of FUS transcripts detected were from the non-target allele. Compared to MOE-modified AOs, the TMO-modified AOs also demonstrated reduced formation of structured nuclear inclusions and SFPQ aggregation that can be triggered by phosphorothioate-containing AOs. How overall length and gap length of the TMO-modified AOs affected allele selectivity, efficiency and off-target gene knockdown was also evaluated. We have shown that allele-selective knockdown of FUS may be a viable therapeutic strategy for treating FUS-ALS and demonstrated the benefits of the TMO modification for allele-selective applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40bef67258a89242d3fafd4ece2697b7aa372f10" target='_blank'>
              Allele-Selective Thiomorpholino Antisense Oligonucleotides as a Therapeutic Approach for Fused-in-Sarcoma Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            R. Mejzini, Marvin H. Caruthers, Balazs Schafer, Ondrej Kostov, Kavitha Sudheendran, Marija Ciba, Mathias Danielsen, Steve D. Wilton, P. A. Akkari, L. L. Flynn
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="INTRODUCTION
Leukaemia is the most common cancer in children, however, there is still a big gap in knowledge about the genomic alterations in childhood acute myeloid leukaemia (AML) compared to adult AML. Relapsed AML remains as a leading cause of cancer deaths among children. This study aims to understand the molecular mechanisms of relapsed AML by elucidating the mutational landscape before and during relapse.


MATERIALS AND METHODS
Whole genome sequencing was performed on matched samples collected at diagnosis, remission and relapse from three patients of de novo childhood AML. Sanger sequencing was performed for validation in 47 patients' samples, followed by functional analysis.


RESULTS
Overall, we identified 312 somatic mutations including synonymous single nucleotide variants (SNVs), missense SNVs, deletions and insertion frameshifts, stopgains and splice sites. After prioritisation, only 46 variants were present at diagnosis (13-17 mutations per patient) and 49 variants at relapse (12-20 mutations per patient). Out of 81 variants, there were 35 new variants detected at relapse but not present at diagnosis. Six potential driver mutations (KIT, CDC73, HNF1A, RBM10, ZMYM4 and ETV6) were identified in predicting relapse for the 3 patients, with recurrent mutations of the ETV6 gene in 2 patients. Functional analysis of the ETV6 mutation showed that ETV6 lost its tumour suppressive function when both mutant ETV6 p.P25fs and ETV6 p.N75fs were tested in vitro.


CONCLUSION
This study has uncovered the mutational landscape in three local childhood AML patients and contributes to a better understanding of the molecular mechanisms of relapsed AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe66d9e1447f89554dbfaa72d09784bc4662e53" target='_blank'>
              Whole genome sequencing reveals the mutational landscape from disease diagnosis to relapse in patients with childhood acute myeloid leukaemia.
              </a>
            </td>
          <td>
            H. Aziz, N. Ab Mutalib, H. Alias, R. Jamal
          </td>
          <td>2024-08-01</td>
          <td>The Malaysian journal of pathology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Pediatric cancers are estimated to be related to a genetic predisposition in about 8 to 10% of cases. This estimation mostly relies on already discovered cancer predisposing genes. However, the numerous new cancer predisposing genes that have been discovered in the past decade thanks to high throughput sequencing in vast cohort of pediatric patients illustrate that new genes may still be unravelled. Moreover, the frequency of early post-zygotic mosaicism is being readdressed thanks to more dedicated sequencing and research. The actual clinical impact of all those discoveries is the main challenge for the community: defining the penetrance, delineating the cancer spectrum and therefore the appropriate surveillance, and setting up innovative follow-up techniques are necessary. Here, we’ll describe recent findings on brain tumor predisposing genes (ELP1 and SMARCB1, among others) to illustrate the increasing recognition of genetic predisposition in pediatric cancers and how the clinical translation of these laboratory findings remain challenging at clinical and ethical levels.
 Citation Format: Franck Bourdeaut. Genetic predisposition to pediatric cancer: the long way from discovery to surveillance guidelines [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr IA003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc6b854ba212ba8f4846fc2365983567ae6c2486" target='_blank'>
              Abstract IA003: Genetic predisposition to pediatric cancer: the long way from discovery to surveillance guidelines
              </a>
            </td>
          <td>
            F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background and Objective Lung cancer stands as the main cause of cancer-related deaths worldwide. With the advent of immunotherapy and the discovery of targetable oncogenic driver genes, although prognosis has changed in the last few years, survival rates remain dismal for most patients. This emphasizes the urgent need for new strategies that could enhance treatment in precision medicine. The role of the microbiota in carcinogenesis constitutes an evolving landscape of which little is known. It has been suggested these microorganisms may influence in responses, resistance, and adverse effects to cancer treatments, particularly to immune checkpoint blockers. However, evidence on the impact of microbiota composition in oncogene-addicted tumors is lacking. This review aims to provide an overview of the relationship between microbiota, daily habits, the immune system, and oncogene-addicted tumors, focusing on lung cancer. Methods A PubMed and Google Scholar search from 2013 to 2024 was conducted. Relevant articles were reviewed in order to guide our research and generate hypothesis of clinical applicability. Key Content and Findings Microbiota is recognized to participate in immune reprogramming, fostering inflammatory, immunosuppressive, or anti-tumor responses. Therefore, identifying the microbiota that impact response to treatment and modulating its composition by interventions such as dietary modifications, probiotics or antibiotics, could potentially yield better outcomes for cancer patients. Additionally, targeted therapies that modulate molecular signaling pathways may impact both immunity and microbiota. Understanding this intricate interplay could unveil new therapeutic strategies. Conclusions By comprehending how microbiota may influence efficacy of targeted therapies, even though current evidence is scarce, we may generate interesting hypotheses that could improve clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c51e9b0c6cc73dbc9879341142a717c957faa91" target='_blank'>
              Oncogene-addicted solid tumors and microbiome—lung cancer as a main character: a narrative review
              </a>
            </td>
          <td>
            Mora Guardamagna, May-Lucie Meyer, M. Berciano-Guerrero, Andres Mesas-Ruiz, M. Cobo-Dols, Elisabeth Pérez-Ruiz, Alexandra Cantero González, R. Lavado-Valenzuela, Isabel Barragán, Javier Oliver, Alicia Garrido-Aranda, Martina Álvarez, A. Rueda-Domínguez, M. I. Queipo-Ortuño, Emilio Alba Conejo, J. C. Benítez
          </td>
          <td>2024-08-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Myelodysplastic syndrome and acute myeloid leukemia (AML) belong to a continuous disease spectrum of myeloid malignancies with poor prognosis in the relapsed/refractory setting necessitating novel therapies. Natural killer (NK) cells from patients with myeloid malignancies display global dysfunction with impaired killing capacity, altered metabolism, and an exhausted phenotype at the single-cell transcriptomic and proteomic levels. In this study, we identified that this dysfunction was mediated through a cross-talk between NK cells and myeloid blasts necessitating cell-cell contact. NK cell dysfunction could be prevented by targeting the αvβ-integrin/TGF-β/SMAD pathway but, once established, was persistent because of profound epigenetic reprogramming. We identified BATF as a core transcription factor and the main mediator of this NK cell dysfunction in AML. Mechanistically, we found that BATF was directly regulated and induced by SMAD2/3 and, in turn, bound to key genes related to NK cell exhaustion, such as HAVCR2, LAG3, TIGIT, and CTLA4. BATF deletion enhanced NK cell function against AML in vitro and in vivo. Collectively, our findings reveal a previously unidentified mechanism of NK immune evasion in AML manifested by epigenetic rewiring and inactivation of NK cells by myeloid blasts. This work highlights the importance of using healthy allogeneic NK cells as an adoptive cell therapy to treat patients with myeloid malignancies combined with strategies aimed at preventing the dysfunction by targeting the TGF-β pathway or BATF. BATF deletion prevents epigenetic rewiring and enhances the antitumor activity of NK cells in AML. Editor’s summary Patients with myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML), often relapse after standard of care. To improve outcomes, Kumar et al. have investigated the role of natural killer (NK) cells in antitumor immunity against myeloid blasts and have identified global dysfunction of NK cells in these patients. They identified BATF as responsible for this dysfunction, and by knocking out BATF downstream of the TGF-β canonical pathway in NK cells, they were able to enhance NK cell function against AML in vitro and in vivo. This has led to the initiation of a promising clinical trial using these TGFBR2 knockout NK cells against myeloid malignancies. —Dorothy Hallberg">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1bf3fa2449127d2f41a09515bf651d31d92c80" target='_blank'>
              BATF is a major driver of NK cell epigenetic reprogramming and dysfunction in AML
              </a>
            </td>
          <td>
            Bijender Kumar, Anand Singh, R. Başar, N. Uprety, Ye Li, Huihui Fan, Ana Karen Nunez Cortes, Mecit Kaplan, S. Acharya, Hila Shaim, Anna C Xu, Manrong Wu, Emily Ensley, Dexing Fang, P. Banerjee, L. M. Garcia, S. Tiberti, Paul Lin, H. Rafei, Maliha Nuzhat Munir, Madison Moore, M. Shanley, M. Mendt, L. Kerbauy, Bin Liu, Alexander Biederstädt, Elif Gokdemir, Susmita Ghosh, Kiran Kundu, Francia Reyes-Silva, X. Jiang, Xinhai Wan, April L. Gilbert, Merve Dede, V. Mohanty, J. Dou, Patrick Zhang, Enli Liu, Luis Muniz-Feliciano, Gary M. Deyter, Abhinav K. Jain, J. J. Rodríguez-Sevilla, S. Colla, Guillermo Garcia-Manero, Elizabeth J. Shpall, Ken Chen, Hussein A. Abbas, Kunal Rai, Katayoun Rezvani, M. Daher
          </td>
          <td>2024-09-11</td>
          <td>Science Translational Medicine</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT Changes in gene dosage can have tremendous evolutionary potential (e.g. whole-genome duplications), but without compensatory mechanisms, they can also lead to gene dysregulation and pathologies. Sex chromosomes are a paradigmatic example of naturally occurring gene dosage differences and their compensation. In species with chromosome-based sex determination, individuals within the same population necessarily show ‘natural’ differences in gene dosage for the sex chromosomes. In this Review, we focus on the mammalian X chromosome and discuss recent new insights into the dosage-compensation mechanisms that evolved along with the emergence of sex chromosomes, namely X-inactivation and X-upregulation. We also discuss the evolution of the genetic loci and molecular players involved, as well as the regulatory diversity and potentially different requirements for dosage compensation across mammalian species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016ee497a5dab2410dc213e6a7a7d311ff9fabfc" target='_blank'>
              Compensation of gene dosage on the mammalian X
              </a>
            </td>
          <td>
            Daniela Cecalev, Beatriz Viçoso, Rafael Galupa
          </td>
          <td>2024-08-01</td>
          <td>Development (Cambridge, England)</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Background: Transposable elements (TEs) are dynamic repetitive regions which generate mutations and structural variants. TP53 plays a crucial role in suppressing TE movement to maintain genomic stability. The relationship between TP53 and TEs has been extensively studied in tumours, but not the germline. Individuals with germline TP53 pathogenic variants have Li-Fraumeni Syndrome (LFS), a cancer predisposition syndrome with a high lifetime risk of cancer in various tissues. This study aims to characterize the TE landscape in individuals with Li-Fraumeni Syndrome and determine how this contributes to their increased cancer risk. Methods: MELT and xTea were used to identify TEs in children with (n=48) or without (n=198) a germline TP53 variant. TE calls were merged with SURVIVOR and variants were annotated with AnnotSV. To assess the influence of TP53 on TE location, we quantified TEs across genomic windows and identified the top 100 significantly different regions with the Mann-Whitney U test, adjusting for multiple comparisons. We used these regions to develop a gradient-boosted tree model with 5-fold cross-validation to predict TP53 status. Results: Individuals with germline TP53 variants harboured significantly fewer ALU and LINE1 elements in their germline genome compared to the control dataset (p<0.001). The phenomenon was consistent across chromosomes and significant in chromosomes 5 and 11 (FDR<0.05). A gradient-boosted tree model was able to differentiate patients with and without a germline TP53 variant with an AUPRC of 0.77 on an unseen test set. Conclusion: Germline TP53 variants may effect the frequency and location of germline TE insertions, which may influence nearby genomic variations and lead to cancer development. Analyzing TEs in individuals with LFS will enhance our understanding of accelerated cancer development in these patients, informing future research for diagnostic and therapeutic approaches.
 Citation Format: Brianne Laverty, Shilpa Yadahalli, Vallijah Subasri, David Malkin. The role of TP53 on transposable elements in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4995c29bd2050c705f9a5b8da916aacb0aa2601c" target='_blank'>
              Abstract A010 The role of TP53 on transposable elements in pediatric cancer
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Vallijah Subasri, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
FLT3 mutations are among the most common myeloid drivers identified in adult acute myeloid leukemia (AML). Their identification is crucial for the precise risk assessment because of the strong prognostic significance of the most recurrent type of FLT3 alterations, namely internal tandem duplications (ITDs). Recent advances in the pathogenesis and biology of FLT3-mutated AML have opened an opportunity for development and application of selective inhibition of FLT3 pathway.


RECENT FINDINGS
In the last decade, at least three targeted treatments have been approved by regulatory agencies and several others are currently under investigations. Here, we review the latest advance in the role of FLT3 mutations in AML, providing an outline of the available therapeutic strategies, their mechanisms of actions and of resistance, as well as routes for potential improvement.


SUMMARY
The availability of FLT3 inhibitors has improved outcomes in AML harboring such mutations, currently also reflected in disease stratification and recommendations. Newer inhibitors are under investigations, and combinations with chemotherapy or other targeted treatments are being explored to further improve disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732f19a10a5bb85b8ee615c65a99e7f2184b3ca7" target='_blank'>
              Advances in the pathogenesis of FLT3-mutated acute myeloid leukemia and targeted treatments.
              </a>
            </td>
          <td>
            S. Travaglini, C. Gurnari, T. Ottone, M. Voso
          </td>
          <td>2024-08-27</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma (HCC) is one of the most common malignant tumors of the digestive system. RNA methylation plays an important role in tumorigenesis and metastasis, which could alter gene expression and even function at multiple levels, such as RNA splicing, stability, translocation, and translation. In this study, we aimed to conduct a comprehensive analysis of RNA methylation-related genes (RMGs) in HCC and their relationship with survival and clinical features. Methods A retrospective analysis was performed using publicly available HCC-related datasets. The differentially expressed genes (DEGs) between HCC and controls were identified from TCGA-LlHC and intersected with RMGs to obtain differentially expressed RNA methylation-related genes (DERMGs). Regression analysis was used to screen for prognostic genes and construct risk models. Simultaneously, clinical, immune infiltration and therapeutic efficacy analyses were performed. Finally, multivariate cox regression was used to identify independent risk factors, and quantitative real-time polymerase chain reaction (qRT-PCR) was used to validate the expression levels of the core genes of the model. Results A 21-gene risk model for HCC was established with excellent performance based on ROC curves and survival analysis. Risk scores correlated with tumor grade, pathologic T, and TNM stage. Immune infiltration analysis showed correlations with immune scores, 11 immune cells, and 30 immune checkpoints. Low-risk patients showed a higher susceptibility to immunotherapy. The risk score and TNM stage were independent prognostic factors. qRT-PCR confirmed higher expression of PRDM9, ALPP, and GAD1 in HCC. Conclusions This study identified RNA methylation-related signature genes in HCC and constructed a risk model that predicts patient outcomes and reflects the immune microenvironment. Prognostic genes are involved in complex regulatory mechanisms, which may be useful for cancer diagnosis, prognosis, and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288e5a95cc7c2059fde0835f504837f6d0a5d61b" target='_blank'>
              A Novel RNA Methylation-Related Prognostic Signature and its Tumor Microenvironment Characterization in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Luzheng Liu, Jiacheng Chen, Fei Ye, Yanggang Yan, Yong Wang, Jincai Wu
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) maintain homeostasis in the hematopoietic ecosystem, which is tightly regulated at multiple layers. Acute myeloid leukemia (AML) is a severe hematologic malignancy driven by genetic and epigenetic changes that lead to the transformation of leukemia stem cells (LSCs). Since somatic mutations in DNA methylation-related genes frequently occur in AML, DNA methylation is widely altered and functions as a starting engine for initiating AML. Additionally, RNA modifications, especially N6-methyladenosine (m6A), also play an important role in the generation and maintenance of the hematopoietic ecosystem, and AML development requires reprogramming of m6A modifications to facilitate cells with hallmarks of cancer. Given the complex pathogenesis and poor prognosis of AML, it is important to fully understand its pathogenesis. Here, we mainly focus on DNA methylation and RNA m6A modification in hematopoiesis and AML and summarize recent advances in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92f8b80ecba345e57d84b39e3572cdbd8a3dd106" target='_blank'>
              Epigenetic modifications in hematopoietic ecosystem: a key tuner from homeostasis to acute myeloid leukemia
              </a>
            </td>
          <td>
            Shu-Qi Yao, Rongxia Guo, Wen Tian, Yanbing Zheng, Jin Hu, Guoqiang Han, Rong Yin, Fuling Zhou, Haojian Zhang
          </td>
          <td>2024-09-12</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND AND OBJECTIVE
The identification of mutation hot spots in the isocitrate dehydrogenase (IDH) genes is one of the most important cancer genome-wide sequencing discoveries with relevant impact in the treatment of some orphan tumors. These genes were mostly found mutated in lower-grade gliomas (LGGs), acute myeloid leukaemia (AML), myelodysplastic syndromes (MDS) and myeloproliferative neoplasms (MPNs) and in cholangiocarcinoma. This aberrant genomic condition represents a therapeutic target of great interest in cancer research, especially in AML, given the limitations of currently approved therapies in this field. In this review, we investigate the role of IDH mutation and the mutant IDH (mIDH)- targeted therapies for cholangiocarcinoma and glioma.


METHODS
Here, we provide an overview of the IDH mutation role and discuss its role in tumorigenesis and progression of some solid cancers, in which the therapeutic strategy can be completely changed thanks to these brand-new therapeutic options.


KEY CONTENT AND FINDINGS
The encouraging early clinical data demonstrated to be a proof of concept for investigational mIDH1/2 inhibitors in tumors with a paucity of therapeutic possibilities.


CONCLUSIONS
Moreover, we list the most important randomised clinical trials still active with their preliminary results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a20edde7b8a46ad9d9cedbb5e90b751d02d2ad2f" target='_blank'>
              Therapeutic inhibition of isocitrate dehydrogenase mutations in glioma and cholangiocarcinoma: new insights and promises-a narrative review.
              </a>
            </td>
          <td>
            V. Internò, A. Melaccio, Pasquale Vitale, Roberta Spedaliere, Massimo Buonfantino, R. Messina, Anna Laura Lippolis, Francesco Signorelli, Raffaele Addeo, Francesco Giuliani
          </td>
          <td>2023-01-01</td>
          <td>Chinese clinical oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background Current diagnostic tools are unable to distinguish low‐grade indolent prostate cancer (PrCa) from that with a propensity to become metastatic and/or lethal. Recent evidence suggests that reprogramming of the transcriptome may drive the metastatic phenotype, and that this reprogramming is controlled, at least in part, by epigenetic changes to the DNA of cancer cells, including methylation. These changes, referred to as ‘epigenetic drivers,’ have previously been associated with cancer cell survival. Methods Here, using Illumina Methylation EPIC array data of paired primary PrCa and metastatic bone samples, we identified WNT5A as a putative epi‐driver of PrCa metastasis to the bone, which was further validated in vitro. Results Significantly higher WNT5A methylation was observed in primary PrCa samples and 22Rv1 cells compared to metastatic bone samples and PC‐3 cells. This higher methylation was associated with significantly lower WNT5A gene expression. Conclusion Given the limited effective therapies available for metastatic cancer sufferers, particularly those whose disease has metastasised to the bone, WNT5A presents as a potential putative target for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b055e884a7d4d2969ce7c73d7f93b2c051be4c83" target='_blank'>
              WNT5A is a putative epi‐driver of prostate cancer metastasis to the bone
              </a>
            </td>
          <td>
            Emma J Wilkinson, Kelsie Raspin, R. Malley, S. Donovan, Louise M Nott, Adele F Holloway, Joanne L Dickinson
          </td>
          <td>2024-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e16f71ff68478a0f30f71105d86b66a3bf8d6b91" target='_blank'>
              Single-Cell Sequencing: High-Resolution Analysis of Cellular Heterogeneity in Autoimmune Diseases.
              </a>
            </td>
          <td>
            Xuening Tang, Yudi Zhang, Hao Zhang, Nan Zhang, Ziyu Dai, Q. Cheng, Yongzhen Li
          </td>
          <td>2024-08-26</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65c4eb6bdd3ce497963dc5d53f41cd09ae3cc914" target='_blank'>
              Engineering mouse cell fate controller by rational design
              </a>
            </td>
          <td>
            Tao Huang, Dong Liu, Xiaomin Wang, J. Kuang, Manqi Wu, Beibei Wang, Zechuan Liang, Yixin Fan, Bo Chen, Zhaoyi Ma, Yu Fu, Wenhui Zhang, Jin Ming, Yue Qin, Chengchen Zhao, Bo Wang, Duanqing Pei
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a5cebb3dacd4f74ab63e4dfea0d28d7bda5262" target='_blank'>
              N6-methyladenosine (m6A) RNA modification in chronic myeloid leukemia: unveiling a novel therapeutic target
              </a>
            </td>
          <td>
            Guillermo Fernandez Rodriguez, Marco Tarullo, A. Fatica
          </td>
          <td>2024-07-31</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Despite the availability of life-extending treatments for B cell leukemias and lymphomas, many of these cancers remain incurable. Thus, the development of new molecular targets and therapeutics is needed to expand treatment options. To identify new molecular targets, we used a forward genetic screen in mice to identify genes required for development or survival of lymphocytes. Here, we describe Zfp574, an essential gene encoding a zinc finger protein necessary for normal and malignant lymphocyte survival. We show that ZFP574 interacts with zinc finger protein THAP12 and promotes the G1-to-S-phase transition during cell cycle progression. Mutation of ZFP574 impairs nuclear localization of the ZFP574-THAP12 complex. ZFP574 or THAP12 deficiency results in cell cycle arrest and impaired lymphoproliferation. Germline mutation, acute gene deletion, or targeted degradation of ZFP574 suppressed Myc-driven B cell leukemia in mice, but normal B cells were largely spared, permitting long-term survival, whereas complete lethality was observed in control animals. Our findings support the identification of drugs targeting ZFP574-THAP12 as a unique strategy to treat B cell malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0480486c2217b6dc5e3cddbe10c5e1e4dd9761e0" target='_blank'>
              Disruption of the ZFP574-THAP12 complex suppresses B cell malignancies in mice.
              </a>
            </td>
          <td>
            Xue Zhong, James J Moresco, J. SoRelle, Ran Song, Yiao Jiang, Mylinh T Nguyen, Jianhui Wang, C. Bu, E. Moresco, Bruce Beutler, J. Choi
          </td>
          <td>2024-07-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cystic fibrosis (CF) is caused by mutations in the cystic fibrosis transmembrane conductance regulator (CFTR) gene. Although many people with CF (pwCF) are treated using CFTR modulators, some are non-responsive due to their genotype or other uncharacterized reasons. Autologous airway stem cell therapies, in which the CFTR cDNA has been replaced, may enable a durable therapy for all pwCF. Previously, CRISPR-Cas9 with two AAVs was used to sequentially insert two halves of the CFTR cDNA and an enrichment cassette into the CFTR locus. However, the editing efficiency was <10% and required enrichment to restore CFTR function. Further improvement in gene insertion may enhance cell therapy production. To improve CFTR cDNA insertion in human airway basal stem cells (ABCs), we evaluated the use of the small molecules AZD7648 and ART558 which inhibit non-homologous end joining (NHEJ) and micro-homology mediated end joining (MMEJ). Adding AZD7648 alone improved gene insertion by 2-3-fold. Adding both ART558 and AZD7648 improved gene insertion but induced toxicity. ABCs edited in the presence of AZD7648 produced differentiated airway epithelial sheets with restored CFTR function after enrichment. Adding AZD7648 did not increase off-target editing. Further studies are necessary to validate if AZD7648 treatment enriches cells with oncogenic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b76a5973a8822632977941f52ade0b5a68b63d15" target='_blank'>
              DNA-PKcs Inhibition Improves Sequential Gene Insertion of the Full-Length CFTR cDNA in Airway Stem Cells
              </a>
            </td>
          <td>
            Jacob T. Stack, R. Rayner, Reza Nouri, Carlos J. Suarez, Sun Hee Kim, K. Kanke, Tatyana A. Vetter, E. Cormet-Boyaka, S. Vaidyanathan
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Stratified Medicine Paediatrics (SMPaeds) is the UK precision medicine programme offering rapid turnaround sequencing with clinical reporting for patients with paediatric-type tumours at the time of relapse or disease progression. A defined aim of SMPaeds was to identify patients likely to respond to molecularly targeted therapies, and to discover genomic and epigenetic determinants of cancer relapse. Alongside the clinical programme, an archival tissue sample and a temporally matched plasma sample was obtained. Here we report profiling results from a large cohort of matched diagnostic-relapse tumour tissue (n=280) and matched tumour and cell free DNA (cfDNA) (n=402) pairs from patients with relapsed and progressive solid tumours of childhood. Serial tumour sequencing identified putative drivers of relapse - with alterations in CHEK2 and epigenetic drivers being a common feature across the whole cohort. Tissue and cfDNA sequencing results were concordant, with a wider spectrum of mutant alleles and higher degree of intra-tumour heterogeneity captured by the latter, if sufficient circulating tumour-derived DNA was present. In patients with neuroblastoma, who had the highest levels of circulating tumour derived DNA there was a high frequency of cfDNA unique mutations in genes known to be important in neuroblastoma biology, including TP53 and RAS pathway genes associated with ultra high-risk disease. Finally, in this heterogenous cohort, we asked if cfDNA analysis could be used to infer epigenetic characteristics relating to tissue of origin and cancer-specific expression signatures. Using low-coverage whole genome sequencing (lcWGS), we identified cfDNA fragmentation patterns by analysing differences in sequencing coverage between open and closed chromatin, then combined this with a database of binding sites for 425 transcription factors. This approach clearly identified cancer-specific chromatin accessibility, with specific transcription factor associated clusters relating to cell of origin identified in neuroblastoma, rhabdomyosarcoma, non-Hodgkin’s lymphoma and hepatoblastoma. Increased accessibility in cfDNA for experimentally verified core regulatory circuit (CRC) transcription factor binding sites was identified, including MYOD1 and MYOG in rhabdomyosarcoma and ASCL1, HAND2, ISL1, MYCN and TBX2 in neuroblastoma. In Wilms tumour, only one transcription factor was identified as highly specific: SIX2 - a known poor prognostic biomarker and blastemal subtype associated transcription factor. This demonstrates the potential of low-cost clinical cfDNA sequencing in multiple applications including supporting diagnosis, providing prognostic information, and understanding how therapies alter epigenetic programming. In summary, this study leverages a large and well-annotated genomic dataset of aggressive childhood malignancies, identifies genomic and epigenetic drivers of childhood cancer relapse, and highlights the power and practicality of cfDNA analysis to capture both intra-tumoral heterogeneity and the epigenetic state of cancer cells.
 Citation Format: Sally L. George, Claire Lynn, Reda Stankunaite, Debbie Hughes, Jane Chalker, Saira Waqar Ahmed, Minou Oostveen, Paula Proszek, Lina Yuan, Ridwan Shaikh, Sabri Jamal, Jennifer Tall, Janet Shipley, Deborah Tweddle, Lynley Marshall, Chris Jones, Susanne Gatz, Aditi Vedi, Paola Angelini, John Anderson, George Cresswell, Trevor A. Graham, Bissan Al-Lazikani, Pamela Kearns, J. Ciaran Hutchinson, Darren Hargrave, Thomas Jacques, Michael Hubank, Andrea Sottoriva, Louis Chesler. The Stratified Medicine Paediatrics programme for cancers of childhood: Cell free DNA and serial tumour sequencing identifies subtype specific evolution and epigenetic states [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A036.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8afd7fbd1e54bb5adce2e7ffb381373842943873" target='_blank'>
              Abstract A036 The Stratified Medicine Paediatrics programme for cancers of childhood: Cell free DNA and serial tumour sequencing identifies subtype specific evolution and epigenetic states
              </a>
            </td>
          <td>
            Sally L. George, Claire Lynn, Reda Stankunaite, Debbie Hughes, Jane Chalker, Saira W. Ahmed, M. Oostveen, P. Proszek, Lina Yuan, Ridwan Shaikh, S. Jamal, Jennifer R. Tall, Janet Shipley, Deborah Tweddle, L. Marshall, Chris Jones, Susanne Gatz, Aditi Vedi, Paola Angelini, John Anderson, George D. Cresswell, T. Graham, B. Al-Lazikani, Pamela Kearns, J. C. Hutchinson, Darren Hargrave, Thomas Jacques, Michael Hubank, A. Sottoriva, Louis Chesler
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a disease characterized by spatiotemporal heterogeneity of tumor clones. Different genetic aberrations can be observed simultaneously in tumor cells from different loci, and as the disease progresses, new subclones may appear. The role of liquid biopsy, which is based on the analysis of tumor DNA circulating in the blood plasma, continues to be explored in MM. Here, we present an analysis of the STR profiles and mutation status of the KRAS, NRAS, and BRAF genes, evaluated in plasma free circulating tumor DNA (ctDNA), CD138+ bone marrow cells, and plasmacytomas. The prospective single-center study included 97 patients, with a median age of 55 years. Of these, 94 had newly diagnosed symptomatic MM, and three had primary plasma cell leukemia. It should be noted that if mutations were detected only in ctDNA, “non-classical” codons were more often affected. A variety of adverse laboratory and clinical factors have been associated with the detection of rare KRAS or NRAS gene mutations in bone marrow or ctDNA, suggesting that these mutations may be factors of an unfavorable prognosis for MM. Liquid biopsy studies provide undeniable fundamental information about tumor heterogeneity and clonal evolution in MM. Moreover, we focus on using liquid biopsy to identify new high-risk factors for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef6c42652b61bb294400045b1d3bc2249d75f9b" target='_blank'>
              Loss of Heterozygosity and Mutations in the RAS-ERK Pathway Genes in Tumor Cells of Various Loci in Multiple Myeloma
              </a>
            </td>
          <td>
            M. Soloveva, M. Solovev, N. Risinskaya, E. Nikulina, I. Yakutik, B. Biderman, Tatiana Obukhova, Y. Chabaeva, Sergej Kulikov, Andrey Sudarikov, L. Mendeleeva
          </td>
          <td>2024-08-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Sarcomas constitute a large group of rare, heterogeneous cancers of the bone and soft tissue affecting children and young adults. There are over 80 clinically distinct subtypes of sarcoma, and their heterogeneity is a contributing factor to the moderate and poor survival outcomes observed in primary and metastatic disease, respectively. While they are heterogeneous, many subtypes harbor a balanced chromosomal translocation fusing the Ewing Sarcoma RNA binding protein 1 (EWSR1) gene with a gene encoding a transcription factor (TF). This translocation generates two distinct fusions: a canonical fusion consisting of the N-terminal transactivation domain of EWSR1 and the C-terminal DNA-binding domain of a TF, and a reciprocal fusion fusing the N-terminus of the TF and C-terminal RNA-binding domain of EWSR1. Canonical fusion protein products exhibit aberrant TF and pioneer factor activity, leading to alterations in transcriptional programs and the epigenetic landscape within a cell that contributes to oncogenesis. The functions of the reciprocal fusion proteins, however, have remained elusive. Ewing sarcoma (ES) and clear cell sarcoma (CCS), a bone and soft tissue sarcoma, respectively, are two subtypes that have been shown to express reciprocal fusions. The ES fusion implicates EWSR1 and Friend Leukemia Integration Factor 1 (FLI1), while the CCS fusion involves Activating Transcription Factor 1 (ATF1). A challenge in studying the roles of these and other fusion proteins in sarcoma development is the lack of effective model systems, primarily due to their toxicity upon exogenous introduction into fusion-naïve cells. Here, we outline the development of a cumate-inducible lentiviral vector to study the canonical and reciprocal EWSR1 fusion proteins present in ES and CCS. To track cellular localization of the fusion proteins and distinguish fusions from their respective wildtype proteins, we designed fusion sequences with N-terminal fluorescent tags. Tightly controlled expression of canonical and reciprocal fusion proteins reduces their toxicity when expressed in cells. We have successfully established 293T and HT-1080 cells that stably express these vectors. We have also validated cumate-inducible expression of fluorescently tagged canonical and reciprocal FLI1 and ATF1 fusions at the RNA and protein levels in both cell lines. The canonical fusion proteins produced by our model have the expected effects on expression levels of known downstream canonical fusion target genes. Initial cellular localization studies using our model and fluorescence microscopy indicate that reciprocal fusion proteins may localize in the nucleus, which was previously unknown. Overall, our cumate-inducible model represents a viable system for fusion protein expression. Using this model, we aim to gain insight into the roles of reciprocal fusions in ES and CCS development with hopes of uncovering potentially targetable vulnerabilities, opening the door for the development of new treatments in a patient population in need of novel and efficacious therapies.
 Citation Format: Sarah Gawlak, Joyce Ohm. Development of model systems to investigate the roles of canonical and reciprocal EWSR1 fusion proteins in sarcomagenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B079.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6104d5296aa07ecd63e21655a418e97199e7a8ba" target='_blank'>
              Abstract B079: Development of model systems to investigate the roles of canonical and reciprocal EWSR1 fusion proteins in sarcomagenesis
              </a>
            </td>
          <td>
            Sarah Gawlak, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>44</td>
        </tr>

        <tr id="Philadelphia chromosome-positive B-cell acute lymphoblastic leukemia (Ph(+)B-ALL) is a hematological malignancy with a poor prognosis. Epigenetic abnormalities, especially abnormal histone acetylation and microRNAs (miRNAs) dysregulation, are a group of epigenetic patterns that contribute to leukemia progression. However, their regulatory mechanisms in Ph(+)B-ALL have not been fully elucidated. In this study, we identified that miR-183-5p is significantly downregulated in Ph(+)B-ALL and associated with poor prognosis. Moreover, we found that the BCR-ABL fusion gene is a key target gene of miR-183-5p. MiR-183-5p directly targets BCR-ABL gene and induces cell apoptosis via PTEN/AKT and c-MYC signaling pathways. In addition, histone deacetylase inhibitor (HADCi) could mitigate the suppressive effects of HDAC2 on miR-183-5p by promoting promoter acetylation, thereby enhancing cell apoptosis. In conclusion, our results indicate that miR-183-5p is a potential biomarker and suggest that a novel "HDAC2-miR-183-5p epigenetic circuitry regulation" may be involved in the pathogenesis of Ph(+)B-ALL. Taken together, These findings provide new insights into the design of promising molecular-targeted drugs for Ph(+)B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d13162cad787b4066b1367ec7518bbaa77c34746" target='_blank'>
              HDAC2-miR183-5p Epigenetic Circuit Contributes to the Growth of Philadelphia Chromosome-Positive B-cell Acute Lymphoblastic Leukemia via PTEN/AKT and c-MYC Signaling Pathway.
              </a>
            </td>
          <td>
            Yangyang Ding, Xiangjiang Feng, Zelin Liu, Ya Liao, Lianfang Pu, Jun Liu, Huiping Wang, Zhimin Zhai, Shu-Dao Xiong
          </td>
          <td>2024-09-11</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 Fibrolamellar carcinoma (FLC) is characterized by a single genomic alteration, a 400 kB deletion resulting in the fusion transcript DNAJB1::PRKACA, which encodes a fusion oncoprotein essential for tumor initiation and maintenance. This study aims to find therapeutics that can kill FLC cells. We use three model systems: Patient tumors, fresh from resection, that we have made into organoids, that we have implanted into immune compromised mice (PDX), or we have screened directly, immediately after resection. We found lack of efficacy of agents current in the clinic. We have previously characterized the transcriptome of FLC and identified a number of oncogenic genes and pathways that are upregulated including the wnt pathway, EGF, and the wnt pathway. However, agents that blocked these had no effect on tumor survival. We switched to using three different approaches for therapy: i) A functional precision medicine screen using a drug-repurposing library; ii) antisense oligonucleotides against the RNA junction of DNAJB1::PRKACA transcript; iii) Degrader of the DNAJB1::PRKACA fusion protein. Functional precision medicine: We found a number of agents that were extremely efficacious. What they shared in common was pathways of metabolism of these drugs that were down-regulated in FLC. For example, irinotecan, a topoisomerase I inhibitor, was extremely effective. It is removed from liver cells through the addition of a sugar group by UGT1A1 which is decreased at the transcript and protein level. There were some variations in the extent to which patient tumors responded to irinotecan, but the variations could be eliminated by blocking the anti-apoptotic pathway Bcl-xL. We are currently preparing the combination of irinotecan and a PROTAC (proteolysis targeting chimeras) against Bcl-xL for a clinical trial. Antisense oligos: We created shRNA that tiled across the DNAJB1::PRKACA junction and identified some that eliminated DNAJB1::PRKACA at the RNA and protein level with no effect on DNAJB1 and no effect on PRKACA. When these were induced in FLC tumors cells grown as PDX, the tumors not only stopped growing, but shrank. This demonstrates that DNAJB1::PRKACA not only triggers FLC, but also continues to drive FLC and the FLC tumors are oncogenically addicted to DNAJB1::PRKACA. The same shRNA had no detectable effects on non-FLC liver tumors. We next tested siRNA against FLC grown as PDX. The efficacy of the siRNA were increased by conjugation to the sugar GalNAc which binds to the asialyoglycoprotein receptor on FLC cells. Degraders of the oncoprotein: We developed a degrader that selectively degraded the DNAJB1::PRKACA with no detectable effects on the wt PRKACA. This degrader effectively killed FLC tumors growing as PDX. Each of these three approaches represent emerging technologies for pediatric tumors. For each we now have a proof of principle, and our efforts are now focused on improving delivery and studies of efficacy and safety in the hope of moving these into the clinic to provide respite for this usually lethal childhood tumor.
 Citation Format: Mahsa Shirani, Michael Tomasini, Christoph Neumayer, Denise Ng, Gadi Lalazar, Bassem Shebl, Philip Coffino, Barbara A. Lyons, Sanford Simon. Emerging therapies for the treatment of the fusion protein driven cancer, fibrolamellar carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2958d83c3d5c2896a0eac3c8e8513329bef49b6" target='_blank'>
              Abstract A068 Emerging therapies for the treatment of the fusion protein driven cancer, fibrolamellar carcinoma
              </a>
            </td>
          <td>
            Mahsa Shirani, Michael D Tomasini, Christoph Neumayer, D. Ng, G. Lalazar, Bassem Shebl, Philip Coffino, Barbara Lyons, Sanford M Simon
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Gene therapy is being successfully developed for the treatment of several genetic disorders. Various methods of gene transfer have been developed to enable the production of the deficient enzyme or protein. One of the most important is adeno-associated virus vectors, which have been shown to be viable for use in in vivo gene therapy. Several gene therapies have already been approved. They are also promising for acquired diseases. Important examples include gene therapy for haemophilia A and B, X-linked myotubular myopathy, spinal muscular atrophy and several liver diseases such as Criggler-Najjar disease, alpha-1 antitrypsin deficiency and Fabry disease. However, the introduction of a foreign compound into hepatocytes leads to hepatic reactions with heterogeneous phenotypic expression and a wide spectrum of severity, ranging from mild transaminase elevation to acute liver failure. Several mechanisms appear to be involved in liver injury, including an immune response, but also direct toxicity depending on the method of gene transfer. As a result, the incidence, expression and severity of liver injury vary from indication to indication and from patient to patient. Patients treated for haemophilia A are more prone to transaminase elevation than those treated for haemophilia B. Corticosteroids are successfully used to correct liver reactions but also to prevent degradation of the transferred gene and loss of therapeutic activity. The aim of this review is to describe the risk of liver injury according to the indication for gene therapy and the short- and long-term management currently proposed to prevent or correct liver reactions in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d89d93ec8ec242d48ff2f6db70e44a383301bf5" target='_blank'>
              Drug-induced liver injury related to gene therapy: A new challenge to be managed.
              </a>
            </td>
          <td>
            Dominique Larrey, B. Delire, L. Meunier, Amel Zahhaf, Eleonora de Martin, Yves Horsmans
          </td>
          <td>2024-09-09</td>
          <td>Liver international : official journal of the International Association for the Study of the Liver</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Retrons are a retroelement class found in diverse prokaryotes that can be adapted to augment CRISPR-Cas9 genome engineering technology to efficiently rewrite short stretches of genetic information in bacteria and yeast; however, efficiency in human cells has been limited by unknown factors. We identified non-coding RNA (ncRNA) instability and impaired Cas9 activity as major contributors to poor retron editor efficiency. We re-engineered the Eco1 ncRNA to incorporate an exoribonuclease-resistant RNA pseudoknot from the Zika virus 3’ UTR and devised an RNA processing strategy using Csy4 ribonuclease to liberate the sgRNA and ncRNA. These modifications yielded a ncRNA with 5’- and 3’-end protection and an sgRNA with minimal 5’ extension. This strategy increased steady-state ncRNA levels and rescued Cas9 activity leading to enhanced efficiency of the Eco1 retron editor in human cells. The enhanced Eco1 retron editor enabled the insertion of missense mutations in human cells from a single integrated lentivirus, thereby ensuring genotype-phenotype linkage over multiple cell divisions. This work reveals a previously unappreciated role for ncRNA stability in retron editor efficiency in human cells. Here we present an enhanced Eco1 retron editor that enables efficient introduction of missense mutations in human cells from a single heritable genome copy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48b385cb15513c278a5a709586963aacd0c2ed6b" target='_blank'>
              An enhanced Eco1 retron editor enables precision genome engineering in human cells from a single-copy integrated lentivirus
              </a>
            </td>
          <td>
            Matthew A. Cattle, Lauren C. Aguado, Samantha Sze, Dylan Yueyang Wang, T. Papagiannakopoulos, Susan Smith, Charles M. Rice, William M. Schneider, John T. Poirier
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c47eca070ef4fc936f0fdb8bd32925e209619e" target='_blank'>
              Self-consistent signal transduction analysis for modeling context-specific signaling cascades and perturbations
              </a>
            </td>
          <td>
            John Cole
          </td>
          <td>2024-07-19</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Thymomas and thymic carcinomas (together referred to as thymic epithelial tumors, TETs) are a heterogeneous group of rare tumors of the anterior mediastinum. Most treatment regimens for advanced TETs are empirical and patients have not benefited from developments in personalized medicine. We believe that one of the major unresolved problems in TET research is the lack of suitable cellular or animal models that represent the full heterogeneity of TETs, help us to study the driving mechanisms behind TET biology, and explore vulnerabilities. In this narrative review, we discuss the current state of 2D and 3D TET cell culture models and protocols. To move beyond the current state, innovative high-throughput techniques are mandatory for both systematic optimization of cell culture conditions and high-throughput drug screening. Abstract Cell culture model systems are fundamental tools for studying cancer biology and identifying therapeutic vulnerabilities in a controlled environment. TET cells are notoriously difficult to culture, with only a few permanent cell lines available. The optimal conditions and requirements for the ex vivo establishment and permanent expansion of TET cells have not been systematically studied, and it is currently unknown whether different TET subtypes require different culture conditions or specific supplements. The few permanent cell lines available represent only type AB thymomas and thymic carcinomas, while attempts to propagate tumor cells derived from type B thymomas so far have been frustrated. It is conceivable that epithelial cells in type B thymomas are critically dependent on their interaction with immature T cells or their three-dimensional scaffold. Extensive studies leading to validated cell culture protocols would be highly desirable and a major advance in the field. Alternative methods such as tumor cell organoid models, patient-derived xenografts, or tissue slices have been sporadically used in TETs, but their specific contributions and advantages remain to be shown.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89ce3ee9aaf655d07e4e9a399628eb007a235cdd" target='_blank'>
              Cell Culture Models for Translational Research on Thymomas and Thymic Carcinomas: Current Status and Future Perspectives
              </a>
            </td>
          <td>
            Denise Müller, Jürgen Loskutov, Stefan Küffer, Alex Marx, C. Regenbrecht, Philipp Ströbel, Manuela J. Regenbrecht
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Aim: Breast cancer remains a significant cause of mortality worldwide, necessitating the development of innovative therapeutic approaches. Epigenetic and epitranscriptomic regulation have emerged as promising avenues for novel treatments. Sodium Butyrate (NaB) and Meclofenamic Acid (MFA) have gained attention for their respective roles in epigenetic and epitranscriptomic modulation. NaB, a histone deacetylase inhibitor, serves as a critical regulator of chromatin remodeling and gene expression. MFA has been identified to be a potent inhibitor of the FTO enzyme. This inhibitory potential marks its role in epitranscriptomic regulation. This study aimed to investigate the potential effects of MFA and NaB, individually and in combination, on the MCF7 breast cancer cell line.
Method: In order to investigate the cytotoxic and apoptotic effects of the combination treatment of MFA and NaB, cell viability assay, Annexin V analysis and Acridine Orange/DAPI staining were executed.
Results: The results revealed that the combination treatment unexpectedly exhibited antagonistic effects. This was evidenced by a remarkable increase in cell viability and a decreased apoptotic response compared to individual treatments. The strongest antagonistic effect was observed when the cells were treated with 100 μM MFA and 2 mM NaB for a period of 48 hours (CI = 88.3). 
Conclusion: This study, for the first time, sheds light on the complex interaction between meclofenamic acid and sodium butyrate that reveals an unexpected antagonistic effect on MCF7 breast cancer cells. These findings challenge conventional concepts of synergistic interactions and underscore the complexity of drug combinations in breast cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de13a198c5af52a5f49e57064408241f9c84f336" target='_blank'>
              The Impact of Simultaneous Epigenetic and Epitranscriptomic Intervention in Breast Cancer Cells
              </a>
            </td>
          <td>
            Sevinç Yanar, Asuman Deveci Özkan, Merve Gülşen Bal Albayrak, Zeynep Betts
          </td>
          <td>2024-08-31</td>
          <td>İstanbul Gelişim Üniversitesi Sağlık Bilimleri Dergisi</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: Significant progress has been made in identifying mutations and defining biological heterogeneity in neuroblastoma (NB) using high-throughput sequencing approaches. However, functional roles for many specific genetic alterations in NB initiation, growth, and metastasis remain to be fully defined. Pathogenic and/or likely pathogenic (P/LP) alterations in genes involved in DNA Damage Response (DDR) pathways were identified in approximately 30% of patients enrolled in the SickKids Cancer Sequencing Program (KiCS). Despite being well studied in many adult cancers, roles for DDR disruption and potential therapeutic implications remains poorly understood for NB and other pediatric solid tumors. Aims: We aimed to define roles for DDR-deficiency in NB progression and assess the therapeutic potential of targeted agents using patient-specific DDR loss-of-function NB models. Methods: We incorporated loss-of-function mutations in atm, brca2, palb2, and bard1 into an established zebrafish MYCN-driven model of human NB and characterized tumor progression, transcriptional signatures, and response to candidate therapeutics in vivo. Human cell lines with DDR protein knock-down were used to assess candidate drug combinations in vitro. Results: Mutations in DDR genes were found to enhance tumor penetrance and metastasis in MYCN-driven zebrafish NB, and resulted in upregulation of proliferation, cell cycle checkpoint, and DNA damage repair transcriptional signatures. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Conclusions: Our data supports an in vivo functional role for DDR-deficiency in NB, as well as potential molecular vulnerabilities. Pre-clinical assessments in vitro and in vivo suggest therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes.
 Citation Format: Madeline N. Hayes, Sarah Cohen-Gogo, Lynn Kee, Mehdi Layeghifard, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, Anita Villani, Adam Shlien, Daniel A. Morgenstern, Meredith S Irwin. DNA damage response deficiency enhances neuroblastoma progression and sensitivity to combination PARP and ATR inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cc0578406ca9a79efb4aae714485b02b1ea3da3" target='_blank'>
              Abstract A007 DNA damage response deficiency enhances neuroblastoma progression and sensitivity to combination PARP and ATR inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, M. Layeghifard, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The seven human APOBEC3 (hA3) genes encode polynucleotide cytidine deaminases that play vital roles in restricting replication of viruses and retrotransposons. However, off-target A3 deamination of the cellular genome is a major source of somatic mutations in human cancer. The ability to study A3 biology in vivo is hindered by the fact that the solitary murine Apobec3 gene (mA3) encodes a cytoplasmic enzyme, with no apparent mutagenic activity. Transgenic expression of individual hA3 genes in mice has helped to confirm their oncogenic potential but important questions including which hA3 genes are active in different tissue contexts and how they function in concert when under control of their cognate promoters cannot be addressed using these models. Here we describe humanization of the mouse mA3 locus by integration of a modified BAC clone encompassing the entire 7-gene hA3 locus from human chromosome 22 replacing mA3 on mouse chromosome 15. APOBEC3 mice are viable and fertile and hA3 gene expression in cells and tissues correlates strongly with expression in corresponding human cells and tissues, indicating human-like regulation of hA3 gene expression in the mice. Splenocytes from this line display a functional human A3 response to Friend Murine Leukaemia Virus (F-MLV) infection. We propose that the Hs-APOBEC3 mouse will uniquely model the function of the complete hA3 locus in a living organism and that it will serve as a useful background upon which to model human cancer, as well as assisting drug discovery efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8034f8597ff29dbab41cdb895f637bb191838ad2" target='_blank'>
              Human-like APOBEC3 gene expression and anti-viral responses following replacement of mouse Apobec3 with the 7-gene human APOBEC3 locus
              </a>
            </td>
          <td>
            Nerissa K. Kirkwood, H. Prosser, Melvyn W Yap, Jane Gibson, Ross Cook, Ian Reddin, Ane Stranger, Emma Howes, Nur Zainal, M. Periyasamy, S. Conticello, Gareth J. Thomas, James Scott, Kate N. Bishop, Simak Ali, Allan Bradley, Tim R. Fenton
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Colorectal cancer (CRC) ranks third in terms of cancer incidence worldwide and is responsible for 8% of all deaths globally. Approximately 10% of CRC cases are caused by inherited pathogenic mutations in driver genes involved in pathways that are crucial for CRC tumorigenesis and progression. These hereditary mutations significantly increase the risk of initial benign polyps or adenomas developing into cancer. In recent years, the rapid and accurate sequencing of CRC-specific multigene panels by next-generation sequencing (NGS) technologies has enabled the identification of several recurrent pathogenic variants with established functional consequences. In parallel, rare genetic variants that are not characterized and are, therefore, called variants of uncertain significance (VUSs) have also been detected. The classification of VUSs is a challenging task because each amino acid has specific biochemical properties and uniquely contributes to the structural stability and functional activity of proteins. In this scenario, the ability to computationally predict the effect of a VUS is crucial. In particular, in silico prediction methods can provide useful insights to assess the potential impact of a VUS and support additional clinical evaluation. This approach can further benefit from recent advances in artificial intelligence-based technologies. In this review, we describe the main in silico prediction tools that can be used to evaluate the structural and functional impact of VUSs and provide examples of their application in the analysis of gene variants involved in hereditary CRC syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c45c256e9f192af359d986d6c8e8e5f1f020547" target='_blank'>
              In Silico Deciphering of the Potential Impact of Variants of Uncertain Significance in Hereditary Colorectal Cancer Syndromes
              </a>
            </td>
          <td>
            C. Fasano, Martina Lepore Signorile, Katia De Marco, G. Forte, Vittoria Disciglio, P. Sanese, V. Grossi, Cristiano Simone
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4d41c964ff5dfe59fec9c54a2237fb6fabb8d4" target='_blank'>
              Telomerase related molecular subtype and risk model reveal immune activity and evaluate prognosis and immunotherapy response in prostate cancer
              </a>
            </td>
          <td>
            Dongze Liu, Zheng Qin, Bo Yi, Hongbo Xie, Yunan Liang, Liang Zhu, Kuo Yang, Yong Xu, Hongtuan Zhang
          </td>
          <td>2024-08-17</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Lentiviral vector-transduced T-cells were approved by the FDA as gene therapy anti-cancer medications. Little is known about the host genetic variation effects on the safety and efficacy of the lentiviral vector gene delivery system. To narrow this knowledge-gap, we characterized hepatic gene delivery by lentiviral vectors across the Collaborative Cross (CC) mouse genetic reference population. For 24 weeks, we periodically measured hepatic luciferase expression from lentiviral vectors in 41 CC mouse strains. Hepatic and splenic vector copy numbers were determined. We report that CC mouse strains showed highly diverse outcomes following lentiviral gene delivery. For the first time, moderate correlation between mouse strain-specific sleeping patterns and transduction efficiency was observed. We associated two quantitative trait loci (QTLs) with intra-strain variations in transduction phenotypes, which mechanistically relates to the phenomenon of metastable epialleles. An additional QTL was associated with the kinetics of hepatic transgene expression. Genes comprised in the above QTLs are potential targets to personalize gene therapy protocols. Importantly, we identified two mouse strains that open new directions in characterizing continuous viral vector silencing and HIV latency. Our findings suggest that wide-range patient-specific outcomes of viral vector-based gene therapy should be expected. Thus, novel escalating dose-based clinical protocols should be considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0029d66d0e5477dfaed003e789d5d1e323d24295" target='_blank'>
              Analysis of hepatic lentiviral vector transduction; implications for preclinical studies and clinical gene therapy protocols
              </a>
            </td>
          <td>
            Peirong Hu, Yajing Hao, Wei Tang, Graham H. Diering, Fei Zou, T. Kafri
          </td>
          <td>2024-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) stands as one of the most prevalent and aggressive forms of renal malignancies, accounting for a substantial portion of kidney cancer-related mortality worldwide. The heterogeneity within ccRCC poses a significant challenge to understanding its molecular mechanisms and devising effective therapeutic strategies. [1] Previous studies utilizing single nucleus RNA sequencing (snRNA-seq) have revealed transcriptional contributions to cell-type specificity in both mature human and mouse kidneys. Moreover, recent advancements have expanded this approach to single-cell profiling of chromatin accessibility. Single-nuclei assay for transposase-accessible chromatin using sequencing (snATAC-seq) has enabled the measurement of chromatin accessibility in thousands of individual cells, providing insights into the dynamic process that drives nephron development and differentiation. Integration and analysis of multimodal single-cell datasets, such as snRNA-seq and snATAC-seq, allows for the prediction of cell-type-specific cis-regulatory DNA interactions and transcription factor activity, complementing the transcriptional information obtained by snRNA-seq.[2] [3] [1] Yang, Wang, and Yang, “Treatment Strategies for Clear Cell Renal Cell Carcinoma.” [2] Muto et al., “Single Cell Transcriptional and Chromatin Accessibility Profiling Redefine Cellular Heterogeneity in the Adult Human Kidney.” [3] Monteagudo-Sánchez, Noordermeer, and Greenberg, “The Impact of DNA Methylation on CTCF-Mediated 3D Genome Organization.” Methods This study leveraged paired snRNA/ATAC-seq in tumor and normal-adjacent samples from the Renal Tumor Biobank at Dartmouth (n= 63) to explore the ccRCC landscape at single-cell resolution using 10x multiome technology. The snRNA-seq dataset was processed with Seurat v5.0 to remove low-quality nuclei. Doublets were removed with DoubletFinder v2.02, and ambient DNA was removed using decontX from celda. Normalization was conducted via SCTransform, and missing data was imputed with ALRA. Clustering was performed by constructing a KNN graph and running the Louvain algorithm. Cell type assignments were predicted using Azimuth with the human kidney reference dataset. The snATAC-seq data was processed with ArchR v1.0.1, which performed dimensionality reduction and clustering. Cluster identities defined in the Seurat workflow were cross-annotated to the ATAT-seq data, and open peaks were identified within these cell-type clusters using MACS3 v3.0.1. Identification of marker peaks, motif enrichment, and motif enrichment deviations were inferred utilizing ArchR’s getmarkerFeatures, peakAnnoEnrichment, and addMotifAnnotations functions, respectively. Results Motif enrichment analysis revealed canonical cancer-associated transcription factor (TF) dysregulation in tumor cells (nephron loop-like), including enrichment of HNF1B, SMARCC1, FOSL1, and JUND. Interestingly, HNF and SMARCC were also enriched in the lymphoid compartment, indicating possible crosstalk between these two cell types. Previous work has investigated the involvement of the HNF1 in ccRCC and in chromophobe RCC for prognosis,and we plan on investigating the relationship of these enrichments to tumor grade, stage and outcome.[1] Additional TF enrichments included BCL families and ELF within the myeloid compartment, and vascularization related TF families in endothelial cell types including enrichment of various ETS TFs. Investigation of cell-cell interactions between immune and endothelial cell populations as well as between tumor and normal adjacent tissues will be performed to further disentangle the role differentially accessable chromatin plays in ccRCC. [1] Buchner et al., “Downregulation of HNF-1B in Renal Cell Carcinoma Is Associated With Tumor Progression and Poor Prognosis.” Conclusions Our multi-omic approach has several advantages. First, it enables the characterization of cellular heterogeneity within the tumor microenvironment, identifying cell populations and their contributions to disease progression. Secondly, by integrating transcriptomic and epigenomic data, we gain insights into the dynamic interplay between gene expression and chromatin accessibility, uncovering potential therapeutic vulnerabilities and biomarkers for personalized treatment strategies. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e88d9a75048a69eb2afe5150d509757fff83b4" target='_blank'>
              60 Dissecting the Cellular and Epigenetic Landscape of Clear Cell Renal Cell Carcinoma Through Paired snRNA/ATAC-Seq Analysis
              </a>
            </td>
          <td>
            Ian Gingerich
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The AKT1 oncogene is related to various cancers due to its critical role in the PIC3CA/AKT1 pathway; however, most of the studies screened the hotspot mutation AKT1 (E17K) with various incidences. Low frequency or lack of AKT1 (E17K) mutation was reported in prostate cancer (PC) patients. This study aims to explore genetic alterations in the AKT1 PH domain by extending the sequencing to include AKT1 gene exons 3 and 4. Genomic DNA was extracted from 84 Formalin-Fixed Paraffin-Embedded samples of PC patients in Jordan, and then subjected to PCR and sequencing for the targeted exons. This study revealed the presence of two novel mutations (N53Y and Q59K) and a high frequency of mutations in exon 4, with a lack of mutations in the E17K hotspot. Nine missense and two synonymous mutations were detected in exon 4 (Phe27Tyr, Phe27Leu, Ala58Thr, Ser56Phe, Arg41Trp, Phe35Leu, Asp32Glu, Phe35Tyr, and Gln43Lys) and (Ser56 and Glu40), respectively. Two synonymous mutations were detected in exon 3 (Leu12 and Ser2). It is concluded that there is a high frequency of AKT1 mutation in PC patients in Jordan with two novel missense mutations in the Pleckstrin homology (PH) domain. E17K hotspot mutation was not detected in any tested samples, which underlined the significant role of mutations in other AKT1 exons in PC development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/205851db262e85bd8c99969aa7adb86b40d5b2f6" target='_blank'>
              Novel Mutations in AKT1 Gene in Prostate Cancer Patients in Jordan
              </a>
            </td>
          <td>
            A. Alasmar, Zina M. Al-Alami, S. Zein, Asmaa Al-Smadi, Samir Al Bashir, Mohammed S. Alorjani, R. Al‐Zoubi, M. A. Al Zoubi
          </td>
          <td>2024-09-04</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="DNA base lesions, such as incorporation of uracil into DNA or base mismatches, can be mutagenic and toxic to replicating cells. To discover factors in repair of genomic uracil, we performed a CRISPR knockout screen in the presence of floxuridine, a chemotherapeutic agent that incorporates uracil and fluoro-uracil into DNA. We identified known factors, such as uracil DNA N-glycosylase (UNG), but also unknown factors, such as the N6-adenosine methyltransferase, METTL3, as required to overcome floxuridine-driven cytotoxicity. Visualized with immunofluorescence, the product of METTL3 activity, N6-methyladenosine, formed nuclear foci in cells treated with floxuridine. The observed N6-methyladenosine was embedded in DNA, called 6mA, which was confirmed using mass spectrometry. METTL3 and 6mA were required for repair of lesions driven by additional base damaging agents, including raltitrexed, gemcitabine, and hydroxyurea. Our results establish a role for METTL3 and 6mA to promote genome stability in mammalian cells, specially in response to base damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db989576812beb4e3ed7a27aef984122d9c71b2" target='_blank'>
              N6-methyladenosine in DNA promotes genome stability
              </a>
            </td>
          <td>
            Brooke A. Conti, Leo Novikov, Deyan Tong, Qing Xiang, Savon Vigil, Thomas J. McLellan, Chuong Nguyen, Nancy De La Cruz, Reshma T. Veettil, Prashant Pradhan, Parag Sahasrabudhe, Jason D. Arroyo, Lei Shang, Benjamin R. Sabari, David J. Shields, Mariano Oppikofer
          </td>
          <td>2024-07-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Major epigenetic changes are associated with carcinogenesis, including aberrant DNA methylations and post-translational modifications of histone. Indeed evidence accumulated in recent years indicates that inactivating DNA hypermethylation preferentially targets the subset of polycomb group (PcG) genes that are regulators of developmental processes. Conversely, activating DNA hypomethylation targets oncogenic signaling pathway genes, but outcomes of both events lead in the overexpression of oncogenic signaling pathways that contribute to the stem-like state of cancer cells. On the basis of recent evidence from population-basedclinical and experimental studies, we hypothesize that factors associated with risk for developing a hematologic malignancy (HM), such as metabolic syndrome and chronic inflammation, may trigger epigenetic mechanisms to increase the transcriptional expression of oncogenes and activate oncogenic signaling pathways. Signaling pathways associated with such risk factors include but are not limited to pro-inflammatory nuclear factor κB (NF-κB) and mitogenic, growth, and survival Janus kinase (JAK) intracellular non-receptor tyrosine kinase-triggered pathways. The latter includes signaling pathways such as transducer and activator of transcription (STAT), Ras GTPases/mitogen-activated protein kinases (MAPKs)/extracellular signal-related kinases (ERKs), phosphatidylinositol 3-kinase (PI3K)/Akt/mammalian target of rapamycin (mTOR), and β-catenin pathways. Recent findings on epigenetic mechanisms at work in the biology of cancer and in HMs and their importance in the etiology and pathogenesis of these diseases are herein summarized and discussed. Furthermore, the role of epigenetic processes in the determination of biological identity, the consequences for interindividual variability in disease clinical profile, and the potential of epigenetic drugs in HMs are also considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135146cf67f22c155000b7b9f10163440a9d3db5" target='_blank'>
              Role of epigenetic in cancer biology, in hematologic malignancies and in anticancer therapy
              </a>
            </td>
          <td>
            A. H. Nwabo Kamdje, Hervet P Dongmo Fogang, Patrice N. Mimche
          </td>
          <td>2024-09-06</td>
          <td>Frontiers in Molecular Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer continues to present a substantial global health challenge, with its incidence and mortality rates persistently reflecting its significant impact. The emergence of precision oncology has provided a breakthrough in targeting oncogenic drivers previously deemed “undruggable” by conventional therapeutics and by limiting off-target cytotoxicity. Two groundbreaking technologies that have revolutionized the field of precision oncology are primarily CRISPR-Cas9 gene editing and more recently PROTAC (PROteolysis TArgeting Chimeras) targeted protein degradation technology. CRISPR-Cas9, in particular, has gained widespread recognition and acclaim due to its remarkable ability to modify DNA sequences precisely. Rather than editing the genetic code, PROTACs harness the ubiquitin proteasome degradation machinery to degrade proteins of interest selectively. Even though CRISPR-Cas9 and PROTAC technologies operate on different principles, they share a common goal of advancing precision oncology whereby both approaches have demonstrated remarkable potential in preclinical and promising data in clinical trials. CRISPR-Cas9 has demonstrated its clinical potential in this field due to its ability to modify genes directly and indirectly in a precise, efficient, reversible, adaptable, and tissue-specific manner, and its potential as a diagnostic tool. On the other hand, the ability to administer in low doses orally, broad targeting, tissue specificity, and controllability have reinforced the clinical potential of PROTAC. Thus, in the field of precision oncology, gene editing using CRISPR technology has revolutionized targeted interventions, while the emergence of PROTACs has further expanded the therapeutic landscape by enabling selective protein degradation. Rather than viewing them as mutually exclusive or competing methods in the field of precision oncology, their use is context-dependent (i.e., based on the molecular mechanisms of the disease) and they potentially could be used synergistically complementing the strengths of CRISPR and vice versa. Herein, we review the current status of CRISPR and PROTAC designs and their implications in the field of precision oncology in terms of clinical potential, clinical trial data, limitations, and compare their implications in precision clinical oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad1fa36c97b78fd15ef1e65afe80aa88f0e0443" target='_blank'>
              Precision oncology revolution: CRISPR-Cas9 and PROTAC technologies unleashed
              </a>
            </td>
          <td>
            Karim Kanbar, Roy El Darzi, Diana E. Jaalouk
          </td>
          <td>2024-07-31</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Triplex DNA formation is a useful genomic targeting tool that is expected to have a wide range of applications, including the antigene method; however, there are fundamental limitations in its forming sequence. We recently extended the triplex DNA-forming sequence to methylated DNA sequences containing 5mCG base pairs by developing guanidino-dN, which is capable of recognizing a 5mCG base pair with high affinity. We herein investigated the effect of triplex DNA formation using TFOs with guanidino-dN on methylated DNA sequences at the promoter of the RASSF1A gene, whose expression is epigenetically suppressed by DNA methylation in MCF-7 cells, on gene expression. Interestingly, triplex DNA formation increased the expression of the RASSF1A gene at the transcript and protein levels. Furthermore, RASSF1A-activated MCF-7 cells exhibited cell growth suppressing activity. Changes in the expression of various genes associated with the promotion of apoptosis and breast cancer survival accompanied the activation of RASSF1A in cells exhibited antiproliferative activity. These results suggest the potential of increases in gene expression through triplex DNA formation as a new genomic targeting tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/367d9640afcf3f528a05a1512247caf159507678" target='_blank'>
              Novel strategy for activating gene expression through triplex DNA formation targeting epigenetically suppressed genes
              </a>
            </td>
          <td>
            Ryotaro Notomi, Shigeki Sasaki, Yosuke Taniguchi
          </td>
          <td>2024-07-31</td>
          <td>RSC Chemical Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer treatment has long been fraught with challenges, including drug resistance, metastasis, and recurrence, making it one of the most difficult diseases to treat effectively. Traditional therapeutic approaches often fall short due to their inability to target cancer stem cells and the complex genetic and epigenetic landscape of tumors. In recent years, cancer immunotherapy has revolutionized the field, offering new hope and viable alternatives to conventional treatments. A particularly promising area of research focuses on non-coding RNAs (ncRNAs), especially long non-coding RNAs (lncRNAs), and their role in cancer resistance and the modulation of signaling pathways. To address these challenges, we performed a comprehensive review of recent studies on lncRNAs and their impact on cancer immunotherapy. Our review highlights the crucial roles that lncRNAs play in affecting both innate and adaptive immunity, thereby influencing the outcomes of cancer treatments. Key observations from our review indicate that lncRNAs can modify the tumor immune microenvironment, enhance immune cell infiltration, and regulate cytokine production, all of which contribute to tumor growth and resistance to therapies. These insights suggest that lncRNAs could serve as potential targets for precision medicine, opening up new avenues for developing more effective cancer immunotherapies. By compiling recent research on lncRNAs across various cancers, this review aims to shed light on their mechanisms within the tumor immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e39ed55d88fbecad6fec7ba9004faa9355e61712" target='_blank'>
              Exploring potential roles of long non-coding RNAs in cancer immunotherapy: a comprehensive review
              </a>
            </td>
          <td>
            A. Arshi, E. Mahmoudi, Farzaneh Raeisi, Masoud Dehghan Tezerjani, Elham Bahramian, Yeasin Ahmed, Chun Peng
          </td>
          <td>2024-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="CRISPR/Cas9 gene editing in the model green alga Chlamydomonas reinhardtii relies on the use of selective marker genes to enrich for non-selectable target mutations. This becomes challenging when many sequential modifications are required in a single cell line, as useful markers are limited. Here, we demonstrate a cyclical selection process which only requires a single marker gene to identify an almost infinite sequential series of CRISPR-based target gene modifications. We used the NIA1 (Nit1, NR; nitrate reductase) gene as the selectable marker in this study. In the forward stage of the cycle, a stop codon was engineered into the NIA1 gene at the CRISPR target location. Cells retaining the wild-type NIA1 gene were killed by chlorate, while NIA1 knockout mutants survived. In the reverse phase of the cycle, the stop codon engineered into the NIA1 gene during the forward phase was edited back to the wild-type sequence. Using nitrate as the sole nitrogen source, only the reverted wild-type cells survived. By using CRISPR to specifically deactivate and reactivate the NIA1 gene, a marker system was established that flipped back and forth between chlorate- and auxotrophic (nitrate)-based selection. This provided a scarless cyclical marker system that enabled an indefinite series of CRISPR edits in other, non-selectable genes. We demonstrate that this 'Sequential CRISPR via Recycling Endogenous Auxotrophic Markers (SCREAM)' technology enables an essentially limitless series of genetic modifications to be introduced into a single cell lineage of C. reinhardtii in a fast and efficient manner to complete complex genetic engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f75908314649d1188956bda26ea29417db966e6a" target='_blank'>
              A cyclical marker system enables indefinite series of oligonucleotide-directed gene editing in Chlamydomonas reinhardtii.
              </a>
            </td>
          <td>
            Ian L. Ross, Hong Phuong Le, Sabar Budiman, Dake Xiong, Fritz Hemker, Elizabeth A. Millen, Melanie Oey, Ben Hankamer
          </td>
          <td>2024-08-23</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af76c1aab1fec450dc5bad42237d6e428e7def0" target='_blank'>
              Applying whole-genome and whole-exome sequencing in breast cancer: a review of the landscape.
              </a>
            </td>
          <td>
            Hetvi Ganatra, J. K. Tan, Ana Simmons, Carola Maria Bigogno, Vatsala Khurana, Aruni Ghose, Adheesh Ghosh, Ishika Mahajan, S. Boussios, Akash Maniam, Olubukola Ayodele
          </td>
          <td>2024-08-27</td>
          <td>Breast cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The CYCS gene is highly evolutionarily conserved, with only a few pathogenic variants that cause thrombocytopenia-4 (THC4). Here, we report a novel CYCS variant NM_018947.6: c.59C>T [NP_061820.1:p.(Thr20Ile)] segregating with thrombocytopenia in three generations of a Czech family. The phenotype of the patients corresponds to THC4 with platelets of normal size and morphology and dominant inheritance. Intriguingly, a gradual decline in platelet counts was observed across generations. CRISPR/Cas9-mediated gene editing was used to introduce the new CYCS gene variant into a megakaryoblast cell line (MEG-01). Subsequently, the adhesion, shape, size, ploidy, viability, mitochondrial respiration, cytochrome c protein (CYCS) expression, cell surface antigen expression and caspase activity were analysed in cells carrying the studied variant. Interestingly, the variant decreases the expression of CYCS while increasing mitochondrial respiration and the expression of CD9 cell surface antigen. Surprisingly, the variant abates caspase activation, contrasting with previously known effects of other CYCS variants. Some reports indicate that caspases may be involved in thrombopoiesis; thus, the observed dysregulation of caspase activity might contribute to thrombocytopenia. The findings significantly enhance our understanding of the molecular mechanisms underlying inherited thrombocytopenia and may have implications for diagnosis, prognosis and future targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce3e9c59495e4ea384e724a498851b8202b3cc7" target='_blank'>
              A novel thrombocytopenia-4-causing CYCS gene variant decreases caspase activity: Three-generation study.
              </a>
            </td>
          <td>
            Jiří Štika, M. Pešová, K. S. Kozubík, Magdalena Skalníková, Lenka Dostálová, Tomáš Loja, Lenka Radova, Veronika Palušová, Kamila Reblova, Z. Vrzalova, I. Blaháková, J. Trizuljak, S. Uldrijan, Jan Blatný, Michal Šmída, Š. Pospíšilová, M. Doubek
          </td>
          <td>2024-08-27</td>
          <td>British journal of haematology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="High-throughput transcriptomics (HTTr) is increasingly being used to identify molecular targets of chemicals that can be linked to adverse outcomes. Cell proliferation is an important key event in chemical carcinogenesis. Here, we describe the construction and characterization of a gene expression biomarker that is predictive of the cell proliferation (CP) status in human and rodent tissues. The biomarker was constructed from 30 genes known to be increased in expression in prostate cancers relative to surrounding tissues and in cycling human MCF-7 cells after estrogen receptor (ER) agonist exposure. Using a large compendium of gene expression profiles to test utility, the biomarker could identify increases in cell proliferation in 1) 367 tumor vs. normal surrounding tissue comparisons from 6 human organs, 2) MCF-7 cells after activation of ER, 3) after partial hepatectomy in mice and rats, and 4) the livers of mice after exposure to nongenotoxic hepatocarcinogens. The biomarker identified suppression of CP 1) under conditions of p53 activation by DNA damaging agents in human cells, 2) in human A549 lung cells exposed to therapeutic anticancer kinase inhibitors (dasatinib, nilotnib) and 3) in the mouse liver when comparing high levels of CP at birth to the low background levels in the adult. The responses using the biomarker were similar to those observed using conventional markers of CP including PCNA, Ki67, and BrdU labeling. The CP biomarker will be a useful tool for interpretation of HTTr data streams to identify cell proliferation status after exposure to chemicals in human cells or in rodent tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e67eda2694923151226daf541a133cfb12edda84" target='_blank'>
              A transcriptomic biomarker predictive of cell proliferation for use in adverse outcome pathway-informed testing and assessment.
              </a>
            </td>
          <td>
            J. Corton, Victoria Ledbetter, Samuel M. Cohen, E. Atlas, Carole L Yauk, Jie Liu
          </td>
          <td>2024-08-13</td>
          <td>Toxicological sciences : an official journal of the Society of Toxicology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a2685872da8abf8d89c45e2e00bd648c1daaeb" target='_blank'>
              DNA methylation and gene expression as determinants of genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Michaël Noë, Dimitrios Mathios, A. Annapragada, Shashikant Koul, Zachariah H. Foda, J. Medina, S. Cristiano, Christopher Cherry, D. C. Bruhm, N. Niknafs, V. Adleff, Leonardo Ferreira, Hari Easwaran, Stephen Baylin, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="Enhancer elements are specific DNA sequences that play a crucial role in regulating gene expression. Located upstream or downstream in the genomic context, enhancers enhance the transcription of linked genes. This is achieved by providing binding sites for transcription factors and other proteins, acting as a nucleation point that promotes the assembly of the transcriptional machinery.
What makes enhancers unique is that located even thousands of base pairs away, they can exert their function. They can act over long distances, looping to interact with the promoter region of their target genes. Enhancers are often involved in cell type-specific gene expression as different cells express different sets of genes by activating and repressing cell type-specific enhancers.
Cancer cells seem to co-opt this mechanism and hijack it for their own survival. In this process, an aberrant enhancer element activates the transcription of oncogenes (genes that have the potential to cause cancer) due to some alterations in genomic structure or regulatory elements that happen in cancer, contributing to tumorigenesis and cancer progression. Enhancer hijacking often occurs through complex genomic rearrangements such as chromosomal translocations and other structural rearrangements such as deletions or amplifications.8 These alterations can bring a usually distant, far-located enhancer into proximity with an oncogene, leading to its inappropriate activation. This is often seen in various cancers, including leukemias and solid tumors.
Enhancer hijacking and its effects on gene regulation in cancer are frequently studied using methods such as RNA sequencing, chromatin conformation capture, and CRISPR-Cas9. Finding an enhancer hijacking event may be used to inform treatment choices, especially in precision medicine settings, and may also function as a biomarker for particular cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefe88575de8420490e0e1dd10a24dbefe2eef41" target='_blank'>
              Enhancer hijacking: Innovative ways of carcinogenesis
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-09-02</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1f5936217eac5b4043bb1665424b2339a9f20d" target='_blank'>
              MYC and KRAS cooperation: from historical challenges to therapeutic opportunities in cancer
              </a>
            </td>
          <td>
            S. Casacuberta-Serra, Íñigo González-Larreategui, Daniel Capitán-Leo, Laura Soucek
          </td>
          <td>2024-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Since the identification of DNA as the hereditary material, therapeutic advancements have focused on gene therapy (GT) and allogeneic hematopoietic cell transplants for inherited diseases. Despite successes in treating conditions like β-thalassemia and sickle cell disease, challenges such as insertional oncogenesis and immunogenicity persist. Future efforts must prioritize improving vector safety, refining manufacturing processes, and expanding access to GT to fully realize its potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9198c613dac012277fe80dd191ba49946328eeff" target='_blank'>
              Audacity of Gene Therapy.
              </a>
            </td>
          <td>
            Marketa Dimitrov, C. Ebens, Jakub Tolar
          </td>
          <td>2024-08-20</td>
          <td>The British journal of dermatology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Background Minichromosome maintenance complex component 3 associated protein (MCM3AP) is a gene in which mutations can result in autosomal recessive peripheral neuropathy with or without impaired intellectual development. The MCM3AP genotype-phenotype correlation and prognosis remain unclear. The aim of this study was to explore the genotype–phenotype correlations pertaining to MCM3AP. Methods Whole-exome sequencing (WES) combined with copy number variation sequencing (CNV-seq) were performed on the genomic DNA isolated from a Chinese family, and Sanger sequencing, quantitative PCR and cDNA analyses were performed to examine the mutations. The retrospective study was conducted on 28 individuals with biallelic MCM3AP mutation-related diseases, including features such as mutations, motor development impairment, intellectual disability, weakness/atrophy, and cerebral magnetic resonance imaging abnormalities. Results Sequencing identified novel compound heterozygous mutations in MCM3AP, namely, a paternal variant c.1_5426del (loss of exons 1–25) and a maternal splicing variant c.1858 + 3A>G. Functional studies revealed that the variant c.1858 + 3A>G resulted in the heterozygous deletion of exon 5, thereby affecting splicing functionality. Furthermore, the compound heterozygous mutation may affect the functionality of the protein domain. Retrospective analysis revealed different genotype–phenotype correlations for the pathogenic variants in biallelic MCM3AP: all individuals (100%) with mutations outside the Sac3 domain exhibited early-onset symptoms, motor developmental delays, and cognitive abnormalities, conversely, the proportions of individuals carrying mutations within the domain were 26.7% (motor delays) and 46.7% (cognitive abnormalities). Conclusion Our findings further expand the genetic mutation spectrum of biallelic MCM3AP and highlight the genotype-phenotype associations. Additionally, we elaborate on the importance of rehabilitation intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc14d90ab7ba7ceb9ba0f635353a560c5f6b553f" target='_blank'>
              Identification of biallelic mutations in MCM3AP and comprehensive literature analysis
              </a>
            </td>
          <td>
            Chan Liu, Qingfeng Xie, Quan Hu, Bingwu Xiang, Kaiyi Zhao, Xiang Chen, Feixia Zheng
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Metabolic changes during tumour development lead to acidification of the extracellular environment and a smaller increase of intracellular pH. Searches for somatic missense mutations that could reveal adaptation to altered pH have focussed on arginine to histidine changes, part of a general arginine depletion that originates from DNA mutational mechanisms. Analysis of mutations to histidine, potentially a simple route to the introduction of pH-sensing, shows no clear biophysical separation overall of subsets that are more and less frequently mutated in cancer genomes. Within the more frequently mutated subset, individual sites predicted to mediate pH-dependence upon mutation include NDST1 (a Golgi-resident heparan sulphate modifying enzyme), the HLA-C chain of MHCI complex, and the water channel AQP-7. Arginine depletion is a general feature that persists in the more frequently mutated subset, and is complemented by over-representation of mutations to lysine. Arginine to lysine balance is a known factor in determining protein solubility, with higher lysine content being more favourable. Proteins with greater change in arginine to lysine balance are enriched for cell periphery location, where proteostasis is likely to be challenged in tumour cells. Somatic missense mutations in a cancer genome number only in the 10s typically, although can be much higher. Whether the altered arginine to lysine balance is of sufficient scale to play a role in tumour development is unknown.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e382817ce528739f587257a2ec3f22edf1d558d1" target='_blank'>
              Computational investigation of missense somatic mutations in cancer and potential links to pH-dependence and proteostasis
              </a>
            </td>
          <td>
            Shalaw R. Sallah, J. Warwicker
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43402d24d8edf2c96b4d1c5838f23fbcb2866024" target='_blank'>
              Somatic and germline aberrations in homologous recombination repair genes among Chinese high-risk breast cancer patients by multi-gene next-generation sequencing.
              </a>
            </td>
          <td>
            Ling Xie, Jie Chen, Yanying Zheng, Yi Sun, Xiang Zhang, LeLe Chu, Yifen Zhang
          </td>
          <td>2024-07-24</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Disruption of the epigenome is a hallmark of human disease including liver cirrhosis and hepatocellular carcinoma (HCC). While genetic heterogeneity is an established effector of pathologic phenotypes, epigenetic heterogeneity is less well understood. Environmental exposures alter the liver-specific DNA methylation landscape and influence the onset of liver cancer. Given that currently available treatments are unable to target frequently mutated genes in HCC, there is an unmet need for novel therapeutics to prevent or reverse liver damage leading to hepatic tumorigenesis, which the epigenome may provide. We performed genome-wide profiling of DNA methylation, copy number, and gene expression from multiple liver regions from 31 patients with liver disease to examine their crosstalk and define the individual and combinatorial contribution of these processes to liver disease progression. We identify epigenetic heterogeneity hotspots that are conserved across patients. Elevated epigenetic heterogeneity associates with increased gene expression heterogeneity. Cirrhotic regions comprise two distinct cohorts, one exclusively epigenetic, and a second where epigenetic and copy number variations collaborate. Epigenetic heterogeneity hotspots are enriched for genes central to liver function (e.g., HNF1A) and known tumor suppressors (e.g., RASSF1A). These hotspots encompass genes including ACSL1, ACSL5, MAT1A, and ELFN1, which have phenotypic effects in functional screens, supporting their relevance to hepatocarcinogenesis. Moreover, epigenetic heterogeneity hotspots are linked to clinical measures of outcome.


CONCLUSION
Substantial epigenetic heterogeneity arises early in liver disease development, targeting key pathways in the progression and initiation of both cirrhosis and HCC. Integration of epigenetic and transcriptional heterogeneity unveils putative epigenetic regulators of hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46b475bb15d42990ae95e5c459fd1ea63a3f93cd" target='_blank'>
              Epigenetic heterogeneity hotspots in human liver disease progression.
              </a>
            </td>
          <td>
            R. Hlady, Xia Zhao, Louis El Khoury, Ryan T. Wagner, Aesis M. Luna, Kien Pham, Nikolaos Pyrosopoulos, Dhanpat Jain, Liguo Wang, Chen Liu, Keith D. Robertson
          </td>
          <td>2024-07-19</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37af8177474f5214e32408bc13029fb9fea726d5" target='_blank'>
              Gene therapy for ultrarare diseases: a geneticist’s perspective
              </a>
            </td>
          <td>
            W. Hwu
          </td>
          <td>2024-08-13</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b0e84c70be39b55c5ab372a108ddda6ab970765" target='_blank'>
              Polθ: emerging synthetic lethal partner in homologous recombination-deficient tumors.
              </a>
            </td>
          <td>
            T. B. Bazan Russo, Clarissa Mujacic, Emilia Di Giovanni, Maria Concetta Vitale, Carla Ferrante Bannera, U. Randazzo, Silvia Contino, M. Bono, V. Gristina, A. Galvano, A. Perez, G. Badalamenti, A. Russo, V. Bazan, L. Incorvaia
          </td>
          <td>2024-08-09</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, Roberto Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Canine gastrointestinal lymphoma is known to be of T-cell origin in most cases, but the molecular biological aberrations have not been clarified. In human intestinal T-cell lymphoma, the mutations in the genes associated with Janus kinase/signal transducer and activator of transcription (JAK-STAT) pathway have been frequently observed. In this study, the gene mutations were investigated in 31 dogs with large cell gastrointestinal lymphoma (LCGIL) by focusing on the genes involved in JAK-STAT pathway. Next-generation sequencing analysis to examine the mutations in STAT3, STAT5B, and JAK1 genes throughout the exon regions revealed the mutations in STAT3 gene in two dogs and JAK1 gene in one dog. In conclusion, this study could not indicate the associations of gene mutations in JAK-STAT pathway with LCGIL in most canine cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af29815f4def444284d27afce69a73b99fc5ca9d" target='_blank'>
              Investigation of the mutations in the genes involved in Janus kinase/signal transducer and activator of transcription pathway in canine large cell gastrointestinal lymphoma.
              </a>
            </td>
          <td>
            Takumi Tsuruta, Naoki Matsumura, Keijiro Mizukami, Y. Goto-Koshino, T. Aoi, Ryoko Yamada, Itsuma Nagao, Megumi Sakamoto, T. Nakagawa, Ray Fukuoka, A. Ohmi, J. Chambers, Kazuyuki Uchida, Y. Momozawa, H. Tomiyasu
          </td>
          <td>2024-08-13</td>
          <td>The Journal of veterinary medical science</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Reduced responsiveness of precursor B-acute lymphoblastic leukemia (BCP-ALL) to chemotherapy can be first detected in the form of minimal residual disease leukemia cells that persist after 28 days of initial treatment. The ability of these cells to resist chemotherapy is partly due to the microenvironment of the bone marrow, which promotes leukemia cell growth and provides protection, particularly under these conditions of stress. It is unknown if and how the glycocalyx of such cells is remodelled during the development of tolerance to drug treatment, even though glycosylation is the most abundant cell surface post-translational modification present on the plasma membrane. To investigate this, we performed omics analysis of BCP-ALL cells that survived a 30-day vincristine chemotherapy treatment while in co-culture with bone marrow stromal cells. Proteomics showed decreased levels of some metabolic enzymes. Overall glycocalyx changes included a shift from Core-2 to less complex Core-1 O-glycans, and reduced overall sialylation, with a shift from α2-6 to α2-3 linked Neu5Ac. Interestingly, there was a clear increase in bisecting complex N-glycans with a concomitant increased mRNA expression of MGAT3, the only enzyme known to form bisecting N-glycans. These small but reproducible quantitative differences suggest that individual glycoproteins become differentially glycosylated. Glycoproteomics confirmed glycosite-specific modulation of cell surface and lysosomal proteins in drug-tolerant BCP-ALL cells, including HLA-DRA, CD38, LAMP1 and PPT1. We conclude that drug-tolerant persister leukemia cells that grow under continuous chemotherapy stress have characteristic glycotraits that correlate with and perhaps contribute to their ability to survive and could be tested as neoantigens in drug-resistant leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43518aa376606ca1b8957a3131c117df65f4de6f" target='_blank'>
              Remodelling of the glycome of B-cell precursor acute lymphoblastic leukemia cells developing drug-tolerance
              </a>
            </td>
          <td>
            Tiago Oliveira, Mingfeng Zhang, Chun-Wei Chen, N. Packer, M. von Itzstein, Nora Heisterkamp, D. Kolarich
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa9a049feca8fb402096fb6d84d95a6531213f49" target='_blank'>
              CHO cell engineering via targeted integration of circular miR-21 decoy using CRISPR/RMCE hybrid system
              </a>
            </td>
          <td>
            Setare Adibzadeh, S. Amiri, F. Barkhordari, S. Mowla, Hadi Bayat, S. Ghanbari, F. Faghihi, Fatemeh Davami
          </td>
          <td>2024-08-09</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920a368f09f7a44e43a098b28f1b2509afbc2a36" target='_blank'>
              The genomic and immunogenomic landscape of mechanics pathway informs clinical prognosis and response to mechanotherapy.
              </a>
            </td>
          <td>
            Peidong Zhang, Peiwei Li, Muya Tang, Ryan C. Gimple, Liang Huang, Jing Yue, Qiuhong Shen, Yiwei Du, Qian Zhang, Zhengnan Yang, Haihuai He, Kailin Yang, Linjie Zhao, Shengtao Zhou
          </td>
          <td>2024-07-19</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc721cd925975049ceb0cd27f9ffa4efdce8611" target='_blank'>
              Synthetic lethality by PARP inhibitors: new mechanism uncovered based on unresolved transcription-replication conflicts
              </a>
            </td>
          <td>
            Marina Kolesnichenko, Claus Scheidereit
          </td>
          <td>2024-07-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/263cda8d76b200b36443f700782153fb43cb23ea" target='_blank'>
              Increased AID results in mutations at the CRLF2 locus implicated in Latin American ALL health disparities
              </a>
            </td>
          <td>
            Valeria Rangel, Jason N. Sterrenberg, Aya Garawi, Vyanka Mezcord, Melissa L. Folkerts, Sabrina E. Calderon, Yadhira E. Garcia, Jinglong Wang, Eli M. Soyfer, Oliver S Eng, Jennifer B. Valerin, S. Tanjasiri, Fabiola Quintero-Rivera, M. Seldin, Selma Masri, Richard L. Frock, Angela G. Fleischman, Nicholas R Pannunzio
          </td>
          <td>2024-07-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e1b808c0ad8ebc0432fdf9718775113703bf08" target='_blank'>
              Multi-regional genomic and transcriptomic characterization of a melanoma-associated oral cavity cancer provide evidence for CASP8 alteration-mediated field cancerization
              </a>
            </td>
          <td>
            Shouvik Chakravarty, Arnab Ghosh, Chitrarpita Das, Subrata Das, S. Patra, Arindam Maitra, S. Ghose, N. Biswas
          </td>
          <td>2024-09-07</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7401c81aea35658fe72a5961ba5c3e31cae1d2fa" target='_blank'>
              The genomic basis of childhood T-lineage acute lymphoblastic leukaemia.
              </a>
            </td>
          <td>
            P. Pölönen, Danika Di Giacomo, A. Seffernick, Abdelrahman A. Elsayed, Shunsuke Kimura, Francesca Benini, Lindsey E Montefiori, Brent L Wood, Jason Xu, Changya Chen, Zhong-Yi Cheng, H. Newman, Jason R Myers, Ilaria Iacobucci, Elizabeth Li, Jonathan H. Sussman, Dale Hedges, Yawei Hui, Caroline Diorio, Lahari Uppuluri, David Frank, Yiping Fan, Yunchao Chang, S. Meshinchi, R. Ries, R. Shraim, Alexander Li, Kathrin M Bernt, M. Devidas, S. Winter, K. Dunsmore, H. Inaba, William L Carroll, Nilsa C Ramirez, Aaron H. Phillips, Richard W. Kriwacki, Jun J Yang, T. Vincent, Yaqi Zhao, P. S. Ghate, Jian Wang, Colleen Reilly, Xin Zhou, Mathijs A. Sanders, Junko Takita, Motohiro Kato, Nao Takasugi, Bill H Chang, Richard D Press, M. Loh, E. Rampersaud, E. Raetz, S P Hunger, Kai Tan, Ti-Cheng Chang, Gang Wu, S. B. Pounds, C. Mullighan, D. Teachey
          </td>
          <td>2024-08-14</td>
          <td>Nature</td>
          <td>2</td>
          <td>102</td>
        </tr>

        <tr id="Longevity, the length of an organism's lifespan, is impacted by environmental factors, metabolic processes, and genetic determinants. The base excision repair (BER) pathway is crucial for maintaining genomic integrity by repairing oxidatively modified base lesions. Nei-like DNA Glycosylase 1 (NEIL1), part of the BER pathway, is vital in repairing oxidative bases in G-rich DNA regions, such as telomeres and promoters. Hence, in this comprehensive review, it have undertaken a meticulous investigation of the intricate association between NEIL1 and longevity. The analysis delves into the multifaceted aspects of the NEIL1 gene, its various RNA transcripts, and the diverse protein isoforms. In addition, a combination of bioinformatic analysis is conducted to identify NEIL1 mutations, transcription factors, and epigenetic modifications, as well as its lncRNA/pseudogene/circRNA-miRNA-mRNA regulatory network. The findings suggest that the normal function of NEIL1 is a significant factor in human health and longevity, with defects in NEIL1 potentially leading to various cancers and related syndromes, Alzheimer's disease, obesity, and diabetes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c7b949676b21576a435aa8624516850b916fe0" target='_blank'>
              Decoding the Role of NEIL1 Gene in DNA Repair and Lifespan: A Literature Review with Bioinformatics Analysis.
              </a>
            </td>
          <td>
            Amirhossein Mohajeri Khorasani, Alireza Raghibi, Behzad Haj Mohammad Hassani, Pedram Bolbolizadeh, A. Amali, Mahboubeh Sadeghi, Narges Farshidi, A. Dehghani, P. Mousavi
          </td>
          <td>2024-08-20</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Dyslipidemia, characterized by abnormal lipid levels in the blood, significantly escalates the risk of atherosclerotic cardiovascular disease and requires effective treatment strategies. While existing therapies can be effective, long-term adherence is often challenging. There has been an interest in developing enduring and more efficient solutions. In this context, gene editing, particularly clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) technology, emerges as a groundbreaking approach, offering potential long-term control of dyslipidemia by directly modifying gene expression. This review delves into the mechanistic insights of various gene-editing tools. We comprehensively analyze various pre-clinical and clinical studies, evaluating the safety, efficacy, and therapeutic implications of gene editing in dyslipidemia management. Key genetic targets, such as low-density lipoprotein receptor (LDLR), proprotein convertase subtilisin/kexin type 9 (PCSK9), angiopoietin-like protein 3 (ANGPTL3), apolipoprotein C3 (APOC3), and lipoprotein (a) (Lp(a)), known for their pivotal roles in lipid metabolism, are scrutinized. The paper highlights the promising outcomes of gene editing in achieving sustained lipid homeostasis, discusses the challenges and ethical considerations in genome editing, and envisions the future of gene therapy in revolutionizing dyslipidemia treatment and cardiovascular risk reduction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be2067ec1007c41b495bdef122d7e3e947027684" target='_blank'>
              Therapeutic Gene Editing in Dyslipidemias
              </a>
            </td>
          <td>
            Seyed Saeed Tamehri Zadeh, Michael D. Shapiro
          </td>
          <td>2024-08-01</td>
          <td>Reviews in Cardiovascular Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befebce4fc2c4b9cfeedafc2e4927bdb0544dbd9" target='_blank'>
              Mechanisms that clear mutations drive field cancerization in mammary tissue
              </a>
            </td>
          <td>
            Marta Ciwinska, H. Messal, Hristina R Hristova, C. Lutz, Laura Bornes, Theofilos Chalkiadakis, Rolf Harkes, Nathalia S M Langedijk, S. Hutten, Renée X Menezes, Jos Jonkers, Stefan Prekovic, Ben Simons, C. Scheele, J. van Rheenen
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1cc33bb53a8950fd830eced26d315b8502330bf" target='_blank'>
              A single-cell and spatially resolved atlas of human osteosarcomas
              </a>
            </td>
          <td>
            Xuejing Zheng, Xu Liu, Xin-xin Zhang, Zhen-guo Zhao, Wence Wu, Shengji Yu
          </td>
          <td>2024-08-20</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84febdf4c77dd510d0a0709248848e1311db0643" target='_blank'>
              Biological and clinical relevance of correlated expression levels of coding and long noncoding RNAs in HPV16 positive cervical cancers
              </a>
            </td>
          <td>
            Abhisikta Ghosh, Abarna Sinha, Arnab Ghosh, Somrita Roy, Sumana Mallick, Vinoth Kumar, S. Mathai, Jaydip Bhaumik, Asima Mukhopadhyay, Saugata Sen, Aditi Chandra, A. Maitra, N. Biswas, Partha P. Majumder, Sharmila Sengupta
          </td>
          <td>2024-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="By analyzing two large atlases of almost 4 million cells, we show that immune-senescence involves a gradual loss of cellular identity, reflecting increased cellular heterogeneity, for effector, and cytotoxic immune cells. The effects are largely similar in both males and females and were robustly reproduced in two atlases, one assembled from 35 diverse studies including 678 adults, the other the OneK1K study of 982 adults. Since the mean transcriptional differences among cell-types remain constant across age deciles, there is little evidence for the alternative mechanism of convergence of cell-type identity. Key pathways promoting activation and stemness are down-regulated in aged T cells, while CD8 TEM and CD4 CTLs exhibited elevated inflammatory, and cytotoxicity in older individuals. Elevated inflammatory signaling pathways, such as MAPK and TNF-alpha signaling via NF-kB, also occur across all aged immune cells, particularly amongst effector immune cells. This finding of lost transcriptional identity with age carries several implications, spanning from a fundamental biological understanding of aging mechanisms to clinical perspectives on the efficacy of immunomodulation in elderly people.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d768b64d5bd77b266a4e1004a43d2aff5288db6d" target='_blank'>
              Loss of immune cell identity with age inferred from large atlases of single cell transcriptomes.
              </a>
            </td>
          <td>
            Erin Connolly, Tony Pan, Maneesha Aluru, Sriram P. Chockalingam, Vishal Dhere, Greg Gibson
          </td>
          <td>2024-08-14</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>10</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 14],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>