<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-09 06:33:00 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>7</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Most human cancers are heterogeneous consisting of cancer cells at different epigenetic and transcriptional states and with distinct phenotypes, functions, and drug sensitivities. This inherent cancer cell heterogeneity contributes to tumor resistance to clinical treatment, especially the molecularly targeted therapies such as tyrosine kinase inhibitors (TKIs) and androgen receptor signaling inhibitors (ARSIs). Therapeutic interventions, in turn, induce lineage plasticity (also called lineage infidelity) in cancer cells that also drives therapy resistance. In this Perspective, we focus our discussions on cancer cell lineage plasticity manifested as treatment-induced switching of epithelial cancer cells to basal/stem-like, mesenchymal, and neural lineages. We employ prostate cancer (PCa) as the prime example to highlight ARSI-induced lineage plasticity during and towards development of castration-resistant PCa (CRPC). We further discuss how the tumor microenvironment (TME) influences therapy-induced lineage plasticity. Finally, we offer an updated summary on the regulators and mechanisms driving cancer cell lineage infidelity, which should be therapeutically targeted to extend the therapeutic window and improve patients’ survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ea1fde8339eba5fa332bd913bbc774ac173a97e" target='_blank'>
              Treatment-induced stemness and lineage plasticity in driving prostate cancer therapy resistance
              </a>
            </td>
          <td>
            Anmbreen Jamroze, Xiaozhuo Liu, Dean G. Tang
          </td>
          <td>2024-08-25</td>
          <td>Cancer Heterogeneity and Plasticity</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c3db9b2bc9f499477ff494631d94e30c054c0bf" target='_blank'>
              A Boolean model explains phenotypic plasticity changes underlying hepatic cancer stem cells emergence
              </a>
            </td>
          <td>
            Alexis Hernández-Magaña, Antonio Bensussen, Juan Carlos Martínez-García, E. Álvarez-Buylla
          </td>
          <td>2024-09-02</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Prostate cancer (PC) depicts a major health challenge all over the globe due to its complexities in the treatment and diverse clinical trajectories. Even in the advances in the modern treatment strategies, the spectrum of resistance to the therapies continues to be a significant challenge. This review comprehensively examines the underlying mechanisms of the therapy resistance occurred in PC, focusing on both the tumor microenvironment and the signaling pathways implicated in the resistance. Tumor microenvironment comprises of stromal and epithelial cells, which influences tumor growth, response to therapy and progression. Mechanisms such as microenvironmental epithelial-mesenchymal transition (EMT), anoikis suppression and stimulation of angiogenesis results in therapy resistance. Moreover, dysregulation of signaling pathways including androgen receptor (AR), mammalian target of rapamycin/phosphoinositide 3 kinase/AKT (mTOR/PI3K/AKT), DNA damage repair and Janus kinase/signal transducers and activators of transcription (JAK/STAT) pathways drive therapy resistance by promoting tumor survival and proliferation. Understanding these molecular pathways is important for developing targeted therapeutic interventions which overcomes resistance. In conclusion, a complete grasp of mechanisms and pathways underlying medication resistance in PC is important for the development of individualized treatment plans and enhancements of clinical outcomes. By studying and understanding the complex mechanisms of signaling pathways and microenvironmental factors contributing to therapy resistance, this study focuses and aims to guide the development of innovative therapeutic approaches to effectively overcome the PC progression and improve the survival rate of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27b4a025bb1904ed23371786225a1e4cfcd5778f" target='_blank'>
              Therapy resistance in prostate cancer: mechanism, signaling and reversal strategies
              </a>
            </td>
          <td>
            Neha Thakur, Pallavi Singh, Aditi Bagri, Saumya Srivastava, Vinay Dwivedi, Asha Singh, Sunil Kumar Jaiswal, S. Dholpuria
          </td>
          <td>2024-08-29</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer resistance refers to the ability of cancer cells to evade the effects of treatments, such as chemotherapy and radiation, making it challenging to eliminate tumors. Cancer cells develop resistance through genetic modifications, alterations in cellular pathways, or microenvironmental adaptations that allow cancer cells to persist, proliferate, and even thrive despite the application of therapies designed to eradicate them. Despite efforts to create more potent target-based drugs, the flexibility of acquired resistance necessitates advanced options. Combination therapy and precision immunotherapy has revolutionized treatment but is limited by patient specificity and requires further development. New approaches like small molecules, peptides, and nanotherapeutics aim to overcome resistance by enhancing site-specific delivery and increasing drug concentration inside cancer cells. This review provides a comprehensive overview of the molecular mechanisms underlying cancer drug resistance, including genetic mutations, epigenetic alterations, efflux pump activity, and the influence of the tumor microenvironment. Additionally, the roles of microRNAs, long noncoding RNAs, and cellular processes such as autophagy and hypoxia in mediating resistance are examined. This review seeks to improve patient care and help to the development of more effective cancer medicines by explaining these complicated systems and investigating novel therapeutic alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c262ab9bdd31ff19f10e8cd7a734094b9985bc60" target='_blank'>
              An overview of molecular mechanisms in cancer drug resistance and therapeutic strategies
              </a>
            </td>
          <td>
            Navin Kumar, Ranjeeta Bansala
          </td>
          <td>2024-07-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Outcomes for women with breast cancer have improved dramatically in recent decades. However, many patients present with intrinsic drug resistance and others are initially sensitive to anti-cancer drugs but acquire resistance during the course of their treatment, leading to recurrence and/or metastasis. Drug therapy-induced senescence (TIS) is a form of drug resistance characterised by the induction of cell cycle arrest and the emergence of a senescence-associated secretory phenotype (SASP) that can develop in response to chemo- and targeted- therapies. A wide range of anticancer interventions can lead to cell cycle arrest and SASP induction, by inducing genotoxic stress, hyperactivation of signalling pathways or oxidative stress. TIS can be anti-tumorigenic in the short-term, but pro-tumorigenic in the long-term by creating a pro-inflammatory and immunosuppressive microenvironment. Moreover, the SASP can promote angiogenesis and epithelial-mesenchymal transition in neighbouring cells. In this review, we will describe the characteristics of TIS in breast cancer and detail the changes in phenotype that accompany its induction. We also discuss strategies for targeting senescent cancer cells in order to prevent or delay tumour recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1902ff7d1c0379c707774d89c56e002ae6793dc1" target='_blank'>
              Therapy-induced senescence in breast cancer: an overview
              </a>
            </td>
          <td>
            Suraj Narayanan Chembukavu, Andrew J Lindsay
          </td>
          <td>2024-07-25</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is the second leading cause of global mortality and claims approximately 10 million lives annually. Despite advances in treatments such as surgery, chemotherapy, and immunotherapy, resistance to these methods has emerged. Multidrug resistance (MDR), where cancer cells resist diverse treatments, undermines therapy effectiveness, escalating mortality rates. MDR mechanisms include, among others, drug inactivation, reduced drug uptake, enhanced DNA repair, and activation of survival pathways such as autophagy. Moreover, MDR mechanisms can confer resistance to other therapies like radiotherapy. Understanding these mechanisms is crucial for improving treatment efficacy and identifying new therapeutic targets. Extracellular vesicles (EVs) have gathered attention for their role in cancer progression, including MDR development through protein transfer and genetic reprogramming. Autophagy, a process balancing cellular resources, also influences MDR. The intersection of EVs and autophagy further complicates the understanding of MDR. Both extracellular (exosomes, microvesicles) and intracellular (autophagic) vesicles contribute to therapy resistance by regulating the tumor microenvironment, facilitating cell communication, and modulating drug processing. While much is known about these pathways, there is still a need to explore their potential for predicting treatment responses and understanding tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5249ba1d82fab28ebb5cc330d8e07a3024195f" target='_blank'>
              The Role of Intracellular and Extracellular Vesicles in the Development of Therapy Resistance in Cancer.
              </a>
            </td>
          <td>
            Magdalena Wilczak, Magdalena Surman, M. Przybyło
          </td>
          <td>2024-08-07</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Abstract Phenotypic plasticity, which involves phenotypic transformation in the absence of genetic change, may serve as a strategy for organisms to survive in complex and highly fluctuating environments. However, its reaction norm, molecular basis, and evolution remain unclear in most organisms, especially microbial eukaryotes. In this study, we explored these questions by investigating the reaction norm, regulation, and evolution of phenotypic plasticity in the cosmopolitan marine free-living ciliates Glauconema spp., which undergo significant phenotypic changes in response to food shortages. This study led to the de novo assembly of macronuclear genomes using long-read sequencing, identified hundreds of differentially expressed genes associated with phenotypic plasticity in different life stages, validated the function of two of these genes, and revealed that the reaction norm of body shape in response to food density follows a power-law distribution. Purifying selection may be the dominant evolutionary force acting on the genes associated with phenotypic plasticity, and the overall data support the hypothesis that phenotypic plasticity is a trait maintained by natural selection. This study provides novel insight into the developmental genetics of phenotypic plasticity in non-model unicellular eukaryotes and sheds light on the complexity and long evolutionary history of this important survival strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e73848beeb6bba8e81a74587358c522dfa36f0a" target='_blank'>
              Molecular basis of phenotypic plasticity in a marine ciliate
              </a>
            </td>
          <td>
            Jiaoyi Pan, Yaohai Wang, Chao Li, Simo Zhang, Zhiqiang Ye, Jiahao Ni, Haichao Li, Yichen Li, Hongwei Yue, Chenchen Ruan, Dange Zhao, Yujian Jiang, Xiaolin Wu, Xiaopeng Shen, R. Zufall, Yu Zhang, Weiyi Li, Michael Lynch, Hongan Long
          </td>
          <td>2024-01-01</td>
          <td>The ISME Journal</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Cellular responses to environmental stimuli are typically thought to be governed by genetically encoded programs. We demonstrate that melanoma cells can form and maintain cellular memories during the acquisition of therapy resistance that exhibit characteristics of cellular learning and are dependent on the transcription factor AP-1. We show that cells exposed to a low dose of therapy adapt to become resistant to a high dose, demonstrating that resistance was not purely selective. The application of therapy itself results in the encoding of transient gene expression into cellular memory and that this encoding occurs for both transiently induced and probabilistically arising expression. Chromatin accessibility showed concomitant persistence. A two-color AP-1 reporter system showed that these memories are encoded in cis, constituting an example of activating cis epigenetics. Our findings establish the formation and maintenance of cellular memories as a critical aspect of gene regulation during the development of therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a15dd395d0e2a350d250357de4baa259bd2f20" target='_blank'>
              AP-1 Mediates Cellular Adaptation and Memory Formation During Therapy Resistance
              </a>
            </td>
          <td>
            Jingxin Li, Pavithran T. Ravindran, Aoife O’Farrell, Gianna T. Busch, Ryan H. Boe, Zijian Niu, Sean Woo, Margaret C. Dunagin, Naveen Jain, Yogesh Goyal, K. Sarma, M. Herlyn, Arjun Raj
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>113</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14f62e543f9ec01d9f73f5eef49615f95671e727" target='_blank'>
              Cancer drug-tolerant persister cells: from biological questions to clinical opportunities.
              </a>
            </td>
          <td>
            Mariangela Russo, Mengnuo Chen, Elisa Mariella, Haoning Peng, Sumaiyah K Rehman, E. Sancho, Alberto Sogari, Tzen S Toh, Nathalie Q Balaban, Eduard Batlle, R. Bernards, Mathew J. Garnett, Matthew J Hangauer, E. Leucci, J. Marine, Catherine A O'Brien, Yaara Oren, E. Patton, C. Robert, Susan M Rosenberg, Shensi Shen, Alberto Bardelli
          </td>
          <td>2024-09-02</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Abstract Phenotypic plasticity is a recognized mechanism driving therapeutic resistance in patients with prostate cancer. Although underlying molecular causations driving phenotypic plasticity have been identified, therapeutic success is yet to be achieved. To identify putative master regulator transcription factors (MR-TF) driving phenotypic plasticity in prostate cancer, this work utilized a multiomic approach using genetically engineered mouse models of prostate cancer combined with patient data to identify MYB proto-oncogene like 2 (MYBL2) as a significantly enriched transcription factor in prostate cancer exhibiting phenotypic plasticity. Genetic inhibition of Mybl2 using independent murine prostate cancer cell lines representing phenotypic plasticity demonstrated Mybl2 loss significantly decreased in vivo growth as well as cell fitness and repressed gene expression signatures involved in pluripotency and stemness. Because MYBL2 is currently not druggable, a MYBL2 gene signature was employed to identify cyclin-dependent kinase-2 (CDK2) as a potential therapeutic target. CDK2 inhibition phenocopied genetic loss of Mybl2 and significantly decreased in vivo tumor growth associated with enrichment of DNA damage. Together, this work demonstrates MYBL2 as an important MR-TF driving phenotypic plasticity in prostate cancer. Furthermore, high MYBL2 activity identifies prostate cancer that would be responsive to CDK2 inhibition. Significance: Prostate cancers that escape therapy targeting the androgen receptor signaling pathways via phenotypic plasticity are currently untreatable. Our study identifies MYBL2 as a MR-TF in phenotypic plastic prostate cancer and implicates CDK2 inhibition as a novel therapeutic target for this most lethal subtype of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d7169da5236ce487e4af9bc41a5465e1a1c863" target='_blank'>
              MYBL2 Drives Prostate Cancer Plasticity: Inhibiting Its Transcriptional Target CDK2 for RB1-Deficient Neuroendocrine Prostate Cancer
              </a>
            </td>
          <td>
            Beatriz German, S. A. Alaiwi, Kun-Lin Ho, Jagpreet S Nanda, Marcos A S Fonseca, Deborah L. Burkhart, Anjali V. Sheahan, Hannah E Bergom, Katherine L. Morel, H. Beltran, Justin H. Hwang, M. Freedman, K. Lawrenson, Leigh Ellis
          </td>
          <td>2024-08-08</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single-nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosome 20q or 1q, both known for conferring an in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings showed higher loads of both CNVs and SNVs at later passages, which were due to the cumulative acquisition of mutations over a longer time in culture, and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs followed the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid of cancer-associated mutations. This suggests that recurrent chromosomal abnormalities are potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36cc208bebf01a60a96b6bcb671a8cdcf9d8bbf6" target='_blank'>
              De Novo Cancer Mutations Frequently Associate with Recurrent Chromosomal Abnormalities during Long-Term Human Pluripotent Stem Cell Culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huyghebaert, M. Regin, Mai Chi Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Connecting genetic variation to phenotypes and understanding the underlying biological mechanisms has been a fundamental goal of biological genetics. Here, we used the association analysis to identify a Vip3Aa resistance-associated genomic region in a strain of fall armyworm, JC-R, which exhibits >5000-fold resistance to the Bt toxin Vip3Aa. However, through various analytical approaches and fine-scale mapping across different populations, we demonstrated that this genomic region exhibits strong genetic linkage. The chromosome-level genome of JC-R and its parent strain JC-S were assembled, and extensive structural variations in the linkage regions were identified, which could be responsible for maintaining the linkage. To identify the causal variation within this linked region, a chromosome fragment stepwise knockout strategy based on CRISPR/Cas9 was developed. By crossing with the resistant strain and phenotyping segregating offspring on Vip3Aa-containing diet, we identified a chromosomal segment, KO8, containing the resistant gene. Subsequently, we conducted a comprehensive analysis of the variations in the KO8 region using multi-omics approaches, including genomic data, RNA-seq, proteomic, PacBio long read Iso-seq, and phosphoproteomic data. This analysis identified multiple variations in the chitin synthase gene CHS2, including amino acid substitution, alternative splicing, and changes in phosphorylation sites. After knocking out the CHS2, larvae exhibited over 6777-fold resistance to Vip3Aa. These results demonstrate that the chromosome fragment stepwise knockout strategy is a viable approach for studying complex genomic regions, and highlight the value of comprehensive analysis of genetic variations using multi-omics data. The identified candidate gene could potentially advance monitoring and management of pest resistance to Vip3Aa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca50d6fcbaf49cb004ae82c9568f261b121333e" target='_blank'>
              CRISPR-mediated chromosome deletion facilitates genetic mapping of Vip3Aa resistance gene within complex genomic region in an invasive global pest
              </a>
            </td>
          <td>
            M. Jin, Yinxue Shan, Yan Peng, Senlin Chen, Xuanhe Zhou, Kaiyu Liu, Yutao Xiao
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Directed evolution is an efficient strategy to steer protein function to either understand specific biological properties or develop new biotechnology tools. Currently available methods for targeted mutagenesis in human cells rely on deaminases which can only modify specific bases, limiting the region of sequence space explored during evolution. By leveraging CRISPR-Cas9 coupled with an error-prone variant of human DNA polymerase λ, here we developed CRISPR-λ, an unbiased mutagenesis tool for directed evolution in human cells. We evaluated CRISPR-λ by reverting the fluorescence of a mutated EGFP and characterized it using ultra-deep sequencing. The mutagenic activity of CRISPR-λ spans 36-46 nucleotides from the target site, with a mutation frequency as high as 1.4e-4 substitutions per base and with no bias for specific nucleotide substitutions. The versatility of CRISPR-λ extends beyond base substitution, enabling modifications of the target gene through insertions and deletions, thereby broadening its potential for genetic diversification. We validated the efficacy of CRISPR-λ in directed evolution approaches by functionally reverting a mutated blasticidin resistance gene. Furthermore, we demonstrated the sequence diversification power of CRISPR-λ by steering the syncytia formation activity of the SARS-CoV-2 Spike envelope protein in cultured cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d747466dd7f2edf4242403524fc8d70fe775ed9" target='_blank'>
              DNA polymerase λ-driven targeted mutagenesis for directed evolution in human cells
              </a>
            </td>
          <td>
            Davide Aiello, Matteo Ciciani, Federica Marelli, Marta Stancampiano, Veronica De Sanctis, R. Bertorelli, E. Kheir, G. Maule, A. Cereseto, Daniele Arosio
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c323e127988809654aaf5a547e1a3c8242712a2" target='_blank'>
              Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells
              </a>
            </td>
          <td>
            Ian A. Mellis, Madeline E. Melzer, Nicholas Bodkin, Yogesh Goyal
          </td>
          <td>2024-08-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23cf92719cd49496400fba609df94fa13b009008" target='_blank'>
              Precision medicine in AML: overcoming resistance.
              </a>
            </td>
          <td>
            Samuel Urrutia, Koichi Takahashi
          </td>
          <td>2024-08-01</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Ionizing radiation (IR) in high doses is generally lethal to most organisms. Investigating mechanisms of radiation resistance is crucial for gaining insights into the underlying cellular responses and understanding the damaging effects of IR. In this study, we conducted a comprehensive analysis of sequencing data from an evolutionary experiment aimed at understanding the genetic adaptations to ionizing radiation in Escherichia coli. By including previously neglected synonymous mutations, we identified the rpsH c.294T > G variant, which emerged in all 17 examined isolates across four subpopulations. The identified variant is a synonymous mutation affecting the 30S ribosomal protein S8, and consistently exhibited high detection and low allele frequencies in all subpopulations. This variant, along with two additional rpsH variants, potentially influences translational control of the ribosomal spc operon. The early emergence and stability of these variants suggest their role in adapting to environmental stress, possibly contributing to radiation resistance. Our findings shed light on the dynamics of ribosomal variants during the evolutionary process and their potential role in stress adaptation, providing valuable implications for understanding clinical radiation sensitivity and improving radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10a94830c53f3f47e59574a18dce14e20ea55a06" target='_blank'>
              Synonymous rpsH variants: the common denominator in Escherichia coli adapting to ionizing radiation
              </a>
            </td>
          <td>
            Katharina Stemwedel, N. Haase, Simon Christ, Natalia V Bogdanova, Sophia Rudorf
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Allopolyploidy in plants involves the merging of two or more distinct parental genomes into a single nucleus, a significant evolutionary process in the plant kingdom. Transcriptomic analysis provides invaluable insights into allopolyploid plants by elucidating the fate of duplicated genes, revealing evolutionary novelties and uncovering their environmental adaptations. By examining gene expression profiles, scientists can discern how duplicated genes have evolved to acquire new functions or regulatory roles. This process often leads to the development of novel traits and adaptive strategies that allopolyploid plants leverage to thrive in diverse ecological niches. Understanding these molecular mechanisms not only enhances our appreciation of the genetic complexity underlying allopolyploidy but also underscores their importance in agriculture and ecosystem resilience. However, transcriptome profiling is challenging due to genomic redundancy, which is further complicated by the presence of multiple chromosomes sets and the variations among homoeologs and allelic genes. Prior to transcriptome analysis, sub-genome phasing and homoeology inference are essential for obtaining a comprehensive view of gene expression. This review aims to clarify the terminology in this field, identify the most challenging aspects of transcriptome analysis, explain their inherent difficulties, and suggest reliable analytic strategies. Furthermore, bulk RNA-seq is highlighted as a primary method for studying allopolyploid gene expression, focusing on critical steps like read mapping and normalization in differential gene expression analysis. This approach effectively captures gene expression from both parental genomes, facilitating a comprehensive analysis of their combined profiles. Its sensitivity in detecting low-abundance transcripts allows for subtle differences between parental genomes to be identified, crucial for understanding regulatory dynamics and gene expression balance in allopolyploids.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8b5801676d1808c357ebdb7b80d09901e3fd903" target='_blank'>
              Homoeologs in Allopolyploids: Navigating Redundancy as Both an Evolutionary Opportunity and a Technical Challenge—A Transcriptomics Perspective
              </a>
            </td>
          <td>
            Gaetano Aufiero, Carmine Fruggiero, Davide D’Angelo, Nunzio D’Agostino
          </td>
          <td>2024-07-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e223c600817ad1cd6259f50a38399720abf1f7d" target='_blank'>
              Drug tolerant persister cell plasticity in cancer: A revolutionary strategy for more effective anticancer therapies
              </a>
            </td>
          <td>
            Jun He, Zejing Qiu, Jingjing Fan, Xiaohong Xie, Qinsong Sheng, Xinbing Sui
          </td>
          <td>2024-08-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/589af1da53c29b65656184828eec590acfbf3b30" target='_blank'>
              Genetic and epigenetic instability as an underlying driver of progression and aggressive behavior in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            Timothy E Richardson, Jamie M. Walker, Dolores Hambardzumyan, Steven Brem, K. Hatanpaa, Mariano S Viapiano, Balagopal Pai, M. Umphlett, Oren J. Becher, M. Snuderl, S. McBrayer, Kalil G. Abdullah, N. Tsankova
          </td>
          <td>2024-07-16</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Interspecific genomic introgression is an important evolutionary process with respect to the generation of novel phenotypic diversity and adaptation. A key question is how gene flow perturbs gene expression networks and regulatory interactions. Here, an introgression population of two species of allopolyploid cotton (Gossypium) to delineate the regulatory perturbations of gene expression regarding fiber development accompanying fiber quality change is utilized. De novo assembly of the recipient parent (G. hirsutum Emian22) genome allowed the identification of genomic variation and introgression segments (ISs) in 323 introgression lines (ILs) from the donor parent (G. barbadense 3-79). It documented gene expression dynamics by sequencing 1,284 transcriptomes of developing fibers and characterized genetic regulatory perturbations mediated by genomic introgression using a multi-locus model. Introgression of individual homoeologous genes exhibiting extreme low or high expression bias can lead to a parallel expression bias in their non-introgressed duplicates, implying a shared yet divergent regulatory fate of duplicated genes following allopolyploidy. Additionally, the IL N182 with improved fiber quality is characterized, and the candidate gene GhFLAP1 related to fiber length is validated. This study outlines a framework for understanding introgression-mediated regulatory perturbations in polyploids, and provides insights for targeted breeding of superior upland cotton fiber.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1405a67fdbcebbc2402da2663113cf18f89f6fba" target='_blank'>
              Genetic Regulatory Perturbation of Gene Expression Impacted by Genomic Introgression in Fiber Development of Allotetraploid Cotton.
              </a>
            </td>
          <td>
            Xinyuan Chen, Xiubao Hu, Guo Li, C. Grover, J. You, Ruipeng Wang, Zhenping Liu, Zhengyang Qi, Xuanxuan Luo, Yabing Peng, Mengmeng Zhu, Yuqi Zhang, Sifan Lu, Yuan-Ming Zhang, Zhongxu Lin, Jonathan F. Wendel, Xianlong Zhang, Maojun Wang
          </td>
          <td>2024-08-28</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="INTRODUCTION
Colon cancer, ranked as the fourth leading global cause of cancer death, exhibits a complex progression marked by genetic variations. Over the past decade, the utilization of diverse CRISPR systems has propelled accelerated research into colorectal cancer (CRC) treatment.


AREAS COVERED
CRISPR/Cas9, a key player in this research, identifies new oncogenes, tumor suppressor genes (TSGs), and drug-resistance genes. Additionally, it facilitates the construction of experimental models, conducts genome-wide library screening, and develops new therapeutic targets, especially for targeted knockout in vivo or molecular targeted drug delivery, contributing to personalized treatments and significantly enhancing the care of colon cancer patients. In this review, we provide insights into the mechanism of the CRISPR/Cas9 system, offering a comprehensive exploration of its applications in CRC, spanning screening, modeling, gene functions, diagnosis, and gene therapy. While acknowledging its transformative potential, the article  highlights the challenges and limitations of CRISPR systems.


EXPERT OPINION
The application of CRISPR/Cas9 in CRC research provides a promising avenue for personalized treatments. Its potential for identifying key genes and enabling experimental models and genome-wide screening enhances patient care. This review underscores the significance of CRISPR-Cas9 gene editing technology across basic research, diagnosis, and the treatment landscape of colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e832452c0de49427c039c9545ac3618de43c5948" target='_blank'>
              Recent advances in CRISPR-Cas systems for colorectal cancer research and therapeutics.
              </a>
            </td>
          <td>
            Saeideh Khorshid Sokhangouy, Farzaneh Alizadeh, Malihe Lotfi, Samaneh Sharif, Atefeh Ashouri, Yasamin Yoosefi, Saeed Bozorg Qomi, M. Abbaszadegan
          </td>
          <td>2024-08-02</td>
          <td>Expert review of molecular diagnostics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A. Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Using five complementary short- and long-read sequencing technologies, we phased and assembled >95% of each diploid human genome in a four-generation, 28-member family (CEPH 1463) allowing us to systematically assess de novo mutations (DNMs) and recombination. From this family, we estimate an average of 192 DNMs per generation, including 75.5 de novo single-nucleotide variants (SNVs), 7.4 non-tandem repeat indels, 79.6 de novo indels or structural variants (SVs) originating from tandem repeats, 7.7 centromeric de novo SVs and SNVs, and 12.4 de novo Y chromosome events per generation. STRs and VNTRs are the most mutable with 32 loci exhibiting recurrent mutation through the generations. We accurately assemble 288 centromeres and six Y chromosomes across the generations, documenting de novo SVs, and demonstrate that the DNM rate varies by an order of magnitude depending on repeat content, length, and sequence identity. We show a strong paternal bias (75-81%) for all forms of germline DNM, yet we estimate that 17% of de novo SNVs are postzygotic in origin with no paternal bias. We place all this variation in the context of a high-resolution recombination map (∼3.5 kbp breakpoint resolution). We observe a strong maternal recombination bias (1.36 maternal:paternal ratio) with a consistent reduction in the number of crossovers with increasing paternal (r=0.85) and maternal (r=0.65) age. However, we observe no correlation between meiotic crossover locations and de novo SVs, arguing against non-allelic homologous recombination as a predominant mechanism. The use of multiple orthogonal technologies, near-telomere-to-telomere phased genome assemblies, and a multi-generation family to assess transmission has created the most comprehensive, publicly available “truth set” of all classes of genomic variants. The resource can be used to test and benchmark new algorithms and technologies to understand the most fundamental processes underlying human genetic variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcb8df79fd9bfc629f05070b595811f041bfe33" target='_blank'>
              A familial, telomere-to-telomere reference for human de novo mutation and recombination from a four-generation pedigree
              </a>
            </td>
          <td>
            David Porubsky, Harriet Dashnow, T. Sasani, Glennis A. Logsdon, P. Hallast, Michelle D. Noyes, Z. Kronenberg, T. Mokveld, Nidhi Koundinya, Cillian Nolan, C. Steely, A. Guarracino, E. Dolzhenko, William T. Harvey, W. Rowell, K. Grigorev, T. J. Nicholas, Keisuke K. Oshima, Jiadong Lin, Peter Ebert, W. Watkins, Tiffany Y. Leung, Vincent C. T. Hanlon, Sean Mcgee, Brent S. Pedersen, Michael E. Goldberg, Hannah C. Happ, Hyeonsoo Jeong, Katherine M. Munson, Kendra Hoekzema, Daniel D. Chan, Yanni Wang, Jordan Knuth, Gage H. Garcia, Cairbre Fanslow, Christine Lambert, Charles Lee, Josh Smith, Shawn Levy, C. Mason, Erik Garrison, Peter Lansdorp, D. Neklason, L. Jorde, Aaron R. Quinlan, M. Eberle, Evan E. Eichler
          </td>
          <td>2024-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="A multitude of exogenous and endogenous processes have the potential to result in DNA damage. While the repair mechanisms are typically capable of correcting this damage, errors in the repair process can result in mutations. The findings of research conducted in 2012 indicate that mutations do not occur randomly but rather follow specific patterns that can be attributed to known or inferred mutational processes. The process of mutational signature analysis allows for the inference of the predominant mutational process for a given cancer sample, with significant potential for clinical applications. A deeper comprehension of these mutational signatures in CRC could facilitate enhanced prevention strategies, facilitate the comprehension of genotoxic drug activity, predict responses to personalized treatments, and, in the future, inform the development of targeted therapies in the context of precision oncology. The efforts of numerous researchers have led to the identification of several mutational signatures, which can be categorized into different mutational signature references. In CRC, distinct mutational signatures are identified as correlating with mismatch repair deficiency, polymerase mutations, and chemotherapy treatment. In this context, a mutational signature analysis offers considerable potential for enhancing minimal residual disease (MRD) tests in stage II (high-risk) and stage III CRC post-surgery, stratifying CRC based on the impacts of genetic and epigenetic alterations for precision oncology, identifying potential therapeutic vulnerabilities, and evaluating drug efficacy and guiding therapy, as illustrated in a proof-of-concept clinical trial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9f4817ac8c40910eefa1198ffb3e184664ea3d" target='_blank'>
              Mutational Signatures in Colorectal Cancer: Translational Insights, Clinical Applications, and Limitations
              </a>
            </td>
          <td>
            G. Crisafulli
          </td>
          <td>2024-08-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The genomic landscape associated with early adaptation to ciprofloxacin is poorly understood. Although the interplay between core metabolism and antimicrobial resistance is being increasingly recognized, mutations in metabolic genes and their biological role remain elusive. Here, we exposed Escherichia coli to increasing gradients of ciprofloxacin with intermittent transfer-bottlenecking and identified mutations in three non-canonical targets linked to metabolism including a deletion (tRNA-ArgΔ414-bp) and point mutations in the regulatory regions of argI (ARG box) and narU. Our findings suggest that these mutations modulate arginine and carbohydrate metabolism, facilitate anaerobiosis and increased ATP production during ciprofloxacin stress. Furthermore, mutations in the regulatory regions of argI and narU were detected in over 70% of sequences from clinical E. coli isolates and were overrepresented among ciprofloxacin-resistant isolates. In sum, we have identified clinically relevant mutations in the regulatory regions of metabolic genes as a central theme that drives physiological changes necessary for adaptation to ciprofloxacin stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ed3143db4faeda9db095beee9dcccbe75dc30c4" target='_blank'>
              Clinically relevant mutations in regulatory regions of metabolic genes facilitate early adaptation to ciprofloxacin in Escherichia coli.
              </a>
            </td>
          <td>
            Arijit Pal, Dipannita Ghosh, Pratyusha Thakur, Priya Nagpal, Madhumathi Irulappan, Karthik Maruthan, Sanket Mukherjee, Nikita G Patil, Tanmay Dutta, B. Veeraraghavan, Perumal Vivekanandan
          </td>
          <td>2024-08-24</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d317e0704a11a531980e863a514ed7e3ab276870" target='_blank'>
              Advances in pathogenesis research and challenges in treatment development for acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hiroki Yamaguchi
          </td>
          <td>2024-09-03</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Microbial genomes are continuously being rearranged by mobile genetic elements (MGEs), leading to genetic configurations that may confer novel phenotypic traits such as antibiotic resistance, degradation of novel compounds, or other metabolic features. Standard genomic sequencing provides a snapshot of a genome in one configuration, but this static image does not give insight into the dynamics of genomic evolution and whether MGEs are actively changing a given genome. We applied single-strain mobilome sequencing to Escherichia coli K-12 substrain MG1655 under various stress conditions: UV, SDS, nalidixic acid, tetracycline, cetrimide, and copper. Under these conditions, we quantified the activity of a range of genetic elements, including extrachromosomal circular DNA (eccDNA) from IS elements, RNA genes, the UV-inducible e14 prophage, and intergenic repetitive sites (REP). Of the investigated stressors, copper and SDS are among the largest inducers of eccDNA formation from groups of IS elements, while elevated levels of hypothetical RNA/DNA heteroduplexes of ribosomal and transfer RNAs, and Rhs-nuclease proteins are induced under stress various stressors, especially copper and SDS. This approach holds promise for quantifying the genetic response to environmental stress and implications for genome plasticity. The observed mobilization of IS elements upon copper and other stressors helps to explain co-selection of heavy metals with antibiotic resistance genes and MGEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef3703588c90abb531d5e3a142eca82f17774739" target='_blank'>
              Single-strain mobilome sequencing quantifies bacterial genetic response to stress, including activity of IS elements, prophages, RNAs, and REPINs
              </a>
            </td>
          <td>
            T. K. Nielsen, Lars Hestbjerg Hansen
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Ovarian cancer ecosystems are exceedingly complex, consisting of a high heterogeneity of cancer cells. Development of drugs such as poly ADP‐ribose polymerase (PARP) inhibitors, targeted therapies and immunotherapies offer more options for sequential or combined treatments. Nevertheless, mortality in metastatic ovarian cancer patients remains high because cancer cells consistently develop resistance to single and combination therapies, urging a need for treatment designs that target the evolvability of cancer cells. The evolutionary dynamics that lead to resistance emerge from the complex tumour microenvironment, the heterogeneous populations, and the individual cancer cell's plasticity. We propose that successful management of ovarian cancer requires consideration of the ecological and evolutionary dynamics of the disease. Here, we review current options and challenges in ovarian cancer treatment and discuss principles of tumour evolution. We conclude by proposing evolutionarily designed strategies for ovarian cancer, with the goal of integrating such principles with longitudinal, quantitative data to improve the treatment design and management of drug resistance. Key points/Highlights Tumours are ecosystems in which cancer and non‐cancer cells interact and evolve in complex and dynamic ways. Conventional therapies for ovarian cancer inevitably lead to the development of resistance because they fail to consider tumours’ heterogeneity and cellular plasticity. Eco‐evolutionarily designed therapies should consider cancer cell plasticity and patient‐specific characteristics to improve clinical outcome and prevent relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f584cd6ae71db959a312c8365ae2a9ada15cf5" target='_blank'>
              Ecological and evolutionary dynamics to design and improve ovarian cancer treatment
              </a>
            </td>
          <td>
            Grace Y Q Han, Monica Alexander, Julia Gattozzi, Marilyn Day, Elayna Kirsch, Narges Tafreshi, Raafat Chalar, Soraya Rahni, Gabrielle Gossner, William Burke, Mehdi Damaghi
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Antimicrobial resistance (AMR) poses a significant threat to global public health, necessitating innovative strategies to combat this escalating issue. This review outlines next-generation approaches integrating genomics, CRISPR technology, and novel therapeutics to effectively address AMR. Genomic techniques enable comprehensive understanding of the genetic mechanisms underpinning resistance, facilitating the development of targeted interventions. By sequencing the genomes of resistant pathogens, researchers can identify resistance genes, track their spread, and predict emerging resistance patterns. CRISPR-Cas systems offer a revolutionary tool for combating AMR through precise genome editing. This technology can disrupt resistance genes, restore antibiotic sensitivity, and develop bacteriophage therapies that selectively target resistant bacteria. Moreover, CRISPR-based diagnostics enable rapid, accurate detection of resistant strains, enhancing infection control measures. The advent of novel therapeutics, such as antimicrobial peptides, bacteriophage therapy, and synthetic biology-derived compounds, provides alternative treatment options. These therapeutics can bypass traditional resistance mechanisms and exhibit efficacy against multi-drug resistant organisms. Additionally, integrating artificial intelligence (AI) and machine learning with genomics and CRISPR can accelerate the discovery of new antibiotics and predict resistance trends, optimizing treatment regimens. Implementing these next-generation strategies requires robust global collaboration, regulatory frameworks, and investment in research and development. By combining genomics, CRISPR, and novel therapeutics, we can create a multifaceted approach to overcome AMR, ensuring effective treatments and safeguarding public health. This integration represents a paradigm shift in antimicrobial strategy, offering hope for a future where resistant infections can be effectively managed and treated. 
Keywords: Integrating Genomics, Antimicrobial Resistance, CRISPR, Therapeutic">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/268811efc6356d749ee7754e1ba9d746c977ebdb" target='_blank'>
              Next-Generation strategies to combat antimicrobial resistance: Integrating genomics, CRISPR, and novel therapeutics for effective treatment
              </a>
            </td>
          <td>
            Aliu Olalekan Olatunji, Janet Aderonke Olaboye, Chukwudi Cosmos Maha, Tolulope Olagoke Kolawole, Samira Abdul
          </td>
          <td>2024-07-24</td>
          <td>Engineering Science &amp; Technology Journal</td>
          <td>2</td>
          <td>4</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/699986230662d4afba97aea6b14e6128a02adbf7" target='_blank'>
              Molecular mechanisms of DNA lesion and repair during antibody somatic hypermutation.
              </a>
            </td>
          <td>
            Qian Hao, Jinfeng Li, Leng-Siew Yeap
          </td>
          <td>2024-07-23</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Transcription and splicing errors in cancer cells generate erroneous transcripts. Since erroneous transcripts are degraded by the nonsense-mediated mRNA decay (NMD) pathway, whether they are junk or could be functional has been overlooked and understudied. We addressed this question by first performing a pan-cancer analysis and identified substantial erroneous noncoding transcripts (ENT) in cancers. Given that RNA/DNA binding domains (DBD) were predicted in ENTs, we deleted predicted DBDs in multiple ENTs in multiple cell lines, with RNA-sequencing and cell experiments before and after DBD deletion. DBD deletion caused significantly changed expression of ENTs’ target genes (whose promoter regions contain ENTs’ DNA binding sites, DBS) and changed cell migration and proliferation ability, indicating that many ENTs can transcriptionally regulate genes. Tightly coupled data analysis and experiments reveal that ENTs’ functions are highly cancer- and cellular-context specific, making ENTs a new class of safe and specific targets for noncoding RNA-based cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e45af8d1c765a560ff555893aa2ed200e47f3c" target='_blank'>
              Many erroneous noncoding transcripts in cancer cells can highly specifically regulate cancer-related genes and pathways
              </a>
            </td>
          <td>
            Sha He, Wei Xiong, Jianping Huo, Jie Lin, Jianmin Li, Hao Zhu
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Microsatellite unstable colorectal cancer (MSI-CRC) can arise through germline mutations in mismatch repair (MMR) genes in individuals with Lynch syndrome (LS), or sporadically through promoter methylation of the MMR gene MLH1. Despite the different origins of hereditary and sporadic MSI tumours, their genomic features have not been extensively compared. A prominent feature of MMR-deficient genomes is the occurrence of many indels in short repeat sequences, an understudied mutation type due to the technical challenges of variant calling in these regions. In this study, we performed whole genome sequencing and RNA-sequencing on 29 sporadic and 14 hereditary MSI-CRCs. We compared the tumour groups by analysing genome-wide mutation densities, microsatellite repeat indels, recurrent protein-coding variants, signatures of single base, doublet base, and indel mutations, and changes in gene expression. We show that the mutational landscapes of hereditary and sporadic MSI-CRCs, including mutational signatures and mutation densities genome-wide and in microsatellites, are highly similar. Only a low number of differentially expressed genes were found, enriched to interferon-γ regulated immune response pathways. Analysis of the variance in allelic fractions of somatic variants in each tumour group revealed higher clonal heterogeneity in sporadic MSI-CRCs. Our results suggest that the differing molecular origins of MMR deficiency in hereditary and sporadic MSI-CRCs do not result in substantial differences in the mutational landscapes of these tumours. The divergent patterns of clonal evolution between the tumour groups may have clinical implications, as high clonal heterogeneity has been associated with decreased tumour immunosurveillance and reduced responsiveness to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43c6128f5465fd71306e14170da497710701fc42" target='_blank'>
              Lynch syndrome-associated and sporadic microsatellite unstable colorectal cancers: different patterns of clonal evolution yield highly similar tumours.
              </a>
            </td>
          <td>
            Samantha Martin, Riku Katainen, Aurora Taira, Niko Välimäki, Ari Ristimäki, T. Seppälä, L. Renkonen-Sinisalo, A. Lepistö, Kyösti Tahkola, Anne Mattila, S. Koskensalo, J. Mecklin, K. Rajamäki, Kimmo Palin, L. Aaltonen
          </td>
          <td>2024-08-24</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Despite the critical importance of essential genes, systems-level investigations of their contribution to antibiotic sensitivity have been limited. Using CRISPR Adaptation-mediated Library Manufacturing (CALM), we generated ultra-dense CRISPR interference (CRISPRi) libraries in methicillin-sensitive and -resistant strains of Staphylococcus aureus, which allowed us to quantify gene fitness on a global scale across ten clinically relevant antibiotics. This led to the identification of a comprehensive set of known and novel biological processes modulating bacterial fitness in the antibiotics. Notably, we found that essential genes from diverse processes dominated antibiotic-gene interactions, including a large number of synergistic interactions between bactericidal antibiotics and processes such as cell wall synthesis/cell division (CC), DNA replication/DNA recombination (DD), protein export, and coenzyme A biosynthesis. Simultaneous genetic perturbations of diverse CC and DD processes aggravated bacterial fitness, revealing a widespread synergy between the two highly coordinated processes. In contrast, perturbation of transcriptional, translational, and select energy processes antagonized the effects of bactericidal antibiotics. Finally, we show that small molecule inhibitors recapitulated synergistic antibiotic-gene interactions, providing a rational foundation for developing novel combinatorial antimicrobial therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d56a4c312c5116223ff7d382ce48c403626fb7c5" target='_blank'>
              The genetic landscape of antibiotic sensitivity in Staphylococcus aureus
              </a>
            </td>
          <td>
            Wan Li, Menghan Liu, P. Oikonomou, S. Blattman, Julia Hettleman, Joseph Gonzalez, Howe Chen, Saeed Tavazoie, Wenyan Jiang
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Methods to engineer the genomes of human cells for therapeutic intervention continue to advance at a remarkable pace. Chimeric antigen receptor-engineered T lymphocytes have pioneered the way for these therapies, initially beginning with insertions of chimeric antigen receptor transgenes into T-cell genomes using classical gene therapy vectors. The broad use of clustered regularly interspaced short palindromic repeats (CRISPR)-based technologies to edit endogenous genes has now opened the door to a new era of precision medicine. To add complexity, many engineered cellular therapies under development integrate gene therapy with genome editing to introduce novel biological functions and enhance therapeutic efficacy. Here, we review the current state of scientific, translational, and regulatory oversight of gene-edited cell products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2d1ab3d9838377779d6ccf7a7d2e82b715e992" target='_blank'>
              Regulatory Considerations for Genome-Edited T-cell Therapies.
              </a>
            </td>
          <td>
            J. Jadlowsky, Ju-Fang Chang, DH Spencer, John M Warrington, Bruce L. Levine, Carl H. June, J. Fraietta, Nathan Singh
          </td>
          <td>2024-07-17</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract The molecular underpinnings and consequences of cycles of whole-genome duplication (WGD) and subsequent gene loss through subgenome fractionation remain largely elusive. Endogenous drivers, such as transposable elements (TEs), have been postulated to shape genome-wide dominance and biased fractionation, leading to a conserved least-fractionated (LF) subgenome and a degenerated most-fractionated (MF) subgenome. In contrast, the role of exogenous factors, such as those induced by environmental stresses, has been overlooked. In this study, a chromosome-scale assembly of the alpine buckler mustard (Biscutella laevigata; Brassicaceae) that underwent a WGD event about 11 million years ago is coupled with transcriptional responses to heat, cold, drought, and herbivory to assess how gene expression is associated with differential gene retention across the MF and LF subgenomes. Counteracting the impact of TEs in reducing the expression and retention of nearby genes across the MF subgenome, dosage balance is highlighted as a main endogenous promoter of the retention of duplicated gene products under purifying selection. Consistent with the “turn a hobby into a job” model, about one-third of environment-responsive duplicates exhibit novel expression patterns, with one copy typically remaining conditionally expressed, whereas the other copy has evolved constitutive expression, highlighting exogenous factors as a major driver of gene retention. Showing uneven patterns of fractionation, with regions remaining unbiased, but with others showing high bias and significant enrichment in environment-responsive genes, this mesopolyploid genome presents evolutionary signatures consistent with an interplay of endogenous and exogenous factors having driven gene content following WGD-fractionation cycles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96aa827a7920fbae90dd91cda53e844f68c493ea" target='_blank'>
              Biased Retention of Environment-Responsive Genes Following Genome Fractionation
              </a>
            </td>
          <td>
            Marc Beringer, R. R. Choudhury, T. Mandáková, Sandra Grünig, Manuel Poretti, I. Leitch, M. Lysak, C. Parisod
          </td>
          <td>2024-07-29</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Cell immortalization, a hallmark of cancer development, is a process that cells can undergo on their path to carcinogenesis. Spontaneously immortalized mouse embryonic fibroblasts (MEFs) have been used for decades; however, changes in the global transcriptome during this process have been poorly described. In our research, we characterized the poly-A RNA transcriptome changes after spontaneous immortalization. To this end, differentially expressed genes (DEGs) were screened using DESeq2 and characterized by gene ontology enrichment analysis and protein–protein interaction (PPI) network analysis to identify the potential hub genes. In our study, we identified changes in the expression of genes involved in proliferation regulation, cell adhesion, immune response and transcriptional regulation in immortalized MEFs. In addition, we performed a comparative analysis with previously reported MEF immortalization data, where we propose a predicted gene regulatory network model in immortalized MEFs based on the altered expression of Mapk11, Cdh1, Chl1, Zic1, Hoxd10 and the novel hub genes Il6 and Itgb2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/edae843236a102c56d19f5e2f20e7de0a5418029" target='_blank'>
              Transcriptome Profiling of Mouse Embryonic Fibroblast Spontaneous Immortalization: A Comparative Analysis
              </a>
            </td>
          <td>
            Jocshan Loaiza-Moss, U. Braun, Michael Leitges
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/860a1cc9d077ab2536411e26d625b8a5e8cd579d" target='_blank'>
              Epigenome-wide impact of MAT2A sustains the androgen-indifferent state and confers synthetic vulnerability in ERG fusion-positive prostate cancer
              </a>
            </td>
          <td>
            A. Cacciatore, D. Shinde, C. Musumeci, G. Sandrini, L. Guarrera, D. Albino, G. Civenni, Elisa Storelli, S. Mosole, Elisa Federici, Alessio Fusina, Marta Iozzo, Andrea Rinaldi, Matteo Pecoraro, Roger Geiger, Marco Bolis, C. Catapano, G. M. Carbone
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="The intrinsic oncogenic mechanisms and properties of the tumor microenvironment (TME) have been extensively investigated. Primary features of the TME include metabolic reprogramming, hypoxia, chronic inflammation, and tumor immunosuppression. Previous studies suggest that senescence-associated secretory phenotypes that mediate intercellular information exchange play a role in the dynamic evolution of the TME. Specifically, hypoxic adaptation, metabolic dysregulation, and phenotypic shifts in immune cells regulated by cellular senescence synergistically contribute to the development of an immunosuppressive microenvironment and chronic inflammation, thereby promoting the progression of tumor events. This review provides a comprehensive summary of the processes by which cellular senescence regulates the dynamic evolution of the tumor-adapted TME, with focus on the complex mechanisms underlying the relationship between senescence and changes in the biological functions of tumor cells. The available findings suggest that components of the TME collectively contribute to the progression of tumor events. The potential applications and challenges of targeted cellular senescence-based and combination therapies in clinical settings are further discussed within the context of advancing cellular senescence-related research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f69d28c4893741d69a2e81f7df756fd68f976626" target='_blank'>
              Cellular senescence and metabolic reprogramming: Unraveling the intricate crosstalk in the immunosuppressive tumor microenvironment.
              </a>
            </td>
          <td>
            Fusheng Zhang, Junchen Guo, Shengmiao Yu, Youwei Zheng, Meiqi Duan, Liang Zhao, Yihan Wang, Zhi Yang, Xiaofeng Jiang
          </td>
          <td>2024-07-12</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome flexibility is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and flexibility determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin a more prone to form SVs than those located in euchromatin, highlighting that genome stability and flexibility differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA Damage Response (DDR), ATM and ATR, suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure UV-B, UV-C and protons induced the Microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a78aa5b539dcd98a3297c9111dc41b5d43d56b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, M. Pellicioli, Jacky Schuler, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d65bf85005c8a5a2fb214830aeec58d6f7820d7" target='_blank'>
              Key genes and molecular mechanisms related to Paclitaxel Resistance
              </a>
            </td>
          <td>
            Adel I Alalawy
          </td>
          <td>2024-07-13</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions.We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters.Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations.Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, Weisen Fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Genome merging is a common phenomenon in many organisms, causing a wide range of consequences on phenotype, adaptation, and gene expression, among other effects, yet its broader implications are not well understood. Two consequences of genome merging on gene expression remain poorly understood: dosage effects and evolution of expression. In this study, we employed Chlamydomonas reinhardtii as a model to investigate the effects of asymmetric genome merging by crossing a diploid with a haploid strain to create a novel triploid line. Five independent clonal lineages derived from this triploid line were evolved for 425 asexual generations in a laboratory natural selection (LNS) experiment. Utilizing fitness assays, qPCR, and RNA-Seq, we assessed the immediate consequences of genome merging and subsequent evolution over time. Our findings reveal substantial alterations in gene expression, protein homeostasis (proteostasis) and cytonuclear stoichiometry. Notably, gene expression exhibited expression level dominance and transgressivity (i.e., expression level higher or lower than either parent). Ongoing expression level dominance and a pattern of “functional dominance” from the haploid parent was observed, alongside remarkable stability in expression patterns across generations. Despite major nucleo-cytoplasmic disruptions, enhanced fitness was detected in the triploid strain. By comparing gene expression across generations, our results indicate that proteostasis restoration is a critical component of rapid adaptation following genome merging in Chlamydomonas reinhardtii and possibly other systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15e55d2c045a09190880662b2365054897839f5" target='_blank'>
              Asymmetric genome merging leads to gene expression novelty through nucleo-cytoplasmic disruptions and transcriptomic shock in Chlamydomonas triploids
              </a>
            </td>
          <td>
            Lucas Prost-Boxoen, Quinten Bafort, Antoine Van de Vloet, Fabrício Almeida-Silva, Yunn Thet Paing, G. Casteleyn, S. D’hondt, O. De Clerck, Yves Van de Peer
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a121d12ee972f1d2e2291a3bbf0c1c888699c1d3" target='_blank'>
              Writers, readers, and erasers RNA modifications and drug resistance in cancer
              </a>
            </td>
          <td>
            Di Chen, Xinyu Gu, Yeltai Nurzat, Lixia Xu, Xueyuan Li, Lixin Wu, Henan Jiao, Peng Gao, Xuqiang Zhu, Dongming Yan, Shaohua Li, Chen Xue
          </td>
          <td>2024-08-30</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract Understanding the evolutionary potential of mutations in gene regulatory networks is essential to furthering the study of evolution and development. However, in multicellular systems, genetic manipulation of regulatory networks in a targeted and high-throughput way remains challenging. In this study, we designed TF-High-Evolutionary (HighEvo), a transcription factor (TF) fused with a base editor (activation-induced deaminase), to continuously induce germline mutations at TF-binding sites across regulatory networks in Drosophila. Populations of flies expressing TF-HighEvo in their germlines accumulated mutations at rates an order of magnitude higher than natural populations. Importantly, these mutations accumulated around the targeted TF-binding sites across the genome, leading to distinct morphological phenotypes consistent with the developmental roles of the tagged TFs. As such, this TF-HighEvo method allows the interrogation of the mutational space of gene regulatory networks at scale and can serve as a powerful reagent for experimental evolution and genetic screens focused on the regulatory genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2f5224e39a793a9098b6100dcfef52f5acedf2" target='_blank'>
              TF-High-Evolutionary: In Vivo Mutagenesis of Gene Regulatory Networks for the Study of the Genetics and Evolution of the Drosophila Regulatory Genome
              </a>
            </td>
          <td>
            Xueying C. Li, Vani Srinivasan, Ian Laiker, Natalia Misunou, Nicolás Frankel, Luisa F. Pallares, Justin Crocker
          </td>
          <td>2024-08-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cancer is a multifaceted disease influenced by both intrinsic cellular traits and extrinsic factors, with the tumor microenvironment (TME) being crucial in its progression. To satisfy their high proliferation and aggressiveness, cancer cell always plunders large amounts of nutrition and releases various signals to the surrounding, forming a dynamic TME with special metabolic, immune, microbial and physical characteristics. Due to the neglect of interactions between tumor cell and TME, traditional cancer therapies often struggle with challenges such as drug resistance, low efficacy, and recurrence. Importantly, with the development of gene editing technologies, particularly the CRISPR-Cas system, offers promising new strategies for cancer treatment. Combined with nanomaterials strategies, CRISPR-Cas technology exhibits precision, affordability, and user-friendliness with reduced side effects, which holds great promise for profoundly altering the TME at a genetic level, potentially leading to lasting anticancer outcomes. This review will delve into how CRISPR-Cas can be leveraged to manipulate the TME, examining its potential as a transformative anticancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b4994cb649ea8f94220d3676ac8b2eeb79b1144" target='_blank'>
              Harnessing Gene Editing Technology for Tumor Microenvironment Modulation: An Emerging Anticancer Strategy.
              </a>
            </td>
          <td>
            Shujun Feng, Yu Zhang, Yanyi Wang, Yanfeng Gao, Yujun Song
          </td>
          <td>2024-09-03</td>
          <td>Chemistry</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer treatment has long been fraught with challenges, including drug resistance, metastasis, and recurrence, making it one of the most difficult diseases to treat effectively. Traditional therapeutic approaches often fall short due to their inability to target cancer stem cells and the complex genetic and epigenetic landscape of tumors. In recent years, cancer immunotherapy has revolutionized the field, offering new hope and viable alternatives to conventional treatments. A particularly promising area of research focuses on non-coding RNAs (ncRNAs), especially long non-coding RNAs (lncRNAs), and their role in cancer resistance and the modulation of signaling pathways. To address these challenges, we performed a comprehensive review of recent studies on lncRNAs and their impact on cancer immunotherapy. Our review highlights the crucial roles that lncRNAs play in affecting both innate and adaptive immunity, thereby influencing the outcomes of cancer treatments. Key observations from our review indicate that lncRNAs can modify the tumor immune microenvironment, enhance immune cell infiltration, and regulate cytokine production, all of which contribute to tumor growth and resistance to therapies. These insights suggest that lncRNAs could serve as potential targets for precision medicine, opening up new avenues for developing more effective cancer immunotherapies. By compiling recent research on lncRNAs across various cancers, this review aims to shed light on their mechanisms within the tumor immune microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e39ed55d88fbecad6fec7ba9004faa9355e61712" target='_blank'>
              Exploring potential roles of long non-coding RNAs in cancer immunotherapy: a comprehensive review
              </a>
            </td>
          <td>
            A. Arshi, E. Mahmoudi, Farzaneh Raeisi, Masoud Dehghan Tezerjani, Elham Bahramian, Yeasin Ahmed, Chun Peng
          </td>
          <td>2024-08-27</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 <p>Our recent observations show that both resistant and stem-like cancer cells predominantly responsible for metastasis differ from chemotherapy-sensitive cells. We have shown bioinformatically and experimentally that mitochondria of such cells are much more prone to oxidative phosphorylation (OXPHOS) than radio- or chemotherapy-sensitive cancer cells from which they evolved during therapeutic interventions. Specifically, in triple-negative breast cancer models, we observed that such resistant cells exhibit higher mitochondrial membrane potential, higher OXPHOS and respiration, and increased resistance to oxidative stress, allowing them to survive chemo-radiotherapy. &nbsp;These findings of increased expression of OXPHOS-associated genes and proteins in chemoresistant cells and biopsies of relapsed tumors suggest an alternative druggable target. Our in vitro and in vivo (nude mice and Artemia salina) data suggest that certain antibiotics, inducers of mitochondrial dysfunction, create additive oxidative stress and can reduce the growth rate of tumors developed from resistant or stem-like cancer cells. Such repurposed drugs, selected from a chemical library, are also able to resensitize resistant tumors, allowing reuse of chemotherapeutic agents. In addition, their modification with a specific moiety (TPP) allows for increased delivery to mitochondria to reduce cytotoxic pressure on normal cells. Thus, research from our laboratory offers an alternative strategy for anticancer therapy of resistant tumors.</p>
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/795a93e4ffbe5463213b126e0efc54a08e0a2441" target='_blank'>
              MITOCHONDRIAL TARGETING AS A MEANS OF OVERCOMING CANCER DRUG RESISTANCE
              </a>
            </td>
          <td>
            Cemile Uslu, Eda Kapan, Hagar Fadda, Raheema Hassan Khan, Yara Shqair, Zeynep Ülker, Can Tunçay, Etna Abad, A. Lyakhovich
          </td>
          <td>2024-08-29</td>
          <td>RedoXplore</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="A central challenge in the quest for precise gene regulation within mammalian cells is the development of regulatory networks that can achieve perfect adaptation-where outputs consistently return to a set baseline post-stimulus. Here, we present such a system that leverages the CRISPR activation (CRISPRa) and anti-CRISPR proteins as two antithetic elements to establish perfect adaptation in mammalian cells and dynamically regulate gene expression. We demonstrate that this system can maintain stable expression levels of target genes in the face of external perturbations, thus providing a robust platform for biological applications. The versatility of our system is further showcased through its integration with endogenous regulatory mechanisms in T cells, such as the NF-κB-mediated immune response, and its ability to program apoptosis responses for precise spatial and temporal control of cellular growth and death. This study not only advances our understanding of gene regulation in mammalian cells but also opens new avenues for therapeutic intervention, particularly in diseases characterized by dysregulated gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9bdcf363536b5c7b6585a226adb59f423a716ec" target='_blank'>
              CRISPR perfect adaptation for robust control of cellular immune and apoptotic responses.
              </a>
            </td>
          <td>
            Yichi Zhang, Shuyi Zhang
          </td>
          <td>2024-08-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Kirsten Rat Sarcoma (KRAS) is the most commonly mutated oncogene in colorectal carcinoma (CRC). We have previously reported the interactions between microsatellite instability (MSI), DNA promoter methylation, and gene expression. In this study, we looked for associations between KRAS mutation, gene expression, and methylation that may help with precision medicine. Genome-wide gene expression and DNA methylation were done in paired CRC tumor and surrounding healthy tissues. The results suggested that (a) the magnitude of dysregulation of many major gene pathways in CRC was significantly greater in patients with the KRAS mutation, (b) the up- and down-regulation of these dysregulated gene pathways could be correlated with the corresponding hypo- and hyper-methylation, and (c) the up-regulation of CDKN2A was more pronounced in tumors with the KRAS mutation. A recent cell line study showed that there were higher CDKN2A levels in 5-FU-resistant CRC cells and that these could be down-regulated by Villosol. Our findings suggest the possibility of a better response to anti-CDKN2A therapy with Villosol in KRAS-mutant CRC. Also, the more marked up-regulation of genes in the proteasome pathway in CRC tissue, especially with the KRAS mutation and MSI, may suggest a potential role of a proteasome inhibitor (bortezomib, carfilzomib, or ixazomib) in selected CRC patients if necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40b3e427194ea41aedea0dfcb0cf1b6c9f955b47" target='_blank'>
              Association of KRAS Mutation and Gene Pathways in Colorectal Carcinoma: A Transcriptome- and Methylome-Wide Study and Potential Implications for Therapy
              </a>
            </td>
          <td>
            F. Jasmine, Armando Almazan, Yuliia Khamkevych, Marc Bissonnette, Habibul Ahsan, M. Kibriya
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Antibiotic resistance is the leading cause of death globally, with a higher possibility of the emergence of highly resistant pathogens, leading to epidemics. Several antibiotic resistance mechanisms have been discovered, such as enhanced efflux of antibiotics, reduced influx of antibiotics, alteration of antibiotics or their targets, and adaptation to antibiotics. However, this mechanism cannot fully explain the development of antibiotic resistance because the genes associated with this mechanism have been elucidated. However, the factors governing their regulation are not yet fully understood. Recent studies have highlighted the epigenetic and epitranscriptomic roles of antibiotic resistance development-associated genes. Epigenetic modification is associated with DNA modification, whereas epitranscriptomic modification is associated with RNA modification to control gene expression by regulating various biological phenomena such as splicing, translation, and stability. Therefore, this review will focus on the discovery of epigenetic modifications, particularly by DNA methyltransferases, such as restriction-modification (R-M) systems associated with methyltransferases, orphan DNA methyltransferases, and nucleoid-associated proteins that contribute to the development of antibiotic resistance. This scrutinization further expands to epitranscriptomic modification of non-coding RNA, which has a role in the regulation of antibiotic resistance. Epitranscriptomic modification of ribosomal RNA (rRNA), which is a major target of antibiotics, has been well explored. while non-coding RNA such as cis and trans small non coding RNA, and riboswitches are poorly explored. This epigenetic and epitranscriptomic modification will help to understand the regulation of antibiotic resistance-associated genes, which will help to identify key regulators of antibiotic resistance, paving the way for new antibiotic discovery, leading to decreased antibiotic mortality globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b16c6a0ffde83445547b9ffde6ff971186ffa3" target='_blank'>
              A comprehensive review on epigenetic and epitranscriptomic-mediated regulation of antibiotic resistance
              </a>
            </td>
          <td>
            P. K. Giri, Shahil Alam, Madhav Dhakal
          </td>
          <td>2024-07-24</td>
          <td>F1000Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chronic Lymphocytic Leukemia (CLL) is the most common B-cell malignancy in the Western world, characterized by frequent relapses despite temporary remissions. Our study integrated publicly available proteomic, transcriptomic, and patient survival datasets to identify key differences between healthy and CLL samples. We exposed approximately 1000 proteins that differentiate healthy from cancerous cells, with 608 upregulated and 415 downregulated in CLL cases. Notable upregulated proteins include YEATS2 (an epigenetic regulator), PIGR (Polymeric immunoglobulin receptor), and SNRPA (a splicing factor), which may serve as prognostic biomarkers for this disease. Key pathways implicated in CLL progression involve RNA processing, stress resistance, and immune response deficits. Furthermore, we identified three existing drugs—Bosutinib, Vorinostat, and Panobinostat—for potential further investigation in drug repurposing in CLL. We also found limited correlation between transcriptomic and proteomic data, emphasizing the importance of proteomics in understanding gene expression regulation mechanisms. This generally known disparity highlights once again that mRNA levels do not accurately predict protein abundance due to many regulatory factors, such as protein degradation, post-transcriptional modifications, and differing rates of translation. These results demonstrate the value of integrating omics data to uncover deregulated proteins and pathways in cancer and suggest new therapeutic avenues for CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524d87cfc3fcaee95b1a168a590bc69c95feb91a" target='_blank'>
              Integrative Analysis of Multi-Omics Data to Identify Deregulated Molecular Pathways and Druggable Targets in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            D. Mavridou, K. Psatha, M. Aivaliotis
          </td>
          <td>2024-08-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554fe13daafca3f151510eb83bbcfb8701b7284c" target='_blank'>
              Genome-scale CRISPR-Cas9 screening in stem cells: theories, applications and challenges
              </a>
            </td>
          <td>
            Heng Zhou, Peng Ye, W. Xiong, Xingxiang Duan, Shuili Jing, Yan He, Zhi Zeng, Yen Wei, Qingsong Ye
          </td>
          <td>2024-07-19</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A major goal in biology is to uncover the relationship between genotype and phenotype. However, identifying gene function is often hampered by genetic redundancy. For example, under standard laboratory conditions, three-quarters of the genes in the human pathogen Streptococcus pneumoniae are non-essential. A powerful approach to unravel genetic redundancy is by identifying gene-gene interactions. To uncover genetic interactions (GIs) in S. pneumoniae on a genome-wide scale, a generally applicable dual CRISPRi-Seq method and associated analysis pipeline was developed. Specifically, we created a library of 869 dual sgRNAs targeting high-confidence operons that encode essential and non-essential genes, covering over 70% of the genetic elements in the pneumococcal genome. Testing these 378,015 unique combinations, 4,026 significant GIs were identified, including 1,935 negative and 2,091 positive interactions. Besides known GIs, we found and confirmed previously unknown interactions involving genes responsible for fundamental cellular processes such as cell division, cell shape maintenance, and chromosome segregation. The presented methods and bioinformatic approaches can serve as a roadmap for genome-wide gene interaction studies in other organisms. Lastly, all interactions are available for exploration via the Pneumococcal Genetic Interaction Network (PneumoGIN) at https://veeninglab.shinyapps.io/PneumoGIN, which can serve as a starting point for new biological discoveries and translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dd78fdd99cdc75fb8cdbb9e56ea5f0c3b73930" target='_blank'>
              Dual CRISPRi-Seq for genome-wide genetic interaction studies identifies key genes involved in the pneumococcal cell cycle
              </a>
            </td>
          <td>
            Julien Dénéréaz, Elise Eray, Bimal Jana, Vincent de Bakker, Horia Todor, Tim van Opijnen, Xue Liu, J. Veening
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/553f4560eb657fca2fe86385e460c71427df430c" target='_blank'>
              GLI1 confers resistance to PARP inhibitors by activating the DNA damage repair pathway.
              </a>
            </td>
          <td>
            Hiroshi Ikeuchi, Yusuke Matsuno, Rika Kusumoto-Matsuo, S. Kojima, Toshihide Ueno, Masachika Ikegami, Rina Kitada, Hitomi Sumiyoshi-Okuma, Yuki Kojima, K. Yonemori, Y. Yatabe, K. Takamochi, Kenji Suzuki, Ken‐ichi Yoshioka, H. Mano, S. Kohsaka
          </td>
          <td>2024-08-03</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="This report presents the largest collection of gamma-delta T cell receptor (γδ TCR) reads in human cancer to date, analyzing about 11,000 patient tumor samples across 33 cancer types using the TRUST4 algorithm. Despite γδ T cells being a small fraction of the T cell population, they play a key role in both innate and adaptive immunity. Our comprehensive analysis reveals their significant presence across all cancer types, specifically highlighting the diverse spectrum and clonality patterns of their γδ receptors. This research highlights the complex roles of γδ T cells in tumor tissues and their potential as prognostic biomarkers. We also demonstrate the utility of T cell receptor gamma (TRG) and delta (TRD) gene expression values from standard RNA-seq data. Ultimately, our work establishes a fundamental resource for future tumor-infiltrating γδ T cell research and may facilitate the development of novel γδ-T-cell-based therapeutic strategies. Together, we demonstrate the strong diversity and prognostic potential of γδ T cells in multiple cancer types. Highlights Comprehensive analysis of γδ TCRs from 11,473 tumor samples Significant variability and overall consistency in γδ gene expression and clonotype γδ TCR expression and diversity as prognostic biomarkers across multiple cancers Centralized γδ TCR repertoire database for future therapeutic discovery">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f81b0341b0c9b28947c09231986dc63cf346223" target='_blank'>
              A pan-cancer gamma delta T cell repertoire
              </a>
            </td>
          <td>
            Xiaoqing Yu, Li Song, Ling Cen, Biwei Cao, Ranran Tao, Yuanyuan Shen, Daniel Abate- Daga, Paulo C. Rodriguez, Jose R Conejo-Garcia, Xuefeng Wang
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d6d23242f7d6bebc1af13225dc6901681b6a7ca" target='_blank'>
              The roles and mechanisms of coding and noncoding RNA variations in cancer.
              </a>
            </td>
          <td>
            Sang Yean Kim, M. Na, Sungpil Yoon, Eunbi Shin, Jin Woong Ha, Soyoung Jeon, Suk Woo Nam
          </td>
          <td>2024-09-02</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2fd505ae2ea1de627590bede77cfbae2e3ab46" target='_blank'>
              CRISPR screen of venetoclax response-associated genes identifies transcription factor ZNF740 as a key functional regulator
              </a>
            </td>
          <td>
            Lixia Zhang, Xinyue Zhou, Sajesan Aryal, Virginia Veasey, Pengcheng Zhang, Fu Jun Li, Yu Luan, Ravi Bhatia, Yang Zhou, Rui Lu
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Stable isotope-resolved metabolomics delineates reprogrammed intersecting metabolic networks in human cancers. Knowledge gained from in vivo patient studies provides the "benchmark" for cancer models to recapitulate. It is particularly difficult to model patients' tumor microenvironment (TME) with its complex cell-cell/cell-matrix interactions, which shapes metabolic reprogramming crucial to cancer development/drug resistance. Patient-derived organotypic tissue cultures (PD-OTCs) represent a unique model that retains an individual patient's TME. PD-OTCs of non-small-cell lung cancer better recapitulated the in vivo metabolic reprogramming of patient tumors than the patient-derived tumor xenograft (PDTX), while enabling interrogation of immunometabolic response to modulators and TME-dependent resistance development. Patient-derived organoids (PDOs) are also good models for reconstituting TME-dependent metabolic reprogramming and for evaluating therapeutic responses. Single-cell based 'omics on combinations of PD-OTC and PDO models will afford an unprecedented understanding on TME dependence of human cancer metabolic reprogramming, which should translate into the identification of novel metabolic targets for regulating TME interactions and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6597c82613b1bff5869b53a6356a5e51a2359cc" target='_blank'>
              Metabolic Reprogramming in Human Cancer Patients and Patient-Derived Models.
              </a>
            </td>
          <td>
            T. W. Fan, R. Higashi, A. Lane
          </td>
          <td>2024-07-15</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ac60c2f3f46f34f9c357bd7653935d6b617e019" target='_blank'>
              Harnessing the evolving CRISPR/Cas9 for precision oncology
              </a>
            </td>
          <td>
            Tianye Li, Shuiquan Li, Yue Kang, Jianwei Zhou, Ming Yi
          </td>
          <td>2024-08-08</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340cca23bf7d37c3ef78e0576f93a838fbeeb239" target='_blank'>
              Integrated analysis of spatial transcriptomics and CT phenotypes for unveiling the novel molecular characteristics of recurrent and non-recurrent high-grade serous ovarian cancer
              </a>
            </td>
          <td>
            Hye-Yeon Ju, Seo Yeon Youn, Jun Kang, Min Yeop Whang, Youn Jin Choi, Mi-Ryung Han
          </td>
          <td>2024-08-12</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2256152e45d11bd82f68a7e3810e940b41cb02" target='_blank'>
              Essential gene screening identifies the bromodomain-containing protein BRPF1 as a new actionable target for endocrine therapy-resistant breast cancers
              </a>
            </td>
          <td>
            A. Salvati, G. Giurato, J. Lamberti, Ilaria Terenzi, Laura Crescenzo, Viola Melone, L. Palo, A. Giordano, Francesco Sabbatino, G. Roscigno, C. Quintavalle, G. Condorelli, Francesca Rizzo, Roberta Tarallo, G. Nassa, Alessandro Weisz
          </td>
          <td>2024-08-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b62bbc42b475a59548fce65b7db966d5dba12012" target='_blank'>
              CRISPRi–TnSeq maps genome-wide interactions between essential and non-essential genes in bacteria
              </a>
            </td>
          <td>
            Bimal Jana, Xue Liu, Julien Dénéréaz, Hongshik Park, Dmitry Leshchiner, Bruce Liu, Clément Gallay, Junhao Zhu, J. Veening, Tim van Opijnen
          </td>
          <td>2024-07-19</td>
          <td>Nature Microbiology</td>
          <td>2</td>
          <td>46</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="The search for genetic variants that act as causative factors in human diseases by disrupting the normal splicing process has primarily focused on single nucleotide variants (SNVs). It is worth noting that insertions or deletions (indels) have also been sporadically reported as causative disease variants through their potential impact on the splicing process. In this study, to perform identification of indels inducing exon extension/shrinkage events, we used individual-specific genomes and RNA sequencing (RNA-seq) data pertaining to the corresponding individuals and identified 12 exon extension/shrinkage events that were potentially induced by indels that disrupted authentic splice sites or created novel splice sites in 235 normal individuals. By evaluating the impact of these abnormal splicing events on the resulting transcripts, we found that five events led to the generation of premature termination codons (PTCs), including those occurring within genes associated with genetic disorders. Our analysis revealed that the potential functions of indels have been underexamined, and it is worth considering the possibility that indels may affect splice site usage, using RNA-seq data to discover novel potentially disease-associated mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62ba041d07636b21a672bb99a3107362cc90dc0c" target='_blank'>
              Identification and analysis of short indels inducing exon extension/shrinkage events.
              </a>
            </td>
          <td>
            Zhuo Qu, Narumi Sakaguchi, Chie Kikutake, Mikita Suyama
          </td>
          <td>2024-07-31</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Identifying host genetic factors modulating immune checkpoint inhibitor (ICI) efficacy has been experimentally challenging because of variations in both host and tumor genomes, differences in the microbiome, and patient life exposures. Utilizing the Collaborative Cross (CC) multi-parent mouse genetic resource population, we developed an approach that fixes the tumor genomic configuration while varying host genetics. With this approach, we discovered that response to anti-PD-1 (aPD1) immunotherapy was significantly heritable in four distinct murine tumor models (H2 between 0.18-0.40). For the MC38 colorectal carcinoma system (H2 = 0.40), we mapped four significant ICI response quantitative trait loci (QTL) localized to mouse chromosomes (mChr) 5, 9, 15 and 17, and identified significant epistatic interactions between specific QTL pairs. Differentially expressed genes within these QTL were highly enriched for immune genes and pathways mediating allograft rejection and graft vs host disease. Using a cross species analytical approach, we found a core network of 48 genes within the four QTLs that showed significant prognostic value for overall survival in aPD1 treated human cohorts that outperformed all other existing validated immunotherapy biomarkers, especially in human tumors of the previously defined immune subtype 4. Functional blockade of two top candidate immune targets within the 48 gene network, GM-CSF and high affinity IL-2/IL-15 signaling, completely abrogated the MC38 tumor transcriptional response to aPD1 therapy in vivo. Thus, we have established a powerful cross species in vivo platform capable of uncovering host genetic factors that establish the tumor immune microenvironment configuration propitious for ICI response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f7a70d713da2ee90a6af227b41627e893515c66" target='_blank'>
              Mapping the genetic landscape establishing a tumor immune microenvironment favorable for anti-PD-1 response in mice and humans
              </a>
            </td>
          <td>
            Daniel A. Skelly, John P Graham, Mingshan Cheng, Mayuko Furuta, Andrew Walter, Thomas A. Stoklasek, Hongyuan Yang, Timothy M. Stearns, Olivier B. Poirion, Ji-Gang Zhang, J. Grassmann, D. Luo, William F. Flynn, Elise T. Courtois, Chih-Hao Chang, D. Serreze, F. Menghi, L. Reinholdt, Edison T. Liu
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac67a3ed7d66d72ed2d1b0afc516f030fad809a" target='_blank'>
              Immunological hide-and-seek: epigenetically reprogrammed cancer cells and the dynamics of CD8+ T cells.
              </a>
            </td>
          <td>
            Jie-Ting Low, Michael W Y Chan, Cheng-Huang Shen, Kuo-Liang Wei
          </td>
          <td>2024-09-04</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Mutation in p53 is the most frequent event in cancer development and a leading cause of cancer therapy resistance due to evasion of the apoptosis cascade. Beyond chemotherapies and radiation therapies, growing evidence indicates that p53-mutant tumors are resistant to a broad range of immune-based therapies, such as immune checkpoint inhibitors, chimeric antigen receptor (CAR) T, and hematopoietic stem cell transplantation (HSCT). This highlights the role of p53 mutations in driving immune evasion of tumor cells. In this review, we first summarize recent studies revealing mechanisms by which p53-mutant tumors evade immune surveillance from T cells, natural killer (NK) cells, and macrophages. We then review how these mutant tumor cells reshape the tumor microenvironment (TME), modulating bystander cells such as macrophages, neutrophils, and regulatory T (Treg) cells to foster immunosuppression. Additionally, we review clinical observations indicative of immune evasion associated with p53 loss or mutations. Finally, we discuss therapeutic strategies to enhance immune response in p53 wild-type (WT) or mutant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ac96dfaadf6e3a60d7f3146c1f00c2f180c35b" target='_blank'>
              TP53 Mutation-Mediated Immune Evasion in Cancer: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            Chuqi Wang, Jordan Yong Ming Tan, Nishtha Chitkara, Shruti Bhatt
          </td>
          <td>2024-09-03</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66407cd2989bbfac769e0f4b6906811d3ffd112d" target='_blank'>
              Epigenetic modification of ferroptosis by non-coding RNAs in cancer drug resistance
              </a>
            </td>
          <td>
            Hongquan Wang, Joshua S Fleishman, Sihang Cheng, Weixue Wang, Fan Wu, Yumin Wang, Yu Wang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Changes in gene expression play a fundamental role in the process of adaptation. How-ever, single gene expression levels cannot evolve independently but form highly correlated groups, often corresponding to functional modules. Any evolutionary response is therefore the product of direct selection acting on a respective gene and indirect selection acting on genetically correlated partners. Here, we studied the mechanisms behind evolution in expression in response to a change in climate conditions using whole transcriptomes and fitness data of the red flour beetle Tribolium castaneum before and after adaptation in seven independent selection lines. Genetic covariance between expression levels, as well as with fitness, could predict evolutionary responses and allowed us to quantify contri-butions from direct and indirect selection. These predictions were highly consistent with observed changes after 20 generations of adaptation and indicate a pivotal role of indirect selection. Consequently, genes with central positions in gene networks experienced the strongest selection and exhibited larger evolutionary changes in expression. Our genomic analysis revealed that selection on expression levels drives parallel allele frequency changes in the respective genes. More connected genes and those carrying expression quantita-tive trait loci (eQTLs) showed a higher degree of parallel evolution. We demonstrated how a quantitative genetics approach combined with transcriptomics and network anal-ysis provides deep insights into the evolution of gene expression: Network structure can greatly enhance expression evolution through indirect selection when concerted selection pressures act on the functional properties of expression modules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a2b684fa22c2ccb17581fe35f4bf849c110280b" target='_blank'>
              Gene expression evolution is predicted by selection, genetic covariance and network topology
              </a>
            </td>
          <td>
            E. Koch, Charles Rocabert, Champak Beeravolu Reddy, F. Guillaume
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) has a poor survival rate for both pediatric and adult patients due to its frequent relapse. To elucidate the bioenergetic principle underlying AML relapse, we investigated the transcriptional regulation of mitochondrial–nuclear dual genomes responsible for metabolic plasticity in treatment-resistant blasts. Both the gain and loss of function results demonstrated that NFκB2, a noncanonical transcription factor (TF) of the NFκB (nuclear factor kappa-light-chain-enhancer of activated B cells) family, can control the expression of TFAM (mitochondrial transcription factor A), which is known to be essential for metabolic biogenesis. Furthermore, genetic tracking and promoter assays revealed that NFκB2 is in the mitochondria and can bind the specific “TTGGGGGGTG” region of the regulatory D-loop domain to activate the light-strand promoter (LSP) and heavy-strand promoter 1 (HSP1), promoters of the mitochondrial genome. Based on our discovery of NFκB2′s novel function of regulating mitochondrial–nuclear dual genomes, we explored a novel triplet therapy including inhibitors of NFκB2, tyrosine kinase, and mitochondrial ATP synthase that effectively eliminated primary AML blasts with mutations of the FMS-related receptor tyrosine kinase 3 (FLT3) and displayed minimum toxicity to control cells ex vivo. As such, effective treatments for AML must include strong inhibitory actions on the dual genomes mediating metabolic plasticity to improve leukemia prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd91cb6b195c34903e27bb3147d1ba2db984023c" target='_blank'>
              Discovery of NFκB2-Coordinated Dual Regulation of Mitochondrial and Nuclear Genomes Leads to an Effective Therapy for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Yi Xu, David J. Baylink, Jeffrey Xiao, Lily Tran, Vinh Nguyen, Brandon Park, Ismael Valladares, Scott Lee, Kevin Codorniz, Laren D Tan, Chien-Shing Chen, Hisham Abdel-Azim, Mark E. Reeves, Hamid Mirshahidi, Guido Marcucci, H. Cao
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, André Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Mutations in subunits of the SWItch Sucrose Non-Fermentable (SWI/SNF) chromatin remodeling complex occur in ≈20% of cancers and represent a highly unmet medical need. To identify novel therapeutic approaches, we systematically characterized transcriptomic and proteomic changes caused by the loss of SWI/SNF subunits or other epigenetic enzymes in isogenic cell lines, which we subsequently integrated with high-throughput drug screening and independent genetic screens of the DepMap project. Using an optimized bioinformatics pipeline for pathway enrichment, we identified Metabolism of proteins as the most frequently dysregulated Reactome pathway category in SWI/SNF-defective cell lines. Drug screening and multiomic integration revealed multiple chemicals selectively cytotoxic for SWI/SNF-defective models, including CBP/EP300 or mitochondrial respiration inhibitors. A novel algorithm for the analysis of DepMap CRISPR screens independently identified synthetic lethality between SWI/SNF defects and EP300 or mitochondrial respiration genes, which we further revalidated in disease-relevant models. These results unravel novel genetic dependencies for SWI/SNF-defective cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/494552ad585e5d90eeb24a7f96c16c3381361c3e" target='_blank'>
              Integrated multiomic profiling reveals SWI/SNF subunit-specific pathway alterations and targetable vulnerabilities
              </a>
            </td>
          <td>
            Jorge Bretones Santamarina, Clémence Astier, M. Garrido, L. C. Daage, Theodoros I. Roumeliotis, Elodie Anthony, M. Pardo, Marianne Chasseriaud, P. Gestraud, Carine Ngo, Daphné Morel, Roman Chabanon, Jyoti S. Choudhary, E. Del Nery, Sophie Postel-Vinay, Annabelle Ballesta
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Malignant gliomas present great difficulties in treatment, with little change over the past 30 years in the median survival time of 15 months. Current treatment options include surgery, radiotherapy (RT), and chemotherapy. New therapies aimed at suppressing the formation of new vasculature (antiangiogenic treatments) or destroying formed tumor vasculature (vascular disrupting agents) show promise. This study summarizes the existing knowledge regarding the processes by which glioblastoma (GBM) tumors acquire resistance to antiangiogenic treatments. The discussion encompasses the activation of redundant proangiogenic pathways, heightened tumor cell invasion and metastasis, resistance induced by hypoxia, creation of vascular mimicry channels, and regulation of the tumor immune microenvironment. Subsequently, we explore potential strategies to overcome this resistance, such as combining antiangiogenic therapies with other treatment methods, personalizing treatments for each patient, focusing on new therapeutic targets, incorporating immunotherapy, and utilizing drug delivery systems based on nanoparticles. Additionally, we would like to discuss the limitations of existing methods and potential future directions to enhance the beneficial effects of antiangiogenic treatments for patients with GBM. Therefore, this review aims to enhance the research outcome for GBM and provide a more promising opportunity by thoroughly exploring the mechanisms of resistance and investigating novel therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fb43147dc6c7d13576f5540a149d0b8ee6fb97f" target='_blank'>
              Recent Treatment Strategies and Molecular Pathways in Resistance Mechanisms of Antiangiogenic Therapies in Glioblastoma
              </a>
            </td>
          <td>
            Md Ataur Rahman, Meser M. Ali
          </td>
          <td>2024-08-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The fusion expression of DNA replication-related proteins with nucleotide deaminase enzymes promotes random mutations in bacterial genomes, thereby increasing genetic diversity among population. Most previous studies have focused on cytosine deaminase, which produces only C→T mutations, significantly limiting the variety of mutation types. In this study, we developed a fusion expression system by combining DnaG (RNA primase) with adenine deaminase TadA-8e (DnaG-TadA) in Escherichia coli, which is capable of rapidly introducing A→G mutations into the E. coli genome, resulting in a 664-fold increase in terms of mutation rate. Additionally, we engineered a dual-functional TadA variant, TadAD, and then fused it with DnaG. This construct introduced both C→T and A→G mutations into the E. coli genome, with the mutation rate further increased by 370-fold upon co-expression with an uracil glycosylase inhibitor (DnaG-TadAD-UGI). We applied DnaG-TadA and DnaG-TadAD-UGI systems to the adaptive laboratory evolution for Cd2+ and kanamycin resistance, achieving an 8.0 mM Cd2+ and 200 μg/mL kanamycin tolerance within just 17 days and 132 hours, respectively. Compared to conventional evolution methods, the final tolerance levels were increased by 320% and 266%, respectively. Our work offers a novel strategy for random mutagenesis in E. coli and potentially other prokaryotic species. For TOC only">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8dc3e641872269a76809a2a9f58812a1c2451c2" target='_blank'>
              Genome-wide A→G and C→T Mutations Induced by Functional TadA Variants in Escherichia coli
              </a>
            </td>
          <td>
            Hao Wang, Zhengxin Dong, Jingyi Shi, Lei Chen, Tao Sun, Weiwen Zhang
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Combined CDK4/6 inhibitor (CDK4/6i) and endocrine therapy significantly improves outcome for patients with estrogen receptor-positive (ER+) metastatic breast cancer, but drug resistance and thus disease progression inevitably occur. Herein, we aimed to identify genomic alterations associated with combined CDK4/6i and endocrine therapy resistance, and follow the levels of specific mutations in longitudinal circulating tumor DNA (ctDNA) for early detection of progression. From a cohort of 86 patients with ER+ metastatic breast cancer we performed whole exome sequencing or targeted sequencing of paired tumor (N = 8) or blood samples (N = 5) obtained before initiation of combined CDK4/6i and endocrine therapy and at disease progression. Mutations in oncogenic genes at progression were rare, while amplifications of growth-regulating genes were more frequent. The most frequently acquired alterations observed were PIK3CA and TP53 mutations and PDK1 amplification. Longitudinal ctDNA dynamics of mutant PIK3CA or private mutations revealed increased mutation levels at progression in 8 of 10 patients (80%). Impressively, rising levels of PIK3CA-mutated ctDNA were detected 4-17 months before imaging. Our data add to the growing evidence supporting longitudinal ctDNA analysis for real-time monitoring of CDK4/6i response and early detection of progression in advanced breast cancer. Further, our analysis suggests that amplification of growth-related genes may contribute to combined CDK4/6i and endocrine therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4f8f900b692158e74287adf0a693a5a4d08296" target='_blank'>
              Genomic alterations associated with resistance and circulating tumor DNA dynamics for early detection of progression on CDK4/6 inhibitor in advanced breast cancer.
              </a>
            </td>
          <td>
            Charlotte K Kindt, Carla L. Alves, S. Ehmsen, Amalie Kragh, T. Reinert, Marianne Vogsen, A. R. Kodahl, Jeanette D Rønlev, Dilan Ardik, Anna L Sørensen, Kirstine Evald, Mia L Clemmensen, J. Staaf, Henrik J Ditzel
          </td>
          <td>2024-08-11</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Phenotypic differences between individuals of a species are often caused by differences in gene expression, which are in turn caused by genetic variation. Expression quantitative trait locus (eQTL) analysis is a methodology by which we can identify such causal variants. Scaling eQTL analysis is costly due to the expense of generating mapping populations, and the collection of matched transcriptomic and genomic information. We developed a rapid eQTL analysis approach using single-cell/nucleus RNA sequencing of gametes from a small number of heterozygous individuals. Patterns of inherited polymorphisms are used to infer the recombinant genomes of thousands of individual gametes and identify how different haplotypes correlate with variation in gene expression. Applied to Arabidopsis pollen nuclei, our approach uncovers both cis- and trans-eQTLs, ultimately mapping variation in a master regulator of sperm cell development that affects the expression of hundreds of genes. This establishes snRNA-sequencing as a powerful, cost-effective method for the mapping of meiotic recombination, addressing the scalability challenges of eQTL analysis and enabling eQTL mapping in specific cell-types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8df71f2a99165419e506f46db58d2987757dd044" target='_blank'>
              Expression quantitative trait locus mapping in recombinant gametes using single nucleus RNA sequencing
              </a>
            </td>
          <td>
            Matthew T. Parker, Samija Amar, J. A. Campoy, Kristin Krause, Sergio Tusso, Magdalena Marek, Bruno Huettel, Korbinian Schneeberger
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, A. Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f217995bbe1b34f9aeac0d831228d08545af9e" target='_blank'>
              Decoding single-cell molecular mechanisms in astrocyte-to-iN reprogramming via Ngn2- and Pax6-mediated direct lineage switching
              </a>
            </td>
          <td>
            Rongxing Qin, Yingdan Zhang, Yue Yang, Jiafeng Chen, Lijuan Huang, Wei Xu, Qingchun Qin, Xiaojun Liang, Xinyu Lai, Xiaoying Huang, Minshan Xie, Li Chen
          </td>
          <td>2024-07-27</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer drug resistance poses a significant obstacle to successful chemotherapy, primarily driven by the activity of ATP-binding cassette (ABC) transporters, which actively efflux chemotherapeutic agents from cancer cells, reducing their intracellular concentrations and therapeutic efficacy. Recent studies have highlighted the pivotal role of long noncoding RNAs (lncRNAs) in regulating this resistance, positioning them as crucial modulators of ABC transporter function. lncRNAs, once considered transcriptional noise, are now recognized for their complex regulatory capabilities at various cellular levels, including chromatin modification, transcription, and post-transcriptional processing. This review synthesizes current research demonstrating how lncRNAs influence cancer drug resistance by modulating the expression and activity of ABC transporters. lncRNAs can act as molecular sponges, sequestering microRNAs that would otherwise downregulate ABC transporter genes. Additionally, they can alter the epigenetic landscape of these genes, affecting their transcriptional activity. Mechanistic insights reveal that lncRNAs contribute to the activity of ABC transporters, thereby altering the efflux of chemotherapeutic drugs and promoting drug resistance. Understanding these interactions provides a new perspective on the molecular basis of chemoresistance, emphasizing the regulatory network of lncRNAs and ABC transporters. This knowledge not only deepens our understanding of the biological mechanisms underlying drug resistance but also suggests novel therapeutic strategies. In conclusion, the intricate interplay between lncRNAs and ABC transporters is crucial for developing innovative solutions to combat cancer drug resistance, underscoring the importance of continued research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/964e0709fe20c21443ecd0c53a8a7ab6cf10b8c7" target='_blank'>
              lncRNAs: New players of cancer drug resistance via targeting ABC transporters.
              </a>
            </td>
          <td>
            Mohammad Ebrahimnezhad, Sanaz Hassanzadeh Asl, Maede Rezaie, Mehran Molavand, Bahman Yousefi, M. Majidinia
          </td>
          <td>2024-08-01</td>
          <td>IUBMB life</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="A fundamental challenge in biomedicine is understanding the mechanisms predisposing individuals to disease. While previous research has suggested that switch-like gene expression is crucial in driving biological variation and disease susceptibility, a systematic analysis across multiple tissues is still lacking. By analyzing transcriptomes from 943 individuals across 27 tissues, we identified 1,013 switch-like genes. We found that only 31 (3.1%) of these genes exhibit switch-like behavior across all tissues. These universally switch-like genes appear to be genetically driven, with large exonic genomic structural variants explaining five (∼18%) of them. The remaining switch-like genes exhibit tissue-specific expression patterns. Notably, tissue-specific switch-like genes tend to be switched on or off in unison within individuals, likely under the influence of tissue-specific master regulators, including hormonal signals. Among our most significant findings, we identified hundreds of concordantly switched-off genes in the stomach and vagina that are linked to gastric cancer (41-fold, p<10-4) and vaginal atrophy (44-fold, p<10-4), respectively. Experimental analysis of vaginal tissues revealed that low systemic levels of estrogen lead to a significant reduction in both the epithelial thickness and the expression of the switch-like gene ALOX12. We propose a model wherein the switching off of driver genes in basal and parabasal epithelium suppresses cell proliferation therein, leading to epithelial thinning and, therefore, vaginal atrophy. Our findings underscore the significant biomedical implications of switch-like gene expression and lay the groundwork for potential diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d158a86f2a151fa830972300750ba84c1a11dbfc" target='_blank'>
              Switch-like Gene Expression Modulates Disease Susceptibility
              </a>
            </td>
          <td>
            Alber Aqil, Yanyan Li, Zhiliang Wang, Saiful Islam, Madison Russell, T. Kunovac Kallak, Marie Saitou, Omer Gokcumen, N. Masuda
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Successful tuberculosis therapy requires treatment with an unwieldy multidrug combination for several months. Thus, there is a growing need to identify novel genetic vulnerabilities that can be leveraged to develop new, more effective antitubercular drugs. Consequently, recent efforts to optimize TB therapy have exploited Mtb chemical genetics to identify pathways influencing antibiotic efficacy, novel mechanisms of antibiotic action, and new targets for TB drug discovery. However, the influence of the complex host environment on these interactions remains largely unknown, leaving the therapeutic potential of the identified targets unclear. In this study, we leveraged a library of conditional mutants targeting 467 essential Mtb genes to characterize the chemical-genetic interactions (CGIs) with TB drugs directly in the mouse infection model. We found that these in vivo CGIs differ significantly from those identified in vitro. Both drug-specific and drug-agnostic effects were identified, and many were preserved during treatment with a multidrug combination, suggesting numerous strategies for enhancing therapy. This work also elucidated the complex effects of pyrazinamide (PZA), a drug that relies on aspects of the infection environment for efficacy. Specifically, our work supports the importance of coenzyme A synthesis inhibition during infection, as well as the antagonistic effect of iron limitation on PZA activity. In addition, we found that inhibition of thiamine and purine synthesis increases PZA efficacy, suggesting novel therapeutically exploitable metabolic dependencies. Our findings present a map of the unique in vivo CGIs, characterizing the mechanism of PZA activity in vivo and identifying novel targets for TB drug development. Significance The inevitable rise of multi-drug-resistant tuberculosis underscores the urgent need for new TB drugs and novel drug targets while prioritizing synergistic drug combinations. Chemical-genetic interaction (CGI) studies have delineated bacterial pathways influencing antibiotic efficacy and uncovered druggable pathways that synergize with TB drugs. However, most studies are conducted in vitro, limiting our understanding of how the host environment influences drug-mutant interactions. Using an inducible mutant library targeting essential Mtb genes to characterize CGIs during infection, this study reveals that CGIs are both drug-specific and drug-agnostic and differ significantly from those observed in vitro. Synergistic CGIs comprised distinct metabolic pathways mediating antibiotic efficacy, revealing novel drug mechanisms of action, and defining potential drug targets that would synergize with frontline antitubercular drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d045b0a24f1162d0aaf03742b7cbebc2dd9fdd94" target='_blank'>
              Chemical genetic interactions elucidate pathways controlling tuberculosis antibiotic efficacy during infection
              </a>
            </td>
          <td>
            P. Oluoch, Eun-Ik Koh, Megan M Proulx, Charlotte J. Reames, K. Papavinasasundaram, Kenan C. Murphy, Matthew D. Zimmerman, Véronique Dartois, Chris Sassetti
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Ewing sarcoma (EwS) is an aggressive cancer of adolescents in need of effective treatments. Insulin like growth factor (IGF) 1 was previously reported an autocrine growth factor for EwS, but only 10% of patients responded to IGF-1 receptor blockade. Although presumed to originate from mesenchymal progenitors during bone development, targeting of the EwS driver oncogene EWS::FLI1 to the mesenchymal lineage in a conditional mouse model did not result in tumor formation but led to skeletal malformations and perinatal death. We report that transient exposure to IGF-1 concentrations mimicking serum levels during puberty reprogrammed limb-derived mesenchymal cells of EWS::FLI1-mutant mice to stable transformation and tumorigenicity. We identified a modular mechanism of IGF-1-driven tumor promotion in the early steps of EwS pathogenesis, in which Yap1 plays a central role. Pharmacologic Yap1/Tead inhibition reversed the transformed phenotype of EWS::FLI1 expressing cells. Our data provide a rationale for combined IGF-1R and YAP/TEAD inhibition in the treatment of EwS patients. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88becabeb7089545462f322b84b8f946831753a9" target='_blank'>
              YAP1 is a key regulator of EWS::FLI1-dependent malignant transformation upon IGF-1 mediated reprogramming of bone mesenchymal stem cells
              </a>
            </td>
          <td>
            Rahil Noorizadeh, Barbara Sax, T. Javaheri, Branka Radic Sarikas, Valerie Fock, M. Kauer, Aleksandr Bykov, Veveeyan Suresh, M. Schlederer, Lukas Kenner, Gerhard Weber, W. Mikulits, F. Halbritter, R. Moriggl, Heinrich Kovar
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W. Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Mismatch repair proficient (MMRp) tumors of colorectal origin are one of the prevalent yet unpredictable clinical challenges. Despite earnest efforts, optimal treatment modalities have yet to emerge for this class. The poor prognosis and limited actionability of MMRp are ascribed to a low neoantigen burden and a desert-like microenvironment. This review focuses on the critical roadblocks orchestrated by an immune evasive mechanistic milieu in the context of MMRp. The low density of effector immune cells, their weak spatiotemporal underpinnings, and the high-handedness of the IL-17-TGF-β signaling are intertwined and present formidable challenges for the existing therapies. Microbiome niche decorated by Fusobacterium nucleatum alters the metabolic program to maintain an immunosuppressive state. We also highlight the evolving strategies to repolarize and reinvigorate this microenvironment. Reconstruction of anti-tumor chemokine signaling, rational drug combinations eliciting T cell activation, and reprograming the maladapted microbiome are exciting developments in this direction. Alternative vulnerability of other DNA damage repair pathways is gaining momentum. Integration of liquid biopsy and ex vivo functional platforms provide precision oncology insights. We illustrated the perspectives and changing landscape of MMRp-CRC. The emerging opportunities discussed in this review can turn the tide in favor of fighting the treatment dilemma for this elusive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823c184571d6413de5f857c9ff7de13bd6953768" target='_blank'>
              Mismatch repair-proficient tumor footprints in the sands of immune desert: mechanistic constraints and precision platforms
              </a>
            </td>
          <td>
            Biswanath Majumder, N. Nataraj, Leela Maitreyi, Santanu Datta
          </td>
          <td>2024-07-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bbe642fe1a5320aff996d83dd54d13a05e4037" target='_blank'>
              The intersection of homologous recombination (HR) and mismatch repair (MMR) pathways in DNA repair-defective tumors.
              </a>
            </td>
          <td>
            L. Incorvaia, T. B. Bazan Russo, V. Gristina, A. Perez, C. Brando, Clarissa Mujacic, Emilia Di Giovanni, M. Bono, Silvia Contino, Carla Ferrante Bannera, Maria Concetta Vitale, Andrea Gottardo, M. Peri, A. Galvano, D. Fanale, G. Badalamenti, Antonio Russo, V. Bazan
          </td>
          <td>2024-09-05</td>
          <td>NPJ precision oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="PURPOSE
High-grade complex karyotype sarcomas are a heterogeneous group of tumors with a uniformly poor prognosis. Within complex karyotype sarcomas, there are innumerable genetic changes but identifying those that are clinically relevant has been challenging.


EXPERIMENTAL DESIGN
To address this, we utilized a pooled genetic screening approach, informed by TCGA data, to identify key drivers and modifiers of sarcoma development that were validated in vivo.


RESULTS
YAP1 and wildtype KRAS were validated as drivers and transformed human mesenchymal stem cells into two distinct sarcoma subtypes, undifferentiated pleomorphic sarcoma (UPS) and myxofibrosarcoma (MFS), respectively. A subset of tumors driven by CDK4 and PIK3CA reflected leiomyosarcoma (LMS) and osteosarcoma (OS) demonstrating the plasticity of this approach and the potential to investigate sarcoma subtype heterogeneity. All generated tumors histologically reflected human sarcomas and had increased aneuploidy as compared to simple karyotype sarcomas. Comparing differential gene expression of TCGA samples to model data identified increased oxidative phosphorylation signaling in YAP1 tumors. Treatment of a panel of soft tissue sarcomas with a combination of YAP1 and oxidative phosphorylation inhibitors led to significantly decreased viability.


CONCLUSIONS
Transcriptional co-analysis of TCGA patient samples to YAP1 and KRAS model tumors support that these sarcoma subtypes lie along a spectrum of disease and adds guidance for further transcriptome-based refinement of sarcoma subtyping. This approach can be used to begin to understand pathways and mechanisms driving human sarcoma development, the relationship between sarcoma subtypes and to identify and validate new therapeutic vulnerabilities for this aggressive and heterogeneous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdaada75e0a2b742920f7cff889cd677616ee1d5" target='_blank'>
              Genetic Screen in a Pre-Clinical Model of Sarcoma Development Defines Drivers and Therapeutic Vulnerabilities.
              </a>
            </td>
          <td>
            Jack Freeland, Maria Muñoz, Edmond O'Donnell, J. Langerman, Morgan Darrow, Jessica Bergonio, J. Suarez-Navarro, Steven Thorpe, Robert Canter, R. L. Randall, Kathrin Plath, Kermit L Carraway, Owen N. Witte, Thomas G. Graeber, Janai R. Carr-Ascher
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35393ed7857060735a43bccb7ec1795bc1e251a9" target='_blank'>
              Comprehensive molecular characterization of TFE3-rearranged renal cell carcinoma
              </a>
            </td>
          <td>
            Cho-Rong Lee, J. Suh, Dongjun Jang, Bo-Yeong Jin, Jaeso Cho, Moses Lee, Hyungtai Sim, Minyong Kang, Jueun Lee, Ju Hyun Park, Kyoung-Hwa Lee, Geum-Sook Hwang, Kyung Chul Moon, Cheryn Song, J. Ku, C. Kwak, Hyeon-Hoe Kim, Sung-Yup Cho, Murim Choi, Chang Wook Jeong
          </td>
          <td>2024-08-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1334e8d1bab52080f2aa5225537b4b61dc872565" target='_blank'>
              Single-cell RNA-seq reveals the transcriptional program underlying tumor progression and metastasis in neuroblastoma.
              </a>
            </td>
          <td>
            Zhe Nian, Dan Wang, Hao Wang, Wenxu Liu, Zhenyi Ma, Jie Yan, Yanna Cao, Jie Li, Qiang Zhao, Zhe Liu
          </td>
          <td>2024-07-17</td>
          <td>Frontiers of medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Gene expression and complex phenotypes are determined by the activity of cis-regulatory elements. However, an understanding of how extant genetic variants affect cis-regulatory activity remains limited. Here, we investigated the consequences of cis-regulatory diversity using single-cell genomics of >0.7 million nuclei across 172 maize inbreds. Our analyses pinpointed cis-regulatory elements distinct to domesticated maize and how transposons rewired the regulatory landscape. We found widespread chromatin accessibility variation associated with >4.6 million genetic variants with largely cell-type-specific effects. Variants in TEOSINTE BRANCHED1/CYCLOIDEA/PROLIFERATING CELL FACTOR binding sites were the most prevalent determinants of chromatin accessibility. Finally, integration of genetic variants associated with chromatin accessibility, organismal trait variation, and population differentiation revealed how local adaptation has rewired regulatory networks in unique cellular context to alter maize flowering phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f126351cde084205c5fe8f0c5b1a2468aa66fe35" target='_blank'>
              The genetic architecture of cell-type-specific cis-regulation
              </a>
            </td>
          <td>
            Alexandre P. Marand, Luguang Jiang, Fabio Gomez-Cano, Mark A. A. Minow, Xuan Zhang, J. P. Mendieta, Ziliang Luo, Sohyun Bang, Haidong Yan, Cullan Meyer, Luca Schlegel, F. Johannes, Robert J. Schmitz
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Genetic editing of the germline using CRISPR/Cas9 technology has made it possible to alter livestock traits, including the creation of resistance to viral diseases. However, virus adaptability could present a major obstacle in this effort. Recently, chickens resistant to avian leukosis virus subgroup J (ALV-J) were developed by deleting a single amino acid, W38, within the ALV-J receptor NHE1 using CRISPR/Cas9 genome editing. This resistance was confirmed both in vitro and in vivo. In vitro resistance of W38-/- chicken embryonic fibroblasts to all tested ALV-J strains was shown. To investigate the capacity of ALV-J for further adaptation, we used a retrovirus reporter-based assay to select adapted ALV-J variants. We assumed that adaptive mutations overcoming the cellular resistance would occur within the envelope protein. In accordance with this assumption, we isolated and sequenced numerous adapted virus variants and found within their envelope genes eight independent single nucleotide substitutions. To confirm the adaptive capacity of these substitutions, we introduced them into the original retrovirus reporter. All eight variants replicated effectively in W38-/- chicken embryonic fibroblasts in vitro while in vivo, W38-/- chickens were sensitive to tumor induction by two of the variants. Importantly, receptor alleles with more extensive modifications have remained resistant to the virus. These results demonstrate an important strategy in livestock genome engineering towards antivirus resistance and illustrate that cellular resistance induced by minor receptor modifications can be overcome by adapted virus variants. We conclude that more complex editing will be necessary to attain robust resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8a5c8f973522317dce2248fd8edd93f9386c51" target='_blank'>
              Rapid adaptive evolution of avian leukosis virus subgroup J in response to biotechnologically induced host resistance
              </a>
            </td>
          <td>
            M. Matoušková, Jiří Plachý, Dana Kučerová, Ľ. Pecnová, M. Reinišová, J. Geryk, Vít Karafiát, T. Hron, J. Hejnar
          </td>
          <td>2024-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The DNA damage repair (DDR) pathway is a complex signaling cascade that can sense DNA damage and trigger cellular responses to DNA damage to maintain genome stability and integrity. A typical hallmark of cancer is genomic instability or nonintegrity, which is closely related to the accumulation of DNA damage within cancer cells. The treatment principles of radiotherapy and chemotherapy for cancer are based on their cytotoxic effects on DNA damage, which are accompanied by severe and unnecessary side effects on normal tissues, including dysregulation of the DDR and induced therapeutic tolerance. As a driving factor for oncogenes or tumor suppressor genes, noncoding RNA (ncRNA) have been shown to play an important role in cancer cell resistance to radiotherapy and chemotherapy. Recently, it has been found that ncRNA can regulate tumor treatment tolerance by altering the DDR induced by radiotherapy or chemotherapy in cancer cells, indicating that ncRNA are potential regulatory factors targeting the DDR to reverse tumor treatment tolerance. This review provides an overview of the basic information and functions of the DDR and ncRNAs in the tolerance or sensitivity of tumors to chemotherapy and radiation therapy. We focused on the impact of ncRNA (mainly microRNA [miRNA], long noncoding RNA [lncRNA], and circular RNA [circRNA]) on cancer treatment by regulating the DDR and the underlying molecular mechanisms of their effects. These findings provide a theoretical basis and new insights for tumor-targeted therapy and the development of novel drugs targeting the DDR or ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63af0c8bea30d510199d8c53e8bb1fd81c1002b8" target='_blank'>
              DNA damage response-related ncRNAs as regulators of therapy resistance in cancer
              </a>
            </td>
          <td>
            Ziru Gao, Xinchi Luan, Xuezhe Wang, Tianyue Han, Xiaoyuan Li, Zeyang Li, Peifeng Li, Zhixia Zhou
          </td>
          <td>2024-08-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive and lethal form of brain cancer with few effective treatments. In this context, Zika virus has emerged as a promising therapeutic agent due to its ability to selectively infect and kill GBM cells. To elucidate these mechanisms and expand the landscape of oncolytic virotherapy, we pursued a transcriptomic meta-analysis comparing the molecular signatures of Zika infection in GBM and neuroblastoma (NBM). Over-representation analysis of dysregulated coding genes showed significant enrichment of tumor necrosis factor (TNF), NF-κB, and p53 signaling pathways. A refined list of long non-coding RNAs consistently dysregulated in Zika-infected GBMs was also developed. Functional review of these candidates revealed their potential regulatory role in Zika-mediated oncolysis. We performed validation of the less-researched targets in adult and pediatric GBM cell lines and found significant differential regulation, as predicted. Altogether, our results provide novel insights into the molecular mechanisms underlying the effect of Zika on GBM. We highlight potential therapeutic targets that could be further interrogated to improve the efficacy of tumor cell death and the utility of Zika as an adjuvant virotherapy for GBM and other related cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5fb2f82f19cb82c54f2e078dc5fd330417c5a7" target='_blank'>
              Transcriptomic Meta-analysis Identifies Long Non-Coding RNAs Mediating Zika’s Oncolytic Impact in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Youssef A. Kousa, Shriya Singh, Allison Horvath, Federica Tomasso, Javad Nazarian, Lisa Henderson, Tamer A. Mansour
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer drug resistance constitutes a severe limitation for the satisfactory outcome of these patients. This is a complex problem due to the co-existence in cancer cells of multiple and synergistic mechanisms of chemoresistance (MOC). These mechanisms are accounted for by the expression of a set of genes included in the so-called resistome, whose effectiveness often leads to a lack of response to pharmacological treatment. Additionally, genetic variants affecting these genes further increase the complexity of the question. This review focuses on a set of genes encoding members of the transportome involved in drug uptake, which have been classified into the MOC-1A subgroup of the resistome. These proteins belong to the solute carrier (SLC) superfamily. More precisely, we have considered here several members of families SLC2, SLC7, SLC19, SLC22, SLCO, SLC28, SLC29, SLC31, SLC46, and SLC47 due to the impact of their expression and genetic variants in anticancer drug uptake by tumor cells or, in some cases, general bioavailability. Changes in their expression levels and the appearance of genetic variants can contribute to the Darwinian selection of more resistant clones and, hence, to the development of a more malignant phenotype. Accordingly, to address this issue in future personalized medicine, it is necessary to characterize both changes in resistome genes that can affect their function. It is also essential to consider the time-dependent dimension of these features, as the genetic expression and the appearance of genetic variants can change during tumor progression and in response to treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96b3489c776763d0625940057b010e5e5524219f" target='_blank'>
              Impact of genetic variants in the solute carrier (SLC) genes encoding drug uptake transporters on the response to anticancer chemotherapy
              </a>
            </td>
          <td>
            Jose J. G. Marin, Maria A. Serrano, E. Herráez, E. Lozano, Sara Ortiz-Rivero, Laura Perez-Silva, María Reviejo, O. Briz
          </td>
          <td>2024-07-18</td>
          <td>Cancer Drug Resistance</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/354d0cada1a73a94d156b70d215246eda0c2a524" target='_blank'>
              T cell dysfunction and therapeutic intervention in cancer.
              </a>
            </td>
          <td>
            Caitlin C. Zebley, D. Zehn, Stephen Gottshalk, Hongbo Chi
          </td>
          <td>2024-07-18</td>
          <td>Nature immunology</td>
          <td>1</td>
          <td>42</td>
        </tr>

        <tr id="Cells must adapt to environmental changes to maintain homeostasis. One of the most striking environmental adaptations is entry into hibernation during which core body temperature can decrease from 37°C to as low at 4°C. How mammalian cells, which evolved to optimally function within a narrow range of temperatures, adapt to this profound decrease in temperature remains poorly understood. In this study, we conducted the first genome-scale CRISPR-Cas9 screen in cells derived from Syrian hamster, a facultative hibernator, as well as human cells to investigate the genetic basis of cold tolerance in a hibernator and a non-hibernator in an unbiased manner. Both screens independently revealed glutathione peroxidase 4 (GPX4), a selenium-containing enzyme, and associated proteins as critical for cold tolerance. We utilized genetic and pharmacological approaches to demonstrate that GPX4 is active in the cold and its catalytic activity is required for cold tolerance. Furthermore, we show that the role of GPX4 as a suppressor of cold-induced cell death extends across hibernating species, including 13-lined ground squirrels and greater horseshoe bats, highlighting the evolutionary conservation of this mechanism of cold tolerance. This study identifies GPX4 as a central modulator of mammalian cold tolerance and advances our understanding of the evolved mechanisms by which cells mitigate cold-associated damage—one of the most common challenges faced by cells and organisms in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181b39f570d649216102dc87d265ba81bbf8c6ba" target='_blank'>
              Multi-species genome-wide CRISPR screens identify GPX4 as a conserved suppressor of cold-induced cell death
              </a>
            </td>
          <td>
            Breanna Lam, Kathrin M. Kajderowicz, Heather R. Keys, Julian M. Roessler, Evgeni M. Frenkel, Adina Kirkland, Punam Bisht, Mohamed A. El-Brolosy, Rudolf Jaenisch, George W. Bell, Jonathan S Weissman, Eric C. Griffith, Siniša Hrvatin
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Objectives: Neuroblastoma (NB), a pediatric malignancy of the peripheral nervous system, is characterized by epigenetic and transcriptional (EP-TF) anomalies. This study aimed to develop an EP-TF clinical prognostic model for NB using CRISPR-Cas9 knockout screening. Results: An integrative analysis was conducted using CRISPR-Cas9 screening in vitro and in vivo with public NB datasets to identify 35 EP-TF genes that exhibited the highest expression in NB and were highly dependent on cancer viability. After univariate analysis, 27 of these 35 genes were included in the least absolute shrinkage and selection operator screen. We established and biologically validated a prognostic EP-TF model encompassing RUVBL1, LARP7, GTF3C4, THAP10, SUPT16H, TIGD1, SUV39H2, TAF1A, SMAD9, and FEM1B across diverse NB cohorts. MYCN serves a potential upstream regulator of EP-TF genes. The high-risk subtype exhibited traits associated with the malignant cell cycle, MYCN-linked signaling and chromatin remodeling, all of which are correlated with poor prognosis and immunosuppression. MEK inhibitors have emerged as promising therapeutic agents for targeting most EP-TF risk genes in NB. Conclusion: Our novel prognostic model shows significant potential for predicting and evaluating the overall survival of NB patients, offering insights into therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec96366563af269a504d1c7dc22ef594a4897d48" target='_blank'>
              CRISPR-Cas9 screening develops an epigenetic and transcriptional gene signature for risk stratification and target prediction in neuroblastoma
              </a>
            </td>
          <td>
            Liaoran Zhang, Jialin Mo, Hao Shi, Jing Xiong, Yeerfan Aierken, Feng Chen, Yujie Tang, Kewen Zhao, Zhibao Lv, Kezhe Tan
          </td>
          <td>2024-08-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Advancements in our comprehension of tumor biology and chemoresistance have spurred the development of treatments that precisely target specific molecules within the body. Despite the expanding landscape of therapeutic options, there persists a demand for innovative approaches to address unmet clinical needs. RNA therapeutics have emerged as a promising frontier in this realm, offering novel avenues for intervention such as RNA interference and the utilization of antisense oligonucleotides (ASOs). ASOs represent a versatile class of therapeutics capable of selectively targeting messenger RNAs (mRNAs) and silencing disease-associated proteins, thereby disrupting pathogenic processes at the molecular level. Recent advancements in chemical modification and carrier molecule design have significantly enhanced the stability, biodistribution, and intracellular uptake of ASOs, thereby bolstering their therapeutic potential. While ASO therapy holds promise across various disease domains, including oncology, coronary angioplasty, neurological disorders, viral, and parasitic diseases, our review manuscript focuses specifically on the application of ASOs in targeted cancer therapies. Through a comprehensive examination of the latest research findings and clinical developments, we delve into the intricacies of ASO-based approaches to cancer treatment, shedding light on their mechanisms of action, therapeutic efficacy, and prospects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6da866506eed600010d2a20b6f1bd9afd547dc" target='_blank'>
              Therapeutic Antisense Oligonucleotides in Oncology: From Bench to Bedside
              </a>
            </td>
          <td>
            Elif Çakan, Olivia D. Lara, A. Szymanowska, E. Bayraktar, Arturo Chavez-Reyes, G. Lopez-Berestein, P. Amero, Cristian Rodríguez-Aguayo
          </td>
          <td>2024-08-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>97</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d47d5363206f0a4fb9e459a4bf61f772ff9c57" target='_blank'>
              Overexpression of REC8 induces aberrant gamete meiotic division and contributes to AML pathogenesis - a multiplexed microarray analysis and mendelian randomization study.
              </a>
            </td>
          <td>
            Wenxi Hua, J. Qi, Meng Zhou, Shiyu Han, Xiao-Yun Xu, Jinwen Su, Tingting Pan, Depei Wu, Yue Han
          </td>
          <td>2024-07-16</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="High risk - neuroblastoma (HR-NB) is a pediatric solid tumor with high lethality. Half of HR-NB are driven by MYCN gene amplification (MNA). These HR-NBs require high dosage chemotherapy and often relapse. Moreover, current therapies can cause severe long-term side effects and new therapies are urgently needed. This study investigates a novel therapeutic approach targeting the metabolic vulnerabilities of MNA NB cells. We discovered that Diphenyleneiodonium chloride (DPI), an inhibitor of flavoprotein enzymes and mitochondrial complex I, synergizes with mitoquinone mesylate (MitoQ), a mitochondria-targeted antioxidant in 2D and 3D in vitro models of NB. Similarly to DPI, MitoQ affects MNA cells in a MYCN-dependent way, being more toxic when MYCN levels are high. Furthermore, low nanomolar concentrations of MitoQ significantly decrease MYCN protein expression and induce differentiation of MNA cells. The DPI and MitoQ combination further synergizes with vincristine, a chemotherapeutic agent used in NB treatment. Phosphoproteomics and proteomics analysis suggests that the drug combination induces MNA NB cell death by arresting the cell cycle and inhibiting oxidative phosphorylation (OXPHOS) in the mitochondria. Thus, interference with mitochondrial metabolism may represent an effective strategy to enhance the activity of chemotherapeutic drugs in MNA-NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84bd27bba09b6f802fb233fccdb2001e1425eb1" target='_blank'>
              OXPHOS TARGETING OF MYCN-AMPLIFIED NEUROBLASTOMA
              </a>
            </td>
          <td>
            Soraya Epp, Donagh Egan, E. Poon, Amirah Adlina Abdul Aziz, Kieran Wynne, Melinda Halasz, Walter Kolch
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Simple Summary With advanced bioinformatic and sequencing technologies, many non-coding RNAs have been found to harbor non-canonical open reading frames (ncORFs). Some studies suggest that many ncORF-encoded microproteins or micropeptides possess important roles in cancer biology, yet the functional roles of these ncORF-encoded peptides are mostly unknown. The purpose of this review is to provide a comprehensive overview of existing knowledge on ncORF-encoded microproteins in cancer biology and highlight their roles as potential prognostic and therapeutic targets in cancer. Abstract Cancer develops from abnormal cell growth in the body, causing significant mortalities every year. To date, potent therapeutic approaches have been developed to eradicate tumor cells, but intolerable toxicity and drug resistance can occur in treated patients, limiting the efficiency of existing treatment strategies. Therefore, searching for novel genes critical for cancer progression and therapeutic response is urgently needed for successful cancer therapy. Recent advances in bioinformatics and proteomic techniques have allowed the identification of a novel category of peptides encoded by non-canonical open reading frames (ncORFs) from historically non-coding genomic regions. Surprisingly, many ncORFs express functional microproteins that play a vital role in human cancers. In this review, we provide a comprehensive description of different ncORF types with coding capacity and technological methods in discovering ncORFs among human genomes. We also summarize the carcinogenic role of ncORFs such as pTINCR and HOXB-AS3 in regulating hallmarks of cancer, as well as the roles of ncORFs such as HOXB-AS3 and CIP2A-BP in cancer diagnosis and prognosis. We also discuss how ncORFs such as AKT-174aa and DDUP are involved in anti-cancer drug response and the underestimated potential of ncORFs as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d95f26adc077c4e04bfda2c2e30184b8f21906d" target='_blank'>
              Exploring the Dark Matter of Human Proteome: The Emerging Role of Non-Canonical Open Reading Frame (ncORF) in Cancer Diagnosis, Biology, and Therapy
              </a>
            </td>
          <td>
            Anni Ge, Curtis Chan, Xiaolong Yang
          </td>
          <td>2024-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is the most lethal type of lung cancer. Paradoxically, this tumor displays an initial exquisite response to chemotherapy; however, at relapse, the tumor is highly resistant to subsequent available therapies. Here, we report that the expression of three prime repair exonuclease 1 (TREX1) is strongly induced in chemoresistant SCLCs. ATAC-seq and ChIP-seq revealed a significant increase in chromatin accessibility and transcriptional activity of TREX1 gene locus in chemoresistant SCLC. Analyses of human SCLC tumors and patient-derived xenografts (PDXs) also showed increase in TREX1 expression in post-chemotherapy samples. TREX1 depletion caused the activation of cyclic GMP-AMP synthase (cGAS)-stimulator of interferon genes (STING) pathway due to cytoplasmic accumulation of damage-associated double-stranded DNA, inducing immunogenicity and enhancing the sensitivity of drug resistant cells to chemotherapy. These findings suggest TREX1 upregulation may partially contribute to the survival of resistant cells, and its inhibition may represent a promising therapeutic strategy to enhance antitumor immunity and potentiate the efficacy of chemotherapy and/or immunotherapy in chemoresistant SCLCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920774aa4db00913c78963d621212ab8e7a2e837" target='_blank'>
              Targeting TREX1 induces innate immune response in drug-resistant Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Takahiko Murayama, N. Mahadevan, Catherine B Meador, Elena V. Ivanova, Yuqiao Pan, Erik H. Knelson, Tetsuo Tani, Jun Nakayama, Xueying Ma, Tran C Thai, Yin P Hung, William Kim, Hideo Watanabe, Kathy Q Cai, Aaron N Hata, C. Paweletz, D. Barbie, Israel Cañadas
          </td>
          <td>2024-08-23</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) possess a significant ability to renew themselves, which gives them a strong capacity to form tumors and expand to encompass additional body areas. In addition, they possess inherent resistance to chemotherapy and radiation therapies used to treat many forms of cancer. Scientists have focused on investigating the signaling pathways that are highly linked to the ability of CSCs to renew themselves and maintain their stem cell properties. The pathways encompassed are Notch, Wnt/β-catenin, hedgehog, STAT3, NF-κB, PI-3K/Akt/mTOR, sirtuin, ALDH, MDM2, and ROS. Recent studies indicate that directing efforts towards CSC cells is essential in eradicating the overall cancer cell population and reducing the likelihood of tumor metastasis. As our comprehension of the mechanisms that stimulate CSC activity, growth, and resistance to chemotherapy advances, the discovery of therapeutic drugs specifically targeting CSCs, such as small-molecule compounds, holds the potential to revolutionize cancer therapy. This review article examines and analyzes the novel anti-CSC compounds that have demonstrated effective and selective targeting of pathways associated with the renewal and stemness of CSCs. We also discussed their special drug metabolism and absorption mechanisms. CSCs have been the subject of much study in cancer biology. As a possible treatment for malignancies, small-molecule drugs that target CSCs are gaining more and more attention. This article provides a comprehensive review of the current state of key small-molecule compounds, summarizes their recent developments, and anticipates the future discovery of even more potent and targeted compounds, opening up new avenues for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c2ec47520bdf5746027b1d46298064477df151" target='_blank'>
              Novel Anti-Cancer Stem Cell Compounds: A Comprehensive Review
              </a>
            </td>
          <td>
            Shanchun Guo, Shilong Zheng, Mingli Liu, Guangdi Wang
          </td>
          <td>2024-08-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="DNA modifications drive aging, neurodegeneration, carcinogenesis, and chemotherapy drug action. To understand the functional genomic roles of DNA modifications, it is critical to accurately map their diverse chemical forms with single-nucleotide precision in complex genomes, but it remains challenging. Click-code-seq is a click-chemistry-aided single-nucleotide-resolution strategy for guanine-oxidation mapping, used in yeast DNA but having poor applicability to human genomes. Here, we upgraded click-code-seq to enable its first application for sequencing DNA oxidation and depurination in human genomes. For this, we developed a companion fluorescence assay, click-fluoro-quant, to rapidly quantify different common DNA modifications, and devised novel adapters to minimize false modification detection and assess modification frequency in cell populations. We uncovered that endogenous DNA oxidation in a human cell line has a highly similar pattern to cancer mutational signatures associated with reactive oxygen species. We established that the DNA-alkylating chemotherapy drug irofulven preferentially induces depurination in ApA dimers and promoter regions. Intriguingly, we revealed that oxidized guanines and apurinic sites, both irofulven-induced and endogenous, are depleted in gene transcribed strands, and the strand bias widens with increasing gene expression. This work substantially advances click-code-seq for deciphering the impacts of key modifications in human DNA on cellular physiology and toxicological responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6b6bbbe3beae3b0b23fbadc72da494f15786faa" target='_blank'>
              Click-chemistry-aided quantitation and sequencing of oxidized guanines and apurinic sites uncovers their transcription-linked strand bias in human cells
              </a>
            </td>
          <td>
            V. Takhaveev, N. Püllen, Navnit K. Singh, Sabrina M. Huber, Stefan Schauer, Hailey L. Gahlon, A. Poetsch, S. Sturla
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Genomic variations contribute to the phenotypic diversity of individuals. A number of polymorphisms in protein-coding regions that alter drug efficacy or lead to adverse reactions have been characterized; however, noncoding regions that affect drug responses are largely overlooked, except for a limited number of well-studied enhancers. Results We conducted a quantitative assessment of cis-regulatory elements (CREs) based on transcription initiation profiling of mRNAs and noncoding RNAs, including enhancer RNAs, by using CAGE (Cap Analysis of Gene Expression). Candidate CREs identified in a hepatocellular carcinoma HepG2 cell line with stable expression of drug-responsive transcription factor pregnane X receptor (PXR) were further narrowed down by integrating data of PXR-binding sites in human primary hepatocytes and genome-wide association studies. We found more than 100-fold enrichments of the candidates to genetically associated loci with circulating levels of bilirubin and vitamin D, which implicated a link to adverse reactions of PXR ligands. We uncovered novel enhancers of UGT1A1 and TSKU through CRISPR/Cas9 knockout experiments. We identified alleles altering regulatory activities of UGT1A1 and CYP24A1enhancers by using luciferase reporter assay. Furthermore, our siRNA experiments revealed an unexpected impact of TSKU on the expression of vitamin D-metabolizing enzymes. Conclusions Our transcriptome-based assessment of CREs expanded the list of drug-inducible and PXR-mediated enhancers and super-enhancers. We identified regulatory alleles that alter drug-induced gene expressions, and discovered a novel molecular cascade associated with an adverse reaction. Our results contribute a precise understanding of the noncoding elements of the human genome underlying drug responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f12494b9c7c48c33bdacd730ac5e83409d9983a" target='_blank'>
              Drug-induced cis-regulatory elements in human hepatocytes affect molecular phenotypes associated with adverse reactions
              </a>
            </td>
          <td>
            Saki Gotoh-Saito, Ryoko Wada, Hideya Kawaji
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In clinics, chemotherapy is often combined with surgery and radiation to increase the chances of curing cancers. In the case of glioblastoma (GBM), patients are treated with a combination of radiotherapy and TMZ over several weeks. Despite its common use, the mechanism of action of the alkylating agent TMZ has not been well understood when it comes to its cytotoxic effects in tumor cells that are mostly non-dividing. The cellular response to alkylating DNA damage is operated by an intricate protein network involving multiple DNA repair pathways and numerous checkpoint proteins that are dependent on the type of DNA lesion, the cell type, and the cellular proliferation state. Among the various alkylating damages, researchers have placed a special on O6-methylguanine (O6-mG). Indeed, this lesion is efficiently removed via direct reversal by O6-methylguanine-DNA methyltransferase (MGMT). As the level of MGMT expression was found to be directly correlated with TMZ efficiency, O6-mG was identified as the critical lesion for TMZ mode of action. Initially, the mode of action of TMZ was proposed as follows: when left on the genome, O6-mG lesions form O6-mG: T mispairs during replication as T is preferentially mis-inserted across O6-mG. These O6-mG: T mispairs are recognized and tentatively repaired by a post-replicative mismatched DNA correction system (i.e., the MMR system). There are two models (futile cycle and direct signaling models) to account for the cytotoxic effects of the O6-mG lesions, both depending upon the functional MMR system in replicating cells. Alternatively, to explain the cytotoxic effects of alkylating agents in non-replicating cells, we have proposed a “repair accident model” whose molecular mechanism is dependent upon crosstalk between the MMR and the base excision repair (BER) systems. The accidental encounter between these two repair systems will cause the formation of cytotoxic DNA double-strand breaks (DSBs). In this review, we summarize these non-exclusive models to explain the cytotoxic effects of alkylating agents and discuss potential strategies to improve the clinical use of alkylating agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5875d4f5d1a942887726707d85e1f6dc7b35520" target='_blank'>
              Accidental Encounter of Repair Intermediates in Alkylated DNA May Lead to Double-Strand Breaks in Resting Cells
              </a>
            </td>
          <td>
            Shingo Fujii, Robert P. Fuchs
          </td>
          <td>2024-07-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b7a1b60f27d8f207e63fcabad98fcf75b1aff6" target='_blank'>
              Deciphering the impact of genomic variation on function.
              </a>
            </td>
          <td>
            J. Engreitz, Heather A. Lawson, Harinder Singh, Lea M. Starita, Gary C. Hon, Hannah Carter, Nidhi Sahni, Timothy E. Reddy, Xihong Lin, Yun Li, Nikhil V. Munshi, Maria H. Chahrour, Alan P. Boyle, B. Hitz, Ali Mortazavi, Mark Craven, K. Mohlke, Luca Pinello, Ting Wang, A. Kundaje, Feng Yue, Sarah Cody, Nina P. Farrell, Michael I. Love, Lara A Muffley, M. Pazin, Fairlie Reese, Eric Van Buren, Kushal K. Dey, Martin Kircher, Jian Ma, P. Radivojac, Brunilda Balliu, Brian A. Williams, Danwei Huangfu, Chong Y. Park, Thomas Quertermous, Jishnu Das, M. Calderwood, Douglas M. Fowler, Marc Vidal, Lucas Ferreira, Sean D. Mooney, V. Pejaver, Jingjing Zhao, Steven Gazal, Evan Koch, Steven K. Reilly, S. Sunyaev, Anne E. Carpenter, Jason D. Buenrostro, Christina S. Leslie, Rachel E. Savage, Stefanija Giric, Chongyuan Luo, Kathrin Plath, Alejandro Barrera, Max Schubach, A. Gschwind, Jill E. Moore, N. Ahituv, S. Yi, Ingileif B. Hallgrímsdóttir, K. Gaulton, S. Sakaue, Sina Booeshaghi, Eugenio Mattei, Surag Nair, L. Pachter, Austin T. Wang, J. Shendure, Vikram Agarwal, Andrew Blair, Theofilos Chalkiadakis, Florence M. Chardon, P. Dash, Chengyu Deng, Nobuhiko Hamazaki, Pia Keukeleire, Connor A Kubo, Jean-Benoît Lalanne, Thorben Maass, Beth Martin, Troy A. McDiarmid, Mai Nobuhara, Nicholas F. Page, Sam Regalado, Jasmine Sims, A. Ushiki, Sabrina M. Best, Gabe Boyle, Nathan Camp, Silvia Casadei, Estelle Y. Da, Moez Dawood, Samantha C. Dawson, Shawn Fayer, Audrey Hamm, Richard G. James, Gail P. Jarvik, Abbye E. McEwen, Nick Moore, Sriram Pendyala, Nicholas A Popp, Mason Post, Alan F. Rubin, , Jeremy Stone, Malvika Tejura, Ziyu R. Wang, Melinda K. Wheelock, Ivan Woo, Brendan D. Zapp, Dulguun Amgalan, Aradhana Aradhana, Sophia M. Arana, M. Bassik, Julia R. Bauman, Asmita Bhattacharya, Xiangmeng S. Cai, Ziwei Chen, Stephanie Conley, Salil Deshpande, Benjamin R. Doughty, Peter P. Du, James A. Galante, Casey Gifford, W.J. Greenleaf, Katherine Guo, Revant Gupta, Sarasa Isobe, Evelyn Jagoda, Nimit Jain, Hank Jones, Helen Kang, Samuel H. Kim, YeEun Kim, Sandy Klemm, Ramen Kundu, Soumya Kundu, Mauro Lago-Docampo, Yannick C. Lee-Yow, Roni Levin-Konigsberg, Daniel Y. Li, Dominik Lindenhofer, X. R. Ma, Georgi K. Marinov, Gabriella E. Martyn, Chloe V. McCreery, Eyal Metzl-Raz, João P. Monteiro, Michael T. Montgomery, Kristy S. Mualim, Chad J. Munger, Glen Munson, Tri C. Nguyen, Tri C. Nguyen, Brian T. Palmisano, Anusri Pampari, Marlene Rabinovitch, Markus Ramste, Judhajeet Ray, Kevin R. Roy, Oriane M. Rubio, Julia M. Schaepe, Gavin Schnitzler, Jacob M Schreiber, Disha Sharma, Maya U. Sheth, Huitong Shi, Vasundhara Singh, Riya Sinha, Lars M Steinmetz, Jason Tan, Anthony Tan, Josh Tycko, Raeline Valbuena, Valeh Valiollah Pour Amiri, Mariëlle J. F. M. van Kooten, Alun Vaughan-Jackson, Anthony Venida, Chad S. Weldy, Matthew D. Worssam, Fan Xia, David Yao, Tony Zeng, Quanyi Zhao, Ronghao Zhou, Zitong Sam Chen, B. Cimini, Georges Coppin, Atina G. Coté, Marzieh Haghighi, Tong Hao, David E. Hill, Jessica Lacoste, Florent Laval, Chloe Reno, Fritz Roth, Shantanu Singh, Kerstin Spirohn-Fitzgerald, Mikko Taipale, Tanisha Teelucksingh, Maxime Tixhon, Anupama Yadav, Zhipeng Yang, W. L. Kraus, Daniel A. Armendariz, Ashley E. Dederich, Ashlesha Gogate, Lauretta El Hayek, S. Goetsch, Kiran Kaur, Hyung Bum Kim, Melissa K. McCoy, Mpathi Z. Nzima, C. Pinzon-Arteaga, Bruce A. Posner, Daniel A. Schmitz, Sushama Sivakumar, Anjana Sundarrajan, Lei Wang, Yihan Wang, Jun Wu, Lin Xu, Jian Xu, Leqian Yu, Yanfeng Zhang, Huan Zhao, Qinbo Zhou, Hyejung Won, Jessica L Bell, K. A. Broadaway, Katherine N. Degner, A. Etheridge, Beverly H. Koller, Won Mah, Wancen Mu, Kimberly D. Ritola, Jonathan D. Rosen, Sarah A. Schoenrock, Rachel A. Sharp, Daniel Bauer, G. Lettre, Richard Sherwood, Basheer Becerra, Logan J. Blaine, Eric Che, M. Francoeur, Ellie N. Gibbs, Nahye Kim, Emily M. King, B. Kleinstiver, Estelle Lecluze, Zhijian Li, Zain M. Patel, Quang Vinh Phan, Jayoung Ryu, Marlena L. Starr, Ting Wu, C. Gersbach, Gregory E. Crawford, Andrew S. Allen, William H. Majoros, Nahid Iglesias, Ruhi Rai, Revathy Venukuttan, Boxun Li, Taylor Anglen, Lexi R. Bounds, Marisa C. Hamilton, Siyan Liu, Sean R. McCutcheon, Christian McRoberts Amador, Samuel J. Reisman, Maria A. ter Weele, Josephine C. Bodle, Helen L Streff, Keith Siklenka, Kari Strouse, Bradley E. Bernstein, Juliana Babu, Guillermo Barreto Corona, Kevin Dong, Fabiana M. Duarte, Neva C. Durand, Charles B. Epstein, K. Fan, Elizabeth A. Gaskell, Amelia W. Hall, Alexandra M. Ham, Mei K. Knudson, N. Shoresh, Siddarth Wekhande, Cassandra M. White, Wang Xi, Ansuman T. Satpathy, M. Corces, Serena H. Chang, Iris M. Chin, James M Gardner, Zachary A. Gardell, Jacob C. Gutierrez, , Lucas Kampman, Maya Kasowski, Caleb A. Lareau, Vincent Liu, Leif S. Ludwig, Christopher S. McGinnis, Shreya Menon, Anita Qualls, Katalin D. Sandor, Adam W. Turner, , Yajie Yin, Wenxi Zhang, Barbara J. Wold, Maria Carilli, Dayeon Cheong, Ghassan Filibam, Kim Green, S. Kawauchi, Charlene Kim, Heidi Liang, Rebekah Loving, L. Luebbert, Grant MacGregor, Angel G. Merchan, Elisabeth Rebboah, Narges Rezaie, Jasmine Sakr, Delaney K. Sullivan, Nikki Swarna, Diane Trout, S. Upchurch, Ryan Weber, Christopher P. Castro, Elysia Chou, Fan Feng, Andre Guerra, Yuanhao Huang, Linghua Jiang, Jie Liu, Ryan E. Mills, Weizhou Qian, Tingting Qin, Maureen A. Sartor, Rintsen N. Sherpa, Jinhao Wang, Yiqun Wang, Joshua D. Welch, Zhenhao Zhang, Nanxiang Zhao, Sayan Mukherjee, C. D. Page, Shannon Clarke, Richard W. Doty, Yuncheng Duan, Raluca Gordân, Kuei-Yueh Ko, Shengyu Li, Boyao Li, Alexander Thomson, Soumya Raychaudhuri, Alkes Price, Thahmina A. Ali, Arun Durvasula, M. Kellis, L. Iakoucheva, Tulika Kakati, Yile Chen, Mariam Benazouz, Shantanu Jain, Daniel Zeiberg, M. C. De Paolis Kaluza, Michelle Velyunskiy, Audrey Gasch, Kunling Huang, Yiyang Jin, Qiongshi Lu, Jiacheng Miao, Michael Ohtake, Eduardo Scopel, Robert D. Steiner, Yuriy Sverchkov, Zhiping Weng, Manuel Garber, Yu Fu, Natalie Haas, Xihao Li, Nishigandha Phalke, Shuo C. Shan, Nicole Shedd, Tianxiong Yu, Yi Zhang, Hufeng Zhou, Alexis Battle, Livnat Jerby, Eran Kotler, Andrew R. Marderstein, Stephen B. Montgomery, AkshatKumar Nigam, Evin M. Padhi, Aman Patel, Jonathan Pritchard, Ivy Raine, Vivekanandan Ramalingam, Kameron B. Rodrigues, Jacob M Schreiber, Arpita Singhal, Marisa Abundis, Deepa Bisht, Trirupa Chakraborty, Jingyu Fan, David R. Hall, Zarifeh H. Rarani, Abhinav K. Jain, Babita Kaundal, Swapnil Keshari, Daniel McGrail, Nicholas A. Pease, Vivian F. Yi, Hao Wu, Sreeram Kannan, Hongjun Song, Jingli Cai, Ziyue Gao, Ronni Kurzion, Julia I. Leu, Fan Li, Dongming Liang, Guo-Li Ming, K. Musunuru, Qi Qiu, Junwei Shi, Yijing Su, Sarah Tishkoff, Ning Xie, Qian Yang, Wenli Yang, Hongjie Zhang, Zhijian Zhang, Michael A Beer, A. Hadjantonakis, Sharon Adeniyi, Hyein Cho, Ronald Cutler, Rachel A. Glenn, David Godovich, Nan Hu, Svetlana Jovanic, Renhe Luo, Jin Woo Oh, Milad Razavi-Mohseni, Dustin Shigaki, Simone Sidoli, Thomas Vierbuchen, Xianming Wang, Breanna Williams, Jielin Yan, Dapeng Yang, Yunxiao Yang, Maike Sander, Bing Ren, W. Bartosik, Hannah S Indralingam, A. Klie, Hannah M Mummey, Mei-Lin Okino, Gaowei Wang, Nathan R. Zemke, Kai Zhang, Han Zhu, Noah Zaitlen, Jason Ernst, J. Langerman, Terence Li, Yu Sun, Alexander Y. Rudensky, P. Periyakoil, Vianne R. Gao, Melanie H. Smith, Norman M. Thomas, L. Donlin, Amit Lakhanpal, Kaden M. Southard, Rico C. Ardy, J. M. Cherry, Mark B. Gerstein, Kalina Andreeva, Pedro Assis, Beatrice Borsari, Eric Douglass, Shengcheng Dong, I. Gabdank, Keenan Graham, Otto Jolanki, Jennifer Jou, Meenakshi S. Kagda
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c785685c384edfe64958536f2b96ef40bece9a9d" target='_blank'>
              Let’s make it personal: CRISPR tools in manipulating cell death pathways for cancer treatment
              </a>
            </td>
          <td>
            Mobina Bayat, J. S. Nahand
          </td>
          <td>2024-07-29</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c02145e3ec2ea1a6d60bd9b435d738b941f9a4fa" target='_blank'>
              CRISPR-based genetic screens advance cancer immunology.
              </a>
            </td>
          <td>
            Yuanfang Cao, Xueting Li, Yumu Pan, Huahe Wang, Siyu Yang, Lingjuan Hong, Lupeng Ye
          </td>
          <td>2024-07-23</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The human protein Abelson kinase (Abl), a tyrosine kinase, plays a pivotal role in developing chronic myeloid leukemia (CML). Abl's involvement in various signaling pathways underscores its significance in regulating fundamental biological processes, including DNA damage responses, actin polymerization, and chromatin structural changes. The discovery of the Bcr-Abl oncoprotein, resulting from a chromosomal translocation in CML patients, revolutionized the understanding and treatment of the disease. The introduction of targeted therapies, starting with interferon-alpha and culminating in the development of tyrosine kinase inhibitors (TKIs) like imatinib, significantly improved patient outcomes. However, challenges such as drug resistance and side effects persist, indicating the necessity of research into novel therapeutic strategies. This review describes advancements in Abl kinase inhibitor development, emphasizing rational compound design from structural and regulatory information. Strategies, including bivalent inhibitors, PROTACs, and compounds targeting regulatory domains, promise to overcome resistance and minimize side effects. Additionally, leveraging the intricate structure and interactions of Bcr-Abl may provide insights into developing inhibitors for other kinases. Overall, this review highlights the importance of continued research into Abl kinase inhibition and its broader implications for therapeutic interventions targeting kinase-driven diseases. It provides valuable insights and strategies that may guide the development of next-generation therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f11fab61ce74a2d73dd9a6e9080be3c85e97ad" target='_blank'>
              Structure-guided drug design targeting Abl kinase: how structure and regulation can assist in designing new drugs.
              </a>
            </td>
          <td>
            D. M. Martins, P. O. Fernandes, Lucas A Vieira, Vinícius G Maltarollo, Adolfo Henrique Moraes
          </td>
          <td>2024-07-15</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Strict control over hematopoietic stem cell decision making is essential for healthy life‐long blood production and underpins the origins of hematopoietic diseases. Acute myeloid leukemia (AML) in particular is a devastating hematopoietic malignancy that arises from the clonal evolution of disease‐initiating primitive cells which acquire compounding genetic changes over time and culminate in the generation of leukemic stem cells (LSCs). Understanding the molecular underpinnings of these driver cells throughout their development will be instrumental in the interception of leukemia, the enabling of effective treatment of pre‐leukemic conditions, as well as the development of strategies to target frank AML disease. To this point, a number of precancerous myeloid disorders and age‐related alterations are proving as instructive models to gain insights into the initiation of LSCs. Here, we explore this myeloid dysregulation at the level of post–transcriptional control, where RNA‐binding proteins (RBPs) function as core effectors. Through regulating the interplay of a myriad of RNA metabolic processes, RBPs orchestrate transcript fates to govern gene expression in health and disease. We describe the expanding appreciation of the role of RBPs and their post–transcriptional networks in sustaining healthy hematopoiesis and their dysregulation in the pathogenesis of clonal myeloid disorders and AML, with a particular emphasis on findings described in human stem cells. Lastly, we discuss key breakthroughs that highlight RBPs and post–transcriptional control as actionable targets for precision therapy of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6dddad312b3774cf0fac0198aa7e076e582021" target='_blank'>
              RNA binding protein‐directed control of leukemic stem cell evolution and function
              </a>
            </td>
          <td>
            Pratik Joshi, Ava Keyvani Chahi, Lina Liu, Steven Moreira, Ana Vujovic, Kristin J Hope
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Due to the heterogeneity of cancer, precision medicine has been a major challenge for cancer treatment. Determining medication regimens based on patient genotypes has become a research hotspot in cancer genomics. In this study, we aim to identify key biomarkers for targeted therapies based on single nucleotide variants (SNVs) and copy number variants (CNVs) of genes. The experiment is carried out on 7 cancers on the Encyclopedia of Cancer Cell Lines (CCLE) dataset. Considering the high mutability of driver genes which result in abundant mutated samples, the effect of data sparsity can be eliminated to a large extent. Therefore, we focus on discovering the relationship between driver mutation patterns and three measures of drug response, namely area under the curve (AUC), half maximal effective concentration (EC50), and log2-fold change (LFC). First, multiple statistical methods are applied to assess the significance of difference in drug response between sample groups. Next, for each driver gene, we analyze the extent to which its mutations can affect drug response. Based on the results of multiple hypothesis tests and correlation analyses, our main findings include the validation of several known drug response biomarkers such as BRAF, NRAS, MAP2K1, MAP2K2, and CDKN2A, as well as genes with huge potential to infer drug responses. It is worth emphasizing that we identify a list of genes including SALL4, B2M, BAP1, CCDC6, ERBB4, FOXA1, GRIN2A, and PTPRT, whose impact on drug response spans multiple cancers and should be prioritized as key biomarkers for targeted therapies. Furthermore, based on the statistical p-values and correlation coefficients, we construct gene-drug sensitivity maps for cancer drug recommendation. In this work, we show that driver mutation patterns could be used to tailor therapeutics for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa993499162c59f7b76c5c6ffab6d2e07409c25b" target='_blank'>
              Single nucleotide and copy number variants of cancer driver genes inform drug response in multiple cancers
              </a>
            </td>
          <td>
            Zeyuan Wang, Hong Gu, Pan Qin, Jia Wang
          </td>
          <td>2024-07-31</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The use of antitumor drugs represents a reliable strategy for cancer therapy. Unfortunately, drug resistance has become increasingly common and contributes to tumor metastasis and local recurrence. The tumor immune microenvironment (TME) consists of immune cells, cytokines and immunomodulators, and collectively they influence the response to treatment. Epigenetic changes including DNA methylation and histone modification, as well as increased drug exportation have been reported to contribute to the development of drug resistance in cancers. In the past few years, the majority of studies on tumors have only focused on the development and progression of a tumor from a mechanistic standpoint; few studies have examined whether the changes in the TME can also affect tumor growth and drug resistance. Recently, emerging evidence have raised more concerns regarding the role of TME in the development of drug resistance. In the present review, it was discussed how the suppressive TME adapts to drug resistance characterized by the cooperation of immune cells, cytokines, immunomodulators, stromal cells and extracellular matrix. Furthermore, it was reviewed how these immunological or metabolic changes alter immuno‑surveillance and thus facilitate tumor drug resistance. In addition, potential targets present in the TME for developing novel therapeutic strategies to improve individualized therapy for cancer treatment were revealed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbdfa9d18c0bfe84e176528ce335232eb883a650" target='_blank'>
              Drug resistance and tumor immune microenvironment: An overview of current understandings (Review).
              </a>
            </td>
          <td>
            Yan Liu, Jun Liang, Yanping Zhang, Q. Guo
          </td>
          <td>2024-08-30</td>
          <td>International journal of oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background and aims: Colorectal carcinomas (CRCs) in patients with Lynch syndrome (LS) exhibit heightened immunogenicity due to mismatch repair deficiency (MMR-d), often resulting in favorable responses to T cell immune checkpoint therapies. Recent studies indicate that the phenotype and genotype of LS-associated CRCs vary depending on the specific MMR gene mutated. Here, we investigated whether the immune profiles of LS-associated CRCs differ based on the MMR gene defects. Methods: Tissue material from 18 MLH1-, 16 MSH2-, 40 MSH6-, and 23 PMS2-mutated CRCs and 35 sporadic MMR-d CRCs were included in the study. Imaging mass cytometry (IMC) analysis, along with targeted multiplex immunofluorescence imaging (mIF) and immunohistochemistry, were applied to examine the tumor immune microenvironment, including Human Leukocyte Antigen (HLA) class I and programmed death-ligand 1 (PD-L1) expression. Results: Unsupervised hierarchical clustering of cell phenotypes identified by IMC, followed by mIF validation, revealed comparable lymphoid and myeloid cell infiltration levels across CRCs from all MMR groups. Infiltrating T cell levels negatively correlated with the number of mutations at coding microsatellite sequences, particularly in MLH1-mutated CRCs. HLA class I defects were observed in 76% of all CRCs. These defects were more frequently accompanied by {beta}2M defects in hereditary MMR-d CRCs (67%) compared to sporadic MMR-d CRCs (37%), and did not associate with the number of {gamma}{delta} T cells, which were present in CRCs from all MMR groups. PD-L1 expression in tumor cells was only detected in 8% of all CRCs. Conclusion: Our findings illustrate that, from an immunological perspective, there is no evidence of differing immunogenic features across MMR defects. This is important to consider when developing preventive vaccine strategies and evaluating immunotherapy for LS patients and those with MMR-d CRCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0cdd08de3e8c7b38b9b131001cb730c9f4dade" target='_blank'>
              Immunological profiles in Lynch syndrome colorectal cancers are not specific to mismatch repair gene defects
              </a>
            </td>
          <td>
            Noah C. Helderman, M. Ijsselsteijn, Madalina Cabuta, Manon van der Ploeg, T. Wezel, A. Ahadova, Matthias Kloor, H. Morreau, Maartje Nielsen, N. F. D. Miranda
          </td>
          <td>2024-08-28</td>
          <td>None</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Biallelic pathogenic variants in the essential DNA repair gene BRCA2 causes Fanconi anemia, complementation group FA-D1. Patients in this group are highly prone to develop embryonal tumors, most commonly medulloblastoma arising from the cerebellar granule cell progenitors (GCPs). GCPs undergo high proliferation in the postnatal cerebellum under SHH activation, but the type of DNA lesions that require the function of the BRCA2 to prevent tumorigenesis remains unknown. To identify such lesions, we assessed both GCP neurodevelopment and tumor formation using a mouse model with deletion of exons three and four of Brca2 in the central nervous system, coupled with global Trp53 loss. Brca2Δex3-4;Trp53-/- animals developed SHH subgroup medulloblastomas with complete penetrance. Whole-genome sequencing of the tumors identified structural variants with breakpoints enriched in areas overlapping G-quadruplexes (G4s). Brca2-deficient GCPs exhibited decreased replication speed in the presence of the G4-stabilizer pyridostatin. Pif1 helicase, which resolves G4s during replication, was highly upregulated in tumors, and Pif1 knockout in primary MB tumor cells resulted in increased genome instability upon pyridostatin treatment. These data suggest that G4s may represent sites prone to replication stalling in highly proliferative GCPs and without BRCA2, G4s become a source of genome instability. Tumor cells upregulate G4-resolving helicases to facilitate rapid proliferation through G4s highlighting PIF1 helicase as a potential therapeutic target for treatment of BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d543ffa68779aadc1c04b11411614442736e5f08" target='_blank'>
              G-quadruplexes are a source of vulnerability in BRCA2 deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, Carolina Sandoval-Garcia, Margaret L. MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Clustered regularly interspaced short palindromic repeats (CRISPR) has revolutionized the field of genome editing. To circumvent the permanent modifications made by traditional CRISPR techniques and facilitate the study of both essential and non-essential genes, CRISPR interference (CRISPRi) was developed. This gene silencing technique employs a deactivated Cas effector protein and a guide RNA to block transcription initiation or elongation. Continuous improvements and a better understanding of the mechanism of CRISPRi have expanded its scope, facilitating genome-wide high-throughput screens to investigate the genetic basis of phenotypes. Additionally, emerging CRISPR-based alternatives have further expanded the possibilities for genetic screening. This review delves into the mechanism of CRISPRi, compares it with other high-throughput gene-perturbation techniques and highlights its superior capacities for studying complex microbial traits. We also explore the evolution of CRISPRi, emphasizing enhancements that have increased its capabilities, including multiplexing, inducibility, titratability, predictable knockdown efficacy, and adaptability to non-model microorganisms. Beyond CRISPRi, we discuss CRISPR activation, RNA-targeting CRISPR systems and single-nucleotide resolution perturbation techniques for their potential in genome-wide high-throughput screens in microorganisms. Collectively, this review gives a comprehensive overview of the general workflow of a genome-wide CRISPRi screen, with an extensive discussion of strengths and weaknesses, future directions and potential alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/436bc367c0ac1bbbb6f21c89e0aa519ec2663998" target='_blank'>
              The rise and future of CRISPR-based approaches for high-throughput genomics.
              </a>
            </td>
          <td>
            Silke Vercauteren, Simon Fiesack, Laetitia Maroc, N. Verstraeten, Liselot Dewachter, Jan Michiels, Sibylle C. Vonesch
          </td>
          <td>2024-07-31</td>
          <td>FEMS microbiology reviews</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1fa9923fb71606ea55e83b6b8cbb7a9e56b764" target='_blank'>
              Competition shapes the landscape of X-chromosome-linked genetic diversity
              </a>
            </td>
          <td>
            Teresa Buenaventura, Hakan Bagci, Ilinca Patrascan, Joshua J Graham, Kelsey D Hipwell, Roel Oldenkamp, James W D King, Jesus Urtasun, G. Young, Daniel Mouzo, D. Gómez-Cabrero, B. Rowland, Daniel Panne, Amanda G Fisher, Matthias Merkenschlager
          </td>
          <td>2024-07-26</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Alveolar rhabdomyosarcoma (ARMS) patients harboring PAX3-FOXO1 and PAX7-FOXO1 fusion proteins exhibit a greater incidence of tumor relapse, metastasis, and poor survival outcome, thereby underscoring the urgent need to develop effective therapies to treat this subtype of childhood cancer. To uncover mechanisms that contribute to tumor initiation, we developed a novel muscle progenitor model and used epigenomic approaches to unravel genome re-wiring events mediated by PAX3/7 fusion proteins. Importantly, these regulatory mechanisms are conserved across established ARMS cell lines, primary tumors, and orthotopic-patient derived xenografts. Among the key targets of PAX3- and PAX7-fusion proteins, we identified a cohort of oncogenes, FGF receptors, and genes essential for mitochondrial metabolism and protein translation, which we successfully targeted in preclinical trials. Our data suggest an explanation for the relative paucity of recurring mutations in this tumor, provide a compelling list of actionable targets, and suggest promising new strategies to treat this tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f840f071bc2696a99f7c83f33f72c0a963e3308" target='_blank'>
              PAX fusion proteins deregulate gene networks controlling mitochondrial translation in pediatric rhabdomyosarcoma
              </a>
            </td>
          <td>
            Bhargab Kalita, Gerard Martinez-Cebrian, Justina McEvoy, Melody Allensworth, Michelle Knight, Alessandro Magli, Rita C. R. Perlingeiro, Michael A. Dyer, Elizabeth Stewart, B. Dynlacht
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Advances in melanoma research have unveiled critical insights into its genetic and molecular landscape, leading to significant therapeutic innovations. This review explores the intricate interplay between genetic alterations, such as mutations in BRAF, NRAS, and KIT, and melanoma pathogenesis. The MAPK and PI3K/Akt/mTOR signaling pathways are highlighted for their roles in tumor growth and resistance mechanisms. Additionally, this review delves into the impact of epigenetic modifications, including DNA methylation and histone changes, on melanoma progression. The tumor microenvironment, characterized by immune cells, stromal cells, and soluble factors, plays a pivotal role in modulating tumor behavior and treatment responses. Emerging technologies like single-cell sequencing, CRISPR-Cas9, and AI-driven diagnostics are transforming melanoma research, offering precise and personalized approaches to treatment. Immunotherapy, particularly immune checkpoint inhibitors and personalized mRNA vaccines, has revolutionized melanoma therapy by enhancing the body’s immune response. Despite these advances, resistance mechanisms remain a challenge, underscoring the need for combined therapies and ongoing research to achieve durable therapeutic responses. This comprehensive overview aims to highlight the current state of melanoma research and the transformative impacts of these advancements on clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32ea057809622a436f9ea97ccea29c8077c2cd13" target='_blank'>
              Advances in Melanoma: From Genetic Insights to Therapeutic Innovations
              </a>
            </td>
          <td>
            Fernando Valdez-Salazar, Luis A. Jiménez-Del Rio, J. Padilla-Gutiérrez, Y. Valle, J. Muñóz-Valle, Emmanuel Valdés-Alvarado
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="INTRODUCTION
Advances in understanding of tumor biology shed light on hallmarks of cancer development and progression that include dysregulated DNA damage repair (DDR) machinery. Leveraging underlying tumor genomic instability and tumor specific defects in DDR, Poly (ADP-ribose) polymerases (PARP) inhibitors (PARPi) induced DNA damage emerges as a novel non-chemotherapy therapeutic opportunity. PARPis are currently approved in multiple tumor types, with the largest benefit seen in tumors with homologous recombination repair (HRR) deficiency, including germline and somatic mutations in BRCA1/2 genes (BRCA) and other pathway members such as PALB2 and Rad51c.


AREAS COVERED
This review article summarizes the current approval landscape and known and proposed mechanisms of resistance to PARPi. Further, therapeutic strategies to overcome PARPi resistance are discussed, including ongoing clinical trials.


EXPERT OPINION
PARPi have proven to be a safe and effective therapy and represent a cornerstone treatment across multiple solid tumor types. Elucidating innate and acquired mechanisms of resistance, coupled with the emergence of novel therapeutic options to capitalize on the activity of PARPi and prevent or reverse the acquisition of resistance, provides an opportunity to further expand the role of PARPi in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec7393511b29f158c45a3ee7ca1c8ffe71c659b" target='_blank'>
              Strategies for the prevention or reversal of PARP inhibitor resistance.
              </a>
            </td>
          <td>
            Zahi I. Mitri, Shaun M Goodyear, Gordon B. Mills
          </td>
          <td>2024-08-15</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Pediatric cancers are frequently driven by genomic alterations that result in aberrant transcription factor activity. While direct targeting of transcription factors is difficult, one method to circumvent this problem is to target co-factors and chromatin complexes that are necessary for their oncogenic function. To evaluate protein complex dependencies enriched in MYCN-amplified neuroblastoma, a disease of dysregulated development driven by aberrant expression of an oncogenic expression program, we curated a list of protein complexes using the CORUM database and mined the Dependency Map using single sample gene set enrichment analysis. This analysis identified the KAT module of the transcriptional coactivator Spt-Ada-Gcn5-acetyltransferase (SAGA) complex as a selective dependency in MYCN-amplified neuroblastoma. Loss of SAGA complex activity in neuroblastoma through both genetic knockout and induced protein degradation of the lysine acetyltransferase (KAT) module subunit TADA2B resulted in a global reduction of histone lysine acetylation and impaired neuroblastoma cell growth. Genetic knockout of TADA2B did not impair the growth of pediatric sarcoma cell lines, suggesting that SAGA loss-of-function is not broadly deleterious. Furthermore, SAGA loss-of-function reduced MYCN binding to chromatin, resulting in suppression of the MYCN-driven gene expression program and impaired cell cycle progression. Importantly, the SAGA complex is pharmacologically targetable in vitro and in vivo with GSK-699, a PROTAC that induces proteolysis of both KAT2A and KAT2B paralogs. As observed with genetic knockout studies, loss of SAGA activity through KAT2A/B degradation reduced the MYCN-driven transcriptional signature and impaired cell cycle progression. Our findings expand our understanding of the chromatin complexes that maintain the oncogenic transcriptional state in MYCN-amplified neuroblastoma and suggest therapeutic potential for inhibitors of SAGA KAT activity in MYCN-amplified neuroblastoma.
 Citation Format: Nathaniel W. Mabe, Clare F. Malone, Alexandra B. Forman, Gabriela Alexe, Kathleen L. Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar1, Bin Liu, Sharon YR Dent, Kimberly Stegmaier. Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be30c933e33904854e0410865fe3ee874fab512e" target='_blank'>
              Abstract A062 Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program
              </a>
            </td>
          <td>
            Nathaniel W. Mabe, Clare F Malone, Alexandra B Forman, G. Alexe, Kathleen Engel, , Melinda Soeung, Silvi Salhotra, Allen Basanthakumar1, Bin Liu, S. Y. Dent, K. Stegmaier
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The phenomenon wherein innate immune cells adopt long-term inflammatory phenotypes following the first stimuli is named trained immunity and can improve host defense against infections. Transcriptional and epigenetic reprogramming are critical mechanisms of trained immunity; however, the regulatory networks are not entirely clear at present. The human endogenous retroviruses (HERVs) provide large amounts of transcriptional regulators in the regulatory pathways. In this study, we analyzed published large omics data to explore the roles of such "dark matter" of the human genome in trained and tolerant macrophages. We collected 80 RNA sequencing data and 62 sequencing data to detect histone modifications and active regulatory regions from nine published studies on trained and tolerant macrophages. By analyzing the characteristics of transcription and epigenetic modification of HERVs, as well as their association with gene expression, we found that 15.3% of HERVs were transcribed nonrandomly from noncoding regions and enriched in specific HERV families and specific chromosomes, such as chromosomes 11, 15, 17, and 19, and they were highly related with the expression of adjacent genes. We found that 295 differentially expressed HERVs are located in 50-kbp flanking regions of 142 differentially expressed genes. We found epigenetic changes of these HERVs and that overlap with predicted enhancers and identified 35 enhancer-like HERVs. The related genes were highly involved in the activation and inflammatory responses, such as the TLR pathway. Other pathways including phosphoinositide signaling and transport of folate and K+ might be also related with trained immunity, which require further study. These results demonstrated that HERVs might play important roles in trained immunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61b6533ef1e61d4a01667368e408563dc5079fa4" target='_blank'>
              Potential Contributions of Human Endogenous Retroviruses in Innate Immune Memory.
              </a>
            </td>
          <td>
            Pengcheng Du, Jiarui Li, M. Hua, Liuluan Zhu, Chen Chen, H. Zeng
          </td>
          <td>2024-09-04</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e7e1fc16e696eaffc0fa02c1362c0f5c07aa1" target='_blank'>
              The complex interplay of tumor-infiltrating cells in driving therapeutic resistance pathways
              </a>
            </td>
          <td>
            Dengxiong Li, Fanglin Shao, Qingxin Yu, Rui-cheng Wu, Zhou-ting Tuo, Jie Wang, Luxia Ye, Yiqing Guo, Koo Han Yoo, Mang Ke, U. Okoli, Chaipanichkul Premkamon, Yubo Yang, Wuran Wei, Susan Heavey, William C. Cho, Dechao Feng
          </td>
          <td>2024-08-19</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: Cellular therapies leveraging genetically engineered immune effector cells have witnessed a remarkable surge in success, particularly evident in the notable high rates of remission induction and durable remissions observed in a substantial proportion of heavily pretreated patients with refractory B-lineage malignancies. A diverse array of effector cells and therapeutic strategies are now at our disposal, representing the culmination of advancements made over the past 3 decades. The swift pace of development in modern genetic diagnostics, the emergence of spatial proteomics, and the expanding capabilities and precision of computational sciences have profoundly enriched our comprehension of tumor biology and the intricate workings of our immune system. Empowered by advancements in synthetic biology and genome editing, we can expedite the development of next-generation immune effector cells tailored for clinical applications, balancing safety with efficacy. Summary: Universal adapter chimeric antigen receptor (CAR) technologies present the most straightforward solution to tackle antigen heterogeneity and antigen evasion mechanisms employed by tumors. Moreover, due to the decoupling of antigen recognition and signaling in adapter CAR technologies, additional effector functions can safely enhance anticancer activity and most importantly, synergistic combination of patient-specific cellular products with off-the-shelf manufactured antibodies promise increased cost-efficiency. The pivotal collaboration between clinical trial units and regulatory institutions holds the key to surmounting contemporary challenges in trial design, potentially paving the way for the exploration of patient-individualized therapies. Key Messages: In this review, we elaborate on the concept of antibody-dependent cellular cytotoxicity mediated by universal adapter CARs and delineate how recent strides in CAR engineering have the potential to furnish a versatile cellular platform, ushering in an era of cancer-adapted, multitargeted immunotherapies employing universal CAR effector cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ba3c830f8a68c2398c80db069cc5f8b269b4526" target='_blank'>
              Effectors of the Future: Universal Chimeric Antigen Receptor
              </a>
            </td>
          <td>
            Lara Sophie Schlegel, Patrick Schlegel
          </td>
          <td>2024-07-15</td>
          <td>Transfusion Medicine and Hemotherapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The Ten-Eleven Translocation (TET) family genes are implicated in a wide array of biological functions across various human cancers. Nonetheless, there is a scarcity of studies that comprehensively analyze the correlation between TET family members and the molecular phenotypes and clinical characteristics of different cancers. Leveraging updated public databases and employing several bioinformatics analysis methods, we assessed the expression levels, somatic variations, methylation levels, and prognostic values of TET family genes. Additionally, we explored the association between the expression of TET family genes and pathway activity, tumor microenvironment (TME), stemness score, immune subtype, clinical staging, and drug sensitivity in pan-cancer. Molecular biology and cytology experiments were conducted to validate the potential role of TET3 in tumor progression. Each TET family gene displayed distinct expression patterns across at least ten detected tumors. The frequency of Single Nucleotide Variant (SNV) in TET genes was found to be 91.24%, primarily comprising missense mutation types, with the main types of copy number variant (CNV) being heterozygous amplifications and deletions. TET1 gene exhibited high methylation levels, whereas TET2 and TET3 genes displayed hypomethylation in most cancers, which correlated closely with patient prognosis. Pathway activity analysis revealed the involvement of TET family genes in multiple signaling pathways, including cell cycle, apoptosis, DNA damage response, hormone AR, PI3K/AKT, and RTK. Furthermore, the expression levels of TET family genes were shown to impact the clinical staging of tumor patients, modulate the sensitivity of chemotherapy drugs, and thereby influence patient prognosis by participating in the regulation of the tumor microenvironment, cellular stemness potential, and immune subtype. Notably, TET3 was identified to promote cancer progression across various tumors, and its silencing was found to inhibit tumor malignancy and enhance chemotherapy sensitivity. These findings shed light on the role of TET family genes in cancer progression and offer insights for further research on TET3 as a potential therapeutic target for pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4df11d8b0544aeb2264d4da3fba3e8f78e1111" target='_blank'>
              Pan-cancer analyses reveal the molecular and clinical characteristics of TET family members and suggests that TET3 maybe a potential therapeutic target
              </a>
            </td>
          <td>
            Chunyan Zhang, Jie Zheng, Jin Liu, Yanxia Li, G. Xing, Shupeng Zhang, Hekai Chen, Jian Wang, Z. Shao, Yongyuan Li, Zhongmin Jiang, Yingzi Pan, Xiaozhi Liu, Ping Xu, Wenhan Wu
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The endonuclease activity of Pms1 directs mismatch repair by generating a nick in the newly replicated DNA strand. Inactivating Pms2, the human homologue of yeast Pms1, increases the chances of colorectal and uterine cancers. Here we use whole genome sequencing to show that loss of this endonuclease activity, via the pms1-DE variant, results in strong mutator effects throughout the Saccharomyces cerevisiae genome. Mutation rates are strongly increased for mutations resulting from all types of single-base substitutions and for a wide variety of single- and multi-base indel mutations. Rates for these events are further increased in strains combining pms1-DE with mutator variants of each of the three major leading and lagging strand replicases. In all cases, mutation rates, spectra, biases, and context preferences are statistically indistinguishable from strains with equivalent polymerases but lacking initial mismatch recognition due to deletion of MSH2. This implies that, across the nuclear genome, strand discrimination via the Pms1 endonuclease is as important for MMR as is initial mismatch recognition by Msh2 heterodimers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c81863bf8b4bcd0533ef4d09eddc5a93c0da4b" target='_blank'>
              Instability throughout the Saccharomyces cerevisiae genome resulting from Pms1 endonuclease deficiency.
              </a>
            </td>
          <td>
            S. Lujan, Marta A. Garbacz, S. E. Liberti, Adam B Burkholder, Thomas A. Kunkel
          </td>
          <td>2024-07-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Understanding bacterial gene function remains a major biological challenge. Double-mutant genetic interaction (GI) analysis addresses this challenge by uncovering the functional partners of targeted genes, allowing us to associate genes of unknown function with novel pathways and unravel connections between well-studied pathways, but is difficult to implement at the genome-scale. Here, we develop and use double-CRISPRi to systematically quantify genetic interactions at scale in the Bacillus subtilis envelope, including essential genes. We discover > 1000 known and novel genetic interactions. Our analysis pipeline and experimental follow-ups reveal the distinct roles of paralogous genes such as the mreB and mbl actin homologs, and identify new genes involved in the well-studied process of cell division. Overall, our study provides valuable insights into gene function and demonstrates the utility of double-CRISPRi for high-throughput dissection of bacterial gene networks, providing a blueprint for future studies in diverse bacterial species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed882175f2414bae232e1b044a85ff77b9c005c" target='_blank'>
              Comprehensive double-mutant analysis of the Bacillus subtilis envelope using double-CRISPRi
              </a>
            </td>
          <td>
            B. Koo, Horia Todor, Jiawei Sun, Jordi van Gestel, John S. Hawkins, Cameron C. Hearne, Amy B. Banta, K. C. Huang, Jason M. Peters, Carol A Gross
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfda792d36cb0c85bc9dd50884926690c096c37e" target='_blank'>
              RNA polymerase stalling-derived genome instability underlies ribosomal antibiotic efficacy and resistance evolution
              </a>
            </td>
          <td>
            Yayun Zheng, Ruochen Chai, Tian Wang, Zeqi Xu, Yihui He, P. Shen, Jintao Liu
          </td>
          <td>2024-08-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d59a35bbbd452d19f5eb530222e3178b220b08" target='_blank'>
              Tumour mutational burden: clinical utility, challenges and emerging improvements.
              </a>
            </td>
          <td>
            J. Budczies, D. Kazdal, Michael Menzel, Susanne Beck, K. Kluck, Christian Altbürger, C. Schwab, M. Allgäuer, A. Ahadova, Matthias Kloor, P. Schirmacher, Solange Peters, Alwin Krämer, P. Christopoulos, A. Stenzinger
          </td>
          <td>2024-08-27</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved DNA repair system, in which aberrations can lead to the accumulation of DNA damage and genomic scars known as homologous recombination deficiency (HRD). The identification of mutations in key genes (i.e., BRCA1, and BRCA2 (BRCA)) and the quantification of large-scale structural variants (e.g., loss of heterozygosity) are indicators of the HRD phenotype. HRD is a stable biomarker and remains unchanged during recurrence, but fails to reveal the molecular profile of tumor progression. Moreover, interpretation of the current HRD score lacks comprehensiveness, especially for the HR-proficient group. Poly (ADP-ribose) polymerase (PARP) enzymes play an important role in the repair of DNA single-strand breaks, the blockage of which using PARP inhibitors (PARPi) can generate synthetic lethality in cancer cells with HRD. Although numerous studies have demonstrated that the benefit of PARPi is substantial in ovarian cancer (OC) patients, the efficacy is limited by the development of resistance, and seems to be irrespective of HR and/or BRCA mutation status. Moreover, in addition to improving progression-free survival, long-term benefit as overall survival brought by PARPi for advanced, recurrent and refractory OC patients remains unclear. Therefore, further investigations are needed to uncover the role of HR genes beyond BRCA and their interactions with other oncogenic pathways, to determine the value of HRD in the recurrent setting, and to identify alternative strategies for the precise management of advanced, refractory OC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fe4c9f0bc408e017d25339a0121e8f087ab765" target='_blank'>
              Limitations of homologous recombination status testing and poly (ADP-ribose) polymerase inhibitor treatment in the current management of ovarian cancer
              </a>
            </td>
          <td>
            Qianying Zhao, Liping Bai, Yu Tan, Mingrong Qie
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="What Is the Issue? 
The first therapeutics based on clustered regularly interspaced short palindromic repeats (CRISPR) technologies are entering the market. These gene editing technologies have the potential to change treatment paradigms and may be used to treat conditions that cannot be treated or cured with current methods. This report aims to provide an overview of the technologies and their current and potential roles in health care. 
What Are the Technologies? 
CRISPR is a part of bacterial immune systems that can cut DNA strands and is used as a gene editing tool. A guide ribonucleic acid (RNA) sequence leads the CRISPR-associated nuclease to the target DNA sequence where the cut is made. These edits change the function of the gene, making genes nonfunctional or replacing the coding sequence for 1 gene with another. CRISPR can also be used to increase or decrease the expression of specific genes. 
What Is the Potential Impact? 
CRISPR-based technologies have a variety of potential applications in health care, including: 

treating genetic diseases 
understanding the genetic mechanisms of diseases and investigating the relevance of potential drug treatments 
managing infectious diseases through detection, treatment, and elimination. 

What Else Do We Need to Know? 
Ethical issues pertinent to the use of CRISPR include the ability to obtain adequately informed consent, the potential future consequences of gene editing and its potential unintended effects, and the impact gene editing could have on future generations. The long-term effects of CRISPR-based therapies are currently unknown. Further research into emerging applications is required. Long-term follow-up of the patients who have received the first CRISPR-based therapeutics will help inform understanding of the safety and effectiveness of these treatments. While the first of these therapies have been granted regulatory authorization, the next viable CRISPR-based therapies are still in the early phases of development, with the pivotal clinical trials not expected to be completed until at least 2027.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0814cd12a7b72cf57aa46663d5db2b368ed9efd" target='_blank'>
              CRISPR Technologies for In Vivo and Ex Vivo Gene Editing
              </a>
            </td>
          <td>
            Cadth Horizon, Scan
          </td>
          <td>2024-07-19</td>
          <td>Canadian Journal of Health Technologies</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8867a66b53ced02c166da3c33e0be6474929e7d1" target='_blank'>
              Somatic CpG hypermutation is associated with mismatch repair deficiency in cancer
              </a>
            </td>
          <td>
            Aidan Flynn, Sebastian M. Waszak, Joachim Weischenfeldt
          </td>
          <td>2024-07-18</td>
          <td>Molecular Systems Biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="137 Background: Resistance to RT remains a clinical challenge, given our limited understanding of the molecular pathways underpinning radioresistance. Here, we investigated for mechanisms linked to RR using paired experimental and clinico-molecular HNC and PCa datasets for discovery and validation of novel pathways that may be enriched in RR cancers. Methods: For experimental data, we generated RR HNC (FaDu, HK1) and PCa (22Rv1, DU145) cell lines following high-dose X-irradiation (90 Gy in 45 fr). Genotypic characterization of RR and wildtype (WT) cells was performed by genomic and transcriptomic sequencing using WES (100X) and RNAseq (Illumina, CA), respectively, followed by functional characterization by western blot (WB) and immunofluorescence (IF). For clinico-molecular data, we utilized two prospectively recruited RT cohorts, consisting of 311 patients with HNC (n=158) and PCa (n=153). Tumor transcriptomes were profiled using RNAseq and Affymetrix ST array (ThermoFisher, CA), respectively. Comparative analyses of molecular profiles were first performed between RR and WT in vitro models to identify dysregulated pathways that were linked to radioresistance, and subsequently tested for association with disease-free survival (DFS) in the clinical cohorts. Results: Comparative genomic analyses between RR and WT HNC and PCa models revealed an abundance of acquired mutations in the RR compared with WT models, with a higher mutation count observed in PCa than HNC cells (SNV counts: 2,158 [DU145-RR] and 1,387 [22Rv1-RR] vs 396 [FaDu-RR] and 25 [HK1-RR], P<0.0001). Mutational signature analyses indicated a common enrichment of DNA mismatch repair-related mutational signatures (SBS3, SBS14, and SBS21) across the RR HNC and PCa models. Transcriptomic profiling revealed an upregulation of the BAHD1 gene, which is involved in heterochromatin formation, across the 4 RR cell lines, corroborated by an enrichment of heterochromatin-related genesets. These results were consistent with functional characterization of RR versus WT cells by WB and IF indicating increased DNA repair capacity and heterochromatin response post-4 Gy irradiation. We further confirmed the dependency of the RR phenotype on the heterochromatin response with BAHD1 knockdown, resulting in reversal of these cellular responses. Finally, our results were supported by survival analyses indicating an inferior DFS post-RT in patients with HNC and PCa harboring a higher expression of heterochromatin-related geneset (HNC: HRhigh vs low 1.51 ; PCa: HRhigh vs low 1.37 ). Conclusions: Herein, by leveraging on paired experimental and clinico-molecular datasets, we have uncovered a novel BAHD1-dependent heterochromatin response that underpins resistance to RT in HNC and PCa, which may be amendable to systemic agents targeting chromatin remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea1270e10cc26b39aabc2ffd1632147257d9664" target='_blank'>
              Analysis of heterochromatin remodeling as a mechanism of radiotherapy (RT) resistance through molecular profiling of radioresistant (RR) head and neck (HNC) and prostate cancers (PCa).
              </a>
            </td>
          <td>
            Evelyn Tan, Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, Dennis J J Poon, Dewi Susanti, E. Yeo, Pek L Chu, Melvin L K Chua
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="microRNAs (miRNAs), tiny, non-coding RNA molecules, fine-tune the expression of target genes through interacting with mRNAs. These miRNAs are involved in a wide range of biological processes, encompassing cell division, death, blood cell production, and tumor development. When these miRNAs become dysfunctional, they can promote the invasion and spread of cancer cells in various human malignancies, including leukemia. Acute lymphoblastic leukemia (ALL), the preeminent malignancy affecting children, is a blood cancer marked by the uncontrollable growth of immature lymphoid cells that displace healthy blood precursors in the bone marrow. Despite a decline in ALL mortality rates over the past two decades, a significant proportion of deaths still results from a lack of effective diagnostic and prognostic markers that can guide treatment decisions and overcome drug resistance. The analysis of miRNA expression patterns in ALL could lead to more precise disease classification, earlier diagnosis, and better prognostic outcomes in the near future. The connection between miRNA dysfunction and the biology of ALL suggests that these molecules could represent promising therapeutic targets. Therefore, this review delves into the regulatory mechanisms of miRNAs in pediatric ALL, exploring how miRNA-based diagnostic, prognostic, and therapeutic strategies offer unique advantages and hold promise for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b51d3730adcae846d16d92db76f51f153c5c1d0" target='_blank'>
              Advancements in the Regulatory Role of microRNAs in Childhood Acute Lymphoblastic Leukemia: Mechanisms and Clinical Implications
              </a>
            </td>
          <td>
            Wei Deng
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ed5339b18c8cb661ce34053233bc7343f92ee3a" target='_blank'>
              1q jumping translocation as a biomarker in myeloid malignancy: frequently mutated genes associated with bad prognosis and low survival
              </a>
            </td>
          <td>
            Eitan Halper-Stromberg, V. Stinnett, L. Morsberger, Aparna Pallavajjala, M. Levis, A. DeZern, Michelle Lei, Brian Phan, Rena R. Xian, Christopher D. Gocke, Guilin Tang, Ying S Zou
          </td>
          <td>2024-08-01</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8eb55432a81ae9f11fc653e3722684f5e8a7ed1" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Ganciclovir (GCV) is a clinically important drug as it is used to treat viral infections. GCV is incorporated into the DNA during replication, where it interferes with subsequent replication on GCV-incorporated templates. However, the effects of GCV on the host genome and the mechanisms underlying cellular tolerance to GCV remain unclear. In this study, we explored these mechanisms using a collection of mutant DT40 cells. We identified RAD17/-, BRCA1-/-, and RAD18-/- cells as highly GCV-sensitive. RAD17, a component of the alternative checkpoint-clamp loader RAD17-RFC, was required for the activation of the intra-S checkpoint following GCV treatment. BRCA1, a critical factor for promoting homologous recombination (HR), was required for suppressing DNA double-strand breaks (DSBs). Moreover, RAD18, an E3-ligase involved in DNA repair, was critical in suppressing the aberrant ligation of broken chromosomes caused by GCV. We found that BRCA1 suppresses DSBs through HR-mediated repair and template switching (TS)-mediated damage bypass. Moreover, the strong GCV sensitivity of BRCA1-/- cells was rescued by the loss of 53BP1, despite the only partial restoration in the sister chromatid exchange events which are hallmarks of HR. These results indicate that BRCA1 promotes cellular tolerance to GCV through two mechanisms, TS and HR-mediated repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb90975f7c5cd2ebcb118dd3d319f550cae4f8b1" target='_blank'>
              RAD18- and BRCA1-dependent pathways promote cellular tolerance to the nucleoside analog ganciclovir.
              </a>
            </td>
          <td>
            Tasnim Ahmad, Ryotaro Kawasumi, Kouji Hirota
          </td>
          <td>2024-08-22</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Gene expression is a dynamic and stochastic process characterized by transcriptional bursting followed by periods of silence. Single-cell RNA sequencing (scRNA-seq) is a powerful tool to measure transcriptional bursting and silencing at the individual cell level. In this study, we introduce the single-cell Stochastic Gene Silencing (scSGS) method, which leverages the natural variability in single-cell gene expression to decipher gene function. For a target gene g under investigation, scSGS classifies cells into transcriptionally active (g+) and silenced (g-) samples. It then compares these cell samples to identify differentially expressed genes, referred to as SGS- responsive genes, which are used to infer the function of the target gene g. Analysis of real data demonstrates that scSGS can reveal direct regulatory relationships up- and downstream of target genes, circumventing the survivorship bias that often affects gene knockout and perturbation studies. scSGS thus offers an efficient approach for gene function prediction, with significant potential to reduce the use of genetically modified animals in gene function research. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74ad5f9a5377bde93caa62447ac3ad5e5e37e99c" target='_blank'>
              Gene Function Revealed at the Moment of Stochastic Gene Silencing
              </a>
            </td>
          <td>
            Shreyan Gupta, James J. Cai
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="One of the main causes of cancer-related mortality for women worldwide is breast cancer (BC). The XRCC2 gene, essential for DNA repair, has been implicated in cancer susceptibility. This study aims to evaluate the association between XRCC2 and BC risk. The study was conducted at Zheen International Hospital in Erbil, Iraq, between 2021 and 2024 with a total of 88 samples, including 44 paired normal and cancer tissue samples. Mutation analysis was performed using Next-Generation Sequencing, coupled with in silico tools for variant impact prediction. Expression levels were assessed through RT-PCR, and methylation status was determined using methylation-sensitive restriction enzyme digestion PCR. The study identified seven inherited germline variants in the XRCC2 gene, with five of these mutations being Uncertain Significance, one being Likely Pathogenic, and one being Likely benign. RNA purity was found high with mean A260/280 ratios of 1.986 ± 0.097 in normal (N) and 1.963 ± 0.092 in tumor (T) samples. Tumor samples exhibited a higher RNA concentration (78.56 ± 40.87 ng/µL) than normal samples (71.44 ± 40.79 ng/µL). XRCC2 gene expression was significantly upregulated in tumor tissue, with marked increases in patients aged 40-55 and >56 years and in higher cancer grades (II and III) and invasive ductal carcinoma (p-values ranging from <0.0001 to 0.0392). DNA methylation rates in tumor tissues were low (7%), suggesting limited regulation by methylation. The study suggests that XRCC2 can be classified as an oncogene and that its structural investigation by targeted NGS and expression evaluation can be used as a potential biomarker in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81c449dfd85b137aba37b45f3107a8eae5435a35" target='_blank'>
              Association of XRCC2 with breast cancer, a multi-omics analysis at genomic, transcriptomic, and epigenomic level.
              </a>
            </td>
          <td>
            N. Gilani, Mehmet Ozaslan
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Due to the significant morbidity and mortality of hemoglobinopathies, curative options have long been pursued. The overall goal of gene therapy is to modify a patient's own hematopoietic stem cells to overcome the deleterious effects of the underlying genetic defect by gene addition, gene editing, or gene silencing. Gene addition incorporates genes with superior function than the abnormal gene; gene editing takes advantage of molecular tools such as zinc finger proteins, Transcription Activator-Like Effector Nucleases and Clustered Regularly Interspaced Short Palindromic Repeats coupled with Cas9 proteins (CRISPR-Cas9) which allow for sequence-specific breaks in DNA that disrupt gene function; and gene silencing suppresses gene expression by interference with mRNA transcription/protein translation or epigenetic modification. The majority of gene therapy strategies for hemoglobinopathies have targeted erythroid-specific BCL11A, a major regulator of fetal hemoglobin repression at the gamma-globin locus, in the normal fetal-to-adult hemoglobin switch that occurs shortly after birth. Other goals have involved the incorporation of anti-sickling globins, such as βT87Q or βAS3. Landmark clinical trials of gene therapy in transfusion-dependent thalassemia and sickle cell disease have shown remarkable efficacy and acceptable safety and culminated in recent regulatory approvals of gene therapy for both diseases in Europe and the United States.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f812c9e54de47feca0c8615f3dda20f25f85b41" target='_blank'>
              A Review of Gene Therapies for Hemoglobinopathies.
              </a>
            </td>
          <td>
            Boubini Jones-Wonni, Amar H Kelkar, Maureen O Achebe
          </td>
          <td>2024-08-15</td>
          <td>Hemoglobin</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aberrant gene expression patterns in acute myeloid leukemia (AML) with balanced chromosomal translocations are often associated with dysregulation of epigenetic modifiers. The AML1/ETO (RUNX1/MTG8) fusion protein, caused by the translocation (8;21)(q22;q22), leads to the epigenetic repression of its target genes. We aimed in this work to identify critical epigenetic modifiers, on which AML1/ETO-positive AML cells depend on for proliferation and survival using shRNA library screens and global transcriptomics approaches. Using shRNA library screens, we identified 41 commonly depleted genes in two AML1/ETO-positive cell lines Kasumi-1 and SKNO-1. We validated, genetically and pharmacologically, DNMT1 and ATR using several AML1/ETO-positive and negative cell lines. We also demonstrated in vivo differentiation of myeloblasts after treatment with the DNMT1 inhibitor decitabine in a patient with an AML1/ETO-positive AML. Bioinformatic analysis of global transcriptomics after AML1/ETO induction in 9/14/18-U937 cells identified 973 differentially expressed genes (DEGs). Three genes (PARP2, PRKCD, and SMARCA4) were both downregulated after AML1/ETO induction, and identified in shRNA screens. In conclusion, using unbiased shRNA library screens and global transcriptomics, we have identified several driver epigenetic regulators for proliferation in AML1/ETO-positive AML. DNMT1 and ATR were validated and are susceptible to pharmacological inhibition by small molecules showing promising preclinical and clinical efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4848d5afaf1e1f3f5c753596a2ed8d1e491ca9f0" target='_blank'>
              Identification of epigenetic modifiers essential for growth and survival of AML1/ETO-positive leukemia.
              </a>
            </td>
          <td>
            J. Duque-Afonso, P. Veratti, Usama-Ur Rehman, Heike Herzog, J. Mitschke, G. Greve, Julian Eble, Bettina Berberich, Johanna Thomas, Milena Pantic, Miguel Waterhouse, Gaia Gentile, Olaf Heidenreich, Cornelius Miething, Michael Lübbert
          </td>
          <td>2024-08-15</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The elucidation of genetic architecture and molecular regulatory networks underlying complex traits remains a significant challenge in life science, largely due to the substantial background effects that arise from epistasis and gene-environment interactions. The chromosome segment substitution line (CSSL) is an ideal material for genetic and molecular dissection of complex traits due to its near-isogenic properties; yet a comprehensive analysis, from the basic identification of substitution segments to advanced regulatory network, is still insufficient. Here, we developed two cotton CSSL populations on the Gossypium hirsutum background, representing wide adaptation and high lint yield, with introgression from G. barbadense, representing superior fibre quality. We sequenced 99 CSSLs that demonstrated significant differences from G. hirsutum in fibre, and characterized 836 dynamic fibre transcriptomes in three crucial developmental stages. We developed a workflow for precise resolution of chromosomal substitution segments; the genome sequencing revealed substitutions collectively representing 87.25% of the G. barbadense genome. Together, the genomic and transcriptomic survey identified 18 novel fibre-quality-related quantitative trait loci with high genetic contributions and the comprehensive landscape of fibre development regulation. Furthermore, analysis determined unique cis-expression patterns in CSSLs to be the driving force for fibre quality alteration; building upon this, the co-expression regulatory network revealed biological relationships among the noted pathways and accurately described the molecular interactions of GhHOX3, GhRDL1 and GhEXPA1 during fibre elongation, along with reliable predictions for their interactions with GhTBA8A5. Our study will enhance more strategic employment of CSSL in crop molecular biology and breeding programmes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce2cb624684b99188c000abce2399d54db33d8c" target='_blank'>
              Unravelling the genetic basis and regulation networks related to fibre quality improvement using chromosome segment substitution lines in cotton.
              </a>
            </td>
          <td>
            Guoan Qi, Zhanfeng Si, Lisha Xuan, Z. Han, Yan Hu, Lei Fang, Fan Dai, Tianzhen Zhang
          </td>
          <td>2024-07-24</td>
          <td>Plant biotechnology journal</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="ABSTRACT The human T-cell lymphotropic virus type 1 (HTLV-1) is a single-stranded positive-sense RNA virus that belongs to the Retroviridae family, genus Deltaretro, and infects approximately five to 10 million people worldwide. Although a significant number of individuals living with HTLV-1 remain asymptomatic throughout their lives, some develop one or more severe clinical conditions, such as HTLV-1-associated myelopathy/tropical spastic paraparesis (HAM/TSP), a progressive and debilitating disease, and/or a subtype of non-Hodgkin’s lymphoma with a more threatening course known as adult T-cell leukemia/lymphoma (ATLL). Moreover, current therapeutic options are limited and focus primarily on treating symptoms and controlling viral latency. CRISPR-Cas9 gene editing is proposed as a promising tool to address the intricate links associated with HTLV-1. By targeting or silencing key genes during initial infection and dysregulating immune signaling pathways, CRISPR-Cas9 offers potential intervention opportunities. In this review, we address the therapeutic potential of CRISPR-Cas9 gene editing, as well as examine the primary mechanisms involved in editing potential target genes and discuss the existing evidence in the current scientific literature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa8304d3f00cdbae60023f6fc20b04a24438fa0" target='_blank'>
              Novel approaches for HTLV-1 therapy: innovative applications of CRISPR-Cas9
              </a>
            </td>
          <td>
            Wilson Domingues, V. A. Folgosi, S. Sanabani, Pedro Domingos Leite Junior, Tatiane Assone, J. Casseb
          </td>
          <td>2024-08-26</td>
          <td>Revista do Instituto de Medicina Tropical de São Paulo</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/761ad1cbab42eef53a71e8966883fae3c426f6a1" target='_blank'>
              Novel therapeutic agents in clinical trials: emerging approaches in cancer therapy
              </a>
            </td>
          <td>
            Deepak Chandra Joshi, Anurag Sharma, Sonima Prasad, Karishma Singh, Mayank Kumar, Kajal Sherawat, H. Tuli, Madhu Gupta
          </td>
          <td>2024-08-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Li-Fraumeni syndrome (LFS) is a hereditary cancer predisposition syndrome associated with a highly penetrant cancer spectrum characterized by germline TP53 mutations. We characterized the first LFS zebrafish hotspot mutants, tp53 R217H and R242H (human R248H and R273H), and found these mutants exhibit partial-to-no activation of p53 target genes, have a defective G1 cell-cycle checkpoint, and are resistant to apoptosis. Spontaneous tumor development histologically resembling human sarcomas developed as early as 6 months. tp53 R242H mutants had a higher lifetime tumor incidence compared to tp53 null and R217H mutants, suggesting it is a more aggressive mutation. We observed mutation-specific tumor phenotypes across tp53 mutants with associated diverse transcriptomic and DNA methylome profiles, impacting metabolism, cell signalling, and biomacromolecule synthesis and degradation. These tp53 zebrafish mutants demonstrate fidelity to their human counterparts and provide new insights into underlying tumorigenesis mechanisms and kinetics, which will guide targeted therapeutics for LFS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe84da849b9bf9707014c32841afe469ca3c5e99" target='_blank'>
              tp53 R217H and R242H Mutant Zebrafish Exhibit Dysfunctional p53 Hallmarks and Recapitulate Li-Fraumeni Syndrome Phenotypes
              </a>
            </td>
          <td>
            Kim Kobar, Lissandra Tuzi, Erin Burnley, Jennifer A. Fiene, Kristianne J. C. Galpin, C. Midgen, B. Laverty, Vallijah Subasri, Timmy T. Wen, Martin Hirst, M. Moksa, A. Carles, Qi Cao, A. Shlien, David Malkin, Sergey V. Prykhozhij, Jason N Berman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The XmnI Gg -158 C/T polymorphism has been widely associated with fetal hemoglobin (HbF) levels, the severity of disease, and the response to the drug hydroxyurea (HU) in both β-thalassemia (β-thal) and sickle cell disease (SCD) patients. However, the functional significance of this single nucleotide polymorphism (SNP) remains unclear. To gain insight, green fluorescence protein (GFP) cassettes harboring the XmnI C or T alleles in their left homology arms (i.e. Gg promoters) were knocked into the Gg gene(s) of K562 cells via CRISPR/Cas9. Subsequently, the GFP fluorescence levels were compared in the ensuing cell populations and isolated clones. In both instances, median fluorescence intensities (MFI) of the knockin cells having the inserted XmnI T allele were higher than those having the XmnI C allele. Our results suggest that the XmnI T allele can increase Gg expression in K562 cells. The possible functional significance of the XmnI Gg -158 C/T polymorphism provides a rationale for the aforementioned associations. Furthermore, the XmnI polymorphism as a functional SNP substantiates its importance as a prognostic marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bedc9fe1b28e8e9a54e59969786c8981bcb8830d" target='_blank'>
              Genome editing in K562 cells suggests a functional role for the XmnI Gg polymorphism: a widely used genetic marker in β-thalassemia and sickle cell disease patients.
              </a>
            </td>
          <td>
            A. Ahmadifard, Nahal Maroofi, Maryam Maleki Tehrani, Tahere Dabestani, Masoumeh Sadat Mousavi Maleki, Sepideh Bayrami, M. Banan
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Multidrug resistance (MDR) frequently occurs during cancer therapy and remains a major obstacle for the cure of most cancers(1, 2). Drug resistance could exist intrinsically or be acquired by drug treatment(3–5), yet factors that regulate the resistance remain elusive. Here, we show that most anticancer drugs damage neosynthesized proteins prior to reaching their canonic targets and elicit profound cytotoxicity, which is largely compensated by protein damage response (PDR). We demonstrate that the PDR includes damage recognition and clearance that are mainly mediated by ubiquitin and proteasome systems, although some other factors, including cellular ATP levels and proliferation status, are also involved. We show that cancer stem cells (CSCs), which have lower protein synthesis, and drug resistance acquired cells (DRAC), which have higher proteasome activity, are more resistant than other cells. We further demonstrate that ATP promotes protein synthesis and suppresses proteasome activity, thus, increasing mitochondrial ATP production by PDK1 inhibition and using proteasome inhibitor to block protein damage clearance render CSCs and DRACs more vulnerable to anticancer drugs. Thus, patients with drug-resistant cancers and treatment-naïve patients with low ATP levels and/or high proteasome activity can be identified and subtyped, and therapies containing PDK1-I and/or proteasome-I may be effective options for these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e9626795c580f97c2519d1342441e9145cbcfc" target='_blank'>
              A critical role of protein damage response in mediating cancer drug resistance
              </a>
            </td>
          <td>
            Fangyuan Shao, Jie Li, Hao Xiao, Ling Li, Bo Li, YuJun Chen, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Yuzhong Peng, Yujian Huang, Lijian Wang, Yanxia Shi, Nan Shao, Kai Miao, Changhua Zhang, Ying Lin, Jun Yan, Kin Yip Tam, Xiaoling Xu, Chu-Xia Deng
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Genome editing has demonstrated its utility in generating isogenic cell-based disease models, enabling the precise introduction of genetic alterations into wild-type cells to mimic disease phenotypes and explore underlying mechanisms. However, its application in liver-related diseases has been limited by challenges in genetic modification of mature hepatocytes in a dish. Here, we conducted a systematic comparison of various methods for primary hepatocyte culture and gene delivery to achieve robust genome editing of hepatocytes ex vivo. Our efforts yielded editing efficiencies of up to 80% in primary murine hepatocytes cultured in monolayer and 20% in organoids. To model human hepatic tumorigenesis, we utilized hepatocytes differentiated from human pluripotent stem cells (hPSCs) as an alternative human hepatocyte source. We developed a series of cellular models by introducing various single or combined oncogenic alterations into hPSC-derived hepatocytes. Our findings demonstrated that distinct mutational patterns led to phenotypic variances, affecting both overgrowth and transcriptional profiles. Notably, we discovered that the PI3KCA E542K mutant, whether alone or in combination with exogenous c-MYC, significantly impaired hepatocyte functions and facilitated cancer metabolic reprogramming, highlighting the critical roles of these frequently mutated genes in driving liver neoplasia. In conclusion, our study demonstrates genome-engineered hepatocytes as valuable cellular models of hepatocarcinoma, providing insights into early tumorigenesis mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0098e14922960e96572df20d519a8a83fa8956" target='_blank'>
              Genome Engineering of Primary and Pluripotent Stem Cell-Derived Hepatocytes for Modeling Liver Tumor Formation
              </a>
            </td>
          <td>
            Lulu Zhang, Xunting Wang, Xuelian Yang, Yijia Chi, Yihang Chu, Yi Zhang, Yufan Gong, Fei Wang, Qian Zhao, Dongxin Zhao
          </td>
          <td>2024-09-02</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09486473d5ae00655afa329978bc6a48b65d161f" target='_blank'>
              The response to influenza vaccination is associated with DNA methylation-driven regulation of T cell innate antiviral pathways
              </a>
            </td>
          <td>
            Hongxiang Fu, H. Pickering, L. Rubbi, Ted M. Ross, Wanding Zhou, Elaine F. Reed, Matteo Pellegrini
          </td>
          <td>2024-08-21</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bae52efab4bf3e15cee8010e7024c2247940163f" target='_blank'>
              Epigenetic gene regulation in plants and its potential applications in crop improvement.
              </a>
            </td>
          <td>
            Heng Zhang, Jian-Kang Zhu
          </td>
          <td>2024-08-27</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Alexander Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii-DerIda Chen, C. Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="Abstract CRISPR/Cas9 genome editing is a pervasive research tool due to its relative ease of use. However, some systems are not amenable to generating edited clones due to genomic complexity and/or difficulty in establishing clonal lines. For example, Drosophila Schneider 2 (S2) cells possess a segmental aneuploid genome and are challenging to single-cell select. Here, we describe a streamlined CRISPR/Cas9 methodology for knock-in and knock-out experiments in S2 cells, whereby an antibiotic resistance gene is inserted in-frame with the coding region of a gene-of-interest. By using selectable markers, we have improved the ease and efficiency for the positive selection of null cells using antibiotic selection in feeder layers followed by cell expansion to generate clonal lines. Using this method, we generated the first acentrosomal S2 cell lines by knocking-out centriole genes Polo-like Kinase 4/Plk4 or Ana2 as proof of concept. These strategies for generating gene-edited clonal lines will add to the collection of CRISPR tools available for cultured Drosophila cells by making CRISPR more practical and therefore improving gene function studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d9bec46a5fefb60dc99b1e4c11b9d69666ffa32" target='_blank'>
              Generating CRISPR-edited clonal lines of cultured Drosophila S2 cells
              </a>
            </td>
          <td>
            John M. Ryniawec, Anastasia Amoiroglou, Gregory C. Rogers
          </td>
          <td>2024-08-17</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Background: Dynamic regulation of gene expression is fundamental for cellular adaptation to exogenous stressors. PTEFb-mediated promoter proximal pause-release of Pol II is a conserved regulatory mechanism for synchronous transcriptional induction best described in response to heat shock, but this pro-survival role has not been examined in the applied context of cancer therapy. Design/Method: In order to examine the dynamics of chromatin reorganization following radiotherapy, we performed a combination of ChIP-, ATAC-, and RNA-seq in model systems of diffuse intrinsic pontine glioma (DIPG) and other pediatric high-grade gliomas (pHGG) following IR exposure. We interrogated IR-induced gene expression in the presence or absence of PTEFb blockade, including both mechanistic and functional consequences of concurrent inhibition or genetic depletion. We utilized culture models with live cell imaging to assess the therapeutic synergy of PTEFb inhibition with IR, as well as the therapeutic index of this intervention relative to normal controls. Finally, we employed orthotopic models of pHGG treated with conformal radiotherapy and CNS-penetrant PTEFb inhibitors in order to assess tolerability and anti-tumor effect in vivo. Results: Rapid genome-wide redistribution of active chromatin features and PTEFb facilitates Pol II pause-release to drive nascent transcriptional induction within hours of exposure to therapeutic ionizing radiation. Concurrent inhibition of PTEFb imparts a transcription elongation defect, abrogating canonical adaptive programs such as DNA damage repair and cell cycle regulation. This combination demonstrates a potent, synergistic therapeutic potential agnostic of glioma subtype, leading to a marked induction of tumor cell apoptosis and prolongation of xenograft survival.Conclusion: These studies reveal a central role for PTEFb underpinning the early adaptive response to radiotherapy, opening new avenues for combinatorial treatment in these lethal malignancies.
 Citation Format: Faye M. Walker, Lays Martin Sobral, Etienne Danis, Bridget Sanford, Ilango Balakrishnan, Dong Wang, Angela Pierce, Sana Karam, Natalie J. Serkova, Nicholas K. Foreman, Sujatha Venkataraman, Robin Dowell, Rajeev Vibhakar, Nathan A. Dahl. Rapid PTEFb-dependent transcriptional reorganization underpins the glioma adaptive response to radiotherapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cbc389c63c005639d0ccef6f75bb33d60a2a03" target='_blank'>
              Abstract B078: Rapid PTEFb-dependent transcriptional reorganization underpins the glioma adaptive response to radiotherapy
              </a>
            </td>
          <td>
            Faye Walker, L. M. Sobral, Etienne P. Danis, Bridget Sanford, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Sana D Karam, Natalie J Serkova, Nicholas K. Foreman, S. Venkataraman, Robin Dowell, Rajeev Vibhakar, N. Dahl
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Precision medicine approaches such as gene editing via clustered regularly interspaced short palindromic repeat (CRISPR) and their associated protein (Cas-9) are revolutionizing treatment strategies for complex diseases such as Alzheimer and cancer. In this review, we explore the application of CRISPR-Cas9 in advancing therapies for these conditions, as well as its potential in targeting senescent cells. Since the risk of Alzheimer’s disease is strongly associated with genetic mutations and variations, the use of gene editing technologies to rectify these genetic errors by modifying disease-associated proteins becomes relevant. For cancer, to enhance immunotherapy approaches, modification of immune cells have been utilized to improve their anti-tumor efficacy. Additionally, the review also investigates the role of CRISPR-Cas9 in targeting senescent cells, which are implicated in both aging-related disorders and cancer progression. While challenges remain in introducing delivery methods and specificity, CRISPR-Cas9 represents a significant advancement in developing targeted, personalized treatments for these challenging health issues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0022d5a1dcbe7db7f0bd9b012fe358d45973badd" target='_blank'>
              Precision Gene Editing Strategies with CRISPR-Cas9 for Advancing Cancer Immunotherapy and Alzheimer's Disease
              </a>
            </td>
          <td>
            Priyanka Gupte, Karan Dhingra, Saloni
          </td>
          <td>2024-07-28</td>
          <td>Journal of Knowledge Learning and Science Technology ISSN: 2959-6386 (online)</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c24f0b42670eaa7d910971701aacf3a8d5213b" target='_blank'>
              Targeting RBM39 through indisulam induced mis-splicing of mRNA to exert anti-cancer effects in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tongting Ji, Yang Yang, Juanjuan Yu, Hongli Yin, Xinran Chu, Pengju Yang, Ling Xu, Xiaodong Wang, Shaoyan Hu, Yizhen Li, Xiaochen Wu, Wengyuan Liu, Bi Zhou, Wenjuan Wang, Shuqi Zhang, Wei Cheng, Yanling Chen, Lei Shi, Zhiheng Li, Ran Zhuo, Yongping Zhang, Yanfang Tao, Di Wu, Xiao-Lu Li, Zimu Zhang, Jun-Jie Fan, Jian Pan, Jun Lu
          </td>
          <td>2024-07-24</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Metabolic dysfunction-associated Fatty Liver Disease (MAFLD) has emerged as one of the leading cardiometabolic diseases. Friend of GATA2 (FOG2) is a transcriptional co-regulator that has been shown to regulate hepatic lipid metabolism and accumulation. Using meta-analysis from several different biobank datasets, we identified a coding variant of FOG2 (rs28374544, A1969G, S657G) predominantly found in individuals of African ancestry (minor allele frequency~20%), which is associated with liver failure/cirrhosis phenotype and liver injury. To gain insight into potential pathways associated with this variant, we interrogated a previously published genomics dataset of 38 human induced pluripotent stem cell (iPSCs) lines differentiated into hepatocytes (iHeps). Using Differential Gene Expression Analysis and Gene Set Enrichment Analysis, we identified the mTORC1 pathway as differentially regulated between iHeps from individuals with and without the variant. Transient lipid-based transfections were performed on the human hepatoma cell line (Huh7) using wild-type FOG2 and FOG2S657G and demonstrated that FOG2S657G increased mTORC1 signaling, de novo lipogenesis, and cellular triglyceride synthesis and mass. In addition, we observed a significant downregulation of oxidative phosphorylation in FOG2S657G cells in fatty acid-loaded cells but not untreated cells, suggesting that FOG2S657G may also reduce fatty acid to promote lipid accumulation. Taken together, our multi-pronged approach suggests a model whereby the FOG2S657G may promote MAFLD through mTORC1 activation, increased de novo lipogenesis, and lipid accumulation. Our results provide insights into the molecular mechanisms by which FOG2S657G may affect the complex molecular landscape underlying MAFLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2945851076c2f51a928c3f7a7537ad4ec5a83e4e" target='_blank'>
              Genotype-First Approach Identifies an Association between rs28374544/FOG2S657G and Liver Disease through Alterations in mTORC1 Signaling
              </a>
            </td>
          <td>
            Donna Conlon, Siri Kanakala, Tess A. Cherlin, Yi-An Ko, Cecilia Vitali, Sharavana Gurunathan, Rasika Venkatesh, Jakob Woerner, Lindsay Guare, Penn Medicine Biobank, Anurag Verma, S. Verma, Marie A. Guerraty
          </td>
          <td>2024-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2dde0164af0025d5f19bf691294f177add4f7b" target='_blank'>
              Low-pass whole genome sequencing of circulating tumor cells to evaluate chromosomal instability in triple-negative breast cancer
              </a>
            </td>
          <td>
            S. Di Cosimo, M. Silvestri, C. De Marco, Alessia Calzoni, M. C. De Santis, Maria Grazia Carnevale, C. Reduzzi, Massimo Cristofanilli, Vera Cappelletti
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Inflammation is activated by diverse triggers that induce the expression of cytokines and adhesion molecules, which permit a succession of molecules and cells to deliver stimuli and functions that help the immune system clear the primary cause of tissue damage, whether this is an infection, a tumor, or a trauma. During inflammation, short-term changes in the expression and secretion of strong mediators of inflammation occur, while long-term changes occur to specific groups of cells. Long-term changes include cellular transdifferentiation for some types of cells that need to regenerate damaged tissue, as well as death for specific immune cells that can be detrimental to tissue integrity if they remain active beyond the boundaries of essential function. The transcriptional regulator NFκB enables some of the fundamental gene expression changes during inflammation, as well as during tissue development. During recurrence of malignant disease, cell stress-induced alterations enable the growth of cancer cell clones that are substantially resistant to therapeutic intervention and to the immune system. A number of those alterations occur due to significant defects in feedback signal cascades that control the activity of NFκB. Specifically, cell stress contributes to feedback defects as it overrides modules that otherwise control inflammation to protect host tissue. NFκB is involved in both the suppression and promotion of cancer, and the key distinctive feature that determines its net effect remains unclear. This paper aims to provide a clear answer to at least one aspect of this question, namely the mechanism that enables a divergent response of cancer cells to critical inflammatory stimuli and to cell stress in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06e3d0a6a36e79544ef60b790aea44247ff4d140" target='_blank'>
              Divergent Processing of Cell Stress Signals as the Basis of Cancer Progression: Licensing NFκB on Chromatin
              </a>
            </td>
          <td>
            S. Vlahopoulos
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G. Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genetic suppression occurs when the phenotypic defects caused by a deleterious mutation are rescued by another mutation. Suppression interactions are of particular interest for genetic diseases, as they identify ways to reduce disease severity, thereby potentially highlighting avenues for therapeutic intervention. To what extent suppression interactions are influenced by the genetic background in which they operate remains largely unknown. However, a high degree of suppression conservation would be crucial for developing therapeutic strategies that target suppressors. To gain an understanding of the effect of the genetic context on suppression, we isolated spontaneous suppressor mutations of temperature sensitive alleles of SEC17, TAO3, and GLN1 in three genetically diverse natural isolates of the budding yeast Saccharomyces cerevisiae. After identifying and validating the genomic variants responsible for suppression, we introduced the suppressors in all three genetic backgrounds, as well as in a laboratory strain, to assess their specificity. Ten out of eleven tested suppression interactions were conserved in the four yeast strains, although the extent to which a suppressor could rescue the temperature sensitive mutant varied across genetic backgrounds. These results suggest that suppression mechanisms are highly conserved across genetic contexts, a finding that is potentially reassuring for the development of therapeutics that mimic genetic suppressors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b338b4576ed119eb8873cb1fa25b1257f15a3993" target='_blank'>
              Genetic suppression interactions are highly conserved across genetic backgrounds
              </a>
            </td>
          <td>
            Claire Paltenghi, Jolanda van Leeuwen
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Non-coding regulatory sequences play essential roles in adjusting gene output to cellular needs and are thus critical to animal development and health. Numerous such sequences have been identified in mammalian genomes ranging from transcription factors binding motifs to recognition sites for RNA-binding proteins and non-coding RNAs. The advent of CRISPR has raised the possibility of assigning functionality to individual endogenous regulatory sites by facilitating the generation of isogenic cell lines that differ by a defined set of genetic modifications. Here we investigate the usefulness of this approach to assign function to individual miRNA binding sites. We find that the process of generating isogenic pairs of mammalian cell lines with CRISPR-mediated mutations introduces extensive molecular and phenotypic variability between biological replicates making any attempt of assigning function to the binding site essentially impossible. Our work highlights an important consideration when employing CRISPR editing to characterize non-coding regulatory sequences in cell lines and calls for the development and adoption of alternative strategies to address this question in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1d3a055f759b974342766678fe87bc5f92b153" target='_blank'>
              CRISPR-based dissection of miRNA binding sites using isogenic cell lines is hampered by pervasive noise
              </a>
            </td>
          <td>
            Mahendra Prajapat, Joana A. Vidigal
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Elucidating the genetic architecture of DNA methylation (DNAm) is crucial for decoding the etiology of complex diseases. However, current epigenomic studies often suffer from incomplete coverage of methylation sites and the use of tissues containing heterogeneous cell populations. To address these challenges, we present a comprehensive human methylome atlas based on deep whole-genome bisulfite sequencing (WGBS) and whole-genome sequencing (WGS) of purified monocytes from 298 European Americans (EA) and 160 African Americans (AA) in the Louisiana Osteoporosis Study. Our atlas enables the analysis of over 25 million DNAm sites. We identified 1,383,250 and 1,721,167 methylation quantitative trait loci (meQTLs) in cis-regions for EA and AA populations, respectively, with 880,108 sites shared between ancestries. While cis-meQTLs exhibited population-specific patterns, primarily due to differences in minor allele frequencies, shared cis-meQTLs showed high concordance across ancestries. Notably, cis-heritability estimates revealed significantly higher mean values in the AA population (0.09) compared to the EA population (0.04). Furthermore, we developed population-specific DNAm imputation models using Elastic Net, enabling methylome-wide association studies (MWAS) for 1,976,046 and 2,657,581 methylation sites in EA and AA, respectively. The performance of our MWAS models was validated through a systematic multi-ancestry analysis of 41 complex traits from the Million Veteran Program. Our findings bridge the gap between genomics and the monocyte methylome, uncovering novel methylation-phenotype associations and their transferability across diverse ancestries. The identified meQTLs, MWAS models, and data resources are freely available at www.gcbhub.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02561d4c021777b32cc89ad698157b2f333e71dc" target='_blank'>
              An atlas of genetic effects on the monocyte methylome across European and African populations
              </a>
            </td>
          <td>
            Wanheng Zhang, Xiao Zhang, Chuan Qiu, Zichen Zhang, Kuan-Jui Su, Zhe Luo, Minghui Liu, Bingxin Zhao, Lang Wu, Qing Tian, Hui Shen, Chong Wu, Hong-Wen Deng
          </td>
          <td>2024-08-13</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Human inborn errors of immunity (IEI) represent a diverse group of genetic disorders affecting the innate and/or adaptive immune system. Some IEI entities comprise defects in DNA repair factors, resulting in (severe) combined immunodeficiencies, bone marrow failure, predisposition to malignancies, and potentially result in radiosensitivity (RS). While other IEI subcategories such as common variable immunodeficiency (CVID) and immune dysregulation disorders also associate with lymphoproliferative and malignant complications, the occurrence of RS phenotypes in the broader IEI population is not well characterized. Nonetheless, identifying RS in IEI patients through functional testing is crucial to reconsider radiation-related therapeutic protocols and to improve overall patient management. This study aimed to investigate chromosomal RS in a diverse cohort of 107 IEI patients using the G0 cytokinesis-block micronucleus (MN) assay. Our findings indicate significant variability in RS across specific genetic and phenotypical subgroups. Severe RS was detected in all ataxia-telangiectasia (AT) patients, a FANCI deficient and ERCC6L2 deficient patient, but not in any other IEI patient included in this cohort. Age emerged as the single influencing factor for both spontaneous and radiation-induced MN yields, while the manifestation of additional clinical features, including infection susceptibility, immune dysregulation, or malignancies did not associate with increased MN levels. Our extensive analysis of RS in the IEI population underscores the clinical importance of RS assessment in AT patients and supports RS testing in all IEI patients suspected of having a DNA repair disorder associated with radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b4f50595c5c29676228d24464dde5631290eb13" target='_blank'>
              Investigating chromosomal radiosensitivity in inborn errors of immunity: insights from DNA repair disorders and beyond.
              </a>
            </td>
          <td>
            E. Beyls, Evi Duthoo, Lynn Backers, Karlien Claes, Marieke De Bruyne RAPID clinicians, L. Pottie, Victoria Bordon, C. Bonroy, S. Tavernier, Kathleen BM Claes, A. Vral, A. Baeyens, F. Haerynck, T. Kerre, B. Lambrecht, L. Hoste, B. Meertens, T. V. Genechten, A. Ferster, K. V. Driessche, C. D. Vriendt, L. Dedeken, I. Moors, C. Dhooge, L. Kornreich, C. Heijmans, M. L. Roux, M. Colard, A. V. Damme, K. V. Schil, B. D. Wilde
          </td>
          <td>2024-08-20</td>
          <td>None</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b273ccaf2ba1c9f2d8cb9b79c03b12ee6eed17c1" target='_blank'>
              Modulatory effects of miRNAs in doxorubicin resistance: A mechanistic view.
              </a>
            </td>
          <td>
            Fatemeh Ebadi Meinag, Mina Fatahi, Vahid Vahedian, N. F. Maroufi, Bashir Mosayyebi, Elham Ahmadi, Mohammad Rahmati
          </td>
          <td>2024-09-02</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background Glycosyltransferase-associated genes play a crucial role in hepatocellular carcinoma (HCC) pathogenesis. This study investigates their impact on the tumor microenvironment and molecular mechanisms, offering insights into innovative immunotherapeutic strategies for HCC. Methods We utilized cutting-edge single-cell and spatial transcriptomics to examine HCC heterogeneity. Four single-cell scoring techniques were employed to evaluate glycosyltransferase genes. Spatial transcriptomic findings were validated, and bulk RNA-seq analysis was conducted to identify prognostic glycosyltransferase-related genes and potential immunotherapeutic targets. MGAT1’s role was further explored through various functional assays. Results Our analysis revealed diverse cell subpopulations in HCC with distinct glycosyltransferase gene activities, particularly in macrophages. Key glycosyltransferase genes specific to macrophages were identified. Temporal analysis illustrated macrophage evolution during tumor progression, while spatial transcriptomics highlighted reduced expression of these genes in core tumor macrophages. Integrating scRNA-seq, bulk RNA-seq, and spatial transcriptomics, MGAT1 emerged as a promising therapeutic target, showing significant potential in HCC immunotherapy. Conclusion This comprehensive study delves into glycosyltransferase-associated genes in HCC, elucidating their critical roles in cellular dynamics and immune cell interactions. Our findings open new avenues for immunotherapeutic interventions and personalized HCC management, pushing the boundaries of HCC immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/763f4f63b6204f3c799fef3e16a59e2fdcfb352e" target='_blank'>
              Elucidating the multifaceted role of MGAT1 in hepatocellular carcinoma: integrative single-cell and spatial transcriptomics reveal novel therapeutic insights
              </a>
            </td>
          <td>
            Yang Li, Yuan Chen, Danqiong Wang, Ling Wu, Tao Li, Na An, Haikun Yang
          </td>
          <td>2024-07-16</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/609bb25145d5246600ce4feb000ce7f09c4474b7" target='_blank'>
              Building a translational cancer dependency map for The Cancer Genome Atlas
              </a>
            </td>
          <td>
            Xu Shi, Christos Gekas, Daniel Verduzco, Sakina M. Petiwala, Cynthia Jeffries, Charles Lu, Erin Murphy, Tifani Anton, Andy H Vo, Zhiguang Xiao, Padmini Narayanan, Bee-Chun Sun, Aloma L D'Souza, J. M. Barnes, Somdutta Roy, Cyril Ramathal, Michael J. Flister, Zoltan Dezso
          </td>
          <td>2024-07-15</td>
          <td>Nature Cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is an incurable malignancy characterised by altered expression of coding and non-coding genes promoting tumour growth and drug resistance. Although the crucial role of long non-coding RNAs (lncRNAs) in MM is clearly established, the function of the non-coding RNAome, which might allow the design of novel therapeutics, is largely unknown. We performed an unbiased CRISPR-Cas9 loss-of-function screen of 671 lncRNAs in MM cells and their Bortezomib (BZB)-resistant derivative. To rank functionally and clinically relevant candidates, we designed and used a bioinformatic prioritisation pipeline combining functional data from cellular screens with prognostic and transcriptional data from MM patients. With this approach, we unveiled and prioritised 8 onco-lncRNAs essential for MM cell fitness, associated with high expression and poor prognosis in MM patients. The previously uncharacterised RP11-350G8.5 emerged as the most promising target, irrespective of BZB resistance. We i) demonstrated the anti-tumoral effect obtained by RP11-350G8.5 inhibition in vitro and in vivo; ii) highlighted a modulation of the unfolded protein response and the induction of immunogenic cell death triggered by the RP11-350G8.5 knock-out, via RNA-sequencing and molecular studies; iii) characterised its cytoplasmic homing through RNA-FISH; iv) predicted its 2D structure and identified 2 G-quadruplex and 3 hairpin-forming regions by biophysical assays, including Thioflavin T, 1H-NMR and circular dichroism to pave the way to the development of novel targeted therapeutics. Overall we provided innovative insights about unexplored lncRNAs in MM and identified RP11-350G8.5 as an oncogenic target for treatment-naïve and BZB-resistant MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b3038ab25be0410e0d1aec773f7b5564cf954f8" target='_blank'>
              An unbiased lncRNAs dropout CRISPR-Cas9 screen reveals RP11-350G8.5 as a novel therapeutic target for Multiple Myeloma.
              </a>
            </td>
          <td>
            K. Grillone, S. Ascrizzi, Paolo Cremaschi, Jussara Amato, N. Polerà, Ottavio Croci, R. Rocca, C. Riillo, Francesco Conforti, Raffaele Graziano, Diego Brancaccio, Daniele Caracciolo, Stefano Alcaro, Bruno Pagano, Antonio Randazzo, P. Tagliaferri, F. Iorio, P. Tassone
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 Personalized medicine programs, including the Zero Childhood Cancer Program (ZERO), perform molecular analysis of individual patient tumours to identify molecular targets for therapy. The comprehensive and unbiased sequencing approaches used by ZERO, including whole genome sequencing (WGS) and RNA sequencing (RNAseq), enables detection of novel structural variants (SVs) that may be cryptic to standard cytogenetic techniques or panel sequencing approaches. Identification of Receptor Tyrosine Kinase (RTK) activating variants in paediatric cancers, for example NTRK2 gene fusions, is a high priority given the availability and remarkable clinical success of RTK inhibitors. Here we describe a novel internal tandem duplication (ITD) in NTRK2, encoding TRKB, in a ZERO patient with CNS neuroblastoma. This SV was detected by WGS and was notably missed by RNAseq-based gene fusion and SV detection. The ITD spans exons 10-13 of NTRK2, is in-frame, and includes the juxtamembrane and transmembrane protein domains. We hypothesized that this ITD would result in constitutive activation of TRKB and would be sensitive to TRK-targeted therapies. We overexpressed NTRK2 ITD, wild-type NTRK2, and SPECC1L-NTRK2 (an established TRK-activating fusion), in the interleukin-3 (IL-3)-dependent cell line, Ba/F3, and the neuroblastoma cell line, SH-SY5Y. In these models, we showed that like wild-type NTRK2, NTRK2 ITD was expressed on the cell surface. NTRK2 ITD was sufficient to transform Ba/F3 cells to IL-3 independence through constitutive activation of TRKB and downstream signaling pathways PI3K/AKT and MEK/ERK. NTRK2 ITD-expressing cells were sensitive to TRK inhibitors, including larotrectinib, at similar doses to SPECC1L-NTRK2-expressing cells. Interestingly, we observed that cells expressing NTRK2 ITD were specifically sensitive to MEK inhibition, which was not observed in SPECC1L-NTRK2-expressing cells, suggesting the mechanisms by which these variants drive transformation is different. Indeed, in silico structural analysis showed that the duplicated region of NTRK2 ITD contains two internalized tyrosine residues whose phosphorylation could both drive autophosphorylation of the receptor and act as docking sites for adaptor proteins. Mutation of these tyrosine residues delayed transformation of Ba/F3 cells. This study functionally characterizes a novel NTRK2 ITD and shows that this variant is transforming and sensitive to TRK inhibition. While ITDs have been described in other RTK genes, notably FGFR1 and FLT3, structurally these have involved the kinase domain or the juxtamembrane domain. This is the first report of an ITD that spans the transmembrane domain of an RTK, characterizing an additional mechanism by which RTKs can be activated in cancer. This study highlights the value of unbiased WGS approaches to identify novel SVs and identifies another class of patients that may benefit from RTK inhibitor therapy.
 Citation Format: Lauren M. Brown, Gabor Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah Jalud, Teresa Sadras, Pamela Ajuyah, Paulette Barahona, M. Emmy M. Dolman, Marie Wong, Chelsea Mayoh, Mark J. Cowley, Paul G. Ekert. A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ae609e3759161918ab8221b777b148ec5605447" target='_blank'>
              Abstract B002: A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation
              </a>
            </td>
          <td>
            , G. Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah B Jalud, Teresa Sadras, P. Ajuyah, P. Barahona, M. E. M. Dolman, Marie Wong, C. Mayoh, Mark J. Cowley, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8245d23a6d26f49afe8896fd7ee25b1dbf95a8d" target='_blank'>
              Tailored antisense oligonucleotides designed to correct aberrant splicing reveal actionable groups of mutations for rare genetic disorders
              </a>
            </td>
          <td>
            H. Wai, Eliska Svobodova, Natalia Romero Herrera, Andrew G L Douglas, John W Holloway, F. Baralle, Marco Baralle, D. Baralle
          </td>
          <td>2024-08-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Introduction: The role of genetic variants in response to chemotherapy has been investigated in several studies. This study aimed to investigate genetic variants associated with response to chemotherapy in breast cancer (BC) patients. Methods: Significant variants (p < 5 × 10-8) associated with response to chemotherapy were obtained from GWA studies. Candidate variants were identified by haplotype analysis (r2 ≥ 0.9, D'≥0.9) using 1000Genome LD data. To determine the effects of the variants on gene expression, expression quantitative trait loci (eQTL) were evaluated. To compare the expression of the identified genes in tumor samples, expression levels were compared between TCGA tumor types and adjacent normal tissues. Results: Six rs3820706, rs147451859, rs4784750, rs17587029, rs16830728, and rs16972207 variants were significantly associated with response to chemotherapy in BC patients (p < 5 × 10-8). Seven novel haplotypic structures were identified to be associated with adverse response to chemotherapy in BC patients. These haplotypes formed two genetic structures associated with neutropenia, leukopenia, chemotherapy-induced cytotoxicity (GAG-TTAT), and chemotherapy-induced alopecia (CC-CAACTCCCGTTGCGG). These variants are located on PPCDC, NLRC5, STAM2, and TNFSF13B genes, and the expression of these genes significantly changed in BC tissues than normal tissues (P ≤ 0.05), also showing gene-gene correlation (P ≤ 0.05). Conclusions: These genetic variants and their associated novel haplotypic structures can predict adverse response to chemotherapy in BC patients and could potentially form BC-associated genetic panel for adverse response to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f6f813884867edf6f2b79ca7de2ac3b63826948" target='_blank'>
              Novel genetic structures associated with adverse response to chemotherapy in breast cancer.
              </a>
            </td>
          <td>
            Morteza Gholami, Mohsen Asouri, A. Ahmadi, Mehrab Nasirikenari
          </td>
          <td>2024-08-28</td>
          <td>Journal of oncology pharmacy practice : official publication of the International Society of Oncology Pharmacy Practitioners</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c25293ccda82c6d001fe5bae96c805b5e1fa7f9a" target='_blank'>
              The role of DNA polymerase I in tolerating single-strand breaks generated at clustered DNA damage in Escherichia coli
              </a>
            </td>
          <td>
            N. Shikazono, Ken Akamatsu
          </td>
          <td>2024-08-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Pleiotropy, the phenomenon in which a single gene influences multiple traits, is a fundamental concept in genetics. However, the evolutionary mechanisms underlying pleiotropy require further investigation. In this study, we conducted parallel gene knockouts targeting 100 transcription factors in two strains of Saccharomyces cerevisiae. We systematically examined and quantified the pleiotropic effects of these knockouts on gene expression levels for each transcription factor. Our results showed that the knockout of a single gene generally affected the expression levels of multiple genes in both strains, indicating various degrees of pleiotropic effects. Strikingly, the pleiotropic effects of the knockouts change rapidly between strains in different genetic backgrounds, and ∼85% of them were non-conserved. Further analysis revealed that the conserved effects tended to be functionally associated with the deleted transcription factors, while the non-conserved effects appeared to be more ad hoc responses. In addition, we measured 184 yeast cell morphological traits in these knockouts and found consistent patterns. In order to investigate the evolutionary processes underlying pleiotropy, we examined the pleiotropic effects of standing genetic variations in a population consisting of ∼1000 hybrid progenies of the two strains. We observed that newly evolved expression quantitative trait loci (eQTLs) impacted the expression of a greater number of genes than did old eQTLs, suggesting that natural selection is gradually eliminating maladaptive or slightly deleterious pleiotropic responses. Overall, our results show that, although being prevalent for new mutations, the majority of pleiotropic effects observed are evolutionarily transient, which explains how evolution proceeds despite complicated pleiotropic effects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/74979a3b8c99e59fe3aefc2b4e6cac48414c9093" target='_blank'>
              Most pleiotropic effects of gene knockouts are evolutionarily transient in yeasts.
              </a>
            </td>
          <td>
            Li Liu, Yao Liu, Lulu Min, Zhenzhen Zhou, Xingxing He, YunHan Xie, Waifang Cao, Shuyun Deng, Xiaoju Lin, Xionglei He, Xiaoshu Chen
          </td>
          <td>2024-09-06</td>
          <td>Molecular biology and evolution</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="For several decades, induction therapy with nucleoside analogs, in particular cytarabine (Ara-C) and, to a lesser extent, fludarabine, has been the standard of care for patients diagnosed with acute myeloid leukemia (AML). Still, the anti-tumor efficacy of nucleoside analogs is often limited by intrinsic and acquired drug resistance, thereby leading to poor therapeutic response and suboptimal clinical outcomes. Here, we used genome-wide CRISPR-based pharmacogenomic screening to map the genetic factors that modulate the response to nucleoside analogs in AML and identified the E3 ubiquitin ligase Herc1 as a key modulator of Ara-C response in the MLL/AF9 (MA) and the HOXA9/MEIS1 (HM) murine AML models both in vitro and in vivo. Loss of HERC1 enhanced nucleoside analog-induced cell death in both murine and human AML cell lines by compromising cell cycle progression. In-depth proteomic analysis and subsequent validation identified deoxycytidine kinase Dck as a novel target of Herc1 in MA and HM murine cells. We observed that HERC1 is overexpressed in AML compared to other cancer types and higher HERC1 expression is associated with shorter overall survival of patients with AML in the TCGA and BEAT-AML cohorts. Collectively, this study highlights the importance of HERC1 in the response of AML cells to nucleoside analogs, thereby establishing this E3 ubiquitin ligase as a novel predictive biomarker and potential therapeutic target for the treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5245f1077766a809fae0247565289dd67c6fcad" target='_blank'>
              The E3 ubiquitin ligase Herc1 modulates the response to nucleoside analogs in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Maja Jankovic, William Wl Poon, Cristobal Gonzales-Losada, Gabriela Galicia Vazquez, Bahram Sharif-Askari, Yi Ding, Constance Craplet-Desombre, Alexandru Ilie, Jiantao Shi, Yongie Wang, A. Jayavelu, Alexandre Orthwein, F. E. Mercier
          </td>
          <td>2024-08-02</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e7718e7c250fe926c8cdf48ae45f72605c99d8" target='_blank'>
              REXO5 promotes genomic integrity through regulating R-loop using its exonuclease activity.
              </a>
            </td>
          <td>
            Ye Jin Lee, Seo Yun Lee, Soomi Kim, Soomi Kim, Soo Hyeon Lee, Sungho Park, Jae Jin Kim, Dong-Wook Kim, Hongtae Kim
          </td>
          <td>2024-07-30</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Simple Summary Treatment options for lung cancer have greatly expanded in the last decade. Immune checkpoint inhibitors for patients without driver mutations have become the standard of care in widespread disease and, more recently, localized disease. However, resistance can develop, and new treatments are needed. We discussed current combination treatments with radiation and chemotherapy, as well as newer agents. Abstract Lung cancer continues to contribute to the highest percentage of cancer-related deaths worldwide. Advancements in the treatment of non-small cell lung cancer like immune checkpoint inhibitors have dramatically improved survival and long-term disease response, even in curative and perioperative settings. Unfortunately, resistance develops either as an initial response to treatment or more commonly as a progression after the initial response. Several modalities have been utilized to combat this. This review will focus on the various combination treatments with immune checkpoint inhibitors including the addition of chemotherapy, various immunotherapies, radiation, antibody–drug conjugates, bispecific antibodies, neoantigen vaccines, and tumor-infiltrating lymphocytes. We discuss the status of these agents when used in combination with immune checkpoint inhibitors with an emphasis on lung cancer. The early toxicity signals, tolerability, and feasibility of implementation are also reviewed. We conclude with a discussion of the next steps in treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b5223c1539571632a1ba426f2baf523569759ec" target='_blank'>
              Overcoming Resistance to Checkpoint Inhibitors with Combination Strategies in the Treatment of Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Amanda Reyes, R. Muddasani, E. Massarelli
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common pediatric malignancy, with relapse being a major obstacle to successful treatment. Our understanding of the mechanisms driving chemotherapy resistance and ultimately relapse in leukemia remains incomplete. Herein, we investigate the impact of the tumor microenvironment on leukemia cell drug responses using human plasma-like media (HPLM), designed to mimic physiological conditions more accurately ex vivo. We demonstrate that while most chemotherapeutics maintain an efficacy in HPLM comparable to standard tissue culture media, the thiopurines 6-mercaptopurine (6-MP) and 6-thioguanine (6-TG) exhibit significantly reduced potency and efficacy against both B- and T- leukemia cells in HPLM. By merging our understanding of thiopurines’ mechanism of action with the metabolites supplemented in HPLM compared to standard media, we proposed and subsequently validated the hypothesis that hypoxanthine, a purine derivative, is responsible for conferring resistance to the thiopurines. Importantly, the concentration of hypoxanthine required for resistance is comparable to physiological levels found in vivo, supporting clinical relevance. Our findings demonstrate the utility of a more physiologic media in identifying and characterizing mechanisms by which the microenvironment can enable resistance. Understanding such interactions may inform strategies to overcome drug resistance and improve therapeutic outcomes in pediatric leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb07c169f4ccfcca21fef5a4fe805a099e13bb6" target='_blank'>
              Hypoxanthine in the microenvironment can enable thiopurine resistance in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Xiaohong Wang, Jason Ostergaard, Jongseok Kang, Grace Sagong, Rachel Twite, Andrea Vargas-Morales, Peter M. Gordon
          </td>
          <td>2024-07-19</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background The methyltransferase SETD2 was originally identified as a tumor suppressor in clear cell renal cell carcinoma (ccRCC), and ccRCC with SETD2 mutations have increased metastatic potential and poor progression free survival. A known epigenetic regulator, SETD2 interacts with actively transcribing RNA polymerase II (RNAPII) and tri-methylates Histone 3 at Lysine 36 (H3K36me3), a modification that correlates with active gene transcription and is recognized by mRNA methylation and splicing factors. Despite these known roles in mRNA transcription, loss of SETD2 does not alter genic transcription, raising the question of whether or how SETD2 regulates RNA biology. Methods Given that SETD2 influences RNA processing, we sought to identify RNA-protein complexes that are regulated by SETD2. To do this in an unbiased and high-throughput manner, we used orthogonal organic phase separation (OOPS) coupled with mass spectrometry to identify changes in RNA binding proteins and pathways between control and SETD2-knockout cells. Results Corroborating previous findings, we identified that RNA binding proteins governing mRNA splicing and cellular metabolism were dysregulated in SETD2 mutant cells. However, the most robust changes in SETD2 knockout cells occurred in pathways regulated ribosome biogenesis (See Figure, orange highlights), resulting in a significant decrease in binding of ribosomal RNA (rRNA) processing factors to RNA. We confirm these alterations lead to defects in rRNA transcription, processing, and ribosome biogenesis. We identify that loss of SETD2 catalytic activity phenocopies these rRNA processing defects, suggesting H3K36me3 is essential for rRNA processing. Disruptions occur to both the large (60S) and small (40S) ribosome subunits, modifying protein translation. Lastly, we identify synthetic lethality between SETD2 deletion or inhibition and small molecules that interfere with rRNA transcription. Conclusions While the majority of research emphasizes the role SETD2 in the regulation of mRNA, our data identify that SETD2 and its catalytic activity play a critical role in the transcription and processing of rRNA. Importantly, rRNA comprises 80-90% of cellular RNA yet is typically overlooked and understudied. Next-generation RNA sequencing or transcriptional assays investigate mRNA or genic transcription while ignoring rRNA, demonstrating why SETD2-mediated rRNA processing has been unexplored. This may also have important implications for prospective clinical trials that utilized RNA-sequencing to identify optimal therapeutic regimens. Our work also identified that SETD2-mutated cells are more sensitive to compounds that inhibit rRNA transcription, potentially uncovering a new therapeutic vulnerability of ccRCC with mono- and bi-allelic loss of SETD2. Alterations in ribosome biogenesis leads to epithelial-to-mesenchymal transition and metastasis in many tumor types, and our future goal will be to investigate the role of ribosomes and rRNA transcription in transformation and metastasis in ccRCC as well as the molecular mechanisms connecting SETD2 to ribosome fidelity. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30cb284d919d6bd0de496d90d21e1c0abae77b34" target='_blank'>
              51 Epigenetic regulation of ribosomal RNA transcription and processing by the tumor suppressor SETD2
              </a>
            </td>
          <td>
            Logan Vlach, Scott Haake, K. Rathmell, Frank M. Mason
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mutations in the male germline are a driving force behind rare genetic diseases. Driver mutations enjoying a selective advantage expand to mutant clusters within the aged testis and are thus overrepresented in sperm with age. Other kinds of driver mutations, occurring prepubescently, have been the focus of recent attention given their high occurrence rate independent of age. Here, we investigated the ErbB2 gene via error-corrected sequencing and detected a high percentage of missense mutations, including recurrent mutations, which were observed mainly in the tyrosine kinase domain with likely functional consequences, as we verified for a subset via biophysical methods. While these mutations increased with age, we found no evidence that they originated from mutational clusters in the aged testis, and young donors also showed an accumulation of driver mutations, suggesting that mutational enrichment is not exclusive to the sexually mature germline but can occur earlier during germline development, and are likely evenly distributed in the testis and stable in size.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa9f975fbf6b3e62de966f4ae5746c898526f4b3" target='_blank'>
              Mutations in ErbB2 accumulating in the male germline measured by error-corrected sequencing
              </a>
            </td>
          <td>
            Atena Yasari, Monika Heinzl, Theresa Mair, Tina Karimian, Shehab Moukbel Ali Aldawla, Ingrid Hartl, Andrea J. Betancourt, Peter Lanzerstorfer, Irene Tiemann-Boege
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The size and composition of the intracellular DNA precursor pool is integral to the maintenance of genome stability, and this relationship is fundamental to our understanding of cancer. Key aspects of carcinogenesis, including elevated mutation rates and induction of certain types of DNA damage in cancer cells, can be linked to disturbances in deoxynucleoside triphosphate (dNTP) pools. Furthermore, our approaches to treat cancer heavily exploit the metabolic interplay between the DNA and the dNTP pool, with a long-standing example being the use of antimetabolite-based cancer therapies, and this strategy continues to show promise with the development of new targeted therapies. In this Review, we compile the current knowledge on both the causes and consequences of dNTP pool perturbations in cancer cells, together with their impact on genome stability. We outline several outstanding questions remaining in the field, such as the role of dNTP catabolism in genome stability and the consequences of dNTP pool expansion. Importantly, we detail how our mechanistic understanding of these processes can be utilised with the aim of providing better informed treatment options to patients with cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489114e77099687246712fffa583dd2206547a4f" target='_blank'>
              Understanding the interplay between dNTP metabolism and genome stability in cancer.
              </a>
            </td>
          <td>
            Miriam Yagüe-Capilla, S. G. Rudd
          </td>
          <td>2024-08-01</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Human induced pluripotent stem cells (hiPSCs) represent a powerful tool to investigate neuropathological disorders in which the cells of interest are inaccessible, such as in the Charcot–Marie–Tooth disease (CMT), the most common inherited peripheral neuropathy. Developing appropriate cellular models becomes crucial in order to both study the disease’s pathophysiology and test new therapeutic approaches. The generation of hiPS cellular models for disorders caused by a single nucleotide variation has been significantly improved following the development of CRISPR-based editing tools. In this study, we efficiently and quickly generated, by CRISPR editing, the two first hiPSCs cellular models carrying alterations involved in CMT4C, also called AR-CMTde-SH3TC2. This subtype of CMT is associated with alterations in the SH3TC2 gene and represents the most prevalent form of autosomal recessive demyelinating CMT. We aimed to develop models for two different SH3TC2 nonsense variants, c.211C>T, p.Gln71* and the most common AR-CMTde-SH3TC2 alteration, c.2860C>T, p.Arg954*. First, in order to determine the best CRISPR strategy to adopt on hiPSCs, we first tested a variety of sgRNAs combined with a selection of recent base editors using the conveniently cultivable and transfectable HEK-293T cell line. The chosen CRISPR base-editing strategy was then applied to hiPSCs derived from healthy individuals to generate isogenic CMT disease models with up to 93% editing efficiency. For point mutation generation, we first recommend to test your strategies on alternative cell line such as HEK-293T before hiPSCs to evaluate a variety of sgRNA-BE combinations, thus boosting the chance of achieving edited cellular clones with the hard-to-culture and to transfect hiPSCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe956c657405832b052a7c5be2f405359785783c" target='_blank'>
              CRISPR Base Editing to Create Potential Charcot–Marie–Tooth Disease Models with High Editing Efficiency: Human Induced Pluripotent Stem Cell Harboring SH3TC2 Variants
              </a>
            </td>
          <td>
            Camille Loret, Amandine Pauset, Pierre-Antoine Faye, Valérie Prouzet-Mauleon, I. Pyromali, A. Nizou, Federica Miressi, Franck Sturtz, Frédéric Favreau, Béatrice Turcq, A. Lia
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Menin inhibitors that disrupt Menin-MLL interaction hold promise for treating specific acute myeloid leukemia subtypes, including KMT2A rearrangements (KMT2A-r), yet resistance remains a challenge. Here, through systematic chromatin-focused CRISPR screens, along with genetic, epigenetic, and pharmacologic studies in a variety of human and mouse KMT2A-r AML models, we uncover a potential resistance mechanism independent of canonical Menin-MLL targets. We show that a group of non-canonical Menin targets, which are bivalently co-occupied by active Menin and repressive H2AK119ub marks, are typically downregulated following Menin inhibition. The loss of Polycomb Repressive Complex 1.1 (PRC1.1) subunits, such as PCGF1 or BCOR, leads to Menin inhibitor resistance by epigenetic reactivation of these non-canonical targets, including MYC. Genetic and pharmacological inhibition of MYC can resensitize PRC1.1-deficent leukemia cells to Menin inhibition. Moreover, we demonstrate that leukemia cells with the loss of PRC1.1 subunits exhibit reduced monocytic gene signatures and are susceptible to the BCL2 inhibition, and combinational treatment of venetoclax overcomes the resistance to Menin inhibition in PRC1.1-deficient leukemia cells. These findings highlight the important roles of PRC1.1 and its regulated non-canonical Menin targets in modulating Menin inhibitor response and provide potential strategies to treat leukemias with compromised PRC1.1 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fba94eb1d6d5cb6e0a8de6969fbdc894d28222" target='_blank'>
              Epigenetic Regulation of Non-canonical Menin Targets Modulates Menin Inhibitor Response in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Xinyue Zhou, Lixia Zhang, Sajesan Aryal, Virginia Veasey, Amanda Tajik, Cecilia Restelli, Steven Moreira, Pengcheng Zhang, Yanfeng Zhang, Kristin J Hope, Yang Zhou, Changde Cheng, Ravi Bhatia, Rui Lu
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40fcdfbbc2df4270401ef8beed0b78736d658d46" target='_blank'>
              DNA methylation in mammalian development and disease.
              </a>
            </td>
          <td>
            Zachary D. Smith, Sara Hetzel, Alexander Meissner
          </td>
          <td>2024-08-12</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="SF3B1 is the most recurrently mutated RNA splicing gene in cancer; However, the study of its pathogenic role has been hindered by a lack of disease-relevant cell line models. Here, we compared four genome engineering platforms to establish SF3B1 mutant cell lines: CRISPR-Cas9 editing, AAV HDR editing, base editing (ABEmax, ABE8e), and prime editing (PE2, PE3, PE5Max). We showed that prime editing via PE5max achieved the most efficient SF3B1 K700E editing across a wide range of cell lines. We further refined our approach by coupling prime editing with a fluorescent reporter that leverages a SF3B1 mutation-responsive synthetic intron to mark successfully edited cells. By applying this approach, called prime editing coupled intron-assisted selection (PRECIS), we introduced the K700E hotspot mutation into two chronic lymphocytic leukemia (CLL) cell lines, HG-3 and MEC-1. We demonstrated that our PRECIS-engineered cells faithfully recapitulate known mutant SF3B1 phenotypes including altered splicing, copy number variations, and cell growth defect. Moreover, we uncovered that SF3B1 mutation can cause the loss of Y chromosome in CLL. Our results showcase PRECIS as an efficient and generalizable method for engineering genetically faithful SF3B1 mutant models. Our approach provides new insights on the role of SF3B1 mutation in cancer and enables the generation of SF3B1 mutant cell lines in relevant cellular context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca0b3ac1820b19171ca9e6d9370b57efca6b03" target='_blank'>
              Engineering oncogenic hotspot mutations on SF3B1 via CRISPR-directed PRECIS mutagenesis.
              </a>
            </td>
          <td>
            Mikel Fernandez, Lei Yu, Qiong Jia, Xuesong Wang, Kevyn L Hart, Zhenyu Jia, Ren-Jang Lin, Lili Wang
          </td>
          <td>2024-08-28</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aab9291d16699439a2d6b53e165a3605cd9a24fb" target='_blank'>
              Unraveling the spatial organization and development of human thymocytes through integration of spatial transcriptomics and single-cell multi-omics profiling.
              </a>
            </td>
          <td>
            Yanchuan Li, Huamei Li, Cheng Peng, Ge Meng, Yijun Lu, Honglin Liu, Li Cui, Huan Zhou, Zhu Xu, Lingyun Sun, Lihong Liu, Qing Xiong, Beicheng Sun, Shiping Jiao
          </td>
          <td>2024-09-06</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Resistance to standard therapies represents a major challenge in melanoma treatment. This is mainly related to the high heterogeneity of this malignancy due to the presence of different types of cells in the tumor mass. Cancer stem cells (CSCs) are a small subset of cells endowed with self-renewal and tumorigenic capacity and the ability to escape anticancer therapies. Melanoma CSCs can be identified by the expression of biomarkers and intracellular pathways deeply involved in their malignant phenotype. Dissecting the molecular mechanisms underlying CSC biological functions might pave the way for novel therapeutic approaches. Herein, we highlight the intricate bidirectional communication between melanoma CSCs and their surrounding cells in the tumor microenvironment and discuss how this interplay might impact the processes of drug resistance and tumor relapse. We also address the potential benefits of CSC-targeted strategies, in combination with conventional drugs, for improving melanoma treatment strategies. Abstract Cutaneous melanoma still represents a significant health burden worldwide, being responsible for the majority of skin cancer deaths. Key advances in therapeutic strategies have significantly improved patient outcomes; however, most patients experience drug resistance and tumor relapse. Cancer stem cells (CSCs) are a small subpopulation of cells in different tumors, including melanoma, endowed with distinctive capacities of self-renewal and differentiation into bulk tumor cells. Melanoma CSCs are characterized by the expression of specific biomarkers and intracellular pathways; moreover, they play a pivotal role in tumor onset, progression and drug resistance. In recent years, great efforts have been made to dissect the molecular mechanisms underlying the protumor activities of melanoma CSCs to provide the basis for novel CSC-targeted therapies. Herein, we highlight the intricate crosstalk between melanoma CSCs and bystander cells in the tumor microenvironment (TME), including immune cells, endothelial cells and cancer-associated fibroblasts (CAFs), and its role in melanoma progression. Specifically, we discuss the peculiar capacities of melanoma CSCs to escape the host immune surveillance, to recruit immunosuppressive cells and to educate immune cells toward an immunosuppressive and protumor phenotype. We also address currently investigated CSC-targeted strategies that could pave the way for new promising therapeutic approaches for melanoma care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7340de903da58d80075cbbac264d846e831d74b" target='_blank'>
              Unveiling the Dynamic Interplay between Cancer Stem Cells and the Tumor Microenvironment in Melanoma: Implications for Novel Therapeutic Strategies
              </a>
            </td>
          <td>
            Patrizia Limonta, Raffaella Chiaramonte, Lavinia Casati
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a5fb073ed81392e0d208c32c77f69b33b47ec" target='_blank'>
              The contributions of cytogenetics, genetics, and epigenetics to the stability of plants polyploidy
              </a>
            </td>
          <td>
            Arrashid Harun, Zhongming Fang, Chunli Chen
          </td>
          <td>2024-07-30</td>
          <td>Discover Plants</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Rhabdoid tumors (RTs) are rare, highly aggressive pediatric malignancies which occur in kidney, soft-parts and brain. They are characterized by a complete inactivation of the SMARCB1 tumor suppressor gene encoding a core subunit of the chromatin remodeling SWI/SNF complex. Prognosis for children with RTs is poor as, in many instances, these tumors are resistant to conventional type chemotherapy. Pharmacological inhibition of EZH2 is a promising strategy to treat some tumors with loss of function in SMARCB1. However, the clinical response rate remains low, and both primary and secondary resistance have been reported. Understanding the mechanisms of resistance to EZH2 inhibition may allow new therapeutic hypothesis. For this purpose, we realized a genome-wide CRISPR-Cas9 knockout screening on two rhabdoid cell lines treated with EZH2 inhibitor. We identified TP53 loss as the sole consistent gene knock-out able to confer some resistance to EZH2 inhibition in both cell lines. Conversely, we demonstrated that MDM2/MDM4 inhibition, in a TP53-dependant manner, strongly synergized with EZH2 inhibition to control cell lines viability in vitro. We further treated 4 rhabdoid patient-derived xenografts with UNC1999, Idasanutlin and the combination of both and observed a potent tumor growth control upon combined EZH2 and MDM2/MDM4 inhibitions. To conclude, our results strongly encourage to assess the actual efficacy of combined MDM2/MDM4 and EZH2 inhibitors in the treatment of patients with SMARCB1-deficient rhabdoid tumors.
 Citation Format: Céline Chauvin, Tiphaine Hery, Rachida Bouarich, Fariba Nemati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, Sergio Roman-Roman, Didier Decaudin, Raphaël Margueron, Franck Bourdeaut. Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A051.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85bf34da504cd580597c4be8039c1663b824b215" target='_blank'>
              Abstract A051 Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors
              </a>
            </td>
          <td>
            C. Chauvin, Tiphaine Héry, Rachida Bouarich, F. Némati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, , Sakina Zaidi, Didier Surdez, S. Roman-Roman, Didier Decaudin, Raphaël Margueron, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5096f65eca5658128881222e362e03cc7a2beb2" target='_blank'>
              H3K27ac acts as a molecular switch for doxorubicin-induced activation of cardiotoxic genes
              </a>
            </td>
          <td>
            Yu Hong, Xinlan Li, Jia Li, Qiuyi He, Manbing Huang, Yubo Tang, Xiao Chen, Jie Chen, Ke-Jing Tang, Chao Wei
          </td>
          <td>2024-07-16</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Desmoplastic small round cell tumor (DSRCT) is a highly aggressive cancer predominantly occurring in male adolescents and young adults. The lack of a comprehensive understanding on the biology of the disease is paralleled by its dismal survival rates (5–20%). To overcome this challenge, we first identified and prioritized urgently needed resources for clinicians and researchers. Thus, we established genome-wide single-cell RNA-sequencing and bulk proteomic data of in vitro and in vivo-generated knockdown models of the pathognomonic DSRCT fusion oncoprotein (EWSR1::WT1) and combined them with an original systems-biology-based pipeline including patient data and the largest histology collection of DSRCTs and morphological mimics available to date. These novel tools were enriched with curated public datasets including patient- and cell line-derived ChIP-seq, bulk and single-cell RNA-seq studies resulting in a multi-model and multi-omic toolbox for discovery analyses. As a proof of concept, our approach revealed the alpha-2/delta subunit of the voltage-dependent calcium channel complex, CACNA2D2, as a highly overexpressed, super-enhancer driven, direct target of EWSR1::WT1. Single-cell and bulk-level analyses of patient samples and xenografted cell lines highlighted CACNA2D2 as a critical component of our newly established EWSR1::WT1 oncogenic signature, that can be employed to robustly identify DSRCT in reference sets. Finally, we show that CACNA2D2 is a highly sensitive and specific single biomarker for fast, simple, and cost-efficient diagnosis of DSRCT. Collectively, we establish a large-scale multi-omics dataset for this devastating disease and provide a blueprint of how such toolbox can be used to identify new and clinically relevant diagnostic markers, which may significantly reduce misdiagnoses, and thus improve patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1adb9c7dca4062ab1492b77bcd56eb7a98c207ea" target='_blank'>
              Super-enhancer-driven CACNA2D2 is an EWSR1::WT1 signature gene encoding a diagnostic marker for desmoplastic small round cell tumor (DSRCT)
              </a>
            </td>
          <td>
            Florian H. Geyer, Alina Ritter, Seneca S Kinn-Gurzo, Tobias Faehling, Jing Li, Armin Jarosch, Carine Ngo, Endrit Vinca, Karim Aljakouch, Azhar Orynbek, S. Ohmura, Thomas Kirchner, R. Imle, L. Romero-Pérez, Stefanie Bertram, Enrique de Álava, Sophie Postel-Vilnay, A. Banito, M. Sill, Y. Versleijen-Jonkers, Benjamin F B Mayer, Martin Ebinger, M. Sparber-Sauer, Sabine Stegmaier, Daniel Baumhoer, W. Hartmann, Jeroen Krijgsveld, David Horst, Olivier Delattre, P. Grohar, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2024-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="ABSTRACT Changes in gene dosage can have tremendous evolutionary potential (e.g. whole-genome duplications), but without compensatory mechanisms, they can also lead to gene dysregulation and pathologies. Sex chromosomes are a paradigmatic example of naturally occurring gene dosage differences and their compensation. In species with chromosome-based sex determination, individuals within the same population necessarily show ‘natural’ differences in gene dosage for the sex chromosomes. In this Review, we focus on the mammalian X chromosome and discuss recent new insights into the dosage-compensation mechanisms that evolved along with the emergence of sex chromosomes, namely X-inactivation and X-upregulation. We also discuss the evolution of the genetic loci and molecular players involved, as well as the regulatory diversity and potentially different requirements for dosage compensation across mammalian species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016ee497a5dab2410dc213e6a7a7d311ff9fabfc" target='_blank'>
              Compensation of gene dosage on the mammalian X
              </a>
            </td>
          <td>
            Daniela Cecalev, Beatriz Viçoso, Rafael Galupa
          </td>
          <td>2024-08-01</td>
          <td>Development (Cambridge, England)</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Peptide-based antibiotics (PBAs), including antimicrobial peptides (AMPs) and their synthetic mimics, have received significant interest due to their diverse and unique bioactivities. The integration of high-throughput sequencing and bioinformatics tools has dramatically enhanced the discovery of enzymes, allowing researchers to identify specific genes and metabolic pathways responsible for producing novel PBAs more precisely. Cell-free systems (CFSs) that allow precise control over transcription and translation in vitro are being adapted, which accelerate the identification, characterization, selection, and production of novel PBAs. Furthermore, these platforms offer an ideal solution for overcoming the limitations of small-molecule antibiotics, which often lack efficacy against a broad spectrum of pathogens and contribute to the development of antibiotic resistance. In this review, we highlight recent examples of how CFSs streamline these processes while expanding our ability to access new antimicrobial agents that are effective against antibiotic-resistant infections.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69136d617718b9427194d4834266868834ff8487" target='_blank'>
              Cell-Free Systems: Ideal Platforms for Accelerating the Discovery and Production of Peptide-Based Antibiotics
              </a>
            </td>
          <td>
            Hyeongwoo Park, Haneul Jin, Dayeong Kim, Joongoo Lee
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b903232278019fc20d595adb95c47c4a0e3a7532" target='_blank'>
              Deletion of IRC19 Causes Defects in DNA Double-Strand Break Repair Pathways in Saccharomyces cerevisiae.
              </a>
            </td>
          <td>
            Ju-Hee Choi, Oyungoo Bayarmagnai, Sung-Ho Bae
          </td>
          <td>2024-07-12</td>
          <td>Journal of microbiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Background: Transposable elements (TEs) are dynamic repetitive regions which generate mutations and structural variants. TP53 plays a crucial role in suppressing TE movement to maintain genomic stability. The relationship between TP53 and TEs has been extensively studied in tumours, but not the germline. Individuals with germline TP53 pathogenic variants have Li-Fraumeni Syndrome (LFS), a cancer predisposition syndrome with a high lifetime risk of cancer in various tissues. This study aims to characterize the TE landscape in individuals with Li-Fraumeni Syndrome and determine how this contributes to their increased cancer risk. Methods: MELT and xTea were used to identify TEs in children with (n=48) or without (n=198) a germline TP53 variant. TE calls were merged with SURVIVOR and variants were annotated with AnnotSV. To assess the influence of TP53 on TE location, we quantified TEs across genomic windows and identified the top 100 significantly different regions with the Mann-Whitney U test, adjusting for multiple comparisons. We used these regions to develop a gradient-boosted tree model with 5-fold cross-validation to predict TP53 status. Results: Individuals with germline TP53 variants harboured significantly fewer ALU and LINE1 elements in their germline genome compared to the control dataset (p<0.001). The phenomenon was consistent across chromosomes and significant in chromosomes 5 and 11 (FDR<0.05). A gradient-boosted tree model was able to differentiate patients with and without a germline TP53 variant with an AUPRC of 0.77 on an unseen test set. Conclusion: Germline TP53 variants may effect the frequency and location of germline TE insertions, which may influence nearby genomic variations and lead to cancer development. Analyzing TEs in individuals with LFS will enhance our understanding of accelerated cancer development in these patients, informing future research for diagnostic and therapeutic approaches.
 Citation Format: Brianne Laverty, Shilpa Yadahalli, Vallijah Subasri, David Malkin. The role of TP53 on transposable elements in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4995c29bd2050c705f9a5b8da916aacb0aa2601c" target='_blank'>
              Abstract A010 The role of TP53 on transposable elements in pediatric cancer
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Vallijah Subasri, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Salih Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Long terminal repeat retroelements (LTR-REs) have profound effects on DNA methylation and gene regulation. Despite the vast abundance of LTR-REs in the genome of Moso bamboo (Phyllostachys edulis), an industrial crop in underdeveloped countries, their precise implication of the LTR-RE mobility in stress response and development remains unknown. We investigated the RNA and DNA products of LTR-REs in Moso bamboo under various developmental stages and stressful conditions. Surprisingly, our analyses identified thousands of active LTR-REs, particularly those located near genes involved in stress response and developmental regulation. These genes adjacent to active LTR-REs exhibited an increased expression under stress and are associated with reduced DNA methylation that is likely affected by the induced LTR-REs. Moreover, the analyses of simultaneous mapping of insertions and DNA methylation showed that the LTR-REs effectively alter the epigenetic status of the genomic regions where they inserted, and concomitantly their transcriptional competence which might impact the stress resilience and growth of the host. Our work unveils the unusually strong LTR-RE mobility in Moso bamboo and its close association with (epi)genetic changes, which supports the co-evolution of the parasitic DNAs and host genome in attaining stress tolerance and developmental robustness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8156b2311d35a48deb53050724e71cd41e2e56c" target='_blank'>
              Genetic and epigenetic reprogramming in response to internal and external cues by induced transposon mobilization in Moso bamboo.
              </a>
            </td>
          <td>
            L. Zou, Bailiang Zhu, Yaxin Chen, Yaping Lu, Muthusamy Ramkrishnan, Chao Xu, Xiaohong Zhou, Yiqian Ding, Jungnam Cho, Mingbing Zhou
          </td>
          <td>2024-09-05</td>
          <td>The New phytologist</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer is a broad category of diseases characterized through the unchecked proliferation and dissemination of atypical cells. It is among the primary causes of morbidity and mortality globally. The multistage genesis of cancer, which begins with genetic abnormalities that cause normal cells to become malignant, is what gives cancer its complexity. There are several steps involved in the genesis of cancer, including six key cancer distinguishing features known to influence malignant change has been identified. Anticancer drug development is a laborious process that includes numerous in vitro, in vivo, and clinical trials. In vitro assays provide a foundation for cancer medication development techniques. Numerous in vitro procedures and tests have been developed to analyze every defining aspect of cancer; the choice of a specific in vitro technique or assay is largely based on the research question(s) under investigation. Currently, oncology researchers are attempting to create cancer nanomedicines that are both safe and effective. While nanoparticles have opened up new therapeutic and diagnostic avenues, stem cell treatment has demonstrated potential usefulness in renewing and repairing defective or damaged tissues by tackling primary and metastatic cancer sites. With minimal harm to healthy cells, targeted therapy has the potential to stop the development and propagation of specific cancer cells. In place of open surgery, ablation therapy has become a popular minimally invasive method for destroying or freezing tumors. Naturally occurring antioxidants have demonstrated the ability to find free radicals and counteract their harmful effects, potentially treating or preventing cancer. A number of novel technologies have previously received authorization, and some are presently the subject of clinical trials. This review article's objective is to provide an extensive overview of the state of our knowledge on cancer, covering its causes, kinds, diagnosis, therapy, in vitro assays to screen cancer and most recent scientific developments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1f94580685a6bea79e8df46a55cf04537a375f9" target='_blank'>
              A Comprehensive Review of In vitro Testing and Emerging Strategies Employed in Anticancer Drug Discovery Therapy
              </a>
            </td>
          <td>
            N. Reddy V, M. G. Raju, Ashritha V, Sowmika Kv, Anirudh M, Avinash A, Ashritha B
          </td>
          <td>2024-07-26</td>
          <td>International Journal of Innovative Science and Research Technology (IJISRT)</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e16f71ff68478a0f30f71105d86b66a3bf8d6b91" target='_blank'>
              Single-Cell Sequencing: High-Resolution Analysis of Cellular Heterogeneity in Autoimmune Diseases.
              </a>
            </td>
          <td>
            Xuening Tang, Yudi Zhang, Hao Zhang, Nan Zhang, Ziyu Dai, Q. Cheng, Yongzhen Li
          </td>
          <td>2024-08-26</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43914f4be0505268e85d82eb83b340875b29e63e" target='_blank'>
              Integrated systematic functional screen and fine-mapping decipher the role and genetic regulation of RPS19 in colorectal cancer development.
              </a>
            </td>
          <td>
            Can Chen, Wenzhuo Wang, Caibo Ning, Zequn Lu, Ming Zhang, Ying Zhu, Jianbo Tian, Haijie Li, Yue Ge, Beifang Yang, Xiaoping Miao
          </td>
          <td>2024-07-16</td>
          <td>Archives of toxicology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="ABSTRACT Natural killer (NK) cells play a crucial role in antitumoral and antiviral responses. Yet, cancer cells can alter themselves or the microenvironment through the secretion of cytokines or other factors, hindering NK cell activation and promoting a less cytotoxic phenotype. These resistance mechanisms, often referred to as the “hallmarks of cancer” are significantly influenced by the activation of oncogenes, impacting most, if not all, of the described hallmarks. Along with oncogenes, other types of genes, the tumor suppressor genes are frequently mutated or modified during cancer. Traditionally, these genes have been associated with uncontrollable tumor growth and apoptosis resistance. Recent evidence suggests oncogenic mutations extend beyond modulating cell death/proliferation programs, influencing cancer immunosurveillance. While T cells have been more studied, the results obtained highlight NK cells as emerging key protagonists for enhancing tumor cell elimination by modulating oncogenic activity. A few recent studies highlight the crucial role of oncogenic mutations in NK cell-mediated cancer recognition, impacting angiogenesis, stress ligands, and signaling balance within the tumor microenvironment. This review will critically examine recent discoveries correlating oncogenic mutations to NK cell-mediated cancer immunosurveillance, a relatively underexplored area, particularly in the era dominated by immune checkpoint inhibitors and CAR-T cells. Building on these insights, we will explore opportunities to improve NK cell-based immunotherapies, which are increasingly recognized as promising alternatives for treating low-antigenic tumors, offering significant advantages in terms of safety and manufacturing suitability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfefe1207e6a05f83b08c29611ae01cb2a8d9e94" target='_blank'>
              In-depth analysis of the interplay between oncogenic mutations and NK cell-mediated cancer surveillance in solid tumors
              </a>
            </td>
          <td>
            C. Pesini, Laura Artal, Jorge Paúl Bernal, Diego Sánchez Martinez, Julián Pardo, A. Ramírez-Labrada
          </td>
          <td>2024-07-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Despite the availability of life-extending treatments for B cell leukemias and lymphomas, many of these cancers remain incurable. Thus, the development of new molecular targets and therapeutics is needed to expand treatment options. To identify new molecular targets, we used a forward genetic screen in mice to identify genes required for development or survival of lymphocytes. Here, we describe Zfp574, an essential gene encoding a zinc finger protein necessary for normal and malignant lymphocyte survival. We show that ZFP574 interacts with zinc finger protein THAP12 and promotes the G1-to-S-phase transition during cell cycle progression. Mutation of ZFP574 impairs nuclear localization of the ZFP574-THAP12 complex. ZFP574 or THAP12 deficiency results in cell cycle arrest and impaired lymphoproliferation. Germline mutation, acute gene deletion, or targeted degradation of ZFP574 suppressed Myc-driven B cell leukemia in mice, but normal B cells were largely spared, permitting long-term survival, whereas complete lethality was observed in control animals. Our findings support the identification of drugs targeting ZFP574-THAP12 as a unique strategy to treat B cell malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0480486c2217b6dc5e3cddbe10c5e1e4dd9761e0" target='_blank'>
              Disruption of the ZFP574-THAP12 complex suppresses B cell malignancies in mice.
              </a>
            </td>
          <td>
            Xue Zhong, James J Moresco, J. SoRelle, Ran Song, Yiao Jiang, Mylinh T Nguyen, Jianhui Wang, C. Bu, E. Moresco, Bruce Beutler, J. Choi
          </td>
          <td>2024-07-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The nature of microRNA (miRNA) dysfunction in carcinogenesis remains controversial because of the complex connection between miRNA structural diversity and biological processes. Here, we found that oncofetal IGF2BP3 regulates the selective production of a subset of 3'-isoforms (3'-isomiRs), including miR-21-5p and Let-7 family, which induces significant changes in their cellular seed occupancy and structural components, establishing a cancer-specific gene expression profile. The D-score, reflecting dominant production of a representative miR-21-5p+C (a 3'-isomiR), discriminated between clinical early-stage lung adenocarcinoma (LUAD) cases with low and high recurrence risks, and was associated with molecular features of cell cycle progression, epithelial-mesenchymal transition pressure, and immune evasion. We found that IGF2BP3 controls the production of miR-21-5p+C by directing the nuclear Drosha complex to select the cleavage site. IGF2BP3 was also involved in the production of 3'-isomiRs of miR-425-5p and miR-454-3p. IGF2BP3-regulated these three miRNAs are suggested to be associated with the regulation of p53, TGF-β, and TNF pathways in LUAD. Knockdown of IGF2BP3 also induced a selective upregulation of Let-7 3'-isomiRs, leading to increased cellular Let-7 seed occupancy and broad repression of its target genes encoding cell cycle regulators. The D-score is an index that reflects this cellular situation. Our results suggest that the aberrant regulation of miRNA structural diversity is a critical component for controlling cellular networks, thus supporting the establishment of a malignant gene expression profile in early stage LUAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/878f17dcf28ac138d1fb1c11fc04b13b2ee0cbcc" target='_blank'>
              Oncofetal IGF2BP3-mediated control of microRNA structural diversity in the malignancy of early-stage lung adenocarcinoma.
              </a>
            </td>
          <td>
            Yuko Fujiwara, Ryou-u Takahashi, M. Saito, Michinobu Umakoshi, Yoko Shimada, Kei Koyama, Yasushi Yatabe, Shun-ichi Watanabe, Souichi Koyota, Yoshihiro Minamiya, H. Tahara, Koji Kono, K. Shiraishi, Takashi Kohno, Akiteru Goto, Naoto Tsuchiya
          </td>
          <td>2024-08-28</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer.
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Gene expression regulation is one of the most fundamental cellular processes, enabling the activation of a gene to produce either the translatable protein-coding transcript (mRNA) or a functional non-coding RNA with gene regulatory functions, ultimately determining cell identity and function. Although gene expression regulation can occur at transcriptional, translational, and post-translational levels, transcription initiation is the first and the most important step in gene expression, facilitating the transfer of biological information from DNA to protein. Enhancers and super-enhancers are among the master regulators of tissue- and cell-specific transcription regulation involved in cell differentiation and tumor formation. Despite four decades passing since the first discovery of enhancers in eukaryotes and extensive efforts undertaken to identify enhancers on a genomic scale during the last decade, the discovery of enhancers still faces certain limitations and needs further investigation. The perturbation of enhancer function due to genetic or epigenetic changes is closely linked to a range of human disorders, including the development and progression of cancers. Thus, the detection of early cancer-related enhancer activity and the subsequent normalization of expression abnormalities using enhancer-targeting CRISPR epigenetic editing, as well as enhancer-targeting pharmaceuticals, are regarded as groundbreaking therapeutic tactics in preclinical stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782ddfe76cab2a16620844e3e84b112d8c90071b" target='_blank'>
              Enhancers and super-enhancers as master regulators in cancer
              </a>
            </td>
          <td>
            P. Sarvari, Pourya Sarvari, Ivonne Ramírez-Díaz, Karla Rubio
          </td>
          <td>2024-07-24</td>
          <td>INNOSC Theranostics and Pharmacological Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Each piece of cell-free DNA (cfDNA) has a length determined by the exact metabolic conditions in the cell it belonged to at the time of cell death. The changes in cellular regulation leading to a variety of patterns, which are based on the different number of fragments with lengths up to several hundred base pairs at each of the almost three billion genomic positions, allow for the detection of disease and also the precise identification of the tissue of their origin. A Kullback-Leibler (KL) divergence computation identifies different fragment lengths and areas of the human genome, depending on the stage, for which disease samples, starting from pre-clinical disease stages, diverge from healthy donor samples. We provide examples of genes related to colorectal cancer (CRC), which our algorithm detected to belong to divergent genomic bins. The staging of CRC can be viewed as a Markov Chain and that provides a framework for studying disease progression and the types of epigenetic changes occurring longitudinally at each stage, which might aid the correct classification of a new hospital sample. In a new look to treat such data as grayscale value images, pattern recognition using artificial intelligence (AI) could be one approach to classification. In CRC, Stage I disease does not, for the most part, shed any tumor circulation, making detection difficult for established machine learning (ML) methods. This leads to the deduction that early detection, where we can only rely on changes in the metabolic patterns, can be accomplished when the information is considered in its entirety, for example by applying computer vision methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769c868029bf18fe644fcae9a2141c21f2b5a763" target='_blank'>
              Channel Capacity of Genome-Wide Cell-Free DNA Fragment Length Distribution in Colorectal Cancer
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-07-18</td>
          <td>None</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The seven human APOBEC3 (hA3) genes encode polynucleotide cytidine deaminases that play vital roles in restricting replication of viruses and retrotransposons. However, off-target A3 deamination of the cellular genome is a major source of somatic mutations in human cancer. The ability to study A3 biology in vivo is hindered by the fact that the solitary murine Apobec3 gene (mA3) encodes a cytoplasmic enzyme, with no apparent mutagenic activity. Transgenic expression of individual hA3 genes in mice has helped to confirm their oncogenic potential but important questions including which hA3 genes are active in different tissue contexts and how they function in concert when under control of their cognate promoters cannot be addressed using these models. Here we describe humanization of the mouse mA3 locus by integration of a modified BAC clone encompassing the entire 7-gene hA3 locus from human chromosome 22 replacing mA3 on mouse chromosome 15. APOBEC3 mice are viable and fertile and hA3 gene expression in cells and tissues correlates strongly with expression in corresponding human cells and tissues, indicating human-like regulation of hA3 gene expression in the mice. Splenocytes from this line display a functional human A3 response to Friend Murine Leukaemia Virus (F-MLV) infection. We propose that the Hs-APOBEC3 mouse will uniquely model the function of the complete hA3 locus in a living organism and that it will serve as a useful background upon which to model human cancer, as well as assisting drug discovery efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8034f8597ff29dbab41cdb895f637bb191838ad2" target='_blank'>
              Human-like APOBEC3 gene expression and anti-viral responses following replacement of mouse Apobec3 with the 7-gene human APOBEC3 locus
              </a>
            </td>
          <td>
            Nerissa K. Kirkwood, H. Prosser, Melvyn W Yap, Jane Gibson, Ross Cook, Ian Reddin, Ane Stranger, Emma Howes, Nur Zainal, M. Periyasamy, S. Conticello, Gareth J. Thomas, James Scott, Kate N. Bishop, Simak Ali, Allan Bradley, Tim R. Fenton
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The global race against antimicrobial resistance requires novel antimicrobials that are not only effective in killing specific bacteria, but also minimize the emergence of new resistances. Recently, CRISPR/Cas-based antimicrobials were proposed to address killing specificity with encouraging results. However, the emergence of target sequence mutations triggered by Cas-cleavage was identified as an escape strategy, posing the risk of generating new antibiotic-resistance gene (ARG) variants. Here, we evaluated an antibiotic re-sensitization strategy based on CRISPR interference (CRISPRi), which inhibits gene expression without damaging target DNA. The resistance to four antibiotics, including last resort drugs, was significantly reduced by individual and multi-gene targeting of ARGs in low- to high-copy numbers in recombinant E. coli. Escaper analysis confirmed the absence of mutations in target sequence, corroborating the harmless role of CRISPRi in the selection of new resistances. E. coli clinical isolates carrying ARGs of severe clinical concern were then used to test the robustness of CRISPRi under different growth conditions. Meropenem, colistin and cefotaxime susceptibility was successfully increased in terms of MIC (up to >4-fold) and growth delay (up to 11-hours) in a medium-dependent fashion. To our knowledge, this is the first demonstration of CRISPRi-mediated re-sensitization to last-resort drugs in clinical isolates. This study laid the foundations for further leveraging CRISPRi as antimicrobial agent or research tool to selectively repress ARGs and investigate resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2989ebf7e464eb8144da23fd1f19b024a23610d8" target='_blank'>
              Harnessing CRISPR interference to re-sensitize laboratory strains and clinical isolates to last resort antibiotics
              </a>
            </td>
          <td>
            Angelica Frusteri Chiacchiera, Michela Casanova, Massimo Bellato, Aurora Piazza, Roberta Migliavacca, Gregory Batt, P. Magni, L. Pasotti
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Colorectal cancer (CRC) ranks third in terms of cancer incidence worldwide and is responsible for 8% of all deaths globally. Approximately 10% of CRC cases are caused by inherited pathogenic mutations in driver genes involved in pathways that are crucial for CRC tumorigenesis and progression. These hereditary mutations significantly increase the risk of initial benign polyps or adenomas developing into cancer. In recent years, the rapid and accurate sequencing of CRC-specific multigene panels by next-generation sequencing (NGS) technologies has enabled the identification of several recurrent pathogenic variants with established functional consequences. In parallel, rare genetic variants that are not characterized and are, therefore, called variants of uncertain significance (VUSs) have also been detected. The classification of VUSs is a challenging task because each amino acid has specific biochemical properties and uniquely contributes to the structural stability and functional activity of proteins. In this scenario, the ability to computationally predict the effect of a VUS is crucial. In particular, in silico prediction methods can provide useful insights to assess the potential impact of a VUS and support additional clinical evaluation. This approach can further benefit from recent advances in artificial intelligence-based technologies. In this review, we describe the main in silico prediction tools that can be used to evaluate the structural and functional impact of VUSs and provide examples of their application in the analysis of gene variants involved in hereditary CRC syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c45c256e9f192af359d986d6c8e8e5f1f020547" target='_blank'>
              In Silico Deciphering of the Potential Impact of Variants of Uncertain Significance in Hereditary Colorectal Cancer Syndromes
              </a>
            </td>
          <td>
            C. Fasano, Martina Lepore Signorile, Katia De Marco, G. Forte, Vittoria Disciglio, P. Sanese, V. Grossi, Cristiano Simone
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Persister cells (PS) selected for anticancer therapy have been recognized as a significant contributor to the development of treatment-resistant malignancies. It is found that imposing glutamine restriction induces the generation of PS, which paradoxically bestows heightened resistance to glutamine restriction treatment by activating the integrated stress response and initiating the general control nonderepressible 2-activating transcription factor 4-alanine, serine, cysteine-preferring transporter 2 (GCN2-ATF4-ASCT2) axis. Central to this phenomenon is the stress-induced ATF4 translational reprogramming. Unfortunately, directly targeting ATF4 protein has proven to be a formidable challenge because of its flat surface. Nonetheless, a G-quadruplex structure located within the promoter region of ATF4 (ATF4-G4) is uncovered and resolved, which functions as a transcriptional regulator and can be targeted by small molecules. The investigation identifies the natural compound coptisine (COP) as a potent binder that interacts with and stabilizes ATF4-G4. For the first time, the high-resolution structure of the COP-ATF4-G4 complex is determined. The formation of this stable complex disrupts the interaction between transcription factor AP-2 alpha (TFAP2A) and ATF4-G4, resulting in a substantial reduction in intracellular ATF4 levels and the eventual death of cancer cells. These seminal findings underscore the potential of targeting the ATF4-G4 structure to yield significant therapeutic advantages within the realm of persister cancer cells induced by glutamine-restricted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e9a3aeeaaba15f0673e425713df633d5da2043" target='_blank'>
              Overcoming Cancer Persister Cells by Stabilizing the ATF4 Promoter G-quadruplex.
              </a>
            </td>
          <td>
            Chengmei Xiao, Yipu Li, Yushuang Liu, Ruifang Dong, Xiaoyu He, Qing Lin, Xin Zang, Kaibo Wang, Yuanzheng Xia, Lingyi Kong
          </td>
          <td>2024-07-12</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, B. van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba856bced2e48ff7a94743b3236b182aa83cc934" target='_blank'>
              Epigenome editing technologies for discovery and medicine.
              </a>
            </td>
          <td>
            Sean R. McCutcheon, Dahlia Rohm, Nahid Iglesias, C. Gersbach
          </td>
          <td>2024-07-29</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="RNA and protein expressed from the same gene can have diverse isoforms due to various post-transcriptional and post-translational modifications. For the vast majority of alternative isoforms, It is unknown whether they are adaptive or simply biological noise. As we cannot experimentally probe the function of each isoform, we can ask whether the distribution of isoforms across genes and across species is consistent with expectations from different evolutionary processes. However, there is currently no theoretical framework that can generate such predictions. To address this, we developed a mathematical model where isoform abundances are determined collectively by cis-acting loci, trans-acting factors, gene expression levels, and isoform decay rates to predict isoform abundance distributions across species and genes in the face of mutation, genetic drift, and selection. We found that factors beyond selection, such as effective population size and the number of cis-acting loci, significantly influence evolutionary outcomes. Notably, suboptimal phenotypes are more likely to evolve when the population is small and/or when the number of cis-loci is large. We also explored scenarios where modification processes have both beneficial and detrimental effects, revealing a non-monotonic relationship between effective population size and optimization, demonstrating how opposing selection pressures on cis- and trans-acting loci can constrain the optimization of gene product diversity. As a demonstration of the power of our theory, we compared the expected distribution of A-to-I RNA editing levels in coleoids and found this to be largely consistent with non-adaptive explanations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f7debd21002031945dc5d351bbceebd6fc011cff" target='_blank'>
              Genetic and selective constraints on the optimization of gene product diversity
              </a>
            </td>
          <td>
            Daohan Jiang, Nevraj S. Kejiou, Yi Qiu, Alexander F. Palazzo, Matt Pennell
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) are the remnants of retroviral germline infections and are highly abundant in the genomes of vertebrates. At one time considered to be nothing more than inert ‘junk’ within genomes, ERVs have been tolerated within host genomes over vast timescales, and their study continues to reveal complex co-evolutionary histories within their respective host species. For example, multiple instances have been characterized of ERVs having been ‘borrowed’ for normal physiology, from single copies to ones involved in various regulatory networks such as innate immunity and during early development. Within the cell, the accessibility of ERVs is normally tightly controlled by epigenetic mechanisms such as DNA methylation or histone modifications. However, these silencing mechanisms of ERVs are reversible, and epigenetic alterations to the chromatin landscape can thus lead to their aberrant expression, as is observed in abnormal cellular environments such as in tumors. In this review, we focus on ERV transcriptional control and draw parallels and distinctions concerning the loss of regulation in disease, as well as their precise regulation in early development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88fcb4c839dae363d4185cc295fc1a0003652e42" target='_blank'>
              Transcription of Endogenous Retroviruses: Broad and Precise Mechanisms of Control
              </a>
            </td>
          <td>
            Abigail S. Jarosz, Julia V. Halo
          </td>
          <td>2024-08-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb85026de0a1065b160b9eed08ef28f424a5cc3" target='_blank'>
              The AKT inhibitor AZD5363 elicits synthetic lethality in ARID1A-deficient gastric cancer cells via induction of pyroptosis.
              </a>
            </td>
          <td>
            Menghan Fang, Youfen Lin, Chaorong Xue, Kaiqin Sheng, Zegeng Guo, Yuting Han, Hanbin Lin, Yuecheng Wu, Yuchao Sang, Xintan Chen, Stephen B. Howell, Xu Lin, Xinjian Lin
          </td>
          <td>2024-07-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer continues to present a substantial global health challenge, with its incidence and mortality rates persistently reflecting its significant impact. The emergence of precision oncology has provided a breakthrough in targeting oncogenic drivers previously deemed “undruggable” by conventional therapeutics and by limiting off-target cytotoxicity. Two groundbreaking technologies that have revolutionized the field of precision oncology are primarily CRISPR-Cas9 gene editing and more recently PROTAC (PROteolysis TArgeting Chimeras) targeted protein degradation technology. CRISPR-Cas9, in particular, has gained widespread recognition and acclaim due to its remarkable ability to modify DNA sequences precisely. Rather than editing the genetic code, PROTACs harness the ubiquitin proteasome degradation machinery to degrade proteins of interest selectively. Even though CRISPR-Cas9 and PROTAC technologies operate on different principles, they share a common goal of advancing precision oncology whereby both approaches have demonstrated remarkable potential in preclinical and promising data in clinical trials. CRISPR-Cas9 has demonstrated its clinical potential in this field due to its ability to modify genes directly and indirectly in a precise, efficient, reversible, adaptable, and tissue-specific manner, and its potential as a diagnostic tool. On the other hand, the ability to administer in low doses orally, broad targeting, tissue specificity, and controllability have reinforced the clinical potential of PROTAC. Thus, in the field of precision oncology, gene editing using CRISPR technology has revolutionized targeted interventions, while the emergence of PROTACs has further expanded the therapeutic landscape by enabling selective protein degradation. Rather than viewing them as mutually exclusive or competing methods in the field of precision oncology, their use is context-dependent (i.e., based on the molecular mechanisms of the disease) and they potentially could be used synergistically complementing the strengths of CRISPR and vice versa. Herein, we review the current status of CRISPR and PROTAC designs and their implications in the field of precision oncology in terms of clinical potential, clinical trial data, limitations, and compare their implications in precision clinical oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ad1fa36c97b78fd15ef1e65afe80aa88f0e0443" target='_blank'>
              Precision oncology revolution: CRISPR-Cas9 and PROTAC technologies unleashed
              </a>
            </td>
          <td>
            Karim Kanbar, Roy El Darzi, Diana E. Jaalouk
          </td>
          <td>2024-07-31</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
The genetic material within cells plays a pivotal role in shaping the structure and function of living organisms. Manipulating an organism's genome to correct inherited abnormalities or introduce new traits holds great promise. Genetic engineering techniques offers promising pathways for precisely altering cellular genetics. Among these methodologies, clustered regularly interspaced short palindromic repeat (CRISPR), honored with the 2020 Nobel Prize in Chemistry, has garnered significant attention for its precision in editing genomes. However, the CRISPR system faces challenges when applied in vivo, including low delivery efficiency, off‐target effects, and instability. To address these challenges, innovative technologies for targeted and precise delivery of CRISPR have emerged. Engineered carrier platforms represent a substantial advancement, improving stability, precision, and reducing the side effects associated with genome editing. These platforms facilitate efficient local and systemic genome engineering of various tissues and cells, including immune cells. This review explores recent advances, benefits, and challenges of CRISPR‐based genome editing delivery. It examines various carriers including nanocarriers (polymeric, lipid‐derived, metallic, and bionanoparticles), viral particles, virus‐like particles, and exosomes, providing insights into their clinical utility and future prospects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f48664bd4bcab7f27ff34a25f2e1c64fa38e6d" target='_blank'>
              Exploring Advanced CRISPR Delivery Technologies for Therapeutic Genome Editing
              </a>
            </td>
          <td>
            N. Rostami, M. M. Gomari, Edris Choupani, Shadi Abkhiz, Mahmood Fadaie, Seyed Sadegh Eslami, Zahra Mahmoudi, Yapei Zhang, Madhu Puri, Fatemeh Nafe Monfared, Elena Demireva, V. Uversky, B. R. Smith, S. Bencherif
          </td>
          <td>2024-07-25</td>
          <td>Small Science</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="ABSTRACT Many genes are known to regulate retinal regeneration after widespread tissue damage. Conversely, genes controlling regeneration after limited cell loss, as per degenerative diseases, are undefined. As stem/progenitor cell responses scale to injury levels, understanding how the extent and specificity of cell loss impact regenerative processes is important. Here, transgenic zebrafish enabling selective retinal ganglion cell (RGC) ablation were used to identify genes that regulate RGC regeneration. A single cell multiomics-informed screen of 100 genes identified seven knockouts that inhibited and 11 that promoted RGC regeneration. Surprisingly, 35 out of 36 genes known and/or implicated as being required for regeneration after widespread retinal damage were not required for RGC regeneration. The loss of seven even enhanced regeneration kinetics, including the proneural factors neurog1, olig2 and ascl1a. Mechanistic analyses revealed that ascl1a disruption increased the propensity of progenitor cells to produce RGCs, i.e. increased ‘fate bias’. These data demonstrate plasticity in the mechanism through which Müller glia convert to a stem-like state and context specificity in how genes function during regeneration. Increased understanding of how the regeneration of disease-relevant cell types is specifically controlled will support the development of disease-tailored regenerative therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4308685e50dc9ea339cdb924d4c527f4fee7a99" target='_blank'>
              A large-scale CRISPR screen reveals context-specific genetic regulation of retinal ganglion cell regeneration
              </a>
            </td>
          <td>
            Kevin Emmerich, John Hageter, Thanh V. Hoang, Pin Lyu, Abigail V. Sharrock, Anneliese Ceisel, James Thierer, Zeeshaan Chunawala, Saumya Nimmagadda, Isabella Palazzo, F. Matthews, Liyun Zhang, David T. White, Catalina Rodriguez, Gianna Graziano, Patrick Marcos, Adam May, Tim Mulligan, Barak Reibman, M. Saxena, D. Ackerley, Jiang Qian, Seth Blackshaw, Eric Horstick, Jeff S Mumm
          </td>
          <td>2024-07-15</td>
          <td>Development (Cambridge, England)</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Triplex DNA formation is a useful genomic targeting tool that is expected to have a wide range of applications, including the antigene method; however, there are fundamental limitations in its forming sequence. We recently extended the triplex DNA-forming sequence to methylated DNA sequences containing 5mCG base pairs by developing guanidino-dN, which is capable of recognizing a 5mCG base pair with high affinity. We herein investigated the effect of triplex DNA formation using TFOs with guanidino-dN on methylated DNA sequences at the promoter of the RASSF1A gene, whose expression is epigenetically suppressed by DNA methylation in MCF-7 cells, on gene expression. Interestingly, triplex DNA formation increased the expression of the RASSF1A gene at the transcript and protein levels. Furthermore, RASSF1A-activated MCF-7 cells exhibited cell growth suppressing activity. Changes in the expression of various genes associated with the promotion of apoptosis and breast cancer survival accompanied the activation of RASSF1A in cells exhibited antiproliferative activity. These results suggest the potential of increases in gene expression through triplex DNA formation as a new genomic targeting tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/367d9640afcf3f528a05a1512247caf159507678" target='_blank'>
              Novel strategy for activating gene expression through triplex DNA formation targeting epigenetically suppressed genes
              </a>
            </td>
          <td>
            Ryotaro Notomi, Shigeki Sasaki, Yosuke Taniguchi
          </td>
          <td>2024-07-31</td>
          <td>RSC Chemical Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer mortality worldwide. KRAS oncogenes are responsible for at least a quarter of lung adenocarcinomas, the main subtype of lung cancer. After four decades of intense research, selective inhibitors of KRAS oncoproteins are finally reaching the clinic. Yet, their effect on overall survival is limited due to the rapid appearance of drug resistance, a likely consequence of the high intratumoral heterogeneity characteristic of these tumors. In this study, we have attempted to identify those functional alterations that result from KRAS oncoprotein expression during the earliest stages of tumor development. Such functional changes are likely to be maintained during the entire process of tumor progression regardless of additional co-occurring mutations. Single-cell RNA sequencing analysis of murine alveolar type 2 cells expressing a resident Kras oncogene revealed impairment of the type I interferon pathway, a feature maintained throughout tumor progression. This alteration was also present in advanced murine and human tumors harboring additional mutations in the p53 or LKB1 tumor suppressors. Restoration of type I interferon (IFN) signaling by IFN-β or constitutive active stimulator of interferon genes (STING) expression had a profound influence on the tumor microenvironment, switching them from immunologically "cold" to immunologically "hot" tumors. Therefore, enhancement of the type I IFN pathway predisposes KRAS mutant lung tumors to immunotherapy treatments, regardless of co-occurring mutations in p53 or LKB1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4ab14ca9fcef735ebd7c09bd8738b90b51f2b7" target='_blank'>
              Type I interferon signaling pathway enhances immune-checkpoint inhibition in KRAS mutant lung tumors.
              </a>
            </td>
          <td>
            Fernando Fernández-García, Ana Fernández-Rodríguez, C. Fustero-Torre, Elena Piñeiro-Yáñez, Haiyun Wang, C. Lechuga, S. Callejas, Rebeca Álvarez, Alejandra López-García, Laura Esteban-Burgos, Marina Salmón, Marta San Román, Carmen Guerra, Chiara Ambrogio, M. Drosten, David Santamaría, Fátima Al-Shahrour, Ana Dopazo, Mariano Barbacid, M. Musteanu
          </td>
          <td>2024-08-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Retrons are promising gene editing tools because they can produce multi-copy single-stranded DNA in cells via self-primed reverse transcription. However, their potential for inserting genetic cargos in eukaryotes remains largely unexplored. Here we report the discovery and engineering of highly efficient retron-based gene editors for mammalian cells and vertebrates. Through bioinformatic analysis of metagenomic data and functional screening, we identified novel retron reverse transcriptases (RTs) that are highly active in mammalian cells. Rational design further improved the editing efficiency to levels comparable with conventional single-stranded oligodeoxynucleotide donors but from a genetically encoded cassette. Small molecule inhibitors of non-homologous end joining factors and Cas9-DNA repair protein fusions further increase homology-directed repair. Retron editors also exhibited robust activity with Cas12a nuclease and Cas9 nickase, expanding the genomic target scope and bypassing the need for a DNA double-stranded break. Using a rationally engineered retron editor, we incorporate a split GFP epitope tag for live cell imaging. Finally, we develop an all-RNA delivery strategy to enable DNA-free gene editing in cells and vertebrate embryos. This work establishes retron editors as a versatile and efficient tool for precise gene editing, offering new opportunities for biotechnology and biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8f2c620f5c7bf4a518c60425b0c63267672964" target='_blank'>
              Discovery and Engineering of Retrons for Precise Genome Editing
              </a>
            </td>
          <td>
            Jesse D. Buffington, Hung-Che Kuo, Kuang Hu, You-Chiun Chang, Kamyab Javanmardi, Brittney Voigt, Yi-Ru Li, Mary E. Little, Sravan K Devanathan, Blerta Xhemalçe, R. Gray, Ilya J. Finkelstein
          </td>
          <td>2024-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af76c1aab1fec450dc5bad42237d6e428e7def0" target='_blank'>
              Applying whole-genome and whole-exome sequencing in breast cancer: a review of the landscape.
              </a>
            </td>
          <td>
            Hetvi Ganatra, J. K. Tan, Ana Simmons, Carola Maria Bigogno, Vatsala Khurana, Aruni Ghose, Adheesh Ghosh, Ishika Mahajan, S. Boussios, Akash Maniam, Olubukola Ayodele
          </td>
          <td>2024-08-27</td>
          <td>Breast cancer</td>
          <td>0</td>
          <td>39</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [17],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>