<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-26 06:29:59 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>3</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2f585a6d644d3577541d684c98016ab2b165e02" target='_blank'>
              Sensing and guiding cell-state transitions by using genetically encoded endoribonuclease-mediated microRNA sensors.
              </a>
            </td>
          <td>
            Lei Wang, Wenlong Xu, Shun Zhang, Gregory C Gundberg, Christine R Zheng, Zhengpeng Wan, Kamila Mustafina, Fabio Caliendo, Hayden Sandt, Roger Kamm, Ron Weiss
          </td>
          <td>2024-07-09</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Therapeutic resistance is the leading cause of treatment failure and death from cancer. While resistance can be driven by genetic mutations, mounting evidence also points to an epigenetic basis of resistance. Much of this epigenetic, or non-genetic, resistance has been attributed to drug-resistant transcriptional cell states that are either induced by drug treatment or pre-exist in a fraction of cells selected by treatment. However, the extent to which long-term resistance is manifested in the early inherent cellular response to drugs is poorly understood, and which aspects, if any, of this early response to drug-induced transcriptional response are evolutionarily conserved properties of cells. To address these questions, we integrate datasets of long-term drug resistance and early drug response data across multiple cell lines with drug response and resistance data from bacteria and yeast. Our findings suggest that cancer cell states in both the drug-naive populations as well as in populations shortly after treatment share transcriptional properties with fully established resistant cell populations, and CRISPR-cas9 knockout of transcription factors predicted to regulate the resistant transcriptional programs result in inceased drug sensitivity. Furthermore, the resistance states manifested as early drug response are evolutionarily conserved. Finally, we show that early resistant states discriminate responders from non-responders across multiple human cancer trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad4a4d49f9d38cdc5b6bcd770ac201bba19fb9f" target='_blank'>
              Transcriptional Characterization of Resistance in Early Drug Response
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Michael M. Gottesman, Sridhar Hannenhalli
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Summary Phenotypic adaptation, the ability of cells to change phenotype in response to external pressures, has been identified as a driver of drug resistance in cancer. To quantify phenotypic adaptation in BRAFV600E-mutant melanoma, we develop a theoretical model that emerges from data analysis of WM239A-BRAFV600E growth rates in response to drug challenge with the BRAF-inhibitor encorafenib. Our model constitutes a cell population model in which each cell is individually described by one of multiple discrete and plastic phenotype states that are directly linked to drug-dependent net growth rates and, by extension, drug resistance. Data-matched simulations reveal that phenotypic adaptation in the cells is directed towards states of high net growth rates, which enables evasion of drug-effects. The model subsequently provides an explanation for when and why intermittent treatments outperform continuous treatments in vitro, and demonstrates the benefits of not only targeting, but also leveraging, phenotypic adaptation in treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db72a236ec0fc5bd479b42430df6525df1d343c7" target='_blank'>
              Growth rate-driven modelling reveals how phenotypic adaptation drives drug resistance in BRAFV600E-mutant melanoma
              </a>
            </td>
          <td>
            Sara Hamis, A. Browning, A. Jenner, Chiara Villa, Philip Maini, Tyler Cassidy
          </td>
          <td>2024-08-18</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Cancer resistance refers to the ability of cancer cells to evade the effects of treatments, such as chemotherapy and radiation, making it challenging to eliminate tumors. Cancer cells develop resistance through genetic modifications, alterations in cellular pathways, or microenvironmental adaptations that allow cancer cells to persist, proliferate, and even thrive despite the application of therapies designed to eradicate them. Despite efforts to create more potent target-based drugs, the flexibility of acquired resistance necessitates advanced options. Combination therapy and precision immunotherapy has revolutionized treatment but is limited by patient specificity and requires further development. New approaches like small molecules, peptides, and nanotherapeutics aim to overcome resistance by enhancing site-specific delivery and increasing drug concentration inside cancer cells. This review provides a comprehensive overview of the molecular mechanisms underlying cancer drug resistance, including genetic mutations, epigenetic alterations, efflux pump activity, and the influence of the tumor microenvironment. Additionally, the roles of microRNAs, long noncoding RNAs, and cellular processes such as autophagy and hypoxia in mediating resistance are examined. This review seeks to improve patient care and help to the development of more effective cancer medicines by explaining these complicated systems and investigating novel therapeutic alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c262ab9bdd31ff19f10e8cd7a734094b9985bc60" target='_blank'>
              An overview of molecular mechanisms in cancer drug resistance and therapeutic strategies
              </a>
            </td>
          <td>
            Navin Kumar, Ranjeeta Bansala
          </td>
          <td>2024-07-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epithelial-to-mesenchymal transition (EMT) is a major axis of phenotypic plasticity not only in diseased conditions such as cancer metastasis and fibrosis but also during normal development and wound healing. Yet-another important axis of plasticity with metastatic implications includes the cancer stem cell (CSCs) and non-CSC transitions. However, in both processes, epithelial (E) and mesenchymal (M) phenotypes are not merely binary states. Cancer cells acquire a spectrum of phenotypes with traits, properties, and markers of both E and M phenotypes, giving rise to intermediary hybrid (E/M) phenotypes. E/M cells play an important role in tumor initiation, metastasis, and disease progression in multiple cancers. Furthermore, the hybrid phenotypes also play a major role in causing therapeutic resistance in cancer. Here, we discuss how a systems biology perspective on the problem, which is implicit in the ‘Team Medicine’ approach outlined in the theme of this Special Issue of The Journal of Clinical Medicine and includes an interdisciplinary team of experts, is more likely to shed new light on EMT in cancer and help us to identify novel therapeutics and strategies to target phenotypic plasticity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6867e0a77d67eee55a122d53d387d619f33219b" target='_blank'>
              Phenotypic Plasticity and Cancer: A System Biology Perspective
              </a>
            </td>
          <td>
            A. R. Subbalakshmi, S. Ramisetty, Atish Mohanty, Siddhika Pareek, Dana Do, Sagun Shrestha, Ajaz Khan, Neel Talwar, Tingting Tan, Priya Vishnubhotla, Sharad S Singhal, Ravi Salgia, Prakash Kulkarni
          </td>
          <td>2024-07-23</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Outcomes for women with breast cancer have improved dramatically in recent decades. However, many patients present with intrinsic drug resistance and others are initially sensitive to anti-cancer drugs but acquire resistance during the course of their treatment, leading to recurrence and/or metastasis. Drug therapy-induced senescence (TIS) is a form of drug resistance characterised by the induction of cell cycle arrest and the emergence of a senescence-associated secretory phenotype (SASP) that can develop in response to chemo- and targeted- therapies. A wide range of anticancer interventions can lead to cell cycle arrest and SASP induction, by inducing genotoxic stress, hyperactivation of signalling pathways or oxidative stress. TIS can be anti-tumorigenic in the short-term, but pro-tumorigenic in the long-term by creating a pro-inflammatory and immunosuppressive microenvironment. Moreover, the SASP can promote angiogenesis and epithelial-mesenchymal transition in neighbouring cells. In this review, we will describe the characteristics of TIS in breast cancer and detail the changes in phenotype that accompany its induction. We also discuss strategies for targeting senescent cancer cells in order to prevent or delay tumour recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1902ff7d1c0379c707774d89c56e002ae6793dc1" target='_blank'>
              Therapy-induced senescence in breast cancer: an overview
              </a>
            </td>
          <td>
            Suraj Narayanan Chembukavu, Andrew J Lindsay
          </td>
          <td>2024-07-25</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is the second leading cause of global mortality and claims approximately 10 million lives annually. Despite advances in treatments such as surgery, chemotherapy, and immunotherapy, resistance to these methods has emerged. Multidrug resistance (MDR), where cancer cells resist diverse treatments, undermines therapy effectiveness, escalating mortality rates. MDR mechanisms include, among others, drug inactivation, reduced drug uptake, enhanced DNA repair, and activation of survival pathways such as autophagy. Moreover, MDR mechanisms can confer resistance to other therapies like radiotherapy. Understanding these mechanisms is crucial for improving treatment efficacy and identifying new therapeutic targets. Extracellular vesicles (EVs) have gathered attention for their role in cancer progression, including MDR development through protein transfer and genetic reprogramming. Autophagy, a process balancing cellular resources, also influences MDR. The intersection of EVs and autophagy further complicates the understanding of MDR. Both extracellular (exosomes, microvesicles) and intracellular (autophagic) vesicles contribute to therapy resistance by regulating the tumor microenvironment, facilitating cell communication, and modulating drug processing. While much is known about these pathways, there is still a need to explore their potential for predicting treatment responses and understanding tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5249ba1d82fab28ebb5cc330d8e07a3024195f" target='_blank'>
              The Role of Intracellular and Extracellular Vesicles in the Development of Therapy Resistance in Cancer.
              </a>
            </td>
          <td>
            Magdalena Wilczak, Magdalena Surman, M. Przybyło
          </td>
          <td>2024-08-07</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="All routine clinical treatments for colorectal cancer include 5-fluorouracil (5-FU), which cannot counteract recurrence and metastases formation. As the pyrimidine analog 5-FU can impact multiple pathways including both DNA and RNA metabolism, studying its mode of actions could lead to improved therapies. Using a dedicated reporter system for lineage-tracing and deep translatome profiling we demonstrate that 5-FU causes some colorectal cancer cells to tolerate the drug, due to a durable translational reprogramming that sustains cell plasticity. This period of drug tolerance coincides with specific translational activation of genes coding for proteins with major pro-tumoral functions. We unravel a major unexpected translational overexpression of the pro-inflammatory and pro-tumoral IL-8 cytokine, alongside other anti-apoptotic, senescence-associated secretory phenotype and cancer-related senescence phenotype genes. Given the adverse prognostic implications of elevated IL-8 levels across various cancers, our findings suggest IL-8 targeting could counteract 5-FU resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3f76dde38afaccb213072b4b26b46e5ff73bd2b" target='_blank'>
              Translational control of cell plasticity drives 5-FU tolerance
              </a>
            </td>
          <td>
            Mounira Chalabi-Dchar, Olivia Villeronce, J. Ripoll, A. Vincent, T. Fenouil, Rita Khoueiry, Jérôme Kucharczak, Laura Jentschel, Frédéric Catez, Arnaud M Vigneron, Julie Tréguier, Céline Mandier, Céline Bouclier, Jihane Vitre, Louise Lagerqvist, Armelle Choquet, Z. Herceg, C. Machon, Jérôme Guitton, Alexandre David, Eric Solary, David Bernard, Nadine Martin, Eric Rivals, N. D. Venezia, Julie Pannequin, J- J. Diaz
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Cellular responses to environmental stimuli are typically thought to be governed by genetically encoded programs. We demonstrate that melanoma cells can form and maintain cellular memories during the acquisition of therapy resistance that exhibit characteristics of cellular learning and are dependent on the transcription factor AP-1. We show that cells exposed to a low dose of therapy adapt to become resistant to a high dose, demonstrating that resistance was not purely selective. The application of therapy itself results in the encoding of transient gene expression into cellular memory and that this encoding occurs for both transiently induced and probabilistically arising expression. Chromatin accessibility showed concomitant persistence. A two-color AP-1 reporter system showed that these memories are encoded in cis, constituting an example of activating cis epigenetics. Our findings establish the formation and maintenance of cellular memories as a critical aspect of gene regulation during the development of therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a15dd395d0e2a350d250357de4baa259bd2f20" target='_blank'>
              AP-1 Mediates Cellular Adaptation and Memory Formation During Therapy Resistance
              </a>
            </td>
          <td>
            Jingxin Li, Pavithran T. Ravindran, Aoife O’Farrell, Gianna T. Busch, Ryan H. Boe, Zijian Niu, Sean Woo, Margaret C. Dunagin, Naveen Jain, Yogesh Goyal, K. Sarma, M. Herlyn, Arjun Raj
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>113</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eaccde1d6ccb6dacff234a0f6ddffc263c341a3" target='_blank'>
              Targeting oncogene-induced cellular plasticity for tumor therapy
              </a>
            </td>
          <td>
            Bin Li, Lingling Zheng, Jianhua Yang, Lianghu Qu
          </td>
          <td>2024-07-15</td>
          <td>Advanced Biotechnology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ae5740675a47153265ae470439ce458fde35c02" target='_blank'>
              Single-cell sensor analyses reveal signaling programs enabling Ras-G12C drug resistance.
              </a>
            </td>
          <td>
            Jason Z Zhang, Shao-En Ong, D. Baker, D. Maly
          </td>
          <td>2024-08-05</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A. Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M. As Sobeai
          </td>
          <td>2024-08-05</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Epigenetic evolution is a common mechanism used by cancer cells to evade the therapeutic effects of drug treatment. In ovarian cancers, epigenetically-driven resistance may be responsible for a large number of late-stage patient deaths. Here, we describe the first investigation into the role of G-quadruplex (G4) DNA secondary structures in mediating epigenetic regulation in drug-resistant ovarian cancer cells. Through genome-wide mapping of G4s in paired drug-sensitive and drug-resistant cell lines, we find that increased G4 formation is associated with significant increase in gene expression, with high enrichment in signalling pathways previously established to promote drug-resistant states. However, in contrast to previous studies, the expression-enhancing effects of G4s were not found at gene promoters, but intergenic and intronic regions, indicating that G4s promote long-range transcriptional regulation in drug-resistant cells. Furthermore, we discovered that clusters of G4s (super-G4s) are associated with particularly high levels of transcriptional enhancement that surpass the effects of super-enhancers, which act as well established regulatory sites in many cancers. Finally, we demonstrate that targeting G4s with small molecules results in significant down-regulation of pathways associated with drug-resistance, which results in resensitisation of resistant cells to chemotherapy agents. These findings indicate that G4 structures are critical for the epigenetic regulatory networks of drug-resistant cells and may represent a promising target to treat drug-tolerant ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef77f99d4d1172d6a547e45d37053b1f52fb2ad" target='_blank'>
              G-quadruplex structures regulate long-range transcriptional reprogramming to promote drug resistance in ovarian cancer
              </a>
            </td>
          <td>
            Jenna Robinson, Gem Flint, I. Garner, Silvia Galli, Thomas E. Maher, M. Kuimova, Ramon Vilar, I. McNeish, Robert Brown, Hector Keun, Marco Di Antonio
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Multiple myeloma (MM) is a genetically heterogeneous disease and the management of relapses is one of the biggest clinical challenges. TP53 alterations are established high‐risk markers and are included in the current disease staging criteria. KRAS is the most frequently mutated gene affecting around 20% of MM patients. Applying Clonal Competition Assays (CCA) by co‐culturing color‐labeled genetically modified cell models, we recently showed that mono‐ and biallelic alterations in TP53 transmit a fitness advantage to the cells. Here, we report a similar dynamic for two mutations in KRAS (G12A and A146T), providing a biological rationale for the high frequency of KRAS and TP53 alterations at MM relapse. Resistance mutations, on the other hand, did not endow MM cells with a general fitness advantage but rather presented a disadvantage compared to the wild‐type. CUL4B KO and IKZF1 A152T transmit resistance against immunomodulatory agents, PSMB5 A20T to proteasome inhibition. However, MM cells harboring such lesions only outcompete the culture in the presence of the respective drug. To better prevent the selection of clones with the potential of inducing relapse, these results argue in favor of treatment‐free breaks or a switch of the drug class given as maintenance therapy. In summary, the fitness benefit of TP53 and KRAS mutations was not treatment‐related, unlike patient‐derived drug resistance alterations that may only induce an advantage under treatment. CCAs are suitable models for the study of clonal evolution and competitive (dis)advantages conveyed by a specific genetic lesion of interest, and their dependence on external factors such as the treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736cb2bf2240ae0e1d6133faf0f979a9afbbf92" target='_blank'>
              Clonal competition assays identify fitness signatures in cancer progression and resistance in multiple myeloma
              </a>
            </td>
          <td>
            L. Haertle, U. Munawar, Hipólito N. C. Hernández, Andrés Arroyo-Barea, T. Heckel, I. Cuenca, Lucia Martin, Carlotta Höschle, Nicole Müller, C. Vogt, Thorsten Bischler, Paula L. del Campo, Seungbin Han, Natalia Buenache, Xiang Zhou, Florian Bassermann, J. Waldschmidt, Torsten Steinbrunn, L. Rasche, T. Stühmer, J. Martínez-López, K. Martin Kortüm, S. Barrio
          </td>
          <td>2024-07-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23cf92719cd49496400fba609df94fa13b009008" target='_blank'>
              Precision medicine in AML: overcoming resistance.
              </a>
            </td>
          <td>
            Samuel Urrutia, Koichi Takahashi
          </td>
          <td>2024-08-01</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e387009a4ed97bf637b9dd6a3591f151035e10" target='_blank'>
              Insights into gemcitabine resistance in pancreatic cancer: association with metabolic reprogramming and TP53 pathogenicity in patient derived xenografts
              </a>
            </td>
          <td>
            Mariam M. Konaté, Julia Krushkal, Ming-Chung Li, Li Chen, Yuri Kotliarov, Alida Palmisano, Rini Pauly, Qian Xie, P. M. Williams, Lisa M. McShane, Yingdong Zhao
          </td>
          <td>2024-08-05</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In this review, we revisit the pivotal role of fibroblast growth factor receptor 3 (FGFR3) in bladder cancer (BLCA), underscoring its prevalence in both non-muscle-invasive and muscle-invasive forms of the disease. FGFR3 mutations in up to half of BLCAs play a well-established role in tumorigenesis, shaping distinct tumor initiation patterns and impacting the tumor microenvironment (TME). Emphasizing the importance of considering epithelial-mesenchymal transition profile and TME status, we revisit their relevance in predicting responses to immune checkpoint inhibitors in FGFR3-mutated BLCAs. This writing highlights the initially promising yet transient efficacy of the FGFR inhibitor Erdafitinib on FGFR3-mutated BLCA, stressing the pressing need to unravel resistance mechanisms and identify co-targets for future combinatorial studies. A thorough analysis of recent preclinical and clinical evidence reveals resistance mechanisms, including secondary mutations, epigenetic alterations in pathway effectors, phenotypic heterogeneity, and population-specific variations within FGFR3 mutational status. Lastly, we discuss the potential of combinatorial treatments and concepts like synthetic lethality for discovering more effective targeted therapies against FGFR3-mutated BLCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d32b54c415deb1782db39b2832fb5448db0b31" target='_blank'>
              FGFR3 alterations in bladder cancer: Sensitivity and resistance to targeted therapies.
              </a>
            </td>
          <td>
            Maxim Noeraparast, Katarina Krajina, Renate Pichler, Dora Niedersüß-Beke, S. Shariat, Viktor Grünwald, S. Ahyai, Martin Pichler
          </td>
          <td>2024-08-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fba17e91584024423ac0c6e3cc0e42afd48ee9aa" target='_blank'>
              Emerging advances in defining the molecular and therapeutic landscape of small-cell lung cancer.
              </a>
            </td>
          <td>
            Triparna Sen, N. Takahashi, Subhamoy Chakraborty, Naoko Takebe, Amin H. Nassar, Nagla A Karim, S. Puri, Abdul Rafeh Naqash
          </td>
          <td>2024-07-04</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8428e89731398de2122358b2387bfd15c652d8d" target='_blank'>
              Homopolymer switches mediate adaptive mutability in mismatch repair-deficient colorectal cancer
              </a>
            </td>
          <td>
            H. Kayhanian, William Cross, Suzanne E M van der Horst, Panagiotis Barmpoutis, E. Lakatos, G. Caravagna, L. Zapata, Arne van Hoeck, Sjors Middelkamp, Kevin Litchfield, Christopher Steele, W. Waddingham, Dominic Patel, Salvatore Milite, Chen Jin, A. Baker, Daniel C Alexander, Khurum Khan, Daniel Hochhauser, Marco Novelli, Benjamin Werner, R. van Boxtel, Joris H. Hageman, Julian R Buissant des Amorie, Josep Linares, M. Ligtenberg, I. Nagtegaal, M. Laclé, Leon M G Moons, L. Brosens, N. Pillay, A. Sottoriva, Trevor A Graham, M. Rodriguez-Justo, K. Shiu, Hugo J G Snippert, Marnix Jansen
          </td>
          <td>2024-07-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0807e64db184649a4b0df3bd08502c4d34be0a53" target='_blank'>
              Programming tumor evolution with selection gene drives to proactively combat drug resistance.
              </a>
            </td>
          <td>
            Scott M. Leighow, Josh Reynolds, Ivan Sokirniy, Shun Yao, Zeyu Yang, Haider Inam, Dominik Wodarz, Marco Archetti, Justin Pritchard
          </td>
          <td>2024-07-04</td>
          <td>Nature biotechnology</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to sustain rapid proliferation, resist cell death, and adapt to varying microenvironmental conditions. This review elucidates the key mechanisms underlying metabolic reprogramming in cancer cells, including alterations in glucose metabolism, glutamine addiction, and lipid biosynthesis. A central feature of cancer metabolism is the Warburg effect, where cancer cells preferentially utilize aerobic glycolysis over oxidative phosphorylation, even in the presence of oxygen. This metabolic shift is driven by oncogenes and tumor suppressor genes, such as MYC and TP53, which modulate the expression and activity of enzymes involved in glycolysis and mitochondrial function. Additionally, cancer cells exhibit increased glutaminolysis, relying on glutamine as a carbon and nitrogen source to support anabolic processes and redox balance. Lipid metabolism is also reprogrammed, with enhanced de novo lipogenesis supplying membrane components and signaling molecules critical for tumor growth and survival. Understanding these metabolic alterations provides a basis for developing targeted therapies. Several therapeutic strategies have emerged, including inhibitors of key metabolic enzymes, such as hexokinase 2 (HK2), pyruvate kinase M2 (PKM2), and glutaminase. Metabolic interventions can disrupt the metabolic flexibility of cancer cells, sensitizing them to conventional therapies and overcoming resistance mechanisms. This review highlights the potential of metabolic targets in cancer treatment and emphasizes the need for personalized approaches considering tumor-specific metabolic profiles. Future research should focus on identifying biomarkers of metabolic vulnerabilities and developing combination therapies that exploit the intricate metabolic dependencies of cancer cells. Ultimately, targeting metabolic reprogramming holds promise for improving cancer prognosis and achieving more effective and durable therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d015236b22d21b366aae600c9f9e8d4b15a3c83" target='_blank'>
              A review of metabolic reprogramming in cancer cells: Mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Kelly Osayi Otakhor, Elizabeth O. Soladoye
          </td>
          <td>2024-07-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Our research aimed to identify new therapeutic targets for Lung adenocarcinoma (LUAD), a major subtype of non-small cell lung cancer known for its low 5-year survival rate of 22%. By employing a comprehensive methodological approach, we analyzed bulk RNA sequencing data from 513 LUAD and 59 non-tumorous tissues, identifying 2,688 differentially expressed genes. Using Mendelian randomization (MR), we identified 74 genes with strong evidence for a causal effect on risk of LUAD. Survival analysis on these genes revealed significant differences in survival rates for 13 of them. Our pathway enrichment analysis highlighted their roles in immune response and cell communication, deepening our understanding. We also utilized single-cell RNA sequencing (scRNA-seq) to uncover cell type-specific gene expression patterns within LUAD, emphasizing the tumor microenvironment’s heterogeneity. Pseudotime analysis further assisted in assessing the heterogeneity of tumor cell populations. Additionally, protein-protein interaction (PPI) network analysis was conducted to evaluate the potential druggability of these identified genes. The culmination of our efforts led to the identification of five genes (tier 1) with the most compelling evidence, including SECISBP2L, PRCD, SMAD9, C2orf91, and HSD17B13, and eight genes (tier 2) with convincing evidence for their potential as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e174b3e000455b97e2cc57b71e1d6819743b30f" target='_blank'>
              Identifying potential therapeutic targets in lung adenocarcinoma: a multi-omics approach integrating bulk and single-cell RNA sequencing with Mendelian randomization
              </a>
            </td>
          <td>
            Youpeng Chen, Enzhong Li, Zhenglin Chang, Tingting Zhang, Zhenfeng Song, Haojie Wu, Zhangkai J. Cheng, Baoqing Sun
          </td>
          <td>2024-07-18</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite development of several effective therapies for multiple myeloma (MM), the prognosis of patients with partial deletion of chromosome 17 (del(17p)) and TP53 aberrations remains poor. By applying comprehensive multi-omics profiling analyses (whole exome and transcriptome sequencing plus proteomics) and functional ex vivo drug screening to samples from 167 patients with MM, we uncovered novel therapeutic vulnerabilities specific to TP53 mutated MM. Our findings revealed a distinct sensitivity profile to a range of inhibitors (mitotic, topoisomerase, HDAC, HSP90, IGF1R and PI3K/AKT/mTOR inhibitors) irrespective of 17p deletion status. Conversely, no increase in sensitivity was observed for monoallelic TP53 (del(17p) with WT TP53) when compared to other samples, highlighting the remaining unmet clinical need. Notably, plicamycin, an RNA synthesis inhibitor linked to modulation of chromatin structure and increased transcription, emerged as particularly efficacious for TP53 mutated MM. The increased sensitivity correlated with higher protein expression of the drug targets: HDAC2, HSP90AA1 and multiple ribosomal subunits. Additionally, we observed increased RNA expression of G2M checkpoint, E2F targets and mTORC1 signaling in our cohort and the MMRF-CoMMpass (NCT01454297) study in TP53 mutated MM. Harmonization of multi-omics data with ex vivo drug screening results revealed that TP53 mutated MM is functionally distinct from MM with monoallelic TP53, and demonstrate that MM with mutated TP53, with and without del(17p), may be targetable by approved drugs, and further indicates the need for regular monitoring by sequencing to identify these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a52d0d33d267850e10246d4013cb5921df1165" target='_blank'>
              Multi-omic and functional screening reveal targetable vulnerabilities in TP53 mutated multiple myeloma
              </a>
            </td>
          <td>
            Dimitrios Tsallos, Nemo Ikonen, Juho J. Mienen, M. M. Majumder, S. Eldfors, Imre Västrik, A. Parsons, M. Suvela, Katie Dunphy, Paul Dowling, D. Bazou, Peter O’Gorman, J. Lievonen, R. Silvennoinen, Pekka Anla, C. Heckman
          </td>
          <td>2024-08-23</td>
          <td>None</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e223c600817ad1cd6259f50a38399720abf1f7d" target='_blank'>
              Drug tolerant persister cell plasticity in cancer: A revolutionary strategy for more effective anticancer therapies
              </a>
            </td>
          <td>
            Jun He, Zejing Qiu, Jingjing Fan, Xiaohong Xie, Qinsong Sheng, Xinbing Sui
          </td>
          <td>2024-08-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Simple Summary Diffuse large B-cell lymphoma is the most common type of lymphoma. Despite initial treatment, up to 40% of patients do not achieve a cure and need second-line therapy. For those experiencing late relapse or lacking access to CAR-T cell therapy, platinum-based chemotherapy followed by stem cell transplantation remains the standard of care. In this study, we used genomewide CRISPR/Cas9 screens and single-gene knockout experiments to identify the genes associated with responses to platinum-based drugs. We provide a comprehensive list of genes involved in the response to cisplatin in DLBCL. Our functional experiments highlight the critical roles of the DNA damage response genes XPA and ERCC6, in addition to BTK, in the response to platinum-based drugs. Additionally, we show that inhibition of BTK at lower concentrations sensitizes DLBCL cells to platinum-based drugs. Abstract The recurrence of diffuse large B-cell lymphoma (DLBCL) has been observed in 40% of cases. The standard of care for refractory/relapsed DLBCL (RR-DLBCL) is platinum-based treatment prior to autologous stem cell transplantation; however, the prognosis for RR-DLBCL patients remains poor. Thus, to identify genes affecting the cisplatin response in DLBCL, cisplatin-based whole-genome CRISPR-Cas9 knockout screens were performed in this study. We discovered DNA damage response (DDR) pathways as enriched among identified sensitizing CRISPR-mediated gene knockouts. In line, the knockout of the nucleotide excision repair genes XPA and ERCC6 sensitized DLBCL cells to platinum drugs irrespective of proliferation rate, thus documenting DDR as essential for cisplatin sensitivity in DLBCL. Functional analysis revealed that the loss of XPA and ERCC6 increased DNA damage levels and altered cell cycle distribution. Interestingly, we also identified BTK, which is involved in B-cell receptor signaling, to affect cisplatin response. The knockout of BTK increased cisplatin sensitivity in DLBCL cells, and combinatory drug screens revealed a synergistic effect of the BTK inhibitor, ibrutinib, with platinum drugs at low concentrations. Applying local and external DLBCL cohorts, we addressed the clinical relevance of the genes identified in the CRISPR screens. BTK was among the most frequently mutated genes with a frequency of 3–5%, and XPA and ERCC6 were also mutated, albeit at lower frequencies. Furthermore, 27–54% of diagnostic DLBCL samples had mutations in pathways that can sensitize cells to cisplatin. In conclusion, this study shows that XPA and ERCC6, in addition to BTK, are essential for the response to platinum-based drugs in DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2794e056513b2d174f98b17ca50338733fe72c8" target='_blank'>
              CRISPR-Cas9 Knockout Screens Identify DNA Damage Response Pathways and BTK as Essential for Cisplatin Response in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            I. I. Issa, Hanne Due, R. Brøndum, Vidthdyan Veeravakaran, Hulda Haraldsdóttir, Cathrine Sylvester, Asta Brogaard, Soniya Dhanjal, Bernhard Schmierer, K. Dybkær
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Simple Summary Lung adenocarcinoma, the most prevalent type of lung cancer, presents a significant treatment challenge owing to drug resistance. This study aimed to investigate the role of long non-coding RNAs (lncRNAs) in promoting intrinsic resistance in three lung adenocarcinoma cell lines from the onset of treatment. Drug-tolerant persister (DTP) cells, a subset of cancer cells with the ability to survive and proliferate after exposure to therapeutic drugs, were generated. RNA sequencing was used to investigate the differential expression of lncRNAs, and the clinical relevance of lncRNAs was assessed in a cohort of lung adenocarcinoma patients from The Cancer Genome Atlas. Knockdown of these lncRNAs increases the sensitivity of the analyzed cell lines to therapeutic drugs. This study provides an opportunity to investigate the role of lncRNAs in the genetic and epigenetic mechanisms underlying intrinsic resistance. Abstract Lung adenocarcinoma is the most prevalent form of lung cancer, and drug resistance poses a significant obstacle in its treatment. This study aimed to investigate the overexpression of long non-coding RNAs (lncRNAs) as a mechanism that promotes intrinsic resistance in tumor cells from the onset of treatment. Drug-tolerant persister (DTP) cells are a subset of cancer cells that survive and proliferate after exposure to therapeutic drugs, making them an essential object of study in cancer treatment. The molecular mechanisms underlying DTP cell survival are not fully understood; however, long non-coding RNAs (lncRNAs) have been proposed to play a crucial role. DTP cells from lung adenocarcinoma cell lines were obtained after single exposure to tyrosine kinase inhibitors (TKIs; erlotinib or osimertinib). After establishing DTP cells, RNA sequencing was performed to investigate the differential expression of the lncRNAs. Some lncRNAs and one mRNA were overexpressed in DTP cells. The clinical relevance of lncRNAs was evaluated in a cohort of patients with lung adenocarcinoma from The Cancer Genome Atlas (TCGA). RT–qPCR validated the overexpression of lncRNAs and mRNA in the residual DTP cells and LUAD biopsies. Knockdown of these lncRNAs increases the sensitivity of DTP cells to therapeutic drugs. This study provides an opportunity to investigate the involvement of lncRNAs in the genetic and epigenetic mechanisms that underlie intrinsic resistance. The identified lncRNAs and CD74 mRNA may serve as potential prognostic markers or therapeutic targets to improve the overall survival (OS) of patients with lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47f1b958113e3cfa109f6bab13ad3bc4cd8c921" target='_blank'>
              Transcriptomic Analysis Reveals Early Alterations Associated with Intrinsic Resistance to Targeted Therapy in Lung Adenocarcinoma Cell Lines
              </a>
            </td>
          <td>
            Mario Perez-Medina, J. Lopez-Gonzalez, Jesus J. Benito-Lopez, Santiago Ávila-Ríos, Maribel Soto-Nava, M. Matías-Florentino, A. Méndez-Tenorio, M. Galicia-Velasco, R. Chavez-Dominguez, S. Meza-Toledo, D. Aguilar-Cázares
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Lung cancer patients treated with targeted therapies frequently respond well but invariably relapse due to the development of drug resistance. Drug resistance is in part mediated by a subset of cancer cells termed “drug-tolerant persisters” (DTPs), which enter a dormant, slow-cycling state that enables them to survive drug exposure. DTPs also exhibit stem cell-like characteristics, broad epigenetic reprogramming, altered metabolism, and a mutagenic phenotype mediated by adaptive mutability. While several studies have characterised the transcriptional changes that lead to the altered phenotypes exhibited in DTPs, these studies have focused predominantly on protein coding changes. As long non-coding RNAs (lncRNAs) are also implicated in the phenotypes altered in DTPs, it is likely that they play a role in the biology of drug tolerance. In this review, we outline how lncRNAs may contribute to the key characteristics of DTPs, their potential roles in tolerance to targeted therapies, and the emergence of genetic resistance in lung adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83141f0410de3ec7e33ac04641da48080cc097e5" target='_blank'>
              The Potential Links between lncRNAs and Drug Tolerance in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            William J. H. Davis, Catherine J. Drummond, Sarah D. Diermeier, Glen Reid
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="INTRODUCTION
Colon cancer, ranked as the fourth leading global cause of cancer death, exhibits a complex progression marked by genetic variations. Over the past decade, the utilization of diverse CRISPR systems has propelled accelerated research into colorectal cancer (CRC) treatment.


AREAS COVERED
CRISPR/Cas9, a key player in this research, identifies new oncogenes, tumor suppressor genes (TSGs), and drug-resistance genes. Additionally, it facilitates the construction of experimental models, conducts genome-wide library screening, and develops new therapeutic targets, especially for targeted knockout in vivo or molecular targeted drug delivery, contributing to personalized treatments and significantly enhancing the care of colon cancer patients. In this review, we provide insights into the mechanism of the CRISPR/Cas9 system, offering a comprehensive exploration of its applications in CRC, spanning screening, modeling, gene functions, diagnosis, and gene therapy. While acknowledging its transformative potential, the article  highlights the challenges and limitations of CRISPR systems.


EXPERT OPINION
The application of CRISPR/Cas9 in CRC research provides a promising avenue for personalized treatments. Its potential for identifying key genes and enabling experimental models and genome-wide screening enhances patient care. This review underscores the significance of CRISPR-Cas9 gene editing technology across basic research, diagnosis, and the treatment landscape of colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e832452c0de49427c039c9545ac3618de43c5948" target='_blank'>
              Recent advances in CRISPR-Cas systems for colorectal cancer research and therapeutics.
              </a>
            </td>
          <td>
            Saeideh Khorshid Sokhangouy, Farzaneh Alizadeh, Malihe Lotfi, Samaneh Sharif, Atefeh Ashouri, Yasamin Yoosefi, Saeed Bozorg Qomi, M. Abbaszadegan
          </td>
          <td>2024-08-12</td>
          <td>Expert review of molecular diagnostics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A. Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2239c71d70f51fc31d667365569c847abafe0d7a" target='_blank'>
              Novel drug resistance mechanisms and drug targets in BRAF-mutated peritoneal metastasis from colorectal cancer
              </a>
            </td>
          <td>
            Christin Lund-Andersen, A. Torgunrud, Chakravarthi Kanduri, V. Dagenborg, I. S. Frøysnes, Mette M. Larsen, Ben Davidson, Stein G Larsen, K. Flatmark
          </td>
          <td>2024-07-09</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Using five complementary short- and long-read sequencing technologies, we phased and assembled >95% of each diploid human genome in a four-generation, 28-member family (CEPH 1463) allowing us to systematically assess de novo mutations (DNMs) and recombination. From this family, we estimate an average of 192 DNMs per generation, including 75.5 de novo single-nucleotide variants (SNVs), 7.4 non-tandem repeat indels, 79.6 de novo indels or structural variants (SVs) originating from tandem repeats, 7.7 centromeric de novo SVs and SNVs, and 12.4 de novo Y chromosome events per generation. STRs and VNTRs are the most mutable with 32 loci exhibiting recurrent mutation through the generations. We accurately assemble 288 centromeres and six Y chromosomes across the generations, documenting de novo SVs, and demonstrate that the DNM rate varies by an order of magnitude depending on repeat content, length, and sequence identity. We show a strong paternal bias (75-81%) for all forms of germline DNM, yet we estimate that 17% of de novo SNVs are postzygotic in origin with no paternal bias. We place all this variation in the context of a high-resolution recombination map (∼3.5 kbp breakpoint resolution). We observe a strong maternal recombination bias (1.36 maternal:paternal ratio) with a consistent reduction in the number of crossovers with increasing paternal (r=0.85) and maternal (r=0.65) age. However, we observe no correlation between meiotic crossover locations and de novo SVs, arguing against non-allelic homologous recombination as a predominant mechanism. The use of multiple orthogonal technologies, near-telomere-to-telomere phased genome assemblies, and a multi-generation family to assess transmission has created the most comprehensive, publicly available “truth set” of all classes of genomic variants. The resource can be used to test and benchmark new algorithms and technologies to understand the most fundamental processes underlying human genetic variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcb8df79fd9bfc629f05070b595811f041bfe33" target='_blank'>
              A familial, telomere-to-telomere reference for human de novo mutation and recombination from a four-generation pedigree
              </a>
            </td>
          <td>
            David Porubsky, Harriet Dashnow, T. Sasani, Glennis A. Logsdon, P. Hallast, Michelle D. Noyes, Z. Kronenberg, T. Mokveld, Nidhi Koundinya, Cillian Nolan, C. Steely, A. Guarracino, E. Dolzhenko, William T. Harvey, W. Rowell, K. Grigorev, T. J. Nicholas, Keisuke K. Oshima, Jiadong Lin, Peter Ebert, W. Watkins, Tiffany Y. Leung, Vincent C. T. Hanlon, Sean Mcgee, Brent S. Pedersen, Michael E. Goldberg, Hannah C. Happ, Hyeonsoo Jeong, Katherine M. Munson, Kendra Hoekzema, Daniel D. Chan, Yanni Wang, Jordan Knuth, Gage H. Garcia, Cairbre Fanslow, Christine Lambert, Charles Lee, Josh Smith, Shawn Levy, C. Mason, Erik Garrison, Peter Lansdorp, D. Neklason, L. Jorde, Aaron R. Quinlan, M. Eberle, Evan E. Eichler
          </td>
          <td>2024-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c654e20750311088b55e69ff370d60ba4dd73ded" target='_blank'>
              Diabetes Driven Oncogenesis and Anticancer Potential of Repurposed Antidiabetic Drug: A Systemic Review.
              </a>
            </td>
          <td>
            Iqra Khan, Aisha Kamal, Salman Akhtar
          </td>
          <td>2024-07-02</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Benzo(a)pyrene (BaP) is a prevalent food and environmental carcinogen. Chronic low-dose BaP exposure can promote the migratory and invasive capacities of human hepatocellular carcinoma (HCC) cells, yet its intricate molecular mechanisms remain elusive. Utilizing the established BaP-exposed HCC cell model, we analyzed the gene expression alteration, exosomal RNA cargo, and genetic variants induced by BaP through transcriptomic and whole-genome sequencing. Transcriptomic analysis revealed significant dysregulation in genes and pathways associated with tumor metastasis, particularly those involved in steroidal lipid metabolism and cell migration. BaP exposure enriched PI3K-AKT, mTOR, and NF-κB signaling pathways and disrupted genes implicated in cellular secretory processes, suggesting the potential involvement of exosomes in metastasis. Exosome analysis depicted the RNA profiling in exosomes of HCC cells altered by BaP, and the exosomal circRNA-miRNA-mRNA interaction network was constructed. Finally, whole-genome sequencing delineated BaP-induced gene mutations and genomic instability in HCC cells. In summary, prolonged low-dose BaP exposure induces intricate molecular alterations in gene mutation and expression profiles in HCC cells, notably those secreted in exosomes, which may potentially remodel the tumor microenvironment and foster HCC metastasis. Our findings offer new insights into the molecular underpinnings of BaP-induced HCC metastasis, thereby advancing the comprehensive understanding of BaP toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e7aa79dbaa0e9a217dc4d37ffdecdec77e97e8e" target='_blank'>
              Molecular Characteristics of Aberrant Gene Mutations and Expression Profiles Induced by Benzo(a)pyrene in Hepatocellular Carcinoma Cells
              </a>
            </td>
          <td>
            Xinyi Cao, Ying Zhu, S. Cheng, Kunxiao Zhang, Hui Wang, Qian Ba
          </td>
          <td>2024-07-01</td>
          <td>Toxics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The genetic integrity of pluripotent stem cells (PSC) is integral to their applications in research and therapy, but it is compromised by frequent development of copy number variations. Little is known about the basis of the variable genomic integrity among different PSC lines. Here we identify aneuploidies using RNA-seq and proteomics data from a panel of mouse embryonic stem cell (mESC) lines derived from 170 Diversity Outbred mice. We identified 62 lines with detectable aneuploid subpopulations and a subset of originally XX lines that lost one Chromosome X (XO). Strikingly, a much lower proportion of XX lines were aneuploid, compared to XY or XO lines. Two single-cell RNA-seq data sets demonstrated that aneuploid XY DO mESC also show lower Chromosome X gene expression and that XY mESC accumulate higher aneuploid proportions in culture than isogenic XX lines. Possible mechanisms for this protective effect of X chromosome dosage include our discovery that the lines with two active X Chromosomes have a higher expression of 2-cell-like state genes, and differential expression of X-linked tumor suppressor genes associated with DNA damage response. Highlights First genetic analysis of predisposition to aneuploidy in pluripotent stem cell cultures Chromosomal regions duplicated in aneuploid mouse embryonic stem cells are syntenic with regions overrepresented in human pluripotent stem cell lines bearing recurrent genetic abnormalities X-Chromosome dosage strongly influences susceptibility to aneuploidy in mouse embryonic stem cells and to a lesser degree in human pluripotent stem cells XX mouse embryonic stem cell lines show a higher proportion of cells in 2 cell-like state and higher expression of tumor suppressor genes associated with DNA damage response">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f853ae6e95a31cb9ad3712a21b0838a6601fe33" target='_blank'>
              Chromosome X Dosage Modulates Development of Aneuploidy in Genetically Diverse Mouse Embryonic Stem Cells
              </a>
            </td>
          <td>
            Alexander Stanton, Selcan Aydin, Daniel A. Skelly, Dylan Stavish, Kim Leonhard, Seth M. Taapken, Erik M. McIntire, Matthew Pankratz, A. Czechanski, T. Ludwig, Ted Choi, Steve P. Gygi, Ivana Barbaric, Steven C. Munger, L. Reinholdt, Martin F. Pera
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/699986230662d4afba97aea6b14e6128a02adbf7" target='_blank'>
              Molecular mechanisms of DNA lesion and repair during antibody somatic hypermutation.
              </a>
            </td>
          <td>
            Qian Hao, Jinfeng Li, Leng-Siew Yeap
          </td>
          <td>2024-07-23</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Introduction Glioblastomas utilize malignant gene expression pathways to drive growth. Many of these gene pathways are not directly accessible with molecularly targeted pharmacological agents. Chromatin-modifying compounds can alter gene expression and target glioblastoma growth pathways. In this study, we utilize a systematic screen of chromatin-modifying compounds on a panel of patient-derived glioblastoma lines to identify promising compounds and their associated gene targets. Methods Five glioblastoma cell lines were subjected to a drug screen of 106 chromatin-modifying compounds representing 36 unique drug classes to determine the twelve most promising drug classes and the best candidate inhibitors in each class. These twelve drugs were then tested with a panel of twelve patient-derived gliomasphere lines to identify growth inhibition and corresponding gene expression patterns. Overlap analysis and weighted co-expression network analysis (WCGNA) were utilized to determine potential target genes and gene pathways. Results The initial drug screen identified twelve candidate pharmacologic agents for further testing. Drug sensitivity testing indicated an overall high degree of variability between gliomasphere lines. However, CPI203 was the most consistently effective compound, and the BET inhibitor class was the most consistently effective class of compounds across the gliomasphere panel. Correspondingly, most of the compounds tested had highly variable effects on gene expression between gliomasphere lines. CPI203 stood out as the only compound to induce a consistent effect on gene expression across different gliomasphere lines, specifically down-regulation of DNA-synthesis genes. Amongst the twelve tested cell lines, high expression of CDKN2A and CDKN2B distinguished more drug sensitive from more drug resistant lines. WCGNA identified two oncogenic gene modules (FBXO5 and MELK) that were effectively downregulated by CPI203 (FBXO5) and ML228 (FBXO5 and MELK). Conclusions The bromodomain inhibitor CPI203 induced relatively consistent effects on gene expression and growth across a variety of glioblastoma lines, specifically down-regulating genes associated with DNA replication. We propose that clinically effective BET inhibitors have the potential to induce consistent beneficial effects across a spectrum of glioblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1561fa27def3a17181508712df5ccd39c21a9f63" target='_blank'>
              CPI203, a BET inhibitor, down-regulates a consistent set of DNA synthesis genes across a wide array of glioblastoma lines
              </a>
            </td>
          <td>
            Matthew C. Garrett, Troy P Carnwath, Rebecca R Albano, Yonghua Zhuang, Catherine A Behrmann, Merissa Pemberton, Farah Barakat, Robert Lober, Mark Hoeprich, Anthony Paravati, Marilyn Reed, Hailey Spry, Daniel Woo, Eric O’Brien, Brett VanCauwenbergh, John Perentesis, Rani Nasser, Mario Medvedovic, D. Plas
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Transcription and splicing errors in cancer cells generate erroneous transcripts. Since erroneous transcripts are degraded by the nonsense-mediated mRNA decay (NMD) pathway, whether they are junk or could be functional has been overlooked and understudied. We addressed this question by first performing a pan-cancer analysis and identified substantial erroneous noncoding transcripts (ENT) in cancers. Given that RNA/DNA binding domains (DBD) were predicted in ENTs, we deleted predicted DBDs in multiple ENTs in multiple cell lines, with RNA-sequencing and cell experiments before and after DBD deletion. DBD deletion caused significantly changed expression of ENTs’ target genes (whose promoter regions contain ENTs’ DNA binding sites, DBS) and changed cell migration and proliferation ability, indicating that many ENTs can transcriptionally regulate genes. Tightly coupled data analysis and experiments reveal that ENTs’ functions are highly cancer- and cellular-context specific, making ENTs a new class of safe and specific targets for noncoding RNA-based cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e45af8d1c765a560ff555893aa2ed200e47f3c" target='_blank'>
              Many erroneous noncoding transcripts in cancer cells can highly specifically regulate cancer-related genes and pathways
              </a>
            </td>
          <td>
            Sha He, Wei Xiong, Jianping Huo, Jie Lin, Jianmin Li, Hao Zhu
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="T cell acute lymphoblastic leukemia (T-ALL) is an aggressive leukemia predominantly affecting adolescents and young adults. T-ALL relapses are characterized by chemotherapy resistance, cellular heterogeneity and dismal outcome. To gain a deeper understanding of the cellular heterogeneity and mechanisms driving relapse, we conducted single-cell full-length RNA sequencing of 13 matched pediatric T-ALL patient-derived xenografts (PDX) samples obtained at initial diagnosis and relapse, generating the to date most comprehensive longitudinal single cell study in paired T-ALL samples, along with 5 non-relapsing PDX samples collected at initial diagnosis. This dataset identifies considerable transcriptomic diversity among individual T-ALL cell populations. Notably however, 11 of the 18 patients exhibit a small T-ALL cell subpopulation with a shared set of gene regulatory networks characterized by a common set of active regulons, expression patterns and splice isoforms. This profile involves the upregulation of a stem-like cell signature with enrichment of cell adhesion, and inhibition of the cell cycle and metabolic activity. Comprehensive investigations of these networks identify transcripts enforcing drug resistance through NF-κB expression and TGF-β signaling and of anti-apoptotic T cell signaling. Longitudinal monitoring of these stem-like cells demonstrates this subpopulation to account for only a small proportion of leukemia cells initially with a substantial expansion at relapse suggesting resistance to first line therapy. Chemotherapy resistance is functionally corroborated through in vitro and in vivo drug testing. We thus report the discovery of inherently treatment-resistant stem-like T-ALL cell populations underscoring the potential for devising future therapeutic strategies aimed at targeting stemness pathways in pediatric T-ALL. Key Points Single-cell full-length RNA sequencing reveals dormant, treatment-resistant cells expanding upon T cell acute lymphoblastic leukemia relapse. Stem-like T-ALL cells exhibit a shared gene regulatory network conferring chemotherapy resistance and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88dbfdc32eaada9aa463b5fbbffd8e6808956ec" target='_blank'>
              Role of Stem-Like Cells in Chemotherapy Resistance and Relapse in pediatric T Cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Julia Costea, Kerstin K. Rauwolf, Pietro Zafferani, T. Rausch, A. Mathioudaki, Judith B. Zaugg, M. Schrappe, C. Eckert, G. Escherich, Jean-Pierre Bourquin, B. Bornhauser, Andreas E. Kulozik, J. Korbel
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135134a62bac76cb46dae2f04090c849f2c01172" target='_blank'>
              Targeting chromosomal instability in patients with cancer.
              </a>
            </td>
          <td>
            Duaa H Al-Rawi, Emanuele Lettera, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-11</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="CRISPR gene editing strategies are shaping cell therapies through precise and tunable control over gene expression. However, achieving reliable therapeutic effects with improved safety and efficacy requires informed target gene selection. This depends on a thorough understanding of the involvement of target genes in gene regulatory networks (GRNs) that regulate cell phenotype and function. Machine learning models have been previously used for GRN reconstruction using RNA- seq data, but current techniques are limited to single cell types and focus mainly on transcription factors. This restriction overlooks many potential CRISPR target genes, such as those encoding extracellular matrix components, growth factors, and signaling molecules, thus limiting the applicability of these models for CRISPR strategies. To address these limitations, we have developed CRISPR-GEM, a multi-layer perceptron (MLP)-based synthetic GRN constructed to accurately predict the downstream effects of CRISPR gene editing. First, input and output nodes are identified as differentially expressed genes between defined experimental and target cell/tissue types respectively. Then, MLP training learns regulatory relationships in a black-box approach allowing accurate prediction of output gene expression using only input gene expression. Finally, CRISPR-mimetic perturbations are made to each input gene individually and the resulting model predictions are compared to those for the target group to score and assess each input gene as a CRISPR candidate. The top scoring genes provided by CRISPR-GEM therefore best modulate experimental group GRNs to motivate transcriptomic shifts towards a target group phenotype. This machine learning model is the first of its kind for predicting optimal CRISPR target genes and serves as a powerful tool for enhanced CRISPR strategies across a range of cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/486423f8c90770082bb7f4dc4e4e7334da919173" target='_blank'>
              CRISPR-GEM: A Novel Machine Learning Model for CRISPR Genetic Target Discovery and Evaluation
              </a>
            </td>
          <td>
            Josh P. Graham, Yu Zhang, Lifang He, Tomas Gonzalez-Fernandez
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Increased access to high-throughput DNA sequencing platforms has transformed the diagnostic landscape of pediatric malignancies by identifying and integrating actionable genomic or transcriptional features that refine diagnosis, classification, and treatment. Yet less than 10% of treated patients show a positive response and translating precision oncology data into feasible and effective therapies for hard-to-cure childhood, adolescent, and young adult malignancies remains a significant challenge. Combining the identification of therapeutic targets at the protein and pathway levels with demonstration of treatment response in personalized models holds great promise. Here we present the case for combining proteomics with patient-derived xenograft (PDX) models to identify personalized treatment options that were not apparent at genomic and transcriptomic levels. Proteome analysis with immunohistochemistry (IHC) validation of formalin-fixed paraffin-embedded sections from an adolescent with primary and metastatic spindle epithelial tumor with thymus-like elements (SETTLE) was completed within two weeks of biopsy. The results identified an elevated protein level of SHMT2 as a possible target for therapy with the commercially available anti-depressant sertraline. Within 2 months and ahead of a molecular tumor board, we confirmed a positive drug response in a personalized chick chorioallantoic membrane (CAM) model of the SETTLE tumor (CAM-PDX). Following the failure of cytotoxic chemotherapy and second-line therapy, a treatment of sertraline was initiated for the patient. After 3 months of sertraline treatment the patient showed decreased tumor growth rates, albeit with clinically progressive disease. Significance: Overall, we demonstrate that proteomics and fast-track personalized xenograft models can provide supportive pre-clinical data in a clinically meaningful timeframe to support medical decision-making and impact the clinical practice. By this we show that proteome-guided and functional precision oncology are feasible and valuable complements to the current genome- driven precision oncology practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6528441c9ef77e1a91eff560c6531d2a938e5fc1" target='_blank'>
              Proteomics and personalized patient-derived xenograft models identify treatment opportunities for a progressive malignancy within a clinically actionable timeframe and change care
              </a>
            </td>
          <td>
            G. D. Barnabas, T. A. Bhat, V. Goebeler, P. Leclair, N. Azzam, N. Melong, C. Anderson, A. Gom, S. An, E. Ergin, Y. Shen, A. Mungall, K. L. Mungall, C. A. Maxwell, G. S. D. Reid, M. Hirst, S. Jones, J. A. Chan, D. L. Senger, J. N. Berman, S. Parker, J. Bush, C. Strahlendorf, R. Deyell, C. J. Lim, P. F. Lange, Profyle Program
          </td>
          <td>2024-07-08</td>
          <td>None</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Single nucleotide variants (SNVs) contribute to cancer by altering the coding and the non-coding regions of the genome. Connecting SNVs to transcriptomic and epigenetic changes at the single-cell level remains challenging. To enable studies of rare cell populations harboring specific point mutations, we developed STAR-FACS, Specific-To-Allele PCR-FACS, to sort cells based on genomic allele alterations. We show that STAR-FACS can separate cells based on TERT promoter mutation status and is compatible with bulk and single-cell transcriptomic and epigenetic profiling. We demonstrate that glioblastoma cell lines derived from the same tumor but harboring distinct TERT promoter SNVs have different transcriptional programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f2ef09f49e055964d96a3768d156e4cf968931" target='_blank'>
              Cell sorting based on single nucleotide variation enables characterization of mutation-dependent transcriptome and chromatin states
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, Oszkar Szentirmai, M. Janiszewska
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Methods to engineer the genomes of human cells for therapeutic intervention continue to advance at a remarkable pace. Chimeric antigen receptor-engineered T lymphocytes have pioneered the way for these therapies, initially beginning with insertions of chimeric antigen receptor transgenes into T-cell genomes using classical gene therapy vectors. The broad use of clustered regularly interspaced short palindromic repeats (CRISPR)-based technologies to edit endogenous genes has now opened the door to a new era of precision medicine. To add complexity, many engineered cellular therapies under development integrate gene therapy with genome editing to introduce novel biological functions and enhance therapeutic efficacy. Here, we review the current state of scientific, translational, and regulatory oversight of gene-edited cell products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2d1ab3d9838377779d6ccf7a7d2e82b715e992" target='_blank'>
              Regulatory Considerations for Genome-Edited T-cell Therapies.
              </a>
            </td>
          <td>
            J. Jadlowsky, Ju-Fang Chang, DH Spencer, John M Warrington, Bruce L. Levine, Carl H. June, J. Fraietta, Nathan Singh
          </td>
          <td>2024-07-17</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="BACKGROUND
Metastatic colorectal cancer (mCRC) is increasingly characterized by myriad genomic alterations beyond the well-known factors such as RAS, BRAF, and microsatellite instability (MSI). Novel genomic changes, including ERBB2 amplifications, mutations, and gene fusions, are now recognized as potential targets for precision therapy. This study aims to explore the genomic landscape of a Chinese cohort with mCRC to identify potentially targetable genetic alterations for personalized treatment strategies.


METHODS
A total of 500 mCRC patients in China were enrolled, based on which genomic profiling was performed using capture-based targeted sequencing across a panel of 520 genes on tumor tissues to identify prevalent genomic alterations. The mutations were analyzed by optimized proprietary algorithms. MSI and mismatch repair deficiency status were analyzed using the read-count-distribution approach. Besides, the overall survival (OS) related to these molecular changes was estimated.


RESULTS
The cohort's genomic profiling revealed TP53 mutations in 78%, APC in 60%, and KRAS in 47% of the patients. MSI-High status was confirmed in 5.8% of cases via a next-generation sequencing (NGS)-based algorithm. ERBB2/HER2 amplifications were found in 12% (60/500) of patients, with potential therapeutic implications for those without concurrent KRAS mutations. A subset of patients (1.2%; 6/500) showed fusions and DNA damage response (DDR) gene mutations (except TP53) that could be targeted therapeutically. The KRAS (G12C) variant was detected in 14 patients (2.8%), and 61 (12.2%) had a BRAF V600E mutation. Notably, survival analysis showed no significant differences in OS between KRAS mutant loci and NRAS mutations (p = 0.436). However, BRAF V600E mutations were associated with a poorer prognosis than BRAF wild-type and non-V600E mutations (16.3 months vs. 29.5 and 31.1 months, respectively; p < 0.001).


CONCLUSIONS
This study validates the feasibility of using NGS to detect prognostic and therapeutically actionable genetic variants in Chinese mCRC patients, contributing to understanding the genomic variation within this population and highlighting the potential for personalized medicine in managing mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d75b3be37f6d3fb50ba2d4fcbf63055e7d0a5a8b" target='_blank'>
              Genomic Alterations and Clinical Characterization in Chinese Patients with Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            Xu-hui Zhang, Jie-Qiong Zhou, Qing Wei, Jian Li, Ting Xu, Chun-mei Bai, Jian-Feng Zhou, Xi-cheng Wang
          </td>
          <td>2024-07-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ffdcafc4ed89cea9e479cfcd544ea8dcf0cb5d2" target='_blank'>
              Towards genomic medicine: a tailored next-generation sequencing panel for hydroxyurea pharmacogenomics in Tanzania
              </a>
            </td>
          <td>
            S. Nkya, Collin Nzunda, E. Saukiwa, Frida Kaywanga, Eliud Buberwa, David Solomon, H. Christopher, Doreen Ngowi, Julieth Johansen, Florence Urio, J. Mgaya, Salman Karim, M. Alimohamed, Raphael Zozimus Sangeda, C. Chamba, Emile R. Chimusa, Enrico Novelli, J. Makani
          </td>
          <td>2024-07-18</td>
          <td>BMC Medical Genomics</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c91dfeec839321e76219264c96627fff2fc054a" target='_blank'>
              Non-coding RNAs in BRAF-mutant melanoma: targets, indicators, and therapeutic potential.
              </a>
            </td>
          <td>
            S. Afsar, Rahamat Unissa Syed, Weam M A Khojali, Najat Masood, Mhdia Elhadi Osman, J. S. Jyothi, Mohd Abdul Hadi, Amna Abakar Suleiman Khalifa, Nayla Ahmed Mohamed Aboshouk, Hessa Ahmed Alsaikhan, Aljuri Saleh Alafnan, Bushra Abdullah Alrashidi
          </td>
          <td>2024-08-21</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Resistance to cancer therapy is driven by both cell-intrinsic and microenvironmental factors. Previous work has revealed that multiple resistant cell fates emerge in melanoma following treatment with targeted therapy and that, in vitro, these resistant fates are determined by the transcriptional state of individual cells prior to exposure to treatment. What remains unclear is whether these resistant fates are shared across different genetic backgrounds and how, if at all, these resistant fates interact with the tumor microenvironment. Through spatial transcriptomics and single-cell RNA sequencing, we uncovered distinct resistance programs in melanoma cells shaped by both intrinsic cellular states and the tumor microenvironment. Consensus non-negative matrix factorization revealed shared intrinsic resistance programs across different cell lines, highlighting the presence of universal and unique resistance pathways. In patient samples, we demonstrated that these resistance programs coexist within individual tumors and associate with diverse immune signatures, suggesting that the tumor microenvironment and distribution of resistant fates are closely connected. Single-cell resolution spatial transcriptomics in xenograft models revealed both intrinsically determined and extrinsically influenced resistant fates. Overall, this work demonstrates that each therapy resistant fate coexists with a distinct immune microenvironment in tumors and that, in vivo, tissue features, such as regions of necrosis, can influence which resistant fate is adopted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5979abc00b349ddf0161079fda7fee453bd0ce0a" target='_blank'>
              Spatial transcriptomics reveals influence of microenvironment on intrinsic fates in melanoma therapy resistance
              </a>
            </td>
          <td>
            Ryan H. Boe, Catherine G. Triandafillou, R. Lazcano, J. Wargo, Arjun Raj
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="Imatinib (IM), a breakthrough in chronic myeloid leukemia (CML) treatment, is accompanied by discontinuation challenges owing to drug intolerance. Although BCR-ABL1 mutation is a key cause of CML resistance, understanding mechanisms independent of BCR-ABL1 is also important. This study investigated the sphingosine-1-phosphate (S1P) signaling-associated genes (SphK1 and S1PRs) and their role in BCR-ABL1-independent resistant CML, an area currently lacking investigation. Through comprehensive transcriptomic analysis of IM-sensitive and IM-resistant CML groups, we identified the differentially expressed genes and found a notable upregulation of SphK1, S1PR2, and S1PR5 in IM-resistant CML. Functional annotation revealed their roles in critical cellular processes such as proliferation and GPCR activity. Their network analysis uncovered significant clusters, emphasizing the interconnectedness of the S1P signaling genes. Further, we identified interactors such as BIRC3, TRAF6, and SRC genes, with potential implications for IM resistance. Additionally, receiver operator characteristic curve analysis suggested these genes' potential as biomarkers for predicting IM resistance. Network pharmacology analysis identified six herbal compounds-ampelopsin, ellagic acid, colchicine, epigallocatechin-3-gallate, cucurbitacin B, and evodin-as potential drug candidates targeting the S1P signaling genes. In summary, this study contributes to efforts to better understand the molecular mechanisms underlying BCR-ABL1-independent CML resistance. Moreover, the S1P signaling genes are promising therapeutic targets and plausible new innovation avenues to combat IM resistance in cancer clinical care in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6358a3e0033ae2acc0726edb73a83bd9b4186018" target='_blank'>
              S1P Signaling Genes as Prominent Drivers of BCR-ABL1-Independent Imatinib Resistance and Six Herbal Compounds as Potential Drugs for Chronic Myeloid Leukemia.
              </a>
            </td>
          <td>
            Sikha Morang, Manisha Bisht, Vikas Upadhyay, Surabhi Thapliyal, S. Handu
          </td>
          <td>2024-07-01</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af8a5e84090feb0d13451624ef0b08cb6aa245a3" target='_blank'>
              Epigenetic regulation of DNA repair gene program by Hippo/YAP1-TET1 axis mediates sorafenib resistance in HCC
              </a>
            </td>
          <td>
            Chunli Mo, Weixin You, Yipeng Rao, Zhenping Lin, Shuai Wang, Ting He, Huanming Shen, Xun Li, Rui Zhang, Boan Li
          </td>
          <td>2024-07-05</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technique has enabled detailed analysis of gene expression at the single cell level, enhancing the understanding of subtle mechanisms that underly pathologies and drug resistance. To derive such biological meaning from sequencing data in oncology, some critical processing must be performed, including identification of the tumor cells by markers and algorithms that infer copy number variations (CNVs). We compared the performance of sciCNV, InferCNV, CopyKAT and SCEVAN tools that identify tumor cells by inferring CNVs from scRNA-seq data. Sequencing data from Pancreatic Ductal Adenocarcinoma (PDAC) patients, adjacent and healthy tissues were analyzed, and the predicted tumor cells were compared to those identified by well-assessed PDAC markers. Results from InferCNV, CopyKAT and SCEVAN overlapped by less than 30% with InferCNV showing the highest sensitivity (0.72) and SCEVAN the highest specificity (0.75). We show that the predictions are highly dependent on the sample and the software used, and that they return so many false positives hence are of little use in verifying or filtering predictions made via tumor biomarkers. We highlight how critical this processing can be, warn against the blind use of these software and point out the great need for more reliable algorithms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f15f4004aa5098686401484d91f98cd52ad6608" target='_blank'>
              A Comparison of Tools That Identify Tumor Cells by Inferring Copy Number Variations from Single-Cell Experiments in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Daisy J. A. Oketch, M. Giulietti, Francesco Piva
          </td>
          <td>2024-08-05</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="FOXL2 is a transcription factor essential for sex determination and ovary development and maintenance. Mutations in this gene are implicated in syndromes involving premature ovarian failure and granulosa cell tumors (GCTs). This rare cancer accounts for less than 5% of diagnosed ovarian cancers and is causally associated with the FOXL2 c.402C>G, p.C134W mutation in 97% of the adult cases (AGCTs). In this study, we employed CRISPR technology to specifically eliminate the FOXL2 c.402C>G mutation in granulosa tumor cells. Our results show that this Cas9-mediated strategy selectively targets the mutation without affecting the wild-type allele. Granulosa cells lacking the FOXL2 c.402C>G exhibit a reduced malignant phenotype, with significant changes in cell proliferation, invasion, and cell death. Furthermore, these modified cells are more susceptible to Dasatinib and Ketoconazole. Transcriptomic and proteomic analyses reveal that CRISPR-modified granulosa tumor cells shift their expression profiles towards a wild-type like phenotype. Additionally, this altered expression signature has led to the identification of new compounds with antiproliferative and pro-apoptotic effects on granulosa tumor cells. Our findings demonstrate the potential of CRISPR technology for the specific targeting and elimination of a mutation causing granulosa cell tumors, highlighting its therapeutic promise for treating this rare ovarian cancer. Simple Summary Adult granulosa cell tumors (AGCTs), characterized by a specific point mutation (C134W) in the gene FOXL2, are less than 5% of all the diagnosed ovarian cancers. Surgery is the cornerstone treatment for AGCT even at relapse, with systemic therapy showing poor results. The aim of our study is to explore the potential therapeutic effect of the elimination of the C134W mutation. To achieve our goal, we have eliminated the mutant allele using CRISPR technology. Our results demonstrate that CRISPR-mediated elimination of FOXL2-C134W reduces the malignant phenotype of granulosa tumor cells, which change their transcriptional, proteomic, and cellular phenotype to a wild type like, granulosa type. Moreover, the induced changes allowed us to find new compounds with antitumoral activity. This work highlights the therapeutic potential of CRISPR mediated technology for the treatment of AGCT.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/634dedb74022a549efd5ac39c51b17c70bcdf23c" target='_blank'>
              CRISPR targeting of FOXL2 c.402C>G mutation reduces malignant phenotype in granulosa tumor cells and identifies anti-tumoral compounds
              </a>
            </td>
          <td>
            Sandra Amarilla-Quintana, Paloma Navarro, Alejandra Ramos, A. Montero‐Calle, P. Cabezas-Sáinz, Maria J Barrero, D. Megías, Borja Vilaplana-Martí, Carolina Epifano, Déborah Gómez-Domínguez, Iván Hernández, S. Monzón, Isabel Cuesta, Laura Sánchez, R. Barderas, J. García-Donas, Alberto Martín, Ignacio Pérez de Castro
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/435232aea775847959662a12da4da5a9a69c73c7" target='_blank'>
              A whole-genome CRISPR screen identifies the spindle accessory checkpoint as a locus of nab-paclitaxel resistance in a pancreatic cancer cell line
              </a>
            </td>
          <td>
            Priya Mondal, George Alyateem, Allison V. Mitchell, Michael M. Gottesman
          </td>
          <td>2024-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acquired chemoresistance remains a significant challenge in the clinics as most of the treated cancers eventually emerge as hard-to-treat phenotypes. Therefore, identifying chemoresistance targets is highly warranted to manage the disease better. In this study, we employed a label-free LC-MS/MS-based quantitative proteomics analysis to identify potential targets and signaling pathways underlying acquired chemoresistance in a sub-cell line (A549DR) derived from the parental lung adenocarcinoma cells (A549) treated with gradually increasing doses of doxorubicin (DOX). Our proteomics analysis identified 146 upregulated and 129 downregulated targets in A549DR cells. The KEGG pathway and Gene ontology (GO) analysis of differentially expressed upregulated and downregulated proteins showed that most abundant upregulated pathways were related to metabolic pathways, cellular senescence, cell cycle, and p53 signaling. Meanwhile, the downregulated pathways were related to spliceosome, nucleotide metabolism, DNA replication, nucleotide excision repair, and nuclear-cytoplasmic transport. Further, STRING analysis of upregulated biological processes showed a protein-protein interaction (PPI) between CDK1, AKT2, SRC, STAT1, HDAC1, FDXR, FDX1, NPC1, ALDH2, GPx1, CDK4, and B2M, proteins. The identified proteins in this study might be the potential therapeutic targets for mitigating DOX resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/974ec76bceeb820f708d654bf77904bfd435f4c2" target='_blank'>
              Identification of chemoresistance targets in doxorubicin-resistant lung adenocarcinoma cells using LC-MS/MS-based proteomics.
              </a>
            </td>
          <td>
            Nuzhat Bano, K. Kainat, Mohammad Imran Ansari, Anjali Pal, Sana Sarkar, Pradeep Kumar Sharma
          </td>
          <td>2024-08-05</td>
          <td>Journal of chemotherapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Understanding the evolutionary potential of mutations in gene regulatory networks is essential to furthering the study of evolution and development. However, in multicellular systems, genetic manipulation of regulatory networks in a targeted and high-throughput way remains challenging. In this study, we designed TF-High-Evolutionary (HighEvo), a transcription factor (TF) fused with a base editor (activation-induced deaminase), to continuously induce germline mutations at TF-binding sites across regulatory networks in Drosophila. Populations of flies expressing TF-HighEvo in their germlines accumulated mutations at rates an order of magnitude higher than natural populations. Importantly, these mutations accumulated around the targeted TF-binding sites across the genome, leading to distinct morphological phenotypes consistent with the developmental roles of the tagged TFs. As such, this TF-HighEvo method allows the interrogation of the mutational space of gene regulatory networks at scale and can serve as a powerful reagent for experimental evolution and genetic screens focused on the regulatory genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2f5224e39a793a9098b6100dcfef52f5acedf2" target='_blank'>
              TF-High-Evolutionary: In Vivo Mutagenesis of Gene Regulatory Networks for the Study of the Genetics and Evolution of the Drosophila Regulatory Genome
              </a>
            </td>
          <td>
            Xueying C. Li, Vani Srinivasan, Ian Laiker, Natalia Misunou, Nicolás Frankel, Luisa F. Pallares, Justin Crocker
          </td>
          <td>2024-08-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671f339111b7b478e20a7b9554c095e50b64b22b" target='_blank'>
              Cellular and molecular events in colorectal cancer: biological mechanisms, cell death pathways, drug resistance and signalling network interactions
              </a>
            </td>
          <td>
            Lei Yan, Jia Shi, Jiazuo Zhu
          </td>
          <td>2024-07-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Currently, cancer is still a leading cause of human death globally. Tumor deterioration comprises multiple events including metastasis, therapeutic resistance and immune evasion, all of which are tightly related to the phenotypic plasticity especially epithelial–mesenchymal plasticity (EMP). Tumor cells with EMP are manifest in three states as epithelial–mesenchymal transition (EMT), partial EMT, and mesenchymal–epithelial transition, which orchestrate the phenotypic switch and heterogeneity of tumor cells via transcriptional regulation and a series of signaling pathways, including transforming growth factor‐β, Wnt/β‐catenin, and Notch. However, due to the complicated nature of EMP, the diverse process of EMP is still not fully understood. In this review, we systematically conclude the biological background, regulating mechanisms of EMP as well as the role of EMP in therapy response. We also summarize a range of small molecule inhibitors, immune‐related therapeutic approaches, and combination therapies that have been developed to target EMP for the outstanding role of EMP‐driven tumor deterioration. Additionally, we explore the potential technique for EMP‐based tumor mechanistic investigation and therapeutic research, which may burst vigorous prospects. Overall, we elucidate the multifaceted aspects of EMP in tumor progression and suggest a promising direction of cancer treatment based on targeting EMP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32fcd891c5d88b251baf8fc511031710e0fc1629" target='_blank'>
              Epithelial–mesenchymal plasticity in cancer: signaling pathways and therapeutic targets
              </a>
            </td>
          <td>
            Xiangpeng Wang, Xiao-yong Xue, Mingshi Pang, Liuchunyang Yu, Jinxiu Qian, Xiaoyu Li, Meng Tian, Aiping Lyu, Cheng Lu, Yuanyan Liu
          </td>
          <td>2024-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background In the absence of targeted mutations and immune checkpoints, platinum-based chemotherapy remains a gold standard agent in the treatment of patients with lung squamous cell carcinoma (LUSC). However, cisplatin resistance greatly limits its therapeutic efficacy and presents challenges in the treatment of lung cancer patients. Therefore, the potential clinical needs for this research focus on identifying novel molecular signatures to further elucidate the underlying mechanisms of cisplatin resistance in LUSC. A growing body of evidence indicates that alternative splicing (AS) events significantly influence the tumor progression and survival of patients with LUSC. However, there are few systematic analyses of AS reported in LUSC. This study aims to explore the role of messenger RNA (mRNA), microRNA (miRNA), and AS in predicting prognosis in patients with cisplatin-resistant LUSC and provide potential therapeutic targets and drugs. Methods Gene expression and miRNA expression, using RNA sequencing (RNA-seq), and SpliceSeq data were downloaded from The Cancer Genome Atlas (TCGA) database. The least absolute shrinkage and selection operator (LASSO) Cox regression analysis were used to construct predictive models. Kaplan-Meier survival analyses were used to evaluate patients’ prognosis. Single-sample gene set enrichment analysis (ssGSEA) conducted via the R package “GSEAbase” was used to evaluate the immune-related characteristics. Immunohistochemistry was used to examine protein expression. The Connectivity Map (CMap) database was used to screen for potential drugs. The 3-(4,5)-dimethylthiahiazo (-z-y1)-3,5-di-phenytetrazoliumromide (MTT) assay was used to determine and calculate the half-maximal inhibitory concentration (IC50) of the drugs, sulforaphane and parthenolide. Results In this study, bioinformatics were used to identify mRNAs, miRNAs, and AS events related to response to cisplatin and to establish an integrated prognostic signature for 70 patients with LUSC and cisplatin resistance. The prognostic signature served as an independent prognostic factor with high accuracy [hazard ratio (HR) =2.346, 95% confidence interval (CI): 1.568–3.510; P<0.001], yielding an area under the curve (AUC) of 0.825, 0.829, and 0.877 for 1-, 3-, and 5-year survival, respectively. It also demonstrated high predictive performance in this cohort of patients with LUSC, with an AUC of 0.734, 0.767, and 0.776 for 1-, 3-, and 5-year survival, respectively. This integrated signature was also found to be an independent indicator among conventional clinical features (HR =2.288, 95% CI: 1.547–3.383; P<0.001). In addition, we analyzed the correlation of the signature with immune infiltration and identified several small-molecule drugs that had the potential to improve the survival of patients with LUSC. Conclusions This study provides a framework for the mRNA-, miRNA-, and AS-based evaluation of cisplatin response and several potential therapeutic drugs for targeting cisplatin resistance in LUSC. These findings may serve as a theoretical basis for the clinical alleviation of cisplatin resistance and thus help to improve treatment responses to chemotherapy in patients with LUSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9e4be62d5ffa939a89b32b21f09c073e553ca1" target='_blank'>
              Identification of a prognostic gene signature in patients with cisplatin resistant squamous cell lung cancer
              </a>
            </td>
          <td>
            Yi Mu, Yinan Dong, Mingyang Zheng, Martin P. Barr, G. Roviello, Zhihuang Hu, Jia Liu
          </td>
          <td>2024-07-01</td>
          <td>Journal of Thoracic Disease</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Melanoma being one of the most common and deadliest skin cancers, has been rising since the past decade. Patients at advanced stages of the disease have very poor prognoses, as opposed to at the earlier stages. Nowadays the standard-of-care of advanced melanoma is resection followed by immune checkpoint inhibition based immunotherapy. However, a substantial proportion of patients either do not respond or develop resistances. This underscores a need for novel approaches and therapeutic targets as well as a better understanding of the mechanisms of melanoma pathogenesis. Long non-coding RNAs (lncRNAs) comprise a poorly characterized class of functional players and promising targets in promoting malignancy. Certain lncRNAs have been identified to play integral roles in melanoma progression and drug resistances, however systematic screens to uncover novel functional lncRNAs are scarce. Here, we profile differentially expressed lncRNAs in patient derived short-term metastatic cultures and BRAF-MEK-inhibition resistant cells. We conduct a focused growth-related CRISPR-inhibition screen of overexpressed lncRNAs, validate and functionally characterize lncRNA hits with respect to cellular growth, invasive capacities and apoptosis in vitro as well as the transcriptomic impact of our lead candidate the novel lncRNA XLOC_030781. In sum, we extend the current knowledge of ncRNAs and their potential relevance on melanoma. Significance Previously considered as transcriptional noise, lncRNAs have emerged as novel players in regulating many cellular aspects in health and disease including melanoma. However, the number and as well as the extent of functional significance of most lncRNAs remains elusive. We provide a comprehensive strategy to identify functionally relevant lncRNAs in melanoma by combining expression profiling with CRISPR-inhibition growths screens lowering the experimental effort. We also provide a larger resource of differentially expressed lncRNAs with potential implications in melanoma growth and invasion. Our results broaden the characterized of lncRNAs as potential targets for future therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768f16ebbf2c3ec424c154b1baf19ad0e931615" target='_blank'>
              CRISPR-inhibition screen for lncRNAs linked to melanoma growth and metastasis
              </a>
            </td>
          <td>
            Stavroula Petroulia, K. Hockemeyer, Shashank Tiwari, Pietro Berico, Sama Shamloo, Seyedeh Elnaz Banijamali, Eleazar Vega-Saenz de Miera, Yixiao Gong, Palaniraja Thandapani, Eric Wang, Michael Schulz, A. Tsirigos, Iman Osman, Ioannis D. Aifantis, Jochen Imig
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Deficient (d) DNA mismatch repair (MMR) is a biomarker predictive of better response to PD-1 blockade immunotherapy in solid tumors. dMMR can be caused by mutations in MMR genes or by protein inactivation, which can be detected by sequencing and immunohistochemistry, respectively. To investigate the role of dMMR in diffuse large B-cell lymphoma (DLBCL), MMR gene mutations and expression of MSH6, MSH2, MLH1, and PMS2 proteins were evaluated by targeted next-generation sequencing and immunohistochemistry in a large cohort of DLBCL patients treated with standard chemoimmunotherapy, and correlated with the tumor immune microenvironment characteristics quantified by fluorescent multiplex immunohistochemistry and gene-expression profiling. The results showed that genetic dMMR was infrequent in DLBCL and was significantly associated with increased cancer gene mutations and favorable immune microenvironment, but not prognostic impact. Phenotypic dMMR was also infrequent, and MMR proteins were commonly expressed in DLBCL. However, intratumor heterogeneity existed, and increased DLBCL cells with phenotypic dMMR correlated with significantly increased T cells and PD-1+ T cells, higher average nearest neighbor distance between T cells and PAX5+ cells, upregulated immune gene signatures, LE4 and LE7 ecotypes and their underlying Ecotyper-defined cell states, suggesting the possibility that increased T cells targeted only tumor cell subsets with dMMR. Only in patients with MYC¯ DLBCL, high MSH6/PMS2 expression showed significant adverse prognostic effects. This study shows the immunologic and prognostic effects of genetic/phenotypic dMMR in DLBCL, and raises a question on whether DLBCL-infiltrating PD-1+ T cells target only tumor subclones, relevant for the efficacy of PD-1 blockade immunotherapy in DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398a69413a5280f59d02312c6f8460b206f61fde" target='_blank'>
              DNA mismatch repair defect and intratumor heterogeneous deficiency differently impact immune responses in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Z. Xu-Monette, Cancan Luo, Li Yu, Yong Li, Govind Bhagat, A. Tzankov, Carlo Visco, Xiangshan Fan, K. Dybkaer, Ali Sakhdari, Nicholas T Wang, Alyssa F. Yuan, April Chiu, Wayne Tam, Y. Zu, Eric D. Hsi, Anamarija M. Perry, Wenting Song, Dennis O'Malley, Q. Au, H. Nunns, H. Go, Michael B. Møller, Benjamin Marshall Parsons, S. Montes-Moreno, M. Ponzoni, A. Ferreri, A. Sohani, J. Abramson, Bing Xu, Ken H. Young
          </td>
          <td>2024-08-05</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Background Relapsed/refractory acute myeloid leukemia (R/R-AML) has dismal prognosis due to chemotherapy resistance. Circular RNAs (circRNAs) have shown emerging roles in chemotherapy resistance in various cancers including hematologic malignancies. However, the potential roles of circRNAs in AML progression and drug resistance remain largely undetermined. Methods In this study, circulating circRNAs expression profiles were analyzed among R/R-AML, de novo AML and healthy controls (HC) using a human circRNA Array. Bioinformatic analysis was carried out to explore the differentially expressed circRNAs (DE-circRNAs). GO, KEGG pathway analysis, along with circRNA-miRNA-mRNA network analysis, were conducted to identify the potential biological pathways involved in R/R-AML. Finally, the UALCAN database was used to assess the prognosis of different target DE-circRNAs-related mRNAs. Results Forty-eight DE-circRNAs were upregulated, whereas twenty-seven DE-circRNAs were downregulated in R/R-AML samples. Up-regulated DE-circRNAs in R/R-AML samples were mainly enrichment in the biological processes and pathways of cell migration, microRNAs in cancers, Rap1 and Ras signaling pathways. Six DE-circRNAs were randomly selected to further explore their relationships with R/R-AML. GO and KEGG pathway analyses of the six candidate DE-circRNAs-related target mRNAs were mainly involved in the regulation of signal transduction and Ras signaling pathway. By overlapping our RNA-sequencing results of differentially expressed genes (DEGs) in R/R-AML samples with the candidate DE-circRNAs-predicted target mRNAs, we identified sixty-eight overlapping targeted mRNAs. Using UALCAN database analysis, we identified that AML patients with six upregulated DE-circRNA-related genes (ECE1, PI4K2A, SLC9A6, CCND3, PPP1R16B, and TRIM32) and one downregulated gene DE-circRNA-related genes (ARHGAP10) might have a poor prognosis. Conclusion This study revealed the overall alterations of circRNAs in R/R-AML. DE-circRNAs and their related genes might be used as potential early, sensitive and stable biomarkers for AML diagnosis, R/R-AML monitoring, and even as novel treatment targets for R/R-AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68cf9c31c5780881aca1b710dce6fd3cd806578f" target='_blank'>
              Screening of a Prognostic Gene Signature for Relapsed/Refractory Acute Myeloid Leukemia Based on Altered Circulating CircRNA Profiles
              </a>
            </td>
          <td>
            Honggang Guo, Yabin Cui, Yanliang Bai, Fan Yan, Wenhui Zhang, Yu-qing Chen, M. Shi
          </td>
          <td>2024-07-01</td>
          <td>International Journal of General Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="A central challenge in the quest for precise gene regulation within mammalian cells is the development of regulatory networks that can achieve perfect adaptation-where outputs consistently return to a set baseline post-stimulus. Here, we present such a system that leverages the CRISPR activation (CRISPRa) and anti-CRISPR proteins as two antithetic elements to establish perfect adaptation in mammalian cells and dynamically regulate gene expression. We demonstrate that this system can maintain stable expression levels of target genes in the face of external perturbations, thus providing a robust platform for biological applications. The versatility of our system is further showcased through its integration with endogenous regulatory mechanisms in T cells, such as the NF-κB-mediated immune response, and its ability to program apoptosis responses for precise spatial and temporal control of cellular growth and death. This study not only advances our understanding of gene regulation in mammalian cells but also opens new avenues for therapeutic intervention, particularly in diseases characterized by dysregulated gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9bdcf363536b5c7b6585a226adb59f423a716ec" target='_blank'>
              CRISPR perfect adaptation for robust control of cellular immune and apoptotic responses.
              </a>
            </td>
          <td>
            Yichi Zhang, Shuyi Zhang
          </td>
          <td>2024-08-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Kirsten Rat Sarcoma (KRAS) is the most commonly mutated oncogene in colorectal carcinoma (CRC). We have previously reported the interactions between microsatellite instability (MSI), DNA promoter methylation, and gene expression. In this study, we looked for associations between KRAS mutation, gene expression, and methylation that may help with precision medicine. Genome-wide gene expression and DNA methylation were done in paired CRC tumor and surrounding healthy tissues. The results suggested that (a) the magnitude of dysregulation of many major gene pathways in CRC was significantly greater in patients with the KRAS mutation, (b) the up- and down-regulation of these dysregulated gene pathways could be correlated with the corresponding hypo- and hyper-methylation, and (c) the up-regulation of CDKN2A was more pronounced in tumors with the KRAS mutation. A recent cell line study showed that there were higher CDKN2A levels in 5-FU-resistant CRC cells and that these could be down-regulated by Villosol. Our findings suggest the possibility of a better response to anti-CDKN2A therapy with Villosol in KRAS-mutant CRC. Also, the more marked up-regulation of genes in the proteasome pathway in CRC tissue, especially with the KRAS mutation and MSI, may suggest a potential role of a proteasome inhibitor (bortezomib, carfilzomib, or ixazomib) in selected CRC patients if necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40b3e427194ea41aedea0dfcb0cf1b6c9f955b47" target='_blank'>
              Association of KRAS Mutation and Gene Pathways in Colorectal Carcinoma: A Transcriptome- and Methylome-Wide Study and Potential Implications for Therapy
              </a>
            </td>
          <td>
            F. Jasmine, Armando Almazan, Yuliia Khamkevych, Marc Bissonnette, Habibul Ahsan, M. Kibriya
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a type of blood cancer with diverse genetic variations and DNA methylation alterations. By studying the interaction of gene mutations, expression, and DNA methylation, we aimed to gain valuable insights into the processes that lead to block differentiation in AML. We analyzed TCGA-LAML data (173 samples) with RNA sequencing and DNA methylation arrays, comparing FLT3 mutant (48) and wild-type (125) cases. We conducted differential gene expression analysis using cBioPortal, identified DNA methylation differences with ChAMP tool, and correlated them with gene expression changes. Gene set enrichment analysis (g:Profiler) revealed significant biological processes and pathways. ShinyGo and GeneCards were used to find potential transcription factors and their binding sites among significant genes. We found significant differentially expressed genes (DEGs) negatively correlated with their most significant methylation probes (Pearson correlation coefficient of -0.49, P-value <0.001) between FLT3 mutant and wild-type groups. Moreover, our exploration of 450 k CpG sites uncovered a global hypo-methylated status in 168 DEGs. Notably, these methylation changes were enriched in the promoter regions of Homebox superfamily gene, which are crucial in transcriptional-regulating pathways in blood cancer. Furthermore, in FLT3 mutant AML patient samples, we observed overexpress of WT1, a transcription factor known to bind homeobox gene family. This finding suggests a potential mechanism by which WT1 recruits TET2 to demethylate specific genomic regions. Integrating gene expression and DNA methylation analyses shed light on the impact of FLT3 mutations on cancer cell development and differentiation, supporting a two-hit model in AML. This research advances understanding of AML and fosters targeted therapeutic strategy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d3ff3b479ed6f746656e6ba790e6039d0322f3" target='_blank'>
              Crosstalk between genomic variants and DNA methylation in FLT3 mutant acute myeloid leukemia.
              </a>
            </td>
          <td>
            Bac Dao, Van Ngu Trinh, H. Nguyen, Hoa L. Nguyen, Thuc Duy Le, Phuc Loi Luu
          </td>
          <td>2024-06-30</td>
          <td>Briefings in functional genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background. Lung adenocarcinoma (LUAD) is the most common subtype of non-small cell lung cancer (NSCLC) and frequently affects non-smokers, especially women. It is characterized by a complex genetic profile and interactions with its microenvironment, which contribute to its aggressive and adaptable nature. Early symptoms are often subtle, leading to late diagnoses. Treatment approaches have advanced with targeted therapies and immunotherapy supplementing traditional chemotherapy and radiation. Despite these advancements, the prognosis remains variable, highlighting the need for continued research into new treatment strategies to improve outcomes. Method. In this study, we employed Single-cell RNA Sequencing (scRNA-seq) to comprehensively analyze the impact of chemotherapy on lung adenocarcinoma at the individual cell level. By comparing before and after treatment samples, we assessed the differential expression of genes and pathways, revealing insights into how different cell types within the tumour respond to chemotherapy. This approach enabled us to pinpoint specific mechanisms of drug resistance and highlight potential therapeutic targets for overcoming these challenges. Results. Our analysis uncovered substantial changes in gene expression between primary tumour cells and metastatic cells following chemotherapy. Notably, we observed that 45 pathways were shared between the top 50 upregulated pathways in the primary tumour and the top 50 downregulated pathways in the metastatic tumour post-chemotherapy. Conversely, there was no overlap between the top 50 downregulated pathways in the primary tumour and the top 50 upregulated pathways in the metastatic tumour after chemotherapy. This suggests that chemotherapy effectively downregulated the major upregulated pathways but did not upregulate the key downregulated pathways in metastatic tumours. Conclusions. Integrating single-cell transcriptomics into LUAD research offers detailed insights into the tumour's response to chemotherapy and its interaction with the immune system. This approach enhances our understanding of LUAD and aids in developing targeted and effective treatments. Based on our analysis, we hypothesize that combining chemotherapy with drugs designed to upregulate the downregulated pathways in primary tumour cells could significantly enhance treatment efficacy and improve patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c9748c84cdcd0c997de55e9e674c83813950dc2" target='_blank'>
              Unveiling Chemotherapy's Impact on Lung Cancer through Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            MD Saed Sayad, PhD Mark Hiatt, Mba MS Hazem Mustafa Md
          </td>
          <td>2024-07-10</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Therapy-induced acquired resistance limits the clinical effectiveness of mutation-specific RAS inhibitors in colorectal cancer. It is unknown whether broad-spectrum active-state RAS inhibitors meet similar limitations. Here, we identify and categorize mechanisms of resistance to the broad-spectrum active-state RAS inhibitor RMC-7977 in colorectal cancer cell lines. We found that KRAS-mutant colorectal cancer cell lines are universally sensitive to RMC-7977, inhibiting the RAS-RAF-MEK-ERK axis, halting proliferation and in some cases inducing apoptosis. To monitor KRAS downstream effector pathway activity, we developed a compartment-specific dual-color ERK activity reporter. RMC-7977 treatment reduced reporter activity. However, long-term dose escalation with RMC-7977 revealed multiple patterns of reporter reactivation in emerging resistant cell populations that correlated with phosphorylation states of compartment-specific ERK targets. Cells sorted for high, low, or cytoplasmic reporter activity exhibited distinct patterns of genomic mutations, phospho-protein, and transcriptional activities. Notably, all resistant subpopulations showed dynamic ERK regulation in the presence of the RAS inhibitor, unlike the parental sensitive cell lines. High levels of RAS downstream activities were observed in cells characterized by a KRAS Y71H resistance mutation. In contrast, RAS inhibitor-resistant populations with low, or cytoplasmic ERK reporter reactivation displayed different genetic alterations, among them RAF1 S257L and S259P mutations. Colorectal cancer cells resistant to RMC-7977 and harboring the RAF1 mutation specifically exhibited synergistic sensitivity to concurrent RAS and RAF inhibition. Our findings endorse reporter-assisted screening together with single-cell analyses as a powerful approach for dissecting the complex landscape of therapy resistance. The strategy offers opportunities to develop clinically relevant combinatorial treatments to counteract emergence of resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58f2dfe945929a08a6260606b0df62f40daad49b" target='_blank'>
              Reporter-based screening identifies RAS-RAF mutations as drivers of resistance to active-state RAS inhibition in colorectal cancer
              </a>
            </td>
          <td>
            Oleksandra Aust, Eric Blanc, Mareen Lüthen, Viola Hollek, Rosario Astaburuaga-García, B. Klinger, Alexandra Trinks, D. Beule, Björn Papke, David Horst, Nils Blüthgen, Christine Sers, Channing Der, M. Morkel
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSC) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosomes 20q or 1q, both known for conferring in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings show higher loads of both CNVs and SNVs at later passages which are due to the cumulative acquisition of mutations over a longer time in culture and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs follow the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid cancer-associated mutations. This suggests that the recurrent chromosomal abnormalities are the potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f552b60bd87e180c4d6a5dd522428893e4b81b" target='_blank'>
              De novo cancer mutations frequently associate with recurrent chromosomal abnormalities during long-term human pluripotent stem cell culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huygebaert, M. Regin, Chi Mai Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cancer is a complex and heterogeneous disease, in which a number of genetic and epigenetic changes occur in tumor onset and progression. Recent studies indicate that changes at the RNA level are also involved in tumorigenesis, such as adenosine-to-inosine (A-to-I) RNA editing. Here, we systematically investigate transcriptome-wide A-to-I editing events in a large number of samples from Non-Hodgkin lymphomas (NHLs). Using a computational pipeline that determines significant differences in editing level between NHL and normal samples at known A-to-I editing sites, we identify a number of differentially edited editing sites between NHL subtypes and normal samples. Most of the differentially edited sites are located in non-coding regions, and many such sites show a strong correlation between gene expression level and editing efficiency, indicating that RNA editing might have direct consequences for the cancer cell’s aberrant gene regulation status in these cases. Moreover, we establish a strong link between RNA editing and NHL by demonstrating that NHL and normal samples and even NHL subtypes can be distinguished based on genome-wide RNA editing profiles alone. Our study establishes a strong link between RNA editing, cancer and aberrant gene regulation in NHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3709fa38f5f1119b1fcb4db3f2e4cd77244ecc08" target='_blank'>
              A-to-I Editing Is Subtype-Specific in Non-Hodgkin Lymphomas
              </a>
            </td>
          <td>
            Cai Chen, Ralf Bundschuh
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Purpose: Homologous recombination deficiency (HRD) impacts cancer treatment strategies, particularly the effectiveness of PARP inhibitors. However, the variability different HRD assays has hampered the selection of oncology patients who may benefit from these therapies. Our study aims to assess the whole genome landscape to better define HRD in a pan-cancer cohort and to contribute to harmonization of HRD detection. Methods: We employed a whole-genome sequencing WGS HRD classifier that included genome-wide features associated with HRD to analyze 580 tumor/normal paired pan-cancer samples. The HRD results were correlated retrospectively with treatment responses and were compared with commercial HRD tests in a subset of cases. Results: HRD phenotype was identified in 62 samples across various cancers including breast (19%), pancreaticobiliary (17%), gynecological (15%), prostate (8%), upper gastrointestinal (GI) (2%), and other cancers (1%). HRD cases were not confined to BRCA1/2 mutations; 24% of HRD cases were BRCA1/2 wild-type. A diverse range of HRR pathway gene alterations involved in HRD were elucidated, including biallelic mutations in FANCF, XRCC2, and FANCC, and deleterious structural variants. Comparison with results from commercial HRD assays suggests a better performance of WGS to detect HRD, based on treatment response. Conclusion: HRD is a biomarker used to determine which cancer patients would benefit from PARPi and platinum-based chemotherapy. However, a lack of harmonization of tests to determine HRD status makes it challenging to interpret their results. Our study highlights the use of comprehensive WGS analysis to predict HRD in a pan-cancer cohort, elucidates new genomic mechanisms associated with HRD, and enables an accurate identification of this phenotype, paving the way for improved outcomes in oncology care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80d347a4838be16d195accf2ef681565a4b6929" target='_blank'>
              Whole Genome Landscape Analysis of Homologous Recombination Deficiency in a Pan-Cancer Cohort
              </a>
            </td>
          <td>
            MD Majd Al Assaad, PhD Kevin Hadi, MS Max F. Levine, MD Daniela Guevara, Mph Minal Patel, MS Marvel Tranquille, BS Abigail King, BS John Otilano, MS Alissa Semaan, PhD Gunes Gundem, MS Juan S. Medina-Martínez, BS Michael Sigouros, Mph Jyothi Manohar, BS Hui-Hsuan Kuo, PhD David C. Wilkes, MD Eleni Andreopoulou, MD Eloise Chapman-Davis, MD Scott T. Tagawa, PhD Andrea Sboner, MD Allyson J. Ocean, MD Manish Shah, PhD Elli Papaemmanuil, Cora N. Sternberg, MD Kevin Holcomb, MD David M. Nanus, PhD Olivier Elemento, Juan Miguel Mosquera, J. Medina-Martinez
          </td>
          <td>2024-06-30</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Meiotic recombination is a fundamental process that generates genetic diversity by creating new combinations of existing alleles. Although human crossovers have been studied at the pedigree, population and single-cell level, the more frequent non-crossover events that lead to gene conversion are harder to study, particularly at the individual level. Here we show that single high-fidelity long sequencing reads from sperm can capture both crossovers and non-crossovers, allowing effectively arbitrary sample sizes for analysis from one male. Using fifteen sperm samples from thirteen donors we demonstrate variation between and within donors for the rates of different types of recombination. Intriguingly, we observe a tendency for non-crossover gene conversions to occur upstream of nearby PRDM9 binding sites, whereas crossover locations have a slight downstream bias. We further provide evidence for two distinct non-crossover processes. One gives rise to the vast majority of non-crossovers with mean conversion tract length under 50bp, which we suggest is an outcome of standard PRDM9-induced meiotic recombination. In contrast ∼2% of non-crossovers have much longer mean tract length, and potentially originate from the same process as complex events with more than two haplotype switches, which is not associated with PRDM9 binding sites and is also seen in somatic cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7a7057073d62495af1df781d99095c76dca90d" target='_blank'>
              Insights into non-crossover recombination from long-read sperm sequencing
              </a>
            </td>
          <td>
            Regev Schweiger, Sangjin Lee, Chenxi Zhou, Tsun-Po Yang, Katie Smith, Stacy Li, R. Sanghvi, Matthew Neville, Emily Mitchell, Ayrun Nessa, Sam Wadge, K. Small, Peter J Campbell, Peter H. Sudmant, R. Rahbari, R. Durbin
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: Bladder cancer (BLCA) is a highly aggressive and heterogeneous disease, posing challenges for diagnosis and treatment. Cancer immunotherapy has recently emerged as a promising option for patients with advanced and drug-resistant cancers. Fibroblasts, a significant component of the tumor microenvironment, play a crucial role in tumor progression, but their precise function in BLCA remains uncertain. Methods: Single-cell RNA sequencing (scRNA-seq) data for BLCA were obtained from the Gene Expression Omnibus database. The R package “Seurat” was used for processing scRNA-seq data, with uniform manifold approximation and projection (UMAP) for downscaling and cluster identification. The FindAllMarkers function identified marker genes for each cluster. Differentially expressed genes influencing overall survival (OS) of BLCA patients were identified using the limma package. Differences in clinicopathological characteristics, immune microenvironment, immune checkpoints, and chemotherapeutic drug sensitivity between high- and low-risk groups were investigated. RT-qPCR and immunohistochemistry validated the expression of prognostic genes. Results: Fibroblast marker genes identified three molecular subtypes in the testing set. A prognostic signature comprising ten genes stratified BLCA patients into high- and low-score groups. This signature was validated in one internal and two external validation sets. High-score patients exhibited increased immune cell infiltration, elevated chemokine expression, and enhanced immune checkpoint expression but had poorer OS and a reduced response to immunotherapy. Six sensitive anti-tumor drugs were identified for the high-score group. RT-qPCR and immunohistochemistry showed that CERCAM, TM4SF1, FN1, ANXA1, and LOX were highly expressed, while EMP1, HEYL, FBN1, and SLC2A3 were downregulated in BLCA. Conclusion: A novel fibroblast marker gene-based signature was established, providing robust predictions of survival and immunotherapeutic response in BLCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66f9a0c86d6c80279d0927f1ba673a71e962a0e6" target='_blank'>
              Integrating single-cell RNA-seq to identify fibroblast-based molecular subtypes for predicting prognosis and therapeutic response in bladder cancer
              </a>
            </td>
          <td>
            Jia Wang, Zhiyong Tan, Yinglong Huang, Charles Li, Peiqin Zhan, Haifeng Wang, Haihao Li
          </td>
          <td>2024-07-18</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="A major goal in biology is to uncover the relationship between genotype and phenotype. However, identifying gene function is often hampered by genetic redundancy. For example, under standard laboratory conditions, three-quarters of the genes in the human pathogen Streptococcus pneumoniae are non-essential. A powerful approach to unravel genetic redundancy is by identifying gene-gene interactions. To uncover genetic interactions (GIs) in S. pneumoniae on a genome-wide scale, a generally applicable dual CRISPRi-Seq method and associated analysis pipeline was developed. Specifically, we created a library of 869 dual sgRNAs targeting high-confidence operons that encode essential and non-essential genes, covering over 70% of the genetic elements in the pneumococcal genome. Testing these 378,015 unique combinations, 4,026 significant GIs were identified, including 1,935 negative and 2,091 positive interactions. Besides known GIs, we found and confirmed previously unknown interactions involving genes responsible for fundamental cellular processes such as cell division, cell shape maintenance, and chromosome segregation. The presented methods and bioinformatic approaches can serve as a roadmap for genome-wide gene interaction studies in other organisms. Lastly, all interactions are available for exploration via the Pneumococcal Genetic Interaction Network (PneumoGIN) at https://veeninglab.shinyapps.io/PneumoGIN, which can serve as a starting point for new biological discoveries and translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dd78fdd99cdc75fb8cdbb9e56ea5f0c3b73930" target='_blank'>
              Dual CRISPRi-Seq for genome-wide genetic interaction studies identifies key genes involved in the pneumococcal cell cycle
              </a>
            </td>
          <td>
            Julien Dénéréaz, Elise Eray, Bimal Jana, Vincent de Bakker, Horia Todor, Tim van Opijnen, Xue Liu, J. Veening
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/553f4560eb657fca2fe86385e460c71427df430c" target='_blank'>
              GLI1 confers resistance to PARP inhibitors by activating the DNA damage repair pathway.
              </a>
            </td>
          <td>
            Hiroshi Ikeuchi, Yusuke Matsuno, Rika Kusumoto-Matsuo, S. Kojima, Toshihide Ueno, Masachika Ikegami, Rina Kitada, Hitomi Sumiyoshi-Okuma, Yuki Kojima, K. Yonemori, Y. Yatabe, K. Takamochi, Kenji Suzuki, Ken‐ichi Yoshioka, H. Mano, S. Kohsaka
          </td>
          <td>2024-08-03</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/460b0c287af289dbf822999647d9d57a5f2960b9" target='_blank'>
              Integrative single-cell RNA-seq and spatial transcriptomics analyses reveal diverse apoptosis-related gene expression profiles in EGFR-mutated lung cancer
              </a>
            </td>
          <td>
            Motohiro Izumi, Masanori Fujii, I. Kobayashi, Vivian Ho, Y. Kashima, Hibiki Udagawa, Daniel B. Costa, Susumu S. Kobayashi
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Alterations in the FMS-like tyrosine kinase 3 (FLT3) gene are the most frequent driver mutations in acute myeloid leukaemia (AML), linked to a high risk of relapse in patients with internal tandem duplications (FLT3-ITD). Tyrosine kinase inhibitors (TKIs) targeting the FLT3 protein are approved for clinical use, yet resistance often emerges. This resistance is mainly seen following the acquisition of additional point mutations in the tyrosine kinase domain (TKD), resulting in a double mutant FLT3-ITD/TKD, which sustains cell signalling and survival despite the presence of FLT3 inhibitors. Here, we developed a FLT3-mutant AML model with adaptive resistance to type II TKIs, sorafenib, and quizartinib by in vitro drug selection. Through global multiomic profiling, we identified upregulation of proteins involved in reactive oxygen species (ROS) production, particularly NADPH-oxidases, driving cellular ’ROS-addiction’, with resistant cells relying on ROS for survival, and genome fidelity preserved by ATM-driven DNA repair. Transcriptomic analysis of adult and paediatric AML (pAML) patients identified high ATM expression as a biomarker for shorter median overall survival in both the de novo and relapsed settings. Inhibition of ATM with clinically relevant therapy WSD-0628 effectively killed TKI- and chemotherapy-resistant AML cells in vitro and significantly extended the survival of mice with sorafenib- and quizartinib-resistant FLT3-ITD AML in vivo. We propose a new treatment strategy to improve survival of patients who develop resistance to sorafenib and quizartinib, as well as relapsed and refractory pAML, exploiting resistance mechanisms to precision therapies and cell-intrinsic features of high-risk cases, highlighting a clinically relevant salvage strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0bcd4e64b15b98143cfc995f3b3d32e9c623b66" target='_blank'>
              Adaptive resistance to FLT3 inhibitors is potentiated by ROS-driven DNA repair signalling
              </a>
            </td>
          <td>
            Dilana E. Staudt, Zacary P. Germon, Abdul Mannan, Tabitha Mclachlan, Heather C Murray, Ryan J. Duchatel, Bryce C. Thomas, Tyrone S Beitaki, Holly P. McEwen, Mika L Persson, Leah Calvert, Izac J. Findlay, Evangeline R. Jackson, Nathan D. Smith, D. Skerrett‐Byrne, David Mossman, Brett Nixon, Geoffry De Iullis, Alicia M. Douglas, A. Enjeti, Jonathan R. Sillar, Janis Chamberlain, Frank Alvaro, Andrew H. Wei, Patrick Connerty, N. Verrills, Matthew D. Dun
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2256152e45d11bd82f68a7e3810e940b41cb02" target='_blank'>
              Essential gene screening identifies the bromodomain-containing protein BRPF1 as a new actionable target for endocrine therapy-resistant breast cancers
              </a>
            </td>
          <td>
            A. Salvati, G. Giurato, J. Lamberti, Ilaria Terenzi, Laura Crescenzo, Viola Melone, L. Palo, A. Giordano, Francesco Sabbatino, G. Roscigno, C. Quintavalle, G. Condorelli, Francesca Rizzo, Roberta Tarallo, G. Nassa, Alessandro Weisz
          </td>
          <td>2024-08-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b62bbc42b475a59548fce65b7db966d5dba12012" target='_blank'>
              CRISPRi-TnSeq maps genome-wide interactions between essential and non-essential genes in bacteria.
              </a>
            </td>
          <td>
            Bimal Jana, Xue Liu, Julien Dénéréaz, Hongshik Park, Dmitry Leshchiner, Bruce Liu, Clément Gallay, Junhao Zhu, J. Veening, Tim van Opijnen
          </td>
          <td>2024-07-19</td>
          <td>Nature microbiology</td>
          <td>1</td>
          <td>46</td>
        </tr>

        <tr id="Drug resistance is currently one of the biggest challenges in cancer treatment. With the deepening understanding of drug resistance, various mechanisms have been revealed, including metabolic reprogramming and alterations of redox balance. Notably, metabolic reprogramming mediates the survival of tumor cells in harsh environments, thereby promoting the development of drug resistance. In addition, the changes during metabolic pattern shift trigger reactive oxygen species (ROS) production, which in turn regulates cellular metabolism, DNA repair, cell death, and drug metabolism in direct or indirect ways to influence the sensitivity of tumors to therapies. Therefore, the intersection of metabolism and ROS profoundly affects tumor drug resistance, and clarifying the entangled mechanisms may be beneficial for developing drugs and treatment methods to thwart drug resistance. In this review, we will summarize the regulatory mechanism of redox and metabolism on tumor drug resistance and highlight recent therapeutic strategies targeting metabolic–redox circuits, including dietary interventions, novel chemosynthetic drugs, drug combination regimens, and novel drug delivery systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77cc6698aebc91736fc846603c2b274976902d8b" target='_blank'>
              Targeting Metabolic–Redox Nexus to Regulate Drug Resistance: From Mechanism to Tumor Therapy
              </a>
            </td>
          <td>
            Yuke Wang, Jingqiu He, Shan Lian, Yan Zeng, Sheng He, Jue Xu, Li Luo, Wenyong Yang, Jingwen Jiang
          </td>
          <td>2024-07-01</td>
          <td>Antioxidants</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="The search for genetic variants that act as causative factors in human diseases by disrupting the normal splicing process has primarily focused on single nucleotide variants (SNVs). It is worth noting that insertions or deletions (indels) have also been sporadically reported as causative disease variants through their potential impact on the splicing process. In this study, to perform identification of indels inducing exon extension/shrinkage events, we used individual-specific genomes and RNA sequencing (RNA-seq) data pertaining to the corresponding individuals and identified 12 exon extension/shrinkage events that were potentially induced by indels that disrupted authentic splice sites or created novel splice sites in 235 normal individuals. By evaluating the impact of these abnormal splicing events on the resulting transcripts, we found that five events led to the generation of premature termination codons (PTCs), including those occurring within genes associated with genetic disorders. Our analysis revealed that the potential functions of indels have been underexamined, and it is worth considering the possibility that indels may affect splice site usage, using RNA-seq data to discover novel potentially disease-associated mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62ba041d07636b21a672bb99a3107362cc90dc0c" target='_blank'>
              Identification and analysis of short indels inducing exon extension/shrinkage events.
              </a>
            </td>
          <td>
            Zhuo Qu, Narumi Sakaguchi, Chie Kikutake, Mikita Suyama
          </td>
          <td>2024-07-31</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In contrast to microarray methods, sequence-based technologies directly determine the nucleic acid sequence. A number of modern sequencing technologies are referred to collectively as "next-generation sequencing" (NGS), often known as "high-throughput sequencing." Compared to conventional Sanger sequencing, NGS gives orders of magnitude more data at a much lower ongoing cost. These new technologies allow for much faster and more affordable sequencing of DNA and RNA, revolutionizing the study of genomics and molecular biology. Technical improvements in NGS sequencing methods have rapidly increased sequencing volume to several billion nucleotides within a short period and at a reasonable cost. Currently, NGS is developing into a molecular microscope that is permeating almost all areas of biological research. The last ten years have seen the development of NGS platforms and methodologies, and the quality of the sequences has increased to the point where NGS is now utilized in human clinical diagnosis. Due to significant cost reductions and greater community acceptance of NGS, the utilization of NGS techniques in studying clinical trials has significantly increased. NGS is a useful tool for detecting mutations in people with cancer and genetic abnormalities. To ascertain whether NGS can cost-effectively improve patient outcomes, more thorough cost-effectiveness studies of NGS applied to patient care management are required.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01e4a56c187e0cf2684ef024ce554931a3b25b8e" target='_blank'>
              A Review of Next-Generation Sequencing Technologies and Their Impact on Clinical Research: Assessing Clinical Efficacy and Cost-Effectiveness
              </a>
            </td>
          <td>
            Pratibha Mambatta Sankaranarayanan, Saranya Kanukollu, Anand Babu Vangala, R. K. Vangala, Pramod Nair
          </td>
          <td>2024-07-02</td>
          <td>Trends in Current Biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic regulatory mechanism that plays a critical role in a variety of cellular processes, including the regulation of cell fate during development, maintenance of cell identity, and genome stability. DNA methylation is tightly regulated by enzymatic reactions and its deregulation plays an important role in the development of cancer. Specific DNA methylation alterations have been found in pediatric solid tumors, providing new insights into the development of these tumors. In addition, DNA methylation profiles have greatly contributed to tune the diagnosis of pediatric solid tumors and to define subgroups of patients with different risks of progression, leading to the reduction in unwanted toxicity and the improvement of treatment efficacy. This review highlights the dysregulated DNA methylome in pediatric solid tumors and how this information provides promising targets for epigenetic therapies, particularly inhibitors of DNMT enzymes (DNMTis). Opportunities and limitations are considered, including the ability of DNMTis to induce viral mimicry and immune signaling by tumors. Besides intrinsic action against cancer cells, DNMTis have the potential to sensitize immune-cold tumors to immunotherapies and may represent a remarkable option to improve the treatment of challenging pediatric solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1eac857dfcc5a5e3b672ddb17d6097e48bc70f" target='_blank'>
              Targeting DNA Methylation Machinery in Pediatric Solid Tumors
              </a>
            </td>
          <td>
            C. Cristalli, Katia Scotlandi
          </td>
          <td>2024-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Melanoma showcases a complex interplay of genetic alterations and intra- and inter-cellular morphological changes during metastatic transformation. While pivotal, the role of specific mutations in dictating these changes still needs to be fully elucidated. Telomerase promoter mutations (TERTp mutations) significantly influence melanoma’s progression, invasiveness, and resistance to various emerging treatments, including chemical inhibitors, telomerase inhibitors, targeted therapy, and immunotherapies. We aim to understand the morphological and phenotypic implications of the two dominant monoallelic TERTp mutations, C228T and C250T, enriched in melanoma metastasis. We developed isogenic clonal cell lines containing the TERTp mutations and utilized dual-color expression reporters steered by the endogenous Telomerase promoter, giving us allelic resolution. This approach allowed us to monitor morpholomic variations induced by these mutations. TERTp mutation-bearing cells exhibited significant morpholome differences from their wild-type counterparts, with increased allele expression patterns, augmented wound-healing rates, and unique spatiotemporal dynamics. Notably, the C250T mutation exerted more pronounced changes in the morpholome than C228T, suggesting a differential role in metastatic potential. Our findings underscore the distinct influence of TERTp mutations on melanoma’s cellular architecture and behavior. The C250T mutation may offer a unique morpholomic and systems-driven advantage for metastasis. These insights provide a foundational understanding of how a non-coding mutation in melanoma metastasis affects the system, manifesting in cellular morpholome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b66856ef0863e2a38d3da5fdaad88a280ff968" target='_blank'>
              Using deep learning to decipher the impact of telomerase promoter mutations on the dynamic metastatic morpholome
              </a>
            </td>
          <td>
            Andres J. Nevarez, Anusorn Mudla, Sabrina A. Diaz, Nan Hao
          </td>
          <td>2024-07-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Cancer cells often become resistant to treatment, making it harder to fight the disease effectively. This resistance can happen due to genetic changes, proteins that pump out drugs, or the environment around the tumor. Crizotinib is a drug that helps overcome this resistance in certain lung cancers and lymphomas by targeting specific proteins. It can also break down a harmful protein found in some cancers. Researchers have found that combining crizotinib with other drugs can improve its effectiveness, especially for cancers with specific genetic mutations. Abstract Chemoresistance is a major obstacle in cancer treatment, often leading to disease progression and poor outcomes. It arises through various mechanisms such as genetic mutations, drug efflux pumps, enhanced DNA repair, and changes in the tumor microenvironment. These processes allow cancer cells to survive despite chemotherapy, underscoring the need for new strategies to overcome resistance and improve treatment efficacy. Crizotinib, a first-generation multi-target kinase inhibitor, is approved by the FDA for the treatment of ALK-positive or ROS1-positive non-small cell lung cancer (NSCLC), refractory inflammatory (ALK)-positive myofibroblastic tumors (IMTs) and relapsed/refractory ALK-positive anaplastic large cell lymphoma (ALCL). Crizotinib exists in two enantiomeric forms: (R)-crizotinib and its mirror image, (S)-crizotinib. It is assumed that the R-isomer is responsible for the carrying out various processes reviewed here The S-isomer, on the other hand, shows a strong inhibition of MTH1, an enzyme important for DNA repair mechanisms. Studies have shown that crizotinib is an effective multi-kinase inhibitor targeting various kinases such as c-Met, native/T315I Bcr/Abl, and JAK2. Its mechanism of action involves the competitive inhibition of ATP binding and allosteric inhibition, particularly at Bcr/Abl. Crizotinib showed synergistic effects when combined with the poly ADP ribose polymerase inhibitor (PARP), especially in ovarian cancer harboring BRCA gene mutations. In addition, crizotinib targets a critical vulnerability in many p53-mutated cancers. Unlike its wild-type counterpart, the p53 mutant promotes cancer cell survival. Crizotinib can cause the degradation of the p53 mutant, sensitizing these cancer cells to DNA-damaging substances and triggering apoptosis. Interestingly, other reports demonstrated that crizotinib exhibits anti-bacterial activity, targeting Gram-positive bacteria. Also, it is active against drug-resistant strains. In summary, crizotinib exerts anti-tumor effects through several mechanisms, including the inhibition of kinases and the restoration of drug sensitivity. The potential of crizotinib in combination therapies is emphasized, particularly in cancers with a high prevalence of the p53 mutant, such as triple-negative breast cancer (TNBC) and high-grade serous ovarian cancer (HGSOC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c55a5da7409fe84d8ab6b88da7dafb5e0032e27" target='_blank'>
              Overcoming Chemoresistance in Cancer: The Promise of Crizotinib
              </a>
            </td>
          <td>
            Sanaa Musa, Noor Amara, Adan Selawi, Junbiao Wang, Cristina Marchini, A. Agbarya, J. Mahajna
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) has a poor survival rate for both pediatric and adult patients due to its frequent relapse. To elucidate the bioenergetic principle underlying AML relapse, we investigated the transcriptional regulation of mitochondrial–nuclear dual genomes responsible for metabolic plasticity in treatment-resistant blasts. Both the gain and loss of function results demonstrated that NFκB2, a noncanonical transcription factor (TF) of the NFκB (nuclear factor kappa-light-chain-enhancer of activated B cells) family, can control the expression of TFAM (mitochondrial transcription factor A), which is known to be essential for metabolic biogenesis. Furthermore, genetic tracking and promoter assays revealed that NFκB2 is in the mitochondria and can bind the specific “TTGGGGGGTG” region of the regulatory D-loop domain to activate the light-strand promoter (LSP) and heavy-strand promoter 1 (HSP1), promoters of the mitochondrial genome. Based on our discovery of NFκB2′s novel function of regulating mitochondrial–nuclear dual genomes, we explored a novel triplet therapy including inhibitors of NFκB2, tyrosine kinase, and mitochondrial ATP synthase that effectively eliminated primary AML blasts with mutations of the FMS-related receptor tyrosine kinase 3 (FLT3) and displayed minimum toxicity to control cells ex vivo. As such, effective treatments for AML must include strong inhibitory actions on the dual genomes mediating metabolic plasticity to improve leukemia prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd91cb6b195c34903e27bb3147d1ba2db984023c" target='_blank'>
              Discovery of NFκB2-Coordinated Dual Regulation of Mitochondrial and Nuclear Genomes Leads to an Effective Therapy for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Yi Xu, David J. Baylink, Jeffrey Xiao, Lily Tran, Vinh Nguyen, Brandon Park, Ismael Valladares, Scott Lee, Kevin Codorniz, Laren D Tan, Chien-Shing Chen, Hisham Abdel-Azim, Mark E. Reeves, Hamid Mirshahidi, Guido Marcucci, H. Cao
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ec02f56417ae90246c83ab25ae98a1c2c7e30" target='_blank'>
              Targeting treatment resistance: unveiling the potential of RNA methylation regulators and TG-101,209 in pan-cancer neoadjuvant therapy
              </a>
            </td>
          <td>
            Yaoyao Zhou, Ziyun Liu, Cheng Gong, Jie Zhang, Jing Zhao, Xia Zhang, Xiangyu Liu, Bin Li, Rui Li, Zhenyu Shi, Yongjie Xie, Li Bao
          </td>
          <td>2024-08-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Fungi can cause devastating invasive infections, typically in immunocompromised patients. Treatment is complicated both by the evolutionary similarity between humans and fungi and by the frequent emergence of drug resistance. Studies in fungal pathogens have long been slowed by a lack of high-throughput tools and community resources that are common in model organisms. Here we demonstrate a high-throughput transposon mutagenesis and sequencing (TN-seq) system in Cryptococcus neoformans that enables genome-wide determination of gene essentiality. We employed a random forest machine learning approach to classify the Cryptococcus neoformans genome as essential or nonessential, predicting 1,465 essential genes, including 302 that lack human orthologs. These genes are ideal targets for new antifungal drug development. TN-seq also enables genome-wide measurement of the fitness contribution of genes to phenotypes of interest. As proof of principle, we demonstrate the genome-wide contribution of genes to growth in fluconazole, a clinically used antifungal. We show a novel role for the well-studied RIM101 pathway in fluconazole susceptibility. We also show that 5’ insertions of transposons can drive sensitization of essential genes, enabling screenlike assays of both essential and nonessential components of the genome. Using this approach, we demonstrate a role for mitochondrial function in fluconazole sensitivity, such that tuning down many essential mitochondrial genes via 5’ insertions can drive resistance to fluconazole. Our assay system will be valuable in future studies of C. neoformans, particularly in examining the consequences of genotypic diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126840639a843aec8469a2a2562239beea433d1f" target='_blank'>
              Saturation transposon mutagenesis enables genome-wide identification of genes required for growth and fluconazole resistance in the human fungal pathogen Cryptococcus neoformans
              </a>
            </td>
          <td>
            R. B. Billmyre, Caroline J. Craig, Joshua Lyon, Claire Reichardt, Michael T. Eickbush, Sarah E. Zanders
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, Andre Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Introduction Breast cancer (BRCA) is a significant cause of cancer-associated mortality across the globe. Current therapeutic approaches face challenges such as drug resistance and metastasis. Immune signaling is triggered by chromosomal instability (CIN) generates misplaced DNA structures that activate the cyclic GMP–AMP synthase–stimulator of interferon genes (cGAS-STING) pathway, triggering. Studies have linked STING activation to BRCA treatment. Methods The bulk RNA-seq data for patients with BRCA were collected from the TCGA-BRCA cohort, GSE20685, and GSE96058 cohorts. STING pathway-related genes (SRGs) were obtained from the Reactome database. Differentially expressed genes were analyzed using the limma package. Immune cell infiltration was analyzed using the IOBR package. Gene Ontology biological processes, Kyoto Encyclopedia of Genes and Genomes pathways, and cancer hallmark pathways were analyzed using the MSigDB database. Prognostic models were prepared using the least absolute shrinkage and selection operator and multiple-factor Cox regression analysis. Single-cell analysis was performed using the Seurat and SCP pipeline. Results The expression patterns and clinical relevance of SRGs were analyzed in patients with BRCA. Transcriptional differences in the SRGs were observed between normal and tumorous tissues, with global down-regulated STING1 and up-regulated TBK1 in BRCA tissue. Tumor tissues were classified through consensus clustering analysis into two distinct groups, with differences in clinical characteristics and immune infiltration. A prognostic model related to the differences in STING pathway activity—high prognostic stratification potency—was developed and validated. Correlation analysis revealed suppressed overall immune activation in patients with BRCA having higher risk scores. Gemcitabine had a more favorable outcome in the low-risk group. The activity of the prognostic model at the single-cell level was confirmed through single-cell analysis, particularly in CD8 T cells and intratumor natural killer cells. Conclusion A STING pathway-related prognostic model developed and validated and the model could accurately predict BRCA patient outcomes. These findings have important implications for the personalized treatment and management of patients with BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dda048ba9b29653abf25b1272c86792b3a07e71" target='_blank'>
              Investigating the impact of STING pathway activation on breast cancer treatment outcomes: development and validation of a prognostic model
              </a>
            </td>
          <td>
            Yangyan Zhong, Hong Cao, Wei Li, Jian Deng, Dan Li, Junjie Deng
          </td>
          <td>2024-08-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Combined CDK4/6 inhibitor (CDK4/6i) and endocrine therapy significantly improves outcome for patients with estrogen receptor-positive (ER+) metastatic breast cancer, but drug resistance and thus disease progression inevitably occur. Herein, we aimed to identify genomic alterations associated with combined CDK4/6i and endocrine therapy resistance, and follow the levels of specific mutations in longitudinal circulating tumor DNA (ctDNA) for early detection of progression. From a cohort of 86 patients with ER+ metastatic breast cancer we performed whole exome sequencing or targeted sequencing of paired tumor (N = 8) or blood samples (N = 5) obtained before initiation of combined CDK4/6i and endocrine therapy and at disease progression. Mutations in oncogenic genes at progression were rare, while amplifications of growth-regulating genes were more frequent. The most frequently acquired alterations observed were PIK3CA and TP53 mutations and PDK1 amplification. Longitudinal ctDNA dynamics of mutant PIK3CA or private mutations revealed increased mutation levels at progression in 8 of 10 patients (80%). Impressively, rising levels of PIK3CA-mutated ctDNA were detected 4-17 months before imaging. Our data add to the growing evidence supporting longitudinal ctDNA analysis for real-time monitoring of CDK4/6i response and early detection of progression in advanced breast cancer. Further, our analysis suggests that amplification of growth-related genes may contribute to combined CDK4/6i and endocrine therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4f8f900b692158e74287adf0a693a5a4d08296" target='_blank'>
              Genomic alterations associated with resistance and circulating tumor DNA dynamics for early detection of progression on CDK4/6 inhibitor in advanced breast cancer.
              </a>
            </td>
          <td>
            Charlotte K Kindt, Carla L. Alves, S. Ehmsen, Amalie Kragh, T. Reinert, Marianne Vogsen, A. R. Kodahl, Jeanette D Rønlev, Dilan Ardik, Anna L Sørensen, Kirstine Evald, Mia L Clemmensen, J. Staaf, Henrik J Ditzel
          </td>
          <td>2024-08-11</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Patients with chronic lymphocytic leukemia (CLL) exhibit diverse clinical outcomes. An expanding array of genetic tests is now employed to facilitate the identification of patients with high‐risk disease and inform treatment decisions. These tests encompass molecular cytogenetic analysis, focusing on recurrent chromosomal alterations, particularly del(17p). Additionally, sequencing is utilized to identify TP53 mutations and to determine the somatic hypermutation status of the immunoglobulin heavy variable gene. Concurrently, a swift advancement of targeted treatment has led to the implementation of novel strategies for patients with CLL, including kinase and BCL2 inhibitors. This review explores both current and emerging diagnostic tests aimed at identifying high‐risk patients who should benefit from targeted therapies. We outline existing treatment paradigms, emphasizing the importance of matching the right treatment to the right patient beyond genetic stratification, considering the crucial balance between safety and efficacy. We also take into consideration the practical and logistical issues when choosing a management strategy for each individual patient. Furthermore, we delve into the mechanisms underlying therapy resistance and stress the relevance of monitoring measurable residual disease to guide treatment decisions. Finally, we underscore the necessity of aggregating real‐world data, adopting a global perspective, and ensuring patient engagement. Taken together, we argue that precision medicine is not the mere application of precision diagnostics and accessibility of precision therapies in CLL but encompasses various aspects of the patient journey (e.g., lifestyle exposures and comorbidities) and their preferences toward achieving true personalized medicine for patients with CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dc7d85ef5969ecec3da1b9d71510953a1f31440" target='_blank'>
              Realizing precision medicine in chronic lymphocytic leukemia: Remaining challenges and potential opportunities
              </a>
            </td>
          <td>
            K. Stamatopoulos, S. Pavlova, O. Al-Sawaf, T. Chatzikonstantinou, C. Karamanidou, G. Gaidano, F. Cymbalista, A. Kater, Andy Rawstron, L. Scarfò, P. Ghia, Richard Rosenquist
          </td>
          <td>2024-07-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>84</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, Amy Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Cancer drug resistance poses a significant obstacle to successful chemotherapy, primarily driven by the activity of ATP-binding cassette (ABC) transporters, which actively efflux chemotherapeutic agents from cancer cells, reducing their intracellular concentrations and therapeutic efficacy. Recent studies have highlighted the pivotal role of long noncoding RNAs (lncRNAs) in regulating this resistance, positioning them as crucial modulators of ABC transporter function. lncRNAs, once considered transcriptional noise, are now recognized for their complex regulatory capabilities at various cellular levels, including chromatin modification, transcription, and post-transcriptional processing. This review synthesizes current research demonstrating how lncRNAs influence cancer drug resistance by modulating the expression and activity of ABC transporters. lncRNAs can act as molecular sponges, sequestering microRNAs that would otherwise downregulate ABC transporter genes. Additionally, they can alter the epigenetic landscape of these genes, affecting their transcriptional activity. Mechanistic insights reveal that lncRNAs contribute to the activity of ABC transporters, thereby altering the efflux of chemotherapeutic drugs and promoting drug resistance. Understanding these interactions provides a new perspective on the molecular basis of chemoresistance, emphasizing the regulatory network of lncRNAs and ABC transporters. This knowledge not only deepens our understanding of the biological mechanisms underlying drug resistance but also suggests novel therapeutic strategies. In conclusion, the intricate interplay between lncRNAs and ABC transporters is crucial for developing innovative solutions to combat cancer drug resistance, underscoring the importance of continued research in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/964e0709fe20c21443ecd0c53a8a7ab6cf10b8c7" target='_blank'>
              lncRNAs: New players of cancer drug resistance via targeting ABC transporters.
              </a>
            </td>
          <td>
            Mohammad Ebrahimnezhad, Sanaz Hassanzadeh Asl, Maede Rezaie, Mehran Molavand, Bahman Yousefi, M. Majidinia
          </td>
          <td>2024-08-01</td>
          <td>IUBMB life</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c05cd3b7eb71af823cd51dee708a69c95084a142" target='_blank'>
              The role of lncRNA NEAT1 in human cancer chemoresistance
              </a>
            </td>
          <td>
            Feng Long, Xue Li, Jingyu Pan, Hailin Ye, Cuixia Di, Yong Huang, Jiawei Li, Xuan Zhou, Huiyi Yi, Qiaozhen Huang, Jing Si
          </td>
          <td>2024-07-05</td>
          <td>Cancer Cell International</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323dfa5e3ca434563b74541a5db544a4116607d2" target='_blank'>
              Disrupting YAP1-mediated glutamine metabolism induces synthetic lethality alongside ODC1 inhibition in osteosarcoma.
              </a>
            </td>
          <td>
            Hongsheng Wang, Yining Tao, Jing Han, Jiakang Shen, Haoran Mu, Zhuoying Wang, Jinzeng Wang, Xinmeng Jin, Qi Zhang, Yuqin Yang, Jun Lin, M. Sun, Xiao‐jun Ma, Ling Ren, A. K. LeBlanc, Jing Xu, Y. Hua, Wei Sun
          </td>
          <td>2024-08-08</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f8be9a3b2a2fa9867435314cacdcbe80a7eb4d" target='_blank'>
              Multiple mechanisms contribute to acquired TRAIL resistance in multiple myeloma
              </a>
            </td>
          <td>
            Fany V Ticona-Pérez, Xi Chen, A. Pandiella, Elena Díaz-Rodríguez
          </td>
          <td>2024-08-05</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is a highly prevalent and incurable form of cancer that arises from malignant plasma cells, with over 35,000 new cases diagnosed annually in the United States. While there are a growing number of approved therapies, MM remains incurable and nearly all patients will relapse and exhaust all available treatment options. Mechanisms for disease progression are unclear and in particular, little is known regarding the role of long non-coding RNAs (lncRNA) in mediating disease progression and response to treatment. In this study, we used transcriptome sequencing to compare newly diagnosed MM patients who had short progression- free survival (PFS) to standard first-line treatment (PFS < 24 months) to patients who had prolonged PFS (PFS > 24 months). We identified 157 differentially upregulated lncRNAs with short PFS and focused our efforts on characterizing the most upregulated lncRNA, LINC01432. We investigated LINC01432 overexpression and CRISPR/Cas9 knockdown in MM cell lines to show that LINC01432 overexpression significantly increases cell viability and reduces apoptosis, while knockdown significantly reduces viability and increases apoptosis, supporting the clinical relevance of this lncRNA. Next, we used individual-nucleotide resolution cross-linking immunoprecipitation with RT-qPCR to show that LINC01432 directly interacts with the RNA binding protein, CELF2. Lastly, we showed that LINC01432-targeted locked nucleic acid antisense oligonucleotides reduce viability and increases apoptosis. In summary, this fundamental study identified lncRNAs associated with short PFS to standard NDMM treatment and further characterized LINC01432, which inhibits apoptosis. Key points: lncRNA expression was found to be dysregulated in patients with short PFS to standard multiple myeloma therapy. LINC01432-bound CELF2 inhibits apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847cba66ef27ec89828bb55803b54ea81c4c449e" target='_blank'>
              LINC01432 binds to CELF2 in newly diagnosed multiple myeloma promoting short progression-free survival to standard therapy
              </a>
            </td>
          <td>
            Richa Mishra, Prasanth Thunuguntla, Alani Perkin, Dhanusha Duraiyan, Katelyn Bagwill, Savannah Gonzales, Vanessa Brizuela, Steve Daly, Yoon Jae Chang, Mahdote Abebe, Yash Rajana, Kelly Wichmann, Catheryn Bolick, Jaiyana King, M. Fiala, J. Fortier, R. Jayasinghe, Mark Schroeder, Li Ding, Ravi Vij, Jessica M. Silva-Fisher
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers.
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Simple Summary Neuregulin-1 (NRG1) is an important regulator of ErbB-mediated pathways involved in cancer development. Recently, there have been several studies analyzing NRG-1 gene fusions engaged in altering the dimerization of HER family proteins and the consecutive results of their activation in different types of cancer. Non-small cell lung cancer (NSCLC) patients can benefit from pan-HER inhibitors, and knowledge of NRG-1 fusions can help tailor the treatment to a specific group of patients. New drugs targeting cells with NRG-1 fusions are under clinical trials and show effectiveness in NSCLC treatment. Abstract Non-small cell lung cancer (NSCLC) presents a variety of druggable genetic alterations that revolutionized the treatment approaches. However, identifying new alterations may broaden the group of patients benefitting from such novel treatment options. Recently, the interest focused on the neuregulin-1 gene (NRG1), whose fusions may have become a potential predictive factor. To date, the occurrence of NRG1 fusions has been considered a negative prognostic marker in NSCLC treatment; however, many premises remain behind the targetability of signaling pathways affected by the NRG1 gene. The role of NRG1 fusions in ErbB-mediated cell proliferation especially seems to be considered as a main target of treatment. Hence, NSCLC patients harboring NRG1 fusions may benefit from targeted therapies such as pan-HER family inhibitors, which have shown efficacy in previous studies in various cancers, and anti-HER monoclonal antibodies. Considering the increased interest in the NRG1 gene as a potential clinical target, in the following review, we highlight its biology, as well as the potential clinical implications that were evaluated in clinics or remained under consideration in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302eaed4f1e7bdac2f2c9a0f86b9a6908f0e39da" target='_blank'>
              NRG1 Gene Fusions—What Promise Remains Behind These Rare Genetic Alterations? A Comprehensive Review of Biology, Diagnostic Approaches, and Clinical Implications
              </a>
            </td>
          <td>
            Tomasz Kucharczyk, M. Nicoś, Marek Kucharczyk, E. Kalinka
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a disorder of the monoclonal plasma cells and is the second most common hematologic malignancy. MM initiation and progression are dependent upon complex genomic abnormalities. The current pathogenic model of MM includes two types of primary events, represented by chromosome translocations or chromosome number alterations resulting in hyperdiploidy. These primary molecular events are observed both in MM and in monoclonal gammopathy, its premalignant precursor. Subsequent genetic events allow the progression of monoclonal gammopathy to MM and, together with primary events, contribute to the genetic complexity and heterogeneity of MM. Newer therapies have considerably improved patient outcomes; however, MM remains an incurable disease and most patients experience multiple relapses. The dramatic progresses achieved in the analysis of the heterogeneous molecular features of different MM patients allowed a comprehensive molecular classification of MM and the definition of an individualized prognostic model to predict an individual MM patient’s response to different therapeutic options. Despite these progresses, prognostic models fail to identify a significant proportion of patients destined to early relapse. Treatment strategies are increasingly. Based on disease biology, trials are enriched for high-risk MMs, whose careful definition and categorization requires DNA sequencing studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9bfe253151b2949dea161b10b654c58bbbd1b1" target='_blank'>
              Recent Advances in The Definition of the Molecular Alterations Occurring in Multiple Myeloma
              </a>
            </td>
          <td>
            Ugo Testa, E. Pelosi, G. Castelli, Giuseppe Leone
          </td>
          <td>2024-06-29</td>
          <td>Mediterranean Journal of Hematology and Infectious Diseases</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="PURPOSE
High-grade complex karyotype sarcomas are a heterogeneous group of tumors with a uniformly poor prognosis. Within complex karyotype sarcomas, there are innumerable genetic changes but identifying those that are clinically relevant has been challenging.


EXPERIMENTAL DESIGN
To address this, we utilized a pooled genetic screening approach, informed by TCGA data, to identify key drivers and modifiers of sarcoma development that were validated in vivo.


RESULTS
YAP1 and wildtype KRAS were validated as drivers and transformed human mesenchymal stem cells into two distinct sarcoma subtypes, undifferentiated pleomorphic sarcoma (UPS) and myxofibrosarcoma (MFS), respectively. A subset of tumors driven by CDK4 and PIK3CA reflected leiomyosarcoma (LMS) and osteosarcoma (OS) demonstrating the plasticity of this approach and the potential to investigate sarcoma subtype heterogeneity. All generated tumors histologically reflected human sarcomas and had increased aneuploidy as compared to simple karyotype sarcomas. Comparing differential gene expression of TCGA samples to model data identified increased oxidative phosphorylation signaling in YAP1 tumors. Treatment of a panel of soft tissue sarcomas with a combination of YAP1 and oxidative phosphorylation inhibitors led to significantly decreased viability.


CONCLUSIONS
Transcriptional co-analysis of TCGA patient samples to YAP1 and KRAS model tumors support that these sarcoma subtypes lie along a spectrum of disease and adds guidance for further transcriptome-based refinement of sarcoma subtyping. This approach can be used to begin to understand pathways and mechanisms driving human sarcoma development, the relationship between sarcoma subtypes and to identify and validate new therapeutic vulnerabilities for this aggressive and heterogeneous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdaada75e0a2b742920f7cff889cd677616ee1d5" target='_blank'>
              Genetic Screen in a Pre-Clinical Model of Sarcoma Development Defines Drivers and Therapeutic Vulnerabilities.
              </a>
            </td>
          <td>
            Jack Freeland, Maria Muñoz, Edmond O'Donnell, J. Langerman, Morgan Darrow, Jessica Bergonio, J. Suarez-Navarro, Steven Thorpe, Robert Canter, R. L. Randall, Kathrin Plath, Kermit L Carraway, Owen N. Witte, Thomas G. Graeber, Janai R. Carr-Ascher
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Linking disease phenotypes with molecular targets is key to the rational design of treatment interventions. Resistance to the chemotherapeutic cisplatin is one of the major factors limiting the clinical utility of this therapy, which is central to the treatment of a variety of solid malignancies. In this study, we couple the upregulation of a chemoresistant factor, the glutamate-cystine antiporter SLC7A11, with the addiction of cisplatin-resistant cancer cells to extracellular glutamine. In doing so, we thus provide a putative biomarker for this acquired metabolic dependency of chemoresistance. Subsequently, we evaluate various therapeutic strategies to selectively kill SLC7A11high cisplatin-resistant cancer cells, identifying cross-resistance to ferroptosis-inducing compounds and hypersensitivity to glutaminase inhibitor CB-839. We identify enzymatic depletion of extracellular glutamine using the long-standing anti-leukemic therapy asparaginase (ASNase), which possesses glutaminase activity, as a potential approach, and show this can be successfully combined with cisplatin in cell models. In summary, this study mechanistically links an acquired metabolic dependency of chemoresistant cancer cells with a putative biomarker and provides a potentially actionable strategy to target these drug resistant cells warranting further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c836fb51ec4a12788e118211188bb2a1ae2ab5" target='_blank'>
              Glutamine-addiction in cisplatin resistant cancer cells is mediated by SLC7A11 and can be targeted with asparaginase therapy
              </a>
            </td>
          <td>
            Jiantao Wang, Robert Strauss, Jiri Bartek, S. G. Rudd
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="


 Can novel genomic analysis methods be developed to identify genetic variants that predispose to specific types of infertility and genomic instability in embryos?



 Whole genome analysis and the creation of a genomic variation database of highly targeted populations yielded several genomic links to human infertility and genomic instability.



 Infertility is a phenotypic manifestation that originates from highly variable causes, like repeated miscarriage (RM), repeated implantation failure (RIF) or various male and female factors. Specific genetic variations can predispose carriers to distinct types of infertility however quantifiable genetic risks are difficult to be established clinically due to the high amount of variables that exist in any relevant infertility studies. In addition, the combination of the maternal and paternal genetics further complicates the interpretation of genomics results especially when genetic instability in embryos are studied. Several studies have indicated multi[le risk variants however few have managed to establish causal relationships.



 The causes of infertility are extremely variable, which presents a challenge in the design and analysis of genome sequencing studies that involve large populations of participants with similar characteristics. In this study a targeted approach highly selected populations allows more detailed examination of the whole genome of the participants and the different subgroups that exist among them. The participants were recruited using specific selection criteria and samples were collected over a period of 5 years.



 Couples presenting with infertility and went through preimplantation genetic testing for aneuploidy (PGT- A) were invited to participate. Selection criteria were applied: i) referred for PGT-A after >3 assisted reproduction failed cycles, ii) Normal karyotypes, iii) Female age less than 37, iv) Embryonic instability in their subsequent PGT-A cycles in more than 40% of the produced embryos. After whole genome sequencing (WGS) and variant calling, a bespoke genomic SQL database was constructed for genomic analysis.



 From 118 individuals that consented to participate 28 individuals with repeated implantation failure (RIF) and 8 individuals RIF/RM passed the selection criteria and their DNA was processed with WGS using a short read platform. In addition, WGS Validation was performed using genomes with known mutations. After the WGS quality control and validation. In total 50 genomes were processed and produced variant results. A novel MySQL database was successfully constructed and validated. Variant z scores were calculated for each participant and the significantly flagged variants were pulled and analysed and comparison with other bespoke and publicly available databases was performed. Ethnicity bias in novel variations detected and corrected. Several smaller cohorts were identified with similar genomic characteristics and accumulation of specific sets of polymorphisms, that were associated with certain types of infertility and preimplantation embryo abnormalities. Of particular interest were variants in synaptonemal complex genes associated with gamete failure, meiotic errors and instability in embryos. Structural variations were significant for male factor infertility. The WGS approach also revealed novel intronic and extragenic variants that would not be usually picked up by other methods like exome sequencing but can play a vital role in gene expression.



 As with all genomic studies relating to infertility, association does not mean causation. Further functional analysis would be required to establish definitive causal pathways based on significant risk variants. In addition, although our study is highly targeted more genomic data can enhance to validation of these results.



 This highly targeted study using a personalised approach to genome analysis combined with WGS can lead to protocols that determine the cumulative clinical risk for infertility, by revelling the accumulation of specific sets of polymorphisms, intronic and exonic, can be associated with certain types of infertility and preimplantation embryo abnormalities.



 not applicable
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31f47cee873a70e90fde4db0907ac6385b8599fa" target='_blank'>
              P-537 Linking genetic variation to infertility and embryonic genomic instability. Lessons from a database of whole genomes from targeted populations: challenges, and future directions
              </a>
            </td>
          <td>
            A. Mantzouratou, J. Wieland, S. Buchan, P. Serhl, S. Sioban
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) possess a significant ability to renew themselves, which gives them a strong capacity to form tumors and expand to encompass additional body areas. In addition, they possess inherent resistance to chemotherapy and radiation therapies used to treat many forms of cancer. Scientists have focused on investigating the signaling pathways that are highly linked to the ability of CSCs to renew themselves and maintain their stem cell properties. The pathways encompassed are Notch, Wnt/β-catenin, hedgehog, STAT3, NF-κB, PI-3K/Akt/mTOR, sirtuin, ALDH, MDM2, and ROS. Recent studies indicate that directing efforts towards CSC cells is essential in eradicating the overall cancer cell population and reducing the likelihood of tumor metastasis. As our comprehension of the mechanisms that stimulate CSC activity, growth, and resistance to chemotherapy advances, the discovery of therapeutic drugs specifically targeting CSCs, such as small-molecule compounds, holds the potential to revolutionize cancer therapy. This review article examines and analyzes the novel anti-CSC compounds that have demonstrated effective and selective targeting of pathways associated with the renewal and stemness of CSCs. We also discussed their special drug metabolism and absorption mechanisms. CSCs have been the subject of much study in cancer biology. As a possible treatment for malignancies, small-molecule drugs that target CSCs are gaining more and more attention. This article provides a comprehensive review of the current state of key small-molecule compounds, summarizes their recent developments, and anticipates the future discovery of even more potent and targeted compounds, opening up new avenues for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c2ec47520bdf5746027b1d46298064477df151" target='_blank'>
              Novel Anti-Cancer Stem Cell Compounds: A Comprehensive Review
              </a>
            </td>
          <td>
            Shanchun Guo, Shilong Zheng, Mingli Liu, Guangdi Wang
          </td>
          <td>2024-08-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Drug resistance is a major challenge in the treatment of advanced cholangiocarcinoma (CCA). Understanding the mechanisms of drug resistance can aid in identifying novel prognostic biomarkers and therapeutic targets to improve treatment efficacy. This study established 5-fluorouracil- (5-FU) and gemcitabine-resistant CCA cell lines, KKU-213FR and KKU-213GR, and utilized comparative proteomics to identify differentially expressed proteins in drug-resistant cells compared to parental cells. Additionally, bioinformatics analyses were conducted to explore the biological and clinical significance of key proteins. The drug-resistant phenotypes of KKU-213FR and KKU-213GR cell lines were confirmed. In addition, these cells demonstrated increased migration and invasion abilities. Proteomics analysis identified 81 differentially expressed proteins in drug-resistant cells, primarily related to binding functions, biological regulation, and metabolic processes. Protein–protein interaction analysis revealed a highly interconnected network involving MET, LAMB1, ITGA3, NOTCH2, CDH2, and NDRG1. siRNA-mediated knockdown of these genes in drug-resistant cell lines attenuated cell migration and cell invasion abilities and increased sensitivity to 5-FU and gemcitabine. The mRNA expression of these genes is upregulated in CCA patient samples and is associated with poor prognosis in gastrointestinal cancers. Furthermore, the functions of these proteins are closely related to the epithelial–mesenchymal transition (EMT) pathway. These findings elucidate the potential molecular mechanisms underlying drug resistance and tumor progression in CCA, providing insights into potential therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6dfaa4af4b64a761bf16d4bce1645ee438c64a04" target='_blank'>
              Proteomics and Bioinformatics Identify Drug-Resistant-Related Genes with Prognostic Potential in Cholangiocarcinoma
              </a>
            </td>
          <td>
            Kankamol Kerdkumthong, S. Roytrakul, Kawinnath Songsurin, Kandawasri Pratummanee, Phanthipha Runsaeng, S. Obchoei
          </td>
          <td>2024-08-08</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Adoptive cell therapy is becoming a cornerstone of tumour immunotherapy. It relies on the relatively long-term (> 2 week) ex vivo expansion of T cells either in the form of tumour-infiltrating cells, or bulk cells modified with the expression of heterologous signalling proteins, e.g., chimeric antigen receptors. However, relatively little is known about the developmental trajectories of T cells under these conditions at the system level, or whether the pathways governing these trajectories could be manipulated for clinical advantage. Using bulk RNA-seq analysis of T cells expanded and rested over a 17-day period, we produce a resource revealing how gene expression changes as cells transition through distinct cellular states over the course of activation and ex vivo expansion. By integrating this resource with published single-cell RNA-seq data, we identify a member of the AP1 transcription factor (TF) family, FOSL1, that primes CD8+ T-cells towards an effector/killing phenotype. Remarkably, FOSL1 over-expression during T-cell expansion produced ‘super engager-like’ T-cells, evidenced by their gene-expression signatures and enhanced cancer-cell killing capacity. This establishes proof-of-principle for the rational engineering of T cells via TF modification during ex vivo expansion, offering a route to improving adoptive T-cell therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87c6308d2b13138e23c30c7f12340afe1d6a3ca9" target='_blank'>
              Cell state and transcription factor modulation during extended ex vivo CD8+ T-cell expansion
              </a>
            </td>
          <td>
            Yuan Lui, Edward Jenkins, Emily Zhi Qing Ng, M. Kotowski, Sydney J Mullin, Joseph Clarke, Simon J. Davis, Ana Mafalda Santos, Sumana Sharma
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1334e8d1bab52080f2aa5225537b4b61dc872565" target='_blank'>
              Single-cell RNA-seq reveals the transcriptional program underlying tumor progression and metastasis in neuroblastoma.
              </a>
            </td>
          <td>
            Zhe Nian, Dan Wang, Hao Wang, Wenxu Liu, Zhenyi Ma, Jie Yan, Yanna Cao, Jie Li, Qiang Zhao, Zhe Liu
          </td>
          <td>2024-07-17</td>
          <td>Frontiers of medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Somatic mutation phasing informs our understanding of cancer-related events, like driver mutations. We generated linked-read whole genome sequencing data for 23 samples across disease stages from 14 multiple myeloma (MM) patients and systematically assigned somatic mutations to haplotypes using linked-reads. Here, we report the reconstructed cancer haplotypes and phase blocks from several MM samples and show how phase block length can be extended by integrating samples from the same individual. We also uncover phasing information in genes frequently mutated in MM, including DIS3, HIST1H1E, KRAS, NRAS, and TP53, phasing 79.4% of 20,705 high-confidence somatic mutations. In some cases, this enabled us to interpret clonal evolution models at higher resolution using pairs of phased somatic mutations. For example, our analysis of one patient suggested that two NRAS hotspot mutations occurred on the same haplotype but were independent events in different subclones. Given sufficient tumor purity and data quality, our framework illustrates how haplotype-aware analysis of somatic mutations in cancer can be beneficial for some cancer cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2870d2d1518a03384eeca2d75b0b4975453b6c" target='_blank'>
              Somatic mutation phasing and haplotype extension using linked-reads in multiple myeloma
              </a>
            </td>
          <td>
            S. Foltz, Yize Li, Lijun Yao, Nadezhda V. Terekhanova, A. Weerasinghe, Qingsong Gao, Guanlan Dong, Moses Schindler, Song Cao, Hua Sun, R. Jayasinghe, Robert S Fulton, C. Fronick, Justin King, Daniel R. Kohnen, M. Fiala, Ken Chen, J. Dipersio, Ravi Vij, Li Ding
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Essential gene products carry out fundamental cellular activities in interaction with other components. However, the lack of essential gene mutants and appropriate methodologies to link essential gene functions with their partners poses significant challenges. Here, we have generated deletion mutants in 32 genes previously identified as essential, with 23 mutants showing extremely slow growth in the SK36 strain of Streptococcus sanguinis. Whole-genome sequencing of 243 independently evolved populations of these mutants has identified >1000 spontaneous suppressor mutations in experimental evolution, many of which define new gene and pathway relationships, such as F1Fo-ATPase/V1Vo-ATPase/TrkA1-H1. Patterns of spontaneous mutations occurring in essential gene mutants differed from those found in wildtype. While gene duplications occurred rarely and appeared most often at later stages of evolution, substitutions, deletions, and insertions were prevalent in evolved populations. These essential gene deletion mutants and spontaneous mutations fixed in the mutant populations during evolution establish a foundation for understanding gene essentiality and the interaction of essential genes in networks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b8085c4071c7f49004500689bcbfd19d2769747" target='_blank'>
              Experimental evolution of gene essentiality in bacteria
              </a>
            </td>
          <td>
            Liang Bao, Zan Zhu, Ahmed Ismail, Bin Zhu, Vysakh Anandan, Marvin Whiteley, T. Kitten, Ping Xu
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Patients with class I V600EBRAF-mutant (MT) colorectal cancer (CRC) have a poor prognosis and their response to combined anti-BRAF/EGFR inhibition remains limited. There is clearly an unmet need in further understanding the biology of V600EBRAFMT CRC. We have used differential gene expression of BRAFWT and MT CRC cells to identify pathways underpinning BRAFMT CRC. We tested a panel of molecularly/genetically subtyped CRC cells for their sensitivity to the Unfolded Protein Response (UPR) activator BOLD-100. To identify novel combination strategies for BOLD-100, we performed RNA sequencing and high-throughput drug screening. Pathway enrichment analysis identified that the UPR and DNA repair pathways were significantly enriched in BRAFMT CRC. We found that oncogenic BRAF plays a crucial role in mediating response to BOLD-100. Using a systems biology approach, we identified V600EBRAFMT-dependent activation of the replication stress response kinase ATR as a key mediator of resistance to BOLD-100. Further analysis identified acute increases in BRAFMT-dependent-reactive oxygen species (ROS) levels following treatment with BOLD-100 that was demonstrated to promote ATR/CHK1 activation and apoptosis. Furthermore, activation of ROS/ATR/CHK1 following BOLD-100 was found to be mediated through the AHR transcription factor and CYP1A1. Importantly, pharmacological blockade of this resistance pathway with ATR inhibitors synergistically increased BOLD-100-induced apoptosis and growth inhibition in BRAFMT models. These results unveil possible novel therapeutic opportunity for BRAFMT CRC. Implications: BOLD-100 induces BRAFMT-dependent replication stress, and targeted strategies against replication stress (eg. by using ATR inhibitors) in combination with BOLD-100 may serve as a potential novel therapeutic strategy for clinically aggressive BRAFMT CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a0c515d9a8242560fe5d20e397d7ea5de365fc8" target='_blank'>
              Ruthenium drug BOLD-100 regulates BRAFMT colorectal cancer cell apoptosis through AhR/ROS/ATR signaling axis modulation.
              </a>
            </td>
          <td>
            Daryl Griffin, R. Carson, Deborah Y. Moss, T. Sessler, Deborah Lavin, Vijay K Tiwari, Shivaali Karelia, Richard Kennedy, K. Savage, S. McDade, Adam Carie, J. Pankovich, Mark Bazett, S. Van Schaeybroeck
          </td>
          <td>2024-07-31</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive and lethal form of brain cancer with few effective treatments. In this context, Zika virus has emerged as a promising therapeutic agent due to its ability to selectively infect and kill GBM cells. To elucidate these mechanisms and expand the landscape of oncolytic virotherapy, we pursued a transcriptomic meta-analysis comparing the molecular signatures of Zika infection in GBM and neuroblastoma (NBM). Over-representation analysis of dysregulated coding genes showed significant enrichment of tumor necrosis factor (TNF), NF-κB, and p53 signaling pathways. A refined list of long non-coding RNAs consistently dysregulated in Zika-infected GBMs was also developed. Functional review of these candidates revealed their potential regulatory role in Zika-mediated oncolysis. We performed validation of the less-researched targets in adult and pediatric GBM cell lines and found significant differential regulation, as predicted. Altogether, our results provide novel insights into the molecular mechanisms underlying the effect of Zika on GBM. We highlight potential therapeutic targets that could be further interrogated to improve the efficacy of tumor cell death and the utility of Zika as an adjuvant virotherapy for GBM and other related cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5fb2f82f19cb82c54f2e078dc5fd330417c5a7" target='_blank'>
              Transcriptomic Meta-analysis Identifies Long Non-Coding RNAs Mediating Zika’s Oncolytic Impact in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Youssef A. Kousa, Shriya Singh, Allison Horvath, Federica Tomasso, Javad Nazarian, Lisa Henderson, Tamer A. Mansour
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Chronic myeloid leukaemia (CML) is a haematological malignancy characterized by the constitutive tyrosine kinase activity of the BCR‐ABL1 fusion protein. Flumatinib, a second‐generation tyrosine kinase inhibitor, has exhibited superior clinical efficacy compared to its precursor, imatinib. However, with increased clinical use, resistance to flumatinib has emerged as a significant challenge. To investigate the mechanisms of flumatinib resistance in CML, we induced the human CML cell line K562 using a flumatinib concentration gradient method in vitro, successfully establishing a flumatinib‐resistant K562/FLM cell line. This cell line exhibited cross‐resistance to imatinib and doxorubicin, but remained sensitive to the antiparasitic agent ivermectin, which possesses antitumoural effects. Through cellular experimentation, we explored the resistance mechanisms, which indicated that K562/FLM cells evade flumatinib cytotoxicity by enhancing autophagy, increasing the expression of membrane transport proteins, particularly P‐glycoprotein, ABCC1 and ABCC4, as well as enhancing phosphorylation of p‐EGFR, p‐ERK and p‐STAT3 proteins. Moreover, it was found that ivermectin effectively suppressed the expression of autophagy and transport proteins in K562/FLM cells, reduced the activity of the aforementioned phosphoproteins, and promoted apoptotic cell death. Collectively, the increased autophagy, higher expression of drug‐efflux proteins and hyperactivation of the EGFR/ERK/STAT3 signalling pathway were identified as pivotal elements promoting resistance to flumatinib. The significant effects of ivermectin might offer a novel therapeutic strategy to overcome flumatinib resistance and optimize the treatment outcomes of CML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c1bd54816137ee6beae3e8ec7bc7d66c0509108" target='_blank'>
              Overcoming flumatinib resistance in chronic myeloid leukaemia: Insights into cellular mechanisms and ivermectin's therapeutic potential
              </a>
            </td>
          <td>
            Jixian Huang, Jie Xiao, Lifeng He, Wenjie Dai, Jian Xiao, Yuquan Li, Ying He, Liang Yu
          </td>
          <td>2024-07-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66eb86e9bf90c107f44800e55996f2d2a686c181" target='_blank'>
              Integrative transcriptome analysis reveals the molecular events underlying impaired T-cell responses in EGFR-mutant lung cancer
              </a>
            </td>
          <td>
            Yu Zhao, Gu Tang, Jun Li, Xiaonan Bian, Xiaorong Zhou, Jian Feng
          </td>
          <td>2024-08-07</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/354d0cada1a73a94d156b70d215246eda0c2a524" target='_blank'>
              T cell dysfunction and therapeutic intervention in cancer.
              </a>
            </td>
          <td>
            Caitlin C. Zebley, D. Zehn, Stephen Gottshalk, Hongbo Chi
          </td>
          <td>2024-07-18</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Cells must adapt to environmental changes to maintain homeostasis. One of the most striking environmental adaptations is entry into hibernation during which core body temperature can decrease from 37°C to as low at 4°C. How mammalian cells, which evolved to optimally function within a narrow range of temperatures, adapt to this profound decrease in temperature remains poorly understood. In this study, we conducted the first genome-scale CRISPR-Cas9 screen in cells derived from Syrian hamster, a facultative hibernator, as well as human cells to investigate the genetic basis of cold tolerance in a hibernator and a non-hibernator in an unbiased manner. Both screens independently revealed glutathione peroxidase 4 (GPX4), a selenium-containing enzyme, and associated proteins as critical for cold tolerance. We utilized genetic and pharmacological approaches to demonstrate that GPX4 is active in the cold and its catalytic activity is required for cold tolerance. Furthermore, we show that the role of GPX4 as a suppressor of cold-induced cell death extends across hibernating species, including 13-lined ground squirrels and greater horseshoe bats, highlighting the evolutionary conservation of this mechanism of cold tolerance. This study identifies GPX4 as a central modulator of mammalian cold tolerance and advances our understanding of the evolved mechanisms by which cells mitigate cold-associated damage—one of the most common challenges faced by cells and organisms in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181b39f570d649216102dc87d265ba81bbf8c6ba" target='_blank'>
              Multi-species genome-wide CRISPR screens identify GPX4 as a conserved suppressor of cold-induced cell death
              </a>
            </td>
          <td>
            Breanna Lam, Kathrin M. Kajderowicz, Heather R. Keys, Julian M. Roessler, Evgeni M. Frenkel, Adina Kirkland, Punam Bisht, Mohamed A. El-Brolosy, Rudolf Jaenisch, George W. Bell, Jonathan S Weissman, Eric C. Griffith, Siniša Hrvatin
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The expression of eukaryotic genes relies on the precise 3’-terminal cleavage and polyadenylation of newly synthesized pre-mRNA transcripts. Defects in these processes have been associated with various diseases, including cancer. While cancer-focused sequencing studies have identified numerous driver mutations in protein-coding sequences, noncoding drivers – particularly those affecting the cis-elements required for pre-mRNA cleavage and polyadenylation – have received less attention. Here, we systematically analysed cancer somatic mutations affecting 3’UTR polyadenylation signals using the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset. We found a striking enrichment of cancer-specific somatic mutations that disrupt strong and evolutionarily conserved cleavage and polyadenylation signals within tumour suppressor genes. Further bioinformatics and experimental analyses conducted as a part of our study suggest that these mutations have a profound capacity to downregulate the expression of tumour suppressor genes. Thus, this work uncovers a novel class of noncoding somatic mutations with significant potential to drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4b3938c7f84b24956b4223cbd287669eff1caf" target='_blank'>
              Recurrent disruption of tumour suppressor genes in cancer by somatic mutations in cleavage and polyadenylation signals
              </a>
            </td>
          <td>
            Yaroslav A. Kainov, F. Hamid, E. Makeyev
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Introduction: Circulating tumor cells (CTCs) represent the sub-population of cells shed into the vasculature and able to survive in the bloodstream, adhere to target vascular endothelial cells, and re-growth into the distant organ. CTCs have been found in the blood of most solid tumor-bearing patients and are used as a diagnostic marker. Although a complex genotypic and phenotypic signature characterizes CTCs, the ability to survive in suspension constitutes the most critical property, known as resistance to anoikis, e.g., the ability to resist apoptosis resulting from a loss of substrate adhesion. Here, we selected melanoma cells resistant to anoikis, and we studied their metabolic reprogramming, with the aim of identifying new metabolic targets of CTCs. Methods: Subpopulations of melanoma cells expressing a high anoikis-resistant phenotype were selected by three consecutive rocking exposures in suspension and studied for their phenotypic and metabolic characteristics. Moreover, we tested the efficacy of different metabolic inhibitors targeting glycolysis (2DG), LDHA (LDHA-in-3), the mitochondrial electron transport chain complex I (rotenone), glutaminase (BPTES), fatty acid transporter (SSO), fatty acid synthase (denifanstat), CPT1 (etomoxir), to inhibit cell survival and colony formation ability after 24 h of rocking condition. Results: Anoikis-resistant cells displayed higher ability to grow in suspension on agarose-covered dishes respect to control cells, and higher cell viability and colony formation capability after a further step in rocking condition. They showed also an epithelial-to-mesenchymal transition associated with high invasiveness and a stemness-like phenotype. Anoikis-resistant melanoma cells in suspension showed a metabolic reprogramming from a characteristic glycolytic metabolism toward a more oxidative metabolism based on the use of glutamine and fatty acids, while re-adhesion on the dishes reversed the metabolism to glycolysis. The treatment with metabolic inhibitors highlighted the effectiveness of rotenone, BPTES, SSO, and etomoxir in reducing the viability and the colony formation ability of cells capable of surviving in suspension, confirming the dependence of their metabolism on oxidative phosphorylation, using glutamine and fatty acids as the most important fuels. Discussion: This finding opens up new therapeutic strategies based on metabolic inhibitors of glutaminase and fatty acid oxidation for the treatment of CTCs and melanoma metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95e52231b69241d2e256add02fe49ee1f3ba1d9" target='_blank'>
              The critical role of glutamine and fatty acids in the metabolic reprogramming of anoikis-resistant melanoma cells
              </a>
            </td>
          <td>
            S. Peppicelli, T. Kersikla, G. Menegazzi, E. Andreucci, J. Ruzzolini, C. Nediani, F. Bianchini, L. Calorini, Valentina Audrito, Joanna Kopecka, Alessandra Fiore, Huakan Zhao
          </td>
          <td>2024-08-08</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Simple Summary With advanced bioinformatic and sequencing technologies, many non-coding RNAs have been found to harbor non-canonical open reading frames (ncORFs). Some studies suggest that many ncORF-encoded microproteins or micropeptides possess important roles in cancer biology, yet the functional roles of these ncORF-encoded peptides are mostly unknown. The purpose of this review is to provide a comprehensive overview of existing knowledge on ncORF-encoded microproteins in cancer biology and highlight their roles as potential prognostic and therapeutic targets in cancer. Abstract Cancer develops from abnormal cell growth in the body, causing significant mortalities every year. To date, potent therapeutic approaches have been developed to eradicate tumor cells, but intolerable toxicity and drug resistance can occur in treated patients, limiting the efficiency of existing treatment strategies. Therefore, searching for novel genes critical for cancer progression and therapeutic response is urgently needed for successful cancer therapy. Recent advances in bioinformatics and proteomic techniques have allowed the identification of a novel category of peptides encoded by non-canonical open reading frames (ncORFs) from historically non-coding genomic regions. Surprisingly, many ncORFs express functional microproteins that play a vital role in human cancers. In this review, we provide a comprehensive description of different ncORF types with coding capacity and technological methods in discovering ncORFs among human genomes. We also summarize the carcinogenic role of ncORFs such as pTINCR and HOXB-AS3 in regulating hallmarks of cancer, as well as the roles of ncORFs such as HOXB-AS3 and CIP2A-BP in cancer diagnosis and prognosis. We also discuss how ncORFs such as AKT-174aa and DDUP are involved in anti-cancer drug response and the underestimated potential of ncORFs as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d95f26adc077c4e04bfda2c2e30184b8f21906d" target='_blank'>
              Exploring the Dark Matter of Human Proteome: The Emerging Role of Non-Canonical Open Reading Frame (ncORF) in Cancer Diagnosis, Biology, and Therapy
              </a>
            </td>
          <td>
            Anni Ge, Curtis Chan, Xiaolong Yang
          </td>
          <td>2024-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="NF1 is recurrently mutated in glioblastoma yet the molecular landscape and efficacy of targeted therapies remain unclear. Here, we combine bulk and single cell genomics of human somatic NF1 mutant, IDH-wildtype glioblastomas with functional genomic analysis of cell lines and mouse intracranial tumor models to identify molecular subgroups within NF1 mutant glioblastomas and mechanisms underlying MEK inhibitor response. Targeted DNA sequencing showed homozygous deletion of the cell cycle regulator CDKN2A/B is a poor prognostic marker in somatic NF1 mutant, but not NF1 wildtype, tumors. DNA methylation array profiling revealed three epigenetic groups highlighted by distinct clinical features, co-mutation patterns, and reference methylation classifier identities. Genome-wide CRISPRi screens in glioblastoma cells revealed cell cycle regulators are conserved mediators of cell growth while response to the MEK inhibitor selumetinib converges on Ras/RAF/MEK pathway activation. Repression of the RAF regulator SHOC2 sensitizes glioblastomas to selumetinib in vitro and in vivo in mouse intracranial glioblastoma models. Single cell RNA-sequencing of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib reveals distinct responses between mesenchymal-like (MES-like) and non MES-like subpopulations suggesting non-MES like cells are intrinsically resistant to MEK inhibition. Finally, single nuclear RNA-sequencing (snRNA-seq) of human NF1 mutant, CDKN2A/B deleted glioblastomas reveals MES-like tumor cells are associated with selumetinib sensitivity signatures while non MES-like cells exhibit increased cell cycle progression and lack selumetinib sensitivity, further supporting the notion that MEK inhibition specifically targets MES-like tumor cell subpopulations. Taken together, our data underscores the heterogeneity between and within somatic NF1 mutant glioblastomas and delineates mechanisms of MEK inhibitor response across distinct tumor subpopulations, guiding the development of future therapeutic strategies that may synergize with MEK inhibition for NF1 mutant tumors. The tumor suppressor NF1 is mutated in 15% of glioblastomas,1–3 the most common malignant brain tumor with poor outcomes and few effective treatments.4 NF1 is a GTPase activating protein (GAP) that negatively regulates Ras, and thus, NF1 loss leads to induction of Ras/RAF/MEK/ERK signaling, driving tumorigenesis and comprising a targetable molecular cascade.5,6 Genomic analysis of glioblastoma demonstrates NF1 mutation is associated with a mesenchymal-like (MES-like) transcriptomic tumor cell subpopulation and altered tumor microenvironment.7,8 More broadly, DNA methylation analysis reveals multiple epigenetic subgroups with overlapping relationships to transcriptomic subtype and DNA alterations, underscoring the complex relationship between genetic drivers and molecular signatures.9 While the updated 2021 Central Nervous System WHO tumor classification incorporates an ever increasing amount of molecular criteria for diffuse astrocytic tumors,10 the existence and clinical significance of molecular subgroups within somatic NF1 mutant, IDH-wildtype glioblastomas based on genetic co-mutations, epigenetic profile, or transcriptomic signatures remain unclear. Preclinical data support the utility of MEK inhibition in NF1 mutant gliomas,11,12 and the MEK inhibitor selumetinib is FDA approved for tumors arising in patients with syndromic neurofibromatosis type 1 (NF-1) harboring a germline NF1 mutation.13,14 In NF-1 associated gliomas, MEK inhibition demonstrates efficacy in a limited case series,15 and combined BRAF/MEK inhibition shows efficacy in BRAF p.V600E mutant gliomas,16 further supporting the translational potential of Ras/Raf/MEK/ERK blockade within genetically defined glioma subtypes. Nevertheless, treatment resistance to molecular monotherapy remains a challenge,17–20 and the mechanisms underlying MEK inhibitor resistance in NF1 mutant glioma are unknown. Here, we integrate targeted DNA sequencing, DNA methylation profiling, and single nuclear RNA-sequencing (snRNA-seq) of human patient somatic NF1 mutant, IDH-wildtype glioblastomas with single cell RNA-sequencing (scRNA-seq), genome-wide clustered regularly interspaced short palindromic repeats interference (CRISPRi) screens, and pharmacologic studies in cell lines and mouse intracranial glioblastoma models to define molecular subgroups and functional mediators of MEK inhibitor response. Targeted DNA sequencing of NF1 mutant, IDH-wildtype glioblastomas (n=186 tumors) revealed CDKN2A/B deletion was associated with poor outcomes in NF1 mutant, but not NF1 wildtype, glioblastomas. DNA methylation profiling (n=129 tumors) demonstrated three epigenetic subgroups with distinct clinical features, co- mutation patterns across cell cycle genes, and reference methylation classifier identities. Genome-wide CRISPRi screens in mouse SB28 and human GBM43 glioblastoma cells identified a conserved cell cycle gene network mediating cell growth, consistent with the clinical importance of additional hits affecting the cell cycle in human somatic NF1 mutant glioblastomas. Moreover, genome-wide mediators of selumetinib response converged upon two Ras pathway effectors mediating selumetinib sensitivity: BRAF and SHOC2. SHOC2 repression in glioblastoma cells significantly improved selumetinib response both in vitro and in intracranial allografts in vivo. Single cell RNA-sequencing (scRNA-seq) of mouse intracranial glioblastomas treated with the MEK inhibitor selumetinib revealed MES-like tumor cells correlated with CDKN2A retention and the CRISPRi screen selumetinib sensitivity signature, with selumetinib resistant cells displaying Ras pathway induction. In contrast, non-MES like tumor cells were CDKN2A deficient and lacked expression of the CRISPRi screen selumetinib sensitivity signature, with selumetinib resistant cells inducing a glial de-differentiation program. Finally, snRNA-seq of NF1 mutant, CDKN2A/B deleted, IDH-wildtype glioblastomas (n=9) showed non MES-like tumor cells exhibit increased cell cycle progression and were not associated with the CRISPRi screen selumetinib sensitivity signature. MES-like tumor cells within newly diagnosed, but not recurrent, tumors retained expression of the CRISPRi screen selumetinib sensitivity signature, suggesting resistance can arise both between and within specific transcriptomic glioblastoma cell tumor cell subpopulations. Taken together, our data identifies clinically important subgroups of NF1 mutant, IDH-wildtype glioblastomas and supports a model in which heterogeneity between tumors and within tumor cell subpopulations underlies MEK inhibitor response, supporting the need for additional synergistic therapeutic approaches beyond maximal Ras pathway blockade for NF1 mutant glioblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71e53f09e755435876ce84b544a3051562b27f2f" target='_blank'>
              Multiplatform molecular profiling and functional genomic screens identify prognostic signatures and mechanisms underlying MEK inhibitor response in somatic NF1 mutant glioblastoma
              </a>
            </td>
          <td>
            Sixuan Pan, Kanish Mirchia, E. Payne, S. J. Liu, Nadeem N Al-Adli, Zain Peeran, Poojan Shukla, Jacob S. Young, Rohit Gupta, Jasper Wu, J. Pak, K. Seo, T. Ozawa, Brian Na, Alyssa T. Reddy, S. Braunstein, Joanna J. Phillips, Susan Chang, D. A. Solomon, Arie Perry, D. Raleigh, Mitchell S. Berger, A. Abate, H. Vasudevan
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Mutually exclusive loss-of-function alterations in gene pairs are those that occur together less frequently than may be expected and may denote a synthetically lethal relationship (SSL) between the genes. SSLs can be exploited therapeutically to selectively kill cancer cells. Here, we analysed mutation, copy number variation, and methylation levels in samples from The Cancer Genome Atlas, using the hypergeometric and the Poisson binomial tests to identify mutually exclusive inactivated genes. We focused on gene pairs where one is an inactivated tumour suppressor and the other a gene whose protein product can be inhibited by known drugs. This provided an abundance of potential targeted therapeutics and repositioning opportunities for several cancers. These data are available on the MexDrugs website, https://bioinformaticslab.sussex.ac.uk/mexdrugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7250b34d6949747e0dee284668cf7df5c1a39253" target='_blank'>
              Identifying actionable synthetically lethal cancer gene pairs using mutual exclusivity.
              </a>
            </td>
          <td>
            Sarah K. Wooller, Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2024-07-08</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Small cell lung cancer (SCLC) is the most lethal type of lung cancer. Paradoxically, this tumor displays an initial exquisite response to chemotherapy; however, at relapse, the tumor is highly resistant to subsequent available therapies. Here, we report that the expression of three prime repair exonuclease 1 (TREX1) is strongly induced in chemoresistant SCLCs. ATAC-seq and ChIP-seq revealed a significant increase in chromatin accessibility and transcriptional activity of TREX1 gene locus in chemoresistant SCLC. Analyses of human SCLC tumors and patient-derived xenografts (PDXs) also showed increase in TREX1 expression in post-chemotherapy samples. TREX1 depletion caused the activation of cyclic GMP-AMP synthase (cGAS)-stimulator of interferon genes (STING) pathway due to cytoplasmic accumulation of damage-associated double-stranded DNA, inducing immunogenicity and enhancing the sensitivity of drug resistant cells to chemotherapy. These findings suggest TREX1 upregulation may partially contribute to the survival of resistant cells, and its inhibition may represent a promising therapeutic strategy to enhance antitumor immunity and potentiate the efficacy of chemotherapy and/or immunotherapy in chemoresistant SCLCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920774aa4db00913c78963d621212ab8e7a2e837" target='_blank'>
              Targeting TREX1 induces innate immune response in drug-resistant Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Takahiko Murayama, N. Mahadevan, Catherine B Meador, Elena V. Ivanova, Yuqiao Pan, Erik H. Knelson, Tetsuo Tani, Jun Nakayama, Xueying Ma, Tran C Thai, Yin P Hung, William Kim, Hideo Watanabe, Kathy Q Cai, Aaron N Hata, C. Paweletz, D. Barbie, Israel Cañadas
          </td>
          <td>2024-08-23</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="DNA methylation is traditionally associated with gene silencing, but its causal relationship and role in shaping cell fate decisions still need to be fully elucidated. Here, we conducted a genome-wide analysis to investigate the relationship between DNA methylation and gene expression at gene regulatory regions in human immune cells. By utilizing CRISPR-dCas9 DNA methylation editing tools, we successfully established a cause-and-effect relationship between the methylation levels of the promoter of the Interleukin1-receptor antagonist (IL1RN) gene and its expression. Notably, we observed that modifying the DNA methylation status of the IL1RN promoter is sufficient to alter the acquisition of the human myeloid cell fate and change the cellular response to inflammatory stimuli, resulting in abnormal cytokine release and distinctive capacity to support cancer growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0466661c0180cda161896839ec2c88a8975d7da9" target='_blank'>
              Modulating immune cell fate and inflammation through CRISPR-mediated DNA methylation editing
              </a>
            </td>
          <td>
            Gemma Valcárcel, Anna V. López-Rubio, Aleksey Lazarenkov, Clara Berenguer, Josep Calafell, J. Rodríguez-Ubreva, E. Ballestar, J. L. Sardina
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA modifications drive aging, neurodegeneration, carcinogenesis, and chemotherapy drug action. To understand the functional genomic roles of DNA modifications, it is critical to accurately map their diverse chemical forms with single-nucleotide precision in complex genomes, but it remains challenging. Click-code-seq is a click-chemistry-aided single-nucleotide-resolution strategy for guanine-oxidation mapping, used in yeast DNA but having poor applicability to human genomes. Here, we upgraded click-code-seq to enable its first application for sequencing DNA oxidation and depurination in human genomes. For this, we developed a companion fluorescence assay, click-fluoro-quant, to rapidly quantify different common DNA modifications, and devised novel adapters to minimize false modification detection and assess modification frequency in cell populations. We uncovered that endogenous DNA oxidation in a human cell line has a highly similar pattern to cancer mutational signatures associated with reactive oxygen species. We established that the DNA-alkylating chemotherapy drug irofulven preferentially induces depurination in ApA dimers and promoter regions. Intriguingly, we revealed that oxidized guanines and apurinic sites, both irofulven-induced and endogenous, are depleted in gene transcribed strands, and the strand bias widens with increasing gene expression. This work substantially advances click-code-seq for deciphering the impacts of key modifications in human DNA on cellular physiology and toxicological responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6b6bbbe3beae3b0b23fbadc72da494f15786faa" target='_blank'>
              Click-chemistry-aided quantitation and sequencing of oxidized guanines and apurinic sites uncovers their transcription-linked strand bias in human cells
              </a>
            </td>
          <td>
            V. Takhaveev, N. Püllen, Navnit K. Singh, Sabrina M. Huber, Stefan Schauer, Hailey L. Gahlon, A. Poetsch, S. Sturla
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Genetically identical cells can respond heterogeneously to cancer therapy, with a subpopulation of cells often entering a temporarily arrested treatment-tolerant state before repopulating the tumor. To investigate how heterogeneity in the cell cycle arrest protein p21 arises, we imaged the dynamics of p21 transcription and protein expression along with those of p53, its transcriptional regulator, in single cells using live cell fluorescence microscopy. Surprisingly, we found that the rate of p21 transcription depends on the change in p53 rather than its absolute level. Through combined theoretical and experimental modeling, we determined that p21 transcription is governed by an incoherent feedforward loop mediated by MDM2. This network architecture facilitates rapid induction of p21 expression and variability in p21 transcription. Abrogating the feedforward loop overcomes rapid S-phase p21 degradation, with cells transitioning into a quiescent state that transcriptionally resembles a treatment-tolerant persister state. Our findings have important implications for therapeutic strategies based on activating p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b16e761f3e0e9fd2df2ece960b5d2bfc57dcb6" target='_blank'>
              Functional consequences of a p53-MDM2-p21 incoherent feedforward loop
              </a>
            </td>
          <td>
            Jamie A. Dean, José Reyes, Michael Tsabar, A. Jambhekar, G. Lahav, Franziska Michor
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Abstract The use of k-mers to capture genetic variation in bacterial genome-wide association studies (bGWAS) has demonstrated its effectiveness in overcoming the plasticity of bacterial genomes by providing a comprehensive array of genetic variants in a genome set that is not confined to a single reference genome. However, little attempt has been made to interpret k-mers in the context of genome rearrangements, partly due to challenges in the exhaustive and high-throughput identification of genome structure and individual rearrangement events. Here, we present GWarrange, a pre- and post-bGWAS processing methodology that leverages the unique properties of k-mers to facilitate bGWAS for genome rearrangements. Repeat sequences are common instigators of genome rearrangements through intragenomic homologous recombination, and they are commonly found at rearrangement boundaries. Using whole-genome sequences, repeat sequences are replaced by short placeholder sequences, allowing the regions flanking repeats to be incorporated into relatively short k-mers. Then, locations of flanking regions in significant k-mers are mapped back to complete genome sequences to visualise genome rearrangements. Four case studies based on two bacterial species (Bordetella pertussis and Enterococcus faecium) and a simulated genome set are presented to demonstrate the ability to identify phenotype-associated rearrangements. GWarrange is available at https://github.com/DorothyTamYiLing/GWarrangehttps://github.com/DorothyTamYiLing/genome_rearrangement.git.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42162bc2b2f883e8a6834a9dc952dc10d3392ba" target='_blank'>
              GWarrange: a pre- and post- genome-wide association studies pipeline for detecting phenotype-associated genome rearrangement events
              </a>
            </td>
          <td>
            Yi Ling Tam, Sarah Cameron, Andrew Preston, Lauren A. Cowley
          </td>
          <td>2024-07-01</td>
          <td>Microbial Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recurrent mutations in serine / arginine-rich splicing factor 2 (SRSF2), particularly the proline-to-histidine substitution at position 95 (P95H), have been proposed to drive neoplastic diseases. SRSF2 plays pivotal roles in pre-mRNA processing and gene transcription. However, the precise impact of these diverse functions of SRSF2 on cancer cell behaviour remains unclear. Here, we show that deletion or homozygous P95H mutation of SRSF2 both cause extensive DNA damage leading to cell cycle arrest in vitro and in vivo, regardless of cell genotype. We mechanistically demonstrate that SRSF2 is required for efficient bi-directional transcription of DNA replication and repair genes, independent of its function in splicing. In contrast, SRSF2 haploinsufficiency induces DNA damage without halting the cell cycle in cancer cells. Inducing the heterozygous Srsf2 P95H mutation leads to clonal expansion of epidermal cells in mouse skin, but tumor formation is inhibited following exposure to carcinogens. To survive carcinogen treatment, cells containing the Srsf2 P95H mutation undergo substantial transcription rewiring that restores bi-directional gene expression. Thus, our study reveals crucial roles of SRSF2 beyond splicing and underscores its importance in regulating transcription to orchestrate the cell cycle along with the DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32907ead7270689c743bba801c421a14be049a66" target='_blank'>
              SRSF2 transcriptional function maintains genome integrity during cell division
              </a>
            </td>
          <td>
            Rebecca E. Wagner, Leonie Arnetzl, Thiago Britto-Borges, Anke Heit-Mondrzyk, Ali Bakr, Etienne Sollier, Jasper Panten, Sylvain Delaunay, Daniela Sohn, P. Schmezer, D. T. Odom, Karin Müller-Decker, C. Plass, Christoph Dieterich, P. Lutsik, Susanne Bornelöv, Michaela Frye
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c785685c384edfe64958536f2b96ef40bece9a9d" target='_blank'>
              Let’s make it personal: CRISPR tools in manipulating cell death pathways for cancer treatment
              </a>
            </td>
          <td>
            Mobina Bayat, J. S. Nahand
          </td>
          <td>2024-07-29</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cce12d4cb261e95e6f4ee6a3b370aa98326787b3" target='_blank'>
              Investigating druggable kinases for targeted therapy in retinoblastoma.
              </a>
            </td>
          <td>
            Kumar Jeyaprakash, M. Kumaran, U. Kim, Radhakrishnan Santhi, V. Muthukkaruppan, B. Devarajan, A. Vanniarajan
          </td>
          <td>2024-07-01</td>
          <td>Journal of human genetics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a dreadful disease, marked by the uncontrolled proliferation of clonal plasma cells (PCs) within the bone marrow (BM). MM is characterized by a highly heterogeneous clinical and molecular background, supported by severe genomic alterations. Important deregulation of long non-coding RNAs (lncRNAs) expression has been reported in MM patients, influencing progression and therapy resistance. NEAT1 is a lncRNA essential for nuclear paraspeckles and involved in gene expression regulation. We showed that NEAT1 supports MM proliferation making this lncRNA an attractive therapeutic candidate. Here, we used a combinatorial strategy integrating transcriptomic and computational approaches with functional high-throughput drug screening, to identify compounds that synergize with NEAT1 inhibition in restraining MM cells growth. AUKA inhibitors were identified as top-scoring drugs in these analyses. We showed that the combination of NEAT1 silencing and AURKA inhibitors in MM profoundly impairs microtubule organization and mitotic spindle assembly, finally leading to cell death. Analysis of the large publicly CoMMpass dataset showed that in MM patients AURKA expression is strongly associated with reduced progression-free (p < 0.0001) and overall survival probability (p < 0.0001) and patients displaying high expression levels of both NEAT1 and AURKA have a worse clinical outcome. Finally, using RNA-sequencing data from NEAT1 knockdown (KD) MM cells, we identified the AURKA allosteric regulator TPX2 as a new NEAT1 target in MM and as a mediator of the interplay between AURKA and NEAT1, therefore providing a possible explanation of the synergistic activity observed upon their combinatorial inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a62f32e75aca50b8f306a5cfaa25789b960cc17d" target='_blank'>
              Combinatorial strategies targeting NEAT1 and AURKA as new potential therapeutic options for multiple myeloma.
              </a>
            </td>
          <td>
            N. Puccio, G. Manzotti, Elisabetta Mereu, Federica Torricelli, D. Ronchetti, Michela Cumerlato, Ilaria Craparotta, L. Di Rito, Marco Bolis, Valentina Traini, V. Manicardi, V. Fragliasso, Yvan Torrente, N. Amodio, N. Bolli, E. Taiana, Alessia Ciarrochi, Roberto Piva, A. Neri
          </td>
          <td>2024-07-11</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="The human protein Abelson kinase (Abl), a tyrosine kinase, plays a pivotal role in developing chronic myeloid leukemia (CML). Abl's involvement in various signaling pathways underscores its significance in regulating fundamental biological processes, including DNA damage responses, actin polymerization, and chromatin structural changes. The discovery of the Bcr-Abl oncoprotein, resulting from a chromosomal translocation in CML patients, revolutionized the understanding and treatment of the disease. The introduction of targeted therapies, starting with interferon-alpha and culminating in the development of tyrosine kinase inhibitors (TKIs) like imatinib, significantly improved patient outcomes. However, challenges such as drug resistance and side effects persist, indicating the necessity of research into novel therapeutic strategies. This review describes advancements in Abl kinase inhibitor development, emphasizing rational compound design from structural and regulatory information. Strategies, including bivalent inhibitors, PROTACs, and compounds targeting regulatory domains, promise to overcome resistance and minimize side effects. Additionally, leveraging the intricate structure and interactions of Bcr-Abl may provide insights into developing inhibitors for other kinases. Overall, this review highlights the importance of continued research into Abl kinase inhibition and its broader implications for therapeutic interventions targeting kinase-driven diseases. It provides valuable insights and strategies that may guide the development of next-generation therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35f11fab61ce74a2d73dd9a6e9080be3c85e97ad" target='_blank'>
              Structure-guided drug design targeting Abl kinase: how structure and regulation can assist in designing new drugs.
              </a>
            </td>
          <td>
            D. M. Martins, P. O. Fernandes, Lucas A Vieira, Vinícius G Maltarollo, Adolfo Henrique Moraes
          </td>
          <td>2024-07-15</td>
          <td>Chembiochem : a European journal of chemical biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Strict control over hematopoietic stem cell decision making is essential for healthy life‐long blood production and underpins the origins of hematopoietic diseases. Acute myeloid leukemia (AML) in particular is a devastating hematopoietic malignancy that arises from the clonal evolution of disease‐initiating primitive cells which acquire compounding genetic changes over time and culminate in the generation of leukemic stem cells (LSCs). Understanding the molecular underpinnings of these driver cells throughout their development will be instrumental in the interception of leukemia, the enabling of effective treatment of pre‐leukemic conditions, as well as the development of strategies to target frank AML disease. To this point, a number of precancerous myeloid disorders and age‐related alterations are proving as instructive models to gain insights into the initiation of LSCs. Here, we explore this myeloid dysregulation at the level of post–transcriptional control, where RNA‐binding proteins (RBPs) function as core effectors. Through regulating the interplay of a myriad of RNA metabolic processes, RBPs orchestrate transcript fates to govern gene expression in health and disease. We describe the expanding appreciation of the role of RBPs and their post–transcriptional networks in sustaining healthy hematopoiesis and their dysregulation in the pathogenesis of clonal myeloid disorders and AML, with a particular emphasis on findings described in human stem cells. Lastly, we discuss key breakthroughs that highlight RBPs and post–transcriptional control as actionable targets for precision therapy of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6dddad312b3774cf0fac0198aa7e076e582021" target='_blank'>
              RNA binding protein‐directed control of leukemic stem cell evolution and function
              </a>
            </td>
          <td>
            Pratik Joshi, Ava Keyvani Chahi, Lina Liu, Steven Moreira, Ana Vujovic, Kristin J Hope
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8e1be4efe22c9fbd98a98ceb3e0939364442527" target='_blank'>
              Genome-wide CRISPR screening identifies tyrosylprotein sulfotransferase-2 as a target for augmenting anti-PD1 efficacy
              </a>
            </td>
          <td>
            Yumi Oh, Sujeong Kim, Yunjae Kim, Hyun Kim, Dongjun Jang, Seungjae Shin, Soo Jin Lee, Jiwon Kim, Sang Eun Lee, Jaeik Oh, Yoojin Yang, Dohee Kim, Hae-Rim Jung, Sangjin Kim, Jihui Kim, K. Min, Beomki Cho, Hoseok Seo, Dohyun Han, Hansoo Park, Sung-Yup Cho
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND/AIM
There are no established biomarkers for immune checkpoint inhibitors (ICI) in colorectal cancer (CRC) with microsatellite stability (MSS) or proficient mismatch repair (pMMR). Therefore, this study aimed to identify biomarkers for ICI benefit in patients with pMMR by analyzing the down-regulated DNA repair-related genes involved in highly immunogenic and immune responses, and comparing their expression levels and clinical features.


MATERIALS AND METHODS
Mismatch repair (MMR), tumor-infiltrating lymphocytes (TIL), and tumor mutation burden (TMB) were evaluated in 13 CRC cases and mRNA expression levels of 95 DNA repair-related genes were measured. DNA repair-related genes with reduced mRNA expression in the high immunogenicity and high immune response groups were identified. Then, the mRNA expression levels of the identified DNA repair-related genes were measured in 135 patients with CRC. Hierarchical cluster analysis was performed using the mRNA expression levels to compare the clinicopathological characteristics of each cluster.


RESULTS
ATR, LIG4, and RAD52 mRNA levels were significantly down-regulated in the high immunogenicity group. GADD45B, SMUG1, and XRCC6 mRNA levels were significantly down-regulated in the high immune response group. Cases in the cluster with reduced mRNA expression of the six genes were pMMR cases. CD8 mRNA expression level was higher in this cluster than in the other clusters.


CONCLUSION
Decreased mRNA expression levels of ATR, LIG4, RAD52, GADD45B, SMUG1, and XRCC6 genes were associated with high cytotoxic T cell and TMB levels, suggesting that these genes could serve as biomarkers for ICI efficacy in pMMR cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb38d024ab94655fe817bc6cd4f0bb4ad2d969e2" target='_blank'>
              Identification of DNA Repair-related Genes as Biomarkers Reflecting MSI, TMB, and TIL in Colorectal Cancer.
              </a>
            </td>
          <td>
            Junya Fukuda, Tomoya Sudo, Maako Kikuchi, Akihiko Kawahara, Hirona Shigyo, Yusuke Kawamoto, Satoshi Shimamura, Fumiki Koga, Takasi Noguchi, Hiroyuki Nakane, Takahiro Shigaki, Kenji Fujiyoshi, T. Yomoda, Naohiro Yoshida, Kenichi Koushi, Takefumi Yoshida, Jun Akiba, Fumihiko Fujita
          </td>
          <td>2024-07-26</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Biallelic pathogenic variants in the essential DNA repair gene BRCA2 causes Fanconi anemia, complementation group FA-D1. Patients in this group are highly prone to develop embryonal tumors, most commonly medulloblastoma arising from the cerebellar granule cell progenitors (GCPs). GCPs undergo high proliferation in the postnatal cerebellum under SHH activation, but the type of DNA lesions that require the function of the BRCA2 to prevent tumorigenesis remains unknown. To identify such lesions, we assessed both GCP neurodevelopment and tumor formation using a mouse model with deletion of exons three and four of Brca2 in the central nervous system, coupled with global Trp53 loss. Brca2Δex3-4;Trp53-/- animals developed SHH subgroup medulloblastomas with complete penetrance. Whole-genome sequencing of the tumors identified structural variants with breakpoints enriched in areas overlapping G-quadruplexes (G4s). Brca2-deficient GCPs exhibited decreased replication speed in the presence of the G4-stabilizer pyridostatin. Pif1 helicase, which resolves G4s during replication, was highly upregulated in tumors, and Pif1 knockout in primary MB tumor cells resulted in increased genome instability upon pyridostatin treatment. These data suggest that G4s may represent sites prone to replication stalling in highly proliferative GCPs and without BRCA2, G4s become a source of genome instability. Tumor cells upregulate G4-resolving helicases to facilitate rapid proliferation through G4s highlighting PIF1 helicase as a potential therapeutic target for treatment of BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d543ffa68779aadc1c04b11411614442736e5f08" target='_blank'>
              G-quadruplexes are a source of vulnerability in BRCA2 deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, Carolina Sandoval-Garcia, Margaret L. MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4597b2c9168e5b3eb619c4010a4d29dad0a9c28b" target='_blank'>
              mTORC2-driven chromatin cGAS mediates chemoresistance through epigenetic reprogramming in colorectal cancer.
              </a>
            </td>
          <td>
            Guoqing Lv, Qian Wang, Lin Lin, Qiao Ye, Xi Li, Qian Zhou, Xiangzhen Kong, Hongxia Deng, Fuping You, Hebing Chen, Song Wu, Lin Yuan
          </td>
          <td>2024-07-30</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite the better understanding of the molecular mechanisms contributing to the pathogenesis of acute myeloid leukemia (AML) and improved patient survival in recent years, AML therapy still remains a clinical challenge. For this reason, it is important to search for new therapies that will enable the achievement of remission. Recently, the Food and Drug Administration approved three mutant IDH (mIDH) inhibitors for the treatment of AML. However, the use of mIDH inhibitors in monotherapy usually leads to the development of resistance and the subsequent recurrence of the cancer, despite the initial effectiveness of the therapy. A complete understanding of the mechanisms by which IDH mutations influence the development of leukemia, as well as the processes that enable resistance to mIDH inhibitors, may significantly improve the efficacy of this therapy through the use of an appropriate synergistic approach. The aim of this literature review is to present the role of IDH1/IDH2 mutations in the pathogenesis of AML and the results of clinical trials using mIDH1/IDH2 inhibitors in AML and to discuss the challenges related to the use of mIDH1/IDH2 inhibitors in practice and future prospects related to the potential methods of overcoming resistance to these agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/956158b91569ef3848247f4315ca0b54ae5a5903" target='_blank'>
              Clinical Implications of Isocitrate Dehydrogenase Mutations and Targeted Treatment of Acute Myeloid Leukemia with Mutant Isocitrate Dehydrogenase Inhibitors—Recent Advances, Challenges and Future Prospects
              </a>
            </td>
          <td>
            Adrian Kowalczyk, Julia Zarychta, Monika Lejman, Eryk Latoch, Joanna Zawitkowska
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The use of tyrosine kinase inhibitors, such as imatinib, against the chronic myeloid leukemia (CML)-causing kinase BCR::ABL1 has become the model for successful targeted therapy. Nevertheless, drug resistance remains a clinical problem. Analysis of genome-wide expression and genetic aberrations of an in vitro imatinib-resistant CML cell line revealed downregulation of Bruton's tyrosine kinase (BTK), predominantly associated with B cell malignancies, and a novel BTK kinase domain variant in imatinib resistance. This raised the question of the role of BTK in imatinib-resistant CML. In the present study, BTK downregulation and the presence of the BTK variant c.1699_1700delinsAG p.(Glu567Arg) were confirmed in imatinib resistance in vitro. Similarly, BTK inhibition or small interfering RNA-mediated BTK knockdown reduced imatinib susceptibility by 84 and 71%, respectively. BTK overexpression was detrimental to CML cells, as proliferation was significantly reduced by 20.5% under imatinib treatment. In addition, BTK rescue in imatinib-resistant cells restored imatinib sensitivity. The presence of the BTK p.(Glu567Arg) variant increased cell numbers (57%) and proliferation (37%) under imatinib exposure. These data demonstrate that BTK is important for the development of imatinib resistance in CML: Its presence increased drug response, while its absence promotes imatinib resistance. Moreover, the BTK p.(Glu567Arg) variant abrogates imatinib sensitivity. These findings demonstrate a context-dependent role for BTK as an oncogene in B cell malignancies, but as a tumor suppressor in other neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/788e8517fbc0ee9685d2742cf678b90e5864d99a" target='_blank'>
              BTK acts as a modulator of the response to imatinib in chronic myeloid leukemia.
              </a>
            </td>
          <td>
            Lena Schmidlechner, Inga Nagel, I. Vater, Ingolf Cascorbi, Meike Kaehler
          </td>
          <td>2024-07-04</td>
          <td>Oncology letters</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is a significant public health challenge, with 5-fluorouracil (5-FU) resistance being a major obstacle to effective treatment. Despite advancements, resistance to 5-FU remains formidable due to complex mechanisms such as alterations in drug transport, evasion of apoptosis, dysregulation of cell cycle dynamics, tumor microenvironment (TME) interactions, and extracellular vesicle (EV)-mediated resistance pathways. Traditional chemotherapy often results in high toxicity, highlighting the need for alternative approaches with better efficacy and safety. Phytochemicals (PCs) and EVs offer promising CRC therapeutic strategies. PCs, derived from natural sources, often exhibit lower toxicity and can target multiple pathways involved in cancer progression and drug resistance. EVs can facilitate targeted drug delivery, modulate the immune response, and interact with the TME to sensitize cancer cells to treatment. However, the potential of PCs and engineered EVs in overcoming 5-FU resistance and reshaping the immunosuppressive TME in CRC remains underexplored. Addressing this gap is crucial for identifying innovative therapies with enhanced efficacy and reduced toxicities. This review explores the multifaceted mechanisms of 5-FU resistance in CRC and evaluates the synergistic effects of combining PCs with 5-FU to improve treatment efficacy while minimizing adverse effects. Additionally, it investigates engineered EVs in overcoming 5-FU resistance by serving as drug delivery vehicles and modulating the TME. By synthesizing the current knowledge and addressing research gaps, this review enhances the academic understanding of 5-FU resistance in CRC, highlighting the potential of interdisciplinary approaches involving PCs and EVs for revolutionizing CRC therapy. Further research and clinical validation are essential for translating these findings into improved patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b42d2539e636dd666b34c979ca54ba15797bab1" target='_blank'>
              Innovative Strategies to Combat 5-Fluorouracil Resistance in Colorectal Cancer: The Role of Phytochemicals and Extracellular Vesicles
              </a>
            </td>
          <td>
            M. Barathan, A. K. Zulpa, Sook-Luan Ng, Y. Lokanathan, Min Hwei Ng, J. Law
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4af8490fe3598124f1ff1f6ff633491cbc08e2" target='_blank'>
              Engineered targeting OIP5 sensitizes bladder cancer to chemotherapy resistance via TRIP12-PPP1CB-YBX1 axis.
              </a>
            </td>
          <td>
            Xianteng Wang, Ting Guo, Liman Niu, Binbin Zheng, Wei Huang, Haibo Xu, Weiren Huang
          </td>
          <td>2024-08-18</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="INTRODUCTION
Advances in understanding of tumor biology shed light on hallmarks of cancer development and progression that include dysregulated DNA damage repair (DDR) machinery. Leveraging underlying tumor genomic instability and tumor specific defects in DDR, Poly (ADP-ribose) polymerases (PARP) inhibitors (PARPi) induced DNA damage emerges as a novel non-chemotherapy therapeutic opportunity. PARPis are currently approved in multiple tumor types, with the largest benefit seen in tumors with homologous recombination repair (HRR) deficiency, including germline and somatic mutations in BRCA1/2 genes (BRCA) and other pathway members such as PALB2 and Rad51c.


AREAS COVERED
This review article summarizes the current approval landscape and known and proposed mechanisms of resistance to PARPi. Further, therapeutic strategies to overcome PARPi resistance are discussed, including ongoing clinical trials.


EXPERT OPINION
PARPi have proven to be a safe and effective therapy and represent a cornerstone treatment across multiple solid tumor types. Elucidating innate and acquired mechanisms of resistance, coupled with the emergence of novel therapeutic options to capitalize on the activity of PARPi and prevent or reverse the acquisition of resistance, provides an opportunity to further expand the role of PARPi in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec7393511b29f158c45a3ee7ca1c8ffe71c659b" target='_blank'>
              Strategies for the prevention or reversal of PARP inhibitor resistance.
              </a>
            </td>
          <td>
            Zahi I. Mitri, Shaun M Goodyear, Gordon B. Mills
          </td>
          <td>2024-08-15</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Poly(ADP-ribose) polymerase (PARP) inhibitors are targeted therapies that accumulate DNA damage by interfering with DNA repair mechanisms and are approved for treating several cancers with BRCA1/2 mutations. In this study, we utilized CRISPR-dCas9 interference screening to identify genes regulating sensitivity to PARP inhibitors in breast cancer cell lines. Our findings indicated that the interferon (IFN) signaling gene IRF9 was critically involved in modulating sensitivity to these inhibitors. We revealed that the loss of IRF9 leads to increased resistance to the PARP inhibitor in MDA-MB-468 cells, and a similar desensitization was observed in another breast cancer cell line, MDA-MB-231. Further analysis indicated that while the basal expression of IRF9 did not correlate with the response to the PARP inhibitor olaparib, its transcriptional induction was significantly associated with increased sensitivity to the DNA-damaging agent cisplatin in the NCI-60 cell line panel. This finding suggests a mechanistic link between IRF9 induction and cellular responses to DNA damage. Additionally, data from the METABRIC patient tissue study revealed a complex network of IFN-responsive gene expressions postchemotherapy, with seven upregulated genes, including IRF9, and three downregulated genes. These findings underscore the intricate role of IFN signaling in the cellular response to chemotherapy. Collectively, our CRISPR screening data and subsequent bioinformatic analyses suggest that IRF9 is a novel biomarker for sensitivity to DNA-damaging agents, such as olaparib and platinum-based chemotherapeutic agents. Our findings for IRF9 not only enhance our understanding of the genetic basis of drug sensitivity, but also elucidate the role of IRF9 as a critical effector within IFN signaling pathways, potentially influencing the association between the host immune system and chemotherapeutic efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/269c63f48793fead5c71d6f165049d60737829ca" target='_blank'>
              The Role of IRF9 Upregulation in Modulating Sensitivity to Olaparib and Platinum-Based Chemotherapies in Breast Cancer
              </a>
            </td>
          <td>
            SeokGyeong Choi, Han-Gyu Bae, Dong-Kyu Jo, Woo-Young Kim
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e7e1fc16e696eaffc0fa02c1362c0f5c07aa1" target='_blank'>
              The complex interplay of tumor-infiltrating cells in driving therapeutic resistance pathways
              </a>
            </td>
          <td>
            Dengxiong Li, Fanglin Shao, Qingxin Yu, Rui-cheng Wu, Zhou-ting Tuo, Jie Wang, Luxia Ye, Yiqing Guo, Koo Han Yoo, Mang Ke, U. Okoli, Chaipanichkul Premkamon, Yubo Yang, Wuran Wei, Susan Heavey, William C. Cho, Dechao Feng
          </td>
          <td>2024-08-19</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cytogenomic characterization is crucial for the classification and risk stratification of acute myeloid leukemia (AML), thereby facilitating therapeutic decision-making. We examined the clinical utility of optical genome mapping (OGM) in 159 AML patients (103 newly diagnosed and 56 refractory/relapsed), all of whom also underwent chromosomal banding analysis (CBA), fluorescence in situ hybridization, and targeted next-generation sequencing. OGM detected nearly all clinically relevant cytogenetic abnormalities that SCG identified with >99% sensitivity, provided the clonal burden was above 20%. OGM identified additional cytogenomic aberrations and/or provided information on fusion genes in 77 (48%) patients, including eight patients with normal karyotypes and four with failed karyotyping. The most common additional alterations identified by OGM included chromoanagenesis (n = 23), KMT2A partial tandem duplication (n = 11), rearrangements involving MECOM (n = 7), NUP98 (n = 2), KMT2A (n = 2), JAK2 (n = 2), and other gene fusions in 17 patients, with 10 showing novel fusion gene partners. OGM also pinpointed fusion genes in 17 (11%) patients where chromosomal rearrangements were concurrently detected by OGM and CBA. Overall, 24 (15%) aberrations were identified exclusively by OGM and had the potential to alter AML classification, risk stratification, and/or clinical trial eligibility. OGM emerges as a powerful tool for identifying fusion genes and detecting subtle or cryptic cytogenomic aberrations that may otherwise remain undetectable by CBA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d116f1be018b74ce05fc5532f2fa6f900955563" target='_blank'>
              Optical genome mapping improves the accuracy of classification, risk stratification, and personalized treatment strategies for patients with acute myeloid leukemia.
              </a>
            </td>
          <td>
            S. Loghavi, Qing Wei, F. Ravandi, Andres E. Quesada, M. Routbort, Shimin Hu, G. Toruner, Sa A Wang, Wei Wang, Roberto N. Miranda, Shaoying Li, Jie Xu, Courtney D. DiNardo, Naval G Daver, T. Kadia, G. Issa, H. Kantarjian, L. J. Medeiros, G. Tang
          </td>
          <td>2024-07-17</td>
          <td>American journal of hematology</td>
          <td>0</td>
          <td>224</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79dbe68de75bb03fc187c37800cd2699920eb720" target='_blank'>
              Genomic and immune determinants of resistance to daratumumab-based therapy in relapsed refractory multiple myeloma
              </a>
            </td>
          <td>
            B. Ziccheddu, C. Giannotta, M. D'Agostino, G. Bertuglia, E. Saraci, S. Oliva, E. Genuardi, M. Papadimitriou, B. Diamond, P. Corradini, D. Coffey, O. Landgren, N. Bolli, B. Bruno, M. Boccadoro, Massimo Massaia, Francesco Maura, A. Larocca
          </td>
          <td>2024-07-19</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The endonuclease activity of Pms1 directs mismatch repair by generating a nick in the newly replicated DNA strand. Inactivating Pms2, the human homologue of yeast Pms1, increases the chances of colorectal and uterine cancers. Here we use whole genome sequencing to show that loss of this endonuclease activity, via the pms1-DE variant, results in strong mutator effects throughout the Saccharomyces cerevisiae genome. Mutation rates are strongly increased for mutations resulting from all types of single-base substitutions and for a wide variety of single- and multi-base indel mutations. Rates for these events are further increased in strains combining pms1-DE with mutator variants of each of the three major leading and lagging strand replicases. In all cases, mutation rates, spectra, biases, and context preferences are statistically indistinguishable from strains with equivalent polymerases but lacking initial mismatch recognition due to deletion of MSH2. This implies that, across the nuclear genome, strand discrimination via the Pms1 endonuclease is as important for MMR as is initial mismatch recognition by Msh2 heterodimers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c81863bf8b4bcd0533ef4d09eddc5a93c0da4b" target='_blank'>
              Instability throughout the Saccharomyces cerevisiae genome resulting from Pms1 endonuclease deficiency.
              </a>
            </td>
          <td>
            S. Lujan, Marta A. Garbacz, S. E. Liberti, Adam B Burkholder, Thomas A. Kunkel
          </td>
          <td>2024-07-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Leukemia represents a diverse group of hematopoietic neoplasms that can be classified into different subtypes based on the molecular aberration in the affected cell population. Identification of these molecular classification is required to identify specific targeted therapeutic approaches for each leukemic subtype. In general, targeted therapy approaches achieve good responses in some leukemia subgroups, however, resistance against these targeted therapies is common. In this review, we summarize molecular drug resistance biomarkers in targeted therapies in BCR::ABL1-driven chronic myeloid leukemia (CML) and JAK2-driven myeloproliferative neoplasms (MPNs). While acquisition of secondary mutations in the BCR::ABL1 kinase domain is the a common mechanism associated with TKI resistance in CML, in JAK2-driven MPNs secondary mutations in JAK2 are rare. Due to high prevalence and lack of specific therapy approaches in MPNs compared to CML, identification of crucial pathways leading to inhibitor persistence in MPN model is utterly important. In this review, we focus on different alternative signaling pathways activated in both, BCR::ABL1-mediated CML and JAK2-mediated MPNs, by combining data from in vitro and in vivo-studies that could be used as potential biomarkers of drug resistance. In a nutshell, some common similarities, especially activation of PDGFR, Ras, PI3K/Akt signaling pathways, have been demonstrated in both leukemias. In addition, induction of the nucleoprotein YBX1 was shown to be involved in TKI-resistant JAK2-mediated MPN, as well as TKI-resistant CML highlighting deubiquitinating enzymes as potential biomarkers of TKI resistance. Taken together, whole exome sequencing of cell-based or patients-derived samples are highly beneficial to define specific resistance markers. Additionally, this might be helpful for the development of novel diagnostic tools, e.g., liquid biopsy, and novel therapeutic agents, which could be used to overcome TKI resistance in molecularly distinct leukemia subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b47e0df361f50995af94c5ef4667538d96cf6be" target='_blank'>
              Molecular biomarkers of leukemia: convergence-based drug resistance mechanisms in chronic myeloid leukemia and myeloproliferative neoplasms
              </a>
            </td>
          <td>
            Meike Kaehler, N. von Bubnoff, Ingolf Cascorbi, S. P. Gorantla
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="With age, hematopoietic stem cells can acquire somatic mutations in leukemogenic genes that confer a proliferative advantage in a phenomenon termed “clonal hematopoiesis of indeterminate potential” (CHIP). How these mutations confer a proliferative advantage and result in increased risk for numerous age-related diseases remains poorly understood. We conducted a multiracial meta-analysis of epigenome-wide association studies (EWAS) of CHIP and its subtypes in four cohorts (N=8196) to elucidate the molecular mechanisms underlying CHIP and illuminate how these changes influence cardiovascular disease risk. The EWAS findings were functionally validated using human hematopoietic stem cell (HSC) models of CHIP. A total of 9615 CpGs were associated with any CHIP, 5990 with DNMT3A CHIP, 5633 with TET2 CHIP, and 6078 with ASXL1 CHIP (P <1×10−7). CpGs associated with CHIP subtypes overlapped moderately, and the genome-wide DNA methylation directions of effect were opposite for TET2 and DNMT3A CHIP, consistent with their opposing effects on global DNA methylation. There was high directional concordance between the CpGs shared from the meta-EWAS and human edited CHIP HSCs. Expression quantitative trait methylation analysis further identified transcriptomic changes associated with CHIP-associated CpGs. Causal inference analyses revealed 261 CHIP-associated CpGs associated with cardiovascular traits and all-cause mortality (FDR adjusted p-value <0.05). Taken together, our study sheds light on the epigenetic changes impacted by CHIP and their associations with age-related disease outcomes. The novel genes and pathways linked to the epigenetic features of CHIP may serve as therapeutic targets for preventing or treating CHIP-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb071f9470db1b0d43e08fc0664a4d81582bdd97" target='_blank'>
              Epigenome-wide DNA Methylation Association Study of CHIP Provides Insight into Perturbed Gene Regulation
              </a>
            </td>
          <td>
            Daniel Levy, Sara Kirmani, T. Huan, Joseph Van Amburg, R. Joehanes, M. Uddin, N. Nguyen, Bing Yu, Jennifer A. Brody, M. Fornage, Jan Bressler, N. Sotoodehnia, David Ong, Fabio Puddu, James Floyd, Christie M. Ballantyne, B. Psaty, L. Raffield, Pradeep Natarajan, Karen Conneely, April P. Carson, Leslie Lange, Kendra Ferrier, N. HEARD-COSTA, Joanne M Murabito, A. Bick
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Understanding bacterial gene function remains a major biological challenge. Double-mutant genetic interaction (GI) analysis addresses this challenge by uncovering the functional partners of targeted genes, allowing us to associate genes of unknown function with novel pathways and unravel connections between well-studied pathways, but is difficult to implement at the genome-scale. Here, we develop and use double-CRISPRi to systematically quantify genetic interactions at scale in the Bacillus subtilis envelope, including essential genes. We discover > 1000 known and novel genetic interactions. Our analysis pipeline and experimental follow-ups reveal the distinct roles of paralogous genes such as the mreB and mbl actin homologs, and identify new genes involved in the well-studied process of cell division. Overall, our study provides valuable insights into gene function and demonstrates the utility of double-CRISPRi for high-throughput dissection of bacterial gene networks, providing a blueprint for future studies in diverse bacterial species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed882175f2414bae232e1b044a85ff77b9c005c" target='_blank'>
              Comprehensive double-mutant analysis of the Bacillus subtilis envelope using double-CRISPRi
              </a>
            </td>
          <td>
            B. Koo, Horia Todor, Jiawei Sun, Jordi van Gestel, John S. Hawkins, Cameron C. Hearne, Amy B. Banta, K. C. Huang, Jason M. Peters, Carol A Gross
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e81b6b5c5ed75ea0aa53f16f320c4ca6faed6d1d" target='_blank'>
              Intricate effects of post-translational modifications in liver cancer: mechanisms to clinical applications
              </a>
            </td>
          <td>
            Yu Zhang, Weihao Xu, Chuanhui Peng, Shenli Ren, Chengpeng Zhang
          </td>
          <td>2024-07-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f33bf381d41bc1cd559dbc45f08c43df9d1a19d" target='_blank'>
              Comprehensive insights on genetic alterations and immunotherapy prognosis in Chinese melanoma patients
              </a>
            </td>
          <td>
            Dong-Dong Jia, Tao Li
          </td>
          <td>2024-07-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aa3ff406547a34fa6946daf5b9a8ed1bbbd6216" target='_blank'>
              Loss of the tumour suppressor LKB1/STK11 uncovers a leptin-mediated sensitivity mechanism to mitochondrial uncouplers for targeted cancer therapy
              </a>
            </td>
          <td>
            A. Angelopoulou, Giorgos Theocharous, Dimitrios Valakos, A. Polyzou, Sophia Magkouta, V. Myrianthopoulos, Sophia Havaki, Marco Fiorillo, Ioanna Tremi, Konstantinos Vachlas, Theodoros Nisotakis, Dimitris-Foivos Thanos, Anastasia Pantazaki, Dimitris Kletsas, Jiri Bartek, R. Petty, Dimitris-Foivos Thanos, R. McCrimmon, A. Papaspyropoulos, V. Gorgoulis
          </td>
          <td>2024-07-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8867a66b53ced02c166da3c33e0be6474929e7d1" target='_blank'>
              Somatic CpG hypermutation is associated with mismatch repair deficiency in cancer.
              </a>
            </td>
          <td>
            Aidan Flynn, Sebastian M. Waszak, Joachim Weischenfeldt
          </td>
          <td>2024-07-18</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Glioblastoma (GBM) is an aggressive form of brain cancer that is highly resistant to therapy due to significant intra-tumoral heterogeneity. The lack of robust in vitro models to study early tumor progression has hindered the development of effective therapies. Here, we develop engineered GBM organoids (eGBOs) harboring GBM subtype-specific oncogenic mutations to investigate the underlying transcriptional regulation of tumor progression. Single-cell and spatial transcriptomic analyses revealed that these mutations disrupt normal neurodevelopment gene regulatory networks resulting in changes in cellular composition and spatial organization. Upon xenotransplantation into immunodeficient mice, eGBOs form tumors that recapitulate the transcriptional and spatial landscape of human GBM samples. Integrative single-cell trajectory analysis of both eGBO-derived tumor cells and patient GBM samples revealed the dynamic gene expression changes in developmental cell states underlying tumor progression. This analysis of eGBOs provides an important validation of engineered cancer organoid models and demonstrates their utility as a model of GBM tumorigenesis for future preclinical development of therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07db8d239a0fe7347f57d226178f79c43e99e297" target='_blank'>
              Modeling glioblastoma tumor progression via CRISPR-engineered brain organoids
              </a>
            </td>
          <td>
            Matthew Ishahak, Rowland H. Han, Devi Annamalai, Timothy Woodiwiss, Colin McCornack, Ryan Cleary, Patrick A. DeSouza, Xuan Qu, Sonika Dahiya, Albert H. Kim, J. Millman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="microRNAs (miRNAs), tiny, non-coding RNA molecules, fine-tune the expression of target genes through interacting with mRNAs. These miRNAs are involved in a wide range of biological processes, encompassing cell division, death, blood cell production, and tumor development. When these miRNAs become dysfunctional, they can promote the invasion and spread of cancer cells in various human malignancies, including leukemia. Acute lymphoblastic leukemia (ALL), the preeminent malignancy affecting children, is a blood cancer marked by the uncontrollable growth of immature lymphoid cells that displace healthy blood precursors in the bone marrow. Despite a decline in ALL mortality rates over the past two decades, a significant proportion of deaths still results from a lack of effective diagnostic and prognostic markers that can guide treatment decisions and overcome drug resistance. The analysis of miRNA expression patterns in ALL could lead to more precise disease classification, earlier diagnosis, and better prognostic outcomes in the near future. The connection between miRNA dysfunction and the biology of ALL suggests that these molecules could represent promising therapeutic targets. Therefore, this review delves into the regulatory mechanisms of miRNAs in pediatric ALL, exploring how miRNA-based diagnostic, prognostic, and therapeutic strategies offer unique advantages and hold promise for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b51d3730adcae846d16d92db76f51f153c5c1d0" target='_blank'>
              Advancements in the Regulatory Role of microRNAs in Childhood Acute Lymphoblastic Leukemia: Mechanisms and Clinical Implications
              </a>
            </td>
          <td>
            Wei Deng
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd083194686b61ccc53ca0d4c394b67971b4cdb7" target='_blank'>
              Targeted gene therapy for cancer: the impact of microRNA multipotentiality
              </a>
            </td>
          <td>
            Nourhan A. Abou Madawi, Zeinab E Darwish, Enas M Omar
          </td>
          <td>2024-08-01</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Naïve pluripotent stem cells (nPSC) frequently undergo pathological and not readily reversible loss of DNA methylation marks at imprinted gene loci. This abnormality poses a hurdle for using pluripotent cell lines in biomedical applications and underscores the need to identify the causes of imprint instability in these cells. We show that nPSCs from inbred mouse strains exhibit pronounced strain-specific susceptibility to locus-specific deregulation of imprinting marks during reprogramming to pluripotency and upon culture with MAP kinase inhibitors, a common approach to maintain naïve pluripotency. Analysis of genetically highly diverse nPSCs from the Diversity Outbred (DO) stock confirms that genetic variation is a major determinant of epigenome stability in pluripotent cells. We leverage the variable DNA hypomethylation in DO lines to identify several trans-acting quantitative trait loci (QTLs) that determine epigenome stability at either specific target loci or genome-wide. Candidate factors encoded by two multi-target QTLs on chromosomes 4 and 17 suggest specific transcriptional regulators that contribute to DNA methylation maintenance in nPSCs. We propose that genetic variants represent candidate biomarkers to identify pluripotent cell lines with desirable properties and might serve as entry points for the targeted engineering of nPSCs with stable epigenomes. Highlights Naïve pluripotent stem cells from distinct inbred mouse strains exhibit variable DNA methylation levels at imprinted gene loci. The vulnerability of pluripotent stem cells to loss of genomic imprinting caused by MAP kinase inhibition strongly differs between inbred mouse strains. Genetically diverse pluripotent stem cell lines from Diversity Outbred mouse stock allow the identification of quantitative trait loci controlling DNA methylation stability. Genetic variants may serve as biomarkers to identify naïve pluripotent stem cell lines that are epigenetically stable in specific culture conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fcf4c9c2e254cef424889809e18c41add6362bb" target='_blank'>
              Genetic variation modulates susceptibility to aberrant DNA hypomethylation and imprint deregulation in naïve pluripotent stem cells
              </a>
            </td>
          <td>
            C. Parikh, RA Glenn, Y. Shi, K. Chatterjee, EE Swanzey, S. Singer, SC Do, Y. Zhan, Y. Furuta, M. Tahiliani, E. Apostolou, A. Polyzos, R. Koche, JG Mezey, T. Vierbuchen, M. Stadtfeld
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daaaeb917f1640a6de2396b29c1a5a0f942bd758" target='_blank'>
              Enhanced cancer cell proliferation and aggressive phenotype counterbalance in breast cancer with high BRCA1 gene expression.
              </a>
            </td>
          <td>
            Kohei Chida, M. Oshi, A. M. Roy, Takumi Sato, Maya Penelope Takabe, Li Yan, I. Endo, Kenichi Hakamada, K. Takabe
          </td>
          <td>2024-07-07</td>
          <td>Breast cancer research and treatment</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4da881111cc4055f065c04d5a3336c85756775d1" target='_blank'>
              Somatic mutational landscape across Indian breast cancer cases by whole exome sequencing
              </a>
            </td>
          <td>
            Rahul Kumar, Supriya Awasthi, Dibyabhaba Pradhan, Rakesh Kumar, Harsh Goel, J. Singh, Imran Haider, S. Deo, Chitresh Kumar, Anurag Srivastava, Amar Bhatnagar, Rakesh Kumar, S. Lakshmi, Paul Augustine, Amar Ranjan, Anita Chopra, A. Gogia, A. Batra, S. Mathur, G. Rath, Tanvir Kaur, R. Dhaliwal, Aleyamma Mathew, Usha Agrawal, Showket Hussain, P. Tanwar
          </td>
          <td>2024-08-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b139e901b81d232b7592efe9dd25a1948b27ddfe" target='_blank'>
              IκBα kinase inhibitor BAY 11-7082 promotes anti-tumor effect in RAS-driven cancers
              </a>
            </td>
          <td>
            P. Guruvaiah, Romi Gupta
          </td>
          <td>2024-07-09</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Transfer RNA (tRNA) modifications have emerged as critical posttranscriptional regulators of gene expression affecting diverse biological and disease processes. While there is extensive knowledge about the enzymes installing the dozens of post-transcriptional tRNA modifications – the tRNA epitranscriptome – very little is known about how metabolic, signaling, and other networks integrate to regulate tRNA modification levels. Here we took a comprehensive first step at understanding epitranscriptome regulatory networks by developing a high-throughput tRNA isolation and mass spectrometry-based modification profiling platform and applying it to a Pseudomonas aeruginosa transposon insertion mutant library comprising 5,746 strains. Analysis of >200,000 tRNA modification data points validated the annotations of predicted tRNA modification genes, uncovered novel tRNA-modifying enzymes, and revealed tRNA modification regulatory networks in P. aeruginosa. Platform adaptation for RNA-seq library preparation would complement epitranscriptome studies, while application to human cell and mouse tissue demonstrates its utility for biomarker and drug discovery and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cd74a5ca2ea04241d0e2f28ffe633a7113d027f" target='_blank'>
              tRNA modification profiling reveals epitranscriptome regulatory networks in Pseudomonas aeruginosa
              </a>
            </td>
          <td>
            Jingjing Sun, Junzhou Wu, Yifeng Yuan, Leon Fan, Wei Lin Patrina Chua, Yan Han Sharon Ling, S. Balamkundu, Dwija priya, Hazel Chay Suen Suen, V. de Crécy-Lagard, Agnieszka Dziergowska, Peter C. Dedon
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Local inflammation in the pancreas is transient but imprints a durable epigenetic memory on epithelial cells, making them more amenable to oncogenic transformation. However, it is unclear whether epithelial cell heterogeneity is impacted by acute pancreatitis (AP) or whether population dynamics during regeneration contributes to the establishment of inflammation memory. To tackle those questions, we deployed experimental pancreatitis in mice and performed paired sequencing of transcriptomic and chromatin accessibility profiles at single nucleus resolution. We documented cell type abundance but also applied integrative analyses to infer phenotypically-distinct clusters of mesenchymal and exocrine cells. We found that AP perturbs a subset of “idling” acinar cells, which separate from more canonical “secretory” acini based on a more diversified proteome, which include elevated expression of signal transduction receptors. We linked acinar cell heterogeneity to epigenetic differences that also endow idling cells with superior plasticity. These constitute about 40% of acinar cells but can proliferate and skew their phenotype in response to AP. This leads to a remarkable recovery of pancreas histology and function, but also to the dissemination of idling-like features across the exocrine parenchyma. Mechanistically, idling acinar cells are characterized by enhanced transcriptional activity and protein synthesis. After recovery from pancreatitis, acini show elevation of both and establishment of chronic Unfolded Protein Response (UPR). We finally demonstrated that AP-primed pancreata show signs of elevated UPR and that ER stress promotes acinar cell metaplasia. Our data interrogate phenotypical dynamics during tissue regeneration to identify cell states amenable to epigenetic imprinting. They also suggest that UPR-alleviating strategies might curtail the risk of developing pancreatic cancer for individuals who experiences AP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0889f4e0ef737079e14a1df3e2a78bed4691d30d" target='_blank'>
              Population dynamics after pancreatitis dictates long-lasting epigenetic reprogramming and mediates tumor predisposition
              </a>
            </td>
          <td>
            Marco Fantuz, J. Liebig, Giovanni Fontana, Valerio Iebba, Carmine Carbone, Sören Lukassen, Christian Conrad, Alessandro Carrer
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Therapy-induced senescence (TIS) represents a major cellular response to anticancer treatments. Both malignant and non-malignant cells in the tumor microenvironment undergo TIS and may be harmful for cancer patients since TIS cells develop a senescence-associated secretory phenotype (SASP) that can sustain tumor growth. The SASP also modulates anti-tumor immunity, although the immune populations involved and the final results appear to be context-dependent. In addition, senescent cancer cells are able to evade senescence growth arrest and to resume proliferation, likely contributing to relapse. So, research data suggest that TIS induction negatively affects therapy outcomes in cancer patients. In line with this, new interventions aimed at the removal of senescent cells or the reprogramming of their SASP, called senotherapy, have become attractive therapeutic options. To date, the lack of reliable, cost-effective, and easy-to-use TIS biomarkers hinders the application of recent anti-senescence therapeutic approaches in the clinic. Hence, the identification of biomarkers for the detection of TIS tumor cells and TIS non-neoplastic cells is a high priority in cancer research. In this review article, we describe the current knowledge about TIS, outline critical gaps in our knowledge, and address recent advances and novel approaches for the discovery of TIS biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80b8874610a137e048d67c39fb5ecaa332e59e90" target='_blank'>
              Therapy-Induced Senescence: Novel Approaches for Markers Identification
              </a>
            </td>
          <td>
            F. Pacifico, Fulvio Magni, Antonio Leonardi, E. Crescenzi
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background: Actionable driver mutations account for 40–50% of NSCLC cases, and their identification clearly affects treatment choices and outcomes. Conversely, non-actionable mutations are genetic alterations that do not currently have established treatment implications. Among co-occurring alterations, the identification of concurrent actionable genomic alterations is a rare event, potentially impacting prognosis and treatment outcomes. Methods: We retrospectively evaluated the prevalence and patterns of concurrent driver genomic alterations in a large series of NSCLCs to investigate their association with clinicopathological characteristics, to assess the prognosis of patients whose tumor harbors concurrent alterations in the genes of interest and to explore their potential therapeutic implications. Results: Co-occurring driver alterations were identified in 26 out of 1520 patients with at least one gene alteration (1.7%). Within these cases, the incidence of concurrent actionable gene alterations was 39% (0.7% of the overall cohort). Among compound actionable gene mutations, EGFR was the most frequently involved gene (70%). The most frequent association was EGFR mutations with ROS1 rearrangement. Front-line targeted treatments were the preferred approach in patients with compound actionable mutations, with dismal median PFS observed (6 months). Conclusions: Advances in genomic profiling technologies are facilitating the identification of concurrent mutations. In patients with concurrent actionable gene alterations, integrated molecular and clinical data should be used to guide treatment decisions, always considering rebiopsy at the moment of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59022475d568d630ac2b88c69a13d9f08f1373da" target='_blank'>
              Co-Occurring Driver Genomic Alterations in Advanced Non-Small-Cell Lung Cancer (NSCLC): A Retrospective Analysis
              </a>
            </td>
          <td>
            I. Attili, Riccardo Asnaghi, D. Vacirca, Riccardo Adorisio, A. Rappa, Alberto Ranghiero, Mariano Lombardi, C. Corvaja, V. Fuorivia, Ambra Carnevale Schianca, P. Trillo Aliaga, G. Spitaleri, E. del Signore, J. Guarize, Lorenzo Spaggiari, E. Guerini-Rocco, Nicola Fusco, F. de Marinis, A. Passaro
          </td>
          <td>2024-07-31</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="One of the main causes of cancer-related mortality for women worldwide is breast cancer (BC). The XRCC2 gene, essential for DNA repair, has been implicated in cancer susceptibility. This study aims to evaluate the association between XRCC2 and BC risk. The study was conducted at Zheen International Hospital in Erbil, Iraq, between 2021 and 2024 with a total of 88 samples, including 44 paired normal and cancer tissue samples. Mutation analysis was performed using Next-Generation Sequencing, coupled with in silico tools for variant impact prediction. Expression levels were assessed through RT-PCR, and methylation status was determined using methylation-sensitive restriction enzyme digestion PCR. The study identified seven inherited germline variants in the XRCC2 gene, with five of these mutations being Uncertain Significance, one being Likely Pathogenic, and one being Likely benign. RNA purity was found high with mean A260/280 ratios of 1.986 ± 0.097 in normal (N) and 1.963 ± 0.092 in tumor (T) samples. Tumor samples exhibited a higher RNA concentration (78.56 ± 40.87 ng/µL) than normal samples (71.44 ± 40.79 ng/µL). XRCC2 gene expression was significantly upregulated in tumor tissue, with marked increases in patients aged 40-55 and >56 years and in higher cancer grades (II and III) and invasive ductal carcinoma (p-values ranging from <0.0001 to 0.0392). DNA methylation rates in tumor tissues were low (7%), suggesting limited regulation by methylation. The study suggests that XRCC2 can be classified as an oncogene and that its structural investigation by targeted NGS and expression evaluation can be used as a potential biomarker in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81c449dfd85b137aba37b45f3107a8eae5435a35" target='_blank'>
              Association of XRCC2 with breast cancer, a multi-omics analysis at genomic, transcriptomic, and epigenomic level.
              </a>
            </td>
          <td>
            N. Gilani, Mehmet Ozaslan
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Due to the significant morbidity and mortality of hemoglobinopathies, curative options have long been pursued. The overall goal of gene therapy is to modify a patient's own hematopoietic stem cells to overcome the deleterious effects of the underlying genetic defect by gene addition, gene editing, or gene silencing. Gene addition incorporates genes with superior function than the abnormal gene; gene editing takes advantage of molecular tools such as zinc finger proteins, Transcription Activator-Like Effector Nucleases and Clustered Regularly Interspaced Short Palindromic Repeats coupled with Cas9 proteins (CRISPR-Cas9) which allow for sequence-specific breaks in DNA that disrupt gene function; and gene silencing suppresses gene expression by interference with mRNA transcription/protein translation or epigenetic modification. The majority of gene therapy strategies for hemoglobinopathies have targeted erythroid-specific BCL11A, a major regulator of fetal hemoglobin repression at the gamma-globin locus, in the normal fetal-to-adult hemoglobin switch that occurs shortly after birth. Other goals have involved the incorporation of anti-sickling globins, such as βT87Q or βAS3. Landmark clinical trials of gene therapy in transfusion-dependent thalassemia and sickle cell disease have shown remarkable efficacy and acceptable safety and culminated in recent regulatory approvals of gene therapy for both diseases in Europe and the United States.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f812c9e54de47feca0c8615f3dda20f25f85b41" target='_blank'>
              A Review of Gene Therapies for Hemoglobinopathies.
              </a>
            </td>
          <td>
            Boubini Jones-Wonni, Amar H Kelkar, Maureen O Achebe
          </td>
          <td>2024-08-15</td>
          <td>Hemoglobin</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Current treatments for KRAS-mutant colorectal cancers (CRCs) are often limited by cellular plasticity and rewiring responses. Here we describe a promising therapeutic strategy that simultaneously targets epigenetic and oncogenic signals. Specifically, we show that inhibitors of the histone methyltransferase, EZH2, synergize with various RAS pathway inhibitors and promote dramatic tumor regression in vivo. Together these agents cooperatively suppress WNT-driven transcription and drive CRCs into a more differentiated cell state by inducing the Groucho/TLE corepressor, TLE4, along with a network of WNT pathway inhibitors and intestinal differentiation proteins. However, these agents also induce the pro-apoptotic protein BMF, which subsequently kills these more differentiated cells. Accordingly, cell death can be prevented by activating β-catenin, blocking differentiation, or by ablating BMF expression. Collectively, these studies reveal a new therapeutic approach for treating KRAS-mutant CRCs and illustrate a critical convergence of EZH2 and RAS on oncogenic WNT signals, intestinal differentiation, and apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47a75ff581332b9384e613ce0d1c80e31add042e" target='_blank'>
              Epigenetic and oncogenic inhibitors cooperatively drive differentiation and kill KRAS-mutant colorectal cancers.
              </a>
            </td>
          <td>
            P. Loi, Amy E. Schade, Carrie L Rodriguez, Anjana Krishnan, N. Perurena, Van T M Nguyen, Yilin Xu, Marina Watanabe, Rachel A Davis, Alycia Gardner, Natalie F Pilla, Kaia Mattioli, Olesja Popow, Nuray Gunduz, T. Lannagan, Samantha Fitzgerald, Ewa T Sicinska, Jia-Ren Lin, William Tan, L. Brais, Kevin Haigis, M. Giannakis, K. Ng, Sandro Santagata, Kristian Helin, Owen J. Sansom, K. Cichowski
          </td>
          <td>2024-08-12</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Aberrant gene expression patterns in acute myeloid leukemia (AML) with balanced chromosomal translocations are often associated with dysregulation of epigenetic modifiers. The AML1/ETO (RUNX1/MTG8) fusion protein, caused by the translocation (8;21)(q22;q22), leads to the epigenetic repression of its target genes. We aimed in this work to identify critical epigenetic modifiers, on which AML1/ETO-positive AML cells depend on for proliferation and survival using shRNA library screens and global transcriptomics approaches. Using shRNA library screens, we identified 41 commonly depleted genes in two AML1/ETO-positive cell lines Kasumi-1 and SKNO-1. We validated, genetically and pharmacologically, DNMT1 and ATR using several AML1/ETO-positive and negative cell lines. We also demonstrated in vivo differentiation of myeloblasts after treatment with the DNMT1 inhibitor decitabine in a patient with an AML1/ETO-positive AML. Bioinformatic analysis of global transcriptomics after AML1/ETO induction in 9/14/18-U937 cells identified 973 differentially expressed genes (DEGs). Three genes (PARP2, PRKCD, and SMARCA4) were both downregulated after AML1/ETO induction, and identified in shRNA screens. In conclusion, using unbiased shRNA library screens and global transcriptomics, we have identified several driver epigenetic regulators for proliferation in AML1/ETO-positive AML. DNMT1 and ATR were validated and are susceptible to pharmacological inhibition by small molecules showing promising preclinical and clinical efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4848d5afaf1e1f3f5c753596a2ed8d1e491ca9f0" target='_blank'>
              Identification of epigenetic modifiers essential for growth and survival of AML1/ETO-positive leukemia.
              </a>
            </td>
          <td>
            J. Duque-Afonso, P. Veratti, Usama-Ur Rehman, Heike Herzog, J. Mitschke, G. Greve, Julian Eble, Bettina Berberich, Johanna Thomas, Milena Pantic, Miguel Waterhouse, Gaia Gentile, Olaf Heidenreich, Cornelius Miething, Michael Lübbert
          </td>
          <td>2024-08-15</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/761ad1cbab42eef53a71e8966883fae3c426f6a1" target='_blank'>
              Novel therapeutic agents in clinical trials: emerging approaches in cancer therapy
              </a>
            </td>
          <td>
            Deepak Chandra Joshi, Anurag Sharma, Sonima Prasad, Karishma Singh, Mayank Kumar, Kajal Sherawat, H. Tuli, Madhu Gupta
          </td>
          <td>2024-08-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR-T) cells that are obtained from the specific patient, modified genetically ex vivo, and possess the remarkable capability to identify and eradicate targeted cancer cells. These modified cells are subsequently reintroduced into the patient, effectively treating blood cancer. CAR-T therapy is approved to be applied in leukemia due to its great clinical therapeutic effect on B cell hematological malignancies. However, solid tumors are more resistant to this therapy for many reasons. The abnormal vascular structure of solid tumors hampers CAR-T cell trafficking. Various kinds of immunosuppressive cells and chemicals in the tumor microenvironment (TME) accelerates CAR-T cell exhaustion, showing poor persistence in vivo. More and more researches have demonstrated that T cell fate is strongly associated with epigenetic regulation. Epigenetic modification is not a direct addition or deletion of DNA, but a reversible method including modifications on DNA and histones, and non-coding RNA (ncRNA)-mediated regulations. The change of epigenetic landscape in CAR-T cells largely determines the therapeutic performance in vivo. This research outlines three major barriers in CAR-T therapy, including T cell exhaustion, differentiation and infiltration. Additionally, the research elucidates several promising epigenetic reprogramming strategies to reduce CAR-T cells exhaustion, modulate the cell differentiation process, and enhance their infiltration into solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4165f9901246426b44e74fa80485927b8d025d28" target='_blank'>
              Epigenetic Strategies to Optimize CAR-T Therapy
              </a>
            </td>
          <td>
            Danyang Song
          </td>
          <td>2024-07-11</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The XmnI Gg -158 C/T polymorphism has been widely associated with fetal hemoglobin (HbF) levels, the severity of disease, and the response to the drug hydroxyurea (HU) in both β-thalassemia (β-thal) and sickle cell disease (SCD) patients. However, the functional significance of this single nucleotide polymorphism (SNP) remains unclear. To gain insight, green fluorescence protein (GFP) cassettes harboring the XmnI C or T alleles in their left homology arms (i.e. Gg promoters) were knocked into the Gg gene(s) of K562 cells via CRISPR/Cas9. Subsequently, the GFP fluorescence levels were compared in the ensuing cell populations and isolated clones. In both instances, median fluorescence intensities (MFI) of the knockin cells having the inserted XmnI T allele were higher than those having the XmnI C allele. Our results suggest that the XmnI T allele can increase Gg expression in K562 cells. The possible functional significance of the XmnI Gg -158 C/T polymorphism provides a rationale for the aforementioned associations. Furthermore, the XmnI polymorphism as a functional SNP substantiates its importance as a prognostic marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bedc9fe1b28e8e9a54e59969786c8981bcb8830d" target='_blank'>
              Genome editing in K562 cells suggests a functional role for the XmnI Gg polymorphism: a widely used genetic marker in β-thalassemia and sickle cell disease patients.
              </a>
            </td>
          <td>
            A. Ahmadifard, Nahal Maroofi, Maryam Maleki Tehrani, Tahere Dabestani, Masoumeh Sadat Mousavi Maleki, Sepideh Bayrami, M. Banan
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Multidrug resistance (MDR) frequently occurs during cancer therapy and remains a major obstacle for the cure of most cancers(1, 2). Drug resistance could exist intrinsically or be acquired by drug treatment(3–5), yet factors that regulate the resistance remain elusive. Here, we show that most anticancer drugs damage neosynthesized proteins prior to reaching their canonic targets and elicit profound cytotoxicity, which is largely compensated by protein damage response (PDR). We demonstrate that the PDR includes damage recognition and clearance that are mainly mediated by ubiquitin and proteasome systems, although some other factors, including cellular ATP levels and proliferation status, are also involved. We show that cancer stem cells (CSCs), which have lower protein synthesis, and drug resistance acquired cells (DRAC), which have higher proteasome activity, are more resistant than other cells. We further demonstrate that ATP promotes protein synthesis and suppresses proteasome activity, thus, increasing mitochondrial ATP production by PDK1 inhibition and using proteasome inhibitor to block protein damage clearance render CSCs and DRACs more vulnerable to anticancer drugs. Thus, patients with drug-resistant cancers and treatment-naïve patients with low ATP levels and/or high proteasome activity can be identified and subtyped, and therapies containing PDK1-I and/or proteasome-I may be effective options for these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e9626795c580f97c2519d1342441e9145cbcfc" target='_blank'>
              A critical role of protein damage response in mediating cancer drug resistance
              </a>
            </td>
          <td>
            Fangyuan Shao, Jie Li, Hao Xiao, Ling Li, Bo Li, YuJun Chen, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Yuzhong Peng, Yujian Huang, Lijian Wang, Yanxia Shi, Nan Shao, Kai Miao, Changhua Zhang, Ying Lin, Jun Yan, Kin Yip Tam, Xiaoling Xu, Chu-Xia Deng
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Clonality assessment is currently the major molecular analysis utilized to support the diagnosis of suspicious lymphoid malignancies. Clonal rearrangements of the V-J segments of T-cell receptor G chain locus (TCRγ or TRG) have been observed in almost all types of T neoplasms, such as T-cell-related non-Hodgkin lymphomas and leukemias. At present, the gold standard for clonality evaluation is multiplex polymerase chain reaction (PCR), plus subsequent capillary electrophoresis/heteroduplex analyses, and/or Sanger sequencing. This approach overcomes the problem with the conventional Southern blot hybridization and is more efficient, simple, fast, and reproducible. In the recent years, the new next-generation sequencing (NGS) technologies provided alternative techniques for the analysis of antigen receptors genes, which presented several advantages, such as increased efficiency, specificity (SP), sensitivity (ST), resolution, and objectivity of the results, leading to a better classification, stratification, and monitoring of lymphoid malignancies. Nonetheless, these technologies are still far from being the new gold standard since further studies are warranted to prove their utility. The present study aimed to assess the diagnostic accuracy of these two methods by comparing a commercial NGS-based assay for the evaluation of TRG locus with the gold standard PCR-based one, to fulfill the requirements of a phase 3 diagnostic accuracy study. Methods: We assessed the TRG gene rearrangements in 72 cases using the conventional and highly-validated PCR-based assay proposed by EuroClonality consortium, an alternative commercial PCR-based assay, namely, IdentiClone® TCR Gamma Gene Rearrangement Assay 2.0, and a commercial NGS-based assay, that is, Invivoscribe LymphoTrack® Dx MiSeq® (both by Invivoscribe Technologies Inc., San Diego, CA, USA), to determine the diagnostic accuracy of the latter, and compare them with reference diagnoses made based on observation of clinical manifestations, cytohistological, and immunohistochemical analyses. Statistical values were calculated using the Oxford CATmaker software package. Results: Using standardized criteria of interpretation, the obtained results showed a diagnostic accuracy of 90.3% (correspondence in 65 out of 72 cases) of the test under investigation, with a ST of 86%, a SP of 95%, a positive predicting value of 94%, and a negative predicting value of 88%, demonstrating that Invivoscribe LymphoTrack® Dx MiSeq® assay had high efficiency and reliability in detecting clonal TRG gene rearrangements in T-cell non-Hodgkin lymphomas. Conclusions: This diagnostic accuracy study yielded comparable results using a validated PCR-based approach and a new NGS-based one. Subsequent studies and cost-effectiveness evaluation are needed to put the NGS-based clonality assessment into routine diagnostic practice.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2544bd060f9d54663078977efefd9d44ca908208" target='_blank'>
              Conventional PCR-based versus next-generation sequencing-based approach for T-cell receptor γ gene clonality assessment in mature T-cell lymphomas: A phase 3 diagnostic accuracy study
              </a>
            </td>
          <td>
            Riccardo Donelli, A. Gazzola, C. Mannu, Maryam Etebari, M. Navari, P. Piccaluga
          </td>
          <td>2024-07-10</td>
          <td>Journal of Biological Methods</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="CRISPR-Cas9 has revolutionized genome engineering by allowing precise introductions of DNA double-strand breaks (DSBs). However, genome engineering in bacteria is still a complex, multi-step process requiring a donor DNA template for homology-directed repair of DSBs. Prime editing circumvents this need as the repair template is provided within the prime editing guide RNA (pegRNA). Here, we developed make-or-break Prime Editing (mbPE) that allows for precise and effective genetic engineering in the opportunistic human pathogen Streptococcus pneumoniae. In contrast to traditional prime editing in which a nicking Cas9 is employed, mbPE harnesses wild type Cas9 in combination with a pegRNA that destroys the seed region or protospacer adjacent motif. Since most bacteria poorly perform template-independent end joining, correctly genome-edited clones are selectively enriched during mbPE. We show that mbPE can be used to introduce point mutations, deletions and targeted insertions, including protein tags such as a split luciferase, at selection efficiencies of over 93%. mbPE enables sequential genome editing, is scalable, and can be used to generate pools of mutants in a high-throughput manner. The mbPE system and pegRNA design guidelines described here will ameliorate future bacterial genome editing endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e8cb2090cc4138a6d6d7cd040330c66ebea98de" target='_blank'>
              Make-or-break prime editing for bacterial genome engineering
              </a>
            </td>
          <td>
            Monica Rengifo-Gonzalez, Maria-Vittoria Mazzuoli, Axel B. Janssen, Anne-Stéphanie Rueff, Xue Liu, J. Veening
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii Der, Ida Chen, C. Charles, Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>None</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="Abstract Radiotherapy is a key treatment option for a wide variety of human tumors, employed either alone or alongside with other therapeutic interventions. Radiotherapy uses high-energy particles to destroy tumor cells, blocking their ability to divide and proliferate. The effectiveness of radiotherapy is due to genetic and epigenetic factors that determine how tumor cells respond to ionizing radiation. These factors contribute to the establishment of resistance to radiotherapy, which increases the risk of poor clinical prognosis of patients. Although the mechanisms by which tumor cells induce radioresistance are unclear, evidence points out several contributing factors including the overexpression of DNA repair systems, increased levels of reactive oxygen species, alterations in the tumor microenvironment, and enrichment of cancer stem cell populations. In this context, dysregulation of microRNAs or miRNAs, critical regulators of gene expression, may influence how tumors respond to radiation. There is increasing evidence that miRNAs may act as sensitizers or enhancers of radioresistance, regulating key processes such as the DNA damage response and the cell death signaling pathway. Furthermore, expression and activity of miRNAs have shown informative value in overcoming radiotherapy and long-term radiotoxicity, revealing their potential as biomarkers. In this review, we will discuss the molecular mechanisms associated with the response to radiotherapy and highlight the central role of miRNAs in regulating the molecular mechanisms responsible for cellular radioresistance. We will also review radio-miRs, radiotherapy-related miRNAs, either as sensitizers or enhancers of radioresistance that hold promise as biomarkers or pharmacological targets to sensitize radioresistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f11e14f281c48ffc0ef7d72af9b361e4a40e0c9" target='_blank'>
              Radio-miRs: a comprehensive view of radioresistance-related microRNAs
              </a>
            </td>
          <td>
            Abraham Pedroza-Torres, Sandra L Romero-Córdoba, S. Montaño, O. Peralta-Zaragoza, Dora Emma Vélez-Uriza, Cristian Arriaga-Canon, Xiadani Guajardo-Barreto, Diana Bautista-Sánchez, Rodrigo Sosa-León, Olivia Hernández-González, J. Díaz-Chávez, R. M. Álvarez-Gómez, Luis A Herrera
          </td>
          <td>2024-07-04</td>
          <td>Genetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3ef3973d270e32f48aaa19a00bef4e18bca5be" target='_blank'>
              Research Progress of Long Non-coding RNA-ZFAS1 in Malignant Tumors.
              </a>
            </td>
          <td>
            Xin Liu, Zhong Ma, Xianxu Zhang, Shicheng Li, Jiangdong An, Zhiqiang Luo
          </td>
          <td>2024-07-26</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="While the triggers for the metastatic transformation of breast cancer (BC) cells remain unknown, recent evidence suggests that intrinsic cellular metabolism could be a crucial driver of migratory disposition and chemoresistance. Aiming to decode the molecular mechanisms involved in BC cell metabolic maneuvering, we study how a ketomimetic (ketone body-rich, low glucose) medium affects Doxorubicin (DOX) susceptibility and invasive disposition of BC cells. We quantified glycocalyx sialylation and found an inverse correlation with DOX-induced cytotoxicity and DOX internalization. These measurements were coupled with single-cell metabolic imaging, bulk migration studies, along with transcriptomic and metabolomic analyses. Our findings revealed that a ketomimetic medium enhances chemoresistance and invasive disposition of BC cells via two main oncogenic pathways: hypersialylation and lipid synthesis. We propose that the crosstalk between these pathways, juxtaposed at the synthesis of the glycan precursor UDP-GlcNAc, furthers advancement of a metastatic phenotype in BC cells under ketomimetic conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fa6f2b5143747f6467f4d23175994fac910eade" target='_blank'>
              Ketomimetic Nutrients Trigger a Dual Metabolic Defense in Breast Cancer Cells
              </a>
            </td>
          <td>
            Mohini Kamra, Yuan-I Chen, Paula C. Delgado, Erin H. Seeley, S. Seidlits, Hsin-Chin Yeh, Amy Brock, S. Parekh
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c24f0b42670eaa7d910971701aacf3a8d5213b" target='_blank'>
              Targeting RBM39 through indisulam induced mis-splicing of mRNA to exert anti-cancer effects in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tongting Ji, Yang Yang, Juanjuan Yu, Hongli Yin, Xinran Chu, Pengju Yang, Ling Xu, Xiaodong Wang, Shaoyan Hu, Yizhen Li, Xiaochen Wu, Wengyuan Liu, Bi Zhou, Wenjuan Wang, Shuqi Zhang, Wei Cheng, Yanling Chen, Lei Shi, Zhiheng Li, Ran Zhuo, Yongping Zhang, Yanfang Tao, Di Wu, Xiao-Lu Li, Zimu Zhang, Jun-Jie Fan, Jian Pan, Jun Lu
          </td>
          <td>2024-07-24</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Introduction. It has been established that the presence of homologous recombination deficiency in a breast tumor is associated with the effectiveness of treatment. But despite the high chemosensitivity of the tumor to DNA-damaging agents, complete pathological responses to treatment are very rare. And this process may be based on a change in the somatic status of BRCA1, that is, a reversion and return of the wild-type allele occurs and the DNA repair function is restored.Aim. To evaluate changes in the presence of chromosomal aberrations and the expression profile of the main genes of homologous recombination in cell models of breast cancer under the influence of cisplatin and docetaxel.Materials and methods. The study was conducted on breast cancer tumor cell cultures: MCF-7, MDA-MB-231 and MDA-MB-468. A cell model of drug resistance was obtained for two drugs: cisplatin and docetaxel. RNA and DNA were isolated from cell suspension using the RNeasy Plus Mini Kit and QIAamp DNA Mini Kit (Qiagen, Germany), respectively. The expression level of homologous recombination genes was assessed using reverse transcription polymerase chain reaction. To assess the presence of chromosomal aberrations, microarray analysis was performed on DNA chips.Results. Restoration of normal copy number for the BRCA1, CDK12, CHEK1 and RAD51D genes in MCF-7 under the influence of cisplatin was shown. For BRCA2 and PALB2, amplifications were detected. A statistically significant increase in the expression of the BRCA1 (p = 0.04), BRCA2 (p = 0.02), PALB2 (p = 0.01) and RAD51D (p = 0.05) genes was also shown. MDAMB-231 shows that all identified loci with deletions, where the BRCA2, BARD1, CHEK2, PALB2 and RAD54L genes are localized, are restored to normal copy number by cisplatin. The appearance of amplifications was registered for BRCA1, BRIP1, FANCL, RAD51B, PARP1. A similar result was shown for docetaxel. An increase in the expression level is typical for the genes BRCA1 (p = 0.02), BRCA2 (p = 0.02), CHEK2 (p = 0.05), FANCL (p = 0.04), PALB2 (p = 0.05), RAD51C (p = 0.02), PARP1 (p = 0.02), which corresponds to the appearance of amplifications. In the MDA-MB-468 cell culture, an increase in the copy number of only the BRCA1 gene is observed. The effect of docetaxel has no effect on this cell culture. The level of BRCA1 expression increases in direct proportion to the duration of drug action.Conclusion. Thus, the study showed that under the influence of cisplatin, reversion of not only homologous recombination gene mutations, but also other disorders can occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/767d30042e565f8e61dafca9671915ee6d03ecf7" target='_blank'>
              Modulation of homologous recombination gene activity in breast tumor cells in an in vitro model
              </a>
            </td>
          <td>
            M. Tsyganov, A. Frolova, E. A. Kravtsova, I. A. Tsydenova, M. Ibragimova
          </td>
          <td>2024-06-28</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="ABSTRACT Ras genes are important oncogenes that are frequently mutated in cancer. Human oncogenic variants exhibit functional distinctions in terms of their representation in different cancer types, impact on cellular targets and sensitivity to pharmacological treatments. However, how these distinct variants influence and respond to the cellular networks in which they are embedded is poorly understood. To identify novel participants in the complex interplay between Ras genotype and cell interaction networks in vivo, we have developed and tested an experimental framework using a simple vulva-development assay in the nematode C. elegans. Using this system, we evaluated a set of Ras oncogenic substitution changes at G12, G13 and Q61. We found that these variants fall into distinct groups based on phenotypic differences, sensitivity to gene dosage and inhibition of the downstream kinase MEK and their response to genetic modulators that influence Ras activity in a non-autonomous manner. Together, our results demonstrated that oncogenic C. elegans Ras variants exhibit clear distinctions in how they interface with the vulva-development network and showed that extracellular modulators yield variant-restricted effects in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/675fab343bc71b75d1eb1f6e179c6e3a20afa2a0" target='_blank'>
              Functional distinction in oncogenic Ras variant activity in Caenorhabditis elegans
              </a>
            </td>
          <td>
            Haimeng Lyu, H. Chamberlin
          </td>
          <td>2024-07-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Motivation Predicting cancer drug response requires a comprehensive assessment of many mutations present across a tumor genome. While current drug response models generally use a binary mutated/unmutated indicator for each gene, not all mutations in a gene are equivalent. Results Here, we construct and evaluate a series of predictive models based on leading methods for quantitative mutation scoring. Such methods include VEST4 and CADD, which score the impact of a mutation on gene function, and CHASMplus, which scores the likelihood a mutation drives cancer. The resulting predictive models capture cellular responses to dabrafenib, which targets BRAF-V600 mutations, whereas models based on binary mutation status do not. Performance improvements generalize to other drugs, extending genetic indications for PIK3CA, ERBB2, EGFR, PARP1, and ABL1 inhibitors. Introducing quantitative mutation features in drug response models increases performance and mechanistic understanding. Availability and implementation Code and example datasets are available at https://github.com/pgwall/qms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b78691ef04081b6ccf1c8699cd4747a45bf1d4f" target='_blank'>
              Representing mutations for predicting cancer drug response
              </a>
            </td>
          <td>
            Patrick Wall, T. Ideker
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="OBJECTIVE
Acute Lymphoblastic Leukemia (ALL) is the most common malignancy occurring in children. Copy number alterations (CNA) like PAX5, CDKN2A/2B, PAR1 Region, ETV6, IKZF1, BTG1, and RB1 gene deletion are important genetic events that define and prognosticate B-cell ALL. Thus, this study aimed to evaluate associations of CNA with induction phase remission status in childhood B-cell ALL.


METHODS
This study was observational with a cross-sectional design at the Dharmais Cancer Hospital, Harapan Kita Mother and Children Hospital, and Tangerang Regional Public Hospital. We evaluated 74 pediatric B-cell ALL cases with 1-18-year-olds. Genomic DNA was analyzed by Multiplex Ligation Dependent Probe Amplification Assay (MLPA). This study used the P335 ALL-IKZF1 panel kit, which contains several ALL-related genes. The patient's clinical and laboratory characteristics were collected from medical records from January to December 2019.


RESULT
We observed gene copy number alteration in children with B-Cell ALL. PAX5 was the most commonly observed gene deletion, followed by CDKN21/2B, ETV6, IKZF1, BTG1, RB1, and PAR1 Region. Based on gene mutations, only the PAX5 had a significant association with the remission status of pediatric B-cell ALL (p-value <0.05; OR = 3.91). It showed that patients with PAX5 gene mutations have 3.9 times the risk of no remission and/or relapse compared to those without PAX5 gene mutations.


CONCLUSION
Patients with mutations in the PAX5 gene have a higher chance of not achieving remission and/or experiencing relapse than those without such mutations. The MLPA method can be utilized for examining copy number alterations, which is valuable for achieving more precise stratification in diagnosis.. Further research is needed to expand upon this finding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd752c14237c4bf075eb8efe30658a5f3ec960c" target='_blank'>
              Unraveling Copy Number Alterations in Pediatric B-Cell Acute Lymphoblastic Leukemia: Correlation with Induction Phase Remission Using MLPA.
              </a>
            </td>
          <td>
            M. Aisyi, Murti Andriastuti, A. Kosasih, Ahmad Rusdan Handoyo Utomo, Fahreza Saputra, Teny Tjitra Sari, H. Sjakti, Fifi Dwijayanti, K. Harimurti, D. Gatot
          </td>
          <td>2024-07-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract The ability to deliver large transgenes to a single genomic sequence with high efficiency would accelerate biomedical interventions. Current methods suffer from low insertion efficiency and most rely on undesired double-strand DNA breaks. Serine integrases catalyze the insertion of large DNA cargos at attachment (att) sites. By targeting att sites to the genome using technologies such as prime editing, integrases can target safe loci while avoiding double-strand breaks. We developed a method of phage-assisted continuous evolution we call IntePACE, that we used to rapidly perform hundreds of rounds of mutagenesis to systematically improve activity of PhiC31 and Bxb1 serine integrases. Novel hyperactive mutants were generated by combining synergistic mutations resulting in integration of a multi-gene cargo at rates as high as 80% of target chromosomes. Hyperactive integrases inserted a 15.7 kb therapeutic DNA cargo containing von Willebrand Factor. This technology could accelerate gene delivery therapeutics and our directed evolution strategy can easily be adapted to improve novel integrases from nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1115be639d45f9efbece4cb3bd061894c38e5687" target='_blank'>
              Directed evolution of hyperactive integrases for site specific insertion of transgenes
              </a>
            </td>
          <td>
            Brian E. Hew, Sabranth Gupta, Ryuei Sato, David F. Waller, I. Stoytchev, James E. Short, Lisa Sharek, Christopher T. Tran, Ahmed H. Badran, Jesse B Owens
          </td>
          <td>2024-07-02</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac57fc3852f4e168d66236506c1da4f107212ec1" target='_blank'>
              A CRISPR/Cas9 screen in embryonic stem cells reveals that Mdm2 regulates totipotency exit
              </a>
            </td>
          <td>
            Chen Gao, Xingke Gao, Fei Gao, Xuguang Du, Sen Wu
          </td>
          <td>2024-07-03</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Emerging immunotherapies such as immune checkpoint blockade (ICB) and chimeric antigen receptor T-cell (CAR-T) therapy have revolutionized cancer treatment and have improved the survival of patients with multiple cancer types. Despite this success many patients are unresponsive to these treatments or relapse following treatment. CRISPR activation and knockout (KO) screens have been used to identify novel single gene targets that can enhance effector T cell function and promote immune cell targeting and eradication of tumors. However, cancer cells often employ multiple genes to promote an immunosuppressive pathway and thus modulating individual genes often has a limited effect. Paralogs are genes that originate from common ancestors and retain similar functions. They often have complex effects on a particular phenotype depending on factors like gene family similarity, each individual gene’s expression and the physiological or pathological context. Some paralogs exhibit synthetic lethal interactions in cancer cell survival; however, a thorough investigation of paralog pairs that could enhance the efficacy of cancer immunotherapy is lacking. Here we introduce a sensitive computational approach that uses sgRNA sets enrichment analysis to identify cancer-intrinsic paralog pairs which have the potential to synergistically enhance T cell-mediated tumor destruction. We have further developed an ensemble learning model that uses an XGBoost classifier and incorporates features such as gene characteristics, sequence and structural similarities, protein-protein interaction (PPI) networks, and gene coevolution data to predict paralog pairs that are likely to enhance immunotherapy efficacy. We experimentally validated the functional significance of these predicted paralog pairs using double knockout (DKO) of identified paralog gene pairs as compared to single gene knockouts (SKOs). These data and analyses collectively provide a sensitive approach to identify previously undetected paralog pairs that can enhance cancer immunotherapy even when individual genes within the pair has a limited effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9398918df5088c04d21b0123727e4ddeb607a57" target='_blank'>
              Sensitive detection of synthetic response to cancer immunotherapy driven by gene paralog pairs
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Emily He, Ping Ren, Nipun Verma, Xinxin Zhu, Di Feng, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="T-prolymphocytic leukemia (T-PLL) is a mature T-cell neoplasm associated with marked chemotherapy resistance and continued poor clinical outcomes. Current treatments, i.e. the CD52-antibody alemtuzumab, offer transient responses, with relapses being almost inevitable without consolidating allogeneic transplantation. Recent more detailed concepts of T-PLL's pathobiology fostered the identification of actionable vulnerabilities: (i) altered epigenetics, (ii) defective DNA damage responses, (iii) aberrant cell-cycle regulation, and (iv) deregulated pro-survival pathways, including TCR and JAK/STAT signaling. To further develop related pre-clinical therapeutic concepts, we studied inhibitors of (H)DACs, BCL2, CDK, MDM2, and clas-sical cytostatics, utilizing (a) single-agent and combinatorial compound testing in 20 well-characterized and molecularly-profiled primary T-PLL (validated by additional 42 cases), and (b) 2 independent murine models (syngeneic transplants and patient-derived xenografts). Overall, the most efficient/selective single-agents and combinations (in vitro and in mice) in-cluded Cladribine, Romidepsin ((H)DAC), Venetoclax (BCL2), and/or Idasanutlin (MDM2). Cladribine sensitivity correlated with expression of its target RRM2. T-PLL cells revealed low overall apoptotic priming with heterogeneous dependencies on BCL2 proteins. In additional 38 T-cell leukemia/lymphoma lines, TP53 mutations were associated with resistance towards MDM2 inhibitors. P53 of T-PLL cells, predominantly in wild-type configuration, was amenable to MDM2 inhibition, which increased its MDM2-unbound fraction. This facilitated P53 activa-tion and down-stream signals (including enhanced accessibility of target-gene chromatin re-gions), in particular synergy with insults by Cladribine. Our data emphasize the therapeutic potential of pharmacologic strategies to reinstate P53-mediated apoptotic responses. The identified efficacies and their synergies provide an informative background on compound and patient selection for trial designs in T-PLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28856a6bd0f85fac41c622c5fe8a6c6a100dae44" target='_blank'>
              Optimizing Drug Combinations for T-PLL: Restoring DNA Damage and P53-Mediated Apoptotic Responses.
              </a>
            </td>
          <td>
            J. von Jan, S. Timonen, T. Braun, Qu Jiang, Aleksandr Ianevski, Yayi Peng, Kathleen McConnell, Paola Sindaco, T. Müller, S. Pützer, Hanna Klepzig, Dennis Jungherz, Annika Dechow, Linus Wahnschaffe, Anil K Giri, M. Kankainen, H. Kuusanmäki, H. Neubauer, Richard H Moriggl, P. Mazzeo, Nicole Schmidt, Raphael Koch, Michael Hallek, A. Chebel, David Armisén, Laurent Genestier, E. Bachy, Anjali Mishra, A. Schrader, T. Aittokallio, S. Mustjoki, Marco Herling
          </td>
          <td>2024-06-28</td>
          <td>Blood</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Hypoxia contributes to tumor progression and metastasis, and hypoxically dysregulated RNA molecules may, thus, be implicated in poor outcomes. Canine oral melanoma (COM) has a particularly poor prognosis, and some hypoxia-mediated miRNAs are known to exist in this cancer; however, equivalent data on other hypoxically dysregulated non-coding RNAs (ncRNAs) are lacking. Accordingly, we aimed to elucidate non-miRNA ncRNAs that may be mediated by hypoxia, targeting primary-site and metastatic COM cell lines and clinical COM tissue samples in next-generation sequencing (NGS), with subsequent qPCR validation and quantification in COM primary and metastatic cells and plasma and extracellular vesicles (EVs) for any identified ncRNA of interest. The findings suggest that a number of non-miRNA ncRNA species are hypoxically up- or downregulated in COM. We identified one ncRNA, the long ncRNA fragment ENSCAFT00000084705.1, as a molecule of interest due to its consistent downregulation in COM tissues, hypoxically and normoxically cultured primary and metastatic cell lines, when compared to the oral tissues from healthy dogs. However, this molecule was undetectable in plasma and plasma EVs, suggesting that its expression may be tumor tissue-specific, and it has little potential as a biomarker. Here, we provide evidence of hypoxic transcriptional dysregulation for ncRNAs other than miRNA in COM for the first time and suggest that ncRNA ENSCAFT00000084705.1 is a molecule of interest for future research on the role of the transcriptome in the hypoxia-mediated progression of this aggressive cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c9166eb36c9cc287e103c53abc4362015b387f3" target='_blank'>
              Hypoxia-Mediated Long Non-Coding RNA Fragment Identified in Canine Oral Melanoma through Transcriptome Analysis
              </a>
            </td>
          <td>
            Yasunori Hino, Mohammad Arif, Md Mahfuzur Rahman, A. Husna, Md Nazmul Hasan, Naoki Miura
          </td>
          <td>2024-08-09</td>
          <td>Veterinary Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is an incurable malignancy characterised by altered expression of coding and non-coding genes promoting tumour growth and drug resistance. Although the crucial role of long non-coding RNAs (lncRNAs) in MM is clearly established, the function of the non-coding RNAome, which might allow the design of novel therapeutics, is largely unknown. We performed an unbiased CRISPR-Cas9 loss-of-function screen of 671 lncRNAs in MM cells and their Bortezomib (BZB)-resistant derivative. To rank functionally and clinically relevant candidates, we designed and used a bioinformatic prioritisation pipeline combining functional data from cellular screens with prognostic and transcriptional data from MM patients. With this approach, we unveiled and prioritised 8 onco-lncRNAs essential for MM cell fitness, associated with high expression and poor prognosis in MM patients. The previously uncharacterised RP11-350G8.5 emerged as the most promising target, irrespective of BZB resistance. We i) demonstrated the anti-tumoral effect obtained by RP11-350G8.5 inhibition in vitro and in vivo; ii) highlighted a modulation of the unfolded protein response and the induction of immunogenic cell death triggered by the RP11-350G8.5 knock-out, via RNA-sequencing and molecular studies; iii) characterised its cytoplasmic homing through RNA-FISH; iv) predicted its 2D structure and identified 2 G-quadruplex and 3 hairpin-forming regions by biophysical assays, including Thioflavin T, 1H-NMR and circular dichroism to pave the way to the development of novel targeted therapeutics. Overall we provided innovative insights about unexplored lncRNAs in MM and identified RP11-350G8.5 as an oncogenic target for treatment-naïve and BZB-resistant MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b3038ab25be0410e0d1aec773f7b5564cf954f8" target='_blank'>
              An unbiased lncRNAs dropout CRISPR-Cas9 screen reveals RP11-350G8.5 as a novel therapeutic target for Multiple Myeloma.
              </a>
            </td>
          <td>
            K. Grillone, S. Ascrizzi, Paolo Cremaschi, Jussara Amato, N. Polerà, Ottavio Croci, R. Rocca, C. Riillo, Francesco Conforti, Raffaele Graziano, Diego Brancaccio, Daniele Caracciolo, Stefano Alcaro, Bruno Pagano, Antonio Randazzo, P. Tagliaferri, F. Iorio, P. Tassone
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52ab4a838ea1b172020974ecfeeabbd74706716e" target='_blank'>
              Polyamine and EIF5A hypusination downstream of c-Myc confers targeted therapy resistance in BRAF mutant melanoma
              </a>
            </td>
          <td>
            Byung-Sun Park, Heeju Jeon, Yeonseo Kim, Haejin Kwon, Ga-Eun Choi, Sung-Gil Chi, H. Park, Hyunbeom Lee, Tackhoon Kim
          </td>
          <td>2024-07-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Human endogenous retroviruses (HERVs), the remnants of ancient germline retroviral integrations, comprise almost 8% of the human genome. The elucidation of their biological roles is hampered by our inability to link HERV mRNA and protein production with specific HERV loci. To solve the riddle of the integration-specific RNA expression of HERVs, several bioinformatics approaches have been proposed; however, no single process seems to yield optimal results due to the repetitiveness of HERV integrations. The performance of existing data-bioinformatics pipelines has been evaluated against real world datasets whose true expression profile is unknown, thus the accuracy of widely-used approaches remains unclear. Here, we simulated mRNA production from specific HERV integrations to evaluate second and third generation sequencing technologies along with widely used bioinformatic approaches to estimate the accuracy in describing integration-specific expression. We demonstrate that, while a HERV-family approach offers accurate results, per-integration analyses of HERV expression suffer from substantial expression bias, which is only partially mitigated by algorithms developed for calculating the per-integration HERV expression, and is more pronounced in recent integrations. Hence, this bias could erroneously result into biologically meaningful inferences. Finally, we demonstrate the merits of accurate long-read high-throughput sequencing technologies in the resolution of per-locus HERV expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3e71c5502b568bd36436f0dbbe38d91d92145df" target='_blank'>
              Limitations of current high-throughput sequencing technologies lead to biased expression estimates of endogenous retroviral elements
              </a>
            </td>
          <td>
            Konstantina Kitsou, A. Katzourakis, G. Magiorkinis
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT The adaptation of gene deletion methods based on the CRISPR-Cas9 system has facilitated the genetic manipulation of the pathogenic yeast Candida albicans, because homozygous mutants of this diploid fungus can now be generated in a single step, allowing the rapid screening of candidate genes for their involvement in a phenotype of interest. However, the Cas9-mediated double-strand breaks at the target site may result in an undesired loss of heterozygosity (LOH) on the affected chromosome and cause phenotypic alterations that are not related to the function of the investigated gene. In our present study, we harnessed Cas9-facilitated gene deletion to probe a set of genes that are constitutively overexpressed in strains containing hyperactive forms of the transcription factor Mrr1 for a possible contribution to the fluconazole resistance of such strains. To this aim, we used gene deletion cassettes containing two different dominant selection markers, caSAT1 and HygB, which confer resistance to nourseothricin and hygromycin, respectively, for simultaneous genomic integration in a single step, hypothesizing that this would minimize undesired LOH events at the target locus. We found that selection for resistance to both nourseothricin and hygromycin strongly increased the proportion of homozygous deletion mutants among the transformants compared with selection on media containing only one of the antibiotics, but it did not avoid undesired LOH events. Our results demonstrate that LOH on the target chromosome is a significant problem when using Cas9 for the generation of C. albicans gene deletion mutants, which demands a thorough examination of recombination events at the target site. IMPORTANCE Candida albicans is one of the medically most important fungi and a model organism to study fungal pathogenicity. Investigating gene function in this diploid yeast has been facilitated by the adaptation of gene deletion methods based on the bacterial CRISPR-Cas9 system, because they enable the generation of homozygous mutants in a single step. We found that, in addition to increasing the efficiency of gene replacement by selection markers, the Cas9-mediated double-strand breaks also result in frequent loss of heterozygosity on the same chromosome, even when two different selection markers were independently integrated into the two alleles of the target gene. Since loss of heterozygosity for other genes can result in phenotypic alterations that are not caused by the absence of the target gene, these findings show that it is important to thoroughly analyze recombination events at the target locus when using Cas9 to generate gene deletion mutants in C. albicans. Candida albicans is one of the medically most important fungi and a model organism to study fungal pathogenicity. Investigating gene function in this diploid yeast has been facilitated by the adaptation of gene deletion methods based on the bacterial CRISPR-Cas9 system, because they enable the generation of homozygous mutants in a single step. We found that, in addition to increasing the efficiency of gene replacement by selection markers, the Cas9-mediated double-strand breaks also result in frequent loss of heterozygosity on the same chromosome, even when two different selection markers were independently integrated into the two alleles of the target gene. Since loss of heterozygosity for other genes can result in phenotypic alterations that are not caused by the absence of the target gene, these findings show that it is important to thoroughly analyze recombination events at the target locus when using Cas9 to generate gene deletion mutants in C. albicans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9746ed21f0eca5161e3fdd0eda593a6edeb2e8a6" target='_blank'>
              Probing gene function in Candida albicans wild-type strains by Cas9-facilitated one-step integration of two dominant selection markers: a systematic analysis of recombination events at the target locus
              </a>
            </td>
          <td>
            Bernardo Ramírez-Zavala, Anna Hoffmann, Ines Krüger, S. Schwanfelder, K. Barker, P. D. Rogers, J. Morschhäuser
          </td>
          <td>2024-06-28</td>
          <td>mSphere</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="For several decades, induction therapy with nucleoside analogs, in particular cytarabine (Ara-C) and, to a lesser extent, fludarabine, has been the standard of care for patients diagnosed with acute myeloid leukemia (AML). Still, the anti-tumor efficacy of nucleoside analogs is often limited by intrinsic and acquired drug resistance, thereby leading to poor therapeutic response and suboptimal clinical outcomes. Here, we used genome-wide CRISPR-based pharmacogenomic screening to map the genetic factors that modulate the response to nucleoside analogs in AML and identified the E3 ubiquitin ligase Herc1 as a key modulator of Ara-C response in the MLL/AF9 (MA) and the HOXA9/MEIS1 (HM) murine AML models both in vitro and in vivo. Loss of HERC1 enhanced nucleoside analog-induced cell death in both murine and human AML cell lines by compromising cell cycle progression. In-depth proteomic analysis and subsequent validation identified deoxycytidine kinase Dck as a novel target of Herc1 in MA and HM murine cells. We observed that HERC1 is overexpressed in AML compared to other cancer types and higher HERC1 expression is associated with shorter overall survival of patients with AML in the TCGA and BEAT-AML cohorts. Collectively, this study highlights the importance of HERC1 in the response of AML cells to nucleoside analogs, thereby establishing this E3 ubiquitin ligase as a novel predictive biomarker and potential therapeutic target for the treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5245f1077766a809fae0247565289dd67c6fcad" target='_blank'>
              The E3 ubiquitin ligase Herc1 modulates the response to nucleoside analogs in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Maja Jankovic, William Wl Poon, Cristobal Gonzales-Losada, Gabriela Galicia Vazquez, Bahram Sharif-Askari, Yi Ding, Constance Craplet-Desombre, Alexandru Ilie, Jiantao Shi, Yongie Wang, A. Jayavelu, Alexandre Orthwein, F. E. Mercier
          </td>
          <td>2024-08-02</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Ovarian cancer (OC) poses a significant health risk to women worldwide, with late diagnoses and chemotherapy resistance leading to high mortality rates. Despite several histological subtypes, the primary challenge remains the subtle nature of its symptoms, resulting in advanced-stage diagnosis and reduced treatment success rates. With platinum-based therapies showing relative efficacy but limited survival enhancements, the emergence of chemotherapy resistance during recurrence remains a critical obstacle. Precision medicine development has aimed to address these challenges in the context of the molecular diversity of OC. The present review explored the landscape of microRNA (miRNA)-mediated approaches in OC treatment. miRNAs have emerged as regulators of gene expression, serving as both oncogenes and tumor suppressors in OC. Dysregulated miRNAs are associated with disease progression and chemotherapy resistance, underscoring their significance in diagnosis and tailored treatment strategies. The present review extracted 295 publications from the PUBMED database. Out of the 73 eligible studies, 55 miRNAs were assessed. A total of three of these miRNAs were not associated with any disease or cancer, whilst eight were associated with OC, albeit also associated with other diseases. The present review encompassed three dimensions: i) The role of miRNAs in treatment efficacy; ii) the use of miRNAs to enhance therapy outcomes; and iii) adjunctive strategies for improved treatment results. Furthermore, it offered insights into potential avenues for improving OC treatment using miRNA-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e04bb3c4e63f1fb22f64e7d076de1f329b943b90" target='_blank'>
              MicroRNA‑mediated approaches in ovarian cancer therapy: A comprehensive systematic review
              </a>
            </td>
          <td>
            Henny Meitri Andrie Rachmasari Putri, Putri Wikie Novianti, Heru Pradjatmo, S. Haryana
          </td>
          <td>2024-08-12</td>
          <td>Oncology Letters</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background: Integrating immune checkpoint inhibitors with multi-target tyrosine kinase inhibitors presents an innovative and hopeful strategy in liver cancer treatment. Nonetheless, a degree of resistance to this treatment is noticeable in certain patients. Alternative splicing (AS) represents a common biological process that controls the variety of life functions via isoforms. Purpose: Investigating how gene AS affects the effectiveness of combined immunotherapy in treating hepatocellular carcinoma (HCC). Methods: Our retrospective examination focused on AS's effect on immune therapy effectiveness, utilizing accessible tissue sequencing and clinical records for HCC. For corroborating our results, we gathered samples of drug-resistant HCC tissue, nearby tissues, HCC tissue with high drug responsiveness, and healthy liver tissue from clinical studies. Results: The study revealed a link between the frequency of AS occurrences, the expression levels of programmed cell death 1 ligand 1, and the resistance to tumor medications. Our study detailed the AS occurrences in HCC, leading to the creation of a risk-assessment function and a predictive model using AS data. The results of our study revealed that the risk score effectively distinguished between various immune subtypes and the effectiveness of immune therapy. Additional examination of the chosen AS occurrences uncovered their effects on both the immune microenvironment and cellular immunity. Our investigation also delved into the regulatory framework of AS, uncovering the role of stringently controlled splicing factors in the emergence of tumors and the modulation of the body's immune response. Conclusions: Increased AS in HCC diminishes the efficacy of immunotherapy; conversely, more AS in peritumoral tissue elevates the likelihood of tumor immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc39d2a7e8075c89a1f4a7ca29b083b1fc5166ab" target='_blank'>
              Prediction Model for Immunotherapy Efficacy in Hepatocellular Carcinoma Based on Alternative Splicing Sequencing Data
              </a>
            </td>
          <td>
            Huizhong Shi, Yang Liu, Zhao Liu, Xinlan Ge, Jushan Wu, Haowen Tang, Yi Zhang, Shi-yong Lu
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background The methyltransferase SETD2 was originally identified as a tumor suppressor in clear cell renal cell carcinoma (ccRCC), and ccRCC with SETD2 mutations have increased metastatic potential and poor progression free survival. A known epigenetic regulator, SETD2 interacts with actively transcribing RNA polymerase II (RNAPII) and tri-methylates Histone 3 at Lysine 36 (H3K36me3), a modification that correlates with active gene transcription and is recognized by mRNA methylation and splicing factors. Despite these known roles in mRNA transcription, loss of SETD2 does not alter genic transcription, raising the question of whether or how SETD2 regulates RNA biology. Methods Given that SETD2 influences RNA processing, we sought to identify RNA-protein complexes that are regulated by SETD2. To do this in an unbiased and high-throughput manner, we used orthogonal organic phase separation (OOPS) coupled with mass spectrometry to identify changes in RNA binding proteins and pathways between control and SETD2-knockout cells. Results Corroborating previous findings, we identified that RNA binding proteins governing mRNA splicing and cellular metabolism were dysregulated in SETD2 mutant cells. However, the most robust changes in SETD2 knockout cells occurred in pathways regulated ribosome biogenesis (See Figure, orange highlights), resulting in a significant decrease in binding of ribosomal RNA (rRNA) processing factors to RNA. We confirm these alterations lead to defects in rRNA transcription, processing, and ribosome biogenesis. We identify that loss of SETD2 catalytic activity phenocopies these rRNA processing defects, suggesting H3K36me3 is essential for rRNA processing. Disruptions occur to both the large (60S) and small (40S) ribosome subunits, modifying protein translation. Lastly, we identify synthetic lethality between SETD2 deletion or inhibition and small molecules that interfere with rRNA transcription. Conclusions While the majority of research emphasizes the role SETD2 in the regulation of mRNA, our data identify that SETD2 and its catalytic activity play a critical role in the transcription and processing of rRNA. Importantly, rRNA comprises 80-90% of cellular RNA yet is typically overlooked and understudied. Next-generation RNA sequencing or transcriptional assays investigate mRNA or genic transcription while ignoring rRNA, demonstrating why SETD2-mediated rRNA processing has been unexplored. This may also have important implications for prospective clinical trials that utilized RNA-sequencing to identify optimal therapeutic regimens. Our work also identified that SETD2-mutated cells are more sensitive to compounds that inhibit rRNA transcription, potentially uncovering a new therapeutic vulnerability of ccRCC with mono- and bi-allelic loss of SETD2. Alterations in ribosome biogenesis leads to epithelial-to-mesenchymal transition and metastasis in many tumor types, and our future goal will be to investigate the role of ribosomes and rRNA transcription in transformation and metastasis in ccRCC as well as the molecular mechanisms connecting SETD2 to ribosome fidelity. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30cb284d919d6bd0de496d90d21e1c0abae77b34" target='_blank'>
              51 Epigenetic regulation of ribosomal RNA transcription and processing by the tumor suppressor SETD2
              </a>
            </td>
          <td>
            Logan Vlach, Scott Haake, K. Rathmell, Frank M. Mason
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01413a6e42d02eb690fa59aa79101fc898cc9619" target='_blank'>
              RNA sequestration in P-bodies sustains myeloid leukaemia.
              </a>
            </td>
          <td>
            Srikanth Kodali, Ludovica Proietti, Gemma Valcárcel, Anna V. López-Rubio, Patrizia Pessina, Thomas Eder, Junchao Shi, Annie Jen, Núria Lupión-Garcia, Anne C. Starner, Mason Bartels, Yingzhi Cui, Caroline M Sands, Ainoa Planas-Riverola, Alba Martínez, T. Velasco-Hernández, Laureano Tomás-Daza, Bernhard Alber, G. Manhart, I. Mayer, K. Kollmann, A. Fatica, Pablo Menendez, Evgenia Shishkova, Rachel E Rau, B. Javierre, Joshua J. Coon, Qi Chen, Eric L. Van Nostrand, J. L. Sardina, F. Grebien, Bruno Di Stefano
          </td>
          <td>2024-08-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Targeting tumor metabolism through dietary interventions is an area of growing interest, and may help to improve the significant mortality of aggressive cancers, including non-small cell lung cancer (NSCLC). Here we show that the restriction of methionine in the aggressive KRAS/Lkb1-mutant NSCLC autochthonous mouse model drives decreased tumor progression and increased carboplatin treatment efficacy. Importantly, methionine restriction during early stages of tumorigenesis prevents the lineage switching known to occur in the model, and alters the tumor immune microenvironment (TIME) to have fewer tumor-infiltrating neutrophils. Mechanistically, mutations in LKB1 are linked to anti-oxidant production through changes to cystathionine-β-synthase (CBS) expression. Human cell lines with rescued LKB1 show increased CBS levels and resistance to carboplatin, which can be partially rescued by methionine restriction. Furthermore, LKB1 rescued cells, but not mutant cells, show less G2- M arrest and apoptosis in high methionine conditions. Knock-down of CBS sensitized both LKB1 mutant and non-mutated lines to carboplatin, again rescuing the carboplatin resistance of the LKB1 rescued lines. Given that immunotherapy is commonly combined with chemotherapy for NSCLC, we next wanted to understand if T cells are impaired by MR. Therefore, we examined the ability of T cells from MR and control tumor bearing mice to proliferate in culture and found that T cells from MR treated mice had no defects in proliferation, even though we continued the MR conditions ex vivo. We also identified that CBS is most highly correlated with smoking, adenocarcinomas with alveolar and bronchiolar features, and adenosquamous cell carcinomas, implicating its roles in oxidative stress response and lineage fate in human tumors. Taken together, we have shown the importance of MR as a dietary intervention to slow tumor growth and improve treatment outcomes for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0df2616d8ba063728db0011a0955e84576b33e17" target='_blank'>
              Methionine Restriction Reduces Lung Cancer Progression and Increases Chemotherapy Response
              </a>
            </td>
          <td>
            Kassandra J Naughton, Xiulong Song, Avery R Childress, Erika M Skaggs, Aria L. Byrd, Christian M. Gosser, Dave-Preston Esoe, Tanner J DuCote, Daniel Plaugher, Alexsandr Lukyanchuk, Ryan A. Goettl, Jinpeng Liu, C. F. Brainson
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e4d778cae3d7e3203dfad059efb7f4865240dc4" target='_blank'>
              Insights into treatment-specific prognostic somatic mutations in NSCLC from the AACR NSCLC GENIE BPC cohort analysis
              </a>
            </td>
          <td>
            Yi Liu, Sindhu Yalavarthi, Fan Yang, Yusif Abdul-Rashid, Shenkun Tang, Zihe Long, Yongkai Qin, Kerui Wu, Zhifei Wang
          </td>
          <td>2024-07-02</td>
          <td>BMC Pulmonary Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Menin inhibitors that disrupt Menin-MLL interaction hold promise for treating specific acute myeloid leukemia subtypes, including KMT2A rearrangements (KMT2A-r), yet resistance remains a challenge. Here, through systematic chromatin-focused CRISPR screens, along with genetic, epigenetic, and pharmacologic studies in a variety of human and mouse KMT2A-r AML models, we uncover a potential resistance mechanism independent of canonical Menin-MLL targets. We show that a group of non-canonical Menin targets, which are bivalently co-occupied by active Menin and repressive H2AK119ub marks, are typically downregulated following Menin inhibition. The loss of Polycomb Repressive Complex 1.1 (PRC1.1) subunits, such as PCGF1 or BCOR, leads to Menin inhibitor resistance by epigenetic reactivation of these non-canonical targets, including MYC. Genetic and pharmacological inhibition of MYC can resensitize PRC1.1-deficent leukemia cells to Menin inhibition. Moreover, we demonstrate that leukemia cells with the loss of PRC1.1 subunits exhibit reduced monocytic gene signatures and are susceptible to the BCL2 inhibition, and combinational treatment of venetoclax overcomes the resistance to Menin inhibition in PRC1.1-deficient leukemia cells. These findings highlight the important roles of PRC1.1 and its regulated non-canonical Menin targets in modulating Menin inhibitor response and provide potential strategies to treat leukemias with compromised PRC1.1 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fba94eb1d6d5cb6e0a8de6969fbdc894d28222" target='_blank'>
              Epigenetic Regulation of Non-canonical Menin Targets Modulates Menin Inhibitor Response in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Xinyue Zhou, Lixia Zhang, Sajesan Aryal, Virginia Veasey, Amanda Tajik, Cecilia Restelli, Steven Moreira, Pengcheng Zhang, Yanfeng Zhang, Kristin J Hope, Yang Zhou, Changde Cheng, Ravi Bhatia, Rui Lu
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40fcdfbbc2df4270401ef8beed0b78736d658d46" target='_blank'>
              DNA methylation in mammalian development and disease.
              </a>
            </td>
          <td>
            Zachary D. Smith, Sara Hetzel, Alexander Meissner
          </td>
          <td>2024-08-12</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The combination of CDK4/6 and MEK inhibition as a therapeutic strategy has shown promise in various cancer models, particularly in those harboring RAS mutations. An initial high-throughput drug screen identified a high synergy between the CDK4/6 inhibitor palbociclib and the MEK inhibitor trametinib when used in combination in soft tissue sarcomas. In RAS mutant models, combination treatment with palbociclib and trametinib induced significant G1 cell cycle arrest, resulting in a marked reduction in cell proliferation and growth. CRISPR-mediated RB1 depletion resulted in a decreased response to CDK4/6 and MEK inhibition, which was validated in both cell culture and xenograft models. Beyond its cell cycle inhibitory effects, pathway enrichment analysis revealed the robust activation of interferon pathways upon CDK4/6 and MEK inhibition. This induction of gene expression was associated with the upregulation of retroviral elements. The TBK1(TANK-binding kinase 1) inhibitor GSK8612 selectively blocked the induction of interferon-related genes induced by palbociclib and trametinib treatment, and highlighted the separable epigenetic responses elicited by combined CDK4/6 and MEK inhibition. Together, these findings provide key mechanistic insights into the therapeutic potential of CDK4/6 and MEK inhibition in soft tissue sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/721445995cd4f6ad7aae623c32a235fcb7645dbc" target='_blank'>
              Separable cell cycle arrest and immune response elicited through pharmacological CDK4/6 and MEK inhibition in RASmut disease models.
              </a>
            </td>
          <td>
            Jin Wu, Jianxin Wang, Thomas N O'Connor, Stephanie L. Tzetzo, Katerina V Gurova, Erik S. Knudsen, Agnieszka K Witkiewicz
          </td>
          <td>2024-08-16</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract This article provides an in-depth review of computational methods for predicting transcriptional regulators (TRs) with query gene sets. Identification of TRs is of utmost importance in many biological applications, including but not limited to elucidating biological development mechanisms, identifying key disease genes, and predicting therapeutic targets. Various computational methods based on next-generation sequencing (NGS) data have been developed in the past decade, yet no systematic evaluation of NGS-based methods has been offered. We classified these methods into two categories based on shared characteristics, namely library-based and region-based methods. We further conducted benchmark studies to evaluate the accuracy, sensitivity, coverage, and usability of NGS-based methods with molecular experimental datasets. Results show that BART, ChIP-Atlas, and Lisa have relatively better performance. Besides, we point out the limitations of NGS-based methods and explore potential directions for further improvement.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82904f760d288c0625f9f0cb781bd39bde2e6f1e" target='_blank'>
              Assessing next-generation sequencing-based computational methods for predicting transcriptional regulators with query gene sets
              </a>
            </td>
          <td>
            Zeyu Lu, Xue Xiao, Qiang Zheng, Xinlei Wang, Lin Xu
          </td>
          <td>2024-07-25</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Recent advances in ''omics'' technologies allow for the identification of an increasing number of individuals with diseases caused by nano-rare mutations. These difficult-to-diagnose individuals are uniquely disadvantaged and pose significant challenges to healthcare systems and society. Despite having diseases caused by actionable single gene mutations, in many cases, there is no commercial path for treatments for such small patient populations. Since antisense oligonucleotide (ASO) technology has proven to be suited to address the needs of a portion of these patients, the n-Lorem Foundation is establishing an industrialized approach that couples detailed genotypic and phenotypic data to the immediate potential for ASO therapy. In this manuscript we have leveraged our experience in assessing the causality of nano-rare genetic variants and associated proximal molecular pathological events to attempt a correlation between detailed genetic data with patient specific phenotypic observations in 173 nano-rare individuals from diverse age groups evaluated for experimental ASO therapy. We found that the time required to achieve a molecular diagnosis varies from 1 month to 36 years, with the mean and median times from symptom onset to diagnosis estimated to be 4.32 years and 2 years, respectively. Amongst submitted cases there is a significant bias toward neurological diseases, with diverse genes and functional families involved and a marked preponderance of mutations in ion channel genes. The variability in phenotypic expression associated with nano-rare variants in genes such as GNAO1, H3F3A, GBE1, UBTF, or PACS1 clearly supports previous observations that phenotypes associated with same variants in the same gene can vary. We also observe that different, but functionally equivalent variants can result in both similar (e.g., TARDBP) and different phenotypes (e.g., GNAO1). Despite the relatively small size of the patient population investigated, this first compilation of its kind allows a variety of insights into the genotype and phenotype relationships in nano-rare conditions. Moreover, we show that our unique patient population presents a remarkable opportunity to apply ''modern omics'' approaches to begin to understand the various homeostatic, compensatory, and secondary effects of these genetic variants on the networks that result in expression of their unique phenotypes. To provide a more detailed description of the processes involved to provide a personalized antisense medicine, we have included nonclinical and clincal data on four exemplary patients who display disease in three different organs, the CNS, the eye and the kidney and are treated with ASOs of different designs. In contrast to traditional drug development, each patient presents unique genomic, ASO design, clinical treatment and management and evaluation challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d6386d30baeff2ca96c279e928ec4153f9291f" target='_blank'>
              Genotypic and phenotypic analysis of 173 patients with extremely rare pathogenic mutations who applied for experimental antisense oligonucleotide treatment
              </a>
            </td>
          <td>
            S. Crooke, T. A. Cole, J. B. Carroll, J. G. Gleeson, L. Mignon, J. Douville, W. Chung, J. Bain, E. M. Berry-Kravis, N. Leung, S. Demarest, E. McCourt, A. Watt, B. Powers, C. Ngongang
          </td>
          <td>2024-08-06</td>
          <td>None</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary The atavistic theory of cancer suggests that cancer development proceeds through cells reverting to ancient survival mechanisms. The Serial Atavism Model (SAM) expands on this, proposing that cancer progresses through multiple stages of reversion to earlier evolutionary forms with cells losing modern traits and regaining primitive ones. One example is the Warburg effect, where cancer cells prefer a type of energy production used by ancient cells before Earth’s atmosphere had oxygen. However, this review argues that cancer metabolism is too complex to be fully explained by this theory. Cancer cells exhibit a wide range of metabolic behaviors that do not fit neatly into a pattern of reverting to an ancient state, indicating that the SAM may not provide a complete understanding of cancer. Abstract The atavistic theory of cancer posits that cancer emerges and progresses through the reversion of cellular phenotypes to more ancestral types with genomic and epigenetic changes deactivating recently evolved genetic modules and activating ancient survival mechanisms. This theory aims at explaining the known cancer hallmarks and the paradox of cancer’s predictable progression despite the randomness of genetic mutations. Lineweaver and colleagues recently proposed the Serial Atavism Model (SAM), an enhanced version of the atavistic theory, which suggests that cancer progression involves multiple atavistic reversions where cells regress through evolutionary stages, losing recently evolved traits first and reactivating primitive ones later. The Warburg effect, where cancer cells upregulate glycolysis and lactate production in the presence of oxygen instead of using oxidative phosphorylation, is one of the key feature of the SAM. It is associated with the metabolism of ancient cells living on Earth before the oxygenation of the atmosphere. This review addresses the question of whether cancer metabolism can be considered as an atavistic reversion. By analyzing several known characteristics of cancer metabolism, we reach the conclusion that this version of the atavistic theory does not provide an adequate conceptual frame for cancer research. Cancer metabolism spans a whole spectrum of metabolic states which cannot be fully explained by a sequential reversion to an ancient state. Moreover, we interrogate the nature of cancer metabolism and discuss its characteristics within the framework of the SAM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5f41e098ff498aa32258ffc44af055a9872bf7" target='_blank'>
              Is Cancer Metabolism an Atavism?
              </a>
            </td>
          <td>
            Eric Fanchon, A. Stéphanou
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The main epidemiological and clinical data on colorectal cancer, as well as the features of molecular pathology, are discussed in the literature review. Efforts are being putto identify promising targets, particularly small non-coding nucleotide sequences, which can lead to new treatments for this disease. The discovery of significant mutations that contribute to the development of colorectal tumors is a major step in the advancement of molecular oncology, as these mutations give rise to heterogeneous tumors that differ in their origin. These mutations play a significant role in the progression of the disease and are now being targeted for treatment. The prognosis for a disease is influenced by the patient's sensitivity to antitumor therapy. However, new approaches to finding effective targets for antitumor treatments face new fundamental challenges due to clinical issues. These issues include the epigenetic regulation of markers of oncogenesis, which allows for the development of new therapeutic strategies. RNA interference, in particular, has been linked to non-copying RNA sequences such as microRNAs. These microRNAs are associated with certain processes that can influence all aspects of oncogenesis. The diversity of microRNAs allows for a differentiated approach when treating tumors in various locations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4796e80e11216dc287a04bf6cbff831c3d542a28" target='_blank'>
              New Perspectives in Colorectal Cancers Treatment, the Role of MicroRNAs.
              </a>
            </td>
          <td>
            V. A. Belova, Liudmila V Spirina, A. Avgustinovich, S. G. Afanasiev, Maxim Yu Volkov M, D. I. Azovsky, Alexander M Volkov, T. S. Klyushina
          </td>
          <td>2024-07-24</td>
          <td>Current drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92d22ec19c45e84786f29e00379dcc92c97dbcbf" target='_blank'>
              Identifying key genes and functionally enriched pathways in acute myeloid leukemia by weighted gene co-expression network analysis.
              </a>
            </td>
          <td>
            Jimo Jian, Chenglu Yuan, Hongyuan Hao
          </td>
          <td>2024-07-09</td>
          <td>Journal of applied genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hutchinson-Gilford Progeria Syndrome (HGPS) is a rare genetic condition characterized by features of accelerated aging, and individuals with HGPS seldom live beyond their mid-teens. The syndrome is commonly caused by a point mutation in the LMNA gene which codes for lamin A and its splice variant lamin C, components of the nuclear lamina. The mutation causing HGPS leads to production of a truncated, farnesylated form of lamin A referred to as “progerin.” Progerin is also expressed at low levels in healthy individuals and appears to play a role in normal aging. HGPS is associated with an accumulation of genomic DNA double-strand breaks (DSBs) and alterations in the nature of DSB repair. The source of DSBs in HGPS is often attributed to stalling and subsequent collapse of replication forks in conjunction with faulty recruitment of repair factors to damage sites. In this work, we used a model system involving immortalized human cell lines to investigate progerin-induced genomic damage. Using an immunofluorescence approach to visualize phosphorylated histone H2AX foci which mark sites of genomic damage, we report that cells engineered to express progerin displayed a significant elevation of endogenous damage in the absence of any change in the cell cycle profile or doubling time of cells. Genomic damage was enhanced and persistent in progerin-expressing cells treated with hydroxyurea. Overexpression of wild-type lamin A did not elicit the outcomes associated with progerin expression. Our results show that DNA damage caused by progerin can occur independently from global changes in replication or cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4814100043718a6a0e30f1cd9c037307d90666" target='_blank'>
              Progerin Can Induce DNA Damage in the Absence of Global Changes in Replication or Cell Proliferation
              </a>
            </td>
          <td>
            Liza A. Joudeh, P. Logan Schuck, Nina M. Van, Alannah J. DiCintio, Jason A. Stewart, Alan S. Waldman
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="G-quadruplex (G4) sequences, which can fold into higher-order G4 structures, are abundant in the human genome and are over-represented in the promoter regions of many genes involved in human cancer initiation, progression, and metastasis. They are plausible targets for G4-binding small molecules, which would, in the case of promoter G4s, result in the transcriptional downregulation of these genes. However, structural information is currently available on only a very small number of G4s and their ligand complexes. This limitation, coupled with the currently restricted information on the G4-containing genes involved in most complex human cancers, has led to the development of a phenotypic-led approach to G4 ligand drug discovery. This approach was illustrated by the discovery of several generations of tri- and tetra-substituted naphthalene diimide (ND) ligands that were found to show potent growth inhibition in pancreatic cancer cell lines and are active in in vivo models for this hard-to-treat disease. The cycles of discovery have culminated in a highly potent tetra-substituted ND derivative, QN-302, which is currently being evaluated in a Phase 1 clinical trial. The major genes whose expression has been down-regulated by QN-302 are presented here: all contain G4 propensity and have been found to be up-regulated in human pancreatic cancer. Some of these genes are also upregulated in other human cancers, supporting the hypothesis that QN-302 is a pan-G4 drug of potential utility beyond pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f12d7617e98441c66bfb53bc671f9147eed0b1fe" target='_blank'>
              A Phenotypic Approach to the Discovery of Potent G-Quadruplex Targeted Drugs
              </a>
            </td>
          <td>
            Stephen Neidle
          </td>
          <td>2024-08-01</td>
          <td>Molecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract The comprehensive genomic analysis of the head and neck squamous cell carcinoma (HNSCC) oncogenome revealed the frequent loss of p16INK4A (CDKN2A) and amplification of cyclin D1 genes in most human papillomavirus–negative HNSCC lesions. However, cyclin-dependent kinase 4 and 6 (CDK4/6) inhibitors have shown modest effects in the clinic. The aberrant activation of the PI3K/mTOR pathway is highly prevalent in HNSCC, and recent clinical trials have shown promising clinical efficacy of mTOR inhibitors (mTORi) in the neoadjuvant and adjuvant settings but not in patients with advanced HNSCC. By implementing a kinome-wide CRISPR/Cas9 screen, we identified cell-cycle inhibition as a synthetic lethal target for mTORis. A combination of mTORi and palbociclib, a CDK4/6-specific inhibitor, showed strong synergism in HNSCC-derived cells in vitro and in vivo. Remarkably, we found that an adaptive increase in cyclin E1 (CCNE1) expression upon palbociclib treatment underlies the rapid acquired resistance to this CDK4/6 inhibitor. Mechanistically, mTORi inhibits the formation of eIF4G–CCNE1 mRNA complexes, with the consequent reduction in mRNA translation and CCNE1 protein expression. Our findings suggest that mTORi reverts the adaptive resistance to palbociclib. This provides a multimodal therapeutic option for HNSCC by cotargeting mTOR and CDK4/6, which in turn may halt the emergence of palbociclib resistance. Significance: A kinome-wide CRISPR/Cas9 screen identified cell-cycle inhibition as a synthetic lethal target of mTORis. A combination of mTORi and palbociclib, a CDK4/6-specific inhibitor, showed strong synergistic effects in HNSCC. Mechanistically, mTORis inhibited palbociclib-induced increase in CCNE1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d8015b7ff294926111ccbf929e88b6d333407e" target='_blank'>
              A Kinome-Wide Synthetic Lethal CRISPR/Cas9 Screen Reveals That mTOR Inhibition Prevents Adaptive Resistance to CDK4/CDK6 Blockade in HNSCC
              </a>
            </td>
          <td>
            Yusuke Goto, Keiichi Koshizuka, Toshinori Ando, Hiroki Izumi, Xingyu Wu, Kuniaki Sato, Tomohiko Ishikawa, Kyle Ford, Xiaodong Feng, Zhiyong Wang, Nadia Arang, M. Allevato, Ayush Kishore, Prashant Mali, J. Gutkind
          </td>
          <td>2024-07-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5096f65eca5658128881222e362e03cc7a2beb2" target='_blank'>
              H3K27ac acts as a molecular switch for doxorubicin-induced activation of cardiotoxic genes
              </a>
            </td>
          <td>
            Yu Hong, Xinlan Li, Jia Li, Qiuyi He, Manbing Huang, Yubo Tang, Xiao Chen, Jie Chen, Ke-Jing Tang, Chao Wei
          </td>
          <td>2024-07-16</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary This study explores the detection of gene fusions in acute leukemia using targeted RNA sequencing and compares its effectiveness with traditional methods like karyotyping and RT-PCR. Gene fusions, which result from chromosomal rearrangements, are crucial for diagnosing and treating leukemia. Recent updates in diagnostic criteria emphasize the importance of identifying these gene fusions. Targeted RNA sequencing offers advantages such as rapid diagnosis, minimal training requirements, and cost-effectiveness, making it a practical method in clinical settings. Our findings suggest that targeted RNA sequencing can accurately identify gene fusions, potentially benefiting nearly half of leukemia patients and improving diagnostic precision. This method may significantly impact the research community by providing a reliable and efficient tool for leukemia diagnosis. Abstract Gene fusions are key drivers in acute leukemia, impacting diagnosis and treatment decisions. We analyzed 264 leukemia patients using targeted RNA sequencing with conventional karyotyping and reverse transcription polymerase chain reaction (RT-PCR). Leukemic fusions were detected in 127 patients (48.1%). The new guidelines introduced additional diagnostic criteria, expanding the spectrum of gene fusions. We discovered three novel fusions (RUNX1::DOPEY2, RUNX1::MACROD2, and ZCCHC7::LRP1B). We analyzed recurrent breakpoints for the KMT2A and NUP98 rearrangements. Targeted RNA sequencing showed consistent results with RT-PCR in all tested samples. However, when compared to conventional karyotyping, we observed an 83.3% concordance rate, with 29 cases found only in targeted RNA sequencing, 7 cases with discordant results, and 5 cases found only in conventional karyotyping. For the five cases where known leukemic gene rearrangements were suspected only in conventional karyotyping, we conducted additional messenger RNA sequencing in four cases and proved no pathogenic gene rearrangements. Targeted RNA sequencing proved advantageous for the rapid and accurate interpretation of gene rearrangements. The concurrent use of multiple methods was essential for a comprehensive evaluation. Comprehensive molecular analysis enhances our understanding of leukemia’s genetic basis, aiding diagnosis and classification. Advanced molecular techniques improve clinical decision-making, offering potential benefits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f0977b67570006625235107c94ecedeb3ed6b3" target='_blank'>
              Real-World Clinical Utility of Targeted RNA Sequencing in Leukemia Diagnosis and Management
              </a>
            </td>
          <td>
            Seo Wan Kim, N. Kim, Yu Jeong Choi, Seung-Tae Lee, Jong Rak Choi, Saeam Shin
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b903232278019fc20d595adb95c47c4a0e3a7532" target='_blank'>
              Deletion of IRC19 Causes Defects in DNA Double-Strand Break Repair Pathways in Saccharomyces cerevisiae.
              </a>
            </td>
          <td>
            Ju-Hee Choi, Oyungoo Bayarmagnai, Sung-Ho Bae
          </td>
          <td>2024-07-12</td>
          <td>Journal of microbiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="OBJECTIVE
Ionizing radiation generates genomic instability by promoting the accumulation of chromosomal rearrangements. The oncogenic translocation RET/PTC1 is present in more than 70% of radiation-induced thyroid cancers. Both RET and CCDC6, the genes implicated in RET/PTC1, are found within common fragile sites - chromosomal regions prone to DNA breakage during slight replication stress. Given that irradiated cells become more susceptible to genomic destabilization due to the accumulation of replication-stress-related double-strand breaks (DSBs), we explored whether RET and CCDC6 exhibit DNA breakage under replicative stress several days post-irradiation of thyroid cells.


METHODS
We analyzed the dynamic of DNA replication in human thyroid epithelial cells (HThy-ori-3.1) 4 days post a 5-Gy exposure using molecular DNA combing. The DNA replication schedule was evaluated through replication-timing experiments. We implemented a ChIP-qPCR assay to determine whether the RET and CCDC6 genes break following irradiation.


RESULTS
Our study indicates that replicative stress, occurring several days post-irradiation in thyroid cells, primarily causes DSBs in the RET gene. We discovered that both the RET and CCDC6 genes undergo late replication in thyroid cells. However, only RET's replication rate is notably delayed after irradiation.


CONCLUSION
The findings suggest that post-irradiation in the RET gene causes a breakage in the replication fork, which could potentially invade another genomic area, including CCDC6. As a result, this could greatly contribute to the high prevalence of chromosomal RET/PTC rearrangements seen in patients exposed to external radiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eea49490c4342b6b738c402ce79c72ea23e5f552" target='_blank'>
              A post-irradiation-induced replication stress promotes RET proto-oncogene breakage.
              </a>
            </td>
          <td>
            Fabio Hecht, Laura Valerio, Carlos Frederico Lima Gonçalves, Marylin Harinquet, Rabii Ameziane El Hassani, Denise P. Carvalho, Stephane Koundrioukoff, Jean-Charles Cadoret, Corinne Dupuy
          </td>
          <td>2024-07-01</td>
          <td>European thyroid journal</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Introduction: To determine the appropriate treatment for patients with advanced/recurrent nonsquamous non‒small-cell lung cancer (NSCLC), a companion diagnostic was conducted to detect driver mutations through genetic testing. In Japan, Oncomine Dx Target Test (DxTT) using next-generation sequencing (NGS) that can comprehensively detect gene mutations or single-gene tests are conducted as companion diagnostics. Furthermore, cost-effectiveness analysis was conducted to compare the cost-effectiveness of Oncomine DxTT using NGS with that of single-gene test in Japan. Methods: The target population included patients with advanced/recurrent nonsquamous NSCLC. A model structure was constructed for the Oncomine DxTT strategy and three single-gene tests (i.e., epidermal growth factor receptor (EGFR) mutations and anaplastic lymphoma kinase (ALK)/c-ros oncogene 1 (ROS1) rearrangements) with reference to previous studies and the Clinical Practice Guidelines of Lung Cancer 2022 in Japan. The model structure assumed that genetic testing would be conducted and first-line treatment used the drug most recommended in the 2022 Japanese Lung Cancer Clinical Practice Guidelines, depending on the driver mutation,. Model inputs were obtained from the literature and price list in Japan, and cost-utility analysis was conducted. Results: For the Oncomine DxTT strategy, the expected incremental costs and effectiveness were estimated to be approximately JPY 172,361 (JPY 12,285,228 vs. JPY 12,112,867 for strategies A and B, respectively) and −0.51 quality-adjusted life-year (QALY) per patient (21.93 QALY vs. 22.44 QALY for strategies A and B). As a result, the costs increased but the effectiveness decreased. Therefore, the Oncomine DxTT strategy was dominated by the three single-gene tests. Sensitivity and scenario analyses revealed that the test success rate of Oncomine DxTT affected the results. Conclusions: The genetic test using Oncomine DxTT before the first-line treatment is not cost-effective compared with the three single-gene tests (EGFR/ALK/ROS1) for patients with advanced/recurrent nonsquamous NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b412c1d1df4b8b78941591a11bb880b51a38410" target='_blank'>
              Cost-effectiveness Analysis of the Oncomine™ Dx Target Test MultiCDx System Using Next-generation Sequencing and Single-gene Test in Advanced and Recurrent Nonsquamous Non-small-cell Lung Cancer
              </a>
            </td>
          <td>
            Tomomi Kohara, Shunya Ikeda, Koichi Benjamin Ishikawa
          </td>
          <td>2024-07-10</td>
          <td>JMA Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Despite the availability of life-extending treatments for B cell leukemias and lymphomas, many of these cancers remain incurable. Thus, the development of new molecular targets and therapeutics is needed to expand treatment options. To identify new molecular targets, we used a forward genetic screen in mice to identify genes required for development or survival of lymphocytes. Here, we describe Zfp574, an essential gene encoding a zinc finger protein necessary for normal and malignant lymphocyte survival. We show that ZFP574 interacts with zinc finger protein THAP12 and promotes the G1-to-S-phase transition during cell cycle progression. Mutation of ZFP574 impairs nuclear localization of the ZFP574-THAP12 complex. ZFP574 or THAP12 deficiency results in cell cycle arrest and impaired lymphoproliferation. Germline mutation, acute gene deletion, or targeted degradation of ZFP574 suppressed Myc-driven B cell leukemia in mice, but normal B cells were largely spared, permitting long-term survival, whereas complete lethality was observed in control animals. Our findings support the identification of drugs targeting ZFP574-THAP12 as a unique strategy to treat B cell malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0480486c2217b6dc5e3cddbe10c5e1e4dd9761e0" target='_blank'>
              Disruption of the ZFP574-THAP12 complex suppresses B cell malignancies in mice.
              </a>
            </td>
          <td>
            Xue Zhong, James J Moresco, J. SoRelle, Ran Song, Yiao Jiang, Mylinh T Nguyen, Jianhui Wang, C. Bu, E. Moresco, Bruce Beutler, J. Choi
          </td>
          <td>2024-07-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="BACKGROUND
The identification of the most appropriate targeted therapies for advanced cancers is challenging. We performed a molecular profiling of metastatic solid tumors utilizing a comprehensive next-generation sequencing (NGS) assay to determine genomic alterations' type, frequency, actionability, and potential correlations with PD-L1 expression.


METHODS
A total of 304 adult patients with heavily pretreated metastatic cancers treated between January 2019 and March 2021 were recruited. The CLIA-/UKAS-accredit Oncofocus assay targeting 505 genes was used on newly obtained or archived biopsies. Chi-square, Kruskal-Wallis, and Wilcoxon rank-sum tests were used where appropriate. Results were significant for P < .05.


RESULTS
A total of 237 tumors (78%) harbored potentially actionable genomic alterations. Tumors were positive for PD-L1 in 68.9% of cases. The median number of mutant genes/tumor was 2.0 (IQR: 1.0-3.0). Only 34.5% were actionable ESCAT Tier I-II with different prevalence according to cancer type. The DNA damage repair (14%), the PI3K/AKT/mTOR (14%), and the RAS/RAF/MAPK (12%) pathways were the most frequently altered. No association was found among PD-L1, ESCAT, age, sex, and tumor mutational status. Overall, 62 patients underwent targeted treatment, with 37.1% obtaining objective responses. The same molecular-driven treatment for different cancer types could be associated with opposite clinical outcomes.


CONCLUSIONS
We highlight the clinical value of molecular profiling in metastatic solid tumors using comprehensive NGS-based panels to improve treatment algorithms in situations of uncertainty and facilitate clinical trial recruitment. However, interpreting genomic alterations in a tumor type-specific manner is critical.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b13a3de4d8a4a44c9264bbeddda708851f093875" target='_blank'>
              Next-generation sequencing-based evaluation of the actionable landscape of genomic alterations in solid tumors: the "MOZART" prospective observational study.
              </a>
            </td>
          <td>
            Francesco Schettini, Marianna Sirico, M. Loddo, Gareth H Williams, Keeda-Marie Hardisty, Paul Scorer, R. Thatcher, Pablo Rivera, Manuela Milani, C. Strina, G. Ferrero, Marco Ungari, C. Bottin, Fabrizio Zanconati, Nicolò de Manzini, S. Aguggini, Richard Tancredi, E. Fiorio, Antonio Fioravanti, Maurizio Scaltriti, Daniele Generali
          </td>
          <td>2024-08-23</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Background/Aim: BRCA1/2 mutations in breast cancer cells impair homologous recombination and promote alternative end joining (Alt-EJ) for DNA-damage repair. DNA polymerase theta, encoded by POLQ, plays a crucial role in Alt-EJ, making it a potential therapeutic target, particularly in BRCA1/2-mutant cancers. Methionine restriction is a promising approach to target cancer cells due to their addiction to this amino acid. The present study investigated the expression of POLQ in BRCA1/2 wild-type and BRCA1-mutant breast cancer cells under methionine restriction. Materials and Methods: POLQ mRNA expression was measured using qRT-PCR in BRCA1/2 wild-type (MDA-MB-231) and BRCA1- mutant (HCC1937 and MDA-MB-436) breast-cancer cells under normal, or serum-restricted, or serum- and methionine-restricted conditions. Results: Compared to BRCA1/2 wild-type cells, BRCA1-mutant cells displayed significantly higher basal POLQ expression in normal medium. Methionine restriction further increased POLQ expression in the BRCA1-mutant cells but decreased it in the BRCA1/2 wild-type cells. Conclusion: The present findings suggest that methionine restriction showed differential effects on POLQ expression, potentially impacting Alt-EJ activity, in BRCA1/2 wild-type and BRCA1-mutant breast-cancer cells. Further investigation is needed to explore the potential of combining methionine restriction with DNA-repair inhibitors, such as PARP inhibitors, to overcome drug resistance in BRCA1/2 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0523a0d152aaab9d94d339ede11927c41a78c4b" target='_blank'>
              Induction of the DNA-Repair Gene POLQ only in BRCA1-mutant Breast-Cancer Cells by Methionine Restriction
              </a>
            </td>
          <td>
            T. Kunihisa, Sachiko Inubushi, Hirokazu Tanino, Robert M. Hoffman
          </td>
          <td>2024-06-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Human telomerase reverse transcriptase (hTERT) may have noncanonical functions in transcriptional regulation and metabolic reprogramming in cancer cells, but it is a challenging target. We thus developed small-molecule ligands targeting hTERT promoter G-quadruplex DNA structures (hTERT G4) to downregulate hTERT expression. Ligand 5 showed high affinity toward hTERT G4 (Kd = 1.1 μM) and potent activity against triple-negative breast cancer cells (MDA-MB-231, IC50 = 1 μM). In cell-based assays, 5 not only exerts markedly inhibitory activity on classical telomere functions including decreased telomerase activity, shortened telomere length, and cellular senescence but also induces DNA damage, acute cellular senescence, and apoptosis. This study reveals that hTERT G4-targeting ligand may cause mitochondrial dysfunction, disrupt iron metabolism and activate ferroptosis in cancer cells. The in vivo antitumor efficacy of 5 was also evaluated in an MDA-MB-231 xenograft mouse model and approximately 78.7% tumor weight reduction was achieved. No observable toxicity against the major organs was observed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1d3d47d8b1822b8c2901a320f1107b7691744c1" target='_blank'>
              Targeting hTERT Promoter G-Quadruplex DNA Structures with Small-Molecule Ligand to Downregulate hTERT Expression for Triple-Negative Breast Cancer Therapy.
              </a>
            </td>
          <td>
            W. Long, Yao-Xun Zeng, Bo-Xin Zheng, Yu-Bo Li, Ya-Kun Wang, K. Chan, Meng-Ting She, Yu-Jing Lu, Chunyang Cao, Wing-Leung Wong
          </td>
          <td>2024-07-10</td>
          <td>Journal of medicinal chemistry</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fcc48036440e068784b3b542811aace316e8c88" target='_blank'>
              Clinical application of whole-genome sequencing of solid tumors for precision oncology.
              </a>
            </td>
          <td>
            R. Kim, Seokhwi Kim, B. B. Oh, Woo Sik Yu, Chang Woo Kim, Hoon Hur, Sang-Yong Son, Min Jae Yang, Dae Sung Cho, Taeyang Ha, Subin Heo, Jeon Yeob Jang, Jae Sung Yun, Kyu-sung Kwack, Jai Keun Kim, Jimi Huh, S. Lim, Sang-Uk Han, Hyun Woo Lee, Ji Eun Park, Chul-Ho Kim, J. Roh, Young Wha Koh, Dakeun Lee, Jang-Hee Kim, G. Lee, Choong-Kyun Noh, Y. Jung, Ji Won Park, Seungsoo Sheen, M. S. Ahn, Yong Won Choi, Tae-Hwan Kim, Seok Yun Kang, Jin-Hyuk Choi, Soo Yeon Baek, K. Lee, Sun Il Kim, Sung Hyun Noh, Se-Hyuk Kim, Hyemin Hwang, Eunjung Joo, Shinjung Lee, Jong-Yeon Shin, Ji-Young Yun, Jung-Woon Park, Kijong Yi, Youngoh Kwon, W. Lee, H. Park, Joonoh Lim, Boram Yi, Jaemo Koo, June-Young Koh, Sangmoon Lee, Yuna Lee, Bo-Rahm Lee, E. Connolly-Strong, Y. S. Ju, Minsuk Kwon
          </td>
          <td>2024-08-13</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Osteosarcoma is a common malignancy that often occurs in children, teenagers and young adults. Although the treatment strategy has improved, the results are still poor for most patients with metastatic or recurrent osteosarcomas. Therefore, it is necessary to identify new and effective prognostic biomarkers and therapeutic targets for diseases. Human genomes contain lncRNAs, transcripts with limited or insufficient capacity to encode proteins. They have been implicated in tumorigenesis, particularly regarding the onset, advancement, resistance to treatment, recurrence and remote dissemination of malignancies. Aberrant lncRNA expression in osteosarcomas has been reported by numerous researchers; lncRNAs have the potential to exhibit either oncogenic or tumor-suppressing behaviors and thus, to govern the advancement of this skeletal cancer. They are suspected to influence osteosarcoma cell growth, replication, invasion, migration, remote dissemination and programmed cell death. Additionally, they have been recognized as clinical markers, and may participate in the development of multidrug resistance. Therefore, the study of lncRNAs in the growth, metastasis, treatment and prognosis of osteosarcoma is very important for the active prevention and treatment of osteosarcoma. Consequently, this work reviews the functions of lncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fb1ad397401fa91f3d7611746eaf23c6fb5fede" target='_blank'>
              LncRNAs as potential prognosis/diagnosis markers and factors driving drug resistance of osteosarcoma, a review
              </a>
            </td>
          <td>
            Siwang Hu, Xuebing Han, Gang Liu, Shuangshuang Wang
          </td>
          <td>2024-07-02</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b7ae78768e89d2af68c787eeb3b2e8190ce9d8" target='_blank'>
              Single-cell resolution characterization of myeloid-derived cell states with implication in cancer outcome
              </a>
            </td>
          <td>
            Gabriela Rapozo Guimarães, Giovanna Resk Maklouf, Cristiane Esteves Teixeira, Leandro de Oliveira Santos, N. Tessarollo, Nayara Evelin de Toledo, A. F. Serain, C. A. de Lanna, Marco Antônio M. Pretti, Jéssica Gonçalves Vieira da Cruz, M. Falchetti, Mylla M Dimas, I. Filgueiras, Otávio Cabral-Marques, Rodrigo Nalio Ramos, F. C. de Macedo, F. Rodrigues, Nina Carrossini Bastos, J. L. da Silva, Edroaldo Lummertz da Rocha, Cláudia Bessa Pereira Chaves, A. C. de Melo, Pedro M M Moraes-Vieira, Marcelo A Mori, Marianna Boroni
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metabolic reprogramming is recognized as a hallmark of cancer, enabling cancer cells to acquire essential biomolecules for cell growth, often characterized by upregulated glycolysis and/or fatty acid synthesis-related genes. The transcription factor forkhead box M1 (FOXM1) has been implicated in various cancers, contributing significantly to their development, including colorectal cancer (CRC), a major global health concern. Despite FOXM1's established role in cancer, its specific involvement in the Warburg effect and fatty acid biosynthesis in CRC remains unclear. We analyzed The Cancer Genome Atlas (TCGA) Colonic Adenocarcinoma and Rectal Adenocarcinoma (COADREAD) datasets to to derive the correlation of the expression levels between FOXM1 and multiple genes and the survival prognosis based on FOXM1 expression. Using two human CRC cell lines, HT29 and HCT116, we conducted RNAi or plasmid transfection procedures, followed by a series of assays, including RNA extraction, quantitative real-time polymerase chain reaction, Western blot analysis, cell metabolic assays, and immunofluorescence analysis. Higher expression levels of FOXM1 correlated with a poorer survival prognosis, and the expression of FOXM1 was positively correlated with glycolysis-related genes SLC2A1 and LDHA, de novo lipogenesis-related genes ACACA and FASN, and MYC. FOXM1 appeared to modulate AKT/mTOR signaling, the expression of c-Myc, proteins related to glycolysis and fatty acid biosynthesis, as well as extracellular acidification rate in HT29 and HCT116 cells. In summary, FOXM1 plays a regulatory role in glycolysis, fatty acid biosynthesis, and cellular energy consumption, thereby influencing CRC cell growth and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e3baf49a68626e2bd59e387a99b97ce66e27bb" target='_blank'>
              Forkhead box M1 mediates metabolic reprogramming in human colorectal cancer cells.
              </a>
            </td>
          <td>
            Po-Chen Li, Sheng-Yu Dai, Yu-Shun Lin, Yu-Tsen Chang, Chen-Chia Liu, I-Ching Wang, Ming-Fen Lee
          </td>
          <td>2024-07-02</td>
          <td>American journal of physiology. Gastrointestinal and liver physiology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background This study comprehensively investigates the association between the expression of nicotinamide N-methyltransferase (NNMT) and clinical outcomes of urothelial bladder cancer (UBC), as well as the molecular mechanisms by which NNMT in cancer-associated fibroblast (CAF) modulates tumor progression and immunotherapy resistance in UBC. Methods Single-cell transcriptomic analyses, immunohistochemical and immunofluorescence assays were performed on bladder cancer samples to validate the relationship between NNMT expression and clinical outcomes. A series of experiments, including chromatin immunoprecipitation assay, liquid chromatography tandem mass spectrometry assay, and CRISPR‒Cas9 (Clustered Regularly Interspaced Short Palindromic Repeats and CRISPR-associated protein 9) knockout, together with in vivo models, have been established to determine the molecular functions of NNMT in CAFs in UBC. Results We demonstrated that elevated expression of the nicotinamide adenine dinucleotide (NAD+) metabolism enzyme NNMT in CAFs (NNMT+ CAFs) was significantly associated with non-response to programmed death-ligand 1 (PD-L1) blockade immunotherapy in patients with UBC and predicted the unfavorable prognosis of UBC in two independent large cohorts. Targeting NNMT using the inhibitor 5-Amino-1-methylquinolinium iodide significantly reduced tumor growth and enhanced the apoptotic effects of the anti-PD-L1 antibody in UBC mouse models. Mechanistically, NNMT+ CAFs recruit tumor-associated macrophages via epigenetic reprogramming of serum amyloid A (SAA) to drive tumor cell proliferation and confer resistance to programmed death-1/PD-L1 blockade immunotherapy. Conclusions NNMT+ CAFs were significantly associated with non-response to PD-L1 blockade immunotherapy in patients with UBC. Elevated NNMT, specifically in CAFs, upregulates SAA expression and enhances the recruitment and differentiation of macrophages in the tumor microenvironment, thereby directly or indirectly promoting tumor progression and conferring resistance to immunotherapies in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff788af3187bc82c0f0f61490e783d098bccd55c" target='_blank'>
              NAD+ metabolism enzyme NNMT in cancer-associated fibroblasts drives tumor progression and resistance to immunotherapy by modulating macrophages in urothelial bladder cancer
              </a>
            </td>
          <td>
            Meihua Yang, Bo Wang, Weibin Hou, Honghui Zeng, Wang He, Xin-Ke Zhang, D. Yan, Hao Yu, Long Huang, Lu Pei, Kaiwen Li, Haide Qin, Tianxin Lin, Jian Huang
          </td>
          <td>2024-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Persister cells (PS) selected for anticancer therapy have been recognized as a significant contributor to the development of treatment-resistant malignancies. It is found that imposing glutamine restriction induces the generation of PS, which paradoxically bestows heightened resistance to glutamine restriction treatment by activating the integrated stress response and initiating the general control nonderepressible 2-activating transcription factor 4-alanine, serine, cysteine-preferring transporter 2 (GCN2-ATF4-ASCT2) axis. Central to this phenomenon is the stress-induced ATF4 translational reprogramming. Unfortunately, directly targeting ATF4 protein has proven to be a formidable challenge because of its flat surface. Nonetheless, a G-quadruplex structure located within the promoter region of ATF4 (ATF4-G4) is uncovered and resolved, which functions as a transcriptional regulator and can be targeted by small molecules. The investigation identifies the natural compound coptisine (COP) as a potent binder that interacts with and stabilizes ATF4-G4. For the first time, the high-resolution structure of the COP-ATF4-G4 complex is determined. The formation of this stable complex disrupts the interaction between transcription factor AP-2 alpha (TFAP2A) and ATF4-G4, resulting in a substantial reduction in intracellular ATF4 levels and the eventual death of cancer cells. These seminal findings underscore the potential of targeting the ATF4-G4 structure to yield significant therapeutic advantages within the realm of persister cancer cells induced by glutamine-restricted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e9a3aeeaaba15f0673e425713df633d5da2043" target='_blank'>
              Overcoming Cancer Persister Cells by Stabilizing the ATF4 Promoter G-quadruplex.
              </a>
            </td>
          <td>
            Chengmei Xiao, Yipu Li, Yushuang Liu, Ruifang Dong, Xiaoyu He, Qing Lin, Xin Zang, Kaibo Wang, Yuanzheng Xia, Lingyi Kong
          </td>
          <td>2024-07-12</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd17af76ab151f83c8653ba7afa98d9d3324c71" target='_blank'>
              Simultaneous multi-site editing of individual genomes using retron arrays.
              </a>
            </td>
          <td>
            A. González-Delgado, Santiago C. Lopez, Matías Rojas-Montero, C. Fishman, Seth L. Shipman
          </td>
          <td>2024-07-09</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, Bert van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3e129666ae56c033aa207d599e42711f40eae8" target='_blank'>
              Gene body DNA hydroxymethylation restricts the magnitude of transcriptional changes during aging
              </a>
            </td>
          <td>
            James R. Occean, Na Yang, Yan Sun, Marshall S. Dawkins, R. Munk, Cedric Belair, Showkat A. Dar, C. Anerillas, Lin Wang, Changyou Shi, Christopher Dunn, Michel Bernier, Nathan L. Price, Julie S Kim, Chang-Yi Cui, Jinshui Fan, Moitrayee Bhattacharyya, Supriyo De, Manolis Maragkakis, R. de Cabo, Simone Sidoli, P. Sen
          </td>
          <td>2024-07-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="A significant number of the genetic alterations observed in cancer patients lie within nonprotein-coding segments of the genome, including regions coding for long noncoding RNAs (lncRNAs). LncRNAs display aberrant expression in breast cancer (BrCa), but the functional implications of this altered expression remain to be elucidated. By performing transcriptome screen in a triple negative BrCa (TNBC) isogenic 2D and 3D spheroid model, we observed aberrant expression of >1000 lncRNAs during BrCa progression. The chromatin-associated lncRNA MANCR shows elevated expression in metastatic TNBC. MANCR is upregulated in response to cellular stress and modulates DNA repair and cell proliferation. MANCR promotes metastasis as MANCR-depleted cells show reduced cell migration, invasion, and wound healing in vitro, and reduced metastatic lung colonization in xenograft experiments in vivo. Transcriptome analyses reveal that MANCR modulates expression and pre-mRNA splicing of genes, controlling DNA repair and checkpoint response. MANCR promotes the transcription of NET1A, a Rho-GEF that regulates DNA damage checkpoint and metastatic processes in cis, by differential promoter usage. Experiments suggest that MANCR regulates the expression of cancer-associated genes by modulating the association of various transcription factors and RNA-binding proteins. Our results identified the metastasis-promoting activities of MANCR in TNBC by cis-regulation of gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917a434f9bc31bbdd4efde82c1a4158058876264" target='_blank'>
              MANCR lncRNA Modulates Cell-Cycle Progression and Metastasis by Cis-Regulation of Nuclear Rho-GEF.
              </a>
            </td>
          <td>
            D. K. Singh, Zhengmin Cong, Y. Song, Minxue Liu, Ritu Chaudhary, Dazhen Liu, Yu Wang, Rishabh Prasanth, Rajendra K C, Simon Lizarazo, Miriam Akhnoukh, Omid Gholamalamdari, Anurupa Moitra, Lisa M. Jenkins, Rohit Bhargava, Erik R Nelson, Kevin Van Bortle, S. Prasanth, K. Prasanth
          </td>
          <td>2024-08-12</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="DNA methylation is an important epigenetic regulatory mechanism which plays a crucial role in cell differentiation and development. Its function is closely related to DNA methyltransferase 3 alpha (DNMT3A), which can affect gene expression and stem cell differentiation. The mutation rate of the DNMT3A gene is relatively high in Acute myeloid leukemia (AML), but its type and pathogenic mechanism are not yet clear. Further research on DNMT3A may help to identify its pathogenic targets and provide a basis for precise treatment of AML. This article has provided a review for the research progress on the expression of the DNMT3A gene in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ec69c604a676713e1a7e86038752ff556999862" target='_blank'>
              [Research progress on DNMT3A gene expression in Acute myeloid leukemia].
              </a>
            </td>
          <td>
            Jiawei Zhou, Tao Wu, Wenhui Liu
          </td>
          <td>2024-08-10</td>
          <td>Zhonghua yi xue yi chuan xue za zhi = Zhonghua yixue yichuanxue zazhi = Chinese journal of medical genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Clustered regularly interspersed short palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) technology has revolutionized the field of gene therapy as it has enabled precise genome editing with unprecedented accuracy and efficiency, paving the way for clinical applications to treat otherwise incurable genetic disorders. Typically, precise genome editing requires the delivery of multiple components to the target cells that, depending on the editing platform used, may include messenger RNA (mRNA), protein complexes, and DNA fragments. For clinical purposes, these have to be efficiently delivered into transplantable cells, such as primary T lymphocytes or hematopoietic stem and progenitor cells that are typically sensitive to exogenous substances. This challenge has limited the broad applicability of precise gene therapy applications to those strategies for which efficient delivery methods are available. Electroporation-based methodologies have been generally applied for gene editing applications, but procedure-associated toxicity has represented a major burden. With the advent of novel and less disruptive methodologies to deliver genetic cargo to transplantable cells, it is now possible to safely and efficiently deliver multiple components for precise genome editing, thus expanding the applicability of these strategies. In this review, we describe the different delivery systems available for genome editing components, including viral and non-viral systems, highlighting their advantages, limitations, and recent clinical applications. Recent improvements to these delivery methods to achieve cell specificity represent a critical development that may enable in vivo targeting in the future and will certainly play a pivotal role in the gene therapy field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff8c03481dd2684d89eea9dc5f829d3697ccc125" target='_blank'>
              Viral and Non-Viral Systems to Deliver Gene Therapeutics to Clinical Targets
              </a>
            </td>
          <td>
            Maryam Taghdiri, C. Mussolino
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cycling cells replicate their DNA during the S phase through a defined temporal program known as replication timing. Mutation frequencies, epigenetic chromatin states, and transcriptional activities are different for genomic regions that are replicated early and late in the S phase. Here, we found from ChIP-Seq analysis that DNA polymerase (Pol) κ is enriched in early-replicating genomic regions in HEK293T cells. In addition, by feeding cells with N 2-heptynyl-2′-deoxyguanosine followed by click chemistry–based enrichment and high-throughput sequencing, we observed elevated Pol κ activities in genomic regions that are replicated early in the S phase. On the basis of the established functions of Pol κ in accurate and efficient nucleotide insertion opposite endogenously induced N 2-modified dG lesions, our work suggests that active engagement of Pol κ may contribute to diminished mutation rates observed in early-replicating regions of the human genome, including cancer genomes. Together, our work expands the functions of Pol κ and offered a plausible mechanism underlying replication timing–dependent mutation accrual in the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417d907a79911389824959d3e561a3c8846017ac" target='_blank'>
              DNA polymerase κ participates in early S-phase DNA replication in human cells
              </a>
            </td>
          <td>
            Feng Tang, Yinan Wang, Ting Zhao, Jun Yuan, Andrew H. Kellum, Yinsheng Wang
          </td>
          <td>2024-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Lentiviral vector-transduced T-cells were approved by the FDA as gene therapy anti-cancer medications. Little is known about the host genetic variation effects on the safety and efficacy of the lentiviral vector gene delivery system. To narrow this knowledge-gap, we characterized hepatic gene delivery by lentiviral vectors across the Collaborative Cross (CC) mouse genetic reference population. For 24 weeks, we periodically measured hepatic luciferase expression from lentiviral vectors in 41 CC mouse strains. Hepatic and splenic vector copy numbers were determined. We report that CC mouse strains showed highly diverse outcomes following lentiviral gene delivery. For the first time, moderate correlation between mouse strain-specific sleeping patterns and transduction efficiency was observed. We associated two quantitative trait loci (QTLs) with intra-strain variations in transduction phenotypes, which mechanistically relates to the phenomenon of metastable epialleles. An additional QTL was associated with the kinetics of hepatic transgene expression. Genes comprised in the above QTLs are potential targets to personalize gene therapy protocols. Importantly, we identified two mouse strains that open new directions in characterizing continuous viral vector silencing and HIV latency. Our findings suggest that wide-range patient-specific outcomes of viral vector-based gene therapy should be expected. Thus, novel escalating dose-based clinical protocols should be considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5142c06fb0ffd0684ab6270a583153574874529" target='_blank'>
              Analysis of hepatic lentiviral vector transduction; implications on preclinical studies and clinical gene therapy protocols
              </a>
            </td>
          <td>
            Peirong Hu, Yajing Hao, Wei Tang, Graham H. Diering, Fei Zou, T. Kafri
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/beb85026de0a1065b160b9eed08ef28f424a5cc3" target='_blank'>
              The AKT inhibitor AZD5363 elicits synthetic lethality in ARID1A-deficient gastric cancer cells via induction of pyroptosis.
              </a>
            </td>
          <td>
            Menghan Fang, Youfen Lin, Chaorong Xue, Kaiqin Sheng, Zegeng Guo, Yuting Han, Hanbin Lin, Yuecheng Wu, Yuchao Sang, Xintan Chen, Stephen B. Howell, Xu Lin, Xinjian Lin
          </td>
          <td>2024-07-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0f22523498a31f883e04ffefa6645c4ecc8fbfb" target='_blank'>
              Systematic optimization of prime editing for the efficient functional correction of CFTR F508del in human airway epithelial cells.
              </a>
            </td>
          <td>
            Alexander A Sousa, Colin Hemez, Lei Lei, Soumba Traore, Katarina Kulhankova, Gregory A. Newby, Jordan L. Doman, Keyede Oye, Smriti Pandey, Philip H Karp, P. B. Mccray, David R. Liu
          </td>
          <td>2024-07-10</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Circular RNAs are a novel class of RNA transcripts, which regulate important cellular functions in health and disease. Herein, we report on the functional relevance of the circPCMTD1 transcript in acute leukemias. In screening experiments, we found that circPCMTD1 depletion strongly inhibited the proliferative capacity of leukemic cells with BCR-ABL translocations. Mass cytometry experiments identified the aberrant activation of the DNA damage response as an early downstream event of circPCMTD1 depletion. In in vivo experiments, circPCMTD1 targeting prolonged the survival of mice engrafted with leukemic blasts harboring the Philadelphia chromosome. Mechanistically, we found that circPCMTD1 was enriched in the cytoplasm and associated with the ribosomes of the leukemic cells. We detected a cryptic open reading frame within the circPCMTD1 sequence and found that circPCMTD1 could generate a peptide product. The circPCMTD1-derived peptide interacted with proteins of the BTR complex and enhanced BTR complex formation, thereby increasing tolerance to genotoxic stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015fb549e9df8d93cfd780c7c6932ae735f6848d" target='_blank'>
              circPCMTD1: A protein-coding circular RNA that regulates DNA damage response in BCR/ABL-positive leukemias
              </a>
            </td>
          <td>
            D. Papaioannou, Amog P. Urs, Rémi Buisson, A. Petri, R. Kulkarni, D. Nicolet, Lauren A. Woodward, C. Goda, Krzysztof Mrózek, Gregory K. Behbehani, Sakari Kauppinen, Ann-Kathrin Eisfeld, I. Aifantis, Guramrit Singh, A. Dorrance, Ramiro Garzon
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="Targeted therapy remains the future of anti-cancer drug development, owing to the lack of specificity of current treatments which lead to damage in healthy normal tissues. ATR inhibitors have in recent times demonstrated promising clinical potential, and are currently being evaluated in the clinic. However, despite the considerable optimism for clinical success of these inhibitors, reports of associated normal tissues toxicities remain a concern and can compromise their utility. Here, ICT10336 is reported, a newly developed hypoxia-responsive prodrug of ATR inhibitor, AZD6738, which is hypoxia-activated and specifically releases AZD6738 only in hypoxic conditions, in vitro. This hypoxia-selective release of AZD6738 inhibited ATR activation (T1989 and S428 phosphorylation) and subsequently abrogated HIF1a-mediated adaptation of hypoxic cancers cells, thus selectively inducing cell death in 2D and 3D cancer models. Importantly, in normal tissues, ICT10336 is demonstrated to be metabolically stable and less toxic to normal cells than its active parent agent, AZD6738. In addition, ICT10336 exhibited a superior and efficient multicellular penetration ability in 3D tumor models, and selectively eradicated cells at the hypoxic core compared to AZD6738. In summary, the preclinical data demonstrate a new strategy of tumor-targeted delivery of ATR inhibitors with significant potential of enhancing the therapeutic index.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11c3850ab4cd76099d6f0c2a6cba33e3c2f0419b" target='_blank'>
              Hypoxia-Responsive Prodrug of ATR Inhibitor, AZD6738, Selectively Eradicates Treatment-Resistant Cancer Cells.
              </a>
            </td>
          <td>
            Francis M Barnieh, G. Morais, P. Loadman, Robert A. Falconer, Sherif F El-Khamisy
          </td>
          <td>2024-07-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Cancer is one of the main reasons for death, and it threatens human life and health. Both the environment and genes can lead to cancers. It dates back more than a million years; more importantly, tumor cells can not be detected until they grow to a large number. Currently, cancers are treated with surgical excision or non-surgical procedures. By studying the interaction between ncRNAs and PKM2, we aim to provide new targets for diagnosis, treatment, and prognosis for cancers. Read relevant articles and made a summary and classification. Non-coding RNAs (ncRNAs) are RNAs that do not code for proteins. They perform a function in transcription and translation and can be used as targets for cancer therapy. Pyruvate kinase M2 (PKM2) is a form of PKM, and it catalyzes the glycolysis of the final cellular processes to promote tumorigenesis. Not only that, but it also plays non-metabolic functions, including the expression of the gene, cell proliferation, cell migration, and tumor angiogenesis in cancer cells. The existing studies have found that microRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs) can promote or inhibit the aerobic glycolysis of cancer cells by affecting PKM2, which increases or decrease the risk of cancers and affect the progression of cancers. This review focuses on the mechanism of ncRNAs regulating PKM2 in cancers and summarizes the roles of some ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135ae3dc56b2a8f0f10f7d3ae82e64ec70f96983" target='_blank'>
              Emerging Roles of ncRNAs Regulating PKM2 in Cancer Progression.
              </a>
            </td>
          <td>
            Haimei Xie, Jia Yu, Zhiwen Ou, Xiaoyong Lei, Xiaoyan Yang
          </td>
          <td>2024-07-15</td>
          <td>Current pharmaceutical biotechnology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Interactions between multiple genes or cis-regulatory elements (CREs) underlie a wide range of biological processes in both health and disease. High-throughput screens using dCas9 fused to epigenome editing domains have allowed researchers to assess the impact of activation or repression of both coding and non-coding genomic regions on a phenotype of interest, but assessment of genetic interactions between those elements has been limited to pairs. Here, we combine a hyper-efficient version of Lachnospiraceae bacterium dCas12a (dHyperLbCas12a) with RNA Polymerase II expression of long CRISPR RNA (crRNA) arrays to enable efficient highly-multiplexed epigenome editing. We demonstrate that this system is compatible with several activation and repression domains, including the P300 histone acetyltransferase domain and SIN3A interacting domain (SID). We also show that the dCas12a platform can perform simultaneous activation and repression using a single crRNA array via co-expression of multiple dCas12a orthologues. Lastly, demonstrate that the dCas12a system is highly effective for high-throughput screens. We use dHyperLbCas12a-KRAB and a ∼19,000-member barcoded library of crRNA arrays containing six crRNAs each to dissect the independent and combinatorial contributions of CREs to the dose-dependent control of gene expression at a glucocorticoid-responsive locus. The tools and methods introduced here create new possibilities for highly multiplexed control of gene expression in a wide variety of biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c036dd7ee414b69b8ef2f177d76c80471f4540e3" target='_blank'>
              HyperCas12a enables highly-multiplexed epigenome editing screens
              </a>
            </td>
          <td>
            Schuyler M. Melore, Marisa C. Hamilton, Timothy E. Reddy
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Retrons are promising gene editing tools because they can produce multi-copy single-stranded DNA in cells via self-primed reverse transcription. However, their potential for inserting genetic cargos in eukaryotes remains largely unexplored. Here we report the discovery and engineering of highly efficient retron-based gene editors for mammalian cells and vertebrates. Through bioinformatic analysis of metagenomic data and functional screening, we identified novel retron reverse transcriptases (RTs) that are highly active in mammalian cells. Rational design further improved the editing efficiency to levels comparable with conventional single-stranded oligodeoxynucleotide donors but from a genetically encoded cassette. Small molecule inhibitors of non-homologous end joining factors and Cas9-DNA repair protein fusions further increase homology-directed repair. Retron editors also exhibited robust activity with Cas12a nuclease and Cas9 nickase, expanding the genomic target scope and bypassing the need for a DNA double-stranded break. Using a rationally engineered retron editor, we incorporate a split GFP epitope tag for live cell imaging. Finally, we develop an all-RNA delivery strategy to enable DNA-free gene editing in cells and vertebrate embryos. This work establishes retron editors as a versatile and efficient tool for precise gene editing, offering new opportunities for biotechnology and biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8f2c620f5c7bf4a518c60425b0c63267672964" target='_blank'>
              Discovery and Engineering of Retrons for Precise Genome Editing
              </a>
            </td>
          <td>
            Jesse D. Buffington, Hung-Che Kuo, Kuang Hu, You-Chiun Chang, Kamyab Javanmardi, Brittney Voigt, Yi-Ru Li, Mary E. Little, Sravan K Devanathan, Blerta Xhemalçe, R. Gray, Ilya J. Finkelstein
          </td>
          <td>2024-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>