<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-19 06:29:19 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental & Molecular Medicine, Experimental and Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>2</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2f585a6d644d3577541d684c98016ab2b165e02" target='_blank'>
              Sensing and guiding cell-state transitions by using genetically encoded endoribonuclease-mediated microRNA sensors.
              </a>
            </td>
          <td>
            Lei Wang, Wenlong Xu, Shun Zhang, Gregory C Gundberg, Christine R Zheng, Zhengpeng Wan, Kamila Mustafina, Fabio Caliendo, Hayden Sandt, Roger Kamm, Ron Weiss
          </td>
          <td>2024-07-09</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Therapeutic resistance is the leading cause of treatment failure and death from cancer. While resistance can be driven by genetic mutations, mounting evidence also points to an epigenetic basis of resistance. Much of this epigenetic, or non-genetic, resistance has been attributed to drug-resistant transcriptional cell states that are either induced by drug treatment or pre-exist in a fraction of cells selected by treatment. However, the extent to which long-term resistance is manifested in the early inherent cellular response to drugs is poorly understood, and which aspects, if any, of this early response to drug-induced transcriptional response are evolutionarily conserved properties of cells. To address these questions, we integrate datasets of long-term drug resistance and early drug response data across multiple cell lines with drug response and resistance data from bacteria and yeast. Our findings suggest that cancer cell states in both the drug-naive populations as well as in populations shortly after treatment share transcriptional properties with fully established resistant cell populations, and CRISPR-cas9 knockout of transcription factors predicted to regulate the resistant transcriptional programs result in inceased drug sensitivity. Furthermore, the resistance states manifested as early drug response are evolutionarily conserved. Finally, we show that early resistant states discriminate responders from non-responders across multiple human cancer trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ad4a4d49f9d38cdc5b6bcd770ac201bba19fb9f" target='_blank'>
              Transcriptional Characterization of Resistance in Early Drug Response
              </a>
            </td>
          <td>
            Cole Ruoff, Allison V. Mitchell, Priya Mondal, Vishaka Gopalan, Michael M. Gottesman, Sridhar Hannenhalli
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cancer resistance refers to the ability of cancer cells to evade the effects of treatments, such as chemotherapy and radiation, making it challenging to eliminate tumors. Cancer cells develop resistance through genetic modifications, alterations in cellular pathways, or microenvironmental adaptations that allow cancer cells to persist, proliferate, and even thrive despite the application of therapies designed to eradicate them. Despite efforts to create more potent target-based drugs, the flexibility of acquired resistance necessitates advanced options. Combination therapy and precision immunotherapy has revolutionized treatment but is limited by patient specificity and requires further development. New approaches like small molecules, peptides, and nanotherapeutics aim to overcome resistance by enhancing site-specific delivery and increasing drug concentration inside cancer cells. This review provides a comprehensive overview of the molecular mechanisms underlying cancer drug resistance, including genetic mutations, epigenetic alterations, efflux pump activity, and the influence of the tumor microenvironment. Additionally, the roles of microRNAs, long noncoding RNAs, and cellular processes such as autophagy and hypoxia in mediating resistance are examined. This review seeks to improve patient care and help to the development of more effective cancer medicines by explaining these complicated systems and investigating novel therapeutic alternatives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c262ab9bdd31ff19f10e8cd7a734094b9985bc60" target='_blank'>
              An overview of molecular mechanisms in cancer drug resistance and therapeutic strategies
              </a>
            </td>
          <td>
            Navin Kumar, Ranjeeta Bansala
          </td>
          <td>2024-07-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Purpose: Complex chromosomal rearrangements (CCRs) are rare structural variants involving three or more chromosomal break- points. Most de novo reported CCRs pose challenges for diagnosis and management. They often require karyotyping, fluorescence in situ hybridization (FISH), and chromosomal microarray analysis (CMA) for clinical diagnosis because of the limitations of each method. Here we report an inherited exceptionally complex CCR involving 4 chromosomes and 11 breakpoints in a family with multisystem anomalies. Methods: We evaluated the CCRs using karyotyping, FISH, CMA, and two emerging genomic technologies: high-throughput chromosome conformation capture sequencing (Hi-C; aka genomic proximity mapping, GPM) and optical genome mapping (OGM). We also performed functional studies using transcriptome and methylome analyses. Results: The proband, who had intellectual disability and immune deficiency, shared CCRs with her unaffected mother involving chromosomes 1, 7, and 11 by karyotyping. However, CMA revealed a duplication and three deletions in the proband in contrast to her mother's balanced genome. Hi-C (GPM) and OGM detected the CCRs and copy number alterations but also uncovered additional breakpoints at high resolution, including an insertion in 4p and two cryptic rearrangements at 7p. Transcriptome and methylome analyses identified likely biological pathways associated with the proband's phenotypes. Conclusion: Combining cytogenetic and genomic methods provided comprehensive characterization and defined the breakpoints at high resolution in both proband and mother. This underscores the value of novel cytogenetic and genomic techniques in deciphering complex genome rearrangements and the significance of integrative genomic analysis and functional characterization in understanding clinical phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d8dc703c96a52057ab6829a72726f61eb74bd56" target='_blank'>
              Genetic and functional characterization of inherited complex chromosomal rearrangements in a family with multisystem anomalies
              </a>
            </td>
          <td>
            H. Fang, S. M. Eacker, Y. Wu, W. Neufeld-Kaiser, M. Laurino, S. Keel, M. S. Horwitz, Y. Liu
          </td>
          <td>2024-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epithelial-to-mesenchymal transition (EMT) is a major axis of phenotypic plasticity not only in diseased conditions such as cancer metastasis and fibrosis but also during normal development and wound healing. Yet-another important axis of plasticity with metastatic implications includes the cancer stem cell (CSCs) and non-CSC transitions. However, in both processes, epithelial (E) and mesenchymal (M) phenotypes are not merely binary states. Cancer cells acquire a spectrum of phenotypes with traits, properties, and markers of both E and M phenotypes, giving rise to intermediary hybrid (E/M) phenotypes. E/M cells play an important role in tumor initiation, metastasis, and disease progression in multiple cancers. Furthermore, the hybrid phenotypes also play a major role in causing therapeutic resistance in cancer. Here, we discuss how a systems biology perspective on the problem, which is implicit in the ‘Team Medicine’ approach outlined in the theme of this Special Issue of The Journal of Clinical Medicine and includes an interdisciplinary team of experts, is more likely to shed new light on EMT in cancer and help us to identify novel therapeutics and strategies to target phenotypic plasticity in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6867e0a77d67eee55a122d53d387d619f33219b" target='_blank'>
              Phenotypic Plasticity and Cancer: A System Biology Perspective
              </a>
            </td>
          <td>
            A. R. Subbalakshmi, S. Ramisetty, Atish Mohanty, Siddhika Pareek, Dana Do, Sagun Shrestha, Ajaz Khan, Neel Talwar, Tingting Tan, Priya Vishnubhotla, Sharad S Singhal, Ravi Salgia, Prakash Kulkarni
          </td>
          <td>2024-07-23</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Outcomes for women with breast cancer have improved dramatically in recent decades. However, many patients present with intrinsic drug resistance and others are initially sensitive to anti-cancer drugs but acquire resistance during the course of their treatment, leading to recurrence and/or metastasis. Drug therapy-induced senescence (TIS) is a form of drug resistance characterised by the induction of cell cycle arrest and the emergence of a senescence-associated secretory phenotype (SASP) that can develop in response to chemo- and targeted- therapies. A wide range of anticancer interventions can lead to cell cycle arrest and SASP induction, by inducing genotoxic stress, hyperactivation of signalling pathways or oxidative stress. TIS can be anti-tumorigenic in the short-term, but pro-tumorigenic in the long-term by creating a pro-inflammatory and immunosuppressive microenvironment. Moreover, the SASP can promote angiogenesis and epithelial-mesenchymal transition in neighbouring cells. In this review, we will describe the characteristics of TIS in breast cancer and detail the changes in phenotype that accompany its induction. We also discuss strategies for targeting senescent cancer cells in order to prevent or delay tumour recurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1902ff7d1c0379c707774d89c56e002ae6793dc1" target='_blank'>
              Therapy-induced senescence in breast cancer: an overview
              </a>
            </td>
          <td>
            Suraj Narayanan Chembukavu, Andrew J Lindsay
          </td>
          <td>2024-07-25</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfd5cecdc67c5ea14eee4cb838269e77eddb6f01" target='_blank'>
              Farnesyltransferase inhibition overcomes oncogene-addicted non-small cell lung cancer adaptive resistance to targeted therapies
              </a>
            </td>
          <td>
            S. Figarol, C. Delahaye, R. Gence, A. Doussine, J. Cerapio, Mathylda Brachais, Claudine Tardy, N. Béry, R. Asslan, J. Colinge, J. Villemin, Antonio Maraver, Irene Ferrer, Luis Paz-Ares, Linda Kessler, Francis Burrows, Isabelle Lajoie-Mazenc, Vincent Dongay, Clara Morin, Amélie Florent, S. Pagano, E. Taranchon-Clermont, A. Casanova, A. Pradines, Julien Mazieres, Gilles Favre, O. Calvayrac
          </td>
          <td>2024-06-27</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Cancer is the second leading cause of global mortality and claims approximately 10 million lives annually. Despite advances in treatments such as surgery, chemotherapy, and immunotherapy, resistance to these methods has emerged. Multidrug resistance (MDR), where cancer cells resist diverse treatments, undermines therapy effectiveness, escalating mortality rates. MDR mechanisms include, among others, drug inactivation, reduced drug uptake, enhanced DNA repair, and activation of survival pathways such as autophagy. Moreover, MDR mechanisms can confer resistance to other therapies like radiotherapy. Understanding these mechanisms is crucial for improving treatment efficacy and identifying new therapeutic targets. Extracellular vesicles (EVs) have gathered attention for their role in cancer progression, including MDR development through protein transfer and genetic reprogramming. Autophagy, a process balancing cellular resources, also influences MDR. The intersection of EVs and autophagy further complicates the understanding of MDR. Both extracellular (exosomes, microvesicles) and intracellular (autophagic) vesicles contribute to therapy resistance by regulating the tumor microenvironment, facilitating cell communication, and modulating drug processing. While much is known about these pathways, there is still a need to explore their potential for predicting treatment responses and understanding tumor heterogeneity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5249ba1d82fab28ebb5cc330d8e07a3024195f" target='_blank'>
              The Role of Intracellular and Extracellular Vesicles in the Development of Therapy Resistance in Cancer.
              </a>
            </td>
          <td>
            Magdalena Wilczak, Magdalena Surman, M. Przybyło
          </td>
          <td>2024-08-07</td>
          <td>Current pharmaceutical design</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Metabolic reprogramming is a hallmark of cancer, enabling tumor cells to sustain rapid proliferation, resist cell death, and adapt to varying microenvironmental conditions. This review elucidates the key mechanisms underlying metabolic reprogramming in cancer cells, including alterations in glucose metabolism, glutamine addiction, and lipid biosynthesis. A central feature of cancer metabolism is the Warburg effect, where cancer cells preferentially utilize aerobic glycolysis over oxidative phosphorylation, even in the presence of oxygen. This metabolic shift is driven by oncogenes and tumor suppressor genes, such as MYC and TP53, which modulate the expression and activity of enzymes involved in glycolysis and mitochondrial function. Additionally, cancer cells exhibit increased glutaminolysis, relying on glutamine as a carbon and nitrogen source to support anabolic processes and redox balance. Lipid metabolism is also reprogrammed, with enhanced de novo lipogenesis supplying membrane components and signaling molecules critical for tumor growth and survival. Understanding these metabolic alterations provides a basis for developing targeted therapies. Several therapeutic strategies have emerged, including inhibitors of key metabolic enzymes, such as hexokinase 2 (HK2), pyruvate kinase M2 (PKM2), and glutaminase. Metabolic interventions can disrupt the metabolic flexibility of cancer cells, sensitizing them to conventional therapies and overcoming resistance mechanisms. This review highlights the potential of metabolic targets in cancer treatment and emphasizes the need for personalized approaches considering tumor-specific metabolic profiles. Future research should focus on identifying biomarkers of metabolic vulnerabilities and developing combination therapies that exploit the intricate metabolic dependencies of cancer cells. Ultimately, targeting metabolic reprogramming holds promise for improving cancer prognosis and achieving more effective and durable therapeutic outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d015236b22d21b366aae600c9f9e8d4b15a3c83" target='_blank'>
              A review of metabolic reprogramming in cancer cells: Mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Kelly Osayi Otakhor, Elizabeth O. Soladoye
          </td>
          <td>2024-07-30</td>
          <td>World Journal of Advanced Research and Reviews</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="All routine clinical treatments for colorectal cancer include 5-fluorouracil (5-FU), which cannot counteract recurrence and metastases formation. As the pyrimidine analog 5-FU can impact multiple pathways including both DNA and RNA metabolism, studying its mode of actions could lead to improved therapies. Using a dedicated reporter system for lineage-tracing and deep translatome profiling we demonstrate that 5-FU causes some colorectal cancer cells to tolerate the drug, due to a durable translational reprogramming that sustains cell plasticity. This period of drug tolerance coincides with specific translational activation of genes coding for proteins with major pro-tumoral functions. We unravel a major unexpected translational overexpression of the pro-inflammatory and pro-tumoral IL-8 cytokine, alongside other anti-apoptotic, senescence-associated secretory phenotype and cancer-related senescence phenotype genes. Given the adverse prognostic implications of elevated IL-8 levels across various cancers, our findings suggest IL-8 targeting could counteract 5-FU resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3f76dde38afaccb213072b4b26b46e5ff73bd2b" target='_blank'>
              Translational control of cell plasticity drives 5-FU tolerance
              </a>
            </td>
          <td>
            Mounira Chalabi-Dchar, Olivia Villeronce, J. Ripoll, A. Vincent, T. Fenouil, Rita Khoueiry, Jérôme Kucharczak, Laura Jentschel, Frédéric Catez, Arnaud M Vigneron, Julie Tréguier, Céline Mandier, Céline Bouclier, Jihane Vitre, Louise Lagerqvist, Armelle Choquet, Z. Herceg, C. Machon, Jérôme Guitton, Alexandre David, Eric Solary, David Bernard, Nadine Martin, Eric Rivals, N. D. Venezia, Julie Pannequin, J- J. Diaz
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Copy number alterations (CNAs), resulting from the gain or loss of genetic material from as little as 50 base pairs or as big as entire chromosome(s), have been associated with many congenital diseases, de novo syndromes and cancer. It is established that CNAs disturb the dosage of genomic regions including enhancers/promoters, long non-coding RNA and gene(s) among others, ultimately leading to an altered balance of key cellular functions. In cancer, CNAs have been associated with almost all steps of the disease: predisposition, initiation, development, maintenance, response to treatment, resistance, and relapse. Therefore, understanding how specific CNAs contribute to tumourigenesis may provide prognostic insight and ultimately lead to the development of new therapeutic approaches to improve patient outcomes. In this review, we provide a snapshot of what is currently known about CNAs and cancer, incorporating topics regarding their detection, clinical impact, origin, and nature, and discuss the integration of innovative genetic engineering strategies, to highlight the potential for targeting CNAs using novel, dosage-sensitive and less toxic therapies for CNA-driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b107cf5f7a3a29e8e0fe916d4102734572eefc3a" target='_blank'>
              Insights into the Clinical, Biological and Therapeutic Impact of Copy Number Alteration in Cancer
              </a>
            </td>
          <td>
            Shannon L. Carey-Smith, R. Kotecha, L. C. Cheung, Sébastien Malinge
          </td>
          <td>2024-06-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cellular responses to environmental stimuli are typically thought to be governed by genetically encoded programs. We demonstrate that melanoma cells can form and maintain cellular memories during the acquisition of therapy resistance that exhibit characteristics of cellular learning and are dependent on the transcription factor AP-1. We show that cells exposed to a low dose of therapy adapt to become resistant to a high dose, demonstrating that resistance was not purely selective. The application of therapy itself results in the encoding of transient gene expression into cellular memory and that this encoding occurs for both transiently induced and probabilistically arising expression. Chromatin accessibility showed concomitant persistence. A two-color AP-1 reporter system showed that these memories are encoded in cis, constituting an example of activating cis epigenetics. Our findings establish the formation and maintenance of cellular memories as a critical aspect of gene regulation during the development of therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34a15dd395d0e2a350d250357de4baa259bd2f20" target='_blank'>
              AP-1 Mediates Cellular Adaptation and Memory Formation During Therapy Resistance
              </a>
            </td>
          <td>
            Jingxin Li, Pavithran T. Ravindran, Aoife O’Farrell, Gianna T. Busch, Ryan H. Boe, Zijian Niu, Sean Woo, Margaret C. Dunagin, Naveen Jain, Yogesh Goyal, K. Sarma, M. Herlyn, Arjun Raj
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>113</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821cee27b336dd922ca1d195350feabe9168efcf" target='_blank'>
              CRISPR-Cas9 potential for identifying novel therapeutic targets in muscle-invasive bladder cancer.
              </a>
            </td>
          <td>
            Danielle J Smith, S. Lunj, Antony D Adamson, Sankari Nagarajan, Tim A D Smith, Kimberley J Reeves, P. Hoskin, Ananya Choudhury
          </td>
          <td>2024-07-01</td>
          <td>Nature reviews. Urology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7eaccde1d6ccb6dacff234a0f6ddffc263c341a3" target='_blank'>
              Targeting oncogene-induced cellular plasticity for tumor therapy
              </a>
            </td>
          <td>
            Bin Li, Lingling Zheng, Jianhua Yang, Lianghu Qu
          </td>
          <td>2024-07-15</td>
          <td>Advanced Biotechnology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ae5740675a47153265ae470439ce458fde35c02" target='_blank'>
              Single-cell sensor analyses reveal signaling programs enabling Ras-G12C drug resistance.
              </a>
            </td>
          <td>
            Jason Z Zhang, Shao-En Ong, D. Baker, D. Maly
          </td>
          <td>2024-08-05</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A. Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M. As Sobeai
          </td>
          <td>2024-08-05</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="A fundamental obstacle to tackling the antimicrobial resistance crsisis is identifying mutations that lead to resistance in a given genomic background and environment. We present a high-throughput technique – Quantitative Mutational Scan Sequencing (QMS-Seq) – that enables quantitative comparison of which genes are under antibiotic selection and captures how genetic background influences resistance evolution. We compared four E. coli strains exposed to ciprofloxacin, cycloserine, or nitrofurantoin and identified 975 resistance mutations, many in genes and regulatory regions not previously associated with resistance. QMS-Seq revealed that multi-drug and antibiotic-specific resistance are acquired through categorically different types of mutations, and that minor genotypic differences significantly influence evolutionary routes to resistance. By quantifying mutation frequency with single base pair resolution, QMS-Seq informs about the underlying mechanisms of resistance and identifies mutational hotspots within genes. Our method provides a way to rapidly screen for resistance mutations while assessing the impact of multiple confounding factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baea8e2ed712d39dbedd898121a034064f71df57" target='_blank'>
              High-throughput method rapidly characterizes hundreds of novel antibiotic resistance mutations
              </a>
            </td>
          <td>
            Matthew J. Jago, Jake K. Soley, Stepan Denisov, Calum J. Walsh, Danna R. Gifford, Benjamin P. Howden, M. Lagator
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Epigenetic evolution is a common mechanism used by cancer cells to evade the therapeutic effects of drug treatment. In ovarian cancers, epigenetically-driven resistance may be responsible for a large number of late-stage patient deaths. Here, we describe the first investigation into the role of G-quadruplex (G4) DNA secondary structures in mediating epigenetic regulation in drug-resistant ovarian cancer cells. Through genome-wide mapping of G4s in paired drug-sensitive and drug-resistant cell lines, we find that increased G4 formation is associated with significant increase in gene expression, with high enrichment in signalling pathways previously established to promote drug-resistant states. However, in contrast to previous studies, the expression-enhancing effects of G4s were not found at gene promoters, but intergenic and intronic regions, indicating that G4s promote long-range transcriptional regulation in drug-resistant cells. Furthermore, we discovered that clusters of G4s (super-G4s) are associated with particularly high levels of transcriptional enhancement that surpass the effects of super-enhancers, which act as well established regulatory sites in many cancers. Finally, we demonstrate that targeting G4s with small molecules results in significant down-regulation of pathways associated with drug-resistance, which results in resensitisation of resistant cells to chemotherapy agents. These findings indicate that G4 structures are critical for the epigenetic regulatory networks of drug-resistant cells and may represent a promising target to treat drug-tolerant ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef77f99d4d1172d6a547e45d37053b1f52fb2ad" target='_blank'>
              G-quadruplex structures regulate long-range transcriptional reprogramming to promote drug resistance in ovarian cancer
              </a>
            </td>
          <td>
            Jenna Robinson, Gem Flint, I. Garner, Silvia Galli, Thomas E. Maher, Marina K. Kuimova, Ramon Vilar, I. McNeish, Robert Brown, Hector Keun, Marco Di Antonio
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Multiple myeloma (MM) is a genetically heterogeneous disease and the management of relapses is one of the biggest clinical challenges. TP53 alterations are established high‐risk markers and are included in the current disease staging criteria. KRAS is the most frequently mutated gene affecting around 20% of MM patients. Applying Clonal Competition Assays (CCA) by co‐culturing color‐labeled genetically modified cell models, we recently showed that mono‐ and biallelic alterations in TP53 transmit a fitness advantage to the cells. Here, we report a similar dynamic for two mutations in KRAS (G12A and A146T), providing a biological rationale for the high frequency of KRAS and TP53 alterations at MM relapse. Resistance mutations, on the other hand, did not endow MM cells with a general fitness advantage but rather presented a disadvantage compared to the wild‐type. CUL4B KO and IKZF1 A152T transmit resistance against immunomodulatory agents, PSMB5 A20T to proteasome inhibition. However, MM cells harboring such lesions only outcompete the culture in the presence of the respective drug. To better prevent the selection of clones with the potential of inducing relapse, these results argue in favor of treatment‐free breaks or a switch of the drug class given as maintenance therapy. In summary, the fitness benefit of TP53 and KRAS mutations was not treatment‐related, unlike patient‐derived drug resistance alterations that may only induce an advantage under treatment. CCAs are suitable models for the study of clonal evolution and competitive (dis)advantages conveyed by a specific genetic lesion of interest, and their dependence on external factors such as the treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b736cb2bf2240ae0e1d6133faf0f979a9afbbf92" target='_blank'>
              Clonal competition assays identify fitness signatures in cancer progression and resistance in multiple myeloma
              </a>
            </td>
          <td>
            L. Haertle, U. Munawar, Hipólito N. C. Hernández, Andrés Arroyo-Barea, T. Heckel, I. Cuenca, Lucia Martin, Carlotta Höschle, Nicole Müller, C. Vogt, Thorsten Bischler, Paula L. del Campo, Seungbin Han, Natalia Buenache, Xiang Zhou, Florian Bassermann, J. Waldschmidt, Torsten Steinbrunn, L. Rasche, T. Stühmer, J. Martínez-López, K. Martin Kortüm, S. Barrio
          </td>
          <td>2024-07-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23cf92719cd49496400fba609df94fa13b009008" target='_blank'>
              Precision medicine in AML: overcoming resistance.
              </a>
            </td>
          <td>
            Samuel Urrutia, Koichi Takahashi
          </td>
          <td>2024-08-01</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Breast cancer heterogeneity presents a significant challenge in clinical therapy, such as over-treatment and drug resistance. These challenges are largely due to its obscure normal epithelial origins, evolutionary stability, and transitions on the cancer subtypes. This study aims to elucidate the cellular emergence and maintenance of heterogeneous breast cancer via quantitative bio-process modeling, with potential benefit to therapeutic strategies for the disease. An endogenous molecular–cellular hypothesis posits that both pathological and physiological states are phenotypes evolved from and shaped by interactions among a number of conserved modules and cellular factors within a biological network. We hereby developed a model of core endogenous network for breast cancer in accordance with the theory, quantifying its intrinsic dynamic properties with dynamic modeling. The model spontaneously generates cell states that align with molecular classifications at both the molecular and modular level, replicating four widely recognized molecular subtypes of the cancer and validating against data extracted from the TCGA database. Further analysis shows that topologically, a singular progression gateway from normal breast cells to cancerous states is identified as the Luminal A-type breast cancer. Activated positive feedback loops are found to stabilize cellular states, while negative feedback loops facilitate state transitions. Overall, more routes are revealed on the cellular transition between stable states, and a traceable count explains the origin of breast cancer heterogeneity. Ultimately, the research intended to strength the search for therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36bf7724db6a608fda16eecf1282fe4b12434998" target='_blank'>
              Heterogeneous Evolution of Breast Cancer Cells—An Endogenous Molecular-Cellular Network Study
              </a>
            </td>
          <td>
            Tianqi Li, Yong-Cong Chen, Ping Ao
          </td>
          <td>2024-07-26</td>
          <td>Biology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e387009a4ed97bf637b9dd6a3591f151035e10" target='_blank'>
              Insights into gemcitabine resistance in pancreatic cancer: association with metabolic reprogramming and TP53 pathogenicity in patient derived xenografts
              </a>
            </td>
          <td>
            Mariam M. Konaté, Julia Krushkal, Ming-Chung Li, Li Chen, Yuri Kotliarov, Alida Palmisano, Rini Pauly, Qian Xie, P. M. Williams, Lisa M. McShane, Yingdong Zhao
          </td>
          <td>2024-08-05</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Epitranscriptomics is a field that delves into post-transcriptional changes. Among these modifications, the conversion of adenosine to inosine, traduced as guanosine (A>I(G)), is one of the known RNA-editing mechanisms, catalyzed by ADARs. This type of RNA editing is the most common type of editing in mammals and contributes to biological diversity. Disruption in the A>I(G) RNA-editing balance has been linked to diseases, including several types of cancer. Drug resistance in patients with cancer represents a significant public health concern, contributing to increased mortality rates resulting from therapy non-responsiveness and disease progression, representing the greatest challenge for researchers in this field. The A>I(G) RNA editing is involved in several mechanisms over the immunotherapy and genotoxic drug response and drug resistance. This review investigates the relationship between ADAR1 and specific A>I(G) RNA-edited sites, focusing particularly on breast cancer, and the impact of these sites on DNA damage repair and the immune response over anti-cancer therapy. We address the underlying mechanisms, bioinformatics, and in vitro strategies for the identification and validation of A>I(G) RNA-edited sites. We gathered databases related to A>I(G) RNA editing and cancer and discussed the potential clinical and research implications of understanding A>I(G) RNA-editing patterns. Understanding the intricate role of ADAR1-mediated A>I(G) RNA editing in breast cancer holds significant promise for the development of personalized treatment approaches tailored to individual patients’ A>I(G) RNA-editing profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e391d01dc14e829f34fcad7c86a78c7ddae70fe2" target='_blank'>
              ADAR-Mediated A>I(G) RNA Editing in the Genotoxic Drug Response of Breast Cancer
              </a>
            </td>
          <td>
            Yanara A. Bernal, Eduardo Durán, Isidora Solar, Eduardo A. Sagredo, R. Armisén
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fba17e91584024423ac0c6e3cc0e42afd48ee9aa" target='_blank'>
              Emerging advances in defining the molecular and therapeutic landscape of small-cell lung cancer.
              </a>
            </td>
          <td>
            Triparna Sen, N. Takahashi, Subhamoy Chakraborty, Naoko Takebe, Amin H. Nassar, Nagla A Karim, S. Puri, Abdul Rafeh Naqash
          </td>
          <td>2024-07-04</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8428e89731398de2122358b2387bfd15c652d8d" target='_blank'>
              Homopolymer switches mediate adaptive mutability in mismatch repair-deficient colorectal cancer
              </a>
            </td>
          <td>
            H. Kayhanian, William Cross, Suzanne E M van der Horst, Panagiotis Barmpoutis, E. Lakatos, G. Caravagna, L. Zapata, Arne van Hoeck, Sjors Middelkamp, Kevin Litchfield, Christopher Steele, W. Waddingham, Dominic Patel, Salvatore Milite, Chen Jin, A. Baker, Daniel C Alexander, Khurum Khan, Daniel Hochhauser, Marco Novelli, Benjamin Werner, R. van Boxtel, Joris H. Hageman, Julian R Buissant des Amorie, Josep Linares, M. Ligtenberg, I. Nagtegaal, M. Laclé, Leon M G Moons, L. Brosens, N. Pillay, A. Sottoriva, Trevor A Graham, M. Rodriguez-Justo, K. Shiu, Hugo J G Snippert, Marnix Jansen
          </td>
          <td>2024-07-03</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Genetic and epigenetic alterations are cancer hallmark characteristics. However, the role of inherited cancer predisposition alleles in co-opting lineage factor epigenetic reprogramming and tumor progression remains elusive. Here the FinnGen cohort phenome-wide analysis, along with multiple genome-wide association studies, has consistently identified the rs339331-RFX6/6q22 locus associated with prostate cancer (PCa) risk across diverse populations. It is uncovered that rs339331 resides in a reprogrammed androgen receptor (AR) binding site in PCa tumors, with the T risk allele enhancing AR chromatin occupancy. RFX6, an AR-regulated gene linked to rs339331, exhibits synergistic prognostic value for PCa recurrence and metastasis. This comprehensive in vitro and in vivo studies demonstrate the oncogenic functions of RFX6 in promoting PCa cell proliferation and metastasis. Mechanistically, RFX6 upregulates HOXA10 that profoundly correlates with adverse PCa outcomes and is pivotal in RFX6-mediated PCa progression, facilitating the epithelial-mesenchymal transition (EMT) and modulating the TGFβ/SMAD signaling axis. Clinically, HOXA10 elevation is associated with increased EMT scores, tumor advancement and PCa recurrence. Remarkably, reducing RFX6 expression restores enzalutamide sensitivity in resistant PCa cells and tumors. This findings reveal a complex interplay of genetic and epigenetic mechanisms in PCa pathogenesis and drug resistance, centered around disrupted prostate lineage AR signaling and abnormal RFX6 expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/643608faf0fae4b398ea3c97d76a7dd5d2fe17e3" target='_blank'>
              An Inherited Allele Confers Prostate Cancer Progression and Drug Resistance via RFX6/HOXA10-Orchestrated TGFβ Signaling.
              </a>
            </td>
          <td>
            Mengjie Zhong, Wenjie Xu, Pan Tian, Qin Zhang, Zixian Wang, Limiao Liang, Qixiang Zhang, Yuehong Yang, Ying Lu, Gong-Hong Wei
          </td>
          <td>2024-06-26</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cf8519f1fd7b9c6b19231a2f05380848e656a1a" target='_blank'>
              Altered ribosomal profile in acquired resistance and reversal associates with pathological response to chemotherapy in inflammatory breast cancer
              </a>
            </td>
          <td>
            G. Devi, Pritha Pai, Seayoung Lee, Matthew W. Foster, Dorababu S. Sannareddy, F. Bertucci, Naoto Ueno, S. V. van Laere
          </td>
          <td>2024-07-29</td>
          <td>NPJ Breast Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Next-generation sequencing (NGS) technology has revolutionized cancer research and treatment by enabling comprehensive analysis of genetic mutations, alterations, and expression profiles. It allows for the identification of cancer-driving mutations, helping in the development of targeted therapies. NGS provides detailed insights into tumor heterogeneity, resistance mechanisms, and clonal evolution. Its high-throughput capacity facilitates large-scale studies, improving our understanding of cancer genomics. By enabling personalized treatment plans based on individual genetic profiles, NGS holds promise for more effective and tailored cancer therapies. Early reviews on cancer genomics often lacked comprehensive coverage of emerging technologies. They missed in-depth analysis of NGS advancements, their impact on cancer research, and clinical applications. The review addresses this gap by reviving a thorough examination of NGS methods, their role in identifying genetic mutations, and their potential in personalized cancer treatment, thus providing essential insights into the evolving landscape of cancer genomics. The article covers the advancements in technology and bioinformatic approaches for NGS data analysis. It delves into NGS applications in research and diagnostics, particularly for solid cancer diagnosis. The review highlights specific cancer types, including hereditary breast cancer, melanoma, prostate cancer, thyroid cancer, lung cancer, and colorectal cancer. It explores NGS contribution in understanding the genetic basis of these cancers and its potential for enhancing personalized diagnosis and treatment strategies. This review rectifies early lacunas by providing a comprehensive and updated examination of NGS technology, addressing gaps in previous analyses and emphasizes bioinformatic approaches for NGS data analysis, crucial for interpreting vast genomic data accurately. The review meets the current need for a thorough understanding of NGS’s role in personalized cancer treatment and research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39fa1bb9c8e82b88cec555956d178d8b56b6683c" target='_blank'>
              Next-generation sequencing technology in cancer
              </a>
            </td>
          <td>
            P. Krubaa, Dr. Anand Mohan Jha, Prof Dr.Ammar A.Razzak Mahmood, Dr. Anil Kumar, D. Abraham
          </td>
          <td>2024-07-18</td>
          <td>International Journal of Trends in OncoScience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0807e64db184649a4b0df3bd08502c4d34be0a53" target='_blank'>
              Programming tumor evolution with selection gene drives to proactively combat drug resistance.
              </a>
            </td>
          <td>
            Scott M. Leighow, Josh Reynolds, Ivan Sokirniy, Shun Yao, Zeyu Yang, Haider Inam, Dominik Wodarz, Marco Archetti, Justin Pritchard
          </td>
          <td>2024-07-04</td>
          <td>Nature biotechnology</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e223c600817ad1cd6259f50a38399720abf1f7d" target='_blank'>
              Drug tolerant persister cell plasticity in cancer: A revolutionary strategy for more effective anticancer therapies
              </a>
            </td>
          <td>
            Jun He, Zejing Qiu, Jingjing Fan, Xiaohong Xie, Qinsong Sheng, Xinbing Sui
          </td>
          <td>2024-08-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Simple Summary Diffuse large B-cell lymphoma is the most common type of lymphoma. Despite initial treatment, up to 40% of patients do not achieve a cure and need second-line therapy. For those experiencing late relapse or lacking access to CAR-T cell therapy, platinum-based chemotherapy followed by stem cell transplantation remains the standard of care. In this study, we used genomewide CRISPR/Cas9 screens and single-gene knockout experiments to identify the genes associated with responses to platinum-based drugs. We provide a comprehensive list of genes involved in the response to cisplatin in DLBCL. Our functional experiments highlight the critical roles of the DNA damage response genes XPA and ERCC6, in addition to BTK, in the response to platinum-based drugs. Additionally, we show that inhibition of BTK at lower concentrations sensitizes DLBCL cells to platinum-based drugs. Abstract The recurrence of diffuse large B-cell lymphoma (DLBCL) has been observed in 40% of cases. The standard of care for refractory/relapsed DLBCL (RR-DLBCL) is platinum-based treatment prior to autologous stem cell transplantation; however, the prognosis for RR-DLBCL patients remains poor. Thus, to identify genes affecting the cisplatin response in DLBCL, cisplatin-based whole-genome CRISPR-Cas9 knockout screens were performed in this study. We discovered DNA damage response (DDR) pathways as enriched among identified sensitizing CRISPR-mediated gene knockouts. In line, the knockout of the nucleotide excision repair genes XPA and ERCC6 sensitized DLBCL cells to platinum drugs irrespective of proliferation rate, thus documenting DDR as essential for cisplatin sensitivity in DLBCL. Functional analysis revealed that the loss of XPA and ERCC6 increased DNA damage levels and altered cell cycle distribution. Interestingly, we also identified BTK, which is involved in B-cell receptor signaling, to affect cisplatin response. The knockout of BTK increased cisplatin sensitivity in DLBCL cells, and combinatory drug screens revealed a synergistic effect of the BTK inhibitor, ibrutinib, with platinum drugs at low concentrations. Applying local and external DLBCL cohorts, we addressed the clinical relevance of the genes identified in the CRISPR screens. BTK was among the most frequently mutated genes with a frequency of 3–5%, and XPA and ERCC6 were also mutated, albeit at lower frequencies. Furthermore, 27–54% of diagnostic DLBCL samples had mutations in pathways that can sensitize cells to cisplatin. In conclusion, this study shows that XPA and ERCC6, in addition to BTK, are essential for the response to platinum-based drugs in DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2794e056513b2d174f98b17ca50338733fe72c8" target='_blank'>
              CRISPR-Cas9 Knockout Screens Identify DNA Damage Response Pathways and BTK as Essential for Cisplatin Response in Diffuse Large B-Cell Lymphoma
              </a>
            </td>
          <td>
            I. I. Issa, Hanne Due, R. Brøndum, Vidthdyan Veeravakaran, Hulda Haraldsdóttir, Cathrine Sylvester, Asta Brogaard, Soniya Dhanjal, Bernhard Schmierer, K. Dybkær
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Simple Summary Lung adenocarcinoma, the most prevalent type of lung cancer, presents a significant treatment challenge owing to drug resistance. This study aimed to investigate the role of long non-coding RNAs (lncRNAs) in promoting intrinsic resistance in three lung adenocarcinoma cell lines from the onset of treatment. Drug-tolerant persister (DTP) cells, a subset of cancer cells with the ability to survive and proliferate after exposure to therapeutic drugs, were generated. RNA sequencing was used to investigate the differential expression of lncRNAs, and the clinical relevance of lncRNAs was assessed in a cohort of lung adenocarcinoma patients from The Cancer Genome Atlas. Knockdown of these lncRNAs increases the sensitivity of the analyzed cell lines to therapeutic drugs. This study provides an opportunity to investigate the role of lncRNAs in the genetic and epigenetic mechanisms underlying intrinsic resistance. Abstract Lung adenocarcinoma is the most prevalent form of lung cancer, and drug resistance poses a significant obstacle in its treatment. This study aimed to investigate the overexpression of long non-coding RNAs (lncRNAs) as a mechanism that promotes intrinsic resistance in tumor cells from the onset of treatment. Drug-tolerant persister (DTP) cells are a subset of cancer cells that survive and proliferate after exposure to therapeutic drugs, making them an essential object of study in cancer treatment. The molecular mechanisms underlying DTP cell survival are not fully understood; however, long non-coding RNAs (lncRNAs) have been proposed to play a crucial role. DTP cells from lung adenocarcinoma cell lines were obtained after single exposure to tyrosine kinase inhibitors (TKIs; erlotinib or osimertinib). After establishing DTP cells, RNA sequencing was performed to investigate the differential expression of the lncRNAs. Some lncRNAs and one mRNA were overexpressed in DTP cells. The clinical relevance of lncRNAs was evaluated in a cohort of patients with lung adenocarcinoma from The Cancer Genome Atlas (TCGA). RT–qPCR validated the overexpression of lncRNAs and mRNA in the residual DTP cells and LUAD biopsies. Knockdown of these lncRNAs increases the sensitivity of DTP cells to therapeutic drugs. This study provides an opportunity to investigate the involvement of lncRNAs in the genetic and epigenetic mechanisms that underlie intrinsic resistance. The identified lncRNAs and CD74 mRNA may serve as potential prognostic markers or therapeutic targets to improve the overall survival (OS) of patients with lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b47f1b958113e3cfa109f6bab13ad3bc4cd8c921" target='_blank'>
              Transcriptomic Analysis Reveals Early Alterations Associated with Intrinsic Resistance to Targeted Therapy in Lung Adenocarcinoma Cell Lines
              </a>
            </td>
          <td>
            Mario Perez-Medina, J. Lopez-Gonzalez, Jesus J. Benito-Lopez, Santiago Ávila-Ríos, Maribel Soto-Nava, M. Matías-Florentino, A. Méndez-Tenorio, M. Galicia-Velasco, R. Chavez-Dominguez, S. Meza-Toledo, D. Aguilar-Cázares
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Male sex chromosome aneuploidies are frequent genetic aberrations in humans characterized by additional Y or X chromosome complements. Jacob (JS) and Klinefelter syndromes (KS), carrying 47,XYY and 47,XXY chromosomes, respectively, share several clinical features, including sterility, hormonal deficits, neurocognitive delay, and skeletal-muscle defects. Despite the high incidence, a complete understanding of the molecular mechanisms underlying the clinical manifestations in sex aneuploid patients is still elusive. In this study, we generated and characterized the inaugural cohort of 47,XYY human induced pluripotent stem cells (iPSCs). We performed a comprehensive transcriptional analysis, including JS primary fibroblasts, iPSCs, and neural stem cells (NSCs) alongside KS cells. Our study revealed a transcriptional feedback mechanism tuning non-PAR X chromosome genes (NPX) homologs in Y supernumerary cells, a phenomenon not detected in X aneuploid iPSCs. By ectopically modulating the expression of selected NPY genes, we demonstrated a direct transcriptional link between the UTY-UTX gene pair. Furthermore, our analyses identified a shared transcriptomic signature between JS and KS, discernible already at the iPSC stage, with a notable enrichment for processes related to neurological functions. This transcriptomic convergence underscores potential commonalities in the molecular pathways underpinning the pathophysiology of male sex chromosome aneuploidies. Finally, through genome-wide DNA methylation profiling of JS iPSCs, we demonstrated that a supernumerary Y chromosome only minimally impacts the methylation status of 47,XYY cells at the pluripotent stage. Our work reveals critical transcriptional feedback mechanisms and shared gene expression signatures in male sex chromosome aneuploidies, paving the way for a better understanding of their common phenotypic features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4a1e46b6ba1f6258afb3db5dff0a58811a485a6" target='_blank'>
              An iPSC-based model of Jacob Syndrome reveals a DNA methylation-independent transcriptional dysregulation shared with X aneuploid cells
              </a>
            </td>
          <td>
            V. Astro, K. Cardona-Londoño, L. V. Cortés-Medina, R. Alghamdi, G. Ramírez-Calderón, F. Kefalas, J. Dilmé-Capó, S. Radío, A. Adamo
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Lung cancer patients treated with targeted therapies frequently respond well but invariably relapse due to the development of drug resistance. Drug resistance is in part mediated by a subset of cancer cells termed “drug-tolerant persisters” (DTPs), which enter a dormant, slow-cycling state that enables them to survive drug exposure. DTPs also exhibit stem cell-like characteristics, broad epigenetic reprogramming, altered metabolism, and a mutagenic phenotype mediated by adaptive mutability. While several studies have characterised the transcriptional changes that lead to the altered phenotypes exhibited in DTPs, these studies have focused predominantly on protein coding changes. As long non-coding RNAs (lncRNAs) are also implicated in the phenotypes altered in DTPs, it is likely that they play a role in the biology of drug tolerance. In this review, we outline how lncRNAs may contribute to the key characteristics of DTPs, their potential roles in tolerance to targeted therapies, and the emergence of genetic resistance in lung adenocarcinoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83141f0410de3ec7e33ac04641da48080cc097e5" target='_blank'>
              The Potential Links between lncRNAs and Drug Tolerance in Lung Adenocarcinoma
              </a>
            </td>
          <td>
            William J. H. Davis, Catherine J. Drummond, Sarah D. Diermeier, Glen Reid
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Despite significant advances in our understanding of genetic cancer susceptibility, known inherited cancer predisposition syndromes explain at most 20% of early-onset cancers. As early-onset cancer prevalence continues to increase, the need to assess previously inaccessible areas of the human genome, harnessing a trio or quad family-based architecture for variant filtration, may reveal further insights into cancer susceptibility. To assess a broader spectrum of variation than can be ascertained by multi-gene panel sequencing, or even whole genome sequencing with short reads, we employed long read whole genome sequencing using an Oxford Nanopore Technology (ONT) PromethION of 3 families containing an early-onset cancer proband using a trio or quad family architecture. Analysis included 2 early-onset colorectal cancer family trios and one quad consisting of two siblings with testicular cancer, all with unaffected parents. Structural variants (SVs), epigenetic profiles and single nucleotide variants (SNVs) were determined for each individual, and a filtering strategy was employed to refine and prioritize candidate variants based on the family architecture. The family architecture enabled us to focus on inapposite variants while filtering variants shared with the unaffected parents, significantly decreasing background variation that can hamper identification of potentially disease causing differences. Candidate de novo and compound heterozygous variants were identified in this way. Gene expression, in matched neoplastic and pre-neoplastic lesions, was assessed for one trio. Our study demonstrates the feasibility of a streamlined analysis of genomic variants from long read ONT whole genome sequencing and a way to prioritize key variants for further evaluation of pathogenicity, while revealing what may be missing from panel based analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579a5b24c1657cfe092825293758535eef92a0f6" target='_blank'>
              Exploring the genetic and epigenetic underpinnings of early-onset cancers: Variant prioritization for long read whole genome sequencing from family cancer pedigrees
              </a>
            </td>
          <td>
            M. Kramer, Sara Goodwin, R. Wappel, Matilde Borio, K. Offit, D. Feldman, Z. Stadler, W. R. McCombie
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="INTRODUCTION
Colon cancer, ranked as the fourth leading global cause of cancer death, exhibits a complex progression marked by genetic variations. Over the past decade, the utilization of diverse CRISPR systems has propelled accelerated research into colorectal cancer (CRC) treatment.


AREAS COVERED
CRISPR/Cas9, a key player in this research, identifies new oncogenes, tumor suppressor genes (TSGs), and drug-resistance genes. Additionally, it facilitates the construction of experimental models, conducts genome-wide library screening, and develops new therapeutic targets, especially for targeted knockout in vivo or molecular targeted drug delivery, contributing to personalized treatments and significantly enhancing the care of colon cancer patients. In this review, we provide insights into the mechanism of the CRISPR/Cas9 system, offering a comprehensive exploration of its applications in CRC, spanning screening, modeling, gene functions, diagnosis, and gene therapy. While acknowledging its transformative potential, the article  highlights the challenges and limitations of CRISPR systems.


EXPERT OPINION
The application of CRISPR/Cas9 in CRC research provides a promising avenue for personalized treatments. Its potential for identifying key genes and enabling experimental models and genome-wide screening enhances patient care. This review underscores the significance of CRISPR-Cas9 gene editing technology across basic research, diagnosis, and the treatment landscape of colon cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e832452c0de49427c039c9545ac3618de43c5948" target='_blank'>
              Recent advances in CRISPR-Cas systems for colorectal cancer research and therapeutics.
              </a>
            </td>
          <td>
            Saeideh Khorshid Sokhangouy, Farzaneh Alizadeh, Malihe Lotfi, Samaneh Sharif, Atefeh Ashouri, Yasamin Yoosefi, Saeed Bozorg Qomi, M. Abbaszadegan
          </td>
          <td>2024-08-12</td>
          <td>Expert review of molecular diagnostics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="The genetic integrity of pluripotent stem cells (PSC) is integral to their applications in research and therapy, but it is compromised by frequent development of copy number variations. Little is known about the basis of the variable genomic integrity among different PSC lines. Here we identify aneuploidies using RNA-seq and proteomics data from a panel of mouse embryonic stem cell (mESC) lines derived from 170 Diversity Outbred mice. We identified 62 lines with detectable aneuploid subpopulations and a subset of originally XX lines that lost one Chromosome X (XO). Strikingly, a much lower proportion of XX lines were aneuploid, compared to XY or XO lines. Two single-cell RNA-seq data sets demonstrated that aneuploid XY DO mESC also show lower Chromosome X gene expression and that XY mESC accumulate higher aneuploid proportions in culture than isogenic XX lines. Possible mechanisms for this protective effect of X chromosome dosage include our discovery that the lines with two active X Chromosomes have a higher expression of 2-cell-like state genes, and differential expression of X-linked tumor suppressor genes associated with DNA damage response. Highlights First genetic analysis of predisposition to aneuploidy in pluripotent stem cell cultures Chromosomal regions duplicated in aneuploid mouse embryonic stem cells are syntenic with regions overrepresented in human pluripotent stem cell lines bearing recurrent genetic abnormalities X-Chromosome dosage strongly influences susceptibility to aneuploidy in mouse embryonic stem cells and to a lesser degree in human pluripotent stem cells XX mouse embryonic stem cell lines show a higher proportion of cells in 2 cell-like state and higher expression of tumor suppressor genes associated with DNA damage response">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f853ae6e95a31cb9ad3712a21b0838a6601fe33" target='_blank'>
              Chromosome X Dosage Modulates Development of Aneuploidy in Genetically Diverse Mouse Embryonic Stem Cells
              </a>
            </td>
          <td>
            Alexander Stanton, Selcan Aydin, Daniel A. Skelly, Dylan Stavish, Kim Leonhard, Seth M. Taapken, Erik M. McIntire, Matthew Pankratz, A. Czechanski, T. Ludwig, Ted Choi, Steve P. Gygi, Ivana Barbaric, Steven C. Munger, L. Reinholdt, Martin F. Pera
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Using five complementary short- and long-read sequencing technologies, we phased and assembled >95% of each diploid human genome in a four-generation, 28-member family (CEPH 1463) allowing us to systematically assess de novo mutations (DNMs) and recombination. From this family, we estimate an average of 192 DNMs per generation, including 75.5 de novo single-nucleotide variants (SNVs), 7.4 non-tandem repeat indels, 79.6 de novo indels or structural variants (SVs) originating from tandem repeats, 7.7 centromeric de novo SVs and SNVs, and 12.4 de novo Y chromosome events per generation. STRs and VNTRs are the most mutable with 32 loci exhibiting recurrent mutation through the generations. We accurately assemble 288 centromeres and six Y chromosomes across the generations, documenting de novo SVs, and demonstrate that the DNM rate varies by an order of magnitude depending on repeat content, length, and sequence identity. We show a strong paternal bias (75-81%) for all forms of germline DNM, yet we estimate that 17% of de novo SNVs are postzygotic in origin with no paternal bias. We place all this variation in the context of a high-resolution recombination map (∼3.5 kbp breakpoint resolution). We observe a strong maternal recombination bias (1.36 maternal:paternal ratio) with a consistent reduction in the number of crossovers with increasing paternal (r=0.85) and maternal (r=0.65) age. However, we observe no correlation between meiotic crossover locations and de novo SVs, arguing against non-allelic homologous recombination as a predominant mechanism. The use of multiple orthogonal technologies, near-telomere-to-telomere phased genome assemblies, and a multi-generation family to assess transmission has created the most comprehensive, publicly available “truth set” of all classes of genomic variants. The resource can be used to test and benchmark new algorithms and technologies to understand the most fundamental processes underlying human genetic variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcb8df79fd9bfc629f05070b595811f041bfe33" target='_blank'>
              A familial, telomere-to-telomere reference for human de novo mutation and recombination from a four-generation pedigree
              </a>
            </td>
          <td>
            David Porubsky, Harriet Dashnow, T. Sasani, Glennis A. Logsdon, P. Hallast, Michelle D. Noyes, Z. Kronenberg, T. Mokveld, Nidhi Koundinya, Cillian Nolan, C. Steely, A. Guarracino, E. Dolzhenko, William T. Harvey, William J. Rowell, K. Grigorev, T. J. Nicholas, Keisuke K. Oshima, Jiadong Lin, Peter Ebert, W. Watkins, Tiffany Y. Leung, Vincent C. T. Hanlon, Sean Mcgee, Brent S. Pedersen, Michael E. Goldberg, Hannah C. Happ, Hyeonsoo Jeong, Katherine M. Munson, Kendra Hoekzema, Daniel D. Chan, Yanni Wang, Jordan Knuth, Gage H. Garcia, Cairbre Fanslow, Christine Lambert, Charles Lee, Josh Smith, Shawn Levy, C. Mason, Erik Garrison, Peter Lansdorp, D. Neklason, L. Jorde, Aaron R. Quinlan, M. Eberle, Evan E. Eichler
          </td>
          <td>2024-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8868db406f4d297083b3336bdacf7e70f944e3" target='_blank'>
              Characterize molecular signatures and establish a prognostic signature of gastric cancer by integrating single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Zhiwei Wang, Zhiyan Weng, Luping Lin, Xianyi Wu, Wenju Liu, Yong Zhuang, J. Jian, Changhua Zhuo
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/699986230662d4afba97aea6b14e6128a02adbf7" target='_blank'>
              Molecular mechanisms of DNA lesion and repair during antibody somatic hypermutation.
              </a>
            </td>
          <td>
            Qian Hao, Jinfeng Li, Leng-Siew Yeap
          </td>
          <td>2024-07-23</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Introduction Glioblastomas utilize malignant gene expression pathways to drive growth. Many of these gene pathways are not directly accessible with molecularly targeted pharmacological agents. Chromatin-modifying compounds can alter gene expression and target glioblastoma growth pathways. In this study, we utilize a systematic screen of chromatin-modifying compounds on a panel of patient-derived glioblastoma lines to identify promising compounds and their associated gene targets. Methods Five glioblastoma cell lines were subjected to a drug screen of 106 chromatin-modifying compounds representing 36 unique drug classes to determine the twelve most promising drug classes and the best candidate inhibitors in each class. These twelve drugs were then tested with a panel of twelve patient-derived gliomasphere lines to identify growth inhibition and corresponding gene expression patterns. Overlap analysis and weighted co-expression network analysis (WCGNA) were utilized to determine potential target genes and gene pathways. Results The initial drug screen identified twelve candidate pharmacologic agents for further testing. Drug sensitivity testing indicated an overall high degree of variability between gliomasphere lines. However, CPI203 was the most consistently effective compound, and the BET inhibitor class was the most consistently effective class of compounds across the gliomasphere panel. Correspondingly, most of the compounds tested had highly variable effects on gene expression between gliomasphere lines. CPI203 stood out as the only compound to induce a consistent effect on gene expression across different gliomasphere lines, specifically down-regulation of DNA-synthesis genes. Amongst the twelve tested cell lines, high expression of CDKN2A and CDKN2B distinguished more drug sensitive from more drug resistant lines. WCGNA identified two oncogenic gene modules (FBXO5 and MELK) that were effectively downregulated by CPI203 (FBXO5) and ML228 (FBXO5 and MELK). Conclusions The bromodomain inhibitor CPI203 induced relatively consistent effects on gene expression and growth across a variety of glioblastoma lines, specifically down-regulating genes associated with DNA replication. We propose that clinically effective BET inhibitors have the potential to induce consistent beneficial effects across a spectrum of glioblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1561fa27def3a17181508712df5ccd39c21a9f63" target='_blank'>
              CPI203, a BET inhibitor, down-regulates a consistent set of DNA synthesis genes across a wide array of glioblastoma lines
              </a>
            </td>
          <td>
            Matthew C. Garrett, Troy P Carnwath, Rebecca R Albano, Yonghua Zhuang, Catherine A Behrmann, Merissa Pemberton, Farah Barakat, Robert Lober, Mark Hoeprich, Anthony Paravati, Marilyn Reed, Hailey Spry, Daniel Woo, Eric O’Brien, Brett VanCauwenbergh, John Perentesis, Rani Nasser, Mario Medvedovic, D. Plas
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Transcription and splicing errors in cancer cells generate erroneous transcripts. Since erroneous transcripts are degraded by the nonsense-mediated mRNA decay (NMD) pathway, whether they are junk or could be functional has been overlooked and understudied. We addressed this question by first performing a pan-cancer analysis and identified substantial erroneous noncoding transcripts (ENT) in cancers. Given that RNA/DNA binding domains (DBD) were predicted in ENTs, we deleted predicted DBDs in multiple ENTs in multiple cell lines, with RNA-sequencing and cell experiments before and after DBD deletion. DBD deletion caused significantly changed expression of ENTs’ target genes (whose promoter regions contain ENTs’ DNA binding sites, DBS) and changed cell migration and proliferation ability, indicating that many ENTs can transcriptionally regulate genes. Tightly coupled data analysis and experiments reveal that ENTs’ functions are highly cancer- and cellular-context specific, making ENTs a new class of safe and specific targets for noncoding RNA-based cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e45af8d1c765a560ff555893aa2ed200e47f3c" target='_blank'>
              Many erroneous noncoding transcripts in cancer cells can highly specifically regulate cancer-related genes and pathways
              </a>
            </td>
          <td>
            Sha He, Wei Xiong, Jianping Huo, Jie Lin, Jianmin Li, Hao Zhu
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Drug resistance is still an imminent issue for melanoma patients even after the use of BRAFi/MEKi combination therapy in clinic. Tumor heterogeneity and heterogeneity in treatment responses make it difficult to find consensus genes and pathways in resistance to therapy. This study used an objective method to analyze published gene expression data from pre-treated tumors and drug-resistant tumors, and identified possible targets and markers for resistant tumors, which centered on the PLXNC1 gene, which promotes a pro-inflammatory tumor microenvironment. Abstract Melanoma tumors exhibit a wide range of heterogeneity in genomics even with shared mutations in the MAPK pathway, including BRAF mutations. Consistently, adaptive drug resistance to BRAF inhibitors and/or BRAF plus MEK inhibitors also exhibits a wide range of heterogeneous responses, which poses an obstacle for discovering common genes and pathways that can be used in clinic for overcoming drug resistance. This study objectively analyzed two sets of previously published tumor genomics data comparing pre-treated melanoma tumors and BRAFi- and/or MEKi-resistant tumors. Heterogeneity in response to BRAFi and BRAFi/MEKi was evident because the pre-treated tumors and resistant tumors did not exhibit a tendency of clustering together. Differentially expressed gene (DEG) analysis revealed eight genes and two related enriched signature gene sets (matrisome and matrisome-associated signature gene sets) shared by both sets of data. The matrisome was closely related to the tumor microenvironment and immune response, and five out of the eight shared genes were also related to immune response. The PLXNC1 gene links the shared gene set and the enriched signature gene sets as it presented in all analysis results. As the PLXNC1 gene was up-regulated in the resistant tumors, we validated the up-regulation of this gene in a laboratory using vemurafenib-resistant cell lines. Given its role in promoting inflammation, this study suggests that resistant tumors exhibit an inflammatory tumor microenvironment. The involvement of the matrisome and the specific set of immune genes identified in this study may provide new opportunities for developing future therapeutic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fabe8a302d27af7d60acbb39b872cfcbb898453" target='_blank'>
              Unsupervised Analysis Reveals the Involvement of Key Immune Response Genes and the Matrisome in Resistance to BRAF and MEK Inhibitors in Melanoma
              </a>
            </td>
          <td>
            F. Liu-Smith, Jianjian Lin
          </td>
          <td>2024-06-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="T cell acute lymphoblastic leukemia (T-ALL) is an aggressive leukemia predominantly affecting adolescents and young adults. T-ALL relapses are characterized by chemotherapy resistance, cellular heterogeneity and dismal outcome. To gain a deeper understanding of the cellular heterogeneity and mechanisms driving relapse, we conducted single-cell full-length RNA sequencing of 13 matched pediatric T-ALL patient-derived xenografts (PDX) samples obtained at initial diagnosis and relapse, generating the to date most comprehensive longitudinal single cell study in paired T-ALL samples, along with 5 non-relapsing PDX samples collected at initial diagnosis. This dataset identifies considerable transcriptomic diversity among individual T-ALL cell populations. Notably however, 11 of the 18 patients exhibit a small T-ALL cell subpopulation with a shared set of gene regulatory networks characterized by a common set of active regulons, expression patterns and splice isoforms. This profile involves the upregulation of a stem-like cell signature with enrichment of cell adhesion, and inhibition of the cell cycle and metabolic activity. Comprehensive investigations of these networks identify transcripts enforcing drug resistance through NF-κB expression and TGF-β signaling and of anti-apoptotic T cell signaling. Longitudinal monitoring of these stem-like cells demonstrates this subpopulation to account for only a small proportion of leukemia cells initially with a substantial expansion at relapse suggesting resistance to first line therapy. Chemotherapy resistance is functionally corroborated through in vitro and in vivo drug testing. We thus report the discovery of inherently treatment-resistant stem-like T-ALL cell populations underscoring the potential for devising future therapeutic strategies aimed at targeting stemness pathways in pediatric T-ALL. Key Points Single-cell full-length RNA sequencing reveals dormant, treatment-resistant cells expanding upon T cell acute lymphoblastic leukemia relapse. Stem-like T-ALL cells exhibit a shared gene regulatory network conferring chemotherapy resistance and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b88dbfdc32eaada9aa463b5fbbffd8e6808956ec" target='_blank'>
              Role of Stem-Like Cells in Chemotherapy Resistance and Relapse in pediatric T Cell Acute Lymphoblastic Leukemia
              </a>
            </td>
          <td>
            Julia Costea, Kerstin K. Rauwolf, Pietro Zafferani, T. Rausch, A. Mathioudaki, Judith B. Zaugg, M. Schrappe, C. Eckert, G. Escherich, Jean-Pierre Bourquin, B. Bornhauser, Andreas E. Kulozik, J. Korbel
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135134a62bac76cb46dae2f04090c849f2c01172" target='_blank'>
              Targeting chromosomal instability in patients with cancer.
              </a>
            </td>
          <td>
            Duaa H Al-Rawi, Emanuele Lettera, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-11</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="CRISPR gene editing strategies are shaping cell therapies through precise and tunable control over gene expression. However, achieving reliable therapeutic effects with improved safety and efficacy requires informed target gene selection. This depends on a thorough understanding of the involvement of target genes in gene regulatory networks (GRNs) that regulate cell phenotype and function. Machine learning models have been previously used for GRN reconstruction using RNA- seq data, but current techniques are limited to single cell types and focus mainly on transcription factors. This restriction overlooks many potential CRISPR target genes, such as those encoding extracellular matrix components, growth factors, and signaling molecules, thus limiting the applicability of these models for CRISPR strategies. To address these limitations, we have developed CRISPR-GEM, a multi-layer perceptron (MLP)-based synthetic GRN constructed to accurately predict the downstream effects of CRISPR gene editing. First, input and output nodes are identified as differentially expressed genes between defined experimental and target cell/tissue types respectively. Then, MLP training learns regulatory relationships in a black-box approach allowing accurate prediction of output gene expression using only input gene expression. Finally, CRISPR-mimetic perturbations are made to each input gene individually and the resulting model predictions are compared to those for the target group to score and assess each input gene as a CRISPR candidate. The top scoring genes provided by CRISPR-GEM therefore best modulate experimental group GRNs to motivate transcriptomic shifts towards a target group phenotype. This machine learning model is the first of its kind for predicting optimal CRISPR target genes and serves as a powerful tool for enhanced CRISPR strategies across a range of cell therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/486423f8c90770082bb7f4dc4e4e7334da919173" target='_blank'>
              CRISPR-GEM: A Novel Machine Learning Model for CRISPR Genetic Target Discovery and Evaluation
              </a>
            </td>
          <td>
            Josh P. Graham, Yu Zhang, Lifang He, Tomas Gonzalez-Fernandez
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Increased access to high-throughput DNA sequencing platforms has transformed the diagnostic landscape of pediatric malignancies by identifying and integrating actionable genomic or transcriptional features that refine diagnosis, classification, and treatment. Yet less than 10% of treated patients show a positive response and translating precision oncology data into feasible and effective therapies for hard-to-cure childhood, adolescent, and young adult malignancies remains a significant challenge. Combining the identification of therapeutic targets at the protein and pathway levels with demonstration of treatment response in personalized models holds great promise. Here we present the case for combining proteomics with patient-derived xenograft (PDX) models to identify personalized treatment options that were not apparent at genomic and transcriptomic levels. Proteome analysis with immunohistochemistry (IHC) validation of formalin-fixed paraffin-embedded sections from an adolescent with primary and metastatic spindle epithelial tumor with thymus-like elements (SETTLE) was completed within two weeks of biopsy. The results identified an elevated protein level of SHMT2 as a possible target for therapy with the commercially available anti-depressant sertraline. Within 2 months and ahead of a molecular tumor board, we confirmed a positive drug response in a personalized chick chorioallantoic membrane (CAM) model of the SETTLE tumor (CAM-PDX). Following the failure of cytotoxic chemotherapy and second-line therapy, a treatment of sertraline was initiated for the patient. After 3 months of sertraline treatment the patient showed decreased tumor growth rates, albeit with clinically progressive disease. Significance: Overall, we demonstrate that proteomics and fast-track personalized xenograft models can provide supportive pre-clinical data in a clinically meaningful timeframe to support medical decision-making and impact the clinical practice. By this we show that proteome-guided and functional precision oncology are feasible and valuable complements to the current genome- driven precision oncology practices.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6528441c9ef77e1a91eff560c6531d2a938e5fc1" target='_blank'>
              Proteomics and personalized patient-derived xenograft models identify treatment opportunities for a progressive malignancy within a clinically actionable timeframe and change care
              </a>
            </td>
          <td>
            G. D. Barnabas, T. A. Bhat, V. Goebeler, P. Leclair, N. Azzam, N. Melong, C. Anderson, A. Gom, S. An, E. Ergin, Y. Shen, A. Mungall, K. L. Mungall, C. A. Maxwell, G. S. D. Reid, M. Hirst, S. Jones, J. A. Chan, D. L. Senger, J. N. Berman, S. Parker, J. Bush, C. Strahlendorf, R. Deyell, C. J. Lim, P. F. Lange, Profyle Program
          </td>
          <td>2024-07-08</td>
          <td>None</td>
          <td>0</td>
          <td>98</td>
        </tr>

        <tr id="Single nucleotide variants (SNVs) contribute to cancer by altering the coding and the non-coding regions of the genome. Connecting SNVs to transcriptomic and epigenetic changes at the single-cell level remains challenging. To enable studies of rare cell populations harboring specific point mutations, we developed STAR-FACS, Specific-To-Allele PCR-FACS, to sort cells based on genomic allele alterations. We show that STAR-FACS can separate cells based on TERT promoter mutation status and is compatible with bulk and single-cell transcriptomic and epigenetic profiling. We demonstrate that glioblastoma cell lines derived from the same tumor but harboring distinct TERT promoter SNVs have different transcriptional programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f2ef09f49e055964d96a3768d156e4cf968931" target='_blank'>
              Cell sorting based on single nucleotide variation enables characterization of mutation-dependent transcriptome and chromatin states
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, Oszkar Szentirmai, M. Janiszewska
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Methods to engineer the genomes of human cells for therapeutic intervention continue to advance at a remarkable pace. Chimeric antigen receptor-engineered T lymphocytes have pioneered the way for these therapies, initially beginning with insertions of chimeric antigen receptor transgenes into T-cell genomes using classical gene therapy vectors. The broad use of clustered regularly interspaced short palindromic repeats (CRISPR)-based technologies to edit endogenous genes has now opened the door to a new era of precision medicine. To add complexity, many engineered cellular therapies under development integrate gene therapy with genome editing to introduce novel biological functions and enhance therapeutic efficacy. Here, we review the current state of scientific, translational, and regulatory oversight of gene-edited cell products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2d1ab3d9838377779d6ccf7a7d2e82b715e992" target='_blank'>
              Regulatory Considerations for Genome-Edited T-cell Therapies.
              </a>
            </td>
          <td>
            J. Jadlowsky, Ju-Fang Chang, DH Spencer, John M Warrington, Bruce L. Levine, Carl H. June, J. Fraietta, Nathan Singh
          </td>
          <td>2024-07-17</td>
          <td>Cancer immunology research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="BACKGROUND
Metastatic colorectal cancer (mCRC) is increasingly characterized by myriad genomic alterations beyond the well-known factors such as RAS, BRAF, and microsatellite instability (MSI). Novel genomic changes, including ERBB2 amplifications, mutations, and gene fusions, are now recognized as potential targets for precision therapy. This study aims to explore the genomic landscape of a Chinese cohort with mCRC to identify potentially targetable genetic alterations for personalized treatment strategies.


METHODS
A total of 500 mCRC patients in China were enrolled, based on which genomic profiling was performed using capture-based targeted sequencing across a panel of 520 genes on tumor tissues to identify prevalent genomic alterations. The mutations were analyzed by optimized proprietary algorithms. MSI and mismatch repair deficiency status were analyzed using the read-count-distribution approach. Besides, the overall survival (OS) related to these molecular changes was estimated.


RESULTS
The cohort's genomic profiling revealed TP53 mutations in 78%, APC in 60%, and KRAS in 47% of the patients. MSI-High status was confirmed in 5.8% of cases via a next-generation sequencing (NGS)-based algorithm. ERBB2/HER2 amplifications were found in 12% (60/500) of patients, with potential therapeutic implications for those without concurrent KRAS mutations. A subset of patients (1.2%; 6/500) showed fusions and DNA damage response (DDR) gene mutations (except TP53) that could be targeted therapeutically. The KRAS (G12C) variant was detected in 14 patients (2.8%), and 61 (12.2%) had a BRAF V600E mutation. Notably, survival analysis showed no significant differences in OS between KRAS mutant loci and NRAS mutations (p = 0.436). However, BRAF V600E mutations were associated with a poorer prognosis than BRAF wild-type and non-V600E mutations (16.3 months vs. 29.5 and 31.1 months, respectively; p < 0.001).


CONCLUSIONS
This study validates the feasibility of using NGS to detect prognostic and therapeutically actionable genetic variants in Chinese mCRC patients, contributing to understanding the genomic variation within this population and highlighting the potential for personalized medicine in managing mCRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d75b3be37f6d3fb50ba2d4fcbf63055e7d0a5a8b" target='_blank'>
              Genomic Alterations and Clinical Characterization in Chinese Patients with Metastatic Colorectal Cancer.
              </a>
            </td>
          <td>
            Xu-hui Zhang, Jie-Qiong Zhou, Qing Wei, Jian Li, Ting Xu, Chun-mei Bai, Jian-Feng Zhou, Xi-cheng Wang
          </td>
          <td>2024-07-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cell-cell interactions, which allow cells to communicate with each other through molecules in their microenvironment, are critical for the growth, health, and functions of cells. Previous studies show that drug-resistant cells can interact with drug-sensitive cells to elevate their drug resistance level, which is partially responsible for cancer recurrence. Studying protein targets and pathways involved in cell-cell communication provides essential information for fundamental cell biology studies and therapeutics of human diseases. In the current studies, we performed direct coculture and indirect coculture of drug-resistant and drug-sensitive cell lines, aiming to investigate intracellular proteins responsible for cell communication. Comparative studies were carried out using monoculture cells. Shotgun bottom-up proteomics results indicate that the P53 signaling pathway has a strong association with drug resistance mechanisms, and multiple TP53-related proteins were upregulated in both direct and indirect coculture systems. In addition, cell-cell communication pathways, including the phagosome and the HIF-signaling pathway, contribute to both direct and indirect coculture systems. Consequently, AK3 and H3-3A proteins were identified as potential targets for cell-cell interactions that are relevant to drug resistance mechanisms. We propose that the P53 signaling pathway, in which mitochondrial proteins play an important role, is responsible for inducing drug resistance through communication between drug-resistant and drug-sensitive cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cc9726c157261dbc51a02e3ad073aa8b2f6d16b" target='_blank'>
              Proteomics Analysis of Interactions between Drug-Resistant and Drug-Sensitive Cancer Cells: Comparative Studies of Monoculture and Coculture Cell Systems.
              </a>
            </td>
          <td>
            Zongkai Peng, Nagib Ahsan, Zhibo Yang
          </td>
          <td>2024-06-22</td>
          <td>Journal of proteome research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c331995b31749051950071b62fbbb790286c2472" target='_blank'>
              Metabolic reprogramming of poly(morpho)nuclear giant cells determines glioblastoma recovery from doxorubicin-induced stress
              </a>
            </td>
          <td>
            Maciej Pudełek, D. Ryszawy, K. Piwowarczyk, S. Lasota, Z. Madeja, S. Kędracka-Krok, Jarosław Czyż
          </td>
          <td>2024-08-12</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Resistance to cancer therapy is driven by both cell-intrinsic and microenvironmental factors. Previous work has revealed that multiple resistant cell fates emerge in melanoma following treatment with targeted therapy and that, in vitro, these resistant fates are determined by the transcriptional state of individual cells prior to exposure to treatment. What remains unclear is whether these resistant fates are shared across different genetic backgrounds and how, if at all, these resistant fates interact with the tumor microenvironment. Through spatial transcriptomics and single-cell RNA sequencing, we uncovered distinct resistance programs in melanoma cells shaped by both intrinsic cellular states and the tumor microenvironment. Consensus non-negative matrix factorization revealed shared intrinsic resistance programs across different cell lines, highlighting the presence of universal and unique resistance pathways. In patient samples, we demonstrated that these resistance programs coexist within individual tumors and associate with diverse immune signatures, suggesting that the tumor microenvironment and distribution of resistant fates are closely connected. Single-cell resolution spatial transcriptomics in xenograft models revealed both intrinsically determined and extrinsically influenced resistant fates. Overall, this work demonstrates that each therapy resistant fate coexists with a distinct immune microenvironment in tumors and that, in vivo, tissue features, such as regions of necrosis, can influence which resistant fate is adopted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5979abc00b349ddf0161079fda7fee453bd0ce0a" target='_blank'>
              Spatial transcriptomics reveals influence of microenvironment on intrinsic fates in melanoma therapy resistance
              </a>
            </td>
          <td>
            Ryan H. Boe, Catherine G. Triandafillou, R. Lazcano, J. Wargo, Arjun Raj
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea3f7e6f9fd3216c09d0776dc7d2e0f2d441e9d9" target='_blank'>
              Insights into immune microenvironment and therapeutic targeting in androgen-associated prostate cancer subtypes
              </a>
            </td>
          <td>
            Liang Huang, Yu Xie, Shusuan Jiang, Tao Dai, Zhenzhou Xu, Hong Shan
          </td>
          <td>2024-08-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The intrinsic oncogenic mechanisms and properties of the tumor microenvironment (TME) have been extensively investigated. Primary features of the TME include metabolic reprogramming, hypoxia, chronic inflammation, and tumor immunosuppression. Previous studies suggest that senescence-associated secretory phenotypes that mediate intercellular information exchange play a role in the dynamic evolution of the TME. Specifically, hypoxic adaptation, metabolic dysregulation, and phenotypic shifts in immune cells regulated by cellular senescence synergistically contribute to the development of an immunosuppressive microenvironment and chronic inflammation, thereby promoting the progression of tumor events. This review provides a comprehensive summary of the processes by which cellular senescence regulates the dynamic evolution of the tumor-adapted TME, with focus on the complex mechanisms underlying the relationship between senescence and changes in the biological functions of tumor cells. The available findings suggest that components of the TME collectively contribute to the progression of tumor events. The potential applications and challenges of targeted cellular senescence-based and combination therapies in clinical settings are further discussed within the context of advancing cellular senescence-related research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f69d28c4893741d69a2e81f7df756fd68f976626" target='_blank'>
              Cellular senescence and metabolic reprogramming: Unraveling the intricate crosstalk in the immunosuppressive tumor microenvironment.
              </a>
            </td>
          <td>
            Fusheng Zhang, Junchen Guo, Shengmiao Yu, Youwei Zheng, Meiqi Duan, Liang Zhao, Yihan Wang, Zhi Yang, Xiaofeng Jiang
          </td>
          <td>2024-07-12</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genome merging is a common phenomenon in many organisms, causing a wide range of consequences on phenotype, adaptation, and gene expression, among other effects, yet its broader implications are not well understood. Two consequences of genome merging on gene expression remain poorly understood: dosage effects and evolution of expression. In this study, we employed Chlamydomonas reinhardtii as a model to investigate the effects of asymmetric genome merging by crossing a diploid with a haploid strain to create a novel triploid line. Five independent clonal lineages derived from this triploid line were evolved for 425 asexual generations in a laboratory natural selection (LNS) experiment. Utilizing fitness assays, qPCR, and RNA-Seq, we assessed the immediate consequences of genome merging and subsequent evolution over time. Our findings reveal substantial alterations in gene expression, protein homeostasis (proteostasis) and cytonuclear stoichiometry. Notably, gene expression exhibited expression level dominance and transgressivity (i.e., expression level higher or lower than either parent). Ongoing expression level dominance and a pattern of “functional dominance” from the haploid parent was observed, alongside remarkable stability in expression patterns across generations. Despite major nucleo-cytoplasmic disruptions, enhanced fitness was detected in the triploid strain. By comparing gene expression across generations, our results indicate that proteostasis restoration is a critical component of rapid adaptation following genome merging in Chlamydomonas reinhardtii and possibly other systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15e55d2c045a09190880662b2365054897839f5" target='_blank'>
              Asymmetric genome merging leads to gene expression novelty through nucleo-cytoplasmic disruptions and transcriptomic shock in Chlamydomonas triploids
              </a>
            </td>
          <td>
            Lucas Prost-Boxoen, Quinten Bafort, Antoine Van de Vloet, Fabrício Almeida-Silva, Yunn Thet Paing, G. Casteleyn, S. D’hondt, O. De Clerck, Yves Van de Peer
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technique has enabled detailed analysis of gene expression at the single cell level, enhancing the understanding of subtle mechanisms that underly pathologies and drug resistance. To derive such biological meaning from sequencing data in oncology, some critical processing must be performed, including identification of the tumor cells by markers and algorithms that infer copy number variations (CNVs). We compared the performance of sciCNV, InferCNV, CopyKAT and SCEVAN tools that identify tumor cells by inferring CNVs from scRNA-seq data. Sequencing data from Pancreatic Ductal Adenocarcinoma (PDAC) patients, adjacent and healthy tissues were analyzed, and the predicted tumor cells were compared to those identified by well-assessed PDAC markers. Results from InferCNV, CopyKAT and SCEVAN overlapped by less than 30% with InferCNV showing the highest sensitivity (0.72) and SCEVAN the highest specificity (0.75). We show that the predictions are highly dependent on the sample and the software used, and that they return so many false positives hence are of little use in verifying or filtering predictions made via tumor biomarkers. We highlight how critical this processing can be, warn against the blind use of these software and point out the great need for more reliable algorithms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f15f4004aa5098686401484d91f98cd52ad6608" target='_blank'>
              A Comparison of Tools That Identify Tumor Cells by Inferring Copy Number Variations from Single-Cell Experiments in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Daisy J. A. Oketch, M. Giulietti, Francesco Piva
          </td>
          <td>2024-08-05</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/435232aea775847959662a12da4da5a9a69c73c7" target='_blank'>
              A whole-genome CRISPR screen identifies the spindle accessory checkpoint as a locus of nab-paclitaxel resistance in a pancreatic cancer cell line
              </a>
            </td>
          <td>
            Priya Mondal, George Alyateem, Allison V. Mitchell, Michael M. Gottesman
          </td>
          <td>2024-07-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Acquired chemoresistance remains a significant challenge in the clinics as most of the treated cancers eventually emerge as hard-to-treat phenotypes. Therefore, identifying chemoresistance targets is highly warranted to manage the disease better. In this study, we employed a label-free LC-MS/MS-based quantitative proteomics analysis to identify potential targets and signaling pathways underlying acquired chemoresistance in a sub-cell line (A549DR) derived from the parental lung adenocarcinoma cells (A549) treated with gradually increasing doses of doxorubicin (DOX). Our proteomics analysis identified 146 upregulated and 129 downregulated targets in A549DR cells. The KEGG pathway and Gene ontology (GO) analysis of differentially expressed upregulated and downregulated proteins showed that most abundant upregulated pathways were related to metabolic pathways, cellular senescence, cell cycle, and p53 signaling. Meanwhile, the downregulated pathways were related to spliceosome, nucleotide metabolism, DNA replication, nucleotide excision repair, and nuclear-cytoplasmic transport. Further, STRING analysis of upregulated biological processes showed a protein-protein interaction (PPI) between CDK1, AKT2, SRC, STAT1, HDAC1, FDXR, FDX1, NPC1, ALDH2, GPx1, CDK4, and B2M, proteins. The identified proteins in this study might be the potential therapeutic targets for mitigating DOX resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/974ec76bceeb820f708d654bf77904bfd435f4c2" target='_blank'>
              Identification of chemoresistance targets in doxorubicin-resistant lung adenocarcinoma cells using LC-MS/MS-based proteomics.
              </a>
            </td>
          <td>
            Nuzhat Bano, K. Kainat, Mohammad Imran Ansari, Anjali Pal, Sana Sarkar, Pradeep Kumar Sharma
          </td>
          <td>2024-08-05</td>
          <td>Journal of chemotherapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671f339111b7b478e20a7b9554c095e50b64b22b" target='_blank'>
              Cellular and molecular events in colorectal cancer: biological mechanisms, cell death pathways, drug resistance and signalling network interactions
              </a>
            </td>
          <td>
            Lei Yan, Jia Shi, Jiazuo Zhu
          </td>
          <td>2024-07-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Currently, cancer is still a leading cause of human death globally. Tumor deterioration comprises multiple events including metastasis, therapeutic resistance and immune evasion, all of which are tightly related to the phenotypic plasticity especially epithelial–mesenchymal plasticity (EMP). Tumor cells with EMP are manifest in three states as epithelial–mesenchymal transition (EMT), partial EMT, and mesenchymal–epithelial transition, which orchestrate the phenotypic switch and heterogeneity of tumor cells via transcriptional regulation and a series of signaling pathways, including transforming growth factor‐β, Wnt/β‐catenin, and Notch. However, due to the complicated nature of EMP, the diverse process of EMP is still not fully understood. In this review, we systematically conclude the biological background, regulating mechanisms of EMP as well as the role of EMP in therapy response. We also summarize a range of small molecule inhibitors, immune‐related therapeutic approaches, and combination therapies that have been developed to target EMP for the outstanding role of EMP‐driven tumor deterioration. Additionally, we explore the potential technique for EMP‐based tumor mechanistic investigation and therapeutic research, which may burst vigorous prospects. Overall, we elucidate the multifaceted aspects of EMP in tumor progression and suggest a promising direction of cancer treatment based on targeting EMP.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32fcd891c5d88b251baf8fc511031710e0fc1629" target='_blank'>
              Epithelial–mesenchymal plasticity in cancer: signaling pathways and therapeutic targets
              </a>
            </td>
          <td>
            Xiangpeng Wang, Xiao-yong Xue, Mingshi Pang, Liuchunyang Yu, Jinxiu Qian, Xiaoyu Li, Meng Tian, Aiping Lyu, Cheng Lu, Yuanyan Liu
          </td>
          <td>2024-08-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Abstract Aneuploidy, or a change in the number of whole chromosomes or chromosome arms, is a near-universal feature of cancer. Chromosomes affected by aneuploidy are not random, with observed cancer-specific and tissue-specific patterns. Recent advances in genome engineering methods have allowed the creation of models with targeted aneuploidy events. These models can be used to uncover the downstream effects of individual aneuploidies on cancer phenotypes including proliferation, apoptosis, metabolism, and immune signaling. Here, we review the current state of research into the patterns of aneuploidy in cancer and their impact on signaling pathways and biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53976b63387a7558e4fe16be8478ae589398c516" target='_blank'>
              Patterns of Aneuploidy and Signaling Consequences in Cancer
              </a>
            </td>
          <td>
            Nadja Zhakula-Kostadinova, Alison M Taylor
          </td>
          <td>2024-06-26</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background In the absence of targeted mutations and immune checkpoints, platinum-based chemotherapy remains a gold standard agent in the treatment of patients with lung squamous cell carcinoma (LUSC). However, cisplatin resistance greatly limits its therapeutic efficacy and presents challenges in the treatment of lung cancer patients. Therefore, the potential clinical needs for this research focus on identifying novel molecular signatures to further elucidate the underlying mechanisms of cisplatin resistance in LUSC. A growing body of evidence indicates that alternative splicing (AS) events significantly influence the tumor progression and survival of patients with LUSC. However, there are few systematic analyses of AS reported in LUSC. This study aims to explore the role of messenger RNA (mRNA), microRNA (miRNA), and AS in predicting prognosis in patients with cisplatin-resistant LUSC and provide potential therapeutic targets and drugs. Methods Gene expression and miRNA expression, using RNA sequencing (RNA-seq), and SpliceSeq data were downloaded from The Cancer Genome Atlas (TCGA) database. The least absolute shrinkage and selection operator (LASSO) Cox regression analysis were used to construct predictive models. Kaplan-Meier survival analyses were used to evaluate patients’ prognosis. Single-sample gene set enrichment analysis (ssGSEA) conducted via the R package “GSEAbase” was used to evaluate the immune-related characteristics. Immunohistochemistry was used to examine protein expression. The Connectivity Map (CMap) database was used to screen for potential drugs. The 3-(4,5)-dimethylthiahiazo (-z-y1)-3,5-di-phenytetrazoliumromide (MTT) assay was used to determine and calculate the half-maximal inhibitory concentration (IC50) of the drugs, sulforaphane and parthenolide. Results In this study, bioinformatics were used to identify mRNAs, miRNAs, and AS events related to response to cisplatin and to establish an integrated prognostic signature for 70 patients with LUSC and cisplatin resistance. The prognostic signature served as an independent prognostic factor with high accuracy [hazard ratio (HR) =2.346, 95% confidence interval (CI): 1.568–3.510; P<0.001], yielding an area under the curve (AUC) of 0.825, 0.829, and 0.877 for 1-, 3-, and 5-year survival, respectively. It also demonstrated high predictive performance in this cohort of patients with LUSC, with an AUC of 0.734, 0.767, and 0.776 for 1-, 3-, and 5-year survival, respectively. This integrated signature was also found to be an independent indicator among conventional clinical features (HR =2.288, 95% CI: 1.547–3.383; P<0.001). In addition, we analyzed the correlation of the signature with immune infiltration and identified several small-molecule drugs that had the potential to improve the survival of patients with LUSC. Conclusions This study provides a framework for the mRNA-, miRNA-, and AS-based evaluation of cisplatin response and several potential therapeutic drugs for targeting cisplatin resistance in LUSC. These findings may serve as a theoretical basis for the clinical alleviation of cisplatin resistance and thus help to improve treatment responses to chemotherapy in patients with LUSC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac9e4be62d5ffa939a89b32b21f09c073e553ca1" target='_blank'>
              Identification of a prognostic gene signature in patients with cisplatin resistant squamous cell lung cancer
              </a>
            </td>
          <td>
            Yi Mu, Yinan Dong, Mingyang Zheng, Martin P. Barr, G. Roviello, Zhihuang Hu, Jia Liu
          </td>
          <td>2024-07-01</td>
          <td>Journal of Thoracic Disease</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2397c5b9b7fc70e44360b5033c3acbbcc326aa19" target='_blank'>
              Identification of exosomal microRNAs and related hub genes associated with imatinib resistance in chronic myeloid leukemia.
              </a>
            </td>
          <td>
            A. Karabay, T. Ozkan, Aynur Karadag Gurel, A. Koç, Y. Hekmatshoar, A. Sunguroglu, F. Aktan, Zeliha Buyukbingöl
          </td>
          <td>2024-06-25</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Melanoma being one of the most common and deadliest skin cancers, has been rising since the past decade. Patients at advanced stages of the disease have very poor prognoses, as opposed to at the earlier stages. Nowadays the standard-of-care of advanced melanoma is resection followed by immune checkpoint inhibition based immunotherapy. However, a substantial proportion of patients either do not respond or develop resistances. This underscores a need for novel approaches and therapeutic targets as well as a better understanding of the mechanisms of melanoma pathogenesis. Long non-coding RNAs (lncRNAs) comprise a poorly characterized class of functional players and promising targets in promoting malignancy. Certain lncRNAs have been identified to play integral roles in melanoma progression and drug resistances, however systematic screens to uncover novel functional lncRNAs are scarce. Here, we profile differentially expressed lncRNAs in patient derived short-term metastatic cultures and BRAF-MEK-inhibition resistant cells. We conduct a focused growth-related CRISPR-inhibition screen of overexpressed lncRNAs, validate and functionally characterize lncRNA hits with respect to cellular growth, invasive capacities and apoptosis in vitro as well as the transcriptomic impact of our lead candidate the novel lncRNA XLOC_030781. In sum, we extend the current knowledge of ncRNAs and their potential relevance on melanoma. Significance Previously considered as transcriptional noise, lncRNAs have emerged as novel players in regulating many cellular aspects in health and disease including melanoma. However, the number and as well as the extent of functional significance of most lncRNAs remains elusive. We provide a comprehensive strategy to identify functionally relevant lncRNAs in melanoma by combining expression profiling with CRISPR-inhibition growths screens lowering the experimental effort. We also provide a larger resource of differentially expressed lncRNAs with potential implications in melanoma growth and invasion. Our results broaden the characterized of lncRNAs as potential targets for future therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a768f16ebbf2c3ec424c154b1baf19ad0e931615" target='_blank'>
              CRISPR-inhibition screen for lncRNAs linked to melanoma growth and metastasis
              </a>
            </td>
          <td>
            Stavroula Petroulia, K. Hockemeyer, Shashank Tiwari, Pietro Berico, Sama Shamloo, Seyedeh Elnaz Banijamali, Eleazar Vega-Saenz de Miera, Yixiao Gong, Palaniraja Thandapani, Eric Wang, Michael Schulz, A. Tsirigos, Iman Osman, Ioannis D. Aifantis, Jochen Imig
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The genome of a cell is continuously battered by a plethora of exogenous and endogenous processes that can lead to damaged DNA. Repair mechanisms correct this damage most of the time, but failure to do so leaves mutations. Mutations do not occur in random manner, but rather typically follow a more or less specific pattern due to known or imputed mutational processes. Mutational signature analysis is the process by which the predominant mutational process can be inferred for a cancer and can be used in several contexts to study both the genesis of cancer and its response to therapy. Recent pan-cancer genomic efforts such as "The Cancer Genome Atlas" have identified numerous mutational signatures that can be categorized into single base substitutions, doublet base substitutions, or small insertions/deletions. Understanding these mutational signatures as they occur in non-small lung cancer could improve efforts at prevention, predict treatment response to personalized treatments, and guide the development of therapies targeting tumor evolution. For non-small cell lung cancer, several mutational signatures have been identified that correlate with exposures such as tobacco smoking and radon and can also reflect endogenous processes such as aging, APOBEC activity, and loss of mismatch repair. Herein, we provide an overview of the current knowledge of mutational signatures in non-small lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37777d22dfc5a2bd99dc42f300aad4d0a01e922f" target='_blank'>
              Topography of mutational signatures in non-small cell lung cancer: emerging concepts, clinical applications, and limitations.
              </a>
            </td>
          <td>
            Ritujith Jayakrishnan, David J. Kwiatkowski, Michal G Rose, Amin H. Nassar
          </td>
          <td>2024-06-22</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The fitness effects of new mutations determine key properties of evolutionary processes. Beneficial mutations drive evolution, yet selection is also shaped by the frequency of small-effect deleterious mutations, whose combined effect can burden otherwise adaptive lineages and alter evolutionary trajectories and outcomes in clonally evolving organisms such as viruses, microbes, and tumors. The small effect sizes of these important mutations have made accurate measurements of their rates difficult. In microbes, assessing the effect of mutations on growth can be especially instructive, as this complex phenotype is closely linked to fitness in clonally evolving organisms. Here, we perform high-throughput time-lapse microscopy on cells from mutation-accumulation strains to precisely infer the distribution of mutational effects on growth rate in the budding yeast, Saccharomyces cerevisiae. We show that mutational effects on growth rate are overwhelmingly negative, highly skewed towards very small effect sizes, and frequent enough to suggest that deleterious hitchhikers may impose a significant burden on evolving lineages. By using lines that accumulated mutations in either wild-type or slippage repair-defective backgrounds, we further disentangle the effects of 2 common types of mutations, single-nucleotide substitutions and simple sequence repeat indels, and show that they have distinct effects on yeast growth rate. Although the average effect of a simple sequence repeat mutation is very small (approximately 0.3%), many do alter growth rate, implying that this class of frequent mutations has an important evolutionary impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e2d4d1c86097c9432759848b7286f55d338b931" target='_blank'>
              Spontaneous single-nucleotide substitutions and microsatellite mutations have distinct distributions of fitness effects
              </a>
            </td>
          <td>
            Y. Plavskin, M. S. de Biase, Naomi Ziv, Libuše Janská, Yuan O. Zhu, David W. Hall, Roland F Schwarz, Daniel Tranchina, M. Siegal
          </td>
          <td>2024-07-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db5f5f80053a63fc1633772174de85b7485c0712" target='_blank'>
              Saturation genome editing of BAP1 functionally classifies somatic and germline variants
              </a>
            </td>
          <td>
            Andrew J Waters, Timothy Brendler-Spaeth, Danielle Smith, Victoria Offord, Hong Kee Tan, Yajie Zhao, Sofia Obolenski, Maartje Nielsen, R. van Doorn, Jo-Ellen Murphy, Prashant Gupta, C. F. Rowlands, H. Hanson, E. Delage, Mark Thomas, Elizabeth J Radford, S. Gerety, Clare Turnbull, J. R. Perry, M. Hurles, David J. Adams
          </td>
          <td>2024-07-05</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="Background Relapsed/refractory acute myeloid leukemia (R/R-AML) has dismal prognosis due to chemotherapy resistance. Circular RNAs (circRNAs) have shown emerging roles in chemotherapy resistance in various cancers including hematologic malignancies. However, the potential roles of circRNAs in AML progression and drug resistance remain largely undetermined. Methods In this study, circulating circRNAs expression profiles were analyzed among R/R-AML, de novo AML and healthy controls (HC) using a human circRNA Array. Bioinformatic analysis was carried out to explore the differentially expressed circRNAs (DE-circRNAs). GO, KEGG pathway analysis, along with circRNA-miRNA-mRNA network analysis, were conducted to identify the potential biological pathways involved in R/R-AML. Finally, the UALCAN database was used to assess the prognosis of different target DE-circRNAs-related mRNAs. Results Forty-eight DE-circRNAs were upregulated, whereas twenty-seven DE-circRNAs were downregulated in R/R-AML samples. Up-regulated DE-circRNAs in R/R-AML samples were mainly enrichment in the biological processes and pathways of cell migration, microRNAs in cancers, Rap1 and Ras signaling pathways. Six DE-circRNAs were randomly selected to further explore their relationships with R/R-AML. GO and KEGG pathway analyses of the six candidate DE-circRNAs-related target mRNAs were mainly involved in the regulation of signal transduction and Ras signaling pathway. By overlapping our RNA-sequencing results of differentially expressed genes (DEGs) in R/R-AML samples with the candidate DE-circRNAs-predicted target mRNAs, we identified sixty-eight overlapping targeted mRNAs. Using UALCAN database analysis, we identified that AML patients with six upregulated DE-circRNA-related genes (ECE1, PI4K2A, SLC9A6, CCND3, PPP1R16B, and TRIM32) and one downregulated gene DE-circRNA-related genes (ARHGAP10) might have a poor prognosis. Conclusion This study revealed the overall alterations of circRNAs in R/R-AML. DE-circRNAs and their related genes might be used as potential early, sensitive and stable biomarkers for AML diagnosis, R/R-AML monitoring, and even as novel treatment targets for R/R-AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/68cf9c31c5780881aca1b710dce6fd3cd806578f" target='_blank'>
              Screening of a Prognostic Gene Signature for Relapsed/Refractory Acute Myeloid Leukemia Based on Altered Circulating CircRNA Profiles
              </a>
            </td>
          <td>
            Honggang Guo, Yabin Cui, Yanliang Bai, Fan Yan, Wenhui Zhang, Yu-qing Chen, M. Shi
          </td>
          <td>2024-07-01</td>
          <td>International Journal of General Medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Androgen receptor- (AR-) indifference is a mechanism of resistance to hormonal therapy in prostate cancer (PC). Here we demonstrate that ONECUT2 (OC2) activates resistance through multiple drivers associated with adenocarcinoma, stem-like and neuroendocrine (NE) variants. Direct OC2 gene targets include the glucocorticoid receptor (GR; NR3C1) and the NE splicing factor SRRM4, which are key drivers of lineage plasticity. Thus, OC2, despite its previously described NEPC driver function, can indirectly activate a portion of the AR cistrome through epigenetic activation of GR. Mechanisms by which OC2 regulates gene expression include promoter binding, enhancement of genome-wide chromatin accessibility, and super-enhancer reprogramming. Pharmacologic inhibition of OC2 suppresses lineage plasticity reprogramming induced by the AR signaling inhibitor enzalutamide. These results demonstrate that OC2 activation promotes a range of drug resistance mechanisms associated with treatment-emergent lineage variation in PC and support enhanced efforts to therapeutically target OC2 as a means of suppressing treatment-resistant disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/afa57725f4c34a58c8bfb81365718e7515db8a48" target='_blank'>
              ONECUT2 acts as a lineage plasticity driver in adenocarcinoma as well as neuroendocrine variants of prostate cancer
              </a>
            </td>
          <td>
            Chen Qian, Qian Yang, Mirja Rotinen, Rongrong Huang, Hyoyoung Kim, Brad Gallent, Yiwu Yan, Radu M. Cadaneanu, Baohui Zhang, S. Kaochar, Stephen J. Freedland, Edwin M Posadas, Leigh Ellis, Dolores Di Vizio, C. Morrissey, Peter S. Nelson, Lauren Brady, Ramachandran Murali, Moray J. Campbell, Wei Yang, Beatrice S Knudsen, E. Mostaghel, Huihui Ye, I. Garraway, Sungyong You, Michael R Freeman
          </td>
          <td>2024-06-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="A central challenge in the quest for precise gene regulation within mammalian cells is the development of regulatory networks that can achieve perfect adaptation-where outputs consistently return to a set baseline post-stimulus. Here, we present such a system that leverages the CRISPR activation (CRISPRa) and anti-CRISPR proteins as two antithetic elements to establish perfect adaptation in mammalian cells and dynamically regulate gene expression. We demonstrate that this system can maintain stable expression levels of target genes in the face of external perturbations, thus providing a robust platform for biological applications. The versatility of our system is further showcased through its integration with endogenous regulatory mechanisms in T cells, such as the NF-κB-mediated immune response, and its ability to program apoptosis responses for precise spatial and temporal control of cellular growth and death. This study not only advances our understanding of gene regulation in mammalian cells but also opens new avenues for therapeutic intervention, particularly in diseases characterized by dysregulated gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9bdcf363536b5c7b6585a226adb59f423a716ec" target='_blank'>
              CRISPR perfect adaptation for robust control of cellular immune and apoptotic responses.
              </a>
            </td>
          <td>
            Yichi Zhang, Shuyi Zhang
          </td>
          <td>2024-08-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Kirsten Rat Sarcoma (KRAS) is the most commonly mutated oncogene in colorectal carcinoma (CRC). We have previously reported the interactions between microsatellite instability (MSI), DNA promoter methylation, and gene expression. In this study, we looked for associations between KRAS mutation, gene expression, and methylation that may help with precision medicine. Genome-wide gene expression and DNA methylation were done in paired CRC tumor and surrounding healthy tissues. The results suggested that (a) the magnitude of dysregulation of many major gene pathways in CRC was significantly greater in patients with the KRAS mutation, (b) the up- and down-regulation of these dysregulated gene pathways could be correlated with the corresponding hypo- and hyper-methylation, and (c) the up-regulation of CDKN2A was more pronounced in tumors with the KRAS mutation. A recent cell line study showed that there were higher CDKN2A levels in 5-FU-resistant CRC cells and that these could be down-regulated by Villosol. Our findings suggest the possibility of a better response to anti-CDKN2A therapy with Villosol in KRAS-mutant CRC. Also, the more marked up-regulation of genes in the proteasome pathway in CRC tissue, especially with the KRAS mutation and MSI, may suggest a potential role of a proteasome inhibitor (bortezomib, carfilzomib, or ixazomib) in selected CRC patients if necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40b3e427194ea41aedea0dfcb0cf1b6c9f955b47" target='_blank'>
              Association of KRAS Mutation and Gene Pathways in Colorectal Carcinoma: A Transcriptome- and Methylome-Wide Study and Potential Implications for Therapy
              </a>
            </td>
          <td>
            F. Jasmine, Armando Almazan, Yuliia Khamkevych, Marc Bissonnette, Habibul Ahsan, M. Kibriya
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a type of blood cancer with diverse genetic variations and DNA methylation alterations. By studying the interaction of gene mutations, expression, and DNA methylation, we aimed to gain valuable insights into the processes that lead to block differentiation in AML. We analyzed TCGA-LAML data (173 samples) with RNA sequencing and DNA methylation arrays, comparing FLT3 mutant (48) and wild-type (125) cases. We conducted differential gene expression analysis using cBioPortal, identified DNA methylation differences with ChAMP tool, and correlated them with gene expression changes. Gene set enrichment analysis (g:Profiler) revealed significant biological processes and pathways. ShinyGo and GeneCards were used to find potential transcription factors and their binding sites among significant genes. We found significant differentially expressed genes (DEGs) negatively correlated with their most significant methylation probes (Pearson correlation coefficient of -0.49, P-value <0.001) between FLT3 mutant and wild-type groups. Moreover, our exploration of 450 k CpG sites uncovered a global hypo-methylated status in 168 DEGs. Notably, these methylation changes were enriched in the promoter regions of Homebox superfamily gene, which are crucial in transcriptional-regulating pathways in blood cancer. Furthermore, in FLT3 mutant AML patient samples, we observed overexpress of WT1, a transcription factor known to bind homeobox gene family. This finding suggests a potential mechanism by which WT1 recruits TET2 to demethylate specific genomic regions. Integrating gene expression and DNA methylation analyses shed light on the impact of FLT3 mutations on cancer cell development and differentiation, supporting a two-hit model in AML. This research advances understanding of AML and fosters targeted therapeutic strategy development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5d3ff3b479ed6f746656e6ba790e6039d0322f3" target='_blank'>
              Crosstalk between genomic variants and DNA methylation in FLT3 mutant acute myeloid leukemia.
              </a>
            </td>
          <td>
            Bac Dao, Van Ngu Trinh, H. Nguyen, Hoa L. Nguyen, Thuc Duy Le, Phuc Loi Luu
          </td>
          <td>2024-06-30</td>
          <td>Briefings in functional genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cell plasticity (CP), describing a dynamic cell state, plays a crucial role in maintaining homeostasis during organ morphogenesis, regeneration and damage-to-repair biological process. Single-cell-omics datasets provide unprecedented resource to empowers analysis on CP. Hematopoiesis offers fertile opportunities to develop quantitative methods for understanding CP with rich supports from experimental ground-truths. In this study we generated high-quality lineage-negative (Lin−) single-cell RNA-sequencing datasets under various conditions and introduced a working pipeline named Snapdragon to interrogate naïve and disturbed plasticity of hematopoietic stem and progenitor cells (HSPCs) with mutational or environmental challenges. Utilizing embedding methods UMAP or FA, a continuum of hematopoietic development is visually observed in wildtype where the pipeline confirms a very low Proportion of hybrid-cells (Phc, with bias range: 0.4-0.6) on a transition trajectory. Upon Tet2 mutation, a driver of leukemia, or treatment of DSS, an inducer of colitis, Phc is increased and plasticity of HSPCs was enhanced. Quantitative analysis indicates that Tet2 mutation enhances HSC self-renewal capability while DSS treatment results in an enhanced myeloid-skewing trajectory, suggesting their similar but different consequences. We prioritized several transcription factors (i.e the EGR family) and signaling pathways (i.e. receptors IL1R1 and ADRB, inflammation and sympathy-sensing respectively) which are responsible for Phc alterations. CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation. In conclusion, the study provides high-quality datasets with single-cell transcriptomic matrices for diversified hematopoietic simulations and a computational pipeline Snapdragon for quantifying disturbed Phc and CP. (247 words) Highlights To guide CP analysis, we introduce a quantizable parameter Phc and a pipeline Snapdragon, which discriminate naive and disturbed hematopoiesis; The Snapdragon pipeline analysis on Tet2+/-Lin− cells demonstrates many novel insights, including enhanced HSC plasticity and increased PHC; similar trends are observed in inflammatory Lin− cells; Regulon analysis suggests that transcriptional factor EGR1 is significantly activated to elevated the HSC plasticity and change hematopoietic trajectory; Stress-response-related signaling pathways mediated by receptors IL1R1 or ADRB were obviously activated in the challenged hematopoiesis; CellOracle-based simulation suggests that knocking-out EGR regulons or pathways of IL1R1 and ADRB partially reverses Phc promoted by Tet2 mutation and inflammation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96e8f484ca5cec1e51d0d19b3a9a4df71e03c5eb" target='_blank'>
              Computing hematopoietic stem and progenitor cell plasticity in response to genetic mutations and environmental stimulations
              </a>
            </td>
          <td>
            Yuchen Wen, Hang He, Yunxi Ma, Lorie Chen Cai, Huaquan Wang, Yanmei Li, Baobing Zhao, Zhigang Cai
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Antibiotic resistance is the leading cause of death globally, with a higher possibility of the emergence of highly resistant pathogens, leading to epidemics. Several antibiotic resistance mechanisms have been discovered, such as enhanced efflux of antibiotics, reduced influx of antibiotics, alteration of antibiotics or their targets, and adaptation to antibiotics. However, this mechanism cannot fully explain the development of antibiotic resistance because the genes associated with this mechanism have been elucidated. However, the factors governing their regulation are not yet fully understood. Recent studies have highlighted the epigenetic and epitranscriptomic roles of antibiotic resistance development-associated genes. Epigenetic modification is associated with DNA modification, whereas epitranscriptomic modification is associated with RNA modification to control gene expression by regulating various biological phenomena such as splicing, translation, and stability. Therefore, this review will focus on the discovery of epigenetic modifications, particularly by DNA methyltransferases, such as restriction-modification (R-M) systems associated with methyltransferases, orphan DNA methyltransferases, and nucleoid-associated proteins that contribute to the development of antibiotic resistance. This scrutinization further expands to epitranscriptomic modification of non-coding RNA, which has a role in the regulation of antibiotic resistance. Epitranscriptomic modification of ribosomal RNA (rRNA), which is a major target of antibiotics, has been well explored. while non-coding RNA such as cis and trans small non coding RNA, and riboswitches are poorly explored. This epigenetic and epitranscriptomic modification will help to understand the regulation of antibiotic resistance-associated genes, which will help to identify key regulators of antibiotic resistance, paving the way for new antibiotic discovery, leading to decreased antibiotic mortality globally.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24b16c6a0ffde83445547b9ffde6ff971186ffa3" target='_blank'>
              A comprehensive review on epigenetic and epitranscriptomic-mediated regulation of antibiotic resistance
              </a>
            </td>
          <td>
            P. K. Giri, Shahil Alam, Madhav Dhakal
          </td>
          <td>2024-07-24</td>
          <td>F1000Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33af20a7d0aac05ac83adef2c51ac9ffcee3995c" target='_blank'>
              Enhancer engagement sustains oncogenic transformation and progression of B-cell precursor acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            G. Corleone, C. Sorino, M. Caforio, S. Di Giovenale, F. de Nicola, F. Goeman, V. Bertaina, A. Pitisci, C. Cortile, Franco Locatelli, V. Folgiero, Maurizio Fanciulli
          </td>
          <td>2024-06-27</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Therapy-induced acquired resistance limits the clinical effectiveness of mutation-specific RAS inhibitors in colorectal cancer. It is unknown whether broad-spectrum active-state RAS inhibitors meet similar limitations. Here, we identify and categorize mechanisms of resistance to the broad-spectrum active-state RAS inhibitor RMC-7977 in colorectal cancer cell lines. We found that KRAS-mutant colorectal cancer cell lines are universally sensitive to RMC-7977, inhibiting the RAS-RAF-MEK-ERK axis, halting proliferation and in some cases inducing apoptosis. To monitor KRAS downstream effector pathway activity, we developed a compartment-specific dual-color ERK activity reporter. RMC-7977 treatment reduced reporter activity. However, long-term dose escalation with RMC-7977 revealed multiple patterns of reporter reactivation in emerging resistant cell populations that correlated with phosphorylation states of compartment-specific ERK targets. Cells sorted for high, low, or cytoplasmic reporter activity exhibited distinct patterns of genomic mutations, phospho-protein, and transcriptional activities. Notably, all resistant subpopulations showed dynamic ERK regulation in the presence of the RAS inhibitor, unlike the parental sensitive cell lines. High levels of RAS downstream activities were observed in cells characterized by a KRAS Y71H resistance mutation. In contrast, RAS inhibitor-resistant populations with low, or cytoplasmic ERK reporter reactivation displayed different genetic alterations, among them RAF1 S257L and S259P mutations. Colorectal cancer cells resistant to RMC-7977 and harboring the RAF1 mutation specifically exhibited synergistic sensitivity to concurrent RAS and RAF inhibition. Our findings endorse reporter-assisted screening together with single-cell analyses as a powerful approach for dissecting the complex landscape of therapy resistance. The strategy offers opportunities to develop clinically relevant combinatorial treatments to counteract emergence of resistant cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58f2dfe945929a08a6260606b0df62f40daad49b" target='_blank'>
              Reporter-based screening identifies RAS-RAF mutations as drivers of resistance to active-state RAS inhibition in colorectal cancer
              </a>
            </td>
          <td>
            Oleksandra Aust, Eric Blanc, Mareen Lüthen, Viola Hollek, Rosario Astaburuaga-García, B. Klinger, Alexandra Trinks, D. Beule, Björn Papke, David Horst, Nils Blüthgen, Christine Sers, Channing Der, M. Morkel
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSC) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosomes 20q or 1q, both known for conferring in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings show higher loads of both CNVs and SNVs at later passages which are due to the cumulative acquisition of mutations over a longer time in culture and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs follow the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid cancer-associated mutations. This suggests that the recurrent chromosomal abnormalities are the potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f552b60bd87e180c4d6a5dd522428893e4b81b" target='_blank'>
              De novo cancer mutations frequently associate with recurrent chromosomal abnormalities during long-term human pluripotent stem cell culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huygebaert, M. Regin, Chi Mai Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Cancer is a complex and heterogeneous disease, in which a number of genetic and epigenetic changes occur in tumor onset and progression. Recent studies indicate that changes at the RNA level are also involved in tumorigenesis, such as adenosine-to-inosine (A-to-I) RNA editing. Here, we systematically investigate transcriptome-wide A-to-I editing events in a large number of samples from Non-Hodgkin lymphomas (NHLs). Using a computational pipeline that determines significant differences in editing level between NHL and normal samples at known A-to-I editing sites, we identify a number of differentially edited editing sites between NHL subtypes and normal samples. Most of the differentially edited sites are located in non-coding regions, and many such sites show a strong correlation between gene expression level and editing efficiency, indicating that RNA editing might have direct consequences for the cancer cell’s aberrant gene regulation status in these cases. Moreover, we establish a strong link between RNA editing and NHL by demonstrating that NHL and normal samples and even NHL subtypes can be distinguished based on genome-wide RNA editing profiles alone. Our study establishes a strong link between RNA editing, cancer and aberrant gene regulation in NHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3709fa38f5f1119b1fcb4db3f2e4cd77244ecc08" target='_blank'>
              A-to-I Editing Is Subtype-Specific in Non-Hodgkin Lymphomas
              </a>
            </td>
          <td>
            Cai Chen, Ralf Bundschuh
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Aneuploidy, the presence of an aberrant number of chromosomes, has been associated with tumorigenesis for over a century. More recently, advances in karyotyping techniques have revealed its high prevalence in cancer: About 90% of solid tumors and 50–70% of hematopoietic cancers exhibit chromosome gains or losses. When analyzed at the level of specific chromosomes, there are strong patterns that are observed in cancer karyotypes both pan-cancer and for specific cancer types. These specific aneuploidy patterns correlate strongly with outcomes for tumor initiation, progression, metastasis formation, immune evasion and resistance to therapeutic treatment. Despite their prominence, understanding the basis underlying aneuploidy patterns in cancer has been challenging. Advances in genetic engineering and bioinformatic analyses now offer insights into the genetic determinants of aneuploidy pattern selection. Overall, there is substantial evidence that expression changes of particular genes can act as the positive selective forces for adaptation through aneuploidy. Recent findings suggest that multiple genes contribute to the selection of specific aneuploid chromosomes in cancer; however, further research is necessary to identify the most impactful driver genes. Determining the genetic basis and accompanying vulnerabilities of specific aneuploidy patterns is an essential step in selectively targeting these hallmarks of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0123dd290f56b3193001c1283aea1071e6f411ed" target='_blank'>
              Selection forces underlying aneuploidy patterns in cancer
              </a>
            </td>
          <td>
            Tamara C. Klockner, Christopher S. Campbell
          </td>
          <td>2024-06-24</td>
          <td>Molecular & Cellular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Purpose: Homologous recombination deficiency (HRD) impacts cancer treatment strategies, particularly the effectiveness of PARP inhibitors. However, the variability different HRD assays has hampered the selection of oncology patients who may benefit from these therapies. Our study aims to assess the whole genome landscape to better define HRD in a pan-cancer cohort and to contribute to harmonization of HRD detection. Methods: We employed a whole-genome sequencing WGS HRD classifier that included genome-wide features associated with HRD to analyze 580 tumor/normal paired pan-cancer samples. The HRD results were correlated retrospectively with treatment responses and were compared with commercial HRD tests in a subset of cases. Results: HRD phenotype was identified in 62 samples across various cancers including breast (19%), pancreaticobiliary (17%), gynecological (15%), prostate (8%), upper gastrointestinal (GI) (2%), and other cancers (1%). HRD cases were not confined to BRCA1/2 mutations; 24% of HRD cases were BRCA1/2 wild-type. A diverse range of HRR pathway gene alterations involved in HRD were elucidated, including biallelic mutations in FANCF, XRCC2, and FANCC, and deleterious structural variants. Comparison with results from commercial HRD assays suggests a better performance of WGS to detect HRD, based on treatment response. Conclusion: HRD is a biomarker used to determine which cancer patients would benefit from PARPi and platinum-based chemotherapy. However, a lack of harmonization of tests to determine HRD status makes it challenging to interpret their results. Our study highlights the use of comprehensive WGS analysis to predict HRD in a pan-cancer cohort, elucidates new genomic mechanisms associated with HRD, and enables an accurate identification of this phenotype, paving the way for improved outcomes in oncology care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80d347a4838be16d195accf2ef681565a4b6929" target='_blank'>
              Whole Genome Landscape Analysis of Homologous Recombination Deficiency in a Pan-Cancer Cohort
              </a>
            </td>
          <td>
            MD Majd Al Assaad, PhD Kevin Hadi, MS Max F. Levine, MD Daniela Guevara, Mph Minal Patel, MS Marvel Tranquille, BS Abigail King, BS John Otilano, MS Alissa Semaan, PhD Gunes Gundem, MS Juan S. Medina-Martínez, BS Michael Sigouros, Mph Jyothi Manohar, BS Hui-Hsuan Kuo, PhD David C. Wilkes, MD Eleni Andreopoulou, MD Eloise Chapman-Davis, MD Scott T. Tagawa, PhD Andrea Sboner, MD Allyson J. Ocean, MD Manish Shah, PhD Elli Papaemmanuil, Cora N. Sternberg, MD Kevin Holcomb, MD David M. Nanus, PhD Olivier Elemento, Juan Miguel Mosquera, J. Medina-Martinez
          </td>
          <td>2024-06-30</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Meiotic recombination is a fundamental process that generates genetic diversity by creating new combinations of existing alleles. Although human crossovers have been studied at the pedigree, population and single-cell level, the more frequent non-crossover events that lead to gene conversion are harder to study, particularly at the individual level. Here we show that single high-fidelity long sequencing reads from sperm can capture both crossovers and non-crossovers, allowing effectively arbitrary sample sizes for analysis from one male. Using fifteen sperm samples from thirteen donors we demonstrate variation between and within donors for the rates of different types of recombination. Intriguingly, we observe a tendency for non-crossover gene conversions to occur upstream of nearby PRDM9 binding sites, whereas crossover locations have a slight downstream bias. We further provide evidence for two distinct non-crossover processes. One gives rise to the vast majority of non-crossovers with mean conversion tract length under 50bp, which we suggest is an outcome of standard PRDM9-induced meiotic recombination. In contrast ∼2% of non-crossovers have much longer mean tract length, and potentially originate from the same process as complex events with more than two haplotype switches, which is not associated with PRDM9 binding sites and is also seen in somatic cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c7a7057073d62495af1df781d99095c76dca90d" target='_blank'>
              Insights into non-crossover recombination from long-read sperm sequencing
              </a>
            </td>
          <td>
            Regev Schweiger, Sangjin Lee, Chenxi Zhou, Tsun-Po Yang, Katie Smith, Stacy Li, R. Sanghvi, Matthew Neville, Emily Mitchell, Ayrun Nessa, Sam Wadge, K. Small, Peter J Campbell, Peter H. Sudmant, R. Rahbari, R. Durbin
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Populations of the malaria parasite Plasmodium falciparum regularly confront orchestrated changes in frontline drug treatment that drastically alter the parasite’s selection landscape. When this has occurred, the parasite has successfully adapted to the new drugs through novel resistance mutations. These novel mutations, however, may emerge in a genetic background already shaped by prior drug selection. In some instances, selection imposed by distinct drugs has targeted the same loci in either synergistic or antagonistic ways, resulting in genomic signatures that can be hard to attribute to a specific agent. Here, we use two approaches for detecting sequential bouts of drug adaptation: haplotype-based selection testing and temporal changes in allele frequencies. Using a set of longitudinally acquired samples from French Guiana, we determine that since the introduction of the drug artemether-lumefantrine (AL) in 2007 there have been rapid hard selective sweeps at both known and novel loci. We additionally identify genomic regions where selection acted in opposing directions before and after widespread AL introduction. At four high-profile genes with demonstrated involvement in drug resistance (crt, mdr1, aat1, and gch1), we saw strong selection before and after drug regime change; however, selection favored different haplotypes in the two time periods. Similarly, the allele frequency analysis identified coding variants whose frequency trajectory changed sign under the new drug pressure. These selected alleles were enriched for genes implicated in artemisinin and/or partner drug resistance in other global populations. Overall, these results suggest that drug resistance in P. falciparum is governed by known alleles of large effect along with a polygenic architecture of more subtle variants, any of which can experience fitness reversals under distinct drug regimes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97ef8ddfecc3630e73f74049765e533a41788087" target='_blank'>
              Temporal patterns of haplotypic and allelic diversity reflect the changing selection landscape of the malaria parasite Plasmodium falciparum
              </a>
            </td>
          <td>
            A. Early, Stéphane Pelleau, Lise Musset, D. Neafsey
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background: Tamoxifen is commonly used in the treatment of hormonal-positive breast cancer. However, 30%-40% of tumors treated with tamoxifen develop resistance; therefore, an important step to overcome this resistance is to understand the underlying molecular and metabolic mechanisms. In the present work, we used metabolic profiling to determine potential biomarkers of tamoxifen resistance, and gene expression levels of enzymes important to these metabolites and then correlated the expression to the survival of patients receiving tamoxifen. Methods: Tamoxifen-resistant cell lines previously developed and characterized in our laboratory were metabolically profiled with nuclear magnetic resonance spectroscopy (NMR) using cryogenic probe, and the findings were correlated with the expression of genes that encode the key enzymes of the significant metabolites. Moreover, the effect of significantly altered genes on the overall survival of patients was assessed using the Kaplan-Meier plotter web tool. Results: We observed a significant increase in the levels of glutamine, taurine, glutathione, and xanthine, and a significant decrease in the branched-chain amino acids, valine, and isoleucine, as well as glutamate and cysteine in the tamoxifen-resistant cells compared to tamoxifen sensitive cells. Moreover, xanthine dehydrogenase and glutathione synthase gene expression were downregulated, whereas glucose-6-phosphate dehydrogenase was upregulated compared to control. Additionally, increased expression of xanthine dehydrogenase was associated with a better outcome for breast cancer patients. Conclusion: Overall, this study sheds light on metabolic pathways that are dysregulated in tamoxifen-resistant cell lines and the potential role of each of these pathways in the development of resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18acaa27d6e37cf1939c43a13b10ab67089b4a01" target='_blank'>
              Glutathione and Xanthine Metabolic Changes in Tamoxifen Resistant Breast Cancer Cell Lines are Mediated by Down-Regulation of GSS and XDH and Correlated to Poor Prognosis
              </a>
            </td>
          <td>
            M. Alwahsh, Yazan Hamadneh, Rosemarie Marchan, Lina A. Dahabiyeh, Ala A. Alhusban, Aya Hasan, Jawad Alrawabdeh, R. Hergenröder, Lama A. Hamadneh
          </td>
          <td>2024-05-30</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="A major goal in biology is to uncover the relationship between genotype and phenotype. However, identifying gene function is often hampered by genetic redundancy. For example, under standard laboratory conditions, three-quarters of the genes in the human pathogen Streptococcus pneumoniae are non-essential. A powerful approach to unravel genetic redundancy is by identifying gene-gene interactions. To uncover genetic interactions (GIs) in S. pneumoniae on a genome-wide scale, a generally applicable dual CRISPRi-Seq method and associated analysis pipeline was developed. Specifically, we created a library of 869 dual sgRNAs targeting high-confidence operons that encode essential and non-essential genes, covering over 70% of the genetic elements in the pneumococcal genome. Testing these 378,015 unique combinations, 4,026 significant GIs were identified, including 1,935 negative and 2,091 positive interactions. Besides known GIs, we found and confirmed previously unknown interactions involving genes responsible for fundamental cellular processes such as cell division, cell shape maintenance, and chromosome segregation. The presented methods and bioinformatic approaches can serve as a roadmap for genome-wide gene interaction studies in other organisms. Lastly, all interactions are available for exploration via the Pneumococcal Genetic Interaction Network (PneumoGIN) at https://veeninglab.shinyapps.io/PneumoGIN, which can serve as a starting point for new biological discoveries and translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dd78fdd99cdc75fb8cdbb9e56ea5f0c3b73930" target='_blank'>
              Dual CRISPRi-Seq for genome-wide genetic interaction studies identifies key genes involved in the pneumococcal cell cycle
              </a>
            </td>
          <td>
            Julien Dénéréaz, Elise Eray, Bimal Jana, Vincent de Bakker, Horia Todor, Tim van Opijnen, Xue Liu, J. Veening
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/553f4560eb657fca2fe86385e460c71427df430c" target='_blank'>
              GLI1 confers resistance to PARP inhibitors by activating the DNA damage repair pathway.
              </a>
            </td>
          <td>
            Hiroshi Ikeuchi, Yusuke Matsuno, Rika Kusumoto-Matsuo, S. Kojima, Toshihide Ueno, Masachika Ikegami, Rina Kitada, Hitomi Sumiyoshi-Okuma, Yuki Kojima, K. Yonemori, Y. Yatabe, K. Takamochi, Kenji Suzuki, Ken‐ichi Yoshioka, H. Mano, S. Kohsaka
          </td>
          <td>2024-08-03</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA damage-sensing (DDS) and DNA damage-repair (DDR) mechanisms are essential for the fidelity of genetic information transmission. Failure to accomplish an effective DDS/DDR mechanism can lead to cell death or otherwise to cell transformation and cancer development. microRNAs (miRs) are short noncoding RNAs that primarily function as micromanagers of gene expression. Herein, we aimed to investigate the links between miRs and the translation of specific mRNAs encoding proteins involved in genomic DDS and DDR and to screen drugs that have high binding affinity to the selected miRs, which may serve as cancer therapeutics. To accomplish these aims, we used a variety of computational methods spanning data analysis, molecular modeling, and simulation tools (i.e., PyRx, Biopython, ViennaRNA, RNAComposer, AutoDock Vina, OpenBabel, PyMOL, Discovery Studio, MarvinSketch). The genes and miRNAs involved in the DDS and DDR mechanisms were retrieved from either the literature or various online databases (e.g., miRDB). miR data were further cleaned and prepared using scripts, and various libraries were used to obtain their 3-D structures. Genes interacting with miRs were enriched based on multiple database annotations using Enrichr KG. Then, we used docking analyses to virtually screen compounds to serve as ligands for the miRs. Finally, we generated gene-disease-miR-drug networks to study the linkages between the compounds and miR molecules under investigation. For the first time, we were able to identify five compounds that could be repurposed for downregulating miRs that are linked to inhibition of translation of mRNA involved in the DDS and DDR processes. The gathered candidate drugs can be useful for preventing cell transformation and cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04982a355813b15d80368bb235c815920aa1fe9d" target='_blank'>
              Drugs targeting microRNAs that regulate DNA damage sensing and repair mechanisms: A computational drug discovery approach
              </a>
            </td>
          <td>
            S. Ravindranath, Ramirez, Gomez Mejiba S.E, S. Gomez Mejiba, ●. Ravindranath, ●. Ramirez, ●. G. Mejiba
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/460b0c287af289dbf822999647d9d57a5f2960b9" target='_blank'>
              Integrative single-cell RNA-seq and spatial transcriptomics analyses reveal diverse apoptosis-related gene expression profiles in EGFR-mutated lung cancer
              </a>
            </td>
          <td>
            Motohiro Izumi, Masanori Fujii, I. Kobayashi, Vivian Ho, Y. Kashima, Hibiki Udagawa, Daniel B. Costa, Susumu S. Kobayashi
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Background Germline mutations have been identified in a small number of hereditary cancers, but the genetic predisposition for many familial cancers remains to be elucidated. Methods This study identified a Chinese pedigree that presented different cancers (breast cancer, BRCA; adenocarcinoma of the esophagogastric junction, AEG; and B‐cell acute lymphoblastic leukemia, B‐ALL) in each of the three generations. Whole‐genome sequencing and whole‐exome sequencing were performed on peripheral blood or bone marrow and cancer biopsy samples. Whole‐genome bisulfite sequencing was conducted on the monozygotic twin brothers, one of whom developed B‐ALL. Results According to the ACMG guidelines, bioinformatic analysis of the genome sequencing revealed 20 germline mutations, particularly mutations in the DNAH11 (c.9463G > A) and CFH (c.2314G > A) genes that were documented in the COSMIC database and validated by Sanger sequencing. Forty‐one common somatic mutated genes were identified in the cancer samples, displaying the same type of single nucleotide substitution Signature 5. Meanwhile, hypomethylation of PLEK2, MRAS, and RXRA as well as hypermethylation of CpG island associated with WT1 was shown in the twin with B‐ALL. Conclusions These findings reveal genomic alterations in a pedigree with multiple cancers. Mutations found in the DNAH11, CFH genes, and other genes predispose to malignancies in this family. Dysregulated methylation of WT1, PLEK2, MRAS, and RXRA in the twin with B‐ALL increases cancer susceptibility. The similarity of the somatic genetic changes among the three cancers indicates a hereditary impact on the pedigree. These familial cancers with germline and somatic mutations, as well as epigenomic alterations, represent a common molecular basis for many multiple cancer pedigrees.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3f0ae27a463e75a2f599e816c3a044e8b7c954" target='_blank'>
              Impact of genomic and epigenomic alterations of multigene on a multicancer pedigree
              </a>
            </td>
          <td>
            Jinyu Gao, Yongzhang Wu, Jieming Yu, Yinbin Qiu, Tiantian Yi, Chaochao Luo, Junxiao Zhang, Gary Lu, Xu Li, Fu Xiong, Xuedong Wu, Xinghua Pan
          </td>
          <td>2024-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Alterations in the FMS-like tyrosine kinase 3 (FLT3) gene are the most frequent driver mutations in acute myeloid leukaemia (AML), linked to a high risk of relapse in patients with internal tandem duplications (FLT3-ITD). Tyrosine kinase inhibitors (TKIs) targeting the FLT3 protein are approved for clinical use, yet resistance often emerges. This resistance is mainly seen following the acquisition of additional point mutations in the tyrosine kinase domain (TKD), resulting in a double mutant FLT3-ITD/TKD, which sustains cell signalling and survival despite the presence of FLT3 inhibitors. Here, we developed a FLT3-mutant AML model with adaptive resistance to type II TKIs, sorafenib, and quizartinib by in vitro drug selection. Through global multiomic profiling, we identified upregulation of proteins involved in reactive oxygen species (ROS) production, particularly NADPH-oxidases, driving cellular ’ROS-addiction’, with resistant cells relying on ROS for survival, and genome fidelity preserved by ATM-driven DNA repair. Transcriptomic analysis of adult and paediatric AML (pAML) patients identified high ATM expression as a biomarker for shorter median overall survival in both the de novo and relapsed settings. Inhibition of ATM with clinically relevant therapy WSD-0628 effectively killed TKI- and chemotherapy-resistant AML cells in vitro and significantly extended the survival of mice with sorafenib- and quizartinib-resistant FLT3-ITD AML in vivo. We propose a new treatment strategy to improve survival of patients who develop resistance to sorafenib and quizartinib, as well as relapsed and refractory pAML, exploiting resistance mechanisms to precision therapies and cell-intrinsic features of high-risk cases, highlighting a clinically relevant salvage strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0bcd4e64b15b98143cfc995f3b3d32e9c623b66" target='_blank'>
              Adaptive resistance to FLT3 inhibitors is potentiated by ROS-driven DNA repair signalling
              </a>
            </td>
          <td>
            Dilana E. Staudt, Zacary P. Germon, Abdul Mannan, Tabitha Mclachlan, Heather C Murray, Ryan J. Duchatel, Bryce C. Thomas, Tyrone S Beitaki, Holly P. McEwen, Mika L Persson, Leah Calvert, Izac J. Findlay, Evangeline R. Jackson, Nathan D. Smith, D. Skerrett‐Byrne, David Mossman, Brett Nixon, Geoffry De Iullis, Alicia M. Douglas, A. Enjeti, Jonathan R. Sillar, Janis Chamberlain, Frank Alvaro, Andrew H. Wei, Patrick Connerty, N. Verrills, Matthew D. Dun
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Stable isotope-resolved metabolomics delineates reprogrammed intersecting metabolic networks in human cancers. Knowledge gained from in vivo patient studies provides the "benchmark" for cancer models to recapitulate. It is particularly difficult to model patients' tumor microenvironment (TME) with its complex cell-cell/cell-matrix interactions, which shapes metabolic reprogramming crucial to cancer development/drug resistance. Patient-derived organotypic tissue cultures (PD-OTCs) represent a unique model that retains an individual patient's TME. PD-OTCs of non-small-cell lung cancer better recapitulated the in vivo metabolic reprogramming of patient tumors than the patient-derived tumor xenograft (PDTX), while enabling interrogation of immunometabolic response to modulators and TME-dependent resistance development. Patient-derived organoids (PDOs) are also good models for reconstituting TME-dependent metabolic reprogramming and for evaluating therapeutic responses. Single-cell based 'omics on combinations of PD-OTC and PDO models will afford an unprecedented understanding on TME dependence of human cancer metabolic reprogramming, which should translate into the identification of novel metabolic targets for regulating TME interactions and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6597c82613b1bff5869b53a6356a5e51a2359cc" target='_blank'>
              Metabolic Reprogramming in Human Cancer Patients and Patient-Derived Models.
              </a>
            </td>
          <td>
            T. W. Fan, R. Higashi, A. Lane
          </td>
          <td>2024-07-15</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>1</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2256152e45d11bd82f68a7e3810e940b41cb02" target='_blank'>
              Essential gene screening identifies the bromodomain-containing protein BRPF1 as a new actionable target for endocrine therapy-resistant breast cancers
              </a>
            </td>
          <td>
            A. Salvati, G. Giurato, J. Lamberti, Ilaria Terenzi, Laura Crescenzo, Viola Melone, L. Palo, A. Giordano, Francesco Sabbatino, G. Roscigno, C. Quintavalle, G. Condorelli, Francesca Rizzo, Roberta Tarallo, G. Nassa, Alessandro Weisz
          </td>
          <td>2024-08-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b62bbc42b475a59548fce65b7db966d5dba12012" target='_blank'>
              CRISPRi-TnSeq maps genome-wide interactions between essential and non-essential genes in bacteria.
              </a>
            </td>
          <td>
            Bimal Jana, Xue Liu, Julien Dénéréaz, Hongshik Park, Dmitry Leshchiner, Bruce Liu, Clément Gallay, Junhao Zhu, J. Veening, Tim van Opijnen
          </td>
          <td>2024-07-19</td>
          <td>Nature microbiology</td>
          <td>1</td>
          <td>46</td>
        </tr>

        <tr id="The search for genetic variants that act as causative factors in human diseases by disrupting the normal splicing process has primarily focused on single nucleotide variants (SNVs). It is worth noting that insertions or deletions (indels) have also been sporadically reported as causative disease variants through their potential impact on the splicing process. In this study, to perform identification of indels inducing exon extension/shrinkage events, we used individual-specific genomes and RNA sequencing (RNA-seq) data pertaining to the corresponding individuals and identified 12 exon extension/shrinkage events that were potentially induced by indels that disrupted authentic splice sites or created novel splice sites in 235 normal individuals. By evaluating the impact of these abnormal splicing events on the resulting transcripts, we found that five events led to the generation of premature termination codons (PTCs), including those occurring within genes associated with genetic disorders. Our analysis revealed that the potential functions of indels have been underexamined, and it is worth considering the possibility that indels may affect splice site usage, using RNA-seq data to discover novel potentially disease-associated mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62ba041d07636b21a672bb99a3107362cc90dc0c" target='_blank'>
              Identification and analysis of short indels inducing exon extension/shrinkage events.
              </a>
            </td>
          <td>
            Zhuo Qu, Narumi Sakaguchi, Chie Kikutake, Mikita Suyama
          </td>
          <td>2024-07-31</td>
          <td>FEBS open bio</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic regulatory mechanism that plays a critical role in a variety of cellular processes, including the regulation of cell fate during development, maintenance of cell identity, and genome stability. DNA methylation is tightly regulated by enzymatic reactions and its deregulation plays an important role in the development of cancer. Specific DNA methylation alterations have been found in pediatric solid tumors, providing new insights into the development of these tumors. In addition, DNA methylation profiles have greatly contributed to tune the diagnosis of pediatric solid tumors and to define subgroups of patients with different risks of progression, leading to the reduction in unwanted toxicity and the improvement of treatment efficacy. This review highlights the dysregulated DNA methylome in pediatric solid tumors and how this information provides promising targets for epigenetic therapies, particularly inhibitors of DNMT enzymes (DNMTis). Opportunities and limitations are considered, including the ability of DNMTis to induce viral mimicry and immune signaling by tumors. Besides intrinsic action against cancer cells, DNMTis have the potential to sensitize immune-cold tumors to immunotherapies and may represent a remarkable option to improve the treatment of challenging pediatric solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1eac857dfcc5a5e3b672ddb17d6097e48bc70f" target='_blank'>
              Targeting DNA Methylation Machinery in Pediatric Solid Tumors
              </a>
            </td>
          <td>
            C. Cristalli, Katia Scotlandi
          </td>
          <td>2024-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed1698b10d6cf746129577cdea369845de24a960" target='_blank'>
              Signaling dynamics in coexisting monoclonal cell subpopulations unveil mechanisms of resistance to anti-cancer compounds
              </a>
            </td>
          <td>
            Claire E Blanchard, Alison T Gomeiz, Kyle Avery, Emna El Gazzah, Abduljalil M. Alsubaie, M. Sikaroodi, Ylenia Chiari, Chelsea Ward, Jonathan Sanchez, Virginia Espina, Emanuel F. Petricoin, E. Baldelli, M. Pierobon
          </td>
          <td>2024-07-26</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b2c5c7d1cda920e03f84144f36e08d5e8bdd9f" target='_blank'>
              Genomic barcoding for clonal diversity monitoring and control in cell-based complex antibody production
              </a>
            </td>
          <td>
            Niels Bauer, Christoph Oberist, Michaela Poth, Julian Stingele, Oliver Popp, Simon Ausländer
          </td>
          <td>2024-06-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af8a5e84090feb0d13451624ef0b08cb6aa245a3" target='_blank'>
              Epigenetic regulation of DNA repair gene program by Hippo/YAP1-TET1 axis mediates sorafenib resistance in HCC
              </a>
            </td>
          <td>
            Chunli Mo, Weixin You, Yipeng Rao, Zhenping Lin, Shuai Wang, Ting He, Huanming Shen, Xun Li, Rui Zhang, Boan Li
          </td>
          <td>2024-07-05</td>
          <td>Cellular and Molecular Life Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary Cancer cells often become resistant to treatment, making it harder to fight the disease effectively. This resistance can happen due to genetic changes, proteins that pump out drugs, or the environment around the tumor. Crizotinib is a drug that helps overcome this resistance in certain lung cancers and lymphomas by targeting specific proteins. It can also break down a harmful protein found in some cancers. Researchers have found that combining crizotinib with other drugs can improve its effectiveness, especially for cancers with specific genetic mutations. Abstract Chemoresistance is a major obstacle in cancer treatment, often leading to disease progression and poor outcomes. It arises through various mechanisms such as genetic mutations, drug efflux pumps, enhanced DNA repair, and changes in the tumor microenvironment. These processes allow cancer cells to survive despite chemotherapy, underscoring the need for new strategies to overcome resistance and improve treatment efficacy. Crizotinib, a first-generation multi-target kinase inhibitor, is approved by the FDA for the treatment of ALK-positive or ROS1-positive non-small cell lung cancer (NSCLC), refractory inflammatory (ALK)-positive myofibroblastic tumors (IMTs) and relapsed/refractory ALK-positive anaplastic large cell lymphoma (ALCL). Crizotinib exists in two enantiomeric forms: (R)-crizotinib and its mirror image, (S)-crizotinib. It is assumed that the R-isomer is responsible for the carrying out various processes reviewed here The S-isomer, on the other hand, shows a strong inhibition of MTH1, an enzyme important for DNA repair mechanisms. Studies have shown that crizotinib is an effective multi-kinase inhibitor targeting various kinases such as c-Met, native/T315I Bcr/Abl, and JAK2. Its mechanism of action involves the competitive inhibition of ATP binding and allosteric inhibition, particularly at Bcr/Abl. Crizotinib showed synergistic effects when combined with the poly ADP ribose polymerase inhibitor (PARP), especially in ovarian cancer harboring BRCA gene mutations. In addition, crizotinib targets a critical vulnerability in many p53-mutated cancers. Unlike its wild-type counterpart, the p53 mutant promotes cancer cell survival. Crizotinib can cause the degradation of the p53 mutant, sensitizing these cancer cells to DNA-damaging substances and triggering apoptosis. Interestingly, other reports demonstrated that crizotinib exhibits anti-bacterial activity, targeting Gram-positive bacteria. Also, it is active against drug-resistant strains. In summary, crizotinib exerts anti-tumor effects through several mechanisms, including the inhibition of kinases and the restoration of drug sensitivity. The potential of crizotinib in combination therapies is emphasized, particularly in cancers with a high prevalence of the p53 mutant, such as triple-negative breast cancer (TNBC) and high-grade serous ovarian cancer (HGSOC).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c55a5da7409fe84d8ab6b88da7dafb5e0032e27" target='_blank'>
              Overcoming Chemoresistance in Cancer: The Promise of Crizotinib
              </a>
            </td>
          <td>
            Sanaa Musa, Noor Amara, Adan Selawi, Junbiao Wang, Cristina Marchini, A. Agbarya, J. Mahajna
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6bed6d0cd70aa3e466ea83d4896b497ee7efbf" target='_blank'>
              N6-methyladenosine modified TGFB2 triggers lipid metabolism reprogramming to confer pancreatic ductal adenocarcinoma gemcitabine resistance
              </a>
            </td>
          <td>
            Ming-Jian Ma, Yin-Hao Shi, Zhi-De Liu, Ying-Qin Zhu, Guang-Yin Zhao, Jing-Yuan Ye, Fu-Xi Li, Xi-Tai Huang, Xi-Yu Wang, Jie-Qin Wang, Qiong-Cong Xu, Xiao-Yu Yin
          </td>
          <td>2024-06-24</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Fungi can cause devastating invasive infections, typically in immunocompromised patients. Treatment is complicated both by the evolutionary similarity between humans and fungi and by the frequent emergence of drug resistance. Studies in fungal pathogens have long been slowed by a lack of high-throughput tools and community resources that are common in model organisms. Here we demonstrate a high-throughput transposon mutagenesis and sequencing (TN-seq) system in Cryptococcus neoformans that enables genome-wide determination of gene essentiality. We employed a random forest machine learning approach to classify the Cryptococcus neoformans genome as essential or nonessential, predicting 1,465 essential genes, including 302 that lack human orthologs. These genes are ideal targets for new antifungal drug development. TN-seq also enables genome-wide measurement of the fitness contribution of genes to phenotypes of interest. As proof of principle, we demonstrate the genome-wide contribution of genes to growth in fluconazole, a clinically used antifungal. We show a novel role for the well-studied RIM101 pathway in fluconazole susceptibility. We also show that 5’ insertions of transposons can drive sensitization of essential genes, enabling screenlike assays of both essential and nonessential components of the genome. Using this approach, we demonstrate a role for mitochondrial function in fluconazole sensitivity, such that tuning down many essential mitochondrial genes via 5’ insertions can drive resistance to fluconazole. Our assay system will be valuable in future studies of C. neoformans, particularly in examining the consequences of genotypic diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/126840639a843aec8469a2a2562239beea433d1f" target='_blank'>
              Saturation transposon mutagenesis enables genome-wide identification of genes required for growth and fluconazole resistance in the human fungal pathogen Cryptococcus neoformans
              </a>
            </td>
          <td>
            R. B. Billmyre, Caroline J. Craig, Joshua Lyon, Claire Reichardt, Michael T. Eickbush, Sarah E. Zanders
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, Andre Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Background and aim. Lung cancer remains one of the most threatening malignancies, ranking as the second most diagnosed cancer, and it continues to be the leading cause of cancer-related deaths worldwide. Challenges persist with late diagnosis and the high mutational burden characteristic of lung cancer. 
Methods. Our study focuses on identifying the mutational spectrum of a cohort of advanced-stage non-small cell lung cancer (NSCLC) patients using a minimally invasive method through blood collection. To analyze the mutational landscape of these patients, we employed plasma DNA for the next-generation sequencing (NGS) cancer panel Ion Torrent, which contains 50 of the most mutated genes in lung cancer. All protocols for extraction, quality and quantity control, and library preparation follow the manufacturer’s rules. Bioinformatics analysis was performed to select pathogenic mutations versus non-pathogenic-benign ones. 
Results. This approach is particularly valuable for patients in advanced stages (III and IV, n=10) of lung adenocarcinoma and lung squamous cell carcinoma, who lack surgical options and limited therapeutic avenues. The comprehensive sequencing analysis revealed that nine of the ten lung cancer patients carried a TP53 mutation. Also, several other mutations exist in various cases, showing heterogeneous profiling. 
Conclusions. Our findings demonstrate the potential of liquid biopsies in providing crucial genetic insights that can guide personalized treatment strategies, improving the management and outcomes for patients with advanced lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6844d155d0da77d29ae773874578536a65b3932" target='_blank'>
              Next-generation sequencing as a valuable tool for mutational spectrum in advanced-stage NSCLC patients
              </a>
            </td>
          <td>
            Ioana Iurca, Ecaterina Isakescu, L. Pop, Liviuţa Budişan, R. Pîrlog, A. Haranguș, T. Ciuleanu, C. Braicu, I. Berindan-Neagoe
          </td>
          <td>2024-07-10</td>
          <td>Medicine and Pharmacy Reports</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Mutations in subunits of the SWItch Sucrose Non-Fermentable (SWI/SNF) chromatin remodeling complex occur in ≈20% of cancers and represent a highly unmet medical need. To identify novel therapeutic approaches, we systematically characterized transcriptomic and proteomic changes caused by the loss of SWI/SNF subunits or other epigenetic enzymes in isogenic cell lines, which we subsequently integrated with high-throughput drug screening and independent genetic screens of the DepMap project. Using an optimized bioinformatics pipeline for pathway enrichment, we identified Metabolism of proteins as the most frequently dysregulated Reactome pathway category in SWI/SNF-defective cell lines. Drug screening and multiomic integration revealed multiple chemicals selectively cytotoxic for SWI/SNF-defective models, including CBP/EP300 or mitochondrial respiration inhibitors. A novel algorithm for the analysis of DepMap CRISPR screens independently identified synthetic lethality between SWI/SNF defects and EP300 or mitochondrial respiration genes, which we further revalidated in disease-relevant models. These results unravel novel genetic dependencies for SWI/SNF-defective cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/494552ad585e5d90eeb24a7f96c16c3381361c3e" target='_blank'>
              Integrated multiomic profiling reveals SWI/SNF subunit-specific pathway alterations and targetable vulnerabilities
              </a>
            </td>
          <td>
            Jorge Bretones Santamarina, Clémence Astier, M. Garrido, L. C. Daage, Theodoros I. Roumeliotis, Elodie Anthony, M. Pardo, Marianne Chasseriaud, P. Gestraud, Carine Ngo, Daphné Morel, Roman Chabanon, Jyoti S. Choudhary, E. Del Nery, Sophie Postel-Vinay, Annabelle Ballesta
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Whole-genome sequencing efforts has during the past decade unveiled the central role of genomic rearrangements—such as chromosomal inversions—in evolutionary processes, including local adaptation in a wide range of taxa. However, employment of reference genomes from distantly or even closely related species for mapping and the subsequent variant calling, can lead to errors and/or biases in the datasets generated for downstream analyses. Here, we capitalize on the recently generated chromosome-anchored genome assemblies for Arctic cod (Arctogadus glacialis), polar cod (Boreogadus saida), and Atlantic cod (Gadus morhua) to evaluate the extent and consequences of reference bias on population sequencing datasets (approx. 15-20x coverage) for both Arctic cod and polar cod. Our findings demonstrate that the choice of reference genome impacts population genetic statistics, including individual mapping depth, heterozygosity levels, and cross-species comparisons of nucleotide diversity (π) and genetic divergence (DXY). Further, it became evident that using a more distantly related reference genome can lead to inaccurate detection and characterization of chromosomal inversions, i.e., in terms of size (length) and location (position), due to inter-chromosomal reorganizations between species. Additionally, we observe that several of the detected species-specific inversions were split into multiple genomic regions when mapped towards a heterospecific reference. Inaccurate identification of chromosomal rearrangements as well as biased population genetic measures could potentially lead to erroneous interpretation of species-specific genomic diversity, impede the resolution of local adaptation, and thus, impact predictions of their genomic potential to respond to climatic and other environmental perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e083b1a379647c0ff911211cf65b6e9e0db94c8" target='_blank'>
              Reference genome bias in light of species-specific chromosomal reorganization and translocations
              </a>
            </td>
          <td>
            Marius F. Maurstad, S. Hoff, José Cerca, M. Ravinet, Ian R. Bradbury, K. Jakobsen, K. Præbel, S. Jentoft
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (pAML) is a heterogeneous malignancy driven by diverse cytogenetic mutations. While risk stratification improved by identifying cytogenetic lesions, prognostication remains inadequate with 30% of standard-risk patients experiencing relapse within 5 years. Single-cell RNA sequencing (scRNAseq) enabled the interrogation of malignant cell heterogeneity in pAML and characterization of the immune microenvironment. Herein we report the largest pAML scRNAseq analysis to date with 708,285 cells from 164 bone marrow biopsies of 95 patients and 11 healthy controls. We uncovered treatment-resistant (TR) subtypes of pAML specific to RUNX1-RUNX1T1, FLT3-ITD, and CBFB-MYH11 patients. The enrichment of TR subtype gene signatures on the TARGET pAML data supported an association with significantly poor outcomes. Intriguingly, in addition to leukemic stem cells, we identified mast cell-like pAML associated with treatment resistance and poor outcomes. Together, immature and mature pAML subtypes are promising biomarkers for identifying patients at increased risk of relapse within cytogenetic categories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70fa48d362939fe5d28efc31302329cc356df67f" target='_blank'>
              Single-cell analysis of pediatric acute myeloid leukemia samples uncovers treatment-resistant stem and mast cells
              </a>
            </td>
          <td>
            Denis Ohlstrom, Mojtaba Bakhtia, Hope L. Mumme, Marina E. Michaud, Franklin Chien, William Pilcher, Sarthak Satpathy, Sean C Jordan, Swati S. Bhasin, Manoj K. Bhasin
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Combined CDK4/6 inhibitor (CDK4/6i) and endocrine therapy significantly improves outcome for patients with estrogen receptor-positive (ER+) metastatic breast cancer, but drug resistance and thus disease progression inevitably occur. Herein, we aimed to identify genomic alterations associated with combined CDK4/6i and endocrine therapy resistance, and follow the levels of specific mutations in longitudinal circulating tumor DNA (ctDNA) for early detection of progression. From a cohort of 86 patients with ER+ metastatic breast cancer we performed whole exome sequencing or targeted sequencing of paired tumor (N = 8) or blood samples (N = 5) obtained before initiation of combined CDK4/6i and endocrine therapy and at disease progression. Mutations in oncogenic genes at progression were rare, while amplifications of growth-regulating genes were more frequent. The most frequently acquired alterations observed were PIK3CA and TP53 mutations and PDK1 amplification. Longitudinal ctDNA dynamics of mutant PIK3CA or private mutations revealed increased mutation levels at progression in 8 of 10 patients (80%). Impressively, rising levels of PIK3CA-mutated ctDNA were detected 4-17 months before imaging. Our data add to the growing evidence supporting longitudinal ctDNA analysis for real-time monitoring of CDK4/6i response and early detection of progression in advanced breast cancer. Further, our analysis suggests that amplification of growth-related genes may contribute to combined CDK4/6i and endocrine therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4f8f900b692158e74287adf0a693a5a4d08296" target='_blank'>
              Genomic alterations associated with resistance and circulating tumor DNA dynamics for early detection of progression on CDK4/6 inhibitor in advanced breast cancer.
              </a>
            </td>
          <td>
            Charlotte K Kindt, Carla L. Alves, S. Ehmsen, Amalie Kragh, T. Reinert, Marianne Vogsen, A. R. Kodahl, Jeanette D Rønlev, Dilan Ardik, Anna L Sørensen, Kirstine Evald, Mia L Clemmensen, J. Staaf, Henrik J Ditzel
          </td>
          <td>2024-08-11</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Intra-tumoral heterogeneity in pancreatic ductal adenocarcinoma (PDAC) is characterized by a balance between basal and classical epithelial cancer cell states, with basal dominance associating with chemoresistance and a dismal prognosis. Targeting oncogenic KRAS, the primary driver of pancreatic cancer, shows early promise in clinical trials but efficacy is limited by acquired resistance. Using genetically engineered mouse models and patient-derived xenografts, we find that basal PDAC cells are highly sensitive to KRAS inhibitors. Employing fluorescent and bioluminescent reporter systems, we longitudinally track cell-state dynamics in vivo and reveal a rapid, KRAS inhibitor-induced enrichment of the classical state. Lineage-tracing identifies these enriched classical PDAC cells to be a reservoir for disease relapse. Genetic ablation of the classical cell-state is synergistic with KRAS inhibition, providing a pre-clinical proof-of-concept for this therapeutic strategy. Our findings motivate combining classical-state directed therapies with KRAS inhibitors to deepen responses and counteract resistance in pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04662024790cb8192cdf2a7ce3c66046d31ffca3" target='_blank'>
              A classical epithelial state drives acute resistance to KRAS inhibition in pancreas cancer.
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Stefan R. Torborg, Jung Yun Kim, Olivera Grbovic-Huezo, Huijin Feng, Z. Tarcan, Hulya Sahin Ozkan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, I. I. C. Chio, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-07-08</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/79f8be9a3b2a2fa9867435314cacdcbe80a7eb4d" target='_blank'>
              Multiple mechanisms contribute to acquired TRAIL resistance in multiple myeloma
              </a>
            </td>
          <td>
            Fany V Ticona-Pérez, Xi Chen, A. Pandiella, Elena Díaz-Rodríguez
          </td>
          <td>2024-08-05</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is a highly prevalent and incurable form of cancer that arises from malignant plasma cells, with over 35,000 new cases diagnosed annually in the United States. While there are a growing number of approved therapies, MM remains incurable and nearly all patients will relapse and exhaust all available treatment options. Mechanisms for disease progression are unclear and in particular, little is known regarding the role of long non-coding RNAs (lncRNA) in mediating disease progression and response to treatment. In this study, we used transcriptome sequencing to compare newly diagnosed MM patients who had short progression- free survival (PFS) to standard first-line treatment (PFS < 24 months) to patients who had prolonged PFS (PFS > 24 months). We identified 157 differentially upregulated lncRNAs with short PFS and focused our efforts on characterizing the most upregulated lncRNA, LINC01432. We investigated LINC01432 overexpression and CRISPR/Cas9 knockdown in MM cell lines to show that LINC01432 overexpression significantly increases cell viability and reduces apoptosis, while knockdown significantly reduces viability and increases apoptosis, supporting the clinical relevance of this lncRNA. Next, we used individual-nucleotide resolution cross-linking immunoprecipitation with RT-qPCR to show that LINC01432 directly interacts with the RNA binding protein, CELF2. Lastly, we showed that LINC01432-targeted locked nucleic acid antisense oligonucleotides reduce viability and increases apoptosis. In summary, this fundamental study identified lncRNAs associated with short PFS to standard NDMM treatment and further characterized LINC01432, which inhibits apoptosis. Key points: lncRNA expression was found to be dysregulated in patients with short PFS to standard multiple myeloma therapy. LINC01432-bound CELF2 inhibits apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/847cba66ef27ec89828bb55803b54ea81c4c449e" target='_blank'>
              LINC01432 binds to CELF2 in newly diagnosed multiple myeloma promoting short progression-free survival to standard therapy
              </a>
            </td>
          <td>
            Richa Mishra, Prasanth Thunuguntla, Alani Perkin, Dhanusha Duraiyan, Katelyn Bagwill, Savannah Gonzales, Vanessa Brizuela, Steve Daly, Yoon Jae Chang, Mahdote Abebe, Yash Rajana, Kelly Wichmann, Catheryn Bolick, Jaiyana King, M. Fiala, J. Fortier, R. Jayasinghe, Mark Schroeder, Li Ding, Ravi Vij, Jessica M. Silva-Fisher
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b3ee353d402daa697bb21b6a4c992778ca5f0e4" target='_blank'>
              The role of AMPK in pancreatic cancer: from carcinogenesis to treatment.
              </a>
            </td>
          <td>
            Shahrzad Shahrokhi Nejad, S. Razi, N. Rezaei
          </td>
          <td>2024-06-26</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers.
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Abnormal transcription initiation from alternative first exon has been reported to promote tumorigenesis. However, the prevalence and impact of gene expression regulation mediated by alternative tandem transcription initiation were mostly unknown in cancer. Here, we developed a robust computational method to analyze alternative tandem transcription start site (TSS) usage from standard RNA sequencing data. Applying this method to pan-cancer RNA sequencing datasets, we observed widespread dysregulation of tandem TSS usage in tumors, many of which were independent of changes in overall expression level or alternative first exon usage. We showed that the dynamics of tandem TSS usage was associated with epigenomic modulation. We found that significant 5′ untranslated region shortening of gene TIMM13 contributed to increased protein production, and up-regulation of TIMM13 by CRISPR-mediated transcriptional activation promoted proliferation and migration of lung cancer cells. Our findings suggest that dysregulated tandem TSS usage represents an addtional layer of cancer-associated transcriptome alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e10e26b2a90c7cf14b59fe18386ef2bb8242323d" target='_blank'>
              Pan-cancer transcriptome analysis reveals widespread regulation through alternative tandem transcription initiation
              </a>
            </td>
          <td>
            Zhaozhao Zhao, Yu Chen, Xudong Zou, Limin Lin, Xiaolan Zhou, Xiaomeng Cheng, Guangrui Yang, Qiushi Xu, Lihai Gong, Lei Li, Ting Ni
          </td>
          <td>2024-07-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a9aeaed6ad9fc67eb22008185616631f4e626ac" target='_blank'>
              Inverse FASN and LDHA correlation drives metabolic resistance in breast cancer
              </a>
            </td>
          <td>
            Chiara Papulino, U. Chianese, Ahmad Ali, Gregorio Favale, C. Tuccillo, Fortunato Ciardiello, A. Di Mauro, Chiara Mignogna, Gerardo Ferrara, Alfredo Budillon, Wouter Leonard Megchelenbrink, Nunzio Del Gaudio, Mariarosaria Conte, Fabrizio Merciai, Pietro Campiglia, Lucia Altucci, V. Carafa, E. Sommella, R. Benedetti
          </td>
          <td>2024-07-24</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Simple Summary Neuregulin-1 (NRG1) is an important regulator of ErbB-mediated pathways involved in cancer development. Recently, there have been several studies analyzing NRG-1 gene fusions engaged in altering the dimerization of HER family proteins and the consecutive results of their activation in different types of cancer. Non-small cell lung cancer (NSCLC) patients can benefit from pan-HER inhibitors, and knowledge of NRG-1 fusions can help tailor the treatment to a specific group of patients. New drugs targeting cells with NRG-1 fusions are under clinical trials and show effectiveness in NSCLC treatment. Abstract Non-small cell lung cancer (NSCLC) presents a variety of druggable genetic alterations that revolutionized the treatment approaches. However, identifying new alterations may broaden the group of patients benefitting from such novel treatment options. Recently, the interest focused on the neuregulin-1 gene (NRG1), whose fusions may have become a potential predictive factor. To date, the occurrence of NRG1 fusions has been considered a negative prognostic marker in NSCLC treatment; however, many premises remain behind the targetability of signaling pathways affected by the NRG1 gene. The role of NRG1 fusions in ErbB-mediated cell proliferation especially seems to be considered as a main target of treatment. Hence, NSCLC patients harboring NRG1 fusions may benefit from targeted therapies such as pan-HER family inhibitors, which have shown efficacy in previous studies in various cancers, and anti-HER monoclonal antibodies. Considering the increased interest in the NRG1 gene as a potential clinical target, in the following review, we highlight its biology, as well as the potential clinical implications that were evaluated in clinics or remained under consideration in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302eaed4f1e7bdac2f2c9a0f86b9a6908f0e39da" target='_blank'>
              NRG1 Gene Fusions—What Promise Remains Behind These Rare Genetic Alterations? A Comprehensive Review of Biology, Diagnostic Approaches, and Clinical Implications
              </a>
            </td>
          <td>
            Tomasz Kucharczyk, M. Nicoś, Marek Kucharczyk, E. Kalinka
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cancer cells show remarkable plasticity and can switch lineages in response to the tumor microenvironment. Cellular plasticity drives invasiveness and metastasis and helps cancer cells to evade therapy by developing resistance to radiation and cytotoxic chemotherapy. Increased understanding of cell fate determination through epigenetic reprogramming is critical to discover how cancer cells achieve transcriptomic and phenotypic plasticity. Glioblastoma is a perfect example of cancer evolution where cells retain an inherent level of plasticity through activation or maintenance of progenitor developmental programs. However, the principles governing epigenetic drivers of cellular plasticity in glioblastoma remain poorly understood. Here, using machine learning (ML) we employ cross-patient prediction of transcript expression using a combination of epigenetic features (ATAC-seq, CTCF ChIP-seq, RNAPII ChIP-seq, H3K27Ac ChIP-seq, and RNA-seq) of glioblastoma stem cells (GSCs). We investigate different ML and deep learning (DL) models for this task and build our final pipeline using XGBoost. The model trained on one patient generalizes to another one suggesting that the epigenetic signals governing gene transcription are consistent across patients even if GSCs can be very different. We demonstrate that H3K27Ac is the epigenetic feature providing the most significant contribution to cross-patient prediction of gene expression. In addition, using H3K27Ac signals from patients-derived GSCs, we can predict gene expression of human neural crest stem cells suggesting a shared developmental epigenetic trajectory between subpopulations of these malignant and benign stem cells. Our cross-patient ML/DL models determine weighted patterns of influence of epigenetic marks on gene expression across patients with glioblastoma and between GSCs and neural crest stem cells. We propose that broader application of this analysis could reshape our view of glioblastoma tumor evolution and inform the design of new epigenetic targeting therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f06de9705bc503e591aae9a45d9fab127b04d79" target='_blank'>
              Machine learning on multiple epigenetic features reveals H3K27Ac as a driver of gene expression prediction across patients with glioblastoma
              </a>
            </td>
          <td>
            Yusuke Suita, Hardy Bright, Yuan Pu, M. D. Toruner, Jordan Idehen, Nikos Tapinos, Ritambhara Singh
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Linking disease phenotypes with molecular targets is key to the rational design of treatment interventions. Resistance to the chemotherapeutic cisplatin is one of the major factors limiting the clinical utility of this therapy, which is central to the treatment of a variety of solid malignancies. In this study, we couple the upregulation of a chemoresistant factor, the glutamate-cystine antiporter SLC7A11, with the addiction of cisplatin-resistant cancer cells to extracellular glutamine. In doing so, we thus provide a putative biomarker for this acquired metabolic dependency of chemoresistance. Subsequently, we evaluate various therapeutic strategies to selectively kill SLC7A11high cisplatin-resistant cancer cells, identifying cross-resistance to ferroptosis-inducing compounds and hypersensitivity to glutaminase inhibitor CB-839. We identify enzymatic depletion of extracellular glutamine using the long-standing anti-leukemic therapy asparaginase (ASNase), which possesses glutaminase activity, as a potential approach, and show this can be successfully combined with cisplatin in cell models. In summary, this study mechanistically links an acquired metabolic dependency of chemoresistant cancer cells with a putative biomarker and provides a potentially actionable strategy to target these drug resistant cells warranting further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32c836fb51ec4a12788e118211188bb2a1ae2ab5" target='_blank'>
              Glutamine-addiction in cisplatin resistant cancer cells is mediated by SLC7A11 and can be targeted with asparaginase therapy
              </a>
            </td>
          <td>
            Jiantao Wang, Robert Strauss, Jiri Bartek, S. G. Rudd
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Cancer stem cells (CSCs) possess a significant ability to renew themselves, which gives them a strong capacity to form tumors and expand to encompass additional body areas. In addition, they possess inherent resistance to chemotherapy and radiation therapies used to treat many forms of cancer. Scientists have focused on investigating the signaling pathways that are highly linked to the ability of CSCs to renew themselves and maintain their stem cell properties. The pathways encompassed are Notch, Wnt/β-catenin, hedgehog, STAT3, NF-κB, PI-3K/Akt/mTOR, sirtuin, ALDH, MDM2, and ROS. Recent studies indicate that directing efforts towards CSC cells is essential in eradicating the overall cancer cell population and reducing the likelihood of tumor metastasis. As our comprehension of the mechanisms that stimulate CSC activity, growth, and resistance to chemotherapy advances, the discovery of therapeutic drugs specifically targeting CSCs, such as small-molecule compounds, holds the potential to revolutionize cancer therapy. This review article examines and analyzes the novel anti-CSC compounds that have demonstrated effective and selective targeting of pathways associated with the renewal and stemness of CSCs. We also discussed their special drug metabolism and absorption mechanisms. CSCs have been the subject of much study in cancer biology. As a possible treatment for malignancies, small-molecule drugs that target CSCs are gaining more and more attention. This article provides a comprehensive review of the current state of key small-molecule compounds, summarizes their recent developments, and anticipates the future discovery of even more potent and targeted compounds, opening up new avenues for cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c2ec47520bdf5746027b1d46298064477df151" target='_blank'>
              Novel Anti-Cancer Stem Cell Compounds: A Comprehensive Review
              </a>
            </td>
          <td>
            Shanchun Guo, Shilong Zheng, Mingli Liu, Guangdi Wang
          </td>
          <td>2024-08-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1334e8d1bab52080f2aa5225537b4b61dc872565" target='_blank'>
              Single-cell RNA-seq reveals the transcriptional program underlying tumor progression and metastasis in neuroblastoma.
              </a>
            </td>
          <td>
            Zhe Nian, Dan Wang, Hao Wang, Wenxu Liu, Zhenyi Ma, Jie Yan, Yanna Cao, Jie Li, Qiang Zhao, Zhe Liu
          </td>
          <td>2024-07-17</td>
          <td>Frontiers of medicine</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Somatic mutation phasing informs our understanding of cancer-related events, like driver mutations. We generated linked-read whole genome sequencing data for 23 samples across disease stages from 14 multiple myeloma (MM) patients and systematically assigned somatic mutations to haplotypes using linked-reads. Here, we report the reconstructed cancer haplotypes and phase blocks from several MM samples and show how phase block length can be extended by integrating samples from the same individual. We also uncover phasing information in genes frequently mutated in MM, including DIS3, HIST1H1E, KRAS, NRAS, and TP53, phasing 79.4% of 20,705 high-confidence somatic mutations. In some cases, this enabled us to interpret clonal evolution models at higher resolution using pairs of phased somatic mutations. For example, our analysis of one patient suggested that two NRAS hotspot mutations occurred on the same haplotype but were independent events in different subclones. Given sufficient tumor purity and data quality, our framework illustrates how haplotype-aware analysis of somatic mutations in cancer can be beneficial for some cancer cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2870d2d1518a03384eeca2d75b0b4975453b6c" target='_blank'>
              Somatic mutation phasing and haplotype extension using linked-reads in multiple myeloma
              </a>
            </td>
          <td>
            S. Foltz, Yize Li, Lijun Yao, Nadezhda V. Terekhanova, A. Weerasinghe, Qingsong Gao, Guanlan Dong, Moses Schindler, Song Cao, Hua Sun, R. Jayasinghe, Robert S Fulton, C. Fronick, Justin King, Daniel R. Kohnen, M. Fiala, Ken Chen, J. Dipersio, Ravi Vij, Li Ding
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Patients with class I V600EBRAF-mutant (MT) colorectal cancer (CRC) have a poor prognosis and their response to combined anti-BRAF/EGFR inhibition remains limited. There is clearly an unmet need in further understanding the biology of V600EBRAFMT CRC. We have used differential gene expression of BRAFWT and MT CRC cells to identify pathways underpinning BRAFMT CRC. We tested a panel of molecularly/genetically subtyped CRC cells for their sensitivity to the Unfolded Protein Response (UPR) activator BOLD-100. To identify novel combination strategies for BOLD-100, we performed RNA sequencing and high-throughput drug screening. Pathway enrichment analysis identified that the UPR and DNA repair pathways were significantly enriched in BRAFMT CRC. We found that oncogenic BRAF plays a crucial role in mediating response to BOLD-100. Using a systems biology approach, we identified V600EBRAFMT-dependent activation of the replication stress response kinase ATR as a key mediator of resistance to BOLD-100. Further analysis identified acute increases in BRAFMT-dependent-reactive oxygen species (ROS) levels following treatment with BOLD-100 that was demonstrated to promote ATR/CHK1 activation and apoptosis. Furthermore, activation of ROS/ATR/CHK1 following BOLD-100 was found to be mediated through the AHR transcription factor and CYP1A1. Importantly, pharmacological blockade of this resistance pathway with ATR inhibitors synergistically increased BOLD-100-induced apoptosis and growth inhibition in BRAFMT models. These results unveil possible novel therapeutic opportunity for BRAFMT CRC. Implications: BOLD-100 induces BRAFMT-dependent replication stress, and targeted strategies against replication stress (eg. by using ATR inhibitors) in combination with BOLD-100 may serve as a potential novel therapeutic strategy for clinically aggressive BRAFMT CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a0c515d9a8242560fe5d20e397d7ea5de365fc8" target='_blank'>
              Ruthenium drug BOLD-100 regulates BRAFMT colorectal cancer cell apoptosis through AhR/ROS/ATR signaling axis modulation.
              </a>
            </td>
          <td>
            Daryl Griffin, R. Carson, Deborah Y. Moss, T. Sessler, Deborah Lavin, Vijay K Tiwari, Shivaali Karelia, Richard Kennedy, K. Savage, S. McDade, Adam Carie, J. Pankovich, Mark Bazett, S. Van Schaeybroeck
          </td>
          <td>2024-07-31</td>
          <td>Molecular cancer research : MCR</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive and lethal form of brain cancer with few effective treatments. In this context, Zika virus has emerged as a promising therapeutic agent due to its ability to selectively infect and kill GBM cells. To elucidate these mechanisms and expand the landscape of oncolytic virotherapy, we pursued a transcriptomic meta-analysis comparing the molecular signatures of Zika infection in GBM and neuroblastoma (NBM). Over-representation analysis of dysregulated coding genes showed significant enrichment of tumor necrosis factor (TNF), NF-κB, and p53 signaling pathways. A refined list of long non-coding RNAs consistently dysregulated in Zika-infected GBMs was also developed. Functional review of these candidates revealed their potential regulatory role in Zika-mediated oncolysis. We performed validation of the less-researched targets in adult and pediatric GBM cell lines and found significant differential regulation, as predicted. Altogether, our results provide novel insights into the molecular mechanisms underlying the effect of Zika on GBM. We highlight potential therapeutic targets that could be further interrogated to improve the efficacy of tumor cell death and the utility of Zika as an adjuvant virotherapy for GBM and other related cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5fb2f82f19cb82c54f2e078dc5fd330417c5a7" target='_blank'>
              Transcriptomic Meta-analysis Identifies Long Non-Coding RNAs Mediating Zika’s Oncolytic Impact in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Youssef A. Kousa, Shriya Singh, Allison Horvath, Federica Tomasso, Javad Nazarian, Lisa Henderson, Tamer A. Mansour
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cells must adapt to environmental changes to maintain homeostasis. One of the most striking environmental adaptations is entry into hibernation during which core body temperature can decrease from 37°C to as low at 4°C. How mammalian cells, which evolved to optimally function within a narrow range of temperatures, adapt to this profound decrease in temperature remains poorly understood. In this study, we conducted the first genome-scale CRISPR-Cas9 screen in cells derived from Syrian hamster, a facultative hibernator, as well as human cells to investigate the genetic basis of cold tolerance in a hibernator and a non-hibernator in an unbiased manner. Both screens independently revealed glutathione peroxidase 4 (GPX4), a selenium-containing enzyme, and associated proteins as critical for cold tolerance. We utilized genetic and pharmacological approaches to demonstrate that GPX4 is active in the cold and its catalytic activity is required for cold tolerance. Furthermore, we show that the role of GPX4 as a suppressor of cold-induced cell death extends across hibernating species, including 13-lined ground squirrels and greater horseshoe bats, highlighting the evolutionary conservation of this mechanism of cold tolerance. This study identifies GPX4 as a central modulator of mammalian cold tolerance and advances our understanding of the evolved mechanisms by which cells mitigate cold-associated damage—one of the most common challenges faced by cells and organisms in nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181b39f570d649216102dc87d265ba81bbf8c6ba" target='_blank'>
              Multi-species genome-wide CRISPR screens identify GPX4 as a conserved suppressor of cold-induced cell death
              </a>
            </td>
          <td>
            Breanna Lam, Kathrin M. Kajderowicz, Heather R. Keys, Julian M. Roessler, Evgeni M. Frenkel, Adina Kirkland, Punam Bisht, Mohamed A. El-Brolosy, Rudolf Jaenisch, George W. Bell, Jonathan S Weissman, Eric C. Griffith, Siniša Hrvatin
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40cca13757b86bc8061147520627319e4d8e4c62" target='_blank'>
              Single-cell sequencing reveals novel proliferative cell type: a key player in renal cell carcinoma prognosis and therapeutic response
              </a>
            </td>
          <td>
            Bicheng Ye, Hongsheng Ji, Meng Zhu, Anbang Wang, Jingsong Tang, Yong Liang, Qing Zhang
          </td>
          <td>2024-07-25</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The expression of eukaryotic genes relies on the precise 3’-terminal cleavage and polyadenylation of newly synthesized pre-mRNA transcripts. Defects in these processes have been associated with various diseases, including cancer. While cancer-focused sequencing studies have identified numerous driver mutations in protein-coding sequences, noncoding drivers – particularly those affecting the cis-elements required for pre-mRNA cleavage and polyadenylation – have received less attention. Here, we systematically analysed cancer somatic mutations affecting 3’UTR polyadenylation signals using the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset. We found a striking enrichment of cancer-specific somatic mutations that disrupt strong and evolutionarily conserved cleavage and polyadenylation signals within tumour suppressor genes. Further bioinformatics and experimental analyses conducted as a part of our study suggest that these mutations have a profound capacity to downregulate the expression of tumour suppressor genes. Thus, this work uncovers a novel class of noncoding somatic mutations with significant potential to drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4b3938c7f84b24956b4223cbd287669eff1caf" target='_blank'>
              Recurrent disruption of tumour suppressor genes in cancer by somatic mutations in cleavage and polyadenylation signals
              </a>
            </td>
          <td>
            Yaroslav A. Kainov, F. Hamid, E. Makeyev
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Introduction: Circulating tumor cells (CTCs) represent the sub-population of cells shed into the vasculature and able to survive in the bloodstream, adhere to target vascular endothelial cells, and re-growth into the distant organ. CTCs have been found in the blood of most solid tumor-bearing patients and are used as a diagnostic marker. Although a complex genotypic and phenotypic signature characterizes CTCs, the ability to survive in suspension constitutes the most critical property, known as resistance to anoikis, e.g., the ability to resist apoptosis resulting from a loss of substrate adhesion. Here, we selected melanoma cells resistant to anoikis, and we studied their metabolic reprogramming, with the aim of identifying new metabolic targets of CTCs.Methods: Subpopulations of melanoma cells expressing a high anoikis-resistant phenotype were selected by three consecutive rocking exposures in suspension and studied for their phenotypic and metabolic characteristics. Moreover, we tested the efficacy of different metabolic inhibitors targeting glycolysis (2DG), LDHA (LDHA-in-3), the mitochondrial electron transport chain complex I (rotenone), glutaminase (BPTES), fatty acid transporter (SSO), fatty acid synthase (denifanstat), CPT1 (etomoxir), to inhibit cell survival and colony formation ability after 24 h of rocking condition.Results:Anoikis-resistant cells displayed higher ability to grow in suspension on agarose-covered dishes respect to control cells, and higher cell viability and colony formation capability after a further step in rocking condition. They showed also an epithelial-to-mesenchymal transition associated with high invasiveness and a stemness-like phenotype. Anoikis-resistant melanoma cells in suspension showed a metabolic reprogramming from a characteristic glycolytic metabolism toward a more oxidative metabolism based on the use of glutamine and fatty acids, while re-adhesion on the dishes reversed the metabolism to glycolysis. The treatment with metabolic inhibitors highlighted the effectiveness of rotenone, BPTES, SSO, and etomoxir in reducing the viability and the colony formation ability of cells capable of surviving in suspension, confirming the dependence of their metabolism on oxidative phosphorylation, using glutamine and fatty acids as the most important fuels.Discussion: This finding opens up new therapeutic strategies based on metabolic inhibitors of glutaminase and fatty acid oxidation for the treatment of CTCs and melanoma metastases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a95e52231b69241d2e256add02fe49ee1f3ba1d9" target='_blank'>
              The critical role of glutamine and fatty acids in the metabolic reprogramming of anoikis-resistant melanoma cells
              </a>
            </td>
          <td>
            S. Peppicelli, T. Kersikla, G. Menegazzi, E. Andreucci, J. Ruzzolini, C. Nediani, F. Bianchini, L. Calorini, Valentina Audrito, Joanna Kopecka, Alessandra Fiore, Huakan Zhao
          </td>
          <td>2024-08-08</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d47d5363206f0a4fb9e459a4bf61f772ff9c57" target='_blank'>
              Overexpression of REC8 induces aberrant gamete meiotic division and contributes to AML pathogenesis - a multiplexed microarray analysis and mendelian randomization study.
              </a>
            </td>
          <td>
            Wenxi Hua, J. Qi, Meng Zhou, Shiyu Han, Xiao-Yun Xu, Jinwen Su, Tingting Pan, Depei Wu, Yue Han
          </td>
          <td>2024-07-16</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary With advanced bioinformatic and sequencing technologies, many non-coding RNAs have been found to harbor non-canonical open reading frames (ncORFs). Some studies suggest that many ncORF-encoded microproteins or micropeptides possess important roles in cancer biology, yet the functional roles of these ncORF-encoded peptides are mostly unknown. The purpose of this review is to provide a comprehensive overview of existing knowledge on ncORF-encoded microproteins in cancer biology and highlight their roles as potential prognostic and therapeutic targets in cancer. Abstract Cancer develops from abnormal cell growth in the body, causing significant mortalities every year. To date, potent therapeutic approaches have been developed to eradicate tumor cells, but intolerable toxicity and drug resistance can occur in treated patients, limiting the efficiency of existing treatment strategies. Therefore, searching for novel genes critical for cancer progression and therapeutic response is urgently needed for successful cancer therapy. Recent advances in bioinformatics and proteomic techniques have allowed the identification of a novel category of peptides encoded by non-canonical open reading frames (ncORFs) from historically non-coding genomic regions. Surprisingly, many ncORFs express functional microproteins that play a vital role in human cancers. In this review, we provide a comprehensive description of different ncORF types with coding capacity and technological methods in discovering ncORFs among human genomes. We also summarize the carcinogenic role of ncORFs such as pTINCR and HOXB-AS3 in regulating hallmarks of cancer, as well as the roles of ncORFs such as HOXB-AS3 and CIP2A-BP in cancer diagnosis and prognosis. We also discuss how ncORFs such as AKT-174aa and DDUP are involved in anti-cancer drug response and the underestimated potential of ncORFs as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d95f26adc077c4e04bfda2c2e30184b8f21906d" target='_blank'>
              Exploring the Dark Matter of Human Proteome: The Emerging Role of Non-Canonical Open Reading Frame (ncORF) in Cancer Diagnosis, Biology, and Therapy
              </a>
            </td>
          <td>
            Anni Ge, Curtis Chan, Xiaolong Yang
          </td>
          <td>2024-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pyrimidine nucleotide biosynthesis is a druggable metabolic dependency of cancer cells, and chemotherapy agents targeting pyrimidine metabolism are the backbone of treatment for many cancers. Dihydroorotate dehydrogenase (DHODH) is an essential enzyme in the de novo pyrimidine biosynthesis pathway that can be targeted by clinically approved inhibitors. However, despite robust preclinical anticancer efficacy, DHODH inhibitors have shown limited single-agent activity in phase 1 and 2 clinical trials. Therefore, novel combination therapy strategies are necessary to realize the potential of these drugs. To search for therapeutic vulnerabilities induced by DHODH inhibition, we examined gene expression changes in cancer cells treated with the potent and selective DHODH inhibitor brequinar (BQ). This revealed that BQ treatment causes upregulation of antigen presentation pathway genes and cell surface MHC class I expression. Mechanistic studies showed that this effect is (1) strictly dependent on pyrimidine nucleotide depletion, (2) independent of canonical antigen presentation pathway transcriptional regulators, and (3) mediated by RNA polymerase II elongation control by positive transcription elongation factor B (P-TEFb). Furthermore, BQ showed impressive single-agent efficacy in the immunocompetent B16F10 melanoma model, and combination treatment with BQ and dual immune checkpoint blockade (anti-CTLA-4 plus anti-PD-1) significantly prolonged mouse survival compared to either therapy alone. Our results have important implications for the clinical development of DHODH inhibitors and provide a rationale for combination therapy with BQ and immune checkpoint blockade.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b0c0126387b9def9714026a2af50898ea415781" target='_blank'>
              DHODH inhibition enhances the efficacy of immune checkpoint blockade by increasing cancer cell antigen presentation
              </a>
            </td>
          <td>
            Nick Mullen, Surendra K. Shukla, Ravi Thakur, S. S. Kollala, Dezhen Wang, Nina V. Chaika, Juan F Santana, William R Miklavcic, Drew A. LaBreck, J. Mallareddy, David H. Price, A. Natarajan, Kamiya Mehla, David B. Sykes, Michael A. Hollingsworth, Pankaj K Singh
          </td>
          <td>2024-07-08</td>
          <td>eLife</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Mutually exclusive loss-of-function alterations in gene pairs are those that occur together less frequently than may be expected and may denote a synthetically lethal relationship (SSL) between the genes. SSLs can be exploited therapeutically to selectively kill cancer cells. Here, we analysed mutation, copy number variation, and methylation levels in samples from The Cancer Genome Atlas, using the hypergeometric and the Poisson binomial tests to identify mutually exclusive inactivated genes. We focused on gene pairs where one is an inactivated tumour suppressor and the other a gene whose protein product can be inhibited by known drugs. This provided an abundance of potential targeted therapeutics and repositioning opportunities for several cancers. These data are available on the MexDrugs website, https://bioinformaticslab.sussex.ac.uk/mexdrugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7250b34d6949747e0dee284668cf7df5c1a39253" target='_blank'>
              Identifying actionable synthetically lethal cancer gene pairs using mutual exclusivity.
              </a>
            </td>
          <td>
            Sarah K. Wooller, Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2024-07-08</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA methylation is traditionally associated with gene silencing, but its causal relationship and role in shaping cell fate decisions still need to be fully elucidated. Here, we conducted a genome-wide analysis to investigate the relationship between DNA methylation and gene expression at gene regulatory regions in human immune cells. By utilizing CRISPR-dCas9 DNA methylation editing tools, we successfully established a cause-and-effect relationship between the methylation levels of the promoter of the Interleukin1-receptor antagonist (IL1RN) gene and its expression. Notably, we observed that modifying the DNA methylation status of the IL1RN promoter is sufficient to alter the acquisition of the human myeloid cell fate and change the cellular response to inflammatory stimuli, resulting in abnormal cytokine release and distinctive capacity to support cancer growth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0466661c0180cda161896839ec2c88a8975d7da9" target='_blank'>
              Modulating immune cell fate and inflammation through CRISPR-mediated DNA methylation editing
              </a>
            </td>
          <td>
            Gemma Valcárcel, Anna V. López-Rubio, Aleksey Lazarenkov, Clara Berenguer, Josep Calafell, J. Rodríguez-Ubreva, E. Ballestar, J. L. Sardina
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="RNAs undergo a complex choreography of metabolic processes in human cells that are regulated by thousands of RNA-associated proteins. While the effects of individual RNA-associated proteins on RNA metabolism have been extensively characterized, the full complement of regulators for most RNA metabolic events remain unknown. Here we present a massively parallel RNA-linked CRISPR (ReLiC) screening approach to measure the responses of diverse RNA metabolic events to knockout of 2,092 human genes encoding all known RNA-associated proteins. ReLiC screens highlight modular interactions between gene networks regulating splicing, translation, and decay of mRNAs. When combined with biochemical fractionation of polysomes, ReLiC reveals striking pathway-specific coupling between growth fitness and mRNA translation. Perturbing different components of the translation and proteostasis machineries have distinct effects on ribosome occupancy, while perturbing mRNA transcription leaves ribosome occupancy largely intact. Isoform-selective ReLiC screens capture differential regulation of intron retention and exon skipping by SF3b complex subunits. Chemogenomic screens using ReLiC decipher translational regulators upstream of mRNA decay and uncover a role for the ribosome collision sensor GCN1 during treatment with the anti-leukemic drug homoharringtonine. Our work demonstrates ReLiC as a versatile platform for discovering and dissecting regulatory principles of human RNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d61d936925a83dd1c8b255f5e6a3f00dce8861" target='_blank'>
              Decoding RNA Metabolism by RNA-linked CRISPR Screening in Human Cells
              </a>
            </td>
          <td>
            Patrick J. Nugent, Heungwon Park, Cynthia L. Wladyka, Katharine Y. Chen, Christine Bynum, Grace Quarterman, Andrew C. Hsieh, A. Subramaniam
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract The use of k-mers to capture genetic variation in bacterial genome-wide association studies (bGWAS) has demonstrated its effectiveness in overcoming the plasticity of bacterial genomes by providing a comprehensive array of genetic variants in a genome set that is not confined to a single reference genome. However, little attempt has been made to interpret k-mers in the context of genome rearrangements, partly due to challenges in the exhaustive and high-throughput identification of genome structure and individual rearrangement events. Here, we present GWarrange, a pre- and post-bGWAS processing methodology that leverages the unique properties of k-mers to facilitate bGWAS for genome rearrangements. Repeat sequences are common instigators of genome rearrangements through intragenomic homologous recombination, and they are commonly found at rearrangement boundaries. Using whole-genome sequences, repeat sequences are replaced by short placeholder sequences, allowing the regions flanking repeats to be incorporated into relatively short k-mers. Then, locations of flanking regions in significant k-mers are mapped back to complete genome sequences to visualise genome rearrangements. Four case studies based on two bacterial species (Bordetella pertussis and Enterococcus faecium) and a simulated genome set are presented to demonstrate the ability to identify phenotype-associated rearrangements. GWarrange is available at https://github.com/DorothyTamYiLing/GWarrangehttps://github.com/DorothyTamYiLing/genome_rearrangement.git.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42162bc2b2f883e8a6834a9dc952dc10d3392ba" target='_blank'>
              GWarrange: a pre- and post- genome-wide association studies pipeline for detecting phenotype-associated genome rearrangement events
              </a>
            </td>
          <td>
            Yi Ling Tam, Sarah Cameron, Andrew Preston, Lauren A. Cowley
          </td>
          <td>2024-07-01</td>
          <td>Microbial Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Recurrent mutations in serine / arginine-rich splicing factor 2 (SRSF2), particularly the proline-to-histidine substitution at position 95 (P95H), have been proposed to drive neoplastic diseases. SRSF2 plays pivotal roles in pre-mRNA processing and gene transcription. However, the precise impact of these diverse functions of SRSF2 on cancer cell behaviour remains unclear. Here, we show that deletion or homozygous P95H mutation of SRSF2 both cause extensive DNA damage leading to cell cycle arrest in vitro and in vivo, regardless of cell genotype. We mechanistically demonstrate that SRSF2 is required for efficient bi-directional transcription of DNA replication and repair genes, independent of its function in splicing. In contrast, SRSF2 haploinsufficiency induces DNA damage without halting the cell cycle in cancer cells. Inducing the heterozygous Srsf2 P95H mutation leads to clonal expansion of epidermal cells in mouse skin, but tumor formation is inhibited following exposure to carcinogens. To survive carcinogen treatment, cells containing the Srsf2 P95H mutation undergo substantial transcription rewiring that restores bi-directional gene expression. Thus, our study reveals crucial roles of SRSF2 beyond splicing and underscores its importance in regulating transcription to orchestrate the cell cycle along with the DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32907ead7270689c743bba801c421a14be049a66" target='_blank'>
              SRSF2 transcriptional function maintains genome integrity during cell division
              </a>
            </td>
          <td>
            Rebecca E. Wagner, Leonie Arnetzl, Thiago Britto-Borges, Anke Heit-Mondrzyk, Ali Bakr, Etienne Sollier, Jasper Panten, Sylvain Delaunay, Daniela Sohn, P. Schmezer, D. T. Odom, Karin Müller-Decker, C. Plass, Christoph Dieterich, P. Lutsik, Susanne Bornelöv, Michaela Frye
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c785685c384edfe64958536f2b96ef40bece9a9d" target='_blank'>
              Let’s make it personal: CRISPR tools in manipulating cell death pathways for cancer treatment
              </a>
            </td>
          <td>
            Mobina Bayat, J. S. Nahand
          </td>
          <td>2024-07-29</td>
          <td>Cell Biology and Toxicology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c5f226ca297006300a3de7681b5b8c136da7920" target='_blank'>
              The chromatin accessibility and transcriptomic landscape of the aging mice cochlea and the identification of potential functional super-enhancers in age-related hearing loss
              </a>
            </td>
          <td>
            Chanyuan Zhang, Ting Yang, Xiaoqin Luo, Xiaoqing Zhou, Menglong Feng, Wei Yuan
          </td>
          <td>2024-07-04</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8e1be4efe22c9fbd98a98ceb3e0939364442527" target='_blank'>
              Genome-wide CRISPR screening identifies tyrosylprotein sulfotransferase-2 as a target for augmenting anti-PD1 efficacy
              </a>
            </td>
          <td>
            Yumi Oh, Sujeong Kim, Yunjae Kim, Hyun Kim, Dongjun Jang, Seungjae Shin, Soo Jin Lee, Jiwon Kim, Sang Eun Lee, Jaeik Oh, Yoojin Yang, Dohee Kim, Hae-Rim Jung, Sangjin Kim, Jihui Kim, K. Min, Beomki Cho, Hoseok Seo, Dohyun Han, Hansoo Park, Sung-Yup Cho
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Due to the heterogeneity of cancer, precision medicine has been a major challenge for cancer treatment. Determining medication regimens based on patient genotypes has become a research hotspot in cancer genomics. In this study, we aim to identify key biomarkers for targeted therapies based on single nucleotide variants (SNVs) and copy number variants (CNVs) of genes. The experiment is carried out on 7 cancers on the Encyclopedia of Cancer Cell Lines (CCLE) dataset. Considering the high mutability of driver genes which result in abundant mutated samples, the effect of data sparsity can be eliminated to a large extent. Therefore, we focus on discovering the relationship between driver mutation patterns and three measures of drug response, namely area under the curve (AUC), half maximal effective concentration (EC50), and log2-fold change (LFC). First, multiple statistical methods are applied to assess the significance of difference in drug response between sample groups. Next, for each driver gene, we analyze the extent to which its mutations can affect drug response. Based on the results of multiple hypothesis tests and correlation analyses, our main findings include the validation of several known drug response biomarkers such as BRAF, NRAS, MAP2K1, MAP2K2, and CDKN2A, as well as genes with huge potential to infer drug responses. It is worth emphasizing that we identify a list of genes including SALL4, B2M, BAP1, CCDC6, ERBB4, FOXA1, GRIN2A, and PTPRT, whose impact on drug response spans multiple cancers and should be prioritized as key biomarkers for targeted therapies. Furthermore, based on the statistical p-values and correlation coefficients, we construct gene-drug sensitivity maps for cancer drug recommendation. In this work, we show that driver mutation patterns could be used to tailor therapeutics for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa993499162c59f7b76c5c6ffab6d2e07409c25b" target='_blank'>
              Single nucleotide and copy number variants of cancer driver genes inform drug response in multiple cancers
              </a>
            </td>
          <td>
            Zeyuan Wang, Hong Gu, Pan Qin, Jia Wang
          </td>
          <td>2024-07-31</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Technological advancements in cell-free DNA (cfDNA) liquid biopsy have triggered exponential growth in numerous clinical applications. While cfDNA-based liquid biopsy has made significant strides in personalizing cancer treatment, the exploration and translation of epigenetics in liquid biopsy to clinical practice is still nascent. This comprehensive review seeks to provide a broad yet in-depth narrative of the present status of epigenetics in cfDNA liquid biopsy and its associated challenges. It highlights the potential of epigenetics in cfDNA liquid biopsy technologies with the hopes of enhancing its clinical translation. The momentum of cfDNA liquid biopsy technologies in recent years has propelled epigenetics to the forefront of molecular biology. We have only begun to reveal the true potential of epigenetics in both our understanding of disease and leveraging epigenetics in the diagnostic and therapeutic domains. Recent clinical applications of epigenetics-based cfDNA liquid biopsy revolve around DNA methylation in screening and early cancer detection, leading to the development of multi-cancer early detection tests and the capability to pinpoint tissues of origin. The clinical application of epigenetics in cfDNA liquid biopsy in minimal residual disease, monitoring, and surveillance are at their initial stages. A notable advancement in fragmentation patterns analysis has created a new avenue for epigenetic biomarkers. However, the widespread application of cfDNA liquid biopsy has many challenges, including biomarker sensitivity, specificity, logistics including infrastructure and personnel, data processing, handling, results interpretation, accessibility, and cost effectiveness. Exploring and translating epigenetics in cfDNA liquid biopsy technology can transform our understanding and perception of cancer prevention and management. cfDNA liquid biopsy has great potential in precision oncology to revolutionize conventional ways of early cancer detection, monitoring residual disease, treatment response, surveillance, and drug development. Adapting the implementation of liquid biopsy workflow to the local policy worldwide and developing point-of-care testing holds great potential to overcome global cancer disparity and improve cancer outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa7f61a66ee2f452017bc6813b88e2fe69622c60" target='_blank'>
              Translation of Epigenetics in Cell-Free DNA Liquid Biopsy Technology and Precision Oncology
              </a>
            </td>
          <td>
            Wan Ying Tan, Snigdha Nagabhyrava, Olivia Ang-Olson, Paromita Das, Luisa Ladel, B. Sailo, Linda He, Anup Sharma, Nita Ahuja
          </td>
          <td>2024-06-27</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) leads as a primary cause of cancer-related premature mortality in Western populations. This study leverages cutting-edge gene-expression-profiling technologies to perform an in-depth molecular characterization of NSCLC specimens, with the objective of uncovering tumor-specific genomic alterations. By employing DNA microarray analysis, our research aims to refine the classification of NSCLC for early detection, guide molecular-targeted treatment approaches, enhance prognostication, and broaden the scientific understanding of the disease’s biology. We identified widespread genomic abnormalities in our samples, including the recurrent loss of chromosomal regions 3p, 5q, 13q, and 21q and the gain of 12p. Furthermore, utilizing Metascape for bioinformatic analysis revealed critical biological pathways disrupted in NSCLC, offering promising leads for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489eff9e89a558831da14bfa1d262e700e732bd4" target='_blank'>
              Specific Deletions of Chromosomes 3p, 5q, 13q, and 21q among Patients with G2 Grade of Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Agata Kolecka-Bednarczyk, M. Frydrychowicz, B. Budny, M. Ruciński, Claudia Dompe, Piotr Gabryel, B. Płachno, M. Ruchała, K. Ziemnicka, Paweł Zieliński, Joanna Budna-Tukan
          </td>
          <td>2024-08-08</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="BACKGROUND/AIM
There are no established biomarkers for immune checkpoint inhibitors (ICI) in colorectal cancer (CRC) with microsatellite stability (MSS) or proficient mismatch repair (pMMR). Therefore, this study aimed to identify biomarkers for ICI benefit in patients with pMMR by analyzing the down-regulated DNA repair-related genes involved in highly immunogenic and immune responses, and comparing their expression levels and clinical features.


MATERIALS AND METHODS
Mismatch repair (MMR), tumor-infiltrating lymphocytes (TIL), and tumor mutation burden (TMB) were evaluated in 13 CRC cases and mRNA expression levels of 95 DNA repair-related genes were measured. DNA repair-related genes with reduced mRNA expression in the high immunogenicity and high immune response groups were identified. Then, the mRNA expression levels of the identified DNA repair-related genes were measured in 135 patients with CRC. Hierarchical cluster analysis was performed using the mRNA expression levels to compare the clinicopathological characteristics of each cluster.


RESULTS
ATR, LIG4, and RAD52 mRNA levels were significantly down-regulated in the high immunogenicity group. GADD45B, SMUG1, and XRCC6 mRNA levels were significantly down-regulated in the high immune response group. Cases in the cluster with reduced mRNA expression of the six genes were pMMR cases. CD8 mRNA expression level was higher in this cluster than in the other clusters.


CONCLUSION
Decreased mRNA expression levels of ATR, LIG4, RAD52, GADD45B, SMUG1, and XRCC6 genes were associated with high cytotoxic T cell and TMB levels, suggesting that these genes could serve as biomarkers for ICI efficacy in pMMR cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb38d024ab94655fe817bc6cd4f0bb4ad2d969e2" target='_blank'>
              Identification of DNA Repair-related Genes as Biomarkers Reflecting MSI, TMB, and TIL in Colorectal Cancer.
              </a>
            </td>
          <td>
            Junya Fukuda, Tomoya Sudo, Maako Kikuchi, Akihiko Kawahara, Hirona Shigyo, Yusuke Kawamoto, Satoshi Shimamura, Fumiki Koga, Takasi Noguchi, Hiroyuki Nakane, Takahiro Shigaki, Kenji Fujiyoshi, T. Yomoda, Naohiro Yoshida, Kenichi Koushi, Takefumi Yoshida, Jun Akiba, Fumihiko Fujita
          </td>
          <td>2024-07-26</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia expressing the gamma delta T cell receptor (yo T-ALL) is a poorly understood disease. We studied 200 children with yo T-ALL from 13 clinical study groups to understand the clinical and genetic features of this disease. We found age and genetic drivers were significantly associated with outcome. yo T-ALL diagnosed in children under three years of age was extremely high-risk and enriched for genetic alterations that result in both LMO2 activation and STAG2 inactivation. Mechanistically, using patient samples and isogenic cell lines, we show that inactivation of STAG2 profoundly perturbs chromatin organization by altering enhancer-promoter looping, resulting in deregulation of gene expression associated with T-cell differentiation. High throughput drug screening identified a vulnerability in DNA repair pathways arising from STAG2 inactivation, which can be targeted by Poly(ADP-ribose) polymerase (PARP) inhibition. These data provide a diagnostic framework for classification and risk stratification of pediatric yo T-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed40523605c31fb96201abbfd6abddb8593a044" target='_blank'>
              Biologic and clinical analysis of childhood gamma delta T-ALL identifies LMO2/STAG2 rearrangements as extremely high-risk.
              </a>
            </td>
          <td>
            Shunsuke Kimura, Chun Shik Park, Lindsey E Montefiori, Ilaria Iacobucci, Petri Polonen, Qingsong Gao, Elizabeth D Arnold, A. Attarbaschi, Anthony Brown, B. Buldini, Kenneth J Caldwell, Yunchao Chang, Chelsey Chen, Cheng Cheng, Zhong-Yi Cheng, John K. Choi, V. Conter, K. Crews, H. D. de Groot-Kruseman, Takao Deguchi, Mariko Eguchi, Hannah Elisa Muhle, S. Elitzur, G. Escherich, Burgess B Freeman, Zhaohui Gu, Katie Han, K. Horibe, Toshihiko Imamura, S. Jeha, Motohiro Kato, Kean-Hui Chiew, Tanya Khan, M. Kicinski, Stefan Kohrer, S. Kornblau, R. Kotecha, Chi-Kong Li, Yen-Chun Liu, F. Locatelli, S. Luger, E. Paietta, Atsushi Manabe, H. Marquart, R. Masetti, Mellissa Maybury, Pauline Mazilier, Jules P P Meijerink, Sharnise Mitchell, T. Miyamura, Andrew S Moore, Koichi Oshima, K. Pawińska-Wąsikowska, Rob Pieters, Mollie S Prater, Shondra M. Pruett-Miller, Ching-Hon Pui, Chunxu Qu, Michaela Reiterova, Noemi Reyes, K. Roberts, J. M. Rowe, A. Sato, Kjeld Schmiegelow, M. Schrappe, Shuhong Shen, S. Skoczeń, O. Spinelli, Jan Stary, M. Svatoň, Masatoshi Takagi, Junko Takita, Yanjing Tang, D. Teachey, Paul G Thomas, D. Tomizawa, J. Trka, Elena Varotto, T. Vincent, Jun J Yang, Allen E J Yeoh, Yinmei Zhou, Martin Zimmermann, H. Inaba, C. Mullighan
          </td>
          <td>2024-06-25</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4597b2c9168e5b3eb619c4010a4d29dad0a9c28b" target='_blank'>
              mTORC2-driven chromatin cGAS mediates chemoresistance through epigenetic reprogramming in colorectal cancer.
              </a>
            </td>
          <td>
            Guoqing Lv, Qian Wang, Lin Lin, Qiao Ye, Xi Li, Qian Zhou, Xiangzhen Kong, Hongxia Deng, Fuping You, Hebing Chen, Song Wu, Lin Yuan
          </td>
          <td>2024-07-30</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="INTRODUCTION
Advances in understanding of tumor biology shed light on hallmarks of cancer development and progression that include dysregulated DNA damage repair (DDR) machinery. Leveraging underlying tumor genomic instability and tumor specific defects in DDR, Poly (ADP-ribose) polymerases (PARP) inhibitors (PARPi) induced DNA damage emerges as a novel non-chemotherapy therapeutic opportunity. PARPis are currently approved in multiple tumor types, with the largest benefit seen in tumors with homologous recombination repair (HRR) deficiency, including germline and somatic mutations in BRCA1/2 genes (BRCA) and other pathway members such as PALB2 and Rad51c.


AREAS COVERED
This review article summarizes the current approval landscape and known and proposed mechanisms of resistance to PARPi. Further, therapeutic strategies to overcome PARPi resistance are discussed, including ongoing clinical trials.


EXPERT OPINION
PARPi have proven to be a safe and effective therapy and represent a cornerstone treatment across multiple solid tumor types. Elucidating innate and acquired mechanisms of resistance, coupled with the emergence of novel therapeutic options to capitalize on the activity of PARPi and prevent or reverse the acquisition of resistance, provides an opportunity to further expand the role of PARPi in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec7393511b29f158c45a3ee7ca1c8ffe71c659b" target='_blank'>
              Strategies for the prevention or reversal of PARP inhibitor resistance.
              </a>
            </td>
          <td>
            Zahi I. Mitri, Shaun M Goodyear, Gordon B. Mills
          </td>
          <td>2024-08-15</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Cytogenomic characterization is crucial for the classification and risk stratification of acute myeloid leukemia (AML), thereby facilitating therapeutic decision-making. We examined the clinical utility of optical genome mapping (OGM) in 159 AML patients (103 newly diagnosed and 56 refractory/relapsed), all of whom also underwent chromosomal banding analysis (CBA), fluorescence in situ hybridization, and targeted next-generation sequencing. OGM detected nearly all clinically relevant cytogenetic abnormalities that SCG identified with >99% sensitivity, provided the clonal burden was above 20%. OGM identified additional cytogenomic aberrations and/or provided information on fusion genes in 77 (48%) patients, including eight patients with normal karyotypes and four with failed karyotyping. The most common additional alterations identified by OGM included chromoanagenesis (n = 23), KMT2A partial tandem duplication (n = 11), rearrangements involving MECOM (n = 7), NUP98 (n = 2), KMT2A (n = 2), JAK2 (n = 2), and other gene fusions in 17 patients, with 10 showing novel fusion gene partners. OGM also pinpointed fusion genes in 17 (11%) patients where chromosomal rearrangements were concurrently detected by OGM and CBA. Overall, 24 (15%) aberrations were identified exclusively by OGM and had the potential to alter AML classification, risk stratification, and/or clinical trial eligibility. OGM emerges as a powerful tool for identifying fusion genes and detecting subtle or cryptic cytogenomic aberrations that may otherwise remain undetectable by CBA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d116f1be018b74ce05fc5532f2fa6f900955563" target='_blank'>
              Optical genome mapping improves the accuracy of classification, risk stratification, and personalized treatment strategies for patients with acute myeloid leukemia.
              </a>
            </td>
          <td>
            S. Loghavi, Qing Wei, F. Ravandi, Andres E. Quesada, M. Routbort, Shimin Hu, G. Toruner, Sa A Wang, Wei Wang, Roberto N. Miranda, Shaoying Li, Jie Xu, Courtney D. DiNardo, Naval G Daver, T. Kadia, G. Issa, H. Kantarjian, L. J. Medeiros, G. Tang
          </td>
          <td>2024-07-17</td>
          <td>American journal of hematology</td>
          <td>0</td>
          <td>224</td>
        </tr>

        <tr id="The Ten-Eleven Translocation (TET) family genes are implicated in a wide array of biological functions across various human cancers. Nonetheless, there is a scarcity of studies that comprehensively analyze the correlation between TET family members and the molecular phenotypes and clinical characteristics of different cancers. Leveraging updated public databases and employing several bioinformatics analysis methods, we assessed the expression levels, somatic variations, methylation levels, and prognostic values of TET family genes. Additionally, we explored the association between the expression of TET family genes and pathway activity, tumor microenvironment (TME), stemness score, immune subtype, clinical staging, and drug sensitivity in pan-cancer. Molecular biology and cytology experiments were conducted to validate the potential role of TET3 in tumor progression. Each TET family gene displayed distinct expression patterns across at least ten detected tumors. The frequency of Single Nucleotide Variant (SNV) in TET genes was found to be 91.24%, primarily comprising missense mutation types, with the main types of copy number variant (CNV) being heterozygous amplifications and deletions. TET1 gene exhibited high methylation levels, whereas TET2 and TET3 genes displayed hypomethylation in most cancers, which correlated closely with patient prognosis. Pathway activity analysis revealed the involvement of TET family genes in multiple signaling pathways, including cell cycle, apoptosis, DNA damage response, hormone AR, PI3K/AKT, and RTK. Furthermore, the expression levels of TET family genes were shown to impact the clinical staging of tumor patients, modulate the sensitivity of chemotherapy drugs, and thereby influence patient prognosis by participating in the regulation of the tumor microenvironment, cellular stemness potential, and immune subtype. Notably, TET3 was identified to promote cancer progression across various tumors, and its silencing was found to inhibit tumor malignancy and enhance chemotherapy sensitivity. These findings shed light on the role of TET family genes in cancer progression and offer insights for further research on TET3 as a potential therapeutic target for pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4df11d8b0544aeb2264d4da3fba3e8f78e1111" target='_blank'>
              Pan-cancer analyses reveal the molecular and clinical characteristics of TET family members and suggests that TET3 maybe a potential therapeutic target
              </a>
            </td>
          <td>
            Chunyan Zhang, Jie Zheng, Jin Liu, Yanxia Li, G. Xing, Shupeng Zhang, Hekai Chen, Jian Wang, Z. Shao, Yongyuan Li, Zhongmin Jiang, Yingzi Pan, Xiaozhi Liu, Ping Xu, Wenhan Wu
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="With age, hematopoietic stem cells can acquire somatic mutations in leukemogenic genes that confer a proliferative advantage in a phenomenon termed “clonal hematopoiesis of indeterminate potential” (CHIP). How these mutations confer a proliferative advantage and result in increased risk for numerous age-related diseases remains poorly understood. We conducted a multiracial meta-analysis of epigenome-wide association studies (EWAS) of CHIP and its subtypes in four cohorts (N=8196) to elucidate the molecular mechanisms underlying CHIP and illuminate how these changes influence cardiovascular disease risk. The EWAS findings were functionally validated using human hematopoietic stem cell (HSC) models of CHIP. A total of 9615 CpGs were associated with any CHIP, 5990 with DNMT3A CHIP, 5633 with TET2 CHIP, and 6078 with ASXL1 CHIP (P <1×10−7). CpGs associated with CHIP subtypes overlapped moderately, and the genome-wide DNA methylation directions of effect were opposite for TET2 and DNMT3A CHIP, consistent with their opposing effects on global DNA methylation. There was high directional concordance between the CpGs shared from the meta-EWAS and human edited CHIP HSCs. Expression quantitative trait methylation analysis further identified transcriptomic changes associated with CHIP-associated CpGs. Causal inference analyses revealed 261 CHIP-associated CpGs associated with cardiovascular traits and all-cause mortality (FDR adjusted p-value <0.05). Taken together, our study sheds light on the epigenetic changes impacted by CHIP and their associations with age-related disease outcomes. The novel genes and pathways linked to the epigenetic features of CHIP may serve as therapeutic targets for preventing or treating CHIP-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb071f9470db1b0d43e08fc0664a4d81582bdd97" target='_blank'>
              Epigenome-wide DNA Methylation Association Study of CHIP Provides Insight into Perturbed Gene Regulation
              </a>
            </td>
          <td>
            Daniel Levy, Sara Kirmani, T. Huan, Joseph Van Amburg, R. Joehanes, M. Uddin, N. Nguyen, Bing Yu, Jennifer A. Brody, M. Fornage, Jan Bressler, N. Sotoodehnia, David Ong, Fabio Puddu, James Floyd, Christie M. Ballantyne, B. Psaty, L. Raffield, Pradeep Natarajan, Karen Conneely, April P. Carson, Leslie Lange, Kendra Ferrier, N. HEARD-COSTA, Joanne M Murabito, A. Bick
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Cancer stem cells represent a resilient subset within the tumor microenvironment capable of differentiation, regeneration, and resistance to chemotherapeutic agents, often using dormancy as a shield. Their unique properties, including drug resistance and metastatic potential, pose challenges for effective targeting. These cells exploit certain metabolic processes for their maintenance and survival. One of these processes is autophagy, which generally helps in energy homeostasis but when hijacked by CSCs can help maintain their stemness. Thus, it is often referred as an Achilles heel in CSCs, as certain cancers tend to depend on autophagy for survival. Autophagy, while crucial for maintaining stemness in cancer stem cells (CSCs), can also serve as a vulnerability in certain contexts, making it a complex target for therapy. Regulators of autophagy like AMPK (5′ adenosine monophosphate-activated protein kinase) also play a crucial role in maintaining CSCs stemness by helping CSCs in metabolic reprogramming in harsh environments. The purpose of this review is to elucidate the interplay between autophagy and AMPK in CSCs, highlighting the challenges in targeting autophagy and discussing therapeutic strategies to overcome these limitations. This review focuses on previous research on autophagy and its regulators in cancer biology, particularly in CSCs, addresses the remaining unanswered questions, and potential targets for therapy are also brought to attention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/522385875fb4c3e94cc6d48f89e884f2a675c1e3" target='_blank'>
              Role of Autophagy and AMPK in Cancer Stem Cells: Therapeutic Opportunities and Obstacles in Cancer
              </a>
            </td>
          <td>
            Lochana Kovale, Manish Kumar Singh, Joungmok Kim, Joohun Ha
          </td>
          <td>2024-08-08</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e81b6b5c5ed75ea0aa53f16f320c4ca6faed6d1d" target='_blank'>
              Intricate effects of post-translational modifications in liver cancer: mechanisms to clinical applications
              </a>
            </td>
          <td>
            Yu Zhang, Weihao Xu, Chuanhui Peng, Shenli Ren, Chengpeng Zhang
          </td>
          <td>2024-07-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aa3ff406547a34fa6946daf5b9a8ed1bbbd6216" target='_blank'>
              Loss of the tumour suppressor LKB1/STK11 uncovers a leptin-mediated sensitivity mechanism to mitochondrial uncouplers for targeted cancer therapy
              </a>
            </td>
          <td>
            A. Angelopoulou, Giorgos Theocharous, Dimitrios Valakos, A. Polyzou, Sophia Magkouta, V. Myrianthopoulos, Sophia Havaki, Marco Fiorillo, Ioanna Tremi, Konstantinos Vachlas, Theodoros Nisotakis, Dimitris-Foivos Thanos, Anastasia Pantazaki, Dimitris Kletsas, Jiri Bartek, R. Petty, Dimitris-Foivos Thanos, R. McCrimmon, A. Papaspyropoulos, V. Gorgoulis
          </td>
          <td>2024-07-25</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Simple Summary This study explored how genetic changes influence treatment effectiveness and survival in patients with advanced pancreatic cancer. By examining tumors from 142 patients, researchers identified specific genetic mutations that can predict how well a patient responds to chemotherapy. Findings showed that mutations in certain genetic pathways are associated with longer survival and better treatment outcomes. This research highlights the importance of tailoring cancer treatment based on individual genetic profiles, potentially leading to more personalized and effective therapies for pancreatic cancer patients. Abstract (1) Background: Pancreatic ductal adenocarcinoma (PDAC) has low survival rates despite treatment advancements. Aim: This study aims to show how molecular profiling could possibly guide personalized treatment strategies, which may help improve survival outcomes in patients with PDAC. (2) Materials and Methods: A retrospective analysis of 142 PDAC patients from a single academic center was conducted. Patients underwent chemotherapy and next-generation sequencing for molecular profiling. Key oncogenic pathways were identified using the Reactome pathway database. Survival analysis was performed using Kaplan–Meier curves and Cox Proportional Hazards Regression. (3) Results: Patients mainly received FOLFIRINOX (n = 62) or gemcitabine nab-paclitaxel (n = 62) as initial chemotherapy. The median OS was 13.6 months. Longer median OS was noted in patients with NOTCH (15 vs. 12.3 months, p = 0.007) and KIT pathway mutations (21.3 vs. 12.12 months, p = 0.04). Combinatorial pathway analysis indicated potential synergistic effects on survival. In the PFS, PI3K pathway (6.6 vs. 5.7 months, p = 0.03) and KIT pathway (10.3 vs. 6.2 months, p = 0.03) mutations correlated with improved PFS within the gemcitabine nab-paclitaxel subgroup. (4) Conclusions: Molecular profiling could play a role in PDAC for predicting outcomes and responses to therapies like FOLFIRINOX and gemcitabine nab-paclitaxel. Integrating genomic data into clinical decision-making can benefit PDAC treatment, though further validation is needed to fully utilize precision oncology in PDAC management.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dba99f42629474b1327eb80add6a1bd44549f4f2" target='_blank'>
              Somatic Mutation Profile as a Predictor of Treatment Response and Survival in Unresectable Pancreatic Ductal Adenocarcinoma Treated with FOLFIRINOX and Gemcitabine Nab-Paclitaxel
              </a>
            </td>
          <td>
            Rodrigo Paredes de la Fuente, Santiago Sucre, Cristina Ponce, Ahmed Anwer Ali Rattani, M. L. Peters
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8867a66b53ced02c166da3c33e0be6474929e7d1" target='_blank'>
              Somatic CpG hypermutation is associated with mismatch repair deficiency in cancer.
              </a>
            </td>
          <td>
            Aidan Flynn, Sebastian M. Waszak, Joachim Weischenfeldt
          </td>
          <td>2024-07-18</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="microRNAs (miRNAs), tiny, non-coding RNA molecules, fine-tune the expression of target genes through interacting with mRNAs. These miRNAs are involved in a wide range of biological processes, encompassing cell division, death, blood cell production, and tumor development. When these miRNAs become dysfunctional, they can promote the invasion and spread of cancer cells in various human malignancies, including leukemia. Acute lymphoblastic leukemia (ALL), the preeminent malignancy affecting children, is a blood cancer marked by the uncontrollable growth of immature lymphoid cells that displace healthy blood precursors in the bone marrow. Despite a decline in ALL mortality rates over the past two decades, a significant proportion of deaths still results from a lack of effective diagnostic and prognostic markers that can guide treatment decisions and overcome drug resistance. The analysis of miRNA expression patterns in ALL could lead to more precise disease classification, earlier diagnosis, and better prognostic outcomes in the near future. The connection between miRNA dysfunction and the biology of ALL suggests that these molecules could represent promising therapeutic targets. Therefore, this review delves into the regulatory mechanisms of miRNAs in pediatric ALL, exploring how miRNA-based diagnostic, prognostic, and therapeutic strategies offer unique advantages and hold promise for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b51d3730adcae846d16d92db76f51f153c5c1d0" target='_blank'>
              Advancements in the Regulatory Role of microRNAs in Childhood Acute Lymphoblastic Leukemia: Mechanisms and Clinical Implications
              </a>
            </td>
          <td>
            Wei Deng
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd083194686b61ccc53ca0d4c394b67971b4cdb7" target='_blank'>
              Targeted gene therapy for cancer: the impact of microRNA multipotentiality
              </a>
            </td>
          <td>
            Nourhan A. Abou Madawi, Zeinab E Darwish, Enas M Omar
          </td>
          <td>2024-08-01</td>
          <td>Medical Oncology (Northwood, London, England)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Naïve pluripotent stem cells (nPSC) frequently undergo pathological and not readily reversible loss of DNA methylation marks at imprinted gene loci. This abnormality poses a hurdle for using pluripotent cell lines in biomedical applications and underscores the need to identify the causes of imprint instability in these cells. We show that nPSCs from inbred mouse strains exhibit pronounced strain-specific susceptibility to locus-specific deregulation of imprinting marks during reprogramming to pluripotency and upon culture with MAP kinase inhibitors, a common approach to maintain naïve pluripotency. Analysis of genetically highly diverse nPSCs from the Diversity Outbred (DO) stock confirms that genetic variation is a major determinant of epigenome stability in pluripotent cells. We leverage the variable DNA hypomethylation in DO lines to identify several trans-acting quantitative trait loci (QTLs) that determine epigenome stability at either specific target loci or genome-wide. Candidate factors encoded by two multi-target QTLs on chromosomes 4 and 17 suggest specific transcriptional regulators that contribute to DNA methylation maintenance in nPSCs. We propose that genetic variants represent candidate biomarkers to identify pluripotent cell lines with desirable properties and might serve as entry points for the targeted engineering of nPSCs with stable epigenomes. Highlights Naïve pluripotent stem cells from distinct inbred mouse strains exhibit variable DNA methylation levels at imprinted gene loci. The vulnerability of pluripotent stem cells to loss of genomic imprinting caused by MAP kinase inhibition strongly differs between inbred mouse strains. Genetically diverse pluripotent stem cell lines from Diversity Outbred mouse stock allow the identification of quantitative trait loci controlling DNA methylation stability. Genetic variants may serve as biomarkers to identify naïve pluripotent stem cell lines that are epigenetically stable in specific culture conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fcf4c9c2e254cef424889809e18c41add6362bb" target='_blank'>
              Genetic variation modulates susceptibility to aberrant DNA hypomethylation and imprint deregulation in naïve pluripotent stem cells
              </a>
            </td>
          <td>
            C. Parikh, RA Glenn, Y. Shi, K. Chatterjee, EE Swanzey, S. Singer, SC Do, Y. Zhan, Y. Furuta, M. Tahiliani, E. Apostolou, A. Polyzos, R. Koche, JG Mezey, T. Vierbuchen, M. Stadtfeld
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daaaeb917f1640a6de2396b29c1a5a0f942bd758" target='_blank'>
              Enhanced cancer cell proliferation and aggressive phenotype counterbalance in breast cancer with high BRCA1 gene expression.
              </a>
            </td>
          <td>
            Kohei Chida, M. Oshi, A. M. Roy, Takumi Sato, Maya Penelope Takabe, Li Yan, I. Endo, Kenichi Hakamada, K. Takabe
          </td>
          <td>2024-07-07</td>
          <td>Breast cancer research and treatment</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Transfer RNA (tRNA) modifications have emerged as critical posttranscriptional regulators of gene expression affecting diverse biological and disease processes. While there is extensive knowledge about the enzymes installing the dozens of post-transcriptional tRNA modifications – the tRNA epitranscriptome – very little is known about how metabolic, signaling, and other networks integrate to regulate tRNA modification levels. Here we took a comprehensive first step at understanding epitranscriptome regulatory networks by developing a high-throughput tRNA isolation and mass spectrometry-based modification profiling platform and applying it to a Pseudomonas aeruginosa transposon insertion mutant library comprising 5,746 strains. Analysis of >200,000 tRNA modification data points validated the annotations of predicted tRNA modification genes, uncovered novel tRNA-modifying enzymes, and revealed tRNA modification regulatory networks in P. aeruginosa. Platform adaptation for RNA-seq library preparation would complement epitranscriptome studies, while application to human cell and mouse tissue demonstrates its utility for biomarker and drug discovery and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cd74a5ca2ea04241d0e2f28ffe633a7113d027f" target='_blank'>
              tRNA modification profiling reveals epitranscriptome regulatory networks in Pseudomonas aeruginosa
              </a>
            </td>
          <td>
            Jingjing Sun, Junzhou Wu, Yifeng Yuan, Leon Fan, Wei Lin Patrina Chua, Yan Han Sharon Ling, S. Balamkundu, Dwija priya, Hazel Chay Suen Suen, V. de Crécy-Lagard, Agnieszka Dziergowska, Peter C. Dedon
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Therapy-induced senescence (TIS) represents a major cellular response to anticancer treatments. Both malignant and non-malignant cells in the tumor microenvironment undergo TIS and may be harmful for cancer patients since TIS cells develop a senescence-associated secretory phenotype (SASP) that can sustain tumor growth. The SASP also modulates anti-tumor immunity, although the immune populations involved and the final results appear to be context-dependent. In addition, senescent cancer cells are able to evade senescence growth arrest and to resume proliferation, likely contributing to relapse. So, research data suggest that TIS induction negatively affects therapy outcomes in cancer patients. In line with this, new interventions aimed at the removal of senescent cells or the reprogramming of their SASP, called senotherapy, have become attractive therapeutic options. To date, the lack of reliable, cost-effective, and easy-to-use TIS biomarkers hinders the application of recent anti-senescence therapeutic approaches in the clinic. Hence, the identification of biomarkers for the detection of TIS tumor cells and TIS non-neoplastic cells is a high priority in cancer research. In this review article, we describe the current knowledge about TIS, outline critical gaps in our knowledge, and address recent advances and novel approaches for the discovery of TIS biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80b8874610a137e048d67c39fb5ecaa332e59e90" target='_blank'>
              Therapy-Induced Senescence: Novel Approaches for Markers Identification
              </a>
            </td>
          <td>
            F. Pacifico, Fulvio Magni, Antonio Leonardi, E. Crescenzi
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Background: Actionable driver mutations account for 40–50% of NSCLC cases, and their identification clearly affects treatment choices and outcomes. Conversely, non-actionable mutations are genetic alterations that do not currently have established treatment implications. Among co-occurring alterations, the identification of concurrent actionable genomic alterations is a rare event, potentially impacting prognosis and treatment outcomes. Methods: We retrospectively evaluated the prevalence and patterns of concurrent driver genomic alterations in a large series of NSCLCs to investigate their association with clinicopathological characteristics, to assess the prognosis of patients whose tumor harbors concurrent alterations in the genes of interest and to explore their potential therapeutic implications. Results: Co-occurring driver alterations were identified in 26 out of 1520 patients with at least one gene alteration (1.7%). Within these cases, the incidence of concurrent actionable gene alterations was 39% (0.7% of the overall cohort). Among compound actionable gene mutations, EGFR was the most frequently involved gene (70%). The most frequent association was EGFR mutations with ROS1 rearrangement. Front-line targeted treatments were the preferred approach in patients with compound actionable mutations, with dismal median PFS observed (6 months). Conclusions: Advances in genomic profiling technologies are facilitating the identification of concurrent mutations. In patients with concurrent actionable gene alterations, integrated molecular and clinical data should be used to guide treatment decisions, always considering rebiopsy at the moment of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59022475d568d630ac2b88c69a13d9f08f1373da" target='_blank'>
              Co-Occurring Driver Genomic Alterations in Advanced Non-Small-Cell Lung Cancer (NSCLC): A Retrospective Analysis
              </a>
            </td>
          <td>
            I. Attili, Riccardo Asnaghi, D. Vacirca, Riccardo Adorisio, A. Rappa, Alberto Ranghiero, Mariano Lombardi, C. Corvaja, V. Fuorivia, Ambra Carnevale Schianca, P. Trillo Aliaga, G. Spitaleri, E. del Signore, J. Guarize, Lorenzo Spaggiari, E. Guerini-Rocco, Nicola Fusco, F. de Marinis, A. Passaro
          </td>
          <td>2024-07-31</td>
          <td>Journal of Clinical Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Due to the significant morbidity and mortality of hemoglobinopathies, curative options have long been pursued. The overall goal of gene therapy is to modify a patient's own hematopoietic stem cells to overcome the deleterious effects of the underlying genetic defect by gene addition, gene editing, or gene silencing. Gene addition incorporates genes with superior function than the abnormal gene; gene editing takes advantage of molecular tools such as zinc finger proteins, Transcription Activator-Like Effector Nucleases and Clustered Regularly Interspaced Short Palindromic Repeats coupled with Cas9 proteins (CRISPR-Cas9) which allow for sequence-specific breaks in DNA that disrupt gene function; and gene silencing suppresses gene expression by interference with mRNA transcription/protein translation or epigenetic modification. The majority of gene therapy strategies for hemoglobinopathies have targeted erythroid-specific BCL11A, a major regulator of fetal hemoglobin repression at the gamma-globin locus, in the normal fetal-to-adult hemoglobin switch that occurs shortly after birth. Other goals have involved the incorporation of anti-sickling globins, such as βT87Q or βAS3. Landmark clinical trials of gene therapy in transfusion-dependent thalassemia and sickle cell disease have shown remarkable efficacy and acceptable safety and culminated in recent regulatory approvals of gene therapy for both diseases in Europe and the United States.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f812c9e54de47feca0c8615f3dda20f25f85b41" target='_blank'>
              A Review of Gene Therapies for Hemoglobinopathies.
              </a>
            </td>
          <td>
            Boubini Jones-Wonni, Amar H Kelkar, Maureen O Achebe
          </td>
          <td>2024-08-15</td>
          <td>Hemoglobin</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Current treatments for KRAS-mutant colorectal cancers (CRCs) are often limited by cellular plasticity and rewiring responses. Here we describe a promising therapeutic strategy that simultaneously targets epigenetic and oncogenic signals. Specifically, we show that inhibitors of the histone methyltransferase, EZH2, synergize with various RAS pathway inhibitors and promote dramatic tumor regression in vivo. Together these agents cooperatively suppress WNT-driven transcription and drive CRCs into a more differentiated cell state by inducing the Groucho/TLE corepressor, TLE4, along with a network of WNT pathway inhibitors and intestinal differentiation proteins. However, these agents also induce the pro-apoptotic protein BMF, which subsequently kills these more differentiated cells. Accordingly, cell death can be prevented by activating β-catenin, blocking differentiation, or by ablating BMF expression. Collectively, these studies reveal a new therapeutic approach for treating KRAS-mutant CRCs and illustrate a critical convergence of EZH2 and RAS on oncogenic WNT signals, intestinal differentiation, and apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47a75ff581332b9384e613ce0d1c80e31add042e" target='_blank'>
              Epigenetic and oncogenic inhibitors cooperatively drive differentiation and kill KRAS-mutant colorectal cancers.
              </a>
            </td>
          <td>
            P. Loi, Amy E. Schade, Carrie L Rodriguez, Anjana Krishnan, N. Perurena, Van T M Nguyen, Yilin Xu, Marina Watanabe, Rachel A Davis, Alycia Gardner, Natalie F Pilla, Kaia Mattioli, Olesja Popow, Nuray Gunduz, T. Lannagan, Samantha Fitzgerald, Ewa T Sicinska, Jia-Ren Lin, William Tan, L. Brais, Kevin Haigis, M. Giannakis, K. Ng, Sandro Santagata, Kristian Helin, Owen J. Sansom, K. Cichowski
          </td>
          <td>2024-08-12</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="BACKGROUND
Prenatal folate supplementation has been consistently associated with a reduced risk of childhood lymphoblastic leukemia (ALL). Previous germline genetic studies examining the one carbon (folate) metabolism pathway were limited in sample size, scope, and population diversity, and led to inconclusive results.


METHODS
We evaluated whether ~2,900 single nucleotide polymorphisms (SNPs) within 46 candidate genes involved in the folate metabolism pathway influence the risk of childhood ALL, using genome-wide data from nine case-control-studies in the Childhood Cancer and Leukemia International Consortium (n=9,058 cases including 4,510 children of European ancestry, 3,018 Latinx, and 1,406 Asians, and 92,364 controls). Each study followed a standardized protocol for quality control and imputation of genome-wide data, and summary statistics were meta-analyzed for all children combined and by major ancestry group using METAL software.


RESULTS
None of the selected SNPs reached statistical significance, overall and for major ancestry groups (using adjusted Bonferroni p-value of 5x10-6 and less stringent p-value of 3.5x10-5 accounting for the number of "independent" SNPs). None of the 10 top (non-significant) SNPs and corresponding genes overlapped across ancestry groups.


CONCLUSION
This large meta-analysis of original data does not reveal associations between many common genetic variants in the folate metabolism pathway and childhood ALL in various ancestry groups.


IMPACT
Genetic variants in the folate pathway alone do not appear to substantially influence childhood ALL risk. Other mechanisms such as gene-folate interaction, DNA methylation or maternal genetic effects may explain the observed associations with self-reported prenatal folate intake.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a898e87257022dcda41ec075ff464556084a0e7a" target='_blank'>
              Folate metabolism and risk of childhood acute lymphoblastic leukemia: a genetic pathway analysis from the Childhood Cancer and Leukemia International Consortium.
              </a>
            </td>
          <td>
            C. Metayer, Logan G Spector, Michael E Scheurer, Soyoung Jeon, Rodney J. Scott, M. Takagi, J. Clavel, Atsushi Manabe, Xiaomei Ma, E. Hailu, Philip J Lupo, K. Urayama, A. Bonaventure, Motohiro Kato, Aline Meirhaeghe, Charleston W. K. Chiang, L. Morimoto, Joseph L Wiemels
          </td>
          <td>2024-06-21</td>
          <td>Cancer epidemiology, biomarkers & prevention : a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/761ad1cbab42eef53a71e8966883fae3c426f6a1" target='_blank'>
              Novel therapeutic agents in clinical trials: emerging approaches in cancer therapy
              </a>
            </td>
          <td>
            Deepak Chandra Joshi, Anurag Sharma, Sonima Prasad, Karishma Singh, Mayank Kumar, Kajal Sherawat, H. Tuli, Madhu Gupta
          </td>
          <td>2024-08-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Glioblastoma poses a formidable challenge among primary brain tumors: its tumorigenic stem cells, capable of self-renewal, proliferation, and differentiation, contribute substantially to tumor initiation and therapy resistance. These glioblastoma stem cells (GSCs), resembling conventional stem and progenitor cells, adopt pathways critical for tissue development and repair, promoting uninterrupted tumor expansion. Long non-coding RNAs (lncRNAs), a substantial component of the human transcriptome, have garnered considerable interest for their pivotal roles in normal physiological processes and cancer pathogenesis. They display cell- or tissue-specific expression patterns, and extensive investigations have highlighted their impact on regulating GSC properties and cellular differentiation, thus offering promising avenues for therapeutic interventions. Consequently, lncRNAs, with their ability to exert regulatory control over tumor initiation and progression, have emerged as promising targets for innovative glioblastoma therapies. This review explores notable examples of GSC-associated lncRNAs and elucidates their functional roles in driving glioblastoma progression. Additionally, we delved deeper into utilizing a 3D in vitro model for investigating GSC biology and elucidated four primary methodologies for targeting lncRNAs as potential therapeutics in managing glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04ed3ed8d6a1c37f2db5f6f03648c49bfe8a7955" target='_blank'>
              Glioblastoma stem cell long non-coding RNAs: therapeutic perspectives and opportunities
              </a>
            </td>
          <td>
            R. Hazra, Rinku Debnath, Arati Tuppad
          </td>
          <td>2024-07-02</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chimeric antigen receptor T (CAR-T) cells that are obtained from the specific patient, modified genetically ex vivo, and possess the remarkable capability to identify and eradicate targeted cancer cells. These modified cells are subsequently reintroduced into the patient, effectively treating blood cancer. CAR-T therapy is approved to be applied in leukemia due to its great clinical therapeutic effect on B cell hematological malignancies. However, solid tumors are more resistant to this therapy for many reasons. The abnormal vascular structure of solid tumors hampers CAR-T cell trafficking. Various kinds of immunosuppressive cells and chemicals in the tumor microenvironment (TME) accelerates CAR-T cell exhaustion, showing poor persistence in vivo. More and more researches have demonstrated that T cell fate is strongly associated with epigenetic regulation. Epigenetic modification is not a direct addition or deletion of DNA, but a reversible method including modifications on DNA and histones, and non-coding RNA (ncRNA)-mediated regulations. The change of epigenetic landscape in CAR-T cells largely determines the therapeutic performance in vivo. This research outlines three major barriers in CAR-T therapy, including T cell exhaustion, differentiation and infiltration. Additionally, the research elucidates several promising epigenetic reprogramming strategies to reduce CAR-T cells exhaustion, modulate the cell differentiation process, and enhance their infiltration into solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4165f9901246426b44e74fa80485927b8d025d28" target='_blank'>
              Epigenetic Strategies to Optimize CAR-T Therapy
              </a>
            </td>
          <td>
            Danyang Song
          </td>
          <td>2024-07-11</td>
          <td>Highlights in Science, Engineering and Technology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genome editing is poised to revolutionize treatment of genetic diseases, but poor understanding and control of DNA repair outcomes hinders its therapeutic potential. DNA repair is especially understudied in nondividing cells like neurons, which must withstand decades of DNA damage without replicating. This lack of knowledge limits the efficiency and precision of genome editing in clinically relevant cells. To address this, we used induced pluripotent stem cells (iPSCs) and iPSC-derived neurons to examine how postmitotic human neurons repair Cas9-induced DNA damage. We discovered that neurons can take weeks to fully resolve this damage, compared to just days in isogenic iPSCs. Furthermore, Cas9-treated neurons upregulated unexpected DNA repair genes, including factors canonically associated with replication. Manipulating this response with chemical or genetic perturbations allowed us to direct neuronal repair toward desired editing outcomes. By studying DNA repair in postmitotic human cells, we uncovered unforeseen challenges and opportunities for precise therapeutic editing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6434cf3661015c386f770a07ae2b52c8bf820f87" target='_blank'>
              Neuronal DNA repair reveals strategies to influence CRISPR editing outcomes
              </a>
            </td>
          <td>
            Gokul N. Ramadoss, Samali J Namaganda, Jennifer R Hamilton, Rohit Sharma, Karena G Chow, Bria L Macklin, Mengyuan Sun, Jia-Cheng Liu, Christof Fellmann, Hannah L. Watry, Julianne Jin, Barbara S Perez, Cindy R Sandoval Espinoza, Madeline Matia, Serena H Lu, Luke M. Judge, A. Nussenzweig, Britt Adamson, Niren Murthy, Jennifer A. Doudna, Martin Kampmann, Bruce R. Conklin
          </td>
          <td>2024-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Multidrug resistance (MDR) frequently occurs during cancer therapy and remains a major obstacle for the cure of most cancers(1, 2). Drug resistance could exist intrinsically or be acquired by drug treatment(3–5), yet factors that regulate the resistance remain elusive. Here, we show that most anticancer drugs damage neosynthesized proteins prior to reaching their canonic targets and elicit profound cytotoxicity, which is largely compensated by protein damage response (PDR). We demonstrate that the PDR includes damage recognition and clearance that are mainly mediated by ubiquitin and proteasome systems, although some other factors, including cellular ATP levels and proliferation status, are also involved. We show that cancer stem cells (CSCs), which have lower protein synthesis, and drug resistance acquired cells (DRAC), which have higher proteasome activity, are more resistant than other cells. We further demonstrate that ATP promotes protein synthesis and suppresses proteasome activity, thus, increasing mitochondrial ATP production by PDK1 inhibition and using proteasome inhibitor to block protein damage clearance render CSCs and DRACs more vulnerable to anticancer drugs. Thus, patients with drug-resistant cancers and treatment-naïve patients with low ATP levels and/or high proteasome activity can be identified and subtyped, and therapies containing PDK1-I and/or proteasome-I may be effective options for these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7e9626795c580f97c2519d1342441e9145cbcfc" target='_blank'>
              A critical role of protein damage response in mediating cancer drug resistance
              </a>
            </td>
          <td>
            Fangyuan Shao, Jie Li, Hao Xiao, Ling Li, Bo Li, YuJun Chen, Xiangpeng Chu, Maoxin Ran, Dongyang Tang, Yuzhong Peng, Yujian Huang, Lijian Wang, Yanxia Shi, Nan Shao, Kai Miao, Changhua Zhang, Ying Lin, Jun Yan, Kin Yip Tam, Xiaoling Xu, Chu-Xia Deng
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Cancer, a multifactorial disease characterized by uncontrolled cellular proliferation, remains a global health challenge with significant morbidity and mortality. Genomic and molecular aberrations, coupled with environmental factors, contribute to its heterogeneity and complexity. Chemotherapeutic agents like doxorubicin (Dox) have shown efficacy against various cancers but are hindered by dose-dependent cytotoxicity, particularly on vital organs like the heart and brain. Autophagy, a cellular process involved in self-degradation and recycling, emerges as a promising therapeutic target in cancer therapy and neurodegenerative diseases. Dysregulation of autophagy contributes to cancer progression and drug resistance, while its modulation holds the potential to enhance treatment outcomes and mitigate adverse effects. Additionally, emerging evidence suggests a potential link between autophagy, DNA damage, and caretaker breast cancer genes BRCA1/2, highlighting the interplay between DNA repair mechanisms and cellular homeostasis. This review explores the intricate relationship between cancer, Dox-induced cytotoxicity, autophagy modulation, and the potential implications of autophagy in DNA damage repair pathways, particularly in the context of BRCA1/2 mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f00613c7c992a22afe5ec78107b8d19206649c8" target='_blank'>
              Complex Interplay between DNA Damage and Autophagy in Disease and Therapy
              </a>
            </td>
          <td>
            Aman Singh, Naresh Ravendranathan, Jefferson C. Frisbee, Krishna K. Singh
          </td>
          <td>2024-07-29</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Bacteriophages are the most abundant entities on earth and exhibit vast genetic and phenotypic diversity. Exploitation of this largely unexplored molecular space requires identification and functional characterisation of genes that act at the phage-host interface. To date, this is restricted to few model phage-host systems that are amenable to genetic manipulation. To overcome this limitation, we introduce a non-genetic mRNA targeting approach using exogenous delivery of programmable antisense oligomers (ASOs) to silence genes of both DNA and RNA phages. A systematic knockdown screen of core and accessory genes of the nucleus-forming jumbo phage ΦKZ, coupled to RNA-sequencing and microscopy analyses, reveals previously unrecognised proteins that are essential for phage replication and that, upon silencing, elicit distinct phenotypes at the level of the phage and host response. One of these factors is a ΦKZ-encoded nuclease that acts at a major decision point during infection, linking the formation of the protective phage nucleus to phage genome amplification. The non-genetic ASO-based gene silencing used here promises to be a versatile tool for molecular discovery in phage biology, will help elucidate defence and anti-defence mechanisms in non-model phage-host pairs, and offers potential for optimising phage therapy and biotechnological procedures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08d6a132616204d6fa19229a092ac9afe5732d73" target='_blank'>
              Non-genetic messenger RNA silencing reveals essential genes in phage-host interplay
              </a>
            </td>
          <td>
            Milan Gerovac, Svetlana Ðurica-Mitić, Leandro Buhlmann, Valentin Rech, Yan Zhu, Samuel Carien, Linda Popella, Jörg Vogel
          </td>
          <td>2024-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Ras genes are important oncogenes that are frequently mutated in cancer. Human oncogenic variants exhibit functional distinctions in terms of their representation in different cancer types, impact on cellular targets, and sensitivity to pharmacological treatments. However, how these distinct variants influence and respond to the cellular networks in which they are embedded is poorly understood. To identify novel participants in the complex interplay between Ras genotype and cell interaction networks in vivo, we have developed and tested an experimental framework using a simple vulva development assay in the nematode C. elegans. Using this system, we evaluate a set of Ras oncogenic substitution changes at G12, G13 and Q61. We find that these variants fall into distinct groups based on phenotypic differences, sensitivity to gene dose and inhibition of the downstream kinase MEK, and their response to genetic modulators that influence Ras activity in a non-autonomous manner. Together, our results demonstrate oncogenic C. elegans Ras variants exhibit clear distinctions in how they interface with the vulva development network, and show that extracellular modulators yield variant-restricted effects in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/675fab343bc71b75d1eb1f6e179c6e3a20afa2a0" target='_blank'>
              Functional distinction in oncogenic Ras variant activity in Caenorhabditis elegans.
              </a>
            </td>
          <td>
            Haimeng Lyu, H. Chamberlin
          </td>
          <td>2024-07-01</td>
          <td>Disease models & mechanisms</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Down syndrome is a genetic-based disorder that results from the triplication of chromosome 21, leading to an overexpression of many triplicated genes, including the gene encoding Dual-Specificity Tyrosine Phosphorylation-Regulated Kinase 1A (DYRK1A). This protein has been observed to regulate numerous cellular processes, including cell proliferation, cell functioning, differentiation, and apoptosis. Consequently, an overexpression of DYRK1A has been reported to result in cognitive impairment, a key phenotype of individuals with Down syndrome. Therefore, downregulating DYRK1A has been explored as a potential therapeutic strategy for Down syndrome, with promising results observed from in vivo mouse models and human clinical trials that administered epigallocatechin gallate. Current DYRK1A inhibitors target the protein function directly, which tends to exhibit low specificity and selectivity, making them unfeasible for clinical or research purposes. On the other hand, antisense oligonucleotides (ASOs) offer a more selective therapeutic strategy to downregulate DYRK1A expression at the gene transcript level. Advances in ASO research have led to the discovery of numerous chemical modifications that increase ASO potency, specificity, and stability. Recently, several ASOs have been approved by the U.S. Food and Drug Administration to address neuromuscular and neurological conditions, laying the foundation for future ASO therapeutics. The limitations of ASOs, including their high production cost and difficulty delivering to target tissues can be overcome by further advances in ASO design. DYRK1A targeted ASOs could be a viable therapeutic approach to improve the quality of life for individuals with Down syndrome and their families.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6325ca7451f96825cdc01a440a3d532d3e9fb0b3" target='_blank'>
              Down syndrome and DYRK1A overexpression: relationships and future therapeutic directions
              </a>
            </td>
          <td>
            Aidan J. Murphy, Steve D. Wilton, M. Aung-Htut, Craig S McIntosh
          </td>
          <td>2024-07-24</td>
          <td>Frontiers in Molecular Neuroscience</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Radiotherapy is a key treatment option for a wide variety of human tumors, employed either alone or alongside with other therapeutic interventions. Radiotherapy uses high-energy particles to destroy tumor cells, blocking their ability to divide and proliferate. The effectiveness of radiotherapy is due to genetic and epigenetic factors that determine how tumor cells respond to ionizing radiation. These factors contribute to the establishment of resistance to radiotherapy, which increases the risk of poor clinical prognosis of patients. Although the mechanisms by which tumor cells induce radioresistance are unclear, evidence points out several contributing factors including the overexpression of DNA repair systems, increased levels of reactive oxygen species, alterations in the tumor microenvironment, and enrichment of cancer stem cell populations. In this context, dysregulation of microRNAs or miRNAs, critical regulators of gene expression, may influence how tumors respond to radiation. There is increasing evidence that miRNAs may act as sensitizers or enhancers of radioresistance, regulating key processes such as the DNA damage response and the cell death signaling pathway. Furthermore, expression and activity of miRNAs have shown informative value in overcoming radiotherapy and long-term radiotoxicity, revealing their potential as biomarkers. In this review, we will discuss the molecular mechanisms associated with the response to radiotherapy and highlight the central role of miRNAs in regulating the molecular mechanisms responsible for cellular radioresistance. We will also review radio-miRs, radiotherapy-related miRNAs, either as sensitizers or enhancers of radioresistance that hold promise as biomarkers or pharmacological targets to sensitize radioresistant cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f11e14f281c48ffc0ef7d72af9b361e4a40e0c9" target='_blank'>
              Radio-miRs: a comprehensive view of radioresistance-related microRNAs
              </a>
            </td>
          <td>
            Abraham Pedroza-Torres, Sandra L Romero-Córdoba, S. Montaño, O. Peralta-Zaragoza, Dora Emma Vélez-Uriza, Cristian Arriaga-Canon, Xiadani Guajardo-Barreto, Diana Bautista-Sánchez, Rodrigo Sosa-León, Olivia Hernández-González, J. Díaz-Chávez, R. M. Álvarez-Gómez, Luis A Herrera
          </td>
          <td>2024-07-04</td>
          <td>Genetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3ef3973d270e32f48aaa19a00bef4e18bca5be" target='_blank'>
              Research Progress of Long Non-coding RNA-ZFAS1 in Malignant Tumors.
              </a>
            </td>
          <td>
            Xin Liu, Zhong Ma, Xianxu Zhang, Shicheng Li, Jiangdong An, Zhiqiang Luo
          </td>
          <td>2024-07-26</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="While the triggers for the metastatic transformation of breast cancer (BC) cells remain unknown, recent evidence suggests that intrinsic cellular metabolism could be a crucial driver of migratory disposition and chemoresistance. Aiming to decode the molecular mechanisms involved in BC cell metabolic maneuvering, we study how a ketomimetic (ketone body-rich, low glucose) medium affects Doxorubicin (DOX) susceptibility and invasive disposition of BC cells. We quantified glycocalyx sialylation and found an inverse correlation with DOX-induced cytotoxicity and DOX internalization. These measurements were coupled with single-cell metabolic imaging, bulk migration studies, along with transcriptomic and metabolomic analyses. Our findings revealed that a ketomimetic medium enhances chemoresistance and invasive disposition of BC cells via two main oncogenic pathways: hypersialylation and lipid synthesis. We propose that the crosstalk between these pathways, juxtaposed at the synthesis of the glycan precursor UDP-GlcNAc, furthers advancement of a metastatic phenotype in BC cells under ketomimetic conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fa6f2b5143747f6467f4d23175994fac910eade" target='_blank'>
              Ketomimetic Nutrients Trigger a Dual Metabolic Defense in Breast Cancer Cells
              </a>
            </td>
          <td>
            Mohini Kamra, Yuan-I Chen, Paula C. Delgado, Erin H. Seeley, S. Seidlits, Hsin-Chin Yeh, Amy Brock, S. Parekh
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53c24f0b42670eaa7d910971701aacf3a8d5213b" target='_blank'>
              Targeting RBM39 through indisulam induced mis-splicing of mRNA to exert anti-cancer effects in T-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Tongting Ji, Yang Yang, Juanjuan Yu, Hongli Yin, Xinran Chu, Pengju Yang, Ling Xu, Xiaodong Wang, Shaoyan Hu, Yizhen Li, Xiaochen Wu, Wengyuan Liu, Bi Zhou, Wenjuan Wang, Shuqi Zhang, Wei Cheng, Yanling Chen, Lei Shi, Zhiheng Li, Ran Zhuo, Yongping Zhang, Yanfang Tao, Di Wu, Xiao-Lu Li, Zimu Zhang, Jun-Jie Fan, Jian Pan, Jun Lu
          </td>
          <td>2024-07-24</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Despite rapid advances in the field of immunotherapy, including the success of immune checkpoint inhibition in treating multiple cancer types, clinical response in high-grade gliomas (HGGs) has been disappointing. This has been in part attributed to the low tumor mutational burden (TMB) of the majority of HGGs. Hypermutation is a recently characterized glioma signature that occurs in a small subset of cases, which may open an avenue to immunotherapy. The substantially elevated TMB of these tumors most commonly results from alterations in the DNA mismatch repair pathway in the setting of extensive exposure to temozolomide or, less frequently, from inherited cancer predisposition syndromes. In this review, we discuss the genetics and etiology of hypermutation in HGGs, with an emphasis on the resulting genomic signatures, and the state and future directions of immuno-oncology research in these patient populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2f2c2e78fe8478861279397510414edbdfa255" target='_blank'>
              Pathways to hypermutation in high-grade gliomas: Mechanisms, syndromes, and opportunities for immunotherapy.
              </a>
            </td>
          <td>
            Tuesday Haynes, Mark R Gilbert, Kevin Breen, Chunzhang Yang
          </td>
          <td>2024-06-24</td>
          <td>Neuro-oncology advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction. It has been established that the presence of homologous recombination deficiency in a breast tumor is associated with the effectiveness of treatment. But despite the high chemosensitivity of the tumor to DNA-damaging agents, complete pathological responses to treatment are very rare. And this process may be based on a change in the somatic status of BRCA1, that is, a reversion and return of the wild-type allele occurs and the DNA repair function is restored.Aim. To evaluate changes in the presence of chromosomal aberrations and the expression profile of the main genes of homologous recombination in cell models of breast cancer under the influence of cisplatin and docetaxel.Materials and methods. The study was conducted on breast cancer tumor cell cultures: MCF-7, MDA-MB-231 and MDA-MB-468. A cell model of drug resistance was obtained for two drugs: cisplatin and docetaxel. RNA and DNA were isolated from cell suspension using the RNeasy Plus Mini Kit and QIAamp DNA Mini Kit (Qiagen, Germany), respectively. The expression level of homologous recombination genes was assessed using reverse transcription polymerase chain reaction. To assess the presence of chromosomal aberrations, microarray analysis was performed on DNA chips.Results. Restoration of normal copy number for the BRCA1, CDK12, CHEK1 and RAD51D genes in MCF-7 under the influence of cisplatin was shown. For BRCA2 and PALB2, amplifications were detected. A statistically significant increase in the expression of the BRCA1 (p = 0.04), BRCA2 (p = 0.02), PALB2 (p = 0.01) and RAD51D (p = 0.05) genes was also shown. MDAMB-231 shows that all identified loci with deletions, where the BRCA2, BARD1, CHEK2, PALB2 and RAD54L genes are localized, are restored to normal copy number by cisplatin. The appearance of amplifications was registered for BRCA1, BRIP1, FANCL, RAD51B, PARP1. A similar result was shown for docetaxel. An increase in the expression level is typical for the genes BRCA1 (p = 0.02), BRCA2 (p = 0.02), CHEK2 (p = 0.05), FANCL (p = 0.04), PALB2 (p = 0.05), RAD51C (p = 0.02), PARP1 (p = 0.02), which corresponds to the appearance of amplifications. In the MDA-MB-468 cell culture, an increase in the copy number of only the BRCA1 gene is observed. The effect of docetaxel has no effect on this cell culture. The level of BRCA1 expression increases in direct proportion to the duration of drug action.Conclusion. Thus, the study showed that under the influence of cisplatin, reversion of not only homologous recombination gene mutations, but also other disorders can occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/767d30042e565f8e61dafca9671915ee6d03ecf7" target='_blank'>
              Modulation of homologous recombination gene activity in breast tumor cells in an in vitro model
              </a>
            </td>
          <td>
            M. Tsyganov, A. Frolova, E. A. Kravtsova, I. A. Tsydenova, M. Ibragimova
          </td>
          <td>2024-06-28</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Studies of bacterial adaptation and evolution are hampered by the difficulty of measuring traits such as virulence, drug resistance, and transmissibility in large populations. In contrast, it is now feasible to obtain high-quality complete assemblies of many bacterial genomes thanks to scalable high-accuracy long-read sequencing technologies. To exploit this opportunity, we introduce a phenotype- and alignment-free method for discovering coselected and epistatically interacting genomic variation from genome assemblies covering both core and accessory parts of genomes. Our approach uses a compact colored de Bruijn graph to approximate the intragenome distances between pairs of loci for a collection of bacterial genomes to account for the impacts of linkage disequilibrium (LD). We demonstrate the versatility of our approach to efficiently identify associations between loci linked with drug resistance and adaptation to the hospital niche in the major human bacterial pathogens Streptococcus pneumoniae and Enterococcus faecalis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ba999d01a93235712556c85e81e52a6ac50e10" target='_blank'>
              Pangenome-spanning epistasis and coselection analysis via de Bruijn graphs.
              </a>
            </td>
          <td>
            Juri Kuronen, Samuel T. Horsfield, A. K. Pöntinen, S. Mallawaarachchi, S. Arredondo-Alonso, Harry A. Thorpe, R. Gladstone, Rob J. L. Willems, Stephen D. Bentley, N. Croucher, J. Pensar, John A. Lees, G. Tonkin-Hill, Jukka Corander
          </td>
          <td>2024-08-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d50edc29971b3d5ad46253451d29ee88c9075e8" target='_blank'>
              A metabolic reprogramming-related gene signature correlates with prognosis and proliferation of BLCA
              </a>
            </td>
          <td>
            Yaoxin Wu, Yi Luo, Tinghao Li
          </td>
          <td>2024-08-08</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="OBJECTIVE
Acute Lymphoblastic Leukemia (ALL) is the most common malignancy occurring in children. Copy number alterations (CNA) like PAX5, CDKN2A/2B, PAR1 Region, ETV6, IKZF1, BTG1, and RB1 gene deletion are important genetic events that define and prognosticate B-cell ALL. Thus, this study aimed to evaluate associations of CNA with induction phase remission status in childhood B-cell ALL.


METHODS
This study was observational with a cross-sectional design at the Dharmais Cancer Hospital, Harapan Kita Mother and Children Hospital, and Tangerang Regional Public Hospital. We evaluated 74 pediatric B-cell ALL cases with 1-18-year-olds. Genomic DNA was analyzed by Multiplex Ligation Dependent Probe Amplification Assay (MLPA). This study used the P335 ALL-IKZF1 panel kit, which contains several ALL-related genes. The patient's clinical and laboratory characteristics were collected from medical records from January to December 2019.


RESULT
We observed gene copy number alteration in children with B-Cell ALL. PAX5 was the most commonly observed gene deletion, followed by CDKN21/2B, ETV6, IKZF1, BTG1, RB1, and PAR1 Region. Based on gene mutations, only the PAX5 had a significant association with the remission status of pediatric B-cell ALL (p-value <0.05; OR = 3.91). It showed that patients with PAX5 gene mutations have 3.9 times the risk of no remission and/or relapse compared to those without PAX5 gene mutations.


CONCLUSION
Patients with mutations in the PAX5 gene have a higher chance of not achieving remission and/or experiencing relapse than those without such mutations. The MLPA method can be utilized for examining copy number alterations, which is valuable for achieving more precise stratification in diagnosis.. Further research is needed to expand upon this finding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bd752c14237c4bf075eb8efe30658a5f3ec960c" target='_blank'>
              Unraveling Copy Number Alterations in Pediatric B-Cell Acute Lymphoblastic Leukemia: Correlation with Induction Phase Remission Using MLPA.
              </a>
            </td>
          <td>
            M. Aisyi, Murti Andriastuti, A. Kosasih, Ahmad Rusdan Handoyo Utomo, Fahreza Saputra, Teny Tjitra Sari, H. Sjakti, Fifi Dwijayanti, K. Harimurti, D. Gatot
          </td>
          <td>2024-07-01</td>
          <td>Asian Pacific journal of cancer prevention : APJCP</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract The ability to deliver large transgenes to a single genomic sequence with high efficiency would accelerate biomedical interventions. Current methods suffer from low insertion efficiency and most rely on undesired double-strand DNA breaks. Serine integrases catalyze the insertion of large DNA cargos at attachment (att) sites. By targeting att sites to the genome using technologies such as prime editing, integrases can target safe loci while avoiding double-strand breaks. We developed a method of phage-assisted continuous evolution we call IntePACE, that we used to rapidly perform hundreds of rounds of mutagenesis to systematically improve activity of PhiC31 and Bxb1 serine integrases. Novel hyperactive mutants were generated by combining synergistic mutations resulting in integration of a multi-gene cargo at rates as high as 80% of target chromosomes. Hyperactive integrases inserted a 15.7 kb therapeutic DNA cargo containing von Willebrand Factor. This technology could accelerate gene delivery therapeutics and our directed evolution strategy can easily be adapted to improve novel integrases from nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1115be639d45f9efbece4cb3bd061894c38e5687" target='_blank'>
              Directed evolution of hyperactive integrases for site specific insertion of transgenes
              </a>
            </td>
          <td>
            Brian E. Hew, Sabranth Gupta, Ryuei Sato, David F. Waller, I. Stoytchev, James E. Short, Lisa Sharek, Christopher T. Tran, Ahmed H. Badran, Jesse B Owens
          </td>
          <td>2024-07-02</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="BACKGROUND
The DNA mismatch repair (MMR) system serves as a sophisticated guardian of the precise functioning of the human genome. Dysregulation within this system is linked to the oncogenesis process. Reduced expression of MMR system proteins identified in salivary gland tumors (SGTs) suggests an increased risk of tumoral occurrence. This study aims to analyze the expression of MMR proteins in SGTs and discuss the relevance of this association to the development of these neoplasms.


MATERIAL AND METHODS
This review was conducted following the PRISMA guidelines and was registered in PROSPERO (CRD42023465590). A comprehensive search of the PubMed/MEDLINE, Web of Science, Scopus, Embase, and ProQuest (non-peer reviewed platform) was performed to answer the question "Do DNA MMR system proteins exhibit expression in SGTs?". The methodological quality of the selected studies was assessed using the JBI's Critical Appraisal Tool.


RESULTS
A total of 142 patients with benign SGTs and 84 with malignant SGTs were included in this review. The literature analysis showed a notable reduction in the expression of DNA MMR system proteins (hHMS2, hMLH1, hMSH3 and hMSH6) in the percentage of marked cells.


CONCLUSIONS
The reduction in the expression of the DNA MMR system proteins suggests an interesting correlation with the development of malignant and benign SGTs. Nevertheless, further investigations are warranted to better clarify the precision of measuring biomarker protein expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfbcf6ccd15abbdea0ae296578920af07be7eb3d" target='_blank'>
              DNA mismatch repair system expression in salivary gland tumors: A Systematic Review.
              </a>
            </td>
          <td>
            G-M-T-T Alves, R-J Severino-Lazo, G-K Amaral-Silva, F-M Silveira, M-D Carvalho
          </td>
          <td>2024-06-22</td>
          <td>Medicina oral, patologia oral y cirugia bucal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="INTRODUCTION
Defining the chromosomal and molecular changes associated with myeloid neoplasms (MNs) optimizes clinical care through improved diagnosis, prognosis, treatment planning, and patient monitoring. This review will concisely describe the techniques used to profile MNs clinically today, with descriptions of challenges and emerging approaches that may soon become standard-of-care.


AREAS COVERED
In this review, the authors discuss molecular assessment of MNs using non-sequencing techniques, including conventional cytogenetic analysis, fluorescence in situ hybridization, chromosomal genomic microarray testing; as well as DNA- or RNA-based next-generation sequencing (NGS) assays; and sequential monitoring via digital PCR or measurable residual disease assays. The authors explain why distinguishing somatic from germline alleles is critical for optimal management. Finally, they introduce emerging technologies, such as long-read, whole exome/genome, and single-cell sequencing, which are reserved for research purposes currently but will become clinical tests soon.


EXPERT OPINION
The authors describe challenges to the adoption of comprehensive genomic tests for those in resource-constrained environments and for inclusion into clinical trials. In the future, all aspects of patient care will likely be influenced by the adaptation of artificial intelligence and mathematical modeling, fueled by rapid advances in telecommunications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce5821258a38d2a15a1cce4cdbe2844d4e8f9b60" target='_blank'>
              The impact of next-generation sequencing for diagnosis and disease understanding of myeloid malignancies.
              </a>
            </td>
          <td>
            Erica Vormittag-Nocito, M. Sukhanova, Lucy A. Godley
          </td>
          <td>2024-07-25</td>
          <td>Expert review of molecular diagnostics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="B cell progenitor acute lymphoblastic leukemia (BCP-ALL) is the most common childhood malignancy, driven by multiple genetic alterations that cause maturation arrest and accumulation of abnormal progenitor B cells. Current treatment protocols with chemotherapy have led to favorable outcomes but are associated with significant toxicity and risk of side effects, highlighting the necessity for highly effective, less toxic, targeted drugs, even in subtypes with a favorable outcome. Here, we used multimodal single-cell sequencing to delineate the transcriptional, epigenetic, and immunophenotypic characteristics of 23 childhood BCP-ALLs, belonging to the BCR::ABL1-positive, ETV6::RUNX1-positive, high hyperdiploid, and recently discovered DUX4-rearranged (DUX4-r) subtypes. Projection of the ALL cells along the normal hematopoietic differentiation axis revealed a diversity in the maturation pattern between the different BCP-ALL subtypes. Whereas the BCR::ABL1-, ETV6::RUNX1-positive, and high hyperdiploidy cells mainly showed similarities to normal pro-B cells, the DUX4-r ALL cells also displayed transcriptional signatures resembling mature B cells. Focusing on the DUX4-r subtype, we found that the blast population displayed multilineage priming toward non-hematopoietic cells, myeloid, and T cell lineages, but also an activation of PI3K/AKT signaling that sensitized the cells to PI3K inhibition in vivo. Given the multilineage priming of the DUX4-r blasts with aberrant expression of the myeloid marker CD371 (CLL-1), we generated chimeric antigen receptor T cells, which effectively eliminated DUX4-r ALL cells in vivo. These results provide a detailed characterization of BCP-ALL at the single-cell level and reveal therapeutic vulnerabilities in the DUX4-r subtype with implications for the understanding of ALL biology and new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1d7a915d43d3c2ff8b079040d662a79ad558986" target='_blank'>
              Single-cell genomics details the maturation block in BCP-ALL and identifies therapeutic vulnerabilities in DUX4-r cases.
              </a>
            </td>
          <td>
            Hanna Thorsson, Rasmus Henningsson, Noelia Puente-Moncada, Pablo Peña-Martínez, Ludvig Sjöström, H. Ågerstam, Carl Sandén, M. Rissler, A. Castor, H. Marquart, S. Modvig, K. Paulsson, C. Pronk, Kjeld Schmiegelow, Axel Hyrenius-Wittsten, Christina Orsmark-Pietras, H. Lilljebjörn, T. Fioretos
          </td>
          <td>2024-07-05</td>
          <td>Blood</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Emerging immunotherapies such as immune checkpoint blockade (ICB) and chimeric antigen receptor T-cell (CAR-T) therapy have revolutionized cancer treatment and have improved the survival of patients with multiple cancer types. Despite this success many patients are unresponsive to these treatments or relapse following treatment. CRISPR activation and knockout (KO) screens have been used to identify novel single gene targets that can enhance effector T cell function and promote immune cell targeting and eradication of tumors. However, cancer cells often employ multiple genes to promote an immunosuppressive pathway and thus modulating individual genes often has a limited effect. Paralogs are genes that originate from common ancestors and retain similar functions. They often have complex effects on a particular phenotype depending on factors like gene family similarity, each individual gene’s expression and the physiological or pathological context. Some paralogs exhibit synthetic lethal interactions in cancer cell survival; however, a thorough investigation of paralog pairs that could enhance the efficacy of cancer immunotherapy is lacking. Here we introduce a sensitive computational approach that uses sgRNA sets enrichment analysis to identify cancer-intrinsic paralog pairs which have the potential to synergistically enhance T cell-mediated tumor destruction. We have further developed an ensemble learning model that uses an XGBoost classifier and incorporates features such as gene characteristics, sequence and structural similarities, protein-protein interaction (PPI) networks, and gene coevolution data to predict paralog pairs that are likely to enhance immunotherapy efficacy. We experimentally validated the functional significance of these predicted paralog pairs using double knockout (DKO) of identified paralog gene pairs as compared to single gene knockouts (SKOs). These data and analyses collectively provide a sensitive approach to identify previously undetected paralog pairs that can enhance cancer immunotherapy even when individual genes within the pair has a limited effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9398918df5088c04d21b0123727e4ddeb607a57" target='_blank'>
              Sensitive detection of synthetic response to cancer immunotherapy driven by gene paralog pairs
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Emily He, Ping Ren, Nipun Verma, Xinxin Zhu, Di Feng, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="T-prolymphocytic leukemia (T-PLL) is a mature T-cell neoplasm associated with marked chemotherapy resistance and continued poor clinical outcomes. Current treatments, i.e. the CD52-antibody alemtuzumab, offer transient responses, with relapses being almost inevitable without consolidating allogeneic transplantation. Recent more detailed concepts of T-PLL's pathobiology fostered the identification of actionable vulnerabilities: (i) altered epigenetics, (ii) defective DNA damage responses, (iii) aberrant cell-cycle regulation, and (iv) deregulated pro-survival pathways, including TCR and JAK/STAT signaling. To further develop related pre-clinical therapeutic concepts, we studied inhibitors of (H)DACs, BCL2, CDK, MDM2, and clas-sical cytostatics, utilizing (a) single-agent and combinatorial compound testing in 20 well-characterized and molecularly-profiled primary T-PLL (validated by additional 42 cases), and (b) 2 independent murine models (syngeneic transplants and patient-derived xenografts). Overall, the most efficient/selective single-agents and combinations (in vitro and in mice) in-cluded Cladribine, Romidepsin ((H)DAC), Venetoclax (BCL2), and/or Idasanutlin (MDM2). Cladribine sensitivity correlated with expression of its target RRM2. T-PLL cells revealed low overall apoptotic priming with heterogeneous dependencies on BCL2 proteins. In additional 38 T-cell leukemia/lymphoma lines, TP53 mutations were associated with resistance towards MDM2 inhibitors. P53 of T-PLL cells, predominantly in wild-type configuration, was amenable to MDM2 inhibition, which increased its MDM2-unbound fraction. This facilitated P53 activa-tion and down-stream signals (including enhanced accessibility of target-gene chromatin re-gions), in particular synergy with insults by Cladribine. Our data emphasize the therapeutic potential of pharmacologic strategies to reinstate P53-mediated apoptotic responses. The identified efficacies and their synergies provide an informative background on compound and patient selection for trial designs in T-PLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28856a6bd0f85fac41c622c5fe8a6c6a100dae44" target='_blank'>
              Optimizing Drug Combinations for T-PLL: Restoring DNA Damage and P53-Mediated Apoptotic Responses.
              </a>
            </td>
          <td>
            J. von Jan, S. Timonen, T. Braun, Qu Jiang, Aleksandr Ianevski, Yayi Peng, Kathleen McConnell, Paola Sindaco, T. Müller, S. Pützer, Hanna Klepzig, Dennis Jungherz, Annika Dechow, Linus Wahnschaffe, Anil K Giri, M. Kankainen, H. Kuusanmäki, H. Neubauer, Richard H Moriggl, P. Mazzeo, Nicole Schmidt, Raphael Koch, Michael Hallek, A. Chebel, David Armisén, Laurent Genestier, E. Bachy, Anjali Mishra, A. Schrader, T. Aittokallio, S. Mustjoki, Marco Herling
          </td>
          <td>2024-06-28</td>
          <td>Blood</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbd2ebf2aed5dc3874b70ddd0ac9b07d92d51408" target='_blank'>
              Short Tandem Repeats in the era of next-generation sequencing: from historical loci to population databases.
              </a>
            </td>
          <td>
            K. Uguen, Jacques L Michaud, Emmanuelle Génin
          </td>
          <td>2024-07-10</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Abstract Human endogenous retroviruses (HERVs), the remnants of ancient germline retroviral integrations, comprise almost 8% of the human genome. The elucidation of their biological roles is hampered by our inability to link HERV mRNA and protein production with specific HERV loci. To solve the riddle of the integration-specific RNA expression of HERVs, several bioinformatics approaches have been proposed; however, no single process seems to yield optimal results due to the repetitiveness of HERV integrations. The performance of existing data-bioinformatics pipelines has been evaluated against real world datasets whose true expression profile is unknown, thus the accuracy of widely-used approaches remains unclear. Here, we simulated mRNA production from specific HERV integrations to evaluate second and third generation sequencing technologies along with widely used bioinformatic approaches to estimate the accuracy in describing integration-specific expression. We demonstrate that, while a HERV-family approach offers accurate results, per-integration analyses of HERV expression suffer from substantial expression bias, which is only partially mitigated by algorithms developed for calculating the per-integration HERV expression, and is more pronounced in recent integrations. Hence, this bias could erroneously result into biologically meaningful inferences. Finally, we demonstrate the merits of accurate long-read high-throughput sequencing technologies in the resolution of per-locus HERV expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f3e71c5502b568bd36436f0dbbe38d91d92145df" target='_blank'>
              Limitations of current high-throughput sequencing technologies lead to biased expression estimates of endogenous retroviral elements
              </a>
            </td>
          <td>
            Konstantina Kitsou, A. Katzourakis, G. Magiorkinis
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ee3ef93f75d18cca805d005d653d62f6f76b88e" target='_blank'>
              Comprehensive pan-cancer analysis reveals that C5orf34 regulates the proliferation and mortality of lung cancer.
              </a>
            </td>
          <td>
            Mengmeng Yang, Yuhan Deng, Yu Ma, Chunli Song, Zhenhua Wu, Xiayimaierdan Yibulayin, Xiaohong Sun, Yunquan Guo, Dan He
          </td>
          <td>2024-06-29</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Disease progression after intensive multi modal clinical therapy poses grave concern not only on the quality of life but on the very survival of children presented with neuroblastoma. Acquired modifications of energy metabolism in the tumor cells that defy clinical therapy plays pivotal role in frequent and rapid tumor relapse. We investigated the mechanisms of how these cells adopt metabolic rearrangements with therapy pressure. Our earlier findings showed the constitutive availability of retinal degeneration protein 3 (RD3) in human fetal tissues beyond retina, displayed its functional role in tumor pathogenesis, and unearthed its significance in neuroblastoma clinical outcomes. Herein, with the gene surrogate strategy, the whole genome RNA sequencing in reverse engineered (bed-to-bench) patient derived cell lines showed a patient-independent, RD3-dependent stabilization of metabolic programming. Loss of RD3 altered crucial metabolic pathways and coordinated metabolic mayhem in high-risk stage-4 disease. Evaluating the glycolysis phenotype with seahorse revealed that RD3 loss dependent increase in extracellular acidification rate (EACR) associated with glycolysis, glycolytic capacity and glycolytic reserves in therapy defying progressive disease. Critically, therapeutic delivery of RD3 (custom archived and characterized RD3 peptide candidates) significantly deregulated glycolysis in such progressive disease that lack RD3, recognizing a RD3-dependent regulation of metabolic activity. Our study outcomes clearly portrayed the thus far unrealized significance of therapy pressure driven RD3 loss mediated metabolism deregulation in neuroblastoma. Crucially, these new findings will pave way for the identification of a novel targeted maintenance therapeutic strategy for treatment of this deadly disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9ddc349052e0fda38c77640ff0b290e468ff1c6" target='_blank'>
              Metabolic Reprogramming and Extracranial Solid Tumor Evolution in Infants
              </a>
            </td>
          <td>
            Poorvi Subramanian, Natarajan Aravindan
          </td>
          <td>2024-07-01</td>
          <td>JCO Global Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cae8abbc9e79803db49d453f861eb5943682ed07" target='_blank'>
              Mitochondrial genetics through the lens of single-cell multi-omics.
              </a>
            </td>
          <td>
            Lena Nitsch, Caleb A. Lareau, Leif S. Ludwig
          </td>
          <td>2024-07-01</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>23</td>
        </tr>

        <tr id="Near-haploidization, that is, loss of one copy of most chromosomes, is a relatively rare phenomenon in most tumors, but is enriched among certain soft tissue sarcomas, including undifferentiated pleomorphic sarcoma (UPS). Presumably, near-haploidization can arise through many mechanisms. This study aimed to identify gene rearrangements that could cause near-haploidization. We here present two UPS in which near-haploidization was an early event, identified through single nucleotide polymorphism (SNP) array analysis. One of the cases was studied further using whole genome and transcriptome sequencing, as well as cytogenetic and molecular cytogenetic methods. Both tumors had chromosomal rearrangements in the form of copy number shifts/structural variants affecting the SMC1A gene. These findings suggest that cohesin defects could contribute to mitotic errors resulting in massive loss of chromosomes. SMC1A encodes one of the components of the cohesin multiprotein complex, which is critical for proper alignment of the sister chromatids during S-phase and separation to opposite spindle poles. Further studies should explore the role of cohesin defects in near-haploidization in other sarcomas and to clarify its role in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac46389944e17adf0a38d1c92f41322a72ad7ff" target='_blank'>
              Structural Variants in the SMC1A Gene Associated With Near-Haploidy in Undifferentiated Pleomorphic Sarcomas.
              </a>
            </td>
          <td>
            Sebastian Ibstedt, Paul Piccinelli, Saskia Sydow, Jan Köster, Fredrik Mertens
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Background: Integrating immune checkpoint inhibitors with multi-target tyrosine kinase inhibitors presents an innovative and hopeful strategy in liver cancer treatment. Nonetheless, a degree of resistance to this treatment is noticeable in certain patients. Alternative splicing (AS) represents a common biological process that controls the variety of life functions via isoforms. Purpose: Investigating how gene AS affects the effectiveness of combined immunotherapy in treating hepatocellular carcinoma (HCC). Methods: Our retrospective examination focused on AS's effect on immune therapy effectiveness, utilizing accessible tissue sequencing and clinical records for HCC. For corroborating our results, we gathered samples of drug-resistant HCC tissue, nearby tissues, HCC tissue with high drug responsiveness, and healthy liver tissue from clinical studies. Results: The study revealed a link between the frequency of AS occurrences, the expression levels of programmed cell death 1 ligand 1, and the resistance to tumor medications. Our study detailed the AS occurrences in HCC, leading to the creation of a risk-assessment function and a predictive model using AS data. The results of our study revealed that the risk score effectively distinguished between various immune subtypes and the effectiveness of immune therapy. Additional examination of the chosen AS occurrences uncovered their effects on both the immune microenvironment and cellular immunity. Our investigation also delved into the regulatory framework of AS, uncovering the role of stringently controlled splicing factors in the emergence of tumors and the modulation of the body's immune response. Conclusions: Increased AS in HCC diminishes the efficacy of immunotherapy; conversely, more AS in peritumoral tissue elevates the likelihood of tumor immune evasion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc39d2a7e8075c89a1f4a7ca29b083b1fc5166ab" target='_blank'>
              Prediction Model for Immunotherapy Efficacy in Hepatocellular Carcinoma Based on Alternative Splicing Sequencing Data
              </a>
            </td>
          <td>
            Huizhong Shi, Yang Liu, Zhao Liu, Xinlan Ge, Jushan Wu, Haowen Tang, Yi Zhang, Shi-yong Lu
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a976d04ebae96418666fe6d36b4350c7aceba4b4" target='_blank'>
              Single-cell transcriptomic insights into chemotherapy-induced remodeling of the osteosarcoma tumor microenvironment
              </a>
            </td>
          <td>
            Xuejing Zheng, Wence Wu, Zhen-guo Zhao, Xin-xin Zhang, Shengji Yu
          </td>
          <td>2024-07-20</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Background The methyltransferase SETD2 was originally identified as a tumor suppressor in clear cell renal cell carcinoma (ccRCC), and ccRCC with SETD2 mutations have increased metastatic potential and poor progression free survival. A known epigenetic regulator, SETD2 interacts with actively transcribing RNA polymerase II (RNAPII) and tri-methylates Histone 3 at Lysine 36 (H3K36me3), a modification that correlates with active gene transcription and is recognized by mRNA methylation and splicing factors. Despite these known roles in mRNA transcription, loss of SETD2 does not alter genic transcription, raising the question of whether or how SETD2 regulates RNA biology. Methods Given that SETD2 influences RNA processing, we sought to identify RNA-protein complexes that are regulated by SETD2. To do this in an unbiased and high-throughput manner, we used orthogonal organic phase separation (OOPS) coupled with mass spectrometry to identify changes in RNA binding proteins and pathways between control and SETD2-knockout cells. Results Corroborating previous findings, we identified that RNA binding proteins governing mRNA splicing and cellular metabolism were dysregulated in SETD2 mutant cells. However, the most robust changes in SETD2 knockout cells occurred in pathways regulated ribosome biogenesis (See Figure, orange highlights), resulting in a significant decrease in binding of ribosomal RNA (rRNA) processing factors to RNA. We confirm these alterations lead to defects in rRNA transcription, processing, and ribosome biogenesis. We identify that loss of SETD2 catalytic activity phenocopies these rRNA processing defects, suggesting H3K36me3 is essential for rRNA processing. Disruptions occur to both the large (60S) and small (40S) ribosome subunits, modifying protein translation. Lastly, we identify synthetic lethality between SETD2 deletion or inhibition and small molecules that interfere with rRNA transcription. Conclusions While the majority of research emphasizes the role SETD2 in the regulation of mRNA, our data identify that SETD2 and its catalytic activity play a critical role in the transcription and processing of rRNA. Importantly, rRNA comprises 80-90% of cellular RNA yet is typically overlooked and understudied. Next-generation RNA sequencing or transcriptional assays investigate mRNA or genic transcription while ignoring rRNA, demonstrating why SETD2-mediated rRNA processing has been unexplored. This may also have important implications for prospective clinical trials that utilized RNA-sequencing to identify optimal therapeutic regimens. Our work also identified that SETD2-mutated cells are more sensitive to compounds that inhibit rRNA transcription, potentially uncovering a new therapeutic vulnerability of ccRCC with mono- and bi-allelic loss of SETD2. Alterations in ribosome biogenesis leads to epithelial-to-mesenchymal transition and metastasis in many tumor types, and our future goal will be to investigate the role of ribosomes and rRNA transcription in transformation and metastasis in ccRCC as well as the molecular mechanisms connecting SETD2 to ribosome fidelity. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30cb284d919d6bd0de496d90d21e1c0abae77b34" target='_blank'>
              51 Epigenetic regulation of ribosomal RNA transcription and processing by the tumor suppressor SETD2
              </a>
            </td>
          <td>
            Logan Vlach, Scott Haake, K. Rathmell, Frank M. Mason
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The aim of this study was to evaluate the mutation spectrum of homologous recombination repair (HRR) genes and its association with tumor immune infiltration and prognosis in triple-negative breast cancer (TNBC). TNBC patients (434 patients from Ruijin cohort) were evaluated with targeted next-generating sequencing for mutations in HRR genes. The frequencies of mutations were compared with public reference cohorts (320 TNBC patients from METABRIC, 105 from TCGA, and 225 from MSKCC 2018). Associations between mutation status and tumor immune infiltration and prognosis were analyzed. HRR genes mutations were seen in 21.89% patients, with BRCA1/2 mutations significantly enriched in tumors with breast/ovarian cancer family history (P = 0.025) and high Ki-67 levels (P = 0.018). HRR genes mutations were not related with recurrence-free survival (RFS) (adjusted P = 0.070) and overall survival (OS) (adjusted P = 0.318) for TNBC patients, regardless of carboplatin treatment (P > 0.05). Moreover, tumor immune infiltration and PD-L1 expression was positively associated with HRR or BRCA1/2 mutation (all P < 0.001). Patients with both HRR mutation and high CD8+ T cell counts had the best RFS and OS, whereas patients with no HRR mutation and low CD8+ T cell counts had the worst outcomes (RFS P < 0.001, OS P = 0.019). High frequency of HRR gene mutations was found in early TNBC, with no prognostic significance. Immune infiltration and PD-L1 expression was positively associated with HRR mutation, and both HRR mutation and high CD8+ T cell infiltration levels were associated with superior disease outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/754b9595e6f9b757c8c2106ab35e651d7b39b608" target='_blank'>
              Association of tumor immune infiltration and prognosis with homologous recombination repair genes mutations in early triple-negative breast cancer
              </a>
            </td>
          <td>
            Zheng Wang, A. Li, Yujie Lu, Mengyuan Han, Miao Ruan, Chaofu Wang, Xiaotian Zhang, Changbin Zhu, Kunwei Shen, Lei Dong, Xiao-song Chen
          </td>
          <td>2024-07-04</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Background Acute myeloid leukaemia (AML) is a disease of the haematopoietic stem cells(HSCs) that is characterised by the uncontrolled proliferation and impaired differentiation of normal haematopoietic stem/progenitor cells. Several pathways that control the proliferation and differentiation of HSCs are impaired in AML. Activation of the Wnt/beta-catenin signalling pathway has been shown in AML and beta-catenin, which is thought to be the key element of this pathway, has been frequently highlighted. The present study was designed to determine beta-catenin expression levels and beta-catenin-related genes in AML. Methods In this study, beta-catenin gene expression levels were determined in 19 AML patients and 3 controls by qRT-PCR. Transcriptome analysis was performed on AML grouped according to beta-catenin expression levels. Differentially expressed genes(DEGs) were investigated in detail using the Database for Annotation Visualisation and Integrated Discovery(DAVID), Gene Ontology(GO), Kyoto Encyclopedia of Genes and Genomes(KEGG), STRING online tools. Results The transcriptome profiles of our AML samples showed different molecular signature profiles according to their beta-catenin levels(high-low). A total of 20 genes have been identified as hub genes. Among these, TTK, HJURP, KIF14, BTF3, RPL17 and RSL1D1 were found to be associated with beta-catenin and poor survival in AML. Furthermore, for the first time in our study, the ELOV6 gene, which is the most highly up-regulated gene in human AML samples, was correlated with a poor prognosis via high beta-catenin levels. Conclusion It is suggested that the identification of beta-catenin-related gene profiles in AML may help to select new therapeutic targets for the treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ccef612e7374e44e05f4d3d5061c210406c1fc8" target='_blank'>
              Transcriptome Analysis of Beta-Catenin-Related Genes in CD34+ Haematopoietic Stem and Progenitor Cells from Patients with AML
              </a>
            </td>
          <td>
            Buket Altinok Gunes, T. Ozkan, Aynur Karadag Gurel, S. Dalkılıç, Nevin Belder, Zeynep Ozkeserli, Hilal Ozdag Sevgili, M. Beksaç, Nilgün Sayınalp, Abdullah Munci Yagci, A. Sunguroglu
          </td>
          <td>2024-06-29</td>
          <td>Mediterranean Journal of Hematology and Infectious Diseases</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Targeting tumor metabolism through dietary interventions is an area of growing interest, and may help to improve the significant mortality of aggressive cancers, including non-small cell lung cancer (NSCLC). Here we show that the restriction of methionine in the aggressive KRAS/Lkb1-mutant NSCLC autochthonous mouse model drives decreased tumor progression and increased carboplatin treatment efficacy. Importantly, methionine restriction during early stages of tumorigenesis prevents the lineage switching known to occur in the model, and alters the tumor immune microenvironment (TIME) to have fewer tumor-infiltrating neutrophils. Mechanistically, mutations in LKB1 are linked to anti-oxidant production through changes to cystathionine-β-synthase (CBS) expression. Human cell lines with rescued LKB1 show increased CBS levels and resistance to carboplatin, which can be partially rescued by methionine restriction. Furthermore, LKB1 rescued cells, but not mutant cells, show less G2- M arrest and apoptosis in high methionine conditions. Knock-down of CBS sensitized both LKB1 mutant and non-mutated lines to carboplatin, again rescuing the carboplatin resistance of the LKB1 rescued lines. Given that immunotherapy is commonly combined with chemotherapy for NSCLC, we next wanted to understand if T cells are impaired by MR. Therefore, we examined the ability of T cells from MR and control tumor bearing mice to proliferate in culture and found that T cells from MR treated mice had no defects in proliferation, even though we continued the MR conditions ex vivo. We also identified that CBS is most highly correlated with smoking, adenocarcinomas with alveolar and bronchiolar features, and adenosquamous cell carcinomas, implicating its roles in oxidative stress response and lineage fate in human tumors. Taken together, we have shown the importance of MR as a dietary intervention to slow tumor growth and improve treatment outcomes for NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0df2616d8ba063728db0011a0955e84576b33e17" target='_blank'>
              Methionine Restriction Reduces Lung Cancer Progression and Increases Chemotherapy Response
              </a>
            </td>
          <td>
            Kassandra J Naughton, Xiulong Song, Avery R Childress, Erika M Skaggs, Aria L. Byrd, Christian M. Gosser, Dave-Preston Esoe, Tanner J DuCote, Daniel Plaugher, Alexsandr Lukyanchuk, Ryan A. Goettl, Jinpeng Liu, C. F. Brainson
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e4d778cae3d7e3203dfad059efb7f4865240dc4" target='_blank'>
              Insights into treatment-specific prognostic somatic mutations in NSCLC from the AACR NSCLC GENIE BPC cohort analysis
              </a>
            </td>
          <td>
            Yi Liu, Sindhu Yalavarthi, Fan Yang, Yusif Abdul-Rashid, Shenkun Tang, Zihe Long, Yongkai Qin, Kerui Wu, Zhifei Wang
          </td>
          <td>2024-07-02</td>
          <td>BMC Pulmonary Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Genome sequencing has revealed frequent mutations in Ras homolog family member A (RHOA) among various cancers with unique aberrant profiles and pathogenic effects, especially in peripheral T-cell lymphoma (PTCL). The discrete positional distribution and types of RHOA amino acid substitutions vary according to the tumor type, thereby leading to different functional and biological properties, which provide new insight into the molecular pathogenesis and potential targeted therapies for various tumors. However, the similarities and discrepancies in characteristics of RHOA mutations among various histologic subtypes of PTCL have not been fully elucidated. Herein we highlight the inconsistencies and complexities of the type and location of RHOA mutations and demonstrate the contribution of RHOA variants to the pathogenesis of PTCL by combining epigenetic abnormalities and activating multiple downstream pathways. The promising potential of targeting RHOA as a therapeutic modality is also outlined. This review provides new insight in the field of personalized medicine to improve the clinical outcomes for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd71cf13763be0cc23a4dc23d32cf828f50620b5" target='_blank'>
              Mutations in Ras homolog family member A in patients with peripheral T-cell lymphoma and implications for personalized medicine.
              </a>
            </td>
          <td>
            Lina Hu, Xuanye Zhang, Shengbing Zang
          </td>
          <td>2024-08-09</td>
          <td>Cancer biology & medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40fcdfbbc2df4270401ef8beed0b78736d658d46" target='_blank'>
              DNA methylation in mammalian development and disease.
              </a>
            </td>
          <td>
            Zachary D. Smith, Sara Hetzel, Alexander Meissner
          </td>
          <td>2024-08-12</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The combination of CDK4/6 and MEK inhibition as a therapeutic strategy has shown promise in various cancer models, particularly in those harboring RAS mutations. An initial high-throughput drug screen identified a high synergy between the CDK4/6 inhibitor palbociclib and the MEK inhibitor trametinib when used in combination in soft tissue sarcomas. In RAS mutant models, combination treatment with palbociclib and trametinib induced significant G1 cell cycle arrest, resulting in a marked reduction in cell proliferation and growth. CRISPR-mediated RB1 depletion resulted in a decreased response to CDK4/6 and MEK inhibition, which was validated in both cell culture and xenograft models. Beyond its cell cycle inhibitory effects, pathway enrichment analysis revealed the robust activation of interferon pathways upon CDK4/6 and MEK inhibition. This induction of gene expression was associated with the upregulation of retroviral elements. The TBK1(TANK-binding kinase 1) inhibitor GSK8612 selectively blocked the induction of interferon-related genes induced by palbociclib and trametinib treatment, and highlighted the separable epigenetic responses elicited by combined CDK4/6 and MEK inhibition. Together, these findings provide key mechanistic insights into the therapeutic potential of CDK4/6 and MEK inhibition in soft tissue sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/721445995cd4f6ad7aae623c32a235fcb7645dbc" target='_blank'>
              Separable cell cycle arrest and immune response elicited through pharmacological CDK4/6 and MEK inhibition in RASmut disease models.
              </a>
            </td>
          <td>
            Jin Wu, Jianxin Wang, Thomas N O'Connor, Stephanie L. Tzetzo, Katerina V Gurova, Erik S. Knudsen, Agnieszka K Witkiewicz
          </td>
          <td>2024-08-16</td>
          <td>Molecular cancer therapeutics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Simple Summary The atavistic theory of cancer suggests that cancer development proceeds through cells reverting to ancient survival mechanisms. The Serial Atavism Model (SAM) expands on this, proposing that cancer progresses through multiple stages of reversion to earlier evolutionary forms with cells losing modern traits and regaining primitive ones. One example is the Warburg effect, where cancer cells prefer a type of energy production used by ancient cells before Earth’s atmosphere had oxygen. However, this review argues that cancer metabolism is too complex to be fully explained by this theory. Cancer cells exhibit a wide range of metabolic behaviors that do not fit neatly into a pattern of reverting to an ancient state, indicating that the SAM may not provide a complete understanding of cancer. Abstract The atavistic theory of cancer posits that cancer emerges and progresses through the reversion of cellular phenotypes to more ancestral types with genomic and epigenetic changes deactivating recently evolved genetic modules and activating ancient survival mechanisms. This theory aims at explaining the known cancer hallmarks and the paradox of cancer’s predictable progression despite the randomness of genetic mutations. Lineweaver and colleagues recently proposed the Serial Atavism Model (SAM), an enhanced version of the atavistic theory, which suggests that cancer progression involves multiple atavistic reversions where cells regress through evolutionary stages, losing recently evolved traits first and reactivating primitive ones later. The Warburg effect, where cancer cells upregulate glycolysis and lactate production in the presence of oxygen instead of using oxidative phosphorylation, is one of the key feature of the SAM. It is associated with the metabolism of ancient cells living on Earth before the oxygenation of the atmosphere. This review addresses the question of whether cancer metabolism can be considered as an atavistic reversion. By analyzing several known characteristics of cancer metabolism, we reach the conclusion that this version of the atavistic theory does not provide an adequate conceptual frame for cancer research. Cancer metabolism spans a whole spectrum of metabolic states which cannot be fully explained by a sequential reversion to an ancient state. Moreover, we interrogate the nature of cancer metabolism and discuss its characteristics within the framework of the SAM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f5f41e098ff498aa32258ffc44af055a9872bf7" target='_blank'>
              Is Cancer Metabolism an Atavism?
              </a>
            </td>
          <td>
            Eric Fanchon, A. Stéphanou
          </td>
          <td>2024-06-29</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The main epidemiological and clinical data on colorectal cancer, as well as the features of molecular pathology, are discussed in the literature review. Efforts are being putto identify promising targets, particularly small non-coding nucleotide sequences, which can lead to new treatments for this disease. The discovery of significant mutations that contribute to the development of colorectal tumors is a major step in the advancement of molecular oncology, as these mutations give rise to heterogeneous tumors that differ in their origin. These mutations play a significant role in the progression of the disease and are now being targeted for treatment. The prognosis for a disease is influenced by the patient's sensitivity to antitumor therapy. However, new approaches to finding effective targets for antitumor treatments face new fundamental challenges due to clinical issues. These issues include the epigenetic regulation of markers of oncogenesis, which allows for the development of new therapeutic strategies. RNA interference, in particular, has been linked to non-copying RNA sequences such as microRNAs. These microRNAs are associated with certain processes that can influence all aspects of oncogenesis. The diversity of microRNAs allows for a differentiated approach when treating tumors in various locations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4796e80e11216dc287a04bf6cbff831c3d542a28" target='_blank'>
              New Perspectives in Colorectal Cancers Treatment, the Role of MicroRNAs.
              </a>
            </td>
          <td>
            V. A. Belova, Liudmila V Spirina, A. Avgustinovich, S. G. Afanasiev, Maxim Yu Volkov M, D. I. Azovsky, Alexander M Volkov, T. S. Klyushina
          </td>
          <td>2024-07-24</td>
          <td>Current drug targets</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92d22ec19c45e84786f29e00379dcc92c97dbcbf" target='_blank'>
              Identifying key genes and functionally enriched pathways in acute myeloid leukemia by weighted gene co-expression network analysis.
              </a>
            </td>
          <td>
            Jimo Jian, Chenglu Yuan, Hongyuan Hao
          </td>
          <td>2024-07-09</td>
          <td>Journal of applied genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/502a5fb073ed81392e0d208c32c77f69b33b47ec" target='_blank'>
              The contributions of cytogenetics, genetics, and epigenetics to the stability of plants polyploidy
              </a>
            </td>
          <td>
            Arrashid Harun, Zhongming Fang, Chunli Chen
          </td>
          <td>2024-07-30</td>
          <td>Discover Plants</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="G-quadruplex (G4) sequences, which can fold into higher-order G4 structures, are abundant in the human genome and are over-represented in the promoter regions of many genes involved in human cancer initiation, progression, and metastasis. They are plausible targets for G4-binding small molecules, which would, in the case of promoter G4s, result in the transcriptional downregulation of these genes. However, structural information is currently available on only a very small number of G4s and their ligand complexes. This limitation, coupled with the currently restricted information on the G4-containing genes involved in most complex human cancers, has led to the development of a phenotypic-led approach to G4 ligand drug discovery. This approach was illustrated by the discovery of several generations of tri- and tetra-substituted naphthalene diimide (ND) ligands that were found to show potent growth inhibition in pancreatic cancer cell lines and are active in in vivo models for this hard-to-treat disease. The cycles of discovery have culminated in a highly potent tetra-substituted ND derivative, QN-302, which is currently being evaluated in a Phase 1 clinical trial. The major genes whose expression has been down-regulated by QN-302 are presented here: all contain G4 propensity and have been found to be up-regulated in human pancreatic cancer. Some of these genes are also upregulated in other human cancers, supporting the hypothesis that QN-302 is a pan-G4 drug of potential utility beyond pancreatic cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f12d7617e98441c66bfb53bc671f9147eed0b1fe" target='_blank'>
              A Phenotypic Approach to the Discovery of Potent G-Quadruplex Targeted Drugs
              </a>
            </td>
          <td>
            Stephen Neidle
          </td>
          <td>2024-08-01</td>
          <td>Molecules</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract The comprehensive genomic analysis of the head and neck squamous cell carcinoma (HNSCC) oncogenome revealed the frequent loss of p16INK4A (CDKN2A) and amplification of cyclin D1 genes in most human papillomavirus–negative HNSCC lesions. However, cyclin-dependent kinase 4 and 6 (CDK4/6) inhibitors have shown modest effects in the clinic. The aberrant activation of the PI3K/mTOR pathway is highly prevalent in HNSCC, and recent clinical trials have shown promising clinical efficacy of mTOR inhibitors (mTORi) in the neoadjuvant and adjuvant settings but not in patients with advanced HNSCC. By implementing a kinome-wide CRISPR/Cas9 screen, we identified cell-cycle inhibition as a synthetic lethal target for mTORis. A combination of mTORi and palbociclib, a CDK4/6-specific inhibitor, showed strong synergism in HNSCC-derived cells in vitro and in vivo. Remarkably, we found that an adaptive increase in cyclin E1 (CCNE1) expression upon palbociclib treatment underlies the rapid acquired resistance to this CDK4/6 inhibitor. Mechanistically, mTORi inhibits the formation of eIF4G–CCNE1 mRNA complexes, with the consequent reduction in mRNA translation and CCNE1 protein expression. Our findings suggest that mTORi reverts the adaptive resistance to palbociclib. This provides a multimodal therapeutic option for HNSCC by cotargeting mTOR and CDK4/6, which in turn may halt the emergence of palbociclib resistance. Significance: A kinome-wide CRISPR/Cas9 screen identified cell-cycle inhibition as a synthetic lethal target of mTORis. A combination of mTORi and palbociclib, a CDK4/6-specific inhibitor, showed strong synergistic effects in HNSCC. Mechanistically, mTORis inhibited palbociclib-induced increase in CCNE1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45d8015b7ff294926111ccbf929e88b6d333407e" target='_blank'>
              A Kinome-Wide Synthetic Lethal CRISPR/Cas9 Screen Reveals That mTOR Inhibition Prevents Adaptive Resistance to CDK4/CDK6 Blockade in HNSCC
              </a>
            </td>
          <td>
            Yusuke Goto, Keiichi Koshizuka, Toshinori Ando, Hiroki Izumi, Xingyu Wu, Kuniaki Sato, Tomohiko Ishikawa, Kyle Ford, Xiaodong Feng, Zhiyong Wang, Nadia Arang, M. Allevato, Ayush Kishore, Prashant Mali, J. Gutkind
          </td>
          <td>2024-07-01</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5096f65eca5658128881222e362e03cc7a2beb2" target='_blank'>
              H3K27ac acts as a molecular switch for doxorubicin-induced activation of cardiotoxic genes
              </a>
            </td>
          <td>
            Yu Hong, Xinlan Li, Jia Li, Qiuyi He, Manbing Huang, Yubo Tang, Xiao Chen, Jie Chen, Ke-Jing Tang, Chao Wei
          </td>
          <td>2024-07-16</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Survival differences exist in colorectal cancer (CRC) patients by sex and disease stage. However, the potential molecular mechanism(s) are not well understood. Here we show that asparagine synthetase (ASNS) and G protein-coupled estrogen receptor-1 (GPER1) are critical sensors of nutrient depletion and linked to poorer outcomes for females with CRC. Using a 3D spheroid model of isogenic SW48 KRAS wild-type (WT) and G12A mutant (MT) cells grown under a restricted nutrient supply, we found that glutamine depletion inhibited cell growth in both cell lines, whereas ASNS and GPER1 expression were upregulated in KRAS MT versus WT. Estradiol decreased growth in KRAS WT but had no effect on MT cells. Selective GPER1 and ASNS inhibitors suppressed cell proliferation with increased caspase-3 activity of MT cells under glutamine depletion condition particularly in the presence of estradiol. In a clinical colon cancer cohort from The Cancer Genome Atlas, both high GPER1 and ASNS expression were associated with poorer overall survival for females only in advanced stage tumors. These results suggest KRAS MT cells have mechanisms in place that respond to decreased nutrient supply, typically observed in advanced tumors, by increasing the expression of ASNS and GPER1 to drive cell growth. Furthermore, KRAS MT cells are resistant to the protective effects of estradiol under nutrient deplete conditions. The findings indicate that GPER1 and ASNS expression, along with the interaction between nutrient supply and KRAS mutations shed additional light on the mechanisms underlying sex differences in metabolism and growth in CRC, and have clinical implications in the precision management of KRAS mutant CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09b210dc926e8881982ade4ae07595a6df40faf4" target='_blank'>
              Asparagine synthetase and G-protein coupled estrogen receptor are critical responders to nutrient supply in KRAS mutant colorectal cancer.
              </a>
            </td>
          <td>
            Lingeng Lu, Qian Zhang, Oladimeji Aladelokun, Domenica Berardi, Xinyi Shen, Audrey Marin, Rolando Garcia-Milian, Jatin Roper, Sajid A. Khan, Caroline H. Johnson
          </td>
          <td>2024-07-22</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d2a4f583e73c9388ee06b9414fc44f7c5cdf6bf" target='_blank'>
              Whole genome sequencing refines stratification and therapy of patients with clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            R. Culliford, Sam Lawrence, Charlie Mills, Z. Tippu, D. Chubb, A. Cornish, Lisa Browning, B. Kinnersley, R. Bentham, Amit Sud, H. Pallikonda, A. Frangou, Andreas J. Gruber, K. Litchfield, David C Wedge, James Larkin, S. Turajlic, R. Houlston
          </td>
          <td>2024-07-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Introduction Esophageal cancer presents significant challenges due to limited treatment options and poor prognosis, particularly in advanced stages. Dysregulated long non-coding RNAs (lncRNAs) are implicated in cancer progression and treatment resistance. This study investigated the roles of dysregulated lncRNA NONHSAT227443.1, identified through lncRNA-seq, and its downstream target gene MRTFB in esophageal squamous cell carcinoma (ESCC). Methods Dysregulated lncRNAs were identified through lncRNA-seq in esophageal cancer tissues with varying chemotherapy response. The regulatory interaction of overexpressed NONHSAT227443.1 was assessed using quantitative real-time polymerase chain reaction (qRT-PCR) and western blotting. Functional assays, including cell viability, cell proliferation, and flow cytometry analyses, were performed to comprehensively investigate the influence of NONHSAT227443.1 and its downstream molecules on ESCC. Results NONHSAT227443.1 was significantly overexpressed in paclitaxel plus platinum chemotherapy non-responders and esophageal cancer cell lines. Chemotherapy exposure led to diminished NONHSAT227443.1 expression. NONHSAT227443.1 negatively regulated MRTFB expression, and their combined dysregulation correlated with increased cancer activity, proliferation, and suppressed apoptosis. Diminished MRTFB expression was associated with PI3K/AKT pathway activation. Conclusion Our study provides insights into NONHSAT227443.1 and MRTFB roles in esophageal cancer, contributing to aggressive traits and treatment resistance. NONHSAT227443.1 and MRTFB may serve as potential therapeutic targets to enhance the response to paclitaxel plus platinum chemotherapy in non-responsive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/76b8641f3f03ccc5e27fc76600d3a4f6465206b9" target='_blank'>
              LncRNA NONHSAT227443.1 Confers Esophageal Squamous Cell Carcinoma Chemotherapy Resistance by Activating PI3K/AKT Signaling via Targeting MRTFB
              </a>
            </td>
          <td>
            Yuchen Wang, Yingying Wang, Jinze Zhang, Zhihua Shi, Junfeng Liu
          </td>
          <td>2024-08-16</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary This study explores the detection of gene fusions in acute leukemia using targeted RNA sequencing and compares its effectiveness with traditional methods like karyotyping and RT-PCR. Gene fusions, which result from chromosomal rearrangements, are crucial for diagnosing and treating leukemia. Recent updates in diagnostic criteria emphasize the importance of identifying these gene fusions. Targeted RNA sequencing offers advantages such as rapid diagnosis, minimal training requirements, and cost-effectiveness, making it a practical method in clinical settings. Our findings suggest that targeted RNA sequencing can accurately identify gene fusions, potentially benefiting nearly half of leukemia patients and improving diagnostic precision. This method may significantly impact the research community by providing a reliable and efficient tool for leukemia diagnosis. Abstract Gene fusions are key drivers in acute leukemia, impacting diagnosis and treatment decisions. We analyzed 264 leukemia patients using targeted RNA sequencing with conventional karyotyping and reverse transcription polymerase chain reaction (RT-PCR). Leukemic fusions were detected in 127 patients (48.1%). The new guidelines introduced additional diagnostic criteria, expanding the spectrum of gene fusions. We discovered three novel fusions (RUNX1::DOPEY2, RUNX1::MACROD2, and ZCCHC7::LRP1B). We analyzed recurrent breakpoints for the KMT2A and NUP98 rearrangements. Targeted RNA sequencing showed consistent results with RT-PCR in all tested samples. However, when compared to conventional karyotyping, we observed an 83.3% concordance rate, with 29 cases found only in targeted RNA sequencing, 7 cases with discordant results, and 5 cases found only in conventional karyotyping. For the five cases where known leukemic gene rearrangements were suspected only in conventional karyotyping, we conducted additional messenger RNA sequencing in four cases and proved no pathogenic gene rearrangements. Targeted RNA sequencing proved advantageous for the rapid and accurate interpretation of gene rearrangements. The concurrent use of multiple methods was essential for a comprehensive evaluation. Comprehensive molecular analysis enhances our understanding of leukemia’s genetic basis, aiding diagnosis and classification. Advanced molecular techniques improve clinical decision-making, offering potential benefits.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f0977b67570006625235107c94ecedeb3ed6b3" target='_blank'>
              Real-World Clinical Utility of Targeted RNA Sequencing in Leukemia Diagnosis and Management
              </a>
            </td>
          <td>
            Seo Wan Kim, N. Kim, Yu Jeong Choi, Seung-Tae Lee, Jong Rak Choi, Saeam Shin
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b903232278019fc20d595adb95c47c4a0e3a7532" target='_blank'>
              Deletion of IRC19 Causes Defects in DNA Double-Strand Break Repair Pathways in Saccharomyces cerevisiae.
              </a>
            </td>
          <td>
            Ju-Hee Choi, Oyungoo Bayarmagnai, Sung-Ho Bae
          </td>
          <td>2024-07-12</td>
          <td>Journal of microbiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background DNA methylation alterations within the VMP1/MIR21 gene region, a potential epigenetic biomarker of systemic inflammation, have been demonstrated in mononuclear blood cells from early breast cancer (BC) patients after chemotherapy. Whether these changes are present in granulocytes, persist in the years after treatment, and affect VMP1 or MIR21 gene expression, remains unknown. Aim We aimed to investigate whether adjuvant chemotherapy alters the DNA methylation and gene expression of VMP1/MIR21 in granulocytes from postmenopausal BC patients and, if so, whether these treatment-induced changes are reversible in the first two years after completed chemotherapy. Methods Whole blood samples were obtained from 30 postmenopausal BC patients before chemotherapy and every six months for two years, and from 10 healthy age- and BMI-matched controls. DNA and RNA was extracted from isolated granulocytes, and DNA methylation of four CpG sites located in the gene body of VMP1, which is situated in the promoter region of MIR21, was assessed through bisulfite pyrosequencing. qPCR was used for assessment of VMP1 and MIR21 expression. Results VMP1/MIR21 was significantly demethylated in granulocytes from BC patients shortly after completed chemotherapy compared to before (10 percentage points decrease, p<0.0001). Six months thereafter, DNA methylation values were significantly increased (6 percentage points, p = 0.002), and they were further increased to pre-chemotherapy levels 12, 18 and 24 months post chemotherapy. Chemotherapy did not cause significant changes in the expression of VMP1 or MIR21. Conclusion The unique follow-up samples in this study demonstrated that chemotherapy induced a transient reduction in DNA methylation of the VMP1/MIR21 region in granulocytes from postmenopausal BC patients. Although transient, chemotherapy-induced epigenetic changes in blood cells may contribute to the increased risk of inflammatory-related comorbidities in BC survivors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2639ac538ad006164281d688a94e8e3cef9a9357" target='_blank'>
              Chemotherapy causes a reversible decrease in VMP1/MIR21 DNA methylation in granulocytes from breast cancer survivors
              </a>
            </td>
          <td>
            Marie-Louise Abrahamsen, Zuzanna Jachowicz, Kristian Buch-Larsen, Djordje Marina, Michael Andersson, Peter Schwarz, F. Dela, Linn Gillberg
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="OBJECTIVE
Ionizing radiation generates genomic instability by promoting the accumulation of chromosomal rearrangements. The oncogenic translocation RET/PTC1 is present in more than 70% of radiation-induced thyroid cancers. Both RET and CCDC6, the genes implicated in RET/PTC1, are found within common fragile sites - chromosomal regions prone to DNA breakage during slight replication stress. Given that irradiated cells become more susceptible to genomic destabilization due to the accumulation of replication-stress-related double-strand breaks (DSBs), we explored whether RET and CCDC6 exhibit DNA breakage under replicative stress several days post-irradiation of thyroid cells.


METHODS
We analyzed the dynamic of DNA replication in human thyroid epithelial cells (HThy-ori-3.1) 4 days post a 5-Gy exposure using molecular DNA combing. The DNA replication schedule was evaluated through replication-timing experiments. We implemented a ChIP-qPCR assay to determine whether the RET and CCDC6 genes break following irradiation.


RESULTS
Our study indicates that replicative stress, occurring several days post-irradiation in thyroid cells, primarily causes DSBs in the RET gene. We discovered that both the RET and CCDC6 genes undergo late replication in thyroid cells. However, only RET's replication rate is notably delayed after irradiation.


CONCLUSION
The findings suggest that post-irradiation in the RET gene causes a breakage in the replication fork, which could potentially invade another genomic area, including CCDC6. As a result, this could greatly contribute to the high prevalence of chromosomal RET/PTC rearrangements seen in patients exposed to external radiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eea49490c4342b6b738c402ce79c72ea23e5f552" target='_blank'>
              A post-irradiation-induced replication stress promotes RET proto-oncogene breakage.
              </a>
            </td>
          <td>
            Fabio Hecht, Laura Valerio, Carlos Frederico Lima Gonçalves, Marylin Harinquet, Rabii Ameziane El Hassani, Denise P. Carvalho, Stephane Koundrioukoff, Jean-Charles Cadoret, Corinne Dupuy
          </td>
          <td>2024-07-01</td>
          <td>European thyroid journal</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background/Aim: BRCA1/2 mutations in breast cancer cells impair homologous recombination and promote alternative end joining (Alt-EJ) for DNA-damage repair. DNA polymerase theta, encoded by POLQ, plays a crucial role in Alt-EJ, making it a potential therapeutic target, particularly in BRCA1/2-mutant cancers. Methionine restriction is a promising approach to target cancer cells due to their addiction to this amino acid. The present study investigated the expression of POLQ in BRCA1/2 wild-type and BRCA1-mutant breast cancer cells under methionine restriction. Materials and Methods: POLQ mRNA expression was measured using qRT-PCR in BRCA1/2 wild-type (MDA-MB-231) and BRCA1- mutant (HCC1937 and MDA-MB-436) breast-cancer cells under normal, or serum-restricted, or serum- and methionine-restricted conditions. Results: Compared to BRCA1/2 wild-type cells, BRCA1-mutant cells displayed significantly higher basal POLQ expression in normal medium. Methionine restriction further increased POLQ expression in the BRCA1-mutant cells but decreased it in the BRCA1/2 wild-type cells. Conclusion: The present findings suggest that methionine restriction showed differential effects on POLQ expression, potentially impacting Alt-EJ activity, in BRCA1/2 wild-type and BRCA1-mutant breast-cancer cells. Further investigation is needed to explore the potential of combining methionine restriction with DNA-repair inhibitors, such as PARP inhibitors, to overcome drug resistance in BRCA1/2 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0523a0d152aaab9d94d339ede11927c41a78c4b" target='_blank'>
              Induction of the DNA-Repair Gene POLQ only in BRCA1-mutant Breast-Cancer Cells by Methionine Restriction
              </a>
            </td>
          <td>
            T. Kunihisa, Sachiko Inubushi, Hirokazu Tanino, Robert M. Hoffman
          </td>
          <td>2024-06-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer characterized by late diagnosis, rapid progression, and a high mortality rate. Its complex biology, characterized by a dense, stromal tumor environment with an immunosuppressive milieu, contributes to resistance against standard treatments like chemotherapy and radiation. This comprehensive review explores the dynamic role of the microbiome in modulating chemotherapy efficacy and outcomes in PDAC. It delves into the microbiome’s impact on drug metabolism and resistance, and the interaction between microbial elements, drugs, and human biology. We also highlight the significance of specific bacterial species and microbial enzymes in influencing drug action and the immune response in the tumor microenvironment. Cutting-edge methodologies, including artificial intelligence, low-biomass microbiome analysis and patient-derived organoid models, are discussed, offering insights into the nuanced interactions between microbes and cancer cells. The potential of microbiome-based interventions as adjuncts to conventional PDAC treatments are discussed, paving the way for personalized therapy approaches. This review synthesizes recent research to provide an in-depth understanding of how the microbiome affects chemotherapy efficacy. It focuses on elucidating key mechanisms and identifying existing knowledge gaps. Addressing these gaps is crucial for enhancing personalized medicine and refining cancer treatment strategies, ultimately improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33468af8b48c7327c4b215eb7932772751d9192f" target='_blank'>
              Microbiome and pancreatic cancer: time to think about chemotherapy
              </a>
            </td>
          <td>
            J. de Castilhos, Katharina Tillmanns, Jana Blessing, Arnelyn Laraño, Vadim Borisov, Christoph K Stein-Thoeringer
          </td>
          <td>2024-07-18</td>
          <td>Gut Microbes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer (mCRPC) is a lethal disease that resists therapy targeting androgen signaling, the primary driver of prostate cancer. mCRPC resists androgen receptor (AR) inhibitors by amplifying AR signaling or by evolving into therapy-resistant subtypes that do not depend on AR. Elucidation of the epigenetic underpinnings of these subtypes could provide important insights into the drivers of therapy resistance. In this study, we produced chromatin accessibility maps linked to the binding of lineage-specific transcription factors (TF) by performing ATAC sequencing on 70 mCRPC tissue biopsies integrated with transcriptome and whole genome sequencing. mCRPC had a distinct global chromatin accessibility profile linked to AR function. Analysis of TF occupancy across accessible chromatin revealed 203 TFs associated with mCRPC subtypes. Notably, ZNF263 was identified as a putative prostate cancer TF with a significant impact on gene activity in the double-negative (AR- neuroendocrine-) subtype, potentially activating MYC targets. Overall, this analysis of chromatin accessibility in mCRPC provides valuable insights into epigenetic changes that occur during progression to mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac71bf183535cdb84fd367ef22969e3b56650628" target='_blank'>
              An Atlas of Accessible Chromatin in Advanced Prostate Cancer Reveals the Epigenetic Evolution during Tumor Progression.
              </a>
            </td>
          <td>
            Raunak Shrestha, L. Chesner, Meng Zhang, Stanley Zhou, A. Foye, A. Lundberg, Alana S Weinstein, M. Sjöström, Xiaolin Zhu, Thaidy Moreno-Rodriguez, Haolong Li, Su C/Pcf West Coast Prostate Cancer Dream Team, J. Alumkal, Rahul Aggarwal, Eric J Small, Mathieu Lupien, David A Quigley, Felix Y Feng
          </td>
          <td>2024-07-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Human telomerase reverse transcriptase (hTERT) may have noncanonical functions in transcriptional regulation and metabolic reprogramming in cancer cells, but it is a challenging target. We thus developed small-molecule ligands targeting hTERT promoter G-quadruplex DNA structures (hTERT G4) to downregulate hTERT expression. Ligand 5 showed high affinity toward hTERT G4 (Kd = 1.1 μM) and potent activity against triple-negative breast cancer cells (MDA-MB-231, IC50 = 1 μM). In cell-based assays, 5 not only exerts markedly inhibitory activity on classical telomere functions including decreased telomerase activity, shortened telomere length, and cellular senescence but also induces DNA damage, acute cellular senescence, and apoptosis. This study reveals that hTERT G4-targeting ligand may cause mitochondrial dysfunction, disrupt iron metabolism and activate ferroptosis in cancer cells. The in vivo antitumor efficacy of 5 was also evaluated in an MDA-MB-231 xenograft mouse model and approximately 78.7% tumor weight reduction was achieved. No observable toxicity against the major organs was observed.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1d3d47d8b1822b8c2901a320f1107b7691744c1" target='_blank'>
              Targeting hTERT Promoter G-Quadruplex DNA Structures with Small-Molecule Ligand to Downregulate hTERT Expression for Triple-Negative Breast Cancer Therapy.
              </a>
            </td>
          <td>
            W. Long, Yao-Xun Zeng, Bo-Xin Zheng, Yu-Bo Li, Ya-Kun Wang, K. Chan, Meng-Ting She, Yu-Jing Lu, Chunyang Cao, Wing-Leung Wong
          </td>
          <td>2024-07-10</td>
          <td>Journal of medicinal chemistry</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7189b9bbbd6276b2c06531ac44b4da640c222158" target='_blank'>
              Small cell lung cancer: emerging subtypes, signaling pathways, and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Jing Zhang, Xiaoping Zeng, Qiji Guo, Zhenxin Sheng, Yan Chen, Shiyue Wan, Lele Zhang, Peng Zhang
          </td>
          <td>2024-08-05</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="To realize the potential of engineered cells in therapeutic applications, transgenes must be expressed within the window of therapeutic efficacy. Differences in copy number and other sources of extrinsic noise generate variance in transgene expression and limit the performance of synthetic gene circuits. In a therapeutic context, supraphysiological expression of transgenes can compromise engineered phenotypes and lead to toxicity. To ensure a narrow range of transgene expression, we design and characterize Compact microRNA-Mediated Attenuator of Noise and Dosage (ComMAND), a single-transcript, microRNA-based incoherent feedforward loop. We experimentally tune the ComMAND output profile, and we model the system to explore additional tuning strategies. By comparing ComMAND to two-gene implementations, we highlight the precise control afforded by the single-transcript architecture, particularly at relatively low copy numbers. We show that ComMAND tightly regulates transgene expression from lentiviruses and precisely controls expression in primary human T cells, primary rat neurons, primary mouse embryonic fibroblasts, and human induced pluripotent stem cells. Finally, ComMAND effectively sets levels of the clinically relevant transgenes FMRP1 and FXN within a narrow window. Together, ComMAND is a compact tool well-suited to precisely specify expression of therapeutic cargoes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/736723e302bd2cb6b51bf128d0280bc0a6cf2378" target='_blank'>
              Model-guided design of microRNA-based gene circuits supports precise dosage of transgenic cargoes into diverse primary cells
              </a>
            </td>
          <td>
            K. S. Love, Christopher P. Johnstone, Emma L. Peterman, Stephanie Gaglione, Kate E. Galloway
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fcc48036440e068784b3b542811aace316e8c88" target='_blank'>
              Clinical application of whole-genome sequencing of solid tumors for precision oncology.
              </a>
            </td>
          <td>
            R. Kim, Seokhwi Kim, B. B. Oh, Woo Sik Yu, Chang Woo Kim, Hoon Hur, Sang-Yong Son, Min Jae Yang, Dae Sung Cho, Taeyang Ha, Subin Heo, Jeon Yeob Jang, Jae Sung Yun, Kyu-sung Kwack, Jai Keun Kim, Jimi Huh, S. Lim, Sang-Uk Han, Hyun Woo Lee, Ji Eun Park, Chul-Ho Kim, J. Roh, Young Wha Koh, Dakeun Lee, Jang-Hee Kim, G. Lee, Choong-Kyun Noh, Y. Jung, Ji Won Park, Seungsoo Sheen, M. S. Ahn, Yong Won Choi, Tae-Hwan Kim, Seok Yun Kang, Jin-Hyuk Choi, Soo Yeon Baek, K. Lee, Sun Il Kim, Sung Hyun Noh, Se-Hyuk Kim, Hyemin Hwang, Eunjung Joo, Shinjung Lee, Jong-Yeon Shin, Ji-Young Yun, Jung-Woon Park, Kijong Yi, Youngoh Kwon, W. Lee, H. Park, Joonoh Lim, Boram Yi, Jaemo Koo, June-Young Koh, Sangmoon Lee, Yuna Lee, , E. Connolly-Strong, Y. S. Ju, Minsuk Kwon
          </td>
          <td>2024-08-13</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Changes in gene dosage can have tremendous evolutionary potential (e.g. whole-genome duplications), but without compensatory mechanisms, they can also lead to gene dysregulation and pathologies. Sex chromosomes are a paradigmatic example of naturally occurring gene dosage differences and their compensation. In species with chromosome-based sex determination, individuals within the same population necessarily show 'natural' differences in gene dosage for the sex chromosomes. In this Review, we focus on the mammalian X chromosome and discuss recent new insights into the dosage-compensation mechanisms that evolved along with the emergence of sex chromosomes, namely X-inactivation and X-upregulation. We also discuss the evolution of the genetic loci and molecular players involved, as well as the regulatory diversity and potentially different requirements for dosage compensation across mammalian species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016ee497a5dab2410dc213e6a7a7d311ff9fabfc" target='_blank'>
              Compensation of gene dosage on the mammalian X.
              </a>
            </td>
          <td>
            Daniela Cecalev, Beatriz Viçoso, Rafael Galupa
          </td>
          <td>2024-08-01</td>
          <td>Development</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Metabolic reprogramming is recognized as a hallmark of cancer, enabling cancer cells to acquire essential biomolecules for cell growth, often characterized by upregulated glycolysis and/or fatty acid synthesis-related genes. The transcription factor forkhead box M1 (FOXM1) has been implicated in various cancers, contributing significantly to their development, including colorectal cancer (CRC), a major global health concern. Despite FOXM1's established role in cancer, its specific involvement in the Warburg effect and fatty acid biosynthesis in CRC remains unclear. We analyzed The Cancer Genome Atlas (TCGA) Colonic Adenocarcinoma and Rectal Adenocarcinoma (COADREAD) datasets to to derive the correlation of the expression levels between FOXM1 and multiple genes and the survival prognosis based on FOXM1 expression. Using two human CRC cell lines, HT29 and HCT116, we conducted RNAi or plasmid transfection procedures, followed by a series of assays, including RNA extraction, quantitative real-time polymerase chain reaction, Western blot analysis, cell metabolic assays, and immunofluorescence analysis. Higher expression levels of FOXM1 correlated with a poorer survival prognosis, and the expression of FOXM1 was positively correlated with glycolysis-related genes SLC2A1 and LDHA, de novo lipogenesis-related genes ACACA and FASN, and MYC. FOXM1 appeared to modulate AKT/mTOR signaling, the expression of c-Myc, proteins related to glycolysis and fatty acid biosynthesis, as well as extracellular acidification rate in HT29 and HCT116 cells. In summary, FOXM1 plays a regulatory role in glycolysis, fatty acid biosynthesis, and cellular energy consumption, thereby influencing CRC cell growth and patient prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60e3baf49a68626e2bd59e387a99b97ce66e27bb" target='_blank'>
              Forkhead box M1 mediates metabolic reprogramming in human colorectal cancer cells.
              </a>
            </td>
          <td>
            Po-Chen Li, Sheng-Yu Dai, Yu-Shun Lin, Yu-Tsen Chang, Chen-Chia Liu, I-Ching Wang, Ming-Fen Lee
          </td>
          <td>2024-07-02</td>
          <td>American journal of physiology. Gastrointestinal and liver physiology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Pancreatic ductal adenocarcinoma-(PDAC) needs innovative approaches due to its 12% 5-year survival despite current therapies. We show marked sensitivity of pancreatic cancer cells to the combination of a novel eIF4A inhibitor, des-methyl pateamine A (DMPatA), and a histone deacetylase inhibitor, romidepsin, inducing epigenetic reprogramming as an innovative therapeutic strategy. Exploring the mechanistic activity of this combination showed that with a short duration of romidepsin at low doses, robust acetylation persisted up to 48h with the combination, while histone acetylation rapidly faded with monotherapy. This represents an unexpected mechanism of action against PDAC cells that triggers transcriptional overload, metabolic stress, and augmented DNA damage. Structurally different class I HDAC inhibitors exhibit the same hyperacetylation patterns when co-administered with DMPatA, suggesting a class effect. We show efficacy of this combination regimen against tumor growth in a MIA PaCa-2 xenograft model of PDAC with persistent hyperacetylation confirmed in tumor samples. STATEMENT OF SIGNIFICANCE Pancreatic ductal adenocarcinoma, a significant clinical challenge, could benefit from the latent potential of epigenetic therapies like HDAC inhibitors-(HDIs), typically limited to hematological malignancies. Our study shows that a synergistic low dose combination of HDIs with an eIF4A-inhibitor in pancreatic cancer models results in marked pre-clinical efficacy, offering a promising new treatment strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e39a30fe843d024db3df2071d132b06103faa3c8" target='_blank'>
              Combined HDAC and eIF4A inhibition: A novel epigenetic therapy for pancreatic adenocarcinoma
              </a>
            </td>
          <td>
            Maryam Safari, Luigi Scotto, A. Basseville, Thomas Litman, Haoran Xue, Luba Petrukhin, Ping Zhou, Diana V. Morales, Christopher Damoci, Mingzhao Zhu, Kenneth Hull, K. Olive, Tito Fojo, Daniel Romo, Susan E. Bates
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Abstract Background This study comprehensively investigates the association between the expression of nicotinamide N-methyltransferase (NNMT) and clinical outcomes of urothelial bladder cancer (UBC), as well as the molecular mechanisms by which NNMT in cancer-associated fibroblast (CAF) modulates tumor progression and immunotherapy resistance in UBC. Methods Single-cell transcriptomic analyses, immunohistochemical and immunofluorescence assays were performed on bladder cancer samples to validate the relationship between NNMT expression and clinical outcomes. A series of experiments, including chromatin immunoprecipitation assay, liquid chromatography tandem mass spectrometry assay, and CRISPR‒Cas9 (Clustered Regularly Interspaced Short Palindromic Repeats and CRISPR-associated protein 9) knockout, together with in vivo models, have been established to determine the molecular functions of NNMT in CAFs in UBC. Results We demonstrated that elevated expression of the nicotinamide adenine dinucleotide (NAD+) metabolism enzyme NNMT in CAFs (NNMT+ CAFs) was significantly associated with non-response to programmed death-ligand 1 (PD-L1) blockade immunotherapy in patients with UBC and predicted the unfavorable prognosis of UBC in two independent large cohorts. Targeting NNMT using the inhibitor 5-Amino-1-methylquinolinium iodide significantly reduced tumor growth and enhanced the apoptotic effects of the anti-PD-L1 antibody in UBC mouse models. Mechanistically, NNMT+ CAFs recruit tumor-associated macrophages via epigenetic reprogramming of serum amyloid A (SAA) to drive tumor cell proliferation and confer resistance to programmed death-1/PD-L1 blockade immunotherapy. Conclusions NNMT+ CAFs were significantly associated with non-response to PD-L1 blockade immunotherapy in patients with UBC. Elevated NNMT, specifically in CAFs, upregulates SAA expression and enhances the recruitment and differentiation of macrophages in the tumor microenvironment, thereby directly or indirectly promoting tumor progression and conferring resistance to immunotherapies in bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff788af3187bc82c0f0f61490e783d098bccd55c" target='_blank'>
              NAD+ metabolism enzyme NNMT in cancer-associated fibroblasts drives tumor progression and resistance to immunotherapy by modulating macrophages in urothelial bladder cancer
              </a>
            </td>
          <td>
            Meihua Yang, Bo Wang, Weibin Hou, Honghui Zeng, Wang He, Xin-Ke Zhang, D. Yan, Hao Yu, Long Huang, Lu Pei, Kaiwen Li, Haide Qin, Tianxin Lin, Jian Huang
          </td>
          <td>2024-07-01</td>
          <td>Journal for Immunotherapy of Cancer</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Persister cells (PS) selected for anticancer therapy have been recognized as a significant contributor to the development of treatment-resistant malignancies. It is found that imposing glutamine restriction induces the generation of PS, which paradoxically bestows heightened resistance to glutamine restriction treatment by activating the integrated stress response and initiating the general control nonderepressible 2-activating transcription factor 4-alanine, serine, cysteine-preferring transporter 2 (GCN2-ATF4-ASCT2) axis. Central to this phenomenon is the stress-induced ATF4 translational reprogramming. Unfortunately, directly targeting ATF4 protein has proven to be a formidable challenge because of its flat surface. Nonetheless, a G-quadruplex structure located within the promoter region of ATF4 (ATF4-G4) is uncovered and resolved, which functions as a transcriptional regulator and can be targeted by small molecules. The investigation identifies the natural compound coptisine (COP) as a potent binder that interacts with and stabilizes ATF4-G4. For the first time, the high-resolution structure of the COP-ATF4-G4 complex is determined. The formation of this stable complex disrupts the interaction between transcription factor AP-2 alpha (TFAP2A) and ATF4-G4, resulting in a substantial reduction in intracellular ATF4 levels and the eventual death of cancer cells. These seminal findings underscore the potential of targeting the ATF4-G4 structure to yield significant therapeutic advantages within the realm of persister cancer cells induced by glutamine-restricted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e9a3aeeaaba15f0673e425713df633d5da2043" target='_blank'>
              Overcoming Cancer Persister Cells by Stabilizing the ATF4 Promoter G-quadruplex.
              </a>
            </td>
          <td>
            Chengmei Xiao, Yipu Li, Yushuang Liu, Ruifang Dong, Xiaoyu He, Qing Lin, Xin Zang, Kaibo Wang, Yuanzheng Xia, Lingyi Kong
          </td>
          <td>2024-07-12</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fd17af76ab151f83c8653ba7afa98d9d3324c71" target='_blank'>
              Simultaneous multi-site editing of individual genomes using retron arrays.
              </a>
            </td>
          <td>
            A. González-Delgado, Santiago C. Lopez, Matías Rojas-Montero, C. Fishman, Seth L. Shipman
          </td>
          <td>2024-07-09</td>
          <td>Nature chemical biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, Bert van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3e129666ae56c033aa207d599e42711f40eae8" target='_blank'>
              Gene body DNA hydroxymethylation restricts the magnitude of transcriptional changes during aging
              </a>
            </td>
          <td>
            James R. Occean, Na Yang, Yan Sun, Marshall S. Dawkins, R. Munk, Cedric Belair, Showkat A. Dar, C. Anerillas, Lin Wang, Changyou Shi, Christopher Dunn, Michel Bernier, Nathan L. Price, Julie S Kim, Chang-Yi Cui, Jinshui Fan, Moitrayee Bhattacharyya, Supriyo De, Manolis Maragkakis, R. de Cabo, Simone Sidoli, P. Sen
          </td>
          <td>2024-07-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba856bced2e48ff7a94743b3236b182aa83cc934" target='_blank'>
              Epigenome editing technologies for discovery and medicine.
              </a>
            </td>
          <td>
            Sean R. McCutcheon, Dahlia Rohm, Nahid Iglesias, C. Gersbach
          </td>
          <td>2024-07-29</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Knowledge of locations and activities of cis-regulatory elements (CREs) is needed to decipher basic mechanisms of gene regulation and to understand the impact of genetic variants on complex traits. Previous studies identified candidate CREs (cCREs) using epigenetic features in one species, making comparisons difficult between species. In contrast, we conducted an interspecies study defining epigenetic states and identifying cCREs in blood cell types to generate regulatory maps that are comparable between species, using integrative modeling of eight epigenetic features jointly in human and mouse in our Validated Systematic Integration (VISION) Project. The resulting catalogs of cCREs are useful resources for further studies of gene regulation in blood cells, indicated by high overlap with known functional elements and strong enrichment for human genetic variants associated with blood cell phenotypes. The contribution of each epigenetic state in cCREs to gene regulation, inferred from a multivariate regression, was used to estimate epigenetic state Regulatory Potential (esRP) scores for each cCRE in each cell type, which were used to categorize dynamic changes in cCREs. Groups of cCREs displaying similar patterns of regulatory activity in human and mouse cell types, obtained by joint clustering on esRP scores, harbored distinctive transcription factor binding motifs that were similar between species. An interspecies comparison of cCREs revealed both conserved and species-specific patterns of epigenetic evolution. Finally, we showed that comparisons of the epigenetic landscape between species can reveal elements with similar roles in regulation, even in the absence of genomic sequence alignment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e8211dd3cbaad431cfef28f6397a18c06fd052" target='_blank'>
              Interspecies regulatory landscapes and elements revealed by novel joint systematic integration of human and mouse blood cell epigenomes.
              </a>
            </td>
          <td>
            Guanjue Xiang, Xi He, B. Giardine, Kathryn J. Isaac, Dylan J. Taylor, R. McCoy, Camden Jansen, C. Keller, A. Wixom, April Cockburn, Amber Miller, Qian Qi, Yanghua He, Yichao Li, Jens Lichtenberg, Elisabeth F. Heuston, Stacie M. Anderson, Jing Luan, M. Vermunt, Feng Yue, M. Sauria, Michael C. Schatz, James Taylor, B. Göttgens, Jim R. Hughes, D. Higgs, Mitchell J. Weiss, Yong Cheng, G. Blobel, D. Bodine, Yu Zhang, Qunhua Li, Shaun Mahony, R. Hardison
          </td>
          <td>2024-07-01</td>
          <td>Genome research</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="A significant number of the genetic alterations observed in cancer patients lie within nonprotein-coding segments of the genome, including regions coding for long noncoding RNAs (lncRNAs). LncRNAs display aberrant expression in breast cancer (BrCa), but the functional implications of this altered expression remain to be elucidated. By performing transcriptome screen in a triple negative BrCa (TNBC) isogenic 2D and 3D spheroid model, we observed aberrant expression of >1000 lncRNAs during BrCa progression. The chromatin-associated lncRNA MANCR shows elevated expression in metastatic TNBC. MANCR is upregulated in response to cellular stress and modulates DNA repair and cell proliferation. MANCR promotes metastasis as MANCR-depleted cells show reduced cell migration, invasion, and wound healing in vitro, and reduced metastatic lung colonization in xenograft experiments in vivo. Transcriptome analyses reveal that MANCR modulates expression and pre-mRNA splicing of genes, controlling DNA repair and checkpoint response. MANCR promotes the transcription of NET1A, a Rho-GEF that regulates DNA damage checkpoint and metastatic processes in cis, by differential promoter usage. Experiments suggest that MANCR regulates the expression of cancer-associated genes by modulating the association of various transcription factors and RNA-binding proteins. Our results identified the metastasis-promoting activities of MANCR in TNBC by cis-regulation of gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917a434f9bc31bbdd4efde82c1a4158058876264" target='_blank'>
              MANCR lncRNA Modulates Cell-Cycle Progression and Metastasis by Cis-Regulation of Nuclear Rho-GEF.
              </a>
            </td>
          <td>
            D. K. Singh, Zhengmin Cong, Y. Song, Minxue Liu, Ritu Chaudhary, Dazhen Liu, Yu Wang, Rishabh Prasanth, Rajendra K C, Simon Lizarazo, Miriam Akhnoukh, Omid Gholamalamdari, Anurupa Moitra, Lisa M. Jenkins, Rohit Bhargava, Erik R Nelson, Kevin Van Bortle, S. Prasanth, K. Prasanth
          </td>
          <td>2024-08-12</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Upper aerodigestive squamous cell carcinoma (UASCC) is a common and aggressive malignancy with few effective therapeutic options. Here, we investigate amino acid metabolism in this cancer, surprisingly noting that UASCC exhibits the highest methionine level across all human cancers, driven by its transporter LAT1. We show that LAT1 is also expressed at the highest level in UASCC, transcriptionally activated by UASCC-specific promoter and enhancers, which are directly coregulated by SCC master regulators TP63/KLF5/SREBF1. Unexpectedly, unbiased bioinformatic screen identifies EZH2 as the most significant target downstream of the LAT1-methionine pathway, directly linking methionine metabolism to epigenomic reprogramming. Importantly, this cascade is indispensable for the survival and proliferation of UASCC patient-derived tumor organoids. In addition, LAT1 expression is closely associated with cellular sensitivity to inhibition of the LAT1-methionine-EZH2 axis. Notably, this unique LAT1-methionine-EZH2 cascade can be targeted effectively by either pharmacological approaches or dietary intervention in vivo. In summary, this work maps a unique mechanistic cross talk between epigenomic reprogramming with methionine metabolism, establishes its biological significance in the biology of UASCC, and identifies a unique tumor-specific vulnerability which can be exploited both pharmacologically and dietarily.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/946f71e29ffbbe5a8edd3fbc4dfdae6ae2d0dcbe" target='_blank'>
              A druggable cascade links methionine metabolism to epigenomic reprogramming in squamous cell carcinoma.
              </a>
            </td>
          <td>
            Chehyun Nam, Li-Yan Li, Qian Yang, Benjamin Ziman, Hua Zhao, Boyan Hu, Casey Collet, Pei Jing, Qifang Lei, Liyan Xu, E. Li, H. Koeffler, Uttam K Sinha, De-Chen Lin
          </td>
          <td>2024-06-20</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="Clustered regularly interspersed short palindromic repeats (CRISPR)/CRISPR-associated protein 9 (Cas9) technology has revolutionized the field of gene therapy as it has enabled precise genome editing with unprecedented accuracy and efficiency, paving the way for clinical applications to treat otherwise incurable genetic disorders. Typically, precise genome editing requires the delivery of multiple components to the target cells that, depending on the editing platform used, may include messenger RNA (mRNA), protein complexes, and DNA fragments. For clinical purposes, these have to be efficiently delivered into transplantable cells, such as primary T lymphocytes or hematopoietic stem and progenitor cells that are typically sensitive to exogenous substances. This challenge has limited the broad applicability of precise gene therapy applications to those strategies for which efficient delivery methods are available. Electroporation-based methodologies have been generally applied for gene editing applications, but procedure-associated toxicity has represented a major burden. With the advent of novel and less disruptive methodologies to deliver genetic cargo to transplantable cells, it is now possible to safely and efficiently deliver multiple components for precise genome editing, thus expanding the applicability of these strategies. In this review, we describe the different delivery systems available for genome editing components, including viral and non-viral systems, highlighting their advantages, limitations, and recent clinical applications. Recent improvements to these delivery methods to achieve cell specificity represent a critical development that may enable in vivo targeting in the future and will certainly play a pivotal role in the gene therapy field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff8c03481dd2684d89eea9dc5f829d3697ccc125" target='_blank'>
              Viral and Non-Viral Systems to Deliver Gene Therapeutics to Clinical Targets
              </a>
            </td>
          <td>
            Maryam Taghdiri, C. Mussolino
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cycling cells replicate their DNA during the S phase through a defined temporal program known as replication timing. Mutation frequencies, epigenetic chromatin states, and transcriptional activities are different for genomic regions that are replicated early and late in the S phase. Here, we found from ChIP-Seq analysis that DNA polymerase (Pol) κ is enriched in early-replicating genomic regions in HEK293T cells. In addition, by feeding cells with N 2-heptynyl-2′-deoxyguanosine followed by click chemistry–based enrichment and high-throughput sequencing, we observed elevated Pol κ activities in genomic regions that are replicated early in the S phase. On the basis of the established functions of Pol κ in accurate and efficient nucleotide insertion opposite endogenously induced N 2-modified dG lesions, our work suggests that active engagement of Pol κ may contribute to diminished mutation rates observed in early-replicating regions of the human genome, including cancer genomes. Together, our work expands the functions of Pol κ and offered a plausible mechanism underlying replication timing–dependent mutation accrual in the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417d907a79911389824959d3e561a3c8846017ac" target='_blank'>
              DNA polymerase κ participates in early S-phase DNA replication in human cells
              </a>
            </td>
          <td>
            Feng Tang, Yinan Wang, Ting Zhao, Jun Yuan, Andrew H. Kellum, Yinsheng Wang
          </td>
          <td>2024-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="High-throughput DNA transformation techniques are invaluable when creating high-diversity mutant libraries, and the success rate of any protein engineering endeavors is directly dependent on both the size and diversity of the mutant library that is to be screened. It is also widely accepted that in both bacteria and yeast there is an inverse correlation between the DNA transformation efficiency and the likelihood of transforming multiple DNA molecules into each cell. However, most successful high-throughput mutant screening efforts require high quality libraries, i.e., libraries comprised of cells with a clear phenotype-to-genotype relationship (one genotype/cell). Thus, DNA transformation methods with a high multiplicity of transformation are highly undesirable and detrimental to most mutant screening assays. Here we describe a simple, robust, and highly efficient yeast plasmid DNA transformation methodology, using a dual heat-shock and electroporation approach (HEEL) that generates more than 2 x 107 plasmid-transformed yeast cells per reaction, while simultaneously increasing the fraction of mono-transformed cells from 20% to more than 70% of the transformed population. By also using an automated yeast genotyping workflow coupled with a dual-barcoding approach, consisting of a SNP and high-diversity barcode (10N), we can consistently identify and enumerate unique plasmid genotypes within a heterogeneous population merely through Sanger sequencing. We demonstrate here that the size and quality of a transformed library no longer need to be inversely correlated when transforming large mutant DNA libraries in yeast using highly efficient DNA electroporation methods. Significance With the recent expansion of artificial intelligence in the field of synthetic biology there has never been a greater need for high-quality data and reliable measurements of phenotype-to-genotype relationships. However, one major obstacle to creating accurate computer-based models is the current abundance of low-quality phenotypic measurements originating from numerous high-throughput, but low-resolution assays. Rather than increasing the quantity of measurements, new studies should aim to generate as accurate measurements as possible. The HEEL methodology presented here aims to address this issue by minimizing the problem of multi-plasmid uptake during high-throughput yeast DNA transformations, which leads to the creation of heterogeneous cellular genotypes. HEEL should enable highly accurate phenotype-to-genotype measurements going forward, which could be used to construct better computer-based models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d31585740ca58ec4265cb646f1e7e7eda782580" target='_blank'>
              Heat pre-treatment reduces multiplicity of plasmid transformations in yeast during electroporation, without diminishing the transformation efficiency
              </a>
            </td>
          <td>
            Marcus Wäneskog, E. E. Hoch-Schneider, Shilpa Garg, Christian Kronborg Cantalapiedra, Elena Schaefer, Michael Krogh Jensen, Emil Damgaard Jensen
          </td>
          <td>2024-07-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Premature aging is a hallmark of Down syndrome, caused by trisomy of human chromosome 21, but the reason is unclear and difficult to study in humans. We used an aneuploid model in wild yeast to show that chromosome amplification disrupts nutrient-induced cell-cycle arrest, quiescence entry, and healthy aging, across genetic backgrounds and amplified chromosomes. We discovered that these defects are due in part to aneuploidy-induced dysfunction in Ribosome Quality Control (RQC). Compared to euploids, aneuploids entering quiescence display aberrant ribosome profiles, accumulate RQC intermediates, and harbor an increased load of protein aggregates. Although they have normal proteasome capacity, aneuploids show signs of ubiquitin dysregulation, which impacts cyclin abundance to disrupt arrest. Remarkably, inducing ribosome stalling in euploids produces similar aberrations, while up-regulating limiting RQC subunits or proteins in ubiquitin metabolism alleviates many of the aneuploid defects. Our results provide implications for other aneuploidy disorders including Down syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c6d2e4dd1921e05fad92cf19e5a9d35d322983a" target='_blank'>
              Premature aging in aneuploid yeast is caused in part by aneuploidy-induced defects in Ribosome Quality Control
              </a>
            </td>
          <td>
            Leah E. Escalante, James Hose, Hollis Howe, Norah Paulsen, Michael Place, A. Gasch
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="

A pan-genome is a compilation of the common and unique genomes found in a given
species. It incorporates the genetic information from all of the genomes sampled, producing a big
and diverse set of genetic material. Pan-genomic analysis has various advantages over typical genomics
research. It creates a vast and varied spectrum of genetic material by combining the genetic
data from all the sampled genomes. Comparing pan-genomics analysis to conventional genomic
research, there are a number of benefits. Although the most recent era of pan-genomic studies has
used cutting-edge sequencing technology to shed fresh light on biological variety and improvement,
the potential uses of pan-genomics in improvement have not yet been fully realized. Pangenome
research in various organisms has demonstrated that missing genetic components and the
detection of significant Structural Variants (SVs) can be investigated using pan-genomic methods.
Many individual-specific sequences have been linked to biological adaptability, phenotypic, and
key economic attributes. This study aims to focus on how pangenome analysis uncovers genetic
differences in various organisms, including human, and their effects on phenotypes, as well as
how this might help us comprehend the diversity of species. The review also concentrated on potential
problems and the prospects for future pangenome research.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00278941260cac72d0ac34306dd0ec12767d6c83" target='_blank'>
              Pan-Genomics: Insight into the Functional Genome, Applications, Advancements, and Challenges
              </a>
            </td>
          <td>
            Akansha Sarwad, Spoorti Hosgoudar, Prachi P. Parvatikar
          </td>
          <td>2024-07-03</td>
          <td>Current Genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="For many years, scientific research has dedicated significant attention to unraveling the complexities of inheritance, the process by which traits are passed from one generation to the next. At the heart of this exploration lies DNA methylation, a pivotal mechanism that exerts profound influence over gene expression, cellular identity, and the overarching development of organisms. This in-depth review meticulously probes the intricate dynamics of DNA methylation, meticulously dissecting its multifaceted impact on inheritance patterns and the spectrum of phenotypic variations observed. The review scrutinizes the underlying mechanisms governing the establishment and maintenance of DNA methylation, offering a nuanced understanding of its regulatory roles. Through a synthesis of insights gleaned from the fields of molecular biology, epigenetics, and genomics, it illuminates the far-reaching implications of DNA methylation for both health and disease. By peering into the molecular machinery that orchestrates these epigenetic modifications, the review provides a comprehensive framework for comprehending the interplay between DNA methylation and the transmission of hereditary traits. Moreover, by contextualizing these findings within the broader landscape of genetics and inheritance, the review offers valuable perspectives for guiding future research endeavors. These insights not only deepen our understanding of the intricate processes underlying heredity but also hold potential implications for the development of novel therapeutic interventions aimed at modulating DNA methylation patterns. In essence, this review serves as a cornerstone in the ongoing quest to unravel the mysteries of inheritance, paving the way for innovative approaches to address complex biological phenomena.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f158b12af137fd58de0918eb48149c92baa1bb" target='_blank'>
              Dynamics of DNA Methylation During Gametogenesis: Insights and Detection Methods
              </a>
            </td>
          <td>
            Gangadhara Doggalli, D. S. Shailaja, Keerthana S, Manojkumar H G, Santhoshinii E, Mitali Srivastava, Ritama Kundu, Arfa Ameen, Varun Kumar K
          </td>
          <td>2024-06-25</td>
          <td>Journal of Advances in Biology &amp; Biotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract In ovarian and breast cancer, promoter methylation of BRCA1 or RAD51C is a promising biomarker for PARP inhibitor response, as high levels lead to homologous recombination deficiency (HRD). Yet the extent and role of such methylation in other cancers is not clear. This study comprehensively investigated promoter methylation of eight homologous recombination repair genes across 23 solid cancer types. Here, we showed that BRCA1 methylated cancers were associated with reduced gene expression, loss of heterozygosity (LOH), TP53 mutations and genomic features of HRD. We identified BRCA1 methylation in 3% of the copy-number high subtype of endometrial cancer, and as a rare event in six other cancer types, including lung squamous cell, pancreatic, bladder and stomach cancer. RAD51C promoter methylation was widespread across multiple cancer types, but HRD features were only observed for cases which contained high-level tumour methylation and LOH of RAD51C. While RAD51C methylation was frequent in stomach adenocarcinoma (6%) and low-grade glioma (2.5%), it was mostly detected at a low tumour level, suggestive of heterozygous methylation, and was associated with CpG island methylator phenotype. Our findings indicate that high-level tumour methylation of BRCA1 and RAD51C should be explored as a PARP inhibitor biomarker across multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/374a7a0b66d69ba1f5dc2c4d8219381d1c2a8cfc" target='_blank'>
              High-level tumour methylation of BRCA1 and RAD51C is required for homologous recombination deficiency in solid cancers
              </a>
            </td>
          <td>
            Lijun Xu, Brett Liddell, K. Nešić, Franziska Geissler, Lauren M. Ashwood, Matthew J. Wakefield, Clare L Scott, Nicola Waddell, O. Kondrashova
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Type 1 diabetes (T1D) serves as an exemplar of chronic autoimmune disease characterized by insulin deficiency due to pancreatic β-cell destruction, leading to hyperglycemia and progressive organ failure. Until recently, therapeutic efforts to mitigate the root cause of disease have been limited by the challenges in studying mechanisms involved in immune tolerance in humans. The current clinical advances, and existing challenges, highlight a need to incorporate new insights into mechanisms into correlative studies that assess immune tolerance in the setting of delayed β-cell destruction. Among several factors known to promote T1D, autoreactive T cells play a critical role in initiating and sustaining disease through their direct recognition and destruction of β cells. Emerging research defining the genetic and epigenetic etiology of long-lived β-cell-specific T cells is providing new insight into mechanisms that promote lifelong disease and future opportunities for targeted therapeutic intervention. This article will provide an overview of recent progress toward understanding the development of autoreactive T cells and epigenetic mechanisms stabilizing their developmental state during T1D pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c968e457a95efbe4a140cd4340ccdd42be2982d" target='_blank'>
              Genetics and Epigenetics of Type 1 Diabetes Self-Reactive T Cells.
              </a>
            </td>
          <td>
            Tae Gun Kang, Benjamin Youngblood
          </td>
          <td>2024-08-12</td>
          <td>Cold Spring Harbor perspectives in medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbea514dd68ac66e3d2deacc382cefe854e64feb" target='_blank'>
              Mechanism insights and therapeutic intervention of tumor metastasis: latest developments and perspectives
              </a>
            </td>
          <td>
            Xiaoli Shi, Xinyi Wang, Wentao Yao, Dongmin Shi, Xihuan Shao, Zhengqing Lu, Yue Chai, Jinhua Song, Weiwei Tang, Xuehao Wang
          </td>
          <td>2024-08-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Interactions between multiple genes or cis-regulatory elements (CREs) underlie a wide range of biological processes in both health and disease. High-throughput screens using dCas9 fused to epigenome editing domains have allowed researchers to assess the impact of activation or repression of both coding and non-coding genomic regions on a phenotype of interest, but assessment of genetic interactions between those elements has been limited to pairs. Here, we combine a hyper-efficient version of Lachnospiraceae bacterium dCas12a (dHyperLbCas12a) with RNA Polymerase II expression of long CRISPR RNA (crRNA) arrays to enable efficient highly-multiplexed epigenome editing. We demonstrate that this system is compatible with several activation and repression domains, including the P300 histone acetyltransferase domain and SIN3A interacting domain (SID). We also show that the dCas12a platform can perform simultaneous activation and repression using a single crRNA array via co-expression of multiple dCas12a orthologues. Lastly, demonstrate that the dCas12a system is highly effective for high-throughput screens. We use dHyperLbCas12a-KRAB and a ∼19,000-member barcoded library of crRNA arrays containing six crRNAs each to dissect the independent and combinatorial contributions of CREs to the dose-dependent control of gene expression at a glucocorticoid-responsive locus. The tools and methods introduced here create new possibilities for highly multiplexed control of gene expression in a wide variety of biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c036dd7ee414b69b8ef2f177d76c80471f4540e3" target='_blank'>
              HyperCas12a enables highly-multiplexed epigenome editing screens
              </a>
            </td>
          <td>
            Schuyler M. Melore, Marisa C. Hamilton, Timothy E. Reddy
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background: B-cell acute lymphoblastic leukemia (B-ALL) is a complex disorder that includes multiple genetic changes, one of the main causes of which is rare and common chromosomal translocations that lead to abnormal gene fusions. This abnormal fusion produces a new protein that causes the leukemia cells. These types of rearrangements usually occur in lymphoma and result in the movement of genetic material between different chromosomes or within chromosomes. This systematic review aims to evaluate published studies and investigate the role and importance of common and rare chromosomal translocations in the occurrence of B-ALL. 
Material and Methods: This systematic review investigated and evaluated the evidence regarding the effect of chromosomal translocations in adults and pediatrics with B-ALL. This review was based on the preferred reporting items for systematic reviews and meta-analysis checklists. A literature search was conducted using international databases (such as PubMed, Web of Science, Scopus, Research Gate, Google Scholar, and Cochrane Systematic Reviews database). Only English-language articles published between January 2010 and January 2023 including MESH terms such as ALL, B-ALL, and chromosomal translocations were selected. Seventy-five related studies had the necessary criteria to be examined in the present study. 
Results: A total of 237 articles were retrieved in the online search. The excluded articles included research on other types of leukemia in adults and children, descriptive studies, case studies, studies related to animal models of leukemogenesis, and comparisons of common treatment methods of leukemia grouping cancers. Finally, seventy-five studies were identified as eligible for inclusion in this systematic review. 
Conclusion: This review provides a comprehensive assessment of common and rare chromosomal translocations that can lead to the development of cancer cells. Chromosomal translocations play a role as diagnostic markers and important prognostic indicators and help specialists adjust treatment approaches according to their occurrence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a4f4fe423595ea176d9dfc545456ffbffddf2b8" target='_blank'>
              Analysis and Identification of Rare and Prevalent Breakpoints in Chromosomal rearrangements in Adult and Pediatric with B-Acute Lymphoblastic Leukemia (B-ALL): A Systematic Review
              </a>
            </td>
          <td>
            Roghayeh Shahshahani, Mehri Khatami, M. Heidari, Parisa Naji
          </td>
          <td>2024-06-30</td>
          <td>Iranian Journal of Pediatric Hematology &amp; Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Retrons are promising gene editing tools because they can produce multi-copy single-stranded DNA in cells via self-primed reverse transcription. However, their potential for inserting genetic cargos in eukaryotes remains largely unexplored. Here we report the discovery and engineering of highly efficient retron-based gene editors for mammalian cells and vertebrates. Through bioinformatic analysis of metagenomic data and functional screening, we identified novel retron reverse transcriptases (RTs) that are highly active in mammalian cells. Rational design further improved the editing efficiency to levels comparable with conventional single-stranded oligodeoxynucleotide donors but from a genetically encoded cassette. Small molecule inhibitors of non-homologous end joining factors and Cas9-DNA repair protein fusions further increase homology-directed repair. Retron editors also exhibited robust activity with Cas12a nuclease and Cas9 nickase, expanding the genomic target scope and bypassing the need for a DNA double-stranded break. Using a rationally engineered retron editor, we incorporate a split GFP epitope tag for live cell imaging. Finally, we develop an all-RNA delivery strategy to enable DNA-free gene editing in cells and vertebrate embryos. This work establishes retron editors as a versatile and efficient tool for precise gene editing, offering new opportunities for biotechnology and biomedical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8f2c620f5c7bf4a518c60425b0c63267672964" target='_blank'>
              Discovery and Engineering of Retrons for Precise Genome Editing
              </a>
            </td>
          <td>
            Jesse D. Buffington, Hung-Che Kuo, Kuang Hu, You-Chiun Chang, Kamyab Javanmardi, Brittney Voigt, Yi-Ru Li, Mary E. Little, Sravan K Devanathan, Blerta Xhemalçe, R. Gray, Ilya J. Finkelstein
          </td>
          <td>2024-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>