<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-30 06:32:29 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>7</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Summary Phenotypic adaptation, the ability of cells to change phenotype in response to external pressures, has been identified as a driver of drug resistance in cancer. To quantify phenotypic adaptation in BRAFV600E-mutant melanoma, we develop a theoretical model that emerges from data analysis of WM239A-BRAFV600E growth rates in response to drug challenge with the BRAF-inhibitor encorafenib. Our model constitutes a cell population model in which each cell is individually described by one of multiple discrete and plastic phenotype states that are directly linked to drug-dependent net growth rates and, by extension, drug resistance. Data-matched simulations reveal that phenotypic adaptation in the cells is directed towards states of high net growth rates, which enables evasion of drug-effects. The model subsequently provides an explanation for when and why intermittent treatments outperform continuous treatments in vitro, and demonstrates the benefits of not only targeting, but also leveraging, phenotypic adaptation in treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db72a236ec0fc5bd479b42430df6525df1d343c7" target='_blank'>
              Growth rate-driven modelling reveals how phenotypic adaptation drives drug resistance in BRAFV600E-mutant melanoma
              </a>
            </td>
          <td>
            Sara Hamis, A. Browning, A. Jenner, Chiara Villa, P. Maini, Tyler Cassidy
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Drug resistance is the major cause of therapeutic failure in high-grade serous ovarian cancer (HGSOC). Yet, the mechanisms by which tumors evolve to drug resistant states remains largely unknown. To address this, we aimed to exploit clone-specific genomic structural variations by combining scaled single-cell whole genome sequencing with longitudinally collected cell-free DNA (cfDNA), enabling clonal tracking before, during and after treatment. We developed a cfDNA hybrid capture, deep sequencing approach based on leveraging clone-specific structural variants as endogenous barcodes, with orders of magnitude lower error rates than single nucleotide variants in ctDNA (circulating tumor DNA) detection, demonstrated on 19 patients at baseline. We then applied this to monitor and model clonal evolution over several years in ten HGSOC patients treated with systemic therapy from diagnosis through recurrence. We found drug resistance to be polyclonal in most cases, but frequently dominated by a single high-fitness and expanding clone, reducing clonal diversity in the relapsed disease state in most patients. Drug-resistant clones frequently displayed notable genomic features, including high-level amplifications of oncogenes such as CCNE1, RAB25, NOTCH3, and ERBB2. Using a population genetics Wright-Fisher model, we found evolutionary trajectories of these features were consistent with drug-induced positive selection. In select cases, these alterations impacted selection of secondary lines of therapy with positive patient outcomes. For cases with matched single-cell RNA sequencing data, pre-existing and genomically encoded phenotypic states such as upregulation of EMT and VEGF were linked to drug resistance. Together, our findings indicate that drug resistant states in HGSOC pre-exist at diagnosis and lead to dramatic clonal expansions that alter clonal composition at the time of relapse. We suggest that combining tumor single cell sequencing with cfDNA enables clonal tracking in patients and harbors potential for evolution-informed adaptive treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1441270a54104f75e7713e2bb6b7323866959" target='_blank'>
              Tracking clonal evolution of drug resistance in ovarian cancer patients by exploiting structural variants in cfDNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Gryte Satas, Hannah J. Lees, Jake June-Koo Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, N. Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitriy Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a diverse malignancy originating from myeloid progenitor cells, with significant genetic and clinical variability. Modern classification systems like those from the World Health Organization (WHO) and European LeukemiaNet use immunophenotyping, molecular genetics, and clinical features to categorize AML subtypes. This classification highlights crucial genetic markers such as FLT3, NPM1 mutations, and MLL-AF9 fusion, which are essential for prognosis and directing targeted therapies. The MLL-AF9 fusion protein is often linked with therapy-resistant AML, highlighting the risk of relapse due to standard chemotherapeutic regimes. In this sense, factors like the ZEB, SNAI, and TWIST gene families, known for their roles in epithelial–mesenchymal transition (EMT) and cancer metastasis, also regulate hematopoiesis and may serve as effective therapeutic targets in AML. These genes contribute to cell proliferation, differentiation, and extramedullary hematopoiesis, suggesting new possibilities for treatment. Advancing our understanding of the molecular mechanisms that promote AML, especially how the bone marrow microenvironment affects invasion and drug resistance, is crucial. This comprehensive insight into the molecular and environmental interactions in AML emphasizes the need for ongoing research and more effective treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b827ac4bebbcac825ffc1c60412a291161ab8775" target='_blank'>
              The Role of Epithelial-to-Mesenchymal Transition Transcription Factors (EMT-TFs) in Acute Myeloid Leukemia Progression
              </a>
            </td>
          <td>
            Diego Cuevas, Roberto Amigo, Adolfo Agurto, Adan Andreu Heredia, Catherine Guzmán, Antonia Recabal-Beyer, V. González-Pecchi, T. Caprile, Jody J. Haigh, Carlos Farkas
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Phenotypic plasticity is a recognized mechanism driving therapeutic resistance in patients with prostate cancer. Although underlying molecular causations driving phenotypic plasticity have been identified, therapeutic success is yet to be achieved. To identify putative master regulator transcription factors (MR-TF) driving phenotypic plasticity in prostate cancer, this work utilized a multiomic approach using genetically engineered mouse models of prostate cancer combined with patient data to identify MYB proto-oncogene like 2 (MYBL2) as a significantly enriched transcription factor in prostate cancer exhibiting phenotypic plasticity. Genetic inhibition of Mybl2 using independent murine prostate cancer cell lines representing phenotypic plasticity demonstrated Mybl2 loss significantly decreased in vivo growth as well as cell fitness and repressed gene expression signatures involved in pluripotency and stemness. Because MYBL2 is currently not druggable, a MYBL2 gene signature was employed to identify cyclin-dependent kinase-2 (CDK2) as a potential therapeutic target. CDK2 inhibition phenocopied genetic loss of Mybl2 and significantly decreased in vivo tumor growth associated with enrichment of DNA damage. Together, this work demonstrates MYBL2 as an important MR-TF driving phenotypic plasticity in prostate cancer. Furthermore, high MYBL2 activity identifies prostate cancer that would be responsive to CDK2 inhibition. Significance: Prostate cancers that escape therapy targeting the androgen receptor signaling pathways via phenotypic plasticity are currently untreatable. Our study identifies MYBL2 as a MR-TF in phenotypic plastic prostate cancer and implicates CDK2 inhibition as a novel therapeutic target for this most lethal subtype of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d7169da5236ce487e4af9bc41a5465e1a1c863" target='_blank'>
              MYBL2 Drives Prostate Cancer Plasticity: Inhibiting Its Transcriptional Target CDK2 for RB1-Deficient Neuroendocrine Prostate Cancer
              </a>
            </td>
          <td>
            Beatriz German, S. A. Alaiwi, Kun-Lin Ho, Jagpreet S Nanda, Marcos A S Fonseca, Deborah L. Burkhart, Anjali V. Sheahan, Hannah E Bergom, Katherine L. Morel, H. Beltran, Justin H. Hwang, M. Freedman, K. Lawrenson, Leigh Ellis
          </td>
          <td>2024-08-08</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 We aim to offer new insights on the regulation of metabolic plasticity in renal cell carcinoma (RCC) and identify novel metabolic regulatory targets that can be used for potential treatment therapies of the disease. RCC affects an estimated 400,000 people worldwide each year with over 100,00 deaths annually that disproportionately affects racial/ethnic minorities in the United States. RCC is a term that covers an array of kidney cancer subtypes differing in incidence, histopathology, genetic and molecular alterations, as well as in clinical outcomes and prognoses. However, a common characteristic of many RCCs is that they are driven by metabolic rewiring, due to a high frequency of mutations in genes that regulate major metabolic processes in the cell. It was recently discovered that patient overall survival and prognosis has been linked to the expression of certain metabolic signatures in RCC tumor cells. Metabolic plasticity allows cancer cells to survive and adapt to different environments and metabolic stress conditions, such as nutrient deprivation. Therefore, exploring and exploiting metabolic rewiring in these cells may reveal genetic vulnerabilities and dependencies. This could lead to discovery of novel biomarkers and molecular targets that can be implicated as future therapeutic interventions for this disease. To this end, we applied functional genomic screening to expose genetic dependencies and vulnerabilities under different physiologic and metabolic conditions in patient derived medullary (PDM) RCC cells. The medullary RCC is a rare and aggressive RCC subtype that disproportionately affects the African American/Black population. We applied a CRISPR- based combinatorial screening platform that is based on co-expression of Cas9 and Cas12a nucleases and libraries of hybrid guide RNAs (hgRNA) to achieve ultra-efficient gene knockout (Aregger et al., 2021; Gonatopoulos-Pournatzis et al., 2020) in PDM RCC cells. Additionally, cells were screened under different media conditions that mimic human physiological nutrient conditions and a media deprived of lipids, one of the major metabolic substrates of the cell. By utilizing a genome-wide pooled CRISPR gene knockout screen we were able to systematically uncover genes required for cell survival and proliferation in a panel of patient-derived RCCs. We have begun to uncover genetic dependencies and major metabolic pathways that can be elucidated and exploited for future studies, such as those involving ferroptosis which is a targetable pathway for the development of cancer treatment therapies. Our findings demonstrate some of the key genetic drivers underlying cellular adaptations to different physiological environments and nutrient deprivation. In future studies we will further explore the relationship between different key metabolic regulatory genes in RCC and map for genetic interactions in major metabolic pathways.
 Citation Format: Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger. Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C091.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cd9b94600307109e3720c07a3b1a13c91abd8ae" target='_blank'>
              Abstract C091: Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma
              </a>
            </td>
          <td>
            Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The rapid advancement of high-throughput technologies, particularly next-generation sequencing (NGS), has revolutionized cancer research by enabling the investigation of genetic variations such as SNPs, copy number variations, gene expression, and protein levels. These technologies have elevated the significance of precision oncology, creating a demand for biomarker identification and validation. This review explores the complex interplay of oncology, cancer biology, and bioinformatics tools, highlighting the challenges in statistical learning, experimental validation, data processing, and quality control that underpin this transformative field. This review outlines the methodologies and applications of bioinformatics tools in cancer genomics research, encompassing tools for data structuring, pathway analysis, network analysis, tools for analyzing biomarker signatures, somatic variant interpretation, genomic data analysis, and visualization tools. Open-source tools and repositories like The Cancer Genome Atlas (TCGA), Genomic Data Commons (GDC), cBioPortal, UCSC Genome Browser, Array Express, and Gene Expression Omnibus (GEO) have emerged to streamline cancer omics data analysis. Bioinformatics has significantly impacted cancer research, uncovering novel biomarkers, driver mutations, oncogenic pathways, and therapeutic targets. Integrating multi-omics data, network analysis, and advanced ML will be pivotal in future biomarker discovery and patient prognosis prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d1868b128ca158d078b4dfd206561eae801a47a" target='_blank'>
              A Comprehensive Review of Bioinformatics Tools for Genomic Biomarker Discovery Driving Precision Oncology
              </a>
            </td>
          <td>
            Alexis J. Clark, James W. Lillard
          </td>
          <td>2024-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this review, we revisit the pivotal role of fibroblast growth factor receptor 3 (FGFR3) in bladder cancer (BLCA), underscoring its prevalence in both non-muscle-invasive and muscle-invasive forms of the disease. FGFR3 mutations in up to half of BLCAs play a well-established role in tumorigenesis, shaping distinct tumor initiation patterns and impacting the tumor microenvironment (TME). Emphasizing the importance of considering epithelial-mesenchymal transition profile and TME status, we revisit their relevance in predicting responses to immune checkpoint inhibitors in FGFR3-mutated BLCAs. This writing highlights the initially promising yet transient efficacy of the FGFR inhibitor Erdafitinib on FGFR3-mutated BLCA, stressing the pressing need to unravel resistance mechanisms and identify co-targets for future combinatorial studies. A thorough analysis of recent preclinical and clinical evidence reveals resistance mechanisms, including secondary mutations, epigenetic alterations in pathway effectors, phenotypic heterogeneity, and population-specific variations within FGFR3 mutational status. Lastly, we discuss the potential of combinatorial treatments and concepts like synthetic lethality for discovering more effective targeted therapies against FGFR3-mutated BLCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d32b54c415deb1782db39b2832fb5448db0b31" target='_blank'>
              FGFR3 alterations in bladder cancer: Sensitivity and resistance to targeted therapies.
              </a>
            </td>
          <td>
            Maxim Noeraparast, Katarina Krajina, Renate Pichler, Dora Niedersüß-Beke, S. Shariat, Viktor Grünwald, S. Ahyai, Martin Pichler
          </td>
          <td>2024-08-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M As Sobeai
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematologic neoplasia with a complex polyclonal architecture. Among driver lesions, those involving the FLT3 gene represent the most frequent mutations identified at diagnosis. The development of tyrosine kinase inhibitors (TKIs) has improved the clinical outcomes of FLT3-mutated patients (Pt). However, overcoming resistance to these drugs remains a challenge. To unravel the molecular mechanisms underlying therapy resistance and clonal selection, we conducted a longitudinal analysis using a single-cell DNA sequencing approach (MissionBioTapestri® platform, San Francisco, CA, USA) in two patients with FLT3-mutated AML. To this end, samples were collected at the time of diagnosis, during TKI therapy, and at relapse or complete remission. For Pt #1, disease resistance was associated with clonal expansion of minor clones, and 2nd line TKI therapy with gilteritinib provided a proliferative advantage to the clones carrying NRAS and KIT mutations, thereby responsible for relapse. In Pt #2, clonal architecture was less complex, and 1st line TKI therapy with midostaurin was able to eradicate the leukemic clones. Our results corroborate previous findings about clonal selection driven by TKIs, highlighting the importance of a deeper characterization of individual clonal architectures for choosing the best treatment plan for personalized approaches aimed at optimizing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4202a0b52113f492de1c5d891aefedd2f36b44c" target='_blank'>
              Tracking Response and Resistance in Acute Myeloid Leukemia through Single-Cell DNA Sequencing Helps Uncover New Therapeutic Targets
              </a>
            </td>
          <td>
            S. Bruno, E. Borsi, A. Patuelli, L. Bandini, M. Mancini, D. Forte, J. Nanni, Martina Barone, Alessandra Grassi, G. Cristiano, C. Venturi, V. Robustelli, Giulia Atzeni, Cristina Mosca, S. De Santis, C. Monaldi, A. Poletti, C. Terragna, A. Curti, M. Cavo, Simona Soverini, E. Ottaviani
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Despite development of several effective therapies for multiple myeloma (MM), the prognosis of patients with partial deletion of chromosome 17 (del(17p)) and TP53 aberrations remains poor. By applying comprehensive multi-omics profiling analyses (whole exome and transcriptome sequencing plus proteomics) and functional ex vivo drug screening to samples from 167 patients with MM, we uncovered novel therapeutic vulnerabilities specific to TP53 mutated MM. Our findings revealed a distinct sensitivity profile to a range of inhibitors (mitotic, topoisomerase, HDAC, HSP90, IGF1R and PI3K/AKT/mTOR inhibitors) irrespective of 17p deletion status. Conversely, no increase in sensitivity was observed for monoallelic TP53 (del(17p) with WT TP53) when compared to other samples, highlighting the remaining unmet clinical need. Notably, plicamycin, an RNA synthesis inhibitor linked to modulation of chromatin structure and increased transcription, emerged as particularly efficacious for TP53 mutated MM. The increased sensitivity correlated with higher protein expression of the drug targets: HDAC2, HSP90AA1 and multiple ribosomal subunits. Additionally, we observed increased RNA expression of G2M checkpoint, E2F targets and mTORC1 signaling in our cohort and the MMRF-CoMMpass (NCT01454297) study in TP53 mutated MM. Harmonization of multi-omics data with ex vivo drug screening results revealed that TP53 mutated MM is functionally distinct from MM with monoallelic TP53, and demonstrate that MM with mutated TP53, with and without del(17p), may be targetable by approved drugs, and further indicates the need for regular monitoring by sequencing to identify these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a52d0d33d267850e10246d4013cb5921df1165" target='_blank'>
              Multi-omic and functional screening reveal targetable vulnerabilities in TP53 mutated multiple myeloma
              </a>
            </td>
          <td>
            Dimitrios Tsallos, Nemo Ikonen, Juho J. Mienen, M. M. Majumder, S. Eldfors, Imre Västrik, A. Parsons, M. Suvela, Katie Dunphy, Paul Dowling, D. Bazou, Peter O’Gorman, J. Lievonen, R. Silvennoinen, Pekka Anla, C. Heckman
          </td>
          <td>2024-08-23</td>
          <td>None</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Relapsed or refractory B-cell acute lymphoblastic leukemia (B-ALL) is a major cause of pediatric cancer-related deaths. Relapse-specific mutations do not account for all chemotherapy failures in B-ALL patients, suggesting additional mechanisms of resistance. By mining RNA-seq datasets of paired diagnostic/relapse pediatric B-ALL samples, we discovered pervasive alternative splicing (AS) patterns linked to relapse and affecting drivers of resistance to glucocorticoids, anti-folates, and thiopurines. Most splicing variations represented cassette exon skipping, "poison" exon inclusion, and intron retention, phenocopying well-documented loss-of-function mutations. In contrast, relapse-associated AS of NT5C2 mRNA yielded an isoform with the functionally uncharacterized in-frame exon 6a. Incorporation of the 8-amino acid sequence SQVAVQKR into this enzyme created a putative phosphorylation site and resulted in elevated nucleosidase activity, which is a known consequence of gain-of-function mutations in NT5C2 and a common determinant of 6-mercaptopurine (6-MP) resistance. Consistent with this finding, NT5C2ex6a and the R238W hotspot variant conferred comparable levels of resistance to 6-MP in B-ALL cells both in vitro and in vivo. Furthermore, both the NT5C2ex6a and R238W variants induced collateral sensitivity to the inosine monophosphate dehydrogenase (IMPDH) inhibitor mizoribine. These results ascribe an important role for splicing perturbations in chemotherapy resistance in relapsed B-ALL and suggest that IMPDH inhibitors, including the commonly used immunosuppressive agent mycophenolate mofetil, could be a valuable therapeutic option for treating thiopurine-resistant leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4042b39078340516bf6531df7b8969e5feddb46a" target='_blank'>
              An Alternatively Spliced Gain-of-Function NT5C2 Isoform Contributes to Chemoresistance in Acute Lymphoblastic Leukemia.
              </a>
            </td>
          <td>
            M. Torres-Diz, Clara Reglero, C. Falkenstein, Annette Castro, Katharina E. Hayer, Caleb M. Radens, Mathieu Quesnel-Vallières, Zhiwei Ang, Priyanka Sehgal, Marilyn M. Li, Yoseph Barash, S. Tasian, Adolfo Ferrando, A. Thomas-Tikhonenko
          </td>
          <td>2024-08-02</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Background/Objectives: The rat osteosarcoma cell line UMR-106 is widely used for the study of bone cancer biology but it has not been well characterized with modern genomic methods. Methods: To better understand the biology of UMR-106 cells we used a combination of optical genome mapping (OGM), long-read sequencing nanopore sequencing and RNA sequencing.The UMR-106 genome was compared to a strain-matched Sprague-Dawley rat for variants associated with human osteosarcoma while expression data were contrasted with a public osteoblast dataset. Results: Using the COSMIC database to identify the most affected genes in human osteosarcomas we found somatic mutations in Tp53 and H3f3a. OGM identified a relatively small number of differences between the cell line and a strain-matched control animal but did detect a ~45 Mb block of amplification that included Myc on chromosome 7 which was confirmed by long-read sequencing. The amplified region showed several blocks of non-contiguous rearranged sequence implying complex rearrangements during their formation and included 14 genes reported as biomarkers in human osteosarcoma, many of which also showed increased transcription. A comparison of 5mC methylation from the nanopore reads of tumor and control samples identified genes with distinct differences including the OS marker Cdkn2a. Conclusions: This dataset illustrates the value of long DNA methods for the characterization of cell lines and how inter-species analysis can inform us about the genetic nature underlying mutations that underpin specific tumor types. The data should be a valuable resource for investigators studying osteosarcoma, in general, and specifically the UMR-106 model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0957ae8779454bb6e3009d9a818af23c55e01e" target='_blank'>
              Characterization of the Rat Osteosarcoma Cell Line UMR-106 by Long-Read Technologies Identifies a Large Block of Amplified Genes Associated with Human Disease
              </a>
            </td>
          <td>
            Alan F. Scott, David W. Mohr, William A. Littrell, Reshma Babu, Michelle Kokosinski, V. Stinnett, Janvi Madhiwala, John Anderson, Ying S. Zou, Kathleen L. Gabrielson
          </td>
          <td>2024-09-26</td>
          <td>Genes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Carcinogenesis predicts 30 million new cases by 2040, making it the second leading cause of death globally. In Brazil, neoplastic diseases resulted in more than 229,000 deaths in 2020, with 704,000 new cases expected between 2023 and 2025. Factors such as smoking, stress and genetic predisposition influence oncogenesis. Oncology interventions face challenges such as therapeutic resistance and tumor heterogeneity. Around 75% of the human genome is made up of non-coding RNAs (ncRNAs), with a focus on long ncRNAs. Previously considered 'evolutionary junk', lncRNAs regulate genes and affect cancer. Nuclear lncRNAs impact chromatin architecture, transcription and RNA processing. They function via complex pathways, modulating oncogenes and affecting resistance to treatment. Several lncRNAs, including MALAT1, ANRIL, HOTAIR, GAS5, MEG3 and H19, modulate oncogenic pathways and influence cellular processes and treatment resistance. Although promising as biomarkers, the structural complexity of lncRNAs hinders their clinical application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e63b0d46ddc30af66e31a8059fc320b5d198137c" target='_blank'>
              DECODING THE CANCER CODE:
              </a>
            </td>
          <td>
            Bruna Custódio Dias Duarte, Laurence Rodrigues do Amaral, Pedro Luiz Lima Bertarini, M. de Souza Gomes, Fábio Ribeiro Queiroz, Letícia da Conceição Braga
          </td>
          <td>2024-08-16</td>
          <td>Mário Penna Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Pancreatic cancer is an aggressive disease with limited therapeutic options. Gemcitabine is a clinically used chemotherapy for pancreatic cancer treatment but has limited efficacy. The molecular mechanisms responsible for gemcitabine resistance in pancreatic cancer are still unclear. Here, we analyzed the genomics of drug sensitivity in cancer dataset and identified various pancreatic cancer cells with distinct sensitivity to gemcitabine. Validation with proliferation and clonogenic assays confirmed that while CFPAC1 and PANC0327 are highly gemcitabine-sensitive, ASPC1 and PANC1 display intrinsic gemcitabine resistance. Analysis of the gene expression profile of these cells identified several genes that are differentially upregulated in resistant versus sensitive cells. Pathway enrichment analysis using the differential genes revealed the upregulation of Hippo, Wnt, MAPK signaling pathways as well as autophagy and glutathione metabolism in the resistant cells. Using quantitative PCR, immunoblotting and biochemical assays, and extracellular flux analysis methods, we have characterized the molecular distinctions between gemcitabine sensitive and resistant cells. Our results provide insight on the molecular mechanisms driving gemcitabine resistance in pancreatic cancer and could lead to novel therapeutic strategies to overcome resistance and improve patients’ treatment.
 Citation Format: Amina D Bilalbegovic, Bowen Fu, Zeribe Nwosu. Identification of novel drivers of gemcitabine resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B089.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52052963f11a91114395dd33df22ba52dd0b89f" target='_blank'>
              Abstract B089: Identification of novel drivers of gemcitabine resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Amina D Bilalbegovic, Bowen Fu, Z. Nwosu
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d317e0704a11a531980e863a514ed7e3ab276870" target='_blank'>
              Advances in pathogenesis research and challenges in treatment development for acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hiroki Yamaguchi
          </td>
          <td>2024-09-03</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Despite a critical role for tumor-initiating cancer stem cells (CSCs) in breast cancer progression, major questions remain about the properties and signaling pathways essential for their function. Recent discoveries highlighting mechanisms of CSC-resistance to the stress caused by chromosomal instability (CIN) may provide valuable new insight into the underlying forces driving stemness properties. While stress tolerance is a well-known attribute of CSCs, CIN-induced stress is distinctive since levels appear to increase during tumor initiation and metastasis. These dynamic changes in CIN levels may serve as a barrier constraining the effects of non-CSCs and shaping the stemness landscape during the early stages of disease progression. In contrast to most other stresses, CIN can also paradoxically activate pro-tumorigenic antiviral signaling. Though seemingly contradictory, this may indicate that mechanisms of CIN tolerance and pro-tumorigenic inflammatory signaling closely collaborate to define the CSC state. Together, these unique features may form the basis for a critical relationship between CIN and stemness properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d491eba9730d9558923c4e53a3264dd576b3117a" target='_blank'>
              Chromosomal instability as an architect of the cancer stemness landscape
              </a>
            </td>
          <td>
            Shahnawaz A. Baba, Aran Zakeri, J. Desgrosellier
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Ovarian cancer ecosystems are exceedingly complex, consisting of a high heterogeneity of cancer cells. Development of drugs such as poly ADP‐ribose polymerase (PARP) inhibitors, targeted therapies and immunotherapies offer more options for sequential or combined treatments. Nevertheless, mortality in metastatic ovarian cancer patients remains high because cancer cells consistently develop resistance to single and combination therapies, urging a need for treatment designs that target the evolvability of cancer cells. The evolutionary dynamics that lead to resistance emerge from the complex tumour microenvironment, the heterogeneous populations, and the individual cancer cell's plasticity. We propose that successful management of ovarian cancer requires consideration of the ecological and evolutionary dynamics of the disease. Here, we review current options and challenges in ovarian cancer treatment and discuss principles of tumour evolution. We conclude by proposing evolutionarily designed strategies for ovarian cancer, with the goal of integrating such principles with longitudinal, quantitative data to improve the treatment design and management of drug resistance. Key points/Highlights Tumours are ecosystems in which cancer and non‐cancer cells interact and evolve in complex and dynamic ways. Conventional therapies for ovarian cancer inevitably lead to the development of resistance because they fail to consider tumours’ heterogeneity and cellular plasticity. Eco‐evolutionarily designed therapies should consider cancer cell plasticity and patient‐specific characteristics to improve clinical outcome and prevent relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f584cd6ae71db959a312c8365ae2a9ada15cf5" target='_blank'>
              Ecological and evolutionary dynamics to design and improve ovarian cancer treatment
              </a>
            </td>
          <td>
            Grace Y Q Han, Monica Alexander, Julia Gattozzi, Marilyn Day, Elayna Kirsch, Narges Tafreshi, Raafat Chalar, Soraya Rahni, Gabrielle Gossner, William Burke, Mehdi Damaghi
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Microsatellite unstable colorectal cancer (MSI-CRC) can arise through germline mutations in mismatch repair (MMR) genes in individuals with Lynch syndrome (LS), or sporadically through promoter methylation of the MMR gene MLH1. Despite the different origins of hereditary and sporadic MSI tumours, their genomic features have not been extensively compared. A prominent feature of MMR-deficient genomes is the occurrence of many indels in short repeat sequences, an understudied mutation type due to the technical challenges of variant calling in these regions. In this study, we performed whole genome sequencing and RNA-sequencing on 29 sporadic and 14 hereditary MSI-CRCs. We compared the tumour groups by analysing genome-wide mutation densities, microsatellite repeat indels, recurrent protein-coding variants, signatures of single base, doublet base, and indel mutations, and changes in gene expression. We show that the mutational landscapes of hereditary and sporadic MSI-CRCs, including mutational signatures and mutation densities genome-wide and in microsatellites, are highly similar. Only a low number of differentially expressed genes were found, enriched to interferon-γ regulated immune response pathways. Analysis of the variance in allelic fractions of somatic variants in each tumour group revealed higher clonal heterogeneity in sporadic MSI-CRCs. Our results suggest that the differing molecular origins of MMR deficiency in hereditary and sporadic MSI-CRCs do not result in substantial differences in the mutational landscapes of these tumours. The divergent patterns of clonal evolution between the tumour groups may have clinical implications, as high clonal heterogeneity has been associated with decreased tumour immunosurveillance and reduced responsiveness to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43c6128f5465fd71306e14170da497710701fc42" target='_blank'>
              Lynch syndrome-associated and sporadic microsatellite unstable colorectal cancers: different patterns of clonal evolution yield highly similar tumours.
              </a>
            </td>
          <td>
            Samantha Martin, Riku Katainen, Aurora Taira, Niko Välimäki, Ari Ristimäki, T. Seppälä, L. Renkonen-Sinisalo, A. Lepistö, Kyösti Tahkola, Anne Mattila, S. Koskensalo, J. Mecklin, K. Rajamäki, Kimmo Palin, L. Aaltonen
          </td>
          <td>2024-08-24</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, P. Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technique has enabled detailed analysis of gene expression at the single cell level, enhancing the understanding of subtle mechanisms that underly pathologies and drug resistance. To derive such biological meaning from sequencing data in oncology, some critical processing must be performed, including identification of the tumor cells by markers and algorithms that infer copy number variations (CNVs). We compared the performance of sciCNV, InferCNV, CopyKAT and SCEVAN tools that identify tumor cells by inferring CNVs from scRNA-seq data. Sequencing data from Pancreatic Ductal Adenocarcinoma (PDAC) patients, adjacent and healthy tissues were analyzed, and the predicted tumor cells were compared to those identified by well-assessed PDAC markers. Results from InferCNV, CopyKAT and SCEVAN overlapped by less than 30% with InferCNV showing the highest sensitivity (0.72) and SCEVAN the highest specificity (0.75). We show that the predictions are highly dependent on the sample and the software used, and that they return so many false positives hence are of little use in verifying or filtering predictions made via tumor biomarkers. We highlight how critical this processing can be, warn against the blind use of these software and point out the great need for more reliable algorithms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f15f4004aa5098686401484d91f98cd52ad6608" target='_blank'>
              A Comparison of Tools That Identify Tumor Cells by Inferring Copy Number Variations from Single-Cell Experiments in Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Daisy J. A. Oketch, M. Giulietti, Francesco Piva
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The study of chromosomal shape, characteristics, and behavior in somatic cell division (mitosis) during growth and development and in germ cell division (meiosis) during reproduction is known as cytogenetics. Many techniques can be used for cytogenetics, including fluorescent in situ hybridization (FISH), spectral karyotyping (SKY), multicolor FISH (M-FISH), microarray, and optical genome mapping (OGM). OGM is a novel genome-wide method that can identify structural variants (SVs) and copy number variants (CNVs) with only one test. Genomic structural information that is difficult to obtain with DNA sequencing can be promptly obtained with OGM, in which large molecule lengths can be mapped at a reasonable cost. OGM is increasingly being used to investigate chromosome abnormalities in genetic disorders and human cancer, but it was first utilized in genome assembly and research. According to recent research, OGM is capable of identifying every clinically significant variation seen in trials using conventional care. OGM is being utilized to identify genomic abnormalities in patients with malignancies and constitutional illnesses. It is regarded as a revolution in the field of cytogenetics. Rather than sequencing DNA, OGM relies on DNA labeling. Currently, the OGM technique with the Saphyr system from Bionano Genomics is a widely utilized platform for cytogenetic analysis. In conclusion, OGM can now be considered a highly reliable method for the identification of chromosomal abnormalities in the diagnosis of tumors and hematological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e6e0a2264e6747c604c9c870d801b4fcbe0131" target='_blank'>
              Cytogenetics and the Revolution of Optical Genome Mapping in the Diagnosis of Diseases.
              </a>
            </td>
          <td>
            Osama M Al-Amer, Yahya Khubrani
          </td>
          <td>2024-09-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pediatric brain cancer is the leading cause of disease-related mortality in children, and many aggressive tumors still lack effective treatment strategies. Despite extensive studies characterizing these tumor genomes, alternative transcriptional splicing patterns remain underexplored. Here, we systematically characterized aberrant alternative splicing across pediatric brain tumors, identifying pediatric high-grade gliomas (HGGs) among the most heterogeneous. Through integration with UniProt Knowledgebase annotations, we identified 12,145 splice events in 5,424 genes, leading to functional changes in protein activation, folding, and localization. We discovered that the master splicing factor and cell-cycle modulator, CDC- like kinase 1 (CLK1), is aberrantly spliced in HGGs to include exon 4, resulting in a gain of two phosphorylation sites and subsequent activation of CLK1. Inhibition of CLK1 with Cirtuvivint in the pediatric HGG KNS-42 cell line significantly decreased both cell viability and proliferation in a dose-dependent manner. Morpholino-mediated depletion of CLK1 exon 4 splicing reduced RNA expression, protein abundance, and cell viability. Notably, KNS-42 cells treated with the CLK1 exon 4 morpholino demonstrated differential expression 78 genes and differential splicing with loss or gain of a functional site in 193 genes annotated as oncogene or tumor suppressor genes (TSGs). These genes were enriched for cancer-associated pathways, with 20 identified as significant gene dependencies in pediatric HGGs. Our findings highlight a dependency of pediatric HGGs on CLK1 and its roles contributing to tumor splicing heterogeneity through transcriptional dysregulation of splicing factors and transcriptional modulation of oncogenes. Overall, aberrant splicing in HGGs and other pediatric brain tumors represents a potentially targetable oncogenic pathway contributing to tumor growth and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624af22ec9a4e826208669859a67ff2ad429da0e" target='_blank'>
              Characterization of aberrant splicing in pediatric central nervous system tumors reveals CLK1 as a candidate oncogenic dependency
              </a>
            </td>
          <td>
            Ammar S. Naqvi, Ryan J. Corbett, Priyanka Seghal, Karina L. Conkrite, Komal S. Rathi, Brian M. Ennis, Katharina E. Hayer, Bo Zhang, Miguel A. Brown, Daniel P. Miller, Adam A. Kraya, Joseph M. Dybas, Zhuangzhuang Geng, Christopher Blackden, Shehbeel Arif, Antonia Chroni, Aditya Lahiri, Madison L. Hollawell, Phillip B. Storm, J. Foster, Mateusz Koptyra, Peter Madsen, S. Diskin, A. Thomas-Tikhonenko, Adam C. Resnick, J. L. Rokita
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Crizotinib, as the first-generation of anaplastic lymphoma kinase (ALK) inhibitor, effectively improves the survival time of ALK-positive non-small cell lung cancer (NSCLC) patients. However, its efficacy is severely limited by drug resistance caused by secondary mutations. G1202R and L1196M are classical mutation sites located in ALK kinase domain. They may hinder the binding of ALK inhibitors to the target kinase domain, resulting in drug resistance in patients. However, the exact mechanism of drug resistance mediated by these mutations remains unclear. In this study, we aimed to evaluate how G1202R and L1196M mutations mediate crizotinib resistance. To explore the resistance mechanism, we constructed EML4-ALK G1202R and L1196M mutant cell lines with A549 cells. The results showed that the mutant cells exhibited significant epithelial-mesenchymal transition (EMT) and metastasis compared to control (A549-vector) or wild type (A549-EML4-ALK) cells. Subsequently, it was found that the occurrence of EMT was correlated to the high expression of murine double minute 2 (MDM2) protein and the activation of mitogen-activated protein kinase (MEK)/extracellular signal-regulated kinase (ERK) pathway in mutant cells. Down-regulation of MDM2 inhibited the activation of MEK/ERK pathway, thus reversed the EMT process and markedly increased the inhibitory effect of crizotinib on the growth of mutant cells. Collectively, resistance of ALK-positive NSCLC cells to crizotinib is induced by G1202R and L1196M mutations through activation of the MDM2/MEK/ERK signalling axis, promoting EMT process and metastasis. These findings suggest that the combination of MDM2 inhibitors and crizotinib could be a potential therapeutic strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/241304b35cd14de845e9005a8b17e3d2b9e3840d" target='_blank'>
              EML4-ALK G1202R and EML4-ALK L1196M mutations induce crizotinib resistance in non-small cell lung cancer cells through activating epithelial-mesenchymal transition mediated by MDM2/MEK/ERK signal axis.
              </a>
            </td>
          <td>
            Yuying Yang, Huan Yang, Yunhui Gao, Qian Yang, Xinya Zhu, Qianying Miao, Xiaobo Xu, Zengqiang Li, Daiying Zuo
          </td>
          <td>2024-09-24</td>
          <td>Cell biology international</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="TP53 functions as a central regulator in response to DNA damage and other cell stressors by inducing the expression of many protective pathways such as cell cycle arrest and apoptosis. Consequently, this gene is often found disrupted in human cancers. Elephants are a particularly interesting species for the study of cancer, by virtue of their large number of cell divisions and long lives yet low incidence of cancer. Elephants also possess multiple retrogene copies of TP53, which have previously been shown to induce strong cellular responses to DNA damage. However, most previous studies have largely focused only on African elephant TP53 retrogenes and often in non-native backgrounds. Here we generated CRISPR-Cas9 knockouts of TP53, all 29 TP53 retrogenes, or both in combination in Asian elephant fibroblasts. We find that while there is considerable overlap in the DNA damage responses of the TP53 and retrogene knockouts, there are also many unique pathways enriched in both. In particular, the retrogene knockouts exhibit strong enrichment of many extracellular pathways suggesting they may play a large role in the tumor microenvironment and mitigating metastatic growth. We also find that only a small fraction of these 29 retrogenes appear to be expressed across a variety of tissues and identify three loci that are likely driving this response. This work shows for the first time the transcriptomic effect of these retrogenes within their native background and establishes a foundation for future research into the relative contributions of these genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01cd22e9caf6290532e5ca297543c409aee6bcfc" target='_blank'>
              Engineering Asian elephant TP53: TP53 retrogene knockouts activate common and unique cancer-relevant pathways
              </a>
            </td>
          <td>
            Emil Karpinski, Nikil Badey, Esther Mintzer, Asaf Ashkenazy-Titelman, George M. Church
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU- 023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention. Author Summary Bile duct cancer has the highest incidence in northeastern Thailand. In this study, we explored the mtDNA of CCA cell lines, as mitochondria play a key role in cellular energy production. We aimed to identify mutations that could serve as therapeutic targets or biomarkers. Our analysis revealed that each CCA cell line has unique mtDNA profiles, including INDELs, SVs, and SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. yet they also shared some common mutations. Future research is needed to understand how these mutations affect cell function, interact with nuclear DNA, and their potential for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eafb79f26f32f1aeaf6d648b336c327747f051f" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapan, A. Jareanrat, V. Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Simple Summary This review provides a comprehensive overview of the current knowledge on mutational signatures in colorectal cancer (CRC), highlighting their potential clinical applications. It discusses the challenges and limitations in translating these analyses into clinical practice and proposes strategies to overcome these obstacles. Additionally, it provides insights into future directions and emerging proof-of-concept studies that highlight the translational potential of mutational signature analysis in improving patient care and outcomes in CRC. Abstract A multitude of exogenous and endogenous processes have the potential to result in DNA damage. While the repair mechanisms are typically capable of correcting this damage, errors in the repair process can result in mutations. The findings of research conducted in 2012 indicate that mutations do not occur randomly but rather follow specific patterns that can be attributed to known or inferred mutational processes. The process of mutational signature analysis allows for the inference of the predominant mutational process for a given cancer sample, with significant potential for clinical applications. A deeper comprehension of these mutational signatures in CRC could facilitate enhanced prevention strategies, facilitate the comprehension of genotoxic drug activity, predict responses to personalized treatments, and, in the future, inform the development of targeted therapies in the context of precision oncology. The efforts of numerous researchers have led to the identification of several mutational signatures, which can be categorized into different mutational signature references. In CRC, distinct mutational signatures are identified as correlating with mismatch repair deficiency, polymerase mutations, and chemotherapy treatment. In this context, a mutational signature analysis offers considerable potential for enhancing minimal residual disease (MRD) tests in stage II (high-risk) and stage III CRC post-surgery, stratifying CRC based on the impacts of genetic and epigenetic alterations for precision oncology, identifying potential therapeutic vulnerabilities, and evaluating drug efficacy and guiding therapy, as illustrated in a proof-of-concept clinical trial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9f4817ac8c40910eefa1198ffb3e184664ea3d" target='_blank'>
              Mutational Signatures in Colorectal Cancer: Translational Insights, Clinical Applications, and Limitations
              </a>
            </td>
          <td>
            G. Crisafulli
          </td>
          <td>2024-08-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Objective The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions. Methods We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters. Results Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations. Conclusion Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, weisen fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background: Transcriptomics can reveal much about cellular activity, and cancer transcriptomics have been useful in investigating tumor cell behaviors. Patterns in transcriptome-wide gene expression can be used to investigate biological mechanisms and pathways that can explain the variability in patient response to cancer therapies. Methods: We identified gene expression patterns related to patient drug response by clustering tumor gene expression data and selecting from the resulting gene clusters those where expression of cluster genes was related to patient survival on specific drugs. We then investigated these gene clusters for biological meaning using several approaches, including identifying common genomic locations and transcription factors whose targets were enriched in these clusters and performing survival analyses to support these candidate transcription factor-drug relationships. Results: We identified gene clusters related to drug-specific survival, and through these, we were able to associate observed variations in patient drug response to specific known biological phenomena. Specifically, our analysis implicated 2 stem cell-related transcription factors, HOXB4 and SALL4, in poor response to temozolomide in brain cancers. In addition, expression of SNRNP70 and its targets were implicated in cetuximab response by 3 different analyses, although the mechanism remains unclear. We also found evidence that 2 cancer-related chromosomal structural changes may impact drug efficacy. Conclusion: In this study, we present the gene clusters identified and the results of our systematic analysis linking drug efficacy to specific transcription factors, which are rich sources of potential mechanistic relationships impacting patient outcomes. We also highlight the most promising of these results, which were supported by multiple analyses and by previous research. We report these findings as promising avenues for independent validation and further research into cancer treatments and patient response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12d8ad110c71ebbda4c529cf36eba8f982598ee0" target='_blank'>
              Characterization of Expression-Based Gene Clusters Gives Insights into Variation in Patient Response to Cancer Therapies
              </a>
            </td>
          <td>
            B. Neary, Peng Qiu
          </td>
          <td>2024-01-01</td>
          <td>Cancer Informatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Genomic rearrangements are chromosomal abnormalities that alter the arrangement of genes and are important for diagnosing multiple cancer types and guiding therapy selection. However, current diagnostic tests may not detect all genomic rearrangements. Hi-C sequencing is a promising new method for detecting genomic rearrangements. First, we examined whether Hi-C can detect known genomic rearrangements in pediatric leukemia and sarcoma specimens. Next, we evaluated whether Hi-C can detect genomic rearrangements that were not found using standard diagnostic testing. Hi-C showed complete agreement with other diagnostic methods in identifying known rearrangements. Hi-C also detected previously unknown genomic rearrangements in 5/11 pediatric leukemia cases that could impact diagnosis, prognosis, or treatment choices. These data suggest Hi-C could be beneficial for medical diagnostic testing of pediatric cancers, but more extensive clinical validation is needed. Abstract Hi-C sequencing is a DNA-based next-generation sequencing method that preserves the 3D genome conformation and has shown promise in detecting genomic rearrangements in translational research studies. To evaluate Hi-C as a potential clinical diagnostic platform, analytical concordance with routine laboratory testing was assessed using primary pediatric leukemia and sarcoma specimens. Archived viable and non-viable frozen leukemic cells and formalin-fixed paraffin-embedded (FFPE) tumor specimens were analyzed. Pediatric acute myeloid leukemia (AML) and alveolar rhabdomyosarcoma (A-RMS) specimens with known genomic rearrangements were subjected to Hi-C to assess analytical concordance. Subsequently, a discovery cohort consisting of AML and acute lymphoblastic leukemia (ALL) cases without known genomic rearrangements based on prior clinical diagnostic testing was evaluated to determine whether Hi-C could detect rearrangements. Using a standard sequencing depth of 50 million raw read-pairs per sample, or approximately 5X raw genomic coverage, we observed 100% concordance between Hi-C and previous clinical cytogenetic and molecular testing. In the discovery cohort, a clinically relevant gene fusion was detected in 45% of leukemia cases (5/11). This study provides an institutional proof of principle evaluation of Hi-C sequencing to medical diagnostic testing as it identified several clinically relevant rearrangements, including those that were missed by current clinical testing workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af941e63f57e2e2bb71ed7543d231768d840bed5" target='_blank'>
              Evaluation of Hi-C Sequencing for Detection of Gene Fusions in Hematologic and Solid Tumor Pediatric Cancer Samples
              </a>
            </td>
          <td>
            Anthony D. Schmitt, Kristin Sikkink, Atif A. Ahmed, Shadi Melnyk, Derek Reid, Logan Van Meter, Erin M. Guest, Lisa A. Lansdon, Tomi Pastinen, Irina Pushel, B. Yoo, M. Farooqi
          </td>
          <td>2024-08-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/860a1cc9d077ab2536411e26d625b8a5e8cd579d" target='_blank'>
              Epigenome-wide impact of MAT2A sustains the androgen-indifferent state and confers synthetic vulnerability in ERG fusion-positive prostate cancer
              </a>
            </td>
          <td>
            A. Cacciatore, D. Shinde, C. Musumeci, G. Sandrini, L. Guarrera, D. Albino, G. Civenni, Elisa Storelli, S. Mosole, Elisa Federici, Alessio Fusina, Marta Iozzo, Andrea Rinaldi, Matteo Pecoraro, Roger Geiger, Marco Bolis, C. Catapano, G. M. Carbone
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d61b6a296f0f4f8539e1a19d49c6d27ddf36337d" target='_blank'>
              Deciphering the role of sphingolipid metabolism in the immune microenvironment and prognosis of esophageal cancer via single-cell sequencing and bulk data analysis
              </a>
            </td>
          <td>
            Rongzhang He, Jing Tang, Haotian Lai, Tianchi Zhang, Linjuan Du, Siqi Wei, Ping Zhao, Guobin Tang, Jie Liu, Xiufang Luo
          </td>
          <td>2024-09-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8e1be4efe22c9fbd98a98ceb3e0939364442527" target='_blank'>
              Genome-wide CRISPR screening identifies tyrosylprotein sulfotransferase-2 as a target for augmenting anti-PD1 efficacy
              </a>
            </td>
          <td>
            Yumi Oh, Sujeong Kim, Yunjae Kim, Hyun Kim, Dongjun Jang, Seungjae Shin, Soo Jin Lee, Jiwon Kim, Sang Eun Lee, Jaeik Oh, Yoojin Yang, Dohee Kim, Hae-Rim Jung, Sangjin Kim, Jihui Kim, K. Min, Beomki Cho, Hoseok Seo, Dohyun Han, Hansoo Park, Sung-Yup Cho
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Pediatric acute myeloid leukemia (AML) exhibits distinct characteristics between infants and children, manifested by variations in clinical features, cytogenetic abnormalities, and molecular aberrations. While most studies have focused on the associations between genomic abnormalities, age groups, and disease status, the transcriptional profiles and gene regulatory mechanisms underlying these differences remain underexplored. In this study, through differential gene expression analysis and Weighted Gene Co-expression Network Analysis, I have identified gene groups associated with AML and further categorized them by infant and childhood subgroups. The findings reveal three distinct gene groups with age- and tissue-specific expression patterns. Additionally, I propose a gene regulatory circuitry that elucidates the differences between infant and childhood AML. Several novel markers demonstrating significant expression changes in tumors were identified. Moreover, a comprehensive gene regulatory network for pediatric AML was constructed using differentially expressed transcription factors and protein-gene interaction data. This study highlights new gene regulation mechanisms in pediatric AML and offers potential avenues for developing novel biomarkers and therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdcb7517a7ea15fd2b0b9215c835acfed1519a12" target='_blank'>
              Dissecting the transcriptional regulation of infant and childhood acute myeloid leukemia
              </a>
            </td>
          <td>
            Rasoul Godini
          </td>
          <td>2024-08-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer is a multifaceted disease influenced by both intrinsic cellular traits and extrinsic factors, with the tumor microenvironment (TME) being crucial in its progression. To satisfy their high proliferation and aggressiveness, cancer cell always plunders large amounts of nutrition and releases various signals to the surrounding, forming a dynamic TME with special metabolic, immune, microbial and physical characteristics. Due to the neglect of interactions between tumor cell and TME, traditional cancer therapies often struggle with challenges such as drug resistance, low efficacy, and recurrence. Importantly, with the development of gene editing technologies, particularly the CRISPR-Cas system, offers promising new strategies for cancer treatment. Combined with nanomaterials strategies, CRISPR-Cas technology exhibits precision, affordability, and user-friendliness with reduced side effects, which holds great promise for profoundly altering the TME at a genetic level, potentially leading to lasting anticancer outcomes. This review will delve into how CRISPR-Cas can be leveraged to manipulate the TME, examining its potential as a transformative anticancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b4994cb649ea8f94220d3676ac8b2eeb79b1144" target='_blank'>
              Harnessing Gene Editing Technology for Tumor Microenvironment Modulation: An Emerging Anticancer Strategy.
              </a>
            </td>
          <td>
            Shujun Feng, Yu Zhang, Yanyi Wang, Yanfeng Gao, Yujun Song
          </td>
          <td>2024-09-03</td>
          <td>Chemistry</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecf274ccb3cfdd7406ad686ecfcdad4f5ec5eb3b" target='_blank'>
              Evolutionary dependency of cancer mutations in gene pairs inferred by nonsynonymous-synonymous mutation ratios
              </a>
            </td>
          <td>
            Dong-Jin Han, Sunmin Kim, Seo-Young Lee, Youngbeen Moon, Su Jung Kang, Jinseon Yoo, Hye Young Jeong, Hae Jin Cho, Jeong Yang Jeon, Byeong Chang Sim, Jaehoon Kim, Seungho Lee, Ruibin Xi, Tae-Min Kim
          </td>
          <td>2024-08-19</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Approximately 50% of patients diagnosed with ovarian cancer harbor tumors with mutations in BRCA1, BRCA2, or other genes involved in homologous recombination repair (HR). The presence of homologous recombination deficiency (HRD) is an approved biomarker for poly-ADP-ribose polymerase inhibitors (PARPis) as a maintenance treatment following a positive response to initial platinum-based chemotherapy. Despite this treatment option, the development of resistance to PARPis is common among recurrent disease patients, leading to a poor prognosis. In this study, we conducted a comprehensive analysis using publicly available datasets to elucidate the molecular mechanisms driving PARPi resistance in BRCA1-deficient ovarian cancer. Our findings reveal a central role for the interferon (IFN) pathway in mediating resistance in the context of BRCA1 deficiency. Through integrative bioinformatics approaches, we identified LY6E, an interferon-stimulated gene, as a key mediator of PARPi resistance, with its expression linked to an immunosuppressive tumor microenvironment (TME) encouraging tumor progression and invasion. LY6E amplification correlates with poor prognosis and increased expression of immune-related gene signatures, which is predictive of immunotherapy response. Interestingly, LY6E expression upon PARPi treatment resistance was found to be dependent on BRCA1 status. Gene expression analysis in the Orien/cBioPortal database revealed an association between LY6E and genes involved in DNA repair, such as Rad21 and PUF60, emphasizing the interplay between DNA repair pathways and immune modulation. Moreover, PUF60, Rad21, and LY6E are located on chromosome 8q24, a locus often amplified and associated with the progression of ovarian cancer. Overall, our study provides novel insights into the molecular determinants of PARPi resistance and highlights LY6E as a promising prognostic biomarker in the management of HRD ovarian cancer. Future studies are needed to fully elucidate the molecular mechanisms underlying the role of LY6E in PARPi resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4052305f8854485a3b85613abf68315aff09bfc" target='_blank'>
              Exploring Molecular Drivers of PARPi Resistance in BRCA1-Deficient Ovarian Cancer: The Role of LY6E and Immunomodulation
              </a>
            </td>
          <td>
            T. Petta, Joseph Carlson
          </td>
          <td>2024-09-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pediatric acute lymphoblastic leukemia (ALL) is a highly heterogeneous disease at the molecular level, with over 20 subtypes of B-cell ALL (B-ALL) identified to date. These subtypes are critical for guiding risk-adapted therapy and precision medicine, as they are defined by specific chromosomal rearrangements, gene expression profiles, aneuploidies and point mutations. However, the ability to accurately classify these subtypes is limited by the technical and economic challenges faced by many centers, particularly in low- and middle-income countries. In this study, we employed transcriptome sequencing to perform molecular classification of B-ALL at diagnosis in pediatric patients enrolled in the multicentric ALLIC-GATLA-2010 clinical protocol in Argentina. Using a combination of bioinformatic tools, we successfully identified single nucleotide variants, fusion transcripts and gene expression profiles, achieving molecular classification in over 90% of patients. Our analysis also revealed high-risk molecular features and novel genetic alterations. These findings hold potential clinical value for improving risk stratification and identifying therapeutic targets, particularly for patients who remain unclassified by conventional diagnostic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f39fa251d291498250d854c37a9028d4e85f79" target='_blank'>
              Molecular Characterization of Pediatric Acute Lymphoblastic Leukemia via Integrative Transcriptomics: A Multicenter Study in Argentina
              </a>
            </td>
          <td>
            Sol Ruiz, María Mercedes Abbate, Ezequiel Sosa, Daniel Avendaño, Ignacio Gomez Mercado, María Laura Lacreu, Mariano Riccheri, Virginia Schuttenberg, Luis Aversa, E. Vazquez, G. Gueron, J. Cotignola
          </td>
          <td>2024-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Current therapies for neuroblastoma are often ineffective and survivors suffer from severe long-term therapy related side-effects, underscoring the need for identification of novel drugging strategies. We performed an in-depth evaluation of phenotypic and molecular responses following exposure of neuroblastoma cells to the rocaglate CR-1-31-B, scrutinizing its mode-of-action through integrative ribosome footprinting and shotgun proteome profiling. We could show that CR-1-31-B significantly reduces tumor growth in vivo without apparent toxicity. By means of combined ribosome footprinting and transcriptome analysis we uncovered that CR-1-31-B treatment downregulates translation efficiencies of several major neuroblastoma dependencies including MYCN, CCND1 and ALK as well as factors involved in the G2/M checkpoint. Upregulated targets are enriched for oxidative phosphorylation pathway components and DNA repair. At the proteome level, CR-1-31-B imposed downregulation of a FOXM1 driven signature, including the FOXM1 target gene TPX2. We show that neuroblastoma cells are dependent on TPX2 for growth and DNA repair and further demonstrate enhanced CHK1 sensitivity upon TPX2 knockdown. Next, we also observed synergistic effects of CHK1 inhibition with CR-1-31-B. In conclusion, our data support CR-1-31-B as a potent novel therapeutic agent in neuroblastoma, in particular in combination with DNA damage or replication stress inducing agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf8560705e95704cc4eb577ffabb3bc018ccfcd1" target='_blank'>
              5’UTR translational inhibition of neuroblastoma dependency factors using the CR-1-31-B rocaglate
              </a>
            </td>
          <td>
            C. Nunes, SL Bekaert, S. Parsa, I. Nelen, F. Martens, E. de Stanchina, E. Sanders, E. Hilgert, S. T’sas, P. Zhao, F. De Vloed, A. Eggermont, S. Goossens, A. Sablina, L. Depestel, HG Wendel, K. Durinck
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Metastasis is the leading cause of cancer-related deaths. It is unclear how intratumor heterogeneity (ITH) contributes to metastasis and how metastatic cells adapt to distant tissue environments. The study of these adaptations is challenged by the limited access to patient material and a lack of experimental models that appropriately recapitulate ITH. To investigate metastatic cell adaptations and the contribution of ITH to metastasis, we analyzed single-cell transcriptomes of matched primary tumors and metastases from patient-derived xenograft models of breast cancer. We found profound transcriptional differences between the primary tumor and metastatic cells. Primary tumors upregulated several metabolic genes, whereas motility pathway genes were upregulated in micrometastases, and stress response signaling was upregulated during progression. Additionally, we identified primary tumor gene signatures that were associated with increased metastatic potential and correlated with patient outcomes. Immune-regulatory control pathways were enriched in poorly metastatic primary tumors, whereas genes involved in epithelial-mesenchymal transition were upregulated in highly metastatic tumors. We found that ITH was dominated by epithelial-mesenchymal plasticity (EMP), which presented as a dynamic continuum with intermediate EMP cell states characterized by specific genes such as CRYAB and S100A2. Elevated expression of an intermediate EMP signature correlated with worse patient outcomes. Our findings identified inhibition of the intermediate EMP cell state as a potential therapeutic target to block metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4040c30de029de626b99afe51b1e8ce55165671f" target='_blank'>
              Single-cell analysis of breast cancer metastasis reveals epithelial-mesenchymal plasticity signatures associated with poor outcomes
              </a>
            </td>
          <td>
            Juliane Winkler, Weilun Tan, Catherine M. M. Diadhiou, Christopher S. McGinnis, Aamna J. Abbasi, Saad Hasnain, Sophia Durney, Elena Atamaniuc, Daphne A. Superville, L. Awni, Joyce V. Lee, Johanna H. Hinrichs, Patrick S. Wagner, Namrata Singh, Marco Y. Hein, M. Borja, Angela M Detweiler, Su-Yang Liu, Ankitha Nanjaraj, Vaishnavi Sitarama, H. Rugo, Norma Neff, Zev J. Gartner, Angela Oliveira Pisco, Andrei Goga, S. Darmanis, Z. Werb
          </td>
          <td>2024-09-03</td>
          <td>The Journal of Clinical Investigation</td>
          <td>1</td>
          <td>116</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M. Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="
 Background: Hispanic American (HA) men with prostate cancer (PC) are more likely to have more aggressive and advanced diseases and greater mortality rates compared to non-Hispanic white (NHW) men. Resistance to the current anticancer therapy is the major hurdle that leads to poor clinical outcomes. Hedgehog, Notch, and Wnt signaling pathways play a crucial role in development, progression, and resistance to the various anticancer therapies. Here, we characterized the enrichment of Hedgehog, Notch, and Wnt signaling pathways and molecular tumors differences between HA and NHW patients. Methods: Our data consist of 88 PC tumors samples (HA=34, NHW=54) obtained from treatment-naive metastatic hormone-sensitive prostate cancer (mHSPC) that were analyzed by next-generation sequencing (592, NextSeq; Whole Exome Sequencing, NovaSeq) and Whole Transcriptome Sequencing (WTS; NovaSeq) (Caris Life Sciences, Phoenix, AZ). Pathway enrichment was determined by Gene Set Enrichment Analysis (GSEA, Broad Institute) by measurable TPM (transcripts per kilobase million). The medical records were reviewed in a deidentified fashion for clinical features. Statistical significance was determined using chi-square and Mann-Whitney U (p<0.05). Results: HA PC had enrichment of Hedgehog signaling (NES: 1.45, FDR=0.04), Notch signaling (NES: 1.41, FDR=0.07), and Wnt/β-catenin signaling (NES: 1.38, FDR=0.11) pathways compared to NHW tumors. HA tumors had higher expression of genes (HES1, Notch1, WNT1, WNT2; FC:1.1-2.5) associated with crosstalk between Hedgehog, Notch, and Wnt/β- catenin signaling pathways (all p<0.05). Moreover, HA tumors had higher expression of stemness (ALDH1A2, ALDH1A1, PROM1; FC: 1.7-3.0), drug efflux (ABCC1, ABCB1; FC: 1.1- 1.5), epithelial to mesenchymal transition (EMT) (TWIST1, MMP2; FC: 1.1-1.5), cell cycle (CCND2; FC: 1.2), inhibition of apoptosis (BIRC2, XIAP; FC: 1.1) and epigenetic (EP300, TET1, TET2; FC: 1.3-1.4) related genes compared to NHW tumors (all p<0.05). HA PC had higher frequency of Hedgehog and Wnt/β-catenin associated SPOP (16% vs 6.6%, p=0.2) and CTNNB1 (12% vs 2.2%, p=0.1) mutation compared to NHW PC. Conclusion: These results demonstrate that tumors from HA patients had enrichment of oncogenic Hedgehog, Notch, and Wnt/β-catenin signaling pathways, higher expression of stemness, drug efflux, EMT, cell cycle, inhibition of apoptosis and epigenetic-related genes, and mutations directly involved in activating the Hedgehog and Wnt/β-catenin signaling pathways. Together, these findings suggest that tumors from HA patients are enriched for gene signatures that promote cancer progression and therapy resistance. Additional research on collaboration and crosstalk between Hedgehog, Notch, and Wnt/β-catenin signaling pathways may help in designing better therapeutic strategies for HA men with mHSPC.
 Citation Format: Alejandro Recio-Boiles, Sachin Kumar Deshmukh, Ricardo J. Estrada-Mendizabal, Kenneth Barker, Sharon Wu, Joanne Xiu, Milan Radovich, Elisabeth I. Heath, Rana McKay, Juan Chipollini, Cynthia K. Miranti. Defining the role of Hedgehog, Notch, and Wnt signaling pathways in aggressive lethal prostate cancer of Hispanic men [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C100.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084e712d5ece7637137ae910e70858f7d21562a6" target='_blank'>
              Abstract C100: Defining the role of Hedgehog, Notch, and Wnt signaling pathways in aggressive lethal prostate cancer of Hispanic men
              </a>
            </td>
          <td>
            Alejandro Recio-Boiles, S. Deshmukh, Ricardo J. Estrada-Mendizabal, Kenneth Barker, Sharon Wu, Joanne Xiu, M. Radovich, Elisabeth I. Heath, Rana McKay, Juan Chipollini, Cynthia K. Miranti
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5edea2d9a83527bbebc5d48f3ba44b53a9b424d3" target='_blank'>
              Single-cell profiling uncovers proliferative cells as key determinants of survival outcomes in lower-grade glioma patients
              </a>
            </td>
          <td>
            Jian-Bing Peng, Qing Zhang, Xiaofeng Zhu, Zhu Yan, Meng Zhu
          </td>
          <td>2024-09-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="TP53 is a tumor suppressor gene that is mutated in most cancer types and has been extensively studied in cancer research. p53 plays a critical role in regulating the expression of target genes and is involved in key processes such as apoptosis, cell cycle regulation, and genomic stability, earning it the title “guardian of the genome.” Numerous studies have demonstrated p53’s influence on and regulation of autophagy, ferroptosis, the tumor microenvironment, and cell metabolism, all of which contribute to tumor suppression. Alterations in p53, specifically mutant p53 (mutp53), not only impair its tumor-suppressing functions but also enhance oncogenic characteristics. Recent data indicate that mutp53 is strongly associated with poor prognosis and advanced cancers, making it an ideal target for the development of novel cancer therapies. This review summarizes the post-translational modifications of p53, the mechanisms of mutp53 accumulation, and its gain-of-function, based on previous findings. Additionally, this review discusses its impact on metabolic homeostasis, ferroptosis, genomic instability, the tumor microenvironment, and cancer stem cells, and highlights recent advancements in mutp53 research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea389a27a85074a44af76e077628c890d00a2c8" target='_blank'>
              Roles of Mutant TP53 Gene in Cancer Development and Progression
              </a>
            </td>
          <td>
            Muhammad Abubakar, Baqaur Rehman
          </td>
          <td>2024-09-25</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chronic Lymphocytic Leukemia (CLL) is the most common B-cell malignancy in the Western world, characterized by frequent relapses despite temporary remissions. Our study integrated publicly available proteomic, transcriptomic, and patient survival datasets to identify key differences between healthy and CLL samples. We exposed approximately 1000 proteins that differentiate healthy from cancerous cells, with 608 upregulated and 415 downregulated in CLL cases. Notable upregulated proteins include YEATS2 (an epigenetic regulator), PIGR (Polymeric immunoglobulin receptor), and SNRPA (a splicing factor), which may serve as prognostic biomarkers for this disease. Key pathways implicated in CLL progression involve RNA processing, stress resistance, and immune response deficits. Furthermore, we identified three existing drugs—Bosutinib, Vorinostat, and Panobinostat—for potential further investigation in drug repurposing in CLL. We also found limited correlation between transcriptomic and proteomic data, emphasizing the importance of proteomics in understanding gene expression regulation mechanisms. This generally known disparity highlights once again that mRNA levels do not accurately predict protein abundance due to many regulatory factors, such as protein degradation, post-transcriptional modifications, and differing rates of translation. These results demonstrate the value of integrating omics data to uncover deregulated proteins and pathways in cancer and suggest new therapeutic avenues for CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524d87cfc3fcaee95b1a168a590bc69c95feb91a" target='_blank'>
              Integrative Analysis of Multi-Omics Data to Identify Deregulated Molecular Pathways and Druggable Targets in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            D. Mavridou, K. Psatha, M. Aivaliotis
          </td>
          <td>2024-08-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/553f4560eb657fca2fe86385e460c71427df430c" target='_blank'>
              GLI1 confers resistance to PARP inhibitors by activating the DNA damage repair pathway.
              </a>
            </td>
          <td>
            Hiroshi Ikeuchi, Yusuke Matsuno, Rika Kusumoto-Matsuo, S. Kojima, Toshihide Ueno, Masachika Ikegami, Rina Kitada, Hitomi Sumiyoshi-Okuma, Yuki Kojima, K. Yonemori, Y. Yatabe, K. Takamochi, Kenji Suzuki, Ken‐ichi Yoshioka, H. Mano, S. Kohsaka
          </td>
          <td>2024-08-03</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 METTL1 is an RNA methyltransferase which catalyzes N7-methylguanosine (m7G) methylation on various RNAs, particularly on tRNAs. METTL1 has been reported to be amplified and/or overexpressed and high expression correlates with poor patient prognosis in different types of adult cancers. Our ZERO Childhood Cancer Program focuses on identifying and validating potential new drug targets and developing novel therapies for clinical use. ZERO has sequenced the whole genome and transcriptome in over 500 high-risk pediatric tumors and found that METTL1 gene copy number and gene expression are significantly higher in a subset of high-grade glioma, neuroblastoma, diffuse midline glioma and fusion positive rhabdomyosarcoma. Analysis of the publicly available SEQC microarray dataset showed that high METTL1 expression correlates with poor outcomes in MYCN amplified neuroblastoma patients. We validated the functional roles of METTL1 gene amplification and overexpression in a few pediatric cancer cell line models using the Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) Cas13d knockdown system, in dual lentiviral constructs. We found that knocking down METTL1 significantly decreased METTL1 amplified neuroblastoma and rhabdomyosarcoma cell viability and colony formation but not in METTL1 non-amplified neuroblastoma and rhabdomyosarcoma cells. Gene Set Enrichment Analysis from RNA sequencing data revealed that knocking down METTL1 resulted in a decrease in Myc target gene expression, and an increased in apoptosis, interferon alpha response and p53 pathway hallmarks. Western blot analysis confirmed that knocking down METTL1 led to a decrease in the MCL-1 antiapoptotic protein expression and an increase in the proapoptotic protein expression such as cleaved PARP and caspase 3. Taken together, our data demonstrate that METTL1 is a potential therapeutic target in METTL1 amplified pediatric cancers.
 Citation Format: Pei Y. Liu, Steven He, Chelsea Mayoh, Alice Salib, Antoine De Weck, Paul G. Ekert. Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64085373809830d692bd2bd083b18380125ea0bf" target='_blank'>
              Abstract A012 Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d
              </a>
            </td>
          <td>
            Pei Y. Liu, Steven He, C. Mayoh, A. Salib, Antoine de Weck, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/460b0c287af289dbf822999647d9d57a5f2960b9" target='_blank'>
              Integrative single-cell RNA-seq and spatial transcriptomics analyses reveal diverse apoptosis-related gene expression profiles in EGFR-mutated lung cancer
              </a>
            </td>
          <td>
            Motohiro Izumi, Masanori Fujii, I. Kobayashi, Vivian Ho, Y. Kashima, H. Udagawa, Daniel B. Costa, Susumu S. Kobayashi
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="TP53, the most frequently mutated gene in human cancer, encodes a transcriptional activator that induces myriad downstream target genes. Despite the importance of p53 in tumor suppression, the specific p53 target genes important for tumor suppression remain unclear. Recent studies have identified the p53-inducible gene Zmat3 as a critical effector of tumor suppression, but many questions remain regarding its p53-dependence, activity across contexts, and mechanism of tumor suppression alone and in cooperation with other p53-inducible genes. To address these questions, we used Tuba-seqUltra somatic genome editing and tumor barcoding in a mouse lung adenocarcinoma model, combinatorial in vivo CRISPR/Cas9 screens, meta-analyses of gene expression and Cancer Dependency Map data, and integrative RNA-sequencing and shotgun proteomic analyses. We established Zmat3 as a core component of p53-mediated tumor suppression and identified Cdkn1a as the most potent cooperating p53-induced gene in tumor suppression. We discovered that ZMAT3/CDKN1A serve as near-universal effectors of p53-mediated tumor suppression that regulate cell division, migration, and extracellular matrix organization. Accordingly, combined Zmat3-Cdkn1a inactivation dramatically enhanced cell proliferation and migration compared to controls, akin to p53 inactivation. Together, our findings place ZMAT3 and CDKN1A as hubs of a p53-induced gene program that opposes tumorigenesis across various cellular and genetic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5afaeddf5354538c432a3c29c86495b8e7485c" target='_blank'>
              Integrative multiomic approaches reveal ZMAT3 and p21 as conserved hubs in the p53 tumor suppression network
              </a>
            </td>
          <td>
            A. Boutelle, Aicha R. Mabene, David Yao, Haiqing Xu, Mengxiong Wang, Yuning J. Tang, Steven S. Lopez, Sauradeep Sinha, J. Demeter, Ran Cheng, Brooks A. Benard, Liz J. Valente, Alexandros P. Drainas, Martin Fischer, Ravi Majeti, D. Petrov, Peter K. Jackson, Fan Yang, M. Winslow, Mike C Bassik, L. Attardi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/244d976555710935c004ec15262f6387c6d24c28" target='_blank'>
              Identifying prognostic biomarkers in oral squamous cell carcinoma: an integrated single-cell and bulk RNA sequencing study on mitophagy-related genes
              </a>
            </td>
          <td>
            Minsi Li, Yi Wei, Wenhua Huang, Cen Wang, Shixi He, Shuwen Bi, Shuangyu Hu, Ling You, Xuan-ping Huang
          </td>
          <td>2024-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This study leverages bioinformatics and medical big data to integrate datasets from the Gene Expression Omnibus (GEO) and The Cancer Genome Atlas (TCGA), providing a comprehensive overview of immunogenic cell death (ICD)-related gene expression in colorectal cancer (CRC). The research aims to elucidate the molecular pathways and gene networks associated with ICD in CRC, with a focus on the therapeutic potential of cell death inducers, including ferroptosis agents, and their implications for precision medicine.We conducted differential expression analysis and utilized advanced bioinformatic techniques to analyze ICD-related gene expression in CRC tissues. Unsupervised consensus clustering was applied to categorize CRC patients into distinct ICD-associated subtypes, followed by an in-depth immune microenvironment analysis and single-cell RNA sequencing to investigate immune responses and cell infiltration patterns. Experimental validation was performed to assess the impact of cell death inducers on ICD gene expression and their interaction with ferroptosis inducers in combination with other clinical drugs.Distinct ICD gene expression profiles were identified in CRC tissues, revealing molecular pathways and intricate gene networks. Unsupervised consensus clustering refined the CRC cohort into unique ICD-associated subtypes, each characterized by distinct clinical and immunological features. Immune microenvironment analysis and single-cell RNA sequencing revealed significant variations in immune responses and cell infiltration patterns across these subtypes. Experimental validation confirmed that cell death inducers directly affect ICD gene expression, highlighting their therapeutic potential. Additionally, combinatorial therapies with ferroptosis inducers and clinical drugs were shown to influence drug sensitivity and resistance in CRC.Our findings underscore the importance of ICD-related genes in CRC prognosis and therapeutic targeting. The study provides actionable insights into the efficacy of cell death-inducing therapies, particularly ferroptosis inducers, and their regulatory mechanisms in CRC. These discoveries support the development of precision medicine strategies targeting ICD genes and offer valuable guidance for translating these therapies into clinical practice, with the potential to enhance CRC treatment outcomes and patient survival.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9db03dd2d9c7081d5e57b96873900ea952f94958" target='_blank'>
              Prognosis of colorectal cancer, prognostic index of immunogenic cell death associated genes in response to immunotherapy, and potential therapeutic effects of ferroptosis inducers
              </a>
            </td>
          <td>
            Mengjie Lei, Meihua Xiao, Zhiqing Long, Taolin Lin, Ran Ding, Qi Quan
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2256152e45d11bd82f68a7e3810e940b41cb02" target='_blank'>
              Essential gene screening identifies the bromodomain-containing protein BRPF1 as a new actionable target for endocrine therapy-resistant breast cancers
              </a>
            </td>
          <td>
            A. Salvati, G. Giurato, J. Lamberti, Ilaria Terenzi, Laura Crescenzo, Viola Melone, L. Palo, A. Giordano, Francesco Sabbatino, G. Roscigno, C. Quintavalle, G. Condorelli, Francesca Rizzo, Roberta Tarallo, G. Nassa, Alessandro Weisz
          </td>
          <td>2024-08-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bbe642fe1a5320aff996d83dd54d13a05e4037" target='_blank'>
              The intersection of homologous recombination (HR) and mismatch repair (MMR) pathways in DNA repair-defective tumors
              </a>
            </td>
          <td>
            L. Incorvaia, T. B. Bazan Russo, V. Gristina, A. Perez, C. Brando, Clarissa Mujacic, Emilia Di Giovanni, M. Bono, Silvia Contino, Carla Ferrante Bannera, Maria Concetta Vitale, Andrea Gottardo, M. Peri, A. Galvano, D. Fanale, G. Badalamenti, Antonio Russo, V. Bazan
          </td>
          <td>2024-09-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Background: The fate and functions of RNAs are coordinately regulated by RNA-binding proteins (RBPs), which are often dysregulated in various cancers. Known as a splicing regulator, RNA-binding motif protein 6 (RBM6) harbors tumor-suppressor activity in many cancers; however, there is a lack of research on the molecular targets and regulatory mechanisms of RBM6. Methods: In this study, we constructed an RBM6 knock-down (shRBM6) model in the HeLa cell line to investigate its functions and molecular targets. Then we applied improved RNA immunoprecipitation coupled with sequencing (iRIP-seq) and whole transcriptome sequencing approaches to investigate the potential role and RNA targets of RBM6. Results: Using The Cancer Genome Atlas dataset, we found that higher expression of RBM6 is associated with a better prognosis in many cancer types. In addition, we found that RBM6 knockdown promoted cell proliferation and inhibited apoptosis, demonstrating that RBM6 may act as an anti-oncogenic protein in cancer cells. RBM6 can regulate the alternative splicing (AS) of genes involved in DNA damage response, proliferation, and apoptosis-associated pathways. Meanwhile, RBM6 knockdown activated type I interferon signaling pathways and inhibited the expression of genes involved in the cell cycle, cellular responses to DNA damage, and DNA repair pathways. The differentially expressed genes (DEGs) by shRBM6 and their involved pathways were likely regulated by the transcription factors undergoing aberrant AS by RBM6 knockdown. For iRIP-seq analysis, we found that RBM6 could interact with a large number of mRNAs, with a tendency for binding motifs GGCGAUG and CUCU. RBM6 bound to the mRNA of cell proliferation- and apoptosis-associated genes with dysregulated AS after RBM6 knockdown. Conclusions: In summary, our study highlights the important role of RBM6, as well as the downstream targets and regulated pathways, suggesting the potential regulatory mechanisms of RBM6 in the development of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f429e0c09165b4883518646756d39032c7ca6d4" target='_blank'>
              Global RNA Interaction and Transcriptome Profiles Demonstrate the Potential Anti-Oncogenic Targets and Pathways of RBM6 in HeLa Cells
              </a>
            </td>
          <td>
            Ping Peng, Qingqing Yin, Wei Sun, Jing Han, Hao Guo, Chao Cheng, Dongbo Liu
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Bioscience-Landmark</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Introduction Type B1 and B2 thymomas are lymphocyte-rich malignant tumors with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, non-coding regions remain uncharacterized. Here, we developed a rigorous tumor isolation method from lymphocyte-rich thymoma tissues and identified somatic mutations in non-coding and mitochondrial DNA. Methods CD205+CD45- pure tumor cells were isolated from fresh-frozen tissues using DEPArray system. Deep whole-genome sequencing was performed, and recurrent somatic alterations in coding, non-coding, and mitochondria regions were systemically identified by computational framework. The mutations were classified according to gene function, cis-regulatory element, and mutational signature. Results The total number of somatic mutations was approximately 80 times higher in non-coding regions than in coding regions in type B1-2 thymomas (1,671.3 vs. 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of cases exhibited fewer than four mutations in coding regions. A systematic non-coding analysis identified a total of 405.0 mutations per case on cis-regulatory elements, and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Mitochondrial heteroplasmy was observed in 90% of cases, with a significant proportion of mutations located in D-loop region. The single-base substitution pattern was signature 12. Conclusions Numerous non-coding mutations and mitochondrial heteroplasmy were detected in type B1 and B2 thymomas. Given the paucity of coding mutations observed in this disease entity, disruption of the non-coding landscape and mitochondrial heteroplasmic shift may be the primary cause of thymoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/559b9056aa5e8bd421441f43ad9a55ae5b38dbc9" target='_blank'>
              Genome-wide analysis of somatic non-coding mutation patterns and mitochondrial heteroplasmy in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Juliet King, Emma McLean, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutations commonly found in AML such as DNMT3A, TET2 and ASXL1 can be found in the peripheral blood of otherwise healthy adults a phenomenon referred to as clonal hematopoiesis (CH). These mutations are thought to represent the earliest genetic events in the evolution of AML. Genomic studies on samples acquired at diagnosis, remission, and at relapse have demonstrated significant stability of CH mutations following induction chemotherapy. Meanwhile, later mutations in genes such as NPM1 and FLT3, have been shown to contract at remission and in the case of FLT3 often are absent at relapse. We sought to understand how early CH mutations influence subsequent evolutionary trajectories throughout remission and relapse in response to induction chemotherapy. Here, we assembled a retrospective cohort of patients diagnosed with de novo AML at our institution that underwent genomic sequencing at diagnosis as well as at the time of remission and/or relapse (total n = 182 patients). Corroborating prior studies, FLT3 and NPM1 mutations were generally eliminated at the time of cytologic complete remission but subsequently reemerged upon relapse, whereas DNMT3A, TET2 and ASXL1 mutations often persisted through remission. Early CH-related mutations exhibited distinct constellations of co-occurring genetic alterations, with NPM1 and FLT3 mutations enriched in DNMT3Amut AML, while CBL and SRSF2 mutations were enriched in TET2mut and ASXL1mut AML, respectively. In the case of NPM1 and FLT3 mutations, these differences vanished at the time of complete remission yet readily reemerged upon relapse, indicating the reproducible nature of these genetic interactions. Thus, early CH-associated mutations that precede malignant transformation subsequently shape the evolutionary trajectories of AML through diagnosis, therapy, and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f35657f001405231a2b4d996ed0ac3f0415703" target='_blank'>
              Early drivers of clonal hematopoiesis shape the evolutionary trajectories of de novo acute myeloid leukemia
              </a>
            </td>
          <td>
            Ryan D. Chow, Priya Velu, Safoora Deihimi, Jonathan Belman, Angela Youn, Nisargbhai Shah, Selina M. Luger, Martin P. Carroll, Jennifer J D Morrissette, Robert L Bowman
          </td>
          <td>2024-09-01</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Leukemia is a malignant tumor with high heterogeneity and a complex evolutionary process. It is difficult to resolve the heterogeneity and clonal evolution of leukemia cells by applying traditional bulk sequencing techniques, thus preventing a deep understanding of the mechanisms of leukemia development and the identification of potential therapeutic targets. However, with the development and application of single-cell sequencing technology, it is now possible to investigate the gene expression profile, mutations, and epigenetic features of leukemia at the single-cell level, thus providing a new perspective for leukemia research. In this article, we review the recent applications and advances of single-cell sequencing technology in leukemia research, discuss its potential for enhancing our understanding of the mechanisms of leukemia development, discovering therapeutic targets and personalized treatment, and provide reference guidelines for the significance of this technology in clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059e4fa52cbb0003a82e3f417ae0958def767ace" target='_blank'>
              Application and research progress of single cell sequencing technology in leukemia
              </a>
            </td>
          <td>
            Dan Xie, Bangquan An, Mingyue Yang, Lei Wang, Min Guo, Heng Luo, Shengwen Huang, Fa Sun
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) has a poor survival rate for both pediatric and adult patients due to its frequent relapse. To elucidate the bioenergetic principle underlying AML relapse, we investigated the transcriptional regulation of mitochondrial–nuclear dual genomes responsible for metabolic plasticity in treatment-resistant blasts. Both the gain and loss of function results demonstrated that NFκB2, a noncanonical transcription factor (TF) of the NFκB (nuclear factor kappa-light-chain-enhancer of activated B cells) family, can control the expression of TFAM (mitochondrial transcription factor A), which is known to be essential for metabolic biogenesis. Furthermore, genetic tracking and promoter assays revealed that NFκB2 is in the mitochondria and can bind the specific “TTGGGGGGTG” region of the regulatory D-loop domain to activate the light-strand promoter (LSP) and heavy-strand promoter 1 (HSP1), promoters of the mitochondrial genome. Based on our discovery of NFκB2′s novel function of regulating mitochondrial–nuclear dual genomes, we explored a novel triplet therapy including inhibitors of NFκB2, tyrosine kinase, and mitochondrial ATP synthase that effectively eliminated primary AML blasts with mutations of the FMS-related receptor tyrosine kinase 3 (FLT3) and displayed minimum toxicity to control cells ex vivo. As such, effective treatments for AML must include strong inhibitory actions on the dual genomes mediating metabolic plasticity to improve leukemia prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd91cb6b195c34903e27bb3147d1ba2db984023c" target='_blank'>
              Discovery of NFκB2-Coordinated Dual Regulation of Mitochondrial and Nuclear Genomes Leads to an Effective Therapy for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Yi Xu, David J. Baylink, Jeffrey Xiao, Lily Tran, Vinh Nguyen, Brandon Park, Ismael Valladares, Scott Lee, Kevin Codorniz, Laren D Tan, Chien-Shing Chen, Hisham Abdel-Azim, Mark E. Reeves, Hamid Mirshahidi, Guido Marcucci, H. Cao
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ec02f56417ae90246c83ab25ae98a1c2c7e30" target='_blank'>
              Targeting treatment resistance: unveiling the potential of RNA methylation regulators and TG-101,209 in pan-cancer neoadjuvant therapy
              </a>
            </td>
          <td>
            Yaoyao Zhou, Ziyun Liu, Cheng Gong, Jie Zhang, Jing Zhao, Xia Zhang, Xiangyu Liu, Bin Li, Rui Li, Zhenyu Shi, Yongjie Xie, Li Bao
          </td>
          <td>2024-08-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Upon infection, human papillomavirus (HPV) manipulates host cell gene expression to create an environment that is supportive of a productive and persistent infection. The virus-induced changes to the host cell’s transcriptome are thought to contribute to carcinogenesis. Here, we show by RNA-sequencing that oncogenic HPV18 episome replication in primary human foreskin keratinocytes (HFKs) drives host transcriptional changes that are consistent between multiple HFK donors. We have previously shown that HPV18 episome replication in HFKs results in post-transcriptional stabilisation of the host chromatin insulation protein CTCF. Since CTCF is an important regulator of host cell transcription via coordination of epigenetic boundaries and long-range chromosomal interactions, we hypothesised that HPV18-induced stabilisation of CTCF may contribute to host transcription reprogramming. Analysis of CTCF binding in the host cell genome by ChIP-Seq revealed that while the total number of CTCF binding sites is not altered by the virus, there are a sub-set of CTCF binding sites that are either enriched or depleted of CTCF. Many of these altered sites are clustered within regulatory elements of differentially expressed genes, including the tumour suppressor gene cell adhesion molecule 1 (CADM1), which supresses epithelial cell growth and invasion. We show that HPV18 establishment results in reduced CTCF binding at the CADM1 promoter and upstream enhancer. Loss of CTCF binding is coincident with epigenetic repression of CADM1, in the absence of CpG hypermethylation, while adjacent genes including the transcriptional regulator ZBTB16 are activated. These data indicate that the CADM1 locus is subject to topological rearrangement following HPV18 establishment. We tested this hypothesis using 4C-Seq (circular chromosome confirmation capture-sequencing) and show that HPV18 establishment causes a loss of long-range chromosomal interactions between the CADM1 transcriptional start site and the upstream transcriptional enhancer. These data show that HPV18 manipulates host cell promoter-enhancer interactions to drive transcriptional reprogramming that may contribute to HPV-induced disease progression. AUTHOR SUMMARY Infection with oncogenic HPV is the cause of numerous cancer types, which generally arise after persistent HPV infection. Upon infection, HPV alters the gene expression profile of infected cells to facilitate virus replication and persistence. Multiple mechanisms of HPV-induced host cell reprogramming have been previously suggested. Here, we show that HPV infection induces rearrangement of specific genomic loci by altering the chromatin binding of the host cell protein CTCF, an important regulator of chromatin architecture. Loss of CTCF binding to a cluster of binding sites at the CADM1 locus on chromosome 11 is coincident with epigenetic reprogramming and disruption of long-range chromatin interactions, resulting in transcriptional repression of CADM1. Our data show that repression of CADM1 is an early event in HPV-driven disease, preceding hypermethylation of the CADM1 transcriptional promoter that is frequently observed in HPV-driven cancers, demonstrating a novel mechanism of HPV-induced host cell transcriptional reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296755d464d45666368bedcd29a51e084375e664" target='_blank'>
              Repression of CADM1 transcription by HPV type 18 is mediated by three-dimensional rearrangement of promoter-enhancer interactions
              </a>
            </td>
          <td>
            Karen Campos-León, J. Ferguson, Thomas Günther, C. D. Wood, S. Wingett, Selin Pekel, C. S. Varghese, C. Várnai, Michelle West, Andrew Beggs, A. Grundhoff, B. Noyvert, Sally Roberts, Joanna L. Parish
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Richter's transformation (RT) is defined as the evolution of chronic lymphocytic leukemia (CLL) or small lymphocytic lymphoma (SLL) into an aggressive lymphoma, most commonly diffuse large B-cell lymphoma. This complication is rare and aggressive, with poor prognosis and dismal survival. Clonal relationship with the underlying CLL/SLL, observed in ∼80% of cases, represents one of the main factors affecting prognosis. Treatment has been historically based on chemoimmunotherapy, but frequent mutations in genes involved in cell survival and proliferation-such as TP53, NOTCH1, MYC, CDKN2A-confer resistance to standard treatments. During the last years, advances in the knowledge of the biological mechanisms underlying RT allowed to identify genetic and molecular lesions that can potentially be targeted by novel selective agents. Pathway and checkpoint inhibitors, bispecific antibodies and CAR T-cell therapy are currently under investigation and represent promising treatment options. This review summarizes current biological evidence and available data on novel therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a1bac15f3b796b2ef346e32ef1ee6b7161f4b12" target='_blank'>
              Advances in the understanding of molecular genetics and therapy of Richter transformation in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            M. Deodato, A. Frustaci, Arianna Zappaterra, Alberto Rapella, C. Gambacorti-Passerini, R. Cairoli, Marco Montillo, Alessandra Tedeschi
          </td>
          <td>2024-09-02</td>
          <td>Leukemia & lymphoma</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most common malignant tumours. Identification of new effective drug targets for CRC and exploration of bioactive small-molecules are clinically urgent. The human dCTP pyrophosphatase 1 (DCTPP1) is a newly identified pyrophosphatase regulating the cellular nucleotide pool but remains unexplored as potential target for CRC treatment. Here, twelve unprecedented chemical architectures terpene-nonadride heterodimers (1-12) and their monomers (13-20) were isolated from endophyte Bipolaris victoriae S27. Compounds 1-12 represented the first example of terpene-nonadride heterodimers, in which nonadride monomers of 1 and 2 were also first example of 5/6 bicyclic nonadrides. A series of assays showed that 2 could repress proliferation and induce cell cycle arrest, apoptotic and autophagic CRC cell death in vitro and in vivo. Clinical cancer samples data revealed that DCTPP1 was a novel target associated with poor survival in CRC. DCTPP1 was also identified as a new target protein of 2. Mechanistically, compound 2 bound to DCTPP1, inhibited its enzymatic activity, intervened with amino acid metabolic reprogramming, and exerted anti-CRC activity. Our study demonstrates that DCTPP1 was a novel potential biomarker and therapeutic target in CRC, and 2 was the first natural anti-CRC drug candidate targeting DCTPP1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc9fd435f5c840a052cb7ac9a9e3b5ba2d4e289b" target='_blank'>
              Identification of Novel Target DCTPP1 for Colorectal Cancer Therapy with the Natural Small-molecule Inhibitors Regulating Metabolic Reprogramming.
              </a>
            </td>
          <td>
            Li Feng, Xinjia Wang, Xinrui Guo, Li Shi, Shihuang Su, Xinjing Li, Jia Wang, Ninghua Tan, Yi Ma, Zhe Wang
          </td>
          <td>2024-08-14</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Acute myeloid leukemia with complex karyotype (ckAML) is characterized by high genomic complexity, including frequent TP53 mutations and chromothripsis. We hypothesized that the numerous genomic rearrangements could reposition active enhancers near proto-oncogenes, leading to their aberrant expression. We developed pyjacker, a computational tool for the detection of enhancer hijacking events, and applied it to a cohort of 39 ckAML samples. Pyjacker identified motor neuron and pancreas homeobox 1 (MNX1), a gene aberrantly expressed in 1.4% of AML patients, often as a result of del(7)(q22q36) associated with hijacking of a CDK6 enhancer. MNX1-activated cases show significant co-occurrence with BCOR mutations and a gene signature shared with t(7;12)(q36;p13) pediatric AML. We demonstrated that MNX1 is a dependency gene, as its knockdown in a xenograft model reduces leukemia cell fitness. In conclusion, enhancer hijacking is a frequent mechanism for oncogene activation in AML. Statement of significance This study examines the consequences of structural alterations and demonstrates that proto-oncogene activation by enhancer hijacking is an overlooked pathomechanism in AML. MNX1 overexpression demonstrates that deletions on chromosome 7q can not only lead to haploinsufficiency, but also to activation of oncogenes by enhancer hijacking, providing a novel leukemogenic mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb3e7eaaf436e861083291af3f4d69f9c709309" target='_blank'>
              Pyjacker identifies enhancer hijacking events in acute myeloid leukemia including MNX1 activation via deletion 7q
              </a>
            </td>
          <td>
            Etienne Sollier, Anna Riedel, U. Toprak, Justyna A. Wierzbinska, D. Weichenhan, J. P. Schmid, M. Hakobyan, A. Touzart, E. Jahn, B. Vick, Fiona Brown-Burke, Katherine Kelly, Simge Kelekçi, Anastasija Pejkovska, Ashish Goyal, Marion Bähr, Kersten Breuer, Mei-Ju May Chen, Maria Llamazares-Prada, M. Hartmann, M. Schönung, Nádia Correia, A. Trumpp, Yomn Abdullah, Ursula Klingmüller, S. S. Mughal, B. Brors, Frank Westermann, Matthias Schlesner, S. Vosberg, T. Herold, P. Greif, Dietmar Pfeifer, Michael Lübbert, Thomas Fischer, F. H. Heidel, C. Gebhard, W. Walter, Torsten Haferlach, Ann-Kathrin Eisfeld, K. Mrózek, D. Nicolet, L. Bullinger, L. Smeenk, Claudia Erpelinck, Roger Mulet-Lazaro, Ruud Delwel, Aurélie Ernst, Michael Scherer, P. Lutsik, I. Jeremias, Konstanze Döhner, H. Döhner, D. Lipka, C. Plass
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="The fusion expression of DNA replication-related proteins with nucleotide deaminase enzymes promotes random mutations in bacterial genomes, thereby increasing genetic diversity among population. Most previous studies have focused on cytosine deaminase, which produces only C→T mutations, significantly limiting the variety of mutation types. In this study, we developed a fusion expression system by combining DnaG (RNA primase) with adenine deaminase TadA-8e (DnaG-TadA) in Escherichia coli, which is capable of rapidly introducing A→G mutations into the E. coli genome, resulting in a 664-fold increase in terms of mutation rate. Additionally, we engineered a dual-functional TadA variant, TadAD, and then fused it with DnaG. This construct introduced both C→T and A→G mutations into the E. coli genome, with the mutation rate further increased by 370-fold upon co-expression with an uracil glycosylase inhibitor (DnaG-TadAD-UGI). We applied DnaG-TadA and DnaG-TadAD-UGI systems to the adaptive laboratory evolution for Cd2+ and kanamycin resistance, achieving an 8.0 mM Cd2+ and 200 μg/mL kanamycin tolerance within just 17 days and 132 hours, respectively. Compared to conventional evolution methods, the final tolerance levels were increased by 320% and 266%, respectively. Our work offers a novel strategy for random mutagenesis in E. coli and potentially other prokaryotic species. For TOC only">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8dc3e641872269a76809a2a9f58812a1c2451c2" target='_blank'>
              Genome-wide A→G and C→T Mutations Induced by Functional TadA Variants in Escherichia coli
              </a>
            </td>
          <td>
            Hao Wang, Zhengxin Dong, Jingyi Shi, Lei Chen, Tao Sun, Weiwen Zhang
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cell atlas projects have nominated recurrent transcriptional states as drivers of biological processes and disease, but their origins, regulation, and properties remain unclear. To enable complementary functional studies, we developed a scalable approach for recapitulating cell states in vitro using CRISPR activation (CRISPRa) Perturb-seq. Aided by a novel multiplexing method, we activated 1,836 transcription factors in two cell types. Measuring 21,958 perturbations showed that CRISPRa activated targets within physiological ranges, that epigenetic features predicted activatable genes, and that the protospacer seed region drove an off-target effect. Perturbations recapitulated in vivo fibroblast states, including universal and inflammatory states, and identified KLF4 and KLF5 as key regulators of the universal state. Inducing the universal state suppressed disease-associated states, highlighting its therapeutic potential. Our findings cement CRISPRa as a tool for perturbing differentiated cells and indicate that in vivo states can be elicited via perturbation, enabling studies of clinically relevant states ex vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec1895f5c09e2fea09dd8630472743d3965d841b" target='_blank'>
              Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states
              </a>
            </td>
          <td>
            Kaden M. Southard, Rico C. Ardy, Anran Tang, Deirdre D. O’Sullivan, Eli Metzner, Karthik Guruvayurappan, Thomas M. Norman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Combined CDK4/6 inhibitor (CDK4/6i) and endocrine therapy significantly improves outcome for patients with estrogen receptor-positive (ER+) metastatic breast cancer, but drug resistance and thus disease progression inevitably occur. Herein, we aimed to identify genomic alterations associated with combined CDK4/6i and endocrine therapy resistance, and follow the levels of specific mutations in longitudinal circulating tumor DNA (ctDNA) for early detection of progression. From a cohort of 86 patients with ER+ metastatic breast cancer we performed whole exome sequencing or targeted sequencing of paired tumor (N = 8) or blood samples (N = 5) obtained before initiation of combined CDK4/6i and endocrine therapy and at disease progression. Mutations in oncogenic genes at progression were rare, while amplifications of growth-regulating genes were more frequent. The most frequently acquired alterations observed were PIK3CA and TP53 mutations and PDK1 amplification. Longitudinal ctDNA dynamics of mutant PIK3CA or private mutations revealed increased mutation levels at progression in 8 of 10 patients (80%). Impressively, rising levels of PIK3CA-mutated ctDNA were detected 4-17 months before imaging. Our data add to the growing evidence supporting longitudinal ctDNA analysis for real-time monitoring of CDK4/6i response and early detection of progression in advanced breast cancer. Further, our analysis suggests that amplification of growth-related genes may contribute to combined CDK4/6i and endocrine therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4f8f900b692158e74287adf0a693a5a4d08296" target='_blank'>
              Genomic alterations associated with resistance and circulating tumor DNA dynamics for early detection of progression on CDK4/6 inhibitor in advanced breast cancer.
              </a>
            </td>
          <td>
            Charlotte K Kindt, Carla L. Alves, S. Ehmsen, Amalie Kragh, T. Reinert, Marianne Vogsen, A. R. Kodahl, Jeanette D Rønlev, Dilan Ardik, Anna L Sørensen, Kirstine Evald, Mia L Clemmensen, J. Staaf, Henrik J Ditzel
          </td>
          <td>2024-08-11</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Translocation renal cell carcinoma (tRCC) is an aggressive subtype of kidney cancer driven by TFE3 gene fusions, which act via poorly characterized downstream mechanisms. Here we report that TFE3 fusions transcriptionally rewire tRCCs toward oxidative phosphorylation (OXPHOS), contrasting with the highly glycolytic metabolism of most other renal cancers. This TFE3 fusion-driven OXPHOS program, together with heightened glutathione levels found in renal cancers, renders tRCCs sensitive to reductive stress – a metabolic stress state induced by an imbalance of reducing equivalents. Genome-scale CRISPR screening identifies tRCC-selective vulnerabilities linked to this metabolic state, including EGLN1, which hydroxylates HIF-1α and targets it for proteolysis. Inhibition of EGLN1 compromises tRCC cell growth by stabilizing HIF-1a and promoting metabolic reprogramming away from OXPHOS, thus representing a vulnerability to OXPHOS-dependent tRCC cells. Our study defines a distinctive tRCC-essential metabolic program driven by TFE3 fusions and nominates EGLN1 inhibition as a therapeutic strategy to counteract fusion-induced metabolic rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2568e01eb416e6b7b22ba26a8e1971911ede6d44" target='_blank'>
              TFE3 fusions direct an oncogenic transcriptional program that drives OXPHOS and unveils vulnerabilities in translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Jiao Li, Kaimeng Huang, Fiona McBride, Ananthan Sadagopan, Daniel S. Gallant, Meha Thakur, Prateek Khanna, Bingchen Li, Maolin Ge, Cary N. Weiss, M. Achom, Qingru Xu, Kun Huang, Birgitta A. Ryback, Miao Gui, L. Bar-Peled, S. Viswanathan
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>1</td>
          <td>62</td>
        </tr>

        <tr id="Single-cell genomic analysis enables researchers to gain novel insights across diverse research areas, including developmental biology, tumor heterogeneity, and disease pathogenesis. Conducting single-cell genomic analysis using next-generation sequencing (NGS) methods has traditionally been challenging as the amount of genomic DNA present in a single cell is limited. Advancements in multiple displacement amplification (MDA) technologies allow the unbiased amplification of limited quantities of DNA under conditions that maintain its integrity. This method of amplification results in high yield and facilitates the generation of high-complexity NGS libraries that ensure the highest coverage to effectively allow variant calling. With the introduction of new sequencing platforms and chemistry, whole genome sequencing became a more cost-effective application, but enrichment of specific regions of interest further reduces the amount of required sequencing output and associated costs. There are two enrichment methods, polymerase chain reaction (PCR)-based and hybrid-capture-based methods. PCR-based methods are very flexible and highly effective but focus on specific loci, typically known to be associated with disease. Inherited diseases of unknown genetic origin require a more comprehensive approach to capture the genetic variation that is not yet associated with a specific disease. Hybrid capture enrichment methods require considerable amounts of DNA such that exome enrichment from single cells is only possible after preamplification of this limited material. This article describes the complete workflow from single cells and small quantities of DNA to exome-NGS libraries for Illumina sequencing instruments and includes the following protocols: © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Whole genome amplification from single cells or small amounts of gDNA Basic Protocol 2: NGS library generation of MDA-amplified material Basic Protocol 3: Exome enrichment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891307a5d438a6375e34496a127912e0af6f10c1" target='_blank'>
              Exome Sequencing Starting from Single Cells.
              </a>
            </td>
          <td>
            Ioanna Andreou, Markus Storbeck, Peter Hahn, Samuel J. Rulli, Eric Lader
          </td>
          <td>2024-09-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 INTRODUCTION Utilizing whole genome (WGS) and transcriptome sequencing (RNA-seq) to identify the molecular features of individual cancers sits at the heart of pediatric cancer precision medicine. Whilst the utility of WGS for mutation detection is well established, most precision medicine programs utilize RNA-seq for fusion detection, which underutilizes the potential of RNA-seq information. In ZERO (Australia’s national precision medicine program for children with high-risk cancer), we showed that combining WGS with RNA-seq identified the molecular driver of their cancer in 94% of patients (n=247). We have sought to measure the added utility integrating a comprehensive RNA-seq pipeline provides in a precision medicine program (n=477). METHODS We developed a comprehensive RNA-seq pipeline (Carbonite) to identify expressed single nucleotide variants (SNV), small insertion and deletions (InDels) and structural variant (SV) events. SVs extend beyond fusions to include intragenic deletions, duplications, inversions and insertions, previously only identifiable through WGS. Fusion detection was expanded beyond gold-standard algorithms by incorporating a reference-free WGS guided approach that identifies cryptic SVs and intragenic alternate splicing events. The application of RNA-seq measured gene abundance was applied to identify enhancer hijacking events and biallelic deletions resulting from an SV. SNV and InDel analysis reports the zygosity in comparison to germline and tumor DNA. RESULTS A total of 218 SVs and 537 SNVs/InDels were identified as either pathogenic or likely pathogenic in our cohort of 477 patients. Carbonite identified 96% of SVs in comparison to 54% identified using gold-standard fusion detection algorithms only. In addition, Carbonite resolved the expressed chimeric transcript from complex chromosome shattering events, multi-hop complex rearrangements and SVs occurring in homologous and difficult to sequence regions in 33 patients. This information also aided the functional interpretation of SVs through accurate frame calling of potential activating oncogenic SV events, and disruption of key tumor suppressor genes. Of the driver SNV/InDels identified in WGS, Carbonite corroborated 92%. Those not validated resulted in either non-sense mediated decay or the gene was not expressed. Importantly, in 22% of somatic SNV/InDels, RNA-seq altered the assessment of pathogenicity, principally through resolving allele specific expression (ASE) and confirmation of the transcriptional consequence of splice site mutations. Germline SNVs/InDels exhibited ASE in the tumor RNA in 48%, aiding in interpretation of penetrance and oncogenicity. CONCLUSIONS Our results show that the enhanced RNA-seq pipeline has clinical applicability in precision medicine beyond fusion detection, providing functional insights derived from the transcriptional consequences of SVs and SNVs/InDels. We propose that RNA-seq is an essential tool in interpretation of the pathogenicity prediction of molecular drivers of cancer.
 Citation Format: Chelsea Mayoh, Paulette Barahona, Angela Lin, Lujing Cui, Pamela Ajuyah, Ann Altekoester, Loretta MS Lau, Dong-Anh Khuong-Quang, Vanessa Tyrrell, Michelle Haber, Marie Wong-Erasmus, Paul G. Ekert, Mark J. Cowley. Increasing the clinical utility of transcriptome analysis in high-risk childhood precision oncology [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B014.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f46d1e40d467fbbfb28ea4fdb28248d01c9dacb2" target='_blank'>
              Abstract B014: Increasing the clinical utility of transcriptome analysis in high-risk childhood precision oncology
              </a>
            </td>
          <td>
            C. Mayoh, P. Barahona, Angela Lin, Lujing Cui, P. Ajuyah, Ann-Kristin Altekoester, Loretta M S Lau, Dong-Anh Khuong-Quang, V. Tyrrell, Michelle Haber, Marie Wong-Erasmus, Paul G. Ekert, Mark J. Cowley
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The DNA damage repair (DDR) pathway is a complex signaling cascade that can sense DNA damage and trigger cellular responses to DNA damage to maintain genome stability and integrity. A typical hallmark of cancer is genomic instability or nonintegrity, which is closely related to the accumulation of DNA damage within cancer cells. The treatment principles of radiotherapy and chemotherapy for cancer are based on their cytotoxic effects on DNA damage, which are accompanied by severe and unnecessary side effects on normal tissues, including dysregulation of the DDR and induced therapeutic tolerance. As a driving factor for oncogenes or tumor suppressor genes, noncoding RNA (ncRNA) have been shown to play an important role in cancer cell resistance to radiotherapy and chemotherapy. Recently, it has been found that ncRNA can regulate tumor treatment tolerance by altering the DDR induced by radiotherapy or chemotherapy in cancer cells, indicating that ncRNA are potential regulatory factors targeting the DDR to reverse tumor treatment tolerance. This review provides an overview of the basic information and functions of the DDR and ncRNAs in the tolerance or sensitivity of tumors to chemotherapy and radiation therapy. We focused on the impact of ncRNA (mainly microRNA [miRNA], long noncoding RNA [lncRNA], and circular RNA [circRNA]) on cancer treatment by regulating the DDR and the underlying molecular mechanisms of their effects. These findings provide a theoretical basis and new insights for tumor-targeted therapy and the development of novel drugs targeting the DDR or ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63af0c8bea30d510199d8c53e8bb1fd81c1002b8" target='_blank'>
              DNA damage response-related ncRNAs as regulators of therapy resistance in cancer
              </a>
            </td>
          <td>
            Ziru Gao, Xinchi Luan, Xuezhe Wang, Tianyue Han, Xiaoyuan Li, Zeyang Li, Peifeng Li, Zhixia Zhou
          </td>
          <td>2024-08-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="PURPOSE
High-grade complex karyotype sarcomas are a heterogeneous group of tumors with a uniformly poor prognosis. Within complex karyotype sarcomas, there are innumerable genetic changes but identifying those that are clinically relevant has been challenging.


EXPERIMENTAL DESIGN
To address this, we utilized a pooled genetic screening approach, informed by TCGA data, to identify key drivers and modifiers of sarcoma development that were validated in vivo.


RESULTS
YAP1 and wildtype KRAS were validated as drivers and transformed human mesenchymal stem cells into two distinct sarcoma subtypes, undifferentiated pleomorphic sarcoma (UPS) and myxofibrosarcoma (MFS), respectively. A subset of tumors driven by CDK4 and PIK3CA reflected leiomyosarcoma (LMS) and osteosarcoma (OS) demonstrating the plasticity of this approach and the potential to investigate sarcoma subtype heterogeneity. All generated tumors histologically reflected human sarcomas and had increased aneuploidy as compared to simple karyotype sarcomas. Comparing differential gene expression of TCGA samples to model data identified increased oxidative phosphorylation signaling in YAP1 tumors. Treatment of a panel of soft tissue sarcomas with a combination of YAP1 and oxidative phosphorylation inhibitors led to significantly decreased viability.


CONCLUSIONS
Transcriptional co-analysis of TCGA patient samples to YAP1 and KRAS model tumors support that these sarcoma subtypes lie along a spectrum of disease and adds guidance for further transcriptome-based refinement of sarcoma subtyping. This approach can be used to begin to understand pathways and mechanisms driving human sarcoma development, the relationship between sarcoma subtypes and to identify and validate new therapeutic vulnerabilities for this aggressive and heterogeneous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdaada75e0a2b742920f7cff889cd677616ee1d5" target='_blank'>
              Genetic Screen in a Pre-Clinical Model of Sarcoma Development Defines Drivers and Therapeutic Vulnerabilities.
              </a>
            </td>
          <td>
            Jack Freeland, Maria Muñoz, Edmond O'Donnell, J. Langerman, Morgan Darrow, Jessica Bergonio, J. Suarez-Navarro, Steven Thorpe, Robert Canter, R. L. Randall, Kathrin Plath, Kermit L Carraway, Owen N. Witte, Thomas G. Graeber, Janai R. Carr-Ascher
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Most head and neck squamous cell carcinoma (HNSCC) patients are diagnosed at an advanced local stage. While immunotherapy has improved survival rates, only a minority of patients respond durably to targeted immunotherapies, posing substantial clinical challenges. We investigated the heterogeneity of the tumor microenvironment in HNSCC cohorts before and after immunotherapy by analyzing single-cell RNA sequencing (scRNA-seq) data and bulk RNA sequencing datasets retrieved from public databases. Methods We constructed a single-cell transcriptome landscape of HNSCC patients before and after immunotherapy and analyzed the cellular composition, developmental trajectories, gene regulatory networks, and communication patterns of different cell type subpopulations. Additionally, we assessed the expression levels of relevant indicators in HNSCC cells via western blot, ELISA, and fluorescent probe techniques. Results At the single-cell level, we identified a subpopulation of TP63+ SLC7A5+ HNSCC that exhibited a ferroptosis-resistant phenotype. This subpopulation suppresses ferroptosis in malignant cells through the transcriptional upregulation of SLC7A5 mediated by high TP63 expression, thereby promoting tumor growth and resistance to immunotherapy. The experimental results demonstrated that the overexpression of TP63 upregulated the expression of SLC7A5 and suppressed the concentrations of Fe2+ and ROS in HNSCC cells. By integrating bulk transcriptome data, we developed a clinical scoring model based on TP63 and SLC7A5, which are closely associated with tumor stage, revealing the significant prognostic efficacy of the TP63+ SLC7A5+ HNSCC-mediated ferroptosis mechanism in HNSCC patients. Conclusion Our research elucidates the TME in HNSCC before and after immunotherapy, revealing a novel mechanism by which TP63+ SLC7A5+ HNSCC inhibits ferroptosis and enhances tumor resistance via TP63-induced SLC7A5 upregulation. These insights lay the foundation for the development of more effective treatments for HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea63ddf8e02932d2e36f67e9aabda84b6c8ce55c" target='_blank'>
              TP63 transcriptionally regulates SLC7A5 to suppress ferroptosis in head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Zilong Chen, Haoxi Cai, Weiwei Ye, Junming Wu, Jing Liu, Yun Xie, Shiqiang Feng, Yuanpei Jin, Yunxia Lv, Hui Ye, Chengfu Cai, Gengming Cai
          </td>
          <td>2024-08-21</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Mycosis fungoides (MF) is the most common subtype of cutaneous T-cell lymphoma (CTCL). Comprehensive analysis of MF cells in situ and ex vivo is complicated by the fact that is challenging to distinguish malignant from reactive T cells with certainty. Methods To overcome this limitation, we performed combined single-cell RNA (scRNAseq) and T-cell receptor TCR sequencing (scTCRseq) of skin lesions of cutaneous MF lesions from 12 patients. A sufficient quantity of living T cells was obtained from 9 patients, but 2 had to be excluded due to unclear diagnoses (coexisting CLL or revision to a fixed toxic drug eruption). Results From the remaining patients we established single-cell mRNA expression profiles and the corresponding TCR repertoire of 18,630 T cells. TCR clonality unequivocally identified 13,592 malignant T cells. Reactive T cells of all patients clustered together, while malignant cells of each patient formed a unique cluster expressing genes typical of naive/memory, such as CD27, CCR7 and IL7R, or cytotoxic T cells, e.g., GZMA, NKG7 and GNLY. Genes encoding classic CTCL markers were not detected in all clusters, consistent with the fact that mRNA expression does not correlate linearly with protein expression. Nevertheless, we successfully pinpointed distinctive gene signatures differentiating reactive malignant from malignant T cells: keratins (KRT81, KRT86), galectins (LGALS1, LGALS3) and S100 genes (S100A4, S100A6) being overexpressed in malignant cells. Conclusions Combined scRNAseq and scTCRseq not only allows unambiguous identification of MF cells, but also revealed marked heterogeneity between and within patients with unexpected functional phenotypes. While the correlation between mRNA and protein abundance was limited with respect to established MF markers, we were able to identify a single-cell gene expression signature that distinguishes malignant from reactive T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/212bc68159cf864c0e23d5af31516d6565474ee3" target='_blank'>
              Single-cell RNA and T-cell receptor sequencing unveil mycosis fungoides heterogeneity and a possible gene signature
              </a>
            </td>
          <td>
            Nalini Srinivas, Lukas Peiffer, K. Horny, Kuan Cheok Lei, T. B. Buus, Linda Kubat, Meng Luo, Menghong Yin, Ivelina Spassova, A. Sucker, Farnoush Farahpour, Jan Kehrmann, Selma Ugurel, E. Livingstone, T. Gambichler, Niels Ødum, Jürgen C. Becker
          </td>
          <td>2024-08-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Of the more than 100 types of brain cancer, glioblastoma (GBM) is the deadliest. As GBM stem cells (GSCs) are considered to be responsible for therapeutic resistance and tumor recurrence, effective targeting and elimination of GSCs could hold promise for preventing GBM recurrence and achieving potential cures. We show here that SUV39H1, which encodes a histone-3, lysine-9 methyltransferase, plays a critical role in GSC maintenance and GBM progression. Upregulation of SUV39H1 was observed in GBM samples compared to normal brain tissues, and knockdown of SUV39H1 in patient-derived GSCs impaired their proliferation and stemness. Single-cell RNA-seq analysis demonstrated restricted expression of SUV39H1 is in GSCs relative to non-stem GBM cells, likely due to super-enhancer-mediated transcriptional activation, while whole cell RNA-seq analysis revealed that SUV39H1 regulates G2/M cell cycle progression, stem cell maintenance, and cell death pathways in GSCs. By integrating the RNA-seq data with ATAC-seq (assay for transposase-accessible chromatin followed by sequencing), we further demonstrated altered chromatin accessibility in key genes associated with these pathways following SUV39H1 knockdown. Treatment with chaetocin, a SUV39H1 inhibitor, mimicked the functional effects of SUV39H1 knockdown in GSCs and sensitized GSCs to the GBM chemotherapy drug temozolomide. Furthermore, targeting SUV39H1 in vivo using a patient-derived xenograft model for GBM inhibited GSC-driven tumor formation. This is the first report demonstrating a critical role for SUV39H1 in GSC maintenance. SUV39H1-mediated targeting of GSCs could enhance the efficacy of existing chemotherapy, presenting a promising strategy for improving GBM treatment and patient outcomes. Highlights SUV39H1 is upregulated in GBM, especially GSCs Targeting SUV39H1 disrupts GSC maintenance and sensitizes GSCs to TMZ Targeting SUV39H1 alters chromatin accessibility at cell cycle and stemness genes Targeting SUV39H1 suppresses GSC-driven tumors in a patient-derived xenograft model">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f51a1ca26253028a59eb7d526ac8f76a7e04a6" target='_blank'>
              SUV39H1 Preserves Cancer Stem Cell Chromatin State and Properties in Glioblastoma
              </a>
            </td>
          <td>
            Chunying Li, Qiqi Xie, Sugata Ghosh, Bihui Cao, Yuanning Du, G. V. Vo, Timothy Y. Huang, Charles Spruck, Y. A. Wang, K. Nephew, Jia Shen
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Abstract Acute myeloid leukemia (AML) is an aggressive hematological malignancy with a heterogeneous molecular landscape. In the pediatric context, the NUP98 gene is a frequent target of chromosomal rearrangements that are linked to poor prognosis and unfavorable treatment outcomes in different AML subtypes. The translocations fuse NUP98 to a diverse array of partner genes, resulting in fusion proteins with novel functions. NUP98 fusion oncoproteins induce aberrant biomolecular condensation, abnormal gene expression programs, and re‐wired protein interactions which ultimately cause alterations in the cell cycle and changes in cellular structures, all of which contribute to leukemia development. The extent of these effects is steered by the functional domains of the fusion partners and the influence of concomitant somatic mutations. In this review, we discuss the complex characteristics of NUP98 fusion proteins and potential novel therapeutic approaches for NUP98 fusion‐driven AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c58bea08a706bedb5af088f9b2cdc67d8db6f7ce" target='_blank'>
              NUP98 oncofusions in myeloid malignancies: An update on molecular mechanisms and therapeutic opportunities
              </a>
            </td>
          <td>
            Milad Rasouli, Selina Troester, Florian Grebien, B. Goemans, C. M. Zwaan, Olaf Heidenreich
          </td>
          <td>2024-09-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tracing cell lineages has become a valuable tool for studying biological processes. Among the available tools for human data, mitochondria DNA (mtDNA) has a high potential due to its ability to be used in conjunction with single-cell chromatin accessibility data, giving access to the cell phenotype. Nonetheless, the existing mutation calling tools are ill-equipped to deal with the polyploid nature of the mtDNA and lack a robust statistical framework. Here we introduce scMitoMut, an innovative R package that leverages statistical methodologies to accurately identify mitochondrial lineage related mutations at the single-cell level. scMitoMut assigns a mutation quality q-value based on beta-binomial distribution to each mutation at each locus within individual cells, ensuring higher sensitivity and precision of lineage related mutation calling in comparison to current methodologies. We tested scMitoMut using single-cell DNA sequencing, scATAC sequencing and 10× Genomics single cell multiome datasets. Using a single-cell DNA sequencing dataset from a mixed population of cell lines, scMitoMut demonstrated superior sensitivity in identifying small proportion of cancer cell lines compared to existing methods. In a human colorectal cancer scATAC dataset, scMitoMut identified more mutations than state-of-the-art methods. Applied to 10× Genomics multiome datasets, scMitoMut effectively measured the lineage distance in cells from blood or brain tissues. Thus, the scMitoMut is a free available (https://www.bioconductor.org/packages/devel/bioc/html/scMitoMut.html.), well-engineered toolkit for mtDNA mutation calling with high memory and CPU efficiency. Consequently, it will significantly advance the application of single-cell sequencing, facilitating the precise delineation of mitochondrial mutations for lineage tracing purposes in development, tumor and stem cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a057cdbd6d7399eed5dec23b92e83e85fe7d0c9" target='_blank'>
              scMitoMut for calling mitochondrial lineage–related mutations in single cells
              </a>
            </td>
          <td>
            Wenjie Sun, Daphne van Ginneken, Leïla Perié
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e335090c3e9a1ddfa4a9c87a808106af39d43ca" target='_blank'>
              Gene regulatory networks reveal sex difference in lung adenocarcinoma
              </a>
            </td>
          <td>
            Enakshi Saha, Marouen Ben Guebila, Viola Fanfani, Jonas Fischer, K.H. Shutta, Panagiotis Mandros, Dawn L. DeMeo, John Quackenbush, C. Lopes-Ramos
          </td>
          <td>2024-08-06</td>
          <td>Biology of Sex Differences</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive and lethal form of brain cancer with few effective treatments. In this context, Zika virus has emerged as a promising therapeutic agent due to its ability to selectively infect and kill GBM cells. To elucidate these mechanisms and expand the landscape of oncolytic virotherapy, we pursued a transcriptomic meta-analysis comparing the molecular signatures of Zika infection in GBM and neuroblastoma (NBM). Over-representation analysis of dysregulated coding genes showed significant enrichment of tumor necrosis factor (TNF), NF-κB, and p53 signaling pathways. A refined list of long non-coding RNAs consistently dysregulated in Zika-infected GBMs was also developed. Functional review of these candidates revealed their potential regulatory role in Zika-mediated oncolysis. We performed validation of the less-researched targets in adult and pediatric GBM cell lines and found significant differential regulation, as predicted. Altogether, our results provide novel insights into the molecular mechanisms underlying the effect of Zika on GBM. We highlight potential therapeutic targets that could be further interrogated to improve the efficacy of tumor cell death and the utility of Zika as an adjuvant virotherapy for GBM and other related cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5fb2f82f19cb82c54f2e078dc5fd330417c5a7" target='_blank'>
              Transcriptomic Meta-analysis Identifies Long Non-Coding RNAs Mediating Zika’s Oncolytic Impact in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Youssef A. Kousa, Shriya Singh, Allison Horvath, Federica Tomasso, Javad Nazarian, Lisa Henderson, Tamer A. Mansour
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6f6c8a044b42cc74a45de30bf8885e624a6a702" target='_blank'>
              Single-cell transcriptional atlas of tumor-associated macrophages in breast cancer
              </a>
            </td>
          <td>
            Yupeng Zhang, Fan Zhong, Lei Liu
          </td>
          <td>2024-09-04</td>
          <td>Breast Cancer Research : BCR</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ab20abff3d5ab2e3023b99dd1dd6ab980ad6081" target='_blank'>
              Mutual exclusivity and co-occurrence patterns of immune checkpoints indicate NKG2A relates to anti-PD-1 resistance in gastric cancer
              </a>
            </td>
          <td>
            Guanjun Li, Xiaohan Liu, Chunhui Gu, Guichuang Ma, Shaowei Li, Zhenfeng Ma, Yanqiu Xiong, Yu Jiang, Qiong Huang, Jianhua Wu, Zhenzhen Wu, Wangjun Liao, Qijing Wu, Min Shi
          </td>
          <td>2024-08-03</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="Objectives: Neuroblastoma (NB), a pediatric malignancy of the peripheral nervous system, is characterized by epigenetic and transcriptional (EP-TF) anomalies. This study aimed to develop an EP-TF clinical prognostic model for NB using CRISPR-Cas9 knockout screening. Results: An integrative analysis was conducted using CRISPR-Cas9 screening in vitro and in vivo with public NB datasets to identify 35 EP-TF genes that exhibited the highest expression in NB and were highly dependent on cancer viability. After univariate analysis, 27 of these 35 genes were included in the least absolute shrinkage and selection operator screen. We established and biologically validated a prognostic EP-TF model encompassing RUVBL1, LARP7, GTF3C4, THAP10, SUPT16H, TIGD1, SUV39H2, TAF1A, SMAD9, and FEM1B across diverse NB cohorts. MYCN serves a potential upstream regulator of EP-TF genes. The high-risk subtype exhibited traits associated with the malignant cell cycle, MYCN-linked signaling and chromatin remodeling, all of which are correlated with poor prognosis and immunosuppression. MEK inhibitors have emerged as promising therapeutic agents for targeting most EP-TF risk genes in NB. Conclusion: Our novel prognostic model shows significant potential for predicting and evaluating the overall survival of NB patients, offering insights into therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec96366563af269a504d1c7dc22ef594a4897d48" target='_blank'>
              CRISPR-Cas9 screening develops an epigenetic and transcriptional gene signature for risk stratification and target prediction in neuroblastoma
              </a>
            </td>
          <td>
            Liaoran Zhang, Jialin Mo, Hao Shi, Jing Xiong, Yeerfan Aierken, Feng Chen, Yujie Tang, Kewen Zhao, Zhibao Lv, Kezhe Tan
          </td>
          <td>2024-08-08</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="High risk - neuroblastoma (HR-NB) is a pediatric solid tumor with high lethality. Half of HR-NB are driven by MYCN gene amplification (MNA). These HR-NBs require high dosage chemotherapy and often relapse. Moreover, current therapies can cause severe long-term side effects and new therapies are urgently needed. This study investigates a novel therapeutic approach targeting the metabolic vulnerabilities of MNA NB cells. We discovered that Diphenyleneiodonium chloride (DPI), an inhibitor of flavoprotein enzymes and mitochondrial complex I, synergizes with mitoquinone mesylate (MitoQ), a mitochondria-targeted antioxidant in 2D and 3D in vitro models of NB. Similarly to DPI, MitoQ affects MNA cells in a MYCN-dependent way, being more toxic when MYCN levels are high. Furthermore, low nanomolar concentrations of MitoQ significantly decrease MYCN protein expression and induce differentiation of MNA cells. The DPI and MitoQ combination further synergizes with vincristine, a chemotherapeutic agent used in NB treatment. Phosphoproteomics and proteomics analysis suggests that the drug combination induces MNA NB cell death by arresting the cell cycle and inhibiting oxidative phosphorylation (OXPHOS) in the mitochondria. Thus, interference with mitochondrial metabolism may represent an effective strategy to enhance the activity of chemotherapeutic drugs in MNA-NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84bd27bba09b6f802fb233fccdb2001e1425eb1" target='_blank'>
              OXPHOS TARGETING OF MYCN-AMPLIFIED NEUROBLASTOMA
              </a>
            </td>
          <td>
            Soraya Epp, Donagh Egan, E. Poon, Amirah Adlina Abdul Aziz, Kieran Wynne, Melinda Halasz, Walter Kolch
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND The mitotic intra-S DNA damage checkpoint signaling gene set is potentially involved in cancers where the genes play an important role. 17 total genes are involved in this gene set: ATF2, CHEK2, EME1, EME2, FANCD2, HUS1, HUS1B, MDC1, MRE11, MSH2, MUS81, NEK11, RAD17, RAD9A, RAD9B, TIPIN, XPC. The aim of this study is to complete a pan-cancer profile of each gene in the mitotic intra-S DNA damage checkpoint signaling gene set in order to determine potential diagnostic and prognostic purposes, while also determining how they could be used in a clinical setting as therapeutic targets to help patients. METHODS Multiomic data was acquired for the 17 genes; over 9000 samples of 33 types of cancer were analyzed to create pan-cancer profiles of CNV, mRNA expression, and pathway analysis. RESULTS The CNVs of some of these genes are associated with the survival of MESO, PCPG, BLCA, SKCM, LUAD, HNSC, LUSC, OV, and BRCA could be affected by the mRNA expression of the genes which can involve regulation of copy number. CONCLUSION With sufficient investigation, the genes involved in mitotic intra-S DNA damage checkpoint signaling may contribute to the development of cancer and may be used as biomarkers for cancer prognosis and diagnosis. To prove their clinical use for diagnosis and prognosis, however, and to create workable applications in clinical settings, further work is required. However, these pan-cancer profiles provide a more comprehensive knowledge of the mitotic intra-S DNA damage checkpoint signaling gene set in cancer as well as valuable information for future reference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f222ed09772900dec985568ac443adca2980e33" target='_blank'>
              Potential Cancer Biomarkers: Mitotic Intra-S DNA Damage Checkpoint Genes
              </a>
            </td>
          <td>
            Kashvi Agarwal, Hengrui Liu
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Precision oncology promises individually tailored drugs and clinical care for patients with cancer: That is, "the right drug, for the right patient, at the right dose, and at the right time." Although stratification of the risk for treatment resistance and toxicity is key to precision oncology, there are multiple ways in which such stratification can be achieved, for example, genetic, functional pathway based, among others. Moving toward precision oncology is sorely needed in the case of acute lymphoblastic leukemia (ALL) wherein adult patients display survival rates ranging from 30% to 70%. The present study reports on the pathway activity signature of adult B-ALL, with an eye to precision oncology. Transcriptome profiles from three different expression datasets, comprising 346 patients who were adolescents or adults with B-ALL, were harnessed to determine the activity of signaling pathways commonly disrupted in B-ALL. Pathway activity analyses revealed that Ph-like ALL closely resembles Ph-positive ALL. Although this was the case at the average pathway activity level, the pathway activity patterns in B-ALL differ from genetic subtypes. Importantly, clustering analysis revealed that five distinct clusters exist in B-ALL patients based on pathway activity, with each cluster displaying a unique pattern of pathway activation. Identifying pathway-based subtypes thus appears to be crucial, considering the inherent heterogeneity among patients with the same genetic subtype. In conclusion, a pathway-based stratification of the B-ALL could potentially allow for simultaneously targeting highly active pathways within each ALL subtype, and thus might open up new avenues of innovation for personalized/precision medicine in this cancer that continues to have poor prognosis in adult patients compared with the children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72b528a52f9a1c25407f3a5bc1baf86dc4c7b9e9" target='_blank'>
              Expanding Beyond Genetic Subtypes in B-Cell Acute Lymphoblastic Leukemia: A Pathway-Based Stratification of Patients for Precision Oncology.
              </a>
            </td>
          <td>
            Ozlem Ulucan
          </td>
          <td>2024-08-19</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The fundamental transcription factor p53 regulates cellular processes and integrates signals of cellular stress, triggering a coordinated response to ensure survival of cells restored to healthy function and programmed death of those that couldn’t be repaired. Unsurprisingly, this is one of the most mutated genes in human cancers, with most changes occurring in the DNA-binding domain of the protein. In this work, we take a genome-wide approach and use available resources to identify high confidence p53-target genes, that we examine in three breast cancer cell lines with different p53 status, wild type (MCF-7) and different mutations in the DNA-binding domain (MDA-MB231, T47D). Comparison of p53-targets expression in response to DNA damage by RNAseq and cellular assays reveals that MDA-MB231 have a severely impaired p53-dependent pathway functionality while T47D are much less affected. MDA-MB231 are more resistant to DNA damage yet unable to repair and able to override cell cycle arrest leading to survival while T47D are sensitive only to high dose and exposure to genotoxic agents. This data shows the variability of effects of different p53 mutations and highlight the importance of understanding the mechanisms of p53 in the context of genotoxicity-based treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6785633c891564b2ca32ca3538063162d5c2bd" target='_blank'>
              Differential gene expression in cells with different p53 mutations identifies genome-wide p53 targets and shows distinct modulation of cellular pathways in response to DNA damage
              </a>
            </td>
          <td>
            Patricia Eror Barnes, Maria Jose de la Concha, Kioko Mwikali, B. L. Ng, Hannes Ponstingl, Alena Pance
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39f338bea38e6dfbcc4292c9fad6a48a280f8ab6" target='_blank'>
              A Bruton tyrosine kinase inhibitor-resistance gene signature predicts prognosis and identifies TRIP13 as a potential therapeutic target in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Yangyang Ding, Keke Huang, Cheng Sun, Zelin Liu, Jinli Zhu, Xunyi Jiao, Ya Liao, Xiangjiang Feng, Jingjing Guo, Chunhua Zhu, Zhimin Zhai, Shudao Xiong
          </td>
          <td>2024-09-11</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Breast cancer is the most prevalent malignant tumor among women with high heterogeneity. Traditional techniques frequently struggle to comprehensively capture the intricacy and variety of cellular states and interactions within breast cancer. As global precision medicine rapidly advances, single-cell RNA sequencing (scRNA-seq) has become a highly effective technique, revolutionizing breast cancer research by offering unprecedented insights into the cellular heterogeneity and complexity of breast cancer. This cutting-edge technology facilitates the analysis of gene expression profiles at the single-cell level, uncovering diverse cell types and states within the tumor microenvironment. By dissecting the cellular composition and transcriptional signatures of breast cancer cells, scRNA-seq provides new perspectives for understanding the mechanisms behind tumor therapy, drug resistance and metastasis in breast cancer. In this review, we summarized the working principle and workflow of scRNA-seq and emphasized the major applications and discoveries of scRNA-seq in breast cancer research, highlighting its impact on our comprehension of breast cancer biology and its potential for guiding personalized treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d20b5e78664c0194ad04247bdfc15101c7c5f63" target='_blank'>
              Single-Cell RNA-Sequencing: Opening New Horizons for Breast Cancer Research
              </a>
            </td>
          <td>
            Lingyan Xiang, Jie Rao, Jingping Yuan, Ting Xie, Honglin Yan
          </td>
          <td>2024-08-31</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Abstract Small-cell lung cancer (SCLC) is the most lethal type of lung cancer. Paradoxically, this tumor displays an initial exquisite response to chemotherapy; however, at relapse, the tumor is highly resistant to subsequent available therapies. Here, we report that the expression of three prime repair exonuclease 1 (TREX1) is strongly induced in chemoresistant SCLCs. Assay for transposase-accessible chromatin using sequencing and chromatin immunoprecipitation sequencing revealed a significant increase in chromatin accessibility and transcriptional activity of TREX1 gene locus in chemoresistant SCLCs. Analyses of human SCLC tumors and patient-derived xenografts (PDX) also showed an increase in TREX1 expression in postchemotherapy samples. TREX1 depletion caused the activation of cyclic GMP-AMP synthase stimulator of interferon gene pathway due to cytoplasmic accumulation of damage-associated double-stranded DNA, inducing immunogenicity and enhancing the sensitivity of drug-resistant cells to chemotherapy. These findings suggest TREX1 upregulation may partially contribute to the survival of resistant cells, and its inhibition may represent a promising therapeutic strategy to enhance antitumor immunity and potentiate the efficacy of chemotherapy and/or immunotherapy in chemoresistant SCLCs. Significance: In this study, we show that targeting TREX1 induces an innate immune response and resensitizes SCLC cells to chemotherapy, representing a promising novel target for “immunologically” cold tumors, such as SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920774aa4db00913c78963d621212ab8e7a2e837" target='_blank'>
              Targeting TREX1 Induces Innate Immune Response in Drug-Resistant Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Takahiko Murayama, N. Mahadevan, Catherine B Meador, Elena V. Ivanova, Yuqiao Pan, Erik H. Knelson, Tetsuo Tani, Jun Nakayama, Xueying Ma, Tran C Thai, Yin P Hung, William Kim, Hideo Watanabe, Kathy Q Cai, Aaron N Hata, C. Paweletz, D. Barbie, Israel Cañadas
          </td>
          <td>2024-08-23</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chimeric genes and transcripts can be biological markers as well as the reasons for tumor progression and development. Modern algorithms and high-throughput sequencing are the complementary clues to the question of the tumor origin and cancer detection as well as to the fundamental question of chimeric genes origin and their influence on molecular processes of the cell. A wide-range of algorithms for chimeric genes detection was developed, with various differences in computing speed, sensitivity, specificity, and focus on the experimental design. There exist three main types of bioinformatic approaches, which act according to the sequencing read length. Algorithms, which focus on short-read high-throughput sequencing (about 50–300 bр of read length) or long-read sequencing (about 5000–100000 bр of read length) exclusively or algorithms, which combine the results of both short and long-read sequencing. These algorithms are further subdivided into: 1) mapping-first approaches (STAR-Fusion, Arriba), which map reads to the genome or transcriptome directly and search the reads supporting the fused gene or transcript; 2) assembly-first approaches (Fusion-Bloom), which assemble the genome or transcriptome from the overlapping reads, and then compare the results to the reference transcriptome or genome to find transcripts or genes not present in the reference and therefore raising questions; 3) pseudoalignment approaches, which do not make local alignment, but just search for the closest transcript subsequence to the reads seed, following the precomputed index for all reference transcripts and provides the results. This article describes the main classes of available software tools for chimeric gene detection, provides the characteristics of these programs, their advantages and disadvantages. To date the most resource intensive and slowest are still assembly-first algorithms. Mapping-first approaches are quite fast and rather accurate at fusion detection, still the fastest and resource-saving are the pseudoalignment algorithms, but, worth noting, that the quick search is carried out at the expense of chimeras search quality decrease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40da4e0fb6446cdfd7e663fbf9214c9d51fb3a2" target='_blank'>
              Bioinformatic approaches for detection of fusion genes and trans-splicing products
              </a>
            </td>
          <td>
            I. Y. Musatov, M. I. Sorokin, А. A. Buzdin
          </td>
          <td>2024-08-11</td>
          <td>Биоорганическая химия</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tyrosine kinase inhibitors (TKIs) have revolutionized the treatment landscape for chronic myeloid leukemia (CML). However, TKI resistance poses a significant challenge, leading to treatment failure and disease progression. Resistance mechanisms include both BCR::ABL1-dependent and BCR::ABL1-independent pathways. The mechanisms underlying BCR::ABL1 independence remain incompletely understood, with CML cells potentially activating alternative signaling pathways, including the AKT/mTOR and JAK2/STAT5 pathways, to compensate for the loss of BCR::ABL1 kinase activity. This study explored tumoral VISTA (encoded by VSIR) as a contributing factor to TKI resistance in CML patients and identified elevated tumoral VISTA levels as a marker of resistance and poor survival. Through in vitro and in vivo analyses, we demonstrated that VSIR knockdown and the application of NSC-622608, a novel VISTA inhibitor, significantly impeded CML cell proliferation and induced apoptosis by attenuating the AKT/mTOR and JAK2/STAT5 pathways, which are crucial for CML cell survival independent of BCR::ABL1 kinase activity. Moreover, VSIR overexpression promoted TKI resistance in CML cells. Importantly, the synergistic effect of NSC-622608 with TKIs offers a potent therapeutic avenue against both imatinib-sensitive and imatinib-resistant CML cells, including those harboring the challenging T315I mutation. Our findings highlight the role of tumoral VISTA in mediating TKI resistance in CML, suggesting that inhibition of VISTA, particularly in combination with TKIs, is an innovative approach to enhancing treatment outcomes in CML patients, irrespective of BCR::ABL1 mutation status. This study not only identified a new pathway contributing to TKI resistance but also revealed the possibility of targeting tumoral VISTA as a means of overcoming this significant clinical challenge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41cf4290358e1af5f36f27cd7397321411a29f7e" target='_blank'>
              Inhibition of Tumoral VISTA to Overcome TKI Resistance via Downregulation of the AKT/mTOR and JAK2/STAT5 Pathways in Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            Kexin Ai, Mu Chen, Zhao Liang, Xiangyang Ding, Yang Gao, Honghao Zhang, Suwan Wu, Yanjie He, Yuhua Li
          </td>
          <td>2024-08-06</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e5e2db57a9a9a8ae1d5af55bcc4e63ea2e814f6" target='_blank'>
              The K9 lymphoma assay allows a genetic subgrouping of canine lymphomas with improved risk classification
              </a>
            </td>
          <td>
            A. Fanelli, L. Licenziato, Eugenio Mazzone, S. Divari, A. Rinaldi, Michele Marino, Ilaria Maga, Francesco Bertoni, L. Marconato, Luca Aresu
          </td>
          <td>2024-08-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="This review offers an in-depth analysis of mitochondrial DNA (mtDNA) mutations in colorectal cancer stem cells (CSCs), emphasizing their significant impact on tumor dynamics and potential therapeutic strategies. CSCs are a special subpopulation due to their unique capabilities for self-renewal, differentiation, and resistance to conventional therapies. Given that CSCs significantly differ from other tumor cell subpopulations, particularly in their metabolic properties, and considering that colorectal cancer is a malignancy characterized by mitochondrial dysfunction, this review aims to put together existing data on the differences in the mitochondrial genome of CSCs compared to other colorectal tumor cell subpopulations. Additionally, the review seeks to explore the potential roles of these differences and to identify new ideas for therapeutic strategies. Key topics include the identification and properties of CSCs in colorectal cancer, the distinctive features of the mitochondrial genome, and the functional consequences of mtDNA mutations. The review hypothesizes that CSCs rely on well-functioning mitochondria for crucial aspects like energy production; yet, mtDNA mutations can lead to mitochondrial dysfunction, altering CSC characteristics and influencing cancer progression. The article discusses emerging therapeutic approaches targeting mitochondrial function in colorectal CSCs and highlights the need for advanced research, including the development of preclinical models and exploration of targeted therapies, to improve the understanding and treatment of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebae925d5c7100b4c278e4586e85603f7ee40d86" target='_blank'>
              Mitochondrial DNA Mutations in Colorectal Cancer Stem Cells: Implications for Tumor Dynamics and Therapeutic Strategies.
              </a>
            </td>
          <td>
            N. Shakhpazyan, L. Mikhaleva, Arcady L Bedzhanyan, Z. Gioeva, A. Mikhalev, K. Midiber, A. Konyukova, D. Atiakshin, Igor Buchwalow, Markus Tiemann, Alexander N. Orekhov
          </td>
          <td>2024-09-11</td>
          <td>Current medicinal chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Alzheimer's disease (AD) is a common and increasing societal problem due to the extending human lifespan. In males, loss of chromosome Y (LOY) in leukocytes is strongly associated with AD. We studied here DNA methylation and RNA expression in sorted monocytes and granulocytes with and without LOY from male AD patients. Through multi-omic analysis, we identified new candidate genes and confirmed the involvement of numerous genes previously associated with AD. Our findings highlight LOY-related differences in DNA methylation that occur in gene regulatory regions and are predominantly accompanied by down-regulation of affected genes. Specifically, we observed alterations in key genes involved in leukocyte differentiation: FLI1, involved in early hematopoiesis; RUNX1, essential for blood cell development; RARA, regulating gene expression in response to retinoic acid; CANX, crucial for protein folding; CEBPB, a transcription factor important for immune responses; and MYADM, implicated in cell adhesion and migration. Moreover, protein-protein interaction analysis in granulocytes identified that products of two of these genes, CANX and CEBPB, are key hub proteins. Thus, LOY appears to dysregulate genes involved in leukocyte differentiation and induce higher-level epigenetic changes. This research underscores the potential of multi-omic approaches in pure cell populations to uncover the molecular underpinnings of AD and reinforces the significance of LOY as a pathogenic factor in this disease. Overall, results support the hypothesis that age-related immune cell dysfunction contributes to AD development. Finally, our results link previous analysis showing impact of LOY on leukocyte differentiation, LOY-associated transcriptional dysregulation and GWAS studies of LOY.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9899fb011458e044621821bc8b19932591bd72f" target='_blank'>
              DNA methylation patterns contribute to changes of cellular differentiation pathways in leukocytes with LOY from patients with Alzheimer''s disease
              </a>
            </td>
          <td>
            M. Jakalski, B. Bruhn-Olszewska, E. Rychlicka-Buniowska, H. Davies, D. Sarkisyan, M. Siedlar, J. Baran, K. Weglarczyk, J. Jaszczynski, J. Rys, V. Gedraitis, N. Filipowicz, A. Klich-Raczka, L. Kilander, M. Ingelsson, J. P. Dumanski
          </td>
          <td>2024-08-20</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a major cause of cancer-related deaths in developed nations. Factors such as unhealthy lifestyle choices, particularly smoking, contribute to the development of this disease. Epigenetic abnormalities greatly affect gene expression and disrupt important cellular signaling pathways that are responsible for the proper growth, regulation, and functioning of cells. In the case of lung cancer, specifically non-small cell lung cancer (NSCLC), CBX3 acts as an epigenetic oncoprotein, promoting the growth and progression of tumors. Numerous studies have shown that CBX3 is overexpressed in NSCLC and is associated with a poor prognosis. It interacts with key oncogenic pathways leading to increased proliferation, reduced apoptosis, and enhanced resistance to therapy. Further research on the mechanisms and functions of CBX3 holds the potential to reveal new insights into the development of the disease and uncover novel therapeutic opportunities. Abstract Epigenetic changes regulate gene expression through histone modifications, chromatin remodeling, and protein translation of these modifications. The PRC1 and PRC2 complexes shape gene repression via histone modifications. Specifically, the CBX protein family aids PRC1 recruitment to chromatin, impacting the progressive multistep process driving chromatin silencing. Among family members, CBX3 is a complex protein involved in aberrant epigenetic mechanisms that drive lung cancer progression. CBX3 promotes lung tumorigenesis by interacting with key pathways such as PI3K/AKT, Ras/KRAS, Wnt/β-catenin, MAPK, Notch, and p53, leading to increased proliferation, inhibition of apoptosis, and enhanced resistance to therapy. Given our current lack of knowledge, additional research is required to uncover the intricate mechanisms underlying CBX3 activity, as well as its involvement in molecular pathways and its potential biomarker evaluation. Specifically, the dissimilar roles of CBX3 could be reexamined to gain a greater insight into lung cancer pathogenesis. This review aims to provide a clear overview of the context-related molecular profile of CBX3, which could be useful for addressing clinical challenges and developing novel targeted therapies based on personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b8cecc66992d19bc95a90f28c7e4981384afb5" target='_blank'>
              Exploring the Role of CBX3 as a Potential Therapeutic Target in Lung Cancer
              </a>
            </td>
          <td>
            Muhammad Aamir Wahab, Nunzio Del Gaudio, Biagio Gargiulo, V. Quagliariello, N. Maurea, A. Nebbioso, L. Altucci, Mariarosaria Conte
          </td>
          <td>2024-08-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 Barriers in access to clinical care among under-served populations in combination with a vast underrepresentation at a scientific research level perpetuate inequalities in cancer treatment. These disparities are further disseminated due to the lack of understanding underlying genomic differences contributing to cancer outcome among these patients. The contributions of germline and somatic variations in cancer often shape treatment regimens, and these variations are not well investigated in non-European populations. We have described a class of potential therapeutic targets (GEMINI genes) that reflect a combination of germline alterations with somatic loss of heterozygosity (LOH). GEMINI genes are essential genes with frequent germline polymorphisms, whose frequency varies by ancestry. Indeed, because genomes of African ancestry tend to be the most diverse, many GEMINI genes are the most polymorphic in this patient population. When normal cells are heterozygous, LOH leaves cancer cells reliant on the gene products encoded by a single allele, where as healthy cells retain both copies. LOH is a common genetic event, often affecting over 20% of the genome in certain cancers, making this approach widely useful. Targeting GEMINI vulnerabilities opens up possibilities for allele-specific inhibitors or degrader therapeutics that can be catered in an ancestry-dependent manner, to ameliorate inequities in cancer genomics research. One approach to readily target individual alleles is to utilize nucleic acid-based methods, as we’ve previously demonstrated using PRIM1. This therapeutic is expected to be the most specific when targeting polymorphisms that create the greatest differences between alleles. Here, we have prioritized a set of GEMINI targets based upon the extent of differences between alleles, and rates of heterozygosity and LOH across populations. Our initial analysis of TCGA data identified 5664 variants in 1278 essential genes that undergo LOH in cancer. We filtered GEMINI vulnerabilities by means of insertion-deletion (indel) difference of the alternate allele compared to the reference allele, followed by sorting based off maximum allele frequency by genetic ancestry. Here, we identify 31 variants across 23 genes enriched in African ancestry, 7 variants across 7 genes enriched in American ancestry, 12 variants across 12 genes enriched in East Asian ancestry, 4 variants enriched in 4 genes in South Asian ancestry, and 3 variants enriched in 3 genes in European ancestry. In addition, each of these ancestry-specific GEMINI genes exhibit a minimum 10% pan-cancer LOH rate, with the highest LOH frequencies observed in ovarian and kidney chromophobe cancers. As both these cancers exhibit disparities in mortality at a population level, targeting GEMINI genes as a therapeutic strategy may aid in lessening the disproportionate rate of outcome resultant of these cancers. Overall, we identify novel GEMINI genes to assess ancestry-specific genomic variations to serve as targets for precision treatment specific to underserved patient populations.
 Citation Format: Nicole Peiris, Catherine Hazard, Charlotte E. Farquhar, Andrei Loas, Bradley L. Pentelute, Rameen Beroukhim. Genetic ancestry specific loss of heterozygosity of essential genes as therapeutic vulnerabilities in cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C099.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e337d55b2fcfc1ea91f26641b13ccce525e55ce5" target='_blank'>
              Abstract C099: Genetic ancestry specific loss of heterozygosity of essential genes as therapeutic vulnerabilities in cancer
              </a>
            </td>
          <td>
            Nicole Peiris, Catherine Hazard, C. Farquhar, A. Loas, Bradley L. Pentelute, R. Beroukhim
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb14f52e0abb7fcb6b84cd46a03fa1b6ced7483" target='_blank'>
              Genomic and molecular alterations associated with primary resistance to immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Jyoti Malhotra, Subhajyoti De, Kim Nguyen, Percy Lee, Victoria Villaflor
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Understanding cancer evolution is fundamental to predicting cancer progression. However, the evolution of paediatric cancers is still under-researched. Large cohorts of patients are required to determine consistent evolutionary trajectories that shed light on key steps in cancer development and reveal underlying biology, especially in rare cancers. Additionally, well annotated clinical data is necessary for determining if evolutionary biomarkers are predictive of patient outcome. Methods We performed detailed evolutionary analysis of 64 paediatric kidney cancers, including 60 Wilms tumours (WT), using DNA microarrays and, in a subset of 30 patients, a WT-specific targeted sequencing assay, to detect copy number alterations (CNA) and mutations, respectively. By analysing multiple tissue samples in the majority of cases we could detect mutation heterogeneity in each tumour. We reconstructed clones across the cohort and described their phylogenetic histories, in addition to detecting mirrored subclonal allelic imbalance. Results Our results highlight pervasive evidence of parallel evolution in WTs affecting CNAs, and CTNNB1 and TP53 mutations. Furthermore, we demonstrate that stromal-type WTs often evolve from a consistent series of events (WT1 mutation, 11p uniparental disomy and CTNNB1 mutation) and we suggest that 19q uniparental disomy is an important ancestral event in both epithelial and diffuse anaplastic WTs. Finally, we propose the total number of evolutionary CNA events as a prognostic biomarker in WTs for event-free survival, particularly in high-risk WT. Conclusions Overall, this study sheds light on the evolution of the most common paediatric kidney cancer and links evolutionary analysis to fundamental clinical and biological questions in a large cohort of WTs. We conclude that histological subtypes of WT are often defined by consistent evolutionary sequences. We present evidence that a key marker of evolvability, namely CNA diversity measured phylogenetically across multiple tumour sites, is prognostic of patient outcome and should be considered for clinical use to detect the most aggressive blastemal and diffuse anaplastic type WTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24a5fbaf8b2a5b212183cb8a0bf89d97d1b5f3b" target='_blank'>
              Subtype-specific Patterns of Evolution and Clinically Relevant Genomic Instability in Wilms Tumour
              </a>
            </td>
          <td>
            George D. Cresswell, T. Chagtai, Reem Al-Saadi, T. Treger, G. Madhan, Borbala Mifsud, Gordan M. Vujanic, R. Williams, N. Luscombe, Kathy Pritchard-Jones, William Mifsud
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Cancer cell dormancy (CCD) in colorectal cancer (CRC) poses a significant challenge to effective treatment. In CRC, CCD contributes to tumour recurrence, drug resistance, and amplifying the disease's burden. The molecular mechanisms governing CCD and strategies for eliminating dormant cancer cells remain largely unexplored. Therefore, understanding the molecular mechanisms governing dormancy is crucial for improving patient outcomes and developing targeted therapies. This editorial highlights the complex interplay of signalling pathways and factors involved in colorectal CCD, emphasizing the roles of Hippo/YAP, pluripotent transcription factors such as NANOG, HIF-1α signalling, and Notch signalling pathways. Additionally, ERK/p38α/β/MAPK pathways, AKT signalling pathway, and Extracellular Matrix Metalloproteinase Inducer, along with some potential less explored pathways such as STAT/p53 switch and canonical and non-canonical Wnt and SMAD signalling, are also involved in promoting colorectal CCD. Highlighting their clinical significance, these findings may offer the potential for identifying key dormancy regulator pathways, improving treatment strategies, surmounting drug resistance, and advancing personalized medicine approaches. Moreover, insights into dormancy mechanisms could lead to the development of predictive biomarkers for identifying patients at risk of recurrence and the tailoring of targeted therapies based on individual dormancy profiles. It is essential to conduct further research into these pathways and their modulation to fully comprehend CRC dormancy mechanisms and enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5a1fd8a1c24adda33f9d39ee28ef2dc82663eff" target='_blank'>
              Colorectal cancer cell dormancy: An insight into pathways
              </a>
            </td>
          <td>
            Anil Kumar, L. Saha
          </td>
          <td>2024-09-07</td>
          <td>World Journal of Gastroenterology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML), an aggressive malignancy of hematopoietic stem cells, is characterized by the blockade of cell differentiation, uncontrolled proliferation, and cell expansion that impairs healthy hematopoiesis and results in pancytopenia and susceptibility to infections. Several genetic and chromosomal aberrations play a role in AML and influence patient outcomes. TP53 is a key tumor suppressor gene involved in a variety of cell features, such as cell-cycle regulation, genome stability, proliferation, differentiation, stem-cell homeostasis, apoptosis, metabolism, senescence, and the repair of DNA damage in response to cellular stress. In AML, TP53 alterations occur in 5%–12% of de novo AML cases. These mutations form an important molecular subgroup, and patients with these mutations have the worst prognosis and shortest overall survival among patients with AML, even when treated with aggressive chemotherapy and allogeneic stem cell transplant. The frequency of TP53-mutations increases in relapsed and recurrent AML and is associated with chemoresistance. Progress in AML genetics and biology has brought the novel therapies, however, the clinical benefit of these agents for patients whose disease is driven by TP53 mutations remains largely unexplored. This review focuses on the molecular characteristics of TP53-mutated disease; the impact of TP53 on selected hallmarks of leukemia, particularly metabolic rewiring and immune evasion, the clinical importance of TP53 mutations; and the current progress in the development of preclinical and clinical therapeutic strategies to treat TP53-mutated disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/442da7a2ffb734ac40ab31ab13428c2a4af47cbd" target='_blank'>
              Impact of p53-associated acute myeloid leukemia hallmarks on metabolism and the immune environment
              </a>
            </td>
          <td>
            Monika Chomczyk, Luca Gazzola, Shubhankar Dash, Patryk Firmanty, Binsah S. George, V. Mohanty, Hussein A. Abbas, Natalia Baran
          </td>
          <td>2024-08-05</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer death by 2030. PDAC lacks effective treatment options and innovative strategies for therapeutic intervention are urgently needed. Chimeric Antigen Receptor (CAR)-T cell therapy has shown remarkable success in treating hematologic malignancies, yet achieving efficacy in solid tumors remains a significant challenge. Early results from clinical trials show limited efficacy of CAR T-cell therapy in PDAC. To uncover tumor-intrinsic mechanisms of resistance to CAR T-cell therapy in vivo, we performed iterative CRISPR/Cas9-based pooled screens in a fully immunocompetent, orthotopic model of PDAC. We identified a variety of genes influencing CAR T-cell treatment efficacy. In particular, loss of genes involved in redox biology and oxidative stress sensitized PDAC tumors to CAR T-cell therapy in vivo. Additionally, single-cell RNA sequencing of PDAC tumors subjected to CAR T-cell therapy unveiled a distinct subset of tumor cells resilient to the treatment, characterized by changes in oxidative stress. Our findings indicate a potential avenue for therapeutic synergy by targeting pathways involved in oxidative stress to increase the efficacy of CAR T cell therapy in PDAC.
 Citation Format: Julia Fröse, Charlie Whittaker, Paul Leclerc, Adam Langenbucher, Riley Hellinger, Daniel R. Goulet, Michael T. Hemann. Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B060.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a130506e26e2da6b9c6338dc0032b746f1f4ca46" target='_blank'>
              Abstract B060: Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening
              </a>
            </td>
          <td>
            Julia Fröse, Charles A. Whittaker, Paul Leclerc, Adam Langenbucher, Riley D Hellinger, Daniel R. Goulet, M. Hemann
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c7724f7614b28025fae5c6a3ee774acac9c238f" target='_blank'>
              Genome-wide CRISPR-Cas9 knockout screens identify DNMT1 as a druggable dependency in sonic hedgehog medulloblastoma
              </a>
            </td>
          <td>
            F. Tsiami, C. Lago, Noemi Pozza, F. Piccioni, Xuesong Zhao, Fabienne Lülsberg, David E. Root, Luca Tiberi, Marcel Kool, Jens Schittenhelm, P. Bandopadhayay, R. Segal, Ghazaleh Tabatabai, Daniel J. Merk
          </td>
          <td>2024-08-07</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Alveolar rhabdomyosarcoma (ARMS) patients harboring PAX3-FOXO1 and PAX7-FOXO1 fusion proteins exhibit a greater incidence of tumor relapse, metastasis, and poor survival outcome, thereby underscoring the urgent need to develop effective therapies to treat this subtype of childhood cancer. To uncover mechanisms that contribute to tumor initiation, we developed a novel muscle progenitor model and used epigenomic approaches to unravel genome re-wiring events mediated by PAX3/7 fusion proteins. Importantly, these regulatory mechanisms are conserved across established ARMS cell lines, primary tumors, and orthotopic-patient derived xenografts. Among the key targets of PAX3- and PAX7-fusion proteins, we identified a cohort of oncogenes, FGF receptors, and genes essential for mitochondrial metabolism and protein translation, which we successfully targeted in preclinical trials. Our data suggest an explanation for the relative paucity of recurring mutations in this tumor, provide a compelling list of actionable targets, and suggest promising new strategies to treat this tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f840f071bc2696a99f7c83f33f72c0a963e3308" target='_blank'>
              PAX fusion proteins deregulate gene networks controlling mitochondrial translation in pediatric rhabdomyosarcoma
              </a>
            </td>
          <td>
            Bhargab Kalita, Gerard Martinez-Cebrian, Justina McEvoy, Melody Allensworth, Michelle Knight, Alessandro Magli, Rita C. R. Perlingeiro, Michael A. Dyer, Elizabeth Stewart, B. Dynlacht
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Parallel single-cell multimodal sequencing is the most intuitive and precise tool for cellular status research. In this study, we propose AMAR-seq to automate methylation, chromatin accessibility, and RNA expression coanalysis with single-cell precision. We validated the accuracy and robustness of AMAR-seq in comparison with standard single-omics methods. The high gene detection rate and genome coverage of AMAR-seq enabled us to establish a genome-wide gene expression regulatory atlas and triple-omics landscape with single base resolution and implement single-cell copy number variation analysis. Applying AMAR-seq to investigate the process of mouse embryonic stem cell differentiation, we revealed the dynamic coupling of the epigenome and transcriptome, which may contribute to unraveling the molecular mechanisms of early embryonic development. Collectively, we propose AMAR-seq for the in-depth and accurate establishment of single-cell multiomics regulatory patterns in a cost-effective, efficient, and automated manner, paving the way for insightful dissection of complex life processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daa2f0b9dd274ee3a6047b970e14792c843b3fd0" target='_blank'>
              AMAR-seq: Automated Multimodal Sequencing of DNA Methylation, Chromatin Accessibility, and RNA Expression with Single-Cell Resolution.
              </a>
            </td>
          <td>
            Xi Zeng, Xiaoping Yang, Zhixing Zhong, Xinrui Lin, Qiuyue Chen, Shaowei Jiang, Mengwu Mo, Shichao Lin, Huimin Zhang, Zhi Zhu, Jin Li, Jia Song, Chao Yang
          </td>
          <td>2024-09-09</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6728cb375ed8ebc4bb5bd4c0c454f7b4f90445d" target='_blank'>
              SUV39H1 epigenetically modulates the MCPIP1-AURKA signaling axis to enhance neuroblastoma tumorigenesis.
              </a>
            </td>
          <td>
            Mengzhen Li, F. Sun, Juan Wang, Suying Lu, Yi Que, Mengjia Song, Hui-qin Chen, Xiangyu Xiong, Weiji Xie, Jia Zhu, Junting Huang, Yu Zhang, Yizhuo Zhang
          </td>
          <td>2024-09-19</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/311bc754350204df7366f1cc87322a6aee4bc5a6" target='_blank'>
              Assessment of SWI/SNF chromatin remodeling complex related genes as potential biomarkers and therapeutic targets in pan-cancer
              </a>
            </td>
          <td>
            Kai Zhuang, Lishan Wang, Chengyu Lu, Zhiping Liu, Dongli Yang, Hao Zhong, Jiami Zou, Aamir Fahira, Jiaojiao Wang, Zunnan Huang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Epigenetics denotes heritable alterations in gene expression patterns independent of changes in DNA sequence. Epigenetic therapy seeks to reprogram malignant cells to a normal phenotype and has been extensively investigated in oncology. This study conducts a bibliometric analysis of epigenetic therapy in cancer, providing a comprehensive overview of current research, identifying trends, and highlighting key areas of investigation. Methods Publications concerning epigenetic inhibitors in cancer spanning 2004 to 2023 were retrieved from the Web of Science Core Collection (WoSCC). Co-occurrence analysis using VOSviewer assessed current status and focal points. Evolutionary trends and bursts in the knowledge domain were analyzed using CiteSpace. Bibliometrix facilitated topic evolution and revealed trends in keywords. National, institutional, and author affiliations and collaborations were also examined. Results A total of 2,153 articles and reviews on epigenetic therapy in oncology were identified, demonstrating a consistent upward trend over time. The United States (745 papers), University of Texas MD Anderson Cancer Center (57 papers), and Stephen B. Baylin (27 papers) emerged as the most productive country, institution, and author, respectively. Keyword co-occurrence analysis identified five primary clusters: tumor, DNA methylation, epigenetic therapy, expression, and immunotherapy. In the past 5 years, newly emerging themes with increased centrality and density include “drug resistance,” “immunotherapy,” and “combination therapy.” The most cited publication reviewed current understanding of potential causes of epigenetic diseases and proposed future therapeutic strategies. Conclusion In the past two decades, the importance of epigenetic therapy in cancer research has become increasingly prominent. The United States occupies a key position in this field, while China, despite having published a large number of related papers, still has relatively limited influence. Current research focuses on the “combination therapy” of epigenetic drugs. Future studies should further explore the sequencing and scheduling of combination therapies, optimize trial designs and dosing regimens to improve clinical efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df9baed6fff272ce0f1cbc502a348d307f715a03" target='_blank'>
              Research hotspots and trends of epigenetic therapy in oncology: a bibliometric analysis from 2004 to 2023
              </a>
            </td>
          <td>
            Sisi Li, Xinrui Liang, Qing Shao, Guanwen Wang, Yuxin Huang, Ping Wen, Dongping Jiang, Xiaohua Zeng
          </td>
          <td>2024-09-12</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pediatric cancers are frequently driven by genomic alterations that result in aberrant transcription factor activity. While direct targeting of transcription factors is difficult, one method to circumvent this problem is to target co-factors and chromatin complexes that are necessary for their oncogenic function. To evaluate protein complex dependencies enriched in MYCN-amplified neuroblastoma, a disease of dysregulated development driven by aberrant expression of an oncogenic expression program, we curated a list of protein complexes using the CORUM database and mined the Dependency Map using single sample gene set enrichment analysis. This analysis identified the KAT module of the transcriptional coactivator Spt-Ada-Gcn5-acetyltransferase (SAGA) complex as a selective dependency in MYCN-amplified neuroblastoma. Loss of SAGA complex activity in neuroblastoma through both genetic knockout and induced protein degradation of the lysine acetyltransferase (KAT) module subunit TADA2B resulted in a global reduction of histone lysine acetylation and impaired neuroblastoma cell growth. Genetic knockout of TADA2B did not impair the growth of pediatric sarcoma cell lines, suggesting that SAGA loss-of-function is not broadly deleterious. Furthermore, SAGA loss-of-function reduced MYCN binding to chromatin, resulting in suppression of the MYCN-driven gene expression program and impaired cell cycle progression. Importantly, the SAGA complex is pharmacologically targetable in vitro and in vivo with GSK-699, a PROTAC that induces proteolysis of both KAT2A and KAT2B paralogs. As observed with genetic knockout studies, loss of SAGA activity through KAT2A/B degradation reduced the MYCN-driven transcriptional signature and impaired cell cycle progression. Our findings expand our understanding of the chromatin complexes that maintain the oncogenic transcriptional state in MYCN-amplified neuroblastoma and suggest therapeutic potential for inhibitors of SAGA KAT activity in MYCN-amplified neuroblastoma.
 Citation Format: Nathaniel W. Mabe, Clare F. Malone, Alexandra B. Forman, Gabriela Alexe, Kathleen L. Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar, Bin Liu, Sharon YR Dent, Kimberly Stegmaier. Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be30c933e33904854e0410865fe3ee874fab512e" target='_blank'>
              Abstract A062 Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program
              </a>
            </td>
          <td>
            Nathaniel W. Mabe, Clare F Malone, Alexandra B Forman, G. Alexe, Kathleen Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar, Bin Liu, S. Y. Dent, K. Stegmaier
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="B-cell acute lymphoblastic leukemia (B-ALL) is the most common pediatric cancer, with long-term overall survival rates of ~85%. However, B-ALL harboring rearrangements of the MLL gene (also known as KMT2A), referred to as MLLr B-ALL, is common in infants and is associated with poor 5-year survival (<30%), frequent relapses, and refractoriness to glucocorticoids (GCs). GCs are an essential part of the treatment backbone for B-ALL and GC resistance is a major clinical predictor of poor outcome. Elucidating the mechanisms of GC resistance in MLLr B-ALL is, therefore, critical to guide therapeutic strategies that deepen the response after induction therapy. Neuron-glial antigen-2 (NG2) expression is a hallmark of MLLr B-ALL and is minimally expressed in healthy hematopoietic cells. We recently reported that NG2 expression is associated with poor prognosis and that anti-NG2 immunotherapy strongly reduces/delays relapse in MLLr B-ALL xenograft models. Despite its contribution to MLLr B-ALL pathogenesis and its diagnostic utility, the role of NG2 in MLLr-mediated leukemogenesis/chemoresistance remains elusive. Here we show that NG2 is an epigenetically regulated direct target gene of the leukemic MLL-AF4 fusion protein. NG2 negatively regulates the expression of the GC receptor NR3C1 and confers GC resistance to MLLr B-ALL cells in vitro and in vivo. Mechanistically, NG2 interacts with FLT3 to render ligand-independent activation of FLT3 signaling (a hallmark of MLLr B-ALL) and downregulation of NR3C1 via AP-1-mediated trans-repression. Collectively, our study elucidates the role of NG2 in GC resistance in MLLr B-ALL through FLT3/AP-1-mediated downregulation of NR3C1, providing novel therapeutic avenues for MLLr B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0170423741ccbe8da38d4d329c1b108fb9d545" target='_blank'>
              NG2 is a target gene of MLL-AF4 and underlies glucocorticoid resistance in MLL-r B-ALL by regulating NR3C1 expression.
              </a>
            </td>
          <td>
            B. López‐Millán, A. Rubio-Gayarre, M. Vinyoles, J. Trincado, Mario F Fraga, N. Fernández-Fuentes, Mercedes Guerrero-Murillo, A. Martínez-Moreno, T. Velasco-Hernández, A. Falgàs, Carla Panisello, Gemma Valcárcel, J. L. Sardina, Paula López-Martí, B. Javierre, Beatriz Del Valle-Pérez, Antonio García de Herreros, Franco Locatelli, Rob Pieters, M. Bardini, G. Cazzaniga, J. C. Rodríguez-Manzaneque, T. Hanewald, R. Marschalek, Thomas A. Milne, R. Stam, J. R. Tejedor, Pablo Menendez, C. Bueno
          </td>
          <td>2024-08-02</td>
          <td>Blood</td>
          <td>1</td>
          <td>59</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma A. Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="An important goal in cancer research is to identify driver genes and mutations. Reasoning that such mutations often alter enzymatic functions, we investigated the cancer driver role of enzyme families. Using pan-cancer genomic data and established driver mutation catalogues, we found an unexpectedly high rate of mutations in helicases, making helicases the most frequently mutated enzyme family in cancer. Based on both functional perturbation screens and cancer genomic analyses, we provide evidence that cancers with mutated helicases converge on increased genomic instability and faulty DNA repair. We identify a striking phenotype in cells with loss of the helicase Aquarius (AQR). AQR was exclusively hemizygous lost in cancer genomes, which was associated with elevated levels of structural variants and point mutation signatures indicative of homologous recombination deficiency. Finally, we leverage large dependency maps to show that hemizygous loss is a common tumour suppression mechanism among helicases. In summary, we uncover a striking frequency of mutated helicases with key roles in genomic maintenance, and we nominate novel hemizygous cancer driver genes including AQR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/381ed30f7e9d5d197bbe8fa9f74d37852c71bb4a" target='_blank'>
              Enzyme family-centred approach identifies helicases as recurrent hemizygous tumour suppressor genes
              </a>
            </td>
          <td>
            Karolin Voßgröne, F. Favero, Krushanka Kashyap, F. G. Rodríguez-González, André Vidas Olsen, Xin Li, Balca R. Mardin, Joachim Weischenfeldt, Claus S. Sørensen
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a42363dd5b8cd99837d35c9de6b31b67925b7368" target='_blank'>
              Impaired TGF-β signaling via AHNAK family mutations elicits an esophageal cancer subtype with sensitivities to genotoxic therapy and immunotherapy
              </a>
            </td>
          <td>
            Z.-H. Mai, K. Luo, Xinye Wang, Xiuying Xie, Lanlan Pang, Hong Yang, Jing Wen, Jianhua Fu
          </td>
          <td>2024-09-05</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d59a35bbbd452d19f5eb530222e3178b220b08" target='_blank'>
              Tumour mutational burden: clinical utility, challenges and emerging improvements.
              </a>
            </td>
          <td>
            J. Budczies, D. Kazdal, Michael Menzel, Susanne Beck, K. Kluck, Christian Altbürger, C. Schwab, M. Allgäuer, A. Ahadova, Matthias Kloor, P. Schirmacher, Solange Peters, Alwin Krämer, P. Christopoulos, A. Stenzinger
          </td>
          <td>2024-08-27</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is characterized by its aggressive behavior and complex molecular heterogeneity, posing significant challenges for treatment and prognostication. This study offers a comprehensive analysis of ccRCC by leveraging both bulk and single-cell RNA sequencing data, with a specific aim to unravel the complexities of sphingolipid metabolism and the intricate dynamics within the tumor microenvironment (TME). By examining ccRCC samples sourced from public databases, our investigation delves deep into the genetic and transcriptomic landscape of this cancer type. Employing advanced analytical techniques, we have identified pivotal patterns in gene expression and cellular heterogeneity, with a special focus on the roles and interactions of various immune cells within the TME. Significantly, our research has unearthed insights into the dynamics of sphingolipid metabolism in ccRCC, shedding light on its potential implications for tumor progression and strategies for immune evasion. A novel aspect of this study is the development of a risk score model designed to enhance prognostic predictions for ccRCC patients, which is currently pending external validation to ascertain its clinical utility. Despite its contributions, the study is mindful of its limitations, including a reliance on observational data from public sources and a primary focus on RNA sequencing data, which may constrain the depth and generalizability of the findings. The study does not encompass critical aspects, such as protein expression, posttranslational modifications, and comprehensive metabolic profiles. Moreover, its retrospective design underscores the necessity for future prospective studies to solidify these preliminary conclusions. Our findings illuminate the intricate interplay between genetic alterations, sphingolipid metabolism, and immune responses in ccRCC. This research not only enhances our understanding of the molecular foundations of ccRCC but also paves the way for the development of targeted therapies and personalized treatment modalities. The study underlines the importance of cautious interpretation of results and champions ongoing research using diverse methodologies to thoroughly comprehend and effectively combat this formidable cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d290ea11babde7963cf17fade3be71816bf90d65" target='_blank'>
              Integrative analysis of bulk and single-cell RNA sequencing reveals sphingolipid metabolism and immune landscape in clear cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Dongdong Xie, Zhitao Han, Yu Wang, Haoyu Shi, Xiang Wu, Jiaqing Wu, Y. Dai
          </td>
          <td>2024-09-04</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
Oncogenic-addicted non-small cell lung cancer (NSCLC) has emerged as the most prevalent form of lung cancer, presenting a dynamic landscape in treatment modalities. Among these, epidermal growth factor receptor (EGFR)-mutant NSCLC remains the predominant oncogenic mutation, particularly prevalent in regions such as Asia and Latin America.


CASE PRESENTATION
This case study highlights the experience of a woman diagnosed with EGFR-sensitive (del exon 19) mutant NSCLC who demonstrated an extended duration of response (DOR) to third-generation EGFR-TKI therapy. Upon disease progression, detection of MET gene amplification prompted the addition of a selective MET inhibitor to the existing EGFR-TKI regimen, resulting in a complete response for the patient.


DISCUSSION/CONCLUSION
The molecular heterogeneity of this condition has significantly increased in complexity over recent years, marked by the identification of baseline co-alterations and development of a broad spectrum of resistance mechanisms post-EGFR tyrosine kinase inhibitor (TKI) therapy. This complexity poses a substantial challenge to clinicians. Despite the rapid advancement of targeted therapies and the implementation of treatment escalation through combination strategies, there remains an ongoing debate regarding which patients would benefit most from combination therapies, both in the initial treatment phase and in the setting of disease progression, particularly when off-target resistance mechanisms or co-alterations are identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aee7b0dfd69142858e9dd93edb7ec143688a294" target='_blank'>
              EGFR plus MET targeted therapies for overcoming treatment resistance in EGFR mutant NSCLC. A Case Report.
              </a>
            </td>
          <td>
            Maria F Martínez-Hernandez, L. Lara-Mejía, C. Izquierdo-Tolosa, L. Cabrera-Miranda, Oscar Arrieta
          </td>
          <td>2024-09-18</td>
          <td>Oncology research and treatment</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="137 Background: Resistance to RT remains a clinical challenge, given our limited understanding of the molecular pathways underpinning radioresistance. Here, we investigated for mechanisms linked to RR using paired experimental and clinico-molecular HNC and PCa datasets for discovery and validation of novel pathways that may be enriched in RR cancers. Methods: For experimental data, we generated RR HNC (FaDu, HK1) and PCa (22Rv1, DU145) cell lines following high-dose X-irradiation (90 Gy in 45 fr). Genotypic characterization of RR and wildtype (WT) cells was performed by genomic and transcriptomic sequencing using WES (100X) and RNAseq (Illumina, CA), respectively, followed by functional characterization by western blot (WB) and immunofluorescence (IF). For clinico-molecular data, we utilized two prospectively recruited RT cohorts, consisting of 311 patients with HNC (n=158) and PCa (n=153). Tumor transcriptomes were profiled using RNAseq and Affymetrix ST array (ThermoFisher, CA), respectively. Comparative analyses of molecular profiles were first performed between RR and WT in vitro models to identify dysregulated pathways that were linked to radioresistance, and subsequently tested for association with disease-free survival (DFS) in the clinical cohorts. Results: Comparative genomic analyses between RR and WT HNC and PCa models revealed an abundance of acquired mutations in the RR compared with WT models, with a higher mutation count observed in PCa than HNC cells (SNV counts: 2,158 [DU145-RR] and 1,387 [22Rv1-RR] vs 396 [FaDu-RR] and 25 [HK1-RR], P<0.0001). Mutational signature analyses indicated a common enrichment of DNA mismatch repair-related mutational signatures (SBS3, SBS14, and SBS21) across the RR HNC and PCa models. Transcriptomic profiling revealed an upregulation of the BAHD1 gene, which is involved in heterochromatin formation, across the 4 RR cell lines, corroborated by an enrichment of heterochromatin-related genesets. These results were consistent with functional characterization of RR versus WT cells by WB and IF indicating increased DNA repair capacity and heterochromatin response post-4 Gy irradiation. We further confirmed the dependency of the RR phenotype on the heterochromatin response with BAHD1 knockdown, resulting in reversal of these cellular responses. Finally, our results were supported by survival analyses indicating an inferior DFS post-RT in patients with HNC and PCa harboring a higher expression of heterochromatin-related geneset (HNC: HRhigh vs low 1.51 ; PCa: HRhigh vs low 1.37 ). Conclusions: Herein, by leveraging on paired experimental and clinico-molecular datasets, we have uncovered a novel BAHD1-dependent heterochromatin response that underpins resistance to RT in HNC and PCa, which may be amendable to systemic agents targeting chromatin remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea1270e10cc26b39aabc2ffd1632147257d9664" target='_blank'>
              Analysis of heterochromatin remodeling as a mechanism of radiotherapy (RT) resistance through molecular profiling of radioresistant (RR) head and neck (HNC) and prostate cancers (PCa).
              </a>
            </td>
          <td>
            Evelyn Tan, Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, Dennis J J Poon, Dewi Susanti, E. Yeo, Pek L Chu, Melvin L K Chua
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="microRNAs (miRNAs), tiny, non-coding RNA molecules, fine-tune the expression of target genes through interacting with mRNAs. These miRNAs are involved in a wide range of biological processes, encompassing cell division, death, blood cell production, and tumor development. When these miRNAs become dysfunctional, they can promote the invasion and spread of cancer cells in various human malignancies, including leukemia. Acute lymphoblastic leukemia (ALL), the preeminent malignancy affecting children, is a blood cancer marked by the uncontrollable growth of immature lymphoid cells that displace healthy blood precursors in the bone marrow. Despite a decline in ALL mortality rates over the past two decades, a significant proportion of deaths still results from a lack of effective diagnostic and prognostic markers that can guide treatment decisions and overcome drug resistance. The analysis of miRNA expression patterns in ALL could lead to more precise disease classification, earlier diagnosis, and better prognostic outcomes in the near future. The connection between miRNA dysfunction and the biology of ALL suggests that these molecules could represent promising therapeutic targets. Therefore, this review delves into the regulatory mechanisms of miRNAs in pediatric ALL, exploring how miRNA-based diagnostic, prognostic, and therapeutic strategies offer unique advantages and hold promise for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b51d3730adcae846d16d92db76f51f153c5c1d0" target='_blank'>
              Advancements in the Regulatory Role of microRNAs in Childhood Acute Lymphoblastic Leukemia: Mechanisms and Clinical Implications
              </a>
            </td>
          <td>
            Wei Deng
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background: Genetic studies of ovarian cancer (OC) have historically focused on BRCA1/2 mutations, lacking other studies of homologous recombination repair (HRR). Poly (ADP-ribose) polymerase inhibitors (PARPi) exploit synthetic lethality to significantly improve OC treatment outcomes, especially in BRCA1/2 deficiency patients. Objectives: Our study aims to construct a mutation map of HRR genes in OC and identify factors influencing the efficacy of PARPi. Design: A retrospective observational analysis of HRR gene variation data from 695 OC patients from March 2019 to February 2022 was performed. Methods: The HRR gene variation data of 695 OC patients who underwent next-generation sequencing (NGS) in the First Affiliated Hospital of Zhengzhou University were retrospectively collected. Clinical data on the use of PARPi in these patients were also gathered to identify factors that may interfere with the efficacy of PARPi. Results: Out of 127 pathogenic variants in the BRCA1/2 genes, 104 (81.9%) were BRCA1 mutations, and 23 (18.1%) were BRCA2 mutations. Among the 59 variants of uncertain significance (VUS), 20 (33.9%) were BRCA1, while 39 (66.1%) were BRCA2 mutations. In addition to BRCA1/2, HRR gene results showed that 9 (69%) of 13 were HRR pathway pathogenic variants; and 16 (1.7%) of 116 VUS were Food and Drug Administration (FDA)-approved mutated HRR genes. Notably, the treatment regimen significantly influenced the effectiveness of PARPi, especially when using first-line maintenance therapy, leading to enhanced progression-free survival (PFS) compared to alternative protocols. Conclusion: Focusing on HRR gene mutations and supporting clinical research about PARPi in OC patients is crucial for developing precision treatment strategies and enhancing prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d41e2645926ac18cf48bd055b15a6838f0eb8b2d" target='_blank'>
              Genetic landscape of homologous recombination repair and practical outcomes of PARPi therapy in ovarian cancer management
              </a>
            </td>
          <td>
            Mengyu Chen, Ningjing Lei, Ruixia Guo, Liping Han, Qinghe Zhao, Yang Zhao, Luojie Qiu, Fengling Wu, Shan Jiang, Ningyao Tong, Kunmei Wang, Siyu Li, Lei Chang
          </td>
          <td>2024-01-01</td>
          <td>Therapeutic Advances in Medical Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Multiple myeloma is the second most hematological cancer. RUVBL1 and RUVBL2 form a subcomplex of many chromatin remodeling complexes implicated in cancer progression. As an inhibitor specific to the RUVBL1/2 complex, CB-6644 exhibits remarkable anti-tumor activity in xenograft models of Burkitt’s lymphoma and multiple myeloma (MM). In this work, we defined transcriptional signatures corresponding to CB-6644 treatment in MM cells and determined underlying epigenetic changes in terms of chromatin accessibility. CB-6644 upregulated biological processes related to interferon response and downregulated those linked to cell proliferation in MM cells. Transcriptional regulator inference identified E2Fs as regulators for downregulated genes and MED1 and MYC as regulators for upregulated genes. CB-6644-induced changes in chromatin accessibility occurred mostly in non-promoter regions. Footprinting analysis identified transcription factors implied in modulating chromatin accessibility in response to CB-6644 treatment, including ATF4/CEBP and IRF4. Lastly, integrative analysis of transcription responses to various chemical compounds of the molecular signature genes from public gene expression data identified CB-5083, a p97 inhibitor, as a synergistic candidate with CB-6644 in MM cells, but experimental validation refuted this hypothesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aff9ee9c96de0ec030a3d9d628090554b2bbd4f2" target='_blank'>
              Molecular Signatures of CB-6644 Inhibition of the RUVBL1/2 Complex in Multiple Myeloma
              </a>
            </td>
          <td>
            Weijun Yi, Sebastian A. Dziadowicz, Rachel S. Mangano, Lei Wang, Joseph C. McBee, Steven M. Frisch, Lori A. Hazlehurst, Donald A Adjeroh, Gangqing Hu
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8eb55432a81ae9f11fc653e3722684f5e8a7ed1" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The detection of mitochondrial DNA (mtDNA) mutations in single cells holds considerable potential to define clonal relationships coupled with information on cell state in humans. Previous methods focused on higher heteroplasmy mutations that are limited in number and can be influenced by functional selection, introducing biases for lineage tracing. Although more challenging to detect, intermediate to low heteroplasmy mtDNA mutations are valuable due to their high diversity, abundance, and lower propensity to selection. To enhance mtDNA mutation detection and facilitate fine-scale lineage tracing, we developed the single-cell Regulatory multi-omics with Deep Mitochondrial mutation profiling (ReDeeM) approach, an integrated experimental and computational framework. Recently, some concerns have been raised about the analytical workflow in the ReDeeM framework. Specifically, it was noted that the mutations detected in a single molecule per cell are enriched on edges of mtDNA molecules, suggesting they resemble artifacts reported in other sequencing approaches. It was then proposed that all mutations found in one molecule per cell should be removed. We detail our error correction method, demonstrating that the observed edge mutations are distinct from previously reported sequencing artifacts. We further show that the proposed removal leads to massive elimination of bona fide and informative mutations. Indeed, mutations accumulating on edges impact a minority of all mutation calls (for example, in hematopoietic stem cells, the excess mutations on the edge account for only 4.3%-7.6% of the total). Recognizing the value of addressing edge mutations even after applying consensus correction, we provide an additional filtering option in the ReDeeM-R package. This approach effectively eliminates the position biases, leads to a mutational signature indistinguishable from bona fide mitochondrial mutations, and removes excess low molecule high connectedness mutations. Importantly, this option preserves the large majority of unique mutations identified by ReDeeM, maintaining the ability of ReDeeM to provide a more than 10-fold increase in variant detection compared to previous methods. Additionally, the cells remain well-connected. While there is room for further refinement in mutation calling strategies, the significant advances and biological insights provided by the ReDeeM framework are unique and remain intact. We hope that this detailed discussion and analysis enables the community to employ this approach and contribute to its further development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94914bd4d265668ea63fab9b4a1aa10d1231c91c" target='_blank'>
              Robustness and reliability of single-cell regulatory multi-omics with deep mitochondrial mutation profiling
              </a>
            </td>
          <td>
            Chen Weng, Jonathan S. Weissman, V. Sankaran
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) is a common leukemia, mainly affecting the elderly. Originating in the bone marrow, CLL involves the accumulation of B lymphocytes and progresses slowly, though 50-60% of patients will require therapy. At diagnosis, the presence of p53 protein aberrations, such as 17p deletion and TP53 mutation, arises in approximately one out of 10 patients. Even in the era of targeted therapies, these aberrations remain the most important prognostic factors. Current guidelines favor continuous BTK inhibitor therapy in patients with CLLp53, though adverse events and drug resistance may lead to discontinuation. Herein, we discuss the effects of B-cell receptor and BCL-2 inhibition, as well as the role of the immune system, in two elderly CLLp53 patients with prolonged responses to different therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d74118b8b647e856bb83b0c75e1871e781ba904" target='_blank'>
              Holding the therapy in CLLp53: mechanisms to achieve durable responses.
              </a>
            </td>
          <td>
            Rodrigo Cantera, Tatiana Fernández-Barge, Jon Salmanton-García, L. Yáñez
          </td>
          <td>2024-08-09</td>
          <td>Anti-cancer drugs</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="The fruit fly Drosophila melanogaster has long served as a model organism for studying genetic and developmental processes. The Drosophila eye, with its well-characterized structure and genetic tractability, is a prime system for exploring the principles of differentiation, cell type specification, and neurogenesis. Recent advances in single-cell RNA sequencing (scRNA-seq) technology have revolutionized our ability to study gene expression at the individual cell level, offering unprecedented insights into cellular heterogeneity within complex tissues. This review highlights the application of scRNA-seq to the adult Drosophila eye, emphasizing its role in delineating the transcriptomic landscape of its diverse cell types. By profiling individual cells, researchers have identified novel marker genes for all major cell types in the eye, providing a detailed cellular atlas and uncovering previously unknown cellular subtypes. The review also discusses the broader implications of these methodologies for developmental biology, the importance of functional validation of identified markers, and the potential of integrating scRNA-seq data with other omics approaches. Overall, single-cell genomics of the Drosophila eye sets a new standard for cellular resolution, offering valuable insights into the complexities of gene regulation and cellular diversity, with significant implications for understanding development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2298d38c02aec31b5a1d78d27d7720f3d07a1065" target='_blank'>
              Advances in understanding the cellular composition and molecular signatures of the adult Drosophila eye through single-cell RNA sequencing
              </a>
            </td>
          <td>
            K. K. B. Raja, Blessie Pradeeka Gudelly
          </td>
          <td>2024-08-30</td>
          <td>Magna Scientia Advanced Research and Reviews</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16db7fe8f3bc4329cf344c224d8e623bae436c6d" target='_blank'>
              Elucidating the pan-oncologic landscape of S100A9: prognostic and therapeutic corollaries from an integrative bioinformatics and Mendelian randomization analysis
              </a>
            </td>
          <td>
            Yingying Chen, Zixuan Wu, Xingxing Yi
          </td>
          <td>2024-08-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Due to the significant morbidity and mortality of hemoglobinopathies, curative options have long been pursued. The overall goal of gene therapy is to modify a patient's own hematopoietic stem cells to overcome the deleterious effects of the underlying genetic defect by gene addition, gene editing, or gene silencing. Gene addition incorporates genes with superior function than the abnormal gene; gene editing takes advantage of molecular tools such as zinc finger proteins, Transcription Activator-Like Effector Nucleases and Clustered Regularly Interspaced Short Palindromic Repeats coupled with Cas9 proteins (CRISPR-Cas9) which allow for sequence-specific breaks in DNA that disrupt gene function; and gene silencing suppresses gene expression by interference with mRNA transcription/protein translation or epigenetic modification. The majority of gene therapy strategies for hemoglobinopathies have targeted erythroid-specific BCL11A, a major regulator of fetal hemoglobin repression at the gamma-globin locus, in the normal fetal-to-adult hemoglobin switch that occurs shortly after birth. Other goals have involved the incorporation of anti-sickling globins, such as βT87Q or βAS3. Landmark clinical trials of gene therapy in transfusion-dependent thalassemia and sickle cell disease have shown remarkable efficacy and acceptable safety and culminated in recent regulatory approvals of gene therapy for both diseases in Europe and the United States.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f812c9e54de47feca0c8615f3dda20f25f85b41" target='_blank'>
              A Review of Gene Therapies for Hemoglobinopathies.
              </a>
            </td>
          <td>
            Boubini Jones-Wonni, Amar H Kelkar, Maureen O Achebe
          </td>
          <td>2024-08-15</td>
          <td>Hemoglobin</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) presents a formidable clinical challenge owing to its aggressive progression and resistance to conventional therapies. A key driver of NEPC is the overexpression of MYCN, a well-established oncogene associated with neuroendocrine tumors. However, efforts to directly inhibit the N-Myc protein encoded by this gene have resulted in limited success, thereby hindering therapeutic advancements. To overcome this obstacle, we conducted unbiased genome-wide screening using isogenic prostate cancer cell lines to identify the synthetic vulnerabilities of MYCN. Among the identified candidates, NEUROG2 emerged as a significant candidate. Neurog2 is a proneural transcription factor (PTF) known for its role in developmental processes and trans-differentiation of adult cells. Our findings demonstrate that Neurog2 depletion does not affect non-malignant cells, but significantly suppresses the growth of MYCN-overexpressing cells and tumors in orthotopic NEPC models. Furthermore, our observations indicate that the Neurog2-mediated regulation of PTFs can facilitate NEPC development. Thus, targeting Neurog2 holds promise as an effective therapeutic strategy for MYCN-overexpressing NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6178b90a3b078d38756c518ec592d0bd4f1191a1" target='_blank'>
              A novel role for Neurog2 in MYCN driven neuroendocrine plasticity of prostate cancer
              </a>
            </td>
          <td>
            Prachi Walke, Jared D.W. Price, Frederick S. Vizeacoumar, Nickson Joseph, Vincent Maranda, Bari Chowdhury, Jay Patel, Yue Zhang, He Dong, Lara New, Ashtalakshmi Ganapathysamy, Lihui Gong, Hussain Elhasasna, Kalpana K Bhanumathy, Yuliang Wu, A. Freywald, Anand Krishnan, Franco J Vizeacoumar
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Current treatments for KRAS-mutant colorectal cancers (CRCs) are often limited by cellular plasticity and rewiring responses. Here we describe a promising therapeutic strategy that simultaneously targets epigenetic and oncogenic signals. Specifically, we show that inhibitors of the histone methyltransferase, EZH2, synergize with various RAS pathway inhibitors and promote dramatic tumor regression in vivo. Together these agents cooperatively suppress WNT-driven transcription and drive CRCs into a more differentiated cell state by inducing the Groucho/TLE corepressor, TLE4, along with a network of WNT pathway inhibitors and intestinal differentiation proteins. However, these agents also induce the pro-apoptotic protein BMF, which subsequently kills these more differentiated cells. Accordingly, cell death can be prevented by activating β-catenin, blocking differentiation, or by ablating BMF expression. Collectively, these studies reveal a new therapeutic approach for treating KRAS-mutant CRCs and illustrate a critical convergence of EZH2 and RAS on oncogenic WNT signals, intestinal differentiation, and apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47a75ff581332b9384e613ce0d1c80e31add042e" target='_blank'>
              Epigenetic and oncogenic inhibitors cooperatively drive differentiation and kill KRAS-mutant colorectal cancers.
              </a>
            </td>
          <td>
            P. Loi, Amy E. Schade, Carrie L Rodriguez, Anjana Krishnan, N. Perurena, Van T M Nguyen, Yilin Xu, Marina Watanabe, Rachel A Davis, Alycia Gardner, Natalie F Pilla, Kaia Mattioli, Olesja Popow, Nuray Gunduz, T. Lannagan, Samantha Fitzgerald, Ewa T Sicinska, Jia-Ren Lin, William Tan, L. Brais, Kevin Haigis, M. Giannakis, K. Ng, Sandro Santagata, Kristian Helin, Owen J. Sansom, K. Cichowski
          </td>
          <td>2024-08-12</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Aberrant gene expression patterns in acute myeloid leukemia (AML) with balanced chromosomal translocations are often associated with dysregulation of epigenetic modifiers. The AML1/ETO (RUNX1/MTG8) fusion protein, caused by the translocation (8;21)(q22;q22), leads to the epigenetic repression of its target genes. We aimed in this work to identify critical epigenetic modifiers, on which AML1/ETO-positive AML cells depend on for proliferation and survival using shRNA library screens and global transcriptomics approaches. Using shRNA library screens, we identified 41 commonly depleted genes in two AML1/ETO-positive cell lines Kasumi-1 and SKNO-1. We validated, genetically and pharmacologically, DNMT1 and ATR using several AML1/ETO-positive and negative cell lines. We also demonstrated in vivo differentiation of myeloblasts after treatment with the DNMT1 inhibitor decitabine in a patient with an AML1/ETO-positive AML. Bioinformatic analysis of global transcriptomics after AML1/ETO induction in 9/14/18-U937 cells identified 973 differentially expressed genes (DEGs). Three genes (PARP2, PRKCD, and SMARCA4) were both downregulated after AML1/ETO induction, and identified in shRNA screens. In conclusion, using unbiased shRNA library screens and global transcriptomics, we have identified several driver epigenetic regulators for proliferation in AML1/ETO-positive AML. DNMT1 and ATR were validated and are susceptible to pharmacological inhibition by small molecules showing promising preclinical and clinical efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4848d5afaf1e1f3f5c753596a2ed8d1e491ca9f0" target='_blank'>
              Identification of epigenetic modifiers essential for growth and survival of AML1/ETO-positive leukemia.
              </a>
            </td>
          <td>
            J. Duque-Afonso, P. Veratti, Usama-Ur Rehman, Heike Herzog, J. Mitschke, G. Greve, Julian Eble, Bettina Berberich, Johanna Thomas, Milena Pantic, Miguel Waterhouse, Gaia Gentile, Olaf Heidenreich, Cornelius Miething, Michael Lübbert
          </td>
          <td>2024-08-15</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ABSTRACT The human T-cell lymphotropic virus type 1 (HTLV-1) is a single-stranded positive-sense RNA virus that belongs to the Retroviridae family, genus Deltaretro, and infects approximately five to 10 million people worldwide. Although a significant number of individuals living with HTLV-1 remain asymptomatic throughout their lives, some develop one or more severe clinical conditions, such as HTLV-1-associated myelopathy/tropical spastic paraparesis (HAM/TSP), a progressive and debilitating disease, and/or a subtype of non-Hodgkin’s lymphoma with a more threatening course known as adult T-cell leukemia/lymphoma (ATLL). Moreover, current therapeutic options are limited and focus primarily on treating symptoms and controlling viral latency. CRISPR-Cas9 gene editing is proposed as a promising tool to address the intricate links associated with HTLV-1. By targeting or silencing key genes during initial infection and dysregulating immune signaling pathways, CRISPR-Cas9 offers potential intervention opportunities. In this review, we address the therapeutic potential of CRISPR-Cas9 gene editing, as well as examine the primary mechanisms involved in editing potential target genes and discuss the existing evidence in the current scientific literature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa8304d3f00cdbae60023f6fc20b04a24438fa0" target='_blank'>
              Novel approaches for HTLV-1 therapy: innovative applications of CRISPR-Cas9
              </a>
            </td>
          <td>
            Wilson Domingues, V. A. Folgosi, S. Sanabani, Pedro Domingos Leite Junior, Tatiane Assone, J. Casseb
          </td>
          <td>2024-08-26</td>
          <td>Revista do Instituto de Medicina Tropical de São Paulo</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Li-Fraumeni syndrome (LFS) is a hereditary cancer predisposition syndrome associated with a highly penetrant cancer spectrum characterized by germline TP53 mutations. We characterized the first LFS zebrafish hotspot mutants, tp53 R217H and R242H (human R248H and R273H), and found these mutants exhibit partial-to-no activation of p53 target genes, have a defective G1 cell-cycle checkpoint, and are resistant to apoptosis. Spontaneous tumor development histologically resembling human sarcomas developed as early as 6 months. tp53 R242H mutants had a higher lifetime tumor incidence compared to tp53 null and R217H mutants, suggesting it is a more aggressive mutation. We observed mutation-specific tumor phenotypes across tp53 mutants with associated diverse transcriptomic and DNA methylome profiles, impacting metabolism, cell signalling, and biomacromolecule synthesis and degradation. These tp53 zebrafish mutants demonstrate fidelity to their human counterparts and provide new insights into underlying tumorigenesis mechanisms and kinetics, which will guide targeted therapeutics for LFS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe84da849b9bf9707014c32841afe469ca3c5e99" target='_blank'>
              tp53 R217H and R242H Mutant Zebrafish Exhibit Dysfunctional p53 Hallmarks and Recapitulate Li-Fraumeni Syndrome Phenotypes
              </a>
            </td>
          <td>
            Kim Kobar, Lissandra Tuzi, Erin Burnley, Jennifer A. Fiene, Kristianne J. C. Galpin, C. Midgen, B. Laverty, Vallijah Subasri, Timmy T. Wen, Martin Hirst, M. Moksa, A. Carles, Qi Cao, A. Shlien, David Malkin, Sergey V. Prykhozhij, Jason N Berman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="OBJECTIVES
The TP53 mutation, a prevalent tumor suppressor gene alteration, is linked to chemotherapy resistance, increased relapse rates and diminished overall survival (OS) in acute myeloid leukemia (AML) patients.


METHODS
In this study, we characterize the TP53 mutation phenotypes across various AML cohorts utilizing The Cancer Genome Atlas (TCGA) data. We devised a TP53-related prognostic signature derived from differentially expressed genes between mutated and wild-type TP53 AML specimens. In-depth analyses were conducted, encompassing genetic variation, immune cell infiltration and prognostic stratification.


RESULTS
A six-gene TP53-related signature was established using least absolute shrinkage and selection operator (LASSO)-Cox regression, demonstrating robust prognostic predictability. This signature exhibited strong performance in both the OHSU validation cohorts, an independent Gene Expression Omnibus (GEO) validation cohort (GSE71014) and proved by results of the in vivo experiment. Finally, we used single cell database (GSE198681) to observe the characteristics of these six genes.


DISCUSSION
Our study may facilitate the development of efficacious therapeutic approaches and provide a novel idea for future research. Conclusion: The TP53-related signature and pattern hold the potential to refine prognostic stratification and underscore emerging targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57809ff1c4369159931bf563a0e583afd0ffd0c" target='_blank'>
              Elucidating the immune landscape and potential prognostic model in acute myeloid leukemia with TP53 mutation.
              </a>
            </td>
          <td>
            Gelan Zhu, Jiayi Cai, Wanbin Fu, Yue Sun, Ting Wang, Hua Zhong
          </td>
          <td>2024-09-27</td>
          <td>Hematology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ab10bbc0562e400afe55f02a7e1bbeddfbcff21" target='_blank'>
              Reinvigoration of cytotoxic T lymphocytes in microsatellite instability-high colon adenocarcinoma through lysosomal degradation of PD-L1
              </a>
            </td>
          <td>
            Dan Liu, Jin Yan, Fang Ma, Jingmei Wang, Siqi Yan, Wangxiao He
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9a93850c21d9d18806483d79ec8a05dda4a648d" target='_blank'>
              Treatment resistance to melanoma therapeutics on a single cell level
              </a>
            </td>
          <td>
            Lijun Yao, B. Krasnick, Ye Bi, Sunantha Sethuraman, S. Goedegebuure, A. Weerasinghe, Christopher Wetzel, Qingsong Gao, A. Oyedeji, Jacqueline L Mudd, Matthew A. Wyczalkowski, M. Wendl, Li Ding, Ryan C. Fields
          </td>
          <td>2024-09-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Dysfunction of epigenetic modulators, such as the SWI/SNF complex, is a wide-spread but relatively ill-defined feature of a broad spectrum of cancer entities. Among SWI/SNF-mutant entities, SMARCB1-deficient cancers, such as the highly aggressive Epithelioid Sarcoma (EpS), are characterized by this genetic event in an otherwise rather silent mutational landscape. This renders EpS an ideal model to study how epigenetic reprogramming by a single mutation can contribute to tumorigenesis. Hence, to characterize and compare the function of the SMARCB1-deficient, residual and the physiological SWI/SNF complex in cancer, we generated a panel of SMARCB1 re-expressing EpS cell lines and employed a functional multi-omics approach. Here, we show that SWI/SNF holds canonical characteristics of both tumor-suppressors and proto-oncogenes due to its multi-faceted role in the regulation of the epigenome. Our data indicates that the loss of SMARCB1 causes an overall loss of SWI/SNF chromatin affinity at cis-regulatory enhancer elements, inducing a preference for uncontrolled proliferation and cell cycle progression as opposed to development and differentiation. We further demonstrate that EpS cell lines depend on residual SWI/SNF action to maintain clonogenicity and proliferation. Consequently, EpS cell lines exhibit markedly increased sensitivity to pharmacological inhibition of the residual SWI/SNF when compared with SWI/SNF-proficient cancer entities. Collectively, our results from the EpS model shed new light on how a single mutation can rewire the pleiotropic effects of an epigenetic master regulator and provide inroads for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ba81f1fa7d5881ab59ab9ddb07d9a1d9290af17" target='_blank'>
              Loss of SMARCB1 evokes targetable epigenetic vulnerabilities in Epithelioid Sarcoma
              </a>
            </td>
          <td>
            Jia Xiang Jin, Fabia Fuchslocher, Martha J. Carreño-Gonzalez, Felina Zahnow, A. K. Ceranski, Rainer Will, Dominic Helm, Felix Bestvater, A. Banito, R. Imle, S. Ohmura, F. Cidre-Aranaz, T. G. Grünewald
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The effectiveness of drug treatments is profoundly influenced by individual responses, which are shaped by gene expression variability, particularly within pharmacogenes. Leveraging single-cell RNA sequencing (scRNA-seq) data, our study explores the extent of expression variability among pharmacogenes in a wide array of cell types across eight different human tissues, shedding light on their impact on drug responses. Our findings broaden the established link between variability in pharmacogene expression and drug efficacy to encompass variability at the cellular level. Moreover, we unveil a promising approach to enhance drug efficacy prediction. This is achieved by leveraging a combination of cross-cell and cross-individual pharmacogene expression variation measurements. Our study opens avenues for more precise forecasting of drug performance, facilitating tailored and more effective treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b12a7bcbec6921ec291037e369d1c317c0110c" target='_blank'>
              Deciphering single-cell gene expression variability and its role in drug response.
              </a>
            </td>
          <td>
            Sizhe Liu, Liang Chen
          </td>
          <td>2024-09-15</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Metastatic breast cancer remains largely incurable, and the mechanisms underlying the transition from primary to metastatic breast cancer remain elusive. We analyzed the complex landscape of primary and metastatic breast cancer using scRNA-seq data from twenty-three female patients with either primary or metastatic disease to elucidate the genetic and molecular mechanisms underlying changes in the metastatic tumor ecosystem. We identify specific subtypes of stromal and immune cells critical to forming a pro-tumor microenvironment in metastatic lesions, including CCL2+ macrophages, cytotoxic T cells with an exhausted gene signature, and FOXP3+ regulatory T cells. Analysis of cell-cell communication highlights a marked decrease in tumor-immune cell interactions in metastatic tissues, likely strengthening the immunosuppressive microenvironment. In contrast, primary breast cancer samples displayed increased activation of the TNF- signaling pathway via NF-kB, indicating a potential therapeutic target. Our study comprehensively characterizes the transcriptional landscape encompassing primary and metastatic breast cancer">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f83bcd576506ee2a8774cb0cfd1433b6936e5b1" target='_blank'>
              Single-cell RNA sequencing reveals immunosuppressive pathways associated with metastatic breast cancer
              </a>
            </td>
          <td>
            F. Ozmen, T. Y. Ozmen, A. Ors, M. Janghorban, M. J. Rames, X. Li, F. Behbod, G. B. Mills, H. Mohammed
          </td>
          <td>2024-09-26</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Alexander Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii-DerIda Chen, C. Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="FMS-like tyrosine kinase 3 (FLT3) internal tandem duplication (ITD) mutations in acute myeloid leukemia (AML) are associated with poor prognosis and therapy resistance. This study aimed to demonstrate that inhibiting the deubiquitinating enzymes ubiquitin-specific peptidase 14 (USP14) and ubiquitin C-terminal hydrolase L5 (UCHL5) (USP14/UCHL5) with b-AP15 or the organogold compound auranofin (AUR) induces apoptosis in the ITD-transformed human leukemia cell line MV4-11 and mononuclear leukocytes derived from patients with FLT3-ITD-positive AML. This study included patients diagnosed with AML at Tokyo Medical and Dental University Hospital between January 2018 and July 2024. Both treatments blocked downstream FLT3 pathway events, with the effects potentiated by USP14 knockdown. Both treatments inhibited FLT3 deubiquitination via K48 and disrupted translation initiation via 4EBP1, a downstream FLT3 target. FLT3 was downregulated in the leukemic cells, with the associated activation of stress-related MAP kinase pathways and increased NF-E2-related factor 2. Furthermore, the overexpression of B-cell lymphoma-extra-large and myeloid cell leukemia-1 prevented the cell death caused by b-AP15 and AUR. These results suggest that inhibiting USP14/UCHL5, which involves multiple regulatory mechanisms, is a promising target for novel therapies for treatment-resistant FLT3-ITD-positive AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/540b8cc6a2106b3e135bb1081baf0b46a1a8976c" target='_blank'>
              Targeting USP14/UCHL5: A Breakthrough Approach to Overcoming Treatment-Resistant FLT3-ITD-Positive AML
              </a>
            </td>
          <td>
            Ayako Nogami, Hideki Jose Amemiya, Hiroki Fujiwara, Yoshihiro Umezawa, Shuji Tohda, Toshikage Nagao
          </td>
          <td>2024-09-26</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54370344e70d12f9e093a0f96e506dd839c6df0" target='_blank'>
              Deciphering breast cancer dynamics: insights from single-cell and spatial profiling in the multi-omics era
              </a>
            </td>
          <td>
            Xin Xiong, Xin Wang, Cui-Cui Liu, Zhi-Ming Shao, Ke-Da Yu
          </td>
          <td>2024-09-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite the success of CRISPR/Cas9 in inducing DNA double-strand breaks (DSBs) for genome editing, achieving targeted recombination in somatic cells remains challenging, particularly at recombination cold spots like the Tomato Mosaic Virus (ToMV) resistance locus in Solanum lycopersicum. We investigated the potential of CRISPR/Cas9-induced targeted recombination in somatic cells to overcome linkage drag surrounding the ToMV locus. We employed two strategies: first, inducing DSBs in both alleles of F1 tomato seedlings to promote non-homologous end joining (NHEJ) and homology-directed repair (HDR); second, targeting a single allele in a heterozygous background to induce HDR in seedlings. CRISPR/Cas9 activity was confirmed in F₁ seedlings by detecting NHEJ-mediated mutations at the target sites in ToMV. We developed a bioinformatics pipeline to identify targeted recombinants by analyzing single nucleotide polymorphisms (SNPs) between parental haplotypes, allowing precise tracking of SNP variations. A two-dimensional pooling strategy was employed to distinguish genuine recombination events from PCR artifacts. Despite these advances and the active CRISPR/Cas9 system in F1 progeny, no increase in recombination frequency was observed compared to wild-type plants. We extended our research to protoplasts to assess whether CRISPR/Cas9 could induce targeted recombination under different cellular conditions at the same locus. Consistent with our findings in F1 plants, we observed no increase in recombinant patterns compared to wild-type controls in protoplasts. Our findings suggest that CRISPR/Cas9-induced DSBs are insufficient to break the genetic linkage at the ToMV locus on chromosome 9 in recombination cold spots within somatic cells. Article Summary This research targets plant biologists and geneticists interested in enhancing plant breeding techniques. The study used CRISPR/Cas9 technology to induce DNA breaks in tomato plants. It specifically targeted the Tomato Mosaic Virus (ToMV) resistance gene, which resists natural recombination. The aim was to induce genetic recombination via CRISPR/Cas9. The highly active CRISPR/Cas9 system did not increase the expected genetic changes, indicating challenges in achieving targeted recombination. These findings highlight the challenges in breaking genetic linkages in specific genome regions using current CRISPR methods. These findings are relevant for developing techniques for targeted recombination in plant breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0b58aaaa05e1e2ef0cf4dbd56fab2aa20c1e65" target='_blank'>
              CRISPR/Cas9-induced breaks are insufficient to break linkage drag surrounding the ToMV locus of Solanum lycopersicum
              </a>
            </td>
          <td>
            Jillis Grubben, Gerard Bijsterbosch, Burak Aktürk, Richard G.F. Visser, Henk J. Schouten
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Mutations in DNA are a known cause of cancer. However, they become oncogenic only under specific cellular conditions and within certain tissue contexts. Tissue injury often leads to the aberrant activation of developmental pathways, eroding markers of cellular differentiation and giving rise to transient cellular states. These states resemble developmental progenitors, characterized by higher transcriptional plasticity. Oncogenes, particularly KRAS, co-opt cellular plasticity by acting on tissue-specific enhancers, preventing the resolution of acinar plasticity and maintaining the pancreas in a pro-inflammatory phase that progresses to cancer1,2. However, the specific role of tissue-specific enhancers in pancreas regeneration and cancer initiation remains unexplored, and strategies to target these regenerative processes are yet to be developed. Our laboratory investigates non-mutational mechanisms of cancer, with a special interest in the intersection of developmental biology and cancer, aiming to elucidate the processes of cellular plasticity and cancer initiation. Recent studies have shown that acinar cells possess remarkable cellular plasticity, giving rise to a diverse population of cellular states in pancreas regeneration and cancer contexts3-5. We have discovered that the transcription factor Sox4 is necessary for the specification of tuft cells and gastric metaplasia found in chronic pancreatitis and in pancreatic intraepithelial lesions (PanINs). Moreover, we have shown that cellular plasticity has a non-cell autonomous protective effect; losing Sox4 and cellular heterogeneity in PanINs results in a fibrotic and immunosuppressive tumor microenvironment and disease progression. Our results demonstrated that targeting cellular plasticity accelerates or prevents cancer initiation through autonomous and non-autonomous cellular processes6. In this study, we leverage a comprehensive resource of ncRNAs transcribed from enhancer elements of pancreas development, together with their inferred target genes and correlations with clinical outcomes. We focused on an ncRNA associated with the risk of pancreatic cancer. To study its function, we developed two novel mouse strains for tissue-specific deletion, enabling precise manipulation and study of its role in the pancreas and other tissues. Our phenotypic analysis demonstrates that the ncRNA is activated following tissue injury and is essential for developing PanINs. Moreover, we show that it regulates cell-cell communication after tissue injury. This mechanism operates independently of cellular plasticity but significantly creates a pro-inflammatory environment that favors cancer development. Our results demonstrate that enhancer elements are actionable targets to regulate cell fate decisions in regeneration and cancer. Additionally, KRAS-induced cancer initiation depends on developmental regulatory elements, opening the window to develop prophylactic interventions to delay tumor onset.
 Citation Format: Luis Arnes. Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/698dc456f639e681131533a5b64aac4181ad1d22" target='_blank'>
              Abstract C059: Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Luis Arnes
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most prevalent malignant tumors of the digestive system globally, with a particularly high incidence recorded in China due to the prevalence of chronic hepatitis virus infections. Recent developments in single-cell RNA sequencing (scRNA-seq) technology have provided new perspectives and approaches for cancer research, particularly showing great potential in understanding the heterogeneity of HCC. scRNA-seq technology, through detailed gene expression analysis at the single-cell level, reveals the cellular heterogeneity of hepatocellular carcinoma, identify key drivers of tumor progression, and elucidates the complex features of the tumor microenvironment. Such insights are pivotal for decoding the underlying mechanisms of hepatocellular carcinoma, thereby guiding the development of precise therapeutic strategies and personalized treatments. Furthermore, identifying key drivers of tumor progression and analyzing the gene expression characteristics of the surrounding microenvironment at single-cell resolution is expected to provide clues for developing new therapeutic strategies. Therefore, this article aims to provide a systematic overview of the fundamental principles of scRNA-seq, review its progress in HCC research, and explore the challenges and future directions in this field to offer researchers a comprehensive perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b441257e4cdb8264be1952074191341186a913" target='_blank'>
              Research on single-cell transcriptomics in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Nauman Shabbir
          </td>
          <td>2024-08-07</td>
          <td>Asia-Pacific Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metabolic dysfunction-associated Fatty Liver Disease (MAFLD) has emerged as one of the leading cardiometabolic diseases. Friend of GATA2 (FOG2) is a transcriptional co-regulator that has been shown to regulate hepatic lipid metabolism and accumulation. Using meta-analysis from several different biobank datasets, we identified a coding variant of FOG2 (rs28374544, A1969G, S657G) predominantly found in individuals of African ancestry (minor allele frequency~20%), which is associated with liver failure/cirrhosis phenotype and liver injury. To gain insight into potential pathways associated with this variant, we interrogated a previously published genomics dataset of 38 human induced pluripotent stem cell (iPSCs) lines differentiated into hepatocytes (iHeps). Using Differential Gene Expression Analysis and Gene Set Enrichment Analysis, we identified the mTORC1 pathway as differentially regulated between iHeps from individuals with and without the variant. Transient lipid-based transfections were performed on the human hepatoma cell line (Huh7) using wild-type FOG2 and FOG2S657G and demonstrated that FOG2S657G increased mTORC1 signaling, de novo lipogenesis, and cellular triglyceride synthesis and mass. In addition, we observed a significant downregulation of oxidative phosphorylation in FOG2S657G cells in fatty acid-loaded cells but not untreated cells, suggesting that FOG2S657G may also reduce fatty acid to promote lipid accumulation. Taken together, our multi-pronged approach suggests a model whereby the FOG2S657G may promote MAFLD through mTORC1 activation, increased de novo lipogenesis, and lipid accumulation. Our results provide insights into the molecular mechanisms by which FOG2S657G may affect the complex molecular landscape underlying MAFLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2945851076c2f51a928c3f7a7537ad4ec5a83e4e" target='_blank'>
              Genotype-First Approach Identifies an Association between rs28374544/FOG2S657G and Liver Disease through Alterations in mTORC1 Signaling
              </a>
            </td>
          <td>
            Donna Conlon, Siri Kanakala, Tess A. Cherlin, Yi-An Ko, Cecilia Vitali, Sharavana Gurunathan, Rasika Venkatesh, Jakob Woerner, Lindsay Guare, Penn Medicine Biobank, Anurag Verma, S. Verma, Marie A. Guerraty
          </td>
          <td>2024-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd0f703bff7b7dc043c4099be82b88f90d5c4d73" target='_blank'>
              Trametinib Sensitivity is Defined by a Myeloid Differentiation Profile in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Mathieu Quesnel-Vallières, David C Schultz, Alena Orlenko, Yancy Lo, Jason Moore, Marylyn Ritchie, David Roth, Martin Carroll, Yoseph Barash, Kristen W Lynch, Sara Cherry
          </td>
          <td>2024-09-24</td>
          <td>Drugs in R&D</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2235e4530843ef03fa58ea8c3a36f0f96aa2515f" target='_blank'>
              miR-149-3p Enhances Drug Sensitivity of AML Cells by Inhibiting Warburg Effect Through PI3K/AKT Pathway.
              </a>
            </td>
          <td>
            Xi Chen, Yan Song, Yaoyao Tian, Xiushuai Dong, Yuying Chang, Wei Wang
          </td>
          <td>2024-08-17</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Juvenile myelomonocytic leukemia (JMML), a clonal hematologic malignancy, originates from mutated hematopoietic stem cells (HSCs). The mechanism sustaining the persistence of mutant stem cells, leading to leukemia development, remains elusive. In this study, we conducted comprehensive examination of gene expression profiles, transcriptional factor regulons, and cell compositions/interactions throughout various stages of tumor cell development in Ptpn11 mutation-associated JMML. Our analyses revealed that leukemia-initiating Ptpn11E76K/+ mutant stem cells exhibited de novo activation of the myeloid transcriptional program and aberrant developmental trajectories. These mutant stem cells displayed significantly elevated expression of innate immunity-associated anti-microbial peptides and pro-inflammatory proteins, particularly S100a9 and S100a8. Biological experiments confirmed that S100a9/S100a8 conferred a selective advantage to the leukemia-initiating cells through autocrine effects and facilitated immune evasion by recruiting and promoting immune suppressive myeloid-derived suppressor cells (MDSCs) in the microenvironment. Importantly, pharmacological inhibition of S100a9/S100a8 signaling effectively impeded leukemia development from Ptpn11E76K/+ mutant stem cells. These findings collectively suggest that JMML tumor-initiating cells exploit evolutionarily conserved innate immune and inflammatory mechanisms to establish clonal dominance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14ee0ff31f4868484243a76f8cf02c1e5b54c52" target='_blank'>
              Prototypical innate immune mechanism hijacked by leukemia-initiating mutant stem cells for selective advantage and immune evasion in Ptpn11-associated juvenile myelomonocytic leukemia
              </a>
            </td>
          <td>
            Cheng-Kui Qu, Hong Zheng, Peng Zhao, Zhenya Tan, Wen‐mei Yu, Juwita Werner, Elliot Stieglitz, Christopher C Porter, S. Chandrakasan, Daniel Wechsler, Simon Mendez-Ferrer
          </td>
          <td>2024-08-02</td>
          <td>Research Square</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="In the ever-evolving landscape of oncology, the battle against prostate cancer (PCa) stands at a transformative juncture, propelled by the integration of molecular diagnostics into traditional cytopathological frameworks. This synthesis not only heralds a new epoch of precision medicine but also significantly enhances our understanding of the disease’s genetic intricacies. Our comprehensive review navigates through the latest advancements in molecular biomarkers and their detection technologies, illuminating the potential these innovations hold for the clinical realm. With PCa persisting as one of the most common malignancies among men globally, the quest for early and precise diagnostic methods has never been more critical. The spotlight in this endeavor shines on the molecular diagnostics that reveal the genetic underpinnings of PCa, offering insights into its onset, progression, and resistance to conventional therapies. Among the genetic aberrations, the TMPRSS2-ERG fusion and mutations in genes such as phosphatase and tensin homolog (PTEN) and myelocytomatosis viral oncogene homolog (MYC) are identified as significant players in the disease’s pathology, providing not only diagnostic markers but also potential therapeutic targets. This review underscores a multimodal diagnostic approach, merging molecular diagnostics with cytopathology, as a cornerstone in managing PCa effectively. This strategy promises a future where treatment is not only tailored to the individual’s genetic makeup but also anticipates the disease’s trajectory, offering hope for improved prognosis and quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/040ce4884f4e6bdcb8041b33d1c1868319f07888" target='_blank'>
              From microscopes to molecules: The evolution of prostate cancer diagnostics
              </a>
            </td>
          <td>
            Junyue Tao, Xiaokang Bian, Jun Zhou, Meng Zhang
          </td>
          <td>2024-08-29</td>
          <td>Cytojournal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genome instability is a characteristic trait of tumours and includes changes in DNA and in chromosomes. The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for the assessment of chromatin stability in healthy and sick dogs. The assays identified genomic instabilities in dogs with cancer (squamous cell carcinoma) and in healthy dogs. The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable the more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. The identification of instabilities can be used in routine diagnostic examination in dogs with cancer for more accurate diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7548d405e861a202a9ad5c3f6e2cc2fac0cc66" target='_blank'>
              Genomic Instability in the Lymphocytes of Dogs with Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Ewa Wójcik, Emilia Kot, Iga Wójcik, A. Wysokińska, P. Matusevičius
          </td>
          <td>2024-09-24</td>
          <td>Animals</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Elucidating the genetic architecture of DNA methylation (DNAm) is crucial for decoding the etiology of complex diseases. However, current epigenomic studies often suffer from incomplete coverage of methylation sites and the use of tissues containing heterogeneous cell populations. To address these challenges, we present a comprehensive human methylome atlas based on deep whole-genome bisulfite sequencing (WGBS) and whole-genome sequencing (WGS) of purified monocytes from 298 European Americans (EA) and 160 African Americans (AA) in the Louisiana Osteoporosis Study. Our atlas enables the analysis of over 25 million DNAm sites. We identified 1,383,250 and 1,721,167 methylation quantitative trait loci (meQTLs) in cis-regions for EA and AA populations, respectively, with 880,108 sites shared between ancestries. While cis-meQTLs exhibited population-specific patterns, primarily due to differences in minor allele frequencies, shared cis-meQTLs showed high concordance across ancestries. Notably, cis-heritability estimates revealed significantly higher mean values in the AA population (0.09) compared to the EA population (0.04). Furthermore, we developed population-specific DNAm imputation models using Elastic Net, enabling methylome-wide association studies (MWAS) for 1,976,046 and 2,657,581 methylation sites in EA and AA, respectively. The performance of our MWAS models was validated through a systematic multi-ancestry analysis of 41 complex traits from the Million Veteran Program. Our findings bridge the gap between genomics and the monocyte methylome, uncovering novel methylation-phenotype associations and their transferability across diverse ancestries. The identified meQTLs, MWAS models, and data resources are freely available at www.gcbhub.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02561d4c021777b32cc89ad698157b2f333e71dc" target='_blank'>
              An atlas of genetic effects on the monocyte methylome across European and African populations
              </a>
            </td>
          <td>
            Wanheng Zhang, Xiao Zhang, Chuan Qiu, Zichen Zhang, Kuan-Jui Su, Zhe Luo, Minghui Liu, Bingxin Zhao, Lang Wu, Qing Tian, Hui Shen, Chong Wu, Hong-Wen Deng
          </td>
          <td>2024-08-13</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Human inborn errors of immunity (IEI) represent a diverse group of genetic disorders affecting the innate and/or adaptive immune system. Some IEI entities comprise defects in DNA repair factors, resulting in (severe) combined immunodeficiencies, bone marrow failure, predisposition to malignancies, and potentially result in radiosensitivity (RS). While other IEI subcategories such as common variable immunodeficiency (CVID) and immune dysregulation disorders also associate with lymphoproliferative and malignant complications, the occurrence of RS phenotypes in the broader IEI population is not well characterized. Nonetheless, identifying RS in IEI patients through functional testing is crucial to reconsider radiation-related therapeutic protocols and to improve overall patient management. This study aimed to investigate chromosomal RS in a diverse cohort of 107 IEI patients using the G0 cytokinesis-block micronucleus (MN) assay. Our findings indicate significant variability in RS across specific genetic and phenotypical subgroups. Severe RS was detected in all ataxia-telangiectasia (AT) patients, a FANCI deficient and ERCC6L2 deficient patient, but not in any other IEI patient included in this cohort. Age emerged as the single influencing factor for both spontaneous and radiation-induced MN yields, while the manifestation of additional clinical features, including infection susceptibility, immune dysregulation, or malignancies did not associate with increased MN levels. Our extensive analysis of RS in the IEI population underscores the clinical importance of RS assessment in AT patients and supports RS testing in all IEI patients suspected of having a DNA repair disorder associated with radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b4f50595c5c29676228d24464dde5631290eb13" target='_blank'>
              Investigating chromosomal radiosensitivity in inborn errors of immunity: insights from DNA repair disorders and beyond.
              </a>
            </td>
          <td>
            E. Beyls, Evi Duthoo, Lynn Backers, Karlien Claes, Marieke De Bruyne RAPID clinicians, L. Pottie, Victoria Bordon, C. Bonroy, S. Tavernier, Kathleen BM Claes, A. Vral, A. Baeyens, F. Haerynck, T. Kerre, B. Lambrecht, L. Hoste, B. Meertens, T. V. Genechten, A. Ferster, K. V. Driessche, C. D. Vriendt, L. Dedeken, I. Moors, C. Dhooge, L. Kornreich, C. Heijmans, M. L. Roux, M. Colard, A. V. Damme, K. V. Schil, B. D. Wilde
          </td>
          <td>2024-08-20</td>
          <td>None</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Simple Summary p53 mutations are prevalent across a variety of human cancers and are regarded as a major obstacle in cancer therapy. These mutations can confer resistance to apoptosis and cell cycle arrest, contributing to multidrug resistance in tumors. Recent studies have uncovered important immunomodulatory functions of p53, but these functions are still underappreciated compared to its other well-known roles. This review aims to summarize the latest literature on immune evasion in p53-mutant tumors and explore the potential of targeting p53 to enhance anti-tumor immunity. Abstract Mutation in p53 is the most frequent event in cancer development and a leading cause of cancer therapy resistance due to evasion of the apoptosis cascade. Beyond chemotherapies and radiation therapies, growing evidence indicates that p53-mutant tumors are resistant to a broad range of immune-based therapies, such as immune checkpoint inhibitors, chimeric antigen receptor (CAR) T, and hematopoietic stem cell transplantation (HSCT). This highlights the role of p53 mutations in driving immune evasion of tumor cells. In this review, we first summarize recent studies revealing mechanisms by which p53-mutant tumors evade immune surveillance from T cells, natural killer (NK) cells, and macrophages. We then review how these mutant tumor cells reshape the tumor microenvironment (TME), modulating bystander cells such as macrophages, neutrophils, and regulatory T (Treg) cells to foster immunosuppression. Additionally, we review clinical observations indicative of immune evasion associated with p53 loss or mutations. Finally, we discuss therapeutic strategies to enhance immune response in p53 wild-type (WT) or mutant tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ac96dfaadf6e3a60d7f3146c1f00c2f180c35b" target='_blank'>
              TP53 Mutation-Mediated Immune Evasion in Cancer: Mechanisms and Therapeutic Implications
              </a>
            </td>
          <td>
            Chuqi Wang, Jordan Yong Ming Tan, Nishtha Chitkara, Shruti Bhatt
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Objective: Colon cancer is associated with multiple levels of molecular heterogeneity. RNA processing converts primary transcriptional RNA to mature RNA, which drives tumourigenesis and its maintenance. The characterisation of RNA processing genes in colon cancer urgently needs to be elucidated. Methods: In this study, we obtained 1033 relevant samples from The Cancer Genome Atlas (TCGA) and Gene Expression Omnibus (GEO) databases to explore the heterogeneity of RNA processing phenotypes in colon cancer. Firstly, Unsupervised hierarchical cluster analysis detected 4 subtypes with specific clinical outcomes and biological features via analysis of 485 RNA processing genes. Next, we adopted the least absolute shrinkage and selection operator (LASSO) as well as Cox regression model with penalty to characterise RNA processing-related prognostic features. Results: An RNA processing-related prognostic risk model based on 10 genes including FXR1, MFAP1, RBM17, SAGE1, SNRPA1, SRRM4, ADAD1, DDX52, ERI1, and EXOSC7 was identified finally. A composite prognostic nomogram was constructed by combining this feature with the remaining clinical variables including TNM, age, sex, and stage. Genetic variation, pathway activation, and immune heterogeneity with risk signatures were also analysed via bioinformatics methods. The outcomes indicated that the high-risk subgroup was associated with higher genomic instability, increased proliferative and cycle characteristics, decreased tumour killer CD8+ T cells and poorer clinical prognosis than the low-risk group. Conclusion: This prognostic classifier based on RNA-edited genes facilitates stratification of colon cancer into specific subgroups according to TNM and clinical outcomes, genetic variation, pathway activation, and immune heterogeneity. It can be used for diagnosis, classification and targeted treatment strategies comparable to current standards in precision medicine. It provides a rationale for elucidation of the role of RNA editing genes and their clinical significance in colon cancer as prognostic markers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/120a9cc9c01f30852bfe106a395264eb722362ff" target='_blank'>
              Characterization of RNA Processing Genes in Colon Cancer for Predicting Clinical Outcomes
              </a>
            </td>
          <td>
            Jianwen Hu, Yingze Ning, Yongchen Ma, Lie Sun, Guowei Chen
          </td>
          <td>2024-01-01</td>
          <td>Biomarker Insights</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="BACKGROUND
Lung cancer is a prevalent and severe form of malignant tumors worldwide. tRF-Leu-CAG, a recently discovered non-coding single-stranded small RNA derived from transfer RNA, has sparked interest in exploring its biological functions and potential molecular mechanisms in lung cancer.


METHODS
The abundance of tRF-Leu-CAG was measured via quantitative real-time polymerase chain reaction (qRT-PCR) in 96 sets of lung cancer tissue samples obtained from clinical patients. Subsequently, both in vivo and in vitro experiments were conducted to validate the biological functions of tRF-Leu-CAG in lung cancer. Furthermore, an exploration of the potential target genes of tRF-Leu-CAG and its association with autophagy and drug resistance in lung cancer was undertaken.


RESULTS
Our analysis revealed a significant upregulation of tRF-Leu-CAG in non-small cell lung cancer (NSCLC) tissues. Additionally, we observed that heightened expression of tRF-Leu-CAG significantly augmented the proliferation and migration of NSCLC cells, facilitated cell cycle progression, and suppressed apoptosis. Furthermore, we identified transcription elongation factor A3 (TCEA3) as a direct target gene of tRF-Leu-CAG. TCEA3 inhibited the proliferation and migration of NSCLC, and tRF-Leu-CAG promoted the proliferation and migration of NSCLC by mediating the silencing of TCEA3. Moreover, we demonstrated that the augmentation of paclitaxel resistance by tRF-Leu-CAG was contingent on autophagy. Finally, tRF-Leu-CAG notably accelerated tumor growth and promoted the process of epithelial-mesenchymal transition (EMT) in vivo.


CONCLUSIONS
tRF-Leu-CAG promotes NSCLC tumor growth and metastasis by targeting TCEA3 and promotes paclitaxel resistance by enhancing cellular autophagy. These results provide potentially effective targets and therapeutic options for the clinical treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a8cbdbaa838f4eb2da1ce3af02d5ba168916d9" target='_blank'>
              Oncogenic tRNA-derived fragment tRF-Leu-CAG promotes tumorigenesis of lung cancer via targeting TCEA3 and increasing autophagy.
              </a>
            </td>
          <td>
            Fan Wu, Binshu Chai, Pengfei Qi, Yaqi Han, Z. Gu, Weikang Pan, Hui Zhang, Xianyi Wang, Xiaomin Liu, Heng Zou, Chen Liang, Yanli Li, Wen-xu Fang, Zhongliang Ma
          </td>
          <td>2024-08-28</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/431b8d9fdd349b2296cea4bbf8173c562f2afaeb" target='_blank'>
              Comprehensive molecular profiling of multiple myeloma identifies refined copy number and expression subtypes
              </a>
            </td>
          <td>
            Sheri Skerget, Daniel A. Peñaherrera, A. Chari, Sundar Jagannath, David Siegel, Ravi Vij, G. Orloff, A. Jakubowiak, R. Niesvizky, Darla Liles, J. Berdeja, Moshe Levy, J. Wolf, Saad Z. Usmani, Austin Christofferson, Sara Nasser, J. Aldrich, Christophe Legendre, Brooks A. Benard, Chase Miller, Bryce Turner, Ahmet A. Kurdoglu, M. Washington, Venkata Yellapantula, J. Adkins, Lori Cuyugan, Martin Boateng, A. Helland, Shari Kyman, Jackie McDonald, Rebecca Reiman, Kristi Stephenson, Erica E Tassone, A. Blanski, Brianne Livermore, Meghan Kirchhoff, D. Rohrer, M. D'Agostino, Manuela Gamella, K. Collison, Jennifer Stumph, Pam Kidd, A. Donnelly, B. Zaugg, Maureen Toone, K. McBride, M. Derome, Jennifer Rogers, David Craig, Winnie S Liang, N. Gutiérrez, Scott D. Jewell, J. Carpten, Kenneth C Anderson, H. Cho, D. Auclair, S. Lonial, J. Keats
          </td>
          <td>2024-08-19</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2b8937040c82aaaf4474d35899e8a2ae8efb870" target='_blank'>
              Genomic, epigenomic and transcriptomic landscape of glioblastoma.
              </a>
            </td>
          <td>
            T. C. Dakal, Ganesh S. Kakde, Pawan Kumar Maurya
          </td>
          <td>2024-08-24</td>
          <td>Metabolic brain disease</td>
          <td>1</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="Genetic suppression occurs when the phenotypic defects caused by a deleterious mutation are rescued by another mutation. Suppression interactions are of particular interest for genetic diseases, as they identify ways to reduce disease severity, thereby potentially highlighting avenues for therapeutic intervention. To what extent suppression interactions are influenced by the genetic background in which they operate remains largely unknown. However, a high degree of suppression conservation would be crucial for developing therapeutic strategies that target suppressors. To gain an understanding of the effect of the genetic context on suppression, we isolated spontaneous suppressor mutations of temperature sensitive alleles of SEC17, TAO3, and GLN1 in three genetically diverse natural isolates of the budding yeast Saccharomyces cerevisiae. After identifying and validating the genomic variants responsible for suppression, we introduced the suppressors in all three genetic backgrounds, as well as in a laboratory strain, to assess their specificity. Ten out of eleven tested suppression interactions were conserved in the four yeast strains, although the extent to which a suppressor could rescue the temperature sensitive mutant varied across genetic backgrounds. These results suggest that suppression mechanisms are highly conserved across genetic contexts, a finding that is potentially reassuring for the development of therapeutics that mimic genetic suppressors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b338b4576ed119eb8873cb1fa25b1257f15a3993" target='_blank'>
              Genetic suppression interactions are highly conserved across genetic backgrounds
              </a>
            </td>
          <td>
            Claire Paltenghi, Jolanda van Leeuwen
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="There is increasing recognition that the sex chromosomes, X and Y, play an important role in health and disease that goes beyond the determination of biological sex. Loss of the Y chromosome (LOY) in blood, which occurs naturally in aging men, has been found to be a driver of cardiac fibrosis and heart failure mortality. LOY also occurs in most solid tumors in males and is often associated with worse survival, suggesting that LOY may give tumor cells a growth or survival advantage. We analyzed LOY in lung adenocarcinoma (LUAD) using both bulk and single-cell expression data and found evidence suggesting that LOY affects the tumor immune environment by altering cancer/testis antigen expression and consequently facilitating tumor immune evasion. Analyzing immunotherapy data, we show that LOY and changes in expression of particular cancer/testis antigens are associated with response to pembrolizumab treatment and outcome, providing a new and powerful biomarker for predicting immunotherapy response in LUAD tumors in males.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03627d4b1b03380930579b6c3a6ee53571e9ff88" target='_blank'>
              Selective loss of Y chromosomes in lung adenocarcinoma modulates the tumor immune environment through cancer/testis antigens
              </a>
            </td>
          <td>
            Jonas Fischer, K. Shutta, Chen Chen, Viola Fanfani, Enakshi Saha, Panagiotis Mandros, Marouen Ben Guebila, Joanne Xiu, Jorge Nieva, Stephen V. Liu, D. Uprety, D. Spetzler, C. Lopes-Ramos, Dawn L. DeMeo, John Quackenbush
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c12c760f08b0716217f59b590788ec67fee6b6e" target='_blank'>
              Double-negative T cells with a distinct transcriptomic profile are abundant in the peripheral blood of patients with breast cancer.
              </a>
            </td>
          <td>
            Hui-Ru Zhu, Yun-Bo Wei, Jia-Qi Guo, Xiao-Fei Liu
          </td>
          <td>2024-09-10</td>
          <td>Breast cancer research and treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is an incurable malignancy characterised by altered expression of coding and non-coding genes promoting tumour growth and drug resistance. Although the crucial role of long non-coding RNAs (lncRNAs) in MM is clearly established, the function of the non-coding RNAome, which might allow the design of novel therapeutics, is largely unknown. We performed an unbiased CRISPR-Cas9 loss-of-function screen of 671 lncRNAs in MM cells and their Bortezomib (BZB)-resistant derivative. To rank functionally and clinically relevant candidates, we designed and used a bioinformatic prioritisation pipeline combining functional data from cellular screens with prognostic and transcriptional data from MM patients. With this approach, we unveiled and prioritised 8 onco-lncRNAs essential for MM cell fitness, associated with high expression and poor prognosis in MM patients. The previously uncharacterised RP11-350G8.5 emerged as the most promising target, irrespective of BZB resistance. We i) demonstrated the anti-tumoral effect obtained by RP11-350G8.5 inhibition in vitro and in vivo; ii) highlighted a modulation of the unfolded protein response and the induction of immunogenic cell death triggered by the RP11-350G8.5 knock-out, via RNA-sequencing and molecular studies; iii) characterised its cytoplasmic homing through RNA-FISH; iv) predicted its 2D structure and identified 2 G-quadruplex and 3 hairpin-forming regions by biophysical assays, including Thioflavin T, 1H-NMR and circular dichroism to pave the way to the development of novel targeted therapeutics. Overall we provided innovative insights about unexplored lncRNAs in MM and identified RP11-350G8.5 as an oncogenic target for treatment-naïve and BZB-resistant MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b3038ab25be0410e0d1aec773f7b5564cf954f8" target='_blank'>
              An unbiased lncRNAs dropout CRISPR-Cas9 screen reveals RP11-350G8.5 as a novel therapeutic target for Multiple Myeloma.
              </a>
            </td>
          <td>
            K. Grillone, S. Ascrizzi, Paolo Cremaschi, Jussara Amato, N. Polerà, Ottavio Croci, R. Rocca, C. Riillo, Francesco Conforti, Raffaele Graziano, Diego Brancaccio, Daniele Caracciolo, Stefano Alcaro, Bruno Pagano, Antonio Randazzo, P. Tagliaferri, F. Iorio, P. Tassone
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 Personalized medicine programs, including the Zero Childhood Cancer Program (ZERO), perform molecular analysis of individual patient tumours to identify molecular targets for therapy. The comprehensive and unbiased sequencing approaches used by ZERO, including whole genome sequencing (WGS) and RNA sequencing (RNAseq), enables detection of novel structural variants (SVs) that may be cryptic to standard cytogenetic techniques or panel sequencing approaches. Identification of Receptor Tyrosine Kinase (RTK) activating variants in paediatric cancers, for example NTRK2 gene fusions, is a high priority given the availability and remarkable clinical success of RTK inhibitors. Here we describe a novel internal tandem duplication (ITD) in NTRK2, encoding TRKB, in a ZERO patient with CNS neuroblastoma. This SV was detected by WGS and was notably missed by RNAseq-based gene fusion and SV detection. The ITD spans exons 10-13 of NTRK2, is in-frame, and includes the juxtamembrane and transmembrane protein domains. We hypothesized that this ITD would result in constitutive activation of TRKB and would be sensitive to TRK-targeted therapies. We overexpressed NTRK2 ITD, wild-type NTRK2, and SPECC1L-NTRK2 (an established TRK-activating fusion), in the interleukin-3 (IL-3)-dependent cell line, Ba/F3, and the neuroblastoma cell line, SH-SY5Y. In these models, we showed that like wild-type NTRK2, NTRK2 ITD was expressed on the cell surface. NTRK2 ITD was sufficient to transform Ba/F3 cells to IL-3 independence through constitutive activation of TRKB and downstream signaling pathways PI3K/AKT and MEK/ERK. NTRK2 ITD-expressing cells were sensitive to TRK inhibitors, including larotrectinib, at similar doses to SPECC1L-NTRK2-expressing cells. Interestingly, we observed that cells expressing NTRK2 ITD were specifically sensitive to MEK inhibition, which was not observed in SPECC1L-NTRK2-expressing cells, suggesting the mechanisms by which these variants drive transformation is different. Indeed, in silico structural analysis showed that the duplicated region of NTRK2 ITD contains two internalized tyrosine residues whose phosphorylation could both drive autophosphorylation of the receptor and act as docking sites for adaptor proteins. Mutation of these tyrosine residues delayed transformation of Ba/F3 cells. This study functionally characterizes a novel NTRK2 ITD and shows that this variant is transforming and sensitive to TRK inhibition. While ITDs have been described in other RTK genes, notably FGFR1 and FLT3, structurally these have involved the kinase domain or the juxtamembrane domain. This is the first report of an ITD that spans the transmembrane domain of an RTK, characterizing an additional mechanism by which RTKs can be activated in cancer. This study highlights the value of unbiased WGS approaches to identify novel SVs and identifies another class of patients that may benefit from RTK inhibitor therapy.
 Citation Format: Lauren M. Brown, Gabor Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah Jalud, Teresa Sadras, Pamela Ajuyah, Paulette Barahona, M. Emmy M. Dolman, Marie Wong, Chelsea Mayoh, Mark J. Cowley, Paul G. Ekert. A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ae609e3759161918ab8221b777b148ec5605447" target='_blank'>
              Abstract B002: A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation
              </a>
            </td>
          <td>
            Lauren M. Brown, G. Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah B Jalud, Teresa Sadras, P. Ajuyah, P. Barahona, M. E. M. Dolman, Marie Wong, C. Mayoh, Mark J. Cowley, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Gene expression patterns are very sensitive to external influences and are reflected in phenotypic changes. It was previously described that transferring melanoma cells from a plastic surface to Matrigel led to formation of de novo vascular networks—vasculogenic mimicry—that are characteristic to a stemness phenotype in aggressive tumors. Up to now there was no detailed data about the gene signature accompanying this process. Here, we show that this transfer shortly led to extremely strong epigenetic changes in gene expression in the melanoma cells. We observed that on Matrigel numerous genes controlling ribosome biogenesis were upregulated. However, most of the activated genes were inhibitors of the differentiation genes (ID1, ID2, and ID3). At the same time, the genes that control differentiation were downregulated. Both the upregulated and the downregulated genes are simultaneously targeted by different transcription factors shaping sets of co-expressed genes. The specific group of downregulated genes shaping contacts with rDNA genes are also associated with the H3K27me3 mark and with numerous lincRNAs and miRNAs. We conclude that the stemness phenotype of melanoma cells is due to the downregulation of developmental genes and formation of dedifferentiated cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e3983f49ac992e0e94ffb5e7e55f9651ff1ef6" target='_blank'>
              Strong Activation of ID1, ID2, and ID3 Genes Is Coupled with the Formation of Vasculogenic Mimicry Phenotype in Melanoma Cells
              </a>
            </td>
          <td>
            N. Tchurikov, Amalia A. Vartanian, E. S. Klushevskaya, I. R. Alembekov, A. N. Kretova, Viktoriya N. Lukicheva, V. Chechetkin, G. I. Kravatskaya, V. S. Kosorukov, Y. V. Kravatsky
          </td>
          <td>2024-08-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Children born with Down syndrome (DS), a genetic condition caused by the presence of an additional chromosome 21 (trisomy 21 or T21), have a substantially higher risk of developing childhood leukaemia. Approximately 30% of DS infants develop a preleukemic condition called transient abnormal myelopoiesis (TAM). TAM is strictly associated with GATA1 truncating mutations on a T21 background. Most TAM cases spontaneously resolve within the first few months of life. However, by the age of five, approximately 10% of cases progress to megakaryoblastic/erythroid leukaemia, known as myeloid leukaemia of DS (ML-DS). This final step of transformation is often driven by specific third-hit somatic mutations, most of which affects genes encoding JAK kinases, cohesin complexes, or epigenetic regulators. These well-defined stepwise genetic aberrations associated with ML-DS pathogenesis provide a unique and highly tractable system to elucidate the molecular changes underlying leukaemogenesis. Using single-cell mRNA sequencing of primary human samples, we directly interrogate the transcriptional changes underpinning the multi-step pathogenesis of ML-DS. We observe an expansion of megakaryocyte-erythrocyte progenitors (MEPs) specifically in T21 foetal liver compared to diploid or other trisomies. However, it is the GATA1 mutation that accounts for the majority of the transcriptomic changes towards that of the leukaemic blasts. Furthermore, investigation of an aggressive TAM case reveals an enrichment in expression of the leukaemogenic gene module associated with the full-blown leukaemia state ML-DS. We also demonstrate that TAM and ML-DS blasts still retain transcriptional signatures of normal haematopoiesis, showing heterogeneous features of differentiation towards erythrocytes, megakaryocytes, and mast cells. Finally, by integrating genomic and transcriptomic data, we directly defined the genomic evolution, and its corresponding transcriptional consequences, that underpins a treatment-refractory case of ML-DS. Overall, we have generated the first comprehensive single-cell mRNA atlas of TAM / ML-DS, enabling unbiased quantitative characterisation of the transcriptional transformation along ML-DS pathogenesis. Our study explores the molecular signature associated with progression of TAM to ML-DS, as well as the mechanism underlying differences in treatment response. These insights offer the possibility of more meaningful clinical interpretations and potential to explore novel therapeutic targets for these conditions.
 Citation Format: Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, Angus Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, Jan-Henning Klusmann, Jack Bartram, Sam Behjati. Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f4dde5c53348eeefdb9808ec633733f445cffc" target='_blank'>
              Abstract A018 Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome
              </a>
            </td>
          <td>
            Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, A. Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, J. Klusmann, Jack Bartram, Sam Behjati
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="For several decades, induction therapy with nucleoside analogs, in particular cytarabine (Ara-C) and, to a lesser extent, fludarabine, has been the standard of care for patients diagnosed with acute myeloid leukemia (AML). Still, the anti-tumor efficacy of nucleoside analogs is often limited by intrinsic and acquired drug resistance, thereby leading to poor therapeutic response and suboptimal clinical outcomes. Here, we used genome-wide CRISPR-based pharmacogenomic screening to map the genetic factors that modulate the response to nucleoside analogs in AML and identified the E3 ubiquitin ligase Herc1 as a key modulator of Ara-C response in the MLL/AF9 (MA) and the HOXA9/MEIS1 (HM) murine AML models both in vitro and in vivo. Loss of HERC1 enhanced nucleoside analog-induced cell death in both murine and human AML cell lines by compromising cell cycle progression. In-depth proteomic analysis and subsequent validation identified deoxycytidine kinase Dck as a novel target of Herc1 in MA and HM murine cells. We observed that HERC1 is overexpressed in AML compared to other cancer types and higher HERC1 expression is associated with shorter overall survival of patients with AML in the TCGA and BEAT-AML cohorts. Collectively, this study highlights the importance of HERC1 in the response of AML cells to nucleoside analogs, thereby establishing this E3 ubiquitin ligase as a novel predictive biomarker and potential therapeutic target for the treatment of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5245f1077766a809fae0247565289dd67c6fcad" target='_blank'>
              The E3 ubiquitin ligase Herc1 modulates the response to nucleoside analogs in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Maja Jankovic, William Wl Poon, Cristobal Gonzales-Losada, Gabriela Galicia Vazquez, Bahram Sharif-Askari, Yi Ding, Constance Craplet-Desombre, Alexandru Ilie, Jiantao Shi, Yongie Wang, A. Jayavelu, Alexandre Orthwein, F. E. Mercier
          </td>
          <td>2024-08-02</td>
          <td>Blood advances</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="The family of LINE1 transposable elements underwent a massive expansion in mammalian genomes. While traditionally viewed as a mutagenic selfish element, recent studies point to roles for LINE1 in early mouse development, T cell quiescence and neurogenesis. Here we show that human LINE1 RNA is essential for self-renewal and identity of human embryonic stem cells (hESCs). Silencing of LINE1 using either antisense oligonucleotides or CRISPR interference in naïve hESCs leads to a strong induction of 8C-like cells (8CLCs). We found that genes derepressed upon LINE1 KD are not uniformly distributed across the genome, with an enrichment for chromosome 19, which includes key markers of the 8C state such as TPRX1. Silencing of TPRX1, but not other putative 8C regulators p53 or H3.XY, suppresses the induction of the 8C program in LINE1 KD hESCs. We found that LINE1 RNA is preferentially localized to the lamina and periphery of the nucleolus in hESCs. Sequencing of Lamina-Associated Domains (LADs) and Nucleolus-Associated Domains (NADs) reveals a preferential association of chromosome 19 with NADs in hESCs. However, 8CLCs have a distinct nucleolar morphology and a lower association of chromosome 19 and TPRX1 loci with the nucleolus relative to naïve and primed hESCs, suggesting a role for nucleolar dynamics in the 8CLC-hESC transition. In agreement, LINE1 KD leads to disruption of nucleolar architecture with signs of nucleolar stress. Independent perturbations of the nucleolus induce the 8C program in hESCs. Genes induced by LINE1 KD are enriched for targets of Polycomb Repressive Complex (PRC2), and inhibition of PRC2 leads to a strong induction of 8C genes. Our results indicate that LINE1 coordinates nuclear compartmentalization and chromatin-mediated gene repression to prevent developmental reversion of hESCs. Highlights Knockdown of LINE1 induces TPRX1-dependent emergence of 8C-like cells in hESCs. Genes de-repressed upon LINE1 KD are enriched for Chr 19 and PRC2 targets. 8CLCs display dissociation of Chromosome 19 from the nucleolus. Disruption of the nucleolus or inhibition of PRC2 strongly induce the 8C program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34781a9454759e1cbbdf47b8d98d4f390cccbdd4" target='_blank'>
              LINE1 promotes nuclear compartmentalization to repress the 8-cell state in embryonic stem cells
              </a>
            </td>
          <td>
            Juan Zhang, Lamisa Ataei, Liang Wu, Kirti Mittal, Linh Huynh, Shahil Sarajideen, Abdul Mazid, David P. Cook, D. Trcka, Kevin Tse, Jeffrey L. Wrana, Michael M. Hoffman, Miguel A Esteban, M. Ramalho-Santos
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Background Alterations in DNA damage repair genes in advanced prostate cancer (PC) may impact responses to therapy and clinical outcomes. This study described homologous recombination repair (HRR) testing patterns and clinical outcomes among patients with metastatic castration-resistant prostate cancer (mCRPC) by HRR alteration status and race in the United States (US). Methods Clinical data in the nationwide (US-based) Flatiron Health-Foundation Medicine, Inc. (FMI) Metastatic PC Clinico-Genomic Database were evaluated (01/01/2011–12/31/2022). Patients initiating first-line (1L) mCRPC therapy on or after mCRPC diagnosis were included. Testing patterns, time-to-next treatment, overall survival (OS), and time-to-prostate specific antigen response were described. Results Of the 1367 patients with mCRPC and at least one HRR panel test prior to or on the date of 1L mCRPC therapy initiation, 332 (24.3%) were HRR positive (White patients: n = 219 [66.0%]; Black patients: n = 37 [11.1%]) and 1035 (75.7%) were HRR negative (White patients: n = 702 [67.8%]; Black patients: n = 84 [8.1%]). The mean time between first positive test and 1L mCRPC therapy initiation date was 588 days (White patients: 589 days; Black patients: 639 days). Among HRR positive relative to negative patients, trends for faster progression (respective 12-month rate overall: 71.1% and 63.7%; White patients: 72.5% and 64.0%; Black patients: 65.4% and 56.4%), shorter OS (respective 24-month rate overall: 46.8% and 51.9%; White patients: 48.6% and 46.2%; Black patients: 52.8% and 54.1%), and decreased treatment response (respective 12-month rate overall: 24.3% and 37.9%; White patients: 24.5% and 35.2%; Black patients: 17.0% and 43.9%) were observed. Conclusion Patients with mCRPC positive for HRR alterations tended to exhibit poorer treatment responses and clinical outcomes than those with a negative status. These findings highlight the importance of timely genetic testing in mCRPC, particularly among Black patients, and the need for improved 1L targeted therapies to address the unmet need in HRR positive mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b5b000d1e3f13e0b5a22c9691a361f7dc5edd2" target='_blank'>
              Homologous Recombination Repair Testing Patterns and Outcomes in mCRPC by Alteration Status and Race
              </a>
            </td>
          <td>
            M. Bilen, I. Khilfeh, C. Rossi, Erik Muser, L. Morrison, A. Hilts, Lilian Diaz, Patrick Lefebvre, D. Pilon, Daniel George
          </td>
          <td>2024-09-01</td>
          <td>ClinicoEconomics and Outcomes Research: CEOR</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Abstract Pediatric classic Hodgkin lymphoma (cHL) patients have a high survival rate but suffer from severe long‐term side effects induced by chemo‐ and radiotherapy. cHL tumors are characterized by the low fraction (0.1%–10%) of malignant Hodgkin and Reed–Sternberg (HRS) cells in the tumor. The HRS cells depend on the surrounding immune cells for survival and growth. This dependence is leveraged by current treatments that target the PD‐1/PD‐L1 axis in cHL tumors. The development of more targeted therapies that are specific for the tumor and are therefore less toxic for healthy tissue compared with conventional chemotherapy could improve the quality of life of pediatric cHL survivors. Here, we applied single‐cell RNA sequencing (scRNA‐seq) on isolated HRS cells and the immune cells from the same cHL tumors. Besides TNFRSF8 (CD30), we identified other genes of cell surface proteins that are consistently overexpressed in HRS cells, such as NRXN3 and LRP8, which can potentially be used as alternative targets for antibody–drug conjugates or CAR T cells. Finally, we identified potential interactions by which HRS cells inhibit T cells, among which are the galectin‐1/CD69 and HLA‐II/LAG3 interactions. RNAscope was used to validate the enrichment of CD69 and LAG3 expression on T cells near HRS cells and indicated large variability of the interaction strength with the corresponding ligands between patients and between tumor tissue regions. In conclusion, this study identifies new potential therapeutic targets for cHL and highlights the importance of studying heterogeneity when identifying therapy targets, specifically those that target tumor‐immune cell interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/963141cd2c957924d6cb5b66200fd817e1bf6a44" target='_blank'>
              Single‐cell RNA sequencing of pediatric Hodgkin lymphoma to study the inhibition of T cell subtypes
              </a>
            </td>
          <td>
            Jurrian K de Kanter, A. Steemers, D. M. Gonzalez, Ravian L. van Ineveld, Catharina Blijleven, N. Groenen, L. Trabut, M. Scheijde-Vermeulen, Liset Westera, Auke Beishuizen, Anne C. Rios, F. C. Holstege, A. Brandsma, T. Margaritis, R. van Boxtel, F. Meyer-Wentrup
          </td>
          <td>2024-09-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01413a6e42d02eb690fa59aa79101fc898cc9619" target='_blank'>
              RNA sequestration in P-bodies sustains myeloid leukaemia.
              </a>
            </td>
          <td>
            Srikanth Kodali, Ludovica Proietti, Gemma Valcárcel, Anna V. López-Rubio, Patrizia Pessina, Thomas Eder, Junchao Shi, Annie Jen, Núria Lupión-Garcia, Anne C. Starner, Mason Bartels, Yingzhi Cui, Caroline M Sands, Ainoa Planas-Riverola, Alba Martínez, T. Velasco-Hernández, Laureano Tomás-Daza, Bernhard Alber, G. Manhart, I. Mayer, K. Kollmann, A. Fatica, Pablo Menendez, Evgenia Shishkova, Rachel E Rau, B. Javierre, Joshua J. Coon, Qi Chen, Eric L. Van Nostrand, J. L. Sardina, F. Grebien, Bruno Di Stefano
          </td>
          <td>2024-08-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Genetic or epigenetic variations in regulatory enhancer elements increase susceptibility to a range of pathologies, including pancreas cancer. Pinpointing genes affecting pancreas cancer risk holds promise for early detection, prevention, and effective therapies. Despite recent advances, linking enhancer elements to target genes and predicting the transcriptional outcome of enhancer dysfunction remain significant challenges. Using 3D chromatin assays, we generated an extensive enhancer interaction dataset for the human pancreas, spanning more than 20 donors and five major cell types, including both the exocrine and endocrine compartments. We employed a network approach to parse chromatin interactions into enhancer-promoter tree models, facilitating quantitative, genome-wide analysis of enhancer connectivity. Using the tree models, we developed a machine learning algorithm capable of estimating the impact of enhancer perturbations on cell-type specific gene expression in the human pancreas. Complementing this computational approach, we streamlined an experimental platform using CRISPR-interference and RNA-FISH coupled with high-throughput imaging to quantitatively measure the effect size of enhancer perturbations in primary human pancreas cells. Our enhancer tree models enabled functional annotation of genetic variants associated with pancreas diseases, revealing novel target genes in specific cell types. For pancreas cancer (PDAC), our data suggests a stronger association of disease susceptibility variants with acinar cells, even though ductal cells historically have been the focus of PDAC. This discovery linked three potentially functional variants associated with PDAC risk to acinar cell enhancers, revealing novel target genes to further investigation. By integrating genomics with mechanistic findings into enhancer function, our work offers a framework for understanding genetic basis of pancreas cancer disease risk.
 Citation Format: Li Wang, Songjoon Baek, Gauri Prasad, Thucnhi Truongvo, Jason Hoskins, Laufey Amundadottir, Efsun Arda. A novel approach to understanding pancreas cancer risk through enhancer- promoter interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f97224f03d537588713402ad00fc72aae371ba05" target='_blank'>
              Abstract C058: A novel approach to understanding pancreas cancer risk through enhancer- promoter interactions
              </a>
            </td>
          <td>
            Li Wang, Songjoon Baek, Gauri Prasad, Thucnhi Truongvo, J. Hoskins, L. Amundadottir, Efsun Arda
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d65d9d5a5ac50ea71ca0cdd687057235e734679" target='_blank'>
              Limb reduction in an Esco2 cohesinopathy mouse model is mediated by p53-dependent apoptosis and vascular disruption
              </a>
            </td>
          <td>
            Arielle S Strasser, A. Gonzalez-Reiche, Xianxiao Zhou, Braulio Valdebenito-Maturana, Xiaoqian Ye, Bin Zhang, Meng Wu, H. van Bakel, E. Jabs
          </td>
          <td>2024-08-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Mutations in the male germline are a driving force behind rare genetic diseases. Driver mutations enjoying a selective advantage expand to mutant clusters within the aged testis and are thus overrepresented in sperm with age. Other kinds of driver mutations, occurring prepubescently, have been the focus of recent attention given their high occurrence rate independent of age. Here, we investigated the ErbB2 gene via error-corrected sequencing and detected a high percentage of missense mutations, including recurrent mutations, which were observed mainly in the tyrosine kinase domain with likely functional consequences, as we verified for a subset via biophysical methods. While these mutations increased with age, we found no evidence that they originated from mutational clusters in the aged testis, and young donors also showed an accumulation of driver mutations, suggesting that mutational enrichment is not exclusive to the sexually mature germline but can occur earlier during germline development, and are likely evenly distributed in the testis and stable in size.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa9f975fbf6b3e62de966f4ae5746c898526f4b3" target='_blank'>
              Mutations in ErbB2 accumulating in the male germline measured by error-corrected sequencing
              </a>
            </td>
          <td>
            Atena Yasari, Monika Heinzl, Theresa Mair, Tina Karimian, Shehab Moukbel Ali Aldawla, Ingrid Hartl, Andrea J. Betancourt, Peter Lanzerstorfer, Irene Tiemann-Boege
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background/Aim Despite therapeutic advancements, metastatic colorectal cancer is usually fatal, necessitating novel approaches based on the molecular pathogenesis to improve outcomes. Some colorectal cancers have no mutations in the extended RAS panel (KRAS, NRAS, BRAF) genes and represent a special subset, which deserves particular therapeutic considerations. Materials and Methods The genomic landscape of colorectal cancers from publicly available genomic series was interrogated, using the cBioportal platform. Colorectal cancer cohorts with cancers devoid of KRAS/NRAS or BRAF mutations were evaluated for the presence of mutations in the catalytic sub-unit alpha of kinase PI3K, encoded by the gene PIK3CA. Results PIK3CA mutations in the absence of KRAS/NRAS/BRAF mutations were observed in 3.7% to 7.6% of colorectal cancers in the different series examined. Patients with all four genes in wildtype configuration (quadruple wild type) represented 32.2% to 39.9% of cases in the different series examined. Compared with quadruple wild type cancers, triple (KRAS/NRAS/BRAF) wild type/PIK3CA mutated cancers had a higher prevalence of high TMB cases and additional mutations in colorectal cancer associated genes except for mutations in TP53. Mutations in genes encoding for epigenetic modifiers and the DNA damage response (DDR) were also more frequent in triple wild type/PIK3CA mutated cancers. The prognosis of the two groups was comparable. Conclusion Colorectal cancers with PIK3CA mutations in the absence of KRAS/NRAS/BRAF mutations have frequently mutations in epigenetic modifiers and DDR response genes, which may provide opportunities for targeting. These mutations are present in a smaller subset of quadruple wild type cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fec366ce24e54405084d6854f8aadb7604b8213" target='_blank'>
              PIK3CA Mutated Colorectal Cancers Without KRAS, NRAS and BRAF Mutations Possess Common and Potentially Targetable Mutations in Epigenetic Modifiers and DNA Damage Response Genes
              </a>
            </td>
          <td>
            I. Voutsadakis
          </td>
          <td>2024-08-27</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/094bbe114db0279ee40c26eb6fe6398adcd7e861" target='_blank'>
              Pediatric cancer—pathology and microenvironment influence: a perspective into osteosarcoma and non-osteogenic mesenchymal malignant neoplasms
              </a>
            </td>
          <td>
            Consolato M. Sergi
          </td>
          <td>2024-08-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, Jens Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Menin inhibitors that disrupt Menin-MLL interaction hold promise for treating specific acute myeloid leukemia subtypes, including KMT2A rearrangements (KMT2A-r), yet resistance remains a challenge. Here, through systematic chromatin-focused CRISPR screens, along with genetic, epigenetic, and pharmacologic studies in a variety of human and mouse KMT2A-r AML models, we uncover a potential resistance mechanism independent of canonical Menin-MLL targets. We show that a group of non-canonical Menin targets, which are bivalently co-occupied by active Menin and repressive H2AK119ub marks, are typically downregulated following Menin inhibition. The loss of Polycomb Repressive Complex 1.1 (PRC1.1) subunits, such as PCGF1 or BCOR, leads to Menin inhibitor resistance by epigenetic reactivation of these non-canonical targets, including MYC. Genetic and pharmacological inhibition of MYC can resensitize PRC1.1-deficent leukemia cells to Menin inhibition. Moreover, we demonstrate that leukemia cells with the loss of PRC1.1 subunits exhibit reduced monocytic gene signatures and are susceptible to the BCL2 inhibition, and combinational treatment of venetoclax overcomes the resistance to Menin inhibition in PRC1.1-deficient leukemia cells. These findings highlight the important roles of PRC1.1 and its regulated non-canonical Menin targets in modulating Menin inhibitor response and provide potential strategies to treat leukemias with compromised PRC1.1 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fba94eb1d6d5cb6e0a8de6969fbdc894d28222" target='_blank'>
              Epigenetic Regulation of Non-canonical Menin Targets Modulates Menin Inhibitor Response in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Xinyue Zhou, Lixia Zhang, Sajesan Aryal, Virginia Veasey, Amanda Tajik, Cecilia Restelli, Steven Moreira, Pengcheng Zhang, Yanfeng Zhang, Kristin J Hope, Yang Zhou, Changde Cheng, Ravi Bhatia, Rui Lu
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chromatin is not randomly distributed within the nucleus, but organized in a three-dimensional structure that plays a critical role in genome functions. Сohesin and condensins are conserved multi-subunit protein complexes that participate in mammalian genome organization by extruding chromatin loops. The fine temporal regulation of these complexes is facilitated by a number of other proteins, one of which is microcephalin (Mcph1). Mcph1 prevents condensin II from associating with chromatin through interphase. Loss of Mcph1 induces chromosome hypercondensation; it is not clear to what extent this reorganization affects gene expression. In this study, we generated several mouse embryonic stem cell (mESC) lines with knockout of the Mcph1 gene and analyzed their gene expression profile. Gene Ontology analyses of differentially expressed genes (DEGs) after Mcph1 knockout revealed gene categories related to general metabolism and olfactory receptor function but not to cell cycle control previously described for Mcph1. We did not find a correlation between the DEGs and their frequency of lamina association. Thus, this evidence questions the hypothesis that Mcph1 knockout-mediated chromatin reorganization governs gene expression in mESCs. Among the negative effects of Mcph1 knockout, we observed numerous chromosomal aberrations, including micronucleus formation and chromosome fusion. This confirms the role of Mcph1 in maintaining genome integrity described previously. In our opinion, dysfunction of Mcph1 may be a kind of “Rosetta stone” for deciphering the function of condensin II in the interphase nucleus. Thus, the cell lines with knocked-out Mcph1 can be used to further study the influence of chromatin structural proteins on gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34d65d4d4365d60abd6b73f946f30c7616d79df9" target='_blank'>
              Generation and analysis of mouse embryonic stem cells with knockout of the Mcph1 (microcephalin) gene
              </a>
            </td>
          <td>
            A. Yunusova, A. V. Smirnov, T. Shnaider, I. Pristyazhnuk, S.Y. Korableva, N. R. Battulin
          </td>
          <td>2024-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The SAGA complex is an evolutionarily conserved histone acetyltransferase complex and transcription coactivator essential for development and disease. Dysregulation of SAGA is implicated in various human diseases, including cancer. In this issue of Genes & Development, Chen et al. (doi/10.1101/gad.351789.124) uncover a critical role for SAGA in multiple myeloma wherein SAGA's ADA2B component is required for the expression of mTORC1 pathway genes and targets of the MYC, E2F, and MAF (musculoaponeurotic fibrosarcoma) transcription factors. SAGA cooperates with MYC and MAF to sustain oncogenic gene expression programs vital for multiple myeloma survival and thus may serve as a therapeutic target for future cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa0500e20ce37a63c1e23fe219d04a1b016c2f1c" target='_blank'>
              The MYC-MAF-SAGA axis drives oncogenic gene expression in multiple myeloma.
              </a>
            </td>
          <td>
            Hongkuan Wang, Hong Wen, Xiaobing Shi
          </td>
          <td>2024-08-01</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Background: To date, targeted therapies have largely failed in eliciting sustained responses in advanced pancreatic ductal adenocarcinoma (PDAC). KRAS-targeted therapies have the potential to drastically transform clinical management. Emerging data indicate that only a subset of patients respond, and that acquired resistance is common. Here, we used a combination of in vivo CRISPR activation (CRISPRa) screening and in vitro/preclinical models to identify synergistic combinations with KRAS inhibition (KRASi) in pancreatic and lung cancer. Material and Methods: We used our CRISPRa-competent PPKS (P53(F/F), Kras(LSL-G12D/+), R26(LSL-SAM)) model of adenocarcinoma to conduct targeted screening for drivers of resistance to MRTX133. Our library consisted of ∼450 frequently mutated and amplified putative oncogenes across human adenocarcinoma based on TCGA data. Mice were treated with 1e8 TU of lentiviral library through nasal instillation. After 6 weeks, treatment was initiated with 3 mg/kg MRTX1133 daily, 5 days/week (i.p.). Mice were sacrificed after 4 weeks of treatment and the guide distribution determined through bioinformatic analysis. Genes statistically significantly enriched (FDR<0.05) in the treatment group were determined. Cell lines with acquired resistance to MRTX1133 derived from the KPC PDAC model and the KPP lung adenocarcinoma (LUAD) model were generated using long-term, low-dose treatment. Resistance was confirmed with MTT drug response assays. In vitro combinatorial drug response was determined using a 72h, 96-well MTT assay, and clonogenic response using a 2 week, 384-well FACS assay. Mice were treated with 20 mg/kg MRTX1133 and/or 20mg/kg Temuterkib once daily, 7 days/week (i.p.) for 2 weeks. Results: We performed targeted in vivo CRISPRa screening for drivers of resistance to KRASi in an autochthons model of lung adenocarcinoma (LUAD) using a library targeting 452 genes commonly amplified or mutated in cancer. We identified a set of genes that were specifically enriched in at least 50% of tumors treated with MRTX1133, including MAPK1/ERK2. We also generated 16 cell lines with acquired resistance to MRTX1133, derived from KPC PDAC or KPP LUAD tumors. Transcriptomic analysis identified a large set of genes differentially expressed in cell with acquired resistance relative to controls. GSEA indicated that reactivation of the MAPK cascade as a common trait in resistant cells. To test if ERK inhibition synergizes with KRASi, we quantified the combinatorial response to MRTX1133 and Temuterkib, a specific inhibitor of ERK1 and ERK2, in a set of KPC and KPP derived cell lines, revealing significant synergy and reduced clonogenic potential in a majority of cell lines. A majority of cell lines with acquired resistance to MRTX1133 remained sensitive to Temuterkib. Finally, we tested combinatorial treatment of MRTX1133 and Temuterkib in a syngenetic subcutaneous model of PDAC, which showed at minimum additive effect. In conclusion we have identified ERK inhibition as a promising combination with KRAS inhibition in KRAS-mutant PDAC and LUAD.
 Citation Format: Fredrik I Thege, Amber Hoskins, Sonja M Woermann, Anirban Maitra. Identifying synergistic combinations with KRAS inhibition in PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr PR-15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/172a9d516cee228f00e1cc1c95f94a25e631e8bb" target='_blank'>
              Abstract PR-15: Identifying synergistic combinations with KRAS inhibition in PDAC
              </a>
            </td>
          <td>
            Fredrik I. Thege, Amber Hoskins, Sonja M. Woermann, Anirban Maitra
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Glioblastoma is the most common primary malignant brain tumor in adults, with a median survival of just over 1 year. The failure of available treatments to achieve remission in patients with glioblastoma (GBM) has been attributed to the presence of cancer stem cells (CSCs), which are thought to play a central role in tumor development and progression and serve as a treatment-resistant cell repository capable of driving tumor recurrence. In fact, the property of "stemness" itself may be responsible for treatment resistance. In this study, we identify a novel long noncoding RNA (lncRNA), cancer stem cell-associated distal enhancer of SOX2 (CASCADES), that functions as an epigenetic regulator in glioma CSCs (GSCs). CASCADES is expressed in isocitrate dehydrogenase (IDH)-wild-type GBM and is significantly enriched in GSCs. Knockdown of CASCADES in GSCs results in differentiation towards a neuronal lineage in a cell- and cancer-specific manner. Bioinformatics analysis reveals that CASCADES functions as a super-enhancer-associated lncRNA epigenetic regulator of SOX2. Our findings identify CASCADES as a critical regulator of stemness in GSCs that represents a novel epigenetic and therapeutic target for disrupting the CSC compartment in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5be2f4cc8ff228e53f5dc871cb0a020d6528f65a" target='_blank'>
              CASCADES, a novel SOX2 super-enhancer-associated long noncoding RNA, regulates cancer stem cell specification and differentiation in glioblastoma.
              </a>
            </td>
          <td>
            Uswa Shahzad, Marina Nikolopoulos, Christopher Li, Michael J Johnston, J. Wang, N. Sabha, Frederick Varn, Alexandra N Riemenschneider, Stacey L. Krumholtz, P. M. Krishnamurthy, Christian A. Smith, Jason Karamchandani, Jonathan K. Watts, R. Verhaak, Marco Gallo, James T. Rutka, Sunit Das
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Rhabdoid tumors (RTs) are rare, highly aggressive pediatric malignancies which occur in kidney, soft-parts and brain. They are characterized by a complete inactivation of the SMARCB1 tumor suppressor gene encoding a core subunit of the chromatin remodeling SWI/SNF complex. Prognosis for children with RTs is poor as, in many instances, these tumors are resistant to conventional type chemotherapy. Pharmacological inhibition of EZH2 is a promising strategy to treat some tumors with loss of function in SMARCB1. However, the clinical response rate remains low, and both primary and secondary resistance have been reported. Understanding the mechanisms of resistance to EZH2 inhibition may allow new therapeutic hypothesis. For this purpose, we realized a genome-wide CRISPR-Cas9 knockout screening on two rhabdoid cell lines treated with EZH2 inhibitor. We identified TP53 loss as the sole consistent gene knock-out able to confer some resistance to EZH2 inhibition in both cell lines. Conversely, we demonstrated that MDM2/MDM4 inhibition, in a TP53-dependant manner, strongly synergized with EZH2 inhibition to control cell lines viability in vitro. We further treated 4 rhabdoid patient-derived xenografts with UNC1999, Idasanutlin and the combination of both and observed a potent tumor growth control upon combined EZH2 and MDM2/MDM4 inhibitions. To conclude, our results strongly encourage to assess the actual efficacy of combined MDM2/MDM4 and EZH2 inhibitors in the treatment of patients with SMARCB1-deficient rhabdoid tumors.
 Citation Format: Céline Chauvin, Tiphaine Hery, Rachida Bouarich, Fariba Nemati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, Sergio Roman-Roman, Didier Decaudin, Raphaël Margueron, Franck Bourdeaut. Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A051.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85bf34da504cd580597c4be8039c1663b824b215" target='_blank'>
              Abstract A051 Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors
              </a>
            </td>
          <td>
            C. Chauvin, Tiphaine Héry, Rachida Bouarich, F. Némati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, S. Roman-Roman, Didier Decaudin, Raphaël Margueron, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Relapse remains a determinant of treatment failure and contributes significantly to mortality in acute myeloid leukemia (AML) patients. Despite efforts to understand AML progression and relapse mechanisms, findings on acquired gene mutations in relapse vary, suggesting inherent genetic heterogeneity and emphasizing the role of epigenetic modifications. We conducted a multi-omic analysis using Omni-C, ATAC-seq, and RNA-seq on longitudinal samples from two adult AML patients at diagnosis and relapse. Herein, we characterized genetic and epigenetic changes in AML progression to elucidate the underlying mechanisms of relapse. Differential interaction analysis showed significant 3D chromatin landscape reorganization between relapse and diagnosis samples. Comparing global open chromatin profiles revealed that relapse samples had significantly fewer accessible chromatin regions than diagnosis samples. In addition, we discovered that relapse-related upregulation was achieved either by forming new active enhancer contacts or by losing interactions with poised enhancers/potential silencers. Altogether, our study highlights the impact of genetic and epigenetic changes on AML progression, underlining the importance of multi-omic approaches in understanding disease relapse mechanisms and guiding potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce349ab8147752454b7d482ecbf7663d5857bab7" target='_blank'>
              Multi-omic analysis of longitudinal acute myeloid leukemia patient samples reveals potential prognostic markers linked to disease progression
              </a>
            </td>
          <td>
            Nisar Ahmed, Irene Cavattoni, William Villiers, Chiara Cugno, S. Deola, Borbala Mifsud
          </td>
          <td>2024-09-27</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="Prostate cancer (PC), which is a disease driven by the activity of the androgen receptor (AR), is the most commonly diagnosed malignancy and despite advances in diagnostic and treatment strategies, PC is the second most common cause of cancer mortality in men (Bray et al., 2018). Taxane-based chemotherapy is the only chemotherapy that prolongs survival in metastatic PC patients (Petrylak et al., 2004; Tannock et al., 2004). At the cellular level, taxanes bind to and stabilize microtubules (MTs) inhibiting all MT-dependent intracellular pathways. MTs are highly dynamic polymers that stochastically switch between phases of growth, shrinkage, and pause (Jordan and Wilson, 2004). Altered MT dynamics endow cancer cells with both survival and migratory advantages (Mitchison, 2012). Taxanes inhibit MT dynamics and alter the spatial organization of the MT network, thereby inhibiting intracellular trafficking of molecular cargo critical for tumor survival. In PC specifically, taxanes inhibit transcriptional activity downstream of MT stabilization (Thadani-Mulero et al., 2012) and AR nuclear accumulation (Darshan et al., 2011; Zhu et al., 2010). Different tubulin inhibitors, even from within the same structural class as the taxanes, affect distinct parameters of MT dynamics (Jordan and Wilson, 2004), yet the selection of taxane for chemotherapy is not based on the particular patterns of dynamic behavior of the MT cytoskeleton in individual patients. We envisage that systematic characterization using quantitative analysis of MT dynamics in PC patient cells expressing clinically relevant protein isoforms (Matov et al., 2024; Thoma et al., 2010), before and after treatment with each of the taxanes, will allow us to identify criteria for the selection of the most suitable drug combination at the onset of treatment. We link MT dynamics in the presence of AR variants and sensitivity/resistance to taxanes and connect fundamental research with clinically relevant concepts to elucidate cellular mechanisms of clinical response to taxanes and, thus, advance the customization of therapy. Our computational live-cell analysis addresses questions in the context of the inherent differences in MT homeostasis as a function of AR content in PC cells, the specific parameters of MT dynamics each of the taxanes affects, and how can this information be used to match endogenous patterns of MT dynamics with drug-modulated MT behavior. We investigate whether the sensitivity to taxanes, evaluated by computational analysis of MTs, can be linked to gene expression driven by AR and its variants, and whether the resistance to taxanes be linked to the presence of a specific AR splice variant, and can we identify which of the taxanes will be most effective based on the endogenous patterns of MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e90d5e5c268c6138c1b50b9fd253e67162113bd" target='_blank'>
              Computational Analysis of Treatment Resistant Cancer Cells
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-08-31</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In recent decades, the field of genetics has revolutionized our understanding of biology. In particular, we know about how genetic information encodes the blueprint for life and how variations and mutations in these sequences contribute to disease or health. What is unique is that the genetic trait changes do not always caused by any changes in genetic sequences. 
Epigenetics refers to the study of how environmental factors and lifestyle choices can affect the gene activity. Changes in the heritable traits of organisms can be caused by modification of gene expression rather than alteration to the underlying DNA sequence. Therefore, this change does not involve any DNA mutation. 
DNA methylation is a biological process of adding a methyl group to the DNA molecule. The activity of DNA segments can change without changing the sequence by the process of methylation. This process plays an important role in normal development and aberrant DNA methylation patterns are associated with various diseases. Histones are highly basic proteins around which DNA is wound. The modifications in histone protein plays a fundamental role in most biological processes and can alter the accessibility of the DNA for transcription, as well as influencing gene expression. There are non-coding RNA molecules that do not encode proteins but play vital role in regulating the gene expression by interacting with other molecules and influencing chromatin structure. 
Over the past few decades, mechanisms of epigenetics have emerged as a crucial factor in governing every biological process in the human body. Throughout human life, these mechanisms have been impacted various physiological and pathological conditions. For example, cancer is often characterized by epigenetics abnormalities, such as hypermethylation of tumor suppressor genes or hypomethylation of oncogenes. Epigenetic modifications are also implicated in complex diseases like cardiovascular disease, autoimmune conditions and neurodegenerative disorder. Beyond these diseases, epigenetics also offers insights into health and development. Furthermore, understanding the epigenetic mechanisms influence human health, leading to new preventive strategies and treatment of various diseases. Early life experiences, including nutrition and stress, can lead to epigenetic changes that affect long-term health outcomes of the individual.   
Advances in this research can pinpoint any epigenetic changes that affect long-term health outcomes. The advances of epigenetic research can lead to early diagnosis and preventive measures. Identifying the epigenetic markers associated with the diseases enable early detection of the diseases and more effective preventative strategies. Some epigenetic therapies like; drugs that modify histone acetylation or DNA methylation, are already in development and provide future treatments for disease. To understand how life style factors, influence epigenetic regulation can lead to personalized lifestyle recommendations that influence the public health strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6963fc644883e01d78b4f2a627be037841fc025" target='_blank'>
              The Role of Epigenetics in Disease and Health: Beyond DNA Sequences
              </a>
            </td>
          <td>
            .. Diki
          </td>
          <td>2024-08-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Pathogenic variations in the fused in sarcoma (FUS) gene are associated with rare and aggressive forms of amyotrophic lateral sclerosis (ALS). As FUS-ALS is a dominant disease, a targeted, allele-selective approach to FUS knockdown is most suitable. Antisense oligonucleotides (AOs) are a promising therapeutic platform for treating such diseases. In this study, we have explored the potential for allele-selective knockdown of FUS. Gapmer-type AOs targeted to two common neutral polymorphisms in FUS were designed and evaluated in human fibroblasts. AOs had either methoxyethyl (MOE) or thiomorpholino (TMO) modifications. We found that the TMO modification improved allele selectivity and efficacy for the lead sequences when compared to the MOE counterparts. After TMO-modified gapmer knockdown of the target allele, up to 93% of FUS transcripts detected were from the non-target allele. Compared to MOE-modified AOs, the TMO-modified AOs also demonstrated reduced formation of structured nuclear inclusions and SFPQ aggregation that can be triggered by phosphorothioate-containing AOs. How overall length and gap length of the TMO-modified AOs affected allele selectivity, efficiency and off-target gene knockdown was also evaluated. We have shown that allele-selective knockdown of FUS may be a viable therapeutic strategy for treating FUS-ALS and demonstrated the benefits of the TMO modification for allele-selective applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40bef67258a89242d3fafd4ece2697b7aa372f10" target='_blank'>
              Allele-Selective Thiomorpholino Antisense Oligonucleotides as a Therapeutic Approach for Fused-in-Sarcoma Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            R. Mejzini, Marvin H. Caruthers, Balazs Schafer, Ondrej Kostov, Kavitha Sudheendran, Marija Ciba, Mathias Danielsen, Steve D. Wilton, P. A. Akkari, L. L. Flynn
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Pediatric cancers are estimated to be related to a genetic predisposition in about 8 to 10% of cases. This estimation mostly relies on already discovered cancer predisposing genes. However, the numerous new cancer predisposing genes that have been discovered in the past decade thanks to high throughput sequencing in vast cohort of pediatric patients illustrate that new genes may still be unravelled. Moreover, the frequency of early post-zygotic mosaicism is being readdressed thanks to more dedicated sequencing and research. The actual clinical impact of all those discoveries is the main challenge for the community: defining the penetrance, delineating the cancer spectrum and therefore the appropriate surveillance, and setting up innovative follow-up techniques are necessary. Here, we’ll describe recent findings on brain tumor predisposing genes (ELP1 and SMARCB1, among others) to illustrate the increasing recognition of genetic predisposition in pediatric cancers and how the clinical translation of these laboratory findings remain challenging at clinical and ethical levels.
 Citation Format: Franck Bourdeaut. Genetic predisposition to pediatric cancer: the long way from discovery to surveillance guidelines [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr IA003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc6b854ba212ba8f4846fc2365983567ae6c2486" target='_blank'>
              Abstract IA003: Genetic predisposition to pediatric cancer: the long way from discovery to surveillance guidelines
              </a>
            </td>
          <td>
            F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="In the era of precision oncology, identifying abnormal genetic and epigenetic alterations has transformed the way cancer is diagnosed, managed, and treated. 5-hydroxymethylcytosine (5hmC) is an emerging epigenetic modification formed through the oxidation of 5-methylcytosine (5mC) by ten-eleven translocase (TET) enzymes. DNA hydroxymethylation exhibits tissue- and cancer-specific patterns and is essential in DNA demethylation and gene regulation. Recent advancements in 5hmC detection methods and the discovery of 5hmC in cell-free DNA (cfDNA) have highlighted the potential for cell-free 5hmC as a cancer biomarker. This review explores the current and emerging techniques and applications of DNA hydroxymethylation in cancer, particularly in the context of cfDNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6579b28c064cf45b3665a8e9a1ea5968846de367" target='_blank'>
              Cell-Free DNA Hydroxymethylation in Cancer: Current and Emerging Detection Methods and Clinical Applications
              </a>
            </td>
          <td>
            Janice J. N. Li, Geoffrey Liu, Benjamin H. Lok
          </td>
          <td>2024-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31c537bce8789a914d584c53915429d4b9a4119c" target='_blank'>
              Prognosis and treatment in acute myeloid leukemia: a comprehensive review
              </a>
            </td>
          <td>
            Ankita Debnath, S. Nath
          </td>
          <td>2024-08-06</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Myelodysplastic syndrome and acute myeloid leukemia (AML) belong to a continuous disease spectrum of myeloid malignancies with poor prognosis in the relapsed/refractory setting necessitating novel therapies. Natural killer (NK) cells from patients with myeloid malignancies display global dysfunction with impaired killing capacity, altered metabolism, and an exhausted phenotype at the single-cell transcriptomic and proteomic levels. In this study, we identified that this dysfunction was mediated through a cross-talk between NK cells and myeloid blasts necessitating cell-cell contact. NK cell dysfunction could be prevented by targeting the αvβ-integrin/TGF-β/SMAD pathway but, once established, was persistent because of profound epigenetic reprogramming. We identified BATF as a core transcription factor and the main mediator of this NK cell dysfunction in AML. Mechanistically, we found that BATF was directly regulated and induced by SMAD2/3 and, in turn, bound to key genes related to NK cell exhaustion, such as HAVCR2, LAG3, TIGIT, and CTLA4. BATF deletion enhanced NK cell function against AML in vitro and in vivo. Collectively, our findings reveal a previously unidentified mechanism of NK immune evasion in AML manifested by epigenetic rewiring and inactivation of NK cells by myeloid blasts. This work highlights the importance of using healthy allogeneic NK cells as an adoptive cell therapy to treat patients with myeloid malignancies combined with strategies aimed at preventing the dysfunction by targeting the TGF-β pathway or BATF. BATF deletion prevents epigenetic rewiring and enhances the antitumor activity of NK cells in AML. Editor’s summary Patients with myeloid malignancies, including myelodysplastic syndrome (MDS) and acute myeloid leukemia (AML), often relapse after standard of care. To improve outcomes, Kumar et al. have investigated the role of natural killer (NK) cells in antitumor immunity against myeloid blasts and have identified global dysfunction of NK cells in these patients. They identified BATF as responsible for this dysfunction, and by knocking out BATF downstream of the TGF-β canonical pathway in NK cells, they were able to enhance NK cell function against AML in vitro and in vivo. This has led to the initiation of a promising clinical trial using these TGFBR2 knockout NK cells against myeloid malignancies. —Dorothy Hallberg">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1bf3fa2449127d2f41a09515bf651d31d92c80" target='_blank'>
              BATF is a major driver of NK cell epigenetic reprogramming and dysfunction in AML
              </a>
            </td>
          <td>
            Bijender Kumar, A. Singh, R. Başar, N. Uprety, Ye Li, Huihui Fan, Ana Karen Nunez Cortes, Mecit Kaplan, S. Acharya, Hila Shaim, Anna C Xu, Manrong Wu, Emily Ensley, Dexing Fang, P. Banerjee, L. M. Garcia, S. Tiberti, Paul Lin, H. Rafei, Maliha Nuzhat Munir, Madison Moore, M. Shanley, M. Mendt, L. Kerbauy, Bin Liu, Alexander Biederstädt, Elif Gokdemir, Susmita Ghosh, Kiran Kundu, Francia Reyes-Silva, X. Jiang, Xinhai Wan, April L. Gilbert, Merve Dede, V. Mohanty, J. Dou, Patrick Zhang, Enli Liu, Luis Muniz-Feliciano, Gary M. Deyter, Abhinav K. Jain, J. J. Rodríguez-Sevilla, S. Colla, Guillermo Garcia-Manero, Elizabeth J. Shpall, Ken Chen, Hussein A. Abbas, Kunal Rai, Katayoun Rezvani, M. Daher
          </td>
          <td>2024-09-11</td>
          <td>Science Translational Medicine</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24d8d8f2f89f8ee70047ce83a13996f46168eafe" target='_blank'>
              Star wars against leukemia: attacking the clones.
              </a>
            </td>
          <td>
            Monika M. Toma, Tomasz Skorski
          </td>
          <td>2024-09-02</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genome sequencing has revealed frequent mutations in Ras homolog family member A (RHOA) among various cancers with unique aberrant profiles and pathogenic effects, especially in peripheral T-cell lymphoma (PTCL). The discrete positional distribution and types of RHOA amino acid substitutions vary according to the tumor type, thereby leading to different functional and biological properties, which provide new insight into the molecular pathogenesis and potential targeted therapies for various tumors. However, the similarities and discrepancies in characteristics of RHOA mutations among various histologic subtypes of PTCL have not been fully elucidated. Herein we highlight the inconsistencies and complexities of the type and location of RHOA mutations and demonstrate the contribution of RHOA variants to the pathogenesis of PTCL by combining epigenetic abnormalities and activating multiple downstream pathways. The promising potential of targeting RHOA as a therapeutic modality is also outlined. This review provides new insight in the field of personalized medicine to improve the clinical outcomes for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd71cf13763be0cc23a4dc23d32cf828f50620b5" target='_blank'>
              Mutations in Ras homolog family member A in patients with peripheral T-cell lymphoma and implications for personalized medicine
              </a>
            </td>
          <td>
            Lina Hu, Xuanye Zhang, Shengbing Zang
          </td>
          <td>2024-08-09</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-cell transcriptomics is valuable for uncovering individual cell properties, particularly in highly heterogeneous systems. However, this technique often results in the analysis of many well- characterized cells, increasing costs and diluting rare cell populations. To address this, we developed PURE-seq (PIP-seq for Rare-cell Enrichment and Sequencing) for scalable sequencing of rare cells. PURE-seq allows direct cell loading from FACS into PIP-seq reactions, minimizing handling and reducing cell loss. PURE-seq reliably captures rare cells, with 60 minutes of sorting capturing tens of cells at a rarity of 1 in 1,000,000. Using PURE-seq, we investigated murine long- term hematopoietic stem cells and their transcriptomes in the context of hematopoietic aging, identifying Egr1 as a potential master regulator of hematopoiesis in the aging context. PURE-seq offers an accessible and reliable method for isolating and sequencing cells that are currently too rare to capture successfully with existing methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3803f1982914d2f84b2b9f714aff12641ae7e08e" target='_blank'>
              PURE-seq identifies Egr1 as a Potential Master Regulator in Murine Aging by Sequencing Long-Term Hematopoietic Stem Cells
              </a>
            </td>
          <td>
            Sixuan Pan, Kai-chun Chang, Ines Fernandez-Maestre, S. Haver, Matthew Wereski, Robert L. Bowman, Ross L. Levine, A. Abate
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Background: Transposable elements (TEs) are dynamic repetitive regions which generate mutations and structural variants. TP53 plays a crucial role in suppressing TE movement to maintain genomic stability. The relationship between TP53 and TEs has been extensively studied in tumours, but not the germline. Individuals with germline TP53 pathogenic variants have Li-Fraumeni Syndrome (LFS), a cancer predisposition syndrome with a high lifetime risk of cancer in various tissues. This study aims to characterize the TE landscape in individuals with Li-Fraumeni Syndrome and determine how this contributes to their increased cancer risk. Methods: MELT and xTea were used to identify TEs in children with (n=48) or without (n=198) a germline TP53 variant. TE calls were merged with SURVIVOR and variants were annotated with AnnotSV. To assess the influence of TP53 on TE location, we quantified TEs across genomic windows and identified the top 100 significantly different regions with the Mann-Whitney U test, adjusting for multiple comparisons. We used these regions to develop a gradient-boosted tree model with 5-fold cross-validation to predict TP53 status. Results: Individuals with germline TP53 variants harboured significantly fewer ALU and LINE1 elements in their germline genome compared to the control dataset (p<0.001). The phenomenon was consistent across chromosomes and significant in chromosomes 5 and 11 (FDR<0.05). A gradient-boosted tree model was able to differentiate patients with and without a germline TP53 variant with an AUPRC of 0.77 on an unseen test set. Conclusion: Germline TP53 variants may effect the frequency and location of germline TE insertions, which may influence nearby genomic variations and lead to cancer development. Analyzing TEs in individuals with LFS will enhance our understanding of accelerated cancer development in these patients, informing future research for diagnostic and therapeutic approaches.
 Citation Format: Brianne Laverty, Shilpa Yadahalli, Vallijah Subasri, David Malkin. The role of TP53 on transposable elements in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4995c29bd2050c705f9a5b8da916aacb0aa2601c" target='_blank'>
              Abstract A010 The role of TP53 on transposable elements in pediatric cancer
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Vallijah Subasri, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Predicting the effects of non-coding mutations on DNA methylation is crucial for advancing our understanding of gene expression, epigenetic inheritance, and its role in disease mechanisms. Current methods lack the capability to predict the impact of non-coding mutations on DNA methylation at single-cell resolution and long range, while remain challenges in tracking SNP influences throughout disease progression. Here, we introduce Methven, a deep learning-based framework designed to predict the effects of non-coding mutations on DNA methylation at single-cell resolution, to overcome the challenges. Methven integrates DNA sequences and ATAC-seq data, employing a divide-and-conquer approach to handle varying scales of SNP-CpG interactions. By leveraging a pretrained DNA language model, Methven accurately predicts both the direction and magnitude of methylation changes across a 100kbp range with a lightweight architecture. The evaluation results demonstrate the superior performance of Methven in prioritizing functional non-coding mutation, model interpretability, and its potential for revealing personalized mutation-disease associations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/90c29a5330f80b31c8ec7716b2f8598345b09c61" target='_blank'>
              Predicting the effect of non-coding mutations on single-cell DNA methylation using deep learning
              </a>
            </td>
          <td>
            Zhe Liu, An Gu, Yihang Bao, Guan Ning Lin
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma (HCC) is one of the most common malignant tumors of the digestive system. RNA methylation plays an important role in tumorigenesis and metastasis, which could alter gene expression and even function at multiple levels, such as RNA splicing, stability, translocation, and translation. In this study, we aimed to conduct a comprehensive analysis of RNA methylation-related genes (RMGs) in HCC and their relationship with survival and clinical features. Methods A retrospective analysis was performed using publicly available HCC-related datasets. The differentially expressed genes (DEGs) between HCC and controls were identified from TCGA-LlHC and intersected with RMGs to obtain differentially expressed RNA methylation-related genes (DERMGs). Regression analysis was used to screen for prognostic genes and construct risk models. Simultaneously, clinical, immune infiltration and therapeutic efficacy analyses were performed. Finally, multivariate cox regression was used to identify independent risk factors, and quantitative real-time polymerase chain reaction (qRT-PCR) was used to validate the expression levels of the core genes of the model. Results A 21-gene risk model for HCC was established with excellent performance based on ROC curves and survival analysis. Risk scores correlated with tumor grade, pathologic T, and TNM stage. Immune infiltration analysis showed correlations with immune scores, 11 immune cells, and 30 immune checkpoints. Low-risk patients showed a higher susceptibility to immunotherapy. The risk score and TNM stage were independent prognostic factors. qRT-PCR confirmed higher expression of PRDM9, ALPP, and GAD1 in HCC. Conclusions This study identified RNA methylation-related signature genes in HCC and constructed a risk model that predicts patient outcomes and reflects the immune microenvironment. Prognostic genes are involved in complex regulatory mechanisms, which may be useful for cancer diagnosis, prognosis, and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288e5a95cc7c2059fde0835f504837f6d0a5d61b" target='_blank'>
              A Novel RNA Methylation-Related Prognostic Signature and its Tumor Microenvironment Characterization in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Luzheng Liu, Jiacheng Chen, Fei Ye, Yanggang Yan, Yong Wang, Jincai Wu
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND AND OBJECTIVE
The identification of mutation hot spots in the isocitrate dehydrogenase (IDH) genes is one of the most important cancer genome-wide sequencing discoveries with relevant impact in the treatment of some orphan tumors. These genes were mostly found mutated in lower-grade gliomas (LGGs), acute myeloid leukaemia (AML), myelodysplastic syndromes (MDS) and myeloproliferative neoplasms (MPNs) and in cholangiocarcinoma. This aberrant genomic condition represents a therapeutic target of great interest in cancer research, especially in AML, given the limitations of currently approved therapies in this field. In this review, we investigate the role of IDH mutation and the mutant IDH (mIDH)- targeted therapies for cholangiocarcinoma and glioma.


METHODS
Here, we provide an overview of the IDH mutation role and discuss its role in tumorigenesis and progression of some solid cancers, in which the therapeutic strategy can be completely changed thanks to these brand-new therapeutic options.


KEY CONTENT AND FINDINGS
The encouraging early clinical data demonstrated to be a proof of concept for investigational mIDH1/2 inhibitors in tumors with a paucity of therapeutic possibilities.


CONCLUSIONS
Moreover, we list the most important randomised clinical trials still active with their preliminary results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a20edde7b8a46ad9d9cedbb5e90b751d02d2ad2f" target='_blank'>
              Therapeutic inhibition of isocitrate dehydrogenase mutations in glioma and cholangiocarcinoma: new insights and promises-a narrative review.
              </a>
            </td>
          <td>
            V. Internò, A. Melaccio, Pasquale Vitale, Roberta Spedaliere, Massimo Buonfantino, R. Messina, Anna Laura Lippolis, Francesco Signorelli, Raffaele Addeo, Francesco Giuliani
          </td>
          <td>2023-01-01</td>
          <td>Chinese clinical oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e16f71ff68478a0f30f71105d86b66a3bf8d6b91" target='_blank'>
              Single-Cell Sequencing: High-Resolution Analysis of Cellular Heterogeneity in Autoimmune Diseases.
              </a>
            </td>
          <td>
            Xuening Tang, Yudi Zhang, Hao Zhang, Nan Zhang, Ziyu Dai, Q. Cheng, Yongzhen Li
          </td>
          <td>2024-08-26</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract Single-cell transcriptomics is valuable for uncovering individual cell properties, particularly in highly heterogeneous systems. However, this technique often results in the analysis of many well-characterized cells, increasing costs and diluting rare cell populations. To address this, we developed PURE-seq (PIP-seq for Rare-cell Enrichment and Sequencing) for scalable sequencing of rare cells. PURE-seq allows direct cell loading from FACS into PIP-seq reactions, minimizing handling and reducing cell loss. PURE-seq reliably captures rare cells, with 60 minutes of sorting capturing tens of cells at a rarity of 1 in 1,000,000. Using PURE-seq, we investigated murine long-term hematopoietic stem cells and their transcriptomes in the context of hematopoietic aging, identifying Egr1 as a potential master regulator of hematopoiesis in the aging context. PURE-seq offers an accessible and reliable method for isolating and sequencing cells that are currently too rare to capture successfully with existing methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/745f57640b639dfa9ea54ff6bc7c8ea207a49f2c" target='_blank'>
              PURE-seq identifies Egr1 as a Potential Master Regulator in Murine Aging by Sequencing Long-Term Hematopoietic Stem Cells
              </a>
            </td>
          <td>
            A. Abate, Sixuan Pan, Kai-chun Chang, Ines Fernandez-Maestre, S. Haver, Matthew Wereski, Robert L. Bowman, Ross Levine
          </td>
          <td>2024-08-13</td>
          <td>Research Square</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cystic fibrosis (CF) is caused by mutations in the cystic fibrosis transmembrane conductance regulator (CFTR) gene. Although many people with CF (pwCF) are treated using CFTR modulators, some are non-responsive due to their genotype or other uncharacterized reasons. Autologous airway stem cell therapies, in which the CFTR cDNA has been replaced, may enable a durable therapy for all pwCF. Previously, CRISPR-Cas9 with two AAVs was used to sequentially insert two halves of the CFTR cDNA and an enrichment cassette into the CFTR locus. However, the editing efficiency was <10% and required enrichment to restore CFTR function. Further improvement in gene insertion may enhance cell therapy production. To improve CFTR cDNA insertion in human airway basal stem cells (ABCs), we evaluated the use of the small molecules AZD7648 and ART558 which inhibit non-homologous end joining (NHEJ) and micro-homology mediated end joining (MMEJ). Adding AZD7648 alone improved gene insertion by 2-3-fold. Adding both ART558 and AZD7648 improved gene insertion but induced toxicity. ABCs edited in the presence of AZD7648 produced differentiated airway epithelial sheets with restored CFTR function after enrichment. Adding AZD7648 did not increase off-target editing. Further studies are necessary to validate if AZD7648 treatment enriches cells with oncogenic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b76a5973a8822632977941f52ade0b5a68b63d15" target='_blank'>
              DNA-PKcs Inhibition Improves Sequential Gene Insertion of the Full-Length CFTR cDNA in Airway Stem Cells
              </a>
            </td>
          <td>
            Jacob T. Stack, R. Rayner, Reza Nouri, Carlos J. Suarez, Sun Hee Kim, K. Kanke, Tatyana A. Vetter, E. Cormet-Boyaka, S. Vaidyanathan
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Olaparib and temozolomide (OT) combination therapy is in clinical trial evaluation for adolescent rhabdomyosarcoma (RMS) muscle cancers. Unfortunately, resistance to OT has been reported in other cancers, with no counterstrategies available. Using preclinical mouse xenograft experiments, we show that OT is effective at curbing RMS growth, yet a subset of tumors develop resistance that is associated with transcriptomic changes that occur in the absence of recurrent genomic mutation. Importantly, a vast majority of resistant RMS models upregulate the PIK3CA/AKT pathway, which in turn activates NRF2 transcription factor phosphorylation and subsequent transcriptional expression of multidrug resistance ABC transport proteins that rapidly efflux drugs from cells. Using dynamic real-time imaging of tumor growth and assessing drug responses at single-cell resolution in human cell line models, we found that the PIK3CA inhibitor alpelisib re-sensitized resistant RMS cells to OT therapy by suppressing the expression of ABC transport proteins. Excitingly, RMS uses the same PIK3CA/AKT pathway activation to drive resistance to standard-of-care combination therapy vincristine, actinomycin D, and cyclophosphamide (VAC) and the combination of OT + alpelisib effectively killed VAC-resistant RMS. Alpelisib also re-sensitized resistant RMS to chemotherapy in preclinical xenograft mouse models resulting in reduced tumor burden and extended disease-free survival. Our work defines a common resistance pathway in RMS and has credentialled a new preclinical strategy to kill therapy-resistant RMS.
 Citation Format: Yueyang Wang, Qiqi Yang, Chuan Yan, David M. Langenau. The PIK3CA/AKT pathway drives therapy resistance in rhabdomyosarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A072.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de779864c638c41dc5737016f4c37a97b2fb0f5c" target='_blank'>
              Abstract A072 The PIK3CA/AKT pathway drives therapy resistance in rhabdomyosarcoma
              </a>
            </td>
          <td>
            Yueyang Wang, Qiqi Yang, Chuan Yan, D. Langenau
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a disease characterized by spatiotemporal heterogeneity of tumor clones. Different genetic aberrations can be observed simultaneously in tumor cells from different loci, and as the disease progresses, new subclones may appear. The role of liquid biopsy, which is based on the analysis of tumor DNA circulating in the blood plasma, continues to be explored in MM. Here, we present an analysis of the STR profiles and mutation status of the KRAS, NRAS, and BRAF genes, evaluated in plasma free circulating tumor DNA (ctDNA), CD138+ bone marrow cells, and plasmacytomas. The prospective single-center study included 97 patients, with a median age of 55 years. Of these, 94 had newly diagnosed symptomatic MM, and three had primary plasma cell leukemia. It should be noted that if mutations were detected only in ctDNA, “non-classical” codons were more often affected. A variety of adverse laboratory and clinical factors have been associated with the detection of rare KRAS or NRAS gene mutations in bone marrow or ctDNA, suggesting that these mutations may be factors of an unfavorable prognosis for MM. Liquid biopsy studies provide undeniable fundamental information about tumor heterogeneity and clonal evolution in MM. Moreover, we focus on using liquid biopsy to identify new high-risk factors for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef6c42652b61bb294400045b1d3bc2249d75f9b" target='_blank'>
              Loss of Heterozygosity and Mutations in the RAS-ERK Pathway Genes in Tumor Cells of Various Loci in Multiple Myeloma
              </a>
            </td>
          <td>
            M. Soloveva, M. Solovev, N. Risinskaya, E. Nikulina, I. Yakutik, B. Biderman, Tatiana Obukhova, Y. Chabaeva, Sergej Kulikov, Andrey Sudarikov, L. Mendeleeva
          </td>
          <td>2024-08-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Sarcomas constitute a large group of rare, heterogeneous cancers of the bone and soft tissue affecting children and young adults. There are over 80 clinically distinct subtypes of sarcoma, and their heterogeneity is a contributing factor to the moderate and poor survival outcomes observed in primary and metastatic disease, respectively. While they are heterogeneous, many subtypes harbor a balanced chromosomal translocation fusing the Ewing Sarcoma RNA binding protein 1 (EWSR1) gene with a gene encoding a transcription factor (TF). This translocation generates two distinct fusions: a canonical fusion consisting of the N-terminal transactivation domain of EWSR1 and the C-terminal DNA-binding domain of a TF, and a reciprocal fusion fusing the N-terminus of the TF and C-terminal RNA-binding domain of EWSR1. Canonical fusion protein products exhibit aberrant TF and pioneer factor activity, leading to alterations in transcriptional programs and the epigenetic landscape within a cell that contributes to oncogenesis. The functions of the reciprocal fusion proteins, however, have remained elusive. Ewing sarcoma (ES) and clear cell sarcoma (CCS), a bone and soft tissue sarcoma, respectively, are two subtypes that have been shown to express reciprocal fusions. The ES fusion implicates EWSR1 and Friend Leukemia Integration Factor 1 (FLI1), while the CCS fusion involves Activating Transcription Factor 1 (ATF1). A challenge in studying the roles of these and other fusion proteins in sarcoma development is the lack of effective model systems, primarily due to their toxicity upon exogenous introduction into fusion-naïve cells. Here, we outline the development of a cumate-inducible lentiviral vector to study the canonical and reciprocal EWSR1 fusion proteins present in ES and CCS. To track cellular localization of the fusion proteins and distinguish fusions from their respective wildtype proteins, we designed fusion sequences with N-terminal fluorescent tags. Tightly controlled expression of canonical and reciprocal fusion proteins reduces their toxicity when expressed in cells. We have successfully established 293T and HT-1080 cells that stably express these vectors. We have also validated cumate-inducible expression of fluorescently tagged canonical and reciprocal FLI1 and ATF1 fusions at the RNA and protein levels in both cell lines. The canonical fusion proteins produced by our model have the expected effects on expression levels of known downstream canonical fusion target genes. Initial cellular localization studies using our model and fluorescence microscopy indicate that reciprocal fusion proteins may localize in the nucleus, which was previously unknown. Overall, our cumate-inducible model represents a viable system for fusion protein expression. Using this model, we aim to gain insight into the roles of reciprocal fusions in ES and CCS development with hopes of uncovering potentially targetable vulnerabilities, opening the door for the development of new treatments in a patient population in need of novel and efficacious therapies.
 Citation Format: Sarah Gawlak, Joyce Ohm. Development of model systems to investigate the roles of canonical and reciprocal EWSR1 fusion proteins in sarcomagenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B079.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6104d5296aa07ecd63e21655a418e97199e7a8ba" target='_blank'>
              Abstract B079: Development of model systems to investigate the roles of canonical and reciprocal EWSR1 fusion proteins in sarcomagenesis
              </a>
            </td>
          <td>
            Sarah Gawlak, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Combined tracking of clonal evolution and chimeric cell phenotypes could enable detection of the key cellular populations associated with response following therapy, including after allogeneic hematopoietic stem cell transplantation (HSCT). We demonstrate that mitochondrial DNA (mtDNA) mutations co-evolve with somatic nuclear DNA mutations at relapse post-HSCT and provide a sensitive means to monitor these cellular populations. Further, detection of mtDNA mutations via single-cell ATAC with select antigen profiling by sequencing (ASAP-seq) simultaneously determines not only donor and recipient cells, but also their phenotype, at frequencies of 0.1-1%. Finally, integration of mtDNA mutations, surface markers, and chromatin accessibility profiles enables the phenotypic resolution of leukemic populations from normal immune cells, thereby providing fresh insights into residual donor-derived engraftment and short-term clonal evolution following therapy for post-transplant leukemia relapse. As throughput evolves, we envision future development of single-cell sequencing-based post-transplant monitoring as a powerful approach for guiding clinical decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454ae3e4512380b5fd1a0c7892e2f418e6fca333" target='_blank'>
              Tracking rare single donor and recipient immune and leukemia cells after allogeneic hematopoietic cell transplantation using mitochondrial DNA mutations.
              </a>
            </td>
          <td>
            Livius Penter, Nicoletta Cieri, Katie Maurer, M. Kwok, Haoxiang Lyu, Wesley S Lu, Giacomo Oliveira, Satyen H Gohil, I. Leshchiner, Caleb A. Lareau, Leif S. Ludwig, Donna S. Neuberg, Haesook T Kim, Shuqiang Li, L. Bullinger, Jerome Ritz, Gaddy Getz, Jacqueline S Garcia, Robert J. Soiffer, K. Livak, Catherine J. Wu
          </td>
          <td>2024-09-05</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Philadelphia chromosome-positive B-cell acute lymphoblastic leukemia (Ph(+)B-ALL) is a hematological malignancy with a poor prognosis. Epigenetic abnormalities, especially abnormal histone acetylation and microRNAs (miRNAs) dysregulation, are a group of epigenetic patterns that contribute to leukemia progression. However, their regulatory mechanisms in Ph(+)B-ALL have not been fully elucidated. In this study, we identified that miR-183-5p is significantly downregulated in Ph(+)B-ALL and associated with poor prognosis. Moreover, we found that the BCR-ABL fusion gene is a key target gene of miR-183-5p. MiR-183-5p directly targets BCR-ABL gene and induces cell apoptosis via PTEN/AKT and c-MYC signaling pathways. In addition, histone deacetylase inhibitor (HADCi) could mitigate the suppressive effects of HDAC2 on miR-183-5p by promoting promoter acetylation, thereby enhancing cell apoptosis. In conclusion, our results indicate that miR-183-5p is a potential biomarker and suggest that a novel "HDAC2-miR-183-5p epigenetic circuitry regulation" may be involved in the pathogenesis of Ph(+)B-ALL. Taken together, These findings provide new insights into the design of promising molecular-targeted drugs for Ph(+)B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d13162cad787b4066b1367ec7518bbaa77c34746" target='_blank'>
              HDAC2-miR183-5p Epigenetic Circuit Contributes to the Growth of Philadelphia Chromosome-Positive B-cell Acute Lymphoblastic Leukemia via PTEN/AKT and c-MYC Signaling Pathway.
              </a>
            </td>
          <td>
            Yangyang Ding, Xiangjiang Feng, Zelin Liu, Ya Liao, Lianfang Pu, Jun Liu, Huiping Wang, Zhimin Zhai, Shu-Dao Xiong
          </td>
          <td>2024-09-11</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7189b9bbbd6276b2c06531ac44b4da640c222158" target='_blank'>
              Small cell lung cancer: emerging subtypes, signaling pathways, and therapeutic vulnerabilities
              </a>
            </td>
          <td>
            Jing Zhang, Xiaoping Zeng, Qiji Guo, Zhenxin Sheng, Yan Chen, Shiyue Wan, Lele Zhang, Peng Zhang
          </td>
          <td>2024-08-05</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8b3281c13d1ffc75f1261f03703dc8638c833" target='_blank'>
              A protein expression atlas on tissue samples and cell lines from cancer patients provides insights into tumor heterogeneity and dependencies.
              </a>
            </td>
          <td>
            Jun Li, Wei Liu, K. Mojumdar, Hong Kim, Zhicheng Zhou, Z. Ju, Shweth V. Kumar, P. Ng, Han Chen, Michael A. Davies, Yiling Lu, Rehan Akbani, Gordon B. Mills, Han Liang
          </td>
          <td>2024-09-03</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="TET2 mutations (mTET2) are common genetic events in myeloid malignancies and clonal hematopoiesis (CH). These mutations arise in the founding clone and are implicated in many clinical sequelae associated with oncogenic feedforward inflammatory circuits. However, the direct downstream effector of mTET2 responsible for the potentiation of this inflammatory circuit is unknown. To address this, we performed scRNA and scATAC-seq in COVID-19 patients with and without TET2-mutated CH reasoning that the inflammation from COVID-19 may highlight critical downstream transcriptional targets of mTET2. Using this approach, we identified MALAT1, a therapeutically tractable lncRNA, as a central downstream effector of mTET2 that is both necessary and sufficient to induce the oncogenic pro-inflammatory features of mTET2 in vivo. We also elucidate the mechanism by which mTET2 upregulate MALAT1 and describe an interaction between MALAT1 and P65 which leads to RNA "shielding" from PP2A dephosphorylation thus preventing resolution of inflammatory signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e647627f6805fe19dbf59a6bc50dfd7de16f1c5" target='_blank'>
              RNA shielding of P65 is required to potentiate oncogenic inflammation in TET2 mutated clonal hematopoiesis.
              </a>
            </td>
          <td>
            N. Ben-Crentsil, Wazim Mohammed Ismail, M. Balasis, Hannah Newman, Ariel Quintana, M. Binder, Traci L. Kruer, Surendra Neupane, M. Ferrall-Fairbanks, Jenna A. Fernandez, T. Lasho, C. Finke, Mohammed L Ibrahim, K. McGraw, Michael Wysota, Amy L Aldrich, Christopher B Ryder, Christopher T Letson, Joshua Traina, Amy F. McLemore, N. Droin, A. Shastri, Seongseok Yun, E. Solary, D. Sallman, Amer A Beg, Li Ma, Alexandre Gaspar-Maia, Mirinal S Patnaik, E. Padron
          </td>
          <td>2024-08-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The question whether interference with the ubiquitous splicing machinery can lead to cell-type specific perturbation of cellular function is addressed here by T cell specific ablation of the general U5 snRNP assembly factor CD2BP2/U5–52K. This protein defines the family of nuclear GYF domain containing proteins that are ubiquitously expressed in eukaryotes with essential functions ascribed to early embryogenesis and organ function. Abrogating CD2BP2/U5–52K in T cells, allows us to delineate the consequences of splicing machinery interferences for T cell development and function. Increased T cell lymphopenia and T cell death are observed upon depletion of CD2BP2/U5–52K. A substantial increase in exon skipping coincides with the observed defect in the proliferation/differentiation balance in the absence of CD2BP2/U5–52K. Prominently, skipping of exon 7 in Mdm4 is observed, coinciding with upregulation of pro-apoptotic gene expression profiles upon CD2BP2/U5–52K depletion. Furthermore, we observe enhanced sensitivity of naïve T cells compared to memory T cells to changes in CD2BP2/U5–52K levels, indicating that depletion of this general splicing factor leads to modulation of T cell homeostasis. Given the recent structural characterization of the U5 snRNP and the crosslinking mass spectrometry data given here, design of inhibitors of the U5 snRNP conceivably offers new ways to manipulate T cell function in settings of disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e3f51e578d8a7f3d740bbe93224c2b622e1de6d" target='_blank'>
              The nuclear GYF protein CD2BP2/U5–52K is required for T cell homeostasis
              </a>
            </td>
          <td>
            Miriam Bertazzon, Almudena Hurtado-Pico, C. Plaza-Sirvent, M. Schuster, Marco Preussner, Benno Kuropka, Fan Liu, Andor Zenon Amandus Kirsten, X. J. Schmitt, Benjamin König, Miguel Álvaro-Benito, Esam T. Abualrous, G. I. Albert, S. Kliche, F. Heyd, Ingo Schmitz, Christian Freund
          </td>
          <td>2024-09-06</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Inborn errors of metabolism (IEMs) and immunity (IEIs) are Mendelian diseases in which complex phenotypes and patient rarity have limited clinical understanding. Whereas few genes have been annotated as contributing to both IEMs and IEIs, immunometabolic demands suggested greater functional overlap. Here, CRISPR screens tested IEM genes for immunologic roles and IEI genes for metabolic effects and found considerable previously unappreciated crossover. Analysis of IEMs showed that N-linked glycosylation and the hexosamine pathway enzyme Gfpt1 are critical for T cell expansion and function. Further, T helper (TH1) cells synthesized uridine diphosphate N-acetylglucosamine more rapidly and were more impaired by Gfpt1 deficiency than TH17 cells. Screening IEI genes found that Bcl11b promotes the CD4 T cell mitochondrial activity and Mcl1 expression necessary to prevent metabolic stress. Thus, a high degree of functional overlap exists between IEM and IEI genes, and immunometabolic mechanisms may underlie a previously underappreciated intersection of these disorders. The functional overlap of genes that cause inborn errors of metabolism and immunity reveals T cell metabolic regulators. Editor’s summary Inborn errors of metabolism (IEMs) and immunity (IEIs) are two groups of monogenic disorders whose study has offered insights into fundamental human biology. Patterson et al. used CRISPR screens of mouse CD4 T cells disrupting known IEM and IEI genes to test potential overlapping immunometabolic regulators of T cell function. This approach revealed hundreds of previously unappreciated players, including N-linked glycosylation and the de novo hexosamine synthesis enzyme Gfpt1, which were critical for T cell expansion and function. Moreover, Bcl11b, a player in thymic T cell differentiation, promoted mitochondrial activity and prevented metabolic stress in CD4 T cells. —Seth Thomas Scanlon">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92efd6d1fbfc773538594e8af1740d7df031781" target='_blank'>
              Functional overlap of inborn errors of immunity and metabolism genes defines T cell metabolic vulnerabilities
              </a>
            </td>
          <td>
            Andrew R. Patterson, Gabriel A. Needle, Ayaka Sugiura, Erin Q. Jennings, Channing Chi, KayLee K. Steiner, Emilie L. Fisher, Gabriella L. Robertson, Caroline Bodnya, Janet G. Markle, Ryan D. Sheldon, Russell G. Jones, Vivian Gama, J.C. Rathmell
          </td>
          <td>2024-08-16</td>
          <td>Science Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="
 Background: Significant progress has been made in identifying mutations and defining biological heterogeneity in neuroblastoma (NB) using high-throughput sequencing approaches. However, functional roles for many specific genetic alterations in NB initiation, growth, and metastasis remain to be fully defined. Pathogenic and/or likely pathogenic (P/LP) alterations in genes involved in DNA Damage Response (DDR) pathways were identified in approximately 30% of patients enrolled in the SickKids Cancer Sequencing Program (KiCS). Despite being well studied in many adult cancers, roles for DDR disruption and potential therapeutic implications remains poorly understood for NB and other pediatric solid tumors. Aims: We aimed to define roles for DDR-deficiency in NB progression and assess the therapeutic potential of targeted agents using patient-specific DDR loss-of-function NB models. Methods: We incorporated loss-of-function mutations in atm, brca2, palb2, and bard1 into an established zebrafish MYCN-driven model of human NB and characterized tumor progression, transcriptional signatures, and response to candidate therapeutics in vivo. Human cell lines with DDR protein knock-down were used to assess candidate drug combinations in vitro. Results: Mutations in DDR genes were found to enhance tumor penetrance and metastasis in MYCN-driven zebrafish NB, and resulted in upregulation of proliferation, cell cycle checkpoint, and DNA damage repair transcriptional signatures. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Conclusions: Our data supports an in vivo functional role for DDR-deficiency in NB, as well as potential molecular vulnerabilities. Pre-clinical assessments in vitro and in vivo suggest therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes.
 Citation Format: Madeline N. Hayes, Sarah Cohen-Gogo, Lynn Kee, Mehdi Layeghifard, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, Anita Villani, Adam Shlien, Daniel A. Morgenstern, Meredith S Irwin. DNA damage response deficiency enhances neuroblastoma progression and sensitivity to combination PARP and ATR inhibition [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A007.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1cc0578406ca9a79efb4aae714485b02b1ea3da3" target='_blank'>
              Abstract A007 DNA damage response deficiency enhances neuroblastoma progression and sensitivity to combination PARP and ATR inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, M. Layeghifard, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4d41c964ff5dfe59fec9c54a2237fb6fabb8d4" target='_blank'>
              Telomerase related molecular subtype and risk model reveal immune activity and evaluate prognosis and immunotherapy response in prostate cancer
              </a>
            </td>
          <td>
            Dongze Liu, Zheng Qin, Bo Yi, Hongbo Xie, Yunan Liang, Liang Zhu, Kuo Yang, Yong Xu, Hongtuan Zhang
          </td>
          <td>2024-08-17</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="
 Objective: Structural variations (SVs) constitute the greatest source of genomic alterations leading to the oncogenesis of many cancers, including the deadliest pancreatic cancer (PC). However, SVs in PC remain largely undefined due to technological limitations of the traditional short-read sequencing. This study aims to establish a comprehensive signature of genomic SVs in PC. Methods: PacBio whole genome long-read sequencing was employed to investigate the genomic landscape of SVs in eight tissues from four pancreatic cancer patients. A variety of SVs callers, including pbsv, Sniffles, cuteSV, and svim, were comprehensively adopted to improve the accuracy of SVs detection. Many undefined characteristics of germline and somatic SVs were revealed by comparing with multiple previously published databases. Besides, the impacts of structural variations on 3D chromatin organization were analyzed by integrating multi-omics data including Hi-C, RNA seq and ChIP seq. Results: A total of 26,844 and 27,028 non-redundant SVs were identified in human pancreatic cancer and matched para-cancerous tissues, respectively. Notably, among these SVs, 23.97% were novel and had not been previously reported. Germline SVs were further characterized and found enriched in 25 well-known susceptible genes, such as TP63, PVT1, and ABO. Additionally, we proposed a panel of germline SVs with predicted highest pathogenicity, involving STOX1, DSPP, and WRN. Totally, 616 somatic SVs were identified and presented high burdens in patients with lymph node metastasis. We observed that somatic DELs and INSs were significantly more prevalent in repetitive regions than in non-repetitive regions. Interestingly, DELs showed a decrease in the proportion occurring in simple repeat regions but an increase in the proportion occurring in LINE and SINE regions compared to INSs. Moreover, it was observed that deletions within topologically associated domains were associated with local enhanced interactions and disrupted chromatin loops, which might influence H3K27ac modification of histone contributing to enhancer or silencer hijacking. Conclusion: This study provides insights into the genomic signature of SVs in human pancreatic cancer. These findings might shed new light on the complexity of SVs and their pathogenesis by interplaying with chromatin organization.
 Citation Format: Yongxing Du, Xiaohao Zheng, Yunjie Duan, Chengfeng Wang. Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A083.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40a9fb18df1c1c8a73e8c29f87767e5466baa02" target='_blank'>
              Abstract A083: Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing
              </a>
            </td>
          <td>
            Yongxing Du, Xiaohao Zheng, Yunjie Duan, Cheng-Guang Wang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) stands as one of the most prevalent and aggressive forms of renal malignancies, accounting for a substantial portion of kidney cancer-related mortality worldwide. The heterogeneity within ccRCC poses a significant challenge to understanding its molecular mechanisms and devising effective therapeutic strategies. [1] Previous studies utilizing single nucleus RNA sequencing (snRNA-seq) have revealed transcriptional contributions to cell-type specificity in both mature human and mouse kidneys. Moreover, recent advancements have expanded this approach to single-cell profiling of chromatin accessibility. Single-nuclei assay for transposase-accessible chromatin using sequencing (snATAC-seq) has enabled the measurement of chromatin accessibility in thousands of individual cells, providing insights into the dynamic process that drives nephron development and differentiation. Integration and analysis of multimodal single-cell datasets, such as snRNA-seq and snATAC-seq, allows for the prediction of cell-type-specific cis-regulatory DNA interactions and transcription factor activity, complementing the transcriptional information obtained by snRNA-seq.[2] [3] [1] Yang, Wang, and Yang, “Treatment Strategies for Clear Cell Renal Cell Carcinoma.” [2] Muto et al., “Single Cell Transcriptional and Chromatin Accessibility Profiling Redefine Cellular Heterogeneity in the Adult Human Kidney.” [3] Monteagudo-Sánchez, Noordermeer, and Greenberg, “The Impact of DNA Methylation on CTCF-Mediated 3D Genome Organization.” Methods This study leveraged paired snRNA/ATAC-seq in tumor and normal-adjacent samples from the Renal Tumor Biobank at Dartmouth (n= 63) to explore the ccRCC landscape at single-cell resolution using 10x multiome technology. The snRNA-seq dataset was processed with Seurat v5.0 to remove low-quality nuclei. Doublets were removed with DoubletFinder v2.02, and ambient DNA was removed using decontX from celda. Normalization was conducted via SCTransform, and missing data was imputed with ALRA. Clustering was performed by constructing a KNN graph and running the Louvain algorithm. Cell type assignments were predicted using Azimuth with the human kidney reference dataset. The snATAC-seq data was processed with ArchR v1.0.1, which performed dimensionality reduction and clustering. Cluster identities defined in the Seurat workflow were cross-annotated to the ATAT-seq data, and open peaks were identified within these cell-type clusters using MACS3 v3.0.1. Identification of marker peaks, motif enrichment, and motif enrichment deviations were inferred utilizing ArchR’s getmarkerFeatures, peakAnnoEnrichment, and addMotifAnnotations functions, respectively. Results Motif enrichment analysis revealed canonical cancer-associated transcription factor (TF) dysregulation in tumor cells (nephron loop-like), including enrichment of HNF1B, SMARCC1, FOSL1, and JUND. Interestingly, HNF and SMARCC were also enriched in the lymphoid compartment, indicating possible crosstalk between these two cell types. Previous work has investigated the involvement of the HNF1 in ccRCC and in chromophobe RCC for prognosis,and we plan on investigating the relationship of these enrichments to tumor grade, stage and outcome.[1] Additional TF enrichments included BCL families and ELF within the myeloid compartment, and vascularization related TF families in endothelial cell types including enrichment of various ETS TFs. Investigation of cell-cell interactions between immune and endothelial cell populations as well as between tumor and normal adjacent tissues will be performed to further disentangle the role differentially accessable chromatin plays in ccRCC. [1] Buchner et al., “Downregulation of HNF-1B in Renal Cell Carcinoma Is Associated With Tumor Progression and Poor Prognosis.” Conclusions Our multi-omic approach has several advantages. First, it enables the characterization of cellular heterogeneity within the tumor microenvironment, identifying cell populations and their contributions to disease progression. Secondly, by integrating transcriptomic and epigenomic data, we gain insights into the dynamic interplay between gene expression and chromatin accessibility, uncovering potential therapeutic vulnerabilities and biomarkers for personalized treatment strategies. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e88d9a75048a69eb2afe5150d509757fff83b4" target='_blank'>
              60 Dissecting the Cellular and Epigenetic Landscape of Clear Cell Renal Cell Carcinoma Through Paired snRNA/ATAC-Seq Analysis
              </a>
            </td>
          <td>
            Ian Gingerich
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The AKT1 oncogene is related to various cancers due to its critical role in the PIC3CA/AKT1 pathway; however, most of the studies screened the hotspot mutation AKT1 (E17K) with various incidences. Low frequency or lack of AKT1 (E17K) mutation was reported in prostate cancer (PC) patients. This study aims to explore genetic alterations in the AKT1 PH domain by extending the sequencing to include AKT1 gene exons 3 and 4. Genomic DNA was extracted from 84 Formalin-Fixed Paraffin-Embedded samples of PC patients in Jordan, and then subjected to PCR and sequencing for the targeted exons. This study revealed the presence of two novel mutations (N53Y and Q59K) and a high frequency of mutations in exon 4, with a lack of mutations in the E17K hotspot. Nine missense and two synonymous mutations were detected in exon 4 (Phe27Tyr, Phe27Leu, Ala58Thr, Ser56Phe, Arg41Trp, Phe35Leu, Asp32Glu, Phe35Tyr, and Gln43Lys) and (Ser56 and Glu40), respectively. Two synonymous mutations were detected in exon 3 (Leu12 and Ser2). It is concluded that there is a high frequency of AKT1 mutation in PC patients in Jordan with two novel missense mutations in the Pleckstrin homology (PH) domain. E17K hotspot mutation was not detected in any tested samples, which underlined the significant role of mutations in other AKT1 exons in PC development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/205851db262e85bd8c99969aa7adb86b40d5b2f6" target='_blank'>
              Novel Mutations in AKT1 Gene in Prostate Cancer Patients in Jordan
              </a>
            </td>
          <td>
            A. Alasmar, Zina M. Al-Alami, S. Zein, Asmaa Al-Smadi, Samir Al Bashir, Mohammed S. Alorjani, Raed M. Al‐Zoubi, M. A. Al Zoubi
          </td>
          <td>2024-09-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24eccfe89ea8e14aa08f4e83d855de3df06a6b0" target='_blank'>
              EWS-WT1 fusion isoforms establish oncogenic programs and therapeutic vulnerabilities in desmoplastic small round cell tumors
              </a>
            </td>
          <td>
            Gaylor Boulay, Liliane C. Broye, Rui Dong, Sowmya Iyer, Rajendran Sanalkumar, Yu-Hang Xing, Rémi Buisson, Shruthi Rengarajan, Beverly Naigles, Benoît Duc, A. Volorio, Mary E. Awad, R. Renella, I. Chebib, G. P. Nielsen, Edwin Choy, Gregory M Cote, Lee Zou, Igor Letovanec, Ivan Stamenkovic, Miguel N. Rivera, Nicolò Riggi
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbea514dd68ac66e3d2deacc382cefe854e64feb" target='_blank'>
              Mechanism insights and therapeutic intervention of tumor metastasis: latest developments and perspectives
              </a>
            </td>
          <td>
            Xiaoli Shi, Xinyi Wang, Wentao Yao, Dongmin Shi, Xihuan Shao, Zhengqing Lu, Yue Chai, Jinhua Song, Weiwei Tang, Xuehao Wang
          </td>
          <td>2024-08-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) can manifest as de novo AML (dn-AML) or secondary AML (s-AML), with s-AML being associated with inferior survival and distinct genomic characteristics. The underlying reasons for this disparity remain to be elucidated. In this multicenter study, next-generation sequencing (NGS) was employed to investigate the mutational landscape of AML in 721 patients from June 2020 to May 2023.Genetic mutations were observed in 93.34% of the individuals, with complex variations (more than three gene mutations) present in 63.10% of them. TET2, ASXL1, DNMT3A, TP53 and SRSF2 mutations showed a higher prevalence among older individuals, whereas WT1 and KIT mutations were more commonly observed in younger patients. BCOR, BCORL1, ZRSR2, ASXL1 and SRSF2 exhibited higher mutation frequencies in males. Additionally, ASXL1, NRAS, PPMID, SRSF2, TP53 and U2AF1 mutations were more common in patients with s-AML, which PPM1D was more frequently associated with therapy-related AML (t-AML). Advanced age and hyperleukocytosis independently served as adverse prognostic factors for both types of AML; however, s-AML patients demonstrated a greater number of monogenic adverse prognostic factors compared to dn-AML cases (ASXL1, PPM1D, TP53 and U2AF1 in s-AML vs. FLT3, TP53 and U2AF1 in dn-AML). Age and sex-related gene mutations suggest epigenetic changes may be key in AML pathogenesis. The worse prognosis of s-AML compared to dn-AML could be due to the older age of s-AML patients and more poor-prognosis gene mutations. These findings could improve AML diagnosis and treatment by identifying potential therapeutic targets and risk stratification biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f06fb6d75db866688d32fbb738b2b52aacbc718" target='_blank'>
              Genomic mutation patterns and prognostic value in de novo and secondary acute myeloid leukemia: A multicenter study from China.
              </a>
            </td>
          <td>
            Xi Dou, Chunli Dan, Duanzhong Zhang, Hong-Sheng Zhou, Renke He, Guangyu Zhou, Yu Zhu, Nan Fu, Ben Niu, Shuangnian Xu, Yi Liao, Zhangqin Luo, Lihua Yang, Haiguo Zhang, Yizhi Xu, Q. Zhan, Wei Chen, Zesong Yang, Xiaoqiong Tang, Hong-bin Zhang, Qing Xiao, Jianbin Chen, Lin Liu, Yi Wang, Li Pei, Li Wang
          </td>
          <td>2024-08-07</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 In the United States and globally, prostate cancer (PCa) mortality rates are the highest among men of African descent. 24% of the disparities in PCa remains even when controlled for access to care and stage at presentation. Moreover, the tumor microenvironment plays an essential role in tumor progression, aggressiveness, therapeutic response, and patient outcomes. Additionally, the association of the genomic findings with patient ancestry and other characteristics, such as tumor biology and transcriptomic alterations, remains poorly understood. Here, we performed a multi-Omics approach (N=447) to unravel the complexity of tumor heterogeneity and understand disease progression & distinct tumor biology influenced by genetic ancestry. We performed Whole Exome (Normal/tumor paired) Sequencing matched with Methyl Seq and Whole transcriptomic Sequencing for three datasets of our African, African American Men (AAM), and European Men (EAM). Additionally, we ran Spatial NanoString high-plex GeoMx-DSP for a total of 118 treatment-naive PCa patients (around 500 ROIs). Simultaneously, we added matched whole transcriptome Sequencing for these patients. The cohort comprised 87 AAM, 3 unknown, and 28 EAM self-reported individuals. To verify the self-reported race, the genomic ancestry was qualified using genotype and Admixture analysis. To further validate differentially expressed genes at the protein level, we performed multi-plex histological staining of 40 markers to determine the spatial resolution and neighborhood clustering within the tumor and the microenvironment. In parallel, we performed scMultiOmics sequencing (scRNA & scATAC-Seq) at a single-cell resolution (6000 cells/sample at 25K reads per cell) from an additional 12 patients (9 AAM & 3 EAM). We reconstructed a 3D model of the distinct tumor microenvironment at subcellular resolution using serially sectioned hematoxylin and eosin- stained tissue sections (CODA technology). Our results demonstrate that patients who self-report as AAM or Nigerian are assigned to high African (> 70%) Ancestry with either Yoruba (Nigeria) and/or Bantu subpopulation in the Sub-Saharan area. Additionally, high African Ancestry patients are diagnosed at a younger age and show advanced pathology stages compared to patients with European Ancestry. AAM of Yoruba descent expresses significantly higher immune-inflammatory signatures (STAT/IFNG-signaling pathway) compared to the Nigerian-Yoruba subpopulation. Our scRNA-Seq analysis shows that AAM has suppressive myeloid cells infiltrate within the tumor cells. The infiltrations of these cells change with age, Gleason Grade, and pathology stage. Moreover, AAM-tumor cells (scATAC-Seq) have increased open Chromatin accessibility with STAT motifs enrichment (p-value;0.0001), which could be the driver regulators of the immune-suppressive signature as well as influence the upregulation of STAT/IFNG signature within African Ancestry patients. Our study provides new insight into how genetic ancestry impacts immune signatures in AAM/African and contributes to PCa racial disparities.
 Citation Format: Isra Elhussin, Ezra Baraban1, Ashley Kiemen, Tamara L Lotan, Cathy Handy Marshall, Emmanuel Antonarakis, Moray J Campbell, Melissa Davis, Michael Dixon, Isaac Kim, Stefan Ambs, Rick Kittles, Adam B Murphy, Clayton Yates. Integrative multi-omics profiling in patients of African descent diagnosed with prostate cancer reveals distinct tumor-promoting immunosuppressive niches at a single-cell level and spatial resolution [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C117.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4100a5d71a839b36fc483aaee40ec75da9220882" target='_blank'>
              Abstract C117: Integrative multi-omics profiling in patients of African descent diagnosed with prostate cancer reveals distinct tumor-promoting immunosuppressive niches at a single-cell level and spatial resolution
              </a>
            </td>
          <td>
            Isra Elhussin, Ezra G. Baraban, Ashley Kiemen, T. Lotan, Cathy Handy Marshall, E. Antonarakis, Moray J. Campbell, Melissa B. Davis, Michael Dixon, Isaac Kim, S. Ambs, Rick A. Kittles, Adam B. Murphy, Clayton C. Yates
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Lung cancer is the leading cause of cancer mortality worldwide. KRAS oncogenes are responsible for at least a quarter of lung adenocarcinomas, the main subtype of lung cancer. After four decades of intense research, selective inhibitors of KRAS oncoproteins are finally reaching the clinic. Yet, their effect on overall survival is limited due to the rapid appearance of drug resistance, a likely consequence of the high intratumoral heterogeneity characteristic of these tumors. In this study, we have attempted to identify those functional alterations that result from KRAS oncoprotein expression during the earliest stages of tumor development. Such functional changes are likely to be maintained during the entire process of tumor progression regardless of additional co-occurring mutations. Single-cell RNA sequencing analysis of murine alveolar type 2 cells expressing a resident Kras oncogene revealed impairment of the type I interferon pathway, a feature maintained throughout tumor progression. This alteration was also present in advanced murine and human tumors harboring additional mutations in the p53 or LKB1 tumor suppressors. Restoration of type I interferon (IFN) signaling by IFN-β or constitutive active stimulator of interferon genes (STING) expression had a profound influence on the tumor microenvironment, switching them from immunologically "cold" to immunologically "hot" tumors. Therefore, enhancement of the type I IFN pathway predisposes KRAS mutant lung tumors to immunotherapy treatments, regardless of co-occurring mutations in p53 or LKB1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe4ab14ca9fcef735ebd7c09bd8738b90b51f2b7" target='_blank'>
              Type I interferon signaling pathway enhances immune-checkpoint inhibition in KRAS mutant lung tumors.
              </a>
            </td>
          <td>
            Fernando Fernández-García, Ana Fernández-Rodríguez, C. Fustero-Torre, Elena Piñeiro-Yáñez, Haiyun Wang, C. Lechuga, S. Callejas, Rebeca Álvarez, Alejandra López-García, Laura Esteban-Burgos, Marina Salmón, Marta San Román, Carmen Guerra, Chiara Ambrogio, M. Drosten, David Santamaría, Fátima Al-Shahrour, Ana Dopazo, Mariano Barbacid, M. Musteanu
          </td>
          <td>2024-08-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Cellular crosstalk mediated by ligand–receptor interactions largely complicates the tumour ecosystem, resulting in heterogeneous tumour microenvironments that affect immune response and clinical benefits from immunotherapy. Epigenetic mechanisms are pivotal to expression changes of immune‐related genes and can modulate the anti‐tumour immune response. However, the functional consequences of disrupted epigenetic regulators (ERs) on ligand–receptor interactions in the tumour microenvironment remain largely unexplored. Here, we proposed mutations of ERs in perturbed interactions (MERIN), a molecular network‐based approach that incorporates multi‐omics data, to infer the potential consequences of ER mutations on ligand–receptor interaction perturbations. Leveraging cancer genomic profiles and molecular interaction data, we comprehensively decoded the functional consequences of ER mutations on dysregulated ligand–receptor interactions across 33 cancers. The dysregulated ligand–receptor genes were indeed enriched in cancer and immune‐related function. We demonstrated the potential significance of PD1–PDL1 interaction‐related ER mutations in stratifying cancer patients from multiple independent data cohorts. The ER mutation group showed distinct immunological characterizations and prognoses. Furthermore, we highlighted that the ER mutations could potentially predict clinical outcomes of immunotherapy. Our computational and clinical assessment underscore the utility of MERIN for elucidating the functional relevance of ER mutations in cancer immune response, potentially aiding patients' stratification for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1c94edf4edb1035d9676cb0af0e88e572eaf00" target='_blank'>
              Decoding functional impact of epigenetic regulator mutations on ligand–receptor interaction perturbations for evaluation of cancer immunotherapy
              </a>
            </td>
          <td>
            Aiai Shi, Chaohuan Lin, Jie Lyu
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Background: Prostate cancer (PCa) poses unique challenges for Non-Hispanic Black Americans (BA), who often face poorer outcomes than Non-Hispanic White Americans (WA). Even after adjusting for socioeconomic factors, BAs are diagnosed younger and experience more severe clinical effects, potentially due to the tumor microenvironment. Within PCa, Endogenous Retroviruses (ERVs) have been shown to regulate immune response genes and have prognostic value in PCa. Despite constituting 8% of the human, ERVs' role in the tumor microenvironment remains incompletely understood. These findings highlight the need to target microenvironment dysregulation in solid tumor therapies, especially for ERV expression in prostate cancer. Methods: Using RNA-sequencing, we investigated the expression of ERVs in BA (n=18) and WA (n=14) tumor adjacent stroma (TAS) samples. We analyzed reads by mapping them to the human genome (hg38). We then used bedtools and data from the Human Endogenous Retrovirus database (HERVd) to calculate the number of reads at each ERV locus (n=519,060). Differentially expressed ERV elements were then identified in an Audic Claverie Test (fold- change cutoff of 2 and padj < 0.05). The expression of protein-coding genes in the same BA (n=18) and WA (n=14) TAS samples was investigated using RNA-sequencing. Regulation of significant ERV elements were compared to the regulation of nearby immune response genes between the two races. We developed primary stroma (carcinoma associated fibroblast, CAFs) culture of different races with PCa using fresh prostatectomy tissues. To investigate the biological functions of ERVs in CAFs, we developed FANA oligonucleotides (synthetic single- stranded nucleic acid analogs that can modulate gene expression by enzymatic degradation of a target RNA) and corresponding primers to knock down specific ERV sequences near key immune response genes in prostate cancer. Results: We examined 32 prostatectomy specimens of BA (n=18) and WA (n=14) PCa. Of the 5,786 statistically significant differentially expressed ERV elements between BA and WA TAS that were identified, 3,274 elements had an increase in ERV expression for BA TAS, and 2,512 elements had a decrease in ERV expression for BA TAS. 61 MER41 elements were found to be differentially expressed between the two races. The MER family sequences contain binding sites for transcription factors that are crucial in cancer, such as STAT1. This transcription factor can regulate genes involved in cell proliferation, apoptosis, and DNA repair. MER41 is in close proximity (within 5000 base pairs) to IFNA6, an immune response gene. We found both MER41 and IFNA6 to be downregulated in BA TAS, as compared to their WA counterparts. We developed FANA oligonucleotides to knockdown MER41E, MER61-Int, HERVL-Int, MLT1A, and LTR61E1 to investigate their effect on nearby gene expression. Conclusion: The CAF models from PCa patients of different races can be used to explore the biological function of ERV inactivation on anti-tumor immune response genes within the tumor microenvironment.
 Citation Format: Tara S. K. Jennings, Vinay Kumar, Anton N. Nguyen, Michael M. Ittmann, Patricia Castro, Farah Rahmatpanah. Transcriptome analysis of human endogenous retroviral elements in tumor microenvironment of prostate cancer patients of different ancestry [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C092.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba34051bd7dedd2fa8c67fcaee44595e97f04e7" target='_blank'>
              Abstract C092: Transcriptome analysis of human endogenous retroviral elements in tumor microenvironment of prostate cancer patients of different ancestry
              </a>
            </td>
          <td>
            Tara S. K. Jennings, Vinay Kumar, Anton N. Nguyen, Michael M. Ittmann, Patricia Castro, Farah Rahmatpanah
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Topologically associated domains (TADs) are believed to be involved in the regulation of gene expression. While the impact of TAD perturbations is usually studied in developmental genes with highly cell-type-specific expression patterns, this study examines genes with broad expression profiles divided by a strong insulatory boundary. We focused on the Slc29a3/Unc5b locus, which encompasses two distinct TADs. Our analysis demonstrates that deletions of CTCF binding sites within this locus lead to alterations in local chromatin architecture, disrupting existing loops and forming novel long-range interactions. We evaluated the transcription changes of Unc5b, Slc29a3, Psap, Vsir, Cdh23, and Sgpl1 genes across various organs, finding that TAD boundary disruption results in variable transcriptional responses, where not only magnitude, but also direction of gene expression changes are tissue-specific. Current models of genome architecture, including enhancer competition and hijacking, only partially account for these transcriptional changes, indicating the need for further investigation into the mechanisms underlying TAD function and gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c912d81a6af96e3ee3f74faa2fc1bde4c441880" target='_blank'>
              Direction and modality of transcription changes caused by TAD boundary disruption in Slc29a3/Unc5b locus depends on tissue-specific epigenetic context
              </a>
            </td>
          <td>
            Paul Salnikov, P. Belokopytova, Alexandra Yan, E. Viesná, Alexey Korablev, Irina Serova, V. Lukyanchikova, Yana Stepanchuk, N. Torgunakov, Savelii Tikhomirov, V. Fishman
          </td>
          <td>2024-08-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Genomic medicine has transformed the lives of patients with cancer by enabling individualised and evidence-based clinical decision-making. Despite this progress, the implementation of precision cancer medicine is limited by its dependence on isolated biomarkers. The development of bulk and single-cell multiomic technologies has revealed the enormous complexity of the cancer ecosystem. Beyond the cancer cell, the tumour microenvironment, macroenvironment and host factors, including the microbiome, profoundly influence the cancer phenotype, and accounting for these enhances the resolution of precision medicine. The advent of robust multiomic profiling and interpretable machine learning algorithms mark the dawn of a new postgenomic era of personalised cancer medicine. In Precision Cancer Medicine 2.0, high-resolution personalised clinical decision-making is informed by the comprehensive multiomic profiling of tumour and host, integrated using artificial intelligence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfe2911ddad4c6278f32e1d4a4df6b899200b9a9" target='_blank'>
              Precision Cancer Medicine 2.0-Oncology in the postgenomic era.
              </a>
            </td>
          <td>
            Riccardo Masina, Carlos Caldas
          </td>
          <td>2024-08-07</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec914b4189eafe0cce08165c6a7c2a53bfc159e" target='_blank'>
              Unveiling FRG1’s DNA repair role in breast cancer
              </a>
            </td>
          <td>
            Shubhanjali Shubhanjali, Talina Mohapatra, Rehan Khan, Manjusha Dixit
          </td>
          <td>2024-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Background: For effective screening of breast cancer patients for candidates for target therapy with alpelisib, it is necessary to identify activating somatic mutations in the PIK3CA gene by allele specific polymerase chain reaction (PCR); this requires that an optimal list of mutations should be compiled. 
Aim: To determine the spectrum of somatic mutations in the PIK3CA gene in breast cancer tumors by means of high performance sequencing (next generation sequencing, NGS) and to identify their minimal set for clinical diagnostic testing by allele specific PCR. 
Methods: Targeted NGS was used to identify mutations in the PIK3CA gene in DNA obtained from paraffin blocks with tumor material from 431 patients with HR+HER2- breast cancer. A set of the most common somatic mutations was also detected by allele specific PCR. 
Results: We have developed a set of reagents and a protocol for targeted NGS of frequently mutating regions of the PIK3CA and ESR1 genes, which was used to analyze samples from 451 HR+/HER2- breast cancer patients. Clinically significant activating mutations in the PIK3CA gene were found in 32.7% of the samples (141/431). The frequency of the mutant allele ranged from 0.15 to 0.65. Six mutations were most common: c.3140AG p.His1047Arg (69), c.1633GA p.Glu545Lys (32), c.1035TA p.Asn345Lys (12), c.1624GA p.Glu542Lys (9), c.3140AT p.His1047Leu (8), Cys420Arg c.1258TC (3). In total, these mutations amounted to 94.3% (133/141). In 3.5% of the samples (15/431), there were clinically significant somatic mutations in the ESR1 gene: c.1613AG p, Asp538Gly (7), c.1610AC p.Tyr537Ser (6), c.1609TA p.Tyr537Asn (1), c.1610AG p.Tyr537Cys (1), causing resistance to hormone therapy in patients with breast cancer. While rare mutations comprised only 5.7% of our sample, we validated a set of reagents to identify the six mutations described above by allele specific PCR. NGS and PCR were completely concordant. 
Conclusion: PCR testing of activating somatic mutations of the PIK3CA gene meets the requirements for sensitivity ( 90%) and specificity (100%) for a clinical test and can be used in the selection of patients for targeted therapy with PIK3CA inhibitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb6ff9ea9759ebdc593e75158f675bc13151f83f" target='_blank'>
              Identification of activating somatic mutations in the PIK3CA gene in breast tumors and determination of their minimal set for clinical diagnostic testing
              </a>
            </td>
          <td>
            U. A. Boyarskih, A. Kechin, Alyona V. Zyuzyukina, Yevgeny A. Khrapov, I. Oscorbin, Yefim Y. Alexeenok, Galina A. Avdiyuk, R. Zukov, N. E. Kushlinskii, M. Filipenko
          </td>
          <td>2024-09-10</td>
          <td>Almanac of Clinical Medicine</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Genes on sex chromosomes have higher evolutionary rates than those on autosomes. However, this does not necessarily apply to somatic evolution in cancer. Many dominant mutations have been described in the so-called proto-oncogenes (OGs), while recessive mutations are typically described in tumor-suppressor genes (TSGs). Evidence indicates that mutations in X-chromosome TSGs are more likely to contribute to cancer than those in autosomal TSGs. Here, we formalize this in several dynamic models and predict, as expected, that mutations spread faster in TSGs located on the X chromosome than on autosomes (faster-X effect). Conversely, mutations in OGs spread faster on autosomes than on the X chromosome, but under high selective pressure, this difference is negligible. Published genomic screenings of cancer samples show evidence of the faster-X effect in TSGs. This pattern is observed in both sexes, suggesting that the maintenance of X-chromosome inactivation during cancer progression plays an important role in the evolution of TSGs. Strikingly, the relative mutation incidence in X-linked TSGs among females across individual studies is bimodal, with one group of studies showing a faster-X effect and another group showing similar incidences for X-linked and autosomal TSGs. This differentiation between cancer samples is not associated with the specific type of cancer or the tissue of origin. This may indicate that X-chromosome inactivation plays a differential role in the involvement of X-linked TSGs across individual cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e60393ec97b0f4b9cf086f3cab7030b54b21b01" target='_blank'>
              Somatic evolution of cancer genes in sex chromosomes
              </a>
            </td>
          <td>
            Amarachi Akachukwu, Jasmine S. Ratcliff, Antonio Marco
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4af76c1aab1fec450dc5bad42237d6e428e7def0" target='_blank'>
              Applying whole-genome and whole-exome sequencing in breast cancer: a review of the landscape.
              </a>
            </td>
          <td>
            Hetvi Ganatra, J. K. Tan, Ana Simmons, Carola Maria Bigogno, Vatsala Khurana, Aruni Ghose, Adheesh Ghosh, Ishika Mahajan, S. Boussios, Akash Maniam, Olubukola Ayodele
          </td>
          <td>2024-08-27</td>
          <td>Breast cancer</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="High-throughput transcriptomics (HTTr) is increasingly being used to identify molecular targets of chemicals that can be linked to adverse outcomes. Cell proliferation is an important key event in chemical carcinogenesis. Here, we describe the construction and characterization of a gene expression biomarker that is predictive of the cell proliferation (CP) status in human and rodent tissues. The biomarker was constructed from 30 genes known to be increased in expression in prostate cancers relative to surrounding tissues and in cycling human MCF-7 cells after estrogen receptor (ER) agonist exposure. Using a large compendium of gene expression profiles to test utility, the biomarker could identify increases in cell proliferation in 1) 367 tumor vs. normal surrounding tissue comparisons from 6 human organs, 2) MCF-7 cells after activation of ER, 3) after partial hepatectomy in mice and rats, and 4) the livers of mice after exposure to nongenotoxic hepatocarcinogens. The biomarker identified suppression of CP 1) under conditions of p53 activation by DNA damaging agents in human cells, 2) in human A549 lung cells exposed to therapeutic anticancer kinase inhibitors (dasatinib, nilotnib) and 3) in the mouse liver when comparing high levels of CP at birth to the low background levels in the adult. The responses using the biomarker were similar to those observed using conventional markers of CP including PCNA, Ki67, and BrdU labeling. The CP biomarker will be a useful tool for interpretation of HTTr data streams to identify cell proliferation status after exposure to chemicals in human cells or in rodent tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e67eda2694923151226daf541a133cfb12edda84" target='_blank'>
              A transcriptomic biomarker predictive of cell proliferation for use in adverse outcome pathway-informed testing and assessment.
              </a>
            </td>
          <td>
            J. Corton, Victoria Ledbetter, Samuel M. Cohen, E. Atlas, Carole L Yauk, Jie Liu
          </td>
          <td>2024-08-13</td>
          <td>Toxicological sciences : an official journal of the Society of Toxicology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="
 Paediatric T-cell acute lymphoblastic leukaemia (T-ALL) is a T cell malignancy for which there currently is no feature that reliably risk-stratifies patients at the point of diagnosis. The earliest reliable indicator of high-risk T-ALL is the presence of residual disease (≥ 5% cancer cells) after four weeks of induction chemotherapy. Various diagnostic biomarkers have been proposed to identify children with a poor prognosis, however, none have been sufficiently robust to enter clinical practice. We used single-cell RNA-sequencing to profile bone marrow aspirates both at diagnosis and after induction failure (at day 28) for 13 refractory patients, as well as 8 children who were responsive to chemotherapy. This high-resolution phenotyping allowed us to identify a ZBTB16+ cancer cell (blast) population that is present only in the diagnostic samples of induction failure patients and dominates their day 28 bone marrows. Comparison with normal immune cells revealed a strong similarity to unconventional innate-like lymphoid cells, indicating ZBTB16+ blasts represent a non-canonical, previously unrecognised, lymphoblast. To examine the clinical significance of ZBTB16+ blasts in large and publicly available bulk RNA-seq cohorts, we derived a 29-gene module that represents the ZBTB16+ non-canonical cell state. Using independent bulk RNA-seq cohorts for validation, we show that our gene module segregates patients who experience induction failure, independent of ETP status. Importantly, we show that ZBTB16+ blasts can be detected at the point of diagnosis and can be measured by intracellular flow cytometry, which can easily be incorporated into diagnostic panels. For one particular child, our index case, ZBTB16+ blasts were a minority population at diagnosis but dramatically expanded during induction treatment. We performed whole genome sequencing on the diagnostic, post-induction, and remission bone marrow of this child to investigate the phylogenetic relation of treatment-resistant ZBTB16+ blasts and sensitive ZBTB16- blasts that disappeared during induction. Using single nucleotide variants and copy number changes, we determined that ZBTB16+ blasts did not derive from diagnostic (treatment sensitive ZBTB16-) blasts. Our findings suggest that treatment-resistant blasts represent a distinct cancer cell state, rather than a clone that derives from the initial diagnostic population, which has the potential to transform risk stratification in T-ALL and offers a novel therapuetic target.
 Citation Format: Holly J. Whitfield, Bram Lim, Mi Trinh, Angus Hodder, Petri Polonen, David Teachy, Charles Mullighan, Melanie M. Hagleitner, Lennart Kester, Frank van Leeuwen, Jack Bartram, Owen Williams, Nathaniel D. Anderson, Laura Jardine, Sam Behjati, David O'Connor. Single cell transcriptomics reveals a non-canonical lymphoblast in refractory childhood T cell leukaemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A025.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c561ee124832d325d8b303a4dd3c549dc55a3bd0" target='_blank'>
              Abstract A025 Single cell transcriptomics reveals a non-canonical lymphoblast in refractory childhood T cell leukaemia
              </a>
            </td>
          <td>
            Holly J. Whitfield, Bram Lim, Mi K. Trinh, A. Hodder, Petri Polonen, David Teachy, C. Mullighan, Melanie M. Hagleitner, Lennart A Kester, Frank van Leeuwen, Jack Bartram, Owen Williams, Nathaniel D. Anderson, Laura Jardine, Sam Behjati, David O'Connor
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e1516a70ceb6cf40f46931b69cd383f8042a08" target='_blank'>
              PTBP1-mediated repression of neuron-specific CDC42 splicing constitutes a genomic alteration-independent, developmentally conserved vulnerability in IDH-wildtype glioblastoma.
              </a>
            </td>
          <td>
            Junjie Yang, Jing Feng, Jing Lv, Xiaojing Chu, Yanfei Wei, Yunqiu Zhang, Jiuyi Li, Yingyu Sun, Guanzhang Li, Tao Jiang, Jinyan Huang, Xiaolong Fan
          </td>
          <td>2024-08-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecd90e67bdef5377062f3b39198a321851836aa3" target='_blank'>
              Ultra-sensitive molecular residual disease detection through whole genome sequencing with single-read error correction
              </a>
            </td>
          <td>
            Xinxing Li, Tao Liu, A. Bacchiocchi, Mengxing Li, W. Cheng, T. Wittkop, F. Mendez, Yingyu Wang, P. Tang, Q. Yao, M. W. Bosenberg, M. Sznol, Q. Yan, M. Faham, L. Weng, R. Halaban, H. Jin, Zhiqian Hu
          </td>
          <td>2024-08-20</td>
          <td>EMBO Molecular Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a2685872da8abf8d89c45e2e00bd648c1daaeb" target='_blank'>
              DNA methylation and gene expression as determinants of genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Michaël Noë, Dimitrios Mathios, A. Annapragada, Shashikant Koul, Zachariah H. Foda, J. Medina, S. Cristiano, Christopher Cherry, D. C. Bruhm, N. Niknafs, V. Adleff, Leonardo Ferreira, Hari Easwaran, Stephen Baylin, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="The mammalian mitochondrial genome (mtDNA) is multicopy and its copy number (mtCN) varies widely across tissues, in development and in disease. Here, we systematically catalog this variation by assaying mtCN in 52 human tissues across 952 donors (10,499 samples from the Genotype-Tissue Expression project) and 20 murine tissues using qPCR, capturing 50- and 200-fold variation, respectively. We also estimate per cell mtCN across 173 human cell lines from the Cancer Cell Line Encyclopedia using whole-genome sequencing data and observe >50-fold variation. We then leverage the vast amount of genomics data available for these repositories to credential our resource and uncover mtDNA-related biology. Using already existing proteomics data, we show that variation in mtCN can be predicted by variation in TFAM, histone, and mitochondrial ribosome protein abundance. We also integrate mtCN estimates with the CRISPR gene dependency measurements to find that cell lines with high mtCN are resistant to loss of GPX4, a glutathione phospholipid hydroperoxidase. Our resource captures variation in mtCN across mammalian tissues and should be broadly useful to the research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407c1ab023792825943a400486c262ae1672c036" target='_blank'>
              Mitochondrial genome copy number variation across tissues in mice and humans
              </a>
            </td>
          <td>
            S. Rath, Rahul Gupta, E. Todres, Hong Wang, Alexis A. Jourdain, K. Ardlie, Sarah E. Calvo, V. Mootha
          </td>
          <td>2024-08-06</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3ed677c11e6abc17ba45e11a7ae14f8ec6c668d" target='_blank'>
              Targeting MDM2-mediated suppression of p53 with idasanutlin: a promising therapeutic approach for acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            Seyda Gungordu, E. Aptullahoglu
          </td>
          <td>2024-09-21</td>
          <td>Investigational new drugs</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Enhancer elements are specific DNA sequences that play a crucial role in regulating gene expression. Located upstream or downstream in the genomic context, enhancers enhance the transcription of linked genes. This is achieved by providing binding sites for transcription factors and other proteins, acting as a nucleation point that promotes the assembly of the transcriptional machinery.
What makes enhancers unique is that located even thousands of base pairs away, they can exert their function. They can act over long distances, looping to interact with the promoter region of their target genes. Enhancers are often involved in cell type-specific gene expression as different cells express different sets of genes by activating and repressing cell type-specific enhancers.
Cancer cells seem to co-opt this mechanism and hijack it for their own survival. In this process, an aberrant enhancer element activates the transcription of oncogenes (genes that have the potential to cause cancer) due to some alterations in genomic structure or regulatory elements that happen in cancer, contributing to tumorigenesis and cancer progression. Enhancer hijacking often occurs through complex genomic rearrangements such as chromosomal translocations and other structural rearrangements such as deletions or amplifications.8 These alterations can bring a usually distant, far-located enhancer into proximity with an oncogene, leading to its inappropriate activation. This is often seen in various cancers, including leukemias and solid tumors.
Enhancer hijacking and its effects on gene regulation in cancer are frequently studied using methods such as RNA sequencing, chromatin conformation capture, and CRISPR-Cas9. Finding an enhancer hijacking event may be used to inform treatment choices, especially in precision medicine settings, and may also function as a biomarker for particular cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefe88575de8420490e0e1dd10a24dbefe2eef41" target='_blank'>
              Enhancer hijacking: Innovative ways of carcinogenesis
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-09-02</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823edc7b7f2e48466de67d70b29a3e25297b1b88" target='_blank'>
              Cooperative genomic lesions in HRAS-mutant cancers predict resistance to farnesyltransferase inhibitors.
              </a>
            </td>
          <td>
            Arad Nigam, Gnana P. Krishnamoorthy, Walid K. Chatila, Katherine Berman, Mahesh Saqcena, H. Walch, Mandakini Venkatramani, Alan L. Ho, N. Schultz, James A. Fagin, B. Untch
          </td>
          <td>2024-08-16</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1f5936217eac5b4043bb1665424b2339a9f20d" target='_blank'>
              MYC and KRAS cooperation: from historical challenges to therapeutic opportunities in cancer
              </a>
            </td>
          <td>
            S. Casacuberta-Serra, Íñigo González-Larreategui, Daniel Capitán-Leo, Laura Soucek
          </td>
          <td>2024-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Minichromosome maintenance complex component 3 associated protein (MCM3AP) is a gene in which mutations can result in autosomal recessive peripheral neuropathy with or without impaired intellectual development. The MCM3AP genotype-phenotype correlation and prognosis remain unclear. The aim of this study was to explore the genotype–phenotype correlations pertaining to MCM3AP. Methods Whole-exome sequencing (WES) combined with copy number variation sequencing (CNV-seq) were performed on the genomic DNA isolated from a Chinese family, and Sanger sequencing, quantitative PCR and cDNA analyses were performed to examine the mutations. The retrospective study was conducted on 28 individuals with biallelic MCM3AP mutation-related diseases, including features such as mutations, motor development impairment, intellectual disability, weakness/atrophy, and cerebral magnetic resonance imaging abnormalities. Results Sequencing identified novel compound heterozygous mutations in MCM3AP, namely, a paternal variant c.1_5426del (loss of exons 1–25) and a maternal splicing variant c.1858 + 3A>G. Functional studies revealed that the variant c.1858 + 3A>G resulted in the heterozygous deletion of exon 5, thereby affecting splicing functionality. Furthermore, the compound heterozygous mutation may affect the functionality of the protein domain. Retrospective analysis revealed different genotype–phenotype correlations for the pathogenic variants in biallelic MCM3AP: all individuals (100%) with mutations outside the Sac3 domain exhibited early-onset symptoms, motor developmental delays, and cognitive abnormalities, conversely, the proportions of individuals carrying mutations within the domain were 26.7% (motor delays) and 46.7% (cognitive abnormalities). Conclusion Our findings further expand the genetic mutation spectrum of biallelic MCM3AP and highlight the genotype-phenotype associations. Additionally, we elaborate on the importance of rehabilitation intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc14d90ab7ba7ceb9ba0f635353a560c5f6b553f" target='_blank'>
              Identification of biallelic mutations in MCM3AP and comprehensive literature analysis
              </a>
            </td>
          <td>
            Chan Liu, Qingfeng Xie, Quan Hu, Bingwu Xiang, Kaiyi Zhao, Xiang Chen, Feixia Zheng
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>