<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-23 06:32:00 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>7</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Drug resistance is the major cause of therapeutic failure in high-grade serous ovarian cancer (HGSOC). Yet, the mechanisms by which tumors evolve to drug resistant states remains largely unknown. To address this, we aimed to exploit clone-specific genomic structural variations by combining scaled single-cell whole genome sequencing with longitudinally collected cell-free DNA (cfDNA), enabling clonal tracking before, during and after treatment. We developed a cfDNA hybrid capture, deep sequencing approach based on leveraging clone-specific structural variants as endogenous barcodes, with orders of magnitude lower error rates than single nucleotide variants in ctDNA (circulating tumor DNA) detection, demonstrated on 19 patients at baseline. We then applied this to monitor and model clonal evolution over several years in ten HGSOC patients treated with systemic therapy from diagnosis through recurrence. We found drug resistance to be polyclonal in most cases, but frequently dominated by a single high-fitness and expanding clone, reducing clonal diversity in the relapsed disease state in most patients. Drug-resistant clones frequently displayed notable genomic features, including high-level amplifications of oncogenes such as CCNE1, RAB25, NOTCH3, and ERBB2. Using a population genetics Wright-Fisher model, we found evolutionary trajectories of these features were consistent with drug-induced positive selection. In select cases, these alterations impacted selection of secondary lines of therapy with positive patient outcomes. For cases with matched single-cell RNA sequencing data, pre-existing and genomically encoded phenotypic states such as upregulation of EMT and VEGF were linked to drug resistance. Together, our findings indicate that drug resistant states in HGSOC pre-exist at diagnosis and lead to dramatic clonal expansions that alter clonal composition at the time of relapse. We suggest that combining tumor single cell sequencing with cfDNA enables clonal tracking in patients and harbors potential for evolution-informed adaptive treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1441270a54104f75e7713e2bb6b7323866959" target='_blank'>
              Tracking clonal evolution of drug resistance in ovarian cancer patients by exploiting structural variants in cfDNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Gryte Satas, Hannah J. Lees, Jake June-Koo Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, N. Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitriy Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Abstract Phenotypic plasticity is a recognized mechanism driving therapeutic resistance in patients with prostate cancer. Although underlying molecular causations driving phenotypic plasticity have been identified, therapeutic success is yet to be achieved. To identify putative master regulator transcription factors (MR-TF) driving phenotypic plasticity in prostate cancer, this work utilized a multiomic approach using genetically engineered mouse models of prostate cancer combined with patient data to identify MYB proto-oncogene like 2 (MYBL2) as a significantly enriched transcription factor in prostate cancer exhibiting phenotypic plasticity. Genetic inhibition of Mybl2 using independent murine prostate cancer cell lines representing phenotypic plasticity demonstrated Mybl2 loss significantly decreased in vivo growth as well as cell fitness and repressed gene expression signatures involved in pluripotency and stemness. Because MYBL2 is currently not druggable, a MYBL2 gene signature was employed to identify cyclin-dependent kinase-2 (CDK2) as a potential therapeutic target. CDK2 inhibition phenocopied genetic loss of Mybl2 and significantly decreased in vivo tumor growth associated with enrichment of DNA damage. Together, this work demonstrates MYBL2 as an important MR-TF driving phenotypic plasticity in prostate cancer. Furthermore, high MYBL2 activity identifies prostate cancer that would be responsive to CDK2 inhibition. Significance: Prostate cancers that escape therapy targeting the androgen receptor signaling pathways via phenotypic plasticity are currently untreatable. Our study identifies MYBL2 as a MR-TF in phenotypic plastic prostate cancer and implicates CDK2 inhibition as a novel therapeutic target for this most lethal subtype of prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d7169da5236ce487e4af9bc41a5465e1a1c863" target='_blank'>
              MYBL2 Drives Prostate Cancer Plasticity: Inhibiting Its Transcriptional Target CDK2 for RB1-Deficient Neuroendocrine Prostate Cancer
              </a>
            </td>
          <td>
            Beatriz German, S. A. Alaiwi, Kun-Lin Ho, Jagpreet S Nanda, Marcos A S Fonseca, Deborah L. Burkhart, Anjali V. Sheahan, Hannah E Bergom, Katherine L. Morel, H. Beltran, Justin H. Hwang, M. Freedman, K. Lawrenson, Leigh Ellis
          </td>
          <td>2024-08-08</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single-nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosome 20q or 1q, both known for conferring an in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings showed higher loads of both CNVs and SNVs at later passages, which were due to the cumulative acquisition of mutations over a longer time in culture, and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs followed the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid of cancer-associated mutations. This suggests that recurrent chromosomal abnormalities are potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36cc208bebf01a60a96b6bcb671a8cdcf9d8bbf6" target='_blank'>
              De Novo Cancer Mutations Frequently Associate with Recurrent Chromosomal Abnormalities during Long-Term Human Pluripotent Stem Cell Culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huyghebaert, M. Regin, Mai Chi Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35a77ea00b4d522f22f234858138bc7f414f0459" target='_blank'>
              Integrative Bioinformatics Analysis: Unraveling Variant Signatures and Single-Nucleotide Polymorphism Markers Associated with 5-FU-Based Chemotherapy Resistance in Colorectal Cancer Patients.
              </a>
            </td>
          <td>
            Masomeh Askari, Ebrahim Mirzaei, Leila Navapour, M. Karimpour, L. Rejali, Somayeh Sarirchi, E. Nazemalhosseini-Mojarad, Stefania Nobili, Claudia Cava, Amir Sadeghi, N. Fatemi
          </td>
          <td>2024-09-06</td>
          <td>Journal of gastrointestinal cancer</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Connecting genetic variation to phenotypes and understanding the underlying biological mechanisms has been a fundamental goal of biological genetics. Here, we used the association analysis to identify a Vip3Aa resistance-associated genomic region in a strain of fall armyworm, JC-R, which exhibits >5000-fold resistance to the Bt toxin Vip3Aa. However, through various analytical approaches and fine-scale mapping across different populations, we demonstrated that this genomic region exhibits strong genetic linkage. The chromosome-level genome of JC-R and its parent strain JC-S were assembled, and extensive structural variations in the linkage regions were identified, which could be responsible for maintaining the linkage. To identify the causal variation within this linked region, a chromosome fragment stepwise knockout strategy based on CRISPR/Cas9 was developed. By crossing with the resistant strain and phenotyping segregating offspring on Vip3Aa-containing diet, we identified a chromosomal segment, KO8, containing the resistant gene. Subsequently, we conducted a comprehensive analysis of the variations in the KO8 region using multi-omics approaches, including genomic data, RNA-seq, proteomic, PacBio long read Iso-seq, and phosphoproteomic data. This analysis identified multiple variations in the chitin synthase gene CHS2, including amino acid substitution, alternative splicing, and changes in phosphorylation sites. After knocking out the CHS2, larvae exhibited over 6777-fold resistance to Vip3Aa. These results demonstrate that the chromosome fragment stepwise knockout strategy is a viable approach for studying complex genomic regions, and highlight the value of comprehensive analysis of genetic variations using multi-omics data. The identified candidate gene could potentially advance monitoring and management of pest resistance to Vip3Aa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ca50d6fcbaf49cb004ae82c9568f261b121333e" target='_blank'>
              CRISPR-mediated chromosome deletion facilitates genetic mapping of Vip3Aa resistance gene within complex genomic region in an invasive global pest
              </a>
            </td>
          <td>
            M. Jin, Yinxue Shan, Yan Peng, Senlin Chen, Xuanhe Zhou, Kaiyu Liu, Yutao Xiao
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23cf92719cd49496400fba609df94fa13b009008" target='_blank'>
              Precision medicine in AML: overcoming resistance.
              </a>
            </td>
          <td>
            Samuel Urrutia, Koichi Takahashi
          </td>
          <td>2024-08-01</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In this review, we revisit the pivotal role of fibroblast growth factor receptor 3 (FGFR3) in bladder cancer (BLCA), underscoring its prevalence in both non-muscle-invasive and muscle-invasive forms of the disease. FGFR3 mutations in up to half of BLCAs play a well-established role in tumorigenesis, shaping distinct tumor initiation patterns and impacting the tumor microenvironment (TME). Emphasizing the importance of considering epithelial-mesenchymal transition profile and TME status, we revisit their relevance in predicting responses to immune checkpoint inhibitors in FGFR3-mutated BLCAs. This writing highlights the initially promising yet transient efficacy of the FGFR inhibitor Erdafitinib on FGFR3-mutated BLCA, stressing the pressing need to unravel resistance mechanisms and identify co-targets for future combinatorial studies. A thorough analysis of recent preclinical and clinical evidence reveals resistance mechanisms, including secondary mutations, epigenetic alterations in pathway effectors, phenotypic heterogeneity, and population-specific variations within FGFR3 mutational status. Lastly, we discuss the potential of combinatorial treatments and concepts like synthetic lethality for discovering more effective targeted therapies against FGFR3-mutated BLCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31d32b54c415deb1782db39b2832fb5448db0b31" target='_blank'>
              FGFR3 alterations in bladder cancer: Sensitivity and resistance to targeted therapies.
              </a>
            </td>
          <td>
            Maxim Noeraparast, Katarina Krajina, Renate Pichler, Dora Niedersüß-Beke, S. Shariat, Viktor Grünwald, S. Ahyai, Martin Pichler
          </td>
          <td>2024-08-19</td>
          <td>Cancer communications</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>3</td>
          <td>36</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M As Sobeai
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematologic neoplasia with a complex polyclonal architecture. Among driver lesions, those involving the FLT3 gene represent the most frequent mutations identified at diagnosis. The development of tyrosine kinase inhibitors (TKIs) has improved the clinical outcomes of FLT3-mutated patients (Pt). However, overcoming resistance to these drugs remains a challenge. To unravel the molecular mechanisms underlying therapy resistance and clonal selection, we conducted a longitudinal analysis using a single-cell DNA sequencing approach (MissionBioTapestri® platform, San Francisco, CA, USA) in two patients with FLT3-mutated AML. To this end, samples were collected at the time of diagnosis, during TKI therapy, and at relapse or complete remission. For Pt #1, disease resistance was associated with clonal expansion of minor clones, and 2nd line TKI therapy with gilteritinib provided a proliferative advantage to the clones carrying NRAS and KIT mutations, thereby responsible for relapse. In Pt #2, clonal architecture was less complex, and 1st line TKI therapy with midostaurin was able to eradicate the leukemic clones. Our results corroborate previous findings about clonal selection driven by TKIs, highlighting the importance of a deeper characterization of individual clonal architectures for choosing the best treatment plan for personalized approaches aimed at optimizing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4202a0b52113f492de1c5d891aefedd2f36b44c" target='_blank'>
              Tracking Response and Resistance in Acute Myeloid Leukemia through Single-Cell DNA Sequencing Helps Uncover New Therapeutic Targets
              </a>
            </td>
          <td>
            S. Bruno, E. Borsi, A. Patuelli, L. Bandini, M. Mancini, D. Forte, J. Nanni, Martina Barone, Alessandra Grassi, G. Cristiano, C. Venturi, V. Robustelli, Giulia Atzeni, Cristina Mosca, S. De Santis, C. Monaldi, A. Poletti, C. Terragna, A. Curti, M. Cavo, Simona Soverini, E. Ottaviani
          </td>
          <td>2024-09-17</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Next generation sequencing of neuroblastoma (NB) tumors have revealed frequent somatic and germline genetic alterations in genes encoding proteins involved in DNA damage response (DDR) pathways. Despite being well-studied in many adult cancers, roles for DDR disruption in pediatric solid tumors remains poorly understood. To address this, patient-relevant loss-of-function mutations in DDR pathway components including Brca2, Atm, and Palb2 were incorporated into an established zebrafish MYCN transgenic model (Tg(dbh:EGFP-MYCN)). These mutations were found to enhance NB formation and metastasis in vivo, and result in upregulation of proliferation, cell cycle checkpoint and DNA damage repair transcriptional signatures, revealing potential molecular vulnerabilities in DDR-deficient NB. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Altogether, our data supports a functional role for DDR-deficiency in NB in vivo and therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes. Significance This work provides the first in vivo evidence supporting a functional role for DDR-deficiency in NB by demonstrating that alterations in certain DDR pathway genes promote NB formation and metastasis. NGS and pre-clinical drug testing also provides rationale for PARP + ATR inhibitor therapy combinations for patients with NB and pathogenic DDR pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdec5d0ffc7f47bcd4a92b02bbe2b4a531d6b2a" target='_blank'>
              DNA Damage Response Deficiency Enhances Neuroblastoma Progression and Sensitivity to Combination PARP and ATR Inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, Alex Weiss, M. Layeghifard, Yagnesh Ladumor, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Using five complementary short- and long-read sequencing technologies, we phased and assembled >95% of each diploid human genome in a four-generation, 28-member family (CEPH 1463) allowing us to systematically assess de novo mutations (DNMs) and recombination. From this family, we estimate an average of 192 DNMs per generation, including 75.5 de novo single-nucleotide variants (SNVs), 7.4 non-tandem repeat indels, 79.6 de novo indels or structural variants (SVs) originating from tandem repeats, 7.7 centromeric de novo SVs and SNVs, and 12.4 de novo Y chromosome events per generation. STRs and VNTRs are the most mutable with 32 loci exhibiting recurrent mutation through the generations. We accurately assemble 288 centromeres and six Y chromosomes across the generations, documenting de novo SVs, and demonstrate that the DNM rate varies by an order of magnitude depending on repeat content, length, and sequence identity. We show a strong paternal bias (75-81%) for all forms of germline DNM, yet we estimate that 17% of de novo SNVs are postzygotic in origin with no paternal bias. We place all this variation in the context of a high-resolution recombination map (∼3.5 kbp breakpoint resolution). We observe a strong maternal recombination bias (1.36 maternal:paternal ratio) with a consistent reduction in the number of crossovers with increasing paternal (r=0.85) and maternal (r=0.65) age. However, we observe no correlation between meiotic crossover locations and de novo SVs, arguing against non-allelic homologous recombination as a predominant mechanism. The use of multiple orthogonal technologies, near-telomere-to-telomere phased genome assemblies, and a multi-generation family to assess transmission has created the most comprehensive, publicly available “truth set” of all classes of genomic variants. The resource can be used to test and benchmark new algorithms and technologies to understand the most fundamental processes underlying human genetic variation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bcb8df79fd9bfc629f05070b595811f041bfe33" target='_blank'>
              A familial, telomere-to-telomere reference for human de novo mutation and recombination from a four-generation pedigree
              </a>
            </td>
          <td>
            David Porubsky, Harriet Dashnow, T. Sasani, Glennis A. Logsdon, P. Hallast, Michelle D. Noyes, Z. Kronenberg, T. Mokveld, Nidhi Koundinya, Cillian Nolan, C. Steely, A. Guarracino, E. Dolzhenko, William T. Harvey, W. Rowell, K. Grigorev, T. J. Nicholas, Keisuke K. Oshima, Jiadong Lin, Peter Ebert, W. Watkins, Tiffany Y. Leung, Vincent C. T. Hanlon, Sean Mcgee, Brent S. Pedersen, Michael E. Goldberg, Hannah C. Happ, Hyeonsoo Jeong, Katherine M. Munson, Kendra Hoekzema, Daniel D. Chan, Yanni Wang, Jordan Knuth, Gage H. Garcia, Cairbre Fanslow, Christine Lambert, Charles Lee, Josh Smith, Shawn Levy, Christopher E. Mason, Erik Garrison, Peter Lansdorp, D. Neklason, L. Jorde, Aaron R. Quinlan, M. Eberle, Evan E. Eichler
          </td>
          <td>2024-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Pancreatic cancer is an aggressive disease with limited therapeutic options. Gemcitabine is a clinically used chemotherapy for pancreatic cancer treatment but has limited efficacy. The molecular mechanisms responsible for gemcitabine resistance in pancreatic cancer are still unclear. Here, we analyzed the genomics of drug sensitivity in cancer dataset and identified various pancreatic cancer cells with distinct sensitivity to gemcitabine. Validation with proliferation and clonogenic assays confirmed that while CFPAC1 and PANC0327 are highly gemcitabine-sensitive, ASPC1 and PANC1 display intrinsic gemcitabine resistance. Analysis of the gene expression profile of these cells identified several genes that are differentially upregulated in resistant versus sensitive cells. Pathway enrichment analysis using the differential genes revealed the upregulation of Hippo, Wnt, MAPK signaling pathways as well as autophagy and glutathione metabolism in the resistant cells. Using quantitative PCR, immunoblotting and biochemical assays, and extracellular flux analysis methods, we have characterized the molecular distinctions between gemcitabine sensitive and resistant cells. Our results provide insight on the molecular mechanisms driving gemcitabine resistance in pancreatic cancer and could lead to novel therapeutic strategies to overcome resistance and improve patients’ treatment.
 Citation Format: Amina D Bilalbegovic, Bowen Fu, Zeribe Nwosu. Identification of novel drivers of gemcitabine resistance in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B089.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b52052963f11a91114395dd33df22ba52dd0b89f" target='_blank'>
              Abstract B089: Identification of novel drivers of gemcitabine resistance in pancreatic cancer
              </a>
            </td>
          <td>
            Amina D Bilalbegovic, Bowen Fu, Z. Nwosu
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d317e0704a11a531980e863a514ed7e3ab276870" target='_blank'>
              Advances in pathogenesis research and challenges in treatment development for acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hiroki Yamaguchi
          </td>
          <td>2024-09-03</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Microsatellite unstable colorectal cancer (MSI-CRC) can arise through germline mutations in mismatch repair (MMR) genes in individuals with Lynch syndrome (LS), or sporadically through promoter methylation of the MMR gene MLH1. Despite the different origins of hereditary and sporadic MSI tumours, their genomic features have not been extensively compared. A prominent feature of MMR-deficient genomes is the occurrence of many indels in short repeat sequences, an understudied mutation type due to the technical challenges of variant calling in these regions. In this study, we performed whole genome sequencing and RNA-sequencing on 29 sporadic and 14 hereditary MSI-CRCs. We compared the tumour groups by analysing genome-wide mutation densities, microsatellite repeat indels, recurrent protein-coding variants, signatures of single base, doublet base, and indel mutations, and changes in gene expression. We show that the mutational landscapes of hereditary and sporadic MSI-CRCs, including mutational signatures and mutation densities genome-wide and in microsatellites, are highly similar. Only a low number of differentially expressed genes were found, enriched to interferon-γ regulated immune response pathways. Analysis of the variance in allelic fractions of somatic variants in each tumour group revealed higher clonal heterogeneity in sporadic MSI-CRCs. Our results suggest that the differing molecular origins of MMR deficiency in hereditary and sporadic MSI-CRCs do not result in substantial differences in the mutational landscapes of these tumours. The divergent patterns of clonal evolution between the tumour groups may have clinical implications, as high clonal heterogeneity has been associated with decreased tumour immunosurveillance and reduced responsiveness to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43c6128f5465fd71306e14170da497710701fc42" target='_blank'>
              Lynch syndrome-associated and sporadic microsatellite unstable colorectal cancers: different patterns of clonal evolution yield highly similar tumours.
              </a>
            </td>
          <td>
            Samantha Martin, Riku Katainen, Aurora Taira, Niko Välimäki, Ari Ristimäki, T. Seppälä, L. Renkonen-Sinisalo, A. Lepistö, Kyösti Tahkola, Anne Mattila, S. Koskensalo, J. Mecklin, K. Rajamäki, Kimmo Palin, L. Aaltonen
          </td>
          <td>2024-08-24</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, Paul Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Leukemias are a group of heterogeneous hematological malignancies driven by diverse genetic variations, and the advent of genomic sequencing technologies facilitates the investigation of genetic abnormalities in leukemia. However, these sequencing-based studies mainly focus on nuclear DNAs. Increasing evidence indicates that mitochondrial dysfunction is an important mechanism of leukemia pathogenesis, which is closely related to the mitochondrial genome variations. Here, we provide an overview of current research progress concerning mitochondrial genetic variations in leukemia, encompassing gene mutations and copy number variations. We also summarize currently accessible mitochondrial DNA (mtDNA) sequencing methods. Notably, somatic mtDNA mutations may serve as natural genetic barcodes for lineage tracing and longitudinal assessment of clonal dynamics. Collectively, these findings enhance our understanding of leukemia pathogenesis and foster the identification of novel therapeutic targets and interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a51e65e882f1837d1adb0460175f694592b51dd7" target='_blank'>
              Mitochondrial genetic variations in leukemia: a comprehensive overview
              </a>
            </td>
          <td>
            Ao Zhang, Wenbing Liu, S. Qiu
          </td>
          <td>2024-09-05</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="TP53 functions as a central regulator in response to DNA damage and other cell stressors by inducing the expression of many protective pathways such as cell cycle arrest and apoptosis. Consequently, this gene is often found disrupted in human cancers. Elephants are a particularly interesting species for the study of cancer, by virtue of their large number of cell divisions and long lives yet low incidence of cancer. Elephants also possess multiple retrogene copies of TP53, which have previously been shown to induce strong cellular responses to DNA damage. However, most previous studies have largely focused only on African elephant TP53 retrogenes and often in non-native backgrounds. Here we generated CRISPR-Cas9 knockouts of TP53, all 29 TP53 retrogenes, or both in combination in Asian elephant fibroblasts. We find that while there is considerable overlap in the DNA damage responses of the TP53 and retrogene knockouts, there are also many unique pathways enriched in both. In particular, the retrogene knockouts exhibit strong enrichment of many extracellular pathways suggesting they may play a large role in the tumor microenvironment and mitigating metastatic growth. We also find that only a small fraction of these 29 retrogenes appear to be expressed across a variety of tissues and identify three loci that are likely driving this response. This work shows for the first time the transcriptomic effect of these retrogenes within their native background and establishes a foundation for future research into the relative contributions of these genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01cd22e9caf6290532e5ca297543c409aee6bcfc" target='_blank'>
              Engineering Asian elephant TP53: TP53 retrogene knockouts activate common and unique cancer-relevant pathways
              </a>
            </td>
          <td>
            Emil Karpinski, Nikil Badey, Esther Mintzer, Asaf Ashkenazy-Titelman, George M. Church
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU- 023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention. Author Summary Bile duct cancer has the highest incidence in northeastern Thailand. In this study, we explored the mtDNA of CCA cell lines, as mitochondria play a key role in cellular energy production. We aimed to identify mutations that could serve as therapeutic targets or biomarkers. Our analysis revealed that each CCA cell line has unique mtDNA profiles, including INDELs, SVs, and SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. yet they also shared some common mutations. Future research is needed to understand how these mutations affect cell function, interact with nuclear DNA, and their potential for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eafb79f26f32f1aeaf6d648b336c327747f051f" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapan, A. Jareanrat, V. Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Simple Summary This review provides a comprehensive overview of the current knowledge on mutational signatures in colorectal cancer (CRC), highlighting their potential clinical applications. It discusses the challenges and limitations in translating these analyses into clinical practice and proposes strategies to overcome these obstacles. Additionally, it provides insights into future directions and emerging proof-of-concept studies that highlight the translational potential of mutational signature analysis in improving patient care and outcomes in CRC. Abstract A multitude of exogenous and endogenous processes have the potential to result in DNA damage. While the repair mechanisms are typically capable of correcting this damage, errors in the repair process can result in mutations. The findings of research conducted in 2012 indicate that mutations do not occur randomly but rather follow specific patterns that can be attributed to known or inferred mutational processes. The process of mutational signature analysis allows for the inference of the predominant mutational process for a given cancer sample, with significant potential for clinical applications. A deeper comprehension of these mutational signatures in CRC could facilitate enhanced prevention strategies, facilitate the comprehension of genotoxic drug activity, predict responses to personalized treatments, and, in the future, inform the development of targeted therapies in the context of precision oncology. The efforts of numerous researchers have led to the identification of several mutational signatures, which can be categorized into different mutational signature references. In CRC, distinct mutational signatures are identified as correlating with mismatch repair deficiency, polymerase mutations, and chemotherapy treatment. In this context, a mutational signature analysis offers considerable potential for enhancing minimal residual disease (MRD) tests in stage II (high-risk) and stage III CRC post-surgery, stratifying CRC based on the impacts of genetic and epigenetic alterations for precision oncology, identifying potential therapeutic vulnerabilities, and evaluating drug efficacy and guiding therapy, as illustrated in a proof-of-concept clinical trial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9f4817ac8c40910eefa1198ffb3e184664ea3d" target='_blank'>
              Mutational Signatures in Colorectal Cancer: Translational Insights, Clinical Applications, and Limitations
              </a>
            </td>
          <td>
            G. Crisafulli
          </td>
          <td>2024-08-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d044e99711a824bcff1790b729c952743e0ce14" target='_blank'>
              Atorvastatin lowers breast cancer risk by reversing an early tumorigenic signature
              </a>
            </td>
          <td>
            Mohamed Y Foda, Mohamed L Salem, Fadhl Alakwaa, O. El-khawaga
          </td>
          <td>2024-08-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Objective The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions. Methods We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters. Results Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations. Conclusion Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, Weisen Fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Genomic rearrangements are chromosomal abnormalities that alter the arrangement of genes and are important for diagnosing multiple cancer types and guiding therapy selection. However, current diagnostic tests may not detect all genomic rearrangements. Hi-C sequencing is a promising new method for detecting genomic rearrangements. First, we examined whether Hi-C can detect known genomic rearrangements in pediatric leukemia and sarcoma specimens. Next, we evaluated whether Hi-C can detect genomic rearrangements that were not found using standard diagnostic testing. Hi-C showed complete agreement with other diagnostic methods in identifying known rearrangements. Hi-C also detected previously unknown genomic rearrangements in 5/11 pediatric leukemia cases that could impact diagnosis, prognosis, or treatment choices. These data suggest Hi-C could be beneficial for medical diagnostic testing of pediatric cancers, but more extensive clinical validation is needed. Abstract Hi-C sequencing is a DNA-based next-generation sequencing method that preserves the 3D genome conformation and has shown promise in detecting genomic rearrangements in translational research studies. To evaluate Hi-C as a potential clinical diagnostic platform, analytical concordance with routine laboratory testing was assessed using primary pediatric leukemia and sarcoma specimens. Archived viable and non-viable frozen leukemic cells and formalin-fixed paraffin-embedded (FFPE) tumor specimens were analyzed. Pediatric acute myeloid leukemia (AML) and alveolar rhabdomyosarcoma (A-RMS) specimens with known genomic rearrangements were subjected to Hi-C to assess analytical concordance. Subsequently, a discovery cohort consisting of AML and acute lymphoblastic leukemia (ALL) cases without known genomic rearrangements based on prior clinical diagnostic testing was evaluated to determine whether Hi-C could detect rearrangements. Using a standard sequencing depth of 50 million raw read-pairs per sample, or approximately 5X raw genomic coverage, we observed 100% concordance between Hi-C and previous clinical cytogenetic and molecular testing. In the discovery cohort, a clinically relevant gene fusion was detected in 45% of leukemia cases (5/11). This study provides an institutional proof of principle evaluation of Hi-C sequencing to medical diagnostic testing as it identified several clinically relevant rearrangements, including those that were missed by current clinical testing workflows.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af941e63f57e2e2bb71ed7543d231768d840bed5" target='_blank'>
              Evaluation of Hi-C Sequencing for Detection of Gene Fusions in Hematologic and Solid Tumor Pediatric Cancer Samples
              </a>
            </td>
          <td>
            Anthony D. Schmitt, Kristin Sikkink, Atif A. Ahmed, Shadi Melnyk, Derek Reid, Logan Van Meter, Erin M. Guest, Lisa A. Lansdon, Tomi Pastinen, Irina Pushel, B. Yoo, M. Farooqi
          </td>
          <td>2024-08-23</td>
          <td>Cancers</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/860a1cc9d077ab2536411e26d625b8a5e8cd579d" target='_blank'>
              Epigenome-wide impact of MAT2A sustains the androgen-indifferent state and confers synthetic vulnerability in ERG fusion-positive prostate cancer
              </a>
            </td>
          <td>
            A. Cacciatore, D. Shinde, C. Musumeci, G. Sandrini, L. Guarrera, D. Albino, G. Civenni, Elisa Storelli, S. Mosole, Elisa Federici, Alessio Fusina, Marta Iozzo, Andrea Rinaldi, Matteo Pecoraro, Roger Geiger, Marco Bolis, C. Catapano, G. M. Carbone
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Expanding tandem gene arrays facilitates adaptation through dosage effects and gene family formation via sequence diversification. However, experimental induction of such expansions remains challenging. Here we introduce a method termed break-induced replication (BIR)-mediated tandem repeat expansion (BITREx) to address this challenge. BITREx strategically places Cas9 nickase adjacent to a tandem gene array to break the replication fork that has replicated the array, forming a single-end double-strand break. This break is subsequently end-resected to become single-stranded. Since there is no repeat unit downstream of the break, the single-stranded DNA often invades an upstream unit to initiate ectopic BIR, resulting in array expansion. BITREx has successfully expanded gene arrays in budding yeast, with the CUP1 array reaching ∼1 Mb. Furthermore, appropriate splint DNA allows BITREx to generate tandem arrays de novo from single-copy genes. We have also demonstrated BITREx in mammalian cells. Therefore, BITREx will find various unique applications in genome engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33093f987bef3bbf4086ae4d4df3919c786e836c" target='_blank'>
              Strategic targeting of Cas9 nickase expands tandem gene arrays
              </a>
            </td>
          <td>
            Hiroaki Takesue, Satoshi Okada, Goro Doi, Yuki Sugiyama, Emiko Kusumoto, Takashi Ito
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a diverse malignancy originating from myeloid progenitor cells, with significant genetic and clinical variability. Modern classification systems like those from the World Health Organization (WHO) and European LeukemiaNet use immunophenotyping, molecular genetics, and clinical features to categorize AML subtypes. This classification highlights crucial genetic markers such as FLT3, NPM1 mutations, and MLL-AF9 fusion, which are essential for prognosis and directing targeted therapies. The MLL-AF9 fusion protein is often linked with therapy-resistant AML, highlighting the risk of relapse due to standard chemotherapeutic regimes. In this sense, factors like the ZEB, SNAI, and TWIST gene families, known for their roles in epithelial–mesenchymal transition (EMT) and cancer metastasis, also regulate hematopoiesis and may serve as effective therapeutic targets in AML. These genes contribute to cell proliferation, differentiation, and extramedullary hematopoiesis, suggesting new possibilities for treatment. Advancing our understanding of the molecular mechanisms that promote AML, especially how the bone marrow microenvironment affects invasion and drug resistance, is crucial. This comprehensive insight into the molecular and environmental interactions in AML emphasizes the need for ongoing research and more effective treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b827ac4bebbcac825ffc1c60412a291161ab8775" target='_blank'>
              The Role of Epithelial-to-Mesenchymal Transition Transcription Factors (EMT-TFs) in Acute Myeloid Leukemia Progression
              </a>
            </td>
          <td>
            Diego Cuevas, Roberto Amigo, Adolfo Agurto, Adan Andreu Heredia, , , , Teresa Caprile, Jody J Haigh, Carlos Farkas
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Psoriasis is a chronic skin disease with an increasing prevalence each year. However, the mechanisms underlying its onset and progression remain unclear, and effective therapeutic targets are lacking. Therefore, we employs an innovative approach by combining single-cell RNA sequencing (scRNA-seq) with meta-analysis. This not only elucidates the potential mechanisms of psoriasis at the cellular level but also identifies immunoregulatory marker genes that play a statistically significant role in driving psoriasis progression through comprehensive analysis of multiple datasets. Skin tissue samples from 12 psoriasis patients underwent scRNA-seq, followed by quality control, filtering, PCA dimensionality reduction, and tSNE clustering analysis to identify T cell subtypes and differentially expressed genes (DEGs) in psoriatic skin tissue. Next, three psoriasis datasets were standardised and merged to identify differentially expressed genes (DEGs). Subsequently, weighted gene co-expression network analysis (WGCNA) was applied for clustering analysis of gene co-expression network modules and to assess the correlation between these modules and DEGs. Least absolute shrinkage and selection operator (LASSO) regression and receiver operating characteristic (ROC) curve analyses were conducted to select disease-specific genes and evaluate their diagnostic value. Single-cell data revealed nine cell types in psoriatic skin tissue, with seven T cell subtypes identified. Intersection analysis identified ADAM8 and G3BP2 as key genes. Through the integration of scRNA-seq and Meta analysis, we identified the immunoregulatory marker gene G3BP2, which is associated with the onset and progression of psoriasis and holds clinical significance. G3BP2 is speculated to promote the development of psoriasis by increasing the proportion of CD8+ T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84a0f4299cf8246872d71f67ae65b28acc87e89e" target='_blank'>
              Identification of psoriasis-associated immune marker G3BP2 through single-cell RNA sequencing and meta analysis.
              </a>
            </td>
          <td>
            Shuangshuang Gao, Hua-Yu Fan, Ting Wang, Jinguang Chen
          </td>
          <td>2024-09-12</td>
          <td>Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract Understanding the evolutionary potential of mutations in gene regulatory networks is essential to furthering the study of evolution and development. However, in multicellular systems, genetic manipulation of regulatory networks in a targeted and high-throughput way remains challenging. In this study, we designed TF-High-Evolutionary (HighEvo), a transcription factor (TF) fused with a base editor (activation-induced deaminase), to continuously induce germline mutations at TF-binding sites across regulatory networks in Drosophila. Populations of flies expressing TF-HighEvo in their germlines accumulated mutations at rates an order of magnitude higher than natural populations. Importantly, these mutations accumulated around the targeted TF-binding sites across the genome, leading to distinct morphological phenotypes consistent with the developmental roles of the tagged TFs. As such, this TF-HighEvo method allows the interrogation of the mutational space of gene regulatory networks at scale and can serve as a powerful reagent for experimental evolution and genetic screens focused on the regulatory genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2f5224e39a793a9098b6100dcfef52f5acedf2" target='_blank'>
              TF-High-Evolutionary: In Vivo Mutagenesis of Gene Regulatory Networks for the Study of the Genetics and Evolution of the Drosophila Regulatory Genome
              </a>
            </td>
          <td>
            Xueying C. Li, Vani Srinivasan, Ian Laiker, Natalia Misunou, Nicolás Frankel, Luisa F. Pallares, Justin Crocker
          </td>
          <td>2024-08-01</td>
          <td>Molecular Biology and Evolution</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Pediatric acute lymphoblastic leukemia (ALL) is a highly heterogeneous disease at the molecular level, with over 20 subtypes of B-cell ALL (B-ALL) identified to date. These subtypes are critical for guiding risk-adapted therapy and precision medicine, as they are defined by specific chromosomal rearrangements, gene expression profiles, aneuploidies and point mutations. However, the ability to accurately classify these subtypes is limited by the technical and economic challenges faced by many centers, particularly in low- and middle-income countries. In this study, we employed transcriptome sequencing to perform molecular classification of B-ALL at diagnosis in pediatric patients enrolled in the multicentric ALLIC-GATLA-2010 clinical protocol in Argentina. Using a combination of bioinformatic tools, we successfully identified single nucleotide variants, fusion transcripts and gene expression profiles, achieving molecular classification in over 90% of patients. Our analysis also revealed high-risk molecular features and novel genetic alterations. These findings hold potential clinical value for improving risk stratification and identifying therapeutic targets, particularly for patients who remain unclassified by conventional diagnostic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f39fa251d291498250d854c37a9028d4e85f79" target='_blank'>
              Molecular Characterization of Pediatric Acute Lymphoblastic Leukemia via Integrative Transcriptomics: A Multicenter Study in Argentina
              </a>
            </td>
          <td>
            Sol Ruiz, María Mercedes Abbate, Ezequiel Sosa, Daniel Avendaño, Ignacio Gomez Mercado, María Laura Lacreu, Mariano Riccheri, Virginia Schuttenberg, Luis Aversa, E. Vazquez, G. Gueron, J. Cotignola
          </td>
          <td>2024-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT Deficient (d) DNA mismatch repair (MMR) is a biomarker predictive of better response to PD-1 blockade immunotherapy in solid tumors. dMMR can be caused by mutations in MMR genes or by protein inactivation, which can be detected by sequencing and immunohistochemistry, respectively. To investigate the role of dMMR in diffuse large B-cell lymphoma (DLBCL), MMR gene mutations and expression of MSH6, MSH2, MLH1, and PMS2 proteins were evaluated by targeted next-generation sequencing and immunohistochemistry in a large cohort of DLBCL patients treated with standard chemoimmunotherapy, and correlated with the tumor immune microenvironment characteristics quantified by fluorescent multiplex immunohistochemistry and gene-expression profiling. The results showed that genetic dMMR was infrequent in DLBCL and was significantly associated with increased cancer gene mutations and favorable immune microenvironment, but not prognostic impact. Phenotypic dMMR was also infrequent, and MMR proteins were commonly expressed in DLBCL. However, intratumor heterogeneity existed, and increased DLBCL cells with phenotypic dMMR correlated with significantly increased T cells and PD-1+ T cells, higher average nearest neighbor distance between T cells and PAX5+ cells, upregulated immune gene signatures, LE4 and LE7 ecotypes and their underlying Ecotyper-defined cell states, suggesting the possibility that increased T cells targeted only tumor cell subsets with dMMR. Only in patients with MYC¯ DLBCL, high MSH6/PMS2 expression showed significant adverse prognostic effects. This study shows the immunologic and prognostic effects of genetic/phenotypic dMMR in DLBCL, and raises a question on whether DLBCL-infiltrating PD-1+ T cells target only tumor subclones, relevant for the efficacy of PD-1 blockade immunotherapy in DLBCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/398a69413a5280f59d02312c6f8460b206f61fde" target='_blank'>
              DNA mismatch repair defect and intratumor heterogeneous deficiency differently impact immune responses in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Z. Xu-Monette, Cancan Luo, Li Yu, Yong Li, Govind Bhagat, A. Tzankov, Carlo Visco, Xiangshan Fan, K. Dybkaer, Ali Sakhdari, Nicholas T Wang, Alyssa F. Yuan, April Chiu, Wayne Tam, Y. Zu, Eric D Hsi, Anamarija M. Perry, Wenting Song, Dennis O'Malley, Q. Au, H. Nunns, H. Go, Michael B Møller, Benjamin Marshall Parsons, S. Montes-Moreno, M. Ponzoni, A. Ferreri, A. Sohani, J. Abramson, Bing Xu, Ken H. Young
          </td>
          <td>2024-08-05</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M. Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5edea2d9a83527bbebc5d48f3ba44b53a9b424d3" target='_blank'>
              Single-cell profiling uncovers proliferative cells as key determinants of survival outcomes in lower-grade glioma patients
              </a>
            </td>
          <td>
            Jian-Bing Peng, Qing Zhang, Xiaofeng Zhu, Zhu Yan, Meng Zhu
          </td>
          <td>2024-09-14</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Chronic Lymphocytic Leukemia (CLL) is the most common B-cell malignancy in the Western world, characterized by frequent relapses despite temporary remissions. Our study integrated publicly available proteomic, transcriptomic, and patient survival datasets to identify key differences between healthy and CLL samples. We exposed approximately 1000 proteins that differentiate healthy from cancerous cells, with 608 upregulated and 415 downregulated in CLL cases. Notable upregulated proteins include YEATS2 (an epigenetic regulator), PIGR (Polymeric immunoglobulin receptor), and SNRPA (a splicing factor), which may serve as prognostic biomarkers for this disease. Key pathways implicated in CLL progression involve RNA processing, stress resistance, and immune response deficits. Furthermore, we identified three existing drugs—Bosutinib, Vorinostat, and Panobinostat—for potential further investigation in drug repurposing in CLL. We also found limited correlation between transcriptomic and proteomic data, emphasizing the importance of proteomics in understanding gene expression regulation mechanisms. This generally known disparity highlights once again that mRNA levels do not accurately predict protein abundance due to many regulatory factors, such as protein degradation, post-transcriptional modifications, and differing rates of translation. These results demonstrate the value of integrating omics data to uncover deregulated proteins and pathways in cancer and suggest new therapeutic avenues for CLL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/524d87cfc3fcaee95b1a168a590bc69c95feb91a" target='_blank'>
              Integrative Analysis of Multi-Omics Data to Identify Deregulated Molecular Pathways and Druggable Targets in Chronic Lymphocytic Leukemia
              </a>
            </td>
          <td>
            D. Mavridou, K. Psatha, M. Aivaliotis
          </td>
          <td>2024-08-01</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/553f4560eb657fca2fe86385e460c71427df430c" target='_blank'>
              GLI1 confers resistance to PARP inhibitors by activating the DNA damage repair pathway.
              </a>
            </td>
          <td>
            Hiroshi Ikeuchi, Yusuke Matsuno, Rika Kusumoto-Matsuo, S. Kojima, Toshihide Ueno, Masachika Ikegami, Rina Kitada, Hitomi Sumiyoshi-Okuma, Yuki Kojima, K. Yonemori, Y. Yatabe, K. Takamochi, Kenji Suzuki, Ken‐ichi Yoshioka, H. Mano, S. Kohsaka
          </td>
          <td>2024-08-03</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/244d976555710935c004ec15262f6387c6d24c28" target='_blank'>
              Identifying prognostic biomarkers in oral squamous cell carcinoma: an integrated single-cell and bulk RNA sequencing study on mitophagy-related genes
              </a>
            </td>
          <td>
            Minsi Li, Yi Wei, Wenhua Huang, Cen Wang, Shixi He, Shuwen Bi, Shuangyu Hu, Ling You, Xuan-ping Huang
          </td>
          <td>2024-08-28</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb2fd505ae2ea1de627590bede77cfbae2e3ab46" target='_blank'>
              CRISPR screen of venetoclax response-associated genes identifies transcription factor ZNF740 as a key functional regulator
              </a>
            </td>
          <td>
            Lixia Zhang, Xinyue Zhou, Sajesan Aryal, Virginia Veasey, Pengcheng Zhang, Fu Jun Li, Yu Luan, Ravi Bhatia, Yang Zhou, Rui Lu
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Alterations in the FMS-like tyrosine kinase 3 (FLT3) gene are the most frequent driver mutations in acute myeloid leukaemia (AML), linked to a high risk of relapse in patients with internal tandem duplications (FLT3-ITD). Tyrosine kinase inhibitors (TKIs) targeting the FLT3 protein are approved for clinical use, yet resistance often emerges. This resistance is mainly seen following the acquisition of additional point mutations in the tyrosine kinase domain (TKD), resulting in a double mutant FLT3-ITD/TKD, which sustains cell signalling and survival despite the presence of FLT3 inhibitors. Here, we developed a FLT3-mutant AML model with adaptive resistance to type II TKIs, sorafenib, and quizartinib by in vitro drug selection. Through global multiomic profiling, we identified upregulation of proteins involved in reactive oxygen species (ROS) production, particularly NADPH-oxidases, driving cellular ’ROS-addiction’, with resistant cells relying on ROS for survival, and genome fidelity preserved by ATM-driven DNA repair. Transcriptomic analysis of adult and paediatric AML (pAML) patients identified high ATM expression as a biomarker for shorter median overall survival in both the de novo and relapsed settings. Inhibition of ATM with clinically relevant therapy WSD-0628 effectively killed TKI- and chemotherapy-resistant AML cells in vitro and significantly extended the survival of mice with sorafenib- and quizartinib-resistant FLT3-ITD AML in vivo. We propose a new treatment strategy to improve survival of patients who develop resistance to sorafenib and quizartinib, as well as relapsed and refractory pAML, exploiting resistance mechanisms to precision therapies and cell-intrinsic features of high-risk cases, highlighting a clinically relevant salvage strategy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0bcd4e64b15b98143cfc995f3b3d32e9c623b66" target='_blank'>
              Adaptive resistance to FLT3 inhibitors is potentiated by ROS-driven DNA repair signalling
              </a>
            </td>
          <td>
            Dilana E. Staudt, Zacary P. Germon, Abdul Mannan, Tabitha Mclachlan, Heather C Murray, Ryan J. Duchatel, Bryce C. Thomas, Tyrone S Beitaki, Holly P. McEwen, Mika L Persson, Leah Calvert, Izac J. Findlay, Evangeline R. Jackson, Nathan D. Smith, D. Skerrett‐Byrne, David Mossman, Brett Nixon, Geoffry De Iullis, Alicia M. Douglas, A. Enjeti, Jonathan R. Sillar, Janis Chamberlain, Frank Alvaro, Andrew H. Wei, Patrick Connerty, N. Verrills, Matthew D. Dun
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5ea42f9ac2f6d502e99cce5e348de337e22eda" target='_blank'>
              Effects of HOX family regulator-mediated modification patterns and immunity characteristics on tumor-associated cell type in endometrial cancer
              </a>
            </td>
          <td>
            Jiaolin Yang, Jinpeng Li, Sufen Li, Yutong Yang, Huancheng Su, Hongrui Guo, Jing Lei, YaLin Wang, KaiTing Wen, Xia Li, Sanyuan Zhang, Zhe Wang
          </td>
          <td>2024-08-14</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bbe642fe1a5320aff996d83dd54d13a05e4037" target='_blank'>
              The intersection of homologous recombination (HR) and mismatch repair (MMR) pathways in DNA repair-defective tumors
              </a>
            </td>
          <td>
            L. Incorvaia, T. B. Bazan Russo, V. Gristina, A. Perez, C. Brando, Clarissa Mujacic, Emilia Di Giovanni, M. Bono, Silvia Contino, Carla Ferrante Bannera, Maria Concetta Vitale, Andrea Gottardo, M. Peri, A. Galvano, D. Fanale, G. Badalamenti, Antonio Russo, V. Bazan
          </td>
          <td>2024-09-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Introduction Type B1 and B2 thymomas are lymphocyte-rich malignant tumors with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, non-coding regions remain uncharacterized. Here, we developed a rigorous tumor isolation method from lymphocyte-rich thymoma tissues and identified somatic mutations in non-coding and mitochondrial DNA. Methods CD205+CD45- pure tumor cells were isolated from fresh-frozen tissues using DEPArray system. Deep whole-genome sequencing was performed, and recurrent somatic alterations in coding, non-coding, and mitochondria regions were systemically identified by computational framework. The mutations were classified according to gene function, cis-regulatory element, and mutational signature. Results The total number of somatic mutations was approximately 80 times higher in non-coding regions than in coding regions in type B1-2 thymomas (1,671.3 vs. 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of cases exhibited fewer than four mutations in coding regions. A systematic non-coding analysis identified a total of 405.0 mutations per case on cis-regulatory elements, and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Mitochondrial heteroplasmy was observed in 90% of cases, with a significant proportion of mutations located in D-loop region. The single-base substitution pattern was signature 12. Conclusions Numerous non-coding mutations and mitochondrial heteroplasmy were detected in type B1 and B2 thymomas. Given the paucity of coding mutations observed in this disease entity, disruption of the non-coding landscape and mitochondrial heteroplasmic shift may be the primary cause of thymoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/559b9056aa5e8bd421441f43ad9a55ae5b38dbc9" target='_blank'>
              Genome-wide analysis of somatic non-coding mutation patterns and mitochondrial heteroplasmy in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Juliet King, Emma McLean, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mutations commonly found in AML such as DNMT3A, TET2 and ASXL1 can be found in the peripheral blood of otherwise healthy adults a phenomenon referred to as clonal hematopoiesis (CH). These mutations are thought to represent the earliest genetic events in the evolution of AML. Genomic studies on samples acquired at diagnosis, remission, and at relapse have demonstrated significant stability of CH mutations following induction chemotherapy. Meanwhile, later mutations in genes such as NPM1 and FLT3, have been shown to contract at remission and in the case of FLT3 often are absent at relapse. We sought to understand how early CH mutations influence subsequent evolutionary trajectories throughout remission and relapse in response to induction chemotherapy. Here, we assembled a retrospective cohort of patients diagnosed with de novo AML at our institution that underwent genomic sequencing at diagnosis as well as at the time of remission and/or relapse (total n = 182 patients). Corroborating prior studies, FLT3 and NPM1 mutations were generally eliminated at the time of cytologic complete remission but subsequently reemerged upon relapse, whereas DNMT3A, TET2 and ASXL1 mutations often persisted through remission. Early CH-related mutations exhibited distinct constellations of co-occurring genetic alterations, with NPM1 and FLT3 mutations enriched in DNMT3Amut AML, while CBL and SRSF2 mutations were enriched in TET2mut and ASXL1mut AML, respectively. In the case of NPM1 and FLT3 mutations, these differences vanished at the time of complete remission yet readily reemerged upon relapse, indicating the reproducible nature of these genetic interactions. Thus, early CH-associated mutations that precede malignant transformation subsequently shape the evolutionary trajectories of AML through diagnosis, therapy, and relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45f35657f001405231a2b4d996ed0ac3f0415703" target='_blank'>
              Early drivers of clonal hematopoiesis shape the evolutionary trajectories of de novo acute myeloid leukemia
              </a>
            </td>
          <td>
            Ryan D. Chow, Priya Velu, Safoora Deihimi, Jonathan Belman, Angela Youn, Nisargbhai Shah, Selina M. Luger, Martin P. Carroll, Jennifer J D Morrissette, Robert L. Bowman
          </td>
          <td>2024-09-01</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Leukemia is a malignant tumor with high heterogeneity and a complex evolutionary process. It is difficult to resolve the heterogeneity and clonal evolution of leukemia cells by applying traditional bulk sequencing techniques, thus preventing a deep understanding of the mechanisms of leukemia development and the identification of potential therapeutic targets. However, with the development and application of single-cell sequencing technology, it is now possible to investigate the gene expression profile, mutations, and epigenetic features of leukemia at the single-cell level, thus providing a new perspective for leukemia research. In this article, we review the recent applications and advances of single-cell sequencing technology in leukemia research, discuss its potential for enhancing our understanding of the mechanisms of leukemia development, discovering therapeutic targets and personalized treatment, and provide reference guidelines for the significance of this technology in clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059e4fa52cbb0003a82e3f417ae0958def767ace" target='_blank'>
              Application and research progress of single cell sequencing technology in leukemia
              </a>
            </td>
          <td>
            Dan Xie, Bangquan An, Mingyue Yang, Lei Wang, Min Guo, Heng Luo, Shengwen Huang, Fa Sun
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) has a poor survival rate for both pediatric and adult patients due to its frequent relapse. To elucidate the bioenergetic principle underlying AML relapse, we investigated the transcriptional regulation of mitochondrial–nuclear dual genomes responsible for metabolic plasticity in treatment-resistant blasts. Both the gain and loss of function results demonstrated that NFκB2, a noncanonical transcription factor (TF) of the NFκB (nuclear factor kappa-light-chain-enhancer of activated B cells) family, can control the expression of TFAM (mitochondrial transcription factor A), which is known to be essential for metabolic biogenesis. Furthermore, genetic tracking and promoter assays revealed that NFκB2 is in the mitochondria and can bind the specific “TTGGGGGGTG” region of the regulatory D-loop domain to activate the light-strand promoter (LSP) and heavy-strand promoter 1 (HSP1), promoters of the mitochondrial genome. Based on our discovery of NFκB2′s novel function of regulating mitochondrial–nuclear dual genomes, we explored a novel triplet therapy including inhibitors of NFκB2, tyrosine kinase, and mitochondrial ATP synthase that effectively eliminated primary AML blasts with mutations of the FMS-related receptor tyrosine kinase 3 (FLT3) and displayed minimum toxicity to control cells ex vivo. As such, effective treatments for AML must include strong inhibitory actions on the dual genomes mediating metabolic plasticity to improve leukemia prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd91cb6b195c34903e27bb3147d1ba2db984023c" target='_blank'>
              Discovery of NFκB2-Coordinated Dual Regulation of Mitochondrial and Nuclear Genomes Leads to an Effective Therapy for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Yi Xu, David J. Baylink, Jeffrey Xiao, Lily Tran, Vinh Nguyen, Brandon Park, Ismael Valladares, Scott Lee, Kevin Codorniz, Laren D Tan, Chien-Shing Chen, Hisham Abdel-Azim, Mark E. Reeves, Hamid Mirshahidi, Guido Marcucci, H. Cao
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ec02f56417ae90246c83ab25ae98a1c2c7e30" target='_blank'>
              Targeting treatment resistance: unveiling the potential of RNA methylation regulators and TG-101,209 in pan-cancer neoadjuvant therapy
              </a>
            </td>
          <td>
            Yaoyao Zhou, Ziyun Liu, Cheng Gong, Jie Zhang, Jing Zhao, Xia Zhang, Xiangyu Liu, Bin Li, Rui Li, Zhenyu Shi, Yongjie Xie, Li Bao
          </td>
          <td>2024-08-19</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Numerous genetic mutations have been documented in HCC, and associated with clinical characters. Herein, we demonstrated that XIRP2 mutation is one of the high-frequency mutations in HCC, and is associated with poor prognosis and drug resistance. The presence of the XIRP2 mutation leads to enhanced stability of the XIRP2 protein, whereas its depletion significantly heightens sensitivity to oxaliplatin by inducing an excessive influx of zinc and calcium ions. XIRP2 mutation holds potential as a prognostic biomarker and that targeting the XIRP2 protein may offer therapeutic benefits in the management of HCC. Abstract Hepatocellular carcinoma (HCC) is a prevalent malignant digestive tumor. Numerous genetic mutations have been documented in HCC, yet the clinical significance of these mutations remains largely unexplored. The objective of this study is to ascertain the clinical value and biological effects of xin actin binding repeat containing 2 (XIRP2) mutation in HCC. The gene mutation landscape of HCC was examined using data from the Cancer Genome Atlas and the International Cancer Genome Consortium databases. The prognostic significance of the XIRP2 mutation was assessed through KM plot analysis. The association between drug sensitivity and the XIRP2 mutation was investigated using the TIDE algorithm and CCK-8 experiments. The biological effects of the XIRP2 mutation were evaluated through qRT-PCR, protein stability experiments, and relevant biological experiments. The XIRP2 mutation is one of the high-frequency mutations in HCC, and is associated with poor prognosis. A total of 72 differentially expressed genes (DEGs) were observed in HCC tissues with the XIRP2 mutation as compared to those with the XIRP2 wildtype, and these DEGs were closely related to ion metabolic processes. The XIRP2 mutation was linked to alterations in the sensitivity of fludarabine, oxaliplatin, WEHI-539, and LCL-161. CCK-8 assays demonstrated that HCC cells carrying the XIRP2 mutation exhibited increased resistance to fludarabine and oxaliplatin, but enhanced sensitivity to WEHI-539 and LCL-161 as compared with those HCC cells with the XIRP2 wildtype. The XIRP2 mutation was found to have no impact on the mRNA levels of XIRP2 in tissues and cells, but it did enhance the stability of the XIRP2 protein. Mechanically, the inhibition of XIRP2 resulted in a significant increase in sensitivity to oxaliplatin through an elevation in zinc ions and a calcium ion overload. In conclusion, the XIRP2 mutation holds potential as a biomarker for predicting the prognosis and drug sensitivity of HCC and serves as a therapeutic target to enhance the efficacy of oxaliplatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b9ed326e9255f6375da439690c4e53906be89c6" target='_blank'>
              Identification of Clinical Value and Biological Effects of XIRP2 Mutation in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Dahuan Li, Xin Bao, Shan Lei, Wenpeng Cao, Zhirui Zeng, Tengxiang Chen
          </td>
          <td>2024-08-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Richter's transformation (RT) is defined as the evolution of chronic lymphocytic leukemia (CLL) or small lymphocytic lymphoma (SLL) into an aggressive lymphoma, most commonly diffuse large B-cell lymphoma. This complication is rare and aggressive, with poor prognosis and dismal survival. Clonal relationship with the underlying CLL/SLL, observed in ∼80% of cases, represents one of the main factors affecting prognosis. Treatment has been historically based on chemoimmunotherapy, but frequent mutations in genes involved in cell survival and proliferation-such as TP53, NOTCH1, MYC, CDKN2A-confer resistance to standard treatments. During the last years, advances in the knowledge of the biological mechanisms underlying RT allowed to identify genetic and molecular lesions that can potentially be targeted by novel selective agents. Pathway and checkpoint inhibitors, bispecific antibodies and CAR T-cell therapy are currently under investigation and represent promising treatment options. This review summarizes current biological evidence and available data on novel therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a1bac15f3b796b2ef346e32ef1ee6b7161f4b12" target='_blank'>
              Advances in the understanding of molecular genetics and therapy of Richter transformation in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            M. Deodato, A. Frustaci, Arianna Zappaterra, Alberto Rapella, C. Gambacorti-Passerini, R. Cairoli, Marco Montillo, Alessandra Tedeschi
          </td>
          <td>2024-09-02</td>
          <td>Leukemia & lymphoma</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Acute myeloid leukemia with complex karyotype (ckAML) is characterized by high genomic complexity, including frequent TP53 mutations and chromothripsis. We hypothesized that the numerous genomic rearrangements could reposition active enhancers near proto-oncogenes, leading to their aberrant expression. We developed pyjacker, a computational tool for the detection of enhancer hijacking events, and applied it to a cohort of 39 ckAML samples. Pyjacker identified motor neuron and pancreas homeobox 1 (MNX1), a gene aberrantly expressed in 1.4% of AML patients, often as a result of del(7)(q22q36) associated with hijacking of a CDK6 enhancer. MNX1-activated cases show significant co-occurrence with BCOR mutations and a gene signature shared with t(7;12)(q36;p13) pediatric AML. We demonstrated that MNX1 is a dependency gene, as its knockdown in a xenograft model reduces leukemia cell fitness. In conclusion, enhancer hijacking is a frequent mechanism for oncogene activation in AML. Statement of significance This study examines the consequences of structural alterations and demonstrates that proto-oncogene activation by enhancer hijacking is an overlooked pathomechanism in AML. MNX1 overexpression demonstrates that deletions on chromosome 7q can not only lead to haploinsufficiency, but also to activation of oncogenes by enhancer hijacking, providing a novel leukemogenic mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb3e7eaaf436e861083291af3f4d69f9c709309" target='_blank'>
              Pyjacker identifies enhancer hijacking events in acute myeloid leukemia including MNX1 activation via deletion 7q
              </a>
            </td>
          <td>
            Etienne Sollier, Anna Riedel, U. Toprak, Justyna A. Wierzbinska, D. Weichenhan, J. P. Schmid, M. Hakobyan, Aurore Touzart, E. Jahn, B. Vick, Fiona Brown-Burke, Katherine Kelly, Simge Kelekçi, Anastasija Pejkovska, Ashish Goyal, Marion Bähr, Kersten Breuer, , Maria Llamazares-Prada, M. Hartmann, M. Schönung, Nádia Correia, A. Trumpp, Yomn Abdullah, Ursula Klingmüller, S. S. Mughal, B. Brors, Frank Westermann, Matthias Schlesner, S. Vosberg, T. Herold, P. Greif, Dietmar Pfeifer, Michael Lübbert, Thomas Fischer, F. H. Heidel, C. Gebhard, W. Walter, Torsten Haferlach, Ann-Kathrin Eisfeld, K. Mrózek, D. Nicolet, L. Bullinger, L. Smeenk, Claudia Erpelinck, Roger Mulet-Lazaro, Ruud Delwel, Aurélie Ernst, Michael Scherer, P. Lutsik, I. Jeremias, Konstanze Döhner, H. Döhner, D. Lipka, C. Plass
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="The fusion expression of DNA replication-related proteins with nucleotide deaminase enzymes promotes random mutations in bacterial genomes, thereby increasing genetic diversity among population. Most previous studies have focused on cytosine deaminase, which produces only C→T mutations, significantly limiting the variety of mutation types. In this study, we developed a fusion expression system by combining DnaG (RNA primase) with adenine deaminase TadA-8e (DnaG-TadA) in Escherichia coli, which is capable of rapidly introducing A→G mutations into the E. coli genome, resulting in a 664-fold increase in terms of mutation rate. Additionally, we engineered a dual-functional TadA variant, TadAD, and then fused it with DnaG. This construct introduced both C→T and A→G mutations into the E. coli genome, with the mutation rate further increased by 370-fold upon co-expression with an uracil glycosylase inhibitor (DnaG-TadAD-UGI). We applied DnaG-TadA and DnaG-TadAD-UGI systems to the adaptive laboratory evolution for Cd2+ and kanamycin resistance, achieving an 8.0 mM Cd2+ and 200 μg/mL kanamycin tolerance within just 17 days and 132 hours, respectively. Compared to conventional evolution methods, the final tolerance levels were increased by 320% and 266%, respectively. Our work offers a novel strategy for random mutagenesis in E. coli and potentially other prokaryotic species. For TOC only">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8dc3e641872269a76809a2a9f58812a1c2451c2" target='_blank'>
              Genome-wide A→G and C→T Mutations Induced by Functional TadA Variants in Escherichia coli
              </a>
            </td>
          <td>
            Hao Wang, Zhengxin Dong, Jingyi Shi, Lei Chen, Tao Sun, Weiwen Zhang
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Breast cancer (BRCA) is a significant cause of cancer-associated mortality across the globe. Current therapeutic approaches face challenges such as drug resistance and metastasis. Immune signaling is triggered by chromosomal instability (CIN) generates misplaced DNA structures that activate the cyclic GMP–AMP synthase–stimulator of interferon genes (cGAS-STING) pathway, triggering. Studies have linked STING activation to BRCA treatment. Methods The bulk RNA-seq data for patients with BRCA were collected from the TCGA-BRCA cohort, GSE20685, and GSE96058 cohorts. STING pathway-related genes (SRGs) were obtained from the Reactome database. Differentially expressed genes were analyzed using the limma package. Immune cell infiltration was analyzed using the IOBR package. Gene Ontology biological processes, Kyoto Encyclopedia of Genes and Genomes pathways, and cancer hallmark pathways were analyzed using the MSigDB database. Prognostic models were prepared using the least absolute shrinkage and selection operator and multiple-factor Cox regression analysis. Single-cell analysis was performed using the Seurat and SCP pipeline. Results The expression patterns and clinical relevance of SRGs were analyzed in patients with BRCA. Transcriptional differences in the SRGs were observed between normal and tumorous tissues, with global down-regulated STING1 and up-regulated TBK1 in BRCA tissue. Tumor tissues were classified through consensus clustering analysis into two distinct groups, with differences in clinical characteristics and immune infiltration. A prognostic model related to the differences in STING pathway activity—high prognostic stratification potency—was developed and validated. Correlation analysis revealed suppressed overall immune activation in patients with BRCA having higher risk scores. Gemcitabine had a more favorable outcome in the low-risk group. The activity of the prognostic model at the single-cell level was confirmed through single-cell analysis, particularly in CD8 T cells and intratumor natural killer cells. Conclusion A STING pathway-related prognostic model developed and validated and the model could accurately predict BRCA patient outcomes. These findings have important implications for the personalized treatment and management of patients with BRCA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dda048ba9b29653abf25b1272c86792b3a07e71" target='_blank'>
              Investigating the impact of STING pathway activation on breast cancer treatment outcomes: development and validation of a prognostic model
              </a>
            </td>
          <td>
            Yangyan Zhong, Hong Cao, Wei Li, Jian Deng, Dan Li, Junjie Deng
          </td>
          <td>2024-08-09</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Combined CDK4/6 inhibitor (CDK4/6i) and endocrine therapy significantly improves outcome for patients with estrogen receptor-positive (ER+) metastatic breast cancer, but drug resistance and thus disease progression inevitably occur. Herein, we aimed to identify genomic alterations associated with combined CDK4/6i and endocrine therapy resistance, and follow the levels of specific mutations in longitudinal circulating tumor DNA (ctDNA) for early detection of progression. From a cohort of 86 patients with ER+ metastatic breast cancer we performed whole exome sequencing or targeted sequencing of paired tumor (N = 8) or blood samples (N = 5) obtained before initiation of combined CDK4/6i and endocrine therapy and at disease progression. Mutations in oncogenic genes at progression were rare, while amplifications of growth-regulating genes were more frequent. The most frequently acquired alterations observed were PIK3CA and TP53 mutations and PDK1 amplification. Longitudinal ctDNA dynamics of mutant PIK3CA or private mutations revealed increased mutation levels at progression in 8 of 10 patients (80%). Impressively, rising levels of PIK3CA-mutated ctDNA were detected 4-17 months before imaging. Our data add to the growing evidence supporting longitudinal ctDNA analysis for real-time monitoring of CDK4/6i response and early detection of progression in advanced breast cancer. Further, our analysis suggests that amplification of growth-related genes may contribute to combined CDK4/6i and endocrine therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4f8f900b692158e74287adf0a693a5a4d08296" target='_blank'>
              Genomic alterations associated with resistance and circulating tumor DNA dynamics for early detection of progression on CDK4/6 inhibitor in advanced breast cancer.
              </a>
            </td>
          <td>
            Charlotte K Kindt, Carla L. Alves, S. Ehmsen, Amalie Kragh, T. Reinert, Marianne Vogsen, A. R. Kodahl, Jeanette D Rønlev, Dilan Ardik, Anna L Sørensen, Kirstine Evald, Mia L Clemmensen, J. Staaf, Henrik J Ditzel
          </td>
          <td>2024-08-11</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="While T cell lymphomas are classified as mature neoplasms, emerging evidence indicates that malignant transformation may occur at an earlier stage of T cell maturation. In this study, we determined clonal architectures in a broad range of T cell lymphomas. Our multidimensional profiling indicates that a large part of these lymphomas in fact emerge from an immature lymphoid T cell precursor at a maturation stage prior to V(D)J rearrangement that undergoes branching evolution. Consequently, at single cell resolution we observed considerable clonal tiding under selective therapeutic pressure. T cell receptor next-generation sequencing suggested a highly biased usage of TRBV20-1 gene segments as part of multiple antigen receptor rearrangements per patient. The predominance of TRBV20-1 was found across all major T cell lymphoma subtypes analyzed. This suggested that this particular V gene - independently of complementarity-determining region 3 (CDR3) configuration - may represent a driver of malignant transformation. Together, our data indicate that T cell lymphomas derive from immature lymphoid precursors and display considerable intratumoral heterogeneity that may provide the basis for relapse and resistance in these hard-to-treat cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5cc9e8d39a7b3561d529df56bcd87a167e0b44e" target='_blank'>
              T-cell receptor architecture and clonal tiding provide insight into the transformation trajectory of peripheral T-cell lymphomas.
              </a>
            </td>
          <td>
            E. Willscher, C. Schultheiss, L. Paschold, Franziska Lea Schümann, Paul Schmidt-Barbo, Benjamin Thiele, M. Bauer, Claudia Wickenhauser, Thomas Weber, Mascha Binder
          </td>
          <td>2024-08-29</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f217995bbe1b34f9aeac0d831228d08545af9e" target='_blank'>
              Decoding single-cell molecular mechanisms in astrocyte-to-iN reprogramming via Ngn2- and Pax6-mediated direct lineage switching
              </a>
            </td>
          <td>
            Rongxing Qin, Yingdan Zhang, Yue Yang, Jiafeng Chen, Lijuan Huang, Wei Xu, Qingchun Qin, Xiaojun Liang, Xinyu Lai, Xiaoying Huang, Minshan Xie, Li Chen
          </td>
          <td>2024-07-27</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Translocation renal cell carcinoma (tRCC) is an aggressive subtype of kidney cancer driven by TFE3 gene fusions, which act via poorly characterized downstream mechanisms. Here we report that TFE3 fusions transcriptionally rewire tRCCs toward oxidative phosphorylation (OXPHOS), contrasting with the highly glycolytic metabolism of most other renal cancers. This TFE3 fusion-driven OXPHOS program, together with heightened glutathione levels found in renal cancers, renders tRCCs sensitive to reductive stress – a metabolic stress state induced by an imbalance of reducing equivalents. Genome-scale CRISPR screening identifies tRCC-selective vulnerabilities linked to this metabolic state, including EGLN1, which hydroxylates HIF-1α and targets it for proteolysis. Inhibition of EGLN1 compromises tRCC cell growth by stabilizing HIF-1a and promoting metabolic reprogramming away from OXPHOS, thus representing a vulnerability to OXPHOS-dependent tRCC cells. Our study defines a distinctive tRCC-essential metabolic program driven by TFE3 fusions and nominates EGLN1 inhibition as a therapeutic strategy to counteract fusion-induced metabolic rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2568e01eb416e6b7b22ba26a8e1971911ede6d44" target='_blank'>
              TFE3 fusions direct an oncogenic transcriptional program that drives OXPHOS and unveils vulnerabilities in translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Jiao Li, Kaimeng Huang, Fiona McBride, Ananthan Sadagopan, Daniel S. Gallant, Meha Thakur, Prateek Khanna, Bingchen Li, Maolin Ge, Cary N. Weiss, M. Achom, Qingru Xu, Kun Huang, Birgitta A. Ryback, Miao Gui, L. Bar-Peled, S. Viswanathan
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Acute lymphoblastic leukemia (ALL) is an aggressive hematological malignancy that affects thousands of pediatric and adult patients each year. A major hurdle to developing more effective treatments is a limited understanding of the genes and pathways that drive ALL malignancy, and which are required for continued tumor growth. ALL is often diagnosed with the onset of advanced symptoms, making it difficult to study the early drivers of oncogenic initiation. Identifying novel genetic interactions and pathways capable of driving ALL will help to develop new diagnostic criteria and identify actionable drug targets for clinical treatments. In addition, genetic interactions between co-expressed oncogenes and their impact on initiation, progression, and therapy resistance is difficult to study and requires the costly use of genetically engineered animal models, usually mice. Here, we report a novel screening method using transgenic zebrafish to unbiasedly identify collaborating oncogenic drivers that when co-expressed lead to T- and B-ALL. We screened a transgene pool of 68 putative oncogenes identified from human ALL for synergies in driving the initiation of leukemia using a large-scale F0 transgenic gain-of-function screen in zebrafish. We recovered zebrafish with tumors that were predominantly T- and B-ALL, but also identified several fish with thymic T cell lymphomas, muscle sarcomas, and liposarcomas, demonstrating the robust capability of our approach to discover novel oncogenic collaborations. Comparative transcriptome analysis using bulk RNA sequencing between zebrafish tumors (>150 tumors) revealed candidate transgene combinations that likely act as drivers of tumor formation in T- and B-ALL. When microinjected for validation studies, two transgene combinations: 1) rag2:SET + rag2:intracellular notch1a and 2) rag2:SET + rag2:mutationally-activated IL7R lead to robust ALL formation in zebrafish, validating the oncogenic role of these factors in leukemia initiation. SET is a multifunctional protein involved in transcriptional regulation, histone binding, and cell death. While SET has been well characterized in other cancers, we found that SET is capable of collaborating with known drivers to initiate T-ALL in vivo and highly expressed in a vast majority of human T-ALL. Our SET models also revealed an enrichment of key pathways that drive proliferation in both zebrafish and human disease. We are currently performing molecular characterization of SET in human ALL cell lines to investigate SET driven ALL growth and maintenance. In addition, our panel of unique zebrafish tumors will allow help establish new models of hematologic malignancies, launch future investigations into leukemia pathology, and identify novel targets of therapy resistance.
 Citation Format: James R. Allen, Luis Antonio Corchete Sanchez, Mohamed N. Bakr, Miriam Fernandez-Lajarin, Alexandra Hazelwood, Nathan Ford, Alexandra Veloso, Victoriano Mulero, Maria L. Cayuela, Esther Rheinbay, David M. Langenau. A transgenic zebrafish screen identifies new collaborating oncogenic drivers in acute lymphoblastic leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/091a253fa76c63ad6deaa84d3f6f708b6e59ab16" target='_blank'>
              Abstract A002 A transgenic zebrafish screen identifies new collaborating oncogenic drivers in acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            J. R. Allen, Luis Antonio Corchete Sanchez, Mohamed N. Bakr, Miriam Fernández-Lajarín, Alexandra Hazelwood, Nathan Ford, Alexandra Veloso, Victoriano Mulero, M. Cayuela, Esther Rheinbay, D. Langenau
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="A fundamental challenge in biomedicine is understanding the mechanisms predisposing individuals to disease. While previous research has suggested that switch-like gene expression is crucial in driving biological variation and disease susceptibility, a systematic analysis across multiple tissues is still lacking. By analyzing transcriptomes from 943 individuals across 27 tissues, we identified 1,013 switch-like genes. We found that only 31 (3.1%) of these genes exhibit switch-like behavior across all tissues. These universally switch-like genes appear to be genetically driven, with large exonic genomic structural variants explaining five (∼18%) of them. The remaining switch-like genes exhibit tissue-specific expression patterns. Notably, tissue-specific switch-like genes tend to be switched on or off in unison within individuals, likely under the influence of tissue-specific master regulators, including hormonal signals. Among our most significant findings, we identified hundreds of concordantly switched-off genes in the stomach and vagina that are linked to gastric cancer (41-fold, p<10-4) and vaginal atrophy (44-fold, p<10-4), respectively. Experimental analysis of vaginal tissues revealed that low systemic levels of estrogen lead to a significant reduction in both the epithelial thickness and the expression of the switch-like gene ALOX12. We propose a model wherein the switching off of driver genes in basal and parabasal epithelium suppresses cell proliferation therein, leading to epithelial thinning and, therefore, vaginal atrophy. Our findings underscore the significant biomedical implications of switch-like gene expression and lay the groundwork for potential diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d158a86f2a151fa830972300750ba84c1a11dbfc" target='_blank'>
              Switch-like Gene Expression Modulates Disease Susceptibility
              </a>
            </td>
          <td>
            Alber Aqil, Yanyan Li, Zhiliang Wang, Saiful Islam, Madison Russell, T. Kunovac Kallak, Marie Saitou, Omer Gokcumen, N. Masuda
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Simple Summary Neuregulin-1 (NRG1) is an important regulator of ErbB-mediated pathways involved in cancer development. Recently, there have been several studies analyzing NRG-1 gene fusions engaged in altering the dimerization of HER family proteins and the consecutive results of their activation in different types of cancer. Non-small cell lung cancer (NSCLC) patients can benefit from pan-HER inhibitors, and knowledge of NRG-1 fusions can help tailor the treatment to a specific group of patients. New drugs targeting cells with NRG-1 fusions are under clinical trials and show effectiveness in NSCLC treatment. Abstract Non-small cell lung cancer (NSCLC) presents a variety of druggable genetic alterations that revolutionized the treatment approaches. However, identifying new alterations may broaden the group of patients benefitting from such novel treatment options. Recently, the interest focused on the neuregulin-1 gene (NRG1), whose fusions may have become a potential predictive factor. To date, the occurrence of NRG1 fusions has been considered a negative prognostic marker in NSCLC treatment; however, many premises remain behind the targetability of signaling pathways affected by the NRG1 gene. The role of NRG1 fusions in ErbB-mediated cell proliferation especially seems to be considered as a main target of treatment. Hence, NSCLC patients harboring NRG1 fusions may benefit from targeted therapies such as pan-HER family inhibitors, which have shown efficacy in previous studies in various cancers, and anti-HER monoclonal antibodies. Considering the increased interest in the NRG1 gene as a potential clinical target, in the following review, we highlight its biology, as well as the potential clinical implications that were evaluated in clinics or remained under consideration in clinical trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302eaed4f1e7bdac2f2c9a0f86b9a6908f0e39da" target='_blank'>
              NRG1 Gene Fusions—What Promise Remains Behind These Rare Genetic Alterations? A Comprehensive Review of Biology, Diagnostic Approaches, and Clinical Implications
              </a>
            </td>
          <td>
            Tomasz Kucharczyk, M. Nicoś, Marek Kucharczyk, E. Kalinka
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a848be54c4ea82e1661c7913223b39ce6de8b05" target='_blank'>
              Small RNA activation of CDH13 expression overcome BCR-ABL1-independent imatinib-resistance and their signaling pathway studies in chronic myeloid leukemia
              </a>
            </td>
          <td>
            Rui Su, Ziqi Wen, Xingri Zhan, Yiling Long, Xiuyuan Wang, Chuting Li, Yubin Su, Jia Fei
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
High-grade complex karyotype sarcomas are a heterogeneous group of tumors with a uniformly poor prognosis. Within complex karyotype sarcomas, there are innumerable genetic changes but identifying those that are clinically relevant has been challenging.


EXPERIMENTAL DESIGN
To address this, we utilized a pooled genetic screening approach, informed by TCGA data, to identify key drivers and modifiers of sarcoma development that were validated in vivo.


RESULTS
YAP1 and wildtype KRAS were validated as drivers and transformed human mesenchymal stem cells into two distinct sarcoma subtypes, undifferentiated pleomorphic sarcoma (UPS) and myxofibrosarcoma (MFS), respectively. A subset of tumors driven by CDK4 and PIK3CA reflected leiomyosarcoma (LMS) and osteosarcoma (OS) demonstrating the plasticity of this approach and the potential to investigate sarcoma subtype heterogeneity. All generated tumors histologically reflected human sarcomas and had increased aneuploidy as compared to simple karyotype sarcomas. Comparing differential gene expression of TCGA samples to model data identified increased oxidative phosphorylation signaling in YAP1 tumors. Treatment of a panel of soft tissue sarcomas with a combination of YAP1 and oxidative phosphorylation inhibitors led to significantly decreased viability.


CONCLUSIONS
Transcriptional co-analysis of TCGA patient samples to YAP1 and KRAS model tumors support that these sarcoma subtypes lie along a spectrum of disease and adds guidance for further transcriptome-based refinement of sarcoma subtyping. This approach can be used to begin to understand pathways and mechanisms driving human sarcoma development, the relationship between sarcoma subtypes and to identify and validate new therapeutic vulnerabilities for this aggressive and heterogeneous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdaada75e0a2b742920f7cff889cd677616ee1d5" target='_blank'>
              Genetic Screen in a Pre-Clinical Model of Sarcoma Development Defines Drivers and Therapeutic Vulnerabilities.
              </a>
            </td>
          <td>
            Jack Freeland, Maria Muñoz, Edmond O'Donnell, J. Langerman, Morgan Darrow, Jessica Bergonio, J. Suarez-Navarro, Steven Thorpe, Robert Canter, R. L. Randall, Kathrin Plath, Kermit L Carraway, Owen N. Witte, Thomas G. Graeber, Janai R. Carr-Ascher
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35393ed7857060735a43bccb7ec1795bc1e251a9" target='_blank'>
              Comprehensive molecular characterization of TFE3-rearranged renal cell carcinoma
              </a>
            </td>
          <td>
            Cho-Rong Lee, J. Suh, Dongjun Jang, Bo-Yeong Jin, Jaeso Cho, Moses Lee, Hyungtai Sim, Minyong Kang, Jueun Lee, Ju Hyun Park, Kyoung-Hwa Lee, Geum-Sook Hwang, Kyung Chul Moon, Cheryn Song, J. Ku, C. Kwak, Hyeon-Hoe Kim, Sung-Yup Cho, Murim Choi, Chang Wook Jeong
          </td>
          <td>2024-08-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Mycosis fungoides (MF) is the most common subtype of cutaneous T-cell lymphoma (CTCL). Comprehensive analysis of MF cells in situ and ex vivo is complicated by the fact that is challenging to distinguish malignant from reactive T cells with certainty. Methods To overcome this limitation, we performed combined single-cell RNA (scRNAseq) and T-cell receptor TCR sequencing (scTCRseq) of skin lesions of cutaneous MF lesions from 12 patients. A sufficient quantity of living T cells was obtained from 9 patients, but 2 had to be excluded due to unclear diagnoses (coexisting CLL or revision to a fixed toxic drug eruption). Results From the remaining patients we established single-cell mRNA expression profiles and the corresponding TCR repertoire of 18,630 T cells. TCR clonality unequivocally identified 13,592 malignant T cells. Reactive T cells of all patients clustered together, while malignant cells of each patient formed a unique cluster expressing genes typical of naive/memory, such as CD27, CCR7 and IL7R, or cytotoxic T cells, e.g., GZMA, NKG7 and GNLY. Genes encoding classic CTCL markers were not detected in all clusters, consistent with the fact that mRNA expression does not correlate linearly with protein expression. Nevertheless, we successfully pinpointed distinctive gene signatures differentiating reactive malignant from malignant T cells: keratins (KRT81, KRT86), galectins (LGALS1, LGALS3) and S100 genes (S100A4, S100A6) being overexpressed in malignant cells. Conclusions Combined scRNAseq and scTCRseq not only allows unambiguous identification of MF cells, but also revealed marked heterogeneity between and within patients with unexpected functional phenotypes. While the correlation between mRNA and protein abundance was limited with respect to established MF markers, we were able to identify a single-cell gene expression signature that distinguishes malignant from reactive T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/212bc68159cf864c0e23d5af31516d6565474ee3" target='_blank'>
              Single-cell RNA and T-cell receptor sequencing unveil mycosis fungoides heterogeneity and a possible gene signature
              </a>
            </td>
          <td>
            Nalini Srinivas, Lukas Peiffer, K. Horny, Kuan Cheok Lei, T. B. Buus, Linda Kubat, Meng Luo, Menghong Yin, Ivelina Spassova, A. Sucker, Farnoush Farahpour, Jan Kehrmann, Selma Ugurel, E. Livingstone, T. Gambichler, Niels Ødum, Jürgen C. Becker
          </td>
          <td>2024-08-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0fe672a884b6c86c84e083c9b2ab93ac24f65f" target='_blank'>
              Multiplex, single-cell CRISPRa screening for cell type specific regulatory elements
              </a>
            </td>
          <td>
            Florence M. Chardon, Troy A. McDiarmid, Nicholas F. Page, R. Daza, Beth K Martin, Silvia Domcke, Samuel G Regalado, Jean-Benoît Lalanne, Diego Calderon, Xiaoyi Li, L. Starita, Stephan J. Sanders, N. Ahituv, J. Shendure
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Genetic editing of the germline using CRISPR/Cas9 technology has made it possible to alter livestock traits, including the creation of resistance to viral diseases. However, virus adaptability could present a major obstacle in this effort. Recently, chickens resistant to avian leukosis virus subgroup J (ALV-J) were developed by deleting a single amino acid, W38, within the ALV-J receptor NHE1 using CRISPR/Cas9 genome editing. This resistance was confirmed both in vitro and in vivo. In vitro resistance of W38-/- chicken embryonic fibroblasts to all tested ALV-J strains was shown. To investigate the capacity of ALV-J for further adaptation, we used a retrovirus reporter-based assay to select adapted ALV-J variants. We assumed that adaptive mutations overcoming the cellular resistance would occur within the envelope protein. In accordance with this assumption, we isolated and sequenced numerous adapted virus variants and found within their envelope genes eight independent single nucleotide substitutions. To confirm the adaptive capacity of these substitutions, we introduced them into the original retrovirus reporter. All eight variants replicated effectively in W38-/- chicken embryonic fibroblasts in vitro while in vivo, W38-/- chickens were sensitive to tumor induction by two of the variants. Importantly, receptor alleles with more extensive modifications have remained resistant to the virus. These results demonstrate an important strategy in livestock genome engineering towards antivirus resistance and illustrate that cellular resistance induced by minor receptor modifications can be overcome by adapted virus variants. We conclude that more complex editing will be necessary to attain robust resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8a5c8f973522317dce2248fd8edd93f9386c51" target='_blank'>
              Rapid adaptive evolution of avian leukosis virus subgroup J in response to biotechnologically induced host resistance
              </a>
            </td>
          <td>
            M. Matoušková, Jiří Plachý, Dana Kučerová, Ľ. Pecnová, M. Reinišová, J. Geryk, Vít Karafiát, T. Hron, J. Hejnar
          </td>
          <td>2024-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive and lethal form of brain cancer with few effective treatments. In this context, Zika virus has emerged as a promising therapeutic agent due to its ability to selectively infect and kill GBM cells. To elucidate these mechanisms and expand the landscape of oncolytic virotherapy, we pursued a transcriptomic meta-analysis comparing the molecular signatures of Zika infection in GBM and neuroblastoma (NBM). Over-representation analysis of dysregulated coding genes showed significant enrichment of tumor necrosis factor (TNF), NF-κB, and p53 signaling pathways. A refined list of long non-coding RNAs consistently dysregulated in Zika-infected GBMs was also developed. Functional review of these candidates revealed their potential regulatory role in Zika-mediated oncolysis. We performed validation of the less-researched targets in adult and pediatric GBM cell lines and found significant differential regulation, as predicted. Altogether, our results provide novel insights into the molecular mechanisms underlying the effect of Zika on GBM. We highlight potential therapeutic targets that could be further interrogated to improve the efficacy of tumor cell death and the utility of Zika as an adjuvant virotherapy for GBM and other related cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5fb2f82f19cb82c54f2e078dc5fd330417c5a7" target='_blank'>
              Transcriptomic Meta-analysis Identifies Long Non-Coding RNAs Mediating Zika’s Oncolytic Impact in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Youssef A. Kousa, Shriya Singh, Allison Horvath, Federica Tomasso, Javad Nazarian, Lisa Henderson, Tamer A. Mansour
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Background Pediatric precision medicine programs iTHER (The Netherlands), INFORM (international), and ZERO (Australia), report actionable molecular drug targets in 70-86% of children with cancer. This paradigm-changing approach has led to clinical benefit in selected groups. However, no relevant molecular targets are identified in subsets of patients e.g., malignant rhabdoid tumors or ependymoma, and clinical impact for individual patients remains hard to predict. This is the rationale underpinning our aim to integrate functional approaches into therapeutic decision making. Here, we report on drug sensitivity screening integrated with matched molecular profiles from the global collaboration between iTHER, INFORM and ZERO. Methods Functional profiling was performed on patient-derived viable material from high-risk, relapsed or refractory pediatric tumors after short- or long-term culture, and/or in vivo expansion. Long-term cultured and in vivo expanded samples were authenticated using short tandem repeat analysis and validated using single nucleotide polymorphism array, immunohistochemistry and/or flow cytometry. Samples were exposed to clinically relevant drug libraries and drug efficacy parameters including half-maximal inhibitory concentration (IC50), area under the dose-response curve value (AUC), and drug sensitivity score (DSS) were obtained. Tumor molecular profiles were established using whole-genome sequencing or whole exome sequencing, RNA sequencing and/or methylation profiling. All data were integrated, after which clustering analysis and gene set enrichment analysis were used to identify drug sensitivity patterns. Results Drug sensitivity screening was performed for 270 (iTHER=71; INFORM=109; ZERO=90) solid tumors, 105 (iTHER=19 ; INFORM=45; ZERO=41) CNS tumors, and 23 (ZERO) hematological malignancies. Results were collected in the R2 platform (http://r2platform.com), which incorporates dedicated visualization and analysis tools. As a result, a powerful reference set is available, reflecting both known pharmacologic vulnerabilities, such as sensitivity of NTRK-fusion positive samples to NTRK inhibition, and novel vulnerabilities including sensitivity of PIK3R1 mutated brain tumors to MEK inhibition. Additionally, tumor-type specific drug sensitivities were discovered, including sensitivity to MEK inhibitors in Wilms tumors and high-grade and diffuse midline gliomas without Ras-MAPK pathway alterations. Moreover, in vitro non-responsiveness of heavily pre-treated samples to chemotherapy was confirmed, which potentially could avoid ineffective treatments. Clinical follow-up of a subset of patients confirmed correlation with in vitro drug sensitivity. Conclusions Our data support complementary functional profiling to omics-guided precision medicine by strengthening molecular results; identifying new treatment options; and avoiding ineffective treatments. Global collaboration and data sharing is essential as childhood cancer remains rare, and innovative approaches are urgently needed to improve outcome for future patients.
 Citation Format: Eleonora J. Looze, Jie Mao, Heike Peterziel, Arjan Boltjes, Bianca Koopmans, Jan Koster, Marcel Kool, Max M. Van Noesel, Olaf Witt, Jan J. Molenaar, Ina Oehme, M. Emmy M. Dolman, Karin P.S. Langenberg. Next generation pediatric precision oncology: Functional profiling of patient-derived viable tumor material to link genotype and phenotype [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr PR004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97a32fb89ef5b994398005d64b51559a4eec1006" target='_blank'>
              Abstract PR004: Next generation pediatric precision oncology: Functional profiling of patient-derived viable tumor material to link genotype and phenotype
              </a>
            </td>
          <td>
            Eleonora J. Looze, Jie Mao, H. Peterziel, A. Boltjes, B. Koopmans, Jan Koster, Marcel Kool, Max M. van Noesel, Olaf Witt, J. Molenaar, Ina Oehme, M. Dolman, Karin P S Langenberg
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="High risk - neuroblastoma (HR-NB) is a pediatric solid tumor with high lethality. Half of HR-NB are driven by MYCN gene amplification (MNA). These HR-NBs require high dosage chemotherapy and often relapse. Moreover, current therapies can cause severe long-term side effects and new therapies are urgently needed. This study investigates a novel therapeutic approach targeting the metabolic vulnerabilities of MNA NB cells. We discovered that Diphenyleneiodonium chloride (DPI), an inhibitor of flavoprotein enzymes and mitochondrial complex I, synergizes with mitoquinone mesylate (MitoQ), a mitochondria-targeted antioxidant in 2D and 3D in vitro models of NB. Similarly to DPI, MitoQ affects MNA cells in a MYCN-dependent way, being more toxic when MYCN levels are high. Furthermore, low nanomolar concentrations of MitoQ significantly decrease MYCN protein expression and induce differentiation of MNA cells. The DPI and MitoQ combination further synergizes with vincristine, a chemotherapeutic agent used in NB treatment. Phosphoproteomics and proteomics analysis suggests that the drug combination induces MNA NB cell death by arresting the cell cycle and inhibiting oxidative phosphorylation (OXPHOS) in the mitochondria. Thus, interference with mitochondrial metabolism may represent an effective strategy to enhance the activity of chemotherapeutic drugs in MNA-NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c84bd27bba09b6f802fb233fccdb2001e1425eb1" target='_blank'>
              OXPHOS TARGETING OF MYCN-AMPLIFIED NEUROBLASTOMA
              </a>
            </td>
          <td>
            Soraya Epp, Donagh Egan, E. Poon, Amirah Adlina Abdul Aziz, Kieran Wynne, Melinda Halasz, Walter Kolch
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background: Comprehensive genomic profiling (CGP) identifies genetic alterations and patterns that are crucial for therapy selection and precise treatment development. In Colombia, limited access to CGP tests underscores the necessity of documenting the prevalence of treatable genetic alterations. This study aimed to describe the somatic genetic profile of specific cancer types in Colombian patients and assess its impact on treatment selection. Methods: A retrospective cohort study was conducted at Clínica Colsanitas S.A. from March 2023 to June 2024. Sequencing was performed on the NextSeq2000 platform with the TruSight Oncology 500 (TSO500) assay, which simultaneously evaluates 523 genes for DNA analysis and 55 for RNA; additionally, analyses were performed with the SOPHiA DDM software. The tumor mutational burden (TMB), microsatellite instability (MSI), and programmed cell death ligand 1 (PDL1) were assessed. Results: Among 111 patients, 103 were evaluated, with gastrointestinal (27.93%), respiratory (13.51%), and central nervous system cancers (10.81%) being the most prevalent. TP53 (37%), KMT2C (28%), and KRAS (21%) were frequent mutations. Actionable findings were detected in 76.7% of cases, notably in digestive (20 patients) and lung cancers (8 patients). MSI was stable at 82.52% and high at 2.91%, whilst TMB was predominantly low (91.26%). Conclusions: The test has facilitated access to targeted therapies, improving clinical outcomes in Colombian patients. This profiling test is expected to increase opportunities for personalized medicine in Colombia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93aa6bc1a8a2f18dd281151deeb80b4be0a23d78" target='_blank'>
              Advancing Cancer Care in Colombia: Results of the First In Situ Implementation of Comprehensive Genomic Profiling
              </a>
            </td>
          <td>
            Juan Javier López Rivera, Paula Rueda-Gaitán, Laura Camila Rios Pinto, Diego Alejandro Rodríguez Gutiérrez, Natalia Gomez-Lopera, J. Lamilla, Fabio Andrés Rojas Aguirre, Laura Bernal Vaca, M. Isaza-Ruget
          </td>
          <td>2024-09-14</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Gastric cancer (GC) represents a major global health challenge as a highly prevalent disease with high mortality whose global incidence and mortality are predicted to worsen over the coming years. To date, our standard of care for advanced gastric cancer of combination chemotherapy and immunotherapy has a 1-year overall survival rate of 55%. Significant efforts have gone into identifying targetable alterations in gastric cancer, ultimately yielding the Fibroblast Growth Factor Receptors (FGFRs) family, specifically FGFR2 as a promising target. FGFR2 is overexpressed in GC, particularly diffuse-type GC, and is associated with poor prognostic outcomes. In recent years, there has been an increasing number of small molecule inhibitors and monoclonal antibodies targeting FGFR2 that have entered into clinical trials. Specifically for GC, these agents are currently being trialed in various phases as monotherapies or with standard-of-care treatments to make a clinically meaningful impact on what appears to be an important biological axis of GC. In this review, we outline the underlying biology of FGFR2, its putative role in GC, and the various FGFR2-targeted agents currently in clinical trials for gastric cancer patients as well as postulate some challenges in adopting these therapeutics for clinically meaningful benefit.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0e38261e8052f219b50e3aed4b759b6cc116998" target='_blank'>
              Deciphering the FGFR2 Code: Innovative Targets in Gastric Cancer Therapy
              </a>
            </td>
          <td>
            Alireza Tojjari, S. Nagdas, Ali Saeed, A. Saeed
          </td>
          <td>2024-07-29</td>
          <td>Current Oncology</td>
          <td>1</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00f59f40899be50a5ab608a38abd5cef15d3af87" target='_blank'>
              The ATOM-Seq sequence capture panel can accurately predict microsatellite instability status in formalin-fixed tumour samples, alongside routine gene mutation testing
              </a>
            </td>
          <td>
            Kanishta Srihar, A. Gusnanto, Susan D Richman, Nicholas P West, Leanne Galvin, D. Bottomley, G. Hemmings, Amy Glover, Subaashini Natarajan, Rebecca Miller, Sameira Arif, H. Rossington, Thomas L Dunwell, Simon C. Dailey, Gracielle Fontarum, Agnes George, Winnie Wu, Phil Quirke, H.M. Wood
          </td>
          <td>2024-09-19</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Non-small cell lung cancer (NSCLC) leads as a primary cause of cancer-related premature mortality in Western populations. This study leverages cutting-edge gene-expression-profiling technologies to perform an in-depth molecular characterization of NSCLC specimens, with the objective of uncovering tumor-specific genomic alterations. By employing DNA microarray analysis, our research aims to refine the classification of NSCLC for early detection, guide molecular-targeted treatment approaches, enhance prognostication, and broaden the scientific understanding of the disease’s biology. We identified widespread genomic abnormalities in our samples, including the recurrent loss of chromosomal regions 3p, 5q, 13q, and 21q and the gain of 12p. Furthermore, utilizing Metascape for bioinformatic analysis revealed critical biological pathways disrupted in NSCLC, offering promising leads for novel therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489eff9e89a558831da14bfa1d262e700e732bd4" target='_blank'>
              Specific Deletions of Chromosomes 3p, 5q, 13q, and 21q among Patients with G2 Grade of Non-Small Cell Lung Cancer
              </a>
            </td>
          <td>
            Agata Kolecka-Bednarczyk, M. Frydrychowicz, B. Budny, M. Ruciński, Claudia Dompe, Piotr Gabryel, B. Płachno, M. Ruchała, K. Ziemnicka, Paweł Zieliński, Joanna Budna-Tukan
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Precision oncology promises individually tailored drugs and clinical care for patients with cancer: That is, "the right drug, for the right patient, at the right dose, and at the right time." Although stratification of the risk for treatment resistance and toxicity is key to precision oncology, there are multiple ways in which such stratification can be achieved, for example, genetic, functional pathway based, among others. Moving toward precision oncology is sorely needed in the case of acute lymphoblastic leukemia (ALL) wherein adult patients display survival rates ranging from 30% to 70%. The present study reports on the pathway activity signature of adult B-ALL, with an eye to precision oncology. Transcriptome profiles from three different expression datasets, comprising 346 patients who were adolescents or adults with B-ALL, were harnessed to determine the activity of signaling pathways commonly disrupted in B-ALL. Pathway activity analyses revealed that Ph-like ALL closely resembles Ph-positive ALL. Although this was the case at the average pathway activity level, the pathway activity patterns in B-ALL differ from genetic subtypes. Importantly, clustering analysis revealed that five distinct clusters exist in B-ALL patients based on pathway activity, with each cluster displaying a unique pattern of pathway activation. Identifying pathway-based subtypes thus appears to be crucial, considering the inherent heterogeneity among patients with the same genetic subtype. In conclusion, a pathway-based stratification of the B-ALL could potentially allow for simultaneously targeting highly active pathways within each ALL subtype, and thus might open up new avenues of innovation for personalized/precision medicine in this cancer that continues to have poor prognosis in adult patients compared with the children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72b528a52f9a1c25407f3a5bc1baf86dc4c7b9e9" target='_blank'>
              Expanding Beyond Genetic Subtypes in B-Cell Acute Lymphoblastic Leukemia: A Pathway-Based Stratification of Patients for Precision Oncology.
              </a>
            </td>
          <td>
            Ozlem Ulucan
          </td>
          <td>2024-08-19</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The fundamental transcription factor p53 regulates cellular processes and integrates signals of cellular stress, triggering a coordinated response to ensure survival of cells restored to healthy function and programmed death of those that couldn’t be repaired. Unsurprisingly, this is one of the most mutated genes in human cancers, with most changes occurring in the DNA-binding domain of the protein. In this work, we take a genome-wide approach and use available resources to identify high confidence p53-target genes, that we examine in three breast cancer cell lines with different p53 status, wild type (MCF-7) and different mutations in the DNA-binding domain (MDA-MB231, T47D). Comparison of p53-targets expression in response to DNA damage by RNAseq and cellular assays reveals that MDA-MB231 have a severely impaired p53-dependent pathway functionality while T47D are much less affected. MDA-MB231 are more resistant to DNA damage yet unable to repair and able to override cell cycle arrest leading to survival while T47D are sensitive only to high dose and exposure to genotoxic agents. This data shows the variability of effects of different p53 mutations and highlight the importance of understanding the mechanisms of p53 in the context of genotoxicity-based treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d6785633c891564b2ca32ca3538063162d5c2bd" target='_blank'>
              Differential gene expression in cells with different p53 mutations identifies genome-wide p53 targets and shows distinct modulation of cellular pathways in response to DNA damage
              </a>
            </td>
          <td>
            Patricia Eror Barnes, Maria Jose de la Concha, Kioko Mwikali, B. L. Ng, Hannes Ponstingl, Alena Pance
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b7a1b60f27d8f207e63fcabad98fcf75b1aff6" target='_blank'>
              Deciphering the impact of genomic variation on function.
              </a>
            </td>
          <td>
            J. Engreitz, Heather A. Lawson, Harinder Singh, Lea M. Starita, Gary C. Hon, Hannah Carter, Nidhi Sahni, Timothy E. Reddy, Xihong Lin, Yun Li, Nikhil V. Munshi, Maria H. Chahrour, Alan P. Boyle, B. Hitz, Ali Mortazavi, Mark Craven, K. Mohlke, Luca Pinello, Ting Wang, A. Kundaje, Feng Yue, Sarah Cody, Nina P. Farrell, Michael I. Love, Lara A Muffley, M. Pazin, Fairlie Reese, Eric Van Buren, Kushal K. Dey, Martin Kircher, Jian Ma, P. Radivojac, Brunilda Balliu, Brian A. Williams, Danwei Huangfu, Chong Y. Park, Thomas Quertermous, Jishnu Das, M. Calderwood, Douglas M. Fowler, Marc Vidal, Lucas Ferreira, Sean D. Mooney, V. Pejaver, Jingjing Zhao, Steven Gazal, Evan Koch, Steven K. Reilly, S. Sunyaev, Anne E. Carpenter, Jason D. Buenrostro, Christina S. Leslie, Rachel E. Savage, Stefanija Giric, Chongyuan Luo, Kathrin Plath, Alejandro Barrera, Max Schubach, A. Gschwind, Jill E. Moore, N. Ahituv, S. Yi, Ingileif B. Hallgrímsdóttir, K. Gaulton, S. Sakaue, Sina Booeshaghi, Eugenio Mattei, Surag Nair, L. Pachter, Austin T. Wang, J. Shendure, Vikram Agarwal, Andrew Blair, Theofilos Chalkiadakis, Florence M. Chardon, P. Dash, Chengyu Deng, Nobuhiko Hamazaki, Pia Keukeleire, Connor A Kubo, Jean-Benoît Lalanne, Thorben Maass, Beth Martin, Troy A. McDiarmid, Mai Nobuhara, Nicholas F. Page, Sam Regalado, Jasmine Sims, A. Ushiki, Sabrina M. Best, Gabe Boyle, Nathan Camp, Silvia Casadei, Estelle Y. Da, Moez Dawood, Samantha C. Dawson, Shawn Fayer, Audrey Hamm, Richard G. James, Gail P. Jarvik, Abbye E. McEwen, Nick Moore, Sriram Pendyala, Nicholas A Popp, Mason Post, Alan F. Rubin, Nahum T. Smith, J. Stone, Malvika Tejura, Ziyu R. Wang, Melinda K. Wheelock, Ivan Woo, Brendan D. Zapp, Dulguun Amgalan, Aradhana Aradhana, Sophia M. Arana, M. Bassik, Julia R. Bauman, Asmita Bhattacharya, Xiangmeng S. Cai, Ziwei Chen, Stephanie Conley, Salil Deshpande, Benjamin R. Doughty, Peter P. Du, James A. Galante, Casey Gifford, W.J. Greenleaf, Katherine Guo, Revant Gupta, Sarasa Isobe, Evelyn Jagoda, Nimit Jain, Hank Jones, Helen Kang, Samuel H. Kim, YeEun Kim, Sandy Klemm, Ramen Kundu, Soumya Kundu, Mauro Lago-Docampo, Yannick C. Lee-Yow, Roni Levin-Konigsberg, Daniel Y. Li, Dominik Lindenhofer, X. R. Ma, Georgi K. Marinov, Gabriella E. Martyn, Chloe V. McCreery, Eyal Metzl-Raz, João P. Monteiro, Michael T. Montgomery, Kristy S. Mualim, Chad J. Munger, Glen Munson, Tri C. Nguyen, Tri C. Nguyen, Brian T. Palmisano, Anusri Pampari, Marlene Rabinovitch, Markus Ramste, Judhajeet Ray, Kevin R. Roy, Oriane M. Rubio, Julia M. Schaepe, Gavin Schnitzler, Jacob M Schreiber, Disha Sharma, Maya U. Sheth, Huitong Shi, Vasundhara Singh, Riya Sinha, Lars M Steinmetz, Jason Tan, Anthony Tan, Josh Tycko, Raeline Valbuena, Valeh Valiollah Pour Amiri, Mariëlle J. F. M. van Kooten, Alun Vaughan-Jackson, Anthony Venida, Chad S. Weldy, Matthew D. Worssam, Fan Xia, David Yao, Tony Zeng, Quanyi Zhao, Ronghao Zhou, Zitong Sam Chen, B. Cimini, Georges Coppin, Atina G. Coté, Marzieh Haghighi, Tong Hao, David E. Hill, Jessica Lacoste, Florent Laval, Chloe Reno, Fritz Roth, Shantanu Singh, Kerstin Spirohn-Fitzgerald, Mikko Taipale, Tanisha Teelucksingh, Maxime Tixhon, Anupama Yadav, Zhipeng Yang, W. L. Kraus, Daniel A. Armendariz, Ashley E. Dederich, Ashlesha Gogate, Lauretta El Hayek, S. Goetsch, Kiran Kaur, Hyung Bum Kim, Melissa K. McCoy, Mpathi Z. Nzima, C. Pinzon-Arteaga, Bruce A. Posner, Daniel A. Schmitz, Sushama Sivakumar, Anjana Sundarrajan, Lei Wang, Yihan Wang, Jun Wu, Lin Xu, Jian Xu, Leqian Yu, Yanfeng Zhang, Huan Zhao, Qinbo Zhou, Hyejung Won, Jessica L Bell, K. A. Broadaway, Katherine N. Degner, A. Etheridge, Beverly H. Koller, Won Mah, Wancen Mu, Kimberly D. Ritola, Jonathan D. Rosen, Sarah A. Schoenrock, Rachel A. Sharp, Daniel Bauer, G. Lettre, Richard Sherwood, Basheer Becerra, Logan J. Blaine, Eric Che, M. Francoeur, Ellie N. Gibbs, Nahye Kim, Emily M. King, B. Kleinstiver, Estelle Lecluze, Zhijian Li, Zain M. Patel, Quang Vinh Phan, Jayoung Ryu, Marlena L. Starr, Ting Wu, C. Gersbach, Gregory E. Crawford, Andrew S. Allen, William H. Majoros, Nahid Iglesias, Ruhi Rai, Revathy Venukuttan, Boxun Li, Taylor Anglen, Lexi R. Bounds, Marisa C. Hamilton, Siyan Liu, Sean R. McCutcheon, Christian McRoberts Amador, Samuel J. Reisman, Maria A. ter Weele, Josephine C. Bodle, Helen L Streff, Keith Siklenka, Kari Strouse, Bradley E. Bernstein, Juliana Babu, Guillermo Barreto Corona, Kevin Dong, Fabiana M. Duarte, Neva C. Durand, Charles B. Epstein, K. Fan, Elizabeth A. Gaskell, Amelia W. Hall, Alexandra M. Ham, Mei K. Knudson, N. Shoresh, Siddarth Wekhande, Cassandra M. White, Wang Xi, Ansuman T. Satpathy, M. Corces, Serena H. Chang, Iris M. Chin, James M Gardner, Zachary A. Gardell, Jacob C. Gutierrez, Alia W. Johnson, Lucas Kampman, Maya Kasowski, Caleb A. Lareau, Vincent Liu, Leif S. Ludwig, Christopher S. McGinnis, Shreya Menon, Anita Qualls, Katalin D. Sandor, Adam W. Turner, C. J. Ye, Yajie Yin, Wenxi Zhang, Barbara J. Wold, Maria Carilli, Dayeon Cheong, Ghassan Filibam, Kim Green, S. Kawauchi, Charlene Kim, Heidi Liang, Rebekah Loving, L. Luebbert, Grant MacGregor, Angel G. Merchan, Elisabeth Rebboah, Narges Rezaie, Jasmine Sakr, Delaney K. Sullivan, Nikki Swarna, Diane Trout, S. Upchurch, Ryan Weber, Christopher P. Castro, Elysia Chou, Fan Feng, Andre Guerra, Yuanhao Huang, Linghua Jiang, Jie Liu, Ryan E. Mills, Weizhou Qian, Tingting Qin, Maureen A. Sartor, Rintsen N. Sherpa, Jinhao Wang, Yiqun Wang, Joshua D. Welch, Zhenhao Zhang, Nanxiang Zhao, Sayan Mukherjee, C. D. Page, Shannon Clarke, Richard W. Doty, Yuncheng Duan, Raluca Gordân, Kuei-Yueh Ko, Shengyu Li, Boyao Li, Alexander Thomson, Soumya Raychaudhuri, Alkes Price, Thahmina A. Ali, Arun Durvasula, M. Kellis, L. Iakoucheva, Tulika Kakati, Yile Chen, Mariam Benazouz, Shantanu Jain, Daniel Zeiberg, M. C. De Paolis Kaluza, Michelle Velyunskiy, Audrey Gasch, Kunling Huang, Yiyang Jin, Qiongshi Lu, Jiacheng Miao, Michael Ohtake, Eduardo Scopel, Robert D. Steiner, Yuriy Sverchkov, Zhiping Weng, Manuel Garber, Yu Fu, Natalie Haas, Xihao Li, Nishigandha Phalke, Shuo C. Shan, Nicole Shedd, Tianxiong Yu, Yi Zhang, Hufeng Zhou, Alexis Battle, Livnat Jerby, Eran Kotler, Andrew R. Marderstein, Stephen B. Montgomery, AkshatKumar Nigam, Evin M. Padhi, Aman Patel, Jonathan Pritchard, Ivy Raine, Vivekanandan Ramalingam, Kameron B. Rodrigues, Jacob M Schreiber, Arpita Singhal, Marisa Abundis, Deepa Bisht, Trirupa Chakraborty, Jingyu Fan, David R. Hall, Zarifeh H. Rarani, Abhinav K. Jain, Babita Kaundal, Swapnil Keshari, Daniel McGrail, Nicholas A. Pease, Vivian F. Yi, Hao Wu, Sreeram Kannan, Hongjun Song, Jingli Cai, Ziyue Gao, Ronni Kurzion, Julia I. Leu, Fan Li, Dongming Liang, Guo-Li Ming, K. Musunuru, Qi Qiu, Junwei Shi, Yijing Su, Sarah Tishkoff, Ning Xie, Qian Yang, Wenli Yang, Hongjie Zhang, Zhijian Zhang, Michael A Beer, A. Hadjantonakis, Sharon Adeniyi, Hyein Cho, Ronald Cutler, Rachel A. Glenn, David Godovich, Nan Hu, Svetlana Jovanic, Renhe Luo, Jin Woo Oh, Milad Razavi-Mohseni, Dustin Shigaki, Simone Sidoli, Thomas Vierbuchen, Xianming Wang, Breanna Williams, Jielin Yan, Dapeng Yang, Yunxiao Yang, Maike Sander, Bing Ren, W. Bartosik, Hannah S Indralingam, A. Klie, Hannah M. Mummey, Mei-Lin Okino, Gaowei Wang, Nathan R. Zemke, Kai Zhang, Han Zhu, Noah Zaitlen, Jason Ernst, J. Langerman, Terence Li, Yu Sun, Alexander Y. Rudensky, P. Periyakoil, Vianne R. Gao, Melanie H. Smith, Norman M. Thomas, L. Donlin, Amit Lakhanpal, Kaden M. Southard, Rico C. Ardy, J. M. Cherry, Mark B. Gerstein, Kalina Andreeva, Pedro Assis, Beatrice Borsari, Eric Douglass, Shengcheng Dong, I. Gabdank, Keenan Graham, Otto Jolanki, Jennifer Jou, Meenakshi S. Kagda
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>1</td>
          <td>109</td>
        </tr>

        <tr id="Abstract Small-cell lung cancer (SCLC) is the most lethal type of lung cancer. Paradoxically, this tumor displays an initial exquisite response to chemotherapy; however, at relapse, the tumor is highly resistant to subsequent available therapies. Here, we report that the expression of three prime repair exonuclease 1 (TREX1) is strongly induced in chemoresistant SCLCs. Assay for transposase-accessible chromatin using sequencing and chromatin immunoprecipitation sequencing revealed a significant increase in chromatin accessibility and transcriptional activity of TREX1 gene locus in chemoresistant SCLCs. Analyses of human SCLC tumors and patient-derived xenografts (PDX) also showed an increase in TREX1 expression in postchemotherapy samples. TREX1 depletion caused the activation of cyclic GMP-AMP synthase stimulator of interferon gene pathway due to cytoplasmic accumulation of damage-associated double-stranded DNA, inducing immunogenicity and enhancing the sensitivity of drug-resistant cells to chemotherapy. These findings suggest TREX1 upregulation may partially contribute to the survival of resistant cells, and its inhibition may represent a promising therapeutic strategy to enhance antitumor immunity and potentiate the efficacy of chemotherapy and/or immunotherapy in chemoresistant SCLCs. Significance: In this study, we show that targeting TREX1 induces an innate immune response and resensitizes SCLC cells to chemotherapy, representing a promising novel target for “immunologically” cold tumors, such as SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920774aa4db00913c78963d621212ab8e7a2e837" target='_blank'>
              Targeting TREX1 Induces Innate Immune Response in Drug-Resistant Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Takahiko Murayama, N. Mahadevan, Catherine B Meador, Elena V. Ivanova, Yuqiao Pan, Erik H. Knelson, Tetsuo Tani, Jun Nakayama, Xueying Ma, Tran C Thai, Yin P Hung, William Kim, Hideo Watanabe, Kathy Q Cai, Aaron N Hata, C. Paweletz, D. Barbie, Israel Cañadas
          </td>
          <td>2024-08-23</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ce3d50fc0ab92951015dbededde57814fc28edd" target='_blank'>
              Systemic lupus erythematosus genetics: insights into pathogenesis and implications for therapy.
              </a>
            </td>
          <td>
            Y. Ghodke-Puranik, M. Olferiev, Mary K. Crow
          </td>
          <td>2024-09-04</td>
          <td>Nature reviews. Rheumatology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D'Amato, Yi Xue, Kristina Fontanez, Aaron A May-Zhang, Trinity Smithers, Yigal Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract Strict control over hematopoietic stem cell decision making is essential for healthy life‐long blood production and underpins the origins of hematopoietic diseases. Acute myeloid leukemia (AML) in particular is a devastating hematopoietic malignancy that arises from the clonal evolution of disease‐initiating primitive cells which acquire compounding genetic changes over time and culminate in the generation of leukemic stem cells (LSCs). Understanding the molecular underpinnings of these driver cells throughout their development will be instrumental in the interception of leukemia, the enabling of effective treatment of pre‐leukemic conditions, as well as the development of strategies to target frank AML disease. To this point, a number of precancerous myeloid disorders and age‐related alterations are proving as instructive models to gain insights into the initiation of LSCs. Here, we explore this myeloid dysregulation at the level of post–transcriptional control, where RNA‐binding proteins (RBPs) function as core effectors. Through regulating the interplay of a myriad of RNA metabolic processes, RBPs orchestrate transcript fates to govern gene expression in health and disease. We describe the expanding appreciation of the role of RBPs and their post–transcriptional networks in sustaining healthy hematopoiesis and their dysregulation in the pathogenesis of clonal myeloid disorders and AML, with a particular emphasis on findings described in human stem cells. Lastly, we discuss key breakthroughs that highlight RBPs and post–transcriptional control as actionable targets for precision therapy of AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c6dddad312b3774cf0fac0198aa7e076e582021" target='_blank'>
              RNA binding protein‐directed control of leukemic stem cell evolution and function
              </a>
            </td>
          <td>
            Pratik Joshi, Ava Keyvani Chahi, Lina Liu, Steven Moreira, Ana Vujovic, Kristin J Hope
          </td>
          <td>2024-08-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Tyrosine kinase inhibitors (TKIs) have revolutionized the treatment landscape for chronic myeloid leukemia (CML). However, TKI resistance poses a significant challenge, leading to treatment failure and disease progression. Resistance mechanisms include both BCR::ABL1-dependent and BCR::ABL1-independent pathways. The mechanisms underlying BCR::ABL1 independence remain incompletely understood, with CML cells potentially activating alternative signaling pathways, including the AKT/mTOR and JAK2/STAT5 pathways, to compensate for the loss of BCR::ABL1 kinase activity. This study explored tumoral VISTA (encoded by VSIR) as a contributing factor to TKI resistance in CML patients and identified elevated tumoral VISTA levels as a marker of resistance and poor survival. Through in vitro and in vivo analyses, we demonstrated that VSIR knockdown and the application of NSC-622608, a novel VISTA inhibitor, significantly impeded CML cell proliferation and induced apoptosis by attenuating the AKT/mTOR and JAK2/STAT5 pathways, which are crucial for CML cell survival independent of BCR::ABL1 kinase activity. Moreover, VSIR overexpression promoted TKI resistance in CML cells. Importantly, the synergistic effect of NSC-622608 with TKIs offers a potent therapeutic avenue against both imatinib-sensitive and imatinib-resistant CML cells, including those harboring the challenging T315I mutation. Our findings highlight the role of tumoral VISTA in mediating TKI resistance in CML, suggesting that inhibition of VISTA, particularly in combination with TKIs, is an innovative approach to enhancing treatment outcomes in CML patients, irrespective of BCR::ABL1 mutation status. This study not only identified a new pathway contributing to TKI resistance but also revealed the possibility of targeting tumoral VISTA as a means of overcoming this significant clinical challenge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41cf4290358e1af5f36f27cd7397321411a29f7e" target='_blank'>
              Inhibition of Tumoral VISTA to Overcome TKI Resistance via Downregulation of the AKT/mTOR and JAK2/STAT5 Pathways in Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            Kexin Ai, Mu Chen, Zhao Liang, Xiangyang Ding, Yang Gao, Honghao Zhang, Suwan Wu, Yanjie He, Yuhua Li
          </td>
          <td>2024-08-06</td>
          <td>Biomolecules & Therapeutics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background Alternative and complementary therapies play an imperative role in the clinical management of Type 2 diabetes mellitus (T2DM), and exploring and utilizing natural products from a genetic perspective may yield novel insights into the mechanisms and interventions of the disorder. Methods To identify the therapeutic target of baicalin for T2DM, we conducted a Mendelian randomization study. Druggable targets of baicalin were obtained by integrating multiple databases, and target-associated cis-expression quantitative trait loci (cis-eQTL) originated from the eQTLGen consortium. Summary statistics for T2DM were derived from two independent genome-wide association studies available through the DIAGRAM Consortium (74,124 cases vs. 824,006 controls) and the FinnGen R9 repository (9,978 cases vs. 12,348 controls). Network construction and enrichment analysis were applied to the therapeutic targets of baicalin. Colocalization analysis was utilized to assess the potential for the therapeutic targets and T2DM to share causative genetic variations. Molecular docking was performed to validate the potency of baicalin. Single-cell RNA sequencing was employed to seek evidence of therapeutic targets’ involvement in islet function. Results Eight baicalin-related targets proved to be significant in the discovery and validation cohorts. Genetic evidence indicated the expression of ANPEP, BECN1, HNF1A, and ST6GAL1 increased the risk of T2DM, and the expression of PGF, RXRA, SREBF1, and USP7 decreased the risk of T2DM. In particular, SREBF1 has significant interaction properties with other therapeutic targets and is supported by strong colocalization. Baicalin had favorable combination activity with eight therapeutic targets. The expression patterns of the therapeutic targets were characterized in cellular clusters of pancreatic tissues that exhibited a pseudo-temporal dependence on islet cell formation and development. Conclusion This study identified eight potential targets of baicalin for treating T2DM from a genetic perspective, contributing an innovative analytical framework for the development of natural products. We have offered fresh insights into the connections between therapeutic targets and islet cells. Further, fundamental experiments and clinical research are warranted to delve deeper into the molecular mechanisms of T2DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee3b249fa81dffddaf199c1f12f9929a26ce0336" target='_blank'>
              Integrating Mendelian randomization and single-cell RNA sequencing to identify therapeutic targets of baicalin for type 2 diabetes mellitus
              </a>
            </td>
          <td>
            Ying-Chao Liang, Ling Li, Jia-Lin Liang, De-Liang Liu, Shu-Fang Chu, Hui-Lin Li
          </td>
          <td>2024-07-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e5e2db57a9a9a8ae1d5af55bcc4e63ea2e814f6" target='_blank'>
              The K9 lymphoma assay allows a genetic subgrouping of canine lymphomas with improved risk classification
              </a>
            </td>
          <td>
            A. Fanelli, L. Licenziato, Eugenio Mazzone, S. Divari, A. Rinaldi, Michele Marino, Ilaria Maga, Francesco Bertoni, L. Marconato, Luca Aresu
          </td>
          <td>2024-08-12</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Secondary acute myeloid leukemia (sAML) presents as a complex and multifaceted ensemble of disorders, positioning itself as both a challenge and an intriguing frontier within hematologic oncology. Its origins are diverse, stemming from antecedent hematologic conditions, germline predisposing mutations, or the sequelae of cytotoxic therapies, and its development is driven by intricate genetic and epigenetic modifications. This complexity necessitates a diverse array of therapeutic strategies, each meticulously tailored to address the distinctive challenges sAML introduces. Such strategies require a personalized approach, considering the variegated clinical backgrounds of patients and the inherent intricacies of the disease. Allogeneic stem cell transplantation stands as a cornerstone, offering the potential for curative outcomes. This is complemented by the emergence of innovative treatments such as CPX-351, venetoclax, and glasdegib, which have demonstrated promising results in enhancing prognosis. The evolving landscape of sAML treatment underscores the importance of continued research and innovation in the field, aiming not only to improve patient outcomes but also to deepen our understanding of the disease’s biological underpinnings, thereby illuminating pathways toward more effective and individualized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e411e905c73882bdcd7c22537e611482f6836fd2" target='_blank'>
              Novel insights and therapeutic approaches in secondary AML
              </a>
            </td>
          <td>
            Giovanni Marconi, Michela Rondoni, B. Zannetti, Irene Zacheo, Davide Nappi, Agnese Mattei, S. Rocchi, Francesco Lanza
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Simple Summary Lung cancer is a major cause of cancer-related deaths in developed nations. Factors such as unhealthy lifestyle choices, particularly smoking, contribute to the development of this disease. Epigenetic abnormalities greatly affect gene expression and disrupt important cellular signaling pathways that are responsible for the proper growth, regulation, and functioning of cells. In the case of lung cancer, specifically non-small cell lung cancer (NSCLC), CBX3 acts as an epigenetic oncoprotein, promoting the growth and progression of tumors. Numerous studies have shown that CBX3 is overexpressed in NSCLC and is associated with a poor prognosis. It interacts with key oncogenic pathways leading to increased proliferation, reduced apoptosis, and enhanced resistance to therapy. Further research on the mechanisms and functions of CBX3 holds the potential to reveal new insights into the development of the disease and uncover novel therapeutic opportunities. Abstract Epigenetic changes regulate gene expression through histone modifications, chromatin remodeling, and protein translation of these modifications. The PRC1 and PRC2 complexes shape gene repression via histone modifications. Specifically, the CBX protein family aids PRC1 recruitment to chromatin, impacting the progressive multistep process driving chromatin silencing. Among family members, CBX3 is a complex protein involved in aberrant epigenetic mechanisms that drive lung cancer progression. CBX3 promotes lung tumorigenesis by interacting with key pathways such as PI3K/AKT, Ras/KRAS, Wnt/β-catenin, MAPK, Notch, and p53, leading to increased proliferation, inhibition of apoptosis, and enhanced resistance to therapy. Given our current lack of knowledge, additional research is required to uncover the intricate mechanisms underlying CBX3 activity, as well as its involvement in molecular pathways and its potential biomarker evaluation. Specifically, the dissimilar roles of CBX3 could be reexamined to gain a greater insight into lung cancer pathogenesis. This review aims to provide a clear overview of the context-related molecular profile of CBX3, which could be useful for addressing clinical challenges and developing novel targeted therapies based on personalized medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1b8cecc66992d19bc95a90f28c7e4981384afb5" target='_blank'>
              Exploring the Role of CBX3 as a Potential Therapeutic Target in Lung Cancer
              </a>
            </td>
          <td>
            Muhammad Aamir Wahab, Nunzio Del Gaudio, Biagio Gargiulo, V. Quagliariello, N. Maurea, A. Nebbioso, L. Altucci, Mariarosaria Conte
          </td>
          <td>2024-08-30</td>
          <td>Cancers</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="
 At the Gene Editing Institute, we are advancing a novel strategy for the treatment of pancreatic ductile adenocarcinoma by employing CRISPR-directed gene editing to disable the functions of key genes that block effective drug application including chemotherapy, radiation and immunotherapy. No matter how careful one is at designing a specific drug against a specific protein, ultimately, the tumor cell develops a workaround. A paradigm shift is required to truly overcome this formidable barrier. We’ve accomplished this by leveraging the effectiveness of standard care and augmenting it, thereby reducing patient suffering, increasing treatment tolerance and, most importantly, improving outcomes. With an enthusiastic response for this concept from the FDA- INTERFACT meeting, we have demonstrated that a carefully selected CRISPR/Cas complex disables the NRF2 gene in the tumor cell and significantly reduces the dose of chemotherapy required to halt pancreatic tumor cell growth, and consequently promoting cell death. The efficiency of gene editing approaches 70 % routinely, which destroys the function of NRF2 and enables additional killing of residual tumor cells by chemotherapeutic agents. As a consequence of CRISPR/Cas activity, we effectively reduce the amount of chemotherapy needed to kill tumor cells by over 20-fold. This level of reduction should enable patients to complete their treatment regimens more successfully, at lower dosages and with far fewer adverse side effects. In addition to reduced suffering, this will invariably prolong survival as we are augmenting already proven treatments. This approach is agnostic to the haplotype of the tumor cell and does not rely on specific biomarker composition; radically expanding the patient population that would be suitable for this type of treatment. We will discuss the impact of CRISPR/Cas-gene editing to disable specific genes, including NRF2 and mutant KRAS, to enable current therapies to work at greater efficacy and at lower dosages.
 Citation Format: London P McGill, Kelly H Banas, Gregory Tiesi, Eric B Kmiec. A patient first approach using CRISPR-directed gene editing as an augmentative therapy for the treatment of pancreatic ductile adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/159a6092b196d29302afbdfaca37a6f4ece01dff" target='_blank'>
              Abstract A010: A patient first approach using CRISPR-directed gene editing as an augmentative therapy for the treatment of pancreatic ductile adenocarcinoma
              </a>
            </td>
          <td>
            L. McGill, Kelly Banas, Gregory Tiesi, Eric B Kmiec
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="R-loops are trimeric nucleic acid structures that form when an RNA molecule hybridizes with its complementary DNA strand, displacing the opposite strand. These structures regulate transcription as well as replication, but aberrant R-loops can form, leading to DNA breaks and genomic instability if unresolved. R-loop levels are elevated in many cancers as well as cells that maintain high-risk human papillomaviruses. We investigated how the distribution as well as function of R-loops changed between normal keratinocytes and HPV positive cells derived from a precancerous lesion of the cervix (CIN I). The levels of R-loops associated with cellular genes were found to be up to 10-fold higher in HPV positive cells than in normal keratinocytes while increases at ALU1 elements increased by up to 500-fold. The presence of enhanced R-loops resulted in altered levels of gene transcription, with equal numbers increased as decreased. While no uniform global effects on transcription due to the enhanced levels of R-loops were detected, genes in several pathways were coordinately increased or decreased in expression only in the HPV positive cells. This included the downregulation of genes in the innate immune pathway, such as DDX58, IL-6, STAT1, IFN-β, and NLRP3. All differentially expressed innate immune genes dependent on R-loops were also associated with H3K36me3 modified histones. Genes that were upregulated by the presence of R-loops in HPV positive cells included those in the DNA damage repair such as ATM, ATRX, and members of the Fanconi Anemia pathway. These genes exhibited a linkage between R-loops and H3K36me3 as well as γH2AX histone marks only in HPV positive cells. These studies identify a potential link in HPV positive cells between DNA damage repair as well as innate immune regulatory pathways with R-loops and γH2AX/H3K36me3 histone marks that may contribute to regulating important functions for HPV pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d30cc12d40eed32c558ede6628c4c8dfd335ab28" target='_blank'>
              HPV induced R-loop formation represses innate immune gene expression while activating DNA damage repair pathways
              </a>
            </td>
          <td>
            Conor W Templeton, Lou Laimins
          </td>
          <td>2024-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cb14f52e0abb7fcb6b84cd46a03fa1b6ced7483" target='_blank'>
              Genomic and molecular alterations associated with primary resistance to immune checkpoint inhibitors
              </a>
            </td>
          <td>
            Jyoti Malhotra, Subhajyoti De, Kim Nguyen, Percy Lee, Victoria Villaflor
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Due to the heterogeneity of cancer, precision medicine has been a major challenge for cancer treatment. Determining medication regimens based on patient genotypes has become a research hotspot in cancer genomics. In this study, we aim to identify key biomarkers for targeted therapies based on single nucleotide variants (SNVs) and copy number variants (CNVs) of genes. The experiment is carried out on 7 cancers on the Encyclopedia of Cancer Cell Lines (CCLE) dataset. Considering the high mutability of driver genes which result in abundant mutated samples, the effect of data sparsity can be eliminated to a large extent. Therefore, we focus on discovering the relationship between driver mutation patterns and three measures of drug response, namely area under the curve (AUC), half maximal effective concentration (EC50), and log2-fold change (LFC). First, multiple statistical methods are applied to assess the significance of difference in drug response between sample groups. Next, for each driver gene, we analyze the extent to which its mutations can affect drug response. Based on the results of multiple hypothesis tests and correlation analyses, our main findings include the validation of several known drug response biomarkers such as BRAF, NRAS, MAP2K1, MAP2K2, and CDKN2A, as well as genes with huge potential to infer drug responses. It is worth emphasizing that we identify a list of genes including SALL4, B2M, BAP1, CCDC6, ERBB4, FOXA1, GRIN2A, and PTPRT, whose impact on drug response spans multiple cancers and should be prioritized as key biomarkers for targeted therapies. Furthermore, based on the statistical p-values and correlation coefficients, we construct gene-drug sensitivity maps for cancer drug recommendation. In this work, we show that driver mutation patterns could be used to tailor therapeutics for precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa993499162c59f7b76c5c6ffab6d2e07409c25b" target='_blank'>
              Single nucleotide and copy number variants of cancer driver genes inform drug response in multiple cancers
              </a>
            </td>
          <td>
            Zeyuan Wang, Hong Gu, Pan Qin, Jia Wang
          </td>
          <td>2024-07-31</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Fragile X Syndrome (FXS) is a genetic neurodevelopmental disorder closely associated with intellectual disability and autism spectrum disorders. The core of the disease lies in the abnormal expansion of the CGG trinucleotide repeat sequence at the 5′end of the FMR1 gene. When the repetition exceeds 200 times, it causes the silencing of the FMR1 gene, leading to the absence of the encoded Fragile X mental retardation protein 1 (FMRP). Although the detailed mechanism by which the CGG repeat expansion triggers gene silencing is yet to be fully elucidated, it is known that this process does not alter the promoter region or the coding sequence of the FMR1 gene. This discovery provides a scientific basis for the potential reversal of FMR1 gene silencing through interventional approaches, thereby improving the symptoms of FXS. Epigenetics, a mechanism of genetic regulation that does not depend on changes in the DNA sequence, has become a new focus in FXS research by modulating gene expression in a reversible manner. The latest progress in molecular genetics has revealed that epigenetics plays a key role in the pathogenesis and pathophysiological processes of FXS. This article compiles the existing research findings on the role of epigenetics in Fragile X Syndrome (FXS) with the aim of deepening the understanding of the pathogenesis of FXS to identify potential targets for new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64f2a0a0d20e60f40e1927dda3be0710c0759d2d" target='_blank'>
              Epigenetic insights into Fragile X Syndrome
              </a>
            </td>
          <td>
            Liangqun Xie, Huiying Li, MengLiang Xiao, Ningjing Chen, Xiaoxiao Zang, Yingying Liu, Hong Ye, Chaogang Tang
          </td>
          <td>2024-08-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Understanding cancer evolution is fundamental to predicting cancer progression. However, the evolution of paediatric cancers is still under-researched. Large cohorts of patients are required to determine consistent evolutionary trajectories that shed light on key steps in cancer development and reveal underlying biology, especially in rare cancers. Additionally, well annotated clinical data is necessary for determining if evolutionary biomarkers are predictive of patient outcome. Methods We performed detailed evolutionary analysis of 64 paediatric kidney cancers, including 60 Wilms tumours (WT), using DNA microarrays and, in a subset of 30 patients, a WT-specific targeted sequencing assay, to detect copy number alterations (CNA) and mutations, respectively. By analysing multiple tissue samples in the majority of cases we could detect mutation heterogeneity in each tumour. We reconstructed clones across the cohort and described their phylogenetic histories, in addition to detecting mirrored subclonal allelic imbalance. Results Our results highlight pervasive evidence of parallel evolution in WTs affecting CNAs, and CTNNB1 and TP53 mutations. Furthermore, we demonstrate that stromal-type WTs often evolve from a consistent series of events (WT1 mutation, 11p uniparental disomy and CTNNB1 mutation) and we suggest that 19q uniparental disomy is an important ancestral event in both epithelial and diffuse anaplastic WTs. Finally, we propose the total number of evolutionary CNA events as a prognostic biomarker in WTs for event-free survival, particularly in high-risk WT. Conclusions Overall, this study sheds light on the evolution of the most common paediatric kidney cancer and links evolutionary analysis to fundamental clinical and biological questions in a large cohort of WTs. We conclude that histological subtypes of WT are often defined by consistent evolutionary sequences. We present evidence that a key marker of evolvability, namely CNA diversity measured phylogenetically across multiple tumour sites, is prognostic of patient outcome and should be considered for clinical use to detect the most aggressive blastemal and diffuse anaplastic type WTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a24a5fbaf8b2a5b212183cb8a0bf89d97d1b5f3b" target='_blank'>
              Subtype-specific Patterns of Evolution and Clinically Relevant Genomic Instability in Wilms Tumour
              </a>
            </td>
          <td>
            George D. Cresswell, T. Chagtai, Reem Al-Saadi, T. Treger, G. Madhan, Borbala Mifsud, Gordan M. Vujanic, R. Williams, N. Luscombe, Kathy Pritchard-Jones, William Mifsud
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Cancer cell dormancy (CCD) in colorectal cancer (CRC) poses a significant challenge to effective treatment. In CRC, CCD contributes to tumour recurrence, drug resistance, and amplifying the disease's burden. The molecular mechanisms governing CCD and strategies for eliminating dormant cancer cells remain largely unexplored. Therefore, understanding the molecular mechanisms governing dormancy is crucial for improving patient outcomes and developing targeted therapies. This editorial highlights the complex interplay of signalling pathways and factors involved in colorectal CCD, emphasizing the roles of Hippo/YAP, pluripotent transcription factors such as NANOG, HIF-1α signalling, and Notch signalling pathways. Additionally, ERK/p38α/β/MAPK pathways, AKT signalling pathway, and Extracellular Matrix Metalloproteinase Inducer, along with some potential less explored pathways such as STAT/p53 switch and canonical and non-canonical Wnt and SMAD signalling, are also involved in promoting colorectal CCD. Highlighting their clinical significance, these findings may offer the potential for identifying key dormancy regulator pathways, improving treatment strategies, surmounting drug resistance, and advancing personalized medicine approaches. Moreover, insights into dormancy mechanisms could lead to the development of predictive biomarkers for identifying patients at risk of recurrence and the tailoring of targeted therapies based on individual dormancy profiles. It is essential to conduct further research into these pathways and their modulation to fully comprehend CRC dormancy mechanisms and enhance patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5a1fd8a1c24adda33f9d39ee28ef2dc82663eff" target='_blank'>
              Colorectal cancer cell dormancy: An insight into pathways
              </a>
            </td>
          <td>
            Anil Kumar, L. Saha
          </td>
          <td>2024-09-07</td>
          <td>World Journal of Gastroenterology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND/AIM
There are no established biomarkers for immune checkpoint inhibitors (ICI) in colorectal cancer (CRC) with microsatellite stability (MSS) or proficient mismatch repair (pMMR). Therefore, this study aimed to identify biomarkers for ICI benefit in patients with pMMR by analyzing the down-regulated DNA repair-related genes involved in highly immunogenic and immune responses, and comparing their expression levels and clinical features.


MATERIALS AND METHODS
Mismatch repair (MMR), tumor-infiltrating lymphocytes (TIL), and tumor mutation burden (TMB) were evaluated in 13 CRC cases and mRNA expression levels of 95 DNA repair-related genes were measured. DNA repair-related genes with reduced mRNA expression in the high immunogenicity and high immune response groups were identified. Then, the mRNA expression levels of the identified DNA repair-related genes were measured in 135 patients with CRC. Hierarchical cluster analysis was performed using the mRNA expression levels to compare the clinicopathological characteristics of each cluster.


RESULTS
ATR, LIG4, and RAD52 mRNA levels were significantly down-regulated in the high immunogenicity group. GADD45B, SMUG1, and XRCC6 mRNA levels were significantly down-regulated in the high immune response group. Cases in the cluster with reduced mRNA expression of the six genes were pMMR cases. CD8 mRNA expression level was higher in this cluster than in the other clusters.


CONCLUSION
Decreased mRNA expression levels of ATR, LIG4, RAD52, GADD45B, SMUG1, and XRCC6 genes were associated with high cytotoxic T cell and TMB levels, suggesting that these genes could serve as biomarkers for ICI efficacy in pMMR cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb38d024ab94655fe817bc6cd4f0bb4ad2d969e2" target='_blank'>
              Identification of DNA Repair-related Genes as Biomarkers Reflecting MSI, TMB, and TIL in Colorectal Cancer.
              </a>
            </td>
          <td>
            Junya Fukuda, Tomoya Sudo, Maako Kikuchi, Akihiko Kawahara, Hirona Shigyo, Yusuke Kawamoto, Satoshi Shimamura, Fumiki Koga, Takasi Noguchi, Hiroyuki Nakane, Takahiro Shigaki, Kenji Fujiyoshi, T. Yomoda, Naohiro Yoshida, Kenichi Koushi, Takefumi Yoshida, Jun Akiba, Fumihiko Fujita
          </td>
          <td>2024-07-26</td>
          <td>Anticancer research</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1fa9923fb71606ea55e83b6b8cbb7a9e56b764" target='_blank'>
              Competition shapes the landscape of X-chromosome-linked genetic diversity
              </a>
            </td>
          <td>
            Teresa Buenaventura, Hakan Bagci, Ilinca Patrascan, Joshua J Graham, Kelsey D Hipwell, Roel Oldenkamp, James W D King, Jesus Urtasun, G. Young, Daniel Mouzo, D. Gómez-Cabrero, B. Rowland, Daniel Panne, Amanda G Fisher, Matthias Merkenschlager
          </td>
          <td>2024-07-26</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML), an aggressive malignancy of hematopoietic stem cells, is characterized by the blockade of cell differentiation, uncontrolled proliferation, and cell expansion that impairs healthy hematopoiesis and results in pancytopenia and susceptibility to infections. Several genetic and chromosomal aberrations play a role in AML and influence patient outcomes. TP53 is a key tumor suppressor gene involved in a variety of cell features, such as cell-cycle regulation, genome stability, proliferation, differentiation, stem-cell homeostasis, apoptosis, metabolism, senescence, and the repair of DNA damage in response to cellular stress. In AML, TP53 alterations occur in 5%–12% of de novo AML cases. These mutations form an important molecular subgroup, and patients with these mutations have the worst prognosis and shortest overall survival among patients with AML, even when treated with aggressive chemotherapy and allogeneic stem cell transplant. The frequency of TP53-mutations increases in relapsed and recurrent AML and is associated with chemoresistance. Progress in AML genetics and biology has brought the novel therapies, however, the clinical benefit of these agents for patients whose disease is driven by TP53 mutations remains largely unexplored. This review focuses on the molecular characteristics of TP53-mutated disease; the impact of TP53 on selected hallmarks of leukemia, particularly metabolic rewiring and immune evasion, the clinical importance of TP53 mutations; and the current progress in the development of preclinical and clinical therapeutic strategies to treat TP53-mutated disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/442da7a2ffb734ac40ab31ab13428c2a4af47cbd" target='_blank'>
              Impact of p53-associated acute myeloid leukemia hallmarks on metabolism and the immune environment
              </a>
            </td>
          <td>
            Monika Chomczyk, Luca Gazzola, Shubhankar Dash, Patryk Firmanty, Binsah S. George, V. Mohanty, Hussein A. Abbas, Natalia Baran
          </td>
          <td>2024-08-05</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Global healthcare systems have a great challenge in the form of inherited hematological diseases, which necessitates the development of new remedial strategies. By precisely targeting inherited abnormalities, Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-mediated ex vivo gene editing has surfaced as a promising approach to treat these diseases. This review offers a comprehensive examination of the advancements, challenges, and clinical eventuality of CRISPR-Cas9-intermediated ex vivo gene editing for inherited hematological diseases. With advancements in CRISPR-Cas9 technology, the eventuality to correct inheritable mutations responsible for inherited hematological diseases is within reach. However, challenges such as off-target effects, immune responses, and ethical considerations need to be addressed for the safe and effective perpetration of this technology. A promising understanding of how CRISPR-Cas9-intermediated gene editing functions in practice is handed by ongoing clinical studies, giving rise to the possibility of advanced remedial approaches and bettered patient issues. By addressing these complications in a human-readable format, this review attempts to provide greater understanding and appreciation for the eventuality of CRISPR-Cas9 technology in revolutionizing the treatment landscape for these challenging disorders and contribute to the ongoing discussion in the field and facilitate further exploration towards effective treatments for these challenging disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4d496cce2c3ddc8c2b97b6be801d229f168c15" target='_blank'>
              CRISPR-CAS9-MEDIATED EX VIVO GENE EDITING FOR INHERITED HEMATOLOGICAL DISORDERS: ADVANCEMENTS, CHALLENGES, AND CLINICAL POTENTIAL
              </a>
            </td>
          <td>
            Atasi Ranjan Panda, Shreeya Das
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Pharmacy and Pharmaceutical Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is projected to become the second leading cause of cancer death by 2030. PDAC lacks effective treatment options and innovative strategies for therapeutic intervention are urgently needed. Chimeric Antigen Receptor (CAR)-T cell therapy has shown remarkable success in treating hematologic malignancies, yet achieving efficacy in solid tumors remains a significant challenge. Early results from clinical trials show limited efficacy of CAR T-cell therapy in PDAC. To uncover tumor-intrinsic mechanisms of resistance to CAR T-cell therapy in vivo, we performed iterative CRISPR/Cas9-based pooled screens in a fully immunocompetent, orthotopic model of PDAC. We identified a variety of genes influencing CAR T-cell treatment efficacy. In particular, loss of genes involved in redox biology and oxidative stress sensitized PDAC tumors to CAR T-cell therapy in vivo. Additionally, single-cell RNA sequencing of PDAC tumors subjected to CAR T-cell therapy unveiled a distinct subset of tumor cells resilient to the treatment, characterized by changes in oxidative stress. Our findings indicate a potential avenue for therapeutic synergy by targeting pathways involved in oxidative stress to increase the efficacy of CAR T cell therapy in PDAC.
 Citation Format: Julia Fröse, Charlie Whittaker, Paul Leclerc, Adam Langenbucher, Riley Hellinger, Daniel R. Goulet, Michael T. Hemann. Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B060.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a130506e26e2da6b9c6338dc0032b746f1f4ca46" target='_blank'>
              Abstract B060: Unveiling resistance mechanisms to CAR T-cell therapy in pancreatic cancer through in vivo CRISPR/Cas9 knockout screening
              </a>
            </td>
          <td>
            Julia Fröse, Charles A. Whittaker, Paul Leclerc, Adam Langenbucher, Riley D Hellinger, Daniel R. Goulet, M. Hemann
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Alveolar rhabdomyosarcoma (ARMS) patients harboring PAX3-FOXO1 and PAX7-FOXO1 fusion proteins exhibit a greater incidence of tumor relapse, metastasis, and poor survival outcome, thereby underscoring the urgent need to develop effective therapies to treat this subtype of childhood cancer. To uncover mechanisms that contribute to tumor initiation, we developed a novel muscle progenitor model and used epigenomic approaches to unravel genome re-wiring events mediated by PAX3/7 fusion proteins. Importantly, these regulatory mechanisms are conserved across established ARMS cell lines, primary tumors, and orthotopic-patient derived xenografts. Among the key targets of PAX3- and PAX7-fusion proteins, we identified a cohort of oncogenes, FGF receptors, and genes essential for mitochondrial metabolism and protein translation, which we successfully targeted in preclinical trials. Our data suggest an explanation for the relative paucity of recurring mutations in this tumor, provide a compelling list of actionable targets, and suggest promising new strategies to treat this tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f840f071bc2696a99f7c83f33f72c0a963e3308" target='_blank'>
              PAX fusion proteins deregulate gene networks controlling mitochondrial translation in pediatric rhabdomyosarcoma
              </a>
            </td>
          <td>
            Bhargab Kalita, Gerard Martinez-Cebrian, Justina McEvoy, Melody Allensworth, Michelle Knight, Alessandro Magli, Rita C. R. Perlingeiro, Michael A. Dyer, Elizabeth Stewart, B. Dynlacht
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d22a32ba49d7731185468f66f4a9a15ddb1efe55" target='_blank'>
              Is cross-species horizontal gene transfer responsible for gallbladder carcinogenesis
              </a>
            </td>
          <td>
            Monika Rajput, Manoj Pandey, R. Dixit, Vijay K. Shukla
          </td>
          <td>2024-07-30</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Richter’s transformation (RT) is a progression of chronic lymphocytic leukemia (CLL) to aggressive lymphoma. MGA (Max gene associated), a functional MYC suppressor, is mutated at 3% in CLL and 36% in RT. However, genetic models and molecular mechanisms of MGA deletion that drive CLL to RT remain elusive. We established an RT mouse model by knockout of Mga in the Sf3b1/Mdr CLL model using CRISPR-Cas9 to determine the role of Mga in RT. Murine RT cells exhibited mitochondrial aberrations with elevated oxidative phosphorylation (OXPHOS). Through RNA sequencing and functional characterization, we identified Nme1 (nucleoside diphosphate kinase) as an Mga target, which drives RT by modulating OXPHOS. Given that NME1 is also a known MYC target without targetable compounds, we found that concurrent inhibition of MYC and electron transport chain complex II substantially prolongs the survival of RT mice in vivo. Our results suggest that the Mga-Nme1 axis drives murine CLL-to-RT transition via modulating OXPHOS, highlighting a potential therapeutic avenue for RT. A MGA/NME1 regulatory axis can promote CLL-to-RT transition by modulating OXPHOS, highlighting this pathway as a target for RT treatment. Editor’s summary Richter’s transformation (RT) involves the progression of chronic lymphocytic leukemia (CLL) to a more aggressive lymphoma with poor outcomes. Understanding the transition into RT is key to more targeted therapeutic strategies. Iyer et al. examined the role of the MYC suppressor MGA, which is differentially expressed in RT, in an in vivo CLL model with MGA dysfunction. MGA loss resulted in increased mitochondrial function and reactive oxygen species and led to aberrant cell morphology and up-regulated MYC as commonly observed in patients. Combined inhibition of MYC and mitochondrial respiration prolonged in vivo survival in this model, highlighting potential therapeutic strategies for RT with this genetic profile. —Catherine Charneski">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75d1c8fe0890ca3dad978b633b25c5e7b5178f59" target='_blank'>
              MGA deletion leads to Richter’s transformation by modulating mitochondrial OXPHOS
              </a>
            </td>
          <td>
            Prajish Iyer, Bo Zhang, Tingting Liu, Meiling Jin, Kevyn L Hart, Jibin Zhang, Viola Siegert, Marianne Remke, Xuesong Wang, Lei Yu, Joo Y Song, Girish Venkataraman, Wing C. Chan, Zhenyu Jia, M. Buchner, Tanya Siddiqi, Steven T. Rosen, Alexey Danilov, Lili Wang
          </td>
          <td>2024-07-31</td>
          <td>Science Translational Medicine</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/311bc754350204df7366f1cc87322a6aee4bc5a6" target='_blank'>
              Assessment of SWI/SNF chromatin remodeling complex related genes as potential biomarkers and therapeutic targets in pan-cancer
              </a>
            </td>
          <td>
            Kai Zhuang, Lishan Wang, Chengyu Lu, Zhiping Liu, Dongli Yang, Hao Zhong, Jiami Zou, Aamir Fahira, Jiaojiao Wang, Zunnan Huang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78087b18cde872814454e116c4aae9d273fd362a" target='_blank'>
              Modeling X chromosome inactivation using t5iLA naive human pluripotent stem cells
              </a>
            </td>
          <td>
            Yudan Shang, Nannan Wang, Haoyi Wang, Chenrui An, Wen Sun
          </td>
          <td>2024-09-18</td>
          <td>BMC Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Understanding bacterial gene function remains a major biological challenge. Double-mutant genetic interaction (GI) analysis addresses this challenge by uncovering the functional partners of targeted genes, allowing us to associate genes of unknown function with novel pathways and unravel connections between well-studied pathways, but is difficult to implement at the genome-scale. Here, we develop and use double-CRISPRi to systematically quantify genetic interactions at scale in the Bacillus subtilis envelope, including essential genes. We discover > 1000 known and novel genetic interactions. Our analysis pipeline and experimental follow-ups reveal the distinct roles of paralogous genes such as the mreB and mbl actin homologs, and identify new genes involved in the well-studied process of cell division. Overall, our study provides valuable insights into gene function and demonstrates the utility of double-CRISPRi for high-throughput dissection of bacterial gene networks, providing a blueprint for future studies in diverse bacterial species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed882175f2414bae232e1b044a85ff77b9c005c" target='_blank'>
              Comprehensive double-mutant analysis of the Bacillus subtilis envelope using double-CRISPRi
              </a>
            </td>
          <td>
            B. Koo, Horia Todor, Jiawei Sun, Jordi van Gestel, John S. Hawkins, Cameron C. Hearne, Amy B. Banta, K. C. Huang, Jason M. Peters, Carol A Gross
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfda792d36cb0c85bc9dd50884926690c096c37e" target='_blank'>
              RNA polymerase stalling-derived genome instability underlies ribosomal antibiotic efficacy and resistance evolution
              </a>
            </td>
          <td>
            Yayun Zheng, Ruochen Chai, Tian Wang, Zeqi Xu, Yihui He, P. Shen, Jintao Liu
          </td>
          <td>2024-08-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="An important goal in cancer research is to identify driver genes and mutations. Reasoning that such mutations often alter enzymatic functions, we investigated the cancer driver role of enzyme families. Using pan-cancer genomic data and established driver mutation catalogues, we found an unexpectedly high rate of mutations in helicases, making helicases the most frequently mutated enzyme family in cancer. Based on both functional perturbation screens and cancer genomic analyses, we provide evidence that cancers with mutated helicases converge on increased genomic instability and faulty DNA repair. We identify a striking phenotype in cells with loss of the helicase Aquarius (AQR). AQR was exclusively hemizygous lost in cancer genomes, which was associated with elevated levels of structural variants and point mutation signatures indicative of homologous recombination deficiency. Finally, we leverage large dependency maps to show that hemizygous loss is a common tumour suppression mechanism among helicases. In summary, we uncover a striking frequency of mutated helicases with key roles in genomic maintenance, and we nominate novel hemizygous cancer driver genes including AQR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/381ed30f7e9d5d197bbe8fa9f74d37852c71bb4a" target='_blank'>
              Enzyme family-centred approach identifies helicases as recurrent hemizygous tumour suppressor genes
              </a>
            </td>
          <td>
            Karolin Voßgröne, F. Favero, Krushanka Kashyap, F. G. Rodríguez-González, André Vidas Olsen, Xin Li, Balca R. Mardin, Joachim Weischenfeldt, Claus S. Sørensen
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4568f027a9bf73e7d22e0523c166497f5edf7122" target='_blank'>
              Perturbing TET2 condensation promotes aberrant genome-wide DNA methylation and curtails leukaemia cell growth.
              </a>
            </td>
          <td>
            Lei Guo, Tingting Hong, Yi-Tsang Lee, Xue Hu, Guokai Pan, Rongjie Zhao, Yuhan Yang, Jingwen Yang, Xiaoli Cai, Logan Rivera, Jie Liang, Rui Wang, Yaling Dou, Srikanth Kodali, Wenbo Li, Leng Han, Bruno Di Stefano, Yubin Zhou, Jia Li, Yun Huang
          </td>
          <td>2024-09-09</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7a9a7a504580b14d647edcfb85c539f3ac48" target='_blank'>
              Enhanced depletion of MLL-fusion proteins in acute leukemia: potential for improved therapeutic outcomes
              </a>
            </td>
          <td>
            Noelia Che, S. Cantilena, R. Looi-Somoye, Danesh Sundar, Kent Fung, J. de Boer, Owen Williams
          </td>
          <td>2024-08-16</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7d59a35bbbd452d19f5eb530222e3178b220b08" target='_blank'>
              Tumour mutational burden: clinical utility, challenges and emerging improvements.
              </a>
            </td>
          <td>
            J. Budczies, D. Kazdal, Michael Menzel, Susanne Beck, K. Kluck, Christian Altbürger, C. Schwab, M. Allgäuer, A. Ahadova, Matthias Kloor, P. Schirmacher, Solange Peters, Alwin Krämer, P. Christopoulos, A. Stenzinger
          </td>
          <td>2024-08-27</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="INTRODUCTION
Oncogenic-addicted non-small cell lung cancer (NSCLC) has emerged as the most prevalent form of lung cancer, presenting a dynamic landscape in treatment modalities. Among these, epidermal growth factor receptor (EGFR)-mutant NSCLC remains the predominant oncogenic mutation, particularly prevalent in regions such as Asia and Latin America.


CASE PRESENTATION
This case study highlights the experience of a woman diagnosed with EGFR-sensitive (del exon 19) mutant NSCLC who demonstrated an extended duration of response (DOR) to third-generation EGFR-TKI therapy. Upon disease progression, detection of MET gene amplification prompted the addition of a selective MET inhibitor to the existing EGFR-TKI regimen, resulting in a complete response for the patient.


DISCUSSION/CONCLUSION
The molecular heterogeneity of this condition has significantly increased in complexity over recent years, marked by the identification of baseline co-alterations and development of a broad spectrum of resistance mechanisms post-EGFR tyrosine kinase inhibitor (TKI) therapy. This complexity poses a substantial challenge to clinicians. Despite the rapid advancement of targeted therapies and the implementation of treatment escalation through combination strategies, there remains an ongoing debate regarding which patients would benefit most from combination therapies, both in the initial treatment phase and in the setting of disease progression, particularly when off-target resistance mechanisms or co-alterations are identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aee7b0dfd69142858e9dd93edb7ec143688a294" target='_blank'>
              EGFR plus MET targeted therapies for overcoming treatment resistance in EGFR mutant NSCLC. A Case Report.
              </a>
            </td>
          <td>
            Maria F Martínez-Hernandez, L. Lara-Mejía, C. Izquierdo-Tolosa, L. Cabrera-Miranda, Oscar Arrieta
          </td>
          <td>2024-09-18</td>
          <td>Oncology research and treatment</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="137 Background: Resistance to RT remains a clinical challenge, given our limited understanding of the molecular pathways underpinning radioresistance. Here, we investigated for mechanisms linked to RR using paired experimental and clinico-molecular HNC and PCa datasets for discovery and validation of novel pathways that may be enriched in RR cancers. Methods: For experimental data, we generated RR HNC (FaDu, HK1) and PCa (22Rv1, DU145) cell lines following high-dose X-irradiation (90 Gy in 45 fr). Genotypic characterization of RR and wildtype (WT) cells was performed by genomic and transcriptomic sequencing using WES (100X) and RNAseq (Illumina, CA), respectively, followed by functional characterization by western blot (WB) and immunofluorescence (IF). For clinico-molecular data, we utilized two prospectively recruited RT cohorts, consisting of 311 patients with HNC (n=158) and PCa (n=153). Tumor transcriptomes were profiled using RNAseq and Affymetrix ST array (ThermoFisher, CA), respectively. Comparative analyses of molecular profiles were first performed between RR and WT in vitro models to identify dysregulated pathways that were linked to radioresistance, and subsequently tested for association with disease-free survival (DFS) in the clinical cohorts. Results: Comparative genomic analyses between RR and WT HNC and PCa models revealed an abundance of acquired mutations in the RR compared with WT models, with a higher mutation count observed in PCa than HNC cells (SNV counts: 2,158 [DU145-RR] and 1,387 [22Rv1-RR] vs 396 [FaDu-RR] and 25 [HK1-RR], P<0.0001). Mutational signature analyses indicated a common enrichment of DNA mismatch repair-related mutational signatures (SBS3, SBS14, and SBS21) across the RR HNC and PCa models. Transcriptomic profiling revealed an upregulation of the BAHD1 gene, which is involved in heterochromatin formation, across the 4 RR cell lines, corroborated by an enrichment of heterochromatin-related genesets. These results were consistent with functional characterization of RR versus WT cells by WB and IF indicating increased DNA repair capacity and heterochromatin response post-4 Gy irradiation. We further confirmed the dependency of the RR phenotype on the heterochromatin response with BAHD1 knockdown, resulting in reversal of these cellular responses. Finally, our results were supported by survival analyses indicating an inferior DFS post-RT in patients with HNC and PCa harboring a higher expression of heterochromatin-related geneset (HNC: HRhigh vs low 1.51 ; PCa: HRhigh vs low 1.37 ). Conclusions: Herein, by leveraging on paired experimental and clinico-molecular datasets, we have uncovered a novel BAHD1-dependent heterochromatin response that underpins resistance to RT in HNC and PCa, which may be amendable to systemic agents targeting chromatin remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea1270e10cc26b39aabc2ffd1632147257d9664" target='_blank'>
              Analysis of heterochromatin remodeling as a mechanism of radiotherapy (RT) resistance through molecular profiling of radioresistant (RR) head and neck (HNC) and prostate cancers (PCa).
              </a>
            </td>
          <td>
            Evelyn Tan, Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, Dennis J J Poon, Dewi Susanti, E. Yeo, Pek L Chu, Melvin L K Chua
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="microRNAs (miRNAs), tiny, non-coding RNA molecules, fine-tune the expression of target genes through interacting with mRNAs. These miRNAs are involved in a wide range of biological processes, encompassing cell division, death, blood cell production, and tumor development. When these miRNAs become dysfunctional, they can promote the invasion and spread of cancer cells in various human malignancies, including leukemia. Acute lymphoblastic leukemia (ALL), the preeminent malignancy affecting children, is a blood cancer marked by the uncontrollable growth of immature lymphoid cells that displace healthy blood precursors in the bone marrow. Despite a decline in ALL mortality rates over the past two decades, a significant proportion of deaths still results from a lack of effective diagnostic and prognostic markers that can guide treatment decisions and overcome drug resistance. The analysis of miRNA expression patterns in ALL could lead to more precise disease classification, earlier diagnosis, and better prognostic outcomes in the near future. The connection between miRNA dysfunction and the biology of ALL suggests that these molecules could represent promising therapeutic targets. Therefore, this review delves into the regulatory mechanisms of miRNAs in pediatric ALL, exploring how miRNA-based diagnostic, prognostic, and therapeutic strategies offer unique advantages and hold promise for clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b51d3730adcae846d16d92db76f51f153c5c1d0" target='_blank'>
              Advancements in the Regulatory Role of microRNAs in Childhood Acute Lymphoblastic Leukemia: Mechanisms and Clinical Implications
              </a>
            </td>
          <td>
            Wei Deng
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ed5339b18c8cb661ce34053233bc7343f92ee3a" target='_blank'>
              1q jumping translocation as a biomarker in myeloid malignancy: frequently mutated genes associated with bad prognosis and low survival
              </a>
            </td>
          <td>
            Eitan Halper-Stromberg, V. Stinnett, L. Morsberger, Aparna Pallavajjala, M. Levis, A. DeZern, Michelle Lei, Brian Phan, Rena R. Xian, Christopher D. Gocke, Guilin Tang, Ying S Zou
          </td>
          <td>2024-08-01</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8eb55432a81ae9f11fc653e3722684f5e8a7ed1" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Ganciclovir (GCV) is a clinically important drug as it is used to treat viral infections. GCV is incorporated into the DNA during replication, where it interferes with subsequent replication on GCV-incorporated templates. However, the effects of GCV on the host genome and the mechanisms underlying cellular tolerance to GCV remain unclear. In this study, we explored these mechanisms using a collection of mutant DT40 cells. We identified RAD17/-, BRCA1-/-, and RAD18-/- cells as highly GCV-sensitive. RAD17, a component of the alternative checkpoint-clamp loader RAD17-RFC, was required for the activation of the intra-S checkpoint following GCV treatment. BRCA1, a critical factor for promoting homologous recombination (HR), was required for suppressing DNA double-strand breaks (DSBs). Moreover, RAD18, an E3-ligase involved in DNA repair, was critical in suppressing the aberrant ligation of broken chromosomes caused by GCV. We found that BRCA1 suppresses DSBs through HR-mediated repair and template switching (TS)-mediated damage bypass. Moreover, the strong GCV sensitivity of BRCA1-/- cells was rescued by the loss of 53BP1, despite the only partial restoration in the sister chromatid exchange events which are hallmarks of HR. These results indicate that BRCA1 promotes cellular tolerance to GCV through two mechanisms, TS and HR-mediated repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb90975f7c5cd2ebcb118dd3d319f550cae4f8b1" target='_blank'>
              RAD18- and BRCA1-dependent pathways promote cellular tolerance to the nucleoside analog ganciclovir.
              </a>
            </td>
          <td>
            Tasnim Ahmad, Ryotaro Kawasumi, Kouji Hirota
          </td>
          <td>2024-08-22</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Huntington's disease (HD) is an inherited neurodegenerative disorder caused by the abnormal expansion of CAG trinucleotide repeats in the Huntingtin gene (HTT) located on chromosome 4. It is transmitted in an autosomal dominant manner and is characterized by motor dysfunction, cognitive decline, and emotional disturbances. To date, there are no curative treatments for HD have been developed; current therapeutic approaches focus on symptom relief and comprehensive care through coordinated pharmacological and non-pharmacological methods to manage the diverse phenotypes of the disease. International clinical guidelines for the treatment of HD are continually being revised in an effort to enhance care within a multidisciplinary framework. Additionally, innovative gene and cell therapy strategies are being actively researched and developed to address the complexities of the disorder and improve treatment outcomes. This review endeavours to elucidate the current and emerging gene and cell therapy strategies for HD, offering a detailed insight into the complexities of the disorder and looking forward to future treatment paradigms. Considering the complexity of the underlying mechanisms driving HD, a synergistic treatment strategy that integrates various factors-such as distinct cell types, epigenetic patterns, genetic components, and methods to improve the cerebral microenvironment-may significantly enhance therapeutic outcomes. In the future, we eagerly anticipate ongoing innovations in interdisciplinary research that will bring profound advancements and refinements in the treatment of HD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6618affd7d51107e033086146a23c1aab41c1ab" target='_blank'>
              Advances in Gene and Cellular Therapeutic Approaches for Huntington's Disease.
              </a>
            </td>
          <td>
            Xuejiao Piao, Dan Li, Hui Liu, Qing Guo, Yang Yu
          </td>
          <td>2024-08-09</td>
          <td>Protein & cell</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Background: Actionable driver mutations account for 40–50% of NSCLC cases, and their identification clearly affects treatment choices and outcomes. Conversely, non-actionable mutations are genetic alterations that do not currently have established treatment implications. Among co-occurring alterations, the identification of concurrent actionable genomic alterations is a rare event, potentially impacting prognosis and treatment outcomes. Methods: We retrospectively evaluated the prevalence and patterns of concurrent driver genomic alterations in a large series of NSCLCs to investigate their association with clinicopathological characteristics, to assess the prognosis of patients whose tumor harbors concurrent alterations in the genes of interest and to explore their potential therapeutic implications. Results: Co-occurring driver alterations were identified in 26 out of 1520 patients with at least one gene alteration (1.7%). Within these cases, the incidence of concurrent actionable gene alterations was 39% (0.7% of the overall cohort). Among compound actionable gene mutations, EGFR was the most frequently involved gene (70%). The most frequent association was EGFR mutations with ROS1 rearrangement. Front-line targeted treatments were the preferred approach in patients with compound actionable mutations, with dismal median PFS observed (6 months). Conclusions: Advances in genomic profiling technologies are facilitating the identification of concurrent mutations. In patients with concurrent actionable gene alterations, integrated molecular and clinical data should be used to guide treatment decisions, always considering rebiopsy at the moment of disease progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59022475d568d630ac2b88c69a13d9f08f1373da" target='_blank'>
              Co-Occurring Driver Genomic Alterations in Advanced Non-Small-Cell Lung Cancer (NSCLC): A Retrospective Analysis
              </a>
            </td>
          <td>
            I. Attili, Riccardo Asnaghi, D. Vacirca, Riccardo Adorisio, A. Rappa, Alberto Ranghiero, Mariano Lombardi, C. Corvaja, Valeria Fuorivia, Ambra Carnevale Schianca, P. Trillo Aliaga, G. Spitaleri, E. del Signore, J. Guarize, Lorenzo Spaggiari, E. Guerini-Rocco, Nicola Fusco, F. de Marinis, A. Passaro
          </td>
          <td>2024-07-31</td>
          <td>Journal of Clinical Medicine</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="One of the main causes of cancer-related mortality for women worldwide is breast cancer (BC). The XRCC2 gene, essential for DNA repair, has been implicated in cancer susceptibility. This study aims to evaluate the association between XRCC2 and BC risk. The study was conducted at Zheen International Hospital in Erbil, Iraq, between 2021 and 2024 with a total of 88 samples, including 44 paired normal and cancer tissue samples. Mutation analysis was performed using Next-Generation Sequencing, coupled with in silico tools for variant impact prediction. Expression levels were assessed through RT-PCR, and methylation status was determined using methylation-sensitive restriction enzyme digestion PCR. The study identified seven inherited germline variants in the XRCC2 gene, with five of these mutations being Uncertain Significance, one being Likely Pathogenic, and one being Likely benign. RNA purity was found high with mean A260/280 ratios of 1.986 ± 0.097 in normal (N) and 1.963 ± 0.092 in tumor (T) samples. Tumor samples exhibited a higher RNA concentration (78.56 ± 40.87 ng/µL) than normal samples (71.44 ± 40.79 ng/µL). XRCC2 gene expression was significantly upregulated in tumor tissue, with marked increases in patients aged 40-55 and >56 years and in higher cancer grades (II and III) and invasive ductal carcinoma (p-values ranging from <0.0001 to 0.0392). DNA methylation rates in tumor tissues were low (7%), suggesting limited regulation by methylation. The study suggests that XRCC2 can be classified as an oncogene and that its structural investigation by targeted NGS and expression evaluation can be used as a potential biomarker in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81c449dfd85b137aba37b45f3107a8eae5435a35" target='_blank'>
              Association of XRCC2 with breast cancer, a multi-omics analysis at genomic, transcriptomic, and epigenomic level.
              </a>
            </td>
          <td>
            N. Gilani, Mehmet Ozaslan
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16db7fe8f3bc4329cf344c224d8e623bae436c6d" target='_blank'>
              Elucidating the pan-oncologic landscape of S100A9: prognostic and therapeutic corollaries from an integrative bioinformatics and Mendelian randomization analysis
              </a>
            </td>
          <td>
            Yingying Chen, Zixuan Wu, Xingxing Yi
          </td>
          <td>2024-08-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Due to the significant morbidity and mortality of hemoglobinopathies, curative options have long been pursued. The overall goal of gene therapy is to modify a patient's own hematopoietic stem cells to overcome the deleterious effects of the underlying genetic defect by gene addition, gene editing, or gene silencing. Gene addition incorporates genes with superior function than the abnormal gene; gene editing takes advantage of molecular tools such as zinc finger proteins, Transcription Activator-Like Effector Nucleases and Clustered Regularly Interspaced Short Palindromic Repeats coupled with Cas9 proteins (CRISPR-Cas9) which allow for sequence-specific breaks in DNA that disrupt gene function; and gene silencing suppresses gene expression by interference with mRNA transcription/protein translation or epigenetic modification. The majority of gene therapy strategies for hemoglobinopathies have targeted erythroid-specific BCL11A, a major regulator of fetal hemoglobin repression at the gamma-globin locus, in the normal fetal-to-adult hemoglobin switch that occurs shortly after birth. Other goals have involved the incorporation of anti-sickling globins, such as βT87Q or βAS3. Landmark clinical trials of gene therapy in transfusion-dependent thalassemia and sickle cell disease have shown remarkable efficacy and acceptable safety and culminated in recent regulatory approvals of gene therapy for both diseases in Europe and the United States.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f812c9e54de47feca0c8615f3dda20f25f85b41" target='_blank'>
              A Review of Gene Therapies for Hemoglobinopathies.
              </a>
            </td>
          <td>
            Boubini Jones-Wonni, Amar H Kelkar, Maureen O Achebe
          </td>
          <td>2024-08-15</td>
          <td>Hemoglobin</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aberrant gene expression patterns in acute myeloid leukemia (AML) with balanced chromosomal translocations are often associated with dysregulation of epigenetic modifiers. The AML1/ETO (RUNX1/MTG8) fusion protein, caused by the translocation (8;21)(q22;q22), leads to the epigenetic repression of its target genes. We aimed in this work to identify critical epigenetic modifiers, on which AML1/ETO-positive AML cells depend on for proliferation and survival using shRNA library screens and global transcriptomics approaches. Using shRNA library screens, we identified 41 commonly depleted genes in two AML1/ETO-positive cell lines Kasumi-1 and SKNO-1. We validated, genetically and pharmacologically, DNMT1 and ATR using several AML1/ETO-positive and negative cell lines. We also demonstrated in vivo differentiation of myeloblasts after treatment with the DNMT1 inhibitor decitabine in a patient with an AML1/ETO-positive AML. Bioinformatic analysis of global transcriptomics after AML1/ETO induction in 9/14/18-U937 cells identified 973 differentially expressed genes (DEGs). Three genes (PARP2, PRKCD, and SMARCA4) were both downregulated after AML1/ETO induction, and identified in shRNA screens. In conclusion, using unbiased shRNA library screens and global transcriptomics, we have identified several driver epigenetic regulators for proliferation in AML1/ETO-positive AML. DNMT1 and ATR were validated and are susceptible to pharmacological inhibition by small molecules showing promising preclinical and clinical efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4848d5afaf1e1f3f5c753596a2ed8d1e491ca9f0" target='_blank'>
              Identification of epigenetic modifiers essential for growth and survival of AML1/ETO-positive leukemia.
              </a>
            </td>
          <td>
            J. Duque-Afonso, P. Veratti, Usama-Ur Rehman, Heike Herzog, J. Mitschke, G. Greve, Julian Eble, Bettina Berberich, Johanna Thomas, Milena Pantic, Miguel Waterhouse, Gaia Gentile, Olaf Heidenreich, Cornelius Miething, Michael Lübbert
          </td>
          <td>2024-08-15</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Li-Fraumeni syndrome (LFS) is a hereditary cancer predisposition syndrome associated with a highly penetrant cancer spectrum characterized by germline TP53 mutations. We characterized the first LFS zebrafish hotspot mutants, tp53 R217H and R242H (human R248H and R273H), and found these mutants exhibit partial-to-no activation of p53 target genes, have a defective G1 cell-cycle checkpoint, and are resistant to apoptosis. Spontaneous tumor development histologically resembling human sarcomas developed as early as 6 months. tp53 R242H mutants had a higher lifetime tumor incidence compared to tp53 null and R217H mutants, suggesting it is a more aggressive mutation. We observed mutation-specific tumor phenotypes across tp53 mutants with associated diverse transcriptomic and DNA methylome profiles, impacting metabolism, cell signalling, and biomacromolecule synthesis and degradation. These tp53 zebrafish mutants demonstrate fidelity to their human counterparts and provide new insights into underlying tumorigenesis mechanisms and kinetics, which will guide targeted therapeutics for LFS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe84da849b9bf9707014c32841afe469ca3c5e99" target='_blank'>
              tp53 R217H and R242H Mutant Zebrafish Exhibit Dysfunctional p53 Hallmarks and Recapitulate Li-Fraumeni Syndrome Phenotypes
              </a>
            </td>
          <td>
            Kim Kobar, Lissandra Tuzi, Erin Burnley, Jennifer A. Fiene, Kristianne J. C. Galpin, C. Midgen, B. Laverty, Vallijah Subasri, Timmy T. Wen, Martin Hirst, M. Moksa, A. Carles, Qi Cao, A. Shlien, David Malkin, Sergey V. Prykhozhij, Jason N Berman
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Long non‐coding RNAs (lncRNAs) can crucially regulate activation and transformation of cancer‐associated fibroblasts (CAFs) but have not been systematically investigated at single cell resolution. Here, by utilizing integrated single‐cell sequencing datasets, we screened the aberrantly expressed lncRNAs in CAFs, which are the major component of tumor microenvironment. Our findings revealed a consistent CAF‐specific downregulation of Maternally Expressed Gene 3 (MEG3) expression and increased MEG3+ proportion at the pan‐cancer level, which may be attributed to m6A‐related post‐transcriptional modifications. Through activation trajectory analysis of the major CAF subtypes, it was determined that elevated MEG3 expression in CAFs leads to an increase in PDGFRA expression. This, in turn, promotes CAF activation and transformation into an MEG3+ adipogenic CAF (MACAF) subtype, which is more sensitive to Dasatinib. MACAF‐related cell–cell interactions highlighted that MACAF could enhance the epithelial‐mesenchymal transition process in tumor cells via the TGF‐β pathway, promoting tumor cell migration and possibly contributing to tumor progression and invasiveness. Notably, patients with higher MACAF scores experience unfavorable prognoses and poor response rates to checkpoint inhibitor‐based immunotherapy, suggesting a correlation between MACAF and immunosuppressive microenvironment shaping. Our findings provide novel insights of the MEG3 in CAF activation and highlight the potential value of the MACAF score for therapeutic strategies design involving Dasatinib and immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60513de4f9e6ad73bba4418e0a42f0b64c44877e" target='_blank'>
              The role of long non‐coding RNA Maternally Expressed Gene 3 in cancer‐associated fibroblasts at single cell pan‐cancer level
              </a>
            </td>
          <td>
            Tao Zhou, Huayun Yan, Yiqi Deng, Yunfeng Zhu, Xuyang Xia, Wanchun Wu, Weihan Zhang, Hai-Ning Chen, Jian‐Kun Hu, Zong‐Guang Zhou, Yang Shu, Yuan Li, Heng Xu
          </td>
          <td>2024-09-09</td>
          <td>Interdisciplinary Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="The XmnI Gg -158 C/T polymorphism has been widely associated with fetal hemoglobin (HbF) levels, the severity of disease, and the response to the drug hydroxyurea (HU) in both β-thalassemia (β-thal) and sickle cell disease (SCD) patients. However, the functional significance of this single nucleotide polymorphism (SNP) remains unclear. To gain insight, green fluorescence protein (GFP) cassettes harboring the XmnI C or T alleles in their left homology arms (i.e. Gg promoters) were knocked into the Gg gene(s) of K562 cells via CRISPR/Cas9. Subsequently, the GFP fluorescence levels were compared in the ensuing cell populations and isolated clones. In both instances, median fluorescence intensities (MFI) of the knockin cells having the inserted XmnI T allele were higher than those having the XmnI C allele. Our results suggest that the XmnI T allele can increase Gg expression in K562 cells. The possible functional significance of the XmnI Gg -158 C/T polymorphism provides a rationale for the aforementioned associations. Furthermore, the XmnI polymorphism as a functional SNP substantiates its importance as a prognostic marker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bedc9fe1b28e8e9a54e59969786c8981bcb8830d" target='_blank'>
              Genome editing in K562 cells suggests a functional role for the XmnI Gg polymorphism: a widely used genetic marker in β-thalassemia and sickle cell disease patients.
              </a>
            </td>
          <td>
            A. Ahmadifard, Nahal Maroofi, Maryam Maleki Tehrani, Tahere Dabestani, Masoumeh Sadat Mousavi Maleki, Sepideh Bayrami, M. Banan
          </td>
          <td>2024-07-28</td>
          <td>Cellular and molecular biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Genome editing has demonstrated its utility in generating isogenic cell-based disease models, enabling the precise introduction of genetic alterations into wild-type cells to mimic disease phenotypes and explore underlying mechanisms. However, its application in liver-related diseases has been limited by challenges in genetic modification of mature hepatocytes in a dish. Here, we conducted a systematic comparison of various methods for primary hepatocyte culture and gene delivery to achieve robust genome editing of hepatocytes ex vivo. Our efforts yielded editing efficiencies of up to 80% in primary murine hepatocytes cultured in monolayer and 20% in organoids. To model human hepatic tumorigenesis, we utilized hepatocytes differentiated from human pluripotent stem cells (hPSCs) as an alternative human hepatocyte source. We developed a series of cellular models by introducing various single or combined oncogenic alterations into hPSC-derived hepatocytes. Our findings demonstrated that distinct mutational patterns led to phenotypic variances, affecting both overgrowth and transcriptional profiles. Notably, we discovered that the PI3KCA E542K mutant, whether alone or in combination with exogenous c-MYC, significantly impaired hepatocyte functions and facilitated cancer metabolic reprogramming, highlighting the critical roles of these frequently mutated genes in driving liver neoplasia. In conclusion, our study demonstrates genome-engineered hepatocytes as valuable cellular models of hepatocarcinoma, providing insights into early tumorigenesis mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc0098e14922960e96572df20d519a8a83fa8956" target='_blank'>
              Genome Engineering of Primary and Pluripotent Stem Cell-Derived Hepatocytes for Modeling Liver Tumor Formation
              </a>
            </td>
          <td>
            Lulu Zhang, Xunting Wang, Xuelian Yang, Yijia Chi, Yihang Chu, Yi Zhang, Yufan Gong, Fei Wang, Qian Zhao, Dongxin Zhao
          </td>
          <td>2024-09-02</td>
          <td>Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Introduction Detection of tumor-derived targetable alterations from peripheral blood samples represents a unique opportunity for patients (pts) with hard-to-biopsy neuroblastoma (NBL). The overall aim of LIBERTY is to describe the molecular alteration landscape identified by circulating tumor DNA (ctDNA) profiling in pts with relapsed/refractory NBL by providing rapid turnaround results to Canadian patients, using a clinically validated commercial liquid biopsy platform. Methods LIBERTY is a prospective non-interventional study of molecular profiling of ctDNA for pts with relapsed, refractory or not amenable to biopsy NBL, treated in a Canadian oncology center. The Tempus next-generation sequencing xF+ assay is performed on ctDNA extracted from blood collected at a single timepoint. The 523-gene panel includes single nucleotide variants, insertions and deletions, copy number variants and chromosomal rearrangements, using 30-50 ng of input DNA. An analytically validated report of genomic findings is returned to the study team and may be used to inform treatment recommendations. Results To date, 15 pts have been enrolled (6 male), with a median age of 6.7 y (range, 2.8-24.6). Disease status at enrolment was primary refractory/progressive NBL (n=2), first relapse (n=4), second or greater relapse (n=9). Nine pts were primarily treated at the Hospital for Sick Children. Remote consent was performed for remaining 6 pts, with blood sample being collected at and shipped from their institution to TEMPUS. Thirteen of 15 (86.6%) pts had been previously enrolled in another pediatric precision oncology program (KiCS (n=10), PROFYLE (n=1), KiCS/PROFYLE (n=2)), which provided sequencing results of tumor tissue-derived DNA. Median turnaround time was 13 days from blood draw to available report (range, 9-27). ctDNA extraction and sequencing led to generation of a research report for 14/15 pts. Biologically relevant variants, as defined by the TEMPUS knowledge database as variants with functional significance or association with disease state, and may have relevant therapeutic, prognostic or diagnostic evidence, were reported for 8/14 (57%), with a median variant allele fraction of 10.6% (range, 0.2%-47.4%). At least one biologically relevant variant was a novel finding for 6 pts (not in previous tumor tissue sequencing (n=5); no previous sequencing (n=1)) and in 3 pts, was deemed therapeutically actionable – variants in PIK3R1, RET, CHEK2 for SK-0010; ALK for SK-0013; PTPN11 for SK-0015. For one pt with ALK p.F1174L, ctDNA sequencing led to a change in therapy in the context of first relapse. Conclusion Using a commercially available liquid biopsy tool within the confines of a research study for pts with relapsed/refractory NBL is feasible in an expansive country like Canada. Remote consent and shipping of blood samples directly from pt’s institution facilitated access from across the country. Enrolment in LIBERTY gave pts timely access to ctDNA next-generation sequencing and led to identification of novel findings with implications for therapy.
 Citation Format: Sarah Cohen-Gogo, Gabriel Revon-Riviere, Jathishinie Jegathisawaran, Adam Shlien, Anita Villani, Meredith Irwin, Daniel A. Morgenstern. LIBERTY: LIquid Biopsy for rElapsed/RefracTorY neuroblastoma - a pilot study of characterization of circulating tumor DNA alterations in patients with relapsed and refractory neuroblastoma using a clinically validated liquid biopsy platform [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A034.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe4ebd557ae7a40899d5d252816419d95ed1e00" target='_blank'>
              Abstract A034 LIBERTY: LIquid Biopsy for rElapsed/RefracTorY neuroblastoma - a pilot study of characterization of circulating tumor DNA alterations in patients with relapsed and refractory neuroblastoma using a clinically validated liquid biopsy platform
              </a>
            </td>
          <td>
            S. Cohen-Gogo, G. Revon-Rivière, J. Jegathisawaran, A. Shlien, A. Villani, Meredith S Irwin, Daniel A. Morgenstern
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09486473d5ae00655afa329978bc6a48b65d161f" target='_blank'>
              The response to influenza vaccination is associated with DNA methylation-driven regulation of T cell innate antiviral pathways
              </a>
            </td>
          <td>
            Hongxiang Fu, H. Pickering, L. Rubbi, Ted M. Ross, Wanding Zhou, Elaine F. Reed, Matteo Pellegrini
          </td>
          <td>2024-08-21</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Alexander Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii-DerIda Chen, C. Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="Abstract CRISPR/Cas9 genome editing is a pervasive research tool due to its relative ease of use. However, some systems are not amenable to generating edited clones due to genomic complexity and/or difficulty in establishing clonal lines. For example, Drosophila Schneider 2 (S2) cells possess a segmental aneuploid genome and are challenging to single-cell select. Here, we describe a streamlined CRISPR/Cas9 methodology for knock-in and knock-out experiments in S2 cells, whereby an antibiotic resistance gene is inserted in-frame with the coding region of a gene-of-interest. By using selectable markers, we have improved the ease and efficiency for the positive selection of null cells using antibiotic selection in feeder layers followed by cell expansion to generate clonal lines. Using this method, we generated the first acentrosomal S2 cell lines by knocking-out centriole genes Polo-like Kinase 4/Plk4 or Ana2 as proof of concept. These strategies for generating gene-edited clonal lines will add to the collection of CRISPR tools available for cultured Drosophila cells by making CRISPR more practical and therefore improving gene function studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d9bec46a5fefb60dc99b1e4c11b9d69666ffa32" target='_blank'>
              Generating CRISPR-edited clonal lines of cultured Drosophila S2 cells
              </a>
            </td>
          <td>
            John M. Ryniawec, Anastasia Amoiroglou, Gregory C. Rogers
          </td>
          <td>2024-08-17</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Next-generation sequencing of samples from patients with acute myeloid leukemia (AML) has revealed several driver gene mutations in adult AML. However, unlike other cancers, AML is defined by relatively few mutations per patient, with a median of 4–5 depending on subtype. In this review, we will discuss the most common driver genes found in patients with AML and focus on the most clinically relevant ones that impact treatment strategies. The most common driver gene mutations in AML occur in NPM1 and FLT3, accounting for ~30% each. There are now targeted therapies being tested or already approved for these driver genes. Menin inhibitors, a novel targeted therapy that blocks the function of the menin protein, are in clinical trials for NPM1 driver gene mutant AML after relapse. A number of FLT3 inhibitors are now approved for FLT3 driver gene mutant AML in combination with chemotherapy in the frontline and also as single agent in relapse. Although mutations in IDH1/2 and TP53 only occur in around 10–20% of patients with AML each, they can affect the treatment strategy due to their association with prognosis and availability of targeted agents. While the impact of other driver gene mutations in AML is recognized, there is a lack of data on the actionable impact of those mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb073fa841295c9a09a75adda3fd8b5c5ace677d" target='_blank'>
              Common Driver Mutations in AML: Biological Impact, Clinical Considerations, and Treatment Strategies
              </a>
            </td>
          <td>
            Tiffany Nong, Shefali Mehra, Justin Taylor
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Despite the success of CRISPR/Cas9 in inducing DNA double-strand breaks (DSBs) for genome editing, achieving targeted recombination in somatic cells remains challenging, particularly at recombination cold spots like the Tomato Mosaic Virus (ToMV) resistance locus in Solanum lycopersicum. We investigated the potential of CRISPR/Cas9-induced targeted recombination in somatic cells to overcome linkage drag surrounding the ToMV locus. We employed two strategies: first, inducing DSBs in both alleles of F1 tomato seedlings to promote non-homologous end joining (NHEJ) and homology-directed repair (HDR); second, targeting a single allele in a heterozygous background to induce HDR in seedlings. CRISPR/Cas9 activity was confirmed in F₁ seedlings by detecting NHEJ-mediated mutations at the target sites in ToMV. We developed a bioinformatics pipeline to identify targeted recombinants by analyzing single nucleotide polymorphisms (SNPs) between parental haplotypes, allowing precise tracking of SNP variations. A two-dimensional pooling strategy was employed to distinguish genuine recombination events from PCR artifacts. Despite these advances and the active CRISPR/Cas9 system in F1 progeny, no increase in recombination frequency was observed compared to wild-type plants. We extended our research to protoplasts to assess whether CRISPR/Cas9 could induce targeted recombination under different cellular conditions at the same locus. Consistent with our findings in F1 plants, we observed no increase in recombinant patterns compared to wild-type controls in protoplasts. Our findings suggest that CRISPR/Cas9-induced DSBs are insufficient to break the genetic linkage at the ToMV locus on chromosome 9 in recombination cold spots within somatic cells. Article Summary This research targets plant biologists and geneticists interested in enhancing plant breeding techniques. The study used CRISPR/Cas9 technology to induce DNA breaks in tomato plants. It specifically targeted the Tomato Mosaic Virus (ToMV) resistance gene, which resists natural recombination. The aim was to induce genetic recombination via CRISPR/Cas9. The highly active CRISPR/Cas9 system did not increase the expected genetic changes, indicating challenges in achieving targeted recombination. These findings highlight the challenges in breaking genetic linkages in specific genome regions using current CRISPR methods. These findings are relevant for developing techniques for targeted recombination in plant breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0b58aaaa05e1e2ef0cf4dbd56fab2aa20c1e65" target='_blank'>
              CRISPR/Cas9-induced breaks are insufficient to break linkage drag surrounding the ToMV locus of Solanum lycopersicum
              </a>
            </td>
          <td>
            Jillis Grubben, Gerard Bijsterbosch, Burak Aktürk, Richard G.F. Visser, Henk J. Schouten
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Mutations in DNA are a known cause of cancer. However, they become oncogenic only under specific cellular conditions and within certain tissue contexts. Tissue injury often leads to the aberrant activation of developmental pathways, eroding markers of cellular differentiation and giving rise to transient cellular states. These states resemble developmental progenitors, characterized by higher transcriptional plasticity. Oncogenes, particularly KRAS, co-opt cellular plasticity by acting on tissue-specific enhancers, preventing the resolution of acinar plasticity and maintaining the pancreas in a pro-inflammatory phase that progresses to cancer1,2. However, the specific role of tissue-specific enhancers in pancreas regeneration and cancer initiation remains unexplored, and strategies to target these regenerative processes are yet to be developed. Our laboratory investigates non-mutational mechanisms of cancer, with a special interest in the intersection of developmental biology and cancer, aiming to elucidate the processes of cellular plasticity and cancer initiation. Recent studies have shown that acinar cells possess remarkable cellular plasticity, giving rise to a diverse population of cellular states in pancreas regeneration and cancer contexts3-5. We have discovered that the transcription factor Sox4 is necessary for the specification of tuft cells and gastric metaplasia found in chronic pancreatitis and in pancreatic intraepithelial lesions (PanINs). Moreover, we have shown that cellular plasticity has a non-cell autonomous protective effect; losing Sox4 and cellular heterogeneity in PanINs results in a fibrotic and immunosuppressive tumor microenvironment and disease progression. Our results demonstrated that targeting cellular plasticity accelerates or prevents cancer initiation through autonomous and non-autonomous cellular processes6. In this study, we leverage a comprehensive resource of ncRNAs transcribed from enhancer elements of pancreas development, together with their inferred target genes and correlations with clinical outcomes. We focused on an ncRNA associated with the risk of pancreatic cancer. To study its function, we developed two novel mouse strains for tissue-specific deletion, enabling precise manipulation and study of its role in the pancreas and other tissues. Our phenotypic analysis demonstrates that the ncRNA is activated following tissue injury and is essential for developing PanINs. Moreover, we show that it regulates cell-cell communication after tissue injury. This mechanism operates independently of cellular plasticity but significantly creates a pro-inflammatory environment that favors cancer development. Our results demonstrate that enhancer elements are actionable targets to regulate cell fate decisions in regeneration and cancer. Additionally, KRAS-induced cancer initiation depends on developmental regulatory elements, opening the window to develop prophylactic interventions to delay tumor onset.
 Citation Format: Luis Arnes. Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/698dc456f639e681131533a5b64aac4181ad1d22" target='_blank'>
              Abstract C059: Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Luis Arnes
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2dde0164af0025d5f19bf691294f177add4f7b" target='_blank'>
              Low-pass whole genome sequencing of circulating tumor cells to evaluate chromosomal instability in triple-negative breast cancer
              </a>
            </td>
          <td>
            S. Di Cosimo, M. Silvestri, C. De Marco, Alessia Calzoni, M. C. De Santis, Maria Grazia Carnevale, C. Reduzzi, Massimo Cristofanilli, Vera Cappelletti
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, , Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2235e4530843ef03fa58ea8c3a36f0f96aa2515f" target='_blank'>
              miR-149-3p Enhances Drug Sensitivity of AML Cells by Inhibiting Warburg Effect Through PI3K/AKT Pathway.
              </a>
            </td>
          <td>
            Xi Chen, Yan Song, Yaoyao Tian, Xiushuai Dong, Yuying Chang, Wei Wang
          </td>
          <td>2024-08-17</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G. Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f88ec1cfcf4f474ca746d0bea5d45f08c037f3cd" target='_blank'>
              LIM-domain-only 4 (LMO4) enhances CD8+ T-cell stemness and tumor rejection by boosting IL-21-STAT3 signaling
              </a>
            </td>
          <td>
            R. Schelker, J. Fioravanti, Fabio Mastrogiovanni, Jeremy G. Baldwin, Nisha Rana, Peng Li, Ping Chen, Timea Vadász, Rosanne Spolski, Christoph Heuser-Loy, Dragana Slavkovic-Lukic, Pedro Noronha, Giuseppe Damiano, Laura Raccosta, Daniela Maggioni, Sree Pullugula, Jian-Xin Lin, Jangsuk Oh, Patrick Grandinetti, Mario Lecce, Leo Hesse, Emilia Kocks, Azucena Martín-Santos, C. Gebhard, William Telford, Yun Ji, Nicholas P Restifo, Vincenzo Russo, Michael Rehli, Wolfgang Herr, Warren J. Leonard, Luca Gattinoni
          </td>
          <td>2024-08-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Juvenile myelomonocytic leukemia (JMML), a clonal hematologic malignancy, originates from mutated hematopoietic stem cells (HSCs). The mechanism sustaining the persistence of mutant stem cells, leading to leukemia development, remains elusive. In this study, we conducted comprehensive examination of gene expression profiles, transcriptional factor regulons, and cell compositions/interactions throughout various stages of tumor cell development in Ptpn11 mutation-associated JMML. Our analyses revealed that leukemia-initiating Ptpn11E76K/+ mutant stem cells exhibited de novo activation of the myeloid transcriptional program and aberrant developmental trajectories. These mutant stem cells displayed significantly elevated expression of innate immunity-associated anti-microbial peptides and pro-inflammatory proteins, particularly S100a9 and S100a8. Biological experiments confirmed that S100a9/S100a8 conferred a selective advantage to the leukemia-initiating cells through autocrine effects and facilitated immune evasion by recruiting and promoting immune suppressive myeloid-derived suppressor cells (MDSCs) in the microenvironment. Importantly, pharmacological inhibition of S100a9/S100a8 signaling effectively impeded leukemia development from Ptpn11E76K/+ mutant stem cells. These findings collectively suggest that JMML tumor-initiating cells exploit evolutionarily conserved innate immune and inflammatory mechanisms to establish clonal dominance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e14ee0ff31f4868484243a76f8cf02c1e5b54c52" target='_blank'>
              Prototypical innate immune mechanism hijacked by leukemia-initiating mutant stem cells for selective advantage and immune evasion in Ptpn11-associated juvenile myelomonocytic leukemia
              </a>
            </td>
          <td>
            Cheng-Kui Qu, Hong Zheng, Peng Zhao, Zhenya Tan, Wen‐mei Yu, Juwita Werner, Elliot Stieglitz, Christopher C Porter, S. Chandrakasan, Daniel Wechsler, Simon Mendez-Ferrer
          </td>
          <td>2024-08-02</td>
          <td>Research Square</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="In the ever-evolving landscape of oncology, the battle against prostate cancer (PCa) stands at a transformative juncture, propelled by the integration of molecular diagnostics into traditional cytopathological frameworks. This synthesis not only heralds a new epoch of precision medicine but also significantly enhances our understanding of the disease’s genetic intricacies. Our comprehensive review navigates through the latest advancements in molecular biomarkers and their detection technologies, illuminating the potential these innovations hold for the clinical realm. With PCa persisting as one of the most common malignancies among men globally, the quest for early and precise diagnostic methods has never been more critical. The spotlight in this endeavor shines on the molecular diagnostics that reveal the genetic underpinnings of PCa, offering insights into its onset, progression, and resistance to conventional therapies. Among the genetic aberrations, the TMPRSS2-ERG fusion and mutations in genes such as phosphatase and tensin homolog (PTEN) and myelocytomatosis viral oncogene homolog (MYC) are identified as significant players in the disease’s pathology, providing not only diagnostic markers but also potential therapeutic targets. This review underscores a multimodal diagnostic approach, merging molecular diagnostics with cytopathology, as a cornerstone in managing PCa effectively. This strategy promises a future where treatment is not only tailored to the individual’s genetic makeup but also anticipates the disease’s trajectory, offering hope for improved prognosis and quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/040ce4884f4e6bdcb8041b33d1c1868319f07888" target='_blank'>
              From microscopes to molecules: The evolution of prostate cancer diagnostics
              </a>
            </td>
          <td>
            Junyue Tao, Xiaokang Bian, Jun Zhou, Meng Zhang
          </td>
          <td>2024-08-29</td>
          <td>Cytojournal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Non-coding regulatory sequences play essential roles in adjusting gene output to cellular needs and are thus critical to animal development and health. Numerous such sequences have been identified in mammalian genomes ranging from transcription factors binding motifs to recognition sites for RNA-binding proteins and non-coding RNAs. The advent of CRISPR has raised the possibility of assigning functionality to individual endogenous regulatory sites by facilitating the generation of isogenic cell lines that differ by a defined set of genetic modifications. Here we investigate the usefulness of this approach to assign function to individual miRNA binding sites. We find that the process of generating isogenic pairs of mammalian cell lines with CRISPR-mediated mutations introduces extensive molecular and phenotypic variability between biological replicates making any attempt of assigning function to the binding site essentially impossible. Our work highlights an important consideration when employing CRISPR editing to characterize non-coding regulatory sequences in cell lines and calls for the development and adoption of alternative strategies to address this question in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1d3a055f759b974342766678fe87bc5f92b153" target='_blank'>
              CRISPR-based dissection of miRNA binding sites using isogenic cell lines is hampered by pervasive noise
              </a>
            </td>
          <td>
            Mahendra Prajapat, Joana A. Vidigal
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Elucidating the genetic architecture of DNA methylation (DNAm) is crucial for decoding the etiology of complex diseases. However, current epigenomic studies often suffer from incomplete coverage of methylation sites and the use of tissues containing heterogeneous cell populations. To address these challenges, we present a comprehensive human methylome atlas based on deep whole-genome bisulfite sequencing (WGBS) and whole-genome sequencing (WGS) of purified monocytes from 298 European Americans (EA) and 160 African Americans (AA) in the Louisiana Osteoporosis Study. Our atlas enables the analysis of over 25 million DNAm sites. We identified 1,383,250 and 1,721,167 methylation quantitative trait loci (meQTLs) in cis-regions for EA and AA populations, respectively, with 880,108 sites shared between ancestries. While cis-meQTLs exhibited population-specific patterns, primarily due to differences in minor allele frequencies, shared cis-meQTLs showed high concordance across ancestries. Notably, cis-heritability estimates revealed significantly higher mean values in the AA population (0.09) compared to the EA population (0.04). Furthermore, we developed population-specific DNAm imputation models using Elastic Net, enabling methylome-wide association studies (MWAS) for 1,976,046 and 2,657,581 methylation sites in EA and AA, respectively. The performance of our MWAS models was validated through a systematic multi-ancestry analysis of 41 complex traits from the Million Veteran Program. Our findings bridge the gap between genomics and the monocyte methylome, uncovering novel methylation-phenotype associations and their transferability across diverse ancestries. The identified meQTLs, MWAS models, and data resources are freely available at www.gcbhub.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02561d4c021777b32cc89ad698157b2f333e71dc" target='_blank'>
              An atlas of genetic effects on the monocyte methylome across European and African populations
              </a>
            </td>
          <td>
            Wanheng Zhang, Xiao Zhang, Chuan Qiu, Zichen Zhang, Kuan-Jui Su, Zhe Luo, Minghui Liu, Bingxin Zhao, Lang Wu, Qing Tian, Hui Shen, Chong Wu, Hong-Wen Deng
          </td>
          <td>2024-08-13</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Human inborn errors of immunity (IEI) represent a diverse group of genetic disorders affecting the innate and/or adaptive immune system. Some IEI entities comprise defects in DNA repair factors, resulting in (severe) combined immunodeficiencies, bone marrow failure, predisposition to malignancies, and potentially result in radiosensitivity (RS). While other IEI subcategories such as common variable immunodeficiency (CVID) and immune dysregulation disorders also associate with lymphoproliferative and malignant complications, the occurrence of RS phenotypes in the broader IEI population is not well characterized. Nonetheless, identifying RS in IEI patients through functional testing is crucial to reconsider radiation-related therapeutic protocols and to improve overall patient management. This study aimed to investigate chromosomal RS in a diverse cohort of 107 IEI patients using the G0 cytokinesis-block micronucleus (MN) assay. Our findings indicate significant variability in RS across specific genetic and phenotypical subgroups. Severe RS was detected in all ataxia-telangiectasia (AT) patients, a FANCI deficient and ERCC6L2 deficient patient, but not in any other IEI patient included in this cohort. Age emerged as the single influencing factor for both spontaneous and radiation-induced MN yields, while the manifestation of additional clinical features, including infection susceptibility, immune dysregulation, or malignancies did not associate with increased MN levels. Our extensive analysis of RS in the IEI population underscores the clinical importance of RS assessment in AT patients and supports RS testing in all IEI patients suspected of having a DNA repair disorder associated with radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b4f50595c5c29676228d24464dde5631290eb13" target='_blank'>
              Investigating chromosomal radiosensitivity in inborn errors of immunity: insights from DNA repair disorders and beyond.
              </a>
            </td>
          <td>
            E. Beyls, Evi Duthoo, Lynn Backers, Karlien Claes, Marieke De Bruyne RAPID clinicians, L. Pottie, Victoria Bordon, C. Bonroy, S. Tavernier, Kathleen BM Claes, A. Vral, A. Baeyens, F. Haerynck, T. Kerre, B. Lambrecht, L. Hoste, B. Meertens, T. V. Genechten, A. Ferster, K. V. Driessche, C. D. Vriendt, L. Dedeken, I. Moors, C. Dhooge, L. Kornreich, C. Heijmans, M. L. Roux, M. Colard, A. V. Damme, K. V. Schil, B. D. Wilde
          </td>
          <td>2024-08-20</td>
          <td>None</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62121140d30406add5d3e63d73916661305bd4d7" target='_blank'>
              Comprehensive genetic analysis by targeted sequencing identifies risk factors and predicts patient outcome in Mantle Cell Lymphoma: results from the EU-MCL network trials.
              </a>
            </td>
          <td>
            M. Khouja, Linmiao Jiang, Karol Pál, P. J. Stewart, Binaya Regmi, Martin Schwarz, Wolfram Klapper, Stefan K Alig, N. Darzentas, H. Kluin-Nelemans, Olivier Hermine, M. Dreyling, David Gonzalez de Castro, E. Hoster, Christiane Pott
          </td>
          <td>2024-09-16</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>83</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/431b8d9fdd349b2296cea4bbf8173c562f2afaeb" target='_blank'>
              Comprehensive molecular profiling of multiple myeloma identifies refined copy number and expression subtypes
              </a>
            </td>
          <td>
            Sheri Skerget, Daniel A. Peñaherrera, A. Chari, Sundar Jagannath, David Siegel, Ravi Vij, G. Orloff, A. Jakubowiak, R. Niesvizky, Darla Liles, J. Berdeja, Moshe Levy, J. Wolf, Saad Z Usmani, Austin Christofferson, Sara Nasser, J. Aldrich, Christophe Legendre, Brooks A. Benard, Chase Miller, Bryce Turner, Ahmet A. Kurdoglu, M. Washington, Venkata Yellapantula, J. Adkins, Lori Cuyugan, Martin Boateng, A. Helland, Shari Kyman, Jackie McDonald, Rebecca Reiman, Kristi Stephenson, Erica E Tassone, A. Blanski, Brianne Livermore, Meghan Kirchhoff, D. Rohrer, M. D'Agostino, Manuela Gamella, K. Collison, Jennifer Stumph, Pam Kidd, A. Donnelly, B. Zaugg, Maureen Toone, K. McBride, M. Derome, Jennifer Rogers, David Craig, Winnie S Liang, Norma C. Gutiérrez, Scott D. Jewell, J. Carpten, Kenneth C Anderson, H. J. Cho, D. Auclair, S. Lonial, J. Keats
          </td>
          <td>2024-08-19</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>107</td>
        </tr>

        <tr id="Abstract Disruptions in normal development and the emergence of health conditions often result from the malfunction of vital genes in the human body. Decades of scientific research have focused on techniques to modify or substitute defective genes with healthy alternatives, marking a new era in disease treatment, prevention and cure. Recent strides in science and technology have reshaped our understanding of disorders, medication development and treatment recommendations, with human gene and cell therapy at the forefront of this transformative shift. Its primary objective is the modification of genes or adjustment of cell behaviour for therapeutic purposes. In this review, we focus on the latest advances in gene and cell therapy for treating human genetic diseases, with a particular emphasis on FDA and EMA‐approved therapies and the evolving landscape of genome editing. We examine the current state of innovative gene editing technologies, particularly the CRISPR‐Cas systems. As we explore the progress, ethical considerations and prospects of these innovations, we gain insight into their potential to revolutionize the treatment of genetic diseases, along with a discussion of the challenges associated with their regulatory pathways. This review traces the origins and evolution of these therapies, from conceptual ideas to practical clinical applications, marking a significant milestone in the field of medical science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/605617113766e025c90c18b7b7452f93c13a0842" target='_blank'>
              Gene and cell therapy of human genetic diseases: Recent advances and future directions
              </a>
            </td>
          <td>
            Busra Cetin, Fulya Erendor, Y. E. Eksi, A. Şanlioğlu, S. Sanlioglu
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2b8937040c82aaaf4474d35899e8a2ae8efb870" target='_blank'>
              Genomic, epigenomic and transcriptomic landscape of glioblastoma.
              </a>
            </td>
          <td>
            T. C. Dakal, Ganesh S. Kakde, Pawan Kumar Maurya
          </td>
          <td>2024-08-24</td>
          <td>Metabolic brain disease</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77653d072a56bf5996520a64df22766fb41edd0" target='_blank'>
              CDC7 inhibition impairs neuroendocrine transformation in lung and prostate tumors through MYC degradation
              </a>
            </td>
          <td>
            Á. Quintanal-Villalonga, Kenta Kawasaki, E. Redin, F. Uddin, Swanand Rakhade, Vidushi Durani, Amin Sabet, Moniquetta Shafer, W. Karthaus, S. Zaidi, Y. Zhan, P. Manoj, Harsha Sridhar, Dennis Kinyua, Hong Zhong, Barbara P. Mello, M. Ciampricotti, Umeshkumar K. Bhanot, I. Linkov, Juan Qiu, Radhika A Patel, C. Morrissey, Sanjoy Mehta, Jesse Barnes, Michael C. Haffner, N. Socci, R. Koche, E. de Stanchina, S. Molina-Pinelo, Sohrab Salehi, Helena A Yu, J. Chan, Charles M. Rudin
          </td>
          <td>2024-07-26</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>94</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Personalized medicine programs, including the Zero Childhood Cancer Program (ZERO), perform molecular analysis of individual patient tumours to identify molecular targets for therapy. The comprehensive and unbiased sequencing approaches used by ZERO, including whole genome sequencing (WGS) and RNA sequencing (RNAseq), enables detection of novel structural variants (SVs) that may be cryptic to standard cytogenetic techniques or panel sequencing approaches. Identification of Receptor Tyrosine Kinase (RTK) activating variants in paediatric cancers, for example NTRK2 gene fusions, is a high priority given the availability and remarkable clinical success of RTK inhibitors. Here we describe a novel internal tandem duplication (ITD) in NTRK2, encoding TRKB, in a ZERO patient with CNS neuroblastoma. This SV was detected by WGS and was notably missed by RNAseq-based gene fusion and SV detection. The ITD spans exons 10-13 of NTRK2, is in-frame, and includes the juxtamembrane and transmembrane protein domains. We hypothesized that this ITD would result in constitutive activation of TRKB and would be sensitive to TRK-targeted therapies. We overexpressed NTRK2 ITD, wild-type NTRK2, and SPECC1L-NTRK2 (an established TRK-activating fusion), in the interleukin-3 (IL-3)-dependent cell line, Ba/F3, and the neuroblastoma cell line, SH-SY5Y. In these models, we showed that like wild-type NTRK2, NTRK2 ITD was expressed on the cell surface. NTRK2 ITD was sufficient to transform Ba/F3 cells to IL-3 independence through constitutive activation of TRKB and downstream signaling pathways PI3K/AKT and MEK/ERK. NTRK2 ITD-expressing cells were sensitive to TRK inhibitors, including larotrectinib, at similar doses to SPECC1L-NTRK2-expressing cells. Interestingly, we observed that cells expressing NTRK2 ITD were specifically sensitive to MEK inhibition, which was not observed in SPECC1L-NTRK2-expressing cells, suggesting the mechanisms by which these variants drive transformation is different. Indeed, in silico structural analysis showed that the duplicated region of NTRK2 ITD contains two internalized tyrosine residues whose phosphorylation could both drive autophosphorylation of the receptor and act as docking sites for adaptor proteins. Mutation of these tyrosine residues delayed transformation of Ba/F3 cells. This study functionally characterizes a novel NTRK2 ITD and shows that this variant is transforming and sensitive to TRK inhibition. While ITDs have been described in other RTK genes, notably FGFR1 and FLT3, structurally these have involved the kinase domain or the juxtamembrane domain. This is the first report of an ITD that spans the transmembrane domain of an RTK, characterizing an additional mechanism by which RTKs can be activated in cancer. This study highlights the value of unbiased WGS approaches to identify novel SVs and identifies another class of patients that may benefit from RTK inhibitor therapy.
 Citation Format: Lauren M. Brown, Gabor Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah Jalud, Teresa Sadras, Pamela Ajuyah, Paulette Barahona, M. Emmy M. Dolman, Marie Wong, Chelsea Mayoh, Mark J. Cowley, Paul G. Ekert. A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ae609e3759161918ab8221b777b148ec5605447" target='_blank'>
              Abstract B002: A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation
              </a>
            </td>
          <td>
            Lauren M. Brown, G. Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah B Jalud, Teresa Sadras, P. Ajuyah, P. Barahona, M. E. M. Dolman, Marie Wong, C. Mayoh, Mark J. Cowley, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Treatment of acute leukemia is gradually moving away from a "one-size-fits-all" approach, as scientific and clinical advances expand the arsenal of available targeted therapies. One of the recent additions are the menin inhibitors; oral, selective, small molecules that disrupt the interaction between the chromatin adapter menin, and an epigenetic regulator, the Lysine Methyltransferase 2A (KMT2A) complex. Two susceptible leukemia subtypes have been identified: 1) Acute Myeloid Leukemia (AML) with a mutation in Nucleophosmin 1 (NPM1), and 2) any acute leukemia, myeloid or lymphoid, with a translocation resulting in the rearrangement of KMT2A. These leukemias share a distinct genetic expression, maintained by the KMT2A-menin interaction. Together they account for approximately 40% of patients with AML, and 10% of patients with Acute Lymphoblastic Leukemia (ALL). This review follows the journey of revumenib, as a representative of menin inhibitors, from bench to bedside. It will focus on the pathophysiology of leukemias sensitive to menin inhibition, delineation of how this understanding led to targeted drug development, and data from clinical trials. The important discovery of resistance mechanisms will also be explored, as well as future directions in using menin inhibitors for treating leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/323b377f2a21066c86ab2fc4983bf5a1e9c606c6" target='_blank'>
              Revumenib for patients with acute leukemia: a new tool for differentiation therapy.
              </a>
            </td>
          <td>
            Meira Yisraeli Salman, E. Stein
          </td>
          <td>2024-08-01</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Gene expression patterns are very sensitive to external influences and are reflected in phenotypic changes. It was previously described that transferring melanoma cells from a plastic surface to Matrigel led to formation of de novo vascular networks—vasculogenic mimicry—that are characteristic to a stemness phenotype in aggressive tumors. Up to now there was no detailed data about the gene signature accompanying this process. Here, we show that this transfer shortly led to extremely strong epigenetic changes in gene expression in the melanoma cells. We observed that on Matrigel numerous genes controlling ribosome biogenesis were upregulated. However, most of the activated genes were inhibitors of the differentiation genes (ID1, ID2, and ID3). At the same time, the genes that control differentiation were downregulated. Both the upregulated and the downregulated genes are simultaneously targeted by different transcription factors shaping sets of co-expressed genes. The specific group of downregulated genes shaping contacts with rDNA genes are also associated with the H3K27me3 mark and with numerous lincRNAs and miRNAs. We conclude that the stemness phenotype of melanoma cells is due to the downregulation of developmental genes and formation of dedifferentiated cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e3983f49ac992e0e94ffb5e7e55f9651ff1ef6" target='_blank'>
              Strong Activation of ID1, ID2, and ID3 Genes Is Coupled with the Formation of Vasculogenic Mimicry Phenotype in Melanoma Cells
              </a>
            </td>
          <td>
            N. Tchurikov, Amalia A. Vartanian, E. S. Klushevskaya, I. R. Alembekov, A. N. Kretova, Viktoriya N. Lukicheva, V. Chechetkin, G. I. Kravatskaya, V. S. Kosorukov, Y. V. Kravatsky
          </td>
          <td>2024-08-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="
 Children born with Down syndrome (DS), a genetic condition caused by the presence of an additional chromosome 21 (trisomy 21 or T21), have a substantially higher risk of developing childhood leukaemia. Approximately 30% of DS infants develop a preleukemic condition called transient abnormal myelopoiesis (TAM). TAM is strictly associated with GATA1 truncating mutations on a T21 background. Most TAM cases spontaneously resolve within the first few months of life. However, by the age of five, approximately 10% of cases progress to megakaryoblastic/erythroid leukaemia, known as myeloid leukaemia of DS (ML-DS). This final step of transformation is often driven by specific third-hit somatic mutations, most of which affects genes encoding JAK kinases, cohesin complexes, or epigenetic regulators. These well-defined stepwise genetic aberrations associated with ML-DS pathogenesis provide a unique and highly tractable system to elucidate the molecular changes underlying leukaemogenesis. Using single-cell mRNA sequencing of primary human samples, we directly interrogate the transcriptional changes underpinning the multi-step pathogenesis of ML-DS. We observe an expansion of megakaryocyte-erythrocyte progenitors (MEPs) specifically in T21 foetal liver compared to diploid or other trisomies. However, it is the GATA1 mutation that accounts for the majority of the transcriptomic changes towards that of the leukaemic blasts. Furthermore, investigation of an aggressive TAM case reveals an enrichment in expression of the leukaemogenic gene module associated with the full-blown leukaemia state ML-DS. We also demonstrate that TAM and ML-DS blasts still retain transcriptional signatures of normal haematopoiesis, showing heterogeneous features of differentiation towards erythrocytes, megakaryocytes, and mast cells. Finally, by integrating genomic and transcriptomic data, we directly defined the genomic evolution, and its corresponding transcriptional consequences, that underpins a treatment-refractory case of ML-DS. Overall, we have generated the first comprehensive single-cell mRNA atlas of TAM / ML-DS, enabling unbiased quantitative characterisation of the transcriptional transformation along ML-DS pathogenesis. Our study explores the molecular signature associated with progression of TAM to ML-DS, as well as the mechanism underlying differences in treatment response. These insights offer the possibility of more meaningful clinical interpretations and potential to explore novel therapeutic targets for these conditions.
 Citation Format: Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, Angus Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, Jan-Henning Klusmann, Jack Bartram, Sam Behjati. Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f4dde5c53348eeefdb9808ec633733f445cffc" target='_blank'>
              Abstract A018 Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome
              </a>
            </td>
          <td>
            Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, A. Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, J. Klusmann, Jack Bartram, Sam Behjati
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f0f651c9fac9c8c40b22fc5b21b4578646df27" target='_blank'>
              Multi-omics features of immunogenic cell death in gastric cancer identified by combining single-cell sequencing analysis and machine learning
              </a>
            </td>
          <td>
            Shu-Long Dai, Jian-Qiang Pan, Zhen-Rong Su
          </td>
          <td>2024-09-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a8245d23a6d26f49afe8896fd7ee25b1dbf95a8d" target='_blank'>
              Tailored antisense oligonucleotides designed to correct aberrant splicing reveal actionable groups of mutations for rare genetic disorders
              </a>
            </td>
          <td>
            H. Wai, Eliska Svobodova, Natalia Romero Herrera, Andrew G L Douglas, John W Holloway, F. Baralle, Marco Baralle, D. Baralle
          </td>
          <td>2024-08-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction: The role of genetic variants in response to chemotherapy has been investigated in several studies. This study aimed to investigate genetic variants associated with response to chemotherapy in breast cancer (BC) patients. Methods: Significant variants (p < 5 × 10-8) associated with response to chemotherapy were obtained from GWA studies. Candidate variants were identified by haplotype analysis (r2 ≥ 0.9, D'≥0.9) using 1000Genome LD data. To determine the effects of the variants on gene expression, expression quantitative trait loci (eQTL) were evaluated. To compare the expression of the identified genes in tumor samples, expression levels were compared between TCGA tumor types and adjacent normal tissues. Results: Six rs3820706, rs147451859, rs4784750, rs17587029, rs16830728, and rs16972207 variants were significantly associated with response to chemotherapy in BC patients (p < 5 × 10-8). Seven novel haplotypic structures were identified to be associated with adverse response to chemotherapy in BC patients. These haplotypes formed two genetic structures associated with neutropenia, leukopenia, chemotherapy-induced cytotoxicity (GAG-TTAT), and chemotherapy-induced alopecia (CC-CAACTCCCGTTGCGG). These variants are located on PPCDC, NLRC5, STAM2, and TNFSF13B genes, and the expression of these genes significantly changed in BC tissues than normal tissues (P ≤ 0.05), also showing gene-gene correlation (P ≤ 0.05). Conclusions: These genetic variants and their associated novel haplotypic structures can predict adverse response to chemotherapy in BC patients and could potentially form BC-associated genetic panel for adverse response to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f6f813884867edf6f2b79ca7de2ac3b63826948" target='_blank'>
              Novel genetic structures associated with adverse response to chemotherapy in breast cancer.
              </a>
            </td>
          <td>
            Morteza Gholami, Mohsen Asouri, A. Ahmadi, Mehrab Nasirikenari
          </td>
          <td>2024-08-28</td>
          <td>Journal of oncology pharmacy practice : official publication of the International Society of Oncology Pharmacy Practitioners</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c25293ccda82c6d001fe5bae96c805b5e1fa7f9a" target='_blank'>
              The role of DNA polymerase I in tolerating single-strand breaks generated at clustered DNA damage in Escherichia coli
              </a>
            </td>
          <td>
            N. Shikazono, Ken Akamatsu
          </td>
          <td>2024-08-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Near-haploidization, that is, loss of one copy of most chromosomes, is a relatively rare phenomenon in most tumors, but is enriched among certain soft tissue sarcomas, including undifferentiated pleomorphic sarcoma (UPS). Presumably, near-haploidization can arise through many mechanisms. This study aimed to identify gene rearrangements that could cause near-haploidization. We here present two UPS in which near-haploidization was an early event, identified through single nucleotide polymorphism (SNP) array analysis. One of the cases was studied further using whole genome and transcriptome sequencing, as well as cytogenetic and molecular cytogenetic methods. Both tumors had chromosomal rearrangements in the form of copy number shifts/structural variants affecting the SMC1A gene. These findings suggest that cohesin defects could contribute to mitotic errors resulting in massive loss of chromosomes. SMC1A encodes one of the components of the cohesin multiprotein complex, which is critical for proper alignment of the sister chromatids during S-phase and separation to opposite spindle poles. Further studies should explore the role of cohesin defects in near-haploidization in other sarcomas and to clarify its role in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac46389944e17adf0a38d1c92f41322a72ad7ff" target='_blank'>
              Structural Variants in the SMC1A Gene Associated With Near-Haploidy in Undifferentiated Pleomorphic Sarcomas.
              </a>
            </td>
          <td>
            Sebastian Ibstedt, Paul Piccinelli, Saskia Sydow, Jan Köster, Fredrik Mertens
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dca29cb8a3526737bad31e766fb17c818ee6f4c" target='_blank'>
              Significance of targeting DNMT3A mutations in AML.
              </a>
            </td>
          <td>
            Guiqin Huang, Xiaoya Cai, Dengju Li
          </td>
          <td>2024-07-30</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8e7718e7c250fe926c8cdf48ae45f72605c99d8" target='_blank'>
              REXO5 promotes genomic integrity through regulating R-loop using its exonuclease activity.
              </a>
            </td>
          <td>
            Ye Jin Lee, Seo Yun Lee, Soomi Kim, Soomi Kim, Soo Hyeon Lee, Sungho Park, Jae Jin Kim, Dong-Wook Kim, Hongtae Kim
          </td>
          <td>2024-07-30</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Alterations in DNA damage repair genes in advanced prostate cancer (PC) may impact responses to therapy and clinical outcomes. This study described homologous recombination repair (HRR) testing patterns and clinical outcomes among patients with metastatic castration-resistant prostate cancer (mCRPC) by HRR alteration status and race in the United States (US). Methods Clinical data in the nationwide (US-based) Flatiron Health-Foundation Medicine, Inc. (FMI) Metastatic PC Clinico-Genomic Database were evaluated (01/01/2011–12/31/2022). Patients initiating first-line (1L) mCRPC therapy on or after mCRPC diagnosis were included. Testing patterns, time-to-next treatment, overall survival (OS), and time-to-prostate specific antigen response were described. Results Of the 1367 patients with mCRPC and at least one HRR panel test prior to or on the date of 1L mCRPC therapy initiation, 332 (24.3%) were HRR positive (White patients: n = 219 [66.0%]; Black patients: n = 37 [11.1%]) and 1035 (75.7%) were HRR negative (White patients: n = 702 [67.8%]; Black patients: n = 84 [8.1%]). The mean time between first positive test and 1L mCRPC therapy initiation date was 588 days (White patients: 589 days; Black patients: 639 days). Among HRR positive relative to negative patients, trends for faster progression (respective 12-month rate overall: 71.1% and 63.7%; White patients: 72.5% and 64.0%; Black patients: 65.4% and 56.4%), shorter OS (respective 24-month rate overall: 46.8% and 51.9%; White patients: 48.6% and 46.2%; Black patients: 52.8% and 54.1%), and decreased treatment response (respective 12-month rate overall: 24.3% and 37.9%; White patients: 24.5% and 35.2%; Black patients: 17.0% and 43.9%) were observed. Conclusion Patients with mCRPC positive for HRR alterations tended to exhibit poorer treatment responses and clinical outcomes than those with a negative status. These findings highlight the importance of timely genetic testing in mCRPC, particularly among Black patients, and the need for improved 1L targeted therapies to address the unmet need in HRR positive mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b5b000d1e3f13e0b5a22c9691a361f7dc5edd2" target='_blank'>
              Homologous Recombination Repair Testing Patterns and Outcomes in mCRPC by Alteration Status and Race
              </a>
            </td>
          <td>
            M. Bilen, I. Khilfeh, C. Rossi, Erik Muser, L. Morrison, A. Hilts, Lilian Diaz, Patrick Lefebvre, D. Pilon, Daniel George
          </td>
          <td>2024-09-01</td>
          <td>ClinicoEconomics and Outcomes Research: CEOR</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Minor introns constitute 0.4% of all introns in human cells, but they are unique in their ability to regulate the genes in which they are embedded such that their low splicing efficiency can be a rate limiting step in gene expression. The first intron of the tumor suppressor gene, PTEN, has been documented to be a minor intron, but very little is known about its regulation. Regulation of PTEN levels in cancer cells, especially breast cancer is tightly controlled as a very small reduction in its expression can lead to tumorigenesis. Indeed, many genetic, epigenetic, post-transcriptional and post-translational mechanisms are employed to reduce PTEN levels or activities, leading to cancer. Here, we uncover a previously unexplored mechanism for modulating PTEN expression utilizing its minor intron. The minor intron of PTEN minor intron has one of the lowest splicing efficacies in breast cancer cells, causing at least 50% of PTEN pre-mRNA to retain it, and thus not produce a functional protein. We also show that, unlike other minor introns, the retained intron in PTEN pre-mRNA is not used as a molecular switch that would be spliced out when needed but is rather processed by premature cleavage and polyadenylation into a long noncoding RNA (lncRNA) that we termed PINC. Exogenous expression of PINC caused significant alterations to cellular proliferation, including breast cancer cells that harbor a genetic mutation in the PTEN gene and do not produce a functional PTEN protein. This shows that this novel lncRNA functions independently of the encoded protein of the host gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1318f9f43dd61e458c57076f614249c11e1f55c5" target='_blank'>
              Premature cleavage and polyadenylation in the minor intron of PTEN modulate its expression and generates a functional long non-coding RNA in breast cancer
              </a>
            </td>
          <td>
            Mariam Elesnawy, Rim Elghandour, Hana Hasna, Aisha Fakhroo, Boshra Al-Sulaiti, Ihab Younis
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/315c73079e1d95c257190420cacdda2569b17936" target='_blank'>
              RNA Editing by ADAR Adenosine Deaminases in the Cell Models of CAG Repeat Expansion Diseases: Significant Effect of Differentiation from Stem Cells into Brain Organoids in the Absence of Substantial Influence of CAG Repeats on the Level of Editing.
              </a>
            </td>
          <td>
            Viacheslav V. Kudriavskii, A. O. Goncharov, A. V. Eremeev, Evgenii S. Ruchko, V. Veselovsky, Ksenia M Klimina, A. Bogomazova, M. Lagarkova, Sergei A. Moshkovskii, A. A. Kliuchnikova
          </td>
          <td>2024-08-01</td>
          <td>Biochemistry. Biokhimiia</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Genetic or epigenetic variations in regulatory enhancer elements increase susceptibility to a range of pathologies, including pancreas cancer. Pinpointing genes affecting pancreas cancer risk holds promise for early detection, prevention, and effective therapies. Despite recent advances, linking enhancer elements to target genes and predicting the transcriptional outcome of enhancer dysfunction remain significant challenges. Using 3D chromatin assays, we generated an extensive enhancer interaction dataset for the human pancreas, spanning more than 20 donors and five major cell types, including both the exocrine and endocrine compartments. We employed a network approach to parse chromatin interactions into enhancer-promoter tree models, facilitating quantitative, genome-wide analysis of enhancer connectivity. Using the tree models, we developed a machine learning algorithm capable of estimating the impact of enhancer perturbations on cell-type specific gene expression in the human pancreas. Complementing this computational approach, we streamlined an experimental platform using CRISPR-interference and RNA-FISH coupled with high-throughput imaging to quantitatively measure the effect size of enhancer perturbations in primary human pancreas cells. Our enhancer tree models enabled functional annotation of genetic variants associated with pancreas diseases, revealing novel target genes in specific cell types. For pancreas cancer (PDAC), our data suggests a stronger association of disease susceptibility variants with acinar cells, even though ductal cells historically have been the focus of PDAC. This discovery linked three potentially functional variants associated with PDAC risk to acinar cell enhancers, revealing novel target genes to further investigation. By integrating genomics with mechanistic findings into enhancer function, our work offers a framework for understanding genetic basis of pancreas cancer disease risk.
 Citation Format: Li Wang, Songjoon Baek, Gauri Prasad, Thucnhi Truongvo, Jason Hoskins, Laufey Amundadottir, Efsun Arda. A novel approach to understanding pancreas cancer risk through enhancer- promoter interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f97224f03d537588713402ad00fc72aae371ba05" target='_blank'>
              Abstract C058: A novel approach to understanding pancreas cancer risk through enhancer- promoter interactions
              </a>
            </td>
          <td>
            Li Wang, Songjoon Baek, Gauri Prasad, Thucnhi Truongvo, J. Hoskins, L. Amundadottir, Efsun Arda
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Mutations in the male germline are a driving force behind rare genetic diseases. Driver mutations enjoying a selective advantage expand to mutant clusters within the aged testis and are thus overrepresented in sperm with age. Other kinds of driver mutations, occurring prepubescently, have been the focus of recent attention given their high occurrence rate independent of age. Here, we investigated the ErbB2 gene via error-corrected sequencing and detected a high percentage of missense mutations, including recurrent mutations, which were observed mainly in the tyrosine kinase domain with likely functional consequences, as we verified for a subset via biophysical methods. While these mutations increased with age, we found no evidence that they originated from mutational clusters in the aged testis, and young donors also showed an accumulation of driver mutations, suggesting that mutational enrichment is not exclusive to the sexually mature germline but can occur earlier during germline development, and are likely evenly distributed in the testis and stable in size.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa9f975fbf6b3e62de966f4ae5746c898526f4b3" target='_blank'>
              Mutations in ErbB2 accumulating in the male germline measured by error-corrected sequencing
              </a>
            </td>
          <td>
            Atena Yasari, Monika Heinzl, Theresa Mair, Tina Karimian, Shehab Moukbel Ali Aldawla, Ingrid Hartl, Andrea J. Betancourt, Peter Lanzerstorfer, Irene Tiemann-Boege
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background/Aim Despite therapeutic advancements, metastatic colorectal cancer is usually fatal, necessitating novel approaches based on the molecular pathogenesis to improve outcomes. Some colorectal cancers have no mutations in the extended RAS panel (KRAS, NRAS, BRAF) genes and represent a special subset, which deserves particular therapeutic considerations. Materials and Methods The genomic landscape of colorectal cancers from publicly available genomic series was interrogated, using the cBioportal platform. Colorectal cancer cohorts with cancers devoid of KRAS/NRAS or BRAF mutations were evaluated for the presence of mutations in the catalytic sub-unit alpha of kinase PI3K, encoded by the gene PIK3CA. Results PIK3CA mutations in the absence of KRAS/NRAS/BRAF mutations were observed in 3.7% to 7.6% of colorectal cancers in the different series examined. Patients with all four genes in wildtype configuration (quadruple wild type) represented 32.2% to 39.9% of cases in the different series examined. Compared with quadruple wild type cancers, triple (KRAS/NRAS/BRAF) wild type/PIK3CA mutated cancers had a higher prevalence of high TMB cases and additional mutations in colorectal cancer associated genes except for mutations in TP53. Mutations in genes encoding for epigenetic modifiers and the DNA damage response (DDR) were also more frequent in triple wild type/PIK3CA mutated cancers. The prognosis of the two groups was comparable. Conclusion Colorectal cancers with PIK3CA mutations in the absence of KRAS/NRAS/BRAF mutations have frequently mutations in epigenetic modifiers and DDR response genes, which may provide opportunities for targeting. These mutations are present in a smaller subset of quadruple wild type cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fec366ce24e54405084d6854f8aadb7604b8213" target='_blank'>
              PIK3CA Mutated Colorectal Cancers Without KRAS, NRAS and BRAF Mutations Possess Common and Potentially Targetable Mutations in Epigenetic Modifiers and DNA Damage Response Genes
              </a>
            </td>
          <td>
            I. Voutsadakis
          </td>
          <td>2024-08-27</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, Jens Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Menin inhibitors that disrupt Menin-MLL interaction hold promise for treating specific acute myeloid leukemia subtypes, including KMT2A rearrangements (KMT2A-r), yet resistance remains a challenge. Here, through systematic chromatin-focused CRISPR screens, along with genetic, epigenetic, and pharmacologic studies in a variety of human and mouse KMT2A-r AML models, we uncover a potential resistance mechanism independent of canonical Menin-MLL targets. We show that a group of non-canonical Menin targets, which are bivalently co-occupied by active Menin and repressive H2AK119ub marks, are typically downregulated following Menin inhibition. The loss of Polycomb Repressive Complex 1.1 (PRC1.1) subunits, such as PCGF1 or BCOR, leads to Menin inhibitor resistance by epigenetic reactivation of these non-canonical targets, including MYC. Genetic and pharmacological inhibition of MYC can resensitize PRC1.1-deficent leukemia cells to Menin inhibition. Moreover, we demonstrate that leukemia cells with the loss of PRC1.1 subunits exhibit reduced monocytic gene signatures and are susceptible to the BCL2 inhibition, and combinational treatment of venetoclax overcomes the resistance to Menin inhibition in PRC1.1-deficient leukemia cells. These findings highlight the important roles of PRC1.1 and its regulated non-canonical Menin targets in modulating Menin inhibitor response and provide potential strategies to treat leukemias with compromised PRC1.1 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fba94eb1d6d5cb6e0a8de6969fbdc894d28222" target='_blank'>
              Epigenetic Regulation of Non-canonical Menin Targets Modulates Menin Inhibitor Response in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Xinyue Zhou, Lixia Zhang, Sajesan Aryal, Virginia Veasey, Amanda Tajik, Cecilia Restelli, Steven Moreira, Pengcheng Zhang, Yanfeng Zhang, Kristin J Hope, Yang Zhou, Changde Cheng, Ravi Bhatia, Rui Lu
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="SF3B1 is the most recurrently mutated RNA splicing gene in cancer; However, the study of its pathogenic role has been hindered by a lack of disease-relevant cell line models. Here, we compared four genome engineering platforms to establish SF3B1 mutant cell lines: CRISPR-Cas9 editing, AAV HDR editing, base editing (ABEmax, ABE8e), and prime editing (PE2, PE3, PE5Max). We showed that prime editing via PE5max achieved the most efficient SF3B1 K700E editing across a wide range of cell lines. We further refined our approach by coupling prime editing with a fluorescent reporter that leverages a SF3B1 mutation-responsive synthetic intron to mark successfully edited cells. By applying this approach, called prime editing coupled intron-assisted selection (PRECIS), we introduced the K700E hotspot mutation into two chronic lymphocytic leukemia (CLL) cell lines, HG-3 and MEC-1. We demonstrated that our PRECIS-engineered cells faithfully recapitulate known mutant SF3B1 phenotypes including altered splicing, copy number variations, and cell growth defect. Moreover, we uncovered that SF3B1 mutation can cause the loss of Y chromosome in CLL. Our results showcase PRECIS as an efficient and generalizable method for engineering genetically faithful SF3B1 mutant models. Our approach provides new insights on the role of SF3B1 mutation in cancer and enables the generation of SF3B1 mutant cell lines in relevant cellular context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca0b3ac1820b19171ca9e6d9370b57efca6b03" target='_blank'>
              Engineering oncogenic hotspot mutations on SF3B1 via CRISPR-directed PRECIS mutagenesis.
              </a>
            </td>
          <td>
            Mikel Fernandez, Lei Yu, Qiong Jia, Xuesong Wang, Kevyn L Hart, Zhenyu Jia, Ren-Jang Lin, Lili Wang
          </td>
          <td>2024-08-28</td>
          <td>Cancer research communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Recent advances in ''omics'' technologies allow for the identification of an increasing number of individuals with diseases caused by nano-rare mutations. These difficult-to-diagnose individuals are uniquely disadvantaged and pose significant challenges to healthcare systems and society. Despite having diseases caused by actionable single gene mutations, in many cases, there is no commercial path for treatments for such small patient populations. Since antisense oligonucleotide (ASO) technology has proven to be suited to address the needs of a portion of these patients, the n-Lorem Foundation is establishing an industrialized approach that couples detailed genotypic and phenotypic data to the immediate potential for ASO therapy. In this manuscript we have leveraged our experience in assessing the causality of nano-rare genetic variants and associated proximal molecular pathological events to attempt a correlation between detailed genetic data with patient specific phenotypic observations in 173 nano-rare individuals from diverse age groups evaluated for experimental ASO therapy. We found that the time required to achieve a molecular diagnosis varies from 1 month to 36 years, with the mean and median times from symptom onset to diagnosis estimated to be 4.32 years and 2 years, respectively. Amongst submitted cases there is a significant bias toward neurological diseases, with diverse genes and functional families involved and a marked preponderance of mutations in ion channel genes. The variability in phenotypic expression associated with nano-rare variants in genes such as GNAO1, H3F3A, GBE1, UBTF, or PACS1 clearly supports previous observations that phenotypes associated with same variants in the same gene can vary. We also observe that different, but functionally equivalent variants can result in both similar (e.g., TARDBP) and different phenotypes (e.g., GNAO1). Despite the relatively small size of the patient population investigated, this first compilation of its kind allows a variety of insights into the genotype and phenotype relationships in nano-rare conditions. Moreover, we show that our unique patient population presents a remarkable opportunity to apply ''modern omics'' approaches to begin to understand the various homeostatic, compensatory, and secondary effects of these genetic variants on the networks that result in expression of their unique phenotypes. To provide a more detailed description of the processes involved to provide a personalized antisense medicine, we have included nonclinical and clincal data on four exemplary patients who display disease in three different organs, the CNS, the eye and the kidney and are treated with ASOs of different designs. In contrast to traditional drug development, each patient presents unique genomic, ASO design, clinical treatment and management and evaluation challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6d6386d30baeff2ca96c279e928ec4153f9291f" target='_blank'>
              Genotypic and phenotypic analysis of 173 patients with extremely rare pathogenic mutations who applied for experimental antisense oligonucleotide treatment
              </a>
            </td>
          <td>
            S. Crooke, T. A. Cole, J. B. Carroll, J. G. Gleeson, L. Mignon, J. Douville, W. Chung, J. Bain, E. M. Berry-Kravis, N. Leung, S. Demarest, E. McCourt, A. Watt, B. Powers, C. Ngongang
          </td>
          <td>2024-08-06</td>
          <td>None</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8787bff41d07bad7870367afd2e6ff19a0d0ec2d" target='_blank'>
              CDC27 gene expression patterns as a potential biomarker in Acute Leukemia.
              </a>
            </td>
          <td>
            Yasaman Pouriafar, S. Rostami, Nasrin Alizadghandforoush, Mahmood Barati, Ali Amini, Majid Safa
          </td>
          <td>2024-07-29</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Background: To date, targeted therapies have largely failed in eliciting sustained responses in advanced pancreatic ductal adenocarcinoma (PDAC). KRAS-targeted therapies have the potential to drastically transform clinical management. Emerging data indicate that only a subset of patients respond, and that acquired resistance is common. Here, we used a combination of in vivo CRISPR activation (CRISPRa) screening and in vitro/preclinical models to identify synergistic combinations with KRAS inhibition (KRASi) in pancreatic and lung cancer. Material and Methods: We used our CRISPRa-competent PPKS (P53(F/F), Kras(LSL-G12D/+), R26(LSL-SAM)) model of adenocarcinoma to conduct targeted screening for drivers of resistance to MRTX133. Our library consisted of ∼450 frequently mutated and amplified putative oncogenes across human adenocarcinoma based on TCGA data. Mice were treated with 1e8 TU of lentiviral library through nasal instillation. After 6 weeks, treatment was initiated with 3 mg/kg MRTX1133 daily, 5 days/week (i.p.). Mice were sacrificed after 4 weeks of treatment and the guide distribution determined through bioinformatic analysis. Genes statistically significantly enriched (FDR<0.05) in the treatment group were determined. Cell lines with acquired resistance to MRTX1133 derived from the KPC PDAC model and the KPP lung adenocarcinoma (LUAD) model were generated using long-term, low-dose treatment. Resistance was confirmed with MTT drug response assays. In vitro combinatorial drug response was determined using a 72h, 96-well MTT assay, and clonogenic response using a 2 week, 384-well FACS assay. Mice were treated with 20 mg/kg MRTX1133 and/or 20mg/kg Temuterkib once daily, 7 days/week (i.p.) for 2 weeks. Results: We performed targeted in vivo CRISPRa screening for drivers of resistance to KRASi in an autochthons model of lung adenocarcinoma (LUAD) using a library targeting 452 genes commonly amplified or mutated in cancer. We identified a set of genes that were specifically enriched in at least 50% of tumors treated with MRTX1133, including MAPK1/ERK2. We also generated 16 cell lines with acquired resistance to MRTX1133, derived from KPC PDAC or KPP LUAD tumors. Transcriptomic analysis identified a large set of genes differentially expressed in cell with acquired resistance relative to controls. GSEA indicated that reactivation of the MAPK cascade as a common trait in resistant cells. To test if ERK inhibition synergizes with KRASi, we quantified the combinatorial response to MRTX1133 and Temuterkib, a specific inhibitor of ERK1 and ERK2, in a set of KPC and KPP derived cell lines, revealing significant synergy and reduced clonogenic potential in a majority of cell lines. A majority of cell lines with acquired resistance to MRTX1133 remained sensitive to Temuterkib. Finally, we tested combinatorial treatment of MRTX1133 and Temuterkib in a syngenetic subcutaneous model of PDAC, which showed at minimum additive effect. In conclusion we have identified ERK inhibition as a promising combination with KRAS inhibition in KRAS-mutant PDAC and LUAD.
 Citation Format: Fredrik I Thege, Amber Hoskins, Sonja M Woermann, Anirban Maitra. Identifying synergistic combinations with KRAS inhibition in PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr PR-15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/172a9d516cee228f00e1cc1c95f94a25e631e8bb" target='_blank'>
              Abstract PR-15: Identifying synergistic combinations with KRAS inhibition in PDAC
              </a>
            </td>
          <td>
            Fredrik I. Thege, Amber Hoskins, Sonja M. Woermann, Anirban Maitra
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="
 Rhabdoid tumors (RTs) are rare, highly aggressive pediatric malignancies which occur in kidney, soft-parts and brain. They are characterized by a complete inactivation of the SMARCB1 tumor suppressor gene encoding a core subunit of the chromatin remodeling SWI/SNF complex. Prognosis for children with RTs is poor as, in many instances, these tumors are resistant to conventional type chemotherapy. Pharmacological inhibition of EZH2 is a promising strategy to treat some tumors with loss of function in SMARCB1. However, the clinical response rate remains low, and both primary and secondary resistance have been reported. Understanding the mechanisms of resistance to EZH2 inhibition may allow new therapeutic hypothesis. For this purpose, we realized a genome-wide CRISPR-Cas9 knockout screening on two rhabdoid cell lines treated with EZH2 inhibitor. We identified TP53 loss as the sole consistent gene knock-out able to confer some resistance to EZH2 inhibition in both cell lines. Conversely, we demonstrated that MDM2/MDM4 inhibition, in a TP53-dependant manner, strongly synergized with EZH2 inhibition to control cell lines viability in vitro. We further treated 4 rhabdoid patient-derived xenografts with UNC1999, Idasanutlin and the combination of both and observed a potent tumor growth control upon combined EZH2 and MDM2/MDM4 inhibitions. To conclude, our results strongly encourage to assess the actual efficacy of combined MDM2/MDM4 and EZH2 inhibitors in the treatment of patients with SMARCB1-deficient rhabdoid tumors.
 Citation Format: Céline Chauvin, Tiphaine Hery, Rachida Bouarich, Fariba Nemati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, Sergio Roman-Roman, Didier Decaudin, Raphaël Margueron, Franck Bourdeaut. Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A051.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85bf34da504cd580597c4be8039c1663b824b215" target='_blank'>
              Abstract A051 Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors
              </a>
            </td>
          <td>
            C. Chauvin, Tiphaine Héry, Rachida Bouarich, F. Némati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, S. Roman-Roman, Didier Decaudin, Raphaël Margueron, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="Prostate cancer (PC), which is a disease driven by the activity of the androgen receptor (AR), is the most commonly diagnosed malignancy and despite advances in diagnostic and treatment strategies, PC is the second most common cause of cancer mortality in men (Bray et al., 2018). Taxane-based chemotherapy is the only chemotherapy that prolongs survival in metastatic PC patients (Petrylak et al., 2004; Tannock et al., 2004). At the cellular level, taxanes bind to and stabilize microtubules (MTs) inhibiting all MT-dependent intracellular pathways. MTs are highly dynamic polymers that stochastically switch between phases of growth, shrinkage, and pause (Jordan and Wilson, 2004). Altered MT dynamics endow cancer cells with both survival and migratory advantages (Mitchison, 2012). Taxanes inhibit MT dynamics and alter the spatial organization of the MT network, thereby inhibiting intracellular trafficking of molecular cargo critical for tumor survival. In PC specifically, taxanes inhibit transcriptional activity downstream of MT stabilization (Thadani-Mulero et al., 2012) and AR nuclear accumulation (Darshan et al., 2011; Zhu et al., 2010). Different tubulin inhibitors, even from within the same structural class as the taxanes, affect distinct parameters of MT dynamics (Jordan and Wilson, 2004), yet the selection of taxane for chemotherapy is not based on the particular patterns of dynamic behavior of the MT cytoskeleton in individual patients. We envisage that systematic characterization using quantitative analysis of MT dynamics in PC patient cells expressing clinically relevant protein isoforms (Matov et al., 2024; Thoma et al., 2010), before and after treatment with each of the taxanes, will allow us to identify criteria for the selection of the most suitable drug combination at the onset of treatment. We link MT dynamics in the presence of AR variants and sensitivity/resistance to taxanes and connect fundamental research with clinically relevant concepts to elucidate cellular mechanisms of clinical response to taxanes and, thus, advance the customization of therapy. Our computational live-cell analysis addresses questions in the context of the inherent differences in MT homeostasis as a function of AR content in PC cells, the specific parameters of MT dynamics each of the taxanes affects, and how can this information be used to match endogenous patterns of MT dynamics with drug-modulated MT behavior. We investigate whether the sensitivity to taxanes, evaluated by computational analysis of MTs, can be linked to gene expression driven by AR and its variants, and whether the resistance to taxanes be linked to the presence of a specific AR splice variant, and can we identify which of the taxanes will be most effective based on the endogenous patterns of MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e90d5e5c268c6138c1b50b9fd253e67162113bd" target='_blank'>
              Computational Analysis of Treatment Resistant Cancer Cells
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-08-31</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pathogenic variations in the fused in sarcoma (FUS) gene are associated with rare and aggressive forms of amyotrophic lateral sclerosis (ALS). As FUS-ALS is a dominant disease, a targeted, allele-selective approach to FUS knockdown is most suitable. Antisense oligonucleotides (AOs) are a promising therapeutic platform for treating such diseases. In this study, we have explored the potential for allele-selective knockdown of FUS. Gapmer-type AOs targeted to two common neutral polymorphisms in FUS were designed and evaluated in human fibroblasts. AOs had either methoxyethyl (MOE) or thiomorpholino (TMO) modifications. We found that the TMO modification improved allele selectivity and efficacy for the lead sequences when compared to the MOE counterparts. After TMO-modified gapmer knockdown of the target allele, up to 93% of FUS transcripts detected were from the non-target allele. Compared to MOE-modified AOs, the TMO-modified AOs also demonstrated reduced formation of structured nuclear inclusions and SFPQ aggregation that can be triggered by phosphorothioate-containing AOs. How overall length and gap length of the TMO-modified AOs affected allele selectivity, efficiency and off-target gene knockdown was also evaluated. We have shown that allele-selective knockdown of FUS may be a viable therapeutic strategy for treating FUS-ALS and demonstrated the benefits of the TMO modification for allele-selective applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40bef67258a89242d3fafd4ece2697b7aa372f10" target='_blank'>
              Allele-Selective Thiomorpholino Antisense Oligonucleotides as a Therapeutic Approach for Fused-in-Sarcoma Amyotrophic Lateral Sclerosis
              </a>
            </td>
          <td>
            R. Mejzini, Marvin H. Caruthers, Balazs Schafer, Ondrej Kostov, Kavitha Sudheendran, Marija Ciba, Mathias Danielsen, Steve D. Wilton, P. A. Akkari, L. L. Flynn
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="INTRODUCTION
Leukaemia is the most common cancer in children, however, there is still a big gap in knowledge about the genomic alterations in childhood acute myeloid leukaemia (AML) compared to adult AML. Relapsed AML remains as a leading cause of cancer deaths among children. This study aims to understand the molecular mechanisms of relapsed AML by elucidating the mutational landscape before and during relapse.


MATERIALS AND METHODS
Whole genome sequencing was performed on matched samples collected at diagnosis, remission and relapse from three patients of de novo childhood AML. Sanger sequencing was performed for validation in 47 patients' samples, followed by functional analysis.


RESULTS
Overall, we identified 312 somatic mutations including synonymous single nucleotide variants (SNVs), missense SNVs, deletions and insertion frameshifts, stopgains and splice sites. After prioritisation, only 46 variants were present at diagnosis (13-17 mutations per patient) and 49 variants at relapse (12-20 mutations per patient). Out of 81 variants, there were 35 new variants detected at relapse but not present at diagnosis. Six potential driver mutations (KIT, CDC73, HNF1A, RBM10, ZMYM4 and ETV6) were identified in predicting relapse for the 3 patients, with recurrent mutations of the ETV6 gene in 2 patients. Functional analysis of the ETV6 mutation showed that ETV6 lost its tumour suppressive function when both mutant ETV6 p.P25fs and ETV6 p.N75fs were tested in vitro.


CONCLUSION
This study has uncovered the mutational landscape in three local childhood AML patients and contributes to a better understanding of the molecular mechanisms of relapsed AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fe66d9e1447f89554dbfaa72d09784bc4662e53" target='_blank'>
              Whole genome sequencing reveals the mutational landscape from disease diagnosis to relapse in patients with childhood acute myeloid leukaemia.
              </a>
            </td>
          <td>
            H. Aziz, N. Ab Mutalib, H. Alias, R. Jamal
          </td>
          <td>2024-08-01</td>
          <td>The Malaysian journal of pathology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="
 Pediatric cancers are estimated to be related to a genetic predisposition in about 8 to 10% of cases. This estimation mostly relies on already discovered cancer predisposing genes. However, the numerous new cancer predisposing genes that have been discovered in the past decade thanks to high throughput sequencing in vast cohort of pediatric patients illustrate that new genes may still be unravelled. Moreover, the frequency of early post-zygotic mosaicism is being readdressed thanks to more dedicated sequencing and research. The actual clinical impact of all those discoveries is the main challenge for the community: defining the penetrance, delineating the cancer spectrum and therefore the appropriate surveillance, and setting up innovative follow-up techniques are necessary. Here, we’ll describe recent findings on brain tumor predisposing genes (ELP1 and SMARCB1, among others) to illustrate the increasing recognition of genetic predisposition in pediatric cancers and how the clinical translation of these laboratory findings remain challenging at clinical and ethical levels.
 Citation Format: Franck Bourdeaut. Genetic predisposition to pediatric cancer: the long way from discovery to surveillance guidelines [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr IA003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc6b854ba212ba8f4846fc2365983567ae6c2486" target='_blank'>
              Abstract IA003: Genetic predisposition to pediatric cancer: the long way from discovery to surveillance guidelines
              </a>
            </td>
          <td>
            F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Background and Objective Lung cancer stands as the main cause of cancer-related deaths worldwide. With the advent of immunotherapy and the discovery of targetable oncogenic driver genes, although prognosis has changed in the last few years, survival rates remain dismal for most patients. This emphasizes the urgent need for new strategies that could enhance treatment in precision medicine. The role of the microbiota in carcinogenesis constitutes an evolving landscape of which little is known. It has been suggested these microorganisms may influence in responses, resistance, and adverse effects to cancer treatments, particularly to immune checkpoint blockers. However, evidence on the impact of microbiota composition in oncogene-addicted tumors is lacking. This review aims to provide an overview of the relationship between microbiota, daily habits, the immune system, and oncogene-addicted tumors, focusing on lung cancer. Methods A PubMed and Google Scholar search from 2013 to 2024 was conducted. Relevant articles were reviewed in order to guide our research and generate hypothesis of clinical applicability. Key Content and Findings Microbiota is recognized to participate in immune reprogramming, fostering inflammatory, immunosuppressive, or anti-tumor responses. Therefore, identifying the microbiota that impact response to treatment and modulating its composition by interventions such as dietary modifications, probiotics or antibiotics, could potentially yield better outcomes for cancer patients. Additionally, targeted therapies that modulate molecular signaling pathways may impact both immunity and microbiota. Understanding this intricate interplay could unveil new therapeutic strategies. Conclusions By comprehending how microbiota may influence efficacy of targeted therapies, even though current evidence is scarce, we may generate interesting hypotheses that could improve clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c51e9b0c6cc73dbc9879341142a717c957faa91" target='_blank'>
              Oncogene-addicted solid tumors and microbiome—lung cancer as a main character: a narrative review
              </a>
            </td>
          <td>
            Mora Guardamagna, May-Lucie Meyer, M. Berciano-Guerrero, Andres Mesas-Ruiz, M. Cobo-Dols, Elisabeth Pérez-Ruiz, Alexandra Cantero González, R. Lavado-Valenzuela, Isabel Barragán, Javier Oliver, Alicia Garrido-Aranda, Martina Álvarez, A. Rueda-Domínguez, M. I. Queipo-Ortuño, Emilio Alba Conejo, J. C. Benítez
          </td>
          <td>2024-08-01</td>
          <td>Translational Lung Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31c537bce8789a914d584c53915429d4b9a4119c" target='_blank'>
              Prognosis and treatment in acute myeloid leukemia: a comprehensive review
              </a>
            </td>
          <td>
            Ankita Debnath, S. Nath
          </td>
          <td>2024-08-06</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24d8d8f2f89f8ee70047ce83a13996f46168eafe" target='_blank'>
              Star wars against leukemia: attacking the clones.
              </a>
            </td>
          <td>
            Monika M Toma, Tomasz Skorski
          </td>
          <td>2024-09-02</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="ABSTRACT Changes in gene dosage can have tremendous evolutionary potential (e.g. whole-genome duplications), but without compensatory mechanisms, they can also lead to gene dysregulation and pathologies. Sex chromosomes are a paradigmatic example of naturally occurring gene dosage differences and their compensation. In species with chromosome-based sex determination, individuals within the same population necessarily show ‘natural’ differences in gene dosage for the sex chromosomes. In this Review, we focus on the mammalian X chromosome and discuss recent new insights into the dosage-compensation mechanisms that evolved along with the emergence of sex chromosomes, namely X-inactivation and X-upregulation. We also discuss the evolution of the genetic loci and molecular players involved, as well as the regulatory diversity and potentially different requirements for dosage compensation across mammalian species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016ee497a5dab2410dc213e6a7a7d311ff9fabfc" target='_blank'>
              Compensation of gene dosage on the mammalian X
              </a>
            </td>
          <td>
            Daniela Cecalev, Beatriz Viçoso, Rafael Galupa
          </td>
          <td>2024-08-01</td>
          <td>Development (Cambridge, England)</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Genome sequencing has revealed frequent mutations in Ras homolog family member A (RHOA) among various cancers with unique aberrant profiles and pathogenic effects, especially in peripheral T-cell lymphoma (PTCL). The discrete positional distribution and types of RHOA amino acid substitutions vary according to the tumor type, thereby leading to different functional and biological properties, which provide new insight into the molecular pathogenesis and potential targeted therapies for various tumors. However, the similarities and discrepancies in characteristics of RHOA mutations among various histologic subtypes of PTCL have not been fully elucidated. Herein we highlight the inconsistencies and complexities of the type and location of RHOA mutations and demonstrate the contribution of RHOA variants to the pathogenesis of PTCL by combining epigenetic abnormalities and activating multiple downstream pathways. The promising potential of targeting RHOA as a therapeutic modality is also outlined. This review provides new insight in the field of personalized medicine to improve the clinical outcomes for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd71cf13763be0cc23a4dc23d32cf828f50620b5" target='_blank'>
              Mutations in Ras homolog family member A in patients with peripheral T-cell lymphoma and implications for personalized medicine
              </a>
            </td>
          <td>
            Lina Hu, Xuanye Zhang, Shengbing Zang
          </td>
          <td>2024-08-09</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Transposable elements (TEs) are dynamic repetitive regions which generate mutations and structural variants. TP53 plays a crucial role in suppressing TE movement to maintain genomic stability. The relationship between TP53 and TEs has been extensively studied in tumours, but not the germline. Individuals with germline TP53 pathogenic variants have Li-Fraumeni Syndrome (LFS), a cancer predisposition syndrome with a high lifetime risk of cancer in various tissues. This study aims to characterize the TE landscape in individuals with Li-Fraumeni Syndrome and determine how this contributes to their increased cancer risk. Methods: MELT and xTea were used to identify TEs in children with (n=48) or without (n=198) a germline TP53 variant. TE calls were merged with SURVIVOR and variants were annotated with AnnotSV. To assess the influence of TP53 on TE location, we quantified TEs across genomic windows and identified the top 100 significantly different regions with the Mann-Whitney U test, adjusting for multiple comparisons. We used these regions to develop a gradient-boosted tree model with 5-fold cross-validation to predict TP53 status. Results: Individuals with germline TP53 variants harboured significantly fewer ALU and LINE1 elements in their germline genome compared to the control dataset (p<0.001). The phenomenon was consistent across chromosomes and significant in chromosomes 5 and 11 (FDR<0.05). A gradient-boosted tree model was able to differentiate patients with and without a germline TP53 variant with an AUPRC of 0.77 on an unseen test set. Conclusion: Germline TP53 variants may effect the frequency and location of germline TE insertions, which may influence nearby genomic variations and lead to cancer development. Analyzing TEs in individuals with LFS will enhance our understanding of accelerated cancer development in these patients, informing future research for diagnostic and therapeutic approaches.
 Citation Format: Brianne Laverty, Shilpa Yadahalli, Vallijah Subasri, David Malkin. The role of TP53 on transposable elements in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4995c29bd2050c705f9a5b8da916aacb0aa2601c" target='_blank'>
              Abstract A010 The role of TP53 on transposable elements in pediatric cancer
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Vallijah Subasri, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
FLT3 mutations are among the most common myeloid drivers identified in adult acute myeloid leukemia (AML). Their identification is crucial for the precise risk assessment because of the strong prognostic significance of the most recurrent type of FLT3 alterations, namely internal tandem duplications (ITDs). Recent advances in the pathogenesis and biology of FLT3-mutated AML have opened an opportunity for development and application of selective inhibition of FLT3 pathway.


RECENT FINDINGS
In the last decade, at least three targeted treatments have been approved by regulatory agencies and several others are currently under investigations. Here, we review the latest advance in the role of FLT3 mutations in AML, providing an outline of the available therapeutic strategies, their mechanisms of actions and of resistance, as well as routes for potential improvement.


SUMMARY
The availability of FLT3 inhibitors has improved outcomes in AML harboring such mutations, currently also reflected in disease stratification and recommendations. Newer inhibitors are under investigations, and combinations with chemotherapy or other targeted treatments are being explored to further improve disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732f19a10a5bb85b8ee615c65a99e7f2184b3ca7" target='_blank'>
              Advances in the pathogenesis of FLT3-mutated acute myeloid leukemia and targeted treatments.
              </a>
            </td>
          <td>
            S. Travaglini, C. Gurnari, T. Ottone, M. Voso
          </td>
          <td>2024-08-27</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Hepatocellular carcinoma (HCC) is one of the most common malignant tumors of the digestive system. RNA methylation plays an important role in tumorigenesis and metastasis, which could alter gene expression and even function at multiple levels, such as RNA splicing, stability, translocation, and translation. In this study, we aimed to conduct a comprehensive analysis of RNA methylation-related genes (RMGs) in HCC and their relationship with survival and clinical features. Methods A retrospective analysis was performed using publicly available HCC-related datasets. The differentially expressed genes (DEGs) between HCC and controls were identified from TCGA-LlHC and intersected with RMGs to obtain differentially expressed RNA methylation-related genes (DERMGs). Regression analysis was used to screen for prognostic genes and construct risk models. Simultaneously, clinical, immune infiltration and therapeutic efficacy analyses were performed. Finally, multivariate cox regression was used to identify independent risk factors, and quantitative real-time polymerase chain reaction (qRT-PCR) was used to validate the expression levels of the core genes of the model. Results A 21-gene risk model for HCC was established with excellent performance based on ROC curves and survival analysis. Risk scores correlated with tumor grade, pathologic T, and TNM stage. Immune infiltration analysis showed correlations with immune scores, 11 immune cells, and 30 immune checkpoints. Low-risk patients showed a higher susceptibility to immunotherapy. The risk score and TNM stage were independent prognostic factors. qRT-PCR confirmed higher expression of PRDM9, ALPP, and GAD1 in HCC. Conclusions This study identified RNA methylation-related signature genes in HCC and constructed a risk model that predicts patient outcomes and reflects the immune microenvironment. Prognostic genes are involved in complex regulatory mechanisms, which may be useful for cancer diagnosis, prognosis, and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/288e5a95cc7c2059fde0835f504837f6d0a5d61b" target='_blank'>
              A Novel RNA Methylation-Related Prognostic Signature and its Tumor Microenvironment Characterization in Hepatocellular Carcinoma
              </a>
            </td>
          <td>
            Luzheng Liu, Jiacheng Chen, Fei Ye, Yanggang Yan, Yong Wang, Jincai Wu
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="BACKGROUND AND OBJECTIVE
The identification of mutation hot spots in the isocitrate dehydrogenase (IDH) genes is one of the most important cancer genome-wide sequencing discoveries with relevant impact in the treatment of some orphan tumors. These genes were mostly found mutated in lower-grade gliomas (LGGs), acute myeloid leukaemia (AML), myelodysplastic syndromes (MDS) and myeloproliferative neoplasms (MPNs) and in cholangiocarcinoma. This aberrant genomic condition represents a therapeutic target of great interest in cancer research, especially in AML, given the limitations of currently approved therapies in this field. In this review, we investigate the role of IDH mutation and the mutant IDH (mIDH)- targeted therapies for cholangiocarcinoma and glioma.


METHODS
Here, we provide an overview of the IDH mutation role and discuss its role in tumorigenesis and progression of some solid cancers, in which the therapeutic strategy can be completely changed thanks to these brand-new therapeutic options.


KEY CONTENT AND FINDINGS
The encouraging early clinical data demonstrated to be a proof of concept for investigational mIDH1/2 inhibitors in tumors with a paucity of therapeutic possibilities.


CONCLUSIONS
Moreover, we list the most important randomised clinical trials still active with their preliminary results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a20edde7b8a46ad9d9cedbb5e90b751d02d2ad2f" target='_blank'>
              Therapeutic inhibition of isocitrate dehydrogenase mutations in glioma and cholangiocarcinoma: new insights and promises-a narrative review.
              </a>
            </td>
          <td>
            V. Internò, A. Melaccio, Pasquale Vitale, Roberta Spedaliere, Massimo Buonfantino, R. Messina, Anna Laura Lippolis, Francesco Signorelli, Raffaele Addeo, Francesco Giuliani
          </td>
          <td>2023-01-01</td>
          <td>Chinese clinical oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Background Current diagnostic tools are unable to distinguish low‐grade indolent prostate cancer (PrCa) from that with a propensity to become metastatic and/or lethal. Recent evidence suggests that reprogramming of the transcriptome may drive the metastatic phenotype, and that this reprogramming is controlled, at least in part, by epigenetic changes to the DNA of cancer cells, including methylation. These changes, referred to as ‘epigenetic drivers,’ have previously been associated with cancer cell survival. Methods Here, using Illumina Methylation EPIC array data of paired primary PrCa and metastatic bone samples, we identified WNT5A as a putative epi‐driver of PrCa metastasis to the bone, which was further validated in vitro. Results Significantly higher WNT5A methylation was observed in primary PrCa samples and 22Rv1 cells compared to metastatic bone samples and PC‐3 cells. This higher methylation was associated with significantly lower WNT5A gene expression. Conclusion Given the limited effective therapies available for metastatic cancer sufferers, particularly those whose disease has metastasised to the bone, WNT5A presents as a potential putative target for therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b055e884a7d4d2969ce7c73d7f93b2c051be4c83" target='_blank'>
              WNT5A is a putative epi‐driver of prostate cancer metastasis to the bone
              </a>
            </td>
          <td>
            Emma J Wilkinson, Kelsie Raspin, R. Malley, S. Donovan, Louise M Nott, Adele F Holloway, Joanne L Dickinson
          </td>
          <td>2024-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e16f71ff68478a0f30f71105d86b66a3bf8d6b91" target='_blank'>
              Single-Cell Sequencing: High-Resolution Analysis of Cellular Heterogeneity in Autoimmune Diseases.
              </a>
            </td>
          <td>
            Xuening Tang, Yudi Zhang, Hao Zhang, Nan Zhang, Ziyu Dai, Q. Cheng, Yongzhen Li
          </td>
          <td>2024-08-26</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95a5cebb3dacd4f74ab63e4dfea0d28d7bda5262" target='_blank'>
              N6-methyladenosine (m6A) RNA modification in chronic myeloid leukemia: unveiling a novel therapeutic target
              </a>
            </td>
          <td>
            Guillermo Fernandez Rodriguez, Marco Tarullo, A. Fatica
          </td>
          <td>2024-07-31</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="BACKGROUND
The identification of the most appropriate targeted therapies for advanced cancers is challenging. We performed a molecular profiling of metastatic solid tumors utilizing a comprehensive next-generation sequencing (NGS) assay to determine genomic alterations' type, frequency, actionability, and potential correlations with PD-L1 expression.


METHODS
A total of 304 adult patients with heavily pretreated metastatic cancers treated between January 2019 and March 2021 were recruited. The CLIA-/UKAS-accredit Oncofocus assay targeting 505 genes was used on newly obtained or archived biopsies. Chi-square, Kruskal-Wallis, and Wilcoxon rank-sum tests were used where appropriate. Results were significant for P < .05.


RESULTS
A total of 237 tumors (78%) harbored potentially actionable genomic alterations. Tumors were positive for PD-L1 in 68.9% of cases. The median number of mutant genes/tumor was 2.0 (IQR: 1.0-3.0). Only 34.5% were actionable ESCAT Tier I-II with different prevalence according to cancer type. The DNA damage repair (14%), the PI3K/AKT/mTOR (14%), and the RAS/RAF/MAPK (12%) pathways were the most frequently altered. No association was found among PD-L1, ESCAT, age, sex, and tumor mutational status. Overall, 62 patients underwent targeted treatment, with 37.1% obtaining objective responses. The same molecular-driven treatment for different cancer types could be associated with opposite clinical outcomes.


CONCLUSIONS
We highlight the clinical value of molecular profiling in metastatic solid tumors using comprehensive NGS-based panels to improve treatment algorithms in situations of uncertainty and facilitate clinical trial recruitment. However, interpreting genomic alterations in a tumor type-specific manner is critical.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b13a3de4d8a4a44c9264bbeddda708851f093875" target='_blank'>
              Next-generation sequencing-based evaluation of the actionable landscape of genomic alterations in solid tumors: the "MOZART" prospective observational study.
              </a>
            </td>
          <td>
            Francesco Schettini, Marianna Sirico, M. Loddo, Gareth H Williams, Keeda-Marie Hardisty, Paul Scorer, R. Thatcher, Pablo Rivera, Manuela Milani, C. Strina, G. Ferrero, Marco Ungari, C. Bottin, Fabrizio Zanconati, Nicolò de Manzini, S. Aguggini, Richard Tancredi, E. Fiorio, Antonio Fioravanti, Maurizio Scaltriti, Daniele Generali
          </td>
          <td>2024-08-23</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3335db864e73ace960f7e153ca0a3871e879b0c6" target='_blank'>
              Hyper-recombination in ribosomal DNA is driven by long-range resection-independent RAD51 accumulation
              </a>
            </td>
          <td>
            Zita Gál, Stavroula Boukoura, Kezia Catharina Oxe, Sara Badawi, Blanca Nieto, L. Korsholm, Sille Blangstrup Geisler, E. Dulina, Anna Vestergaard Rasmussen, Christina Dahl, Wei Lv, Huixin Xu, Xiaoguang Pan, Stefanos Arampatzis, Danai-Eleni Stratou, P. Galanos, Lin Lin, Per Guldberg, Jiri Bartek, Yonglun Luo, D. H. Larsen
          </td>
          <td>2024-09-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="PURPOSE In metastatic colorectal cancer (mCRC), RAS mutations drive resistance to anti–epidermal growth factor receptor antibodies. It is unclear whether RAS mutations ever become clonally undetectable. METHODS CO.26 was a phase II clinical trial that assessed durvalumab + tremelimumab in heavily pretreated mCRC. RAS mutation status was tracked over time using circulating tumor DNA (ctDNA) sequencing at baseline, week 8, and on progression. RESULTS Among the 95 patients with KRAS/NRAS mutations in their archival tumor tissue, 6.3% (6/95) had undetectable RAS mutations in ctDNA collected at baseline or week 8 of the CO.26 study. Of these, 67% (4/6) of disappearances were transient, with the same mutation reappearing with progressive disease. In three cases, the simultaneous persistence of other preexisting CRC-associated truncal mutations could not be demonstrated, suggestive of low tumor shedding of ctDNA, leaving the incidence of true clonal reversion to RAS-wildtype (WT) possibly as low as 3.2% (3/95). Fewer patients in the neo-RAS-WT group (33%) had greater than four lesions at trial baseline compared with patients with persistent RAS mutations (75%), P = .046. The likelihood of synchronous metastases at cancer diagnosis (33% v 63%; P = .15) or liver metastases at trial baseline (50% v 68.5%; P = .17) was not significantly different between patients with disappearing versus persistent RAS mutations. Overall survival from stage IV diagnosis (hazard ratio, 0.77 [95% CI, 0.35 to 1.72]; P = .52) was not significantly different between those with disappearing versus persistent RAS mutations. The disappearance of RAS mutations was not associated with primary tumor sidedness (P = .41), archival BRAF/MEK/ERK-mutant status (P = .16/1.00/.09), nor baseline ctDNA HER2 amplifications (P = 1.00). CONCLUSION We identified a 3.2%-6.3% prevalence of the neo-RAS-WT phenomenon in the CO.26 trial. However, 67% of apparent cases were transient with subsequent re-emergence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6c51668f82416dc0a222d299f79ee86f027730" target='_blank'>
              Kinetic Profiling of RAS Mutations With Circulating Tumor DNA in the Canadian Cancer Trials Group CO.26 Trial Suggests the Loss of RAS Mutations in Neo-RAS-Wildtype Metastatic Colorectal Cancer Is Transient
              </a>
            </td>
          <td>
            Florence T H Wu, J. Topham, C. O'Callaghan, Harriet Feilotter, Hagen F Kennecke, L. Drusbosky, D. Renouf, Derek J Jonker, Dongsheng Tu, Eric X Chen, J. Loree
          </td>
          <td>2024-08-01</td>
          <td>JCO Precision Oncology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a disease characterized by spatiotemporal heterogeneity of tumor clones. Different genetic aberrations can be observed simultaneously in tumor cells from different loci, and as the disease progresses, new subclones may appear. The role of liquid biopsy, which is based on the analysis of tumor DNA circulating in the blood plasma, continues to be explored in MM. Here, we present an analysis of the STR profiles and mutation status of the KRAS, NRAS, and BRAF genes, evaluated in plasma free circulating tumor DNA (ctDNA), CD138+ bone marrow cells, and plasmacytomas. The prospective single-center study included 97 patients, with a median age of 55 years. Of these, 94 had newly diagnosed symptomatic MM, and three had primary plasma cell leukemia. It should be noted that if mutations were detected only in ctDNA, “non-classical” codons were more often affected. A variety of adverse laboratory and clinical factors have been associated with the detection of rare KRAS or NRAS gene mutations in bone marrow or ctDNA, suggesting that these mutations may be factors of an unfavorable prognosis for MM. Liquid biopsy studies provide undeniable fundamental information about tumor heterogeneity and clonal evolution in MM. Moreover, we focus on using liquid biopsy to identify new high-risk factors for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef6c42652b61bb294400045b1d3bc2249d75f9b" target='_blank'>
              Loss of Heterozygosity and Mutations in the RAS-ERK Pathway Genes in Tumor Cells of Various Loci in Multiple Myeloma
              </a>
            </td>
          <td>
            M. Soloveva, M. Solovev, N. Risinskaya, E. Nikulina, I. Yakutik, B. Biderman, Tatiana Obukhova, Y. Chabaeva, Sergej Kulikov, Andrey Sudarikov, L. Mendeleeva
          </td>
          <td>2024-08-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Sarcomas constitute a large group of rare, heterogeneous cancers of the bone and soft tissue affecting children and young adults. There are over 80 clinically distinct subtypes of sarcoma, and their heterogeneity is a contributing factor to the moderate and poor survival outcomes observed in primary and metastatic disease, respectively. While they are heterogeneous, many subtypes harbor a balanced chromosomal translocation fusing the Ewing Sarcoma RNA binding protein 1 (EWSR1) gene with a gene encoding a transcription factor (TF). This translocation generates two distinct fusions: a canonical fusion consisting of the N-terminal transactivation domain of EWSR1 and the C-terminal DNA-binding domain of a TF, and a reciprocal fusion fusing the N-terminus of the TF and C-terminal RNA-binding domain of EWSR1. Canonical fusion protein products exhibit aberrant TF and pioneer factor activity, leading to alterations in transcriptional programs and the epigenetic landscape within a cell that contributes to oncogenesis. The functions of the reciprocal fusion proteins, however, have remained elusive. Ewing sarcoma (ES) and clear cell sarcoma (CCS), a bone and soft tissue sarcoma, respectively, are two subtypes that have been shown to express reciprocal fusions. The ES fusion implicates EWSR1 and Friend Leukemia Integration Factor 1 (FLI1), while the CCS fusion involves Activating Transcription Factor 1 (ATF1). A challenge in studying the roles of these and other fusion proteins in sarcoma development is the lack of effective model systems, primarily due to their toxicity upon exogenous introduction into fusion-naïve cells. Here, we outline the development of a cumate-inducible lentiviral vector to study the canonical and reciprocal EWSR1 fusion proteins present in ES and CCS. To track cellular localization of the fusion proteins and distinguish fusions from their respective wildtype proteins, we designed fusion sequences with N-terminal fluorescent tags. Tightly controlled expression of canonical and reciprocal fusion proteins reduces their toxicity when expressed in cells. We have successfully established 293T and HT-1080 cells that stably express these vectors. We have also validated cumate-inducible expression of fluorescently tagged canonical and reciprocal FLI1 and ATF1 fusions at the RNA and protein levels in both cell lines. The canonical fusion proteins produced by our model have the expected effects on expression levels of known downstream canonical fusion target genes. Initial cellular localization studies using our model and fluorescence microscopy indicate that reciprocal fusion proteins may localize in the nucleus, which was previously unknown. Overall, our cumate-inducible model represents a viable system for fusion protein expression. Using this model, we aim to gain insight into the roles of reciprocal fusions in ES and CCS development with hopes of uncovering potentially targetable vulnerabilities, opening the door for the development of new treatments in a patient population in need of novel and efficacious therapies.
 Citation Format: Sarah Gawlak, Joyce Ohm. Development of model systems to investigate the roles of canonical and reciprocal EWSR1 fusion proteins in sarcomagenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B079.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6104d5296aa07ecd63e21655a418e97199e7a8ba" target='_blank'>
              Abstract B079: Development of model systems to investigate the roles of canonical and reciprocal EWSR1 fusion proteins in sarcomagenesis
              </a>
            </td>
          <td>
            Sarah Gawlak, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Combined tracking of clonal evolution and chimeric cell phenotypes could enable detection of the key cellular populations associated with response following therapy, including after allogeneic hematopoietic stem cell transplantation (HSCT). We demonstrate that mitochondrial DNA (mtDNA) mutations co-evolve with somatic nuclear DNA mutations at relapse post-HSCT and provide a sensitive means to monitor these cellular populations. Further, detection of mtDNA mutations via single-cell ATAC with select antigen profiling by sequencing (ASAP-seq) simultaneously determines not only donor and recipient cells, but also their phenotype, at frequencies of 0.1-1%. Finally, integration of mtDNA mutations, surface markers, and chromatin accessibility profiles enables the phenotypic resolution of leukemic populations from normal immune cells, thereby providing fresh insights into residual donor-derived engraftment and short-term clonal evolution following therapy for post-transplant leukemia relapse. As throughput evolves, we envision future development of single-cell sequencing-based post-transplant monitoring as a powerful approach for guiding clinical decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454ae3e4512380b5fd1a0c7892e2f418e6fca333" target='_blank'>
              Tracking rare single donor and recipient immune and leukemia cells after allogeneic hematopoietic cell transplantation using mitochondrial DNA mutations.
              </a>
            </td>
          <td>
            Livius Penter, Nicoletta Cieri, Katie Maurer, M. Kwok, Haoxiang Lyu, Wesley S Lu, Giacomo Oliveira, Satyen H Gohil, I. Leshchiner, Caleb A. Lareau, Leif S. Ludwig, Donna S. Neuberg, Haesook T Kim, Shuqiang Li, L. Bullinger, Jerome Ritz, Gaddy Getz, Jacqueline S Garcia, Robert J. Soiffer, K. Livak, Catherine J. Wu
          </td>
          <td>2024-09-05</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Philadelphia chromosome-positive B-cell acute lymphoblastic leukemia (Ph(+)B-ALL) is a hematological malignancy with a poor prognosis. Epigenetic abnormalities, especially abnormal histone acetylation and microRNAs (miRNAs) dysregulation, are a group of epigenetic patterns that contribute to leukemia progression. However, their regulatory mechanisms in Ph(+)B-ALL have not been fully elucidated. In this study, we identified that miR-183-5p is significantly downregulated in Ph(+)B-ALL and associated with poor prognosis. Moreover, we found that the BCR-ABL fusion gene is a key target gene of miR-183-5p. MiR-183-5p directly targets BCR-ABL gene and induces cell apoptosis via PTEN/AKT and c-MYC signaling pathways. In addition, histone deacetylase inhibitor (HADCi) could mitigate the suppressive effects of HDAC2 on miR-183-5p by promoting promoter acetylation, thereby enhancing cell apoptosis. In conclusion, our results indicate that miR-183-5p is a potential biomarker and suggest that a novel "HDAC2-miR-183-5p epigenetic circuitry regulation" may be involved in the pathogenesis of Ph(+)B-ALL. Taken together, These findings provide new insights into the design of promising molecular-targeted drugs for Ph(+)B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d13162cad787b4066b1367ec7518bbaa77c34746" target='_blank'>
              HDAC2-miR183-5p Epigenetic Circuit Contributes to the Growth of Philadelphia Chromosome-Positive B-cell Acute Lymphoblastic Leukemia via PTEN/AKT and c-MYC Signaling Pathway.
              </a>
            </td>
          <td>
            Yangyang Ding, Xiangjiang Feng, Zelin Liu, Ya Liao, Lianfang Pu, Jun Liu, Huiping Wang, Zhimin Zhai, Shu-Dao Xiong
          </td>
          <td>2024-09-11</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer.
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Rationale: PSMA-targeting radioligand therapy (PSMA-RLT) has shown promise in metastatic castration-resistant prostate cancer (mCRPC), particularly in PSMA-avid tumours. However, predicting response remains challenging. Preclinical data suggests aberrant p53-signalling as a predictor of poor response. Methods: The patient population of this pre-planned retrospective cohort study consists of 96 patients with mCRPC who underwent treatment with PSMA-RLT and were molecularly profiled by whole-genome sequencing and or targeted next-generation sequencing. Response to PSMA-RLT was assessed per molecular subtype, including TP53-mutational status. Results: Patients with TP53 loss-of-function alterations had a shorter median progression-free survival (3.7 versus 6.2 months, P<0.001), a lower median PSA change (-55% vs. -75%, P=0.012) and shorter overall survival from initiation of PMSA-RLT (7.6 vs. 13.9 months, P=0.003) compared to TP53-wildtype patients. Pathogenic alterations in AR, MYC, BRCA1, or BRCA2 as well as in genes linked to the PI3K or MAPK pathways or genes involved in homologous recombination repair, were not associated with response. Only lactate dehydrogenase was, alongside TP53-status, significantly associated with response. Transcriptome analysis of 21 patients, identified six p53 signalling genes whose low expression was associated to a shorter progression-free survival (P<0.05). Conclusion: TP53 loss-of-function may serve as a prognostic factor for PSMA-RLT outcomes in patients with mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c11495faf656b283d4b121b320ac0068523d4f0" target='_blank'>
              Impact of TP53 loss-of-function alterations on the response to PSMA radioligand therapy in metastatic castration-resistant prostate cancer patients
              </a>
            </td>
          <td>
            P. Slootbeek, M. V. Luna-Velez, B. Privé, M. J. van der Doelen, I. Kloots, Samhita Pamidimarri Naga, Hilde E. Onstenk, James Nagarajah, H. Westdorp, I. V. van Oort, L. Kroeze, J. Schalken, H. Bloemendal, N. Mehra
          </td>
          <td>2024-08-01</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Simple Summary Thymomas and thymic carcinomas (together referred to as thymic epithelial tumors, TETs) are a heterogeneous group of rare tumors of the anterior mediastinum. Most treatment regimens for advanced TETs are empirical and patients have not benefited from developments in personalized medicine. We believe that one of the major unresolved problems in TET research is the lack of suitable cellular or animal models that represent the full heterogeneity of TETs, help us to study the driving mechanisms behind TET biology, and explore vulnerabilities. In this narrative review, we discuss the current state of 2D and 3D TET cell culture models and protocols. To move beyond the current state, innovative high-throughput techniques are mandatory for both systematic optimization of cell culture conditions and high-throughput drug screening. Abstract Cell culture model systems are fundamental tools for studying cancer biology and identifying therapeutic vulnerabilities in a controlled environment. TET cells are notoriously difficult to culture, with only a few permanent cell lines available. The optimal conditions and requirements for the ex vivo establishment and permanent expansion of TET cells have not been systematically studied, and it is currently unknown whether different TET subtypes require different culture conditions or specific supplements. The few permanent cell lines available represent only type AB thymomas and thymic carcinomas, while attempts to propagate tumor cells derived from type B thymomas so far have been frustrated. It is conceivable that epithelial cells in type B thymomas are critically dependent on their interaction with immature T cells or their three-dimensional scaffold. Extensive studies leading to validated cell culture protocols would be highly desirable and a major advance in the field. Alternative methods such as tumor cell organoid models, patient-derived xenografts, or tissue slices have been sporadically used in TETs, but their specific contributions and advantages remain to be shown.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89ce3ee9aaf655d07e4e9a399628eb007a235cdd" target='_blank'>
              Cell Culture Models for Translational Research on Thymomas and Thymic Carcinomas: Current Status and Future Perspectives
              </a>
            </td>
          <td>
            Denise Müller, Jürgen Loskutov, Stefan Küffer, Alex Marx, C. Regenbrecht, Philipp Ströbel, Manuela J. Regenbrecht
          </td>
          <td>2024-08-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fcc48036440e068784b3b542811aace316e8c88" target='_blank'>
              Clinical application of whole-genome sequencing of solid tumors for precision oncology
              </a>
            </td>
          <td>
            Ryul Kim, Seokhwi Kim, Brian Baek-Lok Oh, Woo Sik Yu, Chang Woo Kim, Hoon Hur, Sang-Yong Son, Min Jae Yang, Dae Sung Cho, Taeyang Ha, Subin Heo, Jeon Yeob Jang, Jae Sung Yun, Kyu-sung Kwack, Jai Keun Kim, Jimi Huh, S. Lim, Sang-Uk Han, Hyun Woo Lee, Ji Eun Park, Chul-Ho Kim, Jin Roh, Young Wha Koh, Dakeun Lee, Jang-Hee Kim, G. Lee, Choong-Kyun Noh, Y. Jung, Ji Won Park, Seungsoo Sheen, Mi Sun Ahn, Yong Won Choi, Tae-Hwan Kim, Seok Yun Kang, Jin-Hyuk Choi, Soo Yeon Baek, K. Lee, Sun Il Kim, Sung Hyun Noh, Se-Hyuk Kim, Hyemin Hwang, Eunjung Joo, Shinjung Lee, Jong-Yeon Shin, Ji-Young Yun, Jung-Woon Park, Kijong Yi, Youngoh Kwon, W. Lee, H. Park, Joonoh Lim, Boram Yi, Jaemo Koo, June-Young Koh, Sangmoon Lee, Yuna Lee, Bo-Rahm Lee, Erin Connolly-Strong, Young Seok Ju, Minsuk Kwon
          </td>
          <td>2024-08-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The seven human APOBEC3 (hA3) genes encode polynucleotide cytidine deaminases that play vital roles in restricting replication of viruses and retrotransposons. However, off-target A3 deamination of the cellular genome is a major source of somatic mutations in human cancer. The ability to study A3 biology in vivo is hindered by the fact that the solitary murine Apobec3 gene (mA3) encodes a cytoplasmic enzyme, with no apparent mutagenic activity. Transgenic expression of individual hA3 genes in mice has helped to confirm their oncogenic potential but important questions including which hA3 genes are active in different tissue contexts and how they function in concert when under control of their cognate promoters cannot be addressed using these models. Here we describe humanization of the mouse mA3 locus by integration of a modified BAC clone encompassing the entire 7-gene hA3 locus from human chromosome 22 replacing mA3 on mouse chromosome 15. APOBEC3 mice are viable and fertile and hA3 gene expression in cells and tissues correlates strongly with expression in corresponding human cells and tissues, indicating human-like regulation of hA3 gene expression in the mice. Splenocytes from this line display a functional human A3 response to Friend Murine Leukaemia Virus (F-MLV) infection. We propose that the Hs-APOBEC3 mouse will uniquely model the function of the complete hA3 locus in a living organism and that it will serve as a useful background upon which to model human cancer, as well as assisting drug discovery efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8034f8597ff29dbab41cdb895f637bb191838ad2" target='_blank'>
              Human-like APOBEC3 gene expression and anti-viral responses following replacement of mouse Apobec3 with the 7-gene human APOBEC3 locus
              </a>
            </td>
          <td>
            Nerissa K. Kirkwood, H. Prosser, Melvyn W Yap, Jane Gibson, Ross Cook, Ian Reddin, Ane Stranger, Emma Howes, Nur Zainal, M. Periyasamy, S. Conticello, Gareth J. Thomas, James Scott, Kate N. Bishop, Simak Ali, Allan Bradley, Tim R. Fenton
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Colorectal cancer (CRC) ranks third in terms of cancer incidence worldwide and is responsible for 8% of all deaths globally. Approximately 10% of CRC cases are caused by inherited pathogenic mutations in driver genes involved in pathways that are crucial for CRC tumorigenesis and progression. These hereditary mutations significantly increase the risk of initial benign polyps or adenomas developing into cancer. In recent years, the rapid and accurate sequencing of CRC-specific multigene panels by next-generation sequencing (NGS) technologies has enabled the identification of several recurrent pathogenic variants with established functional consequences. In parallel, rare genetic variants that are not characterized and are, therefore, called variants of uncertain significance (VUSs) have also been detected. The classification of VUSs is a challenging task because each amino acid has specific biochemical properties and uniquely contributes to the structural stability and functional activity of proteins. In this scenario, the ability to computationally predict the effect of a VUS is crucial. In particular, in silico prediction methods can provide useful insights to assess the potential impact of a VUS and support additional clinical evaluation. This approach can further benefit from recent advances in artificial intelligence-based technologies. In this review, we describe the main in silico prediction tools that can be used to evaluate the structural and functional impact of VUSs and provide examples of their application in the analysis of gene variants involved in hereditary CRC syndromes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c45c256e9f192af359d986d6c8e8e5f1f020547" target='_blank'>
              In Silico Deciphering of the Potential Impact of Variants of Uncertain Significance in Hereditary Colorectal Cancer Syndromes
              </a>
            </td>
          <td>
            C. Fasano, Martina Lepore Signorile, Katia De Marco, G. Forte, Vittoria Disciglio, P. Sanese, V. Grossi, Cristiano Simone
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4d41c964ff5dfe59fec9c54a2237fb6fabb8d4" target='_blank'>
              Telomerase related molecular subtype and risk model reveal immune activity and evaluate prognosis and immunotherapy response in prostate cancer
              </a>
            </td>
          <td>
            Dongze Liu, Zheng Qin, Bo Yi, Hongbo Xie, Yunan Liang, Liang Zhu, Kuo Yang, Yong Xu, Hongtuan Zhang
          </td>
          <td>2024-08-17</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Overall survival for patients with advanced PDAC remains less than 1 year. Biomarkers are needed to select chemotherapy regimens, either alone or in combination with novel agents. PASS-01 compared mFOLFIRINOX and Gemcitabine/nab-Paclitaxel in a phase II randomised control trial while prospectively exploring tissue, liquid and organoid biospecimens to interrogate putative biomarkers of chemotherapy response and resistance. Methods PASS-01 was conducted across 6 sites in North America. Patients with de novo metastatic disease and a lesion amenable to core biopsy were included. Cases with germline pathogenic variants in BRCA1/2 or PALB2 were excluded. Up to 6 core biopsies were obtained with priorities for clinical diagnosis, patient derived organoids (PDOs), and whole genome and transcriptome sequencing (WGTS). WGTS were performed at a single site following laser capture microdissection. WGS reports were clinically (CAP) accredited and annotated mutations, copy number alterations, mutational signatures, TMB and structural variant patterns. KRAS allelic states were documented with major imbalances in mutant KRAS (KRASmaj) defined as mutant: WT copy number >3. Transcriptomic subtypes (classical vs. basal-like) were classified as previously described (Moffitt et al., 2015). WGTS results were discussed at a molecular tumor board with PDO pharmacotyping to determine best second line options. Results: 160 patients were included in the intention to treat population, of these WGS was available in 127 (79%) and additional NGS reports in 20 (13%). RNA-seq was successful in 119 (74%) and the basal-like subtype accounted for 29% and was associated with inferior outcomes. KRASm were evident in 91% and KRASG12D represented 45%. Tumor suppressors TP53, CDKN2A and SMAD4 were lost in 81%, 76% and 39% respectively. MTAP deletion was present in 40%, largely synonymous with CDKN2A deletions. Commonly co-altered genes in the dataset included epigenetic regulators (ARID1A, KMT2C, KDM6A) and TGFBR2. KRASmaj was present in 30% of cases and polyploid tumors in 50%. 50% of basal-like PDAC harboured KRASmaj compared to 22% of classical PDAC, p<0.01. Dominant signatures (SBS) included age related SBS1, 5 and 8. Additionally, we saw a prevalence of SBS17 and APOBEC SBS2/13, signatures which occurred more frequently compared to a similarly characterised resected cohort. Unstable genotypes were present in 24% with somatic-HRD evident in 3%. Of KRASWT PDAC, 58% harboured class I-II BRAF variants. KRAS WT PDAC had fewer co-mutations and in one case where an NGS report did not find a driver, a novel fusion in SLC4A4-RASGRF2 was identified. 28 molecular tumor boards took place throughout the trial influencing 50% obtaining a 2nd-line treatment. Conclusion: The PASS-01 cohort exhibits a heterogeneous group of genotypes and molecular subgroups. Certain mutational processes are more evident in advanced compared to resected PDAC. The impact of subgroups on survival and chemotherapy response is ongoing with a planned data lock on July 15th. Results will be presented at the meeting.
 Citation Format: Grainne M O'Kane, Gunho Jang, Trevor J Pugh, Julie Wilson, Stephanie Ramotar, Daniel A King, Dan Laheru, Ken H Yu, Kimberly J Perez, Amber Habowski, Erica S Tsang, Robert C Grant, Andrew J Aguirre, Raymond W-J Jang, Craig E Devoe, Eileen M O'Reilly, Anna Dodd, Brian M Wolpin, Xiang Y Ye, Elizabeth M Jaffee, David Tuveson, Steven Gallinger, Jennifer J Knox, Faiyaz Notta. Genomic and transcriptomic characterization of pancreatic cancer patients on the PASS-01 trial [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr PR-17.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65f655824e6bccfd53e0df1cf2553965b9da5ae0" target='_blank'>
              Abstract PR-17: Genomic and transcriptomic characterization of pancreatic cancer patients on the PASS-01 trial
              </a>
            </td>
          <td>
            G. O’Kane, G. Jang, T. J. Pugh, Julie M. Wilson, S. Ramotar, Daniel A King, Daniel A Laheru, Kenneth H Yu, Kimberly Perez, Amber N Habowski, Erica S Tsang, Robert C Grant, A. Aguirre, Raymond W-J Jang, Craig E. Devoe, E. O'Reilly, A. Dodd, B. Wolpin, X. Y. Ye, Elizabeth M. Jaffee, D. Tuveson, Stephen Gallinger, Jennifer J Knox, F. Notta
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>105</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="
 Objective: Structural variations (SVs) constitute the greatest source of genomic alterations leading to the oncogenesis of many cancers, including the deadliest pancreatic cancer (PC). However, SVs in PC remain largely undefined due to technological limitations of the traditional short-read sequencing. This study aims to establish a comprehensive signature of genomic SVs in PC. Methods: PacBio whole genome long-read sequencing was employed to investigate the genomic landscape of SVs in eight tissues from four pancreatic cancer patients. A variety of SVs callers, including pbsv, Sniffles, cuteSV, and svim, were comprehensively adopted to improve the accuracy of SVs detection. Many undefined characteristics of germline and somatic SVs were revealed by comparing with multiple previously published databases. Besides, the impacts of structural variations on 3D chromatin organization were analyzed by integrating multi-omics data including Hi-C, RNA seq and ChIP seq. Results: A total of 26,844 and 27,028 non-redundant SVs were identified in human pancreatic cancer and matched para-cancerous tissues, respectively. Notably, among these SVs, 23.97% were novel and had not been previously reported. Germline SVs were further characterized and found enriched in 25 well-known susceptible genes, such as TP63, PVT1, and ABO. Additionally, we proposed a panel of germline SVs with predicted highest pathogenicity, involving STOX1, DSPP, and WRN. Totally, 616 somatic SVs were identified and presented high burdens in patients with lymph node metastasis. We observed that somatic DELs and INSs were significantly more prevalent in repetitive regions than in non-repetitive regions. Interestingly, DELs showed a decrease in the proportion occurring in simple repeat regions but an increase in the proportion occurring in LINE and SINE regions compared to INSs. Moreover, it was observed that deletions within topologically associated domains were associated with local enhanced interactions and disrupted chromatin loops, which might influence H3K27ac modification of histone contributing to enhancer or silencer hijacking. Conclusion: This study provides insights into the genomic signature of SVs in human pancreatic cancer. These findings might shed new light on the complexity of SVs and their pathogenesis by interplaying with chromatin organization.
 Citation Format: Yongxing Du, Xiaohao Zheng, Yunjie Duan, Chengfeng Wang. Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A083.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40a9fb18df1c1c8a73e8c29f87767e5466baa02" target='_blank'>
              Abstract A083: Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing
              </a>
            </td>
          <td>
            Yongxing Du, Xiaohao Zheng, Yunjie Duan, Cheng-Guang Wang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="DNA methylation is an important epigenetic regulatory mechanism which plays a crucial role in cell differentiation and development. Its function is closely related to DNA methyltransferase 3 alpha (DNMT3A), which can affect gene expression and stem cell differentiation. The mutation rate of the DNMT3A gene is relatively high in Acute myeloid leukemia (AML), but its type and pathogenic mechanism are not yet clear. Further research on DNMT3A may help to identify its pathogenic targets and provide a basis for precise treatment of AML. This article has provided a review for the research progress on the expression of the DNMT3A gene in AML.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ec69c604a676713e1a7e86038752ff556999862" target='_blank'>
              [Research progress on DNMT3A gene expression in Acute myeloid leukemia].
              </a>
            </td>
          <td>
            Jiawei Zhou, Tao Wu, Wenhui Liu
          </td>
          <td>2024-08-10</td>
          <td>Zhonghua yi xue yi chuan xue za zhi = Zhonghua yixue yichuanxue zazhi = Chinese journal of medical genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The AKT1 oncogene is related to various cancers due to its critical role in the PIC3CA/AKT1 pathway; however, most of the studies screened the hotspot mutation AKT1 (E17K) with various incidences. Low frequency or lack of AKT1 (E17K) mutation was reported in prostate cancer (PC) patients. This study aims to explore genetic alterations in the AKT1 PH domain by extending the sequencing to include AKT1 gene exons 3 and 4. Genomic DNA was extracted from 84 Formalin-Fixed Paraffin-Embedded samples of PC patients in Jordan, and then subjected to PCR and sequencing for the targeted exons. This study revealed the presence of two novel mutations (N53Y and Q59K) and a high frequency of mutations in exon 4, with a lack of mutations in the E17K hotspot. Nine missense and two synonymous mutations were detected in exon 4 (Phe27Tyr, Phe27Leu, Ala58Thr, Ser56Phe, Arg41Trp, Phe35Leu, Asp32Glu, Phe35Tyr, and Gln43Lys) and (Ser56 and Glu40), respectively. Two synonymous mutations were detected in exon 3 (Leu12 and Ser2). It is concluded that there is a high frequency of AKT1 mutation in PC patients in Jordan with two novel missense mutations in the Pleckstrin homology (PH) domain. E17K hotspot mutation was not detected in any tested samples, which underlined the significant role of mutations in other AKT1 exons in PC development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/205851db262e85bd8c99969aa7adb86b40d5b2f6" target='_blank'>
              Novel Mutations in AKT1 Gene in Prostate Cancer Patients in Jordan
              </a>
            </td>
          <td>
            A. Alasmar, Zina M. Al-Alami, S. Zein, Asmaa Al-Smadi, Samir Al Bashir, Mohammed S. Alorjani, Raed M. Al‐Zoubi, M. A. Al Zoubi
          </td>
          <td>2024-09-04</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Prostate cancer (PC) is one of the most commonly diagnosed tumours among men. Second-generation androgen receptor axis-targeted (ARAT) agents, namely abiraterone acetate (AbA) and enzalutamide (ENZ), are currently used in the management of metastatic castration-resistant PC (mCRPC). However, the treatment is challenging due to the lack of prognostic biomarkers. Meanwhile, single-nucleotide polymorphisms (SNPs) have emerged as potential prognostic indicators of mCRPC. Thus, this study evaluated the impact of relevant SNPs on the treatment outcomes of 123 mCRPC patients enrolled in a hospital-based cohort study. The CYP17A1 rs2486758 C allele was associated with a 50% reduction in the risk of developing castration resistance (hazard ratio (HR) = 0.55; p = 0.003). Among patients without metastasis at tumour diagnosis and under AbA, a marginal association between YBX1 rs10493112 and progression-free survival was detected (log-rank test, p = 0.056). In the same subgroup, significant associations of HSD3B1 rs1047303 (CC/CA vs. AA; HR = 3.41; p = 0.025), YBX1 rs12030724 (AT vs. AA; HR = 3.54; p = 0.039) and YBX1 rs10493112 (log-rank test, p = 0.041; CC vs. AA/AC; HR = 3.22; p = 0.053) with overall survival were also observed, which were confirmed by multivariate Cox analyses. Although validation with larger cohorts is required, these findings suggest that SNPs could enhance the prognosis assessment of mCRPC patients, leading to a more personalised treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70a3c0fa449c43e80a192951ef4bc8a7cc97f2bc" target='_blank'>
              Connecting Gene Variation to Treatment Outcomes in Metastatic Castration-Resistant Prostate Adenocarcinoma: Insights into Second-Generation Androgen Receptor Axis-Targeted Therapies
              </a>
            </td>
          <td>
            Ana Vaz-Ferreira, Valéria Tavares, Inês Guerra de Melo, Patrícia Rafaela Rodrigues, Ana Afonso, Maria Joaquina Maurício, Rui Medeiros
          </td>
          <td>2024-09-12</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Impaired control of the G1/S checkpoint allows initiation of DNA replication under non-permissive conditions. Unscheduled S-phase entry is associated with DNA replication stress, demanding for other checkpoints or cellular pathways to maintain proliferation. Here, we uncovered a requirement for ADARp150 to sustain proliferation of G1/S-checkpoint-defective cells under growth-restricting conditions. Besides its well-established mRNA editing function in inversely oriented short interspersed nuclear elements (SINEs), we found ADARp150 to exert a critical function in mitosis. ADARp150 depletion resulted in tetraploidization, impeding cell proliferation in mitogen-deprived conditions. Mechanistically we show that ADAR1 depletion induced aberrant expression of Cyclin B3, which was causative for mitotic failure and whole-genome duplication. Finally, we find that also in vivo ADAR1-depletion-provoked tetraploidization hampers tumor outgrowth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d2bb8be1edc7659c2bc78816fc95a0c749bf23" target='_blank'>
              ADARp150 counteracts whole genome duplication.
              </a>
            </td>
          <td>
            Frank van Gemert, Alexandra Drakaki, Isabel Morales Lozano, Daniel de Groot, Maud Schoot Uiterkamp, N. Proost, C. Lieftink, M. van de Ven, R. Beijersbergen, Heinz Jacobs, H. te Riele
          </td>
          <td>2024-08-27</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Triplex DNA formation is a useful genomic targeting tool that is expected to have a wide range of applications, including the antigene method; however, there are fundamental limitations in its forming sequence. We recently extended the triplex DNA-forming sequence to methylated DNA sequences containing 5mCG base pairs by developing guanidino-dN, which is capable of recognizing a 5mCG base pair with high affinity. We herein investigated the effect of triplex DNA formation using TFOs with guanidino-dN on methylated DNA sequences at the promoter of the RASSF1A gene, whose expression is epigenetically suppressed by DNA methylation in MCF-7 cells, on gene expression. Interestingly, triplex DNA formation increased the expression of the RASSF1A gene at the transcript and protein levels. Furthermore, RASSF1A-activated MCF-7 cells exhibited cell growth suppressing activity. Changes in the expression of various genes associated with the promotion of apoptosis and breast cancer survival accompanied the activation of RASSF1A in cells exhibited antiproliferative activity. These results suggest the potential of increases in gene expression through triplex DNA formation as a new genomic targeting tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/367d9640afcf3f528a05a1512247caf159507678" target='_blank'>
              Novel strategy for activating gene expression through triplex DNA formation targeting epigenetically suppressed genes
              </a>
            </td>
          <td>
            Ryotaro Notomi, Shigeki Sasaki, Yosuke Taniguchi
          </td>
          <td>2024-07-31</td>
          <td>RSC Chemical Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) can manifest as de novo AML (dn-AML) or secondary AML (s-AML), with s-AML being associated with inferior survival and distinct genomic characteristics. The underlying reasons for this disparity remain to be elucidated. In this multicenter study, next-generation sequencing (NGS) was employed to investigate the mutational landscape of AML in 721 patients from June 2020 to May 2023.Genetic mutations were observed in 93.34% of the individuals, with complex variations (more than three gene mutations) present in 63.10% of them. TET2, ASXL1, DNMT3A, TP53 and SRSF2 mutations showed a higher prevalence among older individuals, whereas WT1 and KIT mutations were more commonly observed in younger patients. BCOR, BCORL1, ZRSR2, ASXL1 and SRSF2 exhibited higher mutation frequencies in males. Additionally, ASXL1, NRAS, PPMID, SRSF2, TP53 and U2AF1 mutations were more common in patients with s-AML, which PPM1D was more frequently associated with therapy-related AML (t-AML). Advanced age and hyperleukocytosis independently served as adverse prognostic factors for both types of AML; however, s-AML patients demonstrated a greater number of monogenic adverse prognostic factors compared to dn-AML cases (ASXL1, PPM1D, TP53 and U2AF1 in s-AML vs. FLT3, TP53 and U2AF1 in dn-AML). Age and sex-related gene mutations suggest epigenetic changes may be key in AML pathogenesis. The worse prognosis of s-AML compared to dn-AML could be due to the older age of s-AML patients and more poor-prognosis gene mutations. These findings could improve AML diagnosis and treatment by identifying potential therapeutic targets and risk stratification biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f06fb6d75db866688d32fbb738b2b52aacbc718" target='_blank'>
              Genomic mutation patterns and prognostic value in de novo and secondary acute myeloid leukemia: A multicenter study from China.
              </a>
            </td>
          <td>
            Xi Dou, Chunli Dan, Duanzhong Zhang, Hong-Sheng Zhou, Renke He, Guangyu Zhou, Yu Zhu, Nan Fu, Ben Niu, Shuangnian Xu, Yi Liao, Zhangqin Luo, Lihua Yang, Haiguo Zhang, Yizhi Xu, Q. Zhan, Wei Chen, Zesong Yang, Xiaoqiong Tang, Hong-bin Zhang, Qing Xiao, Jianbin Chen, Lin Liu, Yi Wang, Li Pei, Li Wang
          </td>
          <td>2024-08-07</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genes on sex chromosomes have higher evolutionary rates than those on autosomes. However, this does not necessarily apply to somatic evolution in cancer. Many dominant mutations have been described in the so-called proto-oncogenes (OGs), while recessive mutations are typically described in tumor-suppressor genes (TSGs). Evidence indicates that mutations in X-chromosome TSGs are more likely to contribute to cancer than those in autosomal TSGs. Here, we formalize this in several dynamic models and predict, as expected, that mutations spread faster in TSGs located on the X chromosome than on autosomes (faster-X effect). Conversely, mutations in OGs spread faster on autosomes than on the X chromosome, but under high selective pressure, this difference is negligible. Published genomic screenings of cancer samples show evidence of the faster-X effect in TSGs. This pattern is observed in both sexes, suggesting that the maintenance of X-chromosome inactivation during cancer progression plays an important role in the evolution of TSGs. Strikingly, the relative mutation incidence in X-linked TSGs among females across individual studies is bimodal, with one group of studies showing a faster-X effect and another group showing similar incidences for X-linked and autosomal TSGs. This differentiation between cancer samples is not associated with the specific type of cancer or the tissue of origin. This may indicate that X-chromosome inactivation plays a differential role in the involvement of X-linked TSGs across individual cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e60393ec97b0f4b9cf086f3cab7030b54b21b01" target='_blank'>
              Somatic evolution of cancer genes in sex chromosomes
              </a>
            </td>
          <td>
            Amarachi Akachukwu, Jasmine S. Ratcliff, Antonio Marco
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CRISPR/Cas9 gene editing in the model green alga Chlamydomonas reinhardtii relies on the use of selective marker genes to enrich for non-selectable target mutations. This becomes challenging when many sequential modifications are required in a single cell line, as useful markers are limited. Here, we demonstrate a cyclical selection process which only requires a single marker gene to identify an almost infinite sequential series of CRISPR-based target gene modifications. We used the NIA1 (Nit1, NR; nitrate reductase) gene as the selectable marker in this study. In the forward stage of the cycle, a stop codon was engineered into the NIA1 gene at the CRISPR target location. Cells retaining the wild-type NIA1 gene were killed by chlorate, while NIA1 knockout mutants survived. In the reverse phase of the cycle, the stop codon engineered into the NIA1 gene during the forward phase was edited back to the wild-type sequence. Using nitrate as the sole nitrogen source, only the reverted wild-type cells survived. By using CRISPR to specifically deactivate and reactivate the NIA1 gene, a marker system was established that flipped back and forth between chlorate- and auxotrophic (nitrate)-based selection. This provided a scarless cyclical marker system that enabled an indefinite series of CRISPR edits in other, non-selectable genes. We demonstrate that this 'Sequential CRISPR via Recycling Endogenous Auxotrophic Markers (SCREAM)' technology enables an essentially limitless series of genetic modifications to be introduced into a single cell lineage of C. reinhardtii in a fast and efficient manner to complete complex genetic engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f75908314649d1188956bda26ea29417db966e6a" target='_blank'>
              A cyclical marker system enables indefinite series of oligonucleotide-directed gene editing in Chlamydomonas reinhardtii.
              </a>
            </td>
          <td>
            Ian L. Ross, Hong Phuong Le, Sabar Budiman, Dake Xiong, Fritz Hemker, Elizabeth A. Millen, Melanie Oey, Ben Hankamer
          </td>
          <td>2024-08-23</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequent and mortality-causing neoplasia, with various distributions between populations. Strong hereditary predispositions are the causatives of a small percentage of CRC, and most cases have no transparent genetic background. This is a vast arena for exploring cancer low-susceptibility genetic variants. Nonetheless, the research that has been conducted to date has failed to deliver consistent conclusions and often features conflicting messages, causing chaos in this field. Therefore, we decided to organize the existing knowledge on this topic. We screened the PubMed and Google Scholar databases. We drew up markers by gene locus gathered by hallmark: oncogenes, tumor suppressor genes, genes involved in DNA damage repair, genes involved in metabolic pathways, genes involved in methylation, genes that modify the colonic microenvironment, and genes involved in the immune response. Low-penetration genetic variants increasing the risk of cancer are often population-specific, hence the urgent need for large-scale testing. Such endeavors can be successful only when financial decision-makers are united with social educators, medical specialists, genetic consultants, and the scientific community. Countries’ policies should prioritize research on this subject regardless of cost because it is the best investment. In this review, we listed potential low-penetrance CRC susceptibility alleles whose role remains to be established.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0541d34727973134a35fb48d39ef83a36856b18" target='_blank'>
              Low-Penetrance Susceptibility Variants in Colorectal Cancer—Current Outlook in the Field
              </a>
            </td>
          <td>
            Marcin Szuman, M. Kaczmarek-Ryś, S. Hryhorowicz, Alicja Kryszczyńska, Natalia Grot, Andrzej Pławski
          </td>
          <td>2024-07-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>12</td>
        </tr>

        <tr id="High-throughput transcriptomics (HTTr) is increasingly being used to identify molecular targets of chemicals that can be linked to adverse outcomes. Cell proliferation is an important key event in chemical carcinogenesis. Here, we describe the construction and characterization of a gene expression biomarker that is predictive of the cell proliferation (CP) status in human and rodent tissues. The biomarker was constructed from 30 genes known to be increased in expression in prostate cancers relative to surrounding tissues and in cycling human MCF-7 cells after estrogen receptor (ER) agonist exposure. Using a large compendium of gene expression profiles to test utility, the biomarker could identify increases in cell proliferation in 1) 367 tumor vs. normal surrounding tissue comparisons from 6 human organs, 2) MCF-7 cells after activation of ER, 3) after partial hepatectomy in mice and rats, and 4) the livers of mice after exposure to nongenotoxic hepatocarcinogens. The biomarker identified suppression of CP 1) under conditions of p53 activation by DNA damaging agents in human cells, 2) in human A549 lung cells exposed to therapeutic anticancer kinase inhibitors (dasatinib, nilotnib) and 3) in the mouse liver when comparing high levels of CP at birth to the low background levels in the adult. The responses using the biomarker were similar to those observed using conventional markers of CP including PCNA, Ki67, and BrdU labeling. The CP biomarker will be a useful tool for interpretation of HTTr data streams to identify cell proliferation status after exposure to chemicals in human cells or in rodent tissues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e67eda2694923151226daf541a133cfb12edda84" target='_blank'>
              A transcriptomic biomarker predictive of cell proliferation for use in adverse outcome pathway-informed testing and assessment.
              </a>
            </td>
          <td>
            J. Corton, Victoria Ledbetter, Samuel M. Cohen, E. Atlas, Carole L Yauk, Jie Liu
          </td>
          <td>2024-08-13</td>
          <td>Toxicological sciences : an official journal of the Society of Toxicology</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Two new diagnostic classifications of acute myeloid leukemia (AML) were published in 2022 to update current knowledge on disease biology. In previous 2017-edition categories of AML with myelodysplasia-related changes, AML was not otherwise specified, but AML with mutated RUNX1 experienced profound changes. We performed whole exome sequencing on a cohort of 69 patients with cytogenetic intermediate-risk AML that belonged to these diagnostic categories to correlate their mutational pattern and copy-number alterations with their new diagnostic distribution. Our results show that 45% of patients changed their diagnostic category, being AML myelodysplasia-related the most enlarged, mainly due to a high frequency of myelodysplasia-related mutations (58% of patients). These showed a good correlation with multilineage dysplasia and/or myelodysplastic syndrome history, but at the same time, 21% of de novo patients without dysplasia also presented them. RUNX1 was the most frequently mutated gene, with a high co-occurrence rate with other myelodysplasia-related mutations. We found a high prevalence of copy-neutral loss of heterozygosity, frequently inducing a homozygous state in particular mutated genes. Mild differences in current classifications explain the diagnostic disparity in 10% of patients, claiming a forthcoming unified classification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96bc336d75b06c106306099959e9bc35ef3c19e7" target='_blank'>
              Whole Exome Sequencing of Intermediate-Risk Acute Myeloid Leukemia without Recurrent Genetic Abnormalities Offers Deeper Insights into New Diagnostic Classifications
              </a>
            </td>
          <td>
            F. Guijarro, Sandra Castaño-Díez, C. Jiménez-Vicente, M. Garrote, J. Álamo, Marta Gómez-Hernando, I. López-Oreja, Jordi Morata, M. López-Guerra, Cristina López, S. Beà, D. Costa, Dolors Colomer, M. Díaz-Beyá, M. Rozman, Jordi Esteve
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9be07b4d9d0fbf94fc6b0404b66be597056986cc" target='_blank'>
              Extended replicative lifespan of primary resting T cells by CRISPR/dCas9-based epigenetic modifiers and transcriptional activators
              </a>
            </td>
          <td>
            Siping Huang, Cia-Hin Lau, C. Tin, Raymond H W Lam
          </td>
          <td>2024-09-17</td>
          <td>Cellular and Molecular Life Sciences: CMLS</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1e1516a70ceb6cf40f46931b69cd383f8042a08" target='_blank'>
              PTBP1-mediated repression of neuron-specific CDC42 splicing constitutes a genomic alteration-independent, developmentally conserved vulnerability in IDH-wildtype glioblastoma.
              </a>
            </td>
          <td>
            Junjie Yang, Jing Feng, Jing Lv, Xiaojing Chu, Yanfei Wei, Yunqiu Zhang, Jiuyi Li, Yingyu Sun, Guanzhang Li, Tao Jiang, Jinyan Huang, Xiaolong Fan
          </td>
          <td>2024-08-01</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Enhancer elements are specific DNA sequences that play a crucial role in regulating gene expression. Located upstream or downstream in the genomic context, enhancers enhance the transcription of linked genes. This is achieved by providing binding sites for transcription factors and other proteins, acting as a nucleation point that promotes the assembly of the transcriptional machinery.
What makes enhancers unique is that located even thousands of base pairs away, they can exert their function. They can act over long distances, looping to interact with the promoter region of their target genes. Enhancers are often involved in cell type-specific gene expression as different cells express different sets of genes by activating and repressing cell type-specific enhancers.
Cancer cells seem to co-opt this mechanism and hijack it for their own survival. In this process, an aberrant enhancer element activates the transcription of oncogenes (genes that have the potential to cause cancer) due to some alterations in genomic structure or regulatory elements that happen in cancer, contributing to tumorigenesis and cancer progression. Enhancer hijacking often occurs through complex genomic rearrangements such as chromosomal translocations and other structural rearrangements such as deletions or amplifications.8 These alterations can bring a usually distant, far-located enhancer into proximity with an oncogene, leading to its inappropriate activation. This is often seen in various cancers, including leukemias and solid tumors.
Enhancer hijacking and its effects on gene regulation in cancer are frequently studied using methods such as RNA sequencing, chromatin conformation capture, and CRISPR-Cas9. Finding an enhancer hijacking event may be used to inform treatment choices, especially in precision medicine settings, and may also function as a biomarker for particular cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefe88575de8420490e0e1dd10a24dbefe2eef41" target='_blank'>
              Enhancer hijacking: Innovative ways of carcinogenesis
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-09-02</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823edc7b7f2e48466de67d70b29a3e25297b1b88" target='_blank'>
              Cooperative genomic lesions in HRAS-mutant cancers predict resistance to farnesyltransferase inhibitors.
              </a>
            </td>
          <td>
            Arad Nigam, Gnana P. Krishnamoorthy, Walid K. Chatila, Katherine Berman, Mahesh Saqcena, H. Walch, Mandakini Venkatramani, Alan L. Ho, N. Schultz, James A. Fagin, B. Untch
          </td>
          <td>2024-08-16</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1f5936217eac5b4043bb1665424b2339a9f20d" target='_blank'>
              MYC and KRAS cooperation: from historical challenges to therapeutic opportunities in cancer
              </a>
            </td>
          <td>
            S. Casacuberta-Serra, Íñigo González-Larreategui, Daniel Capitán-Leo, Laura Soucek
          </td>
          <td>2024-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Background Minichromosome maintenance complex component 3 associated protein (MCM3AP) is a gene in which mutations can result in autosomal recessive peripheral neuropathy with or without impaired intellectual development. The MCM3AP genotype-phenotype correlation and prognosis remain unclear. The aim of this study was to explore the genotype–phenotype correlations pertaining to MCM3AP. Methods Whole-exome sequencing (WES) combined with copy number variation sequencing (CNV-seq) were performed on the genomic DNA isolated from a Chinese family, and Sanger sequencing, quantitative PCR and cDNA analyses were performed to examine the mutations. The retrospective study was conducted on 28 individuals with biallelic MCM3AP mutation-related diseases, including features such as mutations, motor development impairment, intellectual disability, weakness/atrophy, and cerebral magnetic resonance imaging abnormalities. Results Sequencing identified novel compound heterozygous mutations in MCM3AP, namely, a paternal variant c.1_5426del (loss of exons 1–25) and a maternal splicing variant c.1858 + 3A>G. Functional studies revealed that the variant c.1858 + 3A>G resulted in the heterozygous deletion of exon 5, thereby affecting splicing functionality. Furthermore, the compound heterozygous mutation may affect the functionality of the protein domain. Retrospective analysis revealed different genotype–phenotype correlations for the pathogenic variants in biallelic MCM3AP: all individuals (100%) with mutations outside the Sac3 domain exhibited early-onset symptoms, motor developmental delays, and cognitive abnormalities, conversely, the proportions of individuals carrying mutations within the domain were 26.7% (motor delays) and 46.7% (cognitive abnormalities). Conclusion Our findings further expand the genetic mutation spectrum of biallelic MCM3AP and highlight the genotype-phenotype associations. Additionally, we elaborate on the importance of rehabilitation intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc14d90ab7ba7ceb9ba0f635353a560c5f6b553f" target='_blank'>
              Identification of biallelic mutations in MCM3AP and comprehensive literature analysis
              </a>
            </td>
          <td>
            Chan Liu, Qingfeng Xie, Quan Hu, Bingwu Xiang, Kaiyi Zhao, Xiang Chen, Feixia Zheng
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Major epigenetic changes are associated with carcinogenesis, including aberrant DNA methylations and post-translational modifications of histone. Indeed evidence accumulated in recent years indicates that inactivating DNA hypermethylation preferentially targets the subset of polycomb group (PcG) genes that are regulators of developmental processes. Conversely, activating DNA hypomethylation targets oncogenic signaling pathway genes, but outcomes of both events lead in the overexpression of oncogenic signaling pathways that contribute to the stem-like state of cancer cells. On the basis of recent evidence from population-basedclinical and experimental studies, we hypothesize that factors associated with risk for developing a hematologic malignancy (HM), such as metabolic syndrome and chronic inflammation, may trigger epigenetic mechanisms to increase the transcriptional expression of oncogenes and activate oncogenic signaling pathways. Signaling pathways associated with such risk factors include but are not limited to pro-inflammatory nuclear factor κB (NF-κB) and mitogenic, growth, and survival Janus kinase (JAK) intracellular non-receptor tyrosine kinase-triggered pathways. The latter includes signaling pathways such as transducer and activator of transcription (STAT), Ras GTPases/mitogen-activated protein kinases (MAPKs)/extracellular signal-related kinases (ERKs), phosphatidylinositol 3-kinase (PI3K)/Akt/mammalian target of rapamycin (mTOR), and β-catenin pathways. Recent findings on epigenetic mechanisms at work in the biology of cancer and in HMs and their importance in the etiology and pathogenesis of these diseases are herein summarized and discussed. Furthermore, the role of epigenetic processes in the determination of biological identity, the consequences for interindividual variability in disease clinical profile, and the potential of epigenetic drugs in HMs are also considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135146cf67f22c155000b7b9f10163440a9d3db5" target='_blank'>
              Role of epigenetic in cancer biology, in hematologic malignancies and in anticancer therapy
              </a>
            </td>
          <td>
            A. H. Nwabo Kamdje, Hervet P Dongmo Fogang, Patrice N. Mimche
          </td>
          <td>2024-09-06</td>
          <td>Frontiers in Molecular Medicine</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="BACKGROUND
Chronic myeloid leukemia (CML), a myeloproliferative neoplasm defined by the BCR::ABL1 fusion gene arising from the Philadelphia chromosome (Ph) translocation t(9:22)(q34;q11), exhibits diverse clinical courses often influenced by additional chromosomal aberrations (ACAs). This report presents a case of CML har-boring a novel four-way Ph translocation involving the X chromosome, offering insights into the interplay between complex karyotypes and treatment response and emphasizing the need for further research into the role of ACAs in CML management.


METHODS
A 42-year-old man diagnosed with CML in the accelerated phase presented a novel four-way Ph translocation involving chromosomes X, 5, 9, and 22: 46,Y,t(X;5;9;22)(q26;q15;q34;q11.2). Despite achieving a major molecular response initially with imatinib and nilotinib, BCR::ABL1 levels (international scale) increased up to 24.0%, which prompted the use of second-line nilotinib.


RESULTS
Follow-up bone marrow (BM) studies revealed clonal evolution with trisomy 8 and an unclassified ABL1 mutation (E292V), potentially contributing to resistance. Though a transient major molecular response (MMR) occurred after a switch to third-line dasatinib, this change failed to achieve a deep molecular response, and BCR-ABL1 levels were elevated above the MMR.


CONCLUSIONS
This case highlights the challenge of ACAs impacting CML treatment response and prognosis. Limited knowledge exists on complex Ph translocations involving the X chromosome, but this report contributes data for further research. Understanding ACA effects on therapeutic response and prognosis requires a detailed study of such complex chromosomal aberrations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25ceb43112ac67026716ad67804a2634518ff113" target='_blank'>
              A Rare Case of X-Linked Four-Way Philadelphia Chromosome Translocation with Therapeutic Challenges and Clonal Evolution.
              </a>
            </td>
          <td>
            Soo Eung Park, J. Lee, Hye Jin Kang, Y. Hong, Ae-Chin Oh, Heyjin Kim
          </td>
          <td>2024-09-01</td>
          <td>Clinical laboratory</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Simple Summary Individualized medicine means understanding how each tumor is different from normal cells and how each tumor is different from other tumors, including profiling mutations, non-mutational epigenetic changes, and differences in gene expression. This allows the discovery of key processes each tumor absolutely depends on for survival and growth, which are intrinsic weaknesses. This profiling means selecting treatments to specifically target each tumor’s survival-dependent pathways, killing them. Tip60 is a master controller of processes that maintain genome stability and signaling regulating gene expression. While disrupted in many cancers, Tip60 is essential for cell survival, and inhibiting Tip60 kills tumors. While we understand some key aspects of the molecular roles Tip60 plays, much more remains to be discovered. A more complete understanding of the diverse roles and functions of Tip60 in cancer, and how targeting Tip60 kills cancer cells, will lead to better treatments for patients and increased survival. Abstract Precision (individualized) medicine relies on the molecular profiling of tumors’ dysregulated characteristics (genomic, epigenetic, transcriptomic) to identify the reliance on key pathways (including genome stability and epigenetic gene regulation) for viability or growth, and then utilises targeted therapeutics to disrupt these survival-dependent pathways. Non-mutational epigenetic changes alter cells’ transcriptional profile and are a key feature found in many tumors. In contrast to genetic mutations, epigenetic changes are reversable, and restoring a normal epigenetic profile can inhibit tumor growth and progression. Lysine acetyltransferases (KATs or HATs) protect genome stability and integrity, and Tip60 is an essential acetyltransferase due to its roles as an epigenetic and transcriptional regulator, and as master regulator of the DNA double-strand break response. Tip60 is commonly downregulated and mislocalized in many cancers, and the roles that mislocalized Tip60 plays in cancer are not well understood. Here we categorize and discuss Tip60-regulated genes, evaluate Tip60-interacting proteins based on cellular localization, and explore the therapeutic potential of Tip60-targeting compounds as epigenetic inhibitors. Understanding the multiple roles Tip60 plays in tumorigenesis will improve our understanding of tumor progression and will inform therapeutic options, including informing potential combinatorial regimes with current chemotherapeutics, leading to improvements in patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0558cedd75f278d6feed6575b276aad74d694cf" target='_blank'>
              Evaluating the Cellular Roles of the Lysine Acetyltransferase Tip60 in Cancer: A Multi-Action Molecular Target for Precision Oncology
              </a>
            </td>
          <td>
            Nazanin Zohourian, Erin Coll, Muiread Dever, Anna Sheahan, Petra Burns-Lane, James A. L. Brown
          </td>
          <td>2024-07-27</td>
          <td>Cancers</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37af8177474f5214e32408bc13029fb9fea726d5" target='_blank'>
              Gene therapy for ultrarare diseases: a geneticist’s perspective
              </a>
            </td>
          <td>
            W. Hwu
          </td>
          <td>2024-08-13</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b0e84c70be39b55c5ab372a108ddda6ab970765" target='_blank'>
              Polθ: emerging synthetic lethal partner in homologous recombination-deficient tumors.
              </a>
            </td>
          <td>
            T. B. Bazan Russo, Clarissa Mujacic, Emilia Di Giovanni, Maria Concetta Vitale, Carla Ferrante Bannera, U. Randazzo, Silvia Contino, M. Bono, V. Gristina, A. Galvano, A. Perez, G. Badalamenti, A. Russo, V. Bazan, L. Incorvaia
          </td>
          <td>2024-08-09</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa9a049feca8fb402096fb6d84d95a6531213f49" target='_blank'>
              CHO cell engineering via targeted integration of circular miR-21 decoy using CRISPR/RMCE hybrid system
              </a>
            </td>
          <td>
            Setare Adibzadeh, S. Amiri, F. Barkhordari, S. Mowla, Hadi Bayat, S. Ghanbari, F. Faghihi, Fatemeh Davami
          </td>
          <td>2024-08-09</td>
          <td>Applied Microbiology and Biotechnology</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc721cd925975049ceb0cd27f9ffa4efdce8611" target='_blank'>
              Synthetic lethality by PARP inhibitors: new mechanism uncovered based on unresolved transcription-replication conflicts
              </a>
            </td>
          <td>
            Marina Kolesnichenko, Claus Scheidereit
          </td>
          <td>2024-07-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/263cda8d76b200b36443f700782153fb43cb23ea" target='_blank'>
              Increased AID results in mutations at the CRLF2 locus implicated in Latin American ALL health disparities
              </a>
            </td>
          <td>
            Valeria Rangel, Jason N. Sterrenberg, Aya Garawi, Vyanka Mezcord, Melissa L. Folkerts, Sabrina E. Calderon, Yadhira E. Garcia, Jinglong Wang, Eli M. Soyfer, Oliver S Eng, J. Valerin, S. Tanjasiri, Fabiola Quintero-Rivera, M. Seldin, Selma Masri, Richard L. Frock, Angela G. Fleischman, Nicholas R Pannunzio
          </td>
          <td>2024-07-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83e1b808c0ad8ebc0432fdf9718775113703bf08" target='_blank'>
              Multi-regional genomic and transcriptomic characterization of a melanoma-associated oral cavity cancer provide evidence for CASP8 alteration-mediated field cancerization
              </a>
            </td>
          <td>
            Shouvik Chakravarty, Arnab Ghosh, Chitrarpita Das, Subrata Das, S. Patra, Arindam Maitra, S. Ghose, N. Biswas
          </td>
          <td>2024-09-07</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7401c81aea35658fe72a5961ba5c3e31cae1d2fa" target='_blank'>
              The genomic basis of childhood T-lineage acute lymphoblastic leukaemia.
              </a>
            </td>
          <td>
            P. Pölönen, Danika Di Giacomo, A. Seffernick, Abdelrahman A. Elsayed, Shunsuke Kimura, Francesca Benini, Lindsey E Montefiori, Brent L Wood, Jason Xu, Changya Chen, Zhong-Yi Cheng, H. Newman, Jason R Myers, Ilaria Iacobucci, Elizabeth Li, Jonathan H. Sussman, Dale Hedges, Yawei Hui, Caroline Diorio, Lahari Uppuluri, David Frank, Yiping Fan, Yunchao Chang, S. Meshinchi, R. Ries, R. Shraim, Alexander Li, Kathrin M Bernt, M. Devidas, S. Winter, K. Dunsmore, H. Inaba, William L Carroll, Nilsa C Ramirez, Aaron H. Phillips, Richard W. Kriwacki, Jun J Yang, T. Vincent, Yaqi Zhao, P. S. Ghate, Jian Wang, Colleen Reilly, Xin Zhou, Mathijs A. Sanders, Junko Takita, Motohiro Kato, Nao Takasugi, Bill H Chang, Richard D Press, M. Loh, E. Rampersaud, E. Raetz, S P Hunger, Kai Tan, Ti-Cheng Chang, Gang Wu, S. B. Pounds, C. Mullighan, D. Teachey
          </td>
          <td>2024-08-14</td>
          <td>Nature</td>
          <td>2</td>
          <td>102</td>
        </tr>

        <tr id="Background. Prostate cancer (PCa) is one of the most common cancers of men. In 2022, 48,025 new PCa cases and 12,896 deaths from the disease were reported in Russia. Metastatic castration-resistant PCa still is one of the common causes of death in men.Aim. To evaluate prevalence of homologous recombination repair gene (HRR) mutations in patients with metastatic castration-resistant PCa in Russia and to identify differences in clinical characteristics and treatment outcomes of patients with these mutations and without them.Materials and methods. The study included 329 patients with metastatic castration-resistant PCa from 20 centers. Patients underwent molecular genetic analysis using next-generation sequencing to identify mutations in 14 HRR genes. The mutation status was determined in 3 routine laboratories and further validated in a central independent laboratory.Results. The HRR gene mutations were detected in 59 (19.28 %) of 329 patients. The most common were ATM mutations – in 14 (4.57 %), BRCA2 – in 14 (4.57 %), BRCA1 – in 6 (1.96 %). All the patients received first-line therapy, with newgeneration antiandrogens and taxanes being used most frequently. The patients with HRR gene mutations had nearly1.5 times lower median of progression-free survival (PFS) with first-line therapy: 12.8 months vs. 20.8 months in patients without mutations (p = 0.048). There was statistically significant difference in PFS between the observed groups with first-line therapy depending on the treatment group. In patients of general group having been treated with new-generation antiandrogen therapy, the median of PFS was 22 months, while in patients who had received taxanes it was 8.22 months (p <0.05). In patients with HRR gene mutations having been treated with first-line antiandrogenes of novel generation, the median of PFS was 20.5 months, while in patients without mutations the PFS median was 23.1 months (p = 0.14). In patients with mutations and having been treated with first-line taxanes, the PFS median was 6.15 months, while in patients without mutations it was 8.55 months (p = 0.4).Conclusion. In our ADAM study in the Russian population, the proportion of carriers of the HRR gene mutations was slightly lower (19.28 %) than that reported in published randomized trials. There is also a different distribution structure and frequency of the most frequent mutations. Patients with the HRR gene mutations exhibited worse PFS rates and more aggressive course of the disease, which requires a special approach to treatment of this group of patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9726667fcd620604b264c956c112f113815dddcc" target='_blank'>
              Multicenter non-interventional study of prevalence of homologous recombination gene mutations and approaches to treatment of metastatic castration-resistant prostate cancer in Russia (ADAM)
              </a>
            </td>
          <td>
            B. Alekseev, Yu. V. Anzhiganova, A. V. Sultanbaev, A. Bystrov, A. Orlov, G. Gopp, A. V. Kopyltsov, A. Lykov, V. A. Atduev, G. Alekseeva, L. M. Rodygin, S. A. Varlamov, A. V. Shestakov, M. R. Maturov, M. F. Urmantsev, L. V. Lankina, E. Parsadanova, V. Y. Startsev, O. Vedrova, S. Andreev, A. A. Perevoshchikov, O. A. Mailyan, A. Kalpinskiy
          </td>
          <td>2024-08-17</td>
          <td>Cancer Urology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c4d24999230a049c47c4ad63da565877c2b583c" target='_blank'>
              PPM1D activity promotes cellular transformation by preventing senescence and cell death.
              </a>
            </td>
          <td>
            Miroslav Stoyanov, Andra S. Martinikova, Kateřina Matějková, K. Horackova, P. Zemánková, Kamila Burdová, Zuzana Zemanova, P. Kleiblova, Z. Kleibl, L. Macurek
          </td>
          <td>2024-09-05</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/befebce4fc2c4b9cfeedafc2e4927bdb0544dbd9" target='_blank'>
              Mechanisms that clear mutations drive field cancerization in mammary tissue
              </a>
            </td>
          <td>
            Marta Ciwinska, H. Messal, Hristina R Hristova, C. Lutz, Laura Bornes, Theofilos Chalkiadakis, Rolf Harkes, Nathalia S M Langedijk, S. Hutten, Renée X Menezes, Jos Jonkers, Stefan Prekovic, Ben Simons, C. Scheele, J. van Rheenen
          </td>
          <td>2024-09-01</td>
          <td>Nature</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="By analyzing two large atlases of almost 4 million cells, we show that immune-senescence involves a gradual loss of cellular identity, reflecting increased cellular heterogeneity, for effector, and cytotoxic immune cells. The effects are largely similar in both males and females and were robustly reproduced in two atlases, one assembled from 35 diverse studies including 678 adults, the other the OneK1K study of 982 adults. Since the mean transcriptional differences among cell-types remain constant across age deciles, there is little evidence for the alternative mechanism of convergence of cell-type identity. Key pathways promoting activation and stemness are down-regulated in aged T cells, while CD8 TEM and CD4 CTLs exhibited elevated inflammatory, and cytotoxicity in older individuals. Elevated inflammatory signaling pathways, such as MAPK and TNF-alpha signaling via NF-kB, also occur across all aged immune cells, particularly amongst effector immune cells. This finding of lost transcriptional identity with age carries several implications, spanning from a fundamental biological understanding of aging mechanisms to clinical perspectives on the efficacy of immunomodulation in elderly people.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d768b64d5bd77b266a4e1004a43d2aff5288db6d" target='_blank'>
              Loss of immune cell identity with age inferred from large atlases of single cell transcriptomes.
              </a>
            </td>
          <td>
            Erin Connolly, Tony Pan, Maneesha Aluru, Sriram P. Chockalingam, Vishal Dhere, Greg Gibson
          </td>
          <td>2024-08-14</td>
          <td>Aging cell</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Chimeric RNAs, distinct from DNA gene fusions, have emerged as promising therapeutic targets with diverse functions in cancer treatment. However, the functional significance and therapeutic potential of most chimeric RNAs remain unclear. Here we identify a novel fusion transcript of solute carrier family 2‐member 11 (SLC2A11) and macrophage migration inhibitory factor (MIF). In this study, we investigated the upregulation of SLC2A11–MIF in The Cancer Genome Atlas cohort and a cohort of patients from Sun Yat‐Sen Memorial Hospital. Subsequently, functional investigations demonstrated that SLC2A11–MIF enhanced the proliferation, antiapoptotic effects, and metastasis of bladder cancer cells in vitro and in vivo. Mechanistically, the fusion protein encoded by SLC2A11–MIF interacted with polypyrimidine tract binding protein 1 (PTBP1) and regulated the mRNA half‐lives of Polo Like Kinase 1, Roundabout guidance receptor 1, and phosphoinositide‐3‐kinase regulatory subunit 3 in BCa cells. Moreover, PTBP1 knockdown abolished the enhanced impact of SLC2A11–MIF on biological function and mRNA stability. Furthermore, the expression of SLC2A11–MIF mRNA is regulated by CCCTC‐binding factor and stabilized through RNA N4‐acetylcytidine modification facilitated by N‐acetyltransferase 10. Overall, our findings revealed a significant fusion protein orchestrated by the SLC2A11–MIF–PTBP1 axis that governs mRNA stability during the multistep progression of bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024d970fb32d3f01ae87aaf1382c1c0690e3d1f6" target='_blank'>
              Oncogenic SLC2A11–MIF fusion protein interacts with polypyrimidine tract binding protein 1 to facilitate bladder cancer proliferation and metastasis by regulating mRNA stability
              </a>
            </td>
          <td>
            Liang Cheng, Chenwei Yang, Junlin Lu, Ming Huang, Ruihui Xie, Sarah Lynch, Justin Elfman, Yuhang Huang, Sen Liu, Siting Chen, Baoqing He, Tianxin Lin, Hui Li, Xu Chen, Jian-Ying Huang
          </td>
          <td>2024-08-14</td>
          <td>MedComm</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86982bea800325c140fbd22502a2e518da1a3167" target='_blank'>
              Novel method for detecting frequent TERT promoter hot spot mutations in bladder cancer samples
              </a>
            </td>
          <td>
            Á. Kovács, Farkas Sükösd, Levente Kuthi, Imre M Boros, B. Vedelek
          </td>
          <td>2024-08-14</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Transcription factor 3 (TCF3), a pivotal member of the TCF/LEF family, plays a critical role in tumorigenesis. Nonetheless, its impact on the tumor microenvironment (TME) and cancer phenotypes remains elusive. We perform an exhaustive analysis of TCF3 expression, DNA variation profiles, prognostic implications, and associations with the TME and immunological aspects. This study is based on a large-scale pan-cancer cohort, encompassing over 17,000 cancer patients from multiple independent datasets, validated by in vitro assays. Our results show that TCF3/4/7 exhibits differential expression patterns between normal and tumor tissues across pan-cancer analyses. Mutational analysis of TCF3 across diverse cancer types reveals the highest alteration rates in biliary tract cancer. Additionally, mutations and single nucleotide variants in TCF3/4/7 are found to exert varied effects on patient prognosis. Importantly, TCF3 emerges as a robust predictor of survival across all cancer cohorts and among patients receiving immune checkpoint inhibitors. Elevated TCF3 expression is correlated with more aggressive cancer subtypes, as validated by immunohistochemistry and diverse cohort data. Furthermore, TCF3 expression is positively correlated with intratumoral heterogeneity and angiogenesis. In vitro investigations demonstrate that TCF3 is involved in epithelial-mesenchymal transition, migration, invasion, and angiogenesis. These effects are likely mediated through the interaction of TCF3 with the NF-κB/MMP2 pathway, which is modulated by IL-17A in human uveal melanoma MUM2B cells. This study elucidates, for the first time, the significant associations of TCF3 with DNA variation profiles, prognostic outcomes, and the TME in multiple cancer contexts. TCF3 holds promise as a molecular marker for diagnosis and as a potential target for novel therapeutic strategies, particularly in uveal melanoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02bec429a74c0a32bc4300e0227ccff7d9af711e" target='_blank'>
              TCF3 as a multidimensional biomarker: oncogenicity, genomic alterations, and immune landscape in pan-cancer analysis.
              </a>
            </td>
          <td>
            Huiling Nie, Yang Yu, Siqi Zhou, Yue Xu, Xi Chen, Xun Qin, Zhangyu Liu, Jiayu Huang, Hailiang Zhang, Jin Yao, Qin Jiang, Bingbing Wei, Xiaojian Qin
          </td>
          <td>2024-08-01</td>
          <td>Acta biochimica et biophysica Sinica</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d417bff4628adb0291d6a29056ca242d46664396" target='_blank'>
              Molecular characteristics and oncogenic role of CHD family genes: a pan-cancer analysis based on bioinformatic and biological analysis
              </a>
            </td>
          <td>
            Yujia Lu, Jiebang Jiang, Zhihong He, Zhouzhou Bao, Xin Chen, Jie Cheng
          </td>
          <td>2024-08-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Current therapy in chronic myeloid leukemia (CML) has improved patient life expectancy close to that of healthy individuals. However, molecular alterations other than BCR::ABL1 fusion gene in CML are barely known. MicroRNAs are important regulators of gene expression, and variants in some of the components of microRNA biosynthesis pathways have been associated with genetic susceptibility to different types of cancer. Thus, the aim of this study was to evaluate the association of variants located in genes involved in the biogenesis of microRNAs with susceptibility to CML. Fifteen variants in eight genes involved in the biogenesis of miRNAs were genotyped in 296 individuals with CML and 485 healthy participants using TaqMan probes. The association of gene variants with CML and clinical variables was evaluated by a Chi-square test, and odds ratios and 95% confidence intervals were estimated by logistic regression. The variant rs13078 in DICER1 was significantly higher among CML individuals than in healthy participants. In addition, the variants rs7813 and rs2740349 were significantly associated with worse prognosis, according to their Hasford scores, whereas the rs2740349 variant was also associated with a later age at diagnosis. These findings suggest that variants in components of the microRNA biogenesis pathway could be involved in CML genetic risk.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcd4acdd732157884e8097b744843748d83fc720" target='_blank'>
              Gene Variants in Components of the microRNA Processing Pathway in Chronic Myeloid Leukemia
              </a>
            </td>
          <td>
            Guillermina Chavaro-Francisco, Araceli Hernández-Zavala, Camila E. Bravo-Cidro, Sandybel Rios-Rodriguez, Mabel Muciño-Sánchez, Marisol López-López, X. H. Castro-Martínez, Irma Olarte-Carrillo, A. García-Laguna, Gilberto Barranco-Lampón, Adrián de la Cruz-Rosas, Adolfo Martínez-Tovar, Emilio J. Córdova
          </td>
          <td>2024-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>6</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 16],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>