<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../ecDNA%20forecasting/">
      
      
        <link rel="next" href="../Spatial-Heterogeneity%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Drug-Resistance forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../ecDNA%20forecasting/" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../ecDNA%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Drug-Resistance forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-07 06:32:21 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Drug-Resistance forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Drug-Resistance forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on Drug-Resistance forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Drug-Resistance forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1df3a39ae2dafa8a1bd079f121166dd223decd13" target='_blank'>
                Genomic and transcriptomic analyses reveal a tandem amplification unit of 11 genes and mutations in mismatch repair genes in methotrexate-resistant HT-29 cells
                </a>
              </td>
          <td>
            Ahreum Kim, Jong-Yeon Shin, Jeong-Sun Seo
          </td>
          <td>2020-02-27</td>
          <td>Experimental and Molecular Medicine, Experimental & Molecular Medicine</td>
          <td>4</td>
          <td>48</td>

            <td><a href='../recommendations/1df3a39ae2dafa8a1bd079f121166dd223decd13' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee" target='_blank'>
                Cellular adaptation to cancer therapy along a resistance continuum.
                </a>
              </td>
          <td>
            Gustavo S França, Maayan Baron, B. King, Jozef P Bossowski, Alicia Bjornberg, Maayan Pour, A. Rao, Ayushi S. Patel, Selim Misirlioglu, Dalia Barkley, Kwan Ho Tang, Igor Dolgalev, D. A. Liberman, Gal Avital, Felicia Kuperwaser, Marta Chiodin, Douglas A. Levine, T. Papagiannakopoulos, A. Marusyk, T. Lionnet, Itai Yanai
          </td>
          <td>2024-07-10</td>
          <td>Nature</td>
          <td>8</td>
          <td>41</td>

            <td><a href='../recommendations/2b8945ae1e53ee1f3d36bd049fb804f06d8fb7ee' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Drug-Resistance forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Drug-Resistance forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6866382555382de9b8b8e3c83922b3e82d715085" target='_blank'>
              Defining heritability, plasticity, and transition dynamics of cellular phenotypes in somatic evolution.
              </a>
            </td>
          <td>
            Joshua S. Schiffman, Andrew R D'Avino, Tamara Prieto, Yakun Pang, Yilin Fan, Srinivas Rajagopalan, Catherine Potenski, Toshiro Hara, M. Suvà, Charles Gawad, Dan A Landau
          </td>
          <td>2024-09-24</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Summary Phenotypic adaptation, the ability of cells to change phenotype in response to external pressures, has been identified as a driver of drug resistance in cancer. To quantify phenotypic adaptation in BRAFV600E-mutant melanoma, we develop a theoretical model that emerges from data analysis of WM239A-BRAFV600E growth rates in response to drug challenge with the BRAF-inhibitor encorafenib. Our model constitutes a cell population model in which each cell is individually described by one of multiple discrete and plastic phenotype states that are directly linked to drug-dependent net growth rates and, by extension, drug resistance. Data-matched simulations reveal that phenotypic adaptation in the cells is directed towards states of high net growth rates, which enables evasion of drug-effects. The model subsequently provides an explanation for when and why intermittent treatments outperform continuous treatments in vitro, and demonstrates the benefits of not only targeting, but also leveraging, phenotypic adaptation in treatment protocols.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db72a236ec0fc5bd479b42430df6525df1d343c7" target='_blank'>
              Growth rate-driven modelling reveals how phenotypic adaptation drives drug resistance in BRAFV600E-mutant melanoma
              </a>
            </td>
          <td>
            Sara Hamis, A. Browning, A. Jenner, Chiara Villa, P. Maini, Tyler Cassidy
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>14</td>
        </tr>

        <tr id="Drug resistance is the major cause of therapeutic failure in high-grade serous ovarian cancer (HGSOC). Yet, the mechanisms by which tumors evolve to drug resistant states remains largely unknown. To address this, we aimed to exploit clone-specific genomic structural variations by combining scaled single-cell whole genome sequencing with longitudinally collected cell-free DNA (cfDNA), enabling clonal tracking before, during and after treatment. We developed a cfDNA hybrid capture, deep sequencing approach based on leveraging clone-specific structural variants as endogenous barcodes, with orders of magnitude lower error rates than single nucleotide variants in ctDNA (circulating tumor DNA) detection, demonstrated on 19 patients at baseline. We then applied this to monitor and model clonal evolution over several years in ten HGSOC patients treated with systemic therapy from diagnosis through recurrence. We found drug resistance to be polyclonal in most cases, but frequently dominated by a single high-fitness and expanding clone, reducing clonal diversity in the relapsed disease state in most patients. Drug-resistant clones frequently displayed notable genomic features, including high-level amplifications of oncogenes such as CCNE1, RAB25, NOTCH3, and ERBB2. Using a population genetics Wright-Fisher model, we found evolutionary trajectories of these features were consistent with drug-induced positive selection. In select cases, these alterations impacted selection of secondary lines of therapy with positive patient outcomes. For cases with matched single-cell RNA sequencing data, pre-existing and genomically encoded phenotypic states such as upregulation of EMT and VEGF were linked to drug resistance. Together, our findings indicate that drug resistant states in HGSOC pre-exist at diagnosis and lead to dramatic clonal expansions that alter clonal composition at the time of relapse. We suggest that combining tumor single cell sequencing with cfDNA enables clonal tracking in patients and harbors potential for evolution-informed adaptive treatment decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e1441270a54104f75e7713e2bb6b7323866959" target='_blank'>
              Tracking clonal evolution of drug resistance in ovarian cancer patients by exploiting structural variants in cfDNA
              </a>
            </td>
          <td>
            Marc J. Williams, I. Vázquez-García, Grittney K. Tam, Michelle Wu, Nancy Varice, Eliyahu Havasov, Hongyu Shi, Gryte Satas, Hannah J. Lees, Jake June-Koo Lee, Matthew A. Myers, Matthew Zatzman, Nicole Rusk, Emily Ali, Ronak H Shah, Michael F. Berger, N. Mohibullah, Y. Lakhman, Dennis S Chi, N. Abu-Rustum, Carol Aghajanian, Andrew McPherson, Dmitriy Zamarin, Brian Loomis, Britta Weigelt, C. Friedman, Sohrab P. Shah
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="Tumors consist of cancer cells with different genetic, epigenetic, and phenotypic properties. Cancer stem cells are an important subpopulation of heterogeneous cancer cells and are capable of initiating and propagating tumors. The term cancer stem cells has become broader in efforts to understand their phenotypic plasticity to switch fates between self-renewal and differentiation. Cancer stem cell plasticity is significantly associated with the initiation of metastasis, resistance to therapy, and tumor recurrence. With our broadened knowledge of epigenetic regulation and metabolic reprogramming as key elements enabling such capabilities, an expansive body of literature has demonstrated the functional importance of each element in contributing to cancer stem cell characteristics. Recently, the direct interplay between epigenetic regulation and metabolic reprogramming has begun to be appreciated in the context of cancer stem cells with growing interest. In this review, we discuss the mechanisms by which cancer stem cells orchestrate the reciprocal regulation of cellular metabolism and epigenetic alterations. In the discussion, compelling, unanswered questions on this topic have been elaborated for the interest of the research community and how recent technological developments help tackle such research ideas. A comprehensive understanding of cancer stem cell attributes that are largely governed by epigenetic and metabolic reprogramming would enable the advancement of precise therapeutic options and the prediction of better responses to drugs, holding great promise in cancer treatment and cure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a352f514fc793046331539c8fc8b91b79542075" target='_blank'>
              Interplay between epigenetics and metabolism controls cancer stem cell plasticity
              </a>
            </td>
          <td>
            Jee-Eun Choi, Inwha Baek
          </td>
          <td>2024-09-11</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e459f679b760cbc6c5f4d15f5d578e006d0e8b9" target='_blank'>
              Establishment and characterization of 18 Sarcoma Cell Lines: Unraveling the Molecular Mechanisms of Doxorubicin Resistance in Sarcoma Cell Lines
              </a>
            </td>
          <td>
            Young-Eun Cho, Soon-Chan Kim, Ha Jeong Kim, Ilkyu Han, Ja-Lok Ku
          </td>
          <td>2024-10-02</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is a diverse malignancy originating from myeloid progenitor cells, with significant genetic and clinical variability. Modern classification systems like those from the World Health Organization (WHO) and European LeukemiaNet use immunophenotyping, molecular genetics, and clinical features to categorize AML subtypes. This classification highlights crucial genetic markers such as FLT3, NPM1 mutations, and MLL-AF9 fusion, which are essential for prognosis and directing targeted therapies. The MLL-AF9 fusion protein is often linked with therapy-resistant AML, highlighting the risk of relapse due to standard chemotherapeutic regimes. In this sense, factors like the ZEB, SNAI, and TWIST gene families, known for their roles in epithelial–mesenchymal transition (EMT) and cancer metastasis, also regulate hematopoiesis and may serve as effective therapeutic targets in AML. These genes contribute to cell proliferation, differentiation, and extramedullary hematopoiesis, suggesting new possibilities for treatment. Advancing our understanding of the molecular mechanisms that promote AML, especially how the bone marrow microenvironment affects invasion and drug resistance, is crucial. This comprehensive insight into the molecular and environmental interactions in AML emphasizes the need for ongoing research and more effective treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b827ac4bebbcac825ffc1c60412a291161ab8775" target='_blank'>
              The Role of Epithelial-to-Mesenchymal Transition Transcription Factors (EMT-TFs) in Acute Myeloid Leukemia Progression
              </a>
            </td>
          <td>
            Diego Cuevas, Roberto Amigo, Adolfo Agurto, Adan Andreu Heredia, Catherine Guzmán, Antonia Recabal-Beyer, V. González-Pecchi, T. Caprile, Jody J. Haigh, Carlos Farkas
          </td>
          <td>2024-08-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Neuroblastoma (NB) is the most common pediatric solid tumor, responsible for about 15% of childhood cancer-related mortality in the United States. Multiple therapies have been established, which include chemotherapy, radiotherapy, immunotherapy and molecularly-targeted therapy, however resistance to these therapies is the major hinderance in curing NB. RAS pathway mutations are prevalent in relapsed NB and are associated with a poorer overall survival. Therefore, comprehending the mechanisms leading to therapy resistance has become an urgent need. In this study, we attempted to identify the resistance mechanism of NB cell lines towards RAS inhibitors. RAS-altered NB cell lines are treated with trametinib, a MAP kinase pathway inhibitor to perform cell viability assays, immunoblotting and RT PCR. Copy number alteration and translocation studies are performed using spectral karyotyping (SKY) and multiplex interphase fluorescent in-situ hybridization (miFISH). We observed that unlike other RAS-altered NB cell lines (SK-N-AS, CHP-212, LA-N-6, SK-N-BE(2)-C), NB-Eb-C1, which harbors a KRAS G12D mutation shows resistance to both MRTX1133 (KRAS G12D direct inhibitor) and trametinib. Immunoblot against crucial genes regulating NB showed high expression of MYCN, the most relevant biomarker of high-risk neuroblastoma. MYCN is unfortunately undruggable hence, standard treatment for MYCN-amplified neuroblastoma patients does not include targeting MYCN itself. We therefore pursued a hypothesis-driven approach towards identifying possible mechanisms responsible for MYCN driven MEKi-resistance in RAS pathway-altered NB cells. MYCN is known to cross talk with the RAS pathway, however this mechanism is not well understood in NB. Therefore, we performed the clonal assessment of NB cell lines using miFISH and to the best of our knowledge are the first to report that, NB-Eb-C1 has a hypertriploid to hypotetraploid baseline with a low copy-number gain of MYCN. To have a profound understanding of the functionality of MYCN in NB, we have taken 3 different NB cell lines with varied MYCN status. CHP-212 (MYCN amplified-extrachromosomal), SK-N-BE(2)-C (MYCN amplified-homogenously staining region), SK-N-AS (MYCN neutral). SKY and FISH revealed that the MYCN copy-number gain in NBEB cells is the result of a translocation between chromosome 2 and 5 involving MYCN. Of note, another frequently abnormally expressed gene in NB, ALK, is co-translocated with MYCN suggesting potential involvement of both genes in imparting chemoresistance. To understand the mechanism of MYCN overexpression in NBEB we will be performing HiC to check for promoter/enhancer hijacking. Also, to better define this translocation and studying other translocations involved we are performing optical genome mapping using BioNano. Other functional experiments will be performed to look at the impact of the RAS/MAPK alteration on MYCN protein stability. We are confident that this study will contribute towards developing new combinations of targeted therapies to improve outcomes in RAS-altered NB.
 Citation Format: Subhra Dash, Kerstin Heselmeyer-Haddad, Stacy Stauffer, Lucas Stauffer, Danny Wangsa, Marielli E. Yohe. Investigating mechanisms responsible for MAP kinase pathway resistance in RAS-altered neuroblastoma cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A053.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b3e4035267dd00bede1ab23d0c552e939dc4a96" target='_blank'>
              Abstract A053 Investigating mechanisms responsible for MAP kinase pathway resistance in RAS-altered neuroblastoma cells
              </a>
            </td>
          <td>
            Subhra Dash, Kerstin Heselmeyer-Haddad, Stacy Stauffer, Lucas Stauffer, Danny Wangsa, Marielli E. Yohe
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfbcb18aaf8922f16ad19ad9eeb7b0ec8bc2789d" target='_blank'>
              Serial single-cell RNA sequencing unveils drug resistance and metastatic traits in stage IV breast cancer
              </a>
            </td>
          <td>
            K. Otsuji, Yoko Takahashi, Tomo Osako, Takayuki Kobayashi, T. Takano, S. Saeki, Liying Yang, Satoko Baba, Kohei Kumegawa, Hiromu Suzuki, Tetsuo Noda, Kengo Takeuchi, Shinji Ohno, Takayuki Ueno, Reo Maruyama
          </td>
          <td>2024-10-03</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Telomeres are critical nucleoprotein structures that safeguard chromosome ends from degradation and fusion, thereby ensuring genomic stability. Without proper maintenance, telomeres progressively shorten with each cell division, leading to dysfunction and triggering chromosomal instability. This dysfunction is a significant driver of tumorigenesis, primarily by facilitating genetic alterations such as aneuploidy, gene amplifications, and chromosomal rearrangements. Cancer cells bypass cellular aging by activating mechanisms like telomerase reactivation or the Alternative Lengthening of Telomeres (ALT) pathway to maintain telomere length. This review explores the mechanisms by which telomere dysfunction contributes to genomic instability and cancer progression, including telomere shortening, breakage-fusion-bridge (B/F/B) cycles, and the ALT pathway. Additionally, it addresses the therapeutic potential of targeting telomere maintenance, highlighting current strategies like telomerase and ALT inhibitors. However, developing telomere-based therapies presents challenges, including resistance mechanisms, off-target effects, and potential impacts on normal stem cells. Emerging research areas such as the development of biomarkers and combination therapies offer promising directions for overcoming these challenges. Understanding telomere dynamics provides novel opportunities to exploit cancer cell vulnerabilities and advance treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b7c5196ace0bbc6bf2d90a9a3ba826ad745b1e" target='_blank'>
              Mechanisms of telomere dysfunction in cancer from genomic instability to therapy: A review
              </a>
            </td>
          <td>
            Amir Mohammad Karimi Forood
          </td>
          <td>2024-09-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) is an aggressive hematologic neoplasia with a complex polyclonal architecture. Among driver lesions, those involving the FLT3 gene represent the most frequent mutations identified at diagnosis. The development of tyrosine kinase inhibitors (TKIs) has improved the clinical outcomes of FLT3-mutated patients (Pt). However, overcoming resistance to these drugs remains a challenge. To unravel the molecular mechanisms underlying therapy resistance and clonal selection, we conducted a longitudinal analysis using a single-cell DNA sequencing approach (MissionBioTapestri® platform, San Francisco, CA, USA) in two patients with FLT3-mutated AML. To this end, samples were collected at the time of diagnosis, during TKI therapy, and at relapse or complete remission. For Pt #1, disease resistance was associated with clonal expansion of minor clones, and 2nd line TKI therapy with gilteritinib provided a proliferative advantage to the clones carrying NRAS and KIT mutations, thereby responsible for relapse. In Pt #2, clonal architecture was less complex, and 1st line TKI therapy with midostaurin was able to eradicate the leukemic clones. Our results corroborate previous findings about clonal selection driven by TKIs, highlighting the importance of a deeper characterization of individual clonal architectures for choosing the best treatment plan for personalized approaches aimed at optimizing outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4202a0b52113f492de1c5d891aefedd2f36b44c" target='_blank'>
              Tracking Response and Resistance in Acute Myeloid Leukemia through Single-Cell DNA Sequencing Helps Uncover New Therapeutic Targets
              </a>
            </td>
          <td>
            S. Bruno, E. Borsi, A. Patuelli, L. Bandini, M. Mancini, D. Forte, J. Nanni, Martina Barone, Alessandra Grassi, G. Cristiano, C. Venturi, V. Robustelli, Giulia Atzeni, Cristina Mosca, S. De Santis, C. Monaldi, A. Poletti, C. Terragna, A. Curti, M. Cavo, Simona Soverini, E. Ottaviani
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c323e127988809654aaf5a547e1a3c8242712a2" target='_blank'>
              Prevalence of and gene regulatory constraints on transcriptional adaptation in single cells
              </a>
            </td>
          <td>
            Ian A. Mellis, Madeline E. Melzer, Nicholas Bodkin, Yogesh Goyal
          </td>
          <td>2024-08-12</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="
 We aim to offer new insights on the regulation of metabolic plasticity in renal cell carcinoma (RCC) and identify novel metabolic regulatory targets that can be used for potential treatment therapies of the disease. RCC affects an estimated 400,000 people worldwide each year with over 100,00 deaths annually that disproportionately affects racial/ethnic minorities in the United States. RCC is a term that covers an array of kidney cancer subtypes differing in incidence, histopathology, genetic and molecular alterations, as well as in clinical outcomes and prognoses. However, a common characteristic of many RCCs is that they are driven by metabolic rewiring, due to a high frequency of mutations in genes that regulate major metabolic processes in the cell. It was recently discovered that patient overall survival and prognosis has been linked to the expression of certain metabolic signatures in RCC tumor cells. Metabolic plasticity allows cancer cells to survive and adapt to different environments and metabolic stress conditions, such as nutrient deprivation. Therefore, exploring and exploiting metabolic rewiring in these cells may reveal genetic vulnerabilities and dependencies. This could lead to discovery of novel biomarkers and molecular targets that can be implicated as future therapeutic interventions for this disease. To this end, we applied functional genomic screening to expose genetic dependencies and vulnerabilities under different physiologic and metabolic conditions in patient derived medullary (PDM) RCC cells. The medullary RCC is a rare and aggressive RCC subtype that disproportionately affects the African American/Black population. We applied a CRISPR- based combinatorial screening platform that is based on co-expression of Cas9 and Cas12a nucleases and libraries of hybrid guide RNAs (hgRNA) to achieve ultra-efficient gene knockout (Aregger et al., 2021; Gonatopoulos-Pournatzis et al., 2020) in PDM RCC cells. Additionally, cells were screened under different media conditions that mimic human physiological nutrient conditions and a media deprived of lipids, one of the major metabolic substrates of the cell. By utilizing a genome-wide pooled CRISPR gene knockout screen we were able to systematically uncover genes required for cell survival and proliferation in a panel of patient-derived RCCs. We have begun to uncover genetic dependencies and major metabolic pathways that can be elucidated and exploited for future studies, such as those involving ferroptosis which is a targetable pathway for the development of cancer treatment therapies. Our findings demonstrate some of the key genetic drivers underlying cellular adaptations to different physiological environments and nutrient deprivation. In future studies we will further explore the relationship between different key metabolic regulatory genes in RCC and map for genetic interactions in major metabolic pathways.
 Citation Format: Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger. Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C091.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cd9b94600307109e3720c07a3b1a13c91abd8ae" target='_blank'>
              Abstract C091: Uncovering genetic regulators of metabolic plasticity in renal cell carcinoma
              </a>
            </td>
          <td>
            Chelsee Holloway, Josef Horak, Youngkyu Jeon, Michael Aregger
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract RNA sequencing technology combining short read and long read analysis can be used to detect chimeric RNAs in malignant cells. Here, we propose an integrated approach that uses k-mers to analyze indexed datasets. This approach is used to identify chimeric RNA in chronic myelomonocytic leukemia (CMML) cells, a myeloid malignancy that associates features of myelodysplastic and myeloproliferative neoplasms. In virtually every CMML patient, new generation sequencing identifies one or several somatic driver mutations, typically affecting epigenetic, splicing and signaling genes. In contrast, cytogenetic aberrations are currently detected in only one third of the cases. Nevertheless, chromosomal abnormalities contribute to patient stratification, some of them being associated with higher risk of poor outcome, e.g. through transformation into acute myeloid leukemia (AML). Our approach selects four chimeric RNAs that have been detected and validated in CMML cells. We further focus on NRIP1-MIR99AHG, as this fusion has also recently been detected in AML cells. We show that this fusion encodes three isoforms, including a novel one. Further studies will decipher the biological significance of such a fusion and its potential to improve disease stratification. Taken together, this report demonstrates the ability of a large-scale approach to detect chimeric RNAs in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20cd2e1e29866db7633afbaeb3e311e423e92cc0" target='_blank'>
              Effective requesting method to detect fusion transcripts in chronic myelomonocytic leukemia RNA-seq
              </a>
            </td>
          <td>
            Florence Rufflé, Jérôme Reboul, A. Boureux, Benoit Guibert, Chloé Bessière, Raíssa Silva, Eric Jourdan, Jean-Baptiste Gaillard, A. Boland, J. Deleuze, Catherine Senamaud-Beaufort, D. Sélimoglu-Buet, Eric Solary, Nicolas Gilbert, Thérèse Commes
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24869ef91b9d9f698a0aa1c1c25527b7385dd280" target='_blank'>
              Deciphering resistance mechanisms in cancer: final report of MATCH-R study with a focus on molecular drivers and PDX development
              </a>
            </td>
          <td>
            Damien Vasseur, L. Bigot, Kristi Beshiri, Juan-David Florez-Arango, F. Facchinetti, A. Hollebecque, L. Tselikas, M. Aldea, F. Blanc-Durand, A. Gazzah, D. Planchard, Ludovic Lacroix, N. Pata-Merci, Catline Nobre, Alice Da Silva, C. Nicotra, M. Ngo-Camus, F. Braye, Sergey Nikolaev, S. Michiels, Gerome Jules-Clement, K. Olaussen, Fabrice André, J. Scoazec, F. Barlesi, Santiago Ponce, Jean-Charles Soria, Benjamin Besse, Yohann Loriot, L. Friboulet
          </td>
          <td>2024-10-04</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Despite development of several effective therapies for multiple myeloma (MM), the prognosis of patients with partial deletion of chromosome 17 (del(17p)) and TP53 aberrations remains poor. By applying comprehensive multi-omics profiling analyses (whole exome and transcriptome sequencing plus proteomics) and functional ex vivo drug screening to samples from 167 patients with MM, we uncovered novel therapeutic vulnerabilities specific to TP53 mutated MM. Our findings revealed a distinct sensitivity profile to a range of inhibitors (mitotic, topoisomerase, HDAC, HSP90, IGF1R and PI3K/AKT/mTOR inhibitors) irrespective of 17p deletion status. Conversely, no increase in sensitivity was observed for monoallelic TP53 (del(17p) with WT TP53) when compared to other samples, highlighting the remaining unmet clinical need. Notably, plicamycin, an RNA synthesis inhibitor linked to modulation of chromatin structure and increased transcription, emerged as particularly efficacious for TP53 mutated MM. The increased sensitivity correlated with higher protein expression of the drug targets: HDAC2, HSP90AA1 and multiple ribosomal subunits. Additionally, we observed increased RNA expression of G2M checkpoint, E2F targets and mTORC1 signaling in our cohort and the MMRF-CoMMpass (NCT01454297) study in TP53 mutated MM. Harmonization of multi-omics data with ex vivo drug screening results revealed that TP53 mutated MM is functionally distinct from MM with monoallelic TP53, and demonstrate that MM with mutated TP53, with and without del(17p), may be targetable by approved drugs, and further indicates the need for regular monitoring by sequencing to identify these patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12a52d0d33d267850e10246d4013cb5921df1165" target='_blank'>
              Multi-omic and functional screening reveal targetable vulnerabilities in TP53 mutated multiple myeloma
              </a>
            </td>
          <td>
            Dimitrios Tsallos, Nemo Ikonen, Juho J. Mienen, M. M. Majumder, S. Eldfors, I. Västrik, A. Parsons, M. Suvela, Katie Dunphy, Paul Dowling, D. Bazou, Peter O’Gorman, J. Lievonen, R. Silvennoinen, Pekka Anla, C. Heckman
          </td>
          <td>2024-08-23</td>
          <td>None</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="BACKGROUND
Glioblastoma (GBM) is an aggressive malignancy. The inherent resistance of GBM to radiotherapy poses great challenges for clinical treatment.


OBJECTIVES
The primary objective of this study is to explore the molecular mechanisms of radiotherapy resistance in GBM and identify the key influencing factors that contribute to this phenomenon.


METHODS
The single-cell RNA sequencing (scRNA-seq) data of GBM were downloaded from the Gene Expression Omnibus (GEO) database. Cells were clustered using the Seurat R package, and the clusters were annotated using the CellMarker database. Pseudotime analysis was conducted using Monocle2. Marker scores were calculated based on the RNA-seq data of GBM from the UCSC database, and the enrichment of Hallmark gene sets was measured with the AUCell package. Furthermore, the most frequently mutated genes were identified using the simple nucleotide variation data from The Cancer Genome Atlas (TCGA) applying the maftools package.


RESULTS
This study identified two oligodendrocyte subsets (ODC3 and ODC4) as radiotherapy-resistant groups in GBM. Enrichment and Pseudotime analysis revealed that the inflammatory response and immune activation pathways were enriched in ODC3, while the cell division and interferon response pathways were enriched in ODC4. The enrichment scores of hallmark gene sets further confirmed that ODC3 and ODC4 subpopulations developed radiotherapy resistance via distinct molecular mechanisms. Analysis of gene mutation frequencies showed that TP53 exhibited the most significant change in mutation frequency, indicating that it was an important risk factor involved in radiotherapy resistance in GBM.


CONCLUSION
We identified two ODC subpopulations that exhibited resistance to radiotherapy, providing a new perspective and potential targets for personalized treatment strategies for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3593a813498357e1fd5d3f302a2e0d7d52d1643a" target='_blank'>
              Analysis of the Mechanism Underlying Radiotherapy Resistance Caused by Oligodendroglia Cells in Glioblastoma by Applying the Single-cell RNA Sequencing Technology.
              </a>
            </td>
          <td>
            Qinghua Yuan, Weida Gao, Mian Guo, Bo Liu
          </td>
          <td>2024-09-12</td>
          <td>Current medicinal chemistry</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d53e6452f948769bae38a66bb4f9a7addad82684" target='_blank'>
              Single-cell RNA sequencing explored potential therapeutic targets by revealing the tumor microenvironment of neuroblastoma and its expression in cell death
              </a>
            </td>
          <td>
            Lei Sun, Wenwen Shao, Zhiheng Lin, Jingheng Lin, Fu Zhao, Juan Yu
          </td>
          <td>2024-09-05</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Next generation sequencing of neuroblastoma (NB) tumors have revealed frequent somatic and germline genetic alterations in genes encoding proteins involved in DNA damage response (DDR) pathways. Despite being well-studied in many adult cancers, roles for DDR disruption in pediatric solid tumors remains poorly understood. To address this, patient-relevant loss-of-function mutations in DDR pathway components including Brca2, Atm, and Palb2 were incorporated into an established zebrafish MYCN transgenic model (Tg(dbh:EGFP-MYCN)). These mutations were found to enhance NB formation and metastasis in vivo, and result in upregulation of proliferation, cell cycle checkpoint and DNA damage repair transcriptional signatures, revealing potential molecular vulnerabilities in DDR-deficient NB. Zebrafish DDR-deficient NB and human NB cells with DDR protein knock-down were sensitive to the poly(ADP-ribose)-polymerase (PARP) inhibitor olaparib, and this effect was further enhanced by inhibition of the ataxia telangiectasia and rad3 related (ATR) kinase. Altogether, our data supports a functional role for DDR-deficiency in NB in vivo and therapeutic potential for combination PARP + ATR inhibition in NB patients with alterations in DDR genes. Significance This work provides the first in vivo evidence supporting a functional role for DDR-deficiency in NB by demonstrating that alterations in certain DDR pathway genes promote NB formation and metastasis. NGS and pre-clinical drug testing also provides rationale for PARP + ATR inhibitor therapy combinations for patients with NB and pathogenic DDR pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2bdec5d0ffc7f47bcd4a92b02bbe2b4a531d6b2a" target='_blank'>
              DNA Damage Response Deficiency Enhances Neuroblastoma Progression and Sensitivity to Combination PARP and ATR Inhibition
              </a>
            </td>
          <td>
            Madeline N Hayes, S. Cohen-Gogo, L. Kee, Alex Weiss, M. Layeghifard, Yagnesh Ladumor, Ivette Valencia-Sama, Anisha Rajaselvam, David Kaplan, A. Villani, A. Shlien, Daniel A. Morgenstern, Meredith S Irwin
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="In the realm of hematopoiesis, hematopoietic stem cells (HSCs) serve as pivotal entities responsible for generating various blood cell types, initiating both the myeloid and lymphoid branches within the hematopoietic lineage. This intricate process is marked by genetic variations that underscore the crucial role of genes in regulating cellular functions and interactions. Recognizing the significance of genetic factors in this context, this article delves into a genetic perspective, aiming to unravel the biological factors that govern the transition from one cell’s fate to another within the hematopoietic system. To gain deeper insights into the genetic traits of three distinct blood cell types—HSCs, erythroblasts (EBs), and megakaryocytes (MKs)—we conducted a comprehensive transcriptomic analysis. Leveraging diverse hematopoietic cell datasets from healthy individuals, sourced from The BLUEPRINT consortium, our investigation targeted the identification of genetic variants responsible for changes in gene expression levels and epigenetic modifications across the entire human genome in each of these cell types. The total number of normalized expressed transcripts includes 14,233 novel trinity lncRNAs, 13,749 mRNAs, and 3092 lncRNAs. This scrutiny revealed a total of 31,074 transcripts, with a notable revelation that 14,233 of them were previously unidentified or novel lncRNAs, highlighting a substantial reservoir of genetic information yet to be explored. Examining their expression across distinct lineages further unveiled 2845 differentially expressed (DE) mRNAs and 354 DE long noncoding RNAs (lncRNAs) notably enriched among the three distinct blood cell types: HSCs, EBs, and MKs. Our investigation extended beyond mRNA to focus on the dynamic expression of lncRNAs, revealing a well-defined pattern that played a significant role in regulating differentiation and cell-fate specification. This coordination of lncRNA dynamics extended to aberrations in both mRNA and lncRNA transcriptomes within HSCs, EBs, and MKs. We specifically characterized lncRNAs with preferential expression in HSCs, as well as in various downstream differentiated lineage progenitors of EBs and MKs, providing a comprehensive perspective on lncRNAs in human hematopoietic cells. Notably, the expression of lncRNAs exhibited substantial cell-to-cell variation, a phenomenon discernible only through single-cell analysis. The comparative analysis undertaken in this study provides valuable insights into the distinctive genetic signatures guiding the differentiation of these crucial hematopoietic cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de7e188cc79e1414360446b250f4a5a454712e4c" target='_blank'>
              Deciphering Transcriptomic Variations in Hematopoietic Lineages: HSCs, EBs, and MKs
              </a>
            </td>
          <td>
            S. Dahariya, Anton Enright, Santosh Kumar, Ravi Kumar Gutti
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d317e0704a11a531980e863a514ed7e3ab276870" target='_blank'>
              Advances in pathogenesis research and challenges in treatment development for acute myeloid leukemia.
              </a>
            </td>
          <td>
            Hiroki Yamaguchi
          </td>
          <td>2024-09-03</td>
          <td>International journal of hematology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a treatment-refractory malignancy with a dismal prognosis. Targeting cancer cell plasticity—the capacity to adapt to cell-extrinsic pressure via non-genetic mechanisms—is a promising therapeutic strategy, yet cell states within PDAC harboring high plasticity remain poorly understood. Using genetically engineered mouse models driven by oncogenic KrasG12D and loss of p53 (“KPC” model), we profiled primary PDAC tumors using single-cell RNA sequencing and single cell ATAC sequencing. We identified a cancer cell differentiation state that co-expresses both epithelial and mesenchymal programs and has a distinct epigenetic landscape with accessible chromatin across gene sets that mark cancer cell states with divergent phenotypes, suggesting that this cell state is primed for cell state switching. Notably, this high-plasticity cell state (HPCS) signature aligns with a "basal" signature in human PDAC, which associates with poor prognosis and chemoresistance. To functionally interrogate the HPCS in PDAC progression, we developed a novel lineage-tracing and ablation system. Lineage-tracing experiments revealed that the HPCS has a robust capacity for differentiation into epithelial and mesenchymal states, indicating that the HPCS is functionally plastic and acts as a source of fixed malignant states. Strikingly, targeted elimination of the HPCS over short time periods resulted in tumor collapse, underscoring a critical role in tumor maintenance. Further, we found that the HPCS is highly dependent on KRAS activity, with acute KRAS inhibitor treatment leading to a selective depletion of the HPCS. Taken together, our data indicates that PDAC harbors a striking dependency on the HPCS and encourages therapeutic approaches aimed at eradicating this critical subset of cancer cells.
 Citation Format: Anupriya Singhal, Hannah C. Styers, Jonathan Rub, Zhuxuan Li, Zeynep Tarcan, Jill Hallin, Olca Basturk, Rona Yaeger, James G. Christensen, Doron Betel, Yan Yan, Elisa de Stanchina, Tuomas Tammela. Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C077.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4987f81530acda2fa882016a6788c57664c546ba" target='_blank'>
              Abstract C077: Functional Dissection of the Highly Plastic Basal Cell State in Pancreatic Cancer
              </a>
            </td>
          <td>
            Anupriya Singhal, Hannah C Styers, Jonathan Rub, Zhuxuan Li, Z. Tarcan, Jill Hallin, O. Basturk, R. Yaeger, James G Christensen, D. Betel, Yan Yan, E. de Stanchina, Tuomas Tammela
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Abstract Ovarian cancer ecosystems are exceedingly complex, consisting of a high heterogeneity of cancer cells. Development of drugs such as poly ADP‐ribose polymerase (PARP) inhibitors, targeted therapies and immunotherapies offer more options for sequential or combined treatments. Nevertheless, mortality in metastatic ovarian cancer patients remains high because cancer cells consistently develop resistance to single and combination therapies, urging a need for treatment designs that target the evolvability of cancer cells. The evolutionary dynamics that lead to resistance emerge from the complex tumour microenvironment, the heterogeneous populations, and the individual cancer cell's plasticity. We propose that successful management of ovarian cancer requires consideration of the ecological and evolutionary dynamics of the disease. Here, we review current options and challenges in ovarian cancer treatment and discuss principles of tumour evolution. We conclude by proposing evolutionarily designed strategies for ovarian cancer, with the goal of integrating such principles with longitudinal, quantitative data to improve the treatment design and management of drug resistance. Key points/Highlights Tumours are ecosystems in which cancer and non‐cancer cells interact and evolve in complex and dynamic ways. Conventional therapies for ovarian cancer inevitably lead to the development of resistance because they fail to consider tumours’ heterogeneity and cellular plasticity. Eco‐evolutionarily designed therapies should consider cancer cell plasticity and patient‐specific characteristics to improve clinical outcome and prevent relapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f584cd6ae71db959a312c8365ae2a9ada15cf5" target='_blank'>
              Ecological and evolutionary dynamics to design and improve ovarian cancer treatment
              </a>
            </td>
          <td>
            Grace Y Q Han, Monica Alexander, Julia Gattozzi, Marilyn Day, Elayna Kirsch, Narges Tafreshi, Raafat Chalar, Soraya Rahni, Gabrielle Gossner, William Burke, Mehdi Damaghi
          </td>
          <td>2024-08-29</td>
          <td>Clinical and Translational Medicine</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most prevalent malignant brain tumor in children that demonstrates clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), and particularly the 3gsubtype, exhibit high levels of MYC, high rates of metastasis, and have the worst prognosis. Despite treatment with surgery, radiation and chemotherapy, Group 3 medulloblastoma patients are more likely to develop recurrent tumors which are far more aggressive resulting in a 5-year survival rate after recurrence of 12%. Thus, there is an imperative need to examine the underlying mechanisms mediating treatment resistance. Recent evidence has identified cellular plasticity, as a major driver of therapy resistance. To investigate whether epigenetic mediated cell plasticity contributes to Myc-MB therapy response we performed Single Nuclei Multiome ATAC + Gene Expression analysis of paired primary and relapsed tumors. We identified the emergence of novel progenitor populations in resistant tumors with stem-like gene expression networks and altered chromatin accessibility. These emergent popuoations demostrated new enhancer-promoer interactions and exhibited increased metabolic reprogramming with potential for theraputic targeting. Because radiation is one of the key treatment modalities for Myc-MB, we chose to model radiation resistance in further studies. Single-cell RNA sequencing of outgrown in vivo irradiated orthotopic xenograft MB tumors showed an overall upregulation of metabolic pathways with the enhancement of genes that mediate oxidative phosphorylation and glutamate metabolism centering on wild-type IDH1 (wtIDH1), consistent with our patient data. IR-resistant cells showed altered chromatin occupancy of H3K3me3 and H3K27Ac. Disruption of IDH1 activity in IR resistant cells in vitro altered the epigenomic landscape, facilitating reversion to the pre-irradiated state. Interestingly, we observed that pluripotent genes with bivalent regions (H3K4me3 and H3K27me3) were enhanced in radiation-resistant cells and suppressed by IDH1 inhibition. Metabolic mass spectrometry reveled that metabolites from the citric acid cycle and synthesis of unsaturated fatty acids were among the main metabolically elevated molecules in radiation-resistant cells consistnat with the observed epigenomic reprograming. IDH1 inhibition reversed the metabolic reprograming and resensitized MB cells to radiation. In total our data identify a novel mechanism of radiation resistance in medulloblastoma with potential theraputic implications.
 Citation Format: Bethany Veo, Dong Wang, John Desisto, Rajeev Vibhakar. Mechanisms of radiation resistance in medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A071.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaed4654cb7c14fb325db4bb243c211a30ba0383" target='_blank'>
              Abstract A071 Mechanisms of radiation resistance in medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Despite the general detriment of aneuploidy to cellular fitness, >90% of solid tumors carry an imbalanced karyotype. Regardless of this existing paradox, our understanding of the molecular responses to aneuploidy remains limited. Here, we explore these cellular stresses and unique vulnerabilities in aneuploid human mammary epithelial cells (HMECs) enriched for breast cancer-associated copy number alterations (CNAs). To uncover the genetic dependencies specific to aneuploid cells, we conducted a comprehensive, genome-wide CRISPR knockout screen targeting isogenic diploid and aneuploid HMEC lines. Our study reveals that aneuploid HMECs exhibit an increased reliance on pyrimidine biosynthesis and mitochondrial oxidative phosphorylation genes, and demonstrate heightened fitness advantages upon loss of tumor suppressor genes. Using an integrative multi-omic analysis, we confirm nucleotide pool insufficiency as a key contributor to widespread cellular dysfunction in aneuploid HMECs with net copy number gain. While diploid cells can switch seamlessly between pyrimidine synthesis and salvage, cells with increased chromosomal content exhibit p53 activation and S-phase arrest when relying on salvage alone, and exhibit increased sensitivity to DNA-damaging chemotherapeutics. This work advances our understanding of the consequences of aneuploidy and uncovers potential avenues for patient stratification and therapeutic intervention based on tumor ploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f75b91959ba71d0bf04e734b4ee5bf5730bc793" target='_blank'>
              An aneuploidy epistasis map reveals metabolic vulnerabilities associated with supernumerary chromosomes in cancer
              </a>
            </td>
          <td>
            R. Magesh, A. N. Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. A. Guertin, J. B. Spinelli, S. Elledge, E. Watson
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Microsatellite unstable colorectal cancer (MSI-CRC) can arise through germline mutations in mismatch repair (MMR) genes in individuals with Lynch syndrome (LS), or sporadically through promoter methylation of the MMR gene MLH1. Despite the different origins of hereditary and sporadic MSI tumours, their genomic features have not been extensively compared. A prominent feature of MMR-deficient genomes is the occurrence of many indels in short repeat sequences, an understudied mutation type due to the technical challenges of variant calling in these regions. In this study, we performed whole genome sequencing and RNA-sequencing on 29 sporadic and 14 hereditary MSI-CRCs. We compared the tumour groups by analysing genome-wide mutation densities, microsatellite repeat indels, recurrent protein-coding variants, signatures of single base, doublet base, and indel mutations, and changes in gene expression. We show that the mutational landscapes of hereditary and sporadic MSI-CRCs, including mutational signatures and mutation densities genome-wide and in microsatellites, are highly similar. Only a low number of differentially expressed genes were found, enriched to interferon-γ regulated immune response pathways. Analysis of the variance in allelic fractions of somatic variants in each tumour group revealed higher clonal heterogeneity in sporadic MSI-CRCs. Our results suggest that the differing molecular origins of MMR deficiency in hereditary and sporadic MSI-CRCs do not result in substantial differences in the mutational landscapes of these tumours. The divergent patterns of clonal evolution between the tumour groups may have clinical implications, as high clonal heterogeneity has been associated with decreased tumour immunosurveillance and reduced responsiveness to immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43c6128f5465fd71306e14170da497710701fc42" target='_blank'>
              Lynch syndrome-associated and sporadic microsatellite unstable colorectal cancers: different patterns of clonal evolution yield highly similar tumours.
              </a>
            </td>
          <td>
            Samantha Martin, Riku Katainen, Aurora Taira, Niko Välimäki, Ari Ristimäki, T. Seppälä, L. Renkonen-Sinisalo, A. Lepistö, Kyösti Tahkola, Anne Mattila, S. Koskensalo, J. Mecklin, K. Rajamäki, Kimmo Palin, L. Aaltonen
          </td>
          <td>2024-08-24</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="Therapeutic resistance in cancer significantly contributes to mortality, with many patients eventually experiencing recurrence after initial treatment responses. Recent studies have identified therapy-resistant large polyploid cancer cells in patient tissues, particularly in late-stage prostate cancer, linking them to advanced disease and relapse. Here, we analyzed bone marrow aspirates from 44 advanced prostate cancer patients and found the presence of circulating tumor cells with increased genomic content (CTC-IGC) was significantly associated with poorer progression- free survival. Single cell copy number profiling of CTC-IGC displayed clonal origins with typical CTCs, suggesting complete polyploidization. Induced polyploid cancer cells from PC3 and MDA-MB-231 cell lines treated with docetaxel or cisplatin were examined through single cell DNA sequencing, RNA sequencing, and protein immunofluorescence. Novel RNA and protein markers, including HOMER1, TNFRSF9, and LRP1, were identified as linked to chemotherapy resistance. These markers were also present in a subset of patient CTCs and associated with recurrence in public gene expression data. This study highlights the prognostic significance of large polyploid tumor cells, their role in chemotherapy resistance, and their expression of markers tied to cancer relapse, offering new potential avenues for therapeutic development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3949721a19b75a2630865e1a7d38618cdb6374" target='_blank'>
              Polyploid cancer cells reveal signatures of chemotherapy resistance
              </a>
            </td>
          <td>
            Michael J. Schmidt, Amin Naghdloo, Rishvanth K. Prabakar, Mohamed Kamal, Radu M. Cadaneanu, I. Garraway, Michael Lewis, Ana Aparicio, A. Zurita-Saavedra, P. Corn, Peter Kuhn, Kenneth J Pienta, SR Amend, James B. Hicks
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Leukemias are a group of heterogeneous hematological malignancies driven by diverse genetic variations, and the advent of genomic sequencing technologies facilitates the investigation of genetic abnormalities in leukemia. However, these sequencing-based studies mainly focus on nuclear DNAs. Increasing evidence indicates that mitochondrial dysfunction is an important mechanism of leukemia pathogenesis, which is closely related to the mitochondrial genome variations. Here, we provide an overview of current research progress concerning mitochondrial genetic variations in leukemia, encompassing gene mutations and copy number variations. We also summarize currently accessible mitochondrial DNA (mtDNA) sequencing methods. Notably, somatic mtDNA mutations may serve as natural genetic barcodes for lineage tracing and longitudinal assessment of clonal dynamics. Collectively, these findings enhance our understanding of leukemia pathogenesis and foster the identification of novel therapeutic targets and interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a51e65e882f1837d1adb0460175f694592b51dd7" target='_blank'>
              Mitochondrial genetic variations in leukemia: a comprehensive overview
              </a>
            </td>
          <td>
            Ao Zhang, Wenbing Liu, S. Qiu
          </td>
          <td>2024-09-05</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The study of chromosomal shape, characteristics, and behavior in somatic cell division (mitosis) during growth and development and in germ cell division (meiosis) during reproduction is known as cytogenetics. Many techniques can be used for cytogenetics, including fluorescent in situ hybridization (FISH), spectral karyotyping (SKY), multicolor FISH (M-FISH), microarray, and optical genome mapping (OGM). OGM is a novel genome-wide method that can identify structural variants (SVs) and copy number variants (CNVs) with only one test. Genomic structural information that is difficult to obtain with DNA sequencing can be promptly obtained with OGM, in which large molecule lengths can be mapped at a reasonable cost. OGM is increasingly being used to investigate chromosome abnormalities in genetic disorders and human cancer, but it was first utilized in genome assembly and research. According to recent research, OGM is capable of identifying every clinically significant variation seen in trials using conventional care. OGM is being utilized to identify genomic abnormalities in patients with malignancies and constitutional illnesses. It is regarded as a revolution in the field of cytogenetics. Rather than sequencing DNA, OGM relies on DNA labeling. Currently, the OGM technique with the Saphyr system from Bionano Genomics is a widely utilized platform for cytogenetic analysis. In conclusion, OGM can now be considered a highly reliable method for the identification of chromosomal abnormalities in the diagnosis of tumors and hematological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e6e0a2264e6747c604c9c870d801b4fcbe0131" target='_blank'>
              Cytogenetics and the Revolution of Optical Genome Mapping in the Diagnosis of Diseases.
              </a>
            </td>
          <td>
            Osama M Al-Amer, Yahya Khubrani
          </td>
          <td>2024-09-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Bone marrow (BM) is a common site for solid tumor metastasis, often causing poor outcome. Here, we define the characteristics of BM-disseminated tumor cells (DTCs) using neuroblastoma as a model. We combined single-cell RNA-sequencing (scRNA-seq) and cell-surface protein analysis using 7 paired BM and primary tumor (PT) samples and found that DTCs contain a higher percentage of cycling cells and higher expression of neurodevelopmental genes compared to corresponding PT cells. In 6 patients, the copy number variation profile differed between PT cells and DTCs, indicating spatial heterogeneity. Within the BM, we detected dormant DTCs with potentially reduced chemosensitivity; this population contained cells expressing low levels of the immunotherapeutic antigen GD2 and increased NGFR expression. In conclusion, we characterized DTCs that are particularly challenging to target, offering new avenues for developing therapeutic strategies designed to target all subpopulations within the highly complex metastatic site, thereby preventing the development of drug-resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fc1c99c4919cd41656eef13d151b98e6e5eb661" target='_blank'>
              Metastatic tumor cells in bone marrow differ from paired neuroblastoma tumor and contain subsets with therapy-resistant characteristics
              </a>
            </td>
          <td>
            Caroline Hochheuser, A. Boltjes, Kaylee M. Keller, Simon Tol, Marieke van de Mheen, Carolina Pita Barros, Zeinab van Gestel-Fadaie, André B. P. van Kuilenburg, Sander van Hooff, C. Voermans, Jan J. Molenaar, G. Tytgat, Ilse Timmerman
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Simple Summary This review provides a comprehensive overview of the current knowledge on mutational signatures in colorectal cancer (CRC), highlighting their potential clinical applications. It discusses the challenges and limitations in translating these analyses into clinical practice and proposes strategies to overcome these obstacles. Additionally, it provides insights into future directions and emerging proof-of-concept studies that highlight the translational potential of mutational signature analysis in improving patient care and outcomes in CRC. Abstract A multitude of exogenous and endogenous processes have the potential to result in DNA damage. While the repair mechanisms are typically capable of correcting this damage, errors in the repair process can result in mutations. The findings of research conducted in 2012 indicate that mutations do not occur randomly but rather follow specific patterns that can be attributed to known or inferred mutational processes. The process of mutational signature analysis allows for the inference of the predominant mutational process for a given cancer sample, with significant potential for clinical applications. A deeper comprehension of these mutational signatures in CRC could facilitate enhanced prevention strategies, facilitate the comprehension of genotoxic drug activity, predict responses to personalized treatments, and, in the future, inform the development of targeted therapies in the context of precision oncology. The efforts of numerous researchers have led to the identification of several mutational signatures, which can be categorized into different mutational signature references. In CRC, distinct mutational signatures are identified as correlating with mismatch repair deficiency, polymerase mutations, and chemotherapy treatment. In this context, a mutational signature analysis offers considerable potential for enhancing minimal residual disease (MRD) tests in stage II (high-risk) and stage III CRC post-surgery, stratifying CRC based on the impacts of genetic and epigenetic alterations for precision oncology, identifying potential therapeutic vulnerabilities, and evaluating drug efficacy and guiding therapy, as illustrated in a proof-of-concept clinical trial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b9f4817ac8c40910eefa1198ffb3e184664ea3d" target='_blank'>
              Mutational Signatures in Colorectal Cancer: Translational Insights, Clinical Applications, and Limitations
              </a>
            </td>
          <td>
            G. Crisafulli
          </td>
          <td>2024-08-24</td>
          <td>Cancers</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Objective The aim of this research was to gain a thorough understanding of the processes involved in cell communication and discover potential indicators for treating multiple myeloma (MM) through the use of single-cell RNA sequencing (scRNA-seq). And explored the expression of multiple myeloma-related subgroups on metal ion-related pathways to explore the relationship between MM and metal ions. Methods We performed a fair examination using single-cell RNA sequencing on 32 bone marrow specimens collected from 22 individuals at different points of MM advancement and 9 individuals without any health issues. To analyze the scRNA-seq data, we employed advanced computational algorithms, including Slingshot, Monocle2, and other methodologies. Specifically, Slingshot and Monocle2 enabled us to simulate the biological functionalities of different cell populations and map trajectories of cell developmental pathways. Additionally, we utilized the UMAP algorithm, a powerful dimension reduction technique, to cluster cells and identify genes that were differentially expressed across clusters. Results Our study revealed distinct gene expression patterns and molecular pathways within each patient, which exhibited associations with disease progression. The analysis provided insights into the tumor microenvironment (TME), intra- and inter-patient heterogeneity, and cell-cell interactions mediated by ligand-receptor signaling. And found that multiple myeloma-related subgroups were expressed higher levels in MMP and TIMP pathways, there were some associations. Conclusion Our study presents a fresh perspective for future research endeavors and clinical interventions in the field of MM. The identified gene expression patterns and molecular pathways hold immense potential as therapeutic targets for the treatment of multiple myeloma. The utilization of scRNA-seq technology has significantly contributed to a more precise understanding of the complex cellular processes and interactions within MM. Through these advancements, we are now better equipped to unravel the underlying mechanisms driving the development and progression of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b00964a82cc272b14f247335d0402dedfb1db2" target='_blank'>
              Unveiling the cellular landscape: insights from single-cell RNA sequencing in multiple myeloma
              </a>
            </td>
          <td>
            Xinhan Li, Zhiheng Lin, Fu Zhao, Tianjiao Huang, weisen fan, Lijun Cen, Jun Ma
          </td>
          <td>2024-08-30</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Drug resistance remains a major clinical challenge in the treatment of colorectal cancer (CRC) with conventional chemotherapy. Analyzing changes within tumor cells and tumor microenvironment (TME) after treatment and in metastases is essential to understanding how resistance develops. In this study, we analyzed scRNA-seq data from 56 CRCs including treatment-naïve tumors and tumors treated with standard chemotherapy with the known response status (18 responders and 6 progressors). In our cohort, primary left-sided CRCs were associated with metastatic potential mesenchymal phenotype and with depleted B cells. In the post-treatment CRC, there was a high prevalence of dendritic cells (DC) in the TME in the response group. The DC-derived signature was associated with better survival in a large CRC cohort from the TCGA. In progressors there was an enrichment of pericyte-like fibroblasts, which appeared to be associated with poor survival in a CRC-TCGA cohort. Progressors also showed elevated fractions of exhausted CD8+ T memory cells suggesting a pro-inflammatory TME. In tumor cells of progressors group, we identified specific expression of chemo-protective markers MTRNR2L1 and CDX1; and their co-expression with stemness-related immune-checkpoint CD24. In summary, scRNA-seq provides a valuable information for the discovery of prognostic markers, and reveals distinct features potentially underlying response to chemotherapy or disease progression in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0d907795ada63036c087467c94e2007531b414d" target='_blank'>
              Single-cell Transcriptome Profiling of Post-treatment and Treatment-naïve Colorectal Cancer: Insights into Putative Mechanisms of Chemoresistance
              </a>
            </td>
          <td>
            Grigory A. Puzanov, Clémence Astier, Andrey A. Yurchenko, Gerome Jules-Clement, Fabrice André, Aurélien Marabelle, A. Hollebecque, Sergey Nikolaev
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Neuroendocrine transdifferentiation (NEtD), also commonly referred to as lineage plasticity, emerges as an acquired resistance mechanism to molecular targeted therapies in multiple cancer types, predominately occurs in metastatic epidermal growth factor receptor (EGFR)‐mutant non‐small cell lung cancer treated with EGFR tyrosine kinase inhibitors and metastatic castration‐resistant prostate cancer treated with androgen receptor targeting therapies. NEtD tumors are the lethal cancer histologic subtype with unfavorable prognosis and limited treatment. A comprehensive understanding of molecular mechanism underlying targeted‐induced plasticity could greatly facilitate the development of novel therapies. In the past few years, increasingly elegant studies indicated that NEtD tumors share key the convergent genomic and phenotypic characteristics irrespective of their site of origin, but also embrace distinct change and function of molecular mechanisms. In this review, we provide a comprehensive overview of the current understanding of molecular mechanism in regulating the NEtD, including genetic alterations, DNA methylation, histone modifications, dysregulated noncoding RNA, lineage‐specific transcription factors regulation, and other proteomic alterations. We also provide the current management of targeted therapies in clinical and preclinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7693a38c0d6b0d0a5a68e9cf1e1598b353f12ed8" target='_blank'>
              Neuroendocrine transdifferentiation in human cancer: molecular mechanisms and therapeutic targets
              </a>
            </td>
          <td>
            Jun Jiang, Donghui Han, Jiawei Wang, Weihong Wen, Rui Zhang, Weijun Qin
          </td>
          <td>2024-10-01</td>
          <td>MedComm</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="While numerous technologies for the characterization of potential off-target editing by CRISPR/Cas9 have been described, the development of new technologies and analytical methods for off-target recombination by Large Serine Integrases (LSIs) are required to advance the application of LSIs for therapeutic gene integration. Here we describe a suite of off-target recombination discovery technologies and a hybrid capture validation approach as a comprehensive framework for off-target characterization of LSIs. HIDE-Seq (High-throughput Integrase-mediated DNA Event Sequencing) is a PCR-free unbiased genome-wide biochemical assay capable of discovering sites with LSI-mediated free DNA ends (FDEs) and off-target recombination events. Cryptic-Seq is a PCR-based unbiased genome-wide biochemical or cellular-based assay that is more sensitive than HIDE-Seq but is limited to the discovery of sites with off-target recombination. HIDE-Seq and Cryptic-Seq discovered 38 and 44,311 potential off-target sites respectively. 2,455 sites were prioritized for validation by hybrid capture NGS in LSI-edited K562 cells and off-target integration was detected at 52 of the sites. We benchmarked the sensitivity of our LSI off-target characterization framework against unbiased whole genome sequencing (WGS) on LSI-edited samples, and off-target integration was detected at 5 sites with an average genome coverage of 40x. This reflects a greater than 10-fold increase in sensitivity for off-target detection compared to WGS, however only 4 of the 5 sites detected by WGS were also validated by hybrid capture NGS. The dissemination of these technologies will help advance the application of LSIs in therapeutic genome editing by establishing methods and benchmarks for the sensitivity of off-target detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d296a2b7012ef9e1a6d1a1f622c6127d7920625" target='_blank'>
              Large Serine Integrase Off-target Discovery and Validation for Therapeutic Genome Editing
              </a>
            </td>
          <td>
            Dane Z. Hazelbaker, Japan B. Mehta, Connor McGinnis, Didac Santesmasses, Anne M. Bara, Xiaoyu Liang, Thomas Biondi, Tim J. Fennell, Nils Homer, Brett Estes, Jenny Xie, Davood Norouzi, Kaivalya Molugu, Ravindra Amunugama, Chong Luo, Parth Amin, Xiarong Shi, Jesse C Cochrane, Sandeep Kumar, Jie Wang, Matthew H. Bakalar, Jonathan D. Finn, Daniel J. O’Connell
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Cancer stem cells (CSCs) are widely acknowledged as the drivers of tumor initiation, epithelial‐mesenchymal transition (EMT) progression, and metastasis. Originating from both hematologic and solid malignancies, CSCs exhibit quiescence, pluripotency, and self‐renewal akin to normal stem cells, thus orchestrating tumor heterogeneity and growth. Through a dynamic interplay with the tumor microenvironment (TME) and intricate signaling cascades, CSCs undergo transitions from differentiated cancer cells, culminating in therapy resistance and disease recurrence. This review undertakes an in‐depth analysis of the multifaceted mechanisms underlying cancer stemness and CSC‐mediated resistance to therapy. Intrinsic factors encompassing the TME, hypoxic conditions, and oxidative stress, alongside extrinsic processes such as drug efflux mechanisms, collectively contribute to therapeutic resistance. An exploration into key signaling pathways, including JAK/STAT, WNT, NOTCH, and HEDGEHOG, sheds light on their pivotal roles in sustaining CSCs phenotypes. Insights gleaned from preclinical and clinical studies hold promise in refining drug discovery efforts and optimizing therapeutic interventions, especially chimeric antigen receptor (CAR)‐T cell therapy, cytokine‐induced killer (CIK) cell therapy, natural killer (NK) cell‐mediated CSC‐targeting and others. Ultimately use of cell sorting and single cell sequencing approaches for elucidating the fundamental characteristics and resistance mechanisms inherent in CSCs will enhance our comprehension of CSC and intratumor heterogeneity, which ultimately would inform about tailored and personalized interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d170930289915a8a14d31763d5c1c28676522808" target='_blank'>
              Intricate relationship between cancer stemness, metastasis, and drug resistance
              </a>
            </td>
          <td>
            T. C. Dakal, Ravi Bhushan, Caiming Xu, Bhana Ram Gadi, S. Cameotra, Vikas Yadav, Jarek Maciaczyk, Ingo G H Schmidt-Wolf, Abhishek Kumar, Amit Sharma
          </td>
          <td>2024-09-21</td>
          <td>MedComm</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d61b6a296f0f4f8539e1a19d49c6d27ddf36337d" target='_blank'>
              Deciphering the role of sphingolipid metabolism in the immune microenvironment and prognosis of esophageal cancer via single-cell sequencing and bulk data analysis
              </a>
            </td>
          <td>
            Rongzhang He, Jing Tang, Haotian Lai, Tianchi Zhang, Linjuan Du, Siqi Wei, Ping Zhao, Guobin Tang, Jie Liu, Xiufang Luo
          </td>
          <td>2024-09-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Despite the crucial role of T cell clones in the anti-tumor activity, their characterization and association with clinical outcome following immune checkpoint inhibitors (ICI) is lacking. Here we analyzed paired single-cell RNA-sequencing/T-cell receptor sequencing of 767,606 T cells from 460 samples spanning 6 cancer types. We found a robust signature of response based on expanded CD8+ clones that differentiates between responders and non-responders. Analysis of persistent clones showed transcriptional changes that are differentially induced by therapy in the different response groups, suggesting an improved reinvigoration capacity in responding patients. Moreover, a gene trajectory analysis revealed changes in the pseudo-temporal state of de-novo clones that are associated with response to therapy. Lastly, we found that clones shared between tumor and blood are more abundant in non-responders and execute distinct transcriptional programs. Overall, our results highlight differences in clonal transcriptional states that are linked to patient response, offering valuable insights into the mechanisms driving effective anti-tumor immunity. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3504c21c4fc5c6f9ee1a5a18b7688458b8f59956" target='_blank'>
              Single-cell meta-analysis of T cells reveals clonal dynamics of response to checkpoint immunotherapy
              </a>
            </td>
          <td>
            Ofir Shorer, Asaf Pinhasi, Keren Yizhak
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Pediatric acute lymphoblastic leukemia (ALL) is a highly heterogeneous disease at the molecular level, with over 20 subtypes of B-cell ALL (B-ALL) identified to date. These subtypes are critical for guiding risk-adapted therapy and precision medicine, as they are defined by specific chromosomal rearrangements, gene expression profiles, aneuploidies and point mutations. However, the ability to accurately classify these subtypes is limited by the technical and economic challenges faced by many centers, particularly in low- and middle-income countries. In this study, we employed transcriptome sequencing to perform molecular classification of B-ALL at diagnosis in pediatric patients enrolled in the multicentric ALLIC-GATLA-2010 clinical protocol in Argentina. Using a combination of bioinformatic tools, we successfully identified single nucleotide variants, fusion transcripts and gene expression profiles, achieving molecular classification in over 90% of patients. Our analysis also revealed high-risk molecular features and novel genetic alterations. These findings hold potential clinical value for improving risk stratification and identifying therapeutic targets, particularly for patients who remain unclassified by conventional diagnostic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f39fa251d291498250d854c37a9028d4e85f79" target='_blank'>
              Molecular Characterization of Pediatric Acute Lymphoblastic Leukemia via Integrative Transcriptomics: A Multicenter Study in Argentina
              </a>
            </td>
          <td>
            Sol Ruiz, María Mercedes Abbate, Ezequiel Sosa, Daniel Avendaño, Ignacio Gomez Mercado, María Laura Lacreu, Mariano Riccheri, Virginia Schuttenberg, Luis Aversa, E. Vazquez, G. Gueron, J. Cotignola
          </td>
          <td>2024-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Introduction Osteosarcoma is a common type of bone cancer characterized by a poor prognosis due to its metastatic nature. The tumor microenvironment (TME) plays a critical role in tumor metastasis and therapy response. Therefore, our study aims to explore the metastatic mechanism of osteosarcoma, potentially opening new avenues for cancer treatment. Methods In this study, we collected data from the GSE152048, GSE14359, and GSE49003 datasets. Differentially expressed genes (DEGs) were identified in osteosarcoma cases with primary and metastatic features using R software and the limma package. Gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses were performed to investigate metastasis-related genes. A protein–protein interaction (PPI) network was established using the STRING database to further analyze these metastasis-associated genes. The abundances of different cell types with a mixed cell population were estimated using the CIBERSORT approach. The scRNA-seq data were analyzed by the Seurat package in R software, and intercellular communications were elucidated using the CellChat R package. Results In this study, 92 DEGs related to metastasis were identified, including 41 upregulated and 51 downregulated genes in both the GSE14359 and GSE49003 datasets. Metastasis-associated pathways were identified, including those involving the cyclin-dependent protein kinase holoenzyme complex, transferase complex, transferring phosphorus-containing groups, SCF ubiquitin ligase complex, and the serine/threonine protein kinase complex. KEGG and PPI network analyses revealed 15 hub genes, including Skp2, KIF20A, CCNF, TROAP, PHB, CKS1B, MCM3, CCNA2, TRIP13, CENPM, Hsp90AB1, JUN, CKS2, TK1, and KIF4A. Skp2 has been known as an E3 ubiquitin ligase involved in osteosarcoma progression. The proportion of CD8+ T cells was found to be higher in metastatic osteosarcoma tissues, and high expression of PHB was associated with a favorable prognosis in osteosarcoma patients. Additionally, 23 cell clusters were classified into eight cell types, including chondrocytes, MSC, T cells, monocytes, tissue stem cells, neurons, endothelial cells, and macrophages. The 15 hub genes were expressed across various cell types, and interactions between different cell types were observed. Conclusion Our study reveals the intricate communication between tumor microenvironment components and tumor metastasis in osteosarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff33ba2ff2e5acd8821d77a975c86bee16bbf2ba" target='_blank'>
              Single-cell RNA sequencing reveals the communications between tumor microenvironment components and tumor metastasis in osteosarcoma
              </a>
            </td>
          <td>
            Jiatong Li, Yang Bai, He Zhang, Ting Chen, Guanning Shang
          </td>
          <td>2024-09-11</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Neuroblastoma (NBL) is the most common extra-cranial solid tumor with <60% long-term survival in high-risk patients. Approximately 15% of high-risk NBL patients develop progressive disease (PD) during induction chemotherapy. As tumor biopsies are often not obtained at time of PD, identifying underlying genetic/genomic features that contribute to disease progression is challenging. Patient-derived xenografts (PDX) established from NBL cells in bone marrow aspirates or blood samples obtained pre-therapy and at time of PD can be phenotypically and molecularly characterized to potentially identify biomarkers predictive of treatment outcome in high-risk NBL patients. Thirty-five high-risk patient tumor, marrow, or blood samples, 16 at diagnosis (DX) and 19 at PD, were received via Children’s Oncology Group (COG) protocol ANBL00B1 and established as PDXs. PDXs were classified by response to chemotherapy (cyclophosphamide, 30mg/kg + topotecan, 0.6 mg/kg daily for 5 days every 21 days, total of 3 cycles) as non-responders (NR) and responders (R), the latter then divided into partial responders (PR) and complete responders (CR). The gene expression in the PDXs was analyzed by RNA-sequencing, and the expression profiles were analyzed by differential analysis (DEGs) via edgeR, binomial regression model, and student’s t-test to identify the marker genes for NR group. The markers were validated using real-time RT-PCR. PDX cyclo/topo response was: non-response (NR, n=13), partial response (PR, n=9), and complete response (CR, n=13). The event-free survival in all treatment groups was significantly extended relative to their control (P < 0.001). Nine of 13 (69%) of the NR PDXs were established from post-mortem PD (PD-PM) samples while nine of thirteen (69%) of the CR group were from diagnosis (DX) samples, demonstrating the clinical relevance of the PDX models. The RNA-seq data analysis identified 639 differentially expressed genes across the NR versus response groups. A regression model and t-test significance further filtered these genes to define a signature comprised of 21 genes for the NR group (P < 1.8e-12). When assessing only DX PDX models, a set of 26 genes was identified as a signature for the NR group (P < 4.1e-05). WNT signaling and Hedgehog pathways were the two most highly activated in the NR group relative to the R group. Of the 639 differentially expressed genes, we investigated nine genes that showed highly significant differential expression by RT-PCR, and seven of the nine genes tested (P < 0.05). Response of PDXs to cyclophosphamide/topotecan was greater for pretherapy (DX) PDXs than PD-PM PDXs. The gene expression signature of NR PDXs can potentially provide a biomarker to identify patients destined to have suboptimal responses to induction chemotherapy. Genes overexpressed in NR PDXs provide potential molecular targets to reverse drug resistance. In addition, the well-characterized and clinically relevant panel of PDXs will be valuable in preclinical studies of novel therapeutic approaches for high-risk neuroblastoma patients.
 Citation Format: Nighat Noureen, Harry May, In-Hyoung Yang, Kristyn McCoy, Jonas Nance, Diana Ixlamati-Nava, Meredith S Irwin, Michael D. Hogarty, Charles P. Reynolds, Min H. Kang. Response of neuroblastoma patient-derived xenografts to cyclophosphamide + topotecan enables identification of gene expression patterns associated with drug resistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/080da9bbd6cc73e1417f72d197d94604906e67f5" target='_blank'>
              Abstract A058 Response of neuroblastoma patient-derived xenografts to cyclophosphamide + topotecan enables identification of gene expression patterns associated with drug resistance
              </a>
            </td>
          <td>
            Nighat Noureen, Harry A. May, In-Hyoung Yang, Kristyn McCoy, Jonas Nance, Diana Ixlamati-Nava, Meredith S Irwin, M. Hogarty, C. P. Reynolds, Min H. Kang
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="DNA rearrangements are thought to arise from two classes of processes. The first class involves DNA breakage and fusion (“cut-and-paste”) without net DNA gain or loss. The second class involves aberrant DNA replication (“copy-and-paste”) and can produce either net DNA gain or loss. We previously demonstrated that the partitioning of chromosomes into aberrant structures of the nucleus, micronuclei or chromosome bridges, can generate cut-and-paste rearrangements by chromosome fragmentation and ligation. Surprisingly, in the progeny clones of single cells that have undergone chromosome bridge breakage, we identified large segmental duplications and short sequence insertions that are commonly attributed to copy-and-paste processes. Here, we demonstrate that both large duplications and short insertions are inherent outcomes of the replication and fusion of unligated DNA ends, a process we term breakage-replication/fusion (B-R/F). We propose that B-R/F provides a unifying explanation for complex rearrangement patterns including chromothripsis and chromoanasynthesis and enables rapid DNA amplification after chromosome fragmentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c002d451b4a62962b702db7317ff655026459317" target='_blank'>
              Chromosome breakage-replication/fusion enables rapid DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, David Pellman
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Background: Immune checkpoint blockades (ICBs) lack clinical efficacy in all-coming pediatric solid tumors. Gene expression-based classification of the tumor immune environment (TiME) can identify a subset of pediatric tumors with a rich immune environment, potentially actionable by ICBs. Methylation reprogramming is a major player in TiME modeling and ICB resistance, its role in pediatric tumors has been little explored. Hypothesis: dual-omic classification, combining DNA-methylation and RNA-sequencing, can identify tumors with epigenetically altered TiME, potentially responsible for ICB resistance. Objective: identify and characterize pediatric solid tumors with epigenetically altered immune phenotypes. Methods: We studied RNA-sequencing and DNA-methylation quantitative data from pediatric extra-cranial solid tumors. We used similarity network fusion (SNF) to individualize immune phenotypes. A preselected customized panel of 1226 immune genes, and their corresponding probes, were used for dual-omics clustering. Differential expression (DE) and methylation (DM) were performed between clusters using volcano3D. Scatter plots visualized the cis-regulation of probe-gene pairs with significant DE-DM. Gene set enrichment analyses (GSEA) were performed with clusterProfiler and gometh packages for DE and DM, respectively. Adjusted p values <0.01 were retained. Results: 184 samples were included. SNF clustering identified 3 phenotypes regrouping 52 (28%), 83 (45%), 49 (27%) samples in clusters (cl) 1, 2 and 3, respectively. Cl1 was characterized by a low expression of immune genes (cold phenotype), cl2 by overexpression in immune genes (hot phenotype), and cl3 by global hypermethylation (epigenetically altered phenotype). Both cl2 and cl3 overexpressed genes of immune checkpoints (CD274, PDCD1) and T-cell activator chemokines (CXCL9, CXCL10), central for ICB sensitivity. However, only cl2 overexpressed major histocompatibility complex (MHC) class I-II genes and their regulators (CIITA, TAP2), essential for antigen (Ag)-presenting machinery and immune recognition. Cl2 and 3 were also enriched for T-, B-cell and pro-inflammatory interferon gamma signaling pathways, known as ICB sensitivity biomarkers. Only cl2 was enriched in Ag processing and presentation pathway, confirming prior observation. DE-DM correlation showed that DNA-methylation programming induced overexpression of most immune genes (78%) in cl2, including CIITA, CXCR3 and MHC genes, when most immune genes (63%) were repressed by methylation in cl3. Methylation-based GSEA confirmed the epigenetic regulation of T- and B-cell signaling in cl2 and cl3, and for Ag presentation in cl2 only. Conclusion We demonstrated that DNA-methylation can reshape the TiME of pediatric solid tumors. We identified a subset of tumors with epigenetically altered immune phenotype characterized by inflamed immune environment but altered for Ag presentation by methylation reprogramming. Future studies should investigate methylation modulators to reverse these mechanisms and enable ICB sensitivity.
 Citation Format: Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-Cazavieille, Charles Joly-Beauparlant, Lara Herrmann, Emeric Texeraud, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Vincent-Philippe Lavallée, Thai Hoa Tran, Sonia Cellot, Daniel Sinnett, Arnaud Droit, Raoul Santiago. Dual-omic characterization of pediatric solid tumors identified a subset of tumors with epigenetically altered immune phenotype [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B057.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/83a9f318a4bf2de77816f3008a669473ec30397b" target='_blank'>
              Abstract B057: Dual-omic characterization of pediatric solid tumors identified a subset of tumors with epigenetically altered immune phenotype
              </a>
            </td>
          <td>
            Stéphanie Bianco, Anas Belaktib, Virgile Raufaste-cazavieille, Charles Joly-Beauparlant, Lara Herrmann, Emeric Texeraud, Sylvie Langlois, Thomas Sontag, Alex Richard-St-Hilaire, Vincent-Philippe Lavallée, T. Tran, Sonia Cellot, D. Sinnett, Arnaud Droit, Raoul Santiago
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Chemoresistance of cancer cells, resulting from various mechanisms, is a significant obstacle to the effectiveness of modern cancer therapies. Targeting fibroblast growth factors (FGFs) and their receptors (FGFRs) is becoming crucial, as their high activity significantly contributes to cancer development and progression by driving cell proliferation and activating signaling pathways that enhance drug resistance. Methods We investigated the potential of honokiol and FGF ligand trap in blocking the FGF1/FGFR1 axis to counteract drug resistance. Using PEAQ-ITC, we verified direct interaction of honokiol with the FGFR1 kinase domain. We then demonstrated the effect of FGF1/FGFR1 inhibition on taltobulin resistance in cells expressing FGFR1. Finally, we generated drug-resistant clones by prolonged exposure of cells with negligible FGFR levels to taltobulin alone, taltobulin and honokiol, or taltobulin and FGF ligand trap. Results We demonstrated for the first time a direct interaction of honokiol with the FGFR1 kinase domain, resulting in inhibition of downstream signaling pathways. We revealed that both honokiol and FGF ligand trap prevent FGF1-dependent protection against taltobulin in cancer cells expressing FGFR1. In addition, we showed that cells obtained by long-term exposure to taltobulin are resistant to both taltobulin and other microtubule-targeting drugs, and exhibit elevated levels of FGFR1 and cyclin D. We also found that the presence of FGF-ligand trap prevents the development of long-term resistance to taltobulin. Conclusion Our results shed light on how blocking the FGF1/FGFR1 axis by honokiol and FGF ligand trap could help develop more effective cancer therapies, potentially preventing the emergence of drug-resistant relapses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfcc03a2b34bad2da53452929f0f653a4ba5ad8b" target='_blank'>
              Overcoming drug resistance of cancer cells by targeting the FGF1/FGFR1 axis with honokiol or FGF ligand trap
              </a>
            </td>
          <td>
            Jakub Szymczyk, Martyna Sochacka, Martyna Biadun, K. Sluzalska, Danuta Witkowska, M. Zakrzewska
          </td>
          <td>2024-09-12</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/827407d1ee135266c2bdec8a7b4b96ce85a8df0f" target='_blank'>
              A genome-wide CRISPR/Cas9 knockout screen identifies SEMA3F gene for resistance to cyclin-dependent kinase 4 and 6 inhibitors in breast cancer.
              </a>
            </td>
          <td>
            Yuko Kawai, Aiko Nagayama, Kazuhiro Miyao, Makoto Takeuchi, Takamichi Yokoe, Tomoe Kameyama, Xinyue Wang, T. Seki, Maiko Takahashi, T. Hayashida, Yuko Kitagawa
          </td>
          <td>2024-10-01</td>
          <td>Breast cancer</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a593c69280acd8e1ddd89d1994ea815f977cea" target='_blank'>
              Chromosomal instability: a key driver in glioma pathogenesis and progression
              </a>
            </td>
          <td>
            Adele Mazzoleni, W. A. Awuah, Vivek Sanker, H. Bharadwaj, Nicholas Aderinto, J. K. Tan, H. Huang, Jeisun Poornaselvan, Muhammad Hamza Shah, O. Atallah, Aya Tawfik, Mohamed Elmanzalawi, Sama Hesham Ghozlan, T. Abdul-Rahman, Jeremiah Adepoju Moyondafoluwa, Athanasios Alexiou, Marios Papadakis
          </td>
          <td>2024-09-04</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8db9094952a2d4596bce488ed054cdce260574fd" target='_blank'>
              Integrative genomic analyses identify neuroblastoma risk genes involved in neuronal differentiation.
              </a>
            </td>
          <td>
            Matilde Tirelli, Ferdinando Bonfiglio, Sueva Cantalupo, Annalaura Montella, M. Avitabile, Teresa Maiorino, S. Diskin, Achille Iolascon, Mario Capasso
          </td>
          <td>2024-08-27</td>
          <td>Human genetics</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="A major goal in biology is to uncover the relationship between genotype and phenotype. However, identifying gene function is often hampered by genetic redundancy. For example, under standard laboratory conditions, three-quarters of the genes in the human pathogen Streptococcus pneumoniae are non-essential. A powerful approach to unravel genetic redundancy is by identifying gene-gene interactions. To uncover genetic interactions (GIs) in S. pneumoniae on a genome-wide scale, a generally applicable dual CRISPRi-Seq method and associated analysis pipeline was developed. Specifically, we created a library of 869 dual sgRNAs targeting high-confidence operons that encode essential and non-essential genes, covering over 70% of the genetic elements in the pneumococcal genome. Testing these 378,015 unique combinations, 4,026 significant GIs were identified, including 1,935 negative and 2,091 positive interactions. Besides known GIs, we found and confirmed previously unknown interactions involving genes responsible for fundamental cellular processes such as cell division, cell shape maintenance, and chromosome segregation. The presented methods and bioinformatic approaches can serve as a roadmap for genome-wide gene interaction studies in other organisms. Lastly, all interactions are available for exploration via the Pneumococcal Genetic Interaction Network (PneumoGIN) at https://veeninglab.shinyapps.io/PneumoGIN, which can serve as a starting point for new biological discoveries and translational research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28dd78fdd99cdc75fb8cdbb9e56ea5f0c3b73930" target='_blank'>
              Dual CRISPRi-Seq for genome-wide genetic interaction studies identifies key genes involved in the pneumococcal cell cycle
              </a>
            </td>
          <td>
            Julien Dénéréaz, Elise Eray, Bimal Jana, Vincent de Bakker, Horia Todor, Tim van Opijnen, Xue Liu, J. Veening
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 METTL1 is an RNA methyltransferase which catalyzes N7-methylguanosine (m7G) methylation on various RNAs, particularly on tRNAs. METTL1 has been reported to be amplified and/or overexpressed and high expression correlates with poor patient prognosis in different types of adult cancers. Our ZERO Childhood Cancer Program focuses on identifying and validating potential new drug targets and developing novel therapies for clinical use. ZERO has sequenced the whole genome and transcriptome in over 500 high-risk pediatric tumors and found that METTL1 gene copy number and gene expression are significantly higher in a subset of high-grade glioma, neuroblastoma, diffuse midline glioma and fusion positive rhabdomyosarcoma. Analysis of the publicly available SEQC microarray dataset showed that high METTL1 expression correlates with poor outcomes in MYCN amplified neuroblastoma patients. We validated the functional roles of METTL1 gene amplification and overexpression in a few pediatric cancer cell line models using the Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) Cas13d knockdown system, in dual lentiviral constructs. We found that knocking down METTL1 significantly decreased METTL1 amplified neuroblastoma and rhabdomyosarcoma cell viability and colony formation but not in METTL1 non-amplified neuroblastoma and rhabdomyosarcoma cells. Gene Set Enrichment Analysis from RNA sequencing data revealed that knocking down METTL1 resulted in a decrease in Myc target gene expression, and an increased in apoptosis, interferon alpha response and p53 pathway hallmarks. Western blot analysis confirmed that knocking down METTL1 led to a decrease in the MCL-1 antiapoptotic protein expression and an increase in the proapoptotic protein expression such as cleaved PARP and caspase 3. Taken together, our data demonstrate that METTL1 is a potential therapeutic target in METTL1 amplified pediatric cancers.
 Citation Format: Pei Y. Liu, Steven He, Chelsea Mayoh, Alice Salib, Antoine De Weck, Paul G. Ekert. Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64085373809830d692bd2bd083b18380125ea0bf" target='_blank'>
              Abstract A012 Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d
              </a>
            </td>
          <td>
            Pei Y. Liu, Steven He, C. Mayoh, A. Salib, Antoine de Weck, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/460b0c287af289dbf822999647d9d57a5f2960b9" target='_blank'>
              Integrative single-cell RNA-seq and spatial transcriptomics analyses reveal diverse apoptosis-related gene expression profiles in EGFR-mutated lung cancer
              </a>
            </td>
          <td>
            Motohiro Izumi, Masanori Fujii, I. Kobayashi, Vivian Ho, Y. Kashima, H. Udagawa, Daniel B. Costa, Susumu S. Kobayashi
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="TP53, the most frequently mutated gene in human cancer, encodes a transcriptional activator that induces myriad downstream target genes. Despite the importance of p53 in tumor suppression, the specific p53 target genes important for tumor suppression remain unclear. Recent studies have identified the p53-inducible gene Zmat3 as a critical effector of tumor suppression, but many questions remain regarding its p53-dependence, activity across contexts, and mechanism of tumor suppression alone and in cooperation with other p53-inducible genes. To address these questions, we used Tuba-seqUltra somatic genome editing and tumor barcoding in a mouse lung adenocarcinoma model, combinatorial in vivo CRISPR/Cas9 screens, meta-analyses of gene expression and Cancer Dependency Map data, and integrative RNA-sequencing and shotgun proteomic analyses. We established Zmat3 as a core component of p53-mediated tumor suppression and identified Cdkn1a as the most potent cooperating p53-induced gene in tumor suppression. We discovered that ZMAT3/CDKN1A serve as near-universal effectors of p53-mediated tumor suppression that regulate cell division, migration, and extracellular matrix organization. Accordingly, combined Zmat3-Cdkn1a inactivation dramatically enhanced cell proliferation and migration compared to controls, akin to p53 inactivation. Together, our findings place ZMAT3 and CDKN1A as hubs of a p53-induced gene program that opposes tumorigenesis across various cellular and genetic contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e5afaeddf5354538c432a3c29c86495b8e7485c" target='_blank'>
              Integrative multiomic approaches reveal ZMAT3 and p21 as conserved hubs in the p53 tumor suppression network
              </a>
            </td>
          <td>
            A. Boutelle, Aicha R. Mabene, David Yao, Haiqing Xu, Mengxiong Wang, Yuning J. Tang, Steven S. Lopez, Sauradeep Sinha, J. Demeter, Ran Cheng, Brooks A. Benard, Liz J. Valente, Alexandros P. Drainas, Martin Fischer, Ravi Majeti, D. Petrov, Peter K. Jackson, Fan Yang, M. Winslow, Mike C Bassik, L. Attardi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="The role of Erythroid cells in immune regulation and immunosuppression is one of the emerging topics in modern immunology that still requires further clarification as Erythroid cells from different tissues and different species express different immunoregulatory molecules. In this study, we performed a thorough investigation of human bone marrow Erythroid cells from adult healthy donors and adult acute lymphoblastic leukemia patients using the state-of-the-art single-cell targeted proteomics and transcriptomics via BD Rhapsody and cancer-related gene copy number variation analysis via NanoString Sprint Profiler. We found that human bone marrow Erythroid cells express the ARG1, LGALS1, LGALS3, LGALS9, and C10orf54 (VISTA) immunosuppressive genes, CXCL5, CXCL8, and VEGFA cytokine genes, as well as the genes involved in antimicrobial immunity and MHC Class II antigen presentation. We also found that ARG1 gene expression was restricted to the single erythroid cell cluster that we termed ARG1-positive Orthochromatic erythroblasts and that late Erythroid cells lose S100A9 and gain MZB1 gene expression in case of acute lymphoblastic leukemia. These findings show that steady-state erythropoiesis bone marrow Erythroid cells express myeloid signature genes even without any transdifferentiating stimulus like cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b037304bfa3fec87e38c7d6b632f2e503c9b8fe" target='_blank'>
              Single-cell multi-omics reveal stage of differentiation and trajectory-dependent immunity-related gene expression patterns in human erythroid cells
              </a>
            </td>
          <td>
            R. Perik-Zavodskii, Olga Perik-Zavodskaia, S. Alrhmoun, M. Volynets, J. Shevchenko, K. Nazarov, Vera Denisova, S. Sennikov
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5ea42f9ac2f6d502e99cce5e348de337e22eda" target='_blank'>
              Effects of HOX family regulator-mediated modification patterns and immunity characteristics on tumor-associated cell type in endometrial cancer
              </a>
            </td>
          <td>
            Jiaolin Yang, Jinpeng Li, Sufen Li, Yutong Yang, Huancheng Su, Hongrui Guo, Jing Lei, YaLin Wang, KaiTing Wen, Xia Li, Sanyuan Zhang, Zhe Wang
          </td>
          <td>2024-08-14</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac67a3ed7d66d72ed2d1b0afc516f030fad809a" target='_blank'>
              Immunological hide-and-seek: epigenetically reprogrammed cancer cells and the dynamics of CD8+ T cells.
              </a>
            </td>
          <td>
            Jie-Ting Low, M. W. Chan, Cheng-Huang Shen, Kuo-Liang Wei
          </td>
          <td>2024-09-04</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Type B1 and B2 thymomas are lymphocyte-rich malignant tumors with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, non-coding regions remain uncharacterized. Here, we developed a rigorous tumor isolation method from lymphocyte-rich thymoma tissues and identified somatic mutations in non-coding and mitochondrial DNA. Methods CD205+CD45- pure tumor cells were isolated from fresh-frozen tissues using DEPArray system. Deep whole-genome sequencing was performed, and recurrent somatic alterations in coding, non-coding, and mitochondria regions were systemically identified by computational framework. The mutations were classified according to gene function, cis-regulatory element, and mutational signature. Results The total number of somatic mutations was approximately 80 times higher in non-coding regions than in coding regions in type B1-2 thymomas (1,671.3 vs. 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of cases exhibited fewer than four mutations in coding regions. A systematic non-coding analysis identified a total of 405.0 mutations per case on cis-regulatory elements, and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Mitochondrial heteroplasmy was observed in 90% of cases, with a significant proportion of mutations located in D-loop region. The single-base substitution pattern was signature 12. Conclusions Numerous non-coding mutations and mitochondrial heteroplasmy were detected in type B1 and B2 thymomas. Given the paucity of coding mutations observed in this disease entity, disruption of the non-coding landscape and mitochondrial heteroplasmic shift may be the primary cause of thymoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/559b9056aa5e8bd421441f43ad9a55ae5b38dbc9" target='_blank'>
              Genome-wide analysis of somatic non-coding mutation patterns and mitochondrial heteroplasmy in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Juliet King, Emma McLean, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancers develop resistance to inhibitors of oncogenes mainly due to target-centric mechanisms such as mutations and splicing. While inhibitors or antagonists force targets to unnatural conformation contributing to protein instability and resistance, activating tumor suppressors may maintain the protein in an agonistic conformation to elicit sustainable growth inhibition. Due to the lack of tumor suppressor agonists, this hypothesis and the mechanisms underlying resistance are not understood. In estrogen receptor (ER)-positive breast cancer (BC), androgen receptor (AR) is a druggable tumor suppressor offering a promising avenue for this investigation. Spatial genomics suggests that the molecular portrait of AR-expressing BC cells in tumor microenvironment corresponds to better overall patient survival, clinically confirming AR's role as a tumor suppressor. Ligand activation of AR in ER-positive BC xenografts reprograms cistromes, inhibits oncogenic pathways, and promotes cellular elasticity toward a more differentiated state. Sustained AR activation results in cistrome rearrangement toward transcription factor PROP paired-like homeobox 1, transformation of AR into oncogene, and activation of the Janus kinase/signal transducer (JAK/STAT) pathway, all culminating in lineage plasticity to an aggressive resistant subtype. While the molecular profile of AR agonist-sensitive tumors corresponds to better patient survival, the profile represented in the resistant phenotype corresponds to shorter survival. Inhibition of activated oncogenes in resistant tumors reduces growth and resensitizes them to AR agonists. These findings indicate that persistent activation of a context-dependent tumor suppressor may lead to resistance through lineage plasticity-driven tumor metamorphosis. Our work provides a framework to explore the above phenomenon across multiple cancer types and underscores the importance of factoring sensitization of tumor suppressor targets while developing agonist-like drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6c58f43e2870ced4e1b54180f1d8d95c299ca56" target='_blank'>
              A molecular switch from tumor suppressor to oncogene in ER+ve breast cancer: Role of androgen receptor, JAK-STAT, and lineage plasticity.
              </a>
            </td>
          <td>
            Sarah Asemota, W. Effah, J. Holt, Daniel Johnson, Linnea Cripe, S. Ponnusamy, T. Thiyagarajan, Yekta Khosrosereshki, Dong-Jin Hwang, Yali He, B. Grimes, Martin D. Fleming, Frances E. Pritchard, Ashley Hendrix, M. Fan, Abhinav K. Jain, Hyo Young Choi, Liza Makowski, D. N. Hayes, Duane D Miller, Lawrence M. Pfeffer, Balaji Santhanam, Ramesh Narayanan
          </td>
          <td>2024-09-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Leukemia is a malignant tumor with high heterogeneity and a complex evolutionary process. It is difficult to resolve the heterogeneity and clonal evolution of leukemia cells by applying traditional bulk sequencing techniques, thus preventing a deep understanding of the mechanisms of leukemia development and the identification of potential therapeutic targets. However, with the development and application of single-cell sequencing technology, it is now possible to investigate the gene expression profile, mutations, and epigenetic features of leukemia at the single-cell level, thus providing a new perspective for leukemia research. In this article, we review the recent applications and advances of single-cell sequencing technology in leukemia research, discuss its potential for enhancing our understanding of the mechanisms of leukemia development, discovering therapeutic targets and personalized treatment, and provide reference guidelines for the significance of this technology in clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059e4fa52cbb0003a82e3f417ae0958def767ace" target='_blank'>
              Application and research progress of single cell sequencing technology in leukemia
              </a>
            </td>
          <td>
            Dan Xie, Bangquan An, Mingyue Yang, Lei Wang, Min Guo, Heng Luo, Shengwen Huang, Fa Sun
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Upon infection, human papillomavirus (HPV) manipulates host cell gene expression to create an environment that is supportive of a productive and persistent infection. The virus-induced changes to the host cell’s transcriptome are thought to contribute to carcinogenesis. Here, we show by RNA-sequencing that oncogenic HPV18 episome replication in primary human foreskin keratinocytes (HFKs) drives host transcriptional changes that are consistent between multiple HFK donors. We have previously shown that HPV18 episome replication in HFKs results in post-transcriptional stabilisation of the host chromatin insulation protein CTCF. Since CTCF is an important regulator of host cell transcription via coordination of epigenetic boundaries and long-range chromosomal interactions, we hypothesised that HPV18-induced stabilisation of CTCF may contribute to host transcription reprogramming. Analysis of CTCF binding in the host cell genome by ChIP-Seq revealed that while the total number of CTCF binding sites is not altered by the virus, there are a sub-set of CTCF binding sites that are either enriched or depleted of CTCF. Many of these altered sites are clustered within regulatory elements of differentially expressed genes, including the tumour suppressor gene cell adhesion molecule 1 (CADM1), which supresses epithelial cell growth and invasion. We show that HPV18 establishment results in reduced CTCF binding at the CADM1 promoter and upstream enhancer. Loss of CTCF binding is coincident with epigenetic repression of CADM1, in the absence of CpG hypermethylation, while adjacent genes including the transcriptional regulator ZBTB16 are activated. These data indicate that the CADM1 locus is subject to topological rearrangement following HPV18 establishment. We tested this hypothesis using 4C-Seq (circular chromosome confirmation capture-sequencing) and show that HPV18 establishment causes a loss of long-range chromosomal interactions between the CADM1 transcriptional start site and the upstream transcriptional enhancer. These data show that HPV18 manipulates host cell promoter-enhancer interactions to drive transcriptional reprogramming that may contribute to HPV-induced disease progression. AUTHOR SUMMARY Infection with oncogenic HPV is the cause of numerous cancer types, which generally arise after persistent HPV infection. Upon infection, HPV alters the gene expression profile of infected cells to facilitate virus replication and persistence. Multiple mechanisms of HPV-induced host cell reprogramming have been previously suggested. Here, we show that HPV infection induces rearrangement of specific genomic loci by altering the chromatin binding of the host cell protein CTCF, an important regulator of chromatin architecture. Loss of CTCF binding to a cluster of binding sites at the CADM1 locus on chromosome 11 is coincident with epigenetic reprogramming and disruption of long-range chromatin interactions, resulting in transcriptional repression of CADM1. Our data show that repression of CADM1 is an early event in HPV-driven disease, preceding hypermethylation of the CADM1 transcriptional promoter that is frequently observed in HPV-driven cancers, demonstrating a novel mechanism of HPV-induced host cell transcriptional reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296755d464d45666368bedcd29a51e084375e664" target='_blank'>
              Repression of CADM1 transcription by HPV type 18 is mediated by three-dimensional rearrangement of promoter-enhancer interactions
              </a>
            </td>
          <td>
            Karen Campos-León, J. Ferguson, Thomas Günther, C. D. Wood, S. Wingett, Selin Pekel, C. S. Varghese, C. Várnai, Michelle West, Andrew Beggs, A. Grundhoff, B. Noyvert, Sally Roberts, Joanna L. Parish
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Richter's transformation (RT) is defined as the evolution of chronic lymphocytic leukemia (CLL) or small lymphocytic lymphoma (SLL) into an aggressive lymphoma, most commonly diffuse large B-cell lymphoma. This complication is rare and aggressive, with poor prognosis and dismal survival. Clonal relationship with the underlying CLL/SLL, observed in ∼80% of cases, represents one of the main factors affecting prognosis. Treatment has been historically based on chemoimmunotherapy, but frequent mutations in genes involved in cell survival and proliferation-such as TP53, NOTCH1, MYC, CDKN2A-confer resistance to standard treatments. During the last years, advances in the knowledge of the biological mechanisms underlying RT allowed to identify genetic and molecular lesions that can potentially be targeted by novel selective agents. Pathway and checkpoint inhibitors, bispecific antibodies and CAR T-cell therapy are currently under investigation and represent promising treatment options. This review summarizes current biological evidence and available data on novel therapeutic agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a1bac15f3b796b2ef346e32ef1ee6b7161f4b12" target='_blank'>
              Advances in the understanding of molecular genetics and therapy of Richter transformation in chronic lymphocytic leukemia.
              </a>
            </td>
          <td>
            M. Deodato, A. Frustaci, Arianna Zappaterra, Alberto Rapella, C. Gambacorti-Passerini, R. Cairoli, Marco Montillo, Alessandra Tedeschi
          </td>
          <td>2024-09-02</td>
          <td>Leukemia & lymphoma</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, Sarah H. Johnson, W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Acute myeloid leukemia with complex karyotype (ckAML) is characterized by high genomic complexity, including frequent TP53 mutations and chromothripsis. We hypothesized that the numerous genomic rearrangements could reposition active enhancers near proto-oncogenes, leading to their aberrant expression. We developed pyjacker, a computational tool for the detection of enhancer hijacking events, and applied it to a cohort of 39 ckAML samples. Pyjacker identified motor neuron and pancreas homeobox 1 (MNX1), a gene aberrantly expressed in 1.4% of AML patients, often as a result of del(7)(q22q36) associated with hijacking of a CDK6 enhancer. MNX1-activated cases show significant co-occurrence with BCOR mutations and a gene signature shared with t(7;12)(q36;p13) pediatric AML. We demonstrated that MNX1 is a dependency gene, as its knockdown in a xenograft model reduces leukemia cell fitness. In conclusion, enhancer hijacking is a frequent mechanism for oncogene activation in AML. Statement of significance This study examines the consequences of structural alterations and demonstrates that proto-oncogene activation by enhancer hijacking is an overlooked pathomechanism in AML. MNX1 overexpression demonstrates that deletions on chromosome 7q can not only lead to haploinsufficiency, but also to activation of oncogenes by enhancer hijacking, providing a novel leukemogenic mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb3e7eaaf436e861083291af3f4d69f9c709309" target='_blank'>
              Pyjacker identifies enhancer hijacking events in acute myeloid leukemia including MNX1 activation via deletion 7q
              </a>
            </td>
          <td>
            Etienne Sollier, Anna Riedel, U. Toprak, Justyna A. Wierzbinska, D. Weichenhan, J. P. Schmid, M. Hakobyan, A. Touzart, E. Jahn, B. Vick, Fiona Brown-Burke, Katherine Kelly, Simge Kelekçi, Anastasija Pejkovska, Ashish Goyal, Marion Bähr, Kersten Breuer, Mei-Ju May Chen, Maria Llamazares-Prada, M. Hartmann, M. Schönung, Nádia Correia, A. Trumpp, Yomn Abdullah, Ursula Klingmüller, S. S. Mughal, B. Brors, Frank Westermann, Matthias Schlesner, S. Vosberg, T. Herold, P. Greif, Dietmar Pfeifer, Michael Lübbert, Thomas Fischer, F. H. Heidel, C. Gebhard, W. Walter, Torsten Haferlach, Ann-Kathrin Eisfeld, K. Mrózek, D. Nicolet, L. Bullinger, L. Smeenk, Claudia Erpelinck, Roger Mulet-Lazaro, Ruud Delwel, Aurélie Ernst, Michael Scherer, P. Lutsik, I. Jeremias, Konstanze Döhner, H. Döhner, D. Lipka, C. Plass
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="While T cell lymphomas are classified as mature neoplasms, emerging evidence indicates that malignant transformation may occur at an earlier stage of T cell maturation. In this study, we determined clonal architectures in a broad range of T cell lymphomas. Our multidimensional profiling indicates that a large part of these lymphomas in fact emerge from an immature lymphoid T cell precursor at a maturation stage prior to V(D)J rearrangement that undergoes branching evolution. Consequently, at single cell resolution we observed considerable clonal tiding under selective therapeutic pressure. T cell receptor next-generation sequencing suggested a highly biased usage of TRBV20-1 gene segments as part of multiple antigen receptor rearrangements per patient. The predominance of TRBV20-1 was found across all major T cell lymphoma subtypes analyzed. This suggested that this particular V gene - independently of complementarity-determining region 3 (CDR3) configuration - may represent a driver of malignant transformation. Together, our data indicate that T cell lymphomas derive from immature lymphoid precursors and display considerable intratumoral heterogeneity that may provide the basis for relapse and resistance in these hard-to-treat cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5cc9e8d39a7b3561d529df56bcd87a167e0b44e" target='_blank'>
              T-cell receptor architecture and clonal tiding provide insight into the transformation trajectory of peripheral T-cell lymphomas.
              </a>
            </td>
          <td>
            E. Willscher, C. Schultheiss, L. Paschold, Franziska Lea Schümann, Paul Schmidt-Barbo, Benjamin Thiele, M. Bauer, Claudia Wickenhauser, Thomas Weber, Mascha Binder
          </td>
          <td>2024-08-29</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Transgenerational epigenetic inheritance (TEI) mediated by transposable elements (TEs) is well established in plants. However, the molecular determinants and functional impact of this additional system of inheritance are poorly characterized. Through comprehensive analyses of severe DNA methylation loss over TEs in an experimental population and in >700 natural strains of Arabidopsis thaliana, we uncovered hundreds of TE epivariants in nature, most of which can be inherited independently of DNA sequence changes and thus are true epialleles. Furthermore, natural epivariants are enriched near genes and we uncovered a novel role for the Paf1 complex as a major facilitator of their prevalence. Conversely, related TE copies limit epiallelic variation, by producing the small RNAs that guide DNA methylation in trans. Crucially, natural epivariants mainly affect stress-responsive genes, making them unique targets for selection. Our findings therefore demonstrate that TEs are important mediators of transgenerational epigenetic variation in nature, with singular properties for plant adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac6bd4e49538ff89fe2173d3326edef5a9f7c98a" target='_blank'>
              Transposable elements are prevalent vectors of transgenerational epigenetic inheritance in nature
              </a>
            </td>
          <td>
            P. Baduel, Louna De Oliveira, E. Caillieux, Grégoire Bohl-Viallefond, Mounia El Messaoudi, Ciana Xu, Matteo Barois, Vipin Singh, Alexis Sarazin, Martine Boccara, Elodie Gilbault, Antoine de France, L. Quadrana, O. Loudet, V. Colot
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Translocation renal cell carcinoma (tRCC) is an aggressive subtype of kidney cancer driven by TFE3 gene fusions, which act via poorly characterized downstream mechanisms. Here we report that TFE3 fusions transcriptionally rewire tRCCs toward oxidative phosphorylation (OXPHOS), contrasting with the highly glycolytic metabolism of most other renal cancers. This TFE3 fusion-driven OXPHOS program, together with heightened glutathione levels found in renal cancers, renders tRCCs sensitive to reductive stress – a metabolic stress state induced by an imbalance of reducing equivalents. Genome-scale CRISPR screening identifies tRCC-selective vulnerabilities linked to this metabolic state, including EGLN1, which hydroxylates HIF-1α and targets it for proteolysis. Inhibition of EGLN1 compromises tRCC cell growth by stabilizing HIF-1a and promoting metabolic reprogramming away from OXPHOS, thus representing a vulnerability to OXPHOS-dependent tRCC cells. Our study defines a distinctive tRCC-essential metabolic program driven by TFE3 fusions and nominates EGLN1 inhibition as a therapeutic strategy to counteract fusion-induced metabolic rewiring.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2568e01eb416e6b7b22ba26a8e1971911ede6d44" target='_blank'>
              TFE3 fusions direct an oncogenic transcriptional program that drives OXPHOS and unveils vulnerabilities in translocation renal cell carcinoma
              </a>
            </td>
          <td>
            Jiao Li, Kaimeng Huang, Fiona McBride, Ananthan Sadagopan, Daniel S. Gallant, Meha Thakur, Prateek Khanna, Bingchen Li, Maolin Ge, Cary N. Weiss, M. Achom, Qingru Xu, Kun Huang, Birgitta A. Ryback, Miao Gui, L. Bar-Peled, S. Viswanathan
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="A fundamental challenge in biomedicine is understanding the mechanisms predisposing individuals to disease. While previous research has suggested that switch-like gene expression is crucial in driving biological variation and disease susceptibility, a systematic analysis across multiple tissues is still lacking. By analyzing transcriptomes from 943 individuals across 27 tissues, we identified 1,013 switch-like genes. We found that only 31 (3.1%) of these genes exhibit switch-like behavior across all tissues. These universally switch-like genes appear to be genetically driven, with large exonic genomic structural variants explaining five (∼18%) of them. The remaining switch-like genes exhibit tissue-specific expression patterns. Notably, tissue-specific switch-like genes tend to be switched on or off in unison within individuals, likely under the influence of tissue-specific master regulators, including hormonal signals. Among our most significant findings, we identified hundreds of concordantly switched-off genes in the stomach and vagina that are linked to gastric cancer (41-fold, p<10-4) and vaginal atrophy (44-fold, p<10-4), respectively. Experimental analysis of vaginal tissues revealed that low systemic levels of estrogen lead to a significant reduction in both the epithelial thickness and the expression of the switch-like gene ALOX12. We propose a model wherein the switching off of driver genes in basal and parabasal epithelium suppresses cell proliferation therein, leading to epithelial thinning and, therefore, vaginal atrophy. Our findings underscore the significant biomedical implications of switch-like gene expression and lay the groundwork for potential diagnostic and therapeutic applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae8ed2dab6e33efaca4d5da4963155c33d9525bc" target='_blank'>
              Switch-like Gene Expression Modulates Disease Susceptibility
              </a>
            </td>
          <td>
            Alber Aqil, Yanyan Li, Zhiliang Wang, Saiful Islam, Madison Russell, T. Kunovac Kallak, Marie Saitou, Omer Gokcumen, N. Masuda
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c443aed28fd474af23dca9ce71d466278127b676" target='_blank'>
              Multi-omics and clustering analyses reveal the mechanisms underlying unmet needs for patients with lung adenocarcinoma and identify potential therapeutic targets
              </a>
            </td>
          <td>
            Ken Asada, S. Kaneko, Ken Takasawa, K. Shiraishi, Norio Shinkai, Yoko Shimada, Satoshi Takahashi, Hidenori Machino, Kazuma Kobayashi, Amina Bolatkan, M. Komatsu, Masayoshi Yamada, M. Miyake, Hirokazu Watanabe, A. Tateishi, T. Mizuno, Y. Okubo, Masami Mukai, Tatsuya Yoshida, Yukihiro Yoshida, H. Horinouchi, Shun-ichi Watanabe, Y. Ohe, Yasushi Yatabe, Takashi Kohno, R. Hamamoto
          </td>
          <td>2024-09-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed3b8831fd786b3f7f846b45187bda3701aa49a" target='_blank'>
              Pan-cancer analysis of m1A writer gene RRP8: implications for immune infiltration and prognosis in human cancers
              </a>
            </td>
          <td>
            Zhihui Huang, Koo Han Yoo, Duohui Li, Qingxin Yu, Luxia Ye, Wuran Wei
          </td>
          <td>2024-09-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents used to treat these cancers are common. One such chemotherapy is doxorubicin. Therefore, this project aims to reveal and understand the mechanisms of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from one ATRT and two RMC primary cell lines (BT16, PEDS005T, and RCRF1009T respectively) utilizing a dose escalation approach. This resulted in DR cell lines that are 10-fold to 30-fold resistant compared to the parental cell line. Upon generation of these stably resistant cell lines, we profiled their transcriptomes by RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1. To further probe these cell lines, we performed whole-genome sequencing (WGS) on two of our DR cell lines (BT16 and PEDS005T DR). Here, we found a chromosome 7 aneuploidy in BT16 DR cells, but no aneuploidy or mutations were seen in PEDS005T DR cells. ABCB1 is located on chromosome 7. We confirmed this aneuploidy via karyotyping. The mechanism of ABCB1 upregulation in the RMC DR cell lines remains unknown, suggesting multiple modalities leading to ABCB1 upregulation. We then overexpressed ABCB1 in two ATRT parental cell lines (BT16 and CHLA-06-ATRT) and one RMC cell line (PEDS005T) which led to enhanced short-term resistance. This contrasted to JMSU, a SMARCB1 wild-type bladder cancer cell line, where there was stable resistance. Together, this suggests that ABCB1 may be necessary but not sufficient to confer doxorubicin resistance in these SMARCB1-deficient cancer cells. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there may be different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, despite the well documented role of ABCB1 in drug resistance, there may be other factors involved in conferring stable resistance in the setting of SMARCB1-deficient cancers.
 Citation Format: Katie T. Skinner, Benjamin P. Lee, Jessica S Yoon, Andrew L. Hong. The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A069.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdc2732137e2a0d6d27507359d94e24f1e66d2f" target='_blank'>
              Abstract A069 The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P Lee, Jessica S Yoon, Andrew L. Hong
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/484b45daaba1111e0c4d69a75c275aee523f917f" target='_blank'>
              Multi-omics profiling of longitudinal samples reveals early genomic changes in follicular lymphoma
              </a>
            </td>
          <td>
            Baoyan Bai, J. F. Wise, Daniel Vodák, S. Nakken, Ankush Sharma, Y. Blaker, M. Brodtkorb, V. Hilden, G. Trøen, W. Ren, S. Lorenz, Michael S Lawrence, O. Myklebost, E. Kimby, Q. Pan-Hammarström, Chloé B Steen, L. Meza-Zepeda, Klaus Beiske, E. Smeland, E. Hovig, O. Lingjærde, H. Holte, J. Myklebust
          </td>
          <td>2024-08-27</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Abstract Gastrointestinal tumors, the second leading cause of human mortality, are characterized by their association with inflammation. Currently, progress in the early diagnosis and effective treatment of gastrointestinal tumors is limited. Recent whole‐genome analyses have underscored their profound heterogeneity and extensive genetic and epigenetic reprogramming. Epigenetic reprogramming pertains to dynamic and hereditable alterations in epigenetic patterns, devoid of concurrent modifications in the underlying DNA sequence. Common epigenetic modifications encompass DNA methylation, histone modifications, noncoding RNA, RNA modifications, and chromatin remodeling. These modifications possess the potential to invoke or suppress a multitude of genes associated with cancer, thereby governing the establishment of chromatin configurations characterized by diverse levels of accessibility. This intricate interplay assumes a pivotal and indispensable role in governing the commencement and advancement of gastrointestinal cancer. This article focuses on the impact of epigenetic reprogramming in the initiation and progression of gastric cancer, esophageal cancer, and colorectal cancer, as well as other uncommon gastrointestinal tumors. We elucidate the epigenetic landscape of gastrointestinal tumors, encompassing DNA methylation, histone modifications, chromatin remodeling, and their interrelationships. Besides, this review summarizes the potential diagnostic, therapeutic, and prognostic targets in epigenetic reprogramming, with the aim of assisting clinical treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa46852e2f3c4f502e85931e028a5b5bde3a9141" target='_blank'>
              Epigenetic reprogramming in gastrointestinal cancer: biology and translational perspectives
              </a>
            </td>
          <td>
            Yingjie Wang, Hongyu Liu, Mengsha Zhang, Jing Xu, Liuxian Zheng, Pengpeng Liu, Jingyao Chen, Hongyu Liu, Chong Chen
          </td>
          <td>2024-08-24</td>
          <td>MedComm</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Single-cell genomic analysis enables researchers to gain novel insights across diverse research areas, including developmental biology, tumor heterogeneity, and disease pathogenesis. Conducting single-cell genomic analysis using next-generation sequencing (NGS) methods has traditionally been challenging as the amount of genomic DNA present in a single cell is limited. Advancements in multiple displacement amplification (MDA) technologies allow the unbiased amplification of limited quantities of DNA under conditions that maintain its integrity. This method of amplification results in high yield and facilitates the generation of high-complexity NGS libraries that ensure the highest coverage to effectively allow variant calling. With the introduction of new sequencing platforms and chemistry, whole genome sequencing became a more cost-effective application, but enrichment of specific regions of interest further reduces the amount of required sequencing output and associated costs. There are two enrichment methods, polymerase chain reaction (PCR)-based and hybrid-capture-based methods. PCR-based methods are very flexible and highly effective but focus on specific loci, typically known to be associated with disease. Inherited diseases of unknown genetic origin require a more comprehensive approach to capture the genetic variation that is not yet associated with a specific disease. Hybrid capture enrichment methods require considerable amounts of DNA such that exome enrichment from single cells is only possible after preamplification of this limited material. This article describes the complete workflow from single cells and small quantities of DNA to exome-NGS libraries for Illumina sequencing instruments and includes the following protocols: © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Whole genome amplification from single cells or small amounts of gDNA Basic Protocol 2: NGS library generation of MDA-amplified material Basic Protocol 3: Exome enrichment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891307a5d438a6375e34496a127912e0af6f10c1" target='_blank'>
              Exome Sequencing Starting from Single Cells.
              </a>
            </td>
          <td>
            Ioanna Andreou, Markus Storbeck, Peter Hahn, Samuel J. Rulli, Eric Lader
          </td>
          <td>2024-09-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="PURPOSE
High-grade complex karyotype sarcomas are a heterogeneous group of tumors with a uniformly poor prognosis. Within complex karyotype sarcomas, there are innumerable genetic changes but identifying those that are clinically relevant has been challenging.


EXPERIMENTAL DESIGN
To address this, we utilized a pooled genetic screening approach, informed by TCGA data, to identify key drivers and modifiers of sarcoma development that were validated in vivo.


RESULTS
YAP1 and wildtype KRAS were validated as drivers and transformed human mesenchymal stem cells into two distinct sarcoma subtypes, undifferentiated pleomorphic sarcoma (UPS) and myxofibrosarcoma (MFS), respectively. A subset of tumors driven by CDK4 and PIK3CA reflected leiomyosarcoma (LMS) and osteosarcoma (OS) demonstrating the plasticity of this approach and the potential to investigate sarcoma subtype heterogeneity. All generated tumors histologically reflected human sarcomas and had increased aneuploidy as compared to simple karyotype sarcomas. Comparing differential gene expression of TCGA samples to model data identified increased oxidative phosphorylation signaling in YAP1 tumors. Treatment of a panel of soft tissue sarcomas with a combination of YAP1 and oxidative phosphorylation inhibitors led to significantly decreased viability.


CONCLUSIONS
Transcriptional co-analysis of TCGA patient samples to YAP1 and KRAS model tumors support that these sarcoma subtypes lie along a spectrum of disease and adds guidance for further transcriptome-based refinement of sarcoma subtyping. This approach can be used to begin to understand pathways and mechanisms driving human sarcoma development, the relationship between sarcoma subtypes and to identify and validate new therapeutic vulnerabilities for this aggressive and heterogeneous disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdaada75e0a2b742920f7cff889cd677616ee1d5" target='_blank'>
              Genetic Screen in a Pre-Clinical Model of Sarcoma Development Defines Drivers and Therapeutic Vulnerabilities.
              </a>
            </td>
          <td>
            Jack Freeland, Maria Muñoz, Edmond O'Donnell, J. Langerman, Morgan Darrow, Jessica Bergonio, J. Suarez-Navarro, Steven Thorpe, Robert Canter, R. L. Randall, Kathrin Plath, Kermit L Carraway, Owen N. Witte, Thomas G. Graeber, Janai R. Carr-Ascher
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e27208f944e21d4b5aca3b4a730bfecbb2ccacaf" target='_blank'>
              Single-cell RNA sequencing to map tumor heterogeneity in gastric carcinogenesis paving roads to individualized therapy
              </a>
            </td>
          <td>
            Jiao Xu, Bixin Yu, Fan Wang, Jin Yang
          </td>
          <td>2024-09-13</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary This research aims to explore the role of the TSGA10 protein in cancer development, specifically in how it might influence the growth and spread of cancer cells. Scientists are particularly interested in TSGA10 because it is found in both normal reproductive tissues and cancer cells, yet seems to slow down cancer progression. The key question is why cancer cells would produce a protein that could hinder their own survival. To investigate this, the authors propose several hypotheses about how TSGA10 might be involved in carcinogenesis. They will analyze both published and unpublished studies and data to understand how TSGA10 functions at different stages of cancer. By uncovering these mechanisms, this research could lead to new targeted therapies that use TSGA10 to combat cancer more effectively, offering fresh insights and potential breakthroughs in cancer treatment. Abstract Cancer-specific antigens have been a significant area of focus in cancer treatment since their discovery in the mid-twentieth century. Cancer germline antigens are a class of antigens specifically overexpressed in germline tissues and cancer cells. Among these, TSGA10 (testis-specific gene antigen 10) is of great interest because of its crucial impact on cancer progression. Early studies explored TSGA10 expression in a variety of cancer types. More recent studies revealed that TSGA10 can suppress tumor progression by blocking cancer cell metabolism, angiogenesis, and metastasis. An open question regarding the TSGA10 is why cancer cells must express a protein that prevents their progression. To answer this question, we conducted a comprehensive review to engage the TSGA10 in the context of the current understanding of “malignant transformation”. This review demonstrated that TSGA10 expression level in cancer cells depends on the cancer stage across malignant transformation. In addition, we evaluated how TSGA10 expression can prevent the “cancer hallmarks”. Given this information, TSGA10 can be of great interest in developing effective targeted anti-cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fbb75a98c933791c688ebdc1263dceb33c21d30" target='_blank'>
              Exploring TSGA10 Function: A Crosstalk or Controlling Mechanism in the Signaling Pathway of Carcinogenesis?
              </a>
            </td>
          <td>
            F. Taghizadeh-Hesary, Mobina Ghadyani, F. Kashanchi, Babak Behnam
          </td>
          <td>2024-08-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Gene expression and complex phenotypes are determined by the activity of cis-regulatory elements. However, an understanding of how extant genetic variants affect cis-regulatory activity remains limited. Here, we investigated the consequences of cis-regulatory diversity using single-cell genomics of >0.7 million nuclei across 172 maize inbreds. Our analyses pinpointed cis-regulatory elements distinct to domesticated maize and how transposons rewired the regulatory landscape. We found widespread chromatin accessibility variation associated with >4.6 million genetic variants with largely cell-type-specific effects. Variants in TEOSINTE BRANCHED1/CYCLOIDEA/PROLIFERATING CELL FACTOR binding sites were the most prevalent determinants of chromatin accessibility. Finally, integration of genetic variants associated with chromatin accessibility, organismal trait variation, and population differentiation revealed how local adaptation has rewired regulatory networks in unique cellular context to alter maize flowering phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f126351cde084205c5fe8f0c5b1a2468aa66fe35" target='_blank'>
              The genetic architecture of cell-type-specific cis-regulation
              </a>
            </td>
          <td>
            Alexandre P. Marand, Luguang Jiang, Fabio Gomez-Cano, Mark A. A. Minow, Xuan Zhang, J. P. Mendieta, Ziliang Luo, Sohyun Bang, Haidong Yan, Cullan Meyer, Luca Schlegel, F. Johannes, Robert J. Schmitz
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e0914eac9368db3a26273ef88f18ab51249d37f" target='_blank'>
              Advances in targeting cancer epigenetics using CRISPR-dCas9 technology: A comprehensive review and future prospects.
              </a>
            </td>
          <td>
            Jeevitha Rajanathadurai, Elumalai Perumal, Jospin Sindya
          </td>
          <td>2024-09-18</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56281f9e2c04f45376f07cc22699136fff5305aa" target='_blank'>
              Exploring the interaction between immune cells in the prostate cancer microenvironment combining weighted correlation gene network analysis and single-cell sequencing: An integrated bioinformatics analysis
              </a>
            </td>
          <td>
            Danial Hashemi Karoii, Sobhan Bavandi, Melika Djamali, Ali Shakeri Abroudi
          </td>
          <td>2024-09-30</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Somatic mutation phasing informs our understanding of cancer-related events, like driver mutations. We generated linked-read whole genome sequencing data for 23 samples across disease stages from 14 multiple myeloma (MM) patients and systematically assigned somatic mutations to haplotypes using linked-reads. Here, we report the reconstructed cancer haplotypes and phase blocks from several MM samples and show how phase block length can be extended by integrating samples from the same individual. We also uncover phasing information in genes frequently mutated in MM, including DIS3, HIST1H1E, KRAS, NRAS, and TP53, phasing 79.4% of 20,705 high-confidence somatic mutations. In some cases, this enabled us to interpret clonal evolution models at higher resolution using pairs of phased somatic mutations. For example, our analysis of one patient suggested that two NRAS hotspot mutations occurred on the same haplotype but were independent events in different subclones. Given sufficient tumor purity and data quality, our framework illustrates how haplotype-aware analysis of somatic mutations in cancer can be beneficial for some cancer cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2870d2d1518a03384eeca2d75b0b4975453b6c" target='_blank'>
              Somatic mutation phasing and haplotype extension using linked-reads in multiple myeloma
              </a>
            </td>
          <td>
            S. Foltz, Yize Li, Lijun Yao, Nadezhda V. Terekhanova, A. Weerasinghe, Qingsong Gao, Guanlan Dong, Moses Schindler, Song Cao, Hua Sun, R. Jayasinghe, Robert S Fulton, C. Fronick, Justin King, Daniel R. Kohnen, M. Fiala, Ken Chen, J. Dipersio, Ravi Vij, Li Ding
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d0fe672a884b6c86c84e083c9b2ab93ac24f65f" target='_blank'>
              Multiplex, single-cell CRISPRa screening for cell type specific regulatory elements
              </a>
            </td>
          <td>
            Florence M. Chardon, Troy A. McDiarmid, Nicholas F. Page, R. Daza, Beth K. Martin, Silvia Domcke, Samuel G Regalado, Jean-Benoît Lalanne, Diego Calderon, Xiaoyi Li, L. Starita, Stephan J. Sanders, N. Ahituv, J. Shendure
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>49</td>
        </tr>

        <tr id="CRISPR/Cas9 gene editing technology is a promising tool for correcting pathogenic variants for autologous cell therapies for Inborn Errors of Immunity (IEI). The present IEI correction strategies mainly focus on the knock-in of therapeutic cDNAs, or knockout of the disease-causing gene when feasible. These strategies address many single-gene defects but may disrupt gene expression and require significant optimization for each newly discovered IEI-causing gene, highlighting the need for complementary platforms that can precisely correct diverse pathogenic variants. Here, we present a safe and efficient T cell single nucleotide variant (SNV) correction pipeline based on homology-directed repair (HDR), suitable for diverse monogenic mutations. By using founder mutations of Deficiency of ADA2 (DADA2), Autoimmune polyendocrinopathy-candidiasis-ectodermal dystrophy (APECED) and Cartilage Hair Hypoplasia (CHH) as IEI models, we show that our pipeline can achieve up to 80% bi-allelic editing, with resultant functional correction of the disease phenotype in patient T cells. We do not find detectable pre-malignant off-target effects or karyotypic, transcriptomic or proteomic aberrations upon profiling patient T cells with GUIDE-seq, single cell RNA sequencing, PacBio based long-read whole genome sequencing, and high-throughput proteomics. This study demonstrates that HDR-based SNV editing is a safe and effective option for IEI T cell correction and that it could be developed to an autologous T cell therapy, as the presented protocol is scalable for a GMP-compatible workflow. This study is a step towards the development of gene correction platform that targets a broad number of monogenic mutations. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3514ea764ac815dfdaa1d9a93597841e9c3e8944" target='_blank'>
              T cell correction pipeline for Inborn Errors of Immunity
              </a>
            </td>
          <td>
            Katariina Mamia, Sólrún Kolbeinsdóttir, Zhuokun Li, Kornel Labun, Anna Komisarczuk, S. Keskitalo, Ganna Reint, Frida Høsøien Haugen, Britt Olaug Lindestad, T. Gjerdingen, Antti Tuhkala, Carolina Wieczorek Ervik, Pavel Kopcil, Nail Fatkhutdinov, Monika Szymańska, Eero Tölö, V. Glumoff, J. Saarela, T. Michelsen, Camilla Schalin-Jäntti, Johanna Olweus, Eira Leinonen, M. Varjosalo, Eivind Valen, Timo Hautala, Martin Enge, T. Martelius, Shiva Dahal-Koirala, E. Haapaniemi
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Patients with microsatellite instability-high (MSI-H) colorectal cancer (CRC) have high tumor mutation burden and tumor immunogenicity, exhibiting a higher response rate to immunotherapy and better survival. However, a portion of MSI-H CRC patients still experience adverse disease outcomes. We aimed to identify the tumor-autonomous regulators determining these heterogeneous clinical outcomes. The Cancer Genome Atlas (TCGA) dataset was used to identify regulators in MSI-H CRC patients with unfavorable outcomes. Stable CRC tumor clones expressing targeted regulators were established to evaluate migratory and stemness properties, immune cell vulnerability, and cell-in-cell (CIC) structure formation. RNA-sequencing (RNA-seq) was used to identify enriched biological pathways in stable CRC tumor clones. Clinicopathological characterization of formalin-fixed paraffin-embedded (FFPE) MSI-H CRC specimens was performed to explore the underlying mechanisms involved. We showed that cancer/testis antigen family 45 member A1 (CT45A1) expression was upregulated in MSI-H CRC patients with poor survival outcomes. CT45A1-expressing microsatellite stable (MSS) CRC cells showed enhanced migratory ability. However, CT45A1-expressing MSI-H CRC cells, but not MSS CRC cells, showed higher resistance to natural killer (NK) cell cytotoxicity and served as outer cells in homotypic CIC structures, preventing exogenous or therapeutic antibody access to inner CRC cells. Inactivating RHO-ROCK/MLCK-MLC2 signaling with small-molecule inhibitors or short-hairpin RNAs (shRNAs) targeting myosin light chain kinase (MYLK) abolished NK cell resistance and reduced the outer cell fate of CT45A1-expressing MSI-H CRC cells. In MSI-H CRC patients, CT45A1-positive tumors exhibited increased MLC2 phosphorylation, increased outer cell fate, and decreased survival. We demonstrated that CT45A1 potentiates the advanced progression of MSI-H CRC, and targeting MLC2 phosphorylation may enhance immunotherapy efficacy in CT45A1-positive MSI-H CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a08f29a07f3e7ccbababc06da74550a73f5f37c3" target='_blank'>
              CT45A1-mediated MLC2 (MYL9) phosphorylation promotes natural killer cell resistance and outer cell fate in a cell-in-cell structure, potentiating the progression of microsatellite instability-high colorectal cancer.
              </a>
            </td>
          <td>
            H. Teng, Hsiang-Yueh Huang, Chun-Chi Lin, Y. Twu, Wen-Hao Yang, Wen-Chun Lin, Hsin-Yi Lan, Yen-Yu Lin, Wei-Lun Hwang
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Introduction Biological processes such as aging, carcinogenesis, and immune response rely on the ability to maintain or rapidly expand cell populations. The fitness of the involved cells is constrained by their replicative potential, which is reflected in the cellular telomere content. Method We apply TelomereHunter to scATAC-seq data to determine telomere content on single-cell level, in a hematopoietic dataset consisting of 35,139 cells from samples of basal cell carcinoma patients receiving programmed cell death protein 1 (PD1) blockade treatment. Integrating information from open-chromatin-based signatures to assess cell identity, we characterize the heterogeneity of telomere length for individual cell populations pre- and post-immunotherapy. Results The extracted telomeric reads reflect the expected telomereome-to-genome fraction. Telomere content distributions differ significantly between cell populations, and the median telomere content in intermediate and terminal exhausted CD8+ T-cells pre-treatment is significantly correlated to response to PD-1 checkpoint blockade. Likewise, telomere content correlates with post-treatment cell proliferation in terminally exhausted and T follicular helper cells from responding patients. Conclusion Telomere content measurement from scATAC-seq data has a sufficiently high signal-to-noise ratio to detect significant differences between cell types. Furthermore, the telomere content of CD8+ exhausted T-cells pre-treatment is a putative biomarker for successful PD-1-based immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c0b43ae3c8ae5883f7e563f1131356d52bd5e2" target='_blank'>
              Quantifying immune cell telomere content at single-cell resolution in context of PD-1 checkpoint immunotherapy
              </a>
            </td>
          <td>
            Niklas L. Engel, Lea Herzel, Julie Surmely, Hanna Frieß, Malte Simon, B. Brors, Charles D. Imbusch, L. Feuerbach
          </td>
          <td>2024-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Small or overlapping genes are prevalent across all domains of life but are often overlooked for annotation and function because of challenges in their detection. The advent of high-density mutagenesis and data-mining studies suggest the existence of further coding potential within bacterial genomes. To overcome limitations in existing protein detection methods, we applied a genetics-based approach. We combined transposon insertion sequencing with a translation reporter to identify translated open reading frames throughout the genome at scale, independent of genome annotation. We applied our method to the well characterised species Escherichia coli and identified ∼200 putative novel protein coding sequences (CDS). These are mostly short CDSs (<50 amino acids) and in some cases highly conserved. We validate the expression of selected CDSs demonstrating the utility of this approach. Despite the extensive study of E. coli, this method revealed proteins that have not been previously described, including proteins that are conserved and neighbouring functionally important genes, suggesting significant functional roles of small proteins that are still overlooked. We present this as a complementary method to whole cell proteomics and ribosome trapping for condition-dependent identification of protein CDSs. We anticipate this technique will be a starting point for future high-throughput genetics investigations to determine the existence of unannotated genes in multiple bacterial species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12d66e376a1ad0bb088385e992deb063164354bb" target='_blank'>
              Identification of new genes on a whole genome scale using saturated reporter transposon mutagenesis
              </a>
            </td>
          <td>
            E. C. Goodall, Freya J. Hodges, Weine J. Kok, Budi Permana, Thom Cuddihy, Zihao Yang, Nicole Kahler, Kenneth Shires, Karthik Pullela, V. Torres, Jessica L. Rooke, A. Delhaye, Jean-François Collet, J. Bryant, Brian Forde, Matthew Hemm, Ian R. Henderson
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751f02ecfd3cada2a90e62224689eeb8eaf33b4" target='_blank'>
              Forward–reverse mutation cycles in cancer cell lines under chemical treatments
              </a>
            </td>
          <td>
            Si Chen, I. S. Tyagi, Wai-Kin Mat, Muhammad A. Khan, Weijian Fan, Zhenggang Wu, Taobo Hu, Can Yang, Hong Xue
          </td>
          <td>2024-09-27</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Heart failure (HF) is a prevalent cardiovascular disease with significant morbidity and mortality rates worldwide. Due to the intricate structure of the heart, diverse cell types, and the complex pathogenesis of HF, further in-depth investigation into the underlying mechanisms  is required. The elucidation of the heterogeneity of cardiomyocytes and the intercellular communication network is particularly important. Traditional high-throughput sequencing methods provide an average measure of gene expression, failing to capture the "heterogeneity" between cells and impacting the accuracy of gene function knowledge. In contrast, single-cell sequencing techniques allow for the amplification of the entire genome or transcriptome at the individual cell level, facilitating the examination of gene structure and expression with unparalleled precision. This approach offers valuable insights into disease mechanisms, enabling the identification of changes in cellular components and gene expressions during hypertrophy associated with HF. Moreover, it reveals distinct cell populations and their unique roles in the HF microenvironment, providing a comprehensive understanding of the cellular landscape that underpins HF pathogenesis. This review focuses on the insights provided by single-cell sequencing techniques into the mechanisms underlying HF and discusses the challenges encountered in current cardiovascular research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1186f68f600fee1e7602ca4a8c686869da13bf" target='_blank'>
              From Cell to Gene: Deciphering the Mechanism of Heart Failure With Single-Cell Sequencing.
              </a>
            </td>
          <td>
            Dan Zhang, Qiang Wen, Rui Zhang, K. Kou, Miao Lin, Shiyu Zhang, Jun Yang, Hangchuan Shi, Yan Yang, Xiaoqiu Tan, Shigang Yin, X. Ou
          </td>
          <td>2024-08-19</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genetically engineered CD8+ T cells are being explored for the treatment of various cancers. Analytical characterization represents a major challenge in the development of genetically engineered cell therapies, especially assessing the potential off-target editing and product heterogeneity. As conventional sequencing techniques only provide information at the bulk level, they are unable to detect off-target CRISPR translocation or editing events occurring in minor cell subpopulations. In this study, we report the analytical development of a single-cell multi-omics DNA and protein assay to characterize genetically engineered cell products for safety and genotoxicity assessment. We were able to quantify on-target edits, off-target events, and potential translocations at the targeting loci with per-cell granularity, providing important characterization data of the final cell product. Conclusion: A single-cell multi-omics approach provides the resolution required to understand the composition of cellular products and identify critical quality attributes (CQAs).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a403ecc7f675cd4ff2179f421e45809aa747f" target='_blank'>
              Assessing a single-cell multi-omic analytic platform to characterize ex vivo-engineered T-cell therapy products
              </a>
            </td>
          <td>
            Maryam Moshref, Jerry Hung-Hao Lo, Andrew McKay, Julien Camperi, Joseph Schroer, Norikiyo Ueno, Shu Wang, Saurabh Gulati, Somayeh Tarighat, Steffen Durinck, Ho Young Lee, Dayue Chen
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Bioengineering and Biotechnology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Precision oncology promises individually tailored drugs and clinical care for patients with cancer: That is, "the right drug, for the right patient, at the right dose, and at the right time." Although stratification of the risk for treatment resistance and toxicity is key to precision oncology, there are multiple ways in which such stratification can be achieved, for example, genetic, functional pathway based, among others. Moving toward precision oncology is sorely needed in the case of acute lymphoblastic leukemia (ALL) wherein adult patients display survival rates ranging from 30% to 70%. The present study reports on the pathway activity signature of adult B-ALL, with an eye to precision oncology. Transcriptome profiles from three different expression datasets, comprising 346 patients who were adolescents or adults with B-ALL, were harnessed to determine the activity of signaling pathways commonly disrupted in B-ALL. Pathway activity analyses revealed that Ph-like ALL closely resembles Ph-positive ALL. Although this was the case at the average pathway activity level, the pathway activity patterns in B-ALL differ from genetic subtypes. Importantly, clustering analysis revealed that five distinct clusters exist in B-ALL patients based on pathway activity, with each cluster displaying a unique pattern of pathway activation. Identifying pathway-based subtypes thus appears to be crucial, considering the inherent heterogeneity among patients with the same genetic subtype. In conclusion, a pathway-based stratification of the B-ALL could potentially allow for simultaneously targeting highly active pathways within each ALL subtype, and thus might open up new avenues of innovation for personalized/precision medicine in this cancer that continues to have poor prognosis in adult patients compared with the children.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72b528a52f9a1c25407f3a5bc1baf86dc4c7b9e9" target='_blank'>
              Expanding Beyond Genetic Subtypes in B-Cell Acute Lymphoblastic Leukemia: A Pathway-Based Stratification of Patients for Precision Oncology.
              </a>
            </td>
          <td>
            Ozlem Ulucan
          </td>
          <td>2024-08-19</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Bone sarcomas, a rare group of malignant neoplasms arising from mesenchymal cells, present significant diagnostic and therapeutic challenges due to their heterogeneity and overlapping characteristics with other tumours. Conventional diagnostic methods are often inadequate, necessitating advanced techniques to improve diagnostic accuracy and treatment outcomes. Objectives: This review aims to evaluate the role of genetic and molecular profiling in bone sarcomas, focusing on its potential to develop personalized treatment approaches and overcome therapeutic resistance. Methods: A systematic search was conducted across PubMed and Google Scholar to identify relevant studies published in the last ten years. Inclusion criteria encompassed randomized controlled trials, cohort studies, and systematic reviews investigating genetic and molecular profiling techniques such as RT-PCR, FISH, and NGS in bone sarcomas. Exclusion criteria included studies on soft tissue sarcomas, animal models, and non-peer-reviewed articles. Discussion: Genetic and molecular profiling has revolutionized the diagnosis and management of bone sarcomas by identifying key genetic alterations and enabling targeted therapies. While early clinical data are promising, larger randomized trials are needed to validate these findings. Resistance mechanisms, such as miRNA dysregulation, present ongoing challenges, highlighting the need for combination therapies to optimize treatment outcomes. Integrating genetic and molecular profiling into clinical practice offers the potential for personalized treatment regimens, improving survival rates and quality of life for patients with bone sarcomas. Further research is essential to fully realize the benefits of these advanced techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab81eb80f85c15cdbb59566542cc86af0d9cf4d8" target='_blank'>
              Genetic and molecular profiling in bone sarcomas: advancing personalized treatment approaches
              </a>
            </td>
          <td>
            Gandhi Dhruv, Aastha N. Raj, Saksham Sharma
          </td>
          <td>2024-08-19</td>
          <td>InterConf</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Small-cell lung cancer (SCLC) is the most lethal type of lung cancer. Paradoxically, this tumor displays an initial exquisite response to chemotherapy; however, at relapse, the tumor is highly resistant to subsequent available therapies. Here, we report that the expression of three prime repair exonuclease 1 (TREX1) is strongly induced in chemoresistant SCLCs. Assay for transposase-accessible chromatin using sequencing and chromatin immunoprecipitation sequencing revealed a significant increase in chromatin accessibility and transcriptional activity of TREX1 gene locus in chemoresistant SCLCs. Analyses of human SCLC tumors and patient-derived xenografts (PDX) also showed an increase in TREX1 expression in postchemotherapy samples. TREX1 depletion caused the activation of cyclic GMP-AMP synthase stimulator of interferon gene pathway due to cytoplasmic accumulation of damage-associated double-stranded DNA, inducing immunogenicity and enhancing the sensitivity of drug-resistant cells to chemotherapy. These findings suggest TREX1 upregulation may partially contribute to the survival of resistant cells, and its inhibition may represent a promising therapeutic strategy to enhance antitumor immunity and potentiate the efficacy of chemotherapy and/or immunotherapy in chemoresistant SCLCs. Significance: In this study, we show that targeting TREX1 induces an innate immune response and resensitizes SCLC cells to chemotherapy, representing a promising novel target for “immunologically” cold tumors, such as SCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/920774aa4db00913c78963d621212ab8e7a2e837" target='_blank'>
              Targeting TREX1 Induces Innate Immune Response in Drug-Resistant Small-Cell Lung Cancer
              </a>
            </td>
          <td>
            Takahiko Murayama, N. Mahadevan, Catherine B Meador, Elena V. Ivanova, Yuqiao Pan, Erik H. Knelson, Tetsuo Tani, Jun Nakayama, Xueying Ma, Tran C Thai, Yin P Hung, William Kim, Hideo Watanabe, Kathy Q Cai, Aaron N Hata, C. Paweletz, D. Barbie, Israel Cañadas
          </td>
          <td>2024-08-23</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Immunotherapy represents a groundbreaking and monumental achievement in the field of cancer therapy, marking a significant advancement in fighting against this devastating disease. Lung cancer has showed consistent clinical improvements in response to immunotherapy treatments, yet, it is undeniable that challenges such as limited response rates acquire resistance, and the unclear fundamental mechanisms were inevitable problems.


METHODS
The cellular composition was defined and distinguished through single-cell RNA sequencing (scRNA-seq) analysis of MPR (major pathologic response) and NMPR (non-major pathologic response) samples in GSE207422, including four primary MPR samples and eight primary NMPR samples.


RESULTS
We found obvious difference in CD8+ T cell population between MPR and NMPR samples, with high expression of TYMS, RRM2, and BIRC5 in NPMR samples. Meanwhile, the proportion of macrophages and tumor epithelial cells infiltration increased in the NMPR samples. We discovered biomarkers (ACTN4, ATF3, BRD2, CDKN1A, and CHMP4B) in epithelial cells which were potentially represented worse outcomes.


CONCLUSIONS
By exploring the difference of tumor microenvironment (TME) in samples with different corresponding degrees of neoadjuvant immunotherapy, this research introduces a number of novel biomarkers for predicting the response of treatment and a theoretical basis for overcoming immunotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6eb58f4e74cf2a065722c81b24475ac3132ac67a" target='_blank'>
              Single-cell RNA sequencing reveals microenvironmental infiltration in non-small cell lung cancer with different responses to immunotherapy.
              </a>
            </td>
          <td>
            Xinnan Hu, Yonghui Wu, Lixin Wang, Fujun Yang, Lingyun Ye, Xiaoxia Chen, Xiao Song, Ping Wei
          </td>
          <td>2024-09-01</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa2963e25bac9e3ab08f5a5b7df7ab88d90b9cc" target='_blank'>
              Multiomic single cell sequencing identifies stemlike nature of mixed phenotype acute leukemia
              </a>
            </td>
          <td>
            C. Peretz, Vanessa E. Kennedy, Anushka Walia, C. Delley, Andrew Koh, Elaine Tran, Iain C Clark, Corey E. Hayford, Chris D'Amato, Yi Xue, Kristina Fontanez, Aaron A May-Zhang, Trinity Smithers, Yigal Agam, Qian Wang, Haixin Dai, Ritu Roy, Aaron C Logan, Alexander E Perl, A. Abate, Adam Olshen, Catherine C Smith
          </td>
          <td>2024-09-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Mammalian genomes are scattered with transposable elements (TEs). TEs are epigenetically silenced to prevent harmful effects caused by either global activation leading to genome instability or insertional mutation disturbing gene transcription. However, whether the activation of a single element can contribute to pathological phenotypes without directly affecting gene expression is largely unknown. Here, we show that tissue-specific expression of a TE in the embryo leads to the production of viral-like particles (VLPs) which can affect organ formation. Failure to silence an LTR retrotransposon inserted upstream of the Fgf8 gene results in its co-expression with Fgf8 in the developing embryo. While local gene regulation is unaffected, the LTR retrotransposon participates in chromatin folding at the locus and adopts the expression of the regulatory domain it is located in. This drives the production of VLPs in the Fgf8-expressing cells of the developing limb, triggering apoptotic cell death at the time of digit outgrowth and resulting in a limb malformation resembling human ectrodactyly. This phenotype can be rescued by knock-out or knock-in of the retrotransposon causing mutations preventing its full retroviral cycle. Insertion of the same element at other developmental loci faithfully recapitulates expression according to the neighboring regulatory activity. Our findings provide a mechanism by which TE insertion is incorporated into the local genomic regulatory landscape and show how VLP production in post-implantation embryos can interfere with organ formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6530941b1bbda9df13ec9f7016e41dec0d4f10bf" target='_blank'>
              Enhancer adoption by an LTR retrotransposon generates viral-like particles causing developmental limb phenotypes
              </a>
            </td>
          <td>
            Juliane Glaser, Giulia Cova, B. Fauler, C. Prada-Medina, Virginie Stanislas, Mai H.Q. Phan, R. Schöpflin, Yasmin Aktas, Martin Franke, Guillaume Andrey, C. Paliou, Verena Laupert, W. Chan, L. Wittler, Thorsten Mielke, Stefan Mundlos
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="The complexity of cancer requires a comprehensive approach to understand its diverse manifestations and underlying mechanisms. Initially outlined by Hanahan and Weinberg in 2000 and updated in 2010, the hallmarks of cancer provide a conceptual basis for understanding inherent variability in cancer biology. Recent expansions have further elucidated additional hallmarks, including phenotypic plasticity and senescent cells. The International Agency for Research on Cancer (IARC) has identified the key characteristics of carcinogens (KCCs) to evaluate their carcinogenic potential. We analyzed chemicals of concern for environmental exposure that interact with specific receptors to induce genomic instability, epigenetic alterations, immune suppression, and receptor-mediated effects, thereby contributing to chronic inflammation. Despite their varying degrees of carcinogenicity, these chemicals have similar KCC profiles. Our analysis highlights the pivotal role of receptor binding in activating most other KCCs, underscoring their significance in cancer initiation. Although KCCs are associated with early molecular or cellular events, they do not encompass processes directly linked to full cellular malignancy. Thus, there is a need to integrate clear endpoints that anchor KCCs to the acquisition of a complete malignant phenotype into chemical testing. From the perspective of toxicology and cancer research, an all-encompassing strategy that incorporates both existing and novel KCCs and cancer hallmarks is essential to enable the targeted identification of prevalent carcinogens and facilitate zone-specific prevention strategies. To achieve this goal, collaboration between the KCC and cancer hallmarks communities becomes essential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a462cc73a8d245ed18a298dc440b46e953accd9" target='_blank'>
              Key characteristics of carcinogens meet hallmarks for prevention-cutting the Gordian knot
              </a>
            </td>
          <td>
            S. Senga, William H. Bisson, Annamaria Colacci
          </td>
          <td>2024-09-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This review offers an in-depth analysis of mitochondrial DNA (mtDNA) mutations in colorectal cancer stem cells (CSCs), emphasizing their significant impact on tumor dynamics and potential therapeutic strategies. CSCs are a special subpopulation due to their unique capabilities for self-renewal, differentiation, and resistance to conventional therapies. Given that CSCs significantly differ from other tumor cell subpopulations, particularly in their metabolic properties, and considering that colorectal cancer is a malignancy characterized by mitochondrial dysfunction, this review aims to put together existing data on the differences in the mitochondrial genome of CSCs compared to other colorectal tumor cell subpopulations. Additionally, the review seeks to explore the potential roles of these differences and to identify new ideas for therapeutic strategies. Key topics include the identification and properties of CSCs in colorectal cancer, the distinctive features of the mitochondrial genome, and the functional consequences of mtDNA mutations. The review hypothesizes that CSCs rely on well-functioning mitochondria for crucial aspects like energy production; yet, mtDNA mutations can lead to mitochondrial dysfunction, altering CSC characteristics and influencing cancer progression. The article discusses emerging therapeutic approaches targeting mitochondrial function in colorectal CSCs and highlights the need for advanced research, including the development of preclinical models and exploration of targeted therapies, to improve the understanding and treatment of colorectal cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebae925d5c7100b4c278e4586e85603f7ee40d86" target='_blank'>
              Mitochondrial DNA Mutations in Colorectal Cancer Stem Cells: Implications for Tumor Dynamics and Therapeutic Strategies.
              </a>
            </td>
          <td>
            N. Shakhpazyan, L. Mikhaleva, Arcady L Bedzhanyan, Z. Gioeva, A. Mikhalev, K. Midiber, A. Konyukova, D. Atiakshin, Igor Buchwalow, Markus Tiemann, Alexander N. Orekhov
          </td>
          <td>2024-09-11</td>
          <td>Current medicinal chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Barriers in access to clinical care among under-served populations in combination with a vast underrepresentation at a scientific research level perpetuate inequalities in cancer treatment. These disparities are further disseminated due to the lack of understanding underlying genomic differences contributing to cancer outcome among these patients. The contributions of germline and somatic variations in cancer often shape treatment regimens, and these variations are not well investigated in non-European populations. We have described a class of potential therapeutic targets (GEMINI genes) that reflect a combination of germline alterations with somatic loss of heterozygosity (LOH). GEMINI genes are essential genes with frequent germline polymorphisms, whose frequency varies by ancestry. Indeed, because genomes of African ancestry tend to be the most diverse, many GEMINI genes are the most polymorphic in this patient population. When normal cells are heterozygous, LOH leaves cancer cells reliant on the gene products encoded by a single allele, where as healthy cells retain both copies. LOH is a common genetic event, often affecting over 20% of the genome in certain cancers, making this approach widely useful. Targeting GEMINI vulnerabilities opens up possibilities for allele-specific inhibitors or degrader therapeutics that can be catered in an ancestry-dependent manner, to ameliorate inequities in cancer genomics research. One approach to readily target individual alleles is to utilize nucleic acid-based methods, as we’ve previously demonstrated using PRIM1. This therapeutic is expected to be the most specific when targeting polymorphisms that create the greatest differences between alleles. Here, we have prioritized a set of GEMINI targets based upon the extent of differences between alleles, and rates of heterozygosity and LOH across populations. Our initial analysis of TCGA data identified 5664 variants in 1278 essential genes that undergo LOH in cancer. We filtered GEMINI vulnerabilities by means of insertion-deletion (indel) difference of the alternate allele compared to the reference allele, followed by sorting based off maximum allele frequency by genetic ancestry. Here, we identify 31 variants across 23 genes enriched in African ancestry, 7 variants across 7 genes enriched in American ancestry, 12 variants across 12 genes enriched in East Asian ancestry, 4 variants enriched in 4 genes in South Asian ancestry, and 3 variants enriched in 3 genes in European ancestry. In addition, each of these ancestry-specific GEMINI genes exhibit a minimum 10% pan-cancer LOH rate, with the highest LOH frequencies observed in ovarian and kidney chromophobe cancers. As both these cancers exhibit disparities in mortality at a population level, targeting GEMINI genes as a therapeutic strategy may aid in lessening the disproportionate rate of outcome resultant of these cancers. Overall, we identify novel GEMINI genes to assess ancestry-specific genomic variations to serve as targets for precision treatment specific to underserved patient populations.
 Citation Format: Nicole Peiris, Catherine Hazard, Charlotte E. Farquhar, Andrei Loas, Bradley L. Pentelute, Rameen Beroukhim. Genetic ancestry specific loss of heterozygosity of essential genes as therapeutic vulnerabilities in cancer [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C099.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e337d55b2fcfc1ea91f26641b13ccce525e55ce5" target='_blank'>
              Abstract C099: Genetic ancestry specific loss of heterozygosity of essential genes as therapeutic vulnerabilities in cancer
              </a>
            </td>
          <td>
            Nicole Peiris, Catherine Hazard, C. Farquhar, A. Loas, Bradley L. Pentelute, R. Beroukhim
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Telomeres are a protective cap that prevents chromosome ends from being recognized as double-stranded breaks. In somatic cells, telomeres shorten with each cell division due to the end replication problem, which eventually leads to senescence, a checkpoint proposed to prevent uncontrolled cell growth. Tumor cells avoid telomere shortening by activating one of two telomere maintenance mechanisms (TMMs): telomerase reactivation or alternative lengthening of telomeres (ALT). TMMs are a viable target for cancer treatment as they are not active in normal, differentiated cells. Whereas there is a telomerase inhibitor currently undergoing clinical trials, there are no known ALT inhibitors in development, partially because the complex ALT pathway is still poorly understood. For cancers such as neuroblastoma and osteosarcoma, the ALT-positive status is associated with an aggressive phenotype and few therapeutic options. Thus, methods that characterize the key biological pathways driving ALT will provide important mechanistic insight. We have developed a first-in-class phenotypic high-throughput screen to identify small-molecule inhibitors of ALT. Our screen measures relative C-circle level, an ALT-specific biomarker, to detect changes in ALT activity induced by compound treatment. To investigate epigenetic mechanisms that contribute to ALT, we screened osteosarcoma and neuroblastoma cells against an epigenetic-targeted compound library. Hits included compounds that target chromatin-regulating proteins and DNA damage repair pathways. Overall, the high-throughput C-circle assay will help expand the repertoire of potential ALT-specific therapeutic targets and increase our understanding of ALT biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a74d0a829596a6c1ee56a1f05003c04780d1b641" target='_blank'>
              A First-in-Class High-Throughput Screen to Discover Modulators of the Alternative Lengthening of Telomeres (ALT) Pathway.
              </a>
            </td>
          <td>
            Merrill M. Froney, Christian R. Cook, Alyssa M. Cadiz, Katherine A. Flinter, Sara T. Ledeboer, Bianca Chan, Lauren E. Burris, Brian P. Hardy, Kenneth H. Pearce, Alexis C. Wardell, B. Golitz, M. Jarstfer, Samantha G. Pattenden
          </td>
          <td>2024-08-13</td>
          <td>ACS pharmacology & translational science</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Fragile X Syndrome (FXS) is a genetic neurodevelopmental disorder closely associated with intellectual disability and autism spectrum disorders. The core of the disease lies in the abnormal expansion of the CGG trinucleotide repeat sequence at the 5′end of the FMR1 gene. When the repetition exceeds 200 times, it causes the silencing of the FMR1 gene, leading to the absence of the encoded Fragile X mental retardation protein 1 (FMRP). Although the detailed mechanism by which the CGG repeat expansion triggers gene silencing is yet to be fully elucidated, it is known that this process does not alter the promoter region or the coding sequence of the FMR1 gene. This discovery provides a scientific basis for the potential reversal of FMR1 gene silencing through interventional approaches, thereby improving the symptoms of FXS. Epigenetics, a mechanism of genetic regulation that does not depend on changes in the DNA sequence, has become a new focus in FXS research by modulating gene expression in a reversible manner. The latest progress in molecular genetics has revealed that epigenetics plays a key role in the pathogenesis and pathophysiological processes of FXS. This article compiles the existing research findings on the role of epigenetics in Fragile X Syndrome (FXS) with the aim of deepening the understanding of the pathogenesis of FXS to identify potential targets for new therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64f2a0a0d20e60f40e1927dda3be0710c0759d2d" target='_blank'>
              Epigenetic insights into Fragile X Syndrome
              </a>
            </td>
          <td>
            Liangqun Xie, Huiying Li, MengLiang Xiao, Ningjing Chen, Xiaoxiao Zang, Yingying Liu, Hong Ye, Chaogang Tang
          </td>
          <td>2024-08-16</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background and aims: Colorectal carcinomas (CRCs) in patients with Lynch syndrome (LS) exhibit heightened immunogenicity due to mismatch repair deficiency (MMR-d), often resulting in favorable responses to T cell immune checkpoint therapies. Recent studies indicate that the phenotype and genotype of LS-associated CRCs vary depending on the specific MMR gene mutated. Here, we investigated whether the immune profiles of LS-associated CRCs differ based on the MMR gene defects. Methods: Tissue material from 18 MLH1-, 16 MSH2-, 40 MSH6-, and 23 PMS2-mutated CRCs and 35 sporadic MMR-d CRCs were included in the study. Imaging mass cytometry (IMC) analysis, along with targeted multiplex immunofluorescence imaging (mIF) and immunohistochemistry, were applied to examine the tumor immune microenvironment, including Human Leukocyte Antigen (HLA) class I and programmed death-ligand 1 (PD-L1) expression. Results: Unsupervised hierarchical clustering of cell phenotypes identified by IMC, followed by mIF validation, revealed comparable lymphoid and myeloid cell infiltration levels across CRCs from all MMR groups. Infiltrating T cell levels negatively correlated with the number of mutations at coding microsatellite sequences, particularly in MLH1-mutated CRCs. HLA class I defects were observed in 76% of all CRCs. These defects were more frequently accompanied by {beta}2M defects in hereditary MMR-d CRCs (67%) compared to sporadic MMR-d CRCs (37%), and did not associate with the number of {gamma}{delta} T cells, which were present in CRCs from all MMR groups. PD-L1 expression in tumor cells was only detected in 8% of all CRCs. Conclusion: Our findings illustrate that, from an immunological perspective, there is no evidence of differing immunogenic features across MMR defects. This is important to consider when developing preventive vaccine strategies and evaluating immunotherapy for LS patients and those with MMR-d CRCs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb0cdd08de3e8c7b38b9b131001cb730c9f4dade" target='_blank'>
              Immunological profiles in Lynch syndrome colorectal cancers are not specific to mismatch repair gene defects
              </a>
            </td>
          <td>
            Noah C. Helderman, M. Ijsselsteijn, Madalina Cabuta, Manon van der Ploeg, T. Wezel, A. Ahadova, Matthias Kloor, H. Morreau, Maartje Nielsen, N. F. D. Miranda
          </td>
          <td>2024-08-28</td>
          <td>None</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf9c919681d9aa45181ae81406b378eac4e35cf6" target='_blank'>
              Deregulation of mitochondrial gene expression in cancer: mechanisms and therapeutic opportunities.
              </a>
            </td>
          <td>
            Mariah J. Berner, Steven W Wall, Gloria V. Echeverria
          </td>
          <td>2024-08-14</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Global healthcare systems have a great challenge in the form of inherited hematological diseases, which necessitates the development of new remedial strategies. By precisely targeting inherited abnormalities, Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR)-CRISPR-associated protein 9 (Cas9)-mediated ex vivo gene editing has surfaced as a promising approach to treat these diseases. This review offers a comprehensive examination of the advancements, challenges, and clinical eventuality of CRISPR-Cas9-intermediated ex vivo gene editing for inherited hematological diseases. With advancements in CRISPR-Cas9 technology, the eventuality to correct inheritable mutations responsible for inherited hematological diseases is within reach. However, challenges such as off-target effects, immune responses, and ethical considerations need to be addressed for the safe and effective perpetration of this technology. A promising understanding of how CRISPR-Cas9-intermediated gene editing functions in practice is handed by ongoing clinical studies, giving rise to the possibility of advanced remedial approaches and bettered patient issues. By addressing these complications in a human-readable format, this review attempts to provide greater understanding and appreciation for the eventuality of CRISPR-Cas9 technology in revolutionizing the treatment landscape for these challenging disorders and contribute to the ongoing discussion in the field and facilitate further exploration towards effective treatments for these challenging disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce4d496cce2c3ddc8c2b97b6be801d229f168c15" target='_blank'>
              CRISPR-CAS9-MEDIATED EX VIVO GENE EDITING FOR INHERITED HEMATOLOGICAL DISORDERS: ADVANCEMENTS, CHALLENGES, AND CLINICAL POTENTIAL
              </a>
            </td>
          <td>
            Atasi Ranjan Panda, Shreeya Das
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Pharmacy and Pharmaceutical Sciences</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Simple Summary This study aimed to determine the gene expression profiles associated with chemotherapeutic responses in conventional osteosarcomas (COS) within South Africa. We observed a significant downregulation in the ATP binding cassette subfamily C members (ABCC3 and ABCB1-p-glycoprotein), excision repair cross-complimenting group 1 (ERCC 1), replication factor C subunit 1 (RFC1), and tumour protein 53 (p53) genes in the COS tumours compared to the healthy donors. Furthermore, an upregulated ERCC1 gene expression level predicted a poor chemotherapeutic response. Additionally, the predictors of COS chemotherapeutic response comprised age, chondroblastic and osteoblastic histological subtypes, and ABCC3, ERCC1, and RFC1 gene expression. Abstract Background: We determined the predictive gene expression profiles associated with chemo-response in conventional osteosarcomas (COS) within South Africa. Materials and methods: In 28 patients, we performed an RNA extraction, cDNA synthesis, and quantitative analysis using the RT-PCR 2−∆∆CT method to determine the fold change in gene expression alongside GAPDH (housekeeping gene). Results: We observed a significant downregulation in the mRNA expression profiles of ABCB1-p-glycoprotein (p = 0.0007), ABCC3 (p = 0.002), ERCC1 (p = 0.007), p-53 (p = 0.007), and RFC1 (p = 0.003) in the COS patients compared to the healthy donors. Furthermore, ABCB1-p-glycoprotein (p = 0.008) and ABCC3 (p = 0.020) exhibited a significant downregulation in the COS tumour tissues when compared to the healthy donors. In our univariate logistic regression, the predictors of chemotherapeutic response comprised ERCC1 [restricted cubic spline (RCS) knot: OR −0.27; CI −0.504 to −0.032; p = 0.036]; osteoblastic subtype [OR −0.36; CI −0.652 to −0.092; p = 0.026); fibroblastic subtype [OR 0.91; CI 0.569 to 1.248; p < 0.001]; and mixed subtype [OR 0.53; CI 0.232 to 0.032; p = 0.032]. In our multivariable logistic regression, the significant predictors of chemotherapeutic response comprised age [RCS knot: OR −2.5; CI −3.616 to −1.378; p = 0.022]; ABCC3 [RCS knot: OR 0.67; CI 0.407 to 0.936, p = 0.016]; ERCC1 [RCS knot: OR 0.57; CI 0.235 to 0.901; p = 0.044]; RFC1 [RCS knot: OR −1.04; CI −1.592 to −0.487; p = 0.035]; chondroblastic subtype [OR −0.83; CI −1.106 to −0.520; p = 0.012]; and osteoblastic subtype [OR −1.28; CI −1.664 to −0.901; p = 0.007]. Conclusions: In this South African cohort, we observed the unique gene expression profiles of osteosarcoma tumourigenesis and chemotherapeutic responses. These may serve as prognostication and therapeutic targets. Larger-scale research is needed on the African continent.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66cda9efe1f101a9fdd82c5518590f36a3653f2a" target='_blank'>
              Unique Gene Expression Profiles within South Africa Are Associated with Varied Chemotherapeutic Responses in Conventional Osteosarcoma
              </a>
            </td>
          <td>
            P. G. Mthethwa, Thilona Arumugam, V. Ramsuran, Anmol Gokul, R. Rodseth, Leonard Marais
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Introduction: High-risk neuroblastoma (NBL) is a leading cause of pediatric cancer death and is characterized by substantial intratumoral heterogeneity leading to therapeutic resistance. Despite the significant insights from recent single-cell sequencing, the evolution of the tumor microenvironment under therapy remains obscure. We longitudinally profiled 22 high-risk NBL patients before and after induction chemotherapy through single-nucleus RNA and ATAC sequencing. We identified profound shifts in tumor and immune cell subpopulations. In particular, macrophages significantly expanded towards pro-angiogenic, immunosuppressive, and metabolic phenotypes. Through a prediction of paracrine signaling using our single-cell sequencing data, we predicted a role of the heparin-binding epidermal growth factor (HB-EGF)/Erb receptor tyrosine kinase 4 (ERBB4) signaling pathway between VCAN+ macrophages and neuroblasts. In this study, we sought to characterize the role of this signaling pathway in therapy response in high-risk neuroblastoma. Methods: We examined the interaction between malignant neuroblasts and macrophages using transwell co-culture assay. Macrophages were differentiated from the THP1 monocytes. Neuroblasts were cultured using NB1643, CHLA15, CHLA20, COG-N-297, and COG-N-590 neuroblastoma cell lines. To determine the source and activity of the HB-EGF/ERBB4/ERK signaling pathway, protein levels were determined using enzyme-linked immunosorbent assay and Western blot using co-culture supernatant and neuroblast and macrophage cell lysates. Pathway activity was modulated using CRM197 as the HB-EGF inhibitor and afatinib as the ERBB tyrosine kinase inhibitor. We examined the role of macrophage-derived HB-EGF on cell proliferation and migration by colony formation and transwell migration assays, respectively. Furthermore, we assessed the adrenergic (PHOX2B, GATA3, ALK, PHOX2A) and mesenchymal (YAP1 and PRRX1) marker expressions by real-time qPCR and Western blot. Results: We found that expression of HB-EGF in macrophages was significantly induced when co-cultured with multiple neuroblast cell lines. Binding of HB-EGF to ERBB4 in neuroblasts led to the activation of the kinase pathway and increased proliferation and migration of neuroblast cells, which can be inhibited by either HB-EGF or tyrosine kinase inhibitors. We examined adrenergic and mesenchymal marker expression in neuroblasts after prolonged co-culture with macrophages. While ALK and PHOX2B expressions were reduced, YAP1 and PRRX1 expressions increased in post-therapy CHLA20 cells but not in the paired diagnostic CHLA15 or NB1643 cells. Inhibition of the HB-EGF/ERBB pathway partially restored PHOX2B expression and reduced YAP1 and PRRX1 expression in macrophage co-cultured CHLA20 cells. Conclusion: We established the HB-EGF/ERBB4 axis between macrophage and neoplastic neuroblast, inducing ERK signaling, tumor cell proliferation, migration, and adrenergic-to-mesenchymal transition. These findings collectively reveal a novel extrinsic mechanism of therapy resistance in high-risk NBL.
 Citation Format: Rumeysa Biyik-Sit, Wenbao Yu, Chia-Hui Chen, Anusha Thadi, Minxing Pang, Liron D. Grossmann, Tasleema Patel, Daniel Martinez, Lea F. Surrey, Michael D. Hogarty, Kathrin Bernt, Nancy Zhang, John M. Maris, Kai Tan. VCAN+ macrophages promote neuroblastoma cell growth, migration, and adrenergic to mesenchymal transition via the HB-EGF/ERBB signaling axis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf38a2142d9aa00bb31bb908d3c2ec97aaec27dc" target='_blank'>
              Abstract B062: VCAN+ macrophages promote neuroblastoma cell growth, migration, and adrenergic to mesenchymal transition via the HB-EGF/ERBB signaling axis
              </a>
            </td>
          <td>
            Rumeysa Biyik‐Sit, Wenbao Yu, Chia-Hui Chen, Anusha Thadi, Minxing Pang, Liron D. Grossmann, Tasleema Patel, Daniel Martinez, Lea Surrey, M. Hogarty, Kathrin M Bernt, Nancy Zhang, John M. Maris, Kai Tan
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Recent studies demonstrate growing roles for genetic mosaicism in neurodevelopmental and neuropsychiatric disorders, with the paradigm being drug-resistant pediatric focal epilepsy related to activating somatic variants in the PI3K-mTOR pathway. While identifying the genotype-associated changes at the single-cell level is fundamental to understanding disease pathophysiology, this remains technically challenging in human tissue samples with existing methods. Here, we performed single-nucleus RNA-sequencing (snRNA-seq) of 20 focal cortical dysplasia (FCD) samples removed surgically for treatment of drug-resistant epilepsy, and 10 non-FCD controls, and we developed a new approach, Genotyping Of Transcriptomes Enhanced with Nanopore sequencing (GO-TEN), that combines targeted complementary (c)DNA sequencing with snRNA-seq to perform concurrent single-nucleus genotyping and transcriptional analysis. We find that mosaic pathogenic variants in FCD do not produce a detectable novel cell identity, but instead we observe conserved cell types present both in FCD cases and non-FCD control specimens. Similarly, GO-TEN analysis shows that most pathogenic variant-carrying cells have well-differentiated neuronal or glial identities and are enriched for layer II-III excitatory neurons. We identify cell-intrinsic disruption of glutamate and GABA-A signaling pathways in variant-carrying neurons and altered intercellular signaling, making potential mechanisms for epileptogenesis in FCD. In summary, by addressing genotype-specific changes in mosaic epilepsy-associated lesions, our study highlights new potential disease mechanisms and therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc181f7065f3cb1a379b3d4368b2d029b87b1b3b" target='_blank'>
              Cell-type-informed genotyping of mosaic focal epilepsies reveals cell-autonomous and non-cell-autonomous disease-associated transcriptional programs
              </a>
            </td>
          <td>
            S. Bizzotto, E. Stronge, Maya Talukdar, Qiwen Hu, Zinan Zhou, August Yue Huang, B. Chhouk, Alissa M. D’Gama, Edward Yang, T. E. Green, David C. Reutens, Saul A. Mullen, Michael S. Hildebrand, Russell J Buono, A. Poduri, S. Khoshkhoo, Christopher A. Walsh
          </td>
          <td>2024-10-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="To comprehensively unravel the temporal relationship between initiating and driver events and its impact on clinical outcomes, we analyzed 421 whole-genome sequencing profiles from 382 patients. Using clock-like mutational signatures, we estimated a time lag of 2-4 decades between initiating events and diagnosis. In patients with hyperdiploidy, we demonstrate that trisomies of odd-numbered chromosomes can be acquired simultaneously with other chromosomal gains, such as 1q gain. We provide evidence that hyperdiploidy is acquired after canonical IGH translocation when both events are present. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (>1 extra-copies), but faring worse than those with late 1q gain. This underscores that the prognostic impact of 1q gain/amp depends more on the timing of acquisition than on the number of extra copies gained. Overall, this study contributes to a better understanding of the life history of MM and may have prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f98e16f03f4610ad7609f6b41fac703153b2f8eb" target='_blank'>
              Temporal Genomic Dynamics Shape Clinical Trajectory in Multiple Myeloma
              </a>
            </td>
          <td>
            Francesco Maura, Marcella Kaddoura, A. Poos, L. Baughn, B. Ziccheddu, Marc-Andrea Bärtsch, Anthony Cirrincione, K. Maclachlan, M. Chojnacka, B. Diamond, M. Papadimitriou, P. Blaney, L. John, Philipp Reichert, S. Huhn, Dylan Gagler, Yanming Zhang, Ahmet Dogan, A. Lesokhin, Faith Davies, Hartmut Goldschmidt, Roland Fenk, Katja C. Weisel, E. Mai, N. Korde, Gareth Morgan, S. Rajkumar, Shaji Kumar, S. Usmani, O. Landgren, Marc-Steffen Raab, Niels Weinhold
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Synovial Sarcoma (SySa) is an aggressive soft tissue sarcoma that accounts for 5 – 10% of all soft tissue sarcomas. Current treatment involves radiation and radical surgery including limb amputation, highlighting the urgent need to develop targeted therapies. We reasoned that transcriptional rewiring by the fusion protein SS18-SSX, the sole oncogenic driver in SySa, creates specific vulnerabilities that can be exploited for treatment. To uncover genes that are selectively essential for SySa, we mined The Cancer Dependency Map (DepMap) data to identify genes that specifically impact the fitness of SySa compared to other tumor cell lines. Targeted CRISPR library screening of SySa-selective candidates revealed that the small ubiquitin-like modifier 2 (SUMO2) was one of the strongest dependencies both in vitro as well as in vivo. TAK-981, a clinical-stage small molecule SUMO2 inhibitor potently inhibited growth and colony-forming ability. Strikingly, transcriptomic studies showed that pharmacological SUMO2 inhibition with TAK-981 treatment elicited a profound reversal of a gene expression program orchestrated by SS18-SSX fusions. Of note, genetic or pharmacological SUMO2 inhibition reduced global and chromatin levels of the SS18-SSX fusion protein with a concomitant reduction in histone 2A lysine 119 ubiquitination (H2AK119ub), an epigenetic mark that plays an important role in SySa pathogenesis. Taken together, our studies identify SUMO2 as a novel, selective vulnerability in SySa. Since SUMO2 inhibitors are currently in Phase 1/2 clinical trials for other cancers, our findings present a novel avenue for targeted treatment of synovial sarcoma. SIGNIFICANCE Our study identifies SUMO2 as a selective dependency in synovial sarcoma. We demonstrate that the SUMO2/3 inhibitor TAK-981 impairs sarcomagenesis and reverses the SS18-SSX fusion-driven oncotranscriptome. Our study indicates that SUMO2 inhibition may be an attractive therapeutic option in synovial sarcoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b1863d8b2a5acaf7a4dc2a730005a178b4452ec" target='_blank'>
              SUMO2 Inhibition Reverses Aberrant Epigenetic Rewiring Driven by Synovial Sarcoma Fusion Oncoproteins and Impairs Sarcomagenesis
              </a>
            </td>
          <td>
            Rema Iyer, Anagha Deshpande, Aditi Pedgaonkar, Pramod Akula Bala, Taehee Kim, Gerard L. Brien, D. Finlay, K. Vuori, A. Soragni, Rabi Murad, Aniruddha J. Deshpande
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="During cancer development, the interplay between the nucleus and the cell cycle leads to a state of genomic instability, often accompanied by observable morphological aberrations. These aberrations can be controlled by tumor cells to evade cell death, either by preventing or eliminating genomic instability. In epithelial ovarian cancer (EOC), overexpression of the multifunctional protein claudin-4 is a key contributor to therapy resistance through mechanisms associated with genomic instability. However, the molecular mechanisms underlying claudin-4 overexpression in EOC remain poorly understood. Here, we altered claudin-4 expression and employed a unique claudin-4 targeting peptide (CMP) to manipulate the function of claudin-4. We found that claudin-4 facilitates genome maintenance by linking the nuclear envelope and cytoskeleton dynamics with cell cycle progression. Claudin-4 caused nuclei constriction by excluding lamin B1 and promoting perinuclear F-actin accumulation, associated with remodeling nuclear architecture, thus altering nuclear envelope dynamics. Consequently, cell cycle modifications due to claudin-4 overexpression resulted in fewer cells entering the S-phase and reduced genomic instability. Importantly, disrupting biological interactions of claudin-4 using CMP and forskolin altered oxidative stress cellular response and increased the efficacy of PARP inhibitor treatment. Our data indicate that claudin-4 protects tumor genome integrity by remodeling the crosstalk between the nuclei and the cell cycle, leading to resistance to genomic instability formation and the effects of genomic instability-inducing agents. Graphical abstract Claudin-4 plays a crucial role in remodeling the cytoskeleton, particularly influencing nuclear architecture. This remodeling appears to act as a regulatory mechanism, limiting the progression of ovarian cancer cells into the S-phase and correlating with reduced genomic instability in ovarian tumors. Moreover, since olaparib treatment triggered a cellular oxidative response, it is likely that this claudin-4-mediated remodeling contributes to resistance against the effects of the PARP inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1779163fc52ccc64570aa04b3ca709410f5487db" target='_blank'>
              Claudin-4 remodeling of nucleus-cell cycle crosstalk maintains ovarian tumor genome stability and drives resistance to genomic instability-inducing agents
              </a>
            </td>
          <td>
            Fabian R. Villagomez, Julie Lang, Daniel Nunez-Avellaneda, K. Behbakht, Hannah L. Dimmick, Patricia Webb, K. Nephew, Margaret Neville, Elizabeth R. Woodruff, Benjamin G. Bitler
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Carcinogenesis is an evolutionary process, and mutations can fix the selected phenotypes in selective microenvironments. Both normal and neoplastic cells are robust to the mutational stressors in the microenvironment to the extent that secure their fitness. To test the robustness of genes under a range of mutagens, we developed a sequential mutation simulator, Sinabro, to simulate single base substitution under a given mutational process. Then, we developed a pipeline to measure the robustness of genes and cells under those mutagenesis processes. We discovered significant human genome robustness to the APOBEC mutational signature SBS2, which is associated with viral defense mechanisms and is implicated in cancer. Robustness evaluations across over 70,000 sequences against 41 signatures showed higher resilience under signatures predominantly causing C-to-T (G-to-A) mutations. Principal component analysis indicates the GC content at the codon’s wobble position significantly influences robustness, with increased resilience noted under transition mutations compared to transversions. Then, we tested our results in bats at extremes of the lifespan-to-mass relationship and found the long-lived bat is more robust to APOBEC than the short-lived one. By revealing robustness to APOBEC ranked highest in human (and bats with much more than number of APOBEC) genome, this work bolsters the key potential role of APOBECs in aging and cancer, as well as evolved countermeasures to this innate mutagenic process. It also provides the baseline of the human and bat genome robustness under mutational processes associated with aging and cancer. Highlights Sinabro, the sequential mutation simulator, facilitates measuring the robustness of human protein-coding sequences under all COSMIC mutational signatures. Robustness under APOBEC mutational signatures showed the largest mean and standard deviation in the human genome. Robustness to mutational signatures analysis reveals the role of APOBECs is complementary to cancer in the evolvability of cancer cells in later stages. Principal component analysis indicates that the GC content at the codon’s wobble position significantly influences robustness. A long-lived bat (Myotis myotis) has higher robustness to APOBECs than a short-lived one (Molossus molossus) than humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/011df1029d192af53d074e293c75b5f6a3b83940" target='_blank'>
              Human and bats genome robustness under COSMIC mutational signatures
              </a>
            </td>
          <td>
            Joon-Hyun Song, Ying Zeng, Liliana M. Dávalos, Thomas MacCarthy, Mani Larijani, Mehdi Damaghi
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Background While immune checkpoint inhibitors (ICIs) are adopted as standard therapy in non-small cell lung cancer (NSCLC) patients, factors that influence variable prognosis still remain elusive. Therefore, a deeper understanding is needed of how germline variants regulate the transcriptomes of circulating immune cells in metastasis, and ultimately influence immunotherapy outcomes. Methods We collected peripheral blood mononuclear cells (PBMCs) from 73 ICI-treated NSCLC patients, conducted single-cell RNA sequencing, and called germline variants via SNP microarray. Determination of expression quantitative trait loci (eQTL) allows elucidating genetic interactions between germline variants and gene expression. Utilizing aggregation-based eQTL mapping and network analysis across eight blood cell types, we sought cell-type-specific and ICI-prognosis-dependent gene regulatory signatures. Results Our sc-eQTL analysis identified 3,616 blood- and 702 lung-cancer-specific eGenes across eight major clusters and treatment conditions, highlighting involvement of immune-related pathways. Network analysis revealed TBX21-EOMES regulons activity in CD8+ T cells and the enrichment of eQTLs in higher-centrality genes as predictive factors of ICI response. Conclusions Our findings suggest that in the circulating immune cells of NSCLC patients, transcriptomic regulation differs in a cell type- and treatment-specific manner. They further highlight the role of eQTL loci as broad controllers of ICI-prognosis-predicting gene networks. The predictive networks and identification of eQTL contributions can lead to deeper understanding and personalized ICI therapy response prediction based on germline variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70aa2fedf9166d7283b9f0e6d961d358ca399349" target='_blank'>
              Systemic CD8+ T cell effector signature predicts prognosis of lung cancer immunotherapy
              </a>
            </td>
          <td>
            Hyungtai Sim, Geun-Ho Park, Woong-Yang Park, Se-Hoon Lee, Murim Choi
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Since their discovery, CRISPR/Cas9 systems have been repurposed for programmable targeted genomic editing. This has led to unprecedented advancement of gene editing for therapeutic benefit. Initial uses of CRISPR/Cas9 were focused on gene disruption via DNA cleavage, but significant engineering led to systems for single base editing as well as insertion, deletion and manipulation of short stretches of genomic sequences using nicking Cas9 and RT-based methods. These technologies allowed safer and more precise editing but were limited to small corrections and showed significantly reduced efficiencies in nondividing cells, presenting difficulty for translation to in vivo therapies. To find an alternate editing strategy that could address these shortcomings, we revisited the mechanism of DNA nicking by nCas9. nCas9 nicking creates a free 5’ phosphate group and a 3’ hydroxyl group on the complementary strand of the target sequence. Under ordinary conditions in the cell these ends are re-joined by endogenously expressed ligases to repair DNA back to wild-type. If, however, a DNA fragment containing the desired edit were present, ligation of the nicked genomic DNA with the delivered fragment could result in gene editing. We demonstrate that optimization of each component and introduction of a chemically modified high affinity splinting DNA allows a variety of ligase-based edits, including longer edits not efficient with RT-based systems, at high efficiencies and fidelities that minimize genomic byproducts in both dividing and nondividing cells as well as in vivo in adult mice. Here we present the first therapeutically relevant ligation-based programmable gene editing technology, L-PGI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05db8d5146707d667516d894067c8635a13e5573" target='_blank'>
              Ligase-mediated programmable genomic integration (L-PGI): an efficient site-specific gene editing system that overcomes the limitations of reverse transcriptase-based editing systems
              </a>
            </td>
          <td>
            Angela Xinyi Nan, Michael Chickering, Christopher Bartolome, Neeta Shadija, Dan Li, Brett Estes, Jessica Von Stetina, Wei Li, Jason Andresen, Jesse C Cochrane, Chen Bai, Jason Gatlin, Jie Wang, Davood Norouzi, Sandeep Kumar, Maike Thamsen Dunyak, Leonard Chavez, Anmol Seth, Shakked Halperin, Jonathan D. Finn, Jenny Xie
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d35e0bab7dec79077bc3332fba4617308d5113" target='_blank'>
              Comprehensive evaluation of immunological attributes and immunotherapy responses of positive T cell function regulators in colorectal cancer
              </a>
            </td>
          <td>
            Ke Pu, Jingyuan Gao, Yang Feng, Jian Hu, Shunli Tang, Guodong Yang, Chuan Xu
          </td>
          <td>2024-10-01</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Preclinical models such as cell lines and mice are the backbone of drug development and experimental-mechanistic oncology. However, we currently lack a detailed understanding of the direct clinical relevance of data collected in most preclinical models, hampering the development of new treatments. Despite this, few formal approaches have been proposed to determine how the various preclinical models represent/resemble primary patient tumors. Here, we present the first comprehensive single-cell RNA-seq analysis of neuroblastoma across an extensive cohort of patient tumors and a variety of preclinical model systems (n = 126 total samples assembled – the largest cohort of its kind). By developing an unsupervised machine learning method, which we term “automatic consensus nonnegative matrix factorization” (acNMF), we have integrated and contrasted the transcriptional landscapes of patient tumors with those of cell lines, patient-derived xenografts (PDX), and genetic mouse models (GEMM). We discovered that the dominant adrenergic gene expression programs commonly found in neuroblastoma patient tumors were generally preserved across all preclinical models. However, the presumptive chemo-resistant mesenchymal-like programs, while identifiable in cell lines, were primarily restricted to subpopulations of cancer-associated fibroblasts and Schwann-like cells in vivo. Surprisingly however, a mesenchymal-like program could be acutely chemotherapy-induced in GEMM and was evident in pre-treated patient and PDX samples, suggesting a previously uncharacterized mechanism of therapy escape resulting from an acute shift in cell state. In addition, our approach could further delineate the classical neuroblastoma adrenergic and mesenchymal gene expression programs, discovering for example, novel subpopulations of cancer associated fibroblasts and reproducible subtypes of adrenergic programs. These behaviors were conserved across tumors and preclinical models, which we validated by RNA in situ hybridization, an ultra-sensitive, high resolution, spatial transcriptomics technology. Overall, we offer a nuanced, high-resolution view of neuroblastoma pre-clinical systems for advancing therapeutic development, as well as a generalizable set of computational tools, which can be applied in other diseases. We have created an open-source web resource, featuring this integrated map to aid the scientific community in further exploration of these integrated data (available at http://pscb.stjude.org).
 Citation Format: Richard H. Chapple, Xueying Liu, Sivaraman Natarajan, Margaret I.M. Alexander, Yuna Kim, Anand G. Patel, Christy W. LaFlamme, Min Pan, William C. Wright, Hyeong-Min Lee, Yinwen Zhang, Meifen Lu, Selene C. Koo, Courtney Long, John Harper, Chandra Savage, Melissa D. Johnson, Thomas Confer, Walter J. Akers, Michael A. Dyer, Heather Sheppard, John Easton, Paul Geeleher. An integrated single-cell RNA-seq map of human neuroblastoma tumors and preclinical models uncovers divergent mesenchymal-like gene expression programs [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b38392d6531b277e9f9ee236ca213de0c124f13" target='_blank'>
              Abstract B006: An integrated single-cell RNA-seq map of human neuroblastoma tumors and preclinical models uncovers divergent mesenchymal-like gene expression programs
              </a>
            </td>
          <td>
            Richard H. Chapple, Xueying Liu, Sivaraman Natarajan, M. I. Alexander, Yuna Kim, Anand G. Patel, Christy W. LaFlamme, Min Pan, William C. Wright, Hyeong-Min Lee, Yinwen Zhang, Meifen Lu, Selene C. Koo, Courtney Long, John Harper, Chandra Savage, Melissa D. Johnson, Thomas Confer, Walter J. Akers, Michael A. Dyer, Heather Sheppard, John Easton, P. Geeleher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma A. Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Understanding bacterial gene function remains a major biological challenge. Double-mutant genetic interaction (GI) analysis addresses this challenge by uncovering the functional partners of targeted genes, allowing us to associate genes of unknown function with novel pathways and unravel connections between well-studied pathways, but is difficult to implement at the genome-scale. Here, we develop and use double-CRISPRi to systematically quantify genetic interactions at scale in the Bacillus subtilis envelope, including essential genes. We discover > 1000 known and novel genetic interactions. Our analysis pipeline and experimental follow-ups reveal the distinct roles of paralogous genes such as the mreB and mbl actin homologs, and identify new genes involved in the well-studied process of cell division. Overall, our study provides valuable insights into gene function and demonstrates the utility of double-CRISPRi for high-throughput dissection of bacterial gene networks, providing a blueprint for future studies in diverse bacterial species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed882175f2414bae232e1b044a85ff77b9c005c" target='_blank'>
              Comprehensive double-mutant analysis of the Bacillus subtilis envelope using double-CRISPRi
              </a>
            </td>
          <td>
            B. Koo, Horia Todor, Jiawei Sun, Jordi van Gestel, John S. Hawkins, Cameron C. Hearne, Amy B. Banta, K. C. Huang, Jason M. Peters, Carol A Gross
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eddab72a32f5e67003082a5b17959fbc71f348a1" target='_blank'>
              Gene therapy for polygenic or complex diseases
              </a>
            </td>
          <td>
            Tingting Wu, Yu Hu, Liang V Tang
          </td>
          <td>2024-09-04</td>
          <td>Biomarker Research</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="An important goal in cancer research is to identify driver genes and mutations. Reasoning that such mutations often alter enzymatic functions, we investigated the cancer driver role of enzyme families. Using pan-cancer genomic data and established driver mutation catalogues, we found an unexpectedly high rate of mutations in helicases, making helicases the most frequently mutated enzyme family in cancer. Based on both functional perturbation screens and cancer genomic analyses, we provide evidence that cancers with mutated helicases converge on increased genomic instability and faulty DNA repair. We identify a striking phenotype in cells with loss of the helicase Aquarius (AQR). AQR was exclusively hemizygous lost in cancer genomes, which was associated with elevated levels of structural variants and point mutation signatures indicative of homologous recombination deficiency. Finally, we leverage large dependency maps to show that hemizygous loss is a common tumour suppression mechanism among helicases. In summary, we uncover a striking frequency of mutated helicases with key roles in genomic maintenance, and we nominate novel hemizygous cancer driver genes including AQR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/381ed30f7e9d5d197bbe8fa9f74d37852c71bb4a" target='_blank'>
              Enzyme family-centred approach identifies helicases as recurrent hemizygous tumour suppressor genes
              </a>
            </td>
          <td>
            Karolin Voßgröne, F. Favero, Krushanka Kashyap, F. G. Rodríguez-González, André Vidas Olsen, Xin Li, Balca R. Mardin, Joachim Weischenfeldt, Claus S. Sørensen
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Gastric cancer (GC) is the fifth most common human cancer worldwide, but the genetic etiology is largely unknown. We performed a Bayesian genome-wide association study and selection analyses in a naturally-occurring canine model of GC, the Belgian Tervuren and Sheepdog breeds, to elucidate underlying genetic risk factors. We identified 15 loci with over 90% predictive accuracy for the GC phenotype. Variant filtering revealed germline putative regulatory variants for the EPAS1 (HIF2A) and PTEN genes and a coding variant in CD101. Although closely related to Tervuren and Sheepdogs, Belgian Malinois rarely develop GC. Across-breed analyses uncovered protective haplotypes under selection in Malinois at SOX2-OT and IGF2BP2. Among Tervuren and Sheepdogs, HDAC2 putative regulatory variants were present at comparatively high frequency and were associated with GC. Here, we describe a complex genetic architecture governing GC in a dog model, including genes such as PDZRN3, that have not been associated with human GC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1797dc7272399029f05047a73a22158a3c2ed9d7" target='_blank'>
              Genomic analyses identify 15 susceptibility loci and reveal HDAC2 , SOX2-OT , and IGF2BP2 in a naturally-occurring canine model of gastric cancer
              </a>
            </td>
          <td>
            S. Cook, S. Hugen, Jessica J. Hayward, T. Famula, Janelle M. Belanger, Elizabeth McNiel, H. Fieten, Anita M. Oberbauer, P. A. Leegwater, Elaine A. Ostrander, P. Mandigers, Jacquelyn M. Evans
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/895b7a9a7a504580b14d647edcfb85c539f3ac48" target='_blank'>
              Enhanced depletion of MLL-fusion proteins in acute leukemia: potential for improved therapeutic outcomes
              </a>
            </td>
          <td>
            Noelia Che, S. Cantilena, R. Looi-Somoye, Danesh Sundar, Kent Fung, J. de Boer, Owen Williams
          </td>
          <td>2024-08-16</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Clear cell renal cell carcinoma (ccRCC) is characterized by its aggressive behavior and complex molecular heterogeneity, posing significant challenges for treatment and prognostication. This study offers a comprehensive analysis of ccRCC by leveraging both bulk and single-cell RNA sequencing data, with a specific aim to unravel the complexities of sphingolipid metabolism and the intricate dynamics within the tumor microenvironment (TME). By examining ccRCC samples sourced from public databases, our investigation delves deep into the genetic and transcriptomic landscape of this cancer type. Employing advanced analytical techniques, we have identified pivotal patterns in gene expression and cellular heterogeneity, with a special focus on the roles and interactions of various immune cells within the TME. Significantly, our research has unearthed insights into the dynamics of sphingolipid metabolism in ccRCC, shedding light on its potential implications for tumor progression and strategies for immune evasion. A novel aspect of this study is the development of a risk score model designed to enhance prognostic predictions for ccRCC patients, which is currently pending external validation to ascertain its clinical utility. Despite its contributions, the study is mindful of its limitations, including a reliance on observational data from public sources and a primary focus on RNA sequencing data, which may constrain the depth and generalizability of the findings. The study does not encompass critical aspects, such as protein expression, posttranslational modifications, and comprehensive metabolic profiles. Moreover, its retrospective design underscores the necessity for future prospective studies to solidify these preliminary conclusions. Our findings illuminate the intricate interplay between genetic alterations, sphingolipid metabolism, and immune responses in ccRCC. This research not only enhances our understanding of the molecular foundations of ccRCC but also paves the way for the development of targeted therapies and personalized treatment modalities. The study underlines the importance of cautious interpretation of results and champions ongoing research using diverse methodologies to thoroughly comprehend and effectively combat this formidable cancer type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d290ea11babde7963cf17fade3be71816bf90d65" target='_blank'>
              Integrative analysis of bulk and single-cell RNA sequencing reveals sphingolipid metabolism and immune landscape in clear cell renal cell carcinoma.
              </a>
            </td>
          <td>
            Dongdong Xie, Zhitao Han, Yu Wang, Haoyu Shi, Xiang Wu, Jiaqing Wu, Y. Dai
          </td>
          <td>2024-09-04</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
Oncogenic-addicted non-small cell lung cancer (NSCLC) has emerged as the most prevalent form of lung cancer, presenting a dynamic landscape in treatment modalities. Among these, epidermal growth factor receptor (EGFR)-mutant NSCLC remains the predominant oncogenic mutation, particularly prevalent in regions such as Asia and Latin America.


CASE PRESENTATION
This case study highlights the experience of a woman diagnosed with EGFR-sensitive (del exon 19) mutant NSCLC who demonstrated an extended duration of response (DOR) to third-generation EGFR-TKI therapy. Upon disease progression, detection of MET gene amplification prompted the addition of a selective MET inhibitor to the existing EGFR-TKI regimen, resulting in a complete response for the patient.


DISCUSSION/CONCLUSION
The molecular heterogeneity of this condition has significantly increased in complexity over recent years, marked by the identification of baseline co-alterations and development of a broad spectrum of resistance mechanisms post-EGFR tyrosine kinase inhibitor (TKI) therapy. This complexity poses a substantial challenge to clinicians. Despite the rapid advancement of targeted therapies and the implementation of treatment escalation through combination strategies, there remains an ongoing debate regarding which patients would benefit most from combination therapies, both in the initial treatment phase and in the setting of disease progression, particularly when off-target resistance mechanisms or co-alterations are identified.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aee7b0dfd69142858e9dd93edb7ec143688a294" target='_blank'>
              EGFR plus MET targeted therapies for overcoming treatment resistance in EGFR mutant NSCLC. A Case Report.
              </a>
            </td>
          <td>
            Maria F Martínez-Hernandez, L. Lara-Mejía, C. Izquierdo-Tolosa, L. Cabrera-Miranda, Oscar Arrieta
          </td>
          <td>2024-09-18</td>
          <td>Oncology research and treatment</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA that is found outside the main chromosomes in a cell's nucleus is known as extrachromosomal DNA. This type of DNA is not part of the standard 23 pairs of chromosomes in humans and instead exists as standalone circular or linear DNA structures. Unlike bacterial plasmids, extrachromosomal DNA in human cells contains important genetic material and holds particular significance in cancer cells. Critical characteristics of ecDNA in Cancer is that they frequently harbour amplified versions of oncogenes such as MYC, EGFR, and CCND1. These oncogenes propel the growth and survival of tumors. Cancer cells utilize ecDNA to generate multiple copies of these oncogenes, amplifying their expression without requiring alterations to the chromosomal structure. This amplification grants the cancer cells a competitive edge in growth. ecDNA plays a crucial role in the advancement and growth of cancer, as it amplifies oncogenes, genetic diversity, and resistance to treatments. Its adaptable and ever-changing characteristics empower cancer cells to adjust to external influences such as medication. Gaining insights into ecDNA's operations and impacts could lead to new possibilities for diagnosing and treating cancer, as well as developing therapies aimed at this distinct DNA structure.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b548c36ec02cb85af82ad09081289b0df56847d0" target='_blank'>
              Extrachromosomal DNA : New Players in Oncology
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-10-01</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="137 Background: Resistance to RT remains a clinical challenge, given our limited understanding of the molecular pathways underpinning radioresistance. Here, we investigated for mechanisms linked to RR using paired experimental and clinico-molecular HNC and PCa datasets for discovery and validation of novel pathways that may be enriched in RR cancers. Methods: For experimental data, we generated RR HNC (FaDu, HK1) and PCa (22Rv1, DU145) cell lines following high-dose X-irradiation (90 Gy in 45 fr). Genotypic characterization of RR and wildtype (WT) cells was performed by genomic and transcriptomic sequencing using WES (100X) and RNAseq (Illumina, CA), respectively, followed by functional characterization by western blot (WB) and immunofluorescence (IF). For clinico-molecular data, we utilized two prospectively recruited RT cohorts, consisting of 311 patients with HNC (n=158) and PCa (n=153). Tumor transcriptomes were profiled using RNAseq and Affymetrix ST array (ThermoFisher, CA), respectively. Comparative analyses of molecular profiles were first performed between RR and WT in vitro models to identify dysregulated pathways that were linked to radioresistance, and subsequently tested for association with disease-free survival (DFS) in the clinical cohorts. Results: Comparative genomic analyses between RR and WT HNC and PCa models revealed an abundance of acquired mutations in the RR compared with WT models, with a higher mutation count observed in PCa than HNC cells (SNV counts: 2,158 [DU145-RR] and 1,387 [22Rv1-RR] vs 396 [FaDu-RR] and 25 [HK1-RR], P<0.0001). Mutational signature analyses indicated a common enrichment of DNA mismatch repair-related mutational signatures (SBS3, SBS14, and SBS21) across the RR HNC and PCa models. Transcriptomic profiling revealed an upregulation of the BAHD1 gene, which is involved in heterochromatin formation, across the 4 RR cell lines, corroborated by an enrichment of heterochromatin-related genesets. These results were consistent with functional characterization of RR versus WT cells by WB and IF indicating increased DNA repair capacity and heterochromatin response post-4 Gy irradiation. We further confirmed the dependency of the RR phenotype on the heterochromatin response with BAHD1 knockdown, resulting in reversal of these cellular responses. Finally, our results were supported by survival analyses indicating an inferior DFS post-RT in patients with HNC and PCa harboring a higher expression of heterochromatin-related geneset (HNC: HRhigh vs low 1.51 ; PCa: HRhigh vs low 1.37 ). Conclusions: Herein, by leveraging on paired experimental and clinico-molecular datasets, we have uncovered a novel BAHD1-dependent heterochromatin response that underpins resistance to RT in HNC and PCa, which may be amendable to systemic agents targeting chromatin remodeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ea1270e10cc26b39aabc2ffd1632147257d9664" target='_blank'>
              Analysis of heterochromatin remodeling as a mechanism of radiotherapy (RT) resistance through molecular profiling of radioresistant (RR) head and neck (HNC) and prostate cancers (PCa).
              </a>
            </td>
          <td>
            Evelyn Tan, Chaw Yee Beh, Celestia P X Yeo, Boon Hao Hong, Dennis J J Poon, Dewi Susanti, E. Yeo, Pek L Chu, Melvin L K Chua
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="


 High-grade gliomas (HGGs) are the most aggressive type of gliomas and have the poorest outcomes. Chromatin remodeling (CR) genes have been implicated in multiple oncogenic pathways in numerous cancer types. In gliomagenesis, CR genes have been implicated in regulating stemness of glioma cells, the tumor microenvironment (TME), and resistance to therapies.



 We performed molecular profiling of 4244 HGGs and evaluated associations of CR mutations with other cancer related biomarkers, infiltration by immune cells, and immune gene expression. We also evaluated the association between CR mutations and survival in IDH WT HGG patients.



 Nearly 10% of HGGs carry mutations in CR genes, with higher prevalence (15%) in HGGs with IDH mutations. Analysis of co-occurrence with other biomarkers revealed that CR-mutated HGGs possess favorable genetic alterations which may have prognostic value. CR-mutated HGGs with wild type IDH demonstrated colder TME and worse OS overall compared to the CR-wild type HGGs.



 Our study reveals the prognostic effects of CR mutations in HGG and points to several biomarker candidates that could suggest sensitivity to emerging therapeutic strategies.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/18a6ec214f04da421e41c3579984b7e7f5d8d94d" target='_blank'>
              Biological and prognostic relevance of epigenetic regulatory genes in high-grade gliomas (HGGs)
              </a>
            </td>
          <td>
            Sonikpreet Aulakh, Joanne Xiu, Andrew Hinton, S. Darabi, M. Demeure, Soma Sengupta, Santosh Kesari, David M Ashley, Ashley Love Sumrall, M. Glantz, D. Spetzler
          </td>
          <td>2024-10-04</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4da881111cc4055f065c04d5a3336c85756775d1" target='_blank'>
              Somatic mutational landscape across Indian breast cancer cases by whole exome sequencing
              </a>
            </td>
          <td>
            Rahul Kumar, Supriya Awasthi, Dibyabhaba Pradhan, Rakesh Kumar, Harsh Goel, Jay Singh, Imran Haider, S. Deo, Chitresh Kumar, Anurag Srivastava, Amar Bhatnagar, Rakesh Kumar, S. Lakshmi, Paul Augustine, Amar Ranjan, Anita Chopra, A. Gogia, A. Batra, S. Mathur, G. Rath, T. Kaur, R. Dhaliwal, Aleyamma Mathew, Usha Agrawal, Showket Hussain, P. Tanwar
          </td>
          <td>2024-08-12</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8eb55432a81ae9f11fc653e3722684f5e8a7ed1" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="The detection of mitochondrial DNA (mtDNA) mutations in single cells holds considerable potential to define clonal relationships coupled with information on cell state in humans. Previous methods focused on higher heteroplasmy mutations that are limited in number and can be influenced by functional selection, introducing biases for lineage tracing. Although more challenging to detect, intermediate to low heteroplasmy mtDNA mutations are valuable due to their high diversity, abundance, and lower propensity to selection. To enhance mtDNA mutation detection and facilitate fine-scale lineage tracing, we developed the single-cell Regulatory multi-omics with Deep Mitochondrial mutation profiling (ReDeeM) approach, an integrated experimental and computational framework. Recently, some concerns have been raised about the analytical workflow in the ReDeeM framework. Specifically, it was noted that the mutations detected in a single molecule per cell are enriched on edges of mtDNA molecules, suggesting they resemble artifacts reported in other sequencing approaches. It was then proposed that all mutations found in one molecule per cell should be removed. We detail our error correction method, demonstrating that the observed edge mutations are distinct from previously reported sequencing artifacts. We further show that the proposed removal leads to massive elimination of bona fide and informative mutations. Indeed, mutations accumulating on edges impact a minority of all mutation calls (for example, in hematopoietic stem cells, the excess mutations on the edge account for only 4.3%-7.6% of the total). Recognizing the value of addressing edge mutations even after applying consensus correction, we provide an additional filtering option in the ReDeeM-R package. This approach effectively eliminates the position biases, leads to a mutational signature indistinguishable from bona fide mitochondrial mutations, and removes excess low molecule high connectedness mutations. Importantly, this option preserves the large majority of unique mutations identified by ReDeeM, maintaining the ability of ReDeeM to provide a more than 10-fold increase in variant detection compared to previous methods. Additionally, the cells remain well-connected. While there is room for further refinement in mutation calling strategies, the significant advances and biological insights provided by the ReDeeM framework are unique and remain intact. We hope that this detailed discussion and analysis enables the community to employ this approach and contribute to its further development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94914bd4d265668ea63fab9b4a1aa10d1231c91c" target='_blank'>
              Robustness and reliability of single-cell regulatory multi-omics with deep mitochondrial mutation profiling
              </a>
            </td>
          <td>
            Chen Weng, Jonathan S. Weissman, V. Sankaran
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Despite a critical role for tumor-initiating cancer stem cells (CSCs) in breast cancer progression, major questions remain about the properties and signaling pathways essential for their function. Recent discoveries highlighting mechanisms of CSC-resistance to the stress caused by chromosomal instability (CIN) may provide valuable new insight into the underlying forces driving stemness properties. While stress tolerance is a well-known attribute of CSCs, CIN-induced stress is distinctive since levels appear to increase during tumor initiation and metastasis. These dynamic changes in CIN levels may serve as a barrier constraining the effects of non-CSCs and shaping the stemness landscape during the early stages of disease progression. In contrast to most other stresses, CIN can also paradoxically activate pro-tumorigenic antiviral signaling. Though seemingly contradictory, this may indicate that mechanisms of CIN tolerance and pro-tumorigenic inflammatory signaling closely collaborate to define the CSC state. Together, these unique features may form the basis for a critical relationship between CIN and stemness properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d491eba9730d9558923c4e53a3264dd576b3117a" target='_blank'>
              Chromosomal instability as an architect of the cancer stemness landscape
              </a>
            </td>
          <td>
            Shahnawaz A. Baba, Aran Zakeri, J. Desgrosellier
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Myelodysplastic syndromes (MDS) are myeloid malignancies with heterogeneous genotypes and phenotypes, characterized by ineffective haematopoiesis and a high risk of progression towards acute myeloid leukaemia (AML). Prognosis for patients treated with hypomethylating agents (HMAs), as is azacytidine, the main drug used as frontline therapy for MDS is mostly based on cytogenetics and next generation sequencing (NGS) of the initial myeloid clone. Although the critical influence of the epigenetic landscape upon cancer cells survival and development as well on tumour environment establishment is currently recognized and approached within current clinical practice in MDS, the heterogenous response of the patients to epigenetic therapy is suggesting a more complex mechanism of action, as is the case of RNA methylation. In this sense, the newly emerging field of epitranscriptomics could provide a more comprehensive perspective upon the modulation of gene expression in malignancies, as is the proof‐of‐concept of MDS. We initially did RNA methylation sequencing on MDS patients (n = 6) treated with azacytidine and compared responders with non‐responders. Afterwards, the genes identified were assessed in vitro and afterwards validated on a larger cohort of MDS patients treated with azacytidine (n = 58). Our data show that a more accurate prognosis could be based on analysing the methylome and thus we used methylation sequencing to differentially split high‐grade MDS patients with identical demographical and cytogenetic features, between azacytidine responders and non‐responders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/302c378a460869cbc4ab47244a1bece112c425c0" target='_blank'>
              RNA methylation sequencing shows different gene expression signatures for response to azacytidine therapy in high‐grade myelodysplastic syndromes
              </a>
            </td>
          <td>
            Diana Gulei, Vlad Moisoiu, David Kegyes, Rareș Drula, Sabina Iluta, A. Țigu, Mădălina Nistor, Ciprian Jitaru, Anamaria Bancos, Petra Rotariu, Corina Popovici, D. Dima, R. Tomai, Ioana Rus, Cătălin Constantinescu, R. Munteanu, Diana Cenariu, U. Sezerman, M. Zdrenghea, Jaroslav Čermák, Hermann Einsele, Gabriel Ghiaur, C. Tomuleasa
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Maintaining engineered cell populations’ genetic stability is a key challenge in synthetic biology. Synthetic genetic constructs compete with a host cell’s native genes for expression resources, burdening the cell and impairing its growth. This creates a selective pressure favouring mutations which alleviate this growth defect by removing synthetic gene expression. Non-functional mutants thus spread in cell populations, eventually making them lose engineered functions. Past work has attempted to limit mutation spread by coupling synthetic gene expression to survival. However, these approaches are highly context-dependent and must be tailor-made for each particular synthetic gene circuit to be retained. In contrast, we develop and analyse a biomolecular controller which depresses mutant cell growth independently of the mutated synthetic gene’s identity. Modelling shows how our design can be deployed alongside various synthetic circuits without any re-engineering of its genetic components, outperforming extant gene-specific mutation spread mitigation strategies. Our controller’s performance is evaluated using a novel simulation approach which leverages resource-aware cell modelling to directly link a circuit’s design parameters to its population-level behaviour. Our design’s adaptability promises to mitigate mutation spread in an expanded range of applications, whilst our analyses provide a blueprint for using resource-aware cell models in circuit design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2420835e594d4a65b9dfcd0a4c88ad37ec47058" target='_blank'>
              Model-guided gene circuit design for engineering genetically stable cell populations in diverse applications
              </a>
            </td>
          <td>
            Kirill Sechkar, Harrison Steel
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16db7fe8f3bc4329cf344c224d8e623bae436c6d" target='_blank'>
              Elucidating the pan-oncologic landscape of S100A9: prognostic and therapeutic corollaries from an integrative bioinformatics and Mendelian randomization analysis
              </a>
            </td>
          <td>
            Yingying Chen, Zixuan Wu, Xingxing Yi
          </td>
          <td>2024-08-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93f34d93ff1b305ad73729aa0a43e0b0a160605f" target='_blank'>
              Characterization of MYC Rearrangements in Multiple Myeloma: an Optical Genome Mapping Approach
              </a>
            </td>
          <td>
            Jung Yoon, Taesung Jeon, J. Kwon, Soo-Young Yoon
          </td>
          <td>2024-09-20</td>
          <td>Blood Cancer Journal</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Due to the significant morbidity and mortality of hemoglobinopathies, curative options have long been pursued. The overall goal of gene therapy is to modify a patient's own hematopoietic stem cells to overcome the deleterious effects of the underlying genetic defect by gene addition, gene editing, or gene silencing. Gene addition incorporates genes with superior function than the abnormal gene; gene editing takes advantage of molecular tools such as zinc finger proteins, Transcription Activator-Like Effector Nucleases and Clustered Regularly Interspaced Short Palindromic Repeats coupled with Cas9 proteins (CRISPR-Cas9) which allow for sequence-specific breaks in DNA that disrupt gene function; and gene silencing suppresses gene expression by interference with mRNA transcription/protein translation or epigenetic modification. The majority of gene therapy strategies for hemoglobinopathies have targeted erythroid-specific BCL11A, a major regulator of fetal hemoglobin repression at the gamma-globin locus, in the normal fetal-to-adult hemoglobin switch that occurs shortly after birth. Other goals have involved the incorporation of anti-sickling globins, such as βT87Q or βAS3. Landmark clinical trials of gene therapy in transfusion-dependent thalassemia and sickle cell disease have shown remarkable efficacy and acceptable safety and culminated in recent regulatory approvals of gene therapy for both diseases in Europe and the United States.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f812c9e54de47feca0c8615f3dda20f25f85b41" target='_blank'>
              A Review of Gene Therapies for Hemoglobinopathies.
              </a>
            </td>
          <td>
            Boubini Jones-Wonni, Amar H Kelkar, Maureen O Achebe
          </td>
          <td>2024-08-15</td>
          <td>Hemoglobin</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2352f41e08e0411c3c2eeaf443ff948521637f4" target='_blank'>
              Deletion of exons 45 to 55 in the DMD gene: from the therapeutic perspective to the in vitro model
              </a>
            </td>
          <td>
            Javier Poyatos-García, Patricia Soblechero-Martín, A. Liquori, A. López-Martínez, Pilar Maestre, Elisa González-Romero, Rafael P. Vázquez-Manrique, Nuria Muelas, Gema García-García, Jessica Ohana, Virginia Arechavala-Gomeza, Juan J. Vílchez
          </td>
          <td>2024-10-01</td>
          <td>Skeletal Muscle</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Gene therapy is advancing at an unprecedented pace, and the recent success of clinical trials reinforces optimism and trust among the scientific community. Recently, the cardiac gene therapy pipeline, which had progressed more slowly than in other fields, has begun to advance, overcoming biological and technical challenges, particularly in treating genetic heart pathologies. The primary rationale behind the focus on monogenic cardiac diseases is the well-defined molecular mechanisms driving their phenotypes, directly linked to the pathogenicity of single genetic mutations. This aspect makes these conditions a remarkable example of "genetically druggable" diseases. Unfortunately, current treatments for these life-threatening disorders are few and often poorly effective, underscoring the need to develop therapies to modulate or correct their molecular substrates. In this review we examine the latest advancements in cardiac gene therapy, discussing the pros and cons of different molecular approaches and delivery vectors, with a focus on their therapeutic application in cardiac inherited diseases. Additionally, we highlight the key factors that may enhance clinical translation, drawing insights from previous trials and the current prospects of cardiac gene therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7decc83e9a8c870c1521b14e063653c9243a518" target='_blank'>
              Gene therapy for cardiac diseases: methods, challenges, and future directions.
              </a>
            </td>
          <td>
            Luca Grisorio, R. Bongianino, Matteo Gianeselli, Silvia G. Priori
          </td>
          <td>2024-09-20</td>
          <td>Cardiovascular research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Current treatments for KRAS-mutant colorectal cancers (CRCs) are often limited by cellular plasticity and rewiring responses. Here we describe a promising therapeutic strategy that simultaneously targets epigenetic and oncogenic signals. Specifically, we show that inhibitors of the histone methyltransferase, EZH2, synergize with various RAS pathway inhibitors and promote dramatic tumor regression in vivo. Together these agents cooperatively suppress WNT-driven transcription and drive CRCs into a more differentiated cell state by inducing the Groucho/TLE corepressor, TLE4, along with a network of WNT pathway inhibitors and intestinal differentiation proteins. However, these agents also induce the pro-apoptotic protein BMF, which subsequently kills these more differentiated cells. Accordingly, cell death can be prevented by activating β-catenin, blocking differentiation, or by ablating BMF expression. Collectively, these studies reveal a new therapeutic approach for treating KRAS-mutant CRCs and illustrate a critical convergence of EZH2 and RAS on oncogenic WNT signals, intestinal differentiation, and apoptosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47a75ff581332b9384e613ce0d1c80e31add042e" target='_blank'>
              Epigenetic and oncogenic inhibitors cooperatively drive differentiation and kill KRAS-mutant colorectal cancers.
              </a>
            </td>
          <td>
            P. Loi, Amy E. Schade, Carrie L Rodriguez, Anjana Krishnan, N. Perurena, Van T M Nguyen, Yilin Xu, Marina Watanabe, Rachel A Davis, Alycia Gardner, Natalie F Pilla, Kaia Mattioli, Olesja Popow, Nuray Gunduz, T. Lannagan, Samantha Fitzgerald, Ewa T Sicinska, Jia-Ren Lin, William Tan, L. Brais, Kevin Haigis, M. Giannakis, K. Ng, Sandro Santagata, Kristian Helin, Owen J. Sansom, K. Cichowski
          </td>
          <td>2024-08-12</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) poses significant challenges in veterinary medicine, with limited treatment options and poor survival rates. While substantial progress has been made in characterizing human AML, translating these advancements to veterinary practice has been hindered by limited molecular understanding and diagnostic tools. The case study presented illustrates the application of whole genome sequencing in diagnosing AML in a dog, showcasing its potential in veterinary oncology. Our approach facilitated comprehensive genomic analysis, identifying mutations in genes that may be associated with AML pathogenesis in dogs, such as KRAS, IKZF1, and RUNX1. However, without supportive evidence of its clinical utility (eg, association with response to treatment or prognosis), the information is limited to exploration. This article reviews the comparative features of canine AML with human AML and discusses strategies to shrink the knowledge gap between human and veterinary medicine with cost-effective next-generation sequencing (NGS) techniques. By utilizing these approaches, the unique and shared molecular features with human AML can be identified, aiding in molecular classification and therapeutic development for both species. Despite the promise of NGS, challenges exist in implementing it into routine veterinary diagnostics. Cost considerations, turnaround times, and the need for robust bioinformatics pipelines and quality control measures must be addressed. Most importantly, analytical and clinical validation processes are essential to ensure the reliability and clinical utility of NGS-based assays. Overall, integrating NGS technologies into veterinary oncology holds great potential for advancing our understanding of AML and improving disease stratification, in hopes of improving clinical outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8bd6fdd9b3631dd5e67d1617f989917515a9291" target='_blank'>
              Advancements in genetic analysis: Insights from a case study and review of next-generation sequencing techniques for veterinary oncology applications.
              </a>
            </td>
          <td>
            R Adam Harris, Jillian Nolan, Dylan Ammons, Samantha Beeson, Douglas Thamm, Anne Avery
          </td>
          <td>2024-10-04</td>
          <td>Veterinary clinical pathology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aberrant gene expression patterns in acute myeloid leukemia (AML) with balanced chromosomal translocations are often associated with dysregulation of epigenetic modifiers. The AML1/ETO (RUNX1/MTG8) fusion protein, caused by the translocation (8;21)(q22;q22), leads to the epigenetic repression of its target genes. We aimed in this work to identify critical epigenetic modifiers, on which AML1/ETO-positive AML cells depend on for proliferation and survival using shRNA library screens and global transcriptomics approaches. Using shRNA library screens, we identified 41 commonly depleted genes in two AML1/ETO-positive cell lines Kasumi-1 and SKNO-1. We validated, genetically and pharmacologically, DNMT1 and ATR using several AML1/ETO-positive and negative cell lines. We also demonstrated in vivo differentiation of myeloblasts after treatment with the DNMT1 inhibitor decitabine in a patient with an AML1/ETO-positive AML. Bioinformatic analysis of global transcriptomics after AML1/ETO induction in 9/14/18-U937 cells identified 973 differentially expressed genes (DEGs). Three genes (PARP2, PRKCD, and SMARCA4) were both downregulated after AML1/ETO induction, and identified in shRNA screens. In conclusion, using unbiased shRNA library screens and global transcriptomics, we have identified several driver epigenetic regulators for proliferation in AML1/ETO-positive AML. DNMT1 and ATR were validated and are susceptible to pharmacological inhibition by small molecules showing promising preclinical and clinical efficacy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4848d5afaf1e1f3f5c753596a2ed8d1e491ca9f0" target='_blank'>
              Identification of epigenetic modifiers essential for growth and survival of AML1/ETO-positive leukemia.
              </a>
            </td>
          <td>
            J. Duque-Afonso, P. Veratti, Usama-Ur Rehman, Heike Herzog, J. Mitschke, G. Greve, Julian Eble, Bettina Berberich, Johanna Thomas, Milena Pantic, Miguel Waterhouse, Gaia Gentile, Olaf Heidenreich, Cornelius Miething, Michael Lübbert
          </td>
          <td>2024-08-15</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ABSTRACT The human T-cell lymphotropic virus type 1 (HTLV-1) is a single-stranded positive-sense RNA virus that belongs to the Retroviridae family, genus Deltaretro, and infects approximately five to 10 million people worldwide. Although a significant number of individuals living with HTLV-1 remain asymptomatic throughout their lives, some develop one or more severe clinical conditions, such as HTLV-1-associated myelopathy/tropical spastic paraparesis (HAM/TSP), a progressive and debilitating disease, and/or a subtype of non-Hodgkin’s lymphoma with a more threatening course known as adult T-cell leukemia/lymphoma (ATLL). Moreover, current therapeutic options are limited and focus primarily on treating symptoms and controlling viral latency. CRISPR-Cas9 gene editing is proposed as a promising tool to address the intricate links associated with HTLV-1. By targeting or silencing key genes during initial infection and dysregulating immune signaling pathways, CRISPR-Cas9 offers potential intervention opportunities. In this review, we address the therapeutic potential of CRISPR-Cas9 gene editing, as well as examine the primary mechanisms involved in editing potential target genes and discuss the existing evidence in the current scientific literature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6aa8304d3f00cdbae60023f6fc20b04a24438fa0" target='_blank'>
              Novel approaches for HTLV-1 therapy: innovative applications of CRISPR-Cas9
              </a>
            </td>
          <td>
            Wilson Domingues, V. A. Folgosi, S. Sanabani, Pedro Domingos Leite Junior, Tatiane Assone, J. Casseb
          </td>
          <td>2024-08-26</td>
          <td>Revista do Instituto de Medicina Tropical de São Paulo</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="OBJECTIVES
The TP53 mutation, a prevalent tumor suppressor gene alteration, is linked to chemotherapy resistance, increased relapse rates and diminished overall survival (OS) in acute myeloid leukemia (AML) patients.


METHODS
In this study, we characterize the TP53 mutation phenotypes across various AML cohorts utilizing The Cancer Genome Atlas (TCGA) data. We devised a TP53-related prognostic signature derived from differentially expressed genes between mutated and wild-type TP53 AML specimens. In-depth analyses were conducted, encompassing genetic variation, immune cell infiltration and prognostic stratification.


RESULTS
A six-gene TP53-related signature was established using least absolute shrinkage and selection operator (LASSO)-Cox regression, demonstrating robust prognostic predictability. This signature exhibited strong performance in both the OHSU validation cohorts, an independent Gene Expression Omnibus (GEO) validation cohort (GSE71014) and proved by results of the in vivo experiment. Finally, we used single cell database (GSE198681) to observe the characteristics of these six genes.


DISCUSSION
Our study may facilitate the development of efficacious therapeutic approaches and provide a novel idea for future research. Conclusion: The TP53-related signature and pattern hold the potential to refine prognostic stratification and underscore emerging targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b57809ff1c4369159931bf563a0e583afd0ffd0c" target='_blank'>
              Elucidating the immune landscape and potential prognostic model in acute myeloid leukemia with TP53 mutation.
              </a>
            </td>
          <td>
            Gelan Zhu, J. Cai, Wanbin Fu, Yue Sun, Ting Wang, Hua Zhong
          </td>
          <td>2024-09-27</td>
          <td>Hematology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44f2554beac8f2504f8636ae14089de94d4df8bc" target='_blank'>
              Deciphering the impact of aggregated autophagy-related genes TUBA1B and HSP90AA1 on colorectal cancer evolution: a single-cell sequencing study of the tumor microenvironment
              </a>
            </td>
          <td>
            Qianping Xu, Chao Liu, Hailin Wang, Shujuan Li, Hanshen Yan, Ziyang Liu, Kexin Chen, Yaoqin Xu, Runqin Yang, Jingfang Zhou, Xiaolin Yang, Jie Liu, Lexin Wang
          </td>
          <td>2024-09-11</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The effectiveness of drug treatments is profoundly influenced by individual responses, which are shaped by gene expression variability, particularly within pharmacogenes. Leveraging single-cell RNA sequencing (scRNA-seq) data, our study explores the extent of expression variability among pharmacogenes in a wide array of cell types across eight different human tissues, shedding light on their impact on drug responses. Our findings broaden the established link between variability in pharmacogene expression and drug efficacy to encompass variability at the cellular level. Moreover, we unveil a promising approach to enhance drug efficacy prediction. This is achieved by leveraging a combination of cross-cell and cross-individual pharmacogene expression variation measurements. Our study opens avenues for more precise forecasting of drug performance, facilitating tailored and more effective treatments in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11b12a7bcbec6921ec291037e369d1c317c0110c" target='_blank'>
              Deciphering single-cell gene expression variability and its role in drug response.
              </a>
            </td>
          <td>
            Sizhe Liu, Liang Chen
          </td>
          <td>2024-09-15</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09486473d5ae00655afa329978bc6a48b65d161f" target='_blank'>
              The response to influenza vaccination is associated with DNA methylation-driven regulation of T cell innate antiviral pathways
              </a>
            </td>
          <td>
            Hongxiang Fu, H. Pickering, L. Rubbi, Ted M. Ross, Wanding Zhou, Elaine F. Reed, Matteo Pellegrini
          </td>
          <td>2024-08-21</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Gene editing technology has gained popularity over the past two decades with the scientific advancements being made in genetics and computational biology. CRISPR-Cas9 technique allows us to target genetic material and perform accurate manipulation of targeted gene function. Cgr operon, commonly found in the gut microbiome bacteria of E. lenta has several attributes which can be modified using the CRISPR-Cas9 methodology. Here, we briefly describe how the Cas9 nuclease can be used to prevent Digoxin from being suppressed by the cgr gene, directly correlating to improved cardiac function. The editing tool is described to assist in mutating the genetic material of cgr gene discussing the inhibition of Na+/K+ ATPase in cardiac myocytes and CG-content. The principle of operation behind CRISPR-Cas9 is described using non-homologous end joining or homology-directed repair cellular mechanisms. Target site selection, designing sgRNAs and introducing various mutations are discussed with limitations of current technology and possible applications in the field of point of care diagnostics and biosensing developing therapeutic interventions using cgr operon.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/627afe18bee815e175f09e1300b0be4d397c2003" target='_blank'>
              CRISPR-Cas9 as a gene editing tool using cardiac glycoside reductase operon for digoxin metabolism
              </a>
            </td>
          <td>
            Tanishka Nale, Karan Dhingra, Saloni Verma
          </td>
          <td>2024-12-25</td>
          <td>Journal of Knowledge Learning and Science Technology ISSN: 2959-6386 (online)</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Alexander Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii-DerIda Chen, C. Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="Abstract CRISPR/Cas9 genome editing is a pervasive research tool due to its relative ease of use. However, some systems are not amenable to generating edited clones due to genomic complexity and/or difficulty in establishing clonal lines. For example, Drosophila Schneider 2 (S2) cells possess a segmental aneuploid genome and are challenging to single-cell select. Here, we describe a streamlined CRISPR/Cas9 methodology for knock-in and knock-out experiments in S2 cells, whereby an antibiotic resistance gene is inserted in-frame with the coding region of a gene-of-interest. By using selectable markers, we have improved the ease and efficiency for the positive selection of null cells using antibiotic selection in feeder layers followed by cell expansion to generate clonal lines. Using this method, we generated the first acentrosomal S2 cell lines by knocking-out centriole genes Polo-like Kinase 4/Plk4 or Ana2 as proof of concept. These strategies for generating gene-edited clonal lines will add to the collection of CRISPR tools available for cultured Drosophila cells by making CRISPR more practical and therefore improving gene function studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d9bec46a5fefb60dc99b1e4c11b9d69666ffa32" target='_blank'>
              Generating CRISPR-edited clonal lines of cultured Drosophila S2 cells
              </a>
            </td>
          <td>
            John M. Ryniawec, Anastasia Amoiroglou, Gregory C. Rogers
          </td>
          <td>2024-08-17</td>
          <td>Biology Methods & Protocols</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Background: Dynamic regulation of gene expression is fundamental for cellular adaptation to exogenous stressors. PTEFb-mediated promoter proximal pause-release of Pol II is a conserved regulatory mechanism for synchronous transcriptional induction best described in response to heat shock, but this pro-survival role has not been examined in the applied context of cancer therapy. Design/Method: In order to examine the dynamics of chromatin reorganization following radiotherapy, we performed a combination of ChIP-, ATAC-, and RNA-seq in model systems of diffuse intrinsic pontine glioma (DIPG) and other pediatric high-grade gliomas (pHGG) following IR exposure. We interrogated IR-induced gene expression in the presence or absence of PTEFb blockade, including both mechanistic and functional consequences of concurrent inhibition or genetic depletion. We utilized culture models with live cell imaging to assess the therapeutic synergy of PTEFb inhibition with IR, as well as the therapeutic index of this intervention relative to normal controls. Finally, we employed orthotopic models of pHGG treated with conformal radiotherapy and CNS-penetrant PTEFb inhibitors in order to assess tolerability and anti-tumor effect in vivo. Results: Rapid genome-wide redistribution of active chromatin features and PTEFb facilitates Pol II pause-release to drive nascent transcriptional induction within hours of exposure to therapeutic ionizing radiation. Concurrent inhibition of PTEFb imparts a transcription elongation defect, abrogating canonical adaptive programs such as DNA damage repair and cell cycle regulation. This combination demonstrates a potent, synergistic therapeutic potential agnostic of glioma subtype, leading to a marked induction of tumor cell apoptosis and prolongation of xenograft survival.Conclusion: These studies reveal a central role for PTEFb underpinning the early adaptive response to radiotherapy, opening new avenues for combinatorial treatment in these lethal malignancies.
 Citation Format: Faye M. Walker, Lays Martin Sobral, Etienne Danis, Bridget Sanford, Ilango Balakrishnan, Dong Wang, Angela Pierce, Sana Karam, Natalie J. Serkova, Nicholas K. Foreman, Sujatha Venkataraman, Robin Dowell, Rajeev Vibhakar, Nathan A. Dahl. Rapid PTEFb-dependent transcriptional reorganization underpins the glioma adaptive response to radiotherapy [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29cbc389c63c005639d0ccef6f75bb33d60a2a03" target='_blank'>
              Abstract B078: Rapid PTEFb-dependent transcriptional reorganization underpins the glioma adaptive response to radiotherapy
              </a>
            </td>
          <td>
            Faye Walker, L. M. Sobral, Etienne P. Danis, Bridget Sanford, Ilango Balakrishnan, Dong Wang, Angela M Pierce, Sana D Karam, Natalie J Serkova, Nicholas K. Foreman, S. Venkataraman, Robin Dowell, Rajeev Vibhakar, N. Dahl
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Amino-terminal (Nt-) acetylation (NTA) is a common protein modification, affecting 80% of cytosolic proteins in humans. The human essential gene, NAA10, encodes the enzyme NAA10, as the catalytic subunit for the N-terminal acetyltransferase A (NatA) complex, including the accessory protein, NAA15. The first human disease directly involving NAA10 was discovered in 2011, and it was named Ogden syndrome (OS), after the location of the first affected family residing in Ogden, Utah, USA. Since that time, other variants have been found in NAA10 and NAA15. Here we describe the generation of 31 iPSC lines, with 16 from females and 15 from males. This cohort includes CRISPR-mediated correction to the wild-type genotype in 4 male lines, along with editing one female line to generate homozygous wild-type or mutant clones. Following the monoclonalizaiton and screening for X-chromosome activation status in female lines, 3 additional pairs of female lines, in which either the wild type allele is on the active X chromosome (Xa) or the pathogenic variant allele is on Xa, have been generated. Subsets of this cohort have been successfully used to make cardiomyocytes and neural progenitor cells (NPCs). These cell lines are made available to the community via the NYSCF Repository.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab1adf8656f9aef233d0cf25ef9cb89566ccecd" target='_blank'>
              A repository of Ogden syndrome patient derived iPSC lines and isogenic pairs by X-chromosome screening and genome-editing
              </a>
            </td>
          <td>
            Josephine Wesely, Tom Rusielewicz, Yu-Ren Chen, Brigham Hartley, Dayna McKenzie, Matthew K. Yim, Colin Maguire, Ryan Bia, Sarah Franklin, Rikhil Makwana, E. Marchi, Manali Nikte, Soha Patil, Maria Sapar, Dorota Moroziewicz, Lauren Bauer, Jeannie T. Lee, Frederick J. Monsma, Dan Paull, G. Lyon
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Despite the success of CRISPR/Cas9 in inducing DNA double-strand breaks (DSBs) for genome editing, achieving targeted recombination in somatic cells remains challenging, particularly at recombination cold spots like the Tomato Mosaic Virus (ToMV) resistance locus in Solanum lycopersicum. We investigated the potential of CRISPR/Cas9-induced targeted recombination in somatic cells to overcome linkage drag surrounding the ToMV locus. We employed two strategies: first, inducing DSBs in both alleles of F1 tomato seedlings to promote non-homologous end joining (NHEJ) and homology-directed repair (HDR); second, targeting a single allele in a heterozygous background to induce HDR in seedlings. CRISPR/Cas9 activity was confirmed in F₁ seedlings by detecting NHEJ-mediated mutations at the target sites in ToMV. We developed a bioinformatics pipeline to identify targeted recombinants by analyzing single nucleotide polymorphisms (SNPs) between parental haplotypes, allowing precise tracking of SNP variations. A two-dimensional pooling strategy was employed to distinguish genuine recombination events from PCR artifacts. Despite these advances and the active CRISPR/Cas9 system in F1 progeny, no increase in recombination frequency was observed compared to wild-type plants. We extended our research to protoplasts to assess whether CRISPR/Cas9 could induce targeted recombination under different cellular conditions at the same locus. Consistent with our findings in F1 plants, we observed no increase in recombinant patterns compared to wild-type controls in protoplasts. Our findings suggest that CRISPR/Cas9-induced DSBs are insufficient to break the genetic linkage at the ToMV locus on chromosome 9 in recombination cold spots within somatic cells. Article Summary This research targets plant biologists and geneticists interested in enhancing plant breeding techniques. The study used CRISPR/Cas9 technology to induce DNA breaks in tomato plants. It specifically targeted the Tomato Mosaic Virus (ToMV) resistance gene, which resists natural recombination. The aim was to induce genetic recombination via CRISPR/Cas9. The highly active CRISPR/Cas9 system did not increase the expected genetic changes, indicating challenges in achieving targeted recombination. These findings highlight the challenges in breaking genetic linkages in specific genome regions using current CRISPR methods. These findings are relevant for developing techniques for targeted recombination in plant breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e0b58aaaa05e1e2ef0cf4dbd56fab2aa20c1e65" target='_blank'>
              CRISPR/Cas9-induced breaks are insufficient to break linkage drag surrounding the ToMV locus of Solanum lycopersicum
              </a>
            </td>
          <td>
            Jillis Grubben, Gerard Bijsterbosch, Burak Aktürk, Richard G.F. Visser, Henk J. Schouten
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Metabolic dysfunction-associated Fatty Liver Disease (MAFLD) has emerged as one of the leading cardiometabolic diseases. Friend of GATA2 (FOG2) is a transcriptional co-regulator that has been shown to regulate hepatic lipid metabolism and accumulation. Using meta-analysis from several different biobank datasets, we identified a coding variant of FOG2 (rs28374544, A1969G, S657G) predominantly found in individuals of African ancestry (minor allele frequency~20%), which is associated with liver failure/cirrhosis phenotype and liver injury. To gain insight into potential pathways associated with this variant, we interrogated a previously published genomics dataset of 38 human induced pluripotent stem cell (iPSCs) lines differentiated into hepatocytes (iHeps). Using Differential Gene Expression Analysis and Gene Set Enrichment Analysis, we identified the mTORC1 pathway as differentially regulated between iHeps from individuals with and without the variant. Transient lipid-based transfections were performed on the human hepatoma cell line (Huh7) using wild-type FOG2 and FOG2S657G and demonstrated that FOG2S657G increased mTORC1 signaling, de novo lipogenesis, and cellular triglyceride synthesis and mass. In addition, we observed a significant downregulation of oxidative phosphorylation in FOG2S657G cells in fatty acid-loaded cells but not untreated cells, suggesting that FOG2S657G may also reduce fatty acid to promote lipid accumulation. Taken together, our multi-pronged approach suggests a model whereby the FOG2S657G may promote MAFLD through mTORC1 activation, increased de novo lipogenesis, and lipid accumulation. Our results provide insights into the molecular mechanisms by which FOG2S657G may affect the complex molecular landscape underlying MAFLD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2945851076c2f51a928c3f7a7537ad4ec5a83e4e" target='_blank'>
              Genotype-First Approach Identifies an Association between rs28374544/FOG2S657G and Liver Disease through Alterations in mTORC1 Signaling
              </a>
            </td>
          <td>
            Donna Conlon, Siri Kanakala, Tess A. Cherlin, Yi-An Ko, Cecilia Vitali, Sharavana Gurunathan, Rasika Venkatesh, Jakob Woerner, Lindsay Guare, Penn Medicine Biobank, Anurag Verma, S. Verma, Marie A. Guerraty
          </td>
          <td>2024-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dd0f703bff7b7dc043c4099be82b88f90d5c4d73" target='_blank'>
              Trametinib Sensitivity is Defined by a Myeloid Differentiation Profile in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Mathieu Quesnel-Vallières, David C Schultz, Alena Orlenko, Yancy Lo, Jason Moore, Marylyn Ritchie, David Roth, Martin Carroll, Yoseph Barash, Kristen W Lynch, Sara Cherry
          </td>
          <td>2024-09-24</td>
          <td>Drugs in R&D</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="In the fields of medicine and bioscience, gene editing is increasingly recognized as a promising therapeutic approach for treating pathogenic variants in humans and other living organisms. With advancements in technology and knowledge, it is now understood that most genetic defects are caused by single-base pair variants. The ability to substitute genes using genome editing tools enables scientists and doctors to cure genetic diseases and disorders. Starting with CRISPR (clustered regularly interspaced short palindromic repeats)/Cas, the technology has evolved to become more efficient and safer, leading to the development of base and prime editors. Furthermore, various approaches are used to treat genetic disorders such as hemophilia, cystic fibrosis, and Duchenne muscular dystrophy. As previously mentioned, most genetic defects leading to specific diseases are caused by single-base pair variants, which can occur at many locations in corresponding gene, potentially causing the same disease. This means that, even when using the same genome editing tool, results in terms of editing efficiency or treatment effectiveness may differ. Therefore, different approaches may need to be applied to different types of diseases. Prevalently, due to the safety of adeno-associated virus (AAV) vectors in gene therapy, most clinical trials of gene therapy are based on AAV delivery methods. However, despite their safety and nonintegration into the host genome, their limitations, such as confined capacity, dosage-dependent viral toxicity, and immunogenicity, necessitate the development of new approaches to enhance treatment effects. This review provides the structure and function of each CRISPR-based gene editing tool and focuses on introducing new approaches in gene therapy associated with improving treatment efficiency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/175c7ca99490d5250dd5cadc163e90b91db26af6" target='_blank'>
              Mechanism of Genome Editing Tools and Their Application on Genetic Inheritance Disorders
              </a>
            </td>
          <td>
            Dae Hwan Oh
          </td>
          <td>2024-09-16</td>
          <td>Global Medical Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="CRISPR-Associated Transposases (CASTs) hold tremendous potential for microbial genome editing due to their ability to integrate large DNA cargos in a programmable and site-specific manner. However, the widespread application of CASTs has been hindered by their low efficiency in diverse, non-model bacteria. In an effort to address this shortcoming, we conducted the first genome-wide screen for host factors impacting Vibrio cholerae CAST (VchCAST) activity and used the findings to increase VchCAST editing efficiency. A genome-wide loss-of-function mutant library in E. coli was screened to identify 15 genes that impact type VchCAST transposition. Of these, seven factors were validated to improve VchCAST activity and two were found to be inhibitory. Informed by homologous recombination involved effectors, RecD and RecA, we tested the λ-Red recombineering system in our VchCAST editing vectors, which increased its insertion meditated-editing efficiency by 25.7-fold in E. coli while maintaining high target specificity and similar insertion arrangements. Furthermore, λ-Red-enhanced VchCAST achieved increased editing efficiency in the industrially important bacteria Pseudomonas putida and the emerging pathogen Klebsiella michiganensis. This study improves understanding of factors impacting VchCAST activity and enhances its efficiency as a bacterial genome editor. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2483d77579dd4ab8c5f977694f46b22d04f0a6" target='_blank'>
              Identification of Proteins Influencing CRISPR-Associated Transposases for Enhanced Genome Editing
              </a>
            </td>
          <td>
            Leo Song, Amanda T.P. Alker, Agnès Oromí-Bosch, Sophia E. Swartz, Jonathan N.V. Martinson, Jigyasa Arora, Abby M. Wang, Rachel Rovinsky, Sara J. Smith, Emily C. Pierce, A. Deutschbauer, Jennifer A. Doudna, Brady F. Cress, Benjamin E. Rubin
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Direct lineage reprogramming into dopaminergic (DA) neurons holds great promise for the more effective production of DA neurons, offering potential therapeutic benefits for conditions such as Parkinson's disease. However, the reprogramming pathway for fully reprogrammed DA neurons remains largely unclear, resulting in immature and dead-end states with low efficiency. In this study, using single-cell RNA sequencing, the trajectory of reprogramming DA neurons at multiple time points, identifying a continuous pathway for their reprogramming is analyzed. It is identified that intermediate cell populations are crucial for resetting host cell fate during early DA neuronal reprogramming. Further, longitudinal dissection uncovered two distinct trajectories: one leading to successful reprogramming and the other to a dead end. Notably, Arid4b, a histone modifier, as a crucial regulator at this branch point, essential for the successful trajectory and acquisition of mature dopaminergic neuronal identity is identified. Consistently, overexpressing Arid4b in the DA neuronal reprogramming process increases the yield of iDA neurons and effectively reverses the disease phenotypes observed in the PD mouse brain. Thus, gaining insights into the cellular trajectory holds significant importance for devising regenerative medicine strategies, particularly in the context of addressing neurodegenerative disorders like Parkinson's disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bfa61a76443945c179bc5c94c72de94ea5c5df" target='_blank'>
              Epigenetic Dynamics in Reprogramming to Dopaminergic Neurons for Parkinson's Disease.
              </a>
            </td>
          <td>
            Byounggook Cho, Junyeop Kim, Sumin Kim, Saemin An, Yerim Hwang, Yunkyung Kim, Daeyeol Kwon, Jongpil Kim
          </td>
          <td>2024-09-16</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Cellular crosstalk mediated by ligand–receptor interactions largely complicates the tumour ecosystem, resulting in heterogeneous tumour microenvironments that affect immune response and clinical benefits from immunotherapy. Epigenetic mechanisms are pivotal to expression changes of immune‐related genes and can modulate the anti‐tumour immune response. However, the functional consequences of disrupted epigenetic regulators (ERs) on ligand–receptor interactions in the tumour microenvironment remain largely unexplored. Here, we proposed mutations of ERs in perturbed interactions (MERIN), a molecular network‐based approach that incorporates multi‐omics data, to infer the potential consequences of ER mutations on ligand–receptor interaction perturbations. Leveraging cancer genomic profiles and molecular interaction data, we comprehensively decoded the functional consequences of ER mutations on dysregulated ligand–receptor interactions across 33 cancers. The dysregulated ligand–receptor genes were indeed enriched in cancer and immune‐related function. We demonstrated the potential significance of PD1–PDL1 interaction‐related ER mutations in stratifying cancer patients from multiple independent data cohorts. The ER mutation group showed distinct immunological characterizations and prognoses. Furthermore, we highlighted that the ER mutations could potentially predict clinical outcomes of immunotherapy. Our computational and clinical assessment underscore the utility of MERIN for elucidating the functional relevance of ER mutations in cancer immune response, potentially aiding patients' stratification for immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1c94edf4edb1035d9676cb0af0e88e572eaf00" target='_blank'>
              Decoding functional impact of epigenetic regulator mutations on ligand–receptor interaction perturbations for evaluation of cancer immunotherapy
              </a>
            </td>
          <td>
            Aiai Shi, Chaohuan Lin, Jie Lyu
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew D Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background/Objectives: Irinotecan, a camptothecin (CPT) derivative, is commonly used as a first-line therapy for colorectal cancer (CRC), but resistance remains a significant challenge. This study aims to explore the therapeutic potential of FL118, another CPT derivative, with a focus on overcoming resistance to irinotecan. Methods: The effects of FL118 on CRC cells were evaluated, and bioinformatics analysis was performed on RNA-seq data. Transfection was conducted to observe the knockdown effect of survivin, and the in vivo efficacy of FL118 was assessed using a xenograft model. Results: FL118 induces apoptosis, G2/M arrest, and DNA damage. A notable mechanism of action of FL118 is a reduction in survivin levels, which downregulates the expression of RAD51, a key marker of homologous recombination, and attenuates DNA repair processes. Given that SN38 is the active metabolite of irinotecan, FL118 reduces cell viability and RAD51 in SN38-resistant LOVO cells. Conclusions: Our findings provide effective insights into the antitumor activity of FL118 and its potential as a therapeutic agent for overcoming irinotecan resistance in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d19093f191d9d1383ff5d9089dfc1860caa3a82" target='_blank'>
              FL118 Enhances Therapeutic Efficacy in Colorectal Cancer by Inhibiting the Homologous Recombination Repair Pathway through Survivin–RAD51 Downregulation
              </a>
            </td>
          <td>
            Jungyoun Kim, Yeyeong Jeong, You Me Shin, Sung Eun Kim, Sang Joon Shin
          </td>
          <td>2024-10-03</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the ever-evolving landscape of oncology, the battle against prostate cancer (PCa) stands at a transformative juncture, propelled by the integration of molecular diagnostics into traditional cytopathological frameworks. This synthesis not only heralds a new epoch of precision medicine but also significantly enhances our understanding of the disease’s genetic intricacies. Our comprehensive review navigates through the latest advancements in molecular biomarkers and their detection technologies, illuminating the potential these innovations hold for the clinical realm. With PCa persisting as one of the most common malignancies among men globally, the quest for early and precise diagnostic methods has never been more critical. The spotlight in this endeavor shines on the molecular diagnostics that reveal the genetic underpinnings of PCa, offering insights into its onset, progression, and resistance to conventional therapies. Among the genetic aberrations, the TMPRSS2-ERG fusion and mutations in genes such as phosphatase and tensin homolog (PTEN) and myelocytomatosis viral oncogene homolog (MYC) are identified as significant players in the disease’s pathology, providing not only diagnostic markers but also potential therapeutic targets. This review underscores a multimodal diagnostic approach, merging molecular diagnostics with cytopathology, as a cornerstone in managing PCa effectively. This strategy promises a future where treatment is not only tailored to the individual’s genetic makeup but also anticipates the disease’s trajectory, offering hope for improved prognosis and quality of life for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/040ce4884f4e6bdcb8041b33d1c1868319f07888" target='_blank'>
              From microscopes to molecules: The evolution of prostate cancer diagnostics
              </a>
            </td>
          <td>
            Junyue Tao, Xiaokang Bian, Jun Zhou, Meng Zhang
          </td>
          <td>2024-08-29</td>
          <td>Cytojournal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Non-coding regulatory sequences play essential roles in adjusting gene output to cellular needs and are thus critical to animal development and health. Numerous such sequences have been identified in mammalian genomes ranging from transcription factors binding motifs to recognition sites for RNA-binding proteins and non-coding RNAs. The advent of CRISPR has raised the possibility of assigning functionality to individual endogenous regulatory sites by facilitating the generation of isogenic cell lines that differ by a defined set of genetic modifications. Here we investigate the usefulness of this approach to assign function to individual miRNA binding sites. We find that the process of generating isogenic pairs of mammalian cell lines with CRISPR-mediated mutations introduces extensive molecular and phenotypic variability between biological replicates making any attempt of assigning function to the binding site essentially impossible. Our work highlights an important consideration when employing CRISPR editing to characterize non-coding regulatory sequences in cell lines and calls for the development and adoption of alternative strategies to address this question in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f1d3a055f759b974342766678fe87bc5f92b153" target='_blank'>
              CRISPR-based dissection of miRNA binding sites using isogenic cell lines is hampered by pervasive noise
              </a>
            </td>
          <td>
            Mahendra Prajapat, Joana A. Vidigal
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Abstract SF3B1 is the most recurrently mutated RNA splicing gene in cancer. However, research of its pathogenic role has been hindered by a lack of disease-relevant cell line models. Here, our study compared four genome engineering platforms to establish SF3B1 mutant cell lines: CRISPR-Cas9 editing, AAV homology-directed repair editing, base editing (ABEmax, ABE8e), and prime editing (PE2, PE3, PE5max). We showed that prime editing via PE5max achieved the most efficient SF3B1 K700E editing across a wide range of cell lines. Our approach was further refined by coupling prime editing with a fluorescent reporter that leverages a SF3B1 mutation-responsive synthetic intron to mark successfully edited cells. By applying this approach, called prime editing coupled intron-assisted selection (PRECIS), we introduced the K700E hotspot mutation into two chronic lymphocytic leukemia cell lines, HG-3 and MEC-1. We demonstrated that our PRECIS-engineered cells faithfully recapitulate known mutant SF3B1 phenotypes, including altered splicing, copy number variations, and cell-growth defect. Moreover, we discovered that the SF3B1 mutation can cause the loss of Y chromosome in chronic lymphocytic leukemia. Our results showcase that PRECIS is an efficient and generalizable method for engineering genetically faithful SF3B1 mutant models. Our approach provides new insights on the role of SF3B1 mutation in cancer and enables the generation of SF3B1 mutant cell lines in relevant cellular context. Significance: This study developed an approach that can reliably and efficiently engineer SF3B1 mutation into different cellular contexts, thereby revealing novel roles of SF3B1 mutation in driving aberrant splicing, clonal evolution, and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2ca0b3ac1820b19171ca9e6d9370b57efca6b03" target='_blank'>
              Engineering Oncogenic Hotspot Mutations on SF3B1 via CRISPR-Directed PRECIS Mutagenesis
              </a>
            </td>
          <td>
            Mikel Fernandez, Lei Yu, Qiong Jia, Xuesong Wang, Kevyn L Hart, Zhenyu Jia, Ren-Jang Lin, Lili Wang
          </td>
          <td>2024-08-28</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Elucidating the genetic architecture of DNA methylation (DNAm) is crucial for decoding the etiology of complex diseases. However, current epigenomic studies often suffer from incomplete coverage of methylation sites and the use of tissues containing heterogeneous cell populations. To address these challenges, we present a comprehensive human methylome atlas based on deep whole-genome bisulfite sequencing (WGBS) and whole-genome sequencing (WGS) of purified monocytes from 298 European Americans (EA) and 160 African Americans (AA) in the Louisiana Osteoporosis Study. Our atlas enables the analysis of over 25 million DNAm sites. We identified 1,383,250 and 1,721,167 methylation quantitative trait loci (meQTLs) in cis-regions for EA and AA populations, respectively, with 880,108 sites shared between ancestries. While cis-meQTLs exhibited population-specific patterns, primarily due to differences in minor allele frequencies, shared cis-meQTLs showed high concordance across ancestries. Notably, cis-heritability estimates revealed significantly higher mean values in the AA population (0.09) compared to the EA population (0.04). Furthermore, we developed population-specific DNAm imputation models using Elastic Net, enabling methylome-wide association studies (MWAS) for 1,976,046 and 2,657,581 methylation sites in EA and AA, respectively. The performance of our MWAS models was validated through a systematic multi-ancestry analysis of 41 complex traits from the Million Veteran Program. Our findings bridge the gap between genomics and the monocyte methylome, uncovering novel methylation-phenotype associations and their transferability across diverse ancestries. The identified meQTLs, MWAS models, and data resources are freely available at www.gcbhub.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02561d4c021777b32cc89ad698157b2f333e71dc" target='_blank'>
              An atlas of genetic effects on the monocyte methylome across European and African populations
              </a>
            </td>
          <td>
            Wanheng Zhang, Xiao Zhang, Chuan Qiu, Zichen Zhang, Kuan-Jui Su, Zhe Luo, Minghui Liu, Bingxin Zhao, Lang Wu, Qing Tian, Hui Shen, Chong Wu, Hong-Wen Deng
          </td>
          <td>2024-08-13</td>
          <td>medRxiv : the preprint server for health sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Human inborn errors of immunity (IEI) represent a diverse group of genetic disorders affecting the innate and/or adaptive immune system. Some IEI entities comprise defects in DNA repair factors, resulting in (severe) combined immunodeficiencies, bone marrow failure, predisposition to malignancies, and potentially result in radiosensitivity (RS). While other IEI subcategories such as common variable immunodeficiency (CVID) and immune dysregulation disorders also associate with lymphoproliferative and malignant complications, the occurrence of RS phenotypes in the broader IEI population is not well characterized. Nonetheless, identifying RS in IEI patients through functional testing is crucial to reconsider radiation-related therapeutic protocols and to improve overall patient management. This study aimed to investigate chromosomal RS in a diverse cohort of 107 IEI patients using the G0 cytokinesis-block micronucleus (MN) assay. Our findings indicate significant variability in RS across specific genetic and phenotypical subgroups. Severe RS was detected in all ataxia-telangiectasia (AT) patients, a FANCI deficient and ERCC6L2 deficient patient, but not in any other IEI patient included in this cohort. Age emerged as the single influencing factor for both spontaneous and radiation-induced MN yields, while the manifestation of additional clinical features, including infection susceptibility, immune dysregulation, or malignancies did not associate with increased MN levels. Our extensive analysis of RS in the IEI population underscores the clinical importance of RS assessment in AT patients and supports RS testing in all IEI patients suspected of having a DNA repair disorder associated with radiosensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b4f50595c5c29676228d24464dde5631290eb13" target='_blank'>
              Investigating chromosomal radiosensitivity in inborn errors of immunity: insights from DNA repair disorders and beyond.
              </a>
            </td>
          <td>
            E. Beyls, Evi Duthoo, Lynn Backers, Karlien Claes, Marieke De Bruyne RAPID clinicians, L. Pottie, Victoria Bordon, C. Bonroy, S. Tavernier, Kathleen BM Claes, A. Vral, A. Baeyens, F. Haerynck, T. Kerre, B. Lambrecht, L. Hoste, B. Meertens, T. V. Genechten, A. Ferster, K. V. Driessche, C. D. Vriendt, L. Dedeken, I. Moors, C. Dhooge, L. Kornreich, C. Heijmans, M. L. Roux, M. Colard, A. V. Damme, K. V. Schil, B. D. Wilde
          </td>
          <td>2024-08-20</td>
          <td>None</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Overexpression of the MYC oncogene, encoding c-MYC protein, contributes to the pathogenesis and drug resistance of acute myeloid leukemia (AML) and many other hematopoietic malignancies. Although standard chemotherapy has predominated in AML therapy over the past five decades, the clinical outcomes and patient response to treatment remain suboptimal. Deeper insight into the molecular basis of this disease should facilitate the development of novel therapeutics targeting specific molecules and pathways that are dysregulated in AML, including fms-like tyrosine kinase 3 (FLT3) gene mutation and cluster of differentiation 33 (CD33) protein expression. Elevated expression of c-MYC is one of the molecular features of AML that determines the clinical prognosis in patients. Increased expression of c-MYC is also one of the cytogenetic characteristics of drug resistance in AML. However, direct targeting of c-MYC has been challenging due to its lack of binding sites for small molecules. In this review, we focused on the mechanisms involving the bromodomain and extra-terminal (BET) and cyclin-dependent kinase 9 (CDK9) proteins, phosphoinositide-Akt-mammalian target of rapamycin (PI3K/AKT/mTOR) and Janus kinase-signal transduction and activation of transcription (JAK/STAT) pathways, as well as various inflammatory cytokines, as an indirect means of regulating MYC overexpression in AML. Furthermore, we highlight Food and Drug Administration (FDA)-approved drugs for AML, and the results of preclinical and clinical studies on novel agents that have been or are currently being tested for efficacy and tolerability in AML therapy. Overall, this review summarizes our current knowledge of the molecular processes that promote leukemogenesis, as well as the various agents that intervene in specific pathways and directly or indirectly modulate c-MYC to disrupt AML pathogenesis and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1135bf881c820d3eca90352e7790a726703df16e" target='_blank'>
              Targeting Molecular Signaling Pathways and Cytokine Responses to Modulate c-MYC in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Kyle Gu, Harry A. May, Min H. Kang
          </td>
          <td>2024-09-14</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Patients with breast cancer, especially triple-negative breast cancer, have a poor prognosis. There is still no effective treatment for this disease. Due to resistance to traditional treatments such as chemotherapy and radiation therapy, there is a need to discover novel treatment strategies to treat this disease. Ribociclib is a selective CDK4/6 inhibitor. Approximately 20% of patients with HR+ breast cancer developed primary resistance to CDK4/6 inhibitors, and more than 30% experienced secondary resistance. Since most patients experience resistance during CDK4/6 inhibitor treatment, managing this disease is becoming more challenging. Many malignant tumors abnormally express microRNA (miR)-141, which participates in several cellular processes, including drug resistance, proliferation, epithelial–mesenchymal transition, migration, and invasion. Materials and methods In the present study, we cultured MDA-MB-231 and MCF-7 cells in DMEM-F12 medium. By performing MTT assay we determined the cytotoxic effects of ribociclib on breast cancer cells, as well as determining the IC50 of it. Then, we treated the cells with ribociclib at two time points: 24 h and 72 h. After that, RNA was isolated and reverse transcribed to cDNA. Finally, we performed qRT‒PCR to evaluate how ribociclib affects the expression level of desired genes. Results and conclusion We found that ribociclib can inhibit cell growth in a dose- and time-dependent manner. We examined the mRNA expression of 4 genes. After ribociclib treatment, the mRNA expression of CDK6 and MYH10 decreased (p < 0.01, p < 0.05). The mRNA expression of CDON increased (p<0.05), but no significant changes were observed in ZEB1 mRNA expression. Furthermore, the qRT‒PCR results for miR-141 showed that the expression of miR-141 increased (p<0.01) after 72 h of treatment with ribociclib.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9af2742d79589071ef6d86e863e12d574976bef5" target='_blank'>
              The effect of ribociclib on the expression levels of miR-141 and CDK4/6-USP51 signaling pathway genes in MCF-7 and MDA-MB-231 cells
              </a>
            </td>
          <td>
            Shayeste Sadat Baghermanesh, Mahmoud Barati, Arshad Hosseini
          </td>
          <td>2024-08-28</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Although single cell RNA-sequencing (scRNA-seq) provides unprecedented insights into the biology of complex tissues, analyzing such data on a gene-by-gene basis is challenging due to the large number of tested hypotheses and consequent low statistical power and difficult interpretation. These issues are magnified by the increased noise, significant sparsity and multi-modal distributions characteristic of single cell data. One promising approach for addressing these challenges is gene set testing, or pathway analysis. Unfortunately, statistical and biological differences between single cell and bulk transcriptomic data make it challenging to use existing gene set collections, which were developed for bulk tissue analysis, on scRNA-seq data. In this paper, we describe a procedure for customizing gene set collections originally created for bulk tissue analysis to reflect the structure of gene activity within specific cell types. Our approach leverages information about mean gene expression in the 81 human cell types profiled via scRNA-seq by the Human Protein Atlas (HPA) Single Cell Type Atlas. This HPA information is used to compute cell type-specific gene and gene set weights that can be used to filter or weight gene set collections. As demonstrated through the analysis of immune cell scRNA-seq data using gene sets from the Molecular Signatures Database (MSigDB), accounting for cell type-specificity can significantly improve gene set testing power and interpretability. An example vignette along with gene and gene set weights for the 81 HPA SCTA cell types and the MSigDB collections are available at https://hrfrost.host.dartmouth.edu/SCGeneSetOpt/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09881903d0a088b514af64709adaa8454ee2f8e2" target='_blank'>
              Leveraging cell type-specificity for gene set analysis of single cell transcriptomics
              </a>
            </td>
          <td>
            H. R. Frost
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Lung cancer is a prevalent and severe form of malignant tumors worldwide. tRF-Leu-CAG, a recently discovered non-coding single-stranded small RNA derived from transfer RNA, has sparked interest in exploring its biological functions and potential molecular mechanisms in lung cancer.


METHODS
The abundance of tRF-Leu-CAG was measured via quantitative real-time polymerase chain reaction (qRT-PCR) in 96 sets of lung cancer tissue samples obtained from clinical patients. Subsequently, both in vivo and in vitro experiments were conducted to validate the biological functions of tRF-Leu-CAG in lung cancer. Furthermore, an exploration of the potential target genes of tRF-Leu-CAG and its association with autophagy and drug resistance in lung cancer was undertaken.


RESULTS
Our analysis revealed a significant upregulation of tRF-Leu-CAG in non-small cell lung cancer (NSCLC) tissues. Additionally, we observed that heightened expression of tRF-Leu-CAG significantly augmented the proliferation and migration of NSCLC cells, facilitated cell cycle progression, and suppressed apoptosis. Furthermore, we identified transcription elongation factor A3 (TCEA3) as a direct target gene of tRF-Leu-CAG. TCEA3 inhibited the proliferation and migration of NSCLC, and tRF-Leu-CAG promoted the proliferation and migration of NSCLC by mediating the silencing of TCEA3. Moreover, we demonstrated that the augmentation of paclitaxel resistance by tRF-Leu-CAG was contingent on autophagy. Finally, tRF-Leu-CAG notably accelerated tumor growth and promoted the process of epithelial-mesenchymal transition (EMT) in vivo.


CONCLUSIONS
tRF-Leu-CAG promotes NSCLC tumor growth and metastasis by targeting TCEA3 and promotes paclitaxel resistance by enhancing cellular autophagy. These results provide potentially effective targets and therapeutic options for the clinical treatment of NSCLC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9a8cbdbaa838f4eb2da1ce3af02d5ba168916d9" target='_blank'>
              Oncogenic tRNA-derived fragment tRF-Leu-CAG promotes tumorigenesis of lung cancer via targeting TCEA3 and increasing autophagy.
              </a>
            </td>
          <td>
            Fan Wu, Binshu Chai, Pengfei Qi, Yaqi Han, Z. Gu, Weikang Pan, Hui Zhang, Xianyi Wang, Xiaomin Liu, Heng Zou, Chen Liang, Yanli Li, Wen-xu Fang, Zhongliang Ma
          </td>
          <td>2024-08-28</td>
          <td>The journal of gene medicine</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/431b8d9fdd349b2296cea4bbf8173c562f2afaeb" target='_blank'>
              Comprehensive molecular profiling of multiple myeloma identifies refined copy number and expression subtypes
              </a>
            </td>
          <td>
            Sheri Skerget, Daniel A. Peñaherrera, A. Chari, Sundar Jagannath, David Siegel, Ravi Vij, G. Orloff, A. Jakubowiak, R. Niesvizky, Darla Liles, J. Berdeja, Moshe Levy, J. Wolf, S. Usmani, Austin Christofferson, Sara Nasser, J. Aldrich, Christophe Legendre, Brooks A. Benard, Chase Miller, Bryce Turner, Ahmet A. Kurdoglu, M. Washington, Venkata Yellapantula, J. Adkins, Lori Cuyugan, Martin Boateng, A. Helland, Shari Kyman, Jackie McDonald, Rebecca Reiman, Kristi Stephenson, Erica E Tassone, A. Blanski, Brianne Livermore, Meghan Kirchhoff, D. Rohrer, M. D'Agostino, Manuela Gamella, K. Collison, Jennifer Stumph, Pam Kidd, A. Donnelly, B. Zaugg, Maureen Toone, K. McBride, M. Derome, Jennifer Rogers, David Craig, Winnie S Liang, N. Gutiérrez, Scott D. Jewell, J. Carpten, Kenneth C Anderson, H. Cho, D. Auclair, S. Lonial, J. Keats
          </td>
          <td>2024-08-19</td>
          <td>Nature Genetics</td>
          <td>1</td>
          <td>108</td>
        </tr>

        <tr id="Abstract Disruptions in normal development and the emergence of health conditions often result from the malfunction of vital genes in the human body. Decades of scientific research have focused on techniques to modify or substitute defective genes with healthy alternatives, marking a new era in disease treatment, prevention and cure. Recent strides in science and technology have reshaped our understanding of disorders, medication development and treatment recommendations, with human gene and cell therapy at the forefront of this transformative shift. Its primary objective is the modification of genes or adjustment of cell behaviour for therapeutic purposes. In this review, we focus on the latest advances in gene and cell therapy for treating human genetic diseases, with a particular emphasis on FDA and EMA‐approved therapies and the evolving landscape of genome editing. We examine the current state of innovative gene editing technologies, particularly the CRISPR‐Cas systems. As we explore the progress, ethical considerations and prospects of these innovations, we gain insight into their potential to revolutionize the treatment of genetic diseases, along with a discussion of the challenges associated with their regulatory pathways. This review traces the origins and evolution of these therapies, from conceptual ideas to practical clinical applications, marking a significant milestone in the field of medical science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/605617113766e025c90c18b7b7452f93c13a0842" target='_blank'>
              Gene and cell therapy of human genetic diseases: Recent advances and future directions
              </a>
            </td>
          <td>
            Busra Cetin, Fulya Erendor, Y. E. Eksi, A. Şanlioğlu, S. Sanlioglu
          </td>
          <td>2024-09-01</td>
          <td>Journal of Cellular and Molecular Medicine</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="BACKGROUND
The clinical management of chronic myeloid leukemia (CML) patients requires the identification of the type of BCR::ABL1 transcript at diagnosis and the monitoring of its expression and potential tyrosine kinase inhibitor (TKI) resistance mutations during treatment. Detection of resistant mutation requires transcript type-specific amplification of BCR::ABL1 from RNA.


METHODS
In this study, a custom RNA-based next-generation sequencing (NGS) assay (Dup-Seq BCR::ABL1) that enables (a) the identification of BCR::ABL1 transcript type and (b) the detection of resistance mutations from common and atypical BCR::ABL1 transcript types was developed and validated. The assay design covers BCR exon 1 to ABL1 exon 10 and employs duplicate PCR amplification for error correction. The custom data analysis pipeline enables breakpoint determination and overlapped mutation calling from duplicates, which minimizes the low-level mutation artifacts.


RESULTS
This study demonstrates that this novel assay achieves high accuracy (positive percent agreement (PPA) for fusion: 98.5%; PPA and negative percent agreement (NPA) for mutation at 97.8% and 100.0%, respectively) and sensitivity (limit of detection (LOD) for mutation detection at 3% from 10 000 copies of BCR::ABL1 input).


CONCLUSIONS
The Dup-Seq BCR::ABL1 assay not only allows for the identification of BCR::ABL1 typical and atypical transcript types and accurate and sensitive detection of TKI-resistant mutations but also simplifies molecular testing work flow for the clinical management of CML patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c413e19fd7d542721e914a796a2298f73b3d650" target='_blank'>
              A Novel Next-Generation Sequencing Assay for the Identification of BCR::ABL1 Transcript Type and Accurate and Sensitive Detection of TKI-Resistant Mutations.
              </a>
            </td>
          <td>
            Zhenyu Yan, Lin Shi, Wei Li, Weihua Liu, C. Galderisi, Cynthia Spittle, Jin Li
          </td>
          <td>2024-09-03</td>
          <td>The journal of applied laboratory medicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0943aff1f710e095be6a9439cd65322a1fc6aa0e" target='_blank'>
              Genomic Balancing Act: deciphering DNA rearrangements in the complex chromosomal aberration involving 5p15.2, 2q31.1, and 18q21.32.
              </a>
            </td>
          <td>
            Zain Dardas, Dana Marafi, Ruizhi Duan, Jawid M Fatih, O. El-Rashidy, Christopher M. Grochowski, Claudia M B Carvalho, S. Jhangiani, W. Bi, Haowei Du, Richard A Gibbs, J. Posey, D. Calame, Maha S. Zaki, James R. Lupski
          </td>
          <td>2024-09-10</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9323b395426f00928696c5f63d0f88e7dffceb5" target='_blank'>
              The evolving genetic landscape of telomere biology disorder dyskeratosis congenita.
              </a>
            </td>
          <td>
            H. Tummala, A. Walne, M. Badat, Manthan Patel, Abigail M Walne, Jenna Alnajar, Chi Ching Chow, Ibtehal Albursan, J. Frost, David Ballard, Sally Killick, Peter Szitányi, Anne M Kelly, Manoj Raghavan, Corrina Powell, Reinier A P Raymakers, Tony Todd, Elpis Mantadakis, Sophia Polychronopoulou, N. Pontikos, Tianyi Liao, Pradeepa M. Madapura, Upal Hossain, T. Vulliamy, I. Dokal
          </td>
          <td>2024-08-28</td>
          <td>EMBO molecular medicine</td>
          <td>1</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88cace8d561b58e28d3ee8aa2930fd2c3ef573b1" target='_blank'>
              Loss of EMI1 compromises chromosome stability and is associated with cellular transformation in colonic epithelial cell contexts.
              </a>
            </td>
          <td>
            Rubi Campos Gudiño, Nicole M. Neudorf, Demi Andromidas, Zelda Lichtensztejn, K. McManus
          </td>
          <td>2024-10-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c12c760f08b0716217f59b590788ec67fee6b6e" target='_blank'>
              Double-negative T cells with a distinct transcriptomic profile are abundant in the peripheral blood of patients with breast cancer.
              </a>
            </td>
          <td>
            Hui-Ru Zhu, Yun-Bo Wei, Jia-Qi Guo, Xiao-Fei Liu
          </td>
          <td>2024-09-10</td>
          <td>Breast cancer research and treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Multiple Myeloma (MM) is an incurable malignancy characterised by altered expression of coding and non-coding genes promoting tumour growth and drug resistance. Although the crucial role of long non-coding RNAs (lncRNAs) in MM is clearly established, the function of the non-coding RNAome, which might allow the design of novel therapeutics, is largely unknown. We performed an unbiased CRISPR-Cas9 loss-of-function screen of 671 lncRNAs in MM cells and their Bortezomib (BZB)-resistant derivative. To rank functionally and clinically relevant candidates, we designed and used a bioinformatic prioritisation pipeline combining functional data from cellular screens with prognostic and transcriptional data from MM patients. With this approach, we unveiled and prioritised 8 onco-lncRNAs essential for MM cell fitness, associated with high expression and poor prognosis in MM patients. The previously uncharacterised RP11-350G8.5 emerged as the most promising target, irrespective of BZB resistance. We i) demonstrated the anti-tumoral effect obtained by RP11-350G8.5 inhibition in vitro and in vivo; ii) highlighted a modulation of the unfolded protein response and the induction of immunogenic cell death triggered by the RP11-350G8.5 knock-out, via RNA-sequencing and molecular studies; iii) characterised its cytoplasmic homing through RNA-FISH; iv) predicted its 2D structure and identified 2 G-quadruplex and 3 hairpin-forming regions by biophysical assays, including Thioflavin T, 1H-NMR and circular dichroism to pave the way to the development of novel targeted therapeutics. Overall we provided innovative insights about unexplored lncRNAs in MM and identified RP11-350G8.5 as an oncogenic target for treatment-naïve and BZB-resistant MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b3038ab25be0410e0d1aec773f7b5564cf954f8" target='_blank'>
              An unbiased lncRNAs dropout CRISPR-Cas9 screen reveals RP11-350G8.5 as a novel therapeutic target for Multiple Myeloma.
              </a>
            </td>
          <td>
            K. Grillone, S. Ascrizzi, Paolo Cremaschi, Jussara Amato, N. Polerà, Ottavio Croci, R. Rocca, C. Riillo, Francesco Conforti, Raffaele Graziano, Diego Brancaccio, Daniele Caracciolo, Stefano Alcaro, Bruno Pagano, Antonio Randazzo, P. Tagliaferri, F. Iorio, P. Tassone
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="
 Personalized medicine programs, including the Zero Childhood Cancer Program (ZERO), perform molecular analysis of individual patient tumours to identify molecular targets for therapy. The comprehensive and unbiased sequencing approaches used by ZERO, including whole genome sequencing (WGS) and RNA sequencing (RNAseq), enables detection of novel structural variants (SVs) that may be cryptic to standard cytogenetic techniques or panel sequencing approaches. Identification of Receptor Tyrosine Kinase (RTK) activating variants in paediatric cancers, for example NTRK2 gene fusions, is a high priority given the availability and remarkable clinical success of RTK inhibitors. Here we describe a novel internal tandem duplication (ITD) in NTRK2, encoding TRKB, in a ZERO patient with CNS neuroblastoma. This SV was detected by WGS and was notably missed by RNAseq-based gene fusion and SV detection. The ITD spans exons 10-13 of NTRK2, is in-frame, and includes the juxtamembrane and transmembrane protein domains. We hypothesized that this ITD would result in constitutive activation of TRKB and would be sensitive to TRK-targeted therapies. We overexpressed NTRK2 ITD, wild-type NTRK2, and SPECC1L-NTRK2 (an established TRK-activating fusion), in the interleukin-3 (IL-3)-dependent cell line, Ba/F3, and the neuroblastoma cell line, SH-SY5Y. In these models, we showed that like wild-type NTRK2, NTRK2 ITD was expressed on the cell surface. NTRK2 ITD was sufficient to transform Ba/F3 cells to IL-3 independence through constitutive activation of TRKB and downstream signaling pathways PI3K/AKT and MEK/ERK. NTRK2 ITD-expressing cells were sensitive to TRK inhibitors, including larotrectinib, at similar doses to SPECC1L-NTRK2-expressing cells. Interestingly, we observed that cells expressing NTRK2 ITD were specifically sensitive to MEK inhibition, which was not observed in SPECC1L-NTRK2-expressing cells, suggesting the mechanisms by which these variants drive transformation is different. Indeed, in silico structural analysis showed that the duplicated region of NTRK2 ITD contains two internalized tyrosine residues whose phosphorylation could both drive autophosphorylation of the receptor and act as docking sites for adaptor proteins. Mutation of these tyrosine residues delayed transformation of Ba/F3 cells. This study functionally characterizes a novel NTRK2 ITD and shows that this variant is transforming and sensitive to TRK inhibition. While ITDs have been described in other RTK genes, notably FGFR1 and FLT3, structurally these have involved the kinase domain or the juxtamembrane domain. This is the first report of an ITD that spans the transmembrane domain of an RTK, characterizing an additional mechanism by which RTKs can be activated in cancer. This study highlights the value of unbiased WGS approaches to identify novel SVs and identifies another class of patients that may benefit from RTK inhibitor therapy.
 Citation Format: Lauren M. Brown, Gabor Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah Jalud, Teresa Sadras, Pamela Ajuyah, Paulette Barahona, M. Emmy M. Dolman, Marie Wong, Chelsea Mayoh, Mark J. Cowley, Paul G. Ekert. A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B002.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ae609e3759161918ab8221b777b148ec5605447" target='_blank'>
              Abstract B002: A novel NTRK2-activating internal tandem duplication characterizes a new mechanism of receptor tyrosine kinase activation
              </a>
            </td>
          <td>
            Lauren M. Brown, G. Tax, Pablo Acera Mateos, Antoine de Weck, Steve Foresto, Fatimah B Jalud, Teresa Sadras, P. Ajuyah, P. Barahona, M. E. M. Dolman, Marie Wong, C. Mayoh, Mark J. Cowley, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="CRISPR-based genome engineering holds enormous promise for basic science and therapeutic applications. Integrating and editing DNA sequences is still challenging in many cellular contexts, largely due to insufficient control of the repair process. We find that repair at the genome-cargo interface is predictable by deep-learning models and adheres to sequence context specific rules. Based on in silico predictions, we devised a strategy of triplet base-pair repeat repair arms that correspond to microhomologies at double-strand breaks (trimologies), which facilitated integration of large cargo (>2 kb) and protected the targeted locus and transgene from excessive damage. Successful integrations occurred in >30 loci in human cells and in in vivo models. Germline transmissible transgene integration in Xenopus, and endogenous tagging of tubulin in adult mice brains demonstrated integration during early embryonic cleavage and in non-dividing differentiated cells. Further, optimal repair arms for single- or double nucleotide edits were predictable, and facilitated small edits in vitro and in vivo using oligonucleotide templates. We provide a design-tool (Pythia, pythia-editing.org) to optimize custom integration, tagging or editing strategies. Pythia will facilitate genomic integration and editing for experimental and therapeutic purposes for a wider range of target cell types and applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67ad57fb5d6450173260d95ec721d1be7ca9efcc" target='_blank'>
              Pythia: Non-random DNA repair allows predictable CRISPR/Cas9 integration and gene editing
              </a>
            </td>
          <td>
            Thomas Naert, Taiyo Yamamoto, Shuting Han, Melanie Horn, Phillip Bethge, Nikita Vladimirov, F. Voigt, Joana Figueiro-Silva, Ruxandra Bachmann-Gagescu, F. Helmchen, S. Lienkamp
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="
 Children born with Down syndrome (DS), a genetic condition caused by the presence of an additional chromosome 21 (trisomy 21 or T21), have a substantially higher risk of developing childhood leukaemia. Approximately 30% of DS infants develop a preleukemic condition called transient abnormal myelopoiesis (TAM). TAM is strictly associated with GATA1 truncating mutations on a T21 background. Most TAM cases spontaneously resolve within the first few months of life. However, by the age of five, approximately 10% of cases progress to megakaryoblastic/erythroid leukaemia, known as myeloid leukaemia of DS (ML-DS). This final step of transformation is often driven by specific third-hit somatic mutations, most of which affects genes encoding JAK kinases, cohesin complexes, or epigenetic regulators. These well-defined stepwise genetic aberrations associated with ML-DS pathogenesis provide a unique and highly tractable system to elucidate the molecular changes underlying leukaemogenesis. Using single-cell mRNA sequencing of primary human samples, we directly interrogate the transcriptional changes underpinning the multi-step pathogenesis of ML-DS. We observe an expansion of megakaryocyte-erythrocyte progenitors (MEPs) specifically in T21 foetal liver compared to diploid or other trisomies. However, it is the GATA1 mutation that accounts for the majority of the transcriptomic changes towards that of the leukaemic blasts. Furthermore, investigation of an aggressive TAM case reveals an enrichment in expression of the leukaemogenic gene module associated with the full-blown leukaemia state ML-DS. We also demonstrate that TAM and ML-DS blasts still retain transcriptional signatures of normal haematopoiesis, showing heterogeneous features of differentiation towards erythrocytes, megakaryocytes, and mast cells. Finally, by integrating genomic and transcriptomic data, we directly defined the genomic evolution, and its corresponding transcriptional consequences, that underpins a treatment-refractory case of ML-DS. Overall, we have generated the first comprehensive single-cell mRNA atlas of TAM / ML-DS, enabling unbiased quantitative characterisation of the transcriptional transformation along ML-DS pathogenesis. Our study explores the molecular signature associated with progression of TAM to ML-DS, as well as the mechanism underlying differences in treatment response. These insights offer the possibility of more meaningful clinical interpretations and potential to explore novel therapeutic targets for these conditions.
 Citation Format: Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, Angus Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, Jan-Henning Klusmann, Jack Bartram, Sam Behjati. Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A018.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f4dde5c53348eeefdb9808ec633733f445cffc" target='_blank'>
              Abstract A018 Transcriptional evolution from normal foetal haematopoiesis to myeloid leukaemia in Down syndrome
              </a>
            </td>
          <td>
            Mi K. Trinh, Matthew D. Young, Conor Parks, Agnes Oszlanczi, Toochi Ogbonnah, Di Zhou, A. Hodder, Konstantin Schuschel, Hasan Issa, Laura Jardine, J. Klusmann, Jack Bartram, Sam Behjati
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88f0f651c9fac9c8c40b22fc5b21b4578646df27" target='_blank'>
              Multi-omics features of immunogenic cell death in gastric cancer identified by combining single-cell sequencing analysis and machine learning
              </a>
            </td>
          <td>
            Shu-Long Dai, Jian-Qiang Pan, Zhen-Rong Su
          </td>
          <td>2024-09-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ba40a48a9694556eef4d0d290c4702c1cf12dc5" target='_blank'>
              Spatially resolved analysis of pancreatic cancer identifies therapy-associated remodeling of the tumor microenvironment.
              </a>
            </td>
          <td>
            C. Shiau, Jingyi Cao, Dennis Gong, M. Gregory, Nicholas J Caldwell, Xunqin Yin, Jae-Won Cho, Peter L. Wang, J. Su, Steven Wang, J. Reeves, T. Kim, Youngmi Kim, Jimmy A. Guo, N. A. Lester, Jung Woo Bae, Ryan Zhao, Nathan Schurman, J. L. Barth, M. L. Ganci, Ralph Weissleder, Tyler Jacks, M. Qadan, Theodore S. Hong, J. Y. Wo, Hannah J Roberts, J. Beechem, Carlos Fernández-del Castillo, M. Mino-Kenudson, David T Ting, M. Hemberg, W. L. Hwang
          </td>
          <td>2024-09-03</td>
          <td>Nature genetics</td>
          <td>2</td>
          <td>50</td>
        </tr>

        <tr id="Introduction: The role of genetic variants in response to chemotherapy has been investigated in several studies. This study aimed to investigate genetic variants associated with response to chemotherapy in breast cancer (BC) patients. Methods: Significant variants (p < 5 × 10-8) associated with response to chemotherapy were obtained from GWA studies. Candidate variants were identified by haplotype analysis (r2 ≥ 0.9, D'≥0.9) using 1000Genome LD data. To determine the effects of the variants on gene expression, expression quantitative trait loci (eQTL) were evaluated. To compare the expression of the identified genes in tumor samples, expression levels were compared between TCGA tumor types and adjacent normal tissues. Results: Six rs3820706, rs147451859, rs4784750, rs17587029, rs16830728, and rs16972207 variants were significantly associated with response to chemotherapy in BC patients (p < 5 × 10-8). Seven novel haplotypic structures were identified to be associated with adverse response to chemotherapy in BC patients. These haplotypes formed two genetic structures associated with neutropenia, leukopenia, chemotherapy-induced cytotoxicity (GAG-TTAT), and chemotherapy-induced alopecia (CC-CAACTCCCGTTGCGG). These variants are located on PPCDC, NLRC5, STAM2, and TNFSF13B genes, and the expression of these genes significantly changed in BC tissues than normal tissues (P ≤ 0.05), also showing gene-gene correlation (P ≤ 0.05). Conclusions: These genetic variants and their associated novel haplotypic structures can predict adverse response to chemotherapy in BC patients and could potentially form BC-associated genetic panel for adverse response to chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f6f813884867edf6f2b79ca7de2ac3b63826948" target='_blank'>
              Novel genetic structures associated with adverse response to chemotherapy in breast cancer.
              </a>
            </td>
          <td>
            Morteza Gholami, Mohsen Asouri, A. Ahmadi, Mehrab Nasirikenari
          </td>
          <td>2024-08-28</td>
          <td>Journal of oncology pharmacy practice : official publication of the International Society of Oncology Pharmacy Practitioners</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c25293ccda82c6d001fe5bae96c805b5e1fa7f9a" target='_blank'>
              The role of DNA polymerase I in tolerating single-strand breaks generated at clustered DNA damage in Escherichia coli
              </a>
            </td>
          <td>
            N. Shikazono, Ken Akamatsu
          </td>
          <td>2024-08-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Sickle cell disease (SCD) is a hereditary blood disorder characterized by the presence of abnormal hemoglobin molecules and thus distortion (sickling) of the red blood cells. SCD causes chronic pain and organ damage and shortens life expectancy. Gene therapy emerges as a potentially curative approach for people with SCD who lack a matched sibling donor for hematopoietic stem cell transplantation. Here, we review recent progress in gene therapy for SCD and focus on innovative technologies that target the genetic roots of the disease. We also review the challenges associated with gene therapy, including oncogenic risks, and the need for refined delivery methods. Despite these hurdles, the rapidly evolving landscape of gene therapy for SCD raises hope for a paradigm shift in the treatment of this debilitating disease. As research progresses, a deeper understanding of the molecular mechanisms involved and continuous improvements in gene-editing technologies promise to bring gene therapy for SCD closer to mainstream clinical application, offering a transformative, curative option for patients with this genetic disorder.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6e785d7cbf41248336e372c5a5fb5eb1bf4e0c" target='_blank'>
              Treating Sickle Cell Disease: Gene Therapy Approaches.
              </a>
            </td>
          <td>
            M. Cavazzana, Alice Corsia, Mégane Brusson, A. Miccio, Michaela Semeraro
          </td>
          <td>2024-09-11</td>
          <td>Annual review of pharmacology and toxicology</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Hepatocellular adenomas (HAs) are tumors that can develop under different conditions, including in patients harboring a germline mutation in HNF1A. However, little is known about the pathogenesis of such disease. This work aims to better define what mechanisms lie under the development of this condition. Six HAs were sampled from the liver of a 17-year-old male affected by diabetes and multiple hepatic adenomatosis harboring the heterozygous pathogenic germline variant c.815G>A, p.(Arg272His) in HNF1A, which has a dominant negative effect. All HAs were molecularly characterized. Four of them were shown to harbor a second somatic HNF1A variant and one had a mutation in the ARID1A gene, while no additional somatic changes were found in the remaining HA and normal parenchyma. A transcriptomic profile of the same HA samples was also performed. HNF1A biallelic mutations were associated with the up-regulation of several pathways including the tricarboxylic acid cycle, the metabolism of fatty acids, and mTOR signaling while angiogenesis, endothelial and vascular proliferation, cell migration/adhesion, and immune response were down-regulated. Contrariwise, in the tumor harboring the ARID1A variant, angiogenesis was up-modulated while fatty acid metabolism was down-modulated. Histological analyses confirmed the molecular data. Independently of the second mutation, energetic processes and cholesterol metabolism were up-modulated, while the immune response was down-modulated. This work provides a complete molecular signature of HNF1A-associated HAs, analyzing the association between specific HNF1A variants and the development of HA while identifying potential new therapeutic targets for non-surgical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26474672d542f89d0fa38cb2c1556ad93882621f" target='_blank'>
              Genomic and Transcriptomic Profile of HNF1A-Mutated Liver Adenomas Highlights Molecular Signature and Potential Therapeutic Implications
              </a>
            </td>
          <td>
            A. Faini, F. Arruga, M. Pinon, Valeria Bracciamà, Francesco Edoardo Vallone, Fiorenza Mioli, M. Sorbini, Martina Migliorero, A. Gambella, Damiano Carota, Isaac Giraudo, P. Cassoni, Silvia Catalano, Renato Romagnoli, Antonio Amoroso, P. L. Calvo, T. Vaisitti, Silvia Deaglio
          </td>
          <td>2024-09-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The family of LINE1 transposable elements underwent a massive expansion in mammalian genomes. While traditionally viewed as a mutagenic selfish element, recent studies point to roles for LINE1 in early mouse development, T cell quiescence and neurogenesis. Here we show that human LINE1 RNA is essential for self-renewal and identity of human embryonic stem cells (hESCs). Silencing of LINE1 using either antisense oligonucleotides or CRISPR interference in naïve hESCs leads to a strong induction of 8C-like cells (8CLCs). We found that genes derepressed upon LINE1 KD are not uniformly distributed across the genome, with an enrichment for chromosome 19, which includes key markers of the 8C state such as TPRX1. Silencing of TPRX1, but not other putative 8C regulators p53 or H3.XY, suppresses the induction of the 8C program in LINE1 KD hESCs. We found that LINE1 RNA is preferentially localized to the lamina and periphery of the nucleolus in hESCs. Sequencing of Lamina-Associated Domains (LADs) and Nucleolus-Associated Domains (NADs) reveals a preferential association of chromosome 19 with NADs in hESCs. However, 8CLCs have a distinct nucleolar morphology and a lower association of chromosome 19 and TPRX1 loci with the nucleolus relative to naïve and primed hESCs, suggesting a role for nucleolar dynamics in the 8CLC-hESC transition. In agreement, LINE1 KD leads to disruption of nucleolar architecture with signs of nucleolar stress. Independent perturbations of the nucleolus induce the 8C program in hESCs. Genes induced by LINE1 KD are enriched for targets of Polycomb Repressive Complex (PRC2), and inhibition of PRC2 leads to a strong induction of 8C genes. Our results indicate that LINE1 coordinates nuclear compartmentalization and chromatin-mediated gene repression to prevent developmental reversion of hESCs. Highlights Knockdown of LINE1 induces TPRX1-dependent emergence of 8C-like cells in hESCs. Genes de-repressed upon LINE1 KD are enriched for Chr 19 and PRC2 targets. 8CLCs display dissociation of Chromosome 19 from the nucleolus. Disruption of the nucleolus or inhibition of PRC2 strongly induce the 8C program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34781a9454759e1cbbdf47b8d98d4f390cccbdd4" target='_blank'>
              LINE1 promotes nuclear compartmentalization to repress the 8-cell state in embryonic stem cells
              </a>
            </td>
          <td>
            Juan Zhang, Lamisa Ataei, Liang Wu, Kirti Mittal, Linh Huynh, Shahil Sarajideen, Abdul Mazid, David P. Cook, D. Trcka, Kevin Tse, Jeffrey L. Wrana, Michael M. Hoffman, Miguel A Esteban, M. Ramalho-Santos
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Retrotransposons are invasive genetic elements, which replicate by copying and pasting themselves throughout the genome in a process called retrotransposition. The most abundant retrotransposons by number in the human genome are Alu and LINE-1 elements, which comprise approximately 40% of the human genome. The ability of retrotransposons to expand and colonize eukaryotic genomes has rendered them evolutionarily successful and is responsible for creating genetic alterations leading to significant impacts on their hosts. Previous research suggested that hypomethylation of Alu and LINE-1 elements is associated with global hypomethylation and genomic instability in several types of cancer and diseases, such as neurodegenerative diseases, obesity, osteoporosis, and diabetes mellitus (DM). With the advancement of sequencing technologies and computational tools, the study of the retrotransposon’s association with physiology and diseases is becoming a hot topic among researchers. Quantifying Alu and LINE-1 methylation is thought to serve as a surrogate measurement of global DNA methylation level. Although Alu and LINE-1 hypomethylation appears to serve as a cellular senescence biomarker promoting genomic instability, there is sparse information available regarding their potential functional and biological significance in DM. This review article summarizes the current knowledge on the involvement of the main epigenetic alterations in the methylation status of Alu and LINE-1 retrotransposons and their potential role as epigenetic markers of global DNA methylation in the pathogenesis of DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07f55c13ed59ce933c2fb2995ef34b1c1fa76b5" target='_blank'>
              Retrotransposons and Diabetes Mellitus
              </a>
            </td>
          <td>
            Andromachi Katsanou, Charilaos Kostoulas, E. Liberopoulos, A. Tsatsoulis, I. Georgiou, S. Tigas
          </td>
          <td>2024-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Genome-wide CRISPR screens have emerged as powerful tools for uncovering the genetic underpinnings of diverse biological processes. Incisive screens often depend on directly measuring molecular phenotypes, such as regulated gene expression changes, provoked by CRISPR-mediated genetic perturbations. Here, we provide quantitative measurements of transcriptional responses in human cells across genome-scale perturbation libraries by coupling CRISPR interference (CRISPRi) with barcoded expression reporter sequencing (CiBER-seq). To enable CiBER-seq in mammalian cells, we optimize the integration of highly complex, barcoded sgRNA libraries into a defined genomic context. CiBER-seq profiling of a nuclear factor kappa B (NF-κB) reporter delineates the canonical signaling cascade linking the transmembrane TNF-alpha receptor to inflammatory gene activation and highlights cell-type-specific factors in this response. Importantly, CiBER-seq relies solely on bulk RNA sequencing to capture the regulatory circuit driving this rapid transcriptional response. Our work demonstrates the accuracy of CiBER-seq and its potential for dissecting genetic networks in mammalian cells with superior time resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f33a5d37dba6ea92909caf57b17709293f40346" target='_blank'>
              CRISPRi with barcoded expression reporters dissects regulatory networks in human cells
              </a>
            </td>
          <td>
            Jinyoung Kim, Ryan Y Muller, Eliana R. Bondra, Nicholas T. Ingolia
          </td>
          <td>2024-09-06</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>50</td>
        </tr>

        <tr id="Background Alterations in DNA damage repair genes in advanced prostate cancer (PC) may impact responses to therapy and clinical outcomes. This study described homologous recombination repair (HRR) testing patterns and clinical outcomes among patients with metastatic castration-resistant prostate cancer (mCRPC) by HRR alteration status and race in the United States (US). Methods Clinical data in the nationwide (US-based) Flatiron Health-Foundation Medicine, Inc. (FMI) Metastatic PC Clinico-Genomic Database were evaluated (01/01/2011–12/31/2022). Patients initiating first-line (1L) mCRPC therapy on or after mCRPC diagnosis were included. Testing patterns, time-to-next treatment, overall survival (OS), and time-to-prostate specific antigen response were described. Results Of the 1367 patients with mCRPC and at least one HRR panel test prior to or on the date of 1L mCRPC therapy initiation, 332 (24.3%) were HRR positive (White patients: n = 219 [66.0%]; Black patients: n = 37 [11.1%]) and 1035 (75.7%) were HRR negative (White patients: n = 702 [67.8%]; Black patients: n = 84 [8.1%]). The mean time between first positive test and 1L mCRPC therapy initiation date was 588 days (White patients: 589 days; Black patients: 639 days). Among HRR positive relative to negative patients, trends for faster progression (respective 12-month rate overall: 71.1% and 63.7%; White patients: 72.5% and 64.0%; Black patients: 65.4% and 56.4%), shorter OS (respective 24-month rate overall: 46.8% and 51.9%; White patients: 48.6% and 46.2%; Black patients: 52.8% and 54.1%), and decreased treatment response (respective 12-month rate overall: 24.3% and 37.9%; White patients: 24.5% and 35.2%; Black patients: 17.0% and 43.9%) were observed. Conclusion Patients with mCRPC positive for HRR alterations tended to exhibit poorer treatment responses and clinical outcomes than those with a negative status. These findings highlight the importance of timely genetic testing in mCRPC, particularly among Black patients, and the need for improved 1L targeted therapies to address the unmet need in HRR positive mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42b5b000d1e3f13e0b5a22c9691a361f7dc5edd2" target='_blank'>
              Homologous Recombination Repair Testing Patterns and Outcomes in mCRPC by Alteration Status and Race
              </a>
            </td>
          <td>
            M. Bilen, I. Khilfeh, C. Rossi, Erik Muser, L. Morrison, A. Hilts, Lilian Diaz, Patrick Lefebvre, D. Pilon, Daniel George
          </td>
          <td>2024-09-01</td>
          <td>ClinicoEconomics and Outcomes Research: CEOR</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="The concept of cancer stemness is undergoing rapid evolution, facilitated by technological advances allowing the tracking and elimination of stem cells in vivo. Following their identification in solid tumors, cancer stem cells were placed at the center of tumor initiation, growth and metastasis. However, increasing evidence suggests that stemness is a cellular state that can be acquired by differentiated cells in a process called plasticity. Here we show that CD44v6 may act as a molecular switch controlling plasticity of colorectal cancer cells. CD44v6/MET signaling controls the reappearance of Lgr5+ cells after ablation, in vitro and in vivo as demonstrated in tumor organoids derived from Lgr5DTR/eGFP mice using a CD44v6 blocking peptide. CD44v6 also affects the YAP/TAZ signaling pathway, involved in the very first steps of plasticity. In view of the essential role of plasticity for the establishment of metastases, blocking CD44v6 signaling may represent a pivotal and promising therapy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04cb2eb85c74641a51555dede7b20eb7d8aa317e" target='_blank'>
              Breaking the cycle: How targeting CD44v6/MET signaling disrupts colorectal cancer cell plasticity
              </a>
            </td>
          <td>
            S. J. Sonnentag, S. Goldbach, D. Hoch, G. Andrieux, J. Becker, H. Fromm, L. Munoz-Sagredo, Y. Heneka, D. Grimm, F. R. Greten, V. Orian-Rousseau
          </td>
          <td>2024-09-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) has revolutionized cell biology by enabling the profiling of transcriptomes at a single-cell resolution, leading to important discoveries that have advanced our understanding of cellular and tissue heterogeneity, developmental trajectories, and disease progression. Despite these important advances, scRNA-seq is limited to measuring the transcriptome providing a partial view of cellular function. To address this limitation, multimodal scRNA-seq assays have emerged, allowing for the simultaneous measurement of RNA expression and protein. Intracellular Transcriptomic and Protein Sequencing (InTraSeq), a novel multimodal scRNA-seq technology described here, enables the concurrent measurement of mRNA, surface markers, cytoplasmic proteins, and nuclear proteins within individual cells through oligo-barcoded antibodies. This method offers a comprehensive approach to studying cellular function by combining RNA and protein profiling from the same sample and utilizing a relatively simple protocol. The InTraSeq method enables researchers to expand their view of critical intracellular protein expression including post-translational modifications (PTMs) and transcription factors, allowing for the identification of novel cellular subtypes and states that may be obscured by RNA-based analyses alone. This is particularly valuable in understanding the heterogeneity of cell populations and identifying distinct functional states. In this report, we used InTraSeq to characterize the complex cellular states and regulatory mechanisms during Th17 cell differentiation. We simultaneously profiled RNA and protein expression in over 85,000 cells, capturing transcriptional changes, changes in protein expression and the dynamics of signaling pathways at a high resolution. Our results revealed novel insights into Th17 cell differentiation, including the identification of key regulatory factors and their target genes. By simultaneously measuring mRNA, extra and intra-cellular proteins, signaling proteins, and PTMs, InTraSeq offers a comprehensive understanding of cellular processes and enables the identification of novel regulatory mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/281131467bb5dc8461ee132b2238aeb8b1cafd01" target='_blank'>
              InTraSeq: A Multimodal Assay that Uncovers New Single-Cell Biology and Regulatory Mechanisms
              </a>
            </td>
          <td>
            Majd M. Ariss, Linglin Huang, Xiaokai Ding, Shivani Sheth, Tyler Levy, Jeremy Fisher, Jean Loebelenz, Keith Arlotta, Karen Dixon, Roberto Polakiewicz, V. K. Kuchroo, Sean A. Beausoleil
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Minor introns constitute 0.4% of all introns in human cells, but they are unique in their ability to regulate the genes in which they are embedded such that their low splicing efficiency can be a rate limiting step in gene expression. The first intron of the tumor suppressor gene, PTEN, has been documented to be a minor intron, but very little is known about its regulation. Regulation of PTEN levels in cancer cells, especially breast cancer is tightly controlled as a very small reduction in its expression can lead to tumorigenesis. Indeed, many genetic, epigenetic, post-transcriptional and post-translational mechanisms are employed to reduce PTEN levels or activities, leading to cancer. Here, we uncover a previously unexplored mechanism for modulating PTEN expression utilizing its minor intron. The minor intron of PTEN minor intron has one of the lowest splicing efficacies in breast cancer cells, causing at least 50% of PTEN pre-mRNA to retain it, and thus not produce a functional protein. We also show that, unlike other minor introns, the retained intron in PTEN pre-mRNA is not used as a molecular switch that would be spliced out when needed but is rather processed by premature cleavage and polyadenylation into a long noncoding RNA (lncRNA) that we termed PINC. Exogenous expression of PINC caused significant alterations to cellular proliferation, including breast cancer cells that harbor a genetic mutation in the PTEN gene and do not produce a functional PTEN protein. This shows that this novel lncRNA functions independently of the encoded protein of the host gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1318f9f43dd61e458c57076f614249c11e1f55c5" target='_blank'>
              Premature cleavage and polyadenylation in the minor intron of PTEN modulate its expression and generates a functional long non-coding RNA in breast cancer
              </a>
            </td>
          <td>
            Mariam Elesnawy, Rim Elghandour, Hana Hasna, Aisha Fakhroo, Boshra Al-Sulaiti, Ihab Younis
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01413a6e42d02eb690fa59aa79101fc898cc9619" target='_blank'>
              RNA sequestration in P-bodies sustains myeloid leukaemia.
              </a>
            </td>
          <td>
            Srikanth Kodali, Ludovica Proietti, Gemma Valcárcel, Anna V. López-Rubio, Patrizia Pessina, Thomas Eder, Junchao Shi, Annie Jen, Núria Lupión-Garcia, Anne C. Starner, Mason Bartels, Yingzhi Cui, Caroline M Sands, Ainoa Planas-Riverola, Alba Martínez, T. Velasco-Hernández, Laureano Tomás-Daza, Bernhard Alber, G. Manhart, I. Mayer, K. Kollmann, A. Fatica, Pablo Menendez, Evgenia Shishkova, Rachel E Rau, B. Javierre, Joshua J. Coon, Qi Chen, Eric L. Van Nostrand, J. L. Sardina, F. Grebien, Bruno Di Stefano
          </td>
          <td>2024-08-21</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Mutations in the male germline are a driving force behind rare genetic diseases. Driver mutations enjoying a selective advantage expand to mutant clusters within the aged testis and are thus overrepresented in sperm with age. Other kinds of driver mutations, occurring prepubescently, have been the focus of recent attention given their high occurrence rate independent of age. Here, we investigated the ErbB2 gene via error-corrected sequencing and detected a high percentage of missense mutations, including recurrent mutations, which were observed mainly in the tyrosine kinase domain with likely functional consequences, as we verified for a subset via biophysical methods. While these mutations increased with age, we found no evidence that they originated from mutational clusters in the aged testis, and young donors also showed an accumulation of driver mutations, suggesting that mutational enrichment is not exclusive to the sexually mature germline but can occur earlier during germline development, and are likely evenly distributed in the testis and stable in size.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa9f975fbf6b3e62de966f4ae5746c898526f4b3" target='_blank'>
              Mutations in ErbB2 accumulating in the male germline measured by error-corrected sequencing
              </a>
            </td>
          <td>
            Atena Yasari, Monika Heinzl, Theresa Mair, Tina Karimian, Shehab Moukbel Ali Aldawla, Ingrid Hartl, Andrea J. Betancourt, Peter Lanzerstorfer, Irene Tiemann-Boege
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Background/Aim Despite therapeutic advancements, metastatic colorectal cancer is usually fatal, necessitating novel approaches based on the molecular pathogenesis to improve outcomes. Some colorectal cancers have no mutations in the extended RAS panel (KRAS, NRAS, BRAF) genes and represent a special subset, which deserves particular therapeutic considerations. Materials and Methods The genomic landscape of colorectal cancers from publicly available genomic series was interrogated, using the cBioportal platform. Colorectal cancer cohorts with cancers devoid of KRAS/NRAS or BRAF mutations were evaluated for the presence of mutations in the catalytic sub-unit alpha of kinase PI3K, encoded by the gene PIK3CA. Results PIK3CA mutations in the absence of KRAS/NRAS/BRAF mutations were observed in 3.7% to 7.6% of colorectal cancers in the different series examined. Patients with all four genes in wildtype configuration (quadruple wild type) represented 32.2% to 39.9% of cases in the different series examined. Compared with quadruple wild type cancers, triple (KRAS/NRAS/BRAF) wild type/PIK3CA mutated cancers had a higher prevalence of high TMB cases and additional mutations in colorectal cancer associated genes except for mutations in TP53. Mutations in genes encoding for epigenetic modifiers and the DNA damage response (DDR) were also more frequent in triple wild type/PIK3CA mutated cancers. The prognosis of the two groups was comparable. Conclusion Colorectal cancers with PIK3CA mutations in the absence of KRAS/NRAS/BRAF mutations have frequently mutations in epigenetic modifiers and DDR response genes, which may provide opportunities for targeting. These mutations are present in a smaller subset of quadruple wild type cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fec366ce24e54405084d6854f8aadb7604b8213" target='_blank'>
              PIK3CA Mutated Colorectal Cancers Without KRAS, NRAS and BRAF Mutations Possess Common and Potentially Targetable Mutations in Epigenetic Modifiers and DNA Damage Response Genes
              </a>
            </td>
          <td>
            I. Voutsadakis
          </td>
          <td>2024-08-27</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Harnessing DNA double-strand breaks (DSBs) is a powerful approach for gene editing, but it may provoke loss of heterozygosity (LOH), which predisposes to tumorigenesis. To interrogate this risk, we developed a two- color flow cytometry-based system (Flo-LOH), detecting LOH in ∼5% of cells following a DSB. After this initial increase, cells with LOH decrease due to a competitive disadvantage with parental cells, but if isolated, they stably propagate. Segmental loss from terminal deletions with de novo telomere addition and nonreciprocal translocations is observed as well as whole chromosome loss, especially following a centromeric DSB. LOH spans megabases distal from the DSB, but also frequently tens of megabases centromere-proximal. Inhibition of microhomology-mediated end joining massively increases LOH, which is synergistically increased with concomitant inhibition of canonical nonhomologous end joining. The capacity for large-scale LOH must therefore be considered when using DSB-based gene editing, especially in conjunction with end joining inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d1b33368b2fb7ae9c66926efe7d4fcbd491246" target='_blank'>
              Megabase-scale loss of heterozygosity provoked by CRISPR-Cas9 DNA double-strand breaks
              </a>
            </td>
          <td>
            Samantha B. Regan, Darpan Medhi, Travis B. White, Yi-Zhen Jiang, Su Jia, Qichen Deng, M. Jasin
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9583e865faa5492c029bc6345e877e21d9be0e7f" target='_blank'>
              Integrative single-cell analysis of human colorectal cancer reveals patient stratification with distinct immune evasion mechanisms.
              </a>
            </td>
          <td>
            Xiaojing Chu, Xiangjie Li, Yu Zhang, Guohui Dang, Yuhui Miao, Wenbin Xu, Jinyu Wang, Zemin Zhang, Si Cheng
          </td>
          <td>2024-08-15</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Chickens are a valuable livestock species, providing an affordable and nutritious food source worldwide through eggs and meat. Additionally, chickens are extensively used in scientific research as model organisms in virology, immunology, epigenetics, development, and conservation biology. Considering this, there are increasing efforts to deepen our understanding of their genome complexity, such as efforts by the Functional Annotation of Animal Genomes (FAANG) Consortium. Toward this goal, we are working to contribute to the annotation of the chicken transcriptome. Allele-specific expression (ASE) is an imbalance of allelic gene expression often driven by genetic and epigenetic changes in cis-regulatory regions. We, therefore, aimed to determine allelic imbalances in gene expression across 20 tissues and cells to shed light on mechanisms regulating gene expression. Using replicate paired-end (PE) RNA-seq samples, we identified around 7 million variants across 20 tissues and cells (e.g., reproductive tissues, intestine tissues, muscle, and immune tissues and cells). We applied quality control measures and filtered out monoallelic and homozygous variants. After that, we calculated read counts per allele to determine allelic expression. We found 365,894 significant ASE SNPs and 11,530 ASE genes in at least one tissue or cell type (FDR ≤ 0.05). We discovered that ASE SNPs are widely distributed throughout the chicken genome, and ASE SNPs and ASE genes showed a varied distribution across tissues and cells. Primary macrophage exhibited the most abundant, while the pectoralis major (breast muscle) had the lowest ASE genes and ASE SNPs, ranging from 7,592 to 32,505 ASE SNPs and 773 to 2,387 ASE genes. Our findings reveal several important pathways affected by allelic imbalance of expression. These pathways included fatty acid biosynthesis and regulation, cell proliferation and differentiation, cell metabolism, adipocytokine signaling, proteolysis and autophagy, and focal adhesion. In summary, our study provides insight into relevant biological processes critical to chickens and can contribute to improving genome annotation, helping to expand the transcriptomic reference source.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34ca25bc7d19a5be733c02d1d2c38f3dd7c192ae" target='_blank'>
              PSVII-1 Widespread allele-specific expression across tissues and cells of the chicken genome
              </a>
            </td>
          <td>
            A. O. de Lima, Theros T. Ng, Yvonne Drechsler, R. D. Hawkins
          </td>
          <td>2024-09-01</td>
          <td>Journal of Animal Science</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Menin inhibitors that disrupt Menin-MLL interaction hold promise for treating specific acute myeloid leukemia subtypes, including KMT2A rearrangements (KMT2A-r), yet resistance remains a challenge. Here, through systematic chromatin-focused CRISPR screens, along with genetic, epigenetic, and pharmacologic studies in a variety of human and mouse KMT2A-r AML models, we uncover a potential resistance mechanism independent of canonical Menin-MLL targets. We show that a group of non-canonical Menin targets, which are bivalently co-occupied by active Menin and repressive H2AK119ub marks, are typically downregulated following Menin inhibition. The loss of Polycomb Repressive Complex 1.1 (PRC1.1) subunits, such as PCGF1 or BCOR, leads to Menin inhibitor resistance by epigenetic reactivation of these non-canonical targets, including MYC. Genetic and pharmacological inhibition of MYC can resensitize PRC1.1-deficent leukemia cells to Menin inhibition. Moreover, we demonstrate that leukemia cells with the loss of PRC1.1 subunits exhibit reduced monocytic gene signatures and are susceptible to the BCL2 inhibition, and combinational treatment of venetoclax overcomes the resistance to Menin inhibition in PRC1.1-deficient leukemia cells. These findings highlight the important roles of PRC1.1 and its regulated non-canonical Menin targets in modulating Menin inhibitor response and provide potential strategies to treat leukemias with compromised PRC1.1 function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5fba94eb1d6d5cb6e0a8de6969fbdc894d28222" target='_blank'>
              Epigenetic Regulation of Non-canonical Menin Targets Modulates Menin Inhibitor Response in Acute Myeloid Leukemia.
              </a>
            </td>
          <td>
            Xinyue Zhou, Lixia Zhang, Sajesan Aryal, Virginia Veasey, Amanda Tajik, Cecilia Restelli, Steven Moreira, Pengcheng Zhang, Yanfeng Zhang, Kristin J Hope, Yang Zhou, Changde Cheng, Ravi Bhatia, Rui Lu
          </td>
          <td>2024-08-16</td>
          <td>Blood</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40fcdfbbc2df4270401ef8beed0b78736d658d46" target='_blank'>
              DNA methylation in mammalian development and disease.
              </a>
            </td>
          <td>
            Zachary D. Smith, Sara Hetzel, Alexander Meissner
          </td>
          <td>2024-08-12</td>
          <td>Nature reviews. Genetics</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Radiation‐induced tissue injury (RITI) is the most common complication in clinical tumor radiotherapy. Due to the heterogeneity in the response of different tissues to radiation (IR), radiotherapy will cause different types and degrees of RITI, which greatly limits the clinical application of radiotherapy. Efforts are continuously ongoing to elucidate the molecular mechanism of RITI and develop corresponding prevention and treatment drugs for RITI. Single‐cell sequencing (Sc‐seq) has emerged as a powerful tool in uncovering the molecular mechanisms of RITI and for identifying potential prevention targets by enhancing our understanding of the complex intercellular relationships, facilitating the identification of novel cell phenotypes, and allowing for the assessment of cell heterogeneity and spatiotemporal developmental trajectories. Based on a comprehensive review of the molecular mechanisms of RITI, we analyzed the molecular mechanisms and regulatory networks of different types of RITI in combination with Sc‐seq and summarized the targeted intervention pathways and therapeutic drugs for RITI. Deciphering the diverse mechanisms underlying RITI can shed light on its pathogenesis and unveil new therapeutic avenues to potentially facilitate the repair or regeneration of currently irreversible RITI. Furthermore, we discuss how personalized therapeutic strategies based on Sc‐seq offer clinical promise in mitigating RITI.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43cb0e2375143a24b5e75c3b8bc90e1f65f24386" target='_blank'>
              Mechanisms of radiation‐induced tissue damage and response
              </a>
            </td>
          <td>
            Lin Zhou, Jiaojiao Zhu, Yuhao Liu, Ping Zhou, Yongqing Gu
          </td>
          <td>2024-09-20</td>
          <td>MedComm</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The use and publication of research utilising CRISPR/cas9 in gene editing and knock outs (KOs) within cell lines is now widespread and has proved extremely powerful in the interrogation of gene function. However, the potential of experimental artefacts resulting from the need to generate single cell clones post gene-manipulation has been somewhat overlooked. In this study, we show that the commonly used pulmonary cell line A549 displays significant heterogeneity in terms of their gene expression, and that individual cells from a population exhibit dramatically different susceptibility to viral infection from a range of viruses including respiratory syncytial virus (RSV), influenza A virus (IAV), Hazara virus (HAZV), lymphocytic choriomeningitis virus (LCMV) and Bunyamwera virus (BUNV). Moreover, we demonstrate the rapid re-emergence of phenotypic heterogeneity even after cloning. These results demonstrate the need for caution in interpreting results from CRISPR screens and CRISPR-KO validation studies, especially in the study of viral infection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0e26c9049ed60692f4658ae9936a4fcc5067296" target='_blank'>
              Single cell variation and rapid emergence of phenotypic heterogeneity in cell lines- a cautionary tale for devotees of CRISPR-Cas9
              </a>
            </td>
          <td>
            Hayley Bradley, John N Barr, Martin Stacey
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Background: To date, targeted therapies have largely failed in eliciting sustained responses in advanced pancreatic ductal adenocarcinoma (PDAC). KRAS-targeted therapies have the potential to drastically transform clinical management. Emerging data indicate that only a subset of patients respond, and that acquired resistance is common. Here, we used a combination of in vivo CRISPR activation (CRISPRa) screening and in vitro/preclinical models to identify synergistic combinations with KRAS inhibition (KRASi) in pancreatic and lung cancer. Material and Methods: We used our CRISPRa-competent PPKS (P53(F/F), Kras(LSL-G12D/+), R26(LSL-SAM)) model of adenocarcinoma to conduct targeted screening for drivers of resistance to MRTX133. Our library consisted of ∼450 frequently mutated and amplified putative oncogenes across human adenocarcinoma based on TCGA data. Mice were treated with 1e8 TU of lentiviral library through nasal instillation. After 6 weeks, treatment was initiated with 3 mg/kg MRTX1133 daily, 5 days/week (i.p.). Mice were sacrificed after 4 weeks of treatment and the guide distribution determined through bioinformatic analysis. Genes statistically significantly enriched (FDR<0.05) in the treatment group were determined. Cell lines with acquired resistance to MRTX1133 derived from the KPC PDAC model and the KPP lung adenocarcinoma (LUAD) model were generated using long-term, low-dose treatment. Resistance was confirmed with MTT drug response assays. In vitro combinatorial drug response was determined using a 72h, 96-well MTT assay, and clonogenic response using a 2 week, 384-well FACS assay. Mice were treated with 20 mg/kg MRTX1133 and/or 20mg/kg Temuterkib once daily, 7 days/week (i.p.) for 2 weeks. Results: We performed targeted in vivo CRISPRa screening for drivers of resistance to KRASi in an autochthons model of lung adenocarcinoma (LUAD) using a library targeting 452 genes commonly amplified or mutated in cancer. We identified a set of genes that were specifically enriched in at least 50% of tumors treated with MRTX1133, including MAPK1/ERK2. We also generated 16 cell lines with acquired resistance to MRTX1133, derived from KPC PDAC or KPP LUAD tumors. Transcriptomic analysis identified a large set of genes differentially expressed in cell with acquired resistance relative to controls. GSEA indicated that reactivation of the MAPK cascade as a common trait in resistant cells. To test if ERK inhibition synergizes with KRASi, we quantified the combinatorial response to MRTX1133 and Temuterkib, a specific inhibitor of ERK1 and ERK2, in a set of KPC and KPP derived cell lines, revealing significant synergy and reduced clonogenic potential in a majority of cell lines. A majority of cell lines with acquired resistance to MRTX1133 remained sensitive to Temuterkib. Finally, we tested combinatorial treatment of MRTX1133 and Temuterkib in a syngenetic subcutaneous model of PDAC, which showed at minimum additive effect. In conclusion we have identified ERK inhibition as a promising combination with KRAS inhibition in KRAS-mutant PDAC and LUAD.
 Citation Format: Fredrik I Thege, Amber Hoskins, Sonja M Woermann, Anirban Maitra. Identifying synergistic combinations with KRAS inhibition in PDAC [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr PR-15.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/172a9d516cee228f00e1cc1c95f94a25e631e8bb" target='_blank'>
              Abstract PR-15: Identifying synergistic combinations with KRAS inhibition in PDAC
              </a>
            </td>
          <td>
            Fredrik I. Thege, Amber Hoskins, Sonja M. Woermann, Anirban Maitra
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Children with Down syndrome (DS) and B-cell acute lymphoblastic leukemia (B-ALL) are at increased risk for treatment-related mortality and relapse, highlighting the need for new therapies. Leukemia cell lines (CLs) have been fundamental to understanding therapeutic responses to pharmacological agents. We generated three DS B-ALL CLs characterized with diverse genomic alterations, including IGH::CRLF2 rearrangement (BCR::ABL1 like), mutations in FLT3 and TP53, and a novel ERG::CEBPD rearrangement. DS CLs had diminished proliferation, metabolism, and mitochondrial function when compared to non-DS (NDS) CLs and interestingly, these findings were similar to NDS Philadelphia chromosome-like (Ph-like) B-ALL CLs. Based on similar mitochondrial defects and prior preclinical data using Venetoclax for Ph-like B-ALL, we hypothesized that Venetoclax would be effective in DS. Intriguingly, Venetoclax was more effective in DS when compared to both NDS and Ph-like CLs. Efficacy was observed in DS patient derived xenografts (PDXs) and diagnostic/relapsed patient samples treated with Venetoclax, which synergized with Trametinib and Vincristine. Mass spectrometry-based multiomics analyses in DS and NDS B-ALL patient samples revealed an enriched metabolite profile in DS, particularly in the hubs of glucose metabolism and polyunsaturated phosphatidylcholines and phosphatidylinositols. Transcriptome analyses in DS B-ALL patients (n=249) supported enhanced glucose and fatty acid metabolism; along with a reliance on BCL2 family members for apoptosis resistance. Venetoclax synergized with inhibition of glucose metabolism in DS B-ALL CLs. In summary, we have generated novel tools for studying DS B-ALL and identify altered metabolism in DS that responds to Venetoclax.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04f404242cc6274ce2ef0f772a4cb26039fa167f" target='_blank'>
              Metabolic vulnerabilities in Down syndrome B-cell acute lymphoblastic leukemia can be targeted using Venetoclax
              </a>
            </td>
          <td>
            Dallas Jones, Elena Woods, Zhenhua Li, Kristin L. Schaller, Eric Hoffmeyer, Ben Kooiman, Joseph Fernandez, Tegan Wharton, Dejene M. Tufa, Spencer C. Hall, George Trahan, Kelly W. Maloney, Holly L. Pacenta, Kelly Sullivan, Joaquin Espinosa, James R. Roede, John O. Marentette, Brett M Stevens, Anagha Inguva, D. Stephenson, J. Reisz, Angelo D’Alessandro, Craig T. Jordan, Jun J Yang, Michael R Verneris
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Glioblastoma is the most common primary malignant brain tumor in adults, with a median survival of just over 1 year. The failure of available treatments to achieve remission in patients with glioblastoma (GBM) has been attributed to the presence of cancer stem cells (CSCs), which are thought to play a central role in tumor development and progression and serve as a treatment-resistant cell repository capable of driving tumor recurrence. In fact, the property of "stemness" itself may be responsible for treatment resistance. In this study, we identify a novel long noncoding RNA (lncRNA), cancer stem cell-associated distal enhancer of SOX2 (CASCADES), that functions as an epigenetic regulator in glioma CSCs (GSCs). CASCADES is expressed in isocitrate dehydrogenase (IDH)-wild-type GBM and is significantly enriched in GSCs. Knockdown of CASCADES in GSCs results in differentiation towards a neuronal lineage in a cell- and cancer-specific manner. Bioinformatics analysis reveals that CASCADES functions as a super-enhancer-associated lncRNA epigenetic regulator of SOX2. Our findings identify CASCADES as a critical regulator of stemness in GSCs that represents a novel epigenetic and therapeutic target for disrupting the CSC compartment in glioblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5be2f4cc8ff228e53f5dc871cb0a020d6528f65a" target='_blank'>
              CASCADES, a novel SOX2 super-enhancer-associated long noncoding RNA, regulates cancer stem cell specification and differentiation in glioblastoma.
              </a>
            </td>
          <td>
            Uswa Shahzad, Marina Nikolopoulos, Christopher Li, Michael J Johnston, J. Wang, N. Sabha, Frederick Varn, Alexandra N Riemenschneider, Stacey L. Krumholtz, P. M. Krishnamurthy, Christian A. Smith, Jason Karamchandani, Jonathan K. Watts, R. Verhaak, Marco Gallo, James T. Rutka, Sunit Das
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Rhabdoid tumors (RTs) are rare, highly aggressive pediatric malignancies which occur in kidney, soft-parts and brain. They are characterized by a complete inactivation of the SMARCB1 tumor suppressor gene encoding a core subunit of the chromatin remodeling SWI/SNF complex. Prognosis for children with RTs is poor as, in many instances, these tumors are resistant to conventional type chemotherapy. Pharmacological inhibition of EZH2 is a promising strategy to treat some tumors with loss of function in SMARCB1. However, the clinical response rate remains low, and both primary and secondary resistance have been reported. Understanding the mechanisms of resistance to EZH2 inhibition may allow new therapeutic hypothesis. For this purpose, we realized a genome-wide CRISPR-Cas9 knockout screening on two rhabdoid cell lines treated with EZH2 inhibitor. We identified TP53 loss as the sole consistent gene knock-out able to confer some resistance to EZH2 inhibition in both cell lines. Conversely, we demonstrated that MDM2/MDM4 inhibition, in a TP53-dependant manner, strongly synergized with EZH2 inhibition to control cell lines viability in vitro. We further treated 4 rhabdoid patient-derived xenografts with UNC1999, Idasanutlin and the combination of both and observed a potent tumor growth control upon combined EZH2 and MDM2/MDM4 inhibitions. To conclude, our results strongly encourage to assess the actual efficacy of combined MDM2/MDM4 and EZH2 inhibitors in the treatment of patients with SMARCB1-deficient rhabdoid tumors.
 Citation Format: Céline Chauvin, Tiphaine Hery, Rachida Bouarich, Fariba Nemati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, Sergio Roman-Roman, Didier Decaudin, Raphaël Margueron, Franck Bourdeaut. Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A051.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/85bf34da504cd580597c4be8039c1663b824b215" target='_blank'>
              Abstract A051 Genome-wide CRISPR/Cas9 library screening identified TP53 as a critical driver for resistance to EZH2 inhibitor in rhabdoid tumors
              </a>
            </td>
          <td>
            C. Chauvin, Tiphaine Héry, Rachida Bouarich, F. Némati, Diego Teyssonneau, Camille Fouassier, Chiara Giudiceandrea, Zhi-Yan Han, Sakina Zaidi, Didier Surdez, S. Roman-Roman, Didier Decaudin, Raphaël Margueron, F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="Background Hepatocellular carcinoma (HCC) is a prevalent and heterogeneous tumor with limited treatment options and unfavorable prognosis. The crucial role of a disintegrin and metalloprotease (ADAM) gene family in the tumor microenvironment of HCC remains unclear. Methods This study employed a novel multi-omics integration strategy to investigate the potential roles of ADAM family signals in HCC. A series of single-cell and spatial omics algorithms were utilized to uncover the molecular characteristics of ADAM family genes within HCC. The GSVA package was utilized to compute the scores for ADAM family signals, subsequently stratified into three categories: high, medium, and low ADAM signal levels through unsupervised clustering. Furthermore, we developed and rigorously validated an innovative and robust clinical prognosis assessment model by employing 99 mainstream machine learning algorithms in conjunction with co-expression feature spectra of ADAM family genes. To validate our findings, we conducted PCR and IHC experiments to confirm differential expression patterns within the ADAM family genes. Results Gene signals from the ADAM family were notably abundant in endothelial cells, liver cells, and monocyte macrophages. Single-cell sequencing and spatial transcriptomics analyses have both revealed the molecular heterogeneity of the ADAM gene family, further emphasizing its significant impact on the development and progression of HCC. In HCC tissues, the expression levels of ADAM9, ADAM10, ADAM15, and ADAM17 were markedly elevated. Elevated ADAM family signal scores were linked to adverse clinical outcomes and disruptions in the immune microenvironment and metabolic reprogramming. An ADAM prognosis signal, developed through the utilization of 99 machine learning algorithms, could accurately forecast the survival duration of HCC, achieving an AUC value of approximately 0.9. Conclusions This study represented the inaugural report on the deleterious impact and prognostic significance of ADAM family signals within the tumor microenvironment of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/000a5d98243250c138efd38742555d600655a9e5" target='_blank'>
              Unraveling the role of ADAMs in clinical heterogeneity and the immune microenvironment of hepatocellular carcinoma: insights from single-cell, spatial transcriptomics, and bulk RNA sequencing
              </a>
            </td>
          <td>
            Junhong Chen, Qihang Yuan, Hewen Guan, Yuying Cui, Chang Fu, Tianfu Wei, Kai Liu
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Relapse remains a determinant of treatment failure and contributes significantly to mortality in acute myeloid leukemia (AML) patients. Despite efforts to understand AML progression and relapse mechanisms, findings on acquired gene mutations in relapse vary, suggesting inherent genetic heterogeneity and emphasizing the role of epigenetic modifications. We conducted a multi-omic analysis using Omni-C, ATAC-seq, and RNA-seq on longitudinal samples from two adult AML patients at diagnosis and relapse. Herein, we characterized genetic and epigenetic changes in AML progression to elucidate the underlying mechanisms of relapse. Differential interaction analysis showed significant 3D chromatin landscape reorganization between relapse and diagnosis samples. Comparing global open chromatin profiles revealed that relapse samples had significantly fewer accessible chromatin regions than diagnosis samples. In addition, we discovered that relapse-related upregulation was achieved either by forming new active enhancer contacts or by losing interactions with poised enhancers/potential silencers. Altogether, our study highlights the impact of genetic and epigenetic changes on AML progression, underlining the importance of multi-omic approaches in understanding disease relapse mechanisms and guiding potential therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce349ab8147752454b7d482ecbf7663d5857bab7" target='_blank'>
              Multi-omic analysis of longitudinal acute myeloid leukemia patient samples reveals potential prognostic markers linked to disease progression
              </a>
            </td>
          <td>
            Nisar Ahmed, Irene Cavattoni, William Villiers, Chiara Cugno, S. Deola, Borbala Mifsud
          </td>
          <td>2024-09-27</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In recent decades, the field of genetics has revolutionized our understanding of biology. In particular, we know about how genetic information encodes the blueprint for life and how variations and mutations in these sequences contribute to disease or health. What is unique is that the genetic trait changes do not always caused by any changes in genetic sequences. 
Epigenetics refers to the study of how environmental factors and lifestyle choices can affect the gene activity. Changes in the heritable traits of organisms can be caused by modification of gene expression rather than alteration to the underlying DNA sequence. Therefore, this change does not involve any DNA mutation. 
DNA methylation is a biological process of adding a methyl group to the DNA molecule. The activity of DNA segments can change without changing the sequence by the process of methylation. This process plays an important role in normal development and aberrant DNA methylation patterns are associated with various diseases. Histones are highly basic proteins around which DNA is wound. The modifications in histone protein plays a fundamental role in most biological processes and can alter the accessibility of the DNA for transcription, as well as influencing gene expression. There are non-coding RNA molecules that do not encode proteins but play vital role in regulating the gene expression by interacting with other molecules and influencing chromatin structure. 
Over the past few decades, mechanisms of epigenetics have emerged as a crucial factor in governing every biological process in the human body. Throughout human life, these mechanisms have been impacted various physiological and pathological conditions. For example, cancer is often characterized by epigenetics abnormalities, such as hypermethylation of tumor suppressor genes or hypomethylation of oncogenes. Epigenetic modifications are also implicated in complex diseases like cardiovascular disease, autoimmune conditions and neurodegenerative disorder. Beyond these diseases, epigenetics also offers insights into health and development. Furthermore, understanding the epigenetic mechanisms influence human health, leading to new preventive strategies and treatment of various diseases. Early life experiences, including nutrition and stress, can lead to epigenetic changes that affect long-term health outcomes of the individual.   
Advances in this research can pinpoint any epigenetic changes that affect long-term health outcomes. The advances of epigenetic research can lead to early diagnosis and preventive measures. Identifying the epigenetic markers associated with the diseases enable early detection of the diseases and more effective preventative strategies. Some epigenetic therapies like; drugs that modify histone acetylation or DNA methylation, are already in development and provide future treatments for disease. To understand how life style factors, influence epigenetic regulation can lead to personalized lifestyle recommendations that influence the public health strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6963fc644883e01d78b4f2a627be037841fc025" target='_blank'>
              The Role of Epigenetics in Disease and Health: Beyond DNA Sequences
              </a>
            </td>
          <td>
            .. Diki
          </td>
          <td>2024-08-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Translation acts as an additional layer of regulation that has an important role in gene expression and function. Due to a phenomenon called codon usage bias, highly expressed genes are thought to be codon-biased to support efficient translation. As more than one codon can code for the same amino acid (synonymous codons), organisms may exhibit preferences for specific codons that facilitate increased expression of important genes due to variations in the availability of corresponding tRNAs. Advances in sequencing technologies have recently permitted the study of the translatome, which refers to the entire population of mRNA associated with ribosomes for protein synthesis and can be investigated through ribosome profiling. This cutting-edge technique allows the identification of actively translated regions, but can also reveal translational pausing events that stem from the presence of SNPs. Our previous research has demonstrated variation in tRNA expression across tissues and different states of health, leading us to consider the connection between tRNA abundance and translational stalling due to SNPs. By coupling ribosome profiling with tRNA sequencing and RNA sequencing, we have investigated all elements of translational machinery to predict translational efficiency, estimate proteome composition, and evaluate tRNA abundance as a source of genetic variation. In this work, we utilized ribosome profiling, tRNAseq, and RNAseq and performed an integrative analysis in bovine tissues (kidney, liver and muscle) as well as murine myoblast cell lines (C2C12). By applying these methods to different bovine tissues, we were able to explore the interplay between tRNA availability and translational stalling events. Moreover, we have identified translationally regulated genes underlying tissue-specific biological processes and found that many upregulated and downregulated genes coincided with high and low translational efficiency respectively. We have also successfully defined stalling sites that depict the regulatory information encoded within the coding sequence of transcripts, which could control translation rate and facilitate proper protein folding. Through the implementation of these next generation sequencing (NGS) methods to cell culture, we were able to evaluate the translatome at various time points (0-min, 30-min, 60-min, and 4-h) post-induction of differentiation. Where most studies have focused on molecular changes between differentiating myoblasts and multinucleated myotubes that are present 7 d after differentiation, we focus on the earliest stages of muscle differentiation that initiate muscle development and therefore kickstart the process of meat production. This work offers an atlas of distinctive stalling sites across bovine tissues and various stages of muscle differentiation, which provides an opportunity to predict codon optimality and understand tissue-specific or time-specific mechanisms of regulating protein synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e43a89a80d89cd089ba12c6ecf74452555c852c4" target='_blank'>
              199 Awardee Talk: Insights into translational regulation through tRNA sequencing, RNA sequencing, and ribosome profiling
              </a>
            </td>
          <td>
            A. K. Goldkamp, L. Okamoto, Kara J Thornton-Kurth, Darren E. Hagen
          </td>
          <td>2024-09-01</td>
          <td>Journal of Animal Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pediatric cancers are estimated to be related to a genetic predisposition in about 8 to 10% of cases. This estimation mostly relies on already discovered cancer predisposing genes. However, the numerous new cancer predisposing genes that have been discovered in the past decade thanks to high throughput sequencing in vast cohort of pediatric patients illustrate that new genes may still be unravelled. Moreover, the frequency of early post-zygotic mosaicism is being readdressed thanks to more dedicated sequencing and research. The actual clinical impact of all those discoveries is the main challenge for the community: defining the penetrance, delineating the cancer spectrum and therefore the appropriate surveillance, and setting up innovative follow-up techniques are necessary. Here, we’ll describe recent findings on brain tumor predisposing genes (ELP1 and SMARCB1, among others) to illustrate the increasing recognition of genetic predisposition in pediatric cancers and how the clinical translation of these laboratory findings remain challenging at clinical and ethical levels.
 Citation Format: Franck Bourdeaut. Genetic predisposition to pediatric cancer: the long way from discovery to surveillance guidelines [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr IA003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc6b854ba212ba8f4846fc2365983567ae6c2486" target='_blank'>
              Abstract IA003: Genetic predisposition to pediatric cancer: the long way from discovery to surveillance guidelines
              </a>
            </td>
          <td>
            F. Bourdeaut
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24d8d8f2f89f8ee70047ce83a13996f46168eafe" target='_blank'>
              Star wars against leukemia: attacking the clones.
              </a>
            </td>
          <td>
            Monika M. Toma, Tomasz Skorski
          </td>
          <td>2024-09-02</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genome sequencing has revealed frequent mutations in Ras homolog family member A (RHOA) among various cancers with unique aberrant profiles and pathogenic effects, especially in peripheral T-cell lymphoma (PTCL). The discrete positional distribution and types of RHOA amino acid substitutions vary according to the tumor type, thereby leading to different functional and biological properties, which provide new insight into the molecular pathogenesis and potential targeted therapies for various tumors. However, the similarities and discrepancies in characteristics of RHOA mutations among various histologic subtypes of PTCL have not been fully elucidated. Herein we highlight the inconsistencies and complexities of the type and location of RHOA mutations and demonstrate the contribution of RHOA variants to the pathogenesis of PTCL by combining epigenetic abnormalities and activating multiple downstream pathways. The promising potential of targeting RHOA as a therapeutic modality is also outlined. This review provides new insight in the field of personalized medicine to improve the clinical outcomes for patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd71cf13763be0cc23a4dc23d32cf828f50620b5" target='_blank'>
              Mutations in Ras homolog family member A in patients with peripheral T-cell lymphoma and implications for personalized medicine
              </a>
            </td>
          <td>
            Lina Hu, Xuanye Zhang, Shengbing Zang
          </td>
          <td>2024-08-09</td>
          <td>Cancer Biology & Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: Transposable elements (TEs) are dynamic repetitive regions which generate mutations and structural variants. TP53 plays a crucial role in suppressing TE movement to maintain genomic stability. The relationship between TP53 and TEs has been extensively studied in tumours, but not the germline. Individuals with germline TP53 pathogenic variants have Li-Fraumeni Syndrome (LFS), a cancer predisposition syndrome with a high lifetime risk of cancer in various tissues. This study aims to characterize the TE landscape in individuals with Li-Fraumeni Syndrome and determine how this contributes to their increased cancer risk. Methods: MELT and xTea were used to identify TEs in children with (n=48) or without (n=198) a germline TP53 variant. TE calls were merged with SURVIVOR and variants were annotated with AnnotSV. To assess the influence of TP53 on TE location, we quantified TEs across genomic windows and identified the top 100 significantly different regions with the Mann-Whitney U test, adjusting for multiple comparisons. We used these regions to develop a gradient-boosted tree model with 5-fold cross-validation to predict TP53 status. Results: Individuals with germline TP53 variants harboured significantly fewer ALU and LINE1 elements in their germline genome compared to the control dataset (p<0.001). The phenomenon was consistent across chromosomes and significant in chromosomes 5 and 11 (FDR<0.05). A gradient-boosted tree model was able to differentiate patients with and without a germline TP53 variant with an AUPRC of 0.77 on an unseen test set. Conclusion: Germline TP53 variants may effect the frequency and location of germline TE insertions, which may influence nearby genomic variations and lead to cancer development. Analyzing TEs in individuals with LFS will enhance our understanding of accelerated cancer development in these patients, informing future research for diagnostic and therapeutic approaches.
 Citation Format: Brianne Laverty, Shilpa Yadahalli, Vallijah Subasri, David Malkin. The role of TP53 on transposable elements in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4995c29bd2050c705f9a5b8da916aacb0aa2601c" target='_blank'>
              Abstract A010 The role of TP53 on transposable elements in pediatric cancer
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Vallijah Subasri, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE OF REVIEW
FLT3 mutations are among the most common myeloid drivers identified in adult acute myeloid leukemia (AML). Their identification is crucial for the precise risk assessment because of the strong prognostic significance of the most recurrent type of FLT3 alterations, namely internal tandem duplications (ITDs). Recent advances in the pathogenesis and biology of FLT3-mutated AML have opened an opportunity for development and application of selective inhibition of FLT3 pathway.


RECENT FINDINGS
In the last decade, at least three targeted treatments have been approved by regulatory agencies and several others are currently under investigations. Here, we review the latest advance in the role of FLT3 mutations in AML, providing an outline of the available therapeutic strategies, their mechanisms of actions and of resistance, as well as routes for potential improvement.


SUMMARY
The availability of FLT3 inhibitors has improved outcomes in AML harboring such mutations, currently also reflected in disease stratification and recommendations. Newer inhibitors are under investigations, and combinations with chemotherapy or other targeted treatments are being explored to further improve disease outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/732f19a10a5bb85b8ee615c65a99e7f2184b3ca7" target='_blank'>
              Advances in the pathogenesis of FLT3-mutated acute myeloid leukemia and targeted treatments.
              </a>
            </td>
          <td>
            S. Travaglini, C. Gurnari, T. Ottone, M. Voso
          </td>
          <td>2024-08-27</td>
          <td>Current opinion in oncology</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="This article describes how the transcriptional alterations of the innate immune system divide dysplasias into aggressive forms that, despite the treatment, relapse quickly and more easily, and others where the progression is slow and more treatable. It elaborates on how the immune system can change the extracellular matrix, favoring neoplastic progression, and how infections can enhance disease progression by increasing epithelial damage due to the loss of surface immunoglobulin and amplifying the inflammatory response. We investigated whether these dysregulated genes were linked to disease progression, delay, or recovery. These transcriptional alterations were observed using the RNA-based next-generation sequencing (NGS) panel Oncomine Immune Response Research Assay (OIRRA) to measure the expression of genes associated with lymphocyte regulation, cytokine signaling, lymphocyte markers, and checkpoint pathways. During the analysis, it became apparent that certain alterations divide dysplasia into two categories: progressive or not. In the future, these biological alterations are the first step to provide new treatment modalities with different classes of drugs currently in use in a systemic or local approach, including classical chemotherapy drugs such as cisplatin and fluorouracile, older drugs like fenretinide, and new checkpoint inhibitor drugs such as nivolumab and pembrolizumab, as well as newer options like T cell therapy (CAR-T). Following these observed alterations, it is possible to differentiate which dysplasias progress or not or relapse quickly. This information could, in the future, be the basis for determining a close follow-up, minimizing surgical interventions, planning a correct and personalized treatment protocol for each patient and, after specific clinical trials, tailoring new drug treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7733a13a6989d51b26a85ab5fd3970e4ce88c18" target='_blank'>
              A Transcriptomic Analysis of Laryngeal Dysplasia
              </a>
            </td>
          <td>
            F. Maffini, D. Lepanto, F. Chu, M. Tagliabue, D. Vacirca, R. De Berardinis, S. Gandini, Silvano Vignati, Alberto Ranghiero, S. Taormina, A. Rappa, M. Cossu Rocca, D. Alterio, Susanna Chiocca, Massimo Barberis, Lorenzo Preda, F. Pagni, Nicola Fusco, M. Ansarin
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="DNA demethylases TET2 and TET3 play a fundamental role in thymic invariant natural killer T (iNKT) cell differentiation by mediating DNA demethylation of genes encoding for lineage specifying factors. Paradoxically, differential gene expression analysis revealed that significant number of genes were upregulated upon TET2 and TET3 loss in iNKT cells. This unexpected finding could be potentially explained if loss of TET proteins was reducing the expression of proteins that suppress gene expression. In this study, we discover that TET2 and TET3 synergistically regulate Drosha expression, by generating 5hmC across the gene body and by impacting chromatin accessibility. As DROSHA is involved in microRNA biogenesis, we proceed to investigate the impact of TET2/3 loss on microRNAs in iNKT cells. We report that among the downregulated microRNAs are members of the Let-7 family that downregulate in vivo the expression of the iNKT cell lineage specifying factor PLZF. Our data link TET proteins with microRNA expression and reveal an additional layer of TET mediated regulation of gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/340235c7cc4f9c622222c731a0e560d74501df72" target='_blank'>
              TET proteins regulate Drosha expression and impact microRNAs in iNKT cells
              </a>
            </td>
          <td>
            Marianthi Gioulbasani, Tarmo Äijö, Jair E. Valenzuela, Julia Buquera Bettes, Ageliki Tsagaratou
          </td>
          <td>2024-09-19</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hematopoietic stem cells (HSCs) maintain homeostasis in the hematopoietic ecosystem, which is tightly regulated at multiple layers. Acute myeloid leukemia (AML) is a severe hematologic malignancy driven by genetic and epigenetic changes that lead to the transformation of leukemia stem cells (LSCs). Since somatic mutations in DNA methylation-related genes frequently occur in AML, DNA methylation is widely altered and functions as a starting engine for initiating AML. Additionally, RNA modifications, especially N6-methyladenosine (m6A), also play an important role in the generation and maintenance of the hematopoietic ecosystem, and AML development requires reprogramming of m6A modifications to facilitate cells with hallmarks of cancer. Given the complex pathogenesis and poor prognosis of AML, it is important to fully understand its pathogenesis. Here, we mainly focus on DNA methylation and RNA m6A modification in hematopoiesis and AML and summarize recent advances in this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92f8b80ecba345e57d84b39e3572cdbd8a3dd106" target='_blank'>
              Epigenetic modifications in hematopoietic ecosystem: a key tuner from homeostasis to acute myeloid leukemia
              </a>
            </td>
          <td>
            Shu-Qi Yao, Rongxia Guo, Wen Tian, Yanbing Zheng, Jin Hu, G-X Han, Rong Yin, Fuling Zhou, Haojian Zhang
          </td>
          <td>2024-09-12</td>
          <td>Blood Science</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="BACKGROUND AND OBJECTIVE
The identification of mutation hot spots in the isocitrate dehydrogenase (IDH) genes is one of the most important cancer genome-wide sequencing discoveries with relevant impact in the treatment of some orphan tumors. These genes were mostly found mutated in lower-grade gliomas (LGGs), acute myeloid leukaemia (AML), myelodysplastic syndromes (MDS) and myeloproliferative neoplasms (MPNs) and in cholangiocarcinoma. This aberrant genomic condition represents a therapeutic target of great interest in cancer research, especially in AML, given the limitations of currently approved therapies in this field. In this review, we investigate the role of IDH mutation and the mutant IDH (mIDH)- targeted therapies for cholangiocarcinoma and glioma.


METHODS
Here, we provide an overview of the IDH mutation role and discuss its role in tumorigenesis and progression of some solid cancers, in which the therapeutic strategy can be completely changed thanks to these brand-new therapeutic options.


KEY CONTENT AND FINDINGS
The encouraging early clinical data demonstrated to be a proof of concept for investigational mIDH1/2 inhibitors in tumors with a paucity of therapeutic possibilities.


CONCLUSIONS
Moreover, we list the most important randomised clinical trials still active with their preliminary results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a20edde7b8a46ad9d9cedbb5e90b751d02d2ad2f" target='_blank'>
              Therapeutic inhibition of isocitrate dehydrogenase mutations in glioma and cholangiocarcinoma: new insights and promises-a narrative review.
              </a>
            </td>
          <td>
            V. Internò, A. Melaccio, Pasquale Vitale, Roberta Spedaliere, Massimo Buonfantino, R. Messina, Anna Laura Lippolis, Francesco Signorelli, Raffaele Addeo, Francesco Giuliani
          </td>
          <td>2023-01-01</td>
          <td>Chinese clinical oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c904b2f681f70c0b549880d31ccc37b010f98cbc" target='_blank'>
              Characterisation of APOBEC3B-Mediated RNA editing in breast cancer cells reveals regulatory roles of NEAT1 and MALAT1 lncRNAs.
              </a>
            </td>
          <td>
            Chi Zhang, Yu-Jing Lu, Mei Wang, Bingjie Chen, Feifei Xiong, C. Mitsopoulos, Olivia Rossanese, Xiuling Li, Paul A Clarke
          </td>
          <td>2024-09-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e16f71ff68478a0f30f71105d86b66a3bf8d6b91" target='_blank'>
              Single-Cell Sequencing: High-Resolution Analysis of Cellular Heterogeneity in Autoimmune Diseases.
              </a>
            </td>
          <td>
            Xuening Tang, Yudi Zhang, Hao Zhang, Nan Zhang, Ziyu Dai, Q. Cheng, Yongzhen Li
          </td>
          <td>2024-08-26</td>
          <td>Clinical reviews in allergy & immunology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cystic fibrosis (CF) is caused by mutations in the cystic fibrosis transmembrane conductance regulator (CFTR) gene. Although many people with CF (pwCF) are treated using CFTR modulators, some are non-responsive due to their genotype or other uncharacterized reasons. Autologous airway stem cell therapies, in which the CFTR cDNA has been replaced, may enable a durable therapy for all pwCF. Previously, CRISPR-Cas9 with two AAVs was used to sequentially insert two halves of the CFTR cDNA and an enrichment cassette into the CFTR locus. However, the editing efficiency was <10% and required enrichment to restore CFTR function. Further improvement in gene insertion may enhance cell therapy production. To improve CFTR cDNA insertion in human airway basal stem cells (ABCs), we evaluated the use of the small molecules AZD7648 and ART558 which inhibit non-homologous end joining (NHEJ) and micro-homology mediated end joining (MMEJ). Adding AZD7648 alone improved gene insertion by 2-3-fold. Adding both ART558 and AZD7648 improved gene insertion but induced toxicity. ABCs edited in the presence of AZD7648 produced differentiated airway epithelial sheets with restored CFTR function after enrichment. Adding AZD7648 did not increase off-target editing. Further studies are necessary to validate if AZD7648 treatment enriches cells with oncogenic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b76a5973a8822632977941f52ade0b5a68b63d15" target='_blank'>
              DNA-PKcs Inhibition Improves Sequential Gene Insertion of the Full-Length CFTR cDNA in Airway Stem Cells
              </a>
            </td>
          <td>
            Jacob T. Stack, R. Rayner, Reza Nouri, Carlos J. Suarez, Sun Hee Kim, K. Kanke, Tatyana A. Vetter, E. Cormet-Boyaka, S. Vaidyanathan
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="BACKGROUND
The identification of the most appropriate targeted therapies for advanced cancers is challenging. We performed a molecular profiling of metastatic solid tumors utilizing a comprehensive next-generation sequencing (NGS) assay to determine genomic alterations' type, frequency, actionability, and potential correlations with PD-L1 expression.


METHODS
A total of 304 adult patients with heavily pretreated metastatic cancers treated between January 2019 and March 2021 were recruited. The CLIA-/UKAS-accredit Oncofocus assay targeting 505 genes was used on newly obtained or archived biopsies. Chi-square, Kruskal-Wallis, and Wilcoxon rank-sum tests were used where appropriate. Results were significant for P < .05.


RESULTS
A total of 237 tumors (78%) harbored potentially actionable genomic alterations. Tumors were positive for PD-L1 in 68.9% of cases. The median number of mutant genes/tumor was 2.0 (IQR: 1.0-3.0). Only 34.5% were actionable ESCAT Tier I-II with different prevalence according to cancer type. The DNA damage repair (14%), the PI3K/AKT/mTOR (14%), and the RAS/RAF/MAPK (12%) pathways were the most frequently altered. No association was found among PD-L1, ESCAT, age, sex, and tumor mutational status. Overall, 62 patients underwent targeted treatment, with 37.1% obtaining objective responses. The same molecular-driven treatment for different cancer types could be associated with opposite clinical outcomes.


CONCLUSIONS
We highlight the clinical value of molecular profiling in metastatic solid tumors using comprehensive NGS-based panels to improve treatment algorithms in situations of uncertainty and facilitate clinical trial recruitment. However, interpreting genomic alterations in a tumor type-specific manner is critical.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b13a3de4d8a4a44c9264bbeddda708851f093875" target='_blank'>
              Next-generation sequencing-based evaluation of the actionable landscape of genomic alterations in solid tumors: the "MOZART" prospective observational study.
              </a>
            </td>
          <td>
            Francesco Schettini, Marianna Sirico, M. Loddo, Gareth H Williams, Keeda-Marie Hardisty, Paul Scorer, R. Thatcher, Pablo Rivera, Manuela Milani, C. Strina, G. Ferrero, Marco Ungari, C. Bottin, Fabrizio Zanconati, Nicolò de Manzini, S. Aguggini, Richard Tancredi, E. Fiorio, Antonio Fioravanti, Maurizio Scaltriti, Daniele Generali
          </td>
          <td>2024-08-23</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Infant ALL (iALL) is initiated in utero, most often by rearrangement of the KMT2A gene (KMT2Ar). It carries a very poor prognosis despite a lack of additional oncogenic driver mutations common in childhood ALL. Here, we aimed to identify specific properties of human fetal hematopoietic stem/progenitor cells (HSPC) that promote leukemic transformation in KMT2Ar iALL using molecular, functional and in vivo assays. First, by comparing transcriptomes of human fetal HSPC to adult HSPC we derived a fetal-specific gene signature and identified the fetal oncogene LIN28B and its downstream effectors among the top hits. These genes were also expressed in iALL. Functional assays revealed that LIN28B was essential in human fetal liver (FL) CD34+ cells to maintain proliferation and stemness, and support B- and NK-lymphopoiesis. To interrogate the role of LIN28B in iALL, we utilised a human FL-derived CRISPR-Cas9 KMT2A::AFF1 model. In this model, LIN28B-expressing leukemias were more proliferative in vitro and in vivo, with this advantage being lost upon LIN28B knockdown. Mechanistic studies showed that LIN28B acts by stabilizing key early B-lymphoid genes, epigenetic regulators, and cell cycle and anti-apoptotic genes. Finally, In the absence of LIN28B, human FL CD34+ cells fail to transform upon induction of KMT2A::AFF1 translocation. Thus, LIN28B has an essential role in normal human fetal B-lymphopoiesis, and is necessary for the initiation of KMT2A::AFF1 iALL in fetal cells in the absence of co-operating mutations. It has a role in making leukemias more aggressive, suggesting it is a potential target in LIN28B-expressing leukemias.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10cc286f51631e81c564d7b507e1a0ef71703dbb" target='_blank'>
              The fetal specific gene LIN28B is essential for human fetal B-lymphopoiesis and initiation of KMT2A::AFF1 infant leukemia
              </a>
            </td>
          <td>
            Rebecca E. Ling, T. Jackson, N. Elliott, Joe Cross, Lucy Hamer, Arundhati Wuppalapati, Alastair L. Smith, Catherine Chahrour, Okan Sevim, Deena Iskander, Guanlin Wang, S. Rice, Sorcha O’Byrne, Joe R Harman, Bethan Psaila, R. Morgan, Irene Roberts, Thomas A. Milne, Anindita Roy
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Background Radiotherapy (RT) is a critical component of treatment for locally advanced rectal cancer (LARC), though patient response varies significantly. The variability in treatment outcomes is partly due to the resistance conferred by cancer stem cells (CSCs) and tumor immune microenvironment (TiME). This study investigates the role of EIF5A in radiotherapy response and its impact on the CSCs and TiME. Methods Predictive models for preoperative radiotherapy (preRT) response were developed using machine learning, identifying EIF5A as a key gene associated with radioresistance. EIF5A expression was analyzed via bulk RNA-seq and single-cell RNA-seq (scRNA-seq). Functional assays and in vivo experiments validated EIF5A’s role in radioresistance and TiME modulation. Results EIF5A was significantly upregulated in radioresistant colorectal cancer (CRC) tissues. EIF5A knockdown in CRC cell lines reduced cell viability, migration, and invasion after radiation, and increased radiation-induced apoptosis. Mechanistically, EIF5A promoted cancer stem cell (CSC) characteristics through the Hedgehog signaling pathway. Analysis of the TiME revealed that the radiation-resistant group had an immune-desert phenotype, characterized by low immune cell infiltration. In vivo experiments showed that EIF5A knockdown led to increased infiltration of CD8+ T cells and M1 macrophages, and decreased M2 macrophages and Tregs following radiation therapy, thereby enhancing the radiotherapy response. Conclusion EIF5A contributes to CRC radioresistance by promoting CSC traits via the Hedgehog pathway and modulating the TiME to an immune-suppressive state. Targeting EIF5A could enhance radiation sensitivity and improve immune responses, offering a potential therapeutic strategy to optimize radiotherapy outcomes in CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f53c097a18755aa6274aec26b4f9ddb497f9ca6" target='_blank'>
              Deciphering colorectal cancer radioresistance and immune microrenvironment: unraveling the role of EIF5A through single-cell RNA sequencing and machine learning
              </a>
            </td>
          <td>
            Yaqi Zhong, Xingte Chen, Shiji Wu, Huipeng Fang, Liang Hong, Lingdong Shao, Lei Wang, Junxin Wu
          </td>
          <td>2024-09-03</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2b01750179731a3467c6191264b3a3625ebdeb" target='_blank'>
              Altered methylation of imprinted genes in neuroblastoma: implications for prognostic refinement
              </a>
            </td>
          <td>
            Medha Suman, Maja Löfgren, S. Fransson, Jewahri Idris Yousuf, Johanna Svensson, A. Djos, Tommy Martinsson, P. Kogner, T. Kling, Helena Carén
          </td>
          <td>2024-08-31</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="TRIM44, a tripartite motif (TRIM) family member, is pivotal in linking the ubiquitin-proteasome system (UPS) to autophagy in multiple myeloma (MM). However, its prognostic impact and therapeutic potential remain underexplored. Here, we report that TRIM44 overexpression is associated with poor prognosis in a Multiple Myeloma Research Foundation (MMRF) cohort of 858 patients, persisting across primary and recurrent MM cases. TRIM44 expression notably increases in advanced MM stages, indicating its potential role in disease progression. Single-cell RNA sequencing across MM stages showed significant TRIM44 upregulation in smoldering MM (SMM) and MM compared to normal bone marrow, especially in patients with t(4;14) cytogenetic abnormalities. This analysis further identified high TRIM44 expression as predictive of lower responsiveness to proteasome inhibitor (PI) treatments, underscoring its critical function in the unfolded protein response (UPR) in TRIM44-high MM cells. Our findings also demonstrate that TRIM44 facilitates SQSTM1 oligomerization under oxidative stress, essential for its phosphorylation and subsequent autophagic degradation. This process supports the survival of PI-resistant MM cells by activating the NRF2 pathway, which is crucial for oxidative stress response and, potentially, other chemotherapy-induced stressors. Additionally, TRIM44 counters the TRIM21-mediated suppression of the antioxidant response, enhancing MM cell survival under oxidative stress. Collectively, our discoveries highlight TRIM44’s significant role in MM progression and resistance to therapy, suggesting its potential value as a therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/016505694bfc0a30037baa711cb807808634a1de" target='_blank'>
              TRIM44, a Novel Prognostic Marker, Supports the Survival of Proteasome-Resistant Multiple Myeloma Cells
              </a>
            </td>
          <td>
            Trung Vu, Yuqin Wang, Annaliese Fowler, Anton Simieou, Nami McCarty
          </td>
          <td>2024-08-26</td>
          <td>Cells</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiple myeloma (MM) is a disease characterized by spatiotemporal heterogeneity of tumor clones. Different genetic aberrations can be observed simultaneously in tumor cells from different loci, and as the disease progresses, new subclones may appear. The role of liquid biopsy, which is based on the analysis of tumor DNA circulating in the blood plasma, continues to be explored in MM. Here, we present an analysis of the STR profiles and mutation status of the KRAS, NRAS, and BRAF genes, evaluated in plasma free circulating tumor DNA (ctDNA), CD138+ bone marrow cells, and plasmacytomas. The prospective single-center study included 97 patients, with a median age of 55 years. Of these, 94 had newly diagnosed symptomatic MM, and three had primary plasma cell leukemia. It should be noted that if mutations were detected only in ctDNA, “non-classical” codons were more often affected. A variety of adverse laboratory and clinical factors have been associated with the detection of rare KRAS or NRAS gene mutations in bone marrow or ctDNA, suggesting that these mutations may be factors of an unfavorable prognosis for MM. Liquid biopsy studies provide undeniable fundamental information about tumor heterogeneity and clonal evolution in MM. Moreover, we focus on using liquid biopsy to identify new high-risk factors for MM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ef6c42652b61bb294400045b1d3bc2249d75f9b" target='_blank'>
              Loss of Heterozygosity and Mutations in the RAS-ERK Pathway Genes in Tumor Cells of Various Loci in Multiple Myeloma
              </a>
            </td>
          <td>
            M. Soloveva, M. Solovev, N. Risinskaya, E. Nikulina, I. Yakutik, B. Biderman, Tatiana Obukhova, Y. Chabaeva, Sergej Kulikov, Andrey Sudarikov, L. Mendeleeva
          </td>
          <td>2024-08-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca9ece1f1388440529e828df574be9eda5a88aff" target='_blank'>
              Network modeling links kidney developmental programs and the cancer type-specificity of VHL mutations
              </a>
            </td>
          <td>
            Xiaobao Dong, Donglei Zhang, Xian Zhang, Yun Liu, Yuanyuan Liu
          </td>
          <td>2024-10-03</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Sarcomas constitute a large group of rare, heterogeneous cancers of the bone and soft tissue affecting children and young adults. There are over 80 clinically distinct subtypes of sarcoma, and their heterogeneity is a contributing factor to the moderate and poor survival outcomes observed in primary and metastatic disease, respectively. While they are heterogeneous, many subtypes harbor a balanced chromosomal translocation fusing the Ewing Sarcoma RNA binding protein 1 (EWSR1) gene with a gene encoding a transcription factor (TF). This translocation generates two distinct fusions: a canonical fusion consisting of the N-terminal transactivation domain of EWSR1 and the C-terminal DNA-binding domain of a TF, and a reciprocal fusion fusing the N-terminus of the TF and C-terminal RNA-binding domain of EWSR1. Canonical fusion protein products exhibit aberrant TF and pioneer factor activity, leading to alterations in transcriptional programs and the epigenetic landscape within a cell that contributes to oncogenesis. The functions of the reciprocal fusion proteins, however, have remained elusive. Ewing sarcoma (ES) and clear cell sarcoma (CCS), a bone and soft tissue sarcoma, respectively, are two subtypes that have been shown to express reciprocal fusions. The ES fusion implicates EWSR1 and Friend Leukemia Integration Factor 1 (FLI1), while the CCS fusion involves Activating Transcription Factor 1 (ATF1). A challenge in studying the roles of these and other fusion proteins in sarcoma development is the lack of effective model systems, primarily due to their toxicity upon exogenous introduction into fusion-naïve cells. Here, we outline the development of a cumate-inducible lentiviral vector to study the canonical and reciprocal EWSR1 fusion proteins present in ES and CCS. To track cellular localization of the fusion proteins and distinguish fusions from their respective wildtype proteins, we designed fusion sequences with N-terminal fluorescent tags. Tightly controlled expression of canonical and reciprocal fusion proteins reduces their toxicity when expressed in cells. We have successfully established 293T and HT-1080 cells that stably express these vectors. We have also validated cumate-inducible expression of fluorescently tagged canonical and reciprocal FLI1 and ATF1 fusions at the RNA and protein levels in both cell lines. The canonical fusion proteins produced by our model have the expected effects on expression levels of known downstream canonical fusion target genes. Initial cellular localization studies using our model and fluorescence microscopy indicate that reciprocal fusion proteins may localize in the nucleus, which was previously unknown. Overall, our cumate-inducible model represents a viable system for fusion protein expression. Using this model, we aim to gain insight into the roles of reciprocal fusions in ES and CCS development with hopes of uncovering potentially targetable vulnerabilities, opening the door for the development of new treatments in a patient population in need of novel and efficacious therapies.
 Citation Format: Sarah Gawlak, Joyce Ohm. Development of model systems to investigate the roles of canonical and reciprocal EWSR1 fusion proteins in sarcomagenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B079.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6104d5296aa07ecd63e21655a418e97199e7a8ba" target='_blank'>
              Abstract B079: Development of model systems to investigate the roles of canonical and reciprocal EWSR1 fusion proteins in sarcomagenesis
              </a>
            </td>
          <td>
            Sarah Gawlak, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Combined tracking of clonal evolution and chimeric cell phenotypes could enable detection of the key cellular populations associated with response following therapy, including after allogeneic hematopoietic stem cell transplantation (HSCT). We demonstrate that mitochondrial DNA (mtDNA) mutations co-evolve with somatic nuclear DNA mutations at relapse post-HSCT and provide a sensitive means to monitor these cellular populations. Further, detection of mtDNA mutations via single-cell ATAC with select antigen profiling by sequencing (ASAP-seq) simultaneously determines not only donor and recipient cells, but also their phenotype, at frequencies of 0.1-1%. Finally, integration of mtDNA mutations, surface markers, and chromatin accessibility profiles enables the phenotypic resolution of leukemic populations from normal immune cells, thereby providing fresh insights into residual donor-derived engraftment and short-term clonal evolution following therapy for post-transplant leukemia relapse. As throughput evolves, we envision future development of single-cell sequencing-based post-transplant monitoring as a powerful approach for guiding clinical decision making.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454ae3e4512380b5fd1a0c7892e2f418e6fca333" target='_blank'>
              Tracking rare single donor and recipient immune and leukemia cells after allogeneic hematopoietic cell transplantation using mitochondrial DNA mutations.
              </a>
            </td>
          <td>
            Livius Penter, Nicoletta Cieri, Katie Maurer, M. Kwok, Haoxiang Lyu, Wesley S Lu, Giacomo Oliveira, Satyen H Gohil, I. Leshchiner, Caleb A. Lareau, Leif S. Ludwig, Donna S. Neuberg, Haesook T Kim, Shuqiang Li, L. Bullinger, Jerome Ritz, Gaddy Getz, Jacqueline S Garcia, Robert J. Soiffer, K. Livak, Catherine J. Wu
          </td>
          <td>2024-09-05</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Philadelphia chromosome-positive B-cell acute lymphoblastic leukemia (Ph(+)B-ALL) is a hematological malignancy with a poor prognosis. Epigenetic abnormalities, especially abnormal histone acetylation and microRNAs (miRNAs) dysregulation, are a group of epigenetic patterns that contribute to leukemia progression. However, their regulatory mechanisms in Ph(+)B-ALL have not been fully elucidated. In this study, we identified that miR-183-5p is significantly downregulated in Ph(+)B-ALL and associated with poor prognosis. Moreover, we found that the BCR-ABL fusion gene is a key target gene of miR-183-5p. MiR-183-5p directly targets BCR-ABL gene and induces cell apoptosis via PTEN/AKT and c-MYC signaling pathways. In addition, histone deacetylase inhibitor (HADCi) could mitigate the suppressive effects of HDAC2 on miR-183-5p by promoting promoter acetylation, thereby enhancing cell apoptosis. In conclusion, our results indicate that miR-183-5p is a potential biomarker and suggest that a novel "HDAC2-miR-183-5p epigenetic circuitry regulation" may be involved in the pathogenesis of Ph(+)B-ALL. Taken together, These findings provide new insights into the design of promising molecular-targeted drugs for Ph(+)B-ALL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d13162cad787b4066b1367ec7518bbaa77c34746" target='_blank'>
              HDAC2-miR183-5p Epigenetic Circuit Contributes to the Growth of Philadelphia Chromosome-Positive B-cell Acute Lymphoblastic Leukemia via PTEN/AKT and c-MYC Signaling Pathway.
              </a>
            </td>
          <td>
            Yangyang Ding, Xiangjiang Feng, Zelin Liu, Ya Liao, Lianfang Pu, Jun Liu, Huiping Wang, Zhimin Zhai, Shu-Dao Xiong
          </td>
          <td>2024-09-11</td>
          <td>Journal of leukocyte biology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8b3281c13d1ffc75f1261f03703dc8638c833" target='_blank'>
              A protein expression atlas on tissue samples and cell lines from cancer patients provides insights into tumor heterogeneity and dependencies.
              </a>
            </td>
          <td>
            Jun Li, Wei Liu, K. Mojumdar, Hong Kim, Zhicheng Zhou, Z. Ju, Shweth V. Kumar, P. Ng, Han Chen, Michael A. Davies, Yiling Lu, Rehan Akbani, Gordon B. Mills, Han Liang
          </td>
          <td>2024-09-03</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="
 Fibrolamellar carcinoma (FLC) is characterized by a single genomic alteration, a 400 kB deletion resulting in the fusion transcript DNAJB1::PRKACA, which encodes a fusion oncoprotein essential for tumor initiation and maintenance. This study aims to find therapeutics that can kill FLC cells. We use three model systems: Patient tumors, fresh from resection, that we have made into organoids, that we have implanted into immune compromised mice (PDX), or we have screened directly, immediately after resection. We found lack of efficacy of agents current in the clinic. We have previously characterized the transcriptome of FLC and identified a number of oncogenic genes and pathways that are upregulated including the wnt pathway, EGF, and the wnt pathway. However, agents that blocked these had no effect on tumor survival. We switched to using three different approaches for therapy: i) A functional precision medicine screen using a drug-repurposing library; ii) antisense oligonucleotides against the RNA junction of DNAJB1::PRKACA transcript; iii) Degrader of the DNAJB1::PRKACA fusion protein. Functional precision medicine: We found a number of agents that were extremely efficacious. What they shared in common was pathways of metabolism of these drugs that were down-regulated in FLC. For example, irinotecan, a topoisomerase I inhibitor, was extremely effective. It is removed from liver cells through the addition of a sugar group by UGT1A1 which is decreased at the transcript and protein level. There were some variations in the extent to which patient tumors responded to irinotecan, but the variations could be eliminated by blocking the anti-apoptotic pathway Bcl-xL. We are currently preparing the combination of irinotecan and a PROTAC (proteolysis targeting chimeras) against Bcl-xL for a clinical trial. Antisense oligos: We created shRNA that tiled across the DNAJB1::PRKACA junction and identified some that eliminated DNAJB1::PRKACA at the RNA and protein level with no effect on DNAJB1 and no effect on PRKACA. When these were induced in FLC tumors cells grown as PDX, the tumors not only stopped growing, but shrank. This demonstrates that DNAJB1::PRKACA not only triggers FLC, but also continues to drive FLC and the FLC tumors are oncogenically addicted to DNAJB1::PRKACA. The same shRNA had no detectable effects on non-FLC liver tumors. We next tested siRNA against FLC grown as PDX. The efficacy of the siRNA were increased by conjugation to the sugar GalNAc which binds to the asialyoglycoprotein receptor on FLC cells. Degraders of the oncoprotein: We developed a degrader that selectively degraded the DNAJB1::PRKACA with no detectable effects on the wt PRKACA. This degrader effectively killed FLC tumors growing as PDX. Each of these three approaches represent emerging technologies for pediatric tumors. For each we now have a proof of principle, and our efforts are now focused on improving delivery and studies of efficacy and safety in the hope of moving these into the clinic to provide respite for this usually lethal childhood tumor.
 Citation Format: Mahsa Shirani, Michael Tomasini, Christoph Neumayer, Denise Ng, Gadi Lalazar, Bassem Shebl, Philip Coffino, Barbara A. Lyons, Sanford Simon. Emerging therapies for the treatment of the fusion protein driven cancer, fibrolamellar carcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A068.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2958d83c3d5c2896a0eac3c8e8513329bef49b6" target='_blank'>
              Abstract A068 Emerging therapies for the treatment of the fusion protein driven cancer, fibrolamellar carcinoma
              </a>
            </td>
          <td>
            Mahsa Shirani, Michael D Tomasini, Christoph Neumayer, D. Ng, G. Lalazar, Bassem Shebl, Philip Coffino, Barbara Lyons, Sanford M Simon
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="TET2 mutations (mTET2) are common genetic events in myeloid malignancies and clonal hematopoiesis (CH). These mutations arise in the founding clone and are implicated in many clinical sequelae associated with oncogenic feedforward inflammatory circuits. However, the direct downstream effector of mTET2 responsible for the potentiation of this inflammatory circuit is unknown. To address this, we performed scRNA and scATAC-seq in COVID-19 patients with and without TET2-mutated CH reasoning that the inflammation from COVID-19 may highlight critical downstream transcriptional targets of mTET2. Using this approach, we identified MALAT1, a therapeutically tractable lncRNA, as a central downstream effector of mTET2 that is both necessary and sufficient to induce the oncogenic pro-inflammatory features of mTET2 in vivo. We also elucidate the mechanism by which mTET2 upregulate MALAT1 and describe an interaction between MALAT1 and P65 which leads to RNA "shielding" from PP2A dephosphorylation thus preventing resolution of inflammatory signaling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e647627f6805fe19dbf59a6bc50dfd7de16f1c5" target='_blank'>
              RNA shielding of P65 is required to potentiate oncogenic inflammation in TET2 mutated clonal hematopoiesis.
              </a>
            </td>
          <td>
            N. Ben-Crentsil, Wazim Mohammed Ismail, M. Balasis, Hannah Newman, Ariel Quintana, M. Binder, Traci L. Kruer, Surendra Neupane, M. Ferrall-Fairbanks, Jenna A. Fernandez, T. Lasho, C. Finke, Mohammed L Ibrahim, K. McGraw, Michael Wysota, Amy L Aldrich, Christopher B Ryder, Christopher T Letson, Joshua Traina, Amy F. McLemore, N. Droin, A. Shastri, Seongseok Yun, E. Solary, D. Sallman, Amer A Beg, Li Ma, Alexandre Gaspar-Maia, Mirinal S Patnaik, E. Padron
          </td>
          <td>2024-08-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Immune evasion is not only critical for tumor initiation and progression, but also determines the efficacy of immunotherapies. Through iterative in vivo CRISPR screens with seven syngeneic tumor models, we identified core and context-dependent immune evasion pathways across cancer types. This valuable high-confidence dataset is available for the further understanding of tumor intrinsic immunomodulators, which may lead to the discovery of effective anticancer therapeutic targets. With a focus on triple-negative breast cancer (TNBC), we found that Mga knock-out significantly enhances antitumor immunity and inhibits tumor growth. Transcriptomics and single-cell RNA sequencing analyses revealed that Mga influences various immune-related pathways in the tumor microenvironment. Our findings suggest that Mga may play a role in modulating the tumor immune landscape, though the precise mechanisms require further investigation. Interestingly, we observed that low MGA expression in breast cancer patients correlates with a favorable prognosis, particularly in those with active interferon-γ signaling. These observations provide insights into tumor immune escape mechanisms and suggest that further exploration of MGA's function could potentially lead to effective therapeutic strategies in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a148d1222506ce8171cce7b1609e8c01da8e2a" target='_blank'>
              In vivo CRISPR screens identify Mga as an immunotherapy target in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Xu Feng, Chang Yang, Yuanjian Huang, Dan Su, Chao Wang, L. L. Wilson, Ling Yin, Mengfan Tang, Siting Li, Zhen Chen, Dandan Zhu, Shimin Wang, Shengzhe Zhang, Jie Zhang, Huimin Zhang, Litong Nie, Min Huang, Jae-Ll Park, Traver Hart, Dadi Jiang, Kuirong Jiang, Junjie Chen
          </td>
          <td>2024-09-19</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Inborn errors of metabolism (IEMs) and immunity (IEIs) are Mendelian diseases in which complex phenotypes and patient rarity have limited clinical understanding. Whereas few genes have been annotated as contributing to both IEMs and IEIs, immunometabolic demands suggested greater functional overlap. Here, CRISPR screens tested IEM genes for immunologic roles and IEI genes for metabolic effects and found considerable previously unappreciated crossover. Analysis of IEMs showed that N-linked glycosylation and the hexosamine pathway enzyme Gfpt1 are critical for T cell expansion and function. Further, T helper (TH1) cells synthesized uridine diphosphate N-acetylglucosamine more rapidly and were more impaired by Gfpt1 deficiency than TH17 cells. Screening IEI genes found that Bcl11b promotes the CD4 T cell mitochondrial activity and Mcl1 expression necessary to prevent metabolic stress. Thus, a high degree of functional overlap exists between IEM and IEI genes, and immunometabolic mechanisms may underlie a previously underappreciated intersection of these disorders. The functional overlap of genes that cause inborn errors of metabolism and immunity reveals T cell metabolic regulators. Editor’s summary Inborn errors of metabolism (IEMs) and immunity (IEIs) are two groups of monogenic disorders whose study has offered insights into fundamental human biology. Patterson et al. used CRISPR screens of mouse CD4 T cells disrupting known IEM and IEI genes to test potential overlapping immunometabolic regulators of T cell function. This approach revealed hundreds of previously unappreciated players, including N-linked glycosylation and the de novo hexosamine synthesis enzyme Gfpt1, which were critical for T cell expansion and function. Moreover, Bcl11b, a player in thymic T cell differentiation, promoted mitochondrial activity and prevented metabolic stress in CD4 T cells. —Seth Thomas Scanlon">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b92efd6d1fbfc773538594e8af1740d7df031781" target='_blank'>
              Functional overlap of inborn errors of immunity and metabolism genes defines T cell metabolic vulnerabilities
              </a>
            </td>
          <td>
            Andrew R. Patterson, Gabriel A. Needle, Ayaka Sugiura, Erin Q. Jennings, Channing Chi, KayLee K. Steiner, Emilie L. Fisher, Gabriella L. Robertson, Caroline Bodnya, Janet G. Markle, Ryan D. Sheldon, Russell G. Jones, Vivian Gama, J.C. Rathmell
          </td>
          <td>2024-08-16</td>
          <td>Science Immunology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b5f230111809df7f6f25d631537bdc75559fcde" target='_blank'>
              Mapping spatial organization and genetic cell-state regulators to target immune evasion in ovarian cancer
              </a>
            </td>
          <td>
            C. Yeh, Karmen Aguirre, Olivia Laveroni, Subin Kim, Aihui Wang, Brooke Liang, Xiaoming Zhang, Lucy Han, Raeline Valbuena, M. Bassik, Young-Min Kim, Sylvia K Plevritis, M. Snyder, Brooke E Howitt, Livnat Jerby
          </td>
          <td>2024-08-23</td>
          <td>Nature Immunology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e692b2096b801074500379b186d120038a0326ab" target='_blank'>
              Chromosome 9p trisomy increases stem cells clonogenic potential and fosters T-cell exhaustion in JAK2-mutant myeloproliferative neoplasms
              </a>
            </td>
          <td>
            C. Carretta, S. Parenti, Matteo Bertesi, S. Rontauroli, Filippo Badii, L. Tavernari, E. Genovese, Marica Malerba, Elisa Papa, S. Sperduti, E. Enzo, M. Mirabile, Francesca Pedrazzi, Anita Neroni, Camilla Tombari, Barbara Mora, M. Maffioli, Marco Mondini, M. Brociner, M. Maccaferri, E. Tenedini, Silvia Martinelli, N. Bartalucci, E. Bianchi, L. Casarini, L. Potenza, Mario Luppi, Enrico Tagliafico, P. Guglielmelli, Manuela Simoni, Francesco Passamonti, Ruggiero Norfo, A. Vannucchi, R. Manfredini
          </td>
          <td>2024-08-23</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>108</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f52f7f8ee82929f6197e0c94074a6bfe46589e7" target='_blank'>
              Derivation of human primary prostate epithelial cell lines by differentially targeting the CDKN2A locus along with expression of hTERT
              </a>
            </td>
          <td>
            Jason S Wasserman, Holly Fowle, Rumesa Hashmi, Diba Atar, Kishan R Patel, Amir Yarmahmoodi, Alexander W. Macfarlane, Yinfei Tan, Edna Cukierman, Bojana Gligorijevic, Adam Karami, Kelly A. Whelan, Kerry S. Campbell, Xavier Graña
          </td>
          <td>2024-09-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Lentiviral vector-transduced T-cells were approved by the FDA as gene therapy anti-cancer medications. Little is known about the host genetic variation effects on the safety and efficacy of the lentiviral vector gene delivery system. To narrow this knowledge-gap, we characterized hepatic gene delivery by lentiviral vectors across the Collaborative Cross (CC) mouse genetic reference population. For 24 weeks, we periodically measured hepatic luciferase expression from lentiviral vectors in 41 CC mouse strains. Hepatic and splenic vector copy numbers were determined. We report that CC mouse strains showed highly diverse outcomes following lentiviral gene delivery. For the first time, moderate correlation between mouse strain-specific sleeping patterns and transduction efficiency was observed. We associated two quantitative trait loci (QTLs) with intra-strain variations in transduction phenotypes, which mechanistically relates to the phenomenon of metastable epialleles. An additional QTL was associated with the kinetics of hepatic transgene expression. Genes comprised in the above QTLs are potential targets to personalize gene therapy protocols. Importantly, we identified two mouse strains that open new directions in characterizing continuous viral vector silencing and HIV latency. Our findings suggest that wide-range patient-specific outcomes of viral vector-based gene therapy should be expected. Thus, novel escalating dose-based clinical protocols should be considered.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0029d66d0e5477dfaed003e789d5d1e323d24295" target='_blank'>
              Analysis of hepatic lentiviral vector transduction; implications for preclinical studies and clinical gene therapy protocols
              </a>
            </td>
          <td>
            Peirong Hu, Yajing Hao, Wei Tang, Graham H. Diering, Fei Zou, T. Kafri
          </td>
          <td>2024-08-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="
 Objective: Structural variations (SVs) constitute the greatest source of genomic alterations leading to the oncogenesis of many cancers, including the deadliest pancreatic cancer (PC). However, SVs in PC remain largely undefined due to technological limitations of the traditional short-read sequencing. This study aims to establish a comprehensive signature of genomic SVs in PC. Methods: PacBio whole genome long-read sequencing was employed to investigate the genomic landscape of SVs in eight tissues from four pancreatic cancer patients. A variety of SVs callers, including pbsv, Sniffles, cuteSV, and svim, were comprehensively adopted to improve the accuracy of SVs detection. Many undefined characteristics of germline and somatic SVs were revealed by comparing with multiple previously published databases. Besides, the impacts of structural variations on 3D chromatin organization were analyzed by integrating multi-omics data including Hi-C, RNA seq and ChIP seq. Results: A total of 26,844 and 27,028 non-redundant SVs were identified in human pancreatic cancer and matched para-cancerous tissues, respectively. Notably, among these SVs, 23.97% were novel and had not been previously reported. Germline SVs were further characterized and found enriched in 25 well-known susceptible genes, such as TP63, PVT1, and ABO. Additionally, we proposed a panel of germline SVs with predicted highest pathogenicity, involving STOX1, DSPP, and WRN. Totally, 616 somatic SVs were identified and presented high burdens in patients with lymph node metastasis. We observed that somatic DELs and INSs were significantly more prevalent in repetitive regions than in non-repetitive regions. Interestingly, DELs showed a decrease in the proportion occurring in simple repeat regions but an increase in the proportion occurring in LINE and SINE regions compared to INSs. Moreover, it was observed that deletions within topologically associated domains were associated with local enhanced interactions and disrupted chromatin loops, which might influence H3K27ac modification of histone contributing to enhancer or silencer hijacking. Conclusion: This study provides insights into the genomic signature of SVs in human pancreatic cancer. These findings might shed new light on the complexity of SVs and their pathogenesis by interplaying with chromatin organization.
 Citation Format: Yongxing Du, Xiaohao Zheng, Yunjie Duan, Chengfeng Wang. Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A083.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40a9fb18df1c1c8a73e8c29f87767e5466baa02" target='_blank'>
              Abstract A083: Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing
              </a>
            </td>
          <td>
            Yongxing Du, Xiaohao Zheng, Yunjie Duan, Cheng-Guang Wang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) are the remnants of retroviral germline infections and are highly abundant in the genomes of vertebrates. At one time considered to be nothing more than inert ‘junk’ within genomes, ERVs have been tolerated within host genomes over vast timescales, and their study continues to reveal complex co-evolutionary histories within their respective host species. For example, multiple instances have been characterized of ERVs having been ‘borrowed’ for normal physiology, from single copies to ones involved in various regulatory networks such as innate immunity and during early development. Within the cell, the accessibility of ERVs is normally tightly controlled by epigenetic mechanisms such as DNA methylation or histone modifications. However, these silencing mechanisms of ERVs are reversible, and epigenetic alterations to the chromatin landscape can thus lead to their aberrant expression, as is observed in abnormal cellular environments such as in tumors. In this review, we focus on ERV transcriptional control and draw parallels and distinctions concerning the loss of regulation in disease, as well as their precise regulation in early development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88fcb4c839dae363d4185cc295fc1a0003652e42" target='_blank'>
              Transcription of Endogenous Retroviruses: Broad and Precise Mechanisms of Control
              </a>
            </td>
          <td>
            Abigail S. Jarosz, Julia V. Halo
          </td>
          <td>2024-08-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9918c8bdd0094035be3fb2b57ac7bbf09b9c90ee" target='_blank'>
              Comprehensive analysis of bulk, single-cell RNA sequencing, and spatial transcriptomics revealed IER3 for predicting malignant progression and immunotherapy efficacy in glioma
              </a>
            </td>
          <td>
            Qi Wang, Chunyu Zhang, Ying Pang, Meng Cheng, Rui Wang, Xu Chen, Tongjie Ji, Yuntong Yang, Jing Zhang, Chunlong Zhong
          </td>
          <td>2024-10-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Women of African descent are at an increased risk of developing and dying from aggressive subtypes of breast cancer. A connection between aggressive disease and Western Sub-Saharan African ancestry has been postulated, but it remains largely unknown to what extent breast cancer in Africa is reminiscent of breast cancer in U.S. African American (AA) women who experience disproportionately high mortality rates. We performed ATAC- and RNA-sequencing on 9 human triple-negative breast cancer cell lines of U.S. origin and discovered that African ancestry influences the chromatin landscape, leading to disparate transcription factor (TF) activity and downstream gene expression patterns indicative of an aggressive tumor biology. Here, we describe an ambitious study that employs single-nucleus (sn) ATAC- and RNA-sequencing (snMultiome) of frozen breast tumors to characterize chromatin accessibility and gene expression patterns with single-cell resolution in AA (n=33), Kenyan (n=25), and European American (EA, n=24) women in relation to genetic ancestry, risk factor exposures, clinical characteristics, and 5-year survival. To achieve this, we successfully isolated intact, high-quality single nuclei from archival frozen breast tumor tissue through an optimized combination of enzymatic digestion and automated tissue homogenization. We performed snMultiome sequencing of 82 tumors using the 10x Genomics platform. Following filtering, normalization (SCT for snRNA; LSI for snATAC), peak calling (MACS2), and integration (Harmony), our dataset includes a total of 296,557 nuclei. Cancerous (163,419 nuclei) and non-cancerous (133,138 nuclei) cells were distinguished based on DNA copy number (CopyKat). Within the microenvironment, 11 major immune, epithelial, and stromal cell types were successfully annotated, exhibiting distinct patterns by population group (e.g. AA tumors showed markedly increased abundance of myeloid and T-cells, while Kenyan tumors showed increased abundance of pericytes and fibroblasts, relative to EA). A large number of enriched TFs within each cell type varied significantly by population group, suggesting distinct chromatin accessibility patterns related to genetic ancestry. CD45+, EPCAM- cells were further extracted, clustered, and manually annotated for subpopulations based on known marker genes. We detected 26 distinct immune subpopulations across our samples, including 2 inflammatory macrophage, 6 immunosuppressive macrophage, 1 fibronectin+/thrombospondin+ monocyte, 6 T-cell, 1 natural killer cell, 2 B-cell, 1 mast cell, and 4 dendritic cell subpopulations. Differential gene expression analyses of each immune cell subpopulation, adjusted for age, BMI, and tumor subtype, revealed ancestry-specific gene expression patterns, which could provide novel insights into functional differences that may impact tumor immune response. Current efforts focus on in-depth molecular characterization of ancestry-related differences in the tumor immune environment and distinct signatures present in lethal disease.
 Citation Format: Alexandra R. Harris, Huaitian Liu, Brittany D. Jenkins, Tiffany H. Dorsey, Francis Makokha, Shahin Sayed, Gretchen Gierach, Stefan Ambs. Characterization of the breast tumor immune microenvironment in women of African descent using single-nucleus RNA- and ATAC-sequencing [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C089.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c691a244278d4d06415c78c2b87c77a2297b77d" target='_blank'>
              Abstract C089: Characterization of the breast tumor immune microenvironment in women of African descent using single-nucleus RNA- and ATAC-sequencing
              </a>
            </td>
          <td>
            Alexandra R Harris, Huaitian Liu, Brittany D. Jenkins, Tiffany H Dorsey, Francis W. Makokha, S. Sayed, Gretchen Gierach, S. Ambs
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Immunotherapy has revolutionised the treatment of multiple cancer types, however, these treatments only work for a proportion of patients and biomarkers to predict response are lacking. One correlate of response is the reinvigoration of a subset of CD8 T cells that have an exhausted phenotype and impaired functionality. In order to develop new therapies, reproducible models are required to identify candidate target genes that enables the reversal of key hallmarks of T cell exhaustion. Here we describe the development of an in vitro model by chronically stimulating T cells with their cognate antigen and performed an in depth temporal phenotypic characterisation. This model recapitulates many of the critical hallmarks of exhaustion, including increased expression of canonical exhaustion surface markers, impaired proliferation, reduced cytokine production, decreased release of cytotoxic granules, and metabolic alterations, including dysfunctional mitochondria. These exhaustion hallmarks were validated using an in vivo model and a gene signature identified which robustly define the shared in vitro and in vivo exhausted state. Critically, this signature is also observed in tumour infiltrating T cells from multiple human tumour types, validating the translational potential of this model for discovering and triaging new therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a18835c6854dccdf9d7801dca652067782db4dd" target='_blank'>
              Simulating CD8 T Cell Exhaustion: A Comprehensive Approach
              </a>
            </td>
          <td>
            Andrea J. Manrique-Rincón, Ben Foster, Stuart Horswell, David A. Goulding, David J. Adams, A. Speak
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24eccfe89ea8e14aa08f4e83d855de3df06a6b0" target='_blank'>
              EWS-WT1 fusion isoforms establish oncogenic programs and therapeutic vulnerabilities in desmoplastic small round cell tumors
              </a>
            </td>
          <td>
            Gaylor Boulay, Liliane C. Broye, Rui Dong, Sowmya Iyer, Rajendran Sanalkumar, Yu-Hang Xing, Rémi Buisson, Shruthi Rengarajan, Beverly Naigles, Benoît Duc, A. Volorio, Mary E. Awad, R. Renella, I. Chebib, G. P. Nielsen, Edwin Choy, Gregory M Cote, Lee Zou, Igor Letovanec, Ivan Stamenkovic, Miguel N. Rivera, Nicolò Riggi
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 In the United States and globally, prostate cancer (PCa) mortality rates are the highest among men of African descent. 24% of the disparities in PCa remains even when controlled for access to care and stage at presentation. Moreover, the tumor microenvironment plays an essential role in tumor progression, aggressiveness, therapeutic response, and patient outcomes. Additionally, the association of the genomic findings with patient ancestry and other characteristics, such as tumor biology and transcriptomic alterations, remains poorly understood. Here, we performed a multi-Omics approach (N=447) to unravel the complexity of tumor heterogeneity and understand disease progression & distinct tumor biology influenced by genetic ancestry. We performed Whole Exome (Normal/tumor paired) Sequencing matched with Methyl Seq and Whole transcriptomic Sequencing for three datasets of our African, African American Men (AAM), and European Men (EAM). Additionally, we ran Spatial NanoString high-plex GeoMx-DSP for a total of 118 treatment-naive PCa patients (around 500 ROIs). Simultaneously, we added matched whole transcriptome Sequencing for these patients. The cohort comprised 87 AAM, 3 unknown, and 28 EAM self-reported individuals. To verify the self-reported race, the genomic ancestry was qualified using genotype and Admixture analysis. To further validate differentially expressed genes at the protein level, we performed multi-plex histological staining of 40 markers to determine the spatial resolution and neighborhood clustering within the tumor and the microenvironment. In parallel, we performed scMultiOmics sequencing (scRNA & scATAC-Seq) at a single-cell resolution (6000 cells/sample at 25K reads per cell) from an additional 12 patients (9 AAM & 3 EAM). We reconstructed a 3D model of the distinct tumor microenvironment at subcellular resolution using serially sectioned hematoxylin and eosin- stained tissue sections (CODA technology). Our results demonstrate that patients who self-report as AAM or Nigerian are assigned to high African (> 70%) Ancestry with either Yoruba (Nigeria) and/or Bantu subpopulation in the Sub-Saharan area. Additionally, high African Ancestry patients are diagnosed at a younger age and show advanced pathology stages compared to patients with European Ancestry. AAM of Yoruba descent expresses significantly higher immune-inflammatory signatures (STAT/IFNG-signaling pathway) compared to the Nigerian-Yoruba subpopulation. Our scRNA-Seq analysis shows that AAM has suppressive myeloid cells infiltrate within the tumor cells. The infiltrations of these cells change with age, Gleason Grade, and pathology stage. Moreover, AAM-tumor cells (scATAC-Seq) have increased open Chromatin accessibility with STAT motifs enrichment (p-value;0.0001), which could be the driver regulators of the immune-suppressive signature as well as influence the upregulation of STAT/IFNG signature within African Ancestry patients. Our study provides new insight into how genetic ancestry impacts immune signatures in AAM/African and contributes to PCa racial disparities.
 Citation Format: Isra Elhussin, Ezra Baraban1, Ashley Kiemen, Tamara L Lotan, Cathy Handy Marshall, Emmanuel Antonarakis, Moray J Campbell, Melissa Davis, Michael Dixon, Isaac Kim, Stefan Ambs, Rick Kittles, Adam B Murphy, Clayton Yates. Integrative multi-omics profiling in patients of African descent diagnosed with prostate cancer reveals distinct tumor-promoting immunosuppressive niches at a single-cell level and spatial resolution [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C117.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4100a5d71a839b36fc483aaee40ec75da9220882" target='_blank'>
              Abstract C117: Integrative multi-omics profiling in patients of African descent diagnosed with prostate cancer reveals distinct tumor-promoting immunosuppressive niches at a single-cell level and spatial resolution
              </a>
            </td>
          <td>
            Isra Elhussin, Ezra G. Baraban, Ashley Kiemen, T. Lotan, Cathy Handy Marshall, E. Antonarakis, Moray J. Campbell, Melissa B. Davis, Michael Dixon, Isaac Kim, S. Ambs, Rick A. Kittles, Adam B. Murphy, Clayton C. Yates
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="
 Background: Prostate cancer (PCa) poses unique challenges for Non-Hispanic Black Americans (BA), who often face poorer outcomes than Non-Hispanic White Americans (WA). Even after adjusting for socioeconomic factors, BAs are diagnosed younger and experience more severe clinical effects, potentially due to the tumor microenvironment. Within PCa, Endogenous Retroviruses (ERVs) have been shown to regulate immune response genes and have prognostic value in PCa. Despite constituting 8% of the human, ERVs' role in the tumor microenvironment remains incompletely understood. These findings highlight the need to target microenvironment dysregulation in solid tumor therapies, especially for ERV expression in prostate cancer. Methods: Using RNA-sequencing, we investigated the expression of ERVs in BA (n=18) and WA (n=14) tumor adjacent stroma (TAS) samples. We analyzed reads by mapping them to the human genome (hg38). We then used bedtools and data from the Human Endogenous Retrovirus database (HERVd) to calculate the number of reads at each ERV locus (n=519,060). Differentially expressed ERV elements were then identified in an Audic Claverie Test (fold- change cutoff of 2 and padj < 0.05). The expression of protein-coding genes in the same BA (n=18) and WA (n=14) TAS samples was investigated using RNA-sequencing. Regulation of significant ERV elements were compared to the regulation of nearby immune response genes between the two races. We developed primary stroma (carcinoma associated fibroblast, CAFs) culture of different races with PCa using fresh prostatectomy tissues. To investigate the biological functions of ERVs in CAFs, we developed FANA oligonucleotides (synthetic single- stranded nucleic acid analogs that can modulate gene expression by enzymatic degradation of a target RNA) and corresponding primers to knock down specific ERV sequences near key immune response genes in prostate cancer. Results: We examined 32 prostatectomy specimens of BA (n=18) and WA (n=14) PCa. Of the 5,786 statistically significant differentially expressed ERV elements between BA and WA TAS that were identified, 3,274 elements had an increase in ERV expression for BA TAS, and 2,512 elements had a decrease in ERV expression for BA TAS. 61 MER41 elements were found to be differentially expressed between the two races. The MER family sequences contain binding sites for transcription factors that are crucial in cancer, such as STAT1. This transcription factor can regulate genes involved in cell proliferation, apoptosis, and DNA repair. MER41 is in close proximity (within 5000 base pairs) to IFNA6, an immune response gene. We found both MER41 and IFNA6 to be downregulated in BA TAS, as compared to their WA counterparts. We developed FANA oligonucleotides to knockdown MER41E, MER61-Int, HERVL-Int, MLT1A, and LTR61E1 to investigate their effect on nearby gene expression. Conclusion: The CAF models from PCa patients of different races can be used to explore the biological function of ERV inactivation on anti-tumor immune response genes within the tumor microenvironment.
 Citation Format: Tara S. K. Jennings, Vinay Kumar, Anton N. Nguyen, Michael M. Ittmann, Patricia Castro, Farah Rahmatpanah. Transcriptome analysis of human endogenous retroviral elements in tumor microenvironment of prostate cancer patients of different ancestry [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C092.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba34051bd7dedd2fa8c67fcaee44595e97f04e7" target='_blank'>
              Abstract C092: Transcriptome analysis of human endogenous retroviral elements in tumor microenvironment of prostate cancer patients of different ancestry
              </a>
            </td>
          <td>
            Tara S. K. Jennings, Vinay Kumar, Anton N. Nguyen, Michael M. Ittmann, Patricia Castro, Farah Rahmatpanah
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chemotherapy resistance remains a significant obstacle that limits the long-term efficacy of cancer therapy, necessitating further investigations into the underlying mechanisms. Here, we find that DNA fragments induced by chemotherapeutic agents trigger the degradation of cGAS, a potent double-strand DNA (dsDNA) sensor, by lysosomes. Mechanically, the lysosome-localized protein LAMTOR1 is up-regulated, and the interaction between LAMTOR1 and cGAS is enhanced upon exposure to DNA fragments, boosting the accumulation and digestion of cGAS in lysosomes through the receptor protein p62. LAMTOR1 deficiency increases cGAS abundance and promotes activation of the cGAS-STING pathway, leading to subsequent production of type I interferons induced by cytosolic DNA stimulation. Loss of LAMTOR1 synergizes with immunotherapy and chemotherapy to inhibit tumor growth and prolong the survival time of tumor-bearing mice by promoting the infiltration of effective T lymphocytes. Thus, our study reveals a regulation of cGAS abundance and provides a potential strategy to overcome chemotherapy resistance by targeting LAMTOR1.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa657115aef3dca8540ce49857eda72154fdebd3" target='_blank'>
              LAMTOR1 ablation impedes cGAS degradation caused by chemotherapy and promotes antitumor immunity.
              </a>
            </td>
          <td>
            Juntao Bie, Yutong Li, Chen Song, Q. Weng, Long Zhao, Li Su, Zhongwei Zhao, Yingjiang Ye, Zhanlong Shen, Jiansong Ji, Jianyuan Luo
          </td>
          <td>2024-10-03</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/545dcd839772fedd6284499e3cba3faacdd096c7" target='_blank'>
              Heat shock proteins as hallmarks of cancer: insights from molecular mechanisms to therapeutic strategies
              </a>
            </td>
          <td>
            Weiguo Zuo, Qiwen Pang, Xinyu Zhu, Qian-Qian Yang, Qian Zhao, Gu He, Bo Han, Wei Huang
          </td>
          <td>2024-09-04</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec914b4189eafe0cce08165c6a7c2a53bfc159e" target='_blank'>
              Unveiling FRG1’s DNA repair role in breast cancer
              </a>
            </td>
          <td>
            Shubhanjali Shubhanjali, Talina Mohapatra, Rehan Khan, Manjusha Dixit
          </td>
          <td>2024-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) disease progression involves complex cell state transitions in both malignant and non-malignant populations within the tumor microenvironment. To uncover PDAC cell states and epigenetic features associated with clinical outcome, we profiled 40 tumor samples (33 primary tumors and 7 metastases) from 39 patients with single cell ATAC-seq, along with 10 sample-matched single cell RNA-seq profiles. Through topic modeling of high dimensional chromatin data, we uncovered distinct cis-regulatory networks underlying the classical and basal-like molecular subtypes of PDAC, which showed differential activity in basal-classical hybrid cells. Furthermore, delineation of epigenetic alterations by basal, classical, and hybrid subtypes revealed a core set of chromatin regions opened in all PDAC subtypes relative to ductal and acinar cells. These universally open regions showed striking overlap with gastrointestinal enhancers, highlighting a common endoderm lineage infidelity process across pancreatic tumors. Within the stromal compartment, we identified novel regulators of myofibroblastic cancer-associated fibroblasts (myCAFs) and inflammatory CAFs (iCAFs) from enhancer networks active in each CAF subtype. Importantly, our data uncover a highly prognostic gene program active in only one of two distinct myCAF chromatin states found in our data, highlighting a putative tumor-promoting myCAF subpopulation. Our work provides a high-resolution description of epigenetic cell state heterogeneity in PDAC, revealing novel regulators and gene programs associated with PDAC subtypes and clinical outcome.
 Citation Format: Kevin MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, Carl Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew Adey, Rosalie Sears. Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55da9db59167776bfc809f1ebb52c0cfa3226d09" target='_blank'>
              Abstract C078: Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes
              </a>
            </td>
          <td>
            Kevin A MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, C. Pelz, Trent Waugh, Ethan Agritelle, John L. Muschler, Andrew C. Adey, Rosalie C. Sears
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Genes on sex chromosomes have higher evolutionary rates than those on autosomes. However, this does not necessarily apply to somatic evolution in cancer. Many dominant mutations have been described in the so-called proto-oncogenes (OGs), while recessive mutations are typically described in tumor-suppressor genes (TSGs). Evidence indicates that mutations in X-chromosome TSGs are more likely to contribute to cancer than those in autosomal TSGs. Here, we formalize this in several dynamic models and predict, as expected, that mutations spread faster in TSGs located on the X chromosome than on autosomes (faster-X effect). Conversely, mutations in OGs spread faster on autosomes than on the X chromosome, but under high selective pressure, this difference is negligible. Published genomic screenings of cancer samples show evidence of the faster-X effect in TSGs. This pattern is observed in both sexes, suggesting that the maintenance of X-chromosome inactivation during cancer progression plays an important role in the evolution of TSGs. Strikingly, the relative mutation incidence in X-linked TSGs among females across individual studies is bimodal, with one group of studies showing a faster-X effect and another group showing similar incidences for X-linked and autosomal TSGs. This differentiation between cancer samples is not associated with the specific type of cancer or the tissue of origin. This may indicate that X-chromosome inactivation plays a differential role in the involvement of X-linked TSGs across individual cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e60393ec97b0f4b9cf086f3cab7030b54b21b01" target='_blank'>
              Somatic evolution of cancer genes in sex chromosomes
              </a>
            </td>
          <td>
            Amarachi Akachukwu, Jasmine S. Ratcliff, Antonio Marco
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Multiple myeloma (MM) is an incurable hematological malignancy with high chromosome instability and heavy dependence on the immunosuppressive bone marrow microenvironment. P53 mutations are adverse prognostic factors in MM; however, clinically, some patients without P53 mutations also exhibit aggressive disease progression. DNp73, an inhibitor of TP53 tumor suppressor family members, drives drug resistance and cancer progression in several solid malignancies. Nevertheless, the biological functions of DNp73 and the molecular mechanisms in myelomagenesis remain unclear.The effects of DNp73 on proliferation and drug sensitivity were assessed using flow cytometry and xenograft models. To investigate the mechanisms of drug resistance, RNA-seq and ChIP-seq analyses were performed in MM cell lines, with validation by Western blot and RT-qPCR. Immunofluorescence and transwell assays were used to assess DNA damage and cell invasion in MM cells. Additionally, in vitro phagocytosis assays were conducted to confirm the role of DNp73 in immune evasion.Our study found that activation of NF-κB-p65 in multiple myeloma cells with different p53 mutation statuses upregulates DNp73 expression at the transcriptional level. Forced expression of DNp73 promoted aggressive proliferation and multidrug resistance in MM cells. Bulk RNA-seq analysis was conducted to assess the levels of MYCN, MYC, and CDK7. A ChIP-qPCR assay was used to reveal that DNp73 acts as a transcription factor regulating MYCN gene expression. Bulk RNA-seq analysis demonstrated increased levels of MYCN, MYC, and CDK7 with forced DNp73 expression in MM cells. A ChIP-qPCR assay revealed that DNp73 upregulates MYCN gene expression as a transcription factor. Additionally, DNp73 promoted immune evasion of MM cells by upregulating MYC target genes CD47 and PD-L1. Blockade of the CD47/SIRPα and PD-1/PD-L1 signaling pathways by the SIRPα-Fc fusion protein IMM01 and monoclonal antibody atezolizumab significantly restored the anti-MM activity of macrophages and T cells in the microenvironment, respectively.In summary, our study demonstrated for the first time that the p53 family member DNp73 remarkably induces proliferation, drug resistance, and immune escape of myeloma cells by directly targeting MYCN and regulating the MYC pathway. The oncogenic function of DNp73 is independent of p53 status in MM cells. These data contribute to a better understanding of the function of TP53 and its family members in tumorigenesis. Moreover, our study clarified that DNp73 overexpression not only promotes aggressive growth of tumor cells but, more importantly, promotes immune escape of MM cells through upregulation of immune checkpoints. DNp73 could serve as a biomarker for immunotherapy targeting PD-L1 and CD47 blockade in MM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29c2317b2b047eb0eb6e4a0882a65574253f5b39" target='_blank'>
              DNp73 enhances tumor progression and immune evasion in multiple myeloma by targeting the MYC and MYCN pathways
              </a>
            </td>
          <td>
            Lanting Liu, Dasen Gong, Hao Sun, Fangshuo Feng, Jie Xu, Xiyue Sun, L. Gong, Zhen Yu, Teng-Ching Fang, Yan Xu, R. Lyu, Tingyu Wang, Wentian Wang, Wenzhi Tian, Lugui Qiu, Gang An, Mu Hao
          </td>
          <td>2024-09-24</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Conventional genome editing tools rely on DNA double-strand breaks (DSBs) and host recombination proteins to achieve large insertions, resulting in a heterogeneous mixture of undesirable editing outcomes. We recently leveraged a type I-F CRISPR-associated transposase (CAST) from the Pseudoalteromonas Tn7016 transposon (PseCAST) for DSB-free, RNA-guided DNA integration in human cells, taking advantage of its programmability and large payload capacity. PseCAST is the only characterized CAST system that has achieved human genomic DNA insertions, but multiple lines of evidence suggest that DNA binding may be a critical bottleneck that limits high-efficiency activity. Here we report structural determinants of target DNA recognition by the PseCAST QCascade complex using single-particle cryogenic electron microscopy (cryoEM), which revealed novel subtype-specific interactions and RNA-DNA heteroduplex features. By combining our structural data with target DNA library screens and rationally engineered protein mutations, we uncovered CAST variants that exhibit increased integration efficiency and modified PAM stringency. Structure predictions of key interfaces in the transpososome holoenzyme also revealed opportunities for the design of hybrid CASTs, which we leveraged to build chimeric systems that combine high-activity DNA binding and DNA integration modules. Collectively, our work provides unique structural insights into type I-F CAST systems while showcasing multiple diverse strategies to investigate and engineer new RNA-guided transposase architectures for human genome editing applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e8033198591678345635234a5ea23aaad28664f" target='_blank'>
              Structure-guided engineering of type I-F CASTs for targeted gene insertion in human cells
              </a>
            </td>
          <td>
            George D. Lampe, Ashley R. Liang, Dennis J. Zhang, Israel S. Fernández, S. H. Sternberg
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [0, 15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>