<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-10-07 06:31:58 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>26</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>53</td>
          <td>132</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>102</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, Jens Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>283</td>
          <td>132</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>244</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, Jens Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) have been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterized the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA were flanked by novel somatic DNA enhancers and HPV L1 enhancers, with strong cis-interaction. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduced E6/E7 expression, and significantly inhibited in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with novel somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c00e1f8a07a4acac2f60dea45935a4972a1645" target='_blank'>
              Inhibition of novel human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Joseph Califano, Takuya Nakagawa, Jens Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, S. Sadat, Sara Javadzadeh, Qian Yang, Allen Wang, Kersi Pestonjamasp, Sara Rosenthal, Kathleen M Fisch, Paul S. Mischel, V. Bafna
          </td>
          <td>2024-09-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Rationale: Extrachromosomal circular DNA is a hallmark of cancer, but its role in shaping the genome heterogeneity of urothelial bladder carcinoma (UBC) remains poorly understood. Here, we comprehensively analyzed the features of extrachromosomal circular DNA in 80 UBC patients. Methods: We performed whole-genome/exome sequencing (WGS/WES), Circle-Seq, single-molecule real-time (SMRT) long-read sequencing of circular DNA, and RNA sequencing (RNA-Seq) on 80 pairs of tumor and AT samples. We used our newly developed circular DNA analysis software, Circle-Map++ to detect small extrachromosomal circular DNA from Circle-Seq data. Results: We observed a high load and significant heterogeneity of extrachromosomal circular DNAs in UBC, including numerous single-locus and complex chimeric circular DNAs originating from different chromosomes. This includes highly chimeric circular DNAs carrying seven oncogenes and circles from nine chromosomes. We also found that large tumor-specific extrachromosomal circular DNAs could influence genome-wide gene expression, and are detectable in time-matched urinary sediments. Additionally, we found that the extrachromosomal circular DNA correlates with hypermutation, copy number variation, oncogene amplification, and clinical outcome. Conclusions: Overall, our study provides a comprehensive extrachromosomal circular DNA map of UBC, along with valuable data resources and bioinformatics tools for future cancer and extrachromosomal circular DNA research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0454d85a5ac1363a4fa012edcd2f47cff5be9b32" target='_blank'>
              Extrachromosomal circular DNA orchestrates genome heterogeneity in urothelial bladder carcinoma
              </a>
            </td>
          <td>
            Wei Lv, Xiaoguang Pan, Peng Han, Shuang Wu, Yuchen Zeng, Qingqing Wang, Lidong Guo, Mengyang Xu, Yanwei Qi, Li Deng, Zhe Xu, Conghui Li, Tianxi Yu, Xin Cui, Huajing Teng, Chongjun Xiang, Haotian Tan, Yue Li, Ning Liang, Huiying Tao, Qingqing Gao, Guohua Yu, Jia Mi, Fuyi Xu, Benjiao Gong, Lei Shi, Tao Wang, Huanming Yang, Wei Dong, Lars Bolund, Lin Lin, Wenting Wang, Hanbo Li, Jinrong Huang, Chunhua Lin, Yonglun Luo
          </td>
          <td>2024-08-19</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="DNA that is found outside the main chromosomes in a cell's nucleus is known as extrachromosomal DNA. This type of DNA is not part of the standard 23 pairs of chromosomes in humans and instead exists as standalone circular or linear DNA structures. Unlike bacterial plasmids, extrachromosomal DNA in human cells contains important genetic material and holds particular significance in cancer cells. Critical characteristics of ecDNA in Cancer is that they frequently harbour amplified versions of oncogenes such as MYC, EGFR, and CCND1. These oncogenes propel the growth and survival of tumors. Cancer cells utilize ecDNA to generate multiple copies of these oncogenes, amplifying their expression without requiring alterations to the chromosomal structure. This amplification grants the cancer cells a competitive edge in growth. ecDNA plays a crucial role in the advancement and growth of cancer, as it amplifies oncogenes, genetic diversity, and resistance to treatments. Its adaptable and ever-changing characteristics empower cancer cells to adjust to external influences such as medication. Gaining insights into ecDNA's operations and impacts could lead to new possibilities for diagnosing and treating cancer, as well as developing therapies aimed at this distinct DNA structure.
 ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b548c36ec02cb85af82ad09081289b0df56847d0" target='_blank'>
              Extrachromosomal DNA : New Players in Oncology
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-10-01</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Enhancer elements are specific DNA sequences that play a crucial role in regulating gene expression. Located upstream or downstream in the genomic context, enhancers enhance the transcription of linked genes. This is achieved by providing binding sites for transcription factors and other proteins, acting as a nucleation point that promotes the assembly of the transcriptional machinery.
What makes enhancers unique is that located even thousands of base pairs away, they can exert their function. They can act over long distances, looping to interact with the promoter region of their target genes. Enhancers are often involved in cell type-specific gene expression as different cells express different sets of genes by activating and repressing cell type-specific enhancers.
Cancer cells seem to co-opt this mechanism and hijack it for their own survival. In this process, an aberrant enhancer element activates the transcription of oncogenes (genes that have the potential to cause cancer) due to some alterations in genomic structure or regulatory elements that happen in cancer, contributing to tumorigenesis and cancer progression. Enhancer hijacking often occurs through complex genomic rearrangements such as chromosomal translocations and other structural rearrangements such as deletions or amplifications.8 These alterations can bring a usually distant, far-located enhancer into proximity with an oncogene, leading to its inappropriate activation. This is often seen in various cancers, including leukemias and solid tumors.
Enhancer hijacking and its effects on gene regulation in cancer are frequently studied using methods such as RNA sequencing, chromatin conformation capture, and CRISPR-Cas9. Finding an enhancer hijacking event may be used to inform treatment choices, especially in precision medicine settings, and may also function as a biomarker for particular cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefe88575de8420490e0e1dd10a24dbefe2eef41" target='_blank'>
              Enhancer hijacking: Innovative ways of carcinogenesis
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-09-02</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4921d9b380eadcd80f19284f6396e453cfba83e" target='_blank'>
              Exploring the potential of extrachromosomal DNA as a novel oncogenic driver.
              </a>
            </td>
          <td>
            Huanbo Zhu, Longtao Huangfu, Junbing Chen, Jiafu Ji, Xiaofang Xing
          </td>
          <td>2024-09-27</td>
          <td>Science China. Life sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="An important goal in cancer research is to identify driver genes and mutations. Reasoning that such mutations often alter enzymatic functions, we investigated the cancer driver role of enzyme families. Using pan-cancer genomic data and established driver mutation catalogues, we found an unexpectedly high rate of mutations in helicases, making helicases the most frequently mutated enzyme family in cancer. Based on both functional perturbation screens and cancer genomic analyses, we provide evidence that cancers with mutated helicases converge on increased genomic instability and faulty DNA repair. We identify a striking phenotype in cells with loss of the helicase Aquarius (AQR). AQR was exclusively hemizygous lost in cancer genomes, which was associated with elevated levels of structural variants and point mutation signatures indicative of homologous recombination deficiency. Finally, we leverage large dependency maps to show that hemizygous loss is a common tumour suppression mechanism among helicases. In summary, we uncover a striking frequency of mutated helicases with key roles in genomic maintenance, and we nominate novel hemizygous cancer driver genes including AQR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/381ed30f7e9d5d197bbe8fa9f74d37852c71bb4a" target='_blank'>
              Enzyme family-centred approach identifies helicases as recurrent hemizygous tumour suppressor genes
              </a>
            </td>
          <td>
            Karolin Voßgröne, F. Favero, Krushanka Kashyap, F. G. Rodríguez-González, André Vidas Olsen, Xin Li, Balca R. Mardin, Joachim Weischenfeldt, Claus S. Sørensen
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="Great apes have maintained a stable karyotype with few large-scale rearrangements; in contrast, gibbons have undergone a high rate of chromosomal rearrangements coincident with rapid centromere turnover. Here we characterize assembled centromeres in the Eastern hoolock gibbon, Hoolock leuconedys (HLE), finding a diverse group of transposable elements (TEs) that differ from the canonical alpha satellites found across centromeres of other apes. We find that HLE centromeres contain a CpG methylation centromere dip region, providing evidence this epigenetic feature is conserved in the absence of satellite arrays; nevertheless, we report a variety of atypical centromeric features, including protein-coding genes and mismatched replication timing. Further, large structural variations define HLE centromeres and distinguish them from other gibbons. Combined with differentially methylated TEs, topologically associated domain boundaries, and segmental duplications at chromosomal breakpoints, we propose that a “perfect storm” of multiple genomic attributes with propensities for chromosome instability shaped gibbon centromere evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ad367a909b0a60178d576c36304f1f927c20ef" target='_blank'>
              Centromeric transposable elements and epigenetic status drive karyotypic variation in the eastern hoolock gibbon
              </a>
            </td>
          <td>
            Gabrielle Hartley, M. Okhovat, Savannah J. Hoyt, Emily Fuller, Nicole Pauloski, Nicolas Alexandre, I. Alexandrov, R. Drennan, Danilo Dubocanin, David M. Gilbert, Yizi Mao, Christine McCann, Shane Neph, Fedor Ryabov, Takayo Sasaki, Jessica M. Storer, Derek Svendsen, William Troy, Jackson Wells, LM Core, A. Stergachis, Lucia Carbone, Rachel J. O’Neill
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, Sarah H. Johnson, W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Micronuclei (MN) are a commonly used marker of chromosome instability that form when missegregated chromatin recruits its own nuclear envelope (NE) after mitosis. MN frequently rupture, which results in genome instability, upregulation of metastatic genes, and increased immune signaling. MN rupture is linked to NE defects, but the cause of these defects is poorly understood. Previous work from our lab found that chromosome identity correlates with rupture timing for small MN, i.e. MN containing a short chromosome, with more euchromatic chromosomes forming more stable MN with fewer nuclear lamina gaps. Here we demonstrate that histone methylation promotes rupture and nuclear lamina defects in small MN. This correlates with increased MN size, and we go on to find that all MN have a constitutive nuclear export defect that drives MN growth and nuclear lamina gap expansion, making the MN susceptible to rupture. We demonstrate that these export defects arise from decreased RCC1 levels in MN and that additional loss of RCC1 caused by low histone methylation in small euchromatic MN results in additional import defects that suppress nuclear lamina gaps and MN rupture. Through analysis of mutational signatures associated with early and late rupturing chromosomes in the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset, we identify an enrichment of APOBEC and DNA polymerase E hypermutation signatures in chromothripsis events on early and mid rupturing chromosomes, respectively, suggesting that MN rupture timing could determine the landscape of structural variation in chromothripsis. Our study defines a new model of MN rupture where increased MN growth, caused by defects in protein export, drives gaps in nuclear lamina organization that make the MN susceptible to membrane rupture with long-lasting effects on genome architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6815113609c0299acce4dadc060265ccb48bb9ff" target='_blank'>
              RCC1 depletion drives protein transport defects and rupture in micronuclei
              </a>
            </td>
          <td>
            Molly G. Zych, Maya Contreras, M. Vashisth, Anna E Mammel, Gavin Ha, Emily M. Hatch
          </td>
          <td>2024-09-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Chromosome aberrations (CAs), a genotoxic potential of carcinogens, are believed to contribute to tumorigenesis by chromosomal rearrangements through micronucleus formation. However, there is no direct evidence that proves the involvement of CAs in tumorigenesis in vivo. In the current study, we sought to clarify the involvement of CAs in chemical carcinogenesis using a rat model with a pure CA-inducer hepatocarcinogen, acetamide. Whole-genome analysis indicated that hepatic tumors induced by acetamide treatment for 26-30 weeks showed a broad range of copy number alterations in various chromosomes. In contrast, hepatic tumors induced by a typical mutagen (diethylnitrosamine) followed by a nonmutagen (phenobarbital) did not show such mutational patterns. Additionally, structural alterations such as translocations were observed more frequently in the acetamide-induced tumors. Moreover, most of the acetamide-induced tumors expressed c-Myc and/or MDM2 protein due to the copy number gain of each oncogene. These results suggest the occurrence of chromosomal rearrangements and subsequent oncogene amplification in the acetamide-induced tumors. Taken together, the results indicate that CAs are directly involved in tumorigenesis through chromosomal rearrangements in an acetamide-induced hepatocarcinogenesis rat model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4992ea68f0d5c57d06c0aa2d2039720d0742395b" target='_blank'>
              Chromosome aberrations cause tumorigenesis through chromosomal rearrangements in a hepatocarcinogenesis rat model.
              </a>
            </td>
          <td>
            Kenji Nakamura, Yuji Ishii, S. Takasu, Moeka Namiki, Meili Soma, N. Takimoto, K. Matsushita, M. Shibutani, Kumiko Ogawa
          </td>
          <td>2024-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction: Copy number variations (CNVs) are structural genomic alterations that involve changes in the number of copies of specific DNA regions. These variations can include deletions, duplications, and more complex rearrangements, and play a critical role in cancer progression by amplifying oncogenes, deleting tumor suppressor genes, or altering other key genomic regions. Despite the importance of CNVs in cancer biology, there is a lack of comprehensive resources that aggregate CNV data across multiple cancer types, impeding the exploration of their role in different malignancies. The PanCNV-Explorer was developed to fill this gap by providing a pan-cancer resource that integrates CNV data for over 21 cancer types. Methods: The PanCNV-Explorer was developed, utilizing data from repositories such as COSMIC, DepMap, DGV, and dbVAR. Amplifications and deletions can be visualized, with detailed chromosome-specific plots highlighting CNV patterns. In addition, a genome browser has been implemented to display regions of interest for searching for genes of potential relevance. Results: The database contains over 5 million CNVs from 15,809 samples, spanning 21 primary cancer types. Pathogenic CNVs from ClinVar and DepMap were compared to benign data from DGV and dbVAR, allowing a comprehensive analysis across tissues. CNV patterns were visualized for individual cancers. We demonstrate with use cases, focusing on common oncogenes such as MYC and tumor suppressor genes such as TP53, the ability of the resource to search for cancer driver genes. For example, chromosome 12 in pancreatic cancer showed frequent amplifications in the region containing the KRAS oncogene. In another example, we show how driver gene candidates already described in certain cancer entities can also be found with high frequency in other entities. Conclusion: The PanCNV-Explorer provides a valuable resource for pan-cancer CNV analysis, facilitating the exploration of CNV-driven tumor heterogeneity. The integrated visualization tools and genome browser enable detailed exploration of genomic regions, enhancing cancer research. As a valuable and easy-to-use platform that allows researchers to quantify and compare CNVs across cancer types, the PanCNV-Explorer is an essential tool for advancing cancer genomics research. The main page is available at https://mtb.bioinf.med.uni-goettingen.de/pancnv-explorer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0740ab5d12642c500ab7be4dd721a73d9e32d198" target='_blank'>
              PanCNV Explorer: A pan-cancer database for copy number variations.
              </a>
            </td>
          <td>
            Kevin Kornrumpf, Jürgen Dönitz
          </td>
          <td>2024-09-24</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Microscopy and genomics are both used to characterize cell function, but approaches to connect the two types of information are lacking, particularly at subnuclear resolution. While emerging multiplexed imaging methods can simultaneously localize genomic regions and nuclear proteins, their ability to accurately measure DNA-protein interactions is constrained by the diffraction limit of optical microscopy. Here, we describe expansion in situ genome sequencing (ExIGS), a technology that enables sequencing of genomic DNA and superresolution localization of nuclear proteins in single cells. We applied ExIGS to fibroblast cells derived from an individual with Hutchinson-Gilford progeria syndrome to characterize how variation in nuclear morphology affects spatial chromatin organization. Using this data, we discovered that lamin abnormalities are linked to hotspots of aberrant euchromatin repression that may erode cell identity. Further, we show that lamin abnormalities heterogeneously increase the repressive environment of the nucleus in tissues and aged cells. These results demonstrate that ExIGS may serve as a generalizable platform for connecting nuclear abnormalities to changes in gene regulation across disease contexts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e86969e327346d3a55aa56def236fca89f4284c6" target='_blank'>
              Expansion in situ genome sequencing links nuclear abnormalities to hotspots of aberrant euchromatin repression
              </a>
            </td>
          <td>
            Ajay S. Labade, Zachary D. Chiang, Caroline Comenho, Paul L. Reginato, Andrew C. Payne, Andrew S. Earl, Rojesh Shrestha, Fabiana M. Duarte, Ehsan Habibi, Ruochi Zhang, George M. Church, E. Boyden, Fei Chen, Jason D. Buenrostro
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosomal instability (CIN) generates micronuclei—aberrant extranuclear structures that catalyze the acquisition of complex chromosomal rearrangements present in cancer. Micronuclei are characterized by persistent DNA damage and catastrophic nuclear envelope collapse, which exposes DNA to the cytoplasm. We found that the autophagic receptor p62/SQSTM1 modulates micronuclear stability, influencing chromosome fragmentation and rearrangements. Mechanistically, proximity of micronuclei to mitochondria led to oxidation-driven homo-oligomerization of p62, limiting endosomal sorting complex required for transport (ESCRT)–dependent micronuclear envelope repair by triggering autophagic degradation. We also found that p62 levels correlate with increased chromothripsis across human cancer cell lines and with increased CIN in colorectal tumors. Thus, p62 acts as a regulator of micronuclei and may serve as a prognostic marker for tumors with high CIN. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION A main feature of cancer cells is their high frequency of chromosome segregation errors, a condition known as chromosomal instability (CIN), which is associated with poor prognosis and chemoresistance. CIN leads to the formation of micronuclei—abnormal extranuclear bodies widely found in cancer cells. The nuclear envelopes of micronuclei are often fragile and compromised, which causes irreparable ruptures and structural collapse. This exposes micronuclear DNA to the cytosol, resulting in DNA damage and extensive chromosomal rearrangements, which fuels genomic instability and cancer progression. In addition, rupture of the micronuclear membrane activates the cGAS-STING [cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING)] pathway, triggering inflammatory responses that promote tumor invasion and metastasis. RATIONALE Although the consequences of loss of micronuclei compartmentalization are well studied, the events leading to micronuclear envelope rupture and the mechanisms involved in their repair are less clear. Given the importance of those mechanisms in maintaining micronuclei integrity, we aimed to identify proteins and pathways that specifically modulate these events. We isolated micronuclei and primary nuclei, analyzed their protein composition using mass spectrometry, and identified p62/SQSTM1—a key autophagic pathway component—as specifically enriched in micronuclei compared with primary nuclei. RESULTS We found that p62 localizes to micronuclei and is crucial for maintaining their integrity. The localization of p62 to micronuclei relied on its oxidation-driven homo-oligomerization, which was induced by reactive oxygen species (ROS) released by mitochondria proximal to micronuclei. We used electron tomography and quantitative imaging to analyze the proximity of mitochondria to micronuclei and found that p62-positive micronuclei were closer to mitochondria compared with those without p62. We tested how mitochondrial ROS affected p62 localization and found that reducing ROS decreased p62-positive micronuclei and prevented their rupture, whereas increasing ROS led to more p62-positive and ruptured micronuclei in both normal and cancer cells. Homo-oligomerization of p62 induced by ROS influenced micronuclear integrity by inhibiting the repair activity of the endosomal sorting complex required for transport–III (ESCRT-III), which is involved in envelope resealing. This control was achieved through the p62-mediated autophagic degradation of ESCRT components. The p62-dependent regulation of micronuclear integrity affected several features associated with micronuclei, such as chromosomal rearrangements and cGAS-STING–mediated inflammation. Furthermore, p62 levels correlated with chromothripsis status in cancer cells and tumor samples and were strongly linked to poor prognosis in colorectal cancer with high CIN. CONCLUSION In this work, we identified a critical mechanism by which p62 regulates micronuclear stability by influencing envelope repair. Given p62’s frequent deregulation in tumors and its control over micronuclear envelope integrity, our findings suggest that p62-mediated effects on micronuclear repair activities could influence cancer development and progression. p62/SQSTM1 drives micronuclei collapse and complex chromosome rearrangements by restraining micronuclear envelope repair. Micronuclei-mitochondria proximity promotes ROS-mediated homo-oligomerization of p62/SQSTM1 through cysteine oxidation. ROS-induced p62 homo-oligomerization affects micronuclear integrity by inhibiting components of ESCRT-III repair through their autophagic degradation. This affects chromosomal rearrangements and cGAS-STING inflammation. High p62 levels correlate with chromothripsis and poor prognosis in colorectal cancer with high CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/180c1ddc6e5e2000c0d5181084de24b29d56eee4" target='_blank'>
              A p62-dependent rheostat dictates micronuclei catastrophe and chromosome rearrangements
              </a>
            </td>
          <td>
            Sara Martin, Simone Scorzoni, Sara Cordone, Alice Mazzagatti, Galina V Beznoussenko, Amanda L. Gunn, Melody Di Bona, Yonatan Eliezer, Gil Leor, Tal Ben-Yishay, A. Loffreda, V. Cancila, Maria Chiara Rainone, M. Ippolito, Valentino Martis, Fabio Bedin, M. Garrè, L. P. Vaites, Paolo Vasapolli, Simona Polo, D. Parazzoli, C. Tripodo, Alexander A. Mironov, A. Cuomo, Uri Ben-David, S. Bakhoum, Emily M. Hatch, Peter Ly, S. Santaguida
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>3</td>
          <td>36</td>
        </tr>

        <tr id="Despite the general detriment of aneuploidy to cellular fitness, >90% of solid tumors carry an imbalanced karyotype. Regardless of this existing paradox, our understanding of the molecular responses to aneuploidy remains limited. Here, we explore these cellular stresses and unique vulnerabilities in aneuploid human mammary epithelial cells (HMECs) enriched for breast cancer-associated copy number alterations (CNAs). To uncover the genetic dependencies specific to aneuploid cells, we conducted a comprehensive, genome-wide CRISPR knockout screen targeting isogenic diploid and aneuploid HMEC lines. Our study reveals that aneuploid HMECs exhibit an increased reliance on pyrimidine biosynthesis and mitochondrial oxidative phosphorylation genes, and demonstrate heightened fitness advantages upon loss of tumor suppressor genes. Using an integrative multi-omic analysis, we confirm nucleotide pool insufficiency as a key contributor to widespread cellular dysfunction in aneuploid HMECs with net copy number gain. While diploid cells can switch seamlessly between pyrimidine synthesis and salvage, cells with increased chromosomal content exhibit p53 activation and S-phase arrest when relying on salvage alone, and exhibit increased sensitivity to DNA-damaging chemotherapeutics. This work advances our understanding of the consequences of aneuploidy and uncovers potential avenues for patient stratification and therapeutic intervention based on tumor ploidy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f75b91959ba71d0bf04e734b4ee5bf5730bc793" target='_blank'>
              An aneuploidy epistasis map reveals metabolic vulnerabilities associated with supernumerary chromosomes in cancer
              </a>
            </td>
          <td>
            R. Magesh, A. N. Kaur, F. Keller, A. Frederick, T. Tseyang, J. A. Haley, A. M. Rivera-Nieves, A. Liang, D. A. Guertin, J. B. Spinelli, S. Elledge, E. Watson
          </td>
          <td>2024-10-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="CIZ1 is part of the large RNA-dependent supramolecular assemblies that form around the inactive X-chromosome (Xi) in female cells, and smaller assemblies throughout the nucleus in males and females. Here we show that CIZ1 C-terminal anchor domain (AD) is elevated in primary human breast tumour transcriptomes, even at stage I. Elevation of AD correlates with deprotection of chromatin, and up-regulation of lncRNA-containing gene clusters in approximately 10Mb regions that are enriched in cancer-associated genes. We modelled the effect of ectopic AD on endogenous CIZ1-Xi assemblies and observed dominant-negative interference with their re-formation after mitosis, leading to abnormal Xi assemblies similar to those in breast cancer cells, and depletion of histone modifications H2AK119ub1 and H3K27me3. Within days consistent alterations in gene expression were evident across the genome, showing that disruption of CIZ1-RNA assemblies has a destabilizing effect, likely by unscheduled exposure of underlying chromatin to modifying enzymes. Together the data argue for a dominant, potent and rapid effect of CIZ1 AD, that can deprogram established patterns of gene expression and which may predispose incipient tumours to epigenetic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44d2a58fd6870a0dd51c6ac1e82db8148da5b9e2" target='_blank'>
              Epigenetic deprogramming driven by disruption of CIZ1-RNA nuclear assemblies in early-stage breast cancers
              </a>
            </td>
          <td>
            Gabrielle L. Turvey, Ernesto López de Alba, Emma R Stewart, Heather Cook, Ahmad Alalti, Richard T. Gawne, J. Ainscough, Andrew S. Mason, Dawn Coverley
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly toxic lesions that underly the efficacy of ionizing radiation (IR) and a large number of cytotoxic chemotherapies1–3. Yet, abnormal repair of DSBs is associated with genomic instability and may contribute to cancer heterogeneity and tumour evolution. Here, we show that DSBs induced by IR, by DSB-inducing chemotherapeutics, or by the expression of a rare-cutting restriction endonuclease induce large-scale genomic amplification in human cancer cells. Importantly, the extent of DSB-induced genomic amplification (DIGA) in a panel of melanoma cell lines correlated with the degree of cytotoxicity elicited by IR, suggesting that DIGA contributes significantly to DSB-induced cancer cell lethality. DIGA, which is mediated through conservative DNA synthesis, does not require origin re-licensing, and is enhanced by the depletion or deletion of the methyltransferases SET8 and SUV4-20H1, which function sequentially to mono- and di-methylate histone H4 lysine 20 (H4K20) at DSBs to facilitate the recruitment of 53BP1-RIF1 and its downstream effector shieldin complex to DSBs to prevent hyper-resection4–11. Consistently, DIGA was enhanced in cells lacking 53BP1 or RIF1, or in cells that lacked components of the shieldin complex or of other factors that help recruit 53BP1 to DSBs. Mechanistically, DIGA requires MRE11/CtIP and EXO1, factors that promote resection and hyper-resection at DSBs, and is dependent on the catalytic activity of the RAD51 recombinase. Furthermore, deletion or depletion of POLD3, POLD4, or RAD52, proteins involved in break-induced replication (BIR), significantly inhibited DIGA, suggesting that DIGA is mediated through a RAD51-dependent BIR-like process. DIGA induction was maximal if the cells encountered DSBs in early and mid S-phase, whereas cells competent for homologous recombination (in late S and G2) exhibited less DIGA induction. We propose that unshielded, hyper-resected ends of DSBs may nucleate a replication-like intermediate that enables cytotoxic long-range genomic DNA amplification mediated through BIR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec06bf7ab705064fc487b428fdb7631566b4ae0f" target='_blank'>
              Break-induced replication drives large-scale genomic amplifications in cancer cells
              </a>
            </td>
          <td>
            Mouadh Benamar, Rebeka Eki, Kangping Du, Tarek Abbas
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The 10q11.22 chromosomal region is a duplication-rich interval of the human genome and one of the last to be fully assembled. It carries copy-number variable genes associated with intellectual disability, bipolar disorder, and obesity. In this study, we characterized the structural diversity at this locus by analyzing 64 haploid assemblies produced by the Human Pangenome Reference Consortium. We identified eleven alternative haplotypes that differ in the copy number and/or orientation of large genomic segments, ranging from hundreds of kilobase pairs (kbp) to over one megabase pair (Mbp). We uncovered a 2.4 Mbp size difference between the shortest and longest haplotypes. Breakpoint analysis revealed that genomic instability results from nonallelic homologous recombination between segmental duplication (SD) pairs with varying similarity (94.4-99.6%). Nonetheless, these pairs generally recombine at positions where their identity is higher (>99.6%). Recurrent inversions occur with varying breakpoints within the same inverted SD pair. Inversion polymorphisms shuffle the entire SD arrangement, creating new predispositions to copy-number variations. The SD architecture is associated with a catarrhine-specific subgroup of the AGAP gene family, which likely triggered the accumulation of SDs at this locus over the past 25 million years of human evolution. Our results reveal extensive structural diversity and genomic instability at the 10q11.22 locus and expand the general understanding of the mutational mechanisms behind SD-mediated rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b45817e56dbf9d949483be26e4d2690cbf8b3395" target='_blank'>
              AGAP duplicons associate with structural diversity at Chromosome 10q11.22.
              </a>
            </td>
          <td>
            Stefania Fornezza, V. S. Delvecchio, William T. Harvey, Philip C. Dishuck, Evan E. Eichler, Giuliana Giannuzzi
          </td>
          <td>2024-09-25</td>
          <td>Genome research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Continuous chromosome missegregation over successive mitotic divisions, known as chromosomal instability (CIN), is common in cancer. Increasing CIN above a maximally tolerated threshold leads to cell death due to loss of essential chromosomes. Here, we show in two tissue contexts that otherwise isogenic cancer cells with higher levels of CIN are more sensitive to ionizing radiation, which itself induces CIN. CIN also sensitizes HPV-positive and HPV-negative head and neck cancer patient derived xenograft (PDX) tumors to radiation. Moreover, laryngeal cancers with higher CIN prior to treatment show improved response to radiation therapy. In addition, we reveal a novel mechanism of radiosensitization by docetaxel, a microtubule stabilizing drug commonly used in combination with radiation. Docetaxel causes cell death by inducing CIN due to abnormal multipolar spindles rather than causing mitotic arrest, as previously assumed. Docetaxel-induced CIN, rather than mitotic arrest, is responsible for the enhanced radiation sensitivity observed in vitro and in vivo, challenging the mechanistic dogma of the last 40 years. These results implicate CIN as a potential biomarker and inducer of radiation response, which could provide valuable cancer therapeutic opportunities. Statement of Significance Cancer cells and laryngeal tumors with higher chromosome missegregation rates are more sensitive to radiation therapy, supporting chromosomal instability as a promising biomarker of radiation response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324e0283c2a65fbf5296691fea31358474092654" target='_blank'>
              Chromosomal instability increases radiation sensitivity
              </a>
            </td>
          <td>
            P. Cosper, M. Paracha, Kathryn M. Jones, Laura C. F. Hrycyniak, Les Henderson, Ava Bryan, D. Eyzaguirre, Emily McCunn, Elizabeth Boulanger, Jun Wan, K. Nickel, Vanessa L Horner, Rong Hu, Paul M Harari, R. Kimple, B. Weaver
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Oncogenic fusions of EML4 and ALK occur in ∼5% of lung adenocarcinomas. More than 15 EML4-ALK variants with distinct breakpoints within EML4 have been identified, but the functional differences between these variants remain poorly understood. Here we use CRISPR/Cas9 somatic genome editing to generate autochthonous mouse models of the two most common EML4-ALK variants, V1 and V3, and show that V3 is more oncogenic than V1. By integrating these models with multiplexed genome editing, we quantify the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth in vivo and show that many tumor suppressor genes have dramatically variant-specific effects on tumorigenesis. Analysis of a novel dataset representing the largest human EML4-ALK lung cancer cohort to date identified alterations in the genomic landscape depending on the EML4-ALK variant. These findings demonstrate functional heterogeneity among EML4-ALK variants, suggesting that EML4-ALK variants behave more like distinct oncogenes than a uniform entity. More broadly, these findings highlight the dramatic impact of oncogenic fusions partner proteins on tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f320bc764c0ea759a77487f2f2744f6c0fd41b3d" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Oscar Gonzalez, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, S. Sivakumar, Saumya D. Sisoudiya, Ethan S. Sokol, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="The family of LINE1 transposable elements underwent a massive expansion in mammalian genomes. While traditionally viewed as a mutagenic selfish element, recent studies point to roles for LINE1 in early mouse development, T cell quiescence and neurogenesis. Here we show that human LINE1 RNA is essential for self-renewal and identity of human embryonic stem cells (hESCs). Silencing of LINE1 using either antisense oligonucleotides or CRISPR interference in naïve hESCs leads to a strong induction of 8C-like cells (8CLCs). We found that genes derepressed upon LINE1 KD are not uniformly distributed across the genome, with an enrichment for chromosome 19, which includes key markers of the 8C state such as TPRX1. Silencing of TPRX1, but not other putative 8C regulators p53 or H3.XY, suppresses the induction of the 8C program in LINE1 KD hESCs. We found that LINE1 RNA is preferentially localized to the lamina and periphery of the nucleolus in hESCs. Sequencing of Lamina-Associated Domains (LADs) and Nucleolus-Associated Domains (NADs) reveals a preferential association of chromosome 19 with NADs in hESCs. However, 8CLCs have a distinct nucleolar morphology and a lower association of chromosome 19 and TPRX1 loci with the nucleolus relative to naïve and primed hESCs, suggesting a role for nucleolar dynamics in the 8CLC-hESC transition. In agreement, LINE1 KD leads to disruption of nucleolar architecture with signs of nucleolar stress. Independent perturbations of the nucleolus induce the 8C program in hESCs. Genes induced by LINE1 KD are enriched for targets of Polycomb Repressive Complex (PRC2), and inhibition of PRC2 leads to a strong induction of 8C genes. Our results indicate that LINE1 coordinates nuclear compartmentalization and chromatin-mediated gene repression to prevent developmental reversion of hESCs. Highlights Knockdown of LINE1 induces TPRX1-dependent emergence of 8C-like cells in hESCs. Genes de-repressed upon LINE1 KD are enriched for Chr 19 and PRC2 targets. 8CLCs display dissociation of Chromosome 19 from the nucleolus. Disruption of the nucleolus or inhibition of PRC2 strongly induce the 8C program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34781a9454759e1cbbdf47b8d98d4f390cccbdd4" target='_blank'>
              LINE1 promotes nuclear compartmentalization to repress the 8-cell state in embryonic stem cells
              </a>
            </td>
          <td>
            Juan Zhang, Lamisa Ataei, Liang Wu, Kirti Mittal, Linh Huynh, Shahil Sarajideen, Abdul Mazid, David P. Cook, D. Trcka, Kevin Tse, Jeffrey L. Wrana, Michael M. Hoffman, Miguel A Esteban, M. Ramalho-Santos
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46a593c69280acd8e1ddd89d1994ea815f977cea" target='_blank'>
              Chromosomal instability: a key driver in glioma pathogenesis and progression
              </a>
            </td>
          <td>
            Adele Mazzoleni, W. A. Awuah, Vivek Sanker, H. Bharadwaj, Nicholas Aderinto, J. K. Tan, H. Huang, Jeisun Poornaselvan, Muhammad Hamza Shah, O. Atallah, Aya Tawfik, Mohamed Elmanzalawi, Sama Hesham Ghozlan, T. Abdul-Rahman, Jeremiah Adepoju Moyondafoluwa, Athanasios Alexiou, Marios Papadakis
          </td>
          <td>2024-09-04</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e11890b80227e402b42551347179de59480a8fc" target='_blank'>
              HMGA1 orchestrates chromatin compartmentalization and sequesters genes into 3D networks coordinating senescence heterogeneity
              </a>
            </td>
          <td>
            Ioana Olan, Masami Ando-Kuri, Aled J. Parry, Testuya Handa, S. Schoenfelder, Peter Fraser, Yasuyuki Ohkawa, Hiroshi Kimura, M. Narita, Masashi Narita
          </td>
          <td>2024-08-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Chromatin is not randomly distributed within the nucleus, but organized in a three-dimensional structure that plays a critical role in genome functions. Сohesin and condensins are conserved multi-subunit protein complexes that participate in mammalian genome organization by extruding chromatin loops. The fine temporal regulation of these complexes is facilitated by a number of other proteins, one of which is microcephalin (Mcph1). Mcph1 prevents condensin II from associating with chromatin through interphase. Loss of Mcph1 induces chromosome hypercondensation; it is not clear to what extent this reorganization affects gene expression. In this study, we generated several mouse embryonic stem cell (mESC) lines with knockout of the Mcph1 gene and analyzed their gene expression profile. Gene Ontology analyses of differentially expressed genes (DEGs) after Mcph1 knockout revealed gene categories related to general metabolism and olfactory receptor function but not to cell cycle control previously described for Mcph1. We did not find a correlation between the DEGs and their frequency of lamina association. Thus, this evidence questions the hypothesis that Mcph1 knockout-mediated chromatin reorganization governs gene expression in mESCs. Among the negative effects of Mcph1 knockout, we observed numerous chromosomal aberrations, including micronucleus formation and chromosome fusion. This confirms the role of Mcph1 in maintaining genome integrity described previously. In our opinion, dysfunction of Mcph1 may be a kind of “Rosetta stone” for deciphering the function of condensin II in the interphase nucleus. Thus, the cell lines with knocked-out Mcph1 can be used to further study the influence of chromatin structural proteins on gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34d65d4d4365d60abd6b73f946f30c7616d79df9" target='_blank'>
              Generation and analysis of mouse embryonic stem cells with knockout of the Mcph1 (microcephalin) gene
              </a>
            </td>
          <td>
            A. Yunusova, A. V. Smirnov, T. Shnaider, I. Pristyazhnuk, S.Y. Korableva, N. R. Battulin
          </td>
          <td>2024-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0943aff1f710e095be6a9439cd65322a1fc6aa0e" target='_blank'>
              Genomic Balancing Act: deciphering DNA rearrangements in the complex chromosomal aberration involving 5p15.2, 2q31.1, and 18q21.32.
              </a>
            </td>
          <td>
            Zain Dardas, Dana Marafi, Ruizhi Duan, Jawid M Fatih, O. El-Rashidy, Christopher M. Grochowski, Claudia M B Carvalho, S. Jhangiani, W. Bi, Haowei Du, Richard A Gibbs, J. Posey, D. Calame, Maha S. Zaki, James R. Lupski
          </td>
          <td>2024-09-10</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Harnessing DNA double-strand breaks (DSBs) is a powerful approach for gene editing, but it may provoke loss of heterozygosity (LOH), which predisposes to tumorigenesis. To interrogate this risk, we developed a two- color flow cytometry-based system (Flo-LOH), detecting LOH in ∼5% of cells following a DSB. After this initial increase, cells with LOH decrease due to a competitive disadvantage with parental cells, but if isolated, they stably propagate. Segmental loss from terminal deletions with de novo telomere addition and nonreciprocal translocations is observed as well as whole chromosome loss, especially following a centromeric DSB. LOH spans megabases distal from the DSB, but also frequently tens of megabases centromere-proximal. Inhibition of microhomology-mediated end joining massively increases LOH, which is synergistically increased with concomitant inhibition of canonical nonhomologous end joining. The capacity for large-scale LOH must therefore be considered when using DSB-based gene editing, especially in conjunction with end joining inhibition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d1b33368b2fb7ae9c66926efe7d4fcbd491246" target='_blank'>
              Megabase-scale loss of heterozygosity provoked by CRISPR-Cas9 DNA double-strand breaks
              </a>
            </td>
          <td>
            Samantha B. Regan, Darpan Medhi, Travis B. White, Yi-Zhen Jiang, Su Jia, Qichen Deng, M. Jasin
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="To comprehensively unravel the temporal relationship between initiating and driver events and its impact on clinical outcomes, we analyzed 421 whole-genome sequencing profiles from 382 patients. Using clock-like mutational signatures, we estimated a time lag of 2-4 decades between initiating events and diagnosis. In patients with hyperdiploidy, we demonstrate that trisomies of odd-numbered chromosomes can be acquired simultaneously with other chromosomal gains, such as 1q gain. We provide evidence that hyperdiploidy is acquired after canonical IGH translocation when both events are present. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (>1 extra-copies), but faring worse than those with late 1q gain. This underscores that the prognostic impact of 1q gain/amp depends more on the timing of acquisition than on the number of extra copies gained. Overall, this study contributes to a better understanding of the life history of MM and may have prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f98e16f03f4610ad7609f6b41fac703153b2f8eb" target='_blank'>
              Temporal Genomic Dynamics Shape Clinical Trajectory in Multiple Myeloma
              </a>
            </td>
          <td>
            Francesco Maura, Marcella Kaddoura, A. Poos, L. Baughn, B. Ziccheddu, Marc-Andrea Bärtsch, Anthony Cirrincione, K. Maclachlan, M. Chojnacka, B. Diamond, M. Papadimitriou, P. Blaney, L. John, Philipp Reichert, S. Huhn, Dylan Gagler, Yanming Zhang, Ahmet Dogan, A. Lesokhin, Faith Davies, Hartmut Goldschmidt, Roland Fenk, Katja C. Weisel, E. Mai, N. Korde, Gareth Morgan, S. Rajkumar, Shaji Kumar, S. Usmani, O. Landgren, Marc-Steffen Raab, Niels Weinhold
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6d9632a110490aa181f31d215bebe837e3899f41" target='_blank'>
              Inferring replication timing and proliferation dynamics from single-cell DNA sequencing data
              </a>
            </td>
          <td>
            A. Weiner, Marc J. Williams, Hongyu Shi, I. Vázquez-García, Sohrab Salehi, Nicole Rusk, Sam Aparicio, Sohrab P. Shah, Andrew McPherson
          </td>
          <td>2024-10-01</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Upon infection, human papillomavirus (HPV) manipulates host cell gene expression to create an environment that is supportive of a productive and persistent infection. The virus-induced changes to the host cell’s transcriptome are thought to contribute to carcinogenesis. Here, we show by RNA-sequencing that oncogenic HPV18 episome replication in primary human foreskin keratinocytes (HFKs) drives host transcriptional changes that are consistent between multiple HFK donors. We have previously shown that HPV18 episome replication in HFKs results in post-transcriptional stabilisation of the host chromatin insulation protein CTCF. Since CTCF is an important regulator of host cell transcription via coordination of epigenetic boundaries and long-range chromosomal interactions, we hypothesised that HPV18-induced stabilisation of CTCF may contribute to host transcription reprogramming. Analysis of CTCF binding in the host cell genome by ChIP-Seq revealed that while the total number of CTCF binding sites is not altered by the virus, there are a sub-set of CTCF binding sites that are either enriched or depleted of CTCF. Many of these altered sites are clustered within regulatory elements of differentially expressed genes, including the tumour suppressor gene cell adhesion molecule 1 (CADM1), which supresses epithelial cell growth and invasion. We show that HPV18 establishment results in reduced CTCF binding at the CADM1 promoter and upstream enhancer. Loss of CTCF binding is coincident with epigenetic repression of CADM1, in the absence of CpG hypermethylation, while adjacent genes including the transcriptional regulator ZBTB16 are activated. These data indicate that the CADM1 locus is subject to topological rearrangement following HPV18 establishment. We tested this hypothesis using 4C-Seq (circular chromosome confirmation capture-sequencing) and show that HPV18 establishment causes a loss of long-range chromosomal interactions between the CADM1 transcriptional start site and the upstream transcriptional enhancer. These data show that HPV18 manipulates host cell promoter-enhancer interactions to drive transcriptional reprogramming that may contribute to HPV-induced disease progression. AUTHOR SUMMARY Infection with oncogenic HPV is the cause of numerous cancer types, which generally arise after persistent HPV infection. Upon infection, HPV alters the gene expression profile of infected cells to facilitate virus replication and persistence. Multiple mechanisms of HPV-induced host cell reprogramming have been previously suggested. Here, we show that HPV infection induces rearrangement of specific genomic loci by altering the chromatin binding of the host cell protein CTCF, an important regulator of chromatin architecture. Loss of CTCF binding to a cluster of binding sites at the CADM1 locus on chromosome 11 is coincident with epigenetic reprogramming and disruption of long-range chromatin interactions, resulting in transcriptional repression of CADM1. Our data show that repression of CADM1 is an early event in HPV-driven disease, preceding hypermethylation of the CADM1 transcriptional promoter that is frequently observed in HPV-driven cancers, demonstrating a novel mechanism of HPV-induced host cell transcriptional reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296755d464d45666368bedcd29a51e084375e664" target='_blank'>
              Repression of CADM1 transcription by HPV type 18 is mediated by three-dimensional rearrangement of promoter-enhancer interactions
              </a>
            </td>
          <td>
            Karen Campos-León, J. Ferguson, Thomas Günther, C. D. Wood, S. Wingett, Selin Pekel, C. S. Varghese, C. Várnai, Michelle West, Andrew Beggs, A. Grundhoff, B. Noyvert, Sally Roberts, Joanna L. Parish
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="H3K36 methylation is emerging as a key epigenetic modification with strong implications in genetic disease and cancer. However, the mechanisms through which H3K36me impacts the epigenome and asserts its functional consequences are far from understood. Here, we use mouse mesenchymal stem cell lines with successive knockouts of the H3K36 methyltransferases: NSD1, NSD2, SETD2, NSD3, and ASH1L, which result in progressive depletion of H3K36me and its complete absence in quintuple knockout cells, to finely dissect the role of H3K36me2 in shaping the epigenome and transcriptome. We show that H3K36me2, which targets active enhancers, is important for maintaining enhancer activity, and its depletion results in downregulation of enhancer-dependent genes. We demonstrate the roles of H3K36me2/3 in preventing the invasion of gene bodies by the repressive H3K27me modifications. Finally, we observe a previously undescribed relationship between H3K36me and H3K9me3: Following the depletion of H3K36me2, H3K9me3 is redistributed away from large heterochromatic domains and towards euchromatin. This results in a drastic decompartmentalization of the genome, weakening the boundaries between active and inactive compartments, and a catastrophic loss of long-range inter-compartment interactions. By studying cells totally devoid of H3K36 methyltransferase activity, we uncover a broad range of crucial functions of H3K36me in maintaining epigenome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c996ce7c85d697b9c8fb195bf8183e8099242ceb" target='_blank'>
              H3K36 Methylation - a Guardian of Epigenome Integrity
              </a>
            </td>
          <td>
            Reinnier Padilla, Gerry A. Shipman, Cynthia Horth, E. Bareke, Jacek Majewski
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background Approximately half of all high-grade serous ovarian carcinomas (HGSCs) have a therapeutically targetable defect in the homologous recombination (HR) DNA repair mechanism. While there are genomic and transcriptomic methods, developed for other cancer types, to identify HR deficient (HRD) samples, there are no gene expression-based tools to predict HR repair status in HGSC specifically. We have built the first HGSC-specific model to predict HR repair status using gene expression. Methods We separated The Cancer Genome Atlas (TCGA) cohort of HGSCs (n = 361) into training (n = 288) and testing (n = 73) sets and labelled each case as HRD or HR proficient (HRP) based on the clinical standard for classification, being a score of HRD genomic damage. Using the training set, we performed differential gene expression analysis between HRD and HRP cases. The 2604 significantly differentially expressed genes were then used to tune and train a penalised logistic regression model. Results IdentifiHR is an elastic net penalised logistic regression model that uses the expression of 209 genes to predict HR status in HGSC. These genes capture known regions of HR-specific copy number alteration, which impact gene expression levels, and preserve the genomic damage signal. IdentifiHR has an accuracy of 85% in the TCGA test set and of 91% in an independent cohort of 99 samples, collected from primary tumours before (n = 74/99) and after autopsy (n = 6/99), in addition to ascites (n = 12/99) and normal fallopian tube samples (n = 7/99). Further, IdentifiHR is 84% accurate in pseudobulked single-cell HGSC sequencing from 37 patients and outperforms existing gene expression-based methods to predict HR status, being BRCAness, MutliscaleHRD and expHRD. Conclusions IdentifiHR is an accurate model to predict HR status in HGSC using gene expression alone, that is available as an R package from https://github.com/DavidsonGroup/IdentifiHR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03bb043799881c3a8ed9a4b69ffa433b94653148" target='_blank'>
              IdentifiHR: predicting homologous recombination deficiency in high-grade serous ovarian carcinoma using gene expression
              </a>
            </td>
          <td>
            Ashley L. Weir, Samuel C. Lee, Mengbo Li, Chin Wee Tan, S. Ramus, Nadia M. Davidson
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Robertsonian chromosomes are a type of variant chromosome found commonly in nature. Present in one in 800 humans, these chromosomes can underlie infertility, trisomies, and increased cancer incidence. Recognized cytogenetically for more than a century, their origins have remained mysterious. Recent advances in genomics allowed us to assemble three human Robertsonian chromosomes completely. We identify a common breakpoint and epigenetic changes in centromeres that provide insight into the formation and propagation of common Robertsonian translocations. Further investigation of the assembled genomes of chimpanzee and bonobo highlights the structural features of the human genome that uniquely enable the specific crossover event that creates these chromosomes. Resolving the structure and epigenetic features of human Robertsonian chromosomes at a molecular level paves the way to understanding how chromosomal structural variation occurs more generally, and how chromosomes evolve.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2a20561e10d74149e6076e6c3bf2d3aed806c4" target='_blank'>
              The formation and propagation of human Robertsonian chromosomes
              </a>
            </td>
          <td>
            Leonardo Gomes de Lima, A. Guarracino, S. Koren, T. Potapova, S. McKinney, A. Rhie, Steven J. Solar, Chris Seidel, Brandon Fagen, B. Walenz, G. Bouffard, Shelise Y. Brooks, Michael Peterson, Kate Hall, Juyun Crawford, Alice C Young, Brandon D. Pickett, Erik Garrison, A. Phillippy, J. Gerton
          </td>
          <td>2024-09-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="It has been observed that men have a shorter lifespan and higher incidence of certain cancers than women. We postulate here that this phenomenon may be attributed to the “toxic Y chromosome”, owing to the expression of normally silent HSATII repeats in cancer, abundantly present on chromosome Y. Investigating the contribution of all repetitive elements to chromatin interactions using a genome-wide HiC set derived from a colon cancer cell line (HCT116), we found several anomalies regarding satellite HSATII elements when compared to other genomic repeat types: (i) HSATII repeats tend to form HiC pairs mostly with their own kind, and depleted in most heterologous repeat pairs, whereas other types of repeats readily form heterologous HiC pairs; (ii) when treated with 5aza, a chemotherapeutic agent, the number of long-distance HSATII-overlapping HiC pairs significantly decreased when compared to the untreated samples; (iii) and most importantly, in the treated samples interchromosomal HiC read pairs involving a chromosome Y-located repeat and an HSATII repeat on either end almost completely disappear. (iv) We also found that HSATII repeats lack overlap with other types of repeats. To account for these anomalies we queried non-B DNA structure forming such as G-quadruplexes and transcription-related triplex forming. We found that Y chromosome-coded HSATII repeats form significantly stronger triplexes than HSATII repeats on other chromosomes, their strength significantly correlating with the number of HiC reads they overlap with. We surmise that the above phenomena may cause the higher incidence of cancer in men and may lead to the disappearance of the Y chromosome observed in cancerous cells in men.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53990a3a3a681b314979b859a2d2941c439f782" target='_blank'>
              Y chromosome-coded HSATII repeats may contribute to higher incidence of cancer in men
              </a>
            </td>
          <td>
            Hedi Hegyi, Matej Lexa
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The recent paradoxical dual citizenships of Runt-related transcription factor 3 (Runx3) in tumorigenesis remains poorly characterized. Here, we report the oncogenic capacity of Runx3 as chromatin modulator in metastatic gastric cancer model. Runx3 exists as homodimer and binds cooperatively to modified nucleosomes. Additionally, we detected a synergistic functional enhancement of octamer transfer, nucleosome sliding and stochiometric integrity of SWI/SNF by Runx3. We found that Runx3 depletion increased nucleosomes occupancy and promoted chromatin silencing by heterochromatin condensation and HP1 oligomerization. ATAC-seq analysis revealed differential accessibility per chromosome due to Runx3 null expression with dysregulation of multiple inflammatory response and DNA repair pathways. Mechanistically, these modulations resulted in aberrant DNA damage repair response, which is rescued by RUNX3 overexpression. These findings reveal a new paradigm in Runx3 biology as dynamic chromatin regulatory element vital for the maintenances of cancerous phenotype. Significance To the best of our knowledge, the present study is the first to explore the role of Runx3 as homodimeic chromatin binding factor and establishes its oncogenic-function as modulator of heterochromatin de-condensation and SWI/SNF chromatin remodeling activities. These emerging features of Runx3 at the epigenetic level imply a promising direction to screen for anti-Runx3 epigenetic drugs “Epi-drugs” in search of novel gastric cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65693fd8ec877e2185ca7654a9454ed2dc3bd82f" target='_blank'>
              Runx3 Acts as Homodimeric Chromatin Binding Factor Regulating Heterochromatin-Mediated Cancerous Phenotype
              </a>
            </td>
          <td>
            Salma Awad Merghani, Isabelle Bonne, Aik Yong Sim, Gehan El-Enain
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d63fe0e1a04ed83bd0b4c69b42c0d270765edd" target='_blank'>
              Human cytomegalovirus harnesses host L1 retrotransposon for efficient replication
              </a>
            </td>
          <td>
            Sung-Yeon Hwang, Hyewon Kim, Danielle Denisko, Boxun Zhao, Dohoon Lee, Jiseok Jeong, Jinuk Kim, Kiwon Park, Junhyun Park, D. Jeong, Sehong Park, Hee-Jung Choi, Sun Kim, E. A. Lee, Kwangseog Ahn
          </td>
          <td>2024-09-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Transposable elements (TEs) drive genome evolution and can affect gene expression through diverse mechanisms. In breast cancer, disrupted regulation of TE sequences may facilitate tumor-specific transcriptomic alterations. We examine 142,514 full-length isoforms derived from long-read RNA sequencing (LR-seq) of 30 breast samples to investigate the effects of TEs on the breast cancer transcriptome. Approximately half of these isoforms contain TE sequences, and these contribute to half of the novel annotated splice junctions. We quantify splicing of these LR-seq derived isoforms in 1,135 breast tumors from The Cancer Genome Atlas (TCGA) and 1,329 healthy tissue samples from the Genotype-Tissue Expression (GTEx), and find 300 TE-overlapping tumor-specific splicing events. Some splicing events are enriched in specific breast cancer subtypes – for example, a TE-driven transcription start site upstream of ERBB2 in HER2+ tumors, and several TE-mediated splicing events are associated with patient survival and poor prognosis. The full-length sequences we capture with LR-seq reveal thousands of isoforms with signatures of RNA editing, including a novel isoform belonging to RHOA; a gene previously implicated in tumor progression. We utilize our full-length isoforms to discover polymorphic TE insertions that alter splicing and validate one of these events in breast cancer cell lines. Together, our results demonstrate the widespread effects of dysregulated TEs on breast cancer transcriptomes and highlight the advantages of long-read isoform sequencing for understanding TE biology. TE-derived isoforms may alter the expression of genes important in cancer and can potentially be used as novel, disease-specific therapeutic targets or biomarkers. One Sentence Summary Transposable elements generate alternative isoforms and alter post-transcriptional regulation in human breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa6498a7e5198294cbede8c8c23e085245d62203" target='_blank'>
              Alternative splicing of transposable elements in human breast cancer
              </a>
            </td>
          <td>
            Alex Nesta, Diogo F. T. Veiga, Jacques Banchereau, Olga Anczuków, Christine R. Beck
          </td>
          <td>2024-09-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Significance Chromosome arm 8q is the most frequently gained chromosome arm in aneuploid prostate cancer and is associated with worse prognosis. Using population science and laboratory approaches, we identified increased RAD21 expression in prostate cancer as a key adverse prognostic factor related to 8q gains that enables tumors to sustain proliferation despite oncogenic stress and DNA damage from oncogenic mutations. These data pinpoint one mechanism through which aneuploidy promotes tumor progression and suggests a potential therapeutic target for aneuploid prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86f061db09f8cb87400e631c0dc03d0078eb8a3c" target='_blank'>
              RAD21 promotes oncogenesis and lethal progression of prostate cancer
              </a>
            </td>
          <td>
            Xiaofeng A Su, Konrad H. Stopsack, Daniel R. Schmidt, Duanduan Ma, Zhe Li, P. Scheet, K. Penney, T. Lotan, W. Abida, Elise G DeArment, Kate Lu, Thomas Janas, Sofia Hu, M. V. Vander Heiden, Massimo Loda, Monica G. Boselli, Angelika Amon, Lorelei Mucci
          </td>
          <td>2024-08-27</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="MYC deregulation occurs in the majority of multiple myeloma (MM) cases and is associated with progression and worse prognosis. Enhanced MYC expression occurs in about 70% of MM patients, but it is known to be driven by translocation or amplification events in only ~40% of myelomas. Here, we used CRISPR interference (CRISPRi) to uncover an epigenetic mechanism of MYC regulation whereby increased accessibility of a plasma cell-type specific enhancer leads to increased MYC expression. This native enhancer activity was not associated with enhancer hijacking events but led to specific binding of c-MAF, IRF4, and SPIB transcription factors that activated MYC expression in the absence of known genetic aberrations. In addition, focal amplification was another mechanism of activation of this enhancer in approximately 3.4% of MM patients. Together, these findings define an epigenetic mechanism of MYC deregulation in MM beyond known translocations or amplifications and point to the importance of non-coding regulatory elements and their associated transcription factor networks as drivers of MM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89586e6b56717e928b88a8526a759effc353dc88" target='_blank'>
              Selective Enhancer Gain of Function Deregulates MYC Expression in Multiple Myeloma.
              </a>
            </td>
          <td>
            M. Rahmat, Kendell Clement, Jean-Baptiste Alberge, Romanos Sklavenitis-Pistofidis, R. Kodgule, C. Fulco, Daniel Heilpern-Mallory, Katarina Nilsson, David Dorfman, Jesse M Engreitz, G. Getz, Luca Pinello, R. Ryan, I. M. Ghobrial
          </td>
          <td>2024-09-23</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Identification of somatic driver mutations in the noncoding genome remains challenging. To comprehensively characterize noncoding driver mutations for pancreatic ductal adenocarcinoma (PDAC), we first created genome-scale maps of accessible chromatin regions (ACRs) and histone modification marks (HMMs) in pancreatic cell lines and purified pancreatic acinar and duct cells. Integration with whole-genome mutation calls from 506 PDACs revealed 314 ACRs/HMMs significantly enriched with 3,614 noncoding somatic mutations (NCSMs). Functional assessment using massively parallel reporter assays (MPRA) identified 178 NCSMs impacting reporter activity (19.45% of those tested). Focused luciferase validation confirmed negative effects on gene regulatory activity for NCSMs near CDKN2A and ZFP36L2. For the latter, CRISPR interference (CRISPRi) further identified ZFP36L2 as a target gene (16.0 - 24.0% reduced expression, P = 0.023 - 0.0047) with disrupted KLF9 binding likely mediating the effect. Our integrative approach provides a catalog of potentially functional noncoding driver mutations and nominates ZFP36L2 as a PDAC driver gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db992992dda460b3aa1dd358fc40078a768ec148" target='_blank'>
              Large-scale multi-omic analysis identifies noncoding somatic driver mutations and nominates ZFP36L2 as a driver gene for pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            J. Zhong, Aidan O'Brien, Minal Patel, Daina Eiser, Michael Mobaraki, I. Collins, Li Wang, Konnie Guo, Thucnhi Truongvo, Ashley A. Jermusyk, Maura J. O’Neill, Courtney D. Dill, A. Wells, M. Leonard, J. Pippin, Struan F. A. Grant, Tongwu Zhang, T. Andresson, Katelyn E. Connelly, Jianxin Shi, H. E. Arda, Jason W. Hoskins, L. Amundadottir
          </td>
          <td>2024-09-24</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits widespread epigenetic alterations, yet their impact on cis-regulatory elements (CREs) and retrotransposons remains poorly understood. Here, we present an integrated epigenomic and transcriptomic analysis of HCC tumors and matched tumor-adjacent normal tissues. We identified extensive DNA hypomethylation coupled with changes in histone modifications at partially methylated domains, CREs, and retrotransposons. These epigenetic aberrations were associated with dysregulated expression of genes involved in cell cycle regulation, immune response, and extracellular matrix organization. Notably, our findings revealed a novel mechanism for the transcriptional dysregulation of GPC3, a key HCC biomarker and immunotherapeutic target. We observed that GPC3 upregulation is driven by both the reactivation of a fetal liver super enhancer and hypomethylation of GPC3-associated CpG islands. Moreover, we found that DNA hypomethylation-driven aberrant expression of retrotransposons carries prognostic significance in HCC. Patients with high expression of a long non-coding RNA driven by a HERVE-int element exhibited more aggressive tumors, poorer clinical outcomes, and molecular features associated with favorable immunotherapy response. Together, our study provides a comprehensive resource for understanding the role of epigenetic dysregulation in HCC and identifies retrotransposon-associated transcripts as potential biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce7deec20d100e57e2cf628c181e276b8e32f748" target='_blank'>
              Epigenomic analysis of hepatocellular carcinoma reveals aberrant cis-regulatory changes and dysregulated retrotransposons with prognostic potentials
              </a>
            </td>
          <td>
            Clooney C. Y. Cheng, Ming Fung Cheung, Ah Young Lee, Qiong Wu, S. Chow, Julie Y. J. Ang, Ignacio Riquelme Medina, Grace Lo, Haoran Wu, Weiqin Yang, Paul B. S. Lai, K. Yip, Alfred S. Cheng, Danny Leung
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="How specific enhancer-promoter pairing is established is still mostly unclear. Besides the CTCF/cohesin machinery, only a few nuclear factors have been studied for a direct role in physically connecting regulatory elements. Here, we show via acute degradation experiments that LDB1 directly and broadly promotes enhancer-promoter loops. Most LDB1-mediated contacts, even those spanning hundreds of kb, can form in the absence of CTCF, cohesin, or YY1 as determined via the use of multiple degron systems. Moreover, an engineered LDB1-driven chromatin loop is cohesin independent. Cohesin-driven loop extrusion does not stall at LDB1 occupied sites but may aid the formation of a subset of LDB1 anchored loops. Leveraging the dynamic reorganization of nuclear architecture during the transition from mitosis to G1-phase, we establish a relationship between LDB1-dependent interactions in the context of TAD organization and gene activation. Lastly, Tri-C and Region Capture Micro-C reveal that LDB1 organizes multi-enhancer networks to activate transcription. This establishes LDB1 as a direct driver of regulatory network inter-connectivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a1fba58a9a80d4788114c71aebd8c713f90c8ba" target='_blank'>
              LDB1 establishes multi-enhancer networks to regulate gene expression
              </a>
            </td>
          <td>
            Nicholas G. Aboreden, Jessica C. Lam, Viraat Y. Goel, Siqing Wang, Xiaokang Wang, Susannah C. Midla, Alma Quijano, C. Keller, B. Giardine, R. Hardison, Haoyue Zhang, A. Hansen, Gerd A. Blobel
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>41</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c120e2af47610364790b472bc22133f0b2b21b" target='_blank'>
              Complex genomic rearrangements of the Y chromosome in a premature infant
              </a>
            </td>
          <td>
            Stephanie A. Balow, Alyxis Coyan, Nicki Smith, Bianca E Russell, Danielle Monteil, Robert J. Hopkin, Teresa A. Smolarek
          </td>
          <td>2024-08-26</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Break-induced replication (BIR) is mutagenic, and thus its use requires tight regulation, yet the underlying mechanisms remain elusive. Here we uncover an important role of 53BP1 in suppressing BIR after end resection at double strand breaks (DSBs), distinct from its end protection activity, providing insight into the mechanisms governing BIR regulation and DSB repair pathway selection. We demonstrate that loss of 53BP1 induces BIR-like hyperrecombination, in a manner dependent on Polα-primase-mediated end fill-in DNA synthesis on single-stranded DNA (ssDNA) overhangs at DSBs, leading to PCNA ubiquitination and PIF1 recruitment to activate BIR. On broken replication forks, where BIR is required for repairing single-ended DSBs (seDSBs), SMARCAD1 displaces 53BP1 to facilitate the localization of ubiquitinated PCNA and PIF1 to DSBs for BIR activation. Hyper BIR associated with 53BP1 deficiency manifests template switching and large deletions, underscoring another aspect of 53BP1 in suppressing genome instability. The synthetic lethal interaction between the 53BP1 and BIR pathways provides opportunities for targeted cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21b30096c06a55bc35f817e3d1b46b909b538103" target='_blank'>
              53BP1 deficiency leads to hyperrecombination using break-induced replication (BIR)
              </a>
            </td>
          <td>
            Sameer Bikram Shah, Youhang Li, Shibo Li, Qing Hu, Tong Wu, Yanmeng Shi, Tran Nguyen, Isaac Ive, Linda Shi, Hailong Wang, Xiaohua Wu
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Genome organization plays a significant role in silencing heterochromatinized genes in the most virulent human malaria parasite, Plasmodium falciparum. However, it remains unclear how heterochromatinized genes spatially cluster or if active genes are also organized in a specific manner. We used Micro-C to achieve a near-nucleosome resolution DNA-DNA contact map, which revealed new inter- and intrachromosomal heterochromatic and euchromatic structures in the blood stage parasite. We observed subtelomeric fold structures that facilitate interactions amongst heterochromatinized genes involved in antigenic variation. In addition, we identified long-range intra- and interchromosomal interactions amongst active, stage-specific genes. Both structures are mediated by AP2-P, an ApiAP2 transcription factor, and a putative MORC chromatin remodeler, and functional specificity is achieved via combinatorial binding with other sequence-specific DNA-binding factors. This study provides unprecedented insight into the organizational machinery used by this medically important eukaryotic parasite to spatially coordinate genes underlying antigenic variation and to co-activate stage-specific genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daca09a9e45914726bf35f825cfbcaaf5259d6fc" target='_blank'>
              Micro-C reveals MORC/ApiAP2-mediated links between distant, functionally related genes in the human malaria parasite
              </a>
            </td>
          <td>
            Parul Singh, Jacques Serizay, J. Couble, Maureen D. Cabahug, Catarina Rosa, Patty Chen, Artur Scherf, Romain Koszul, Sebastian Baumgarten, Jessica M. Bryant
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9a6a91b5f3ac052c57405c773236aad0d8d4b4" target='_blank'>
              Recent insights into the causes and consequences of chromosome mis-segregation.
              </a>
            </td>
          <td>
            Romain Devillers, Alexsandro dos Santos, Quentin Destombes, Mathieu Laplante, S. Elowe
          </td>
          <td>2024-09-15</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Errors in mitosis can contribute to aneuploidy and CIN and play a pivotal role in cancer. So the identification of altered mitotic regulators can contribute to the understanding of the development and progression of breast cancer. In the present study we used an in vitro model of disease progression (the MCF10A series of BC continuum) and analyzed the errors of chromosome segregation that occur during the progression of the disease. Our findings indicated that the MCF10A series exhibited several abnormalities in chromosome segregation and its frequency increased with the disease progression. These errors included anaphase lagging chromosomes, micronuclei, nuclear buds, nucleoplasmic bridges, errors of chromosome alignment, and centrosome loss/amplification. Moreover, the presence of centrosome amplification disrupted the proper orientation of the mitotic spindle, resulting in the generation asymmetrical cell lines and aneuploidy in the MCF10A series. Hyper stable kinetochore-microtubule (kt-MT) attachment was also found in premalignant, preinvasive, and invasive cell lines, which can also explain the presence of errors of chromosome alignment. The human transcriptome array also determined possible negative regulators of ciliogenesis that can explain the mechanism of chromosome missegregation that lead to CIN found in the MCF10A series. Collectively, these findings highlight the importance of mitotic defects in the progression of breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8acf9391e3d0717b055bd26e1631bad45d557802" target='_blank'>
              A characterization of mitotic and centrosomal defects in a continuum model of Breast Cancer
              </a>
            </td>
          <td>
            Alexsandro dos Santos, C. Diorio, Francine Durocher, S. Elowe
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Polypyrimidine tract binding protein 2 (PTBP2) is an RNA-binding protein that controls alternative splicing in neuronal, muscle, and Sertoli cells. Our study unveils a novel role of PTBP2 in promoting the excessive production of the DNA polymerase kappa (Polκ) by stabilizing its 3’UTR. We observed an association between its increased expression and the upregulation of PTBP2 in clinical samples of Chronic Myeloid Leukemia (CML). Ptbp2 knock-out CML cell lines and patient samples treated with hydroxyurea presented with increased DNA damage, as evidenced by long comet tails and higher levels of the DNA damage marker, γH2AX foci, however overexpression of Polκ in the Ptbp2-KO cells restored normal phenotype. The deregulation of the DNA repair pathway is a defining feature of malignancies and is closely associated with genomic instability. POLK was found to interact with MRE11 of the MRN complex, thereby governing the activation of ATM-CHK2. Cells with elevated levels of Ptbp2 and Polκ demonstrated increased sister chromatid exchange and BrdU incorporation in ex-vivo assays, while multinucleated cells with multipolar spindles were observed in in-vivo assays. Our findings confirm the critical role of the PTBP2-POLK axis in driving genomic instability and bolstering the viability of cells with increased malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aa3b48c6a74455a5e894d9a4adb977c32b5b29c" target='_blank'>
              DNA polymerase kappa stabilized by PTBP2 interacts with MRE11 and promotes genomic instability in leukemia cells
              </a>
            </td>
          <td>
            Shristi Lama, Bibhudev Barik, Sajitha Is, Tannistha Sarkar, Sayantan Chanda, Monalisa Behera, Subhankar Priyadarshi Behera, S. Biswas, Sonali Mohapatra, G. Biswas, Soumen Chakraborty
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="B-Myb, also known as MYB proto-oncogene like 2 (MYBL2), is an important transcription factor implicated in transcription regulation, cell cycle and tumorigenesis. However, the molecular mechanism underlying B-Myb-controlled transactivation in different cell contexts as well as its functional implication in cancers remains elusive. In this study, we have conducted a comprehensive genome-wide analysis of B-Myb binding sites in multiple immortalized or cancer cell lines and identified its critical target genes. The results revealed that B-Myb regulates a common set of core cell cycle genes and cell type-specific genes through collaboration with other important transcription factors (e.g. NFY and MuvB complex) and binding to cell type-invariant promoters and cell type-specific enhancers and super-enhancers. KIF2C, UBE2C and MYC were further validated as B-Myb target genes. Loss-of-function analysis demonstrated that KIF2C knockdown inhibited tumor cell growth both in vitro and in vivo, suppressed cell motility and cell cycle progression, accompanied with defects in microtubule organization and mitosis, strongly suggesting that KIF2C is a critical regulator of cancer cell growth and mitosis, and maintains high cancer cell motility ability and microtubule dynamics. Pan-cancer transcriptomic analysis revealed that the overexpression of both B-Myb and KIF2C presents as independent prognostic markers in various types of cancer. Notably, B-Myb associates with NFYB, binds to target gene promoters, enhancers and super-enhancers, and provokes a cascade of oncogenic gene expression profiles in cancers. Overall, our results highly suggest the critical implication of B-Myb-mediated gene regulation in cancers, and the promising therapeutic and prognostic potentials of B-Myb and KIF2C for cancer diagnosis and treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0f31a53d23c8c9c328816abe67e6d92d102de4" target='_blank'>
              Genome-wide binding analysis unveils critical implication of B-Myb-mediated transactivation in cancers
              </a>
            </td>
          <td>
            Chuntao Tao, Tao Liu, Zongrong Zhao, Xuanqi Dou, X. Xia, Kailong Du, Xiaofeng Zuo, Yitao Wang, Tingting Wang, Youquan Bu
          </td>
          <td>2024-09-03</td>
          <td>International Journal of Biological Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="During cancer development, the interplay between the nucleus and the cell cycle leads to a state of genomic instability, often accompanied by observable morphological aberrations. These aberrations can be controlled by tumor cells to evade cell death, either by preventing or eliminating genomic instability. In epithelial ovarian cancer (EOC), overexpression of the multifunctional protein claudin-4 is a key contributor to therapy resistance through mechanisms associated with genomic instability. However, the molecular mechanisms underlying claudin-4 overexpression in EOC remain poorly understood. Here, we altered claudin-4 expression and employed a unique claudin-4 targeting peptide (CMP) to manipulate the function of claudin-4. We found that claudin-4 facilitates genome maintenance by linking the nuclear envelope and cytoskeleton dynamics with cell cycle progression. Claudin-4 caused nuclei constriction by excluding lamin B1 and promoting perinuclear F-actin accumulation, associated with remodeling nuclear architecture, thus altering nuclear envelope dynamics. Consequently, cell cycle modifications due to claudin-4 overexpression resulted in fewer cells entering the S-phase and reduced genomic instability. Importantly, disrupting biological interactions of claudin-4 using CMP and forskolin altered oxidative stress cellular response and increased the efficacy of PARP inhibitor treatment. Our data indicate that claudin-4 protects tumor genome integrity by remodeling the crosstalk between the nuclei and the cell cycle, leading to resistance to genomic instability formation and the effects of genomic instability-inducing agents. Graphical abstract Claudin-4 plays a crucial role in remodeling the cytoskeleton, particularly influencing nuclear architecture. This remodeling appears to act as a regulatory mechanism, limiting the progression of ovarian cancer cells into the S-phase and correlating with reduced genomic instability in ovarian tumors. Moreover, since olaparib treatment triggered a cellular oxidative response, it is likely that this claudin-4-mediated remodeling contributes to resistance against the effects of the PARP inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1779163fc52ccc64570aa04b3ca709410f5487db" target='_blank'>
              Claudin-4 remodeling of nucleus-cell cycle crosstalk maintains ovarian tumor genome stability and drives resistance to genomic instability-inducing agents
              </a>
            </td>
          <td>
            Fabian R. Villagomez, Julie Lang, Daniel Nunez-Avellaneda, K. Behbakht, Hannah L. Dimmick, Patricia Webb, K. Nephew, Margaret Neville, Elizabeth R. Woodruff, Benjamin G. Bitler
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="The tumor suppressor gene TP53 encodes the DNA binding transcription factor p53 and is one of the most commonly mutated genes in human cancer. Tumor suppressor activity requires binding of p53 to its DNA response elements and subsequent transcriptional activation of a diverse set of target genes. Despite decades of close study, the logic underlying p53 interactions with its numerous potential genomic binding sites and target genes is not yet fully understood. Here, we present a database of DNA and chromatin-based information focused on putative p53 binding sites in the human genome to allow users to generate and test new hypotheses related to p53 activity in the genome. Users can query genomic locations based on experimentally observed p53 binding, regulatory element activity, genetic variation, evolutionary conservation, chromatin modification state, and chromatin structure. We present multiple use cases demonstrating the utility of this database for generating novel biological hypotheses, such as chromatin-based determinants of p53 binding and potential cell type-specific p53 activity. All database information is also available as a precompiled sqlite database for use in local analysis or as a Shiny web application.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f24a956f031769841584e7d92965e89a42014c" target='_blank'>
              p53motifDB: integration of genomic information and tumor suppressor p53 binding motifs
              </a>
            </td>
          <td>
            Gabriele Baniulyte, S. M. Hicks, Morgan A. Sammons
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Cancer cells with an activated Alternative Lengthening of Telomeres (ALT) mechanism elongate telomeres via homology-directed repair. Sustained telomeric replication stress is an essential trigger of ALT activity; however, it can lead to cell death if not properly restricted. By analyzing publicly available data from genome-wide CRISPR KO screenings, we have identified the multifunctional protein PC4 as a novel factor essential for ALT cell viability. Depletion of PC4 using siRNAs results in rapid ALT cell death, while telomerase-positive cells show minimal effects. PC4 depletion induces replication stress and telomere fragility primarily in ALT cells, and increases ALT activity. PC4 binds to telomeric DNA in cells, and its binding is enhanced by telomeric replication stress. Finally, a mutant PC4 with partly impaired single stranded DNA binding activity is capable to localize to telomeres and suppress ALT activity and telomeric replication stress. We propose that PC4 supports ALT cell viability, at least partly, by averting telomere dysfunction. Targeted inhibition of PC4 holds promise for innovative therapies to eradicate ALT cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfdc3406a77cef2067d450e6f8d19a7b405859a4" target='_blank'>
              Human PC4 supports telomere stability and viability in cells utilizing the alternative lengthening of telomeres mechanism
              </a>
            </td>
          <td>
            Sara Salgado, Patricia L. Abreu, Beatriz Moleirinho, Lee Larcombe, C. Azzalin
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Transcriptional enhancers are non-coding DNA elements that regulate gene transcription in a temporal and tissue-specific manner. Despite advances in computational and experimental methods, identifying enhancers and their target genes essential for specific biological processes remains challenging. Determining target genes for enhancers is also complex and often relies on indirect, low-resolution, and/or assumptive methodologies. To identify and functionally perturb enhancers at their endogenous sites without altering their sequence, we performed a pooled tiling CRISPR activation (CRISPRa) screen surrounding PHOX2B, a master regulator of neuronal cell fate and a key player in neuroblastoma development. This screen allowed the de novo identification of CRISPRa responsive elements (CaREs) that alter cellular growth within the 2 Mb genomic region. To determine CaRE target genes, we developed TESLA-seq (TargEted SingLe cell Activation), which combines CRISPRa screening with targeted single-cell RNA-sequencing and enables the parallel readout of the effect of hundreds of enhancers on all genes in the locus. While most TESLA-revealed CaRE-gene relationships involved neuroblastoma-related regulatory elements already active in the system, we found many CaREs and target connections normally active only in other tissue types or with no previous evidence and induced out of context by CRISPRa. This highlights the power of TESLA-seq to reveal gene regulatory networks, including edges active outside of a given experimental system. Highlights Systematically perturbed regulatory landscape in a 2 Mb genomic region surrounding PHOX2B to identify hundreds of CRISPRa-responsive elements that affect cellular growth Developed TESLA-seq as a principled molecular approach to find gene targets of dozens of candidate regulatory elements Validated interactions between identified regulatory elements and target genes and characterized their genomic features Integrated a compendium of epigenomic datasets to identify regulatory relationships induced out of context or with no previous evidence">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6f4af902573832fb2aa9994e5101eab3eac7003" target='_blank'>
              Sensitive dissection of a genomic regulatory landscape using bulk and targeted single-cell activation
              </a>
            </td>
          <td>
            Dubravka Vučićević, Che-Wei Hsu, Lorena Sofia Lopez Zepeda, M. Burkert, Antje Hirsekorn, Ilija Bilić, Nicolai Kastelić, M. Landthaler, S. Lacadie, U. Ohler
          </td>
          <td>2024-09-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="In cancer cells, ATR signaling is crucial to tolerate the intrinsically high damage levels that normally block replication fork progression. Assembly of TopBP1, a multifunctional scaffolding protein, into condensates is required to amplify ATR kinase activity to the levels needed to coordinate the DNA damage response and manage DNA replication stress. Many ATR inhibitors are tested for cancer treatment in clinical trials, but their overall effectiveness is often compromised by the emergence of resistance and toxicities. In this proof-of-concept study, we propose to disrupt the ATR pathway by targeting TopBP1 condensation. First, we screened a molecule-based library using a previously developed optogenetic approach and identified several TopBP1 condensation inhibitors. Amongst them, AZD2858 disrupted TopBP1 assembly induced by the clinically relevant topoisomerase I inhibitor SN-38, thereby inhibiting the ATR/Chk1 signaling pathway. We found that AZD2858 exerted its effects by disrupting TopBP1 self-interaction and binding to ATR in mammalian cells, and by increasing its chromatin recruitment in cell-free Xenopus laevis egg extracts. Moreover, AZD2858 prevented S-phase checkpoint induction by SN-38, leading to increased DNA damage and apoptosis in a colorectal cancer cell line. Lastly, AZD2858 showed synergistic effect in combination with the FOLFIRI chemotherapy regimen in a spheroid model of colorectal cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8d12d939b21bc8eddc64b81e916d3c7019a271" target='_blank'>
              TopBP1 biomolecular condensates: a new therapeutic target in advanced-stage colorectal cancer
              </a>
            </td>
          <td>
            Laura Morano, Nadia Vezzio-Vié, Adam Aissanou, Tom Egger, Antoine Aze, Solène Fiachetti, Hervé Seitz, Louis-Antoine Milazzo, V. Garambois, Nathalie Bonnefoy, Céline Gongora, A. Constantinou, J. Basbous
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Histones are essential for maintaining chromatin structure and function. Histone mutations lead to changes in chromatin compaction, gene expression, and the recruitment of DNA repair proteins to the DNA lesion. These disruptions can impair critical DNA repair pathways, such as homologous recombination and non-homologous end joining, resulting in increased genomic instability, which promotes an environment favorable to tumor development and progression. Understanding these mechanisms underscores the potential of targeting DNA repair pathways in cancers harboring mutated histones, offering novel therapeutic strategies to exploit their inherent genomic instability for better treatment outcomes. Here, we examine how mutations in histone H3 disrupt normal chromatin function and DNA damage repair processes and how these mechanisms can be exploited for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae6d859b6a16ea85b27994b62cbff421ebf7a45" target='_blank'>
              Histone H3 mutations and their impact on genome stability maintenance.
              </a>
            </td>
          <td>
            Lucas D Caeiro, Ramiro E Verdun, Lluis Morey
          </td>
          <td>2024-09-09</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA replication often encounters obstacles like the stalled transcription machinery and R-loops. While ribonucleases and DNA-RNA helicases can resolve these structures, the role of chromatin remodelers remains understudied. Through a series of in vitro and in vivo experiments, we show that the chromatin remodeler SMARCAD1, which associates with active replication forks, is crucial for resolving nearby R-loops to maintain fork stability. SMARCAD1 directly binds R-loops via its ATPase domain and associates with the replisome through its N-terminus region. Both interactions are critical for resolving R-loops within cells. Genome-wide assays reveal that cells expressing mutant SMARCAD1 accumulate significantly more R-loops than wild-type cells, particularly in regions distinct from known fork blockage-prone sites. These R-loop-enriched regions in SMARCAD1 mutants also exhibit increased mutagenesis in germline tumors, suggesting they are mutation hotspots in cancer. Therefore, SMARCAD1 acts as an R-loop sensor and resolvase at actively progressing forks, maintaining genome stability and preventing tumorigenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b2647786c65a810f532e08e30214a54e76697fe" target='_blank'>
              SMARCAD1 Regulates R-Loops at Active Replication Forks Linked to Cancer Mutation Hotspots
              </a>
            </td>
          <td>
            Sidrit Uruci, N. M. Hoitsma, María E. Soler-Oliva, Aleix Bayona-Feliú, Vincent Gaggioli, María L. García-Rubio, Calvin Lo, Collin Bakker, Jessica Marinello, E. Manolika, Giovanni Capranico, M. S. Luijsterburg, Karolin Luger, A. Aguilera, Nitika Taneja
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Although the role of G-quadruplex (G4) DNA structures has been suggested in chromosomal looping this was not tested directly. Here, to test causal function, an array of G4s, or control sequence that does not form G4s, were inserted within chromatin in cells. In vivo G4 formation of the inserted G4 sequence array, and not the control sequence, was confirmed using G4-selective antibody. Compared to the control insert, we observed a remarkable increase in the number of 3D chromatin looping interactions from the inserted G4 array. This was evident within the immediate topologically associated domain (TAD) and throughout the genome. Locally, recruitment of enhancer histone marks and the transcriptional coactivator p300/Acetylated-p300 increased in the G4-array, but not in the control insertion. Resulting promoter-enhancer interactions and gene activation were clear up to 5 Mb away from the insertion site. Together, these show the causal role of G4s in enhancer function and long-range chromatin interactions. Mechanisms of 3D topology are primarily based on DNA-bound architectural proteins that induce/stabilize long-range interactions. Involvement of the underlying intrinsic DNA sequence/structure in 3D looping shown here therefore throws new light on how long-range chromosomal interactions might be induced or maintained.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c4a14fedf36592373d312c9d23b12df8fd19b77" target='_blank'>
              Artificially inserted strong promoter containing multiple G-quadruplexes induces long-range chromatin modification
              </a>
            </td>
          <td>
            S. Roy, Sulochana Bagri, Soujanya Vinayagamurthy, Avik Sengupta, Claudia Regina Then, Rahul Kumar, Sriram Sridharan, Shantanu Chowdhury
          </td>
          <td>2024-08-19</td>
          <td>eLife</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Multimeric SWI/SNF chromatin remodelers assemble into discrete conformations with unique complex functionalities difficult to dissect. Distinct cancers harbor mutations in specific subunits, altering the chromatin landscape, such as the PBAF-specific component ARID2 in melanoma. Here, we performed comprehensive epigenomic profiling of SWI/SNF complexes and their associated chromatin states in melanoma and melanocytes and uncovered a subset of PBAF-exclusive regions that coexist with PRC2 and repressive chromatin. Time-resolved approaches revealed that PBAF regions are generally less sensitive to ATPase-mediated remodeling than BAF sites. Moreover, PBAF/PRC2-bound loci are enriched for REST, a transcription factor that represses neuronal genes. In turn, absence of ARID2 and consequent PBAF complex disruption hinders the ability of REST to bind and inactivate its targets, leading to upregulation of synaptic transcripts. Remarkably, this gene signature is conserved in melanoma patients with ARID2 mutations. In sum, we demonstrate a unique role for PBAF in generating accessibility for a silencing transcription factor at repressed chromatin, with important implications for disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523ec2477b747fcc0089e58355e10a3265c7a242" target='_blank'>
              The SWI/SNF PBAF complex facilitates REST occupancy at repressive chromatin
              </a>
            </td>
          <td>
            Elena Grossi, Christie B. Nguyen, Saul Carcamo, Shannon Moran, Valentina Kirigin Callaú, Dan Filipescu, Dan Hasson, Emily Bernstein
          </td>
          <td>2024-08-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Transposable elements (TEs), widespread genetic parasites, pose potential threats to the stability of their host genomes. Hence, the interactions observed today between TEs and their host genomes, as well as among the different TE species coexisting in the same host, likely reflect those that did not lead to the extinction of either the host or the TEs. It is not clear to what extent the expression and integration steps of the TE replication cycles are involved in this ‘peaceful’ coexistence. Here, we show that four Drosophila LTR RetroTransposable Elements (LTR-RTEs), although sharing the same overall integration mechanism, preferentially integrate into distinct open chromatin domains of the host germline. Notably, the differential expressions of the gtwin and ZAM LTR-RTEs in ovarian and embryonic somatic tissues, respectively, result in differential integration timings and targeting of accessible chromatin landscapes that differ between early and late embryonic nuclei, highlighting connections between temporal and spatial LTR-RTEs niche partitionings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10ef122580a3d05b44b334371537616b10346674" target='_blank'>
              Temporal and spatial niche partitioning in a retrotransposon community of the Drosophila genome
              </a>
            </td>
          <td>
            Marion Varoqui, Mourdas Mohamed, B. Mugat, Daniel Gourion, Maëlys Lemoine, Alain Pélisson, Charlotte Grimaud, S. Chambeyron
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="DNAJB6, a major member of the DNAJ/HSP40 family, plays an important role in tumor development. We explored the effect of DNAJB6 expression on the prognosis of patients and its biological role in lung adenocarcinoma (LUAD). mRNA and clinical data were obtained from The Cancer Genome Atlas (TCGA). Enriched pathways were determined by the Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses. A nomogram incorporating DNAJB6 and three clinical features was constructed to predict the survival rate. DNAJB6 expression and function in LUAD were explored using immunohistochemistry, Western blotting, proliferation, cell cycle analysis, RNA sequencing, and xenograft tumor assays. DNAJB6 mRNA levels were elevated in the LUAD-TCGA dataset. DNAJB6 protein levels were higher in LUAD tumor tissues than in normal tissues. A high DNAJB6 level was an independent risk factor for poor prognosis in patients with LUAD. The proportion of tumor-infiltrating immune cells significantly differed between high and low DNAJB6 expression. DNAJB6 was associated with cell cycle pathways; therefore, its knockdown induced G2/M cell cycle arrest and inhibited LUAD cell proliferation. This is the first report of the DNAJB6 requirement for LUAD cell proliferation and its potentially crucial role in LUAD prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd99e6105741818b2b8a1ce73bd38d2fcdc17926" target='_blank'>
              Abnormally High Expression of DNAJB6 Accelerates Malignant Progression of Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Di Wang, Jiayu Xiao, Yang Du, Li Zhang, Xuzhen Qin
          </td>
          <td>2024-09-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Epstein-Barr virus (EBV) uses latency programs to colonize the memory B-cell reservoir, and each program is associated with human malignancies. However, knowledge remains incomplete of epigenetic mechanisms that maintain the highly restricted latency I program, present in memory and Burkitt lymphoma cells, in which EBNA1 is the only EBV-encoded protein expressed. Given increasing appreciation that higher order chromatin architecture is an important determinant of viral and host gene expression, we investigated roles of Wings Apart-Like Protein Homolog (WAPL), a host factor that unloads cohesin to control DNA loop size and that was discovered as an EBNA2-associated protein. WAPL knockout (KO) in Burkitt cells de-repressed LMP1 and LMP2A expression, but not other EBV oncogenes, to yield a viral program reminiscent of EBV latency II, which is rarely observed in B-cells. WAPL KO also increased LMP1/2A levels in latency III lymphoblastoid cells. WAPL KO altered EBV genome architecture, triggering formation of DNA loops between the LMP promoter region and the EBV origins of lytic replication (oriLyt). Hi-C analysis further demonstrated that WAPL KO reprogrammed EBV genomic DNA looping. LMP1 and LMP2A de-repression correlated with decreased histone repressive marks at their promoters. We propose that EBV coopts WAPL to negatively regulate latent membrane protein expression to maintain Burkitt latency I.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ccb753cd4a98829975ef9e5d570c14e44b7ff8" target='_blank'>
              The DNA loop release factor WAPL suppresses Epstein-Barr virus latent membrane protein expression to maintain the highly restricted latency I program
              </a>
            </td>
          <td>
            Laura A. Murray-Nerger, Davide Maestri, Xiang Liu, Zhixuan Li, Nina R Auld, I. Tempera, Mingxiang Teng, B. Gewurz
          </td>
          <td>2024-09-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Aneuploidy typically poses challenges for cell survival and growth. However, recent studies have identified exceptions where aneuploidy is beneficial for cells with mutations in certain regulatory genes. Our research reveals that cells lacking the spindle checkpoint gene BUB3 exhibit aneuploidy of select chromosomes. While the spindle checkpoint is not essential in budding yeast, the loss of BUB3 and BUB1 increases the probability of chromosome missegregation compared to wildtype cells. Contrary to the prevailing assumption that the aneuploid cells would be outcompeted due to growth defects, our findings demonstrate that bub3Δ cells consistently maintained aneuploidy of specific chromosomes over many generations. We investigated whether the persistence of these additional chromosomes in bub3Δ cells resulted from the beneficial elevated expression of certain genes, or mere tolerance. We identified several genes involved in chromosome segregation and cell cycle regulation that confer an advantage to Bub3-depleted cells. Overall, our results suggest that the upregulation of specific genes through aneuploidy may provide a survival and growth advantage to strains with poor chromosome segregation fidelity. AUTHOR SUMMARY Accurate chromosome segregation is crucial for the proper development of all living organisms. Errors in chromosome segregation can lead to aneuploidy, characterized by an abnormal number of chromosomes, which generally impairs cell survival and growth. However, under certain stress conditions, such as in various cancers, cells with specific mutations and extra copies of advantageous chromosomes exhibit improved survival and proliferation. In our study, we discovered that cells lacking the spindle checkpoint protein Bub3 became aneuploid, retaining specific chromosomes. This finding was unexpected because although bub3Δ cells have a higher rate of chromosome mis-segregation, they were not thought to maintain an aneuploid karyotype. We investigated whether the increased copy number of specific genes on these acquired chromosomes offered a benefit to Bub3-deficient cells. Our results revealed that several genes involved in chromosome segregation and cell cycle regulation prevented the gain of chromosomes upon Bub3-depletion, suggesting that these genes confer a survival advantage. Overall, our study demonstrates that cells lacking Bub3 selectively retain specific chromosomes to increase the copy number of genes that promote proper chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fe65305b6ce867bfbed31e843276ed59a3f7ed3" target='_blank'>
              Aneuploidy of Specific Chromosomes is Beneficial to Cells Lacking Spindle Checkpoint Protein Bub3
              </a>
            </td>
          <td>
            Pallavi Gadgil, Olivia Ballew, Timothy J. Sullivan, Soni Lacefield
          </td>
          <td>2024-09-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505f6d9a24bb0cc1b413b063823fa87cd784ad4b" target='_blank'>
              Extracting regulatory active chromatin footprint from cell-free DNA
              </a>
            </td>
          <td>
            Kevin Lai, Katharine Dilger, Rachael Cunningham, Kathy T Lam, Rhea Boquiren, Khiet Truong, Maggie C. Louie, Richard Rava, Diana Abdueva
          </td>
          <td>2024-09-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f6b0780202531821c7a75fa420779272072de3" target='_blank'>
              Retrotransposon addiction promotes centromere function via epigenetically activated small RNAs
              </a>
            </td>
          <td>
            Atsushi Shimada, Jonathan Cahn, E. Ernst, Jason Lynn, D. Grimanelli, Ian R. Henderson, T. Kakutani, R. Martienssen
          </td>
          <td>2024-09-02</td>
          <td>Nature Plants</td>
          <td>1</td>
          <td>93</td>
        </tr>

        <tr id="Genes on sex chromosomes have higher evolutionary rates than those on autosomes. However, this does not necessarily apply to somatic evolution in cancer. Many dominant mutations have been described in the so-called proto-oncogenes (OGs), while recessive mutations are typically described in tumor-suppressor genes (TSGs). Evidence indicates that mutations in X-chromosome TSGs are more likely to contribute to cancer than those in autosomal TSGs. Here, we formalize this in several dynamic models and predict, as expected, that mutations spread faster in TSGs located on the X chromosome than on autosomes (faster-X effect). Conversely, mutations in OGs spread faster on autosomes than on the X chromosome, but under high selective pressure, this difference is negligible. Published genomic screenings of cancer samples show evidence of the faster-X effect in TSGs. This pattern is observed in both sexes, suggesting that the maintenance of X-chromosome inactivation during cancer progression plays an important role in the evolution of TSGs. Strikingly, the relative mutation incidence in X-linked TSGs among females across individual studies is bimodal, with one group of studies showing a faster-X effect and another group showing similar incidences for X-linked and autosomal TSGs. This differentiation between cancer samples is not associated with the specific type of cancer or the tissue of origin. This may indicate that X-chromosome inactivation plays a differential role in the involvement of X-linked TSGs across individual cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e60393ec97b0f4b9cf086f3cab7030b54b21b01" target='_blank'>
              Somatic evolution of cancer genes in sex chromosomes
              </a>
            </td>
          <td>
            Amarachi Akachukwu, Jasmine S. Ratcliff, Antonio Marco
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Sex-chromosome dosage represents a challenge for heterogametic species to maintain correct proportion of gene products across chromosomes in each sex. While therian mammals (XX/XY system) achieve near-perfect balance of X-chromosome mRNAs through X-upregulation and X-inactivation, birds (ZW/ZZ system) have been found to lack efficient compensation at RNA level, challenging the necessity of resolving major gene-dosage discrepancies in avian cells. Through allele-resolved multiome analyses, we comprehensively examined dosage compensation in female (ZW), male (ZZ), and rare intersex (ZZW) chicken. Remarkably, this revealed that females exhibit upregulation of their single Z through increased transcriptional burst frequency similar to mammalian X-upregulation, and that Z-protein levels are further balanced via enhanced translation efficiency in females. Global analyses of transcriptional kinetics elements in birds demonstrate remarkable conservation of the genomic encoding of burst kinetics between mammals and birds. Our study uncovers new mechanisms for achieving sex-chromosome dosage compensation and highlights the importance of gene-dosage balance across diverse species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05f4b40e70c2988331617fc51e819b31adbb3451" target='_blank'>
              Multi-layered dosage compensation of the avian Z chromosome
              </a>
            </td>
          <td>
            Natali Papanicolaou, Antonio Lentini, Sebastian Wettersten, Michael Hagemann-Jensen, Annika Krüger, Jilin Zhang, Christos Coucoravas, Ioannis Petrosian, Xian Xin, Ilhan Ceyhan, Joanna Rorbach, D. Wright, B. Reinius
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="CTCF architects 3D genome folding by anchoring or forestalling cohesin loop extrusion, but the exact mechanism remains obscure. Here, using clustered protocadherins (cPcdh) as model genes, we report that CTCF stabilizes or facilitates cohesin loop extrusion by enhancing its processivity. Specifically, we show that, compared with the Pcdh α and γ clusters, the Pcdhβ cluster is greatly affected upon CTCFY226A/F228A mutation in the N-terminal domain. Given the long-range distance of the Pcdhβ cluster from the distal enhancer, this finding has interesting implications in CTCF regulation of cohesin processivity along the linear chromatin during DNA loop extrusion. In particular, the effect on cohesin processivity upon CTCFY226A/F228A mutation is conspicuously similar to that of WAPL overexpression, demonstrating that, contrary to the general view of blocking or forestalling cohesin, CTCF actually enhances or facilitates cohesin loop extrusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ae55bb55054fa2f97642ac29917aefae44f10b" target='_blank'>
              CTCF N-terminal Domain Regulates Clustered Protocadherin Gene Expression by Enhancing Cohesin Processivity
              </a>
            </td>
          <td>
            Yijun Zhang, Qiang Wu
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) are the remnants of retroviral germline infections and are highly abundant in the genomes of vertebrates. At one time considered to be nothing more than inert ‘junk’ within genomes, ERVs have been tolerated within host genomes over vast timescales, and their study continues to reveal complex co-evolutionary histories within their respective host species. For example, multiple instances have been characterized of ERVs having been ‘borrowed’ for normal physiology, from single copies to ones involved in various regulatory networks such as innate immunity and during early development. Within the cell, the accessibility of ERVs is normally tightly controlled by epigenetic mechanisms such as DNA methylation or histone modifications. However, these silencing mechanisms of ERVs are reversible, and epigenetic alterations to the chromatin landscape can thus lead to their aberrant expression, as is observed in abnormal cellular environments such as in tumors. In this review, we focus on ERV transcriptional control and draw parallels and distinctions concerning the loss of regulation in disease, as well as their precise regulation in early development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88fcb4c839dae363d4185cc295fc1a0003652e42" target='_blank'>
              Transcription of Endogenous Retroviruses: Broad and Precise Mechanisms of Control
              </a>
            </td>
          <td>
            Abigail S. Jarosz, Julia V. Halo
          </td>
          <td>2024-08-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transgenerational epigenetic inheritance (TEI) mediated by transposable elements (TEs) is well established in plants. However, the molecular determinants and functional impact of this additional system of inheritance are poorly characterized. Through comprehensive analyses of severe DNA methylation loss over TEs in an experimental population and in >700 natural strains of Arabidopsis thaliana, we uncovered hundreds of TE epivariants in nature, most of which can be inherited independently of DNA sequence changes and thus are true epialleles. Furthermore, natural epivariants are enriched near genes and we uncovered a novel role for the Paf1 complex as a major facilitator of their prevalence. Conversely, related TE copies limit epiallelic variation, by producing the small RNAs that guide DNA methylation in trans. Crucially, natural epivariants mainly affect stress-responsive genes, making them unique targets for selection. Our findings therefore demonstrate that TEs are important mediators of transgenerational epigenetic variation in nature, with singular properties for plant adaptation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac6bd4e49538ff89fe2173d3326edef5a9f7c98a" target='_blank'>
              Transposable elements are prevalent vectors of transgenerational epigenetic inheritance in nature
              </a>
            </td>
          <td>
            P. Baduel, Louna De Oliveira, E. Caillieux, Grégoire Bohl-Viallefond, Mounia El Messaoudi, Ciana Xu, Matteo Barois, Vipin Singh, Alexis Sarazin, Martine Boccara, Elodie Gilbault, Antoine de France, L. Quadrana, O. Loudet, V. Colot
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="Many integrative mobile genetic elements (MGEs) are sensitive to RecA-DNA filaments, suggesting their activation may enable them to evade deletion through homologous recombination (HR). This was studied using pneumococcal prophage. Most pneumococci contained prophage that inhibited transformation-mediated HR through chromosomal modification. These included a commonly-targeted attB that reduced transformation by modifying the small cytoplasmic RNA. The prophage were almost universally regulated by C1-type or ImmAR-type systems. Representatives with either type were efficiently deleted by HR. Mutations that impaired a C1-regulated prophage increased this deletion rate, reflecting this element being activated by RecA-DNA filaments imported during transformation. ImmAR-regulated prophage instead responded to transient stimuli by excising as deletion-resistant pseudolysogens, with varying RecA sensitivity, gradually committing to lysing cells following longer-duration signals. These processes were modulated by interactions with the DNA-binding protein DprA. Therefore bacterial diversification though HR is impeded by MGEs that selfishly prevent themselves from being eliminated from the chromosome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a246916c36cd9a48942af1a067709e422a31a41c" target='_blank'>
              Chromosomal curing drives prophage evolution and bacterial epidemiology
              </a>
            </td>
          <td>
            M. Kwun, Alexandru V. Ion, Katinka J. Apagyi, N. Croucher
          </td>
          <td>2024-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast Cancer Type 1 Susceptibility Protein (BRCA1) is a tumor-suppressor protein that regulates various cellular pathways, including those that are essential for preserving genome stability. One essential mechanism involves a BRCA1-A complex that is recruited to double-strand breaks (DSBs) by RAP80 before initiating DNA damage repair (DDR). How RAP80 itself is recruited to DNA damage sites, however, is unclear. Here, we demonstrate an intrinsic correlation between a methyltransferase DOT1L-mediated RAP80 methylation and BRCA1-A complex chromatin recruitment that occurs during cancer cell radiotherapy resistance. Mechanistically, DOT1L is quickly recruited onto chromatin and methylates RAP80 at multiple lysines in response to DNA damage. Methylated RAP80 is then indispensable for binding to ubiquitinated H2A and subsequently triggering BRCA1-A complex recruitment onto DSBs. Importantly, DOT1L-catalyzed RAP80 methylation and recruitment of BRCA1 have clinical relevance, as inhibition of DOT1L or RAP80 methylation seems to enhance the radiosensitivity of cancer cells both in vivo and in vitro. These data reveal a crucial role for DOT1L in DDR through initiating recruitment of RAP80 and BRCA1 onto chromatin and underscore a therapeutic strategy based on targeting DOT1L to overcome tumor radiotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7a53d6b1d8429c7df99fb0b7dc3e6119436184d" target='_blank'>
              DOT1L-mediated RAP80 methylation promotes BRCA1 recruitment to elicit DNA repair.
              </a>
            </td>
          <td>
            Huangqi Tang, Ya-Fei Lu, Rongsheng Zeng, Chaohua Liu, Yuxin Shu, Yupei Wu, Jiajie Su, Longjiang Di, Jinqin Qian, Jun Zhang, Yuan Tian, Xiaopeng Lu, Xin-Hai Pei, Qian Zhu, Wei-Guo Zhu
          </td>
          <td>2024-08-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f28b08db9038d5c3400d445c5aadfc149da32c65" target='_blank'>
              Centromeres in cancer: Unraveling the link between chromosomal instability and tumorigenesis.
              </a>
            </td>
          <td>
            Mohsen Karami Fath, Ahmad Nazari, Noushin Parsania, Paria Behboodi, Seyedeh Sara Ketabi, Pegah Razmjouei, Farnoosh Farzam, Seyyed-Ghavam Shafagh, Mohsen Nabi Afjadi
          </td>
          <td>2024-10-01</td>
          <td>Medical oncology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Mammalian genomes are scattered with transposable elements (TEs). TEs are epigenetically silenced to prevent harmful effects caused by either global activation leading to genome instability or insertional mutation disturbing gene transcription. However, whether the activation of a single element can contribute to pathological phenotypes without directly affecting gene expression is largely unknown. Here, we show that tissue-specific expression of a TE in the embryo leads to the production of viral-like particles (VLPs) which can affect organ formation. Failure to silence an LTR retrotransposon inserted upstream of the Fgf8 gene results in its co-expression with Fgf8 in the developing embryo. While local gene regulation is unaffected, the LTR retrotransposon participates in chromatin folding at the locus and adopts the expression of the regulatory domain it is located in. This drives the production of VLPs in the Fgf8-expressing cells of the developing limb, triggering apoptotic cell death at the time of digit outgrowth and resulting in a limb malformation resembling human ectrodactyly. This phenotype can be rescued by knock-out or knock-in of the retrotransposon causing mutations preventing its full retroviral cycle. Insertion of the same element at other developmental loci faithfully recapitulates expression according to the neighboring regulatory activity. Our findings provide a mechanism by which TE insertion is incorporated into the local genomic regulatory landscape and show how VLP production in post-implantation embryos can interfere with organ formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6530941b1bbda9df13ec9f7016e41dec0d4f10bf" target='_blank'>
              Enhancer adoption by an LTR retrotransposon generates viral-like particles causing developmental limb phenotypes
              </a>
            </td>
          <td>
            Juliane Glaser, Giulia Cova, B. Fauler, C. Prada-Medina, Virginie Stanislas, Mai H.Q. Phan, R. Schöpflin, Yasmin Aktas, Martin Franke, Guillaume Andrey, C. Paliou, Verena Laupert, W. Chan, L. Wittler, Thorsten Mielke, Stefan Mundlos
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Structural Maintenance of Chromosomes (SMC) complexes are an evolutionary conserved protein family. In most eukaryotes, three SMC complexes have been characterized, as follows: cohesin, condensin, and SMC5/6 complexes. These complexes are involved in a plethora of functions, and defects in SMC genes can lead to an increased risk of chromosomal abnormalities, infertility, and cancer. To investigate the evolution of SMC complex genes in mammals, we analyzed their selective patterns in an extended phylogeny. Signals of positive selection were identified for condensin NCAPG, for two SMC5/6 complex genes (SMC5 and NSMCE4A), and for all cohesin genes with almost exclusive meiotic expression (RAD21L1, REC8, SMC1B, and STAG3). For the latter, evolutionary rates correlate with expression during female meiosis, and most positively selected sites fall in intrinsically disordered regions (IDRs). Our results support growing evidence that IDRs are fast evolving, and that they most likely contribute to adaptation through modulation of phase separation. We suggest that the natural selection signals identified in SMC complexes may be the result of different selective pressures: a host-pathogen arms race in the condensin and SMC5/6 complexes, and an intragenomic conflict for meiotic cohesin genes that is similar to that described for centromeres and telomeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0cc74a6b808c47f9b7e227a7f90d663f5e5a7ca2" target='_blank'>
              Positive Selection Drives the Evolution of the Structural Maintenance of Chromosomes (SMC) Complexes
              </a>
            </td>
          <td>
            D. Forni, A. Mozzi, M. Sironi, R. Cagliani
          </td>
          <td>2024-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="DNA replication is constantly challenged by a wide variety of endogenous and exogenous stressors that can damage DNA. Such lesions encountered during genome duplication can stall replisomes and convert replication forks into double-strand breaks. If left unrepaired, these toxic DNA breaks can trigger chromosomal rearrangements, leading to heightened genome instability and an increased likelihood of cellular transformation. Additionally, cancer cells exhibit persistent replication stress, making the targeting of replication fork vulnerabilities in tumor cells an attractive strategy for chemotherapy. A highly versatile and powerful technique to study DNA breaks during replication is the comet assay. This gel electrophoresis technique reliably detects the induction and repair of DNA breaks at the single-cell level. Herein, a protocol is outlined that allows investigators to measure the extent of DNA damage in mitotically dividing human cells using fork-stalling agents across multiple cell types. Coupling this with automated comet scoring facilitates rapid analysis and enhances the reliability in studying induction of DNA breaks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bb1e23cd556c7368ee2eef1c5cc145deb5fd207" target='_blank'>
              Detection of DNA Breaks in Dividing Human Cells by Neutral Comet Assay.
              </a>
            </td>
          <td>
            Anthony Nelligan, Huzefa Dungrawala
          </td>
          <td>2024-08-23</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Topoisomerases alter DNA topology by making transient DNA strand breaks (DSBs) in DNA. The DNA cleavage reaction mechanism includes the formation of a reversible protein/DNA complex that allows rapid resealing of the transient break. This mechanism allows changes in DNA topology with minimal risks of persistent DNA damage. Nonetheless, small molecules, alternate DNA structures, or mutations in topoisomerase proteins can impede the resealing of the transient breaks, leading to genome instability and potentially cell death. The consequences of high levels of enzyme/DNA adducts differ for type I and type II topoisomerases. Top1 action on DNA containing ribonucleotides leads to 2–5 nucleotide deletions in repeated sequences, while mutant Top1 enzymes can generate large deletions. By contrast, small molecules that target Top2, or mutant Top2 enzymes with elevated levels of cleavage lead to small de novo duplications. Both Top1 and Top2 have the potential to generate large rearrangements and translocations. Thus, genome instability due to topoisomerase mis-function is a potential pathogenic mechanism especially leading to oncogenic progression. Recent studies support the potential roles of topoisomerases in genetic changes in cancer cells, highlighting the need to understand how cells limit genome instability induced by topoisomerases. This review highlights recent studies that bear on these questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fc620e1f3abd707db6dd8102f3cc18d304a5ef7" target='_blank'>
              Genome Instability Induced by Topoisomerase Misfunction
              </a>
            </td>
          <td>
            K. C. Nitiss, Afif F. Bandak, James M. Berger, J. Nitiss
          </td>
          <td>2024-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56963ef797b360ca0fd377e89bb8feeef832b223" target='_blank'>
              Genome-wide characterization of dynamic DNA 5-hydroxymethylcytosine and TET2-related DNA demethylation during breast tumorigenesis
              </a>
            </td>
          <td>
            Shuang-Ling Wu, Lin Yang, Changcai Huang, Qing Li, Chunhui Ma, Fang Yuan, Yinglin Zhou, Xiaoyue Wang, Wei-Min Tong, Yamei Niu, Feng Jin
          </td>
          <td>2024-09-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Carcinogen-induced mutations are thought near-random, with rare cancer-driver mutations underlying clonal expansion. Using high-fidelity Duplex Sequencing to reach a mutation frequency sensitivity of 4×10-9 per nt, we report that sun exposure creates pervasive mutations at sites with ∼100-fold UV-sensitivity in RNA-processing gene promoters – cyclobutane pyrimidine dimer (CPD) hyperhotspots – and these mutations have a mini-driver clonal expansion phenotype. Numerically, human skin harbored 10-fold more genuine mutations than previously reported, with neonatal skin containing 90,000 per cell; UV signature mutations increased 8,000-fold in sun-exposed skin, averaging 3×10-5 per nt. Clonal expansion by neutral drift or passenger formation was nil. Tumor suppressor gene hotspots reached variant allele frequency 0.1-10% via 30-3,000 fold clonal expansion, in occasional biopsies. CPD hyperhotspots reached those frequencies in every biopsy, with modest clonal expansion. In vitro, tumor hotspot mutations arose occasionally over weeks of chronic low-dose exposure, whereas CPD hyperhotspot mutations arose in days at 1000-fold higher frequencies, growing exponentially. UV targeted mini-drivers in every skin cell.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccbc5cb5ec1678ecc35a610c5268f3895b55100" target='_blank'>
              Pervasive Induction of Regulatory Mutation Microclones in Sun-exposed Skin
              </a>
            </td>
          <td>
            Vijay Menon, Alejandro Garcia-Ruiz, Susan Neveu, Brenda Cartmel, Leah M. Ferrucci, Meg Palmatier, Christine Ko, Kenneth Y. Tsai, Mio Nakamura, Sa Rang Kim, Michael Girardi, Karl Kornacker, Douglas E. Brash
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="There is increasing recognition that the sex chromosomes, X and Y, play an important role in health and disease that goes beyond the determination of biological sex. Loss of the Y chromosome (LOY) in blood, which occurs naturally in aging men, has been found to be a driver of cardiac fibrosis and heart failure mortality. LOY also occurs in most solid tumors in males and is often associated with worse survival, suggesting that LOY may give tumor cells a growth or survival advantage. We analyzed LOY in lung adenocarcinoma (LUAD) using both bulk and single-cell expression data and found evidence suggesting that LOY affects the tumor immune environment by altering cancer/testis antigen expression and consequently facilitating tumor immune evasion. Analyzing immunotherapy data, we show that LOY and changes in expression of particular cancer/testis antigens are associated with response to pembrolizumab treatment and outcome, providing a new and powerful biomarker for predicting immunotherapy response in LUAD tumors in males.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03627d4b1b03380930579b6c3a6ee53571e9ff88" target='_blank'>
              Selective loss of Y chromosomes in lung adenocarcinoma modulates the tumor immune environment through cancer/testis antigens
              </a>
            </td>
          <td>
            Jonas Fischer, K. Shutta, Chen Chen, Viola Fanfani, Enakshi Saha, Panagiotis Mandros, Marouen Ben Guebila, Joanne Xiu, Jorge Nieva, Stephen V. Liu, D. Uprety, D. Spetzler, C. Lopes-Ramos, Dawn L. DeMeo, John Quackenbush
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The transition of detached fragments of mitochondrial DNA into the nucleus and their integration into chromosomal DNA is a special kind of genetic variability that highlights the relation between the two genomes and their interaction in a eukaryotic cell. The human genome contains several hundreds of insertions of mtDNA fragments (NUMTS). This paper presents an overview of the current state of research in this area. To date, evidence has been obtained that the occurrence of new mtDNA insertions in the nuclear genome is a seldom but not exceptionally rare event. The integration of new mtDNA fragments into the nuclear genome occurs during double-strand DNA break repair through the non-homologous end joining mechanism. Along with evolutionarily stable “genetic fossils” that were integrated into the nuclear genome millions of years ago and are shared by many species, there are NUMTS that could be species-specific, polymorphic in a species, or “private”. Partial copies of mitochondrial DNA in the human nuclear genome can interfere with mtDNA during experimental studies of the mitochondrial genome, such as genotyping, heteroplasmy assessment, mtDNA methylation analysis, and mtDNA copy number estimation. In some cases, the insertion of multiple copies of the complete mitochondrial genome sequence may mimic paternal inheritance of mtDNA. The functional significance of NUMTS is poorly understood. For instance, they may be a source of variability for expression and splicing modulation. The role of NUMTS as a cause of hereditary diseases is negligible, since only a few cases of diseases caused by NUMTS have been described so far. In addition, NUMTS can serve as markers for evolutionary genetic studies. Of particular interest is the meaning of NUMTS in eukaryotic genome evolution. The constant flow of functionally inactive DNA sequences from mitochondria into the nucleus and its significance could be studied in view of the modern concepts of evolutionary theory suggesting non-adaptive complexity and the key role of stochastic processes in the formation of genomic structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/629a51f6a4b8a657abed2a512344c0b9fb23810b" target='_blank'>
              Liberties of the genome: insertions of mitochondrial DNA fragments into nuclear genome
              </a>
            </td>
          <td>
            M. Golubenko, V. P. Puzyrev
          </td>
          <td>2024-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Metastatic cancer cells traverse constricted spaces that exert forces on their nucleus and the genomic contents within. Cancerous tumors are highly heterogeneous and not all cells within them can achieve such a feat. Here, we investigated what initial genome architecture characteristics favor the constricted migratory ability of cancer cells and which arise only after passage through multiple constrictions. We identified a cell surface protein (ITGB4) whose expression correlates with increased initial constricted migration ability in human melanoma A375 cells. Sorting out this subpopulation allowed us to identify cellular and nuclear features that pre-exist and favor migration, as well as alterations that only appear after cells have passed through constrictions. We identified specific genomic regions that experienced altered genome spatial compartment profiles only after constricted migration. Our study reveals 3D genome structure contributions to both selection and induction mechanisms of cell fate change during cancer metastasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8e297c703eca1376eadbe1d1a6d4f49a31d4765" target='_blank'>
              Deciphering Pre-existing and Induced 3D Genome Architecture Changes involved in Constricted Melanoma Migration
              </a>
            </td>
          <td>
            Christopher Playter, Rosela Golloshi, Joshua H. Garretson, Alvaro Rodriguez Gonzalez, T. H. Olajide, Ahmed Saad, S. Benson, R. McCord
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba7ee5fba3b3bfc377211f04436065c1bd9deb16" target='_blank'>
              Elucidating the genotoxicity of Fusobacterium nucleatum-secreted mutagens in colorectal cancer carcinogenesis
              </a>
            </td>
          <td>
            Wenye Xu, Yuchen Zhang, Dongjiao Chen, Dan Huang, Yang Zhao, Wei Hu, Ling Lin, Yingzhi Liu, Shilan Wang, Judeng Zeng, Chuan Xie, H. Chan, Qing Li, Huarong Chen, Xiaodong Liu, Sunny H Wong, Jun Yu, Francis K. L. Chan, Matthew T V Chan, Siew C Ng, William K. K. Wu, Lin Zhang
          </td>
          <td>2024-09-27</td>
          <td>Gut Pathogens</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7234a63fd75fc71e91d39632a18b128298ba3fb" target='_blank'>
              Genome-wide identification of copy number variations in wrinkled skin cases of Xiang pigs
              </a>
            </td>
          <td>
            Xiaoli Liu, Fenbin Hu, Wei Wang, Xia Chen, Xi Niu, Shihui Huang, Zhou-cai Wang, Jiafu Wang, Xueqin Ran
          </td>
          <td>2024-08-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chromosomal aneuploidy, that is, numerical chromosome aberrations, is one of the molecular hallmarks of cancer. However, when neoplasms are studied with sequencing- and array-based approaches, chromosome numbers and ploidy states are typically inferred from bulk DNA data. Furthermore, published molecular estimates of neoplasia-associated aneuploidy often also include genomic imbalances resulting from various types of structural rearrangement, which likely result from other mechanisms than numerical chromosome aberrations. We thus analyzed chromosome numbers using single-cell cytogenetic data from 83,862 tumors, and show that both benign and malignant tumors are highly heterogeneous with regard to deviations from the normal, diploid state. Focusing on the chromosome numbers in 112 specific tumor types, defined by both exact morphologic diagnosis and organ location and from which data from ≥50 cases were available, we found two major clusters: one predominated by near-diploid neoplasms and one by neoplasms with extensive aneuploidy and one or more whole genome doublings. The former cluster included most benign solid tumors, myeloid neoplasms, and malignant gene fusion-associated solid tumors, whereas the latter was predominated by malignant solid tumors and lymphomas. For 16 malignant tumor types, the distribution of chromosome numbers could be compared to TCGA ploidy level data. Cytogenetic and molecular data correlated well, but the former indicates a higher level of clonal heterogeneity. The results presented here suggest shared pathogenetic mechanisms in certain tumor types and provide a reference for molecular analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b031c86f32544de4d075d55344d053563ec29d0f" target='_blank'>
              Aneuploidy in neoplasia: Single-cell data on 83,862 tumors.
              </a>
            </td>
          <td>
            Fredrik Mertens, Jakob Hofvander, Nils Mandahl, F. Mitelman
          </td>
          <td>2024-09-02</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="BACKGROUND The mitotic intra-S DNA damage checkpoint signaling gene set is potentially involved in cancers where the genes play an important role. 17 total genes are involved in this gene set: ATF2, CHEK2, EME1, EME2, FANCD2, HUS1, HUS1B, MDC1, MRE11, MSH2, MUS81, NEK11, RAD17, RAD9A, RAD9B, TIPIN, XPC. The aim of this study is to complete a pan-cancer profile of each gene in the mitotic intra-S DNA damage checkpoint signaling gene set in order to determine potential diagnostic and prognostic purposes, while also determining how they could be used in a clinical setting as therapeutic targets to help patients. METHODS Multiomic data was acquired for the 17 genes; over 9000 samples of 33 types of cancer were analyzed to create pan-cancer profiles of CNV, mRNA expression, and pathway analysis. RESULTS The CNVs of some of these genes are associated with the survival of MESO, PCPG, BLCA, SKCM, LUAD, HNSC, LUSC, OV, and BRCA could be affected by the mRNA expression of the genes which can involve regulation of copy number. CONCLUSION With sufficient investigation, the genes involved in mitotic intra-S DNA damage checkpoint signaling may contribute to the development of cancer and may be used as biomarkers for cancer prognosis and diagnosis. To prove their clinical use for diagnosis and prognosis, however, and to create workable applications in clinical settings, further work is required. However, these pan-cancer profiles provide a more comprehensive knowledge of the mitotic intra-S DNA damage checkpoint signaling gene set in cancer as well as valuable information for future reference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f222ed09772900dec985568ac443adca2980e33" target='_blank'>
              Potential Cancer Biomarkers: Mitotic Intra-S DNA Damage Checkpoint Genes
              </a>
            </td>
          <td>
            Kashvi Agarwal, Hengrui Liu
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/327f3b8a50e3e934afb3ca0aaae0fc0fd3ed123e" target='_blank'>
              An elevated rate of whole-genome duplications in cancers from Black patients
              </a>
            </td>
          <td>
            L. M. Brown, R. Hagenson, T. Koklič, I. Urbančič, Lu Qiao, J. Štrancar, J. Sheltzer
          </td>
          <td>2024-09-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Impaired control of the G1/S checkpoint allows initiation of DNA replication under non-permissive conditions. Unscheduled S-phase entry is associated with DNA replication stress, demanding for other checkpoints or cellular pathways to maintain proliferation. Here, we uncovered a requirement for ADARp150 to sustain proliferation of G1/S-checkpoint-defective cells under growth-restricting conditions. Besides its well-established mRNA editing function in inversely oriented short interspersed nuclear elements (SINEs), we found ADARp150 to exert a critical function in mitosis. ADARp150 depletion resulted in tetraploidization, impeding cell proliferation in mitogen-deprived conditions. Mechanistically we show that ADAR1 depletion induced aberrant expression of Cyclin B3, which was causative for mitotic failure and whole-genome duplication. Finally, we find that also in vivo ADAR1-depletion-provoked tetraploidization hampers tumor outgrowth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d2bb8be1edc7659c2bc78816fc95a0c749bf23" target='_blank'>
              ADARp150 counteracts whole genome duplication
              </a>
            </td>
          <td>
            Frank van Gemert, Alexandra Drakaki, Isabel Morales Lozano, Daniel de Groot, Maud Schoot Uiterkamp, N. Proost, C. Lieftink, M. van de Ven, R. Beijersbergen, Heinz Jacobs, H. te Riele
          </td>
          <td>2024-08-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed3b8831fd786b3f7f846b45187bda3701aa49a" target='_blank'>
              Pan-cancer analysis of m1A writer gene RRP8: implications for immune infiltration and prognosis in human cancers
              </a>
            </td>
          <td>
            Zhihui Huang, Koo Han Yoo, Duohui Li, Qingxin Yu, Luxia Ye, Wuran Wei
          </td>
          <td>2024-09-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbc12eb0a2eae0ddf37c9584cc484b594730250" target='_blank'>
              Genomic profiling of cell-free DNA from dogs with benign and malignant tumors
              </a>
            </td>
          <td>
            Hongchao Du, Wenfeng Liu, Yunfei Li, Lijuan Zhang, Fangfang Jiang, Dandan Zhu, Jingshuai Li, Pan Hu, Ningning Yan, Mao Mao, Shiyong Li
          </td>
          <td>2024-09-13</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eba36ecae0bb22585b48631895c433e55a25862" target='_blank'>
              Nucleolus and centromere Tyramide Signal Amplification-Seq reveals variable localization of heterochromatin in different cell types
              </a>
            </td>
          <td>
            Pradeep Kumar, Omid Gholamalamdari, Yang Zhang, Liguo Zhang, Anastassiia Vertii, Tom van Schaik, Daan Peric-Hupkes, Takayo Sasaki, David M. Gilbert, B. van Steensel, Jian Ma, Paul D. Kaufman, A. Belmont
          </td>
          <td>2024-09-13</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="SOWAHA is a protein-coding gene, also known as ANKRD43. Studies have indicated that SOWAHA can serve as a prognostic biomarker in colorectal cancer and pancreatic cancer. However, there are few reports about SOWAHA in other types of cancer and the specific mechanism of action of SOWAHA in cancer is also not clear. Based on National Center for Biotechnology Information (NCBI), The Cancer Genome Atlas (TCGA), Genotype-Tissue Expression Project (GTEx), cBioPortal, Human Protein Atlas (HPA), etc., we adopted bioinformatics methods to uncover the potential tumor genomic features of SOWAHA, including the correlation with prognosis, gene mutation, immune cell infiltration, and DNA methylation in different tumors and evaluated the association with tumor heterogeneity, stemness, chemokines chemokine receptors, and immunomodulators in pan-cancer. Besides, we knocked down SOWAHA in SW620 cells and performed RNA-seq analysis, then we conducted functional enrichment to uncover the biological significance of the gene set. SOWAHA has early diagnostic potential, and low expression of SOWAHA was associated with poor prognosis in was associated with poor prognosis in GBMLGG, PAAD, READ, etc. SOWAHA is associated with most tumor immune-infiltrating cells in pan-cancer. SOWAHA correlates with DNA methylation, tumor heterogeneity, and stemness in many epithelial carcinomas. Furthermore, SOWAHA is involved in many enzyme activity and metabolic pathways, mainly metabolic programming pathways in cancer. Additionally, we identified two potential transcription factors of SOWAHA, TBX4, and FOXP2, which are dysregulated in SW620 cells. Besides, the cell proliferation and viability in siSOWAHA groups are better than in siNC groups.SOWAHA, identified as a suppressor gene, and its role in the progression of colorectal cancer is primarily mediated through metabolic reprogramming mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bb7dfbc90beef304e4ac787c6b60478323925b6" target='_blank'>
              SOWAHA as a Cancer Suppressor Gene Influence Metabolic Reprogramming
              </a>
            </td>
          <td>
            Xiaohong Yi, Xianwen Zhang, Claire H Zhao, Yuhui Chen, Lijun Huang, Hua Zhong, Yumei Wang
          </td>
          <td>2024-09-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec914b4189eafe0cce08165c6a7c2a53bfc159e" target='_blank'>
              Unveiling FRG1’s DNA repair role in breast cancer
              </a>
            </td>
          <td>
            Shubhanjali Shubhanjali, Talina Mohapatra, Rehan Khan, Manjusha Dixit
          </td>
          <td>2024-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e4728df16ec9a48de516d9a37941944d6293186" target='_blank'>
              Intragenic DNA inversions expand bacterial coding capacity.
              </a>
            </td>
          <td>
            Rachael B Chanin, Patrick T. West, Jakob Wirbel, Matthew O Gill, Gabriella Z M Green, Ryan M. Park, Nora J. Enright, Arjun M. Miklos, A. S. Hickey, Erin F Brooks, K. Lum, I. Cristea, A. Bhatt
          </td>
          <td>2024-09-25</td>
          <td>Nature</td>
          <td>1</td>
          <td>58</td>
        </tr>

        <tr id="The CHAMP1 complex, a little-known but highly conserved protein complex consisting of CHAMP1, POGZ, and HP1α, is enriched in heterochromatin though its cellular function in these regions of the genome remain unknown. Here we show that the CHAMP complex promotes heterochromatin assembly at multiple chromosomal sites, including centromeres and telomeres, and promotes homology-directed repair (HDR) of DNA double strand breaks (DSBs) in these regions. The CHAMP1 complex is also required for heterochromatin assembly and DSB repair in highly-specialized chromosomal regions, such as the highly-compacted telomeres of ALT (Alternative Lengthening of Telomeres) positive tumor cells. Moreover, the CHAMP1 complex binds and recruits the writer methyltransferase SETDB1 to heterochromatin regions of the genome and is required for efficient DSB repair at these sites. Importantly, peripheral blood lymphocytes from individuals with CHAMP1 syndrome, an inherited neurologic disorder resulting from heterozygous mutations in CHAMP1, also exhibit defective heterochromatin clustering and defective repair of local DSBs, suggesting that a defect in DNA repair underlies this syndrome. Taken together, the CHAMP1 complex has a novel role in heterochromatin assembly and the enhancement of HDR in heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/454e4622c260f66b26c8cb18854b8ca092214e10" target='_blank'>
              CHAMP1 complex directs heterochromatin assembly and promotes homology-directed DNA repair
              </a>
            </td>
          <td>
            Feng Li, Tianpeng Zhang, Aleem Syed, Amira Elbakry, Noella Holmer, Huy Nguyen, Sirisha Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="During cellular differentiation, enhancers transform overlapping gradients of transcription factors (TFs) to highly specific gene expression patterns. However, the vast complexity of regulatory DNA impedes the identification of the underlying cis-regulatory rules. Here, we have characterized 62,126 fully synthetic DNA sequences to bottom-up dissect design principles of cell-state specific enhancers in the context of the differentiation of blood stem cells to seven myeloid lineages. Focusing on binding sites for 38 TFs and their pairwise interactions, we found that identical sites displayed both repressive and activating function, as a consequence of cellular context, site combinatorics, or simply predicted occupancy of a TF on an enhancer. Surprisingly, we found that combinations of activating sites frequently neutralized each other or even gained repressive function. These negative synergies convert quantitative imbalances in transcription factor expression into binary downstream activity patterns, a principle that can be exploited to build differentiation-state specific enhancers from scratch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab0f40a79f1a6210ac7f2c422c1cc54bfc294a6c" target='_blank'>
              Synthetic enhancers reveal design principles of cell state specific regulatory elements in hematopoiesis
              </a>
            </td>
          <td>
            R. Frömel, Julia Rühle, Aina Bernal Martinez, Chelsea Szu-Tu, Felix Pacheco Pastor, Rosa Martinez Corral, Lars Velten
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2dde0164af0025d5f19bf691294f177add4f7b" target='_blank'>
              Low-pass whole genome sequencing of circulating tumor cells to evaluate chromosomal instability in triple-negative breast cancer
              </a>
            </td>
          <td>
            S. Di Cosimo, M. Silvestri, C. De Marco, Alessia Calzoni, M. C. De Santis, Maria Grazia Carnevale, C. Reduzzi, Massimo Cristofanilli, Vera Cappelletti
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b0e84c70be39b55c5ab372a108ddda6ab970765" target='_blank'>
              Polθ: emerging synthetic lethal partner in homologous recombination-deficient tumors.
              </a>
            </td>
          <td>
            T. B. Bazan Russo, Clarissa Mujacic, Emilia Di Giovanni, Maria Concetta Vitale, Carla Ferrante Bannera, U. Randazzo, Silvia Contino, M. Bono, V. Gristina, A. Galvano, A. Perez, G. Badalamenti, A. Russo, V. Bazan, L. Incorvaia
          </td>
          <td>2024-08-09</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Human stem cells rely on enhanced DNA repair mechanisms to safeguard their ability to replenish somatic tissues. Telomerase counteracts telomere shortening and is a component of the stem cell DNA repair system that is regulated by ATM and ATR kinases. Here, we report that the DNA repair enzyme APEX2, but not its close paralog APEX1, is required for efficient telomerase reverse transcriptase (TERT) gene expression in human embryonic stem cells (hESC) and a melanoma cell line. We also observed that APEX2 knockdown significantly diminished telomerase enzyme activity. While APEX1 is known to regulate certain transcription factors, APEX2 has not been reported to influence gene expression. To gain insight into how APEX2 influences gene expression, we conducted RNA-seq following APEX2 knockdown in hESC. These results indicated that a number of genes, in addition to TERT, relied on APEX2 for efficient expression. Genes affected by APEX2 knockdown were significantly enriched for specific repetitive DNA families. These include mammalian-wide interspersed repeats (MIRs) and Alu elements. Chromatin immunoprecipitation experiments demonstrated the highest APEX2 binding near MIR sequences in TERT intron 2. Surprisingly, binding was low in the TERT proximal promoter, a region known to control TERT transcription. MIR and other repetitive DNA regions are common sites of DNA damage, suggesting that APEX2 recruitment and repair of TERT MIR sequences may play a role in influencing TERT expression. This new role for APEX2 in promoting efficient gene expression deepens our understanding of an emerging cancer therapeutic target. Further, as the TERT gene plays critical roles in stem cell maintenance, organismal development and aging, as well as in short telomere disorders and cancer, our observations provide insight into new strategies to modulate the expression of this important enzyme.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c18f456e9f9ac926ada6d6d91f9b7e89f672dbdc" target='_blank'>
              Apurinic/apyrimidinic endodeoxyribonuclease 2 (APEX2/APE2) is required for efficient expression of TERT in human embryonic stem cells
              </a>
            </td>
          <td>
            Josh L. Stern, Lindsay F. Rizzardi, Natalie R. Gassman
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU- 023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention. Author Summary Bile duct cancer has the highest incidence in northeastern Thailand. In this study, we explored the mtDNA of CCA cell lines, as mitochondria play a key role in cellular energy production. We aimed to identify mutations that could serve as therapeutic targets or biomarkers. Our analysis revealed that each CCA cell line has unique mtDNA profiles, including INDELs, SVs, and SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. yet they also shared some common mutations. Future research is needed to understand how these mutations affect cell function, interact with nuclear DNA, and their potential for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eafb79f26f32f1aeaf6d648b336c327747f051f" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapan, A. Jareanrat, V. Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c4d24999230a049c47c4ad63da565877c2b583c" target='_blank'>
              PPM1D activity promotes cellular transformation by preventing senescence and cell death.
              </a>
            </td>
          <td>
            Miroslav Stoyanov, Andra S. Martinikova, Kateřina Matějková, K. Horackova, P. Zemánková, Kamila Burdová, Zuzana Zemanova, P. Kleiblova, Z. Kleibl, L. Macurek
          </td>
          <td>2024-09-05</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="
 Ovarian cancer is one of the most lethal gynecological malignancies worldwide, posing immense challenges in early detection, displaying grim prognoses, and having high rates of recurrence and mortality. This malignancy exhibits a range of ethnic diversity-mortality rates, each with distinct sub-categories characterized by variations in both molecular attributes and clinical behavior. The most prevalent and also most aggressive among the various ovarian cancer sub-types is high-grade serous ovarian carcinoma (HGSOC). This carcinoma is marked by genomic instability, resulting in the malfunction of tumor suppressor genes and oncogenes, caused by homologous recombination deficiency, copy number variations, chromosomal abnormalities, and epigenetic changes. The pursuit of understanding and targeting genomic instability has significantly guided approaches to screening, prognosis, and therapeutic strategies in ovarian cancer. Further insights from molecular investigations have unveiled the regulatory role of microRNAs (short non-coding RNAs) in modulating the expression of target tumor suppressors and oncogenes by interacting with messenger RNA (mRNA). Studies have shown that miRNAs are dysregulated in ovarian cancer, however, no data has shown significant relationships between these miRNAs to the patient demographics, including race/ethnicity. miRNAs can alter oncogenes, and tumor suppressor gene expression levels, serving as early biomarkers. We utilized the TCGA database from the University of Alabama at Birmingham (UALCAN) software to analyze the expressions of targeted genes and miRNAs. We determined various oncogenes and tumor suppressor genes with higher statistical significance on survival rates, race, cancer stages, tumor grades, and miRNAs linked to them. Additionally, genomic instability was assessed using aneuploidy score, fraction of genome altered, mutation counts, mutation burden, and prognosis (i.e., overall survival and progression-free survival), downloaded from cBioPortal, and the genomic instability parameters were calculated by Spearman correlation, and represented by a heatmap. Further, DNA and RNA from 22 HGSOC samples of paraffin-embedded tissues of ovarian cancer patients were collected for RT-qPCR assays for several miRNAs. Of the genes analyzed, BRCA1, BRAF, CCNE1, and miRNAs linked to these genes showed statistically significant data on survival rates. BRAF/CCNE1 is upregulated while BRCA 1 is downregulated in ovarian high-grade serous carcinoma. We also found quantitative differential expressions of some miRNAs analyzed in African Americans versus White Americans. Successful identification of early molecular markers in ovarian cancer will potentially allow for early detection and decreased mortality rates with HGSOC. Analysis of the appearance and abundance of miRNAs by qPCR assay in early-stage ovarian cancer can aid as an early diagnostic and prognostic tool. In a further study, we hope to verify our results by testing a larger cohort of samples from African Ancestry.
 Citation Format: Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga. miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C075.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f1e0052202e2129f599047f6e77f66413139b6" target='_blank'>
              Abstract C075: miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry
              </a>
            </td>
          <td>
            Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The study of chromosomal shape, characteristics, and behavior in somatic cell division (mitosis) during growth and development and in germ cell division (meiosis) during reproduction is known as cytogenetics. Many techniques can be used for cytogenetics, including fluorescent in situ hybridization (FISH), spectral karyotyping (SKY), multicolor FISH (M-FISH), microarray, and optical genome mapping (OGM). OGM is a novel genome-wide method that can identify structural variants (SVs) and copy number variants (CNVs) with only one test. Genomic structural information that is difficult to obtain with DNA sequencing can be promptly obtained with OGM, in which large molecule lengths can be mapped at a reasonable cost. OGM is increasingly being used to investigate chromosome abnormalities in genetic disorders and human cancer, but it was first utilized in genome assembly and research. According to recent research, OGM is capable of identifying every clinically significant variation seen in trials using conventional care. OGM is being utilized to identify genomic abnormalities in patients with malignancies and constitutional illnesses. It is regarded as a revolution in the field of cytogenetics. Rather than sequencing DNA, OGM relies on DNA labeling. Currently, the OGM technique with the Saphyr system from Bionano Genomics is a widely utilized platform for cytogenetic analysis. In conclusion, OGM can now be considered a highly reliable method for the identification of chromosomal abnormalities in the diagnosis of tumors and hematological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26e6e0a2264e6747c604c9c870d801b4fcbe0131" target='_blank'>
              Cytogenetics and the Revolution of Optical Genome Mapping in the Diagnosis of Diseases.
              </a>
            </td>
          <td>
            Osama M Al-Amer, Yahya Khubrani
          </td>
          <td>2024-09-01</td>
          <td>Discovery medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epigenetic alterations are implicated in the early stages of tumorigenesis and are widely recognized as a ubiquitous phenomenon in cancer development. Aberrant epigenetic modifications can alter the expression of target genes, induce heterochromatin formation, and gradually drive normal cells towards immortalized tumor cells with significant consequences. SETDB1 (SET domain bifurcated histone lysine methyltransferase 1), a typical histone me-thyltransferase, promotes the formation of heterochromatin and inhibits the transcription of genes by modifying the methylation of lysine 9 of histone 3. SETDB1 is usually highly ex-pressed in tumors with high copy numbers, accompanied by poor prognosis and low patient survival rates, which is a typical case of abnormal epigenetic modification. We discuss the mechanism of SETDB1 in a variety of cancers and review the epigenetic inhibitors that have been reported in recent years, along with their anti-tumor effects. In addition, we summarize the role of SETDB1 in a variety of diseases and cell functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8068b1c29211fa575e01e9ddbf87ca49f2785f" target='_blank'>
              Targeting the Tumor Epigenetic Regulator SETDB1 for Tumor Therapy.
              </a>
            </td>
          <td>
            Cheng Peng, Nini Zhou, T. Chen, Jie Lei, Changwen Chen, Shun-qin Zhu
          </td>
          <td>2024-08-27</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The nucleus, a highly organized and dynamic organelle, plays a crucial role in regulating cellular processes. During cell differentiation, profound changes occur in gene expression, chromatin organization, and nuclear morphology. This review explores the intricate relationship between nuclear architecture and cellular function, focusing on the roles of the nuclear lamina, nuclear pore complexes (NPCs), sub-nuclear bodies, and the nuclear scaffold. These components collectively maintain nuclear integrity, organize chromatin, and interact with key regulatory factors. The dynamic remodeling of chromatin, its interactions with nuclear structures, and epigenetic modifications work in concert to modulate gene accessibility and ensure precise spatiotemporal control of gene expression. The nuclear lamina stabilizes nuclear shape and is associated with inactive chromatin regions, while NPCs facilitate selective transport. Sub-nuclear bodies contribute to genome organization and gene regulation, often by influencing RNA processing. The nuclear scaffold provides structural support, impacting 3D genome organization, which is crucial for proper gene expression during differentiation. This review underscores the significance of nuclear architecture in regulating gene expression and guiding cell differentiation. Further investigation into nuclear structure and 3D genome organization will deepen our understanding of the mechanisms governing cell fate determination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e3a4a1b9e55e69b48e38215e900d744d3c0775b" target='_blank'>
              Nuclear structures and their emerging roles in cell differentiation and development
              </a>
            </td>
          <td>
            Hye Ji Cha
          </td>
          <td>2024-08-22</td>
          <td>BMB Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary Human cells are typically diploid, that is, they contain two sets of chromosomes. Some cells, however, contain three or more sets of chromosomes and are called polyploid. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of undergoing asymmetric division via nuclear budding and bursting (amitosis), giving rise to daughter cells that proliferate rapidly and can contribute to tumorigenesis (transformation of normal cells to a neoplastic state). This commentary provides an update on these intriguing discoveries. Abstract Cell division is crucial for the survival of living organisms. Human cells undergo three types of cell division: mitosis, meiosis, and amitosis. The former two types occur in somatic cells and germ cells, respectively. Amitosis involves nuclear budding and occurs in cells that exhibit abnormal nuclear morphology (e.g., polyploidy) with increased cell size. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of producing progeny via amitotic cell division, and that a subset of emerging daughter cells proliferate rapidly, exhibit stem cell-like properties, and can contribute to tumorigenesis. Polyploid cells that arise in solid tumors/tumor-derived cell lines are referred to as polyploid giant cancer cells (PGCCs) and are known to contribute to therapy resistance and disease recurrence following anticancer treatment. This commentary provides an update on some of these intriguing discoveries as a tribute to Drs. Walen and Rajaraman.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149407ed82c685a5b413633998b54affc7e6d4b2" target='_blank'>
              Amitotic Cell Division, Malignancy, and Resistance to Anticancer Agents: A Tribute to Drs. Walen and Rajaraman
              </a>
            </td>
          <td>
            R. Mirzayans, David Murray
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Herpesviruses are ubiquitous pathogens that cause a wide range of disease. Upon nuclear entry, their genomes associate with histones and chromatin modifying enzymes that regulate the progression of viral transcription and outcome of infection. While the composition and modification of viral chromatin has been extensively studied on bulk populations of infected cells by chromatin immunoprecipitation, this key regulatory process remains poorly defined at single-genome resolution. Here we use high-resolution quantitative imaging to investigate the spatial proximity of canonical and variant histones at individual Herpes Simplex Virus 1 (HSV-1) genomes within the first 90 minutes of infection. We identify significant population heterogeneity in the stable enrichment and spatial proximity of canonical histones (H2A, H2B, H3.1) at viral DNA (vDNA) relative to established promyelocytic leukaemia nuclear body (PML-NB) host factors that are actively recruited to viral genomes upon nuclear entry. We show the replication-independent histone H3.3/H4 chaperone Daxx to cooperate with PML to mediate the enrichment and spatial localization of variant histone H3.3 at vDNA that limits the rate of HSV-1 genome decompaction to restrict the progress of immediate-early (IE) transcription. This host response is counteracted by the viral ubiquitin ligase ICP0, which degrades PML to disperse Daxx and variant histone H3.3 from vDNA to stimulate the progression of viral genome expansion, IE transcription, and onset of HSV-1 replication. Our data support a model of intermediate and sequential histone assembly initiated by Daxx that limits the rate of HSV-1 genome decompaction independently of the stable enrichment of histones H2A and H2B at vDNA required to facilitate canonical nucleosome assembly. We identify HSV-1 genome decompaction upon nuclear infection to play a key role in the initiation and functional outcome of HSV-1 lytic infection, findings pertinent to the transcriptional regulation of many nuclear replicating herpesvirus pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd10788fc22834b17fdae65c2c26cce49f7d5c7d" target='_blank'>
              Daxx mediated histone H3.3 deposition on HSV-1 DNA restricts genome decompaction and the progression of immediate-early transcription
              </a>
            </td>
          <td>
            Ashley P.E. Roberts, Anne Orr, V. Iliev, Lauren Orr, S. McFarlane, Zhousiyu Yang, I. Epifano, C. Loney, Milagros Collados Rodriguez, Anna R. Cliffe, Kristen L Conn, Chris Boutell
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10bd05b394b6759a7d6959abc8009eb518a19281" target='_blank'>
              Multifaceted role of the DNA replication protein MCM10 in maintaining genome stability and its implication in human diseases.
              </a>
            </td>
          <td>
            Sumayyah M. Q. Ahmed, Jayaprakash Sasikumar, Suparna Laha, Shankar Prasad Das
          </td>
          <td>2024-09-06</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf251e653d85e2d3fa3eb31d13a78d3467c5985" target='_blank'>
              Bioinformatics advances in eccDNA identification and analysis.
              </a>
            </td>
          <td>
            Fuyu Li, Wenlong Ming, Wenxiang Lu, Ying Wang, Xianjun Dong, Yunfei Bai
          </td>
          <td>2024-08-29</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="
 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents used to treat these cancers are common. One such chemotherapy is doxorubicin. Therefore, this project aims to reveal and understand the mechanisms of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from one ATRT and two RMC primary cell lines (BT16, PEDS005T, and RCRF1009T respectively) utilizing a dose escalation approach. This resulted in DR cell lines that are 10-fold to 30-fold resistant compared to the parental cell line. Upon generation of these stably resistant cell lines, we profiled their transcriptomes by RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1. To further probe these cell lines, we performed whole-genome sequencing (WGS) on two of our DR cell lines (BT16 and PEDS005T DR). Here, we found a chromosome 7 aneuploidy in BT16 DR cells, but no aneuploidy or mutations were seen in PEDS005T DR cells. ABCB1 is located on chromosome 7. We confirmed this aneuploidy via karyotyping. The mechanism of ABCB1 upregulation in the RMC DR cell lines remains unknown, suggesting multiple modalities leading to ABCB1 upregulation. We then overexpressed ABCB1 in two ATRT parental cell lines (BT16 and CHLA-06-ATRT) and one RMC cell line (PEDS005T) which led to enhanced short-term resistance. This contrasted to JMSU, a SMARCB1 wild-type bladder cancer cell line, where there was stable resistance. Together, this suggests that ABCB1 may be necessary but not sufficient to confer doxorubicin resistance in these SMARCB1-deficient cancer cells. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there may be different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, despite the well documented role of ABCB1 in drug resistance, there may be other factors involved in conferring stable resistance in the setting of SMARCB1-deficient cancers.
 Citation Format: Katie T. Skinner, Benjamin P. Lee, Jessica S Yoon, Andrew L. Hong. The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A069.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdc2732137e2a0d6d27507359d94e24f1e66d2f" target='_blank'>
              Abstract A069 The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P Lee, Jessica S Yoon, Andrew L. Hong
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Rationale: Tumor cells remodel transcriptome to construct an ecosystem with stemness features, which maintains tumor growth and highly malignant characteristics. However, the core regulatory factors involved in this process still need to be further discovered. Methods: Single cell RNA-sequncing (scRNA-seq) and bulk RNA-sequencing profiles derived from fetal liver, normal liver, liver tumors, and their adjacent samples were collected to analyze the ecosystem of liver cancer. Mouse models were established to identify molecular functions of oncofetal-related oncogenes using hydrodynamic tail vein injection. Results: We found that liver cancer rebuilt oncofetal ecosystem to maintain malignant features. Interestingly, we identified a group of RNA-binding proteins (RBPs) that were highly overexpressed with oncofetal features. Among them, TRIM71 was specifically expressed in liver cancers and was associated with poor outcomes. TRIM71 drove the carcinogenesis of hepatocellular carcinoma (HCC), and knockdown of TRIM71 significantly abolished liver cancer cell proliferation. Mechanistically, TRIM71 formed a protein complex with IGF2BP1, bound to and stabilized the mRNA of CEBPA in an m6A-dependent manner, enhance the serine/glycine metabolic pathway, and ultimately promoted liver cancer progression. Furthermore, we identified that all-trans-retinoic acid (ATRA) combined with e1A binding protein p300 (EP300) inhibitor A-485 repressed TRIM71, attenuated glycine/serine metabolism, and inhibited liver cancer cell proliferation with high TRIM71 levels. Conclusions: We demonstrated the oncofetal status in liver cancer and highlighted the crucial role of TRIM71 and provided potential therapeutic strategies and liver cancer-specific biomarker for liver cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce6c5051059c8c5192cce1ee450847798b9ec45" target='_blank'>
              Oncofetal TRIM71 drives liver cancer carcinogenesis through remodeling CEBPA-mediated serine/glycine metabolism
              </a>
            </td>
          <td>
            Ying Su, Ziteng Li, Qin Li, Xinyi Guo, He-Yun Zhang, Yan Li, Zhiqiang Meng, Shenglin Huang, Zhixiang Hu
          </td>
          <td>2024-08-12</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromosome-containing micronuclei are a hallmark of aggressive cancers. Micronuclei frequently undergo irreversible collapse, exposing their enclosed chromatin to the cytosol. Micronuclear rupture catalyzes chromosomal rearrangements, epigenetic abnormalities, and inflammation, yet mechanisms safeguarding micronuclear integrity are poorly understood. In this study, we found that mitochondria-derived reactive oxygen species (ROS) disrupt micronuclei by promoting a noncanonical function of charged multivesicular body protein 7 (CHMP7), a scaffolding protein for the membrane repair complex known as endosomal sorting complex required for transport III (ESCRT-III). ROS retained CHMP7 in micronuclei while disrupting its interaction with other ESCRT-III components. ROS-induced cysteine oxidation stimulated CHMP7 oligomerization and binding to the nuclear membrane protein LEMD2, disrupting micronuclear envelopes. Furthermore, this ROS-CHMP7 pathological axis engendered chromosome shattering known to result from micronuclear rupture. It also mediated micronuclear disintegrity under hypoxic conditions, linking tumor hypoxia with downstream processes driving cancer progression. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION Chromosomal instability, a hallmark of aggressive cancers, is characterized by the presence of micronuclei, which are cytosolic rupture-prone structures that contain entire chromosomes or chromosome arms. The irreversible collapse of micronuclear envelopes is a central event in tumor progression. Micronuclear collapse exposes the encapsulated DNA to the cytosol, catalyzing chromosomal rearrangements and epigenetic alterations that drive tumor heterogeneity as well as therapy resistance. Micronuclear rupture also activates inflammatory signaling pathways that reshape the tumor immune microenvironment, promoting metastasis. Despite its importance, the underlying mechanisms of micronuclear collapse are unclear. RATIONALE Maintaining the integrity of the nuclear membrane is essential for cellular functions and organismal viability. It is unclear what cellular safeguards are compromised in micronuclei that lead to the breakdown of nuclear membrane integrity. We posited that inherent dissimilarities between micronuclei and primary nuclei might explain their respective differences in membrane stability. For instance, micronuclei are 5- to 20-fold smaller than the primary nucleus. Secondly, they possess abnormal nuclear envelopes, as evidenced by aberrant nuclear pore function and reduced lamin B1 levels. In addition, once ruptured, micronuclear envelopes are seldom repaired. RESULTS In this study, we integrated prior observations with our findings that micronuclear envelope rupture occurs because of the aberrant interaction between micronuclei and mitochondria, a process that is mediated by reactive oxygen species (ROS). Indeed, ruptured micronuclei were more likely to be intermixed with the mitochondrial network compared with intact ones, and manipulation of ROS levels correspondingly altered the rates of micronuclear rupture. Subsequent proteomic analysis and genetic manipulations led us to the observation that ROS interfere with micronuclear integrity by disrupting the normal activity of the nuclear membrane repair complex known as the endosomal sorting complex required for transport III (ESCRT-III). Increases in ROS levels led to the micronuclear accumulation of the ESCRT-III scaffolding protein, charged multivesicular body protein 7 (CHMP7), promoting a noncanonical function. ROS inhibited the export of CHMP7, leading to its persistence in the micronucleus and aberrant binding to its inner nuclear membrane partner LEM domain nuclear envelope protein 2 (LEMD2). ROS-induced cysteine oxidation catalyzed CHMP7 aggregation and reduced its interaction with its canonical binding partners in the ESCRT-III complex. The binding of CHMP7 aggregates to LEMD2 promoted micronuclear membrane deformation and collapse. This effect was further exacerbated by ROS-dependent recruitment of the autophagy-related protein p62, which mediated the degradation of canonical ESCRT-III members, limiting any chances of subsequent membrane repair. Lastly, we found that this mechanism is relevant to human tumors. Elevated levels of ROS and aberrant CHMP7 function led to complex chromosomal rearrangements, or chromothripsis, known to arise from micronuclear rupture. Furthermore, ROS generated under hypoxia induced micronuclear rupture in a CHMP7-dependent manner. In line with these findings, we observed a significant increase in micronuclear rupture along with CHMP7 accumulation in hypoxic regions of human head and neck cancer as well as in ovarian tumors. CONCLUSION Our results have uncovered a pathological interaction between micronuclei and mitochondria that underlies micronuclear rupture. By implicating ROS as regulators of micronuclear integrity, this work provides a mechanistic link between ROS-inducing conditions (such as free-radical generation and hypoxia) and downstream processes known to arise from the presence of micronuclei, including chromosomal rearrangements, epigenetic dysregulation, and tumor-promoting inflammation. Mechanisms of micronuclear rupture. The proximity of mitochondria to micronuclei drives micronuclear membrane rupture through mitochondria-derived reactive oxygen species (ROS). ROS inhibit micronuclear export, leading to the excessive accumulation of CHMP7, a scaffolding protein associated with the nuclear membrane repair complex ESCRT-III. ROS-dependent cysteine oxidation promotes CHMP7 self-aggregation and its aberrant binding to the membrane protein LEMD2, causing micronuclear collapse. [Figure created with BioRender.com]">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1748efac80bee30de3b1de5b76ab1af0bb5d42d" target='_blank'>
              Micronuclear collapse from oxidative damage
              </a>
            </td>
          <td>
            Melody Di Bona, Yanyang Chen, Albert S. Agustinus, Alice Mazzagatti, Mercedes A. Duran, Matthew Deyell, Daniel Bronder, James Hickling, Christy Hong, Lorenzo Scipioni, Giulia Tedeschi, Sara Martin, Jun Li, Aušrinė Ruzgaitė, N. Riaz, Parin Shah, Edridge K. D’Souza, D. Z. Brodtman, Simone Sidoli, B.H. Diplas, Manisha Jalan, Nancy Y Lee, Alban Ordureau, Benjamin Izar, A. Laughney, Simon N. Powell, Enrico Gratton, S. Santaguida, J. Maciejowski, Peter Ly, Thomas M. Jeitner, S. Bakhoum
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>4</td>
          <td>62</td>
        </tr>

        <tr id="Aflatoxin B1 (AFB1), a potent mycotoxin, is one of the environmental risk factors that cause liver cancer. In the liver, the bioactivated AFB1 intercalates into the DNA double helix to form a bulky DNA adduct which will lead to mutation if left unrepaired. Here, we adapted the tXR-seq method to measure the nucleotide excision repair of AFB1-induced DNA adducts at single-nucleotide resolution on a genome-wide scale, and compared it with repair data obtained from conventional UV-damage XR-seq. Our results showed that transcription-coupled repair plays a major role in the damage removal process. We further analyzed the distribution of nucleotide excision repair sites for AFB1-induced DNA adducts within the 3D human genome organization. Our analysis revealed a heterogeneous AFB1-dG repair across four different organization levels, including chromosome territories, A/B compartments, TADs, and chromatin loops. We found that chromosomes positioned closer to the nuclear center and regions within A compartments have higher levels of nucleotide excision repair. Notably, we observed high repair activity around both TAD boundaries and loop anchors. These findings provide insights into the complex interplay between AFB1-induced DNA damage repair, transcription, and 3D genome organization, shedding light on the mechanisms underlying AFB1-induced mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6fed0c3af4e712a403c0af2af9bb447bf31d09" target='_blank'>
              Nucleotide excision repair of aflatoxin-induced DNA damage within the 3D human genome organization.
              </a>
            </td>
          <td>
            Yiran Wu, Muhammad Muzammal Adeel, Dian Xia, Aziz Sancar, Wentao Li
          </td>
          <td>2024-09-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Genome rearrangement is a hallmark of cancer and ageing. DNA double-strand breaks (DSBs) are highly toxic lesions that can generate genome rearrangements. Several pathways compete for DSB repair, and selection of the appropriate repair process is decisive for genetic stability. DSB repair act according to two steps: 1-the choice between nonhomologous end-joining (NHEJ) versus resection of DNA ends, generating single-stranded DNA (ssDNA); 2-on ssDNA, homologous recombination (HR) and sub-processes. Here we show that 53BP1, which plays a prime role at the first step by protecting against resection and fostering NHEJ, physically interacts with the catalytic subunits of the cAMP dependant protein kinase A (PKAcs). PKA favors the recruitment of 53BP1 at DNA damages sites, and consistently, prevents resection, favoring NHEJ. Inhibition of PKA stimulated resection and coherently reduces NHEJ. Reversely, activation of PKA with 8-Bromo-cAMP stimulate NHEJ and reduces HR. These data open new avenues for potential anticancer strategies. More generally, they underline the high complexity of the regulation of DSB repair, identifying PKA as a novel actor of the DNA damage response, acting at the choice of the DSB repair process, which might be essential for genome integrity maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb615918d35bbed71f2c3e0b64876e126043052" target='_blank'>
              PKA prevents resection of DNA double strand breaks and favor non-homologous end-joining
              </a>
            </td>
          <td>
            Houaria Khater-Sellou, I. Magdalou, Vasily Ogryzko, Bernard S. Lopez
          </td>
          <td>2024-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Introduction Biological processes such as aging, carcinogenesis, and immune response rely on the ability to maintain or rapidly expand cell populations. The fitness of the involved cells is constrained by their replicative potential, which is reflected in the cellular telomere content. Method We apply TelomereHunter to scATAC-seq data to determine telomere content on single-cell level, in a hematopoietic dataset consisting of 35,139 cells from samples of basal cell carcinoma patients receiving programmed cell death protein 1 (PD1) blockade treatment. Integrating information from open-chromatin-based signatures to assess cell identity, we characterize the heterogeneity of telomere length for individual cell populations pre- and post-immunotherapy. Results The extracted telomeric reads reflect the expected telomereome-to-genome fraction. Telomere content distributions differ significantly between cell populations, and the median telomere content in intermediate and terminal exhausted CD8+ T-cells pre-treatment is significantly correlated to response to PD-1 checkpoint blockade. Likewise, telomere content correlates with post-treatment cell proliferation in terminally exhausted and T follicular helper cells from responding patients. Conclusion Telomere content measurement from scATAC-seq data has a sufficiently high signal-to-noise ratio to detect significant differences between cell types. Furthermore, the telomere content of CD8+ exhausted T-cells pre-treatment is a putative biomarker for successful PD-1-based immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8c0b43ae3c8ae5883f7e563f1131356d52bd5e2" target='_blank'>
              Quantifying immune cell telomere content at single-cell resolution in context of PD-1 checkpoint immunotherapy
              </a>
            </td>
          <td>
            Niklas L. Engel, Lea Herzel, Julie Surmely, Hanna Frieß, Malte Simon, B. Brors, Charles D. Imbusch, L. Feuerbach
          </td>
          <td>2024-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca0012968a9023f1a5b2833b62edce18fdae052" target='_blank'>
              LncRNA DNAH17-AS1 promotes gastric cancer proliferation and radioresistance by sponging miR-202-3p to upregulate ONECUT2
              </a>
            </td>
          <td>
            Yugang Ge, Hui Cang, J-Q Xiao, Hongshuai Wu, Biao Wang, Qing Shao
          </td>
          <td>2024-09-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CRISPR-Associated Transposases (CASTs) hold tremendous potential for microbial genome editing due to their ability to integrate large DNA cargos in a programmable and site-specific manner. However, the widespread application of CASTs has been hindered by their low efficiency in diverse, non-model bacteria. In an effort to address this shortcoming, we conducted the first genome-wide screen for host factors impacting Vibrio cholerae CAST (VchCAST) activity and used the findings to increase VchCAST editing efficiency. A genome-wide loss-of-function mutant library in E. coli was screened to identify 15 genes that impact type VchCAST transposition. Of these, seven factors were validated to improve VchCAST activity and two were found to be inhibitory. Informed by homologous recombination involved effectors, RecD and RecA, we tested the λ-Red recombineering system in our VchCAST editing vectors, which increased its insertion meditated-editing efficiency by 25.7-fold in E. coli while maintaining high target specificity and similar insertion arrangements. Furthermore, λ-Red-enhanced VchCAST achieved increased editing efficiency in the industrially important bacteria Pseudomonas putida and the emerging pathogen Klebsiella michiganensis. This study improves understanding of factors impacting VchCAST activity and enhances its efficiency as a bacterial genome editor. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2483d77579dd4ab8c5f977694f46b22d04f0a6" target='_blank'>
              Identification of Proteins Influencing CRISPR-Associated Transposases for Enhanced Genome Editing
              </a>
            </td>
          <td>
            Leo Song, Amanda T.P. Alker, Agnès Oromí-Bosch, Sophia E. Swartz, Jonathan N.V. Martinson, Jigyasa Arora, Abby M. Wang, Rachel Rovinsky, Sara J. Smith, Emily C. Pierce, A. Deutschbauer, Jennifer A. Doudna, Brady F. Cress, Benjamin E. Rubin
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Transgene applications ranging from gene therapy to development of stable cell lines and organisms rely on maintaining expression of transgenes. To-date, the use of plasmid-based transgenes has been limited by loss of their expression shortly after delivery into target cells. This short-lived expression of plasmid-based transgenes has been largely attributed to host cell-mediated degradation and/or silencing of transgenes. To assess the impact of “priming” plasmid-based transgenes to adopt accessible chromatin states to promote gene expression, nucleosome positioning elements were introduced at promoters of transgenes, or vectors were pre-assembled into nucleosomes containing unmodified histones or histone mutants mimicking constitutively acetylated states at residues 9 and 14 of histone H3 or residue 16 of histone H4 prior to their introduction into cells, then transgene expression was monitored over time. DNA sequences capable of positioning nucleosomes could positively impact expression of adjacent transgenes in a distance-dependent manner in the absence of pre-assembly into chromatin. Intriguingly, pre-assembly of plasmids into chromatin facilitated prolonged expression of transgenes relative to plasmids that were not pre-packaged into chromatin. Interactions between pre-assembled chromatin states and nucleosome positioning-derived effects on expression of reporter genes were also assessed and, generally, nucleosome positioning played the predominant role in influencing gene expression relative to priming with hyperacetylated chromatin states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96d54a5f1c4a00f415814edf9a864ffd37484254" target='_blank'>
              Impacts of Nucleosome Positioning Elements and Pre-Assembled Chromatin States on Expression and Retention of Transgenes
              </a>
            </td>
          <td>
            Ronard Kwizera, Junkai Xie, Nathan P Nurse, Chongli Yuan, A. Kirchmaier
          </td>
          <td>2024-08-21</td>
          <td>Genes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="DNA lies at the heart of the central dogma of life. Altering DNA can modify the flow of information in fundamental cellular processes such as transcription and translation. The ability to precisely manipulate DNA has led to remarkable advances in treating incurable human genetic ailments and has changed the landscape of biological research. Genome editors such as CRISPR-Cas nucleases and TALENs have become ubiquitous tools in basic and applied biological research and have been translated to the clinic to treat patients. The specificity and modularity of these genome editors have made it possible to efficiently engineer genomic DNA; however, underlying principles governing editing outcomes in eukaryotes are still being uncovered. Editing efficiency can vary from cell type to cell type for the same DNA target sequence, necessitating de novo design and validation efforts. Chromatin structure and epigenetic modifications have been shown to affect the activity of genome editors because of the role they play in hierarchical organization of the underlying DNA. Understanding the nuclear search mechanism of genome editors and their molecular interactions with higher order chromatin will lead to improved models for predicting precise genome editing outcomes. Insights from such studies will unlock the entire genome to be engineered for the creation of novel therapies to treat critical illnesses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de15652cba672686fe241fc8c1aaf364c6e3c7f6" target='_blank'>
              Impact of Chromatin Organization and Epigenetics on CRISPR-Cas and TALEN Genome Editing.
              </a>
            </td>
          <td>
            Surbhi Jain, Guanhua Xun, Huimin Zhao
          </td>
          <td>2024-09-24</td>
          <td>ACS synthetic biology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5a77337f38c8a116c6d75b10e0eda38d81a91a" target='_blank'>
              ΔNp63 bookmarks and creates an accessible epigenetic environment for TGFβ-induced cancer cell stemness and invasiveness
              </a>
            </td>
          <td>
            Eleftheria Vasilaki, Yu Bai, Mohamad Moustafa Ali, Anders Sundqvist, A. Moustakas, C. Heldin
          </td>
          <td>2024-08-23</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>155</td>
        </tr>

        <tr id="Clustered regularly interspaced short palindromic repeats (CRISPR)-Cas (CRISPR-associated proteins) systems provide acquired heritable protection to bacteria and archaea against selfish DNA elements, such as viruses. These systems must be tightly regulated because they can capture DNA fragments from foreign selfish elements, and also occasionally from self-chromosomes, resulting in autoimmunity. Most known species from the halophilic archaeal genus Haloferax contain type I-B CRISPR-Cas systems, and the strongest hotspot for self-spacer acquisition by H. mediterranei was a locus that contained a putative transposable element, as well as the gene HFX_2341, which was a very frequent target for self-targeting spacers. To test whether this gene is CRISPR-associated, we investigated it using bioinformatics, deletion, over-expression, and comparative transcriptomics. We show that HFX_2341 is a global transcriptional regulator that can repress diverse genes, since its deletion results in significantly higher expression of multiple genes, especially those involved in nutrient transport. When over-expressed, HFX_2341 strongly repressed the transcript production of all cas genes tested, both those involved in spacer acquisition (cas1, 2 and 4) and those required for destroying selfish genetic elements (cas3 and 5–8). Considering that HFX_2341 is highly conserved in haloarchaea, with homologs that are present in species that do not encode the CRISPR-Cas system, we conclude that it is a global regulator that is also involved in cas gene regulation, either directly or indirectly.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00459123b00f7b1a9dd02b717cf9a4770d4090a0" target='_blank'>
              A Haloarchaeal Transcriptional Regulator That Represses the Expression of CRISPR-Associated Genes
              </a>
            </td>
          <td>
            Israela Turgeman-Grott, Yarden Shalev, Netta Shemesh, Rachel Levy, Inbar Eini, M. Pasmanik-Chor, U. Gophna
          </td>
          <td>2024-08-27</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly detrimental to cells, as improper repair can result in inheritable genetic rearrangements or cell death. The role of RNA:DNA hybrids (RDHs) in DSB repair remains poorly understood, but their transient accumulation and subsequent resolution are crucial for accurate repair. The absence of the end-joining factor Nej1 at DSBs significantly reduced RDH levels, which was linked to increased activity of the Dna2 nuclease. Dna2 limits the accumulation of hybrids at DSBs, with levels rising in the presence of a nuclease-dead Dna2 and RNH201 deletion. Dna2 has a heightened preference for resolving RDHs with 5’ RNA overhangs compared to duplex substrates with 5’ DNA overhangs. This selective resolution by Dna2 helps restrict hybrid accumulation at DSBs and promotes resection, a function not shared by Exo1. This study underscores the multifunctional roles of canonical repair factors in ensuring efficient homology-directed repair. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5b2cacdb3d323609fc9f634d2ea46c9da9e5cdc" target='_blank'>
              Dna2 nuclease resolves RNA:DNA hybrids at double-strand breaks
              </a>
            </td>
          <td>
            A. Mojumdar, Courtney Granger, Jennifer A. Cobb
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4568f027a9bf73e7d22e0523c166497f5edf7122" target='_blank'>
              Perturbing TET2 condensation promotes aberrant genome-wide DNA methylation and curtails leukaemia cell growth.
              </a>
            </td>
          <td>
            Lei Guo, Tingting Hong, Yi-Tsang Lee, Xue Hu, Guokai Pan, Rongjie Zhao, Yuhan Yang, Jingwen Yang, Xiaoli Cai, Logan Rivera, Jie Liang, Rui Wang, Yaling Dou, Srikanth Kodali, Wenbo Li, Leng Han, Bruno Di Stefano, Yubin Zhou, Jia Li, Yun Huang
          </td>
          <td>2024-09-09</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Telomeres are critical nucleoprotein structures that safeguard chromosome ends from degradation and fusion, thereby ensuring genomic stability. Without proper maintenance, telomeres progressively shorten with each cell division, leading to dysfunction and triggering chromosomal instability. This dysfunction is a significant driver of tumorigenesis, primarily by facilitating genetic alterations such as aneuploidy, gene amplifications, and chromosomal rearrangements. Cancer cells bypass cellular aging by activating mechanisms like telomerase reactivation or the Alternative Lengthening of Telomeres (ALT) pathway to maintain telomere length. This review explores the mechanisms by which telomere dysfunction contributes to genomic instability and cancer progression, including telomere shortening, breakage-fusion-bridge (B/F/B) cycles, and the ALT pathway. Additionally, it addresses the therapeutic potential of targeting telomere maintenance, highlighting current strategies like telomerase and ALT inhibitors. However, developing telomere-based therapies presents challenges, including resistance mechanisms, off-target effects, and potential impacts on normal stem cells. Emerging research areas such as the development of biomarkers and combination therapies offer promising directions for overcoming these challenges. Understanding telomere dynamics provides novel opportunities to exploit cancer cell vulnerabilities and advance treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/04b7c5196ace0bbc6bf2d90a9a3ba826ad745b1e" target='_blank'>
              Mechanisms of telomere dysfunction in cancer from genomic instability to therapy: A review
              </a>
            </td>
          <td>
            Amir Mohammad Karimi Forood
          </td>
          <td>2024-09-30</td>
          <td>International Journal of Science and Research Archive</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genome instability is a characteristic trait of tumours and includes changes in DNA and in chromosomes. The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for the assessment of chromatin stability in healthy and sick dogs. The assays identified genomic instabilities in dogs with cancer (squamous cell carcinoma) and in healthy dogs. The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable the more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. The identification of instabilities can be used in routine diagnostic examination in dogs with cancer for more accurate diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7548d405e861a202a9ad5c3f6e2cc2fac0cc66" target='_blank'>
              Genomic Instability in the Lymphocytes of Dogs with Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Ewa Wójcik, Emilia Kot, Iga Wójcik, A. Wysokińska, P. Matusevičius
          </td>
          <td>2024-09-24</td>
          <td>Animals</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Long noncoding (lnc)RNAs emerge as regulators of genome stability. The nuclear-enriched abundant transcript 1 (NEAT1) is overexpressed in many tumors and is responsive to genotoxic stress. However, the mechanism that links NEAT1 to DNA damage response (DDR) is unclear. Here, we investigate the expression, modification, localization, and structure of NEAT1 in response to DNA double-strand breaks (DSBs). DNA damage increases the levels and N6-methyladenosine (m6A) marks on NEAT1, which promotes alterations in NEAT1 structure, accumulation of hypermethylated NEAT1 at promoter-associated DSBs, and DSB signaling. The depletion of NEAT1 impairs DSB focus formation and elevates DNA damage. The genome-protective role of NEAT1 is mediated by the RNA methyltransferase 3 (METTL3) and involves the release of the chromodomain helicase DNA binding protein 4 (CHD4) from NEAT1 to fine-tune histone acetylation at DSBs. Our data suggest a direct role for NEAT1 in DDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0891d9260fe499de788b24c8ed246446e39b1f4e" target='_blank'>
              NEAT1 promotes genome stability via m6A methylation-dependent regulation of CHD4.
              </a>
            </td>
          <td>
            V. Mamontova, Barbara Trifault, Anne-Sophie Gribling-Burrer, Patrick Bohn, Lea Boten, Pit Preckwinkel, Peter Gallant, Daniel Solvie, C. Ade, Dimitrios Papadopoulos, Martin Eilers, Tony Gutschner, R. Smyth, Kaspar Burger
          </td>
          <td>2024-10-03</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b7f68d17e46790c6769aa99466f38e2be24e476" target='_blank'>
              An extrinsic motor directs chromatin loop formation by cohesin
              </a>
            </td>
          <td>
            Thomas M. Guérin, Christopher Barrington, G. Pobegalov, Maxim I Molodtsov, F. Uhlmann
          </td>
          <td>2024-08-19</td>
          <td>The EMBO Journal</td>
          <td>1</td>
          <td>59</td>
        </tr>

        <tr id="Accurate monitoring of base excision repair (BER) activity in cancer cells is critical for advancing the comprehension of DNA repair processes, gaining insights into cancer development, and guiding treatment strategies. However, current assay techniques for assessing BER activity in cancer cells face challenges due to the heterogeneous origins and diversity of BER enzymes. In this work, we present a highly reliable triple loop-interlocked DNA codec (GATED) that enables precise assessment of BER activity in cancer cells through signal amplification mediated by multienzyme orthogonal activation. The GATED device features a dumbbell-shaped DNA probe to encode two BER enzymes for BER-related signal conversion as well as two bound circular DNA to decode the apurinic/apyrimidinic sites for apurinic/apyrimidinic endonuclease 1 (APE1)-mediated signal amplification. Importantly, GATED is orthogonally activated by multiple target BER enzymes (i.e., uracil DNA glycosylase, thymine DNA glycosylase, and APE1), resulting in a unified fluorescent signal that significantly improves the detection specificity and sensitivity to BER enzymes. Additionally, we demonstrate that the GATED has exceptional biostability within complex biological systems, where it was successfully employed to monitor BER activity in cancer cells with high specificity and enabled cell-based high-throughput screening for BER inhibitors. The GATED provides a much-needed tool for the real-time monitoring of BER activity and the screening of BER inhibitors in cancer cells, potentially advancing both the investigation and clinical application of BER biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff4455a54c11e6664a89c2b008ca24acc5e5bb02" target='_blank'>
              Multienzymatic Orthogonal Activation of DNA Codec Enables Tumor-Specific Imaging of Base Excision Repair Activity.
              </a>
            </td>
          <td>
            Wei-Jia Zeng, Zhaopeng Chen, Yan-Mei Lei, W. Liang, Yaqin Chai, Ruo Yuan, Ying Zhuo
          </td>
          <td>2024-09-26</td>
          <td>Analytical chemistry</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca19d3c949a93e74e54e4af39448ca2941e944a6" target='_blank'>
              HDAC6 inhibition disrupts HDAC6-P300 interaction reshaping the cancer chromatin landscape
              </a>
            </td>
          <td>
            Michela Gottardi Zamperla, B. Illi, V. Barbi, C. Cencioni, Daniele Santoni, Stella Gagliardi, Maria Garofalo, Gabriele Antonio Zingale, Irene Pandino, Diego Sbardella, Lina Cipolla, Simone Sabbioneda, A. Farsetti, Chiara Ripamonti, Gianluca Fossati, Christian Steinkühler, Carlo Gaetano, S. Atlante
          </td>
          <td>2024-08-18</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667549573815f8b84f3de00bdda0de7e5af3d623" target='_blank'>
              Emerin mislocalization during chromatin bridge resolution can drive prostate cancer cell invasiveness in a collagen-rich microenvironment
              </a>
            </td>
          <td>
            M. Popęda, Kamil Kowalski, Tomasz Wenta, Galina V Beznoussenko, Michał Rychłowski, Alexander A. Mironov, Zeno Lavagnino, Sara Barozzi, Julia Richert, Rebecca Bertolio, Kamil Myszczyński, J. Szade, Michał Bieńkowski, Kevin Miszewski, Marcin Matuszewski, A. Żaczek, Luca Braga, Giannino Del Sal, N. Bednarz-Knoll, P. Maiuri, P. Nastały
          </td>
          <td>2024-09-01</td>
          <td>Experimental & Molecular Medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba675eb14cd995fdc082910df7aeef7f5f1a429b" target='_blank'>
              Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Alva Annett, Elham Karimi, D. G. Topouza, Morteza Rezanejad, Yitong Liu, Michael McNicholas, Eduardo G Gonzalez Santiago, Dhana Llivichuzhca-Loja, Arne Gehlhaar, Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Caterina Russo, D. Faury, Geoffroy Danieau, Evan Puligandla, Yuhong Wei, M. Zeinieh, Qing Wu, S. Hébert, N. Juretic, Emily M. Nakada, Brian Krug, Valérie Larouche, Alexander G Weil, Roy W. R. Dudley, Jason Karamchandani, S. Agnihotri, D. Quail, B. Ellezam, Liza Konnikova, Logan A Walsh, Manav Pathania, Claudia L Kleinman, Nada Jabado
          </td>
          <td>2024-09-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="In eukaryotes, the cellular processes contributing to gamete formation form the building blocks of genetic inheritance across generations. While traditionally viewed as conserved across model organisms, emerging studies reveal significant variation among eukaryotes in meiotic and post-meiotic processes. Extending our knowledge to non-model organisms is therefore critical to improve our understanding of the evolutionary origin, significance and subsequent modifications associated with gamete formation. We describe the mechanisms underlying recombination, chromosome segregation and meiotic sex chromosome inactivation in the stick insect group Timema. Our results reveal that in males, 1) recombination initiates before synapsis, 2) the mechanisms of X inactivation are variable among species and the X remains silenced despite two waves of transcriptional activation in autosomes during spermatogenesis, 3) chromosome segregation is preceded by the recruitment of centromere proteins that either bind locally or longitudinally along the chromosomes. Together, our observations help understand the evolutionary significance of key cellular processes related to spermatogenesis and shed light on the diversity of their underlying mechanisms among species, including Timema stick insects.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c8577ddd5aaf9fc201ec5378fe865f42882b1cd" target='_blank'>
              Dynamics of recombination, X inactivation and centromere proteins during stick insect spermatogenesis
              </a>
            </td>
          <td>
            Zoé Dumas, William Toubiana, Marie Delattre, Tanja Schwander
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Breast cancer (BC) is a prevailing malignancy among women, and its inconspicuous development contributes significantly to mortality. The RNA N6-methyladenosine (m6A) modification represents an emerging mechanism for gene expression regulation, with the active involvement of the YTH N6-methyladenosine RNA binding protein 3 (YTHDF3) in tumor progression across multiple cancer types. Nonetheless, its precise function in breast cancer necessitates further investigation. Methods The expression of YTHDF3 in both cell lines and patient tissues was examined using Western blotting, reverse transcription quantitative PCR (RT-qPCR), and immunohistochemistry (IHC) techniques. Bioinformatics analysis of methylated RNA immunoprecipitation sequencing (MeRIP-seq) and transcriptome RNA sequencing (RNA-seq) data was employed to screen for the target genes of YTHDF3. The main focus of this study was to investigate the in vitro biological functions of YTHDF3. The specific binding of YTHDF3 to its target genes and its correlation with m6A methylation were studied through RNA pull-down, RNA immunoprecipitation, and co-immunoprecipitation experiments. The protein regulatory mechanisms of downstream genes of YTHDF3 were assessed using protein stability analysis. Furthermore, the biological functions of YTHDF3 and its target genes in breast cancer cells were validated through CRISPR-Cas9 technology and rescue experiments. Results By constructing a risk model using the TCGA database, YTHDF3 was identified as a high-risk factor among m6A methylation factors. Subsequent investigations revealed its elevated expression in various subtypes of breast cancer, accompanied by poor prognosis. MeRIP-seq analysis further revealed fibroblast growth factor 2 (FGF2) as a downstream gene of YTHDF3. Knockdown of YTHDF3 in breast cancer cells led to significant inhibition of cell self-renewal, migration, and invasion abilities in vitro. Mechanistically, YTHDF3 specifically recognized the methylated transcript of FGF2 within its coding sequence (CDS) region, leading to the inhibition of FGF2 protein degradation. Moreover, depletion of FGF2 markedly suppressed the biological functions of breast cancer cells, while reducing FGF2 expression in YTHDF3-overexpressing breast cancer cell lines substantially alleviated the malignant progression. Conclusions In summary, our study elucidates the role of YTHDF3 as an oncogene in maintaining FGF2 expression in BC cells through an m6A-dependent mechanism. Additionally, we provide a potential biomarker panel for prognostic prediction in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4a36ec9c5649a0b1a66054fa0cef4c652b513ff" target='_blank'>
              YTHDF3 modulates the progression of breast cancer cells by regulating FGF2 through m6A methylation
              </a>
            </td>
          <td>
            R. F. Gong, Z. H. Zhang, T. T. Sun, Y. X. Zhao, Wen Fang
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary Forkhead-box M1 (FOXM1) is a proliferation-associated transcription factor, overexpressed in almost all the cancers, making its regulation worth investigating. FOXM1 is known to bind to the promoters of certain microRNAs, a class of biomolecules involved in post-transcriptional gene regulation. Database mining followed by Venn analysis led us to the shortlisting of seven microRNAs, among which microRNA-532-3p showed the strongest interaction with FOXM1 3’UTR. The ectopic overexpression of miR-532-3p in colorectal cancer (CRC) cells led to decreased FOXM1 levels which resulted in the alteration of CRC functionalities. These altered phenotypes were supported by a change in the expression of proliferation and EMT markers. In conclusion, the present study suggested that the regulation of FOXM1 by microRNA-532-3p via its interaction with FOXM1 3’UTR inhibited CRC proliferation, migration, and invasion. Abstract Colorectal cancer (CRC) is a heterogeneous disease and classified into various subtypes, among which transcriptional alterations result in CRC progression, metastasis, and drug resistance. Forkhead-box M1 (FOXM1) is a proliferation-associated transcription factor which is overexpressed in CRC and the mechanisms of FOXM1 regulation have been under investigation. Previously, we showed that FOXM1 binds to promoters of certain microRNAs. Database mining led to several microRNAs that might interact with FOXM1 3’UTR. The interactions between shortlisted microRNAs and FOXM1 3’UTR were quantitated by a dual-luciferase reporter assay. MicroRNA-532-3p interacted with the 3’UTR of the FOXM1 mRNA transcript most efficiently. MicroRNA-532-3p was ectopically overexpressed in colorectal cancer (CRC) cell lines, leading to reduced transcript and protein levels of FOXM1 and cyclin B1, a direct transcriptional target of FOXM1. Further, a clonogenic assay was conducted in overexpressed miR-532-3p CRC cells that revealed a decline in the ability of cells to form colonies and a reduction in migratory and invading potential. These alterations were reinforced at molecular levels by the altered transcript and protein levels of the conventional EMT markers E-cadherin and vimentin. Overall, this study identifies the regulation of FOXM1 by microRNA-532-3p via its interaction with FOXM1 3’UTR, resulting in the suppression of proliferation, migration, and invasion, suggesting its role as a tumor suppressor in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/affbc7e03338ebd318b510b5965cce7c6dfa9311" target='_blank'>
              MicroRNA-532-3p Modulates Colorectal Cancer Cell Proliferation and Invasion via Suppression of FOXM1
              </a>
            </td>
          <td>
            Nita Ahuja, Akira Yamauchi, David Wong, Ketakee Mahajan, Ani V. Das, S. Alahari, Ramesh Pothuraju, S. A. Nair
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="BACKGROUND
Extrachromosomal circular DNA (eccDNA) has emerged as a central focus in molecular biology, with various types being found across species through advanced techniques, including high-throughput sequencing. This dynamic molecule exerts significant influence on aging and immune function and plays pivotal roles in autoimmune diseases, type 2 diabetes mellitus, cancer, and genetic disorders.


SUMMARY
This comprehensive review investigates the classification, characteristics, formation processes, and multifaceted functions of eccDNA, providing an in-depth exploration of its mechanisms in diverse diseases.


KEY MESSAGES
The goal of this review is to establish a robust theoretical foundation for a more comprehensive understanding of eccDNA, offering valuable insights for the development of clinical diagnostics and innovative therapeutic strategies in the context of related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f915db54a9ed6fe76aa5229ced4103ebd9df09e" target='_blank'>
              Exploring the Role of Extrachromosomal Circular DNA in Human Diseases.
              </a>
            </td>
          <td>
            Yali Peng, Huihui Tao, Guoying Wang, Mengyao Wu, Tinatin Xu, Chunmei Wen, Xuejia Zheng, Yong Dai
          </td>
          <td>2024-09-30</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Simple Summary By hypothesizing that molecular hyper-recombination and cellular proliferation are among the major features of multi-factorial cancer proneness, by using skin fibroblasts derived from eight major cancer syndromes, a significant correlation was found between the hyper-recombination rate quantified by plasmid assay, proliferation capacity assessed by flow cytometry, and excess of relative cancer risk (ERR). The product of the hyper-recombination rate and capacity of proliferation described a linear function of ERR. Abstract Context: Although carcinogenesis is a multi-factorial process, the mutability and the capacity of cells to proliferate are among the major features of the cells that contribute together to the initiation and promotion steps of cancer formation. Particularly, mutability can be quantified by hyper-recombination rate assessed with specific plasmid assay, hypoxanthine-guanine phosphoribosyltransferase (HPRT) mutations frequency rate, or MRE11 nuclease activities. Cell proliferation can be assessed by flow cytometry by quantifying G2/M, G1 arrests, or global cellular evasion. Methods: All these assays were applied to skin untransformed fibroblasts derived from eight major cancer syndromes characterized by their excess of relative cancer risk (ERR). Results: Significant correlations with ERR were found between hyper-recombination assessed by the plasmid assay and G2/M arrest and described a third-degree polynomial ERR function and a sigmoidal ERR function, respectively. The product of the hyper-recombination rate and capacity of proliferation described a linear ERR function that permits one to better discriminate each cancer syndrome. Conclusions: Hyper-recombination and cell proliferation were found to obey differential equations that better highlight the intrinsic bases of cancer formation. Further investigations to verify their relevance for cancer proneness induced by exogenous agents are in progress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6745d80a485fc8627c999d4c147aa5ba6f6eca06" target='_blank'>
              Prediction of Cancer Proneness under Influence of X-rays with Four DNA Mutability and/or Three Cellular Proliferation Assays
              </a>
            </td>
          <td>
            Laura El Nachef, L. Bodgi, Maxime Estavoyer, Simon Buré, Anne-Catherine Jallas, A. Granzotto, Juliette Restier-Verlet, Laurène Sonzogni, J. Al-Choboq, Michel Bourguignon, Laurent Pujo-Menjouet, N. Foray
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Chromatin organization is important for gene transcription in pig genome. However, its three-dimensional (3D) structure and dynamics are much less investigated than those in human. Here we applied the long-reads chromatin interaction analysis by paired-end tag sequencing (ChIA-PET) method to map the whole-genome chromatin interactions mediated by CCCTC-binding factor (CTCF) and RNA polymerase Ⅱ (RNAPⅡ or POLⅡ) in porcine macrophage cells before and after polyinosinic-polycytidylic acid [Poly(I:C)] induction. Our results revealed that Poly(I:C) induction impacts the 3D genome organization in the 3D4/21 cells at the fine-scale chromatin loop level rather than at the large-scale domain level. Furthermore, our findings underscored the pivotal role of CTCF anchored chromatin interactions in reshaping chromatin architecture during immune responses. Knock-out of the CTCF locus further confirmed that the CTCF anchored enhancers are associated with the activation of immune genes via long-range interactions. Notably, ChIA-PET data also supported the spatial relationship between single nucleotide polymorphisms (SNPs) and the related gene transcription in 3D genome aspect. Our findings in this study provide new clues and potential targets to explore key elements related to diseases in swine and are also likely to shed light on elucidating chromatin organization and dynamics underlying the process of mammalian infectious diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97442a763cf352589738944e6bb5a4d860334082" target='_blank'>
              Virus Infection Induces Immune Gene Activation with CTCF Anchored Enhancers and Chromatin Interactions in Pig Genome.
              </a>
            </td>
          <td>
            Jianhua Cao, Ruimin Ren, Xiaolong Li, Xiaoqian Zhang, Yan Sun, Xiaohuan Tian, Ru Liu, Xiangdong Liu, Yijun Ruan, Guoliang Li, Shuhong Zhao
          </td>
          <td>2024-09-23</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bba6936c2ac2d76bcee0a653b32c7b9ef4038fa" target='_blank'>
              Yeast EndoG prevents genome instability by degrading extranuclear DNA species
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Ruofan Yu, P. Thakre, Brenna S. McCauley, Nic Nikoloutsos, Yang Yu, Qian Li, P. Hastings, Weiwei Dang, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-09-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e48d863d20b27ee2206ff6d80ccbbfa69c29c0c2" target='_blank'>
              3D genome contributes to MHC-II neoantigen prediction
              </a>
            </td>
          <td>
            Mofan Feng, Liangjie Liu, K. Su, Xianbin Su, Luming Meng, Zehua Guo, Dan Cao, Jiayi Wang, Guang He, Yi Shi
          </td>
          <td>2024-09-26</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48137767fd26c2f7051bdcf540a74692bdb8a20b" target='_blank'>
              The role of hexokinases in epigenetic regulation: altered hexokinase expression and chromatin stability in yeast
              </a>
            </td>
          <td>
            Srinivasu Karri, Quinn Dickinson, Jing Jia, Yi Yang, Haiyun Gan, Zhiquan Wang, Yibin Deng, Chuanhe Yu
          </td>
          <td>2024-08-27</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c1e4b7568509ca6d857531ee73ea8a0c9b0585" target='_blank'>
              Decoding the genomic landscape of chromatin-associated biomolecular condensates
              </a>
            </td>
          <td>
            Zhaowei Yu, Qi Wang, Qichen Zhang, Yawen Tian, Guo Yan, Jidong Zhu, Guangya Zhu, Yong Zhang
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c14f4ac3142485cbddf01506f6b88958b2474175" target='_blank'>
              YIPF2 regulates genome integrity
              </a>
            </td>
          <td>
            Xiao Zhang, Tao Wang
          </td>
          <td>2024-09-05</td>
          <td>Cell & Bioscience</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aed930352660788bbfcd1cc9c93c4e4f947acaa" target='_blank'>
              PD-L1 deglycosylation promotes its nuclear translocation and accelerates DNA double-strand-break repair in cancer
              </a>
            </td>
          <td>
            Zhen Shu, Bhakti Dwivedi, Jeffrey M. Switchenko, David S Yu, Xingming Deng
          </td>
          <td>2024-08-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64123a29f4e4d91a84682b8e0733680d1726f975" target='_blank'>
              Phase separation of phospho-HDAC6 drives aberrant chromatin architecture in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Bing Lu, Ru Qiu, Jiatian Wei, Li Wang, Qinkai Zhang, Mingsen Li, Xiudan Zhan, Jian Chen, I-yun Hsieh, Ciqiu Yang, Jing Zhang, Zicheng Sun, Yifan Zhu, Tao Jiang, Han Zhu, Jie Li, Wei Zhao
          </td>
          <td>2024-08-28</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/311bc754350204df7366f1cc87322a6aee4bc5a6" target='_blank'>
              Assessment of SWI/SNF chromatin remodeling complex related genes as potential biomarkers and therapeutic targets in pan-cancer
              </a>
            </td>
          <td>
            Kai Zhuang, Lishan Wang, Chengyu Lu, Zhiping Liu, Dongli Yang, Hao Zhong, Jiami Zou, Aamir Fahira, Jiaojiao Wang, Zunnan Huang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b02e26ddc26802806307ff2a8acf516e9dc75bb" target='_blank'>
              Exosomal transcript cargo and functional correlation with HNSCC patients’ survival
              </a>
            </td>
          <td>
            Joni Yadav, A. Chaudhary, Tanya Tripathi, Divya Janjua, Udit Joshi, N. Aggarwal, Arun Chhokar, Chetkar Chandra Keshavam, Anna Senrung, A. Bharti
          </td>
          <td>2024-09-13</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, S. Shen, S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 The most common cancer in Korea is thyroid cancer, followed by lung cancer, and the most common cancer in men is lung cancer. The cancer with the highest mortality rate is lung cancer. Recently, the precision medical approach is becoming a major treatment strategy for lung cancer. For precision medical approach, the usefulness of next-generation sequencing (NGS) analysis targeting circulating tumor DNA (ctDNA) is being emphasized. Therefore, in this study, NGS analysis was performed on blood and tissues of advanced lung cancer patients to compare and analyze genetic mutation patterns to confirm the usefulness of ctDNA NGS assay.



 A total of 101 patients with advanced lung cancer (stage 3 or 4) were recruited, and ctDNA NGS assays were performed on their plasma using a Pan100 panel (Dxome). Tissues from primary lung cancer, metastatic lymph nodes, or metastatic organs of 22 patients were analyzed using ONCOaccuPanel (NGeneBio). All variants were classified into four tiers based on the level of clinical significance, according to the standards and guidelines established by the Association for Molecular Pathology, American Society of Clinical Oncology, and College of American Pathologists.



 In the ctDNA NGS assay of 101 patients, 70 (69.3%) harbored 154 tier 1 or 2 variants. Mutations in TP53 (45, 29.2%) and EGFR (23, 14.9%) were the most frequently detected, followed by mutations in RB1, CHEK2, ATM, JAK2, ATR, DNMT3A, and KRAS. Among the 22 patients whose tissue was examined, tier 1 or 2 variants were detected in both the plasma and tissue of 14 patients and only in the tissue of six patients. All genes for these variants were targeted using both methods. Identical variants were detected in the plasma and tissue of six of the 14 patients, and discordant results were observed in the remaining eight patients. Twenty copy number variations were detected in 13 of the 101 patients, with copy number gain of EGFR being the most common (6, 30%). Fusion genes were detected in five of the 101 patients, two of which were not detected in the tissue. Overall, 74 (73.3%) of the 101 patients had positive ctDNA NGS assay results.



 Although the NGS assay with ctDNA is less sensitive than that with tissue DNA, some genetic alterations are detected only in ctDNA NGS assays; therefore, it is useful to apply the ctDNA NGS assay as a complement.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8e0e6eed278b41bf977b6ec4d0bb9e829c01bb" target='_blank'>
              B-191 Genomic Profile of Circulating Tumor DNA in Advanced Lung Cancer: Comparison with Tissue DNA
              </a>
            </td>
          <td>
            J. Park, Y. Choi, S. Jeon, S. Kee, E. Na, M. Shin
          </td>
          <td>2024-10-01</td>
          <td>Clinical Chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3ca92a03051caca152f854fcb5c2f6b12d56f76" target='_blank'>
              Integrated genomic/epigenomic analysis stratifies subtypes of clear cell ovarian carcinoma, highlighting their cellular origin
              </a>
            </td>
          <td>
            Akira Nishijima, K. Oda, Kosei Hasegawa, Takahiro Koso, K. Asada, Yuji Ikeda, Ayumi Taguchi, Daichi Maeda, G. Nagae, S. Tsuji, K. Tatsuno, Y. Uehara, A. Kurosaki, S. Sato, M. Tanikawa, K. Sone, M. Mori, M. Ikemura, Keiichi Fujiwara, T. Ushiku, Yutaka Osuga, Hiroyuki Aburatani
          </td>
          <td>2024-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Formation of cytoplasmic inclusions (CIs) of TDP-43 and FUS, along with DNA damage accumulation, is a hallmark of affected motor neurons in Amyotrophic Lateral Sclerosis (ALS). However, the impact of CIs on DNA damage response (DDR) and repair in this pathology remains unprobed. Here, we show that CIs of TDP-43 and FUSP525L, co-localizing with stress granules, lead to a dysfunctional DDR activation associated with physical DNA breakage. Inhibition of the activity of the DDR kinase ATM, but not of ATR, abolishes DDR signaling, indicating that DNA double-strand breaks (DSBs) are the primary source of DDR activation. In addition, cells with TDP-43 and FUSP525L CIs exhibit reduced DNA damage-induced RNA synthesis at DSBs. We previously showed that the two endoribonucleases DROSHA and DICER, also known to interact with TDP-43 and FUS during small RNA processing, contribute to DDR signaling at DSBs. Treatment with enoxacin, which stimulates DDR and repair by boosting the enzymatic activity of DICER, restores a proficient DDR and reduces DNA damage accumulation in cultured cells with CIs and in vivo in a murine model of ALS. In Drosophila melanogaster, Dicer-2 overexpression rescues TDP-43-mediated retinal degeneration. In summary, our results indicate that the harmful effects caused by TDP-43 and FUS proteinopathies include genotoxic stress and that the pharmacological stimulation of the DNA damage signaling and repair counteracts it.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/671ea965eea696457f0c97b98eb2d7ef226c0fae" target='_blank'>
              DNA damage response defects induced by the formation of TDP-43 and mutant FUS cytoplasmic inclusions and their pharmacological rescue
              </a>
            </td>
          <td>
            Stefania Farina, Francesca Esposito, Stefania Modafferi, Ornella Brandi, M. Di Salvio, Ilaria Della Valle, Giada Cicio, Adelaide Riccardi, Federica Pisati, Anna Garbelli, Nadia D’Ambrosi, Mauro Cozzolino, G. Cestra, Fabrizio d’Adda di Fagagna, Ubaldo Gioia, S. Francia
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, Roberto Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9824378cb72ff7a6ac9e3450ad8fb3f2d8abaa3a" target='_blank'>
              Human genomic DNA is widely interspersed with i-motif structures.
              </a>
            </td>
          <td>
            C. D. Peña Martinez, Mahdi Zeraati, R. Rouet, Ohan Mazigi, Jake Y Henry, Brian Gloss, J. Kretzmann, Cameron W Evans, E. Ruggiero, Irene Zanin, Maja Marušič, J. Plavec, Sara N. Richter, Tracy M. Bryan, Nicole M Smith, Marcel E. Dinger, Sarah Kummerfeld, Daniel Christ
          </td>
          <td>2024-08-29</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="This review starts off with the first germline homozygous variants of the Nucleoporin 98 gene (NUP98) in siblings whose clinical presentation recalls Rothmund–Thomson (RTS) and Werner (WS) syndromes. The progeroid phenotype caused by a gene associated with haematological malignancies and neurodegenerative disorders primed the search for interplay between caretakers involved in genome instability syndromes and Nuclear Pore Complex (NPC) components. In the context of basic information on NPC architecture and functions, we discuss the studies on the interdependence of caretakers and gatekeepers in WS and Hereditary Fibrosing Poikiloderma (POIKTMP), both entering in differential diagnosis with RTS. In WS, the WRN/WRNIP complex interacts with nucleoporins of the Y-complex and NDC1 altering NPC architecture. In POIKTMP, the mutated FAM111B, recruited by the Y-complex’s SEC13 and NUP96, interacts with several Nups safeguarding NPC structure. The linkage of both defective caretakers to the NPC highlights the attempt to activate a repair hub at the nuclear periphery to restore the DNA damage. The two separate WS and POIKTMP syndromes are drawn close by the interaction of their damage sensors with the NPC and by the shared hallmark of short fragile telomeres disclosing a major role of both caretakers in telomere maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73f8c0edcf3a3bd5207393373e2885f587217e1c" target='_blank'>
              Interdependence between Nuclear Pore Gatekeepers and Genome Caretakers: Cues from Genome Instability Syndromes
              </a>
            </td>
          <td>
            Lidia Larizza, E. Colombo
          </td>
          <td>2024-08-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62cbbfa74d4b379f3ac75ff31b1584f4b3b436bc" target='_blank'>
              Heterochromatin formation and remodeling by IRTKS condensates counteract cellular senescence.
              </a>
            </td>
          <td>
            Jia Xie, Zhao-Ning Lu, Shi-Hao Bai, Xiao-Fang Cui, He-Yuan Lian, Chen-Yi Xie, Na Wang, Lan Wang, Ze-Guang Han
          </td>
          <td>2024-08-27</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Cancer is a complex disease in which several molecular and cellular pathways converge to foster the tumoral phenotype. Notably, in the latest iteration of the cancer hallmarks, "nonmutational epigenetic reprogramming" was newly added. However, epigenetics, much like genetics, is a broad scientific area that deserves further attention due to its multiple roles in cancer initiation, progression, and adaptive nature. Herein, we present a detailed examination of the epigenetic hallmarks affected in human cancer, elucidating the pathways and genes involved, and dissecting the disrupted landscapes for DNA methylation, histone modifications, and chromatin architecture that define the disease. Significance: Cancer is a disease characterized by constant evolution, spanning from its initial premalignant stages to the advanced invasive and disseminated stages. It is a pathology that is able to adapt and survive amidst hostile cellular microenvironments and diverse treatments implemented by medical professionals. The more fixed setup of the genetic structure cannot fully provide transformed cells with the tools to survive but the rapid and plastic nature of epigenetic changes is ready for the task. This review summarizes the epigenetic hallmarks that define the ecological success of cancer cells in our bodies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed379ec7953480a8811a5c6ea46e05bc377efc5" target='_blank'>
              The Epigenetic Hallmarks of Cancer.
              </a>
            </td>
          <td>
            M. Esteller, M. Dawson, Cigall Kadoch, F. Rassool, P. Jones, Stephen B Baylin
          </td>
          <td>2024-10-04</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="
 Atypical Teratoid Rhabdoid Tumor (ATRT) is a highly aggressive pediatric brain tumor and characterized by SMARCB1 deletion. SMARCB1 is a member of the SWI/SNF chromatin remodeling complex that is responsible for determining cellular pluripotency and lineage commitment. We performed an RNAi-based functional genomic screen to identify key epigenetic regulators that co-operate with SMARCB1 loss and contribute to tumorigenesis in ATRT. 406 genes associated with epigenetic regulation in patient derived ATRT cell lines were targeted. This screen identified BMI1, a component of the Polycomb Repressive Complex 1 (PRC1), as one of the top targets essential for ATRT cell growth. Using RNA-seq and ChIP-Seq analysis we demonstrated that BMI1 co-operates with SMARCB1 deletion to suppress transcription of pro-differentiation pathways and promote self-renewal of tumor stem cells. In SMARCB1 deficient cells BMI1 forms a partial PRC1 complex devoid of DNA binding components. We found that genetic inactivation of BMI1 suppresses ATRT cell clonogenicity in vitro and decreases S-phase length. By Doxycycline-inducible SMARCB1 re-expression system we revealed activation of proapoptotic proteins p16 and p21. With immunoprecipitation for BMI1, followed by mass spectrometry analysis we demonstrated that re-expression of SMARCB1 activates two PRC1 chromatin localizing components CBX4 and CBX8. Moreover, inhibition of BMI1 significantly prolongs mice survival and decreases tumor size as evaluated by IVIS imaging system and T2-MRI. Thus, we demonstrated that SMARCB1 deletion results in reprograming of BMI1 chromatin occupancy away from lineage specification by altering the components of the PRC1 complex and that PRC1/BMI1 inhibition is a novel therapeutic approach in ATRT.
 Citation Format: Irina Irina Alimova, Gillian Murdock, Angela M. Pierce, Dong Wang, Nathan Dahl, Sujatha Venkataraman, Rajeev Vibhakar. PRC1/BMI1 activity alters chromatin accessibility to regulate differentiation in atypical teratoid rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A080.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d643026696b6226900916830deb821d92b3f76f" target='_blank'>
              Abstract A080 PRC1/BMI1 activity alters chromatin accessibility to regulate differentiation in atypical teratoid rhabdoid tumors
              </a>
            </td>
          <td>
            Irina Irina Alimova, Gillian Murdock, Angela M Pierce, Dong Wang, N. Dahl, S. Venkataraman, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="
 Despite recent advances in the treatment of pancreatic adenocarcinoma (PDAC), the median survival remains <12 months. Patients typically present with late-stage disease and are often unable to tolerate drug combination regimens. Large-scale genomic approaches categorise PDAC into two distinct molecular subtypes, based on transcriptomic signatures, and these are termed the classical and squamous (or basal) subtypes. Although it is well-described that PDAC harbours gene mutations (i.e. KRAS G12D, p53 R172H) that initiate the disease, little is known about the influence of lineage-defining Transcription Factors (TF) on PDAC subtypes and their contribution to tumor progression. We took an unbiased approach to discover driving TFs in PDAC, we performed ChIP-seq of the active enhancer mark H3K27Ac from surplus tissue following resection of primary PDAC tumors (n=6) and normal adjacent tissue (n=4). Analysis of the H3K27Ac sites showed that the most enriched motifs within all experimentally mapped enhancer elements were HNF motifs (HNF4G & HNF4A) and Forkhead (i.e., FOXA1), confirming that these two classes of TFs constitute the lineage-defining factors in these clinical samples. Investigation of publicly available clinical datasets revealed HNF4G to be amplified in 9% of PDAC patients. Using Rapid Immunoprecipitation and Mass Spectrometry of Endogenous protein complexes (RIME), our method for unbiased discovery of endogenous protein complexes, we investigated the fundamental differences in the FOXA1 interactome between models of classical and squamous subtypes. HNF4A and HNF4G were only seen as FOXA1-interactors in models of the classical subtype. HNF4G Knockout (KO) clones injected orthotopically into the pancreas of mice, showed a significant increase (p-value<0.0006) in median survival and slower tumor growth compared to Control (Cas-9) (p-value <0.0001). HNF4G-KO dependent genes were specifically expressed in primary tumors (with decreased EMT signalling) versus metastatic samples (PanCRux dataset), implying that the HNF4G target genes were mostly limited to primary tumor contexts. We found Protein Arginine Methylate Transferase (PRMT1) to be a significant and reproducible interactor of FOXA1 and HNF4G. We observed a significant global reduction in PRMT1 chromatin binding in both the HNF4G-KO and FOXA1 depleted cells compared to the control. This data suggests a dependency on both HNF4G and FOXA1 for recruitment of PRMT1 to chromatin. We find HNF4G to be the key driver of classical subtype primary disease, in part by recruiting the methyltransferase PRMT1 to regulatory regions revealing a therapeutically exploitable opportunity. A molecular switch occurs in advanced disease, whereby HNF4G expression decreases, unmasking FOXA1’s transcriptional activity. De-repressed FOXA1 drives late-stage disease, by orchestrating metastasis-specific enhancer-promoter loops to direct metastatic gene regulation. We suggest the existence of stage-dependent TF activity, triggered by molecular compartmentalisation that mediates multistage progression of PDAC.
 Citation Format: Shalini Vasudev Rao, Lisa V Young, Danya Cheeseman, Stephanie V Mack, Dominique-Laurent Couturier, Sean V Flynn, Rebecca Brias, Amy Smith, Jill Temple, Duncan Jodrell, Alasdair Russell V Russell, Igor Chernukhin, Jason Carroll. Transcription factor switching drives progression of the classical subtype of pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C071.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e663bb2d35a48fcc777219947b4d5e9554b7ef19" target='_blank'>
              Abstract C071: Transcription factor switching drives progression of the classical subtype of pancreatic cancer
              </a>
            </td>
          <td>
            Shalini V Rao, Lisa Young, D. Cheeseman, Stephanie V Mack, Dominique-Laurent Couturier, Sean V Flynn, Rebecca Brias, Amy Smith, Jill Temple, Duncan Jodrell, Alasdair Russell V Russell, Igor Chernukhin, Jason Carroll
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Three-dimensional chromatin control of eukaryotic transcription is pivotal for regulating gene expression. This additional layer of epigenetic regulation is also utilized by DNA viruses, including herpesviruses. Dynamic, spatial genomic organization often involves looping of chromatin anchored by host-encoded CCCTC-binding factor (CTCF) and other factors, which control crosstalk between promoters and enhancers. Herein, we review the contribution of CTCF-mediated looping in regulating transcription during herpesvirus infection, with a specific focus on the betaherpesvirus, human cytomegalovirus (HCMV).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ca71344a757b4c351d6ffdff351925d0ab026" target='_blank'>
              Loopy virus or controlled contortionist? 3D regulation of HCMV gene expression by CTCF-driven chromatin interactions.
              </a>
            </td>
          <td>
            Ian J. Groves, Christine M. O’Connor
          </td>
          <td>2024-08-30</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="During meiosis, each chromosome pair experiences at least one crossover (CO), which directs their balanced segregation in addition to shuffling genetic information. COs tend to be away from each other, a phenomenon known as CO interference. The main biochemical pathway for CO formation, which is conserved in distant eukaryotes, involves the ZMM proteins together with the MLH1-MLH3 complex (MutLγ). Here, we aim to clarify the role of MutLγ in CO formation in Arabidopsis thaliana. We show that AtMutLγ is partially dispensable for ZMM-dependant CO formation. HEI10 large foci - that mark CO sites in wild-type-form at a normal level in mlh1 and mlh3 mutants, but are inefficiently maturated into COs. Mutating the MUS81 nuclease in either mlh1 or mlh3 leads to chromosome fragmentation, which is suppressed by further mutating the zmm msh5. This suggests that in the absence of MutLγ, recombination intermediates produced by ZMMs are resolved by MUS81, which does not ensure CO formation. Finally, CO interference is not affected in mlh1, which is compatible with a random sub-sampling of normally patterned CO sites. We conclude that AtMutLγ imposes designated recombination intermediates to be resolved exclusively as COs, supporting the view that MutLγ asymmetrically resolves double-Holliday junctions, yielding COs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/804b8cf2149074d9d9f514d468fba802009e7940" target='_blank'>
              MutLγ enforces meiotic crossovers in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Stéphanie Durand, Qichao Lian, Victor Solier, J. B. Fernandes, Raphael Mercier
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37e3c1ad704a586aa01f4f3735cd6a0421899426" target='_blank'>
              Rise and SINE: roles of transcription factors and retrotransposons in zygotic genome activation.
              </a>
            </td>
          <td>
            Pavel Kravchenko, Kikuë Tachibana
          </td>
          <td>2024-10-02</td>
          <td>Nature reviews. Molecular cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="The fundamental role of cells in safeguarding the genome's integrity against DNA double-strand breaks (DSBs) is crucial for maintaining chromatin homeostasis and the overall genomic stability. Aberrant responses to DNA damage, known as DNA damage responses (DDRs), can result in genomic instability and contribute significantly to tumorigenesis. Unraveling the intricate mechanisms underlying DDRs following severe damage holds the key to identify therapeutic targets for cancer. Chromatin lysine acylation, encompassing diverse modifications such as acetylation, lactylation, crotonylation, succinylation, malonylation, glutarylation, propionylation, and butyrylation, has been extensively studied in the context of DDRs and chromatin homeostasis. Here, we delve into the modifying enzymes and the pivotal roles of lysine acylation and their crosstalk in maintaining chromatin homeostasis and genome integrity in response to DDRs. Moreover, we offer a comprehensive perspective and overview of the latest insights, driven primarily by chromatin acylation modification and associated regulators.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/811404c503e46fe5b51a600e0d64881692754a49" target='_blank'>
              Chromatin lysine acylation: On the path to chromatin homeostasis and genome integrity.
              </a>
            </td>
          <td>
            Feng Chen, Xingkai He, Wenchao Xu, Linmin Zhou, Qi Liu, Weicheng Chen, Wei-Guo Zhu, Jun Zhang
          </td>
          <td>2024-08-18</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966f20b4115e34215d01f56c4a256d96a9d46af1" target='_blank'>
              Mosaic variegated aneuploidy in development, ageing and cancer.
              </a>
            </td>
          <td>
            Marcos Malumbres, Carolina Villarroya-Beltri
          </td>
          <td>2024-08-21</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b44100a40caef5d94745df5dd16e4a9a8f03f0a6" target='_blank'>
              Expression, DNA methylation pattern and transcription factor EPB41L3 in gastric cancer: a study of 262 cases
              </a>
            </td>
          <td>
            Mengdi Cai, Haonan Guo, Dong Wang, Tie Zhao, Xiao Liang, Jiaqi Li, XiaoBo Cui, Songbin Fu, Jingcui Yu
          </td>
          <td>2024-10-01</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Telomeres protect chromosome ends from DNA damage responses, and their dysfunction triggers genomic alterations like chromosome fusion and rearrangement, which can lead to cellular death. Certain cells, including specific cancer cells, adopt alternative lengthening of telomere (ALT) to counteract dysfunctional telomeres and proliferate indefinitely. While telomere instability and ALT activity are likely major sources of genomic alteration, the patterns and consequences of such changes at the nucleotide level in ALT cells remain unexplored. Here we generated haplotype-resolved genome assemblies for type I ALT mouse embryonic stem cells, facilitated by highly accurate or ultra-long reads and Hi-C reads. High-quality genome revealed ALT-specific complex chromosome end structures and various genomic alterations including over 1000 structural variants (SVs). The unique sequence (mTALT) used as a template for type I ALT telomeres showed traces of being recruited into the genome, with mTALT being replicated with remarkably high accuracy. Subtelomeric regions exhibited distinct characteristics: resistance to the accumulation of SVs and small variants. We genotyped SVs at allele resolution, identifying genes (Rgs6, Dpf3 and Tacc2) crucial for maintaining ALT telomere stability. Our genome assembly-based approach elucidated the unique characteristics of ALT genome, offering insights into the genome evolution of cells surviving telomere-derived crisis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d9657866100fc15991f3ce13ef281bd3238fb24" target='_blank'>
              Haplotype-resolved de novo assembly revealed unique characteristics of alternative lengthening of telomeres in mouse embryonic stem cells.
              </a>
            </td>
          <td>
            Hyunji Lee, Hiroyuki Niida, Sanghyun Sung, Junho Lee
          </td>
          <td>2024-10-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The large T antigen (LT) of the Merkel cell polyomavirus (MCPyV) is crucial for Merkel cell carcinoma (MCC), a rare but very aggressive form of neuroendocrine skin cancer. The clonal integration of MCPyV DNA into the host genome is a signature event of this malignancy. The resulting expression of oncogenes, including the small T (sT) antigen and a truncated form of the LT (truncLT), directly contribute to carcinogenesis. The truncation of the C-terminus of LT prevents the virus from replicating due to the loss of the origin binding domain (OBD) and the helicase domain. This precludes cytopathic effects that would lead to DNA damage and ultimately cell death. At the same time, the LxCxE motif in the N-terminus is retained, allowing truncLT to bind the retinoblastoma protein (pRb), a cellular tumor suppressor. The continuously inactivated pRb promotes cell proliferation and tumor development. truncLT exerts several classical functions of an oncogene: altering the host cell cycle, suppressing innate immune responses to viral DNA, causing immune escape, and shifting metabolism in favor of cancer cells. Given its central role in MCC, the LT is a major target for therapeutic interventions with novel approaches, such as immune checkpoint inhibition, T cell-based immunotherapy, and cancer vaccines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ba55aeb0f5e61fa39c51305144cbf085a84b1f2" target='_blank'>
              The Role of the Large T Antigen in the Molecular Pathogenesis of Merkel Cell Carcinoma
              </a>
            </td>
          <td>
            Julia Myrda, Franziska Bremm, N. Schaft, J. Dörrie
          </td>
          <td>2024-08-27</td>
          <td>Genes</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88cace8d561b58e28d3ee8aa2930fd2c3ef573b1" target='_blank'>
              Loss of EMI1 compromises chromosome stability and is associated with cellular transformation in colonic epithelial cell contexts.
              </a>
            </td>
          <td>
            Rubi Campos Gudiño, Nicole M. Neudorf, Demi Andromidas, Zelda Lichtensztejn, K. McManus
          </td>
          <td>2024-10-02</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Intracellular replication is crucial for the success of apicomplexan parasites, including Toxoplasma gondii. Therefore, essential players in parasite replication present potential targets for drug development. In this study, we have characterized TgGSK, a glycogen synthase kinase homolog that plays an important role in Toxoplasma endodyogeny. We have shown that TgGSK has a dynamic localization that is concurrent with the cell cycle. In non-dividing parasites, this kinase is highly concentrated in the nucleus. However, during division, TgGSK displays a cytosolic localization, with concentration foci at the centrosomes, a key organelle involved in parasite division, and the basal end. Conditional knockdown of TgGSK determined that it is essential for the completion of the lytic cycle and proper parasite division. Parasites lacking endogenous protein levels of TgGSK exhibited defects in division synchronicity and the segregation of the nucleus and apicoplast into forming daughter cells. These phenotypes are associated with defects in centrosome duplication and fission. Global phosphoproteomic analysis determined TgGSK-dependent phosphorylation of RNA-processing, basal end, and centrosome proteins. Consistent with the putative regulation of RNA-processing proteins, global transcriptomic analysis suggests that TgGSK is needed for proper splicing. Finally, we show that TgGSK interacts with GCN5b, a well-characterized acetyltransferase with roles in transcriptional control. Conversely, GCN5b chemical inhibition results in specific degradation of TgGSK. Thus, these studies reveal the involvement of TgGSK in various crucial processes, including endodyogeny and splicing, and identify acetylation as a possible mechanism by which this essential kinase is regulated. AUTHOR SUMMARY Toxoplasma gondii infects nearly a third of the world’s human population. While infection is largely asymptomatic in healthy adults, in immunocompromised or immunosuppressed individuals it can lead to brain lesions and even death. Similarly, toxoplasmosis can result in stillbirth, birth defects, and blindness of a developing fetus in the case of a congenital infection. With minimal treatments for Toxoplasmosis available, it is crucial to study parasite-specific processes that could be potential drug targets for the treatment of Toxoplasmosis. In this study, we investigated the protein TgGSK that is essential for parasite survival and proper division. We showed that TgGSK may perform its essential functions through interaction with the centrosome, an organelle that plays a major role in cell division in many organisms. We also show in this study a role for TgGSK in proper processing of messenger RNAs. Taken together, we have performed an in-depth study of the functional role of the essential protein TgGSK in Toxoplasma gondii. Importantly, TgGSK was shown to have more similarity to plant proteins than mammalian proteins which may allow for the possibility of targeting of this protein for therapeutic treatment of toxoplasmosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/062bbaa669e0646fc54f62727d07cc361f6a95c9" target='_blank'>
              THE ESSENTIAL KINASE TGGSK REGULATES CENTROSOME DIVISION AND ENDODYOGENY IN TOXOPLASMA GONDII
              </a>
            </td>
          <td>
            Amanda Krueger, Sofia Horjales, Chunlin Yang, William J. Blakely, M. Francia, G. Arrizabalaga
          </td>
          <td>2024-09-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Telomeres, crucial for chromosomal integrity, have been related to aging and cancer formation, mainly through regulating G-quadruplex structures. G-quadruplexes are structural motifs that can arise as secondary structures of nucleic acids, especially in guanine-rich DNA and RNA regions. Targeting these structures by small compounds shows promise in the selective suppression of cell growth, opening up novel possibilities for anticancer treatment. A comprehensive investigation of the many structural forms of G-quadruplex ligands is required to create ground-breaking anticancer drugs. Recent research into using specific benzimidazole molecules in stabilizing telomeric DNA into G-quadruplex structures has highlighted their ability to influence oncogene expression and demonstrate antiproliferative characteristics against cancer cells. This review describes the benzimidazole derivative, designed to enhance the stability of the G-quadruplex structure DNA to suppress the activity of telomerase enzyme, exhibiting promising potential for anticancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29f95ab2b6e7475ab4ddd44739a0c2d7e5a470ee" target='_blank'>
              Benzimidazole-based small molecules as anticancer agents targeting telomeric G-quadruplex and inhibiting telomerase enzyme.
              </a>
            </td>
          <td>
            Hemanathan Elango, Rabindra Nath Das, Abhijit Saha
          </td>
          <td>2024-09-24</td>
          <td>Future medicinal chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="BACKGROUND
Genome size is influenced by natural selection and genetic drift acting on variations from polyploidy and repetitive DNA sequences. We hypothesized that centromere drive, where centromeres compete for inclusion in the functional gamete during meiosis, may also affect genome and chromosome size. This competition occurs in asymmetric meiosis, where only one of the four meiotic products becomes a gamete. If centromere drive influences chromosome size evolution, it may also impact post-polyploid diploidization, where a polyploid genome is restructured to function more like a diploid through chromosomal rearrangements, including fusions. We tested if plant lineages with asymmetric meiosis exhibit faster chromosome size evolution compared to those with only symmetric meiosis, which lack centromere drive as all four meiotic products become gametes. We also examined if positive selection on centromeric histone H3 (CENH3), a protein that can suppress centromere drive, is more frequent in these asymmetric lineages.


METHODS
We analyzed plant groups with different meiotic modes: asymmetric in gymnosperms and angiosperms, and symmetric in bryophytes, lycophytes, and ferns. We selected species based on available CENH3 gene sequences and chromosome size data. Using Ornstein-Uhlenbeck evolutionary models and phylogenetic regressions, we assessed the rates of chromosome size evolution and the frequency of positive selection on CENH3 in these clades.


RESULTS
Our analyses showed that clades with asymmetric meiosis have a higher frequency of positive selection on CENH3 and increased rates of chromosome size evolution compared to symmetric clades.


CONCLUSIONS
Our findings support the hypothesis that centromere drive accelerates chromosome and genome size evolution, potenatially also influencing the process of post-polyploid diploidization. We propose a model which in a single famework helps explain the stability of chromosome size in symmetric lineages (bryophytes, lycophytes, and ferns) and its variability in asymmetric lineages (gymnosperms and angiosperms), providing a foundation for future research in plant genome evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fabc90ee2c5c9cff34a1bbfeb39ccf65f0a8155b" target='_blank'>
              Centromere drive may propel the evolution of chromosome and genome size in plants.
              </a>
            </td>
          <td>
            Klára Plačková, P. Bureš, M. Lysak, F. Zedek
          </td>
          <td>2024-08-28</td>
          <td>Annals of botany</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="CRISPR-Cas12a, an RNA-guided nuclease, has been repurposed for genome editing and molecular diagnostics due to its capability of cis-cleavage on target DNA and trans-cleavage on non-target single-strand DNA (ssDNA). However, the mechanisms underlying the activation of trans-cleavage activity of Cas12a, particularly in the context of split DNA activators, remain poorly understood. We elucidate the synergistic effect of these activators and introduce the concepts of induced targeting effect and exon-unwinding to describe the phenomenon. We demonstrate that upon binding of split DNA activators adjacent to the Protospacer Adjacent Motif (PAM) to the Cas12a ribonucleoprotein (Cas12a-RNP), a ternary complex form that can capture and interact with distal split DNA activators to achieve synergistic effects. Notably, if the distal activator is double-strand DNA (dsDNA), the complex initiates exon-unwinding, facilitating the RNA-guide sequence's access. Our findings provide a mechanistic insight into action of Cas12a and propose a model that could significantly advance our understanding of its function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21bfca41537975350cb4978fbebd5605ae4be926" target='_blank'>
              Synergistic effect of split DNA activators of Cas12a with exon-unwinding and induced targeting effect.
              </a>
            </td>
          <td>
            Shen Huang, Yongliang Lou, Laibao Zheng
          </td>
          <td>2024-09-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7300fe63a85fac8fe37bf10af42231e58f921f5" target='_blank'>
              Analysis of the role of CHPF in colorectal cancer tumorigenesis and immunotherapy based on bioinformatics and experiments
              </a>
            </td>
          <td>
            Qingyu Song, Pengchao Wang, Jingyu Wu, Ming Lu, Qingcheng Xia, Yexin Shi, Zijun Wang, Xiang Ma, Qinghong Zhao
          </td>
          <td>2024-09-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Study of early pancreas neoplasia in humans had previously been limited by lack of available tissue. Recent work by our group on deceased donor pancreata identified pancreatic intraepithelial neoplasia (PanIN) lesions in non-diseased organs and defined a transcriptomic signature for the microenvironment of these pre-cancerous lesions. Prevalence of PanINs in healthy human tissue was higher than expected and established a novel model to expand understanding of the complex biology of these precursor lesions. Epigenetic changes in chromatin structure and the subsequent effects on gene expression are essential processes in neoplasia that have yet to be described at the single cell level in the human pancreas. We hypothesized that previously defined gene signatures could identify acinar, ductal, and PanIN cells in donor pancreata based on chromatin accessibility profiling by single nuclear transposase-accessible chromatin preparation followed by high-throughput sequencing (snATAC-seq). Comparison between the accessibility patterns, transcription factor motifs, and differences in accessibility versus expression profiles can then be used to expand prior knowledge of early pancreatic neoplasia and identify novel targets for further study and therapy selection. To test this hypothesis, single nuclei from seventeen donor pancreata (twenty total samples) were isolated for snATAC-seq using the 10x Genomics platform. Data analysis was conducted using CellRanger, Seurat, Signac, AUCell, and other programs to identify and label chromatin peaks in individual nuclei corresponding to acinar, ductal, and PanIN origin according to gene signatures identified by scRNA-seq and special transcriptomics on matched donor pancreata. Using this method, cells corresponding to these gene signatures, including PanIN- like cells, were indeed identifiable with differentially accessible regions of the genome then analyzed for transcription factor motif enrichment between these cell types. Motifs enriched in cells identified as PanINs compared to either ductal or acinar cells included several transcription factors implicated in tumorigenesis. These results are similar to previously demonstrated essential factors in pancreatic neoplasia, suggesting this strategy is a relevant method for data analysis and discovery of novel factors implicated in tumorigenesis. Ongoing studies include validation of these findings in vitro using human pancreatic normal epithelial cells, human tumor cell lines, and organoid models derived from human normal and tumor samples. Together, this study demonstrates that PanINs can be identified in human pancreas tissue by snATAC-seq and that differentially expressed motifs can be identified, providing opportunity for further integration with transcriptomic information to elucidate detailed understanding of early neoplasia in the pancreas for future drug development and targeted therapeutics studies.
 Citation Format: Jamie N Mills, Aaron Dendekker, Joyce K Thompson, Hannah Watkoske, Simone Benitz, Padma Kadiyala, Ahmed Elhossiny, Howard Crawford, Eileen Carpenter, Marina Pasca di Magliano, Filip Bednar. Characterization of chromatin accessibility patterns in the human pancreas and early pancreatic neoplastic lesions using snATAC-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C056.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eee7faf599c3d04827a3bdb6a5acb4e73be0ca2f" target='_blank'>
              Abstract C056: Characterization of chromatin accessibility patterns in the human pancreas and early pancreatic neoplastic lesions using snATAC-seq
              </a>
            </td>
          <td>
            Jamie N. Mills, Aaron Dendekker, Joyce K Thompson, Hannah R Watkoske, S. Benitz, Padma Kadiyala, Ahmed M. Elhossiny, Howard C. Crawford, Eileen S Carpenter, M. Pasca di Magliano, Filip Bednar
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background The role of focal amplifications and extrachromosomal circular DNA (eccDNA) is still uncertain in prostate adenocarcinoma (PRAD). Here, we first mapped the global characterizations of eccDNA and then investigate the characterization of eccDNA-amplified key differentially expressed encoded genes (eKDEGs) in the progression, immune response and immunotherapy of PRAD. Methods Circular_seq was used in conjunction with the TCGA-PRAD transcriptome dataset to sequence, annotate, and filter for eccDNA-amplified differentially expressed coding genes (eDEGs) in PRAD and para-cancerous normal prostate tissues. Afterwards, risk models were created and eKDEGs linked to the PRAD prognosis were identified using Cox and Lasso regression analysis. The immune microenvironment of the risk model was quantified using a variety of immunological algorithms, which also identified its characteristics with regard to immunotherapy, immune response, and immune infiltration. Results In this research, there was no significant difference in the size, type, and chromosomal distribution of eccDNA in PRAD and para-cancerous normal prostate tissues. However, 4,290 differentially expressed eccDNAs were identified and 1,981 coding genes were amplified. Following that, 499 eDEGs were tested in conjunction with the transcriptome dataset from TCGA-PRAD. By using Cox and Lasso regression techniques, ZNF330 and PITPNM3 were identified as eKDEGs of PRAD, and a new PRAD risk model was conducted based on this. Survival analysis showed that the high-risk group of this model was associated with poor prognosis and validated in external data. Immune infiltration analysis showed that the model risks affected immune cell infiltration in PRAD, not only mediating changes in immune cell function, but also correlating with immunophenotyping. Furthermore, the high-risk group was negatively associated with anti-CTLA-4/anti-PD-1 response and mutational burden. In addition, Tumor Immune Dysfunction and Exclusion analyses showed that high-risk group was more prone to immune escape. Drug sensitivity analyses identified 10 drugs, which were instructive for PRAD treatment. Conclusion ZNF330 and PITPNM are the eKDEGs for PRAD, which can be used as potential new prognostic markers. The two-factor combined risk model can effectively assess the survival and prognosis of PRAD patients, but also can predict the different responses of immunotherapy to PRAD patients, which may provide new ideas for PRAD immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189d5f00b52f7644f54771a980ef61f81d31dae9" target='_blank'>
              Extrachromosomal circular DNAs in prostate adenocarcinoma: global characterizations and a novel prediction model
              </a>
            </td>
          <td>
            Qingliu He, Qingfu Su, Chengcheng Wei, Pu Zhang, Weihui Liu, Junyi Chen, Xiaoping Su, Zhuang Wei
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Motivation Cancer is driven by somatic mutations that result in the expansion of genomically distinct sub-populations of cells called clones. Identifying the clonal composition of tumours and understanding the evolutionary relationships between clones is a crucial task in cancer genomics. Bulk DNA sequencing is commonly used for studying the clonal composition of tumours, but it is challenging to infer the genetic relationship between different clones due to the mixture of different cell populations. Results In this work, we introduce a new probabilistic model called PhyClone that can infer clonal phylogenies from bulk sequencing data. We demonstrate the performance of PhyClone on simulated and real-world datasets and show that it outperforms previous methods in terms of accuracy and scalability. Availability and implementation Source code is available on Github at: https://github.com/Roth-Lab/PhyClone under the GPL v3.0 license. Contact aroth@bccrc.ca">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2cba6bef4ca455a79d82a594e5b713c434402a" target='_blank'>
              PhyClone: Accurate Bayesian reconstruction of cancer phylogenies from bulk sequencing
              </a>
            </td>
          <td>
            Emilia Hurtado, A. Bouchard-Côté, Andrew Roth
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Accumulating evidence indicates that G‐quadruplexes (G4s) are involved in transcriptional regulation. Previous studies have demonstrated that DHX36 preferentially resolves G4s, suggesting its potential impact on gene transcription mediated by these structures. However, systematic validation is required to establish a link between DHX36 activity and its roles in transcriptional regulation. In this study, we investigate the role of DHX36 in transcription. First, we employ the cleavage under targets and tagmentation (CUT&Tag), an efficient method for mapping protein–DNA interactions, to identify the binding sites in the chromatin of MCF‐7 cells. Subsequently, we use the auxin‐inducible degron (AID) protein degradation system and improved nascent RNA sequencing method acrylonitrile‐mediated uridine‐to‐cytidine conversion sequencing (AMUC‐seq) to pinpoint genes directly regulated by DHX36. Our results reveal a significant enrichment of G4 structures at DHX36 target sites, predominantly located in active genomic regions. In vitro assays further demonstrate DHX36's interaction with G4 sequences from three specific oncogenes. These findings underscore the potential role of DHX36 in modulating gene transcription through G4 structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07f9bb5fca73acfa9f0b936e9e6d763af42a8898" target='_blank'>
              Rapid degradation of DHX36 revealing its transcriptional role by interacting with G‐quadruplex
              </a>
            </td>
          <td>
            Ziang Lu, Jinglei Xu, Yuqi Chen, Yuanyuan Zhou, Xiaolu Zhou, Qi Wang, Qi Wei, Shaoqing Han, Ruiqi Zhao, X. Weng, Xiaolian Zhang, Xiang Zhou
          </td>
          <td>2024-08-19</td>
          <td>Aggregate</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Precise regulation of gene expression is essential for proper development and the maintenance of homeostasis in organisms. Studies have shown that some transcriptional regulatory proteins influence gene expression through the formation of dynamic, locally concentrated assemblies known as condensates, while dysregulation of transcriptional condensates was associated with several cancers, such as Ewing sarcoma and AML [Wang Y et al. (2023) Nat Chem Biol 19, 1223-1234; Chandra B et al. (2022) Cancer Discov 12, 1152-1169]. Mutations in the histone acetylation "reader" eleven-nineteen-leukemia (ENL) have been shown to form discrete condensates at endogenous genomic targets, but it remains unclear how ENL mutations drive tumorigenesis and whether it is correlated with their condensate formation property. Liu et al. now show, using a conditional knock-in mouse model, that ENL YEATS domain mutation is a bona fide oncogenic driver for AML. This mutant ENL forms condensates in hematopoietic stem/progenitor cells at the genomic loci of key leukemogenic genes, including Meis1 and Hoxa cluster genes, and disrupting condensate formation via mutagenesis impairs its chromatin and oncogenic function. Furthermore, they show that small-molecule inhibition of the acetyl-binding activity displaces ENL mutant condensates from oncogenic target loci, and this inhibitor significantly impairs the onset and progression of AML driven by mutant ENL in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9e0f2605685a3812a479056f634ccfed5efb36" target='_blank'>
              ENL mutation and AML: a new model that reveals oncogenic condensate's function in leukemogenesis.
              </a>
            </td>
          <td>
            Zhong Fan, Yanan Jiang, Xiaotian Zhang
          </td>
          <td>2024-09-26</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oxidative damage triggers micronuclear membrane rupture and defective repair Errors in chromosome segregation during cell division (mitosis) can lead to chromosomal instability, a hallmark of human cancer. These mitotic errors give rise to the formation of micronuclei—small cytoplasmic structures that are spatially separated from the primary nucleus and contain lagging chromosomes or chromosome fragments. Unlike the primary nucleus of a cell, micronuclei frequently undergo irreparable rupture and collapse. This breakdown releases micronuclear DNA into the cytosol, causing DNA damage, chromosomal rearrangements, and genomic instability, thereby contributing to cancer progression. The rupture of micronuclei also activates a component of the innate immune system called the cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING) pathway, leading to tumorrelated inflammation and metastasis (1). On pages 951 and 952 of this issue, Di Bona et al. (2) and Martin et al. (3), respectively, report the molecular underpinnings of micronuclear collapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f3bfa9de0e1bf705d54ddc01f0ac3869ca5a791" target='_blank'>
              Micronuclear collapse mechanisms in cancer
              </a>
            </td>
          <td>
            Marianna Maddaluno, C. Settembre
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The therapeutic efficacy of the anticancer drug cisplatin is limited by acquired drug resistance. Cisplatin forms DNA crosslinks, that, if not removed, lead to replication stress. Due to this, the DNA damage response (DDR) gets activated regulating cell cycle arrest, DNA repair, cell death or survival. This makes DDR components promising targets for the development of new therapeutic approaches aiming to overcome acquired drug resistance. To this end, cisplatin-resistant bladder cancer cells were analyzed regarding their sensitivity to combination treatments with selected pharmacological DDR inhibitors. Synergistic cytolethal effects were achieved after combined treatment with low to moderate doses of the non-genotoxic RAD51-inhibitor (RAD51i) B02 and CHK1-inhibitor (CHK1i) PF477736. This effect was also found in cisplatin resistant tumor cells of other origin as well as with other RAD51i and CHK1i. Combined treatments promoted decelerated replication, S-phase blockage, accumulation of DNA strand breaks, DDR activation and stimulation of apoptotic cell death as compared to mono-treatment, which is independent of the expression of RAD51, CHK1, and PrimPol. Based on these data, we suggest combined inhibition of RAD51 and CHK1 to overcome acquired cisplatin resistance of malignant cells. We propose that the molecular mechanism of this synergistic toxicity relies on a simultaneous inactivation of two key DNA damage tolerance pathways regulating replication fork restart, thereby circumventing the activation of alternative compensatory mechanisms and, in consequence, eventually effectively triggering apoptotic cell death by replication fork collapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebd957adf6dd81535bc9cd542c1c1aa09cefb079" target='_blank'>
              Combined inhibition of RAD51 and CHK1 causes synergistic toxicity in cisplatin resistant cancer cells by triggering replication fork collapse.
              </a>
            </td>
          <td>
            Julia Mann, Kathrin Niedermayer, Johannes Krautstrunk, Lena Abbey, Lisa Wiesmüller, Roland P. Piekorz, Gerhard Fritz
          </td>
          <td>2024-09-06</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Objective: Structural variations (SVs) constitute the greatest source of genomic alterations leading to the oncogenesis of many cancers, including the deadliest pancreatic cancer (PC). However, SVs in PC remain largely undefined due to technological limitations of the traditional short-read sequencing. This study aims to establish a comprehensive signature of genomic SVs in PC. Methods: PacBio whole genome long-read sequencing was employed to investigate the genomic landscape of SVs in eight tissues from four pancreatic cancer patients. A variety of SVs callers, including pbsv, Sniffles, cuteSV, and svim, were comprehensively adopted to improve the accuracy of SVs detection. Many undefined characteristics of germline and somatic SVs were revealed by comparing with multiple previously published databases. Besides, the impacts of structural variations on 3D chromatin organization were analyzed by integrating multi-omics data including Hi-C, RNA seq and ChIP seq. Results: A total of 26,844 and 27,028 non-redundant SVs were identified in human pancreatic cancer and matched para-cancerous tissues, respectively. Notably, among these SVs, 23.97% were novel and had not been previously reported. Germline SVs were further characterized and found enriched in 25 well-known susceptible genes, such as TP63, PVT1, and ABO. Additionally, we proposed a panel of germline SVs with predicted highest pathogenicity, involving STOX1, DSPP, and WRN. Totally, 616 somatic SVs were identified and presented high burdens in patients with lymph node metastasis. We observed that somatic DELs and INSs were significantly more prevalent in repetitive regions than in non-repetitive regions. Interestingly, DELs showed a decrease in the proportion occurring in simple repeat regions but an increase in the proportion occurring in LINE and SINE regions compared to INSs. Moreover, it was observed that deletions within topologically associated domains were associated with local enhanced interactions and disrupted chromatin loops, which might influence H3K27ac modification of histone contributing to enhancer or silencer hijacking. Conclusion: This study provides insights into the genomic signature of SVs in human pancreatic cancer. These findings might shed new light on the complexity of SVs and their pathogenesis by interplaying with chromatin organization.
 Citation Format: Yongxing Du, Xiaohao Zheng, Yunjie Duan, Chengfeng Wang. Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A083.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40a9fb18df1c1c8a73e8c29f87767e5466baa02" target='_blank'>
              Abstract A083: Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing
              </a>
            </td>
          <td>
            Yongxing Du, Xiaohao Zheng, Yunjie Duan, Cheng-Guang Wang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Zhao-Ru Dong, Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, Yong-Heng Yang, Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bbe642fe1a5320aff996d83dd54d13a05e4037" target='_blank'>
              The intersection of homologous recombination (HR) and mismatch repair (MMR) pathways in DNA repair-defective tumors
              </a>
            </td>
          <td>
            L. Incorvaia, T. B. Bazan Russo, V. Gristina, A. Perez, C. Brando, Clarissa Mujacic, Emilia Di Giovanni, M. Bono, Silvia Contino, Carla Ferrante Bannera, Maria Concetta Vitale, Andrea Gottardo, M. Peri, A. Galvano, D. Fanale, G. Badalamenti, Antonio Russo, V. Bazan
          </td>
          <td>2024-09-05</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feda6005d00312b29429cd211524a37b760d02e2" target='_blank'>
              Pan-cancer analysis of the immunological and oncogenic roles of ATAD2 with verification in papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Zhecheng Li, Yaxin Zhao, Peng Huang, Zhaoyi Wu, D. Ouyang, AdolphusOsei Nyarko, Lei Ai, Zhejia Zhang, Shi Chang
          </td>
          <td>2024-09-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/962a15086155c288de21a400a171d7f9e76960cc" target='_blank'>
              Specific multivalent molecules boost CRISPR-mediated transcriptional activation
              </a>
            </td>
          <td>
            Rui Chen, Xinyao Shi, Xiangrui Yao, Tong Gao, Guangyu Huang, Duo Ning, Zemin Cao, Youxin Xu, Weizheng Liang, S. Tian, Qionghua Zhu, Liang Fang, Meizhen Zheng, Yuhui Hu, Huanhuan Cui, Wei Chen
          </td>
          <td>2024-08-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b69c52796ad88ee763ab51cb8f0791d90f0f5161" target='_blank'>
              Antagonism between H3K27me3 and genome-lamina association drives atypical spatial genome organization in the totipotent embryo.
              </a>
            </td>
          <td>
            Isabel Guerreiro, Franka J. Rang, Yumiko K Kawamura, Carla Kroon-Veenboer, Jeroen Korving, Femke C Groenveld, Ramada E van Beek, Silke J. A. Lochs, Ellen Boele, A. H. Peters, Jop Kind
          </td>
          <td>2024-09-16</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) is an aggressive primary intestinal malignancy with the third-highest incidence and second-highest mortality among all cancer types worldwide. Transcription factors (TFs) regulate cell development and differentiation owing to their ability to recognize specific DNA sequences upstream of genes. Numerous studies have demonstrated a strong correlation between TFs, the etiology of tumors, and therapeutic approaches. Here, we aimed to explore prognosis-related TFs and comprehend their carcinogenic mechanisms, thereby offering novel insights into the diagnosis and management of CRC. Materials and Methods Differentially expressed TFs between CRC and normal tissues were identified leveraging The Cancer Genome Atlas database, Weighted correlation network analysis and Cox regression analysis were performed to identify prognosis-related TFs. The cellular functions of hub TF zinc finger E-box binding homeobox 1 (ZEB1) were determined using by 5-ethynyl-2′-deoxyuridine and cell invasion assays in CRC cells. RNA-sequencing, Kyoto Encyclopedia of Genes and Genomes enrichment, and gene set enrichment analyses were used to identify the cellular processes in which ZEB1 participates. Immunoaffinity purification, silver staining mass spectrometry, and a chromatin immunoprecipitation assay were conducted to search for proteins that might interact with ZEB1 and the target genes they jointly regulate. Results Thirteen central TFs related to prognosis were identified through bioinformatics analysis techniques. Among these TFs, ZEB1 emerged as the TF most closely associated with CRC, as determined through a combination of regulatory network diagrams, survival curves, and phenotype analyses. ZEB1 promotes CRC cell growth by recruiting the NuRD(MTA1) complex, and the ZEB1/NuRD(MTA1) complex transcriptionally represses glycolysis-associated tumor suppressor genes. Conclusion Our study not only identified a hub biomarker related to CRC prognosis but also revealed the specific molecular mechanisms through which ZEB1 affects cancer progression. These insights provide crucial evidence for the diagnosis of CRC and potential treatment opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/951b486f89f4fbb493f66d9c80785b0eee594498" target='_blank'>
              Transcription factor ZEB1 coordinating with NuRD complex to promote oncogenesis through glycolysis in colorectal cancer
              </a>
            </td>
          <td>
            T. Gao, Xinhui Hao, Jing-yao Zhang, Miaomiao Huo, Ting Hu, Tianyu Ma, Hefen Yu, X. Teng, Yong Wang, Yunkai Yang, Wei Huang, Yan Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="DNA in eukaryotic genomes is under constant assault from both exogenous and endogenous sources, leading to DNA damage, which is considered a major molecular driver of ageing. Fortunately, the genome and the central exome are safeguarded against these attacks by abundant peripheral non-coding DNA. Non-coding DNA codes for small non-coding RNAs that inactivate foreign nucleic acids in the cytoplasm and physically blocks these attacks in the nucleus. Damage to non-coding DNA produced during such blockage is removed in the form of extrachromosomal circular DNA (eccDNA) through nucleic pore complexes. Consequently, non-coding DNA serves as a line of defence for the exome against DNA damage. The total amount of non-coding DNA/heterochromatin declines with age, resulting in a decrease in both physical blockage and eccDNA exclusion, and thus an increase in the accumulation of DNA damage in the nucleus during ageing and in age-related diseases. Here, we summarize recent evidence supporting a protective role of non-coding DNA in healthy and pathological states and argue that DNA damage is the proximate cause of ageing and age-related genetic diseases. Strategies aimed at strengthening the protective role of non-coding DNA/heterochromatin could potentially offer better systematic protection for the dynamic genome and the exome against diverse assaults, reduce the burden of DNA damage to the exome, and thus slow ageing, counteract age-related genetic diseases and promote a healthier life for individuals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a7879deb4075aa031b3e715005da950caf3fffb2" target='_blank'>
              Protection of the genome and the central exome by peripheral non-coding DNA against DNA damage in health, ageing and age-related diseases.
              </a>
            </td>
          <td>
            Guo-Hua Qiu, Mingjun Fu, Xintian Zheng, Cuiqin Huang
          </td>
          <td>2024-09-26</td>
          <td>Biological reviews of the Cambridge Philosophical Society</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The cell cycle-regulated DNA methyltransferase CcrM is conserved in most Alphaproteobacteria, but its role in bacteria with complex or multicentric genomes remains unexplored. Here, we compare the methylome, the transcriptome and the phenotypes of wild-type and CcrM-depleted Agrobacterium tumefaciens cells with a dicentric chromosome with two essential replication origins. We find that DNA methylation has a pleiotropic impact on motility, biofilm formation and viability. Remarkably, CcrM promotes the expression of the repABCCh2 operon, encoding proteins required for replication initiation/partitioning at ori2, and represses gcrA, encoding a conserved global cell cycle regulator. Imaging ori1 and ori2 in live cells, we show that replication from ori2 is often delayed in cells with a hypo-methylated genome, while ori2 over-initiates in cells with a hyper-methylated genome. Further analyses show that GcrA promotes the expression of the RepCCh2 initiator, most likely through the repression of a RepECh2 anti-sense RNA. Altogether, we propose that replication at ori1 leads to a transient hemi-methylation and activation of the gcrA promoter, allowing repCCh2 activation by GcrA and contributing to initiation at ori2. This study then uncovers a novel and original connection between CcrM-dependent DNA methylation, a conserved epigenetic regulator and genome maintenance in an Alphaproteobacterial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad339078add660717cbec9308e097c1844ce59b8" target='_blank'>
              DNA methylation by CcrM contributes to genome maintenance in the Agrobacterium tumefaciens plant pathogen.
              </a>
            </td>
          <td>
            Sandra Martin, Florian Fournes, Giovanna Ambrosini, Christian Iseli, Karolina Bojkowska, Julien Marquis, Nicolas Guex, Justine Collier
          </td>
          <td>2024-09-04</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The cell cycle is a fundamental process essential for cell proliferation, differentiation, and development. It consists of four major phases: G1, S, G2, and M. These phases collectively drive the reproductive cycle and are meticulously regulated by various proteins that play critical roles in both the prevention and progression of cancer. Traditional methods for studying these functions, such as flow cytometry, require a substantial number of cells to ensure accuracy. In this study, we have developed a user-friendly, immunofluorescence-based method for identifying cell cycle stages, providing single-cell resolution and precise identification of G1, early S, late S, early G2, late G2, and each sub-stage of the M phase using fluorescence microscopy. This method provides high-precision cell cycle identification and can serve as an alternative to, or in combination with, traditional flow cytometry to dissect detailed substages of the cell cycle in a variety of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6a0cb1f79ef1a3760e18e969fa1e9d5248adeb4" target='_blank'>
              ImmunoCellCycle-ID: A high-precision immunofluorescence-based method for cell cycle identification
              </a>
            </td>
          <td>
            Yu-Lin Chen, Yu-Chia Chen, Aussie Suzuki
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by fluctuations in chromosome number or structure within cells, stands out as a hallmark of cancer, enabling tumors to thrive in hostile conditions. CIN serves as a driver of genetic diversity, giving rise to clonal heterogeneity (CH). Emerging evidence points to a potential correlation between CIN, CH, and the prognosis of breast cancer (BC) patients, especially in tumors exhibiting overexpression of the human epidermal growth factor receptor 2 (HER2+). However, our understanding of the role of CIN in other subtypes of BC is limited. Furthermore, it remains unclear whether CIN levels above a certain threshold in BC tumors could adversely affect tumor growth, or if lower to moderate levels of CIN might be associated with a more favorable prognosis for BC patients compared to elevated levels. Elucidating these relationships could significantly influence risk assessment and the formulation of future therapeutic approaches targeting CIN in BC. This study aimed to assess CIN and CH in tumor tissue samples obtained from Colombian patients diagnosed with luminal A, luminal B, HER2+, or triple-negative BC, and compare them with established clinicopathological parameters. The findings of this study indicate that BC patients, exhibit intermediate CIN, high CH, and stable aneuploidy. All these characteristics were found to be related with clinicopathological features. Our results suggest that the identification of CIN, CH and aneuploidy could improve cancer risk stratification, which could help to clarify the prediction of clinical outcomes and guiding personalized therapeutic strategies for patients with different BC subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23f4c53715c6bdbb1cc8b85a27c36e138c73e29" target='_blank'>
              Exploring chromosomal instability and clonal heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            Giovanny Castellanos, Laura Valentina Camargo-Herrera, N. Rangel, G. A. Jiménez-Tobón, María Martínez-Agüero, Milena Rondón-Lagos
          </td>
          <td>2024-09-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de9b939e6665a85e15c06d7919e4ac22bd18c3f" target='_blank'>
              Cellular origin and clonal evolution of human dedifferentiated liposarcoma
              </a>
            </td>
          <td>
            N. Gruel, C. Quignot, Laëtitia Lesage, Sophie El Zein, S. Bonvalot, Dimitri Tzanis, Khadija Ait Rais, Fabien Quinquis, Bastien Manciot, J. Vibert, Nadine El Tannir, A. Dahmani, Héloise Derrien, Didier Decaudin, Ivan Bièche, L. Courtois, O. Mariani, Laetitia K Linares, L. Gayte, Sylvain Baulande, J. Waterfall, O. Delattre, G. Pierron, Sarah Watson
          </td>
          <td>2024-09-12</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="Humans are exposed to DNA alkylating agents through endogenous metabolism, environmental exposure and cancer chemotherapy. The resulting alkylated DNA adducts may elicit genome instability by perturbing DNA replication and transcription. R-loops regulate various cellular processes, including transcription, DNA repair, and telomere maintenance. However, unscheduled R-loops are also recognized as potential sources of DNA damage and genome instability. In this study, by employing fluorescence microscopy and R-loop sequencing approaches, we uncovered, for the first time, that minor-groove N2-alkyl-dG lesions elicit elevated R-loop accumulation in chromatin and in plasmid DNA in cells. We also demonstrated that the N2-alkyl-dG-induced R-loops impede transcription elongation and compromise genome integrity. Moreover, genetic depletion of DDX23, a R-loop helicase, renders cells more sensitive toward benzo[a]pyrene diolepoxide, a carcinogen that induces mainly the minor-groove N2-dG adduct. Together, our work unveiled that unrepaired minor-groove N2-alkyl-dG lesions may perturb genome integrity through augmenting R-loop levels in chromatin. Our findings suggest a potential therapeutic strategy involving the combination of R-loop helicase inhibitors with DNA alkylating drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17bca69f7ad1277a7093d1fb762d63dd7f27acf5" target='_blank'>
              N 2-Alkyl-dG lesions elicit R-loop accumulation in the genome.
              </a>
            </td>
          <td>
            Yinan Wang, Feng Tang, Ting Zhao, Jun Yuan, Andrew H. Kellum, Yinsheng Wang
          </td>
          <td>2024-10-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eaaf9c2f335952f249d3df58f150d64ce452ea5c" target='_blank'>
              CircXPO6 promotes breast cancer progression through competitively inhibiting the ubiquitination degradation of c-Myc.
              </a>
            </td>
          <td>
            Aiqi Xu, Xi Li, Qiaoting Cai, Ciqiu Yang, Mei Yang, Hongfei Gao, M. Cheng, Xianzhe Chen, F. Ji, Hailin Tang, Kun Wang
          </td>
          <td>2024-08-23</td>
          <td>Molecular and cellular biochemistry</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Amino-terminal (Nt-) acetylation (NTA) is a common protein modification, affecting 80% of cytosolic proteins in humans. The human essential gene, NAA10, encodes the enzyme NAA10, as the catalytic subunit for the N-terminal acetyltransferase A (NatA) complex, including the accessory protein, NAA15. The first human disease directly involving NAA10 was discovered in 2011, and it was named Ogden syndrome (OS), after the location of the first affected family residing in Ogden, Utah, USA. Since that time, other variants have been found in NAA10 and NAA15. Here we describe the generation of 31 iPSC lines, with 16 from females and 15 from males. This cohort includes CRISPR-mediated correction to the wild-type genotype in 4 male lines, along with editing one female line to generate homozygous wild-type or mutant clones. Following the monoclonalizaiton and screening for X-chromosome activation status in female lines, 3 additional pairs of female lines, in which either the wild type allele is on the active X chromosome (Xa) or the pathogenic variant allele is on Xa, have been generated. Subsets of this cohort have been successfully used to make cardiomyocytes and neural progenitor cells (NPCs). These cell lines are made available to the community via the NYSCF Repository.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eab1adf8656f9aef233d0cf25ef9cb89566ccecd" target='_blank'>
              A repository of Ogden syndrome patient derived iPSC lines and isogenic pairs by X-chromosome screening and genome-editing
              </a>
            </td>
          <td>
            Josephine Wesely, Tom Rusielewicz, Yu-Ren Chen, Brigham Hartley, Dayna McKenzie, Matthew K. Yim, Colin Maguire, Ryan Bia, Sarah Franklin, Rikhil Makwana, E. Marchi, Manali Nikte, Soha Patil, Maria Sapar, Dorota Moroziewicz, Lauren Bauer, Jeannie T. Lee, Frederick J. Monsma, Dan Paull, G. Lyon
          </td>
          <td>2024-09-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb6a4f203d0ffda30fcc3f0dfa56bd3b405a484" target='_blank'>
              Amplified centrosomes-more than just a threat.
              </a>
            </td>
          <td>
            Eva Kiermaier, Isabel Stötzel, Marina A Schapfl, A. Villunger
          </td>
          <td>2024-09-16</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Human tumor samples hold a plethora of information about their microenvironment and immune repertoire. Effective dissociation of human tissue samples into viable cell suspensions is a required input for the single-cell RNA sequencing (scRNAseq) pipeline. Unlike bulk RNA sequencing approaches, scRNAseq enables us to infer the transcriptional heterogeneity in tumor specimens at the single-cell level. Incorporating this approach in recent years has led to many discoveries, such as identifying immune and tumor cellular states and programs associated with clinical responses to immunotherapies and other types of treatments. Moreover, single-cell technologies applied to dissociated tissues can be used to identify accessible chromatin regions T and B cell receptor repertoire, and the expression of proteins, using DNA barcoded antibodies (CITEseq). The viability and quality of the dissociated sample are critical variables when using these technologies, as these can dramatically affect the cross-contamination of single cells with ambient RNA, the quality of the data, and interpretation. Moreover, long dissociation protocols can lead to the elimination of sensitive cell populations and the upregulation of a stress response gene signature. To overcome these limitations, we devised a rapid universal dissociation protocol, which has been validated on multiple types of human and murine tumors. The process begins with mechanical and enzymatic dissociation, followed by filtration, red blood lysis, and live dead enrichment, suitable for samples with a low input of cells (e.g., needle core biopsies). This protocol ensures a clean and viable single-cell suspension paramount to the successful generation of Gel Bead-In Emulsions (GEMs), barcoding, and sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe2078d45af3e9f2f3826e3ee2e17b4140e6605" target='_blank'>
              Dissociation of Human and Mouse Tumor Tissue Samples for Single-cell RNA Sequencing.
              </a>
            </td>
          <td>
            Jacy Fang, Isabella Salinas, Sarah San Vicente, Caroline Zielinski, M. Sade-Feldman
          </td>
          <td>2024-08-16</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="X chromosome inactivation (XCI) is an epigenetic process that results in the transcriptional silencing of one X chromosome in the somatic cells of females. This phenomenon is common to both eutherian and marsupial mammals, but there are fundamental differences. In eutherians, the X chosen for silencing is random. DNA methylation on the eutherian inactive X is high at transcription start sites (TSSs) and their flanking regions, resulting in universally high DNA methylation. This contrasts XCI in marsupials where the paternally derived X is always silenced, and in which DNA methylation is low at TSSs and flanking regions. Here, we examined the DNA methylation status of the tammar wallaby X chromosome during spermatogenesis to determine the DNA methylation profile of the paternal X prior to and at fertilization. Whole genome enzymatic methylation sequencing was carried out on enriched flow-sorted populations of premeiotic, meiotic, and postmeiotic cells. We observed that the X displayed a pattern of DNA methylation from spermatogonia to mature sperm that reflected the inactive X in female somatic tissue. Therefore, the paternal X chromosome arrives at the egg with a DNA methylation profile that reflects the transcriptionally silent X in adult female somatic tissue. We present this epigenetic signature as a candidate for the long sought-after imprint for paternal XCI in marsupials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82e12408d360f28982d362b3fdf85cb94edc8ed7" target='_blank'>
              Imprinted X chromosome inactivation in marsupials: The paternal X arrives at the egg with a silent DNA methylation profile.
              </a>
            </td>
          <td>
            A. M. Milton, Laia Marín-Gual, Nicholas C. Lister, Kim L. McIntyre, Patrick G. S. Grady, Melanie K Laird, Donna M Bond, T. Hore, Rachel J. O’Neill, A. Pask, Marilyn B. Renfree, Aurora Ruiz-Herrera, Paul D. Waters
          </td>
          <td>2024-08-27</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Background Radiotherapy (RT) is the primary treatment for diffuse midline glioma (DMG), a lethal pediatric malignancy defined by histone H3 lysine 27-to-methionine (H3K27M) mutation. Based on the loss of H3K27 trimethylation producing broad epigenomic alterations, we hypothesized that H3K27M causes a functional double-strand break (DSB) repair defect that could be leveraged therapeutically with PARP inhibitor and RT for selective radiosensitization and antitumor immune responses. Methods H3K27M isogenic DMG cells and orthotopic brainstem DMG tumors in immune deficient and syngeneic, immune competent mice were used to evaluate the efficacy and mechanisms of PARP1/2 inhibition by olaparib or PARP1 inhibition by AZD9574 with concurrent RT. Results H3K27M mutation caused an HRR defect characterized by impaired RT-induced K63-linked polyubiquitination of histone H1 and inhibition of HRR protein recruitment. H3K27M DMG cells were selectively radiosensitized by olaparib in comparison to isogenic controls, and this effect translated to efficacy in H3K27M orthotopic brainstem tumors. Olaparib and RT induced an innate immune response and induction of NK cell (NKG2D) activating ligands leading to increased NK cell-mediated lysis of DMG tumor cells. In immunocompetent syngeneic orthotopic DMG tumors, either olaparib or AZD9574 in combination with RT enhanced intratumoral NK cell infiltration and activity in association with NK cell-mediated therapeutic responses and favorable activity of AZD9574. Conclusions The HRR deficiency in H3K27M DMG can be therapeutically leveraged with PARP inhibitors to radiosensitize and induce an NK cell-mediated antitumor immune response selectively in H3K27M DMG, supporting the clinical investigation of best-in-class PARP inhibitors with RT in DMG patients. Key points H3K27M DMG are HRR defective and selectively radiosensitized by PARP inhibitor. PARP inhibitor with RT enhances NKG2D ligand expression and NK cell-mediated lysis. NK cells are required for the therapeutic efficacy of PARP inhibitor and RT. Importance of the Study Radiotherapy is the cornerstone of H3K27M-mutant diffuse midline glioma treatment, but almost all patients succumb to tumor recurrence with poor overall survival, underscoring the need for RT-based precision combination therapy. Here, we reveal HRR deficiency as an H3K27M-mediated vulnerability and identify a novel mechanism linking impaired RT-induced histone H1 polyubiquitination and the subsequent RNF168/BRCA1/RAD51 recruitment in H3K27M DMG. This model is supported by selective radiosensitization of H3K27M DMG by PARP inhibitor. Notably, the combination treatment results in NKG2D ligand expression that confers susceptibility to NK cell killing in H3K27M DMG. We also show that the novel brain penetrant, PARP1-selective inhibitor AZD9574 compares favorably to olaparib when combined with RT, prolonging survival in a syngeneic orthotopic model of H3K27M DMG. This study highlights the ability of PARP1 inhibition to radiosensitize and induce an NK cell-mediated antitumor immunity in H3K27M DMG and supports future clinical investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa409894637aca4cc204fe5ba28199f9da5c66db" target='_blank'>
              H3K27M diffuse midline glioma is homologous recombination defective and sensitized to radiotherapy and NK cell-mediated antitumor immunity by PARP inhibition
              </a>
            </td>
          <td>
            Yupei Guo, Zian Li, L. A. Parsels, Zhuwen Wang, J. Parsels, Anushka Dalvi, Stephanie The, Nan Hu, Victoria M. Valvo, Robert Doherty, Erik Peterson, Xinjun Wang, S. Venkataraman, S. Agnihotri, S. Venneti, Daniel R. Wahl, Michael D. Green, Theodore S. Lawrence, C. Koschmann, Meredith A. Morgan, Qiang Zhang
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Deleterious accumulation of R-loops, a DNA-RNA hybrid structure, contributes to genome instability. They are associated with BRCA1 mutation-related breast cancer, an estrogen receptor α negative (ERα-) tumor type originating from luminal progenitor cells. However, a presumed causality of R-loops in tumorigenesis has not been established in vivo. Here, we overexpress mouse Rnaseh1 (Rh1-OE) in vivo to remove accumulated R-loops in Brca1-deficient mouse mammary epithelium (BKO). R-loop removal exacerbates DNA replication stress in proliferating BKO mammary epithelial cells, with little effect on homology-directed repair of double-strand breaks following ionizing radiation. Compared to their BKO counterparts, BKO-Rh1-OE mammary glands contain fewer luminal progenitor cells but more mature luminal cells. Despite a similar incidence of spontaneous mammary tumors in BKO and BKO-Rh1-OE mice, a significant percentage of BKO-Rh1-OE tumors express ERα and progesterone receptor. Our results suggest that rather than directly elevating the overall tumor incidence, R-loops influence the mammary tumor subtype by shaping the cell of origin for Brca1 tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd59964db49fa8717ee1e81f17faaff9786150ba" target='_blank'>
              R-loop functions in Brca1-associated mammary tumorigenesis.
              </a>
            </td>
          <td>
            Huai-Chin Chiang, Leilei Qi, Payal Mitra, Yimeng Huang, Yanfen Hu, Rong Li
          </td>
          <td>2024-08-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c115fcb82a4ae08aa16b7db64204e01871cc9b8b" target='_blank'>
              Prognostic value of nucleotide excision repair and translesion DNA synthesis proteins in muscle-infiltrating bladder carcinoma
              </a>
            </td>
          <td>
            P. Palacka, Andrea Holíčková, J. Roška, Peter Makovický, M. Vallová, Csaba Biró, Eveline Órásová, J. Obertová, J. Mardiak, Thomas A Ward, Karol Kajo, M. Chovanec
          </td>
          <td>2024-09-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="During tumor development, promoter CpG islands that are normally silenced by Polycomb repressive complexes (PRCs) become DNA-hypermethylated. The molecular mechanism by which de novo DNA methyltransferase(s) [DNMT(s)] catalyze CpG methylation at PRC-regulated regions remains unclear. Here, we report a cryo–electron microscopy structure of the DNMT3A long isoform (DNMT3A1) amino-terminal region in complex with a nucleosome carrying PRC1-mediated histone H2A lysine-119 monoubiquitination (H2AK119Ub). We identify regions within the DNMT3A1 amino terminus that bind H2AK119Ub and the nucleosome acidic patch. This bidentate interaction is required for effective DNMT3A1 engagement with H2AK119Ub-modified chromatin in cells. Further, aberrant redistribution of DNMT3A1 to Polycomb target genes recapitulates the cancer-associated DNA hypermethylation signature and inhibits their transcriptional activation during cell differentiation. This effect is rescued by disruption of the DNMT3A1–acidic patch interaction. Together, our analyses reveal a binding interface critical for mediating promoter CpG island DNA hypermethylation, a major molecular hallmark of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f64e00b184c0e797697cd862ef32ddd61be317" target='_blank'>
              Cancer-associated DNA hypermethylation of Polycomb targets requires DNMT3A dual recognition of histone H2AK119 ubiquitination and the nucleosome acidic patch
              </a>
            </td>
          <td>
            Kristjan H. Gretarsson, S. Abini-Agbomson, Susan L Gloor, Daniel N. Weinberg, Jamie L McCuiston, V. S. Kumary, Allison Hickman, Varun Sahu, Rachel Lee, Xinjing Xu, Natalie Lipieta, Samuel Flashner, Oluwatobi A Adeleke, Irina K Popova, Hailey F Taylor, Kelsey Noll, C. L. Windham, Danielle N Maryanski, Bryan J Venters, Hiroshi Nakagawa, Michael-Christopher Keogh, K. Armache, Chao Lu
          </td>
          <td>2024-08-28</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ABSTRACT Background Prostate cancer (PCa) is among the three main types of cancer. Although prostate-specific antigen (PSA) is routinely tested, it has disadvantages, such as poor prognostic ability. Therefore, finding more PCa markers and therapeutic targets remains a subject of study. CircRNAs have been found to have regulatory roles in various diseases, such as diabetes, Central Nervous System (CNS) neuropathy, etc. where their application in cancer is even more valuable. Therefore, this paper aims to search for differentially expressed circRNAs in PCa and find downstream targeting pathways related to autophagy. Method By detecting the expression of circRNA in the samples, hsa_circ_0119816 was finally identified as the research target. The properties of circRNA were verified by RNase R, actinomycin D, and fluorescence in situ hybridization (FISH). The downstream target miRNAs and target proteins were predicted by an online database, and the targeting relationship was verified using dual luciferase and RNA Immunoprecipitation. The effects of circRNAs and their downstream signalling pathways on prostate cancer cell proliferation, migration, EMT and autophagy were examined by CCK-8, Transwell, immunofluorescence and Western blotting. Results CircBIRC6 is highly expressed in prostate cancer samples. Knockdown of its expression inhibits cell proliferation, invasion, EMT and autophagy and promotes apoptosis. CircBIRC6/miRNA-574-5p/DNAJB1 is a molecular axis that regulates prostate cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9d1bad6dbf3ace4247e9abe87ef37ab3c3b685" target='_blank'>
              CircBIRC6 affects prostate cancer progression by regulating miR-574-5p and DNAJB1
              </a>
            </td>
          <td>
            Bin Zhao, Jinye Yang, Fengming Ran, Yuanlong Shi, Libo Yang, Yuanpeng Duan, Zhiyu Shi, Xin Li, Jianpeng Zhang, Zhiyao Li, Jiansong Wang
          </td>
          <td>2024-09-11</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df67f1c440809cb586a740b578467c0fb41b2a5" target='_blank'>
              Epigenetic therapy potentiates transposable element transcription to create tumor-enriched antigens in glioblastoma cells.
              </a>
            </td>
          <td>
            H. Jang, Nakul M. Shah, Ju Heon Maeng, Yonghao Liang, Noah L. Basri, Jiaxin Ge, Xuan Qu, T. Mahlokozera, Shin-Cheng Tzeng, Russell B Williams, Michael J Moore, Devi Annamalai, Justin Y Chen, H. Lee, Patrick A DeSouza, Daofeng Li, Xiaoyun Xing, Albert H. Kim, Ting Wang
          </td>
          <td>2024-09-02</td>
          <td>Nature genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="The incidence of colorectal cancer (CRC) among individuals under age 50, or early-onset CRC (EOCRC), has been rising over the past few decades for unclear reasons, and the etiology of the disease remains largely unknown. Known genetic risk factors do not explain this increase, pointing to possible environmental and as-yet unidentified genetic contributors and their interactions. Previous research linked genetic variation on chromosome 6 to increased CRC risk. This region harbors multiple immune genes, including the gene encoding Major Histocompatibility Complex (MHC) class I polypeptide-related sequence A (MICA). MICA is a polygenic ligand for the Natural Killer Group 2D receptor (NKG2D), a receptor expressed on Natural Killer (NK) cells and other lymphocytes. Given that intra-tumoral NK cell infiltration correlates with favorable CRC outcomes, we hypothesized that germline genetic variation in MICA could influence CRC risk. In a discovery set of 40,125 cases and controls, we show that the minor G allele at Chr6:31373718C>G (hg19) is associated with increased risk for CRC (odds ratio (OR) = 1.09, 95% confidence interval (CI) 1.04 - 1.15, p = 0.0009). The effect is stronger in EOCRC (OR = 1.26, 95% CI 1.08 - 1.44, p = 0.0023) than in those 50 and over (OR = 1.07, 95% CI 1.02 - 1.13; p = 0.012) (Ratio of ORs = 1.32, 95% CI 1.14 - 1.52, p = 0.0002). In an independent validation set of 77,983 cases and controls, the adjusted interaction by age -of-onset was significant at OR = 1.15 (95% CI 1.03 - 1.34, p = 0.0150) with a higher risk in EOCRC. Expression quantitative trait locus analysis in normal colonic epithelia showed that MICA RNA expression decreases linearly with each additional copy of the minor G allele (p = 3.345 x 10-18). Bulk RNA analysis of the tumor immune microenvironment revealed that tumors from patients with CG or GG genotypes have lower resting and activated NK cell infiltration as compared to tumors from patients with CC genotype. Multiplex immunofluorescence analysis demonstrated that patients with a G allele (i.e. CG or GG genotype, but not CC genotype) have a statistically significant decrease in the number of NK cells in tumor compared to adjacent normal colonic mucosa. Taken together, population-based epidemiologic, molecular, genetic, cellular and immunologic evidence demonstrate that MICA genotype is associated with increased risk of EOCRC and reduced number of NK cells in colorectal tumors, suggesting that patients with a G allele have altered NK cell-mediated immunosurveillance. These novel findings suggest that EOCRC may have a previously unrecognized innate immune-mediated etiology which merits further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a5c740e57eda493d70527270b12cf7926d01624" target='_blank'>
              Genetic Variation and Regulation of MICA Alters Natural Killer Cell-Mediated Immunosurveillance in Early-Onset Colorectal Cancer
              </a>
            </td>
          <td>
            Heather M. McGee, J. Bonner, Yubo Fu, Oscar Colunga Flores, S. Lindsey, Lawrence Shaktah, Ferran Moratalla-Navarro, Yasmin Kamal, Kevin K. Tsang, , G. Idos, K. McDonnell, H. Rennert, Elizabeth L Barry, Hermann Brenner, Daniel D. Buchanan, P. Campbell, , J. Chang-Claude, Jane C Figueiredo, M. Gago-Domínguez, M. Hoffmeister, Li Hsu, J. Huyghe, M. Jenkins, L. Marchand, Heinz-Josef Lenz, Li Li, A. Lindblom, Yun Ru Liu, B. Lynch, Christina C. Newton, K. Offit, Shuji Ogino, Rebeca Sanz Pamplona, Andrew J. Pellatt, Paul D. P. Pharoah, Amanda I. Phipps, Lorena Reynaga, Allyson S. Templeton, C. Um, Alicja Wolk, M. Woods, A. H. Wu, Yen Yun, Wei Zheng, , David V Conti, Ulrike Peters, F. Lejbkowicz, J. Greenson, S. Schmit, W. Gauderman, Stanley R. Hamilton, Victor Moreno, Gadi Rennert, Stephen B. Gruber, †Gad Rennert, B. Gruber, MD M. McGee, MD Stephen B. Gruber
          </td>
          <td>2024-09-26</td>
          <td>None</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Human DNA methylation profiling offers great promises in various biomedical applications, including ageing, cancer and even forensics. So far, most DNA methylation techniques are based on a chemical process called sodium bisulfite conversion, which specifically converts non-methylated cytosines into uracils. However, despite the popularity of this approach, it is known to cause DNA fragmentation and loss affecting standardization, while incomplete conversion may result in potential misinterpretation of methylation-based outcomes. To offer the community a solution, we developed qBiCo - a novel quality-control method to address the quantity and quality of bisulfite-converted DNA. qBiCo is a 5-plex, TaqMan® probe-based, quantitative (q)PCR assay that amplifies single- and multi-copy DNA fragments of converted and non-converted nature. It estimates four parameters: converted DNA concentration, fragmentation, global conversion efficiency, and potential PCR inhibition. We optimized qBiCo using synthetic DNA standards and assessed it using standard developmental validation criteria, showcasing that qBiCo is reliable, robust and sensitive down to picogram level. We also evaluated its performance by testing decreasing DNA amounts using several commercial bisulfite conversion kits. Depending on the starting DNA quantity, bisulfite-converted DNA recoveries ranged from 8.5 – 100 %, conversion efficiencies from 78 - 99.9 %, while certain kits highly fragment DNA, demonstrating large variability in their performance. Towards building a prototype tool, we further optimized key functionalities, for example, by replacing the poorest performing single-plex assay and creating a more representative DNA standard. Aiming to scale-up and move towards implementation, we successfully transferred and validated our novel method in six different qPCR platforms from different major manufacturers. Overall, with the present study, we offer researchers in the epigenetic field a novel long-awaited QC tool that for the first time allows them to measure key quality and quantity parameters of the most popular DNA conversion process. The tool also enables standardization to prevent inconsistent data and false outcomes in the future, regardless of the downstream experimental analysis of DNA methylation-based research and applications across different fields of biology and biomedicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5f2d2f1ff7d9e8b8418047a193bf39b8fe5d70" target='_blank'>
              qBiCo: A method to assess global DNA conversion performance in epigenetics via single-copy genes and repetitive elements
              </a>
            </td>
          <td>
            Faidra Karkala, Roy B. Simons, Floor Claessens, V. Kalamara, M. Kayser, A. Vidaki
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Forkhead box A1 (FOXA1) acts as a pioneer factor for Estrogen Receptor alpha (ERα), playing a crucial role in determining where ERα binds to DNA and regulates gene expression in breast cancer cells. A pioneer factor is a class of transcription factors (TF) that is responsible for opening the chromatin to allow other TFs to attach and activate transcription of that gene. FOXA1 acts as a facilitator for ERα-DNA interactions. It binds to specific DNA regions, opening the chromatin for ERα to bind. It “pre-marks” these sites in the genome even in the absence of estrogen. In fact, FOXA1 is frequently overexpressed in ER+ breast cancers, suggesting it has a crucial role in driving tumor growth. As such, FOXA1 is an attractive therapeutic target that already serves as a great biomarker for many types of breast cancers. Inhibiting FOXA1 can potentially block ER activity and inhibit growth of ER+ breast cancer, especially in those resistant to anti-estrogen therapies, such as tamoxifen. In this project, we identified sites that FOXA1 opens and correlated them with SNP mutations commonly found in the Hispanic population. The presence of these SNPs portends a worse outcome in Hispanic patients. To assess which open chromatin regions are crucial for breast cancer pathogenesis, a ChIP-seq analysis was done on vehicle-treated and estrogen-treated samples to determine FOXA1 binding sites. After identifying gained, lost, and differentially expressed peaks set, these were compared to genome-wide association studies (GWAS) of Hispanic patients to develop an understanding of how much overlap is common between sites of mutations and open sections of chromatin due to FOXA1. Through our analysis, we uncovered a subset of FOXA1 binding sites that overlap with sites of inherited mutation in Hispanic patients. This informs on the subset of genes that are linked to breast cancer and potential therapeutic targets.
 Citation Format: Hemakshat Sharma, Alyssa Maria Arreola, Frances L. Heredia, Hector L. Franco, Julie Dutil. FOXA1 as a pioneer factor in ERα-DNA interactions and its therapeutic potential in ER+ breast cancer: Insights from ChIP-seq and GWAS in Hispanic populations [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr B011.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e221203c9b5991c2cb9afa79cdfe37b8baf68cb3" target='_blank'>
              Abstract B011: FOXA1 as a pioneer factor in ERα-DNA interactions and its therapeutic potential in ER+ breast cancer: Insights from ChIP-seq and GWAS in Hispanic populations
              </a>
            </td>
          <td>
            Hemakshat Sharma, Alyssa Maria Arreola, Frances L. Heredia, Hector L. Franco, Julie Dutil
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Glioblastoma is the most common primary malignant brain tumour. Despite decades of intensive research in the disease, its prognosis remains poor, with an average survival of only 14 months after diagnosis. The remarkable level of intra- and interpatient heterogeneity is certainly contributing to the lack of progress in tackling this tumour. Epigenetic dysregulation plays an important role in glioblastoma biology and significantly contributes to intratumour heterogeneity. However, it is becoming increasingly clear that it also contributes to intertumour heterogeneity, which historically had mainly been linked to diverse genetic events occurring in different patients. In this review, we explore how DNA methylation, chromatin remodelling, microRNA (miRNA) dysregulation, and long noncoding RNA (lncRNA) alterations contribute to intertumour heterogeneity in glioblastoma, including its implications for advanced tumour stratification, which is the essential first step for developing more effective patient-specific therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8a905205728cf89ec2231b745c2dd76f674ecb4" target='_blank'>
              Unravelling the mosaic: Epigenetic diversity in glioblastoma.
              </a>
            </td>
          <td>
            Sara Lucchini, Myrianni Constantinou, Silvia Marino
          </td>
          <td>2024-08-15</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Pediatric acute lymphoblastic leukemia (ALL) is a highly heterogeneous disease at the molecular level, with over 20 subtypes of B-cell ALL (B-ALL) identified to date. These subtypes are critical for guiding risk-adapted therapy and precision medicine, as they are defined by specific chromosomal rearrangements, gene expression profiles, aneuploidies and point mutations. However, the ability to accurately classify these subtypes is limited by the technical and economic challenges faced by many centers, particularly in low- and middle-income countries. In this study, we employed transcriptome sequencing to perform molecular classification of B-ALL at diagnosis in pediatric patients enrolled in the multicentric ALLIC-GATLA-2010 clinical protocol in Argentina. Using a combination of bioinformatic tools, we successfully identified single nucleotide variants, fusion transcripts and gene expression profiles, achieving molecular classification in over 90% of patients. Our analysis also revealed high-risk molecular features and novel genetic alterations. These findings hold potential clinical value for improving risk stratification and identifying therapeutic targets, particularly for patients who remain unclassified by conventional diagnostic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f39fa251d291498250d854c37a9028d4e85f79" target='_blank'>
              Molecular Characterization of Pediatric Acute Lymphoblastic Leukemia via Integrative Transcriptomics: A Multicenter Study in Argentina
              </a>
            </td>
          <td>
            Sol Ruiz, María Mercedes Abbate, Ezequiel Sosa, Daniel Avendaño, Ignacio Gomez Mercado, María Laura Lacreu, Mariano Riccheri, Virginia Schuttenberg, Luis Aversa, E. Vazquez, G. Gueron, J. Cotignola
          </td>
          <td>2024-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Simple Summary Childhood cancer survivors treated with radiotherapy face the likelihood of long-term complications, including mutations. In order to assess the long-term effect of radiotherapy on the capacity of cells to repair their DNA double-strand breaks (DSBs), five lambs received medulloblastoma radiotherapy to the thecal sac, with three lambs serving as controls. Four years later, lung biopsies were taken and fibroblast cells were amplified and re-irradiated. The cells from the previously treated sheep showed a significant impairment of their DNA DSB repair mechanism, highlighting a potential increase in radiosensitivity. Our results show that previous irradiation can impair the DNA DSB repair mechanism of ovine lung fibroblasts. Abstract Children with cancer previously treated with radiotherapy face the likelihood of side effects that can be debilitating or fatal. This study aimed to assess the long-term effect of medulloblastoma radiotherapy on the DNA double-strand break (DSB) repair capability of primary fibroblasts derived from lung biopsies of previously irradiated young sheep. This study included biopsies from three control and five irradiated sheep. The treated sheep had previously received spinal radiotherapy at a total dose of 28 Gy, which is equivalent to pediatric medulloblastoma treatment. Lung biopsies were taken 4 years post-irradiation from high-dose (HD, >18 Gy) and low-dose (LD, <2 Gy) regions. Fifteen cell lines were extracted (six control, four LD and five HD). The cells were irradiated, and DNA DSB repair was analyzed by immunofluorescence. Clonogenic, trypan blue and micronuclei assays were performed. Both the HD and LD cell lines had a significantly higher number of residual γH2AX foci 24 h and a significant decrease in pATM activity post-irradiation compared to the control. There was no statistically significant difference in the clonogenic assay, trypan blue and micronuclei results. Our study showed that a previous irradiation can impair the DNA DSB repair mechanism of ovine lung fibroblasts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5ea88bd1bedea4941e4772289fe7a7d48fca439" target='_blank'>
              Impaired DNA Double-Strand Break Repair in Irradiated Sheep Lung Fibroblasts: Late Effects of Previous Irradiation of the Spinal Thecal Sac
              </a>
            </td>
          <td>
            B. Youssef, C. Feghaly, Joelle Al Choboq, J. Bou-Gharios, R. Challita, J. Azzi, H. Bou Hadir, F. Abi Antoun, T. Araji, Phillip J. Taddei, F. Geara, Pierre Sfeir, Abdo Jurjus, W. Abou-Kheir, Larry Bodgi
          </td>
          <td>2024-08-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Several hundred billion cells die daily in the human body, releasing cell-free chromatin particles (cfChPs) in the process, which then enter the blood circulation and get taken up by healthy cells. We have previously reported that, these internalized cfChPs damage genomic DNA as well cause physical damage to mitochondria, resulting in increased mitochondrial ROS production. In the current study, we evaluated the potential damaging effects of the cfChP-induced increase in ROS production on genomic DNA. NIH3T3 mouse fibroblast cells were treated with cfChPs isolated from the sera of healthy individuals (H-cfChPs) or patients with cancer (C-cfChPs) in the presence or absence of the ROS scavenger Mito-TEMPO. The pre-incubation of cfChP-treated cells with Mito-TEMPO abolished ROS production, but did not prevent genomic DNA damage induced by H-cfChPs and C-cfChPs. Our results suggest that cfChP-induced genomic DNA damage occurs in an ROS-independent manner. These findings align with emerging evidence suggesting that mitochondrial ROS may not be a direct cause of genomic DNA damage and suggest that DNA damage attributed to ROS may in-fact be induced by cfChPs. This possibility opens up new therapeutic approaches involving deactivation of cfChPs to retard ageing and other degenerative conditions traditionally attributed to oxidative stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c2f89cdf5be913b593ba8f17e11056fcb407617" target='_blank'>
              Cell-free chromatin particles damage genomic DNA of healthy cells via an ROS-independent mechanism
              </a>
            </td>
          <td>
            Karishma Jain, G. V. Raghuram, Relestina Lopes, Naveen Kumar Khare, S. Shabrish, Indraneel Mittra
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Mitochondrial DNA (mtDNA) stability, essential for cellular energy production, relies on DNA polymerase gamma (POLγ). Here, we show that the POLγ Y951N disease causing mutation induces replication stalling and severe mtDNA depletion. However, unlike other POLγ disease causing mutations, Y951N does not directly impair exonuclease activity and only mildly affects polymerase activity. Instead, we found that Y951N compromises the enzyme’s ability to efficiently toggle between DNA synthesis and degradation, and is thus the first patient-derived mutation with impaired polymerase-exonuclease switching. These findings provide new insights into the intramolecular switch when POLγ proofreads the newly-synthesized DNA strand, and reveal a new mechanism for causing mitochondrial DNA instability. Significance Statement DNA polymerase gamma (POLγ) is essential for copying mitochondrial DNA (mtDNA), which is crucial for our energy production. POLγ must accurately switch between making new DNA (polymerase activity) and correcting errors (exonuclease activity). While it is known that mutations in POLγ can cause mitochondrial diseases by directly impairing these enzymatic functions, this study reveals a new mechanism. The Y951N mutation disrupts POLγ’s ability to switch between these activities, leading to severe blockages in DNA replication and a loss of mtDNA in human cells, even without significant direct impairment of polymerase or exonuclease activities. These findings provide new insights into the origins of mitochondrial diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0242e3e1606aa26c741b2c57aa12acc8d793383b" target='_blank'>
              The Y951N patient mutation inactivates the intramolecular switch in human mitochondrial DNA POLγ
              </a>
            </td>
          <td>
            Josefin M. E. Forslund, Tran V.H. Nguyen, V. Parkash, A. Berner, S. Goffart, J. Pohjoismäki, Paulina H. Wanrooij, Erik Johansson, Sjoerd Wanrooij
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="TP53 is a tumor suppressor gene that is mutated in most cancer types and has been extensively studied in cancer research. p53 plays a critical role in regulating the expression of target genes and is involved in key processes such as apoptosis, cell cycle regulation, and genomic stability, earning it the title “guardian of the genome.” Numerous studies have demonstrated p53’s influence on and regulation of autophagy, ferroptosis, the tumor microenvironment, and cell metabolism, all of which contribute to tumor suppression. Alterations in p53, specifically mutant p53 (mutp53), not only impair its tumor-suppressing functions but also enhance oncogenic characteristics. Recent data indicate that mutp53 is strongly associated with poor prognosis and advanced cancers, making it an ideal target for the development of novel cancer therapies. This review summarizes the post-translational modifications of p53, the mechanisms of mutp53 accumulation, and its gain-of-function, based on previous findings. Additionally, this review discusses its impact on metabolic homeostasis, ferroptosis, genomic instability, the tumor microenvironment, and cancer stem cells, and highlights recent advancements in mutp53 research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea389a27a85074a44af76e077628c890d00a2c8" target='_blank'>
              Roles of Mutant TP53 Gene in Cancer Development and Progression
              </a>
            </td>
          <td>
            Muhammad Abubakar, Baqaur Rehman
          </td>
          <td>2024-09-25</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcaa5e009963f12c7b3cfac1a62813bb1ec1e2e7" target='_blank'>
              Removal of TREX1 activity enhances CRISPR-Cas9-mediated homologous recombination.
              </a>
            </td>
          <td>
            Mehmet E. Karasu, Eléonore Toufektchan, Yanyang Chen, Alessandra Albertelli, G. Cullot, J. Maciejowski, Jacob E. Corn
          </td>
          <td>2024-08-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Ganciclovir (GCV) is a clinically important drug as it is used to treat viral infections. GCV is incorporated into the DNA during replication, where it interferes with subsequent replication on GCV-incorporated templates. However, the effects of GCV on the host genome and the mechanisms underlying cellular tolerance to GCV remain unclear. In this study, we explored these mechanisms using a collection of mutant DT40 cells. We identified RAD17/-, BRCA1-/-, and RAD18-/- cells as highly GCV-sensitive. RAD17, a component of the alternative checkpoint-clamp loader RAD17-RFC, was required for the activation of the intra-S checkpoint following GCV treatment. BRCA1, a critical factor for promoting homologous recombination (HR), was required for suppressing DNA double-strand breaks (DSBs). Moreover, RAD18, an E3-ligase involved in DNA repair, was critical in suppressing the aberrant ligation of broken chromosomes caused by GCV. We found that BRCA1 suppresses DSBs through HR-mediated repair and template switching (TS)-mediated damage bypass. Moreover, the strong GCV sensitivity of BRCA1-/- cells was rescued by the loss of 53BP1, despite the only partial restoration in the sister chromatid exchange events which are hallmarks of HR. These results indicate that BRCA1 promotes cellular tolerance to GCV through two mechanisms, TS and HR-mediated repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb90975f7c5cd2ebcb118dd3d319f550cae4f8b1" target='_blank'>
              RAD18- and BRCA1-dependent pathways promote cellular tolerance to the nucleoside analog ganciclovir.
              </a>
            </td>
          <td>
            Tasnim Ahmad, Ryotaro Kawasumi, Kouji Hirota
          </td>
          <td>2024-08-22</td>
          <td>Genes to cells : devoted to molecular & cellular mechanisms</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Telomeres, the protective structures at the ends of chromosomes, are crucial for maintaining cellular longevity and genome stability. Their proper function depends on tightly regulated processes of replication, elongation, and damage response. The shelterin complex, especially Telomere Repeat-binding Factor 1 (TRF1) and TRF2, plays a pivotal role in telomere protection and has emerged as a potential anti-cancer target for drug discovery. These proteins bind to the repetitive telomeric DNA motif TTAGGG, facilitating the formation of protective structures and recruitment of other telomeric proteins. Structural methods and advanced imaging techniques have provided insights into telomeric protein-DNA interactions, but probing the dynamic processes requires single-molecule approaches. Tools like magnetic tweezers, optical tweezers, and atomic force microscopy (AFM) have been employed to study telomeric protein-DNA interactions, revealing important details such as TRF2-dependent DNA distortion and telomerase catalysis. However, the preparation of single-molecule constructs with telomeric repetitive motifs continues to be a challenging task, potentially limiting the breadth of studies utilizing single-molecule mechanical methods. To address this, we developed a method to study interactions using full-length human telomeric DNA with magnetic tweezers. This protocol describes how to express and purify TRF2, prepare telomeric DNA, set up single-molecule mechanical assays, and analyze data. This detailed guide will benefit researchers in telomere biology and telomere-targeted drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f2fc75b3e952e11dfa401640112a1b316a2cff" target='_blank'>
              Analyzing Telomeric Protein-DNA Interactions Using Single-Molecule Magnetic Tweezers.
              </a>
            </td>
          <td>
            Han Gao, Yanling Liu, Zhongbo Yu
          </td>
          <td>2024-08-30</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37216ded337bd372b65a91cb1670af84243d9871" target='_blank'>
              Single cell whole-genome sequencing of brain cells: age- and cell-type specific mutational profiles
              </a>
            </td>
          <td>
            Melania Capasso, N. A. Aziz
          </td>
          <td>2024-08-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="PURPOSE
The acid phosphatase 1 (ACP1) gene encodes low-molecular-weight protein tyrosine phosphatase, which is overexpressed in prostate cancer (PC) and a potential therapeutic target. We analyzed ACP1 expression in primary/metastatic PC and its association with molecular profiles and clinical outcomes.


METHODS
NextGen sequencing of DNA (592-gene/whole-exome sequencing)/RNA(whole-transcriptome sequencing) was performed for 5,028 specimens. ACP1-High/ACP1-Low expression was defined as quartile (Q4/1) of RNA transcripts per million (TPM). DNA mutational profiles were analyzed for ACP1-quartile-stratified samples. Gene set enrichment analysis was used for Hallmark collection of pathways. PD-L1+(≥2+, ≥5%; SP142) was tested by immunohistochemistry. Tumor microenvironment's (TME) immune cell fractions were estimated by RNA deconvolution/quanTIseq. Overall survival (OS) was assessed from initial diagnosis/treatment initiation to death/last follow-up.


RESULTS
We included 3,058 (60.8%) samples from the prostate, 634 (12.6%) from lymph node metastases (LNMs), and 1,307 (26.0%) from distant metastases (DMs). ACP1 expression was higher in LNM/DM than prostate (49.8/47.9 v 44.1 TPM; P < .0001). TP53 mutations were enriched in ACP1-Q4 (37.9%[Q4] v 27.0%[Q1]; P < .001) among prostate samples. Pathways associated with cell cycle regulation and oxidative phosphorylation were enriched in ACP1-Q4, whereas epithelial-mesenchymal transition and tumor necrosis factor-alpha signaling via nuclear factor kappa-light-chain-enhancer of activated B-cell pathways were enriched in ACP1-Q1. Neuroendocrine and androgen receptor signaling was increased in ACP1-Q4. M2 macrophages and natural killer cell fractions were increased, whereas T cells and M1 macrophages were decreased in ACP1-Q4. While OS differences between ACP1-Q1/Q4 were not statistically significant, there was a trend for worse OS among ACP1-Q4 prostate samples (Q4 v Q1: hazard ratio [HR], 1.19 [95% CI, 0.99 to 1.42]; P = .06) and DM (HR, 1.12 [95% CI, 0.93 to 1.36]; P = .22) but not LNM (HR, 0.98 [95% CI, 0.74 to 1.29]; P = .87).


CONCLUSION
ACP1-High tumors exhibit a distinct molecular profile and cold TME, highlighting ACP1's potential role in PC pathogenesis and novel therapeutic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52a6914482e098afcc2b12fcf107d627059fc387" target='_blank'>
              Dissecting the Significance of Acid Phosphatase 1 Gene Alterations in Prostate Cancer.
              </a>
            </td>
          <td>
            Nour Abdallah, Andrew Elliott, Norm Smith, Stephanie M Stanford, N. Agarwal, Aditya Bagrodia, R. Garje, Nunzio Bottini, Rana Mckay
          </td>
          <td>2024-10-01</td>
          <td>JCO precision oncology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Despite a critical role for tumor-initiating cancer stem cells (CSCs) in breast cancer progression, major questions remain about the properties and signaling pathways essential for their function. Recent discoveries highlighting mechanisms of CSC-resistance to the stress caused by chromosomal instability (CIN) may provide valuable new insight into the underlying forces driving stemness properties. While stress tolerance is a well-known attribute of CSCs, CIN-induced stress is distinctive since levels appear to increase during tumor initiation and metastasis. These dynamic changes in CIN levels may serve as a barrier constraining the effects of non-CSCs and shaping the stemness landscape during the early stages of disease progression. In contrast to most other stresses, CIN can also paradoxically activate pro-tumorigenic antiviral signaling. Though seemingly contradictory, this may indicate that mechanisms of CIN tolerance and pro-tumorigenic inflammatory signaling closely collaborate to define the CSC state. Together, these unique features may form the basis for a critical relationship between CIN and stemness properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d491eba9730d9558923c4e53a3264dd576b3117a" target='_blank'>
              Chromosomal instability as an architect of the cancer stemness landscape
              </a>
            </td>
          <td>
            Shahnawaz A. Baba, Aran Zakeri, J. Desgrosellier
          </td>
          <td>2024-09-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7401c81aea35658fe72a5961ba5c3e31cae1d2fa" target='_blank'>
              The genomic basis of childhood T-lineage acute lymphoblastic leukaemia.
              </a>
            </td>
          <td>
            P. Pölönen, Danika Di Giacomo, A. Seffernick, Abdelrahman A. Elsayed, Shunsuke Kimura, Francesca Benini, Lindsey E Montefiori, Brent L Wood, Jason Xu, Changya Chen, Zhong-Yi Cheng, H. Newman, Jason R Myers, Ilaria Iacobucci, Elizabeth Li, Jonathan H. Sussman, Dale Hedges, Yawei Hui, Caroline Diorio, Lahari Uppuluri, David Frank, Yiping Fan, Yunchao Chang, S. Meshinchi, R. Ries, R. Shraim, Alexander Li, Kathrin M Bernt, M. Devidas, S. Winter, K. Dunsmore, H. Inaba, William L Carroll, Nilsa C Ramirez, Aaron H. Phillips, Richard W. Kriwacki, Jun J Yang, T. Vincent, Yaqi Zhao, P. S. Ghate, Jian Wang, Colleen Reilly, Xin Zhou, Mathijs A. Sanders, Junko Takita, Motohiro Kato, Nao Takasugi, Bill H Chang, Richard D Press, M. Loh, E. Rampersaud, E. Raetz, S P Hunger, Kai Tan, Ti-Cheng Chang, Gang Wu, S. Pounds, C. Mullighan, D. Teachey
          </td>
          <td>2024-08-14</td>
          <td>Nature</td>
          <td>2</td>
          <td>103</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e99609007bf2bb9e2a0595195e58156218094a1" target='_blank'>
              The significant others of aurora kinase a in cancer: combination is the key
              </a>
            </td>
          <td>
            Kumar Nikhil, Kavita Shah
          </td>
          <td>2024-09-27</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Here, we present a method for enrichment of double-stranded cfDNA with an average length of ∼40 bp from cfDNA for high-throughput DNA sequencing. This class of cfDNA is enriched at gene promoters and binding sites of transcription factors or structural DNA-binding proteins, so that a genome-wide DNA footprint is directly captured from liquid biopsies. In short double-stranded cfDNA from healthy individuals, we find significant enrichment of 203 transcription factor motifs. Additionally, short double-stranded cfDNA signals at specific genomic regions correlate negatively with DNA methylation, positively with H3K4me3 histone modifications and gene transcription. The diagnostic potential of short double-stranded cell-free DNA (cfDNA) in blood plasma has not yet been recognized. When comparing short double-stranded cfDNA from patient samples of pancreatic ductal adenocarcinoma with colorectal carcinoma or septic with postoperative controls, we identify 136 and 241 differentially enriched loci, respectively. Using these differentially enriched loci, the disease types can be clearly distinguished by principal component analysis, demonstrating the diagnostic potential of short double-stranded cfDNA signals as a new class of biomarkers for liquid biopsies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/003bad913ef23b451e8afe9e333721e6d528290e" target='_blank'>
              A novel approach for in vivo DNA footprinting using short double-stranded cell-free DNA from plasma
              </a>
            </td>
          <td>
            Jan Müller, Christina Hartwig, Mirko Sonntag, Lisa Bitzer, Christopher Adelmann, Y. Vainshtein, Karolina Glanz, S. Decker, Thorsten Brenner, Georg F. Weber, A. von Haeseler, Kai Sohn
          </td>
          <td>2024-08-01</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a liver cancer of adolescents and young adults characterized by fusions of the genes encoding the protein kinase A catalytic subunit, PRKACA, and heat shock protein, DNAJB1. The chimeric DNAJB1-PRKACA protein has increased kinase activity and is essential for FLC xenograft growth. Here, we explore the critical oncogenic pathways controlled by DNAJB1-PRKACA using patient-derived FLC models, engineered systems, and patient samples. We show that a core function of DNAJB1-PRKACA is the phosphorylation and inactivation of Salt-inducible kinases (SIKs). This leads to deregulation of the CRTC2 transcriptional co-activator and p300 acetyltransferase, resulting in transcriptional reprogramming and increased global histone acetylation, driving malignant growth. Our studies establish a central oncogenic mechanism of DNAJB1-PRKACA and suggest the potential of targeting CRTC2/p300 in FLC. Notably, these findings link this rare cancer's signature fusion oncoprotein to more common cancer gene alterations involving STK11 and GNAS, which also function via SIK suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a77fde5ecc4bc9340e646816c229c18dfe2fdb5e" target='_blank'>
              DNAJB1-PRKACA fusion drives fibrolamellar liver cancer through impaired SIK signaling and CRTC2/p300-mediated transcriptional reprogramming.
              </a>
            </td>
          <td>
            Ilaria Gritti, Jinkai Wan, Vajira Weeresekara, Joel M Vaz, Giuseppe Tarantino, Tenna Holgersen Bryde, Vindhya Vijay, Ashwin V. Kammula, Prabhat Kattel, Songli Zhu, Phuong Vu, Marina Chan, Meng-Ju Wu, John D Gordan, K. Patra, Vanessa S Silveira, R. Manguso, Marc N. Wein, Christopher J Ott, Jun Qi, David Liu, Kei Sakamoto, Taranjit S. Gujral, N. Bardeesy
          </td>
          <td>2024-09-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Patients with microsatellite instability-high (MSI-H) colorectal cancer (CRC) have high tumor mutation burden and tumor immunogenicity, exhibiting a higher response rate to immunotherapy and better survival. However, a portion of MSI-H CRC patients still experience adverse disease outcomes. We aimed to identify the tumor-autonomous regulators determining these heterogeneous clinical outcomes. The Cancer Genome Atlas (TCGA) dataset was used to identify regulators in MSI-H CRC patients with unfavorable outcomes. Stable CRC tumor clones expressing targeted regulators were established to evaluate migratory and stemness properties, immune cell vulnerability, and cell-in-cell (CIC) structure formation. RNA-sequencing (RNA-seq) was used to identify enriched biological pathways in stable CRC tumor clones. Clinicopathological characterization of formalin-fixed paraffin-embedded (FFPE) MSI-H CRC specimens was performed to explore the underlying mechanisms involved. We showed that cancer/testis antigen family 45 member A1 (CT45A1) expression was upregulated in MSI-H CRC patients with poor survival outcomes. CT45A1-expressing microsatellite stable (MSS) CRC cells showed enhanced migratory ability. However, CT45A1-expressing MSI-H CRC cells, but not MSS CRC cells, showed higher resistance to natural killer (NK) cell cytotoxicity and served as outer cells in homotypic CIC structures, preventing exogenous or therapeutic antibody access to inner CRC cells. Inactivating RHO-ROCK/MLCK-MLC2 signaling with small-molecule inhibitors or short-hairpin RNAs (shRNAs) targeting myosin light chain kinase (MYLK) abolished NK cell resistance and reduced the outer cell fate of CT45A1-expressing MSI-H CRC cells. In MSI-H CRC patients, CT45A1-positive tumors exhibited increased MLC2 phosphorylation, increased outer cell fate, and decreased survival. We demonstrated that CT45A1 potentiates the advanced progression of MSI-H CRC, and targeting MLC2 phosphorylation may enhance immunotherapy efficacy in CT45A1-positive MSI-H CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a08f29a07f3e7ccbababc06da74550a73f5f37c3" target='_blank'>
              CT45A1-mediated MLC2 (MYL9) phosphorylation promotes natural killer cell resistance and outer cell fate in a cell-in-cell structure, potentiating the progression of microsatellite instability-high colorectal cancer.
              </a>
            </td>
          <td>
            H. Teng, Hsiang-Yueh Huang, Chun-Chi Lin, Y. Twu, Wen-Hao Yang, Wen-Chun Lin, Hsin-Yi Lan, Yen-Yu Lin, Wei-Lun Hwang
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Understanding the genetic regulation, for example, gene expressions (GEs) by copy number variations and methylations, is crucial to uncover the development and progression of complex diseases. Advancing from early studies that are mostly focused on homogeneous groups of patients, some recent studies have shifted their focus toward different patient groups, explored their commonalities and differences, and led to insightful findings. However, the analysis can be very challenging with one GE possibly regulated by multiple regulators and one regulator potentially regulating the expressions of multiple genes, leading to two distinct types of commonalities/differences in the patterns of genetic regulation. In addition, the high dimensionality of both sides of regulation poses challenges to computation. In this study, we develop a two-way fusion integrative analysis approach, which innovatively applies two fusion penalties to simultaneously identify commonalities/differences in the regulated pattern of GEs and regulating pattern of regulators, and adopt a Huber loss function to accommodate the possible data contamination. Moreover, a simple yet efficient iterative optimization algorithm is developed, which does not need to introduce any auxiliary variables and extra tuning parameters and is guaranteed to converge to a globally optimal solution. The advantages of the proposed approach are demonstrated in extensive simulations. The analysis of The Cancer Genome Atlas data on melanoma and lung cancer leads to interesting findings and satisfactory prediction performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c30f4bc0f5f2a42888c972380cc183c3c5a04c6" target='_blank'>
              Unveiling Commonalities and Differences in Genetic Regulations via Two-Way Fusion.
              </a>
            </td>
          <td>
            Biao Mei, Yu Jiang, Yifan Sun
          </td>
          <td>2024-08-12</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Genome instability relies on preserving the chromatin structure, with any histone imbalances threating DNA integrity. Histone synthesis occurs in the cytoplasm, followed by a maturation process before their nuclear translocation. This maturation involves protein folding and the establishment of post-translational modifications. Disruptions in this pathway hinder chromatin assembly and contribute to genome instability. JMJD1B, a histone demethylase, not only regulates gene expression but also ensures a proper supply of histones H3 and H4 for the chromatin assembly. Reduced JMJD1B levels lead to the cytoplasmic accumulation of histones, causing defects in the chromatin assembly and resulting in DNA damage. To investigate the role of JMJD1B in regulating genome stability and the malignancy of melanoma tumors, we used a JMJD1B/KDM3B knockout in B16F10 mouse melanoma cells to perform tumorigenic and genome instability assays. Additionally, we analyzed the transcriptomic data of human cutaneous melanoma tumors. Our results show the enhanced tumorigenic properties of JMJD1B knockout melanoma cells both in vitro and in vivo. The γH2AX staining, Micrococcal Nuclease sensitivity, and comet assays demonstrated increased DNA damage and genome instability. The JMJD1B expression in human melanoma tumors correlates with a lower mutational burden and fewer oncogenic driver mutations. Our findings highlight JMJD1B’s role in maintaining genome integrity by ensuring a proper histone supply to the nucleus, expanding its function beyond gene expression regulation. JMJD1B emerges as a crucial player in preserving genome stability and the development of melanoma, with a potential role as a safeguard against oncogenic mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1490db96e0fd741bd69913b1e9fe76e21ec91750" target='_blank'>
              Unraveling the Role of JMJD1B in Genome Stability and the Malignancy of Melanomas
              </a>
            </td>
          <td>
            Perla Cruz, Diego Peña-Lopez, Diego Figueroa, Isidora Riobó, Vincenzo Benedetti, Francisco Saavedra, Claudia Espinoza-Arratia, Thelma M. Escobar, A. Lladser, Alejandra Loyola
          </td>
          <td>2024-10-04</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Retrotransposons are invasive genetic elements, which replicate by copying and pasting themselves throughout the genome in a process called retrotransposition. The most abundant retrotransposons by number in the human genome are Alu and LINE-1 elements, which comprise approximately 40% of the human genome. The ability of retrotransposons to expand and colonize eukaryotic genomes has rendered them evolutionarily successful and is responsible for creating genetic alterations leading to significant impacts on their hosts. Previous research suggested that hypomethylation of Alu and LINE-1 elements is associated with global hypomethylation and genomic instability in several types of cancer and diseases, such as neurodegenerative diseases, obesity, osteoporosis, and diabetes mellitus (DM). With the advancement of sequencing technologies and computational tools, the study of the retrotransposon’s association with physiology and diseases is becoming a hot topic among researchers. Quantifying Alu and LINE-1 methylation is thought to serve as a surrogate measurement of global DNA methylation level. Although Alu and LINE-1 hypomethylation appears to serve as a cellular senescence biomarker promoting genomic instability, there is sparse information available regarding their potential functional and biological significance in DM. This review article summarizes the current knowledge on the involvement of the main epigenetic alterations in the methylation status of Alu and LINE-1 retrotransposons and their potential role as epigenetic markers of global DNA methylation in the pathogenesis of DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07f55c13ed59ce933c2fb2995ef34b1c1fa76b5" target='_blank'>
              Retrotransposons and Diabetes Mellitus
              </a>
            </td>
          <td>
            Andromachi Katsanou, Charilaos Kostoulas, E. Liberopoulos, A. Tsatsoulis, I. Georgiou, S. Tigas
          </td>
          <td>2024-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Leukemia is a malignant tumor with high heterogeneity and a complex evolutionary process. It is difficult to resolve the heterogeneity and clonal evolution of leukemia cells by applying traditional bulk sequencing techniques, thus preventing a deep understanding of the mechanisms of leukemia development and the identification of potential therapeutic targets. However, with the development and application of single-cell sequencing technology, it is now possible to investigate the gene expression profile, mutations, and epigenetic features of leukemia at the single-cell level, thus providing a new perspective for leukemia research. In this article, we review the recent applications and advances of single-cell sequencing technology in leukemia research, discuss its potential for enhancing our understanding of the mechanisms of leukemia development, discovering therapeutic targets and personalized treatment, and provide reference guidelines for the significance of this technology in clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059e4fa52cbb0003a82e3f417ae0958def767ace" target='_blank'>
              Application and research progress of single cell sequencing technology in leukemia
              </a>
            </td>
          <td>
            Dan Xie, Bangquan An, Mingyue Yang, Lei Wang, Min Guo, Heng Luo, Shengwen Huang, Fa Sun
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df4be94d1533f2155dbf3532c23eeba54389f11" target='_blank'>
              Cohesin mutations in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Austin Boucher, J. Murray, Sridhar Rao
          </td>
          <td>2024-09-09</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c751f02ecfd3cada2a90e62224689eeb8eaf33b4" target='_blank'>
              Forward–reverse mutation cycles in cancer cell lines under chemical treatments
              </a>
            </td>
          <td>
            Si Chen, I. S. Tyagi, Wai-Kin Mat, Muhammad A. Khan, Weijian Fan, Zhenggang Wu, Taobo Hu, Can Yang, Hong Xue
          </td>
          <td>2024-09-27</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The detection of mitochondrial DNA (mtDNA) mutations in single cells holds considerable potential to define clonal relationships coupled with information on cell state in humans. Previous methods focused on higher heteroplasmy mutations that are limited in number and can be influenced by functional selection, introducing biases for lineage tracing. Although more challenging to detect, intermediate to low heteroplasmy mtDNA mutations are valuable due to their high diversity, abundance, and lower propensity to selection. To enhance mtDNA mutation detection and facilitate fine-scale lineage tracing, we developed the single-cell Regulatory multi-omics with Deep Mitochondrial mutation profiling (ReDeeM) approach, an integrated experimental and computational framework. Recently, some concerns have been raised about the analytical workflow in the ReDeeM framework. Specifically, it was noted that the mutations detected in a single molecule per cell are enriched on edges of mtDNA molecules, suggesting they resemble artifacts reported in other sequencing approaches. It was then proposed that all mutations found in one molecule per cell should be removed. We detail our error correction method, demonstrating that the observed edge mutations are distinct from previously reported sequencing artifacts. We further show that the proposed removal leads to massive elimination of bona fide and informative mutations. Indeed, mutations accumulating on edges impact a minority of all mutation calls (for example, in hematopoietic stem cells, the excess mutations on the edge account for only 4.3%-7.6% of the total). Recognizing the value of addressing edge mutations even after applying consensus correction, we provide an additional filtering option in the ReDeeM-R package. This approach effectively eliminates the position biases, leads to a mutational signature indistinguishable from bona fide mitochondrial mutations, and removes excess low molecule high connectedness mutations. Importantly, this option preserves the large majority of unique mutations identified by ReDeeM, maintaining the ability of ReDeeM to provide a more than 10-fold increase in variant detection compared to previous methods. Additionally, the cells remain well-connected. While there is room for further refinement in mutation calling strategies, the significant advances and biological insights provided by the ReDeeM framework are unique and remain intact. We hope that this detailed discussion and analysis enables the community to employ this approach and contribute to its further development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94914bd4d265668ea63fab9b4a1aa10d1231c91c" target='_blank'>
              Robustness and reliability of single-cell regulatory multi-omics with deep mitochondrial mutation profiling
              </a>
            </td>
          <td>
            Chen Weng, Jonathan S. Weissman, V. Sankaran
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Background: N6-methyladenosine (m6A), is the most common modification found in mRNA and lncRNA in higher organisms and plays an important role in physiology and pathology. However, its role in pan-cancer has not been explored. Results: A total of 31 m6A modification regulators, including 12 writers, 2 erasers, and 17 readers are identified in the current study. The functional analysis of the regulators results in the enrichment of processes, primarily related to RNA modification and metabolism, and the PPI network reveals multiple interactions among the regulators. The mRNA expression analysis reveals a high expression for most of the regulators in pan-cancer. Most of the m6A regulators are found to be mutated across the cancers, with ZC3H13, VIRMA, and PRRC2A having a higher frequency rate. Significant correlations of the regulators with clinicopathological parameters, such as age, gender, tumor stage, and grade are identified in pan-cancer. The m6A regulators’ expression is found to have significant positive correlations with the miRNAs in pan-cancer. The expression pattern of the m6A regulators is able to classify the tumors into different subclusters as well as into high- and low-risk groups. These tumor groups show differential patterns in terms of their immune cell infiltration, tumor stemness score, genomic heterogeneity score, expression of immune regulatory/checkpoint genes, and correlations between the regulators and the drugs. Conclusions: Our study provide a comprehensive overview of the functional roles, genetic and epigenetic alterations, and prognostic value of the RNA m6A regulators in pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4e0ddd1547e3b4017e3b8a95f733d908f67e777" target='_blank'>
              Interplay of RNA m6A Modification-Related Geneset in Pan-Cancer
              </a>
            </td>
          <td>
            Boyu Zhang, Yajuan Hao, Haiyan Liu, Jiarun Wu, Lu Lu, Xinfeng Wang, A. Bajpai, Xi Yang
          </td>
          <td>2024-09-27</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="The literature review is devoted to the chromosomally integrating viruses, the integration property. These are revealed using modern diagnostic methods, including highthroughput sequencing. The review describes various types of viruses, such as retroviruses, adenoviruses, herpes viruses, papillomaviruses, hepatitis B, SARS-CoV-2 and others. These viruses have the ability to integrate their genomes into the host cell genomes and their prevalence. Modern information on the molecular mechanisms of viral integration and methods of differential diagnostics is presented. Each of these viruses has its own unique ways of interacting with the host cell genome to maintain a latent state, which determines the specifics of the infectious process and pathological consequences for the body due to the possibility of effective reactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6677dd293cf4b5a23bcbb5a7ef88d7a9fe2d542e" target='_blank'>
              The phenomenon of viral chromosomal integration
              </a>
            </td>
          <td>
            O. Goleva, P. V. Cherkasova, E. V. Baziian, A. A. Igolkina, Yu. A. Ejsmont, N. Skripchenko, A. B. Chukhlovin, O. S. Glotov
          </td>
          <td>2024-09-20</td>
          <td>Journal Infectology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="In acute promyelocytic leukemia (APL), the promyelocytic leukemia-retinoic acid receptor alpha (PML/RARα) fusion protein destroys PML nuclear bodies (NBs), leading to the formation of microspeckles. However, our understanding, largely learned from morphological observations, lacks insight into the mechanisms behind PML/RARα-mediated microspeckle formation and its role in APL leukemogenesis. This study presents evidence uncovering liquid-liquid phase separation (LLPS) as a key mechanism in the formation of PML/RARα-mediated microspeckles. This process is facilitated by the intrinsically disordered region containing a large portion of PML and a smaller segment of RARα. We demonstrate the coassembly of bromodomain-containing protein 4 (BRD4) within PML/RARα-mediated condensates, differing from wild-type PML-formed NBs. In the absence of PML/RARα, PML NBs and BRD4 puncta exist as two independent phases, but the presence of PML/RARα disrupts PML NBs and redistributes PML and BRD4 into a distinct phase, forming PML/RARα-assembled microspeckles. Genome-wide profiling reveals a PML/RARα-induced BRD4 redistribution across the genome, with preferential binding to super-enhancers and broad-promoters (SEBPs). Mechanistically, BRD4 is recruited by PML/RARα into nuclear condensates, facilitating BRD4 chromatin binding to exert transcriptional activation essential for APL survival. Perturbing LLPS through chemical inhibition (1, 6-hexanediol) significantly reduces chromatin co-occupancy of PML/RARα and BRD4, attenuating their target gene activation. Finally, a series of experimental validations in primary APL patient samples confirm that PML/RARα forms microspeckles through condensates, recruits BRD4 to coassemble condensates, and co-occupies SEBP regions. Our findings elucidate the biophysical, pathological, and transcriptional dynamics of PML/RARα-assembled microspeckles, underscoring the importance of BRD4 in mediating transcriptional activation that enables PML/RARα to initiate APL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23f8fbc164ec2b07c1fe6493fb810d79f73ea0d8" target='_blank'>
              Phase separation of PML/RARα and BRD4 coassembled microspeckles governs transcriptional dysregulation in acute promyelocytic leukemia.
              </a>
            </td>
          <td>
            Yi Zhang, Jiacheng Lou, Yabin Liu, Peng Jin, Yun Tan, Huan Song, W. Jin, Dan Wang, Fangyi Dong, Shishuang Wu, Hai Fang, Saijuan Chen, Zhu Chen, Kankan Wang
          </td>
          <td>2024-08-13</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac48d4d6b8d1418ab3b28607fa322fa693c4c306" target='_blank'>
              Microproteins unveiling new dimensions in cancer.
              </a>
            </td>
          <td>
            S. Daisy Precilla, Indrani Biswas, T. Anitha, B. Agieshkumar
          </td>
          <td>2024-09-03</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Prostate cancer is frequently treated with radiotherapy. Unfortunately, aggressive radioresistant relapses can arise, and the molecular underpinnings of radioresistance are unknown. Modern clinical radiotherapy is evolving to deliver higher doses of radiation in fewer fractions (hypofractionation). We therefore analyzed genomic, transcriptomic, and proteomic data to characterize prostate cancer radioresistance in cells treated with both conventionally fractionated and hypofractionated radiotherapy. Independent of fractionation schedule, resistance to radiotherapy involved massive genomic instability and abrogation of DNA mismatch repair. Specific prostate cancer driver genes were modulated at the RNA and protein levels, with distinct protein subcellular responses to radiotherapy. Conventional fractionation led to a far more aggressive biomolecular response than hypofractionation. Testing preclinical candidates identified in cell lines, we revealed POLQ (DNA Polymerase Theta) as a radiosensitizer. POLQ-modulated radioresistance in model systems and was predictive of it in large patient cohorts. The molecular response to radiation is highly multimodal and sheds light on prostate cancer lethality. Significance: Radiation is standard of care in prostate cancer. Yet, we have little understanding of its failure. We demonstrate a new paradigm that radioresistance is fractionation specific and identified POLQ as a radioresistance modulator.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9fa9b6b4bc258239503f1fab9f687905a6c0b2b" target='_blank'>
              The Proteogenomics of Prostate Cancer Radioresistance
              </a>
            </td>
          <td>
            Roni Haas, Gavin Frame, Shahbaz Khan, B. Neilsen, Boon Hao Hong, Celestia P X Yeo, Takafumi N. Yamaguchi, E.H.W. Ong, Wenyan Zhao, Benjamin Carlin, E. Yeo, Kah M Tan, Yuan Zhe Bugh, Chenghao Zhu, Rupert Hugh-White, J. Livingstone, Dennis J J Poon, Pek L Chu, Yash Patel, Shu Tao, V. Ignatchenko, Natalie J Kurganovs, G. Higgins, Michelle R Downes, A. Loblaw, Danny Vesprini, A. Kishan, Melvin L K Chua, T. Kislinger, P. Boutros, Stanley K Liu
          </td>
          <td>2024-08-21</td>
          <td>Cancer Research Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Giant cell glioblastoma often exhibits genome instability and is frequently associated with mutations in genes involved in DNA repair pathways including TP53 and DNA mismatch repair genes. Several germline mutations have been identified in giant cell glioblastoma, including mutations of MSH1 and MSH2, TP53, and POLE. We have documented a case of a germline mutation in CHEK2, another gene crucial to DNA repair, in a patient with giant cell glioblastoma. The CHEK2 mutation was inherited from the patient’s father, who had a history of gastric cancer and renal cell carcinoma. In addition to the germline CHEK2 mutation, the giant cell glioblastoma exhibited a genome-wide loss of heterozygosity, a characteristic observed in a subset of giant cell glioblastomas. Additional mutations detected in the tumor included TP53, PTEN, and a PTPRZ1-MET fusion. This represents the first reported case of a CHEK2 germline mutation in giant cell glioblastoma, further supporting the significance of impaired DNA repair mechanisms in the development of this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8785752f439604ae99bf1f0316497dcd1988c0b0" target='_blank'>
              Case report: Germline CHEK2 mutation is associated with a giant cell glioblastoma
              </a>
            </td>
          <td>
            Yongfeng Bi, Dong Wan, Si Chen, Huafei Chen, Lingchuan Guo, Xiaoshun He, Rong Rong, Jinyuan Xiao, Wei Gao, Sheng Xiao
          </td>
          <td>2024-10-01</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823edc7b7f2e48466de67d70b29a3e25297b1b88" target='_blank'>
              Cooperative genomic lesions in HRAS-mutant cancers predict resistance to farnesyltransferase inhibitors.
              </a>
            </td>
          <td>
            Arad Nigam, Gnana P. Krishnamoorthy, Walid K. Chatila, Katherine Berman, Mahesh Saqcena, H. Walch, Mandakini Venkatramani, Alan L. Ho, N. Schultz, James A. Fagin, B. Untch
          </td>
          <td>2024-08-16</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="
 Background: Transposable elements (TEs) are dynamic repetitive regions which generate mutations and structural variants. TP53 plays a crucial role in suppressing TE movement to maintain genomic stability. The relationship between TP53 and TEs has been extensively studied in tumours, but not the germline. Individuals with germline TP53 pathogenic variants have Li-Fraumeni Syndrome (LFS), a cancer predisposition syndrome with a high lifetime risk of cancer in various tissues. This study aims to characterize the TE landscape in individuals with Li-Fraumeni Syndrome and determine how this contributes to their increased cancer risk. Methods: MELT and xTea were used to identify TEs in children with (n=48) or without (n=198) a germline TP53 variant. TE calls were merged with SURVIVOR and variants were annotated with AnnotSV. To assess the influence of TP53 on TE location, we quantified TEs across genomic windows and identified the top 100 significantly different regions with the Mann-Whitney U test, adjusting for multiple comparisons. We used these regions to develop a gradient-boosted tree model with 5-fold cross-validation to predict TP53 status. Results: Individuals with germline TP53 variants harboured significantly fewer ALU and LINE1 elements in their germline genome compared to the control dataset (p<0.001). The phenomenon was consistent across chromosomes and significant in chromosomes 5 and 11 (FDR<0.05). A gradient-boosted tree model was able to differentiate patients with and without a germline TP53 variant with an AUPRC of 0.77 on an unseen test set. Conclusion: Germline TP53 variants may effect the frequency and location of germline TE insertions, which may influence nearby genomic variations and lead to cancer development. Analyzing TEs in individuals with LFS will enhance our understanding of accelerated cancer development in these patients, informing future research for diagnostic and therapeutic approaches.
 Citation Format: Brianne Laverty, Shilpa Yadahalli, Vallijah Subasri, David Malkin. The role of TP53 on transposable elements in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4995c29bd2050c705f9a5b8da916aacb0aa2601c" target='_blank'>
              Abstract A010 The role of TP53 on transposable elements in pediatric cancer
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Vallijah Subasri, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In recent decades, the field of genetics has revolutionized our understanding of biology. In particular, we know about how genetic information encodes the blueprint for life and how variations and mutations in these sequences contribute to disease or health. What is unique is that the genetic trait changes do not always caused by any changes in genetic sequences. 
Epigenetics refers to the study of how environmental factors and lifestyle choices can affect the gene activity. Changes in the heritable traits of organisms can be caused by modification of gene expression rather than alteration to the underlying DNA sequence. Therefore, this change does not involve any DNA mutation. 
DNA methylation is a biological process of adding a methyl group to the DNA molecule. The activity of DNA segments can change without changing the sequence by the process of methylation. This process plays an important role in normal development and aberrant DNA methylation patterns are associated with various diseases. Histones are highly basic proteins around which DNA is wound. The modifications in histone protein plays a fundamental role in most biological processes and can alter the accessibility of the DNA for transcription, as well as influencing gene expression. There are non-coding RNA molecules that do not encode proteins but play vital role in regulating the gene expression by interacting with other molecules and influencing chromatin structure. 
Over the past few decades, mechanisms of epigenetics have emerged as a crucial factor in governing every biological process in the human body. Throughout human life, these mechanisms have been impacted various physiological and pathological conditions. For example, cancer is often characterized by epigenetics abnormalities, such as hypermethylation of tumor suppressor genes or hypomethylation of oncogenes. Epigenetic modifications are also implicated in complex diseases like cardiovascular disease, autoimmune conditions and neurodegenerative disorder. Beyond these diseases, epigenetics also offers insights into health and development. Furthermore, understanding the epigenetic mechanisms influence human health, leading to new preventive strategies and treatment of various diseases. Early life experiences, including nutrition and stress, can lead to epigenetic changes that affect long-term health outcomes of the individual.   
Advances in this research can pinpoint any epigenetic changes that affect long-term health outcomes. The advances of epigenetic research can lead to early diagnosis and preventive measures. Identifying the epigenetic markers associated with the diseases enable early detection of the diseases and more effective preventative strategies. Some epigenetic therapies like; drugs that modify histone acetylation or DNA methylation, are already in development and provide future treatments for disease. To understand how life style factors, influence epigenetic regulation can lead to personalized lifestyle recommendations that influence the public health strategies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6963fc644883e01d78b4f2a627be037841fc025" target='_blank'>
              The Role of Epigenetics in Disease and Health: Beyond DNA Sequences
              </a>
            </td>
          <td>
            .. Diki
          </td>
          <td>2024-08-31</td>
          <td>Pakistan BioMedical Journal</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background/Objectives: Head and neck squamous cell carcinoma (HNSCC) is a highly heterogeneous and the most common form of head and neck cancer, posing significant challenges for disease management. The objective of this review is to assess the utility of single-cell RNA sequencing (scRNAseq) in addressing these challenges by enabling a detailed characterization of the tumor microenvironment (TME) at the cellular level. Methods: This review compiles and analyzes current strategies that utilize scRNAseq and other single-cell technologies in HNSCC research. Results: For HNSCC etiology, scRNAseq allows for the construction of cellular atlases, characterization of different cell types, and investigation of genes and processes involved in cancer initiation, development, and progression within the TME. In terms of HNSCC diagnosis and prognosis, the resolution offered by scRNAseq enables the identification of cell type-specific signatures, enhancing prognostic models and disease stratifiers for patient outcome assessments. Regarding HNSCC treatment, scRNAseq provides insights into cellular responses to various treatments, including radiotherapy, chemotherapy, and immunotherapy, contributing to a better understanding of treatment efficacy and patient outcomes. Conclusions: This review highlights the contributions of scRNAseq to HNSCC research, addressing its cellular and biological complexity, and emphasizes its potential for advancing research and clinical practice in other cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e831682e82f2a5f8a0b8ba438418409c9819bff6" target='_blank'>
              Unravelling the Complexity of HNSCC Using Single-Cell Transcriptomics
              </a>
            </td>
          <td>
            Cristina Conde-Lopez, Divyasree Marripati, M. Elkabets, Jochen Hess, Ina Kurth
          </td>
          <td>2024-09-25</td>
          <td>Cancers</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="


 Recent studies have linked certain colibactin producing subphylas of E. coli to colorectal cancer. The bacterium synthesizes colibactin via the pks genomic island containing a series of clbA to clbS genes. The genotoxin causes inter-strand DNA cross-links, which lead to double strand breaks and promotion of colorectal adenocarcinoma. The published studies have prioritized the colorectal regions of the body where E. coli is commonly found, but none have identified the same genotoxic relationship with other types of adenocarcinoma. In this study, we evaluate DNA damage and cellular changes of mammary breast tissue based on high and low levels of colibactin producing E. coli exposure. The goal is to evaluate if colibactin can affect cells in the mammary breast tissue promoting breast adenocarcinoma growth.



 Two cell lines were acquired from ATCC: primary mammary epithelial cells (PCS-600-010) and MCF-7 breast adenocarcinoma cells (HTB-22). Both cell lines were split into six culture flasks and maintained until ∼80% confluence was reached using the ATCC cell culture protocols. Once all the flasks had reached the target confluence, they were divided into exposure groups of high and low concentrations of colibactin producing E. coli, a group of non-colibactin producing E. coli, a group exposed only to phosphate buffered saline, and a control group with no exposure. Cellular morphology was observed using light microscopy before, after exposure, and every 5 minutes up to 15 minutes. E. coli and colibactin DNA concentrations were quantified using quantitative polymerase chain reaction. Genomic DNA was extracted from each group of cells. The DNA from each group was whole genome sequenced and analyzed for genomic variation.



 The normal cells exposed to E. coli showed immediate lysing and continued to lyse after a 5 minute incubation; however, the adenocarcinoma positive cells showed no lysing up to 30 minutes incubation. Sequencing results show genomic variation between the groups and specific break points associated with the E. coli exposed groups.



 Our results show that the presence of E. coli with mammary breast tissue can destroy normal tissue, promote lysis resistance, and allow adenocarcinoma cells to continue to grow. The colibactin producing E. coli does cause double stranded breaks and can lead to cell death. This new information points out a critical need to monitor the microbiome and infection levels of people at risk or in early stages of cancer to prevent adenocarcinoma growth and destruction of normal tissue. This study indicates that colibactin producing E. coli may play a larger role in tissue damage and promotion of other types of cancer. More research is needed to elucidate the molecular pathway and mechanisms of colibactin, which will lead to new diagnostics or therapeutics that monitor E. coli levels or inhibit colibactin’s toxic effects.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b3fd078061817b8e9827d3c0f82f869fc3cb02" target='_blank'>
              B-351 Genotoxic Producing Escherichia coli Protects and Promotes Breast Adenocarcinoma
              </a>
            </td>
          <td>
            L. Heaton, S. D. Figard, R. A. Garcia
          </td>
          <td>2024-10-01</td>
          <td>Clinical Chemistry</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/554a8d144e8cb33023121b6846104a6440f65c60" target='_blank'>
              SexAnnoDB, a knowledgebase of sex-specific regulations from multi-omics data of human cancers
              </a>
            </td>
          <td>
            Mengyuan Yang, Yuzhou Feng, Jiajia Liu, Hong Wang, Sijia Wu, Weiling Zhao, P. Kim, Xiaobo Zhou
          </td>
          <td>2024-08-22</td>
          <td>Biology of Sex Differences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Background/Objectives: PRKACA alterations have clear diagnostic and biological roles in the fibrolamellar variant of hepatocellular carcinoma and a potential predictive role in that cancer type. However, the roles of PRKACA have not been comprehensively examined in gastric and colorectal cancers (GC and CRC). This study, therefore, sought to investigate the roles of PRKACA expression in GC and CRC. Methods: The clinico-genomic data of 441 GC and 629 CRC cases were analyzed for therapeutic, clinicopathological, and biological correlates using appropriate bioinformatics and statistical tools. Furthermore, the deregulation of PRKACA expression in GC and CRC was investigated using correlative and regression analyses. Results: The results showed that PRKACA expression subsets were enriched for gene targets of chemotherapeutics, tyrosine kinase, and β-adrenergic inhibitors. Moreover, high PRKACA expression was associated with adverse clinicopathological and genomic features of GC and CRC. Gene Ontology Enrichment Analysis also showed that PRKACA-high subsets of the GI cancers were enriched for the biological and molecular functions that are associated with cell motility, invasion, and metastasis but not cell proliferation. Finally, multiple regression analyses identified multiple methylation loci, transcription factors, miRNA species, and PRKACA copy number changes that deregulated PRKACA expression in GC and CRC. Conclusions: This study has identified potential predictive and clinicopathological roles for PRKACA expression in GI cancers and has added to the growing body of knowledge on the deregulation of PRKACA in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a2145719a7583ee255803ff10ebae15008cf52f" target='_blank'>
              Therapeutic, Clinicopathological, and Molecular Correlates of PRKACA Expression in Gastrointestinal Cancers
              </a>
            </td>
          <td>
            A. Othaim, Glowi A Alasiri, A. Alfahed
          </td>
          <td>2024-09-25</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Oncoviruses can integrate into the host genome and cause tumorigenesis. In particular, hepatitis B virus (HBV) infection accounts for more than 50% of hepatocellular carcinoma (HCC) worldwide. We revealed the global geographical disparity of HBV integration that the landscape of HBV integration between HCC tumor and non-tumorous liver varied in regional cohorts, suggesting the different degrees of clonal enrichment. Most HBV integrations were positionally enriched at telomeres and centromeres (T&C) and they highlighted the novel co-involvement of HBV integration, which likely introduces genomic instability in HCC development. This was confirmed by phospho-H2AX staining. We constructed a large meta-cohort of multiple ethnicities to refine the landscape of HBV integration. This enables the gene set/family level exploration. As TERT is the most frequently integrated gene, we further investigated the underlying mechanistic modulation of TERT transcription activation and revealed the concurrent influence by the orientation and relative distance of HBV integration. Additionally, clonal disparity of HBV integration was observed among patients and the higher level of clonal disparity score can indicate poor patients' prognostication. Taken together, our study uncovered the different levels of clonal enrichment of HBV integration, mechanistic insights, and prognostic biomarker signature, to strengthen our understanding in HBV-associated hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a6ec7bd105c83b5d222005af0db05b395d835b64" target='_blank'>
              Disparity landscapes of viral-induced structural variations in hepatocellular carcinoma: Mechanistic characterization and functional implications.
              </a>
            </td>
          <td>
            Xueying Lyu, K. Sze, J. Lee, Abdullah Husain, Lu Tian, S. Imbeaud, J. Zucman-Rossi, I. Ng, DanielWai-Hung Ho
          </td>
          <td>2024-09-13</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Chromosomal linkages formed through crossover recombination are essential for accurate segregation of homologous chromosomes during meiosis1. DNA events of recombination are spatially and functionally linked to structural components of meiotic chromosomes2. Imperatively, biased resolution of double-Holliday junction (dHJ) intermediates into crossovers3,4 occurs within the synaptonemal complex (SC), the meiosis-specific structure that mediates homolog synapsis during the pachytene stage5,6. However, the SC’s role in crossing over remains unclear. Here we show that SC promotes crossover-specific resolution by protecting dHJs from unscheduled and aberrant resolution. When key SC components are conditionally inactivated during pachytene, dHJs are resolved into noncrossover products by Sgs1-Top3-Rmi1 (STR), the yeast ortholog of the human BLM complex7. Cohesin, the core component of SC lateral elements, plays a primary role in chromatin organization and is required to maintain both SCs and crossover recombination complexes (CRCs) during pachytene. SC central region component Zip1 is required to maintain CRCs even when dHJs are stabilized by inactivating STR. Reciprocally, SC stability requires continuous presence of CRCs, an unanticipated interdependence with important implications for SC dynamics. In conclusion, through hierarchical and interdependent functions of its key components, the SC enables crossover-specific dHJ resolution and thereby ensures the linkage and segregation of homologous chromosomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/95e11a6c6b1970e26248ec2273104829b507bc0a" target='_blank'>
              Synaptonemal complex protects double-Holliday junctions during meiosis
              </a>
            </td>
          <td>
            Shangming Tang, Jennifer Koo, Mohammad Pourhosseinzadeh, Emerald Nguyen, Natalie Liu, Christopher Ma, Hanyu Lu, Monica Lee, Neil Hunter
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="During meiosis, RNA polymerase II transcribes pachytene piRNA precursors with unusually long and unspliced transcripts from discrete autosomal loci in the mouse genome. Despite the importance of piRNA for male fertility and a well-defined maturation process, the transcriptional machinery remains poorly understood. Here, we document that D1PAS1, an ATP-dependent RNA helicase, is critical for pachytene piRNA expression from multiple genomic loci and subsequent translocation into the cytoplasm to ensure mature piRNA biogenesis. Depletion of D1PAS1 in gene-edited mice results in the accumulation of R-loops in pachytene spermatocytes, leading to DNA-damage-induced apoptosis, disruption of piRNA biogenesis, spermatogenic arrest, and male infertility. Transcriptome, genome-wide R-loop profiling, and proteomic analyses document that D1PAS1 regulates pachytene piRNA transcript elongation and termination. D1PAS1 subsequently forms a complex with nuclear export components to ensure pachytene piRNA precursor translocation from the nucleus to the cytoplasm for processing into small non-coding RNAs. Thus, our study defines D1PAS1 as a specific transcription activator that promotes R-loop unwinding and is a critical factor in pachytene piRNA biogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1bffe6baa4b004b5f7f7f5c5ad8b7fd3a3302df" target='_blank'>
              RNA helicase D1PAS1 resolves R-loops and forms a complex for mouse pachytene piRNA biogenesis required for male fertility.
              </a>
            </td>
          <td>
            Heejin Choi, Lecong Zhou, Yangu Zhao, Jurrien Dean
          </td>
          <td>2024-08-20</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chimeric genes and transcripts can be biological markers as well as the reasons for tumor progression and development. Modern algorithms and high-throughput sequencing are the complementary clues to the question of the tumor origin and cancer detection as well as to the fundamental question of chimeric genes origin and their influence on molecular processes of the cell. A wide-range of algorithms for chimeric genes detection was developed, with various differences in computing speed, sensitivity, specificity, and focus on the experimental design. There exist three main types of bioinformatic approaches, which act according to the sequencing read length. Algorithms, which focus on short-read high-throughput sequencing (about 50–300 bр of read length) or long-read sequencing (about 5000–100000 bр of read length) exclusively or algorithms, which combine the results of both short and long-read sequencing. These algorithms are further subdivided into: 1) mapping-first approaches (STAR-Fusion, Arriba), which map reads to the genome or transcriptome directly and search the reads supporting the fused gene or transcript; 2) assembly-first approaches (Fusion-Bloom), which assemble the genome or transcriptome from the overlapping reads, and then compare the results to the reference transcriptome or genome to find transcripts or genes not present in the reference and therefore raising questions; 3) pseudoalignment approaches, which do not make local alignment, but just search for the closest transcript subsequence to the reads seed, following the precomputed index for all reference transcripts and provides the results. This article describes the main classes of available software tools for chimeric gene detection, provides the characteristics of these programs, their advantages and disadvantages. To date the most resource intensive and slowest are still assembly-first algorithms. Mapping-first approaches are quite fast and rather accurate at fusion detection, still the fastest and resource-saving are the pseudoalignment algorithms, but, worth noting, that the quick search is carried out at the expense of chimeras search quality decrease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40da4e0fb6446cdfd7e663fbf9214c9d51fb3a2" target='_blank'>
              Bioinformatic approaches for detection of fusion genes and trans-splicing products
              </a>
            </td>
          <td>
            I. Y. Musatov, M. I. Sorokin, А. A. Buzdin
          </td>
          <td>2024-08-11</td>
          <td>Биоорганическая химия</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Philadelphia-positive leukemia is characterized by the translocation t(9;22)(q34;q11), which results in the oncogene BCR-ABL1 and a chromosome called the Philadelphia (Ph) chromosome (chr). The Philadelphia chromosome is best characterized in Chronic Myeloid Leukemia (CML); however, the translocation is also found in Acute Lymphoblastic Leukemia (ALL). Philadelphia-positive ALL has been historically understudied, especially in minority populations. Genome organization is a highly complex process by which chromatin is segregated within the nucleus into specific domains, making stable translocations highly unfavorable. How translocations are stabilized has been historically understudied. Here, we aim to examine how the Ph chr is stabilized and be the first to visualize an entire abnormal chr with high spatial and genomic resolution at the single-cell level. Using OligoSTORM, a super-resolution microscopy technique, we aim to image the entirety of the Ph chr at a nanoscale resolution in both CML and ALL cell lines. In addition to the DNA super-resolution trace, we have designed an RNA library to image a series of RNAs from the Ph chr on the same sample as the DNA trace, allowing us to connect the transcriptional data to the chromatin structure. We have successfully traced the entirety of the Ph chr. We have preliminary data suggesting that chr9 and chr22 segregate close to each other in non-leukemia healthy cells. We have also proven that our library design and methodology work to achieve the primary 500kb resolution. Preliminary data suggests that chr9 and chr22 in cells, that don’t typically have a Ph chr, segregate within proximity to translocate successfully. This study aims to compare the 3D structure of CML, ALL and non-leukemia cell lines to potentially identify genome organization-based biomarkers for identify and targeting the Philadelphia chromosome.
 Citation Format: Gabrielle S. Dewson, Guy Nir. Unraveling the structure of the Philadelphia chromosome using super-resolution microscopy [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C015.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9df4f07534688b618ef1b83a3e8ca35d3688e2a3" target='_blank'>
              Abstract C015: Unraveling the structure of the Philadelphia chromosome using super-resolution microscopy
              </a>
            </td>
          <td>
            Gabrielle S. Dewson, Guy Nir
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c904b2f681f70c0b549880d31ccc37b010f98cbc" target='_blank'>
              Characterisation of APOBEC3B-Mediated RNA editing in breast cancer cells reveals regulatory roles of NEAT1 and MALAT1 lncRNAs.
              </a>
            </td>
          <td>
            Chi Zhang, Yu-Jing Lu, Mei Wang, Bingjie Chen, Feifei Xiong, C. Mitsopoulos, Olivia Rossanese, Xiuling Li, Paul A Clarke
          </td>
          <td>2024-09-25</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3768b90e9608cd8ea1042d3a6a3887d2d3b11d92" target='_blank'>
              Therapeutic Potential of PLK1, KIF4A, CDCA5, UBE2C, CDT1, SKA3, AURKB, and PTTG1 Genes in Triple-Negative Breast Cancer: Correlating Their Expression with Sensitivity to GSK 461364 and IKK 16 Drugs.
              </a>
            </td>
          <td>
            Najmeh Bashari, Mohammadamin Naghizadeh, Mehrnaz Kalhor Chegini, Ensieh Sagheb Sadeghi, Atefeh Zamani, Mohammad Mahdevar
          </td>
          <td>2024-08-30</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be99bcdcbed83ecbe631677cc9b06e583bb332af" target='_blank'>
              Identification of key ferroptosis genes and subtypes in kidney renal clear cell carcinoma
              </a>
            </td>
          <td>
            Biao Wang, Zhuo Wei, Man Xu, H. Shu, Zheqi Fan
          </td>
          <td>2024-09-27</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f67122b5eabe15ca9c58426b09dad3dd570754" target='_blank'>
              Integrative genomics unveils basement membrane-related diagnostic markers and therapeutic targets in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Han Zhang, Xia Zhang, Zhenyu Huang, Hao Zhang
          </td>
          <td>2024-09-11</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Longevity, the length of an organism's lifespan, is impacted by environmental factors, metabolic processes, and genetic determinants. The base excision repair (BER) pathway is crucial for maintaining genomic integrity by repairing oxidatively modified base lesions. Nei-like DNA Glycosylase 1 (NEIL1), part of the BER pathway, is vital in repairing oxidative bases in G-rich DNA regions, such as telomeres and promoters. Hence, in this comprehensive review, it have undertaken a meticulous investigation of the intricate association between NEIL1 and longevity. The analysis delves into the multifaceted aspects of the NEIL1 gene, its various RNA transcripts, and the diverse protein isoforms. In addition, a combination of bioinformatic analysis is conducted to identify NEIL1 mutations, transcription factors, and epigenetic modifications, as well as its lncRNA/pseudogene/circRNA-miRNA-mRNA regulatory network. The findings suggest that the normal function of NEIL1 is a significant factor in human health and longevity, with defects in NEIL1 potentially leading to various cancers and related syndromes, Alzheimer's disease, obesity, and diabetes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c7b949676b21576a435aa8624516850b916fe0" target='_blank'>
              Decoding the Role of NEIL1 Gene in DNA Repair and Lifespan: A Literature Review with Bioinformatics Analysis.
              </a>
            </td>
          <td>
            Amirhossein Mohajeri Khorasani, Alireza Raghibi, Behzad Haj Mohammad Hassani, Pedram Bolbolizadeh, A. Amali, Mahboubeh Sadeghi, Narges Farshidi, A. Dehghani, P. Mousavi
          </td>
          <td>2024-08-20</td>
          <td>Advanced biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="
 Aberrant expression of repeat RNAs in pancreatic ductal adenocarcinoma (PDAC) mimics viral-like responses with implications for tumor cell state and the surrounding microenvironment. To better understand the relationship of repeat RNAs in human PDAC, we employed the NanoString CosMx™ spatial molecular imaging (SMI) platform, which utilizes a 1,000-plex RNA panel with custom repeat RNA probes targeting long interspersed nuclear element 1 (LINE-1) retrotransposon ORF1 and ORF2, HSATII satellite (SAT) repeat, and two human endogenous retroviruses (HERV-K and HERV-H) in 46 primary tumors. This analysis revealed correlations of high repeat RNA expression with alterations in the epithelial state of PDAC cells and the myofibroblast phenotype in cancer-associated fibroblasts (CAFs). The loss of cellular identity observed with dosing of extracellular vesicles (EVs) and individual repeat RNAs in PDAC and CAF cell culture models points to cell-cell intercommunication involving these viral-like elements. Differences in the PDAC and CAF response are driven by distinct innate immune signaling pathways through interferon regulatory transcription factor 3 (IRF3). Altogether, our data indicate that cell context-specific viral-like responses driven by tumor cells have broad impacts on single-cell heterogeneity within cancer cells and the pancreatic cancer microenvironment.
 Citation Format: Eunae You, Patrick Danaher, Chenyue Lu, Siyu Sun, Luli Zou, Ildiko Phillips, Alexandra Rojas, Natalie Ho, Yuhui Song, Michael Raabe, Katherine Xu, Peter Richieri, Hao Li, Natalie Aston, Rebecca Porter, Bidish Patel, Linda Nieman, Nathan Schurman, Briana Hudson, Khrystyna North, Sarah Church, Vikram Deshpande, Andrew Liss, Tae Kim, Yi Cui, Youngmi Kim, Benjamin Greenbaum, Martin Aryee, David Ting. Repeat RNA mediated disruption of cellular plasticity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A039.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffeaf733e03a440a3ea9633bad9509d48160c804" target='_blank'>
              Abstract A039: Repeat RNA mediated disruption of cellular plasticity in pancreatic cancer
              </a>
            </td>
          <td>
            Eunae You, Patrick Danaher, Chenyue Lu, Siyu Sun, Luli Zou, Ildiko E. Phillips, Alexandra Rojas, Natalie Ho, Yuhui Song, Michael J. Raabe, Katherine Xu, Peter Richieri, Hao Li, Natalie Aston, Rebecca L. Porter, Bidish Kumar Patel, Linda T. Nieman, Nathan Schurman, Briana Hudson, Khrystyna North, Sarah Church, Vikram Deshpande, Andrew S Liss, T. Kim, Yi Cui, Youngmi Kim, Benjamin Greenbaum, Martin J. Aryee, David Ting
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Objective
Breast cancer (BC) is highly heterogeneous and one of the most common cancers. Luminal A (LUM A) is a subtype of BC with a better prognosis than other BC subtypes. The molecular mechanisms underlying the initiation and progression of the LUM A subtype are still unclear. Big data generated from microarray and sequencing systems can be re-analyzed, especially with the help of various in silico tools developed in recent years, and made applicable for in vitro and in vivo research. This work aimed to identify genes that may play a role in the progression of LUM A subtype of BC using both computational and laboratory-based methods.


Materials and Methods
Overlapping genes associated with BC were identified from the The Cancer Genome Atlas database, GSE233242, GSE100925 geodata sets, and the geneshot tool. The network functional analysis between overlapping genes was determined with STRING 12.0. Expression levels of overlapping genes in BC were investigated with the TNMplot (https://tnmplot.com/analysis/) in silico tool. The effect of overlapping genes on the overall survival of LUM A cancer patients was defined using the Kaplan-Meier plotter tool. Expressions of genes identified using bioinformatics data were investigated via quantitative real-time -polymerase chain reaction (qRT-PCR) in LUM A tumor and adjacent tissue samples. The data were evaluated using the t-test. Both the sensitivity and specificity of selected genes have been determined using the receiver operating characteristic curve.


Results
In silico investigation showed that eleven genes were possibly associated with BC. Among them CDC25A, AURKB, and TOP2A were considerably increased in LUM A samples according to qRT-PCR results. An overall survival analysis also showed that overexpression of these three genes could reduce the overall survival of LUM A patients.


Conclusion
The genes CDC25A, AURKB, and TOP2A may play crucial functions in LUM A pathogenesis. Therapeutic strategies that diminish the expression of these connected genes may enhance the prognosis of LUM A patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7caf276dd425f22049d300f90bf56bf22e13592" target='_blank'>
              Overexpression of CDC25A, AURKB, and TOP2A Genes Could Be an Important Clue for Luminal A Breast Cancer.
              </a>
            </td>
          <td>
            Murat Kaya, Asmaa Abuaisha, İlknur Süer, Melike Sultan Alptekin, Fahrunnisa Abanoz, S. Emiroğlu, S. Palanduz, K. Çefle, Ş. Öztürk
          </td>
          <td>2024-09-26</td>
          <td>European journal of breast health</td>
          <td>0</td>
          <td>21</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [17],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>