<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-05 12:46:52 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>20</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>50</td>
          <td>130</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>98</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f30ff545a66d3e70df32a4eb65b39b6027a1e6f0" target='_blank'>
                Extrachromosomal circular DNA drives oncogenic genome remodeling in neuroblastoma
                </a>
              </td>
          <td>
            R. Koche, Elias Rodriguez-Fos, K. Helmsauer, M. Burkert, Ian C. MacArthur, J. Maag, Rocio Chamorro, Natalia Munoz-Perez, Montserrat Puiggrós, H. Dorado García, Yi Bei, Claudia Röefzaad, V. Bardinet, A. Szymansky, A. Winkler, Theresa M. Thole, Natalie Timme, K. Kasack, Steffen Fuchs, F. Klironomos, N. Thiessen, E. Blanc, Karin Schmelz, A. Künkele, P. Hundsdörfer, Carolina Rosswog, J. Theißen, D. Beule, H. Deubzer, S. Sauer, J. Toedling, M. Fischer, F. Hertwig, R. Schwarz, A. Eggert, D. Torrents, J. Schulte, A. Henssen
          </td>
          <td>2019-12-16</td>
          <td>Nature Genetics</td>
          <td>197</td>
          <td>61</td>

            <td><a href='../recommendations/f30ff545a66d3e70df32a4eb65b39b6027a1e6f0' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, Jens Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>268</td>
          <td>130</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>233</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA), an extrachromosomal circular structured DNA, is extensively found in eukaryotes. Exploring eccDNA at the single-cell level contributes to understanding heterogeneity, evolution, development, and specific functions within cells. Nevertheless, the high-throughput identification methods for single-cell eccDNA are complex, and currently mature and widely applicable technologies are lacking. Those factors have led to a scarcity of resources for studying eccDNA at the single-cell level. Therefore, using available single-cell whole-genome sequencing (WGS) data, we have constructed a comprehensive eccDNA database named scEccDNAdb (https://lcbb.swjtu.edu.cn/scEccDNAdb/). Presently, scEccDNAdb comprises 3,195,464 single-cell eccDNA entries from both disease/health human and mouse samples, which provides comprehensive annotations including oncogenes, typical enhancers, super-enhancers, CTCF binding sites, SNPs, chromatin accessibility, eQTLs, transcription factor binding sites, motifs, and SVs. Additionally, it provides nine online analysis and visualization tools, facilitating the generation of publication-quality figures for eccDNA analysis through the upload of customized files. Overall, scEccDNAdb represents the first comprehensive database known to us for exploring and analyzing single-cell eccDNA data in diverse cell types, tissues, and species. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322c06c5550a609619dd96de2cba0b3494554b2a" target='_blank'>
              scEccDNAdb: an integrated single-cell eccDNA resource for human and mouse
              </a>
            </td>
          <td>
            Wenqing Wang, Xinyu Zhao, Tianyu Ma, Tengwei Zhong, Junnuo Zheng, Yi Yang, Yuanbiao Guo, Zhiyun Guo
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5988602f52499713b4d1d42d555664a971cb3e9" target='_blank'>
              3D genomic analysis reveals novel enhancer-hijacking caused by complex structural alterations that drive oncogene overexpression
              </a>
            </td>
          <td>
            Katelyn L. Mortenson, Courtney Dawes, Emily R. Wilson, Nathan E. Patchen, Hailey E Johnson, Jason Gertz, Swneke D. Bailey, Yang Liu, K. Varley, Xiaoyang Zhang
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a040a0cb95cb5f3026b82c4853a3fbf7dc09e9f" target='_blank'>
              Structure and transcription of integrated HPV DNA in vulvar carcinomas
              </a>
            </td>
          <td>
            A. Van Arsdale, Lauren Turker, Yoke-Chen Chang, Joshua Gould, Bryan Harmon, E. Maggi, Olga Meshcheryakova, Maxwell P Brown, Dana Luong, Koenraad Van Doorslaer, Mark H. Einstein, Dennis Y. S. Kuo, Deyou Zheng, Brian J. Haas, Jack Lenz, Cristina Montagna
          </td>
          <td>2024-06-19</td>
          <td>NPJ Genomic Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="Somatic structural variations (SVs) in cancer can shuffle DNA content in the genome, relocate regulatory elements, and alter genome organization. Enhancer hijacking occurs when SVs relocate distal enhancers to activate proto-oncogenes. However, most enhancer hijacking studies have only focused on protein-coding genes. Here, we develop a computational algorithm 'HYENA' to identify candidate oncogenes (both protein-coding and non-coding) activated by enhancer hijacking based on tumor whole-genome and transcriptome sequencing data. HYENA detects genes whose elevated expression is associated with somatic SVs by using a rank-based regression model. We systematically analyze 1146 tumors across 25 types of adult tumors and identify a total of 108 candidate oncogenes including many non-coding genes. A long non-coding RNA TOB1-AS1 is activated by various types of SVs in 10% of pancreatic cancers through altered 3-dimensional genome structure. We find that high expression of TOB1-AS1 can promote cell invasion and metastasis. Our study highlights the contribution of genetic alterations in non-coding regions to tumorigenesis and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fb0cf8831bea2891dbca4714836cde381f2ca0" target='_blank'>
              HYENA detects oncogenes activated by distal enhancers in cancer.
              </a>
            </td>
          <td>
            Anqi Yu, Ali E. Yesilkanal, Ashish Thakur, Fan Wang, Yang Yang, William Phillips, Xiaoyang Wu, Alexander Muir, Xin He, Francois Spitz, Lixing Yang
          </td>
          <td>2024-07-25</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/795e6a16077e23dab48be945dcd0a59b697bb23e" target='_blank'>
              Mechanism of ERBB2 gene overexpression by the formation of super-enhancer with genomic structural abnormalities in lung adenocarcinoma without clinically actionable genetic alterations
              </a>
            </td>
          <td>
            S. Kaneko, Ken Takasawa, Ken Asada, K. Shiraishi, Noriko Ikawa, Hidenori Machino, Norio Shinkai, M. Matsuda, Mari Masuda, Shungo Adachi, Satoshi Takahashi, Kazuma Kobayashi, Nobuji Kouno, Amina Bolatkan, M. Komatsu, M. Yamada, M. Miyake, Hirokazu Watanabe, A. Tateishi, T. Mizuno, Y. Okubo, Masami Mukai, Tatsuya Yoshida, Yukihiro Yoshida, H. Horinouchi, Shun-ichi Watanabe, Y. Ohe, Yasushi Yatabe, V. Saloura, Takashi Kohno, R. Hamamoto
          </td>
          <td>2024-06-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="INTRODUCTION
Isodicentric Y chromosomes are relatively common structural variants of the human genome. The underlying mechanism of isodicentric Y chromosomes with short arm breakpoints [idic(Yq)] remains to be clarified.


CASE PRESENTATION
We encountered a Japanese man with azoospermia and mild short stature. G-banding and array-based comparative genomic hybridization indicated that his karyotype was 45,X/46,X,idic(Y)(qter→p11.32::p11.32→qter) with a ~1.8 Mb terminal deletion. Whole genome sequencing suggested that the Y chromosome had four breakpoints in a ~7 kb region of the pseudoautosomal region 1 (PAR1).


CONCLUSION
This case was assumed to have an idic(Yq) resulting from multiple DNA double-strand breaks in PAR1. This rearrangement may have been facilitated by the PAR1-specific chromatin architecture. The clinical features of the patient can be ascribed to SHOX haploinsufficiency and the presence of a 45,X cell line, although copy-number gains of some Yq genes and the size reduction of PAR1 may also contribute to his spermatogenic failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6ffa4553f26785a976ed64f6e1aecf9e157203" target='_blank'>
              Isodicentric Y chromosome with multiple breakpoints in the pseudoautosomal region 1.
              </a>
            </td>
          <td>
            Yasuko Ogiwara, Y. Kobori, E. Suzuki, Atsushi Hattori, Kanako Tanase-Nakao, Akiyoshi Osaka, Toshiyuki Iwahata, Hiroshi Okada, Y. Kuroki, M. Fukami
          </td>
          <td>2024-07-29</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Breast cancer (BCa) is a major global health challenge, characterized by chromosomal instability (CIN) and subsequent acquisition of extensive somatic copy number alterations (CNAs). CNAs including amplifications and deletions, significantly influence intra-tumor heterogeneity and the tumor microenvironment (TME). Among these, the loss of chromosome 13q14.2 emerges as a considerable factor in BCa pathogenesis and treatment responses. We provide evidence that this genomic alteration is under positive selective pressure, correlating with poorer patient survival. Furthermore, through multi-omic and in vitro analyses, we uncover a dual role of 13q14.2 loss: it confers a survival advantage to tumor cells and modulate the cell cycle and pro-apoptotic pathways in cancer cells, affecting macrophages population in the TME, while paradoxically increasing tumor susceptibility to BCL2 inhibitors. These findings suggest that targeting 13q14.2 as a biomarker in BCa could enhance the efficacy of existing treatments and offer a new avenue for improving clinical outcomes in BCa. Figure 1. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e388fe8f21125bbefce3cee2961f295d519ae77" target='_blank'>
              Loss of chromosome cytoband 13q14.2 orchestrates breast cancer pathogenesis and drug response
              </a>
            </td>
          <td>
            Parastoo Shahrouzi, Youness Azimzade, Wioletta Brankiewicz, Sugandha Bhatia, David Kunke, Derek Richard, Xavier Tekpli, Vessela N Kristensen, Pascal H.G. Duijf
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Ovarian cancer is a deadly female cancer with high rates of recurrence. The primary treatment strategy for patients is platinum-based therapy regimens that almost universally develop resistance. Consequently, new therapeutic avenues are needed to overcome the plateau that current therapies have on patient outcomes. We describe a gene amplification involving both HSF1 and MYC, wherein these two genes on chromosome 8q are co-amplified in over 7% of human tumors that is enriched to over 30% of patients with ovarian cancer. We further found that HSF1 and MYC transcriptional activity is correlated in human tumors and ovarian cancer cell lines, suggesting they may cooperate in ovarian cancer cells. CUT&RUN for HSF1 and MYC in co-amplified ovarian cancer cells revealed that HSF1 and MYC have overlapping binding at a substantial number of locations throughout the genome where their binding peaks are near identical. Consistent with these data, a protein-protein interaction between HSF1 and MYC was detected in ovarian cancer cells, implying these two transcription factors have a molecular cooperation. Further supporting their cooperation, growth of HSF1-MYC co-amplified ovarian cancer cells were found to be dependent on both HSF1 and MYC. In an attempt to identify a therapeutic target that could take advantage of this dependency on both HSF1 and MYC, PLK1 was identified as being correlated with HSF1 and MYC in primary human tumor specimens, consistent with a previously established effect of PLK1 on HSF1 and MYC protein levels. Targeting PLK1 with the compound volasertib (BI-6727) revealed a greater than 200-fold increased potency of volasertib in HSF1-MYC co-amplified ovarian cancer cells compared to ovarian cancer cells wild-type HSF1 and MYC copy number, which extended to several growth assays, including spheroid growth. Volasertib, and other PLK1 inhibitors, have not shown great success in clinical trials and this study suggests that targeting PLK1 may be viable in a precision medicine approach using HSF1-MYC co-amplification as a biomarker for response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e2e5303aa923a4a45241efa9a5394dc2df5936" target='_blank'>
              MYC and HSF1 Cooperate to Drive PLK1 inhibitor Sensitivity in High Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Imade Williams, Haddie DeHart, Matthew O’Malley, Bobby Walker, Vrushabh Ulhaskumar, Haimanti Ray, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2024-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Human tumors progress in part by accumulating epigenetic alterations, which include gains and losses of DNA methylation in different parts of the cancer cell genome. Recent work has revealed a link between these two opposite alterations by showing that DNA hypomethylation in tumors can induce the expression of transcripts that overlap downstream gene promoters and thereby induce their hypermethylation. Preliminary in silico evidence prompted us to investigate if this mechanism applies to the locus harboring AGO1, a gene that plays a central role in miRNA biogenesis and RNA interference. Inspection of public RNA-Seq datasets and RT-qPCR experiments show that an alternative transcript starting 13.4 kb upstream of AGO1 (AGO1-V2) is expressed specifically in testicular germ cells, and becomes aberrantly activated in different types of tumors, particularly in tumors of the esophagus, stomach, and lung. This expression pattern classifies AGO1-V2 into the group of “Cancer-Germline” (CG) genes. Analysis of transcriptomic and methylomic datasets provided evidence that transcriptional activation of AGO1-V2 depends on DNA demethylation of its promoter region. Western blot experiments revealed that AGO1-V2 encodes a shortened isoform of AGO1, corresponding to a truncation of 75 aa in the N-terminal domain, and which we therefore referred to as “∆NAGO1”. Interestingly, significant correlations between hypomethylation/activation of AGO1-V2 and hypermethylation/repression of AGO1 were observed upon examination of tumor cell lines and tissue datasets. Overall, our study reveals the existence of a process of interdependent epigenetic alterations in the AGO1 locus, which promotes swapping between two AGO1 protein-coding mRNA isoforms in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27dc2f1e01e40a64b0f8910cba78bb0f22899826" target='_blank'>
              DNA Hypomethylation Underlies Epigenetic Swapping between AGO1 and AGO1-V2 Isoforms in Tumors
              </a>
            </td>
          <td>
            Jean S. Fain, Camille Wangermez, A. Loriot, Claudia Denoue, C. De Smet
          </td>
          <td>2024-06-22</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Transcriptional enhancers can regulate individual or multiple genes through long-range three-dimensional (3D) genome interactions, and these interactions are commonly altered in cancer. Yet, the functional relationship between changes in 3D interactions associated with regulatory regions and differential gene expression appears context-dependent. In this study, we used HiChiP to capture changes in 3D genome interactions between active regulatory regions of endometrial cancer cells in response to estrogen treatment and uncovered significant differential long-range interactions that are strongly enriched for estrogen receptor α (ER) bound sites (ERBS). The ERBS anchoring differential loops with either a gene’s promoter or distal regions were correlated with larger transcriptional responses to estrogen compared to ERBS not involved in differential interactions. To functionally test this observation, CRISPR-based Enhancer-i was used to deactivate specific ERBS, which revealed a wide range of effects on the transcriptional response to estrogen. However, these effects are only subtly and not significantly stronger for ERBS in differential loops. In addition, we observed an enrichment of 3D interactions between the promoters of estrogen up-regulated genes and found that looped promoters can work together cooperatively. Overall, our work suggests that changes in 3D genome structure upon estrogen treatment identify some functionally important regulatory regions; however, these changes aren’t required for a transcriptional response to E2 in endometrial cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d0d32b1b8568f5b7b8778716b0bae7e2d53785" target='_blank'>
              Estrogen-induced chromatin looping changes identify a subset of functional regulatory elements
              </a>
            </td>
          <td>
            Hosiana Abewe, Alexandra Richey, Jeffery M. Vahrenkamp, Matthew Ginley-Hidinger, Craig M Rush, Noel Kitchen, Xiaoyang Zhang, Jason Gertz
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Cancer is a disease underpinned by aberrant gene expression. Enhancers are regulatory elements that play a major role in transcriptional control and changes in active enhancer function are likely critical in the pathogenesis of oesophageal adenocarcinoma (OAC). Here, we utilise STARR-seq to profile the genome-wide enhancer landscape in OAC and identify hundreds of high-confidence enhancer elements. These regions are enriched in enhancer-associated chromatin marks, are actively transcribed and exhibit high levels of associated gene activity in OAC cells. These characteristics are maintained in human patient samples, demonstrating their disease relevance. This relevance is further underlined by their responsiveness to oncogenic ERBB2 inhibition and increased activity compared to the pre-cancerous Barrett’s state. Mechanistically, these enhancers are linked to the core OAC transcriptional network and in particular KLF5 binding is associated with high level activity, providing further support for a role of this transcription factor in defining the OAC transcriptome. Our results therefore uncover a set of enhancer elements with physiological significance, that widen our understanding of the molecular alterations in OAC and point to mechanisms through which response to targeted therapy may occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f476e2900716d7fa8579f8b85d850f7a35ffa1" target='_blank'>
              Massively Parallel Reporter Assays identify enhancer elements in Oesophageal Adenocarcinoma
              </a>
            </td>
          <td>
            SH Yang, Y. Li, CW Bleaney, AD Sharrocks, Andrew Sharrocks
          </td>
          <td>2024-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Aberrant epigenetic regulation is a hallmark of Diffuse Midline Glioma (DMG), an incurable pediatric brain tumor. The H3K27M driver histone mutation leads to transcriptional dysregulation, indicating that targeting the epigenome and transcription may be key therapeutic strategies against this highly aggressive cancer. One such target is the Facilitates Chromatin Transcription (FACT) histone chaperone. We found FACT to be enriched at developmental gene promoters, coinciding with regions of open chromatin and binding motifs of core DMG regulatory transcription factors. Furthermore, FACT interacted and co-localized with the Bromodomain and Extra-Terminal Domain (BET) protein BRD4 at promoters and enhancers, suggesting functional cooperation between FACT and BRD4 in DMG. In vitro, a combinatorial therapeutic approach using the FACT inhibitor CBL0137, coupled with BET inhibition revealed potent and synergistic cytotoxicity across a range of DMG cultures, with H3K27M-mutant cells demonstrating heightened sensitivity. These results were recapitulated in vivo, significantly extending survival in three independent orthotopic PDX models of DMG. Mechanistically, we show that CBL0137 treatment decreased chromatin accessibility, synergizing with BET inhibition to disrupt transcription, silencing several key oncogenes including MYC, PDGFRA and MDM4, as well as causing alterations to the splicing landscape. Combined, these data highlight the therapeutic promise of simultaneously targeting FACT and BRD4 in DMG, proposing a novel strategy for combating this devastating pediatric brain tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6273ba39a45e304340c6436c0631345d003e75f3" target='_blank'>
              Chromatin remodeling with combined FACT and BET inhibition disrupts oncogenic transcription in Diffuse Midline Glioma
              </a>
            </td>
          <td>
            Holly Holliday, Aaminah Khan, A. Ehteda, Samuel E. Ross, Nisitha Jayatilleke, A. Gopalakrishnan, Xinyi Guo, Jie Liu, Hieu Nguyen, Alvin Lee, Yolanda Colino Sanguino, Marcel E. Dinger, C. Mayoh, Benjamin Rayner, M. Tsoli, David S. Ziegler
          </td>
          <td>2024-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, Andre Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4781c7d248ebd4c7b1d88e8451d7cdc1606f06e6" target='_blank'>
              Genome-wide enhancer RNA profiling adds molecular links between genetic variation and human cancers
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Bin Li, Jinyu Huang, Ming Zhang, Can Chen, Linyun Fan, Qian-Ying Ma, Chunyi He, Shuo Chen, Yuan Jiang, Yan-Min Li, Caibo Ning, Fu-Wei Zhang, Wenzhuo Wang, Yizhuo Liu, Heng Zhang, Meng Jin, Xiao-Yang Wang, Jinxin Han, Zhen Xiong, Ming Cai, Chaoqun Huang, Xiaojun Yang, Xu Zhu, Ying Zhu, Xiaoping Miao, Shao-Kai Zhang, Yongchang Wei, Jianbo Tian
          </td>
          <td>2024-06-11</td>
          <td>Military Medical Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33af20a7d0aac05ac83adef2c51ac9ffcee3995c" target='_blank'>
              Enhancer engagement sustains oncogenic transformation and progression of B-cell precursor acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            G. Corleone, C. Sorino, M. Caforio, S. Di Giovenale, F. de Nicola, F. Goeman, V. Bertaina, A. Pitisci, C. Cortile, Franco Locatelli, V. Folgiero, Maurizio Fanciulli
          </td>
          <td>2024-06-27</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Chromosomal translocations in non-Hodgkin lymphoma (NHL) result in activation of oncogenes by placing them under the regulation of immunoglobulin heavy chain (IGH) super-enhancers. Aberrant expression of translocated oncogenes induced by enhancer activity can contribute to lymphomagenesis. The role of the IGH enhancers in normal B-cell development is well established, but knowledge regarding the precise mechanisms of their involvement in control of the translocated oncogenes is limited. The goal of this project was to define the critical regions in the IGH regulatory elements and identify enhancer RNAs (eRNA). We designed a sgRNA library densely covering the IGH enhancers and performed tiling CRISPR interference screens in three NHL cell lines. This revealed three regions crucial for NHL cell growth. With chromatin-enriched RNA-Seq we showed transcription from the core enhancer regions and subsequently validated expression of the eRNAs in a panel of NHL cell lines and tissue samples. Inhibition of the essential IGH enhancer regions decreased expression of eRNAs and translocated oncogenes in several NHL cell lines. The observed expression and growth patterns were consistent with the breakpoints in the IGH locus. Moreover, targeting the Eμ enhancer resulted in loss of B-cell receptor expression. In a Burkitt lymphoma cell line, MYC overexpression partially rescued the phenotype induced by IGH enhancer inhibition. Our results indicated the most critical regions in the IGH enhancers and provided new insights into the current understanding of the role of IGH enhancers in B-cell NHL. As such, this study forms a basis for development of potential therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/801fa2579edbf74a0b66ac42b4a2406cc3439a8b" target='_blank'>
              Core regions in immunoglobulin heavy chain enhancers essential for survival of non-Hodgkin lymphoma cells are identified by a CRISPR interference screen.
              </a>
            </td>
          <td>
            Marta elzbieta Kasprzyk, Weronika Sura, M. Podralska, Marta Kazimierska, Annika Seitz, Wojciech Łosiewski, Tomasz Woźniak, Jeroen E J Guikema, A. Diepstra, Joost L. Kluiver, Anke van den Berg, N. Rozwadowska, A. Dzikiewicz-Krawczyk
          </td>
          <td>2024-06-27</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Many essential functions of organisms are encoded in highly repetitive genomic regions, including histones involved in DNA packaging, centromeres that are core components of chromosome segregation, ribosomal RNA comprising the protein translation machinery, telomeres that ensure chromosome integrity, piRNA clusters encoding host defenses against selfish elements, and virtually the entire Y chromosome. These regions, formed by highly similar tandem arrays, pose significant challenges for experimental and informatic study, impeding sequence-level descriptions essential for understanding genetic variation. Here, we report the assembly and variation analysis of such repetitive regions in Drosophila melanogaster, offering significant improvements to the existing community reference assembly. Our work successfully recovers previously elusive segments, including complete reconstructions of the histone locus and the pericentric heterochromatin of the X chromosome, spanning the Stellate locus to the distal flank of the rDNA cluster. To infer structural changes in these regions where alignments are often not practicable, we introduce landmark anchors based on unique variants that are putatively orthologous. These regions display considerable structural variation between different D. melanogaster strains, exhibiting differences in copy number and organization of homologous repeat units between haplotypes. In the histone cluster, although we observe minimal genetic exchange indicative of crossing over, the variation patterns suggest mechanisms such as unequal sister chromatid exchange. We also examine the prevalence and scale of concerted evolution in the histone and Stellate clusters and discuss the mechanisms underlying these observed patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d432b790a64d5e41e24bfb91fe60924d015814eb" target='_blank'>
              Genetic variation in recalcitrant repetitive regions of the Drosophila melanogaster genome
              </a>
            </td>
          <td>
            Harsh G. Shukla, Mahul Chakraborty, J. Emerson
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer.
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8409b669072e4562ce558ad50b151f454f90ead6" target='_blank'>
              An oncogenic enhancer promotes melanoma progression via regulating ETV4 expression
              </a>
            </td>
          <td>
            Junyou Zhang, Qilin Wang, Sihan Qi, Yingying Duan, Zhaoshuo Liu, Jiaxin Liu, Ziyi Zhang, Chunyan Li
          </td>
          <td>2024-06-07</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Recently, biomolecular condensates formed through liquid–liquid phase separation have been widely reported to regulate key intracellular processes involved in cell biology and pathogenesis. BRD4 is a nuclear protein instrumental to the establishment of phase-separated super-enhancers (SEs) to direct the transcription of important genes. We previously observed that protein droplets of BRD4 became hydrophobic as their size increase, implying an ability of SEs to limit the ionization of water molecules by irradiation. Here, we aim to establish if SEs confer radiation resistance in cancer cells. We established an in vitro DNA damage assay that measures the effect of radicals provoked by the Fenton reaction on DNA integrity. This revealed that DNA damage was markedly reduced when BRD4 underwent phase separation with DNA. Accordingly, co-focal imaging analyses revealed that SE foci and DNA damage foci are mutually exclusive in irradiated cells. Lastly, we observed that the radioresistance of cancer cells was significantly reduced when irradiation was combined with ARV-771, a BRD4 de-stabilizer. Our data revealed the existence of innately radioresistant genomic regions driven by phase separation in cancer cells. The disruption of these phase-separated components enfolding genomic DNA may represent a novel strategy to augment the effects of radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e2dfd016fdeaf5bbe54441d068b3a7b5c2bf81" target='_blank'>
              Phase-separated super-enhancers confer an innate radioresistance on genomic DNA
              </a>
            </td>
          <td>
            Koki Matsumoto, Dini Kurnia Ikliptikawati, Keiichi Makiyama, Kako Mochizuki, Maho Tobita, Isao Kobayashi, D. Voon, Keesiang Lim, Kazuma Ogawa, I. Kashiwakura, Hiroshi I Suzuki, H. Yoshino, Richard W. Wong, M. Hazawa
          </td>
          <td>2024-06-14</td>
          <td>Journal of Radiation Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="ABSTRACT Aneuploidy, the presence of an aberrant number of chromosomes, has been associated with tumorigenesis for over a century. More recently, advances in karyotyping techniques have revealed its high prevalence in cancer: About 90% of solid tumors and 50–70% of hematopoietic cancers exhibit chromosome gains or losses. When analyzed at the level of specific chromosomes, there are strong patterns that are observed in cancer karyotypes both pan-cancer and for specific cancer types. These specific aneuploidy patterns correlate strongly with outcomes for tumor initiation, progression, metastasis formation, immune evasion and resistance to therapeutic treatment. Despite their prominence, understanding the basis underlying aneuploidy patterns in cancer has been challenging. Advances in genetic engineering and bioinformatic analyses now offer insights into the genetic determinants of aneuploidy pattern selection. Overall, there is substantial evidence that expression changes of particular genes can act as the positive selective forces for adaptation through aneuploidy. Recent findings suggest that multiple genes contribute to the selection of specific aneuploid chromosomes in cancer; however, further research is necessary to identify the most impactful driver genes. Determining the genetic basis and accompanying vulnerabilities of specific aneuploidy patterns is an essential step in selectively targeting these hallmarks of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0123dd290f56b3193001c1283aea1071e6f411ed" target='_blank'>
              Selection forces underlying aneuploidy patterns in cancer
              </a>
            </td>
          <td>
            Tamara C. Klockner, Christopher S. Campbell
          </td>
          <td>2024-06-24</td>
          <td>Molecular & Cellular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c57c70b2a1bb317c90f2cc0de26bf7a8a7418b" target='_blank'>
              A microdeletion event at 19q13.43 in IDH-mutant astrocytomas is strongly correlated with MYC overexpression
              </a>
            </td>
          <td>
            Ege Ülgen, U. Gerlevik, Sila Gerlevik, Yavuz Oktay, O. U. Sezerman, Şevin Turcan, K. Ozduman
          </td>
          <td>2024-06-14</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 We sought to identify mechanisms that activate MYC expression in germinal center B cell-(GCB) diffuse large B-cell lymphomas (DLBCL) in the absence of MYC rearrangement. High-throughput CRISPR-interference screens in MYC-rearranged (n=4) and non-MYC-rearranged (n=2) GCB-DLBCL cell lines revealed a distal enhancer complex 436 kb downstream of MYC, GME-1, that was uniquely essential in non-rearranged (MYC-intact) cell lines. CRISPR-interference studies confirmed growth inhibition and MYC downregulation upon GME-1 repression in 4 DLBCL cell lines. ChIP-Seq showed that GME-1 subunits were bound by OCT2, OCA-B, and MEF2B, which are known to activate germinal center-specific enhancers, and knockdown of the corresponding genes reduced MYC expression. GME-1 subunits showed GCB-specific acetylation and accessibility across normal human B cell populations in public chromatin datasets. The mouse ortholog of GME-1 showed germinal center B cell-specific accessibility. GME-1 was strongly acetylated in MYC-intact but not MYC-rearranged GCB-DLBCL cell lines and patient biopsies. Hi-C revealed significant looping of GME-1 to the MYC promoter in MYC protein+, MYC-intact DLBCL biopsies (n=2), but not MYC-rearranged DLBCL (n=2). We used sequence conservation and chromatin accessibility data to define four GME-1 subunits (195A/B and196A/B), which were cloned into luciferase enhancer reporter constructs and compared to two subunits of the BCL6 distal super-enhancer complex that had previously been identified as regulated by the OCT2/OCA-B/MEF2B complex. GME-1 subunit 195B was the only strongly transcription-activating element based on luciferase assays across three MYC-intact cells lines, where it showed activity comparable to BCL6 enhancers. In contrast, GME-1 enhancers were minimally active in reporter assays performed in two MYC-rearranged GCB-DLBCL cell lines. Deletion of a conserved OCT2 binding site or knockdown of the POU2F2 and POU2AF1 genes reduced activity of GME-1. Thus GME-1 trans-regulation is divergent from that of previously characterized GCB enhancers, despite shared dependency on the OCT2 ternary complex. Whole-genome sequencing (WGS) revealed that the GME-1 dependent cell line SUDHL5 bears a 176bp tandem duplication within GME-1 195B. This altered enhancer showed stronger luciferase reporter activity across three MYC-intact cell lines. Published WGS datasets showed isolated genomic tandem copy gains of GME-1 in 2/49 biopsies from recurrent/refractory DLBCL and 1/93 unselected DLBCL. In a cohort of 18 DLBCL with the high-risk DZSig gene expression signature, GME-1 amplification was seen in two biopsies with concurrent deletion of the PVT1 promoter (PVT1-P) and no inter-chromosomal MYC locus rearrangement. CRISPRi showed that PVT1-P repression reduced growth and MYC expression in GME-1-dependent DLBCL, but not in cell lines where rearrangements removed PVT-P from its position in cis with the MYC gene. Our findings identify GME-1 as an activator of MYC expression and target of rare but recurrent genomic copy gains in high-risk DLBCL.
 Citation Format: Aishwarya Gurumurthy, Ashwin R. Iyer, Shih-Chun A. Chu, Marcin P. Cieslik, Russell J.H. Ryan. Characterization of an essential MYC enhancer targeted by focal genomic alterations in diffuse large B-cell lymphoma [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR04.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6db6b409667440c15e28ec2c1062b89225fa14" target='_blank'>
              Abstract PR04: Characterization of an essential MYC enhancer targeted by focal genomic alterations in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Aishwarya Gurumurthy, Ashwin Iyer, Shih-Chun A. Chu, Marcin P. Cieslik, R. Ryan
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Recurrent mutations in serine / arginine-rich splicing factor 2 (SRSF2), particularly the proline-to-histidine substitution at position 95 (P95H), have been proposed to drive neoplastic diseases. SRSF2 plays pivotal roles in pre-mRNA processing and gene transcription. However, the precise impact of these diverse functions of SRSF2 on cancer cell behaviour remains unclear. Here, we show that deletion or homozygous P95H mutation of SRSF2 both cause extensive DNA damage leading to cell cycle arrest in vitro and in vivo, regardless of cell genotype. We mechanistically demonstrate that SRSF2 is required for efficient bi-directional transcription of DNA replication and repair genes, independent of its function in splicing. In contrast, SRSF2 haploinsufficiency induces DNA damage without halting the cell cycle in cancer cells. Inducing the heterozygous Srsf2 P95H mutation leads to clonal expansion of epidermal cells in mouse skin, but tumor formation is inhibited following exposure to carcinogens. To survive carcinogen treatment, cells containing the Srsf2 P95H mutation undergo substantial transcription rewiring that restores bi-directional gene expression. Thus, our study reveals crucial roles of SRSF2 beyond splicing and underscores its importance in regulating transcription to orchestrate the cell cycle along with the DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32907ead7270689c743bba801c421a14be049a66" target='_blank'>
              SRSF2 transcriptional function maintains genome integrity during cell division
              </a>
            </td>
          <td>
            Rebecca E. Wagner, Leonie Arnetzl, Thiago Britto-Borges, Anke Heit-Mondrzyk, Ali Bakr, Etienne Sollier, Jasper Panten, Sylvain Delaunay, Daniela Sohn, P. Schmezer, D. T. Odom, Karin Müller-Decker, C. Plass, Christoph Dieterich, P. Lutsik, Susanne Bornelöv, Michaela Frye
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0a5950869ddd6a21b7f0f323fc01ad3c1371ba9" target='_blank'>
              Altered chromatin topologies caused by balanced chromosomal translocation lead to central iris hypoplasia
              </a>
            </td>
          <td>
            Wenmin Sun, Dan Xiong, Jiamin Ouyang, Xueshan Xiao, Yi Jiang, Yingwei Wang, Shi-qiang Li, Ziying Xie, Junwen Wang, Zhonghui Tang, Qingjiong Zhang
          </td>
          <td>2024-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29e5e59bd894b6791e1caba61c558e842d1e8aab" target='_blank'>
              Mitoferrin2 is a synthetic lethal target for chromosome 8p deleted cancers
              </a>
            </td>
          <td>
            Stephan Krieg, Thomas Rohde, Tobias Rausch, Luise Butthof, Lena Wendler-Link, Christoph Eckert, Kai Breuhahn, Bruno Galy, J. Korbel, Maximilian Billmann, Marco Breinig, Darjus F. Tschaharganeh
          </td>
          <td>2024-06-17</td>
          <td>Genome Medicine</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f701cb6b5c83faca69b958611b0964a6f2f858d" target='_blank'>
              Non-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors
              </a>
            </td>
          <td>
            Qing Hu, Jose Espejo Valle-Inclán, Rashmi Dahiya, Alison Guyer, Alice Mazzagatti, Elizabeth G. Maurais, Justin L. Engel, Huiming Lu, Anthony J. Davis, I. Cortés-Ciriano, P. Ly
          </td>
          <td>2024-07-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Single nucleotide variants (SNVs) contribute to cancer by altering the coding and the non-coding regions of the genome. Connecting SNVs to transcriptomic and epigenetic changes at the single-cell level remains challenging. To enable studies of rare cell populations harboring specific point mutations, we developed STAR-FACS, Specific-To-Allele PCR-FACS, to sort cells based on genomic allele alterations. We show that STAR-FACS can separate cells based on TERT promoter mutation status and is compatible with bulk and single-cell transcriptomic and epigenetic profiling. We demonstrate that glioblastoma cell lines derived from the same tumor but harboring distinct TERT promoter SNVs have different transcriptional programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f2ef09f49e055964d96a3768d156e4cf968931" target='_blank'>
              Cell sorting based on single nucleotide variation enables characterization of mutation-dependent transcriptome and chromatin states
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, Oszkar Szentirmai, M. Janiszewska
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Copy number alterations (CNAs), resulting from the gain or loss of genetic material from as little as 50 base pairs or as big as entire chromosome(s), have been associated with many congenital diseases, de novo syndromes and cancer. It is established that CNAs disturb the dosage of genomic regions including enhancers/promoters, long non-coding RNA and gene(s) among others, ultimately leading to an altered balance of key cellular functions. In cancer, CNAs have been associated with almost all steps of the disease: predisposition, initiation, development, maintenance, response to treatment, resistance, and relapse. Therefore, understanding how specific CNAs contribute to tumourigenesis may provide prognostic insight and ultimately lead to the development of new therapeutic approaches to improve patient outcomes. In this review, we provide a snapshot of what is currently known about CNAs and cancer, incorporating topics regarding their detection, clinical impact, origin, and nature, and discuss the integration of innovative genetic engineering strategies, to highlight the potential for targeting CNAs using novel, dosage-sensitive and less toxic therapies for CNA-driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b107cf5f7a3a29e8e0fe916d4102734572eefc3a" target='_blank'>
              Insights into the Clinical, Biological and Therapeutic Impact of Copy Number Alteration in Cancer
              </a>
            </td>
          <td>
            Shannon L. Carey-Smith, R. Kotecha, L. C. Cheung, Sébastien Malinge
          </td>
          <td>2024-06-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer chimeric, or fusion, transcripts are thought to most frequently appear due to chromosomal aberrations that combine moieties of unrelated normal genes. When being expressed, this results in chimeric RNAs having upstream and downstream parts relatively to the breakpoint position for the 5’- and 3’-fusion components, respectively. As many other types of cancer mutations, fusion genes can be of either driver or passenger type. The driver fusions may have pivotal roles in malignisation by regulating survival, growth, and proliferation of tumor cells, whereas the passenger fusions most likely have no specific function in cancer. The majority of research on fusion gene formation events is concentrated on identifying fusion proteins through chimeric transcripts. However, contemporary studies evidence that fusion events involving non-coding RNA (ncRNA) genes may also have strong oncogenic potential. In this review we highlight most frequent classes of ncRNAs fusions and summarize current understanding of their functional roles. In many cases, cancer ncRNA fusion can result in altered concentration of the non-coding RNA itself, or it can promote protein expression from the protein-coding fusion moiety. Differential splicing, in turn, can enrich the repertoire of cancer chimeric transcripts, e.g. as observed for the fusions of circular RNAs and long non-coding RNAs. These and other ncRNA fusions are being increasingly recognized as cancer biomarkers and even potential therapeutic targets. Finally, we discuss the use of ncRNA fusion genes in the context of cancer detection and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec4b199299758974fdcbbc48ab668c481680ff0" target='_blank'>
              Cancer fusion transcripts with human non-coding RNAs
              </a>
            </td>
          <td>
            Tharaa Mohammad, Marianna A. Zolotovskaia, Maria V. Suntsova, Anton Buzdin
          </td>
          <td>2024-06-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Purpose: Complex chromosomal rearrangements (CCRs) are rare structural variants involving three or more chromosomal break- points. Most de novo reported CCRs pose challenges for diagnosis and management. They often require karyotyping, fluorescence in situ hybridization (FISH), and chromosomal microarray analysis (CMA) for clinical diagnosis because of the limitations of each method. Here we report an inherited exceptionally complex CCR involving 4 chromosomes and 11 breakpoints in a family with multisystem anomalies. Methods: We evaluated the CCRs using karyotyping, FISH, CMA, and two emerging genomic technologies: high-throughput chromosome conformation capture sequencing (Hi-C; aka genomic proximity mapping, GPM) and optical genome mapping (OGM). We also performed functional studies using transcriptome and methylome analyses. Results: The proband, who had intellectual disability and immune deficiency, shared CCRs with her unaffected mother involving chromosomes 1, 7, and 11 by karyotyping. However, CMA revealed a duplication and three deletions in the proband in contrast to her mother's balanced genome. Hi-C (GPM) and OGM detected the CCRs and copy number alterations but also uncovered additional breakpoints at high resolution, including an insertion in 4p and two cryptic rearrangements at 7p. Transcriptome and methylome analyses identified likely biological pathways associated with the proband's phenotypes. Conclusion: Combining cytogenetic and genomic methods provided comprehensive characterization and defined the breakpoints at high resolution in both proband and mother. This underscores the value of novel cytogenetic and genomic techniques in deciphering complex genome rearrangements and the significance of integrative genomic analysis and functional characterization in understanding clinical phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d8dc703c96a52057ab6829a72726f61eb74bd56" target='_blank'>
              Genetic and functional characterization of inherited complex chromosomal rearrangements in a family with multisystem anomalies
              </a>
            </td>
          <td>
            H. Fang, S. M. Eacker, Y. Wu, W. Neufeld-Kaiser, M. Laurino, S. Keel, M. S. Horwitz, Y. Liu
          </td>
          <td>2024-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Introduction Field cancerization is suggested to arise from imbalanced differentiation in individual basal progenitor cells leading to clonal expansion of mutant cells that eventually replace the epithelium, although without evidence. Methods We performed deep sequencing analyses to characterize the genomic and transcriptomic landscapes of field change in two patients with synchronous aerodigestive tract tumors. Results Our data support the emergence of numerous genetic alterations in cancer-associated genes but refutes the hypothesis that founder mutation(s) underpin this phenomenon. Mutational signature analysis identified defective homologous recombination as a common underlying mutational process unique to synchronous tumors. Discussion Our analyses suggest a common etiologic factor defined by mutational signatures and/or transcriptomic convergence, which could provide a therapeutic opportunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae797c96e298d92f976b952e9d1ccd0f24f2a970" target='_blank'>
              Transcriptomic convergence despite genomic divergence drive field cancerization in synchronous squamous tumors
              </a>
            </td>
          <td>
            Qiu Xuan Tan, Nicholas B. Shannon, Weng Khong Lim, J. X. Teo, Daniel R. Y. Yap, Sze Min Lek, Joey W. S. Tan, Shih Jia, Josephine Hendrikson, Ying Liu, Gillian Ng, Clara Y. L. Chong, Wanyu Guo, Kelvin K. N. Koh, C. Ng, Vikneswari Rajasegaran, Jolene S.M. Wong, Chin Jin Seo, C. Ong, Tony K.H. Lim, Bin Tean Teh, Oi Lian Kon, C. Chia, Khee Chee, N. G. Iyer, C. Ong, M. Shammas, Sarah Mazzilli, Prajish Iyer
          </td>
          <td>2024-06-13</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Abnormal cell proliferation and growth leading to cancer primarily result from cumulative genome mutations. Single gene mutations alone do not fully explain cancer onset and progression; instead, clustered mutations—simultaneous occurrences of multiple mutations—are considered to be pivotal in cancer development and advancement. These mutations can affect different genes and pathways, resulting in cells undergoing malignant transformation with multiple functional abnormalities. Clustered mutations influence cancer growth rates, metastatic potential, and drug treatment sensitivity. This summary highlights the various types and characteristics of clustered mutations to understand their associations with carcinogenesis and discusses their potential clinical significance in cancer. As a unique mutation type, clustered mutations may involve genomic instability, DNA repair mechanism defects, and environmental exposures, potentially correlating with responsiveness to immunotherapy. Understanding the characteristics and underlying processes of clustered mutations enhances our comprehension of carcinogenesis and cancer progression, providing new diagnostic and therapeutic approaches for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d085d5b52cda0d463d3c07016cf9320b287c441e" target='_blank'>
              Exploring the Role of Clustered Mutations in Carcinogenesis and Their Potential Clinical Implications in Cancer
              </a>
            </td>
          <td>
            Yi Li, Rui Zhu, Jiaming Jin, Haochuan Guo, Jiaxi Zhang, Zhiheng He, Tingming Liang, Li Guo
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Male sex chromosome aneuploidies are frequent genetic aberrations in humans characterized by additional Y or X chromosome complements. Jacob (JS) and Klinefelter syndromes (KS), carrying 47,XYY and 47,XXY chromosomes, respectively, share several clinical features, including sterility, hormonal deficits, neurocognitive delay, and skeletal-muscle defects. Despite the high incidence, a complete understanding of the molecular mechanisms underlying the clinical manifestations in sex aneuploid patients is still elusive. In this study, we generated and characterized the inaugural cohort of 47,XYY human induced pluripotent stem cells (iPSCs). We performed a comprehensive transcriptional analysis, including JS primary fibroblasts, iPSCs, and neural stem cells (NSCs) alongside KS cells. Our study revealed a transcriptional feedback mechanism tuning non-PAR X chromosome genes (NPX) homologs in Y supernumerary cells, a phenomenon not detected in X aneuploid iPSCs. By ectopically modulating the expression of selected NPY genes, we demonstrated a direct transcriptional link between the UTY-UTX gene pair. Furthermore, our analyses identified a shared transcriptomic signature between JS and KS, discernible already at the iPSC stage, with a notable enrichment for processes related to neurological functions. This transcriptomic convergence underscores potential commonalities in the molecular pathways underpinning the pathophysiology of male sex chromosome aneuploidies. Finally, through genome-wide DNA methylation profiling of JS iPSCs, we demonstrated that a supernumerary Y chromosome only minimally impacts the methylation status of 47,XYY cells at the pluripotent stage. Our work reveals critical transcriptional feedback mechanisms and shared gene expression signatures in male sex chromosome aneuploidies, paving the way for a better understanding of their common phenotypic features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4a1e46b6ba1f6258afb3db5dff0a58811a485a6" target='_blank'>
              An iPSC-based model of Jacob Syndrome reveals a DNA methylation-independent transcriptional dysregulation shared with X aneuploid cells
              </a>
            </td>
          <td>
            V. Astro, K. Cardona-Londoño, L. V. Cortés-Medina, R. Alghamdi, G. Ramírez-Calderón, F. Kefalas, J. Dilmé-Capó, S. Radío, A. Adamo
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61a46af21f4bf12f71e8a9669384dafe3da1b23" target='_blank'>
              CpG island turnover events predict evolutionary changes in enhancer activity
              </a>
            </td>
          <td>
            Acadia A. Kocher, E. Dutrow, Severin Uebbing, Kristina M. Yim, María F. Rosales Larios, Marybeth Baumgartner, Timothy Nottoli, J. Noonan
          </td>
          <td>2024-06-13</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="An obligate step in the life cycle of HIV-1 and other retroviruses is the establishment of the provirus in target cell chromosomes. Transcriptional regulation of proviruses is complex, and understanding the mechanisms underlying this regulation has ramifications for fundamental biology, human health, and gene therapy implementation. The three core components of the Human Silencing Hub (HUSH) complex, TASOR, MPHOSPH8 (MPP8), and PPHLN1 (Periphilin 1), were identified in forward genetic screens for host genes that repress provirus expression. Subsequent loss-of-function screens revealed accessory proteins that collaborate with the HUSH complex to silence proviruses in particular contexts. To identify proteins associated with a HUSH complex-repressed provirus in human cells, we developed a technique, Provirus Proximal Proteomics, based on proximity labeling with C-BERST (dCas9-APEX2 biotinylation at genomic elements by restricted spatial tagging). Our screen exploited a lentiviral reporter that is silenced by the HUSH complex in a manner that is independent of the integration site in chromatin. Our data reveal that proviruses silenced by the HUSH complex are associated with DNA repair, mRNA processing, and transcriptional silencing proteins, including L3MBTL2, a member of the non-canonical polycomb repressive complex 1.6 (PRC1.6). A forward genetic screen confirmed that PRC1.6 components L3MBTL2 and MGA contribute to HUSH complex-mediated silencing. PRC1.6 was then shown to silence HUSH-sensitive proviruses in a promoter-specific manner. Genome wide profiling showed striking colocalization of the PRC1.6 and HUSH complexes on chromatin, primarily at sites of active promoters. Finally, PRC1.6 binding at a subset of genes that are silenced by the HUSH complex was dependent on the core HUSH complex component MPP8. These studies offer new tools with great potential for studying the transcriptional regulation of proviruses and reveal crosstalk between the HUSH complex and PRC1.6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6151d660aa506b4ad3a4c66d40fa26e99798e8" target='_blank'>
              PRC1.6 localizes on chromatin with the human silencing hub (HUSH) complex for promoter-specific silencing
              </a>
            </td>
          <td>
            Tom S Rodriguez, Leonid A Yurkovetskiy, Karthika Nagalekshmi, Chin Hung Oscar Lam, Eva Jazbec, Stacy A. Maitland, S. Wolfe, E. Sontheimer, Jeremy Luban
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) is the most abundant internal modification in mRNAs. Despite accumulating evidence for the profound impact of m6A on cancer biology, there are conflicting reports that alterations in genes encoding the m6A machinery proteins can either promote or suppress cancer, even in the same tumor type. Using data from The Cancer Genome Atlas, we performed a pan-cancer investigation of 15 m6A core factors in nearly 10,000 samples from 31 tumor types to reveal underlying cross-tumor patterns. Altered expression, largely driven by copy number variations at the chromosome arm level, results in the most common mode of dysregulation of these factors. YTHDF1, YTHDF2, YTHDF3, and VIRMA are the most frequently altered factors and the only ones to be uniquely altered when tumors are grouped according to the expression pattern of the m6A factors. These genes are also the only ones with coherent, pan-cancer predictive power for progression-free survival. On the contrary, METTL3, the most intensively studied m6A factor as a cancer target, shows much lower levels of alteration and no predictive power for patient survival. Therefore, we propose the non-enzymatic YTHDF and VIRMA genes as preferred subjects to dissect the role of m6A in cancer and as priority cancer targets. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c9b2f39d5f2496a2d5c8864a272f6e53cb7855" target='_blank'>
              The three YTHDF paralogs and VIRMA are the major tumor drivers among the m6A core genes in a pan-cancer analysis
              </a>
            </td>
          <td>
            E. Destefanis, Denise Sighel, Davide Dalfovo, Riccardo Gilmozzi, F. Broso, Andrea Cappannini, Janusz M. Bujnicki, A. Romanel, E. Dassi, A. Quattrone
          </td>
          <td>2024-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="While genome-wide association studies and expression quantitative trait loci (eQTL) analysis have made significant progress in identifying noncoding variants associated with prostate cancer risk and bulk tissue transcriptome changes, the regulatory effect of these genetic elements on gene expression remains largely unknown. Recent developments in single-cell sequencing have made it possible to perform ATAC-seq and RNA-seq profiling simultaneously to capture functional associations between chromatin accessibility and gene expression. In this study, we tested our hypothesis that this multiome single-cell approach allows for mapping regulatory elements and their target genes at prostate cancer risk loci. We applied a 10X Multiome ATAC + Gene Expression platform to encapsulate Tn5 transposase-tagged nuclei from multiple prostate cell lines for a total of 65,501 high quality single cells from RWPE1, RWPE2, PrEC, BPH1, DU145, PC3, 22Rv1 and LNCaP cell lines. To address data sparsity commonly seen in the single-cell sequencing, we performed targeted sequencing to enrich sequencing data at prostate cancer risk loci involving 2,730 candidate germline variants and 273 associated genes. Although not increasing the number of captured cells, the targeted multiome data did improve eQTL gene expression abundance by about 20% and chromatin accessibility abundance by about 5%. Based on this multiomic profiling, we further associated RNA expression alterations with chromatin accessibility of germline variants at single cell levels. Cross validation analysis showed high overlaps between the multiome associations and the bulk eQTL findings from GTEx prostate cohort. We found that about 20% of GTEx eQTLs were covered within the significant multiome associations (p-value ≤ 0.05, gene abundance percentage ≥ 5%), and roughly 10% of the multiome associations could be identified by significant GTEx eQTLs. We also analyzed accessible regions with available heterozygous SNP reads and observed more frequent association in genomic regions with allelically accessible variants (p = 0.0055). Among these findings were previously reported regulatory variants including rs60464856-RUVBL1 (multiome p-value = 0.0099 in BPH1) and rs7247241-SPINT2 (multiome p-value = 0.0002- 0.0004 in 22Rv1). We also functionally validated a new regulatory SNP and its target gene rs2474694-VPS53 (multiome p-value = 0.00956 in BPH1 and 0.00625 in DU145) by reporter assay and SILAC proteomics sequencing. Taken together, our data demonstrated the feasibility of the multiome single-cell approach for identifying regulatory SNPs and their regulated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936580ce17da8d58425b498ecb0e8504faaa215a" target='_blank'>
              Identify Regulatory eQTLs by Multiome Sequencing in Prostate Single Cells
              </a>
            </td>
          <td>
            Yijun Tian, Lang Wu, Chang-Ching Huang, Liang Wang
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8b04b6a387e12d4ade6bdf8459878237bd6be93" target='_blank'>
              LABS: linear amplification-based bisulfite sequencing for ultrasensitive cancer detection from cell-free DNA
              </a>
            </td>
          <td>
            Xiaolong Cui, Ji Nie, Houxiang Zhu, Krissana Kowitwanich, Alana V. Beadell, Diana C. West-Szymanski, Zhou Zhang, Urszula Dougherty, Akushika Kwesi, Zifeng Deng, Yan Li, Danqing Meng, Kevin Roggin, Teresa Barry, Ryan C. Owyang, Ben Fefferman, C. Zeng, Lu Gao, Carolyn W. T. Zhao, Yuri Malina, Jiangbo Wei, Melanie Weigert, Wenjun Kang, Ajay Goel, Brian Chiu, Marc Bissonnette, Wei Zhang, Mengjie Chen, Chuan He
          </td>
          <td>2024-06-14</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c9abec0157ccbe7ffcea6b2ec6d1e2342588925" target='_blank'>
              Sustained inactivation of the Polycomb PRC1 complex induces DNA repair defects and genomic instability in epigenetic tumors
              </a>
            </td>
          <td>
            C. Rawal, V. Loubiere, N. Butova, Juliette Gracia, V. Parreno, C. Merigliano, Anne-Marie Martinez, G. Cavalli, I. Chiolo
          </td>
          <td>2024-06-18</td>
          <td>Histochemistry and Cell Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The relevant role of LINE-1 (L1) retrotransposition in cancer has been recurrently demonstrated in recent years. However, their repetitive nature hampers their identification and detection, hence remaining inaccessible for clinical practice. Also, its clinical relevance for cancer patients is still limited. Here, we develop a new method to quantify L1 activation, called RetroTest, based on targeted sequencing and a sophisticated bioinformatic pipeline, allowing its application in tumor biopsies. First, we performed the benchmarking of the method and confirmed its high specificity and reliability. Then, we unravel the L1 activation in HNSCC according to a more extensive cohort including all the HNSCC tumor stages. Our results confirm that RetroTest is remarkably efficient for L1 detection in tumor biopsies, reaching a high sensitivity and specificity. In addition, L1 retrotransposition estimation reveals a surprisingly early activation in HNSCC progression, contrary to its classical association with advanced tumor stages. This early activation together with the genomic mutational profiling of normal adjacent tissues supports field cancerization process in this tumor. These results underline the importance of estimating L1 retrotransposition in clinical practice towards an earlier and more efficient diagnosis in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ae21a6395bf2fa53698eb0f53f5dc6c7132bfe" target='_blank'>
              RetroTest unravels LINE-1 retrotransposition in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, Ana Oitaben, Sonia Zumalave, Bernardo Rodriguez-4 Martin, María Gallardo-Gómez, Martín Santamarina, Ana Pequeño-Valtierra, L. Juaneda-Magdalena, Ramón García-Escudero, J. López-Cedrún, Máximo Fraga, J. Tubío, Mónica Martínez-Fernández, Monica Martínez
          </td>
          <td>2024-07-24</td>
          <td>None</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="MOTIVATION
Copy number alterations (CNAs) play an important role in disease progression, especially in cancer. Single-cell DNA sequencing (scDNA-seq) facilitates the detection of CNAs of each cell that is sequenced at a shallow and uneven coverage. However, the state-of-the-art CNA detection tools based on scDNA-seq are still subject to genome-wide errors due to the wrong estimation of the ploidy.


RESULTS
We developed SCCNAInfer, a computational tool that utilizes the subclonal signal inside the tumor cells to more accurately infer each cell's ploidy and CNAs. Given the segmentation result of an existing CNA detection method, SCCNAInfer clusters the cells, infers the ploidy of each subclone, refines the read count by bin clustering, and accurately infers the CNAs for each cell. Both simulated and real datasets show that SCCNAInfer consistently improves upon the state-of-the-art CNA detection tools such as Aneufinder, Ginkgo, SCOPE and SeCNV.


AVAILABILITY AND IMPLEMENTATION
SCCNAInfer is freely available at https://github.com/compbio-mallory/SCCNAInfer.


SUPPLEMENTARY INFORMATION
Supplementary data are available at Bioinformatics online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fcd77fb4e1e5a66f5500cbf69cabe7292f646bc" target='_blank'>
              SCCNAInfer: a robust and accurate tool to infer the absolute copy number on scDNA-seq data.
              </a>
            </td>
          <td>
            Liting Zhang, Xin Maizie Zhou, Xian Mallory
          </td>
          <td>2024-07-27</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome flexibility is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and flexibility determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin a more prone to form SVs than those located in euchromatin, highlighting that genome stability and flexibility differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA Damage Response (DDR), ATM and ATR, suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure UV-B, UV-C and protons induced the Microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a78aa5b539dcd98a3297c9111dc41b5d43d56b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, Michel Pellicioli, Jacky Schuler, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The seven human APOBEC3 (hA3) genes encode polynucleotide cytidine deaminases that play vital roles in restricting replication of viruses and retrotransposons. However, off-target A3 deamination of the cellular genome is a major source of somatic mutations in human cancer. The ability to study A3 biology in vivo is hindered by the fact that the solitary murine Apobec3 gene (mA3) encodes a cytoplasmic enzyme, with no apparent mutagenic activity. Transgenic expression of individual hA3 genes in mice has helped to confirm their oncogenic potential but important questions including which hA3 genes are active in different tissue contexts and how they function in concert when under control of their cognate promoters cannot be addressed using these models. Here we describe humanization of the mouse mA3 locus by integration of a modified BAC clone encompassing the entire 7-gene hA3 locus from human chromosome 22 replacing mA3 on mouse chromosome 15. APOBEC3 mice are viable and fertile and hA3 gene expression in cells and tissues correlates strongly with expression in corresponding human cells and tissues, indicating human-like regulation of hA3 gene expression in the mice. Splenocytes from this line display a functional human A3 response to Friend Murine Leukaemia Virus (F-MLV) infection. We propose that the Hs-APOBEC3 mouse will uniquely model the function of the complete hA3 locus in a living organism and that it will serve as a useful background upon which to model human cancer, as well as assisting drug discovery efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8034f8597ff29dbab41cdb895f637bb191838ad2" target='_blank'>
              Human-like APOBEC3 gene expression and anti-viral responses following replacement of mouse Apobec3 with the 7-gene human APOBEC3 locus
              </a>
            </td>
          <td>
            Nerissa K. Kirkwood, H. Prosser, Melvyn W Yap, Jane Gibson, Ross Cook, Ian Reddin, Ane Stranger, Emma Howes, Nur Zainal, M. Periyasamy, S. Conticello, Gareth J. Thomas, James Scott, Kate N. Bishop, Simak Ali, Allan Bradley, Tim R Fenton
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9895762dc4559c029ea063221ae034c05dcf58ce" target='_blank'>
              DNA replication in early mammalian embryos is patterned, predisposing lamina-associated regions to fragility
              </a>
            </td>
          <td>
            Shuangyi Xu, Ning Wang, M. Zuccaro, Jeannine Gerhardt, R. Iyyappan, Giovanna Nascimento Scatolin, Zongliang Jiang, T. Baslan, A. Koren, Dieter Egli
          </td>
          <td>2024-06-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="The most significant genetic influence on eye color pigmentation is attributed to the intronic SNP rs12913832 in the HERC2 gene, which interacts with the promoter region of the contiguous OCA2 gene. This interaction, through the formation of a chromatin loop, modulates the transcriptional activity of OCA2, directly affecting eye color pigmentation. Recent advancements in technology have elucidated the precise spatial organization of the genome within the cell nucleus, with chromatin architecture playing a pivotal role in regulating various genome functions. In this study, we investigated the organization of the chromatin close to the HERC2/OCA2 locus in human lymphocyte nuclei using fluorescence in situ hybridization (FISH) and high-throughput chromosome conformation capture (Hi-C) data. The 3 Mb of genomic DNA that belonged to the chromosomal region 15q12-q13.1 revealed the presence of three contiguous chromatin loops, which exhibited a different level of compaction depending on the presence of the A or G allele in the SNP rs12913832. Moreover, the analysis of the genomic organization of the genes has demonstrated that this chromosomal region is evolutionarily highly conserved, as evidenced by the analysis of syntenic regions in species from other Vertebrate classes. Thus, the role of rs12913832 variant is relevant not only in determining the transcriptional activation of the OCA2 gene but also in the chromatin compaction of a larger region, underscoring the critical role of chromatin organization in the proper regulation of the involved genes. It is crucial to consider the broader implications of this finding, especially regarding the potential regulatory role of similar polymorphisms located within intronic regions, which do not influence the same gene by modulating the splicing process, but they regulate the expression of adjacent genes. Therefore, caution should be exercised when utilizing whole-exome sequencing for diagnostic purposes, as intron sequences may provide valuable gene regulation information on the region where they reside. Thus, future research efforts should also be directed towards gaining a deeper understanding of the precise mechanisms underlying the role and mode of action of intronic SNPs in chromatin loop organization and transcriptional regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/415e3e0211688fa86f1fcb9648668b6356857209" target='_blank'>
              The Chromatin Organization Close to SNP rs12913832, Involved in Eye Color Variation, Is Evolutionary Conserved in Vertebrates
              </a>
            </td>
          <td>
            Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone, Concetta Federico
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="While genotoxic chemotherapeutic agents are among the most effective tools to combat cancer, they are often associated with severe adverse effects caused by indiscriminate DNA damage in non-tumor tissue as well as increased risk of secondary carcinogenesis. This study builds on our previous work demonstrating that the RNA Polymerase I (Pol I) transcription inhibitor CX-5461 elicits a non-canonical DNA damage response and our discovery of a critical role for Topoisomerase 2α (Top2α) in the initiation of Pol I-dependent transcription. Here, we identify Top2α as a mediator of CX-5461 response in the murine Eµ-Myc B lymphoma model whereby sensitivity to CX-5461 is dependent on cellular Top2α expression/activity. Most strikingly, and in contrast to canonical Top2α poisons, we found that the Top2α-dependent DNA damage induced by CX-5461 is preferentially localized at the ribosomal DNA (rDNA) promoter region, thereby highlighting CX-5461 as a loci-specific DNA damaging agent. This mechanism underpins the efficacy of CX-5461 against certain types of cancer and can be used to develop effective non-genotoxic anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/913f673912030e76f53faa13fb8d57cb178fa48f" target='_blank'>
              CX-5461 Preferentially Induces Top2α-Dependent DNA Breaks at Ribosomal DNA Loci
              </a>
            </td>
          <td>
            Donald P. Cameron, J. Sornkom, Sameerh Alsahafi, D. Drygin, G. Poortinga, Grant A. McArthur, Nadine Hein, Ross D Hannan, Konstantin I. Panov
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="The expression of eukaryotic genes relies on the precise 3’-terminal cleavage and polyadenylation of newly synthesized pre-mRNA transcripts. Defects in these processes have been associated with various diseases, including cancer. While cancer-focused sequencing studies have identified numerous driver mutations in protein-coding sequences, noncoding drivers – particularly those affecting the cis-elements required for pre-mRNA cleavage and polyadenylation – have received less attention. Here, we systematically analysed cancer somatic mutations affecting 3’UTR polyadenylation signals using the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset. We found a striking enrichment of cancer-specific somatic mutations that disrupt strong and evolutionarily conserved cleavage and polyadenylation signals within tumour suppressor genes. Further bioinformatics and experimental analyses conducted as a part of our study suggest that these mutations have a profound capacity to downregulate the expression of tumour suppressor genes. Thus, this work uncovers a novel class of noncoding somatic mutations with significant potential to drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4b3938c7f84b24956b4223cbd287669eff1caf" target='_blank'>
              Recurrent disruption of tumour suppressor genes in cancer by somatic mutations in cleavage and polyadenylation signals
              </a>
            </td>
          <td>
            Yaroslav A. Kainov, F. Hamid, E. Makeyev
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e52325652743d991e4d6523573fd4ef435391561" target='_blank'>
              High-depth whole-genome sequencing identifies structure variants, copy number variants and short tandem repeats associated with Parkinson’s disease
              </a>
            </td>
          <td>
            Chaodong Wang, Hankui Liu, Xu-Ying Li, Jinghong Ma, Z. Gu, Xiuli Feng, Shu Xie, Bei-Sha Tang, Shengdi Chen, Wei Wang, Jian Wang, Jianguo Zhang, Piu Chan
          </td>
          <td>2024-07-23</td>
          <td>NPJ Parkinson's Disease</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Deficiencies in DNA mismatch repair (MMRd) leave characteristic footprints of microsatellite instability (MSI) in cancer genomes. We used data from the Cancer Genome Atlas and International Cancer Genome Consortium to conduct a comprehensive analysis of MSI-associated cancers, focusing on indel mutational signatures. We classified MSI-high genomes into two subtypes based on their indel profiles: deletion-dominant (MMRd-del) and insertion-dominant (MMRd-ins). Compared with MMRd-del genomes, MMRd-ins genomes exhibit distinct mutational and transcriptomic features, including a higher prevalence of T>C substitutions and related mutation signatures. Short insertions and deletions in MMRd-ins and MMRd-del genomes target different sets of genes, resulting in distinct indel profiles between the two subtypes. In addition, indels in the MMRd-ins genomes are enriched with subclonal alterations that provide clues about a distinct evolutionary relationship between the MMRd-ins and MMRd-del genomes. Notably, the transcriptome analysis indicated that MMRd-ins cancers upregulate immune-related genes, show a high level of immune cell infiltration, and display an elevated neoantigen burden. The genomic and transcriptomic distinctions between the two types of MMRd genomes highlight the heterogeneity of genetic mechanisms and resulting genomic footprints and transcriptomic changes in cancers, which has potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b737e04b95df7e27390442dd66d22a5ba373a014" target='_blank'>
              A Subset of Microsatellite Unstable Cancer Genomes Prone to Short Insertions over Deletions Is Associated with Elevated Anticancer Immunity
              </a>
            </td>
          <td>
            Sunmin Kim, Dong-Jin Han, Seo-Young Lee, Youngbeen Moon, Su Jung Kang, Tae-Min Kim
          </td>
          <td>2024-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hutchinson-Gilford Progeria Syndrome (HGPS) is a rare genetic condition characterized by features of accelerated aging, and individuals with HGPS seldom live beyond their mid-teens. The syndrome is commonly caused by a point mutation in the LMNA gene which codes for lamin A and its splice variant lamin C, components of the nuclear lamina. The mutation causing HGPS leads to production of a truncated, farnesylated form of lamin A referred to as “progerin.” Progerin is also expressed at low levels in healthy individuals and appears to play a role in normal aging. HGPS is associated with an accumulation of genomic DNA double-strand breaks (DSBs) and alterations in the nature of DSB repair. The source of DSBs in HGPS is often attributed to stalling and subsequent collapse of replication forks in conjunction with faulty recruitment of repair factors to damage sites. In this work, we used a model system involving immortalized human cell lines to investigate progerin-induced genomic damage. Using an immunofluorescence approach to visualize phosphorylated histone H2AX foci which mark sites of genomic damage, we report that cells engineered to express progerin displayed a significant elevation of endogenous damage in the absence of any change in the cell cycle profile or doubling time of cells. Genomic damage was enhanced and persistent in progerin-expressing cells treated with hydroxyurea. Overexpression of wild-type lamin A did not elicit the outcomes associated with progerin expression. Our results show that DNA damage caused by progerin can occur independently from global changes in replication or cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4814100043718a6a0e30f1cd9c037307d90666" target='_blank'>
              Progerin Can Induce DNA Damage in the Absence of Global Changes in Replication or Cell Proliferation
              </a>
            </td>
          <td>
            Liza A. Joudeh, P. Logan Schuck, Nina M. Van, Alannah J. DiCintio, Jason A. Stewart, Alan S. Waldman
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract DNA Double-strand breaks (DSBs) are harmful lesions and major sources of genomic instability. Studies have suggested that DSBs induce local transcriptional silencing that consequently promotes genomic stability. Several factors have been proposed to actively participate in this process, including ATM and Polycomb repressive complex 1 (PRC1). Here we found that disrupting PRC1 clustering disrupts DSB-induced gene silencing. Interactome analysis of PHC2, a PRC1 subunit that promotes the formation of the Polycomb body, found several nucleoporins that constitute the Nuclear Pore Complex (NPC). Similar to PHC2, depleting the nucleoporins also disrupted the DSB-induced gene silencing. We found that some of these nucleoporins, such as NUP107 and NUP43, which are members of the Y-complex of NPC, localize to DSB sites. These nucleoporin-enriched DSBs were distant from the nuclear periphery. The presence of nucleoporins and PHC2 at DSB regions were inter-dependent, suggesting that they act cooperatively in the DSB-induced gene silencing. We further found two structural components within NUP107 to be necessary for the transcriptional repression at DSBs: ATM/ATR-mediated phosphorylation at Serine37 residue within the N-terminal disordered tail, and the NUP133-binding surface at the C-terminus. These results provide a new functional interplay among nucleoporins, ATM and the Polycomb proteins in the DSB metabolism, and underscore their emerging roles in genome stability maintenance. *Hongseon Song, Yubin Bae, Sangin Kim, and Dante Deascanis contributed equally to this work.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b0e95cd1e0dcce638bc5888df3682c199f2d24" target='_blank'>
              Nucleoporins cooperate with Polycomb silencers to promote transcriptional repression and repair at DNA double strand breaks
              </a>
            </td>
          <td>
            Hongseon Song*, Yubin Bae*, Sangin Kim, Dante Deascanis*, Yujin Lee, G. Róna, Ethan Lane, Seoyeong Lee, Sujung Kim, Michele Pagano, Kyungjae Myung, Y. Kee
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/153ab2ed142f22e69713eea228d0914f5b69386b" target='_blank'>
              Telomere-to-telomere genome assembly of sorghum
              </a>
            </td>
          <td>
            Meng Li, Chunhai Chen, Haigang Wang, Huibin Qin, Sen Hou, Xukui Yang, Jianbo Jian, Peng Gao, Minxuan Liu, Z. Mu
          </td>
          <td>2024-08-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Herpes simplex virus 1 (HSV-1) latently infected neurons show multiple patterns in the distribution of the viral genomes within the nucleus, at least in mouse models. One of the major patterns is characterized by the presence of quiescent HSV-1 genomes trapped in promyelocytic leukemia nuclear bodies (PML NBs) to form viral DNA-containing PML-NBs (vDCP NBs). Using a cellular model reproducing the formation of vDCP NBs we previously showed that viral genomes are chromatinized with the H3.3 histone variant modified on its lysine 9 by tri-methylation (H3.3K9me3) a chromatin mark associated with transcriptional repression. Here we identify an essential role for the HUSH complex and its SETDB1 and MORC2 effectors in the acquisition of the H3K9me3 mark on the PML NBs-associated HSV-1 and in the maintenance of HSV-1 transcriptional repression. ChiP-seq analyses highlight the association of the H3K9me3 mark with the entire viral genome. Inactivating the HUSH-SETDB1-MORC2 repressor complex prior to viral infection results in a significant reduction of H3K9me3 on the viral genome, while the overall impact on the cellular genome is minimal, except for expected changes in families of LINE1 retroelements. Depletion of HUSH, SETDB1, or MORC2, relieves the repressive state of HSV-1 in infected primary human fibroblasts as well as human induced pluripotent stem cell-derived sensory neurons (hiPSDN). We discovered that the viral protein ICP0 induces MORC2 degradation via the proteasome machinery. This process is concurrent with ICP0 and MORC2 depletion capability to reactivate silenced HSV-1 in hiPSDN. Overall, our findings underscore the robust antiviral function of the HUSH-SETDB1-MORC2 repressor complex against a herpesvirus by modulating chromatin marks linked to repression, thus presenting promising avenues for novel anti-herpesvirus therapeutic strategies. Significance statement Herpes simplex virus 1 (HSV-1) is a major human pathogen, which remains latent in the trigeminal ganglia (TG) neurons of the infected individuals. Its reactivation is characterized by a variety of clinical symptoms the most severe ones being keratitis and herpesvirus encephalitis. The colonization of the CNS by the virus during the individual life is a well-known fact but the pathophysiological effects on neurons homeostasis are still underestimated. It is thus paramount to understand the molecular mechanisms that control HSV-1 latency and maintain the virus in a pseudo silent state.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/533427e78d2024eb3c0732866c4276477e3b2ffa" target='_blank'>
              The HUSH epigenetic repressor complex silences PML nuclear bodies-associated HSV-1 quiescent genomes
              </a>
            </td>
          <td>
            Simon Roubille, Tristan Escure, F. Juillard, Armelle Corpet, Rémi Néplaz, Olivier Binda, Coline Seurre, M. Gonin, Stuart Bloor, Camille Cohen, Pascale Texier, Oscar Haigh, Olivier Pascual, Yonatan Ganor, Frédérique Magdinier, M. Labetoulle, Paul J. Lehner, P. Lomonte
          </td>
          <td>2024-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Tumors often show an initial response to chemotherapy, but then develop resistance, leading to relapse and poor prognosis. We hypothesized that a genomic comparison of mutations in pre-treated versus treatment-naive tumors would serve to identify genes that confer resistance. A challenge in such an analysis is that therapy alters mutation burdens and signatures, confounding association studies and complicating identifying causal, selected mutations. We developed DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes. Crucially, DiffInvex draws on a mutation rate baseline that accounts for these shifts in neutral mutagenesis during cancer evolution. We applied DiffInvex to 9,953 cancer whole-genomes from 29 cancer types from 8 studies, containing both WGS of treatment-naive tumors and tumors pre-treated by various drugs, identifying genes where point mutations are under conditional positive or negative selection for a certain chemotherapeutic, suggesting resistance mechanisms occurring via point mutation. DiffInvex confirmed well-known chemoresistance-driver mutations in EGFR, ESR1, KIT and AR genes as being under conditional positive selection, with additional cancer types identified for EGFR and KIT. Additionally, DiffInvex identified 11 genes with treatment-associated selection for different classes of therapeutics. In most cases, these genes were common cancer genes including PIK3CA, APC, MAP2K4 and MAP3K1. This suggests that tumor resistance to therapy via mutation often occurs via selective advantages conferred by known driver genes, rather than via mutations in specialized resistance genes. Various gene-chemotherapy associations were further supported in tests for functional impact of mutations, again implemented in a conditional selection setting, as well as replicating in independent panel or exome sequencing data. In addition to nominating drug resistance genes that could be targeted by future therapeutics, DiffInvex can also be applied to diverse analysis in cancer evolution, such as comparing normal and tumoral tissues, or analyzing subclonal evolution, identifying changes in selection over time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b71e5df0edb673b45bd36f564fa0da7eae40ed" target='_blank'>
              The DiffInvex evolutionary model for conditional somatic selection identifies chemotherapy resistance genes in 10,000 cancer genomes
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2024-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da14688d34e68034cf1e370a5861d8d265cacc" target='_blank'>
              Telomere-to-telomere genome assembly of the goose Anser cygnoides
              </a>
            </td>
          <td>
            Hongchang Zhao, Hao Zhou, Guobo Sun, Biao Dong, Wenqi Zhu, Xiaohui Mu, Xiaoming Li, Jun Wang, Mengli Zhao, Wenhao Yang, Gansheng Zhang, Rongchao Ji, T. Geng, Daoqing Gong, He Meng, Jian Wang
          </td>
          <td>2024-07-07</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc88953f3695ac0c90c5077d3442247bd93ef70" target='_blank'>
              Mixed responses to targeted therapy driven by chromosomal instability through p53 dysfunction and genome doubling
              </a>
            </td>
          <td>
            S. Hobor, M. Al Bakir, C. Hiley, Marcin Skrzypski, A. Frankell, Bjorn Bakker, T. Watkins, A. Markovets, J. Dry, Andrew P. Brown, Jasper van der Aart, H. van den Bos, D. Spierings, D. Oukrif, Marco Novelli, T. Chakrabarti, Adam H. Rabinowitz, Laila Ait Hassou, Saskia Litière, D. Kerr, L. Tan, Gavin P. Kelly, David A Moore, M. Renshaw, Subramanian Venkatesan, William Hill, A. Huebner, C. Martínez-Ruiz, James R. M. Black, Wei Wu, Mihaela Angelova, N. Mcgranahan, Julian Downward, J. Chmielecki, Carl Barrett, K. Litchfield, S. Chew, Collin M Blakely, Elza C de Bruin, F. Foijer, Karen H. Vousden, T. Bivona, , Jason Lester, A. Bajaj, A. Nakas, Azmina Sodha-Ramdeen, M. Tufail, M. Scotland, R. Boyles, S. Rathinam, C. Wilson, Domenic Marrone, S. Dulloo, D. Fennell, Gurdeep Matharu, J. Shaw, E. Boleti, Heather Cheyne, Mohammed Khalil, S. Richardson, Tracey Cruickshank, Gillian Price, K. M. Kerr, Sarah Benafif, Jack French, Kayleigh Gilbert, B. Naidu, Akshay J. Patel, A. Osman, Carol Enstone, G. Langman, Helen Shackleford, M. Djearaman, S. Kadiri, Gary Middleton, Angela E. Leek, Jack Davies Hodgkinson, Nicola Totton, A. Montero, E. Smith, E. Fontaine, F. Granato, A. Paiva-Correia, Juliette Novasio, K. Rammohan, L. Joseph, P. Bishop, Rajesh Shah, Stuart Moss, Vijay Joshi, Philip A J Crosbie, Katherine D. Brown, Mathew Carter, Anshuman Chaturvedi, Pedro Oliveira, Colin R. Lindsay, Fiona H. Blackhall, Matthew G. Krebs, Yvonne Summers, A. Clipson, J. Tugwood, Alastair Kerr, D. Rothwell, C. Dive, H. Aerts, R. Schwarz, Tom L. Kaufmann, G. Wilson, R. Rosenthal, P. Van Loo, Nicolai J. Birkbak, Zoltán Szállási, J. Kisistók, M. Sokač, R. Salgado, M. Dióssy, J. Demeulemeester, Abigail Bunkum, Angela Dwornik, Alastair Magness, Andrew J. Rowan, Angeliki Karamani, A. Toncheva, B. Chain, C. Castignani, Chris Bailey, C. Abbosh, C. Puttick, C. Weeden, Claudia Lee, Corentin Richard, C. Naceur-Lombardelli, D. Pearce, D. Karagianni, D. Biswas, D. Levi, E. Larose Cadieux, E. Lim, Emma C Colliver, E. Nye, F. Gálvez-Cancino, F. Gimeno-Valiente, G. Kassiotis, Georgia Stavrou, Gerasimos Mastrokalos, Helen L Lowe, Ignacio Matos, I. Noorani, J. Goldman, J. Reading, J. Rane, J. Nicod, John A. Hartley, K. Peggs, Katey S. S. Enfield, Kayalvizhi Selvaraju, K. Thol, Kevin W. Ng, Kezhong Chen, K. Dijkstra, Kristiana Grigoriadis, Krupa Thakkar, L. Ensell, Mansi Shah, Maria Litovchenko, M. Jamal-Hanjani, Mariana Werner Sunderland, M. Huska, M. Hill, M. Dietzen, Michelle Leung, M. Escudero, M. Tanić, Monica Sivakumar, O. Chervova, Olivia Lucas, O. Pich, Othman Al-Sawaf, Paulina Prymas, Philip S Hobson, Piotr Pawlik, R. Stone, R. Bentham, Roberto Vendramin, S. Saghafinia, Samuel Gamble, S. Veeriah, S. Ung, Sergio A Quezada, Sharon P. Vanloo, S. Hessey, S. Ward, Sian Harries, S. Boeing, Stephan Beck, S. Bola, Takahiro Karasaki, Tamara Denner, T. Marafioti, Thomas P. Jones, V. Spanswick, Vittorio Barbè, Wei-Ting Lu, Wing Kin Liu, Yin Wu, Yutaka Naito, Zoe Ramsden, C. Veiga, Gary Royle, C. Collins-Fekete, Francesco Fraioli, Paul Ashford, Martin D. Forster, Siow-Ming Lee, E. Borg, M. Falzon, D. Papadatos-Pastos, James Wilson, T. Ahmad, A. Procter, Asia Ahmed, Magali N. Taylor, Arjun Nair, David S. Lawrence, D. Patrini, Neal Navani, R. Thakrar, Sam M. Janes, Emilie Martinoni Hoogenboom, Fleur Monk, James W. Holding, Junaid Choudhary, Kunal Bhakhri, M. Scarci, Pat Gorman, Reena Khiroya, Robert C. M. Stephens, Y. Wong, Zoltán Káplár, S. Bandula, A. Hackshaw, A. Hacker, A. Sharp, Sean Smith, Harjot Kaur Dhanda, Camilla Pilotti, R. Leslie, Anca-Ioana Grapa, Hanyun Zhang, K. AbdulJabbar, Xiaoxi Pan, Yinyin Yuan, David Chuter, M. Mackenzie, S. Chee, A. Alzetani, Judith Cave, J. Richards, Eric Lim, P. De Sousa, Simon Jordan, Alex Rice, H. Raubenheimer, H. Bhayani, L. Ambrose, A. Devaraj, Hema Chavan, S. Begum, Silviu I. Buderi, Daniel Kaniu, Mpho Malima, S. Booth, Andrew G. Nicholson, Nádia Fernandes, Pratibha Shah, C. Proli, M. Hewish, S. Danson, M. Shackcloth, Lily Robinson, P. Russell, K. Blyth, A. Kidd, C. Dick, J. Le Quesne, Alan Kirk, Mohd. Asif, Rocco Bilancia, N. Kostoulas, Mathew Thomas, R. Hynds, N. Kanu, S. Zaccaria, E. Grönroos, C. Swanton
          </td>
          <td>2024-06-13</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320f6fe1cd3bc7742918145765f974259b6508cf" target='_blank'>
              Quantifying DNA replication speeds in single cells by scEdU-seq
              </a>
            </td>
          <td>
            Jeroen van den Berg, Vincent van Batenburg, Christoph Geisenberger, Rinskje B. Tjeerdsma, A. de Jaime-Soguero, S. Acebrón, M. V. van Vugt, Alexander van Oudenaarden
          </td>
          <td>2024-06-17</td>
          <td>Nature Methods</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic regulatory mechanism that plays a critical role in a variety of cellular processes, including the regulation of cell fate during development, maintenance of cell identity, and genome stability. DNA methylation is tightly regulated by enzymatic reactions and its deregulation plays an important role in the development of cancer. Specific DNA methylation alterations have been found in pediatric solid tumors, providing new insights into the development of these tumors. In addition, DNA methylation profiles have greatly contributed to tune the diagnosis of pediatric solid tumors and to define subgroups of patients with different risks of progression, leading to the reduction in unwanted toxicity and the improvement of treatment efficacy. This review highlights the dysregulated DNA methylome in pediatric solid tumors and how this information provides promising targets for epigenetic therapies, particularly inhibitors of DNMT enzymes (DNMTis). Opportunities and limitations are considered, including the ability of DNMTis to induce viral mimicry and immune signaling by tumors. Besides intrinsic action against cancer cells, DNMTis have the potential to sensitize immune-cold tumors to immunotherapies and may represent a remarkable option to improve the treatment of challenging pediatric solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1eac857dfcc5a5e3b672ddb17d6097e48bc70f" target='_blank'>
              Targeting DNA Methylation Machinery in Pediatric Solid Tumors
              </a>
            </td>
          <td>
            C. Cristalli, Katia Scotlandi
          </td>
          <td>2024-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The male mammalian germline is characterized by substantial chromatin remodeling associated with the transition from histones to protamines during spermatogenesis, followed by the reversal to nucleohistones in the male pronucleus preceding the zygotic genome activation. Both transitions are associated with the extensive formation of DNA double-strand breaks (DSBs), requiring an estimated 5 to 10 million transient DSBs per spermatozoa. Additionally, the high transcription rate in early stages of spermatogenesis leads to transcription-coupled damage preceding meiotic homologous recombination, potentially further contributing to the DSB landscape in mature spermatozoa. Once meiosis is completed, spermatozoa remain haploid and therefore cannot rely on error-free homologous recombination, but instead depend on error-prone classical non-homologous end joining (cNHEJ). This DNA damage/repair-scenario is proposed to be one of the main causes of the observed paternal mutation propensity in human evolution. Recent studies have shown that DSBs in the male pronucleus are repaired by maternally provided Polθ in Caenorhabditis elegans through Polθ-mediated end joining (TMEJ). Additionally, population genetic datasets have revealed a preponderance of TMEJ signatures associated with human variation. Since these signatures are the result of the combined effect of TMEJ and DSB formation in spermatozoa and male pronuclei, we used a BLISS-based protocol to analyze recurrent DSBs in mature human sperm heads as a proxy of the male pronucleus before zygotic chromatin remodeling. The DSBs were found to be enriched in (YR)n short tandem repeats and in evolutionarily young SINEs, reminiscent to patterns observed in murine spermatids, indicating evolutionary hotspots of recurrent DSB formation in mammalian spermatozoa. Additionally, we detected a similar DSB pattern in diploid human IMR90 cells when cNHEJ was selectively inhibited, indicating the significant impact of absent cNHEJ on the sperm DSB landscape. Strikingly, regions associated with most retained histones, and therefore less condensed chromatin, were not strongly enriched with recurrent DSBs. In contrast, the fraction of retained H3K27me3 in the mature spermatozoa displayed a strong association with recurrent DSBs. DSBs in H3K27me3 are associated with a preference for TMEJ over cNHEJ during repair. We hypothesize that the retained H3K27me3 may trigger transgenerational DNA repair by priming maternal Polθ to these regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60c153158e8100ec0f4fdb6f1cd7fe2e99666eda" target='_blank'>
              DSB profiles in human spermatozoa highlight the role of TMEJ in the male germline
              </a>
            </td>
          <td>
            Maurice Scheuren, Jonas Möhner, Max Müller, Hans Zischler
          </td>
          <td>2024-07-08</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="New high-quality human genome assemblies derived from lymphoblastoid cell lines (LCLs) provide reference genomes and pangenomes for genomics studies. However, the characteristics of LCLs pose technical challenges to profiling immunoglobulin (IG) genes. IG loci in LCLs contain a mixture of germline and somatically recombined haplotypes, making them difficult to genotype or assemble accurately. To address these challenges, we introduce IGLoo, a software tool that implements novel methods for analyzing sequence data and genome assemblies derived from LCLs. IGLoo characterizes somatic V(D)J recombination events in the sequence data and identifies the breakpoints and missing IG genes in the LCL-based assemblies. Furthermore, IGLoo implements a novel reassembly framework to improve germline assembly quality by integrating information about somatic events and population structural variantions in the IG loci. We applied IGLoo to study the assemblies from the Human Pangenome Reference Consortium, providing new insights into the mechanisms, gene usage, and patterns of V(D)J recombination, causes of assembly fragmentation in the IG heavy chain (IGH) locus, and improved representation of the IGH assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8f48cf5eb22832827a1cb138fb7f409be2963b7" target='_blank'>
              IGLoo: Profiling the Immunoglobulin Heavy chain locus in Lymphoblastoid Cell Lines with PacBio High-Fidelity Sequencing reads
              </a>
            </td>
          <td>
            Mao-Jan Lin, Ben Langmead, Yana Safonova
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Single-cell multi-omics data is a valuable resource to promote the understanding of cancer. However, multimodal data analysis is challenging for most cancer researchers, and there is still a lack of online databases that can bridge the gap. Here we introduced scCancerExplorer, which is a user-friendly database designed for exploring single-cell multi-omics data of human pan-cancer. We processed more than 110 datasets covering single-cell multiomics data from 51 human cancer types, and provides 5 major modules and 12 useful functions for conveniently exploring those data. (1) The “Integrated multi-omics analysis” module enables users explore a gene from multimodal perspectives, such as the gene expression levels, survival analysis, transcription factor activity, and the DNA methylation levels of promoter regions. (2) With single-cell transcriptome module, users can explore the integrated pan-cancer datasets, compare the expression profiles between primary tumor and metastases, and generate customized figures. (3) With single-cell epigenome module, users can explore promoter DNA methylation levels in both normal and cancer cells, DNA methylation state at single CpG resolution, and chromatin accessibility patterns of different cell types. (4) For single-cell genome data, users can visualize the copy number variations of cancer cells for each patient. (5) Moreover, TCGA survival analysis can be performed conveniently. Users can not only group the patients according to gene expression levels, but also select patients by gene mutation states, pathological subtypes, and cancer stages. In summary, scCancerExplorer is a powerful database for convenient data mining by simple clicking, and gaining novel insights into human pan-cancer. scCancerExplorer is freely accessible at https://bianlab.cn/scCancerExplorer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f0bfe81970da09d476152acc53e069683a992ff" target='_blank'>
              scCancerExplorer: a comprehensive database for interactively exploring single-cell multi-omics data of human pan-cancer
              </a>
            </td>
          <td>
            Changzhi Huang, Zekai Liu, Yunlei Guo, Wanchu Wang, Zhen Yuan, Yusheng Guan, Deng Pan, Zhibin Hu, Linhua Sun, Zan Fu, Shuhui Bian
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer is the second leading cause of death worldwide, and although there have been advances in treatments, including immunotherapies, these often require biopsies which can be costly and invasive to obtain. Due to lack of pre-emptive cancer detection methods, many cases of cancer are detected at a late stage when the definitive symptoms appear. Plasma samples are relatively easy to obtain, and they can be used to monitor the molecular signatures of ongoing processes in the body. Profiling cell-free DNA is a popular method for monitoring cancer, but only a few studies have explored the use of cell-free RNA (cfRNA), which shows the recent footprint of systemic transcription. Here we developed FastNeo, a computational method for detecting known neoepitopes in human cfRNA. We show that neoepitopes and other biomarkers detected in cfRNA can discern Hepatocellular carcinoma (HCC) patients from the healthy patients with a sensitivity of 0.84 and a specificity of 0.79. For colorectal cancer we achieve a sensitivity of 0.87 and a specificity of 0.8. An important advantage of our cfRNA based approach is that it also reports putative neoepitopes which are important for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39049eaeea832dff014994be3a821e1580cf6286" target='_blank'>
              Detecting known neoepitopes, gene fusions, transposable elements, and circular RNAs in cell-free RNA
              </a>
            </td>
          <td>
            Mayank Mahajan, Martin Hemberg
          </td>
          <td>2024-06-09</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations in polymerases Pold1 and Pole exonuclease domains in humans are associated with increased cancer incidence, elevated tumor mutation burden (TMB) and response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond. Here we generated Pold1 and Pole proofreading mutator mice and show that ICB treatment of mice with high TMB tumors did not improve survival as only a subset of tumors responded. Similarly, introducing the mutator alleles into mice with Kras/p53 lung cancer did not improve survival, however, passaging mutator tumor cells in vitro without immune editing caused rejection in immune-competent hosts, demonstrating the efficiency by which cells with antigenic mutations are eliminated. Finally, ICB treatment of mutator mice earlier, before observable tumors delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the use of ICB in individuals at high risk for cancer prevention. Highlights Germline somatic and conditional Pold1 and Pole exonuclease domain mutations in mice produce a mutator phenotype. Spontaneous cancers arise in mutator mice that have genomic features comparable to human tumors with these mutations. ICB treatment of mutator mice with tumors did not improve survival as only a subset of tumors respond. Introduction of the mutator alleles into an autochthonous mouse lung cancer model also did not produce immunogenic tumors, whereas passaging mutator tumor cells in vitro caused immune rejection indicating efficient selection against antigenic mutations in vivo. Prophylactic ICB treatment delayed cancer onset, improved survival, and selected for tumors with no aneuploidy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e6ba5a717c7dd3412aa2d3806c5dc666fb03f9" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer and Extends Survival in Murine DNA Polymerase Mutator Syndromes
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. C. Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Abstract Background Germline mutations have been identified in a small number of hereditary cancers, but the genetic predisposition for many familial cancers remains to be elucidated. Methods This study identified a Chinese pedigree that presented different cancers (breast cancer, BRCA; adenocarcinoma of the esophagogastric junction, AEG; and B‐cell acute lymphoblastic leukemia, B‐ALL) in each of the three generations. Whole‐genome sequencing and whole‐exome sequencing were performed on peripheral blood or bone marrow and cancer biopsy samples. Whole‐genome bisulfite sequencing was conducted on the monozygotic twin brothers, one of whom developed B‐ALL. Results According to the ACMG guidelines, bioinformatic analysis of the genome sequencing revealed 20 germline mutations, particularly mutations in the DNAH11 (c.9463G > A) and CFH (c.2314G > A) genes that were documented in the COSMIC database and validated by Sanger sequencing. Forty‐one common somatic mutated genes were identified in the cancer samples, displaying the same type of single nucleotide substitution Signature 5. Meanwhile, hypomethylation of PLEK2, MRAS, and RXRA as well as hypermethylation of CpG island associated with WT1 was shown in the twin with B‐ALL. Conclusions These findings reveal genomic alterations in a pedigree with multiple cancers. Mutations found in the DNAH11, CFH genes, and other genes predispose to malignancies in this family. Dysregulated methylation of WT1, PLEK2, MRAS, and RXRA in the twin with B‐ALL increases cancer susceptibility. The similarity of the somatic genetic changes among the three cancers indicates a hereditary impact on the pedigree. These familial cancers with germline and somatic mutations, as well as epigenomic alterations, represent a common molecular basis for many multiple cancer pedigrees.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3f0ae27a463e75a2f599e816c3a044e8b7c954" target='_blank'>
              Impact of genomic and epigenomic alterations of multigene on a multicancer pedigree
              </a>
            </td>
          <td>
            Jinyu Gao, Yongzhang Wu, Jieming Yu, Yinbin Qiu, Tiantian Yi, Chaochao Luo, Junxiao Zhang, Gary Lu, Xu Li, Fu Xiong, Xuedong Wu, Xinghua Pan
          </td>
          <td>2024-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Aneuploidy – loss or gain of whole chromosomes or chromosome arms, is rare and poorly tolerated in normal cells but occurs in ∼90% of solid tumors; however, the mechanisms through which specific aneuploidies affect cancer development are unclear. Additionally, generating mammalian models of specific chromosome arm alterations is technically difficult, limiting further study. Cancers have tumor, cell, and tissue type-specific patterns of chromosome arm copy-number alterations that influence tumor evolution and sensitivity to anti-cancer therapies. Squamous cell carcinomas (SCCs) affecting lung, head and neck, esophageal, and cervical tissues have been shown to feature conserved early losses and gains of arms 3p and 3q, respectively, and are associated with relatively few oncogenic drivers and treatment options. These aneuploidies are particularly well established in lung SCC and have been shown to promote tumorigenesis, metastasis, and poor prognosis. To study cancer vulnerabilities associated with chromosome 3 arm aneuploidies, we used CRISPR-Cas9 to delete a 3p copy in human immortalized lung epithelial cell lines—the lung SCC cell-of-origin. A subset of clones duplicated a wild type chromosome 3 copy, transitioning to 3q gain and providing us with independent models for chromosome 3 wild type, 3p loss, and 3q gain. Next, we performed a CRISPRi screen and found that 3q gain cells are more sensitive to knockdown of sterol regulatory element-binding factor 1 (SREBF1)—a master regulator of cholesterol and fatty acid biosynthesis, its co-factor SREBP cleavage activating protein (SCAP), and downstream target HMG-CoA reductase (HMGCR)—the rate-limiting enzyme for cholesterol synthesis. 3q gain cells also showed up-regulated fatty acid metabolites, ceramides, triglycerides, and phosphatidylinositols. After performing a chemical screen, 3q gain cells showed reduced proliferation, increased apoptosis, and DNA damage in response to statins (HMGCR inhibitors). Statins have previously been shown to induce tumor-specific apoptosis and reduced cancer risk, although these studies have associated with variable efficacy and underlying mechanisms. We hypothesize that 3q gain is a biomarker of statin response and sensitivity in squamous cancers. We aim to study the mechanism through which aneuploidy alters lipid metabolism and contributes to cancer. Investigating aneuploidy-induced vulnerabilities will help us understand how cancers exploit aneuploidies to promote tumorigenesis and to elucidate novel and specific therapeutic targets for cancers with shared copy-number profiles.
 Citation Format: Nadja Zhakula-Kostadinova, Sejal Jain, Laura Byron, Matthew L. Meyerson, Alison M. Taylor. Investigating vulnerabilities associated with chromosome arm aneuploidy in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca81f5571b9f264386412ab8dcfed6cabd96466" target='_blank'>
              Abstract PR016: Investigating vulnerabilities associated with chromosome arm aneuploidy in cancer
              </a>
            </td>
          <td>
            Nadja Zhakula-Kostadinova, Sejal Jain, Laura Byron, Matthew L. Meyerson, Alison M. Taylor
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdb874383a628cf292c0913e4a06655cd292ef13" target='_blank'>
              Frequency-dependent selection of neoantigens fosters tumor immune escape and predicts immunotherapy response
              </a>
            </td>
          <td>
            Shaoqing Chen, Duo Xie, Zan Li, Jiguang Wang, Zheng Hu, Da Zhou
          </td>
          <td>2024-06-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The centromeric protein-A (CENP-A) is an evolutionary conserved histone H3 variant that marks the identity of the centromeres. Several mechanisms regulate the centromeric deposition of CENP-A as its mislocalization causes erroneous chromosome segregation, leading to aneuploidy-based diseases, including cancers. The most crucial deposition factor is a CENP-A specific chaperone, HJURP (Scm3 in budding yeast), which specifically binds to CENP-A. However, the discovery of HJURP as a DDR (DNA damage repair) protein and evidence of its binding to Holliday junctions in vitro indicate a CENP-A-deposition-independent role of these chaperones. In this study, using budding yeast, we demonstrate that Scm3 is crucial for the DDR pathway as scm3 cells are sensitive to DNA damage. We further observe that the scm3 mutant interacts with the rad52 DDR mutant and is compromised in activating DDR-mediated arrest. We demonstrate that Scm3 associates with the DNA damage sites and undergoes posttranslational modifications upon DNA damage. Overall, from this report and earlier studies on HJURP, we conclude that DDR functions of CENP-A chaperones are conserved across eukaryotes. Thus, the revelation that these chaperones confer genome stability in more than one pathway has clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20b8f002bffe33b7b0eebb870a6636324b784a0d" target='_blank'>
              Evidence of centromeric histone 3 chaperone involved in DNA damage repair pathway
              </a>
            </td>
          <td>
            Prakhar Agarwal, S. Ghosh
          </td>
          <td>2024-07-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b0e09e628f8fcf9bb1ad3d19ff2d5112c171387" target='_blank'>
              Unmutated RRAS2 emerges as a key oncogene in post-partum-associated triple negative breast cancer
              </a>
            </td>
          <td>
            C. Cifuentes, Clara L. Oeste, Isabel Fernández-Pisonero, Alejandro Hortal, C. García-Macías, Jeanne Hochart, Regina Rubira, L. Horndler, Carlos Horndler, Xosé R. Bustelo, Balbino Alarcón
          </td>
          <td>2024-07-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855843ba00188002869d72dc3ec46da982ab1cf4" target='_blank'>
              Genome-wide DNA methylation in relation to ARID1A deficiency in ovarian clear cell carcinoma
              </a>
            </td>
          <td>
            Shang Li, G. Meersma, J. Kupryjańczyk, Steven de Jong, G. Wisman
          </td>
          <td>2024-06-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Cycling cells replicate their DNA during the S phase through a defined temporal program known as replication timing. Mutation frequencies, epigenetic chromatin states, and transcriptional activities are different for genomic regions that are replicated early and late in the S phase. Here, we found from ChIP-Seq analysis that DNA polymerase (Pol) κ is enriched in early-replicating genomic regions in HEK293T cells. In addition, by feeding cells with N 2-heptynyl-2′-deoxyguanosine followed by click chemistry–based enrichment and high-throughput sequencing, we observed elevated Pol κ activities in genomic regions that are replicated early in the S phase. On the basis of the established functions of Pol κ in accurate and efficient nucleotide insertion opposite endogenously induced N 2-modified dG lesions, our work suggests that active engagement of Pol κ may contribute to diminished mutation rates observed in early-replicating regions of the human genome, including cancer genomes. Together, our work expands the functions of Pol κ and offered a plausible mechanism underlying replication timing–dependent mutation accrual in the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417d907a79911389824959d3e561a3c8846017ac" target='_blank'>
              DNA polymerase κ participates in early S-phase DNA replication in human cells
              </a>
            </td>
          <td>
            Feng Tang, Yinan Wang, Ting Zhao, Jun Yuan, Andrew H Kellum, Yinsheng Wang
          </td>
          <td>2024-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96ca99a18dd08dc5d59b7f900d05406ec6c8cd83" target='_blank'>
              Amplifications of EVX2 and HOXD9-HOXD13 on 2q31 in mature cystic teratomas of the ovary identified by array comparative genomic hybridization may explain teratoma characteristics in chondrogenesis and osteogenesis
              </a>
            </td>
          <td>
            Wen-Chung Wang, Tai-Cheng Hou, C. Kuo, Yen-Chein Lai
          </td>
          <td>2024-06-21</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Premature aging is a hallmark of Down syndrome, caused by trisomy of human chromosome 21, but the reason is unclear and difficult to study in humans. We used an aneuploid model in wild yeast to show that chromosome amplification disrupts nutrient-induced cell-cycle arrest, quiescence entry, and healthy aging, across genetic backgrounds and amplified chromosomes. We discovered that these defects are due in part to aneuploidy-induced dysfunction in Ribosome Quality Control (RQC). Compared to euploids, aneuploids entering quiescence display aberrant ribosome profiles, accumulate RQC intermediates, and harbor an increased load of protein aggregates. Although they have normal proteasome capacity, aneuploids show signs of ubiquitin dysregulation, which impacts cyclin abundance to disrupt arrest. Remarkably, inducing ribosome stalling in euploids produces similar aberrations, while up-regulating limiting RQC subunits or proteins in ubiquitin metabolism alleviates many of the aneuploid defects. Our results provide implications for other aneuploidy disorders including Down syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c6d2e4dd1921e05fad92cf19e5a9d35d322983a" target='_blank'>
              Premature aging in aneuploid yeast is caused in part by aneuploidy-induced defects in Ribosome Quality Control
              </a>
            </td>
          <td>
            Leah E. Escalante, James Hose, Hollis Howe, Norah Paulsen, Michael Place, A. Gasch
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="With age, hematopoietic stem cells can acquire somatic mutations in leukemogenic genes that confer a proliferative advantage in a phenomenon termed “clonal hematopoiesis of indeterminate potential” (CHIP). How these mutations confer a proliferative advantage and result in increased risk for numerous age-related diseases remains poorly understood. We conducted a multiracial meta-analysis of epigenome-wide association studies (EWAS) of CHIP and its subtypes in four cohorts (N=8196) to elucidate the molecular mechanisms underlying CHIP and illuminate how these changes influence cardiovascular disease risk. The EWAS findings were functionally validated using human hematopoietic stem cell (HSC) models of CHIP. A total of 9615 CpGs were associated with any CHIP, 5990 with DNMT3A CHIP, 5633 with TET2 CHIP, and 6078 with ASXL1 CHIP (P <1×10−7). CpGs associated with CHIP subtypes overlapped moderately, and the genome-wide DNA methylation directions of effect were opposite for TET2 and DNMT3A CHIP, consistent with their opposing effects on global DNA methylation. There was high directional concordance between the CpGs shared from the meta-EWAS and human edited CHIP HSCs. Expression quantitative trait methylation analysis further identified transcriptomic changes associated with CHIP-associated CpGs. Causal inference analyses revealed 261 CHIP-associated CpGs associated with cardiovascular traits and all-cause mortality (FDR adjusted p-value <0.05). Taken together, our study sheds light on the epigenetic changes impacted by CHIP and their associations with age-related disease outcomes. The novel genes and pathways linked to the epigenetic features of CHIP may serve as therapeutic targets for preventing or treating CHIP-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb071f9470db1b0d43e08fc0664a4d81582bdd97" target='_blank'>
              Epigenome-wide DNA Methylation Association Study of CHIP Provides Insight into Perturbed Gene Regulation
              </a>
            </td>
          <td>
            Daniel Levy, Sara Kirmani, T. Huan, Joseph Van Amburg, R. Joehanes, M. Uddin, Ngoc Quynh Nguyen, Bing Yu, Jennifer A. Brody, M. Fornage, Jan Bressler, N. Sotoodehnia, David Ong, Fabio Puddu, James Floyd, Christie M. Ballantyne, B. Psaty, L. Raffield, Pradeep Natarajan, Karen Conneely, April P. Carson, Leslie Lange, Kendra Ferrier, N. HEARD-COSTA, Joanne M Murabito, A. Bick
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/589af1da53c29b65656184828eec590acfbf3b30" target='_blank'>
              Genetic and epigenetic instability as an underlying driver of progression and aggressive behavior in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            Timothy E Richardson, Jamie M. Walker, Dolores Hambardzumyan, Steven Brem, K. Hatanpaa, Mariano S Viapiano, Balagopal Pai, M. Umphlett, Oren J. Becher, M. Snuderl, S. McBrayer, Kalil G. Abdullah, N. Tsankova
          </td>
          <td>2024-07-16</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3850a10d0e02da63310b269aa1b7eb41498587bc" target='_blank'>
              The first complete mitochondrial genome of Grossulariaceae: Molecular features, structure recombination, and genetic evolution
              </a>
            </td>
          <td>
            Guilong Lu, Wenhua Wang, Shanshan Zhang, Guang Yang, Kun Zhang, Youxiong Que, Lan Deng
          </td>
          <td>2024-07-30</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The genetic integrity of pluripotent stem cells (PSC) is integral to their applications in research and therapy, but it is compromised by frequent development of copy number variations. Little is known about the basis of the variable genomic integrity among different PSC lines. Here we identify aneuploidies using RNA-seq and proteomics data from a panel of mouse embryonic stem cell (mESC) lines derived from 170 Diversity Outbred mice. We identified 62 lines with detectable aneuploid subpopulations and a subset of originally XX lines that lost one Chromosome X (XO). Strikingly, a much lower proportion of XX lines were aneuploid, compared to XY or XO lines. Two single-cell RNA-seq data sets demonstrated that aneuploid XY DO mESC also show lower Chromosome X gene expression and that XY mESC accumulate higher aneuploid proportions in culture than isogenic XX lines. Possible mechanisms for this protective effect of X chromosome dosage include our discovery that the lines with two active X Chromosomes have a higher expression of 2-cell-like state genes, and differential expression of X-linked tumor suppressor genes associated with DNA damage response. Highlights First genetic analysis of predisposition to aneuploidy in pluripotent stem cell cultures Chromosomal regions duplicated in aneuploid mouse embryonic stem cells are syntenic with regions overrepresented in human pluripotent stem cell lines bearing recurrent genetic abnormalities X-Chromosome dosage strongly influences susceptibility to aneuploidy in mouse embryonic stem cells and to a lesser degree in human pluripotent stem cells XX mouse embryonic stem cell lines show a higher proportion of cells in 2 cell-like state and higher expression of tumor suppressor genes associated with DNA damage response">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f853ae6e95a31cb9ad3712a21b0838a6601fe33" target='_blank'>
              Chromosome X Dosage Modulates Development of Aneuploidy in Genetically Diverse Mouse Embryonic Stem Cells
              </a>
            </td>
          <td>
            Alexander Stanton, Selcan Aydin, Daniel A. Skelly, Dylan Stavish, Kim Leonhard, Seth M. Taapken, Erik M. McIntire, Matthew Pankratz, A. Czechanski, T. Ludwig, Ted Choi, Steve P. Gygi, Ivana Barbaric, Steven C. Munger, L. Reinholdt, Martin F. Pera
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Purpose: Homologous recombination deficiency (HRD) impacts cancer treatment strategies, particularly the effectiveness of PARP inhibitors. However, the variability different HRD assays has hampered the selection of oncology patients who may benefit from these therapies. Our study aims to assess the whole genome landscape to better define HRD in a pan-cancer cohort and to contribute to harmonization of HRD detection. Methods: We employed a whole-genome sequencing WGS HRD classifier that included genome-wide features associated with HRD to analyze 580 tumor/normal paired pan-cancer samples. The HRD results were correlated retrospectively with treatment responses and were compared with commercial HRD tests in a subset of cases. Results: HRD phenotype was identified in 62 samples across various cancers including breast (19%), pancreaticobiliary (17%), gynecological (15%), prostate (8%), upper gastrointestinal (GI) (2%), and other cancers (1%). HRD cases were not confined to BRCA1/2 mutations; 24% of HRD cases were BRCA1/2 wild-type. A diverse range of HRR pathway gene alterations involved in HRD were elucidated, including biallelic mutations in FANCF, XRCC2, and FANCC, and deleterious structural variants. Comparison with results from commercial HRD assays suggests a better performance of WGS to detect HRD, based on treatment response. Conclusion: HRD is a biomarker used to determine which cancer patients would benefit from PARPi and platinum-based chemotherapy. However, a lack of harmonization of tests to determine HRD status makes it challenging to interpret their results. Our study highlights the use of comprehensive WGS analysis to predict HRD in a pan-cancer cohort, elucidates new genomic mechanisms associated with HRD, and enables an accurate identification of this phenotype, paving the way for improved outcomes in oncology care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80d347a4838be16d195accf2ef681565a4b6929" target='_blank'>
              Whole Genome Landscape Analysis of Homologous Recombination Deficiency in a Pan-Cancer Cohort
              </a>
            </td>
          <td>
            MD Majd Al Assaad, PhD Kevin Hadi, MS Max F. Levine, MD Daniela Guevara, Mph Minal Patel, MS Marvel Tranquille, BS Abigail King, BS John Otilano, MS Alissa Semaan, PhD Gunes Gundem, MS Juan S. Medina-Martínez, BS Michael Sigouros, Mph Jyothi Manohar, BS Hui-Hsuan Kuo, PhD David C. Wilkes, MD Eleni Andreopoulou, MD Eloise Chapman-Davis, MD Scott T. Tagawa, PhD Andrea Sboner, MD Allyson J. Ocean, MD Manish Shah, PhD Elli Papaemmanuil, Cora N. Sternberg, MD Kevin Holcomb, MD David M. Nanus, PhD Olivier Elemento, Juan Miguel Mosquera, J. Medina-Martinez
          </td>
          <td>2024-06-30</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1310d603b23100b789e20f7104c3139b43adef05" target='_blank'>
              Decoding the genetic program of micronucleus formation: linking chromosomal instability to human disease
              </a>
            </td>
          <td>
            Jörg Fahrer
          </td>
          <td>2024-06-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="We describe exciting recent advances in fusion-driven sarcoma etiology, from an epigenetics perspective. By exploring the current state of the field, we identify and describe the central mechanisms that determine sarcomagenesis. Further, we discuss seminal studies in translational genomics, which enabled epigenetic characterization of fusion-driven sarcomas. Important context for epigenetic mechanisms include, but are not limited to, cell cycle and metabolism, core regulatory circuitry, 3-dimensional chromatin architectural dysregulation, integration with ATP-dependent chromatin remodeling, and translational animal modeling. Paradoxically, while the genetic requirements for oncogenic transformation are highly specific for the fusion partners, the epigenetic mechanisms we as a community have uncovered are categorically very broad. This dichotomy prompts the question of whether the investigation of rare disease epigenomics should prioritize studying individual cell populations, thereby examining whether the mechanisms of chromatin dysregulation are specific to a particular tumor. We review recent advances focusing on rhabdomyosarcoma, synovial sarcoma, alveolar soft part sarcoma, clear cell sarcoma, undifferentiated round cell sarcoma, Ewing sarcoma, myxoid/round liposarcoma, epithelioid hemangioendothelioma and desmoplastic round cell tumor. The growing number of groundbreaking discoveries in the field, motivated us to anticipate further exciting advances in the area of mechanistic epigenomics and direct targeting of fusion transcription factors in the years ahead.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01bc563cf3c1cf1b5a64e47d7f0fdef6c20adb4b" target='_blank'>
              Epigenetic determinants of fusion-driven sarcomas: paradigms and challenges
              </a>
            </td>
          <td>
            Benjamin Z. Stanton, S. Pomella
          </td>
          <td>2024-06-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="The myeloma overexpressed gene (MYEOV) has been proposed to be a proto-oncogene due to high RNA transcript levels found in multiple cancers, including myeloma, breast, lung, pancreas and esophageal cancer. The presence of an open reading frame (ORF) in humans and other primates suggests protein-coding potential. Yet, we still lack evidence of a functional MYEOV protein. It remains undetermined how MYEOV overexpression affects cancerous tissues. In this work, we show that MYEOV has likely originated and may still function as an enhancer, regulating CCND1 and LTO1. Firstly, MYEOV 3′ enhancer activity was confirmed in humans using publicly available ATAC-STARR-seq data, performed on B-cell-derived GM12878 cells. We detected enhancer histone marks H3K4me1 and H3K27ac overlapping MYEOV in multiple healthy human tissues, which include B cells, liver and lung tissue. The analysis of 3D genome datasets revealed chromatin interactions between a MYEOV-3′-putative enhancer and the proto-oncogene CCND1. BLAST searches and multi-sequence alignment results showed that DNA sequence from this human enhancer element is conserved from the amphibians/amniotes divergence, with a 273 bp conserved region also found in all mammals, and even in chickens, where it is consistently located near the corresponding CCND1 orthologues. Furthermore, we observed conservation of an active enhancer state in the MYEOV orthologues of four non-human primates, dogs, rats, and mice. When studying this homologous region in mice, where the ORF of MYEOV is absent, we not only observed an enhancer chromatin state but also found interactions between the mouse enhancer homolog and Ccnd1 using 3D-genome interaction data. This is similar to the interaction observed in humans and, interestingly, coincides with CTCF binding sites in both species. Taken together, this suggests that MYEOV is a primate-specific gene with a de novo ORF that originated at an evolutionarily older enhancer region. This deeply conserved putative enhancer element could regulate CCND1 in both humans and mice, opening the possibility of studying MYEOV regulatory functions in cancer using non-primate animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f17ce82b4897c471d6bebe571f550d964a45b5c" target='_blank'>
              Evolutionarily conserved enhancer-associated features within the MYEOV locus suggest a regulatory role for this non-coding DNA region in cancer
              </a>
            </td>
          <td>
            Brigid S. A. Davidson, Juliana Estefanía Arcila-Galvis, Marco Trevisan-Herraz, Aneta Mikulasova, C. Brackley, Lisa J Russell, Daniel Rico
          </td>
          <td>2024-07-30</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common cancer in children, yet few environmental risk factors have been identified. We previously found an association between early-life tobacco smoke exposure and frequency of somatic deletions of 8 leukemia driver genes among childhood ALL patients in the California Childhood Leukemia Study. To expand analysis genome-wide and examine potential mechanisms, we conducted tumor whole-genome sequencing in 35 ALL patients, including 18 with high prenatal tobacco exposure and 17 with low exposure as determined by established epigenetic biomarkers. High tobacco exposure patients had significantly more structural variants (P < .001) and deletions (P = .001) genome-wide than low exposure patients. Investigation of off-target RAG recombination revealed that 41% of deletions in the high tobacco exposure patients were putatively RAG-mediated (full RAG motif identified at one or both breakpoints) compared with only 21% in the low exposure group (P = .001). In a multilevel model, deletions in high tobacco exposure patients were 2.44-fold (95% CI:1.13–5.38) more likely to be putatively RAG-mediated than deletions in low exposure patients. No point mutational signatures were associated with prenatal tobacco exposure. Our findings suggest that early-life tobacco smoke exposure may promote leukemogenesis by driving development of somatic deletions in pre-leukemic lymphocytes via off-target RAG recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd1dd9d24470ae77cf6358a81bf0ca861d75dad" target='_blank'>
              Early-life tobacco exposure is causally implicated in aberrant RAG-mediated recombination in childhood acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            A. D. de Smith, Tanxin Liu, Keren Xu, A. Pardeshi, Swe Swe Myint, Alice Y Kang, L. Morimoto, Michael Lieber, Joseph L Wiemels, Scott Kogan, C. Metayer
          </td>
          <td>2024-06-20</td>
          <td>Research Square</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Telomeric DNA, composed of short, direct repeats, is of crucial importance for chromosome stability. Due to intrinsic problems with replicating this DNA, the repeat tracts shorten at each cell division. Once repeat tracts become critically short, a telomeric stress signal induces cellular senescence and division arrest, which eventually may lead to devastating age-related degenerative diseases associated with dysfunctional telomers. Conversely, maintenance of telomere length by telomerase upregulation is a hallmark of cancer. Therefore, telomere length is a critical determinant of telomere function. How telomere length is established and molecular mechanisms for telomere-specific length regulation remained unknown. Here we show that subtelomeric chromatin is a determinant for how telomere equilibrium set-length is established in cis. The results demonstrate that telomerase recruitment mediated by the telomere-associated Sir4 protein is modulated on chromosome 3L in a telomere-specific way. Increased Sir4 abundance on subtelomeric heterochromatin of this specific telomere leads to telomere lengthening of only that telomere in cis, but not at other telomeres. Therefore, this work describes a mechanism for a how telomere-specific repeat tract length can be established. Further, our results will force the evaluation of telomere length away from a generalized view to a more telomere-specific consideration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2fb6915328e84462f488e5ccd4496ce592a49a" target='_blank'>
              A mechanism for telomere-specific telomere length regulation
              </a>
            </td>
          <td>
            Gabriela M. Teplitz, Emeline Pasquier, Erin Bonnell, E. D. De Laurentiis, Louise Bartle, Jean-François Lucier, Samantha L. Sholes, C. Greider, R. Wellinger
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facf4dda707b67b3b56d646f95496db5b6cc7df8" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N Anderson
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="OBJECTIVE
Ionizing radiation generates genomic instability by promoting the accumulation of chromosomal rearrangements. The oncogenic translocation RET/PTC1 is present in more than 70% of radiation-induced thyroid cancers. Both RET and CCDC6, the genes implicated in RET/PTC1, are found within common fragile sites - chromosomal regions prone to DNA breakage during slight replication stress. Given that irradiated cells become more susceptible to genomic destabilization due to the accumulation of replication-stress-related double-strand breaks (DSBs), we explored whether RET and CCDC6 exhibit DNA breakage under replicative stress several days post-irradiation of thyroid cells.


METHODS
We analyzed the dynamic of DNA replication in human thyroid epithelial cells (HThy-ori-3.1) 4 days post a 5-Gy exposure using molecular DNA combing. The DNA replication schedule was evaluated through replication-timing experiments. We implemented a ChIP-qPCR assay to determine whether the RET and CCDC6 genes break following irradiation.


RESULTS
Our study indicates that replicative stress, occurring several days post-irradiation in thyroid cells, primarily causes DSBs in the RET gene. We discovered that both the RET and CCDC6 genes undergo late replication in thyroid cells. However, only RET's replication rate is notably delayed after irradiation.


CONCLUSION
The findings suggest that post-irradiation in the RET gene causes a breakage in the replication fork, which could potentially invade another genomic area, including CCDC6. As a result, this could greatly contribute to the high prevalence of chromosomal RET/PTC rearrangements seen in patients exposed to external radiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eea49490c4342b6b738c402ce79c72ea23e5f552" target='_blank'>
              A post-irradiation-induced replication stress promotes RET proto-oncogene breakage.
              </a>
            </td>
          <td>
            Fabio Hecht, Laura Valerio, Carlos Frederico Lima Gonçalves, Marylin Harinquet, Rabii Ameziane El Hassani, Denise P. Carvalho, Stephane Koundrioukoff, Jean-Charles Cadoret, Corinne Dupuy
          </td>
          <td>2024-07-01</td>
          <td>European thyroid journal</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e6bae3255c905d58e9f2d6fd8dca01970a1953f" target='_blank'>
              Chromatin conformation and histone modification profiling across human kidney anatomic regions
              </a>
            </td>
          <td>
            Haikuo Li, Dian Li, Benjamin D. Humphreys
          </td>
          <td>2024-07-18</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bccd8d9ff2cb22d908b272a8863cb4fce2351ba" target='_blank'>
              Monitoring and quantifying replication fork dynamics with high-throughput methods
              </a>
            </td>
          <td>
            Nora Fajri, N. Petryk
          </td>
          <td>2024-06-14</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Endothelial cells (ECs) maintain vessel tone and barrier integrity, regulate blood homeostasis, and prevent the extravasation of leukocytes under normal physiological conditions. Because of the limited lifespans and batch-to-batch differences with respect to the genetic make-up of primary ECs, established immortal EC lines are extensively used for studying endothelial biology. To address this issue, the immortal endothelial cell line EA.hy926 was developed by fusing primary human umbilical vein endothelial cells (HUVECs) with human lung carcinoma A549 cells. EA.hy926 cells share a number of similar endothelial properties with HUVECs and are considered the immortal counterpart to primary HUVECs. However, the cytogenetic integrity of EA.hy926 cells is not fully elucidated. We characterized EA.hy926 cells with conventional G-banding and molecular cytogenetic techniques such as spectral karyotyping and subtelomeric fluorescence in situ hybridization. Cytogenetic analysis revealed an array of numerical and stable structural chromosomal rearrangements including one deletion, one duplication, one isochromosome, seven simple translocations, and five complex translocations in Ea.hy926 cells. These findings will advance comprehension of EA.hy926 cell biology and augment future endothelial studies, specifically in comparison studies between HUVECs and EA.hy926 cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ad1b12e69b1e161748ea2c28f86f896e19f2f6" target='_blank'>
              G-Banding and Molecular Cytogenetics Detect Novel Translocations and Cryptic Aberrations in Human Immortal Endothelial Cells
              </a>
            </td>
          <td>
            Regina Lichti Binz, R. Pathak
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Each piece of cell-free DNA (cfDNA) has a length determined by the exact metabolic conditions in the cell it belonged to at the time of cell death. The changes in cellular regulation leading to a variety of patterns, which are based on the different number of fragments with lengths up to several hundred base pairs at each of the almost three billion genomic positions, allow for the detection of disease and also the precise identification of the tissue of their origin. A Kullback-Leibler (KL) divergence computation identifies different fragment lengths and areas of the human genome, depending on the stage, for which disease samples, starting from pre-clinical disease stages, diverge from healthy donor samples. We provide examples of genes related to colorectal cancer (CRC), which our algorithm detected to belong to divergent genomic bins. The staging of CRC can be viewed as a Markov Chain and that provides a framework for studying disease progression and the types of epigenetic changes occurring longitudinally at each stage, which might aid the correct classification of a new hospital sample. In a new look to treat such data as grayscale value images, pattern recognition using artificial intelligence (AI) could be one approach to classification. In CRC, Stage I disease does not, for the most part, shed any tumor circulation, making detection difficult for established machine learning (ML) methods. This leads to the deduction that early detection, where we can only rely on changes in the metabolic patterns, can be accomplished when the information is considered in its entirety, for example by applying computer vision methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769c868029bf18fe644fcae9a2141c21f2b5a763" target='_blank'>
              Channel Capacity of Genome-Wide Cell-Free DNA Fragment Length Distribution in Colorectal Cancer
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-07-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epigenomic mechanisms are critically involved in mediation of genetic and environmental factors that underlie cancer development. Histone modifications represent highly informative epigenomic marks that reveal activation and repression of gene activities and dysregulation of transcriptional control due to tumorigenesis. Here, we present a comprehensive epigenomic and transcriptomic mapping of 18 tumor and 20 non-neoplastic tissues from non-small cell lung adenocarcinoma patients. Our profiling covers 5 histone marks including activating (H3K4me3, H3K4me1, and H3K27ac) and repressive (H3K27me3 and H3K9me3) marks and the transcriptome using only 20 mg of tissue per sample, enabled by low-input omic technologies. Using advanced integrative bioinformatic analysis, we uncovered cancer-driving signaling cascade networks, changes in 3D genome modularity, and differential expression and functionalities of transcription factors and noncoding RNAs. Many of these identified genes and regulatory molecules showed no significant change in their expression or a single epigenomic modality, emphasizing the power of integrative multimodal and multiomic analysis using patient samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1c31ae6b0dfa593dc1e04630030084d32008118" target='_blank'>
              Comprehensive multimodal and multiomic profiling reveals epigenetic and transcriptional reprogramming in lung tumors
              </a>
            </td>
          <td>
            Peiyao Wu, Zhengzhi Liu, Lina Zheng, Zirui Zhou, Wei Wang, Chang Lu
          </td>
          <td>2024-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic highly polyploid macronucleus is lost. The formation of a new macronucleus involves Programmed Genome Rearrangements, introducing DNA DSBs at approximately 45,000 loci. P. tetraurelia employs a Non-Homologous End Joining (NHEJ)-related mechanism for the systematic repair of these DSBs. Four genes encoding DNA polymerases of family X are present in the genome, one of which was found recently to colocalize with other proteins of NHEJ. The question arises as to how they make almost no error. Here we show that these enzymes are most similar to metazoan DNA polymerase λ and exhibit high fidelity through two different molecular mechanisms. Using X-ray structure determination of polymerase lambda mutants recapitulating sequence determinants of P. tetraurelia PolXs, we find both a local conformational change that involves exchanging partners in a crucial salt bridge in the active site upon binding of correct dNTPs, and a larger conformational change involving the closure of Loop3. This stabilizes the template DNA in the active site, only in the presence of the correct incoming dNTP. Differences with human pol λ and pol β are discussed. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a2c8e3978ae53a388430b008c681d13e31a282" target='_blank'>
              Mechanisms Ensuring Fidelity of Family X DNA Polymerases in Programmed DNA rearrangements in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Antonin Nourisson, Sophia Missoury, A. Haouz, Marc Delarue
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6169884636c48723540d43c960017df0f71986" target='_blank'>
              Hidden secrets of the cancer genome: unlocking the impact of non-coding mutations in gene regulatory elements
              </a>
            </td>
          <td>
            Sandra Íñiguez-Muñoz, P. Llinàs-Arias, Miquel Ensenyat-Mendez, Andrés F. Bedoya-López, Javier I. J. Orozco, Javier Cortés, Ananya Roy, Karin Forsberg-Nilsson, M. DiNome, D. Marzese
          </td>
          <td>2024-06-20</td>
          <td>Cellular and Molecular Life Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Functional copy-number alterations (fCNAs) are DNA copy-number changes with concordant differential gene expression. These are less likely to be bystander genetic lesions and could serve as robust and reproducible tumor biomarkers. To identify candidate fCNAs in neuroendocrine tumors (NETs), we integrated chromosomal microarray (CMA) and RNA-seq differential gene-expression data from 31 pancreatic (pNETs) and 33 small-bowel neuroendocrine tumors (sbNETs). Tumors were resected from 47 early-disease-progression (<24 months) and 17 late-disease-progression (>24 months) patients. Candidate fCNAs that accurately differentiated these groups in this discovery cohort were then replicated using fluorescence in situ hybridization (FISH) on formalin-fixed, paraffin-embedded (FFPE) tissues in a larger validation cohort of 60 pNETs and 82 sbNETs (52 early- and 65 late-disease-progression samples). Logistic regression analysis revealed the predictive ability of these biomarkers, as well as the assay-performance metrics of sensitivity, specificity, and area under the curve. Our results indicate that copy-number changes at chromosomal loci 4p16.3, 7q31.2, 9p21.3, 17q12, 18q21.2, and 19q12 may be used as diagnostic and prognostic NET biomarkers. This involves a rapid, cost-effective approach to determine the primary tumor site for patients with metastatic liver NETs and to guide risk-stratified therapeutic decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/976250a22dce56b41eaaf1761c3d5be047660920" target='_blank'>
              Functional Copy-Number Alterations as Diagnostic and Prognostic Biomarkers in Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Hayley Vaughn, Heather Major, Evangeline Kadera, Kendall Keck, Timothy Dunham, Qining Qian, Bartley Brown, Aaron Scott, Andrew M. Bellizzi, Terry A. Braun, Patrick Breheny, Dawn E. Quelle, James R. Howe, Benjamin W Darbro
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Pancreatic cancer is a malignancy arising from the endocrine or exocrine compartment of this organ. Tumors from exocrine origin comprise over 90% of all pancreatic cancers diagnosed. Of these, pancreatic ductal adenocarcinoma (PDAC) is the most common histological subtype. The five-year survival rate for PDAC ranged between 5 and 9% for over four decades, and only recently saw a modest increase to ∼12-13%, making this a severe and lethal disease. Like other cancers, PDAC initiation stems from genetic changes. However, therapeutic targeting of PDAC genetic drivers has remained relatively unsuccessful, thus the focus in recent years has expanded to the non-genetic factors underlying the disease pathogenesis. Specifically, it has been proposed that dynamic changes in the epigenetic landscape promote tumor growth and metastasis. Emphasis has been given to the re-organization of enhancers, essential regulatory elements controlling oncogenic gene expression, commonly marked my histone 3 lysine 4 monomethylation (H3K4me1). H3K4me1 is typically deposited by histone lysine methyltransferases (KMTs). While well characterized as oncogenes in other cancer types, recent work has expanded the role of KMTs as tumor suppressor in pancreatic cancer. Here, we review the role and translational significance for PDAC development and therapeutics of KMTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a7417d79919ae9f2eab24051f1e99fe379fd6b" target='_blank'>
              Insights into the mechanisms driven by H3K4 KMTs in pancreatic cancer.
              </a>
            </td>
          <td>
            Kayla C LaRue-Nolan, G. L. Raja Arul, Ashley N Sigafoos, Jiaqi Shi, Martin E Fernandez-Zapico
          </td>
          <td>2024-07-30</td>
          <td>The Biochemical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cancer is a heterogeneous disease that results from genetic alteration of cell cycle and proliferation controls. Identifying mutations that drive cancer, understanding cancer type specificities, and delineating how driver mutations interact with each other to establish disease is vital for identifying therapeutic vulnerabilities. Such cancer specific patterns and gene co-occurrences can be identified by studying tumor genome sequences, and networks have proven effective in uncovering relationships between sequences. We present two network-based approaches to identify driver gene patterns among tumor samples. The first approach relies on analysis using the Directed Weighted All Nearest Neighbors (DiWANN) model, which is a variant of sequence similarity network, and the second approach uses bipartite network analysis. A data reduction framework was implemented to extract the minimal relevant information for the sequence similarity network analysis, where a transformed reference sequence is generated for constructing the driver gene network. This data reduction process combined with the efficiency of the DiWANN network model, greatly lowered the computational cost (in terms of execution time and memory usage) of generating the networks enabling us to work at a much larger scale than previously possible. The DiWANN network helped us identify cancer types in which samples were more closely connected to each other suggesting they are less heterogeneous and potentially susceptible to a common drug. The bipartite network analysis provided insight into gene associations and co-occurrences. We identified genes that were broadly mutated in multiple cancer types and mutations exclusive to only a few. Additionally, weighted one-mode gene projections of the bipartite networks revealed a pattern of occurrence of driver genes in different cancers. Our study demonstrates that network-based approaches can be an effective tool in cancer genomics. The analysis identifies co-occurring and exclusive driver genes and mutations for specific cancer types, providing a better understanding of the driver genes that lead to tumor initiation and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26d7d425039cbe06da7fb2e443336941bd0e0ae5" target='_blank'>
              Network analysis of driver genes in human cancers
              </a>
            </td>
          <td>
            S. S. Patil, Steven A. Roberts, A. Gebremedhin
          </td>
          <td>2024-07-08</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Long terminal repeats (LTRs), which often contain promoter and enhancer sequences of intact endogenous retroviruses (ERVs), are known to be co-opted as cis-regulatory elements for fine-tuning host-coding gene expression. Since LTRs are mainly silenced by the deposition of repressive epigenetic marks, substantial activation of LTRs has been found in human cells after treatment with epigenetic inhibitors. Although the LTR12C family makes up the majority of ERVs activated by epigenetic inhibitors, how these epigenetically and transcriptionally activated LTR12C elements can regulate the host-coding gene expression remains unclear due to genome-wide alteration of transcriptional changes after epigenetic inhibitor treatments. Here, we specifically transactivated >600 LTR12C elements by using single guide RNA-based dCas9-SunTag-VP64, a site-specific targeting CRISPR activation (CRISPRa) system, with minimal off-target events. Interestingly, most of the transactivated LTR12C elements acquired the H3K27ac-marked enhancer feature, while only 20% were co-marked with promoter-associated H3K4me3 modifications. The enrichment of the H3K4me3 signal was intricately associated with downstream regions of LTR12C, such as internal regions of intact ERV9 or other types of retrotransposons. Here, we leverage an optimized CRISPRa system to identify two distinct epigenetic signatures that define LTR12C transcriptional activation, which modulate the expression of proximal protein-coding genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e3b49a34cf56368c1fd1327227361deee1a96b09" target='_blank'>
              Efficient activation of hundreds of LTR12C elements reveals cis-regulatory function determined by distinct epigenetic mechanisms.
              </a>
            </td>
          <td>
            Hitoshi Ohtani, Minmin Liu, G. Liang, H. J. Jang, P. Jones
          </td>
          <td>2024-06-14</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Abstract Motivation Eukaryotic cells contain organelles called mitochondria that have their own genome. Most cells contain thousands of mitochondria which replicate, even in nondividing cells, by means of a relatively error-prone process resulting in somatic mutations in their genome. Because of the higher mutation rate compared to the nuclear genome, mitochondrial mutations have been used to track cellular lineage, particularly using single-cell sequencing that measures mitochondrial mutations in individual cells. However, existing methods to infer the cell lineage tree from mitochondrial mutations do not model “heteroplasmy,” which is the presence of multiple mitochondrial clones with distinct sets of mutations in an individual cell. Single-cell sequencing data thus provide a mixture of the mitochondrial clones in individual cells, with the ancestral relationships between these clones described by a mitochondrial clone tree. While deconvolution of somatic mutations from a mixture of evolutionarily related genomes has been extensively studied in the context of bulk sequencing of cancer tumor samples, the problem of mitochondrial deconvolution has the additional constraint that the mitochondrial clone tree must be concordant with the cell lineage tree. Results We formalize the problem of inferring a concordant pair of a mitochondrial clone tree and a cell lineage tree from single-cell sequencing data as the Nested Perfect Phylogeny Mixture (NPPM) problem. We derive a combinatorial characterization of the solutions to the NPPM problem, and formulate an algorithm, MERLIN, to solve this problem exactly using a mixed integer linear program. We show on simulated data that MERLIN outperforms existing methods that do not model mitochondrial heteroplasmy nor the concordance between the mitochondrial clone tree and the cell lineage tree. We use MERLIN to analyze single-cell whole-genome sequencing data of 5220 cells of a gastric cancer cell line and show that MERLIN infers a more biologically plausible cell lineage tree and mitochondrial clone tree compared to existing methods. Availability and implementation https://github.com/raphael-group/MERLIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4e8eaf7dda8383dae6ef320720e71a06bb73607" target='_blank'>
              Joint inference of cell lineage and mitochondrial evolution from single-cell sequencing data
              </a>
            </td>
          <td>
            P. Sashittal, Viola Chen, Amey P. Pasarkar, Benjamin J. Raphael
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Whole-genome sequencing efforts has during the past decade unveiled the central role of genomic rearrangements—such as chromosomal inversions—in evolutionary processes, including local adaptation in a wide range of taxa. However, employment of reference genomes from distantly or even closely related species for mapping and the subsequent variant calling, can lead to errors and/or biases in the datasets generated for downstream analyses. Here, we capitalize on the recently generated chromosome-anchored genome assemblies for Arctic cod (Arctogadus glacialis), polar cod (Boreogadus saida), and Atlantic cod (Gadus morhua) to evaluate the extent and consequences of reference bias on population sequencing datasets (approx. 15-20x coverage) for both Arctic cod and polar cod. Our findings demonstrate that the choice of reference genome impacts population genetic statistics, including individual mapping depth, heterozygosity levels, and cross-species comparisons of nucleotide diversity (π) and genetic divergence (DXY). Further, it became evident that using a more distantly related reference genome can lead to inaccurate detection and characterization of chromosomal inversions, i.e., in terms of size (length) and location (position), due to inter-chromosomal reorganizations between species. Additionally, we observe that several of the detected species-specific inversions were split into multiple genomic regions when mapped towards a heterospecific reference. Inaccurate identification of chromosomal rearrangements as well as biased population genetic measures could potentially lead to erroneous interpretation of species-specific genomic diversity, impede the resolution of local adaptation, and thus, impact predictions of their genomic potential to respond to climatic and other environmental perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e083b1a379647c0ff911211cf65b6e9e0db94c8" target='_blank'>
              Reference genome bias in light of species-specific chromosomal reorganization and translocations
              </a>
            </td>
          <td>
            Marius F. Maurstad, S. Hoff, José Cerca, M. Ravinet, Ian R. Bradbury, K. Jakobsen, K. Præbel, S. Jentoft
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Gene expression regulation is one of the most fundamental cellular processes, enabling the activation of a gene to produce either the translatable protein-coding transcript (mRNA) or a functional non-coding RNA with gene regulatory functions, ultimately determining cell identity and function. Although gene expression regulation can occur at transcriptional, translational, and post-translational levels, transcription initiation is the first and the most important step in gene expression, facilitating the transfer of biological information from DNA to protein. Enhancers and super-enhancers are among the master regulators of tissue- and cell-specific transcription regulation involved in cell differentiation and tumor formation. Despite four decades passing since the first discovery of enhancers in eukaryotes and extensive efforts undertaken to identify enhancers on a genomic scale during the last decade, the discovery of enhancers still faces certain limitations and needs further investigation. The perturbation of enhancer function due to genetic or epigenetic changes is closely linked to a range of human disorders, including the development and progression of cancers. Thus, the detection of early cancer-related enhancer activity and the subsequent normalization of expression abnormalities using enhancer-targeting CRISPR epigenetic editing, as well as enhancer-targeting pharmaceuticals, are regarded as groundbreaking therapeutic tactics in preclinical stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782ddfe76cab2a16620844e3e84b112d8c90071b" target='_blank'>
              Enhancers and super-enhancers as master regulators in cancer
              </a>
            </td>
          <td>
            P. Sarvari, Pourya Sarvari, Ivonne Ramírez-Díaz, Karla Rubio
          </td>
          <td>2024-07-24</td>
          <td>INNOSC Theranostics and Pharmacological Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b3ada799665e69e111a38c4d8d26afa853f4d7" target='_blank'>
              MethPhaser: methylation-based long-read haplotype phasing of human genomes
              </a>
            </td>
          <td>
            Yilei Fu, Sergey Aganezov, M. Mahmoud, J. Beaulaurier, S. Juul, T. Treangen, F. Sedlazeck
          </td>
          <td>2024-06-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Copy number variation (CNV) refers to the number of copies of a specific sequence in a genome and is a type of chromatin structural variation. The development of the Hi‐C technique has empowered research on the spatial structure of chromatins by capturing interactions between DNA fragments. We utilized machine‐learning methods including the linear transformation model and graph convolutional network (GCN) to detect CNV events from Hi‐C data and reveal how CNV is related to three‐dimensional interactions between genomic fragments in terms of the one‐dimensional read count signal and features of the chromatin structure. The experimental results demonstrated a specific linear relation between the Hi‐C read count and CNV for each chromosome that can be well qualified by the linear transformation model. In addition, the GCN‐based model could accurately extract features of the spatial structure from Hi‐C data and infer the corresponding CNV across different chromosomes in a cancer cell line. We performed a series of experiments including dimension reduction, transfer learning, and Hi‐C data perturbation to comprehensively evaluate the utility and robustness of the GCN‐based model. This work can provide a benchmark for using machine learning to infer CNV from Hi‐C data and serves as a necessary foundation for deeper understanding of the relationship between Hi‐C data and CNV.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed5363304e30f626dbaaf6d8f26635e48cef0946" target='_blank'>
              Effectiveness of machine learning at modeling the relationship between Hi‐C data and copy number variation
              </a>
            </td>
          <td>
            Yuyang Wang, Yu Sun, Zeyu Liu, Bijia Chen, Hebing Chen, Chao Ren, Xuanwei Lin, Pengzhen Hu, Peiheng Jia, Xiang Xu, Kang Xu, Ximeng Liu, Hao Li, Xiaochen Bo
          </td>
          <td>2024-07-06</td>
          <td>Quantitative Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In recent years, there has been increased focus on exploring the role the non-protein-coding genome plays in Mendelian disorders. One class of particular interest is long non-coding RNAs (lncRNAs), which has recently been implicated in the regulation of diverse molecular processes. However, because lncRNAs do not encode protein, there is uncertainty regarding what constitutes a pathogenic lncRNA variant, and thus annotating such elements is challenging. The Developmental Genome Anatomy Project (DGAP) and similar projects recruit individuals with apparently balanced chromosomal abnormalities (BCAs) that disrupt or dysregulate genes in order to annotate the human genome. We hypothesized that rearrangements disrupting lncRNAs could be the underlying genetic etiology for the phenotypes of a subset of these individuals. Thus, we assessed 279 cases with BCAs and selected 191 cases with simple BCAs (breakpoints at only two genomic locations) for further analysis of lncRNA disruptions. From these, we identified 66 cases in which the chromosomal rearrangements directly disrupt lncRNAs. Strikingly, the lncRNAs MEF2C-AS1 and ENSG00000257522 are each disrupted in two unrelated cases. Furthermore, in 30 cases, no genes of any other class aside from lncRNAs are directly disrupted, consistent with the hypothesis that lncRNA disruptions could underly the phenotypes of these individuals. To showcase the power of this genomic approach for annotating lncRNAs, here we focus on clinical reports and genetic analysis of two individuals with BCAs and additionally highlight six individuals with likely developmental etiologies due to lncRNA disruptions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4494659e85fb73d7f4a80f0d544797b5da37da2" target='_blank'>
              Rare germline disorders implicate long non-coding RNAs disrupted by chromosomal structural rearrangements
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-06-19</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The expression of tumor-specific antigens during cancer progression can trigger an immune response against the tumor. Here, we investigate if microproteins encoded by noncanonical open reading frames (ncORFs) are a relevant source of tumor-specific antigens. We analyze RNA sequencing data from 117 hepatocellular carcinoma (HCC) tumors and matched healthy tissue together with ribosome profiling and immunopeptidomics data. Combining human leukocyte antigen–epitope binding predictions and experimental validation experiments, we conclude that around 40% of the tumor-specific antigens in HCC are likely to be derived from ncORFs, including two peptides that can trigger an immune response in humanized mice. We identify a subset of 33 tumor-specific long noncoding RNAs expressing novel cancer antigens shared by more than 10% of the HCC samples analyzed, which, when combined, cover a large proportion of the patients. The results of the study open avenues for extending the range of anticancer vaccines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/537c5fce6ae2d9f52e6619d811174a75e25d4f1f" target='_blank'>
              Microproteins encoded by noncanonical ORFs are a major source of tumor-specific antigens in a liver cancer patient meta-cohort
              </a>
            </td>
          <td>
            Marta E. Camarena, Patrick Theunissen, Marta Ruiz, Jorge Ruiz-Orera, Beatriz Calvo-Serra, Robert Castelo, Carla Castro, Pablo Sarobe, Puri Fortes, J. Perera-Bel, M. M. Albà
          </td>
          <td>2024-07-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Oncolytic adenoviruses are in development as immunotherapeutic agents for solid tumors. Their efficacy is in part dependent on their ability to replicate in tumors. It is, however, difficult to obtain evidence for intratumoral oncolytic adenovirus replication if direct access to the tumor is not possible. Detection of systemic adenovirus DNA, which is sometimes used as a proxy, has limited value because it does not distinguish between the product of intratumoral replication and injected virus that did not replicate. Therefore, we investigated if detection of virus-associated RNA (VA RNA) by RT-qPCR on liquid biopsies could be used as an alternative. We found that VA RNA is expressed in adenovirus-infected cells in a replication-dependent manner and is secreted by these cells in association with extracellular vesicles. This allowed VA RNA detection in the peripheral blood of a preclinical in vivo model carrying adenovirus-injected human tumors and on liquid biopsies from a human clinical trial. Our results confirm that VA RNA detection in liquid biopsies can be used for minimally invasive assessment of oncolytic adenovirus replication in solid tumors in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ee6bcefa097df46ba47fa82803bc95fe0eb3034" target='_blank'>
              Quantitative Virus-Associated RNA Detection to Monitor Oncolytic Adenovirus Replication
              </a>
            </td>
          <td>
            Tereza Brachtlova, Jing Li, I. H. van der Meulen-Muileman, Femke Sluiter, Willem von Meijenfeldt, Isabella Witte, Sanne Massaar, Ruben van den Oever, Jeroen de Vrij, V. V. van Beusechem
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f019d23b5adc09f83ebfe644966cdb6ede88d903" target='_blank'>
              Immune evasion impacts the landscape of driver genes during cancer evolution
              </a>
            </td>
          <td>
            Lucie E. Gourmet, A. Sottoriva, Simon Walker-Samuel, M. Secrier, L. Zapata
          </td>
          <td>2024-06-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8e9f04f74a6159b9e49f1e14a0705e366ea67c" target='_blank'>
              Epigenetic alterations affecting hematopoietic regulatory networks as drivers of mixed myeloid/lymphoid leukemia
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Stanley van Herk, Margit Nuetzel, Aniko Sijs-Szabo, Noelia Díaz, Katherine Kelly, Claudia A. J. Erpelinck-Verschueren, Lucia Schwarzfischer-Pfeilschifter, Hanna Stanewsky, Ute Ackermann, Dagmar Glatz, Johanna Raithel, A. Fischer, Sandra Pohl, Anita Rijneveld, Juan M. Vaquerizas, Christian Thiede, C. Plass, Bas J. Wouters, Ruud Delwel, Michael Rehli, C. Gebhard
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Emerging immunotherapies such as immune checkpoint blockade (ICB) and chimeric antigen receptor T-cell (CAR-T) therapy have revolutionized cancer treatment and have improved the survival of patients with multiple cancer types. Despite this success many patients are unresponsive to these treatments or relapse following treatment. CRISPR activation and knockout (KO) screens have been used to identify novel single gene targets that can enhance effector T cell function and promote immune cell targeting and eradication of tumors. However, cancer cells often employ multiple genes to promote an immunosuppressive pathway and thus modulating individual genes often has a limited effect. Paralogs are genes that originate from common ancestors and retain similar functions. They often have complex effects on a particular phenotype depending on factors like gene family similarity, each individual gene’s expression and the physiological or pathological context. Some paralogs exhibit synthetic lethal interactions in cancer cell survival; however, a thorough investigation of paralog pairs that could enhance the efficacy of cancer immunotherapy is lacking. Here we introduce a sensitive computational approach that uses sgRNA sets enrichment analysis to identify cancer-intrinsic paralog pairs which have the potential to synergistically enhance T cell-mediated tumor destruction. We have further developed an ensemble learning model that uses an XGBoost classifier and incorporates features such as gene characteristics, sequence and structural similarities, protein-protein interaction (PPI) networks, and gene coevolution data to predict paralog pairs that are likely to enhance immunotherapy efficacy. We experimentally validated the functional significance of these predicted paralog pairs using double knockout (DKO) of identified paralog gene pairs as compared to single gene knockouts (SKOs). These data and analyses collectively provide a sensitive approach to identify previously undetected paralog pairs that can enhance cancer immunotherapy even when individual genes within the pair has a limited effect.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9398918df5088c04d21b0123727e4ddeb607a57" target='_blank'>
              Sensitive detection of synthetic response to cancer immunotherapy driven by gene paralog pairs
              </a>
            </td>
          <td>
            Chuanpeng Dong, Feifei Zhang, Emily He, Ping Ren, Nipun Verma, Xinxin Zhu, Di Feng, Hongyu Zhao, Sidi Chen
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Meiotic recombination is not only a key mechanism for sexual adaptation in eukaryotes but crucial for the accumulation of beneficial alleles in breeding populations. The effective manipulation of recombination requires, however, a better understanding of the mechanisms regulating the rate and distribution of recombination events in genomes. Here, we identified the genomic features that best explain the recombination variation among a diverse set of segregating populations of barley at a resolution of 1 Mbp and investigated how methylation and structural variants determine recombination hotspots and coldspots at a high-resolution of 10 kb. Hotspots were found to be in proximity to genes and the genetic effects not assigned to methylation were found to be the most important factor explaining differences in recombination rates among populations along with the methylation and the parental sequence divergence. Interestingly, the inheritance of a highly-methylated genomic fragment from one parent only was enough to generate a coldspot, but both parents must be equally low methylated at a genomic segment to allow a hotspot. The parental sequence divergence was shown to have a sigmoidal correlation with recombination indicating an upper limit of mismatch among homologous chromosomes for CO formation. Structural variants (SVs) were shown to suppress COs, and their type and size were not found to influence that effect. Methylation and SVs act jointly determining the location of coldspots in barley and the weight of their relative effect depends on the genomic region. Our findings suggest that recombination in barley is highly predictable, occurring mostly in multiple short sections located in the proximity to genes and being modulated by local levels of methylation and SV load.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff6d3ab3ee7ea3ec56157a8f8804849a64467540" target='_blank'>
              The role of methylation and structural variants in shaping the recombination landscape of barley
              </a>
            </td>
          <td>
            F. Casale, Christopher Arlt, M. Kühl, Jinquan Li, J. Engelhorn, Thomas Hartwig, Benjamin Stich
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Cancer genomic studies have identified frequent alterations in components of the SWI/SNF (SWItch/Sucrose Non- Fermenting) chromatin remodeling complex including SMARCA4 and ARID1A. Importantly, clinical reports indicate that SMARCA4-mutant lung cancers respond poorly to immunotherapy and have dismal prognosis. However, the mechanistic basis of immunotherapy resistance is unknown. Here, we corroborated the clinical findings by using immune-humanized, syngeneic, and genetically engineered mouse models of lung cancer harboring SMARCA4 deficiency. Specifically, we show that SMARCA4 loss caused decreased response to anti-PD1 immunotherapy associated with significantly reduced infiltration of dendritic cells (DCs) and CD4+ T cells into the tumor microenvironment (TME). Mechanistically, we show that SMARCA4 loss in tumor cells led to profound downregulation of STING, IL1β and other components of the innate immune system as well as inflammatory cytokines that are required for efficient recruitment and activity of immune cells. We establish that this deregulation of gene expression is caused by cancer cell-intrinsic reprogramming of the enhancer landscape with marked loss of chromatin accessibility at enhancers of genes involved in innate immune response such as STING, IL1β, type I IFN and inflammatory cytokines. Interestingly, we observed that transcription factor NF-κB binding motif was highly enriched in enhancers that lose accessibility upon SMARCA4 deficiency. Finally, we confirmed that SMARCA4 and NF-κB co-occupy the same genomic loci on enhancers associated with STING and IL1β, indicating a functional interplay between SMARCA4 and NF-κB. Taken together, our findings provide the mechanistic basis for the poor response of SMARCA4-mutant tumors to anti-PD1 immunotherapy and establish a functional link between SMARCA4 and NF-κB on innate immune and inflammatory gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3fdeb7ebdd844e1757256a995b264456319e406" target='_blank'>
              SMARCA4 mutation induces tumor cell-intrinsic defects in enhancer landscape and resistance to immunotherapy
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Yonathan Lissanu
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64f1aced03eb2794aa498d58e60fbc81d9cded4d" target='_blank'>
              Homeobox and Polycomb target gene methylation in human solid tumors
              </a>
            </td>
          <td>
            Reid Blanchett, Kin H. Lau, Gerd P Pfeifer
          </td>
          <td>2024-06-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The significance of p53’s primary and secondary tumor suppressor programs cannot be overstated. A context- and stress-dependent transcription factor, p53 accumulates to mount its most well-characterized programs in response to a variety of stressors, most notably DNA damage. As cells and tissues never exist in a complete absence of stress, a small amount of p53 exists in cells under physiologic stress, detectable by chromatin immunoprecipitation and sequencing, termed basal p53. Recently, we and others have shown that basal p53 is sufficient to regulate tumor suppressor function. Furthermore, others have suggested the possibility that p53 accumulation in response to experimental stress may be dispensable for its tumor suppression. We previously showed basal p53 occupancy and regulation of known tumor suppressor genes, including PTEN and PHLDA3, in non-transformed breast cells, but this study was limited by experimental stress inherent to cell culture. Given the lack of global characterization of the basal p53 landscape under non-malignant physiologic stress in vivo, we utilized a multi-omics approach to define the murine basal p53 epigenome and its transcriptional program in various normal murine tissues. In this study, we observed basal p53 binding to cis regions of multiple tumor suppressor genes in different tissues, of which some showed p53-dependent regulation of their expression, including Phlda3, Bbc3, Xaf1, and itself. Furthermore, the vast majority of basal p53 target genes were not induced upon irradiation, suggesting basal p53 operates a transcriptional program that is largely distinct from its DNA damage response. Similarly, the basal p53 target gene repertoire is unique to each tissue type.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b8d9de29794e5867d3186480a83434095aa1f8b" target='_blank'>
              Basal p53 maintains a distinct transcriptional program from irradiated p53 in tissue, including tumor suppressors
              </a>
            </td>
          <td>
            Royce W. Zhou, Lois Silverman, J. Manfredi, Ramon E. Parsons
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5073eeabc50bea9fdc09e2d2eeb42d91221fda8" target='_blank'>
              Integrative genomics identifies SHPRH as a tumor suppressor gene in lung adenocarcinoma that regulates DNA damage response.
              </a>
            </td>
          <td>
            A. Nagelberg, T. Sihota, Yu-Chi Chuang, Rocky Shi, Justine L M Chow, John English, C. MacAulay, Stephen Lam, Wan L Lam, William W Lockwood
          </td>
          <td>2024-06-18</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Naïve pluripotent stem cells (nPSC) frequently undergo pathological and not readily reversible loss of DNA methylation marks at imprinted gene loci. This abnormality poses a hurdle for using pluripotent cell lines in biomedical applications and underscores the need to identify the causes of imprint instability in these cells. We show that nPSCs from inbred mouse strains exhibit pronounced strain-specific susceptibility to locus-specific deregulation of imprinting marks during reprogramming to pluripotency and upon culture with MAP kinase inhibitors, a common approach to maintain naïve pluripotency. Analysis of genetically highly diverse nPSCs from the Diversity Outbred (DO) stock confirms that genetic variation is a major determinant of epigenome stability in pluripotent cells. We leverage the variable DNA hypomethylation in DO lines to identify several trans-acting quantitative trait loci (QTLs) that determine epigenome stability at either specific target loci or genome-wide. Candidate factors encoded by two multi-target QTLs on chromosomes 4 and 17 suggest specific transcriptional regulators that contribute to DNA methylation maintenance in nPSCs. We propose that genetic variants represent candidate biomarkers to identify pluripotent cell lines with desirable properties and might serve as entry points for the targeted engineering of nPSCs with stable epigenomes. Highlights Naïve pluripotent stem cells from distinct inbred mouse strains exhibit variable DNA methylation levels at imprinted gene loci. The vulnerability of pluripotent stem cells to loss of genomic imprinting caused by MAP kinase inhibition strongly differs between inbred mouse strains. Genetically diverse pluripotent stem cell lines from Diversity Outbred mouse stock allow the identification of quantitative trait loci controlling DNA methylation stability. Genetic variants may serve as biomarkers to identify naïve pluripotent stem cell lines that are epigenetically stable in specific culture conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fcf4c9c2e254cef424889809e18c41add6362bb" target='_blank'>
              Genetic variation modulates susceptibility to aberrant DNA hypomethylation and imprint deregulation in naïve pluripotent stem cells
              </a>
            </td>
          <td>
            C. Parikh, RA Glenn, Y. Shi, K. Chatterjee, EE Swanzey, S. Singer, SC Do, Y. Zhan, Y. Furuta, M. Tahiliani, E. Apostolou, A. Polyzos, R. Koche, JG Mezey, T. Vierbuchen, M. Stadtfeld
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, Bert van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="CRNDE is an oncogene expressed as a long non-coding RNA. However, our team previously reported that the CRNDE gene also encodes a micropeptide, CRNDEP. The amino acid sequence of CRNDEP has recently been revealed by other researchers, too. This study aimed to investigate genetic alterations within the CRNDEP-coding region of the CRNDE gene, methylation profiling of this gene, and CRNDEP expression analysis. All investigations were performed on clinical material from patients with ovarian tumors of diverse aggressiveness. We found that CRNDEP levels were significantly elevated in highly aggressive tumors compared to benign neoplasms. Consistently, a high level of this micropeptide was a negative, independent, prognostic, and predictive factor in high-grade ovarian cancer (hgOvCa) patients. The cancer-promoting role of CRNDE(P), shown in our recent study, was also supported by genetic and epigenetic results obtained herein, revealing no CRNDEP-disrupting mutations in any clinical sample. Moreover, in borderline ovarian tumors (BOTS), but not in ovarian cancers, the presence of a single nucleotide polymorphism in CRNDE, rs115515594, significantly increased the risk of recurrence. Consistently, in BOTS only, the same genetic variant was highly overrepresented compared to healthy individuals. We also discovered that hypomethylation of CRNDE is associated with increased aggressiveness of ovarian tumors. Accordingly, hypomethylation of this gene’s promoter/first exon correlated with hgOvCa resistance to chemotherapy, but only in specimens with accumulation of the TP53 tumor suppressor protein. Taken together, these results contribute to a better understanding of the role of CRNDE(P) in tumorigenesis and potentially may lead to improvements in screening, diagnosis, and treatment of ovarian neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41a5d81fb227910421858aeef5200b2eb49cb75b" target='_blank'>
              The Clinical Significance of CRNDE Gene Methylation, Polymorphisms, and CRNDEP Micropeptide Expression in Ovarian Tumors
              </a>
            </td>
          <td>
            L. A. Szafron, R. Iwanicka-Nowicka, A. Podgórska, Arkadiusz M Bonna, Piotr Sobiczewski, J. Kupryjańczyk, L. Szafron
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="
 Introduction: Kras Gain of Function mutations are frequently detected in lung, colorectal and pancreatic cancers, in addition to others. Activating Kras mutations results in a constantly active protein, which we reasoned is quite unphysiological, yet it is tolerated by cells since this mutation is seen in hyperplasia. Activating this oncogene is insufficient alone for malignant transformation. Mutations in the tumor suppressor gene (TSG) p53 (the most frequently mutated gene in human cancer), cooperate with mutant Kras and is sufficient to permit full display of the actions of oncogenic Ras (as confirmed in Genetically Engineered Mouse Models and in clinical cases). The phenomenon of oncogene addiction may in fact be the result of obligatory requirements (cellular metabolism or otherwise) brought on by the constant oncogene signaling. This required adjustment in cellular circuitry can be afforded by specific cooperating TSG and since it too is a DNA constant change, becomes an new rigid reality for a cell with a given set of oncogene-TSG pair. p53 mutational spectrum was described as ‘enigmatic’ presumably because they almost never completely abrogate the function of this major regulator suggesting to us a possible essential mechanistic role for the retained transcriptional targets within the (well annotated) p53 transcriptional network. This could represent a putative synthetic lethality opportunity against activated Kras, an oncogene that has proven difficult to drug. We hypothesized that specific mutations in p53 with their respective transcriptional lesions cooperate with unique mutant Kras in tissue specific manner identifying a short list of gene targets for synthetic lethality experiments. Methods: To examine this hypothesis, we analyzed the TCGA database to determine a conserved pair cooperation between common Kras 12C; D or V and the top 6 reported p53 (hot spot) mutations [on residues 175; 245;248;249;273 and 282] in a stage-agnostic manner. To examine whether those putative interactions are cell-type specific, we performed this analysis in 3 different histologies (lung, colon, and pancreas). Results: The results suggested a non-random distribution of p53 mutants among the Kras driven cancers. KRAS (12 C, 12 V, 12 D were reported in 80% of Colon, Pancreas, and Lung Cancer. p53 (175 - 30%) (245 - 30%) (248 - 15%) (249 - 12%) (273 - 6.5%) (282 - 6.5%). More detailed analysis is planned for the poster session. Conclusion: Despite having a single activated oncogene, the distribution of the cooperating p53 mutations is nonrandom so examining the transcriptionally retained gene list represents a novel approach to explore for gene editing experiments. This provides and approach to drug the addiction of cancers to their oncogenes in a cancer specific, and occasionally to target essential proto-oncogene such as Myc where direct inhibition is highly undesirable due to its physiologic roles.
 Citation Format: Nishanth Thalambedu, Shallya Anand, Haya Safar, Farah Mazahreh, Ahmad Mazin M. Safar. Exploring the putative Kras-p53 mutational interface for vulnerability [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fa6ccd0c87ee81cbb16a2e3d13e489ca625071" target='_blank'>
              Abstract B009: Exploring the putative Kras-p53 mutational interface for vulnerability
              </a>
            </td>
          <td>
            Nishanth Thalambedu, Shallya Anand, Haya Safar, Farah Mazahreh, A. M. Safar
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Brain tumors and genomics have a long-standing history given that glioblastoma was the first cancer studied by the cancer genome atlas. The numerous and continuous advances through the decades in sequencing technologies have aided in the advanced molecular characterization of brain tumors for diagnosis, prognosis, and treatment. Since the implementation of molecular biomarkers by the WHO CNS in 2016, the genomics of brain tumors has been integrated into diagnostic criteria. Long-read sequencing, also known as third generation sequencing, is an emerging technique that allows for the sequencing of longer DNA segments leading to improved detection of structural variants and epigenetics. These capabilities are opening a way for better characterization of brain tumors. Here, we present a comprehensive summary of the state of the art of third-generation sequencing in the application for brain tumor diagnosis, prognosis, and treatment. We discuss the advantages and potential new implementations of long-read sequencing into clinical paradigms for neuro-oncology patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6562eb4f867a93f803e734b486b9cb5acc9b3bf" target='_blank'>
              Long-read sequencing for brain tumors
              </a>
            </td>
          <td>
            William J. Shelton, Sara Zandpazandi, J. S. Nix, Murat Gokden, Michael Bauer, Katie Rose, Christopher Wardell, O. Vaske, Analiz Rodriguez, Brent T. Harris, Heather Ames, Kathryn Eschbacher
          </td>
          <td>2024-06-10</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The DNA modifications, 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), represent powerful epigenetic regulators of temporal and spatial gene expression. Yet, how the cooperation of these genome-wide, epigenetic marks determine unique transcriptional signatures across different brain cell populations is unclear. Here we applied Nanopore sequencing of native DNA to obtain a complete, genome-wide, single-base resolution atlas of 5mC and 5hmC modifications in neurons, astrocytes and microglia in the mouse cortex (99% genome coverage, 40 million CpG sites). In tandem with RNA sequencing, analysis of 5mC and 5hmC patterns across cell types reveals astrocytes drive uniquely high brain 5hmC levels and support two decades of research regarding methylation patterns, gene expression and alternative splicing, benchmarking this resource. As such, we provide the most comprehensive DNA methylation data in mouse brain as an interactive, online tool (NAM-Me, https://olsenlab.shinyapps.io/NAMME/) to serve as a resource dataset for those interested in the methylome landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60076a55629be60f22fe1afccd00156f5bc051e7" target='_blank'>
              Decoding the Epigenetic Landscape: Insights into 5mC and 5hmC Patterns in Mouse Cortical Cell Types
              </a>
            </td>
          <td>
            Xiaoran Wei, Jiangtao Li, Zuolin Cheng, Songtao Wei, Guoqiang Yu, Michelle L. Olsen
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract The DNA‐dependent protein kinase (DNA‐PK), catalytic subunit, also known as DNA‐PKcs, is complexed with the heterodimer Ku70/Ku80 to form DNA‐PK holoenzyme, which is well recognized as initiator in the nonhomologous end joining (NHEJ) repair after double strand break (DSB). During NHEJ, DNA‐PKcs is essential for both DNA end processing and end joining. Besides its classical function in DSB repair, DNA‐PKcs also shows multifaceted functions in various biological activities such as class switch recombination (CSR) and variable (V) diversity (D) joining (J) recombination in B/T lymphocytes development, innate immunity through cGAS–STING pathway, transcription, alternative splicing, and so on, which are dependent on its function in NHEJ or not. Moreover, DNA‐PKcs deficiency has been proven to be related with human diseases such as neurological pathogenesis, cancer, immunological disorder, and so on through different mechanisms. Therefore, it is imperative to summarize the latest findings about DNA‐PKcs and diseases for better targeting DNA‐PKcs, which have shown efficacy in cancer treatment in preclinical models. Here, we discuss the multifaceted roles of DNA‐PKcs in human diseases, meanwhile, we discuss the progresses of DNA‐PKcs inhibitors and their potential in clinical trials. The most updated review about DNA‐PKcs will hopefully provide insights and ideas to understand DNA‐PKcs associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc4d3c7ebac68c037dd494f0a2f6295735d5a4a5" target='_blank'>
              The multifaceted functions of DNA‐PKcs: implications for the therapy of human diseases
              </a>
            </td>
          <td>
            Jinghong Wu, Liwei Song, Mingjun Lu, Qing Gao, Shaofa Xu, Ping-Kun Zhou, Teng Ma
          </td>
          <td>2024-06-19</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="A recent publication describing the assembly of the Y chromosomes of 43 males was remarkable not only for its ambitious technical scope, but also for the startling suggestion that the boundary of the pseudoautosomal region 1 (PAR1), where the human X and Y chromosomes engage in crossing-over during male meiosis, lies 500 kb distal to its previously reported location1. Where is the boundary of the human PAR1? We first review the evidence that mapped the PAR boundary, or PAB, before the human genome draft sequence was produced, then examine post-genomic datasets for evidence of crossing-over between the X and Y, and lastly re-examine the sequence assemblies presented by Hallast and colleagues to see whether they support a more distal PAB. We find ample evidence of X-Y crossovers throughout the 500 kb in question, some as close as 246 bp to the previously reported PAB. Our new analyses, combined with previous studies over the past 40 years, provide overwhelming evidence to support the original position, and narrow the probable location of the PAB to a 201-bp window.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d811c6ff23bfcae84522ef94ae21f045cd0bf5e" target='_blank'>
              Where is the boundary of the human pseudoautosomal region?
              </a>
            </td>
          <td>
            D. W. Bellott, Jennifer F. Hughes, H. Skaletsky, Erik C. Owen, David C. Page
          </td>
          <td>2024-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198f78f5b5d06b28964e8fe5554ec81d0dc67f78" target='_blank'>
              Pan-cancer exploration of oncogenic and clinical impacts revealed that HOXA9 is a diagnostic indicator of tumorigenesis
              </a>
            </td>
          <td>
            U. S. Shenoy, Dhanraj Salur Basavarajappa, S. Kabekkodu, R. Radhakrishnan
          </td>
          <td>2024-06-21</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Human APOBEC single-strand (ss) specific DNA and RNA cytidine deaminases change cytosines to uracils and function in antiviral innate immunity, RNA editing, and can cause hypermutation in chromosomes. The resulting uracils can be directly replicated, resulting in C to T mutations, or uracil-DNA glycosylase can convert the uracils to abasic (AP) sites which are then fixed as C to T or C to G mutations by translesion DNA polymerases. We noticed that in yeast and in human cancers, contributions of C to T and C to G mutations depends on the origin of ssDNA mutagenized by APOBECs. Since ssDNA in eukaryotic genomes readily binds to replication protein A (RPA) we asked if RPA could affect APOBEC-induced mutation spectrum in yeast. For that purpose, we expressed human APOBECs in the wild-type yeast and in strains carrying a hypomorph mutation rfa1-t33 in the large RPA subunit. We confirmed that the rfa1-t33 allele can facilitate mutagenesis by APOBECs. We also found that the rfa1-t33 mutation changed the ratio of APOBEC3A-induced T to C and T to G mutations in replicating yeast to resemble a ratio observed in long-persistent ssDNA in yeast and in cancers. We present the data suggesting that RPA may shield APOBEC formed uracils in ssDNA from Ung1, thereby facilitating C to T mutagenesis through the accurate copying of uracils by replicative DNA polymerases. Unexpectedly, we also found that for uracils shielded from Ung1 by wild-type RPA the mutagenic outcome is reduced in the presence of translesion DNA polymerase zeta.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106b9d766b6ab541cb0c665a0826fbe3df05d8f1" target='_blank'>
              Hypomorphic mutation in the large subunit of replication protein A affects mutagenesis by human APOBEC cytidine deaminases in yeast
              </a>
            </td>
          <td>
            Matthew S. Dennen, Z. Kockler, Steven A. Roberts, Adam B. Burkholder, L. Klimczak, D. Gordenin
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract CHEK2 is considered to be involved in homologous recombination repair (HRR). Individuals who have germline pathogenic variants (gPVs) in CHEK2 are at increased risk to develop breast cancer and likely other primary cancers. PARP inhibitors (PARPi) have been shown to be effective in the treatment of cancers that present with HRR deficiency—for example, caused by inactivation of BRCA1/2. However, clinical trials have shown little to no efficacy of PARPi in patients with CHEK2 gPVs. Here, we show that both breast and non-breast cancers from individuals who have biallelic gPVs in CHEK2 (germline CHEK2 deficiency) do not present with molecular profiles that fit with HRR deficiency. This finding provides a likely explanation why PARPi therapy is not successful in the treatment of CHEK2-deficient cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d536ac1ca51475b2c207342645867aeb58af501f" target='_blank'>
              The genomic landscape of breast and non-breast cancers from individuals with germline CHEK2 deficiency
              </a>
            </td>
          <td>
            Snežana Hinić, R. S. van der Post, L. Vreede, J. Schuurs-Hoeijmakers, Saskia Koene, Erik A. M. Jansen, Franziska Bervoets-Metge, Arjen R Mensenkamp, N. Hoogerbrugge, M. Ligtenberg, Richarda M. de Voer
          </td>
          <td>2024-06-07</td>
          <td>JNCI Cancer Spectrum</td>
          <td>1</td>
          <td>65</td>
        </tr>

        <tr id="Long-read sequencing technologies have revolutionized genome assembly, addressing the limitations of short reads by spanning large, repetitive, and complex genomic regions. These advances have led to near-complete chromosome assemblies for numerous organisms, which are invaluable to various biological and medical research fields. However, regions with complex repetitive structure continue to represent a challenge for genome assembly algorithms. Specifically in these regions, robust and comprehensive solutions for the assessment of assembly accuracy and completeness do not exist. In this study we focus on the assembly of biomedically important antibody-encoding immunoglobulin (IG) loci, which are characterized by complex duplications and repeat structures. High-quality full-length assemblies for these loci are critical for resolving haplotype-level annotations of IG genes, without which, functional and evolutionary studies of antibody immunity across vertebrates are not tractable. We developed a pipeline, “CloseRead”, that generates multiple assembly verification metrics for analysis and visualization; these metrics expand upon those of existing quality assessment tools. Using CloseRead, we systematically assessed the accuracy and completeness of IG loci in publicly available assemblies of 74 vertebrate species, identifying regions impacted by assembly errors. For a subset of species, we were able to correct assembly errors through targeted reassembly. We also demonstrated that inspecting assembly graphs for problematic regions can both identify the root cause of assembly errors and illuminate solutions for improving erroneous assemblies. Together, our analysis demonstrated the utility of assembly assessment for improving IG locus assembly completeness and accuracy. The application of this approach will be critical to ensuring that we build the most complete and accurate curated sets of IG genes across species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9515ce08ab4aaa7854aed97585cbf854c5828698" target='_blank'>
              Unveiling Assembly Errors in Immunoglobulin Loci: A Comprehensive Evaluation of Long-read Genome Assemblies Across Vertebrates
              </a>
            </td>
          <td>
            Yixin Zhu, Corey T. Watson, Yana Safonova, Matt Pennell, Anton Bankevich
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3e129666ae56c033aa207d599e42711f40eae8" target='_blank'>
              Gene body DNA hydroxymethylation restricts the magnitude of transcriptional changes during aging
              </a>
            </td>
          <td>
            James R. Occean, Na Yang, Yan Sun, Marshall S. Dawkins, R. Munk, Cedric Belair, Showkat A. Dar, C. Anerillas, Lin Wang, Changyou Shi, Christopher Dunn, Michel Bernier, Nathan L. Price, Julie S Kim, Chang-Yi Cui, Jinshui Fan, Moitrayee Bhattacharyya, Supriyo De, Manolis Maragkakis, R. de Cabo, Simone Sidoli, P. Sen
          </td>
          <td>2024-07-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Chromatin states, fundamental to gene regulation and cellular identity, are defined by a unique combination of histone post-translational modifications. Despite their importance, comprehensive patterns within chromatin state sequences, which could provide insights into key biological functions, remain largely unexplored. In this study, we introduce ChromBERT, a BERT-based model specifically designed to detect distinct patterns of chromatin state annotation data sequences. Notably, ChromBERT was pre-trained on promoter regions across a diverse range of epigenomes and subsequently fine-tuned using a dataset from multiple cell lines where RNA-seq data were available, highlighting the model’s ability to discern conserved chromatin state patterns within these regions. In addition to its predictive powers across tasks, evidenced by high AUC scores, ChromBERT provides further analysis through the incorporation of motif clustering using Dynamic Time Warping (DTW). This method enhances the model’s ability to dissect chromatin state sequence motifs, typically involving transcription and enhancer sites. The introduction of motif clustering with DTW into ChromBERT’s workflow is poised to facilitate the discovery of genomic regions linked to novel biological functions, deepening our understanding of chromatin state dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758a0c4bd29ca771e1573835ff87961f8849bdb7" target='_blank'>
              ChromBERT: Uncovering Chromatin State Motifs in the Human Genome Using a BERT-based Approach
              </a>
            </td>
          <td>
            Seohyun Lee, Che Lin, Chien-Yu Chen, Ryuichiro Nakato
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The fidelity of genetic information is essential for cellular function and viability. DNA double-strand breaks (DSBs) pose a significant threat to genome integrity, necessitating efficient repair mechanisms. While the predominant repair strategies are usually accurate, paradoxically, error-prone pathways also exist. This review explores recent advances and our understanding of microhomology-mediated end joining (MMEJ), an intrinsically mutagenic DSB repair pathway conserved across organisms. Central to MMEJ is the activity of DNA polymerase theta (Polθ), a specialized polymerase that fuels MMEJ mutagenicity. We examine the molecular intricacies underlying MMEJ activity and discuss its function during mitosis, where the activity of Polθ emerges as a last-ditch effort to resolve persistent DSBs, especially when homologous recombination is compromised. We explore the promising therapeutic applications of targeting Polθ in cancer treatment and genome editing. Lastly, we discuss the evolutionary consequences of MMEJ, highlighting its delicate balance between protecting genome integrity and driving genomic diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9839d026eb499a9c1802545a526640975b43a205" target='_blank'>
              Microhomology-Mediated End Joining Chronicles: Tracing the Evolutionary Footprints of Genome Protection.
              </a>
            </td>
          <td>
            Agnel Sfeir, M. Tijsterman, M. McVey
          </td>
          <td>2024-06-10</td>
          <td>Annual review of cell and developmental biology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Single-stranded DNA (ssDNA) is essential for various DNA-templated processes in both eukaryotes and prokaryotes. However, comprehensive characterizations of ssDNA still lag in plants compared to non-plant systems. Here, we conducted in situ S1-seq (ISS1-seq), with starting gDNA ranging from 5 µg to 250 ng, followed by comprehensive characterizations of ssDNA in rice (Oryza sativa L.). We found that ssDNA loci were substantially associated with a subset of non-B DNA structures and functional genomic loci. Subtypes of ssDNA loci had distinct epigenetic features. Importantly, ssDNA may act alone or partly coordinate with non-B DNA structures, functional genomic loci, or epigenetic marks to actively or repressively modulate gene transcription, which is genomic-region-dependent and associated with the distinct accumulation of RNA Pol II. Moreover, distinct types of ssDNA had differential impacts on the activities and evolution of TEs (especially common or conserved TEs) in the rice genome. Our study showcases an antibody-independent technique for characterizing non-B DNA structures or functional genomic loci in plants. It lays the groundwork and fills a crucial gap for further exploration of ssDNA, non-B DNA structures, or functional genomic loci, thereby advancing our understanding of their biology in plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a41f2ac1057a9cc0aff5ab06cd10c7413719a7" target='_blank'>
              Genome-wide characterization of single-stranded DNA in rice.
              </a>
            </td>
          <td>
            Yulian Peng, Pengtao Zhao, Zhaoguo Li, Ning Mu, Shentong Tao, Yilong Feng, Xuejiao Cheng, Wenli Zhang
          </td>
          <td>2024-06-25</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="


 What’s the pattern of genome-wide chromatin occupancy of SCP3 during meiotic prophase, and how its interaction with chromatin is regulated?



 We document SCP3 enriched at active chromatin regions with largely inherited from leptotene to pachytene stage, which facilitated by transcription and condensate assembly in spermatocytes.



 Meiosis is a critical step during the process of gametogenesis to form haploid cells which finally develop into functional gametes. The synaptonemal complex (SC) is a meiosis-specific protein assembly which bridges homologous chromosomes to establish the architecture of meiotic chromosome. Genetic ablation of genes encoding SC components impedes meiosis and gametogenesis in mice and mutation of these genes is frequently found in infertile patients. SCP3 localization changes with chromosome morphology in autosomes and is involved in maintaining the association of sex chromosomes. Male mice with null mutation of SCP3 failed to form SCs, resulting in developmental arrest during meiotic prophase.



 This study included male mice at post-natal day 10 (P10), 4 weeks or adult (8 weeks or older) were maintained on C57BL/6J background. Spermatocytes were performed modified CUT&Tag using antibodies. ChIP-seq data was analyzed together with WGBS and ATAC-seq data.



 Spermatocytes were obtained by a discontinuous BSA density gradient. Cell pellet was resuspended in culture medium (MEM with 10% FBS) at a concentration of 105 cells/ml and cultured at 37 °C in a 5% CO2 incubator. Spermatocytes were cultured with or without 1,6-hexanediol or α-amanitin. Spermatocytes with or without drug treatment were collected at indicated time points for further examination and then modified CUT&Tag and co-IP using anti-SCP3 and anti-POL II antibodies.



 Chromatin occupancy of SCP3 in leptotene and pachytene spermatocytes, SCP3-Core sites retained upon treatment of 1,6-hexanediol which dissolves SCP3 condensate, and SCP1 occupancy at meiotic chromatin in pachytene spermatocytes. When we determined inter-peak distances and width of peaks, we observe that: (1) SCP3 occupancy in leptotene stage had greater inter-peak distances and narrower peak width than SCP3 occupancy in pachytene stage, demonstrating gradual accumulated SCP3 at chromatin during meiotic progression; (2) SCP3-Core sites have broader peak width than SCP3-NonCore sites, in agreement with stability of SCP3 occupancy at SCP3-Core sites; (3) SCP1 occupancy had greater inter-peak distances and narrower peak width than SCP3 occupancy in pachytene stage, supporting that SCP1 peaks are a subset of SCP3 peaks and SCP1’s major function in connecting LE and CE respectively. Generally, at pachytene-stage meiosis, SCP3 mainly occupies chromatin regions including transcriptionally active sites, Deu and Alu elements, while SCP1 mainly co-localizes with SCP3 at a subset of regions with open chromatin and high cohesin enrichment to connect meiotic chromosome axes with CE. In this way, a local active chromatin environment is created at meiotic chromosome axes, facilitating homologous chromosome recombination.



 Our test was not successful to examine SCP3 occupancy with as few as less than 1000 spermatocytes using our protocol. It is improvement of efficiency of tagmentation or development of more efficient antibodies against SCP3 will be helpful for clinical sample detection.



 SCP3 occupancy was largely inherited from leptotene to pachytene stage, facilitated by transcription and condensate assembly, and enriched at specific SINE repeats. Further exploration using knockout mouse models of meiotic axial proteins and epigenetic modifiers will provide valuable information on meiotic chromatin architecture and crosstalk among meiotic chromatin events.



 N/A
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd5370d474a768836582b1cb861c8c3d70c98bd" target='_blank'>
              P-045 Deciphering meiotic chromatin organization by SCP3
              </a>
            </td>
          <td>
            S. Guo
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Bacterial chromosomes are large molecules that need to be highly compacted to fit inside the cells. Chromosome compaction must facilitate and maintain key biological processes such as gene expression and DNA transactions (replication, recombination, repair, and segregation). Chromosome and chromatin 3D-organization in bacteria has been a puzzle for decades. Chromosome conformation capture coupled to deep sequencing (Hi-C) in combination with other "omics" approaches has allowed dissection of the structural layers that shape bacterial chromosome organization, from DNA topology to global chromosome architecture. Here we review the latest findings using Hi-C and discuss the main features of bacterial genome folding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d47cf5eae543114153f80223459365d3c29d31d5" target='_blank'>
              Principles of bacterial genome organization, a conformational point of view.
              </a>
            </td>
          <td>
            Sokrich Ponndara, Mounia Kortebi, F. Boccard, S. Bury-Moné, Virginia S. Lioy
          </td>
          <td>2024-06-22</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancer is a complex and heterogeneous disease, in which a number of genetic and epigenetic changes occur in tumor onset and progression. Recent studies indicate that changes at the RNA level are also involved in tumorigenesis, such as adenosine-to-inosine (A-to-I) RNA editing. Here, we systematically investigate transcriptome-wide A-to-I editing events in a large number of samples from Non-Hodgkin lymphomas (NHLs). Using a computational pipeline that determines significant differences in editing level between NHL and normal samples at known A-to-I editing sites, we identify a number of differentially edited editing sites between NHL subtypes and normal samples. Most of the differentially edited sites are located in non-coding regions, and many such sites show a strong correlation between gene expression level and editing efficiency, indicating that RNA editing might have direct consequences for the cancer cell’s aberrant gene regulation status in these cases. Moreover, we establish a strong link between RNA editing and NHL by demonstrating that NHL and normal samples and even NHL subtypes can be distinguished based on genome-wide RNA editing profiles alone. Our study establishes a strong link between RNA editing, cancer and aberrant gene regulation in NHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3709fa38f5f1119b1fcb4db3f2e4cd77244ecc08" target='_blank'>
              A-to-I Editing Is Subtype-Specific in Non-Hodgkin Lymphomas
              </a>
            </td>
          <td>
            Cai Chen, Ralf Bundschuh
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ABSTRACT This research investigates the intricate dynamics of DNA methylation in the hours following CD8+ T cell activation, during a critical yet understudied temporal window. DNA methylation is an epigenetic modification central to regulation of gene expression and directing immune responses. Our investigation spanned 96-h post-activation and unveils a nuanced tapestry of global and site-specific methylation changes. We identified 15,626 significant differentially methylated CpGs spread across the genome, with the most significant changes occurring within the genes ADAM10, ICA1, and LAPTM5. While many changes had modest effect sizes, approximately 120 CpGs exhibited a log2FC above 1.5, with cell activation and proliferation pathways the most affected. Relatively few of the differentially methylated CpGs occurred along adjacent gene regions. The exceptions were seven differentially methylated gene regions, with the Human T cell Receptor Alpha Joining Genes demonstrating consistent methylation change over a 3kb window. We also investigated whether an inflammatory environment could alter DNA methylation during activation, with proliferating cells exposed to the oxidant glycine chloramine. No substantial differential methylation was observed in this context. The temporal perspective of early activation adds depth to the evolving field of epigenetic immunology, offering insights with implications for therapeutic innovation and expanding our understanding of epigenetic modulation in immune function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf69efc7eae76417393a64272f476d27bebaa6b9" target='_blank'>
              Widespread genomic de novo DNA methylation occurs following CD8+ T cell activation and proliferation
              </a>
            </td>
          <td>
            Annika R. Seddon, Olivia M Damiano, Mark B. Hampton, Aaron J. Stevens
          </td>
          <td>2024-06-20</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Maintenance of genome stability is of paramount importance for the survival of an organism. However, genomic integrity is constantly being challenged by various endogenous and exogenous processes that damage DNA. Therefore, cells are heavily reliant on DNA repair pathways that have evolved to deal with every type of genotoxic insult that threatens to compromise genome stability. Notably, inherited mutations in genes encoding proteins involved in these protective pathways trigger the onset of disease that is driven by chromosome instability e.g. neurodevelopmental abnormalities, neurodegeneration, premature ageing, immunodeficiency and cancer development. The ability of cells to regulate the recruitment of specific DNA repair proteins to sites of DNA damage is extremely complex but is primarily mediated by protein post-translational modifications (PTMs). Ubiquitylation is one such PTM, which controls genome stability by regulating protein localisation, protein turnover, protein-protein interactions and intra-cellular signalling. Over the past two decades, numerous ubiquitin (Ub) E3 ligases have been identified to play a crucial role not only in the initiation of DNA replication and DNA damage repair but also in the efficient termination of these processes. In this review, we discuss our current understanding of how different Ub E3 ligases (RNF168, TRAIP, HUWE1, TRIP12, FANCL, BRCA1, RFWD3) function to regulate DNA repair and replication and the pathological consequences arising from inheriting deleterious mutations that compromise the Ub-dependent DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d67379a570ccc6355d856b2e241bb15b1ce2e1c" target='_blank'>
              E3 ligases: a ubiquitous link between DNA repair, DNA replication and human disease.
              </a>
            </td>
          <td>
            Anoop S. Chauhan, Satpal S. Jhujh, Grant S Stewart
          </td>
          <td>2024-07-10</td>
          <td>The Biochemical journal</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca001844a365e7cf93a56cb525afd21d3d9c39ca" target='_blank'>
              Papillary renal neoplasm with reverse polarity has low frequency of alterations in chromosomes 7, 17, and Y.
              </a>
            </td>
          <td>
            Daisuke Kiyozawa, Takeshi Iwasaki, Dai Takamatsu, Kenichi Kohashi, Takumi Miyamoto, Genshiro Fukuchi, Masatoshi Eto, Michifumi Yamashita, Yoshinao Oda
          </td>
          <td>2024-06-14</td>
          <td>Virchows Archiv : an international journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="All bacteria encode a multifunctional DNA-binding protein, DnaA, which initiates chromosomal replication. Despite having the most complex, segmented bacterial genome, little is known about Borrelia burgdorferi DnaA and its role in maintaining the spirochete’s physiology. We utilized inducible CRISPR-interference and overexpression to modulate cellular levels of DnaA to better understand this essential protein. Dysregulation of DnaA, either up or down, significantly slowed replication and increased or decreased cell lengths. Using fluorescent microscopy, we found the DnaA CRISPRi mutants had increased numbers of chromosomes with irregular spacing patterns. The DnaA-depleted spirochetes also exhibited a significant defect in helical morphology. RNA-seq of the conditional mutants showed significant changes in the levels of transcripts involved with flagellar synthesis, elongation, cell division, virulence, and other functions. These findings demonstrate that the DnaA plays a commanding role in maintaining borrelial growth dynamics and protein expression, which are essential for the survival of the Lyme disease spirochete.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef9fa4e6497e13499446134decf2617e58008ee" target='_blank'>
              DnaA modulates gene expression and morphologyof the Lyme Disease spirochete
              </a>
            </td>
          <td>
            A. Krusenstjerna, Nerina Jusufovic, Timothy C. Saylor, Brian Stevenson
          </td>
          <td>2024-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Simple Summary From detection to monitoring therapeutic response, circulating tumor DNA has emerged at the forefront of numerous clinical applications in oncology. It provides a non-invasive, real-time window into the evolving tumor as the cancer progresses. Advances in liquid biopsies and circulating tumor DNA analysis are enhancing their diagnostic applicability with tools like liposomal nanoparticles and antibody priming agents by binding to cell-free DNA in circulation and aiding in evading clearance mechanisms. The development of highly sensitive techniques such as digital droplet PCR and next-generation sequencing-based technologies has enabled the detection of minute ctDNA fractions amid high background, increasing its potential for early detection and minimal residual disease. Abstract Circulating tumor DNA (ctDNA), a fragment of tumor DNA found in the bloodstream, has emerged as a revolutionary tool in cancer management. This review delves into the biology of ctDNA, examining release mechanisms, including necrosis, apoptosis, and active secretion, all of which offer information about the state and nature of the tumor. Comprehensive DNA profiling has been enabled by methods such as whole genome sequencing and methylation analysis. The low abundance of the ctDNA fraction makes alternative techniques, such as digital PCR and targeted next-generation exome sequencing, more valuable and accurate for mutation profiling and detection. There are numerous clinical applications for ctDNA analysis, including non-invasive liquid biopsies for minimal residual disease monitoring to detect cancer recurrence, personalized medicine by mutation profiling for targeted therapy identification, early cancer detection, and real-time evaluation of therapeutic response. Integrating ctDNA analysis into routine clinical practice creates promising avenues for successful and personalized cancer care, from diagnosis to treatment and follow-up.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6061f699175a934ecd4bb96549dc8c649df9865d" target='_blank'>
              Decoding the Dynamics of Circulating Tumor DNA in Liquid Biopsies
              </a>
            </td>
          <td>
            Khadija Turabi, Kelsey Klute, P. Radhakrishnan
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc05d0e416d1c5836cac6aabe3bd3a577d1440ba" target='_blank'>
              Enhanced eMAGE applied to identify genetic factors of nuclear hormone receptor dysfunction via combinatorial gene editing
              </a>
            </td>
          <td>
            Peter N Ciaccia, Zhuobin Liang, Anabel Y. Schweitzer, Eli Metzner, Farren J. Isaacs
          </td>
          <td>2024-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Dysregulation of the tumour suppressor gene Adenomatous Polyposis Coli (APC) is a canonical step in colorectal cancer development. Curiously, most colorectal tumours carry biallelic mutations that result in only partial loss of APC function, suggesting that a ‘just-right’ level of APC inactivation, and hence Wnt signalling, provides the optimal conditions for tumorigenesis. Mutational processes act variably across the APC gene, which could contribute to the bias against complete APC inactivation. Thus the selective consequences of partial APC loss are unclear. Here we propose a mathematical model to quantify the tumorigenic effect of biallelic APC genotypes, controlling for somatic mutational processes. Analysing sequence data from >2500 colorectal cancers, we find that APC genotypes resulting in partial protein function confer about 50 times higher probability of progressing to cancer compared to complete APC inactivation. The optimal inactivation level varies with anatomical location and additional mutations of Wnt pathway regulators. We use this context dependency to assess the regulatory effect of secondary Wnt drivers in combination with APC in vivo, and provide evidence that mutant AMER1 combines with APC genotypes that lead to relatively low Wnt. The fitness landscape of APC inactivation is consistent across microsatellite unstable and POLE-deficient colorectal cancers and tumours in patients with Familial Adenomatous Polyposis suggesting a general ‘just-right’ optimum, and pointing to Wnt hyperactivation as a potential cancer vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5f474ac3dba3c99ab4d1fd2c61a492c95822b9" target='_blank'>
              Quantifying ‘just-right’ APC inactivation for colorectal cancer initiation
              </a>
            </td>
          <td>
            Meritxell Brunet Guasch, Nathalie Feeley, Ignacio Soriano, Steve Thorn, Ian Tomlinson, Michael D. Nicholson, Tibor Antal
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Precise DNA replication is fundamental for genetic inheritance. In eukaryotes, replication initiates at multiple origins that are first "licensed" and subsequently "fired" to activate DNA synthesis. Despite the success in identifying origins with specific DNA motifs in Saccharomyces cerevisiae, no consensus sequence or sequences with a predictive value of replication origins have been recognized in metazoan genomes. Rather, epigenetic rules and chromatin structures are believed to play important roles in governing the selection and activation of replication origins. We propose that replication initiation is facilitated by a group of sequence-specific "replication pioneer factors," which function to increase chromatin accessibility and foster a chromatin environment that is conducive to the loading of the prereplication complex. Dysregulation of the function of these factors may lead to gene duplication, genomic instability, and ultimately the occurrence of pathological conditions such as cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43129f2ef7735b3992e0d45fec31beb44fb71b6c" target='_blank'>
              Pioneer factors for DNA replication initiation in metazoans.
              </a>
            </td>
          <td>
            Yue Wang, Jing Liang
          </td>
          <td>2024-06-16</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC.Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs.Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes.These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Paralogs are prevalent in the human genome and are considered a rich source of synthetic lethality due to functional redundancy. For a cancer cell carrying a gene with Loss-Of-Function (LOF) mutation, inactivation of its paralog using gene editing can induce a selective decrease in viability, leaving normal cells that do not harbor the mutation unharmed. Previous studies have exploited this vulnerability of cancer genomes. However, the cancer-specificity of such synthetic lethality limits its application across different cancer types. Furthermore, the role of functional redundancy which can explain the cancer specificity has remained largely unexplored. In this study, we computationally predicted synthetic lethal paralogs along with mechanistically important functional redundancies between them in a cancer-specific manner. We applied our prediction method to publicly available data for an aggressive subtype of breast cancer called triple-negative breast cancer (TNBC), which lacks the expression key biomarkers and hence has the worst prognosis among other breast cancer subtypes. TNBC is characterized by the highest mutational load and the largest fraction of genome altered among the breast cancer subtypes providing a rich mutational landscape to identify LOF genes. Using the CRISPR inactivation screen data for cancer cell lines obtained from the Cancer Dependency Map (DepMap) project, we predicted sets of synthetic lethal paralogs that show a significantly greater viability decrease if a gene carries LOF and its paralog is inactivated, compared to the viability decrease due to the inactivation of only one of the paralogs. Consistent with previous findings of context-dependent synthetic lethality, we found that a relatively small fraction of TNBC-specific synthetic lethal paralogs overlapped with those found across other cancer types. To uncover the mechanistically important functional redundancies between paralogs, we analyzed the genomics and transcriptomics data from multiple sources: TNBC panel of primary tumors and patient-derived xenografts, Pan-Cancer Analysis of Whole Genomes (PCAWG), and the Cancer Cell Line Encyclopedia (CCLE). The functional redundancies varied across cancer types based on (1) mutual exclusivity of LOFs, (2) backup compensation of deleterious mutations, (3) backup upregulation, and (4) dosage balance. Overall, our findings show a strong context-dependency of synthetic lethal paralogs and estimates of functional redundancy, emphasizing the importance of such cancer-specific predictions in identifying targetable paralog synthetic lethalities. Collectively, the computational method and sets of targetable paralogs are a unique resource for developing precision oncology therapeutic strategies against TNBC and other cancers more broadly.
 Citation Format: Rohan Dandage, Elena Kuzmin. Predicting targetable paralog synthetic lethalities and functional redundancies in cancer genomes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d08601862d27b41b39a782547d739914c1bd271" target='_blank'>
              Abstract B012: Predicting targetable paralog synthetic lethalities and functional redundancies in cancer genomes
              </a>
            </td>
          <td>
            Rohan Dandage, Elena Kuzmin
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Motivation Techniques such as high-throughput chromosome conformation capture (Hi-C) have provided a wealth of information on the organization of the nucleus and the genome important for understanding gene expression regulation. Additionally, Genome-Wide Association Studies (GWASs) have uncovered thousands of loci related to complex traits. Expression quantitative trait loci (eQTL) studies have further linked the genetic variants to alteration in expression levels of associated target genes across individuals. However, the functional roles of many eQTLs located in non-coding regions are unclear. Current joint analyses of Hi-C and eQTLs data lack advanced computational tools, limiting what can be learned from these data. Result In this work, we developed a computational method for simultaneous analysis of Hi-C and eQTL data. Our method can identify a small set of non-random interactions from all Hi-C interactions. Using these non-random interactions, we reconstruct large ensemble (×105) of high-resolution single-cell 3D chromatin conformations with thorough sampling, which accurately replicate Hi-C measurements. Our results revealed the presence of many-body interactions in chromatin conformation at single-cell level in eQTL locus, offering detailed view into how three-dimensional structures of chromatin form the physical foundation for gene regulation, including how genetic variants of eQTLs affect the expression level of their associated eGenes. Furthermore, our method can deconvolve chromatin heterogeneity and investigate the spatial associations of eQTLs and eGenes at subpopulation level to reveal their regulatory impacts on gene expression. Together, ensemble modeling of thoroughly sampled single cell chromatin conformations from Hi-C, along with eQTL data, helps to decipher how chromatin 3D structures provide the physical basis for gene regulation, expression control, and aid in understanding of the overall structure-function relationships of genome organization. Availability and implementation: It is available at https://github.com/uic-liang-lab/3DChromFolding-eQTL-Loci">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9720002233785e5b33a3a1f9b1f74f286c10723" target='_blank'>
              Structural Basis of Differential Gene Expression at eQTLs Loci from High-Resolution Ensemble Models of 3D Single-Cell Chromatin Conformations
              </a>
            </td>
          <td>
            Lin Du, Hammad Farooq, Pourya Delafrouz, Jie Liang
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Tumor protein p53 (TP53) mediates DNA repair and cell proliferation in growing cells. The TP53 gene is a tumor suppressor that regulates the expression of target genes in response to multiple cellular stress factors. Key target genes are involved in crucial cellular events such as DNA repair, cell cycle regulation, apoptosis, metabolism, and senescence. TP53 genetic variants and the activity of the wild-type p53 protein (WT-p53) have been linked to a wide range of tumorigenesis. Various genetic and epigenetic alterations, including germline and somatic mutations, loss of heterozygosity, and DNA methylation, can alter TP53 activity, potentially resulting in cancer initiation and progression. This study was designed to screen three reported mutations in the DNA-binding domain of the p53 protein in breast cancer, to evaluate the relative susceptibility and risk associated with breast cancer in the local population. Genomic DNA was isolated from 30 breast tumor tissues along with controls. Tetra and Tri ARMS PCR were performed to detect mutations in the TP53 coding region. For SNPs c.637C>T and c.733C>T, all analyzed cases were homozygous for the wild-type allele ‘C,’ while for SNP c.745A>G, all cases were homozygous for the wild-type allele ‘A.’ These results indicate no relevance of these three SNPs to cancer progression in our study cohort. Additionally, the findings from whole exon sequencing will help to predict more precise outcomes and assess the importance of TP53 gene mutations in breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bffe267deb645fc61554e684edc8c0bafc2c11ee" target='_blank'>
              Analysis of Genetic Alterations in TP53 Gene in Breast Cancer – A Secondary Publication
              </a>
            </td>
          <td>
            Baqaur Rehman, Muhammad Abubakar, Muhammad Naeem Kiani, Rooma Ayyoub
          </td>
          <td>2024-06-19</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="There remains a large need for a greater understanding of the metastatic process within the prostate cancer field. Our research aims to understand the adaptive – ergo potentially metastatic – responses of cancer to changing microenvironments. Emerging evidence has implicated a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study of prostate cancer patients revealed that PACC presence in the prostate at the time of radical prostatectomy was predictive of future metastatic progression. To test for a causative relationship between PACC state biology and metastasis, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) in subcutaneous, caudal artery, and intracardiac mouse models of metastasis. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. In vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites following dormancy. Additional direct and indirect mechanistic in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence that PACCs are mechanistically linked to metastasis. Statement of Significance We provide the first evidence that cells in the polyaneuploid cancer cell state contribute to increased metastatic competency in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2da05a6c448f502761bb367a94125d4543338" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T.A. Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, SR Amend
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The use of k-mers to capture genetic variation in bacterial genome-wide association studies (bGWAS) has demonstrated its effectiveness in overcoming the plasticity of bacterial genomes by providing a comprehensive array of genetic variants in a genome set that is not confined to a single reference genome. However, little attempt has been made to interpret k-mers in the context of genome rearrangements, partly due to challenges in the exhaustive and high-throughput identification of genome structure and individual rearrangement events. Here, we present GWarrange, a pre- and post-bGWAS processing methodology that leverages the unique properties of k-mers to facilitate bGWAS for genome rearrangements. Repeat sequences are common instigators of genome rearrangements through intragenomic homologous recombination, and they are commonly found at rearrangement boundaries. Using whole-genome sequences, repeat sequences are replaced by short placeholder sequences, allowing the regions flanking repeats to be incorporated into relatively short k-mers. Then, locations of flanking regions in significant k-mers are mapped back to complete genome sequences to visualise genome rearrangements. Four case studies based on two bacterial species (Bordetella pertussis and Enterococcus faecium) and a simulated genome set are presented to demonstrate the ability to identify phenotype-associated rearrangements. GWarrange is available at https://github.com/DorothyTamYiLing/GWarrange.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42162bc2b2f883e8a6834a9dc952dc10d3392ba" target='_blank'>
              GWarrange: a pre- and post- genome-wide association studies pipeline for detecting phenotype-associated genome rearrangement events.
              </a>
            </td>
          <td>
            Yi Ling Tam, Sarah Cameron, Andrew Preston, Lauren A. Cowley
          </td>
          <td>2024-07-01</td>
          <td>Microbial genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35393ed7857060735a43bccb7ec1795bc1e251a9" target='_blank'>
              Comprehensive molecular characterization of TFE3-rearranged renal cell carcinoma.
              </a>
            </td>
          <td>
            Cho-Rong Lee, J. Suh, Dongjun Jang, Bo-Yeong Jin, Jaeso Cho, Moses Lee, Hyungtai Sim, Minyong Kang, Jueun Lee, Ju Hyun Park, Kyoung-Hwa Lee, Geum-Sook Hwang, Kyung Chul Moon, Cheryn Song, J. Ku, C. Kwak, Hyeon-Hoe Kim, , Murim Choi, Chang Wook Jeong
          </td>
          <td>2024-08-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Meiotic errors of relatively small chromosomes in oocytes result in egg aneuploidies that cause miscarriages and congenital diseases. Unlike somatic cells, which preferentially mis-segregate larger chromosomes, aged oocytes preferentially mis-segregate smaller chromosomes through unclear processes. Here, we provide a comprehensive three-dimensional chromosome identifying-and-tracking dataset throughout meiosis I in live mouse oocytes. This analysis reveals a prometaphase pathway that actively moves smaller chromosomes to the inner region of the metaphase plate. In the inner region, chromosomes are pulled by stronger bipolar microtubule forces, which facilitates premature chromosome separation, a major cause of segregation errors in aged oocytes. This study reveals a spatial pathway that facilitates aneuploidy of small chromosomes preferentially in aged eggs and implicates the role of the M phase in creating a chromosome size–based spatial arrangement. Editor’s summary Chromosome segregation errors in oocytes lead to aneuploidy in eggs, a major cause of miscarriage and congenital disease. Errors are more likely to occur with age, especially with smaller chromosomes, but the reasons for this pattern are poorly understood. Takenouchi et al. established a method to identify and track targeted chromosomes throughout meiosis in live mouse oocytes. This method revealed that smaller chromosomes actively move toward the inner region of the spindle equator, creating a chromosome size–based spatial arrangement on the metaphase plate. Inner positioning of smaller chromosomes increases the risk of their premature separation, a hallmark of aging-associated errors in oocytes. —Di Jiang INTRODUCTION Chromosome segregation errors during meiosis in oocytes lead to aneuploidy in eggs. The frequency of errors increases with maternal age. Unlike somatic cells, in which larger chromosomes are more likely to be mis-segregated, aged oocytes exhibit segregation errors preferentially in smaller chromosomes. Egg aneuploidy in a small chromosome is a major cause of miscarriage and congenital diseases such as Down syndrome. How smaller chromosomes are preferentially mis-segregated in aged oocytes is poorly understood. RATIONALE To study individual chromosome dynamics, we developed a comprehensive set of fluorescent probes that specifically label each of the 20 mouse chromosomes (19 autosomes and the X chromosome). Using these probes, we established a live imaging pipeline for complete three-dimensional (3D) tracking of identified chromosomes throughout meiosis in mouse oocytes. This “chromosome identifying-and-tracking” pipeline allowed us to acquire 217 complete 3D trajectories of nine different identified chromosomes and their kinetochores from 93 oocytes. Using this comprehensive dataset, we aimed to quantitatively analyze chromosome size–based spatiotemporal dynamics and test whether they contribute to preferential errors of smaller chromosomes. RESULTS Quantitative analysis revealed that smaller chromosomes prefer to move toward the inner region of the spindle equator during prometaphase, although each chromosome exhibits substantial variability in its dynamics among oocytes. This preference of smaller chromosomes leads to the establishment of a chromosome size–based spatial arrangement, in which smaller chromosomes are preferentially positioned in the inner region of the metaphase plate, in both mouse and human oocytes. Oocytes can establish the chromosome size–based spatial arrangement without the need for a preexisting chromosome positioning bias, demonstrating the existence of a prometaphase pathway that actively creates the spatial arrangement. We found that smaller chromosomes tend to stretch earlier and that stretching is coupled with inward movement more tightly in smaller chromosomes, suggesting that the prometaphase pathway involves earlier bipolar microtubule attachment on smaller chromosomes. The prometaphase pathway is active independent of age. Chromosome tracking in aged mouse oocytes showed that premature chromosome separation—which is due to aging-associated reduction of chromosome cohesion and a major cause of segregation errors—occurs predominantly in the inner region of the metaphase plate. Premature chromosome separation depends on bipolar microtubule pulling forces, which we found to be significantly stronger in the inner region of the metaphase plate, as indicated by increased kinetochore-centromere distances. Preventing the inner positioning of smaller chromosomes with artificial beads significantly suppresses premature chromosome separation in aged oocytes during prolonged metaphase, demonstrating that preferential inner positioning of smaller chromosomes is a risk of segregation errors in aged oocytes. CONCLUSION This study shows that mammalian oocytes possess the prometaphase pathway that preferentially positions smaller chromosomes in the inner region of the metaphase plate, which puts them at risk of segregation errors when chromosome cohesion is weakened by aging. The chromosome size–based 2D radial arrangement created by the prometaphase pathway is similar to the 3D chromosome territory arrangement observed in the interphase nucleus of somatic cells, providing insights into the role of M phase in establishing nuclear-chromosome arrangement. Spatial pathway to meiotic errors. Smaller chromosomes preferentially move toward the inner region of the metaphase plate, creating a chromosome size–based spatial arrangement. In the inner region, microtubule bipolar forces are stronger, facilitating premature chromosome separation, a major cause of aging-associated egg aneuploidy. Pictured are chromosome 18 (the second smallest chromosome, magenta), chromosome 1 (the largest chromosome, blue), and other chromosomes (gray). Scale bar, 5 μm.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9391fa2d43f43ceab0e4b48eea35d62bef4558fa" target='_blank'>
              Live chromosome identifying and tracking reveals size-based spatial pathway of meiotic errors in oocytes
              </a>
            </td>
          <td>
            Osamu Takenouchi, Yogo Sakakibara, Tomoya S. Kitajima
          </td>
          <td>2024-07-19</td>
          <td>Science</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a widely used method for classifying cell types and states and revealing disease mechanisms. However, most contemporary scRNA-seq platforms fail to explore the multi-landscape of RNA. Here, we designed a microfluidic chip combined oligo-dT primers and Random Bridging Co-labeling (RBCL) RNA sequencing to develop an innovative Chigene scRNA-seq technology that can identify gene expression, mutations, and RNA splicing landscapes at the single-cell level. The Chigene scRNA-seq platform demonstrated exceptional performance, with minimal doublet rates of 0.94% (Chigene V1) and 1.93% (Chigene V2). Both versions exhibit high sensitivity, with Chigene V2 achieving nearly 100% RNA coverage and detecting over 1800 genes per cell on average. Targeted capture of single-cell gene mutations enhances mutation detection sensitivity. Moreover, this Chigene V2 platform has been validated in clinical samples for its ability to detect mutations, gene fusions and alternative splicing. The reliability of the platform was further corroborated using known functional gene mutation (CDKN1A) and fusion (FGFR3-TACC). To validate this method’s potential for discovering novel gene mutations in clinical samples, our investigation revealed an intriguing cell subpopulation carrying an ARHGAP5 mutation in urothelial carcinoma. These cells exhibited high-frequency mRNA splicing and exhibited specific crosstalk with T cells, distinguishing them from the subpopulation with the ARHGAP5 wild-type phenotype. Overall, this method provides a robust scRNA-seq platform suitable for comprehensive analyses of clinical specimens at different genetic information levels, thereby offering significant potential in the discovery of novel genes and interactions at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be600ce14b879188e7c67b73a02d6fde03c11a8c" target='_blank'>
              Identification of Multi-landscape and Cell Interactions in the Tumor Microenvironment through High-Coverage Single-Cell Sequencing
              </a>
            </td>
          <td>
            Wenlong Zhong, Ligang Wang, Tonglei Guo, Lihua Zhao, Daqin Wu, Fei Xie, Xiao Wang, Xiuxin Li, FangXiao Wang, Weiyue Gu, Tianxin Lin, Xu Chen
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer (mCRPC) is a lethal disease that resists therapy targeting androgen signaling, the primary driver of prostate cancer. mCRPC resists androgen receptor (AR) inhibitors by amplifying AR signaling or by evolving into therapy-resistant subtypes that do not depend on AR. Elucidation of the epigenetic underpinnings of these subtypes could provide important insights into the drivers of therapy resistance. In this study, we produced chromatin accessibility maps linked to the binding of lineage-specific transcription factors (TF) by performing ATAC sequencing on 70 mCRPC tissue biopsies integrated with transcriptome and whole genome sequencing. mCRPC had a distinct global chromatin accessibility profile linked to AR function. Analysis of TF occupancy across accessible chromatin revealed 203 TFs associated with mCRPC subtypes. Notably, ZNF263 was identified as a putative prostate cancer TF with a significant impact on gene activity in the double-negative (AR- neuroendocrine-) subtype, potentially activating MYC targets. Overall, this analysis of chromatin accessibility in mCRPC provides valuable insights into epigenetic changes that occur during progression to mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac71bf183535cdb84fd367ef22969e3b56650628" target='_blank'>
              An Atlas of Accessible Chromatin in Advanced Prostate Cancer Reveals the Epigenetic Evolution during Tumor Progression.
              </a>
            </td>
          <td>
            Raunak Shrestha, L. Chesner, Meng Zhang, Stanley Zhou, A. Foye, A. Lundberg, Alana S Weinstein, M. Sjöström, Xiaolin Zhu, Thaidy Moreno-Rodriguez, Haolong Li, Su C/Pcf West Coast Prostate Cancer Dream Team, J. Alumkal, Rahul Aggarwal, Eric J Small, Mathieu Lupien, David A Quigley, Felix Y Feng
          </td>
          <td>2024-07-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4fb1e6fb2869d1ffa14cf25654e46c58ec040e" target='_blank'>
              Beyond the code: the role of histone methylation in cancer resistance and therapy
              </a>
            </td>
          <td>
            Daniel Noerenberg, Frederik Damm
          </td>
          <td>2024-06-12</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Tumor genomes often harbor a complex spectrum of single nucleotide mutations, small indels, and large chromosome rearrangements that can perturb coding and non-coding regions of the genome in ways that remain poorly understood. The mutational processes responsible for the genesis of these events are also not static; instead, they continue to operate throughout disease evolution and further diversify the genetic and phenotypic landscape of cancer cells. Mounting evidence suggests that tumor genotype can be an important determinant of disease progression and therapy response, such as by modulating the sensitivity or resistance of cancer cells to mutant-specific drugs. These types of observations have motivated efforts to treat cancers with genome-informed therapies, highlighting the need to understand how different genetic variants precisely affect gene function and overall tumor phenotypes. Variant-function maps that provide a mechanistic understanding of the biology driven by cancer-associated mutations are needed to design these types of treatment strategies. Precision genome editing technologies like base and prime editing are uniquely suited to tackle this problem. Nevertheless, deploying these methods for systematic variant-function studies and disease modeling in vivo has not been straightforward due to lack of robust and scalable platforms capable of assessing editing efficiency and precision, particularly at endogenous loci. With this goal in mind, we previously developed and applied high-throughput base editing ‘sensor’ approaches that link endogenous genome editing outcomes with synthetic DNA-based readouts and cellular fitness measurements. Using these approaches, we showed that several uncharacterized mutant p53 alleles drive cancer cell proliferation and in vivo tumor development. Building upon this work, we recently developed new prime editing guide RNA design tools and sensor-based approaches that similarly couple quantitative editing outcomes to cellular fitness, allowing us to significantly expand the breadth of cancer-associated mutations that can be interrogated using these precision genome editing technologies. In this talk, I will describe ongoing work using base and prime editing sensor libraries to probe the biological impact of thousands of functionally-distinct genetic variants in diverse types of protein-coding genes and families to learn how these influence various cancer phenotypes. I will also discuss how the implementation of these modular technologies will allow researchers to functionally disentangle the impact of endogenous and exogenous mutational processes on functional selection and tumor evolution with close to single base pair resolution. This generalizable precision genome editing framework will facilitate the functional interrogation of genetic variants across diverse biological contexts, providing much needed insight into cancer variant-function relationships that could be leveraged to develop more precise cancer treatment paradigms, including synthetic lethal therapies that exploit tumor genotype.
 Citation Format: Francisco J. Sánchez-Rivera, Samuel I. Gould, Alexandra N. Wuest, Kexin Dong, Grace A. Johnson, Alvin Hsu, Varun K. Narendra, Ondine Atwa, Stuart S. Levine, David R. Liu. Functional studies of genetic variation using precision genome editing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA020.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31918f227a5efe4ffc9748fec8993c778a740c9" target='_blank'>
              Abstract IA020: Functional studies of genetic variation using precision genome editing
              </a>
            </td>
          <td>
            Francisco J. Sánchez-Rivera, Samuel I. Gould, Alexandra N. Wuest, Kexin Dong, Grace A. Johnson, Alvin Hsu, Varun K. Narendra, Ondine Atwa, Stuart S. Levine, David R. Liu
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Our understanding of cancer genomes has allowed for the integration of molecular data into histopathological classifications for routine stratification of patients. In the last 10–15 years, thanks to this systematic implementation of large-scale sequencing, the identification of hotspot somatic mutations in histone genes came into the limelight, underscoring the concept of oncohistones. As drivers in pediatric brain tumors, and in several other types of cancers, oncohistones brought a “new dimension of Strange” into the cancer multiverse, to paraphrase Marvel. An integrative approach to cancer complexity as a multidimensional projection is urgently needed to consider all relevant etiological, developmental, and evolutionary components. Here, we discuss recent progress on histone variants and chaperones, their regulation and alterations in cancers, the available in vivo models, and current treatment strategies. More specifically, we adopt a view through the lens of tissue-specific differences and means for genome expression and integrity maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2eba7c5952135066a16d3e19fbca0058628db86" target='_blank'>
              Histone H3 Variants in the Multiverse of Cancer
              </a>
            </td>
          <td>
            Iva Simeonova, Geneviève Almouzni
          </td>
          <td>2024-06-12</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="MYC has been implicated in the pathogenesis of a wide range of human tumors and has been described for many years as a transcription factor that regulates genes with pleiotropic functions to promote tumorigenic growth. However, despite extensive efforts to identify specific target genes of MYC that alone could be responsible for promoting tumorigenesis, the field is yet to reach a consensus whether this is the crucial function of MYC. Recent work shifts the view on MYC's function from being a gene-specific transcription factor to an essential stress resilience factor. In highly proliferating cells, MYC preserves cell integrity by promoting DNA repair at core promoters, protecting stalled replication forks, and/or preventing transcription-replication conflicts. Furthermore, an increasing body of evidence demonstrates that MYC not only promotes tumorigenesis by driving cell-autonomous growth, but also enables tumors to evade the host's immune system. In this review, we summarize our current understanding of how MYC impairs antitumor immunity and why this function is evolutionarily hard-wired to the biology of the MYC protein family. We show why the cell-autonomous and immune evasive functions of MYC are mutually dependent and discuss ways to target MYC proteins in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877f0298df11cd9063e84174a42a5eb6afca2c71" target='_blank'>
              Immune evasion: An imperative and consequence of MYC deregulation.
              </a>
            </td>
          <td>
            Bastian Krenz, Jongkuen Lee, Toshitha Kannan, Martin Eilers
          </td>
          <td>2024-07-02</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.
 Citation Format: Lixing Yang. Data-driven discoveries of molecular mechanism and therapeutic vulnerabilities of CDK12 mutant tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88abb1d2de21d0d5c6fc7861d116fbf694712a58" target='_blank'>
              Abstract B006: Data-driven discoveries of molecular mechanism and therapeutic vulnerabilities of CDK12 mutant tumors
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c5f226ca297006300a3de7681b5b8c136da7920" target='_blank'>
              The chromatin accessibility and transcriptomic landscape of the aging mice cochlea and the identification of potential functional super-enhancers in age-related hearing loss
              </a>
            </td>
          <td>
            Chanyuan Zhang, Ting Yang, Xiaoqin Luo, Xiaoqing Zhou, Menglong Feng, Wei Yuan
          </td>
          <td>2024-07-04</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="All cell lineages accumulate mutations over time, increasing the probability that some lineages eventually become malignant. Many of the processes responsible for generating mutations are leaving a characteristic footprint in the genome that allows their presence to be detected. However, the mutational pattern in a tumor is usually the combined result of multiple mutational processes being at work simultaneously, and the problem of disentangling the different footprints and their relative impact then becomes a deconvolution problem. Several algorithms have been developed for this purpose, most of them involving a factorization of the mutation count matrix into two nonnegative matrices, representing respectively the underlying mutational signatures and the relative weighting of (or exposure to) these signatures. Here, we introduce the StarSignDNA algorithm for mutational signature analysis, which offers efficient refitting and de novo mutational signature extraction. StarSignDNA is capable of deciphering well-differentiated signatures linked to known mutagenic mechanisms and suggesting clinically relevant predictions for a single patient. The package offers a command line-based interface and data visualization routines. Author summary StarSignDNA is a novel algorithm for identifying mutational signatures from cancer sequencing data. It excels in low mutation count scenarios, balancing signature detection and true signature discovery. StarSignDNA improves prediction accuracy and biological validity by addressing challenges such as overfitting and underfitting, achieving optimal variable selection and shrinkage, and ensuring interpretability. In refitting, it reduces prediction variance and handle sparsity, resulting in more precise predictions. In de novo extraction, it improves the detection of challenging signatures and achieves better alignment between detected and known signatures through unsupervisedcross-validation. Its unique features, including prediction confidence and customizable reference signatures, offer valuable insights for clinical single-sample analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a5a94297c1c9cbe8ea0fed782c76348b449df01" target='_blank'>
              StarSignDNA: Signature tracing for accurate representation of mutational processes
              </a>
            </td>
          <td>
            C. Bope, Sumana Kalyanasundaram, Knut D. Rand, S. Nakken, O. Lingjærde, E. Hovig
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Merkel cell polyomavirus (MCPyV) is the causative agent of the majority of Merkel cell carcinomas (MCC). The virus has limited coding capacity, with its early viral proteins, large T (LT) and small T (sT), being multifunctional and contributing to infection and transformation. A fundamental difference in early viral gene expression between infection and MCPyV-driven tumorigenesis is the expression of a truncated LT (LTtr) in the tumor. In contrast, sT is expressed in both conditions and contributes significantly to oncogenesis. Here, we identified novel functions of early viral proteins by performing genome-wide transcriptome and chromatin studies in primary human fibroblasts. Due to current limitations in infection and tumorigenesis models, we mimic these conditions by ectopically expressing sT, LT or LTtr, individually or in combination, at different time points. In addition to its known function in cell cycle and inflammation modulation, we reveal a fundamentally new function of sT. We show that sT regulates the type I interferon (IFN) response downstream of the type I interferon receptor (IFNAR) by interfering with the interferon-stimulated gene factor 3 (ISGF3)-induced interferon-stimulated gene (ISG) response. Expression of sT leads to a reduction in the expression of interferon regulatory factor 9 (IRF9) which is a central component of the ISGF3 complex. We further show that this function of sT is conserved in BKPyV. We provide a first mechanistic understanding of which early viral proteins trigger and control the type I IFN response, which may influence MCPyV infection, persistence and, during MCC progression, regulation of the tumor microenvironment. Author Summary Merkel cell polyomavirus (MCPyV) is the only human polyomavirus that causes cancer in humans. As with all human polyomaviruses, the available infection models are limited. Thus, many processes such as the host response to infection and its regulation by the virus to establish infection and persistence are poorly understood. To better understand this interplay of viral MCPyV proteins, we performed genome-wide transcriptome and chromatin studies in primary human fibroblasts and simulated infection and tumorigenesis conditions by ectopically expressing the early viral proteins individually or in combination at different time points. This allowed us to uncover a novel, previously undescribed function of polyomavirus sT, namely the reduction of the ISG response by affecting the ISGF3 complex, specifically by reducing IRF9 protein levels. This work sheds light on how early viral proteins influence the type I IFN response and how their interplay may affect MCPyV infection, persistence, and MCC progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc1f2080ed06b5ae44a5926e00540b8bdd290e5e" target='_blank'>
              Merkel cell polyomavirus small tumor antigen contributes to immune evasion by interfering with type I interferon signaling
              </a>
            </td>
          <td>
            Denise Ohnezeit, Jiabin Huang, Ute Westerkamp, Veronika Brinschwitz, Claudia Schmidt, Thomas Günther, Manja Czech‐Sioli, Samira Weißelberg, Tabea Schlemeyer, Jacqueline Nakel, Julia Mai, Sabrina Schreiner, Carola Schneider, C. Friedel, H. Schwanke, Melanie M. Brinkmann, A. Grundhoff, Nicole Fischer
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="The endonuclease activity of Pms1 directs mismatch repair by generating a nick in the newly replicated DNA strand. Inactivating Pms2, the human homologue of yeast Pms1, increases the chances of colorectal and uterine cancers. Here we use whole genome sequencing to show that loss of this endonuclease activity, via the pms1-DE variant, results in strong mutator effects throughout the Saccharomyces cerevisiae genome. Mutation rates are strongly increased for mutations resulting from all types of single-base substitutions and for a wide variety of single- and multi-base indel mutations. Rates for these events are further increased in strains combining pms1-DE with mutator variants of each of the three major leading and lagging strand replicases. In all cases, mutation rates, spectra, biases, and context preferences are statistically indistinguishable from strains with equivalent polymerases but lacking initial mismatch recognition due to deletion of MSH2. This implies that, across the nuclear genome, strand discrimination via the Pms1 endonuclease is as important for MMR as is initial mismatch recognition by Msh2 heterodimers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c81863bf8b4bcd0533ef4d09eddc5a93c0da4b" target='_blank'>
              Instability throughout the Saccharomyces cerevisiae genome resulting from Pms1 endonuclease deficiency.
              </a>
            </td>
          <td>
            S. Lujan, Marta A. Garbacz, S. E. Liberti, Adam B. Burkholder, Thomas A. Kunkel
          </td>
          <td>2024-07-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Radiation cytogenetics has a rich history seldom appreciated by those outside the field. Early radiobiology was dominated by physics and biophysical concepts that borrowed heavily from the study of radiation-induced chromosome aberrations. From such studies, quantitative relationships between biological effect and changes in absorbed dose, dose rate and ionization density were codified into key concepts of radiobiological theory that have persisted for nearly a century. This review aims to provide a historical perspective of some of these concepts, including evidence supporting the contention that chromosome aberrations underlie development of many, if not most, of the biological effects of concern for humans exposed to ionizing radiations including cancer induction, on the one hand, and tumor eradication on the other. The significance of discoveries originating from these studies has widened and extended far beyond their original scope. Chromosome structural rearrangements viewed in mitotic cells were first attributed to the production of breaks by the radiations during interphase, followed by the rejoining or mis-rejoining among ends of other nearby breaks. These relatively modest beginnings eventually led to the discovery and characterization of DNA repair of double-strand breaks by non-homologous end joining, whose importance to various biological processes is now widely appreciated. Two examples, among many, are V(D)J recombination and speciation. Rapid technological advancements in cytogenetics, the burgeoning fields of molecular radiobiology and third-generation sequencing served as a point of confluence between the old and new. As a result, the emergent field of "cytogenomics" now becomes uniquely positioned for the purpose of more fully understanding mechanisms underlying the biological effects of ionizing radiation exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c57db9d0b65c8dcb0db634d392394a245d1aa2a" target='_blank'>
              The Central Role of Cytogenetics in Radiation Biology.
              </a>
            </td>
          <td>
            Susan M. Bailey, Stephen R. Kunkel, Joel S. Bedford, Michael N. Cornforth
          </td>
          <td>2024-07-10</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background/Objectives The complex system of cancer has led to an emphasis on understanding the more general causal relationship within the disease. In this context, concepts of symmetry and symmetry-breaking in distinct biological cell features or components have been examined as an approach to cancer investigation. However, there can be possible limitations in directly interpreting the symmetry-based approach from a physical viewpoint due to the lack of understanding of physical laws governing symmetry in complex systems like cancer. Methods Fractal geometry and DNA walk representation were employed to investigate the geometric features i.e., self-similarity and heterogeneity in DNA nucleotide coding sequences of wild-type and mutated oncogenes, tumour-suppressor, and other unclassified genes. The mutation-facilitated self-similar and heterogenous features were quantified by the fractal dimension and lacunarity coefficient measures, respectively. Additionally, the geometrical orderedness and disorderedness in the analyzed sequences were interpreted from the combination of the fractal measures. Results The findings showed distinct fractal geometric features in the case of fusion mutations. It also highlights the possible interpretation of the observed fractal features as geometric analogues concerning explicit observations corresponding to specific cancer types. In addition, the two-dimensional multi-fractal analysis highlighted the presence of a single exponent in the scaling of mutation-mediated gene sequence self-similarity/complexity and heterogeneity. Conclusions The approach identified mutation-induced geometric features in gene sequences, demonstrating the potential of DNA walks and fractal analysis in translational research regarding cancer. The findings suggest that investigating fractal parameters can capture unique geometric features in nucleotide sequences, contributing to the understanding of cancer’s molecular complexity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a73efc783fc74b512f9c22058780b5d74879423" target='_blank'>
              DNA walk of specific fused oncogenes exhibit distinct fractal geometric characteristics in nucleotide patterns
              </a>
            </td>
          <td>
            Abhijeet Das, Manas Sehgal, A. Singh, Rishabh Goyal, Mallika Prabhakar, Jeremy Fricke, Isa Mambetsariev, Prakash Kulkarni, M. Jolly, Ravi Salgia
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e72b460ab8cbaf67ebb76f72b33e1a322a9c364" target='_blank'>
              MRE11 and TREX1 control senescence by coordinating replication stress and interferon signaling
              </a>
            </td>
          <td>
            Hervé Técher, Diyavarshini Gopaul, Jonathan Heuzé, Nail Bouzalmad, Baptiste Leray, Audrey Vernet, C. Mettling, J. Moreaux, P. Pasero, Yea-Lih Lin
          </td>
          <td>2024-06-26</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b14b3a2b3b2cb05b6f12ac40b8502ecde1ffe45" target='_blank'>
              ASS1 metabolically contributes to the nuclear and cytosolic p53-mediated DNA damage response
              </a>
            </td>
          <td>
            L. Lim, Lital Adler, E. Hajaj, Leandro R Soria, R. B. Perry, Naama Darzi, Ruchama Brody, N. Furth, Michal Lichtenstein, Elizabeta Bab-Dinitz, Z. Porat, Tevie Melman, Alexander Brandis, Yael Aylon, S. Ben-Dor, Irit Orr, Amir Pri-Or, Rony Seger, Y. Shaul, E. Ruppin, Moshe Oren, Minervo Perez, Jordan Meier, N. Brunetti‐Pierri, E. Shema, I. Ulitsky, A. Erez, S. Malitsky, Maxim Itkin
          </td>
          <td>2024-06-10</td>
          <td>Nature Metabolism</td>
          <td>1</td>
          <td>52</td>
        </tr>

        <tr id="Access to DNA is the first level of control in regulating gene transcription, a control that is also critical for maintaining DNA integrity. Cellular senescence is characterized by profound transcriptional rearrangements and accumulation of DNA lesions. Here, we discovered an epigenetic complex between HDAC4 and HDAC1/HDAC2 that is involved in the erase of H2BK120 acetylation. The HDAC4/HDAC1/HDAC2 complex modulates the efficiency of DNA repair by homologous recombination, through dynamic deacetylation of H2BK120. Deficiency of HDAC4 leads to accumulation of H2BK120ac, impaired recruitment of BRCA1 and CtIP to the site of lesions, accumulation of damaged DNA and senescence. In senescent cells this complex is disassembled because of increased proteasomal degradation of HDAC4. Forced expression of HDAC4 during RAS-induced senescence reduces the genomic spread of γH2AX. It also affects H2BK120ac levels, which are increased in DNA-damaged regions that accumulate during RAS-induced senescence. In summary, degradation of HDAC4 during senescence causes the accumulation of damaged DNA and contributes to the activation of the transcriptional program controlled by super-enhancers that maintains senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b53e88311145d59601f68000e7e2fc44b0f2dcda" target='_blank'>
              HDAC4 influences the DNA damage response and counteracts senescence by assembling with HDAC1/HDAC2 to control H2BK120 acetylation and homology-directed repair.
              </a>
            </td>
          <td>
            Eros Di Giorgio, E. Dalla, Vanessa Tolotto, Francesca D'Este, Harikrishnareddy Paluvai, Liliana Ranzino, Claudio Brancolini
          </td>
          <td>2024-06-14</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe60321069cd8bdd4dcd7dd1f6cb9aee5e0996c" target='_blank'>
              Z-DNA formation in promoters conserved between human and mouse are associated with increased transcription reinitiation rates
              </a>
            </td>
          <td>
            N. Beknazarov, Dmitry Konovalov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-08-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Motivation By accounting for variants within measured transcripts, it is possible to evaluate the status of somatic variants using single-cell RNA-sequencing (scRNA-seq) and to characterize their clonality. However, the sparsity (very few reads per transcript) or bias in protocols (favoring 3’ ends of the transcripts) makes the chance of capturing somatic variants very unlikely. This can be overcome by targeted sequencing or the use of mitochondrial variants as natural barcodes for clone identification. Currently, available computational tools focus on genotyping, but do not provide functionality for combined analysis of somatic and mitochondrial variants and functional analysis such as characterization of gene expression changes in detected clones. Results Here, we propose SIGURD, which is an R-based pipeline for the clonal analysis of single-cell RNA-seq data. This allows the quantification of clones by leveraging both somatic and mitochondrial variants. SIGURD also allows for functional analysis after clonal detection: association of clones with cell populations, detection of differentially expressed genes across clones and association of somatic and mitochondrial variants. Here, we demonstrate the power of SIGURD by analyzing single-cell data of colony-forming cells derived from patients with myeloproliferative neoplasms. Availability Code and tutorial of SIGURD are available at GitHub https://github.com/CostaLab/sigurd. Contact ivan.costa@rwth-aachen.de, skoschmieder@ukaachen.de Supplementary Information Supplementary data are available online.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45e9a768e4ded9130265195c996a3adf0b6cd770" target='_blank'>
              SIngle cell level Genotyping Using scRna Data (SIGURD)
              </a>
            </td>
          <td>
            Martina Grasshoff, Milena Kalmer, Nicolas Chatain, K. Kricheldorf, Angela Maurer, Ralf Weiskirchen, Steffen Koschmieder, Ivan G. Costa
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3457bc89fd8d1e79afb89ebab809856dee664511" target='_blank'>
              SPIN1 facilitates chemoresistance and HR repair by promoting Tip60 binding to H3K9me3.
              </a>
            </td>
          <td>
            Yukun Wang, Mengyao Li, Yuhan Chen, Yuhan Jiang, Ziyu Zhang, Zhenzhen Yan, Xiuhua Liu, Chen Wu
          </td>
          <td>2024-08-01</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41728da5eec57442f344cb37688d002e66c10d7e" target='_blank'>
              Tumor initiation and early tumorigenesis: molecular mechanisms and interventional targets
              </a>
            </td>
          <td>
            Shaosen Zhang, Xinyi Xiao, Yonglin Yi, Xinyu Wang, Lingxuan Zhu, Yanrong Shen, Dongxin Lin, Chen Wu
          </td>
          <td>2024-06-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3a5b1503e18f7793a71f75ddd7431eddfa74929" target='_blank'>
              YTHDC1 m6A-dependent and m6A-independent functions converge to preserve the DNA damage response.
              </a>
            </td>
          <td>
            Daniel Elvira-Blázquez, J. M. Fernández-Justel, Aida Arcas, Luisa Statello, Enrique Goñi, Jovanna González, Benedetta Ricci, S. Zaccara, Ivan Raimondi, Maite Huarte
          </td>
          <td>2024-07-01</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Aneuploidy, or a change in the number of whole chromosomes or chromosome arms, is a near-universal feature of cancer. Chromosomes affected by aneuploidy are not random, with observed cancer-specific and tissue-specific patterns. Recent advances in genome engineering methods have allowed the creation of models with targeted aneuploidy events. These models can be used to uncover the downstream effects of individual aneuploidies on cancer phenotypes including proliferation, apoptosis, metabolism, and immune signaling. Here, we review the current state of research into the patterns of aneuploidy in cancer and their impact on signaling pathways and biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53976b63387a7558e4fe16be8478ae589398c516" target='_blank'>
              Patterns of Aneuploidy and Signaling Consequences in Cancer.
              </a>
            </td>
          <td>
            Nadja Zhakula-Kostadinova, Alison M Taylor
          </td>
          <td>2024-06-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Aberrant oncogenic signaling causes cells to transition into oncogene-induced senescence (OIS) to limit uncontrolled proliferation. Despite being a potent tumor suppressor mechanism, OIS is an unstable cell state susceptible to reprogramming that can promote tumorigenesis. Therefore, elucidating the underlying gene regulatory mechanisms that commit cells to OIS is critical to identifying actionable targets to modulate the senescence state. We previously showed that timely execution of the OIS program is governed by hierarchical transcription factor (TF) networks. However, the gene regulatory mechanisms that prime cells to commit to the OIS fate early upon oncogene hyperactivation are currently not known. Here, we leveraged our time-resolved multi-omic profiling approach to generate TF networks during the first 24 h of oncogenic HRASG12V activation. Using this approach, we demonstrate that the commitment to OIS requires the rearrangement of the TF network on a pre-established epigenomic landscape, priming the cells for the substantial chromatin remodeling that underpins the transition to OIS. Our results provide a detailed map of the chromatin landscape before cells transition to OIS thus offering a platform for manipulation of senescence outcomes of potentially therapeutic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412ffff3bb32763efa919fb353d8bfbc6d71328f" target='_blank'>
              Transcription factor network dynamics during the commitment to oncogene-induced senescence
              </a>
            </td>
          <td>
            Themistoklis Vasilopoulos, R. Martínez-Zamudio
          </td>
          <td>2024-07-15</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64dd346087b1d64ab0e5ccbf0c92c3994e5f755a" target='_blank'>
              Characterization and regulation of cell cycle-independent noncanonical gene targeting
              </a>
            </td>
          <td>
            Shinta Saito, N. Adachi
          </td>
          <td>2024-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Non-coding regions of the human genome are important for functional regulations, but their mechanisms remain elusive. We used machine learning to guide a CRISPR screening on hubs (i.e. non-coding loci forming many 3D contacts) and significantly increased the discovery rate of hubs essential for cell growth. We found no clear genetic or epigenetic differences between essential and nonessential hubs, but we observed that some neighboring hubs in the linear genome have distinct spatial contacts and opposite effects on cell growth. One such pair in an epigenetically quiescent region showed different impacts on gene expression, chromatin accessibility and chromatin organization. We also found that deleting the essential hub altered the genetic network activity and increased the entropy of chromatin accessibility, more severe than that caused by deletion of the nonessential hub, suggesting that they are critical for maintaining an ordered chromatin structure. Our study reveals new insights into the system-level roles of non-coding regions in the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37fecffd504fcf6b86c79eeec8f4d7ba8dc1c39" target='_blank'>
              Differential impact of quiescent non-coding loci on chromatin entropy.
              </a>
            </td>
          <td>
            Peiyao Wu, Mina Yao, Wei Wang
          </td>
          <td>2024-06-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Disease risk prediction based on genomic sequence and transcriptional profile can improve disease screening and prevention. Despite identifying many disease-associated DNA variants, distinguishing deleterious non-coding DNA variations remains poor for most common diseases. In this study, we designed in vitro experiments to uncover the significance of occupancy and competitive binding between P53 and cMYC on common target genes. Analyzing publicly available ChIP-seq data for P53 and cMYC in embryonic stem cells showed that ~344-366 regions are co-occupied, and on average, two cis-overlapping motifs (CisOMs) per region were identified, suggesting that co-occupancy is evolutionarily conserved. Using U2OS and Raji cells untreated and treated with doxorubicin to increase P53 protein level while potentially reducing cMYC level, ChIP-seq analysis illustrated that around 16 to 922 genomic regions were co-occupied by P53 and cMYC, and substitutions of cMYC signals by P53 were detected post doxorubicin treatment. Around 187 expressed genes near co-occupied regions were altered at mRNA level according to RNA-seq data analysis. We utilized a computational motif-matching approach to illustrate that changes in predicted P53 binding affinity in CisOMs of co-occupied elements significantly correlate with alterations in reporter gene expression. We performed a similar analysis using SNPs mapped in CisOMs for P53 and cMYC from ChIP-seq data, and expression of target genes from GTEx portal. We found significant correlation between change in cMYC-motif binding affinity in CisOMs and altered expression. Our study brings us closer to developing a generally applicable approach to filter etiological non-coding variations associated with common diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2175dd5f742d55fc20f67f497dba908aa719b17b" target='_blank'>
              Sequence-to-expression approach to identify etiological non-coding DNA variations in P53 and cMYC-driven diseases.
              </a>
            </td>
          <td>
            K. Kin, Shounak Bhogale, Lisha Zhu, Derrick Thomas, Jessica W Bertol, W. J. Zheng, Saurabh Sinha, W. Fakhouri
          </td>
          <td>2024-07-17</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4597b2c9168e5b3eb619c4010a4d29dad0a9c28b" target='_blank'>
              mTORC2-driven chromatin cGAS mediates chemoresistance through epigenetic reprogramming in colorectal cancer.
              </a>
            </td>
          <td>
            Guoqing Lv, Qian Wang, Lin Lin, Qiao Ye, Xi Li, Qian Zhou, Xiangzhen Kong, Hongxia Deng, Fuping You, Hebing Chen, Song Wu, Lin Yuan
          </td>
          <td>2024-07-30</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers. This phenomenon has been investigated since the late 1980s without resolution. Expanding beyond previous gene-centric studies, we investigated the co-occurrence in a genome-wide manner taking an evolutionary perspective. Mining of large-scale tumor aneuploidy data confirmed the previous finding of a small-scale longitudinal study that the most likely order is chromosome 10 loss followed by chromosome 7 gain. Extensive analysis of genomic and transcriptomic data from both patients and cell lines revealed that this co-occurrence can be explained by functional rescue interactions that are highly enriched on chromosome 7, which could potentially compensate for any detrimental consequences arising from the loss of chromosome 10. Transcriptomic data from various normal, non-cancerous human brain tissues were analyzed to assess which tissues may be most predisposed to tolerate compensation of chromosome 10 loss by chromosome 7 gain. The analysis indicated that the pre-existing transcriptomic states in the cortex and frontal cortex, where gliomas arise, are more favorable than other brain regions for compensation by rescuer genes that are active on chromosome 7. Collectively, these findings suggest that the phenomenon of chromosome 10 loss and chromosome 7 gain in gliomas is orchestrated by a complex interaction of many genes residing within these two chromosomes and provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff13af5a4c0be0182481407e6fc3feb597b938be" target='_blank'>
              Chromosome 7 gain compensates for chromosome 10 loss in glioma.
              </a>
            </td>
          <td>
            N. Nair, Alejandro A Schaffer, E. Gertz, Kuoyuan Cheng, Johanna Zerbib, A. Sahu, Gil Leor, E. Shulman, Kenneth D Aldape, Uri Ben-David, E. Ruppin
          </td>
          <td>2024-07-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="53BP1 plays a crucial role in regulating DNA damage repair pathway choice and checkpoint signaling in somatic cells; however, its role in meiosis has remained enigmatic. In this study, we demonstrate that the Caenorhabditis elegans ortholog of 53BP1, HSR-9, associates with chromatin in both proliferating and meiotic germ cells. Notably, HSR-9 is enriched on the X chromosome pair in pachytene oogenic germ cells. HSR-9 is also present at kinetochores during both mitotic and meiotic divisions but does not appear to be essential for monitoring microtubule-kinetochore attachments or tension. Using cytological markers of different steps in recombinational repair, we found that HSR-9 influences the processing of a subset of meiotic double strand breaks into COSA-1-marked crossovers. Additionally, HSR-9 plays a role in meiotic X chromosome segregation under conditions where X chromosomes fail to pair, synapse, and recombine. Together, these results highlight that chromatin-associated HSR-9 has both conserved and unique functions in the regulation of meiotic chromosome behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e700bc11eabfed38f5c48636a6b97e3c3470f7b" target='_blank'>
              The chromatin-associated 53BP1 ortholog, HSR-9, regulates recombinational repair and X chromosome segregation in the Caenorhabditis elegans germ line.
              </a>
            </td>
          <td>
            Qianyan Li, Sara Hariri, Aashna Calidas, Arshdeep Kaur, Erica Huey, J. Engebrecht
          </td>
          <td>2024-06-17</td>
          <td>Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Membrane-associated tyrosine- and threonine-specific Cdc2-inhibitory kinase (PKMYT1) is a cell cycle regulatory kinase that inhibits CDK1/CyclinB activity, delaying entry into mitosis in tumor cells experiencing replication stress (RS). RS is frequently induced by genetic alterations that drive premature transition from G1 to S phase, promoting genome instability and creating a synthetic lethal (SL) relationship between these specific alterations and PKMYT1 inhibition. This relationship has been demonstrated preclinically and clinically with CCNE1 amplification as well as FBXW7 and PPP2R1A deleterious mutations. These genetic alterations are common across a range of tumors including ovarian, endometrial and colorectal cancers. On this basis, we developed lunresertib (RP-6306), a first-in-class, potent and selective oral PKMYT1 inhibitor currently being investigated in phase I clinical trials (NCT04855656, NCT05147272, NCT05147350). Here we discuss the translation of pre-clinical data to the clinical setting as both monotherapy and mechanistic combinations approaches.
 Citation Format: Michael Zinda. Targeting genome instability in cancer: Translating synthetic lethal biology into the clinic [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b2dab79a77a92e7c288ec66eea236365dc8d8a1" target='_blank'>
              Abstract IA010: Targeting genome instability in cancer: Translating synthetic lethal biology into the clinic
              </a>
            </td>
          <td>
            M. Zinda
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 c-MYC (MYC) is a central regulatory protein that is dysregulated in >50% of all human cancers and is linked to aggressive disease. Developing MYC inhibitors would revolutionize cancer treatment; however, developing small molecules that directly target MYC is challenging. An alternative approach is to identify and inhibit critical MYC partner proteins to inactivate MYC and trigger cancer cell death. Inhibiting these targets therapeutically can result in synthetic lethality (MYC-SL), which can be exploited in MYC-dysregulated cancers. To identify MYC-SL targets, we performed a genome-wide CRISPR knock-out screen using an isogenic pair of normal and MYC-driven breast cancer cells. In contrast to other screens, this model is dependent on MYC and recapitulates human disease at both pathological and molecular levels in vivo. We identified high-priority hits to validate from the screen using two independent approaches: 1) a traditional gene-set enrichment analysis to identify highly represented biological pathways; and 2) analyzing the Cancer Dependency Map (DEPMAP) to select hits that are likely to be robust beyond the context of our screening conditions. Using a traditional gene-set enrichment analysis approach, we identified topoisomerase 1 (TOP1) as an actionable vulnerability that can be targeted with clinically approved inhibitors. Genetic and pharmacological inhibition of TOP1 resulted in MYC-driven cell death compared to that in control cells. Finally, drug response to TOP1 inhibitors significantly correlated with MYC levels and activity across panels of breast cancer cell lines and patient-derived organoids, highlighting TOP1 as a promising target for MYC-driven cancers. As a secondary approach to interpreting our CRISPR screen hits, we analyzed DEPMAP to identify MYC-SLs that are differentially essential in MYC-dependent cancer cells. Specifically, data from RNA interference screens in hundreds of cancer cell lines were used to describe the response of these cells to MYC knockdown. These data were used to define MYC-dependent and MYC-independent cell lines within the context of this analysis. These two groups were then investigated for their in silico response to the knockdown of each of our MYC-SL hits. MYC-SLs, which were also differentially essential in MYC-dependent cancer cells from DEPMAP, were prioritized for further investigation. Critical MYC cofactors that have been validated by us and others (e.g., CDK9) were identified, providing confidence in this approach, and rationalizing ongoing investigations. Together, this work features two successful strategies to prioritize hits from hundreds of synthetic-lethal genome-wide CRISPR screens to identify novel MYC-driven vulnerabilities in cancer.
 Citation Format: Peter Lin, Linda Penn. An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fe21fa230f85037dcec9224411cd263d57af781" target='_blank'>
              Abstract B004: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Linda Penn
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54a1de3c94a3135d7ef2bd2539530b5e0e7ee5f8" target='_blank'>
              DNA mismatch and damage patterns revealed by single-molecule sequencing.
              </a>
            </td>
          <td>
            Mei Hong Liu, Benjamin M Costa, Emilia C Bianchini, Una Choi, Rachel C. Bandler, E. Lassen, Marta Grońska-Pęski, Adam Schwing, Z. R. Murphy, Daniel Rosenkjær, Shany Picciotto, V. Bianchi, L. Stengs, M. Edwards, N. Nunes, Caitlin A. Loh, Tina K Truong, Randall E Brand, Tomi Pastinen, J. R. Wagner, A. Skytte, U. Tabori, J. Shoag, G. Evrony
          </td>
          <td>2024-06-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="In this study, we investigated recurrent copy number variations (CNVs) in the 19p12 locus, which are associated with neurodevelopmental disorders. The two genes in this locus, ZNF675 and ZNF681, arose via gene duplication in primates, and their presence in several pathological CNVs in the human population suggests that either or both of these genes are required for normal human brain development. ZNF675 and ZNF681 are members of the Krüppel-associated box zinc finger (KZNF) protein family, a class of transcriptional repressors important for epigenetic silencing of specific genomic regions. About 170 primate-specific KZNFs are present in the human genome. Although KZNFs are primarily associated with repressing retrotransposon-derived DNA, evidence is emerging that they can be co-opted for other gene regulatory processes. We show that genetic deletion of ZNF675 causes developmental defects in cortical organoids, and our data suggest that part of the observed neurodevelopmental phenotype is mediated by a gene regulatory role of ZNF675 on the promoter of the neurodevelopmental gene Hes family BHLH transcription factor 1 (HES1). We also find evidence for the recently evolved regulation of genes involved in neurological disorders, microcephalin 1 and sestrin 3. We show that ZNF675 interferes with HES1 auto-inhibition, a process essential for the maintenance of neural progenitors. As a striking example of how some KZNFs have integrated into preexisting gene expression networks, these findings suggest the emergence of ZNF675 has caused a change in the balance of HES1 autoregulation. The association of ZNF675 CNV with human developmental disorders and ZNF675-mediated regulation of neurodevelopmental genes suggests that it evolved into an important factor for human brain development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2122cac7536c55e0373736baf9954110c9f4179" target='_blank'>
              Emergence of the ZNF675 Gene During Primate Evolution-Influenced Human Neurodevelopment Through Changing HES1 Autoregulation.
              </a>
            </td>
          <td>
            G. A. Lodewijk, Matthijs de Geus, Rita L F P Guimarães, F. Jacobs
          </td>
          <td>2024-07-01</td>
          <td>The Journal of comparative neurology</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Recent advances in DNA sequencing technology have enabled the precise decoding of genomes in non-model organisms, providing a basis for unraveling the patterns and mechanisms of sex chromosome evolution. Studies of different species have yielded conflicting results regarding the traditional theory that sex chromosomes evolve from autosomes via the accumulation of deleterious mutations and degeneration of the Y (or W) chromosome. The concept of the ‘sex chromosome cycle,’ emerging from this context, posits that at any stage of the cycle (i.e., differentiation, degeneration, or loss), sex chromosome turnover can occur while maintaining stable sex determination. Thus, understanding the mechanisms that drive both the persistence and turnover of sex chromosomes at each stage of the cycle is crucial. In this review, we integrate recent findings on the mechanisms underlying maintenance and turnover, with a special focus on several organisms having unique sex chromosomes. Our review suggests that the diversity of sex chromosomes in the maintenance of stable sex determination is underappreciated and emphasizes the need for more research on the sex chromosome cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ad3585db829ba2d4ab84372e9e70791e552b39" target='_blank'>
              Sex chromosome cycle as a mechanism of stable sex determination
              </a>
            </td>
          <td>
            Shun Hayashi, Takuya Abe, Takeshi Igawa, Yukako Katsura, Yusuke Kazama, Masafumi Nozawa
          </td>
          <td>2024-07-10</td>
          <td>Journal of Biochemistry</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Neuroblastoma (NB) is a heterogeneous childhood cancer with a slightly higher incidence in boys than girls, with the reason for this gender disparity unknown. Given the growing evidence for the involvement of loss of the Y chromosome (LoY) in male diseases including cancer, we investigated Y chromosome status in NB. Male NB tumor samples from a Swedish cohort, analyzed using Cytoscan HD SNP-microarray, were selected. Seventy NB tumors were analyzed for aneuploidy of the Y chromosome, and these data were correlated with other genetic, biological, and clinical parameters. LoY was found in 21% of the male NB tumors and it was almost exclusively found in those with high-risk genomic profiles. Furthermore, LoY was associated with increased age at diagnosis and enriched in tumors with 11q-deletion and activated telomere maintenance mechanisms. In contrast, tumors with an MYCN-amplified genomic profile retained their Y chromosome. The understanding of LoY in cancer is limited, making it difficult to conclude whether LoY is a driving event in NB or function of increased genomic instability. Gene expression analysis of Y chromosome genes in male NB tumors showed low expression of certain genes correlating with worse overall survival. KDM5D, encoding a histone demethylase stands out as an interesting candidate for further studies. LoY has been shown to impact the epigenomic layer of autosomal loci in nonreproductive tissues, and KDM5D has been reported as downregulated and/or associated with poor survival in different malignancies. Further studies are needed to explore the mechanisms and functional consequences of LoY in NB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b6c56e8229f3a5587d983a3bab25a468082a5c25" target='_blank'>
              Loss of Chromosome Y in Neuroblastoma Is Associated With High-Risk Disease, 11q-Deletion, and Telomere Maintenance.
              </a>
            </td>
          <td>
            A. Djos, Johanna Svensson, Jennie Gaarder, Ganesh Umapathy, Staffan Nilsson, Torben Ek, Hartmut Vogt, Kleopatra Georgantzi, Ingrid Öra, C. Träger, P. Kogner, Tommy Martinsson, S. Fransson
          </td>
          <td>2024-07-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Turner syndrome (TS) is caused by a complete or partial absence of an X or Y chromosome, including chromosomal mosaicism, affecting 1 in 2500 female live births. Sister chromatid exchange (SCE) is used as a sensitive indicator of spontaneous chromosome instability. Cells from mosaic patients constitute useful material for SCE evaluations as they grow under the influence of the same genetic background and endogenous and exogenous factors. We evaluated the proliferation dynamics and SCE frequencies of 45,X and 46,XN cells of 17 mosaic TS patients. In two participants, the 45,X cells exhibited a proliferative disadvantage in relation to 46,XN cells after 72 h of cultivation. The analysis of the mean proliferation index (PI) showed a trend for a significant difference between the 45,X and 46,X+der(X)/der(Y) cell lineages; however, there were no intra-individual differences. On the other hand, mean SCE frequencies showed that 46,X+der(X) had the highest mean value and 46,XX the lowest, with 45,X occupying an intermediate position among the lineages found in at least three participants; moreover, there were intra-individual differences in five patients. Although 46,X+der(X)/der(Y) cell lineages, found in more than 70% of participants, were the most unstable, they had a slightly higher mean PI than the 45,X cell lineages in younger (≤17 years) mosaic TS participants. This suggests that cells with a karyotype distinct from 45,X may increase with time in mosaic TS children and adolescents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a35420c99ba97fe4f5acd7a759bcc6a732cb8d1" target='_blank'>
              Cell Cycle Kinetics and Sister Chromatid Exchange in Mosaic Turner Syndrome
              </a>
            </td>
          <td>
            M. B. Goulart, Eduardo Vieira Neto, Daniela R. Ney Garcia, Marília Martins Guimarães, Isaías Soares de Paiva, Karina de Ferran, Nathalia Correia Krause dos Santos, Luciana Santos Barbosa, Amanda F. de Figueiredo, M. Ribeiro, Márcia Gonçalves Ribeiro
          </td>
          <td>2024-07-01</td>
          <td>Life</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8868db406f4d297083b3336bdacf7e70f944e3" target='_blank'>
              Characterize molecular signatures and establish a prognostic signature of gastric cancer by integrating single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Zhiwei Wang, Zhiyan Weng, Luping Lin, Xianyi Wu, Wenju Liu, Yong Zhuang, J. Jian, Changhua Zhuo
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87375be953b6b1e62c0930938d4b8c30692440ac" target='_blank'>
              Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants.
              </a>
            </td>
          <td>
            Max Frenkel, James E. Corban, Margaux L. A. Hujoel, Zachary Morris, Srivatsan Raman
          </td>
          <td>2024-07-24</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ad718715d0e23f67c54707d35f7ee63dc651a7" target='_blank'>
              Homeotic DUX4 Genes Shape Dynamic Inter-Chromosomal Contacts with Nucleoli in Human Cells.
              </a>
            </td>
          <td>
            E. S. Klushevskaya, I. R. Alembekov, Y. V. Kravatsky, N. A. Tchurikov
          </td>
          <td>2024-07-13</td>
          <td>Doklady. Biochemistry and biophysics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Deficiency of homologous recombination (HR)-mediated DNA repair occurs through genetic or epigenetic inactivation of the BRCA1 and BRCA2 (BRCA1/2) genes. HR-deficiency also provides unique opportunities for targeted therapy, through the mechanism of synthetic lethality, as exemplified by the hypersensitivity of BRCA1/2-mutated tumors to PARP inhibitors (PARPi). To date, several PARP inhibitors have been approved for clinical use. The promising clinical response of patients with germline BRCA1/2-mutations prompted the use of PARPi for patients with somatic BRCA1/2 mutations as well. In addition, these results extended the use of PARPi for various types of ovarian, breast, pancreatic, and prostate tumors with HR defects. PARPi resistance is emerging as the major obstacle to clinical effectiveness in patients with HR-deficient tumors. PARPi resistance results from several independent mechanisms, leading to the restoration of Homologous Recombination and/or Replication Fork stabilization. The absence of alternative options for patients with tumors with innate or acquired resistance underlines the urgency to develop additional therapeutics. More recent studies have identified new synthetic lethal opportunities for BRCA1/2 deficient tumors. Knockout of the genes for POLQ (DNA polymerase theta) or for USP1 (Ubiquitin Specific Protease 1) results in killing of these tumors, and inhibitors of these enzymes are emerging as promising agents for HR-deficient tumors. POLQ and USP1 expression is particularly high in subtypes of breast and ovarian tumors with defects in HR. As a result, POLQi or USP1i in HR-deficient tumors induces cell death. POLQi or USP1i can synergize with PARPi in killing HR-deficient tumors or PARPi-resistant tumors. BRCA1/2 deficient tumors have an increase in single strand DNA gaps (ssGAPs) near replication forks. Treatment with PARPi results in an increase in the size and number for these ssGAPs, ultimately leading to DSBs and cell death. Interestingly, PARPi resistance results, at least in part, from the ability of resistant cells to close the ssGAPs through enhanced POLQ and USP1 activity, thereby providing a rationale for the combination of PARP1, POLQi, and USP1 in HR-deficient cancers. USP1 inhibition was synergistic with PARP and POLQ inhibition in BRCA1-mutant cells, with enhanced ssDNA gap accumulation. Moreover, this synergy was observed in a set of patient-derived ovarian cancer organoids (PDOs), thus confirming the sensitivity of BRCA1-deficient cells to inhibition by these agents. The accumulation of ssDNA gaps after treatment with a USP1i, PARP, or POLQi correlated with the sensitivity to these drugs in all models tested. Ovarian cancer PDOs provide a powerful tool for detecting drug synergy and for rapid in vitro sensitivity testing. The detection of ssDNA gap accumulation may be a useful predictive biomarker for response to DNA Repair inhibitors as monotherapy or in combination in ongoing clinical trials.
 Citation Format: Alan D. D'Andrea. Single strand DNA GAP accumulation as a functional biomarker for DNA Repair Inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28c9361e88294fef127d64e67bfd18dda9e6b654" target='_blank'>
              Abstract IA009: Single strand DNA GAP accumulation as a functional biomarker for DNA Repair Inhibitors
              </a>
            </td>
          <td>
            Alan D. D'Andrea
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Cytomegalovirus (CMV) has long been thought to have an association with glioblastoma multiforme (GBM), although the exact role of CMV and any subsequent implications for treatment have yet to be fully understood. This study addressed whether IGH complementarity determining region-3 (CDR3)-CMV protein chemical complementarity, with IGH CDR3s representing both tumor resident and blood-sourced IGH recombinations, was associated with overall survival (OS) distinctions. IGH recombination sequencing reads were obtained from (a) the Clinical Proteomic Tumor Analysis Consortium, tumor RNAseq files; and (b) the cancer genome atlas, blood exome-derived files. The Adaptive Match web tool was used to calculate chemical complementarity scores (CSs) based on hydrophobic interactions, and those scores were used to group GBM cases and assess survival probabilities. We found a higher OS probability for cases whose hydrophobic IGH CDR3-CMV protein chemical complementarity scores (Hydro CSs) were in the upper 50th percentile for several CMV proteins, including UL99 and UL123, as well as for CSs based on known B cell epitopes representing these proteins. We also identified multiple immune signature genes, including CD79A and TNFRSF17, for which higher RNA expression was associated with higher Hydro CSs. Results were consistent with the idea that stronger immunoglobulin responses to CMV are associated with better OS probabilities for GBM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3eda5c3758f160f2f49f0717c246c8a859c720bf" target='_blank'>
              IGH Complementarity Determining Region-3-Cytomegalovirus Protein Chemical Complementarity Linked to Better Overall Survival Probabilities for Glioblastoma.
              </a>
            </td>
          <td>
            Tabitha R Hudock, Joanna J Song, Andrea Chobrutskiy, B. Chobrutskiy, G. Blanck
          </td>
          <td>2024-06-07</td>
          <td>Viral immunology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="


 How does accurate, unbiased whole genome amplification and tiered, independent, two component analysis impact PGT-A results?



 When compared to previous methods a PGT-A workflow utilizing PTA followed by a CNV and allelic balance-based analysis generated more conclusive results.



 Accurate PGT-A results require unbiased error-free whole genome amplification from the trophectoderm biopsy and smart bioinformatics utilities to fully analyze and consider the large complex datasets. PTA generates fewer errors, greater coverage, and reduced allele drop-out offering the best starting point to interpret the genome of each embryo. Current PGT-A platforms leverage either normalized CNV data OR allelic balance at single nucleotide polymorphisms (SNPs), to detect alterations in the embryonic genome. A pipeline leveraging improved amplification technology and consolidated CNV and SNP data, particularly for putative mosaic and segmental mosaic calls, could reduce uncertainty and provide more accurate PGT-A results.



 A retrospective analysis of PGT-A results was undertaken with samples originating from multiple clinics and processed in a single laboratory, from January through December, 2023. Approximately 500 samples were processed simultaneously through two independent PGT-A pipelines. The primary objective was to compare the performance of CNV-only PGT-A with a test leveraging both CNV and SNP data. A secondary objective was to observe the resolution of mosaic embryos.



 De-identified samples were amplified with PTA and sequenced with both shotgun (coverage-based CNV) and capture (SNP-based)-approaches. Method specific analysis was applied to the resulting data. An initial bioinformatics approach analyzed CNV data only for variations in copy-number along the genome, proposing stretches of aneuploidy or mosaicism. A second, tiered analysis extended this data and incorporated SNP-based allelic balance to evaluate candidate regions and confirm the presence of aneuploidy and/or quantify mosaic levels.



 Genome-wide read coverage and SNP data confirmed that PTA accurately and completely amplified the genetic material in each sample with broad distribution of informative data and a low allele drop-out rate. High concordance (95%) was observed for euploid and full aneuploid (whole chromosome and segmental) embryos classified with CNV-alone and CNV/SNV combined analysis. SNP data was processed, and B-allele ratios were used to confirm aneuploidies with particular utility in resolving the location of segmental changes. The classification of mosaic samples was altered in the pipeline leveraging both CNV and SNP data. Of 54 mosaic samples tested, 75% remained mosaic after independent consideration of SNP data, ∼15% and 6% of high-level mosaics were resolved to full aneuploid and euploid, respectively, and ∼25% of low-level mosaics reverted to a euploid classification. The deep SNP data produced during targeted sequencing confirmed the altered allele ratio predicted by CNV data and re-confirmed the presence and proportion of aneuploid cells. Mosaic levels were highly correlated in 90% of re-confirmed mosaic samples. This suggests that the classification of these samples as mosaic is precise and accurate and suggests a strong utility for a two component PGT-A pipeline leveraging CNV and SNV data generated from PTA embryos.



 This was a retrospective study with limited sample numbers.



 The study highlights the strong advantages of a hybrid PGT-A pipeline, leveraging PTA and two component analysis of coverage and allelic balance, particularly to resolve mosaic embryos more accurately. Overall, this could reduce uncertainty and deliver more confident PGT-A results for embryo transfer decision making.



 not applicable
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78cfdcbb679168ac6cbb7cde95000c441314f7dd" target='_blank'>
              O-271 Two component resolution of mosaic samples in PGT-A: primary template amplification (PTA) and independent analysis using copy-number variation and allelic balance
              </a>
            </td>
          <td>
            C. Weier, A. Griffith, K. McKissock, K. Brown
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a299a59bda6de4c1c2ea84f852c64d2057c80e3e" target='_blank'>
              Mapping alternative splicing events in colorectal cancer
              </a>
            </td>
          <td>
            Yifeng Zheng, Guoqiang Zhong, Qiuyu Song, Haonan Zhang, Shanping Wang, Chuangzhen Lin, Chengcheng He, Mingsong Li
          </td>
          <td>2024-07-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Gene conversions are broadly defined as the transfer of genetic material from a ‘donor’ to an ‘acceptor’ sequence and can happen both in meiosis and mitosis. They are a subset of non-crossover events and like crossover events, gene conversion can generate new combinations of alleles, erode linkage disequilibrium, and even counteract the mutation load by reverting germline mutations through GC-biased gene conversion. Estimating the rate of gene conversion and the distribution of gene conversion tract lengths remains challenging. Here, we present a new method for estimating tract length, rate and detection probability of non-crossover events directly in HiFi PacBio long read data. The method can be applied with data from a single individual, is unbiased even under low single nucleotide variant densities and does not necessitate any demographic or evolutionary assumptions. We apply the method to gene conversion events observed directly in Pacbio HiFI read data from a human sperm sample and find that human gene conversion tracts are shorter (mean of 50 base pairs) than estimates from yeast or Drosophila. We also estimate that typical human male gametes undergo on average 280 non-crossover events where approximately 7 are expected to become visible as gene conversions moving variants from one donor haplotype to an acceptor haplotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f463a46e1b5590cae1deadecdd92d1f8b1f8321" target='_blank'>
              Estimating gene conversion tract length and rate from PacBio HiFi data
              </a>
            </td>
          <td>
            A. P. Charmouh, Peter Porsborg Sørud, T. Bataillon, A. Hobolth, Lasse Thorup Hansen, Søren Besenbacher, S. B. Winge, K. Almstrup, M. Schierup
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="The p53 family remains a captivating focus of an extensive number of current studies. Accumulating evidence indicates that p53 abnormalities rank among the most prevalent in cancer. Given the numerous existing studies, which mostly focus on the mutations, expression profiles, and functional perturbations exhibited by members of the p53 family across diverse malignancies, this review will concentrate more on less explored facets regarding p53 activation and stabilization by the nuclear pore complex (NPC) in cancer, drawing on several studies. p53 integrates a broad spectrum of signals and is subject to diverse regulatory mechanisms to enact the necessary cellular response. It is widely acknowledged that each stage of p53 regulation, from synthesis to degradation, significantly influences its functionality in executing specific tasks. Over recent decades, a large body of data has established that mechanisms of regulation, closely linked with protein activation and stabilization, involve intricate interactions with various cellular components. These often transcend canonical regulatory pathways. This new knowledge has expanded from the regulation of genes themselves to epigenomics and proteomics, whereby interaction partners increase in number and complexity compared with earlier paradigms. Specifically, studies have recently shown the involvement of the NPC protein in such complex interactions, underscoring the further complexity of p53 regulation. Furthermore, we also discuss therapeutic strategies based on recent developments in this field in combination with established targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f4ce5cb3bd6bcbf7187f9f514ca001d925085c" target='_blank'>
              Unlocking the Gateway: The Spatio-Temporal Dynamics of the p53 Family Driven by the Nuclear Pores and Its Implication for the Therapeutic Approach in Cancer
              </a>
            </td>
          <td>
            Dini Kurnia Ikliptikawati, Keiichi Makiyama, M. Hazawa, Richard W. Wong
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Targeting CDC20 can enhance the radiosensitivity of tumor cells, but the function and mechanism of CDC20 on DNA damage repair response remains vague. To examine that issue, tumor cell lines, including KYSE200, KYSE450, and HCT116, were utilized to detect the expression, function, and underlying mechanism of CDC20 in radio-chemoresistance. Western blot and immunofluorescence staining were employed to confirm CDC20 expression and location, and radiation could upregulate the expression of CDC20 in the cell nucleus. The homologous recombination (HR) and non-homologous end joining (NHEJ) reporter gene systems were utilized to explore the impact of CDC20 on DNA damage repair, indicating that CDC20 could promote HR repair and radio/chemo-resistance. In the early stages of DNA damage, CDC20 stabilizes the RPA1 protein through protein-protein interactions, activating the ATR-mediated signaling cascade, thereby aiding in genomic repair. In the later stages, CDC20 assists in the subsequent steps of damage repair by the ubiquitin-mediated degradation of RPA1. CCK-8 and colony formation assay were used to detect the function of CDC20 in cell vitality and proliferation, and targeting CDC20 can exacerbate the increase in DNA damage levels caused by cisplatin or etoposide. A tumor xenograft model was conducted in BALB/c-nu/nu mice to confirm the function of CDC20 in vivo, confirming the in vitro results. In conclusion, this study provides further validation of the potential clinical significance of CDC20 as a strategy to overcome radio-chemoresistance via uncovering a novel role of CDC20 in regulating RPA1 during DNA damage repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ebddd7f8266727872b7593baec6eddbe1783f9" target='_blank'>
              CDC20 Holds Novel Regulation Mechanism in RPA1 during Different Stages of DNA Damage to Induce Radio-Chemoresistance
              </a>
            </td>
          <td>
            Yang Gao, Pengbo Wen, Chenran Shao, Cheng Ye, Yuji Chen, Junyu You, Zhongjing Su
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Aneuploidy, an imbalanced number of chromosomes or chromosome arms, is a genetic hallmark of cancer cells, yet aneuploidy remains a biological enigma and a missed opportunity for cancer therapy. Aneuploid cells must cope with several types of cellular stresses, potentially creating synthetic lethalities that can be used to target aneuploid cancer cells. Here, I will describe our efforts to identify synthetic lethalities of the aneuploid state (in contrast to synthetic lethalities of specific recurrent aneuploidies. Specifically, I will focus on three recent unpublished studies, where we report that: (1) Aneuploid cells are preferentially sensitive to perturbation of the spindle assembly checkpoint (SAC) and its regulator KIF18A; the expression and activity of CDC20 determine the sensitivity to SAC inhibition. (2) Aneuploid cells depend on MAPK signaling for overcoming aneuploidy-induced DNA damage; targeting MAPK signaling can sensitize aneuploid cells to DNA damage inducing agents and to PARP inhibitors. (3) Aneuploid cells depend on RNA and protein degradation mechanisms to attenuate the cellular consequence of extra chromosomes; this renders aneuploid cells more sensitive to inhibition of nonsense-mediate decay, miRNA-mediated gene silencing, and the proteasome complex.
 Citation Format: Uri Ben-David. Identifying and targeting synthetic lethalities of aneuploid (cancer) cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA015.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac71f1802eee0a6fdce6ada7502686abe3e34f7b" target='_blank'>
              Abstract IA015: Identifying and targeting synthetic lethalities of aneuploid (cancer) cells
              </a>
            </td>
          <td>
            Uri Ben-David
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Genomic instability is one of the hallmarks of cancer. While loss of histone demethylase KDM6A increases the risk of tumorigenesis, its specific role in maintaining genomic stability remains poorly understood. Here, we propose a mechanism in which KDM6A maintains genomic stability independently on its demethylase activity. This occurs through its interaction with SND1, resulting in the establishment of a protective chromatin state that prevents replication fork collapse by recruiting of RPA and Ku70 to nascent DNA strand. Notably, KDM6A–SND1 interaction is up-regulated by KDM6A SUMOylation, while KDM6AK90A mutation almost abolish the interaction. Loss of KDM6A or SND1 leads to increased enrichment of H3K9ac and H4K8ac but attenuates the enrichment of Ku70 and H3K4me3 at nascent DNA strand. This subsequently results in enhanced cellular sensitivity to genotoxins and genomic instability. Consistent with these findings, knockdown of KDM6A and SND1 in esophageal squamous cell carcinoma (ESCC) cells increases genotoxin sensitivity. Intriguingly, KDM6A H101D & P110S, N1156T and D1216N mutations identified in ESCC patients promote genotoxin resistance via increased SND1 association. Our finding provides novel insights into the pivotal role of KDM6A–SND1 in genomic stability and chemoresistance, implying that targeting KDM6A and/or its interaction with SND1 may be a promising strategy to overcome the chemoresistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63d46c928ae17fc244023fddd8a346e2605b19c8" target='_blank'>
              KDM6A–SND1 interaction maintains genomic stability by protecting the nascent DNA and contributes to cancer chemoresistance
              </a>
            </td>
          <td>
            Jian Wu, Yixin Jiang, Qin Zhang, Xiaobing Mao, Tong Wu, Mengqiu Hao, Su Zhang, Yang Meng, Xiaowen Wan, Lei Qiu, Junhong Han
          </td>
          <td>2024-06-08</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer develops from abnormal cell growth in the body, causing significant mortalities every year. To date, potent therapeutic approaches have been developed to eradicate tumor cells, but intolerable toxicity and drug resistance can occur in treated patients, limiting the efficiency of existing treatment strategies. Therefore, searching for novel genes critical for cancer progression and therapeutic response is urgently needed for successful cancer therapy. Recent advances in bioinformatics and proteomic techniques have allowed the identification of a novel category of peptides encoded by non-canonical open reading frames (ncORFs) from historically non-coding genomic regions. Surprisingly, many ncORFs express functional microproteins that play a vital role in human cancers. In this review, we provide a comprehensive description of different ncORF types with coding capacity and technological methods in discovering ncORFs among human genomes. We also summarize the carcinogenic role of ncORFs such as pTINCR and HOXB-AS3 in regulating hallmarks of cancer, as well as the roles of ncORFs such as HOXB-AS3 and CIP2A-BP in cancer diagnosis and prognosis. We also discuss how ncORFs such as AKT-174aa and DDUP are involved in anti-cancer drug response and the underestimated potential of ncORFs as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d95f26adc077c4e04bfda2c2e30184b8f21906d" target='_blank'>
              Exploring the Dark Matter of Human Proteome: The Emerging Role of Non-Canonical Open Reading Frame (ncORF) in Cancer Diagnosis, Biology, and Therapy
              </a>
            </td>
          <td>
            Anni Ge, Curtis Chan, Xiaolong Yang
          </td>
          <td>2024-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genome-level clonal decomposition of a single specimen has been widely studied; however, it is mostly limited to cancer research. In this study, we developed a new algorithm CLEMENT, which conducts accurate decomposition and reconstruction of multiple subclones in genome sequencing of non-tumor (normal) samples. CLEMENT employs the Expectation-Maximization (EM) algorithm with optimization strategies specific to non-tumor subclones, including false variant call identification, non-disparate clone fuzzy clustering, and clonal allele fraction confinement. In the simulation and in vitro cell line mixture data, CLEMENT outperformed current cancer decomposition algorithms in estimating the number of clones (root-mean-square-error = 0.58-0.78 versus 1.43-3.34) and in the variant-clone membership agreement (∼85.5% versus 70.1-76.7%). Additional testing on human multi-clonal normal tissue sequencing confirmed the accurate identification of subclones that originated from different cell types. Clone-level analysis, including mutational burden and signatures, provided a new understanding of normal-tissue composition. We expect that CLEMENT will serve as a crucial tool in the currently emerging field of non-tumor genome analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b10b8eaeda7c6b37ca2e0e9728849f940c0c059" target='_blank'>
              CLEMENT: genomic decomposition and reconstruction of non-tumor subclones.
              </a>
            </td>
          <td>
            Young-Soo Chung, Seungseok Kang, Jisu Kim, Sangbo Lee, Sangwoo Kim
          </td>
          <td>2024-06-26</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Proteasome is an essential organelle in guarding cellular protein homeostasis. Here, we report that inhibition of proteasome leads to alterations in DNA methylation patterns in colorectal cancer (CRC) by surpressing the synthesis of DNA methyltransferases (DNMTs). We found that treating CRC cells with proteasome inhibitors results in attenuated translation of DNMT1 and DNMT3B, mediated by the inactivation of AKT and mammalian target of rapamycin (mTOR), which is dependent on the accumulation of p300, an acetyltransferase that inhibits AKT through acetylation modification. Furthermore, we demonstrated that downregulation of DNMT1 and DNMT3B confers protection against proteasome inhibitor treatment, potentially through reprogramming the transcriptome of CRC cells, highlighting the significant role of DNMTs in response to disruptions in protein homeostasis. Interestingly, the proteasome inhibitor-induced downregulation of DNMT1 and DNMT3B appears to be CRC specific, notwithstanding the underlying mechanism remains unclear. Altogether, our findings reveal an epigenetic effect of proteasome on DNA methylation in CRC through its regulation of DNA methyltransferase synthesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69df09e8bd58ff22a33d502415e4e8aa2f90876c" target='_blank'>
              Proteasome inhibition induces DNA methylation alteration by attenuating the synthesis of DNA methyltransferase 1 and 3B in colorectal cancer
              </a>
            </td>
          <td>
            Wenwen Zhou, Yuling Sheng, Dingxue Hu, Yunyun An, Mengqi Yang, Wanqiu Wang, Shiva Basnet, Jingyu Yan, Shuxia Zhang, Qi Liu, Yunze Li, Yi Tan, Jing Gao, Kun Sun, Changzheng Du
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ee3ef93f75d18cca805d005d653d62f6f76b88e" target='_blank'>
              Comprehensive pan-cancer analysis reveals that C5orf34 regulates the proliferation and mortality of lung cancer.
              </a>
            </td>
          <td>
            Mengmeng Yang, Yuhan Deng, Yu Ma, Chunli Song, Zhenhua Wu, Xiayimaierdan Yibulayin, Xiaohong Sun, Yunquan Guo, Dan He
          </td>
          <td>2024-06-29</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Aneuploidy is nearly ubiquitous in tumor genomes, but the role of aneuploidy in the early stages of cancer evolution remains unclear. Here, by inducing heterogeneous aneuploidy in non-transformed human colon organoids (colonoids), we investigated how the effects of aneuploidy on cell growth and differentiation may promote malignant transformation. Previous work implicated p53 activation as a downstream response to aneuploidy induction. We found that simple aneuploidy, characterized by 1-3 gained or lost chromosomes, resulted in little or modest p53 activation and cell cycle arrest when compared with more complex aneuploid cells. Single-cell RNA sequencing analysis revealed that the degree of p53 activation was strongly correlated with karyotype complexity. Single-cell tracking showed that cells could continue to divide despite the observation of one to a few lagging chromosomes. Unexpectedly, colonoids with simple aneuploidy exhibited impaired differentiation after niche factor withdrawal. These findings demonstrate that simple aneuploid cells can escape p53 surveillance and may contribute to niche factor-independent growth of cancer-initiating colon stem cells. [Media: see text].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69141397c86397dd6c1f0121be2fa0ac82f1c38" target='_blank'>
              Simple aneuploidy evades p53 surveillance and promotes niche factor-independent growth in human intestinal organoids.
              </a>
            </td>
          <td>
            Blake A. Johnson, Albert Z Liu, Tianhao Bi, Yi Dong, Taibo Li, Dingjingyu Zhou, A. Narkar, Yufei Wu, Sean X. Sun, Tatianna C. Larman, Jin Zhu, Rong Li
          </td>
          <td>2024-07-10</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most malignant primary brain tumor in adults, has poor prognosis irrespective of therapeutic advances due to its radio-resistance and infiltrative growth into brain tissue. The present study assessed functions and putative druggability of protein breast cancer type 1 susceptibility protein (BRCA1)-associated Ataxia telangiectasia mutated (ATM)-activator 1 (BRAT1) as a crucial factor driving key aspects of GBM, including enhanced DNA damage response and tumor migration. By a stable depletion of BRAT1 in GBM and glioma stem-like (GSC) cell lines, we observed a delay in DNA double-strand break repair and increased sensitivity to radiation treatment, corroborated by in vitro and in vivo studies demonstrating impaired tumor growth and invasion. Proteomic analyses further emphasize the role of BRAT1’s cell migration and invasion capacity, with a notable proportion of downregulated proteins associated with these processes. In line with the genetic manipulation, we found that treatment with the BRAT1 inhibitor Curcusone D (CurD) significantly reduced GSC migration and invasion in an ex vivo slice culture model, particularly when combined with irradiation, resulting a synergistic inhibition of tumor growth and infiltration. Our results reveal that BRAT1 contributes to GBM growth and invasion and suggest that therapeutic inhibition of BRAT1 with CurD or similar compounds might constitute a novel approach for anti-GBM directed treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6e398faf7890164e1dab60592356505fb1252c" target='_blank'>
              BRAT1 - a new therapeutic target for glioblastoma
              </a>
            </td>
          <td>
            Alicia Haydo, Jennifer Schmidt, Alisha Crider, Tim Kögler, Johanna Ertl, S. Hehlgans, Marina E. Hoffmann, Rajeshwari Rathore, Ömer Güllülü, Yecheng Wang, Xiangke Zhang, Christel Herold-Mende, Francesco Pampaloni, Irmgard Tegeder, Ivan Dikic, Mingji Dai, Franz Rödel, D. Kögel, B. Linder
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc721cd925975049ceb0cd27f9ffa4efdce8611" target='_blank'>
              Synthetic lethality by PARP inhibitors: new mechanism uncovered based on unresolved transcription-replication conflicts
              </a>
            </td>
          <td>
            Marina Kolesnichenko, Claus Scheidereit
          </td>
          <td>2024-07-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="YAP is a central player in cancer development with functions extending beyond its recognized role in cell growth regulation. Recent work has identified a link between YAP/TAZ and the DNA damage response. Here, we investigated the mechanistic underpinnings of the crosstalk between DNA damage repair and YAP activity. Ku70, a key component of the non-homologous end joining pathway to repair DNA damage, engaged in a dynamic competition with TEAD4 for binding to YAP, limiting the transcriptional activity of YAP. Depletion of Ku70 enhanced interaction between YAP and TEAD4 and boosted YAP transcriptional capacity. Consequently, Ku70 loss enhanced tumorigenesis in colon cancer and hepatocellular carcinoma (HCC) in vivo. YAP impeded DNA damage repair and elevated genome instability by inducing PARP1 degradation through the SMURF2-mediated ubiquitin-proteasome pathway. Analysis of HCC patient samples substantiated the link between Ku70 expression, YAP activity, PARP1 levels, and genome instability. In conclusion, this research provides insight into the mechanistic interactions between YAP and key regulators of DNA damage repair, highlighting the role of a Ku70-YAP-PARP1 axis in preserving genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d02fff425b6184aa5f8abb97e7f6cb2ad73ee59a" target='_blank'>
              Ku70 Binding to YAP Alters PARP1 Ubiquitination to Regulate Genome Stability and Tumorigenesis.
              </a>
            </td>
          <td>
            Yinyin Shu, Xiaoni Jin, Mintao Ji, Zhisen Zhang, Xiuxiu Wang, Haisheng Liang, Shuangshuang Lu, Shuai Dong, Yiping Lin, Yuhan Guo, Qiuyu Zhuang, Yuhong Wang, Zhe Lei, Lingchuan Guo, Xuanyu Meng, Guangming Zhou, Wensheng Zhang, Lei Chang
          </td>
          <td>2024-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb4b36c7afe762fcfe4d2e605f3f7060a7a118b" target='_blank'>
              ATM inhibition exploits checkpoint defects and ATM-dependent double strand break repair in TP53-mutant glioblastoma
              </a>
            </td>
          <td>
            D. Laverty, Shiv K Gupta, Gary A. Bradshaw, Alexander S Hunter, B. Carlson, Nery Matias Calmo, Jiajia Chen, Shulan Tian, J. Sarkaria, Zachary D. Nagel
          </td>
          <td>2024-06-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="There remains ongoing debate regarding the association of homologous recombination deficiency (HRD) with patient survival across various malignancies, highlighting the need for a comprehensive understanding of HRD's role in different cancer types. Based on data from databases, we conducted a multivariable omics analysis on HRD in 33 cancer types, focusing mainly on 23 cancers in which HRD was significantly associated with patient overall survival (OS) rates. This analysis included the mechanisms related to patient prognosis, gene expression, gene mutation, and signaling pathways. In this study, HRD was found to be significantly associated with patient prognosis, but its impact varied among different cancers. HRD was linked to different outcomes for patients with distinct tumor subtypes and was correlated with clinical features such as clinical stage and tumor grade. Driver gene mutations, including TP53, MUC4, KRAS, HRAS, FLG, ANK3, BRCA2, ATRX, FGFR3, NFE2L2, MAP3K1, PIK3CA, CIC, FUBP1, ALB, CTNNB1, and MED12, were associated with HRD across specific cancer types. We also analyzed differentially expressed genes (DEGs) and differentially methylated regions (DMRs) in relation to HRD levels in these cancers. Furthermore, we explored the correlation between HRD and signaling pathways, as well as immune cell infiltration. Overall, our findings contribute to a comprehensive understanding of HRD's multifaceted role in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b43dc4d2b27993dbe6ec2fffe344eca1d0221e85" target='_blank'>
              Multiomics analysis of homologous recombination deficiency across cancer types.
              </a>
            </td>
          <td>
            Lin Dong, Lin Li, Linyan Zhu, Fei Xu, Rumeng Zhang, Qiushuang Li, Yong Zhu, Zhutian Zeng, Keshuo Ding
          </td>
          <td>2024-07-28</td>
          <td>Biomolecules & biomedicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3ef3973d270e32f48aaa19a00bef4e18bca5be" target='_blank'>
              Research Progress of Long Non-coding RNA-ZFAS1 in Malignant Tumors.
              </a>
            </td>
          <td>
            Xin Liu, Zhong Ma, Xianxu Zhang, Shicheng Li, Jiangdong An, Zhiqiang Luo
          </td>
          <td>2024-07-26</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="CCCTC-binding factor (CTCF) is an insulator protein that binds to a highly conserved DNA motif and facilitates regulation of three-dimensional (3D) nuclear architecture and transcription. CTCF binding sites (CTCF-BSs) reside in non-coding DNA and are frequently mutated in cancer. Our previous study identified a small subclass of CTCF-BSs that are resistant to CTCF knock down, termed persistent CTCF binding sites (P-CTCF-BSs). P-CTCF-BSs show high binding conservation and potentially regulate cell-type constitutive 3D chromatin architecture. Here, using ICGC sequencing data we made the striking observation that P-CTCF-BSs display a highly elevated mutation rate in breast and prostate cancer when compared to all CTCF-BSs. To address whether P-CTCF-BS mutations are also enriched in other cell-types, we developed CTCF-INSITE-a tool utilising machine learning to predict persistence based on genetic and epigenetic features of experimentally-determined P-CTCF-BSs. Notably, predicted P-CTCF-BSs also show a significantly elevated mutational burden in all 12 cancer-types tested. Enrichment was even stronger for P-CTCF-BS mutations with predicted functional impact to CTCF binding and chromatin looping. Using in vitro binding assays we validated that P-CTCF-BS cancer mutations, predicted to be disruptive, indeed reduced CTCF binding. Together this study reveals a new subclass of cancer specific CTCF-BS DNA mutations and provides insights into their importance in genome organization in a pan-cancer setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994d6f59097fc218409b42c901b1ae72776752f9" target='_blank'>
              Machine learning enables pan-cancer identification of mutational hotspots at persistent CTCF binding sites.
              </a>
            </td>
          <td>
            Wenhan Chen, Y. Zeng, Joanna Achinger-Kawecka, Elyssa Campbell, Alicia K Jones, Alastair G Stewart, Amanda Khoury, Susan J Clark
          </td>
          <td>2024-07-02</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe961497d985de77bb27297cd1364379e55768f" target='_blank'>
              Circular RNA in cancer.
              </a>
            </td>
          <td>
            Vanessa M. Conn, A. Chinnaiyan, Simon J. Conn
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>163</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/529c303e4a8627e247a69d8654c825f73c988e72" target='_blank'>
              Integrative analysis of ultra-deep RNA-seq reveals alternative promoter usage as a mechanism of activating oncogenic programmes during prostate cancer progression.
              </a>
            </td>
          <td>
            Meng Zhang, M. Sjöström, Xiekui Cui, A. Foye, K. Farh, Raunak Shrestha, A. Lundberg, H. Dang, Haolong Li, Phillip G Febbo, Rahul Aggarwal, J. Alumkal, Eric J Small, C. Maher, Felix Y Feng, David A Quigley
          </td>
          <td>2024-06-13</td>
          <td>Nature cell biology</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245be040147fd6375f5eb20fea5e8094910fd2c5" target='_blank'>
              DNA Damage and Chromatin Rearrangement Work Together to Promote Neurodegeneration.
              </a>
            </td>
          <td>
            Harman Sharma, Sushma Koirala, Yee Lian Chew, Anna Konopka
          </td>
          <td>2024-07-08</td>
          <td>Molecular neurobiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The Ten-Eleven Translocation (TET) family genes are implicated in a wide array of biological functions across various human cancers. Nonetheless, there is a scarcity of studies that comprehensively analyze the correlation between TET family members and the molecular phenotypes and clinical characteristics of different cancers. Leveraging updated public databases and employing several bioinformatics analysis methods, we assessed the expression levels, somatic variations, methylation levels, and prognostic values of TET family genes. Additionally, we explored the association between the expression of TET family genes and pathway activity, tumor microenvironment (TME), stemness score, immune subtype, clinical staging, and drug sensitivity in pan-cancer. Molecular biology and cytology experiments were conducted to validate the potential role of TET3 in tumor progression. Each TET family gene displayed distinct expression patterns across at least ten detected tumors. The frequency of Single Nucleotide Variant (SNV) in TET genes was found to be 91.24%, primarily comprising missense mutation types, with the main types of copy number variant (CNV) being heterozygous amplifications and deletions. TET1 gene exhibited high methylation levels, whereas TET2 and TET3 genes displayed hypomethylation in most cancers, which correlated closely with patient prognosis. Pathway activity analysis revealed the involvement of TET family genes in multiple signaling pathways, including cell cycle, apoptosis, DNA damage response, hormone AR, PI3K/AKT, and RTK. Furthermore, the expression levels of TET family genes were shown to impact the clinical staging of tumor patients, modulate the sensitivity of chemotherapy drugs, and thereby influence patient prognosis by participating in the regulation of the tumor microenvironment, cellular stemness potential, and immune subtype. Notably, TET3 was identified to promote cancer progression across various tumors, and its silencing was found to inhibit tumor malignancy and enhance chemotherapy sensitivity. These findings shed light on the role of TET family genes in cancer progression and offer insights for further research on TET3 as a potential therapeutic target for pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4df11d8b0544aeb2264d4da3fba3e8f78e1111" target='_blank'>
              Pan-cancer analyses reveal the molecular and clinical characteristics of TET family members and suggests that TET3 maybe a potential therapeutic target
              </a>
            </td>
          <td>
            Chunyan Zhang, Jie Zheng, Jin Liu, Yanxia Li, G. Xing, Shupeng Zhang, Hekai Chen, Jian Wang, Z. Shao, Yongyuan Li, Zhongmin Jiang, Yingzi Pan, Xiaozhi Liu, Ping Xu, Wenhan Wu
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The fitness effects of new mutations determine key properties of evolutionary processes. Beneficial mutations drive evolution, yet selection is also shaped by the frequency of small-effect deleterious mutations, whose combined effect can burden otherwise adaptive lineages and alter evolutionary trajectories and outcomes in clonally evolving organisms such as viruses, microbes, and tumors. The small effect sizes of these important mutations have made accurate measurements of their rates difficult. In microbes, assessing the effect of mutations on growth can be especially instructive, as this complex phenotype is closely linked to fitness in clonally evolving organisms. Here, we perform high-throughput time-lapse microscopy on cells from mutation-accumulation strains to precisely infer the distribution of mutational effects on growth rate in the budding yeast, Saccharomyces cerevisiae. We show that mutational effects on growth rate are overwhelmingly negative, highly skewed towards very small effect sizes, and frequent enough to suggest that deleterious hitchhikers may impose a significant burden on evolving lineages. By using lines that accumulated mutations in either wild-type or slippage repair-defective backgrounds, we further disentangle the effects of 2 common types of mutations, single-nucleotide substitutions and simple sequence repeat indels, and show that they have distinct effects on yeast growth rate. Although the average effect of a simple sequence repeat mutation is very small (approximately 0.3%), many do alter growth rate, implying that this class of frequent mutations has an important evolutionary impact.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e2d4d1c86097c9432759848b7286f55d338b931" target='_blank'>
              Spontaneous single-nucleotide substitutions and microsatellite mutations have distinct distributions of fitness effects
              </a>
            </td>
          <td>
            Y. Plavskin, M. S. de Biase, Naomi Ziv, Libuše Janská, Yuan O. Zhu, David W. Hall, Roland F Schwarz, Daniel Tranchina, M. Siegal
          </td>
          <td>2024-07-01</td>
          <td>PLOS Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daaaeb917f1640a6de2396b29c1a5a0f942bd758" target='_blank'>
              Enhanced cancer cell proliferation and aggressive phenotype counterbalance in breast cancer with high BRCA1 gene expression.
              </a>
            </td>
          <td>
            Kohei Chida, M. Oshi, A. M. Roy, Takumi Sato, Maya Penelope Takabe, Li Yan, I. Endo, Kenichi Hakamada, K. Takabe
          </td>
          <td>2024-07-07</td>
          <td>Breast cancer research and treatment</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112c0ed37a6197df232678cb308cfb7c6f524b6b" target='_blank'>
              Mechanistic analysis of enhancer sequences in the estrogen receptor transcriptional program
              </a>
            </td>
          <td>
            Shayan Tabe-Bordbar, Y. Song, Bryan J Lunt, Zahra Alavi, K. Prasanth, Saurabh Sinha
          </td>
          <td>2024-06-11</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/522828f236448b54c15f6a0cec58172ce3dd65a1" target='_blank'>
              Epigenetic age acceleration is a distinctive trait of epithelioid sarcoma with potential therapeutic implications.
              </a>
            </td>
          <td>
            Simon Haefliger, O. Chervova, Christopher Davies, Chet Loh, R. Tirabosco, F. Amary, N. Pillay, Steve Horvath, Stephan Beck, Adrienne M. Flanagan, I. Lyskjær
          </td>
          <td>2024-06-16</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e847d1939a2e0502415554f395bfc39a4ea0ac20" target='_blank'>
              Assessment of ID family proteins expression in colorectal cancer of Iraqi patients.
              </a>
            </td>
          <td>
            Saja Ali Abbas, I. H. Hamzah
          </td>
          <td>2024-07-13</td>
          <td>Molecular biology reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Lynch syndrome (LS) is defined by inherited mutations in DNA mismatch repair genes, including MSH2, and carries 60% lifetime risk of developing endometrial cancer (EC). Beyond hypermutability, specific mechanisms for LS-associated endometrial carcinogenesis are not well understood. Here, we assessed the effects of MSH2 loss on EC pathogenesis using a novel mouse model (PR-Cre Msh2flox/flox, abbreviated Msh2KO), primary cell lines established from this model, human tissues, and human EC cell lines with isogenic MSH2 knockdown. Beginning at eight months of age, 30% of Msh2KO mice exhibited endometrial atypical hyperplasia (AH), a precancerous lesion. At 12 to 16 months of age, 47% of Msh2KO mice exhibited either AH or ECs with histologic features similar to human LS-related ECs. Transcriptomic profiling of EC from Msh2KO mice revealed a transcriptomic signature for mitochondrial dysfunction. Studies in vitro and in vivo revealed mitochondrial dysfunction based upon two mechanisms: marked mitochondrial content reduction, along with pronounced disruptions to the integrity of retained mitochondria. Human LS-related ECs also exhibited mitochondrial content reduction compared with non-LS-related ECs. Functional studies revealed metabolic reprogramming of MSH2-deficient EC cells in vitro, including reduced oxidative phosphorylation and increased susceptibility to glycolysis suppression. We are the first to identify mitochondrial dysfunction and metabolic disruption as a consequence of MSH2 deficiency-related EC. Mitochondrial and metabolic aberrations should be evaluated as novel biomarkers for endometrial carcinogenesis or risk stratification and could serve as targets for cancer interception in women with LS. Significance This is the first study to report mitochondrial dysfunction contributing to MSH2-deficient endometrial cancer development, identifying a noncanonical pathway for MSH2 deficient carcinogenesis, which also imparts vulnerability to metabolic targeting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55ccc4896dd3a745a675c80a7b2244614b6d8e30" target='_blank'>
              Mitochondrial defects and metabolic vulnerabilities in Lynch syndrome-associated MSH2-deficient endometrial cancer
              </a>
            </td>
          <td>
            M. Bowen, Brenda Melendez, Qian Zhang, Diana Moreno, Leah Peralta, Wai Kin Chan, Collene R. Jeter, Lin Tan, M. Zal, Philip Lorenzi, Kenneth Dunner, Richard K Yang, Russell R. Broaddus, J. Celestino, N. Gokul, Elizabeth Whitley, R. Schmandt, Karen H. Lu, Hyun-Eui Kim, Melinda S Yates
          </td>
          <td>2024-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA G-quadruplex(G4) is a guanine-rich single-stranded DNA sequence that spontaneously folds into a spherical four-stranded DNA secondary structure in oncogene promoter sequences and telomeres. G4s are highly associated with the occurrence and development of cancer and have emerged as promising anticancer targets. Natural products have long been important sources of anticancer drug development. In recent years, significant progress has been made in the discovery of natural drugs targeting DNA G4s, with many DNA G4s have been confirmed as promising targets of natural products, including MYC-G4, KRAS-G4, PDGFR-β-G4, BCL-2-G4, VEGF-G4, and telomeric G4. This review summarizes the research progress in discovering natural small molecules that target DNA G4s and their binding mechanisms. It also discusses the opportunities of and challenges in developing drugs targeting DNA G4s. This review will serve as a valuable reference for the research on natural products, particularly in the development of novel antitumor medications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0615bcc8ba2b9aa59d7f6c8fe9659c90b7d6290" target='_blank'>
              [Research progress in targeting DNA G-quadruplexes using natural products].
              </a>
            </td>
          <td>
            Zhi-Yuan Chen, Rong-Shuang Cheng, Wubulikasimu Ayixia, Yu-Shuang Liu, Kai-Bo Wang
          </td>
          <td>2024-07-01</td>
          <td>Zhongguo Zhong yao za zhi = Zhongguo zhongyao zazhi = China journal of Chinese materia medica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0bff8174988ff460b1ed14cde5b076ac3673528" target='_blank'>
              Methods and applications of genome-wide profiling of DNA damage and rare mutations.
              </a>
            </td>
          <td>
            Gerd P Pfeifer, Seung-Gi Jin
          </td>
          <td>2024-06-25</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Formaldehyde (FA) is a recognized environmental and metabolic toxin implicated in cancer development and aging. Inherited mutations in the FA-detoxifying enzymes ADH5 and ALDH2 genes lead to FA overload in the severe multisystem AMeD syndrome. FA accumulation causes genome damage including DNA-protein-, inter- and intra-strand crosslinks and oxidative lesions. However, the influence of distinct DNA repair systems on organismal FA resistance remains elusive. We have here investigated the consequence of a range of DNA repair mutants in a model of endogenous FA overload generated by downregulating the orthologs of human ADH5 and ALDH2 in C. elegans. We have focused on the distinct components of nucleotide excision repair (NER) during developmental growth, reproduction and aging. Our results reveal three distinct modes of repair of FA-induced DNA damage: Transcription-coupled repair (TCR) operating NER-independently during developmental growth or through NER during adulthood, and, in concert with global-genome (GG-) NER, in the germline and early embryonic development. Additionally, we show that the Cockayne syndrome B (CSB) factor is involved in the resolution of FA-induced DNA-protein crosslinks, and that the antioxidant and FA quencher N-acetyl-l-cysteine (NAC) reverses the sensitivity of detoxification and DNA repair defects during development, suggesting a therapeutic intervention to revert FA-pathogenic consequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d6ccccfe7604bc347a8ca565bb85bce6101f465" target='_blank'>
              Distinct DNA repair mechanisms prevent formaldehyde toxicity during development, reproduction and aging.
              </a>
            </td>
          <td>
            Matthias Rieckher, Christian Gallrein, Natividad Alquezar-Artieda, Nour Bourached-Silva, P. L. Vaddavalli, Devin Mares, Maria Backhaus, Timon Blindauer, Ksenia Greger, Eva Wiesner, Lucas B. Pontel, B. Schumacher
          </td>
          <td>2024-06-19</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfce29169bac0d7cbc470f70a1b9115230e5e431" target='_blank'>
              Multi-omics analysis reveals the dynamic interplay between Vero host chromatin structure and function during vaccinia virus infection
              </a>
            </td>
          <td>
            Vrinda Venu, Cullen Roth, Samantha H. Adikari, Eric M Small, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-06-11</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="Despite recent advances in treatment, non-small cell lung cancer (NSCLC) continues to have a high mortality rate. Currently, NSCLC pathogenesis requires further investigation, and therapeutic drugs are still under development. Homologous recombination repair (HRR) repairs severe DNA double-strand breaks. Homologous recombination repair deficiency (HRD) occurs when HRR is impaired and causes irreparable double-strand DNA damage, leading to genomic instability and increasing the risk of cancer development. Poly(ADP-ribose) polymerase (PARP) inhibitors can effectively treat HRD-positive tumors. Extracellular heat shock protein 90α (eHSP90α) is highly expressed in hypoxic environments and inhibits apoptosis, thereby increasing cellular tolerance. Here, we investigated the relationship between eHSP90α and HRR in NSCLC. DNA damage models were established in NSCLC cell lines (A549 and H1299). The activation of DNA damage and HRR markers, apoptosis, proliferation, and migration were investigated. In vivo tumor models were established using BALB/c nude mice and A549 cells. We found that human recombinant HSP90α stimulation further activated HRR and reduced DNA damage extent; however, eHSP90α monoclonal antibody, 1G6-D7, effectively inhibited HRR. HRR inhibition and increased apoptosis were observed after LRP1 knockdown; this effect could not be reversed with hrHSP90α addition. The combined use of 1G6-D7 and olaparib caused significant apoptosis and HRR inhibition in vitro and demonstrated promising anti-tumor effects in vivo. Extracellular HSP90α may be involved in HRR in NSCLC through LRP1. The combined use of 1G6-D7 and PARP inhibitors may exert anti-tumor effects by inhibiting DNA repair and further inducing apoptosis of NSCLC cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f32884450f7741dd2fb64543fee8afa9793c35" target='_blank'>
              1G6-D7 Inhibits Homologous Recombination Repair by Targeting Extracellular HSP90α to Promote Apoptosis in Non-Small Cell Lung Cancer.
              </a>
            </td>
          <td>
            Jiangzhou Du, Jinming Zhang, Dongyu Liu, Lin Gao, Hua Liao, Lanhe Chu, Jie Lin, Wei Li, Xiaojing Meng, Fei Zou, Shaoxi Cai, Mengchen Zou, Hangming Dong
          </td>
          <td>2024-06-20</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSC) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosomes 20q or 1q, both known for conferring in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings show higher loads of both CNVs and SNVs at later passages which are due to the cumulative acquisition of mutations over a longer time in culture and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs follow the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid cancer-associated mutations. This suggests that the recurrent chromosomal abnormalities are the potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f552b60bd87e180c4d6a5dd522428893e4b81b" target='_blank'>
              De novo cancer mutations frequently associate with recurrent chromosomal abnormalities during long-term human pluripotent stem cell culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huygebaert, M. Regin, Chi Mai Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16eda9843eb0037407162ede7ade3db9a52371a7" target='_blank'>
              SNM1A is crucial for efficient repair of complex DNA breaks in human cells
              </a>
            </td>
          <td>
            L. P. Swift, B. C. Lagerholm, Lucy R. Henderson, Malitha Ratnaweera, H. T. Baddock, B. Sengerova, Sook Y. Lee, A. Cruz-Migoni, Dominic Waithe, Christian Renz, Helle D. Ulrich, Joseph A. Newman, Christopher J. Schofield, P. McHugh
          </td>
          <td>2024-06-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ca87f16129e49df59c97d9b9e83e846766aff915" target='_blank'>
              iPS cell generation-associated point mutations include many C > T substitutions via different cytosine modification mechanisms
              </a>
            </td>
          <td>
            R. Araki, T. Suga, Y. Hoki, Kaori Imadome, Misato Sunayama, Satoshi Kamimura, Mayumi Fujita, M. Abe
          </td>
          <td>2024-06-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Simple Summary Globally, colorectal cancer is the third and second most common cancer in men and women, respectively. Due to its late diagnosis, morbidity and mortality are on the rise, which legitimizes the search for new markers for early and precise detection of cancer development. The knowledge about the short, non-coding RNA molecule (microRNA) expression profile in tumor cancer cell lines and extracellular vesicles derived from them may help to establish microRNA as useful markers for monitoring tumor signatures in body fluids (i.e., blood). Here, we present a comprehensive analysis of microRNA profiles of four colorectal cancer cell lines, one normal colon epithelium cell line and extracellular vesicles derived from them. Data are supplemented with preliminary results acquired from a group of colorectal cancer patients. Abstract Globally, an increasing prevalence of colorectal cancer (CRC) prompts a need for the development of new methods for early tumor detection. MicroRNAs (also referred to as miRNAs) are short non-coding RNA molecules that play a pivotal role in the regulation of gene expression. MiRNAs are effectively transferred to extracellular vesicle (EVs) membrane sacs commonly released by cells. Our study aimed to examine the expression of miRNAs in four CRC cell lines and EVs derived from them (tumor EVs) in comparison to the normal colon epithelium cell line and its EVs. EVs were isolated by ultracentrifugation from the culture supernatant of SW480, SW620, SW1116, HCT116 and normal CCD841CoN cell lines and characterized according to the MISEV2023 guidelines. MiRNAs were analyzed by small RNA sequencing and validated by quantitative PCR. The performed analysis revealed 22 common miRNAs highly expressed in CRC cell lines and effectively transferred to tumor EVs, including miR-9-5p, miR-182-5p, miR-196b-5p, miR-200b-5p, miR-200c-3p, miR-425-5p and miR-429, which are associated with development, proliferation, invasion and migration of colorectal cancer cells, as well as in vesicle maturation and transport-associated pathways. In parallel, normal cells expressed miRNAs, such as miR-369 and miR-143, which play a role in proinflammatory response and tumor suppression. The analysis of selected miRNAs in plasma-derived EVs and tumor samples from CRC patients showed the similarity of miRNA expression profile between the patients’ samples and CRC cell lines. Moreover, miR-182-5p, miR-196-5p, miR-425-5p and miR-429 were detected in several EV samples isolated from patients’ plasma. Our results suggest that miR-182-5p, miR-196b-5p and miR-429 are differentially expressed between EVs from CRC patients and healthy donors, which might have clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa4ec85d9fe618710e470b0a5005d1432da45ac8" target='_blank'>
              Identification of miRNAs Present in Cell- and Plasma-Derived Extracellular Vesicles—Possible Biomarkers of Colorectal Cancer
              </a>
            </td>
          <td>
            Marzena Lenart, Izabela Siemińska, R. Szatanek, A. Mordel, Antoni M. Szczepanik, M. Rubinkiewicz, Maciej Siedlar, M. Baj-Krzyworzeka
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cancer is one of the main reasons for death, and it threatens human life and health. Both the environment and genes can lead to cancers. It dates back more than a million years; more importantly, tumor cells can not be detected until they grow to a large number. Currently, cancers are treated with surgical excision or non-surgical procedures. By studying the interaction between ncRNAs and PKM2, we aim to provide new targets for diagnosis, treatment, and prognosis for cancers. Read relevant articles and made a summary and classification. Non-coding RNAs (ncRNAs) are RNAs that do not code for proteins. They perform a function in transcription and translation and can be used as targets for cancer therapy. Pyruvate kinase M2 (PKM2) is a form of PKM, and it catalyzes the glycolysis of the final cellular processes to promote tumorigenesis. Not only that, but it also plays non-metabolic functions, including the expression of the gene, cell proliferation, cell migration, and tumor angiogenesis in cancer cells. The existing studies have found that microRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs) can promote or inhibit the aerobic glycolysis of cancer cells by affecting PKM2, which increases or decrease the risk of cancers and affect the progression of cancers. This review focuses on the mechanism of ncRNAs regulating PKM2 in cancers and summarizes the roles of some ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135ae3dc56b2a8f0f10f7d3ae82e64ec70f96983" target='_blank'>
              Emerging Roles of ncRNAs Regulating PKM2 in Cancer Progression.
              </a>
            </td>
          <td>
            Haimei Xie, Jia Yu, Zhiwen Ou, Xiaoyong Lei, Xiaoyan Yang
          </td>
          <td>2024-07-15</td>
          <td>Current pharmaceutical biotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135134a62bac76cb46dae2f04090c849f2c01172" target='_blank'>
              Targeting chromosomal instability in patients with cancer.
              </a>
            </td>
          <td>
            Duaa H Al-Rawi, Emanuele Lettera, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-11</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="Background: Neoantigen targeting therapies including personalized vaccines have shown promise in the treatment of cancers, particularly when used in combination with checkpoint blockade therapy. At least 100 clinical trials involving these therapies are underway globally. Accurate identification and prioritization of neoantigens is highly relevant to designing these trials, predicting treatment response, and understanding mechanisms of resistance. With the advent of massively parallel DNA and RNA sequencing technologies, it is now possible to computationally predict neoantigens based on patient-specific variant information. However, numerous factors must be considered when prioritizing neoantigens for use in personalized therapies. Complexities such as alternative transcript annotations, various binding, presentation and immunogenicity prediction algorithms, and variable peptide lengths/registers all potentially impact the neoantigen selection process. There has been a rapid development of computational tools that attempt to account for these complexities. While these tools generate numerous algorithmic predictions for neoantigen characterization, results from these pipelines are difficult to navigate and require extensive knowledge of the underlying tools for accurate interpretation. This often leads to over-simplification of pipeline outputs to make them tractable, for example limiting prediction to a single RNA isoform or only summarizing the top ranked of many possible peptide candidates. In addition to variant detection, gene expression and predicted peptide binding affinities, recent studies have also demonstrated the importance of mutation location, allele-specific anchor locations, and variation of T-cell response to long versus short peptides. Due to the intricate nature and number of salient neoantigen features, presenting all relevant information to facilitate candidate selection for downstream applications is a difficult challenge that current tools fail to address. Results: We have created pVACview, the first interactive tool designed to aid in the prioritization and selection of neoantigen candidates for personalized neoantigen therapies including cancer vaccines. pVACview has a user-friendly and intuitive interface where users can upload, explore, select and export their neoantigen candidates. The tool allows users to visualize candidates across three different levels, including variant, transcript and peptide information. Conclusions: pVACview will allow researchers to analyze and prioritize neoantigen candidates with greater efficiency and accuracy in basic and translational settings The application is available as part of the pVACtools pipeline at pvactools.org and as an online server at pvacview.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b119733dd552ce3ae0da7e9341790152b9ccd48f" target='_blank'>
              pVACview: an interactive visualization tool for efficient neoantigen prioritization and selection
              </a>
            </td>
          <td>
            Huiming Xia, M. Hoang, Evelyn Schmidt, Susanna Kiwala, Joshua F. McMichael, Zachary L. Skidmore, Bryan Fisk, Jonathan J. Song, J. Hundal, Thomas Mooney, Jason R. Walker, S. Peter Goedegebuure, Christopher A. Miller, W. Gillanders, O. Griffith, M. Griffith
          </td>
          <td>2024-06-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Over 50 hereditary degenerative disorders are caused by expansions of short tandem DNA repeats (STRs). (GAA)n repeat expansions are responsible for Friedreich’s ataxia as well as late-onset cerebellar ataxias (LOCAs). Thus, the mechanisms of (GAA)n repeat expansions attract broad scientific attention. To investigate the role of DNA nicks in this process, we utilized a CRISPR-Cas9 nickase system to introduce targeted nicks adjacent to the (GAA)n repeat tract. We found that DNA nicks 5’ of the (GAA)100 run led to a dramatic increase in both the rate and scale of its expansion in dividing cells. Strikingly, they also promoted large-scale expansions of carrier- and large normal-size (GAA)n repeats, recreating, for the first time in a model system, the expansion events that occur in human pedigrees. DNA nicks 3’ of the (GAA)100 repeat led to a smaller but significant increase in the expansion rate as well. Our genetic analysis implies that in dividing cells, conversion of nicks into double-strand breaks (DSBs) during DNA replication followed by DSB or fork repair leads to repeat expansions. Finally, we showed that 5’ strand nicks increase expansion frequency in non-dividing yeast cells, albeit to a lesser extent that in dividing cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54a99889ad55f19d37b3400e4a1cd5786bdfa79b" target='_blank'>
              DNA Nicks Drive Massive Expansions of (GAA)n Repeats
              </a>
            </td>
          <td>
            Liangzi Li, W. S. Scott, A. N. Khristich, Jillian F. Armenia, S. Mirkin
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0365ae5ee5cc750385ad2b016991d72ddc4452d" target='_blank'>
              Mitochondrial DNA mosaicism in normal human somatic cells.
              </a>
            </td>
          <td>
            J. An, Chang Hyun Nam, Ryul Kim, Y. Lee, Hyein Won, Seongyeol Park, Won Hee Lee, Hansol Park, Christopher J Yoon, Yohan An, Jie-Hyun Kim, Jong Kwan Jun, Jeong Mo Bae, Eui-Cheol Shin, Bun Kim, Yong Jun Cha, Hyun-Woo Kwon, Jihyeon Oh, Jee Yoon Park, Min Jung Kim, Young Seok Ju
          </td>
          <td>2024-07-22</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) are characterized by the presence of either a single enlarged nucleus or multiple nuclei and are closely associated with tumor progression and treatment resistance. These cells contribute significantly to cellular heterogeneity and can arise from various stressors, including radiation, chemotherapy, hypoxia, and environmental factors. The formation of PGCCs can occur through mechanisms such as endoreplication, cell fusion, cytokinesis failure, mitotic slippage, or cell cannibalism. Notably, PGCCs exhibit traits similar to cancer stem cells (CSCs) and generate highly invasive progeny through asymmetric division. The presence of PGCCs and their progeny is pivotal in conferring resistance to chemotherapy and radiation, as well as facilitating tumor recurrence and metastasis. This review provides a comprehensive analysis of the origins, potential formation mechanisms, stressors, unique characteristics, and regulatory pathways of PGCCs, alongside therapeutic strategies targeting these cells. The objective is to enhance the understanding of PGCC initiation and progression, offering novel insights into tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66163bd868b4436fb4d269ccebdc926754441914" target='_blank'>
              Polyploid giant cancer cells: origin, possible pathways of formation, characteristics, and mechanisms of regulation
              </a>
            </td>
          <td>
            Pan Liu, Lili Wang, Huiying Yu
          </td>
          <td>2024-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 KIF18A is a plus-end directed kinesin known to play a role in mitosis by facilitating chromosome alignment and spindle microtubule dynamics. Knockdown or knockout of KIF18A has been shown to inhibit proliferation in cells with whole genome doubling or aneuploidy, cellular states characterized by chromosomal rearrangements. These alterations render cells particularly vulnerable to disrupted mitosis upon KIF18A loss. As such, KIF18A is a compelling synthetic lethality target in a subset of tumors with chromosomal instability (CIN), which have ongoing chromosomal segregation defects. We have identified potent and selective small molecule inhibitors of KIF18A. A proprietary tool compound, ATX020, inhibits KIF18A ATPase activity with an IC50 of 0.014 µM; KIF18A is selectively inhibited over other kinesins including CENPE (IC50 > 10 µM) and EG5 (IC50 5.87 µM). ATX020 exhibits robust cellular activity, with anti-proliferative activity observed in chromosomally instable ovarian cancer cell lines such as OVCAR-3 (IC50 0.053 µM) and OVCAR-8 (IC50 0.54 µM). Upon treatment with ATX020, Annexin V is robustly induced in sensitive cell lines, demonstrating that KIF18A-dependent cell lines undergo apoptosis upon KIF18A inhibition. In KIF18Ai sensitive cells, a dose-dependent upregulation of phospho-Histone H3 and gamma-H2AX is observed upon ATX020 treatment in OVCAR-3 and OVCAR-8 cells, consistent with the induction of mitotic arrest and DNA damage. In these sensitive cell lines, ATX020 induces a dose-dependent cell cycle arrest in G2/M, consistent with the role of KIF18A in facilitating mitosis. OVCAR-3 cells treated with ATX020 exhibit fragmented nuclei and malformed mitotic spindles. In contrast, CIN negative Ovarian cancer cells that are insensitive to KIF18A loss, such as A2780 and OVK18, proliferate normally upon ATX020 treatment, cell division is unaffected, and no induction of Annexin V, phospho-Histone H3, and gamma-H2AX is observed. This selective dependency is also observed in vivo, where administration of ATX020 leads to dose-dependent tumor growth inhibition in an OVCAR-3 cancer cell line-derived xenograft model, with significant tumor regression observed at 100 mpk/day. In contrast, OVK18 tumors are unaffected by ATX020 administration, as expected based on the lack of cell cycle and proliferation effects in that cell line. Together, these data demonstrate a critical role for KIF18A in facilitating mitosis in a subset of chromosomally instable cells. These effects are selective, with KIF18A-independent cell lines proceeding through the cell cycle normally in the presence of inhibitors. Small molecule inhibition of KIF18A is therefore expected to have robust efficacy in solid tumors with high levels of chromosomal instability such as Ovarian cancer.
 Citation Format: Maureen S. Lynes, Laura Ghisolfi, Sanjoy Khan, Taylor Hotz, Brian A. Sparling, Mary-Margaret Zablocki, Deepali Gotur, P. Ann Boriack-Sjodin, Kenneth W. Duncan, Stephen J. Blakemore, Serena J. Silver, Robert A. Copeland. KIF18A inhibition, via ATX020, leads to mitotic arrest and robust anti-tumor activity through a synthetic lethal interaction with chromosome instability [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR001.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c93d87266fb08c0f53f51496b94330637b88929" target='_blank'>
              Abstract PR001: KIF18A inhibition, via ATX020, leads to mitotic arrest and robust anti-tumor activity through a synthetic lethal interaction with chromosome instability
              </a>
            </td>
          <td>
            Maureen S. Lynes*, Laura Ghisolfi†, Sanjoy Khan, Taylor Hotz, Brian A. Sparling, M. Zablocki, D. Gotur, P. Boriack-Sjodin, K. Duncan, Stephen J. Blakemore, Serena J. Silver, Robert A. Copeland
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Breast cancer (BRCA) remains a significant global health challenge due to its prevalence and lethality, exacerbated by the development of resistance to conventional therapies. Therefore, understanding the molecular mechanisms underpinning chemoresistance is crucial for improving therapeutic outcomes. Human deoxycytidine triphosphate pyrophosphatase 1 (DCTPP1) has emerged as a key player in various cancers, including BRCA. DCTPP1, involved in nucleotide metabolism and maintenance of genomic stability, has been linked to cancer cell proliferation, survival, and drug resistance. This study evaluates the role of DCTPP1 in BRCA prognosis and chemotherapy response. Data from the Cancer Genome Atlas Program (TCGA), Genotype-Tissue Expression (GTEx), and Gene Expression Omnibus (GEO) repositories, analyzed using GEPIA and Kaplan–Meier Plotter, indicate that high DCTPP1 expression correlates with poorer overall survival and increased resistance to chemotherapy in BRCA patients. Further analysis reveals that DCTPP1 gene expression is up-regulated in non-responders to chemotherapy, particularly in estrogen receptor (ER)-positive, luminal A subtype patients, with significant predictive power. Additionally, in vitro studies show that DCTPP1 gene expression increases in response to 5-fluorouracil and doxorubicin treatments in luminal A BRCA cell lines, suggesting a hypothetical role in chemoresistance. These findings highlight DCTPP1 as a potential biomarker for predicting chemotherapy response and as a therapeutic target to enhance chemotherapy efficacy in BRCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34c5b36472a08309e9aaa6675581062a8fba187a" target='_blank'>
              DCTPP1 Expression as a Predictor of Chemotherapy Response in Luminal A Breast Cancer Patients
              </a>
            </td>
          <td>
            , Diego Soto-Jiménez, Gloria M. Calaf
          </td>
          <td>2024-08-02</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d747e31bfe10b04fb9f5ac3a09c12a47144e4ffb" target='_blank'>
              guidedNOMe-seq quantifies chromatin states at single allele resolution for hundreds of custom regions in parallel
              </a>
            </td>
          <td>
            Michaela Schwaiger, Fabio Mohn, Marc Bühler, Lucas J. T. Kaaij
          </td>
          <td>2024-07-29</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Transient halting of transcription activity on the damaged chromatin facilitates DNA double-strand break (DSB) repair. However, the molecular mechanisms that facilitate transcription recovery following DSB repair remain largely undefined. Notably, failure to restore gene expression in a timely manner can compromise transcriptome signatures and may impose deleterious impacts on cell identity and cell fate. Here, we report PHF8 as the major demethylase that reverses transcriptionally repressive epigenetic modification laid down by the DYRK1B-EHMT2 pathway. We found that PHF8 concentrates at laser-induced DNA damage tracks in a DYRK1B-dependent manner and promotes timely resolution of local H3K9me2 to facilitate the resumption of transcription. Moreover, PHF8 also assists in the recovery of ribosomal DNA (rDNA) transcription following the repair of nucleolar DSBs. Taken together, our findings uncover PHF8 as a key mediator that coordinates transcription activities during the recovery phase of DSB responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2eb5896d23703a43483dafa3f0c85e759dc8f72" target='_blank'>
              PHF8 facilitates transcription recovery following DNA double-strand break repair.
              </a>
            </td>
          <td>
            Jung Eun Kim, Xiyue Pan, Kwan Yiu Tse, Henry Hei Chan, Chao Dong, M. Huen
          </td>
          <td>2024-08-01</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Maintenance of DNA integrity is essential to all forms of life. DNA damage generated by reaction with genotoxic chemicals results in deleterious mutations, genome instability, and cell death. Pathogenic bacteria encounter several genotoxic agents during infection. In keeping with this, the loss of DNA repair networks results in virulence attenuation in several bacterial species. Interstrand DNA crosslinks (ICLs) are a type of DNA lesion formed by covalent linkage of opposing DNA strands and are particularly toxic as they interfere with replication and transcription. Bacteria have evolved specialized DNA glycosylases that unhook ICLs, thereby initiating their repair. In this study, we describe AlkX, a DNA glycosylase encoded by the multidrug resistant pathogen Acinetobacter baumannii. AlkX exhibits ICL unhooking activity similar to that of its Escherichia coli homolog YcaQ. Interrogation of the in vivo role of AlkX revealed that its loss sensitizes cells to DNA crosslinking and impairs A. baumannii colonization of the lungs and dissemination to distal tissues during pneumonia. These results suggest that AlkX participates in A. baumannii pathogenesis and protects the bacterium from stress conditions encountered in vivo. Consistent with this, we found that acidic pH, an environment encountered during host colonization, results in A. baumannii DNA damage and that alkX is induced by, and contributes to, defense against acidic conditions. Collectively, these studies reveal functions for a recently described class of proteins encoded in a broad range of pathogenic bacterial species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e8859ed92eefb483b7d273e5025bd69777d468" target='_blank'>
              An interstrand DNA crosslink glycosylase aids Acinetobacter baumannii pathogenesis.
              </a>
            </td>
          <td>
            D. E. Kunkle, Yujuan Cai, B. Eichman, Eric P Skaar
          </td>
          <td>2024-06-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="ABSTRACT Natural killer (NK) cells play a crucial role in antitumoral and antiviral responses. Yet, cancer cells can alter themselves or the microenvironment through the secretion of cytokines or other factors, hindering NK cell activation and promoting a less cytotoxic phenotype. These resistance mechanisms, often referred to as the “hallmarks of cancer” are significantly influenced by the activation of oncogenes, impacting most, if not all, of the described hallmarks. Along with oncogenes, other types of genes, the tumor suppressor genes are frequently mutated or modified during cancer. Traditionally, these genes have been associated with uncontrollable tumor growth and apoptosis resistance. Recent evidence suggests oncogenic mutations extend beyond modulating cell death/proliferation programs, influencing cancer immunosurveillance. While T cells have been more studied, the results obtained highlight NK cells as emerging key protagonists for enhancing tumor cell elimination by modulating oncogenic activity. A few recent studies highlight the crucial role of oncogenic mutations in NK cell-mediated cancer recognition, impacting angiogenesis, stress ligands, and signaling balance within the tumor microenvironment. This review will critically examine recent discoveries correlating oncogenic mutations to NK cell-mediated cancer immunosurveillance, a relatively underexplored area, particularly in the era dominated by immune checkpoint inhibitors and CAR-T cells. Building on these insights, we will explore opportunities to improve NK cell-based immunotherapies, which are increasingly recognized as promising alternatives for treating low-antigenic tumors, offering significant advantages in terms of safety and manufacturing suitability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfefe1207e6a05f83b08c29611ae01cb2a8d9e94" target='_blank'>
              In-depth analysis of the interplay between oncogenic mutations and NK cell-mediated cancer surveillance in solid tumors
              </a>
            </td>
          <td>
            C. Pesini, Laura Artal, Jorge Paúl Bernal, Diego Sánchez Martinez, Julián Pardo, A. Ramírez-Labrada
          </td>
          <td>2024-07-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Studies on somatic mutations in cloned animals have revealed slight genetic variances between clones and their originals, but have yet to identify the precise effects of these differences within the organism. Somatic mutations contribute to aging and are implicated in tumor development and other age-related diseases. Thus, we compared whole genome sequencing data from an original dog with that of cloned dogs, identifying candidate somatic mutations that were disproportionately located within genes previously implicated in aging. The substitutional signature of cloning-specific somatic mutations mirrored the uniform distribution characteristic of the signature associated with human aging. Further analysis of genes revealed significant enrichment of traits associated with body size as well as the molecular mechanisms underlying neuronal function and tumorigenesis. Overall, the somatic mutations found in cloned dogs may indicate a conserved mechanism driving aging across species and a broad spectrum of pathway alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f3d14c854be2c4eda75c0ca77517be9684eaeca" target='_blank'>
              Genome-Wide Characterization of Somatic Mutation Patterns in Cloned Dogs Reveals Implications for Neuronal Function, Tumorigenesis, and Aging
              </a>
            </td>
          <td>
            Seung-Wan Woo, Miju Kim, Dayeon Kang, Y. Choe, Seong-Ju Oh, Are-Sun You, S. Lee, Jaemin Kim
          </td>
          <td>2024-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81b50a6de235fd2aad83b4989e6445d7b4c3215a" target='_blank'>
              Cancer-associated Histone H3 N-terminal arginine mutations disrupt PRC2 activity and impair differentiation
              </a>
            </td>
          <td>
            B. A. Nacev, Yakshi Dabas, M. Paul, Christian Pacheco, Michelle Mitchener, Yekaterina Perez, Yan Fang, A. Soshnev, D. Barrows, Thomas S. Carroll, Nicholas D. Socci, Samantha C. St. Jean, Sagarika Tiwari, Michael J. Gruss, S. Monette, W. Tap, Benjamin A Garcia, T. Muir, C. D. Allis
          </td>
          <td>2024-06-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="BACKGROUND
Understanding the impact of clonal hematopoiesis of indeterminate potential (CHIP) and mosaic chromosomal alterations (mCAs) on solid tumor risk and mortality can shed light on novel cancer pathways.


METHODS
The authors analyzed whole genome sequencing data from the Trans-Omics for Precision Medicine Women's Health Initiative study (n = 10,866). They investigated the presence of CHIP and mCA and their association with the development and mortality of breast, lung, and colorectal cancers.


RESULTS
CHIP was associated with higher risk of breast (hazard ratio [HR], 1.30; 95% confidence interval [CI], 1.03-1.64; p = .02) but not colorectal (p = .77) or lung cancer (p = .32). CHIP carriers who developed colorectal cancer also had a greater risk for advanced-stage (p = .01), but this was not seen in breast or lung cancer. CHIP was associated with increased colorectal cancer mortality both with (HR, 3.99; 95% CI, 2.41-6.62; p < .001) and without adjustment (HR, 2.50; 95% CI, 1.32-4.72; p = .004) for advanced-stage and a borderline higher breast cancer mortality (HR, 1.53; 95% CI, 0.98-2.41; p = .06). Conversely, mCA (cell fraction [CF] >3%) did not correlate with cancer risk. With higher CFs (mCA >5%), autosomal mCA was associated with increased breast cancer risk (HR, 1.39; 95% CI, 1.06-1.83; p = .01). There was no association of mCA (>3%) with breast, colorectal, or lung mortality except higher colon cancer mortality (HR, 2.19; 95% CI, 1.11-4.3; p = .02) with mCA >5%.


CONCLUSIONS
CHIP and mCA (CF >5%) were associated with higher breast cancer risk and colorectal cancer mortality individually. These data could inform on novel pathways that impact cancer risk and lead to better risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa8e5d699d9cb31c3abe6a09df01fb728c1626c" target='_blank'>
              Association of clonal hematopoiesis and mosaic chromosomal alterations with solid malignancy incidence and mortality.
              </a>
            </td>
          <td>
            Pinkal M. Desai, Ying Zhou, Justin Grenet, Samuel K Handelman, Cynthia Crispino, Laura N Tarbay, E. Whitsel, Gail Roboz, Ana Barac, Michael Honigberg, A. Bick, Garnet Anderson, Jean Wactawski-Wende, Yasminka A Jakubek Swartzlander, Jason Bacon, Justin Wong, Xiaolong Ma, P. Scheet, Zichan Li, P. Kasi, Ross Prentice, Paul L Auer, J. Manson, Alexander Reiner, Michael S. Simon
          </td>
          <td>2024-07-16</td>
          <td>Cancer</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/116e8815e3f7217557ce7a5fabd86cab6afc311c" target='_blank'>
              CAraCAl: CAMML with the integration of chromatin accessibility
              </a>
            </td>
          <td>
            Courtney Schiebout, H. R. Frost
          </td>
          <td>2024-06-13</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23ed7881dc424017c9838fade2801a5a4db0c319" target='_blank'>
              Bidirectional linkage of DNA barcodes for the multiplexed mapping of higher-order protein interactions in cells.
              </a>
            </td>
          <td>
            Yu Liu, Noah R. Sundah, Nicholas R. Y. Ho, Wanxiang Shen, Yun Xu, Auginia Natalia, Zhonglang Yu, J. Seet, Ching Wan Chan, T. P. Loh, Brian Y Lim, Huilin Shao
          </td>
          <td>2024-06-19</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Stalled replication forks can be processed by several distinct mechanisms collectively called post-replication repair which includes homologous recombination, fork regression, and translesion DNA synthesis. However, the regulation of the usage between these pathways is not fully understood. The Rad51 protein plays a pivotal role in maintaining genomic stability through its roles in HR and in protecting stalled replication forks from degradation. We report the isolation of separation-of-function mutations in Saccharomyces cerevisiae Rad51 that retain their recombination function but display a defect in fork protection leading to a shift in post-replication repair pathway usage from HR to alternate pathways including mutagenic translesion synthesis. Rad51-E135D and Rad51-K305N show normal in vivo and in vitro recombination despite changes in their DNA binding profiles, in particular to dsDNA, with a resulting effect on their ATPase activities. The mutants lead to a defect in Rad51 recruitment to stalled forks in vivo as well as a defect in the protection of dsDNA from degradation by Dna2-Sgs1 and Exo1 in vitro. A high-resolution cryo-electron microscopy structure of the Rad51-ssDNA filament at 2.4 Å resolution provides a structural basis for a mechanistic understanding of the mutant phenotypes. Together, the evidence suggests a model in which Rad51 binding to duplex DNA is critical to control pathway usage at stalled replication forks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0b5df394479fd183dbce68e8ce90f494d910d5a" target='_blank'>
              Rad51 determines pathway usage in post-replication repair
              </a>
            </td>
          <td>
            Damon H. Meyer, Shannon J. Ceballos, Steven K Gore, Jie Liu, Giordano Reginato, María I. Cano-Linares, Katarzyna H. Masłowska, Florencia Villafañez, Christopher Ede, Vincent Pagès, F. Prado, Petr Cejka, W. Heyer
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Skeletal muscle, the largest human organ by weight, is relevant to several polygenic metabolic traits and diseases including type 2 diabetes (T2D). Identifying genetic mechanisms underlying these traits requires pinpointing the relevant cell types, regulatory elements, target genes, and causal variants. Here, we used genetic multiplexing to generate population-scale single nucleus (sn) chromatin accessibility (snATAC-seq) and transcriptome (snRNA-seq) maps across 287 frozen human skeletal muscle biopsies representing 456,880 nuclei. We identified 13 cell types that collectively represented 983,155 ATAC summits, cataloging caQTL peaks that atlas-level snATAC maps often miss. We integrated genetic variation to discover 6,866 expression quantitative trait loci (eQTL) and 100,928 chromatin accessibility QTL (caQTL) (5% FDR) across the five cell types. We identified 1,973 eGenes colocalized with caQTL and used mediation analyses to construct causal directional maps for chromatin accessibility and gene expression. 3,378 genome-wide association study (GWAS) signals across 43 relevant traits colocalized with sn-e/caQTL, 52% in a cell-specific manner. 77% of GWAS signals colocalized with caQTL and not eQTL, highlighting the critical importance of population-scale chromatin profiling for GWAS functional studies. A C2CD4A/B T2D GWAS signal colocalized with caQTL in muscle fibers and multiple chromatin loop models nominated VPS13C, a glucose uptake gene. Sequence of the caQTL peak overlapping caSNP rs7163757 showed allelic regulatory activity differences in a human myocyte cell line massively parallel reporter assay. These results illuminate the genetic regulatory architecture of human skeletal muscle at high-resolution epigenomic, transcriptomic, and cell state scales and serve as a template for population-scale multi-omic mapping in complex tissues and traits.


 A. Varshney: None. N. Manickam: Employee; 10x Genomics. P. Orchard: None. A. Tovar: None. Z. Zhang: None. F. Feng: None. T.A. Lakka: None. M. Laakso: None. J. Tuomilehto: Stock/Shareholder; Orion Pharma, Aktivolabs, Digostics. K.L. Mohlke: None. J. Kitzman: Advisory Panel; Myome Inc. H.A. Koistinen: Other Relationship; AstraZeneca, Novo Nordisk. J. Liu: None. M. Boehnke: None. F.S. Collins: None. L. Scott: None. S.C. Parker: Research Support; Pfizer Inc.



 R01DK072193UM1DK126185R01DK117960P30AR069620
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc8383a53e108ff0e106cd850a1b389d44003926" target='_blank'>
              2013-LB: Population-Scale Skeletal Muscle Single-Nucleus Multiomic Profiling Reveals Extensive Context-Specific Genetic Regulation
              </a>
            </td>
          <td>
            Arushi Varshney, Nandini Manickam, P. Orchard, Adelaide Tovar, Zhenhao Zhang, Fan Feng, Timo A Lakka, Markku Laakso, J. Tuomilehto, K. Mohlke, Jacob Kitzman, H. Koistinen, Jie Liu, M. Boehnke, Francis S. Collins, Laura Scott, Stephen C J Parker
          </td>
          <td>2024-06-14</td>
          <td>Diabetes</td>
          <td>0</td>
          <td>142</td>
        </tr>

        <tr id="Introduction The promoter methylation status of O-6-methylguanine-DNA methyltransferase (MGMTp) is an established predictive and prognostic marker in GBM. Previous studies showed that the expression of MGMT based on immunohistochemistry was variable and lacked association with survival. This in part is because non-tumor cells including endothelial cells and macrophages express MGMT. Advanced technologies such as single-cell RNA (scRNA) sequencing have helped to elucidate the cellular composition of cancer and its microenvironment. scRNA sequencing allows to assess gene expression level in tumor cells specifically. Methods We used publicly available data from two recent GBM scRNA studies that included MGMTp methylation status data for patients to explore and uncover details about MGMT expression at the single-cell level: CPTAC (13 primary samples) and Neftel (20 primary samples). Results In the CPTAC study, MGMT expression ranged from 0.19%-1.43% in the MGMTp methylated group (median 0.82%), and from 2.17%-28.36% in the MGMTp unmethylated group (median 5.7%). It therefore appears that 2% is a reasonable expression cutoff to predict the MGMTp methylation status based on scRNA data. In the Neftel study, MGMT expression ranged from 0-1.26% in the MGMTp methylated group (median 0.59%), and from 0.3-27.67% in the MGMTp unmethylated group (median 12.44%). Three unmethylated samples (out of 16) did not follow the 2% rule. It remains unclear if this is due to technical inaccuracies as the Neftel paper did not specify the method used to detect MGMTp methylation or even mere typos. Alternatively, could it be that truly MGMTp unmethylated samples can have low MGMT expression? Could this explain why some unmethylated MGMTp GBM patients surpass the expected survival? Interestingly, gene set enrichment analysis shows that MGMT expressing cells are enriched with mesenchymal genes, whereas MGMT negative cells are enriched with proneural genes. Conclusion Fewer than 2% of GBM cells express MGMT when MGMTp is methylated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/341ca80b1f0064503114912a62daed828ce1857b" target='_blank'>
              What scRNA sequencing taught us about MGMT expression in glioblastoma
              </a>
            </td>
          <td>
            I. Alnahhas, Mehak M. Khan, Wenyin Shi
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The genome of a cell is continuously battered by a plethora of exogenous and endogenous processes that can lead to damaged DNA. Repair mechanisms correct this damage most of the time, but failure to do so leaves mutations. Mutations do not occur in random manner, but rather typically follow a more or less specific pattern due to known or imputed mutational processes. Mutational signature analysis is the process by which the predominant mutational process can be inferred for a cancer and can be used in several contexts to study both the genesis of cancer and its response to therapy. Recent pan-cancer genomic efforts such as "The Cancer Genome Atlas" have identified numerous mutational signatures that can be categorized into single base substitutions, doublet base substitutions, or small insertions/deletions. Understanding these mutational signatures as they occur in non-small lung cancer could improve efforts at prevention, predict treatment response to personalized treatments, and guide the development of therapies targeting tumor evolution. For non-small cell lung cancer, several mutational signatures have been identified that correlate with exposures such as tobacco smoking and radon and can also reflect endogenous processes such as aging, APOBEC activity, and loss of mismatch repair. Herein, we provide an overview of the current knowledge of mutational signatures in non-small lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37777d22dfc5a2bd99dc42f300aad4d0a01e922f" target='_blank'>
              Topography of mutational signatures in non-small cell lung cancer: emerging concepts, clinical applications, and limitations.
              </a>
            </td>
          <td>
            Ritujith Jayakrishnan, David J. Kwiatkowski, Michal G Rose, Amin H. Nassar
          </td>
          <td>2024-06-22</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution.
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Aneuploidy is an abnormal chromosome composition and a general hallmark of human cancer. Aneuploidy causes detrimental cellular stresses, but cancer cells evolve to cope with these stresses. Consequently, targeting such mitigation mechanisms is a promising potential therapeutic strategy. As an abnormal dosage of gene products from altered chromosomes can cause RNA and proteotoxic stress, dosage compensation (DC) of imbalanced gene products was reported to mitigate these stresses in aneuploid cells. However, the mechanisms that regulate DC remain elusive. To address these mechanisms, we focused on the role(s) of stress granules (SGs) and RNA binding proteins (RBPs) in aneuploid cancer cells. Our recent study revealed that aneuploid cancer cells preferentially depend on RNA and protein metabolism, and need to attenuate translation in order to cope with proteotoxic stress (Ippolito & Zerbib et al. bioRxiv 2023). In yeast, it has been previously reported that Ssd1, one of the RBPs localized in P-bodies, is indispensable to tolerating aneuploidy stress. Based on these findings, we set out to explore SGs as potential regulators of gene expression in aneuploid cancer cells. As expected, we revealed that SGs emerged in cells under acute aneuploidy stress induced by chromosome mis-segregation. These results imply that SGs are formed in cells during and right after aneuploidization, potentially as a way to mediate DC and ameliorate aneuploidy-induced cellular stress. To gain further insight into RNA regulation in aneuploid cells, we analyzed the essentiality of RBPs across cancer cell lines. We found that the RNA-editing enzyme ADAR, a suppressor of SG formation, is preferentially essential in aneuploid cancer cells. Remarkably, ADAR disruption synergized with acute aneuploidy stress to induce SGs, and ADAR exhibited increased RNA editing activity following chromosome mis-segregation. Our results suggest that SGs and their suppressor ADAR might contribute to ameliorating acute aneuploidy-induced stress. We are currently investigating the target RNAs regulated by SGs and ADAR and examining their functional role(s) in DC, with the goal of uncovering potential synthetic lethalities associated with aneuploidy in cancer cells.
 Citation Format: Hajime Okada, Eran Sdeor, Miriam Karmon, Erez Levanon, Uri Ben-David. Targeting mechanisms of dosage compensation to selectively kill aneuploid cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d65eea6dda42eca3bae9912c1560d589405d46e4" target='_blank'>
              Abstract PR008: Targeting mechanisms of dosage compensation to selectively kill aneuploid cancer cells
              </a>
            </td>
          <td>
            Hajime Okada, Eran Sdeor, Miriam Karmon, E. Levanon, Uri Ben-David
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The deoxyribonucleic acid (DNA) carries hereditary codes which is translated by the cells to synthesize the ribonucleic acid (RNA) and polypeptides which can generate and perform vital function. The double helix structure is the most studied model of the DNA that was proposed by Watson and Crick. The capability of DNA to work as a genetic material can be stored and conducted during cell division to permit this information to be doubled and transmitted to the incoming generation. Any damage in the structure of DNA is an essential direct cause for the progression of cancer and other disorders. The factors for DNA damage can be classified as exogenous and endogenous factors. In this review article, we highlight the evidence-supported information about the structure, functions and clinical significance of DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23ff4de2b8e4130d71de29efe87be0afb81a98a9" target='_blank'>
              Structure, Functions And Clinical Significance Of DNA: A Review Article
              </a>
            </td>
          <td>
            Ahmed M. Amshawee, Maryam A. Hussain, Seenaa Taqi Mansour Al Muhtaser, Ali A. Al- Faham
          </td>
          <td>2024-07-22</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Aneuploidy, an imbalanced number of chromosomes or chromosome arms, is the most common genetic aberration in cancer and is associated with a worse prognosis and advanced disease. However, in non-transformed cells, aneuploidy has a substantial fitness cost, leading to cell cycle arrest and cell death. This fitness cost is the result of several cellular stresses associated with aneuploidy, including proteotoxic, metabolic, mitotic and replication stress. Therefore, for aneuploidy to become beneficial, tolerance mechanisms that enable cancer cells to cope with aneuploidy-induced cellular stresses must come into play. Such mechanisms may uncover novel synthetic lethalities of aneuploid cancer cells. To this end, we systematically dissected the cellular mechanisms that facilitate aneuploidy tolerance. We combined large-scale analyses of gene expression, as well as genetic and pharmacological dependency datasets from human tumors, cancer cell lines and non-transformed cells, to identify genes and pathways associated with aneuploidy tolerance. We identified 55 genes associated (with high confidence) with aneuploidy tolerance. Gene set enrichment analysis (GSEA) revealed an enrichment for ribosome biogenesis and translation, DNA repair, replication stress and the p53 pathway. Interestingly, these pathways were all previously implicated with the aforementioned aneuploidy-induced cellular stresses. We plan to conduct a dependency mini-screen to validate the top ‘hits’ from our comprehensive bioinformatic analysis, and to explore the role of validated candidates in conferring aneuploidy tolerance. One of the top ‘hits’ of our exploratory analysis was a novel gene with an unclear cellular function. We experimentally confirmed that this gene was over-expressed in aneuploid cells, across several isogenic systems of matched diploid-aneuploid cell lines. Aneuploidy induction using MPS1 inhibitors led to its significant upregulation, demonstrating that its increased expression is a general feature of aneuploid cells. Mechanistically, we found that the novel gene had a role in regulating chromosome segregation, as its knockdown increased the rate of mitotic aberrations – as well as the sensitivity to mitotic checkpoint inhibition – resulting in increased chromosomal instability. We now study the molecular basis for the effect of this novel gene on aneuploidy tolerance. Specifically, we knocked it down in aneuploid cells, and over-expressed it in diploid cells, and we currently explore how these manipulations affect the above-mentioned aneuploidy tolerance pathways, namely replication stress, ribosome biogenesis, DNA damage repair and tp53 activation. In summary, we revealed a novel gene that is a key regulator of aneuploidy tolerance in cancer cells, likely due to its direct involvement in pathways required for the cellular coping with aneuploidy-induced stresses.
 Citation Format: Yonatan Eliezer, Adi V Tarrab, Tal Ben-Yishay, Hajime Okada, Tom Winkler, Uri Ben-David. Identification of novel genes that regulate aneuploidy tolerance by attenuating aneuploidy-induced stresses [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82fed258fc2885195d90b257738b8f12490644d6" target='_blank'>
              Abstract B003: Identification of novel genes that regulate aneuploidy tolerance by attenuating aneuploidy-induced stresses
              </a>
            </td>
          <td>
            Yonatan Eliezer, Adi V Tarrab, Tal Ben-Yishay, Hajime Okada, Tom Winkler, Uri Ben-David
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Introduction: Synthetically lethal mutations offer unique molecular targets for oncologic therapy. At scale, synthetic lethality (SL) is observed when alterations to one gene alone do not correspond with worse overall survival (OS), but simultaneous expression with another gene does correspond with worse OS. Mutually exclusive gene expression is not a requirement for SL. Existing breast cancer (BC) literature suggests synthetic lethality between CKS1B and PLK1, as well as BRCA1/BRCA2 and PARP1. Unaltered RB1 and altered CCND1 are also known to exhibit SL in the HER2-deficient environment of triple-negative BC. CCND1 and ERBB2 are typically amplified in invasive ductal carcinoma, with ERBB2 amplifications correlating to larger tumor size. This relationship is not observed in invasive lobular carcinoma, although expression of E2F1, a downstream transcription factor of CCND1, is inversely correlated with tumor grade. Methods: Our in silico approach investigated SL between ERBB2 and CCND1 in breast invasive carcinoma without distinction to lobular or ductal origin (TCGA, PanCancer Atlas 2018). We identified 17 genes of interest based on molecular alterations present in >10% of the sample (n=1084) and common mentions in BC literature. SL was determined if alterations to one gene alone did not significantly worsen OS, while alterations to two genes corresponded with worse OS. Results: Twenty-three SL interactions were observed. CCND1 and ERBB2 were selected because neither gene significantly worsened OS alone (p=0.915 & p=0.0951 respectively). However, patients with alterations to both genes had significantly worse OS compared to those with alterations to one gene alone. CCND1 and ERBB2 alterations were present in 14.9% and 13.7% of the sample, respectively. HR=2.568 (p=0.0138) for SL patients compared to those with CCND1 alterations alone; HR=2.244 (p=0.0497) for SL patients compared to those with ERBB2 alterations alone. Mutual exclusivity was not significantly observed (p=0.084). Multivariate analysis was performed using Cox regression models; neither age (older or younger than 55 years) nor race (White, Black, Asian) confounded our results due to p>0.05. Conclusion: Our findings support potential therapeutic use of CCND1 targets in HER2+ BC, specifically for patients with SL between CCND1 and ERBB2. These include PLK1 inhibitors, commonly used in estrogen receptor-positive patients with CDK4/6 inhibitor resistance, along with HER2 inhibitors. A combination of MAP2K and PIK3CA inhibitors should also be investigated in HER2+ BC patients with CCND1 amplification, since HER2+ colorectal cancer patients have been demonstrated to fare better with this treatment.
 Citation Format: Rishi Nair, Evan P. White, Roy Khalife, Anthony M. Magliocco. Synthetic lethality of ERBB2 and CCND1 in breast cancer at scale [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr A022.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55979c4c4c26410bf8b5a7c7beacc6c58c9d670e" target='_blank'>
              Abstract A022: Synthetic lethality of ERBB2 and CCND1 in breast cancer at scale
              </a>
            </td>
          <td>
            Rishi Nair, Evan White, Roy Khalife, Anthony M. Magliocco
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Breast-to-brain metastatic (BTB met) cancer constitutes a great challenge for modern medicine and science, and thus, a quest for the discovery of novel therapeutic targets is underway. As the research on protein-coding genes has brought only incremental progress in discovering such targets, much attention is currently devoted to understanding the role of non-coding RNAs, including a relatively recently discovered group of circular RNAs. These non-coding RNAs, which arise from linear RNA transcripts, possess a covalently closed structure that renders them resistant to RNA turnover and a specific circularization site sequence that allows their precise suppression while keeping the linear parental transcript intact. These features thus make them promising candidates for biomarkers and therapeutic targets. This project aimed to map the circular RNA landscape in a comprehensive model of BTB met cancer cells, primary breast carcinoma, primary brain tumor cells, and relevant non-malignant breast and brain cell controls. Despite different tissue-of-origin, the analysis of global circRNAomes showed striking similarities between BTB mets and glioblastoma stem-like cells (GSCs) and allowed the selection of reliable candidates for mechanistic studies. Interestingly, the circRNAome of BTB mets clustered particularly closely with mesenchymal GSC, the most aggressive and therapy-resistant subtype of GSCs. Efficient knockdown of candidate circular RNAs in BTB met cells by antisense oligonucleotides resulted in molecular and phenotypic alterations such as diminished viability and clonality, arrested cell cycle concurrent with increased expression of cyclin-dependent kinase inhibitors, and changed expression of epithelial-mesenchymal transition markers. These results thus demonstrated the role of circular RNAs in shaping the oncogenic traits of BTB met cells. Further research focuses on circular RNA-dependent chemotherapy resistance, stemness/differentiation balance, and relevant models of metastatic progression (primary site/pleural effusion/brain). Therefore, circular RNAs can serve as valuable biomarkers and promising novel mechanistic candidates whose modulation may become a paradigm-shifting therapeutic approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8da7f6f849a7d637dd2d2df1a3a9af41a6848554" target='_blank'>
              NVTG-02 CIRCULAR RNAS AS NOVEL TARGETS IN BREAST-TO-BRAIN CANCER METASTASIS
              </a>
            </td>
          <td>
            A. Szczepaniak, Agnieszka Bronisz, J. Godlewski
          </td>
          <td>2024-08-01</td>
          <td>Neuro-Oncology Advances</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/358937b5c51be3f8de419eabd20cda30e0675361" target='_blank'>
              Emerging toolkits for decoding the co-occurrence of modified histones and chromatin proteins.
              </a>
            </td>
          <td>
            Anne-Sophie Pépin, Robert Schneider
          </td>
          <td>2024-08-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [15],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>