<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-23 06:31:43 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>24</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>52</td>
          <td>130</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>101</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, Jens Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>279</td>
          <td>130</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>241</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, Jens Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>111</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) have been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterized the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA were flanked by novel somatic DNA enhancers and HPV L1 enhancers, with strong cis-interaction. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduced E6/E7 expression, and significantly inhibited in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with novel somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c00e1f8a07a4acac2f60dea45935a4972a1645" target='_blank'>
              Inhibition of novel human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Joseph Califano, Takuya Nakagawa, Jens Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, Sayed Sadat, Sara Javadzadeh, Qian Yang, Allen Wang, Kersi Pestonjamasp, Sara Rosenthal, Kathleen M Fisch, Paul S. Mischel, V. Bafna
          </td>
          <td>2024-09-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Rationale: Extrachromosomal circular DNA is a hallmark of cancer, but its role in shaping the genome heterogeneity of urothelial bladder carcinoma (UBC) remains poorly understood. Here, we comprehensively analyzed the features of extrachromosomal circular DNA in 80 UBC patients. Methods: We performed whole-genome/exome sequencing (WGS/WES), Circle-Seq, single-molecule real-time (SMRT) long-read sequencing of circular DNA, and RNA sequencing (RNA-Seq) on 80 pairs of tumor and AT samples. We used our newly developed circular DNA analysis software, Circle-Map++ to detect small extrachromosomal circular DNA from Circle-Seq data. Results: We observed a high load and significant heterogeneity of extrachromosomal circular DNAs in UBC, including numerous single-locus and complex chimeric circular DNAs originating from different chromosomes. This includes highly chimeric circular DNAs carrying seven oncogenes and circles from nine chromosomes. We also found that large tumor-specific extrachromosomal circular DNAs could influence genome-wide gene expression, and are detectable in time-matched urinary sediments. Additionally, we found that the extrachromosomal circular DNA correlates with hypermutation, copy number variation, oncogene amplification, and clinical outcome. Conclusions: Overall, our study provides a comprehensive extrachromosomal circular DNA map of UBC, along with valuable data resources and bioinformatics tools for future cancer and extrachromosomal circular DNA research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0454d85a5ac1363a4fa012edcd2f47cff5be9b32" target='_blank'>
              Extrachromosomal circular DNA orchestrates genome heterogeneity in urothelial bladder carcinoma
              </a>
            </td>
          <td>
            Wei Lv, Xiaoguang Pan, Peng Han, Shuang Wu, Yuchen Zeng, Qingqing Wang, Lidong Guo, Mengyang Xu, Yanwei Qi, Li Deng, Zhe Xu, Conghui Li, Tianxi Yu, Xin Cui, Huajing Teng, Chongjun Xiang, Haotian Tan, Yue Li, Ning Liang, Huiying Tao, Qingqing Gao, Guohua Yu, Jia Mi, Fuyi Xu, Benjiao Gong, Lei Shi, Tao Wang, Huanming Yang, Wei Dong, Lars Bolund, Lin Lin, Wenting Wang, Hanbo Li, Jinrong Huang, Chunhua Lin, Yonglun Luo
          </td>
          <td>2024-08-19</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a form of oncogene amplification found across cancer types and associated with poor outcome in patients. ecDNA can be structurally complex and contain rearranged DNA sequences derived from multiple chromosome locations. As the structure of ecDNA can impact oncogene regulation and may indicate mechanisms of its formation, disentangling it at high resolution from sequencing data is essential. Even though methods have been developed to identify and reconstruct ecDNA in cancer genome sequencing, it remains challenging to resolve complex ecDNA structures, in particular amplicons with shared genomic footprints. We here introduce Decoil, a computational method which combines a breakpoint-graph approach with regression to reconstruct complex ecDNA and deconvolve co-occurring ecDNA elements with overlapping genomic footprints from long-read nanopore sequencing. Decoil outperforms de novo assembly and alignment-based methods in simulated long-read sequencing data for both simple and complex ecDNAs. Applying Decoil on whole genome sequencing data uncovered different ecDNA topologies and explored ecDNA structure heterogeneity in neuroblastoma tumors and cell lines, indicating that this method may improve ecDNA structural analyzes in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c914721ddd777cd78df6632d6860e72cff344be" target='_blank'>
              Reconstructing extrachromosomal DNA structural heterogeneity from long-read sequencing data using Decoil.
              </a>
            </td>
          <td>
            M. Giurgiu, Nadine Wittstruck, Elias Rodriguez-Fos, R. Chamorro González, Lotte Brueckner, Annabell Krienelke-Szymansky, K. Helmsauer, Anne Hartebrodt, P. Euskirchen, R. Koche, Kerstin Haase, Knut Reinert, A. Henssen
          </td>
          <td>2024-08-07</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is a pervasive driver event in cancer, yet our understanding of how ecDNA forms is limited. Here, we couple a CRISPR-based method for ecDNA induction with extensive characterization of newly formed ecDNA to examine their biogenesis. We find that DNA circularization is efficient, irrespective of 3D genome context, with formation of 800kb, 1 Mb, and 1.8 Mb ecDNAs reaching or exceeding 15%. We show non-homologous end joining and microhomology-mediated end joining both contribute to ecDNA formation, while inhibition of DNA-PKcs and ATM have opposing impacts on ecDNA formation. EcDNA and the corresponding chromosomal excision scar can form at significantly different rates and respond differently to DNA-PKcs and ATM inhibition. Taken together, our results support a model of ecDNA formation in which double strand break ends dissociate from their legitimate ligation partners prior to joining of illegitimate ends to form the ecDNA and excision scar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e488ec3b1ac40e7e2986571eab100d1986b6e3f" target='_blank'>
              Disparate pathways for extrachromosomal DNA biogenesis and genomic DNA repair.
              </a>
            </td>
          <td>
            John C. Rose, J. Belk, I. Wong, Jens Luebeck, Hudson T Horn, Bence Daniel, Matthew G Jones, Kathryn E Yost, King L. Hung, Kevin S Kolahi, Ellis J Curtis, Calvin J. Kuo, V. Bafna, Paul S. Mischel, Howard Y. Chang
          </td>
          <td>2024-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Enhancer elements are specific DNA sequences that play a crucial role in regulating gene expression. Located upstream or downstream in the genomic context, enhancers enhance the transcription of linked genes. This is achieved by providing binding sites for transcription factors and other proteins, acting as a nucleation point that promotes the assembly of the transcriptional machinery.
What makes enhancers unique is that located even thousands of base pairs away, they can exert their function. They can act over long distances, looping to interact with the promoter region of their target genes. Enhancers are often involved in cell type-specific gene expression as different cells express different sets of genes by activating and repressing cell type-specific enhancers.
Cancer cells seem to co-opt this mechanism and hijack it for their own survival. In this process, an aberrant enhancer element activates the transcription of oncogenes (genes that have the potential to cause cancer) due to some alterations in genomic structure or regulatory elements that happen in cancer, contributing to tumorigenesis and cancer progression. Enhancer hijacking often occurs through complex genomic rearrangements such as chromosomal translocations and other structural rearrangements such as deletions or amplifications.8 These alterations can bring a usually distant, far-located enhancer into proximity with an oncogene, leading to its inappropriate activation. This is often seen in various cancers, including leukemias and solid tumors.
Enhancer hijacking and its effects on gene regulation in cancer are frequently studied using methods such as RNA sequencing, chromatin conformation capture, and CRISPR-Cas9. Finding an enhancer hijacking event may be used to inform treatment choices, especially in precision medicine settings, and may also function as a biomarker for particular cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefe88575de8420490e0e1dd10a24dbefe2eef41" target='_blank'>
              Enhancer hijacking: Innovative ways of carcinogenesis
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-09-02</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia with complex karyotype (ckAML) is characterized by high genomic complexity, including frequent TP53 mutations and chromothripsis. We hypothesized that the numerous genomic rearrangements could reposition active enhancers near proto-oncogenes, leading to their aberrant expression. We developed pyjacker, a computational tool for the detection of enhancer hijacking events, and applied it to a cohort of 39 ckAML samples. Pyjacker identified motor neuron and pancreas homeobox 1 (MNX1), a gene aberrantly expressed in 1.4% of AML patients, often as a result of del(7)(q22q36) associated with hijacking of a CDK6 enhancer. MNX1-activated cases show significant co-occurrence with BCOR mutations and a gene signature shared with t(7;12)(q36;p13) pediatric AML. We demonstrated that MNX1 is a dependency gene, as its knockdown in a xenograft model reduces leukemia cell fitness. In conclusion, enhancer hijacking is a frequent mechanism for oncogene activation in AML. Statement of significance This study examines the consequences of structural alterations and demonstrates that proto-oncogene activation by enhancer hijacking is an overlooked pathomechanism in AML. MNX1 overexpression demonstrates that deletions on chromosome 7q can not only lead to haploinsufficiency, but also to activation of oncogenes by enhancer hijacking, providing a novel leukemogenic mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb3e7eaaf436e861083291af3f4d69f9c709309" target='_blank'>
              Pyjacker identifies enhancer hijacking events in acute myeloid leukemia including MNX1 activation via deletion 7q
              </a>
            </td>
          <td>
            Etienne Sollier, Anna Riedel, U. Toprak, Justyna A. Wierzbinska, D. Weichenhan, J. P. Schmid, M. Hakobyan, Aurore Touzart, E. Jahn, B. Vick, Fiona Brown-Burke, Katherine Kelly, Simge Kelekçi, Anastasija Pejkovska, Ashish Goyal, Marion Bähr, Kersten Breuer, , Maria Llamazares-Prada, M. Hartmann, M. Schönung, Nádia Correia, A. Trumpp, Yomn Abdullah, Ursula Klingmüller, S. S. Mughal, B. Brors, Frank Westermann, Matthias Schlesner, S. Vosberg, T. Herold, P. Greif, Dietmar Pfeifer, Michael Lübbert, Thomas Fischer, F. H. Heidel, C. Gebhard, W. Walter, Torsten Haferlach, Ann-Kathrin Eisfeld, K. Mrózek, D. Nicolet, L. Bullinger, L. Smeenk, Claudia Erpelinck, Roger Mulet-Lazaro, Ruud Delwel, Aurélie Ernst, Michael Scherer, P. Lutsik, I. Jeremias, Konstanze Döhner, H. Döhner, D. Lipka, C. Plass
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="An important goal in cancer research is to identify driver genes and mutations. Reasoning that such mutations often alter enzymatic functions, we investigated the cancer driver role of enzyme families. Using pan-cancer genomic data and established driver mutation catalogues, we found an unexpectedly high rate of mutations in helicases, making helicases the most frequently mutated enzyme family in cancer. Based on both functional perturbation screens and cancer genomic analyses, we provide evidence that cancers with mutated helicases converge on increased genomic instability and faulty DNA repair. We identify a striking phenotype in cells with loss of the helicase Aquarius (AQR). AQR was exclusively hemizygous lost in cancer genomes, which was associated with elevated levels of structural variants and point mutation signatures indicative of homologous recombination deficiency. Finally, we leverage large dependency maps to show that hemizygous loss is a common tumour suppression mechanism among helicases. In summary, we uncover a striking frequency of mutated helicases with key roles in genomic maintenance, and we nominate novel hemizygous cancer driver genes including AQR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/381ed30f7e9d5d197bbe8fa9f74d37852c71bb4a" target='_blank'>
              Enzyme family-centred approach identifies helicases as recurrent hemizygous tumour suppressor genes
              </a>
            </td>
          <td>
            Karolin Voßgröne, F. Favero, Krushanka Kashyap, F. G. Rodríguez-González, André Vidas Olsen, Xin Li, Balca R. Mardin, Joachim Weischenfeldt, Claus S. Sørensen
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19b77f033f8dcc5eb2b738f43988f481891ac30a" target='_blank'>
              Large-scale copy number alterations are enriched for synthetic viability in BRCA1/BRCA2 tumors
              </a>
            </td>
          <td>
            Yingjie Zhu, Xin Pei, Ardijana Novaj, J. Setton, Daniel Bronder, F. Derakhshan, P. Selenica, Niamh McDermott, Mehmet Orman, Sarina Plum, Shyamal Subramanyan, Sara H. Braverman, Biko McMillan, Sonali Sinha, Jennifer Ma, Andrea M Gazzo, Atif Khan, S. Bakhoum, Simon N. Powell, J. Reis-Filho, N. Riaz
          </td>
          <td>2024-08-28</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="To identify cancer-associated gene regulatory changes, we generated single-cell chromatin accessibility landscapes across eight tumor types as part of The Cancer Genome Atlas. Tumor chromatin accessibility is strongly influenced by copy number alterations that can be used to identify subclones, yet underlying cis-regulatory landscapes retain cancer type–specific features. Using organ-matched healthy tissues, we identified the “nearest healthy” cell types in diverse cancers, demonstrating that the chromatin signature of basal-like–subtype breast cancer is most similar to secretory-type luminal epithelial cells. Neural network models trained to learn regulatory programs in cancer revealed enrichment of model-prioritized somatic noncoding mutations near cancer-associated genes, suggesting that dispersed, nonrecurrent, noncoding mutations in cancer are functional. Overall, these data and interpretable gene regulatory models for cancer and healthy tissue provide a framework for understanding cancer-specific gene regulation. Editor’s summary Cancer is caused by mutations in DNA that often accompany profound changes to the nucleoprotein structure of the genome (the epigenome) that governs gene expression. Sundaram et al. mapped active regions of regulatory DNA from single cells in diverse human cancer types. Their analysis revealed distinct patterns of open chromatin in different types and genetic subclones of cancer and immune cells in tumors. The data further provide insight into the regulatory phenotypes of breast cancer subtypes and enable the training of cancer-specific neural network models that can interpret the impact of noncoding mutations on the open chromatin landscape. The deep learning models suggest that inherited or cancer-specific mutations can affect important gene-regulatory elements adjacent to, and possibly regulating, key cancer-causing genes. —Priscilla N. Kelly INTRODUCTION Genetic lesions are the root cause of cancer and drive gene regulatory changes that evince the classical phenotypic “hallmarks” of cancer. The Cancer Genome Atlas (TCGA) has aimed at understanding the diverse molecular features associated with both these genetic mutations and associated gene regulatory transformations that drive phenotypes across diverse cancer types. RATIONALE One means of identifying the regulatory elements and trans factors associated with the gene expression has been chromatin accessibility mapping using the assay of transposase-accessible chromatin using sequencing (ATAC-seq), which identifies the fraction of the genome actively bound by regulatory proteins, such as transcription factors (TFs) and polymerases. Although these regulatory elements have been previously explored by TCGA in “bulk” cancer tissue, such analysis convolves malignant signatures from cancer with the landscape of regulatory elements present in associated stromal or immune cells. By contrast, single-cell chromatin accessibility data allows for deconvolution of these signals. Furthermore, a cancer-specific gene regulatory “grammar” can be learned from these cancer-specific data by using deep learning models, allowing both identification of genomic sequences associated with TF binding and the prediction of the effects of cancer-associated mutation on the accessibility of these elements. RESULTS We generated a single-cell chromatin accessibility atlas from TCGA archival samples spanning eight cancer types and 74 individual cancer samples composed of 227,063 nuclei. Gene accessibility around marker genes allowed us to identify cancer cells versus tumor-infiltrating stromal and immune cells. Megabase-scale changes in chromatin accessibility signals associated with copy number alteration identified tumor subclones on the basis of differential copy number changes. We observed that tumor subclones can exhibit differential gene regulatory programs associated with TFs affected by subclone-specific copy number alterations. Using these data, we trained interpretable neural network–based models of cis regulation that nominate specific TF motifs associated with differential chromatin accessibility signals in cancer (as compared with normal tissue) and cancer subtypes. This analysis provides a platform to compare regulatory grammars and regulatory modules from cancerous cells to those of the most similar healthy tissue type. Analysis of breast cancer subtypes demonstrated that the chromatin signature of the basal-like subtype of breast cancer is most like secretory-type luminal epithelial cells, confirming that this molecular subtype is not most epigenomically similar to basal-type normal cells. We used our neural network models to predict the impact of genetic variation on chromatin accessibility by installing single-nucleotide variants and cancer-specific somatic mutations observed in human cancers or populations into the genome in silico. Mutations predicted to cause either a strong gain or loss of chromatin accessibility were enriched near cancer-associated genes compared with matched, non–cancer-associated gene sets, providing evidence that somatic noncoding mutations can drive changes in cancer-associated chromatin regulation. CONCLUSION The TCGA single-cell atlas of multiple human primary cancer types and interpretable deep learning models for interpreting cis-regulatory elements in cancer comprises new resources for understanding the molecular programs that bring about the malignant phenotypes in primary human cancers. A pan-cancer, single-cell chromatin accessibility atlas of gene regulation in cancer. This atlas identifies tumor cells from other cell types on the basis of gene-scale accessibility patterns and enables the analysis of subclonal heterogeneity in tumors through megabase-scale changes. Neural network models reveal accessible chromatin signatures specific to cancer versus healthy tissue and highlight the role of noncoding somatic mutations in cancer. scATAC-seq, single-cell ATAC-seq; DL Pred., deep learning prediction; Ref, reference; Alt, alternate.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88554b25c16caafc022c58d4c1dc2073d72caa6a" target='_blank'>
              Single-cell chromatin accessibility reveals malignant regulatory programs in primary human cancers
              </a>
            </td>
          <td>
            Laksshman Sundaram, Arvind Kumar, Matthew Zatzman, Adriana Salcedo, Neal Ravindra, Shadi Shams, Brian H. Louie, S. Bagdatli, Matthew A. Myers, Shahab Sarmashghi, Hyo Young Choi, Won-Young Choi, K. Yost, Yanding Zhao, Jeffrey M. Granja, T. Hinoue, D. N. Hayes, Andrew Cherniack, Ina Felau, Hani Choudhry, J. Zenklusen, K. Farh, Andrew McPherson, Christina Curtis, Peter W. Laird, M. Corces, Howard Y. Chang, W.J. Greenleaf, John A. Demchok, Liming Yang, R. Tarnuzzer, Samantha J. Caesar-Johnson, Zhining Wang, A. S. Doane, Ekta Khurana, Mauro A A Castro, Alexander J. Lazar, B. Broom, John N. Weinstein, Rehan Akbani, Shweth V. Kumar, Benjamin J. Raphael, Christopher K. Wong, Joshua M. Stuart, Rojin Safavi, Christopher C. Benz, Benjamin K Johnson, Cindy Kyi, Hui Shen
          </td>
          <td>2024-09-06</td>
          <td>Science</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Chromosome aberrations (CAs), a genotoxic potential of carcinogens, are believed to contribute to tumorigenesis by chromosomal rearrangements through micronucleus formation. However, there is no direct evidence that proves the involvement of CAs in tumorigenesis in vivo. In the current study, we sought to clarify the involvement of CAs in chemical carcinogenesis using a rat model with a pure CA-inducer hepatocarcinogen, acetamide. Whole-genome analysis indicated that hepatic tumors induced by acetamide treatment for 26-30 weeks showed a broad range of copy number alterations in various chromosomes. In contrast, hepatic tumors induced by a typical mutagen (diethylnitrosamine) followed by a nonmutagen (phenobarbital) did not show such mutational patterns. Additionally, structural alterations such as translocations were observed more frequently in the acetamide-induced tumors. Moreover, most of the acetamide-induced tumors expressed c-Myc and/or MDM2 protein due to the copy number gain of each oncogene. These results suggest the occurrence of chromosomal rearrangements and subsequent oncogene amplification in the acetamide-induced tumors. Taken together, the results indicate that CAs are directly involved in tumorigenesis through chromosomal rearrangements in an acetamide-induced hepatocarcinogenesis rat model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4992ea68f0d5c57d06c0aa2d2039720d0742395b" target='_blank'>
              Chromosome aberrations cause tumorigenesis through chromosomal rearrangements in a hepatocarcinogenesis rat model.
              </a>
            </td>
          <td>
            Kenji Nakamura, Yuji Ishii, S. Takasu, Moeka Namiki, Meili Soma, N. Takimoto, K. Matsushita, M. Shibutani, Kumiko Ogawa
          </td>
          <td>2024-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="INTRODUCTION
Isodicentric Y chromosomes are relatively common structural variants of the human genome. The underlying mechanism of isodicentric Y chromosomes with short arm breakpoints [idic(Yq)] remains to be clarified.


CASE PRESENTATION
We encountered a Japanese man with azoospermia and mild short stature. G-banding and array-based comparative genomic hybridization indicated that his karyotype was 45,X/46,X,idic(Y)(qter→p11.32::p11.32→qter) with a ~1.8 Mb terminal deletion. Whole genome sequencing suggested that the Y chromosome had four breakpoints in a ~7 kb region of the pseudoautosomal region 1 (PAR1).


CONCLUSION
This case was assumed to have an idic(Yq) resulting from multiple DNA double-strand breaks in PAR1. This rearrangement may have been facilitated by the PAR1-specific chromatin architecture. The clinical features of the patient can be ascribed to SHOX haploinsufficiency and the presence of a 45,X cell line, although copy-number gains of some Yq genes and the size reduction of PAR1 may also contribute to his spermatogenic failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6ffa4553f26785a976ed64f6e1aecf9e157203" target='_blank'>
              Isodicentric Y chromosome with multiple breakpoints in the pseudoautosomal region 1.
              </a>
            </td>
          <td>
            Yasuko Ogiwara, Y. Kobori, E. Suzuki, Atsushi Hattori, Kanako Tanase-Nakao, Akiyoshi Osaka, Toshiyuki Iwahata, Hiroshi Okada, Y. Kuroki, M. Fukami
          </td>
          <td>2024-07-29</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>3</td>
          <td>36</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chromosomal instability (CIN) generates micronuclei—aberrant extranuclear structures that catalyze the acquisition of complex chromosomal rearrangements present in cancer. Micronuclei are characterized by persistent DNA damage and catastrophic nuclear envelope collapse, which exposes DNA to the cytoplasm. We found that the autophagic receptor p62/SQSTM1 modulates micronuclear stability, influencing chromosome fragmentation and rearrangements. Mechanistically, proximity of micronuclei to mitochondria led to oxidation-driven homo-oligomerization of p62, limiting endosomal sorting complex required for transport (ESCRT)–dependent micronuclear envelope repair by triggering autophagic degradation. We also found that p62 levels correlate with increased chromothripsis across human cancer cell lines and with increased CIN in colorectal tumors. Thus, p62 acts as a regulator of micronuclei and may serve as a prognostic marker for tumors with high CIN. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION A main feature of cancer cells is their high frequency of chromosome segregation errors, a condition known as chromosomal instability (CIN), which is associated with poor prognosis and chemoresistance. CIN leads to the formation of micronuclei—abnormal extranuclear bodies widely found in cancer cells. The nuclear envelopes of micronuclei are often fragile and compromised, which causes irreparable ruptures and structural collapse. This exposes micronuclear DNA to the cytosol, resulting in DNA damage and extensive chromosomal rearrangements, which fuels genomic instability and cancer progression. In addition, rupture of the micronuclear membrane activates the cGAS-STING [cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING)] pathway, triggering inflammatory responses that promote tumor invasion and metastasis. RATIONALE Although the consequences of loss of micronuclei compartmentalization are well studied, the events leading to micronuclear envelope rupture and the mechanisms involved in their repair are less clear. Given the importance of those mechanisms in maintaining micronuclei integrity, we aimed to identify proteins and pathways that specifically modulate these events. We isolated micronuclei and primary nuclei, analyzed their protein composition using mass spectrometry, and identified p62/SQSTM1—a key autophagic pathway component—as specifically enriched in micronuclei compared with primary nuclei. RESULTS We found that p62 localizes to micronuclei and is crucial for maintaining their integrity. The localization of p62 to micronuclei relied on its oxidation-driven homo-oligomerization, which was induced by reactive oxygen species (ROS) released by mitochondria proximal to micronuclei. We used electron tomography and quantitative imaging to analyze the proximity of mitochondria to micronuclei and found that p62-positive micronuclei were closer to mitochondria compared with those without p62. We tested how mitochondrial ROS affected p62 localization and found that reducing ROS decreased p62-positive micronuclei and prevented their rupture, whereas increasing ROS led to more p62-positive and ruptured micronuclei in both normal and cancer cells. Homo-oligomerization of p62 induced by ROS influenced micronuclear integrity by inhibiting the repair activity of the endosomal sorting complex required for transport–III (ESCRT-III), which is involved in envelope resealing. This control was achieved through the p62-mediated autophagic degradation of ESCRT components. The p62-dependent regulation of micronuclear integrity affected several features associated with micronuclei, such as chromosomal rearrangements and cGAS-STING–mediated inflammation. Furthermore, p62 levels correlated with chromothripsis status in cancer cells and tumor samples and were strongly linked to poor prognosis in colorectal cancer with high CIN. CONCLUSION In this work, we identified a critical mechanism by which p62 regulates micronuclear stability by influencing envelope repair. Given p62’s frequent deregulation in tumors and its control over micronuclear envelope integrity, our findings suggest that p62-mediated effects on micronuclear repair activities could influence cancer development and progression. p62/SQSTM1 drives micronuclei collapse and complex chromosome rearrangements by restraining micronuclear envelope repair. Micronuclei-mitochondria proximity promotes ROS-mediated homo-oligomerization of p62/SQSTM1 through cysteine oxidation. ROS-induced p62 homo-oligomerization affects micronuclear integrity by inhibiting components of ESCRT-III repair through their autophagic degradation. This affects chromosomal rearrangements and cGAS-STING inflammation. High p62 levels correlate with chromothripsis and poor prognosis in colorectal cancer with high CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/180c1ddc6e5e2000c0d5181084de24b29d56eee4" target='_blank'>
              A p62-dependent rheostat dictates micronuclei catastrophe and chromosome rearrangements
              </a>
            </td>
          <td>
            Sara Martin, Simone Scorzoni, Sara Cordone, Alice Mazzagatti, Galina V Beznoussenko, Amanda L. Gunn, Melody Di Bona, Yonatan Eliezer, Gil Leor, Tal Ben-Yishay, A. Loffreda, V. Cancila, Maria Chiara Rainone, M. Ippolito, Valentino Martis, Fabio Bedin, M. Garrè, L. P. Vaites, Paolo Vasapolli, Simona Polo, D. Parazzoli, C. Tripodo, Alexander A. Mironov, A. Cuomo, Uri Ben-David, S. Bakhoum, Emily M. Hatch, Peter Ly, S. Santaguida
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>3</td>
          <td>36</td>
        </tr>

        <tr id="Odontogenic myxoma is a rare, benign, and locally aggressive tumor that develops in the tooth-bearing areas of the jaw. The molecular mechanisms underlying odontogenic myxomas are unknown and no diagnostic markers are available to date. The aim of this study was to analyze DNA methylation and copy number variations in odontogenic myxomas to identify new molecular signatures for diagnostic decision-making. We collected a cohort of 16 odontogenic myxomas from 2006 to 2021 located in the mandible (n = 10) and maxilla (n = 6) with available formalin-fixed paraffin-embedded or fresh frozen tumor tissue from a biopsy or resection material. Genome-wide DNA methylation and copy number variation data were generated from 12 odontogenic myxomas using the Illumina Infinium Methylation EPIC array, interrogating >850,000 CpG sites. Unsupervised clustering and dimensionality reduction (Uniform Manifold Approximation and Projection) revealed that odontogenic myxomas formed a distinct DNA methylation class. Copy number profiling showed recurrent whole-chromosome gains (trisomies) of chromosomes 5, 8, and 20 in all cases, and of chromosomes 10, 12, and 17 in all except one case. In conclusion, odontogenic myxomas harbor recurrent copy number patterns and a distinct DNA methylation profile, which can be used as an additional diagnostic tool in the appropriate clinical and radiologic context. Further research is needed to explain the genetic mechanisms caused by these alterations that drive these locally aggressive neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf0f96ab1881b02373738687aea96a145a481bc8" target='_blank'>
              Odontogenic Myxomas Harbor Recurrent Copy Number Alterations and a Distinct Methylation Signature.
              </a>
            </td>
          <td>
            Tony G Kleijn, B. Ameline, Willem H Schreuder, W. Kooistra, J. Doff, Max Witjes, S. Pichardo, Tereza Lausová, S. Koppes, Mari F C M van den Hout, Ilse C H van Engen-van Grunsven, Uta Flucke, Jan de Lange, K. Szuhai, I. B. Briaire-de Bruijn, D. Savci‐Heijink, A. Suurmeijer, J. Bovée, Andreas von Deimling, D. Baumhoer, A. Cleven
          </td>
          <td>2024-10-01</td>
          <td>The American journal of surgical pathology</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="CIZ1 is part of the large RNA-dependent supramolecular assemblies that form around the inactive X-chromosome (Xi) in female cells, and smaller assemblies throughout the nucleus in males and females. Here we show that CIZ1 C-terminal anchor domain (AD) is elevated in primary human breast tumour transcriptomes, even at stage I. Elevation of AD correlates with deprotection of chromatin, and up-regulation of lncRNA-containing gene clusters in approximately 10Mb regions that are enriched in cancer-associated genes. We modelled the effect of ectopic AD on endogenous CIZ1-Xi assemblies and observed dominant-negative interference with their re-formation after mitosis, leading to abnormal Xi assemblies similar to those in breast cancer cells, and depletion of histone modifications H2AK119ub1 and H3K27me3. Within days consistent alterations in gene expression were evident across the genome, showing that disruption of CIZ1-RNA assemblies has a destabilizing effect, likely by unscheduled exposure of underlying chromatin to modifying enzymes. Together the data argue for a dominant, potent and rapid effect of CIZ1 AD, that can deprogram established patterns of gene expression and which may predispose incipient tumours to epigenetic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44d2a58fd6870a0dd51c6ac1e82db8148da5b9e2" target='_blank'>
              Epigenetic deprogramming driven by disruption of CIZ1-RNA nuclear assemblies in early-stage breast cancers
              </a>
            </td>
          <td>
            Gabrielle L. Turvey, Ernesto López de Alba, Emma R Stewart, Heather Cook, Ahmad Alalti, Richard T. Gawne, J. Ainscough, Andrew S. Mason, Dawn Coverley
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly toxic lesions that underly the efficacy of ionizing radiation (IR) and a large number of cytotoxic chemotherapies1–3. Yet, abnormal repair of DSBs is associated with genomic instability and may contribute to cancer heterogeneity and tumour evolution. Here, we show that DSBs induced by IR, by DSB-inducing chemotherapeutics, or by the expression of a rare-cutting restriction endonuclease induce large-scale genomic amplification in human cancer cells. Importantly, the extent of DSB-induced genomic amplification (DIGA) in a panel of melanoma cell lines correlated with the degree of cytotoxicity elicited by IR, suggesting that DIGA contributes significantly to DSB-induced cancer cell lethality. DIGA, which is mediated through conservative DNA synthesis, does not require origin re-licensing, and is enhanced by the depletion or deletion of the methyltransferases SET8 and SUV4-20H1, which function sequentially to mono- and di-methylate histone H4 lysine 20 (H4K20) at DSBs to facilitate the recruitment of 53BP1-RIF1 and its downstream effector shieldin complex to DSBs to prevent hyper-resection4–11. Consistently, DIGA was enhanced in cells lacking 53BP1 or RIF1, or in cells that lacked components of the shieldin complex or of other factors that help recruit 53BP1 to DSBs. Mechanistically, DIGA requires MRE11/CtIP and EXO1, factors that promote resection and hyper-resection at DSBs, and is dependent on the catalytic activity of the RAD51 recombinase. Furthermore, deletion or depletion of POLD3, POLD4, or RAD52, proteins involved in break-induced replication (BIR), significantly inhibited DIGA, suggesting that DIGA is mediated through a RAD51-dependent BIR-like process. DIGA induction was maximal if the cells encountered DSBs in early and mid S-phase, whereas cells competent for homologous recombination (in late S and G2) exhibited less DIGA induction. We propose that unshielded, hyper-resected ends of DSBs may nucleate a replication-like intermediate that enables cytotoxic long-range genomic DNA amplification mediated through BIR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec06bf7ab705064fc487b428fdb7631566b4ae0f" target='_blank'>
              Break-induced replication drives large-scale genomic amplifications in cancer cells
              </a>
            </td>
          <td>
            Mouadh Benamar, Rebeka Eki, Kangping Du, Tarek Abbas
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Introduction Type B1 and B2 thymomas are lymphocyte-rich malignant tumors with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, non-coding regions remain uncharacterized. Here, we developed a rigorous tumor isolation method from lymphocyte-rich thymoma tissues and identified somatic mutations in non-coding and mitochondrial DNA. Methods CD205+CD45- pure tumor cells were isolated from fresh-frozen tissues using DEPArray system. Deep whole-genome sequencing was performed, and recurrent somatic alterations in coding, non-coding, and mitochondria regions were systemically identified by computational framework. The mutations were classified according to gene function, cis-regulatory element, and mutational signature. Results The total number of somatic mutations was approximately 80 times higher in non-coding regions than in coding regions in type B1-2 thymomas (1,671.3 vs. 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of cases exhibited fewer than four mutations in coding regions. A systematic non-coding analysis identified a total of 405.0 mutations per case on cis-regulatory elements, and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Mitochondrial heteroplasmy was observed in 90% of cases, with a significant proportion of mutations located in D-loop region. The single-base substitution pattern was signature 12. Conclusions Numerous non-coding mutations and mitochondrial heteroplasmy were detected in type B1 and B2 thymomas. Given the paucity of coding mutations observed in this disease entity, disruption of the non-coding landscape and mitochondrial heteroplasmic shift may be the primary cause of thymoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/559b9056aa5e8bd421441f43ad9a55ae5b38dbc9" target='_blank'>
              Genome-wide analysis of somatic non-coding mutation patterns and mitochondrial heteroplasmy in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Juliet King, Emma McLean, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Regulating gene expression involves significant and frequent changes in the chromatin environment at the locus level, especially at regulatory sequences. However, their modulation in response to pharmacological treatments or pathological conditions remain mostly undetermined. Here, we report versatile locus-specific proteomics tools to address this knowledge gap, which combine the targeting ability of the CRISPR/Cas9 system and the protein-labelling capability of the highly reactive biotin ligases TurboID (in CasTurbo) and UltraID (in CasUltra). CasTurbo and CasUltra enabled rapid chromatin protein labelling under mild conditions at repetitive sequences like centromeres and telomeres, as well as non-amplified genes. We applied CasUltra to A375 melanoma cell lines to decipher the protein environment of the MYC promoter and characterize the molecular effects of the bromodomain inhibitor JQ1, which targets bromodomain and extra-terminal (BET) proteins that regulate MYC expression. We quantified the consequences of BET protein displacement from the MYC promoter and found that it was associated with a considerable reorganisation of the chromatin composition. In addition, BET protein retention at the MYC promoter was consistent with a model of increased JQ1 resistance. Thus, through the combination of proximity biotinylation and CRISPR-Cas9-dependent genomic targeting, CasTurbo and CasUltra have successfully demonstrated their utility in profiling the proteome associated with a genomic locus in living cells. In Brief Kougnassoukou Tchara et al. report the development and application of CasTurbo and CasUltra, two locus-specific proteomics tools that fuse catalytically dead Cas9 to the engineered biotin ligases TurboID and UltraID. These tools enabled the quantitative mapping of locus-specific chromatin remodelling due to pharmacological inhibition. Highlights CasTurbo and CasUltra were developed for locus-specific label-free proteomics CasTurbo mapped the proteins localized to the centromeres and telomeres Proteins bound to the MYC promoter were quantified in melanoma cells with CasUltra CasUltra is compatible with investigating pharmacological treatment effects Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce2d069fe2369eb465a0930b8b4ef5639c3f142c" target='_blank'>
              Coupling proximity biotinylation with genomic targeting to characterize locus-specific changes in chromatin environments
              </a>
            </td>
          <td>
            Pata-Eting Kougnassoukou Tchara, J. Loehr, Jean-Philippe Lambert
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="RNA polymerase II (Pol II) dysfunction is frequently implied in human disease. Understanding its functional mechanism is essential for designing innovative therapeutic strategies. To visualize its supra-molecular interactions with genes and nascent RNA, we generated a human cell line carrying ~335 consecutive copies of a recombinant β-globin gene. Confocal microscopy showed that Pol II was not homogeneously concentrated around these identical gene copies. Moreover, Pol II signals partially overlapped with the genes and their nascent RNA, revealing extensive compartmentalization. Using a cell line carrying a single copy of the β-globin gene, we also tested if the binding of catalytically dead CRISPR-associated system 9 (dCas9) to different gene regions affected Pol II transcriptional activity. We assessed Pol II localization and nascent RNA levels using chromatin immunoprecipitation and droplet digital reverse transcription PCR, respectively. Some enrichment of transcriptionally paused Pol II accumulated in the promoter region was detected in a strand-specific way of gRNA binding, and there was no decrease in nascent RNA levels. Pol II preserved its transcriptional activity in the presence of DNA-bound dCas9. Our findings contribute further insight into the complex mechanism of mRNA transcription in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b25d997a1002918d7b323e2e8364e23e9d3a4ab" target='_blank'>
              Human RNA Polymerase II Segregates from Genes and Nascent RNA and Transcribes in the Presence of DNA-Bound dCas9
              </a>
            </td>
          <td>
            João Pessoa, C. Carvalho
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e11890b80227e402b42551347179de59480a8fc" target='_blank'>
              HMGA1 orchestrates chromatin compartmentalization and sequesters genes into 3D networks coordinating senescence heterogeneity
              </a>
            </td>
          <td>
            Ioana Olan, Masami Ando-Kuri, Aled J. Parry, Testuya Handa, S. Schoenfelder, Peter Fraser, Yasuyuki Ohkawa, Hiroshi Kimura, M. Narita, Masashi Narita
          </td>
          <td>2024-08-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromatin is not randomly distributed within the nucleus, but organized in a three-dimensional structure that plays a critical role in genome functions. Сohesin and condensins are conserved multi-subunit protein complexes that participate in mammalian genome organization by extruding chromatin loops. The fine temporal regulation of these complexes is facilitated by a number of other proteins, one of which is microcephalin (Mcph1). Mcph1 prevents condensin II from associating with chromatin through interphase. Loss of Mcph1 induces chromosome hypercondensation; it is not clear to what extent this reorganization affects gene expression. In this study, we generated several mouse embryonic stem cell (mESC) lines with knockout of the Mcph1 gene and analyzed their gene expression profile. Gene Ontology analyses of differentially expressed genes (DEGs) after Mcph1 knockout revealed gene categories related to general metabolism and olfactory receptor function but not to cell cycle control previously described for Mcph1. We did not find a correlation between the DEGs and their frequency of lamina association. Thus, this evidence questions the hypothesis that Mcph1 knockout-mediated chromatin reorganization governs gene expression in mESCs. Among the negative effects of Mcph1 knockout, we observed numerous chromosomal aberrations, including micronucleus formation and chromosome fusion. This confirms the role of Mcph1 in maintaining genome integrity described previously. In our opinion, dysfunction of Mcph1 may be a kind of “Rosetta stone” for deciphering the function of condensin II in the interphase nucleus. Thus, the cell lines with knocked-out Mcph1 can be used to further study the influence of chromatin structural proteins on gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34d65d4d4365d60abd6b73f946f30c7616d79df9" target='_blank'>
              Generation and analysis of mouse embryonic stem cells with knockout of the Mcph1 (microcephalin) gene
              </a>
            </td>
          <td>
            A. Yunusova, A. V. Smirnov, T. Shnaider, I. Pristyazhnuk, S.Y. Korableva, N. R. Battulin
          </td>
          <td>2024-09-01</td>
          <td>Vavilov Journal of Genetics and Breeding</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0943aff1f710e095be6a9439cd65322a1fc6aa0e" target='_blank'>
              Genomic Balancing Act: deciphering DNA rearrangements in the complex chromosomal aberration involving 5p15.2, 2q31.1, and 18q21.32.
              </a>
            </td>
          <td>
            Zain Dardas, Dana Marafi, Ruizhi Duan, Jawid M Fatih, O. El-Rashidy, Christopher M. Grochowski, Claudia M B Carvalho, S. Jhangiani, W. Bi, Haowei Du, Richard A Gibbs, J. Posey, D. Calame, Maha S. Zaki, James R. Lupski
          </td>
          <td>2024-09-10</td>
          <td>European journal of human genetics : EJHG</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="Upon infection, human papillomavirus (HPV) manipulates host cell gene expression to create an environment that is supportive of a productive and persistent infection. The virus-induced changes to the host cell’s transcriptome are thought to contribute to carcinogenesis. Here, we show by RNA-sequencing that oncogenic HPV18 episome replication in primary human foreskin keratinocytes (HFKs) drives host transcriptional changes that are consistent between multiple HFK donors. We have previously shown that HPV18 episome replication in HFKs results in post-transcriptional stabilisation of the host chromatin insulation protein CTCF. Since CTCF is an important regulator of host cell transcription via coordination of epigenetic boundaries and long-range chromosomal interactions, we hypothesised that HPV18-induced stabilisation of CTCF may contribute to host transcription reprogramming. Analysis of CTCF binding in the host cell genome by ChIP-Seq revealed that while the total number of CTCF binding sites is not altered by the virus, there are a sub-set of CTCF binding sites that are either enriched or depleted of CTCF. Many of these altered sites are clustered within regulatory elements of differentially expressed genes, including the tumour suppressor gene cell adhesion molecule 1 (CADM1), which supresses epithelial cell growth and invasion. We show that HPV18 establishment results in reduced CTCF binding at the CADM1 promoter and upstream enhancer. Loss of CTCF binding is coincident with epigenetic repression of CADM1, in the absence of CpG hypermethylation, while adjacent genes including the transcriptional regulator ZBTB16 are activated. These data indicate that the CADM1 locus is subject to topological rearrangement following HPV18 establishment. We tested this hypothesis using 4C-Seq (circular chromosome confirmation capture-sequencing) and show that HPV18 establishment causes a loss of long-range chromosomal interactions between the CADM1 transcriptional start site and the upstream transcriptional enhancer. These data show that HPV18 manipulates host cell promoter-enhancer interactions to drive transcriptional reprogramming that may contribute to HPV-induced disease progression. AUTHOR SUMMARY Infection with oncogenic HPV is the cause of numerous cancer types, which generally arise after persistent HPV infection. Upon infection, HPV alters the gene expression profile of infected cells to facilitate virus replication and persistence. Multiple mechanisms of HPV-induced host cell reprogramming have been previously suggested. Here, we show that HPV infection induces rearrangement of specific genomic loci by altering the chromatin binding of the host cell protein CTCF, an important regulator of chromatin architecture. Loss of CTCF binding to a cluster of binding sites at the CADM1 locus on chromosome 11 is coincident with epigenetic reprogramming and disruption of long-range chromatin interactions, resulting in transcriptional repression of CADM1. Our data show that repression of CADM1 is an early event in HPV-driven disease, preceding hypermethylation of the CADM1 transcriptional promoter that is frequently observed in HPV-driven cancers, demonstrating a novel mechanism of HPV-induced host cell transcriptional reprogramming.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/296755d464d45666368bedcd29a51e084375e664" target='_blank'>
              Repression of CADM1 transcription by HPV type 18 is mediated by three-dimensional rearrangement of promoter-enhancer interactions
              </a>
            </td>
          <td>
            Karen Campos-León, J. Ferguson, Thomas Günther, C. D. Wood, S. Wingett, Selin Pekel, C. S. Varghese, C. Várnai, Michelle West, Andrew Beggs, A. Grundhoff, B. Noyvert, Sally Roberts, Joanna L. Parish
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Background Approximately half of all high-grade serous ovarian carcinomas (HGSCs) have a therapeutically targetable defect in the homologous recombination (HR) DNA repair mechanism. While there are genomic and transcriptomic methods, developed for other cancer types, to identify HR deficient (HRD) samples, there are no gene expression-based tools to predict HR repair status in HGSC specifically. We have built the first HGSC-specific model to predict HR repair status using gene expression. Methods We separated The Cancer Genome Atlas (TCGA) cohort of HGSCs (n = 361) into training (n = 288) and testing (n = 73) sets and labelled each case as HRD or HR proficient (HRP) based on the clinical standard for classification, being a score of HRD genomic damage. Using the training set, we performed differential gene expression analysis between HRD and HRP cases. The 2604 significantly differentially expressed genes were then used to tune and train a penalised logistic regression model. Results IdentifiHR is an elastic net penalised logistic regression model that uses the expression of 209 genes to predict HR status in HGSC. These genes capture known regions of HR-specific copy number alteration, which impact gene expression levels, and preserve the genomic damage signal. IdentifiHR has an accuracy of 85% in the TCGA test set and of 91% in an independent cohort of 99 samples, collected from primary tumours before (n = 74/99) and after autopsy (n = 6/99), in addition to ascites (n = 12/99) and normal fallopian tube samples (n = 7/99). Further, IdentifiHR is 84% accurate in pseudobulked single-cell HGSC sequencing from 37 patients and outperforms existing gene expression-based methods to predict HR status, being BRCAness, MutliscaleHRD and expHRD. Conclusions IdentifiHR is an accurate model to predict HR status in HGSC using gene expression alone, that is available as an R package from https://github.com/DavidsonGroup/IdentifiHR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03bb043799881c3a8ed9a4b69ffa433b94653148" target='_blank'>
              IdentifiHR: predicting homologous recombination deficiency in high-grade serous ovarian carcinoma using gene expression
              </a>
            </td>
          <td>
            Ashley L. Weir, Samuel C. Lee, Mengbo Li, Chin Wee Tan, S. Ramus, Nadia M. Davidson
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The three-dimensional (3D) organization of chromatin within the nucleus is crucial for gene regulation. However, the 3D architectural features that coordinate the activation of an entire chromosome remain largely unknown. We introduce an omics method, RNA-associated chromatin DNA-DNA interactions, that integrates RNA polymerase II (RNAPII)–mediated regulome with stochastic optical reconstruction microscopy to investigate the landscape of noncoding RNA roX2-associated chromatin topology for gene equalization to achieve dosage compensation. Our findings reveal that roX2 anchors to the target gene transcription end sites (TESs) and spreads in a distinctive boot-shaped configuration, promoting a more open chromatin state for hyperactivation. Furthermore, roX2 arches TES to transcription start sites to enhance transcriptional loops, potentially facilitating RNAPII convoying and connecting proximal promoter-promoter transcriptional hubs for synergistic gene regulation. These TESs cluster as roX2 compartments, surrounded by inactive domains for coactivation of multiple genes within the roX2 territory. In addition, roX2 structures gradually form and scaffold for stepwise coactivation in dosage compensation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc0d29c3125cfd0f353817880f4b6081993ce9" target='_blank'>
              3D chromatin structures associated with ncRNA roX2 for hyperactivation and coactivation across the entire X chromosome
              </a>
            </td>
          <td>
            S. Tian, Yang Yang, Duo Ning, Ke Fang, Kai Jing, Guangyu Huang, Yewen Xu, Pengfei Yin, Haibo Huang, Gengzhan Chen, Yuqing Deng, Shaohong Zhang, Zhimin Zhang, Zhen-Xia Chen, Tong Gao, Wei Chen, Guoliang Li, Ruilin Tian, Yijun Ruan, Yiming Li, Meizhen Zheng
          </td>
          <td>2024-07-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs) compose approximately 8% of the human genome, co-opted into the dynamic regulatory network of cellular potency in early embryonic development. In recent studies, resurgent HERVs’ transcriptional activity has been frequently observed in many types of human cancers, suggesting their potential functions in the occurrence and progression of malignancy. However, a web resource dedicated to querying the relationship between activation of HERVs and cancer development is lacking. Here, we have constructed a database to explore the sequence information, expression profiles, survival prognosis, and genetic interactions of HERVs in diverse cancer types. Our database currently incorporates RNA sequencing (RNA-seq) data of 580 HERVs across 16246 samples, comprising 151 early embryonic data from the Gene Expression Omnibus (GEO) database, 8051 human adult tissues’ data from the Genotype-Tissue Expression (GTEx) project, 932 cancer cell lines’ data from the Cancer Cell Line Encyclopedia (CCLE) project, 6478 tumoral and 634 normal tissue samples’ data from The Cancer Genome Atlas (TCGA) project. The primary goal is to provide an easily accessible and user-friendly database for professionals in the fields of bioinformatics, pathology, pharmacology, and related areas, enabling them to efficiently screen the activity of HERVs of interest in normal and cancerous tissues and evaluate the clinical relevance. The ERVcancer database is available at http://kyuanlab.com/ervcancer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a360157cc339673766d7e394421cf5342d19b04" target='_blank'>
              ERVcancer: a web resource designed for querying activation of human endogenous retroviruses across major cancer types
              </a>
            </td>
          <td>
            Xiaoyun Lei, Song Mao, Yinshuang Li, Shi Huang, Jinchen Li, Wei Du, Chunmei Kuang, Kai Yuan
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="It has been observed that men have a shorter lifespan and higher incidence of certain cancers than women. We postulate here that this phenomenon may be attributed to the “toxic Y chromosome”, owing to the expression of normally silent HSATII repeats in cancer, abundantly present on chromosome Y. Investigating the contribution of all repetitive elements to chromatin interactions using a genome-wide HiC set derived from a colon cancer cell line (HCT116), we found several anomalies regarding satellite HSATII elements when compared to other genomic repeat types: (i) HSATII repeats tend to form HiC pairs mostly with their own kind, and depleted in most heterologous repeat pairs, whereas other types of repeats readily form heterologous HiC pairs; (ii) when treated with 5aza, a chemotherapeutic agent, the number of long-distance HSATII-overlapping HiC pairs significantly decreased when compared to the untreated samples; (iii) and most importantly, in the treated samples interchromosomal HiC read pairs involving a chromosome Y-located repeat and an HSATII repeat on either end almost completely disappear. (iv) We also found that HSATII repeats lack overlap with other types of repeats. To account for these anomalies we queried non-B DNA structure forming such as G-quadruplexes and transcription-related triplex forming. We found that Y chromosome-coded HSATII repeats form significantly stronger triplexes than HSATII repeats on other chromosomes, their strength significantly correlating with the number of HiC reads they overlap with. We surmise that the above phenomena may cause the higher incidence of cancer in men and may lead to the disappearance of the Y chromosome observed in cancerous cells in men.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53990a3a3a681b314979b859a2d2941c439f782" target='_blank'>
              Y chromosome-coded HSATII repeats may contribute to higher incidence of cancer in men
              </a>
            </td>
          <td>
            Hedi Hegyi, Matej Lexa
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d63fe0e1a04ed83bd0b4c69b42c0d270765edd" target='_blank'>
              Human cytomegalovirus harnesses host L1 retrotransposon for efficient replication
              </a>
            </td>
          <td>
            Sung-Yeon Hwang, Hyewon Kim, Danielle Denisko, Boxun Zhao, Dohoon Lee, Jiseok Jeong, Jinuk Kim, Kiwon Park, Junhyun Park, D. Jeong, Sehong Park, Hee-Jung Choi, Sun Kim, E. A. Lee, Kwangseog Ahn
          </td>
          <td>2024-09-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Centrosomal Protein 55 (CEP55) exhibits various oncogenic activities; it regulates the PI3K-Akt-pathway, midbody abscission, and chromosomal instability (CIN) in cancer cells. Here, we analyzed the mechanism of how CEP55 controls CIN in ovarian and breast cancer (OvCa) cells. Down-regulation of CEP55 reduced CIN in all cell lines analyzed, and CEP55 depletion decreased spindle microtubule (MT)-stability in OvCa cells. Moreover, recombinant CEP55 accelerated MT-polymerization and attenuated cold-induced MT-depolymerization. To analyze a potential relationship between CEP55-controlled CIN and its impact on MT-stability, we identified the CEP55 MT-binding peptides inside the CEP55 protein. Thereafter, a mutant with deficient MT-binding activity was re-expressed in CEP55-depleted OvCa cells and we could show that this mutant did not restore reduced CIN in CEP55-depleted cells. This finding strongly indicates that CEP55 regulates CIN by controlling MT dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43711b62c2fee72142dde9b86575c5f9c1c27732" target='_blank'>
              Centrosomal Protein 55 Regulates Chromosomal Instability in Cancer Cells by Controlling Microtubule Dynamics
              </a>
            </td>
          <td>
            S. Muhs, Themistoklis Paraschiakos, Paula Schäfer, S. Joosse, S. Windhorst
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c4149f430cd50897e383d0f368cb3d1c8239aed" target='_blank'>
              Distinct dynamics of parental 5-hydroxymethylcytosine during human preimplantation development regulate early lineage gene expression
              </a>
            </td>
          <td>
            Dan Liang, Rui Yan, Xin Long, Dongmei Ji, Bing Song, Mengyao Wang, Fan Zhang, Xin Cheng, Fengyuan Sun, Ran Zhu, Xinling Hou, Tianjuan Wang, Weiwei Zou, Ying Zhang, Zhixin Pu, Jing Zhang, Zhiguo Zhang, Ya-Jing Liu, Yuqiong Hu, Xiaojin He, Yunxia Cao, Fan Guo
          </td>
          <td>2024-07-30</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c120e2af47610364790b472bc22133f0b2b21b" target='_blank'>
              Complex genomic rearrangements of the Y chromosome in a premature infant
              </a>
            </td>
          <td>
            Stephanie A. Balow, Alyxis Coyan, Nicki Smith, Bianca E Russell, Danielle Monteil, Robert J. Hopkin, Teresa A. Smolarek
          </td>
          <td>2024-08-26</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve M. Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Near-haploidization, that is, loss of one copy of most chromosomes, is a relatively rare phenomenon in most tumors, but is enriched among certain soft tissue sarcomas, including undifferentiated pleomorphic sarcoma (UPS). Presumably, near-haploidization can arise through many mechanisms. This study aimed to identify gene rearrangements that could cause near-haploidization. We here present two UPS in which near-haploidization was an early event, identified through single nucleotide polymorphism (SNP) array analysis. One of the cases was studied further using whole genome and transcriptome sequencing, as well as cytogenetic and molecular cytogenetic methods. Both tumors had chromosomal rearrangements in the form of copy number shifts/structural variants affecting the SMC1A gene. These findings suggest that cohesin defects could contribute to mitotic errors resulting in massive loss of chromosomes. SMC1A encodes one of the components of the cohesin multiprotein complex, which is critical for proper alignment of the sister chromatids during S-phase and separation to opposite spindle poles. Further studies should explore the role of cohesin defects in near-haploidization in other sarcomas and to clarify its role in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac46389944e17adf0a38d1c92f41322a72ad7ff" target='_blank'>
              Structural Variants in the SMC1A Gene Associated With Near-Haploidy in Undifferentiated Pleomorphic Sarcomas.
              </a>
            </td>
          <td>
            Sebastian Ibstedt, Paul Piccinelli, Saskia Sydow, Jan Köster, Fredrik Mertens
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The nuclear genome is spatially organized into a three-dimensional (3D) architecture by physical association of large chromosomal domains with subnuclear compartments including the nuclear lamina at the radial periphery and nuclear speckles within the nucleoplasm1–5. However, how spatial genome architecture regulates human brain development has been overlooked owing to technical limitations. Here, we generate high-resolution maps of genomic interactions with the lamina and speckles in cells of the neurogenic lineage isolated from midgestational human cortex, uncovering an intimate association between subnuclear genome compartmentalization, chromatin state and transcription. During cortical neurogenesis, spatial genome organization is extensively remodeled, relocating hundreds of neuronal genes from the lamina to speckles including key neurodevelopmental genes bivalent for H3K27me3 and H3K4me3. At the lamina, bivalent genes have exceptionally low expression, and relocation to speckles enhances resolution of bivalent chromatin to H3K4me3 and increases transcription >7-fold. We further demonstrate that proximity to the nuclear periphery – not the presence of H3K27me3 – is the dominant factor in maintaining the lowly expressed, poised state of bivalent genes embedded in the lamina. In addition to uncovering a critical role of subnuclear genome compartmentalization in neurogenic transcriptional regulation, our results establish a new paradigm in which knowing the spatial location of a gene is necessary to understanding its epigenomic regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c884553e9ec75de899ee78b7d899ff800792f997" target='_blank'>
              Spatial 3D genome organization controls the activity of bivalent chromatin during human neurogenesis
              </a>
            </td>
          <td>
            S. Ahanger, Chujing Zhang, Evan R. Semenza, Eugene Gil, Mitchel A. Cole, Li Wang, Arnold R. Kriegstein, Daniel A. Lim
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome integrity is safeguarded by the DNA damage response (DDR). Controlled transcriptional dampening of genes surrounding DNA double-strand breaks (DSBs) has been shown to facilitate DNA repair. This phenomenon, defined as DSB-induced silencing in cis (DISC), involves the DDR apical kinase ATM and the Polycomb Repressive Complex 1 (PRC1). Conversely, DSBs have also been reported to induce de novo transcription of damaged-induced long non-coding RNAs (dilncRNAs) in a MRE11-RAD50-NBS1 (MNR) complex-dependent manner. MRN also controls the recruitment to DSB of the ribonuclease DROSHA, which together with DICER, stimulates DDR signaling and DNA repair. Here, we reconcile these apparently contrasting observations by showing that dilncRNA, together with DROSHA and DICER, but not GW182-like proteins required for miRNA-mediated gene silencing, controls DISC. Indeed, similarly to ATM, MRN inhibition abolishes DISC while pharmacological enhancement of DICER ribonuclease activity by Enoxacin improves DISC. Importantly, Enoxacin administration restores DISC upon ATM inhibition, demonstrating that DICER promotes DISC independently from ATM. Differently, Enoxacin does not restore DISC upon MRN inhibition, suggesting that DICER acts downstream to dilncRNA biogenesis and DROSHA recruitment. Mechanistically, we show that DROSHA and DICER control the recruitment of the PRC1 component BMI1 at DSBs and the consequent H2A-K119 ubiquitination. Upon DSBs formation, BMI1 and DROSHA interact in an RNA-dependent manner. Indeed, BMI1 associates to dilncRNA and do so in a DROSHA- and DICER-dependent manner. Importantly, inhibition of dilncRNA function by antisense oligonucleotides or Cas13-mediated targeting is sufficient to reduce BMI1 recruitment and DISC at individual loci. We propose that dilncRNAs together with DROSHA and DICER control DISC at genomic DSB by supporting PRC1 recruitment and chromatin ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbe46331f4c7eaaba94ede95e2ee9569f12ce8df" target='_blank'>
              DROSHA, DICER and Damage-Induced long ncRNA control BMI1-dependent transcriptional repression at DNA double-strand break
              </a>
            </td>
          <td>
            Francesca Esposito, Ilaria Capozzo, Stefania Modafferi, Ubaldo Gioia, Letizia Manfredi, F. Iannelli, Alessio Colantoni, Adelaide Riccardi, Alessia di Lillo, Sara Tavella, Matteo Cabrini, F. d’Adda di Fagagna, S. Francia
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="In cancer cells, ATR signaling is crucial to tolerate the intrinsically high damage levels that normally block replication fork progression. Assembly of TopBP1, a multifunctional scaffolding protein, into condensates is required to amplify ATR kinase activity to the levels needed to coordinate the DNA damage response and manage DNA replication stress. Many ATR inhibitors are tested for cancer treatment in clinical trials, but their overall effectiveness is often compromised by the emergence of resistance and toxicities. In this proof-of-concept study, we propose to disrupt the ATR pathway by targeting TopBP1 condensation. First, we screened a molecule-based library using a previously developed optogenetic approach and identified several TopBP1 condensation inhibitors. Amongst them, AZD2858 disrupted TopBP1 assembly induced by the clinically relevant topoisomerase I inhibitor SN-38, thereby inhibiting the ATR/Chk1 signaling pathway. We found that AZD2858 exerted its effects by disrupting TopBP1 self-interaction and binding to ATR in mammalian cells, and by increasing its chromatin recruitment in cell-free Xenopus laevis egg extracts. Moreover, AZD2858 prevented S-phase checkpoint induction by SN-38, leading to increased DNA damage and apoptosis in a colorectal cancer cell line. Lastly, AZD2858 showed synergistic effect in combination with the FOLFIRI chemotherapy regimen in a spheroid model of colorectal cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8d12d939b21bc8eddc64b81e916d3c7019a271" target='_blank'>
              TopBP1 biomolecular condensates: a new therapeutic target in advanced-stage colorectal cancer
              </a>
            </td>
          <td>
            Laura Morano, Nadia Vezzio-Vié, Adam Aissanou, Tom Egger, Antoine Aze, Solène Fiachetti, Hervé Seitz, Louis-Antoine Milazzo, V. Garambois, Nathalie Bonnefoy, Céline Gongora, A. Constantinou, J. Basbous
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Animal chromosomes can persist with recognizable homology over hundreds of millions of years, in spite of homology-obfuscating processes such as chromosomal fusion and translocation. The frequency and pace of these major genome structural changes are unknown, and it remains unclear whether or how they impact long-term genome evolution. Here, we compare whole chromosomal sequences of 3,631 genomes from 2,291 species spanning all major animal clades and show that animal karyotypes evolve primarily via karyotype contraction, associated with increased rates of chromosomal fusion-with-mixing and dispersion that largely obey chromosomal algebra1, or karyotype expansion, which breaks up ancestral linkage groups and forms new chromosomal elements via non-algebraic changes. We show that chromosomal changes can be associated with major extinction events. Using a multi-scale encoding of pan-animal genome homology and a manifold representation of genomic changes, we find that genome evolution is not only driven by changes at the chromosomal level, but that subchromosomal mixing and irreversibility define clade-specific evolution. Using this ‘evolutionary genome topology’ approach, we calculate extrema of irreversible genomic configurations and identify species that occupy intermediate manifold positions, providing evidence for distinct macro-evolutionary trajectories. We propose that investigation of mixed state accumulation around important gene loci (such as Hox) will be crucial in capturing and further study of clade-specific regulatory innovations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bcfa759f5e30b92ec5cb430b3076d461c210e1e" target='_blank'>
              Topological mixing and irreversibility in animal chromosome evolution
              </a>
            </td>
          <td>
            D. Schultz, Arno Blümel, Dalila Destanović, Fatih Sarigol, Oleg Simakov
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The myeloma overexpressed gene (MYEOV) has been proposed to be a proto-oncogene due to high RNA transcript levels found in multiple cancers, including myeloma, breast, lung, pancreas and esophageal cancer. The presence of an open reading frame (ORF) in humans and other primates suggests protein-coding potential. Yet, we still lack evidence of a functional MYEOV protein. It remains undetermined how MYEOV overexpression affects cancerous tissues. In this work, we show that MYEOV has likely originated and may still function as an enhancer, regulating CCND1 and LTO1. Firstly, MYEOV 3′ enhancer activity was confirmed in humans using publicly available ATAC-STARR-seq data, performed on B-cell-derived GM12878 cells. We detected enhancer histone marks H3K4me1 and H3K27ac overlapping MYEOV in multiple healthy human tissues, which include B cells, liver and lung tissue. The analysis of 3D genome datasets revealed chromatin interactions between a MYEOV-3′-putative enhancer and the proto-oncogene CCND1. BLAST searches and multi-sequence alignment results showed that DNA sequence from this human enhancer element is conserved from the amphibians/amniotes divergence, with a 273 bp conserved region also found in all mammals, and even in chickens, where it is consistently located near the corresponding CCND1 orthologues. Furthermore, we observed conservation of an active enhancer state in the MYEOV orthologues of four non-human primates, dogs, rats, and mice. When studying this homologous region in mice, where the ORF of MYEOV is absent, we not only observed an enhancer chromatin state but also found interactions between the mouse enhancer homolog and Ccnd1 using 3D-genome interaction data. This is similar to the interaction observed in humans and, interestingly, coincides with CTCF binding sites in both species. Taken together, this suggests that MYEOV is a primate-specific gene with a de novo ORF that originated at an evolutionarily older enhancer region. This deeply conserved putative enhancer element could regulate CCND1 in both humans and mice, opening the possibility of studying MYEOV regulatory functions in cancer using non-primate animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f17ce82b4897c471d6bebe571f550d964a45b5c" target='_blank'>
              Evolutionarily conserved enhancer-associated features within the MYEOV locus suggest a regulatory role for this non-coding DNA region in cancer
              </a>
            </td>
          <td>
            Brigid S. A. Davidson, Juliana Estefanía Arcila-Galvis, Marco Trevisan-Herraz, Aneta Mikulasova, C. Brackley, Lisa J Russell, Daniel Rico
          </td>
          <td>2024-07-30</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSCs) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single-nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosome 20q or 1q, both known for conferring an in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings showed higher loads of both CNVs and SNVs at later passages, which were due to the cumulative acquisition of mutations over a longer time in culture, and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs followed the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid of cancer-associated mutations. This suggests that recurrent chromosomal abnormalities are potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36cc208bebf01a60a96b6bcb671a8cdcf9d8bbf6" target='_blank'>
              De Novo Cancer Mutations Frequently Associate with Recurrent Chromosomal Abnormalities during Long-Term Human Pluripotent Stem Cell Culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huyghebaert, M. Regin, Mai Chi Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Although the role of G-quadruplex (G4) DNA structures has been suggested in chromosomal looping this was not tested directly. Here, to test causal function, an array of G4s, or control sequence that does not form G4s, were inserted within chromatin in cells. In vivo G4 formation of the inserted G4 sequence array, and not the control sequence, was confirmed using G4-selective antibody. Compared to the control insert, we observed a remarkable increase in the number of 3D chromatin looping interactions from the inserted G4 array. This was evident within the immediate topologically associated domain (TAD) and throughout the genome. Locally, recruitment of enhancer histone marks and the transcriptional coactivator p300/Acetylated-p300 increased in the G4-array, but not in the control insertion. Resulting promoter-enhancer interactions and gene activation were clear up to 5 Mb away from the insertion site. Together, these show the causal role of G4s in enhancer function and long-range chromatin interactions. Mechanisms of 3D topology are primarily based on DNA-bound architectural proteins that induce/stabilize long-range interactions. Involvement of the underlying intrinsic DNA sequence/structure in 3D looping shown here therefore throws new light on how long-range chromosomal interactions might be induced or maintained.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c4a14fedf36592373d312c9d23b12df8fd19b77" target='_blank'>
              Artificially inserted strong promoter containing multiple G-quadruplexes induces long-range chromatin modification
              </a>
            </td>
          <td>
            S. Roy, Sulochana Bagri, Soujanya Vinayagamurthy, Avik Sengupta, Claudia Regina Then, Rahul Kumar, Sriram Sridharan, Shantanu Chowdhury
          </td>
          <td>2024-08-19</td>
          <td>eLife</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="A key step of Canonical Nonhomologous End Joining (C-NHEJ) is synapsis of DNA double strand break (DSB) ends for ligation. The DNA-PKcs dimer mediates synapsis in a long-range complex with DSB ends remaining apart, whereas the XLF homodimer can mediate synapsis in both long-range and short-range complexes. Recent structural studies found the PAXX homodimer may also facilitate synapsis in long-range complexes with DNA-PKcs via its interactions with Ku70. Thus, we examined the influence of PAXX in C-NHEJ of chromosomal DSBs, which we compared to another Ku-binding factor, MRI. Using EJ of blunt DSBs with Cas9 reporters as a readout for C-NHEJ, we found that PAXX and/or MRI are dispensable. However, when combined with disruption of DNA-PKcs, particularly with DNA-PKcs kinase inhibition, PAXX becomes important for blunt DSB EJ. In contrast, while DNA-PKcs is also important to suppress short deletion mutations with microhomology, this effect is not magnified with PAXX loss. MRI loss had no effect combined with DNA-PKcs disruption, but becomes important for blunt DSB EJ when combined with disruption of XLF, as is PAXX. Finally, XLF loss causes an increase in larger deletions compared to DNA-PKcs inhibition, which is magnified with combined loss of MRI. Altogether, we suggest that PAXX promotes DSB end synapsis during C-NHEJ in a manner that is partially redundant with DNA-PKcs and XLF, whereas MRI appears to be mainly important in the context of XLF disruption. SIGNIFICANCE STATEMENT Canonical Nonhomologous End Joining (C-NHEJ) is a DNA double strand break repair pathway that can repair diverse types of DNA ends, but is prone to causing genetic mutations. In particular, C-NHEJ can cause insertion or deletion mutations that can cause human disease. Therefore, it is critical to understand the mechanisms of C-NHEJ, including and how various factors within the pathway affect repair outcomes. In this study, we elucidate the role of factors PAXX and MRI during chromosomal EJ and their interplay with the C-NHEJ factors DNA-PKcs and XLF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6710460a50993f582d4fe63d4fd0ff8229a07e" target='_blank'>
              Distinct functions of PAXX and MRI during chromosomal end joining
              </a>
            </td>
          <td>
            Metztli Cisneros-Aguirre, Felicia Wednesday Lopezcolorado, Xiaoli Ping, Ruby Chen, Jeremy M. Stark
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Pediatric brain cancer is the leading cause of disease-related mortality in children, and many aggressive tumors still lack effective treatment strategies. Despite extensive studies characterizing these tumor genomes, alternative transcriptional splicing patterns remain underexplored. Here, we systematically characterized aberrant alternative splicing across pediatric brain tumors, identifying pediatric high-grade gliomas (HGGs) among the most heterogeneous. Through integration with UniProt Knowledgebase annotations, we identified 12,145 splice events in 5,424 genes, leading to functional changes in protein activation, folding, and localization. We discovered that the master splicing factor and cell-cycle modulator, CDC- like kinase 1 (CLK1), is aberrantly spliced in HGGs to include exon 4, resulting in a gain of two phosphorylation sites and subsequent activation of CLK1. Inhibition of CLK1 with Cirtuvivint in the pediatric HGG KNS-42 cell line significantly decreased both cell viability and proliferation in a dose-dependent manner. Morpholino-mediated depletion of CLK1 exon 4 splicing reduced RNA expression, protein abundance, and cell viability. Notably, KNS-42 cells treated with the CLK1 exon 4 morpholino demonstrated differential expression 78 genes and differential splicing with loss or gain of a functional site in 193 genes annotated as oncogene or tumor suppressor genes (TSGs). These genes were enriched for cancer-associated pathways, with 20 identified as significant gene dependencies in pediatric HGGs. Our findings highlight a dependency of pediatric HGGs on CLK1 and its roles contributing to tumor splicing heterogeneity through transcriptional dysregulation of splicing factors and transcriptional modulation of oncogenes. Overall, aberrant splicing in HGGs and other pediatric brain tumors represents a potentially targetable oncogenic pathway contributing to tumor growth and maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/624af22ec9a4e826208669859a67ff2ad429da0e" target='_blank'>
              Characterization of aberrant splicing in pediatric central nervous system tumors reveals CLK1 as a candidate oncogenic dependency
              </a>
            </td>
          <td>
            Ammar S. Naqvi, Ryan J. Corbett, Priyanka Seghal, Karina L. Conkrite, Komal S. Rathi, Brian M. Ennis, Katharina E. Hayer, Bo Zhang, Miguel A. Brown, Daniel P. Miller, Adam A. Kraya, Joseph M. Dybas, Zhuangzhuang Geng, Christopher Blackden, Shehbeel Arif, Antonia Chroni, Aditya Lahiri, Madison L. Hollawell, Phillip B. Storm, J. Foster, Mateusz Koptyra, Peter Madsen, S. Diskin, A. Thomas-Tikhonenko, Adam C. Resnick, J. L. Rokita
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc75014daa55260f9205d28f8165ad0ad389192e" target='_blank'>
              Genetic hallmarks and clinical implications of chromothripsis in childhood T-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            A. Pastorczak, Z. Urbańska, Borys Styka, Karolina Miarka-Walczyk, L. Sędek, K. Wypyszczak, A. Wakulinska, Zuzanna Nowicka, Tomasz Szczepański, Marcin Stańczak, Wojciech Fendler, Jerzy Kowalczyk, W. Młynarski, Monika Lejman
          </td>
          <td>2024-08-27</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The AurkA serine/threonine kinase is a key regulator of cell division controlling mitotic entry, centrosome maturation, and chromosome segregation. The microtubule-associated protein TPX2 controls spindle assembly and is the main AurkA regulator, contributing to AurkA activation, localisation, and stabilisation. Since their identification, AurkA and TPX2 have been described as being overexpressed in cancer, with a significant correlation with highly proliferative and aneuploid tumours. Despite the frequent occurrence of AurkA/TPX2 co-overexpression in cancer, the investigation of their involvement in tumorigenesis and cancer therapy resistance mostly arises from studies focusing only on one at the time. Here, we review the existing literature and discuss the mitotic phenotypes described under conditions of AurkA, TPX2, or AurkA/TPX2 overexpression, to build a picture that may help clarify their oncogenic potential through the induction of chromosome instability. We highlight the relevance of the AurkA/TPX2 complex as an oncogenic unit, based on which we discuss recent strategies under development that aim at disrupting the complex as a promising therapeutic perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1887c3163d0fe8ceb6fc5dcedbb98394ccd0c097" target='_blank'>
              Contribution of AurkA/TPX2 Overexpression to Chromosomal Imbalances and Cancer
              </a>
            </td>
          <td>
            Federica Polverino, Anna Mastrangelo, Giulia Guarguaglini
          </td>
          <td>2024-08-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1082aba026dea72fee57aef4a45e7856e9efcf8" target='_blank'>
              Topoisomerase-modulated genome-wide DNA supercoiling domains colocalize with nuclear compartments and regulate human gene expression.
              </a>
            </td>
          <td>
            Qian Yao, Linying Zhu, Zhen Shi, Subhadra Banerjee, Chongyi Chen
          </td>
          <td>2024-08-16</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G. Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505f6d9a24bb0cc1b413b063823fa87cd784ad4b" target='_blank'>
              Extracting regulatory active chromatin footprint from cell-free DNA
              </a>
            </td>
          <td>
            Kevin Lai, Katharine Dilger, Rachael Cunningham, Kathy T Lam, Rhea Boquiren, Khiet Truong, Maggie C. Louie, Richard Rava, Diana Abdueva
          </td>
          <td>2024-09-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21a24ea5db444de6c4cb2a6e44207610c8118487" target='_blank'>
              RHNO1: at the crossroads of DNA replication stress, DNA repair, and cancer.
              </a>
            </td>
          <td>
            Niphat Jirapongwattana, S. Bunting, Donald R Ronning, Gargi Ghosal, A. Karpf
          </td>
          <td>2024-08-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Genes on sex chromosomes have higher evolutionary rates than those on autosomes. However, this does not necessarily apply to somatic evolution in cancer. Many dominant mutations have been described in the so-called proto-oncogenes (OGs), while recessive mutations are typically described in tumor-suppressor genes (TSGs). Evidence indicates that mutations in X-chromosome TSGs are more likely to contribute to cancer than those in autosomal TSGs. Here, we formalize this in several dynamic models and predict, as expected, that mutations spread faster in TSGs located on the X chromosome than on autosomes (faster-X effect). Conversely, mutations in OGs spread faster on autosomes than on the X chromosome, but under high selective pressure, this difference is negligible. Published genomic screenings of cancer samples show evidence of the faster-X effect in TSGs. This pattern is observed in both sexes, suggesting that the maintenance of X-chromosome inactivation during cancer progression plays an important role in the evolution of TSGs. Strikingly, the relative mutation incidence in X-linked TSGs among females across individual studies is bimodal, with one group of studies showing a faster-X effect and another group showing similar incidences for X-linked and autosomal TSGs. This differentiation between cancer samples is not associated with the specific type of cancer or the tissue of origin. This may indicate that X-chromosome inactivation plays a differential role in the involvement of X-linked TSGs across individual cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e60393ec97b0f4b9cf086f3cab7030b54b21b01" target='_blank'>
              Somatic evolution of cancer genes in sex chromosomes
              </a>
            </td>
          <td>
            Amarachi Akachukwu, Jasmine S. Ratcliff, Antonio Marco
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Sex-chromosome dosage represents a challenge for heterogametic species to maintain correct proportion of gene products across chromosomes in each sex. While therian mammals (XX/XY system) achieve near-perfect balance of X-chromosome mRNAs through X-upregulation and X-inactivation, birds (ZW/ZZ system) have been found to lack efficient compensation at RNA level, challenging the necessity of resolving major gene-dosage discrepancies in avian cells. Through allele-resolved multiome analyses, we comprehensively examined dosage compensation in female (ZW), male (ZZ), and rare intersex (ZZW) chicken. Remarkably, this revealed that females exhibit upregulation of their single Z through increased transcriptional burst frequency similar to mammalian X-upregulation, and that Z-protein levels are further balanced via enhanced translation efficiency in females. Global analyses of transcriptional kinetics elements in birds demonstrate remarkable conservation of the genomic encoding of burst kinetics between mammals and birds. Our study uncovers new mechanisms for achieving sex-chromosome dosage compensation and highlights the importance of gene-dosage balance across diverse species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05f4b40e70c2988331617fc51e819b31adbb3451" target='_blank'>
              Multi-layered dosage compensation of the avian Z chromosome
              </a>
            </td>
          <td>
            Natali Papanicolaou, Antonio Lentini, Sebastian Wettersten, Michael Hagemann-Jensen, Annika Krüger, Jilin Zhang, Christos Coucoravas, Ioannis Petrosian, Xian Xin, Ilhan Ceyhan, Joanna Rorbach, D. Wright, B. Reinius
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) are essential regulatory elements of sex chromosomes that act to equalize gene expression levels between males and females. XIST, RSX, and roX2 regulate X chromosomes in placental mammals, marsupials, and Drosophila, respectively. Because the green anole (Anolis carolinensis) shows complete dosage compensation of its X chromosome, we tested whether a lncRNA was involved. We found an ancient lncRNA, MAYEX, that gained male-specific expression more than 89 million years ago. MAYEX evolved a notable association with the acetylated histone 4 lysine 16 (H4K16ac) epigenetic mark and the ability to loop its locus to the totality of the X chromosome to increase expression levels. MAYEX is the first lncRNA in reptiles linked to a dosage compensation mechanism that balances the expression of sex chromosomes. Editor’s summary Dosage compensation of sex chromosomes can be mediated either by overexpressing genes in the heterogametic sex or by silencing one copy in the homogametic. These processes are complex but are usually mediated by long noncoding RNAs (lncRNAs); however, they are difficult to study because they are not often shared broadly across species. Tenorio et al. examined the green anole and found that genes on the X chromosome are overexpressed in males. This gene expression pattern appeared to be mediated by a lncRNA that they call MAYEX. This lncRNA may be shared beyond anoles to some other species of lizards, thus contributing to our understanding of dosage compensation across animals. —Corinne Simonti">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd6dfb2d4c5ab6b5a06c6a1eff416e4646dc49a" target='_blank'>
              MAYEX is an old long noncoding RNA recruited for X chromosome dosage compensation in a reptile
              </a>
            </td>
          <td>
            Mariela Tenorio, Samantha Cruz-Ruiz, Sergio Encarnación-Guevara, Magdalena Hernández, Jose Antonio Corona-Gomez, Fania Sheccid-Santiago, Joanna Serwatowska, Sinai López-Perdomo, C. D. Flores‐Aguirre, D. Arenas-Moreno, R. Ossiboff, Fausto Méndez-de-la-Cruz, Selene L. Fernandez-Valverde, Mario Zurita, Katarzyna Oktaba, Diego Cortez
          </td>
          <td>2024-09-20</td>
          <td>Science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="R-loops are trimeric nucleic acid structures that form when an RNA molecule hybridizes with its complementary DNA strand, displacing the opposite strand. These structures regulate transcription as well as replication, but aberrant R-loops can form, leading to DNA breaks and genomic instability if unresolved. R-loop levels are elevated in many cancers as well as cells that maintain high-risk human papillomaviruses. We investigated how the distribution as well as function of R-loops changed between normal keratinocytes and HPV positive cells derived from a precancerous lesion of the cervix (CIN I). The levels of R-loops associated with cellular genes were found to be up to 10-fold higher in HPV positive cells than in normal keratinocytes while increases at ALU1 elements increased by up to 500-fold. The presence of enhanced R-loops resulted in altered levels of gene transcription, with equal numbers increased as decreased. While no uniform global effects on transcription due to the enhanced levels of R-loops were detected, genes in several pathways were coordinately increased or decreased in expression only in the HPV positive cells. This included the downregulation of genes in the innate immune pathway, such as DDX58, IL-6, STAT1, IFN-β, and NLRP3. All differentially expressed innate immune genes dependent on R-loops were also associated with H3K36me3 modified histones. Genes that were upregulated by the presence of R-loops in HPV positive cells included those in the DNA damage repair such as ATM, ATRX, and members of the Fanconi Anemia pathway. These genes exhibited a linkage between R-loops and H3K36me3 as well as γH2AX histone marks only in HPV positive cells. These studies identify a potential link in HPV positive cells between DNA damage repair as well as innate immune regulatory pathways with R-loops and γH2AX/H3K36me3 histone marks that may contribute to regulating important functions for HPV pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d30cc12d40eed32c558ede6628c4c8dfd335ab28" target='_blank'>
              HPV induced R-loop formation represses innate immune gene expression while activating DNA damage repair pathways
              </a>
            </td>
          <td>
            Conor W Templeton, Lou Laimins
          </td>
          <td>2024-08-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Many integrative mobile genetic elements (MGEs) are sensitive to RecA-DNA filaments, suggesting their activation may enable them to evade deletion through homologous recombination (HR). This was studied using pneumococcal prophage. Most pneumococci contained prophage that inhibited transformation-mediated HR through chromosomal modification. These included a commonly-targeted attB that reduced transformation by modifying the small cytoplasmic RNA. The prophage were almost universally regulated by C1-type or ImmAR-type systems. Representatives with either type were efficiently deleted by HR. Mutations that impaired a C1-regulated prophage increased this deletion rate, reflecting this element being activated by RecA-DNA filaments imported during transformation. ImmAR-regulated prophage instead responded to transient stimuli by excising as deletion-resistant pseudolysogens, with varying RecA sensitivity, gradually committing to lysing cells following longer-duration signals. These processes were modulated by interactions with the DNA-binding protein DprA. Therefore bacterial diversification though HR is impeded by MGEs that selfishly prevent themselves from being eliminated from the chromosome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a246916c36cd9a48942af1a067709e422a31a41c" target='_blank'>
              Chromosomal curing drives prophage evolution and bacterial epidemiology
              </a>
            </td>
          <td>
            M. Kwun, Alexandru V. Ion, Katinka J. Apagyi, N. Croucher
          </td>
          <td>2024-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d667ed72276b4514cb71b1b131d78ceeae693d2" target='_blank'>
              miR-876-3p is a tumor suppressor on 9p21 that is inactivated in melanoma and targets ERK
              </a>
            </td>
          <td>
            V. Bezrookove, Imran Khan, Anukana Bhattacharjee, Juifang Fan, Robyn Jones, Anima Sharma, M. Nosrati, P. Desprez, Nathan Salomonis, Yihui Shi, A. Dar, M. Kashani-Sabet
          </td>
          <td>2024-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chromatin domain boundaries delimited by CTCF motifs can restrict the range of enhancer action. However, disruption of domain structure often results in mild gene dysregulation and thus predicting the impact of boundary rearrangements on animal development remains challenging. Here, we tested whether structural perturbation of a chromatin domain with multiple developmental regulators can result in more acute gene dysregulation and severe developmental phenotypes. We targeted clusters of CTCF motifs in a domain of the mouse genome containing three FGF ligand genes—Fgf3, Fgf4, and Fgf15—that regulate several developmental processes. Deletion of the 23.9kb cluster that defines the centromeric boundary of this domain resulted in ectopic interactions of the FGF genes with enhancers located across the deleted boundary that are active in the developing brain. This caused strong induction of FGF expression and perinatal lethality with encephalocele and orofacial cleft phenotypes. Heterozygous boundary deletion was sufficient to cause these fully penetrant phenotypes, and strikingly, loss of a single CTCF motif within the cluster also recapitulated ectopic FGF expression and caused encephalocele. However, such phenotypic sensitivity to perturbation of domain structure did not extend to all CTCF clusters of this domain, nor to all developmental processes controlled by these three FGF genes—for example, the ability to undergo lineage specification in the blastocyst and pre-implantation development were not affected. By tracing the impact of different chromosomal rearrangements throughout mouse development, we start to uncover the determinants of phenotypic robustness and sensitivity to perturbation of chromatin boundaries. Our data show how small sequence variants at certain domain boundaries can have a surprisingly outsized effect and must be considered as potential sources of gene dysregulation during development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49d79dff953903821b2cc804c273679c39f02444" target='_blank'>
              Structural perturbation of chromatin domains with multiple developmental regulators can severely impact gene regulation and development
              </a>
            </td>
          <td>
            S. Chakraborty, Nina Wenzlitschke, Matthew J. Anderson, A. Eraso, Manon Baudic, Joyce J. Thompson, Alicia A. Evans, Lilly M. Shatford‑Adams, Raj Chari, Parirokh Awasthi, Ryan K. Dale, M. Lewandoski, Timothy J. Petros, Pedro P. Rocha
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b09daa5b66a34ba0df34226e316e985b17964e4" target='_blank'>
              Satellite DNAs and the evolution of the multiple X1X2Y sex chromosomes in the wolf fish Hoplias malabaricus (Teleostei; Characiformes)
              </a>
            </td>
          <td>
            G. A. Toma, A. Sember, C. A. G. Goes, Rafael Kretschmer, F. Porto‐Foresti, L. Bertollo, Thomas Liehr, R. Utsunomia, Marcelo de Bello Cioffi
          </td>
          <td>2024-09-02</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-01</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 METTL1 is an RNA methyltransferase which catalyzes N7-methylguanosine (m7G) methylation on various RNAs, particularly on tRNAs. METTL1 has been reported to be amplified and/or overexpressed and high expression correlates with poor patient prognosis in different types of adult cancers. Our ZERO Childhood Cancer Program focuses on identifying and validating potential new drug targets and developing novel therapies for clinical use. ZERO has sequenced the whole genome and transcriptome in over 500 high-risk pediatric tumors and found that METTL1 gene copy number and gene expression are significantly higher in a subset of high-grade glioma, neuroblastoma, diffuse midline glioma and fusion positive rhabdomyosarcoma. Analysis of the publicly available SEQC microarray dataset showed that high METTL1 expression correlates with poor outcomes in MYCN amplified neuroblastoma patients. We validated the functional roles of METTL1 gene amplification and overexpression in a few pediatric cancer cell line models using the Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR) Cas13d knockdown system, in dual lentiviral constructs. We found that knocking down METTL1 significantly decreased METTL1 amplified neuroblastoma and rhabdomyosarcoma cell viability and colony formation but not in METTL1 non-amplified neuroblastoma and rhabdomyosarcoma cells. Gene Set Enrichment Analysis from RNA sequencing data revealed that knocking down METTL1 resulted in a decrease in Myc target gene expression, and an increased in apoptosis, interferon alpha response and p53 pathway hallmarks. Western blot analysis confirmed that knocking down METTL1 led to a decrease in the MCL-1 antiapoptotic protein expression and an increase in the proapoptotic protein expression such as cleaved PARP and caspase 3. Taken together, our data demonstrate that METTL1 is a potential therapeutic target in METTL1 amplified pediatric cancers.
 Citation Format: Pei Y. Liu, Steven He, Chelsea Mayoh, Alice Salib, Antoine De Weck, Paul G. Ekert. Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64085373809830d692bd2bd083b18380125ea0bf" target='_blank'>
              Abstract A012 Unravelling the role of METTL1 in Myc driven pediatric cancers using CRISPR Cas13d
              </a>
            </td>
          <td>
            Pei Y. Liu, Steven He, C. Mayoh, A. Salib, Antoine de Weck, Paul G. Ekert
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Mammalian genomes are scattered with transposable elements (TEs). TEs are epigenetically silenced to prevent harmful effects caused by either global activation leading to genome instability or insertional mutation disturbing gene transcription. However, whether the activation of a single element can contribute to pathological phenotypes without directly affecting gene expression is largely unknown. Here, we show that tissue-specific expression of a TE in the embryo leads to the production of viral-like particles (VLPs) which can affect organ formation. Failure to silence an LTR retrotransposon inserted upstream of the Fgf8 gene results in its co-expression with Fgf8 in the developing embryo. While local gene regulation is unaffected, the LTR retrotransposon participates in chromatin folding at the locus and adopts the expression of the regulatory domain it is located in. This drives the production of VLPs in the Fgf8-expressing cells of the developing limb, triggering apoptotic cell death at the time of digit outgrowth and resulting in a limb malformation resembling human ectrodactyly. This phenotype can be rescued by knock-out or knock-in of the retrotransposon causing mutations preventing its full retroviral cycle. Insertion of the same element at other developmental loci faithfully recapitulates expression according to the neighboring regulatory activity. Our findings provide a mechanism by which TE insertion is incorporated into the local genomic regulatory landscape and show how VLP production in post-implantation embryos can interfere with organ formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6530941b1bbda9df13ec9f7016e41dec0d4f10bf" target='_blank'>
              Enhancer adoption by an LTR retrotransposon generates viral-like particles causing developmental limb phenotypes
              </a>
            </td>
          <td>
            Juliane Glaser, Giulia Cova, B. Fauler, C. Prada-Medina, Virginie Stanislas, Mai H.Q. Phan, R. Schöpflin, Yasmin Aktas, Martin Franke, Guillaume Andrey, C. Paliou, Verena Laupert, W. Chan, L. Wittler, Thorsten Mielke, Stefan Mundlos
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56963ef797b360ca0fd377e89bb8feeef832b223" target='_blank'>
              Genome-wide characterization of dynamic DNA 5-hydroxymethylcytosine and TET2-related DNA demethylation during breast tumorigenesis
              </a>
            </td>
          <td>
            Shuang-Ling Wu, Lin Yang, Changcai Huang, Qing Li, Chunhui Ma, Fang Yuan, Yinglin Zhou, Xiaoyue Wang, Wei-Min Tong, Yamei Niu, Feng Jin
          </td>
          <td>2024-09-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The histone variant H3.3 acts in coordination with histone posttranslational modifications and other chromatin features to facilitate appropriate transcription. Canonical histone H3 and histone variant H3.3 are post-translationally modified with the genomic distribution of these marks denoting different features and with more recent evidence suggesting that these modifications may influence transcription. While the majority of posttranslational modifications occur on histone tails, there are defined modifications within the globular domain, such as acetylation of H3K122/H3.3K122. To understand the function of the residue H3.3K122 in transcriptional regulation, we attempted to generate H3.3K122A mouse embryonic stem (mES) cells but were unsuccessful. Through multi-omic profiling of mutant cell lines harboring two or three of four H3.3 targeted alleles, we have uncovered that H3.3K122A is neomorphic and results in lethality. This is surprising as prior studies demonstrate H3.3-null mES cells are viable and pluripotent, albeit with reduced differentiation capacity. Together, these studies have uncovered a novel dependence of a globular domain residue of H3.3 for viability and broadened our understanding of how histone variants contribute to transcription regulation and pluripotency in mES cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efd2d0e0e1e2c91bef0039d2aed277d8828f1da5" target='_blank'>
              H3.3K122A results in a neomorphic phenotype in mouse embryonic stem cells
              </a>
            </td>
          <td>
            Benjamin Patty, Cailin Jordan, Santana M. Lardo, Kris Troy, Sarah Hainer
          </td>
          <td>2024-08-27</td>
          <td>Research Square</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7234a63fd75fc71e91d39632a18b128298ba3fb" target='_blank'>
              Genome-wide identification of copy number variations in wrinkled skin cases of Xiang pigs
              </a>
            </td>
          <td>
            Xiaoli Liu, Fenbin Hu, Wei Wang, Xia Chen, Xi Niu, Shihui Huang, Zhou-cai Wang, Jiafu Wang, Xueqin Ran
          </td>
          <td>2024-08-24</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Interphase mammalian genomes are folded in 3D with complex locus-specific patterns that impact gene regulation. CTCF (CCCTC-binding factor) is a key architectural protein that binds specific DNA sites, halts cohesin-mediated loop extrusion, and enables long-range chromatin interactions. There are hundreds of thousands of annotated CTCF-binding sites in mammalian genomes; disruptions of some result in distinct phenotypes, while others have no visible effect. Despite their importance, the determinants of which CTCF sites are necessary for genome folding and gene regulation remain unclear. Here, we update and utilize Akita, a convolutional neural network model, to extract the sequence preferences and grammar of CTCF contributing to genome folding. Our analyses of individual CTCF sites reveal four predictions: (i) only a small fraction of genomic sites are impactful, (ii) insulation strength is highly dependent on sequences flanking the core CTCF binding motif, (iii) core and flanking sequences are broadly compatible, and (iv) core and flanking nucleotides contribute largely additively to overall strength. Our analysis of collections of CTCF sites make two predictions for multi-motif grammar: (i) insulation strength depends on the number of CTCF sites within a cluster, and (ii) pattern formation is governed by the orientation and spacing of these sites, rather than any inherent specialization of the CTCF motifs themselves. In sum, we present a framework for using neural network models to probe the sequences instructing genome folding and provide a number of predictions to guide future experimental inquiries. Author Summary Mammalian genomes are spatially organized in 3D with profound consequences for all processes involving DNA. CTCF is a key genome organizer, recognizing numerous sites and creating a variety of contact patterns across the genome. Despite the importance of CTCF, the sequence determinants and grammar of how individual sites collectively instruct genome folding remain unclear. This work leverages the ability of Akita, a deep neural network, to make high-throughput predictions for genome folding after DNA sequence perturbations. Using Akita, we make several experimentally testable predictions. First, only a minority of annotated sites individually impact folding, and flanking DNA sequences greatly modulate their impact. Second, multiple sites together influence folding based on their number, orientation, and spacing. In sum, we provide a roadmap for interpreting neural networks to better understand genome folding and important considerations for the design of experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a8a450d11a02a4e12bb1863b2d5f19340a89cc" target='_blank'>
              Interpreting the CTCF-mediated sequence grammar of genome folding with AkitaV2
              </a>
            </td>
          <td>
            Paulina N. Smaruj, Fahad Kamulegeya, David R. Kelley, G. Fudenberg
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Chromosomal aneuploidy, that is, numerical chromosome aberrations, is one of the molecular hallmarks of cancer. However, when neoplasms are studied with sequencing- and array-based approaches, chromosome numbers and ploidy states are typically inferred from bulk DNA data. Furthermore, published molecular estimates of neoplasia-associated aneuploidy often also include genomic imbalances resulting from various types of structural rearrangement, which likely result from other mechanisms than numerical chromosome aberrations. We thus analyzed chromosome numbers using single-cell cytogenetic data from 83,862 tumors, and show that both benign and malignant tumors are highly heterogeneous with regard to deviations from the normal, diploid state. Focusing on the chromosome numbers in 112 specific tumor types, defined by both exact morphologic diagnosis and organ location and from which data from ≥50 cases were available, we found two major clusters: one predominated by near-diploid neoplasms and one by neoplasms with extensive aneuploidy and one or more whole genome doublings. The former cluster included most benign solid tumors, myeloid neoplasms, and malignant gene fusion-associated solid tumors, whereas the latter was predominated by malignant solid tumors and lymphomas. For 16 malignant tumor types, the distribution of chromosome numbers could be compared to TCGA ploidy level data. Cytogenetic and molecular data correlated well, but the former indicates a higher level of clonal heterogeneity. The results presented here suggest shared pathogenetic mechanisms in certain tumor types and provide a reference for molecular analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b031c86f32544de4d075d55344d053563ec29d0f" target='_blank'>
              Aneuploidy in neoplasia: Single-cell data on 83,862 tumors.
              </a>
            </td>
          <td>
            Fredrik Mertens, Jakob Hofvander, Nils Mandahl, F. Mitelman
          </td>
          <td>2024-09-02</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/327f3b8a50e3e934afb3ca0aaae0fc0fd3ed123e" target='_blank'>
              An elevated rate of whole-genome duplications in cancers from Black patients
              </a>
            </td>
          <td>
            L. M. Brown, R. Hagenson, T. Koklič, I. Urbančič, Lu Qiao, J. Štrancar, J. Sheltzer
          </td>
          <td>2024-09-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="RNA-chromatin interactions play crucial roles in gene regulation and genome organization, but the interaction landscape remains poorly understood. In this study, we conducted an in-depth analysis of a previously published dataset on RNase-treated in situ mapping of the RNA–genome interactome in human embryonic stem cells. This dataset globally profiles RNase-insensitive RNA-chromatin interactions. Our analysis revealed that RNase treatment selectively preserved long-range RNA-chromatin interactions while removing promiscuous interactions resulting from the local diffusion of nascent transcripts. RNase-insensitive chromatin-associated RNAs (RI-caRNAs) exhibited high sequence conservation and preferentially localized to functional genomic regions, including promoters, transcription factor binding sites, and regions with specific histone modifications. Interestingly, coding and non-coding RNA transcripts showed distinct sensitivities to RNase, with lncRNAs and disease-associated transcripts being enriched among RI-caRNAs. Furthermore, we identified specific caRNA classes associated with individual transcription factors and histone modifications. Altogether, our findings reveal a RNase-inaccessible regulatory RNA-chromatin interactome and provide a resource for understanding RNA-mediated chromatin regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2c27dbcf0f58a88cf32d2c23a90aa981842c6c6f" target='_blank'>
              Genome-wide identification of stable RNA-chromatin interactions
              </a>
            </td>
          <td>
            Xingzhao Wen, Sheng Zhong
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Excess centrosomes cause defects in mitosis, cell-signaling, and cell migration, and therefore their assembly is tightly regulated. Plk4 controls centriole duplication at the heart of centrosome assembly, and elevation of Plk4 promotes centrosome amplification (CA), a founding event of tumorigenesis. Here, we investigate the transcriptional consequences of elevated Plk4 and find Unkempt, a gene encoding an RNA binding protein with roles in translational regulation, to be one of only two upregulated mRNAs. Unk protein localizes to centrosomes and Cep131-positive centriolar satellites and is required for Plk4-induced centriole overduplication in an RNA-binding dependent manner. Translation is enriched at centrosomes and centriolar satellites with Unk and Cep131 promoting this localized translation. A transient centrosomal downregulation of translation occurs early in Plk4-induced CA. CNOT9, an Unk interactor and component of the translational inhibitory CCR4-NOT complex, localizes to centrosomes at this time. In summary, centriolar satellites and Unk promote local translation as part of a translational program that ensures centriole duplication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c6c9c911209145f33e4cb8a10948a1237a931b7" target='_blank'>
              The Unkempt RNA binding protein reveals a local translation program in centriole overduplication
              </a>
            </td>
          <td>
            Abraham Martinez, Alexander J. Stemm-Wolf, Ryan M. Sheridan, Matthew J. Taliaferro, Chad G. Pearson
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bed3b8831fd786b3f7f846b45187bda3701aa49a" target='_blank'>
              Pan-cancer analysis of m1A writer gene RRP8: implications for immune infiltration and prognosis in human cancers
              </a>
            </td>
          <td>
            Zhihui Huang, Koo Han Yoo, Duohui Li, Qingxin Yu, Luxia Ye, Wuran Wei
          </td>
          <td>2024-09-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic highly polyploid macronucleus is lost. The formation of a new macronucleus involves Programmed Genome Rearrangements, introducing DNA DSBs at approximately 45,000 loci. P. tetraurelia employs a Non-Homologous End Joining (NHEJ)-related mechanism for the systematic repair of these DSBs. Four genes encoding DNA polymerases of family X are present in the genome, one of which was found recently to colocalize with other proteins of NHEJ. The question arises as to how they make almost no error. Here we show that these enzymes are most similar to metazoan DNA polymerase λ and exhibit high fidelity through two different molecular mechanisms. Using X-ray structure determination of polymerase lambda mutants recapitulating sequence determinants of P. tetraurelia PolXs, we find both a local conformational change that involves exchanging partners in a crucial salt bridge in the active site upon binding of correct dNTPs, and a larger conformational change involving the closure of Loop3. This stabilizes the template DNA in the active site, only in the presence of the correct incoming dNTP. Differences with human pol λ and pol β are discussed. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a2c8e3978ae53a388430b008c681d13e31a282" target='_blank'>
              Mechanisms Ensuring Fidelity of Family X DNA Polymerases in Programmed DNA rearrangements in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Antonin Nourisson, Sophia Missoury, A. Haouz, Marc Delarue
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbc12eb0a2eae0ddf37c9584cc484b594730250" target='_blank'>
              Genomic profiling of cell-free DNA from dogs with benign and malignant tumors
              </a>
            </td>
          <td>
            Hongchao Du, Wenfeng Liu, Yunfei Li, Lijuan Zhang, Fangfang Jiang, Dandan Zhu, Jingshuai Li, Pan Hu, Ningning Yan, Mao Mao, Shiyong Li
          </td>
          <td>2024-09-13</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec914b4189eafe0cce08165c6a7c2a53bfc159e" target='_blank'>
              Unveiling FRG1’s DNA repair role in breast cancer
              </a>
            </td>
          <td>
            Shubhanjali Shubhanjali, Talina Mohapatra, Rehan Khan, Manjusha Dixit
          </td>
          <td>2024-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/114a66efadbc9788f06d601c6a1a3abc5635eb72" target='_blank'>
              Evolution of retrocopies in the context of HUSH silencing
              </a>
            </td>
          <td>
            Joanna Kozłowska-Masłoń, Joanna Ciomborowska-Basheer, M. Kubiak, Izabela Makałowska
          </td>
          <td>2024-08-02</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Alveolar rhabdomyosarcoma (ARMS) patients harboring PAX3-FOXO1 and PAX7-FOXO1 fusion proteins exhibit a greater incidence of tumor relapse, metastasis, and poor survival outcome, thereby underscoring the urgent need to develop effective therapies to treat this subtype of childhood cancer. To uncover mechanisms that contribute to tumor initiation, we developed a novel muscle progenitor model and used epigenomic approaches to unravel genome re-wiring events mediated by PAX3/7 fusion proteins. Importantly, these regulatory mechanisms are conserved across established ARMS cell lines, primary tumors, and orthotopic-patient derived xenografts. Among the key targets of PAX3- and PAX7-fusion proteins, we identified a cohort of oncogenes, FGF receptors, and genes essential for mitochondrial metabolism and protein translation, which we successfully targeted in preclinical trials. Our data suggest an explanation for the relative paucity of recurring mutations in this tumor, provide a compelling list of actionable targets, and suggest promising new strategies to treat this tumor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f840f071bc2696a99f7c83f33f72c0a963e3308" target='_blank'>
              PAX fusion proteins deregulate gene networks controlling mitochondrial translation in pediatric rhabdomyosarcoma
              </a>
            </td>
          <td>
            Bhargab Kalita, Gerard Martinez-Cebrian, Justina McEvoy, Melody Allensworth, Michelle Knight, Alessandro Magli, Rita C. R. Perlingeiro, Michael A. Dyer, Elizabeth Stewart, B. Dynlacht
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Multiple myeloma is the second most hematological cancer. RUVBL1 and RUVBL2 form a subcomplex of many chromatin remodeling complexes implicated in cancer progression. As an inhibitor specific to the RUVBL1/2 complex, CB-6644 exhibits remarkable anti-tumor activity in xenograft models of Burkitt’s lymphoma and multiple myeloma (MM). In this work, we defined transcriptional signatures corresponding to CB-6644 treatment in MM cells and determined underlying epigenetic changes in terms of chromatin accessibility. CB-6644 upregulated biological processes related to interferon response and downregulated those linked to cell proliferation in MM cells. Transcriptional regulator inference identified E2Fs as regulators for downregulated genes and MED1 and MYC as regulators for upregulated genes. CB-6644-induced changes in chromatin accessibility occurred mostly in non-promoter regions. Footprinting analysis identified transcription factors implied in modulating chromatin accessibility in response to CB-6644 treatment, including ATF4/CEBP and IRF4. Lastly, integrative analysis of transcription responses to various chemical compounds of the molecular signature genes from public gene expression data identified CB-5083, a p97 inhibitor, as a synergistic candidate with CB-6644 in MM cells, but experimental validation refuted this hypothesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aff9ee9c96de0ec030a3d9d628090554b2bbd4f2" target='_blank'>
              Molecular Signatures of CB-6644 Inhibition of the RUVBL1/2 Complex in Multiple Myeloma
              </a>
            </td>
          <td>
            Weijun Yi, Sebastian A. Dziadowicz, Rachel S. Mangano, Lei Wang, Joseph C. McBee, Steven M. Frisch, Lori A. Hazlehurst, Donald A Adjeroh, Gangqing Hu
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9be5f58320c6cf366bc684bebd0ed4433dc1746c" target='_blank'>
              Pramel15 facilitates zygotic nuclear DNMT1 degradation and DNA demethylation
              </a>
            </td>
          <td>
            Jiajun Tan, Yingfeng Li, Xiang Li, Xiaoxiao Zhu, Liping Liu, Hua Huang, Jiahua Wei, Hailin Wang, Yong Tian, Zhigao Wang, Zhuqiang Zhang, Bing Zhu
          </td>
          <td>2024-08-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2dde0164af0025d5f19bf691294f177add4f7b" target='_blank'>
              Low-pass whole genome sequencing of circulating tumor cells to evaluate chromosomal instability in triple-negative breast cancer
              </a>
            </td>
          <td>
            S. Di Cosimo, M. Silvestri, C. De Marco, Alessia Calzoni, M. C. De Santis, Maria Grazia Carnevale, C. Reduzzi, Massimo Cristofanilli, Vera Cappelletti
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Apurinic/apyrimidinic (AP) sites and single-strand breaks (SSB) arising from base excision repair (BER) during misincorporation of damaged nucleobases may hinder replication fork stability in homologous recombination-deficient (HRD) cells. At templated AP-sites, HMCES DNA-protein crosslinks (DPC) regulate replication fork speed while avoiding APE1-mediated cytotoxic double-strand breaks (DSB). Whereas the role of HMCES at template DNA strand is well studied, its consequences on nascent DNA are less understood. Here, we provide evidence that HMCES play detrimental roles during removal of 5- hydroxymethyl-2’-deoxycytidine (5hmdC)-derived 5-hydroxymethyl-2’-deoxyuridine (5hmdU) by BER at replication forks. HRD cells display heightened HMCES chromatin levels, which increase upon 5hmdC exposure, suggesting that HMCES binds both spontaneous and 5hmdC-induced AP-sites. HMCES depletion largely suppresses 5hmdC- mediated Fancd2-/- replication fork defects, chromosomal aberrations and cell lethality, suggesting that HMCES is responsible for the replication fork impairment and lethality observed in HRD cells. Therefore, HMCES-DPCs are a novel source of BER-initiated PRIMPOL-mediated ssDNA gaps, implying endogenous DPCs as a source of DNA damage in HRD tumours. Teaser Covalent binding of HMCES to nascent DNA blocks replication progression and kills homologous recombination deficient cancer cells">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91301e05a5b2dbeda5a785b9d3a1c934b12bac03" target='_blank'>
              HMCES corrupts replication fork stability during base excision repair in homologous recombination deficient cells
              </a>
            </td>
          <td>
            María José Peña-Gómez, Yaiza Rodríguez-Martin, Marta del Rio Oliva, Jean-Yves Masson, JC Reyes, Iván V. Rosado
          </td>
          <td>2024-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f891979797e3688f35366887366668ec188a50ae" target='_blank'>
              Microsatellite instability at U2AF-binding polypyrimidic tract sites perturbs alternative splicing during colorectal cancer initiation
              </a>
            </td>
          <td>
            V. Jonchère, Hugo Montémont, Enora Le Scanf, A. Siret, Q. Letourneur, Emmanuel Tubacher, Christophe Battail, Assane Fall, Karim Labreche, Victor Renault, T. Ratovomanana, O. Buhard, A. Jolly, Philippe Le Rouzic, Cody Feys, Emmanuelle Despras, Habib Zouali, R. Nicolle, Pascale Cervera, M. Svrcek, P. Bourgoin, Hélène Blanché, A. Boland, Jérémie Lefèvre, Y. Parc, M. Touat, F. Bielle, D. Arzur, Gwennina Cueff, C. Le Jossic-Corcos, Gaël Quéré, Gwendal Dujardin, Marc Blondel, Cédric Le Maréchal, R. Cohen, Thierry André, Florence Coulet, P. de la Grange, A. de Reyniès, Jean-françois Flejou, Florence Renaud, A. Alentorn, Laurent Corcos, J. Deleuze, A. Collura, A. Duval
          </td>
          <td>2024-08-06</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="DNA double strand breaks (DSBs) are critical for the efficacy of radiotherapy as they lead to cell death if not repaired. DSBs caused by ionizing radiation (IR) initiate histone modifications and accumulate DNA repair proteins, including 53BP1, which forms distinct foci at damage sites and serves as a marker for DSBs. DSB repair primarily occurs through Non-Homologous End Joining (NHEJ) and Homologous Recombination (HR). NHEJ directly ligates DNA ends, employing proteins such as DNA-PKcs, while HR, involving proteins such as Rad54, uses a sister chromatid template for accurate repair and functions in the S and G2 phases of the cell cycle. Both pathways are crucial, as illustrated by the IR sensitivity in cells lacking DNA-PKcs or Rad54. We generated mouse embryonic stem (mES) cells which are knockout (KO) for DNA-PKcs and Rad54 to explore the combined role of HR and NHEJ in DSB repair. We found that cells lacking both DNA-PKcs and Rad54 are hypersensitive to X-ray radiation, coinciding with impaired 53BP1 focus resolution and a more persistent G2 phase cell cycle block. Additionally, mES cells deficient in DNA-PKcs or both DNA-PKcs and Rad54 exhibit an increased nuclear size approximately 18–24 h post-irradiation. To further explore the role of Rad54 in the absence of DNA-PKcs, we generated DNA-PKcs KO mES cells expressing GFP-tagged wild-type (WT) or ATPase-defective Rad54 to track the Rad54 foci over time post-irradiation. Cells lacking DNA-PKcs and expressing ATPase-defective Rad54 exhibited a similar phenotypic response to IR as those lacking both DNA-PKcs and Rad54. Despite a strong G2 phase arrest, live-cell imaging showed these cells eventually progress through mitosis, forming micronuclei. Additionally, mES cells lacking DNA-PKcs showed increased Rad54 foci over time post-irradiation, indicating an enhanced reliance on HR for DSB repair without DNA-PKcs. Our findings underscore the essential roles of HR and NHEJ in maintaining genomic stability post-IR in mES cells. The interplay between these pathways is crucial for effective DSB repair and cell cycle progression, highlighting potential targets for enhancing radiotherapy outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4800ec280608dc95677441accad3c221afaf2bd" target='_blank'>
              Synergistic Roles of Non-Homologous End Joining and Homologous Recombination in Repair of Ionizing Radiation-Induced DNA Double Strand Breaks in Mouse Embryonic Stem Cells
              </a>
            </td>
          <td>
            Gerarda van de Kamp, Tim Heemskerk, Roland Kanaar, Jeroen Essers
          </td>
          <td>2024-08-30</td>
          <td>Cells</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f3792b41778f947365ed0dac38ff748d4c49600" target='_blank'>
              L3MBTL1, a polycomb protein, promotes Osimertinib acquired resistance through epigenetic regulation of DNA damage response in lung adenocarcinoma
              </a>
            </td>
          <td>
            Zihe Zhang, Yongwen Li, Ruifeng Shi, Chaoyi Jia, Songlin Xu, Guangsheng Zhu, P. Cao, Hua Huang, Xuanguang Li, Hongbing Zhang, Minghui Liu, Chen Chen, Hongyu Liu, Chunsheng Kang, Jun Chen
          </td>
          <td>2024-09-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU- 023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention. Author Summary Bile duct cancer has the highest incidence in northeastern Thailand. In this study, we explored the mtDNA of CCA cell lines, as mitochondria play a key role in cellular energy production. We aimed to identify mutations that could serve as therapeutic targets or biomarkers. Our analysis revealed that each CCA cell line has unique mtDNA profiles, including INDELs, SVs, and SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. yet they also shared some common mutations. Future research is needed to understand how these mutations affect cell function, interact with nuclear DNA, and their potential for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eafb79f26f32f1aeaf6d648b336c327747f051f" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapan, A. Jareanrat, V. Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c4d24999230a049c47c4ad63da565877c2b583c" target='_blank'>
              PPM1D activity promotes cellular transformation by preventing senescence and cell death.
              </a>
            </td>
          <td>
            Miroslav Stoyanov, Andra S. Martinikova, Kateřina Matějková, K. Horackova, P. Zemánková, Kamila Burdová, Zuzana Zemanova, P. Kleiblova, Z. Kleibl, L. Macurek
          </td>
          <td>2024-09-05</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/141df69604dde0dcf2c796a0436f024e9fb9a66b" target='_blank'>
              Two-polarized roles of transcription factor FOSB in lung cancer progression and prognosis: dependent on p53 status
              </a>
            </td>
          <td>
            Hongchao Zhang, Guopei Zhang, Mingyang Xiao, S. Cui, Cuihong Jin, Jinghua Yang, Sheng-wen Wu, Xiaobo Lu
          </td>
          <td>2024-08-21</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The identification of tumor cells is pivotal to understanding tumor heterogeneity and the tumor microenvironment. Recent advances in spatially resolved transcriptomics (SRT) have revolutionized the way that transcriptomic profiles are characterized and have enabled the simultaneous quantification of transcript locations in intact tissue samples. SRT is a promising alternative method of studying gene expression patterns in spatial domains. Nevertheless, the precise detection of tumor regions within intact tissue remains a great challenge. The common way of identifying tumor cells is via tumor-specific marker gene expression signatures, which is highly dependent on marker accuracy. Another effective approach is through aneuploid copy number events, as most types of cancer exhibit copy number abnormalities. Here, we introduce a novel computational method, called TUSCAN (TUmor Segmentation and Classification ANalysis in spatial transcriptomics), which constructs a spatial copy number variation profile to improve the accuracy of tumor region identification. TUSCAN combines the gene information from SRT data and the hematoxylin-and-eosin-staining image to annotate tumor sections and other benign tissues. We benchmark the performance of TUSCAN and several existing methods through the application to multiple datasets from different SRT platforms. We demonstrate that TUSCAN can effectively delineate tumor regions, with improved accuracy compared to other approaches. Additionally, the output of TUSCAN provides interpretable clonal evolution inferences that may lead to novel insights into disease development and potential druggable targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eadd9edb67ab1fb50a81871f6578ddc5fcbae2db" target='_blank'>
              TUSCAN: Tumor segmentation and classification analysis in spatial transcriptomics
              </a>
            </td>
          <td>
            Chenxuan Zang, Charles C. Guo, Peng Wei, Ziyi Li
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facf4dda707b67b3b56d646f95496db5b6cc7df8" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N. Anderson
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetic alterations are implicated in the early stages of tumorigenesis and are widely recognized as a ubiquitous phenomenon in cancer development. Aberrant epigenetic modifications can alter the expression of target genes, induce heterochromatin formation, and gradually drive normal cells towards immortalized tumor cells with significant consequences. SETDB1 (SET domain bifurcated histone lysine methyltransferase 1), a typical histone me-thyltransferase, promotes the formation of heterochromatin and inhibits the transcription of genes by modifying the methylation of lysine 9 of histone 3. SETDB1 is usually highly ex-pressed in tumors with high copy numbers, accompanied by poor prognosis and low patient survival rates, which is a typical case of abnormal epigenetic modification. We discuss the mechanism of SETDB1 in a variety of cancers and review the epigenetic inhibitors that have been reported in recent years, along with their anti-tumor effects. In addition, we summarize the role of SETDB1 in a variety of diseases and cell functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8068b1c29211fa575e01e9ddbf87ca49f2785f" target='_blank'>
              Targeting the Tumor Epigenetic Regulator SETDB1 for Tumor Therapy.
              </a>
            </td>
          <td>
            Cheng Peng, Nini Zhou, Tengjiang Chen, Jie Lei, Changwen Chen, Shunqin Zhu
          </td>
          <td>2024-08-27</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Transposable elements (TEs) are DNA sequences able to create copies of themselves within the genome. Despite their limited expression due to silencing, TEs still manage to impact the host genome. For instance, some TEs have been shown to act as cis-regulatory elements and be co-opted in the human genome. This highlights that the contributions of TEs to the host might come from their relationship with the epigenome rather than their expression. However, a systematic analysis that relates TEs in the human genome directly with chromatin histone marks across distinct cell types remains lacking. Here we leverage a new dataset from the International Human Epigenome Consortium with 4867 uniformly processed ChIP-seq experiments for 6 histone marks across 175 annotated cell labels and show that TEs have drastically different enrichments levels across marks. Overall, we find that TEs are generally depleted in H3K9me3 histone modification, except for L1s, while MIRs were highly enriched in H3K4me1, H3K27ac and H3K27me3 and Alus were enriched in H3K36me3. Furthermore, we present a generalised profile of the relationship between TEs enrichment and TE age which reveals a few TE families (Alu, MIR, L2) as diverging from expected dynamics. We also find some significant differences in TE enrichment between cell types and that in 20% of the cases, these enrichments were cell-type specific. We report that at least 4% of cell types with healthy and cancer samples featured significant differences. Notably, we identify 456 TE-Cell Type-histone triplet candidates with the strongest cell-type specific enrichments. We show that many of these candidates are associated with relevant biological processes and genes expressed in the relevant cell type. These results further support a role for TE in genome regulation and highlight novel associations between TEs and histone marks across cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa3b0974577b9fd96a44fb8b012fd715ee99dc99" target='_blank'>
              Transposable elements impact the regulatory landscape through cell type specific epigenomic associations
              </a>
            </td>
          <td>
            Jeffrey Hyacinthe, Guillaume Bourque
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8eb55432a81ae9f11fc653e3722684f5e8a7ed1" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Long INterspersed Element-1 (LINE-1; L1) and Alu are two families of transposable elements (TEs) occupying ∼17% and ∼11% of the human genome, respectively. Though only a small fraction of L1 copies is able to produce the machinery to mobilize autonomously, Alu elements and degenerate L1 copies can hijack their functional machinery and mobilize in trans. The expression and subsequent copy number expansion of L1 and Alu can exert pathological effects on their hosts, promoting genome instability, inflammation, and cell cycle alterations. These features have made L1 and Alu promising focus subjects in studies of aging and aging diseases where they can become active. However, the mechanisms regulating variation in their expression and copy number remain incompletely characterized. Moreover, the relevance of known mechanisms to diverse human populations remains unclear, as mechanisms are often characterized in isogenic cell culture models. To address these gaps, we leveraged genomic data from the 1000 Genomes Project to carry out a trans-ethnic GWAS of L1 and Alu insertion global singletons. These singletons are rare insertions observed only once in a population, potentially reflecting recently acquired L1 and Alu integrants or structural variants, and which we used as proxies for L1/Alu-associated copy number variation. Our computational approach identified single nucleotide variants in genomic regions containing genes with potential and known TE regulatory properties, and it enriched for single nucleotide variants in regions containing known regulators of L1 expression. Moreover, we identified many reference TE copies and polymorphic structural variants that were associated with L1/Alu singletons, suggesting their potential contribution to TE copy number variation through transposition-dependent or transposition-independent mechanisms. Finally, a transcriptional analysis of lymphoblastoid cells highlighted potential cell cycle alterations in a subset of samples harboring L1/Alu singletons. Collectively, our results (i) suggest that known TE regulatory mechanisms may also play regulatory roles in diverse human populations, (ii) expand the list of genic and repetitive genomic loci implicated in TE copy number variation, and (iii) reinforce the links between TEs and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c547e837e626947d2c017f057b521d189f2c4fd3" target='_blank'>
              Multi-ancestry GWAS reveals loci linked to human variation in LINE-1- and Alu-copy numbers
              </a>
            </td>
          <td>
            Juan I. Bravo, Lucia Zhang, BA Benayoun
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Human cells are typically diploid, that is, they contain two sets of chromosomes. Some cells, however, contain three or more sets of chromosomes and are called polyploid. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of undergoing asymmetric division via nuclear budding and bursting (amitosis), giving rise to daughter cells that proliferate rapidly and can contribute to tumorigenesis (transformation of normal cells to a neoplastic state). This commentary provides an update on these intriguing discoveries. Abstract Cell division is crucial for the survival of living organisms. Human cells undergo three types of cell division: mitosis, meiosis, and amitosis. The former two types occur in somatic cells and germ cells, respectively. Amitosis involves nuclear budding and occurs in cells that exhibit abnormal nuclear morphology (e.g., polyploidy) with increased cell size. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of producing progeny via amitotic cell division, and that a subset of emerging daughter cells proliferate rapidly, exhibit stem cell-like properties, and can contribute to tumorigenesis. Polyploid cells that arise in solid tumors/tumor-derived cell lines are referred to as polyploid giant cancer cells (PGCCs) and are known to contribute to therapy resistance and disease recurrence following anticancer treatment. This commentary provides an update on some of these intriguing discoveries as a tribute to Drs. Walen and Rajaraman.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149407ed82c685a5b413633998b54affc7e6d4b2" target='_blank'>
              Amitotic Cell Division, Malignancy, and Resistance to Anticancer Agents: A Tribute to Drs. Walen and Rajaraman
              </a>
            </td>
          <td>
            R. Mirzayans, David Murray
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Herpesviruses are ubiquitous pathogens that cause a wide range of disease. Upon nuclear entry, their genomes associate with histones and chromatin modifying enzymes that regulate the progression of viral transcription and outcome of infection. While the composition and modification of viral chromatin has been extensively studied on bulk populations of infected cells by chromatin immunoprecipitation, this key regulatory process remains poorly defined at single-genome resolution. Here we use high-resolution quantitative imaging to investigate the spatial proximity of canonical and variant histones at individual Herpes Simplex Virus 1 (HSV-1) genomes within the first 90 minutes of infection. We identify significant population heterogeneity in the stable enrichment and spatial proximity of canonical histones (H2A, H2B, H3.1) at viral DNA (vDNA) relative to established promyelocytic leukaemia nuclear body (PML-NB) host factors that are actively recruited to viral genomes upon nuclear entry. We show the replication-independent histone H3.3/H4 chaperone Daxx to cooperate with PML to mediate the enrichment and spatial localization of variant histone H3.3 at vDNA that limits the rate of HSV-1 genome decompaction to restrict the progress of immediate-early (IE) transcription. This host response is counteracted by the viral ubiquitin ligase ICP0, which degrades PML to disperse Daxx and variant histone H3.3 from vDNA to stimulate the progression of viral genome expansion, IE transcription, and onset of HSV-1 replication. Our data support a model of intermediate and sequential histone assembly initiated by Daxx that limits the rate of HSV-1 genome decompaction independently of the stable enrichment of histones H2A and H2B at vDNA required to facilitate canonical nucleosome assembly. We identify HSV-1 genome decompaction upon nuclear infection to play a key role in the initiation and functional outcome of HSV-1 lytic infection, findings pertinent to the transcriptional regulation of many nuclear replicating herpesvirus pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd10788fc22834b17fdae65c2c26cce49f7d5c7d" target='_blank'>
              Daxx mediated histone H3.3 deposition on HSV-1 DNA restricts genome decompaction and the progression of immediate-early transcription
              </a>
            </td>
          <td>
            Ashley P.E. Roberts, Anne Orr, V. Iliev, Lauren Orr, S. McFarlane, Zhousiyu Yang, I. Epifano, C. Loney, Milagros Collados Rodriguez, Anna R. Cliffe, Kristen L Conn, Chris Boutell
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="TBC1D3 is a primate-specific gene family that has expanded in the human lineage and has been implicated in neuronal progenitor proliferation and expansion of the frontal cortex. The gene family and its expression have been challenging to investigate because it is embedded in high-identity and highly variable segmental duplications. We sequenced and assembled the gene family using long-read sequencing data from 34 humans and 11 non-human primate species. Our analysis shows that this particular gene family has independently duplicated in at least five primate lineages, and the duplicated loci are enriched at sites of large-scale chromosomal rearrangements on Chromosome 17. We find that all human copy number variation maps to two distinct clusters located at Chromosome 17q12 and that humans are highly structurally variable at this locus, differing by as many as 20 copies and ~1 Mbp in length depending on haplotypes. We also show evidence of positive selection, as well as a significant change in the predicted human TBC1D3 protein sequence. Lastly, we find that, despite multiple duplications, human TBC1D3 expression is limited to a subset of copies and, most notably, from a single paralog group: TBC1D3-CDKL These observations may help explain why a gene potentially important in cortical development can be so variable in the human population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872db936ea87de98bc9c479ae7fb31da39d2f13a" target='_blank'>
              Independent expansion, selection and hypervariability of the TBC1D3 gene family in humans.
              </a>
            </td>
          <td>
            Xavi Guitart, David Porubsky, DongAhn Yoo, Max L. Dougherty, Philip C. Dishuck, Katherine M. Munson, Alexandra P. Lewis, Kendra Hoekzema, Jordan Knuth, Stephen Chang, Tomi Pastinen, Evan E. Eichler
          </td>
          <td>2024-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>1</td>
          <td>87</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf251e653d85e2d3fa3eb31d13a78d3467c5985" target='_blank'>
              Bioinformatics advances in eccDNA identification and analysis.
              </a>
            </td>
          <td>
            Fuyu Li, Wenlong Ming, Wenxiang Lu, Ying Wang, Xianjun Dong, Yunfei Bai
          </td>
          <td>2024-08-29</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="BACKGROUND
Many plant secondary metabolites (PSMs) were shown to intercalate into DNA helix or interact with DNA grooves. This may influence histone-DNA interactions changeing chromatin structure and genome functioning.


METHODS
Nucleosome stability and linker histone H1.2, H1.4 and H1.5 localizations were studied in HeLa cells after the treatment with 15 PSMs, which are DNA-binders and possess anticancer activity according to published data. Chromatin remodeler CBL0137 was used as a control. Effects of PSMs were studied using fluorescent microscopy, flowcytometry, quantitative reverse transcriptase-polymerase chain reaction (RT-qPCR), western-blotting.


RESULTS
We showed that 1-hour treatment with CBL0137 strongly inhibited DNA synthesis and caused intensive linker histone depletion consistent with nucleosome destabilization. None of PSMs caused nucleosome destabilization, while most of them demonstrated significant influence on linker histone localizations. In particular, cell treatment with 11 PSMs at non-toxic concentrations induced significant translocation of the histone H1.5 to nucleoli and most of PSMs caused depletion of the histones H1.2 and H1.4 from chromatin fraction. Curcumin, resveratrol, berberine, naringenin, and quercetin caused significant redistribution of all three variants of the studied linker histones showing some overlap of PSM effects on linker histone DNA-binding. We demonstrated that PSMs, which induced the most significant redistribution of the histone H1.5 (berberine, curcumin and naringenin), influence the proportion of cells synthesizing DNA, expressing or non-expressing cyclin B and influence cell cycle distribution. Berberine induction of H1.5 translocations to nucleoli was shown to occur independently on the phases of cell cycle (metaphase was not analyzed).


CONCLUSIONS
For the first time we revealed PSM influence on linker histone location in cell nuclei that opens a new direction of PSM research as anticancer agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31259052a1bbdcbe810f26e91f0f54bf14b49fe4" target='_blank'>
              Anticancer Plant Secondary Metabolites Induce Linker Histone Depletion from Chromatin.
              </a>
            </td>
          <td>
            O. Vlasova, Irina Antonova, Roman Zenkov, Denis Naberezhnov, Gennady Belitsky, Anna Borunova, Tatiana Zabotina, Daniel García-Gomis, A. Safina, Katerina Gurova, A. Gudkov, Kirill Kirsanov, Albert Jordan, Marianna Yakubovskaya
          </td>
          <td>2024-08-05</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Rationale: Tumor cells remodel transcriptome to construct an ecosystem with stemness features, which maintains tumor growth and highly malignant characteristics. However, the core regulatory factors involved in this process still need to be further discovered. Methods: Single cell RNA-sequncing (scRNA-seq) and bulk RNA-sequencing profiles derived from fetal liver, normal liver, liver tumors, and their adjacent samples were collected to analyze the ecosystem of liver cancer. Mouse models were established to identify molecular functions of oncofetal-related oncogenes using hydrodynamic tail vein injection. Results: We found that liver cancer rebuilt oncofetal ecosystem to maintain malignant features. Interestingly, we identified a group of RNA-binding proteins (RBPs) that were highly overexpressed with oncofetal features. Among them, TRIM71 was specifically expressed in liver cancers and was associated with poor outcomes. TRIM71 drove the carcinogenesis of hepatocellular carcinoma (HCC), and knockdown of TRIM71 significantly abolished liver cancer cell proliferation. Mechanistically, TRIM71 formed a protein complex with IGF2BP1, bound to and stabilized the mRNA of CEBPA in an m6A-dependent manner, enhance the serine/glycine metabolic pathway, and ultimately promoted liver cancer progression. Furthermore, we identified that all-trans-retinoic acid (ATRA) combined with e1A binding protein p300 (EP300) inhibitor A-485 repressed TRIM71, attenuated glycine/serine metabolism, and inhibited liver cancer cell proliferation with high TRIM71 levels. Conclusions: We demonstrated the oncofetal status in liver cancer and highlighted the crucial role of TRIM71 and provided potential therapeutic strategies and liver cancer-specific biomarker for liver cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce6c5051059c8c5192cce1ee450847798b9ec45" target='_blank'>
              Oncofetal TRIM71 drives liver cancer carcinogenesis through remodeling CEBPA-mediated serine/glycine metabolism
              </a>
            </td>
          <td>
            Ying Su, Ziteng Li, Qin Li, Xinyi Guo, He-Yun Zhang, Yan Li, Zhiqiang Meng, Shenglin Huang, Zhixiang Hu
          </td>
          <td>2024-08-12</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Aflatoxin B1 (AFB1), a potent mycotoxin, is one of the environmental risk factors that cause liver cancer. In the liver, the bioactivated AFB1 intercalates into the DNA double helix to form a bulky DNA adduct which will lead to mutation if left unrepaired. Here, we adapted the tXR-seq method to measure the nucleotide excision repair of AFB1-induced DNA adducts at single-nucleotide resolution on a genome-wide scale, and compared it with repair data obtained from conventional UV-damage XR-seq. Our results showed that transcription-coupled repair plays a major role in the damage removal process. We further analyzed the distribution of nucleotide excision repair sites for AFB1-induced DNA adducts within the 3D human genome organization. Our analysis revealed a heterogeneous AFB1-dG repair across four different organization levels, including chromosome territories, A/B compartments, TADs, and chromatin loops. We found that chromosomes positioned closer to the nuclear center and regions within A compartments have higher levels of nucleotide excision repair. Notably, we observed high repair activity around both TAD boundaries and loop anchors. These findings provide insights into the complex interplay between AFB1-induced DNA damage repair, transcription, and 3D genome organization, shedding light on the mechanisms underlying AFB1-induced mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6fed0c3af4e712a403c0af2af9bb447bf31d09" target='_blank'>
              Nucleotide excision repair of aflatoxin-induced DNA damage within the 3D human genome organization.
              </a>
            </td>
          <td>
            Yiran Wu, Muhammad Muzammal Adeel, Dian Xia, Aziz Sancar, Wentao Li
          </td>
          <td>2024-09-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The mechanisms that work alongside BRAFV600E oncogene in melanoma development, in addition to ultraviolet (UV) radiation (UVR), are of great interest. Analysis of human melanoma tumors [data from The Cancer Genome Atlas (TCGA)] revealed that 50% or more of the samples expressed no or low amounts of serine/threonine protein kinase STK11 (also known as LKB1) protein. Here, we report that, in a mouse model, concomitant neonatal BrafV600E activation and Lkb1 tumor suppressor ablation in melanocytes led to full melanoma development. A single postnatal dose of UVB radiation had no effect on melanoma onset in Lkb1-depleted mice compared with BrafV600E-irradiated mice, but increased tumor multiplicity. In concordance with these findings and previous reports, Lkb1-null irradiated mice exhibited deficient DNA damage repair (DDR). Histologically, tumors lacking Lkb1 were enriched in neural-like tumor morphology. Genetic profiling and gene set enrichment analyses of tumor sample mutated genes indicated that loss of Lkb1 promoted the selection of altered genes associated with neural differentiation processes. Thus, these results suggest that the loss of Lkb1 cooperates with BrafV600E and UVR, impairing the DDR and increasing melanoma multiplicity and neural-like dedifferentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/468890566d37157a91439461401c20d4b0665a15" target='_blank'>
              Loss of Lkb1 cooperates with BrafV600E and ultraviolet radiation, increasing melanoma multiplicity and neural-like dedifferentiation.
              </a>
            </td>
          <td>
            K. McGrail, Elena González-Sánchez, Paula Granado-Martínez, Roberto Orsenigo, Yuxin Ding, Berta Ferrer, Javier Hernández-Losa, Iván Ortega, Juan Martín-Caballero, Eva Muñoz-Couselo, Vicente García-Patos, Juan A. Recio
          </td>
          <td>2024-08-08</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Targeting CDC20 can enhance the radiosensitivity of tumor cells, but the function and mechanism of CDC20 on DNA damage repair response remains vague. To examine that issue, tumor cell lines, including KYSE200, KYSE450, and HCT116, were utilized to detect the expression, function, and underlying mechanism of CDC20 in radio-chemoresistance. Western blot and immunofluorescence staining were employed to confirm CDC20 expression and location, and radiation could upregulate the expression of CDC20 in the cell nucleus. The homologous recombination (HR) and non-homologous end joining (NHEJ) reporter gene systems were utilized to explore the impact of CDC20 on DNA damage repair, indicating that CDC20 could promote HR repair and radio/chemo-resistance. In the early stages of DNA damage, CDC20 stabilizes the RPA1 protein through protein-protein interactions, activating the ATR-mediated signaling cascade, thereby aiding in genomic repair. In the later stages, CDC20 assists in the subsequent steps of damage repair by the ubiquitin-mediated degradation of RPA1. CCK-8 and colony formation assay were used to detect the function of CDC20 in cell vitality and proliferation, and targeting CDC20 can exacerbate the increase in DNA damage levels caused by cisplatin or etoposide. A tumor xenograft model was conducted in BALB/c-nu/nu mice to confirm the function of CDC20 in vivo, confirming the in vitro results. In conclusion, this study provides further validation of the potential clinical significance of CDC20 as a strategy to overcome radio-chemoresistance via uncovering a novel role of CDC20 in regulating RPA1 during DNA damage repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12ebddd7f8266727872b7593baec6eddbe1783f9" target='_blank'>
              CDC20 Holds Novel Regulation Mechanism in RPA1 during Different Stages of DNA Damage to Induce Radio-Chemoresistance
              </a>
            </td>
          <td>
            Yang Gao, Pengbo Wen, Chenran Shao, Cheng Ye, Yuji Chen, Junyu You, Zhongjing Su
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="While T cell lymphomas are classified as mature neoplasms, emerging evidence indicates that malignant transformation may occur at an earlier stage of T cell maturation. In this study, we determined clonal architectures in a broad range of T cell lymphomas. Our multidimensional profiling indicates that a large part of these lymphomas in fact emerge from an immature lymphoid T cell precursor at a maturation stage prior to V(D)J rearrangement that undergoes branching evolution. Consequently, at single cell resolution we observed considerable clonal tiding under selective therapeutic pressure. T cell receptor next-generation sequencing suggested a highly biased usage of TRBV20-1 gene segments as part of multiple antigen receptor rearrangements per patient. The predominance of TRBV20-1 was found across all major T cell lymphoma subtypes analyzed. This suggested that this particular V gene - independently of complementarity-determining region 3 (CDR3) configuration - may represent a driver of malignant transformation. Together, our data indicate that T cell lymphomas derive from immature lymphoid precursors and display considerable intratumoral heterogeneity that may provide the basis for relapse and resistance in these hard-to-treat cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5cc9e8d39a7b3561d529df56bcd87a167e0b44e" target='_blank'>
              T-cell receptor architecture and clonal tiding provide insight into the transformation trajectory of peripheral T-cell lymphomas.
              </a>
            </td>
          <td>
            E. Willscher, C. Schultheiss, L. Paschold, Franziska Lea Schümann, Paul Schmidt-Barbo, Benjamin Thiele, M. Bauer, Claudia Wickenhauser, Thomas Weber, Mascha Binder
          </td>
          <td>2024-08-29</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8200f580864196977ce1aaadc3c51eb18d6c8f71" target='_blank'>
              Evolutionary patterns and functional effects of 3D chromatin structures in butterflies with extensive genome rearrangements
              </a>
            </td>
          <td>
            Botong Zhou, Ping Hu, Guichun Liu, Zhou Chang, Zhiwei Dong, Zihe Li, Yuan Yin, Zunzhe Tian, Ge Han, Wen Wang, Xueyan Li
          </td>
          <td>2024-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca0012968a9023f1a5b2833b62edce18fdae052" target='_blank'>
              LncRNA DNAH17-AS1 promotes gastric cancer proliferation and radioresistance by sponging miR-202-3p to upregulate ONECUT2
              </a>
            </td>
          <td>
            Yugang Ge, Hui Cang, J-Q Xiao, Hongshuai Wu, Biao Wang, Qing Shao
          </td>
          <td>2024-09-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CRISPR-Associated Transposases (CASTs) hold tremendous potential for microbial genome editing due to their ability to integrate large DNA cargos in a programmable and site-specific manner. However, the widespread application of CASTs has been hindered by their low efficiency in diverse, non-model bacteria. In an effort to address this shortcoming, we conducted the first genome-wide screen for host factors impacting Vibrio cholerae CAST (VchCAST) activity and used the findings to increase VchCAST editing efficiency. A genome-wide loss-of-function mutant library in E. coli was screened to identify 15 genes that impact type VchCAST transposition. Of these, seven factors were validated to improve VchCAST activity and two were found to be inhibitory. Informed by homologous recombination involved effectors, RecD and RecA, we tested the λ-Red recombineering system in our VchCAST editing vectors, which increased its insertion meditated-editing efficiency by 25.7-fold in E. coli while maintaining high target specificity and similar insertion arrangements. Furthermore, λ-Red-enhanced VchCAST achieved increased editing efficiency in the industrially important bacteria Pseudomonas putida and the emerging pathogen Klebsiella michiganensis. This study improves understanding of factors impacting VchCAST activity and enhances its efficiency as a bacterial genome editor. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2483d77579dd4ab8c5f977694f46b22d04f0a6" target='_blank'>
              Identification of Proteins Influencing CRISPR-Associated Transposases for Enhanced Genome Editing
              </a>
            </td>
          <td>
            Leo Song, Amanda T.P. Alker, Agnès Oromí-Bosch, Sophia E. Swartz, Jonathan N.V. Martinson, Jigyasa Arora, Abby M. Wang, Rachel Rovinsky, Sara J. Smith, Emily C. Pierce, A. Deutschbauer, Jennifer A. Doudna, Brady F. Cress, Benjamin E. Rubin
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Studies of bacterial adaptation and evolution are hampered by the difficulty of measuring traits such as virulence, drug resistance, and transmissibility in large populations. In contrast, it is now feasible to obtain high-quality complete assemblies of many bacterial genomes thanks to scalable high-accuracy long-read sequencing technologies. To exploit this opportunity, we introduce a phenotype- and alignment-free method for discovering coselected and epistatically interacting genomic variation from genome assemblies covering both core and accessory parts of genomes. Our approach uses a compact colored de Bruijn graph to approximate the intragenome distances between pairs of loci for a collection of bacterial genomes to account for the impacts of linkage disequilibrium (LD). We demonstrate the versatility of our approach to efficiently identify associations between loci linked with drug resistance and adaptation to the hospital niche in the major human bacterial pathogens Streptococcus pneumoniae and Enterococcus faecalis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ba999d01a93235712556c85e81e52a6ac50e10" target='_blank'>
              Pangenome-spanning epistasis and coselection analysis via de Bruijn graphs
              </a>
            </td>
          <td>
            Juri Kuronen, Samuel T. Horsfield, A. K. Pöntinen, S. Mallawaarachchi, S. Arredondo-Alonso, Harry A. Thorpe, R. Gladstone, Rob J. L. Willems, Stephen D. Bentley, N. Croucher, J. Pensar, John A. Lees, G. Tonkin-Hill, Jukka Corander
          </td>
          <td>2024-07-01</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="Uterine cancer is the most common gynecologic malignancy in the United States, with endometrioid endometrial adenocarcinoma (EC) being the most common histologic sub-type. Considering the molecular classifications of EC, efforts have been made to identify additional biomarkers that can assist in diagnosis, prognosis, and individualized therapy. We sought to explore the relationship of Repressor Element 1 (RE1) silencing transcription factor (REST), which downregulates neuronal genes in non-neuronal tissue, along with matrix metalloproteinase-24 (MMP24) and EC. We analyzed the expression of REST and MMP24 in 31 cases of endometrial cancer and 16 controls. We then explored the baseline expression of REST and MMP24 in two EC cell lines (Ishikawa and HEC-1-A) compared to a benign cell line (t-HESC) and subsequently evaluated proliferation, migration, and invasion in the setting of loss of REST gene expression. REST and MMP24 expression were significantly lower in human EC samples compared to control samples. REST was highly expressed in EC cell lines, but decreasing REST gene expression increased proliferation (FC: 1.13X, p < 0.0001), migration (1.72X, p < 0.0001), and invasion (FC: 7.77X, p < 0.05) in Ishikawa cells, which are hallmarks of cancer progression and metastasis. These findings elicit a potential role for REST as a putative tumor suppressor in EC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d640a56866f332d95647c4ba33b72698badf61e" target='_blank'>
              Functional Analysis of RE1 Silencing Transcription Factor as a Putative Tumor Suppressor in Human Endometrial Cancer
              </a>
            </td>
          <td>
            Yasmin Abedin, P. Minchella, Riley Peterson, F. Gonnella, Amanda Graham, Ian Cook, Melissa Javellana, Andrea Jewell, Lori Spoozak, Warren B. Nothnick
          </td>
          <td>2024-09-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The epigenetic layer of regulation has become increasingly relevant in the research focused on tumor suppressors. KLF4 is a well-described zinc-finger transcription factor, mainly known for its role in the acquisition of cell pluripotency. Here we report and describe the most relevant epigenetic regulation mechanisms that affect KLF4 expression in tumors. CpG island methylation emerges as the most common mechanism in several tumors including lung adenocarcinoma, hepatocellular carcinoma, non-Hodgkin lymphomas, among others. Further layers of regulation represented by histone methylation and acetylation and by non-coding RNAs are described. Overall, KLF4 emerges as a crucial target in the fight against cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e9c4ef46a7844e1c0cc57e2b10d6f8fefd7118" target='_blank'>
              KLF4 is an epigenetically modulated, context-dependent tumor suppressor
              </a>
            </td>
          <td>
            R. Frazzi
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="CCR4-NOT regulates multiple steps in gene regulation, including transcription, mRNA decay, protein ubiquitylation, and translation. It has been well studied in budding yeast; however, relatively less is known about its regulation and functions in mammals. To characterize the functions of the human CCR4-NOT complex, we developed a rapid auxin-induced degron system to deplete CNOT1 (the scaffold of the complex) and CNOT4 (E3 ubiquitin ligase) in cell culture. Transcriptome-wide measurements of gene-expression revealed that depleting CNOT1 changed several thousand transcripts, wherein most mRNAs were increased and resulted in a global decrease in mRNA decay rates. In contrast to what was observed in CNOT1-depleted cells, CNOT4 depletion only modestly changed RNA steady-state levels and, surprisingly, led to a global acceleration in mRNA decay. To further investigate the role of CCR4-NOT in transcription, we used transient transcriptome sequencing (TT-seq) to measure ongoing RNA synthesis. Depletion of either subunit resulted in increased RNA synthesis of several thousand genes. In contrast to most of the genome, a rapid reduction in the synthesis of KRAB-Zinc-Finger-proteins (KZNFs) genes, especially those clustered on chromosome 19, was observed. KZNFs are transcriptional repressors of retro-transposable elements (rTEs), and consistent with the decreased KZNFs expression, we observed a significant and rapid activation of rTEs, mainly Long interspersed Nuclear Elements (LINEs). Our data reveal that CCR4-NOT regulates gene expression and silences retrotransposons across the genome by maintaining KZNF expression. These data establish CCR4-NOT as a global regulator of gene expression, and we have identified a novel mammalian-specific function of the complex, the suppression of rTEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a32fd100fbc0ada50fb682b865f09d95280e8fa6" target='_blank'>
              Human CCR4-NOT is a global regulator of gene expression and is a novel silencer of retrotransposon activation
              </a>
            </td>
          <td>
            S. Kulkarni, Alexis Morrissey, A. Sebastian, C. Keller, B. Giardine, Courtney Smith, O. Akinniyi, A. Arnaoutov, István Albert, Shaun Mahony, J.C. Reese
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly detrimental to cells, as improper repair can result in inheritable genetic rearrangements or cell death. The role of RNA:DNA hybrids (RDHs) in DSB repair remains poorly understood, but their transient accumulation and subsequent resolution are crucial for accurate repair. The absence of the end-joining factor Nej1 at DSBs significantly reduced RDH levels, which was linked to increased activity of the Dna2 nuclease. Dna2 limits the accumulation of hybrids at DSBs, with levels rising in the presence of a nuclease-dead Dna2 and RNH201 deletion. Dna2 has a heightened preference for resolving RDHs with 5’ RNA overhangs compared to duplex substrates with 5’ DNA overhangs. This selective resolution by Dna2 helps restrict hybrid accumulation at DSBs and promotes resection, a function not shared by Exo1. This study underscores the multifunctional roles of canonical repair factors in ensuring efficient homology-directed repair. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5b2cacdb3d323609fc9f634d2ea46c9da9e5cdc" target='_blank'>
              Dna2 nuclease resolves RNA:DNA hybrids at double-strand breaks
              </a>
            </td>
          <td>
            A. Mojumdar, Courtney Granger, Jennifer A. Cobb
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4568f027a9bf73e7d22e0523c166497f5edf7122" target='_blank'>
              Perturbing TET2 condensation promotes aberrant genome-wide DNA methylation and curtails leukaemia cell growth.
              </a>
            </td>
          <td>
            Lei Guo, Tingting Hong, Yi-Tsang Lee, Xue Hu, Guokai Pan, Rongjie Zhao, Yuhan Yang, Jingwen Yang, Xiaoli Cai, Logan Rivera, Jie Liang, Rui Wang, Yaling Dou, Srikanth Kodali, Wenbo Li, Leng Han, Bruno Di Stefano, Yubin Zhou, Jia Li, Yun Huang
          </td>
          <td>2024-09-09</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407d56e1f37d6b7ede8b27130702ccb149beda6e" target='_blank'>
              Multiomics and machine learning-based analysis of pancancer pseudouridine modifications
              </a>
            </td>
          <td>
            Jiheng Zhang, Lei Xu, Xiuwei Yan, Jiahe Hu, Xin Gao, Hongtao Zhao, Mo Geng, Nan Wang, Shaoshan Hu
          </td>
          <td>2024-08-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667549573815f8b84f3de00bdda0de7e5af3d623" target='_blank'>
              Emerin mislocalization during chromatin bridge resolution can drive prostate cancer cell invasiveness in a collagen-rich microenvironment.
              </a>
            </td>
          <td>
            M. Popęda, Kamil Kowalski, Tomasz Wenta, Galina V Beznoussenko, Michał Rychłowski, Alexander A. Mironov, Zeno Lavagnino, Sara Barozzi, Julia Richert, Rebecca Bertolio, Kamil Myszczyński, J. Szade, Michał Bieńkowski, Kevin Miszewski, Marcin Matuszewski, A. Żaczek, Luca Braga, Giannino Del Sal, N. Bednarz-Knoll, P. Maiuri, P. Nastały
          </td>
          <td>2024-09-02</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c2256152e45d11bd82f68a7e3810e940b41cb02" target='_blank'>
              Essential gene screening identifies the bromodomain-containing protein BRPF1 as a new actionable target for endocrine therapy-resistant breast cancers
              </a>
            </td>
          <td>
            A. Salvati, G. Giurato, J. Lamberti, Ilaria Terenzi, Laura Crescenzo, Viola Melone, L. Palo, A. Giordano, Francesco Sabbatino, G. Roscigno, C. Quintavalle, G. Condorelli, Francesca Rizzo, Roberta Tarallo, G. Nassa, Alessandro Weisz
          </td>
          <td>2024-08-07</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84febdf4c77dd510d0a0709248848e1311db0643" target='_blank'>
              Biological and clinical relevance of correlated expression levels of coding and long noncoding RNAs in HPV16 positive cervical cancers
              </a>
            </td>
          <td>
            Abhisikta Ghosh, Abarna Sinha, Arnab Ghosh, Somrita Roy, Sumana Mallick, Vinoth Kumar, S. Mathai, Jaydip Bhaumik, Asima Mukhopadhyay, Saugata Sen, Aditi Chandra, A. Maitra, N. Biswas, Partha P. Majumder, Sharmila Sengupta
          </td>
          <td>2024-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Satellite DNAs (satDNAs) are tandemly repeated sequences that make up a significant portion of almost all eukaryotic genomes. Although satDNAs have been shown to play a very important role in genome organization and evolution, they are relatively poorly analysed even in model. One of the main reasons for the current lack of in-depth studies on satDNAs is their underrepresentation in genome assemblies. The complexity and highly repetitive nature of satDNAs make their analysis challenging, and there is a need for efficient tools that can ensure accurate annotation and analysis of satDNAs. We present a novel pipeline, named Satellite DNA Exploration (SatXplor), designed to robustly characterize satDNA elements and analyse their arrays and flanking regions. SatXplor is benchmarked against curated satDNA datasets from diverse species, showcasing its versatility across genomes with varying complexities and different satDNA profile. Component algorithms excel in the identification of tandemly repeated sequences and for the first time enable evaluation of satDNA variation and array annotation with the addition of information about surrounding genomic landscape. SatXplor is an innovative pipeline for satDNA analysis that can be paired with any tool used for satDNA detection, offering insights into the structural characteristics, array determination and genomic context of satDNA elements. By integrating various computational techniques, from sequence analysis and homology investigation to advanced clustering and graph-based methods, it provides a versatile and comprehensive approach to explore the complexity of satDNA organization and to understand the underlying mechanisms and evolutionary aspects. It is open-source and freely accessible at https://github.com/mvolar/SatXplor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93bcbb026ce597eb6926d4d48baa8708101680" target='_blank'>
              SatXplor – A comprehensive pipeline for satellite DNA analyses in complex genome assemblies
              </a>
            </td>
          <td>
            Marin Volarić, N. Meštrović, Evelin Despot-Slade
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Simple Summary Forkhead-box M1 (FOXM1) is a proliferation-associated transcription factor, overexpressed in almost all the cancers, making its regulation worth investigating. FOXM1 is known to bind to the promoters of certain microRNAs, a class of biomolecules involved in post-transcriptional gene regulation. Database mining followed by Venn analysis led us to the shortlisting of seven microRNAs, among which microRNA-532-3p showed the strongest interaction with FOXM1 3’UTR. The ectopic overexpression of miR-532-3p in colorectal cancer (CRC) cells led to decreased FOXM1 levels which resulted in the alteration of CRC functionalities. These altered phenotypes were supported by a change in the expression of proliferation and EMT markers. In conclusion, the present study suggested that the regulation of FOXM1 by microRNA-532-3p via its interaction with FOXM1 3’UTR inhibited CRC proliferation, migration, and invasion. Abstract Colorectal cancer (CRC) is a heterogeneous disease and classified into various subtypes, among which transcriptional alterations result in CRC progression, metastasis, and drug resistance. Forkhead-box M1 (FOXM1) is a proliferation-associated transcription factor which is overexpressed in CRC and the mechanisms of FOXM1 regulation have been under investigation. Previously, we showed that FOXM1 binds to promoters of certain microRNAs. Database mining led to several microRNAs that might interact with FOXM1 3’UTR. The interactions between shortlisted microRNAs and FOXM1 3’UTR were quantitated by a dual-luciferase reporter assay. MicroRNA-532-3p interacted with the 3’UTR of the FOXM1 mRNA transcript most efficiently. MicroRNA-532-3p was ectopically overexpressed in colorectal cancer (CRC) cell lines, leading to reduced transcript and protein levels of FOXM1 and cyclin B1, a direct transcriptional target of FOXM1. Further, a clonogenic assay was conducted in overexpressed miR-532-3p CRC cells that revealed a decline in the ability of cells to form colonies and a reduction in migratory and invading potential. These alterations were reinforced at molecular levels by the altered transcript and protein levels of the conventional EMT markers E-cadherin and vimentin. Overall, this study identifies the regulation of FOXM1 by microRNA-532-3p via its interaction with FOXM1 3’UTR, resulting in the suppression of proliferation, migration, and invasion, suggesting its role as a tumor suppressor in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/affbc7e03338ebd318b510b5965cce7c6dfa9311" target='_blank'>
              MicroRNA-532-3p Modulates Colorectal Cancer Cell Proliferation and Invasion via Suppression of FOXM1
              </a>
            </td>
          <td>
            Nita Ahuja, Akira Yamauchi, David Wong, Ketakee Mahajan, Ani V. Das, S. Alahari, Ramesh Pothuraju, S. A. Nair
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Myxofibrosarcoma (MFS) and undifferentiated pleomorphic sarcoma (UPS) are two common and aggressive subtypes of soft tissue sarcoma. The aim of this study was to assess potential transcriptomic differences between MFS and UPS tumours and to evaluate the extent to which differences in gene expression profiles were associated with genomic and clinical features. The study included 162 patients with tumours diagnosed as MFS (N = 62) or UPS (N = 100). The patients had been diagnosed and treated at two Swedish sarcoma centres during a 30-year period. For gene expression profiling and gene fusion detection all tumours were analysed using RNA-sequencing and could be compared with data on clinical outcome (N = 155), global copy number profiles (N = 145), and gene mutations (N = 128). Gene expression profiling revealed three transcriptomic clusters (TCs) without any clear separation of MFS and UPS. One TC was associated with longer metastasis-free survival. These tumours had lower tumour mutation burden (TMB), were enriched for a copy number signature representative of focal LOH and chromosomal instability on a diploid background, and were relatively immune-depleted. MFS and UPS showed extensive genomic overlap, with whole genome doubling occurring more frequently among the latter. The results support the idea that MFS and UPS tumours have largely overlapping genomic and transcriptomic features, with UPS tumours showing more aggressive behaviour and more complex genomes. Independently of the tumour type, clinically relevant subgroups were revealed by gene expression analysis, and the finding of multiple genomic subgroups strongly suggest the existence of subgroups of relevance to treatment stratification. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cdad99fbec5c71439ba17d6c812cff6bcce80b" target='_blank'>
              Transcriptomic profiles of myxofibrosarcoma and undifferentiated pleomorphic sarcoma correlate with clinical and genomic features.
              </a>
            </td>
          <td>
            Shamik Mitra, Akanksha Farswan, Paul Piccinelli, Saskia Sydow, A. Hesla, P. Tsagkozis, F. Vult von Steyern, Martin Almqvist, Mikael Eriksson, L. Magnusson, J. Nilsson, N. Pillay, Fredrik Mertens
          </td>
          <td>2024-09-11</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Characterizing DNA methylation patterns is important for addressing key questions in evolutionary biology, geroscience, and medical genomics. While costs are decreasing, whole-genome DNA methylation profiling remains prohibitively expensive for most population-scale studies, creating a need for cost-effective, reduced representation approaches (i.e., assays that rely on microarrays, enzyme digests, or sequence capture to target a subset of the genome). Most common whole genome and reduced representation techniques rely on bisulfite conversion, which can damage DNA resulting in DNA loss and sequencing biases. Enzymatic methyl sequencing (EM-seq) was recently proposed to overcome these issues, but thorough benchmarking of EM-seq combined with cost-effective, reduced representation strategies has not yet been performed. To do so, we optimized Targeted Methylation Sequencing protocol (TMS)—which profiles ∼4 million CpG sites—for miniaturization, flexibility, and multispecies use at a cost of ∼$80. First, we tested modifications to increase throughput and reduce cost, including increasing multiplexing, decreasing DNA input, and using enzymatic rather than mechanical fragmentation to prepare DNA. Second, we compared our optimized TMS protocol to commonly used techniques, specifically the Infinium MethylationEPIC BeadChip (n=55 paired samples) and whole genome bisulfite sequencing (n=6 paired samples). In both cases, we found strong agreement between technologies (R² = 0.97 and 0.99, respectively). Third, we tested the optimized TMS protocol in three non-human primate species (rhesus macaques, geladas, and capuchins). We captured a high percentage (mean=77.1%) of targeted CpG sites and produced methylation level estimates that agreed with those generated from reduced representation bisulfite sequencing (R² = 0.98). Finally, we applied our protocol to profile age-associated DNA methylation variation in two subsistence-level populations—the Tsimane of lowland Bolivia and the Orang Asli of Peninsular Malaysia—and found age-methylation patterns that were strikingly similar to those reported in high income cohorts, despite known differences in age-health relationships between lifestyle contexts. Altogether, our optimized TMS protocol will enable cost-effective, population-scale studies of genome-wide DNA methylation levels across human and non-human primate species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ea49a3598e0f07df9c655927729fa7dc601194" target='_blank'>
              Cost-effective solutions for high-throughput enzymatic DNA methylation sequencing
              </a>
            </td>
          <td>
            Amy Longtin, Marina M. Watowich, Baptiste Sadoughi, Rachel M. Petersen, Sarah F. Brosnan, Kenneth Buetow, Qiuyin Cai, M. Gurven, Heather M. Highland, Yi-Ting Huang, Hillard Kaplan, Thomas S Kraft, Yvonne A L Lim, J. Long, Amanda D. Melin, Jamie Roberson, Kee-Seong Ng, J. Stieglitz, B. Trumble, Vivek V Venkataraman, Ian J Wallace, Jie Wu, N. Snyder‐Mackler, Angela Jones, Alexander G. Bick, Amanda J. Lea
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bba6936c2ac2d76bcee0a653b32c7b9ef4038fa" target='_blank'>
              Yeast EndoG prevents genome instability by degrading extranuclear DNA species
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Ruofan Yu, P. Thakre, Brenna S. McCauley, Nic Nikoloutsos, Yang Yu, Qian Li, P. Hastings, Weiwei Dang, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-09-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/744ca5586c9c614b8098bc8c2e1803f3bc1c7ab2" target='_blank'>
              An atlas of genetic effects on cellular composition of the tumor microenvironment.
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Can Chen, Ying Zhu, Zhirui Chen, Zuyou Wu, Jingyi Peng, Xu Zhu, Ziying Liu, Bin Li, Ming Zhang, Jinyu Huang, Yanmin Li, Yizhuo Liu, Qianying Ma, Chunyi He, Shuo Chen, Wen Tian, Linyun Fan, Caibo Ning, Hui Geng, Bin Xu, Haijie Li, Xu Zhu, Jun Fang, Xiaoyang Wang, Shao-Kai Zhang, Meng Jin, Chaoqun Huang, Xiaojun Yang, Jianbo Tian, Xiaoping Miao
          </td>
          <td>2024-09-02</td>
          <td>Nature immunology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c1e4b7568509ca6d857531ee73ea8a0c9b0585" target='_blank'>
              Decoding the genomic landscape of chromatin-associated biomolecular condensates
              </a>
            </td>
          <td>
            Zhaowei Yu, Qi Wang, Qichen Zhang, Yawen Tian, Guo Yan, Jidong Zhu, Guangya Zhu, Yong Zhang
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aed930352660788bbfcd1cc9c93c4e4f947acaa" target='_blank'>
              PD-L1 deglycosylation promotes its nuclear translocation and accelerates DNA double-strand-break repair in cancer
              </a>
            </td>
          <td>
            Zhen Shu, Bhakti Dwivedi, Jeffrey M. Switchenko, David S Yu, Xingming Deng
          </td>
          <td>2024-08-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Myc is a major driver of tumor initiation, progression, and maintenance. Up-regulation of Myc protein level rather than acquisition of neomorphic properties appears to underlie most Myc-driven cancers. Cellular mechanisms governing Myc expression remain incompletely defined. In this study, we show that ribosome-associated quality control (RQC) plays a critical role in maintaining Myc protein level. Ribosomes stall during the synthesis of the N-terminal portion of cMyc, generating aberrant cMyc species and necessitating deployment of the early RQC factor ZNF598 to handle translational stress and restore cMyc translation. ZNF598 expression is up-regulated in human glioblastoma (GBM), and its expression positively correlates with that of cMyc. ZNF598 knockdown inhibits human GBM neurosphere formation in cell culture and Myc-dependent tumor growth in vivo in Drosophila. Intriguingly, the SARS-COV-2-encoded translational regulator Nsp1 impinges on ZNF598 to restrain cMyc translation and consequently cMyc-dependent cancer growth. Remarkably, Nsp1 exhibits synthetic toxicity with the translation and RQC-related factor ATP-binding cassette subfamily E member 1, which, despite its normally positive correlation with cMyc in cancer cells, is co-opted by Nsp1 to down-regulate cMyc and inhibit tumor growth. Ribosome stalling during c-myc translation thus offers actionable cancer cell vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3e20c4cb54679180301d5c681441f377ad983fd" target='_blank'>
              Ribosome stalling during c-myc translation presents actionable cancer cell vulnerability
              </a>
            </td>
          <td>
            T. P. Khaket, Suman Rimal, Xingjun Wang, Sunil Bhurtel, Yen-Chi Wu, Bingwei Lu
          </td>
          <td>2024-08-01</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7688d245ed2ef09eafd9e57de454a0d8db59f71a" target='_blank'>
              Transposable elements contribute to tissue-specific gene regulation in humans.
              </a>
            </td>
          <td>
            Arsala Ali, Ping Liang
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Introduction Accurate reference genomes are fundamental to understanding biological evolution, biodiversity, hereditary phenomena and diseases. However, many assembled nuclear chromosomes are often contaminated by organelle genomes, which will mislead bioinformatic analysis, and genomic and transcriptomic data interpretation. Methods To address this issue, we developed a tool named Chlomito, aiming at precise identification and elimination of organelle genome contamination from nuclear genome assembly. Compared to conventional approaches, Chlomito utilized new metrics, alignment length coverage ratio (ALCR) and sequencing depth ratio (SDR), thereby effectively distinguishing true organelle genome sequences from those transferred into nuclear genomes via horizontal gene transfer (HGT). Results The accuracy of Chlomito was tested using sequencing data from Plum, Mango and Arabidopsis. The results confirmed that Chlomito can accurately detect contigs originating from the organelle genomes, and the identified contigs covered most regions of the organelle reference genomes, demonstrating efficiency and precision of Chlomito. Considering user convenience, we further packaged this method into a Docker image, simplified the data processing workflow. Discussion Overall, Chlomito provides an efficient, accurate and convenient method for identifying and removing contigs derived from organelle genomes in genomic assembly data, contributing to the improvement of genome assembly quality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2949fcd41e909221ac38423a6ad3cf2f0aef961d" target='_blank'>
              Chlomito: a novel tool for precise elimination of organelle genome contamination from nuclear genome assembly
              </a>
            </td>
          <td>
            Wei Song, Chong Li, Yanming Lu, Dawei Shen, Yunxiao Jia, Yixin Huo, Weilan Piao, Hua Jin
          </td>
          <td>2024-08-27</td>
          <td>Frontiers in Plant Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64123a29f4e4d91a84682b8e0733680d1726f975" target='_blank'>
              Phase separation of phospho-HDAC6 drives aberrant chromatin architecture in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Bing Lu, Ru Qiu, Jiatian Wei, Li Wang, Qinkai Zhang, Mingsen Li, Xiudan Zhan, Jian Chen, I-yun Hsieh, Ciqiu Yang, Jing Zhang, Zicheng Sun, Yifan Zhu, Tao Jiang, Han Zhu, Jie Li, Wei Zhao
          </td>
          <td>2024-08-28</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/311bc754350204df7366f1cc87322a6aee4bc5a6" target='_blank'>
              Assessment of SWI/SNF chromatin remodeling complex related genes as potential biomarkers and therapeutic targets in pan-cancer
              </a>
            </td>
          <td>
            Kai Zhuang, Lishan Wang, Chengyu Lu, Zhiping Liu, Dongli Yang, Hao Zhong, Jiami Zou, Aamir Fahira, Jiaojiao Wang, Zunnan Huang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b02e26ddc26802806307ff2a8acf516e9dc75bb" target='_blank'>
              Exosomal transcript cargo and functional correlation with HNSCC patients’ survival
              </a>
            </td>
          <td>
            Joni Yadav, A. Chaudhary, Tanya Tripathi, Divya Janjua, Udit Joshi, N. Aggarwal, Arun Chhokar, Chetkar Chandra Keshavam, Anna Senrung, A. Bharti
          </td>
          <td>2024-09-13</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Noncoding single nucleotide variants are the predominant class of genetic variation in whole genome sequencing and are key drivers of phenotypic variation. However, their functional annotation remains challenging. To address this, we develop a hypothesis-driven functional annotation scheme for CTCF binding sites given CTCF’s critical roles in gene regulation and extensive profiling in regulatory datasets. We synthesize CTCF’s binding patterns at 1,063,879 genomic loci across 214 biological contexts into a summary metric, which we refer to as binding activity. We find that binding activity is significantly enriched for both conserved nucleotides (Pearson R = 0.31, p < 2.2 x 10-16) and sequences that contain high-quality CTCF binding motifs (Pearson R = 0.63, p = 2.9 x 10-12). We then integrate binding activity with high confidence change in precision weight matrix scores. By applying this framework to 1,253,330 SNVs in gnomAD, we explore signatures of selection acting against the disruption of CTCF binding. We find a strong, positive relationship between the mutability adjusted proportion of singletons (MAPS) metric and the loss of CTCF binding at loci with high in vitro activity (Pearson R = 0.67, p = 1.5 x 10-14). To contextualize these findings, we apply MAPS to other functional classes of variation and find that a subset of 198,149 loss of CTCF binding variants are observed as infrequently as missense variants. This work implicates these thousands of rare, noncoding variants that disrupt CTCF binding for further functional studies while providing a blueprint for the interpretable annotation of noncoding variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea605606df9f3fb36737c3abacedd277ab50869b" target='_blank'>
              Identifying deleterious noncoding variation through gain and loss of CTCF binding activity
              </a>
            </td>
          <td>
            Colby Tubbs, Mary Lauren Benton, Evonne McArthur, John A. Capra, Douglas M. Ruderfer
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) is a lethal disease with limited therapies. In our screening using patient-derived organoids, we identified cyclin-dependent kinase 12 (CDK12) as a promising target in a subset of PDAC. Both pharmacological and genetic inhibition of CDK12 exerted potent antitumor activity in vitro and in vivo, inducing double-strand DNA breaks and apoptosis. CDK12 is a member of cyclin-dependent kinases that is involved in gene expression regulations at multiple levels. Recently, CDK12 has gained attention as a therapeutic target in several types of cancers including triple-negative breast cancer and Ewing sarcoma. However, little is known about the impact and possibility of CDK12 inhibition in pancreatic cancer treatment. As the lack of specific inhibitors that selectively target CDK12, studying CDK12 has long been challenging. To overcome this, we used CRISPR-Cas9 mediated genome editing to generate FKBP12F36V (dTAG) knock-in PDAC cell lines, enabling rapid elimination of endogenous CDK12 using a degrader against dTAG (dTAG-13). Quantitative-image-based cytometry and time lapse imaging showed that CDK12 ablation induces mitotic arrest, DNA damage and apoptosis (mitotic catastrophe). Specifically, loss of CDK12 kinase activity in the S phase was sufficient to slow DNA replication speed, leading to spindle assembly checkpoint activation and mitotic catastrophe. Importantly, cells that escaped mitotic death accumulate micronuclei, triggering immune response signaling that is mediated by the cGAS-STING-type I interferon pathway. Our results suggest that CDK12 is a promising target in a subset of PDAC patients, where CDK12 inhibition exerts antitumor activity both in cell autonomous and non-cell autonomous ways through inducing mitotic catastrophe as well as triggering immune response.
 Citation Format: Dosuke Iwadate, Keisuke Yamamoto, Hiroyuki Kato, Keisuke Tateishi, Mitsuhiro Fujishiro. CDK12 inhibition drives mitotic catastrophe, triggering genome instability and innate immune response in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A084.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/632d48b78e0c9131037bdfa10d93bd71182ca006" target='_blank'>
              Abstract A084: CDK12 inhibition drives mitotic catastrophe, triggering genome instability and innate immune response in pancreatic cancer
              </a>
            </td>
          <td>
            Dosuke Iwadate, Keisuke Yamamoto, Hiroyuki Kato, Keisuke Tateishi, Mitsuhiro Fujishiro
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de95d0b10e54e93d13fb87142565e8cdb7d31d26" target='_blank'>
              FAM109B plays a tumorigenic role in low-grade gliomas and is associated with tumor-associated macrophages (TAMs)
              </a>
            </td>
          <td>
            Zhe Zhang, Yao Xiao, Siyi Zhao, Jun Liu, Jie Zeng, Feng Xiao, Bin Liao, Xuesong Shan, Hong Zhu, Hua Guo
          </td>
          <td>2024-09-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d24eccfe89ea8e14aa08f4e83d855de3df06a6b0" target='_blank'>
              EWS-WT1 fusion isoforms establish oncogenic programs and therapeutic vulnerabilities in desmoplastic small round cell tumors
              </a>
            </td>
          <td>
            Gaylor Boulay, Liliane C. Broye, Rui Dong, Sowmya Iyer, Rajendran Sanalkumar, Yu-Hang Xing, Rémi Buisson, Shruthi Rengarajan, Beverly Naigles, Benoît Duc, A. Volorio, Mary E. Awad, R. Renella, I. Chebib, G. P. Nielsen, Edwin Choy, Gregory M Cote, Lee Zou, Igor Letovanec, Ivan Stamenkovic, Miguel N. Rivera, Nicolò Riggi
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, Roberto Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9824378cb72ff7a6ac9e3450ad8fb3f2d8abaa3a" target='_blank'>
              Human genomic DNA is widely interspersed with i-motif structures.
              </a>
            </td>
          <td>
            C. D. Peña Martinez, Mahdi Zeraati, R. Rouet, Ohan Mazigi, Jake Y Henry, Brian Gloss, J. Kretzmann, Cameron W Evans, E. Ruggiero, Irene Zanin, Maja Marušič, J. Plavec, Sara N. Richter, Tracy M. Bryan, Nicole M Smith, Marcel E. Dinger, Sarah Kummerfeld, Daniel Christ
          </td>
          <td>2024-08-29</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe60321069cd8bdd4dcd7dd1f6cb9aee5e0996c" target='_blank'>
              Z-DNA formation in promoters conserved between human and mouse are associated with increased transcription reinitiation rates
              </a>
            </td>
          <td>
            N. Beknazarov, Dmitry Konovalov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-08-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) remains a global health concern with persistently high incidence and mortality rates. However, the specific pathogenesis of CRC remains poorly understood. This study aims to investigate the role and pathogenesis of serine and arginine rich splicing factor 10 (SRSF10) in colorectal cancer. Methods Bioinformatics analysis was employed to predict SRSF10 gene expression in CRC patients. Functional experiments involving SRSF10 knockdown and overexpression were conducted using CCK8, transwell, scratch assay, and flow cytometry. Additionally, the PRIdictor website was utilized to predict the SRSF10 interaction site with RFC5. The identification of different transcripts of SRSF10-acting RFC5 pre-mRNA was achieved through agarose gel electrophoresis. Result The knockdown of SRSF10 inhibited the proliferation and migration ability of CRC cells, while promoting apoptosis and altering the DNA replication of CRC cells. Conversely, when SRSF10 was highly expressed, it enhanced the proliferation and migration ability of CRC cells and caused changes in the cell cycle of colorectal cancer cells. This study revealed a change in the replicating factor C subunit 5 (RFC5) gene in colorectal cancer cells following SRSF10 knockdown. Furthermore, it was confirmed that SRSF10 increased RFC5 exon2-AS1(S) transcription variants, thereby promoting the development of colorectal cancer through AS1 exclusion to exon 2 of RFC5. Conclusion In summary, this study demonstrates that SRSF10 promotes the progression of colorectal cancer by generating an aberrantly spliced exclusion isoform of AS1 within RFC5 exon 2. These findings suggest that SRSF10 could serve as a crucial target for the clinical diagnosis and treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183e863439be0cd3db11d90e783d10b2b9aa76e8" target='_blank'>
              High Expression of SRSF10 Promotes Colorectal Cancer Progression by Aberrant Alternative Splicing of RFC5
              </a>
            </td>
          <td>
            Shuai Xu, Fangmin Zhong, Junyao Jiang, Fangyi Yao, Meiyong Li, Mengxin Tang, Ying Cheng, Yulin Yang, Wen Wen, Xueru Zhang, Bo Huang, Xiao-Zhong Wang
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="
 Mutations in DNA are a known cause of cancer. However, they become oncogenic only under specific cellular conditions and within certain tissue contexts. Tissue injury often leads to the aberrant activation of developmental pathways, eroding markers of cellular differentiation and giving rise to transient cellular states. These states resemble developmental progenitors, characterized by higher transcriptional plasticity. Oncogenes, particularly KRAS, co-opt cellular plasticity by acting on tissue-specific enhancers, preventing the resolution of acinar plasticity and maintaining the pancreas in a pro-inflammatory phase that progresses to cancer1,2. However, the specific role of tissue-specific enhancers in pancreas regeneration and cancer initiation remains unexplored, and strategies to target these regenerative processes are yet to be developed. Our laboratory investigates non-mutational mechanisms of cancer, with a special interest in the intersection of developmental biology and cancer, aiming to elucidate the processes of cellular plasticity and cancer initiation. Recent studies have shown that acinar cells possess remarkable cellular plasticity, giving rise to a diverse population of cellular states in pancreas regeneration and cancer contexts3-5. We have discovered that the transcription factor Sox4 is necessary for the specification of tuft cells and gastric metaplasia found in chronic pancreatitis and in pancreatic intraepithelial lesions (PanINs). Moreover, we have shown that cellular plasticity has a non-cell autonomous protective effect; losing Sox4 and cellular heterogeneity in PanINs results in a fibrotic and immunosuppressive tumor microenvironment and disease progression. Our results demonstrated that targeting cellular plasticity accelerates or prevents cancer initiation through autonomous and non-autonomous cellular processes6. In this study, we leverage a comprehensive resource of ncRNAs transcribed from enhancer elements of pancreas development, together with their inferred target genes and correlations with clinical outcomes. We focused on an ncRNA associated with the risk of pancreatic cancer. To study its function, we developed two novel mouse strains for tissue-specific deletion, enabling precise manipulation and study of its role in the pancreas and other tissues. Our phenotypic analysis demonstrates that the ncRNA is activated following tissue injury and is essential for developing PanINs. Moreover, we show that it regulates cell-cell communication after tissue injury. This mechanism operates independently of cellular plasticity but significantly creates a pro-inflammatory environment that favors cancer development. Our results demonstrate that enhancer elements are actionable targets to regulate cell fate decisions in regeneration and cancer. Additionally, KRAS-induced cancer initiation depends on developmental regulatory elements, opening the window to develop prophylactic interventions to delay tumor onset.
 Citation Format: Luis Arnes. Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C059.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/698dc456f639e681131533a5b64aac4181ad1d22" target='_blank'>
              Abstract C059: Regulatory Elements of Pancreas Development License the Initiation of Pancreatic Ductal Adenocarcinoma
              </a>
            </td>
          <td>
            Luis Arnes
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="OBJECTIVES
The present study used single-cell RNA sequencing (scRNA-seq) to characterize the cellular composition of ovarian carcinosarcoma (OCS) and identify its molecular characteristics.


METHODS
scRNA-seq was performed in resected primary OCS for an in-depth analysis of tumor cells and the tumor microenvironment. Immunohistochemistry staining was used for validation. The scRNA-seq data of OCS were compared with those of high-grade serous ovarian carcinoma (HGSOC) tumors and other OCS tumors.


RESULTS
Both malignant epithelial and malignant mesenchymal cells were observed in the OCS patient of this study. We identified four epithelial cell subclusters with different biological roles. Among them, epithelial subcluster 4 presented high levels of breast cancer type 1 susceptibility protein homolog (BRCA1) and DNA topoisomerase 2-α (TOP2A) expression and was related to drug resistance and cell cycle. We analyzed the interaction between epithelial and mesenchymal cells and found that fibroblast growth factor (FGF) and pleiotrophin (PTN) signalings were the main pathways contributing to communication between these cells. Moreover, we compared the malignant epithelial and mesenchymal cells of this OCS tumor with our previous published HGSOC scRNA-seq data and OCS data. All the epithelial subclusters in the OCS tumor could be found in the HGSOC samples. Notably, the mesenchymal subcluster C14 exhibited specific expression patterns in the OCS tumor, characterized by elevated expression of cytochrome P450 family 24 subfamily A member 1 (CYP24A1), collagen type XXIII α1 chain (COL23A1), cholecystokinin (CCK), bone morphogenetic protein 7 (BMP7), PTN, Wnt inhibitory factor 1 (WIF1), and insulin-like growth factor 2 (IGF2). Moreover, this subcluster showed distinct characteristics when compared with both another previously published OCS tumor and normal ovarian tissue.


CONCLUSIONS
This study provides the single-cell transcriptomics signature of human OCS, which constitutes a new resource for elucidating OCS diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf6169030bf9196f7db5cdbd57ccc6a7a671e33" target='_blank'>
              Single-cell transcriptomics reveals tumor landscape in ovarian carcinosarcoma.
              </a>
            </td>
          <td>
            Junfen Xu, Mengyan Tu
          </td>
          <td>2024-08-01</td>
          <td>Journal of Zhejiang University. Science. B</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Phenotypic variability in isogenic bacterial populations helps them to cope with external stress. This variability can stem from gene expression noise and/or unequal partitioning of low-copy-number freely diffusing proteins during cell division. Some high-copy-number components are transiently associated with almost immobile large assemblies (hyperstructures) and thereby may not be distributed equally. We hypothesize that hyperstructure organization of bacterial cells can contribute to bacterial phenotypic variability. We focus on nucleoid hyperstructure containing many DNA-associated proteins. Our previous findings suggested that replication initiation timing may be the main source of variability for nucleoid segregation events. To uncover the role of the initiator DnaA in phenotypic variability in E. coli, we followed its intra- and intercellular distribution using chromosomally encoded fluorescently tagged DnaA and histone-like HU under their native promoters. The mobility of DnaA is low, corresponding to a diffusion-binding mode, but still one order faster than that of HU. The intracellular distribution of DnaA concentration is homogeneous in contrast to the significant asymmetry in the distribution of HU; the latter leading to unequal DNA content of nucleoids in future daughter compartments. The intercellular variability of HU (CV=26%), DnaA, and their ratio (both with CV=18%) is high. The variable DnaA/DNA may cause a variable replication initiation time (initiation noise). Asynchronous initiation at different oriCs may be, in turn, a plausible origin of the asymmetric DNA distribution. We specify that the DnaA/DNA ratio may be a more rigorous feature in determining the variability of initiation time in E. coli than each of them separately. Importance Phenotypic variability is an important phenomenon of bacterial physiology related to adaptation to environmental stresses and antibiotic resistance, not yet completely understood. This work is a continued examination of our hypothesis of the role of hyperstructure organization of bacterial cells in this phenomenon, focusing on the nucleoid hyperstructure and the DNA-associated replication initiator DnaA. Exploiting fluorescently tagged proteins, we analyze intra- and intercellular distributions, mobility, and concentration variabilities of DnaA and DNA in the population of Escherichia coli. The data obtained indicate that the variable DnaA/DNA ratio may serve as an essential source of variability of the replication initiation event and the ensuing chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2d2069c6e7f009db748eca4a82f0ae5bcf52f6c" target='_blank'>
              Relative Distribution of DnaA and DNA in Escherichia coli Cells as a Factor of Their Phenotypic Variability
              </a>
            </td>
          <td>
            Sharanya K. Namboodiri, A. Aranovich, U. Hadad, Levi A. Gheber, Mario Feingold, I. Fishov
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Triplex DNA formation is a useful genomic targeting tool that is expected to have a wide range of applications, including the antigene method; however, there are fundamental limitations in its forming sequence. We recently extended the triplex DNA-forming sequence to methylated DNA sequences containing 5mCG base pairs by developing guanidino-dN, which is capable of recognizing a 5mCG base pair with high affinity. We herein investigated the effect of triplex DNA formation using TFOs with guanidino-dN on methylated DNA sequences at the promoter of the RASSF1A gene, whose expression is epigenetically suppressed by DNA methylation in MCF-7 cells, on gene expression. Interestingly, triplex DNA formation increased the expression of the RASSF1A gene at the transcript and protein levels. Furthermore, RASSF1A-activated MCF-7 cells exhibited cell growth suppressing activity. Changes in the expression of various genes associated with the promotion of apoptosis and breast cancer survival accompanied the activation of RASSF1A in cells exhibited antiproliferative activity. These results suggest the potential of increases in gene expression through triplex DNA formation as a new genomic targeting tool.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/367d9640afcf3f528a05a1512247caf159507678" target='_blank'>
              Novel strategy for activating gene expression through triplex DNA formation targeting epigenetically suppressed genes
              </a>
            </td>
          <td>
            Ryotaro Notomi, Shigeki Sasaki, Yosuke Taniguchi
          </td>
          <td>2024-07-31</td>
          <td>RSC Chemical Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3239b6a328047bdf1fed75ded2d7ba86fd49bc1" target='_blank'>
              RBM8A, a new target of TEAD4, promotes breast cancer progression by regulating IGF1R and IRS-2
              </a>
            </td>
          <td>
            Fang Li, Xiaofei Wang, Jing Zhang, Jin-yan Zhang, X. Jing, Q. Jiang, Jing Zhou, Li Cao, Hang Peng, D. Tong, Chen Huang
          </td>
          <td>2024-09-04</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Bone marrow (BM) is a common site for solid tumor metastasis, often causing poor outcome. Here, we define the characteristics of BM-disseminated tumor cells (DTCs) using neuroblastoma as a model. We combined single-cell RNA-sequencing (scRNA-seq) and cell-surface protein analysis using 7 paired BM and primary tumor (PT) samples and found that DTCs contain a higher percentage of cycling cells and higher expression of neurodevelopmental genes compared to corresponding PT cells. In 6 patients, the copy number variation profile differed between PT cells and DTCs, indicating spatial heterogeneity. Within the BM, we detected dormant DTCs with potentially reduced chemosensitivity; this population contained cells expressing low levels of the immunotherapeutic antigen GD2 and increased NGFR expression. In conclusion, we characterized DTCs that are particularly challenging to target, offering new avenues for developing therapeutic strategies designed to target all subpopulations within the highly complex metastatic site, thereby preventing the development of drug-resistant clones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fc1c99c4919cd41656eef13d151b98e6e5eb661" target='_blank'>
              Metastatic tumor cells in bone marrow differ from paired neuroblastoma tumor and contain subsets with therapy-resistant characteristics
              </a>
            </td>
          <td>
            Caroline Hochheuser, A. Boltjes, Kaylee M. Keller, Simon Tol, Marieke van de Mheen, Carolina Pita Barros, Zeinab van Gestel-Fadaie, André B. P. van Kuilenburg, Sander van Hooff, C. Voermans, Jan J. Molenaar, G. Tytgat, Ilse Timmerman
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="The co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers. This phenomenon has been investigated since the late 1980s without resolution. Expanding beyond previous gene-centric studies, we investigated the co-occurrence in a genome-wide manner taking an evolutionary perspective. Mining of large-scale tumor aneuploidy data confirmed the previous finding of a small-scale longitudinal study that the most likely order is chromosome 10 loss followed by chromosome 7 gain. Extensive analysis of genomic and transcriptomic data from both patients and cell lines revealed that this co-occurrence can be explained by functional rescue interactions that are highly enriched on chromosome 7, which could potentially compensate for any detrimental consequences arising from the loss of chromosome 10. Transcriptomic data from various normal, non-cancerous human brain tissues were analyzed to assess which tissues may be most predisposed to tolerate compensation of chromosome 10 loss by chromosome 7 gain. The analysis indicated that the pre-existing transcriptomic states in the cortex and frontal cortex, where gliomas arise, are more favorable than other brain regions for compensation by rescuer genes that are active on chromosome 7. Collectively, these findings suggest that the phenomenon of chromosome 10 loss and chromosome 7 gain in gliomas is orchestrated by a complex interaction of many genes residing within these two chromosomes and provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff13af5a4c0be0182481407e6fc3feb597b938be" target='_blank'>
              Chromosome 7 gain compensates for chromosome 10 loss in glioma.
              </a>
            </td>
          <td>
            N. Nair, Alejandro A Schaffer, E. Gertz, Kuoyuan Cheng, Johanna Zerbib, A. Sahu, Gil Leor, E. Shulman, Kenneth D Aldape, Uri Ben-David, E. Ruppin
          </td>
          <td>2024-07-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Retrotransposons are invasive genetic elements, which replicate by copying and pasting themselves throughout the genome in a process called retrotransposition. The most abundant retrotransposons by number in the human genome are Alu and LINE-1 elements, which comprise approximately 40% of the human genome. The ability of retrotransposons to expand and colonize eukaryotic genomes has rendered them evolutionarily successful and is responsible for creating genetic alterations leading to significant impacts on their hosts. Previous research suggested that hypomethylation of Alu and LINE-1 elements is associated with global hypomethylation and genomic instability in several types of cancer and diseases, such as neurodegenerative diseases, obesity, osteoporosis, and diabetes mellitus (DM). With the advancement of sequencing technologies and computational tools, the study of the retrotransposon’s association with physiology and diseases is becoming a hot topic among researchers. Quantifying Alu and LINE-1 methylation is thought to serve as a surrogate measurement of global DNA methylation level. Although Alu and LINE-1 hypomethylation appears to serve as a cellular senescence biomarker promoting genomic instability, there is sparse information available regarding their potential functional and biological significance in DM. This review article summarizes the current knowledge on the involvement of the main epigenetic alterations in the methylation status of Alu and LINE-1 retrotransposons and their potential role as epigenetic markers of global DNA methylation in the pathogenesis of DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07f55c13ed59ce933c2fb2995ef34b1c1fa76b5" target='_blank'>
              Retrotransposons and Diabetes Mellitus
              </a>
            </td>
          <td>
            Andromachi Katsanou, Charilaos Kostoulas, E. Liberopoulos, A. Tsatsoulis, I. Georgiou, S. Tigas
          </td>
          <td>2024-09-06</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ac1daefafcb98a739594660ed94aebb04da4a2f" target='_blank'>
              DNA polymerase swapping in Caudoviricetes bacteriophages
              </a>
            </td>
          <td>
            N. Yutin, I. Tolstoy, Pascal Mutz, Y. Wolf, M. Krupovic, Eugene V. Koonin
          </td>
          <td>2024-08-26</td>
          <td>Virology Journal</td>
          <td>0</td>
          <td>109</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc721cd925975049ceb0cd27f9ffa4efdce8611" target='_blank'>
              Synthetic lethality by PARP inhibitors: new mechanism uncovered based on unresolved transcription-replication conflicts
              </a>
            </td>
          <td>
            Marina Kolesnichenko, Claus Scheidereit
          </td>
          <td>2024-07-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Three-dimensional chromatin control of eukaryotic transcription is pivotal for regulating gene expression. This additional layer of epigenetic regulation is also utilized by DNA viruses, including herpesviruses. Dynamic, spatial genomic organization often involves looping of chromatin anchored by host-encoded CCCTC-binding factor (CTCF) and other factors, which control crosstalk between promoters and enhancers. Herein, we review the contribution of CTCF-mediated looping in regulating transcription during herpesvirus infection, with a specific focus on the betaherpesvirus, human cytomegalovirus (HCMV).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c77ca71344a757b4c351d6ffdff351925d0ab026" target='_blank'>
              Loopy virus or controlled contortionist? 3D regulation of HCMV gene expression by CTCF-driven chromatin interactions.
              </a>
            </td>
          <td>
            Ian J. Groves, Christine M. O’Connor
          </td>
          <td>2024-08-30</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3e129666ae56c033aa207d599e42711f40eae8" target='_blank'>
              Gene body DNA hydroxymethylation restricts the magnitude of transcriptional changes during aging
              </a>
            </td>
          <td>
            James R. Occean, Na Yang, Yan Sun, Marshall S. Dawkins, R. Munk, Cedric Belair, Showkat A. Dar, C. Anerillas, Lin Wang, Changyou Shi, Christopher Dunn, Michel Bernier, Nathan L. Price, Julie S Kim, Chang-Yi Cui, Jinshui Fan, Moitrayee Bhattacharyya, Supriyo De, Manolis Maragkakis, R. de Cabo, Simone Sidoli, P. Sen
          </td>
          <td>2024-07-28</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61a1e72cd7656def115fe3bbce4a02021bb17ead" target='_blank'>
              Involvement of the splicing factor SART1 in the BRCA1-dependent homologous recombination repair of DNA double-strand breaks
              </a>
            </td>
          <td>
            Kie Ozaki, Reona Kato, Takaaki Yasuhara, Yuki Uchihara, Miyako Hirakawa, Yu Abe, Hiroki Shibata, Reika Kawabata-Iwakawa, Aizhan Shakayeva, Palina Kot, Kiyoshi Miyagawa, Keiji Suzuki, Naoki Matsuda, Atsushi Shibata, M. Yamauchi
          </td>
          <td>2024-08-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966f20b4115e34215d01f56c4a256d96a9d46af1" target='_blank'>
              Mosaic variegated aneuploidy in development, ageing and cancer.
              </a>
            </td>
          <td>
            Marcos Malumbres, Carolina Villarroya-Beltri
          </td>
          <td>2024-08-21</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3734fdaa3ed3777431c0f723757629a9b7e4aa2" target='_blank'>
              High-complexity of DNA double-strand breaks is key for alternative end-joining choice
              </a>
            </td>
          <td>
            Zhiyang Hou, Tianxiang Yu, Qiyi Yi, Yan Du, Libin Zhou, Ye Zhao, Yuejin Wu, Lijun Wu, Ting Wang, Po Bian
          </td>
          <td>2024-08-03</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tracing cell lineages has become a valuable tool for studying biological processes. Among the available tools for human data, mitochondria DNA (mtDNA) has a high potential due to its ability to be used in conjunction with single-cell chromatin accessibility data, giving access to the cell phenotype. Nonetheless, the existing mutation calling tools are ill-equipped to deal with the polyploid nature of the mtDNA and lack a robust statistical framework. Here we introduce scMitoMut, an innovative R package that leverages statistical methodologies to accurately identify mitochondrial lineage related mutations at the single-cell level. scMitoMut assigns a mutation quality q-value based on beta-binomial distribution to each mutation at each locus within individual cells, ensuring higher sensitivity and precision of lineage related mutation calling in comparison to current methodologies. We tested scMitoMut using single-cell DNA sequencing, scATAC sequencing and 10× Genomics single cell multiome datasets. Using a single-cell DNA sequencing dataset from a mixed population of cell lines, scMitoMut demonstrated superior sensitivity in identifying small proportion of cancer cell lines compared to existing methods. In a human colorectal cancer scATAC dataset, scMitoMut identified more mutations than state-of-the-art methods. Applied to 10× Genomics multiome datasets, scMitoMut effectively measured the lineage distance in cells from blood or brain tissues. Thus, the scMitoMut is a free available (https://www.bioconductor.org/packages/devel/bioc/html/scMitoMut.html.), well-engineered toolkit for mtDNA mutation calling with high memory and CPU efficiency. Consequently, it will significantly advance the application of single-cell sequencing, facilitating the precise delineation of mitochondrial mutations for lineage tracing purposes in development, tumor and stem cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a057cdbd6d7399eed5dec23b92e83e85fe7d0c9" target='_blank'>
              scMitoMut for calling mitochondrial lineage–related mutations in single cells
              </a>
            </td>
          <td>
            Wenjie Sun, Daphne van Ginneken, Leïla Perié
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1f5936217eac5b4043bb1665424b2339a9f20d" target='_blank'>
              MYC and KRAS cooperation: from historical challenges to therapeutic opportunities in cancer
              </a>
            </td>
          <td>
            S. Casacuberta-Serra, Íñigo González-Larreategui, Daniel Capitán-Leo, Laura Soucek
          </td>
          <td>2024-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The role of focal amplifications and extrachromosomal circular DNA (eccDNA) is still uncertain in prostate adenocarcinoma (PRAD). Here, we first mapped the global characterizations of eccDNA and then investigate the characterization of eccDNA-amplified key differentially expressed encoded genes (eKDEGs) in the progression, immune response and immunotherapy of PRAD.Circular_seq was used in conjunction with the TCGA-PRAD transcriptome dataset to sequence, annotate, and filter for eccDNA-amplified differentially expressed coding genes (eDEGs) in PRAD and para-cancerous normal prostate tissues. Afterwards, risk models were created and eKDEGs linked to the PRAD prognosis were identified using Cox and Lasso regression analysis. The immune microenvironment of the risk model was quantified using a variety of immunological algorithms, which also identified its characteristics with regard to immunotherapy, immune response, and immune infiltration.In this research, there was no significant difference in the size, type, and chromosomal distribution of eccDNA in PRAD and para-cancerous normal prostate tissues. However, 4,290 differentially expressed eccDNAs were identified and 1,981 coding genes were amplified. Following that, 499 eDEGs were tested in conjunction with the transcriptome dataset from TCGA-PRAD. By using Cox and Lasso regression techniques, ZNF330 and PITPNM3 were identified as eKDEGs of PRAD, and a new PRAD risk model was conducted based on this. Survival analysis showed that the high-risk group of this model was associated with poor prognosis and validated in external data. Immune infiltration analysis showed that the model risks affected immune cell infiltration in PRAD, not only mediating changes in immune cell function, but also correlating with immunophenotyping. Furthermore, the high-risk group was negatively associated with anti-CTLA-4/anti-PD-1 response and mutational burden. In addition, Tumor Immune Dysfunction and Exclusion analyses showed that high-risk group was more prone to immune escape. Drug sensitivity analyses identified 10 drugs, which were instructive for PRAD treatment.ZNF330 and PITPNM are the eKDEGs for PRAD, which can be used as potential new prognostic markers. The two-factor combined risk model can effectively assess the survival and prognosis of PRAD patients, but also can predict the different responses of immunotherapy to PRAD patients, which may provide new ideas for PRAD immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189d5f00b52f7644f54771a980ef61f81d31dae9" target='_blank'>
              Extrachromosomal circular DNAs in prostate adenocarcinoma: global characterizations and a novel prediction model
              </a>
            </td>
          <td>
            Qingliu He, Qingfu Su, Chengcheng Wei, Pu Zhang, Weihui Liu, Junyi Chen, Xiaoping Su, Zhuang Wei
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Diffuse Large B-cell Lymphoma (DLBCL) is the most prevalent subtype of non-Hodgkin’s lymphoma for which current therapeutic strategies remain insufficient. The diffuse nature of DLBCL, lacking distinct tissue structures, represents a challenge to elucidate the cellular organization and interactions within the tumor microenvironment (TME). In this study, we applied spatial transcriptomics to identify spatially-resolved gene expression profiles in 10 DLBCL tissue samples, identifying distinct immune cell infiltration and colocalization patterns. These profiles were classified into six cellular ecosystems (Cell-Eco) that differ in cellular composition, functional patterns, and neighborhood characteristics. The spatially-resolved Cell-Eco signatures provided prognostic scores that stratified patients with different overall survival rates. We also found that C1q+ tumor-associated macrophages are the primary cells interacting with malignant B cells and influencing the spatial architecture of the TME. This study provides novel biological insights into the complexity of the TME in DLBCL and highlights the potential prognostic value of its spatial organization. Graphical abstract Key findings Spatial transcriptomics classifies DLBCL tissues based on immune cell infiltration and colocalization patterns. DLBCL tumor microenvironment consists of cellular ecosystems (Cell-Eco) that differ in cellular composition, transcriptomic profiles and neighborhood characteristics. Spatially-resolved Cell-Eco signatures stratify patients with different overall survival. C1q+ tumor-associated macrophages primarily interact with malignant B cells and contribute to the spatial organization of the tumor microenvironment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/80ebc56c31f2327f9141f086b678169242b28d1d" target='_blank'>
              Spatial transcriptomics unveils immune cellular ecosystems associated with patient survival in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Alba Díaz Herrero, Hector F Pelaez-Prestel, Lucile Massenet-Regad, Maëva Veyssiere, Julien Calvani, Caterina Cristinelli, Jacqueline Lehmann-Che, Véronique Meignin, Catherine Thieblemont, Véronique Blanc, V. Soumelis, P. Tonnerre
          </td>
          <td>2024-09-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Background Smooth muscle cell (SMC) plasticity and phenotypic switching play prominent roles in the pathogenesis of multiple diseases, but their role in tumorigenesis is unknown. We investigated whether and how SMC diversity and plasticity plays a role in tumor angiogenesis and the tumor microenvironment. Methods and Results We use SMC-specific lineage-tracing mouse models and single cell RNA sequencing to observe the phenotypic diversity of SMCs participating in tumor vascularization. We find that a significant proportion of SMCs adopt a phenotype traditionally associated with macrophage-like cells. These cells are transcriptionally similar to ‘resolution phase’ M2b macrophages, which have been described to have a role in inflammation resolution. Computationally predicted by the ligand-receptor algorithm CellChat, signaling from BST2 on the surface of tumor cells to PIRA2 on SMCs promote this phenotypic transition; in vitro SMC assays demonstrate upregulation of macrophage transcriptional programs, and increased proliferation, migration, and phagocytic ability when exposed to BST2. Knockdown of BST2 in the tumor significantly decreases the transition towards a macrophage-like phenotype, and cells that do transition have a comparatively higher inflammatory signal typically associated with anti-tumor effect. Conclusion As BST2 is known to be a poor prognostic marker in multiple cancers where it is associated with an M2 macrophage-skewed TME, these studies suggest that phenotypically switched SMCs may have a previously unidentified role in this immunosuppressive milieu. Further translational work is needed to understand how this phenotypic switch could influence the response to anti-cancer agents and if targeted inhibition of SMC plasticity would be therapeutically beneficial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcad9e00e00b0b1e9b4d096ed61477beb04a0767" target='_blank'>
              BST2 induces vascular smooth muscle cell plasticity and phenotype switching during cancer progression
              </a>
            </td>
          <td>
            Caitlin F. Bell, Richard A. Baylis, N. G. Lopez, Wei Feng Ma, Hua Gao, Fudi Wang, Sharika Bamezai, Changhao Fu, Yoko Kojima, Shaunak S. Adkar, Lingfeng Luo, Clint L. Miller, N. Leeper
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="CRISPR-Cas12a, an RNA-guided nuclease, has been repurposed for genome editing and molecular diagnostics due to its capability of cis-cleavage on target DNA and trans-cleavage on non-target single-strand DNA (ssDNA). However, the mechanisms underlying the activation of trans-cleavage activity of Cas12a, particularly in the context of split DNA activators, remain poorly understood. We elucidate the synergistic effect of these activators and introduce the concepts of induced targeting effect and exon-unwinding to describe the phenomenon. We demonstrate that upon binding of split DNA activators adjacent to the Protospacer Adjacent Motif (PAM) to the Cas12a ribonucleoprotein (Cas12a-RNP), a ternary complex form that can capture and interact with distal split DNA activators to achieve synergistic effects. Notably, if the distal activator is double-strand DNA (dsDNA), the complex initiates exon-unwinding, facilitating the RNA-guide sequence's access. Our findings provide a mechanistic insight into action of Cas12a and propose a model that could significantly advance our understanding of its function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21bfca41537975350cb4978fbebd5605ae4be926" target='_blank'>
              Synergistic effect of split DNA activators of Cas12a with exon-unwinding and induced targeting effect.
              </a>
            </td>
          <td>
            Shen Huang, Yongliang Lou, Laibao Zheng
          </td>
          <td>2024-09-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Sequencing mitochondrial DNA (mtDNA) variants from single cells has resolved clonality and lineage in native human samples and clinical specimens. Prior work established that heteroplasmic mtDNA variants can be used to delineate clonality in hematopoiesis, but they have limited ability to reconstruct cellular phylogenies. However, a recent report by Weng et al. challenges the current paradigm by describing an unprecedented number of shared mtDNA variants between cells that reportedly resolve high-resolution phylogenetic trees. We re-examined the claims of Weng et al., and identified two major points of concern regarding this unprecedented connectedness. First, shared variants between cells are disproportionately detected in a single molecule per cell, and second, these variants are enriched 10–20-fold at the edges of mtDNA molecules, reminiscent of artifacts reported in other sequencing approaches. Further, our analyses show that pruning low support and likely artificial mtDNA variants removes nearly all of the reported phylogenetic structure. Thus, we strongly caution against using mtDNA variant calling workflows that rely on minimal evidence, including the computational pipeline introduced in Weng et al., as variants with high connectedness and low evidence are likely artifacts that lead to the construction of false phylogenies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae6d387965c56c26ee411bc842b91f11e043e8f3" target='_blank'>
              Artifacts in single-cell mitochondrial DNA mutation analyses misinform phylogenetic inference
              </a>
            </td>
          <td>
            Caleb A. Lareau, M. Chapman, Livius Penter, T. Nawy, D. Pe’er, Leif S. Ludwig
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7300fe63a85fac8fe37bf10af42231e58f921f5" target='_blank'>
              Analysis of the role of CHPF in colorectal cancer tumorigenesis and immunotherapy based on bioinformatics and experiments
              </a>
            </td>
          <td>
            Qingyu Song, Pengchao Wang, Jingyu Wu, Ming Lu, Qingcheng Xia, Yexin Shi, Zijun Wang, Xiang Ma, Qinghong Zhao
          </td>
          <td>2024-09-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Phenethyl isothiocyanate (PEITC), a natural product, exists in biological activities, including anticancer activity in many human cancer cells. No information shows that PEITC affects DNA damage in human retinoblastoma (RB) cells in vitro. In this study, the aim of experiments was to determine whether PEITC decreased total viable cell number or not by inducing protein expressions involved in DNA damage and repair in Y79 RB cells in vitro. Total cell viability was measured by PI exclusion assay, and PEITC reduced the total Y79 viable cell numbers in a dose-dependent manner. DNA condensation and DNA impairment were conducted by DAPI staining and comet assays, respectively, in Y79 cells. The findings show that PEITC induced DNA condensation dose-dependently based on the brighter fluorescence of cell nuclei stained by DAPI staining. PEITC-induced DNA damage showed a more extended DNA migration smears than that of the control, which was performed by a comet assay. Western blotting was performed to measure the protein expressions involved in DNA damage and repair, which showed that PEITC at 2.5-10 μM increased NRF2, HO-1, SOD (Mn), and catalase; however, it decreased SOD (Cu/Zn) except 10 μM PEITC treatment, and decreased glutathione, which were associated with oxidative stress. Furthermore, PEITC increased DNA-PK, MDC1, H2A.XpSer139, ATMpSer1981, p53, p53pSer15, PARP, HSP70, and HSP90, but decreased TOPIIα, TOPIIβ, and MDM2pSer166 that were associated with DNA damage and repair mechanism in Y79 cells. The examination from confocal laser microscopy shows that PEITC increased H2A.XpSer139 and p53pSer15, and decreased glutathione and TOPIIα in Y79 cells. In conclusion, the cytotoxic effects of PEITC on reducing the number of viable cells may be due to the induction of DNA damage and the alteration of DNA repair proteins in Y79 cells in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc6b28a6e7a9bee2d7ae76fad1426f8cde6995ab" target='_blank'>
              PEITC Induces DNA Damage and Inhibits DNA Repair-Associated Proteins in Human Retinoblastoma Cells In Vitro.
              </a>
            </td>
          <td>
            Sheng-Yao Hsu, Yi-Ping Huang, T. Hsia, Jaw-Chyun Chen, Shu-Fen Peng, Wen-Tsong Hsieh, F. Chueh, Chao-Lin Kuo
          </td>
          <td>2024-08-23</td>
          <td>Environmental toxicology</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Study of early pancreas neoplasia in humans had previously been limited by lack of available tissue. Recent work by our group on deceased donor pancreata identified pancreatic intraepithelial neoplasia (PanIN) lesions in non-diseased organs and defined a transcriptomic signature for the microenvironment of these pre-cancerous lesions. Prevalence of PanINs in healthy human tissue was higher than expected and established a novel model to expand understanding of the complex biology of these precursor lesions. Epigenetic changes in chromatin structure and the subsequent effects on gene expression are essential processes in neoplasia that have yet to be described at the single cell level in the human pancreas. We hypothesized that previously defined gene signatures could identify acinar, ductal, and PanIN cells in donor pancreata based on chromatin accessibility profiling by single nuclear transposase-accessible chromatin preparation followed by high-throughput sequencing (snATAC-seq). Comparison between the accessibility patterns, transcription factor motifs, and differences in accessibility versus expression profiles can then be used to expand prior knowledge of early pancreatic neoplasia and identify novel targets for further study and therapy selection. To test this hypothesis, single nuclei from seventeen donor pancreata (twenty total samples) were isolated for snATAC-seq using the 10x Genomics platform. Data analysis was conducted using CellRanger, Seurat, Signac, AUCell, and other programs to identify and label chromatin peaks in individual nuclei corresponding to acinar, ductal, and PanIN origin according to gene signatures identified by scRNA-seq and special transcriptomics on matched donor pancreata. Using this method, cells corresponding to these gene signatures, including PanIN- like cells, were indeed identifiable with differentially accessible regions of the genome then analyzed for transcription factor motif enrichment between these cell types. Motifs enriched in cells identified as PanINs compared to either ductal or acinar cells included several transcription factors implicated in tumorigenesis. These results are similar to previously demonstrated essential factors in pancreatic neoplasia, suggesting this strategy is a relevant method for data analysis and discovery of novel factors implicated in tumorigenesis. Ongoing studies include validation of these findings in vitro using human pancreatic normal epithelial cells, human tumor cell lines, and organoid models derived from human normal and tumor samples. Together, this study demonstrates that PanINs can be identified in human pancreas tissue by snATAC-seq and that differentially expressed motifs can be identified, providing opportunity for further integration with transcriptomic information to elucidate detailed understanding of early neoplasia in the pancreas for future drug development and targeted therapeutics studies.
 Citation Format: Jamie N Mills, Aaron Dendekker, Joyce K Thompson, Hannah Watkoske, Simone Benitz, Padma Kadiyala, Ahmed Elhossiny, Howard Crawford, Eileen Carpenter, Marina Pasca di Magliano, Filip Bednar. Characterization of chromatin accessibility patterns in the human pancreas and early pancreatic neoplastic lesions using snATAC-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C056.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eee7faf599c3d04827a3bdb6a5acb4e73be0ca2f" target='_blank'>
              Abstract C056: Characterization of chromatin accessibility patterns in the human pancreas and early pancreatic neoplastic lesions using snATAC-seq
              </a>
            </td>
          <td>
            Jamie N. Mills, Aaron Dendekker, Joyce K Thompson, Hannah R Watkoske, S. Benitz, Padma Kadiyala, Ahmed M. Elhossiny, Howard C. Crawford, Eileen S Carpenter, M. Pasca di Magliano, Filip Bednar
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="ABSTRACT The size and composition of the intracellular DNA precursor pool is integral to the maintenance of genome stability, and this relationship is fundamental to our understanding of cancer. Key aspects of carcinogenesis, including elevated mutation rates and induction of certain types of DNA damage in cancer cells, can be linked to disturbances in deoxynucleoside triphosphate (dNTP) pools. Furthermore, our approaches to treat cancer heavily exploit the metabolic interplay between the DNA and the dNTP pool, with a long-standing example being the use of antimetabolite-based cancer therapies, and this strategy continues to show promise with the development of new targeted therapies. In this Review, we compile the current knowledge on both the causes and consequences of dNTP pool perturbations in cancer cells, together with their impact on genome stability. We outline several outstanding questions remaining in the field, such as the role of dNTP catabolism in genome stability and the consequences of dNTP pool expansion. Importantly, we detail how our mechanistic understanding of these processes can be utilised with the aim of providing better informed treatment options to patients with cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/489114e77099687246712fffa583dd2206547a4f" target='_blank'>
              Understanding the interplay between dNTP metabolism and genome stability in cancer
              </a>
            </td>
          <td>
            Miriam Yagüe-Capilla, S. G. Rudd
          </td>
          <td>2024-08-01</td>
          <td>Disease Models & Mechanisms</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Motivation Cancer is driven by somatic mutations that result in the expansion of genomically distinct sub-populations of cells called clones. Identifying the clonal composition of tumours and understanding the evolutionary relationships between clones is a crucial task in cancer genomics. Bulk DNA sequencing is commonly used for studying the clonal composition of tumours, but it is challenging to infer the genetic relationship between different clones due to the mixture of different cell populations. Results In this work, we introduce a new probabilistic model called PhyClone that can infer clonal phylogenies from bulk sequencing data. We demonstrate the performance of PhyClone on simulated and real-world datasets and show that it outperforms previous methods in terms of accuracy and scalability. Availability and implementation Source code is available on Github at: https://github.com/Roth-Lab/PhyClone under the GPL v3.0 license. Contact aroth@bccrc.ca">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe2cba6bef4ca455a79d82a594e5b713c434402a" target='_blank'>
              PhyClone: Accurate Bayesian reconstruction of cancer phylogenies from bulk sequencing
              </a>
            </td>
          <td>
            Emilia Hurtado, A. Bouchard-Côté, Andrew Roth
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ed5339b18c8cb661ce34053233bc7343f92ee3a" target='_blank'>
              1q jumping translocation as a biomarker in myeloid malignancy: frequently mutated genes associated with bad prognosis and low survival
              </a>
            </td>
          <td>
            Eitan Halper-Stromberg, V. Stinnett, L. Morsberger, Aparna Pallavajjala, M. Levis, A. DeZern, Michelle Lei, Brian Phan, Rena R. Xian, Christopher D. Gocke, Guilin Tang, Ying S Zou
          </td>
          <td>2024-08-01</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Breast cancer cells exhibit organotropism during metastasis, showing preferential homing to certain organs such as bone, lung, liver, and brain. One potential explanation for this organotropic behavior is that cancer cells gain properties that enable thriving in certain microenvironments. Such specific metastatic traits may arise from gene regulation at the primary tumor site. Spatial genome organization plays a crucial role in oncogenic transformation and progression, but the extent to which chromosome architecture contributes to organ-specific metastatic traits is unclear. This work characterizes chromosome architecture changes associated with organotropic metastatic traits. By comparing a collection of genomic data from different subtypes of localized and lung metastatic breast cancer cells with both normal and cancerous lung cells, we find important trends of genomic reorganization. The most striking differences in 3D genome compartments segregate cell types according to their epithelial vs. mesenchymal status. This EMT compartment signature occurs at genomic regions distinct from transcription-defined EMT signatures, suggesting a separate layer of regulation. Specifically querying organotropism, we find 3D genome changes consistent with adaptations needed to survive in a new microenvironment, with lung metastatic breast cells exhibiting compartment switch signatures that shift the genome architecture to a lung cell-like conformation and brain metastatic prostate cancer cells showing compartment shifts toward a brain-like state. TCGA patient data reveals gene expression changes concordant with these organ-permissive compartment changes. These results suggest that genome architecture provides an additional level of cell fate specification informing organotropism and enabling survival at the metastatic site. SIGNIFICANCE Computational analysis of a cohort of cancer cell lines reveals 3D genome spatial compartment changes are associated with transitions in cancer cell state that favor metastasis (EMT) and enable survival in a new organ context.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e4758661702fdde348d0333dad4378a173eeb39" target='_blank'>
              Rearrangement of 3D genome organization in breast cancer epithelial - mesenchymal transition and metastasis organotropism
              </a>
            </td>
          <td>
            Priyojit Das, Rebeca San Martin, Tian Hong, R. McCord
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c35f97c112d09b7e983874a9efe5c2fc2f69d6c" target='_blank'>
              Enhancers in T Cell development and malignant lesions
              </a>
            </td>
          <td>
            Tong Zhang, Lin Zou
          </td>
          <td>2024-09-17</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Objective: Structural variations (SVs) constitute the greatest source of genomic alterations leading to the oncogenesis of many cancers, including the deadliest pancreatic cancer (PC). However, SVs in PC remain largely undefined due to technological limitations of the traditional short-read sequencing. This study aims to establish a comprehensive signature of genomic SVs in PC. Methods: PacBio whole genome long-read sequencing was employed to investigate the genomic landscape of SVs in eight tissues from four pancreatic cancer patients. A variety of SVs callers, including pbsv, Sniffles, cuteSV, and svim, were comprehensively adopted to improve the accuracy of SVs detection. Many undefined characteristics of germline and somatic SVs were revealed by comparing with multiple previously published databases. Besides, the impacts of structural variations on 3D chromatin organization were analyzed by integrating multi-omics data including Hi-C, RNA seq and ChIP seq. Results: A total of 26,844 and 27,028 non-redundant SVs were identified in human pancreatic cancer and matched para-cancerous tissues, respectively. Notably, among these SVs, 23.97% were novel and had not been previously reported. Germline SVs were further characterized and found enriched in 25 well-known susceptible genes, such as TP63, PVT1, and ABO. Additionally, we proposed a panel of germline SVs with predicted highest pathogenicity, involving STOX1, DSPP, and WRN. Totally, 616 somatic SVs were identified and presented high burdens in patients with lymph node metastasis. We observed that somatic DELs and INSs were significantly more prevalent in repetitive regions than in non-repetitive regions. Interestingly, DELs showed a decrease in the proportion occurring in simple repeat regions but an increase in the proportion occurring in LINE and SINE regions compared to INSs. Moreover, it was observed that deletions within topologically associated domains were associated with local enhanced interactions and disrupted chromatin loops, which might influence H3K27ac modification of histone contributing to enhancer or silencer hijacking. Conclusion: This study provides insights into the genomic signature of SVs in human pancreatic cancer. These findings might shed new light on the complexity of SVs and their pathogenesis by interplaying with chromatin organization.
 Citation Format: Yongxing Du, Xiaohao Zheng, Yunjie Duan, Chengfeng Wang. Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A083.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40a9fb18df1c1c8a73e8c29f87767e5466baa02" target='_blank'>
              Abstract A083: Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing
              </a>
            </td>
          <td>
            Yongxing Du, Xiaohao Zheng, Yunjie Duan, Cheng-Guang Wang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Zhao-Ru Dong, Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, Yong-Heng Yang, Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Centrosome amplification is a feature of cancer cells associated with chromosome instability and invasiveness. Enhancing chromosome instability and subsequent cancer cell death via centrosome unclustering and multipolar divisions is an aimed-for therapeutic approach. Here, we show that centrosome amplification potentiates responses to conventional chemotherapy in addition to its effect on multipolar divisions and chromosome instability. We perform single-cell live imaging of chemotherapy responses in epithelial ovarian cancer cell lines and observe increased cell death when centrosome amplification is induced. By correlating cell fate with mitotic behaviors, we show that enhanced cell death can occur independently of chromosome instability. We identify that cells with centrosome amplification are primed for apoptosis. We show they are dependent on the apoptotic inhibitor BCL-XL and that this is not a consequence of mitotic stresses associated with centrosome amplification. Given the multiple mechanisms that promote chemotherapy responses in cells with centrosome amplification, we assess such a relationship in an epithelial ovarian cancer patient cohort. We show that high centrosome numbers associate with improved treatment responses and longer overall survival. Our work identifies apoptotic priming as a clinically relevant consequence of centrosome amplification, expanding our understanding of this pleiotropic cancer cell feature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a9ce2e6325eaf9b5466f08fa222bbaf8fdd0b70" target='_blank'>
              Centrosome amplification primes ovarian cancer cells for apoptosis and potentiates the response to chemotherapy.
              </a>
            </td>
          <td>
            Frances Edwards, Giulia Fantozzi, Anthony Y Simon, Jean-Philippe Morretton, A. Herbette, Andrea E. Tijhuis, René Wardenaar, Stacy Foulane, Simon Gemble, D. Spierings, F. Foijer, Odette Mariani, A. Vincent-Salomon, S. Roman-Roman, X. Sastre-Garau, O. Goundiam, R. Basto
          </td>
          <td>2024-09-05</td>
          <td>PLoS biology</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Abstract Background Extrachromosomal circular DNAs (eccDNAs), a type of double‐stranded DNAs (dsDNAs) that facilitate the activation of the DNA sensing machinery, have been implicated in the progression and prognosis of various diseases. While the roles of eccDNAs remain contentious, their significance in diffuse large B‐cell lymphoma (DLBCL) has not been reported. Methods Circular DNA sequencing (circle‐seq) was used to demonstrate the expression profile of eccDNAs in DLBCL, and atomic force microscopy to validate the presence of eccDNAs. CCK‐8 and scRNA‐seq techniques were employed to uncover the activation of eccDNA in the STING pathway, leading to enhanced cell proliferation. Chemotherapeutic drugs were used to test the hypothesis that DNA damage induces the production of eccDNA, thereby activating the STING pathway independent of cGAS. GEO databases were used for verification of the prognosis of the eccDNA‐related genes, and animal models were used to investigate the synergistic effects of DNA damage therapy in combination with STING inhibitors on anti‐tumour responses. Results EccDNAs were widely expressed in DLBCL and associated with the prognosis of patients. Elevated abundance of eccDNAs promoted the progression of DLBCL. Chemotherapeutic drugs‐induced DNA damage triggered the generation of eccDNAs, resulting in the activation of the STING signalling in a cGAS‐independent manner. Moreover, inhibition of STING exerted a synergistic anti‐tumour effect with cisplatin. Conclusions EccDNAs induced by DNA damage exert an oncogenic role in DLBCL via activating the STING signalling independently of cGAS. This finding offers a rational therapeutic strategy combining chemotherapy with targeting STING. Highlights EccDNAs induced by DNA damage exert an oncogenic role in DLBCL via activating the STING signalling independently of cGAS. The combined treatment of chemotherapeutic drugs with STING inhibitor significantly delayed the tumor progression, providing new insights into the therapeutic strategy for patients with DLBCL, particularly the relapsed and/or refractory (R/R) ones.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e573ea771bfdb5c81c97bedfb9abf4331d619e8" target='_blank'>
              Multi‐omics integration reveals the oncogenic role of eccDNAs in diffuse large B‐cell lymphoma through STING signalling
              </a>
            </td>
          <td>
            Zijuan Wu, Wei Zhang, Luqiao Wang, Jiayan Leng, Yongle Li, Zhou Fan, Mengtao Zhan, Lei Cao, Yongning Jiang, Yan Jiang, Bing Sun, Jianxin Fu, Jianyong Li, W. Shi, H. Jin
          </td>
          <td>2024-08-01</td>
          <td>Clinical and Translational Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b69c52796ad88ee763ab51cb8f0791d90f0f5161" target='_blank'>
              Antagonism between H3K27me3 and genome-lamina association drives atypical spatial genome organization in the totipotent embryo.
              </a>
            </td>
          <td>
            Isabel Guerreiro, Franka J. Rang, Yumiko K Kawamura, Carla Kroon-Veenboer, Jeroen Korving, Femke C Groenveld, Ramada E van Beek, Silke J. A. Lochs, Ellen Boele, A. H. Peters, Jop Kind
          </td>
          <td>2024-09-16</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc92a939542e292edaa1bc766f527a0552465d51" target='_blank'>
              PTIP epigenetically regulates DNA damage-induced cell cycle arrest by upregulating PRDM1
              </a>
            </td>
          <td>
            Y. Nakata, Shion Nagasawa, Y. Sera, N. Yamasaki, A. Kanai, K. Kobatake, Takeshi Ueda, M. Koizumi, Ichiro Manabe, Osamu Kaminuma, Hiroaki Honda
          </td>
          <td>2024-08-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The cell cycle-regulated DNA methyltransferase CcrM is conserved in most Alphaproteobacteria, but its role in bacteria with complex or multicentric genomes remains unexplored. Here, we compare the methylome, the transcriptome and the phenotypes of wild-type and CcrM-depleted Agrobacterium tumefaciens cells with a dicentric chromosome with two essential replication origins. We find that DNA methylation has a pleiotropic impact on motility, biofilm formation and viability. Remarkably, CcrM promotes the expression of the repABCCh2 operon, encoding proteins required for replication initiation/partitioning at ori2, and represses gcrA, encoding a conserved global cell cycle regulator. Imaging ori1 and ori2 in live cells, we show that replication from ori2 is often delayed in cells with a hypo-methylated genome, while ori2 over-initiates in cells with a hyper-methylated genome. Further analyses show that GcrA promotes the expression of the RepCCh2 initiator, most likely through the repression of a RepECh2 anti-sense RNA. Altogether, we propose that replication at ori1 leads to a transient hemi-methylation and activation of the gcrA promoter, allowing repCCh2 activation by GcrA and contributing to initiation at ori2. This study then uncovers a novel and original connection between CcrM-dependent DNA methylation, a conserved epigenetic regulator and genome maintenance in an Alphaproteobacterial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad339078add660717cbec9308e097c1844ce59b8" target='_blank'>
              DNA methylation by CcrM contributes to genome maintenance in the Agrobacterium tumefaciens plant pathogen.
              </a>
            </td>
          <td>
            Sandra Martin, Florian Fournes, Giovanna Ambrosini, Christian Iseli, Karolina Bojkowska, Julien Marquis, Nicolas Guex, Justine Collier
          </td>
          <td>2024-09-04</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Summary DNA sequencing is becoming more affordable and faster through advances in high-throughput technologies. This rise in data availability has contributed to the development of novel algorithms to elucidate previously obscure features and led to an increased reliance on complex workflows to integrate such tools into analyses pipelines. To facilitate the analysis of DNA sequencing data, we created metapipeline-DNA, a highly configurable and extensible pipeline. It encompasses a broad range of processing including raw sequencing read alignment and recalibration, variant calling, quality control and subclonal reconstruction. Metapipeline-DNA also contains configuration options to select and tune analyses while being robust to failures. This standardizes and simplifies the ability to analyze large DNA sequencing in both clinical and research settings. Availability Metapipeline-DNA is an open-source Nextflow pipeline under the GPLv2 license and is freely available at https://github.com/uclahs-cds/metapipeline-DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bce629f819b8f5832564713fdef250ffa1edd564" target='_blank'>
              Metapipeline-DNA: A Comprehensive Germline & Somatic Genomics Nextflow Pipeline
              </a>
            </td>
          <td>
            Yash Patel, Chenghao Zhu, Takafumi N. Yamaguchi, Nicholas K. Wang, Nicholas Wiltsie, Alfredo Gonzalez, Helena Winata, Nicole Zeltser, Yu Pan, M. F. E. Mootor, Timothy Sanders, C. Kandoth, Sorel T. Fitz-Gibbon, J. Livingstone, Lydia Y. Liu, Benjamin Carlin, Aaron Holmes, Jieun Oh, John Sahrmann, Shu Tao, Stefan E. Eng, Rupert Hugh-White, Kiarod Pashminehazar, Andrew Park, Arpi Beshlikyan, Madison Jordan, Selina Wu, Mao Tian, J. Arbet, B. Neilsen, Yuan Zhe Bugh, Gina Kim, Joseph Salmingo, Wenshu Zhang, Roni Haas, Aakarsh Anand, Edward Hwang, Anna Neiman-Golden, Philippa L. Steinberg, Wenyan Zhao, Prateek Anand, Brandon L. Tsai, P. Boutros
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Transposable elements (TEs) have emerged as important factors in establishing the cell type-specific gene regulatory networks and evolutionary novelty of embryonic and placental development. Recently, studies on the role of TEs and their dysregulation in cancers have shed light on the transcriptional, transpositional, and regulatory activity of TEs, revealing that the activation of developmental transcriptional programs by TEs may have a role in the dedifferentiation of cancer cells to the progenitor-like cell states. This essay reviews the recent evidence of the cis-regulatory TEs (henceforth crTE) in normal development and malignancy as well as the key transcription factors and regulatory pathways that are implicated in both cell states, and presents existing gaps remaining to be studied, limitations of current technologies, and therapeutic possibilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9ae01d2549bee9fa618c02f57bcecdeddf683a" target='_blank'>
              Transposable elements as drivers of dedifferentiation: Connections between enhancers in embryonic stem cells, placenta, and cancer.
              </a>
            </td>
          <td>
            Konsta Karttunen, Divyesh Patel, Biswajyoti Sahu
          </td>
          <td>2024-07-29</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) disease progression involves complex cell state transitions in both malignant and non-malignant populations within the tumor microenvironment. To uncover PDAC cell states and epigenetic features associated with clinical outcome, we profiled 40 tumor samples (33 primary tumors and 7 metastases) from 39 patients with single cell ATAC-seq, along with 10 sample-matched single cell RNA-seq profiles. Through topic modeling of high dimensional chromatin data, we uncovered distinct cis-regulatory networks underlying the classical and basal-like molecular subtypes of PDAC, which showed differential activity in basal-classical hybrid cells. Furthermore, delineation of epigenetic alterations by basal, classical, and hybrid subtypes revealed a core set of chromatin regions opened in all PDAC subtypes relative to ductal and acinar cells. These universally open regions showed striking overlap with gastrointestinal enhancers, highlighting a common endoderm lineage infidelity process across pancreatic tumors. Within the stromal compartment, we identified novel regulators of myofibroblastic cancer-associated fibroblasts (myCAFs) and inflammatory CAFs (iCAFs) from enhancer networks active in each CAF subtype. Importantly, our data uncover a highly prognostic gene program active in only one of two distinct myCAF chromatin states found in our data, highlighting a putative tumor-promoting myCAF subpopulation. Our work provides a high-resolution description of epigenetic cell state heterogeneity in PDAC, revealing novel regulators and gene programs associated with PDAC subtypes and clinical outcome.
 Citation Format: Kevin MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, Carl Pelz, Trent Waugh, Ethan Agritelle, John Muschler, Andrew Adey, Rosalie Sears. Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55da9db59167776bfc809f1ebb52c0cfa3226d09" target='_blank'>
              Abstract C078: Chromatin accessibility profiling of human pancreatic tumors reveals epigenetic features of malignant and stromal cell subtypes
              </a>
            </td>
          <td>
            Kevin A MacPherson-Hawthorne, Jason M. Link, Patrick Worth, Brett Sheppard, Andrew Fields, Colin Daniel, Andrew Nishida, C. Pelz, Trent Waugh, Ethan Agritelle, John L. Muschler, Andrew C. Adey, Rosalie C. Sears
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d22a32ba49d7731185468f66f4a9a15ddb1efe55" target='_blank'>
              Is cross-species horizontal gene transfer responsible for gallbladder carcinogenesis
              </a>
            </td>
          <td>
            Monika Rajput, Manoj Pandey, R. Dixit, Vijay K. Shukla
          </td>
          <td>2024-07-30</td>
          <td>World Journal of Surgical Oncology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Background Mycosis fungoides (MF) is the most common subtype of cutaneous T-cell lymphoma (CTCL). Comprehensive analysis of MF cells in situ and ex vivo is complicated by the fact that is challenging to distinguish malignant from reactive T cells with certainty. Methods To overcome this limitation, we performed combined single-cell RNA (scRNAseq) and T-cell receptor TCR sequencing (scTCRseq) of skin lesions of cutaneous MF lesions from 12 patients. A sufficient quantity of living T cells was obtained from 9 patients, but 2 had to be excluded due to unclear diagnoses (coexisting CLL or revision to a fixed toxic drug eruption). Results From the remaining patients we established single-cell mRNA expression profiles and the corresponding TCR repertoire of 18,630 T cells. TCR clonality unequivocally identified 13,592 malignant T cells. Reactive T cells of all patients clustered together, while malignant cells of each patient formed a unique cluster expressing genes typical of naive/memory, such as CD27, CCR7 and IL7R, or cytotoxic T cells, e.g., GZMA, NKG7 and GNLY. Genes encoding classic CTCL markers were not detected in all clusters, consistent with the fact that mRNA expression does not correlate linearly with protein expression. Nevertheless, we successfully pinpointed distinctive gene signatures differentiating reactive malignant from malignant T cells: keratins (KRT81, KRT86), galectins (LGALS1, LGALS3) and S100 genes (S100A4, S100A6) being overexpressed in malignant cells. Conclusions Combined scRNAseq and scTCRseq not only allows unambiguous identification of MF cells, but also revealed marked heterogeneity between and within patients with unexpected functional phenotypes. While the correlation between mRNA and protein abundance was limited with respect to established MF markers, we were able to identify a single-cell gene expression signature that distinguishes malignant from reactive T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/212bc68159cf864c0e23d5af31516d6565474ee3" target='_blank'>
              Single-cell RNA and T-cell receptor sequencing unveil mycosis fungoides heterogeneity and a possible gene signature
              </a>
            </td>
          <td>
            Nalini Srinivas, Lukas Peiffer, K. Horny, Kuan Cheok Lei, T. B. Buus, Linda Kubat, Meng Luo, Menghong Yin, Ivelina Spassova, A. Sucker, Farnoush Farahpour, Jan Kehrmann, Selma Ugurel, E. Livingstone, T. Gambichler, Niels Ødum, Jürgen C. Becker
          </td>
          <td>2024-08-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Alzheimer's disease (AD) is a common and increasing societal problem due to the extending human lifespan. In males, loss of chromosome Y (LOY) in leukocytes is strongly associated with AD. We studied here DNA methylation and RNA expression in sorted monocytes and granulocytes with and without LOY from male AD patients. Through multi-omic analysis, we identified new candidate genes and confirmed the involvement of numerous genes previously associated with AD. Our findings highlight LOY-related differences in DNA methylation that occur in gene regulatory regions and are predominantly accompanied by down-regulation of affected genes. Specifically, we observed alterations in key genes involved in leukocyte differentiation: FLI1, involved in early hematopoiesis; RUNX1, essential for blood cell development; RARA, regulating gene expression in response to retinoic acid; CANX, crucial for protein folding; CEBPB, a transcription factor important for immune responses; and MYADM, implicated in cell adhesion and migration. Moreover, protein-protein interaction analysis in granulocytes identified that products of two of these genes, CANX and CEBPB, are key hub proteins. Thus, LOY appears to dysregulate genes involved in leukocyte differentiation and induce higher-level epigenetic changes. This research underscores the potential of multi-omic approaches in pure cell populations to uncover the molecular underpinnings of AD and reinforces the significance of LOY as a pathogenic factor in this disease. Overall, results support the hypothesis that age-related immune cell dysfunction contributes to AD development. Finally, our results link previous analysis showing impact of LOY on leukocyte differentiation, LOY-associated transcriptional dysregulation and GWAS studies of LOY.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9899fb011458e044621821bc8b19932591bd72f" target='_blank'>
              DNA methylation patterns contribute to changes of cellular differentiation pathways in leukocytes with LOY from patients with Alzheimer''s disease
              </a>
            </td>
          <td>
            M. Jakalski, B. Bruhn-Olszewska, E. Rychlicka-Buniowska, H. Davies, D. Sarkisyan, M. Siedlar, J. Baran, K. Weglarczyk, J. Jaszczynski, J. Rys, V. Gedraitis, N. Filipowicz, A. Klich-Raczka, L. Kilander, M. Ingelsson, J. P. Dumanski
          </td>
          <td>2024-08-20</td>
          <td>None</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de9b939e6665a85e15c06d7919e4ac22bd18c3f" target='_blank'>
              Cellular origin and clonal evolution of human dedifferentiated liposarcoma
              </a>
            </td>
          <td>
            N. Gruel, C. Quignot, Laëtitia Lesage, Sophie El Zein, S. Bonvalot, Dimitri Tzanis, Khadija Ait Rais, Fabien Quinquis, Bastien Manciot, J. Vibert, Nadine El Tannir, A. Dahmani, Heloise Derrien, Didier Decaudin, Ivan Bièche, L. Courtois, O. Mariani, Laetitia K Linares, L. Gayte, Sylvain Baulande, J. Waterfall, O. Delattre, G. Pierron, Sarah Watson
          </td>
          <td>2024-09-12</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>40</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7d183f3062975c60ce5ba3ada1020c80a06fee4" target='_blank'>
              Prion-like domain mediated phase separation of ARID1A promotes oncogenic potential of Ewing’s sarcoma
              </a>
            </td>
          <td>
            Yong Ryoul Kim, Jaegeon Joo, Hee Jung Lee, Chaelim Kim, Ju-Chan Park, Young Suk Yu, Chang Rok Kim, Do Hui Lee, Joowon Cha, Hyemin Kwon, Kimberley M. Hanssen, T. G. Grünewald, Murim Choi, Ilkyu Han, Sangsu Bae, Inkyung Jung, Yongdae Shin, Sung Hee Baek
          </td>
          <td>2024-08-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb6a4f203d0ffda30fcc3f0dfa56bd3b405a484" target='_blank'>
              Amplified centrosomes-more than just a threat.
              </a>
            </td>
          <td>
            Eva Kiermaier, Isabel Stötzel, Marina A Schapfl, A. Villunger
          </td>
          <td>2024-09-16</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="DNAJB6, a major member of the DNAJ/HSP40 family, plays an important role in tumor development. We explored the effect of DNAJB6 expression on the prognosis of patients and its biological role in lung adenocarcinoma (LUAD). mRNA and clinical data were obtained from The Cancer Genome Atlas (TCGA). Enriched pathways were determined by the Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses. A nomogram incorporating DNAJB6 and three clinical features was constructed to predict the survival rate. DNAJB6 expression and function in LUAD were explored using immunohistochemistry, Western blotting, proliferation, cell cycle analysis, RNA sequencing, and xenograft tumor assays. DNAJB6 mRNA levels were elevated in the LUAD-TCGA dataset. DNAJB6 protein levels were higher in LUAD tumor tissues than in normal tissues. A high DNAJB6 level was an independent risk factor for poor prognosis in patients with LUAD. The proportion of tumor-infiltrating immune cells significantly differed between high and low DNAJB6 expression. DNAJB6 was associated with cell cycle pathways; therefore, its knockdown induced G2/M cell cycle arrest and inhibited LUAD cell proliferation. This is the first report of the DNAJB6 requirement for LUAD cell proliferation and its potentially crucial role in LUAD prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd99e6105741818b2b8a1ce73bd38d2fcdc17926" target='_blank'>
              Abnormally High Expression of DNAJB6 Accelerates Malignant Progression of Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Di Wang, Jiayu Xiao, Yang Du, Li Zhang, Xuzhen Qin
          </td>
          <td>2024-09-02</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Introduction Pediatric acute myeloid leukemia (AML) with NUP98 rearrangements (NUP98r) is associated with dismal prognoses. NUP98 fusion partners are linked with specific AML subtypes, such as NUP98::NSD1 with myeloblastic AML and NUP98::KDM5A with acute megakaryocytic leukemia (AMKL). Although mostly present in AML, various NUP98r are also found in other malignancies such as T-cell acute lymphoblastic leukemia (T-ALL) and myelodysplastic syndrome (MDS). Despite genomic studies showing associations between FLT3 internal tandem duplications (FLT3-ITD) and WT1 mutations with NUP98::NSD1 AML, the genome-wide mutational status of NUP98r leukemias is to be elucidated. Also, experimental models showed that NUP98 fusion oncoproteins (FOs) regulate gene expression by binding to genomic loci such as HOX genes or MEIS1; however, how different partners contribute to specific diseases remains unstudied. Methods We established a pediatric NUP98r leukemia cohort consisting of 185 samples from 177 cases from our institute, public databases, or collaborators. AML is the dominant disease (n=154), whereas it also included T-ALL (n=18), therapy-related myeloid neoplasms (t-MN, n=4), and MDS (n=1). RNA-sequencing (RNA-seq) data was obtained for all samples, and whole-genome/exome sequencing (WGS/WES) data was obtained from 94 samples. Fusion genes were called from RNA-seq. Somatic alterations were called from both WGS/WES and RNA-seq. The epigenetic status and NUP98 FO-binding of 14 cases were studied using CUT&RUN with antibodies against the N-terminal NUP98, H3K27ac, H3K4me, and K3H27ac. Results Among 177 cases, NUP98::NSD1 was most prevalent (n=92) followed by NUP98::KDM5A (n=45). This cohort also included 11 cases with NUP98::RAP1GDS1 enriched in T-ALL and immature AML, rarely reported in previous NUP98r studies. Genomic profiling confirmed known associations with NUP98::NSD1 AML with FLT3-ITD or WT1, along with novel associations represented by NOTCH1, CCND3, and NRAS mutations with NUP98r T-ALL. Transcriptional analyses with other AML subtypes (n=821) showed NUP98r cases are distributed in five large clusters, where fusions and mutations were associated with specific clusters but not exclusively, suggesting that fusion partners, cooperating alterations, and differentiation states contribute to the disease phenotypes. Epigenetic profiling of 14 cases (NSD1: n=5, KDM5A: n=6, RAP1GDS1: n=3) showed that H3K27ac patterns reflect disease types while H3K27me3 and NUP98-FO binding patterns are intrinsic to fusion partners. NUP98 FOs bound to genes commonly expressed in NUP98r leukemia such as HOXA-B genes or ETV6, whereas NUP98-FOs bound to differentiation-related genes depending on the leukemia subtype (e.g., NUP98::KDM5A binding to the GFI1B loci in AMKL). Conclusion These data suggest that NUP98 FOs specify disease types by directly regulating differentiation-related genes. Further investigation by introducing somatic mutations (e.g., WT1 mutations, RB1 loss) in NUP98r models would reveal its contribution to gene regulation and disease types.
 Citation Format: Masayuki Umeda, Nicole Michmerhuizen, Ryan Hiltenbrand, Juan M. Barajas, Michael P. Walsh, Guangchun Song, Jing Ma, Tamara Westover, Cristina Mecucci, Danika Di Giacomo, Franco Locatelli, Riccardo Masetti, Salvatore Bertuccio, Martina Pigazzi, Ilaria Iacobucci, Charles G. Mullighan, Jeffery M. Klco. Genetic and epigenetic characterization of NUP98-rearranged leukemia [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A019.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60d94d1c36d05baa7cc25171e29df8b2f95233b1" target='_blank'>
              Abstract A019 Genetic and epigenetic characterization of NUP98-rearranged leukemia
              </a>
            </td>
          <td>
            Masayuki Umeda, Nicole Michmerhuizen, R. Hiltenbrand, Juan M Barajas, Michael P. Walsh, Guangchun Song, Jing Ma, Tamara Westover, Cristina Mecucci, Danika Di Giacomo, Franco Locatelli, Riccardo Masetti, S. Bertuccio, M. Pigazzi, Ilaria Iacobucci, C. Mullighan, J. Klco
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb1fa9923fb71606ea55e83b6b8cbb7a9e56b764" target='_blank'>
              Competition shapes the landscape of X-chromosome-linked genetic diversity
              </a>
            </td>
          <td>
            Teresa Buenaventura, Hakan Bagci, Ilinca Patrascan, Joshua J Graham, Kelsey D Hipwell, Roel Oldenkamp, James W D King, Jesus Urtasun, G. Young, Daniel Mouzo, D. Gómez-Cabrero, B. Rowland, Daniel Panne, Amanda G Fisher, Matthias Merkenschlager
          </td>
          <td>2024-07-26</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2245c6c9be5dc1954f3bd2bc26c574b380581c40" target='_blank'>
              Comprehensive genomic analysis reveals clonal origin and subtype-specific evolution in a case of sporadic multiple meningiomas.
              </a>
            </td>
          <td>
            M. Sakaguchi, Masafumi Horie, Yukinobu Ito, Shingo Tanaka, Keishi Mizuguchi, Hiroko Ikeda, Etsuko Kiyokawa, Mitsutoshi Nakada, Daichi Maeda
          </td>
          <td>2024-07-27</td>
          <td>Brain tumor pathology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="
 Despite extensive research, the complex biology of pancreatic ductal adenocarcinoma (PDAC) continues to impede progress in improving its prognosis. Its resistance to conventional therapies, along with diverse mutations, necessitate improved molecular stratification to develop tailored approaches and to improve survival of PDAC patients. Our previous studies on ATM- deficient models revealed that serine/threonine kinase ATM loss induces homologous recombination deficiency (HRD), promoting invasiveness and desmoplastic reactions, while also sensitizing tumors to platinum-based therapies and PARP1 inhibition. Interestingly, our data also identified the histone-methyltransferase EZH2 as a direct ATM substrate to operate as a crucial regulator of ATM-specific oncogenic traits. In detail, murine and human ATM-deficient tumors accumulated EZH2, suggesting that ATM status contributes to rewiring the epigenetic landscape of pancreatic tumors, subsequently supporting oncogenic features. To investigate how EZH2 impacts ATM-deficient PDAC pathogenesis, we generated conditional double ATM and EZH2-knockout in the Kras G12D/+ ; Ptf1a Cre/+ mice (AKC, Atm fl/fl ; LSL-Kras G12D/+ ; Ptf1a Cre/+ , and EAKC, Ezh2 fl/fl ; Atm fl/fl ; LSL-Kras G12D/+ ; Ptf1a Cre/+ ). EZH2 depletion in AKC mice prolonged overall survival and reinstated a differentiated phenotype. Furthermore, PDAC patients with negative ATM and EZH2 expression from the TCGA-PAAD cohort (PanCancer Atlas) also showed significantly extended progression-free survival compared to ATM neg EZH2 pos patients. In line, CRISPR/Cas9-mediated EZH2 depletion in ATM-deficient pancreatic cancer cells affected their epigenetic landscape and significantly upregulated transcriptomic programs associated with cell differentiation, thereby restoring differentiated tumor features ex vivo. We showed that ATM status significantly impacts EZH2 activity through a post-translational pathway, promoting tumor supportive EZH2 activity in pancreatic cancer. ATM kinase facilitates EZH2 ubiquitination and degradation by recruiting the Cullin1 E3-ubiquitin ligase complex. Additionally, combining an EZH2 inhibitor with PARP inhibition enhances cytotoxicity in ATM- deficient HRD PDAC. Overall, our findings demonstrate that ATM deficiency drives a pro- oncogenic mesenchymal differentiation program via the histone-methyltransferase EZH2. We emphasize the importance of stratifying patients based on ATM status before considering pharmacological interference with EZH2, offering a promising approach for targeting ATM- deficient HRD tumors.
 Citation Format: Elodie Roger, Anna Härle, Lukas Perkhofer, Elisabeth Hessmann, Alexander Kleger, Johann Gout. Epigenetic reprograming in ATM-deficient pancreatic cancer: targeting EZH2- mediated oncogenic traits [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr B088.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde281aef7a4cfc6dbd17009895cff9b452a2fd" target='_blank'>
              Abstract B088: Epigenetic reprograming in ATM-deficient pancreatic cancer: targeting EZH2- mediated oncogenic traits
              </a>
            </td>
          <td>
            E. Roger, A. Härle, L. Perkhofer, E. Hessmann, A. Kleger, J. Gout
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Deleterious accumulation of R-loops, a DNA-RNA hybrid structure, contributes to genome instability. They are associated with BRCA1 mutation-related breast cancer, an estrogen receptor α negative (ERα-) tumor type originating from luminal progenitor cells. However, a presumed causality of R-loops in tumorigenesis has not been established in vivo. Here, we overexpress mouse Rnaseh1 (Rh1-OE) in vivo to remove accumulated R-loops in Brca1-deficient mouse mammary epithelium (BKO). R-loop removal exacerbates DNA replication stress in proliferating BKO mammary epithelial cells, with little effect on homology-directed repair of double-strand breaks following ionizing radiation. Compared to their BKO counterparts, BKO-Rh1-OE mammary glands contain fewer luminal progenitor cells but more mature luminal cells. Despite a similar incidence of spontaneous mammary tumors in BKO and BKO-Rh1-OE mice, a significant percentage of BKO-Rh1-OE tumors express ERα and progesterone receptor. Our results suggest that rather than directly elevating the overall tumor incidence, R-loops influence the mammary tumor subtype by shaping the cell of origin for Brca1 tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd59964db49fa8717ee1e81f17faaff9786150ba" target='_blank'>
              R-loop functions in Brca1-associated mammary tumorigenesis.
              </a>
            </td>
          <td>
            Huai-Chin Chiang, Leilei Qi, Payal Mitra, Yimeng Huang, Yanfen Hu, Rong Li
          </td>
          <td>2024-08-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="TAD boundaries are genomic elements that separate biological processes in neighboring domains by blocking DNA loops that are formed through Cohesin-mediated loop extrusion. Most TAD boundaries consist of arrays of binding sites for the CTCF protein, whose interaction with the Cohesin complex blocks loop extrusion. TAD boundaries are not fully impermeable though and allow a limited amount of inter-TAD loop formation. Based on the reanalysis of Nano-C data, a multicontact Chromosome Conformation Capture assay, we propose a model whereby clustered CTCF binding sites promote the successive stalling of Cohesin and subsequent dissociation from the chromatin. A fraction of Cohesin nonetheless achieves boundary read-through. Due to a constant rate of Cohesin dissociation elsewhere in the genome, the maximum length of inter-TAD loops is restricted though. We speculate that the DNA-encoded organization of stalling sites regulates TAD boundary permeability and discuss implications for enhancer-promoter loop formation and other genomic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7707b0397caa81eae89d125f02c4b85e10532d65" target='_blank'>
              Permeable TAD boundaries and their impact on genome-associated functions.
              </a>
            </td>
          <td>
            Li-Hsin Chang, D. Noordermeer
          </td>
          <td>2024-08-02</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c115fcb82a4ae08aa16b7db64204e01871cc9b8b" target='_blank'>
              Prognostic value of nucleotide excision repair and translesion DNA synthesis proteins in muscle-infiltrating bladder carcinoma
              </a>
            </td>
          <td>
            P. Palacka, Andrea Holíčková, J. Roška, Peter Makovický, M. Vallová, Csaba Biró, Eveline Órásová, J. Obertová, J. Mardiak, Thomas A Ward, Karol Kajo, M. Chovanec
          </td>
          <td>2024-09-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="ABSTRACT Background Prostate cancer (PCa) is among the three main types of cancer. Although prostate-specific antigen (PSA) is routinely tested, it has disadvantages, such as poor prognostic ability. Therefore, finding more PCa markers and therapeutic targets remains a subject of study. CircRNAs have been found to have regulatory roles in various diseases, such as diabetes, Central Nervous System (CNS) neuropathy, etc. where their application in cancer is even more valuable. Therefore, this paper aims to search for differentially expressed circRNAs in PCa and find downstream targeting pathways related to autophagy. Method By detecting the expression of circRNA in the samples, hsa_circ_0119816 was finally identified as the research target. The properties of circRNA were verified by RNase R, actinomycin D, and fluorescence in situ hybridization (FISH). The downstream target miRNAs and target proteins were predicted by an online database, and the targeting relationship was verified using dual luciferase and RNA Immunoprecipitation. The effects of circRNAs and their downstream signalling pathways on prostate cancer cell proliferation, migration, EMT and autophagy were examined by CCK-8, Transwell, immunofluorescence and Western blotting. Results CircBIRC6 is highly expressed in prostate cancer samples. Knockdown of its expression inhibits cell proliferation, invasion, EMT and autophagy and promotes apoptosis. CircBIRC6/miRNA-574-5p/DNAJB1 is a molecular axis that regulates prostate cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9d1bad6dbf3ace4247e9abe87ef37ab3c3b685" target='_blank'>
              CircBIRC6 affects prostate cancer progression by regulating miR-574-5p and DNAJB1
              </a>
            </td>
          <td>
            Bin Zhao, Jinye Yang, Fengming Ran, Yuanlong Shi, Libo Yang, Yuanpeng Duan, Zhiyu Shi, Xin Li, Jianpeng Zhang, Zhiyao Li, Jiansong Wang
          </td>
          <td>2024-09-11</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Human DNA methylation profiling offers great promises in various biomedical applications, including ageing, cancer and even forensics. So far, most DNA methylation techniques are based on a chemical process called sodium bisulfite conversion, which specifically converts non-methylated cytosines into uracils. However, despite the popularity of this approach, it is known to cause DNA fragmentation and loss affecting standardization, while incomplete conversion may result in potential misinterpretation of methylation-based outcomes. To offer the community a solution, we developed qBiCo - a novel quality-control method to address the quantity and quality of bisulfite-converted DNA. qBiCo is a 5-plex, TaqMan® probe-based, quantitative (q)PCR assay that amplifies single- and multi-copy DNA fragments of converted and non-converted nature. It estimates four parameters: converted DNA concentration, fragmentation, global conversion efficiency, and potential PCR inhibition. We optimized qBiCo using synthetic DNA standards and assessed it using standard developmental validation criteria, showcasing that qBiCo is reliable, robust and sensitive down to picogram level. We also evaluated its performance by testing decreasing DNA amounts using several commercial bisulfite conversion kits. Depending on the starting DNA quantity, bisulfite-converted DNA recoveries ranged from 8.5 – 100 %, conversion efficiencies from 78 - 99.9 %, while certain kits highly fragment DNA, demonstrating large variability in their performance. Towards building a prototype tool, we further optimized key functionalities, for example, by replacing the poorest performing single-plex assay and creating a more representative DNA standard. Aiming to scale-up and move towards implementation, we successfully transferred and validated our novel method in six different qPCR platforms from different major manufacturers. Overall, with the present study, we offer researchers in the epigenetic field a novel long-awaited QC tool that for the first time allows them to measure key quality and quantity parameters of the most popular DNA conversion process. The tool also enables standardization to prevent inconsistent data and false outcomes in the future, regardless of the downstream experimental analysis of DNA methylation-based research and applications across different fields of biology and biomedicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5f2d2f1ff7d9e8b8418047a193bf39b8fe5d70" target='_blank'>
              qBiCo: A method to assess global DNA conversion performance in epigenetics via single-copy genes and repetitive elements
              </a>
            </td>
          <td>
            Faidra Karkala, Roy B. Simons, Floor Claessens, V. Kalamara, M. Kayser, A. Vidaki
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44a8b3281c13d1ffc75f1261f03703dc8638c833" target='_blank'>
              A protein expression atlas on tissue samples and cell lines from cancer patients provides insights into tumor heterogeneity and dependencies.
              </a>
            </td>
          <td>
            Jun Li, Wei Liu, K. Mojumdar, Hong Kim, Zhicheng Zhou, Z. Ju, Shweth V. Kumar, P. Ng, Han Chen, Michael A. Davies, Yiling Lu, Rehan Akbani, Gordon B. Mills, Han Liang
          </td>
          <td>2024-09-03</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Cervical cancer remains a significant health challenge for women worldwide, with a disproportionate impact on developing regions like sub-Saharan Africa. Taking advantage of recent advancements in developing suitable preclinical models to study cell proliferation, differentiation, and gene expression, we used RNA sequencing to compare the transcriptomic profiles of SiHa cervical cancer cells grown in 3D versus 2D culture systems. Pathway analysis of 3D cultures revealed upregulation of immune activation, angiogenesis, and tissue remodeling pathways. The high expression of cytokines, chemokines, matrix metalloproteinases, and immediate early genes, suggests that 3D cultures replicate the tumor microenvironment better than 2D monolayer cultures. HPV gene expression analysis further demonstrated higher expression levels of HPV16 E1, E2, E6, and E7 genes in 3D versus 2D cultures. Further, by using a set of linear models, we identified 79 significantly differentially expressed genes in 3D culture compared to 2D culture conditions, independent of HPV16 viral gene effects. We subsequently validated five of these genes at the protein level in both the SiHa cell line and a newly developed, patient-derived cervical cancer cell line. In addition, correlation analysis identified 26 human genes positively correlated with viral genes across 2D and 3D culture conditions. The top five 3D versus 2D differentially expressed and HPV-correlated genes were validated via qRT-PCR in our patient derived cell line. Altogether, these findings suggest that 3D cultures provide superior model systems to explore mechanisms of immune evasion, cancer progression and antiviral therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72fbc566c536b3ffc1509f8f51df954d9d223184" target='_blank'>
              Deciphering the divergent transcriptomic landscapes of cervical cancer cells grown in 3D and 2D cell culture systems
              </a>
            </td>
          <td>
            Roshan Kumar, Marissa Iden, Shirng-Wern Tsaih, Rachel Schmidt, A. I. Ojesina, Janet S. Rader
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="59 Background: Colorectal cancer (CRC) is the 3rd most common cancer. The consensus molecular subtypes among CRC tumors reflects the heterogeneity of composition and functional states of cells in the tumor microenvironments (TME), which in turn may underlie diverse behaviors of disease progression and responses to anticancer treatments. Cells within a TME communicate through intricate cell-cell communication (CCC) networks. The altered state of one cell can influence the states of other cells through ligand-receptor-mediated signal transduction. Gaining insight into the cellular states of cells and understanding their complex communications in a TME is vital for uncovering the heterogeneous mechanisms governing tumor growth, immune evasion, and therapy resistance. Methods: In this meta-analysis of eight single-cell cohorts encompassing 153 patients and 279 samples with more than 600,000 cells, we advance the understanding of CCC networks in CRC through a novel analytical framework. Employing hierarchical language modeling, we identify gene expression modules (GEMs) that mirror single-cell signaling states, crucial for deciphering the complexity of intercellular interactions. By applying causal discovery methods, we systematically uncover GEMs likely regulated by ligand-receptor signaling and cross-cell-type communication. We further validate the discovered CCC using spatial transcriptomic data by testing the spatial co-localization. Results: This analysis reveals nine cross-cell-type CCC programs, marked by highly correlated GEMs across various cell types, shedding light on the intricate CCC networks within the TME. The discovered CCC programs include malignant program, proliferation program, stromal interactions, stromal-myeloid interactions, innate immunity interactions, regulatory interactions, epithelial-lymphocyte interactions and exhaustion program. Each program is composed of GEMs from various cell types. Spatial transcriptomics further validate these findings by demonstrating the co-localization of GEMs within CCC programs in distinct spatial domains, emphasizing the spatial dynamics of tumor intercellular communication. We successfully construct the CCC subnetworks connected by ligand-receptor signaling. Our interactive website and analytical framework equip researchers with powerful tools to explore complex mechanisms, potentially uncovering novel drug targets and refining strategies for precision immunotherapies. Conclusions: This study provides an in-depth analysis of colorectal cancer by: 1) Cataloging GEMs that precisely depict the transcriptomic processes unique to individual cell clusters or shared among multiple cell clusters. 2) Presenting the CCC networks driven by ligand-receptor interactions within the TME supported by both single-cell RNA-seq data and spatial transcriptomic data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/084c7d8c4635a2cae3a392956680a999782d17a9" target='_blank'>
              Effect of deconvoluting single-cell transcriptomics on cellular programs regulated by cell-cell communication in colorectal cancer.
              </a>
            </td>
          <td>
            Lujia Chen, Han Zhang, Binfeng Lu, Gregory F. Cooper, A. Saeed, Xinghua Lu
          </td>
          <td>2024-08-10</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Signet Ring Cell Carcinoma (SRCC) is a rare and highly malignant form of adenocarcinoma with increasing incidence and poor prognosis due to late diagnosis and limited treatment options. We employed Deep Visual Proteomics (DVP), which combines AI directed cell segmentation and classification with laser microdissection and ultra-high sensitivity mass spectrometry, for cell-type specific proteomic analysis of SRCC across the bladder, prostate, liver, and lymph nodes of a single patient. DVP identified significant alterations in DNA damage response (DDR) proteins, particularly within the ATR and mismatch repair (MMR) pathways, indicating replication stress as a crucial factor in SRCC mutagenicity. Additionally, we observed substantial enrichment of immune-related proteins, reflecting high levels of cytotoxic T lymphocyte infiltration and elevated PD-1 expression. These findings suggest that pembrolizumab immunotherapy may be more effective than conventional chemotherapy for this patient. Our results provide novel insights into the proteomic landscape of SRCC, identifying potential targets and open up for personalized therapeutic strategies in managing SRCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e14daaee3cc1fcece9f6df4b26842260cdd68a8" target='_blank'>
              Deep Visual Proteomics reveals DNA replication stress as a hallmark of Signet Ring Cell Carcinoma
              </a>
            </td>
          <td>
            Sonja Kabatnik, Xiang Zheng, Georgios Pappas, Sophia Steigerwald, M. Padula, Matthias Mann
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1230bf0b0e23a37cb0ef069af27c70638d7b9578" target='_blank'>
              Cell-free DNA from germline TP53 mutation carriers reflect cancer-like fragmentation patterns
              </a>
            </td>
          <td>
            Derek Wong, Maha Tageldein, Ping Luo, Erik Ensminger, Jeffrey P. Bruce, L. Oldfield, Haifan Gong, N. Fischer, B. Laverty, Vallijah Subasri, Scott Davidson, Reem Khan, A. Villani, A. Shlien, R. Kim, David Malkin, Trevor J. Pugh
          </td>
          <td>2024-08-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Glioblastoma is the most common primary malignant brain tumour. Despite decades of intensive research in the disease, its prognosis remains poor, with an average survival of only 14 months after diagnosis. The remarkable level of intra- and interpatient heterogeneity is certainly contributing to the lack of progress in tackling this tumour. Epigenetic dysregulation plays an important role in glioblastoma biology and significantly contributes to intratumour heterogeneity. However, it is becoming increasingly clear that it also contributes to intertumour heterogeneity, which historically had mainly been linked to diverse genetic events occurring in different patients. In this review, we explore how DNA methylation, chromatin remodelling, microRNA (miRNA) dysregulation, and long noncoding RNA (lncRNA) alterations contribute to intertumour heterogeneity in glioblastoma, including its implications for advanced tumour stratification, which is the essential first step for developing more effective patient-specific therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8a905205728cf89ec2231b745c2dd76f674ecb4" target='_blank'>
              Unravelling the mosaic: Epigenetic diversity in glioblastoma.
              </a>
            </td>
          <td>
            Sara Lucchini, Myrianni Constantinou, Silvia Marino
          </td>
          <td>2024-08-15</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer (BC) is the most frequently diagnosed cancer and the primary cause of cancer-related mortality in women. Treatment of triple-negative breast cancer (TNBC) remains particularly challenging due to its resistance to chemotherapy and poor prognosis. Extensive research efforts in BC screening and therapy have improved clinical outcomes for BC patients. Therefore, identifying reliable biomarkers for TNBC is of great clinical importance. Here, we found that tyrosine aminotransferase (TAT) expression was significantly reduced in BC and strongly correlated with the poor prognosis of BC patients, which distinguished BC patients from normal individuals, indicating that TAT is a valuable biomarker for early BC diagnosis. Mechanistically, we uncovered that methylation of the TAT promoter was significantly increased by DNA methyltransferase 3 (DNMT3A/3B). In addition, reduced TAT contributes to DNA replication and cell cycle activation by regulating homologous recombination repair and mismatch repair to ensure genomic stability, which may be one of the reasons for TNBC resistance to chemotherapy. Furthermore, we demonstrated that Diazinon increases TAT expression as an inhibitor of DNMT3A/3B and inhibits the growth of BC by blocking downstream pathways. Taken together, we revealed that TAT is silenced by DNMT3A/3B in BC, especially in TNBC, which promotes the proliferation of tumor cells by supporting DNA replication, activating cell cycle, and enhancing DNA damage repair. These results provide fresh insights and a theoretical foundation for the clinical diagnosis and treatment of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98a77f352d33a7a2b9ba7ce61fe82a2e23ebd12e" target='_blank'>
              Identification of TAT as a Biomarker Involved in Cell Cycle and DNA Repair in Breast Cancer
              </a>
            </td>
          <td>
            Fei Xie, Saiwei Hua, Yajuan Guo, Taoyuan Wang, Changliang Shan, Lianwen Zhang, Tao He
          </td>
          <td>2024-08-30</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pediatric acute lymphoblastic leukemia (ALL) is a highly heterogeneous disease at the molecular level, with over 20 subtypes of B-cell ALL (B-ALL) identified to date. These subtypes are critical for guiding risk-adapted therapy and precision medicine, as they are defined by specific chromosomal rearrangements, gene expression profiles, aneuploidies and point mutations. However, the ability to accurately classify these subtypes is limited by the technical and economic challenges faced by many centers, particularly in low- and middle-income countries. In this study, we employed transcriptome sequencing to perform molecular classification of B-ALL at diagnosis in pediatric patients enrolled in the multicentric ALLIC-GATLA-2010 clinical protocol in Argentina. Using a combination of bioinformatic tools, we successfully identified single nucleotide variants, fusion transcripts and gene expression profiles, achieving molecular classification in over 90% of patients. Our analysis also revealed high-risk molecular features and novel genetic alterations. These findings hold potential clinical value for improving risk stratification and identifying therapeutic targets, particularly for patients who remain unclassified by conventional diagnostic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f39fa251d291498250d854c37a9028d4e85f79" target='_blank'>
              Molecular Characterization of Pediatric Acute Lymphoblastic Leukemia via Integrative Transcriptomics: A Multicenter Study in Argentina
              </a>
            </td>
          <td>
            Sol Ruiz, María Mercedes Abbate, Ezequiel Sosa, Daniel Avendaño, Ignacio Gomez Mercado, María Laura Lacreu, Mariano Riccheri, Virginia Schuttenberg, Luis Aversa, E. Vazquez, G. Gueron, J. Cotignola
          </td>
          <td>2024-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cbd49dc4f182de3dedf2bd9170f18d407483728" target='_blank'>
              VolcanoSV enables accurate and robust structural variant calling in diploid genomes from single-molecule long read sequencing
              </a>
            </td>
          <td>
            Can Luo, Y. Liu, X. Zhou
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Colorectal adenocarcinoma (COAD) has a poor prognosis. Cyclin-dependent kinase inhibitor 2A (CDKN2A) significantly affects the development and progression of various human tumors. However, the significance and pathological mechanisms of CDKN2A in COAD remain to be elucidated. We assessed expression levels, clinical significance, biological function, co-expressed genes, and enrichment of related pathways of CDKN2A in COAD using various databases, including The University of Alabama at Birmingham Cancer Data Analysis Portal, Gene Expression Profiling Interactive Analysis, Tumor Immune Estimation Resource, Human Protein Atlas, STRING, GeneMANIA, cBioPortal, and Linked Omics. Our investigation showed that CDKN2A was highly expressed in colon adenocarcinomas (P < .001). It is weakly expressed or not expressed in normal tissues. The survival time of patients with colon adenocarcinoma with high CDKN2A expression is significantly shorter than that of patients with low expression levels (P = .011). There was a significant positive correlation between the expression level of CDKN2A in colon adenocarcinoma tissues and the infiltration of CD4+ T cells, macrophages, and neutrophils. Moreover, there was a significant negative association between the expression level of CDKN2A in colon adenocarcinoma tissues and B cell infiltration. The ten hub genes included tumor protein 53, V-myc Avian Myelocytomatosis Viral Oncogene Homolog, AKT serine/threonine kinase 1, cyclin-dependent kinase 2, phosphatase and tensin homolog deleted on chromosome ten, cyclin D1, cyclin dependent kinase 4, cyclin dependent kinase inhibitor 1A, catenin beta 1, and B-Raf proto-oncogene, serine/threonine kinase. Mutations in the CDKN2A genome in colon adenocarcinoma reduce survival. Gene ontology and Kyoto Encyclopedia of Genes and Genomes analyses showed that the differentially expressed genes were enriched in apoptotic signaling pathways and multiple pathways related to metabolic progression. Our results indicate that CDKN2A can be used as a marker of poor prognosis in patients with colon adenocarcinoma. CDKN2A may regulate the occurrence and development of colon adenocarcinomas by influencing immune cell infiltration and metabolic pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae51b20c2695c36facac2469a11d7ddc7c4c48d6" target='_blank'>
              Impact of CDKN2A gene expression on colon adenocarcinoma via biosignature analysis.
              </a>
            </td>
          <td>
            Chen Xu
          </td>
          <td>2024-09-06</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcaa5e009963f12c7b3cfac1a62813bb1ec1e2e7" target='_blank'>
              Removal of TREX1 activity enhances CRISPR-Cas9-mediated homologous recombination.
              </a>
            </td>
          <td>
            Mehmet E. Karasu, Eléonore Toufektchan, Yanyang Chen, Alessandra Albertelli, G. Cullot, J. Maciejowski, Jacob E. Corn
          </td>
          <td>2024-08-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37216ded337bd372b65a91cb1670af84243d9871" target='_blank'>
              Single cell whole-genome sequencing of brain cells: age- and cell-type specific mutational profiles
              </a>
            </td>
          <td>
            Melania Capasso, N. A. Aziz
          </td>
          <td>2024-08-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7401c81aea35658fe72a5961ba5c3e31cae1d2fa" target='_blank'>
              The genomic basis of childhood T-lineage acute lymphoblastic leukaemia.
              </a>
            </td>
          <td>
            P. Pölönen, Danika Di Giacomo, A. Seffernick, Abdelrahman A. Elsayed, Shunsuke Kimura, Francesca Benini, Lindsey E Montefiori, Brent L Wood, Jason Xu, Changya Chen, Zhong-Yi Cheng, H. Newman, Jason R Myers, Ilaria Iacobucci, Elizabeth Li, Jonathan H. Sussman, Dale Hedges, Yawei Hui, Caroline Diorio, Lahari Uppuluri, David Frank, Yiping Fan, Yunchao Chang, S. Meshinchi, R. Ries, R. Shraim, Alexander Li, Kathrin M Bernt, M. Devidas, S. Winter, K. Dunsmore, H. Inaba, William L Carroll, Nilsa C Ramirez, Aaron H. Phillips, Richard W. Kriwacki, Jun J Yang, T. Vincent, Yaqi Zhao, P. S. Ghate, Jian Wang, Colleen Reilly, Xin Zhou, Mathijs A. Sanders, Junko Takita, Motohiro Kato, Nao Takasugi, Bill H Chang, Richard D Press, M. Loh, E. Rampersaud, E. Raetz, S P Hunger, Kai Tan, Ti-Cheng Chang, Gang Wu, S. B. Pounds, C. Mullighan, D. Teachey
          </td>
          <td>2024-08-14</td>
          <td>Nature</td>
          <td>2</td>
          <td>102</td>
        </tr>

        <tr id="Here, we present a method for enrichment of double-stranded cfDNA with an average length of ∼40 bp from cfDNA for high-throughput DNA sequencing. This class of cfDNA is enriched at gene promoters and binding sites of transcription factors or structural DNA-binding proteins, so that a genome-wide DNA footprint is directly captured from liquid biopsies. In short double-stranded cfDNA from healthy individuals, we find significant enrichment of 203 transcription factor motifs. Additionally, short double-stranded cfDNA signals at specific genomic regions correlate negatively with DNA methylation, positively with H3K4me3 histone modifications and gene transcription. The diagnostic potential of short double-stranded cell-free DNA (cfDNA) in blood plasma has not yet been recognized. When comparing short double-stranded cfDNA from patient samples of pancreatic ductal adenocarcinoma with colorectal carcinoma or septic with postoperative controls, we identify 136 and 241 differentially enriched loci, respectively. Using these differentially enriched loci, the disease types can be clearly distinguished by principal component analysis, demonstrating the diagnostic potential of short double-stranded cfDNA signals as a new class of biomarkers for liquid biopsies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/003bad913ef23b451e8afe9e333721e6d528290e" target='_blank'>
              A novel approach for in vivo DNA footprinting using short double-stranded cell-free DNA from plasma.
              </a>
            </td>
          <td>
            Jan Müller, Christina Hartwig, Mirko Sonntag, Lisa Bitzer, Christopher Adelmann, Y. Vainshtein, Karolina Glanz, S. Decker, Thorsten Brenner, Georg F. Weber, A. von Haeseler, Kai Sohn
          </td>
          <td>2024-09-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2e1918f5013376fd130c9f594868a7f384598c2" target='_blank'>
              HMOX1: A pivotal regulator of prognosis and immune dynamics in ovarian cancer
              </a>
            </td>
          <td>
            Jinfa Huang, Ruiwan Tan
          </td>
          <td>2024-08-30</td>
          <td>BMC Women's Health</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54370344e70d12f9e093a0f96e506dd839c6df0" target='_blank'>
              Deciphering breast cancer dynamics: insights from single-cell and spatial profiling in the multi-omics era
              </a>
            </td>
          <td>
            Xin Xiong, Xin Wang, Cui-Cui Liu, Zhi-Ming Shao, 
          </td>
          <td>2024-09-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Pancreatic adenocarcinoma (PDAC) is the 3rd leading cause of cancer-related deaths in Canada with a 5-year survival rate at ∼12%. A major cause of this poor survival rate is the inefficiency of current treatments. Only 20% of patients are eligible for surgical resection while the other 80% are treated with chemotherapies (nab-paclitaxel, Gemcitabine, FOLFIRINOX) having limited efficacy. This resistance is due to the PDAC tumour microenvironment (TME), including cancer-associated fibroblasts (CAFs). Numerous studies show CAFs and the TME affect tumour growth, chemoresistance and metastasis, but the mechanisms underlying the effects are unknown. Epigenetic mechanisms, including DNA methylation and histone modifications, link environmental factors to altered gene expression. Genes encoding epigenetic mediators are often mutated or over-expressed in PDAC, but a role in resistance has yet to be defined. We hypothesize that epigenetic reprogramming occurs in pancreatic cancer cells in response to secreted factors from CAFs leading to increased therapy resistance. Methods: Most work to date on PDAC resistance has been performed with immortalized cell lines and in two dimensions. Therefore, to test our hypothesis, we established patient-derived organoids (PDO) and CAFs obtained from endoscopic ultrasound (EUS). Clinical data was obtained through the DERIVE (Determination of Response to Therapy in Individual Patients) database. To examine the effects of CAFs on PDOs, we obtained conditioned media (CM) from four, early passage CAFs, grown 48 hours in culture. Organoid media was supplemented 1:1 with CAF-CM and six PDOs grown in this media for up to 14 days. PDO growth and the response to gemcitabine, FOLFIRINOX or radiation was monitored using an Incucyte system. To identify changes in the epigenome and how this reflects changes in gene expression, DNA methylation arrays and ATAC-Seq was performed before and after treatment with CAF-CM and aligned to RNA-seq analysis on RNA isolated at the same time point. Results: Exposure of PDOs to CAF-CM resulted in altered phenotypes and molecular profiles of PDOs in a patient-specific fashion. Increased resistance to chemotherapy (Gemcitabine) and radiotherapy was observed in multiple PDOs. Changes in DNA methylation and chromatin accessibility was aligned with transcriptomic data and identified known and novel pathways linked to chemoresistance. Metabolomic and cytokine analysis on CAFs-CM identified differential enrichment of metabolites linked to epigenetic mediators. We are currently determining if these factors secreted by the CAFs promote chemo and radio-resistance. Conclusions: This study shows the importance in integrating TME components when examining novel therapeutic options. In addition, we show (1) CAFs secrete factors that enhance therapeutic resistance in PDOs and (2) these changes likely involve altered epigenetic reprogramming. Future experiments will identify the epigenetic factors that can be targeted to counteract chemoresistance.
 Citation Format: Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin. Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41115ec236606c659caa4488fce0af64b664490b" target='_blank'>
              Abstract A058: Primary cancer-associated fibroblasts alter tumor organoid chemosensitivity through epigenetic regulation
              </a>
            </td>
          <td>
            Emilie Jaune-Pons, Zachary Klassen, Joana Ribeiro Pinto, Maria Nica, Nadeem Hussain, Michael Sey, Ken Leslie, Matthew Cecchini, Crystal Engelage, Danielle Porplycia, Stephen Welch, Brian Yan, Christopher Pin
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Humans are exposed to a variety of environmental toxicants which have been implicated in contributing to cancer development. A possible explanation for this link is that adaptive epigenetic changes occur in stem cell populations upon toxicant exposure. These cells are particularly vulnerable during developmental stages, leading to abnormal gene expression later in life. Environmental exposures have been associated with the initiation of sarcoma development. This study will focus on Ewing Sarcoma (ES), as this malignancy has a low mutational burden relative to other cancer types and is thought to be linked to agricultural chemicals. We are also focusing on stromal antigen 2 (STAG2), which is found to be frequently mutated in ES in addition to multiple other tumor types. Patients with STAG2-mutated ES have higher rates of metastatic disease and worse outcomes, but the reason for this remains poorly understood. STAG2 plays a role in the regulation of genomic organization, DNA repair, and modulation of replication stress (RS). It has also been shown that STAG2 loss may lead to dysregulation of the cGAS-STING pathway, which is critical for tumor immunosurveillance. We are testing the effects of environmental toxicants in mesenchymal stem cells (MSCs), the putative cell of origin for ES and many other sarcomas. We have successfully used CRISPR/Cas9 to generate STAG2 knockout MSCs. With this model, we are testing the effects of STAG2 loss and toxicant exposure on levels of RS, epigenomic instability, and innate immune suppression in MSCs. Our results indicate that these toxicants alter the cell cycle dynamics of MSCs, and that these alterations change depending on STAG2 status. We have also observed changes in the DNA damage response of STAG2-mutant MSCs when exposed to environmental toxicants. Interestingly, effects were seen even at sub-lethal doses that may reflect common, low-level exposures. We believe that our data will contribute to the understanding of the mechanism by which environmental toxicants facilitate malignant transformation of Ewing sarcoma and other pediatric sarcomas.
 Citation Format: Rachael Kohrn, Joyce Ohm. Investigating the role of environmental toxicant exposures and STAG2 loss in sarcomagenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed252aa2969d787536854f4f2e71a66da32b809" target='_blank'>
              Abstract A029 Investigating the role of environmental toxicant exposures and STAG2 loss in sarcomagenesis
              </a>
            </td>
          <td>
            Rachael Kohrn, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Leukemia is a malignant tumor with high heterogeneity and a complex evolutionary process. It is difficult to resolve the heterogeneity and clonal evolution of leukemia cells by applying traditional bulk sequencing techniques, thus preventing a deep understanding of the mechanisms of leukemia development and the identification of potential therapeutic targets. However, with the development and application of single-cell sequencing technology, it is now possible to investigate the gene expression profile, mutations, and epigenetic features of leukemia at the single-cell level, thus providing a new perspective for leukemia research. In this article, we review the recent applications and advances of single-cell sequencing technology in leukemia research, discuss its potential for enhancing our understanding of the mechanisms of leukemia development, discovering therapeutic targets and personalized treatment, and provide reference guidelines for the significance of this technology in clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059e4fa52cbb0003a82e3f417ae0958def767ace" target='_blank'>
              Application and research progress of single cell sequencing technology in leukemia
              </a>
            </td>
          <td>
            Dan Xie, Bangquan An, Mingyue Yang, Lei Wang, Min Guo, Heng Luo, Shengwen Huang, Fa Sun
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df4be94d1533f2155dbf3532c23eeba54389f11" target='_blank'>
              Cohesin mutations in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Austin Boucher, J. Murray, Sridhar Rao
          </td>
          <td>2024-09-09</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1066c91ed88f85d1bf07f782fd1e659f00c561c5" target='_blank'>
              B4GALT5 a sialylation-related genes associated with patient prognosis and immune microenvironment in ovarian cancer and pan-cancer
              </a>
            </td>
          <td>
            Di Wu, Li-yuan Sun, Xin-yu Chang, Guangmei Zhang
          </td>
          <td>2024-08-29</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="cfDNA consists of degraded DNA fragments released into body fluids. Its genetic and pathological information makes it useful for prenatal testing and early tumor detection. However, the mechanisms behind cfDNA biology are largely unknown. In this study, for the first time, we conducted a GWAS study to explore the genetic basis of cfDNA features, termed cfGWAS, in 28,016 pregnant women. We identified 84 significant loci, including well-known cfDNA-related genes DFFB and DNASE1L3, and numerous novel genes potentially involved in cfDNA biology, including PANX1 and DNASE1L1. The findings were further verified through independent GWAS and experimental validation in knockout mice and cell lines. Subsequent analyses revealed strong causal relationships of hematological indicators on cfDNA features. In summary, we presented the first cfGWAS, revealing the genetic basis of cfDNA biology from genome-wide scale. Novel knowledge uncovered by this study keep the promise to revolutionize liquid biopsy technology and potential new drug targeted for certain disease. Given exist of the millions cfDNA whole-genome-sequencing data generated from clinical testing, the potential of this paradigm is enormous.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5fbbd847bf08ee9e979bef570652eb8223f56f2a" target='_blank'>
              cfGWAS reveal genetic basis of cell-free DNA features
              </a>
            </td>
          <td>
            Huanhuan Zhu, Yan Zhang, Shuang Zeng, Linxuan Li, Rijing Ou, Xinyi Zhang, Yu Lin, Ying Lin, Chuang Xu, Lin Wang, Guodan Zeng, Jingyu Zeng, Lingguo Li, Yongjian Jia, Yu Wang, Fei Luo, Meng Yang, Yuxuan Hu, Xiameizi Li, Han Xiao, Xun Xu, Jian Wang, Aifen Zhou, Haiqiang Zhang, Xin Jin
          </td>
          <td>2024-08-30</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Complete expression loss of S-methyl-5′-thioadenosine phosphorylase (MTAP) is caused by homozygous 9p21 deletion and results in a critical vulnerability of cancer cells towards drugs targeting multiple different pathways. MTAP deficiency is common in ductal adenocarcinoma of the pancreas, but data on its intratumoral heterogeneity - a potential obstacle for targeted therapies - are lacking. To learn more on the role and potential heterogeneity of MTAP deficiency in pancreatic adenocarcinoma, 236 primary tumors were analyzed in a tissue microarray (TMA) format by immunohistochemistry (IHC) and fluorescence in situ hybridization (FISH) for MTAP expression and 9p21 deletions. The cohort included 236 cancers from which 6 samples were taken from as many as possible different tumor containing primary tumor blocks and 3 samples were added from as many as possible lymph node metastases to construct a “heterogeneity TMA”. All available tumor containing blocks (average 4) from 6 selected cancers were also analyzed by IHC and FISH on whole sections. Complete absence of MTAP staining was seen in 350 (38.6%) of 907 interpretable TMA samples. 284 (98.6%) of 288 samples with a complete MTAP expression loss and FISH data had a homozygous 9p21 deletion while there were only two tumor spots with homozygous deletions within 557 samples with retained MTAP expression (99.6% concordance). On a patient level, an MTAP deficiency was found in 64 (41.0%) of 156 tumors for which at least 3 different samples were interpretable on the TMAs. 62 (96.9%) of these patients showed an MTAP loss in all interpretable samples (homogeneous MTAP deficiency). The TMA data were confirmed on whole sections in 5 of 6 patients. In the discrepant case a focal MTAP loss was identified in a tumor considered MTAP wild type based on the TMA results. It is concluded that MTAP expression loss is frequent and highly homogeneous in ductal adenocarcinomas of the pancreas. Given the high concordance rate between homozygous deletion and MTAP expression loss homozygous deletion may represent the only mechanism for MTAP deficiency in pancreatic cancer. MTAP IHC is an excellent tool for identification of MTPA deficient cases. MTAP IHC may also have considerable diagnostic value as even low-grade cancer glands can be easily identified in small biopsies from MTAP deficient cases. Considering also their aggressive clinical behavior, pancreatic adenocarcinomas may represent an ideal cancer type for studying new drugs targeting MTAP deficient cancer cells in clinical trials.
 Citation Format: Natalia Gorbokon, Katharina Teljuk, Frank Jacobsen, Till Krech, Till Clauditz, Stefan Steurer, Ronald Simon, Guido Sauter, Sarah Minner, Lukas Bubendorf, Thilo Hackert, Nina Schraps, Faik G Uzunoglu, Martina Kluth. MTAP deficiency is frequent and mostly homogeneous in ductal adenocarcinomas of the pancreas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A028.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ece68516216c0dd87294424662694645b72a9ed" target='_blank'>
              Abstract A028: MTAP deficiency is frequent and mostly homogeneous in ductal adenocarcinomas of the pancreas
              </a>
            </td>
          <td>
            N. Gorbokon, Katharina Teljuk, Frank Jacobsen, T. Krech, T. Clauditz, S. Steurer, Ronald Simon, G. Sauter, S. Minner, Lukas Bubendorf, T. Hackert, Nina Schraps, Faik G. Uzunoglu, M. Kluth
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="
 Like many pediatric cancers, Ewing sarcoma (EwS) is a genomically quiet disease. Driven by an oncogenic fusion protein, usually EWSR1-FLI1, the epigenome of this genetically homogenous cancer is a driver of clinical outcomes, where heightened epigenetic heterogeneity within tumors is associated with metastatic disease. While prognosis for localized disease is moderate, outcomes are dismal for patients who present with advanced, relapsed, or refractory disease and there is a lack of targeted therapies in this setting. We have previously described a targeted combination therapy harnessing DDK and WEE1 inhibition exploits elevated endogenous replication stress in EwS to force cells into mitotic catastrophe in vitro. While our experiments evaluating the efficacy of this combination in vivo have been extremely encouraging, the data have shown that individual mice receiving the same treatment have variable patterns of outgrowth after treatment cessation, with some mice demonstrating moderate increases in tumor volume and some mice demonstrating little to no tumor outgrowth. This differential response to therapy is most likely to occur through epigenetic dysregulation. Thus, we aim to interrogate intertumoral epigenetic heterogeneity and examine epigenetic profiles associated with treatment response using samples from in vivo studies. Samples were analyzed from a CDX model using TC32 cells injected subcutaneously. Reduced representation bisulfite sequencing was performed on a control arm (n = 5) and the following three treatment arms, where the DDK inhibitor is TAK-931 (simurosertib) and the WEE1 inhibitor is MK1775 (adavosertib): 1) Irinotecan + Temozolomide (n=9), 2) Irinotecan (full dose) + DDKi + WEE1i (n=7), 3) Irinotecan (half dose) + DDKi + WEE1i (n=9), for a total of 30 samples. Our analysis of this data evaluates epigenetic changes between treatment conditions and increases our understanding of epigenetic heterogeneity in EwS.
 Citation Format: Emily Isenhart, Ajay Gupta, Joyce Ohm. Intertumoral epigenetic heterogeneity in response to a novel therapy in Ewing sarcoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A056.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/366bd5a5e10d37a86e17654dd6f774d1f8bf9952" target='_blank'>
              Abstract A056 Intertumoral epigenetic heterogeneity in response to a novel therapy in Ewing sarcoma
              </a>
            </td>
          <td>
            E. Isenhart, Ajay Gupta, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The tight control of the mitotic phase of differentiation is crucial to prevent tumourigenesis while securing tissue homeostasis. In the Drosophila female germline, differentiation involves precisely four mitotic divisions, and accumulating evidence suggests that bag-of-marbles (bam), the initiator of differentiation, is also involved in controlling the number of divisions. To test this hypothesis, we depleted Bam from differentiating cells and found a reduced number of mitotic divisions. We examined the regulation of Bam using RNA imaging methods and found that the bam 39 UTR conveys instability to the transcript in the 8-cell cyst and early 16-cell cyst. We show that the RNA binding protein, Rbp9, is responsible for timing bam mRNA decay. Rbp9 itself is part of a sequential cascade of RNA binding proteins activated downstream of Bam, and we show that it is regulated through a change in transcription start site, driven by Rbfox1. Altogether, we propose a model in which Bam expression at the dawn of differentiation initiates a series of events that eventually terminates the Bam expression domain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2ac40f2eb322caeece920877298412a69c4a5a9" target='_blank'>
              Destabilisation of bam transcripts terminates the mitotic phase of Drosophila female germline differentiation
              </a>
            </td>
          <td>
            Tamsin J. Samuels, Elizabeth J Torley, Emily L Naden, Phoebe E Blair, Frankjel A Hernandez Frometa, Felipe Karam Teixeira
          </td>
          <td>2024-09-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Pancreatic ductal adenocarcinoma (PDAC) remains a deadly disease with few effective treatments. Even with the advent of mutationally-directed therapies targeting the KRAS oncogene, PDAC transcriptional plasticity remains a major mechanism of therapeutic resistance. As a result, there is a desperate need to develop additional strategies to treat this deadly disease. In addition to the canonical genomic alterations in KRAS, TP53, CDKN2A, and SMAD4, PDAC tumors exhibit diverse transcriptional programs, or "cell states," with prognostic implications: PDAC tumors in the “classical state” are more responsive to upfront chemotherapy and have better outcomes, while those in the “basal state” are more aggressive with poorer prognoses. Currently, PDAC cell states are not used to direct therapy. Cell state is an integrative property shaped by both cell-intrinsic (e.g., mutations, epigenetics) and cell-extrinsic (e.g., tumor microenvironment) factors. In our prior work, single-cell analyses of clinical PDAC samples demonstrated notable cell state heterogeneity and plasticity. In addition, longitudinal monitoring of patient-derived organoids showed that cells can shift from classical to basal or coexpressor states as the environment changes, and that these cell states can drastically influence drug response. However, it remains unknown which signaling and transcriptional pathways induce basal or classical states, and whether these gene regulatory networks can be targeted therapeutically. Thus, there is a pressing need to devise scalable methods to determine essential drivers for prognostically relevant cancer cell states.Here, we aim to systematically and unbiasedly identify transcription factors (TFs) that drive basal and classical cell states in PDAC. We have successfully established a high-throughput screening pipeline to 1) over-express all human TF isoforms in a pooled format within a cohort of PDAC models; 2) sort cells using antibodies against state-specific cell surface markers; and 3) perform single-cell RNA sequencing on sorted cells to nominate individual TFs or combinations that induce specific cell state transitions. Preliminary results identify both previously reported (e.g., GATA4) and new TFs that can induce the classical state. By overexpressing TFs individually or in combination, we are able to engineer PDAC cell line models that more faithfully reflect the transcriptional phenotypes observed in patient tumors. Ultimately, we aim to use this approach to construct high-fidelity isogenic but state-variant PDAC models for use both in therapeutic screening and as a substrate for investigating mechanisms governing cancer cell state plasticity.
 Citation Format: Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E Katan, Julia Joung, Alex K Shalek, Peter S Winter, Srivatsan Raghavan. Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C066.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c88d4d8016aad9bbc47795f53500dd4b1940096b" target='_blank'>
              Abstract C066: Identifying Transcriptional Drivers of Plasticity in Pancreatic Cancer
              </a>
            </td>
          <td>
            Yuzhou Evelyn Tong, Aswanth Mahalingam, Walaa E. Katan, Julia Joung, Alex K. Shalek, Peter S. Winter, Srivatsan Raghavan
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="BACKGROUND
Cellular angiofibroma, a rare benign mesenchymal neoplasm, is classified within the 13q/RB1 family of tumors due to morphological, immunohistochemical, and genetic similarities with spindle cell lipoma. Here, genetic data reveal pathogenetic heterogeneity in cellular angiofibroma.


METHODS
Three cellular angiofibromas were studied using G-banding/Karyotyping, array comparative genomic hybridization, RNA sequencing, and direct cycling sequencing.


RESULTS
The first tumor carried a del(13)(q12) together with heterozygous loss and minimal expression of the RB1 gene. Tumors two and three displayed chromosome 8 abnormalities associated with chimeras of the pleomorphic adenoma gene 1 (PLAG1). In tumor 2, the cathepsin B (CTSB) fused to PLAG1 (CTSB::PLAG1) while in tumor 3, the mir-99a-let-7c cluster host gene (MIR99AHG) fused to PLAG1 (MIR99AHG::PLAG1), both leading to elevated expression of PLAG1 and insulin growth factor 2.


CONCLUSION
This study uncovers two genetic pathways contributing to the pathogenetic heterogeneity within cellular angiofibromas. The first aligns with the 13q/RB1 family of tumors and the second involves PLAG1-chimeras. These findings highlight the diverse genetic landscape of cellular angiofibromas, providing insights into potential diagnostic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72129daa715b46d571770497bea5325bf92afaeb" target='_blank'>
              Genetic Heterogeneity in Cellular Angiofibromas.
              </a>
            </td>
          <td>
            Ioannis Panagopoulos, K. Andersen, I. Lobmaier, M. Lund-Iversen
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) is the most prevalent malignant tumors of the digestive system globally, with a particularly high incidence recorded in China due to the prevalence of chronic hepatitis virus infections. Recent developments in single-cell RNA sequencing (scRNA-seq) technology have provided new perspectives and approaches for cancer research, particularly showing great potential in understanding the heterogeneity of HCC. scRNA-seq technology, through detailed gene expression analysis at the single-cell level, reveals the cellular heterogeneity of hepatocellular carcinoma, identify key drivers of tumor progression, and elucidates the complex features of the tumor microenvironment. Such insights are pivotal for decoding the underlying mechanisms of hepatocellular carcinoma, thereby guiding the development of precise therapeutic strategies and personalized treatments. Furthermore, identifying key drivers of tumor progression and analyzing the gene expression characteristics of the surrounding microenvironment at single-cell resolution is expected to provide clues for developing new therapeutic strategies. Therefore, this article aims to provide a systematic overview of the fundamental principles of scRNA-seq, review its progress in HCC research, and explore the challenges and future directions in this field to offer researchers a comprehensive perspective.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4b441257e4cdb8264be1952074191341186a913" target='_blank'>
              Research on single-cell transcriptomics in hepatocellular carcinoma
              </a>
            </td>
          <td>
            Nauman Shabbir
          </td>
          <td>2024-08-07</td>
          <td>Asia-Pacific Journal of Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Simple Summary It is known that both B cells and myeloblasts from acute myeloid leukemia (AML) express immunoglobulin (Ig). However, the difference between Ig from myeloblasts and B cells has not been explored; therefore, the function and significance of AML-Ig are not well known. Here, we performed 5′ RACE-related PCR coupled with PacBio sequencing to analyze the Ig repertoire of Chinese AML patients. Myeloblasts expressed all five classes of IgH, especially Igγ, with a high expression frequency. Compared with B-Ig in the same AML patient, AML-Ig showed different biased V(D)J usages and mutation patterns. More importantly, mutations of AML-Ig frequently occurred at the sites of post-translational modification. In summary, our results showed that AML-Ig had different sequence characteristics than those of B-Ig and that the unique V(D)J usages of AML-Ig may serve as a novel biomarker for personalized minimal/measurable residual disease monitoring and therapeutic targets in AML patients. Abstract It is common knowledge that immunoglobulin (Ig) is produced by B lymphocytes and mainly functions as an antibody. However, it has been shown recently that myeloblasts from acute myeloid leukemia (AML) could also express Ig and that AML-Ig played a role in leukemogenesis and AML progression. The difference between Ig from myeloblasts and B cells has not been explored. Studying the characteristics of the Ig repertoire in myeloblasts and B cells will be helpful to understand the function and significance of AML-Ig. We performed 5′ RACE-related PCR coupled with PacBio sequencing to analyze the Ig repertoire in myeloblasts and B cells from Chinese AML patients. Myeloblasts expressed all five classes of IgH, especially Igγ, with a high expression frequency. Compared with B-Ig in the same patient, AML-Ig showed different biased V(D)J usages and mutation patterns. In addition, the CDR3 length distribution of AML-Ig was significantly different from those of B-Ig. More importantly, mutations of AML-IgH, especially Igμ, Igα, and Igδ, were different from that of B-IgH in each AML patient, and the mutations frequently occurred at the sites of post-translational modification. AML-Ig has distinct characteristics of variable regions and mutations, which may have implications for disease monitoring and personalized therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2b72edfacf57bf0df201db104f6a0b0c301157e" target='_blank'>
              A Comparative Analysis of the Immunoglobulin Repertoire in Leukemia Cells and B Cells in Chinese Acute Myeloid Leukemia by High-Throughput Sequencing
              </a>
            </td>
          <td>
            Huige Yan, Lina Wu, Pingzhang Wang, Miaoran Xia, Zhan Shi, Xinmei Huang, Sha Yin, Qian Jiang, C. C. Yin, Xiangyu Zhao, Xiaoyan Qiu
          </td>
          <td>2024-08-01</td>
          <td>Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b33c17b9f8ee0a1feff2c09b2ff7e7780f80736" target='_blank'>
              Unlocking the Complexity: Exploration of Acute Lymphoblastic Leukemia at the Single Cell Level.
              </a>
            </td>
          <td>
            Margo Aertgeerts, Sarah Meyers, S. Demeyer, Heidi Segers, Jan Cools
          </td>
          <td>2024-08-27</td>
          <td>Molecular diagnosis & therapy</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="
 Genetic or epigenetic variations in regulatory enhancer elements increase susceptibility to a range of pathologies, including pancreas cancer. Pinpointing genes affecting pancreas cancer risk holds promise for early detection, prevention, and effective therapies. Despite recent advances, linking enhancer elements to target genes and predicting the transcriptional outcome of enhancer dysfunction remain significant challenges. Using 3D chromatin assays, we generated an extensive enhancer interaction dataset for the human pancreas, spanning more than 20 donors and five major cell types, including both the exocrine and endocrine compartments. We employed a network approach to parse chromatin interactions into enhancer-promoter tree models, facilitating quantitative, genome-wide analysis of enhancer connectivity. Using the tree models, we developed a machine learning algorithm capable of estimating the impact of enhancer perturbations on cell-type specific gene expression in the human pancreas. Complementing this computational approach, we streamlined an experimental platform using CRISPR-interference and RNA-FISH coupled with high-throughput imaging to quantitatively measure the effect size of enhancer perturbations in primary human pancreas cells. Our enhancer tree models enabled functional annotation of genetic variants associated with pancreas diseases, revealing novel target genes in specific cell types. For pancreas cancer (PDAC), our data suggests a stronger association of disease susceptibility variants with acinar cells, even though ductal cells historically have been the focus of PDAC. This discovery linked three potentially functional variants associated with PDAC risk to acinar cell enhancers, revealing novel target genes to further investigation. By integrating genomics with mechanistic findings into enhancer function, our work offers a framework for understanding genetic basis of pancreas cancer disease risk.
 Citation Format: Li Wang, Songjoon Baek, Gauri Prasad, Thucnhi Truongvo, Jason Hoskins, Laufey Amundadottir, Efsun Arda. A novel approach to understanding pancreas cancer risk through enhancer- promoter interactions [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C058.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f97224f03d537588713402ad00fc72aae371ba05" target='_blank'>
              Abstract C058: A novel approach to understanding pancreas cancer risk through enhancer- promoter interactions
              </a>
            </td>
          <td>
            Li Wang, Songjoon Baek, Gauri Prasad, Thucnhi Truongvo, J. Hoskins, L. Amundadottir, Efsun Arda
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="DNA methylation is well-established as a major epigenetic mechanism that can control gene expression and is involved in both normal development and disease. Analysis of high-throughput-sequencing-based DNA methylation data is a step toward understanding the relationship between disease and phenotype. Analysis of CpG methylation at single-base resolution is routinely done by bisulfite sequencing, in which methylated Cs remain as C while unmethylated Cs are converted to U, subsequently seen as T nucleotides. Sequence reads are aligned to the reference genome using mapping tools that accept the C-T ambiguity. Then, various statistical packages are used to identify differences in methylation between (groups of) samples. We have previously developed the Differential Methylation Analysis Pipeline (DMAP) as an efficient, fast, and flexible tool for this work, both for whole-genome bisulfite sequencing (WGBS) and reduced-representation bisulfite sequencing (RRBS). The protocol described here includes a series of scripts that simplify the use of DMAP tools and that can accommodate the wider range of input formats now in use to perform analysis of whole-genome-scale DNA methylation sequencing data in various biological and clinical contexts. © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol: DMAP2 workflow for whole-genome bisulfite sequencing (WGBS) and reduced-representation bisulfite sequencing (RRBS).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52b13fb6f53fc80d13d019e959851a23b4cd1b21" target='_blank'>
              DMAP2: A Pipeline for Analysis of Whole-Genome-Scale DNA Methylation Sequencing Data.
              </a>
            </td>
          <td>
            P. Stockwell, Euan J. Rodger, Gregory Gimenez, I. Morison, Aniruddha Chatterjee
          </td>
          <td>2024-09-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="
 Pediatric cancers are frequently driven by genomic alterations that result in aberrant transcription factor activity. While direct targeting of transcription factors is difficult, one method to circumvent this problem is to target co-factors and chromatin complexes that are necessary for their oncogenic function. To evaluate protein complex dependencies enriched in MYCN-amplified neuroblastoma, a disease of dysregulated development driven by aberrant expression of an oncogenic expression program, we curated a list of protein complexes using the CORUM database and mined the Dependency Map using single sample gene set enrichment analysis. This analysis identified the KAT module of the transcriptional coactivator Spt-Ada-Gcn5-acetyltransferase (SAGA) complex as a selective dependency in MYCN-amplified neuroblastoma. Loss of SAGA complex activity in neuroblastoma through both genetic knockout and induced protein degradation of the lysine acetyltransferase (KAT) module subunit TADA2B resulted in a global reduction of histone lysine acetylation and impaired neuroblastoma cell growth. Genetic knockout of TADA2B did not impair the growth of pediatric sarcoma cell lines, suggesting that SAGA loss-of-function is not broadly deleterious. Furthermore, SAGA loss-of-function reduced MYCN binding to chromatin, resulting in suppression of the MYCN-driven gene expression program and impaired cell cycle progression. Importantly, the SAGA complex is pharmacologically targetable in vitro and in vivo with GSK-699, a PROTAC that induces proteolysis of both KAT2A and KAT2B paralogs. As observed with genetic knockout studies, loss of SAGA activity through KAT2A/B degradation reduced the MYCN-driven transcriptional signature and impaired cell cycle progression. Our findings expand our understanding of the chromatin complexes that maintain the oncogenic transcriptional state in MYCN-amplified neuroblastoma and suggest therapeutic potential for inhibitors of SAGA KAT activity in MYCN-amplified neuroblastoma.
 Citation Format: Nathaniel W. Mabe, Clare F. Malone, Alexandra B. Forman, Gabriela Alexe, Kathleen L. Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar, Bin Liu, Sharon YR Dent, Kimberly Stegmaier. Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A062.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be30c933e33904854e0410865fe3ee874fab512e" target='_blank'>
              Abstract A062 Therapeutic targeting of the SAGA KAT module impairs MYCN-amplified neuroblastoma growth through reduction of the MYCN oncogenic gene expression program
              </a>
            </td>
          <td>
            Nathaniel W. Mabe, Clare F Malone, Alexandra B Forman, G. Alexe, Kathleen Engel, Ying-Jiun C. Chen, Melinda Soeung, Silvi Salhotra, Allen Basanthakumar, Bin Liu, S. Y. Dent, K. Stegmaier
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba0c6be6efa212fbb68132ac9fcf2e7d1b9412f9" target='_blank'>
              Sumo-regulatory SENP2 controls the homeostatic squamous mitosis-differentiation checkpoint
              </a>
            </td>
          <td>
            Jesús Galán-Vidal, Lorena García-Gaipo, R. Molinuevo, Samantha Dias, Alex Tsoi, Javier Gómez-Román, James T. Elder, H. Hochegger, Alberto Gandarillas
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/823edc7b7f2e48466de67d70b29a3e25297b1b88" target='_blank'>
              Cooperative genomic lesions in HRAS-mutant cancers predict resistance to farnesyltransferase inhibitors.
              </a>
            </td>
          <td>
            Arad Nigam, Gnana P. Krishnamoorthy, Walid K. Chatila, Katherine Berman, Mahesh Saqcena, H. Walch, Mandakini Venkatramani, Alan L. Ho, N. Schultz, James A. Fagin, B. Untch
          </td>
          <td>2024-08-16</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Bacterial DNA methylation is involved in diverse cellular functions ranging from modulation of gene expression, DNA repair, and restriction-modification systems for defense against viruses and other foreign DNA. Methylome analysis determines sites within a bacterial chromosome that are methylated, revealing motifs that may be targeted by native restriction enzymes. Identification of these motifs is therefore critical to making an organism genetically tractable, where mimicking the methylome patterns in Escherichia coli allows plasmid DNA to be protected from restriction in the target organism and can therefore drastically enhance transformation efficiency. Oxford Nanopore Technologies (ONT) sequencing can detect methylated bases during sequencing, but software is needed to identify the corresponding methylated motifs within the data. Here, we develop MIJAMP (MIJAMP Is Just A MethylBED Parser), a software package that was developed to discover methylated motifs from the output of ONT’s Modkit or other data in the methylBED format. MIJAMP employs a human-driven refinement strategy that empirically validates all motifs against genome-wide methylation data, thus eliminating false, under-, or overexplained motifs. MIJAMP can also report methylation data on a specific, user-defined motif. Using MIJAMP, we determined the methylated motifs both in a control strain (wild type E. coli) and in Picosynecococcus sp. strain PCC7002, laying the foundation for improved transformation in this organism. MIJAMP is available at https://code.ornl.gov/5g6/mijamp/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/560305afe7d74c24c06221b94759147f1622ed98" target='_blank'>
              Discovering methylated DNA motifs in bacterial nanopore sequencing data with MIJAMP
              </a>
            </td>
          <td>
            Alyssa K. Tidwell, Evelyn Faust, Carrie A. Eckert, A. Guss, William G. Alexander
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 Background: Transposable elements (TEs) are dynamic repetitive regions which generate mutations and structural variants. TP53 plays a crucial role in suppressing TE movement to maintain genomic stability. The relationship between TP53 and TEs has been extensively studied in tumours, but not the germline. Individuals with germline TP53 pathogenic variants have Li-Fraumeni Syndrome (LFS), a cancer predisposition syndrome with a high lifetime risk of cancer in various tissues. This study aims to characterize the TE landscape in individuals with Li-Fraumeni Syndrome and determine how this contributes to their increased cancer risk. Methods: MELT and xTea were used to identify TEs in children with (n=48) or without (n=198) a germline TP53 variant. TE calls were merged with SURVIVOR and variants were annotated with AnnotSV. To assess the influence of TP53 on TE location, we quantified TEs across genomic windows and identified the top 100 significantly different regions with the Mann-Whitney U test, adjusting for multiple comparisons. We used these regions to develop a gradient-boosted tree model with 5-fold cross-validation to predict TP53 status. Results: Individuals with germline TP53 variants harboured significantly fewer ALU and LINE1 elements in their germline genome compared to the control dataset (p<0.001). The phenomenon was consistent across chromosomes and significant in chromosomes 5 and 11 (FDR<0.05). A gradient-boosted tree model was able to differentiate patients with and without a germline TP53 variant with an AUPRC of 0.77 on an unseen test set. Conclusion: Germline TP53 variants may effect the frequency and location of germline TE insertions, which may influence nearby genomic variations and lead to cancer development. Analyzing TEs in individuals with LFS will enhance our understanding of accelerated cancer development in these patients, informing future research for diagnostic and therapeutic approaches.
 Citation Format: Brianne Laverty, Shilpa Yadahalli, Vallijah Subasri, David Malkin. The role of TP53 on transposable elements in pediatric cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A010.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4995c29bd2050c705f9a5b8da916aacb0aa2601c" target='_blank'>
              Abstract A010 The role of TP53 on transposable elements in pediatric cancer
              </a>
            </td>
          <td>
            B. Laverty, Shilpa Yadahalli, Vallijah Subasri, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Li-Fraumeni Syndrome (LFS) is a genetic predisposition disorder associated with pathogenic germline Tp53 mutations. A subset of LFS-associated neoplasms occurs at an increased risk within the first 5 years of life, with the most common being soft tissue sarcomas (STS). Histological characterization of STS is one of the primary approaches to guiding treatment. However, the heterogeneity of STS has necessitated a further understanding of the molecular etiology of these tumors. Examining sarcomas using a single-cell transcriptomic approach has shown unique subsets of cells with dysregulations in cell fate and differentiation. Within LFS-associated tumors, including LFS-STS a major event in tumorigenesis is the copy-neutral loss of heterozygosity (LOH). LOH was determined to occur much earlier than cancer onset in LFS, suggesting a dormancy period from the LOH until tumorigenesis; cancer-defining events in LFS are likely occurring prenatally. During embryogenesis, Tp53 is involved in differentiation and pluripotency regulation, as well as cell cycle control and apoptosis. Here we hypothesize that the emergence of STS in LFS mice is associated with distinct disturbances in the cell fate and differentiation pathways. Specifically, there will be deficits in cell fate determination and the organization of cells across multiple cell types in LFS-Sarcomas. Single-cell RNAseq was performed on spontaneous tumors and healthy matched muscle tissue from male and female C57BL/6J Trp53R172H/WT mice. Cell types were determined based on differential gene expression between clusters defined by principal component analysis. Despite tumor heterogeneity between mice, there were consistent disruptions of developmental pathways among tumor cells. Moreover, when comparing matched cell types across tumor and healthy muscle tissue we observed dysregulation of pathways promoting tumorigenesis, and cell fate determination in cells originating from tumor tissue. For the first time to our knowledge, this study examines the single-cell transcriptome of LFS-STS. Current literature has speculated on the presence of an early precancer state of LFS-associated tumors. This study will provide evidence to determine whether this precancer state can be determined in tumor samples, and identify elements that constitute this state. Moreover, we will determine changes in supporting non-tumor cells and whether they also exhibit dysregulations in cell state and differentiation. Together these findings will hopefully contribute to improvements in molecular subtyping across STS, and particularly within LFS individuals.
 Citation Format: Ashby Kissoondoyal, David Malkin. Single cell transcriptomic signature of Li-Fraumeni Syndrome soft tissue sarcomas [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc62fb2bdb474c15b956e6aac5478623a2fc2613" target='_blank'>
              Abstract A009 Single cell transcriptomic signature of Li-Fraumeni Syndrome soft tissue sarcomas
              </a>
            </td>
          <td>
            Ashby Kissoondoyal, D. Malkin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Female cells randomly inactivate one X-chromosome, resulting in cellular mosaicism. Determining the bias in X-chromosomal inactivation (XCI) at single cell level may be relevant for understanding diseases prevalent in females. Here, we introduce a computational method that determines XCI profiles at single-cell level using solely sc/snRNA-Seq data. XCI analysis of skin cells from hybrid mice validates our approach and reveals biased inactivation of X-chromosomes among cell types. In human lung and brain cells, XCI status can be determined in 33.8% and 23.6% of cells. Among the patients, cells with opposite inactivation patterns differently express members of specific gene families and pathways. Alzheimer’s disease patients show reversal of XCI in cortical microglia and regional increase in biallelic expression denoting epigenetic erosion. We provide a robust utility to explore the degree and impact of XCI in single cell expression data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8505b493ed60aed63b31c5d60d74abdddba75957" target='_blank'>
              Single-cell X-chromosome inactivation analysis links biased chimerism to differential gene expression and epigenetic erosion
              </a>
            </td>
          <td>
            Rob H Henning, Thomas M. Rust, Kasper Dijksterhuis, B. Eggen, Victor Guryev
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Aligning DNA sequences retrieved from fossils or other paleontological artifacts, referred to as ancient DNA, is particularly challenging due to the short sequence length and chemical damage which creates a specific pattern of substitution (C→T and G→A) in addition to the heightened divergence between the sample and the reference genome thus exacerbating reference bias. This bias can be mitigated by aligning to pangenome graphs to incorporate documented organismic variation, but this approach still suffers from substitution patterns due to chemical damage. We introduce a novel methodology introducing the RYmer index, a variant of the commonly-used minimizer index which represents purines (A,G) and pyrimidines (C,T) as R and Y respectively. This creates an indexing scheme robust to the aforementioned chemical damage. We implemented SAFARI, an ancient DNA damage-aware version of the pangenome aligner vg giraffe which uses RYmers to rescue alignments containing deaminated seeds. We show that our approach produces more correct alignments from ancient DNA sequences than current approaches while maintaining a tolerable rate of spurious alignments. In addition, we demonstrate that our algorithm improves the estimate of the rate of ancient DNA damage, especially for highly damaged samples. Crucially, we show that this improved alignment can directly translate into better insights gained from the data by showcasing its integration with a number of extant pangenome tools.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f60983de5dcd109f2475f2c8134a67852f35929" target='_blank'>
              SAFARI: Pangenome Alignment of Ancient DNA Using Purine/Pyrimidine Encodings
              </a>
            </td>
          <td>
            J. Rubin, J. van Waaij, Louis Kraft, Jouni Sirén, Peter Wad Sackett, Gabriel Renaud
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [18],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>