<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-05 13:10:18 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>20</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>50</td>
          <td>130</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>98</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5988602f52499713b4d1d42d555664a971cb3e9" target='_blank'>
              3D genomic analysis reveals novel enhancer-hijacking caused by complex structural alterations that drive oncogene overexpression
              </a>
            </td>
          <td>
            Katelyn L. Mortenson, Courtney Dawes, Emily R. Wilson, Nathan E. Patchen, Hailey E Johnson, Jason Gertz, Swneke D. Bailey, Yang Liu, K. Varley, Xiaoyang Zhang
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Somatic structural variations (SVs) in cancer can shuffle DNA content in the genome, relocate regulatory elements, and alter genome organization. Enhancer hijacking occurs when SVs relocate distal enhancers to activate proto-oncogenes. However, most enhancer hijacking studies have only focused on protein-coding genes. Here, we develop a computational algorithm 'HYENA' to identify candidate oncogenes (both protein-coding and non-coding) activated by enhancer hijacking based on tumor whole-genome and transcriptome sequencing data. HYENA detects genes whose elevated expression is associated with somatic SVs by using a rank-based regression model. We systematically analyze 1146 tumors across 25 types of adult tumors and identify a total of 108 candidate oncogenes including many non-coding genes. A long non-coding RNA TOB1-AS1 is activated by various types of SVs in 10% of pancreatic cancers through altered 3-dimensional genome structure. We find that high expression of TOB1-AS1 can promote cell invasion and metastasis. Our study highlights the contribution of genetic alterations in non-coding regions to tumorigenesis and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fb0cf8831bea2891dbca4714836cde381f2ca0" target='_blank'>
              HYENA detects oncogenes activated by distal enhancers in cancer.
              </a>
            </td>
          <td>
            Anqi Yu, Ali E. Yesilkanal, Ashish Thakur, Fan Wang, Yang Yang, William Phillips, Xiaoyang Wu, Alexander Muir, Xin He, Francois Spitz, Lixing Yang
          </td>
          <td>2024-07-25</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA), an extrachromosomal circular structured DNA, is extensively found in eukaryotes. Exploring eccDNA at the single-cell level contributes to understanding heterogeneity, evolution, development, and specific functions within cells. Nevertheless, the high-throughput identification methods for single-cell eccDNA are complex, and currently mature and widely applicable technologies are lacking. Those factors have led to a scarcity of resources for studying eccDNA at the single-cell level. Therefore, using available single-cell whole-genome sequencing (WGS) data, we have constructed a comprehensive eccDNA database named scEccDNAdb (https://lcbb.swjtu.edu.cn/scEccDNAdb/). Presently, scEccDNAdb comprises 3,195,464 single-cell eccDNA entries from both disease/health human and mouse samples, which provides comprehensive annotations including oncogenes, typical enhancers, super-enhancers, CTCF binding sites, SNPs, chromatin accessibility, eQTLs, transcription factor binding sites, motifs, and SVs. Additionally, it provides nine online analysis and visualization tools, facilitating the generation of publication-quality figures for eccDNA analysis through the upload of customized files. Overall, scEccDNAdb represents the first comprehensive database known to us for exploring and analyzing single-cell eccDNA data in diverse cell types, tissues, and species. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322c06c5550a609619dd96de2cba0b3494554b2a" target='_blank'>
              scEccDNAdb: an integrated single-cell eccDNA resource for human and mouse
              </a>
            </td>
          <td>
            Wenqing Wang, Xinyu Zhao, Tianyu Ma, Tengwei Zhong, Junnuo Zheng, Yi Yang, Yuanbiao Guo, Zhiyun Guo
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transcriptional enhancers can regulate individual or multiple genes through long-range three-dimensional (3D) genome interactions, and these interactions are commonly altered in cancer. Yet, the functional relationship between changes in 3D interactions associated with regulatory regions and differential gene expression appears context-dependent. In this study, we used HiChiP to capture changes in 3D genome interactions between active regulatory regions of endometrial cancer cells in response to estrogen treatment and uncovered significant differential long-range interactions that are strongly enriched for estrogen receptor α (ER) bound sites (ERBS). The ERBS anchoring differential loops with either a gene’s promoter or distal regions were correlated with larger transcriptional responses to estrogen compared to ERBS not involved in differential interactions. To functionally test this observation, CRISPR-based Enhancer-i was used to deactivate specific ERBS, which revealed a wide range of effects on the transcriptional response to estrogen. However, these effects are only subtly and not significantly stronger for ERBS in differential loops. In addition, we observed an enrichment of 3D interactions between the promoters of estrogen up-regulated genes and found that looped promoters can work together cooperatively. Overall, our work suggests that changes in 3D genome structure upon estrogen treatment identify some functionally important regulatory regions; however, these changes aren’t required for a transcriptional response to E2 in endometrial cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d0d32b1b8568f5b7b8778716b0bae7e2d53785" target='_blank'>
              Estrogen-induced chromatin looping changes identify a subset of functional regulatory elements
              </a>
            </td>
          <td>
            Hosiana Abewe, Alexandra Richey, Jeffery M. Vahrenkamp, Matthew Ginley-Hidinger, Craig M Rush, Noel Kitchen, Xiaoyang Zhang, Jason Gertz
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Recent advances in high-resolution mapping of spatial interactions among regulatory elements support the existence of complex topological assemblies of enhancers and promoters known as enhancer-promoter hubs or cliques. Yet, organization principles of these multi-interacting enhancer-promoter hubs and their potential role in regulating gene expression in cancer remains unclear. Here, we systematically identified enhancer-promoter hubs in breast cancer, lymphoma, and leukemia. We found that highly interacting enhancer-promoter hubs form at key oncogenes and lineage-associated transcription factors potentially promoting oncogenesis of these diverse cancer types. Genomic and optical mapping of interactions among enhancer and promoter elements further showed that topological alterations in hubs coincide with transcriptional changes underlying acquired resistance to targeted therapy in T cell leukemia and B cell lymphoma. Together, our findings suggest that enhancer-promoter hubs are dynamic and heterogeneous topological assemblies with the potential to control gene expression circuits promoting oncogenesis and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9831a80afed1f71511cdd9f1e5ce45eefd2262c2" target='_blank'>
              Enhancer-promoter hubs organize transcriptional networks promoting oncogenesis and drug resistance
              </a>
            </td>
          <td>
            Brent S. Perlman, Noah G Burget, Yeqiao Zhou, Gregory W. Schwartz, Jelena Petrovic, Z. Modrusan, R. Faryabi
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Enhancers are key drivers of gene regulation thought to act via 3D physical interactions with the promoters of their target genes. However, genome-wide depletions of architectural proteins such as cohesin result in only limited changes in gene expression, despite a loss of contact domains and loops. Consequently, the role of cohesin and 3D contacts in enhancer function remains debated. Here, we developed CRISPRi of regulatory elements upon degron operation (CRUDO), a novel approach to measure how changes in contact frequency impact enhancer effects on target genes by perturbing enhancers with CRISPRi and measuring gene expression in the presence or absence of cohesin. We systematically perturbed all 1,039 candidate enhancers near five cohesin-dependent genes and identified 34 enhancer-gene regulatory interactions. Of 26 regulatory interactions with sufficient statistical power to evaluate cohesin dependence, 18 show cohesin-dependent effects. A decrease in enhancer-promoter contact frequency upon removal of cohesin is frequently accompanied by a decrease in the regulatory effect of the enhancer on gene expression, consistent with a contact-based model for enhancer function. However, changes in contact frequency and regulatory effects on gene expression vary as a function of distance, with distal enhancers (e.g., >50Kb) experiencing much larger changes than proximal ones (e.g., <50Kb). Because most enhancers are located close to their target genes, these observations can explain how only a small subset of genes — those with strong distal enhancers — are sensitive to cohesin. Together, our results illuminate how 3D contacts, influenced by both cohesin and genomic distance, tune enhancer effects on gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/def05c77386b71937e2b9bc99ce09128cc1b33d4" target='_blank'>
              Cohesin-mediated 3D contacts tune enhancer-promoter regulation
              </a>
            </td>
          <td>
            Philine Guckelberger, Benjamin R. Doughty, Glen Munson, Suhas S. P. Rao, Yingxuan Tan, Xiangmeng S. Cai, C. Fulco, J. Nasser, Kristy S. Mualim, Drew T. Bergman, Judhajeet Ray, Evelyn Jagoda, Chad J. Munger, A. Gschwind, Maya U. Sheth, Anthony S. Tan, Lars M. Steinmetz, Eric S. Lander, Alexander Meissner, Erez Lieberman Aiden, J. Engreitz
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Human tumors progress in part by accumulating epigenetic alterations, which include gains and losses of DNA methylation in different parts of the cancer cell genome. Recent work has revealed a link between these two opposite alterations by showing that DNA hypomethylation in tumors can induce the expression of transcripts that overlap downstream gene promoters and thereby induce their hypermethylation. Preliminary in silico evidence prompted us to investigate if this mechanism applies to the locus harboring AGO1, a gene that plays a central role in miRNA biogenesis and RNA interference. Inspection of public RNA-Seq datasets and RT-qPCR experiments show that an alternative transcript starting 13.4 kb upstream of AGO1 (AGO1-V2) is expressed specifically in testicular germ cells, and becomes aberrantly activated in different types of tumors, particularly in tumors of the esophagus, stomach, and lung. This expression pattern classifies AGO1-V2 into the group of “Cancer-Germline” (CG) genes. Analysis of transcriptomic and methylomic datasets provided evidence that transcriptional activation of AGO1-V2 depends on DNA demethylation of its promoter region. Western blot experiments revealed that AGO1-V2 encodes a shortened isoform of AGO1, corresponding to a truncation of 75 aa in the N-terminal domain, and which we therefore referred to as “∆NAGO1”. Interestingly, significant correlations between hypomethylation/activation of AGO1-V2 and hypermethylation/repression of AGO1 were observed upon examination of tumor cell lines and tissue datasets. Overall, our study reveals the existence of a process of interdependent epigenetic alterations in the AGO1 locus, which promotes swapping between two AGO1 protein-coding mRNA isoforms in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27dc2f1e01e40a64b0f8910cba78bb0f22899826" target='_blank'>
              DNA Hypomethylation Underlies Epigenetic Swapping between AGO1 and AGO1-V2 Isoforms in Tumors
              </a>
            </td>
          <td>
            Jean S. Fain, Camille Wangermez, A. Loriot, Claudia Denoue, C. De Smet
          </td>
          <td>2024-06-22</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cancer is a disease underpinned by aberrant gene expression. Enhancers are regulatory elements that play a major role in transcriptional control and changes in active enhancer function are likely critical in the pathogenesis of oesophageal adenocarcinoma (OAC). Here, we utilise STARR-seq to profile the genome-wide enhancer landscape in OAC and identify hundreds of high-confidence enhancer elements. These regions are enriched in enhancer-associated chromatin marks, are actively transcribed and exhibit high levels of associated gene activity in OAC cells. These characteristics are maintained in human patient samples, demonstrating their disease relevance. This relevance is further underlined by their responsiveness to oncogenic ERBB2 inhibition and increased activity compared to the pre-cancerous Barrett’s state. Mechanistically, these enhancers are linked to the core OAC transcriptional network and in particular KLF5 binding is associated with high level activity, providing further support for a role of this transcription factor in defining the OAC transcriptome. Our results therefore uncover a set of enhancer elements with physiological significance, that widen our understanding of the molecular alterations in OAC and point to mechanisms through which response to targeted therapy may occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60f476e2900716d7fa8579f8b85d850f7a35ffa1" target='_blank'>
              Massively Parallel Reporter Assays identify enhancer elements in Oesophageal Adenocarcinoma
              </a>
            </td>
          <td>
            SH Yang, Y. Li, CW Bleaney, AD Sharrocks, Andrew Sharrocks
          </td>
          <td>2024-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Recently, biomolecular condensates formed through liquid–liquid phase separation have been widely reported to regulate key intracellular processes involved in cell biology and pathogenesis. BRD4 is a nuclear protein instrumental to the establishment of phase-separated super-enhancers (SEs) to direct the transcription of important genes. We previously observed that protein droplets of BRD4 became hydrophobic as their size increase, implying an ability of SEs to limit the ionization of water molecules by irradiation. Here, we aim to establish if SEs confer radiation resistance in cancer cells. We established an in vitro DNA damage assay that measures the effect of radicals provoked by the Fenton reaction on DNA integrity. This revealed that DNA damage was markedly reduced when BRD4 underwent phase separation with DNA. Accordingly, co-focal imaging analyses revealed that SE foci and DNA damage foci are mutually exclusive in irradiated cells. Lastly, we observed that the radioresistance of cancer cells was significantly reduced when irradiation was combined with ARV-771, a BRD4 de-stabilizer. Our data revealed the existence of innately radioresistant genomic regions driven by phase separation in cancer cells. The disruption of these phase-separated components enfolding genomic DNA may represent a novel strategy to augment the effects of radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e2dfd016fdeaf5bbe54441d068b3a7b5c2bf81" target='_blank'>
              Phase-separated super-enhancers confer an innate radioresistance on genomic DNA
              </a>
            </td>
          <td>
            Koki Matsumoto, Dini Kurnia Ikliptikawati, Keiichi Makiyama, Kako Mochizuki, Maho Tobita, Isao Kobayashi, D. Voon, Keesiang Lim, Kazuma Ogawa, I. Kashiwakura, Hiroshi I Suzuki, H. Yoshino, Richard W. Wong, M. Hazawa
          </td>
          <td>2024-06-14</td>
          <td>Journal of Radiation Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Most human cells contain two non-identical genomes, and differences in their regulation underlie human development and disease. We demonstrate that Fiber-seq Inferred Regulatory Elements (FIREs) enable the accurate quantification of chromatin accessibility across the 6 Gbp diploid human genome with single-molecule and single-nucleotide precision. We find that cells can harbor >1,000 regulatory elements with haplotype-selective chromatin accessibility (HSCA) and show that these elements preferentially localize to genomic loci containing the most human genetic diversity, with the human leukocyte antigen (HLA) locus showing the largest amount of HSCA genome-wide in immune cells. Furthermore, we uncover HSCA elements with sequence non-deterministic chromatin accessibility, representing likely somatic epimutations, and show that productive transcription from the inactive X chromosome is buttressed by clustered promoter-proximal elements that escape X chromosome inactivation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c0c60a592276d18a9f211d3827524a5d632d729" target='_blank'>
              A haplotype-resolved view of human gene regulation
              </a>
            </td>
          <td>
            Mitchell R. Vollger, Elliott G. Swanson, Shane J. Neph, Jane Ranchalis, Katherine M. Munson, Ching-Huang Ho, Adriana E. Sedeño-Cortés, William E. Fondrie, Stephanie C. Bohaczuk, Yizi Mao, Nancy L. Parmalee, Ben Mallory, William T. Harvey, Younjun Kwon, Gage H. Garcia, Kendra Hoekzema, Jeffrey G. Meyer, Mine Cicek, Evan E. Eichler, W. S. Noble, Daniela M. Witten, James T. Bennett, John P. Ray, A. Stergachis
          </td>
          <td>2024-06-15</td>
          <td>bioRxiv</td>
          <td>2</td>
          <td>29</td>
        </tr>

        <tr id="The three-dimensional (3D) organization of chromatin within the nucleus is crucial for gene regulation. However, the 3D architectural features that coordinate the activation of an entire chromosome remain largely unknown. We introduce an omics method, RNA-associated chromatin DNA-DNA interactions, that integrates RNA polymerase II (RNAPII)–mediated regulome with stochastic optical reconstruction microscopy to investigate the landscape of noncoding RNA roX2-associated chromatin topology for gene equalization to achieve dosage compensation. Our findings reveal that roX2 anchors to the target gene transcription end sites (TESs) and spreads in a distinctive boot-shaped configuration, promoting a more open chromatin state for hyperactivation. Furthermore, roX2 arches TES to transcription start sites to enhance transcriptional loops, potentially facilitating RNAPII convoying and connecting proximal promoter-promoter transcriptional hubs for synergistic gene regulation. These TESs cluster as roX2 compartments, surrounded by inactive domains for coactivation of multiple genes within the roX2 territory. In addition, roX2 structures gradually form and scaffold for stepwise coactivation in dosage compensation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1bc0d29c3125cfd0f353817880f4b6081993ce9" target='_blank'>
              3D chromatin structures associated with ncRNA roX2 for hyperactivation and coactivation across the entire X chromosome
              </a>
            </td>
          <td>
            S. Tian, Yang Yang, Duo Ning, Ke Fang, Kai Jing, Guangyu Huang, Yewen Xu, Pengfei Yin, Haibo Huang, Gengzhan Chen, Yuqing Deng, Shaohong Zhang, Zhimin Zhang, Zhen-Xia Chen, Tong Gao, Wei Chen, Guoliang Li, Ruilin Tian, Yijun Ruan, Yiming Li, Meizhen Zheng
          </td>
          <td>2024-07-26</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33af20a7d0aac05ac83adef2c51ac9ffcee3995c" target='_blank'>
              Enhancer engagement sustains oncogenic transformation and progression of B-cell precursor acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            G. Corleone, C. Sorino, M. Caforio, S. Di Giovenale, F. de Nicola, F. Goeman, V. Bertaina, A. Pitisci, C. Cortile, Franco Locatelli, V. Folgiero, Maurizio Fanciulli
          </td>
          <td>2024-06-27</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Gene expression regulation is one of the most fundamental cellular processes, enabling the activation of a gene to produce either the translatable protein-coding transcript (mRNA) or a functional non-coding RNA with gene regulatory functions, ultimately determining cell identity and function. Although gene expression regulation can occur at transcriptional, translational, and post-translational levels, transcription initiation is the first and the most important step in gene expression, facilitating the transfer of biological information from DNA to protein. Enhancers and super-enhancers are among the master regulators of tissue- and cell-specific transcription regulation involved in cell differentiation and tumor formation. Despite four decades passing since the first discovery of enhancers in eukaryotes and extensive efforts undertaken to identify enhancers on a genomic scale during the last decade, the discovery of enhancers still faces certain limitations and needs further investigation. The perturbation of enhancer function due to genetic or epigenetic changes is closely linked to a range of human disorders, including the development and progression of cancers. Thus, the detection of early cancer-related enhancer activity and the subsequent normalization of expression abnormalities using enhancer-targeting CRISPR epigenetic editing, as well as enhancer-targeting pharmaceuticals, are regarded as groundbreaking therapeutic tactics in preclinical stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782ddfe76cab2a16620844e3e84b112d8c90071b" target='_blank'>
              Enhancers and super-enhancers as master regulators in cancer
              </a>
            </td>
          <td>
            P. Sarvari, Pourya Sarvari, Ivonne Ramírez-Díaz, Karla Rubio
          </td>
          <td>2024-07-24</td>
          <td>INNOSC Theranostics and Pharmacological Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa98c0061b5c115d1ee9f59c1e39ed2f207c47e1" target='_blank'>
              Intertwining roles of R-loops and G-quadruplexes in DNA repair, transcription and genome organization.
              </a>
            </td>
          <td>
            P. Wulfridge, K. Sarma
          </td>
          <td>2024-06-24</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="DNA in bacterial chromosomes is organized into higher-order structures by DNA-binding proteins called nucleoid-associated proteins (NAPs) or bacterial chromatin proteins (BCPs). BCPs often bind to or near DNA loci transcribed by RNA polymerase (RNAP) and can either increase or decrease gene expression. To understand the mechanisms by which BCPs alter transcription, one must consider both steric effects and the topological forces that arise when DNA deviates from its fully relaxed double-helical structure. Transcribing RNAP creates DNA negative (-) supercoils upstream and positive (+) supercoils downstream whenever RNAP and DNA are unable to rotate freely. This (-) and (+) supercoiling generates topological forces that resist forward translocation of DNA through RNAP unless the supercoiling is constrained by BCPs or relieved by topoisomerases. BCPs also may enhance topological stress and overall can either inhibit or aid transcription. Here, we review current understanding of how RNAP, BCPs, and DNA topology interplay to control gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca193a23afae82c35775404364586c5739b5d05" target='_blank'>
              Bacterial chromatin proteins, transcription, and DNA topology: Inseparable partners in the control of gene expression.
              </a>
            </td>
          <td>
            Christine M. Hustmyer, R. Landick
          </td>
          <td>2024-06-07</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="Ovarian cancer is a deadly female cancer with high rates of recurrence. The primary treatment strategy for patients is platinum-based therapy regimens that almost universally develop resistance. Consequently, new therapeutic avenues are needed to overcome the plateau that current therapies have on patient outcomes. We describe a gene amplification involving both HSF1 and MYC, wherein these two genes on chromosome 8q are co-amplified in over 7% of human tumors that is enriched to over 30% of patients with ovarian cancer. We further found that HSF1 and MYC transcriptional activity is correlated in human tumors and ovarian cancer cell lines, suggesting they may cooperate in ovarian cancer cells. CUT&RUN for HSF1 and MYC in co-amplified ovarian cancer cells revealed that HSF1 and MYC have overlapping binding at a substantial number of locations throughout the genome where their binding peaks are near identical. Consistent with these data, a protein-protein interaction between HSF1 and MYC was detected in ovarian cancer cells, implying these two transcription factors have a molecular cooperation. Further supporting their cooperation, growth of HSF1-MYC co-amplified ovarian cancer cells were found to be dependent on both HSF1 and MYC. In an attempt to identify a therapeutic target that could take advantage of this dependency on both HSF1 and MYC, PLK1 was identified as being correlated with HSF1 and MYC in primary human tumor specimens, consistent with a previously established effect of PLK1 on HSF1 and MYC protein levels. Targeting PLK1 with the compound volasertib (BI-6727) revealed a greater than 200-fold increased potency of volasertib in HSF1-MYC co-amplified ovarian cancer cells compared to ovarian cancer cells wild-type HSF1 and MYC copy number, which extended to several growth assays, including spheroid growth. Volasertib, and other PLK1 inhibitors, have not shown great success in clinical trials and this study suggests that targeting PLK1 may be viable in a precision medicine approach using HSF1-MYC co-amplification as a biomarker for response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21e2e5303aa923a4a45241efa9a5394dc2df5936" target='_blank'>
              MYC and HSF1 Cooperate to Drive PLK1 inhibitor Sensitivity in High Grade Serous Ovarian Cancer
              </a>
            </td>
          <td>
            Imade Williams, Haddie DeHart, Matthew O’Malley, Bobby Walker, Vrushabh Ulhaskumar, Haimanti Ray, Joe R. Delaney, K. Nephew, Richard L. Carpenter
          </td>
          <td>2024-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="ABSTRACT Aneuploidy, the presence of an aberrant number of chromosomes, has been associated with tumorigenesis for over a century. More recently, advances in karyotyping techniques have revealed its high prevalence in cancer: About 90% of solid tumors and 50–70% of hematopoietic cancers exhibit chromosome gains or losses. When analyzed at the level of specific chromosomes, there are strong patterns that are observed in cancer karyotypes both pan-cancer and for specific cancer types. These specific aneuploidy patterns correlate strongly with outcomes for tumor initiation, progression, metastasis formation, immune evasion and resistance to therapeutic treatment. Despite their prominence, understanding the basis underlying aneuploidy patterns in cancer has been challenging. Advances in genetic engineering and bioinformatic analyses now offer insights into the genetic determinants of aneuploidy pattern selection. Overall, there is substantial evidence that expression changes of particular genes can act as the positive selective forces for adaptation through aneuploidy. Recent findings suggest that multiple genes contribute to the selection of specific aneuploid chromosomes in cancer; however, further research is necessary to identify the most impactful driver genes. Determining the genetic basis and accompanying vulnerabilities of specific aneuploidy patterns is an essential step in selectively targeting these hallmarks of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0123dd290f56b3193001c1283aea1071e6f411ed" target='_blank'>
              Selection forces underlying aneuploidy patterns in cancer
              </a>
            </td>
          <td>
            Tamara C. Klockner, Christopher S. Campbell
          </td>
          <td>2024-06-24</td>
          <td>Molecular & Cellular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 We sought to identify mechanisms that activate MYC expression in germinal center B cell-(GCB) diffuse large B-cell lymphomas (DLBCL) in the absence of MYC rearrangement. High-throughput CRISPR-interference screens in MYC-rearranged (n=4) and non-MYC-rearranged (n=2) GCB-DLBCL cell lines revealed a distal enhancer complex 436 kb downstream of MYC, GME-1, that was uniquely essential in non-rearranged (MYC-intact) cell lines. CRISPR-interference studies confirmed growth inhibition and MYC downregulation upon GME-1 repression in 4 DLBCL cell lines. ChIP-Seq showed that GME-1 subunits were bound by OCT2, OCA-B, and MEF2B, which are known to activate germinal center-specific enhancers, and knockdown of the corresponding genes reduced MYC expression. GME-1 subunits showed GCB-specific acetylation and accessibility across normal human B cell populations in public chromatin datasets. The mouse ortholog of GME-1 showed germinal center B cell-specific accessibility. GME-1 was strongly acetylated in MYC-intact but not MYC-rearranged GCB-DLBCL cell lines and patient biopsies. Hi-C revealed significant looping of GME-1 to the MYC promoter in MYC protein+, MYC-intact DLBCL biopsies (n=2), but not MYC-rearranged DLBCL (n=2). We used sequence conservation and chromatin accessibility data to define four GME-1 subunits (195A/B and196A/B), which were cloned into luciferase enhancer reporter constructs and compared to two subunits of the BCL6 distal super-enhancer complex that had previously been identified as regulated by the OCT2/OCA-B/MEF2B complex. GME-1 subunit 195B was the only strongly transcription-activating element based on luciferase assays across three MYC-intact cells lines, where it showed activity comparable to BCL6 enhancers. In contrast, GME-1 enhancers were minimally active in reporter assays performed in two MYC-rearranged GCB-DLBCL cell lines. Deletion of a conserved OCT2 binding site or knockdown of the POU2F2 and POU2AF1 genes reduced activity of GME-1. Thus GME-1 trans-regulation is divergent from that of previously characterized GCB enhancers, despite shared dependency on the OCT2 ternary complex. Whole-genome sequencing (WGS) revealed that the GME-1 dependent cell line SUDHL5 bears a 176bp tandem duplication within GME-1 195B. This altered enhancer showed stronger luciferase reporter activity across three MYC-intact cell lines. Published WGS datasets showed isolated genomic tandem copy gains of GME-1 in 2/49 biopsies from recurrent/refractory DLBCL and 1/93 unselected DLBCL. In a cohort of 18 DLBCL with the high-risk DZSig gene expression signature, GME-1 amplification was seen in two biopsies with concurrent deletion of the PVT1 promoter (PVT1-P) and no inter-chromosomal MYC locus rearrangement. CRISPRi showed that PVT1-P repression reduced growth and MYC expression in GME-1-dependent DLBCL, but not in cell lines where rearrangements removed PVT-P from its position in cis with the MYC gene. Our findings identify GME-1 as an activator of MYC expression and target of rare but recurrent genomic copy gains in high-risk DLBCL.
 Citation Format: Aishwarya Gurumurthy, Ashwin R. Iyer, Shih-Chun A. Chu, Marcin P. Cieslik, Russell J.H. Ryan. Characterization of an essential MYC enhancer targeted by focal genomic alterations in diffuse large B-cell lymphoma [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR04.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6db6b409667440c15e28ec2c1062b89225fa14" target='_blank'>
              Abstract PR04: Characterization of an essential MYC enhancer targeted by focal genomic alterations in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Aishwarya Gurumurthy, Ashwin Iyer, Shih-Chun A. Chu, Marcin P. Cieslik, R. Ryan
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cancer chimeric, or fusion, transcripts are thought to most frequently appear due to chromosomal aberrations that combine moieties of unrelated normal genes. When being expressed, this results in chimeric RNAs having upstream and downstream parts relatively to the breakpoint position for the 5’- and 3’-fusion components, respectively. As many other types of cancer mutations, fusion genes can be of either driver or passenger type. The driver fusions may have pivotal roles in malignisation by regulating survival, growth, and proliferation of tumor cells, whereas the passenger fusions most likely have no specific function in cancer. The majority of research on fusion gene formation events is concentrated on identifying fusion proteins through chimeric transcripts. However, contemporary studies evidence that fusion events involving non-coding RNA (ncRNA) genes may also have strong oncogenic potential. In this review we highlight most frequent classes of ncRNAs fusions and summarize current understanding of their functional roles. In many cases, cancer ncRNA fusion can result in altered concentration of the non-coding RNA itself, or it can promote protein expression from the protein-coding fusion moiety. Differential splicing, in turn, can enrich the repertoire of cancer chimeric transcripts, e.g. as observed for the fusions of circular RNAs and long non-coding RNAs. These and other ncRNA fusions are being increasingly recognized as cancer biomarkers and even potential therapeutic targets. Finally, we discuss the use of ncRNA fusion genes in the context of cancer detection and therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec4b199299758974fdcbbc48ab668c481680ff0" target='_blank'>
              Cancer fusion transcripts with human non-coding RNAs
              </a>
            </td>
          <td>
            Tharaa Mohammad, Marianna A. Zolotovskaia, Maria V. Suntsova, Anton Buzdin
          </td>
          <td>2024-06-11</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9b28bde62e7c6ec79bde107b1d678f6c8aab03e" target='_blank'>
              MethNet: a robust approach to identify regulatory hubs and their distal targets from cancer data
              </a>
            </td>
          <td>
            Theodore Sakellaropoulos, Catherine Do, Guimei Jiang, Giulia Cova, Peter Meyn, D. Dimartino, Sitharam Ramaswami, Adriana Heguy, A. Tsirigos, Jane A Skok
          </td>
          <td>2024-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Regulating gene expression involves significant and frequent changes in the chromatin environment at the locus level, especially at regulatory sequences. However, their modulation in response to pharmacological treatments or pathological conditions remain mostly undetermined. Here, we report versatile locus-specific proteomics tools to address this knowledge gap, which combine the targeting ability of the CRISPR/Cas9 system and the protein-labelling capability of the highly reactive biotin ligases TurboID (in CasTurbo) and UltraID (in CasUltra). CasTurbo and CasUltra enabled rapid chromatin protein labelling under mild conditions at repetitive sequences like centromeres and telomeres, as well as non-amplified genes. We applied CasUltra to A375 melanoma cell lines to decipher the protein environment of the MYC promoter and characterize the molecular effects of the bromodomain inhibitor JQ1, which targets bromodomain and extra-terminal (BET) proteins that regulate MYC expression. We quantified the consequences of BET protein displacement from the MYC promoter and found that it was associated with a considerable reorganisation of the chromatin composition. In addition, BET protein retention at the MYC promoter was consistent with a model of increased JQ1 resistance. Thus, through the combination of proximity biotinylation and CRISPR-Cas9-dependent genomic targeting, CasTurbo and CasUltra have successfully demonstrated their utility in profiling the proteome associated with a genomic locus in living cells. In Brief Kougnassoukou Tchara et al. report the development and application of CasTurbo and CasUltra, two locus-specific proteomics tools that fuse catalytically dead Cas9 to the engineered biotin ligases TurboID and UltraID. These tools enabled the quantitative mapping of locus-specific chromatin remodelling due to pharmacological inhibition. Highlights CasTurbo and CasUltra were developed for locus-specific label-free proteomics CasTurbo mapped the proteins localized to the centromeres and telomeres Proteins bound to the MYC promoter were quantified in melanoma cells with CasUltra CasUltra is compatible with investigating pharmacological treatment effects Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce2d069fe2369eb465a0930b8b4ef5639c3f142c" target='_blank'>
              Coupling proximity biotinylation with genomic targeting to characterize locus-specific changes in chromatin environments
              </a>
            </td>
          <td>
            Pata-Eting Kougnassoukou Tchara, J. Loehr, Jean-Philippe Lambert
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a040a0cb95cb5f3026b82c4853a3fbf7dc09e9f" target='_blank'>
              Structure and transcription of integrated HPV DNA in vulvar carcinomas
              </a>
            </td>
          <td>
            A. Van Arsdale, Lauren Turker, Yoke-Chen Chang, Joshua Gould, Bryan Harmon, E. Maggi, Olga Meshcheryakova, Maxwell P Brown, Dana Luong, Koenraad Van Doorslaer, Mark H. Einstein, Dennis Y. S. Kuo, Deyou Zheng, Brian J. Haas, Jack Lenz, Cristina Montagna
          </td>
          <td>2024-06-19</td>
          <td>NPJ Genomic Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer.
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b40bf07806bd0a97226f528a2d387e1d1cd79723" target='_blank'>
              Enhancers associated with unstable RNAs are rare in plants.
              </a>
            </td>
          <td>
            Bayley R McDonald, Colette L. Picard, Ian M. Brabb, Marina I. Savenkova, Robert J. Schmitz, Steve Jacobsen, Sascha H. C. Duttke
          </td>
          <td>2024-07-30</td>
          <td>Nature plants</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f04511465a5fd072722dcf193ecd5af72668beca" target='_blank'>
              CircR-loop: a novel RNA:DNA interaction on genome instability
              </a>
            </td>
          <td>
            Xinming Su, Yaojie Feng, Ruixiu Chen, Shiwei Duan
          </td>
          <td>2024-06-14</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The MYC oncoprotein regulates numerous genes involved in cellular processes such as cell cycle and mitochondrial and ribosomal structure and function. This requires heterodimerization with its partner, MAX, and binding to specific promoter and enhancer elements. Here, we show that MYC and MAX also bind near transcriptional end sites (TESs) of over one-sixth of all annotated genes. These interactions are dose-dependent, evolutionarily conserved, stabilize the normally short-lived MYC protein and regulate expression both in concert with and independent of MYC’s binding elsewhere. MYC’s TES binding occurs in association with other transcription factors, alters the chromatin landscape, increases nuclease susceptibility and can alter transcriptional read-through, particularly in response to certain stresses. MYC-bound TESs can directly contact promoters and may fine-tune gene expression in response to both physiologic and pathologic stimuli. Collectively, these findings support a previously unrecognized role for MYC in regulating transcription and its read-through via direct intragenic contacts between TESs and promoters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4174bbeb7f774b02268693f41e5991ebe810f527" target='_blank'>
              MYC Binding Near Transcriptional End Sites Regulates Basal Gene Expression, Read-Through Transcription and Intragenic Contacts
              </a>
            </td>
          <td>
            Huabo Wang, Bingwei Ma, Taylor Stevens, Jessica Knapp, Jie Lu, E. Prochownik
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Transcribed enhancer maps can reveal nuclear interactions underpinning each cell type and connect specific cell types to diseases. Using a 5′ single-cell RNA sequencing approach, we defined transcription start sites of enhancer RNAs and other classes of coding and noncoding RNAs in human CD4+ T cells, revealing cellular heterogeneity and differentiation trajectories. Integration of these datasets with single-cell chromatin profiles showed that active enhancers with bidirectional RNA transcription are highly cell type–specific and that disease heritability is strongly enriched in these enhancers. The resulting cell type–resolved multimodal atlas of bidirectionally transcribed enhancers, which we linked with promoters using fine-scale chromatin contact maps, enabled us to systematically interpret genetic variants associated with a range of immune-mediated diseases. Editor’s summary Currently, 3′ single-cell RNA sequencing is widely used as a tool for single-cell gene expression analysis. By leveraging a distinctive “cap signature” derived from the 5′-end cap, Oguchi et al. introduced a technique to investigate the precise position of the RNA 5′-ends in single cells, enabling simultaneous profiling of gene expression and enhancer activity. The authors constructed a CD4+ T cell–type resolved multimodal atlas to integrate and elucidate CD4+ T cell diversity and genetic risk of immune-mediated diseases. They also identified rare, uncharacterized CD4+ T cell types that might contribute to the pathology of immune-mediated diseases. —Di Jiang INTRODUCTION Enhancers are distal cis-regulatory elements that increase the expression of target genes. Active enhancers produce bidirectional enhancer RNAs (eRNAs). Transcription of eRNAs can be used to measure enhancer activity and to delineate enhancer architecture at nucleotide resolution. Profiling of transcribed enhancers in single human cells can provide insights into cell type–specific gene regulation and genetic predisposition to diseases. However, comprehensive maps of eRNAs in cell types relevant to diseases are still lacking. RATIONALE The 5′ cap is added to the first nucleotide of an RNA molecule. By leveraging a specific “cap signature,” we developed read-level prefiltering and transcribed enhancer call (ReapTEC), a method for simultaneously profiling gene expression and enhancer activity by using 5′-end single-cell RNA sequencing (5′ scRNA-seq). We applied ReapTEC and single-nucleus assay for transposase-accessible chromatin using sequencing (snATAC-seq) to a total of ~1 million single human CD4+ T cells and constructed a CD4+ T cell type–resolved multimodal atlas to pinpoint genetic and cellular causes of immune-mediated diseases. RESULTS By using ReapTEC, we accurately mapped the transcription start sites of mRNAs, long noncoding RNAs, and eRNAs in single cells, which allowed us to study the expression of not only genes but also bidirectionally transcribed candidate enhancers (btcEnhs). We characterized rare and uncharacterized cell types, including GPR25-expressing regulatory T cells, along with the expression profiles of ~130 surface proteins. We identified 62,803 btcEnhs and 218,508 accessible chromatin regions active across diverse CD4+ T cells. We found higher cell type specificity for RNA transcription than for chromatin accessibility both at promoters and enhancers. We also showed that the heritability of immune-mediated diseases was enriched more in btcEnhs than in open chromatin regions. Using our CD4+ T cell type–resolved atlas of btcEnhs, we systematically interpreted genetic variants associated with a range of immune-mediated diseases. We found that btcEnhs harboring disease SNPs were often active in specific subsets of CD4+ T cells; generally, they were conserved across mammals but in some cases were primate specific. We investigated the target genes of disease-relevant btcEnhs by generating fine-scale chromatin contact maps derived from ultradeep Micro-C data; we further performed CRISPR activation of selected disease-relevant btcEnhs to validate their target genes. CONCLUSION Our study establishes a platform for integrating and elucidating CD4+ T cell diversity and genetic risk of immune-mediated diseases. Although 3′ scRNA-seq is widely used as a major tool for single-cell gene expression analysis, our study demonstrates the strength of 5′ scRNA-seq. We anticipate the utility of our multimodal atlas in addressing variant-to-function challenges. CD4+ T cell type–resolved atlas of btcEnhs for systematic interpretation of the genetic basis of immune-mediated diseases. (A) ReapTEC allows us to detect transcription start sites (TSSs) and active enhancers using 5′ scRNA-seq. (B) Disease-relevant btcEnhs containing genetic variants associated with immune-mediated diseases were linked to target genes by using fine-scale chromatin contact maps. CITE-seq, cellular indexing of transcriptomes and epitopes by sequencing; Ref, reference allele; Alt, alternative allele.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a4b6cb952b21d2e76916c18021dfddb89e4c76" target='_blank'>
              An atlas of transcribed enhancers across helper T cell diversity for decoding human diseases
              </a>
            </td>
          <td>
            Akiko Oguchi, A. Suzuki, Shuichiro Komatsu, Hiroyuki Yoshitomi, S. Bhagat, Raku Son, R. Bonnal, Shohei Kojima, Masaru Koido, Kazuhiro Takeuchi, K. Myouzen, Gyo Inoue, Tomoya Hirai, Hiromi Sano, Y. Takegami, A. Kanemaru, Itaru Yamaguchi, Y. Ishikawa, Nao Tanaka, Shigeki Hirabayashi, Riyo Konishi, Sho Sekito, Takahiro Inoue, Juha Kere, Shunichi Takeda, A. Takaori-Kondo, Itaru Endo, S. Kawaoka, Hideya Kawaji, K. Ishigaki, H. Ueno, Y. Hayashizaki, Massimiliano Pagani, P. Carninci, M. Yanagita, Nicholas Parrish, Chikashi Terao, Kazuhiko Yamamoto, Yasuhiro Murakawa
          </td>
          <td>2024-07-05</td>
          <td>Science</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="The chromatin in the human brain cortex shows more enhancer-enhancer contacts than in macaques and mice, yet the organization of these contacts across cellular states and their mechanisms remain unclear. Here, we developed SCOPE-C, a cost-effective technique designed for mapping DNase I Hypersensitive Sites and their spatial interactions, with the potential for scalability to single-cell resolution. Employing SCOPE-C, we generated a comprehensive chromatin map detailing cis-regulatory elements in four cell types during neurogenesis in human, macaque, and mouse fetal cortex. This map introduces a model of long-range enhancer networks that regulate gene transcription during human cortical neurogenesis. In this model, CTCF-mediated loop extrusion forms ‘stripes’ of enhancer interaction hubs extending up to 10 Mb in human Excitatory Neurons. Enriched with human-specific enhancers and neuropsychiatric disorder-linked SNPs, these remarkably long-range networks predominantly control EN marker genes, suggesting their critical role in robust gene expression during human cortical neurogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57015b434b0a08d1168f68b6cbcda6a638513d66" target='_blank'>
              Long-Range Enhancer Networks Gradually Emerge as Key Regulators During Human Cortical Neurogenesis
              </a>
            </td>
          <td>
            Lumeng Jia, Yingping Hou, Zejia Cui, Xin Luo, Jiali Duan, Jianbin Guo, Minglei Shi, Hebing Chen, Tingting Li, Fengyun Zhang, Bing Su, Lan Zhu, Cheng Li
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Pancreatic cancer is a malignancy arising from the endocrine or exocrine compartment of this organ. Tumors from exocrine origin comprise over 90% of all pancreatic cancers diagnosed. Of these, pancreatic ductal adenocarcinoma (PDAC) is the most common histological subtype. The five-year survival rate for PDAC ranged between 5 and 9% for over four decades, and only recently saw a modest increase to ∼12-13%, making this a severe and lethal disease. Like other cancers, PDAC initiation stems from genetic changes. However, therapeutic targeting of PDAC genetic drivers has remained relatively unsuccessful, thus the focus in recent years has expanded to the non-genetic factors underlying the disease pathogenesis. Specifically, it has been proposed that dynamic changes in the epigenetic landscape promote tumor growth and metastasis. Emphasis has been given to the re-organization of enhancers, essential regulatory elements controlling oncogenic gene expression, commonly marked my histone 3 lysine 4 monomethylation (H3K4me1). H3K4me1 is typically deposited by histone lysine methyltransferases (KMTs). While well characterized as oncogenes in other cancer types, recent work has expanded the role of KMTs as tumor suppressor in pancreatic cancer. Here, we review the role and translational significance for PDAC development and therapeutics of KMTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a7417d79919ae9f2eab24051f1e99fe379fd6b" target='_blank'>
              Insights into the mechanisms driven by H3K4 KMTs in pancreatic cancer.
              </a>
            </td>
          <td>
            Kayla C LaRue-Nolan, G. L. Raja Arul, Ashley N Sigafoos, Jiaqi Shi, Martin E Fernandez-Zapico
          </td>
          <td>2024-07-30</td>
          <td>The Biochemical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Copy number alterations (CNAs), resulting from the gain or loss of genetic material from as little as 50 base pairs or as big as entire chromosome(s), have been associated with many congenital diseases, de novo syndromes and cancer. It is established that CNAs disturb the dosage of genomic regions including enhancers/promoters, long non-coding RNA and gene(s) among others, ultimately leading to an altered balance of key cellular functions. In cancer, CNAs have been associated with almost all steps of the disease: predisposition, initiation, development, maintenance, response to treatment, resistance, and relapse. Therefore, understanding how specific CNAs contribute to tumourigenesis may provide prognostic insight and ultimately lead to the development of new therapeutic approaches to improve patient outcomes. In this review, we provide a snapshot of what is currently known about CNAs and cancer, incorporating topics regarding their detection, clinical impact, origin, and nature, and discuss the integration of innovative genetic engineering strategies, to highlight the potential for targeting CNAs using novel, dosage-sensitive and less toxic therapies for CNA-driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b107cf5f7a3a29e8e0fe916d4102734572eefc3a" target='_blank'>
              Insights into the Clinical, Biological and Therapeutic Impact of Copy Number Alteration in Cancer
              </a>
            </td>
          <td>
            Shannon L. Carey-Smith, R. Kotecha, L. C. Cheung, Sébastien Malinge
          </td>
          <td>2024-06-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="INTRODUCTION
Isodicentric Y chromosomes are relatively common structural variants of the human genome. The underlying mechanism of isodicentric Y chromosomes with short arm breakpoints [idic(Yq)] remains to be clarified.


CASE PRESENTATION
We encountered a Japanese man with azoospermia and mild short stature. G-banding and array-based comparative genomic hybridization indicated that his karyotype was 45,X/46,X,idic(Y)(qter→p11.32::p11.32→qter) with a ~1.8 Mb terminal deletion. Whole genome sequencing suggested that the Y chromosome had four breakpoints in a ~7 kb region of the pseudoautosomal region 1 (PAR1).


CONCLUSION
This case was assumed to have an idic(Yq) resulting from multiple DNA double-strand breaks in PAR1. This rearrangement may have been facilitated by the PAR1-specific chromatin architecture. The clinical features of the patient can be ascribed to SHOX haploinsufficiency and the presence of a 45,X cell line, although copy-number gains of some Yq genes and the size reduction of PAR1 may also contribute to his spermatogenic failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6ffa4553f26785a976ed64f6e1aecf9e157203" target='_blank'>
              Isodicentric Y chromosome with multiple breakpoints in the pseudoautosomal region 1.
              </a>
            </td>
          <td>
            Yasuko Ogiwara, Y. Kobori, E. Suzuki, Atsushi Hattori, Kanako Tanase-Nakao, Akiyoshi Osaka, Toshiyuki Iwahata, Hiroshi Okada, Y. Kuroki, M. Fukami
          </td>
          <td>2024-07-29</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9b534a98269bd9df6b1409e37a459858bd7c03f" target='_blank'>
              Exploring the enigma: history, present, and future of long non-coding RNAs in cancer
              </a>
            </td>
          <td>
            Q. Naseer, Abdul Malik, Fengyuan Zhang, Shengxia Chen
          </td>
          <td>2024-06-07</td>
          <td>Discover Oncology</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6169884636c48723540d43c960017df0f71986" target='_blank'>
              Hidden secrets of the cancer genome: unlocking the impact of non-coding mutations in gene regulatory elements
              </a>
            </td>
          <td>
            Sandra Íñiguez-Muñoz, P. Llinàs-Arias, Miquel Ensenyat-Mendez, Andrés F. Bedoya-López, Javier I. J. Orozco, Javier Cortés, Ananya Roy, Karin Forsberg-Nilsson, M. DiNome, D. Marzese
          </td>
          <td>2024-06-20</td>
          <td>Cellular and Molecular Life Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Rearrangements between genes can yield neomorphic fusions that drive oncogenesis. Fusion oncogenes are made up of fractional segments of the partner genes that comprise them, with each partner potentially contributing some of its own function to the nascent fusion oncoprotein. Clinically, fusion oncoproteins driving one diagnostic entity are typically clustered into a single molecular subset and are often treated a similar fashion. However, knowledge of where specific fusion breakpoints occur in partner genes, and the resulting retention of functional domains in the fusion, is an important determinant of fusion oncoprotein activity and may differ between patients. This study investigates this phenomena through the example of CIC::DUX4, a fusion between the transcriptional repressor capicua (CIC) and the double homeobox 4 gene (DUX4), which drives an aggressive subset of undifferentiated round cell sarcoma. Using a harmonized dataset of over 100 patient fusion breakpoints from the literature, we show that most bona fide CIC::DUX4 fusions retain the C1 domain, which is known to contribute to DNA binding by wild type CIC. Mechanistically, deletion or mutation of the C1 domain reduces, but does not eliminate, activation of CIC target genes by CIC::DUX4. We also find that expression of C1-deleted CIC::DUX4 is capable of exerting intermediate transformation-related phenotypes compared with those imparted by full-length CIC::DUX4, but was not sufficient for tumorigenesis in a subcutaneous mouse model. In summary, our results suggest a supercharging role for the C1 domain in the activity of CIC::DUX4. Significance Statement Fusion oncogenes are neomorphic entities comprised of fractional coding sequences from two partner genes that have been inappropriately rearranged. The functional domains contributed by the partner genes shape the function of the resulting fusion. We use CIC::DUX4, a transcription factor fusion that drives an ultra-rare soft tissue sarcoma, to explore how preferential retention of a partner gene domain may influence the activity of the overall fusion. Our results indicate that the capicua C1 domain is retained in most CIC::DUX4 transcripts and is required for full activity of the CIC::DUX4 oncoprotein. We demonstrate that knowledge of where breakpoints occur and the resulting impact this has on the retention of functional domains can teach us about fusion behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07ff139acac1d619765a5ddb7edf4ef62f9f935" target='_blank'>
              The Capicua C1 Domain is Required for Full Activity of the CIC::DUX4 Fusion Oncoprotein
              </a>
            </td>
          <td>
            Cuyler Luck, Kyle A. Jacobs, Ross A. Okimoto
          </td>
          <td>2024-06-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, Andre Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The most significant genetic influence on eye color pigmentation is attributed to the intronic SNP rs12913832 in the HERC2 gene, which interacts with the promoter region of the contiguous OCA2 gene. This interaction, through the formation of a chromatin loop, modulates the transcriptional activity of OCA2, directly affecting eye color pigmentation. Recent advancements in technology have elucidated the precise spatial organization of the genome within the cell nucleus, with chromatin architecture playing a pivotal role in regulating various genome functions. In this study, we investigated the organization of the chromatin close to the HERC2/OCA2 locus in human lymphocyte nuclei using fluorescence in situ hybridization (FISH) and high-throughput chromosome conformation capture (Hi-C) data. The 3 Mb of genomic DNA that belonged to the chromosomal region 15q12-q13.1 revealed the presence of three contiguous chromatin loops, which exhibited a different level of compaction depending on the presence of the A or G allele in the SNP rs12913832. Moreover, the analysis of the genomic organization of the genes has demonstrated that this chromosomal region is evolutionarily highly conserved, as evidenced by the analysis of syntenic regions in species from other Vertebrate classes. Thus, the role of rs12913832 variant is relevant not only in determining the transcriptional activation of the OCA2 gene but also in the chromatin compaction of a larger region, underscoring the critical role of chromatin organization in the proper regulation of the involved genes. It is crucial to consider the broader implications of this finding, especially regarding the potential regulatory role of similar polymorphisms located within intronic regions, which do not influence the same gene by modulating the splicing process, but they regulate the expression of adjacent genes. Therefore, caution should be exercised when utilizing whole-exome sequencing for diagnostic purposes, as intron sequences may provide valuable gene regulation information on the region where they reside. Thus, future research efforts should also be directed towards gaining a deeper understanding of the precise mechanisms underlying the role and mode of action of intronic SNPs in chromatin loop organization and transcriptional regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/415e3e0211688fa86f1fcb9648668b6356857209" target='_blank'>
              The Chromatin Organization Close to SNP rs12913832, Involved in Eye Color Variation, Is Evolutionary Conserved in Vertebrates
              </a>
            </td>
          <td>
            Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone, Concetta Federico
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4781c7d248ebd4c7b1d88e8451d7cdc1606f06e6" target='_blank'>
              Genome-wide enhancer RNA profiling adds molecular links between genetic variation and human cancers
              </a>
            </td>
          <td>
            Yimin Cai, Zequn Lu, Bin Li, Jinyu Huang, Ming Zhang, Can Chen, Linyun Fan, Qian-Ying Ma, Chunyi He, Shuo Chen, Yuan Jiang, Yan-Min Li, Caibo Ning, Fu-Wei Zhang, Wenzhuo Wang, Yizhuo Liu, Heng Zhang, Meng Jin, Xiao-Yang Wang, Jinxin Han, Zhen Xiong, Ming Cai, Chaoqun Huang, Xiaojun Yang, Xu Zhu, Ying Zhu, Xiaoping Miao, Shao-Kai Zhang, Yongchang Wei, Jianbo Tian
          </td>
          <td>2024-06-11</td>
          <td>Military Medical Research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Many essential functions of organisms are encoded in highly repetitive genomic regions, including histones involved in DNA packaging, centromeres that are core components of chromosome segregation, ribosomal RNA comprising the protein translation machinery, telomeres that ensure chromosome integrity, piRNA clusters encoding host defenses against selfish elements, and virtually the entire Y chromosome. These regions, formed by highly similar tandem arrays, pose significant challenges for experimental and informatic study, impeding sequence-level descriptions essential for understanding genetic variation. Here, we report the assembly and variation analysis of such repetitive regions in Drosophila melanogaster, offering significant improvements to the existing community reference assembly. Our work successfully recovers previously elusive segments, including complete reconstructions of the histone locus and the pericentric heterochromatin of the X chromosome, spanning the Stellate locus to the distal flank of the rDNA cluster. To infer structural changes in these regions where alignments are often not practicable, we introduce landmark anchors based on unique variants that are putatively orthologous. These regions display considerable structural variation between different D. melanogaster strains, exhibiting differences in copy number and organization of homologous repeat units between haplotypes. In the histone cluster, although we observe minimal genetic exchange indicative of crossing over, the variation patterns suggest mechanisms such as unequal sister chromatid exchange. We also examine the prevalence and scale of concerted evolution in the histone and Stellate clusters and discuss the mechanisms underlying these observed patterns.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d432b790a64d5e41e24bfb91fe60924d015814eb" target='_blank'>
              Genetic variation in recalcitrant repetitive regions of the Drosophila melanogaster genome
              </a>
            </td>
          <td>
            Harsh G. Shukla, Mahul Chakraborty, J. Emerson
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>11</td>
        </tr>

        <tr id="We describe exciting recent advances in fusion-driven sarcoma etiology, from an epigenetics perspective. By exploring the current state of the field, we identify and describe the central mechanisms that determine sarcomagenesis. Further, we discuss seminal studies in translational genomics, which enabled epigenetic characterization of fusion-driven sarcomas. Important context for epigenetic mechanisms include, but are not limited to, cell cycle and metabolism, core regulatory circuitry, 3-dimensional chromatin architectural dysregulation, integration with ATP-dependent chromatin remodeling, and translational animal modeling. Paradoxically, while the genetic requirements for oncogenic transformation are highly specific for the fusion partners, the epigenetic mechanisms we as a community have uncovered are categorically very broad. This dichotomy prompts the question of whether the investigation of rare disease epigenomics should prioritize studying individual cell populations, thereby examining whether the mechanisms of chromatin dysregulation are specific to a particular tumor. We review recent advances focusing on rhabdomyosarcoma, synovial sarcoma, alveolar soft part sarcoma, clear cell sarcoma, undifferentiated round cell sarcoma, Ewing sarcoma, myxoid/round liposarcoma, epithelioid hemangioendothelioma and desmoplastic round cell tumor. The growing number of groundbreaking discoveries in the field, motivated us to anticipate further exciting advances in the area of mechanistic epigenomics and direct targeting of fusion transcription factors in the years ahead.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01bc563cf3c1cf1b5a64e47d7f0fdef6c20adb4b" target='_blank'>
              Epigenetic determinants of fusion-driven sarcomas: paradigms and challenges
              </a>
            </td>
          <td>
            Benjamin Z. Stanton, S. Pomella
          </td>
          <td>2024-06-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Dictyostelium discoideum is a unicellular slime mold, developing into a multicellular fruiting body upon starvation. Development is accompanied by large-scale shifts in gene expression program, but underlying features of chromatin spatial organization remain unknown. Here, we report that the Dictyostelium 3D genome is organized into positionally conserved, largely consecutive, non-hierarchical and weakly insulated loops at the onset of multicellular development. The transcription level within the loop interior tends to be higher than in adjacent regions. Loop interiors frequently contain functionally linked genes and genes which coherently change expression level during development. Loop anchors are predominantly positioned by the genes in convergent orientation. Our data suggest that the loop profile may arise from the interplay between transcription and extrusion-driven chromatin folding. In particular, a convergent gene pair serves as a bidirectional extrusion barrier or a “diode” that controls passage of the cohesin extruder by relative transcription level of paired genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cc6bac72041617c8f26b569d3d404fea042317f" target='_blank'>
              Convergent gene pairs restrict chromatin looping in Dictyostelium discoideum, acting as directional barriers for extrusion
              </a>
            </td>
          <td>
            Irina V. Zhegalova, S. Ulianov, Alexandra Galitsyna, Ilya A. Pletenev, Olga Tsoy, Artem V. Luzhin, P. Vasiluev, Egor S. Bulavko, Dmitry N. Ivankov, Alexey A. Gavrilov, E. Khrameeva, Mikhail S. Gelfand, S. V. Razin
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>18</td>
        </tr>

        <tr id="Alu retrotransposons, which form the largest family of mobile DNA elements in the human genome, have recently come to attention as a potential source of regulatory novelties, most notably by participating in enhancer function. Even though Alu transcription by RNA polymerase III is subjected to tight epigenetic silencing, their expression has long been known to increase in response to various types of stress, including viral infection. Here we show that, in primary human fibroblasts, adenovirus small e1a triggered derepression of hundreds of individual Alus by promoting TFIIIB recruitment by Alu-bound TFIIIC. Epigenome profiling revealed an e1a-induced decrease of H3K27 acetylation and increase of H3K4 monomethylation at derepressed Alus, making them resemble poised enhancers. The enhancer nature of e1a-targeted Alus was confirmed by the enrichment, in their upstream regions, of the EP300/CBP acetyltransferase, EP400 chromatin remodeler and YAP1 and FOS transcription factors. The physical interaction of e1a with EP400 was critical for Alu derepression, which was abrogated upon EP400 ablation. Our data suggest that e1a targets a subset of enhancer Alus whose transcriptional activation, which requires EP400 and is mediated by the e1a-EP400 interaction, may participate in the manipulation of enhancer activity by adenoviruses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a36f9b26ed47425d2917decac2468f0de41c5512" target='_blank'>
              Adenovirus small E1A directs activation of Alu transcription at YAP/TEAD- and AP-1-bound enhancers through interactions with the EP400 chromatin remodeler.
              </a>
            </td>
          <td>
            Simona Cantarella, M. Vezzoli, Davide Carnevali, M. Morselli, Nathan R. Zemke, Barbara Montanini, Coralie F. Daussy, Harald Wodrich, Martin Teichmann, M. Pellegrini, Arnold J. Berk, G. Dieci, Roberto Ferrari
          </td>
          <td>2024-07-16</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8409b669072e4562ce558ad50b151f454f90ead6" target='_blank'>
              An oncogenic enhancer promotes melanoma progression via regulating ETV4 expression
              </a>
            </td>
          <td>
            Junyou Zhang, Qilin Wang, Sihan Qi, Yingying Duan, Zhaoshuo Liu, Jiaxin Liu, Ziyi Zhang, Chunyan Li
          </td>
          <td>2024-06-07</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosomal translocations in non-Hodgkin lymphoma (NHL) result in activation of oncogenes by placing them under the regulation of immunoglobulin heavy chain (IGH) super-enhancers. Aberrant expression of translocated oncogenes induced by enhancer activity can contribute to lymphomagenesis. The role of the IGH enhancers in normal B-cell development is well established, but knowledge regarding the precise mechanisms of their involvement in control of the translocated oncogenes is limited. The goal of this project was to define the critical regions in the IGH regulatory elements and identify enhancer RNAs (eRNA). We designed a sgRNA library densely covering the IGH enhancers and performed tiling CRISPR interference screens in three NHL cell lines. This revealed three regions crucial for NHL cell growth. With chromatin-enriched RNA-Seq we showed transcription from the core enhancer regions and subsequently validated expression of the eRNAs in a panel of NHL cell lines and tissue samples. Inhibition of the essential IGH enhancer regions decreased expression of eRNAs and translocated oncogenes in several NHL cell lines. The observed expression and growth patterns were consistent with the breakpoints in the IGH locus. Moreover, targeting the Eμ enhancer resulted in loss of B-cell receptor expression. In a Burkitt lymphoma cell line, MYC overexpression partially rescued the phenotype induced by IGH enhancer inhibition. Our results indicated the most critical regions in the IGH enhancers and provided new insights into the current understanding of the role of IGH enhancers in B-cell NHL. As such, this study forms a basis for development of potential therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/801fa2579edbf74a0b66ac42b4a2406cc3439a8b" target='_blank'>
              Core regions in immunoglobulin heavy chain enhancers essential for survival of non-Hodgkin lymphoma cells are identified by a CRISPR interference screen.
              </a>
            </td>
          <td>
            Marta elzbieta Kasprzyk, Weronika Sura, M. Podralska, Marta Kazimierska, Annika Seitz, Wojciech Łosiewski, Tomasz Woźniak, Jeroen E J Guikema, A. Diepstra, Joost L. Kluiver, Anke van den Berg, N. Rozwadowska, A. Dzikiewicz-Krawczyk
          </td>
          <td>2024-06-27</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="During the metagenomics era, high-throughput sequencing efforts both in mice and humans indicate that non-coding RNAs (ncRNAs) constitute a significant fraction of the transcribed genome. During the past decades, the regulatory role of these non-coding transcripts along with their interactions with other molecules have been extensively characterized. However, the study of long non-coding RNAs (lncRNAs), an ncRNA regulatory class with transcript lengths that exceed 200 nucleotides, revealed that certain non-coding transcripts are transcriptional “by-products”, while their loci exert their downstream regulatory functions through RNA-independent mechanisms. Such mechanisms include, but are not limited to, chromatin interactions and complex promoter-enhancer competition schemes that involve the underlying ncRNA locus with or without its nascent transcription, mediating significant or even exclusive roles in the regulation of downstream target genes in mammals. Interestingly, such RNA-independent mechanisms often drive pathological manifestations, including oncogenesis. In this review, we summarize selective examples of lncRNAs that regulate target genes independently of their produced transcripts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e02f6969d92ee90d00c0c3dd416b8e82fcead5d7" target='_blank'>
              RNA-Independent Regulatory Functions of lncRNA in Complex Disease
              </a>
            </td>
          <td>
            Michaela Kafida, Maria Karela, Antonis Giakountis
          </td>
          <td>2024-07-31</td>
          <td>Cancers</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Breast cancer (BCa) is a major global health challenge, characterized by chromosomal instability (CIN) and subsequent acquisition of extensive somatic copy number alterations (CNAs). CNAs including amplifications and deletions, significantly influence intra-tumor heterogeneity and the tumor microenvironment (TME). Among these, the loss of chromosome 13q14.2 emerges as a considerable factor in BCa pathogenesis and treatment responses. We provide evidence that this genomic alteration is under positive selective pressure, correlating with poorer patient survival. Furthermore, through multi-omic and in vitro analyses, we uncover a dual role of 13q14.2 loss: it confers a survival advantage to tumor cells and modulate the cell cycle and pro-apoptotic pathways in cancer cells, affecting macrophages population in the TME, while paradoxically increasing tumor susceptibility to BCL2 inhibitors. These findings suggest that targeting 13q14.2 as a biomarker in BCa could enhance the efficacy of existing treatments and offer a new avenue for improving clinical outcomes in BCa. Figure 1. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e388fe8f21125bbefce3cee2961f295d519ae77" target='_blank'>
              Loss of chromosome cytoband 13q14.2 orchestrates breast cancer pathogenesis and drug response
              </a>
            </td>
          <td>
            Parastoo Shahrouzi, Youness Azimzade, Wioletta Brankiewicz, Sugandha Bhatia, David Kunke, Derek Richard, Xavier Tekpli, Vessela N Kristensen, Pascal H.G. Duijf
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc88953f3695ac0c90c5077d3442247bd93ef70" target='_blank'>
              Mixed responses to targeted therapy driven by chromosomal instability through p53 dysfunction and genome doubling
              </a>
            </td>
          <td>
            S. Hobor, M. Al Bakir, C. Hiley, Marcin Skrzypski, A. Frankell, Bjorn Bakker, T. Watkins, A. Markovets, J. Dry, Andrew P. Brown, Jasper van der Aart, H. van den Bos, D. Spierings, D. Oukrif, Marco Novelli, T. Chakrabarti, Adam H. Rabinowitz, Laila Ait Hassou, Saskia Litière, D. Kerr, L. Tan, Gavin P. Kelly, David A Moore, M. Renshaw, Subramanian Venkatesan, William Hill, A. Huebner, C. Martínez-Ruiz, James R. M. Black, Wei Wu, Mihaela Angelova, N. Mcgranahan, Julian Downward, J. Chmielecki, Carl Barrett, K. Litchfield, S. Chew, Collin M Blakely, Elza C de Bruin, F. Foijer, Karen H. Vousden, T. Bivona, , Jason Lester, A. Bajaj, A. Nakas, Azmina Sodha-Ramdeen, M. Tufail, M. Scotland, R. Boyles, S. Rathinam, C. Wilson, Domenic Marrone, S. Dulloo, D. Fennell, Gurdeep Matharu, J. Shaw, E. Boleti, Heather Cheyne, Mohammed Khalil, S. Richardson, Tracey Cruickshank, Gillian Price, K. M. Kerr, Sarah Benafif, Jack French, Kayleigh Gilbert, B. Naidu, Akshay J. Patel, A. Osman, Carol Enstone, G. Langman, Helen Shackleford, M. Djearaman, S. Kadiri, Gary Middleton, Angela E. Leek, Jack Davies Hodgkinson, Nicola Totton, A. Montero, E. Smith, E. Fontaine, F. Granato, A. Paiva-Correia, Juliette Novasio, K. Rammohan, L. Joseph, P. Bishop, Rajesh Shah, Stuart Moss, Vijay Joshi, Philip A J Crosbie, Katherine D. Brown, Mathew Carter, Anshuman Chaturvedi, Pedro Oliveira, Colin R. Lindsay, Fiona H. Blackhall, Matthew G. Krebs, Yvonne Summers, A. Clipson, J. Tugwood, Alastair Kerr, D. Rothwell, C. Dive, H. Aerts, R. Schwarz, Tom L. Kaufmann, G. Wilson, R. Rosenthal, P. Van Loo, Nicolai J. Birkbak, Zoltán Szállási, J. Kisistók, M. Sokač, R. Salgado, M. Dióssy, J. Demeulemeester, Abigail Bunkum, Angela Dwornik, Alastair Magness, Andrew J. Rowan, Angeliki Karamani, A. Toncheva, B. Chain, C. Castignani, Chris Bailey, C. Abbosh, C. Puttick, C. Weeden, Claudia Lee, Corentin Richard, C. Naceur-Lombardelli, D. Pearce, D. Karagianni, D. Biswas, D. Levi, E. Larose Cadieux, E. Lim, Emma C Colliver, E. Nye, F. Gálvez-Cancino, F. Gimeno-Valiente, G. Kassiotis, Georgia Stavrou, Gerasimos Mastrokalos, Helen L Lowe, Ignacio Matos, I. Noorani, J. Goldman, J. Reading, J. Rane, J. Nicod, John A. Hartley, K. Peggs, Katey S. S. Enfield, Kayalvizhi Selvaraju, K. Thol, Kevin W. Ng, Kezhong Chen, K. Dijkstra, Kristiana Grigoriadis, Krupa Thakkar, L. Ensell, Mansi Shah, Maria Litovchenko, M. Jamal-Hanjani, Mariana Werner Sunderland, M. Huska, M. Hill, M. Dietzen, Michelle Leung, M. Escudero, M. Tanić, Monica Sivakumar, O. Chervova, Olivia Lucas, O. Pich, Othman Al-Sawaf, Paulina Prymas, Philip S Hobson, Piotr Pawlik, R. Stone, R. Bentham, Roberto Vendramin, S. Saghafinia, Samuel Gamble, S. Veeriah, S. Ung, Sergio A Quezada, Sharon P. Vanloo, S. Hessey, S. Ward, Sian Harries, S. Boeing, Stephan Beck, S. Bola, Takahiro Karasaki, Tamara Denner, T. Marafioti, Thomas P. Jones, V. Spanswick, Vittorio Barbè, Wei-Ting Lu, Wing Kin Liu, Yin Wu, Yutaka Naito, Zoe Ramsden, C. Veiga, Gary Royle, C. Collins-Fekete, Francesco Fraioli, Paul Ashford, Martin D. Forster, Siow-Ming Lee, E. Borg, M. Falzon, D. Papadatos-Pastos, James Wilson, T. Ahmad, A. Procter, Asia Ahmed, Magali N. Taylor, Arjun Nair, David S. Lawrence, D. Patrini, Neal Navani, R. Thakrar, Sam M. Janes, Emilie Martinoni Hoogenboom, Fleur Monk, James W. Holding, Junaid Choudhary, Kunal Bhakhri, M. Scarci, Pat Gorman, Reena Khiroya, Robert C. M. Stephens, Y. Wong, Zoltán Káplár, S. Bandula, A. Hackshaw, A. Hacker, A. Sharp, Sean Smith, Harjot Kaur Dhanda, Camilla Pilotti, R. Leslie, Anca-Ioana Grapa, Hanyun Zhang, K. AbdulJabbar, Xiaoxi Pan, Yinyin Yuan, David Chuter, M. Mackenzie, S. Chee, A. Alzetani, Judith Cave, J. Richards, Eric Lim, P. De Sousa, Simon Jordan, Alex Rice, H. Raubenheimer, H. Bhayani, L. Ambrose, A. Devaraj, Hema Chavan, S. Begum, Silviu I. Buderi, Daniel Kaniu, Mpho Malima, S. Booth, Andrew G. Nicholson, Nádia Fernandes, Pratibha Shah, C. Proli, M. Hewish, S. Danson, M. Shackcloth, Lily Robinson, P. Russell, K. Blyth, A. Kidd, C. Dick, J. Le Quesne, Alan Kirk, Mohd. Asif, Rocco Bilancia, N. Kostoulas, Mathew Thomas, R. Hynds, N. Kanu, S. Zaccaria, E. Grönroos, C. Swanton
          </td>
          <td>2024-06-13</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>81</td>
        </tr>

        <tr id="Zygotic genome activation (ZGA) is a pivotal event in mammalian embryogenesis, marking the transition from maternal to zygotic control of development. During the ZGA process that is characterized by the intricate cascade of gene expression, who tipped the first domino in a meticulously arranged sequence is a subject of paramount interest. Recently, Dux, Obox and Nr5a2 were identified as pioneer transcription factors that reside at the top of transcriptional hierarchy. Through co-option of retrotransposon elements as hubs for transcriptional activation, these pioneer transcription factors rewire the gene regulatory network, thus initiating ZGA. In this review, we provide a snapshot of the mechanisms underlying the functions of these pioneer transcription factors. We propose that ZGA is the starting point where the embryo’s own genome begins to influence development trajectory, therefore in-depth dissecting the functions of pioneer transcription factors during ZGA will form a cornerstone of our understanding for early embryonic development, which will pave the way for advancing our grasp of mammalian developmental biology and optimizing in vitro production (IVP) techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3af6666c5700c3e4caca24092faa96fc80ddc625" target='_blank'>
              Pioneer Transcription Factors: The First Domino in Zygotic Genome Activation
              </a>
            </td>
          <td>
            B. Fu, Hong Ma, Di Liu
          </td>
          <td>2024-06-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic highly polyploid macronucleus is lost. The formation of a new macronucleus involves Programmed Genome Rearrangements, introducing DNA DSBs at approximately 45,000 loci. P. tetraurelia employs a Non-Homologous End Joining (NHEJ)-related mechanism for the systematic repair of these DSBs. Four genes encoding DNA polymerases of family X are present in the genome, one of which was found recently to colocalize with other proteins of NHEJ. The question arises as to how they make almost no error. Here we show that these enzymes are most similar to metazoan DNA polymerase λ and exhibit high fidelity through two different molecular mechanisms. Using X-ray structure determination of polymerase lambda mutants recapitulating sequence determinants of P. tetraurelia PolXs, we find both a local conformational change that involves exchanging partners in a crucial salt bridge in the active site upon binding of correct dNTPs, and a larger conformational change involving the closure of Loop3. This stabilizes the template DNA in the active site, only in the presence of the correct incoming dNTP. Differences with human pol λ and pol β are discussed. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a2c8e3978ae53a388430b008c681d13e31a282" target='_blank'>
              Mechanisms Ensuring Fidelity of Family X DNA Polymerases in Programmed DNA rearrangements in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Antonin Nourisson, Sophia Missoury, A. Haouz, Marc Delarue
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Senescence —the endpoint of replicative lifespan for normal cells— is established via a complex sequence of molecular events. One such event is the dramatic reorganization of CTCF into senescence-induced clusters (SICCs). However, the molecular determinants, genomic consequences, and functional purpose of SICCs remained unknown. Here, we combine functional assays, super-resolution imaging, and 3D genomics with computational modelling to dissect SICC emergence. We establish that the competition between CTCF-bound and non-bound loci dictates clustering propensity. Upon senescence entry, cells repurpose SRRM2 —a key component of nuclear speckles— and BANF1 —a ‘molecular glue’ for chromosomes— to cluster CTCF and rewire genome architecture. This CTCF-centric reorganization in reference to nuclear speckles functionally sustains the senescence splicing program, as SICC disruption fully reverts alternative splicing patterns. We therefore uncover a new paradigm, whereby cells translate changes in nuclear biochemistry into architectural changes directing splicing choices so as to commit to the fate of senescence. GRAPHICAL ABSTRACT HIGHLIGHTS HMGB2-bound loci compete with CTCF-bound ones for nuclear speckle association Senescent cells repurpose SRRM2 and BANF1 to cluster CTCF on speckles BANF1 is essential, but not sufficient for CTCF clustering The SRRM2 RNA-binding domain directs CTCF clustering SICCs rewire chromatin positioning to sustain the senescence splicing program">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8d2e2f24883e3f53705549d5245cfd1c159e708" target='_blank'>
              Senescent cells cluster CTCF on nuclear speckles to sustain their splicing program
              </a>
            </td>
          <td>
            Spiros Palikyras, Vassiliki Varamogiani-Mamatsi, Yajie Zhu, Shyam Ramasamy, Athanasia Mizi, Isabel Liebermann, Athanasia Stavropoulou, Ioanna Papadionysiou, Deniz Bartsch, Yulia Kargapolova, K. Sofiadis, Christoforos Nikolaou, Leo Kurian, A. M. Oudelaar, Mariano Barbieri, A. Papantonis
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The myeloma overexpressed gene (MYEOV) has been proposed to be a proto-oncogene due to high RNA transcript levels found in multiple cancers, including myeloma, breast, lung, pancreas and esophageal cancer. The presence of an open reading frame (ORF) in humans and other primates suggests protein-coding potential. Yet, we still lack evidence of a functional MYEOV protein. It remains undetermined how MYEOV overexpression affects cancerous tissues. In this work, we show that MYEOV has likely originated and may still function as an enhancer, regulating CCND1 and LTO1. Firstly, MYEOV 3′ enhancer activity was confirmed in humans using publicly available ATAC-STARR-seq data, performed on B-cell-derived GM12878 cells. We detected enhancer histone marks H3K4me1 and H3K27ac overlapping MYEOV in multiple healthy human tissues, which include B cells, liver and lung tissue. The analysis of 3D genome datasets revealed chromatin interactions between a MYEOV-3′-putative enhancer and the proto-oncogene CCND1. BLAST searches and multi-sequence alignment results showed that DNA sequence from this human enhancer element is conserved from the amphibians/amniotes divergence, with a 273 bp conserved region also found in all mammals, and even in chickens, where it is consistently located near the corresponding CCND1 orthologues. Furthermore, we observed conservation of an active enhancer state in the MYEOV orthologues of four non-human primates, dogs, rats, and mice. When studying this homologous region in mice, where the ORF of MYEOV is absent, we not only observed an enhancer chromatin state but also found interactions between the mouse enhancer homolog and Ccnd1 using 3D-genome interaction data. This is similar to the interaction observed in humans and, interestingly, coincides with CTCF binding sites in both species. Taken together, this suggests that MYEOV is a primate-specific gene with a de novo ORF that originated at an evolutionarily older enhancer region. This deeply conserved putative enhancer element could regulate CCND1 in both humans and mice, opening the possibility of studying MYEOV regulatory functions in cancer using non-primate animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f17ce82b4897c471d6bebe571f550d964a45b5c" target='_blank'>
              Evolutionarily conserved enhancer-associated features within the MYEOV locus suggest a regulatory role for this non-coding DNA region in cancer
              </a>
            </td>
          <td>
            Brigid S. A. Davidson, Juliana Estefanía Arcila-Galvis, Marco Trevisan-Herraz, Aneta Mikulasova, C. Brackley, Lisa J Russell, Daniel Rico
          </td>
          <td>2024-07-30</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="MYC has been implicated in the pathogenesis of a wide range of human tumors and has been described for many years as a transcription factor that regulates genes with pleiotropic functions to promote tumorigenic growth. However, despite extensive efforts to identify specific target genes of MYC that alone could be responsible for promoting tumorigenesis, the field is yet to reach a consensus whether this is the crucial function of MYC. Recent work shifts the view on MYC's function from being a gene-specific transcription factor to an essential stress resilience factor. In highly proliferating cells, MYC preserves cell integrity by promoting DNA repair at core promoters, protecting stalled replication forks, and/or preventing transcription-replication conflicts. Furthermore, an increasing body of evidence demonstrates that MYC not only promotes tumorigenesis by driving cell-autonomous growth, but also enables tumors to evade the host's immune system. In this review, we summarize our current understanding of how MYC impairs antitumor immunity and why this function is evolutionarily hard-wired to the biology of the MYC protein family. We show why the cell-autonomous and immune evasive functions of MYC are mutually dependent and discuss ways to target MYC proteins in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877f0298df11cd9063e84174a42a5eb6afca2c71" target='_blank'>
              Immune evasion: An imperative and consequence of MYC deregulation.
              </a>
            </td>
          <td>
            Bastian Krenz, Jongkuen Lee, Toshitha Kannan, Martin Eilers
          </td>
          <td>2024-07-02</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c0385fdd940c7e19aa6d1ee44841fd40da2686" target='_blank'>
              Topoisomerase I is an evolutionarily conserved key regulator for satellite DNA transcription
              </a>
            </td>
          <td>
            Zhen Teng, Lu Yang, Qian Zhang, Yujue Chen, Xianfeng Wang, Yiran Zheng, Aiguo Tian, Di Tian, Zhen Lin, Wu-Min Deng, Hong Liu
          </td>
          <td>2024-06-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/795e6a16077e23dab48be945dcd0a59b697bb23e" target='_blank'>
              Mechanism of ERBB2 gene overexpression by the formation of super-enhancer with genomic structural abnormalities in lung adenocarcinoma without clinically actionable genetic alterations
              </a>
            </td>
          <td>
            S. Kaneko, Ken Takasawa, Ken Asada, K. Shiraishi, Noriko Ikawa, Hidenori Machino, Norio Shinkai, M. Matsuda, Mari Masuda, Shungo Adachi, Satoshi Takahashi, Kazuma Kobayashi, Nobuji Kouno, Amina Bolatkan, M. Komatsu, M. Yamada, M. Miyake, Hirokazu Watanabe, A. Tateishi, T. Mizuno, Y. Okubo, Masami Mukai, Tatsuya Yoshida, Yukihiro Yoshida, H. Horinouchi, Shun-ichi Watanabe, Y. Ohe, Yasushi Yatabe, V. Saloura, Takashi Kohno, R. Hamamoto
          </td>
          <td>2024-06-11</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="The seven human APOBEC3 (hA3) genes encode polynucleotide cytidine deaminases that play vital roles in restricting replication of viruses and retrotransposons. However, off-target A3 deamination of the cellular genome is a major source of somatic mutations in human cancer. The ability to study A3 biology in vivo is hindered by the fact that the solitary murine Apobec3 gene (mA3) encodes a cytoplasmic enzyme, with no apparent mutagenic activity. Transgenic expression of individual hA3 genes in mice has helped to confirm their oncogenic potential but important questions including which hA3 genes are active in different tissue contexts and how they function in concert when under control of their cognate promoters cannot be addressed using these models. Here we describe humanization of the mouse mA3 locus by integration of a modified BAC clone encompassing the entire 7-gene hA3 locus from human chromosome 22 replacing mA3 on mouse chromosome 15. APOBEC3 mice are viable and fertile and hA3 gene expression in cells and tissues correlates strongly with expression in corresponding human cells and tissues, indicating human-like regulation of hA3 gene expression in the mice. Splenocytes from this line display a functional human A3 response to Friend Murine Leukaemia Virus (F-MLV) infection. We propose that the Hs-APOBEC3 mouse will uniquely model the function of the complete hA3 locus in a living organism and that it will serve as a useful background upon which to model human cancer, as well as assisting drug discovery efforts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8034f8597ff29dbab41cdb895f637bb191838ad2" target='_blank'>
              Human-like APOBEC3 gene expression and anti-viral responses following replacement of mouse Apobec3 with the 7-gene human APOBEC3 locus
              </a>
            </td>
          <td>
            Nerissa K. Kirkwood, H. Prosser, Melvyn W Yap, Jane Gibson, Ross Cook, Ian Reddin, Ane Stranger, Emma Howes, Nur Zainal, M. Periyasamy, S. Conticello, Gareth J. Thomas, James Scott, Kate N. Bishop, Simak Ali, Allan Bradley, Tim R Fenton
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Non-coding regions of the human genome are important for functional regulations, but their mechanisms remain elusive. We used machine learning to guide a CRISPR screening on hubs (i.e. non-coding loci forming many 3D contacts) and significantly increased the discovery rate of hubs essential for cell growth. We found no clear genetic or epigenetic differences between essential and nonessential hubs, but we observed that some neighboring hubs in the linear genome have distinct spatial contacts and opposite effects on cell growth. One such pair in an epigenetically quiescent region showed different impacts on gene expression, chromatin accessibility and chromatin organization. We also found that deleting the essential hub altered the genetic network activity and increased the entropy of chromatin accessibility, more severe than that caused by deletion of the nonessential hub, suggesting that they are critical for maintaining an ordered chromatin structure. Our study reveals new insights into the system-level roles of non-coding regions in the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37fecffd504fcf6b86c79eeec8f4d7ba8dc1c39" target='_blank'>
              Differential impact of quiescent non-coding loci on chromatin entropy.
              </a>
            </td>
          <td>
            Peiyao Wu, Mina Yao, Wei Wang
          </td>
          <td>2024-06-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29e5e59bd894b6791e1caba61c558e842d1e8aab" target='_blank'>
              Mitoferrin2 is a synthetic lethal target for chromosome 8p deleted cancers
              </a>
            </td>
          <td>
            Stephan Krieg, Thomas Rohde, Tobias Rausch, Luise Butthof, Lena Wendler-Link, Christoph Eckert, Kai Breuhahn, Bruno Galy, J. Korbel, Maximilian Billmann, Marco Breinig, Darjus F. Tschaharganeh
          </td>
          <td>2024-06-17</td>
          <td>Genome Medicine</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="Recurrent mutations in serine / arginine-rich splicing factor 2 (SRSF2), particularly the proline-to-histidine substitution at position 95 (P95H), have been proposed to drive neoplastic diseases. SRSF2 plays pivotal roles in pre-mRNA processing and gene transcription. However, the precise impact of these diverse functions of SRSF2 on cancer cell behaviour remains unclear. Here, we show that deletion or homozygous P95H mutation of SRSF2 both cause extensive DNA damage leading to cell cycle arrest in vitro and in vivo, regardless of cell genotype. We mechanistically demonstrate that SRSF2 is required for efficient bi-directional transcription of DNA replication and repair genes, independent of its function in splicing. In contrast, SRSF2 haploinsufficiency induces DNA damage without halting the cell cycle in cancer cells. Inducing the heterozygous Srsf2 P95H mutation leads to clonal expansion of epidermal cells in mouse skin, but tumor formation is inhibited following exposure to carcinogens. To survive carcinogen treatment, cells containing the Srsf2 P95H mutation undergo substantial transcription rewiring that restores bi-directional gene expression. Thus, our study reveals crucial roles of SRSF2 beyond splicing and underscores its importance in regulating transcription to orchestrate the cell cycle along with the DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32907ead7270689c743bba801c421a14be049a66" target='_blank'>
              SRSF2 transcriptional function maintains genome integrity during cell division
              </a>
            </td>
          <td>
            Rebecca E. Wagner, Leonie Arnetzl, Thiago Britto-Borges, Anke Heit-Mondrzyk, Ali Bakr, Etienne Sollier, Jasper Panten, Sylvain Delaunay, Daniela Sohn, P. Schmezer, D. T. Odom, Karin Müller-Decker, C. Plass, Christoph Dieterich, P. Lutsik, Susanne Bornelöv, Michaela Frye
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Fusion-positive rhabdomyosarcoma is an aggressive pediatric cancer molecularly characterized by arrested myogenesis. The defining genetic driver, PAX3::FOXO1, functions as a chimeric gain-of-function transcription factor. An incomplete understanding of PAX3::FOXO1’s in vivo epigenetic mechanisms has hindered therapeutic development. Here, we establish a PAX3::FOXO1 zebrafish injection model and semi-automated ChIP-seq normalization strategy to evaluate how PAX3::FOXO1 initially interfaces with chromatin in a developmental context. We investigated PAX3::FOXO1’s recognition of chromatin and subsequent transcriptional consequences. We find that PAX3::FOXO1 interacts with inaccessible chromatin through partial/homeobox motif recognition consistent with pioneering activity. However, PAX3::FOXO1-genome binding through a composite paired-box/homeobox motif alters chromatin accessibility and redistributes H3K27ac to activate neural transcriptional programs. We uncover neural signatures that are highly representative of clinical rhabdomyosarcoma gene expression programs that are enriched following chemotherapy. Overall, we identify partial/homeobox motif recognition as a new mode for PAX3::FOXO1 pioneer function and identify neural signatures as a potentially critical PAX3::FOXO1 tumor initiation event.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c34ac92eb278acb6b0ddaf9def7a0af29156e864" target='_blank'>
              Rhabdomyosarcoma fusion oncoprotein initially pioneers a neural signature in vivo
              </a>
            </td>
          <td>
            Jack Kucinski, Alexi Tallan, C. Taslim, Meng Wang, Matthew V. Cannon, Katherine M. Silvius, Benjamin Z. Stanton, Genevieve C. Kendall
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Maintenance of genome stability is of paramount importance for the survival of an organism. However, genomic integrity is constantly being challenged by various endogenous and exogenous processes that damage DNA. Therefore, cells are heavily reliant on DNA repair pathways that have evolved to deal with every type of genotoxic insult that threatens to compromise genome stability. Notably, inherited mutations in genes encoding proteins involved in these protective pathways trigger the onset of disease that is driven by chromosome instability e.g. neurodevelopmental abnormalities, neurodegeneration, premature ageing, immunodeficiency and cancer development. The ability of cells to regulate the recruitment of specific DNA repair proteins to sites of DNA damage is extremely complex but is primarily mediated by protein post-translational modifications (PTMs). Ubiquitylation is one such PTM, which controls genome stability by regulating protein localisation, protein turnover, protein-protein interactions and intra-cellular signalling. Over the past two decades, numerous ubiquitin (Ub) E3 ligases have been identified to play a crucial role not only in the initiation of DNA replication and DNA damage repair but also in the efficient termination of these processes. In this review, we discuss our current understanding of how different Ub E3 ligases (RNF168, TRAIP, HUWE1, TRIP12, FANCL, BRCA1, RFWD3) function to regulate DNA repair and replication and the pathological consequences arising from inheriting deleterious mutations that compromise the Ub-dependent DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d67379a570ccc6355d856b2e241bb15b1ce2e1c" target='_blank'>
              E3 ligases: a ubiquitous link between DNA repair, DNA replication and human disease.
              </a>
            </td>
          <td>
            Anoop S. Chauhan, Satpal S. Jhujh, Grant S Stewart
          </td>
          <td>2024-07-10</td>
          <td>The Biochemical journal</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background Lettuce, an important member of the Asteraceae family, is a globally cultivated cash vegetable crop. With a highly complex genome (∼2.5 Gb; 2n = 18) rich in repeat sequences, current lettuce reference genomes exhibit thousands of gaps, impeding a comprehensive understanding of the lettuce genome. Findings Here, we present a near-complete gapless reference genome for cutting lettuce with high transformability, using long-read PacBio HiFi and Nanopore sequencing data. In comparison to stem lettuce genome, we identify 127,681 structural variations (SVs, present in 0.41 Gb of sequence), reflecting the divergence of leafy and stem lettuce. Interestingly, these SVs are related to transposons and DNA methylation states. Furthermore, we identify 4,612 whole-genome triplication genes exhibiting high expression levels associated with low DNA methylation levels and high N6-methyladenosine RNA modifications. DNA methylation changes are also associated with activation of genes involved in callus formation. Conclusions Our gapless lettuce genome assembly, an unprecedented achievement in the Asteraceae family, establishes a solid foundation for functional genomics, epigenomics, and crop breeding and sheds new light on understanding the complexity of gene regulation associated with the dynamics of DNA and RNA epigenetics in genome evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21297faeee6c28457c9d2a7c7e63a412f5693a42" target='_blank'>
              Gapless genome assembly and epigenetic profiles reveal gene regulation of whole-genome triplication in lettuce
              </a>
            </td>
          <td>
            Shuai Cao, Nunchanoke Sawettalake, Lisha Shen
          </td>
          <td>2024-01-02</td>
          <td>GigaScience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ddfe88dbc57ad9cb3e53a1a082129fee82af037" target='_blank'>
              Determinants of p53 DNA binding, gene regulation, and cell fate decisions
              </a>
            </td>
          <td>
            Martin Fischer, Morgan A. Sammons
          </td>
          <td>2024-06-29</td>
          <td>Cell Death and Differentiation</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4935f35210bf40b045d154a339912e6ddb86073" target='_blank'>
              Decoding high mobility group A2 protein expression regulation and implications in human cancers
              </a>
            </td>
          <td>
            Farah Khazem, A. B. Zetoune
          </td>
          <td>2024-07-31</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe60321069cd8bdd4dcd7dd1f6cb9aee5e0996c" target='_blank'>
              Z-DNA formation in promoters conserved between human and mouse are associated with increased transcription reinitiation rates
              </a>
            </td>
          <td>
            N. Beknazarov, Dmitry Konovalov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-08-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="
 Aneuploidy – loss or gain of whole chromosomes or chromosome arms, is rare and poorly tolerated in normal cells but occurs in ∼90% of solid tumors; however, the mechanisms through which specific aneuploidies affect cancer development are unclear. Additionally, generating mammalian models of specific chromosome arm alterations is technically difficult, limiting further study. Cancers have tumor, cell, and tissue type-specific patterns of chromosome arm copy-number alterations that influence tumor evolution and sensitivity to anti-cancer therapies. Squamous cell carcinomas (SCCs) affecting lung, head and neck, esophageal, and cervical tissues have been shown to feature conserved early losses and gains of arms 3p and 3q, respectively, and are associated with relatively few oncogenic drivers and treatment options. These aneuploidies are particularly well established in lung SCC and have been shown to promote tumorigenesis, metastasis, and poor prognosis. To study cancer vulnerabilities associated with chromosome 3 arm aneuploidies, we used CRISPR-Cas9 to delete a 3p copy in human immortalized lung epithelial cell lines—the lung SCC cell-of-origin. A subset of clones duplicated a wild type chromosome 3 copy, transitioning to 3q gain and providing us with independent models for chromosome 3 wild type, 3p loss, and 3q gain. Next, we performed a CRISPRi screen and found that 3q gain cells are more sensitive to knockdown of sterol regulatory element-binding factor 1 (SREBF1)—a master regulator of cholesterol and fatty acid biosynthesis, its co-factor SREBP cleavage activating protein (SCAP), and downstream target HMG-CoA reductase (HMGCR)—the rate-limiting enzyme for cholesterol synthesis. 3q gain cells also showed up-regulated fatty acid metabolites, ceramides, triglycerides, and phosphatidylinositols. After performing a chemical screen, 3q gain cells showed reduced proliferation, increased apoptosis, and DNA damage in response to statins (HMGCR inhibitors). Statins have previously been shown to induce tumor-specific apoptosis and reduced cancer risk, although these studies have associated with variable efficacy and underlying mechanisms. We hypothesize that 3q gain is a biomarker of statin response and sensitivity in squamous cancers. We aim to study the mechanism through which aneuploidy alters lipid metabolism and contributes to cancer. Investigating aneuploidy-induced vulnerabilities will help us understand how cancers exploit aneuploidies to promote tumorigenesis and to elucidate novel and specific therapeutic targets for cancers with shared copy-number profiles.
 Citation Format: Nadja Zhakula-Kostadinova, Sejal Jain, Laura Byron, Matthew L. Meyerson, Alison M. Taylor. Investigating vulnerabilities associated with chromosome arm aneuploidy in cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR016.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ca81f5571b9f264386412ab8dcfed6cabd96466" target='_blank'>
              Abstract PR016: Investigating vulnerabilities associated with chromosome arm aneuploidy in cancer
              </a>
            </td>
          <td>
            Nadja Zhakula-Kostadinova, Sejal Jain, Laura Byron, Matthew L. Meyerson, Alison M. Taylor
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer is the most common type of cancer globally and presents an escalating problem and a huge burden on societies. Several strategies are implemented in clinics to treat patients and prevent disease incidence. Efforts to understand the underlying causes of disease emergence are pivotal, and the latest examination of human transcriptomic studies showed the involvement of the noncoding RNA regulatory molecules in influencing both pathological and physiological conditions. Several molecular mechanisms are involved in the process and collaborate to develop tumor plasticity and drug resistance. In this review, we highlight for the first time the interplay between long noncoding RNAs and Aurora kinases in breast cancer and review the latest advances in the field in an attempt to pave the way for a better understanding of the course of the disease and to delineate the targets for treatment strategies in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e548fe1625d3f8fbf15ae302f7d6d3f128c4eaf7" target='_blank'>
              Deciphering the Complex Interplay of Long Noncoding RNAs and Aurora Kinases: Novel Insights into Breast Cancer Development and Therapeutic Strategies
              </a>
            </td>
          <td>
            M. K. Saadeldin, G. Curigliano, A. K. Abdel-Aziz
          </td>
          <td>2024-07-30</td>
          <td>Future Pharmacology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/112c0ed37a6197df232678cb308cfb7c6f524b6b" target='_blank'>
              Mechanistic analysis of enhancer sequences in the estrogen receptor transcriptional program
              </a>
            </td>
          <td>
            Shayan Tabe-Bordbar, Y. Song, Bryan J Lunt, Zahra Alavi, K. Prasanth, Saurabh Sinha
          </td>
          <td>2024-06-11</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Bacterial chromosomes are large molecules that need to be highly compacted to fit inside the cells. Chromosome compaction must facilitate and maintain key biological processes such as gene expression and DNA transactions (replication, recombination, repair, and segregation). Chromosome and chromatin 3D-organization in bacteria has been a puzzle for decades. Chromosome conformation capture coupled to deep sequencing (Hi-C) in combination with other "omics" approaches has allowed dissection of the structural layers that shape bacterial chromosome organization, from DNA topology to global chromosome architecture. Here we review the latest findings using Hi-C and discuss the main features of bacterial genome folding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d47cf5eae543114153f80223459365d3c29d31d5" target='_blank'>
              Principles of bacterial genome organization, a conformational point of view.
              </a>
            </td>
          <td>
            Sokrich Ponndara, Mounia Kortebi, F. Boccard, S. Bury-Moné, Virginia S. Lioy
          </td>
          <td>2024-06-22</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="In this study, Chavan et al. investigated novel regulators of nuclear architecture in Drosophila cells and show that the heterochromatin-associated transcription factor Stonewall anchors chromatin to the nuclear periphery and is associated with transcriptional repression. Their findings provide new insights into the mechanism by which Stwl acts in germline stem cell maintenance and ovary homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b98cccf6982181e50a374fdb024592185dad2711" target='_blank'>
              A nuclear architecture screen in Drosophila identifies Stonewall as a link between chromatin position at the nuclear periphery and germline stem cell fate
              </a>
            </td>
          <td>
            Ankita Chavan, Randi L Isenhart, Son C. Nguyen, Noor M Kotb, Jailynn Harke, Anna Sintsova, Gulay Ulukaya, Federico Uliana, C. Ashiono, Ulrike Kutay, Gianluca Pegoraro, P. Rangan, Eric F Joyce, Madhav Jagannathan
          </td>
          <td>2024-05-01</td>
          <td>Genes & Development</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The centromeric protein-A (CENP-A) is an evolutionary conserved histone H3 variant that marks the identity of the centromeres. Several mechanisms regulate the centromeric deposition of CENP-A as its mislocalization causes erroneous chromosome segregation, leading to aneuploidy-based diseases, including cancers. The most crucial deposition factor is a CENP-A specific chaperone, HJURP (Scm3 in budding yeast), which specifically binds to CENP-A. However, the discovery of HJURP as a DDR (DNA damage repair) protein and evidence of its binding to Holliday junctions in vitro indicate a CENP-A-deposition-independent role of these chaperones. In this study, using budding yeast, we demonstrate that Scm3 is crucial for the DDR pathway as scm3 cells are sensitive to DNA damage. We further observe that the scm3 mutant interacts with the rad52 DDR mutant and is compromised in activating DDR-mediated arrest. We demonstrate that Scm3 associates with the DNA damage sites and undergoes posttranslational modifications upon DNA damage. Overall, from this report and earlier studies on HJURP, we conclude that DDR functions of CENP-A chaperones are conserved across eukaryotes. Thus, the revelation that these chaperones confer genome stability in more than one pathway has clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20b8f002bffe33b7b0eebb870a6636324b784a0d" target='_blank'>
              Evidence of centromeric histone 3 chaperone involved in DNA damage repair pathway
              </a>
            </td>
          <td>
            Prakhar Agarwal, S. Ghosh
          </td>
          <td>2024-07-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The incorporation of histone variants has structural ramifications on nucleosome dynamics and stability. Due to their unique sequences, histone variants can alter histone–histone or histone–DNA interactions, impacting the folding of DNA around the histone octamer and the overall higher-order structure of chromatin fibers. These structural modifications alter chromatin compaction and accessibility of DNA by transcription factors and other regulatory proteins to influence gene regulatory processes such as DNA damage and repair, as well as transcriptional activation or repression. Histone variants can also generate a unique interactome composed of histone chaperones and chromatin remodeling complexes. Any of these perturbations can contribute to cellular plasticity and the progression of human diseases. Here, we focus on a frequently overlooked group of histone variants lying within the four human histone gene clusters and their contribution to breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57fb7bdaf615df97d7b072d04e7a749c2e1b4c7b" target='_blank'>
              Beyond the Usual Suspects: Examining the Role of Understudied Histone Variants in Breast Cancer
              </a>
            </td>
          <td>
            Hejer Dhahri, Wesley N. Saintilnord, Darrell Chandler, Yvonne N. Fondufe-Mittendorf
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Male sex chromosome aneuploidies are frequent genetic aberrations in humans characterized by additional Y or X chromosome complements. Jacob (JS) and Klinefelter syndromes (KS), carrying 47,XYY and 47,XXY chromosomes, respectively, share several clinical features, including sterility, hormonal deficits, neurocognitive delay, and skeletal-muscle defects. Despite the high incidence, a complete understanding of the molecular mechanisms underlying the clinical manifestations in sex aneuploid patients is still elusive. In this study, we generated and characterized the inaugural cohort of 47,XYY human induced pluripotent stem cells (iPSCs). We performed a comprehensive transcriptional analysis, including JS primary fibroblasts, iPSCs, and neural stem cells (NSCs) alongside KS cells. Our study revealed a transcriptional feedback mechanism tuning non-PAR X chromosome genes (NPX) homologs in Y supernumerary cells, a phenomenon not detected in X aneuploid iPSCs. By ectopically modulating the expression of selected NPY genes, we demonstrated a direct transcriptional link between the UTY-UTX gene pair. Furthermore, our analyses identified a shared transcriptomic signature between JS and KS, discernible already at the iPSC stage, with a notable enrichment for processes related to neurological functions. This transcriptomic convergence underscores potential commonalities in the molecular pathways underpinning the pathophysiology of male sex chromosome aneuploidies. Finally, through genome-wide DNA methylation profiling of JS iPSCs, we demonstrated that a supernumerary Y chromosome only minimally impacts the methylation status of 47,XYY cells at the pluripotent stage. Our work reveals critical transcriptional feedback mechanisms and shared gene expression signatures in male sex chromosome aneuploidies, paving the way for a better understanding of their common phenotypic features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4a1e46b6ba1f6258afb3db5dff0a58811a485a6" target='_blank'>
              An iPSC-based model of Jacob Syndrome reveals a DNA methylation-independent transcriptional dysregulation shared with X aneuploid cells
              </a>
            </td>
          <td>
            V. Astro, K. Cardona-Londoño, L. V. Cortés-Medina, R. Alghamdi, G. Ramírez-Calderón, F. Kefalas, J. Dilmé-Capó, S. Radío, A. Adamo
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In geminiviruses belonging to the genus Begomovirus, coat protein (CP) expression depends on viral AL2 protein, which derepresses and activates the CP promoter through sequence elements that lie within the viral intergenic region (IR). However, AL2 does not exhibit sequence-specific DNA binding activity but is instead directed to responsive promoters through interactions with host factors, most likely transcriptional activators and/or repressors. In this study, we describe a repressive plant-specific transcription factor, Arabidopsis thaliana TCP24 (AtTCP24), that interacts with AL2 and recognizes a class II TCP binding site in the CP promoter (GTGGTCCC). This motif corresponds to the previously identified conserved late element (CLE). We also report that histone 3 lysine 27 trimethylation (H3K27me3), an epigenetic mark associated with facultative repression, is enriched over the viral IR. H3K27me3 is deposited by Polycomb Repressive Complex 2 (PRC2), a critical regulator of gene expression and development in plants and animals. Remarkably, mutation of the TCP24 binding site (the CLE) in tomato golden mosaic virus (TGMV) and cabbage leaf curl virus (CaLCuV) CP promoters greatly diminishes H3K27me3 levels on viral chromatin and causes a dramatic delay and attenuation of disease symptoms in infected Arabidopsis and Nicotiana benthamiana plants. Symptom remission is accompanied by decreased viral DNA levels in systemically infected tissue. Nevertheless, in transient replication assays CLE mutation delays but does not limit the accumulation of viral double-stranded DNA, although single-stranded DNA and CP mRNA levels are decreased. These findings suggest that TCP24 binding to the CLE leads to CP promoter repression and H3K27me3 deposition, while TCP24-AL2 interaction may recruit AL2 to derepress and activate the promoter. Thus, a repressive host transcription factor may be repurposed to target a viral factor essential for promoter activity. The presence of the CLE in many begomoviruses suggests a common scheme for late promoter regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ddd53cac15774ed5f120c8687e7a74de7cf3b7" target='_blank'>
              Mutation of the conserved late element in geminivirus CP promoters abolishes Arabidopsis TCP24 transcription factor binding and decreases H3K27me3 levels on viral chromatin
              </a>
            </td>
          <td>
            Jacqueline Williams, Elizabeth Regedanz, Natalia Lucinda, Alba Ruth Nava Fereira, Gabriela Lacatus, Mary Berger, Nels O'Connell, Tami L. Coursey, Jianhua Ruan, D. Bisaro, G. Sunter
          </td>
          <td>2024-07-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f61a46af21f4bf12f71e8a9669384dafe3da1b23" target='_blank'>
              CpG island turnover events predict evolutionary changes in enhancer activity
              </a>
            </td>
          <td>
            Acadia A. Kocher, E. Dutrow, Severin Uebbing, Kristina M. Yim, María F. Rosales Larios, Marybeth Baumgartner, Timothy Nottoli, J. Noonan
          </td>
          <td>2024-06-13</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="The intricate genetic regulatory relationship between non-coding RNAs (ncRNAs) and N6-methyladenosine (m6A) modi ﬁ cation opens another exciting chapter in cancer biology. m6A, the most prevalent epigenetic modi ﬁ cation of RNA in eukaryotes, plays a signi ﬁ cant role in controlling gene expression, particularly in cancer progression. The interplay between m6A and various ncRNAs, including microRNAs, circular RNAs, and piRNAs, is crucial for understanding the molecular underpinnings of cancer. This interaction not only affects the stability and function of ncRNAs but also modulates their ability to regulate target genes. By decoding these complex relationships, researchers can identify new diagnostic and prognostic biomarkers, offering new avenues for cancer detection and treatment (Liu and Xiang)">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d92537d2e178f2591f86caf5e875459268796ce8" target='_blank'>
              Editorial: Reviews in non-coding RNA: 2023
              </a>
            </td>
          <td>
            Bodhisattwa Banerjee, Sandip Mukherjee
          </td>
          <td>2024-06-13</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Xist, a pivotal player in X chromosome inactivation (XCI), has long been perceived as a cis-acting long noncoding RNA that binds exclusively to the inactive X chromosome (Xi). However, Xist’s ability to diffuse under select circumstances has also been documented, leading us to suspect that Xist RNA may have targets and functions beyond the Xi. Here, using female mouse embryonic stem cells (ES) and mouse embryonic fibroblasts (MEF) as models, we demonstrate that Xist RNA indeed can localize beyond the Xi. However, its binding is limited to ∼100 genes in cells undergoing XCI (ES cells) and in post-XCI cells (MEFs). The target genes are diverse in function but are unified by their active chromatin status. Xist binds discretely to promoters of target genes in neighborhoods relatively depleted for Polycomb marks, contrasting with the broad, Polycomb-enriched domains reported for human XIST RNA. We find that Xist binding is associated with down-modulation of autosomal gene expression. However, unlike on the Xi, Xist binding does not lead to full silencing and also does not spread beyond the target gene. Over-expressing Xist in transgenic ES cells similarly lead to autosomal gene suppression, while deleting Xist’s Repeat B motif reduces autosomal binding and perturbs autosomal down-regulation. Furthermore, treating female ES cells with the Xist inhibitor, X1, leads to loss of autosomal suppression. Altogether, our findings reveal Xist targets ∼100 genes beyond the Xi, identify Repeat B as a crucial domain for its in-trans function in mice, and indicate that autosomal targeting can be disrupted by the X1 small molecule inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccb2a52596b1d4a0361228a895e9d27e5c7cb55" target='_blank'>
              Xist RNA binds select autosomal genes and depends on Repeat B to regulate their expression
              </a>
            </td>
          <td>
            Shengze Yao, Yesu Jeon, Barry A. Kesner, Jeannie T Lee
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 What’s the pattern of genome-wide chromatin occupancy of SCP3 during meiotic prophase, and how its interaction with chromatin is regulated?



 We document SCP3 enriched at active chromatin regions with largely inherited from leptotene to pachytene stage, which facilitated by transcription and condensate assembly in spermatocytes.



 Meiosis is a critical step during the process of gametogenesis to form haploid cells which finally develop into functional gametes. The synaptonemal complex (SC) is a meiosis-specific protein assembly which bridges homologous chromosomes to establish the architecture of meiotic chromosome. Genetic ablation of genes encoding SC components impedes meiosis and gametogenesis in mice and mutation of these genes is frequently found in infertile patients. SCP3 localization changes with chromosome morphology in autosomes and is involved in maintaining the association of sex chromosomes. Male mice with null mutation of SCP3 failed to form SCs, resulting in developmental arrest during meiotic prophase.



 This study included male mice at post-natal day 10 (P10), 4 weeks or adult (8 weeks or older) were maintained on C57BL/6J background. Spermatocytes were performed modified CUT&Tag using antibodies. ChIP-seq data was analyzed together with WGBS and ATAC-seq data.



 Spermatocytes were obtained by a discontinuous BSA density gradient. Cell pellet was resuspended in culture medium (MEM with 10% FBS) at a concentration of 105 cells/ml and cultured at 37 °C in a 5% CO2 incubator. Spermatocytes were cultured with or without 1,6-hexanediol or α-amanitin. Spermatocytes with or without drug treatment were collected at indicated time points for further examination and then modified CUT&Tag and co-IP using anti-SCP3 and anti-POL II antibodies.



 Chromatin occupancy of SCP3 in leptotene and pachytene spermatocytes, SCP3-Core sites retained upon treatment of 1,6-hexanediol which dissolves SCP3 condensate, and SCP1 occupancy at meiotic chromatin in pachytene spermatocytes. When we determined inter-peak distances and width of peaks, we observe that: (1) SCP3 occupancy in leptotene stage had greater inter-peak distances and narrower peak width than SCP3 occupancy in pachytene stage, demonstrating gradual accumulated SCP3 at chromatin during meiotic progression; (2) SCP3-Core sites have broader peak width than SCP3-NonCore sites, in agreement with stability of SCP3 occupancy at SCP3-Core sites; (3) SCP1 occupancy had greater inter-peak distances and narrower peak width than SCP3 occupancy in pachytene stage, supporting that SCP1 peaks are a subset of SCP3 peaks and SCP1’s major function in connecting LE and CE respectively. Generally, at pachytene-stage meiosis, SCP3 mainly occupies chromatin regions including transcriptionally active sites, Deu and Alu elements, while SCP1 mainly co-localizes with SCP3 at a subset of regions with open chromatin and high cohesin enrichment to connect meiotic chromosome axes with CE. In this way, a local active chromatin environment is created at meiotic chromosome axes, facilitating homologous chromosome recombination.



 Our test was not successful to examine SCP3 occupancy with as few as less than 1000 spermatocytes using our protocol. It is improvement of efficiency of tagmentation or development of more efficient antibodies against SCP3 will be helpful for clinical sample detection.



 SCP3 occupancy was largely inherited from leptotene to pachytene stage, facilitated by transcription and condensate assembly, and enriched at specific SINE repeats. Further exploration using knockout mouse models of meiotic axial proteins and epigenetic modifiers will provide valuable information on meiotic chromatin architecture and crosstalk among meiotic chromatin events.



 N/A
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd5370d474a768836582b1cb861c8c3d70c98bd" target='_blank'>
              P-045 Deciphering meiotic chromatin organization by SCP3
              </a>
            </td>
          <td>
            S. Guo
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1310d603b23100b789e20f7104c3139b43adef05" target='_blank'>
              Decoding the genetic program of micronucleus formation: linking chromosomal instability to human disease
              </a>
            </td>
          <td>
            Jörg Fahrer
          </td>
          <td>2024-06-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Precision (individualized) medicine relies on the molecular profiling of tumors’ dysregulated characteristics (genomic, epigenetic, transcriptomic) to identify the reliance on key pathways (including genome stability and epigenetic gene regulation) for viability or growth, and then utilises targeted therapeutics to disrupt these survival-dependent pathways. Non-mutational epigenetic changes alter cells’ transcriptional profile and are a key feature found in many tumors. In contrast to genetic mutations, epigenetic changes are reversable, and restoring a normal epigenetic profile can inhibit tumor growth and progression. Lysine acetyltransferases (KATs or HATs) protect genome stability and integrity, and Tip60 is an essential acetyltransferase due to its roles as an epigenetic and transcriptional regulator, and as master regulator of the DNA double-strand break response. Tip60 is commonly downregulated and mislocalized in many cancers, and the roles that mislocalized Tip60 plays in cancer are not well understood. Here we categorize and discuss Tip60-regulated genes, evaluate Tip60-interacting proteins based on cellular localization, and explore the therapeutic potential of Tip60-targeting compounds as epigenetic inhibitors. Understanding the multiple roles Tip60 plays in tumorigenesis will improve our understanding of tumor progression and will inform therapeutic options, including informing potential combinatorial regimes with current chemotherapeutics, leading to improvements in patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0558cedd75f278d6feed6575b276aad74d694cf" target='_blank'>
              Evaluating the Cellular Roles of the Lysine Acetyltransferase Tip60 in Cancer: A Multi-Action Molecular Target for Precision Oncology
              </a>
            </td>
          <td>
            Nazanin Zohourian, Erin Coll, Muiread Dever, Anna Sheahan, Petra Burns-Lane, James A. L. Brown
          </td>
          <td>2024-07-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Knowledge of locations and activities of cis-regulatory elements (CREs) is needed to decipher basic mechanisms of gene regulation and to understand the impact of genetic variants on complex traits. Previous studies identified candidate CREs (cCREs) using epigenetic features in one species, making comparisons difficult between species. In contrast, we conducted an interspecies study defining epigenetic states and identifying cCREs in blood cell types to generate regulatory maps that are comparable between species, using integrative modeling of eight epigenetic features jointly in human and mouse in our Validated Systematic Integration (VISION) Project. The resulting catalogs of cCREs are useful resources for further studies of gene regulation in blood cells, indicated by high overlap with known functional elements and strong enrichment for human genetic variants associated with blood cell phenotypes. The contribution of each epigenetic state in cCREs to gene regulation, inferred from a multivariate regression, was used to estimate epigenetic state Regulatory Potential (esRP) scores for each cCRE in each cell type, which were used to categorize dynamic changes in cCREs. Groups of cCREs displaying similar patterns of regulatory activity in human and mouse cell types, obtained by joint clustering on esRP scores, harbored distinctive transcription factor binding motifs that were similar between species. An interspecies comparison of cCREs revealed both conserved and species-specific patterns of epigenetic evolution. Finally, we showed that comparisons of the epigenetic landscape between species can reveal elements with similar roles in regulation, even in the absence of genomic sequence alignment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e8211dd3cbaad431cfef28f6397a18c06fd052" target='_blank'>
              Interspecies regulatory landscapes and elements revealed by novel joint systematic integration of human and mouse blood cell epigenomes.
              </a>
            </td>
          <td>
            Guanjue Xiang, Xi He, B. Giardine, Kathryn J. Isaac, Dylan J. Taylor, R. McCoy, Camden Jansen, C. Keller, A. Wixom, April Cockburn, Amber Miller, Qian Qi, Yanghua He, Yichao Li, Jens Lichtenberg, Elisabeth F. Heuston, Stacie M. Anderson, Jing Luan, M. Vermunt, Feng Yue, M. Sauria, Michael C. Schatz, James Taylor, B. Göttgens, Jim R. Hughes, D. Higgs, Mitchell J. Weiss, Yong Cheng, G. Blobel, D. Bodine, Yu Zhang, Qunhua Li, Shaun Mahony, R. Hardison
          </td>
          <td>2024-07-01</td>
          <td>Genome research</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="DNA methylation is an important epigenetic mechanism essential for transposon silencing and genome integrity. Across evolution, the substrates of DNA methylation have diversified between kingdoms to account for genome complexity. In plants, Chromomethylase3 (CMT3) and CMT2 are the major methyltransferases mediating CHG and CHH methylation, respectively. However, how these two enzymes diverge on substrate specificities during evolution remains unknown. Here, we reveal that CMT2 originates from a duplication of the evolutionarily more ancient CMT3 in flowering plants. Lacking a key arginine residue recognizing CHG in CMT2 impairs its CHG methylation activity in most flowering plants. An engineered V1200R mutation empowers CMT2 to restore both CHG and CHH methylation in Arabidopsis cmt2cmt3 mutant, testifying a loss-of-function effect for CMT2 after ∼200 million years of evolution. Interestingly, CMT2 has evolved a long and unstructured N-terminus critical for balancing protein stability, especially under heat stress. Furthermore, CMT2 N-terminus is plastic and can be tolerant to various natural mutations. Together, this study reveals the mechanism of chromomethylase divergence for context-specific DNA methylation in plants and sheds important lights on DNA methylation evolution and function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/864cdd9615e92b4e5f4ca76c46e050ddaffeafaf" target='_blank'>
              Substrate specificity and protein stability drive the divergence of plant-specific DNA methyltransferases
              </a>
            </td>
          <td>
            Jianjun Jiang, Jia Gwee, Jian Fang, Sarah M. Leichter, Dean Sanders, Xinrui Ji, Jikui Song, Xuehua Zhong
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="N6-methyladenosine (m6A) is the most abundant internal modification in mRNAs. Despite accumulating evidence for the profound impact of m6A on cancer biology, there are conflicting reports that alterations in genes encoding the m6A machinery proteins can either promote or suppress cancer, even in the same tumor type. Using data from The Cancer Genome Atlas, we performed a pan-cancer investigation of 15 m6A core factors in nearly 10,000 samples from 31 tumor types to reveal underlying cross-tumor patterns. Altered expression, largely driven by copy number variations at the chromosome arm level, results in the most common mode of dysregulation of these factors. YTHDF1, YTHDF2, YTHDF3, and VIRMA are the most frequently altered factors and the only ones to be uniquely altered when tumors are grouped according to the expression pattern of the m6A factors. These genes are also the only ones with coherent, pan-cancer predictive power for progression-free survival. On the contrary, METTL3, the most intensively studied m6A factor as a cancer target, shows much lower levels of alteration and no predictive power for patient survival. Therefore, we propose the non-enzymatic YTHDF and VIRMA genes as preferred subjects to dissect the role of m6A in cancer and as priority cancer targets. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c9b2f39d5f2496a2d5c8864a272f6e53cb7855" target='_blank'>
              The three YTHDF paralogs and VIRMA are the major tumor drivers among the m6A core genes in a pan-cancer analysis
              </a>
            </td>
          <td>
            E. Destefanis, Denise Sighel, Davide Dalfovo, Riccardo Gilmozzi, F. Broso, Andrea Cappannini, Janusz M. Bujnicki, A. Romanel, E. Dassi, A. Quattrone
          </td>
          <td>2024-06-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC.Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs.Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes.These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Zinc finger proteins (ZNF), a unique yet diverse group of proteins, play pivotal roles in fundamental cellular mechanisms including transcription regulation, chromatin remodeling, protein/RNA homeostasis, and DNA repair. Consequently, the mis regulation of ZNF proteins can result in a variety of human diseases, ranging from neurodevelopmental disorders to several cancers. Considering the promising results of DNA damage repair (DDR) inhibition in the clinic, as a therapeutic strategy for patients with homologous recombination (HR) deficiency, identifying other potential targetable DDR proteins as emerged vulnerabilities in resistant tumor cells is essential, especially when considering the burden of acquired drug resistance. Importantly, there are a growing number of studies identifying new ZNFs and revealing their significance in several DDR pathways, highlighting their great potential as new targets for DDR-inhibition therapy. Although, there are still many uncharacterized ZNF-containing proteins with unknown biological function. In this review, we highlight the major classes and observed biological functions of ZNF proteins in mammalian cells. We briefly introduce well-known and newly discovered ZNFs and describe their molecular roles and contributions to human health and disease, especially cancer. Finally, we discuss the significance of ZNFs in DNA repair mechanisms, their potential in cancer therapy and advances in exploiting ZNF proteins as future therapeutic targets for human disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b781c237b08f60bdda6af9b1e73143caebb885a" target='_blank'>
              Zinc finger proteins: guardians of genome stability
              </a>
            </td>
          <td>
            Zeeba Kamaliyan, Thomas L. Clarke
          </td>
          <td>2024-07-25</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ad718715d0e23f67c54707d35f7ee63dc651a7" target='_blank'>
              Homeotic DUX4 Genes Shape Dynamic Inter-Chromosomal Contacts with Nucleoli in Human Cells.
              </a>
            </td>
          <td>
            E. S. Klushevskaya, I. R. Alembekov, Y. V. Kravatsky, N. A. Tchurikov
          </td>
          <td>2024-07-13</td>
          <td>Doklady. Biochemistry and biophysics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13152543ddfb5bd39a1ef928614e91f9f037825f" target='_blank'>
              The Role of Long Non-Coding RNF144A-AS1 in Cancer Progression.
              </a>
            </td>
          <td>
            Wei Wu, Xiaolan Li, Xueqin Li, Jiale Zhao, Yibei Gui, Yiyang Luo, Hailin Wang, Lijun Wang, Chengfu Yuan
          </td>
          <td>2024-07-16</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The expression of eukaryotic genes relies on the precise 3’-terminal cleavage and polyadenylation of newly synthesized pre-mRNA transcripts. Defects in these processes have been associated with various diseases, including cancer. While cancer-focused sequencing studies have identified numerous driver mutations in protein-coding sequences, noncoding drivers – particularly those affecting the cis-elements required for pre-mRNA cleavage and polyadenylation – have received less attention. Here, we systematically analysed cancer somatic mutations affecting 3’UTR polyadenylation signals using the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset. We found a striking enrichment of cancer-specific somatic mutations that disrupt strong and evolutionarily conserved cleavage and polyadenylation signals within tumour suppressor genes. Further bioinformatics and experimental analyses conducted as a part of our study suggest that these mutations have a profound capacity to downregulate the expression of tumour suppressor genes. Thus, this work uncovers a novel class of noncoding somatic mutations with significant potential to drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4b3938c7f84b24956b4223cbd287669eff1caf" target='_blank'>
              Recurrent disruption of tumour suppressor genes in cancer by somatic mutations in cleavage and polyadenylation signals
              </a>
            </td>
          <td>
            Yaroslav A. Kainov, F. Hamid, E. Makeyev
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="
 Introduction: Kras Gain of Function mutations are frequently detected in lung, colorectal and pancreatic cancers, in addition to others. Activating Kras mutations results in a constantly active protein, which we reasoned is quite unphysiological, yet it is tolerated by cells since this mutation is seen in hyperplasia. Activating this oncogene is insufficient alone for malignant transformation. Mutations in the tumor suppressor gene (TSG) p53 (the most frequently mutated gene in human cancer), cooperate with mutant Kras and is sufficient to permit full display of the actions of oncogenic Ras (as confirmed in Genetically Engineered Mouse Models and in clinical cases). The phenomenon of oncogene addiction may in fact be the result of obligatory requirements (cellular metabolism or otherwise) brought on by the constant oncogene signaling. This required adjustment in cellular circuitry can be afforded by specific cooperating TSG and since it too is a DNA constant change, becomes an new rigid reality for a cell with a given set of oncogene-TSG pair. p53 mutational spectrum was described as ‘enigmatic’ presumably because they almost never completely abrogate the function of this major regulator suggesting to us a possible essential mechanistic role for the retained transcriptional targets within the (well annotated) p53 transcriptional network. This could represent a putative synthetic lethality opportunity against activated Kras, an oncogene that has proven difficult to drug. We hypothesized that specific mutations in p53 with their respective transcriptional lesions cooperate with unique mutant Kras in tissue specific manner identifying a short list of gene targets for synthetic lethality experiments. Methods: To examine this hypothesis, we analyzed the TCGA database to determine a conserved pair cooperation between common Kras 12C; D or V and the top 6 reported p53 (hot spot) mutations [on residues 175; 245;248;249;273 and 282] in a stage-agnostic manner. To examine whether those putative interactions are cell-type specific, we performed this analysis in 3 different histologies (lung, colon, and pancreas). Results: The results suggested a non-random distribution of p53 mutants among the Kras driven cancers. KRAS (12 C, 12 V, 12 D were reported in 80% of Colon, Pancreas, and Lung Cancer. p53 (175 - 30%) (245 - 30%) (248 - 15%) (249 - 12%) (273 - 6.5%) (282 - 6.5%). More detailed analysis is planned for the poster session. Conclusion: Despite having a single activated oncogene, the distribution of the cooperating p53 mutations is nonrandom so examining the transcriptionally retained gene list represents a novel approach to explore for gene editing experiments. This provides and approach to drug the addiction of cancers to their oncogenes in a cancer specific, and occasionally to target essential proto-oncogene such as Myc where direct inhibition is highly undesirable due to its physiologic roles.
 Citation Format: Nishanth Thalambedu, Shallya Anand, Haya Safar, Farah Mazahreh, Ahmad Mazin M. Safar. Exploring the putative Kras-p53 mutational interface for vulnerability [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B009.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b7fa6ccd0c87ee81cbb16a2e3d13e489ca625071" target='_blank'>
              Abstract B009: Exploring the putative Kras-p53 mutational interface for vulnerability
              </a>
            </td>
          <td>
            Nishanth Thalambedu, Shallya Anand, Haya Safar, Farah Mazahreh, A. M. Safar
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, Bert van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="
 Tumor genomes often harbor a complex spectrum of single nucleotide mutations, small indels, and large chromosome rearrangements that can perturb coding and non-coding regions of the genome in ways that remain poorly understood. The mutational processes responsible for the genesis of these events are also not static; instead, they continue to operate throughout disease evolution and further diversify the genetic and phenotypic landscape of cancer cells. Mounting evidence suggests that tumor genotype can be an important determinant of disease progression and therapy response, such as by modulating the sensitivity or resistance of cancer cells to mutant-specific drugs. These types of observations have motivated efforts to treat cancers with genome-informed therapies, highlighting the need to understand how different genetic variants precisely affect gene function and overall tumor phenotypes. Variant-function maps that provide a mechanistic understanding of the biology driven by cancer-associated mutations are needed to design these types of treatment strategies. Precision genome editing technologies like base and prime editing are uniquely suited to tackle this problem. Nevertheless, deploying these methods for systematic variant-function studies and disease modeling in vivo has not been straightforward due to lack of robust and scalable platforms capable of assessing editing efficiency and precision, particularly at endogenous loci. With this goal in mind, we previously developed and applied high-throughput base editing ‘sensor’ approaches that link endogenous genome editing outcomes with synthetic DNA-based readouts and cellular fitness measurements. Using these approaches, we showed that several uncharacterized mutant p53 alleles drive cancer cell proliferation and in vivo tumor development. Building upon this work, we recently developed new prime editing guide RNA design tools and sensor-based approaches that similarly couple quantitative editing outcomes to cellular fitness, allowing us to significantly expand the breadth of cancer-associated mutations that can be interrogated using these precision genome editing technologies. In this talk, I will describe ongoing work using base and prime editing sensor libraries to probe the biological impact of thousands of functionally-distinct genetic variants in diverse types of protein-coding genes and families to learn how these influence various cancer phenotypes. I will also discuss how the implementation of these modular technologies will allow researchers to functionally disentangle the impact of endogenous and exogenous mutational processes on functional selection and tumor evolution with close to single base pair resolution. This generalizable precision genome editing framework will facilitate the functional interrogation of genetic variants across diverse biological contexts, providing much needed insight into cancer variant-function relationships that could be leveraged to develop more precise cancer treatment paradigms, including synthetic lethal therapies that exploit tumor genotype.
 Citation Format: Francisco J. Sánchez-Rivera, Samuel I. Gould, Alexandra N. Wuest, Kexin Dong, Grace A. Johnson, Alvin Hsu, Varun K. Narendra, Ondine Atwa, Stuart S. Levine, David R. Liu. Functional studies of genetic variation using precision genome editing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA020.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b31918f227a5efe4ffc9748fec8993c778a740c9" target='_blank'>
              Abstract IA020: Functional studies of genetic variation using precision genome editing
              </a>
            </td>
          <td>
            Francisco J. Sánchez-Rivera, Samuel I. Gould, Alexandra N. Wuest, Kexin Dong, Grace A. Johnson, Alvin Hsu, Varun K. Narendra, Ondine Atwa, Stuart S. Levine, David R. Liu
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3ef3973d270e32f48aaa19a00bef4e18bca5be" target='_blank'>
              Research Progress of Long Non-coding RNA-ZFAS1 in Malignant Tumors.
              </a>
            </td>
          <td>
            Xin Liu, Zhong Ma, Xianxu Zhang, Shicheng Li, Jiangdong An, Zhiqiang Luo
          </td>
          <td>2024-07-26</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this study, we investigated recurrent copy number variations (CNVs) in the 19p12 locus, which are associated with neurodevelopmental disorders. The two genes in this locus, ZNF675 and ZNF681, arose via gene duplication in primates, and their presence in several pathological CNVs in the human population suggests that either or both of these genes are required for normal human brain development. ZNF675 and ZNF681 are members of the Krüppel-associated box zinc finger (KZNF) protein family, a class of transcriptional repressors important for epigenetic silencing of specific genomic regions. About 170 primate-specific KZNFs are present in the human genome. Although KZNFs are primarily associated with repressing retrotransposon-derived DNA, evidence is emerging that they can be co-opted for other gene regulatory processes. We show that genetic deletion of ZNF675 causes developmental defects in cortical organoids, and our data suggest that part of the observed neurodevelopmental phenotype is mediated by a gene regulatory role of ZNF675 on the promoter of the neurodevelopmental gene Hes family BHLH transcription factor 1 (HES1). We also find evidence for the recently evolved regulation of genes involved in neurological disorders, microcephalin 1 and sestrin 3. We show that ZNF675 interferes with HES1 auto-inhibition, a process essential for the maintenance of neural progenitors. As a striking example of how some KZNFs have integrated into preexisting gene expression networks, these findings suggest the emergence of ZNF675 has caused a change in the balance of HES1 autoregulation. The association of ZNF675 CNV with human developmental disorders and ZNF675-mediated regulation of neurodevelopmental genes suggests that it evolved into an important factor for human brain development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2122cac7536c55e0373736baf9954110c9f4179" target='_blank'>
              Emergence of the ZNF675 Gene During Primate Evolution-Influenced Human Neurodevelopment Through Changing HES1 Autoregulation.
              </a>
            </td>
          <td>
            G. A. Lodewijk, Matthijs de Geus, Rita L F P Guimarães, F. Jacobs
          </td>
          <td>2024-07-01</td>
          <td>The Journal of comparative neurology</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/358937b5c51be3f8de419eabd20cda30e0675361" target='_blank'>
              Emerging toolkits for decoding the co-occurrence of modified histones and chromatin proteins.
              </a>
            </td>
          <td>
            Anne-Sophie Pépin, Robert Schneider
          </td>
          <td>2024-08-02</td>
          <td>EMBO reports</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c9abec0157ccbe7ffcea6b2ec6d1e2342588925" target='_blank'>
              Sustained inactivation of the Polycomb PRC1 complex induces DNA repair defects and genomic instability in epigenetic tumors
              </a>
            </td>
          <td>
            C. Rawal, V. Loubiere, N. Butova, Juliette Gracia, V. Parreno, C. Merigliano, Anne-Marie Martinez, G. Cavalli, I. Chiolo
          </td>
          <td>2024-06-18</td>
          <td>Histochemistry and Cell Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5a4fb1e6fb2869d1ffa14cf25654e46c58ec040e" target='_blank'>
              Beyond the code: the role of histone methylation in cancer resistance and therapy
              </a>
            </td>
          <td>
            Daniel Noerenberg, Frederik Damm
          </td>
          <td>2024-06-12</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Our understanding of cancer genomes has allowed for the integration of molecular data into histopathological classifications for routine stratification of patients. In the last 10–15 years, thanks to this systematic implementation of large-scale sequencing, the identification of hotspot somatic mutations in histone genes came into the limelight, underscoring the concept of oncohistones. As drivers in pediatric brain tumors, and in several other types of cancers, oncohistones brought a “new dimension of Strange” into the cancer multiverse, to paraphrase Marvel. An integrative approach to cancer complexity as a multidimensional projection is urgently needed to consider all relevant etiological, developmental, and evolutionary components. Here, we discuss recent progress on histone variants and chaperones, their regulation and alterations in cancers, the available in vivo models, and current treatment strategies. More specifically, we adopt a view through the lens of tissue-specific differences and means for genome expression and integrity maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2eba7c5952135066a16d3e19fbca0058628db86" target='_blank'>
              Histone H3 Variants in the Multiverse of Cancer
              </a>
            </td>
          <td>
            Iva Simeonova, Geneviève Almouzni
          </td>
          <td>2024-06-12</td>
          <td>Annual Review of Cancer Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The specialised structure of the centromere is critical for effective chromosome segregation, but its repetitive nature makes it vulnerable to rearrangements. Centromere fragility can drive tumorigenesis, but protective mechanisms preventing fragility are still not fully understood. The PBAF chromatin remodelling complex is frequently misregulated in cancer, but its role in cancer is still not fully characterized. Here, we identify PBAF as a protector of centromere and pericentromere structure with profound consequences for genome stability. A conserved feature of isogenic cell lines lacking PBRM1, a subunit of PBAF, is compromised centromere and pericentromere integrity. PBAF is present at these regions, and the binding pattern changes when PBRM1 is absent. PBRM1 loss creates a dependence on the spindle assembly checkpoint, which represents a therapeutic vulnerability. Importantly, we find that even in the absence of any perturbations, PBRM1 loss leads to centromere fragility, thus identifying a new player in centromere protection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da1343ff6c77eb7b266cc39475b5b2c96f325275" target='_blank'>
              PBRM1 directs PBAF to pericentromeres and protects centromere integrity
              </a>
            </td>
          <td>
            Karen A. Lane, Alison Harrod, Lillian Wu, Theodoros I. Roumeliotis, Hugang Feng, Shane Foo, Katheryn A. G. Begg, Federica Schiavoni, F. Zenke, Alan A. Melcher, Jyoti S. Choudhary, Jessica A. Downs
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Loss-of-function mutations in methyl-CpG binding protein 2 (MECP2) cause Rett syndrome, a postnatal neurodevelopmental disorder that occurs in ∼1/10,000 live female births. MeCP2 binds to methylated cytosines across genomic DNA and recruits various partners to regulate gene expression. MeCP2 has been shown to repress transcription in vitro and interacts with co-repressors such as the Sin3A and NCoR complexes. Based on these observations, MeCP2 has been largely considered as a repressor of transcription. However, a mouse model of RTT displays many down-regulated genes, and those same genes are up-regulated in a MECP2 duplication mouse model. Furthermore, TCF20, which has been associated with transcriptional activation, have recently been identified as a protein interactor of MeCP2. These data broaden the potential functions of MeCP2 as a regulator of gene expression. Yet, the molecular mechanisms underlying MeCP2-dependent gene regulation remain largely unknown. Here, using a human MECP2 gain-of-function Drosophila model, we screened for genetic modifiers of MECP2-induced phenotypes. Our approach identified several subunits of the Drosophila super elongation complex, a P-TEFb containing RNA polymerase II (RNA pol II) elongation factor required for the release of promoter-proximally paused RNA pol II, as genetic interactors of MECP2. We discovered that MeCP2 physically interacts with the SEC in human cells and in the mouse brain. Furthermore, we found that MeCP2 directly binds AFF4, the scaffold of the SEC, via the transcriptional repression domain. Finally, loss of MeCP2 in the mouse cortex caused reduced binding of AFF4 specifically on a subset of genes involved in the regulation of synaptic function, which also displayed the strongest decrease in RNA pol II binding in the genebody. Taken together, our study reveals a previously unrecognized mechanism through which MeCP2 regulates transcription, providing a new dimension to its regulatory role in gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62efed55dc8b31895230549b72d5254a49569042" target='_blank'>
              MeCP2 Interacts with the Super Elongation Complex to Regulate Transcription
              </a>
            </td>
          <td>
            Jun Young Sonn, Wonho Kim, Marta Iwanaszko, Y. Aoi, Yan Li, Luke Parkitny, Janice L. Brissette, Lorin Weiner, I. Al-Ramahi, Juan Botas, A. Shilatifard, Huda Y. Zoghbi
          </td>
          <td>2024-07-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The p53 family remains a captivating focus of an extensive number of current studies. Accumulating evidence indicates that p53 abnormalities rank among the most prevalent in cancer. Given the numerous existing studies, which mostly focus on the mutations, expression profiles, and functional perturbations exhibited by members of the p53 family across diverse malignancies, this review will concentrate more on less explored facets regarding p53 activation and stabilization by the nuclear pore complex (NPC) in cancer, drawing on several studies. p53 integrates a broad spectrum of signals and is subject to diverse regulatory mechanisms to enact the necessary cellular response. It is widely acknowledged that each stage of p53 regulation, from synthesis to degradation, significantly influences its functionality in executing specific tasks. Over recent decades, a large body of data has established that mechanisms of regulation, closely linked with protein activation and stabilization, involve intricate interactions with various cellular components. These often transcend canonical regulatory pathways. This new knowledge has expanded from the regulation of genes themselves to epigenomics and proteomics, whereby interaction partners increase in number and complexity compared with earlier paradigms. Specifically, studies have recently shown the involvement of the NPC protein in such complex interactions, underscoring the further complexity of p53 regulation. Furthermore, we also discuss therapeutic strategies based on recent developments in this field in combination with established targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f4ce5cb3bd6bcbf7187f9f514ca001d925085c" target='_blank'>
              Unlocking the Gateway: The Spatio-Temporal Dynamics of the p53 Family Driven by the Nuclear Pores and Its Implication for the Therapeutic Approach in Cancer
              </a>
            </td>
          <td>
            Dini Kurnia Ikliptikawati, Keiichi Makiyama, M. Hazawa, Richard W. Wong
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8200f580864196977ce1aaadc3c51eb18d6c8f71" target='_blank'>
              Evolutionary patterns and functional effects of 3D chromatin structures in butterflies with extensive genome rearrangements
              </a>
            </td>
          <td>
            Botong Zhou, Ping Hu, Guichun Liu, Zhou Chang, Zhiwei Dong, Zihe Li, Yuan Yin, Zunzhe Tian, Ge Han, Wen Wang, Xueyan Li
          </td>
          <td>2024-07-26</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Motivation Techniques such as high-throughput chromosome conformation capture (Hi-C) have provided a wealth of information on the organization of the nucleus and the genome important for understanding gene expression regulation. Additionally, Genome-Wide Association Studies (GWASs) have uncovered thousands of loci related to complex traits. Expression quantitative trait loci (eQTL) studies have further linked the genetic variants to alteration in expression levels of associated target genes across individuals. However, the functional roles of many eQTLs located in non-coding regions are unclear. Current joint analyses of Hi-C and eQTLs data lack advanced computational tools, limiting what can be learned from these data. Result In this work, we developed a computational method for simultaneous analysis of Hi-C and eQTL data. Our method can identify a small set of non-random interactions from all Hi-C interactions. Using these non-random interactions, we reconstruct large ensemble (×105) of high-resolution single-cell 3D chromatin conformations with thorough sampling, which accurately replicate Hi-C measurements. Our results revealed the presence of many-body interactions in chromatin conformation at single-cell level in eQTL locus, offering detailed view into how three-dimensional structures of chromatin form the physical foundation for gene regulation, including how genetic variants of eQTLs affect the expression level of their associated eGenes. Furthermore, our method can deconvolve chromatin heterogeneity and investigate the spatial associations of eQTLs and eGenes at subpopulation level to reveal their regulatory impacts on gene expression. Together, ensemble modeling of thoroughly sampled single cell chromatin conformations from Hi-C, along with eQTL data, helps to decipher how chromatin 3D structures provide the physical basis for gene regulation, expression control, and aid in understanding of the overall structure-function relationships of genome organization. Availability and implementation: It is available at https://github.com/uic-liang-lab/3DChromFolding-eQTL-Loci">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9720002233785e5b33a3a1f9b1f74f286c10723" target='_blank'>
              Structural Basis of Differential Gene Expression at eQTLs Loci from High-Resolution Ensemble Models of 3D Single-Cell Chromatin Conformations
              </a>
            </td>
          <td>
            Lin Du, Hammad Farooq, Pourya Delafrouz, Jie Liang
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The male mammalian germline is characterized by substantial chromatin remodeling associated with the transition from histones to protamines during spermatogenesis, followed by the reversal to nucleohistones in the male pronucleus preceding the zygotic genome activation. Both transitions are associated with the extensive formation of DNA double-strand breaks (DSBs), requiring an estimated 5 to 10 million transient DSBs per spermatozoa. Additionally, the high transcription rate in early stages of spermatogenesis leads to transcription-coupled damage preceding meiotic homologous recombination, potentially further contributing to the DSB landscape in mature spermatozoa. Once meiosis is completed, spermatozoa remain haploid and therefore cannot rely on error-free homologous recombination, but instead depend on error-prone classical non-homologous end joining (cNHEJ). This DNA damage/repair-scenario is proposed to be one of the main causes of the observed paternal mutation propensity in human evolution. Recent studies have shown that DSBs in the male pronucleus are repaired by maternally provided Polθ in Caenorhabditis elegans through Polθ-mediated end joining (TMEJ). Additionally, population genetic datasets have revealed a preponderance of TMEJ signatures associated with human variation. Since these signatures are the result of the combined effect of TMEJ and DSB formation in spermatozoa and male pronuclei, we used a BLISS-based protocol to analyze recurrent DSBs in mature human sperm heads as a proxy of the male pronucleus before zygotic chromatin remodeling. The DSBs were found to be enriched in (YR)n short tandem repeats and in evolutionarily young SINEs, reminiscent to patterns observed in murine spermatids, indicating evolutionary hotspots of recurrent DSB formation in mammalian spermatozoa. Additionally, we detected a similar DSB pattern in diploid human IMR90 cells when cNHEJ was selectively inhibited, indicating the significant impact of absent cNHEJ on the sperm DSB landscape. Strikingly, regions associated with most retained histones, and therefore less condensed chromatin, were not strongly enriched with recurrent DSBs. In contrast, the fraction of retained H3K27me3 in the mature spermatozoa displayed a strong association with recurrent DSBs. DSBs in H3K27me3 are associated with a preference for TMEJ over cNHEJ during repair. We hypothesize that the retained H3K27me3 may trigger transgenerational DNA repair by priming maternal Polθ to these regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60c153158e8100ec0f4fdb6f1cd7fe2e99666eda" target='_blank'>
              DSB profiles in human spermatozoa highlight the role of TMEJ in the male germline
              </a>
            </td>
          <td>
            Maurice Scheuren, Jonas Möhner, Max Müller, Hans Zischler
          </td>
          <td>2024-07-08</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2287f3da76694ef80e19d9b8e54130589563196" target='_blank'>
              Genome-wide analysis of histone modifications can contribute to the identification of candidate cis-regulatory regions in the threespine stickleback fish
              </a>
            </td>
          <td>
            Genta Okude, Yo Y. Yamasaki, Atsushi Toyoda, Seiichi Mori, Jun Kitano
          </td>
          <td>2024-07-11</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) constitute a distinctive subset of RNA molecules with limited protein-coding potential, which exert crucial impacts on various biological activities. In the context of cancer, dysregulated lncRNAs function as essential regulators that affect tumor initiation and malignant progression. These lncRNAs serve as competitive endogenous RNAs (ceRNAs) through sponging microRNAs and regulating the expression of targeted genes. Moreover, they also directly bind to RNA-binding proteins, which can be integrated into a complex mechanistic network. E2F1, an extensively studied transcription factor, mediates multiple malignant behaviors by regulating cell cycle progression, tumor metastasis, and therapeutic response. Emerging evidence suggests that lncRNAs play a pivotal role in regulating the E2F1 pathway. This review aims to elucidate the intricate gene regulatory programs between lncRNAs and E2F1 in cancer progression. We elaborate on distinct mechanistic networks involved in cancer progression, emphasizing the potential of the lncRNAs/E2F1 axes as promising targets for cancer therapy. Additionally, we provide novel perspectives on current evidence, limitations, and future directions for targeting lncRNAs in human cancers. Fully deciphering the intricate network of lncRNA/E2F1-mediated regulatory mechanisms in cancer could facilitate the translation of current findings into clinical course, such efforts ultimately significantly improve the clinical prognosis of cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b9cb13332ef65d4e861190bb1d5bd1ece6a1d1f" target='_blank'>
              Current concepts of the crosstalk between lncRNA and E2F1: shedding light on the cancer therapy
              </a>
            </td>
          <td>
            Peng Huang, Feng Wen, Qiu Li
          </td>
          <td>2024-07-25</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="While genotoxic chemotherapeutic agents are among the most effective tools to combat cancer, they are often associated with severe adverse effects caused by indiscriminate DNA damage in non-tumor tissue as well as increased risk of secondary carcinogenesis. This study builds on our previous work demonstrating that the RNA Polymerase I (Pol I) transcription inhibitor CX-5461 elicits a non-canonical DNA damage response and our discovery of a critical role for Topoisomerase 2α (Top2α) in the initiation of Pol I-dependent transcription. Here, we identify Top2α as a mediator of CX-5461 response in the murine Eµ-Myc B lymphoma model whereby sensitivity to CX-5461 is dependent on cellular Top2α expression/activity. Most strikingly, and in contrast to canonical Top2α poisons, we found that the Top2α-dependent DNA damage induced by CX-5461 is preferentially localized at the ribosomal DNA (rDNA) promoter region, thereby highlighting CX-5461 as a loci-specific DNA damaging agent. This mechanism underpins the efficacy of CX-5461 against certain types of cancer and can be used to develop effective non-genotoxic anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/913f673912030e76f53faa13fb8d57cb178fa48f" target='_blank'>
              CX-5461 Preferentially Induces Top2α-Dependent DNA Breaks at Ribosomal DNA Loci
              </a>
            </td>
          <td>
            Donald P. Cameron, J. Sornkom, Sameerh Alsahafi, D. Drygin, G. Poortinga, Grant A. McArthur, Nadine Hein, Ross D Hannan, Konstantin I. Panov
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c57c70b2a1bb317c90f2cc0de26bf7a8a7418b" target='_blank'>
              A microdeletion event at 19q13.43 in IDH-mutant astrocytomas is strongly correlated with MYC overexpression
              </a>
            </td>
          <td>
            Ege Ülgen, U. Gerlevik, Sila Gerlevik, Yavuz Oktay, O. U. Sezerman, Şevin Turcan, K. Ozduman
          </td>
          <td>2024-06-14</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aba56364331e9a8e54090b261a21c198f4de3054" target='_blank'>
              ARAP1-AS1: a novel long non-coding RNA with a vital regulatory role in human cancer development
              </a>
            </td>
          <td>
            Jialing Wang, Hongliang Luo, Lu Yang, Huazhao Yuan
          </td>
          <td>2024-08-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Human cell line models, including the neuronal precursor line LUHMES, are important for investigating developmental transcriptional dynamics within imprinted regions, particularly the 15q11-q13 Angelman (AS) and Prader-Willi (PWS) syndrome locus. AS results from loss of maternal UBE3A in neurons, where the paternal allele is silenced by a convergent antisense transcript UBE3A-ATS, a lncRNA that terminates at PWAR1 in non-neurons. qRT-PCR analysis confirmed the exclusive and progressive increase in UBE3A-ATS in differentiating LUHMES neurons, validating their use for studying UBE3A silencing. Genome-wide transcriptome analyses revealed changes to 11 834 genes during neuronal differentiation, including the upregulation of most genes within the 15q11-q13 locus. To identify dynamic changes in chromatin loops linked to transcriptional activity, we performed a HiChIP validated by 4C, which identified two neuron-specific CTCF loops between MAGEL2-SNRPN and PWAR1-UBE3A. To determine if allele-specific differentially methylated regions (DMR) may be associated with CTCF loop anchors, whole genome long-read nanopore sequencing was performed. We identified a paternally hypomethylated DMR near the SNRPN upstream loop anchor exclusive to neurons and a paternally hypermethylated DMR near the PWAR1 CTCF anchor exclusive to undifferentiated cells, consistent with increases in neuronal transcription. Additionally, DMRs near CTCF loop anchors were observed in both cell types, indicative of allele-specific differences in chromatin loops regulating imprinted transcription. These results provide an integrated view of the 15q11-q13 epigenetic landscape during LUHMES neuronal differentiation, underscoring the complex interplay of transcription, chromatin looping, and DNA methylation. They also provide insights for future therapeutic approaches for AS and PWS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c616b0f402f95ed53dc1945c207ee4eb663c1600" target='_blank'>
              Integration of CTCF loops, methylome, and transcriptome in differentiating LUHMES as a model for imprinting dynamics of the 15q11-q13 locus in human neurons.
              </a>
            </td>
          <td>
            Orangel J. Gutierrez Fugón, Osman Sharifi, Nicholas Heath, Daniela C. Soto, J. A. Gomez, Dag Yasui, A. J. P. Mendiola, H. O’Geen, U. Beitnere, Marketa Tomkova, Viktoria Haghani, Greg Dillon, David J. Segal, J. LaSalle
          </td>
          <td>2024-07-24</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f701cb6b5c83faca69b958611b0964a6f2f858d" target='_blank'>
              Non-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors
              </a>
            </td>
          <td>
            Qing Hu, Jose Espejo Valle-Inclán, Rashmi Dahiya, Alison Guyer, Alice Mazzagatti, Elizabeth G. Maurais, Justin L. Engel, Huiming Lu, Anthony J. Davis, I. Cortés-Ciriano, P. Ly
          </td>
          <td>2024-07-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23c60c3e33a282e577264c499ae82c7a724f98d3" target='_blank'>
              Bioinformatic Analysis Reveals the Association of Human N-Terminal Acetyltransferase Complexes with Distinct Transcriptional and Post-Transcriptional Processes.
              </a>
            </td>
          <td>
            C. Koufaris, C. Demetriadou, V. Nicolaidou, A. Kirmizis
          </td>
          <td>2024-06-12</td>
          <td>Biochemical genetics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="While genome-wide association studies and expression quantitative trait loci (eQTL) analysis have made significant progress in identifying noncoding variants associated with prostate cancer risk and bulk tissue transcriptome changes, the regulatory effect of these genetic elements on gene expression remains largely unknown. Recent developments in single-cell sequencing have made it possible to perform ATAC-seq and RNA-seq profiling simultaneously to capture functional associations between chromatin accessibility and gene expression. In this study, we tested our hypothesis that this multiome single-cell approach allows for mapping regulatory elements and their target genes at prostate cancer risk loci. We applied a 10X Multiome ATAC + Gene Expression platform to encapsulate Tn5 transposase-tagged nuclei from multiple prostate cell lines for a total of 65,501 high quality single cells from RWPE1, RWPE2, PrEC, BPH1, DU145, PC3, 22Rv1 and LNCaP cell lines. To address data sparsity commonly seen in the single-cell sequencing, we performed targeted sequencing to enrich sequencing data at prostate cancer risk loci involving 2,730 candidate germline variants and 273 associated genes. Although not increasing the number of captured cells, the targeted multiome data did improve eQTL gene expression abundance by about 20% and chromatin accessibility abundance by about 5%. Based on this multiomic profiling, we further associated RNA expression alterations with chromatin accessibility of germline variants at single cell levels. Cross validation analysis showed high overlaps between the multiome associations and the bulk eQTL findings from GTEx prostate cohort. We found that about 20% of GTEx eQTLs were covered within the significant multiome associations (p-value ≤ 0.05, gene abundance percentage ≥ 5%), and roughly 10% of the multiome associations could be identified by significant GTEx eQTLs. We also analyzed accessible regions with available heterozygous SNP reads and observed more frequent association in genomic regions with allelically accessible variants (p = 0.0055). Among these findings were previously reported regulatory variants including rs60464856-RUVBL1 (multiome p-value = 0.0099 in BPH1) and rs7247241-SPINT2 (multiome p-value = 0.0002- 0.0004 in 22Rv1). We also functionally validated a new regulatory SNP and its target gene rs2474694-VPS53 (multiome p-value = 0.00956 in BPH1 and 0.00625 in DU145) by reporter assay and SILAC proteomics sequencing. Taken together, our data demonstrated the feasibility of the multiome single-cell approach for identifying regulatory SNPs and their regulated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936580ce17da8d58425b498ecb0e8504faaa215a" target='_blank'>
              Identify Regulatory eQTLs by Multiome Sequencing in Prostate Single Cells
              </a>
            </td>
          <td>
            Yijun Tian, Lang Wu, Chang-Ching Huang, Liang Wang
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9895762dc4559c029ea063221ae034c05dcf58ce" target='_blank'>
              DNA replication in early mammalian embryos is patterned, predisposing lamina-associated regions to fragility
              </a>
            </td>
          <td>
            Shuangyi Xu, Ning Wang, M. Zuccaro, Jeannine Gerhardt, R. Iyyappan, Giovanna Nascimento Scatolin, Zongliang Jiang, T. Baslan, A. Koren, Dieter Egli
          </td>
          <td>2024-06-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d0c0b53ddb4794ed24f34e4c2906dc37ebeb850" target='_blank'>
              Dan forms condensates in neuroblasts and regulates nuclear architecture and progenitor competence in vivo
              </a>
            </td>
          <td>
            Gillie Benchorin, Richard Jangwon Cho, M. Li, Natalia Molotkova, M. Kohwi
          </td>
          <td>2024-06-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Cohesins promote proper chromosome segregation, gene transcription, genomic architecture, DNA condensation, and DNA damage repair. Mutations in either cohesin subunit or regulatory genes can give rise to severe developmental abnormalities (such as Robert Syndrome and Cornelia de Lange Syndrome) and also are highly correlated with cancer. Despite this, little is known about cohesin regulation. Eco1 (ESCO2/EFO2 in humans) and Rad61 (WAPL in humans) represent two such regulators but perform opposing roles. Eco1 acetylation of cohesin during S phase, for instance, stabilizes cohesin-DNA binding to promote sister chromatid cohesion. On the other hand, Rad61 promotes the dissociation of cohesin from DNA. While Eco1 is essential, ECO1 and RAD61 co-deletion results in yeast cell viability, but only within a limited temperature range. Here, we report that eco1 rad61 cell lethality is due to reduced levels of the cohesin subunit Mcd1. Results from a suppressor screen further reveals that FDO1 deletion rescues the temperature sensitive (ts) growth defects exhibited by eco1 rad61 double mutant cells by increasing Mcd1 levels. Regulation of MCD1 expression, however, appears more complex. Elevated expression of MBP1, which encodes a subunit of the MBF transcription complex, also rescues eco1 rad61 cell growth defects. Elevated expression of SWI6, however, which encodes the Mbp1-binding partner of MBF, exacerbates eco1 rad61 cell growth and also abrogates the Mpb1-dependent rescue. Finally, we identify two additional transcription factors, Fkh1 and Fkh2, that impact MCD1 expression. In combination, these findings provide new insights into the nuanced and multi-faceted transcriptional pathways that impact MCD1 expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cada658e61da12bf9a49bbb82ccb4ed5e7419997" target='_blank'>
              Fdo1, Fkh1, Fkh2 and the Swi6-Mbp1 MBF complex regulate Mcd1 levels to impact eco1 rad61 cell growth in Saccharomyces cerevisiae
              </a>
            </td>
          <td>
            Gurvir Singh, R. Skibbens
          </td>
          <td>2024-06-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Aberrant oncogenic signaling causes cells to transition into oncogene-induced senescence (OIS) to limit uncontrolled proliferation. Despite being a potent tumor suppressor mechanism, OIS is an unstable cell state susceptible to reprogramming that can promote tumorigenesis. Therefore, elucidating the underlying gene regulatory mechanisms that commit cells to OIS is critical to identifying actionable targets to modulate the senescence state. We previously showed that timely execution of the OIS program is governed by hierarchical transcription factor (TF) networks. However, the gene regulatory mechanisms that prime cells to commit to the OIS fate early upon oncogene hyperactivation are currently not known. Here, we leveraged our time-resolved multi-omic profiling approach to generate TF networks during the first 24 h of oncogenic HRASG12V activation. Using this approach, we demonstrate that the commitment to OIS requires the rearrangement of the TF network on a pre-established epigenomic landscape, priming the cells for the substantial chromatin remodeling that underpins the transition to OIS. Our results provide a detailed map of the chromatin landscape before cells transition to OIS thus offering a platform for manipulation of senescence outcomes of potentially therapeutic value.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/412ffff3bb32763efa919fb353d8bfbc6d71328f" target='_blank'>
              Transcription factor network dynamics during the commitment to oncogene-induced senescence
              </a>
            </td>
          <td>
            Themistoklis Vasilopoulos, R. Martínez-Zamudio
          </td>
          <td>2024-07-15</td>
          <td>Frontiers in Epigenetics and Epigenomics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b14b3a2b3b2cb05b6f12ac40b8502ecde1ffe45" target='_blank'>
              ASS1 metabolically contributes to the nuclear and cytosolic p53-mediated DNA damage response
              </a>
            </td>
          <td>
            L. Lim, Lital Adler, E. Hajaj, Leandro R Soria, R. B. Perry, Naama Darzi, Ruchama Brody, N. Furth, Michal Lichtenstein, Elizabeta Bab-Dinitz, Z. Porat, Tevie Melman, Alexander Brandis, Yael Aylon, S. Ben-Dor, Irit Orr, Amir Pri-Or, Rony Seger, Y. Shaul, E. Ruppin, Moshe Oren, Minervo Perez, Jordan Meier, N. Brunetti‐Pierri, E. Shema, I. Ulitsky, A. Erez, S. Malitsky, Maxim Itkin
          </td>
          <td>2024-06-10</td>
          <td>Nature Metabolism</td>
          <td>1</td>
          <td>52</td>
        </tr>

        <tr id="A considerable proportion of the eukaryotic genome undergoes transcription, leading to the generation of noncoding RNA molecules that lack protein-coding information and are not subjected to translation. These noncoding RNAs (ncRNAs) are well recognized to have essential roles in several biological processes. Long noncoding RNAs (lncRNAs) represent the most extensive category of ncRNAs found in the human genome. Much research has focused on investigating the roles of cis-acting lncRNAs in the regulation of specific target gene expression. In the majority of instances, the regulation of sense gene expression by its corresponding antisense pair occurs in a negative (discordant) manner, resulting in the suppression of the target genes. The notion that a negative correlation exists between sense and antisense pairings is, however, not universally valid. In fact, several recent studies have reported a positive relationship between corresponding cis antisense pairs within plants, budding yeast, and mammalian cancer cells. The positive (concordant) correlation between anti-sense and sense transcripts leads to an increase in the level of the sense transcript within the same genomic loci. In addition, mechanisms such as altering chromatin structure, the formation of R loops, and the recruitment of transcription factors can either enhance transcription or stabilize sense transcripts through their antisense pairs. The primary objective of this work is to provide a comprehensive understanding of both aspects of antisense regulation, specifically focusing on the positive correlation between sense and antisense transcripts in the context of eukaryotic gene expression, including its implications towards cancer progression. This article is categorized under: RNA Processing > 3' End Processing Regulatory RNAs/RNAi/Riboswitches > Regulatory RNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7f42cca4635492a3be12eb31656e764a000d884" target='_blank'>
              Novel insights on the positive correlation between sense and antisense pairs on gene expression.
              </a>
            </td>
          <td>
            Subhadeep Das, M. P. Zea Rojas, Elizabeth Tran
          </td>
          <td>2024-07-01</td>
          <td>Wiley interdisciplinary reviews. RNA</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Heterochromatin is a key feature of eukaryotic genomes that serves important regulatory and structural roles in regions such as centromeres. In fission yeast, maintenance of existing heterochromatic domains relies on positive feedback loops involving histone methylation and non-coding RNAs. However, requirements for de novo establishment of heterochromatin are less well understood. Here, through a cross-based assay we have identified a novel factor influencing the efficiency of heterochromatin establishment. We determine that the previously uncharacterised protein is an ortholog of human Caprin1, an RNA-binding protein linked to stress granule formation. We confirm that the fission yeast ortholog, here named Cpn1, also associates with stress granules, and we uncover evidence of interplay between heterochromatin integrity and ribonucleoprotein (RNP) granule formation, with heterochromatin mutants showing reduced granule formation in the presence of stress, but increased granule formation in the absence of stress. We link this to regulation of non-coding heterochromatic transcripts, since in heterochromatin-deficient cells, absence of Cpn1 leads to hyperaccumulation of centromeric RNAs at centromeres. Together, our findings unveil a novel link between RNP homeostasis and heterochromatin assembly, and implicate Cpn1 and associated factors in facilitating efficient heterochromatin establishment by enabling removal of excess transcripts that would otherwise impair assembly processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ad9fad413ac8508fc3f9242f869a8483bc4bb21" target='_blank'>
              Fission yeast Caprin protein is required for efficient heterochromatin establishment
              </a>
            </td>
          <td>
            Haidao Zhang, Ekaterina Kapitonova, Adriana Orrego, Christos Spanos, Joanna Strachan, E. Bayne
          </td>
          <td>2024-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Abstract The ribosome is a remarkably complex machinery, at the interface with diverse cellular functions and processes. Evolutionarily conserved, yet intricately regulated, ribosomes play pivotal roles in decoding genetic information into the synthesis of proteins and in the generation of biomass critical for cellular physiological functions. Recent insights have revealed the existence of ribosome heterogeneity at multiple levels. Such heterogeneity extends to cancer, where aberrant ribosome biogenesis and function contribute to oncogenesis. This led to the emergence of the concept of ‘onco-ribosomes’, specific ribosomal variants with altered structural dynamics, contributing to cancer initiation and progression. Ribosomal proteins (RPs) are involved in many of these alterations, acting as critical factors for the translational reprogramming of cancer cells. In this review article, we highlight the roles of RPs in ribosome biogenesis, how mutations in RPs and their paralogues reshape the translational landscape, driving clonal evolution and therapeutic resistance. Furthermore, we present recent evidence providing new insights into post-translational modifications of RPs, such as ubiquitylation, UFMylation and phosphorylation, and how they regulate ribosome recycling, translational fidelity and cellular stress responses. Understanding the intricate interplay between ribosome complexity, heterogeneity and RP-mediated regulatory mechanisms in pathology offers profound insights into cancer biology and unveils novel therapeutic avenues targeting the translational machinery in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22a3b2d25463305f9723b9b7d95755debdacbb46" target='_blank'>
              Decoding ribosome complexity: role of ribosomal proteins in cancer and disease
              </a>
            </td>
          <td>
            P. Fuentes, Joffrey Pelletier, A. Gentilella
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>1</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3101f7048ced189804d5ffb2bbda06ceae417ac" target='_blank'>
              CRISPR-Cas9-mediated deletion enhancer of MECOM play a tumor suppressor role in ovarian cancer.
              </a>
            </td>
          <td>
            Yujie Chen, Qiuwen Jiang, Yingzhuo Xue, Weiguan Chen, Minhui Hua
          </td>
          <td>2024-07-12</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Gene position in the nuclear space plays an important role in gene regulation. This is exemplified by repressive chromatin domains frequently contacting nuclear lamina or nucleoli. The nucleolus undergoes structural changes in response to various cellular states, potentially impacting genome organization. However, how the 3D-genome organization responds to nucleolar states has remained underinvestigated due to the lack of methods able to identify nucleolar associated domains (NADs) in single cells and under nucleolar stress. To address this, we developed NoLMseq, a method combining laser-capture microdissection and DNA sequencing to map NADs in single cells. NoLMseq identified many unexplored features of chromosome organization around single nucleoli such as NAD heterogeneity among ESCs, culminating in two major populations with distinct chromatin states. The data also revealed that NADs prevalently contact nucleoli in a monoallelic manner and allelic nucleolar contact frequency sets gene expression and chromatin states. NoLMseq also revealed how chromosomes reorganise around nucleoli under nucleolar stress conditions, highlighting the importance of nucleolus integrity in genome organization and 3D-genome response to nucleolar stress. The results demonstrated that NoLMseq accurately measures chromosome contacts around single healthy and stressed nucleoli and it will be a critical tool to study NADs within biological populations and determine how the 3D-genome responds to nucleolar stress in healthy and disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65d4a9816d5208a9a710d3dadad38f4b35cb1d67" target='_blank'>
              Single-cell dynamics of genome-nucleolus interactions captured by nucleolar laser microdissection (NoLMseq)
              </a>
            </td>
          <td>
            K. Walavalkar, Shivani Gupta, Jelena Kresoja-Rakic, Mathieu Raingeval, Chiara Mungo, Raffaella Santoro
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Both transcription and replication can take place simultaneously on the same DNA template, potentially leading to transcription-replication conflicts (TRCs) and topological problems. Here we asked which topoisomerase(s) is/are the best candidate(s) for sensing TRC. Genome-wide topoisomerase binding sites were mapped in parallel for all the nuclear topoisomerases (TOP1, TOP2A, TOP2B, TOP3A and TOP3B). To increase the signal to noise ratio (SNR), we used ectopic expression of those topoisomerases in H293 cells followed by a modified CUT&Tag method. Although each topoisomerase showed distinct binding patterns, all topoisomerase binding signals positively correlated with gene transcription. TOP3A binding signals were suppressed by DNA replication inhibition. This was also observed but to a lesser extent for TOP2A and TOP2B. Hence, we propose the involvement of TOP3A in sensing both head-on TRCs (HO-TRCs) and codirectional TRCs (CD-TRCs). In which case, the TOP3A signals appear concentrated within the promoters and first 20 kb regions of the 5’ -end of genes, suggesting the prevalence of TRCs and the recruitment of TOP3A in the 5’-regions of transcribed and replicated genes. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8169cbcac60d35ccac774a055807aef2360efe89" target='_blank'>
              Genome-wide Mapping of Topoisomerase Binding Sites Suggests Topoisomerase 3α (TOP3A) as a Reader of Transcription-Replication Conflicts (TRC)
              </a>
            </td>
          <td>
            Hongliang Zhang, Yilun Sun, Sourav Saha, L. Saha, L. Pongor, Anjali Dhall, Yves Pommier
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="CCCTC-binding factor (CTCF) is an insulator protein that binds to a highly conserved DNA motif and facilitates regulation of three-dimensional (3D) nuclear architecture and transcription. CTCF binding sites (CTCF-BSs) reside in non-coding DNA and are frequently mutated in cancer. Our previous study identified a small subclass of CTCF-BSs that are resistant to CTCF knock down, termed persistent CTCF binding sites (P-CTCF-BSs). P-CTCF-BSs show high binding conservation and potentially regulate cell-type constitutive 3D chromatin architecture. Here, using ICGC sequencing data we made the striking observation that P-CTCF-BSs display a highly elevated mutation rate in breast and prostate cancer when compared to all CTCF-BSs. To address whether P-CTCF-BS mutations are also enriched in other cell-types, we developed CTCF-INSITE-a tool utilising machine learning to predict persistence based on genetic and epigenetic features of experimentally-determined P-CTCF-BSs. Notably, predicted P-CTCF-BSs also show a significantly elevated mutational burden in all 12 cancer-types tested. Enrichment was even stronger for P-CTCF-BS mutations with predicted functional impact to CTCF binding and chromatin looping. Using in vitro binding assays we validated that P-CTCF-BS cancer mutations, predicted to be disruptive, indeed reduced CTCF binding. Together this study reveals a new subclass of cancer specific CTCF-BS DNA mutations and provides insights into their importance in genome organization in a pan-cancer setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994d6f59097fc218409b42c901b1ae72776752f9" target='_blank'>
              Machine learning enables pan-cancer identification of mutational hotspots at persistent CTCF binding sites.
              </a>
            </td>
          <td>
            Wenhan Chen, Y. Zeng, Joanna Achinger-Kawecka, Elyssa Campbell, Alicia K Jones, Alastair G Stewart, Amanda Khoury, Susan J Clark
          </td>
          <td>2024-07-02</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Purpose: Complex chromosomal rearrangements (CCRs) are rare structural variants involving three or more chromosomal break- points. Most de novo reported CCRs pose challenges for diagnosis and management. They often require karyotyping, fluorescence in situ hybridization (FISH), and chromosomal microarray analysis (CMA) for clinical diagnosis because of the limitations of each method. Here we report an inherited exceptionally complex CCR involving 4 chromosomes and 11 breakpoints in a family with multisystem anomalies. Methods: We evaluated the CCRs using karyotyping, FISH, CMA, and two emerging genomic technologies: high-throughput chromosome conformation capture sequencing (Hi-C; aka genomic proximity mapping, GPM) and optical genome mapping (OGM). We also performed functional studies using transcriptome and methylome analyses. Results: The proband, who had intellectual disability and immune deficiency, shared CCRs with her unaffected mother involving chromosomes 1, 7, and 11 by karyotyping. However, CMA revealed a duplication and three deletions in the proband in contrast to her mother's balanced genome. Hi-C (GPM) and OGM detected the CCRs and copy number alterations but also uncovered additional breakpoints at high resolution, including an insertion in 4p and two cryptic rearrangements at 7p. Transcriptome and methylome analyses identified likely biological pathways associated with the proband's phenotypes. Conclusion: Combining cytogenetic and genomic methods provided comprehensive characterization and defined the breakpoints at high resolution in both proband and mother. This underscores the value of novel cytogenetic and genomic techniques in deciphering complex genome rearrangements and the significance of integrative genomic analysis and functional characterization in understanding clinical phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d8dc703c96a52057ab6829a72726f61eb74bd56" target='_blank'>
              Genetic and functional characterization of inherited complex chromosomal rearrangements in a family with multisystem anomalies
              </a>
            </td>
          <td>
            H. Fang, S. M. Eacker, Y. Wu, W. Neufeld-Kaiser, M. Laurino, S. Keel, M. S. Horwitz, Y. Liu
          </td>
          <td>2024-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="DNA G-quadruplex(G4) is a guanine-rich single-stranded DNA sequence that spontaneously folds into a spherical four-stranded DNA secondary structure in oncogene promoter sequences and telomeres. G4s are highly associated with the occurrence and development of cancer and have emerged as promising anticancer targets. Natural products have long been important sources of anticancer drug development. In recent years, significant progress has been made in the discovery of natural drugs targeting DNA G4s, with many DNA G4s have been confirmed as promising targets of natural products, including MYC-G4, KRAS-G4, PDGFR-β-G4, BCL-2-G4, VEGF-G4, and telomeric G4. This review summarizes the research progress in discovering natural small molecules that target DNA G4s and their binding mechanisms. It also discusses the opportunities of and challenges in developing drugs targeting DNA G4s. This review will serve as a valuable reference for the research on natural products, particularly in the development of novel antitumor medications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0615bcc8ba2b9aa59d7f6c8fe9659c90b7d6290" target='_blank'>
              [Research progress in targeting DNA G-quadruplexes using natural products].
              </a>
            </td>
          <td>
            Zhi-Yuan Chen, Rong-Shuang Cheng, Wubulikasimu Ayixia, Yu-Shuang Liu, Kai-Bo Wang
          </td>
          <td>2024-07-01</td>
          <td>Zhongguo Zhong yao za zhi = Zhongguo zhongyao zazhi = China journal of Chinese materia medica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Epigenomic mechanisms are critically involved in mediation of genetic and environmental factors that underlie cancer development. Histone modifications represent highly informative epigenomic marks that reveal activation and repression of gene activities and dysregulation of transcriptional control due to tumorigenesis. Here, we present a comprehensive epigenomic and transcriptomic mapping of 18 tumor and 20 non-neoplastic tissues from non-small cell lung adenocarcinoma patients. Our profiling covers 5 histone marks including activating (H3K4me3, H3K4me1, and H3K27ac) and repressive (H3K27me3 and H3K9me3) marks and the transcriptome using only 20 mg of tissue per sample, enabled by low-input omic technologies. Using advanced integrative bioinformatic analysis, we uncovered cancer-driving signaling cascade networks, changes in 3D genome modularity, and differential expression and functionalities of transcription factors and noncoding RNAs. Many of these identified genes and regulatory molecules showed no significant change in their expression or a single epigenomic modality, emphasizing the power of integrative multimodal and multiomic analysis using patient samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1c31ae6b0dfa593dc1e04630030084d32008118" target='_blank'>
              Comprehensive multimodal and multiomic profiling reveals epigenetic and transcriptional reprogramming in lung tumors
              </a>
            </td>
          <td>
            Peiyao Wu, Zhengzhi Liu, Lina Zheng, Zirui Zhou, Wei Wang, Chang Lu
          </td>
          <td>2024-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8e9f04f74a6159b9e49f1e14a0705e366ea67c" target='_blank'>
              Epigenetic alterations affecting hematopoietic regulatory networks as drivers of mixed myeloid/lymphoid leukemia
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Stanley van Herk, Margit Nuetzel, Aniko Sijs-Szabo, Noelia Díaz, Katherine Kelly, Claudia A. J. Erpelinck-Verschueren, Lucia Schwarzfischer-Pfeilschifter, Hanna Stanewsky, Ute Ackermann, Dagmar Glatz, Johanna Raithel, A. Fischer, Sandra Pohl, Anita Rijneveld, Juan M. Vaquerizas, Christian Thiede, C. Plass, Bas J. Wouters, Ruud Delwel, Michael Rehli, C. Gebhard
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facf4dda707b67b3b56d646f95496db5b6cc7df8" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N Anderson
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245be040147fd6375f5eb20fea5e8094910fd2c5" target='_blank'>
              DNA Damage and Chromatin Rearrangement Work Together to Promote Neurodegeneration.
              </a>
            </td>
          <td>
            Harman Sharma, Sushma Koirala, Yee Lian Chew, Anna Konopka
          </td>
          <td>2024-07-08</td>
          <td>Molecular neurobiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Hutchinson-Gilford Progeria Syndrome (HGPS) is a rare genetic condition characterized by features of accelerated aging, and individuals with HGPS seldom live beyond their mid-teens. The syndrome is commonly caused by a point mutation in the LMNA gene which codes for lamin A and its splice variant lamin C, components of the nuclear lamina. The mutation causing HGPS leads to production of a truncated, farnesylated form of lamin A referred to as “progerin.” Progerin is also expressed at low levels in healthy individuals and appears to play a role in normal aging. HGPS is associated with an accumulation of genomic DNA double-strand breaks (DSBs) and alterations in the nature of DSB repair. The source of DSBs in HGPS is often attributed to stalling and subsequent collapse of replication forks in conjunction with faulty recruitment of repair factors to damage sites. In this work, we used a model system involving immortalized human cell lines to investigate progerin-induced genomic damage. Using an immunofluorescence approach to visualize phosphorylated histone H2AX foci which mark sites of genomic damage, we report that cells engineered to express progerin displayed a significant elevation of endogenous damage in the absence of any change in the cell cycle profile or doubling time of cells. Genomic damage was enhanced and persistent in progerin-expressing cells treated with hydroxyurea. Overexpression of wild-type lamin A did not elicit the outcomes associated with progerin expression. Our results show that DNA damage caused by progerin can occur independently from global changes in replication or cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4814100043718a6a0e30f1cd9c037307d90666" target='_blank'>
              Progerin Can Induce DNA Damage in the Absence of Global Changes in Replication or Cell Proliferation
              </a>
            </td>
          <td>
            Liza A. Joudeh, P. Logan Schuck, Nina M. Van, Alannah J. DiCintio, Jason A. Stewart, Alan S. Waldman
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Recent advances in DNA sequencing technology have enabled the precise decoding of genomes in non-model organisms, providing a basis for unraveling the patterns and mechanisms of sex chromosome evolution. Studies of different species have yielded conflicting results regarding the traditional theory that sex chromosomes evolve from autosomes via the accumulation of deleterious mutations and degeneration of the Y (or W) chromosome. The concept of the ‘sex chromosome cycle,’ emerging from this context, posits that at any stage of the cycle (i.e., differentiation, degeneration, or loss), sex chromosome turnover can occur while maintaining stable sex determination. Thus, understanding the mechanisms that drive both the persistence and turnover of sex chromosomes at each stage of the cycle is crucial. In this review, we integrate recent findings on the mechanisms underlying maintenance and turnover, with a special focus on several organisms having unique sex chromosomes. Our review suggests that the diversity of sex chromosomes in the maintenance of stable sex determination is underappreciated and emphasizes the need for more research on the sex chromosome cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ad3585db829ba2d4ab84372e9e70791e552b39" target='_blank'>
              Sex chromosome cycle as a mechanism of stable sex determination
              </a>
            </td>
          <td>
            Shun Hayashi, Takuya Abe, Takeshi Igawa, Yukako Katsura, Yusuke Kazama, Masafumi Nozawa
          </td>
          <td>2024-07-10</td>
          <td>Journal of Biochemistry</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23ed7881dc424017c9838fade2801a5a4db0c319" target='_blank'>
              Bidirectional linkage of DNA barcodes for the multiplexed mapping of higher-order protein interactions in cells.
              </a>
            </td>
          <td>
            Yu Liu, Noah R. Sundah, Nicholas R. Y. Ho, Wanxiang Shen, Yun Xu, Auginia Natalia, Zhonglang Yu, J. Seet, Ching Wan Chan, T. P. Loh, Brian Y Lim, Huilin Shao
          </td>
          <td>2024-06-19</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Cancer develops from abnormal cell growth in the body, causing significant mortalities every year. To date, potent therapeutic approaches have been developed to eradicate tumor cells, but intolerable toxicity and drug resistance can occur in treated patients, limiting the efficiency of existing treatment strategies. Therefore, searching for novel genes critical for cancer progression and therapeutic response is urgently needed for successful cancer therapy. Recent advances in bioinformatics and proteomic techniques have allowed the identification of a novel category of peptides encoded by non-canonical open reading frames (ncORFs) from historically non-coding genomic regions. Surprisingly, many ncORFs express functional microproteins that play a vital role in human cancers. In this review, we provide a comprehensive description of different ncORF types with coding capacity and technological methods in discovering ncORFs among human genomes. We also summarize the carcinogenic role of ncORFs such as pTINCR and HOXB-AS3 in regulating hallmarks of cancer, as well as the roles of ncORFs such as HOXB-AS3 and CIP2A-BP in cancer diagnosis and prognosis. We also discuss how ncORFs such as AKT-174aa and DDUP are involved in anti-cancer drug response and the underestimated potential of ncORFs as therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d95f26adc077c4e04bfda2c2e30184b8f21906d" target='_blank'>
              Exploring the Dark Matter of Human Proteome: The Emerging Role of Non-Canonical Open Reading Frame (ncORF) in Cancer Diagnosis, Biology, and Therapy
              </a>
            </td>
          <td>
            Anni Ge, Curtis Chan, Xiaolong Yang
          </td>
          <td>2024-07-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Transient melting of the duplex-DNA (B-DNA) during DNA transactions allows repeated sequences to fold into non-B DNA structures, including DNA junctions and G-quadruplexes. These noncanonical structures can act as impediments to DNA polymerase progression along the duplex, thereby triggering DNA damage and ultimately jeopardizing genomic stability. Their stabilization by ad hoc ligands is currently being explored as a putative anticancer strategy since it might represent an efficient way to inflict toxic DNA damage specifically to rapidly dividing cancer cells. The relevance of this strategy is only emerging for three-way DNA junctions (TWJs) and, to date, no molecule has been recognized as a reference TWJ ligand, featuring both high affinity and selectivity. Herein, we characterize such reference ligands through a combination of in vitro techniques comprising affinity and selectivity assays (competitive FRET-melting and TWJ Screen assays), functional tests (qPCR and Taq stop assays), and structural analyses (molecular dynamics and NMR investigations). We identify novel azacryptands TrisNP-amphi and TrisNP-ana as the most promising ligands, interacting with TWJs with high affinity and selectivity. These ligands represent new molecular tools to investigate the cellular roles of TWJs and explore how they can be exploited in innovative anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/406c1c1a744a4c655e84090a57ef4d3f115a69ef" target='_blank'>
              Structural Optimization of Azacryptands for Targeting Three-Way DNA Junctions.
              </a>
            </td>
          <td>
            A. Pipier, Titouan Chetot, Apollonia Kalamatianou, Nicolas Martin, Maëlle Caroff, Sébastien Britton, Nicolas Chéron, Lukáš Trantírek, Anton Granzhan, D. Monchaud
          </td>
          <td>2024-06-14</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfce29169bac0d7cbc470f70a1b9115230e5e431" target='_blank'>
              Multi-omics analysis reveals the dynamic interplay between Vero host chromatin structure and function during vaccinia virus infection
              </a>
            </td>
          <td>
            Vrinda Venu, Cullen Roth, Samantha H. Adikari, Eric M Small, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-06-11</td>
          <td>Communications Biology</td>
          <td>1</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba856bced2e48ff7a94743b3236b182aa83cc934" target='_blank'>
              Epigenome editing technologies for discovery and medicine.
              </a>
            </td>
          <td>
            Sean R. McCutcheon, Dahlia Rohm, Nahid Iglesias, C. Gersbach
          </td>
          <td>2024-07-29</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Premature aging is a hallmark of Down syndrome, caused by trisomy of human chromosome 21, but the reason is unclear and difficult to study in humans. We used an aneuploid model in wild yeast to show that chromosome amplification disrupts nutrient-induced cell-cycle arrest, quiescence entry, and healthy aging, across genetic backgrounds and amplified chromosomes. We discovered that these defects are due in part to aneuploidy-induced dysfunction in Ribosome Quality Control (RQC). Compared to euploids, aneuploids entering quiescence display aberrant ribosome profiles, accumulate RQC intermediates, and harbor an increased load of protein aggregates. Although they have normal proteasome capacity, aneuploids show signs of ubiquitin dysregulation, which impacts cyclin abundance to disrupt arrest. Remarkably, inducing ribosome stalling in euploids produces similar aberrations, while up-regulating limiting RQC subunits or proteins in ubiquitin metabolism alleviates many of the aneuploid defects. Our results provide implications for other aneuploidy disorders including Down syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c6d2e4dd1921e05fad92cf19e5a9d35d322983a" target='_blank'>
              Premature aging in aneuploid yeast is caused in part by aneuploidy-induced defects in Ribosome Quality Control
              </a>
            </td>
          <td>
            Leah E. Escalante, James Hose, Hollis Howe, Norah Paulsen, Michael Place, A. Gasch
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe961497d985de77bb27297cd1364379e55768f" target='_blank'>
              Circular RNA in cancer.
              </a>
            </td>
          <td>
            Vanessa M. Conn, A. Chinnaiyan, Simon J. Conn
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>163</td>
        </tr>

        <tr id="Once inside host cells, retroviruses generate a double-stranded DNA copy of their RNA genomes via reverse transcription inside a viral core, and this viral DNA is subsequently integrated into the genome of the host cell. Before integration can occur, the core must cross the cell cortex, be transported through the cytoplasm, and enter the nucleus. Retroviruses have evolved different mechanisms to accomplish this journey. This review examines the various mechanisms retroviruses, especially HIV-1, have evolved to commute throughout the cell. Retroviruses cross the cell cortex while modulating actin dynamics and use microtubules as roads while connecting with microtubule-associated proteins and motors to reach the nucleus. Although a clearer picture exists for HIV-1 compared with other retroviruses, there is still much to learn about how retroviruses accomplish their commute.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14618b907a9962eaa0c1724412e3b1d953962787" target='_blank'>
              From Entry to the Nucleus: How Retroviruses Commute.
              </a>
            </td>
          <td>
            Camila E Osega, F. Bustos, Gloria Arriagada
          </td>
          <td>2024-06-07</td>
          <td>Annual review of virology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Despite significant advances in our understanding of genetic cancer susceptibility, known inherited cancer predisposition syndromes explain at most 20% of early-onset cancers. As early-onset cancer prevalence continues to increase, the need to assess previously inaccessible areas of the human genome, harnessing a trio or quad family-based architecture for variant filtration, may reveal further insights into cancer susceptibility. To assess a broader spectrum of variation than can be ascertained by multi-gene panel sequencing, or even whole genome sequencing with short reads, we employed long read whole genome sequencing using an Oxford Nanopore Technology (ONT) PromethION of 3 families containing an early-onset cancer proband using a trio or quad family architecture. Analysis included 2 early-onset colorectal cancer family trios and one quad consisting of two siblings with testicular cancer, all with unaffected parents. Structural variants (SVs), epigenetic profiles and single nucleotide variants (SNVs) were determined for each individual, and a filtering strategy was employed to refine and prioritize candidate variants based on the family architecture. The family architecture enabled us to focus on inapposite variants while filtering variants shared with the unaffected parents, significantly decreasing background variation that can hamper identification of potentially disease causing differences. Candidate de novo and compound heterozygous variants were identified in this way. Gene expression, in matched neoplastic and pre-neoplastic lesions, was assessed for one trio. Our study demonstrates the feasibility of a streamlined analysis of genomic variants from long read ONT whole genome sequencing and a way to prioritize key variants for further evaluation of pathogenicity, while revealing what may be missing from panel based analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579a5b24c1657cfe092825293758535eef92a0f6" target='_blank'>
              Exploring the genetic and epigenetic underpinnings of early-onset cancers: Variant prioritization for long read whole genome sequencing from family cancer pedigrees
              </a>
            </td>
          <td>
            M. Kramer, Sara Goodwin, R. Wappel, Matilde Borio, K. Offit, D. Feldman, Z. Stadler, W. R. McCombie
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="The fidelity of genetic information is essential for cellular function and viability. DNA double-strand breaks (DSBs) pose a significant threat to genome integrity, necessitating efficient repair mechanisms. While the predominant repair strategies are usually accurate, paradoxically, error-prone pathways also exist. This review explores recent advances and our understanding of microhomology-mediated end joining (MMEJ), an intrinsically mutagenic DSB repair pathway conserved across organisms. Central to MMEJ is the activity of DNA polymerase theta (Polθ), a specialized polymerase that fuels MMEJ mutagenicity. We examine the molecular intricacies underlying MMEJ activity and discuss its function during mitosis, where the activity of Polθ emerges as a last-ditch effort to resolve persistent DSBs, especially when homologous recombination is compromised. We explore the promising therapeutic applications of targeting Polθ in cancer treatment and genome editing. Lastly, we discuss the evolutionary consequences of MMEJ, highlighting its delicate balance between protecting genome integrity and driving genomic diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9839d026eb499a9c1802545a526640975b43a205" target='_blank'>
              Microhomology-Mediated End Joining Chronicles: Tracing the Evolutionary Footprints of Genome Protection.
              </a>
            </td>
          <td>
            Agnel Sfeir, M. Tijsterman, M. McVey
          </td>
          <td>2024-06-10</td>
          <td>Annual review of cell and developmental biology</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Single-stranded DNA (ssDNA) is essential for various DNA-templated processes in both eukaryotes and prokaryotes. However, comprehensive characterizations of ssDNA still lag in plants compared to non-plant systems. Here, we conducted in situ S1-seq (ISS1-seq), with starting gDNA ranging from 5 µg to 250 ng, followed by comprehensive characterizations of ssDNA in rice (Oryza sativa L.). We found that ssDNA loci were substantially associated with a subset of non-B DNA structures and functional genomic loci. Subtypes of ssDNA loci had distinct epigenetic features. Importantly, ssDNA may act alone or partly coordinate with non-B DNA structures, functional genomic loci, or epigenetic marks to actively or repressively modulate gene transcription, which is genomic-region-dependent and associated with the distinct accumulation of RNA Pol II. Moreover, distinct types of ssDNA had differential impacts on the activities and evolution of TEs (especially common or conserved TEs) in the rice genome. Our study showcases an antibody-independent technique for characterizing non-B DNA structures or functional genomic loci in plants. It lays the groundwork and fills a crucial gap for further exploration of ssDNA, non-B DNA structures, or functional genomic loci, thereby advancing our understanding of their biology in plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a41f2ac1057a9cc0aff5ab06cd10c7413719a7" target='_blank'>
              Genome-wide characterization of single-stranded DNA in rice.
              </a>
            </td>
          <td>
            Yulian Peng, Pengtao Zhao, Zhaoguo Li, Ning Mu, Shentong Tao, Yilong Feng, Xuejiao Cheng, Wenli Zhang
          </td>
          <td>2024-06-25</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="G-quadruplexes (G4s) are guanine-rich non-canonical secondary structures of nucleic acids that were identified in vitro almost half a century ago. Starting from the early 1980s, these structures were also observed in eukaryotic cells, first at the telomeric level and later in regulatory regions of cancer-related genes, in regulatory RNAs and within specific cell compartments such as lysosomes, mitochondria, and ribosomes. Because of the involvement of these structures in a large number of biological processes and in the pathogenesis of several diseases, including cancer, the interest in G4 targeting has exponentially increased in the last few years, and a great number of novel G4 ligands have been developed. Notably, G4 ligands represent a large family of heterogeneous molecules that can exert their functions by recognizing, binding, and stabilizing G4 structures in multiple ways. Regarding anti-cancer activity, the efficacy of G4 ligands was originally attributed to the capability of these molecules to inhibit the activity of telomerase, an enzyme that elongates telomeres and promotes endless replication in cancer cells. Thereafter, novel mechanisms through which G4 ligands exert their antitumoral activities have been defined, including the induction of DNA damage, control of gene expression, and regulation of metabolic pathways, among others. Here, we provided a perspective on the structure and function of G4 ligands with particular emphasis on their potential role as antitumoral agents. In particular, we critically examined the problems associated with the clinical translation of these molecules, trying to highlight the main aspects that should be taken into account during the phases of drug design and development. Indeed, taking advantage of the successes and failures, and the more recent technological progresses in the field, it would be possible to hypothesize the development of these molecules in the future that would represent a valid option for those cancers still missing effective therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac0182d1bedc1581bd4f5ff74bba89623ffe175" target='_blank'>
              Therapeutic Use of G4-Ligands in Cancer: State-of-the-Art and Future Perspectives
              </a>
            </td>
          <td>
            S. Iachettini, A. Biroccio, P. Zizza
          </td>
          <td>2024-06-01</td>
          <td>Pharmaceuticals</td>
          <td>2</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/855843ba00188002869d72dc3ec46da982ab1cf4" target='_blank'>
              Genome-wide DNA methylation in relation to ARID1A deficiency in ovarian clear cell carcinoma
              </a>
            </td>
          <td>
            Shang Li, G. Meersma, J. Kupryjańczyk, Steven de Jong, G. Wisman
          </td>
          <td>2024-06-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Cancer genomic studies have identified frequent alterations in components of the SWI/SNF (SWItch/Sucrose Non- Fermenting) chromatin remodeling complex including SMARCA4 and ARID1A. Importantly, clinical reports indicate that SMARCA4-mutant lung cancers respond poorly to immunotherapy and have dismal prognosis. However, the mechanistic basis of immunotherapy resistance is unknown. Here, we corroborated the clinical findings by using immune-humanized, syngeneic, and genetically engineered mouse models of lung cancer harboring SMARCA4 deficiency. Specifically, we show that SMARCA4 loss caused decreased response to anti-PD1 immunotherapy associated with significantly reduced infiltration of dendritic cells (DCs) and CD4+ T cells into the tumor microenvironment (TME). Mechanistically, we show that SMARCA4 loss in tumor cells led to profound downregulation of STING, IL1β and other components of the innate immune system as well as inflammatory cytokines that are required for efficient recruitment and activity of immune cells. We establish that this deregulation of gene expression is caused by cancer cell-intrinsic reprogramming of the enhancer landscape with marked loss of chromatin accessibility at enhancers of genes involved in innate immune response such as STING, IL1β, type I IFN and inflammatory cytokines. Interestingly, we observed that transcription factor NF-κB binding motif was highly enriched in enhancers that lose accessibility upon SMARCA4 deficiency. Finally, we confirmed that SMARCA4 and NF-κB co-occupy the same genomic loci on enhancers associated with STING and IL1β, indicating a functional interplay between SMARCA4 and NF-κB. Taken together, our findings provide the mechanistic basis for the poor response of SMARCA4-mutant tumors to anti-PD1 immunotherapy and establish a functional link between SMARCA4 and NF-κB on innate immune and inflammatory gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3fdeb7ebdd844e1757256a995b264456319e406" target='_blank'>
              SMARCA4 mutation induces tumor cell-intrinsic defects in enhancer landscape and resistance to immunotherapy
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Yonathan Lissanu
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer (mCRPC) is a lethal disease that resists therapy targeting androgen signaling, the primary driver of prostate cancer. mCRPC resists androgen receptor (AR) inhibitors by amplifying AR signaling or by evolving into therapy-resistant subtypes that do not depend on AR. Elucidation of the epigenetic underpinnings of these subtypes could provide important insights into the drivers of therapy resistance. In this study, we produced chromatin accessibility maps linked to the binding of lineage-specific transcription factors (TF) by performing ATAC sequencing on 70 mCRPC tissue biopsies integrated with transcriptome and whole genome sequencing. mCRPC had a distinct global chromatin accessibility profile linked to AR function. Analysis of TF occupancy across accessible chromatin revealed 203 TFs associated with mCRPC subtypes. Notably, ZNF263 was identified as a putative prostate cancer TF with a significant impact on gene activity in the double-negative (AR- neuroendocrine-) subtype, potentially activating MYC targets. Overall, this analysis of chromatin accessibility in mCRPC provides valuable insights into epigenetic changes that occur during progression to mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac71bf183535cdb84fd367ef22969e3b56650628" target='_blank'>
              An Atlas of Accessible Chromatin in Advanced Prostate Cancer Reveals the Epigenetic Evolution during Tumor Progression.
              </a>
            </td>
          <td>
            Raunak Shrestha, L. Chesner, Meng Zhang, Stanley Zhou, A. Foye, A. Lundberg, Alana S Weinstein, M. Sjöström, Xiaolin Zhu, Thaidy Moreno-Rodriguez, Haolong Li, Su C/Pcf West Coast Prostate Cancer Dream Team, J. Alumkal, Rahul Aggarwal, Eric J Small, Mathieu Lupien, David A Quigley, Felix Y Feng
          </td>
          <td>2024-07-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/529c303e4a8627e247a69d8654c825f73c988e72" target='_blank'>
              Integrative analysis of ultra-deep RNA-seq reveals alternative promoter usage as a mechanism of activating oncogenic programmes during prostate cancer progression.
              </a>
            </td>
          <td>
            Meng Zhang, M. Sjöström, Xiekui Cui, A. Foye, K. Farh, Raunak Shrestha, A. Lundberg, H. Dang, Haolong Li, Phillip G Febbo, Rahul Aggarwal, J. Alumkal, Eric J Small, C. Maher, Felix Y Feng, David A Quigley
          </td>
          <td>2024-06-13</td>
          <td>Nature cell biology</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a75dd48ccfcd6591fcd252d1a4a9a583e49480b" target='_blank'>
              Cockayne Syndrome Linked to Elevated R-Loops Induced by Stalled RNA Polymerase II during Transcription Elongation
              </a>
            </td>
          <td>
            Xuan Zhang, Jun Xu, Jing Hu, Sitao Zhang, Yajing Hao, Dongyang Zhang, Hao Qian, Dong Wang, Xiang-Dong Fu
          </td>
          <td>2024-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Transcription and splicing errors in cancer cells generate erroneous transcripts. Since erroneous transcripts are degraded by the nonsense-mediated mRNA decay (NMD) pathway, whether they are junk or could be functional has been overlooked and understudied. We addressed this question by first performing a pan-cancer analysis and identified substantial erroneous noncoding transcripts (ENT) in cancers. Given that RNA/DNA binding domains (DBD) were predicted in ENTs, we deleted predicted DBDs in multiple ENTs in multiple cell lines, with RNA-sequencing and cell experiments before and after DBD deletion. DBD deletion caused significantly changed expression of ENTs’ target genes (whose promoter regions contain ENTs’ DNA binding sites, DBS) and changed cell migration and proliferation ability, indicating that many ENTs can transcriptionally regulate genes. Tightly coupled data analysis and experiments reveal that ENTs’ functions are highly cancer- and cellular-context specific, making ENTs a new class of safe and specific targets for noncoding RNA-based cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e45af8d1c765a560ff555893aa2ed200e47f3c" target='_blank'>
              Many erroneous noncoding transcripts in cancer cells can highly specifically regulate cancer-related genes and pathways
              </a>
            </td>
          <td>
            Sha He, Wei Xiong, Jianping Huo, Jie Lin, Jianmin Li, Hao Zhu
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Circular RNAs are a novel class of RNA transcripts, which regulate important cellular functions in health and disease. Herein, we report on the functional relevance of the circPCMTD1 transcript in acute leukemias. In screening experiments, we found that circPCMTD1 depletion strongly inhibited the proliferative capacity of leukemic cells with BCR-ABL translocations. Mass cytometry experiments identified the aberrant activation of the DNA damage response as an early downstream event of circPCMTD1 depletion. In in vivo experiments, circPCMTD1 targeting prolonged the survival of mice engrafted with leukemic blasts harboring the Philadelphia chromosome. Mechanistically, we found that circPCMTD1 was enriched in the cytoplasm and associated with the ribosomes of the leukemic cells. We detected a cryptic open reading frame within the circPCMTD1 sequence and found that circPCMTD1 could generate a peptide product. The circPCMTD1-derived peptide interacted with proteins of the BTR complex and enhanced BTR complex formation, thereby increasing tolerance to genotoxic stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/015fb549e9df8d93cfd780c7c6932ae735f6848d" target='_blank'>
              circPCMTD1: A protein-coding circular RNA that regulates DNA damage response in BCR/ABL-positive leukemias
              </a>
            </td>
          <td>
            D. Papaioannou, Amog P. Urs, Rémi Buisson, A. Petri, R. Kulkarni, D. Nicolet, Lauren A. Woodward, C. Goda, Krzysztof Mrózek, Gregory K. Behbehani, Sakari Kauppinen, Ann-Kathrin Eisfeld, I. Aifantis, Guramrit Singh, A. Dorrance, Ramiro Garzon
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>86</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5073eeabc50bea9fdc09e2d2eeb42d91221fda8" target='_blank'>
              Integrative genomics identifies SHPRH as a tumor suppressor gene in lung adenocarcinoma that regulates DNA damage response.
              </a>
            </td>
          <td>
            A. Nagelberg, T. Sihota, Yu-Chi Chuang, Rocky Shi, Justine L M Chow, John English, C. MacAulay, Stephen Lam, Wan L Lam, William W Lockwood
          </td>
          <td>2024-06-18</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Emerging evidence indicates that transfer RNA (tRNA)-derived small RNAs (tsRNAs), originated from tRNA with high abundance RNA modifications, play an important role in many complex physiological and pathological processes. However, the biological functions and regulatory mechanisms of modified tsRNAs in cancer remain poorly understood. Here, it is screened for and confirmed the presence of a novel m7G-modified tsRNA, m7G-3'-tiRNA LysTTT (mtiRL), in a variety of chemical carcinogenesis models by combining small RNA sequencing with an m7G small RNA-modified chip. Moreover, it is found that mtiRL, catalyzed by the tRNA m7G-modifying enzyme mettl1, promotes bladder cancer (BC) malignancy in vitro and in vivo. Mechanistically, mtiRL is found to specifically bind the oncoprotein Annexin A2 (ANXA2) to promote its Tyr24 phosphorylation by enhancing the interactions between ANXA2 and Yes proto-oncogene 1 (Yes1), leading to ANXA2 activation and increased p-ANXA2-Y24 nuclear localization in BC cells. Together, these findings define a critical role for mtiRL and suggest that targeting this novel m7G-modified tsRNA can be an efficient way for to treat BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/678ab210e5841f24ef42f868385c9545eca6bd2f" target='_blank'>
              A Novel tsRNA, m7G-3' tiRNA LysTTT, Promotes Bladder Cancer Malignancy Via Regulating ANXA2 Phosphorylation.
              </a>
            </td>
          <td>
            Xiaoling Ying, Wenyu Hu, Yapeng Huang, Yifan Lv, Ding Ji, Cong Chen, Baotong Yang, Chengcheng Zhang, Yaomin Liang, Haiqing Zhang, Mingrui Liu, Gang Yuan, Wenqi Wu, Weidong Ji
          </td>
          <td>2024-06-18</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c5a9b1b536dcc4bfe3e239817233ef120675cf" target='_blank'>
              Werner helicase interacting protein 1 contributes to G-quadruplex processing in human cells
              </a>
            </td>
          <td>
            Lili Hegedűs, Á. Tóth, Gábor M. Harami, J. Pálinkás, Nargis Karatayeva, Enikő Sajben-Nagy, Szabolcs Bene, Sara Afzali Jaktajdinani, Mihaly Kovacs, Szilvia Juhász, Peter Burkovics
          </td>
          <td>2024-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The DNA modifications, 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), represent powerful epigenetic regulators of temporal and spatial gene expression. Yet, how the cooperation of these genome-wide, epigenetic marks determine unique transcriptional signatures across different brain cell populations is unclear. Here we applied Nanopore sequencing of native DNA to obtain a complete, genome-wide, single-base resolution atlas of 5mC and 5hmC modifications in neurons, astrocytes and microglia in the mouse cortex (99% genome coverage, 40 million CpG sites). In tandem with RNA sequencing, analysis of 5mC and 5hmC patterns across cell types reveals astrocytes drive uniquely high brain 5hmC levels and support two decades of research regarding methylation patterns, gene expression and alternative splicing, benchmarking this resource. As such, we provide the most comprehensive DNA methylation data in mouse brain as an interactive, online tool (NAM-Me, https://olsenlab.shinyapps.io/NAMME/) to serve as a resource dataset for those interested in the methylome landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60076a55629be60f22fe1afccd00156f5bc051e7" target='_blank'>
              Decoding the Epigenetic Landscape: Insights into 5mC and 5hmC Patterns in Mouse Cortical Cell Types
              </a>
            </td>
          <td>
            Xiaoran Wei, Jiangtao Li, Zuolin Cheng, Songtao Wei, Guoqiang Yu, Michelle L. Olsen
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="To protect genome integrity from pervasive threats of damage and prevent diseases like cancer, cells employ an integrated network of signalling pathways called the DNA damage response. These pathways involve both protein and RNA components which can act within the damaged cell or be transferred intercellularly to influence population-wide responses to damage. Here, we show that radioprotection can be conferred by damage-derived exosomes and is dependent on YBX1-packaged Y3-derived ysRNA. In recipient cells, ysRNA are methylated on cytosine by an RNA methyltransferase NSUN2, and bound by m5C reader, YBX1. YBX1/ysRNA localises at double strand break (DSB) sites to promote efficient DNA repair and cell survival through complex formation with PARP1. YBX1 modulates PARP1 auto-modification by facilitating ysRNA ADP-ribosylation, promoting increased PARP1 residency at DSBs. Our data highlight an unprecedented role for these under-studied species of small non-coding RNA, identifying them as a novel substrate for PARP1 mediated ADP-ribosylation and their function in DNA repair.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/806820b6f3fdaccb30dad4e1599912f2b6c0fd82" target='_blank'>
              Y RNA-derived fragments in a complex with YBX1 modulate PARP1 residency at DNA double strand breaks
              </a>
            </td>
          <td>
            Annabelle Shaw, Kamal Ajit, M. Gullerová
          </td>
          <td>2024-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Cancer is the second leading cause of death worldwide, and although there have been advances in treatments, including immunotherapies, these often require biopsies which can be costly and invasive to obtain. Due to lack of pre-emptive cancer detection methods, many cases of cancer are detected at a late stage when the definitive symptoms appear. Plasma samples are relatively easy to obtain, and they can be used to monitor the molecular signatures of ongoing processes in the body. Profiling cell-free DNA is a popular method for monitoring cancer, but only a few studies have explored the use of cell-free RNA (cfRNA), which shows the recent footprint of systemic transcription. Here we developed FastNeo, a computational method for detecting known neoepitopes in human cfRNA. We show that neoepitopes and other biomarkers detected in cfRNA can discern Hepatocellular carcinoma (HCC) patients from the healthy patients with a sensitivity of 0.84 and a specificity of 0.79. For colorectal cancer we achieve a sensitivity of 0.87 and a specificity of 0.8. An important advantage of our cfRNA based approach is that it also reports putative neoepitopes which are important for therapeutic purposes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39049eaeea832dff014994be3a821e1580cf6286" target='_blank'>
              Detecting known neoepitopes, gene fusions, transposable elements, and circular RNAs in cell-free RNA
              </a>
            </td>
          <td>
            Mayank Mahajan, Martin Hemberg
          </td>
          <td>2024-06-09</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In recent years, there has been increased focus on exploring the role the non-protein-coding genome plays in Mendelian disorders. One class of particular interest is long non-coding RNAs (lncRNAs), which has recently been implicated in the regulation of diverse molecular processes. However, because lncRNAs do not encode protein, there is uncertainty regarding what constitutes a pathogenic lncRNA variant, and thus annotating such elements is challenging. The Developmental Genome Anatomy Project (DGAP) and similar projects recruit individuals with apparently balanced chromosomal abnormalities (BCAs) that disrupt or dysregulate genes in order to annotate the human genome. We hypothesized that rearrangements disrupting lncRNAs could be the underlying genetic etiology for the phenotypes of a subset of these individuals. Thus, we assessed 279 cases with BCAs and selected 191 cases with simple BCAs (breakpoints at only two genomic locations) for further analysis of lncRNA disruptions. From these, we identified 66 cases in which the chromosomal rearrangements directly disrupt lncRNAs. Strikingly, the lncRNAs MEF2C-AS1 and ENSG00000257522 are each disrupted in two unrelated cases. Furthermore, in 30 cases, no genes of any other class aside from lncRNAs are directly disrupted, consistent with the hypothesis that lncRNA disruptions could underly the phenotypes of these individuals. To showcase the power of this genomic approach for annotating lncRNAs, here we focus on clinical reports and genetic analysis of two individuals with BCAs and additionally highlight six individuals with likely developmental etiologies due to lncRNA disruptions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4494659e85fb73d7f4a80f0d544797b5da37da2" target='_blank'>
              Rare germline disorders implicate long non-coding RNAs disrupted by chromosomal structural rearrangements
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-06-19</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="The precise spatial and temporal orchestration of gene expression is crucial for the ontogeny of an organism and is mainly governed by transcription factors (TFs). The mechanism of recognition of cognate sites amid millions of base pairs in the genome by TFs is still incompletely understood. In this study, we focus on DNA sequence composition, shape, and flexibility preferences of 28 quintessential TFs from Drosophila melanogaster that are critical to development and body patterning mechanisms. Our study finds that TFs exhibit distinct predilections for DNA shape, flexibility, and sequence compositions in the proximity of transcription factor binding sites (TFBSs). Notably, certain zinc finger proteins prefer GC-rich areas with less negative propeller twist, while homeodomains mainly seek AT-rich regions with a more negative propeller twist at their sites. Intriguingly, while numerous cofactors share similar binding site preferences and bind closer to each other in the genome, some cofactors that have different preferences bind farther apart. These findings shed light on TF DNA recognition and provide novel insights into possible cofactor binding and transcriptional regulation mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6708e27e00051dd7963988835acb3d777430c57" target='_blank'>
              Discerning the Role of DNA Sequence, Shape, and Flexibility in Recognition by Drosophila Transcription Factors.
              </a>
            </td>
          <td>
            Smrithi Murthy, Upalabdha Dey, Kaushika Olymon, Eshan Abbas, V. Yella, Aditya Kumar
          </td>
          <td>2024-06-20</td>
          <td>ACS chemical biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="The relevant role of LINE-1 (L1) retrotransposition in cancer has been recurrently demonstrated in recent years. However, their repetitive nature hampers their identification and detection, hence remaining inaccessible for clinical practice. Also, its clinical relevance for cancer patients is still limited. Here, we develop a new method to quantify L1 activation, called RetroTest, based on targeted sequencing and a sophisticated bioinformatic pipeline, allowing its application in tumor biopsies. First, we performed the benchmarking of the method and confirmed its high specificity and reliability. Then, we unravel the L1 activation in HNSCC according to a more extensive cohort including all the HNSCC tumor stages. Our results confirm that RetroTest is remarkably efficient for L1 detection in tumor biopsies, reaching a high sensitivity and specificity. In addition, L1 retrotransposition estimation reveals a surprisingly early activation in HNSCC progression, contrary to its classical association with advanced tumor stages. This early activation together with the genomic mutational profiling of normal adjacent tissues supports field cancerization process in this tumor. These results underline the importance of estimating L1 retrotransposition in clinical practice towards an earlier and more efficient diagnosis in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ae21a6395bf2fa53698eb0f53f5dc6c7132bfe" target='_blank'>
              RetroTest unravels LINE-1 retrotransposition in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, Ana Oitaben, Sonia Zumalave, Bernardo Rodriguez-4 Martin, María Gallardo-Gómez, Martín Santamarina, Ana Pequeño-Valtierra, L. Juaneda-Magdalena, Ramón García-Escudero, J. López-Cedrún, Máximo Fraga, J. Tubío, Mónica Martínez-Fernández, Monica Martínez
          </td>
          <td>2024-07-24</td>
          <td>None</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chromatin states, fundamental to gene regulation and cellular identity, are defined by a unique combination of histone post-translational modifications. Despite their importance, comprehensive patterns within chromatin state sequences, which could provide insights into key biological functions, remain largely unexplored. In this study, we introduce ChromBERT, a BERT-based model specifically designed to detect distinct patterns of chromatin state annotation data sequences. Notably, ChromBERT was pre-trained on promoter regions across a diverse range of epigenomes and subsequently fine-tuned using a dataset from multiple cell lines where RNA-seq data were available, highlighting the model’s ability to discern conserved chromatin state patterns within these regions. In addition to its predictive powers across tasks, evidenced by high AUC scores, ChromBERT provides further analysis through the incorporation of motif clustering using Dynamic Time Warping (DTW). This method enhances the model’s ability to dissect chromatin state sequence motifs, typically involving transcription and enhancer sites. The introduction of motif clustering with DTW into ChromBERT’s workflow is poised to facilitate the discovery of genomic regions linked to novel biological functions, deepening our understanding of chromatin state dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758a0c4bd29ca771e1573835ff87961f8849bdb7" target='_blank'>
              ChromBERT: Uncovering Chromatin State Motifs in the Human Genome Using a BERT-based Approach
              </a>
            </td>
          <td>
            Seohyun Lee, Che Lin, Chien-Yu Chen, Ryuichiro Nakato
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0a5950869ddd6a21b7f0f323fc01ad3c1371ba9" target='_blank'>
              Altered chromatin topologies caused by balanced chromosomal translocation lead to central iris hypoplasia
              </a>
            </td>
          <td>
            Wenmin Sun, Dan Xiong, Jiamin Ouyang, Xueshan Xiao, Yi Jiang, Yingwei Wang, Shi-qiang Li, Ziying Xie, Junwen Wang, Zhonghui Tang, Qingjiong Zhang
          </td>
          <td>2024-06-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="All bacteria encode a multifunctional DNA-binding protein, DnaA, which initiates chromosomal replication. Despite having the most complex, segmented bacterial genome, little is known about Borrelia burgdorferi DnaA and its role in maintaining the spirochete’s physiology. We utilized inducible CRISPR-interference and overexpression to modulate cellular levels of DnaA to better understand this essential protein. Dysregulation of DnaA, either up or down, significantly slowed replication and increased or decreased cell lengths. Using fluorescent microscopy, we found the DnaA CRISPRi mutants had increased numbers of chromosomes with irregular spacing patterns. The DnaA-depleted spirochetes also exhibited a significant defect in helical morphology. RNA-seq of the conditional mutants showed significant changes in the levels of transcripts involved with flagellar synthesis, elongation, cell division, virulence, and other functions. These findings demonstrate that the DnaA plays a commanding role in maintaining borrelial growth dynamics and protein expression, which are essential for the survival of the Lyme disease spirochete.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ef9fa4e6497e13499446134decf2617e58008ee" target='_blank'>
              DnaA modulates gene expression and morphologyof the Lyme Disease spirochete
              </a>
            </td>
          <td>
            A. Krusenstjerna, Nerina Jusufovic, Timothy C. Saylor, Brian Stevenson
          </td>
          <td>2024-06-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Field cancerization is suggested to arise from imbalanced differentiation in individual basal progenitor cells leading to clonal expansion of mutant cells that eventually replace the epithelium, although without evidence. Methods We performed deep sequencing analyses to characterize the genomic and transcriptomic landscapes of field change in two patients with synchronous aerodigestive tract tumors. Results Our data support the emergence of numerous genetic alterations in cancer-associated genes but refutes the hypothesis that founder mutation(s) underpin this phenomenon. Mutational signature analysis identified defective homologous recombination as a common underlying mutational process unique to synchronous tumors. Discussion Our analyses suggest a common etiologic factor defined by mutational signatures and/or transcriptomic convergence, which could provide a therapeutic opportunity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae797c96e298d92f976b952e9d1ccd0f24f2a970" target='_blank'>
              Transcriptomic convergence despite genomic divergence drive field cancerization in synchronous squamous tumors
              </a>
            </td>
          <td>
            Qiu Xuan Tan, Nicholas B. Shannon, Weng Khong Lim, J. X. Teo, Daniel R. Y. Yap, Sze Min Lek, Joey W. S. Tan, Shih Jia, Josephine Hendrikson, Ying Liu, Gillian Ng, Clara Y. L. Chong, Wanyu Guo, Kelvin K. N. Koh, C. Ng, Vikneswari Rajasegaran, Jolene S.M. Wong, Chin Jin Seo, C. Ong, Tony K.H. Lim, Bin Tean Teh, Oi Lian Kon, C. Chia, Khee Chee, N. G. Iyer, C. Ong, M. Shammas, Sarah Mazzilli, Prajish Iyer
          </td>
          <td>2024-06-13</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Aneuploidy, or a change in the number of whole chromosomes or chromosome arms, is a near-universal feature of cancer. Chromosomes affected by aneuploidy are not random, with observed cancer-specific and tissue-specific patterns. Recent advances in genome engineering methods have allowed the creation of models with targeted aneuploidy events. These models can be used to uncover the downstream effects of individual aneuploidies on cancer phenotypes including proliferation, apoptosis, metabolism, and immune signaling. Here, we review the current state of research into the patterns of aneuploidy in cancer and their impact on signaling pathways and biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53976b63387a7558e4fe16be8478ae589398c516" target='_blank'>
              Patterns of Aneuploidy and Signaling Consequences in Cancer.
              </a>
            </td>
          <td>
            Nadja Zhakula-Kostadinova, Alison M Taylor
          </td>
          <td>2024-06-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Different developmental genes shape frequent dynamic inter-chromosomal contacts with rDNA units in human and Drosophila cells. In the course of differentiation, changes in these contacts occur, coupled with changes in the expression of hundreds of rDNA-contacting genes. The data suggest a possible role of nucleoli in the global regulation of gene expression. However, the mechanism behind the specificity of these inter-chromosomal contacts, which are rebuilt in every cell cycle, is not yet known. Here, we describe the strong association of rDNA-contacting genes with numerous long intergenic non-coding RNAs (lincRNAs) in HEK293T cells and in initial and differentiated K562 cells. We observed that up to 600 different lincRNAs were preferentially co-expressed with multiple overlapping sets of rDNA-contacting developmental genes, and there was a strong correlation between the genomic positions of rDNA-contacting genes and lincRNA mappings. These two findings suggest that lincRNAs might guide the corresponding developmental genes toward rDNA clusters. We conclude that the inter-chromosomal interactions of rDNA-contacting genes with nucleoli might be guided by lincRNAs, which might physically link particular genomic regions with rDNA clusters.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d8a741feda6e7fc9cb28c1e7f1e8895309879c0" target='_blank'>
              Preferential Co-Expression and Colocalization of rDNA-Contacting Genes with LincRNAs Suggest Their Involvement in Shaping Inter-Chromosomal Interactions with Nucleoli
              </a>
            </td>
          <td>
            N. Tchurikov, I. R. Alembekov, E. S. Klushevskaya, A. N. Kretova, Viktoriya N. Lukicheva, V. Chechetkin, G. I. Kravatskaya, Y. V. Kravatsky
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract The DNA‐dependent protein kinase (DNA‐PK), catalytic subunit, also known as DNA‐PKcs, is complexed with the heterodimer Ku70/Ku80 to form DNA‐PK holoenzyme, which is well recognized as initiator in the nonhomologous end joining (NHEJ) repair after double strand break (DSB). During NHEJ, DNA‐PKcs is essential for both DNA end processing and end joining. Besides its classical function in DSB repair, DNA‐PKcs also shows multifaceted functions in various biological activities such as class switch recombination (CSR) and variable (V) diversity (D) joining (J) recombination in B/T lymphocytes development, innate immunity through cGAS–STING pathway, transcription, alternative splicing, and so on, which are dependent on its function in NHEJ or not. Moreover, DNA‐PKcs deficiency has been proven to be related with human diseases such as neurological pathogenesis, cancer, immunological disorder, and so on through different mechanisms. Therefore, it is imperative to summarize the latest findings about DNA‐PKcs and diseases for better targeting DNA‐PKcs, which have shown efficacy in cancer treatment in preclinical models. Here, we discuss the multifaceted roles of DNA‐PKcs in human diseases, meanwhile, we discuss the progresses of DNA‐PKcs inhibitors and their potential in clinical trials. The most updated review about DNA‐PKcs will hopefully provide insights and ideas to understand DNA‐PKcs associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc4d3c7ebac68c037dd494f0a2f6295735d5a4a5" target='_blank'>
              The multifaceted functions of DNA‐PKcs: implications for the therapy of human diseases
              </a>
            </td>
          <td>
            Jinghong Wu, Liwei Song, Mingjun Lu, Qing Gao, Shaofa Xu, Ping-Kun Zhou, Teng Ma
          </td>
          <td>2024-06-19</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Cancer is one of the main reasons for death, and it threatens human life and health. Both the environment and genes can lead to cancers. It dates back more than a million years; more importantly, tumor cells can not be detected until they grow to a large number. Currently, cancers are treated with surgical excision or non-surgical procedures. By studying the interaction between ncRNAs and PKM2, we aim to provide new targets for diagnosis, treatment, and prognosis for cancers. Read relevant articles and made a summary and classification. Non-coding RNAs (ncRNAs) are RNAs that do not code for proteins. They perform a function in transcription and translation and can be used as targets for cancer therapy. Pyruvate kinase M2 (PKM2) is a form of PKM, and it catalyzes the glycolysis of the final cellular processes to promote tumorigenesis. Not only that, but it also plays non-metabolic functions, including the expression of the gene, cell proliferation, cell migration, and tumor angiogenesis in cancer cells. The existing studies have found that microRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs) can promote or inhibit the aerobic glycolysis of cancer cells by affecting PKM2, which increases or decrease the risk of cancers and affect the progression of cancers. This review focuses on the mechanism of ncRNAs regulating PKM2 in cancers and summarizes the roles of some ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135ae3dc56b2a8f0f10f7d3ae82e64ec70f96983" target='_blank'>
              Emerging Roles of ncRNAs Regulating PKM2 in Cancer Progression.
              </a>
            </td>
          <td>
            Haimei Xie, Jia Yu, Zhiwen Ou, Xiaoyong Lei, Xiaoyan Yang
          </td>
          <td>2024-07-15</td>
          <td>Current pharmaceutical biotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA, is assaulted by endogenous and exogenous agents that lead to the formation of damage. In order to maintain genome integrity DNA repair pathways must be efficiently activated to prevent mutations and deleterious chromosomal rearrangements. Conversely, genome flexibility is also necessary to allow genetic diversity and evolution. The antagonist interaction between maintenance of genome integrity and flexibility determines genome shape and organization. Therefore, it is of great interest to understand how the whole linear genome structure behaves upon formation and repair of DNA damage. For this, we used long reads sequencing technology to identify and to characterize genomic structural variations (SV) of wild-type Arabidopsis thaliana somatic cells exposed either to UV-B, to UV-C or to protons irradiations. We found that genomic regions located in heterochromatin a more prone to form SVs than those located in euchromatin, highlighting that genome stability and flexibility differs along the chromosome. This holds true in Arabidopsis plants deficient for the expression of master regulators of the DNA Damage Response (DDR), ATM and ATR, suggesting that independent and alternative surveillance processes exist to maintain integrity in genic regions. Finally, the analysis of the radiations-induced deleted regions allowed determining that exposure UV-B, UV-C and protons induced the Microhomology-mediated end joining mechanism (MMEJ) and that both ATM and ATR repress this repair pathway.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2a78aa5b539dcd98a3297c9111dc41b5d43d56b" target='_blank'>
              Characterization of radiations-induced genomic structural variations in Arabidopsis thaliana
              </a>
            </td>
          <td>
            Salimata Ousmane Sall, Abdelmalek Alioua, Sébastien Staerck, Stéfanie Graindorge, Michel Pellicioli, Jacky Schuler, Quentin Raffy, Marc Rousseau, J. Molinier
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The pan-neuronally expressed and phylogenetically conserved CUT homeobox gene ceh-44/CUX orchestrates pan-neuronal gene expression throughout the nervous system of C. elegans. As in many other species, including humans, ceh-44/CUX is encoded by a complex locus that also codes for a Golgi-localized protein. How gene expression from this complex locus is controlled and, in C. elegans, directed to all cells of the nervous system has not been investigated. We show here that pan-neuronal expression of CEH-44/CUX is controlled by a pan-neuronal RNA splicing factor, UNC-75/CELF, the C. elegans homolog of vertebrate CELF proteins. UNC-75/CELF temporally and spatially specifies the production of an alternative, CEH-44/CUX homeobox gene-encoding transcript from a ubiquitously expressed gene locus, which also produces a Golgi apparatus-localized golgin protein, CONE-1 (“CASP of nematodes”). During embryogenesis the cone-1/ceh-44 locus exclusively produces the Golgi-localized CONE-1/CASP protein in all tissues, but upon the onset of postmitotic terminal differentiation of neurons, UNC-75/CELF induces the production of the alternative CEH-44/CUX CUT homeobox gene-encoding transcript, exclusively in the nervous system. Hence, UNC-75/CELF-mediated alternative splicing not only directs pan-neuronal gene expression, but also excludes a phylogenetically deeply conserved golgin from the nervous system, paralleling surprising spatial specificities of another golgin that we describe here as well. In summary, our findings provide novel insights into how all cells in a nervous system acquire pan-neuronal identity features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c64a73d60c20b97bb36dac4de2e26a225a1dd65" target='_blank'>
              Alternative splicing controls pan-neuronal homeobox gene expression
              </a>
            </td>
          <td>
            Eduardo Leyva-Díaz, Michael Cesar, Karinna Pe, Jose Ignacio Jordá-Llorens, Jessica Valdivia, Oliver Hobert
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e6bae3255c905d58e9f2d6fd8dca01970a1953f" target='_blank'>
              Chromatin conformation and histone modification profiling across human kidney anatomic regions
              </a>
            </td>
          <td>
            Haikuo Li, Dian Li, Benjamin D. Humphreys
          </td>
          <td>2024-07-18</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The endonuclease activity of Pms1 directs mismatch repair by generating a nick in the newly replicated DNA strand. Inactivating Pms2, the human homologue of yeast Pms1, increases the chances of colorectal and uterine cancers. Here we use whole genome sequencing to show that loss of this endonuclease activity, via the pms1-DE variant, results in strong mutator effects throughout the Saccharomyces cerevisiae genome. Mutation rates are strongly increased for mutations resulting from all types of single-base substitutions and for a wide variety of single- and multi-base indel mutations. Rates for these events are further increased in strains combining pms1-DE with mutator variants of each of the three major leading and lagging strand replicases. In all cases, mutation rates, spectra, biases, and context preferences are statistically indistinguishable from strains with equivalent polymerases but lacking initial mismatch recognition due to deletion of MSH2. This implies that, across the nuclear genome, strand discrimination via the Pms1 endonuclease is as important for MMR as is initial mismatch recognition by Msh2 heterodimers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c81863bf8b4bcd0533ef4d09eddc5a93c0da4b" target='_blank'>
              Instability throughout the Saccharomyces cerevisiae genome resulting from Pms1 endonuclease deficiency.
              </a>
            </td>
          <td>
            S. Lujan, Marta A. Garbacz, S. E. Liberti, Adam B. Burkholder, Thomas A. Kunkel
          </td>
          <td>2024-07-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Deficiencies in DNA mismatch repair (MMRd) leave characteristic footprints of microsatellite instability (MSI) in cancer genomes. We used data from the Cancer Genome Atlas and International Cancer Genome Consortium to conduct a comprehensive analysis of MSI-associated cancers, focusing on indel mutational signatures. We classified MSI-high genomes into two subtypes based on their indel profiles: deletion-dominant (MMRd-del) and insertion-dominant (MMRd-ins). Compared with MMRd-del genomes, MMRd-ins genomes exhibit distinct mutational and transcriptomic features, including a higher prevalence of T>C substitutions and related mutation signatures. Short insertions and deletions in MMRd-ins and MMRd-del genomes target different sets of genes, resulting in distinct indel profiles between the two subtypes. In addition, indels in the MMRd-ins genomes are enriched with subclonal alterations that provide clues about a distinct evolutionary relationship between the MMRd-ins and MMRd-del genomes. Notably, the transcriptome analysis indicated that MMRd-ins cancers upregulate immune-related genes, show a high level of immune cell infiltration, and display an elevated neoantigen burden. The genomic and transcriptomic distinctions between the two types of MMRd genomes highlight the heterogeneity of genetic mechanisms and resulting genomic footprints and transcriptomic changes in cancers, which has potential clinical implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b737e04b95df7e27390442dd66d22a5ba373a014" target='_blank'>
              A Subset of Microsatellite Unstable Cancer Genomes Prone to Short Insertions over Deletions Is Associated with Elevated Anticancer Immunity
              </a>
            </td>
          <td>
            Sunmin Kim, Dong-Jin Han, Seo-Young Lee, Youngbeen Moon, Su Jung Kang, Tae-Min Kim
          </td>
          <td>2024-06-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Aneuploidy is nearly ubiquitous in tumor genomes, but the role of aneuploidy in the early stages of cancer evolution remains unclear. Here, by inducing heterogeneous aneuploidy in non-transformed human colon organoids (colonoids), we investigated how the effects of aneuploidy on cell growth and differentiation may promote malignant transformation. Previous work implicated p53 activation as a downstream response to aneuploidy induction. We found that simple aneuploidy, characterized by 1-3 gained or lost chromosomes, resulted in little or modest p53 activation and cell cycle arrest when compared with more complex aneuploid cells. Single-cell RNA sequencing analysis revealed that the degree of p53 activation was strongly correlated with karyotype complexity. Single-cell tracking showed that cells could continue to divide despite the observation of one to a few lagging chromosomes. Unexpectedly, colonoids with simple aneuploidy exhibited impaired differentiation after niche factor withdrawal. These findings demonstrate that simple aneuploid cells can escape p53 surveillance and may contribute to niche factor-independent growth of cancer-initiating colon stem cells. [Media: see text].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69141397c86397dd6c1f0121be2fa0ac82f1c38" target='_blank'>
              Simple aneuploidy evades p53 surveillance and promotes niche factor-independent growth in human intestinal organoids.
              </a>
            </td>
          <td>
            Blake A. Johnson, Albert Z Liu, Tianhao Bi, Yi Dong, Taibo Li, Dingjingyu Zhou, A. Narkar, Yufei Wu, Sean X. Sun, Tatianna C. Larman, Jin Zhu, Rong Li
          </td>
          <td>2024-07-10</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Telomeric DNA, composed of short, direct repeats, is of crucial importance for chromosome stability. Due to intrinsic problems with replicating this DNA, the repeat tracts shorten at each cell division. Once repeat tracts become critically short, a telomeric stress signal induces cellular senescence and division arrest, which eventually may lead to devastating age-related degenerative diseases associated with dysfunctional telomers. Conversely, maintenance of telomere length by telomerase upregulation is a hallmark of cancer. Therefore, telomere length is a critical determinant of telomere function. How telomere length is established and molecular mechanisms for telomere-specific length regulation remained unknown. Here we show that subtelomeric chromatin is a determinant for how telomere equilibrium set-length is established in cis. The results demonstrate that telomerase recruitment mediated by the telomere-associated Sir4 protein is modulated on chromosome 3L in a telomere-specific way. Increased Sir4 abundance on subtelomeric heterochromatin of this specific telomere leads to telomere lengthening of only that telomere in cis, but not at other telomeres. Therefore, this work describes a mechanism for a how telomere-specific repeat tract length can be established. Further, our results will force the evaluation of telomere length away from a generalized view to a more telomere-specific consideration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c2fb6915328e84462f488e5ccd4496ce592a49a" target='_blank'>
              A mechanism for telomere-specific telomere length regulation
              </a>
            </td>
          <td>
            Gabriela M. Teplitz, Emeline Pasquier, Erin Bonnell, E. D. De Laurentiis, Louise Bartle, Jean-François Lucier, Samantha L. Sholes, C. Greider, R. Wellinger
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0bff8174988ff460b1ed14cde5b076ac3673528" target='_blank'>
              Methods and applications of genome-wide profiling of DNA damage and rare mutations.
              </a>
            </td>
          <td>
            Gerd P Pfeifer, Seung-Gi Jin
          </td>
          <td>2024-06-25</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Maintenance of DNA integrity is essential to all forms of life. DNA damage generated by reaction with genotoxic chemicals results in deleterious mutations, genome instability, and cell death. Pathogenic bacteria encounter several genotoxic agents during infection. In keeping with this, the loss of DNA repair networks results in virulence attenuation in several bacterial species. Interstrand DNA crosslinks (ICLs) are a type of DNA lesion formed by covalent linkage of opposing DNA strands and are particularly toxic as they interfere with replication and transcription. Bacteria have evolved specialized DNA glycosylases that unhook ICLs, thereby initiating their repair. In this study, we describe AlkX, a DNA glycosylase encoded by the multidrug resistant pathogen Acinetobacter baumannii. AlkX exhibits ICL unhooking activity similar to that of its Escherichia coli homolog YcaQ. Interrogation of the in vivo role of AlkX revealed that its loss sensitizes cells to DNA crosslinking and impairs A. baumannii colonization of the lungs and dissemination to distal tissues during pneumonia. These results suggest that AlkX participates in A. baumannii pathogenesis and protects the bacterium from stress conditions encountered in vivo. Consistent with this, we found that acidic pH, an environment encountered during host colonization, results in A. baumannii DNA damage and that alkX is induced by, and contributes to, defense against acidic conditions. Collectively, these studies reveal functions for a recently described class of proteins encoded in a broad range of pathogenic bacterial species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e8859ed92eefb483b7d273e5025bd69777d468" target='_blank'>
              An interstrand DNA crosslink glycosylase aids Acinetobacter baumannii pathogenesis.
              </a>
            </td>
          <td>
            D. E. Kunkle, Yujuan Cai, B. Eichman, Eric P Skaar
          </td>
          <td>2024-06-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="A recent publication describing the assembly of the Y chromosomes of 43 males was remarkable not only for its ambitious technical scope, but also for the startling suggestion that the boundary of the pseudoautosomal region 1 (PAR1), where the human X and Y chromosomes engage in crossing-over during male meiosis, lies 500 kb distal to its previously reported location1. Where is the boundary of the human PAR1? We first review the evidence that mapped the PAR boundary, or PAB, before the human genome draft sequence was produced, then examine post-genomic datasets for evidence of crossing-over between the X and Y, and lastly re-examine the sequence assemblies presented by Hallast and colleagues to see whether they support a more distal PAB. We find ample evidence of X-Y crossovers throughout the 500 kb in question, some as close as 246 bp to the previously reported PAB. Our new analyses, combined with previous studies over the past 40 years, provide overwhelming evidence to support the original position, and narrow the probable location of the PAB to a 201-bp window.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d811c6ff23bfcae84522ef94ae21f045cd0bf5e" target='_blank'>
              Where is the boundary of the human pseudoautosomal region?
              </a>
            </td>
          <td>
            D. W. Bellott, Jennifer F. Hughes, H. Skaletsky, Erik C. Owen, David C. Page
          </td>
          <td>2024-06-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cytokinesis, the final stage of cell division, serves to physically separate daughter cells while ensuring correct segregation of cellular components. In cultured naïve mouse embryonic stem cells cytokinesis lasts unusually long but the underlying mechanisms are not well understood. Here, using cellular and in vitro approaches, we describe a novel function for the kinesin-13 member KIF2A in this process. In genome-engineered mouse embryonic stem cells we find that KIF2A mainly localises to spindle poles during metaphase and regulates spindle length in a manner consistent with its known role as microtubule minus-end depolymerase. By contrast, during cytokinesis we observe tight binding of KIF2A on the lattices of intercellular bridge microtubules. At this stage KIF2A maintains microtubule length and number, and controls microtubule acetylation. Based on in vitro experiments we propose that the conversion of KIF2A from a depolymerase to a stabiliser is driven both by the inhibition of its ATPase activity, which increases affinity for the lattice, and by a preference of KIF2A for compacted lattices. We propose that during cytokinesis KIF2A maintains the compacted microtubule state, thereby dampening acetylation. As KIF2A depletion causes pluripotency problems and affects mRNA homeostasis our results furthermore indicate that KIF2A-mediated microtubule stabilisation prolongs cytokinesis to maintain pluripotency.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d941e6b6bf9f55a67768f7d757783736ce934663" target='_blank'>
              KIF2A maintains cytokinesis in mouse embryonic stem cells by stabilising intercellular bridge microtubules
              </a>
            </td>
          <td>
            Lieke Stockmann, Hélène Kabbech, G. Kremers, Brent van Herk, Bas Dille, M. van den Hout, W. V. van Ijcken, Dick H W Dekkers, J. Demmers, Ihor Smal, Danny Huylebroeck, Sreya Basu, Niels Galjart
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) are a heterogeneous group, in terms of structure and sequence length, consisting of RNA molecules that do not code for proteins. These ncRNAs have a central role in the regulation of gene expression and are virtually involved in every process analyzed, ensuring cellular homeostasis. Although, over the years, much research has focused on the characterization of non-coding transcripts of nuclear origin, improved bioinformatic tools and next-generation sequencing (NGS) platforms have allowed the identification of hundreds of ncRNAs transcribed from the mitochondrial genome (mt-ncRNA), including long non-coding RNA (lncRNA), circular RNA (circRNA), and microRNA (miR). Mt-ncRNAs have been described in diverse cellular processes such as mitochondrial proteome homeostasis and retrograde signaling; however, the function of the majority of mt-ncRNAs remains unknown. This review focuses on a subgroup of human mt-ncRNAs whose dysfunction is associated with both failures in cell cycle regulation, leading to defects in cell growth, cell proliferation, and apoptosis, and the development of tumor hallmarks, such as cell migration and metastasis formation, thus contributing to carcinogenesis and tumor development. Here we provide an overview of the mt-ncRNAs/cancer relationship that could help the future development of new biomedical applications in the field of oncology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1460fe5d6ce5cff4844c986e13e915b3c8688352" target='_blank'>
              Non-Coding RNAs of Mitochondrial Origin: Roles in Cell Division and Implications in Cancer
              </a>
            </td>
          <td>
            R. Piergentili, S. Sechi
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3850a10d0e02da63310b269aa1b7eb41498587bc" target='_blank'>
              The first complete mitochondrial genome of Grossulariaceae: Molecular features, structure recombination, and genetic evolution
              </a>
            </td>
          <td>
            Guilong Lu, Wenhua Wang, Shanshan Zhang, Guang Yang, Kun Zhang, Youxiong Que, Lan Deng
          </td>
          <td>2024-07-30</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b4719ac7d63f56b0c7cbebda1b463c473c0ad01" target='_blank'>
              Mutations found in cancer patients compromise DNA binding of the winged helix protein STK19
              </a>
            </td>
          <td>
            Jian Li, Xinli Ma, Xiaoyu Wang, Xiaotong Hu, Shaobo Fang, Guoguo Jin, Kangdong Liu, Zigang Dong
          </td>
          <td>2024-06-18</td>
          <td>Scientific Reports</td>
          <td>1</td>
          <td>29</td>
        </tr>

        <tr id="Biomolecular condensates formed via liquid-liquid phase separation are ubiquitous in cells, especially in the nucleus. While condensates containing one or two kinds of biomolecules have been relatively well characterized, those with more heterogenous biomolecular components and interactions between biomolecules inside are largely unknown. This study used residue-resolution molecular dynamics simulations to investigate heterogeneous protein assemblies that include four master transcription factors in mammalian embryonic stem cells: Oct4, Sox2, Klf4, and Nanog. Simulations of the mixture systems showed highly heterogeneous and dynamic behaviors; the condensates mainly contained Sox2, Klf4, and Nanog, while Oct4 was dissolved into the dilute phase. Condensates consisted of loosely interacting clusters in which Klf4 was the most abundant in the cores. We suggest that Klf4 serves as a scaffold of the condensate where Sox2 and Nanog are bound to stabilize the condensate, whereas Oct4 is moderately recruited to the condensate, serving as a client mainly via its interaction with Sox2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf2b6f3c3ee8c5fafe1727b0a8af00c72622a507" target='_blank'>
              Heterogenous organization in condensates of multiple transcription factors in embryonic stem cells
              </a>
            </td>
          <td>
            Azuki Mizutani, Cheng Tan, Yuji Sugita, Shoji Takada
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1575162f33cdbe2adcebbaac6a20b693ec8530" target='_blank'>
              Single-cell EpiChem jointly measures drug-chromatin binding and multimodal epigenome.
              </a>
            </td>
          <td>
            Chao Dong, Xiaoxuan Meng, Tong Zhang, Zhifang Guo, Yaxi Liu, Peihuang Wu, Shiwei Chen, Fanqi Zhou, Yanni Ma, Haiqing Xiong, Shaokun Shu, Aibin He
          </td>
          <td>2024-07-18</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Helicobacter pylori, a leading human pathogen associated with duodenal ulcer and gastric cancer, presents a significant threat to human health due to increasing antibiotic resistance rates. This study investigates G-quadruplexes (G4s), which are non-canonical secondary structures form in G-rich regions within the H. pylori genome. Extensive research on G4s in eukaryotes has revealed their role in epigenetically regulating cellular processes like gene transcription, DNA replication, and oncogene expression. However, understanding of G4-mediated gene regulation in other organisms, especially bacterial pathogens, remains limited. Although G4 motifs have been extensively studied in a few bacterial species such as Mycobacterium, Streptococci, and Helicobacter, research on G4 motifs in other bacterial species is still sparse. Like in other organisms such as archaea, mammals, and viruses, G4s in H. pylori display a non-random distribution primarily situated within open reading frames of various protein-coding genes. The occurrence of G4s in functional regions of the genome and their conservation across different species indicates that their placement is not random, suggesting an evolutionary pressure to maintain these sequences at specific genomic sites. Moreover, G-quadruplexes show enrichment in specific gene classes, suggesting their potential involvement in regulating the expression of genes related to cell wall/membrane/envelope biogenesis, amino acid transport, and metabolism. This indicates a probable regulatory role for G4s in controlling the expression of genes essential for H. pylori survival and virulence. Biophysical techniques such as Circular Dichroism spectroscopy and Nuclear Magnetic Resonance were used to characterize G4 motifs within selected H. pylori genes. The study revealed that G-quadruplex ligand inhibited the growth of H. pylori, with minimal inhibitory concentrations in the low micromolar range. This suggests that targeting G4 structures could offer a promising approach for developing novel anti-H. pylori drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fa6a4ec1b161b97113f3d0f1d450cfd7c868aec" target='_blank'>
              Characterization of G-quadruplexes in the Helicobacter pylori genome and assessment of therapeutic potential of G4 ligands.
              </a>
            </td>
          <td>
            Monika Kumari, Saumya Jaiswal, Uma Shankar, Sharad Gupta, P. Pradeepkumar, Amit Kumar, Debasis Nayak, Vikas Yadav, Puja Yadav
          </td>
          <td>2024-07-23</td>
          <td>Biotechnology and applied biochemistry</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/798b264fbd39508ed9588f5472e886b8b6303884" target='_blank'>
              Navigating the complexity of p53-DNA binding: implications for cancer therapy
              </a>
            </td>
          <td>
            Kelly M. Thayer, Sean Stetson, Fernando Caballero, Christopher Chiu, I. Han
          </td>
          <td>2024-07-11</td>
          <td>Biophysical Reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="TERRA, the lncRNA derived from the ends of chromosomes, has a number of well-described nuclear roles including telomere maintenance and homeostasis. A growing body of evidence now points at its role in human cells outside of nucleus—it has been found to be a component of extracellular vesicles, a player in inflammation signalling and its capacity for translation has been shown. In this work, using a combination of sensitive microscopy methods, cellular fractionation, proteomics and transcriptome analysis, we demonstrate directly for the first time that TERRA is present in the cytoplasm of human telomerase-negative cells, especially upon various stress stimuli, and that it associates with stress granules. Confirming the presence of TERRA in the cytoplasm, our work fills an important gap in the field, and contributes to the discussion about the role of TERRA as a transcript involved in nucleo-cytoplasmic stress communication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb39bcfed08f216165348412fc98ef33cf57f91a" target='_blank'>
              Telomeric lncRNA TERRA localizes to stress granules in human ALT cells
              </a>
            </td>
          <td>
            Luca Larini, Elena Goretti, Eleonora Zulian, Emma Busarello, Stefano Maria Marino, Mona Hajikazemi, K. Paeschke, T. Tebaldi, E. Cusanelli, K. Juríková
          </td>
          <td>2024-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="
 Paralogs are prevalent in the human genome and are considered a rich source of synthetic lethality due to functional redundancy. For a cancer cell carrying a gene with Loss-Of-Function (LOF) mutation, inactivation of its paralog using gene editing can induce a selective decrease in viability, leaving normal cells that do not harbor the mutation unharmed. Previous studies have exploited this vulnerability of cancer genomes. However, the cancer-specificity of such synthetic lethality limits its application across different cancer types. Furthermore, the role of functional redundancy which can explain the cancer specificity has remained largely unexplored. In this study, we computationally predicted synthetic lethal paralogs along with mechanistically important functional redundancies between them in a cancer-specific manner. We applied our prediction method to publicly available data for an aggressive subtype of breast cancer called triple-negative breast cancer (TNBC), which lacks the expression key biomarkers and hence has the worst prognosis among other breast cancer subtypes. TNBC is characterized by the highest mutational load and the largest fraction of genome altered among the breast cancer subtypes providing a rich mutational landscape to identify LOF genes. Using the CRISPR inactivation screen data for cancer cell lines obtained from the Cancer Dependency Map (DepMap) project, we predicted sets of synthetic lethal paralogs that show a significantly greater viability decrease if a gene carries LOF and its paralog is inactivated, compared to the viability decrease due to the inactivation of only one of the paralogs. Consistent with previous findings of context-dependent synthetic lethality, we found that a relatively small fraction of TNBC-specific synthetic lethal paralogs overlapped with those found across other cancer types. To uncover the mechanistically important functional redundancies between paralogs, we analyzed the genomics and transcriptomics data from multiple sources: TNBC panel of primary tumors and patient-derived xenografts, Pan-Cancer Analysis of Whole Genomes (PCAWG), and the Cancer Cell Line Encyclopedia (CCLE). The functional redundancies varied across cancer types based on (1) mutual exclusivity of LOFs, (2) backup compensation of deleterious mutations, (3) backup upregulation, and (4) dosage balance. Overall, our findings show a strong context-dependency of synthetic lethal paralogs and estimates of functional redundancy, emphasizing the importance of such cancer-specific predictions in identifying targetable paralog synthetic lethalities. Collectively, the computational method and sets of targetable paralogs are a unique resource for developing precision oncology therapeutic strategies against TNBC and other cancers more broadly.
 Citation Format: Rohan Dandage, Elena Kuzmin. Predicting targetable paralog synthetic lethalities and functional redundancies in cancer genomes [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B012.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d08601862d27b41b39a782547d739914c1bd271" target='_blank'>
              Abstract B012: Predicting targetable paralog synthetic lethalities and functional redundancies in cancer genomes
              </a>
            </td>
          <td>
            Rohan Dandage, Elena Kuzmin
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87375be953b6b1e62c0930938d4b8c30692440ac" target='_blank'>
              Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants.
              </a>
            </td>
          <td>
            Max Frenkel, James E. Corban, Margaux L. A. Hujoel, Zachary Morris, Srivatsan Raman
          </td>
          <td>2024-07-24</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198f78f5b5d06b28964e8fe5554ec81d0dc67f78" target='_blank'>
              Pan-cancer exploration of oncogenic and clinical impacts revealed that HOXA9 is a diagnostic indicator of tumorigenesis
              </a>
            </td>
          <td>
            U. S. Shenoy, Dhanraj Salur Basavarajappa, S. Kabekkodu, R. Radhakrishnan
          </td>
          <td>2024-06-21</td>
          <td>Clinical and Experimental Medicine</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract The conserved Gsx homeodomain (HD) transcription factors specify neural cell fates in animals from flies to mammals. Like many HD proteins, Gsx factors bind A/T-rich DNA sequences prompting the following question: How do HD factors that bind similar DNA sequences in vitro regulate specific target genes in vivo? Prior studies revealed that Gsx factors bind DNA both as a monomer on individual A/T-rich sites and as a cooperative homodimer to two sites spaced precisely 7 bp apart. However, the mechanistic basis for Gsx–DNA binding and cooperativity is poorly understood. Here, we used biochemical, biophysical, structural and modeling approaches to (i) show that Gsx factors are monomers in solution and require DNA for cooperative complex formation, (ii) define the affinity and thermodynamic binding parameters of Gsx2/DNA interactions, (iii) solve a high-resolution monomer/DNA structure that reveals that Gsx2 induces a 20° bend in DNA, (iv) identify a Gsx2 protein–protein interface required for cooperative DNA binding and (v) determine that flexible spacer DNA sequences enhance Gsx2 cooperativity on dimer sites. Altogether, our results provide a mechanistic basis for understanding the protein and DNA structural determinants that underlie cooperative DNA binding by Gsx factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/452ef310af2a617cf1e269eedc0395b04211add2" target='_blank'>
              Cooperative Gsx2–DNA binding requires DNA bending and a novel Gsx2 homeodomain interface
              </a>
            </td>
          <td>
            Jordan A. Webb, Edward Farrow, Brittany Cain, Zhenyu Yuan, Alexander E. Yarawsky, Emma Schoch, E. Gagliani, Andrew B. Herr, B. Gebelein, R. Kovall
          </td>
          <td>2024-06-14</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="RNA serves as information media as well as molecular scaffold in nature and synthetic systems. The single guide RNA (sgRNA) widely applied in CRISPR techniques exemplifies both functions, with a guide region bearing DNA base-pairing information, and a structural motif for Cas9 protein scaffolding. The scaffold region has been modified by fusing RNA aptamers to the tetra-stem loop. The guide region is typically not regarded as a pluggable module as it encodes the essential function of DNA sequence recognition. Here, we investigate a chimera of two sgRNAs, with distinct guide sequences joined by an RNA linker (dgRNA), regarding its DNA binding function and loop induction capability. First, we studied the sequence bi-specificity of the dgRNA and discovered that the RNA linker allows distal parts of double-stranded DNA to be brought into proximity. To test the activity of the dgRNA in organisms, we used the LacZ gene as a reporter and recapitulated the loop-mediated gene inhibition by LacI in E. coli. We found that the dgRNA can be applied to target distal genomic regions with comparable levels of inhibition. The capability of dgRNA to induce DNA contacts solely requires dCas9 and RNA, making it a minimal system to remodel chromosomal conformation in various organisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523e2eaf342b8bd03ac97c7a4fec3befd8535475" target='_blank'>
              Re-engineered guide RNA enables DNA loops and contacts modulating repression in E. coli.
              </a>
            </td>
          <td>
            Yunshi Yang, Iris Rocamonde-Lago, B. Shen, I. Berzina, Johanna Zipf, Björn Högberg
          </td>
          <td>2024-07-16</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Genetically identical cells can respond heterogeneously to cancer therapy, with a subpopulation of cells often entering a temporarily arrested treatment-tolerant state before repopulating the tumor. To investigate how heterogeneity in the cell cycle arrest protein p21 arises, we imaged the dynamics of p21 transcription and protein expression along with those of p53, its transcriptional regulator, in single cells using live cell fluorescence microscopy. Surprisingly, we found that the rate of p21 transcription depends on the change in p53 rather than its absolute level. Through combined theoretical and experimental modeling, we determined that p21 transcription is governed by an incoherent feedforward loop mediated by MDM2. This network architecture facilitates rapid induction of p21 expression and variability in p21 transcription. Abrogating the feedforward loop overcomes rapid S-phase p21 degradation, with cells transitioning into a quiescent state that transcriptionally resembles a treatment-tolerant persister state. Our findings have important implications for therapeutic strategies based on activating p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/92b16e761f3e0e9fd2df2ece960b5d2bfc57dcb6" target='_blank'>
              Functional consequences of a p53-MDM2-p21 incoherent feedforward loop
              </a>
            </td>
          <td>
            Jamie A. Dean, José Reyes, Michael Tsabar, A. Jambhekar, G. Lahav, Franziska Michor
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Non-coding RNAs (ncRNAs) in human cells do not lead to protein synthesis and constitute a substantial portion of the transcriptome. Human long non-coding RNAs (lncRNAs) orchestrate critical cellular functions influencing development, differentiation, and metabolism. Dysregulation of lncRNAs has been correlated with several pathological conditions such as neurodegenerative and autoimmune disorders, diabetes, and cancer. Recent reports have suggested the involvement of G4s in lncRNAs to regulate colorectal cancer (CRC) carcinogenesis. In this study, we investigate the occurrence and distribution of G4s in the LINC01589, MELTF-AS1, and UXT-AS1 lncRNAs, which have been reported to be dysregulated in CRC. Using a combination of in silico tools and in vitro biophysical techniques, we show that these lncRNAs form stable, parallel, and intramolecular G4s. Furthermore, we establish the formation of G4s within these lncRNAs in CRC using cell-based assays, including RNA G4-Immuno-FISH and G4RP-RT-qPCR. This is the first systematic study of G4s in lncRNAs dysregulated in CRC, and our findings highlight the diagnostic and therapeutic potential of G4s in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24feffea3287f492bc1ccbc98b5a40b322f119ab" target='_blank'>
              G-quadruplex formation in long non-coding RNAs dysregulated in colorectal cancer
              </a>
            </td>
          <td>
            Shubham Sharma, Chinmayee Shukla, Jérémie Mitteaux, A. Pipier, Marc Pirrotta, Marie-José Penouilh, D. Monchaud, Bhaskar Datta
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Tumor protein p53 (TP53) mediates DNA repair and cell proliferation in growing cells. The TP53 gene is a tumor suppressor that regulates the expression of target genes in response to multiple cellular stress factors. Key target genes are involved in crucial cellular events such as DNA repair, cell cycle regulation, apoptosis, metabolism, and senescence. TP53 genetic variants and the activity of the wild-type p53 protein (WT-p53) have been linked to a wide range of tumorigenesis. Various genetic and epigenetic alterations, including germline and somatic mutations, loss of heterozygosity, and DNA methylation, can alter TP53 activity, potentially resulting in cancer initiation and progression. This study was designed to screen three reported mutations in the DNA-binding domain of the p53 protein in breast cancer, to evaluate the relative susceptibility and risk associated with breast cancer in the local population. Genomic DNA was isolated from 30 breast tumor tissues along with controls. Tetra and Tri ARMS PCR were performed to detect mutations in the TP53 coding region. For SNPs c.637C>T and c.733C>T, all analyzed cases were homozygous for the wild-type allele ‘C,’ while for SNP c.745A>G, all cases were homozygous for the wild-type allele ‘A.’ These results indicate no relevance of these three SNPs to cancer progression in our study cohort. Additionally, the findings from whole exon sequencing will help to predict more precise outcomes and assess the importance of TP53 gene mutations in breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bffe267deb645fc61554e684edc8c0bafc2c11ee" target='_blank'>
              Analysis of Genetic Alterations in TP53 Gene in Breast Cancer – A Secondary Publication
              </a>
            </td>
          <td>
            Baqaur Rehman, Muhammad Abubakar, Muhammad Naeem Kiani, Rooma Ayyoub
          </td>
          <td>2024-06-19</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epigenetics are the changes due to DNA methylation, histone modifications, and noncoding RNA regulation, which play a crucial part in the expression of genes. Although these changes do not alter the sequence of the DNA, they are significant in determining the phenotype and inheritance of the genes. The impact of epigenetics on inherited diseases like Fragile X syndrome, Angelman syndrome, and Prader-Willi syndrome emphasizes the significance of studying epigenetic dysregulation in disease pathology. The current review aims to investigate the role of epigenetic mechanisms in inherited diseases and to evaluate the utility of molecular diagnostic tools for epigenetic analysis in disease diagnosis. Case studies of diseases such as Duchenne muscular dystrophy and Prader-Willi syndrome highlight the clinical relevance of epigenetic analysis in disease diagnosis and management. This research is conducted through an extensive analysis of recent literature to explore the epigenetic mechanisms underlying inherited diseases. Additionally, advanced molecular diagnostic techniques like droplet digital PCR (ddPCR), transposase-based bisulfite tagging, and data mining coupled with conventional experimental procedures. However, the intricate relationship between genetic and epigenetic factors and challenges related to inclined data need to be addressed. Future research endeavors to elucidate the role of epigenetic modifications in disease pathogenesis and develop strategies for personalized medicinal treatment and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f24f0960efd210e8ee51cbad54070948fecc2ef" target='_blank'>
              A Review on Epigenetics of Human Inherited Diseases: Molecular Diagnosis
              </a>
            </td>
          <td>
            Bibi Shazia Sabir
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Innovative Science and Research Technology (IJISRT)</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="In light of the post-genomic era, epigenetics brings about an opportunity to better understand how the molecular machinery works and is led by a complex dynamic set of mechanisms, often intricate and complementary in many aspects. In particular, epigenetics links developmental biology and genetics, as well as many other areas of knowledge. The present work highlights substantial scopes and relevant discoveries related to the development of the term from its first notions. To our understanding, the concept of epigenetics needs to be revisited, as it is one of the most relevant and multifaceted terms in human knowledge. To redirect future novel experimental or theoretical efforts, it is crucial to compile all significant issues that could impact human and ecological benefit in the most precise and accurate manner. In this paper, the reader can find one of the widest compilations of the landmarks and epistemic considerations of the knowledge of epigenetics across the history of biology from the earliest epigenetic formulation to genetic determinism until the present. In the present work, we link the current body of knowledge and earlier pre-genomic concepts in order to propose a new definition of epigenetics that is faithful to its regulatory nature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63d95143e930f9759529d2eb02d359dc5d641be6" target='_blank'>
              Revisiting Epigenetics Fundamentals and Its Biomedical Implications
              </a>
            </td>
          <td>
            T. Meza-Menchaca, Arnulfo Albores-Medina, Alma Heredia-Mendez, E. Ruíz-May, Jorge Ricaño-Rodríguez, Veronica Gallegos-García, Adriana Esquivel, Giancarlo Vettoretti-Maldonado, Alma Campos-Parra
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Endothelial cells (ECs) maintain vessel tone and barrier integrity, regulate blood homeostasis, and prevent the extravasation of leukocytes under normal physiological conditions. Because of the limited lifespans and batch-to-batch differences with respect to the genetic make-up of primary ECs, established immortal EC lines are extensively used for studying endothelial biology. To address this issue, the immortal endothelial cell line EA.hy926 was developed by fusing primary human umbilical vein endothelial cells (HUVECs) with human lung carcinoma A549 cells. EA.hy926 cells share a number of similar endothelial properties with HUVECs and are considered the immortal counterpart to primary HUVECs. However, the cytogenetic integrity of EA.hy926 cells is not fully elucidated. We characterized EA.hy926 cells with conventional G-banding and molecular cytogenetic techniques such as spectral karyotyping and subtelomeric fluorescence in situ hybridization. Cytogenetic analysis revealed an array of numerical and stable structural chromosomal rearrangements including one deletion, one duplication, one isochromosome, seven simple translocations, and five complex translocations in Ea.hy926 cells. These findings will advance comprehension of EA.hy926 cell biology and augment future endothelial studies, specifically in comparison studies between HUVECs and EA.hy926 cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ad1b12e69b1e161748ea2c28f86f896e19f2f6" target='_blank'>
              G-Banding and Molecular Cytogenetics Detect Novel Translocations and Cryptic Aberrations in Human Immortal Endothelial Cells
              </a>
            </td>
          <td>
            Regina Lichti Binz, R. Pathak
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The deoxyribonucleic acid (DNA) carries hereditary codes which is translated by the cells to synthesize the ribonucleic acid (RNA) and polypeptides which can generate and perform vital function. The double helix structure is the most studied model of the DNA that was proposed by Watson and Crick. The capability of DNA to work as a genetic material can be stored and conducted during cell division to permit this information to be doubled and transmitted to the incoming generation. Any damage in the structure of DNA is an essential direct cause for the progression of cancer and other disorders. The factors for DNA damage can be classified as exogenous and endogenous factors. In this review article, we highlight the evidence-supported information about the structure, functions and clinical significance of DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23ff4de2b8e4130d71de29efe87be0afb81a98a9" target='_blank'>
              Structure, Functions And Clinical Significance Of DNA: A Review Article
              </a>
            </td>
          <td>
            Ahmed M. Amshawee, Maryam A. Hussain, Seenaa Taqi Mansour Al Muhtaser, Ali A. Al- Faham
          </td>
          <td>2024-07-22</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 c-MYC (MYC) is a central regulatory protein that is dysregulated in >50% of all human cancers and is linked to aggressive disease. Developing MYC inhibitors would revolutionize cancer treatment; however, developing small molecules that directly target MYC is challenging. An alternative approach is to identify and inhibit critical MYC partner proteins to inactivate MYC and trigger cancer cell death. Inhibiting these targets therapeutically can result in synthetic lethality (MYC-SL), which can be exploited in MYC-dysregulated cancers. To identify MYC-SL targets, we performed a genome-wide CRISPR knock-out screen using an isogenic pair of normal and MYC-driven breast cancer cells. In contrast to other screens, this model is dependent on MYC and recapitulates human disease at both pathological and molecular levels in vivo. We identified high-priority hits to validate from the screen using two independent approaches: 1) a traditional gene-set enrichment analysis to identify highly represented biological pathways; and 2) analyzing the Cancer Dependency Map (DEPMAP) to select hits that are likely to be robust beyond the context of our screening conditions. Using a traditional gene-set enrichment analysis approach, we identified topoisomerase 1 (TOP1) as an actionable vulnerability that can be targeted with clinically approved inhibitors. Genetic and pharmacological inhibition of TOP1 resulted in MYC-driven cell death compared to that in control cells. Finally, drug response to TOP1 inhibitors significantly correlated with MYC levels and activity across panels of breast cancer cell lines and patient-derived organoids, highlighting TOP1 as a promising target for MYC-driven cancers. As a secondary approach to interpreting our CRISPR screen hits, we analyzed DEPMAP to identify MYC-SLs that are differentially essential in MYC-dependent cancer cells. Specifically, data from RNA interference screens in hundreds of cancer cell lines were used to describe the response of these cells to MYC knockdown. These data were used to define MYC-dependent and MYC-independent cell lines within the context of this analysis. These two groups were then investigated for their in silico response to the knockdown of each of our MYC-SL hits. MYC-SLs, which were also differentially essential in MYC-dependent cancer cells from DEPMAP, were prioritized for further investigation. Critical MYC cofactors that have been validated by us and others (e.g., CDK9) were identified, providing confidence in this approach, and rationalizing ongoing investigations. Together, this work features two successful strategies to prioritize hits from hundreds of synthetic-lethal genome-wide CRISPR screens to identify novel MYC-driven vulnerabilities in cancer.
 Citation Format: Peter Lin, Linda Penn. An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B004.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3fe21fa230f85037dcec9224411cd263d57af781" target='_blank'>
              Abstract B004: An isogenic CRISPR screen identifies novel MYC-driven vulnerabilities
              </a>
            </td>
          <td>
            Peter Lin, Linda Penn
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Aneuploidy is an abnormal chromosome composition and a general hallmark of human cancer. Aneuploidy causes detrimental cellular stresses, but cancer cells evolve to cope with these stresses. Consequently, targeting such mitigation mechanisms is a promising potential therapeutic strategy. As an abnormal dosage of gene products from altered chromosomes can cause RNA and proteotoxic stress, dosage compensation (DC) of imbalanced gene products was reported to mitigate these stresses in aneuploid cells. However, the mechanisms that regulate DC remain elusive. To address these mechanisms, we focused on the role(s) of stress granules (SGs) and RNA binding proteins (RBPs) in aneuploid cancer cells. Our recent study revealed that aneuploid cancer cells preferentially depend on RNA and protein metabolism, and need to attenuate translation in order to cope with proteotoxic stress (Ippolito & Zerbib et al. bioRxiv 2023). In yeast, it has been previously reported that Ssd1, one of the RBPs localized in P-bodies, is indispensable to tolerating aneuploidy stress. Based on these findings, we set out to explore SGs as potential regulators of gene expression in aneuploid cancer cells. As expected, we revealed that SGs emerged in cells under acute aneuploidy stress induced by chromosome mis-segregation. These results imply that SGs are formed in cells during and right after aneuploidization, potentially as a way to mediate DC and ameliorate aneuploidy-induced cellular stress. To gain further insight into RNA regulation in aneuploid cells, we analyzed the essentiality of RBPs across cancer cell lines. We found that the RNA-editing enzyme ADAR, a suppressor of SG formation, is preferentially essential in aneuploid cancer cells. Remarkably, ADAR disruption synergized with acute aneuploidy stress to induce SGs, and ADAR exhibited increased RNA editing activity following chromosome mis-segregation. Our results suggest that SGs and their suppressor ADAR might contribute to ameliorating acute aneuploidy-induced stress. We are currently investigating the target RNAs regulated by SGs and ADAR and examining their functional role(s) in DC, with the goal of uncovering potential synthetic lethalities associated with aneuploidy in cancer cells.
 Citation Format: Hajime Okada, Eran Sdeor, Miriam Karmon, Erez Levanon, Uri Ben-David. Targeting mechanisms of dosage compensation to selectively kill aneuploid cancer cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR008.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d65eea6dda42eca3bae9912c1560d589405d46e4" target='_blank'>
              Abstract PR008: Targeting mechanisms of dosage compensation to selectively kill aneuploid cancer cells
              </a>
            </td>
          <td>
            Hajime Okada, Eran Sdeor, Miriam Karmon, E. Levanon, Uri Ben-David
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Histone chaperones are integral to chromatin dynamics, facilitating the assembly and disassembly of nucleosomes, thereby playing a crucial role in regulating gene expression and maintaining genomic stability. Moreover, they prevent aberrant histone interactions prior to chromatin assembly. Disruption in histone chaperone function may result in genomic instability, which is implicated in pathogenesis. This review aims to elucidate the role of histone chaperones in cancer pathologies and explore their potential as therapeutic targets. Histone chaperones have been found to be dysregulated in various cancers, with alterations in expression levels, mutations, or aberrant interactions leading to tumorigenesis and cancer progression. In addition, this review intends to highlight the molecular mechanisms of interactions between histone chaperones and oncogenic factors, underscoring their roles in cancer cell survival and proliferation. The dysregulation of histone chaperones is significantly correlated with cancer development, establishing them as active contributors to cancer pathology and viable targets for therapeutic intervention. This review advocates for continued research into histone chaperone-targeted therapies, which hold potential for precision medicine in oncology. Future advancements in understanding chaperone functions and interactions are anticipated to lead to novel cancer treatments, enhancing patient care and outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08dc7b47acaf7ccec187d30ec604ff5ce9a5f189" target='_blank'>
              Comparative Review on Cancer Pathology from Aberrant Histone Chaperone Activity
              </a>
            </td>
          <td>
            Jiho Lee, Xiucong Bao
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The genetic integrity of pluripotent stem cells (PSC) is integral to their applications in research and therapy, but it is compromised by frequent development of copy number variations. Little is known about the basis of the variable genomic integrity among different PSC lines. Here we identify aneuploidies using RNA-seq and proteomics data from a panel of mouse embryonic stem cell (mESC) lines derived from 170 Diversity Outbred mice. We identified 62 lines with detectable aneuploid subpopulations and a subset of originally XX lines that lost one Chromosome X (XO). Strikingly, a much lower proportion of XX lines were aneuploid, compared to XY or XO lines. Two single-cell RNA-seq data sets demonstrated that aneuploid XY DO mESC also show lower Chromosome X gene expression and that XY mESC accumulate higher aneuploid proportions in culture than isogenic XX lines. Possible mechanisms for this protective effect of X chromosome dosage include our discovery that the lines with two active X Chromosomes have a higher expression of 2-cell-like state genes, and differential expression of X-linked tumor suppressor genes associated with DNA damage response. Highlights First genetic analysis of predisposition to aneuploidy in pluripotent stem cell cultures Chromosomal regions duplicated in aneuploid mouse embryonic stem cells are syntenic with regions overrepresented in human pluripotent stem cell lines bearing recurrent genetic abnormalities X-Chromosome dosage strongly influences susceptibility to aneuploidy in mouse embryonic stem cells and to a lesser degree in human pluripotent stem cells XX mouse embryonic stem cell lines show a higher proportion of cells in 2 cell-like state and higher expression of tumor suppressor genes associated with DNA damage response">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f853ae6e95a31cb9ad3712a21b0838a6601fe33" target='_blank'>
              Chromosome X Dosage Modulates Development of Aneuploidy in Genetically Diverse Mouse Embryonic Stem Cells
              </a>
            </td>
          <td>
            Alexander Stanton, Selcan Aydin, Daniel A. Skelly, Dylan Stavish, Kim Leonhard, Seth M. Taapken, Erik M. McIntire, Matthew Pankratz, A. Czechanski, T. Ludwig, Ted Choi, Steve P. Gygi, Ivana Barbaric, Steven C. Munger, L. Reinholdt, Martin F. Pera
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Non-coding RNAs play a significant role in viral infection cycles, with recent attention focused on circular RNAs (circRNAs) originating from various viral families. Notably, these circRNAs have been associated with oncogenesis and alterations in viral fitness. However, identifying their expression has proven more challenging than initially anticipated due to unique viral characteristics. This challenge has the potential to impede progress in our understanding of viral circRNAs. Key hurdles in working with viral genomes include: (1) the presence of repetitive regions that can lead to misalignment of sequencing reads, and (2) unconventional splicing mechanisms that deviate from conserved eukaryotic patterns. To address these challenges, we developed vCircTrappist, a bioinformatic pipeline tailored to identify backsplicing events and pinpoint loci expressing circRNAs in RNA sequencing data. Applying this pipeline, we obtained novel insights from both new and existing datasets encompassing a range of animal and human pathogens belonging to Herpesviridae, Retroviridae, Adenoviridae and Orthomyxoviridae families. Subsequent RT-PCR and Sanger sequencings validated the accuracy of the developed bioinformatic tool for a selection of new candidate viral encoded circRNAs. These findings demonstrate that vCircTrappist is an open and unbiased approach for comprehensive identification of virus-derived circRNAs. Significance Statement Circular RNAs (circRNAs) were revealed to have prominent roles in cellular life in the past decade. They were more recently shown to be expressed by viruses, influencing their infectious cycles and host-pathogen relationship. In this context, viruses that were not previously associated with cellular splicing processes are shown to express circRNAs through unknown mechanisms. These non-canonical circRNAs were already shown to be important in the viral cycle and pathogenesis of the viruses they are encoded from. Here, we propose a bioinformatics pipeline that bypasses the limitations of the existing tools in the identification of viral circRNA. Using this pipeline, we discovered numerous candidates and invite the reader to start its own exploration in the realm of viral encoded circRNAs. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4ea92d444a260798f5779de2f8ef3c3e67a41ef" target='_blank'>
              Unbiased and comprehensive identification of viral encoded circular RNAs in a large range of viral species and families
              </a>
            </td>
          <td>
            Alexis S Chasseur, Maxime Bellefroid, Mathilde Galais, Meijiao Gong, Sarah Mathieu, Camille Ponsard, Laure Vreux, Carlo Yague-Sanz, B. Dewals, N. Gillet, Benoît Muylkens, C. Van Lint, Damien Coupeau
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The p53, a pivotal tumor suppressor, regulates various cellular responses, including DNA repair and apoptosis. Normally, p53 levels are low due to murine double minute clone 2 (MDM2) mediated polyubiquitination. However, stress signals disrupt p53-MDM2 interaction, stabilizing p53 and activating target genes. Dysfunctional p53 is common in cancers, especially colorectal cancer (CRC), with TP53 mutations in 43% of tumors. These mutations impair wild-type p53 function or confer novel activities, promoting cancer progression. Despite drugs targeting p53 entering trials, understanding wild-type and mutant p53 functions is crucial for novel CRC therapies. P53 mutations not only impact DNA repair and apoptosis but also play a crucial role in tumor immunotherapy. While rendering tumors resistant to chemotherapy, p53 mutations provide opportunities for immunotherapy due to neoantigen-rich tumors. Additionally, p53 mutations influence tumor microenvironment cells, such as fibroblasts and immunosuppressive cells, through p53-mediated signaling pathways. Investigating p53 mutations in tumor therapy is vital for personalized medicine and immunotherapy. In cancer treatment research, scientists explore drugs and strategies to restore or enhance p53 function. Targeting wild-type p53 aims to restore DNA repair and cell cycle control, while targeting mutant p53 seeks new drugs to inhibit its detrimental effects, advancing tumor treatment. Understanding p53 drugs and strategies is crucial for cancer therapy progress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4492a2019f4f8e179322826eb5b657b70923676" target='_blank'>
              Unraveling the Role of TP53 in Colorectal Cancer Therapy: From Wild-Type Regulation to Mutant.
              </a>
            </td>
          <td>
            Wenshu Li, Longyuan Li, Huan Yang, Chenxi Shi, Zhe Lei, Lingchuan Guo, Yuhong Wang
          </td>
          <td>2024-07-25</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Background/Objectives The complex system of cancer has led to an emphasis on understanding the more general causal relationship within the disease. In this context, concepts of symmetry and symmetry-breaking in distinct biological cell features or components have been examined as an approach to cancer investigation. However, there can be possible limitations in directly interpreting the symmetry-based approach from a physical viewpoint due to the lack of understanding of physical laws governing symmetry in complex systems like cancer. Methods Fractal geometry and DNA walk representation were employed to investigate the geometric features i.e., self-similarity and heterogeneity in DNA nucleotide coding sequences of wild-type and mutated oncogenes, tumour-suppressor, and other unclassified genes. The mutation-facilitated self-similar and heterogenous features were quantified by the fractal dimension and lacunarity coefficient measures, respectively. Additionally, the geometrical orderedness and disorderedness in the analyzed sequences were interpreted from the combination of the fractal measures. Results The findings showed distinct fractal geometric features in the case of fusion mutations. It also highlights the possible interpretation of the observed fractal features as geometric analogues concerning explicit observations corresponding to specific cancer types. In addition, the two-dimensional multi-fractal analysis highlighted the presence of a single exponent in the scaling of mutation-mediated gene sequence self-similarity/complexity and heterogeneity. Conclusions The approach identified mutation-induced geometric features in gene sequences, demonstrating the potential of DNA walks and fractal analysis in translational research regarding cancer. The findings suggest that investigating fractal parameters can capture unique geometric features in nucleotide sequences, contributing to the understanding of cancer’s molecular complexity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a73efc783fc74b512f9c22058780b5d74879423" target='_blank'>
              DNA walk of specific fused oncogenes exhibit distinct fractal geometric characteristics in nucleotide patterns
              </a>
            </td>
          <td>
            Abhijeet Das, Manas Sehgal, A. Singh, Rishabh Goyal, Mallika Prabhakar, Jeremy Fricke, Isa Mambetsariev, Prakash Kulkarni, M. Jolly, Ravi Salgia
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42da14688d34e68034cf1e370a5861d8d265cacc" target='_blank'>
              Telomere-to-telomere genome assembly of the goose Anser cygnoides
              </a>
            </td>
          <td>
            Hongchang Zhao, Hao Zhou, Guobo Sun, Biao Dong, Wenqi Zhu, Xiaohui Mu, Xiaoming Li, Jun Wang, Mengli Zhao, Wenhao Yang, Gansheng Zhang, Rongchao Ji, T. Geng, Daoqing Gong, He Meng, Jian Wang
          </td>
          <td>2024-07-07</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Transmembrane signaling is a critical process by which changes in the extracellular environment are relayed to intracellular systems that induce changes in homeostasis. One common intracellular system involves guanine nucleotide exchange factors (GEFs), which catalyzes the exchange of GTP for GDP bound to inactive guanine nucleotide binding proteins (G proteins). The resulting active G proteins then interact with downstream targets that control cell proliferation, growth, shape, migration, adhesion, and transcription. Dysregulation of any of these processes is a hallmark of cancer. The Dbl family of GEFs activate Rho family G proteins, which in turn alter the actin cytoskeleton and promote gene transcription. Although they have a common catalytic mechanism exercised by their conserved Dbl homology (DH) domains, Dbl GEFs are regulated in very diverse ways. Often, this regulation involves the release of autoinhibition imposed by accessory domains. Amongst these domains, the pleckstrin homology (PH) domain is the most conserved and is almost always found immediately C-terminal to the DH domain. The domain been associated with both positive and negative regulation. Recently, some atomic structures of Dbl GEFs have been determined which reemphasize the complex and central role that the PH domain can play in orchestrating regulation of the DH domain. Here we discuss these newer structures, put them into context by cataloging the various ways that PH domains are known to contribute to signaling across the Dbl family, and discuss how the PH might be exploited to achieve selective inhibition of this protein family by small molecule therapeutics. Significance Statement Dysregulation via overexpression or mutation of Dbl family RhoGEFs contributes disease. Targeting the Dbl homology (DH) catalytic domain by small molecule therapeutics has been challenging due to its high conservation and the lack of a discrete binding pocket. By evaluating some new autoinhibitory mechanisms in the Dbl family, we demonstrate the great diversity of roles played by the regulatory domains, in particular the PH domain, and how this holds tremendous potential for development of selective therapeutics that modulate GEF activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21ef8c8ae468026b42874719f4b7ad3d348e5774" target='_blank'>
              New mechanisms underlying oncogenesis in Dbl family Rho guanine nucleotide exchange factors.
              </a>
            </td>
          <td>
            Sandeep K. Ravala, J. J. G. Tesmer
          </td>
          <td>2024-06-20</td>
          <td>Molecular pharmacology</td>
          <td>2</td>
          <td>2</td>
        </tr>

        <tr id="Each piece of cell-free DNA (cfDNA) has a length determined by the exact metabolic conditions in the cell it belonged to at the time of cell death. The changes in cellular regulation leading to a variety of patterns, which are based on the different number of fragments with lengths up to several hundred base pairs at each of the almost three billion genomic positions, allow for the detection of disease and also the precise identification of the tissue of their origin. A Kullback-Leibler (KL) divergence computation identifies different fragment lengths and areas of the human genome, depending on the stage, for which disease samples, starting from pre-clinical disease stages, diverge from healthy donor samples. We provide examples of genes related to colorectal cancer (CRC), which our algorithm detected to belong to divergent genomic bins. The staging of CRC can be viewed as a Markov Chain and that provides a framework for studying disease progression and the types of epigenetic changes occurring longitudinally at each stage, which might aid the correct classification of a new hospital sample. In a new look to treat such data as grayscale value images, pattern recognition using artificial intelligence (AI) could be one approach to classification. In CRC, Stage I disease does not, for the most part, shed any tumor circulation, making detection difficult for established machine learning (ML) methods. This leads to the deduction that early detection, where we can only rely on changes in the metabolic patterns, can be accomplished when the information is considered in its entirety, for example by applying computer vision methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769c868029bf18fe644fcae9a2141c21f2b5a763" target='_blank'>
              Channel Capacity of Genome-Wide Cell-Free DNA Fragment Length Distribution in Colorectal Cancer
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-07-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) constitute a class of endogenous non-coding RNAs (ncRNAs) that lack a 5’-ended cap and 3’-ended poly (A) tail and form a closed ring structure with covalent bonds. Due to its special structure, circRNA is resistant to Exonuclease R (RNaseR), making its distribution in the cytoplasm quite rich. Advanced high-throughput sequencing and bioinformatics methods have revealed that circRNA is highly conserved, stable, and disease- and tissue-specific. Furthermore, increasing research has confirmed that circRNA, as a driver or suppressor, regulates cancer onset and progression by modulating a series of pathophysiological mechanisms. As a result, circRNA has emerged as a clinical biomarker and therapeutic intervention target. This article reviews the biological functions and regulatory mechanisms of circRNA in the context of respiratory cancer onset and progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e236d9529d10a4f102e62ec53300a30c33441387" target='_blank'>
              Roles and mechanisms of circular RNA in respiratory system cancers
              </a>
            </td>
          <td>
            Nan Yang, Mengwen Jiao, Yuewen Zhang, Shaokang Mo, Ling Wang, Jianqing Liang
          </td>
          <td>2024-07-15</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mutations in polymerases Pold1 and Pole exonuclease domains in humans are associated with increased cancer incidence, elevated tumor mutation burden (TMB) and response to immune checkpoint blockade (ICB). Although ICB is approved for treatment of several cancers, not all tumors with elevated TMB respond. Here we generated Pold1 and Pole proofreading mutator mice and show that ICB treatment of mice with high TMB tumors did not improve survival as only a subset of tumors responded. Similarly, introducing the mutator alleles into mice with Kras/p53 lung cancer did not improve survival, however, passaging mutator tumor cells in vitro without immune editing caused rejection in immune-competent hosts, demonstrating the efficiency by which cells with antigenic mutations are eliminated. Finally, ICB treatment of mutator mice earlier, before observable tumors delayed cancer onset, improved survival, and selected for tumors without aneuploidy, suggesting the use of ICB in individuals at high risk for cancer prevention. Highlights Germline somatic and conditional Pold1 and Pole exonuclease domain mutations in mice produce a mutator phenotype. Spontaneous cancers arise in mutator mice that have genomic features comparable to human tumors with these mutations. ICB treatment of mutator mice with tumors did not improve survival as only a subset of tumors respond. Introduction of the mutator alleles into an autochthonous mouse lung cancer model also did not produce immunogenic tumors, whereas passaging mutator tumor cells in vitro caused immune rejection indicating efficient selection against antigenic mutations in vivo. Prophylactic ICB treatment delayed cancer onset, improved survival, and selected for tumors with no aneuploidy. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9e6ba5a717c7dd3412aa2d3806c5dc666fb03f9" target='_blank'>
              Immune Checkpoint Blockade Delays Cancer and Extends Survival in Murine DNA Polymerase Mutator Syndromes
              </a>
            </td>
          <td>
            Akshada Sawant, Fuqian Shi, E. C. Lopes, Zhixian Hu, Somer Abdelfattah, Jennele Baul, Jesse Powers, Christian S. Hinrichs, Joshua D. Rabinowitz, Chang S. Chan, Edmund C. Lattime, Shridar Ganesan, Eileen White
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Long noncoding RNA (lncRNA) is a non-coding RNA with a length of more than 200 nucleotides, involved in multiple regulatory processes in vivo, and is related to the physiology and pathology of human diseases. An increasing number of experimental results suggest that when lncRNA is abnormally expressed, it results in the development of tumors. LncRNAs can be divided into five broad categories: sense, antisense, bidirectional, intronic, and intergenic. Studies have found that some antisense lncRNAs are involved in a variety of human tumorigenesis. The newly identified ROR1-AS1, which functions as an antisense RNA of ROR1, is located in the 1p31.3 region of the human genome. Recent studies have reported that abnormal expression of lncRNA ROR1-AS1 can affect cell growth, proliferation, invasion, and metastasis and increase oncogenesis and tumor spread, indicating lncRNA ROR1-AS1 as a promising target for many tumor biological therapies. In this study, the pathophysiology and molecular mechanism of ROR1-AS1 in various malignancies are discussed by retrieving the related literature. ROR1-AS1 is a cancer-associated lncRNA, and studies have found that it is either over- or underexpressed in multiple malignancies, including liver cancer, colon cancer, osteosarcoma, glioma, cervical cancer, bladder cancer, lung adenocarcinoma, and mantle cell lymphoma. Furthermore, it has been demonstrated that lncRNA ROR1-AS1 participates in proliferation, migration, invasion, and suppression of apoptosis of cancer cells. Furthermore, lncRNA ROR1-AS1 promotes the development of tumors by up-regulating or downregulating ROR1-AS1 conjugates and various pathways and miR-504, miR-4686, miR-670-3p, and miR-375 sponges, etc., suggesting that lncRNA ROR1-AS1 may be used as a marker in tumors or a potential therapeutic target for a variety of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7246d4b51bc4f37d4b20e1d534627c70d6f7fbe2" target='_blank'>
              ROR1-AS1: A Meaningful Long Noncoding RNA in Oncogenesis.
              </a>
            </td>
          <td>
            Hong Fan, Yunxi Zhou, Ziyan Zhang, Gang Zhou, Chengfu Yuan
          </td>
          <td>2024-06-10</td>
          <td>Mini reviews in medicinal chemistry</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are cardinal players in numerous physiological and pathological processes. CircRNAs play dual roles as tumor suppressors and oncogenes in different oncological contexts, including hepatocellular carcinoma (HCC). Their roles significantly impact the disease at all stages, including initiation, development, progression, invasion, and metastasis, in addition to the response to treatment. In this review, we discuss the biogenesis and regulatory functional roles of circRNAs, as well as circRNA–protein–mRNA ternary complex formation, elucidating the intricate pathways tuned by circRNAs to modulate gene expression and cellular processes through a comprehensive literature search, in silico search, and bioinformatics analysis. With a particular focus on the interplay between circRNAs, epigenetics, and HCC pathology, the article sets the stage for further exploration of circRNAs as novel investigational theranostic agents in the dynamic realm of HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77ffac464081cbdd21f8bde011de6f9367afe45c" target='_blank'>
              A Comprehensive Insight and In Silico Analysis of CircRNAs in Hepatocellular Carcinoma: A Step Toward ncRNA-Based Precision Medicine
              </a>
            </td>
          <td>
            R. A. Youness, H. A. Hassan, Tasneem Abaza, Ahmed A. Hady, Hekmat M. El Magdoub, Mohamed Ali, Johannes Vogel, M. Thiersch, Max Gassmann, N. Hamdy, Mostafa A. Aboouf
          </td>
          <td>2024-07-24</td>
          <td>Cells</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2c2600dfae0e3e2400091b948e5dbe761b1b615" target='_blank'>
              Current understanding of functional peptides encoded by lncRNA in cancer
              </a>
            </td>
          <td>
            Hua Tian, Lu Tang, Zihan Yang, Yanxi Xiang, Qi Min, Mengshuang Yin, Huili You, Zhangang Xiao, Jing Shen
          </td>
          <td>2024-07-19</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c299e5ae0a72395986d242cc2a31e2707c0b05bd" target='_blank'>
              Three-dimensional chromatin reorganization regulates B cell development during ageing
              </a>
            </td>
          <td>
            Fei Ma, Yaqiang Cao, Hansen Du, Fatima-Zohra Braikia, Le Zong, Noah Ollikainen, Marci Bayer, Xiang Qiu, Bongsoo Park, Roshni Roy, Satabdi Nandi, Dimitra Sarantopoulou, Andrew Ziman, Aisha Haley Bianchi, Isabel Beerman, Keji Zhao, Rudolf Grosschedl, Ranjan Sen
          </td>
          <td>2024-06-12</td>
          <td>Nature Cell Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Aneuploidy, an imbalanced number of chromosomes or chromosome arms, is a genetic hallmark of cancer cells, yet aneuploidy remains a biological enigma and a missed opportunity for cancer therapy. Aneuploid cells must cope with several types of cellular stresses, potentially creating synthetic lethalities that can be used to target aneuploid cancer cells. Here, I will describe our efforts to identify synthetic lethalities of the aneuploid state (in contrast to synthetic lethalities of specific recurrent aneuploidies. Specifically, I will focus on three recent unpublished studies, where we report that: (1) Aneuploid cells are preferentially sensitive to perturbation of the spindle assembly checkpoint (SAC) and its regulator KIF18A; the expression and activity of CDC20 determine the sensitivity to SAC inhibition. (2) Aneuploid cells depend on MAPK signaling for overcoming aneuploidy-induced DNA damage; targeting MAPK signaling can sensitize aneuploid cells to DNA damage inducing agents and to PARP inhibitors. (3) Aneuploid cells depend on RNA and protein degradation mechanisms to attenuate the cellular consequence of extra chromosomes; this renders aneuploid cells more sensitive to inhibition of nonsense-mediate decay, miRNA-mediated gene silencing, and the proteasome complex.
 Citation Format: Uri Ben-David. Identifying and targeting synthetic lethalities of aneuploid (cancer) cells [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA015.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac71f1802eee0a6fdce6ada7502686abe3e34f7b" target='_blank'>
              Abstract IA015: Identifying and targeting synthetic lethalities of aneuploid (cancer) cells
              </a>
            </td>
          <td>
            Uri Ben-David
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc05d0e416d1c5836cac6aabe3bd3a577d1440ba" target='_blank'>
              Enhanced eMAGE applied to identify genetic factors of nuclear hormone receptor dysfunction via combinatorial gene editing
              </a>
            </td>
          <td>
            Peter N Ciaccia, Zhuobin Liang, Anabel Y. Schweitzer, Eli Metzner, Farren J. Isaacs
          </td>
          <td>2024-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3e129666ae56c033aa207d599e42711f40eae8" target='_blank'>
              Gene body DNA hydroxymethylation restricts the magnitude of transcriptional changes during aging
              </a>
            </td>
          <td>
            James R. Occean, Na Yang, Yan Sun, Marshall S. Dawkins, R. Munk, Cedric Belair, Showkat A. Dar, C. Anerillas, Lin Wang, Changyou Shi, Christopher Dunn, Michel Bernier, Nathan L. Price, Julie S Kim, Chang-Yi Cui, Jinshui Fan, Moitrayee Bhattacharyya, Supriyo De, Manolis Maragkakis, R. de Cabo, Simone Sidoli, P. Sen
          </td>
          <td>2024-07-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96ca99a18dd08dc5d59b7f900d05406ec6c8cd83" target='_blank'>
              Amplifications of EVX2 and HOXD9-HOXD13 on 2q31 in mature cystic teratomas of the ovary identified by array comparative genomic hybridization may explain teratoma characteristics in chondrogenesis and osteogenesis
              </a>
            </td>
          <td>
            Wen-Chung Wang, Tai-Cheng Hou, C. Kuo, Yen-Chein Lai
          </td>
          <td>2024-06-21</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Background Germline mutations have been identified in a small number of hereditary cancers, but the genetic predisposition for many familial cancers remains to be elucidated. Methods This study identified a Chinese pedigree that presented different cancers (breast cancer, BRCA; adenocarcinoma of the esophagogastric junction, AEG; and B‐cell acute lymphoblastic leukemia, B‐ALL) in each of the three generations. Whole‐genome sequencing and whole‐exome sequencing were performed on peripheral blood or bone marrow and cancer biopsy samples. Whole‐genome bisulfite sequencing was conducted on the monozygotic twin brothers, one of whom developed B‐ALL. Results According to the ACMG guidelines, bioinformatic analysis of the genome sequencing revealed 20 germline mutations, particularly mutations in the DNAH11 (c.9463G > A) and CFH (c.2314G > A) genes that were documented in the COSMIC database and validated by Sanger sequencing. Forty‐one common somatic mutated genes were identified in the cancer samples, displaying the same type of single nucleotide substitution Signature 5. Meanwhile, hypomethylation of PLEK2, MRAS, and RXRA as well as hypermethylation of CpG island associated with WT1 was shown in the twin with B‐ALL. Conclusions These findings reveal genomic alterations in a pedigree with multiple cancers. Mutations found in the DNAH11, CFH genes, and other genes predispose to malignancies in this family. Dysregulated methylation of WT1, PLEK2, MRAS, and RXRA in the twin with B‐ALL increases cancer susceptibility. The similarity of the somatic genetic changes among the three cancers indicates a hereditary impact on the pedigree. These familial cancers with germline and somatic mutations, as well as epigenomic alterations, represent a common molecular basis for many multiple cancer pedigrees.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3f0ae27a463e75a2f599e816c3a044e8b7c954" target='_blank'>
              Impact of genomic and epigenomic alterations of multigene on a multicancer pedigree
              </a>
            </td>
          <td>
            Jinyu Gao, Yongzhang Wu, Jieming Yu, Yinbin Qiu, Tiantian Yi, Chaochao Luo, Junxiao Zhang, Gary Lu, Xu Li, Fu Xiong, Xuedong Wu, Xinghua Pan
          </td>
          <td>2024-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="tRNA-derived fragments (tRFs) are novel small noncoding RNAs (sncRNAs) that range from approximately 14 to 50 nt. They are generated by the cleavage of mature tRNAs or precursor tRNAs (pre-tRNAs) at specific sites. Based on their origin and length, tRFs can be classified into three categories: (1) tRF-1 s; (2) tRF-3 s, tRF-5 s, and internal tRFs (i-tRFs); and (3) tRNA halves. They play important roles in stress response, signal transduction, and gene expression processes. Recent studies have identified differential expression of tRFs in various tumors. Aberrantly expressed tRFs have critical clinical value and show promise as new biomarkers for tumor diagnosis and prognosis and as therapeutic targets. tRFs regulate the malignant progression of tumors via various mechanisms, primarily including modulation of noncoding RNA biogenesis, global chromatin organization, gene expression regulation, modulation of protein translation, regulation of epigenetic modification, and alternative splicing regulation. In conclusion, tRF-mediated regulatory pathways could present new avenues for tumor treatment, and tRFs could serve as promising therapeutic targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5d61087a4e16d07f2ecf9e86f61657b2d38ec3" target='_blank'>
              tRNA-derived fragments: Unveiling new roles and molecular mechanisms in cancer progression.
              </a>
            </td>
          <td>
            Jingjing Lu, Ping Zhu, Xiufen Zhang, Linzi Zeng, Bujie Xu, Ping Zhou
          </td>
          <td>2024-06-12</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human APOBEC single-strand (ss) specific DNA and RNA cytidine deaminases change cytosines to uracils and function in antiviral innate immunity, RNA editing, and can cause hypermutation in chromosomes. The resulting uracils can be directly replicated, resulting in C to T mutations, or uracil-DNA glycosylase can convert the uracils to abasic (AP) sites which are then fixed as C to T or C to G mutations by translesion DNA polymerases. We noticed that in yeast and in human cancers, contributions of C to T and C to G mutations depends on the origin of ssDNA mutagenized by APOBECs. Since ssDNA in eukaryotic genomes readily binds to replication protein A (RPA) we asked if RPA could affect APOBEC-induced mutation spectrum in yeast. For that purpose, we expressed human APOBECs in the wild-type yeast and in strains carrying a hypomorph mutation rfa1-t33 in the large RPA subunit. We confirmed that the rfa1-t33 allele can facilitate mutagenesis by APOBECs. We also found that the rfa1-t33 mutation changed the ratio of APOBEC3A-induced T to C and T to G mutations in replicating yeast to resemble a ratio observed in long-persistent ssDNA in yeast and in cancers. We present the data suggesting that RPA may shield APOBEC formed uracils in ssDNA from Ung1, thereby facilitating C to T mutagenesis through the accurate copying of uracils by replicative DNA polymerases. Unexpectedly, we also found that for uracils shielded from Ung1 by wild-type RPA the mutagenic outcome is reduced in the presence of translesion DNA polymerase zeta.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/106b9d766b6ab541cb0c665a0826fbe3df05d8f1" target='_blank'>
              Hypomorphic mutation in the large subunit of replication protein A affects mutagenesis by human APOBEC cytidine deaminases in yeast
              </a>
            </td>
          <td>
            Matthew S. Dennen, Z. Kockler, Steven A. Roberts, Adam B. Burkholder, L. Klimczak, D. Gordenin
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Disease risk prediction based on genomic sequence and transcriptional profile can improve disease screening and prevention. Despite identifying many disease-associated DNA variants, distinguishing deleterious non-coding DNA variations remains poor for most common diseases. In this study, we designed in vitro experiments to uncover the significance of occupancy and competitive binding between P53 and cMYC on common target genes. Analyzing publicly available ChIP-seq data for P53 and cMYC in embryonic stem cells showed that ~344-366 regions are co-occupied, and on average, two cis-overlapping motifs (CisOMs) per region were identified, suggesting that co-occupancy is evolutionarily conserved. Using U2OS and Raji cells untreated and treated with doxorubicin to increase P53 protein level while potentially reducing cMYC level, ChIP-seq analysis illustrated that around 16 to 922 genomic regions were co-occupied by P53 and cMYC, and substitutions of cMYC signals by P53 were detected post doxorubicin treatment. Around 187 expressed genes near co-occupied regions were altered at mRNA level according to RNA-seq data analysis. We utilized a computational motif-matching approach to illustrate that changes in predicted P53 binding affinity in CisOMs of co-occupied elements significantly correlate with alterations in reporter gene expression. We performed a similar analysis using SNPs mapped in CisOMs for P53 and cMYC from ChIP-seq data, and expression of target genes from GTEx portal. We found significant correlation between change in cMYC-motif binding affinity in CisOMs and altered expression. Our study brings us closer to developing a generally applicable approach to filter etiological non-coding variations associated with common diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2175dd5f742d55fc20f67f497dba908aa719b17b" target='_blank'>
              Sequence-to-expression approach to identify etiological non-coding DNA variations in P53 and cMYC-driven diseases.
              </a>
            </td>
          <td>
            K. Kin, Shounak Bhogale, Lisha Zhu, Derrick Thomas, Jessica W Bertol, W. J. Zheng, Saurabh Sinha, W. Fakhouri
          </td>
          <td>2024-07-17</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Maintaining the structural integrity of genomic chromosomal DNA is an essential role of cellular life and requires two important biological mechanisms: the DNA damage response (DDR) mechanism and telomere protection mechanism at chromosome ends. Because abnormalities in telomeres and cellular DDR regulation are strongly associated with human aging and cancer, there is a reciprocal regulation of telomeres and cellular DDR. Moreover, several drug treatments for DDR are currently available. This paper reviews the progress in research on the interaction between telomeres and cellular DNA damage repair pathways. The research on the crosstalk between telomere damage and DDR is important for improving the efficacy of tumor treatment. However, further studies are required to confirm this hypothesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5cb84c62db53ba0d298c4d260583f3cee35d1de" target='_blank'>
              Telomere-related DNA damage response pathways in cancer therapy: prospective targets
              </a>
            </td>
          <td>
            Liting Gu, Mingdi Liu, Yuning Zhang, Honglan Zhou, Yishu Wang, Zhi-Xiang Xu
          </td>
          <td>2024-06-07</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The intestines are the largest barrier organ in the human body. The intestinal barrier plays a crucial role in maintaining the balance of the intestinal environment and protecting the intestines from harmful bacterial invasion. Single-cell RNA sequencing technology allows the detection of the different cell types in the intestine in two dimensions and the exploration of cell types that have not been fully characterized. The intestinal mucosa is highly complex in structure, and its proper functioning is linked to multiple structures in the proximal-distal intestinal and luminal-mucosal axes. Spatial localization is at the core of the efforts to explore the interactions between the complex structures. Spatial transcriptomics (ST) is a method that allows for comprehensive tissue analysis and the acquisition of spatially separated genetic information from individual cells, while preserving their spatial location and interactions. This approach also prevents the loss of fragile cells during tissue disaggregation. The emergence of ST technology allows us to spatially dissect enzymatic processes and interactions between multiple cells, genes, proteins and signals in the intestine. This includes the exchange of oxygen and nutrients in the intestine, different gradients of microbial populations and the role of extracellular matrix proteins. This regionally precise approach to tissue studies is gaining more acceptance and is increasingly applied in the investigation of disease mechanisms related to the gastrointestinal tract. Therefore, this review summarized the application of ST in gastrointestinal diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/354ab040d0d2bb0098cc2d055871bbc1b44b39d4" target='_blank'>
              Deciphering the spatiotemporal transcriptional landscape of intestinal diseases (Review)
              </a>
            </td>
          <td>
            Yajing Guo, Chao Ren, Yuxi He, Yue Wu, Xiao-Jun Yang
          </td>
          <td>2024-07-04</td>
          <td>Molecular Medicine Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="T lymphocytes play a critical role in antitumor immunity, but their exhaustion poses a significant challenge for immune evasion by malignant cells. Circular RNAs (circRNAs), characterized by their covalently closed looped structure, have emerged as pivotal regulators within the neoplastic landscape. Recent studies have highlighted their multifaceted roles in cellular processes, including gene expression modulation and protein function regulation, which are often disrupted in cancer. In this review, we systematically explore the intricate interplay between circRNAs and T cell modulation within the tumor microenvironment. By dissecting the regulatory mechanisms through which circRNAs impact T cell exhaustion, we aim to uncover pathways crucial for immune evasion and T cell dysfunction. These insights can inform innovative immunotherapeutic strategies targeting circRNA-mediated molecular pathways. Additionally, we discuss the translational potential of circRNAs as biomarkers for therapeutic response prediction and as intervention targets. Our comprehensive analysis aims to enhance the understanding of immune evasion dynamics in the tumor microenvironment by facilitating the development of precision immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efe914963da3c327fd7451b74772b490e264e429" target='_blank'>
              CircRNA Regulation of T Cells in Cancer: Unraveling Potential Targets
              </a>
            </td>
          <td>
            Zelin Li, Shuanshuan Yin, Kangping Yang, Baojie Zhang, Xuanhuang Wu, Meng Zhang, Dian Gao
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/523beb9c56ee69616ab5249d9438f1967d1b0092" target='_blank'>
              MicroRNAs role in telomere length maintenance and telomerase activity in tumor cells.
              </a>
            </td>
          <td>
            Stefanne M Bortoletto, Emanuelle Nunes-Souza, Rafael Marchi, M. Ruthes, Larissa M. Okano, Maria Vitoria Tofolo, Ariana Centa, A. S. Fonseca, Daiane Rosolen, Luciane R Cavalli
          </td>
          <td>2024-07-23</td>
          <td>Journal of molecular medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ee3ef93f75d18cca805d005d653d62f6f76b88e" target='_blank'>
              Comprehensive pan-cancer analysis reveals that C5orf34 regulates the proliferation and mortality of lung cancer.
              </a>
            </td>
          <td>
            Mengmeng Yang, Yuhan Deng, Yu Ma, Chunli Song, Zhenhua Wu, Xiayimaierdan Yibulayin, Xiaohong Sun, Yunquan Guo, Dan He
          </td>
          <td>2024-06-29</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The "Mlx" and "Myc" transcription factor networks cross-communicate and share many common gene targets. Myc's activity depends upon its heterodimerization with Max, whereas the Mlx Network requires that the Max-like factor Mlx associate with the Myc-like factors MondoA or ChREBP. The current work demonstrates that body-wide Mlx inactivation, like that of Myc, accelerates numerous aging-related phenotypes pertaining to body habitus and metabolism. The deregulation of numerous aging-related Myc target gene sets is also accelerated. Among other functions, these gene sets often regulate ribosomal and mitochondrial structure and function, genomic stability, and aging. Whereas "MycKO" mice have an extended lifespan because of a lower cancer incidence, "MlxKO" mice have normal lifespans and a higher cancer incidence. Like Myc, the expression of Mlx, MondoA, and ChREBP and their control over their target genes deteriorate with age in both mice and humans. Collectively, these findings underscore the importance of lifelong and balanced cross-talk between the two networks to maintain proper function and regulation of the many factors that can affect normal aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d253c974701d0df31f2b604cd9bd6ed6bf34ec98" target='_blank'>
              Body-Wide Inactivation of the Myc-Like Mlx Transcription Factor Network Accelerates Aging and Increases the Lifetime Cancer Incidence.
              </a>
            </td>
          <td>
            Huabo Wang, Taylor Stevens, Jie Lu, Alexander Roberts, Clinton Van't land, Radhika Muzumdar, Zhenwei Gong, Jerry Vockley, E. Prochownik
          </td>
          <td>2024-07-08</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The human Origin Recognition Complex (ORC) is required not only for the initiation of DNA replication, but is also implicated in diverse cellular functions, including chromatin organization, centrosome biology, and cytokinesis. The smallest subunit of ORC, Orc6, is poorly conserved amongst eukaryotes. Recent studies from our laboratory have suggested that human Orc6 is not required for replication licensing, but is needed for S-phase progression. Further, ATR-dependent phosphorylation of Orc6 at T229 is implicated in DNA damage response during S-phase. In this study, we demonstrate that the CDK-dependent phosphorylation of Orc6 at T195 occurs during mitosis. While the phosphorylation at T195 does not seem to be required to exit mitosis, cells expressing the phosphomimetic T195E mutant of Orc6 impede S-phase progression. Moreover, the phosphorylated form of Orc6 associates with ORC more robustly, and Orc6 shows enhanced association with the ORC outside of G1, supporting the view that Orc6 may prevent the role of Orc1-5 in licensing outside of G1. Finally, Orc6 and the phosphorylated Orc6 localize to the nucleolar organizing centers and regulate ribosome biogenesis. Our results suggest that phosphorylated Orc6 at T195 prevents replication.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f45ae10e610dec214a1ea2fb62eeb8f8a4950c19" target='_blank'>
              Phosphorylation of Orc6 During Mitosis Regulates DNA Replication and Ribosome Biogenesis.
              </a>
            </td>
          <td>
            Fredy Kurniawan, Arindam Chakraborty, Humayra Z Oishi, Minxue Liu, Mariam K. Arif, David Chen, Rishabh Prasanth, Yo-Chuen Lin, Godwin Olalaye, K. Prasanth, S. Prasanth
          </td>
          <td>2024-06-12</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="
 The goal of this study was to identify cancer types with synthetic vulnerability to RNA polymerase I (Pol I) inhibitors and to resolve mechanistic underpinnings of this vulnerability. Specific and selective Pol I inhibitors, pioneered by our group, block the transcription of rRNA, which is the rate-limiting step in ribosome biogenesis and protein synthesis. We conducted unbiased large-scale cancer cell line screens with the Pol I inhibitors and performed correlative analyses using DepMap and Achilles databases to identify cancer types and features that sensitize cancer cells to these inhibitors. The most sensitive lineages were colorectal, endometrial, esophageal, leukemia, myeloma, and ovarian. Microsatellite instable (MSI) cancer cell lines were highly sensitive and significantly more sensitive than microsatellite stable lines. Comparison of the cancer cell line responses to the Achilles gene essentiality database showed high correlation of the sensitivity to genetic knock-out of three Pol I subunits. This unbiased finding strongly supports identification of Pol I as the target. Correlative analyses for vulnerability identified two ribosomal proteins, RPL22 and RPL22L1, and MDM4, a p53 transcriptional repressor as the top candidates. Notably, RPL22K15Rfs hotspot mutation, the top-ranking genetic correlate, is present in up 70% MSI cancer cell lines and tumors. RPL22 and RPL22L1 are a synthetic lethal paralog pair. RPL22 is a negative regulator of RPL22L1 and consequently, haploinsufficiency or loss of RPL22 induces RPL22L1. RPL22L1 was the top-ranking expression correlate. Lastly, MDM4 was the top protein marker and is highly expressed in MSI cancers. These findings were validated using genetic and mechanistic analyses. We show that either genetic or chemical inhibition of the Pol I enzyme decreases the expression of RPL22L1 and MDM4 and alters their splicing. RPL22L1 is spliced to an alternative 3’ splice site in intron 2 leading to an early truncation. MDM4 exon 6 skipping converts the full-length p53 repressive form to a short NMD-targeted form. Mechanistically, by using GoldCLIP-seq, rMATS and splicing analyses, we show that RPL22 is a master regulator of splicing by binding to intronic sequences and splice sites of RPL22L1 and MDM4. Finally, we show the efficacy and tolerance of the Pol I inhibitors in xenograft and patient-derived MSI models expressing these markers. In summary, this synthetic vulnerability screen using new Pol I inhibitors uncovered a connection between Pol I transcription and cellular splicing programs and identified perturbation of this connection in MSI cancers. We conclude that RPL22 mutation could serve as tumor-agnostic biomarker for targeting cancers by Pol I inhibitors. Furthermore, this work identifies a new Pol I transcription-dependent ribosomal protein-governed pathway that converges on the regulation of p53.
 Citation Format: Marikki Laiho, Wenjun Fan, Hester Liu, Gregory C. Stachelek, Asma Begum, Catherine E. Davis, Tony Dorado, Pablo de Leon, Rajeshkumar N.V., James C. Barrow. Mechanisms of therapeutic vulnerability of mismatch repair defective cancers to RNA polymerase I inhibitors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr PR003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7cefc87042a8cb7740c0f2cbbcff23d295a38413" target='_blank'>
              Abstract PR003: Mechanisms of therapeutic vulnerability of mismatch repair defective cancers to RNA polymerase I inhibitors
              </a>
            </td>
          <td>
            M. Laiho, Wenjun Fan, Hester Liu, Gregory C. Stachelek, A. Begum, Catherine E. Davis, Tony Dorado, Pablo de León, N.V. Rajeshkumar, James C Barrow
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="The emergence of cancers involves numerous coding and non-coding genes. Understanding the contribution of non-coding RNAs (ncRNAs) to the cancer neighborhood is crucial for interpreting the interaction between molecular markers of cancer. However, there is a lack of systematic studies on the involvement of ncRNAs in the cancer neighborhood. In this paper, we construct an interaction network which encompasses multiple genes. We focus on the fundamental topological indicator, namely connectivity, and evaluate its performance when applied to cancer-affected genes using statistical indices. Our findings reveal that ncRNAs significantly enhance the connectivity of affected genes and mediate the inclusion of more genes in the cancer module. To further explore the role of ncRNAs in the network, we propose a connectivity-based method which leverages the bridging function of ncRNAs across cancer-affected genes and reveals the non-coding RNAs extended omnigenic module (NeOModule). Topologically, this module promotes the formation of cancer patterns involving ncRNAs. Biologically, it is enriched with cancer pathways and treatment targets, providing valuable insights into disease relationships.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c53c94851580faedc72696e1e649d6cf0d594b0" target='_blank'>
              Non-Coding RNAs Extended Omnigenic Module of Cancers
              </a>
            </td>
          <td>
            Jie Li, Bingbo Wang, Xiujuan Ma
          </td>
          <td>2024-07-27</td>
          <td>Entropy</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="
 Somatic structural variations (SVs) are common in cancer. Although a small fraction of SVs in breast and ovarian cancers can be attributed to homologous recombination deficiency, the underlying molecular mechanisms for the vast majority of somatic SVs remain unclear. Here, we focus on the roles of transcription and DNA replication collisions in genomic instability in cancer. Such collisions are unavoidable in cells since both transcription and replication use the same DNA as template. We hypothesized that transcription replication collisions (TRCs), if not properly repaired, would lead to collapsed replication forks and result in SVs. To this end, we studied somatic SVs in 6193 high-coverage whole-genome sequenced primary and metastatic tumors from three independent pan-cancer cohorts. A total of 13 conserved SV signatures, representing independent molecular mechanisms, were deconvoluted from these cohorts using non-negative matrix factorization approach. We detected replicated-strand bias, the expected footprint of transcription-replication collision, in large tandem duplications (TDs) across multiple cohorts. This bias was only observed in expressed genes, consistent with TRCs depending on transcription activity. Large TDs were abundant in female-specific (breast, ovarian and uterus), upper gastric-intestinal tract, and prostate cancers. They were associated with CDK12 mutations and worse patient survival. CDK12 is a cyclin-dependent kinase, a key regulator of transcription elongation and termination. Deleting or suppressing CDK12 using CRISPR-Cas9 in prostate cell lines increased RNA:DNA hybrids (R-loops), promoted TRCs, and ultimately led to large TDs. Finally, we found that cells lacking CDK12 were sensitive to WEE1, CHK1 and ATR inhibitors. In summary, our data suggest that large TDs in cancer form due to impaired TRC repair and can be used as a biomarker for prognosis and treatment.
 Citation Format: Lixing Yang. Data-driven discoveries of molecular mechanism and therapeutic vulnerabilities of CDK12 mutant tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr B006.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88abb1d2de21d0d5c6fc7861d116fbf694712a58" target='_blank'>
              Abstract B006: Data-driven discoveries of molecular mechanism and therapeutic vulnerabilities of CDK12 mutant tumors
              </a>
            </td>
          <td>
            Lixing Yang
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Protein function alteration and protein mislocalization are cancer hallmarks that drive oncogenesis. N6-methyladenosine (m6A) deposition mediated by METTL3, METTL16, and METTL5 together with the contribution of additional subunits of the m6A system, has shown a dramatic impact on cancer development. However, the cellular localization of m6A proteins inside tumor cells has been little studied so far. Interestingly, recent evidence indicates that m6A methyltransferases are not always confined to the nucleus, suggesting that epitranscriptomic factors may also have multiple oncogenic roles beyond m6A that still represent an unexplored field. To date novel epigenetic drugs targeting m6A modifiers, such as METTL3 inhibitors, are entering into clinical trials, therefore, the study of the potential onco-properties of m6A effectors beyond m6A is required. Here we will provide an overview of methylation-independent functions of the m6A players in cancer, describing the molecular mechanisms involved and the future implications for therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/192f2e95959046928a3405d1ee7fcfee78c0ed05" target='_blank'>
              The m6A-independent role of epitranscriptomic factors in cancer.
              </a>
            </td>
          <td>
            Guglielmo Bove, Marco Crepaldi, Sajid Amin, Wouter Leonard Megchelenbrink, A. Nebbioso, V. Carafa, Lucia Altucci, Nunzio Del Gaudio
          </td>
          <td>2024-06-27</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a190a83f30593fab67bd2e0b946b6e5dcca9e7" target='_blank'>
              STAU1-mediated CNBP mRNA degradation by LINC00665 alters stem cell characteristics in ovarian cancer
              </a>
            </td>
          <td>
            Xiaofang Liu, Yang Chen, Ying Li, Jinling Bai, Zhi Zeng, Min Wang, Yaodong Dong, Yingying Zhou
          </td>
          <td>2024-07-30</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96c93109c4afbdef8c94c625e3001beb18190e19" target='_blank'>
              UBQLN1 links proteostasis and mitochondria function to telomere maintenance in human embryonic stem cells
              </a>
            </td>
          <td>
            Shuang Zhao, Jie Li, Songqi Duan, Chang Liu, Hua Wang, Jiangtao Lu, Nannan Zhao, Xiaoyan Sheng, Yiwei Wu, Yanjun Li, Baofa Sun, Lin Liu
          </td>
          <td>2024-06-21</td>
          <td>Stem Cell Research & Therapy</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Small nucleolar RNAs (snoRNAs) are earning increasing attention from research communities due to their critical role in the post-transcriptional modification of various RNAs. These snoRNAs, along with their associated proteins, are crucial in regulating the expression of a vast array of genes in different human diseases. Primarily, snoRNAs facilitate modifications such as 2′-O-methylation, N-4-acetylation, and pseudouridylation, which impact not only ribosomal RNA (rRNA) and their synthesis but also different RNAs. Functionally, snoRNAs bind with core proteins to form small nucleolar ribonucleoproteins (snoRNPs). These snoRNAs then direct the protein complex to specific sites on target RNA molecules where modifications are necessary for either standard cellular operations or the regulation of pathological mechanisms. At these targeted sites, the proteins coupled with snoRNPs perform the modification processes that are vital for controlling cellular functions. The unique characteristics of snoRNAs and their involvement in various non-metabolic and metabolic diseases highlight their potential as therapeutic targets. Moreover, the precise targeting capability of snoRNAs might be harnessed as a molecular tool to therapeutically address various disease conditions. This review delves into the role of snoRNAs in health and disease and explores the broad potential of these snoRNAs as therapeutic agents in human pathologies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7175ab9e45145f1d7688ce168913fbb9c8f548df" target='_blank'>
              SnoRNAs: Exploring Their Implication in Human Diseases
              </a>
            </td>
          <td>
            Waseem Chauhan, SJ Sudharshan, Sweta Kafle, R. Zennadi
          </td>
          <td>2024-06-29</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Clostridium butyricum (CbAgo)-based bioassays are popular due to their programmability and directional cleavage capabilities. However, the relatively compact protein structure of CbAgo limits its cleavage activity (even at the optimal temperature), thus restricting its wider application. Here, we observed that guide DNA (gDNA) with specific structural features significantly enhanced CbAgo cleavage efficiency. Then, we invented a novel gDNA containing DNAzyme segments (gDNAzyme) that substantially enhanced the CbAgo cleavage efficency (by 100%). Using a molecular dynamics simulation system, we found that the augmented cleavage efficiency might be attributed to the large-scale global movement of the PIWI domain of CbAgo and an increased number of cleavage sites. Moreover, this gDNAzyme feature allowed us to create a biosensor that simultaneously and sensitively detected three pathogenic bacteria without DNA extraction and amplification. Our work not only dramatically expands applications of the CbAgo-based biosensor but also provides unique insight into the protein-DNA interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/668827e60198c9cb2e0c1f0177df0d3ca5709c02" target='_blank'>
              Redesigned Guide DNA Enhanced Clostridium butyricum Argonaute Activity for Amplification-Free and Multiplexed Detection of Pathogens.
              </a>
            </td>
          <td>
            Yongqiang He, Shuai Wang, Junping Wen, Niu Feng, Ruxiang Ma, Hetong Zhang, Guoxun Chen, Xiakun Chu, Yiping Chen
          </td>
          <td>2024-07-25</td>
          <td>Nano letters</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="YAP is a central player in cancer development with functions extending beyond its recognized role in cell growth regulation. Recent work has identified a link between YAP/TAZ and the DNA damage response. Here, we investigated the mechanistic underpinnings of the crosstalk between DNA damage repair and YAP activity. Ku70, a key component of the non-homologous end joining pathway to repair DNA damage, engaged in a dynamic competition with TEAD4 for binding to YAP, limiting the transcriptional activity of YAP. Depletion of Ku70 enhanced interaction between YAP and TEAD4 and boosted YAP transcriptional capacity. Consequently, Ku70 loss enhanced tumorigenesis in colon cancer and hepatocellular carcinoma (HCC) in vivo. YAP impeded DNA damage repair and elevated genome instability by inducing PARP1 degradation through the SMURF2-mediated ubiquitin-proteasome pathway. Analysis of HCC patient samples substantiated the link between Ku70 expression, YAP activity, PARP1 levels, and genome instability. In conclusion, this research provides insight into the mechanistic interactions between YAP and key regulators of DNA damage repair, highlighting the role of a Ku70-YAP-PARP1 axis in preserving genome stability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d02fff425b6184aa5f8abb97e7f6cb2ad73ee59a" target='_blank'>
              Ku70 Binding to YAP Alters PARP1 Ubiquitination to Regulate Genome Stability and Tumorigenesis.
              </a>
            </td>
          <td>
            Yinyin Shu, Xiaoni Jin, Mintao Ji, Zhisen Zhang, Xiuxiu Wang, Haisheng Liang, Shuangshuang Lu, Shuai Dong, Yiping Lin, Yuhan Guo, Qiuyu Zhuang, Yuhong Wang, Zhe Lei, Lingchuan Guo, Xuanyu Meng, Guangming Zhou, Wensheng Zhang, Lei Chang
          </td>
          <td>2024-06-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfc721cd925975049ceb0cd27f9ffa4efdce8611" target='_blank'>
              Synthetic lethality by PARP inhibitors: new mechanism uncovered based on unresolved transcription-replication conflicts
              </a>
            </td>
          <td>
            Marina Kolesnichenko, Claus Scheidereit
          </td>
          <td>2024-07-29</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Exosomes, as pivotal entities within the tumor microenvironment, orchestrate intercellular communication through the transfer of diverse molecules, among which non-coding RNAs (ncRNAs) such as miRNAs, lncRNAs, and circRNAs play a crucial role. These ncRNAs, endowed with regulatory functions, are selectively incorporated into exosomes. Emerging evidence underscores the significance of exosomal ncRNAs in modulating key oncogenic processes in thyroid cancer (TC), including proliferation, metastasis, epithelial-mesenchymal transition (EMT), angiogenesis, and immunoediting. The unique composition of exosomes shields their cargo from enzymatic and chemical degradation, ensuring their integrity and facilitating their specific expression in plasma. This positions exosomal ncRNAs as promising candidates for novel diagnostic and prognostic biomarkers in TC. Moreover, the potential of exosomes in the therapeutic landscape of TC is increasingly recognized. This review aims to elucidate the intricate relationship between exosomal ncRNAs and TC, fostering a deeper comprehension of their mechanistic involvement. By doing so, it endeavors to propel forward the exploration of exosomal ncRNAs in TC, ultimately paving the way for innovative diagnostic and therapeutic strategies predicated on exosomes and their ncRNA content.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8060c643c2843671b19bd4873763105a7bbbac8d" target='_blank'>
              Exosomal noncoding RNAs: decoding their role in thyroid cancer progression
              </a>
            </td>
          <td>
            Weiming Sun, Chenjun Jiang, Qianqian Liu, Na Wang, Runchun Huang, Gengchen Jiang, Yuxuan Yang
          </td>
          <td>2024-06-12</td>
          <td>Frontiers in Endocrinology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="This study probes the role of Heme Oxygenase-1 (HO-1) and its nuclear interactors in prostate cancer (PCa). Previous results, demonstrated the anti-tumoral role of HO-1 in PCa and its nuclear translocation, hinting at its potential regulatory roles within the nucleus. Thus, we postulate that HO-1, alongside its interactors, can reprogram PCa cells to a less aggressive phenotype. From the 11 HO-1 nuclear interacting proteins previously identified, 4 were transcription factors Using ChIP Atlas we evaluated the binding affinity of these factors to their downstream gene targets. Gene ontology and KEGG pathway analyses were then performed to associate these targets with specific biological processes and pathologies. Additionally, RNA-seq data from patient-derived xenografts (PDXs) at MD Anderson Cancer Center (MDA PCa PDXs) were analyzed to evaluate the clinical relevance of these genes in the context of PCa heterogeneity. These PDXs offer a clinically relevant model due to their representation of the disease's heterogeneity. We scrutinized the expression of the gene clusters of interest in PDXs, including their association with neuroendocrine (NE) phenotypes. The transcription factors among the HO-1 interactors previously described, were found to bind genes enriched in processes related to histone modification, acetylation, and transcription termination, implicating their role in gene expression regulation. KEGG pathways linked these genes to neurodegenerative diseases (including Parkinson's, Huntington's, and Alzheimer's disease), mirroring the neuroendocrine (NE) phenotype in advanced PCa, which typically emerges in advanced stages and is characterized by the transdifferentiating of epithelial cells into NE cells lacking androgen receptor expression. The unsupervised clustering of gene expression from PDXs showed that high expression of these genes is associated with an AR-negative NE histopathology, suggesting a potential for these interactors in prostate cancer subtyping. HO-1 nuclear interactors, especially transcription factors, appear to influence gene transcription associated with NE differentiation and cancer aggressiveness. The therapeutic relevance of modulating HO-1's interaction with its nuclear partners warrants further investigation to ascertain its role in mitigating the NE of PCa.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/199ab73cb74d73d84b3bf203a09f7e7a339b38e7" target='_blank'>
              Deciphering the Nuclear Role of Heme Oxygenase-1 in Prostate Cancer: Transcriptional Reprogramming and Neuroendocrine Differentiation
              </a>
            </td>
          <td>
            R. Seniuk, Agustina Ayelen Sabater, Gastón Pascual, P. Sanchis, Julia Lechuga, J. Cotignola, N. Anselmino, E. Vazquez, E. Labanca, A. Toro
          </td>
          <td>2024-07-01</td>
          <td>JCO Global Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a299a59bda6de4c1c2ea84f852c64d2057c80e3e" target='_blank'>
              Mapping alternative splicing events in colorectal cancer
              </a>
            </td>
          <td>
            Yifeng Zheng, Guoqiang Zhong, Qiuyu Song, Haonan Zhang, Shanping Wang, Chuangzhen Lin, Chengcheng He, Mingsong Li
          </td>
          <td>2024-07-15</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="

Long non-coding RNAs (lncRNAs), characterized by their length exceeding 200 nucleotides
and lack of protein-coding capacity, are intricately associated with a wide array of cellular
processes, encompassing cell invasion, differentiation, proliferation, migration, apoptosis,
and regeneration. Perturbations in lncRNA expression have been observed in numerous diseases
and have emerged as pivotal players in the pathogenesis of diverse tumor types. Glioblastoma, a
highly malignant primary tumor of the central nervous system (CNS), remains a formidable
challenge even with the advent of novel therapeutic interventions, as primary glioblastomas invariably
exhibit therapy resistance and aggressive behavior. Glioblastomas can arise from progenitor
cells or neuroglial stem cells, revealing profound cellular heterogeneity, notably in the
form of glioblastoma stem cells (GSCs) possessing stem-like properties. Glioblastomas comprise
neural precursors that harbor essential characteristics of neural stem cells (NSCs). Several
signaling pathways have been implicated in the regulation of self-renewal in both cancer cells
and stem cells. In addition to their involvement in therapy resistance and survival of glioblastoma,
lncRNAs are implicated in the modulation of GSC behaviors through diverse pathways and
the intricate regulation of various genes and proteins. This review aims to comprehensively discuss
the interplay between lncRNAs, their associated pathways, and GSCs, shedding light on
their potential implications in glioblastoma.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56eb445d57c7282292f6bf500980b87289b8c234" target='_blank'>
              Functional Roles of Long Non-coding RNAs on Stem Cell-related Pathways in Glioblastoma
              </a>
            </td>
          <td>
            Arya Moftakhar, S. E. Khoshnam, Maryam Farzaneh, Mahrokh Abouali Gale Dari
          </td>
          <td>2024-07-03</td>
          <td>Current Signal Transduction Therapy</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer is a heterogeneous disease that results from genetic alteration of cell cycle and proliferation controls. Identifying mutations that drive cancer, understanding cancer type specificities, and delineating how driver mutations interact with each other to establish disease is vital for identifying therapeutic vulnerabilities. Such cancer specific patterns and gene co-occurrences can be identified by studying tumor genome sequences, and networks have proven effective in uncovering relationships between sequences. We present two network-based approaches to identify driver gene patterns among tumor samples. The first approach relies on analysis using the Directed Weighted All Nearest Neighbors (DiWANN) model, which is a variant of sequence similarity network, and the second approach uses bipartite network analysis. A data reduction framework was implemented to extract the minimal relevant information for the sequence similarity network analysis, where a transformed reference sequence is generated for constructing the driver gene network. This data reduction process combined with the efficiency of the DiWANN network model, greatly lowered the computational cost (in terms of execution time and memory usage) of generating the networks enabling us to work at a much larger scale than previously possible. The DiWANN network helped us identify cancer types in which samples were more closely connected to each other suggesting they are less heterogeneous and potentially susceptible to a common drug. The bipartite network analysis provided insight into gene associations and co-occurrences. We identified genes that were broadly mutated in multiple cancer types and mutations exclusive to only a few. Additionally, weighted one-mode gene projections of the bipartite networks revealed a pattern of occurrence of driver genes in different cancers. Our study demonstrates that network-based approaches can be an effective tool in cancer genomics. The analysis identifies co-occurring and exclusive driver genes and mutations for specific cancer types, providing a better understanding of the driver genes that lead to tumor initiation and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26d7d425039cbe06da7fb2e443336941bd0e0ae5" target='_blank'>
              Network analysis of driver genes in human cancers
              </a>
            </td>
          <td>
            S. S. Patil, Steven A. Roberts, A. Gebremedhin
          </td>
          <td>2024-07-08</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="53BP1 plays a crucial role in regulating DNA damage repair pathway choice and checkpoint signaling in somatic cells; however, its role in meiosis has remained enigmatic. In this study, we demonstrate that the Caenorhabditis elegans ortholog of 53BP1, HSR-9, associates with chromatin in both proliferating and meiotic germ cells. Notably, HSR-9 is enriched on the X chromosome pair in pachytene oogenic germ cells. HSR-9 is also present at kinetochores during both mitotic and meiotic divisions but does not appear to be essential for monitoring microtubule-kinetochore attachments or tension. Using cytological markers of different steps in recombinational repair, we found that HSR-9 influences the processing of a subset of meiotic double strand breaks into COSA-1-marked crossovers. Additionally, HSR-9 plays a role in meiotic X chromosome segregation under conditions where X chromosomes fail to pair, synapse, and recombine. Together, these results highlight that chromatin-associated HSR-9 has both conserved and unique functions in the regulation of meiotic chromosome behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e700bc11eabfed38f5c48636a6b97e3c3470f7b" target='_blank'>
              The chromatin-associated 53BP1 ortholog, HSR-9, regulates recombinational repair and X chromosome segregation in the Caenorhabditis elegans germ line.
              </a>
            </td>
          <td>
            Qianyan Li, Sara Hariri, Aashna Calidas, Arshdeep Kaur, Erica Huey, J. Engebrecht
          </td>
          <td>2024-06-17</td>
          <td>Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Cancer is a complex and heterogeneous disease, in which a number of genetic and epigenetic changes occur in tumor onset and progression. Recent studies indicate that changes at the RNA level are also involved in tumorigenesis, such as adenosine-to-inosine (A-to-I) RNA editing. Here, we systematically investigate transcriptome-wide A-to-I editing events in a large number of samples from Non-Hodgkin lymphomas (NHLs). Using a computational pipeline that determines significant differences in editing level between NHL and normal samples at known A-to-I editing sites, we identify a number of differentially edited editing sites between NHL subtypes and normal samples. Most of the differentially edited sites are located in non-coding regions, and many such sites show a strong correlation between gene expression level and editing efficiency, indicating that RNA editing might have direct consequences for the cancer cell’s aberrant gene regulation status in these cases. Moreover, we establish a strong link between RNA editing and NHL by demonstrating that NHL and normal samples and even NHL subtypes can be distinguished based on genome-wide RNA editing profiles alone. Our study establishes a strong link between RNA editing, cancer and aberrant gene regulation in NHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3709fa38f5f1119b1fcb4db3f2e4cd77244ecc08" target='_blank'>
              A-to-I Editing Is Subtype-Specific in Non-Hodgkin Lymphomas
              </a>
            </td>
          <td>
            Cai Chen, Ralf Bundschuh
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/153ab2ed142f22e69713eea228d0914f5b69386b" target='_blank'>
              Telomere-to-telomere genome assembly of sorghum
              </a>
            </td>
          <td>
            Meng Li, Chunhai Chen, Haigang Wang, Huibin Qin, Sen Hou, Xukui Yang, Jianbo Jian, Peng Gao, Minxuan Liu, Z. Mu
          </td>
          <td>2024-08-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b0e09e628f8fcf9bb1ad3d19ff2d5112c171387" target='_blank'>
              Unmutated RRAS2 emerges as a key oncogene in post-partum-associated triple negative breast cancer
              </a>
            </td>
          <td>
            C. Cifuentes, Clara L. Oeste, Isabel Fernández-Pisonero, Alejandro Hortal, C. García-Macías, Jeanne Hochart, Regina Rubira, L. Horndler, Carlos Horndler, Xosé R. Bustelo, Balbino Alarcón
          </td>
          <td>2024-07-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Significance Cell division is a vital process in development. Errors in cell division can lead to disorders and cancer. At the heart of this process is the centrosome, a membrane-less organelle of hundreds of proteins working together to organize a bipolar spindle. It is unclear how proteins fit and function within the small confines of the centrosome. Here, we demonstrate that a crucial centrosomal protein, Cep57, self-assembles through multivalent interactions. Cep63 regulates Cep57 self-assembly. Mutations in Cep57 are linked to mosaic-variegated aneuploidy (MVA) syndrome. Our study indicates that MVA mutations impair Cep57’s ability to self-assemble. When blocking the self-assembly of Cep57 in cells, we observed abnormal structure and number of centrosomes. These findings contribute to the molecular understanding of centrosome-related diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05d570160ba78312308c376a884de0d944d23a13" target='_blank'>
              Cep57 regulates human centrosomes through multivalent interactions
              </a>
            </td>
          <td>
            Hung-Wei Yeh, Po-Pang Chen, Tzu-Chen Yeh, Shiou-Lan Lin, Yue-Ting Chen, Wan-Ping Lin, Ting Chen, Jia Meng Pang, Kai-Ti Lin, Lily Hui-Ching Wang, Yu-Chun Lin, Orion Shih, U‐Ser Jeng, Kuo-Chiang Hsia, Hui-Chun Cheng
          </td>
          <td>2024-06-10</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a widely used method for classifying cell types and states and revealing disease mechanisms. However, most contemporary scRNA-seq platforms fail to explore the multi-landscape of RNA. Here, we designed a microfluidic chip combined oligo-dT primers and Random Bridging Co-labeling (RBCL) RNA sequencing to develop an innovative Chigene scRNA-seq technology that can identify gene expression, mutations, and RNA splicing landscapes at the single-cell level. The Chigene scRNA-seq platform demonstrated exceptional performance, with minimal doublet rates of 0.94% (Chigene V1) and 1.93% (Chigene V2). Both versions exhibit high sensitivity, with Chigene V2 achieving nearly 100% RNA coverage and detecting over 1800 genes per cell on average. Targeted capture of single-cell gene mutations enhances mutation detection sensitivity. Moreover, this Chigene V2 platform has been validated in clinical samples for its ability to detect mutations, gene fusions and alternative splicing. The reliability of the platform was further corroborated using known functional gene mutation (CDKN1A) and fusion (FGFR3-TACC). To validate this method’s potential for discovering novel gene mutations in clinical samples, our investigation revealed an intriguing cell subpopulation carrying an ARHGAP5 mutation in urothelial carcinoma. These cells exhibited high-frequency mRNA splicing and exhibited specific crosstalk with T cells, distinguishing them from the subpopulation with the ARHGAP5 wild-type phenotype. Overall, this method provides a robust scRNA-seq platform suitable for comprehensive analyses of clinical specimens at different genetic information levels, thereby offering significant potential in the discovery of novel genes and interactions at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be600ce14b879188e7c67b73a02d6fde03c11a8c" target='_blank'>
              Identification of Multi-landscape and Cell Interactions in the Tumor Microenvironment through High-Coverage Single-Cell Sequencing
              </a>
            </td>
          <td>
            Wenlong Zhong, Ligang Wang, Tonglei Guo, Lihua Zhao, Daqin Wu, Fei Xie, Xiao Wang, Xiuxin Li, FangXiao Wang, Weiyue Gu, Tianxin Lin, Xu Chen
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long non-coding RNAs (lncRNAs), defined as RNA molecules exceeding 200 nucleotides in length, have been implicated in the regulation of various biological processes and the progression of tumors. Among them, LINC00518, a recently identified lncRNA encoded by a gene located on chromosome 6p24.3, consists of three exons and is predicted to positively regulate the expression of specific genes. LINC00518 has emerged as a key oncogenic lncRNA in multiple cancer types. It exerts its tumor-promoting effects by modulating the expression of several target genes, primarily through acting as a sponge for microRNAs (miRNAs). Additionally, LINC00518 influences critical signaling pathways, including the Wnt/β-catenin, JAK/STAT, and integrin β3/FAK pathways. Elevated levels of LINC00518 in tumor tissues are associated with increased tumor size, advanced clinical stage, metastasis, and poor survival prognosis. This review provides a comprehensive summary of the genetic characteristics, expression patterns, biological functions, and underlying mechanisms of LINC00518 in human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d93f220783bc0f4834eae797d7fdcb653821aa1b" target='_blank'>
              LINC00518: a key player in tumor progression and clinical outcomes
              </a>
            </td>
          <td>
            Qiang Yi, Gangfeng Zhu, Weijian Zhu, Jiaqi Wang, Xinting Ouyang, Kuan Yang, Jinghua Zhong
          </td>
          <td>2024-07-23</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/909f268f96d508b05ce576d1aa806996848a4469" target='_blank'>
              Beyond traditional translation: ncRNA derived peptides as modulators of tumor behaviors
              </a>
            </td>
          <td>
            Kang Wen, Xin Chen, Jingyao Gu, Zhenyao Chen, Zhaoxia Wang
          </td>
          <td>2024-06-14</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="The p53 genome is a gene that has truly taken the scientific world by storm. Once referred to as an oncogene, p53 has proved itself to be more complex and dubious time and time again. This paper will talk about the beginning doubts and queries revolving around p53 and the discoveries that led to it being referred to as a tumour suppressor gene, rather than being mislabeled as an oncogene. It will touch upon the importance that p53 brings to our daily lives such as repairing the DNA, maintaining the cell cycle and starting apoptosis, as well as highlight the differences between the gene’s two types: wild-type p53 and mutant p53. Furthermore, it will talk about how a slight mutation in p53, can cause it to form many protein isoforms, a leading contributing factor in cancers. It will also mention the effects post-translational modification will have on the two types of p53. In addition to this, it will delve into the negative and potentially harmful effects that p53 could have on our bodies. Finally, it will also talk about p53-mediated therapy and targeting mutant p53 with autophagy, which are 2 processes that have shown successful outcomes in reducing the complications caused by mutant p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dc70ab96424bdf601f1a4ba7f9b8f2ff8a5b550" target='_blank'>
              P53: Our Saviour and Our Destroyer
              </a>
            </td>
          <td>
            Narayani Haldar
          </td>
          <td>2024-07-23</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The histone H3K27 demethylase KDM6A is a tumor suppressor in multiple cancers, including multiple myeloma (MM). We created isogenic MM cells disrupted for KDM6A and tagged the endogenous protein to facilitate genome wide studies. KDM6A binds genes associated with immune recognition and cytokine signaling. Most importantly, KDM6A binds and activates NLRC5 and CIITA encoding regulators of Major Histocompatibility Complex (MHC) genes. Patient data indicate that NLRC5 and CIITA, are downregulated in MM with low KDM6A expression. Chromatin analysis shows that KDM6A binds poised and active enhancers and KDM6A loss led to decreased H3K27ac at enhancers, increased H3K27me3 levels in body of genes bound by KDM6A and decreased gene expression. Reestablishing histone acetylation with an HDAC3 inhibitor leads to upregulation of MHC expression, offering a strategy to restore immunogenicity of KDM6A deficient tumors. Loss of Kdm6a in murine RAS-transformed fibroblasts led to increased growth in vivo associated with decreased T cell infiltration.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b45cb2ef1a013d1559a31e46d5a533af85985cf" target='_blank'>
              KDM6A Regulates Immune Response Genes in Multiple Myeloma.
              </a>
            </td>
          <td>
            Daphné Dupéré-Richer, Alberto Riva, B. Barwick, Sayantan Maji, Heidi L Casellas Roman, Jianping Li, Umasankar De, Amin Sobh, Gabrielle Quickstad, Crissandra Piper, M. Kulis, T. Ezponda, J. Martín-Subero, Giovanni Tonon, Weizhou Zhang, C. Mitsiades, L. Boise, Richard L Bennett, Jonathan D. Licht
          </td>
          <td>2024-07-24</td>
          <td>Blood</td>
          <td>0</td>
          <td>91</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54a1de3c94a3135d7ef2bd2539530b5e0e7ee5f8" target='_blank'>
              DNA mismatch and damage patterns revealed by single-molecule sequencing.
              </a>
            </td>
          <td>
            Mei Hong Liu, Benjamin M Costa, Emilia C Bianchini, Una Choi, Rachel C. Bandler, E. Lassen, Marta Grońska-Pęski, Adam Schwing, Z. R. Murphy, Daniel Rosenkjær, Shany Picciotto, V. Bianchi, L. Stengs, M. Edwards, N. Nunes, Caitlin A. Loh, Tina K Truong, Randall E Brand, Tomi Pastinen, J. R. Wagner, A. Skytte, U. Tabori, J. Shoag, G. Evrony
          </td>
          <td>2024-06-12</td>
          <td>Nature</td>
          <td>0</td>
          <td>72</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the most frequent and mortality-causing neoplasia, with various distributions between populations. Strong hereditary predispositions are the causatives of a small percentage of CRC, and most cases have no transparent genetic background. This is a vast arena for exploring cancer low-susceptibility genetic variants. Nonetheless, the research that has been conducted to date has failed to deliver consistent conclusions and often features conflicting messages, causing chaos in this field. Therefore, we decided to organize the existing knowledge on this topic. We screened the PubMed and Google Scholar databases. We drew up markers by gene locus gathered by hallmark: oncogenes, tumor suppressor genes, genes involved in DNA damage repair, genes involved in metabolic pathways, genes involved in methylation, genes that modify the colonic microenvironment, and genes involved in the immune response. Low-penetration genetic variants increasing the risk of cancer are often population-specific, hence the urgent need for large-scale testing. Such endeavors can be successful only when financial decision-makers are united with social educators, medical specialists, genetic consultants, and the scientific community. Countries’ policies should prioritize research on this subject regardless of cost because it is the best investment. In this review, we listed potential low-penetrance CRC susceptibility alleles whose role remains to be established.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0541d34727973134a35fb48d39ef83a36856b18" target='_blank'>
              Low-Penetrance Susceptibility Variants in Colorectal Cancer—Current Outlook in the Field
              </a>
            </td>
          <td>
            Marcin Szuman, M. Kaczmarek-Ryś, S. Hryhorowicz, Alicja Kryszczyńska, Natalia Grot, Andrzej Pławski
          </td>
          <td>2024-07-30</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Non-coding RNAs are mainly divided into two categories, one is small non-coding RNA represented by miRNA, and the other is long non-coding RNA longer than 200 bp. Further studies on non-coding RNAs have revealed that long non-coding RNAs not only have carcinogenic effects, but also have potential links with miRNAs. Antisense non-coding RNA in the INK4 locus (ANRIL/CDKN2B-AS1), one of the five subtypes of long non-coding RNA, has been proved to play a role of oncogene in many cancers, such as gastric cancer, cervical cancer, prostate cancer and non-small cell lung cancer. Knockdown ANRIL can significantly inhibit the proliferation and migration of cancer cells, while also negatively regulating the expression of related miRNAs. This suggests that ANRIL may serve as a potential target for the development of drugs that provide new strategies to improve the effectiveness of cancer treatment. In our review, we summarize the current association between ANRIL and miRNAs in various cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f81316c09c1ab67e458fd0abb991abce3cd6053f" target='_blank'>
              Potential Links between ANRIL and MiRNAs in Various Cancers.
              </a>
            </td>
          <td>
            Xiaoyan Yang, Liushan Wei, Shijie Wu, Zhizhong Xie, Xiaoyong Lei
          </td>
          <td>2024-06-07</td>
          <td>Combinatorial chemistry & high throughput screening</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="
 The WRN helicase serves as a key target in the treatment of cancers characterized by microsatellite instability (MSI) because it plays a crucial role in resolving harmful non-canonical DNA structures that arise in cells with defective mismatch repair systems. Despite the critical functions of human DNA and RNA helicases, no drugs targeting these enzymes have been approved, largely due to the difficulties in identifying potent and selective inhibitors. In this study, we present the chemoproteomics-based identification of a clinical-stage, covalent allosteric inhibitor of WRN, VVD-133214. This inhibitor specifically interacts with a cysteine residue (C727) within the helicase domain, which undergoes inter-domain movement during the DNA unwinding process. VVD-133214 reacts with the WRN protein in a nucleotide cooperative manner, promoting stable, compact structures that impede the enzyme's dynamic flexibility essential for its helicase activity. Inhibition of WRN by VVD-133214 leads to extensive double-stranded DNA breaks, nuclear enlargement, and ultimately cell death, specifically in MSI-high cells but not in microsatellite stable cells. The inhibitor demonstrated good tolerance in mice and significant tumor reduction in various MSI-high colorectal cancer cell lines and patient-derived xenograft models. Our findings highlight an allosteric strategy to inhibit WRN function that avoids interference from the endogenous ATP cofactor in cancer cells and positions VVD-133214 as a promising therapeutic candidate for patients with MSI-high cancers.
 Citation Format: Matthew P. Patricelli, Kristen A. Baltgalvis, Kelsey N. Lamb, Kent T. Symons, Chu-Chiao Wu, Melissa A. Hoffman, Aaron N. Snead, Xiaodan Song, Thomas Glaza, Shota Kikuchi, Jason C. Green, Donald C. Rogness, Betty Lam, Maria E. Rodriguez-Aguirre, David R. Woody, Christie L. Eissler, Socorro Rodilles, Seth M .Negron, Steffen M. Bernard, Eileen Tran, Jonathan Pollock, Ali Tabatabaei, Victor Contreras, Heather N Williams, Martha K. Pastuszka, John J. Sigler, Piergiorgio Pettazzoni, Markus G. Rudolph, Moritz Classen, Doris Brugger, Christopher Claiborne, Jean-Marc Plancher, Isabel Cuartas, Joan Seoane, Laurence E. Burgess, Robert T. Abraham, David S. Weinstein, Gabriel M. Simon, Todd M. Kinsella. Chemoproteomic discovery of a covalent allosteric inhibitor of WRN helicase [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Expanding and Translating Cancer Synthetic Vulnerabilities; 2024 Jun 10-13; Montreal, Quebec, Canada. Philadelphia (PA): AACR; Mol Cancer Ther 2024;23(6 Suppl):Abstract nr IA003.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4a12df2fc1662921ad0471a69787015bbab12ceb" target='_blank'>
              Abstract IA003: Chemoproteomic discovery of a covalent allosteric inhibitor of WRN helicase
              </a>
            </td>
          <td>
            Matt Patricelli, K. Baltgalvis, Kelsey N. Lamb, Kent T. Symons, Chu-Chiao Wu, Melissa A. Hoffman, Aaron N. Snead, Xiaodan Song, Thomas Glaza, S. Kikuchi, Jason C. Green, Donald C. Rogness, Betty Lam, Maria E Rodriguez-Aguirre, David R. Woody, Christie L. Eissler, Socorro Rodilles, Seth M Negron, Steffen M. Bernard, Eileen Tran, Jonathan Pollock, Ali Tabatabaei, Victor Contreras, Heather N Williams, Martha K. Pastuszka, John J. Sigler, Piergiorgio Pettazzoni, Markus G Rudolph, Moritz Classen, Doris Brugger, Christopher Claiborne, Jean‐Marc Plancher, I. Cuartas, Joan Seoane, Laurence E. Burgess, Robert T. Abraham, David S. Weinstein, Gabe Simon, Todd M Kinsella
          </td>
          <td>2024-06-10</td>
          <td>Molecular Cancer Therapeutics</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Clonorchis sinensis is a significant zoonotic food-borne parasite that causes a range of hepatobiliary diseases, which in severe cases can even lead to cholangiocarcinoma. To explore new diagnostic and treatment strategies, the dynamic RNA regulatory processes across different developmental stages of C. sinensis were analyzed by using whole-transcriptome sequencing. The chromosomal-level genome of C. sinensis was used for sequence alignment and annotation. In this study, we identified a total of 59,103 RNAs in the whole genome, including 2,384 miRNAs, 25,459 mRNAs, 27,564 lncRNAs and 3,696 circRNAs. Differential expression analysis identified 6,556 differentially expressed mRNAs, 2,231 lncRNAs, 877 miRNAs and 20 circRNAs at different developmental stages. Functional enrichment analysis highlighted the critical role of microtubule-related biological processes in the growth and development of C. sinensis. And coexpression analysis revealed 97 lncRNAs and 85 circRNAs that were coexpressed with 42 differentially expressed mRNAs that associated with microtubules at different developmental stages of C. sinensis. The expression of the microtubule-related genes dynein light chain 2 (DLC2) and dynein light chain 4 (DLC4) increased with C. sinensis development, and DLC2/4 could be inhibited by albendazole. Finally, by constructing competing endogenous RNA (ceRNA) networks, the lncRNA-miRNA-mRNA and circRNA-miRNA-mRNA regulatory relationships were constructed, and the ceRNA networks of MSTRG.14258.5-novel_miR_2287-newGene_28215 and MSTRG.14258.5-novel_miR_2216-CSKR_109340 were verified. This study suggests, through whole transcriptome sequencing, that the context of microtubule regulation may play an essential role in the development and growth of C. sinensis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61e43651eb5f05b8bcba907e89576a47ff260ba6" target='_blank'>
              Revealing the dynamic whole transcriptome landscape of Clonorchis sinensis: Insights into the regulatory roles of noncoding RNAs and microtubule-related genes in development
              </a>
            </td>
          <td>
            Yangyuan Qiu, Cunzhou Wang, Jing Wang, Qingbo L V, Lulu Sun, Yaming Yang, Mingyuan Liu, Xiaolei Liu, Chen Li, Bin Tang
          </td>
          <td>2024-07-01</td>
          <td>PLOS Neglected Tropical Diseases</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16eda9843eb0037407162ede7ade3db9a52371a7" target='_blank'>
              SNM1A is crucial for efficient repair of complex DNA breaks in human cells
              </a>
            </td>
          <td>
            L. P. Swift, B. C. Lagerholm, Lucy R. Henderson, Malitha Ratnaweera, H. T. Baddock, B. Sengerova, Sook Y. Lee, A. Cruz-Migoni, Dominic Waithe, Christian Renz, Helle D. Ulrich, Joseph A. Newman, Christopher J. Schofield, P. McHugh
          </td>
          <td>2024-06-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/321126b0cb5f073f8273850a71ea2ffc1df84ce8" target='_blank'>
              Deciphering the role of post-translational modifications in fanconi anemia proteins and their influence on tumorigenesis.
              </a>
            </td>
          <td>
            Rui Ma, Xinlin Xu
          </td>
          <td>2024-06-15</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Fas-activated serine/threonine kinase (FASTK) is the founding member of the FASTKD protein family, which was shown to regulate the fate of mRNA molecules on multiple levels. The mitochondrial variant of FASTK co-localizes with mitochondrial RNA granules and regulates degradation of mitochondrial mRNAs, whereas the cytoplasmic and nuclear forms of FAST are involved in regulation of alternative splicing, cytoplasmic RNA granule formation and mRNA translation. Despite these multiple roles of FASTK in mRNA biology, the exact rules of RNA recognition by this protein remained undetermined. Here, we demonstrate direct RNA binding by purified human FASTK and show its preference for single-stranded G-rich sites and RNA G-quadruplexes. Addition of FASTK alone was sufficient to achieve protection of mitochondrial mRNAs from degradation by the degradosome. Structural characterization by SAXS showed that FASTK in solution is a monomer with an extended conformation. Point mutagenesis studies supported the structural predictions of an exposed RNA-binding interface in the central helical region, preceded by a smaller, flexibly attached, helical N-terminal domain. We provide the first such extensive in vitro characterization of the RNA binding properties for a representative of the FASTKD protein family, and suggest how these intrinsic properties may underly FASTK function in mRNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/af8500234e8aa7d2ce11de44514f8654502ccbed" target='_blank'>
              Human FASTK preferentially binds single-stranded and G-rich RNA
              </a>
            </td>
          <td>
            Daria M. Dawidziak, Dawid A. Dzadz, Mikołaj Kuska, Madhuri Kanavalli, M. M. Klimecka, M. Merski, Katarzyna J. Bandyra, MW Górna
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Telomeres, nucleoprotein complexes protecting the ends of linear chromosomes, are essential for ensuring the stability of eukaryotic genomes. As telomeres generally consist of repetitive DNA associated with specifically bound proteins, the evolution of telomeric repeat motifs hardly occurs. However, a recent study identified nematodes with telomeric repeats other than the canonical TTAGGC motif. Here, we investigated how telomere repeats were able to evolve despite the challenge posed by the specificity of telomere-binding proteins to the telomeric DNA in Nematoda, focusing on CEH-37. Initially, we assessed the conservation of telomere-binding proteins across diverse nematode species and obtained the well-conserved telomere-binding protein CEH-37 from each species with different telomeric repeats. The binding affinities of the proteins, measured by electrophoresis mobility shift assays, revealed that CEH-37 exhibits limited specificity, which cannot distinguish telomeric repeats found in nematodes except for TTAGGG motif. Furthermore, through the analysis of TBP conservation and the structure of CEH-37 in nematodes, we inferred that CEH-37 functioned as a double-stranded telomeric DNA binding protein before TEBP proteins emerged. Through this study, we suggest that telomeric sequences in Nematoda were able to evolve within the range of that does not exceed the sequence-specificity of telomere-binding proteins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10b57f4aa97a7b104fa1ba06966126b4273bc126" target='_blank'>
              Robustness with Limited Specificity of Telomere Binding Proteins Underlies Telomere Evolution in Nematoda
              </a>
            </td>
          <td>
            Hobum Song, Seon Hong Kim, Daisy S. Lim, Hee-Jung Choi, Junho Lee
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135134a62bac76cb46dae2f04090c849f2c01172" target='_blank'>
              Targeting chromosomal instability in patients with cancer.
              </a>
            </td>
          <td>
            Duaa H Al-Rawi, Emanuele Lettera, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-11</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="PBX1 is a transcription factor that can promote the occurrence of various tumors and play a reg-ulatory role in tumor growth, metastasis, invasion, and drug resistance. Furthermore, a variant generated by fusion of E2A and PBX1, E2A-PBX1, has been found in 25% of patients with childhood acute lymphoblastic leukemia. Thus, PBX1 is a potential therapeutic target for many cancers. Here, we describe the structure of PBX1 and E2A-PBX1 as well as the molecular mecha-nisms whereby these proteins promote tumorigenesis to provide future research directions for developing new treatments. We show that PBX1 and E2A-PBX1 induce the development of highly malignant and difficult-to-treat solid and blood tumors. The development of specific drugs against their targets may be a good therapeutic strategy for PBX1-related cancers. Furthermore, we strongly recommend E2A-PBX1 as one of the genes for prenatal screening to reduce the incidence of childhood hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bdaefcf70fff41c7377eae29f3343a4ba69aba0a" target='_blank'>
              Comprehensive summary: the role of PBX1 in development and cancers
              </a>
            </td>
          <td>
            Mingsheng Liu, Yan Xing, Jiufeng Tan, Xiaoliang Chen, Yaming Xue, Licheng Qu, Jianchao Ma, Xuefei Jin
          </td>
          <td>2024-07-26</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [23],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>