<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-12 06:29:11 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>20</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>50</td>
          <td>130</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>98</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, Jens Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>271</td>
          <td>130</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>233</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, V. Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a form of oncogene amplification found across cancer types and associated with poor outcome in patients. ecDNA can be structurally complex and contain rearranged DNA sequences derived from multiple chromosome locations. As the structure of ecDNA can impact oncogene regulation and may indicate mechanisms of its formation, disentangling it at high resolution from sequencing data is essential. Even though methods have been developed to identify and reconstruct ecDNA in cancer genome sequencing, it remains challenging to resolve complex ecDNA structures, in particular amplicons with shared genomic footprints. We here introduce Decoil, a computational method which combines a breakpoint-graph approach with regression to reconstruct complex ecDNA and deconvolve co-occurring ecDNA elements with overlapping genomic footprints from long-read nanopore sequencing. Decoil outperforms de novo assembly and alignment-based methods in simulated long-read sequencing data for both simple and complex ecDNAs. Applying Decoil on whole genome sequencing data uncovered different ecDNA topologies and explored ecDNA structure heterogeneity in neuroblastoma tumors and cell lines, indicating that this method may improve ecDNA structural analyzes in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c914721ddd777cd78df6632d6860e72cff344be" target='_blank'>
              Reconstructing extrachromosomal DNA structural heterogeneity from long-read sequencing data using Decoil.
              </a>
            </td>
          <td>
            M. Giurgiu, Nadine Wittstruck, Elias Rodriguez-Fos, R. Chamorro González, Lotte Brueckner, Annabell Krienelke-Szymansky, K. Helmsauer, Anne Hartebrodt, P. Euskirchen, R. Koche, Kerstin Haase, Knut Reinert, A. Henssen
          </td>
          <td>2024-08-07</td>
          <td>Genome research</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5988602f52499713b4d1d42d555664a971cb3e9" target='_blank'>
              3D genomic analysis reveals novel enhancer-hijacking caused by complex structural alterations that drive oncogene overexpression
              </a>
            </td>
          <td>
            Katelyn L. Mortenson, Courtney Dawes, Emily R. Wilson, Nathan E. Patchen, Hailey E Johnson, Jason Gertz, Swneke D. Bailey, Yang Liu, K. Varley, Xiaoyang Zhang
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA), an extrachromosomal circular structured DNA, is extensively found in eukaryotes. Exploring eccDNA at the single-cell level contributes to understanding heterogeneity, evolution, development, and specific functions within cells. Nevertheless, the high-throughput identification methods for single-cell eccDNA are complex, and currently mature and widely applicable technologies are lacking. Those factors have led to a scarcity of resources for studying eccDNA at the single-cell level. Therefore, using available single-cell whole-genome sequencing (WGS) data, we have constructed a comprehensive eccDNA database named scEccDNAdb (https://lcbb.swjtu.edu.cn/scEccDNAdb/). Presently, scEccDNAdb comprises 3,195,464 single-cell eccDNA entries from both disease/health human and mouse samples, which provides comprehensive annotations including oncogenes, typical enhancers, super-enhancers, CTCF binding sites, SNPs, chromatin accessibility, eQTLs, transcription factor binding sites, motifs, and SVs. Additionally, it provides nine online analysis and visualization tools, facilitating the generation of publication-quality figures for eccDNA analysis through the upload of customized files. Overall, scEccDNAdb represents the first comprehensive database known to us for exploring and analyzing single-cell eccDNA data in diverse cell types, tissues, and species. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322c06c5550a609619dd96de2cba0b3494554b2a" target='_blank'>
              scEccDNAdb: an integrated single-cell eccDNA resource for human and mouse
              </a>
            </td>
          <td>
            Wenqing Wang, Xinyu Zhao, Tianyu Ma, Tengwei Zhong, Junnuo Zheng, Yi Yang, Yuanbiao Guo, Zhiyun Guo
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is a pervasive driver event in cancer, yet our understanding of how ecDNA forms is limited. Here, we couple a CRISPR-based method for ecDNA induction with extensive characterization of newly formed ecDNA to examine their biogenesis. We find that DNA circularization is efficient, irrespective of 3D genome context, with formation of 800kb, 1 Mb, and 1.8 Mb ecDNAs reaching or exceeding 15%. We show non-homologous end joining and microhomology-mediated end joining both contribute to ecDNA formation, while inhibition of DNA-PKcs and ATM have opposing impacts on ecDNA formation. EcDNA and the corresponding chromosomal excision scar can form at significantly different rates and respond differently to DNA-PKcs and ATM inhibition. Taken together, our results support a model of ecDNA formation in which double strand break ends dissociate from their legitimate ligation partners prior to joining of illegitimate ends to form the ecDNA and excision scar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e488ec3b1ac40e7e2986571eab100d1986b6e3f" target='_blank'>
              Disparate pathways for extrachromosomal DNA biogenesis and genomic DNA repair.
              </a>
            </td>
          <td>
            John C. Rose, J. Belk, I. Wong, Jens Luebeck, Hudson T Horn, Bence Daniel, Matthew G Jones, Kathryn E Yost, King L. Hung, Kevin S Kolahi, Ellis J Curtis, Calvin J Kuo, V. Bafna, Paul S. Mischel, Howard Y. Chang
          </td>
          <td>2024-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Recent advances in high-resolution mapping of spatial interactions among regulatory elements support the existence of complex topological assemblies of enhancers and promoters known as enhancer-promoter hubs or cliques. Yet, organization principles of these multi-interacting enhancer-promoter hubs and their potential role in regulating gene expression in cancer remains unclear. Here, we systematically identified enhancer-promoter hubs in breast cancer, lymphoma, and leukemia. We found that highly interacting enhancer-promoter hubs form at key oncogenes and lineage-associated transcription factors potentially promoting oncogenesis of these diverse cancer types. Genomic and optical mapping of interactions among enhancer and promoter elements further showed that topological alterations in hubs coincide with transcriptional changes underlying acquired resistance to targeted therapy in T cell leukemia and B cell lymphoma. Together, our findings suggest that enhancer-promoter hubs are dynamic and heterogeneous topological assemblies with the potential to control gene expression circuits promoting oncogenesis and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9831a80afed1f71511cdd9f1e5ce45eefd2262c2" target='_blank'>
              Enhancer-promoter hubs organize transcriptional networks promoting oncogenesis and drug resistance
              </a>
            </td>
          <td>
            Brent S. Perlman, Noah G Burget, Yeqiao Zhou, Gregory W. Schwartz, Jelena Petrovic, Z. Modrusan, R. Faryabi
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Somatic structural variations (SVs) in cancer can shuffle DNA content in the genome, relocate regulatory elements, and alter genome organization. Enhancer hijacking occurs when SVs relocate distal enhancers to activate proto-oncogenes. However, most enhancer hijacking studies have only focused on protein-coding genes. Here, we develop a computational algorithm 'HYENA' to identify candidate oncogenes (both protein-coding and non-coding) activated by enhancer hijacking based on tumor whole-genome and transcriptome sequencing data. HYENA detects genes whose elevated expression is associated with somatic SVs by using a rank-based regression model. We systematically analyze 1146 tumors across 25 types of adult tumors and identify a total of 108 candidate oncogenes including many non-coding genes. A long non-coding RNA TOB1-AS1 is activated by various types of SVs in 10% of pancreatic cancers through altered 3-dimensional genome structure. We find that high expression of TOB1-AS1 can promote cell invasion and metastasis. Our study highlights the contribution of genetic alterations in non-coding regions to tumorigenesis and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fb0cf8831bea2891dbca4714836cde381f2ca0" target='_blank'>
              HYENA detects oncogenes activated by distal enhancers in cancer.
              </a>
            </td>
          <td>
            Anqi Yu, Ali E. Yesilkanal, Ashish Thakur, Fan Wang, Yang Yang, William Phillips, Xiaoyang Wu, Alexander Muir, Xin He, Francois Spitz, Lixing Yang
          </td>
          <td>2024-07-25</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dec3a33e2b0139c6a02f10afa4c9e80abbaad7a" target='_blank'>
              Extrachromosomal circular DNA promotes prostate cancer progression through the FAM84B/CDKN1B/MYC/WWP1 axis
              </a>
            </td>
          <td>
            Wei Jin, Zhenqun Xu, Yan Song, Fangjie Chen
          </td>
          <td>2024-07-12</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="INTRODUCTION
Isodicentric Y chromosomes are relatively common structural variants of the human genome. The underlying mechanism of isodicentric Y chromosomes with short arm breakpoints [idic(Yq)] remains to be clarified.


CASE PRESENTATION
We encountered a Japanese man with azoospermia and mild short stature. G-banding and array-based comparative genomic hybridization indicated that his karyotype was 45,X/46,X,idic(Y)(qter→p11.32::p11.32→qter) with a ~1.8 Mb terminal deletion. Whole genome sequencing suggested that the Y chromosome had four breakpoints in a ~7 kb region of the pseudoautosomal region 1 (PAR1).


CONCLUSION
This case was assumed to have an idic(Yq) resulting from multiple DNA double-strand breaks in PAR1. This rearrangement may have been facilitated by the PAR1-specific chromatin architecture. The clinical features of the patient can be ascribed to SHOX haploinsufficiency and the presence of a 45,X cell line, although copy-number gains of some Yq genes and the size reduction of PAR1 may also contribute to his spermatogenic failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6ffa4553f26785a976ed64f6e1aecf9e157203" target='_blank'>
              Isodicentric Y chromosome with multiple breakpoints in the pseudoautosomal region 1.
              </a>
            </td>
          <td>
            Yasuko Ogiwara, Y. Kobori, E. Suzuki, Atsushi Hattori, Kanako Tanase-Nakao, Akiyoshi Osaka, Toshiyuki Iwahata, Hiroshi Okada, Y. Kuroki, M. Fukami
          </td>
          <td>2024-07-29</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a040a0cb95cb5f3026b82c4853a3fbf7dc09e9f" target='_blank'>
              Structure and transcription of integrated HPV DNA in vulvar carcinomas
              </a>
            </td>
          <td>
            A. Van Arsdale, Lauren Turker, Yoke-Chen Chang, Joshua Gould, Bryan Harmon, E. Maggi, Olga Meshcheryakova, Maxwell P Brown, Dana Luong, Koenraad Van Doorslaer, Mark H. Einstein, Dennis Y. S. Kuo, Deyou Zheng, Brian J. Haas, Jack Lenz, Cristina Montagna
          </td>
          <td>2024-06-19</td>
          <td>NPJ Genomic Medicine</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Transcriptional enhancers can regulate individual or multiple genes through long-range three-dimensional (3D) genome interactions, and these interactions are commonly altered in cancer. Yet, the functional relationship between changes in 3D interactions associated with regulatory regions and differential gene expression appears context-dependent. In this study, we used HiChiP to capture changes in 3D genome interactions between active regulatory regions of endometrial cancer cells in response to estrogen treatment and uncovered significant differential long-range interactions that are strongly enriched for estrogen receptor α (ER) bound sites (ERBS). The ERBS anchoring differential loops with either a gene’s promoter or distal regions were correlated with larger transcriptional responses to estrogen compared to ERBS not involved in differential interactions. To functionally test this observation, CRISPR-based Enhancer-i was used to deactivate specific ERBS, which revealed a wide range of effects on the transcriptional response to estrogen. However, these effects are only subtly and not significantly stronger for ERBS in differential loops. In addition, we observed an enrichment of 3D interactions between the promoters of estrogen up-regulated genes and found that looped promoters can work together cooperatively. Overall, our work suggests that changes in 3D genome structure upon estrogen treatment identify some functionally important regulatory regions; however, these changes aren’t required for a transcriptional response to E2 in endometrial cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09d0d32b1b8568f5b7b8778716b0bae7e2d53785" target='_blank'>
              Estrogen-induced chromatin looping changes identify a subset of functional regulatory elements
              </a>
            </td>
          <td>
            Hosiana Abewe, Alexandra Richey, Jeffery M. Vahrenkamp, Matthew Ginley-Hidinger, Craig M Rush, Noel Kitchen, Xiaoyang Zhang, Jason Gertz
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Breast cancer (BCa) is a major global health challenge, characterized by chromosomal instability (CIN) and subsequent acquisition of extensive somatic copy number alterations (CNAs). CNAs including amplifications and deletions, significantly influence intra-tumor heterogeneity and the tumor microenvironment (TME). Among these, the loss of chromosome 13q14.2 emerges as a considerable factor in BCa pathogenesis and treatment responses. We provide evidence that this genomic alteration is under positive selective pressure, correlating with poorer patient survival. Furthermore, through multi-omic and in vitro analyses, we uncover a dual role of 13q14.2 loss: it confers a survival advantage to tumor cells and modulate the cell cycle and pro-apoptotic pathways in cancer cells, affecting macrophages population in the TME, while paradoxically increasing tumor susceptibility to BCL2 inhibitors. These findings suggest that targeting 13q14.2 as a biomarker in BCa could enhance the efficacy of existing treatments and offer a new avenue for improving clinical outcomes in BCa. Figure 1. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e388fe8f21125bbefce3cee2961f295d519ae77" target='_blank'>
              Loss of chromosome cytoband 13q14.2 orchestrates breast cancer pathogenesis and drug response
              </a>
            </td>
          <td>
            Parastoo Shahrouzi, Youness Azimzade, Wioletta Brankiewicz, Sugandha Bhatia, David Kunke, Derek Richard, Xavier Tekpli, Vessela N Kristensen, Pascal H.G. Duijf
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Human tumors progress in part by accumulating epigenetic alterations, which include gains and losses of DNA methylation in different parts of the cancer cell genome. Recent work has revealed a link between these two opposite alterations by showing that DNA hypomethylation in tumors can induce the expression of transcripts that overlap downstream gene promoters and thereby induce their hypermethylation. Preliminary in silico evidence prompted us to investigate if this mechanism applies to the locus harboring AGO1, a gene that plays a central role in miRNA biogenesis and RNA interference. Inspection of public RNA-Seq datasets and RT-qPCR experiments show that an alternative transcript starting 13.4 kb upstream of AGO1 (AGO1-V2) is expressed specifically in testicular germ cells, and becomes aberrantly activated in different types of tumors, particularly in tumors of the esophagus, stomach, and lung. This expression pattern classifies AGO1-V2 into the group of “Cancer-Germline” (CG) genes. Analysis of transcriptomic and methylomic datasets provided evidence that transcriptional activation of AGO1-V2 depends on DNA demethylation of its promoter region. Western blot experiments revealed that AGO1-V2 encodes a shortened isoform of AGO1, corresponding to a truncation of 75 aa in the N-terminal domain, and which we therefore referred to as “∆NAGO1”. Interestingly, significant correlations between hypomethylation/activation of AGO1-V2 and hypermethylation/repression of AGO1 were observed upon examination of tumor cell lines and tissue datasets. Overall, our study reveals the existence of a process of interdependent epigenetic alterations in the AGO1 locus, which promotes swapping between two AGO1 protein-coding mRNA isoforms in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27dc2f1e01e40a64b0f8910cba78bb0f22899826" target='_blank'>
              DNA Hypomethylation Underlies Epigenetic Swapping between AGO1 and AGO1-V2 Isoforms in Tumors
              </a>
            </td>
          <td>
            Jean S. Fain, Camille Wangermez, A. Loriot, Claudia Denoue, C. De Smet
          </td>
          <td>2024-06-22</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b93e43a1982115416b67edb36296e1d7dd7f71fe" target='_blank'>
              The DNA methylome of pediatric brain tumors appears shaped by structural variation and predicts survival
              </a>
            </td>
          <td>
            Fengju Chen, Yiqun Zhang, Lanlan Shen, Chad J. Creighton
          </td>
          <td>2024-08-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5073eeabc50bea9fdc09e2d2eeb42d91221fda8" target='_blank'>
              Integrative genomics identifies SHPRH as a tumor suppressor gene in lung adenocarcinoma that regulates DNA damage response
              </a>
            </td>
          <td>
            A. Nagelberg, T. Sihota, Yu-Chi Chuang, Rocky Shi, Justine L M Chow, John English, C. MacAulay, Stephen Lam, Wan L Lam, William W Lockwood
          </td>
          <td>2024-06-18</td>
          <td>British Journal of Cancer</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="ABSTRACT Aneuploidy, the presence of an aberrant number of chromosomes, has been associated with tumorigenesis for over a century. More recently, advances in karyotyping techniques have revealed its high prevalence in cancer: About 90% of solid tumors and 50–70% of hematopoietic cancers exhibit chromosome gains or losses. When analyzed at the level of specific chromosomes, there are strong patterns that are observed in cancer karyotypes both pan-cancer and for specific cancer types. These specific aneuploidy patterns correlate strongly with outcomes for tumor initiation, progression, metastasis formation, immune evasion and resistance to therapeutic treatment. Despite their prominence, understanding the basis underlying aneuploidy patterns in cancer has been challenging. Advances in genetic engineering and bioinformatic analyses now offer insights into the genetic determinants of aneuploidy pattern selection. Overall, there is substantial evidence that expression changes of particular genes can act as the positive selective forces for adaptation through aneuploidy. Recent findings suggest that multiple genes contribute to the selection of specific aneuploid chromosomes in cancer; however, further research is necessary to identify the most impactful driver genes. Determining the genetic basis and accompanying vulnerabilities of specific aneuploidy patterns is an essential step in selectively targeting these hallmarks of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0123dd290f56b3193001c1283aea1071e6f411ed" target='_blank'>
              Selection forces underlying aneuploidy patterns in cancer
              </a>
            </td>
          <td>
            Tamara C. Klockner, Christopher S. Campbell
          </td>
          <td>2024-06-24</td>
          <td>Molecular & Cellular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 We sought to identify mechanisms that activate MYC expression in germinal center B cell-(GCB) diffuse large B-cell lymphomas (DLBCL) in the absence of MYC rearrangement. High-throughput CRISPR-interference screens in MYC-rearranged (n=4) and non-MYC-rearranged (n=2) GCB-DLBCL cell lines revealed a distal enhancer complex 436 kb downstream of MYC, GME-1, that was uniquely essential in non-rearranged (MYC-intact) cell lines. CRISPR-interference studies confirmed growth inhibition and MYC downregulation upon GME-1 repression in 4 DLBCL cell lines. ChIP-Seq showed that GME-1 subunits were bound by OCT2, OCA-B, and MEF2B, which are known to activate germinal center-specific enhancers, and knockdown of the corresponding genes reduced MYC expression. GME-1 subunits showed GCB-specific acetylation and accessibility across normal human B cell populations in public chromatin datasets. The mouse ortholog of GME-1 showed germinal center B cell-specific accessibility. GME-1 was strongly acetylated in MYC-intact but not MYC-rearranged GCB-DLBCL cell lines and patient biopsies. Hi-C revealed significant looping of GME-1 to the MYC promoter in MYC protein+, MYC-intact DLBCL biopsies (n=2), but not MYC-rearranged DLBCL (n=2). We used sequence conservation and chromatin accessibility data to define four GME-1 subunits (195A/B and196A/B), which were cloned into luciferase enhancer reporter constructs and compared to two subunits of the BCL6 distal super-enhancer complex that had previously been identified as regulated by the OCT2/OCA-B/MEF2B complex. GME-1 subunit 195B was the only strongly transcription-activating element based on luciferase assays across three MYC-intact cells lines, where it showed activity comparable to BCL6 enhancers. In contrast, GME-1 enhancers were minimally active in reporter assays performed in two MYC-rearranged GCB-DLBCL cell lines. Deletion of a conserved OCT2 binding site or knockdown of the POU2F2 and POU2AF1 genes reduced activity of GME-1. Thus GME-1 trans-regulation is divergent from that of previously characterized GCB enhancers, despite shared dependency on the OCT2 ternary complex. Whole-genome sequencing (WGS) revealed that the GME-1 dependent cell line SUDHL5 bears a 176bp tandem duplication within GME-1 195B. This altered enhancer showed stronger luciferase reporter activity across three MYC-intact cell lines. Published WGS datasets showed isolated genomic tandem copy gains of GME-1 in 2/49 biopsies from recurrent/refractory DLBCL and 1/93 unselected DLBCL. In a cohort of 18 DLBCL with the high-risk DZSig gene expression signature, GME-1 amplification was seen in two biopsies with concurrent deletion of the PVT1 promoter (PVT1-P) and no inter-chromosomal MYC locus rearrangement. CRISPRi showed that PVT1-P repression reduced growth and MYC expression in GME-1-dependent DLBCL, but not in cell lines where rearrangements removed PVT-P from its position in cis with the MYC gene. Our findings identify GME-1 as an activator of MYC expression and target of rare but recurrent genomic copy gains in high-risk DLBCL.
 Citation Format: Aishwarya Gurumurthy, Ashwin R. Iyer, Shih-Chun A. Chu, Marcin P. Cieslik, Russell J.H. Ryan. Characterization of an essential MYC enhancer targeted by focal genomic alterations in diffuse large B-cell lymphoma [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR04.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c6db6b409667440c15e28ec2c1062b89225fa14" target='_blank'>
              Abstract PR04: Characterization of an essential MYC enhancer targeted by focal genomic alterations in diffuse large B-cell lymphoma
              </a>
            </td>
          <td>
            Aishwarya Gurumurthy, Ashwin Iyer, Shih-Chun A. Chu, Marcin P. Cieslik, R. Ryan
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Abstract Recently, biomolecular condensates formed through liquid–liquid phase separation have been widely reported to regulate key intracellular processes involved in cell biology and pathogenesis. BRD4 is a nuclear protein instrumental to the establishment of phase-separated super-enhancers (SEs) to direct the transcription of important genes. We previously observed that protein droplets of BRD4 became hydrophobic as their size increase, implying an ability of SEs to limit the ionization of water molecules by irradiation. Here, we aim to establish if SEs confer radiation resistance in cancer cells. We established an in vitro DNA damage assay that measures the effect of radicals provoked by the Fenton reaction on DNA integrity. This revealed that DNA damage was markedly reduced when BRD4 underwent phase separation with DNA. Accordingly, co-focal imaging analyses revealed that SE foci and DNA damage foci are mutually exclusive in irradiated cells. Lastly, we observed that the radioresistance of cancer cells was significantly reduced when irradiation was combined with ARV-771, a BRD4 de-stabilizer. Our data revealed the existence of innately radioresistant genomic regions driven by phase separation in cancer cells. The disruption of these phase-separated components enfolding genomic DNA may represent a novel strategy to augment the effects of radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e2dfd016fdeaf5bbe54441d068b3a7b5c2bf81" target='_blank'>
              Phase-separated super-enhancers confer an innate radioresistance on genomic DNA
              </a>
            </td>
          <td>
            Koki Matsumoto, Dini Kurnia Ikliptikawati, Keiichi Makiyama, Kako Mochizuki, Maho Tobita, Isao Kobayashi, D. Voon, Keesiang Lim, Kazuma Ogawa, I. Kashiwakura, Hiroshi I Suzuki, H. Yoshino, Richard W. Wong, M. Hazawa
          </td>
          <td>2024-06-14</td>
          <td>Journal of Radiation Research</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Copy number alterations (CNAs), resulting from the gain or loss of genetic material from as little as 50 base pairs or as big as entire chromosome(s), have been associated with many congenital diseases, de novo syndromes and cancer. It is established that CNAs disturb the dosage of genomic regions including enhancers/promoters, long non-coding RNA and gene(s) among others, ultimately leading to an altered balance of key cellular functions. In cancer, CNAs have been associated with almost all steps of the disease: predisposition, initiation, development, maintenance, response to treatment, resistance, and relapse. Therefore, understanding how specific CNAs contribute to tumourigenesis may provide prognostic insight and ultimately lead to the development of new therapeutic approaches to improve patient outcomes. In this review, we provide a snapshot of what is currently known about CNAs and cancer, incorporating topics regarding their detection, clinical impact, origin, and nature, and discuss the integration of innovative genetic engineering strategies, to highlight the potential for targeting CNAs using novel, dosage-sensitive and less toxic therapies for CNA-driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b107cf5f7a3a29e8e0fe916d4102734572eefc3a" target='_blank'>
              Insights into the Clinical, Biological and Therapeutic Impact of Copy Number Alteration in Cancer
              </a>
            </td>
          <td>
            Shannon L. Carey-Smith, R. Kotecha, L. C. Cheung, Sébastien Malinge
          </td>
          <td>2024-06-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f701cb6b5c83faca69b958611b0964a6f2f858d" target='_blank'>
              Non-homologous end joining shapes the genomic rearrangement landscape of chromothripsis from mitotic errors
              </a>
            </td>
          <td>
            Qing Hu, Jose Espejo Valle-Inclán, Rashmi Dahiya, Alison Guyer, Alice Mazzagatti, Elizabeth G. Maurais, Justin L. Engel, Huiming Lu, Anthony J. Davis, I. Cortés-Ciriano, P. Ly
          </td>
          <td>2024-07-04</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8c57c70b2a1bb317c90f2cc0de26bf7a8a7418b" target='_blank'>
              A microdeletion event at 19q13.43 in IDH-mutant astrocytomas is strongly correlated with MYC overexpression
              </a>
            </td>
          <td>
            Ege Ülgen, U. Gerlevik, Sila Gerlevik, Yavuz Oktay, O. U. Sezerman, Şevin Turcan, K. Ozduman
          </td>
          <td>2024-06-14</td>
          <td>Acta Neuropathologica Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer.
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33af20a7d0aac05ac83adef2c51ac9ffcee3995c" target='_blank'>
              Enhancer engagement sustains oncogenic transformation and progression of B-cell precursor acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            G. Corleone, C. Sorino, M. Caforio, S. Di Giovenale, F. de Nicola, F. Goeman, V. Bertaina, A. Pitisci, C. Cortile, Franco Locatelli, V. Folgiero, Maurizio Fanciulli
          </td>
          <td>2024-06-27</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Repetitive DNA sequences can form non-canonical structures such as H-DNA which is an intramolecular triplex DNA structure. The new Telomere-to-Telomere (T2T) genome assembly for the human genome has eliminated gaps, enabling the examination of highly repetitive regions including centromeric and pericentromeric repeats and ribosomal DNA arrays. This gapless assembly allows for the examination of the distribution of H-DNA sequences in parts of the human genome that were not previously annotated. We find that H-DNA appears once every 30,000 bps in the human genome. Its distribution is highly inhomogeneous with H-DNA motif hotspots being detectable in acrocentric chromosomes. Ribosomal DNA arrays in acrocentric chromosomes are the genomic element with the highest H-DNA enrichment, with 13.22% of total H-DNA motifs being found in ribosomal DNA arrays, representing a 42.65-fold enrichment over what would be expected by chance. Across the acrocentric chromosomes we report that 55.87% of all H-DNA motifs found in these chromosomes are in rDNA array loci. The H-DNA motifs are primarily found in the intergenic spacer regions of the ribosomal DNA arrays, generating repeated clusters. We also discover that binding sites for PRDM9, a protein that regulates the formation of double-strand breaks and determines the meiotic recombination hotspots in humans and most mammals, are over 5-fold enriched for H-DNA motifs. Finally, we provide evidence that our findings are consistent in other non-human great ape genomes. We conclude that ribosomal DNA arrays are the most enriched genomic loci for H-DNA sequences in human and other great ape genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eec5edf88573fda646ad1eb9d23c9425162a3d2f" target='_blank'>
              Ribosomal DNA arrays are the most H-DNA rich element in the human genome
              </a>
            </td>
          <td>
            Nikol Chantzi, Michail Patsakis, Akshatha Nayak, Austin Montgomery, I. Mouratidis, Ilias Georgakopoulos-Soares
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29e5e59bd894b6791e1caba61c558e842d1e8aab" target='_blank'>
              Mitoferrin2 is a synthetic lethal target for chromosome 8p deleted cancers
              </a>
            </td>
          <td>
            Stephan Krieg, Thomas Rohde, Tobias Rausch, Luise Butthof, Lena Wendler-Link, Christoph Eckert, K. Breuhahn, Bruno Galy, J. Korbel, Maximilian Billmann, Marco Breinig, Darjus F. Tschaharganeh
          </td>
          <td>2024-06-17</td>
          <td>Genome Medicine</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="Recurrent mutations in serine / arginine-rich splicing factor 2 (SRSF2), particularly the proline-to-histidine substitution at position 95 (P95H), have been proposed to drive neoplastic diseases. SRSF2 plays pivotal roles in pre-mRNA processing and gene transcription. However, the precise impact of these diverse functions of SRSF2 on cancer cell behaviour remains unclear. Here, we show that deletion or homozygous P95H mutation of SRSF2 both cause extensive DNA damage leading to cell cycle arrest in vitro and in vivo, regardless of cell genotype. We mechanistically demonstrate that SRSF2 is required for efficient bi-directional transcription of DNA replication and repair genes, independent of its function in splicing. In contrast, SRSF2 haploinsufficiency induces DNA damage without halting the cell cycle in cancer cells. Inducing the heterozygous Srsf2 P95H mutation leads to clonal expansion of epidermal cells in mouse skin, but tumor formation is inhibited following exposure to carcinogens. To survive carcinogen treatment, cells containing the Srsf2 P95H mutation undergo substantial transcription rewiring that restores bi-directional gene expression. Thus, our study reveals crucial roles of SRSF2 beyond splicing and underscores its importance in regulating transcription to orchestrate the cell cycle along with the DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32907ead7270689c743bba801c421a14be049a66" target='_blank'>
              SRSF2 transcriptional function maintains genome integrity during cell division
              </a>
            </td>
          <td>
            Rebecca E. Wagner, Leonie Arnetzl, Thiago Britto-Borges, Anke Heit-Mondrzyk, Ali Bakr, Etienne Sollier, Jasper Panten, Sylvain Delaunay, Daniela Sohn, P. Schmezer, D. T. Odom, Karin Müller-Decker, C. Plass, Christoph Dieterich, P. Lutsik, Susanne Bornelöv, Michaela Frye
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c9abec0157ccbe7ffcea6b2ec6d1e2342588925" target='_blank'>
              Sustained inactivation of the Polycomb PRC1 complex induces DNA repair defects and genomic instability in epigenetic tumors
              </a>
            </td>
          <td>
            C. Rawal, V. Loubiere, N. Butova, Juliette Gracia, V. Parreno, C. Merigliano, Anne-Marie Martinez, G. Cavalli, I. Chiolo
          </td>
          <td>2024-06-18</td>
          <td>Histochemistry and Cell Biology</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chromosomal translocations in non-Hodgkin lymphoma (NHL) result in activation of oncogenes by placing them under the regulation of immunoglobulin heavy chain (IGH) super-enhancers. Aberrant expression of translocated oncogenes induced by enhancer activity can contribute to lymphomagenesis. The role of the IGH enhancers in normal B-cell development is well established, but knowledge regarding the precise mechanisms of their involvement in control of the translocated oncogenes is limited. The goal of this project was to define the critical regions in the IGH regulatory elements and identify enhancer RNAs (eRNA). We designed a sgRNA library densely covering the IGH enhancers and performed tiling CRISPR interference screens in three NHL cell lines. This revealed three regions crucial for NHL cell growth. With chromatin-enriched RNA-Seq we showed transcription from the core enhancer regions and subsequently validated expression of the eRNAs in a panel of NHL cell lines and tissue samples. Inhibition of the essential IGH enhancer regions decreased expression of eRNAs and translocated oncogenes in several NHL cell lines. The observed expression and growth patterns were consistent with the breakpoints in the IGH locus. Moreover, targeting the Eμ enhancer resulted in loss of B-cell receptor expression. In a Burkitt lymphoma cell line, MYC overexpression partially rescued the phenotype induced by IGH enhancer inhibition. Our results indicated the most critical regions in the IGH enhancers and provided new insights into the current understanding of the role of IGH enhancers in B-cell NHL. As such, this study forms a basis for development of potential therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/801fa2579edbf74a0b66ac42b4a2406cc3439a8b" target='_blank'>
              Core regions in immunoglobulin heavy chain enhancers essential for survival of non-Hodgkin lymphoma cells are identified by a CRISPR interference screen.
              </a>
            </td>
          <td>
            Marta elzbieta Kasprzyk, Weronika Sura, M. Podralska, Marta Kazimierska, Annika Seitz, Wojciech Łosiewski, Tomasz Woźniak, Jeroen E J Guikema, A. Diepstra, Joost L. Kluiver, Anke van den Berg, N. Rozwadowska, A. Dzikiewicz-Krawczyk
          </td>
          <td>2024-06-27</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Abnormal cell proliferation and growth leading to cancer primarily result from cumulative genome mutations. Single gene mutations alone do not fully explain cancer onset and progression; instead, clustered mutations—simultaneous occurrences of multiple mutations—are considered to be pivotal in cancer development and advancement. These mutations can affect different genes and pathways, resulting in cells undergoing malignant transformation with multiple functional abnormalities. Clustered mutations influence cancer growth rates, metastatic potential, and drug treatment sensitivity. This summary highlights the various types and characteristics of clustered mutations to understand their associations with carcinogenesis and discusses their potential clinical significance in cancer. As a unique mutation type, clustered mutations may involve genomic instability, DNA repair mechanism defects, and environmental exposures, potentially correlating with responsiveness to immunotherapy. Understanding the characteristics and underlying processes of clustered mutations enhances our comprehension of carcinogenesis and cancer progression, providing new diagnostic and therapeutic approaches for cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d085d5b52cda0d463d3c07016cf9320b287c441e" target='_blank'>
              Exploring the Role of Clustered Mutations in Carcinogenesis and Their Potential Clinical Implications in Cancer
              </a>
            </td>
          <td>
            Yi Li, Rui Zhu, Jiaming Jin, Haochuan Guo, Jiaxi Zhang, Zhiheng He, Tingming Liang, Li Guo
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Annotation of the cis-regulatory elements that drive transcriptional dysregulation in cancer cells is critical to improving our understanding of tumor biology. Herein, we present a compendium of matched chromatin accessibility (scATAC-seq) and transcriptome (scRNA-seq) profiles at single-cell resolution from human breast tumors and healthy mammary tissues processed immediately following surgical resection. We identify the most likely cell-of-origin for luminal breast tumors and basal breast tumors and then introduce a novel methodology that implements linear mixed-effects models to systematically quantify associations between regions of chromatin accessibility (i.e. regulatory elements) and gene expression in malignant cells versus normal mammary epithelial cells. These data unveil regulatory elements with that switch from silencers of gene expression in normal cells to enhancers of gene expression in cancer cells, leading to the upregulation of clinically relevant oncogenes. To translate the utility of this dataset into tractable models, we generated matched scATAC-seq and scRNA-seq profiles for breast cancer cell lines, revealing, for each subtype, a conserved oncogenic gene expression program between in vitro and in vivo cells. Together, this work highlights the importance of non-coding regulatory mechanisms that underlie oncogenic processes and the ability of single-cell multi-omics to define the regulatory logic of BC cells at single-cell resolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08bf4929898d9a3caa32f9bca65e96287e05306d" target='_blank'>
              Defining the Regulatory Logic of Breast Cancer Using Single-Cell Epigenetic and Transcriptome Profiling
              </a>
            </td>
          <td>
            Matthew J. Regner, Susana Garcia-Recio, A. Thennavan, K. Wiśniewska, Raul Mendez-Giraldez, Brooke Felsheim, Philip M. Spanheimer, Joel S. Parker, C. Perou, Hector L. Franco
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>160</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Fusion-positive rhabdomyosarcoma is an aggressive pediatric cancer molecularly characterized by arrested myogenesis. The defining genetic driver, PAX3::FOXO1, functions as a chimeric gain-of-function transcription factor. An incomplete understanding of PAX3::FOXO1’s in vivo epigenetic mechanisms has hindered therapeutic development. Here, we establish a PAX3::FOXO1 zebrafish injection model and semi-automated ChIP-seq normalization strategy to evaluate how PAX3::FOXO1 initially interfaces with chromatin in a developmental context. We investigated PAX3::FOXO1’s recognition of chromatin and subsequent transcriptional consequences. We find that PAX3::FOXO1 interacts with inaccessible chromatin through partial/homeobox motif recognition consistent with pioneering activity. However, PAX3::FOXO1-genome binding through a composite paired-box/homeobox motif alters chromatin accessibility and redistributes H3K27ac to activate neural transcriptional programs. We uncover neural signatures that are highly representative of clinical rhabdomyosarcoma gene expression programs that are enriched following chemotherapy. Overall, we identify partial/homeobox motif recognition as a new mode for PAX3::FOXO1 pioneer function and identify neural signatures as a potentially critical PAX3::FOXO1 tumor initiation event.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c34ac92eb278acb6b0ddaf9def7a0af29156e864" target='_blank'>
              Rhabdomyosarcoma fusion oncoprotein initially pioneers a neural signature in vivo
              </a>
            </td>
          <td>
            Jack Kucinski, Alexi Tallan, C. Taslim, Meng Wang, Matthew V. Cannon, Katherine M. Silvius, Benjamin Z. Stanton, Genevieve C. Kendall
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction. It has been established that the presence of homologous recombination deficiency in a breast tumor is associated with the effectiveness of treatment. But despite the high chemosensitivity of the tumor to DNA-damaging agents, complete pathological responses to treatment are very rare. And this process may be based on a change in the somatic status of BRCA1, that is, a reversion and return of the wild-type allele occurs and the DNA repair function is restored.Aim. To evaluate changes in the presence of chromosomal aberrations and the expression profile of the main genes of homologous recombination in cell models of breast cancer under the influence of cisplatin and docetaxel.Materials and methods. The study was conducted on breast cancer tumor cell cultures: MCF-7, MDA-MB-231 and MDA-MB-468. A cell model of drug resistance was obtained for two drugs: cisplatin and docetaxel. RNA and DNA were isolated from cell suspension using the RNeasy Plus Mini Kit and QIAamp DNA Mini Kit (Qiagen, Germany), respectively. The expression level of homologous recombination genes was assessed using reverse transcription polymerase chain reaction. To assess the presence of chromosomal aberrations, microarray analysis was performed on DNA chips.Results. Restoration of normal copy number for the BRCA1, CDK12, CHEK1 and RAD51D genes in MCF-7 under the influence of cisplatin was shown. For BRCA2 and PALB2, amplifications were detected. A statistically significant increase in the expression of the BRCA1 (p = 0.04), BRCA2 (p = 0.02), PALB2 (p = 0.01) and RAD51D (p = 0.05) genes was also shown. MDAMB-231 shows that all identified loci with deletions, where the BRCA2, BARD1, CHEK2, PALB2 and RAD54L genes are localized, are restored to normal copy number by cisplatin. The appearance of amplifications was registered for BRCA1, BRIP1, FANCL, RAD51B, PARP1. A similar result was shown for docetaxel. An increase in the expression level is typical for the genes BRCA1 (p = 0.02), BRCA2 (p = 0.02), CHEK2 (p = 0.05), FANCL (p = 0.04), PALB2 (p = 0.05), RAD51C (p = 0.02), PARP1 (p = 0.02), which corresponds to the appearance of amplifications. In the MDA-MB-468 cell culture, an increase in the copy number of only the BRCA1 gene is observed. The effect of docetaxel has no effect on this cell culture. The level of BRCA1 expression increases in direct proportion to the duration of drug action.Conclusion. Thus, the study showed that under the influence of cisplatin, reversion of not only homologous recombination gene mutations, but also other disorders can occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/767d30042e565f8e61dafca9671915ee6d03ecf7" target='_blank'>
              Modulation of homologous recombination gene activity in breast tumor cells in an in vitro model
              </a>
            </td>
          <td>
            M. Tsyganov, A. Frolova, E. A. Kravtsova, I. A. Tsydenova, M. Ibragimova
          </td>
          <td>2024-06-28</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, Andre Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="The protozoan Trypanosoma cruzi, the causative agent of Chagas disease, exhibits polycistronic transcription and unidimensional genome compartmentalization of core (conserved) and disruptive (virulence factors from multigenic families) genes. Approximately 50% of its genome is repetitive, mainly virulence factor genes. Genomic sequences, including repeats, motifs of architectural proteins, and noncoding RNA loci are crucial for genome folding. Here, we evaluated the genomic features associated with higher-order chromatin organization in T. cruzi through extensive computational processing of high-throughput chromosome conformation capture (Hi-C) data, accounting for repetitive regions and improvements in genome annotation. Our study revealed that repetitive DNA (multimapped reads) influences 3D chromatin folding, particularly in determining the boundaries of topologically associated domains (TAD)-like structures. Virulence factor genes, unlike core genes, form shorter and more compact TAD-like structures enriched in loops, suggesting a gene expression regulatory mechanism. We found nonprotein-coding RNA loci (e.g., tRNAs) and transcription termination sites preferentially located at the boundaries of the TAD-like structures, while pseudogenes and multigenic family genes located in unstructured genomic regions. Our data indicate 3D clustering of tRNA loci, likely optimizing transcription by RNA polymerase III, and a complex interaction between spliced-leader RNA and 18S rRNA loci. Our findings provide insights into 3D genome organization in T. cruzi, contributing to the understanding of supranucleosome-level chromatin organization and suggesting possible links between 3D architecture and gene expression. We draw an analogy to the art of origami (e.g., papers folded into various shapes) resembling the DNA packed in chromatin fibers assuming distinct folds within the nucleus. Importance Despite the knowledge about the linear genome sequence and the identification of numerous virulence factors in the protozoan parasite Trypanosoma cruzi, there has been a limited understanding of how these genomic features are spatially organized within the nucleus and how this organization impacts gene regulation and pathogenicity. By providing a detailed analysis of the three-dimensional chromatin architecture in T. cruzi, our study contributed to filling this gap. We deciphered part of the origami structure hidden in the T. cruzi nucleus, showing the unidimensional genomic features are nonrandomly organized in the nuclear 3D landscape. We revealed the possible role of non-protein-coding RNA loci (e.g., tRNAs, SL-RNA, and 18S RNA) in shaping the genomic architecture. These findings provide insights into an additional epigenetic layer that may influence gene expression. Graphical abstract The spatial organization of chromatin within the nuclei of T. cruzi and its resemblance to origami art. A. Identification of the 3D nuclear architectures within T. cruzi nuclei: topologically associating domains (TADs) and their boundaries; chromatin loops; and 3D networks. Inter- and intrachromosomal interactions reflect DNA‒DNA contacts on the same (cis) and between different (trans) chromosomes. B. Resemblance between origami art and chromatin folding. Steps “a” to “l” show the process of folding a flat piece of paper from its unidimensional view up to its 3D boat form.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e7ca9b9ac7a3e68f2ea6b25af2d09f1d131e151" target='_blank'>
              Hidden origami in Trypanosoma cruzi nuclei highlights its nonrandom 3D genomic organization
              </a>
            </td>
          <td>
            N. K. Bellini, Pedro Leonardo Carvalho de Lima, D. Pires, J. D. da Cunha
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="While genotoxic chemotherapeutic agents are among the most effective tools to combat cancer, they are often associated with severe adverse effects caused by indiscriminate DNA damage in non-tumor tissue as well as increased risk of secondary carcinogenesis. This study builds on our previous work demonstrating that the RNA Polymerase I (Pol I) transcription inhibitor CX-5461 elicits a non-canonical DNA damage response and our discovery of a critical role for Topoisomerase 2α (Top2α) in the initiation of Pol I-dependent transcription. Here, we identify Top2α as a mediator of CX-5461 response in the murine Eµ-Myc B lymphoma model whereby sensitivity to CX-5461 is dependent on cellular Top2α expression/activity. Most strikingly, and in contrast to canonical Top2α poisons, we found that the Top2α-dependent DNA damage induced by CX-5461 is preferentially localized at the ribosomal DNA (rDNA) promoter region, thereby highlighting CX-5461 as a loci-specific DNA damaging agent. This mechanism underpins the efficacy of CX-5461 against certain types of cancer and can be used to develop effective non-genotoxic anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/913f673912030e76f53faa13fb8d57cb178fa48f" target='_blank'>
              CX-5461 Preferentially Induces Top2α-Dependent DNA Breaks at Ribosomal DNA Loci
              </a>
            </td>
          <td>
            Donald P. Cameron, J. Sornkom, Sameerh Alsahafi, D. Drygin, G. Poortinga, Grant A. McArthur, Nadine Hein, Ross D Hannan, Konstantin I. Panov
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Hutchinson-Gilford Progeria Syndrome (HGPS) is a rare genetic condition characterized by features of accelerated aging, and individuals with HGPS seldom live beyond their mid-teens. The syndrome is commonly caused by a point mutation in the LMNA gene which codes for lamin A and its splice variant lamin C, components of the nuclear lamina. The mutation causing HGPS leads to production of a truncated, farnesylated form of lamin A referred to as “progerin.” Progerin is also expressed at low levels in healthy individuals and appears to play a role in normal aging. HGPS is associated with an accumulation of genomic DNA double-strand breaks (DSBs) and alterations in the nature of DSB repair. The source of DSBs in HGPS is often attributed to stalling and subsequent collapse of replication forks in conjunction with faulty recruitment of repair factors to damage sites. In this work, we used a model system involving immortalized human cell lines to investigate progerin-induced genomic damage. Using an immunofluorescence approach to visualize phosphorylated histone H2AX foci which mark sites of genomic damage, we report that cells engineered to express progerin displayed a significant elevation of endogenous damage in the absence of any change in the cell cycle profile or doubling time of cells. Genomic damage was enhanced and persistent in progerin-expressing cells treated with hydroxyurea. Overexpression of wild-type lamin A did not elicit the outcomes associated with progerin expression. Our results show that DNA damage caused by progerin can occur independently from global changes in replication or cell proliferation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4814100043718a6a0e30f1cd9c037307d90666" target='_blank'>
              Progerin Can Induce DNA Damage in the Absence of Global Changes in Replication or Cell Proliferation
              </a>
            </td>
          <td>
            Liza A. Joudeh, P. Logan Schuck, Nina M. Van, Alannah J. DiCintio, Jason A. Stewart, Alan S. Waldman
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Male sex chromosome aneuploidies are frequent genetic aberrations in humans characterized by additional Y or X chromosome complements. Jacob (JS) and Klinefelter syndromes (KS), carrying 47,XYY and 47,XXY chromosomes, respectively, share several clinical features, including sterility, hormonal deficits, neurocognitive delay, and skeletal-muscle defects. Despite the high incidence, a complete understanding of the molecular mechanisms underlying the clinical manifestations in sex aneuploid patients is still elusive. In this study, we generated and characterized the inaugural cohort of 47,XYY human induced pluripotent stem cells (iPSCs). We performed a comprehensive transcriptional analysis, including JS primary fibroblasts, iPSCs, and neural stem cells (NSCs) alongside KS cells. Our study revealed a transcriptional feedback mechanism tuning non-PAR X chromosome genes (NPX) homologs in Y supernumerary cells, a phenomenon not detected in X aneuploid iPSCs. By ectopically modulating the expression of selected NPY genes, we demonstrated a direct transcriptional link between the UTY-UTX gene pair. Furthermore, our analyses identified a shared transcriptomic signature between JS and KS, discernible already at the iPSC stage, with a notable enrichment for processes related to neurological functions. This transcriptomic convergence underscores potential commonalities in the molecular pathways underpinning the pathophysiology of male sex chromosome aneuploidies. Finally, through genome-wide DNA methylation profiling of JS iPSCs, we demonstrated that a supernumerary Y chromosome only minimally impacts the methylation status of 47,XYY cells at the pluripotent stage. Our work reveals critical transcriptional feedback mechanisms and shared gene expression signatures in male sex chromosome aneuploidies, paving the way for a better understanding of their common phenotypic features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4a1e46b6ba1f6258afb3db5dff0a58811a485a6" target='_blank'>
              An iPSC-based model of Jacob Syndrome reveals a DNA methylation-independent transcriptional dysregulation shared with X aneuploid cells
              </a>
            </td>
          <td>
            V. Astro, K. Cardona-Londoño, L. V. Cortés-Medina, R. Alghamdi, G. Ramírez-Calderón, F. Kefalas, J. Dilmé-Capó, S. Radío, A. Adamo
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The most significant genetic influence on eye color pigmentation is attributed to the intronic SNP rs12913832 in the HERC2 gene, which interacts with the promoter region of the contiguous OCA2 gene. This interaction, through the formation of a chromatin loop, modulates the transcriptional activity of OCA2, directly affecting eye color pigmentation. Recent advancements in technology have elucidated the precise spatial organization of the genome within the cell nucleus, with chromatin architecture playing a pivotal role in regulating various genome functions. In this study, we investigated the organization of the chromatin close to the HERC2/OCA2 locus in human lymphocyte nuclei using fluorescence in situ hybridization (FISH) and high-throughput chromosome conformation capture (Hi-C) data. The 3 Mb of genomic DNA that belonged to the chromosomal region 15q12-q13.1 revealed the presence of three contiguous chromatin loops, which exhibited a different level of compaction depending on the presence of the A or G allele in the SNP rs12913832. Moreover, the analysis of the genomic organization of the genes has demonstrated that this chromosomal region is evolutionarily highly conserved, as evidenced by the analysis of syntenic regions in species from other Vertebrate classes. Thus, the role of rs12913832 variant is relevant not only in determining the transcriptional activation of the OCA2 gene but also in the chromatin compaction of a larger region, underscoring the critical role of chromatin organization in the proper regulation of the involved genes. It is crucial to consider the broader implications of this finding, especially regarding the potential regulatory role of similar polymorphisms located within intronic regions, which do not influence the same gene by modulating the splicing process, but they regulate the expression of adjacent genes. Therefore, caution should be exercised when utilizing whole-exome sequencing for diagnostic purposes, as intron sequences may provide valuable gene regulation information on the region where they reside. Thus, future research efforts should also be directed towards gaining a deeper understanding of the precise mechanisms underlying the role and mode of action of intronic SNPs in chromatin loop organization and transcriptional regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/415e3e0211688fa86f1fcb9648668b6356857209" target='_blank'>
              The Chromatin Organization Close to SNP rs12913832, Involved in Eye Color Variation, Is Evolutionary Conserved in Vertebrates
              </a>
            </td>
          <td>
            Desiree Brancato, Francesca Bruno, Elvira Coniglio, V. Sturiale, Salvatore Saccone, Concetta Federico
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The relevant role of LINE-1 (L1) retrotransposition in cancer has been recurrently demonstrated in recent years. However, their repetitive nature hampers their identification and detection, hence remaining inaccessible for clinical practice. Also, its clinical relevance for cancer patients is still limited. Here, we develop a new method to quantify L1 activation, called RetroTest, based on targeted sequencing and a sophisticated bioinformatic pipeline, allowing its application in tumor biopsies. First, we performed the benchmarking of the method and confirmed its high specificity and reliability. Then, we unravel the L1 activation in HNSCC according to a more extensive cohort including all the HNSCC tumor stages. Our results confirm that RetroTest is remarkably efficient for L1 detection in tumor biopsies, reaching a high sensitivity and specificity. In addition, L1 retrotransposition estimation reveals a surprisingly early activation in HNSCC progression, contrary to its classical association with advanced tumor stages. This early activation together with the genomic mutational profiling of normal adjacent tissues supports field cancerization process in this tumor. These results underline the importance of estimating L1 retrotransposition in clinical practice towards an earlier and more efficient diagnosis in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ae21a6395bf2fa53698eb0f53f5dc6c7132bfe" target='_blank'>
              RetroTest unravels LINE-1 retrotransposition in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, Ana Oitaben, Sonia Zumalave, Bernardo Rodriguez-4 Martin, María Gallardo-Gómez, Martín Santamarina, Ana Pequeño-Valtierra, L. Juaneda-Magdalena, Ramón García-Escudero, J. López-Cedrún, Máximo Fraga, J. Tubío, Mónica Martínez-Fernández, Monica Martínez
          </td>
          <td>2024-07-24</td>
          <td>None</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Purpose: Complex chromosomal rearrangements (CCRs) are rare structural variants involving three or more chromosomal break- points. Most de novo reported CCRs pose challenges for diagnosis and management. They often require karyotyping, fluorescence in situ hybridization (FISH), and chromosomal microarray analysis (CMA) for clinical diagnosis because of the limitations of each method. Here we report an inherited exceptionally complex CCR involving 4 chromosomes and 11 breakpoints in a family with multisystem anomalies. Methods: We evaluated the CCRs using karyotyping, FISH, CMA, and two emerging genomic technologies: high-throughput chromosome conformation capture sequencing (Hi-C; aka genomic proximity mapping, GPM) and optical genome mapping (OGM). We also performed functional studies using transcriptome and methylome analyses. Results: The proband, who had intellectual disability and immune deficiency, shared CCRs with her unaffected mother involving chromosomes 1, 7, and 11 by karyotyping. However, CMA revealed a duplication and three deletions in the proband in contrast to her mother's balanced genome. Hi-C (GPM) and OGM detected the CCRs and copy number alterations but also uncovered additional breakpoints at high resolution, including an insertion in 4p and two cryptic rearrangements at 7p. Transcriptome and methylome analyses identified likely biological pathways associated with the proband's phenotypes. Conclusion: Combining cytogenetic and genomic methods provided comprehensive characterization and defined the breakpoints at high resolution in both proband and mother. This underscores the value of novel cytogenetic and genomic techniques in deciphering complex genome rearrangements and the significance of integrative genomic analysis and functional characterization in understanding clinical phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d8dc703c96a52057ab6829a72726f61eb74bd56" target='_blank'>
              Genetic and functional characterization of inherited complex chromosomal rearrangements in a family with multisystem anomalies
              </a>
            </td>
          <td>
            H. Fang, S. M. Eacker, Y. Wu, W. Neufeld-Kaiser, M. Laurino, S. Keel, M. S. Horwitz, Y. Liu
          </td>
          <td>2024-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Melanoma showcases a complex interplay of genetic alterations and intra- and inter-cellular morphological changes during metastatic transformation. While pivotal, the role of specific mutations in dictating these changes still needs to be fully elucidated. Telomerase promoter mutations (TERTp mutations) significantly influence melanoma’s progression, invasiveness, and resistance to various emerging treatments, including chemical inhibitors, telomerase inhibitors, targeted therapy, and immunotherapies. We aim to understand the morphological and phenotypic implications of the two dominant monoallelic TERTp mutations, C228T and C250T, enriched in melanoma metastasis. We developed isogenic clonal cell lines containing the TERTp mutations and utilized dual-color expression reporters steered by the endogenous Telomerase promoter, giving us allelic resolution. This approach allowed us to monitor morpholomic variations induced by these mutations. TERTp mutation-bearing cells exhibited significant morpholome differences from their wild-type counterparts, with increased allele expression patterns, augmented wound-healing rates, and unique spatiotemporal dynamics. Notably, the C250T mutation exerted more pronounced changes in the morpholome than C228T, suggesting a differential role in metastatic potential. Our findings underscore the distinct influence of TERTp mutations on melanoma’s cellular architecture and behavior. The C250T mutation may offer a unique morpholomic and systems-driven advantage for metastasis. These insights provide a foundational understanding of how a non-coding mutation in melanoma metastasis affects the system, manifesting in cellular morpholome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b66856ef0863e2a38d3da5fdaad88a280ff968" target='_blank'>
              Using deep learning to decipher the impact of telomerase promoter mutations on the dynamic metastatic morpholome
              </a>
            </td>
          <td>
            Andres J. Nevarez, Anusorn Mudla, Sabrina A. Diaz, Nan Hao
          </td>
          <td>2024-07-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Alternative DNA conformation formed by sequences called flipons potentially alter the readout of genetic information by directing the shape-specific assembly of complexes on DNA The biological roles of G-quadruplexes formed by motifs rich in guanosine repeats have been investigated experimentally using many different methodologies including G4-seq, G4 ChIP-seq, permanganate nuclease footprinting (KEx), KAS-seq, CUT&Tag with varying degrees of overlap between the results. Here we trained large language model DNABERT on existing data generated by KEx, a rapid chemical footprinting technique performed on live, intact cells using potassium permanganate. The snapshot of flipon state when combined with results from other in vitro methods that are performed on permeabilized cells, allows a high confidence mapping of G-flipons to proximal enhancer and promoter sequences. Using G4-DNABERT predictions,with data from ENdb, Zoonomia cCREs and single cell G4 CUT&Tag experiments, we found support for a model where G4-quadruplexes regulate gene expression through chromatin loop formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea25e2e4cfb927141977cb915b650e3384446d86" target='_blank'>
              Analysis of live cell data with G-DNABERT supports a role for G-quadruplexes in chromatin looping
              </a>
            </td>
          <td>
            Dmitry Konovalov, Dmitry Umerenkov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While genome-wide association studies and expression quantitative trait loci (eQTL) analysis have made significant progress in identifying noncoding variants associated with prostate cancer risk and bulk tissue transcriptome changes, the regulatory effect of these genetic elements on gene expression remains largely unknown. Recent developments in single-cell sequencing have made it possible to perform ATAC-seq and RNA-seq profiling simultaneously to capture functional associations between chromatin accessibility and gene expression. In this study, we tested our hypothesis that this multiome single-cell approach allows for mapping regulatory elements and their target genes at prostate cancer risk loci. We applied a 10X Multiome ATAC + Gene Expression platform to encapsulate Tn5 transposase-tagged nuclei from multiple prostate cell lines for a total of 65,501 high quality single cells from RWPE1, RWPE2, PrEC, BPH1, DU145, PC3, 22Rv1 and LNCaP cell lines. To address data sparsity commonly seen in the single-cell sequencing, we performed targeted sequencing to enrich sequencing data at prostate cancer risk loci involving 2,730 candidate germline variants and 273 associated genes. Although not increasing the number of captured cells, the targeted multiome data did improve eQTL gene expression abundance by about 20% and chromatin accessibility abundance by about 5%. Based on this multiomic profiling, we further associated RNA expression alterations with chromatin accessibility of germline variants at single cell levels. Cross validation analysis showed high overlaps between the multiome associations and the bulk eQTL findings from GTEx prostate cohort. We found that about 20% of GTEx eQTLs were covered within the significant multiome associations (p-value ≤ 0.05, gene abundance percentage ≥ 5%), and roughly 10% of the multiome associations could be identified by significant GTEx eQTLs. We also analyzed accessible regions with available heterozygous SNP reads and observed more frequent association in genomic regions with allelically accessible variants (p = 0.0055). Among these findings were previously reported regulatory variants including rs60464856-RUVBL1 (multiome p-value = 0.0099 in BPH1) and rs7247241-SPINT2 (multiome p-value = 0.0002- 0.0004 in 22Rv1). We also functionally validated a new regulatory SNP and its target gene rs2474694-VPS53 (multiome p-value = 0.00956 in BPH1 and 0.00625 in DU145) by reporter assay and SILAC proteomics sequencing. Taken together, our data demonstrated the feasibility of the multiome single-cell approach for identifying regulatory SNPs and their regulated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936580ce17da8d58425b498ecb0e8504faaa215a" target='_blank'>
              Identify Regulatory eQTLs by Multiome Sequencing in Prostate Single Cells
              </a>
            </td>
          <td>
            Yijun Tian, Lang Wu, Chang-Ching Huang, Liang Wang
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d5a8855e0b26b38ca1d7e3d7f31f3869c8bb2f3" target='_blank'>
              Evaluation of somatic copy number variation detection by NGS technologies and bioinformatics tools on a hyper-diploid cancer genome
              </a>
            </td>
          <td>
            Daniall Masood, Luyao Ren, Cu Nguyen, Francesco G. Brundu, Lily Zheng, Yongmei Zhao, Erich Jaeger, Yong Li, Seong Won Cha, Aaron Halpern, Sean Truong, Michael Virata, Chunhua Yan, Qing-rong Chen, Andy Pang, Reyes Alberto, Chunlin Xiao, Zhaowei Yang, Wanqiu Chen, Charles Wang, Frank Cross, Severine Catreux, Leming Shi, Julia A. Beaver, Wenming Xiao, D. Meerzaman
          </td>
          <td>2024-06-20</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9895762dc4559c029ea063221ae034c05dcf58ce" target='_blank'>
              DNA replication in early mammalian embryos is patterned, predisposing lamina-associated regions to fragility
              </a>
            </td>
          <td>
            Shuangyi Xu, Ning Wang, M. Zuccaro, Jeannine Gerhardt, R. Iyyappan, Giovanna Nascimento Scatolin, Zongliang Jiang, T. Baslan, A. Koren, Dieter Egli
          </td>
          <td>2024-06-19</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>31</td>
        </tr>

        <tr id="Background and Aim: Canine transmissible venereal tumor (CTVT), a unique transmissible cancer in dogs, affects the external genitalia and potentially spreads to other parts of the body. While somatic mutations in oncogenic and tumor-suppressing genes are linked to CTVT development, the impact of DNA methylation, which affects gene expression, remains unclear. This study explored whether DNA methylation in the promoter regions of the MYC oncogene and CDKN2B tumor suppressor genes in CTVTs is associated with their expression, both at the gene and protein levels. 

Materials and Methods: To investigate promoter DNA methylation of MYC and CDKN2B in CTVTs, we analyzed frozen tissue samples from genital CTVT (GTVTs) and extragenital CTVT (ETVTs). Genomic DNA was extracted, bisulfite-treated, and analyzed using bisulfite polymerase chain reaction (PCR) and sequencing. The messenger RNA and protein of MYC and CDKN2B were also extracted and assessed by real-time PCR and Western blotting. Matching formalin-fixed, paraffin-embedded blocks were used for immunohistochemical staining to visualize protein distribution in GTVT and ETVT tissues. 

Results: Although both GTVT and ETVT samples showed MYC promoter methylation, the extent of methylation differed significantly. GTVTs displayed a much higher degree of methylation, potentially explaining the more pronounced downregulation of MYC gene expression and reduction in c-MYC protein levels observed in GTVTs compared with ETVTs. Our data revealed a prevalent hypermethylation pattern in the CDKN2B promoter across both sample types. However, DNA methylation, which was expected to have a suppressive effect, did not correlate with gene/protein expression. GTVTs displayed high protein levels despite significantly reduced CDKN2B expression. Conversely, ETVTs maintained regular CDKN2B expression but exhibited reduced protein production, suggesting a complex interplay between methylation and expression in these tumors. 

Conclusion: MYC demonstrated a clear association between its promoter methylation status, gene expression, and protein levels; however, CDKN2B lacked this correlation, implying the involvement of methylation-independent regulatory mechanisms and highlighting the need for further investigation. 

Keywords: canine transmissible venereal tumor, CDKN2B, DNA methylation, MYC, oncogene, tumor suppressor gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26c5019910fb12afd2000e4151f192b0d24fad38" target='_blank'>
              Divergent DNA methylation patterns and gene expression in MYC and CDKN2B in canine transmissible venereal tumors
              </a>
            </td>
          <td>
            Soukkangna Keopaseuth, K. Pringproa, P. Patchanee, C. Setthawongsin, S. Techangamsuwan, Phongsakorn Chuammitri
          </td>
          <td>2024-07-01</td>
          <td>Veterinary World</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The expression of eukaryotic genes relies on the precise 3’-terminal cleavage and polyadenylation of newly synthesized pre-mRNA transcripts. Defects in these processes have been associated with various diseases, including cancer. While cancer-focused sequencing studies have identified numerous driver mutations in protein-coding sequences, noncoding drivers – particularly those affecting the cis-elements required for pre-mRNA cleavage and polyadenylation – have received less attention. Here, we systematically analysed cancer somatic mutations affecting 3’UTR polyadenylation signals using the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset. We found a striking enrichment of cancer-specific somatic mutations that disrupt strong and evolutionarily conserved cleavage and polyadenylation signals within tumour suppressor genes. Further bioinformatics and experimental analyses conducted as a part of our study suggest that these mutations have a profound capacity to downregulate the expression of tumour suppressor genes. Thus, this work uncovers a novel class of noncoding somatic mutations with significant potential to drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4b3938c7f84b24956b4223cbd287669eff1caf" target='_blank'>
              Recurrent disruption of tumour suppressor genes in cancer by somatic mutations in cleavage and polyadenylation signals
              </a>
            </td>
          <td>
            Yaroslav A. Kainov, F. Hamid, E. Makeyev
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Mutually exclusive loss-of-function alterations in gene pairs are those that occur together less frequently than may be expected and may denote a synthetically lethal relationship (SSL) between the genes. SSLs can be exploited therapeutically to selectively kill cancer cells. Here, we analysed mutation, copy number variation, and methylation levels in samples from The Cancer Genome Atlas, using the hypergeometric and the Poisson binomial tests to identify mutually exclusive inactivated genes. We focused on gene pairs where one is an inactivated tumour suppressor and the other a gene whose protein product can be inhibited by known drugs. This provided an abundance of potential targeted therapeutics and repositioning opportunities for several cancers. These data are available on the MexDrugs website, https://bioinformaticslab.sussex.ac.uk/mexdrugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7250b34d6949747e0dee284668cf7df5c1a39253" target='_blank'>
              Identifying actionable synthetically lethal cancer gene pairs using mutual exclusivity.
              </a>
            </td>
          <td>
            Sarah K. Wooller, Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2024-07-08</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The centromeric protein-A (CENP-A) is an evolutionary conserved histone H3 variant that marks the identity of the centromeres. Several mechanisms regulate the centromeric deposition of CENP-A as its mislocalization causes erroneous chromosome segregation, leading to aneuploidy-based diseases, including cancers. The most crucial deposition factor is a CENP-A specific chaperone, HJURP (Scm3 in budding yeast), which specifically binds to CENP-A. However, the discovery of HJURP as a DDR (DNA damage repair) protein and evidence of its binding to Holliday junctions in vitro indicate a CENP-A-deposition-independent role of these chaperones. In this study, using budding yeast, we demonstrate that Scm3 is crucial for the DDR pathway as scm3 cells are sensitive to DNA damage. We further observe that the scm3 mutant interacts with the rad52 DDR mutant and is compromised in activating DDR-mediated arrest. We demonstrate that Scm3 associates with the DNA damage sites and undergoes posttranslational modifications upon DNA damage. Overall, from this report and earlier studies on HJURP, we conclude that DDR functions of CENP-A chaperones are conserved across eukaryotes. Thus, the revelation that these chaperones confer genome stability in more than one pathway has clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20b8f002bffe33b7b0eebb870a6636324b784a0d" target='_blank'>
              Evidence of centromeric histone 3 chaperone involved in DNA damage repair pathway
              </a>
            </td>
          <td>
            Prakhar Agarwal, S. Ghosh
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/589af1da53c29b65656184828eec590acfbf3b30" target='_blank'>
              Genetic and epigenetic instability as an underlying driver of progression and aggressive behavior in IDH-mutant astrocytoma
              </a>
            </td>
          <td>
            Timothy E Richardson, Jamie M. Walker, Dolores Hambardzumyan, Steven Brem, K. Hatanpaa, Mariano S Viapiano, Balagopal Pai, M. Umphlett, Oren J. Becher, M. Snuderl, S. McBrayer, Kalil G. Abdullah, N. Tsankova
          </td>
          <td>2024-07-16</td>
          <td>Acta Neuropathologica</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Three-stranded DNA-RNA structures known as R-loops that form during papillomavirus transcription can cause transcription-replication conflicts and lead to DNA damage. We found that R-loops accumulated at the viral early promoter in human papillomavirus (HPV) episomal cells but were greatly reduced in cells with integrated HPV genomes. RNA-DNA helicases unwind R-loops and allow for transcription and replication to proceed. Depletion of the RNA-DNA helicase senataxin (SETX) using siRNAs increased the presence of R-loops at the viral early promoter in HPV-31 (CIN612) and HPV-16 (W12) episomal HPV cell lines. Depletion of SETX reduced viral transcripts in episomal HPV cell lines. The viral E2 protein, which binds with high affinity to specific palindromes near the promoter and origin, complexes with SETX, and both SETX and E2 are present at the viral p97 promoter in CIN612 and W12 cells. SETX overexpression increased E2 transcription activity on the p97 promoter. SETX depletion also significantly increased integration of viral genomes in CIN612 cells. Our results demonstrate that SETX resolves viral R-loops to proceed with HPV transcription and prevent genome integration.IMPORTANCEPapillomaviruses contain small circular genomes of approximately 8 kilobase pairs and undergo unidirectional transcription from the sense strand of the viral genome. Co-transcriptional R-loops were recently reported to be present at high levels in cells that maintain episomal HPV and were also detected at the early viral promoter. R-loops can inhibit transcription and DNA replication. The process that removes R-loops from the PV genome and the requisite enzymes are unknown. We propose a model in which the host RNA-DNA helicase senataxin assembles on the HPV genome to resolve R-loops in order to maintain the episomal status of the viral genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3257970da042eeca9c08a9a174f0a9b3a46f14" target='_blank'>
              Senataxin mediates R-loop resolution on HPV episomes.
              </a>
            </td>
          <td>
            L. Jose, Keely Smith, Anaiya Crowner, EJ Androphy, M. DeSmet
          </td>
          <td>2024-07-24</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0807e64db184649a4b0df3bd08502c4d34be0a53" target='_blank'>
              Programming tumor evolution with selection gene drives to proactively combat drug resistance.
              </a>
            </td>
          <td>
            Scott M. Leighow, Josh Reynolds, Ivan Sokirniy, Shun Yao, Zeyu Yang, Haider Inam, Dominik Wodarz, Marco Archetti, Justin Pritchard
          </td>
          <td>2024-07-04</td>
          <td>Nature biotechnology</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Tumors often show an initial response to chemotherapy, but then develop resistance, leading to relapse and poor prognosis. We hypothesized that a genomic comparison of mutations in pre-treated versus treatment-naive tumors would serve to identify genes that confer resistance. A challenge in such an analysis is that therapy alters mutation burdens and signatures, confounding association studies and complicating identifying causal, selected mutations. We developed DiffInvex, a framework for identifying changes in selection acting on individual genes in somatic genomes. Crucially, DiffInvex draws on a mutation rate baseline that accounts for these shifts in neutral mutagenesis during cancer evolution. We applied DiffInvex to 9,953 cancer whole-genomes from 29 cancer types from 8 studies, containing both WGS of treatment-naive tumors and tumors pre-treated by various drugs, identifying genes where point mutations are under conditional positive or negative selection for a certain chemotherapeutic, suggesting resistance mechanisms occurring via point mutation. DiffInvex confirmed well-known chemoresistance-driver mutations in EGFR, ESR1, KIT and AR genes as being under conditional positive selection, with additional cancer types identified for EGFR and KIT. Additionally, DiffInvex identified 11 genes with treatment-associated selection for different classes of therapeutics. In most cases, these genes were common cancer genes including PIK3CA, APC, MAP2K4 and MAP3K1. This suggests that tumor resistance to therapy via mutation often occurs via selective advantages conferred by known driver genes, rather than via mutations in specialized resistance genes. Various gene-chemotherapy associations were further supported in tests for functional impact of mutations, again implemented in a conditional selection setting, as well as replicating in independent panel or exome sequencing data. In addition to nominating drug resistance genes that could be targeted by future therapeutics, DiffInvex can also be applied to diverse analysis in cancer evolution, such as comparing normal and tumoral tissues, or analyzing subclonal evolution, identifying changes in selection over time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/71b71e5df0edb673b45bd36f564fa0da7eae40ed" target='_blank'>
              The DiffInvex evolutionary model for conditional somatic selection identifies chemotherapy resistance genes in 10,000 cancer genomes
              </a>
            </td>
          <td>
            Ahmed Khalil, F. Supek
          </td>
          <td>2024-06-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic regulatory mechanism that plays a critical role in a variety of cellular processes, including the regulation of cell fate during development, maintenance of cell identity, and genome stability. DNA methylation is tightly regulated by enzymatic reactions and its deregulation plays an important role in the development of cancer. Specific DNA methylation alterations have been found in pediatric solid tumors, providing new insights into the development of these tumors. In addition, DNA methylation profiles have greatly contributed to tune the diagnosis of pediatric solid tumors and to define subgroups of patients with different risks of progression, leading to the reduction in unwanted toxicity and the improvement of treatment efficacy. This review highlights the dysregulated DNA methylome in pediatric solid tumors and how this information provides promising targets for epigenetic therapies, particularly inhibitors of DNMT enzymes (DNMTis). Opportunities and limitations are considered, including the ability of DNMTis to induce viral mimicry and immune signaling by tumors. Besides intrinsic action against cancer cells, DNMTis have the potential to sensitize immune-cold tumors to immunotherapies and may represent a remarkable option to improve the treatment of challenging pediatric solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1eac857dfcc5a5e3b672ddb17d6097e48bc70f" target='_blank'>
              Targeting DNA Methylation Machinery in Pediatric Solid Tumors
              </a>
            </td>
          <td>
            C. Cristalli, Katia Scotlandi
          </td>
          <td>2024-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Structural variants (SVs) are important contributors to human disease. Their characterization remains however difficult due to their size and association with repetitive regions. Long-read sequencing (LRS) and optical genome mapping (OGM) can aid as their molecules span multiple kilobases and capture SVs in full. In this study, we selected six individuals who presented with unresolved SVs. We applied LRS onto all individuals and OGM to a subset of three complex cases. LRS detected and fully resolved the interrogated SV in all samples. This enabled a precise molecular diagnosis in two individuals. Overall, LRS identified 100% of the junctions at single-basepair level, providing valuable insights into their formation mechanisms without need for additional data sources. Application of OGM added straightforward variant phasing, aiding in the unravelment of complex rearrangements. These results highlight the potential of LRS and OGM as follow-up molecular tests for complete SV characterization. We show that they can assess clinically relevant structural variation at unprecedented resolution. Additionally, they detect (complex) cryptic rearrangements missed by conventional methods. This ultimately leads to an increased diagnostic yield, emphasizing their added benefit in a diagnostic setting. To aid their rapid adoption, we provide detailed laboratory and bioinformatics workflows in this manuscript.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d491356850d1dca650c1719115928a885707ff0" target='_blank'>
              Full characterization of unresolved structural variation through long-read sequencing and optical genome mapping
              </a>
            </td>
          <td>
            Griet De Clercq, L. Vantomme, Barbara Dewaele, B. Callewaert, Olivier Vanakker, Sandra Janssens, Bart Loeys, Mojca Strazisar, W. D. Coster, J. Vermeesch, A. Dheedene, B. Menten
          </td>
          <td>2024-07-19</td>
          <td>None</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Transcribed enhancer maps can reveal nuclear interactions underpinning each cell type and connect specific cell types to diseases. Using a 5′ single-cell RNA sequencing approach, we defined transcription start sites of enhancer RNAs and other classes of coding and noncoding RNAs in human CD4+ T cells, revealing cellular heterogeneity and differentiation trajectories. Integration of these datasets with single-cell chromatin profiles showed that active enhancers with bidirectional RNA transcription are highly cell type–specific and that disease heritability is strongly enriched in these enhancers. The resulting cell type–resolved multimodal atlas of bidirectionally transcribed enhancers, which we linked with promoters using fine-scale chromatin contact maps, enabled us to systematically interpret genetic variants associated with a range of immune-mediated diseases. Editor’s summary Currently, 3′ single-cell RNA sequencing is widely used as a tool for single-cell gene expression analysis. By leveraging a distinctive “cap signature” derived from the 5′-end cap, Oguchi et al. introduced a technique to investigate the precise position of the RNA 5′-ends in single cells, enabling simultaneous profiling of gene expression and enhancer activity. The authors constructed a CD4+ T cell–type resolved multimodal atlas to integrate and elucidate CD4+ T cell diversity and genetic risk of immune-mediated diseases. They also identified rare, uncharacterized CD4+ T cell types that might contribute to the pathology of immune-mediated diseases. —Di Jiang INTRODUCTION Enhancers are distal cis-regulatory elements that increase the expression of target genes. Active enhancers produce bidirectional enhancer RNAs (eRNAs). Transcription of eRNAs can be used to measure enhancer activity and to delineate enhancer architecture at nucleotide resolution. Profiling of transcribed enhancers in single human cells can provide insights into cell type–specific gene regulation and genetic predisposition to diseases. However, comprehensive maps of eRNAs in cell types relevant to diseases are still lacking. RATIONALE The 5′ cap is added to the first nucleotide of an RNA molecule. By leveraging a specific “cap signature,” we developed read-level prefiltering and transcribed enhancer call (ReapTEC), a method for simultaneously profiling gene expression and enhancer activity by using 5′-end single-cell RNA sequencing (5′ scRNA-seq). We applied ReapTEC and single-nucleus assay for transposase-accessible chromatin using sequencing (snATAC-seq) to a total of ~1 million single human CD4+ T cells and constructed a CD4+ T cell type–resolved multimodal atlas to pinpoint genetic and cellular causes of immune-mediated diseases. RESULTS By using ReapTEC, we accurately mapped the transcription start sites of mRNAs, long noncoding RNAs, and eRNAs in single cells, which allowed us to study the expression of not only genes but also bidirectionally transcribed candidate enhancers (btcEnhs). We characterized rare and uncharacterized cell types, including GPR25-expressing regulatory T cells, along with the expression profiles of ~130 surface proteins. We identified 62,803 btcEnhs and 218,508 accessible chromatin regions active across diverse CD4+ T cells. We found higher cell type specificity for RNA transcription than for chromatin accessibility both at promoters and enhancers. We also showed that the heritability of immune-mediated diseases was enriched more in btcEnhs than in open chromatin regions. Using our CD4+ T cell type–resolved atlas of btcEnhs, we systematically interpreted genetic variants associated with a range of immune-mediated diseases. We found that btcEnhs harboring disease SNPs were often active in specific subsets of CD4+ T cells; generally, they were conserved across mammals but in some cases were primate specific. We investigated the target genes of disease-relevant btcEnhs by generating fine-scale chromatin contact maps derived from ultradeep Micro-C data; we further performed CRISPR activation of selected disease-relevant btcEnhs to validate their target genes. CONCLUSION Our study establishes a platform for integrating and elucidating CD4+ T cell diversity and genetic risk of immune-mediated diseases. Although 3′ scRNA-seq is widely used as a major tool for single-cell gene expression analysis, our study demonstrates the strength of 5′ scRNA-seq. We anticipate the utility of our multimodal atlas in addressing variant-to-function challenges. CD4+ T cell type–resolved atlas of btcEnhs for systematic interpretation of the genetic basis of immune-mediated diseases. (A) ReapTEC allows us to detect transcription start sites (TSSs) and active enhancers using 5′ scRNA-seq. (B) Disease-relevant btcEnhs containing genetic variants associated with immune-mediated diseases were linked to target genes by using fine-scale chromatin contact maps. CITE-seq, cellular indexing of transcriptomes and epitopes by sequencing; Ref, reference allele; Alt, alternative allele.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a4b6cb952b21d2e76916c18021dfddb89e4c76" target='_blank'>
              An atlas of transcribed enhancers across helper T cell diversity for decoding human diseases
              </a>
            </td>
          <td>
            Akiko Oguchi, A. Suzuki, Shuichiro Komatsu, Hiroyuki Yoshitomi, S. Bhagat, Raku Son, R. Bonnal, Shohei Kojima, Masaru Koido, Kazuhiro Takeuchi, K. Myouzen, Gyo Inoue, Tomoya Hirai, Hiromi Sano, Y. Takegami, A. Kanemaru, Itaru Yamaguchi, Y. Ishikawa, Nao Tanaka, Shigeki Hirabayashi, Riyo Konishi, Sho Sekito, Takahiro Inoue, Juha Kere, Shunichi Takeda, A. Takaori-Kondo, Itaru Endo, S. Kawaoka, Hideya Kawaji, K. Ishigaki, H. Ueno, Y. Hayashizaki, Massimiliano Pagani, P. Carninci, M. Yanagita, Nicholas Parrish, Chikashi Terao, Kazuhiko Yamamoto, Yasuhiro Murakawa
          </td>
          <td>2024-07-05</td>
          <td>Science</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Premature aging is a hallmark of Down syndrome, caused by trisomy of human chromosome 21, but the reason is unclear and difficult to study in humans. We used an aneuploid model in wild yeast to show that chromosome amplification disrupts nutrient-induced cell-cycle arrest, quiescence entry, and healthy aging, across genetic backgrounds and amplified chromosomes. We discovered that these defects are due in part to aneuploidy-induced dysfunction in Ribosome Quality Control (RQC). Compared to euploids, aneuploids entering quiescence display aberrant ribosome profiles, accumulate RQC intermediates, and harbor an increased load of protein aggregates. Although they have normal proteasome capacity, aneuploids show signs of ubiquitin dysregulation, which impacts cyclin abundance to disrupt arrest. Remarkably, inducing ribosome stalling in euploids produces similar aberrations, while up-regulating limiting RQC subunits or proteins in ubiquitin metabolism alleviates many of the aneuploid defects. Our results provide implications for other aneuploidy disorders including Down syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c6d2e4dd1921e05fad92cf19e5a9d35d322983a" target='_blank'>
              Premature aging in aneuploid yeast is caused in part by aneuploidy-induced defects in Ribosome Quality Control
              </a>
            </td>
          <td>
            Leah E. Escalante, James Hose, Hollis Howe, Norah Paulsen, Michael Place, A. Gasch
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="The bacterium Deinococcus radiodurans is known to efficiently and accurately reassemble its genome after hundreds of DNA double-strand breaks (DSBs). Only at very large amounts of radiation-induced DSBs is this accuracy affected in the wild-type D. radiodurans, causing rearrangements in its genome structure. However, changes in its genome structure may also be possible during the propagation and storage of cell cultures. We investigate this possibility by listing structural differences between three completely sequenced genomes of D. radiodurans strains with a recent common ancestor—the type strain stored and sequenced in two different laboratories (of the ATCC 13939 lineage) and the first sequenced strain historically used as the reference (ATCC BAA-816). We detected a number of structural differences and found the most likely mechanisms behind them: (i) transposition/copy number change in mobile interspersed repeats—insertion sequences and small non-coding repeats, (ii) variable number of monomers within tandem repeats, (iii) deletions between long direct DNA repeats, and (iv) deletions between short (4–10 bp) direct DNA repeats. The most surprising finding was the deletions between short repeats because it indicates the utilization of a less accurate DSB repair mechanism in conditions in which a more accurate one should be both available and preferred. The detected structural differences, as well as SNPs and short indels, while being important footprints of deinococcal DNA metabolism and repair, are also a valuable resource for researchers using these D. radiodurans strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55030650317fbed25fb2ab5464d15ce87378bc16" target='_blank'>
              Structural Differences between the Genomes of Deinococcus radiodurans Strains from Different Laboratories
              </a>
            </td>
          <td>
            K. Zahradka, D. Zahradka, J. Repar
          </td>
          <td>2024-06-27</td>
          <td>Genes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Desmoplastic small round cell tumor (DSRCT) is a highly aggressive cancer predominantly occurring in male adolescents and young adults. The lack of a comprehensive understanding on the biology of the disease is paralleled by its dismal survival rates (5–20%). To overcome this challenge, we first identified and prioritized urgently needed resources for clinicians and researchers. Thus, we established genome-wide single-cell RNA-sequencing and bulk proteomic data of in vitro and in vivo-generated knockdown models of the pathognomonic DSRCT fusion oncoprotein (EWSR1::WT1) and combined them with an original systems-biology-based pipeline including patient data and the largest histology collection of DSRCTs and morphological mimics available to date. These novel tools were enriched with curated public datasets including patient- and cell line-derived ChIP-seq, bulk and single-cell RNA-seq studies resulting in a multi-model and multi-omic toolbox for discovery analyses. As a proof of concept, our approach revealed the alpha-2/delta subunit of the voltage-dependent calcium channel complex, CACNA2D2, as a highly overexpressed, super-enhancer driven, direct target of EWSR1::WT1. Single-cell and bulk-level analyses of patient samples and xenografted cell lines highlighted CACNA2D2 as a critical component of our newly established EWSR1::WT1 oncogenic signature, that can be employed to robustly identify DSRCT in reference sets. Finally, we show that CACNA2D2 is a highly sensitive and specific single biomarker for fast, simple, and cost-efficient diagnosis of DSRCT. Collectively, we establish a large-scale multi-omics dataset for this devastating disease and provide a blueprint of how such toolbox can be used to identify new and clinically relevant diagnostic markers, which may significantly reduce misdiagnoses, and thus improve patient care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1adb9c7dca4062ab1492b77bcd56eb7a98c207ea" target='_blank'>
              Super-enhancer-driven CACNA2D2 is an EWSR1::WT1 signature gene encoding a diagnostic marker for desmoplastic small round cell tumor (DSRCT)
              </a>
            </td>
          <td>
            Florian H. Geyer, Alina Ritter, Seneca S Kinn-Gurzo, Tobias Faehling, Jing Li, Armin Jarosch, Carine Ngo, Endrit Vinca, Karim Aljakouch, Azhar Orynbek, S. Ohmura, Thomas Kirchner, R. Imle, L. Romero-Pérez, Stefanie Bertram, Enrique de Álava, Sophie Postel-Vilnay, A. Banito, M. Sill, Y. Versleijen-Jonkers, Benjamin F B Mayer, Martin Ebinger, M. Sparber-Sauer, Sabine Stegmaier, Daniel Baumhoer, W. Hartmann, Jeroen Krijgsveld, David Horst, Olivier Delattre, P. Grohar, T. G. Grünewald, F. Cidre-Aranaz
          </td>
          <td>2024-07-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Endothelial cells (ECs) maintain vessel tone and barrier integrity, regulate blood homeostasis, and prevent the extravasation of leukocytes under normal physiological conditions. Because of the limited lifespans and batch-to-batch differences with respect to the genetic make-up of primary ECs, established immortal EC lines are extensively used for studying endothelial biology. To address this issue, the immortal endothelial cell line EA.hy926 was developed by fusing primary human umbilical vein endothelial cells (HUVECs) with human lung carcinoma A549 cells. EA.hy926 cells share a number of similar endothelial properties with HUVECs and are considered the immortal counterpart to primary HUVECs. However, the cytogenetic integrity of EA.hy926 cells is not fully elucidated. We characterized EA.hy926 cells with conventional G-banding and molecular cytogenetic techniques such as spectral karyotyping and subtelomeric fluorescence in situ hybridization. Cytogenetic analysis revealed an array of numerical and stable structural chromosomal rearrangements including one deletion, one duplication, one isochromosome, seven simple translocations, and five complex translocations in Ea.hy926 cells. These findings will advance comprehension of EA.hy926 cell biology and augment future endothelial studies, specifically in comparison studies between HUVECs and EA.hy926 cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ad1b12e69b1e161748ea2c28f86f896e19f2f6" target='_blank'>
              G-Banding and Molecular Cytogenetics Detect Novel Translocations and Cryptic Aberrations in Human Immortal Endothelial Cells
              </a>
            </td>
          <td>
            Regina Lichti Binz, R. Pathak
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="A genome-wide chromatin profiling technology, named as Nicking Enzyme Epitope targeted DNA sequencing (NEED-seq) in which antibody-targeted controlled nicking by Nt.CviPII-pGL is used to study specific protein-DNA complexes. NEED-seq is performed in situ in formaldehyde fixed cells, allowing for both visual and genomic resolution of epitope bound chromatin. When applied to nuclei, NEED-seq yielded genome-wide chromatin associated proteins and histone post-translational modifications (PTMs). NEED-seq of lamin B1/B2 demonstrated their association with heterochromatin. Lamin B1 and B2 associated domains (LAD) segregated to three different states, and states with stronger LAD correlated with heterochromatic marks. Hi-C analysis displayed A and B compartment with equal lamin B1/B2 distribution, although methylated DNA remained high in B compartment. LAD clustering with Hi-C resulted in subcompartments, with lamin B1-B2 partitioning to facultative and constitutive heterochromatin respectively and were associated with neuronal development. Thus, lamin B1 and B2 have structural and functional partitioning in mammalian nucleus. Figure 1: Graphical abstract: Model depicting association of lamin B1 and B2 in A (facultative heterochromatin) and B (constitutive heterochromatin) compartmentalization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c3c50fd768031283d19ccee2e6223fbeee2130c" target='_blank'>
              Distinct structural and functional heterochromatin partitioning of lamin B1 and B2 revealed using genome-wide Nicking Enzyme Epitope targeted DNA sequencing
              </a>
            </td>
          <td>
            Sagnik Sen, P. Estève, Karthikeyan Raman, Julie Beaulieu, H. Chin, G. R. Feehery, U. Vishnu, Shuang-yong Xu, James C. Samuelson, S. Pradhan
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1310d603b23100b789e20f7104c3139b43adef05" target='_blank'>
              Decoding the genetic program of micronucleus formation: linking chromosomal instability to human disease
              </a>
            </td>
          <td>
            Jörg Fahrer
          </td>
          <td>2024-06-14</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers.
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a2685872da8abf8d89c45e2e00bd648c1daaeb" target='_blank'>
              DNA methylation and gene expression as determinants of genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Michaël Noë, Dimitrios Mathios, A. Annapragada, Shashikant Koul, Zachariah H. Foda, J. Medina, S. Cristiano, Christopher Cherry, D. C. Bruhm, N. Niknafs, V. Adleff, Leonardo Ferreira, Hari Easwaran, Stephen Baylin, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="The male mammalian germline is characterized by substantial chromatin remodeling associated with the transition from histones to protamines during spermatogenesis, followed by the reversal to nucleohistones in the male pronucleus preceding the zygotic genome activation. Both transitions are associated with the extensive formation of DNA double-strand breaks (DSBs), requiring an estimated 5 to 10 million transient DSBs per spermatozoa. Additionally, the high transcription rate in early stages of spermatogenesis leads to transcription-coupled damage preceding meiotic homologous recombination, potentially further contributing to the DSB landscape in mature spermatozoa. Once meiosis is completed, spermatozoa remain haploid and therefore cannot rely on error-free homologous recombination, but instead depend on error-prone classical non-homologous end joining (cNHEJ). This DNA damage/repair-scenario is proposed to be one of the main causes of the observed paternal mutation propensity in human evolution. Recent studies have shown that DSBs in the male pronucleus are repaired by maternally provided Polθ in Caenorhabditis elegans through Polθ-mediated end joining (TMEJ). Additionally, population genetic datasets have revealed a preponderance of TMEJ signatures associated with human variation. Since these signatures are the result of the combined effect of TMEJ and DSB formation in spermatozoa and male pronuclei, we used a BLISS-based protocol to analyze recurrent DSBs in mature human sperm heads as a proxy of the male pronucleus before zygotic chromatin remodeling. The DSBs were found to be enriched in (YR)n short tandem repeats and in evolutionarily young SINEs, reminiscent to patterns observed in murine spermatids, indicating evolutionary hotspots of recurrent DSB formation in mammalian spermatozoa. Additionally, we detected a similar DSB pattern in diploid human IMR90 cells when cNHEJ was selectively inhibited, indicating the significant impact of absent cNHEJ on the sperm DSB landscape. Strikingly, regions associated with most retained histones, and therefore less condensed chromatin, were not strongly enriched with recurrent DSBs. In contrast, the fraction of retained H3K27me3 in the mature spermatozoa displayed a strong association with recurrent DSBs. DSBs in H3K27me3 are associated with a preference for TMEJ over cNHEJ during repair. We hypothesize that the retained H3K27me3 may trigger transgenerational DNA repair by priming maternal Polθ to these regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60c153158e8100ec0f4fdb6f1cd7fe2e99666eda" target='_blank'>
              DSB profiles in human spermatozoa highlight the role of TMEJ in the male germline
              </a>
            </td>
          <td>
            Maurice Scheuren, Jonas Möhner, Max Müller, Hans Zischler
          </td>
          <td>2024-07-08</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa98c0061b5c115d1ee9f59c1e39ed2f207c47e1" target='_blank'>
              Intertwining roles of R-loops and G-quadruplexes in DNA repair, transcription and genome organization.
              </a>
            </td>
          <td>
            P. Wulfridge, K. Sarma
          </td>
          <td>2024-06-24</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Head and Neck Squamous Cell Carcinoma (HNSCC) remains a significant health burden due to tumor heterogeneity and treatment resistance, emphasizing the need for improved biological understanding and tailored therapies. This study enrolled 31 HNSCC patients for the establishment of patient-derived tumor organoids (PDOs), which faithfully maintained genomic features and histopathological traits of primary tumors. Long-term culture preserved key characteristics, affirming PDOs as robust representative models. PDOs demonstrated predictive capability for cisplatin treatment responses, correlating ex vivo drug sensitivity with patient outcomes. Bulk and single-cell RNA sequencing unveiled molecular subtypes and intratumor heterogeneity (ITH) in PDOs, paralleling patient tumors. Notably, a hybrid epithelial-mesenchymal transition (hEMT)-like ITH program is associated with cisplatin resistance and poor patient survival. Functional analyses identified amphiregulin (AREG) as a potential regulator of the hybrid epithelial/mesenchymal state. Moreover, AREG contributes to cisplatin resistance via EGFR pathway activation, corroborated by clinical samples. In summary, HNSCC PDOs serve as reliable and versatile models, offer predictive insights into ITH programs and treatment responses, and uncover potential therapeutic targets for personalized medicine. One Sentence Summary This study establishes patient-derived tumor organoids (PDOs) from 31 Head and Neck Squamous Cell Carcinoma (HNSCC) patients, faithfully recapitulating characteristics of primary tumors and accurately predicting clinical responses to cisplatin treatment. We reveal intertumoral heterogeneity within PDOs and a hybrid epithelial-mesenchymal transition (hEMT) program conferring cisplatin resistance, highlighting amphiregulin (AREG) as a regulator of cellular plasticity and potential therapeutic target for HNSCC treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82266d25cc2972a0e3e79d403bbffccb3c4b2f76" target='_blank'>
              Genomic and single-cell characterization of patient-derived tumor organoid models of head and neck squamous cell carcinoma
              </a>
            </td>
          <td>
            Jung Hyun Um, Yueyuan Zheng, Qiong Mao, Chehyun Nam, Hua Zhao, Yoon Woo Koh, Su-Jin Shin, Young Min Park, De-Chen Lin
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Homologous recombination deficiency (HRD) is a predictive biomarker for efficacy of PARP inhibition and platinum chemotherapy but remains challenging to detect from low tumor fraction samples such as liquid biopsies. Here, we describe DirectHRD, a whole-genome sequencing (WGS) scar-based classifier that is 10x more sensitive than state-of-the-art methods. DirectHRD can detect HRD at >=1% tumor fraction using 50x WGS of cell-free DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7695779a93857bebb15b364f3d0d9f0cbe0f6d6f" target='_blank'>
              DirectHRD enables sensitive scar-based classification of homologous recombination deficiency (HRD)
              </a>
            </td>
          <td>
            Ruolin Liu, Eugenia Roberts, Heather A. Parsons, Elizabeth H. Stover, A. D. Choudhury, J. Rhoades, T. Blewett, David D. Yang, Joyce F. Liu, Erica L. Mayer, V. Adalsteinsson
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Our study highlights the discovery of recurrent copy number alterations in noncoding regions, specifically blood enhancer cluster (BENC-CNA), in B-precursor acute lymphoblastic leukemia (BCP-ALL) cell lines. We demonstrate that BENC-CNA acts as a super-enhancer, driving MYC expression and possibly contributing to the immortalization and proliferative advantage of BCP-ALL cells in vitro.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fb20ea45f91ff998875ed91d61008dd2c3ae502" target='_blank'>
              Acquired copy number amplification at the MYC enhancer in human B-precursor acute lymphoblastic leukemia cell lines.
              </a>
            </td>
          <td>
            Atsushi Watanabe, Lu Wang, T. Tan, K. Urayama, Toshihiko Kizuki, Chiaki Komatsu, K. Kagami, Tamao Shinohara, S. Kasai, Minori Tamai, Daisuke Harama, K. Akahane, K. Goi, Hiroaki Goto, K. Satou, Tadashi Kaname, Takaomi Sanda, Takeshi Inukai
          </td>
          <td>2024-06-26</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdb874383a628cf292c0913e4a06655cd292ef13" target='_blank'>
              Frequency-dependent selection of neoantigens fosters tumor immune escape and predicts immunotherapy response
              </a>
            </td>
          <td>
            Shaoqing Chen, Duo Xie, Zan Li, Jiguang Wang, Zheng Hu, Da Zhou
          </td>
          <td>2024-06-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Gene expression regulation is one of the most fundamental cellular processes, enabling the activation of a gene to produce either the translatable protein-coding transcript (mRNA) or a functional non-coding RNA with gene regulatory functions, ultimately determining cell identity and function. Although gene expression regulation can occur at transcriptional, translational, and post-translational levels, transcription initiation is the first and the most important step in gene expression, facilitating the transfer of biological information from DNA to protein. Enhancers and super-enhancers are among the master regulators of tissue- and cell-specific transcription regulation involved in cell differentiation and tumor formation. Despite four decades passing since the first discovery of enhancers in eukaryotes and extensive efforts undertaken to identify enhancers on a genomic scale during the last decade, the discovery of enhancers still faces certain limitations and needs further investigation. The perturbation of enhancer function due to genetic or epigenetic changes is closely linked to a range of human disorders, including the development and progression of cancers. Thus, the detection of early cancer-related enhancer activity and the subsequent normalization of expression abnormalities using enhancer-targeting CRISPR epigenetic editing, as well as enhancer-targeting pharmaceuticals, are regarded as groundbreaking therapeutic tactics in preclinical stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782ddfe76cab2a16620844e3e84b112d8c90071b" target='_blank'>
              Enhancers and super-enhancers as master regulators in cancer
              </a>
            </td>
          <td>
            P. Sarvari, Pourya Sarvari, Ivonne Ramírez-Díaz, Karla Rubio
          </td>
          <td>2024-07-24</td>
          <td>INNOSC Theranostics and Pharmacological Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="We describe exciting recent advances in fusion-driven sarcoma etiology, from an epigenetics perspective. By exploring the current state of the field, we identify and describe the central mechanisms that determine sarcomagenesis. Further, we discuss seminal studies in translational genomics, which enabled epigenetic characterization of fusion-driven sarcomas. Important context for epigenetic mechanisms include, but are not limited to, cell cycle and metabolism, core regulatory circuitry, 3-dimensional chromatin architectural dysregulation, integration with ATP-dependent chromatin remodeling, and translational animal modeling. Paradoxically, while the genetic requirements for oncogenic transformation are highly specific for the fusion partners, the epigenetic mechanisms we as a community have uncovered are categorically very broad. This dichotomy prompts the question of whether the investigation of rare disease epigenomics should prioritize studying individual cell populations, thereby examining whether the mechanisms of chromatin dysregulation are specific to a particular tumor. We review recent advances focusing on rhabdomyosarcoma, synovial sarcoma, alveolar soft part sarcoma, clear cell sarcoma, undifferentiated round cell sarcoma, Ewing sarcoma, myxoid/round liposarcoma, epithelioid hemangioendothelioma and desmoplastic round cell tumor. The growing number of groundbreaking discoveries in the field, motivated us to anticipate further exciting advances in the area of mechanistic epigenomics and direct targeting of fusion transcription factors in the years ahead.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01bc563cf3c1cf1b5a64e47d7f0fdef6c20adb4b" target='_blank'>
              Epigenetic determinants of fusion-driven sarcomas: paradigms and challenges
              </a>
            </td>
          <td>
            Benjamin Z. Stanton, S. Pomella
          </td>
          <td>2024-06-14</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Summary From RNA interference to chromatin silencing, diverse genome defense pathways silence selfish genetic elements to safeguard genome integrity1,2. Despite their diversity, different defense pathways share a modular organization, where numerous specificity factors identify diverse targets and common effectors silence them. In the PIWI-interacting RNA (piRNA) pathway, which controls selfish elements in the metazoan germline, diverse target RNAs are first identified by complementary base pairing with piRNAs and then silenced by PIWI-clade nucleases via enzymatic cleavage1,3. Such a binary architecture allows the defense systems to be readily adaptable, where new targets can be captured via the innovation of new specificity factors4,5. Thus, our current understanding of genome defense against lineage-specific selfish genes has been largely limited to the evolution of specificity factors, while it remains poorly understood whether other types of innovations are required. Here, we describe a new type of innovation, which escalates the defense capacity of the piRNA pathway to control a recently expanded selfish gene in Drosophila melanogaster. Through an in vivo RNAi screen for repressors of Stellate—a recently evolved and expanded selfish meiotic driver6–8—we discovered a novel defense factor, Trailblazer. Trailblazer is a transcription factor that promotes the expression of two PIWI-clade nucleases, Aub and AGO3, to match Stellate in abundance. Recent innovation in the DNA-binding domain of Trailblazer enabled it to drastically elevate Aub and AGO3 expression in the D. melanogaster lineage, thereby escalating the silencing capacity of the piRNA pathway to control expanded Stellate and safeguard fertility. As copy-number expansion is a recurrent feature of diverse selfish genes across the tree of life9–12, we envision that augmenting the defense capacity to quantitatively match selfish genes is likely a repeatedly employed defense strategy in evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc1b38a75ffc4d42a45a0066fa4c7fd09cdbd918" target='_blank'>
              Escalation of genome defense capacity enables control of an expanding meiotic driver
              </a>
            </td>
          <td>
            Peiwei Chen, Katherine C. Pan, Eunice H. Park, Yicheng Luo, Y. C. G. Lee, A. Aravin
          </td>
          <td>2024-06-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96ca99a18dd08dc5d59b7f900d05406ec6c8cd83" target='_blank'>
              Amplifications of EVX2 and HOXD9-HOXD13 on 2q31 in mature cystic teratomas of the ovary identified by array comparative genomic hybridization may explain teratoma characteristics in chondrogenesis and osteogenesis
              </a>
            </td>
          <td>
            Wen-Chung Wang, Tai-Cheng Hou, C. Kuo, Yen-Chein Lai
          </td>
          <td>2024-06-21</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Interphase mammalian genomes are folded in 3D with complex locus-specific patterns that impact gene regulation. CTCF (CCCTC-binding factor) is a key architectural protein that binds specific DNA sites, halts cohesin-mediated loop extrusion, and enables long-range chromatin interactions. There are hundreds of thousands of annotated CTCF-binding sites in mammalian genomes; disruptions of some result in distinct phenotypes, while others have no visible effect. Despite their importance, the determinants of which CTCF sites are necessary for genome folding and gene regulation remain unclear. Here, we update and utilize Akita, a convolutional neural network model, to extract the sequence preferences and grammar of CTCF contributing to genome folding. Our analyses of individual CTCF sites reveal four predictions: (i) only a small fraction of genomic sites are impactful, (ii) insulation strength is highly dependent on sequences flanking the core CTCF binding motif, (iii) core and flanking sequences are broadly compatible, and (iv) core and flanking nucleotides contribute largely additively to overall strength. Our analysis of collections of CTCF sites make two predictions for multi-motif grammar: (i) insulation strength depends on the number of CTCF sites within a cluster, and (ii) pattern formation is governed by the orientation and spacing of these sites, rather than any inherent specialization of the CTCF motifs themselves. In sum, we present a framework for using neural network models to probe the sequences instructing genome folding and provide a number of predictions to guide future experimental inquiries. Author Summary Mammalian genomes are spatially organized in 3D with profound consequences for all processes involving DNA. CTCF is a key genome organizer, recognizing numerous sites and creating a variety of contact patterns across the genome. Despite the importance of CTCF, the sequence determinants and grammar of how individual sites collectively instruct genome folding remain unclear. This work leverages the ability of Akita, a deep neural network, to make high-throughput predictions for genome folding after DNA sequence perturbations. Using Akita, we make several experimentally testable predictions. First, only a minority of annotated sites individually impact folding, and flanking DNA sequences greatly modulate their impact. Second, multiple sites together influence folding based on their number, orientation, and spacing. In sum, we provide a roadmap for interpreting neural networks to better understand genome folding and important considerations for the design of experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a8a450d11a02a4e12bb1863b2d5f19340a89cc" target='_blank'>
              Interpreting the CTCF-mediated sequence grammar of genome folding with AkitaV2
              </a>
            </td>
          <td>
            Paulina N. Smaruj, Fahad Kamulegeya, David R. Kelley, G. Fudenberg
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Purpose: Homologous recombination deficiency (HRD) impacts cancer treatment strategies, particularly the effectiveness of PARP inhibitors. However, the variability different HRD assays has hampered the selection of oncology patients who may benefit from these therapies. Our study aims to assess the whole genome landscape to better define HRD in a pan-cancer cohort and to contribute to harmonization of HRD detection. Methods: We employed a whole-genome sequencing WGS HRD classifier that included genome-wide features associated with HRD to analyze 580 tumor/normal paired pan-cancer samples. The HRD results were correlated retrospectively with treatment responses and were compared with commercial HRD tests in a subset of cases. Results: HRD phenotype was identified in 62 samples across various cancers including breast (19%), pancreaticobiliary (17%), gynecological (15%), prostate (8%), upper gastrointestinal (GI) (2%), and other cancers (1%). HRD cases were not confined to BRCA1/2 mutations; 24% of HRD cases were BRCA1/2 wild-type. A diverse range of HRR pathway gene alterations involved in HRD were elucidated, including biallelic mutations in FANCF, XRCC2, and FANCC, and deleterious structural variants. Comparison with results from commercial HRD assays suggests a better performance of WGS to detect HRD, based on treatment response. Conclusion: HRD is a biomarker used to determine which cancer patients would benefit from PARPi and platinum-based chemotherapy. However, a lack of harmonization of tests to determine HRD status makes it challenging to interpret their results. Our study highlights the use of comprehensive WGS analysis to predict HRD in a pan-cancer cohort, elucidates new genomic mechanisms associated with HRD, and enables an accurate identification of this phenotype, paving the way for improved outcomes in oncology care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80d347a4838be16d195accf2ef681565a4b6929" target='_blank'>
              Whole Genome Landscape Analysis of Homologous Recombination Deficiency in a Pan-Cancer Cohort
              </a>
            </td>
          <td>
            MD Majd Al Assaad, PhD Kevin Hadi, MS Max F. Levine, MD Daniela Guevara, Mph Minal Patel, MS Marvel Tranquille, BS Abigail King, BS John Otilano, MS Alissa Semaan, PhD Gunes Gundem, MS Juan S. Medina-Martínez, BS Michael Sigouros, Mph Jyothi Manohar, BS Hui-Hsuan Kuo, PhD David C. Wilkes, MD Eleni Andreopoulou, MD Eloise Chapman-Davis, MD Scott T. Tagawa, PhD Andrea Sboner, MD Allyson J. Ocean, MD Manish Shah, PhD Elli Papaemmanuil, Cora N. Sternberg, MD Kevin Holcomb, MD David M. Nanus, PhD Olivier Elemento, Juan Miguel Mosquera, J. Medina-Martinez
          </td>
          <td>2024-06-30</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bb0c25603cf78f65def294ef7f4472ac1c4808b" target='_blank'>
              Simultaneous de novo calling and phasing of genetic variants at chromosome-scale using NanoStrand-seq
              </a>
            </td>
          <td>
            Xiuzhen Bai, Zonggui Chen, Kexuan Chen, Zixin Wu, Rui Wang, Jun'e Liu, Liang Chang, Lu Wen, Fuchou Tang
          </td>
          <td>2024-07-09</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>80</td>
        </tr>

        <tr id="The myeloma overexpressed gene (MYEOV) has been proposed to be a proto-oncogene due to high RNA transcript levels found in multiple cancers, including myeloma, breast, lung, pancreas and esophageal cancer. The presence of an open reading frame (ORF) in humans and other primates suggests protein-coding potential. Yet, we still lack evidence of a functional MYEOV protein. It remains undetermined how MYEOV overexpression affects cancerous tissues. In this work, we show that MYEOV has likely originated and may still function as an enhancer, regulating CCND1 and LTO1. Firstly, MYEOV 3′ enhancer activity was confirmed in humans using publicly available ATAC-STARR-seq data, performed on B-cell-derived GM12878 cells. We detected enhancer histone marks H3K4me1 and H3K27ac overlapping MYEOV in multiple healthy human tissues, which include B cells, liver and lung tissue. The analysis of 3D genome datasets revealed chromatin interactions between a MYEOV-3′-putative enhancer and the proto-oncogene CCND1. BLAST searches and multi-sequence alignment results showed that DNA sequence from this human enhancer element is conserved from the amphibians/amniotes divergence, with a 273 bp conserved region also found in all mammals, and even in chickens, where it is consistently located near the corresponding CCND1 orthologues. Furthermore, we observed conservation of an active enhancer state in the MYEOV orthologues of four non-human primates, dogs, rats, and mice. When studying this homologous region in mice, where the ORF of MYEOV is absent, we not only observed an enhancer chromatin state but also found interactions between the mouse enhancer homolog and Ccnd1 using 3D-genome interaction data. This is similar to the interaction observed in humans and, interestingly, coincides with CTCF binding sites in both species. Taken together, this suggests that MYEOV is a primate-specific gene with a de novo ORF that originated at an evolutionarily older enhancer region. This deeply conserved putative enhancer element could regulate CCND1 in both humans and mice, opening the possibility of studying MYEOV regulatory functions in cancer using non-primate animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f17ce82b4897c471d6bebe571f550d964a45b5c" target='_blank'>
              Evolutionarily conserved enhancer-associated features within the MYEOV locus suggest a regulatory role for this non-coding DNA region in cancer
              </a>
            </td>
          <td>
            Brigid S. A. Davidson, Juliana Estefanía Arcila-Galvis, Marco Trevisan-Herraz, Aneta Mikulasova, C. Brackley, Lisa J Russell, Daniel Rico
          </td>
          <td>2024-07-30</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b0e09e628f8fcf9bb1ad3d19ff2d5112c171387" target='_blank'>
              Unmutated RRAS2 emerges as a key oncogene in post-partum-associated triple negative breast cancer
              </a>
            </td>
          <td>
            C. Cifuentes, Clara L. Oeste, Isabel Fernández-Pisonero, Alejandro Hortal, C. García-Macías, Jeanne Hochart, Regina Rubira, L. Horndler, Carlos Horndler, Xosé R. Bustelo, Balbino Alarcón
          </td>
          <td>2024-07-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="During the sexual cycle, programmed genome rearrangement (PGR) in Paramecium tetraurelia involves the non-homologous end joining (NHEJ) DNA repair pathway to eliminate specific germinal Internal Eliminated Sequences (IESs) from the newly developing somatic nucleus. In addition to the core NHEJ factors Ku70/80 and Xrcc4/Lig4, additional enzymes are required to process the 4-base 5’-protruding ends generated following DNA cleavage at IES boundaries, prior to their ligation. Here, we report that PolX (a,b,c,d), four P. tetraurelia distant orthologs of the human Polλ DNA polymerase, are involved in repair of IES excision junctions. During rearrangements, PolX-depleted cells accumulate genome-wide errors, such as unrepaired double-strand breaks, 1-nucleotide deletions and IES retention. Although all PolX paralogs can process DNA ends, two of them (PolXa&b) are induced during PGR and have acquired tight nuclear anchoring properties through their N-terminal region, which contains a predicted BRCT domain. Finally, we show that PolXa accumulates in nuclear foci together with other NHEJ proteins and the Dicer-like enzyme Dcl5, which is involved in the biogenesis of IES-specific small RNAs. We propose that these “DNA repair foci” correspond to the sites where IES concatemers, a by-product of IES excision, are ligated together to produce the precursors of iesRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8a893b216f167d132f42d3e505df3cd9a940a5" target='_blank'>
              A development-specific POLX essential for programmed genome rearrangement in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Baptiste Verron, O. Arnaiz, Coralie Zangarelli, Nathalie Mathy, M. Bétermier, J. Bischerour
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Medulloblastoma (MB) stands as one of the prevalent malignant brain tumors among pediatric patients. Despite its prevalence, the intricate interplay between the regulatory program driving malignancy in MB cells and their interactions with the microenvironment remains insufficiently understood. Leveraging the capabilities of single-cell Assay for Transposase-Accessible Chromatin sequencing (scATAC-seq), the chromatin accessibility landscape is unveiled across 59,015 distinct MB cells. This expansive dataset encompasses cells belonging to discrete molecular subgroups, namely SHH, WNT, Group3, and Group4. Within these chromatin accessibility profiles, specific regulatory elements tied to individual subgroups are uncovered, shedding light on the distinct activities of transcription factors (TFs) that likely orchestrate the tumorigenesis process. Moreover, it is found that certain neurotransmitter receptors (NTRs) are subgroup-specific and can predict MB subgroup classification when combined with their associated transcription factors. Notably, targeting essential NTRs within tumors influences both the in vitro sphere-forming capability and the in vivo tumorigenic capacity of MB cells. These findings collectively provide fresh insights into comprehending the regulatory networks and cellular dynamics within MBs. Furthermore, the significance of the TF-NTR regulatory circuits is underscored as prospective biomarkers and viable therapeutic targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/013b19ab545540365c4fc326bff03e9d828165b6" target='_blank'>
              Single-Cell Chromatin Accessibility Analysis Reveals Subgroup-Specific TF-NTR Regulatory Circuits in Medulloblastoma.
              </a>
            </td>
          <td>
            Xiaoyue Gao, Qiyuan Zhuang, Yun Li, Guochao Li, Zheng Huang, Shenzhi Chen, Shaoxing Sun, Hui Yang, Lan Jiang, Ying Mao
          </td>
          <td>2024-06-17</td>
          <td>Advanced science</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic highly polyploid macronucleus is lost. The formation of a new macronucleus involves Programmed Genome Rearrangements, introducing DNA DSBs at approximately 45,000 loci. P. tetraurelia employs a Non-Homologous End Joining (NHEJ)-related mechanism for the systematic repair of these DSBs. Four genes encoding DNA polymerases of family X are present in the genome, one of which was found recently to colocalize with other proteins of NHEJ. The question arises as to how they make almost no error. Here we show that these enzymes are most similar to metazoan DNA polymerase λ and exhibit high fidelity through two different molecular mechanisms. Using X-ray structure determination of polymerase lambda mutants recapitulating sequence determinants of P. tetraurelia PolXs, we find both a local conformational change that involves exchanging partners in a crucial salt bridge in the active site upon binding of correct dNTPs, and a larger conformational change involving the closure of Loop3. This stabilizes the template DNA in the active site, only in the presence of the correct incoming dNTP. Differences with human pol λ and pol β are discussed. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a2c8e3978ae53a388430b008c681d13e31a282" target='_blank'>
              Mechanisms Ensuring Fidelity of Family X DNA Polymerases in Programmed DNA rearrangements in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Antonin Nourisson, Sophia Missoury, A. Haouz, Marc Delarue
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="The genetic integrity of pluripotent stem cells (PSC) is integral to their applications in research and therapy, but it is compromised by frequent development of copy number variations. Little is known about the basis of the variable genomic integrity among different PSC lines. Here we identify aneuploidies using RNA-seq and proteomics data from a panel of mouse embryonic stem cell (mESC) lines derived from 170 Diversity Outbred mice. We identified 62 lines with detectable aneuploid subpopulations and a subset of originally XX lines that lost one Chromosome X (XO). Strikingly, a much lower proportion of XX lines were aneuploid, compared to XY or XO lines. Two single-cell RNA-seq data sets demonstrated that aneuploid XY DO mESC also show lower Chromosome X gene expression and that XY mESC accumulate higher aneuploid proportions in culture than isogenic XX lines. Possible mechanisms for this protective effect of X chromosome dosage include our discovery that the lines with two active X Chromosomes have a higher expression of 2-cell-like state genes, and differential expression of X-linked tumor suppressor genes associated with DNA damage response. Highlights First genetic analysis of predisposition to aneuploidy in pluripotent stem cell cultures Chromosomal regions duplicated in aneuploid mouse embryonic stem cells are syntenic with regions overrepresented in human pluripotent stem cell lines bearing recurrent genetic abnormalities X-Chromosome dosage strongly influences susceptibility to aneuploidy in mouse embryonic stem cells and to a lesser degree in human pluripotent stem cells XX mouse embryonic stem cell lines show a higher proportion of cells in 2 cell-like state and higher expression of tumor suppressor genes associated with DNA damage response">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f853ae6e95a31cb9ad3712a21b0838a6601fe33" target='_blank'>
              Chromosome X Dosage Modulates Development of Aneuploidy in Genetically Diverse Mouse Embryonic Stem Cells
              </a>
            </td>
          <td>
            Alexander Stanton, Selcan Aydin, Daniel A. Skelly, Dylan Stavish, Kim Leonhard, Seth M. Taapken, Erik M. McIntire, Matthew Pankratz, A. Czechanski, T. Ludwig, Ted Choi, Steve P. Gygi, Ivana Barbaric, Steven C. Munger, L. Reinholdt, Martin F. Pera
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Abstract The DNA‐dependent protein kinase (DNA‐PK), catalytic subunit, also known as DNA‐PKcs, is complexed with the heterodimer Ku70/Ku80 to form DNA‐PK holoenzyme, which is well recognized as initiator in the nonhomologous end joining (NHEJ) repair after double strand break (DSB). During NHEJ, DNA‐PKcs is essential for both DNA end processing and end joining. Besides its classical function in DSB repair, DNA‐PKcs also shows multifaceted functions in various biological activities such as class switch recombination (CSR) and variable (V) diversity (D) joining (J) recombination in B/T lymphocytes development, innate immunity through cGAS–STING pathway, transcription, alternative splicing, and so on, which are dependent on its function in NHEJ or not. Moreover, DNA‐PKcs deficiency has been proven to be related with human diseases such as neurological pathogenesis, cancer, immunological disorder, and so on through different mechanisms. Therefore, it is imperative to summarize the latest findings about DNA‐PKcs and diseases for better targeting DNA‐PKcs, which have shown efficacy in cancer treatment in preclinical models. Here, we discuss the multifaceted roles of DNA‐PKcs in human diseases, meanwhile, we discuss the progresses of DNA‐PKcs inhibitors and their potential in clinical trials. The most updated review about DNA‐PKcs will hopefully provide insights and ideas to understand DNA‐PKcs associated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc4d3c7ebac68c037dd494f0a2f6295735d5a4a5" target='_blank'>
              The multifaceted functions of DNA‐PKcs: implications for the therapy of human diseases
              </a>
            </td>
          <td>
            Jinghong Wu, Liwei Song, Mingjun Lu, Qing Gao, Shaofa Xu, Ping-Kun Zhou, Teng Ma
          </td>
          <td>2024-06-19</td>
          <td>MedComm</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Chromatin bridges are DNA strings that link anaphase poles or daughter nuclei and are associated with carcinogenesis. In response to chromatin bridges, mammalian cells delay abscission by activating the abscission checkpoint which delays completion of cytokinesis in order to prevent chromatin breakage or tetraploidization by furrow regression. We recently showed that in cytokinesis with chromatin bridges a biochemical pathway including MRN complex, ATM and Chk2 promotes INCENP localization to the midbody to delay abscission and prevent chromatin breakage. In this study we used site directed mutagenesis, RNA silencing, time lapse and confocal microscopy, FISH, BrdU labeling, TUNEL assay and differential retention assay. Here, we show that spontaneous or replication stress-induced chromatin bridges derived from catenated DNA exhibit “knots” of tangled, overtwisted DNA next to the midbody. Topoisomerase IIα (Top2a), an enzyme that relaxes supercoils and untangles catenated DNA forms irreversible Top2-DNA cleavage complexes (Top2ccs) on DNA knots. Inhibition of Top2a results in diminished localization of Rad17, MRN, ATM, Chk2 and CPC complex to the DNA bridges and induces chromatin bridge breakage. Furthermore, proteolytic degradation of Top2ccs is required for localization of Rad17 to the bridge DNA. In turn, Rad17 promotes recruitment of the MRN complex to DNA knots and downstream abscission checkpoint signaling to delay abscission and prevent chromatin bridge breakage in cytokinesis. In contrast, chromatin bridges generated by dicentric chromosomes do not exhibit DNA knots or Top2ccs next to the midbody, and fail to recruit Top2a, Rad17 and other downstream proteins and are unable to induce an abscission delay. Our results describe a novel mechanism by which the abscission checkpoint detects chromatin bridges in human cells, through generation of irreversible Top2ccs on DNA knots. Because chromosomal instability is a major cause of cancer, identifying new genes that protect genome integrity is very important for cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34285c04c5509bb4dc8d0ef9ffcf10003592c2a2" target='_blank'>
              A Novel Mechanism for Chromatin Bridge Sensing by the Abscission Checkpoint in Human Cells
              </a>
            </td>
          <td>
            Eleni Petsalaki, Sofia Balafouti, G. Zachos
          </td>
          <td>2024-07-01</td>
          <td>JCO Global Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cancer is a heterogeneous disease that results from genetic alteration of cell cycle and proliferation controls. Identifying mutations that drive cancer, understanding cancer type specificities, and delineating how driver mutations interact with each other to establish disease is vital for identifying therapeutic vulnerabilities. Such cancer specific patterns and gene co-occurrences can be identified by studying tumor genome sequences, and networks have proven effective in uncovering relationships between sequences. We present two network-based approaches to identify driver gene patterns among tumor samples. The first approach relies on analysis using the Directed Weighted All Nearest Neighbors (DiWANN) model, which is a variant of sequence similarity network, and the second approach uses bipartite network analysis. A data reduction framework was implemented to extract the minimal relevant information for the sequence similarity network analysis, where a transformed reference sequence is generated for constructing the driver gene network. This data reduction process combined with the efficiency of the DiWANN network model, greatly lowered the computational cost (in terms of execution time and memory usage) of generating the networks enabling us to work at a much larger scale than previously possible. The DiWANN network helped us identify cancer types in which samples were more closely connected to each other suggesting they are less heterogeneous and potentially susceptible to a common drug. The bipartite network analysis provided insight into gene associations and co-occurrences. We identified genes that were broadly mutated in multiple cancer types and mutations exclusive to only a few. Additionally, weighted one-mode gene projections of the bipartite networks revealed a pattern of occurrence of driver genes in different cancers. Our study demonstrates that network-based approaches can be an effective tool in cancer genomics. The analysis identifies co-occurring and exclusive driver genes and mutations for specific cancer types, providing a better understanding of the driver genes that lead to tumor initiation and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26d7d425039cbe06da7fb2e443336941bd0e0ae5" target='_blank'>
              Network analysis of driver genes in human cancers
              </a>
            </td>
          <td>
            S. S. Patil, Steven A. Roberts, A. Gebremedhin
          </td>
          <td>2024-07-08</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee6169884636c48723540d43c960017df0f71986" target='_blank'>
              Hidden secrets of the cancer genome: unlocking the impact of non-coding mutations in gene regulatory elements
              </a>
            </td>
          <td>
            Sandra Íñiguez-Muñoz, P. Llinàs-Arias, Miquel Ensenyat-Mendez, Andrés F. Bedoya-López, Javier I. J. Orozco, Javier Cortés, Ananya Roy, Karin Forsberg-Nilsson, M. DiNome, D. Marzese
          </td>
          <td>2024-06-20</td>
          <td>Cellular and Molecular Life Sciences</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Neuro-Oncological Ventral Antigen 1 (NOVA1) is best known for its role in mediating an alternative splicing (AS) program in neurons, yet was first discovered as an antigen expressed in breast tumors, causing rare autoimmune reactions and paraneoplastic neurological disorders (PNDs). The PND model suggests a plausible role of the tumor antigen expression in tumor suppression, whereas it has emerged that NOVA may function as an oncogene in a variety of cancers. In addition, whether NOVA mediates AS in breast cancer remains unanswered. Here we examine the AS profiles of breast invasive carcinoma (BRCA) tumor samples and demonstrate that ectopic NOVA1 expression led to the activation of neuron-like splicing patterns in many genes, including exons targeted by NOVA in the brain. The splicing dysregulation is especially prevalent in cell periphery and cytoskeleton genes related to cell-cell communication, actin-based movement, and neuronal functions. We find that NOVA1-mediated AS is most prominent in Luminal A tumors and high NOVA1 expression in this subtype is associated with poorer prognosis. Our results suggest that ectopic NOVA1 in tumors has regulatory activity affecting pathways with high relevance to tumor progression and that this might be a more general mechanism for PND antigens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e50ff50fd94e0098cbe1b6b46456df93e0c5100" target='_blank'>
              Oncogenic NOVA1 expression dysregulates alternative splicing in breast cancer
              </a>
            </td>
          <td>
            Daniel F. Moakley, Chaolin Zhang
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Each piece of cell-free DNA (cfDNA) has a length determined by the exact metabolic conditions in the cell it belonged to at the time of cell death. The changes in cellular regulation leading to a variety of patterns, which are based on the different number of fragments with lengths up to several hundred base pairs at each of the almost three billion genomic positions, allow for the detection of disease and also the precise identification of the tissue of their origin. A Kullback-Leibler (KL) divergence computation identifies different fragment lengths and areas of the human genome, depending on the stage, for which disease samples, starting from pre-clinical disease stages, diverge from healthy donor samples. We provide examples of genes related to colorectal cancer (CRC), which our algorithm detected to belong to divergent genomic bins. The staging of CRC can be viewed as a Markov Chain and that provides a framework for studying disease progression and the types of epigenetic changes occurring longitudinally at each stage, which might aid the correct classification of a new hospital sample. In a new look to treat such data as grayscale value images, pattern recognition using artificial intelligence (AI) could be one approach to classification. In CRC, Stage I disease does not, for the most part, shed any tumor circulation, making detection difficult for established machine learning (ML) methods. This leads to the deduction that early detection, where we can only rely on changes in the metabolic patterns, can be accomplished when the information is considered in its entirety, for example by applying computer vision methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769c868029bf18fe644fcae9a2141c21f2b5a763" target='_blank'>
              Channel Capacity of Genome-Wide Cell-Free DNA Fragment Length Distribution in Colorectal Cancer
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-07-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Germline mutations have been identified in a small number of hereditary cancers, but the genetic predisposition for many familial cancers remains to be elucidated. Methods This study identified a Chinese pedigree that presented different cancers (breast cancer, BRCA; adenocarcinoma of the esophagogastric junction, AEG; and B‐cell acute lymphoblastic leukemia, B‐ALL) in each of the three generations. Whole‐genome sequencing and whole‐exome sequencing were performed on peripheral blood or bone marrow and cancer biopsy samples. Whole‐genome bisulfite sequencing was conducted on the monozygotic twin brothers, one of whom developed B‐ALL. Results According to the ACMG guidelines, bioinformatic analysis of the genome sequencing revealed 20 germline mutations, particularly mutations in the DNAH11 (c.9463G > A) and CFH (c.2314G > A) genes that were documented in the COSMIC database and validated by Sanger sequencing. Forty‐one common somatic mutated genes were identified in the cancer samples, displaying the same type of single nucleotide substitution Signature 5. Meanwhile, hypomethylation of PLEK2, MRAS, and RXRA as well as hypermethylation of CpG island associated with WT1 was shown in the twin with B‐ALL. Conclusions These findings reveal genomic alterations in a pedigree with multiple cancers. Mutations found in the DNAH11, CFH genes, and other genes predispose to malignancies in this family. Dysregulated methylation of WT1, PLEK2, MRAS, and RXRA in the twin with B‐ALL increases cancer susceptibility. The similarity of the somatic genetic changes among the three cancers indicates a hereditary impact on the pedigree. These familial cancers with germline and somatic mutations, as well as epigenomic alterations, represent a common molecular basis for many multiple cancer pedigrees.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3f0ae27a463e75a2f599e816c3a044e8b7c954" target='_blank'>
              Impact of genomic and epigenomic alterations of multigene on a multicancer pedigree
              </a>
            </td>
          <td>
            Jinyu Gao, Yongzhang Wu, Jieming Yu, Yinbin Qiu, Tiantian Yi, Chaochao Luo, Junxiao Zhang, Gary Lu, Xu Li, Fu Xiong, Xuedong Wu, Xinghua Pan
          </td>
          <td>2024-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, Gavin S Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b87ee25b47be8c0f0bda889f181d3721c7eba73b" target='_blank'>
              DNA packaging by molecular motors: from bacteriophage to human chromosomes.
              </a>
            </td>
          <td>
            Bram Prevo, William C. Earnshaw
          </td>
          <td>2024-06-17</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64f1aced03eb2794aa498d58e60fbc81d9cded4d" target='_blank'>
              Homeobox and Polycomb target gene methylation in human solid tumors
              </a>
            </td>
          <td>
            Reid Blanchett, Kin H. Lau, Gerd P Pfeifer
          </td>
          <td>2024-06-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acdebafa769f861e73d2cd78d8c49c2d3b69320" target='_blank'>
              USP53 Affects the Proliferation and Apoptosis of Breast Cancer Cells by Regulating the Ubiquitination Level of ZMYND11
              </a>
            </td>
          <td>
            Xiangchao Meng, Hongye Chen, Zhihui Tan, Weitao Yan, Yinfeng Liu, Ji Lv, Meng Han
          </td>
          <td>2024-07-23</td>
          <td>Biological Procedures Online</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute lymphoblastic leukemia (ALL) is the most common cancer in children, yet few environmental risk factors have been identified. We previously found an association between early-life tobacco smoke exposure and frequency of somatic deletions of 8 leukemia driver genes among childhood ALL patients in the California Childhood Leukemia Study. To expand analysis genome-wide and examine potential mechanisms, we conducted tumor whole-genome sequencing in 35 ALL patients, including 18 with high prenatal tobacco exposure and 17 with low exposure as determined by established epigenetic biomarkers. High tobacco exposure patients had significantly more structural variants (P < .001) and deletions (P = .001) genome-wide than low exposure patients. Investigation of off-target RAG recombination revealed that 41% of deletions in the high tobacco exposure patients were putatively RAG-mediated (full RAG motif identified at one or both breakpoints) compared with only 21% in the low exposure group (P = .001). In a multilevel model, deletions in high tobacco exposure patients were 2.44-fold (95% CI:1.13–5.38) more likely to be putatively RAG-mediated than deletions in low exposure patients. No point mutational signatures were associated with prenatal tobacco exposure. Our findings suggest that early-life tobacco smoke exposure may promote leukemogenesis by driving development of somatic deletions in pre-leukemic lymphocytes via off-target RAG recombination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bd1dd9d24470ae77cf6358a81bf0ca861d75dad" target='_blank'>
              Early-life tobacco exposure is causally implicated in aberrant RAG-mediated recombination in childhood acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            A. D. de Smith, Tanxin Liu, Keren Xu, A. Pardeshi, S. Myint, Alice Y Kang, L. Morimoto, Michael Lieber, Joseph L Wiemels, Scott Kogan, C. Metayer
          </td>
          <td>2024-06-20</td>
          <td>Research Square</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facf4dda707b67b3b56d646f95496db5b6cc7df8" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N. Anderson
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) is increasingly studied for its diverse applications in non-invasive detection. Non-randomly cleaved by nucleases and released into the bloodstream, cfDNA exhibits a variety of intrinsic fragmentation patterns indicative of cell status. Particularly, these fragmentation patterns have recently been demonstrated to be effective in predicting cancer and its tissue-of-origin, owing to increased variation of fragmentation features observed in tumor patients. However, there remains a lack of detailed exploration of altered cfDNA fragmentation profiles in tumors, which consist of a mixture of both non-tumor cfDNA and circulating tumor DNA (ctDNA). Hence, we leveraged the human tumor cell line-derived xenograft (CDX) mouse model, where different tumor cell lines were implanted into different anatomical sites, to isolate pure ctDNA and separately investigate the fragment properties of CDX-induced cfDNA and ctDNA. We found an enrichment of short cfDNA fragments in both CDX-induced cfDNA and ctDNA compared to normal plasma cfDNA, with more elevated short fragments in ctDNA. Moreover, the CDX-induced cfDNA fragmentation features distinguished between CDX models of different tumor cell lines, while the ctDNA fragmentation features conversely discriminate between CDX models of different anatomical sites. The results suggested that both non-tumor cfDNA and ctDNA contribute to the increased variation observed in tumors, and that cfDNA fragmentation may be highly variable and susceptible to regulations by both original cells and cells within the local niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6138f9d167b66612f618b5e26b115af024b7013f" target='_blank'>
              Dissecting cell-free DNA fragmentation variation in tumors using cell line-derived xenograft mouse
              </a>
            </td>
          <td>
            Ruiqing Fu, He Amy Su, Yafei Tian, Hongyan Chen, Daru Lu
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Functional copy-number alterations (fCNAs) are DNA copy-number changes with concordant differential gene expression. These are less likely to be bystander genetic lesions and could serve as robust and reproducible tumor biomarkers. To identify candidate fCNAs in neuroendocrine tumors (NETs), we integrated chromosomal microarray (CMA) and RNA-seq differential gene-expression data from 31 pancreatic (pNETs) and 33 small-bowel neuroendocrine tumors (sbNETs). Tumors were resected from 47 early-disease-progression (<24 months) and 17 late-disease-progression (>24 months) patients. Candidate fCNAs that accurately differentiated these groups in this discovery cohort were then replicated using fluorescence in situ hybridization (FISH) on formalin-fixed, paraffin-embedded (FFPE) tissues in a larger validation cohort of 60 pNETs and 82 sbNETs (52 early- and 65 late-disease-progression samples). Logistic regression analysis revealed the predictive ability of these biomarkers, as well as the assay-performance metrics of sensitivity, specificity, and area under the curve. Our results indicate that copy-number changes at chromosomal loci 4p16.3, 7q31.2, 9p21.3, 17q12, 18q21.2, and 19q12 may be used as diagnostic and prognostic NET biomarkers. This involves a rapid, cost-effective approach to determine the primary tumor site for patients with metastatic liver NETs and to guide risk-stratified therapeutic decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/976250a22dce56b41eaaf1761c3d5be047660920" target='_blank'>
              Functional Copy-Number Alterations as Diagnostic and Prognostic Biomarkers in Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Hayley Vaughn, Heather Major, Evangeline Kadera, Kendall Keck, Timothy Dunham, Qining Qian, Bartley Brown, Aaron Scott, Andrew M. Bellizzi, Terry A. Braun, Patrick Breheny, Dawn E. Quelle, James R. Howe, Benjamin W Darbro
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Bacterial chromosomes are large molecules that need to be highly compacted to fit inside the cells. Chromosome compaction must facilitate and maintain key biological processes such as gene expression and DNA transactions (replication, recombination, repair, and segregation). Chromosome and chromatin 3D-organization in bacteria has been a puzzle for decades. Chromosome conformation capture coupled to deep sequencing (Hi-C) in combination with other "omics" approaches has allowed dissection of the structural layers that shape bacterial chromosome organization, from DNA topology to global chromosome architecture. Here we review the latest findings using Hi-C and discuss the main features of bacterial genome folding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d47cf5eae543114153f80223459365d3c29d31d5" target='_blank'>
              Principles of bacterial genome organization, a conformational point of view.
              </a>
            </td>
          <td>
            Sokrich Ponndara, Mounia Kortebi, F. Boccard, S. Bury-Moné, Virginia S. Lioy
          </td>
          <td>2024-06-22</td>
          <td>Molecular microbiology</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Transcription and splicing errors in cancer cells generate erroneous transcripts. Since erroneous transcripts are degraded by the nonsense-mediated mRNA decay (NMD) pathway, whether they are junk or could be functional has been overlooked and understudied. We addressed this question by first performing a pan-cancer analysis and identified substantial erroneous noncoding transcripts (ENT) in cancers. Given that RNA/DNA binding domains (DBD) were predicted in ENTs, we deleted predicted DBDs in multiple ENTs in multiple cell lines, with RNA-sequencing and cell experiments before and after DBD deletion. DBD deletion caused significantly changed expression of ENTs’ target genes (whose promoter regions contain ENTs’ DNA binding sites, DBS) and changed cell migration and proliferation ability, indicating that many ENTs can transcriptionally regulate genes. Tightly coupled data analysis and experiments reveal that ENTs’ functions are highly cancer- and cellular-context specific, making ENTs a new class of safe and specific targets for noncoding RNA-based cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e45af8d1c765a560ff555893aa2ed200e47f3c" target='_blank'>
              Many erroneous noncoding transcripts in cancer cells can highly specifically regulate cancer-related genes and pathways
              </a>
            </td>
          <td>
            Sha He, Wei Xiong, Jianping Huo, Jie Lin, Jianmin Li, Hao Zhu
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Invasive mucinous adenocarcinoma (IMA) comprises ∼5% of lung adenocarcinoma. There is no effective therapy for IMA when surgical resection is not possible. IMA is sometimes confused with adenocarcinoma with signet ring cell features (SRCC) pathologically since both adenocarcinomas feature tumor cells with abundant intracellular mucin. The molecular mechanisms by which such mucin-producing lung adenocarcinomas develop remain unknown. Methods Using a Visium spatial transcriptomics approach, we analyzed IMA and compared it with SRCC patho-transcriptomically. Combining spatial transcriptomics data with in vitro studies using RNA-seq and ChIP-seq, we assessed downstream targets of transcription factors HNF4A and SPDEF that are highly expressed in IMA and/or SRCC. Results Spatial transcriptomics analysis indicated that there are 6 distinct cell clusters in IMA and SRCC. Notably, two clusters (C1 and C3) of mucinous tumor cells exist in both adenocarcinomas albeit at a different ratio. Importantly, a portion of genes (e.g., NKX2-1, GKN1, HNF4A and FOXA3) are distinctly expressed while some mucous-related genes (e.g., SPDEF and FOXA2) are expressed in both adenocarcinomas. We determined that HNF4A induces MUC3A/B and TM4SF4 and that BI 6015, an HNF4A antagonist, suppressed the growth of IMA cells. Using mutant SPDEF that is associated with COVID-19, we also determined that an intact DNA-binding domain of SPDEF is required for SPDEF-mediated induction of mucin genes (MUC5AC, MUC5B and AGR2). Additionally, we found that XMU-MP-1, a SPDEF inhibitor, suppressed the growth of IMA cells. Conclusion These results revealed that IMA and SRCC contain heterogenous tumor cell types, some of which are targetable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/405c78189e11982c1b68c2b274f03ca88f30d620" target='_blank'>
              Patho-transcriptomic analysis of invasive mucinous adenocarcinoma of the lung (IMA): comparison with lung adenocarcinoma with signet ring cell features (SRCC)
              </a>
            </td>
          <td>
            William D. Stuart, M. Ito, Iris-Fink Baldauf, T. Fukazawa, T. Yamatsuji, Tomoshi Tsuchiya, Hideo Watanabe, Morihito Okada, Eric L. Snyder, M. Mino-Kenudson, M. Guo, Yutaka Maeda
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Maintenance of genome stability is of paramount importance for the survival of an organism. However, genomic integrity is constantly being challenged by various endogenous and exogenous processes that damage DNA. Therefore, cells are heavily reliant on DNA repair pathways that have evolved to deal with every type of genotoxic insult that threatens to compromise genome stability. Notably, inherited mutations in genes encoding proteins involved in these protective pathways trigger the onset of disease that is driven by chromosome instability e.g. neurodevelopmental abnormalities, neurodegeneration, premature ageing, immunodeficiency and cancer development. The ability of cells to regulate the recruitment of specific DNA repair proteins to sites of DNA damage is extremely complex but is primarily mediated by protein post-translational modifications (PTMs). Ubiquitylation is one such PTM, which controls genome stability by regulating protein localisation, protein turnover, protein-protein interactions and intra-cellular signalling. Over the past two decades, numerous ubiquitin (Ub) E3 ligases have been identified to play a crucial role not only in the initiation of DNA replication and DNA damage repair but also in the efficient termination of these processes. In this review, we discuss our current understanding of how different Ub E3 ligases (RNF168, TRAIP, HUWE1, TRIP12, FANCL, BRCA1, RFWD3) function to regulate DNA repair and replication and the pathological consequences arising from inheriting deleterious mutations that compromise the Ub-dependent DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d67379a570ccc6355d856b2e241bb15b1ce2e1c" target='_blank'>
              E3 ligases: a ubiquitous link between DNA repair, DNA replication and human disease.
              </a>
            </td>
          <td>
            Anoop S. Chauhan, Satpal S. Jhujh, Grant S Stewart
          </td>
          <td>2024-07-10</td>
          <td>The Biochemical journal</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Precise DNA replication is fundamental for genetic inheritance. In eukaryotes, replication initiates at multiple origins that are first "licensed" and subsequently "fired" to activate DNA synthesis. Despite the success in identifying origins with specific DNA motifs in Saccharomyces cerevisiae, no consensus sequence or sequences with a predictive value of replication origins have been recognized in metazoan genomes. Rather, epigenetic rules and chromatin structures are believed to play important roles in governing the selection and activation of replication origins. We propose that replication initiation is facilitated by a group of sequence-specific "replication pioneer factors," which function to increase chromatin accessibility and foster a chromatin environment that is conducive to the loading of the prereplication complex. Dysregulation of the function of these factors may lead to gene duplication, genomic instability, and ultimately the occurrence of pathological conditions such as cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/43129f2ef7735b3992e0d45fec31beb44fb71b6c" target='_blank'>
              Pioneer factors for DNA replication initiation in metazoans.
              </a>
            </td>
          <td>
            Yue Wang, Jing Liang
          </td>
          <td>2024-06-16</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, Bert van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Non-SMC condensin I complex subunit D2 (NCAPD2) is a newly identified oncogene; however, the specific biological function and molecular mechanism of NCAPD2 in liver cancer progression remain unknown. In the present study, the aberrant expression of NCAPD2 in liver cancer was investigated using public tumor databases, including TNMplot, The Cancer Genome Atlas and the International Cancer Genome Consortium based on bioinformatics analyses, and it was validated using a clinical cohort. It was revealed that NCAPD2 was significantly upregulated in liver cancer tissues compared with in control liver tissues, and NCAPD2 served as an independent prognostic factor and predicted poor prognosis in liver cancer. In addition, the expression of NCAPD2 was positively correlated with the percentage of Ki67+ cells. Finally, single-cell sequencing data, gene-set enrichment analyses and in vitro investigations, including cell proliferation assay, Transwell assay, wound healing assay, cell cycle experiments, cell apoptosis assay and western blotting, were carried out in human liver cancer cell lines to assess the biological mechanisms of NCAPD2 in patients with liver cancer. The results revealed that the upregulation of NCAPD2 enhanced tumor cell proliferation, invasion and cell cycle progression at the G2/M-phase transition, and inhibited apoptosis in liver cancer cells. Furthermore, NCAPD2 overexpression was closely associated with the phosphatidylinositol 3-kinase (PI3K)-Akt-mammalian target of rapamycin (mTOR)/c-Myc signaling pathway and epithelial-mesenchymal transition (EMT) progression in HepG2 and Huh7 cells. In addition, upregulated NCAPD2 was shown to have adverse effects on overall survival and disease-specific survival in liver cancer. In conclusion, the overexpression of NCAPD2 was shown to lead to cell cycle progression at the G2/M-phase transition, activation of the PI3K-Akt-mTOR/c-Myc signaling pathway and EMT progression in human liver cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6873a90e35d5a8f62be3a51b8f0dd6c3ab4c961" target='_blank'>
              NCAPD2 augments the tumorigenesis and progression of human liver cancer via the PI3K-Akt-mTOR signaling pathway
              </a>
            </td>
          <td>
            Jiangxue Gu, Ke Huang, Wei-Lin Zhao, Xiao-Ming Zheng, Yu-Qin Wu, Shirong Yan, Yu-Gang Huang, Pei Hu
          </td>
          <td>2024-07-31</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e6bae3255c905d58e9f2d6fd8dca01970a1953f" target='_blank'>
              Chromatin conformation and histone modification profiling across human kidney anatomic regions
              </a>
            </td>
          <td>
            Haikuo Li, Dian Li, Benjamin D. Humphreys
          </td>
          <td>2024-07-18</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a190a83f30593fab67bd2e0b946b6e5dcca9e7" target='_blank'>
              STAU1-mediated CNBP mRNA degradation by LINC00665 alters stem cell characteristics in ovarian cancer
              </a>
            </td>
          <td>
            Xiaofang Liu, Yang Chen, Ying Li, Jinling Bai, Zhi Zeng, Min Wang, Yaodong Dong, Yingying Zhou
          </td>
          <td>2024-07-30</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer (mCRPC) is a lethal disease that resists therapy targeting androgen signaling, the primary driver of prostate cancer. mCRPC resists androgen receptor (AR) inhibitors by amplifying AR signaling or by evolving into therapy-resistant subtypes that do not depend on AR. Elucidation of the epigenetic underpinnings of these subtypes could provide important insights into the drivers of therapy resistance. In this study, we produced chromatin accessibility maps linked to the binding of lineage-specific transcription factors (TF) by performing ATAC sequencing on 70 mCRPC tissue biopsies integrated with transcriptome and whole genome sequencing. mCRPC had a distinct global chromatin accessibility profile linked to AR function. Analysis of TF occupancy across accessible chromatin revealed 203 TFs associated with mCRPC subtypes. Notably, ZNF263 was identified as a putative prostate cancer TF with a significant impact on gene activity in the double-negative (AR- neuroendocrine-) subtype, potentially activating MYC targets. Overall, this analysis of chromatin accessibility in mCRPC provides valuable insights into epigenetic changes that occur during progression to mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac71bf183535cdb84fd367ef22969e3b56650628" target='_blank'>
              An Atlas of Accessible Chromatin in Advanced Prostate Cancer Reveals the Epigenetic Evolution during Tumor Progression.
              </a>
            </td>
          <td>
            Raunak Shrestha, L. Chesner, Meng Zhang, Stanley Zhou, A. Foye, A. Lundberg, Alana S Weinstein, M. Sjöström, Xiaolin Zhu, Thaidy Moreno-Rodriguez, Haolong Li, Su C/Pcf West Coast Prostate Cancer Dream Team, J. Alumkal, Rahul Aggarwal, Eric J Small, Mathieu Lupien, David A Quigley, Felix Y Feng
          </td>
          <td>2024-07-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="MYC has been implicated in the pathogenesis of a wide range of human tumors and has been described for many years as a transcription factor that regulates genes with pleiotropic functions to promote tumorigenic growth. However, despite extensive efforts to identify specific target genes of MYC that alone could be responsible for promoting tumorigenesis, the field is yet to reach a consensus whether this is the crucial function of MYC. Recent work shifts the view on MYC's function from being a gene-specific transcription factor to an essential stress resilience factor. In highly proliferating cells, MYC preserves cell integrity by promoting DNA repair at core promoters, protecting stalled replication forks, and/or preventing transcription-replication conflicts. Furthermore, an increasing body of evidence demonstrates that MYC not only promotes tumorigenesis by driving cell-autonomous growth, but also enables tumors to evade the host's immune system. In this review, we summarize our current understanding of how MYC impairs antitumor immunity and why this function is evolutionarily hard-wired to the biology of the MYC protein family. We show why the cell-autonomous and immune evasive functions of MYC are mutually dependent and discuss ways to target MYC proteins in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877f0298df11cd9063e84174a42a5eb6afca2c71" target='_blank'>
              Immune evasion: An imperative and consequence of MYC deregulation.
              </a>
            </td>
          <td>
            Bastian Krenz, Jongkuen Lee, Toshitha Kannan, Martin Eilers
          </td>
          <td>2024-07-02</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is a particularly aggressive mammary neoplasia with a high fatality rate, mainly because of the development of resistance to administered chemotherapy, the standard treatment for this disease. In this study, we employ both bulk RNA-sequencing and single-cell RNA-sequencing (scRNA-seq) to investigate the transcriptional landscape of TNBC cells cultured in two-dimensional monolayers or three-dimensional spheroids, before and after developing resistance to the chemotherapeutic agents paclitaxel and doxorubicin. Our findings reveal significant transcriptional heterogeneity within the TNBC cell populations, with the scRNA-seq identifying rare subsets of cells that express resistance-associated genes not detected by the bulk RNA-seq. Furthermore, we observe a partial shift towards a highly mesenchymal phenotype in chemoresistant cells, suggesting the epithelial-to-mesenchymal transition (EMT) as a prevalent mechanism of resistance in subgroups of these cells. These insights highlight potential therapeutic targets, such as the PDGF signaling pathway mediating EMT, which could be exploited in this setting. Our study underscores the importance of single-cell approaches in understanding tumor heterogeneity and developing more effective, personalized treatment strategies to overcome chemoresistance in TNBC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33e0aae4bc7f6edfa5e8ffddd5c3bb7acfad4a59" target='_blank'>
              Identification of New Chemoresistance-Associated Genes in Triple-Negative Breast Cancer by Single-Cell Transcriptomic Analysis
              </a>
            </td>
          <td>
            S. Foutadakis, Dimitrios Kordias, G. Vatsellas, A. Magklara
          </td>
          <td>2024-06-22</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Cancer genomic studies have identified frequent alterations in components of the SWI/SNF (SWItch/Sucrose Non- Fermenting) chromatin remodeling complex including SMARCA4 and ARID1A. Importantly, clinical reports indicate that SMARCA4-mutant lung cancers respond poorly to immunotherapy and have dismal prognosis. However, the mechanistic basis of immunotherapy resistance is unknown. Here, we corroborated the clinical findings by using immune-humanized, syngeneic, and genetically engineered mouse models of lung cancer harboring SMARCA4 deficiency. Specifically, we show that SMARCA4 loss caused decreased response to anti-PD1 immunotherapy associated with significantly reduced infiltration of dendritic cells (DCs) and CD4+ T cells into the tumor microenvironment (TME). Mechanistically, we show that SMARCA4 loss in tumor cells led to profound downregulation of STING, IL1β and other components of the innate immune system as well as inflammatory cytokines that are required for efficient recruitment and activity of immune cells. We establish that this deregulation of gene expression is caused by cancer cell-intrinsic reprogramming of the enhancer landscape with marked loss of chromatin accessibility at enhancers of genes involved in innate immune response such as STING, IL1β, type I IFN and inflammatory cytokines. Interestingly, we observed that transcription factor NF-κB binding motif was highly enriched in enhancers that lose accessibility upon SMARCA4 deficiency. Finally, we confirmed that SMARCA4 and NF-κB co-occupy the same genomic loci on enhancers associated with STING and IL1β, indicating a functional interplay between SMARCA4 and NF-κB. Taken together, our findings provide the mechanistic basis for the poor response of SMARCA4-mutant tumors to anti-PD1 immunotherapy and establish a functional link between SMARCA4 and NF-κB on innate immune and inflammatory gene expression regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3fdeb7ebdd844e1757256a995b264456319e406" target='_blank'>
              SMARCA4 mutation induces tumor cell-intrinsic defects in enhancer landscape and resistance to immunotherapy
              </a>
            </td>
          <td>
            Yawen Wang, Ismail M. Meraz, Md Qudratullah, Sasi Kotagiri, Yanyan Han, Yuanxin Xi, Jing Wang, Yonathan Lissanu
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Homologous recombination rearranges genetic information during meiosis to generate new combinations of variants. Recombination also causes new mutations, affects the GC content of the genome and reduces selective interference. Here, we use HiFi long-read sequencing to directly detect crossover and gene conversion events from switches between the two haplotypes along single HiFi-reads from testis tissue of humans, chimpanzees and gorillas as well as human sperm samples. Furthermore, based on DNA methylation calls, we classify the cellular origin of reads to either somatic or germline cells in the testis tissue. We identify 1692 crossovers and 1032 gene conversions in nine samples and investigate their chromosomal distribution. Crossovers are more telomeric and correlate better with recombination maps than gene conversions. We show a strong concordance between a human double-strand break map and the human samples, but not for the other species, supporting different PRDM9-programmed double-strand break loci. We estimate the average gene conversion tract lengths to be similar and very short in all three species (means 40-100 bp, fitted well by a geometric distribution) and that 95-98% of non-crossover events do not involve tracts intersecting with polymorphism and are therefore not detectable. Finally, we detect a GC bias in the gene conversion of both single and multiple SNVs and show that the GC-biased gene conversion affects SNVs flanking crossover events. This implies that gene conversion events associated with crossover events are much longer (estimated above 500 bp) than those associated with non-crossover events. Highly accurate long-read sequencing combined with the classification of reads to specific cell types provides a new, powerful way to make individual, detailed maps of gene conversion and crossovers for any species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e343940af051e78f3a88674b4f23d5e3391f5fb" target='_blank'>
              Insights into gene conversion and crossing-over processes from long-read sequencing of human, chimpanzee and gorilla testes and sperm
              </a>
            </td>
          <td>
            Peter Soerud Porsborg, A. P. Charmouh, Vinod Kumar Singh, S. B. Winge, C. Hvilsom, Marta Pelizzola, S. Laurentino, N. Neuhaus, A. Hobolth, Thomas Bataillon, Kristian Almstrup, Søren Besenbacher, M. Schierup
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8e9f04f74a6159b9e49f1e14a0705e366ea67c" target='_blank'>
              Epigenetic alterations affecting hematopoietic regulatory networks as drivers of mixed myeloid/lymphoid leukemia
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Stanley van Herk, Margit Nuetzel, Aniko Sijs-Szabo, Noelia Díaz, Katherine Kelly, Claudia A. J. Erpelinck-Verschueren, Lucia Schwarzfischer-Pfeilschifter, Hanna Stanewsky, Ute Ackermann, Dagmar Glatz, Johanna Raithel, A. Fischer, Sandra Pohl, Anita Rijneveld, Juan M. Vaquerizas, Christian Thiede, C. Plass, Bas J. Wouters, Ruud Delwel, Michael Rehli, C. Gebhard
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="With age, hematopoietic stem cells can acquire somatic mutations in leukemogenic genes that confer a proliferative advantage in a phenomenon termed “clonal hematopoiesis of indeterminate potential” (CHIP). How these mutations confer a proliferative advantage and result in increased risk for numerous age-related diseases remains poorly understood. We conducted a multiracial meta-analysis of epigenome-wide association studies (EWAS) of CHIP and its subtypes in four cohorts (N=8196) to elucidate the molecular mechanisms underlying CHIP and illuminate how these changes influence cardiovascular disease risk. The EWAS findings were functionally validated using human hematopoietic stem cell (HSC) models of CHIP. A total of 9615 CpGs were associated with any CHIP, 5990 with DNMT3A CHIP, 5633 with TET2 CHIP, and 6078 with ASXL1 CHIP (P <1×10−7). CpGs associated with CHIP subtypes overlapped moderately, and the genome-wide DNA methylation directions of effect were opposite for TET2 and DNMT3A CHIP, consistent with their opposing effects on global DNA methylation. There was high directional concordance between the CpGs shared from the meta-EWAS and human edited CHIP HSCs. Expression quantitative trait methylation analysis further identified transcriptomic changes associated with CHIP-associated CpGs. Causal inference analyses revealed 261 CHIP-associated CpGs associated with cardiovascular traits and all-cause mortality (FDR adjusted p-value <0.05). Taken together, our study sheds light on the epigenetic changes impacted by CHIP and their associations with age-related disease outcomes. The novel genes and pathways linked to the epigenetic features of CHIP may serve as therapeutic targets for preventing or treating CHIP-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb071f9470db1b0d43e08fc0664a4d81582bdd97" target='_blank'>
              Epigenome-wide DNA Methylation Association Study of CHIP Provides Insight into Perturbed Gene Regulation
              </a>
            </td>
          <td>
            Daniel Levy, Sara Kirmani, T. Huan, Joseph Van Amburg, R. Joehanes, M. Uddin, N. Nguyen, Bing Yu, Jennifer A. Brody, M. Fornage, Jan Bressler, N. Sotoodehnia, David Ong, Fabio Puddu, James Floyd, Christie M. Ballantyne, B. Psaty, L. Raffield, Pradeep Natarajan, Karen Conneely, April P. Carson, Leslie Lange, Kendra Ferrier, N. HEARD-COSTA, Joanne M Murabito, A. Bick
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Single-strand breaks (SSBs) are one of the most common endogenous lesions and have the potential to give rise to cytotoxic double-strand breaks (DSBs) during DNA replication. To investigate the mechanism of replication fork collapse at SSBs and subsequent repair, we employed Cas9 nickase (nCas9) to generate site and strand-specific nicks in the budding yeast genome. We show that nCas9-induced nicks are converted to mostly double-ended DSBs during S-phase. We find that repair of replication-dependent DSBs requires homologous recombination (HR) and is independent of canonical non-homologous end joining. Consistent with a strong bias to repair these lesions using a sister chromatid template, we observe minimal induction of inter-chromosomal HR by nCas9. Using nCas9 and a gRNA to nick either the leading or lagging strand template, we carried out a genome-wide screen to identify factors necessary for the repair of replication-dependent DSBs. All the core HR genes were recovered in the screen with both gRNAs, but we recovered components of the replication-coupled nucleosome assembly (RCNA) pathway with only the gRNA targeting the leading strand template. By use of additional gRNAs, we find that the RCNA pathway is especially important to repair a leading strand fork collapse.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bb3a784064199e3fa8f658a1f9a023d0b327425" target='_blank'>
              Strand asymmetry in the repair of replication dependent double-strand breaks
              </a>
            </td>
          <td>
            Michael T. Kimble, Aakanksha Sane, Robert J. D. Reid, Matthew J Johnson, Rodney Rothstein, L. Symington
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>54</td>
        </tr>

        <tr id="Size evolution among gigantic genomes involves gain and loss of many gigabases of transposable elements (TEs), sequences that parasitize host genomes. Animals suppress TEs using piRNA and KRAB-ZFP pathways. TEs and hosts coevolve in an arms race, where suppression strength reflects TE fitness costs. In enormous genomes, additional TE costs become miniscule. How, then, do TEs and host suppression invoke further addition of massive DNA amounts? We analyzed TE proliferation histories, deletion rates, and community diversities in six salamander genomes (21.3 - 49.9 Gb), alongside gonadal expression of TEs and suppression pathways. TE activity is higher in testes than ovaries, attributable to lower KRAB-ZFP suppression. Unexpectedly, genome size/expansion is uncorrelated with TE deletion rate, proliferation history, expression, and host suppression. Also, TE community diversity increases with genome size, contrasting theoretical predictions. TE/host antagonism in gigantic genomes likely produces stochastic TE accumulation, determined by noisy intermolecular interactions in huge genomes/cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2d65d2d29d50d5d6dbfbf2dd63d367db919f1f5b" target='_blank'>
              Size evolution of gigantic genomes suggests stochastic outcomes of transposable element/host silencing interactions
              </a>
            </td>
          <td>
            Jie Wang, Guangpu Zhang, Cheng Sun, Liming Chang, Yingyong Wang, Xin Yang, Guiying Chen, Michael W. Itgen, A. Haley, Jiaxing Tang, R. Mueller
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Breast cancer (BC) is a major malignant tumor in females and the incidence rate of BC has increased worldwide in recent years. N6-methyladenosine (m6A) is a methylation modification that occurs extensively in eukaryotic RNA. The abnormal expression of m6A and related regulatory proteins can activate or inhibit certain signal pathways or oncogenes, thus affecting the proliferation, metastasis and prognosis of BC. Numerous studies have shown that m6A regulator disorder exists in BC, and this disorder can be reversed. Therefore, m6A is predicted as a potential therapeutic target for BC. However, the molecular mechanism of m6A RNA methylation regulating the occurrence and development of BC has not been comprehensively elucidated. In this review article, the functions of various m6A regulators and the specific mechanisms of certain regulators of the progress of BC were summarized. Furthermore, the dual role of RNA methylation in tumor progression was discussed, concluding that RNA methylation can not only lead to tumorigenesis but at times give rise to inhibition of tumor formation. In addition, further comprehensive analysis on mechanisms of m6A regulators in BC is conducive to screening effective potential targets and formulating targeted treatment strategies, which will provide new methods for the prevention and treatment of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/94490aedf4c38b8aae57215a539da97485ddaf26" target='_blank'>
              Functions and mechanisms of RNA m6A regulators in breast cancer (Review)
              </a>
            </td>
          <td>
            Yibei Yang, Feng Gao, Lanqi Ren, Ning Ren, Junjie Pan, Qiaoping Xu
          </td>
          <td>2024-07-26</td>
          <td>International Journal of Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="

 In this article, Antoinette van Ouwerkerk and Salvatore Spicuglia from INSERM highlight the significance of regulatory variants within the non-coding genome – often referred to as the ‘dark genome’ – in influencing gene expression and disease. The major genetic cause of human disease has historically been thought to be mutations within coding genes. However, there has been a shift in perspective over recent years, with more attention now being paid to the role of the non-coding regions in the genome. Intense research in this area has revealed that the non-coding genome contributes to gene-regulatory networks that serve to control where and when sets of genes are expressed. In turn, dysregulation of gene expression due to changes or alterations in the non-coding genome drastically impacts the aetiology of different diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1b69d242a16126cd54a0cc28fd1836e10bb273d" target='_blank'>
              How the dark genome enlightens the molecular mechanisms of diseases
              </a>
            </td>
          <td>
            Salvatore Spicuglia
          </td>
          <td>2024-07-09</td>
          <td>Open Access Government</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In recent years, there has been increased focus on exploring the role the non-protein-coding genome plays in Mendelian disorders. One class of particular interest is long non-coding RNAs (lncRNAs), which has recently been implicated in the regulation of diverse molecular processes. However, because lncRNAs do not encode protein, there is uncertainty regarding what constitutes a pathogenic lncRNA variant, and thus annotating such elements is challenging. The Developmental Genome Anatomy Project (DGAP) and similar projects recruit individuals with apparently balanced chromosomal abnormalities (BCAs) that disrupt or dysregulate genes in order to annotate the human genome. We hypothesized that rearrangements disrupting lncRNAs could be the underlying genetic etiology for the phenotypes of a subset of these individuals. Thus, we assessed 279 cases with BCAs and selected 191 cases with simple BCAs (breakpoints at only two genomic locations) for further analysis of lncRNA disruptions. From these, we identified 66 cases in which the chromosomal rearrangements directly disrupt lncRNAs. Strikingly, the lncRNAs MEF2C-AS1 and ENSG00000257522 are each disrupted in two unrelated cases. Furthermore, in 30 cases, no genes of any other class aside from lncRNAs are directly disrupted, consistent with the hypothesis that lncRNA disruptions could underly the phenotypes of these individuals. To showcase the power of this genomic approach for annotating lncRNAs, here we focus on clinical reports and genetic analysis of two individuals with BCAs and additionally highlight six individuals with likely developmental etiologies due to lncRNA disruptions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4494659e85fb73d7f4a80f0d544797b5da37da2" target='_blank'>
              Rare germline disorders implicate long non-coding RNAs disrupted by chromosomal structural rearrangements
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-06-19</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Despite significant advances in our understanding of genetic cancer susceptibility, known inherited cancer predisposition syndromes explain at most 20% of early-onset cancers. As early-onset cancer prevalence continues to increase, the need to assess previously inaccessible areas of the human genome, harnessing a trio or quad family-based architecture for variant filtration, may reveal further insights into cancer susceptibility. To assess a broader spectrum of variation than can be ascertained by multi-gene panel sequencing, or even whole genome sequencing with short reads, we employed long read whole genome sequencing using an Oxford Nanopore Technology (ONT) PromethION of 3 families containing an early-onset cancer proband using a trio or quad family architecture. Analysis included 2 early-onset colorectal cancer family trios and one quad consisting of two siblings with testicular cancer, all with unaffected parents. Structural variants (SVs), epigenetic profiles and single nucleotide variants (SNVs) were determined for each individual, and a filtering strategy was employed to refine and prioritize candidate variants based on the family architecture. The family architecture enabled us to focus on inapposite variants while filtering variants shared with the unaffected parents, significantly decreasing background variation that can hamper identification of potentially disease causing differences. Candidate de novo and compound heterozygous variants were identified in this way. Gene expression, in matched neoplastic and pre-neoplastic lesions, was assessed for one trio. Our study demonstrates the feasibility of a streamlined analysis of genomic variants from long read ONT whole genome sequencing and a way to prioritize key variants for further evaluation of pathogenicity, while revealing what may be missing from panel based analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579a5b24c1657cfe092825293758535eef92a0f6" target='_blank'>
              Exploring the genetic and epigenetic underpinnings of early-onset cancers: Variant prioritization for long read whole genome sequencing from family cancer pedigrees
              </a>
            </td>
          <td>
            M. Kramer, Sara Goodwin, R. Wappel, Matilde Borio, K. Offit, D. Feldman, Z. Stadler, W. R. McCombie
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="ABSTRACT This research investigates the intricate dynamics of DNA methylation in the hours following CD8+ T cell activation, during a critical yet understudied temporal window. DNA methylation is an epigenetic modification central to regulation of gene expression and directing immune responses. Our investigation spanned 96-h post-activation and unveils a nuanced tapestry of global and site-specific methylation changes. We identified 15,626 significant differentially methylated CpGs spread across the genome, with the most significant changes occurring within the genes ADAM10, ICA1, and LAPTM5. While many changes had modest effect sizes, approximately 120 CpGs exhibited a log2FC above 1.5, with cell activation and proliferation pathways the most affected. Relatively few of the differentially methylated CpGs occurred along adjacent gene regions. The exceptions were seven differentially methylated gene regions, with the Human T cell Receptor Alpha Joining Genes demonstrating consistent methylation change over a 3kb window. We also investigated whether an inflammatory environment could alter DNA methylation during activation, with proliferating cells exposed to the oxidant glycine chloramine. No substantial differential methylation was observed in this context. The temporal perspective of early activation adds depth to the evolving field of epigenetic immunology, offering insights with implications for therapeutic innovation and expanding our understanding of epigenetic modulation in immune function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf69efc7eae76417393a64272f476d27bebaa6b9" target='_blank'>
              Widespread genomic de novo DNA methylation occurs following CD8+ T cell activation and proliferation
              </a>
            </td>
          <td>
            Annika R. Seddon, Olivia M Damiano, Mark B. Hampton, Aaron J. Stevens
          </td>
          <td>2024-06-20</td>
          <td>Epigenetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Tumor protein p53 (TP53) mediates DNA repair and cell proliferation in growing cells. The TP53 gene is a tumor suppressor that regulates the expression of target genes in response to multiple cellular stress factors. Key target genes are involved in crucial cellular events such as DNA repair, cell cycle regulation, apoptosis, metabolism, and senescence. TP53 genetic variants and the activity of the wild-type p53 protein (WT-p53) have been linked to a wide range of tumorigenesis. Various genetic and epigenetic alterations, including germline and somatic mutations, loss of heterozygosity, and DNA methylation, can alter TP53 activity, potentially resulting in cancer initiation and progression. This study was designed to screen three reported mutations in the DNA-binding domain of the p53 protein in breast cancer, to evaluate the relative susceptibility and risk associated with breast cancer in the local population. Genomic DNA was isolated from 30 breast tumor tissues along with controls. Tetra and Tri ARMS PCR were performed to detect mutations in the TP53 coding region. For SNPs c.637C>T and c.733C>T, all analyzed cases were homozygous for the wild-type allele ‘C,’ while for SNP c.745A>G, all cases were homozygous for the wild-type allele ‘A.’ These results indicate no relevance of these three SNPs to cancer progression in our study cohort. Additionally, the findings from whole exon sequencing will help to predict more precise outcomes and assess the importance of TP53 gene mutations in breast cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bffe267deb645fc61554e684edc8c0bafc2c11ee" target='_blank'>
              Analysis of Genetic Alterations in TP53 Gene in Breast Cancer – A Secondary Publication
              </a>
            </td>
          <td>
            Baqaur Rehman, Muhammad Abubakar, Muhammad Naeem Kiani, Rooma Ayyoub
          </td>
          <td>2024-06-19</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The Ten-Eleven Translocation (TET) family genes are implicated in a wide array of biological functions across various human cancers. Nonetheless, there is a scarcity of studies that comprehensively analyze the correlation between TET family members and the molecular phenotypes and clinical characteristics of different cancers. Leveraging updated public databases and employing several bioinformatics analysis methods, we assessed the expression levels, somatic variations, methylation levels, and prognostic values of TET family genes. Additionally, we explored the association between the expression of TET family genes and pathway activity, tumor microenvironment (TME), stemness score, immune subtype, clinical staging, and drug sensitivity in pan-cancer. Molecular biology and cytology experiments were conducted to validate the potential role of TET3 in tumor progression. Each TET family gene displayed distinct expression patterns across at least ten detected tumors. The frequency of Single Nucleotide Variant (SNV) in TET genes was found to be 91.24%, primarily comprising missense mutation types, with the main types of copy number variant (CNV) being heterozygous amplifications and deletions. TET1 gene exhibited high methylation levels, whereas TET2 and TET3 genes displayed hypomethylation in most cancers, which correlated closely with patient prognosis. Pathway activity analysis revealed the involvement of TET family genes in multiple signaling pathways, including cell cycle, apoptosis, DNA damage response, hormone AR, PI3K/AKT, and RTK. Furthermore, the expression levels of TET family genes were shown to impact the clinical staging of tumor patients, modulate the sensitivity of chemotherapy drugs, and thereby influence patient prognosis by participating in the regulation of the tumor microenvironment, cellular stemness potential, and immune subtype. Notably, TET3 was identified to promote cancer progression across various tumors, and its silencing was found to inhibit tumor malignancy and enhance chemotherapy sensitivity. These findings shed light on the role of TET family genes in cancer progression and offer insights for further research on TET3 as a potential therapeutic target for pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4df11d8b0544aeb2264d4da3fba3e8f78e1111" target='_blank'>
              Pan-cancer analyses reveal the molecular and clinical characteristics of TET family members and suggests that TET3 maybe a potential therapeutic target
              </a>
            </td>
          <td>
            Chunyan Zhang, Jie Zheng, Jin Liu, Yanxia Li, G. Xing, Shupeng Zhang, Hekai Chen, Jian Wang, Z. Shao, Yongyuan Li, Zhongmin Jiang, Yingzi Pan, Xiaozhi Liu, Ping Xu, Wenhan Wu
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a3d53c60ef5486ebcbf381bfd90970c04135b42" target='_blank'>
              Low-input PacBio sequencing generates high-quality individual fly genomes and characterizes mutational processes
              </a>
            </td>
          <td>
            Hangxing Jia, Shengjun Tan, Yingao Cai, Ya-Dong Guo, Jieyu Shen, Yaqiong Zhang, Huijing Ma, Qingzhu Zhang, Jinfeng Chen, Gexia Qiao, Jue Ruan, Yong E Zhang
          </td>
          <td>2024-07-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4597b2c9168e5b3eb619c4010a4d29dad0a9c28b" target='_blank'>
              mTORC2-driven chromatin cGAS mediates chemoresistance through epigenetic reprogramming in colorectal cancer.
              </a>
            </td>
          <td>
            Guoqing Lv, Qian Wang, Lin Lin, Qiao Ye, Xi Li, Qian Zhou, Xiangzhen Kong, Hongxia Deng, Fuping You, Hebing Chen, Song Wu, Lin Yuan
          </td>
          <td>2024-07-30</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract The Fanconi anemia (FA) repair pathway governs repair of highly genotoxic DNA interstrand crosslinks (ICLs) and relies on translesion synthesis (TLS). TLS is facilitated by REV1 or site-specific monoubiquitination of proliferating cell nuclear antigen (PCNA) (PCNA-Ub) at lysine 164 (K164). A PcnaK164R/K164R but not Rev1−/− mutation renders mammals hypersensitive to ICLs. Besides the FA pathway, alternative pathways have been associated with ICL repair (1, 2), though the decision making between those remains elusive. To study the dependence and relevance of PCNA-Ub in FA repair, we intercrossed PcnaK164R/+; Fancg−/+ mice. A combined mutation (PcnaK164R/K164R; Fancg−/−) was found embryonically lethal. RNA-seq of primary double-mutant (DM) mouse embryonic fibroblasts (MEFs) revealed elevated levels of replication stress-induced checkpoints. To exclude stress-induced confounders, we utilized a Trp53 knock-down to obtain a model to study ICL repair in depth. Regarding ICL-induced cell toxicity, cell cycle arrest, and replication fork progression, single-mutant and DM MEFs were found equally sensitive, establishing PCNA-Ub to be critical for FA-ICL repair. Immunoprecipitation and spectrometry-based analysis revealed an unknown role of PCNA-Ub in excluding mismatch recognition complex MSH2/MSH6 from being recruited to ICLs. In conclusion, our results uncovered a dual function of PCNA-Ub in ICL repair, i.e. exclude MSH2/MSH6 recruitment to channel the ICL toward canonical FA repair, in addition to its established role in coordinating TLS opposite the unhooked ICL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47255c40b1a573923862892c3bf44f93590b73e7" target='_blank'>
              Dual role of proliferating cell nuclear antigen monoubiquitination in facilitating Fanconi anemia-mediated interstrand crosslink repair
              </a>
            </td>
          <td>
            R. Shah, M. A. Aslam, Aldo Spanjaard, Daniel de Groot, Lisa M Zürcher, M. Altelaar, Liesbeth Hoekman, Colin E J Pritchard, B. Pilzecker, P. V. D. van den Berk, Heinz Jacobs
          </td>
          <td>2024-06-18</td>
          <td>PNAS Nexus</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="


 What’s the pattern of genome-wide chromatin occupancy of SCP3 during meiotic prophase, and how its interaction with chromatin is regulated?



 We document SCP3 enriched at active chromatin regions with largely inherited from leptotene to pachytene stage, which facilitated by transcription and condensate assembly in spermatocytes.



 Meiosis is a critical step during the process of gametogenesis to form haploid cells which finally develop into functional gametes. The synaptonemal complex (SC) is a meiosis-specific protein assembly which bridges homologous chromosomes to establish the architecture of meiotic chromosome. Genetic ablation of genes encoding SC components impedes meiosis and gametogenesis in mice and mutation of these genes is frequently found in infertile patients. SCP3 localization changes with chromosome morphology in autosomes and is involved in maintaining the association of sex chromosomes. Male mice with null mutation of SCP3 failed to form SCs, resulting in developmental arrest during meiotic prophase.



 This study included male mice at post-natal day 10 (P10), 4 weeks or adult (8 weeks or older) were maintained on C57BL/6J background. Spermatocytes were performed modified CUT&Tag using antibodies. ChIP-seq data was analyzed together with WGBS and ATAC-seq data.



 Spermatocytes were obtained by a discontinuous BSA density gradient. Cell pellet was resuspended in culture medium (MEM with 10% FBS) at a concentration of 105 cells/ml and cultured at 37 °C in a 5% CO2 incubator. Spermatocytes were cultured with or without 1,6-hexanediol or α-amanitin. Spermatocytes with or without drug treatment were collected at indicated time points for further examination and then modified CUT&Tag and co-IP using anti-SCP3 and anti-POL II antibodies.



 Chromatin occupancy of SCP3 in leptotene and pachytene spermatocytes, SCP3-Core sites retained upon treatment of 1,6-hexanediol which dissolves SCP3 condensate, and SCP1 occupancy at meiotic chromatin in pachytene spermatocytes. When we determined inter-peak distances and width of peaks, we observe that: (1) SCP3 occupancy in leptotene stage had greater inter-peak distances and narrower peak width than SCP3 occupancy in pachytene stage, demonstrating gradual accumulated SCP3 at chromatin during meiotic progression; (2) SCP3-Core sites have broader peak width than SCP3-NonCore sites, in agreement with stability of SCP3 occupancy at SCP3-Core sites; (3) SCP1 occupancy had greater inter-peak distances and narrower peak width than SCP3 occupancy in pachytene stage, supporting that SCP1 peaks are a subset of SCP3 peaks and SCP1’s major function in connecting LE and CE respectively. Generally, at pachytene-stage meiosis, SCP3 mainly occupies chromatin regions including transcriptionally active sites, Deu and Alu elements, while SCP1 mainly co-localizes with SCP3 at a subset of regions with open chromatin and high cohesin enrichment to connect meiotic chromosome axes with CE. In this way, a local active chromatin environment is created at meiotic chromosome axes, facilitating homologous chromosome recombination.



 Our test was not successful to examine SCP3 occupancy with as few as less than 1000 spermatocytes using our protocol. It is improvement of efficiency of tagmentation or development of more efficient antibodies against SCP3 will be helpful for clinical sample detection.



 SCP3 occupancy was largely inherited from leptotene to pachytene stage, facilitated by transcription and condensate assembly, and enriched at specific SINE repeats. Further exploration using knockout mouse models of meiotic axial proteins and epigenetic modifiers will provide valuable information on meiotic chromatin architecture and crosstalk among meiotic chromatin events.



 N/A
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd5370d474a768836582b1cb861c8c3d70c98bd" target='_blank'>
              P-045 Deciphering meiotic chromatin organization by SCP3
              </a>
            </td>
          <td>
            S. Guo
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Background Tandem duplication (TD) is a common and important type of structural variation in the human genome. TDs have been shown to play an essential role in many diseases, including cancer. However, it is difficult to accurately detect TDs due to the uneven distribution of reads and the inherent complexity of next-generation sequencing (NGS) data. Methods This article proposes a method called DTDHM (detection of tandem duplications based on hybrid methods), which utilizes NGS data to detect TDs in a single sample. DTDHM builds a pipeline that integrates read depth (RD), split read (SR), and paired-end mapping (PEM) signals. To solve the problem of uneven distribution of normal and abnormal samples, DTDHM uses the K-nearest neighbor (KNN) algorithm for multi-feature classification prediction. Then, the qualified split reads and discordant reads are extracted and analyzed to achieve accurate localization of variation sites. This article compares DTDHM with three other methods on 450 simulated datasets and five real datasets. Results In 450 simulated data samples, DTDHM consistently maintained the highest F1-score. The average F1-score of DTDHM, SVIM, TARDIS, and TIDDIT were 80.0%, 56.2%, 43.4%, and 67.1%, respectively. The F1-score of DTDHM had a small variation range and its detection effect was the most stable and 1.2 times that of the suboptimal method. Most of the boundary biases of DTDHM fluctuated around 20 bp, and its boundary deviation detection ability was better than TARDIS and TIDDIT. In real data experiments, five real sequencing samples (NA19238, NA19239, NA19240, HG00266, and NA12891) were used to test DTDHM. The results showed that DTDHM had the highest overlap density score (ODS) and F1-score of the four methods. Conclusions Compared with the other three methods, DTDHM achieved excellent results in terms of sensitivity, precision, F1-score, and boundary bias. These results indicate that DTDHM can be used as a reliable tool for detecting TDs from NGS data, especially in the case of low coverage depth and tumor purity samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332aeecb63c38630dbdee5179707b4c99d4bb7e6" target='_blank'>
              DTDHM: detection of tandem duplications based on hybrid methods using next-generation sequencing data
              </a>
            </td>
          <td>
            Tianting Yuan, Jinxin Dong, Baoxian Jia, Hua Jiang, Zuyao Zhao, Mengjiao Zhou
          </td>
          <td>2024-07-26</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4ee6e732452c0ae079ae842747b6a8d042906b7" target='_blank'>
              The contribution of silencer variants to human diseases
              </a>
            </td>
          <td>
            D. Huang, I. Ovcharenko
          </td>
          <td>2024-07-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="CRNDE is an oncogene expressed as a long non-coding RNA. However, our team previously reported that the CRNDE gene also encodes a micropeptide, CRNDEP. The amino acid sequence of CRNDEP has recently been revealed by other researchers, too. This study aimed to investigate genetic alterations within the CRNDEP-coding region of the CRNDE gene, methylation profiling of this gene, and CRNDEP expression analysis. All investigations were performed on clinical material from patients with ovarian tumors of diverse aggressiveness. We found that CRNDEP levels were significantly elevated in highly aggressive tumors compared to benign neoplasms. Consistently, a high level of this micropeptide was a negative, independent, prognostic, and predictive factor in high-grade ovarian cancer (hgOvCa) patients. The cancer-promoting role of CRNDE(P), shown in our recent study, was also supported by genetic and epigenetic results obtained herein, revealing no CRNDEP-disrupting mutations in any clinical sample. Moreover, in borderline ovarian tumors (BOTS), but not in ovarian cancers, the presence of a single nucleotide polymorphism in CRNDE, rs115515594, significantly increased the risk of recurrence. Consistently, in BOTS only, the same genetic variant was highly overrepresented compared to healthy individuals. We also discovered that hypomethylation of CRNDE is associated with increased aggressiveness of ovarian tumors. Accordingly, hypomethylation of this gene’s promoter/first exon correlated with hgOvCa resistance to chemotherapy, but only in specimens with accumulation of the TP53 tumor suppressor protein. Taken together, these results contribute to a better understanding of the role of CRNDE(P) in tumorigenesis and potentially may lead to improvements in screening, diagnosis, and treatment of ovarian neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41a5d81fb227910421858aeef5200b2eb49cb75b" target='_blank'>
              The Clinical Significance of CRNDE Gene Methylation, Polymorphisms, and CRNDEP Micropeptide Expression in Ovarian Tumors
              </a>
            </td>
          <td>
            L. A. Szafron, R. Iwanicka-Nowicka, A. Podgórska, Arkadiusz M Bonna, Piotr Sobiczewski, J. Kupryjańczyk, L. Szafron
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract Summary Mitochondrial DNA sequences are used extensively in phylogeographic and phylogenetic studies for a wide range of organisms. With the advent of low-cost, high-throughput “next generation” DNA sequencing, and user-friendly bioinformatics pipelines for generating and annotating whole mitochondrial genome assemblies, the analysis of whole mitochondrial genomes has become an important component of phylogenomic studies for taxa with high species diversity but limited coverage for other genomic resources. An important step in characterizing de novo mitochondrial genome assemblies is to evaluate and describe structural rearrangements relative to reference taxa. Accessible tools are needed to help visualize gene and non-coding feature complement, their order, and strand orientation. However, there are few dedicated applications that generate high-quality genome diagrams. Here we present circularMT and circularMT-console that allow users to create highly customizable, publication-quality images, of linear and circular mitochondrial genome maps, either individually or integrated into an analysis pipeline. Availability and implementation Both applications are implemented in C#, with binaries, source code, and user guides available on GitHub (https://github.com/msjimc/circularMT). An archive of the published version is available on Zenodo (https://zenodo.org/records/10912319).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ad314515566c3f36d4dbf28b1d1c77340aa083" target='_blank'>
              Drawing mitochondrial genomes with circularMT
              </a>
            </td>
          <td>
            Simon J. Goodman, Ian M. Carr
          </td>
          <td>2024-07-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86376d37d11acefead8c66a577f19b5d0ad22d4f" target='_blank'>
              Circular RNA circ_ARHGEF28 inhibits MST1/2 dimerization to suppress Hippo pathway to induce cisplatin resistance in ovarian cancer
              </a>
            </td>
          <td>
            R. Lei, Yun Long, Qingjian Li, Qingsheng Xie, Xiaoting Ling, Meiqing Xie, Hui Zhou, Bingzhong Zhang
          </td>
          <td>2024-07-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Application of laser-generated electron beams in radiotherapy is a recent development. Accordingly, mechanisms of biological response to radiation damage need to be investigated. In this study, telomere length (TL) as endpoint of genetic damage was analyzed in human blood cells (leukocytes) and K562 leukemic cells irradiated with laser-generated ultrashort electron beam. Metaphases and interphases were analyzed in quantitative fluorescence in situ hybridization (Q-FISH) to assess TL. TLs were shortened compared to non-irradiated controls in both settings (metaphase and interphase) after irradiation with 0.5, 1.5, and 3.0 Gy in blood leukocytes. Radiation also caused a significant TL shortening detectable in the interphase of K562 cells. Overall, a negative correlation between TL and radiation doses was observed in normal and leukemic cells in a dose-dependent manner. K562 cells were more sensitive than normal blood cells to increasing doses of ultrashort electron beam radiation. As telomere shortening leads to genome instability and cell death, the results obtained confirm the suitability of this biomarker for assessing genotoxic effects of accelerated electrons for their further use in radiation therapy. Observed differences in TL shortening between normal and K562 cells provide an opportunity for further development of optimal radiation parameters to reduce side effects in normal cells during radiotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eee43799b74e8b568f5f9ee529deeb7e60bc9753" target='_blank'>
              Changes in Telomere Length in Leukocytes and Leukemic Cells after Ultrashort Electron Beam Radiation
              </a>
            </td>
          <td>
            T. Harutyunyan, A. Sargsyan, Lily Kalashyan, Hovhannes A. Igityan, B. Grigoryan, Hakob Davtyan, R. Aroutiounian, Thomas Liehr, G. Hovhannisyan
          </td>
          <td>2024-06-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Motivation Techniques such as high-throughput chromosome conformation capture (Hi-C) have provided a wealth of information on the organization of the nucleus and the genome important for understanding gene expression regulation. Additionally, Genome-Wide Association Studies (GWASs) have uncovered thousands of loci related to complex traits. Expression quantitative trait loci (eQTL) studies have further linked the genetic variants to alteration in expression levels of associated target genes across individuals. However, the functional roles of many eQTLs located in non-coding regions are unclear. Current joint analyses of Hi-C and eQTLs data lack advanced computational tools, limiting what can be learned from these data. Result In this work, we developed a computational method for simultaneous analysis of Hi-C and eQTL data. Our method can identify a small set of non-random interactions from all Hi-C interactions. Using these non-random interactions, we reconstruct large ensemble (×105) of high-resolution single-cell 3D chromatin conformations with thorough sampling, which accurately replicate Hi-C measurements. Our results revealed the presence of many-body interactions in chromatin conformation at single-cell level in eQTL locus, offering detailed view into how three-dimensional structures of chromatin form the physical foundation for gene regulation, including how genetic variants of eQTLs affect the expression level of their associated eGenes. Furthermore, our method can deconvolve chromatin heterogeneity and investigate the spatial associations of eQTLs and eGenes at subpopulation level to reveal their regulatory impacts on gene expression. Together, ensemble modeling of thoroughly sampled single cell chromatin conformations from Hi-C, along with eQTL data, helps to decipher how chromatin 3D structures provide the physical basis for gene regulation, expression control, and aid in understanding of the overall structure-function relationships of genome organization. Availability and implementation: It is available at https://github.com/uic-liang-lab/3DChromFolding-eQTL-Loci">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9720002233785e5b33a3a1f9b1f74f286c10723" target='_blank'>
              Structural Basis of Differential Gene Expression at eQTLs Loci from High-Resolution Ensemble Models of 3D Single-Cell Chromatin Conformations
              </a>
            </td>
          <td>
            Lin Du, Hammad Farooq, Pourya Delafrouz, Jie Liang
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Despite the established role of DNA methylation in transcriptional silencing, most of the genes remain unchanged upon its global erasing from the genome, while expression of other genes is actually raised. Since its discovery a few decades ago, this unsolved enigma has hampered true understanding of DNA methylation role. By applying an updated mathematical model to the results of the original studies, we showed that this paradoxical effect is due to between-gene differences in the balance between methylation-guided enhancers and silencers, which protect particular disease-causing genes from the otherwise harsh effects of global methylation perturbation. Nevertheless, local methylation alternation may still affect these genes. The results a settle a long-standing wondering and suggest that DNA-methylation regulatory role is significantly greater than previously thought.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f326a9727e723fecbefe9526db8fc37ada53d66" target='_blank'>
              On the enigmatic null effect of global methylation perturbation
              </a>
            </td>
          <td>
            Y. Edrei, A. Hellman
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Radiation cytogenetics has a rich history seldom appreciated by those outside the field. Early radiobiology was dominated by physics and biophysical concepts that borrowed heavily from the study of radiation-induced chromosome aberrations. From such studies, quantitative relationships between biological effect and changes in absorbed dose, dose rate and ionization density were codified into key concepts of radiobiological theory that have persisted for nearly a century. This review aims to provide a historical perspective of some of these concepts, including evidence supporting the contention that chromosome aberrations underlie development of many, if not most, of the biological effects of concern for humans exposed to ionizing radiations including cancer induction, on the one hand, and tumor eradication on the other. The significance of discoveries originating from these studies has widened and extended far beyond their original scope. Chromosome structural rearrangements viewed in mitotic cells were first attributed to the production of breaks by the radiations during interphase, followed by the rejoining or mis-rejoining among ends of other nearby breaks. These relatively modest beginnings eventually led to the discovery and characterization of DNA repair of double-strand breaks by non-homologous end joining, whose importance to various biological processes is now widely appreciated. Two examples, among many, are V(D)J recombination and speciation. Rapid technological advancements in cytogenetics, the burgeoning fields of molecular radiobiology and third-generation sequencing served as a point of confluence between the old and new. As a result, the emergent field of "cytogenomics" now becomes uniquely positioned for the purpose of more fully understanding mechanisms underlying the biological effects of ionizing radiation exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c57db9d0b65c8dcb0db634d392394a245d1aa2a" target='_blank'>
              The Central Role of Cytogenetics in Radiation Biology.
              </a>
            </td>
          <td>
            Susan M. Bailey, Stephen R. Kunkel, Joel S. Bedford, Michael N. Cornforth
          </td>
          <td>2024-07-10</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Introduction Glioblastomas utilize malignant gene expression pathways to drive growth. Many of these gene pathways are not directly accessible with molecularly targeted pharmacological agents. Chromatin-modifying compounds can alter gene expression and target glioblastoma growth pathways. In this study, we utilize a systematic screen of chromatin-modifying compounds on a panel of patient-derived glioblastoma lines to identify promising compounds and their associated gene targets. Methods Five glioblastoma cell lines were subjected to a drug screen of 106 chromatin-modifying compounds representing 36 unique drug classes to determine the twelve most promising drug classes and the best candidate inhibitors in each class. These twelve drugs were then tested with a panel of twelve patient-derived gliomasphere lines to identify growth inhibition and corresponding gene expression patterns. Overlap analysis and weighted co-expression network analysis (WCGNA) were utilized to determine potential target genes and gene pathways. Results The initial drug screen identified twelve candidate pharmacologic agents for further testing. Drug sensitivity testing indicated an overall high degree of variability between gliomasphere lines. However, CPI203 was the most consistently effective compound, and the BET inhibitor class was the most consistently effective class of compounds across the gliomasphere panel. Correspondingly, most of the compounds tested had highly variable effects on gene expression between gliomasphere lines. CPI203 stood out as the only compound to induce a consistent effect on gene expression across different gliomasphere lines, specifically down-regulation of DNA-synthesis genes. Amongst the twelve tested cell lines, high expression of CDKN2A and CDKN2B distinguished more drug sensitive from more drug resistant lines. WCGNA identified two oncogenic gene modules (FBXO5 and MELK) that were effectively downregulated by CPI203 (FBXO5) and ML228 (FBXO5 and MELK). Conclusions The bromodomain inhibitor CPI203 induced relatively consistent effects on gene expression and growth across a variety of glioblastoma lines, specifically down-regulating genes associated with DNA replication. We propose that clinically effective BET inhibitors have the potential to induce consistent beneficial effects across a spectrum of glioblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1561fa27def3a17181508712df5ccd39c21a9f63" target='_blank'>
              CPI203, a BET inhibitor, down-regulates a consistent set of DNA synthesis genes across a wide array of glioblastoma lines
              </a>
            </td>
          <td>
            Matthew C. Garrett, Troy P Carnwath, Rebecca R Albano, Yonghua Zhuang, Catherine A Behrmann, Merissa Pemberton, Farah Barakat, Robert Lober, Mark Hoeprich, Anthony Paravati, Marilyn Reed, Hailey Spry, Daniel Woo, Eric O’Brien, Brett VanCauwenbergh, John Perentesis, Rani Nasser, Mario Medvedovic, D. Plas
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Aneuploidy, or a change in the number of whole chromosomes or chromosome arms, is a near-universal feature of cancer. Chromosomes affected by aneuploidy are not random, with observed cancer-specific and tissue-specific patterns. Recent advances in genome engineering methods have allowed the creation of models with targeted aneuploidy events. These models can be used to uncover the downstream effects of individual aneuploidies on cancer phenotypes including proliferation, apoptosis, metabolism, and immune signaling. Here, we review the current state of research into the patterns of aneuploidy in cancer and their impact on signaling pathways and biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53976b63387a7558e4fe16be8478ae589398c516" target='_blank'>
              Patterns of Aneuploidy and Signaling Consequences in Cancer.
              </a>
            </td>
          <td>
            Nadja Zhakula-Kostadinova, Alison M Taylor
          </td>
          <td>2024-06-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background Lung squamous cell carcinoma (LUSC) ranks among the carcinomas with the highest incidence and dismal survival rates, suffering from a lack of effective therapeutic strategies. Consequently, biomarkers facilitating early diagnosis of LUSC could significantly enhance patient survival. This study aims to identify novel biomarkers for LUSC. Methods Utilizing the TCGA, GTEx, and CGGA databases, we focused on the gene encoding Family with Sequence Similarity 20, Member A (FAM20A) across various cancers. We then corroborated these bioinformatic predictions with clinical samples. A range of analytical tools, including Kaplan-Meier, MethSurv database, Wilcoxon rank-sum, Kruskal-Wallis tests, Gene Set Enrichment Analysis, and TIMER database, were employed to assess the diagnostic and prognostic value of FAM20A in LUSC. These tools also helped evaluate immune cell infiltration, immune checkpoint genes, DNA repair-related genes, DNA methylation, and tumor-related pathways. Results FAM20A expression was found to be significantly reduced in LUSC, correlating with lower survival rates. It exhibited a negative correlation with key proteins in DNA repair signaling pathways, potentially contributing to LUSC’s radiotherapy resistance. Additionally, FAM20A showed a positive correlation with immune checkpoints like CTLA-4, indicating potential heightened sensitivity to immunotherapies targeting these checkpoints. Conclusion FAM20A emerges as a promising diagnostic and prognostic biomarker for LUSC, offering potential clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a51c5b0a9c0f9adcdaf2fcbf78b816cd2d9551bf" target='_blank'>
              FAM20A: a potential diagnostic biomarker for lung squamous cell carcinoma
              </a>
            </td>
          <td>
            Yali Zhang, Qin Sun, Yangbo Liang, Xian Yang, Hailian Wang, Siyuan Song, Yi Wang, Yong Feng
          </td>
          <td>2024-06-25</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic mechanism orchestrating gene expression networks in many biological processes. Nonetheless, studying the role of specific gene methylation events in fish faces challenges. In this study, we validate the regulation of DNA methylation on empty spiracles homeobox 2 (emx2) expression with decitabine treatment in Chinese tongue sole testis cells. We used the emx2 gene as the target gene and developed a new DNA methylation editing system by fusing dnmt3a with catalytic dead Cas9 (dCas9) and demonstrated its ability for sequence-specific DNA methylation editing. Results revealed that utilizing dCas9-dnmt3a to target emx2 promoter region led to increased DNA methylation levels and decreased emx2 expression in Chinese tongue sole testis cells. More importantly, the DNA methylation editing significantly suppressed the expression of MYC proto-oncogene, bHLH transcription factor (myc), one target gene of emx2. Furthermore, we assessed the off-target effects of dCas9-dnmt3a and confirmed no significant impact on the predicted off-target gene expression. Taken together, we developed the first DNA methylation editing system in marine species and demonstrated its effective editing ability in Chinese tongue sole cells. This provides a new strategy for both epigenetic research and molecular breeding of marine species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/418776c51af85059269724bfd4c08f2823970cf0" target='_blank'>
              CRISPR/dCas9-Mediated DNA Methylation Editing on emx2 in Chinese Tongue Sole (Cynoglossus semilaevis) Testis Cells
              </a>
            </td>
          <td>
            Yanxu Sun, Hongyan Wang, Binghua Liu, Bowen Yue, Qian Liu, Yuyan Liu, Ivana F. Rosa, L. Doretto, Shenglei Han, Lei Lin, Xiaoling Gong, Changwei Shao
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41728da5eec57442f344cb37688d002e66c10d7e" target='_blank'>
              Tumor initiation and early tumorigenesis: molecular mechanisms and interventional targets
              </a>
            </td>
          <td>
            Shaosen Zhang, Xinyi Xiao, Yonglin Yi, Xinyu Wang, Lingxuan Zhu, Yanrong Shen, Dongxin Lin, Chen Wu
          </td>
          <td>2024-06-19</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>1</td>
          <td>4</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells. IMPORTANT Manuscripts submitted to Review Commons are peer reviewed in a journal-agnostic way. Upon transfer of the peer reviewed preprint to a journal, the referee reports will be available in full to the handling editor. The identity of the referees will NOT be communicated to the authors unless the reviewers choose to sign their report. The identity of the referee will be confidentially disclosed to any affiliate journals to which the manuscript is transferred. GUIDELINES For reviewers: https://www.reviewcommons.org/reviewers For authors: https://www.reviewcommons.org/authors CONTACT The Review Commons office can be contacted directly at: office@reviewcommons.org">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5af5b334e28dd05ee59f91ab22bbf81d5c9b58d9" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M. Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="53BP1 plays a crucial role in regulating DNA damage repair pathway choice and checkpoint signaling in somatic cells; however, its role in meiosis has remained enigmatic. In this study, we demonstrate that the Caenorhabditis elegans ortholog of 53BP1, HSR-9, associates with chromatin in both proliferating and meiotic germ cells. Notably, HSR-9 is enriched on the X chromosome pair in pachytene oogenic germ cells. HSR-9 is also present at kinetochores during both mitotic and meiotic divisions but does not appear to be essential for monitoring microtubule-kinetochore attachments or tension. Using cytological markers of different steps in recombinational repair, we found that HSR-9 influences the processing of a subset of meiotic double strand breaks into COSA-1-marked crossovers. Additionally, HSR-9 plays a role in meiotic X chromosome segregation under conditions where X chromosomes fail to pair, synapse, and recombine. Together, these results highlight that chromatin-associated HSR-9 has both conserved and unique functions in the regulation of meiotic chromosome behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e700bc11eabfed38f5c48636a6b97e3c3470f7b" target='_blank'>
              The chromatin-associated 53BP1 ortholog, HSR-9, regulates recombinational repair and X chromosome segregation in the Caenorhabditis elegans germ line.
              </a>
            </td>
          <td>
            Qianyan Li, Sara Hariri, Aashna Calidas, Arshdeep Kaur, Erica Huey, J. Engebrecht
          </td>
          <td>2024-06-17</td>
          <td>Genetics</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Non-coding regions of the human genome are important for functional regulations, but their mechanisms remain elusive. We used machine learning to guide a CRISPR screening on hubs (i.e. non-coding loci forming many 3D contacts) and significantly increased the discovery rate of hubs essential for cell growth. We found no clear genetic or epigenetic differences between essential and nonessential hubs, but we observed that some neighboring hubs in the linear genome have distinct spatial contacts and opposite effects on cell growth. One such pair in an epigenetically quiescent region showed different impacts on gene expression, chromatin accessibility and chromatin organization. We also found that deleting the essential hub altered the genetic network activity and increased the entropy of chromatin accessibility, more severe than that caused by deletion of the nonessential hub, suggesting that they are critical for maintaining an ordered chromatin structure. Our study reveals new insights into the system-level roles of non-coding regions in the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37fecffd504fcf6b86c79eeec8f4d7ba8dc1c39" target='_blank'>
              Differential impact of quiescent non-coding loci on chromatin entropy.
              </a>
            </td>
          <td>
            Peiyao Wu, Mina Yao, Wei Wang
          </td>
          <td>2024-06-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-stranded DNA (ssDNA) is essential for various DNA-templated processes in both eukaryotes and prokaryotes. However, comprehensive characterizations of ssDNA still lag in plants compared to non-plant systems. Here, we conducted in situ S1-seq (ISS1-seq), with starting gDNA ranging from 5 µg to 250 ng, followed by comprehensive characterizations of ssDNA in rice (Oryza sativa L.). We found that ssDNA loci were substantially associated with a subset of non-B DNA structures and functional genomic loci. Subtypes of ssDNA loci had distinct epigenetic features. Importantly, ssDNA may act alone or partly coordinate with non-B DNA structures, functional genomic loci, or epigenetic marks to actively or repressively modulate gene transcription, which is genomic-region-dependent and associated with the distinct accumulation of RNA Pol II. Moreover, distinct types of ssDNA had differential impacts on the activities and evolution of TEs (especially common or conserved TEs) in the rice genome. Our study showcases an antibody-independent technique for characterizing non-B DNA structures or functional genomic loci in plants. It lays the groundwork and fills a crucial gap for further exploration of ssDNA, non-B DNA structures, or functional genomic loci, thereby advancing our understanding of their biology in plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a41f2ac1057a9cc0aff5ab06cd10c7413719a7" target='_blank'>
              Genome-wide characterization of single-stranded DNA in rice.
              </a>
            </td>
          <td>
            Yulian Peng, Pengtao Zhao, Zhaoguo Li, Ning Mu, Shentong Tao, Yilong Feng, Xuejiao Cheng, Wenli Zhang
          </td>
          <td>2024-06-25</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc05d0e416d1c5836cac6aabe3bd3a577d1440ba" target='_blank'>
              Enhanced eMAGE applied to identify genetic factors of nuclear hormone receptor dysfunction via combinatorial gene editing
              </a>
            </td>
          <td>
            Peter N Ciaccia, Zhuobin Liang, Anabel Y. Schweitzer, Eli Metzner, Farren J. Isaacs
          </td>
          <td>2024-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The DNA modifications, 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), represent powerful epigenetic regulators of temporal and spatial gene expression. Yet, how the cooperation of these genome-wide, epigenetic marks determine unique transcriptional signatures across different brain cell populations is unclear. Here we applied Nanopore sequencing of native DNA to obtain a complete, genome-wide, single-base resolution atlas of 5mC and 5hmC modifications in neurons, astrocytes and microglia in the mouse cortex (99% genome coverage, 40 million CpG sites). In tandem with RNA sequencing, analysis of 5mC and 5hmC patterns across cell types reveals astrocytes drive uniquely high brain 5hmC levels and support two decades of research regarding methylation patterns, gene expression and alternative splicing, benchmarking this resource. As such, we provide the most comprehensive DNA methylation data in mouse brain as an interactive, online tool (NAM-Me, https://olsenlab.shinyapps.io/NAMME/) to serve as a resource dataset for those interested in the methylome landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60076a55629be60f22fe1afccd00156f5bc051e7" target='_blank'>
              Decoding the Epigenetic Landscape: Insights into 5mC and 5hmC Patterns in Mouse Cortical Cell Types
              </a>
            </td>
          <td>
            Xiaoran Wei, Jiangtao Li, Zuolin Cheng, Songtao Wei, Guoqiang Yu, Michelle L. Olsen
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background: Cervical cancer (CC) stands as a significant health threat to women globally, with high-risk human papillomaviruses as major etiologic agents. The DNA damage repair (DDR) protein topoisomerase I (TOP1) has been linked to various cancers, yet its distinct roles and mechanisms in CC are not fully elucidated.Methods: We investigated TOP1 expression in cervical intraepithelial neoplasia (CIN) and CC tissues utilizing qRT-PCR and IHC, correlating findings with patient prognosis. Subsequent knockdown studies were performed in vitro and in vivo to evaluate the influence of TOP1 on tumor growth, DNA repair, and inflammatory responses.Results: TOP1 was highly expressed in CIN and CC, negatively correlating with patient prognosis. Inhibition of TOP1 impeded CC cell growth and disrupted DNA repair. TOP1 was shown to regulate tumor-promoting inflammation and programmed death-ligand 1 (PD-L1) production in a cGAS-dependent manner. HPV oncoproteins E6 and E7 upregulated TOP1 and activated the cGAS-PD-L1 pathway.Conclusions: TOP1 acts as a DNA repair mediator, promoting CC development and immune evasion. Targeting the TOP1-cGAS-PD-L1 axis could be a potential therapeutic strategy for CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f71748837721c3a12ecd5a0bb258b1048a43b9" target='_blank'>
              Oncoproteins E6 and E7 upregulate topoisomerase I to activate the cGAS-PD-L1 pathway in cervical cancer development
              </a>
            </td>
          <td>
            Ying Luo, Mengda Niu, Yanfei Liu, Miaochang Zhang, Yuanyuan Deng, Dan Mu, Junfen Xu, Shiyuan Hong
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891d9fb97d4fae14734f86d5adf86855004117f2" target='_blank'>
              Clonal inactivation of TERT impairs stem cell competition
              </a>
            </td>
          <td>
            Kazuteru Hasegawa, Yang Zhao, A. Garbuzov, M. Corces, P. Neuhöfer, Victoria M Gillespie, Peggie Cheung, J. Belk, Yung-Hsin Huang, Yuning Wei, Lu Chen, Howard Y. Chang, S. Artandi
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Cancer is a complex and heterogeneous disease, in which a number of genetic and epigenetic changes occur in tumor onset and progression. Recent studies indicate that changes at the RNA level are also involved in tumorigenesis, such as adenosine-to-inosine (A-to-I) RNA editing. Here, we systematically investigate transcriptome-wide A-to-I editing events in a large number of samples from Non-Hodgkin lymphomas (NHLs). Using a computational pipeline that determines significant differences in editing level between NHL and normal samples at known A-to-I editing sites, we identify a number of differentially edited editing sites between NHL subtypes and normal samples. Most of the differentially edited sites are located in non-coding regions, and many such sites show a strong correlation between gene expression level and editing efficiency, indicating that RNA editing might have direct consequences for the cancer cell’s aberrant gene regulation status in these cases. Moreover, we establish a strong link between RNA editing and NHL by demonstrating that NHL and normal samples and even NHL subtypes can be distinguished based on genome-wide RNA editing profiles alone. Our study establishes a strong link between RNA editing, cancer and aberrant gene regulation in NHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3709fa38f5f1119b1fcb4db3f2e4cd77244ecc08" target='_blank'>
              A-to-I Editing Is Subtype-Specific in Non-Hodgkin Lymphomas
              </a>
            </td>
          <td>
            Cai Chen, Ralf Bundschuh
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e1b4639cfb922b131986ddda7afd728b72c924a" target='_blank'>
              Deciphering the molecular landscape: evolutionary progression from gynecomastia to aggressive male breast cancer.
              </a>
            </td>
          <td>
            Chuang Yang, Zhonglin Wang, Lijun Qian, Jingyue Fu, H. Sun
          </td>
          <td>2024-06-18</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe60321069cd8bdd4dcd7dd1f6cb9aee5e0996c" target='_blank'>
              Z-DNA formation in promoters conserved between human and mouse are associated with increased transcription reinitiation rates
              </a>
            </td>
          <td>
            N. Beknazarov, Dmitry Konovalov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-08-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) remains a global health concern with persistently high incidence and mortality rates. However, the specific pathogenesis of CRC remains poorly understood. This study aims to investigate the role and pathogenesis of serine and arginine rich splicing factor 10 (SRSF10) in colorectal cancer. Methods Bioinformatics analysis was employed to predict SRSF10 gene expression in CRC patients. Functional experiments involving SRSF10 knockdown and overexpression were conducted using CCK8, transwell, scratch assay, and flow cytometry. Additionally, the PRIdictor website was utilized to predict the SRSF10 interaction site with RFC5. The identification of different transcripts of SRSF10-acting RFC5 pre-mRNA was achieved through agarose gel electrophoresis. Result The knockdown of SRSF10 inhibited the proliferation and migration ability of CRC cells, while promoting apoptosis and altering the DNA replication of CRC cells. Conversely, when SRSF10 was highly expressed, it enhanced the proliferation and migration ability of CRC cells and caused changes in the cell cycle of colorectal cancer cells. This study revealed a change in the replicating factor C subunit 5 (RFC5) gene in colorectal cancer cells following SRSF10 knockdown. Furthermore, it was confirmed that SRSF10 increased RFC5 exon2-AS1(S) transcription variants, thereby promoting the development of colorectal cancer through AS1 exclusion to exon 2 of RFC5. Conclusion In summary, this study demonstrates that SRSF10 promotes the progression of colorectal cancer by generating an aberrantly spliced exclusion isoform of AS1 within RFC5 exon 2. These findings suggest that SRSF10 could serve as a crucial target for the clinical diagnosis and treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183e863439be0cd3db11d90e783d10b2b9aa76e8" target='_blank'>
              High Expression of SRSF10 Promotes Colorectal Cancer Progression by Aberrant Alternative Splicing of RFC5
              </a>
            </td>
          <td>
            Shuai Xu, Fangmin Zhong, Junyao Jiang, Fangyi Yao, Meiyong Li, Mengxin Tang, Ying Cheng, Yulin Yang, Wen Wen, Xueru Zhang, Bo Huang, Xiao-Zhong Wang
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Recent advances in DNA sequencing technology have enabled the precise decoding of genomes in non-model organisms, providing a basis for unraveling the patterns and mechanisms of sex chromosome evolution. Studies of different species have yielded conflicting results regarding the traditional theory that sex chromosomes evolve from autosomes via the accumulation of deleterious mutations and degeneration of the Y (or W) chromosome. The concept of the ‘sex chromosome cycle,’ emerging from this context, posits that at any stage of the cycle (i.e., differentiation, degeneration, or loss), sex chromosome turnover can occur while maintaining stable sex determination. Thus, understanding the mechanisms that drive both the persistence and turnover of sex chromosomes at each stage of the cycle is crucial. In this review, we integrate recent findings on the mechanisms underlying maintenance and turnover, with a special focus on several organisms having unique sex chromosomes. Our review suggests that the diversity of sex chromosomes in the maintenance of stable sex determination is underappreciated and emphasizes the need for more research on the sex chromosome cycle.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ad3585db829ba2d4ab84372e9e70791e552b39" target='_blank'>
              Sex chromosome cycle as a mechanism of stable sex determination
              </a>
            </td>
          <td>
            Shun Hayashi, Takuya Abe, Takeshi Igawa, Yukako Katsura, Yusuke Kazama, Masafumi Nozawa
          </td>
          <td>2024-07-10</td>
          <td>Journal of Biochemistry</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="In this study, we investigated recurrent copy number variations (CNVs) in the 19p12 locus, which are associated with neurodevelopmental disorders. The two genes in this locus, ZNF675 and ZNF681, arose via gene duplication in primates, and their presence in several pathological CNVs in the human population suggests that either or both of these genes are required for normal human brain development. ZNF675 and ZNF681 are members of the Krüppel-associated box zinc finger (KZNF) protein family, a class of transcriptional repressors important for epigenetic silencing of specific genomic regions. About 170 primate-specific KZNFs are present in the human genome. Although KZNFs are primarily associated with repressing retrotransposon-derived DNA, evidence is emerging that they can be co-opted for other gene regulatory processes. We show that genetic deletion of ZNF675 causes developmental defects in cortical organoids, and our data suggest that part of the observed neurodevelopmental phenotype is mediated by a gene regulatory role of ZNF675 on the promoter of the neurodevelopmental gene Hes family BHLH transcription factor 1 (HES1). We also find evidence for the recently evolved regulation of genes involved in neurological disorders, microcephalin 1 and sestrin 3. We show that ZNF675 interferes with HES1 auto-inhibition, a process essential for the maintenance of neural progenitors. As a striking example of how some KZNFs have integrated into preexisting gene expression networks, these findings suggest the emergence of ZNF675 has caused a change in the balance of HES1 autoregulation. The association of ZNF675 CNV with human developmental disorders and ZNF675-mediated regulation of neurodevelopmental genes suggests that it evolved into an important factor for human brain development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2122cac7536c55e0373736baf9954110c9f4179" target='_blank'>
              Emergence of the ZNF675 Gene During Primate Evolution-Influenced Human Neurodevelopment Through Changing HES1 Autoregulation.
              </a>
            </td>
          <td>
            G. A. Lodewijk, Matthijs de Geus, Rita L F P Guimarães, F. Jacobs
          </td>
          <td>2024-07-01</td>
          <td>The Journal of comparative neurology</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fef3b73f828b4eb7f1c0358c04275f5963ff8a8f" target='_blank'>
              Integrative analysis of genomic and epigenomic regulation reveals miRNA mediated tumor heterogeneity and immune evasion in lower grade glioma
              </a>
            </td>
          <td>
            Zhen Yang, Xiaocen Liu, Hao Xu, Andrew E. Teschendorff, Lingjie Xu, Jingyi Li, Minjie Fu, Jun Liu, Hanyu Zhou, Yingying Wang, Licheng Zhang, Yungang He, Kun Lv, Hui Yang
          </td>
          <td>2024-07-06</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64dd346087b1d64ab0e5ccbf0c92c3994e5f755a" target='_blank'>
              Characterization and regulation of cell cycle-independent noncanonical gene targeting
              </a>
            </td>
          <td>
            Shinta Saito, N. Adachi
          </td>
          <td>2024-06-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Aneuploidy is nearly ubiquitous in tumor genomes, but the role of aneuploidy in the early stages of cancer evolution remains unclear. Here, by inducing heterogeneous aneuploidy in non-transformed human colon organoids (colonoids), we investigated how the effects of aneuploidy on cell growth and differentiation may promote malignant transformation. Previous work implicated p53 activation as a downstream response to aneuploidy induction. We found that simple aneuploidy, characterized by 1-3 gained or lost chromosomes, resulted in little or modest p53 activation and cell cycle arrest when compared with more complex aneuploid cells. Single-cell RNA sequencing analysis revealed that the degree of p53 activation was strongly correlated with karyotype complexity. Single-cell tracking showed that cells could continue to divide despite the observation of one to a few lagging chromosomes. Unexpectedly, colonoids with simple aneuploidy exhibited impaired differentiation after niche factor withdrawal. These findings demonstrate that simple aneuploid cells can escape p53 surveillance and may contribute to niche factor-independent growth of cancer-initiating colon stem cells. [Media: see text].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69141397c86397dd6c1f0121be2fa0ac82f1c38" target='_blank'>
              Simple aneuploidy evades p53 surveillance and promotes niche factor-independent growth in human intestinal organoids.
              </a>
            </td>
          <td>
            Blake A. Johnson, Albert Z Liu, Tianhao Bi, Yi Dong, Taibo Li, Dingjingyu Zhou, A. Narkar, Yufei Wu, Sean X. Sun, Tatianna C. Larman, Jin Zhu, Rong Li
          </td>
          <td>2024-07-10</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35393ed7857060735a43bccb7ec1795bc1e251a9" target='_blank'>
              Comprehensive molecular characterization of TFE3-rearranged renal cell carcinoma.
              </a>
            </td>
          <td>
            Cho-Rong Lee, J. Suh, Dongjun Jang, Bo-Yeong Jin, Jaeso Cho, Moses Lee, Hyungtai Sim, Minyong Kang, Jueun Lee, Ju Hyun Park, Kyoung-Hwa Lee, Geum-Sook Hwang, Kyung Chul Moon, Cheryn Song, J. Ku, C. Kwak, Hyeon-Hoe Kim, Sung-Yup Cho, Murim Choi, Chang Wook Jeong
          </td>
          <td>2024-08-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers. This phenomenon has been investigated since the late 1980s without resolution. Expanding beyond previous gene-centric studies, we investigated the co-occurrence in a genome-wide manner taking an evolutionary perspective. Mining of large-scale tumor aneuploidy data confirmed the previous finding of a small-scale longitudinal study that the most likely order is chromosome 10 loss followed by chromosome 7 gain. Extensive analysis of genomic and transcriptomic data from both patients and cell lines revealed that this co-occurrence can be explained by functional rescue interactions that are highly enriched on chromosome 7, which could potentially compensate for any detrimental consequences arising from the loss of chromosome 10. Transcriptomic data from various normal, non-cancerous human brain tissues were analyzed to assess which tissues may be most predisposed to tolerate compensation of chromosome 10 loss by chromosome 7 gain. The analysis indicated that the pre-existing transcriptomic states in the cortex and frontal cortex, where gliomas arise, are more favorable than other brain regions for compensation by rescuer genes that are active on chromosome 7. Collectively, these findings suggest that the phenomenon of chromosome 10 loss and chromosome 7 gain in gliomas is orchestrated by a complex interaction of many genes residing within these two chromosomes and provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff13af5a4c0be0182481407e6fc3feb597b938be" target='_blank'>
              Chromosome 7 gain compensates for chromosome 10 loss in glioma.
              </a>
            </td>
          <td>
            N. Nair, Alejandro A Schaffer, E. Gertz, Kuoyuan Cheng, Johanna Zerbib, A. Sahu, Gil Leor, E. Shulman, Kenneth D Aldape, Uri Ben-David, E. Ruppin
          </td>
          <td>2024-07-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b3ada799665e69e111a38c4d8d26afa853f4d7" target='_blank'>
              MethPhaser: methylation-based long-read haplotype phasing of human genomes
              </a>
            </td>
          <td>
            Yilei Fu, Sergey Aganezov, M. Mahmoud, J. Beaulaurier, S. Juul, T. Treangen, F. Sedlazeck
          </td>
          <td>2024-06-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The p53 family remains a captivating focus of an extensive number of current studies. Accumulating evidence indicates that p53 abnormalities rank among the most prevalent in cancer. Given the numerous existing studies, which mostly focus on the mutations, expression profiles, and functional perturbations exhibited by members of the p53 family across diverse malignancies, this review will concentrate more on less explored facets regarding p53 activation and stabilization by the nuclear pore complex (NPC) in cancer, drawing on several studies. p53 integrates a broad spectrum of signals and is subject to diverse regulatory mechanisms to enact the necessary cellular response. It is widely acknowledged that each stage of p53 regulation, from synthesis to degradation, significantly influences its functionality in executing specific tasks. Over recent decades, a large body of data has established that mechanisms of regulation, closely linked with protein activation and stabilization, involve intricate interactions with various cellular components. These often transcend canonical regulatory pathways. This new knowledge has expanded from the regulation of genes themselves to epigenomics and proteomics, whereby interaction partners increase in number and complexity compared with earlier paradigms. Specifically, studies have recently shown the involvement of the NPC protein in such complex interactions, underscoring the further complexity of p53 regulation. Furthermore, we also discuss therapeutic strategies based on recent developments in this field in combination with established targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f4ce5cb3bd6bcbf7187f9f514ca001d925085c" target='_blank'>
              Unlocking the Gateway: The Spatio-Temporal Dynamics of the p53 Family Driven by the Nuclear Pores and Its Implication for the Therapeutic Approach in Cancer
              </a>
            </td>
          <td>
            Dini Kurnia Ikliptikawati, Keiichi Makiyama, M. Hazawa, Richard W. Wong
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a widely used method for classifying cell types and states and revealing disease mechanisms. However, most contemporary scRNA-seq platforms fail to explore the multi-landscape of RNA. Here, we designed a microfluidic chip combined oligo-dT primers and Random Bridging Co-labeling (RBCL) RNA sequencing to develop an innovative Chigene scRNA-seq technology that can identify gene expression, mutations, and RNA splicing landscapes at the single-cell level. The Chigene scRNA-seq platform demonstrated exceptional performance, with minimal doublet rates of 0.94% (Chigene V1) and 1.93% (Chigene V2). Both versions exhibit high sensitivity, with Chigene V2 achieving nearly 100% RNA coverage and detecting over 1800 genes per cell on average. Targeted capture of single-cell gene mutations enhances mutation detection sensitivity. Moreover, this Chigene V2 platform has been validated in clinical samples for its ability to detect mutations, gene fusions and alternative splicing. The reliability of the platform was further corroborated using known functional gene mutation (CDKN1A) and fusion (FGFR3-TACC). To validate this method’s potential for discovering novel gene mutations in clinical samples, our investigation revealed an intriguing cell subpopulation carrying an ARHGAP5 mutation in urothelial carcinoma. These cells exhibited high-frequency mRNA splicing and exhibited specific crosstalk with T cells, distinguishing them from the subpopulation with the ARHGAP5 wild-type phenotype. Overall, this method provides a robust scRNA-seq platform suitable for comprehensive analyses of clinical specimens at different genetic information levels, thereby offering significant potential in the discovery of novel genes and interactions at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be600ce14b879188e7c67b73a02d6fde03c11a8c" target='_blank'>
              Identification of Multi-landscape and Cell Interactions in the Tumor Microenvironment through High-Coverage Single-Cell Sequencing
              </a>
            </td>
          <td>
            Wenlong Zhong, Ligang Wang, Tonglei Guo, Lihua Zhao, Daqin Wu, Fei Xie, Xiao Wang, Xiuxin Li, FangXiao Wang, Weiyue Gu, Tianxin Lin, Xu Chen
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/933aeab529c88805bcf4a9c969afbb698b11a342" target='_blank'>
              DNA methylation-mediated suppression of TUSC1 expression regulates the malignant progression of esophagogastric junction cancer
              </a>
            </td>
          <td>
            Zhiqiang Liu, Ganshu Xia, Xiaolong Liang, Baozhong Li, Jingyu Deng
          </td>
          <td>2024-07-23</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69e1db6874aa879e7cfcdc2ee3255c993cf70dae" target='_blank'>
              USP9X regulates the proliferation, survival, migration and invasion of gastric cancer cells by stabilizing MTH1
              </a>
            </td>
          <td>
            Wenji Xu, Yaping Zhang, Yingrui Su, Libin Li, Xinxia Yang, Lixing Wang, Hongzhi Gao
          </td>
          <td>2024-07-29</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Focal gene amplification, such as extrachromosomal DNA (ecDNA), plays an important role in cancer development and therapy resistance. While sequencing-based methodologies enable an unbiased identification of ecDNA, cytogenetic-based techniques, such as fluorescence in situ hybridization (FISH), remain time and cost-effective for identifying ecDNA in clinical specimens. The application of FISH in formalin-fixed paraffin-embedded (FFPE) tissue samples offers a unique avenue for detecting amplified genes, particularly when viable specimens are not available for karyotype examination. However, there is a lack of consensus procedures for this technique. This protocol provides comprehensive, fully optimized, step-by-step instructions for conducting FISH to detect gene amplification, including ecDNA, in FFPE tissue samples which present unique challenges that this protocol aims to overcome and standardize. By following this protocol, researchers can reproducibly acquire high-quality imaging data to assess gene amplification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4b4b06846303f4dbe1ceeb4a2ce5302dc13a1ff" target='_blank'>
              Robust Detection of Gene Amplification in Formalin-Fixed Paraffin-Embedded Samples by Fluorescence in situ Hybridization.
              </a>
            </td>
          <td>
            Collin Gilbreath, Yan Peng, Sihan Wu
          </td>
          <td>2024-07-12</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Immunofluorescence with antibodies against phosphorylated forms of H2AX (γH2AX) is revolutionizing our understanding of repair and signaling of DNA double-strand breaks (DSBs). Unfortunately, the pattern of γH2AX foci depends upon a number of parameters (nature of stress, number of foci, radiation dose, repair time, cell cycle phase, gene mutations, etc…) whose one of the common points is chromatin condensation/decondensation. Here, we endeavored to demonstrate how chromatin conformation affects γH2AX foci pattern and influences immunofluorescence signal. DSBs induced in non-transformed human fibroblasts were analyzed by γH2AX immunofluorescence with sodium butyrate treatment of chromatin applied after the irradiation that decondenses chromatin but does not induce DNA breaks. Our data showed that the pattern of γH2AX foci may drastically change with the experimental protocols in terms of size and brightness. Notably, some γH2AX minifoci resulting from the dispersion of the main signal due to chromatin decondensation may bias the quantification of the number of DSBs. We proposed a model called “Christmas light models” to tentatively explain this diversity of γH2AX foci pattern that may also be considered for any DNA damage marker that relocalizes as nuclear foci.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f2123c88ff3f4eb339edbfea97c8b7ce2af6a7" target='_blank'>
              When Chromatin Decondensation Affects Nuclear γH2AX Foci Pattern and Kinetics and Biases the Assessment of DNA Double-Strand Breaks by Immunofluorescence
              </a>
            </td>
          <td>
            A. Granzotto, Laura El Nachef, Juliette Restier-Verlet, Laurène Sonzogni, J. Al-Choboq, Michel Bourguignon, Nicolas Foray
          </td>
          <td>2024-06-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CCCTC-binding factor (CTCF) is an insulator protein that binds to a highly conserved DNA motif and facilitates regulation of three-dimensional (3D) nuclear architecture and transcription. CTCF binding sites (CTCF-BSs) reside in non-coding DNA and are frequently mutated in cancer. Our previous study identified a small subclass of CTCF-BSs that are resistant to CTCF knock down, termed persistent CTCF binding sites (P-CTCF-BSs). P-CTCF-BSs show high binding conservation and potentially regulate cell-type constitutive 3D chromatin architecture. Here, using ICGC sequencing data we made the striking observation that P-CTCF-BSs display a highly elevated mutation rate in breast and prostate cancer when compared to all CTCF-BSs. To address whether P-CTCF-BS mutations are also enriched in other cell-types, we developed CTCF-INSITE-a tool utilising machine learning to predict persistence based on genetic and epigenetic features of experimentally-determined P-CTCF-BSs. Notably, predicted P-CTCF-BSs also show a significantly elevated mutational burden in all 12 cancer-types tested. Enrichment was even stronger for P-CTCF-BS mutations with predicted functional impact to CTCF binding and chromatin looping. Using in vitro binding assays we validated that P-CTCF-BS cancer mutations, predicted to be disruptive, indeed reduced CTCF binding. Together this study reveals a new subclass of cancer specific CTCF-BS DNA mutations and provides insights into their importance in genome organization in a pan-cancer setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994d6f59097fc218409b42c901b1ae72776752f9" target='_blank'>
              Machine learning enables pan-cancer identification of mutational hotspots at persistent CTCF binding sites.
              </a>
            </td>
          <td>
            Wenhan Chen, Y. Zeng, Joanna Achinger-Kawecka, Elyssa Campbell, Alicia K Jones, Alastair G Stewart, Amanda Khoury, Susan J Clark
          </td>
          <td>2024-07-02</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ad718715d0e23f67c54707d35f7ee63dc651a7" target='_blank'>
              Homeotic DUX4 Genes Shape Dynamic Inter-Chromosomal Contacts with Nucleoli in Human Cells.
              </a>
            </td>
          <td>
            E. S. Klushevskaya, I. R. Alembekov, Y. V. Kravatsky, N. A. Tchurikov
          </td>
          <td>2024-07-13</td>
          <td>Doklady. Biochemistry and biophysics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The hyper-modified DNA base J helps control termination of Pol II transcription at polycistronic transcription units (PTUs) in T. brucei and L. major, allowing epigenetic control of gene expression. The Telomere Repeat-containing RNA (TERRA) is synthesized in T. brucei by Pol I readthrough transcription of a telomeric PTU. While little is understood regarding TERRA synthesis and function, the hyper-modified DNA base J is highly enriched at telomeres in L. major promastigotes. We now show that TERRA is synthesized by Pol II in L. major and loss of base J leads to increased TERRA. For at least one site, the increased TERRA is by Pol II readthrough transcription from an adjacent PTU. Furthermore, Pol II readthrough defects and increased TERRA correlate with increased differentiation of promastigotes to the infectious metacyclic life stage and decreased cell viability. These results help explain the essential nature of base J in Leishmania and provide insight regarding epigenetic control of coding and non-coding RNA expression and parasite development during the life cycle of L. major.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ad169721538f0b20f74e275af5214380d4677b0" target='_blank'>
              Epigenetic regulation of TERRA transcription and metacyclogenesis by base J in Leishmania major
              </a>
            </td>
          <td>
            R. Kieft, D. Reynolds, Robert Sabatini
          </td>
          <td>2024-06-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1238d9815100a9d52edecd0a776b444d4a9b4a2f" target='_blank'>
              N6-methyladenosine modification of circMARK2 enhances cytoplasmic export and stabilizes LIN28B, contributing to the progression of Wilms tumor
              </a>
            </td>
          <td>
            Guannan Shu, Zhang Zhao, Tianxin Zhao, Changmi Deng, Jiangquan Zhu, Yufeng Han, Minyu Chen, Jiajia Jing, Gaochen Bai, Dian Li, Feng Li, Jing He, W. Fu, Guochang Liu
          </td>
          <td>2024-07-11</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3ef3973d270e32f48aaa19a00bef4e18bca5be" target='_blank'>
              Research Progress of Long Non-coding RNA-ZFAS1 in Malignant Tumors.
              </a>
            </td>
          <td>
            Xin Liu, Zhong Ma, Xianxu Zhang, Shicheng Li, Jiangdong An, Zhiqiang Luo
          </td>
          <td>2024-07-26</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87375be953b6b1e62c0930938d4b8c30692440ac" target='_blank'>
              Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants.
              </a>
            </td>
          <td>
            Max Frenkel, James E. Corban, Margaux L. A. Hujoel, Zachary Morris, Srivatsan Raman
          </td>
          <td>2024-07-24</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="All routine clinical treatments for colorectal cancer include 5-fluorouracil (5-FU), which cannot counteract recurrence and metastases formation. As the pyrimidine analog 5-FU can impact multiple pathways including both DNA and RNA metabolism, studying its mode of actions could lead to improved therapies. Using a dedicated reporter system for lineage-tracing and deep translatome profiling we demonstrate that 5-FU causes some colorectal cancer cells to tolerate the drug, due to a durable translational reprogramming that sustains cell plasticity. This period of drug tolerance coincides with specific translational activation of genes coding for proteins with major pro-tumoral functions. We unravel a major unexpected translational overexpression of the pro-inflammatory and pro-tumoral IL-8 cytokine, alongside other anti-apoptotic, senescence-associated secretory phenotype and cancer-related senescence phenotype genes. Given the adverse prognostic implications of elevated IL-8 levels across various cancers, our findings suggest IL-8 targeting could counteract 5-FU resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c3f76dde38afaccb213072b4b26b46e5ff73bd2b" target='_blank'>
              Translational control of cell plasticity drives 5-FU tolerance
              </a>
            </td>
          <td>
            Mounira Chalabi-Dchar, Olivia Villeronce, J. Ripoll, A. Vincent, T. Fenouil, Rita Khoueiry, Jérôme Kucharczak, Laura Jentschel, Frédéric Catez, Arnaud M Vigneron, Julie Tréguier, Céline Mandier, Céline Bouclier, Jihane Vitre, Louise Lagerqvist, Armelle Choquet, Z. Herceg, C. Machon, Jérôme Guitton, Alexandre David, Eric Solary, David Bernard, Nadine Martin, Eric Rivals, N. D. Venezia, Julie Pannequin, J- J. Diaz
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>67</td>
        </tr>

        <tr id="Glioblastoma (GBM), the most malignant primary brain tumor in adults, has poor prognosis irrespective of therapeutic advances due to its radio-resistance and infiltrative growth into brain tissue. The present study assessed functions and putative druggability of protein breast cancer type 1 susceptibility protein (BRCA1)-associated Ataxia telangiectasia mutated (ATM)-activator 1 (BRAT1) as a crucial factor driving key aspects of GBM, including enhanced DNA damage response and tumor migration. By a stable depletion of BRAT1 in GBM and glioma stem-like (GSC) cell lines, we observed a delay in DNA double-strand break repair and increased sensitivity to radiation treatment, corroborated by in vitro and in vivo studies demonstrating impaired tumor growth and invasion. Proteomic analyses further emphasize the role of BRAT1’s cell migration and invasion capacity, with a notable proportion of downregulated proteins associated with these processes. In line with the genetic manipulation, we found that treatment with the BRAT1 inhibitor Curcusone D (CurD) significantly reduced GSC migration and invasion in an ex vivo slice culture model, particularly when combined with irradiation, resulting a synergistic inhibition of tumor growth and infiltration. Our results reveal that BRAT1 contributes to GBM growth and invasion and suggest that therapeutic inhibition of BRAT1 with CurD or similar compounds might constitute a novel approach for anti-GBM directed treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6c6e398faf7890164e1dab60592356505fb1252c" target='_blank'>
              BRAT1 - a new therapeutic target for glioblastoma
              </a>
            </td>
          <td>
            Alicia Haydo, Jennifer Schmidt, Alisha Crider, Tim Kögler, Johanna Ertl, S. Hehlgans, Marina E. Hoffmann, Rajeshwari Rathore, Ömer Güllülü, Yecheng Wang, Xiangke Zhang, Christel Herold-Mende, Francesco Pampaloni, Irmgard Tegeder, Ivan Dikic, Mingji Dai, Franz Rödel, D. Kögel, B. Linder
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Telomeres are specialized structures at the ends of linear chromosomes that protect them from degradation and fusion. It’s replication is a complex process that involves both DNA polymerases and a specialized enzyme called telomerase which is a ribonucleoprotein complex that synthesizes telomeric DNA by using an internal RNA template. However, telomerase alone cannot fully replicate the telomeric DNA, and requires the cooperation of other factors, such as shelterin, CST, and DNA repair proteins. Telomere replication is tightly regulated by various mechanisms, such as cell cycle checkpoints, telomere length homeostasis, and telomere position effect and dysregulation of it can lead to genomic instability, cellular senescence, and cancer. Therefore, understanding the molecular details of telomere replication is crucial for elucidating the role of telomeres in aging and disease. ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e590a2bd1420d5e3b8ed0e5debba5cfdd475b0f8" target='_blank'>
              Unravelling the Intricacies of Telomere Replication: A Molecular Conundrum
              </a>
            </td>
          <td>
            Neelabh Datta
          </td>
          <td>2024-07-16</td>
          <td>Canadian Journal for the Academic Mind</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC.Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs.Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes.These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Glioblastoma multiforme (GBM) is an aggressive and lethal form of brain cancer with few effective treatments. In this context, Zika virus has emerged as a promising therapeutic agent due to its ability to selectively infect and kill GBM cells. To elucidate these mechanisms and expand the landscape of oncolytic virotherapy, we pursued a transcriptomic meta-analysis comparing the molecular signatures of Zika infection in GBM and neuroblastoma (NBM). Over-representation analysis of dysregulated coding genes showed significant enrichment of tumor necrosis factor (TNF), NF-κB, and p53 signaling pathways. A refined list of long non-coding RNAs consistently dysregulated in Zika-infected GBMs was also developed. Functional review of these candidates revealed their potential regulatory role in Zika-mediated oncolysis. We performed validation of the less-researched targets in adult and pediatric GBM cell lines and found significant differential regulation, as predicted. Altogether, our results provide novel insights into the molecular mechanisms underlying the effect of Zika on GBM. We highlight potential therapeutic targets that could be further interrogated to improve the efficacy of tumor cell death and the utility of Zika as an adjuvant virotherapy for GBM and other related cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d5fb2f82f19cb82c54f2e078dc5fd330417c5a7" target='_blank'>
              Transcriptomic Meta-analysis Identifies Long Non-Coding RNAs Mediating Zika’s Oncolytic Impact in Glioblastoma Multiforme
              </a>
            </td>
          <td>
            Youssef A. Kousa, Shriya Singh, Allison Horvath, Federica Tomasso, Javad Nazarian, Lisa Henderson, Tamer A. Mansour
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Sequencing mitochondrial DNA (mtDNA) variants from single cells has resolved clonality and lineage in native human samples and clinical specimens. Prior work established that heteroplasmic mtDNA variants can be used to delineate clonality in hematopoiesis, but they have limited ability to reconstruct cellular phylogenies. However, a recent report by Weng et al. challenges the current paradigm by describing an unprecedented number of shared mtDNA variants between cells that reportedly resolve high-resolution phylogenetic trees. We re-examined the claims of Weng et al., and identified two major points of concern regarding this unprecedented connectedness. First, shared variants between cells are disproportionately detected in a single molecule per cell, and second, these variants are enriched 10–20-fold at the edges of mtDNA molecules, reminiscent of artifacts reported in other sequencing approaches. Further, our analyses show that pruning low support and likely artificial mtDNA variants removes nearly all of the reported phylogenetic structure. Thus, we strongly caution against using mtDNA variant calling workflows that rely on minimal evidence, including the computational pipeline introduced in Weng et al., as variants with high connectedness and low evidence are likely artifacts that lead to the construction of false phylogenies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae6d387965c56c26ee411bc842b91f11e043e8f3" target='_blank'>
              Artifacts in single-cell mitochondrial DNA mutation analyses misinform phylogenetic inference
              </a>
            </td>
          <td>
            Caleb A. Lareau, M. Chapman, Livius Penter, T. Nawy, D. Pe’er, Leif S. Ludwig
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetics are the changes due to DNA methylation, histone modifications, and noncoding RNA regulation, which play a crucial part in the expression of genes. Although these changes do not alter the sequence of the DNA, they are significant in determining the phenotype and inheritance of the genes. The impact of epigenetics on inherited diseases like Fragile X syndrome, Angelman syndrome, and Prader-Willi syndrome emphasizes the significance of studying epigenetic dysregulation in disease pathology. The current review aims to investigate the role of epigenetic mechanisms in inherited diseases and to evaluate the utility of molecular diagnostic tools for epigenetic analysis in disease diagnosis. Case studies of diseases such as Duchenne muscular dystrophy and Prader-Willi syndrome highlight the clinical relevance of epigenetic analysis in disease diagnosis and management. This research is conducted through an extensive analysis of recent literature to explore the epigenetic mechanisms underlying inherited diseases. Additionally, advanced molecular diagnostic techniques like droplet digital PCR (ddPCR), transposase-based bisulfite tagging, and data mining coupled with conventional experimental procedures. However, the intricate relationship between genetic and epigenetic factors and challenges related to inclined data need to be addressed. Future research endeavors to elucidate the role of epigenetic modifications in disease pathogenesis and develop strategies for personalized medicinal treatment and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f24f0960efd210e8ee51cbad54070948fecc2ef" target='_blank'>
              A Review on Epigenetics of Human Inherited Diseases: Molecular Diagnosis
              </a>
            </td>
          <td>
            Bibi Shazia Sabir
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Innovative Science and Research Technology (IJISRT)</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="The endonuclease activity of Pms1 directs mismatch repair by generating a nick in the newly replicated DNA strand. Inactivating Pms2, the human homologue of yeast Pms1, increases the chances of colorectal and uterine cancers. Here we use whole genome sequencing to show that loss of this endonuclease activity, via the pms1-DE variant, results in strong mutator effects throughout the Saccharomyces cerevisiae genome. Mutation rates are strongly increased for mutations resulting from all types of single-base substitutions and for a wide variety of single- and multi-base indel mutations. Rates for these events are further increased in strains combining pms1-DE with mutator variants of each of the three major leading and lagging strand replicases. In all cases, mutation rates, spectra, biases, and context preferences are statistically indistinguishable from strains with equivalent polymerases but lacking initial mismatch recognition due to deletion of MSH2. This implies that, across the nuclear genome, strand discrimination via the Pms1 endonuclease is as important for MMR as is initial mismatch recognition by Msh2 heterodimers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c81863bf8b4bcd0533ef4d09eddc5a93c0da4b" target='_blank'>
              Instability throughout the Saccharomyces cerevisiae genome resulting from Pms1 endonuclease deficiency.
              </a>
            </td>
          <td>
            S. Lujan, Marta A. Garbacz, S. E. Liberti, Adam B. Burkholder, Thomas A. Kunkel
          </td>
          <td>2024-07-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3101f7048ced189804d5ffb2bbda06ceae417ac" target='_blank'>
              CRISPR-Cas9-mediated deletion enhancer of MECOM play a tumor suppressor role in ovarian cancer.
              </a>
            </td>
          <td>
            Yujie Chen, Qiuwen Jiang, Yingzhuo Xue, Weiguan Chen, Minhui Hua
          </td>
          <td>2024-07-12</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) are characterized by the presence of either a single enlarged nucleus or multiple nuclei and are closely associated with tumor progression and treatment resistance. These cells contribute significantly to cellular heterogeneity and can arise from various stressors, including radiation, chemotherapy, hypoxia, and environmental factors. The formation of PGCCs can occur through mechanisms such as endoreplication, cell fusion, cytokinesis failure, mitotic slippage, or cell cannibalism. Notably, PGCCs exhibit traits similar to cancer stem cells (CSCs) and generate highly invasive progeny through asymmetric division. The presence of PGCCs and their progeny is pivotal in conferring resistance to chemotherapy and radiation, as well as facilitating tumor recurrence and metastasis. This review provides a comprehensive analysis of the origins, potential formation mechanisms, stressors, unique characteristics, and regulatory pathways of PGCCs, alongside therapeutic strategies targeting these cells. The objective is to enhance the understanding of PGCC initiation and progression, offering novel insights into tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66163bd868b4436fb4d269ccebdc926754441914" target='_blank'>
              Polyploid giant cancer cells: origin, possible pathways of formation, characteristics, and mechanisms of regulation
              </a>
            </td>
          <td>
            Pan Liu, Lili Wang, Huiying Yu
          </td>
          <td>2024-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70f393647e59fb1238cccba92da569418e311d5e" target='_blank'>
              Deciphering the genetics and mechanisms of predisposition to multiple myeloma
              </a>
            </td>
          <td>
            M. Went, Laura Duran-Lozano, G. Halldorsson, Andrea Gunnell, Nerea Ugidos-Damboriena, P. Law, Ludvig Ekdahl, Amit Sud, G. Thorleifsson, Malte Thodberg, T. Olafsdottir, Antton Lamarca-Arrizabalaga, C. Cafaro, Abhishek Niroula, Ram Ajore, Aitzkoa Lopez de Lapuente Portilla, Zain Ali, M. Pertesi, Hartmut Goldschmidt, L. Stefánsdóttir, S. Kristinsson, S. Stacey, T. Love, Saemundur Rognvaldsson, R. Hájek, Pavel Vodicka, Ulrika Pettersson-Kymmer, Florentin Späth, C. Schinke, F. van Rhee, P. Sulem, Egil Ferkingstad, Grimur Hjorleifsson Eldjarn, U. Mellqvist, I. Jónsdóttir, Gareth Morgan, P. Sonneveld, Anders Waage, N. Weinhold, H. Thomsen, A. Försti, Markus Hansson, Annette Juul-Vangsted, U. Thorsteinsdóttir, Kari Hemminki, M. Kaiser, T. Rafnar, Kári Stefánsson, R. Houlston, Björn Nilsson
          </td>
          <td>2024-08-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>182</td>
        </tr>

        <tr id="Transient melting of the duplex-DNA (B-DNA) during DNA transactions allows repeated sequences to fold into non-B DNA structures, including DNA junctions and G-quadruplexes. These noncanonical structures can act as impediments to DNA polymerase progression along the duplex, thereby triggering DNA damage and ultimately jeopardizing genomic stability. Their stabilization by ad hoc ligands is currently being explored as a putative anticancer strategy since it might represent an efficient way to inflict toxic DNA damage specifically to rapidly dividing cancer cells. The relevance of this strategy is only emerging for three-way DNA junctions (TWJs) and, to date, no molecule has been recognized as a reference TWJ ligand, featuring both high affinity and selectivity. Herein, we characterize such reference ligands through a combination of in vitro techniques comprising affinity and selectivity assays (competitive FRET-melting and TWJ Screen assays), functional tests (qPCR and Taq stop assays), and structural analyses (molecular dynamics and NMR investigations). We identify novel azacryptands TrisNP-amphi and TrisNP-ana as the most promising ligands, interacting with TWJs with high affinity and selectivity. These ligands represent new molecular tools to investigate the cellular roles of TWJs and explore how they can be exploited in innovative anticancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/406c1c1a744a4c655e84090a57ef4d3f115a69ef" target='_blank'>
              Structural Optimization of Azacryptands for Targeting Three-Way DNA Junctions.
              </a>
            </td>
          <td>
            A. Pipier, Titouan Chetot, Apollonia Kalamatianou, Nicolas Martin, Maëlle Caroff, Sébastien Britton, Nicolas Chéron, Lukáš Trantírek, Anton Granzhan, D. Monchaud
          </td>
          <td>2024-06-14</td>
          <td>Angewandte Chemie</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/245be040147fd6375f5eb20fea5e8094910fd2c5" target='_blank'>
              DNA Damage and Chromatin Rearrangement Work Together to Promote Neurodegeneration.
              </a>
            </td>
          <td>
            Harman Sharma, Sushma Koirala, Yee Lian Chew, Anna Konopka
          </td>
          <td>2024-07-08</td>
          <td>Molecular neurobiology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Horizontal gene transfer is a key driver of bacterial evolution, but it also presents severe risks to bacteria by introducing invasive mobile genetic elements. To counter these threats, bacteria have developed various defense systems, including prokaryotic Argonautes (pAgo) and the DNA Defense Module DdmDE system. Through biochemical analysis, structural determination, and in vivo plasmid clearance assays, we elucidate the assembly and activation mechanisms of DdmDE, which eliminates small, multicopy plasmids. We demonstrate that DdmE, a pAgo-like protein, acts as a catalytically inactive, DNA-guided, DNA-targeting defense module. In the presence of guide DNA, DdmE targets plasmids and recruits a dimeric DdmD, which contains nuclease and helicase domains. Upon binding to DNA substrates, DdmD transitions from an autoinhibited dimer to an active monomer, which then translocates along and cleaves the plasmids. Together, our findings reveal the intricate mechanisms underlying DdmDE-mediated plasmid clearance, offering fundamental insights into bacterial defense systems against plasmid invasions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fac230afd0ac05cd0569717ab193d8b0059c7437" target='_blank'>
              DdmDE eliminates plasmid invasion by DNA-guided DNA targeting
              </a>
            </td>
          <td>
            Xiao-Yuan Yang, Zhangfei Shen, Chen Wang, Kotaro Nakanishi, Tian-Min Fu
          </td>
          <td>2024-07-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ee3ef93f75d18cca805d005d653d62f6f76b88e" target='_blank'>
              Comprehensive pan-cancer analysis reveals that C5orf34 regulates the proliferation and mortality of lung cancer.
              </a>
            </td>
          <td>
            Mengmeng Yang, Yuhan Deng, Yu Ma, Chunli Song, Zhenhua Wu, Xiayimaierdan Yibulayin, Xiaohong Sun, Yunquan Guo, Dan He
          </td>
          <td>2024-06-29</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b11c9fdbfd43bb000396d5cb58b5ead9a774473" target='_blank'>
              Hypermethylated RASAL1’s promotive role in chemoresistance and tumorigenesis of choriocarcinoma was regulated by TET2 but not DNMTs
              </a>
            </td>
          <td>
            Xianling Zeng, Rui An, Ruixia Guo, Han Li
          </td>
          <td>2024-08-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Polyploidization has been recognized as a major force in plant evolution. With the continuous progress in sequencing technologies and genome assembly algorithms, high-quality chromosome-level assemblies of polyploid genomes have become increasingly attainable. However, accurately delineating these assemblies into subgenomes remains a challenging task, especially in case where known diploid ancestors are absent. In this study, we introduce a novel approach that leverages long terminal repeat retrotransposons (LTR-RTs) coupled with the Serial Similarity Matrix (SSM) method to assign genome assemblies to subgenomes, particularly beneficial for those without known diploid progenitor genomes. The SSM method helps identify subgenome-specific LTRs and facilitates the inference of the timing of allopolyploidization events. We validated the efficacy of the SSM approach using well-studied allopolyploidy genomes, Eragrostis tef and Gossypium hirsutum, alongside an artificially created allotetraploid genome by merging two closely related diploid species, Glycine max and G. soja. Our results demonstrated the robustness of the method and its effectiveness in assigning chromosomes to subgenomes. We then applied the SSM method to the octoploid strawberry genome. Our analysis revealed three allopolyploidization events in the evolutionary trajectory of the octoploid strawberry genome, shedding light on the evolutionary process of the origin of the octoploid strawberry genome and enhancing our understanding of allopolyploidization in this complex species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e83704c02da2a4c7891265de8e37966482ba3e64" target='_blank'>
              Deciphering octoploid strawberry evolution with serial LTR similarity matrices for subgenome partition
              </a>
            </td>
          <td>
            Haomin Lyu, Shujun Ou, W. C. Yim, Qingyi Yu
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/daaaeb917f1640a6de2396b29c1a5a0f942bd758" target='_blank'>
              Enhanced cancer cell proliferation and aggressive phenotype counterbalance in breast cancer with high BRCA1 gene expression.
              </a>
            </td>
          <td>
            Kohei Chida, M. Oshi, A. M. Roy, Takumi Sato, Maya Penelope Takabe, Li Yan, I. Endo, Kenichi Hakamada, K. Takabe
          </td>
          <td>2024-07-07</td>
          <td>Breast cancer research and treatment</td>
          <td>0</td>
          <td>61</td>
        </tr>

        <tr id="Background: Colorectal cancer (CRC) presents a significant global health burden, with high rates of incidence and mortality, and an urgent need to improve prognosis. STM2457, a novel small molecule inhibitor specific for N6-methyladenosine (m6A) catalytic enzyme Methyltransferase-like 3 (METTL3) has implicated significant treatment potentials in a few of types of cancer. However, its impact and underlying mechanism are still unclear in CRC cells. Methods: We used CCK-8 and colony formation assay to observe cell growth, flow cytometry and TUNEL approaches to detect cell apoptosis under the treatment of STM2457 on CRC cells in vitro or in vivo. RNA-sequencing, qRT-PCR and western blotting were performed to explore downstream effectors of STM2457. Messenger RNA stability was evaluated by qRT-PCR after treatment with actinomycin D. The methylated RNA immunoprecipitation (MeRIP) qPCR, dual-luciferase reporter analyses and m6A dot blotting were carried out to measure the m6A modification. Associated gene expression pattern and clinical relevance in CRC clinical tissue samples were analyzed using online database. Results: STM2457 exhibited a strong influence on cell growth suppression and apoptosis of CRC cells in vitro and subcutaneous xenograft growth in vivo. Asparagine synthetase (ASNS) was markedly downregulated upon STM2457 treatment or METTL3 knockdown and exogenous overexpression of ASNS could rescue the biological defects induced by STM2457. Mechanistically, the downregulation of ASNS by STM2457 may be due to the decrease of m6A modification level in ASNS mRNA mediated by METTL3. Conclusions: Our findings suggest that STM2457 may serve as a potential therapeutic agent and ASNS may be a new promising therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d2ad6ae3e7c70f86f85ea0a48c17dd82345f44" target='_blank'>
              METTL3 Inhibition Suppresses Cell Growth and Survival in Colorectal Cancer via ASNS Downregulation
              </a>
            </td>
          <td>
            Yang Yu, Yanan Hai, Hongfeng Zhou, Wenfang Bao, Xiaowei Hu, Yong Gao, Jin Wu
          </td>
          <td>2024-07-16</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a62302568e13d5d4ea358c501360fb449540886f" target='_blank'>
              Epigenetic regulation of HERVs: Implications for cancer immunotherapy.
              </a>
            </td>
          <td>
            Joo Mi Yi
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Cancer is one of the main reasons for death, and it threatens human life and health. Both the environment and genes can lead to cancers. It dates back more than a million years; more importantly, tumor cells can not be detected until they grow to a large number. Currently, cancers are treated with surgical excision or non-surgical procedures. By studying the interaction between ncRNAs and PKM2, we aim to provide new targets for diagnosis, treatment, and prognosis for cancers. Read relevant articles and made a summary and classification. Non-coding RNAs (ncRNAs) are RNAs that do not code for proteins. They perform a function in transcription and translation and can be used as targets for cancer therapy. Pyruvate kinase M2 (PKM2) is a form of PKM, and it catalyzes the glycolysis of the final cellular processes to promote tumorigenesis. Not only that, but it also plays non-metabolic functions, including the expression of the gene, cell proliferation, cell migration, and tumor angiogenesis in cancer cells. The existing studies have found that microRNAs (miRNAs), long non-coding RNAs (lncRNAs), and circular RNAs (circRNAs) can promote or inhibit the aerobic glycolysis of cancer cells by affecting PKM2, which increases or decrease the risk of cancers and affect the progression of cancers. This review focuses on the mechanism of ncRNAs regulating PKM2 in cancers and summarizes the roles of some ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135ae3dc56b2a8f0f10f7d3ae82e64ec70f96983" target='_blank'>
              Emerging Roles of ncRNAs Regulating PKM2 in Cancer Progression.
              </a>
            </td>
          <td>
            Haimei Xie, Jia Yu, Zhiwen Ou, Xiaoyong Lei, Xiaoyan Yang
          </td>
          <td>2024-07-15</td>
          <td>Current pharmaceutical biotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66955c4f03e0dcce369a0bced796c5c5114b438" target='_blank'>
              Transcriptional dysregulation and insights into clinical implications in melanoma
              </a>
            </td>
          <td>
            Chen Shen, Mengjiao Chen, Xuanbingning Nian, Jun Li, Juan Tao
          </td>
          <td>2024-06-24</td>
          <td>Holistic Integrative Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69d60865849fb8ffb34e49ed542490f33241520a" target='_blank'>
              DNAJC1 facilitates glioblastoma progression by promoting extracellular matrix reorganization and macrophage infiltration
              </a>
            </td>
          <td>
            Han Zhang, Wenjing Zheng, Xu Chen, Long Sa, Yi Huo, Lingling Zhang, Lequn Shan, Tao Wang
          </td>
          <td>2024-06-22</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135134a62bac76cb46dae2f04090c849f2c01172" target='_blank'>
              Targeting chromosomal instability in patients with cancer.
              </a>
            </td>
          <td>
            Duaa H Al-Rawi, Emanuele Lettera, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-11</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/baaa860e0b1c85319e6d97b0864f33c2adb3744d" target='_blank'>
              Long non-coding RNA NEAT1 induced by BHLHE40 activates Wnt/β-catenin signaling and potentiates colorectal cancer progression
              </a>
            </td>
          <td>
            Anlong Ji, Hui Li, Xiangwei Fu, Yourong Zhang, Yanhe Liu
          </td>
          <td>2024-08-04</td>
          <td>Cell Division</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The Chromodomain helicase DNA-binding protein 1-like (CHD1L) is a nucleosome remodeling enzyme, which plays a key role in chromatin relaxation during the DNA damage response. Genome editing has shown that deletion of CHD1L sensitizes cells to PARPi, but the effect of its pharmacological inhibition has not been defined. Triple-negative breast cancer SUM149PT, HCC1937, and MDA-MB-231 cells were used to assess the mechanism of action of the CHD1Li OTI-611. Cytotoxicity as a single agent or in combination with standard-of-care treatments was assessed in tumor organoids. Immunofluorescence was used to assess the translocation of PAR and AIF to the cytoplasm or the nucleus and to study markers of DNA damage or apoptosis. Trapping of PARP1/2 or CHD1L onto chromatin was also assessed by in situ subcellular fractionation and immunofluorescence and validated by Western blot. We show that the inhibition of CHD1L’s ATPase activity by OTI-611 is cytotoxic to triple-negative breast cancer tumor organoids and synergizes with PARPi and chemotherapy independently of the BRCA mutation status. The inhibition of the remodeling function blocks the phosphorylation of H2AX, traps CHD1L on chromatin, and leaves PAR chains on PARP1/2 open for hydrolysis. PAR hydrolysis traps PARP1/2 at DNA damage sites and mediates PAR translocation to the cytoplasm, release of AIF from the mitochondria, and induction of PARthanatos. The targeted inhibition of CHD1L’s oncogenic function by OTI-611 signifies an innovative therapeutic strategy for breast cancer and other cancers. This approach capitalizes on CHD1L-mediated DNA repair and cell survival vulnerabilities, thereby creating synergy with standard-of-care therapies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401b6daf1d2a20088ba1aa72a3182ab37cb9c8e4" target='_blank'>
              CHD1L Regulates Cell Survival in Breast Cancer and Its Inhibition by OTI-611 Impedes the DNA Damage Response and Induces PARthanatos
              </a>
            </td>
          <td>
            Rita Sala, Hector Esquer, Timothy Kellett, Jeffrey T. Kearns, Paul Awolade, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2024-08-06</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Breast cancer (BRCA) remains a significant global health challenge due to its prevalence and lethality, exacerbated by the development of resistance to conventional therapies. Therefore, understanding the molecular mechanisms underpinning chemoresistance is crucial for improving therapeutic outcomes. Human deoxycytidine triphosphate pyrophosphatase 1 (DCTPP1) has emerged as a key player in various cancers, including BRCA. DCTPP1, involved in nucleotide metabolism and maintenance of genomic stability, has been linked to cancer cell proliferation, survival, and drug resistance. This study evaluates the role of DCTPP1 in BRCA prognosis and chemotherapy response. Data from the Cancer Genome Atlas Program (TCGA), Genotype-Tissue Expression (GTEx), and Gene Expression Omnibus (GEO) repositories, analyzed using GEPIA and Kaplan–Meier Plotter, indicate that high DCTPP1 expression correlates with poorer overall survival and increased resistance to chemotherapy in BRCA patients. Further analysis reveals that DCTPP1 gene expression is up-regulated in non-responders to chemotherapy, particularly in estrogen receptor (ER)-positive, luminal A subtype patients, with significant predictive power. Additionally, in vitro studies show that DCTPP1 gene expression increases in response to 5-fluorouracil and doxorubicin treatments in luminal A BRCA cell lines, suggesting a hypothetical role in chemoresistance. These findings highlight DCTPP1 as a potential biomarker for predicting chemotherapy response and as a therapeutic target to enhance chemotherapy efficacy in BRCA patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34c5b36472a08309e9aaa6675581062a8fba187a" target='_blank'>
              DCTPP1 Expression as a Predictor of Chemotherapy Response in Luminal A Breast Cancer Patients
              </a>
            </td>
          <td>
            Juan P Muñoz, Diego Soto-Jiménez, Gloria M. Calaf
          </td>
          <td>2024-08-02</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="DNA replication is regulated by factors that promote or inhibit initiation. In Bacillus subtilis, YabA is a negative regulator of DNA replication initiation while the newly identified kinase CcrZ is a positive regulator. The consequences of under-initiation or over-initiation of DNA replication to genome stability remain unclear. In this work, we measure origin to terminus ratios as a proxy for replication initiation activity. We show that ΔccrZ and several ccrZ alleles under-initiate DNA replication while ablation of yabA or overproduction of CcrZ leads to over-initiation. We find that cells under-initiating DNA replication have few incidents of replication fork stress as determined by low formation of RecA-GFP foci compared with wild type. In contrast, cells over-initiating DNA replication show levels of RecA-GFP foci formation analogous to cells directly challenged with DNA damaging agents. We show that cells under-initiating and over-initiating DNA replication were both sensitive to mitomycin C and that changes in replication initiation frequency cause increased sensitivity to genotoxic stress. With these results, we propose that cells under-initiating DNA replication are sensitive to DNA damage due to a shortage of DNA for repair through homologous recombination. For cells over-initiating DNA replication, we propose that an increase in the number of replication forks leads to replication fork stress which is further exacerbated by chromosomal DNA damage. Together, our study shows that DNA replication initiation frequency must be tightly controlled as changes in initiation influence replication fork fate and the capacity of cells to efficiently repair damage to their genetic material. IMPORTANCE The regulation of DNA replication is fundamental to cell growth and cell cycle control. In eukaryotes under-initiation or over-initiation leads to genome instability. For bacteria, it is unclear how changes in replication initiation frequency impact DNA replication status and genome integrity. We show that tight regulation of DNA replication initiation is critical for maintaining genome integrity. Cells over-initiating or under-initiating DNA replication are sensitive to DNA damage. Further, cells over-initiating DNA replication experience replication fork stress at levels that phenocopy cells encountering DNA damage from the crosslinking agent mitomycin C. Our results establish the critical importance of properly regulating DNA replication initiation frequency because an imbalance in initiation results in replication fork perturbations, deficiencies in DNA repair, and genome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16d74101c8dd440972ee7655bf02cbe81950a771" target='_blank'>
              DNA replication initiation timing is important for maintaining genome integrity
              </a>
            </td>
          <td>
            Tristan T. Reed, Abigail H. Kendal, Katherine J. Wozniak, Lyle A. Simmons
          </td>
          <td>2024-06-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21a2a05c7ef392ef903b88cba17ae6f2c9311cf5" target='_blank'>
              Mechanistic patterns and clinical implications of oncogenic tyrosine kinase fusions in human cancers
              </a>
            </td>
          <td>
            T. Cheong, Ahram Jang, Qi Wang, Giulia Leonardi, B. Ricciuti, J. Alessi, Alessandro Di Federico, Mark M Awad, Maria K Lehtinen, Marian H Harris, Roberto Chiarle
          </td>
          <td>2024-06-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0bff8174988ff460b1ed14cde5b076ac3673528" target='_blank'>
              Methods and applications of genome-wide profiling of DNA damage and rare mutations.
              </a>
            </td>
          <td>
            Gerd P Pfeifer, Seung-Gi Jin
          </td>
          <td>2024-06-25</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/522828f236448b54c15f6a0cec58172ce3dd65a1" target='_blank'>
              Epigenetic age acceleration is a distinctive trait of epithelioid sarcoma with potential therapeutic implications.
              </a>
            </td>
          <td>
            Simon Haefliger, O. Chervova, Christopher Davies, Chet Loh, R. Tirabosco, F. Amary, N. Pillay, Steve Horvath, Stephan Beck, Adrienne M. Flanagan, I. Lyskjær
          </td>
          <td>2024-06-16</td>
          <td>GeroScience</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261ea82053aaf73b2c3fb5ebe769c19bae07d5fa" target='_blank'>
              53BP1 loss elicits cGAS-STING-dependent antitumor immunity in ovarian and pancreatic cancer
              </a>
            </td>
          <td>
            Yajie Sun, Jeffrey Patterson-Fortin, Sen Han, Zhe Li, Zuzanna Nowicka, Yuna Hirohashi, Susan Kilgas, Jae Kyo Yi, A. Spektor, Wojciech Fendler, Panagiotis A Konstantinopoulos, Dipanjan Chowdhury
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23ed7881dc424017c9838fade2801a5a4db0c319" target='_blank'>
              Bidirectional linkage of DNA barcodes for the multiplexed mapping of higher-order protein interactions in cells.
              </a>
            </td>
          <td>
            Yu Liu, Noah R. Sundah, Nicholas R. Y. Ho, Wanxiang Shen, Yun Xu, Auginia Natalia, Zhonglang Yu, J. Seet, Ching Wan Chan, T. P. Loh, Brian Y Lim, Huilin Shao
          </td>
          <td>2024-06-19</td>
          <td>Nature biomedical engineering</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8868db406f4d297083b3336bdacf7e70f944e3" target='_blank'>
              Characterize molecular signatures and establish a prognostic signature of gastric cancer by integrating single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Zhiwei Wang, Zhiyan Weng, Luping Lin, Xianyi Wu, Wenju Liu, Yong Zhuang, J. Jian, Changhua Zhuo
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Recent research indicates a positive correlation between DEP structural domain‐containing 1B (DEPDC1B) and the cell cycle in various tumors. However, the role of DEPDC1B in the infiltration of the tumor immune microenvironment (TIME) remains unexplored. Methods We analyzed the differential expression and prognostic significance of DEPDC1B in colon adenocarcinoma (COAD) using the R package “limma” and the Gene Expression Profiling Interactive Analysis (GEPIA) website. Gene set enrichment analysis (GSEA) was employed to investigate the functions and interactions of DEPDC1B expression in COAD. Cell Counting Kit‐8 (CCK‐8) assays and colony formation assays were utilized to assess the proliferative function of DEPDC1B. Correlations between DEPDC1B expression and tumor‐infiltrating immune cells, immune checkpoints, tumor mutational burden (TMB), and microsatellite instability (MSI) status were examined using Spearman correlation analysis and CIBERSORT. Results DEPDC1B was highly expressed in COAD. Elevated DEPDC1B expression was associated with lower epithelial‐to‐mesenchymal transition (EMT) and TNM stages, leading to a favorable prognosis. DEPDC1B mRNA was prominently expressed in COAD cell lines. CCK‐8 and colony formation assays demonstrated that DEPDC1B inhibited the proliferation of COAD cells. Analysis using the CIBERSORT database and Spearman correlation revealed that DEPDC1B correlated with four types of tumor‐infiltrating immune cells. Furthermore, high DEPDC1B expression was linked to the expression of PD‐L1, CTLA4, SIGLEC15, PD‐L2, TMB, and MSI‐H. High DEPDC1B expression also indicated responsiveness to anti‐PD‐L1 immunotherapy. Conclusions DEPDC1B inhibits the proliferation of COAD cells and positively regulates the cell cycle, showing a positive correlation with CCNB1 and PBK expression. DEPDC1B expression in COAD is associated with tumor‐infiltrating immune cells, immune checkpoints, TMB, and MSI‐H in the tumor immune microenvironment. This suggests that DEPDC1B may serve as a novel prognostic marker and a potential target for immunotherapy in COAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/293cb0f525ffa84f15dbabefe7162aab0692a337" target='_blank'>
              DEPDC1B: A novel tumor suppressor gene associated with immune infiltration in colon adenocarcinoma
              </a>
            </td>
          <td>
            Dandan Zhu, Huolun Feng, Zhixiong Zhang, Jiaqi Li, Yong Li, Tieying Hou
          </td>
          <td>2024-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In transcription-coupled repair, stalled RNA polymerase II (Pol II) is recognized by CSB and CRL4CSA, which co-operate with UVSSSA and ELOF1 to recruit TFIIH for nucleotide excision repair (TC-NER). To explore the mechanism of TC-NER, we recapitulated this reaction in vitro. When a plasmid containing a site-specific lesion is transcribed in frog egg extract, error-free repair is observed that depends on CSB, CRL4CSA, UVSSA, and ELOF1. Repair also depends on STK19, a factor previously implicated in transcription recovery after UV exposure. A 1.9 Å cryo-electron microscopy structure shows that STK19 joins the TC-NER complex by binding CSA and the RPB1 subunit of Pol II. Furthermore, AlphaFold predicts that STK19 interacts with the XPD subunit of TFIIH, and disrupting this interface impairs cell-free repair. Molecular modeling suggests that STK19 positions TFIIH ahead of Pol II for lesion verification. In summary, our analysis of cell-free TC-NER suggests that STK19 couples RNA polymerase II stalling to downstream repair events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297011593f4e81e15ed2ffb10b29598b2dac1064" target='_blank'>
              STK19 positions TFIIH for cell-free transcription-coupled DNA repair
              </a>
            </td>
          <td>
            T. Mevissen, Maximilian Kümmecke, Ernst W. Schmid, L. Farnung, Johannes C. Walter
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Maintenance of DNA integrity is essential to all forms of life. DNA damage generated by reaction with genotoxic chemicals results in deleterious mutations, genome instability, and cell death. Pathogenic bacteria encounter several genotoxic agents during infection. In keeping with this, the loss of DNA repair networks results in virulence attenuation in several bacterial species. Interstrand DNA crosslinks (ICLs) are a type of DNA lesion formed by covalent linkage of opposing DNA strands and are particularly toxic as they interfere with replication and transcription. Bacteria have evolved specialized DNA glycosylases that unhook ICLs, thereby initiating their repair. In this study, we describe AlkX, a DNA glycosylase encoded by the multidrug resistant pathogen Acinetobacter baumannii. AlkX exhibits ICL unhooking activity similar to that of its Escherichia coli homolog YcaQ. Interrogation of the in vivo role of AlkX revealed that its loss sensitizes cells to DNA crosslinking and impairs A. baumannii colonization of the lungs and dissemination to distal tissues during pneumonia. These results suggest that AlkX participates in A. baumannii pathogenesis and protects the bacterium from stress conditions encountered in vivo. Consistent with this, we found that acidic pH, an environment encountered during host colonization, results in A. baumannii DNA damage and that alkX is induced by, and contributes to, defense against acidic conditions. Collectively, these studies reveal functions for a recently described class of proteins encoded in a broad range of pathogenic bacterial species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e8859ed92eefb483b7d273e5025bd69777d468" target='_blank'>
              An interstrand DNA crosslink glycosylase aids Acinetobacter baumannii pathogenesis.
              </a>
            </td>
          <td>
            D. E. Kunkle, Yujuan Cai, B. Eichman, Eric P Skaar
          </td>
          <td>2024-06-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The repair of DNA double strand breaks (DSBs) is crucial for the maintenance of genome stability and cell viability. DSB repair (DSBR) in cells is mediated through several mechanisms: homologous recombination (HR), non-homologous end joining (NHEJ), microhomology-mediated end joining (MMEJ), and single strand annealing (SSA). Cellular assays are essential to measure the proficiency and modulation of these pathways in response to various stimuli. Here, we present a suite of extrachromosomal reporter assays that each measure the reconstitution of a nanoluciferase reporter gene by one of the four major DSBR pathways in cells. Upon transient transfection into cells of interest, repair of pathway-specific reporter substrates can be measured in under 24 h by the detection of Nanoluciferase (NanoLuc) luminescence. These robust assays are quantitative, sensitive, titratable, and amenable to a high-throughput screening format. These properties provide broad applications in DNA repair research and drug discovery, complementing the currently available toolkit of cellular DSBR assays.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/270bb7c41b9ec2cbfe7fe21ae86f536aa1f2b28a" target='_blank'>
              Assessment of DNA Double Strand Break Repair Activity Using High-throughput and Quantitative Luminescence-based Reporter Assays.
              </a>
            </td>
          <td>
            Diego Grande, E. Rajendra, Bethany Mason, Alessandro Galbiati, S. Boulton, Graeme C. M. Smith, Helen M R Robinson
          </td>
          <td>2024-06-14</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16eda9843eb0037407162ede7ade3db9a52371a7" target='_blank'>
              SNM1A is crucial for efficient repair of complex DNA breaks in human cells
              </a>
            </td>
          <td>
            L. P. Swift, B. C. Lagerholm, Lucy R. Henderson, Malitha Ratnaweera, H. T. Baddock, B. Sengerova, Sook Y. Lee, A. Cruz-Migoni, Dominic Waithe, Christian Renz, Helle D. Ulrich, Joseph A. Newman, Christopher J. Schofield, P. McHugh
          </td>
          <td>2024-06-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="This article theoretically speculates that after replication of a pair of DNA they must be allocated to two daughter cells in a transverse division or erect division manner. Transverse division represents differentiation, while erect division represents transformation. The molecular structure determines that DNA has greater hydrophobicity than RNA. The hydrophobic modification of chromatin regulates gene activity and centromere similarity between new and old DNA. A difference in division between cancer cells and embryonic stem cells is discussed">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c6b78e19de2f6d2d4dda38ace7242257dc237cb" target='_blank'>
              On the Cancer
              </a>
            </td>
          <td>
            Xiao En Wang
          </td>
          <td>2024-08-31</td>
          <td>Journal of Cancer Research Reviews &amp; Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene conversions are broadly defined as the transfer of genetic material from a ‘donor’ to an ‘acceptor’ sequence and can happen both in meiosis and mitosis. They are a subset of non-crossover events and like crossover events, gene conversion can generate new combinations of alleles, erode linkage disequilibrium, and even counteract the mutation load by reverting germline mutations through GC-biased gene conversion. Estimating the rate of gene conversion and the distribution of gene conversion tract lengths remains challenging. Here, we present a new method for estimating tract length, rate and detection probability of non-crossover events directly in HiFi PacBio long read data. The method can be applied with data from a single individual, is unbiased even under low single nucleotide variant densities and does not necessitate any demographic or evolutionary assumptions. We apply the method to gene conversion events observed directly in Pacbio HiFI read data from a human sperm sample and find that human gene conversion tracts are shorter (mean of 50 base pairs) than estimates from yeast or Drosophila. We also estimate that typical human male gametes undergo on average 280 non-crossover events where approximately 7 are expected to become visible as gene conversions moving variants from one donor haplotype to an acceptor haplotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f463a46e1b5590cae1deadecdd92d1f8b1f8321" target='_blank'>
              Estimating gene conversion tract length and rate from PacBio HiFi data
              </a>
            </td>
          <td>
            A. P. Charmouh, Peter Porsborg Sørud, T. Bataillon, A. Hobolth, Lasse Thorup Hansen, Søren Besenbacher, S. B. Winge, K. Almstrup, M. Schierup
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf95f1dc0f1af0cb6e8bd77dab713b5fc609919" target='_blank'>
              CircRHBDD1 promotes immune escape via IGF2BP2/PD-L1 signaling and acts as a nanotherapeutic target in gastric cancer
              </a>
            </td>
          <td>
            Yanna Li, Zhixiong Wang, Peng Gao, Danping Cao, R. Dong, Menglin Zhu, Yao Fei, X. Zuo, Juan Cai
          </td>
          <td>2024-07-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract The use of k-mers to capture genetic variation in bacterial genome-wide association studies (bGWAS) has demonstrated its effectiveness in overcoming the plasticity of bacterial genomes by providing a comprehensive array of genetic variants in a genome set that is not confined to a single reference genome. However, little attempt has been made to interpret k-mers in the context of genome rearrangements, partly due to challenges in the exhaustive and high-throughput identification of genome structure and individual rearrangement events. Here, we present GWarrange, a pre- and post-bGWAS processing methodology that leverages the unique properties of k-mers to facilitate bGWAS for genome rearrangements. Repeat sequences are common instigators of genome rearrangements through intragenomic homologous recombination, and they are commonly found at rearrangement boundaries. Using whole-genome sequences, repeat sequences are replaced by short placeholder sequences, allowing the regions flanking repeats to be incorporated into relatively short k-mers. Then, locations of flanking regions in significant k-mers are mapped back to complete genome sequences to visualise genome rearrangements. Four case studies based on two bacterial species (Bordetella pertussis and Enterococcus faecium) and a simulated genome set are presented to demonstrate the ability to identify phenotype-associated rearrangements. GWarrange is available at https://github.com/DorothyTamYiLing/GWarrangehttps://github.com/DorothyTamYiLing/genome_rearrangement.git.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42162bc2b2f883e8a6834a9dc952dc10d3392ba" target='_blank'>
              GWarrange: a pre- and post- genome-wide association studies pipeline for detecting phenotype-associated genome rearrangement events
              </a>
            </td>
          <td>
            Yi Ling Tam, Sarah Cameron, Andrew Preston, Lauren A. Cowley
          </td>
          <td>2024-07-01</td>
          <td>Microbial Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 What role do active enhancers play in determining the fate of early embryonic cells? Are there differences in active enhancers between mouse and human blastocysts?



 Tissue-specific active enhancers in inner cell mass (ICM) and trophoblast (TE) are essential for blastocyst formation, regulating gene expression through interactive loops with promoters.



 At the two-cell stage of nuclear transfer embryos, the absence of interaction between Zscan4d promoter and distal active enhancer leads to reduced Zscan4d expression, resulting in lower developmental rates. Overexpressing the demethylase Kdm4d alleviates reprogramming-resistant H3K9me3 modification, restoring the structural loop between the Zscan4d promoter and active enhancer and enhancing developmental efficiency. These findings strongly suggest a close association between the formation of regulatory chromatin interactions in early embryonic development and histone modifications. Indicating active enhancers marked with H3K27ac may regulate differentiation gene expression through distal interactions in blastocyst.



 Analyze variations in histone H3K27ac modification, chromatin accessibility, higher-order structures, and transcriptomes during mouse embryo differentiation from 8-cell stage to ICM and TE formation. Identify tissue-specific active enhancers in ICM and TE, validate their gene expression regulation, explore their role in early differentiation-associated genes in mice. Compare and clarify similarities and differences in enhancer regulation mechanisms during early embryo development in humans and mice.



 The 8-cell, morula and blastocyst stage embryos were collected from the oviducts and uterus of super-ovulated female ICR mice mated with male mice. The clinical human blastocysts were collected and separated the ICM and TE. Low-input multi-omics sequencing, including Cut-Tag, ATAC-seq, Hi-C, and RNA-seq, was performed on the embryonic samples. After analyzing and identifying active enhancers, tissue-specific ones were selected and validated for their impact on gene expression via CRISPR-Cas9 knockout in embryos.



 Inhibiting ep300 expression in mouse embryos through siRNA interference reduces blastocyst formation from 90% to approximately 40%. Introducing the small molecule inhibitor A485 at the 8-cell stage, blocking the active center of CBP/p300 enzymes, completely arrests embryos at the morula stage, preventing their development into blastocysts. Emphasizing the crucial role of correctly establishing active enhancers marked by H3K27ac modification in embryonic differentiation.
 Multi-omics data demonstrates high repeatability and quality. PCA clustering analysis reveals a clear differentiation path in the distribution of H3K27ac-modified histones, chromatin accessibility sites, and the transcriptome from the 8-cell stage to the blastocyst. Differences in H3K27ac histone modification distribution between the ICM and TE indicate the presence of tissue-specific active enhancers in blastocyst.
 The disparities in H3K27ac modification distribution between blastocyst ICM and TE primarily occur in distal gene regions rather than proximal ones. Moreover, these distal enhancers exhibit a positive correlation with the expression of lineage-specific genes, suggesting a probable regulatory role of distal enhancers in gene modulation during embryonic differentiation.
 Tissue-specific enhancer-promoter (EP) interaction loops emerge during embryonic differentiation, allowing direct interaction between tissue-specific active enhancers and promoters, thereby activating the expression of tissue-specific genes.



 While DNA sequences with H3K27ac modification are currently considered markers for active enhancers, H3K4me modification is another histone marker for enhancers. However, enhancers associated with H3K4me modification may not necessarily be in an activated state. Future research can simultaneously incorporate appropriate detection methods for H3K4me modification.



 After elucidating the mechanisms through which active enhancers regulate mouse differentiation, comparing the differences in active enhancers during the first lineage differentiation process between mice and humans can help identify factors that enhance human blastocyst quality. This can potentially improve clinical outcomes in assisted reproduction.



 not applicable
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06a46c47cbfd5b64e709929a1eba256f72f28a5e" target='_blank'>
              P-548 An investigation into the mechanisms of active enhancers regulating the initial lineage differentiation in mammals
              </a>
            </td>
          <td>
            M. Chen
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa9b85d031e2f46dc36f65fc9e3a327afe81e85" target='_blank'>
              DNA hypo-methylation and expression of GBP4 induces T cell exhaustion in pancreatic cancer
              </a>
            </td>
          <td>
            Yesiboli Tasiheng, Xuan Lin, Xu Wang, Xuan Zou, Yusheng Chen, Yu Yan, Mingjian Ma, Zhen-bo Dai, Xu Wang, Xianjun Yu, He Cheng, Chen Liu
          </td>
          <td>2024-08-07</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Formaldehyde (FA) is a recognized environmental and metabolic toxin implicated in cancer development and aging. Inherited mutations in the FA-detoxifying enzymes ADH5 and ALDH2 genes lead to FA overload in the severe multisystem AMeD syndrome. FA accumulation causes genome damage including DNA-protein-, inter- and intra-strand crosslinks and oxidative lesions. However, the influence of distinct DNA repair systems on organismal FA resistance remains elusive. We have here investigated the consequence of a range of DNA repair mutants in a model of endogenous FA overload generated by downregulating the orthologs of human ADH5 and ALDH2 in C. elegans. We have focused on the distinct components of nucleotide excision repair (NER) during developmental growth, reproduction and aging. Our results reveal three distinct modes of repair of FA-induced DNA damage: Transcription-coupled repair (TCR) operating NER-independently during developmental growth or through NER during adulthood, and, in concert with global-genome (GG-) NER, in the germline and early embryonic development. Additionally, we show that the Cockayne syndrome B (CSB) factor is involved in the resolution of FA-induced DNA-protein crosslinks, and that the antioxidant and FA quencher N-acetyl-l-cysteine (NAC) reverses the sensitivity of detoxification and DNA repair defects during development, suggesting a therapeutic intervention to revert FA-pathogenic consequences.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d6ccccfe7604bc347a8ca565bb85bce6101f465" target='_blank'>
              Distinct DNA repair mechanisms prevent formaldehyde toxicity during development, reproduction and aging.
              </a>
            </td>
          <td>
            Matthias Rieckher, Christian Gallrein, Natividad Alquezar-Artieda, Nour Bourached-Silva, P. L. Vaddavalli, Devin Mares, Maria Backhaus, Timon Blindauer, Ksenia Greger, Eva Wiesner, Lucas B. Pontel, B. Schumacher
          </td>
          <td>2024-06-19</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="
 B-cell lymphoma is the most common hematological malignancy that accounts for the sixth highest mortality among all cancers. Despite recent improvements on combined chemotherapy and immunotherapy, treatments remain challenging due to complex genetic heterogeneity and molecular pathogenesis. To explore novel mechanisms in B-cell lymphomagenesis, we performed a genome-wide CRISPR knockout screen in the murine interleukin-3 (IL-3)-dependent Ba/F3 pro-B cell line and identified Zbtb24 as a putative target whose silencing leads to elevated proliferative signaling and reduced apoptosis, conferring IL-3-independent growth in Ba/F3 cells. Silencing of ZBTB24 also accelerated growth in multiple human B-cell lymphoma cell lines. ZBTB24 is a member of the zinc-finger and BTB domain containing transcription factor that is highly expressed in spleen and bone marrow. While it has been shown that ZBTB24 may play an important role in B-cell development and activation, its involvement in B-cell lymphoma has not been described. Analysis of publicly available datasets found that ZBTB24 copy number deletion occurs in various B-cell lymphomas, which frequently co-occurs with deletion of the tumor suppressor CDKN2A. Hence, we generated B-cell specific Zbtb24 single knockout and Zbtb24/Cdkn2a double knockout mice to explore the role of Zbtb24 in vivo. While Zbtb24 and Cdkn2a single knockout mice remained tumor-free or showed low penetrance of tumorigenesis, the double knockout mice developed lymphadenopathy and splenomegaly accompanied with accelerated mortality. Flow cytometry analysis of bone marrow, spleen, and lymph nodes revealed malignant B-cells. Subsequent transplantation of the splenic cells induced identical malignant B-cell lymphomagenesis and immunophenotype in athymic nude mice, further confirming its malignant identity. These results indicate that ZBTB24 is a novel tumor suppressor in B-cell malignancy. Ongoing studies are being conducted to understand the mechanistic basis for ZBTB24 tumor suppressive function that may involve transcriptional regulation of its target gene expression. This project is supported by the NIH R01CA232246, R01CA129536, and the John Colaizzi Endowment Fund.
 Citation Format: Rongrong Li, Namratha Sheshadri, Jianliang Shen, Junrong Yan, Jaeyong Jung, Muhammad Usama Tariq, Y Lynn Wang, Ping Xie, Wei-Xing Zong. ZBTB24 is a novel tumor suppressor and cooperates with CDKN2A to suppress B-cell lymphomagenesis [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PO-020.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/15e57cab4c9d35d608b79da3c73064617cfb9e73" target='_blank'>
              Abstract PO-020: ZBTB24 is a novel tumor suppressor and cooperates with CDKN2A to suppress B-cell lymphomagenesis
              </a>
            </td>
          <td>
            Rongrong Li, Namratha Sheshadri, Jianliang Shen, Junrong Yan, Jaeyong Jung, Muhammad Usama Tariq, Y. L. Wang, Ping Xie, Wei-Xing Zong
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genome-level clonal decomposition of a single specimen has been widely studied; however, it is mostly limited to cancer research. In this study, we developed a new algorithm CLEMENT, which conducts accurate decomposition and reconstruction of multiple subclones in genome sequencing of non-tumor (normal) samples. CLEMENT employs the Expectation-Maximization (EM) algorithm with optimization strategies specific to non-tumor subclones, including false variant call identification, non-disparate clone fuzzy clustering, and clonal allele fraction confinement. In the simulation and in vitro cell line mixture data, CLEMENT outperformed current cancer decomposition algorithms in estimating the number of clones (root-mean-square-error = 0.58-0.78 versus 1.43-3.34) and in the variant-clone membership agreement (∼85.5% versus 70.1-76.7%). Additional testing on human multi-clonal normal tissue sequencing confirmed the accurate identification of subclones that originated from different cell types. Clone-level analysis, including mutational burden and signatures, provided a new understanding of normal-tissue composition. We expect that CLEMENT will serve as a crucial tool in the currently emerging field of non-tumor genome analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b10b8eaeda7c6b37ca2e0e9728849f940c0c059" target='_blank'>
              CLEMENT: genomic decomposition and reconstruction of non-tumor subclones.
              </a>
            </td>
          <td>
            Young-Soo Chung, Seungseok Kang, Jisu Kim, Sangbo Lee, Sangwoo Kim
          </td>
          <td>2024-06-26</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Strong epigenetic pan-cancer biomarkers are required to meet several current, urgent clinical needs and to further improve the present chemotherapeutic standard. We have concentrated on the investigation of epigenetic alteration of the hTERT gene, which is frequently epigenetically dysregulated in a number of cancers in specific developmental stages. Distinct DNA methylation profiles were identified in our data on early urothelial cancer. An efficient EpihTERT assay could be developed utilizing suitable combinations with sequence-dependent thermodynamic parameters to distinguish between differentially methylated states. We infer from this data set, the epigenetic context, and the related literature that a CpG-rich, 2800 bp region, a prominent CpG island, surrounding the transcription start of the hTERT gene is the crucial epigenetic zone for the development of a potent biomarker. In order to accurately describe this region, we have named it “Acheron” (Ἀχέρων). In Greek mythology, this is the river of woe and misery and the path to the underworld. Exploitation of the DNA methylation profiles focused on this region, e.g., idiolocal normalized Methylation Specific PCR (IDLN-MSP), opens up a wide range of new possibilities for diagnosis, determination of prognosis, follow-up, and detection of residual disease. It may also have broad implications for the choice of chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3ee3e7f8ca8ba62fbce3d513f74a833baab6859" target='_blank'>
              hTERT Epigenetics Provides New Perspectives for Diagnosis and Evidence-Based Guidance of Chemotherapy in Cancer
              </a>
            </td>
          <td>
            S. Santourlidis, Marcos J. Araúzo-Bravo, Robert T. Brodell, Mohamed Hassan, M. Bendhack
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The genome of a cell is continuously battered by a plethora of exogenous and endogenous processes that can lead to damaged DNA. Repair mechanisms correct this damage most of the time, but failure to do so leaves mutations. Mutations do not occur in random manner, but rather typically follow a more or less specific pattern due to known or imputed mutational processes. Mutational signature analysis is the process by which the predominant mutational process can be inferred for a cancer and can be used in several contexts to study both the genesis of cancer and its response to therapy. Recent pan-cancer genomic efforts such as "The Cancer Genome Atlas" have identified numerous mutational signatures that can be categorized into single base substitutions, doublet base substitutions, or small insertions/deletions. Understanding these mutational signatures as they occur in non-small lung cancer could improve efforts at prevention, predict treatment response to personalized treatments, and guide the development of therapies targeting tumor evolution. For non-small cell lung cancer, several mutational signatures have been identified that correlate with exposures such as tobacco smoking and radon and can also reflect endogenous processes such as aging, APOBEC activity, and loss of mismatch repair. Herein, we provide an overview of the current knowledge of mutational signatures in non-small lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37777d22dfc5a2bd99dc42f300aad4d0a01e922f" target='_blank'>
              Topography of mutational signatures in non-small cell lung cancer: emerging concepts, clinical applications, and limitations.
              </a>
            </td>
          <td>
            Ritujith Jayakrishnan, David J. Kwiatkowski, Michal G Rose, Amin H. Nassar
          </td>
          <td>2024-06-22</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bda6b2e4f3ed255839eb1bdc42fe72c67b937b04" target='_blank'>
              Identifying subtypes and developing prognostic models based on N6-methyladenosine and immune microenvironment related genes in breast cancer
              </a>
            </td>
          <td>
            Lizhao Wang, Jianpeng Li, Nan Mei, Heyan Chen, Ligang Niu, Jianjun He, Ru Wang
          </td>
          <td>2024-07-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Asthenoteratozoospermia, a prevalent cause of male infertility, lacks a well-defined etiology. DNAH12 is a special dynein featured by the absence of a microtubule-binding domain, however, its functions in spermatogenesis remain largely unknown. Through comprehensive genetic analyses involving whole-exome sequencing and subsequent Sanger sequencing on infertile patients and fertile controls from six distinct families, we unveiled six biallelic mutations in DNAH12 that co-segregate recessively with male infertility in the studied families. Transmission electron microscopy (TEM) revealed pronounced axonemal abnormalities, including inner dynein arms (IDAs) impairment and central pair (CP) loss in sperm flagella of the patients. Mouse models (Dnah12−/− and Dnah12mut/mut) were generated and recapitulated the reproductive defects in the patients. Noteworthy, DNAH12 deficiency did not show effects on cilium organization and function. Mechanistically, DNAH12 was confirmed to interact with two other IDA components DNALI1 and DNAH1, while disruption of DNAH12 leads to failed recruitment of DNALI1 and DNAH1 to IDAs and compromised sperm development. Furthermore, DNAH12 also interacts with radial spoke head proteins RSPH1, RSPH9, and DNAJB13 to regulate CP stability. Moreover, the infertility of Dnah12−/− mice could be overcome by intracytoplasmic sperm injection (ICSI) treatment. Collectively, DNAH12 plays a crucial role in the proper organization of axoneme in sperm flagella, but not cilia, by recruiting DNAH1 and DNALI1 in both humans and mice. These findings expand our comprehension of dynein component assembly in flagella and cilia and provide a valuable marker for genetic counseling and diagnosis of asthenoteratozoospermia in clinical practice.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea45abcae68145bc277d76b228d4c12d293dba9" target='_blank'>
              Deficiency in a special dynein DNAH12 causes male infertility by impairing DNAH1 and DNALI1 recruitment in humans and mice
              </a>
            </td>
          <td>
            Menglei Yang, H. M. Hussain, Manan Khan, Zubair Muhammad, Jianteng Zhou, Ao Ma, Xiongheng Huang, Jingwei Ye, Min Chen, Aoran Zhi, Tao Liu, R. Khan, Asim Ali, Wasim Shah, A. Zeb, Nisar Ahmad, Huan Zhang, Bo Xu, Hui Ma, Qinghua Shi, Baolu Shi
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSC) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosomes 20q or 1q, both known for conferring in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings show higher loads of both CNVs and SNVs at later passages which are due to the cumulative acquisition of mutations over a longer time in culture and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs follow the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid cancer-associated mutations. This suggests that the recurrent chromosomal abnormalities are the potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f552b60bd87e180c4d6a5dd522428893e4b81b" target='_blank'>
              De novo cancer mutations frequently associate with recurrent chromosomal abnormalities during long-term human pluripotent stem cell culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huygebaert, M. Regin, Chi Mai Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Retinoblastoma, a pediatric ocular malignancy, presents significant challenges in comprehending its molecular underpinnings and targeted therapeutic approaches. The dysregulated activity of histone deacetylases (HDACs) has been associated with retinoblastoma pathogenesis, influencing critical cellular processes like cell cycle regulation or retinal ganglion cell apoptosis. Through their deacetylase activity, HDACs exert control over key tumor suppressors and oncogenes, influencing the delicate equilibrium between proliferation and cell death. Furthermore, the interplay between HDACs and the retinoblastoma protein pathway, a pivotal aspect of retinoblastoma etiology, reveals a complex network of interactions influencing the tumor microenvironment. The examination of HDAC inhibitors, encompassing both established and novel compounds, offers insights into potential approaches to restore acetylation balance and impede retinoblastoma progression. Moreover, the identification of specific HDAC isoforms exhibiting varying expression in retinoblastoma provides avenues for personalized therapeutic strategies, allowing for interventions tailored to individual patient profiles. This review focuses on the intricate interrelationship between HDACs and retinoblastoma, shedding light on epigenetic mechanisms that control tumor development and progression. The exploration of HDAC-targeted therapies underscores the potential for innovative treatment modalities in the pursuit of more efficacious and personalized management strategies for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5855b4c3961d5048496222d3549d192782680eaf" target='_blank'>
              Histone Deacetylases in Retinoblastoma
              </a>
            </td>
          <td>
            Malwina Lisek, Julia Tomczak, Julia Swiatek, Aleksandra Kaluza, T. Boczek
          </td>
          <td>2024-06-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Phylogenetic relationship of cells within tumours can help us to understand how cancer develops in space and time, iden-tify driver mutations and other evolutionary events that enable can-cer growth and spread. Numerous studies have reconstructed phylo-genies from single-cell DNA-seq data. Here we are looking into the problem of phylogenetic analysis of spatially resolved near single-cell RNA-seq data, which is a cost-efficient alternative (or complemen-tary) data source that integrates multiple sources of evolutionary information including point mutations, copy-number changes, and epimutations. Recent attempts to use such data, although promis-ing, raised many methodological challenges. Here, we explored data-preprocessing and modelling approaches for evolutionary analyses of Visium spatial transcriptomics data. We conclude that using only highly variable genes and accounting for heterogeneous RNA capture across tissue-covered spots improves the reconstructed topological relationships and influences estimated branch lengths.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d567ed01155056fa70bf3fe42ae8fbf32987b2be" target='_blank'>
              Filtering for highly variable genes and high quality spots improves phylogenetic analysis of cancer spatial transcriptomics Visium data
              </a>
            </td>
          <td>
            Alexandra “Sasha” Gavryushkina, H. R. Pinkney, Sarah D. Diermeier, A. Gavryushkin
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Recombinant adeno-associated virus (rAAV) is a widely used viral vector for gene therapy. Despite its clinical efficacy, the manufacturing of rAAV faces challenges in productivity and quality, leading to limited availability. To address the growing demand, next-generation process development should be informed by a mechanistic understanding of the cellular response to rAAV. In this study, we performed transcriptomic analysis of 5 cell lines with variable capacities for rAAV production. Using an intersectional approach, we assessed the transcriptional response to rAAV production and compared transcriptional profiles between high and baseline producers to identify possible targets for enhancing production. Modulation of cell cycle and nucleosome components suggested a reduction of proliferative capacity and a shift toward DNA replication to support rAAV production. During rAAV production, we observed upregulation of several core functions including transcription, stress response, and Golgi and endoplasmic reticulum organization. Conversely, inhibitors of DNA-binding proteins and mitochondrial components were consistently downregulated during rAAV production. We next performed a drug connectivity analysis of these results and identified 5 classes of drugs predicted to enhance rAAV production. Validation studies confirmed the efficacy of HDAC and microtubule inhibitors. Our data uncover novel and previously identified pathways that may enhance rAAV productivity, potentially enabling a path to engineer improved processes and cell lines for higher yields and better quality rAAV production. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9b270da94a458df57bb4725ecb5dd57f8a71f23d" target='_blank'>
              Transcriptomics-informed pharmacology identifies epigenetic and cell cycle regulators as enhancers of AAV production
              </a>
            </td>
          <td>
            Joshua Tworig, Francis Grafton, Markus Hörer, Christopher A. Reid, M. Mandegar
          </td>
          <td>2024-06-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Understanding the dark genome is a priority task following the complete sequencing of the human genome. Short open reading frames (sORFs) are a group of largely unexplored elements of the dark genome with the potential for being translated into microproteins. The definitive number of coding and regulatory sORFs is not known, however they could account for up to 1-2% of the human genome. This corresponds to an order of magnitude in the range of canonical coding genes. For a few sORFs a clinical relevance has already been demonstrated, but for the majority of potential sORFs the biological function remains unclear. A major limitation in predicting their disease relevance using large-scale genomic data is the fact that no population-level constraint metrics for genetic variants in sORFs are yet available. To overcome this, we used the recently released gno-mAD 4.0 dataset and analysed the constraint of a consensus set of sORFs and their genomic neighbours. We demonstrate that sORFs are mostly embedded into a moderately constraint genomic context, but within the gencode dataset we identified a subset of highly constrained sORFs comparable to highly constrained canonical genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b69986e14a46e8cde5242362f9f2d287ba4c2f6a" target='_blank'>
              Mutational Constraint Analysis Workflow for Overlapping Short Open Reading Frames and Genomic Neighbours
              </a>
            </td>
          <td>
            Martin Danner, Matthias Begemann, Florian Kraft, Miriam Elbracht, Ingo Kurth, Jeremias Krause
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Simple Summary Individualized medicine means understanding how each tumor is different from normal cells and how each tumor is different from other tumors, including profiling mutations, non-mutational epigenetic changes, and differences in gene expression. This allows the discovery of key processes each tumor absolutely depends on for survival and growth, which are intrinsic weaknesses. This profiling means selecting treatments to specifically target each tumor’s survival-dependent pathways, killing them. Tip60 is a master controller of processes that maintain genome stability and signaling regulating gene expression. While disrupted in many cancers, Tip60 is essential for cell survival, and inhibiting Tip60 kills tumors. While we understand some key aspects of the molecular roles Tip60 plays, much more remains to be discovered. A more complete understanding of the diverse roles and functions of Tip60 in cancer, and how targeting Tip60 kills cancer cells, will lead to better treatments for patients and increased survival. Abstract Precision (individualized) medicine relies on the molecular profiling of tumors’ dysregulated characteristics (genomic, epigenetic, transcriptomic) to identify the reliance on key pathways (including genome stability and epigenetic gene regulation) for viability or growth, and then utilises targeted therapeutics to disrupt these survival-dependent pathways. Non-mutational epigenetic changes alter cells’ transcriptional profile and are a key feature found in many tumors. In contrast to genetic mutations, epigenetic changes are reversable, and restoring a normal epigenetic profile can inhibit tumor growth and progression. Lysine acetyltransferases (KATs or HATs) protect genome stability and integrity, and Tip60 is an essential acetyltransferase due to its roles as an epigenetic and transcriptional regulator, and as master regulator of the DNA double-strand break response. Tip60 is commonly downregulated and mislocalized in many cancers, and the roles that mislocalized Tip60 plays in cancer are not well understood. Here we categorize and discuss Tip60-regulated genes, evaluate Tip60-interacting proteins based on cellular localization, and explore the therapeutic potential of Tip60-targeting compounds as epigenetic inhibitors. Understanding the multiple roles Tip60 plays in tumorigenesis will improve our understanding of tumor progression and will inform therapeutic options, including informing potential combinatorial regimes with current chemotherapeutics, leading to improvements in patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d0558cedd75f278d6feed6575b276aad74d694cf" target='_blank'>
              Evaluating the Cellular Roles of the Lysine Acetyltransferase Tip60 in Cancer: A Multi-Action Molecular Target for Precision Oncology
              </a>
            </td>
          <td>
            Nazanin Zohourian, Erin Coll, Muiread Dever, Anna Sheahan, Petra Burns-Lane, James A. L. Brown
          </td>
          <td>2024-07-27</td>
          <td>Cancers</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Peripheral T-cell lymphomas (PTCL) are heterogeneous and highly aggressive hematologic malignancies with dismal outcomes, highlighting the need for novel targeted therapies. PTCLs are characterized by complex tumor-microenvironment (TME) ecosystems with low tumor cell content, admixed with rich immune infiltrates and stromal elements correlated with clinical outcomes. Direct targeting of the TME is an attractive strategy for PTCLs; however, the specific mechanisms that regulate the interdependence between lymphoma and TME remain poorly understood, hampering the development of TME-directed therapies. Our previous work based on driver genetic alterations in PTCL patients and genetic mouse models identified an important role of these driver mutations in remodeling the TME (Cortes et al., 2022). To address the specific role and mechanisms of tumor cell-TME interactions in the pathogenesis of PTCL, we developed a novel experimental platform and cutting-edge computational methodologies using single-nucleus RNAseq analysis to comprehensively profile lymphoma cells and their microenvironment from 27 PTCL patient samples of two major nodal pathologic subtypes, including 18 Nodal T follicular helper cell lymphomas (TFHcL) and 9 PTCL, not otherwise specified (PTCL, NOS). Our approach allowed the identification and in-depth characterization of tumor cells, lymphoid and myeloid TME, and vascular endothelium and non-endothelial stromal cells, which were challenging to capture with conventional single-cell RNAseq. The malignant T-cells in PTCL, NOS were grouped into patient-specific clusters demonstrating significant inter-tumor heterogeneity associated with diverse and frequently complex chromosomal abnormalities identified by copy number variation (CNV) analysis. In contrast, malignant TFHcL cells show a low degree to no apparent chromosomal abnormalities and divide into two major transcriptional subclusters, with tumor cells from the same patient falling into one of the two subclusters. Additionally, genomic profiles of TFHcL tumors revealed that the inter-tumor transcriptional heterogeneity of TFHcL tumor cells was strongly associated with the driver RHOA G17V mutation status. To profile oncogenic pathways critical for PTCL growth, we compared PTCL tumor cells to normal CD4 T-cells from reactive lymphoid tissue. PTCL, NOS and TFHcL tumor cells were equally enriched in the TCR signaling pathway, mitosis, and cell cycle related genes. However, exploitation of KRAS and cytokine-driven inflammatory signals was prevalent in TFHcL but not PTCL, NOS tumor cells. Analysis of the immune TME identified an increase in T regulatory cells, CD8 T-cells, immunoblasts, and macrophages in TFHcL. Furthermore, while dysregulated B-cells with heightened cytokine-driven inflammatory signals were unique to TFHcL, inflammatory macrophages and classic dendritic cells (cDC) were prominent in the TME of both subtypes. Our results uncover shared and subtype-specific pathogenic features in PTCL and pave the way for designing novel precision therapies for PTCL patients.
 Citation Format: Wen-Hsuan Wendy Lin, Anqi W Wang, Bobby B Shih, Jean-Baptiste F Reynier, Laura Quevedo Palacio, Craig Soderquist, Ryan D Najac, Cindy Ma, Govind Bhagat, Adolfo A Ferrando, Raul Rabadan, Teresa Palomero. Single-cell transcriptomics reveals shared and subtype-specific vulnerabilities of the tumor-microenvironment ecosystems in peripheral T-cell lymphomas [abstract]. In: Proceedings of the Fourth AACR International Meeting on Advances in Malignant Lymphoma: Maximizing the Basic-Translational Interface for Clinical Application; 2024 Jun 19-22; Philadelphia, PA. Philadelphia (PA): AACR; Blood Cancer Discov 2024;5(3_Suppl):Abstract nr PR05.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d17bf46cac83bb1af31141fb58102ffe6ac218af" target='_blank'>
              Abstract PR05: Single-cell transcriptomics reveals shared and subtype-specific vulnerabilities of the tumor-microenvironment ecosystems in peripheral T-cell lymphomas
              </a>
            </td>
          <td>
            Wen-Hsuan Wendy Lin, Anqi W Wang, Bobby B. Shih, Jean-Baptiste F Reynier, Laura Quevedo Palacio, Craig R. Soderquist, Ryan Najac, Cindy Ma, Govind Bhagat, Adolfo A Ferrando, Raul Rabadan, Teresa Palomero
          </td>
          <td>2024-06-19</td>
          <td>Blood Cancer Discovery</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The p53 genome is a gene that has truly taken the scientific world by storm. Once referred to as an oncogene, p53 has proved itself to be more complex and dubious time and time again. This paper will talk about the beginning doubts and queries revolving around p53 and the discoveries that led to it being referred to as a tumour suppressor gene, rather than being mislabeled as an oncogene. It will touch upon the importance that p53 brings to our daily lives such as repairing the DNA, maintaining the cell cycle and starting apoptosis, as well as highlight the differences between the gene’s two types: wild-type p53 and mutant p53. Furthermore, it will talk about how a slight mutation in p53, can cause it to form many protein isoforms, a leading contributing factor in cancers. It will also mention the effects post-translational modification will have on the two types of p53. In addition to this, it will delve into the negative and potentially harmful effects that p53 could have on our bodies. Finally, it will also talk about p53-mediated therapy and targeting mutant p53 with autophagy, which are 2 processes that have shown successful outcomes in reducing the complications caused by mutant p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dc70ab96424bdf601f1a4ba7f9b8f2ff8a5b550" target='_blank'>
              P53: Our Saviour and Our Destroyer
              </a>
            </td>
          <td>
            Narayani Haldar
          </td>
          <td>2024-07-23</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1575162f33cdbe2adcebbaac6a20b693ec8530" target='_blank'>
              Single-cell EpiChem jointly measures drug-chromatin binding and multimodal epigenome.
              </a>
            </td>
          <td>
            Chao Dong, Xiaoxuan Meng, Tong Zhang, Zhifang Guo, Yaxi Liu, Peihuang Wu, Shiwei Chen, Fanqi Zhou, Yanni Ma, Haiqing Xiong, Shaokun Shu, Aibin He
          </td>
          <td>2024-07-18</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A. Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M. As Sobeai
          </td>
          <td>2024-08-05</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="
 Investigating a long-standing conceptual question in bacterial physiology, we examine why DnaA, the bacterial master replication initiator protein, exists in both ATP and ADP forms, despite only the ATP form being essential for initiation. We engineered the Δ4
 Escherichia coli
 strain, devoid of all known external elements facilitating the DnaA-ATP/ADP conversion and found that these cells display nearly wild-type behaviors under nonoverlapping replication cycles. However, during rapid growth with overlapping cycles, Δ4 cells exhibit initiation instability. This aligns with our model predictions, suggesting that the intrinsic ATPase activity of DnaA alone is sufficient for robust initiation control in
 E. coli
 and the DnaA-ATP/ADP conversion regulatory elements extend the robustness to multifork replication, indicating an evolutionary adaptation. Moreover, our experiments revealed constant DnaA concentrations during steady-state cell elongation in both wild-type and Δ4 cells. These insights not only advance our understanding of bacterial cell-cycle regulation and DnaA but also highlight a fundamental divergence from eukaryotic cell-cycle controls, emphasizing protein copy-number sensing in bacteria versus programmed protein concentration oscillations in eukaryotes.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2e026d8cc83866507c0c81f7cd88693227afacd" target='_blank'>
              Dispensability of extrinsic DnaA regulators in
 Escherichia coli
 cell-cycle control
              </a>
            </td>
          <td>
            T. Boesen, Godefroid Charbon, Haochen Fu, Cara Jensen, Michael Sandler, Suckjoon Jun, A. Løbner‐Olesen
          </td>
          <td>2024-08-06</td>
          <td>Proceedings of the National Academy of Sciences</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease.Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion.Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group.This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Non-membrane compartments or biomolecular condensates play an important role in the regulation of cellular processes including DNA repair. Here, an ability of XRCC1, a scaffold protein involved in DNA base excision repair (BER) and single-strand break repair, to form protein-rich microphases in the presence of DNA duplexes was discovered. We also showed that the gap-filling activity of BER-related DNA polymerase λ (Pol λ) is significantly increased by the presence of XRCC1. The stimulation of the Pol λ activity was observed only at micromolar XRCC1 concentrations, which were well above the nanomolar dissociation constant determined for the XRCC1–Pol λ complex and pointed to the presence of an auxiliary stimulatory factor in addition to protein–protein interactions. Indeed, according to dynamic light scattering measurements, the stimulation of the Pol λ activity by XRCC1 was coupled with microphase separation in a protein–DNA mixture. Fluorescence microscopy revealed colocalization of Pol λ, XRCC1, and gapped DNA within the microphases. Thus, stimulation of Pol λ activity is caused both by its interaction with XRCC1 and by specific conditions of microphase separation; this phenomenon is shown for the first time.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a6e72e42baaf9e9f3d9d413842843c132036b34" target='_blank'>
              DNA Repair Protein XRCC1 Stimulates Activity of DNA Polymerase λ under Conditions of Microphase Separation
              </a>
            </td>
          <td>
            N. Lebedeva, R. Anarbaev, E. A. Maltseva, M. Sukhanova, N. Rechkunova, O. I. Lavrik
          </td>
          <td>2024-06-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="ABSTRACT Natural killer (NK) cells play a crucial role in antitumoral and antiviral responses. Yet, cancer cells can alter themselves or the microenvironment through the secretion of cytokines or other factors, hindering NK cell activation and promoting a less cytotoxic phenotype. These resistance mechanisms, often referred to as the “hallmarks of cancer” are significantly influenced by the activation of oncogenes, impacting most, if not all, of the described hallmarks. Along with oncogenes, other types of genes, the tumor suppressor genes are frequently mutated or modified during cancer. Traditionally, these genes have been associated with uncontrollable tumor growth and apoptosis resistance. Recent evidence suggests oncogenic mutations extend beyond modulating cell death/proliferation programs, influencing cancer immunosurveillance. While T cells have been more studied, the results obtained highlight NK cells as emerging key protagonists for enhancing tumor cell elimination by modulating oncogenic activity. A few recent studies highlight the crucial role of oncogenic mutations in NK cell-mediated cancer recognition, impacting angiogenesis, stress ligands, and signaling balance within the tumor microenvironment. This review will critically examine recent discoveries correlating oncogenic mutations to NK cell-mediated cancer immunosurveillance, a relatively underexplored area, particularly in the era dominated by immune checkpoint inhibitors and CAR-T cells. Building on these insights, we will explore opportunities to improve NK cell-based immunotherapies, which are increasingly recognized as promising alternatives for treating low-antigenic tumors, offering significant advantages in terms of safety and manufacturing suitability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfefe1207e6a05f83b08c29611ae01cb2a8d9e94" target='_blank'>
              In-depth analysis of the interplay between oncogenic mutations and NK cell-mediated cancer surveillance in solid tumors
              </a>
            </td>
          <td>
            C. Pesini, Laura Artal, Jorge Paúl Bernal, Diego Sánchez Martinez, Julián Pardo, A. Ramírez-Labrada
          </td>
          <td>2024-07-18</td>
          <td>Oncoimmunology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="BACKGROUND
Understanding the impact of clonal hematopoiesis of indeterminate potential (CHIP) and mosaic chromosomal alterations (mCAs) on solid tumor risk and mortality can shed light on novel cancer pathways.


METHODS
The authors analyzed whole genome sequencing data from the Trans-Omics for Precision Medicine Women's Health Initiative study (n = 10,866). They investigated the presence of CHIP and mCA and their association with the development and mortality of breast, lung, and colorectal cancers.


RESULTS
CHIP was associated with higher risk of breast (hazard ratio [HR], 1.30; 95% confidence interval [CI], 1.03-1.64; p = .02) but not colorectal (p = .77) or lung cancer (p = .32). CHIP carriers who developed colorectal cancer also had a greater risk for advanced-stage (p = .01), but this was not seen in breast or lung cancer. CHIP was associated with increased colorectal cancer mortality both with (HR, 3.99; 95% CI, 2.41-6.62; p < .001) and without adjustment (HR, 2.50; 95% CI, 1.32-4.72; p = .004) for advanced-stage and a borderline higher breast cancer mortality (HR, 1.53; 95% CI, 0.98-2.41; p = .06). Conversely, mCA (cell fraction [CF] >3%) did not correlate with cancer risk. With higher CFs (mCA >5%), autosomal mCA was associated with increased breast cancer risk (HR, 1.39; 95% CI, 1.06-1.83; p = .01). There was no association of mCA (>3%) with breast, colorectal, or lung mortality except higher colon cancer mortality (HR, 2.19; 95% CI, 1.11-4.3; p = .02) with mCA >5%.


CONCLUSIONS
CHIP and mCA (CF >5%) were associated with higher breast cancer risk and colorectal cancer mortality individually. These data could inform on novel pathways that impact cancer risk and lead to better risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa8e5d699d9cb31c3abe6a09df01fb728c1626c" target='_blank'>
              Association of clonal hematopoiesis and mosaic chromosomal alterations with solid malignancy incidence and mortality.
              </a>
            </td>
          <td>
            Pinkal M. Desai, Ying Zhou, Justin Grenet, Samuel K Handelman, Cynthia Crispino, Laura N Tarbay, E. Whitsel, Gail Roboz, Ana Barac, Michael Honigberg, A. Bick, Garnet Anderson, Jean Wactawski-Wende, Yasminka A Jakubek Swartzlander, Jason Bacon, Justin Wong, Xiaolong Ma, P. Scheet, Zichan Li, P. Kasi, Ross Prentice, Paul L Auer, J. Manson, Alexander Reiner, Michael S. Simon
          </td>
          <td>2024-07-16</td>
          <td>Cancer</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Translesion synthesis (TLS) is a mechanism of DNA damage tolerance utilized by eukaryotic cells to replicate DNA across lesions that impede the high-fidelity replication machinery. In TLS, a series of specialized DNA polymerases are employed, which recognize specific DNA lesions, insert nucleotides across the damage, and extend the distorted primer-template. This allows cells to preserve genetic integrity at the cost of mutations. In humans, TLS enzymes include the Y-family, inserter polymerases, Polη, Polι, Polκ, Rev1, and the B-family extender polymerase Polζ, while in S. cerevisiae only Polη, Rev1, and Polζ are present. To bypass DNA lesions, TLS polymerases cooperate, assembling into a complex on the eukaryotic sliding clamp, PCNA, termed the TLS mutasome. The mutasome assembly is contingent on protein–protein interactions (PPIs) between the modular domains and subunits of TLS enzymes, and their interactions with PCNA and DNA. While the structural mechanisms of DNA lesion bypass by the TLS polymerases and PPIs of their individual modules are well understood, the mechanisms by which they cooperate in the context of TLS complexes have remained elusive. This review focuses on structural studies of TLS polymerases and describes the case of TLS holoenzyme assemblies in action emerging from recent high-resolution Cryo-EM studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10007da8b8f351240dd7da3607bdcec34eea8226" target='_blank'>
              Protein Assemblies in Translesion Synthesis
              </a>
            </td>
          <td>
            Gianluca A. Arianna, Dmitry M. Korzhnev
          </td>
          <td>2024-06-24</td>
          <td>Genes</td>
          <td>0</td>
          <td>2</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>