<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-09-30 06:32:17 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>25</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>52</td>
          <td>131</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>102</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, Jens Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>280</td>
          <td>131</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>243</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Extrachromosomal DNA (ecDNA) is a hallmark of aggressive cancer, contributing to both oncogene amplification and tumor heterogeneity. Here, we used Hi-C, super-resolution imaging, and long-read sequencing to explore the nuclear architecture of MYC-amplified ecDNA in colorectal cancer cells. Intriguingly, we observed frequent spatial proximity between ecDNA and 68 repetitive elements which we called ecDNA-interacting elements or EIEs. To characterize a potential regulatory role of EIEs, we focused on a fragment of the L1M4a1#LINE/L1 which we found to be co-amplified with MYC on ecDNA, gaining enhancer-associated chromatin marks in contrast to its normally silenced state. This EIE, in particular, existed as a naturally occurring structural variant upstream of MYC, gaining oncogenic potential in the transcriptionally permissive ecDNA environment. This EIE sequence is sufficient to enhance MYC expression and is required for cancer cell fitness. These findings suggest that silent repetitive genomic elements can be reactivated on ecDNA, leading to functional cooption and amplification. Repeat element activation on ecDNA represents a mechanism of accelerated evolution and tumor heterogeneity and may have diagnostic and therapeutic potential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4ade3bf8fe0fe3ba6189e2762e5b189878e923e" target='_blank'>
              Enhancer activation from transposable elements in extrachromosomal DNA
              </a>
            </td>
          <td>
            Katerina Kraft, Sedona E. Murphy, Matthew G Jones, Quanming Shi, Aarohi Bhargava-Shah, Christy Luong, King L. Hung, B. J. He, Rui Li, Seung K. Park, Natasha E Weiser, Jens Luebeck, V. Bafna, J. Boeke, Paul S. Mischel, A. Boettiger, Howard Y. Chang
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (ecDNA) have been found in most types of human cancers, and ecDNA incorporating viral genomes has recently been described, specifically in human papillomavirus (HPV)-mediated oropharyngeal cancer (OPC). However, the molecular mechanisms of human-viral hybrid ecDNA (hybrid ecDNA) for carcinogenesis remains elusive. We characterized the epigenetic status of hybrid ecDNA using HPVOPC cell lines and patient-derived tumor xenografts, identifying HPV oncogenes E6/E7 in hybrid ecDNA were flanked by novel somatic DNA enhancers and HPV L1 enhancers, with strong cis-interaction. Targeting of these enhancers by clustered regularly interspaced short palindromic repeats interference or hybrid ecDNA by bromodomain and extra-terminal inhibitor reduced E6/E7 expression, and significantly inhibited in vitro and/or in vivo growth only in ecDNA(+) models. HPV DNA in hybrid ecDNA structures are associated with novel somatic and HPV enhancers in hybrid ecDNA that drive HPV ongogene expression and carcinogenesis, and can be targeted with ecDNA disrupting therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9c00e1f8a07a4acac2f60dea45935a4972a1645" target='_blank'>
              Inhibition of novel human-HPV hybrid ecDNA enhancers reduces oncogene expression and tumor growth in oropharyngeal cancer
              </a>
            </td>
          <td>
            Joseph Califano, Takuya Nakagawa, Jens Luebeck, Kaiyuan Zhu, Joshua T. Lange, Roman Sasik, Chad Phillips, Sayed Sadat, Sara Javadzadeh, Qian Yang, Allen Wang, Kersi Pestonjamasp, Sara Rosenthal, Kathleen M Fisch, Paul S. Mischel, V. Bafna
          </td>
          <td>2024-09-05</td>
          <td>Research Square</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="Extrachromosomal, circular DNA (ecDNA) is a prevalent oncogenic alteration in cancer genomes, often associated with aggressive tumor behavior and poor patient outcome. While previous studies proposed a chromatin-based mobile enhancer model for ecDNA-driven oncogenesis, its precise mechanism and impact remains unclear across diverse cancer types. Our study, utilizing advanced multi-omics profiling, epigenetic editing, and imaging approaches in three cancer models, reveals that ecDNA hubs are an integrated part of nuclear condensates and exhibit cancer-type specific chromatin connectivity. Epigenetic silencing of the ecDNA-specific regulatory modules or chemically disrupting liquid-liquid phase separation breaks down ecDNA hubs, displaces MED1 co-activator binding, inhibits oncogenic transcription, and promotes cell death. These findings substantiate the trans-activator function of ecDNA and underscore a structural mechanism driving oncogenesis. This refined understanding expands our views of oncogene regulation and opens potential avenues for novel therapeutic strategies in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5554b124d7186535bee061a70cb36ab7c432ab43" target='_blank'>
              Extrachromosomal DNA Associates with Nuclear Condensates and Reorganizes Chromatin Structures to Enhance Oncogenic Transcription
              </a>
            </td>
          <td>
            Aziz Taghbalout, Chia-Hao Tung, P. A. Clow, Ping Wang, Harianto Tjong, Chee Hong Wong, Diane D. Mao, Rahul Maurya, Meng-Fan Huang, C. Ngan, Albert H. Kim, Chia-Lin Wei
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Rationale: Extrachromosomal circular DNA is a hallmark of cancer, but its role in shaping the genome heterogeneity of urothelial bladder carcinoma (UBC) remains poorly understood. Here, we comprehensively analyzed the features of extrachromosomal circular DNA in 80 UBC patients. Methods: We performed whole-genome/exome sequencing (WGS/WES), Circle-Seq, single-molecule real-time (SMRT) long-read sequencing of circular DNA, and RNA sequencing (RNA-Seq) on 80 pairs of tumor and AT samples. We used our newly developed circular DNA analysis software, Circle-Map++ to detect small extrachromosomal circular DNA from Circle-Seq data. Results: We observed a high load and significant heterogeneity of extrachromosomal circular DNAs in UBC, including numerous single-locus and complex chimeric circular DNAs originating from different chromosomes. This includes highly chimeric circular DNAs carrying seven oncogenes and circles from nine chromosomes. We also found that large tumor-specific extrachromosomal circular DNAs could influence genome-wide gene expression, and are detectable in time-matched urinary sediments. Additionally, we found that the extrachromosomal circular DNA correlates with hypermutation, copy number variation, oncogene amplification, and clinical outcome. Conclusions: Overall, our study provides a comprehensive extrachromosomal circular DNA map of UBC, along with valuable data resources and bioinformatics tools for future cancer and extrachromosomal circular DNA research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0454d85a5ac1363a4fa012edcd2f47cff5be9b32" target='_blank'>
              Extrachromosomal circular DNA orchestrates genome heterogeneity in urothelial bladder carcinoma
              </a>
            </td>
          <td>
            Wei Lv, Xiaoguang Pan, Peng Han, Shuang Wu, Yuchen Zeng, Qingqing Wang, Lidong Guo, Mengyang Xu, Yanwei Qi, Li Deng, Zhe Xu, Conghui Li, Tianxi Yu, Xin Cui, Huajing Teng, Chongjun Xiang, Haotian Tan, Yue Li, Ning Liang, Huiying Tao, Qingqing Gao, Guohua Yu, Jia Mi, Fuyi Xu, Benjiao Gong, Lei Shi, Tao Wang, Huanming Yang, Wei Dong, Lars Bolund, Lin Lin, Wenting Wang, Hanbo Li, Jinrong Huang, Chunhua Lin, Yonglun Luo
          </td>
          <td>2024-08-19</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a form of oncogene amplification found across cancer types and associated with poor outcome in patients. ecDNA can be structurally complex and contain rearranged DNA sequences derived from multiple chromosome locations. As the structure of ecDNA can impact oncogene regulation and may indicate mechanisms of its formation, disentangling it at high resolution from sequencing data is essential. Even though methods have been developed to identify and reconstruct ecDNA in cancer genome sequencing, it remains challenging to resolve complex ecDNA structures, in particular amplicons with shared genomic footprints. We here introduce Decoil, a computational method which combines a breakpoint-graph approach with regression to reconstruct complex ecDNA and deconvolve co-occurring ecDNA elements with overlapping genomic footprints from long-read nanopore sequencing. Decoil outperforms de novo assembly and alignment-based methods in simulated long-read sequencing data for both simple and complex ecDNAs. Applying Decoil on whole genome sequencing data uncovered different ecDNA topologies and explored ecDNA structure heterogeneity in neuroblastoma tumors and cell lines, indicating that this method may improve ecDNA structural analyzes in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c914721ddd777cd78df6632d6860e72cff344be" target='_blank'>
              Reconstructing extrachromosomal DNA structural heterogeneity from long-read sequencing data using Decoil.
              </a>
            </td>
          <td>
            M. Giurgiu, Nadine Wittstruck, Elias Rodriguez-Fos, R. Chamorro González, Lotte Brueckner, Annabell Krienelke-Szymansky, K. Helmsauer, Anne Hartebrodt, P. Euskirchen, R. Koche, Kerstin Haase, Knut Reinert, A. Henssen
          </td>
          <td>2024-08-07</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="DNA rearrangements are thought to arise from two classes of processes. The first class involves DNA breakage and fusion (“cut-and-paste”) without net DNA gain or loss. The second class involves aberrant DNA replication (“copy-and-paste”) and can produce either net DNA gain or loss. We previously demonstrated that the partitioning of chromosomes into aberrant structures of the nucleus, micronuclei or chromosome bridges, can generate cut-and-paste rearrangements by chromosome fragmentation and ligation. Surprisingly, in the progeny clones of single cells that have undergone chromosome bridge breakage, we identified large segmental duplications and short sequence insertions that are commonly attributed to copy-and-paste processes. Here, we demonstrate that both large duplications and short insertions are inherent outcomes of the replication and fusion of unligated DNA ends, a process we term breakage-replication/fusion (B-R/F). We propose that B-R/F provides a unifying explanation for complex rearrangement patterns including chromothripsis and chromoanasynthesis and enables rapid DNA amplification after chromosome fragmentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c002d451b4a62962b702db7317ff655026459317" target='_blank'>
              Chromosome breakage-replication/fusion enables rapid DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, David Pellman
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is a pervasive driver event in cancer, yet our understanding of how ecDNA forms is limited. Here, we couple a CRISPR-based method for ecDNA induction with extensive characterization of newly formed ecDNA to examine their biogenesis. We find that DNA circularization is efficient, irrespective of 3D genome context, with formation of 800kb, 1 Mb, and 1.8 Mb ecDNAs reaching or exceeding 15%. We show non-homologous end joining and microhomology-mediated end joining both contribute to ecDNA formation, while inhibition of DNA-PKcs and ATM have opposing impacts on ecDNA formation. EcDNA and the corresponding chromosomal excision scar can form at significantly different rates and respond differently to DNA-PKcs and ATM inhibition. Taken together, our results support a model of ecDNA formation in which double strand break ends dissociate from their legitimate ligation partners prior to joining of illegitimate ends to form the ecDNA and excision scar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e488ec3b1ac40e7e2986571eab100d1986b6e3f" target='_blank'>
              Disparate pathways for extrachromosomal DNA biogenesis and genomic DNA repair.
              </a>
            </td>
          <td>
            John C. Rose, J. Belk, I. Wong, Jens Luebeck, Hudson T Horn, Bence Daniel, Matthew G Jones, Kathryn E Yost, King L. Hung, Kevin S Kolahi, Ellis J Curtis, Calvin J. Kuo, V. Bafna, Paul S. Mischel, Howard Y. Chang
          </td>
          <td>2024-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>70</td>
        </tr>

        <tr id="L1 retrotransposition is a significant source of genomic variation in human epithelial tumours, which can contribute to tumorigenesis. However, fundamental questions about the causes and consequences of L1 activity in cancer genomes remain unresolved, primarily due to the limitations of short-read sequencing technologies. Here, we employ multiplatform sequencing, with an emphasis on long reads, to analyse a fine selection of 10 tumours exhibiting high rates of somatic retrotransposition, encompassing over 6000 events. The analysis of L1 locus-specific single-nucleotide variants reveals a novel panorama of L1 loci activity. Furthermore, examination of the internal structure of somatic L1s uncovers the mechanisms behind their inactivation. A hidden landscape of chromosomal aberrations emerges in the light of long reads, where reciprocal translocations mediated by L1 insertion represent frequent events. Resolution of L1 bridges’ configuration elucidates the mechanisms of their formation, where typically two independent, but synchronous, somatic L1 insertions drive the reciprocal exchange between non-homologous chromosomes. Timing analyses indicate that L1 retrotransposition is an early driver of chromosomal instability, active before the first whole-genome doubling event. Overall, these findings highlight L1 activity as a more significant contributor to tumour genome plasticity than previously recognized, extending its impact beyond simple insertional mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a02d00b189cce10c8aa198ed86d7194c64bd32e" target='_blank'>
              Synchronous L1 retrotransposition events promote chromosomal crossover early in human tumorigenesis
              </a>
            </td>
          <td>
            Sonia Zumalave, Martín Santamarina, N. P. Espasandin, D. García-Souto, J. Temes, Toby M. Baker, Ana Pequeño-Valtierra, Iago Otero, J. Rodríguez-Castro, Ana Oitaben, Eva G. Álvarez, Paula Otero, Iria Díaz-Arias, Mónica Martínez-Fernández, P. Van Loo, G. Cristofari, Bernardo Rodriguez-Martin, J. Tubío
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Enhancer elements are specific DNA sequences that play a crucial role in regulating gene expression. Located upstream or downstream in the genomic context, enhancers enhance the transcription of linked genes. This is achieved by providing binding sites for transcription factors and other proteins, acting as a nucleation point that promotes the assembly of the transcriptional machinery.
What makes enhancers unique is that located even thousands of base pairs away, they can exert their function. They can act over long distances, looping to interact with the promoter region of their target genes. Enhancers are often involved in cell type-specific gene expression as different cells express different sets of genes by activating and repressing cell type-specific enhancers.
Cancer cells seem to co-opt this mechanism and hijack it for their own survival. In this process, an aberrant enhancer element activates the transcription of oncogenes (genes that have the potential to cause cancer) due to some alterations in genomic structure or regulatory elements that happen in cancer, contributing to tumorigenesis and cancer progression. Enhancer hijacking often occurs through complex genomic rearrangements such as chromosomal translocations and other structural rearrangements such as deletions or amplifications.8 These alterations can bring a usually distant, far-located enhancer into proximity with an oncogene, leading to its inappropriate activation. This is often seen in various cancers, including leukemias and solid tumors.
Enhancer hijacking and its effects on gene regulation in cancer are frequently studied using methods such as RNA sequencing, chromatin conformation capture, and CRISPR-Cas9. Finding an enhancer hijacking event may be used to inform treatment choices, especially in precision medicine settings, and may also function as a biomarker for particular cancer types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eefe88575de8420490e0e1dd10a24dbefe2eef41" target='_blank'>
              Enhancer hijacking: Innovative ways of carcinogenesis
              </a>
            </td>
          <td>
            Ruby Dhar, Arun Kumar, Subhradip Karmakar
          </td>
          <td>2024-09-02</td>
          <td>Asian Journal of Medical Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Acute myeloid leukemia with complex karyotype (ckAML) is characterized by high genomic complexity, including frequent TP53 mutations and chromothripsis. We hypothesized that the numerous genomic rearrangements could reposition active enhancers near proto-oncogenes, leading to their aberrant expression. We developed pyjacker, a computational tool for the detection of enhancer hijacking events, and applied it to a cohort of 39 ckAML samples. Pyjacker identified motor neuron and pancreas homeobox 1 (MNX1), a gene aberrantly expressed in 1.4% of AML patients, often as a result of del(7)(q22q36) associated with hijacking of a CDK6 enhancer. MNX1-activated cases show significant co-occurrence with BCOR mutations and a gene signature shared with t(7;12)(q36;p13) pediatric AML. We demonstrated that MNX1 is a dependency gene, as its knockdown in a xenograft model reduces leukemia cell fitness. In conclusion, enhancer hijacking is a frequent mechanism for oncogene activation in AML. Statement of significance This study examines the consequences of structural alterations and demonstrates that proto-oncogene activation by enhancer hijacking is an overlooked pathomechanism in AML. MNX1 overexpression demonstrates that deletions on chromosome 7q can not only lead to haploinsufficiency, but also to activation of oncogenes by enhancer hijacking, providing a novel leukemogenic mechanism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dbb3e7eaaf436e861083291af3f4d69f9c709309" target='_blank'>
              Pyjacker identifies enhancer hijacking events in acute myeloid leukemia including MNX1 activation via deletion 7q
              </a>
            </td>
          <td>
            Etienne Sollier, Anna Riedel, U. Toprak, Justyna A. Wierzbinska, D. Weichenhan, J. P. Schmid, M. Hakobyan, A. Touzart, E. Jahn, B. Vick, Fiona Brown-Burke, Katherine Kelly, Simge Kelekçi, Anastasija Pejkovska, Ashish Goyal, Marion Bähr, Kersten Breuer, Mei-Ju May Chen, Maria Llamazares-Prada, M. Hartmann, M. Schönung, Nádia Correia, A. Trumpp, Yomn Abdullah, Ursula Klingmüller, S. S. Mughal, B. Brors, Frank Westermann, Matthias Schlesner, S. Vosberg, T. Herold, P. Greif, Dietmar Pfeifer, Michael Lübbert, Thomas Fischer, F. H. Heidel, C. Gebhard, W. Walter, Torsten Haferlach, Ann-Kathrin Eisfeld, K. Mrózek, D. Nicolet, L. Bullinger, L. Smeenk, Claudia Erpelinck, Roger Mulet-Lazaro, Ruud Delwel, Aurélie Ernst, Michael Scherer, P. Lutsik, I. Jeremias, Konstanze Döhner, H. Döhner, D. Lipka, C. Plass
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="An important goal in cancer research is to identify driver genes and mutations. Reasoning that such mutations often alter enzymatic functions, we investigated the cancer driver role of enzyme families. Using pan-cancer genomic data and established driver mutation catalogues, we found an unexpectedly high rate of mutations in helicases, making helicases the most frequently mutated enzyme family in cancer. Based on both functional perturbation screens and cancer genomic analyses, we provide evidence that cancers with mutated helicases converge on increased genomic instability and faulty DNA repair. We identify a striking phenotype in cells with loss of the helicase Aquarius (AQR). AQR was exclusively hemizygous lost in cancer genomes, which was associated with elevated levels of structural variants and point mutation signatures indicative of homologous recombination deficiency. Finally, we leverage large dependency maps to show that hemizygous loss is a common tumour suppression mechanism among helicases. In summary, we uncover a striking frequency of mutated helicases with key roles in genomic maintenance, and we nominate novel hemizygous cancer driver genes including AQR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/381ed30f7e9d5d197bbe8fa9f74d37852c71bb4a" target='_blank'>
              Enzyme family-centred approach identifies helicases as recurrent hemizygous tumour suppressor genes
              </a>
            </td>
          <td>
            Karolin Voßgröne, F. Favero, Krushanka Kashyap, F. G. Rodríguez-González, André Vidas Olsen, Xin Li, Balca R. Mardin, Joachim Weischenfeldt, Claus S. Sørensen
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="Great apes have maintained a stable karyotype with few large-scale rearrangements; in contrast, gibbons have undergone a high rate of chromosomal rearrangements coincident with rapid centromere turnover. Here we characterize assembled centromeres in the Eastern hoolock gibbon, Hoolock leuconedys (HLE), finding a diverse group of transposable elements (TEs) that differ from the canonical alpha satellites found across centromeres of other apes. We find that HLE centromeres contain a CpG methylation centromere dip region, providing evidence this epigenetic feature is conserved in the absence of satellite arrays; nevertheless, we report a variety of atypical centromeric features, including protein-coding genes and mismatched replication timing. Further, large structural variations define HLE centromeres and distinguish them from other gibbons. Combined with differentially methylated TEs, topologically associated domain boundaries, and segmental duplications at chromosomal breakpoints, we propose that a “perfect storm” of multiple genomic attributes with propensities for chromosome instability shaped gibbon centromere evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6ad367a909b0a60178d576c36304f1f927c20ef" target='_blank'>
              Centromeric transposable elements and epigenetic status drive karyotypic variation in the eastern hoolock gibbon
              </a>
            </td>
          <td>
            Gabrielle Hartley, M. Okhovat, Savannah J. Hoyt, Emily Fuller, Nicole Pauloski, Nicolas Alexandre, I. Alexandrov, R. Drennan, Danilo Dubocanin, David M. Gilbert, Yizi Mao, Christine McCann, Shane Neph, Fedor Ryabov, Takayo Sasaki, Jessica M. Storer, Derek Svendsen, William Troy, Jackson Wells, LM Core, A. Stergachis, Lucia Carbone, Rachel J. O’Neill
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Chromosome aberrations (CAs), a genotoxic potential of carcinogens, are believed to contribute to tumorigenesis by chromosomal rearrangements through micronucleus formation. However, there is no direct evidence that proves the involvement of CAs in tumorigenesis in vivo. In the current study, we sought to clarify the involvement of CAs in chemical carcinogenesis using a rat model with a pure CA-inducer hepatocarcinogen, acetamide. Whole-genome analysis indicated that hepatic tumors induced by acetamide treatment for 26-30 weeks showed a broad range of copy number alterations in various chromosomes. In contrast, hepatic tumors induced by a typical mutagen (diethylnitrosamine) followed by a nonmutagen (phenobarbital) did not show such mutational patterns. Additionally, structural alterations such as translocations were observed more frequently in the acetamide-induced tumors. Moreover, most of the acetamide-induced tumors expressed c-Myc and/or MDM2 protein due to the copy number gain of each oncogene. These results suggest the occurrence of chromosomal rearrangements and subsequent oncogene amplification in the acetamide-induced tumors. Taken together, the results indicate that CAs are directly involved in tumorigenesis through chromosomal rearrangements in an acetamide-induced hepatocarcinogenesis rat model.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4992ea68f0d5c57d06c0aa2d2039720d0742395b" target='_blank'>
              Chromosome aberrations cause tumorigenesis through chromosomal rearrangements in a hepatocarcinogenesis rat model.
              </a>
            </td>
          <td>
            Kenji Nakamura, Yuji Ishii, S. Takasu, Moeka Namiki, Meili Soma, N. Takimoto, K. Matsushita, M. Shibutani, Kumiko Ogawa
          </td>
          <td>2024-09-08</td>
          <td>Cancer science</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Understanding the impact of genetic alterations on epigenomic phenotypes during breast cancer progression is challenging with unimodal measurements. Here, we report wellDA-seq, the first high-genomic resolution, high-throughput method that can simultaneously measure the whole genome and chromatin accessibility profiles of thousands of single cells. Using wellDA-seq, we profiled 22,123 single cells from 2 normal and 9 tumors breast tissues. By directly mapping the epigenomic phenotypes to genetic lineages across cancer subclones, we found evidence of both genetic hardwiring and epigenetic plasticity. In 6 estrogen-receptor positive breast cancers, we directly identified the ancestral cancer cells, and found that their epithelial cell-of-origin was Luminal Hormone Responsive cells. We also identified cell types with copy number aberrations (CNA) in normal breast tissues and discovered non-epithelial cell types in the microenvironment with CNAs in breast cancers. These data provide insights into the complex relationship between genetic alterations and epigenomic phenotypes during breast tumor evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d380546a2c5346cf2ff8db012770db28bb0473a0" target='_blank'>
              Single cell genome and epigenome co-profiling reveals hardwiring and plasticity in breast cancer
              </a>
            </td>
          <td>
            Kaile Wang, Yun Yan, Heba Elgamal, Jianzhuo Li, Chenling Tang, S. Bai, Zhenna Xiao, Emi Sei, Yiyun Lin, Junke Wang, Jessica Montalvan, Changandeep Nagi, Alastair M. Thompson, Nicholas E. Navin
          </td>
          <td>2024-09-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Introduction: Copy number variations (CNVs) are structural genomic alterations that involve changes in the number of copies of specific DNA regions. These variations can include deletions, duplications, and more complex rearrangements, and play a critical role in cancer progression by amplifying oncogenes, deleting tumor suppressor genes, or altering other key genomic regions. Despite the importance of CNVs in cancer biology, there is a lack of comprehensive resources that aggregate CNV data across multiple cancer types, impeding the exploration of their role in different malignancies. The PanCNV-Explorer was developed to fill this gap by providing a pan-cancer resource that integrates CNV data for over 21 cancer types. Methods: The PanCNV-Explorer was developed, utilizing data from repositories such as COSMIC, DepMap, DGV, and dbVAR. Amplifications and deletions can be visualized, with detailed chromosome-specific plots highlighting CNV patterns. In addition, a genome browser has been implemented to display regions of interest for searching for genes of potential relevance. Results: The database contains over 5 million CNVs from 15,809 samples, spanning 21 primary cancer types. Pathogenic CNVs from ClinVar and DepMap were compared to benign data from DGV and dbVAR, allowing a comprehensive analysis across tissues. CNV patterns were visualized for individual cancers. We demonstrate with use cases, focusing on common oncogenes such as MYC and tumor suppressor genes such as TP53, the ability of the resource to search for cancer driver genes. For example, chromosome 12 in pancreatic cancer showed frequent amplifications in the region containing the KRAS oncogene. In another example, we show how driver gene candidates already described in certain cancer entities can also be found with high frequency in other entities. Conclusion: The PanCNV-Explorer provides a valuable resource for pan-cancer CNV analysis, facilitating the exploration of CNV-driven tumor heterogeneity. The integrated visualization tools and genome browser enable detailed exploration of genomic regions, enhancing cancer research. As a valuable and easy-to-use platform that allows researchers to quantify and compare CNVs across cancer types, the PanCNV-Explorer is an essential tool for advancing cancer genomics research. The main page is available at https://mtb.bioinf.med.uni-goettingen.de/pancnv-explorer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0740ab5d12642c500ab7be4dd721a73d9e32d198" target='_blank'>
              PanCNV Explorer: A pan-cancer database for copy number variations.
              </a>
            </td>
          <td>
            Kevin Kornrumpf, Jürgen Dönitz
          </td>
          <td>2024-09-24</td>
          <td>None</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcfc9fc5c7ce6b28eb00b12e973f760953fb4c62" target='_blank'>
              Enhancer-promoter hubs organize transcriptional networks promoting oncogenesis and drug resistance
              </a>
            </td>
          <td>
            Brent S Perlman, Noah G Burget, Yeqiao Zhou, Gregory W. Schwartz, Jelena Petrovic, Z. Modrusan, R. Faryabi
          </td>
          <td>2024-09-14</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Lung cancer (LC) is a highly lethal cancer worldwide. Research on the distribution and nature of extrachromosomal DNA molecules (EcDNAm) in early LC is scarce. In this study, after removing linear DNA and mitochondrial circular DNA, EcDNAm were extracted from two paired LC tissue samples and amplified using rolling circle amplification. High throughput extrachromosomal DNA (EcDNA) or RNA sequencing and bioinformatics analysis were subsequently utilized to explore the distribution and nature of the EcDNAm. Additionally, to elucidate the role of oncogenes with large EcDNAm sizes, gene ontology and Kyoto Encyclopedia of Genes and Genomes pathway analyses were performed. The RNA sequencing results revealed significant differences in certain genes between tumors and corresponding normal samples. At the same time, slight distinctions were observed between relapsed and non-relapsed tumor samples. The nature of the EcDNAm was compared between LC samples and matched normal samples. There was a tendency for the number of EcDNAm with longer size (EcDNA) and its containing driver oncogenes to be higher in cancer samples. Enrichment analysis of the cancer samples revealed enrichment in biological processes, such as positive regulation of protein localization, axon development, and in-utero embryonic development. This study highlights the universal distribution and characteristics of EcDNAm in early LC. Moreover, our work fills the investigation of the EcDNAm gap and future studies should focus on the application of EcDNA as a potential biomarker in patients with early LC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c06d1cf4ef24c8fc6225589779b38bcba724aa5" target='_blank'>
              The distribution of the extrachromosomal DNA molecules in early lung cancer
              </a>
            </td>
          <td>
            Jianfei Fang, L. Ying, Zhengxiao Ma, Ying Yang, Rui Zhu, Dan Su
          </td>
          <td>2024-07-01</td>
          <td>Science Progress</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Chromosomal instability (CIN) generates micronuclei—aberrant extranuclear structures that catalyze the acquisition of complex chromosomal rearrangements present in cancer. Micronuclei are characterized by persistent DNA damage and catastrophic nuclear envelope collapse, which exposes DNA to the cytoplasm. We found that the autophagic receptor p62/SQSTM1 modulates micronuclear stability, influencing chromosome fragmentation and rearrangements. Mechanistically, proximity of micronuclei to mitochondria led to oxidation-driven homo-oligomerization of p62, limiting endosomal sorting complex required for transport (ESCRT)–dependent micronuclear envelope repair by triggering autophagic degradation. We also found that p62 levels correlate with increased chromothripsis across human cancer cell lines and with increased CIN in colorectal tumors. Thus, p62 acts as a regulator of micronuclei and may serve as a prognostic marker for tumors with high CIN. Editor’s summary A high rate of chromosome mis-segregation, known as chromosomal instability, is a pervasive feature of tumors. Mis-segregated chromosomes can be encapsulated into micronuclei, abnormal nuclear structures that fuel the acquisition of complex chromosomal rearrangements upon envelope collapse. Two groups of investigators now provide mechanistic insights into micronuclei collapse and the consequences of this process for cancer (see the Perspective by Maddaluno and Settembre). Martin et al. report that mitochondria-derived reactive oxygen species (ROS) oxidize the autophagic protein p62, enhancing its autophagic activity toward components of the repair machinery and thus increasing micronuclear collapse. Elevated p62 levels are associated with increased chromosomal rearrangements in cancer cells and in high–chromosomal instability tumors, pointing to p62 as a potential prognostic marker for such tumors. Di Bona et al. found that ROS interfere with a membrane repair protein called CHMP7, which normally helps to maintain the integrity of the nuclear envelope. ROS cause CHMP7 to cluster and bind with another protein, LEMD2, leading to the breakdown of micronuclei. This collapse contributes to genetic abnormalities and inflammation, linking oxidative damage to cancer’s aggressive behavior. —Stella M. Hurtley INTRODUCTION A main feature of cancer cells is their high frequency of chromosome segregation errors, a condition known as chromosomal instability (CIN), which is associated with poor prognosis and chemoresistance. CIN leads to the formation of micronuclei—abnormal extranuclear bodies widely found in cancer cells. The nuclear envelopes of micronuclei are often fragile and compromised, which causes irreparable ruptures and structural collapse. This exposes micronuclear DNA to the cytosol, resulting in DNA damage and extensive chromosomal rearrangements, which fuels genomic instability and cancer progression. In addition, rupture of the micronuclear membrane activates the cGAS-STING [cyclic guanosine monophosphate–adenosine monophosphate synthase (cGAS)–stimulator of interferon genes (STING)] pathway, triggering inflammatory responses that promote tumor invasion and metastasis. RATIONALE Although the consequences of loss of micronuclei compartmentalization are well studied, the events leading to micronuclear envelope rupture and the mechanisms involved in their repair are less clear. Given the importance of those mechanisms in maintaining micronuclei integrity, we aimed to identify proteins and pathways that specifically modulate these events. We isolated micronuclei and primary nuclei, analyzed their protein composition using mass spectrometry, and identified p62/SQSTM1—a key autophagic pathway component—as specifically enriched in micronuclei compared with primary nuclei. RESULTS We found that p62 localizes to micronuclei and is crucial for maintaining their integrity. The localization of p62 to micronuclei relied on its oxidation-driven homo-oligomerization, which was induced by reactive oxygen species (ROS) released by mitochondria proximal to micronuclei. We used electron tomography and quantitative imaging to analyze the proximity of mitochondria to micronuclei and found that p62-positive micronuclei were closer to mitochondria compared with those without p62. We tested how mitochondrial ROS affected p62 localization and found that reducing ROS decreased p62-positive micronuclei and prevented their rupture, whereas increasing ROS led to more p62-positive and ruptured micronuclei in both normal and cancer cells. Homo-oligomerization of p62 induced by ROS influenced micronuclear integrity by inhibiting the repair activity of the endosomal sorting complex required for transport–III (ESCRT-III), which is involved in envelope resealing. This control was achieved through the p62-mediated autophagic degradation of ESCRT components. The p62-dependent regulation of micronuclear integrity affected several features associated with micronuclei, such as chromosomal rearrangements and cGAS-STING–mediated inflammation. Furthermore, p62 levels correlated with chromothripsis status in cancer cells and tumor samples and were strongly linked to poor prognosis in colorectal cancer with high CIN. CONCLUSION In this work, we identified a critical mechanism by which p62 regulates micronuclear stability by influencing envelope repair. Given p62’s frequent deregulation in tumors and its control over micronuclear envelope integrity, our findings suggest that p62-mediated effects on micronuclear repair activities could influence cancer development and progression. p62/SQSTM1 drives micronuclei collapse and complex chromosome rearrangements by restraining micronuclear envelope repair. Micronuclei-mitochondria proximity promotes ROS-mediated homo-oligomerization of p62/SQSTM1 through cysteine oxidation. ROS-induced p62 homo-oligomerization affects micronuclear integrity by inhibiting components of ESCRT-III repair through their autophagic degradation. This affects chromosomal rearrangements and cGAS-STING inflammation. High p62 levels correlate with chromothripsis and poor prognosis in colorectal cancer with high CIN.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/180c1ddc6e5e2000c0d5181084de24b29d56eee4" target='_blank'>
              A p62-dependent rheostat dictates micronuclei catastrophe and chromosome rearrangements
              </a>
            </td>
          <td>
            Sara Martin, Simone Scorzoni, Sara Cordone, Alice Mazzagatti, Galina V Beznoussenko, Amanda L. Gunn, Melody Di Bona, Yonatan Eliezer, Gil Leor, Tal Ben-Yishay, A. Loffreda, V. Cancila, Maria Chiara Rainone, M. Ippolito, Valentino Martis, Fabio Bedin, M. Garrè, L. P. Vaites, Paolo Vasapolli, Simona Polo, D. Parazzoli, C. Tripodo, Alexander A. Mironov, A. Cuomo, Uri Ben-David, S. Bakhoum, Emily M. Hatch, Peter Ly, S. Santaguida
          </td>
          <td>2024-08-30</td>
          <td>Science</td>
          <td>3</td>
          <td>36</td>
        </tr>

        <tr id="CIZ1 is part of the large RNA-dependent supramolecular assemblies that form around the inactive X-chromosome (Xi) in female cells, and smaller assemblies throughout the nucleus in males and females. Here we show that CIZ1 C-terminal anchor domain (AD) is elevated in primary human breast tumour transcriptomes, even at stage I. Elevation of AD correlates with deprotection of chromatin, and up-regulation of lncRNA-containing gene clusters in approximately 10Mb regions that are enriched in cancer-associated genes. We modelled the effect of ectopic AD on endogenous CIZ1-Xi assemblies and observed dominant-negative interference with their re-formation after mitosis, leading to abnormal Xi assemblies similar to those in breast cancer cells, and depletion of histone modifications H2AK119ub1 and H3K27me3. Within days consistent alterations in gene expression were evident across the genome, showing that disruption of CIZ1-RNA assemblies has a destabilizing effect, likely by unscheduled exposure of underlying chromatin to modifying enzymes. Together the data argue for a dominant, potent and rapid effect of CIZ1 AD, that can deprogram established patterns of gene expression and which may predispose incipient tumours to epigenetic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44d2a58fd6870a0dd51c6ac1e82db8148da5b9e2" target='_blank'>
              Epigenetic deprogramming driven by disruption of CIZ1-RNA nuclear assemblies in early-stage breast cancers
              </a>
            </td>
          <td>
            Gabrielle L. Turvey, Ernesto López de Alba, Emma R Stewart, Heather Cook, Ahmad Alalti, Richard T. Gawne, J. Ainscough, Andrew S. Mason, Dawn Coverley
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly toxic lesions that underly the efficacy of ionizing radiation (IR) and a large number of cytotoxic chemotherapies1–3. Yet, abnormal repair of DSBs is associated with genomic instability and may contribute to cancer heterogeneity and tumour evolution. Here, we show that DSBs induced by IR, by DSB-inducing chemotherapeutics, or by the expression of a rare-cutting restriction endonuclease induce large-scale genomic amplification in human cancer cells. Importantly, the extent of DSB-induced genomic amplification (DIGA) in a panel of melanoma cell lines correlated with the degree of cytotoxicity elicited by IR, suggesting that DIGA contributes significantly to DSB-induced cancer cell lethality. DIGA, which is mediated through conservative DNA synthesis, does not require origin re-licensing, and is enhanced by the depletion or deletion of the methyltransferases SET8 and SUV4-20H1, which function sequentially to mono- and di-methylate histone H4 lysine 20 (H4K20) at DSBs to facilitate the recruitment of 53BP1-RIF1 and its downstream effector shieldin complex to DSBs to prevent hyper-resection4–11. Consistently, DIGA was enhanced in cells lacking 53BP1 or RIF1, or in cells that lacked components of the shieldin complex or of other factors that help recruit 53BP1 to DSBs. Mechanistically, DIGA requires MRE11/CtIP and EXO1, factors that promote resection and hyper-resection at DSBs, and is dependent on the catalytic activity of the RAD51 recombinase. Furthermore, deletion or depletion of POLD3, POLD4, or RAD52, proteins involved in break-induced replication (BIR), significantly inhibited DIGA, suggesting that DIGA is mediated through a RAD51-dependent BIR-like process. DIGA induction was maximal if the cells encountered DSBs in early and mid S-phase, whereas cells competent for homologous recombination (in late S and G2) exhibited less DIGA induction. We propose that unshielded, hyper-resected ends of DSBs may nucleate a replication-like intermediate that enables cytotoxic long-range genomic DNA amplification mediated through BIR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec06bf7ab705064fc487b428fdb7631566b4ae0f" target='_blank'>
              Break-induced replication drives large-scale genomic amplifications in cancer cells
              </a>
            </td>
          <td>
            Mouadh Benamar, Rebeka Eki, Kangping Du, Tarek Abbas
          </td>
          <td>2024-08-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The 10q11.22 chromosomal region is a duplication-rich interval of the human genome and one of the last to be fully assembled. It carries copy-number variable genes associated with intellectual disability, bipolar disorder, and obesity. In this study, we characterized the structural diversity at this locus by analyzing 64 haploid assemblies produced by the Human Pangenome Reference Consortium. We identified eleven alternative haplotypes that differ in the copy number and/or orientation of large genomic segments, ranging from hundreds of kilobase pairs (kbp) to over one megabase pair (Mbp). We uncovered a 2.4 Mbp size difference between the shortest and longest haplotypes. Breakpoint analysis revealed that genomic instability results from nonallelic homologous recombination between segmental duplication (SD) pairs with varying similarity (94.4-99.6%). Nonetheless, these pairs generally recombine at positions where their identity is higher (>99.6%). Recurrent inversions occur with varying breakpoints within the same inverted SD pair. Inversion polymorphisms shuffle the entire SD arrangement, creating new predispositions to copy-number variations. The SD architecture is associated with a catarrhine-specific subgroup of the AGAP gene family, which likely triggered the accumulation of SDs at this locus over the past 25 million years of human evolution. Our results reveal extensive structural diversity and genomic instability at the 10q11.22 locus and expand the general understanding of the mutational mechanisms behind SD-mediated rearrangements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b45817e56dbf9d949483be26e4d2690cbf8b3395" target='_blank'>
              AGAP duplicons associate with structural diversity at Chromosome 10q11.22.
              </a>
            </td>
          <td>
            Stefania Fornezza, V. S. Delvecchio, William T. Harvey, Philip C. Dishuck, Evan E. Eichler, Giuliana Giannuzzi
          </td>
          <td>2024-09-25</td>
          <td>Genome research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Introduction Type B1 and B2 thymomas are lymphocyte-rich malignant tumors with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, non-coding regions remain uncharacterized. Here, we developed a rigorous tumor isolation method from lymphocyte-rich thymoma tissues and identified somatic mutations in non-coding and mitochondrial DNA. Methods CD205+CD45- pure tumor cells were isolated from fresh-frozen tissues using DEPArray system. Deep whole-genome sequencing was performed, and recurrent somatic alterations in coding, non-coding, and mitochondria regions were systemically identified by computational framework. The mutations were classified according to gene function, cis-regulatory element, and mutational signature. Results The total number of somatic mutations was approximately 80 times higher in non-coding regions than in coding regions in type B1-2 thymomas (1,671.3 vs. 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of cases exhibited fewer than four mutations in coding regions. A systematic non-coding analysis identified a total of 405.0 mutations per case on cis-regulatory elements, and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Mitochondrial heteroplasmy was observed in 90% of cases, with a significant proportion of mutations located in D-loop region. The single-base substitution pattern was signature 12. Conclusions Numerous non-coding mutations and mitochondrial heteroplasmy were detected in type B1 and B2 thymomas. Given the paucity of coding mutations observed in this disease entity, disruption of the non-coding landscape and mitochondrial heteroplasmic shift may be the primary cause of thymoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/559b9056aa5e8bd421441f43ad9a55ae5b38dbc9" target='_blank'>
              Genome-wide analysis of somatic non-coding mutation patterns and mitochondrial heteroplasmy in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Juliet King, Emma McLean, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Background: In neuroblastoma (NB), intratumor heterogeneity (ITH) is frequently observed, but the role of cell-to-cell allele-specific copy number alterations in phenotypic variation, clonal evolution and treatment response remains to be determined. Here we investigate ITH, timing of specific genomic aberrations, single-cell replication timing and the co-evolution of the genome and transcriptome in NB tumors at single-cell resolution, with an aim to analyse subclonal dynamics and clone-specific response or resistance under targeted therapeutic pressure. Methods: In addition to germline/tumor bulk whole exome sequencing (WES), ultra-low depth (0.25x) single-cell whole-genome DNA sequencing (scDNAseq) was performed using 10x genomics Chromium single-cell CNV (scCNV) kit and 9410 tumor cells were characterized from 14 patient-derived xenografts (PDX) NB-models and 4 tumor biopsies from NB-patients, either at diagnosis (n=7), progression (n=3) or relapse (n=8). Single-cell RNA sequencing (scRNAseq) data was obtained from the same PDX and patient tumor samples (Thirant et al, 2023). 6/14 PDX models were subjected to different treatment combinations (targeted treatment with/without chemotherapy) and bulk WES was performed at two time-points, pre- and post-treatment. Results: Both monoclonal (n=7) and polyclonal (n=11) genomes were determined by allele and haplotype specific copy number (CN) alteration using both scDNAseq and scRNAseq data analysis, with 2 to 11 clones observed per polyclonal NB tumor. Whole genome duplication events (n=7) were observed in both polyclonal and monoclonal genomes. Known driver CN (segmental loss in chr1p and chr11q and gain at chr17q, or MYCN/ALK amplification) or somatic mutations (ALK/ATRX/TP53/NF1) were early clonal events.Study of replication timing (RT) based on scDNAseq revealed significant differences in RT between the MYCN amplified (n=7) and non-amplified groups (n=5), with NB tumors without MYCN amplification (no MNA) characterized by a predominance of late replicating domains, in contrast to MYCN amplified (MNA) tumors, which are enriched in early replicating domains In a PDX model of interest, scDNAseq analysis showed parallel copy number evolution of two distinct clones, subclone s1/s2. Data integration of clonal mutational profiles with pre- and post-targeted therapy (Lorlatinib) revealed clone-specific treatment response. Subclone s2 was partially responding with extinction of a sub-set of somatic alterations, whereas no change was observed in subclone s1. The replication timing (RT) profile of these two clones, subclone s1 (early-RT) and s2 (late-RT) were mutually exclusive. Genotype to phenotype analysis revealed subclone s1 genotype was preferentially expressed at transcriptomic level. Conclusion: Together, these results determine the evolutionary trajectories of NB tumors, linked to distinct replication timing and highlight opportunities for targetable early clonal alteration detection.
 Citation Format: Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, Didier Surdez, Sakina Zaidi, Gaelle Pierron, Angel Montero Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, Isabelle Janoueix-lerosey, Gudrun Schleiermacher. Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B074.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02f5489383eed60c9973409b18fbcca1b1288062" target='_blank'>
              Abstract B074: Clonal decomposition and DNA replication states defined by scaled single-cell DNA and RNA sequencing suggest clone-specific therapeutic vulnerabilities in neuroblastoma
              </a>
            </td>
          <td>
            Jaydutt Bhalshankar, Angela Bellini, Irene Jimenez, Cécile Thirant, Elnaz Saberi-Ansari, Yasmine Iddir, Alexandra Saint Charles, Charlotte Butterworth, Amira Kramdi, Virginie Raynal, Sylvain Baulande, D. Surdez, Sakina Zaidi, G. Pierron, A. Carcaboso, Birgit Geoerger, Andrei Zinovyev, Olivier Delattre, I. Janoueix-Lerosey, G. Schleiermacher
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>55</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Continuous chromosome missegregation over successive mitotic divisions, known as chromosomal instability (CIN), is common in cancer. Increasing CIN above a maximally tolerated threshold leads to cell death due to loss of essential chromosomes. Here, we show in two tissue contexts that otherwise isogenic cancer cells with higher levels of CIN are more sensitive to ionizing radiation, which itself induces CIN. CIN also sensitizes HPV-positive and HPV-negative head and neck cancer patient derived xenograft (PDX) tumors to radiation. Moreover, laryngeal cancers with higher CIN prior to treatment show improved response to radiation therapy. In addition, we reveal a novel mechanism of radiosensitization by docetaxel, a microtubule stabilizing drug commonly used in combination with radiation. Docetaxel causes cell death by inducing CIN due to abnormal multipolar spindles rather than causing mitotic arrest, as previously assumed. Docetaxel-induced CIN, rather than mitotic arrest, is responsible for the enhanced radiation sensitivity observed in vitro and in vivo, challenging the mechanistic dogma of the last 40 years. These results implicate CIN as a potential biomarker and inducer of radiation response, which could provide valuable cancer therapeutic opportunities. Statement of Significance Cancer cells and laryngeal tumors with higher chromosome missegregation rates are more sensitive to radiation therapy, supporting chromosomal instability as a promising biomarker of radiation response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/324e0283c2a65fbf5296691fea31358474092654" target='_blank'>
              Chromosomal instability increases radiation sensitivity
              </a>
            </td>
          <td>
            P. Cosper, M. Paracha, Kathryn M. Jones, Laura C. F. Hrycyniak, Les Henderson, Ava Bryan, D. Eyzaguirre, Emily McCunn, Elizabeth Boulanger, Jun Wan, K. Nickel, Vanessa L Horner, Rong Hu, Paul M Harari, R. Kimple, B. Weaver
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="High copy number endometrial cancers (HCNEC) are dominated by excessive duplications scattered across the genome, termed here as the HyperDuplication GenomoPhenotype (HDGP). Although correlated with cancer progression, its biological significance and implications for therapy have not yet been established. We identified locations and sizes of duplications in 171 endometrial cancer cases and designated 71 HCNEC cases as HDGP. We also investigated the response to the pan-ERBB inhibitor afatinib in a subset of HDGP-EC cases with ERBB2/ERBB3 duplications using a patient-derived three-dimensional culture model. Our analysis demonstrates that beyond tandem duplications there is a more general pattern involving coordinated duplication of multiple distant regions of the genome, demonstrating preferential selectivity to over-expressed potential oncogenes within a broad network. This suggests that HDGP increases tumor fitness and resistance to therapy by perturbing important gene networks in concert rather than only driver genes, suggesting a mechanistic basis for the ineffectiveness of targeted drugs in these patients and highlighting the need for combination therapies in these highly aggressive cases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4e4f25ff2ac4650bfe407694d21fc4d8e5d3f46" target='_blank'>
              Functional impact of the hyperduplication genomophenotype in high copy number endometrial cancer
              </a>
            </td>
          <td>
            Angela Florio, J. Smadbeck, , W. Lin, Dorsay Sadeghian, Sotiris Sotiriou, Rebeca Salvatori, R. Feathers, Taylor Berry, Lindsey Kinsella, F. Harris, A. Mccune, Stephen J. Murphy, Mohamed F. Ali, Abdulmohammad Pezeshki, Michael T. Barrett, Leah O. Grcevich, Ilaria Capasso, L. D. De Vitis, G. Schivardi, Tommaso Occhiali, A. Larish, John J. Weroha, M. Borad, John C. Cheville, P. Anastasiadis, Andrea Mariani, G. Vasmatzis
          </td>
          <td>2024-09-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="ZFTA-RELA is the most recurrent genetic alteration seen in pediatric supratentorial ependymoma (EPN) and is sufficient to initiate tumors in mice. Despite ZFTA-RELA’s potent oncogenic potential, ZFTA-RELA gene fusions are observed exclusively in childhood EPN, with tumors located distinctly in the supratentorial region of the central nervous system (CNS). We hypothesized that specific chromatin modules accessible during brain development would render distinct cell lineage programs at direct risk of transformation by ZFTA-RELA. To this end, we performed combined single cell ATAC and RNA-seq analysis (scMultiome) of the developing mouse forebrain as compared to ZR-driven mouse and human EPN. We demonstrate that specific developmental lineage programs present in radial glial cells and regulated by Plagl family transcription factors are at risk of neoplastic transformation. Binding of this chromatin network by ZFTA-RELA or other PLAGL family motif targeting fusion proteins leads to persistent chromatin accessibility at oncogenic loci and oncogene expression. Cross-species analysis of mouse and human EPN reveals significant cell type heterogeneity mirroring incomplete neurogenic and gliogenic differentiation, with a small percentage of cycling intermediate progenitor-like cells that establish a putative tumor cell hierarchy. In vivo lineage tracing studies reveal single neoplastic clones that aggressively dominate tumor growth and establish the entire EPN cellular hierarchy. These findings unravel developmental epigenomic states critical for fusion oncoprotein driven transformation and elucidate how these states continue to shape tumor progression. HIGHLIGHTS 1. Specific chromatin modules accessible during brain development render distinct cell lineage programs at risk of transformation by pediatric fusion oncoproteins. 2. Cross-species single cell ATAC and RNA (scMultiome) of mouse and human ependymoma (EPN) reveals diverse patterns of lineage differentiation programs that restrain oncogenic transformation. 3. Early intermediate progenitor-like EPN cells establish a tumor cell hierarchy that mirrors neural differentiation programs. 4. ZFTA-RELA transformation is compatible with distinct developmental epigenetic states requiring precise ‘goldilocks’ levels of fusion oncoprotein expression. 5. Dominant tumor clones establish the entire EPN cellular hierarchy that reflects normal gliogenic and neurogenic differentiation programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa187c646b1e22ee192aeb0a02cb57b2b3624e15" target='_blank'>
              Dominant Malignant Clones Leverage Lineage Restricted Epigenomic Programs to Drive Ependymoma Development
              </a>
            </td>
          <td>
            Alisha Kardian, Hua Sun, Siri Ippagunta, Nic Laboe, Hsiao-Chi Chen, Erik Emanus, S. Varadharajan, T. Zheng, Blake Holcomb, Patrick Connelly, Jon P. Connelly, Michael Wang, Kimberley Lowe, Shondra M. Pruett-Miller, K. Bertrand, Benjamin Deneen, S. Mack
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Oncogenic fusions of EML4 and ALK occur in ∼5% of lung adenocarcinomas. More than 15 EML4-ALK variants with distinct breakpoints within EML4 have been identified, but the functional differences between these variants remain poorly understood. Here we use CRISPR/Cas9 somatic genome editing to generate autochthonous mouse models of the two most common EML4-ALK variants, V1 and V3, and show that V3 is more oncogenic than V1. By integrating these models with multiplexed genome editing, we quantify the effects of 29 putative tumor suppressor genes on V1- and V3-driven lung cancer growth in vivo and show that many tumor suppressor genes have dramatically variant-specific effects on tumorigenesis. Analysis of a novel dataset representing the largest human EML4-ALK lung cancer cohort to date identified alterations in the genomic landscape depending on the EML4-ALK variant. These findings demonstrate functional heterogeneity among EML4-ALK variants, suggesting that EML4-ALK variants behave more like distinct oncogenes than a uniform entity. More broadly, these findings highlight the dramatic impact of oncogenic fusions partner proteins on tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f320bc764c0ea759a77487f2f2744f6c0fd41b3d" target='_blank'>
              EML4-ALK variant-specific genetic interactions shape lung tumorigenesis
              </a>
            </td>
          <td>
            Alberto Diaz-Jimenez, Emily G. Shuldiner, Kálmán Somogyi, Oscar Gonzalez, Filiz Akkas, Christopher W Murray, Laura Andrejka, Min K. Tsai, B. Brors, S. Sivakumar, Saumya D. Sisoudiya, Ethan S. Sokol, D. Petrov, M. Winslow, R. Sotillo
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>82</td>
        </tr>

        <tr id="The family of LINE1 transposable elements underwent a massive expansion in mammalian genomes. While traditionally viewed as a mutagenic selfish element, recent studies point to roles for LINE1 in early mouse development, T cell quiescence and neurogenesis. Here we show that human LINE1 RNA is essential for self-renewal and identity of human embryonic stem cells (hESCs). Silencing of LINE1 using either antisense oligonucleotides or CRISPR interference in naïve hESCs leads to a strong induction of 8C-like cells (8CLCs). We found that genes derepressed upon LINE1 KD are not uniformly distributed across the genome, with an enrichment for chromosome 19, which includes key markers of the 8C state such as TPRX1. Silencing of TPRX1, but not other putative 8C regulators p53 or H3.XY, suppresses the induction of the 8C program in LINE1 KD hESCs. We found that LINE1 RNA is preferentially localized to the lamina and periphery of the nucleolus in hESCs. Sequencing of Lamina-Associated Domains (LADs) and Nucleolus-Associated Domains (NADs) reveals a preferential association of chromosome 19 with NADs in hESCs. However, 8CLCs have a distinct nucleolar morphology and a lower association of chromosome 19 and TPRX1 loci with the nucleolus relative to naïve and primed hESCs, suggesting a role for nucleolar dynamics in the 8CLC-hESC transition. In agreement, LINE1 KD leads to disruption of nucleolar architecture with signs of nucleolar stress. Independent perturbations of the nucleolus induce the 8C program in hESCs. Genes induced by LINE1 KD are enriched for targets of Polycomb Repressive Complex (PRC2), and inhibition of PRC2 leads to a strong induction of 8C genes. Our results indicate that LINE1 coordinates nuclear compartmentalization and chromatin-mediated gene repression to prevent developmental reversion of hESCs. Highlights Knockdown of LINE1 induces TPRX1-dependent emergence of 8C-like cells in hESCs. Genes de-repressed upon LINE1 KD are enriched for Chr 19 and PRC2 targets. 8CLCs display dissociation of Chromosome 19 from the nucleolus. Disruption of the nucleolus or inhibition of PRC2 strongly induce the 8C program.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34781a9454759e1cbbdf47b8d98d4f390cccbdd4" target='_blank'>
              LINE1 promotes nuclear compartmentalization to repress the 8-cell state in embryonic stem cells
              </a>
            </td>
          <td>
            Juan Zhang, Lamisa Ataei, Liang Wu, Kirti Mittal, Linh Huynh, Shahil Sarajideen, Abdul Mazid, David P. Cook, D. Trcka, Kevin Tse, Jeffrey L. Wrana, Michael M. Hoffman, Miguel A Esteban, M. Ramalho-Santos
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e11890b80227e402b42551347179de59480a8fc" target='_blank'>
              HMGA1 orchestrates chromatin compartmentalization and sequesters genes into 3D networks coordinating senescence heterogeneity
              </a>
            </td>
          <td>
            Ioana Olan, Masami Ando-Kuri, Aled J. Parry, Testuya Handa, S. Schoenfelder, Peter Fraser, Yasuyuki Ohkawa, Hiroshi Kimura, M. Narita, Masashi Narita
          </td>
          <td>2024-08-12</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Non-allelic copies of the two major families of repeat elements in the human genome, Alu and L1, recombine somatically at high frequency. Tissue-specific recombination profiles are dynamic in cell differentiation and are altered in neurodegeneration, suggesting that somatic recombination of repeat elements can contribute to functional heterogeneity of cells in health and disease. The study of these genomic variants, however, presents several technical challenges related to their extremely low copy number and their sequence content. Here, we address key issues regarding detecting and annotating structural variants derived from recombining repeat elements in NGS data. We show that PCR introduces significant changes of recombination profiles in sequencing libraries and that recombination profiles are affected by the choice of sequencing platform. We refine previous estimates of recombination in single cells by analyzing recombination profiles in PCR-free HG002 datasets sequenced by Oxford Nanopore Technologies and PacBio sequencers while describing several platform-specific differences. We additionally provide evidence that recombination events annotated in state-of-the-art single-cell HG002 whole-genome sequencing datasets are likely molecular artifacts generated by PCR. By exploring the limits of current technologies, this work establishes essential requirements for future developments to enhance the reliability of detecting somatic recombination of repeat elements in genomic datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc704a41be11711dd33510c7d91088b58be84182" target='_blank'>
              Challenges in Detecting Somatic Recombination of Repeat Elements: Insights from Short and Long Read Datasets
              </a>
            </td>
          <td>
            G. Pascarella, Martin Frith, P. Carninci
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="To comprehensively unravel the temporal relationship between initiating and driver events and its impact on clinical outcomes, we analyzed 421 whole-genome sequencing profiles from 382 patients. Using clock-like mutational signatures, we estimated a time lag of 2-4 decades between initiating events and diagnosis. In patients with hyperdiploidy, we demonstrate that trisomies of odd-numbered chromosomes can be acquired simultaneously with other chromosomal gains, such as 1q gain. We provide evidence that hyperdiploidy is acquired after canonical IGH translocation when both events are present. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (>1 extra-copies), but faring worse than those with late 1q gain. This underscores that the prognostic impact of 1q gain/amp depends more on the timing of acquisition than on the number of extra copies gained. Overall, this study contributes to a better understanding of the life history of MM and may have prognostic implications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f98e16f03f4610ad7609f6b41fac703153b2f8eb" target='_blank'>
              Temporal Genomic Dynamics Shape Clinical Trajectory in Multiple Myeloma
              </a>
            </td>
          <td>
            Francesco Maura, Marcella Kaddoura, A. Poos, L. Baughn, B. Ziccheddu, Marc-Andrea Bärtsch, Anthony Cirrincione, K. Maclachlan, M. Chojnacka, B. Diamond, M. Papadimitriou, P. Blaney, L. John, Philipp Reichert, S. Huhn, Dylan Gagler, Yanming Zhang, Ahmet Dogan, A. Lesokhin, Faith Davies, Hartmut Goldschmidt, Roland Fenk, Katja C. Weisel, E. Mai, N. Korde, Gareth Morgan, S. Rajkumar, Shaji Kumar, Saad Z. Usmani, O. Landgren, Marc-Steffen Raab, Niels Weinhold
          </td>
          <td>2024-09-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>1</td>
          <td>19</td>
        </tr>

        <tr id="H3K36 methylation is emerging as a key epigenetic modification with strong implications in genetic disease and cancer. However, the mechanisms through which H3K36me impacts the epigenome and asserts its functional consequences are far from understood. Here, we use mouse mesenchymal stem cell lines with successive knockouts of the H3K36 methyltransferases: NSD1, NSD2, SETD2, NSD3, and ASH1L, which result in progressive depletion of H3K36me and its complete absence in quintuple knockout cells, to finely dissect the role of H3K36me2 in shaping the epigenome and transcriptome. We show that H3K36me2, which targets active enhancers, is important for maintaining enhancer activity, and its depletion results in downregulation of enhancer-dependent genes. We demonstrate the roles of H3K36me2/3 in preventing the invasion of gene bodies by the repressive H3K27me modifications. Finally, we observe a previously undescribed relationship between H3K36me and H3K9me3: Following the depletion of H3K36me2, H3K9me3 is redistributed away from large heterochromatic domains and towards euchromatin. This results in a drastic decompartmentalization of the genome, weakening the boundaries between active and inactive compartments, and a catastrophic loss of long-range inter-compartment interactions. By studying cells totally devoid of H3K36 methyltransferase activity, we uncover a broad range of crucial functions of H3K36me in maintaining epigenome integrity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c996ce7c85d697b9c8fb195bf8183e8099242ceb" target='_blank'>
              H3K36 Methylation - a Guardian of Epigenome Integrity
              </a>
            </td>
          <td>
            Reinnier Padilla, Gerry A. Shipman, Cynthia Horth, E. Bareke, Jacek Majewski
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="While numerous technologies for the characterization of potential off-target editing by CRISPR/Cas9 have been described, the development of new technologies and analytical methods for off-target recombination by Large Serine Integrases (LSIs) are required to advance the application of LSIs for therapeutic gene integration. Here we describe a suite of off-target recombination discovery technologies and a hybrid capture validation approach as a comprehensive framework for off-target characterization of LSIs. HIDE-Seq (High-throughput Integrase-mediated DNA Event Sequencing) is a PCR-free unbiased genome-wide biochemical assay capable of discovering sites with LSI-mediated free DNA ends (FDEs) and off-target recombination events. Cryptic-Seq is a PCR-based unbiased genome-wide biochemical or cellular-based assay that is more sensitive than HIDE-Seq but is limited to the discovery of sites with off-target recombination. HIDE-Seq and Cryptic-Seq discovered 38 and 44,311 potential off-target sites respectively. 2,455 sites were prioritized for validation by hybrid capture NGS in LSI-edited K562 cells and off-target integration was detected at 52 of the sites. We benchmarked the sensitivity of our LSI off-target characterization framework against unbiased whole genome sequencing (WGS) on LSI-edited samples, and off-target integration was detected at 5 sites with an average genome coverage of 40x. This reflects a greater than 10-fold increase in sensitivity for off-target detection compared to WGS, however only 4 of the 5 sites detected by WGS were also validated by hybrid capture NGS. The dissemination of these technologies will help advance the application of LSIs in therapeutic genome editing by establishing methods and benchmarks for the sensitivity of off-target detection.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d296a2b7012ef9e1a6d1a1f622c6127d7920625" target='_blank'>
              Large Serine Integrase Off-target Discovery and Validation for Therapeutic Genome Editing
              </a>
            </td>
          <td>
            Dane Z. Hazelbaker, Japan B. Mehta, Connor McGinnis, Didac Santesmasses, Anne M. Bara, Xiaoyu Liang, Thomas Biondi, Tim J. Fennell, Nils Homer, Brett Estes, Jenny Xie, Davood Norouzi, Kaivalya Molugu, Ravindra Amunugama, Chong Luo, Parth Amin, Xiarong Shi, Jesse Cochrane, Sandeep Kumar, Jie Wang, Matthew H. Bakalar, Jonathan D. Finn, Daniel J. O’Connell
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a8902ed0a9d947db27c8f559e3db010a5b107b0" target='_blank'>
              Clonal evolution of the 3D chromatin landscape in patients with relapsed pediatric B-cell acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            Sonali Narang, Yohana Ghebrechristos, Nikki A. Evensen, Nina Murrell, Sylwia Jasinski, Talia H Ostrow, D. Teachey, E. Raetz, Timothee Lionnet, Matthew T. Witkowski, I. Aifantis, A. Tsirigos, William L Carroll
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="Background Approximately half of all high-grade serous ovarian carcinomas (HGSCs) have a therapeutically targetable defect in the homologous recombination (HR) DNA repair mechanism. While there are genomic and transcriptomic methods, developed for other cancer types, to identify HR deficient (HRD) samples, there are no gene expression-based tools to predict HR repair status in HGSC specifically. We have built the first HGSC-specific model to predict HR repair status using gene expression. Methods We separated The Cancer Genome Atlas (TCGA) cohort of HGSCs (n = 361) into training (n = 288) and testing (n = 73) sets and labelled each case as HRD or HR proficient (HRP) based on the clinical standard for classification, being a score of HRD genomic damage. Using the training set, we performed differential gene expression analysis between HRD and HRP cases. The 2604 significantly differentially expressed genes were then used to tune and train a penalised logistic regression model. Results IdentifiHR is an elastic net penalised logistic regression model that uses the expression of 209 genes to predict HR status in HGSC. These genes capture known regions of HR-specific copy number alteration, which impact gene expression levels, and preserve the genomic damage signal. IdentifiHR has an accuracy of 85% in the TCGA test set and of 91% in an independent cohort of 99 samples, collected from primary tumours before (n = 74/99) and after autopsy (n = 6/99), in addition to ascites (n = 12/99) and normal fallopian tube samples (n = 7/99). Further, IdentifiHR is 84% accurate in pseudobulked single-cell HGSC sequencing from 37 patients and outperforms existing gene expression-based methods to predict HR status, being BRCAness, MutliscaleHRD and expHRD. Conclusions IdentifiHR is an accurate model to predict HR status in HGSC using gene expression alone, that is available as an R package from https://github.com/DavidsonGroup/IdentifiHR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03bb043799881c3a8ed9a4b69ffa433b94653148" target='_blank'>
              IdentifiHR: predicting homologous recombination deficiency in high-grade serous ovarian carcinoma using gene expression
              </a>
            </td>
          <td>
            Ashley L. Weir, Samuel C. Lee, Mengbo Li, Chin Wee Tan, S. Ramus, Nadia M. Davidson
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="The dynamic three-dimensional (3D) organization of the human genome (the “4D Nucleome”) is closely linked to genome function. Here, we integrate a wide variety of genomic data generated by the 4D Nucleome Project to provide a detailed view of human 3D genome organization in widely used embryonic stem cells (H1-hESCs) and immortalized fibroblasts (HFFc6). We provide extensive benchmarking of 3D genome mapping assays and integrate these diverse datasets to annotate spatial genomic features across scales. The data reveal a rich complexity of chromatin domains and their sub-nuclear positions, and over one hundred thousand structural loops and promoter-enhancer interactions. We developed 3D models of population-based and individual cell-to-cell variation in genome structure, establishing connections between chromosome folding, nuclear organization, chromatin looping, gene transcription, and DNA replication. We demonstrate the use of computational methods to predict genome folding from DNA sequence, uncovering potential effects of genetic variants on genome structure and function. Together, this comprehensive analysis contributes insights into human genome organization and enhances our understanding of connections between the regulation of genome function and 3D genome organization in general.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9b20ae64e7538eb66073395dd2ed80cb8aed036" target='_blank'>
              An integrated view of the structure and function of the human 4D nucleome
              </a>
            </td>
          <td>
            J. Dekker, B. A. Oksuz, Yang Zhang, Ye Wang, Miriam K. Minsk, Shuzhen Kuang, Liyan Yang, J. Gibcus, Nils Krietenstein, Oliver J. Rando, Jie Xu, D. Janssens, S. Henikoff, A. Kukalev, Andréa Willemin, Warren Winick-Ng, Rieke Kempfer, Ana Pombo, Miao Yu, Pradeep Kumar, Liguo Zhang, Andrew S. Belmont, Takayo Sasaki, Tom van Schaik, Laura Brueckner, Daan Peric-Hupkes, B. van Steensel, Ping Wang, Haoxi Chai, Minji Kim, Yijun Ruan, Ran Zhang, Sofia A. Quinodoz, Prashant Bhat, Mitchell Guttman, Wenxin Zhao, Shu Chien, Yuan Liu, Sergey V. Venev, Dariusz Plewczyński, Ibai Irastorza Azcarate, Dominik Szabó, Christoph J. Thieme, Teresa Szczepińska, Mateusz Chiliński, Kaustav Sengupta, Mattia Conte, Andrea Esposito, Alex Abraham, Ruochi Zhang, Yuchuan Wang, Xingzhao Wen, Qiuyang Wu, Yang Yang, Jie Liu, L. Boninsegna, Aslı Yıldırım, Yuxiang Zhan, A. Chiariello, Simona Bianco, Lindsay Lee, Ming Hu, Yun Li, R. J. Barnett, Ashley L. Cook, Daniel J. Emerson, Claire Marchal, Peiyao Zhao, Peter Park, Burak H. Alver, Andrew J. Schroeder, Rahi Navelkar, Clara Bakker, William Ronchetti, Shannon R. Ehmsen, Alexander D. Veit, Nils Gehlenborg, Ting Wang, Daofeng Li, Xiaotao Wang, Mario Nicodemi, Bing Ren, Sheng Zhong, Jennifer E. Phillips-Cremins, David M. Gilbert, Katherine S. Pollard, Frank Alber, Jian Ma, W. S. Noble, Feng Yue
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma A. Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="Chromosome instability (CIN) is frequently observed in many tumors. The breakage-fusion-bridge (BFB) cycle has been proposed to be one of the main drivers of CIN during tumorigenesis and tumor evolution. However, the detailed mechanism for the individual steps of the BFB cycle warrants further investigation. Here, we demonstrate that a nuclease-dead Cas9 (dCas9) coupled with a telomere-specific single-guide RNA (sgTelo) can be used to model the BFB cycle. First, we show that targeting dCas9 to telomeres using sgTelo impedes DNA replication at telomeres and induces a pronounced increase of replication stress and DNA damage. Using Single-Molecule Telomere Assay via Optical Mapping (SMTA-OM), we investigate the genome-wide features of telomeres in the dCas9/sgTelo cells and observe a dramatic increase of chromosome end fusions, including fusion/ITS+ and fusion/ITS-. Consistently, we also observe an increase in the formation of dicentric chromosomes, anaphase bridges, and intercellular telomeric chromosome bridges (ITCBs). Utilizing the dCas9/sgTelo system, we uncover many interesting molecular and structural features of the ITCB and demonstrate that multiple DNA repair pathways are implicated in the formation of ITCBs. Our studies shed new light on the molecular mechanisms of the BFB cycle, which will advance our understanding of tumorigenesis, tumor evolution, and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de961ad81a066749a4fac0d44c89ba341c356870" target='_blank'>
              Elucidation of the molecular mechanism of the breakage-fusion-bridge (BFB) cycle using a CRISPR-dCas9 cellular model.
              </a>
            </td>
          <td>
            Manrose Singh, Kaitlin Raseley, Alexis M. Perez, Danny MacKenzie, Settapong T. Kosiyatrakul, Sanket Desai, Noelle Batista, Navjot Guru, Katherine K. Loomba, Heba Z Abid, Yilin Wang, Lars Udo-Bellner, Randy F. Stout, C. Schildkraut, Ming Xiao, Dong Zhang
          </td>
          <td>2024-08-28</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Human endogenous retroviruses (HERVs) compose approximately 8% of the human genome, co-opted into the dynamic regulatory network of cellular potency in early embryonic development. In recent studies, resurgent HERVs’ transcriptional activity has been frequently observed in many types of human cancers, suggesting their potential functions in the occurrence and progression of malignancy. However, a web resource dedicated to querying the relationship between activation of HERVs and cancer development is lacking. Here, we have constructed a database to explore the sequence information, expression profiles, survival prognosis, and genetic interactions of HERVs in diverse cancer types. Our database currently incorporates RNA sequencing (RNA-seq) data of 580 HERVs across 16246 samples, comprising 151 early embryonic data from the Gene Expression Omnibus (GEO) database, 8051 human adult tissues’ data from the Genotype-Tissue Expression (GTEx) project, 932 cancer cell lines’ data from the Cancer Cell Line Encyclopedia (CCLE) project, 6478 tumoral and 634 normal tissue samples’ data from The Cancer Genome Atlas (TCGA) project. The primary goal is to provide an easily accessible and user-friendly database for professionals in the fields of bioinformatics, pathology, pharmacology, and related areas, enabling them to efficiently screen the activity of HERVs of interest in normal and cancerous tissues and evaluate the clinical relevance. The ERVcancer database is available at http://kyuanlab.com/ervcancer/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a360157cc339673766d7e394421cf5342d19b04" target='_blank'>
              ERVcancer: a web resource designed for querying activation of human endogenous retroviruses across major cancer types
              </a>
            </td>
          <td>
            Xiaoyun Lei, Song Mao, Yinshuang Li, Shi Huang, Jinchen Li, Wei Du, Chunmei Kuang, Kai Yuan
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="It has been observed that men have a shorter lifespan and higher incidence of certain cancers than women. We postulate here that this phenomenon may be attributed to the “toxic Y chromosome”, owing to the expression of normally silent HSATII repeats in cancer, abundantly present on chromosome Y. Investigating the contribution of all repetitive elements to chromatin interactions using a genome-wide HiC set derived from a colon cancer cell line (HCT116), we found several anomalies regarding satellite HSATII elements when compared to other genomic repeat types: (i) HSATII repeats tend to form HiC pairs mostly with their own kind, and depleted in most heterologous repeat pairs, whereas other types of repeats readily form heterologous HiC pairs; (ii) when treated with 5aza, a chemotherapeutic agent, the number of long-distance HSATII-overlapping HiC pairs significantly decreased when compared to the untreated samples; (iii) and most importantly, in the treated samples interchromosomal HiC read pairs involving a chromosome Y-located repeat and an HSATII repeat on either end almost completely disappear. (iv) We also found that HSATII repeats lack overlap with other types of repeats. To account for these anomalies we queried non-B DNA structure forming such as G-quadruplexes and transcription-related triplex forming. We found that Y chromosome-coded HSATII repeats form significantly stronger triplexes than HSATII repeats on other chromosomes, their strength significantly correlating with the number of HiC reads they overlap with. We surmise that the above phenomena may cause the higher incidence of cancer in men and may lead to the disappearance of the Y chromosome observed in cancerous cells in men.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53990a3a3a681b314979b859a2d2941c439f782" target='_blank'>
              Y chromosome-coded HSATII repeats may contribute to higher incidence of cancer in men
              </a>
            </td>
          <td>
            Hedi Hegyi, Matej Lexa
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Diverse sets of complete human genomes are required to construct a pangenome reference and to understand the extent of complex structural variation. Here, we sequence 65 diverse human genomes and build 130 haplotype-resolved assemblies (130 Mbp median continuity), closing 92% of all previous assembly gaps1,2 and reaching telomere-to-telomere (T2T) status for 39% of the chromosomes. We highlight complete sequence continuity of complex loci, including the major histocompatibility complex (MHC), SMN1/SMN2, NBPF8, and AMY1/AMY2, and fully resolve 1,852 complex structural variants (SVs). In addition, we completely assemble and validate 1,246 human centromeres. We find up to 30-fold variation in α-satellite high-order repeat (HOR) array length and characterize the pattern of mobile element insertions into α-satellite HOR arrays. While most centromeres predict a single site of kinetochore attachment, epigenetic analysis suggests the presence of two hypomethylated regions for 7% of centromeres. Combining our data with the draft pangenome reference1 significantly enhances genotyping accuracy from short-read data, enabling whole-genome inference3 to a median quality value (QV) of 45. Using this approach, 26,115 SVs per sample are detected, substantially increasing the number of SVs now amenable to downstream disease association studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6654cdcb0238112d367eba4f362c39559b46653" target='_blank'>
              Complex genetic variation in nearly complete human genomes
              </a>
            </td>
          <td>
            Glennis A. Logsdon, Peter Ebert, P. Audano, Mark Loftus, David Porubsky, Jana Ebler, F. Yilmaz, P. Hallast, Timofey Prodanov, DongAhn Yoo, Carolyn A. Paisie, William T. Harvey, Xuefang Zhao, Gianni V. Martino, Mir Henglin, Katherine M. Munson, Keon Rabbani, Chen-Shan Chin, Bida Gu, Hufsah Ashraf, Olanrewaju Austine-Orimoloye, Parithi Balachandran, M. Bonder, Haoyu Cheng, Zechen Chong, J. Crabtree, Mark B. Gerstein, L. Guethlein, Patrick Hasenfeld, G. Hickey, Kendra Hoekzema, Sarah E. Hunt, Matthew Jensen, Yunzhe Jiang, S. Koren, Youngjun Kwon, Chong Li, Heng Li, Jiaqi Li, Paul J. Norman, Keisuke K. Oshima, B. Paten, A. Phillippy, Nicholas R Pollock, Tobias Rausch, M. Rautiainen, Stephan Scholz, Yuwei Song, Arda Söylev, Arvis Sulovari, Likhitha Surapaneni, Vasiliki Tsapalou, Weichen Zhou, Ying Zhou, Qihui Zhu, Michael C Zody, Ryan E. Mills, S. Devine, Xinghua Shi, M. Talkowski, Mark J. P. Chaisson, Alexander Dilthey, Miriam K. Konkel, J. Korbel, Charles Lee, Christine R. Beck, Evan E. Eichler, T. Marschall
          </td>
          <td>2024-09-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>96</td>
        </tr>

        <tr id="The recent paradoxical dual citizenships of Runt-related transcription factor 3 (Runx3) in tumorigenesis remains poorly characterized. Here, we report the oncogenic capacity of Runx3 as chromatin modulator in metastatic gastric cancer model. Runx3 exists as homodimer and binds cooperatively to modified nucleosomes. Additionally, we detected a synergistic functional enhancement of octamer transfer, nucleosome sliding and stochiometric integrity of SWI/SNF by Runx3. We found that Runx3 depletion increased nucleosomes occupancy and promoted chromatin silencing by heterochromatin condensation and HP1 oligomerization. ATAC-seq analysis revealed differential accessibility per chromosome due to Runx3 null expression with dysregulation of multiple inflammatory response and DNA repair pathways. Mechanistically, these modulations resulted in aberrant DNA damage repair response, which is rescued by RUNX3 overexpression. These findings reveal a new paradigm in Runx3 biology as dynamic chromatin regulatory element vital for the maintenances of cancerous phenotype. Significance To the best of our knowledge, the present study is the first to explore the role of Runx3 as homodimeic chromatin binding factor and establishes its oncogenic-function as modulator of heterochromatin de-condensation and SWI/SNF chromatin remodeling activities. These emerging features of Runx3 at the epigenetic level imply a promising direction to screen for anti-Runx3 epigenetic drugs “Epi-drugs” in search of novel gastric cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65693fd8ec877e2185ca7654a9454ed2dc3bd82f" target='_blank'>
              Runx3 Acts as Homodimeric Chromatin Binding Factor Regulating Heterochromatin-Mediated Cancerous Phenotype
              </a>
            </td>
          <td>
            Salma Awad Merghani, Isabelle Bonne, Aik Yong Sim, Gehan El-Enain
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d2d63fe0e1a04ed83bd0b4c69b42c0d270765edd" target='_blank'>
              Human cytomegalovirus harnesses host L1 retrotransposon for efficient replication
              </a>
            </td>
          <td>
            Sung-Yeon Hwang, Hyewon Kim, Danielle Denisko, Boxun Zhao, Dohoon Lee, Jiseok Jeong, Jinuk Kim, Kiwon Park, Junhyun Park, D. Jeong, Sehong Park, Hee-Jung Choi, Sun Kim, E. A. Lee, Kwangseog Ahn
          </td>
          <td>2024-09-02</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="MYC deregulation occurs in the majority of multiple myeloma (MM) cases and is associated with progression and worse prognosis. Enhanced MYC expression occurs in about 70% of MM patients, but it is known to be driven by translocation or amplification events in only ~40% of myelomas. Here, we used CRISPR interference (CRISPRi) to uncover an epigenetic mechanism of MYC regulation whereby increased accessibility of a plasma cell-type specific enhancer leads to increased MYC expression. This native enhancer activity was not associated with enhancer hijacking events but led to specific binding of c-MAF, IRF4, and SPIB transcription factors that activated MYC expression in the absence of known genetic aberrations. In addition, focal amplification was another mechanism of activation of this enhancer in approximately 3.4% of MM patients. Together, these findings define an epigenetic mechanism of MYC deregulation in MM beyond known translocations or amplifications and point to the importance of non-coding regulatory elements and their associated transcription factor networks as drivers of MM progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89586e6b56717e928b88a8526a759effc353dc88" target='_blank'>
              Selective Enhancer Gain of Function Deregulates MYC Expression in Multiple Myeloma.
              </a>
            </td>
          <td>
            M. Rahmat, Kendell Clement, Jean-Baptiste Alberge, Romanos Sklavenitis-Pistofidis, R. Kodgule, C. Fulco, Daniel Heilpern-Mallory, Katarina Nilsson, David Dorfman, Jesse M Engreitz, G. Getz, Luca Pinello, R. Ryan, I. M. Ghobrial
          </td>
          <td>2024-09-23</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Identification of somatic driver mutations in the noncoding genome remains challenging. To comprehensively characterize noncoding driver mutations for pancreatic ductal adenocarcinoma (PDAC), we first created genome-scale maps of accessible chromatin regions (ACRs) and histone modification marks (HMMs) in pancreatic cell lines and purified pancreatic acinar and duct cells. Integration with whole-genome mutation calls from 506 PDACs revealed 314 ACRs/HMMs significantly enriched with 3,614 noncoding somatic mutations (NCSMs). Functional assessment using massively parallel reporter assays (MPRA) identified 178 NCSMs impacting reporter activity (19.45% of those tested). Focused luciferase validation confirmed negative effects on gene regulatory activity for NCSMs near CDKN2A and ZFP36L2. For the latter, CRISPR interference (CRISPRi) further identified ZFP36L2 as a target gene (16.0 - 24.0% reduced expression, P = 0.023 - 0.0047) with disrupted KLF9 binding likely mediating the effect. Our integrative approach provides a catalog of potentially functional noncoding driver mutations and nominates ZFP36L2 as a PDAC driver gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db992992dda460b3aa1dd358fc40078a768ec148" target='_blank'>
              Large-scale multi-omic analysis identifies noncoding somatic driver mutations and nominates ZFP36L2 as a driver gene for pancreatic ductal adenocarcinoma
              </a>
            </td>
          <td>
            J. Zhong, A. O'Brien, Minal Patel, D. Eiser, Michael Mobaraki, I. Collins, Li Wang, Konnie Guo, Thucnhi Truongvo, Ashley A. Jermusyk, Maura J. O’Neill, Courtney D. Dill, Andrew D. Wells, M. Leonard, J. Pippin, Struan F.A, Grant, Tongwu Zhang, T. Andresson, K. E. Connelly, Jianxin Shi, H. E. Arda, J. Hoskins, L. Amundadottir, Aidan
          </td>
          <td>2024-09-24</td>
          <td>None</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Hepatocellular carcinoma (HCC) exhibits widespread epigenetic alterations, yet their impact on cis-regulatory elements (CREs) and retrotransposons remains poorly understood. Here, we present an integrated epigenomic and transcriptomic analysis of HCC tumors and matched tumor-adjacent normal tissues. We identified extensive DNA hypomethylation coupled with changes in histone modifications at partially methylated domains, CREs, and retrotransposons. These epigenetic aberrations were associated with dysregulated expression of genes involved in cell cycle regulation, immune response, and extracellular matrix organization. Notably, our findings revealed a novel mechanism for the transcriptional dysregulation of GPC3, a key HCC biomarker and immunotherapeutic target. We observed that GPC3 upregulation is driven by both the reactivation of a fetal liver super enhancer and hypomethylation of GPC3-associated CpG islands. Moreover, we found that DNA hypomethylation-driven aberrant expression of retrotransposons carries prognostic significance in HCC. Patients with high expression of a long non-coding RNA driven by a HERVE-int element exhibited more aggressive tumors, poorer clinical outcomes, and molecular features associated with favorable immunotherapy response. Together, our study provides a comprehensive resource for understanding the role of epigenetic dysregulation in HCC and identifies retrotransposon-associated transcripts as potential biomarkers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce7deec20d100e57e2cf628c181e276b8e32f748" target='_blank'>
              Epigenomic analysis of hepatocellular carcinoma reveals aberrant cis-regulatory changes and dysregulated retrotransposons with prognostic potentials
              </a>
            </td>
          <td>
            Clooney C. Y. Cheng, Ming Fung Cheung, Ah Young Lee, Qiong Wu, S. Chow, Julie Y. J. Ang, Ignacio Riquelme Medina, Grace Lo, Haoran Wu, Weiqin Yang, Paul B. S. Lai, K. Yip, Alfred S. Cheng, Danny Leung
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Diffuse large B cell lymphoma, not otherwise specified (DLBCL, NOS) is the most common type of non-Hodgkin lymphoma (NHL). Significant efforts have been focused on utilizing advanced genomic technologies to further subclassify DLBCL, NOS into clinically relevant subtypes. These efforts have led to the implementation of novel algorithms to support optimal risk-oriented therapy and improvement in the overall survival of DLBCL patients. The pathogenesis of DLBCL at the molecular level indicates copy number variation (CNV) as one of the major forms of genetic alterations in the somatic mutational landscape. Random deregulation that results in complex breaks of chromosomes and restructuring of shattered chromosomal segments is called chromothripsis. Gene expression changes influenced by chromothripsis have been reported in cancer and congenital diseases. This chaotic phenomenon results in complex CNV, gene fusions, and amplification and loss of tumor suppressor genes. We present herein a summary of the most clinically relevant genomic aberrations, with particular focus on copy number aberrations in a case that highlights DLBCL, NOS arising from relapsed Hodgkin lymphoma. The focus of our study was to understand the relationship between the clinical, morphological, and genomic abnormalities in DLBCL, NOS through multiple techniques for therapeutic considerations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e29495d2020d233538184aa5ca4d900590a2a9ff" target='_blank'>
              Deregulation and Shattering of Chromosomal Segments Containing Multiple Oncogenic Targets in the Pathogenesis of Diffuse Large B Cell Lymphoma, Not Otherwise Specified (DLBCL, NOS)
              </a>
            </td>
          <td>
            A. Yenamandra, Rebecca B. Smith, Adam C. Seegmiller, Brianna N. Smith, Debra L. Friedman, C. M. Smith
          </td>
          <td>2024-09-18</td>
          <td>DNA</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/17c120e2af47610364790b472bc22133f0b2b21b" target='_blank'>
              Complex genomic rearrangements of the Y chromosome in a premature infant
              </a>
            </td>
          <td>
            Stephanie A. Balow, Alyxis Coyan, Nicki Smith, Bianca E Russell, Danielle Monteil, Robert J. Hopkin, Teresa A. Smolarek
          </td>
          <td>2024-08-26</td>
          <td>Molecular Cytogenetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Break-induced replication (BIR) is mutagenic, and thus its use requires tight regulation, yet the underlying mechanisms remain elusive. Here we uncover an important role of 53BP1 in suppressing BIR after end resection at double strand breaks (DSBs), distinct from its end protection activity, providing insight into the mechanisms governing BIR regulation and DSB repair pathway selection. We demonstrate that loss of 53BP1 induces BIR-like hyperrecombination, in a manner dependent on Polα-primase-mediated end fill-in DNA synthesis on single-stranded DNA (ssDNA) overhangs at DSBs, leading to PCNA ubiquitination and PIF1 recruitment to activate BIR. On broken replication forks, where BIR is required for repairing single-ended DSBs (seDSBs), SMARCAD1 displaces 53BP1 to facilitate the localization of ubiquitinated PCNA and PIF1 to DSBs for BIR activation. Hyper BIR associated with 53BP1 deficiency manifests template switching and large deletions, underscoring another aspect of 53BP1 in suppressing genome instability. The synthetic lethal interaction between the 53BP1 and BIR pathways provides opportunities for targeted cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21b30096c06a55bc35f817e3d1b46b909b538103" target='_blank'>
              53BP1 deficiency leads to hyperrecombination using break-induced replication (BIR)
              </a>
            </td>
          <td>
            Sameer Bikram Shah, Youhang Li, Shibo Li, Qing Hu, Tong Wu, Yanmeng Shi, Tran Nguyen, Isaac Ive, Linda Shi, Hailong Wang, Xiaohua Wu
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1155864d234b3aeabe252449e0eb74460699d3b9" target='_blank'>
              Prediction of the 3D cancer genome from whole-genome sequencing using InfoHiC.
              </a>
            </td>
          <td>
            Yeonghun Lee, Sung-Hye Park, Hyunju Lee
          </td>
          <td>2024-09-25</td>
          <td>Molecular systems biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genome integrity is safeguarded by the DNA damage response (DDR). Controlled transcriptional dampening of genes surrounding DNA double-strand breaks (DSBs) has been shown to facilitate DNA repair. This phenomenon, defined as DSB-induced silencing in cis (DISC), involves the DDR apical kinase ATM and the Polycomb Repressive Complex 1 (PRC1). Conversely, DSBs have also been reported to induce de novo transcription of damaged-induced long non-coding RNAs (dilncRNAs) in a MRE11-RAD50-NBS1 (MNR) complex-dependent manner. MRN also controls the recruitment to DSB of the ribonuclease DROSHA, which together with DICER, stimulates DDR signaling and DNA repair. Here, we reconcile these apparently contrasting observations by showing that dilncRNA, together with DROSHA and DICER, but not GW182-like proteins required for miRNA-mediated gene silencing, controls DISC. Indeed, similarly to ATM, MRN inhibition abolishes DISC while pharmacological enhancement of DICER ribonuclease activity by Enoxacin improves DISC. Importantly, Enoxacin administration restores DISC upon ATM inhibition, demonstrating that DICER promotes DISC independently from ATM. Differently, Enoxacin does not restore DISC upon MRN inhibition, suggesting that DICER acts downstream to dilncRNA biogenesis and DROSHA recruitment. Mechanistically, we show that DROSHA and DICER control the recruitment of the PRC1 component BMI1 at DSBs and the consequent H2A-K119 ubiquitination. Upon DSBs formation, BMI1 and DROSHA interact in an RNA-dependent manner. Indeed, BMI1 associates to dilncRNA and do so in a DROSHA- and DICER-dependent manner. Importantly, inhibition of dilncRNA function by antisense oligonucleotides or Cas13-mediated targeting is sufficient to reduce BMI1 recruitment and DISC at individual loci. We propose that dilncRNAs together with DROSHA and DICER control DISC at genomic DSB by supporting PRC1 recruitment and chromatin ubiquitination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbe46331f4c7eaaba94ede95e2ee9569f12ce8df" target='_blank'>
              DROSHA, DICER and Damage-Induced long ncRNA control BMI1-dependent transcriptional repression at DNA double-strand break
              </a>
            </td>
          <td>
            Francesca Esposito, Ilaria Capozzo, Stefania Modafferi, Ubaldo Gioia, Letizia Manfredi, F. Iannelli, Alessio Colantoni, Adelaide Riccardi, Alessia di Lillo, Sara Tavella, Matteo Cabrini, F. d’Adda di Fagagna, S. Francia
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>37</td>
        </tr>

        <tr id="Polypyrimidine tract binding protein 2 (PTBP2) is an RNA-binding protein that controls alternative splicing in neuronal, muscle, and Sertoli cells. Our study unveils a novel role of PTBP2 in promoting the excessive production of the DNA polymerase kappa (Polκ) by stabilizing its 3’UTR. We observed an association between its increased expression and the upregulation of PTBP2 in clinical samples of Chronic Myeloid Leukemia (CML). Ptbp2 knock-out CML cell lines and patient samples treated with hydroxyurea presented with increased DNA damage, as evidenced by long comet tails and higher levels of the DNA damage marker, γH2AX foci, however overexpression of Polκ in the Ptbp2-KO cells restored normal phenotype. The deregulation of the DNA repair pathway is a defining feature of malignancies and is closely associated with genomic instability. POLK was found to interact with MRE11 of the MRN complex, thereby governing the activation of ATM-CHK2. Cells with elevated levels of Ptbp2 and Polκ demonstrated increased sister chromatid exchange and BrdU incorporation in ex-vivo assays, while multinucleated cells with multipolar spindles were observed in in-vivo assays. Our findings confirm the critical role of the PTBP2-POLK axis in driving genomic instability and bolstering the viability of cells with increased malignancy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5aa3b48c6a74455a5e894d9a4adb977c32b5b29c" target='_blank'>
              DNA polymerase kappa stabilized by PTBP2 interacts with MRE11 and promotes genomic instability in leukemia cells
              </a>
            </td>
          <td>
            Shristi Lama, Bibhudev Barik, Sajitha Is, Tannistha Sarkar, Sayantan Chanda, Monalisa Behera, Subhankar Priyadarshi Behera, S. Biswas, Sonali Mohapatra, G. Biswas, Soumen Chakraborty
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: It is widely accepted that the 3D chromatin organization in human cell nuclei is not random and recent investigations point towards an interactive relation of epigenetic functioning and chromatin (re-)organization. Although chromatin organization seems to be the result of self-organization of the entirety of all molecules available in the cell nucleus, a general question remains open as to what extent chromatin organization might additionally be predetermined by the DNA sequence and, if so, if there are characteristic differences that distinguish typical regions involved in dysfunction-related aberrations from normal ones, since typical DNA breakpoint regions involved in disease-related chromosome aberrations are not randomly distributed along the DNA sequence. Methods: Highly conserved k-mer patterns in intronic and intergenic regions have been reported in eukaryotic genomes. In this article, we search and analyze regions deviating from average spectra (ReDFAS) of k-mer word frequencies in the human genome. This includes all assembled regions, e.g., telomeric, centromeric, genic as well as intergenic regions. Results: A positive correlation between k-mer spectra and 3D contact frequencies, obtained exemplarily from given Hi-C datasets, has been found indicating a relation of ReDFAS to chromatin organization and interactions. We also searched and found correlations of known functional annotations, e.g., genes correlating with ReDFAS. Selected regions known to contain typical breakpoints on chromosomes 9 and 5 that are involved in cancer-related chromosomal aberrations appear to be enriched in ReDFAS. Since transposable elements like ALUs are often assigned as major players in 3D genome organization, we also studied their impact on our examples but could not find a correlation between ALU regions and breakpoints comparable to ReDFAS. Conclusions: Our findings might show that ReDFAS are associated with instable regions of the genome and regions with many chromatin contacts which is in line with current research indicating that chromatin loop anchor points lead to genomic instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d3f4228708cdafa938a5ddafca7f4b52c72767a" target='_blank'>
              Peculiar k-mer Spectra Are Correlated with 3D Contact Frequencies and Breakpoint Regions in the Human Genome
              </a>
            </td>
          <td>
            Wisam Mohammed Hikmat, Aaron Sievers, Michael Hausmann, G. Hildenbrand
          </td>
          <td>2024-09-25</td>
          <td>Genes</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="In cancer cells, ATR signaling is crucial to tolerate the intrinsically high damage levels that normally block replication fork progression. Assembly of TopBP1, a multifunctional scaffolding protein, into condensates is required to amplify ATR kinase activity to the levels needed to coordinate the DNA damage response and manage DNA replication stress. Many ATR inhibitors are tested for cancer treatment in clinical trials, but their overall effectiveness is often compromised by the emergence of resistance and toxicities. In this proof-of-concept study, we propose to disrupt the ATR pathway by targeting TopBP1 condensation. First, we screened a molecule-based library using a previously developed optogenetic approach and identified several TopBP1 condensation inhibitors. Amongst them, AZD2858 disrupted TopBP1 assembly induced by the clinically relevant topoisomerase I inhibitor SN-38, thereby inhibiting the ATR/Chk1 signaling pathway. We found that AZD2858 exerted its effects by disrupting TopBP1 self-interaction and binding to ATR in mammalian cells, and by increasing its chromatin recruitment in cell-free Xenopus laevis egg extracts. Moreover, AZD2858 prevented S-phase checkpoint induction by SN-38, leading to increased DNA damage and apoptosis in a colorectal cancer cell line. Lastly, AZD2858 showed synergistic effect in combination with the FOLFIRI chemotherapy regimen in a spheroid model of colorectal cancer. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8d12d939b21bc8eddc64b81e916d3c7019a271" target='_blank'>
              TopBP1 biomolecular condensates: a new therapeutic target in advanced-stage colorectal cancer
              </a>
            </td>
          <td>
            Laura Morano, Nadia Vezzio-Vié, Adam Aissanou, Tom Egger, Antoine Aze, Solène Fiachetti, Hervé Seitz, Louis-Antoine Milazzo, V. Garambois, Nathalie Bonnefoy, Céline Gongora, A. Constantinou, J. Basbous
          </td>
          <td>2024-09-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Histones are essential for maintaining chromatin structure and function. Histone mutations lead to changes in chromatin compaction, gene expression, and the recruitment of DNA repair proteins to the DNA lesion. These disruptions can impair critical DNA repair pathways, such as homologous recombination and non-homologous end joining, resulting in increased genomic instability, which promotes an environment favorable to tumor development and progression. Understanding these mechanisms underscores the potential of targeting DNA repair pathways in cancers harboring mutated histones, offering novel therapeutic strategies to exploit their inherent genomic instability for better treatment outcomes. Here, we examine how mutations in histone H3 disrupt normal chromatin function and DNA damage repair processes and how these mechanisms can be exploited for therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eae6d859b6a16ea85b27994b62cbff421ebf7a45" target='_blank'>
              Histone H3 mutations and their impact on genome stability maintenance.
              </a>
            </td>
          <td>
            Lucas D Caeiro, Ramiro E Verdun, Lluis Morey
          </td>
          <td>2024-09-09</td>
          <td>Biochemical Society transactions</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Although the role of G-quadruplex (G4) DNA structures has been suggested in chromosomal looping this was not tested directly. Here, to test causal function, an array of G4s, or control sequence that does not form G4s, were inserted within chromatin in cells. In vivo G4 formation of the inserted G4 sequence array, and not the control sequence, was confirmed using G4-selective antibody. Compared to the control insert, we observed a remarkable increase in the number of 3D chromatin looping interactions from the inserted G4 array. This was evident within the immediate topologically associated domain (TAD) and throughout the genome. Locally, recruitment of enhancer histone marks and the transcriptional coactivator p300/Acetylated-p300 increased in the G4-array, but not in the control insertion. Resulting promoter-enhancer interactions and gene activation were clear up to 5 Mb away from the insertion site. Together, these show the causal role of G4s in enhancer function and long-range chromatin interactions. Mechanisms of 3D topology are primarily based on DNA-bound architectural proteins that induce/stabilize long-range interactions. Involvement of the underlying intrinsic DNA sequence/structure in 3D looping shown here therefore throws new light on how long-range chromosomal interactions might be induced or maintained.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c4a14fedf36592373d312c9d23b12df8fd19b77" target='_blank'>
              Artificially inserted strong promoter containing multiple G-quadruplexes induces long-range chromatin modification
              </a>
            </td>
          <td>
            S. Roy, Sulochana Bagri, Soujanya Vinayagamurthy, Avik Sengupta, Claudia Regina Then, Rahul Kumar, Sriram Sridharan, Shantanu Chowdhury
          </td>
          <td>2024-08-19</td>
          <td>eLife</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="A key step of Canonical Nonhomologous End Joining (C-NHEJ) is synapsis of DNA double strand break (DSB) ends for ligation. The DNA-PKcs dimer mediates synapsis in a long-range complex with DSB ends remaining apart, whereas the XLF homodimer can mediate synapsis in both long-range and short-range complexes. Recent structural studies found the PAXX homodimer may also facilitate synapsis in long-range complexes with DNA-PKcs via its interactions with Ku70. Thus, we examined the influence of PAXX in C-NHEJ of chromosomal DSBs, which we compared to another Ku-binding factor, MRI. Using EJ of blunt DSBs with Cas9 reporters as a readout for C-NHEJ, we found that PAXX and/or MRI are dispensable. However, when combined with disruption of DNA-PKcs, particularly with DNA-PKcs kinase inhibition, PAXX becomes important for blunt DSB EJ. In contrast, while DNA-PKcs is also important to suppress short deletion mutations with microhomology, this effect is not magnified with PAXX loss. MRI loss had no effect combined with DNA-PKcs disruption, but becomes important for blunt DSB EJ when combined with disruption of XLF, as is PAXX. Finally, XLF loss causes an increase in larger deletions compared to DNA-PKcs inhibition, which is magnified with combined loss of MRI. Altogether, we suggest that PAXX promotes DSB end synapsis during C-NHEJ in a manner that is partially redundant with DNA-PKcs and XLF, whereas MRI appears to be mainly important in the context of XLF disruption. SIGNIFICANCE STATEMENT Canonical Nonhomologous End Joining (C-NHEJ) is a DNA double strand break repair pathway that can repair diverse types of DNA ends, but is prone to causing genetic mutations. In particular, C-NHEJ can cause insertion or deletion mutations that can cause human disease. Therefore, it is critical to understand the mechanisms of C-NHEJ, including and how various factors within the pathway affect repair outcomes. In this study, we elucidate the role of factors PAXX and MRI during chromosomal EJ and their interplay with the C-NHEJ factors DNA-PKcs and XLF.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa6710460a50993f582d4fe63d4fd0ff8229a07e" target='_blank'>
              Distinct functions of PAXX and MRI during chromosomal end joining
              </a>
            </td>
          <td>
            Metztli Cisneros-Aguirre, Felicia Wednesday Lopezcolorado, Xiaoli Ping, Ruby Chen, Jeremy M. Stark
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc75014daa55260f9205d28f8165ad0ad389192e" target='_blank'>
              Genetic hallmarks and clinical implications of chromothripsis in childhood T-cell acute lymphoblastic leukemia.
              </a>
            </td>
          <td>
            A. Pastorczak, Z. Urbańska, Borys Styka, Karolina Miarka-Walczyk, L. Sędek, K. Wypyszczak, A. Wakulinska, Zuzanna Nowicka, Tomasz Szczepański, Marcin Stańczak, Wojciech Fendler, Jerzy Kowalczyk, W. Młynarski, Monika Lejman
          </td>
          <td>2024-08-27</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1082aba026dea72fee57aef4a45e7856e9efcf8" target='_blank'>
              Topoisomerase-modulated genome-wide DNA supercoiling domains colocalize with nuclear compartments and regulate human gene expression.
              </a>
            </td>
          <td>
            Qian Yao, Linying Zhu, Zhen Shi, Subhadra Banerjee, Chongyi Chen
          </td>
          <td>2024-08-16</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d417bff4628adb0291d6a29056ca242d46664396" target='_blank'>
              Molecular characteristics and oncogenic role of CHD family genes: a pan-cancer analysis based on bioinformatic and biological analysis
              </a>
            </td>
          <td>
            Yujia Lu, Jiebang Jiang, Zhihong He, Zhouzhou Bao, Xin Chen, Jie Cheng
          </td>
          <td>2024-08-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1267dfc8342dfe0bff17f1c6ff48612b497b5563" target='_blank'>
              RNA-mediated double-strand break repair by end-joining mechanisms
              </a>
            </td>
          <td>
            Youngkyu Jeon, Yilin Lu, M. Ferrari, Tejasvi Channagiri, Penghao Xu, Chance Meers, Yiqi Zhang, S. Balachander, Vivian S. Park, Stefania Marsili, Z. Pursell, Nataša Jonoska, F. Storici
          </td>
          <td>2024-09-11</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Transposable elements (TEs), widespread genetic parasites, pose potential threats to the stability of their host genomes. Hence, the interactions observed today between TEs and their host genomes, as well as among the different TE species coexisting in the same host, likely reflect those that did not lead to the extinction of either the host or the TEs. It is not clear to what extent the expression and integration steps of the TE replication cycles are involved in this ‘peaceful’ coexistence. Here, we show that four Drosophila LTR RetroTransposable Elements (LTR-RTEs), although sharing the same overall integration mechanism, preferentially integrate into distinct open chromatin domains of the host germline. Notably, the differential expressions of the gtwin and ZAM LTR-RTEs in ovarian and embryonic somatic tissues, respectively, result in differential integration timings and targeting of accessible chromatin landscapes that differ between early and late embryonic nuclei, highlighting connections between temporal and spatial LTR-RTEs niche partitionings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10ef122580a3d05b44b334371537616b10346674" target='_blank'>
              Temporal and spatial niche partitioning in a retrotransposon community of the Drosophila genome
              </a>
            </td>
          <td>
            Marion Varoqui, Mourdas Mohamed, B. Mugat, Daniel Gourion, Maëlys Lemoine, Alain Pélisson, Charlotte Grimaud, S. Chambeyron
          </td>
          <td>2024-08-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Triple-negative breast cancer (TNBC) is the most therapeutically recalcitrant form of breast cancer, which is due in part to the paucity of targeted therapies. A systematic analysis of regulatory elements that extend beyond protein coding genes could uncover avenues for therapeutic intervention. To this end, we analyzed the regulatory mechanisms of TNBC-specific transcriptional enhancers together with their non-coding enhancer RNA (eRNA) transcripts. The functions of the top 30 eRNA-producing super-enhancers were systematically probed using high-throughput CRISPR-interference assays coupled to RNA-seq that enabled unbiased detection of target genes genome-wide. Generation of high resolution Hi-C chromatin interaction maps enabled annotation of the direct target genes for each super-enhancer, which highlighted their proclivity for genes that portend worse clinical outcomes in TNBC patients. Illustrating the utility of this dataset, deletion of an identified super-enhancer controlling the nearby PODXL gene or specific degradation of its enhancer RNAs led to profound inhibitory effects on target gene expression, cell proliferation, and migration. Furthermore, loss of this super-enhancer suppressed tumor growth and metastasis in TNBC mouse xenograft models. Single-cell RNA-seq and ATAC-seq analyses demonstrated the enhanced activity of this super-enhancer within the malignant cells of TNBC tumor specimens compared to non-malignant cell types. Collectively, this work examines several fundamental questions about how regulatory information encoded into eRNA-producing super-enhancers drives gene expression networks that underlie the biology of triple-negative breast cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4243c6e6d0e13be4b38db7c7847b980032989cf7" target='_blank'>
              CRISPR Screening of Transcribed Super-Enhancers Identifies Drivers of Triple-Negative Breast Cancer Progression.
              </a>
            </td>
          <td>
            Michael W Lewis, Caitlin M King, K. Wiśniewska, Matthew J. Regner, Alisha R. Coffey, Michael R Kelly, R. Méndez-Giráldez, Eric S. Davis, D. Phanstiel, Hector L. Franco
          </td>
          <td>2024-08-26</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Cytosine methylation contributes to the regulation of gene expression and normal hematopoiesis in mammals. It is catalyzed by the family of DNA methyltransferases that include DNMT1, DNMT3A, and DNMT3B. Peripheral T–cell lymphomas (PTCLs) represent aggressive mature T–cell malignancies exhibiting a broad spectrum of clinical features with poor prognosis and inadequately understood molecular pathobiology. To better understand the molecular landscape and identify candidate genes involved in disease maintenance, we profiled DNA methylation and gene expression of PTCLs. We found that the methylation patterns in PTCLs are deregulated and heterogeneous but share 767 hypo- and 567 hypermethylated differentially methylated regions (DMRs) along with 231 genes up- and 91 genes downregulated in all samples, suggesting a potential association with tumor development. We further identified 39 hypomethylated promoters associated with increased gene expression in the majority of PTCLs. This putative oncogenic signature included the TRIP13 (thyroid hormone receptor interactor 13) gene whose genetic and pharmacologic inactivation inhibited the proliferation of T–cell lines by inducing G2-M arrest and apoptosis. Our data thus show that human PTCLs have a significant number of recurrent methylation alterations that may affect the expression of genes critical for proliferation whose targeting might be beneficial in anti-lymphoma treatments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9459abe1347b1ee4432eee2a87abcce083d2c83" target='_blank'>
              Genome-Wide Methylation Profiling of Peripheral T–Cell Lymphomas Identifies TRIP13 as a Critical Driver of Tumor Proliferation and Survival
              </a>
            </td>
          <td>
            P. Nowialis, Julian Tobon, Katarina Lopusna, J. Opavska, Arshee Badar, Duo Chen, Reem Abdelghany, Gene Pozas, Jacob Fingeret, Emma Noel, Alberto Riva, Hiroshi Fujiwara, Alexander Ishov, Rene Opavsky
          </td>
          <td>2024-08-21</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G. Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/505f6d9a24bb0cc1b413b063823fa87cd784ad4b" target='_blank'>
              Extracting regulatory active chromatin footprint from cell-free DNA
              </a>
            </td>
          <td>
            Kevin Lai, Katharine Dilger, Rachael Cunningham, Kathy T Lam, Rhea Boquiren, Khiet Truong, Maggie C. Louie, Richard Rava, Diana Abdueva
          </td>
          <td>2024-09-04</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21a24ea5db444de6c4cb2a6e44207610c8118487" target='_blank'>
              RHNO1: at the crossroads of DNA replication stress, DNA repair, and cancer.
              </a>
            </td>
          <td>
            Niphat Jirapongwattana, S. Bunting, Donald R Ronning, Gargi Ghosal, A. Karpf
          </td>
          <td>2024-08-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Genes on sex chromosomes have higher evolutionary rates than those on autosomes. However, this does not necessarily apply to somatic evolution in cancer. Many dominant mutations have been described in the so-called proto-oncogenes (OGs), while recessive mutations are typically described in tumor-suppressor genes (TSGs). Evidence indicates that mutations in X-chromosome TSGs are more likely to contribute to cancer than those in autosomal TSGs. Here, we formalize this in several dynamic models and predict, as expected, that mutations spread faster in TSGs located on the X chromosome than on autosomes (faster-X effect). Conversely, mutations in OGs spread faster on autosomes than on the X chromosome, but under high selective pressure, this difference is negligible. Published genomic screenings of cancer samples show evidence of the faster-X effect in TSGs. This pattern is observed in both sexes, suggesting that the maintenance of X-chromosome inactivation during cancer progression plays an important role in the evolution of TSGs. Strikingly, the relative mutation incidence in X-linked TSGs among females across individual studies is bimodal, with one group of studies showing a faster-X effect and another group showing similar incidences for X-linked and autosomal TSGs. This differentiation between cancer samples is not associated with the specific type of cancer or the tissue of origin. This may indicate that X-chromosome inactivation plays a differential role in the involvement of X-linked TSGs across individual cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e60393ec97b0f4b9cf086f3cab7030b54b21b01" target='_blank'>
              Somatic evolution of cancer genes in sex chromosomes
              </a>
            </td>
          <td>
            Amarachi Akachukwu, Jasmine S. Ratcliff, Antonio Marco
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology enables the precise analysis of individual cell transcripts with high sensitivity and throughput. When integrated with multiomics technologies, scRNA-seq significantly enhances the understanding of cellular diversity, particularly within the tumor microenvironment. Similarly, single-cell DNA sequencing has emerged as a powerful tool in cancer research, offering unparalleled insights into the genetic heterogeneity and evolution of tumors. In the context of breast cancer, this technology holds substantial promise for decoding the intricate genomic landscape that drives disease progression, treatment resistance, and metastasis. By unraveling the complexities of tumor biology at a granular level, single-cell DNA sequencing provides a pathway to advancing our comprehension of breast cancer and improving patient outcomes through personalized therapeutic interventions. As single-cell sequencing technology continues to evolve and integrate into clinical practice, its application is poised to revolutionize the diagnosis, prognosis, and treatment strategies for breast cancer. This review explores the potential of single-cell sequencing technology to deepen our understanding of breast cancer, highlighting key approaches, recent advancements, and the role of the tumor microenvironment in disease plasticity. Additionally, the review discusses the impact of single-cell sequencing in paving the way for the development of personalized therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad5d30fa3be7681abcc5841abc609350bfd86737" target='_blank'>
              Recent progress and applications of single-cell sequencing technology in breast cancer
              </a>
            </td>
          <td>
            Yousef M. Hawsawi, Basmah Khoja, Abdullah Omar Aljaylani, Raniah Jaha, Rasha Mohammed Alderbi, Huda Alnuman, Mohammed I. Khan
          </td>
          <td>2024-09-18</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Sex-chromosome dosage represents a challenge for heterogametic species to maintain correct proportion of gene products across chromosomes in each sex. While therian mammals (XX/XY system) achieve near-perfect balance of X-chromosome mRNAs through X-upregulation and X-inactivation, birds (ZW/ZZ system) have been found to lack efficient compensation at RNA level, challenging the necessity of resolving major gene-dosage discrepancies in avian cells. Through allele-resolved multiome analyses, we comprehensively examined dosage compensation in female (ZW), male (ZZ), and rare intersex (ZZW) chicken. Remarkably, this revealed that females exhibit upregulation of their single Z through increased transcriptional burst frequency similar to mammalian X-upregulation, and that Z-protein levels are further balanced via enhanced translation efficiency in females. Global analyses of transcriptional kinetics elements in birds demonstrate remarkable conservation of the genomic encoding of burst kinetics between mammals and birds. Our study uncovers new mechanisms for achieving sex-chromosome dosage compensation and highlights the importance of gene-dosage balance across diverse species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05f4b40e70c2988331617fc51e819b31adbb3451" target='_blank'>
              Multi-layered dosage compensation of the avian Z chromosome
              </a>
            </td>
          <td>
            Natali Papanicolaou, Antonio Lentini, Sebastian Wettersten, Michael Hagemann-Jensen, Annika Krüger, Jilin Zhang, Christos Coucoravas, Ioannis Petrosian, Xian Xin, Ilhan Ceyhan, Joanna Rorbach, D. Wright, B. Reinius
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="CTCF architects 3D genome folding by anchoring or forestalling cohesin loop extrusion, but the exact mechanism remains obscure. Here, using clustered protocadherins (cPcdh) as model genes, we report that CTCF stabilizes or facilitates cohesin loop extrusion by enhancing its processivity. Specifically, we show that, compared with the Pcdh α and γ clusters, the Pcdhβ cluster is greatly affected upon CTCFY226A/F228A mutation in the N-terminal domain. Given the long-range distance of the Pcdhβ cluster from the distal enhancer, this finding has interesting implications in CTCF regulation of cohesin processivity along the linear chromatin during DNA loop extrusion. In particular, the effect on cohesin processivity upon CTCFY226A/F228A mutation is conspicuously similar to that of WAPL overexpression, demonstrating that, contrary to the general view of blocking or forestalling cohesin, CTCF actually enhances or facilitates cohesin loop extrusion.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09ae55bb55054fa2f97642ac29917aefae44f10b" target='_blank'>
              CTCF N-terminal Domain Regulates Clustered Protocadherin Gene Expression by Enhancing Cohesin Processivity
              </a>
            </td>
          <td>
            Yijun Zhang, Qiang Wu
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Long noncoding RNAs (lncRNAs) are essential regulatory elements of sex chromosomes that act to equalize gene expression levels between males and females. XIST, RSX, and roX2 regulate X chromosomes in placental mammals, marsupials, and Drosophila, respectively. Because the green anole (Anolis carolinensis) shows complete dosage compensation of its X chromosome, we tested whether a lncRNA was involved. We found an ancient lncRNA, MAYEX, that gained male-specific expression more than 89 million years ago. MAYEX evolved a notable association with the acetylated histone 4 lysine 16 (H4K16ac) epigenetic mark and the ability to loop its locus to the totality of the X chromosome to increase expression levels. MAYEX is the first lncRNA in reptiles linked to a dosage compensation mechanism that balances the expression of sex chromosomes. Editor’s summary Dosage compensation of sex chromosomes can be mediated either by overexpressing genes in the heterogametic sex or by silencing one copy in the homogametic. These processes are complex but are usually mediated by long noncoding RNAs (lncRNAs); however, they are difficult to study because they are not often shared broadly across species. Tenorio et al. examined the green anole and found that genes on the X chromosome are overexpressed in males. This gene expression pattern appeared to be mediated by a lncRNA that they call MAYEX. This lncRNA may be shared beyond anoles to some other species of lizards, thus contributing to our understanding of dosage compensation across animals. —Corinne Simonti">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5dd6dfb2d4c5ab6b5a06c6a1eff416e4646dc49a" target='_blank'>
              MAYEX is an old long noncoding RNA recruited for X chromosome dosage compensation in a reptile
              </a>
            </td>
          <td>
            Mariela Tenorio, Samantha Cruz-Ruiz, Sergio Encarnación-Guevara, Magdalena Hernández, Jose Antonio Corona-Gomez, Fania Sheccid-Santiago, Joanna Serwatowska, Sinai López-Perdomo, C. D. Flores‐Aguirre, D. Arenas-Moreno, R. Ossiboff, Fausto Méndez-de-la-Cruz, Selene L. Fernandez-Valverde, Mario Zurita, Katarzyna Oktaba, Diego Cortez
          </td>
          <td>2024-09-20</td>
          <td>Science</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Telomeric deposition of histone variant H3.3 is controlled by ATRX, DAXX, HIRA, and ASF1 proteins. Point mutations of H3.3 and defective mutants of ATRX are associated with cancers that utilize the alternative lengthening of telomeres (ALT) pathway to protect and maintain chromosome ends. Here, we identify a cascade of events following the expression of a TRF2 dominant negative mutant, TRF2ΔBΔM, which are regulated by H3.3 expression. H3.3 regulates the formation of damaged foci at telomeres and subsequent telomere-to-telomere fusion events controlled by the DNA repair pathway. Telomere fusion events also contributes to the formation of micronuclei and abnormal segregation. Micronuclei formation triggers activation of the cGAS-STING mediated innate immune response to intracellular DNA and inhibits cell growth. All of these phenotypes are controlled by the expression of H3.3. Additionally, disruption of ATRX, DAXX, HIRA, or ASF1 elicits an impairment phenotype similar to that is caused by H3.3 inhibition. Our results indicate that cGAS-STING-mediated cellular senescence is triggered by deprotected telomeres and is controlled by the expression and deposition of H3.3 on telomeres.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f2e3117de410325704b8c5ce4d1cbcd950d45ea" target='_blank'>
              Histone variant H3.3 mediates cGAS-STING pathway activation via telomere deprotection
              </a>
            </td>
          <td>
            Chiao-Ming Huang, Liuh-Yow Chen
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Endogenous retroviruses (ERVs) are the remnants of retroviral germline infections and are highly abundant in the genomes of vertebrates. At one time considered to be nothing more than inert ‘junk’ within genomes, ERVs have been tolerated within host genomes over vast timescales, and their study continues to reveal complex co-evolutionary histories within their respective host species. For example, multiple instances have been characterized of ERVs having been ‘borrowed’ for normal physiology, from single copies to ones involved in various regulatory networks such as innate immunity and during early development. Within the cell, the accessibility of ERVs is normally tightly controlled by epigenetic mechanisms such as DNA methylation or histone modifications. However, these silencing mechanisms of ERVs are reversible, and epigenetic alterations to the chromatin landscape can thus lead to their aberrant expression, as is observed in abnormal cellular environments such as in tumors. In this review, we focus on ERV transcriptional control and draw parallels and distinctions concerning the loss of regulation in disease, as well as their precise regulation in early development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88fcb4c839dae363d4185cc295fc1a0003652e42" target='_blank'>
              Transcription of Endogenous Retroviruses: Broad and Precise Mechanisms of Control
              </a>
            </td>
          <td>
            Abigail S. Jarosz, Julia V. Halo
          </td>
          <td>2024-08-01</td>
          <td>Viruses</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Many integrative mobile genetic elements (MGEs) are sensitive to RecA-DNA filaments, suggesting their activation may enable them to evade deletion through homologous recombination (HR). This was studied using pneumococcal prophage. Most pneumococci contained prophage that inhibited transformation-mediated HR through chromosomal modification. These included a commonly-targeted attB that reduced transformation by modifying the small cytoplasmic RNA. The prophage were almost universally regulated by C1-type or ImmAR-type systems. Representatives with either type were efficiently deleted by HR. Mutations that impaired a C1-regulated prophage increased this deletion rate, reflecting this element being activated by RecA-DNA filaments imported during transformation. ImmAR-regulated prophage instead responded to transient stimuli by excising as deletion-resistant pseudolysogens, with varying RecA sensitivity, gradually committing to lysing cells following longer-duration signals. These processes were modulated by interactions with the DNA-binding protein DprA. Therefore bacterial diversification though HR is impeded by MGEs that selfishly prevent themselves from being eliminated from the chromosome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a246916c36cd9a48942af1a067709e422a31a41c" target='_blank'>
              Chromosomal curing drives prophage evolution and bacterial epidemiology
              </a>
            </td>
          <td>
            M. Kwun, Alexandru V. Ion, Katinka J. Apagyi, N. Croucher
          </td>
          <td>2024-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="The pituitary gland is a main component of the endocrine system and a master controller of hormone production and secretion. Unlike neoplastic formation in other organs, Pituitary Neuroendocrine Tumors (PitNETs) are frequent in the population (16%) and, for unknown reasons, almost exclusively benign. So far, few genes have been identified as drivers for PitNETs, such as GNAS in somatotroph tumors and USP8 in corticotroph tumors. Using whole genome sequencing, we uncover a potential novel driver, the histone methyltransferase KMT2D, in a patient in his 50s suffering from a mixed somato-lactotroph tumor. Coverage ratio between germline and tumor revealed extensive chromosomal alterations. Single-cell RNA sequencing of the tumor shows up-regulation of known tumorigenic pathways compared to a healthy reference, as well as a different immune infiltration profile compared to other PitNETs, more closely resembling the profile of carcinomas than adenomas. Genome-wide DNA methylation analysis identified 792 differentially methylated regions, including notable hypomethylation in the promoter of SPON2, an immune-related gene. Our results show that tumors considered as quiet and non-aggressive can share drivers, features, and epigenetic alterations with metastatic forms of cancer, raising questions about the biological mechanisms controlling their homeostasis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ed149891e8838e85cd3b9f3f97e33858c825df" target='_blank'>
              Whole Genome Sequencing and single-cell transcriptomics identify KMT2D as a potential new driver for pituitary adenomas
              </a>
            </td>
          <td>
            M. Brunner, J. Meylan-Merlini, M. Muriset, S. Oreshkov, A. Messina, M. Messerer, R. T. Daniel, E. Hewer, J. P. Brouland, F. Santoni
          </td>
          <td>2024-09-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast Cancer Type 1 Susceptibility Protein (BRCA1) is a tumor-suppressor protein that regulates various cellular pathways, including those that are essential for preserving genome stability. One essential mechanism involves a BRCA1-A complex that is recruited to double-strand breaks (DSBs) by RAP80 before initiating DNA damage repair (DDR). How RAP80 itself is recruited to DNA damage sites, however, is unclear. Here, we demonstrate an intrinsic correlation between a methyltransferase DOT1L-mediated RAP80 methylation and BRCA1-A complex chromatin recruitment that occurs during cancer cell radiotherapy resistance. Mechanistically, DOT1L is quickly recruited onto chromatin and methylates RAP80 at multiple lysines in response to DNA damage. Methylated RAP80 is then indispensable for binding to ubiquitinated H2A and subsequently triggering BRCA1-A complex recruitment onto DSBs. Importantly, DOT1L-catalyzed RAP80 methylation and recruitment of BRCA1 have clinical relevance, as inhibition of DOT1L or RAP80 methylation seems to enhance the radiosensitivity of cancer cells both in vivo and in vitro. These data reveal a crucial role for DOT1L in DDR through initiating recruitment of RAP80 and BRCA1 onto chromatin and underscore a therapeutic strategy based on targeting DOT1L to overcome tumor radiotherapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7a53d6b1d8429c7df99fb0b7dc3e6119436184d" target='_blank'>
              DOT1L-mediated RAP80 methylation promotes BRCA1 recruitment to elicit DNA repair.
              </a>
            </td>
          <td>
            Huangqi Tang, Ya-Fei Lu, Rongsheng Zeng, Chaohua Liu, Yuxin Shu, Yupei Wu, Jiajie Su, Longjiang Di, Jinqin Qian, Jun Zhang, Yuan Tian, Xiaopeng Lu, Xin-Hai Pei, Qian Zhu, Wei-Guo Zhu
          </td>
          <td>2024-08-22</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Tumors are complex systems characterized by genetic, transcriptomic, phenotypic, and microenvironmental variations. The complexity of this heterogeneity plays a crucial role in metastasis, tumor progression, and recurrence. In this work, we utilized publicly available single-cell transcriptomics data from human breast cancer samples (ER+, HER2+, and triple-negative) to evaluate key concepts pertinent to cancer biology. Quantitative assessments included measures based on copy number alterations (CNAs), entropy, transcriptomic heterogeneity, and different protein-protein interaction networks (PPINs). We found that entropy and PPIN activity related to the cell cycle delineate cell clusters with notably elevated mitotic activity, particularly elevated in aggressive breast cancer subtypes. Additionally, CNA distributions differentiate between ER+ and HER2+/TN subtypes. Further, we identified positive correlations among the CNA score, entropy, and the activities of PPINs associated with the cell cycle, as well as basal and mesenchymal cell lines. These scores reveal associations with tumor characteristics, reflecting the known malignancy spectrum across breast cancer subtypes. By bridging the gap between existing literature and a comprehensive quantitative approach, we present a novel framework for quantifying cancer traits from scRNA-seq data by establishing several scores. This approach highlights the potential for deeper insights into tumor biology compared to conventional marker-based approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13112aeb4b3484a12ac1d7a07211992f2324eedd" target='_blank'>
              Unraveling tumor heterogeneity: Quantitative insights from scRNA-seq analysis in breast cancer subtypes
              </a>
            </td>
          <td>
            Daniela Senra, N. Guisoni, Luis Diambra
          </td>
          <td>2024-09-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Chromatin domain boundaries delimited by CTCF motifs can restrict the range of enhancer action. However, disruption of domain structure often results in mild gene dysregulation and thus predicting the impact of boundary rearrangements on animal development remains challenging. Here, we tested whether structural perturbation of a chromatin domain with multiple developmental regulators can result in more acute gene dysregulation and severe developmental phenotypes. We targeted clusters of CTCF motifs in a domain of the mouse genome containing three FGF ligand genes—Fgf3, Fgf4, and Fgf15—that regulate several developmental processes. Deletion of the 23.9kb cluster that defines the centromeric boundary of this domain resulted in ectopic interactions of the FGF genes with enhancers located across the deleted boundary that are active in the developing brain. This caused strong induction of FGF expression and perinatal lethality with encephalocele and orofacial cleft phenotypes. Heterozygous boundary deletion was sufficient to cause these fully penetrant phenotypes, and strikingly, loss of a single CTCF motif within the cluster also recapitulated ectopic FGF expression and caused encephalocele. However, such phenotypic sensitivity to perturbation of domain structure did not extend to all CTCF clusters of this domain, nor to all developmental processes controlled by these three FGF genes—for example, the ability to undergo lineage specification in the blastocyst and pre-implantation development were not affected. By tracing the impact of different chromosomal rearrangements throughout mouse development, we start to uncover the determinants of phenotypic robustness and sensitivity to perturbation of chromatin boundaries. Our data show how small sequence variants at certain domain boundaries can have a surprisingly outsized effect and must be considered as potential sources of gene dysregulation during development and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/49d79dff953903821b2cc804c273679c39f02444" target='_blank'>
              Structural perturbation of chromatin domains with multiple developmental regulators can severely impact gene regulation and development
              </a>
            </td>
          <td>
            S. Chakraborty, Nina Wenzlitschke, Matthew J. Anderson, A. Eraso, Manon Baudic, Joyce J. Thompson, Alicia A. Evans, Lilly M. Shatford‑Adams, Raj Chari, Parirokh Awasthi, Ryan K. Dale, M. Lewandoski, Timothy J. Petros, Pedro P. Rocha
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Myeloid malignancies carrying somatic DNMT3A mutations (DNMT3Amut) are usually resistant to standard therapy. DNMT3Amut leukemia cells accumulate toxic DNA double strand breaks (DSBs) and collapsed replication forks, rendering them dependent on DNA damage response (DDR). DNA polymerase theta (Polθ), a key element in Polθ-mediated DNA end-joining (TMEJ), is essential for survival and proliferation of DNMT3Amut leukemia cells. Polθ is overexpressed in DNMT3Amut leukemia cells due to abrogation of PARP1 PARylation-dependent UBE2O E3 ligase-mediated ubiquitination and proteasomal degradation of Polθ. In addition, PARP1- mediated recruitment of the SMARCAD1-MSH2/MSH3 repressive complex to DSBs was diminished in DNMT3Amut leukemia cells which facilitated loading of Polθ on DNA damage and promoting TMEJ and replication fork restart. Polθ inhibitors enhanced the anti-leukemic effects of mainstream drugs such as FLT3 kinase inhibitor quizartinib, cytarabine and etoposide in vitro and in mice with FLT3(ITD);DNMT3Amut leukemia. Altogether, Polθ is an attractive target in DNMT3Amut hematological malignancies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a67e19e6f65c3b6d38c0743f352c92d85456a92" target='_blank'>
              DNA polymerase theta-mediated DNA repair is a functional dependency and therapeutic vulnerability in DNMT3A deficient leukemia cells
              </a>
            </td>
          <td>
            Bac Viet Le, Umeshkumar Vekariya, Monika M. Toma, M. Nieborowska-Skorska, Marie-christine Caron, M. Gozdecka, Zayd Haydar, Martin Walsh, Jayashri Ghosh, Elaine Vaughan-Williams, Paulina Podszywalow-Bartnicka, Anna-Mariya Kukuyan, Sylwia Ziolkowska, E. Hadzijusufovic, Gurushankar Chandramouly, K. Piwocka, R. Pomerantz, George S. Vassiliou, B. Huntly, P. Valent, Alfonso Bellacosa, Jean-Yves Masson, Gaorav P. Gupta, Grant A. Challen, Tomasz Skorski
          </td>
          <td>2024-09-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="DNA replication is constantly challenged by a wide variety of endogenous and exogenous stressors that can damage DNA. Such lesions encountered during genome duplication can stall replisomes and convert replication forks into double-strand breaks. If left unrepaired, these toxic DNA breaks can trigger chromosomal rearrangements, leading to heightened genome instability and an increased likelihood of cellular transformation. Additionally, cancer cells exhibit persistent replication stress, making the targeting of replication fork vulnerabilities in tumor cells an attractive strategy for chemotherapy. A highly versatile and powerful technique to study DNA breaks during replication is the comet assay. This gel electrophoresis technique reliably detects the induction and repair of DNA breaks at the single-cell level. Herein, a protocol is outlined that allows investigators to measure the extent of DNA damage in mitotically dividing human cells using fork-stalling agents across multiple cell types. Coupling this with automated comet scoring facilitates rapid analysis and enhances the reliability in studying induction of DNA breaks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bb1e23cd556c7368ee2eef1c5cc145deb5fd207" target='_blank'>
              Detection of DNA Breaks in Dividing Human Cells by Neutral Comet Assay.
              </a>
            </td>
          <td>
            Anthony Nelligan, Huzefa Dungrawala
          </td>
          <td>2024-08-23</td>
          <td>Journal of visualized experiments : JoVE</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a2685872da8abf8d89c45e2e00bd648c1daaeb" target='_blank'>
              DNA methylation and gene expression as determinants of genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Michaël Noë, Dimitrios Mathios, A. Annapragada, Shashikant Koul, Zachariah H. Foda, J. Medina, S. Cristiano, Christopher Cherry, D. C. Bruhm, N. Niknafs, V. Adleff, Leonardo Ferreira, Hari Easwaran, Stephen Baylin, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="Topoisomerases alter DNA topology by making transient DNA strand breaks (DSBs) in DNA. The DNA cleavage reaction mechanism includes the formation of a reversible protein/DNA complex that allows rapid resealing of the transient break. This mechanism allows changes in DNA topology with minimal risks of persistent DNA damage. Nonetheless, small molecules, alternate DNA structures, or mutations in topoisomerase proteins can impede the resealing of the transient breaks, leading to genome instability and potentially cell death. The consequences of high levels of enzyme/DNA adducts differ for type I and type II topoisomerases. Top1 action on DNA containing ribonucleotides leads to 2–5 nucleotide deletions in repeated sequences, while mutant Top1 enzymes can generate large deletions. By contrast, small molecules that target Top2, or mutant Top2 enzymes with elevated levels of cleavage lead to small de novo duplications. Both Top1 and Top2 have the potential to generate large rearrangements and translocations. Thus, genome instability due to topoisomerase mis-function is a potential pathogenic mechanism especially leading to oncogenic progression. Recent studies support the potential roles of topoisomerases in genetic changes in cancer cells, highlighting the need to understand how cells limit genome instability induced by topoisomerases. This review highlights recent studies that bear on these questions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fc620e1f3abd707db6dd8102f3cc18d304a5ef7" target='_blank'>
              Genome Instability Induced by Topoisomerase Misfunction
              </a>
            </td>
          <td>
            K. C. Nitiss, Afif F. Bandak, James M. Berger, J. Nitiss
          </td>
          <td>2024-09-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56963ef797b360ca0fd377e89bb8feeef832b223" target='_blank'>
              Genome-wide characterization of dynamic DNA 5-hydroxymethylcytosine and TET2-related DNA demethylation during breast tumorigenesis
              </a>
            </td>
          <td>
            Shuang-Ling Wu, Lin Yang, Changcai Huang, Qing Li, Chunhui Ma, Fang Yuan, Yinglin Zhou, Xiaoyue Wang, Wei-Min Tong, Yamei Niu, Feng Jin
          </td>
          <td>2024-09-11</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The CHAMP1 complex, a little-known but highly conserved protein complex consisting of CHAMP1, POGZ, and HP1α, is enriched in heterochromatin though its cellular function in these regions of the genome remain unknown. Here we show that the CHAMP complex promotes heterochromatin assembly at multiple chromosomal sites, including centromeres and telomeres, and promotes homology-directed repair (HDR) of DNA double strand breaks (DSBs) in these regions. The CHAMP1 complex is also required for heterochromatin assembly and DSB repair in highly-specialized chromosomal regions, such as the highly-compacted telomeres of ALT (Alternative Lengthening of Telomeres) positive tumor cells. Moreover, the CHAMP1 complex binds and recruits the writer methyltransferase SETDB1 to heterochromatin regions of the genome and is required for efficient DSB repair at these sites. Importantly, peripheral blood lymphocytes from individuals with CHAMP1 syndrome, an inherited neurologic disorder resulting from heterozygous mutations in CHAMP1, also exhibit defective heterochromatin clustering and defective repair of local DSBs, suggesting that a defect in DNA repair underlies this syndrome. Taken together, the CHAMP1 complex has a novel role in heterochromatin assembly and the enhancement of HDR in heterochromatin.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ccfdc180764db46976d4f594601fc01b45d8357" target='_blank'>
              CHAMP1 complex directs heterochromatin assembly and promotes homology-directed DNA repair
              </a>
            </td>
          <td>
            Feng Li, Tianpeng Zhang, Aleem Syed, Amira Elbakry, Noella Holmer, Huy Nguyen, Sirisha Mukkavalli, Roger A. Greenberg, Alan D. D’Andrea
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Persistent DNA double-strand breaks (DSBs) are enigmatically implicated in neurodegenerative diseases including Huntington’s disease (HD), the inherited late-onset disorder caused by CAG repeat elongations in Huntingtin (HTT). Here we combine biochemistry, computation and molecular cell biology to unveil a mechanism whereby HTT coordinates a Transcription-Coupled Non-Homologous End-Joining (TC-NHEJ) complex. HTT joins TC-NHEJ proteins PNKP, Ku70/80, and XRCC4 with chromatin remodeler Brahma-related Gene 1 (BRG1) to resolve transcription-associated DSBs in brain. HTT recruitment to DSBs in transcriptionally active gene- rich regions is BRG1-dependent while efficient TC-NHEJ protein recruitment is HTT-dependent. Notably, mHTT compromises TC-NHEJ interactions and repair activity, promoting DSB accumulation in HD tissues. Importantly, HTT or PNKP overexpression restores TC-NHEJ in a Drosophila HD model dramatically improving genome integrity, motor defects, and lifespan. Collective results uncover HTT stimulation of DSB repair by organizing a TC-NHEJ complex that is impaired by mHTT thereby implicating dysregulation of transcription-coupled DSB repair in mHTT pathophysiology. Highlights • BRG1 recruits HTT and NHEJ components to transcriptionally active DSBs. • HTT joins BRG1 and PNKP to efficiently repair transcription related DSBs in brain. • Mutant HTT impairs the functional integrity of TC-NHEJ complex for DSB repair. • HTT expression improves DSB repair, genome integrity and phenotypes in HD flies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9eb03b67998e606f7a7795acb11a2068118359e" target='_blank'>
              Chromatin remodeler BRG1 recruits huntingtin to repair DNA double-strand breaks in neurons
              </a>
            </td>
          <td>
            Subrata Pradhan, Keegan M. Bush, Nan Zhang, R. Pandita, Chi-lin Tsai, Charlene Smith, Devon F. Pandlebury, Sagar Gaikwad, Francis Leonard, Linghui Nie, Annie Tao, William Russell, Subo Yuan, Sanjeev Choudhary, Kenneth S. Ramos, Cornelis J. Elferink, Yogesh P. Wairkar, John A Tainer, Leslie M. Thompson, T. Pandita, P. Sarkar
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="There is increasing recognition that the sex chromosomes, X and Y, play an important role in health and disease that goes beyond the determination of biological sex. Loss of the Y chromosome (LOY) in blood, which occurs naturally in aging men, has been found to be a driver of cardiac fibrosis and heart failure mortality. LOY also occurs in most solid tumors in males and is often associated with worse survival, suggesting that LOY may give tumor cells a growth or survival advantage. We analyzed LOY in lung adenocarcinoma (LUAD) using both bulk and single-cell expression data and found evidence suggesting that LOY affects the tumor immune environment by altering cancer/testis antigen expression and consequently facilitating tumor immune evasion. Analyzing immunotherapy data, we show that LOY and changes in expression of particular cancer/testis antigens are associated with response to pembrolizumab treatment and outcome, providing a new and powerful biomarker for predicting immunotherapy response in LUAD tumors in males.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/03627d4b1b03380930579b6c3a6ee53571e9ff88" target='_blank'>
              Selective loss of Y chromosomes in lung adenocarcinoma modulates the tumor immune environment through cancer/testis antigens
              </a>
            </td>
          <td>
            Jonas Fischer, K. Shutta, Chen Chen, Viola Fanfani, Enakshi Saha, Panagiotis Mandros, Marouen Ben Guebila, Joanne Xiu, Jorge Nieva, Stephen V. Liu, D. Uprety, D. Spetzler, C. Lopes-Ramos, Dawn L. DeMeo, John Quackenbush
          </td>
          <td>2024-09-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Single-cell methods to assess DNA methylation have not yet achieved the same level of cell throughput compared to other modalities. Here, we describe sciMETv3, a combinatorial indexing-based technique that builds on our prior technology, sciMETv2. SciMETv3 achieves nearly a 100-fold improvement in cell throughput by increasing the index space while simultaneously reducing hands-on time and total costs per experiment. To reduce the sequencing burden of the assay, we demonstrate compatibility of sciMETv3 with capture techniques that enrich for regulatory regions, as well as the ability to leverage enzymatic conversion which can yield higher library diversity. We showcase the throughput of sciMETv3 by producing a >140k cell library from human middle frontal gyrus split across four multiplexed individuals using both Illumina and Ultima sequencing instrumentation. This library was prepared over two days by one individual and required no expensive equipment (e.g. a flow sorter, as required by sciMETv2). The same experiment produced an estimated 650k additional cells that were not sequenced, representing the power of sciMETv3 to meet the throughput needs of the most demanding atlas-scale projects. Finally, we demonstrate the compatibility of sciMETv3 with multimodal assays by introducing sciMET+ATAC, which will enable high- throughput exploration of the interplay between two layers of epigenetic regulation within the same cell, as well as the ability to directly integrate single-cell methylation datasets with existing single-cell ATAC-seq. Highlights Atlas-scale production of single-cell DNA methylation libraries in a single experiment Protocols and evaluation using both Illumina and Ultima Genomics sequencing platforms Compatibility of sciMETv3 with capture techniques to reduce sequencing burden Compatibility of sciMETv3 with enzymatic conversion methods Generation of an integrated >140,000 cell dataset from human middle frontal gyrus across four individuals Ability to profile both ATAC and genome-wide DNA methylation from the same cells and integration with datasets from each modality A novel implementation of the s3-ATAC technology that leverages a nanowell chip for increased throughput Motivation DNA methylation forms a basal layer of epigenomic regulatory control, shaping the genomic permissiveness of mammalian cells during lineage specification and development. Aberrant DNA methylation has been associated with myriad health conditions ranging from developmental disorders to cancer. The high cell type specificity necessitates analysis at the single-cell level, much like transcription or other epigenomic properties. However, robust and cost-effective techniques to produce atlas-scale datasets have not been realized for DNA methylation. Here, we directly meet this need by introducing sciMETv3, a high-throughput protocol capable of producing hundreds of thousands of single-cell DNA methylation profiles in a single experiment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac78c7a01f74f879240ac4385b6f1d516ec40258" target='_blank'>
              Atlas-scale Single-cell DNA Methylation Profiling with sciMETv3
              </a>
            </td>
          <td>
            Ruth V. Nichols, Lauren E. Rylaarsdam, Brendan L O'Connell, Z. Shipony, Nika Iremadze, Sonia N. Acharya, Andrew C. Adey
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Transposable elements (TEs) constitute a significant portion of eukaryotic genomes, yet their role in chromatin organization remains poorly understood. This study investigates the distribution patterns of TEs around chromatin ligation points (LPs) identified through Micro-C experiments in human cells. We analyzed the density of various TE families within a 100kb window centered on LPs, focusing on major families such as Alu and LINE-1 (L1) elements. Our findings reveal distinct, non-random distribution patterns that differ between TE families and exhibit consistent strand-specific biases. These patterns were reproducible across two independent datasets and showed marked differences from random genomic distributions. Notably, we observed family-specific variations in TE density near LPs, with some families showing depletion at LPs followed by periodic fluctuations in density. The consistency of these patterns across TE families and their orientation relative to chromosome arms suggest a fundamental relationship between TEs and higher-order chromatin structure. Our results provide new insights into the potential role of TEs in genome organization and challenge the notion of TEs as passive genomic components. This study lays the groundwork for future investigations into the functional implications of TE distribution in chromatin architecture and gene regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5bdde57a115dc0fd3de477232119284f878cb2a5" target='_blank'>
              Patterns of Transposable Element Distribution Around Chromatin Ligation Points Revealed by Micro-C Data Analysis
              </a>
            </td>
          <td>
            Alexandr V. Vikhorev, Michael M. Rempel, O. O. Polesskaya, Ivan V. Savelev, Max Myakishev-Rempel
          </td>
          <td>2024-08-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Most microorganisms cannot be cultured in isolation, necessitating sophisticated methods for studying their (eco)physiology. While numerous approaches can probe the activity of given microbes in enrichment cultures, no single technique can render simultaneous data on both metabolic capacities and mobile genetic elements. Here, we apply long-read sequencing to monitor the incorporation of non-canonical bases in genome-resolved metagenomic datasets and elucidate the replication patterns of both bacteria and phages. This technology enables the simultaneous reconstruction of both prokaryotic and viral genomes (alongside genomics downstream analyses like metabolic predictions), in addition to providing information regarding their replication in enrichment cultures. By spiking the base analog 5-bromo-2’-deoxyuridine (BrdU) into activated sludge microcosms, we determined that 114 of the 118 high-quality genomes recovered were actively replicating in enrichment cultures from activated sludge and identified both slow (low BrdU incorporation and change in abundance) and rapidly replicating organisms (high BrdU incorporation and change in abundance). Some of the genomes detected exhibited regions rich in BrdU that were predicted to represent prophages in their lytic cycle. Ultimately, this novel means of monitoring the replication responses of microbes, and deciphering their genomes and active mobile genetic elements will advance and empower strategies aimed at isolating previously uncultivated microbes in pure culture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be0a85d84c15b458a44522244dee2adb3e97fbf4" target='_blank'>
              Inferring replication states of bacteria and viruses in enrichment cultures via long-read sequencing
              </a>
            </td>
          <td>
            Sophie A. Simon, A. Soares, Till L. V. Bornemann, Adrian Lange, Lea Griesdorn, Adrián Fuentes, Marie Dieckmann, Beate A. Krok, S. E. Ruff, Michael Hügler, Cristina Moraru, Alexander J. Probst
          </td>
          <td>2024-08-14</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Interphase mammalian genomes are folded in 3D with complex locus-specific patterns that impact gene regulation. CTCF (CCCTC-binding factor) is a key architectural protein that binds specific DNA sites, halts cohesin-mediated loop extrusion, and enables long-range chromatin interactions. There are hundreds of thousands of annotated CTCF-binding sites in mammalian genomes; disruptions of some result in distinct phenotypes, while others have no visible effect. Despite their importance, the determinants of which CTCF sites are necessary for genome folding and gene regulation remain unclear. Here, we update and utilize Akita, a convolutional neural network model, to extract the sequence preferences and grammar of CTCF contributing to genome folding. Our analyses of individual CTCF sites reveal four predictions: (i) only a small fraction of genomic sites are impactful, (ii) insulation strength is highly dependent on sequences flanking the core CTCF binding motif, (iii) core and flanking sequences are broadly compatible, and (iv) core and flanking nucleotides contribute largely additively to overall strength. Our analysis of collections of CTCF sites make two predictions for multi-motif grammar: (i) insulation strength depends on the number of CTCF sites within a cluster, and (ii) pattern formation is governed by the orientation and spacing of these sites, rather than any inherent specialization of the CTCF motifs themselves. In sum, we present a framework for using neural network models to probe the sequences instructing genome folding and provide a number of predictions to guide future experimental inquiries. Author Summary Mammalian genomes are spatially organized in 3D with profound consequences for all processes involving DNA. CTCF is a key genome organizer, recognizing numerous sites and creating a variety of contact patterns across the genome. Despite the importance of CTCF, the sequence determinants and grammar of how individual sites collectively instruct genome folding remain unclear. This work leverages the ability of Akita, a deep neural network, to make high-throughput predictions for genome folding after DNA sequence perturbations. Using Akita, we make several experimentally testable predictions. First, only a minority of annotated sites individually impact folding, and flanking DNA sequences greatly modulate their impact. Second, multiple sites together influence folding based on their number, orientation, and spacing. In sum, we provide a roadmap for interpreting neural networks to better understand genome folding and important considerations for the design of experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a8a450d11a02a4e12bb1863b2d5f19340a89cc" target='_blank'>
              Interpreting the CTCF-mediated sequence grammar of genome folding with AkitaV2
              </a>
            </td>
          <td>
            Paulina N. Smaruj, Fahad Kamulegeya, David R. Kelley, G. Fudenberg
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac7d722a65b854470cf42ddcd045da10745edbaf" target='_blank'>
              RAD52 resolves transcription-replication conflicts to mitigate R-loop induced genome instability
              </a>
            </td>
          <td>
            Manisha Jalan, Aman Sharma, Xin Pei, Nils Weinhold, Erika S. Buechelmaier, Yingjie Zhu, Sana Ahmed-Seghir, A. Ratnakumar, Melody Di Bona, Niamh McDermott, Joan Gomez-Aguilar, Kyrie S Anderson, Charlotte K Y Ng, P. Selenica, S. Bakhoum, Jorge S Reis-Filho, N. Riaz, Simon N. Powell
          </td>
          <td>2024-09-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Chromosomal aneuploidy, that is, numerical chromosome aberrations, is one of the molecular hallmarks of cancer. However, when neoplasms are studied with sequencing- and array-based approaches, chromosome numbers and ploidy states are typically inferred from bulk DNA data. Furthermore, published molecular estimates of neoplasia-associated aneuploidy often also include genomic imbalances resulting from various types of structural rearrangement, which likely result from other mechanisms than numerical chromosome aberrations. We thus analyzed chromosome numbers using single-cell cytogenetic data from 83,862 tumors, and show that both benign and malignant tumors are highly heterogeneous with regard to deviations from the normal, diploid state. Focusing on the chromosome numbers in 112 specific tumor types, defined by both exact morphologic diagnosis and organ location and from which data from ≥50 cases were available, we found two major clusters: one predominated by near-diploid neoplasms and one by neoplasms with extensive aneuploidy and one or more whole genome doublings. The former cluster included most benign solid tumors, myeloid neoplasms, and malignant gene fusion-associated solid tumors, whereas the latter was predominated by malignant solid tumors and lymphomas. For 16 malignant tumor types, the distribution of chromosome numbers could be compared to TCGA ploidy level data. Cytogenetic and molecular data correlated well, but the former indicates a higher level of clonal heterogeneity. The results presented here suggest shared pathogenetic mechanisms in certain tumor types and provide a reference for molecular analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b031c86f32544de4d075d55344d053563ec29d0f" target='_blank'>
              Aneuploidy in neoplasia: Single-cell data on 83,862 tumors.
              </a>
            </td>
          <td>
            Fredrik Mertens, Jakob Hofvander, Nils Mandahl, F. Mitelman
          </td>
          <td>2024-09-02</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="BACKGROUND The mitotic intra-S DNA damage checkpoint signaling gene set is potentially involved in cancers where the genes play an important role. 17 total genes are involved in this gene set: ATF2, CHEK2, EME1, EME2, FANCD2, HUS1, HUS1B, MDC1, MRE11, MSH2, MUS81, NEK11, RAD17, RAD9A, RAD9B, TIPIN, XPC. The aim of this study is to complete a pan-cancer profile of each gene in the mitotic intra-S DNA damage checkpoint signaling gene set in order to determine potential diagnostic and prognostic purposes, while also determining how they could be used in a clinical setting as therapeutic targets to help patients. METHODS Multiomic data was acquired for the 17 genes; over 9000 samples of 33 types of cancer were analyzed to create pan-cancer profiles of CNV, mRNA expression, and pathway analysis. RESULTS The CNVs of some of these genes are associated with the survival of MESO, PCPG, BLCA, SKCM, LUAD, HNSC, LUSC, OV, and BRCA could be affected by the mRNA expression of the genes which can involve regulation of copy number. CONCLUSION With sufficient investigation, the genes involved in mitotic intra-S DNA damage checkpoint signaling may contribute to the development of cancer and may be used as biomarkers for cancer prognosis and diagnosis. To prove their clinical use for diagnosis and prognosis, however, and to create workable applications in clinical settings, further work is required. However, these pan-cancer profiles provide a more comprehensive knowledge of the mitotic intra-S DNA damage checkpoint signaling gene set in cancer as well as valuable information for future reference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f222ed09772900dec985568ac443adca2980e33" target='_blank'>
              Potential Cancer Biomarkers: Mitotic Intra-S DNA Damage Checkpoint Genes
              </a>
            </td>
          <td>
            Kashvi Agarwal, Hengrui Liu
          </td>
          <td>2024-09-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/327f3b8a50e3e934afb3ca0aaae0fc0fd3ed123e" target='_blank'>
              An elevated rate of whole-genome duplications in cancers from Black patients
              </a>
            </td>
          <td>
            L. M. Brown, R. Hagenson, T. Koklič, I. Urbančič, Lu Qiao, J. Štrancar, J. Sheltzer
          </td>
          <td>2024-09-19</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Abstract Impaired control of the G1/S checkpoint allows initiation of DNA replication under non-permissive conditions. Unscheduled S-phase entry is associated with DNA replication stress, demanding for other checkpoints or cellular pathways to maintain proliferation. Here, we uncovered a requirement for ADARp150 to sustain proliferation of G1/S-checkpoint-defective cells under growth-restricting conditions. Besides its well-established mRNA editing function in inversely oriented short interspersed nuclear elements (SINEs), we found ADARp150 to exert a critical function in mitosis. ADARp150 depletion resulted in tetraploidization, impeding cell proliferation in mitogen-deprived conditions. Mechanistically we show that ADAR1 depletion induced aberrant expression of Cyclin B3, which was causative for mitotic failure and whole-genome duplication. Finally, we find that also in vivo ADAR1-depletion-provoked tetraploidization hampers tumor outgrowth.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11d2bb8be1edc7659c2bc78816fc95a0c749bf23" target='_blank'>
              ADARp150 counteracts whole genome duplication
              </a>
            </td>
          <td>
            Frank van Gemert, Alexandra Drakaki, Isabel Morales Lozano, Daniel de Groot, Maud Schoot Uiterkamp, N. Proost, C. Lieftink, M. van de Ven, R. Beijersbergen, Heinz Jacobs, H. te Riele
          </td>
          <td>2024-08-27</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdbc12eb0a2eae0ddf37c9584cc484b594730250" target='_blank'>
              Genomic profiling of cell-free DNA from dogs with benign and malignant tumors
              </a>
            </td>
          <td>
            Hongchao Du, Wenfeng Liu, Yunfei Li, Lijuan Zhang, Fangfang Jiang, Dandan Zhu, Jingshuai Li, Pan Hu, Ningning Yan, Mao Mao, Shiyong Li
          </td>
          <td>2024-09-13</td>
          <td>BMC Research Notes</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="
 Oncogenic protein dosage is tightly regulated to enable cancer cells to adapt and survive. In healthy cells, translation initiation of onco-mRNA transcripts encoding oncogenes and growth factors is tightly regulated to maintain accurate protein expression levels. However, this process is broken in cancer cells, which rely on the maintenance of high dosage of oncogenic proteins for their survival and to drive cancer development. Whether this is regulated at the level of translational control and the key factors in cis and trans remain unknown. The Myc oncogene is a central paradigm of an exquisitely regulated oncogene and a major driver of pancreatic ductal adenocarcinoma (PDAC). While there has been a tremendous effort to target Myc in cancer, clinically relevant Myc inhibition remains a challenge. Therefore, identifying selective promoters of MYC translation may present a novel therapeutic opportunity. Employing a genome-wide CRISPRi screen for modulators of MYC mRNA translation initiation in PDAC cells, we identified regulators of selective MYC translation and validated four RNA binding proteins (RBPs), including epitranscriptome modifiers. We focused on a top hit, a little studied RBP, RBM42, which is highly expressed in PDAC and predicts poor patient survival. We found that RBM42 binds and selectively regulates the translation of MYC and a precise, yet vital suite of pro-oncogenic transcripts, including JUN and EGFR. Mechanistically, employing immunoprecipitation-mass spectrometry analysis, we found that RMB42 is a novel ribosome-associated protein (RAP). Through RNA structure determination and mutagenesis analysis, we showed that RBM42 directly binds and remodels the MYC 5’UTR RNA structure, facilitating the formation of the translation pre-initiation complex and increasing MYC translation. Importantly, we demonstrated that RBM42 is necessary for human PDAC cell growth and fitness and PDAC tumorigenesis in xenograft mouse models in a Myc-dependent manner in vivo. In PDAC patient samples, RBM42 expression is correlated with Myc protein levels and transcriptional activity. This work transforms our understanding of the translational code in cancer and offers a new therapeutic opening to target the expression of oncogenes.
 Citation Format: Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace Hernandez, Gilles Rademaker, Juan A Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero. Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A078.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6940dcc1be9dcaf6b8a1bc0d89818408eab3c22" target='_blank'>
              Abstract A078: Functional screen for mediators of onco-mRNA translation specificity in pancreatic cancer
              </a>
            </td>
          <td>
            Joanna R Kovalski, Goksu Sarioglu, Vishvak Subramanyam, Grace A. Hernandez, Gilles Rademaker, J. Oses-Prieto, Macey Slota, Kwun Wah Wen, Grace E Kim, Alma L. Burlingame, Hani Goodarzi, Rushika M Perera, Davide Ruggero
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D. Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec914b4189eafe0cce08165c6a7c2a53bfc159e" target='_blank'>
              Unveiling FRG1’s DNA repair role in breast cancer
              </a>
            </td>
          <td>
            Shubhanjali Shubhanjali, Talina Mohapatra, Rehan Khan, Manjusha Dixit
          </td>
          <td>2024-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="A major goal in evolutionary biology and biomedicine is to understand the complex interactions between genetic variants, the epigenome, and gene expression. However, the causal relationships between these factors remain poorly understood. mSTARR-seq, a methylation-sensitive massively parallel reporter assay, is capable of identifying methylation-dependent regulatory activity at many thousands of genomic regions simultaneously, and allows for the testing of causal relationships between DNA methylation and gene expression on a region-by-region basis. Here, we developed a multiplexed mSTARR-seq protocol to assay naturally occurring human genetic variation from 25 individuals sampled from 10 localities in Europe and Africa. We identified 6,957 regulatory elements in either the unmethylated or methylated state, and this set was enriched for enhancer and promoter annotations, as expected. The expression of 58% of these regulatory elements was modulated by methylation, which was generally associated with decreased RNA expression. Within our set of regulatory elements, we used allele-specific expression analyses to identify 8,020 sites with genetic effects on gene regulation; further, we found that 42.3% of these genetic effects varied between methylated and unmethylated states. Sites exhibiting methylation-dependent genetic effects were enriched for GWAS and EWAS annotations, implicating them in human disease. Compared to datasets that assay DNA from a single European individual, our multiplexed assay uncovers dramatically more genetic effects and methylation-dependent genetic effects, highlighting the importance of including diverse individuals in assays which aim to understand gene regulatory processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77388b2f974e9f5ed610fdfb8a75a00e44836555" target='_blank'>
              Uncovering methylation-dependent genetic effects on regulatory element function in diverse genomes
              </a>
            </td>
          <td>
            Rachel M. Petersen, Christopher M. Vockley, Amanda J. Lea
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cd1a5615f8c194a58f199eb7268b36734c0e340" target='_blank'>
              Convergent alterations in the tumor microenvironment of MYC-driven human and murine prostate cancer
              </a>
            </td>
          <td>
            M. K. Graham, Rulin Wang, Roshan Chikarmane, Bulouere Abel, A. Vaghasia, Anuj Gupta, Qizhi Zheng, Jessica Hicks, Polina Sysa-Shah, Xin Pan, Nicole Castagna, Jianyong Liu, J. Meyers, Alyza M. Skaist, Yan Zhang, Michael Rubenstein, Kornel Schuebel, Brian W. Simons, C. Bieberich, William G Nelson, Shawn E Lupold, Theodore L. DeWeese, A. D. De Marzo, Srinivasan Yegnasubramanian
          </td>
          <td>2024-08-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/114a66efadbc9788f06d601c6a1a3abc5635eb72" target='_blank'>
              Evolution of retrocopies in the context of HUSH silencing
              </a>
            </td>
          <td>
            Joanna Kozłowska-Masłoń, Joanna Ciomborowska-Basheer, M. Kubiak, Izabela Makałowska
          </td>
          <td>2024-08-02</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="During cellular differentiation, enhancers transform overlapping gradients of transcription factors (TFs) to highly specific gene expression patterns. However, the vast complexity of regulatory DNA impedes the identification of the underlying cis-regulatory rules. Here, we have characterized 62,126 fully synthetic DNA sequences to bottom-up dissect design principles of cell-state specific enhancers in the context of the differentiation of blood stem cells to seven myeloid lineages. Focusing on binding sites for 38 TFs and their pairwise interactions, we found that identical sites displayed both repressive and activating function, as a consequence of cellular context, site combinatorics, or simply predicted occupancy of a TF on an enhancer. Surprisingly, we found that combinations of activating sites frequently neutralized each other or even gained repressive function. These negative synergies convert quantitative imbalances in transcription factor expression into binary downstream activity patterns, a principle that can be exploited to build differentiation-state specific enhancers from scratch.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab0f40a79f1a6210ac7f2c422c1cc54bfc294a6c" target='_blank'>
              Synthetic enhancers reveal design principles of cell state specific regulatory elements in hematopoiesis
              </a>
            </td>
          <td>
            R. Frömel, Julia Rühle, Aina Bernal Martinez, Chelsea Szu-Tu, Felix Pacheco Pastor, Rosa Martinez Corral, Lars Velten
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9be5f58320c6cf366bc684bebd0ed4433dc1746c" target='_blank'>
              Pramel15 facilitates zygotic nuclear DNMT1 degradation and DNA demethylation
              </a>
            </td>
          <td>
            Jiajun Tan, Yingfeng Li, Xiang Li, Xiaoxiao Zhu, Liping Liu, Hua Huang, Jiahua Wei, Hailin Wang, Yong Tian, Zhigao Wang, Zhuqiang Zhang, Bing Zhu
          </td>
          <td>2024-08-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a2dde0164af0025d5f19bf691294f177add4f7b" target='_blank'>
              Low-pass whole genome sequencing of circulating tumor cells to evaluate chromosomal instability in triple-negative breast cancer
              </a>
            </td>
          <td>
            S. Di Cosimo, M. Silvestri, C. De Marco, Alessia Calzoni, M. C. De Santis, Maria Grazia Carnevale, C. Reduzzi, Massimo Cristofanilli, Vera Cappelletti
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Introduction The complexity of tumor cell subclonal structure has been extensively investigated in hepatocellular carcinoma. However, the role of subclonal complexity in reshaping the tumor microenvironment (TME) remains poorly understood. Methods We integrated single-cell transcriptome sequencing data from four independent HCC cohorts, involving 30 samples, to decode the associations between tumor subclonal complexity and the TME. We proposed a robust metric to accurately quantify the degree of subclonal complexity for each sample based on discrete copy number variations (CNVs) profiles. Results We found that tumor cells in the high-complexity group originated from the cell lineage with FGB overexpression and exhibited high levels of transcription factors associated with poor survival. In contrast, tumor cells in low-complexity patients showed activation of more hallmark signaling pathways, more active cell-cell communications within the TME and a higher immune activation status. Additionally, cytokines signaling activity analysis suggested a link between HMGB1 expressed by a specific endothelial subtype and T cell proliferation. Discussion Our study sheds light on the intricate relationship between the complexity of subclonal structure and the TME, offering novel insights into potential therapeutic targets for HCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ed5eee1a75397be2ec5144411bd37b05727e6ac" target='_blank'>
              Single-cell transcriptomics reveals tumor microenvironment remodeling in hepatocellular carcinoma with varying tumor subclonal complexity
              </a>
            </td>
          <td>
            Jian Shi, Yanru Zhang, Lixia Xu, Fang Wang
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f891979797e3688f35366887366668ec188a50ae" target='_blank'>
              Microsatellite instability at U2AF-binding polypyrimidic tract sites perturbs alternative splicing during colorectal cancer initiation
              </a>
            </td>
          <td>
            V. Jonchère, Hugo Montémont, Enora Le Scanf, A. Siret, Q. Letourneur, Emmanuel Tubacher, Christophe Battail, Assane Fall, Karim Labreche, Victor Renault, T. Ratovomanana, O. Buhard, A. Jolly, Philippe Le Rouzic, Cody Feys, Emmanuelle Despras, Habib Zouali, R. Nicolle, Pascale Cervera, M. Svrcek, P. Bourgoin, Hélène Blanché, A. Boland, Jérémie Lefèvre, Y. Parc, M. Touat, F. Bielle, D. Arzur, Gwennina Cueff, C. Le Jossic-Corcos, Gaël Quéré, Gwendal Dujardin, Marc Blondel, Cédric Le Maréchal, R. Cohen, Thierry André, Florence Coulet, P. de la Grange, A. de Reyniès, Jean-françois Flejou, Florence Renaud, A. Alentorn, Laurent Corcos, J. Deleuze, A. Collura, A. Duval
          </td>
          <td>2024-08-06</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b0e84c70be39b55c5ab372a108ddda6ab970765" target='_blank'>
              Polθ: emerging synthetic lethal partner in homologous recombination-deficient tumors.
              </a>
            </td>
          <td>
            T. B. Bazan Russo, Clarissa Mujacic, Emilia Di Giovanni, Maria Concetta Vitale, Carla Ferrante Bannera, U. Randazzo, Silvia Contino, M. Bono, V. Gristina, A. Galvano, A. Perez, G. Badalamenti, A. Russo, V. Bazan, L. Incorvaia
          </td>
          <td>2024-08-09</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Background The mitochondrial DNA repair has gained attention for its potential impact on pan-cancer genetic analysis. This study investigates the clinical relevance of mitochondrial DNA repair genes: PARP1, DNA 2, PRIMPOL, TP53, MGME1. Methods Using multi-omics profiling data and Gene Set Cancer Analysis (GSCA) with normalized SEM mRNA expression, this research analyzes differential expression, gene mutation, and drug correlation. Results TP53 was the most commonly mutated mitochondrial-related gene in cancer, with UCS and OV having the highest mutation rates. CPG mutations linked to lowest survival rates. Breast cancer, with various subtypes, was potentially influenced by mitochondrial DNA repair genes. ACC was shown to be high in gene survival analysis. BRCA, USC, LUCS, COAD, and OV showed CNV levels impacting survival. A negative gene expression-methylation correlation was observed and was weakest in KIRC. Mitochondrial DNA repair genes were linked to Cell cycle_A activation. A weak correlation was found between immune infiltration and mitochondrial genes. Few drug compounds were shown to be affected by mitochondrial-related genes. Conclusion Understanding mitochondrial-related genes could redefine cancer diagnosis, and prognosis, and serve as therapeutic biomarkers, potentially altering cancer cell behavior and treatment outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b24d1546c726bf1a29f7cc51df4338f8e47ed456" target='_blank'>
              Pan-Cancer Genetic Analysis of Mitochondrial DNA Repair Gene Set
              </a>
            </td>
          <td>
            Angela Dong, Ayana Meegol Rasteh, Hengrui Liu
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/65abef66cfc3b531cbb5f7b34f625f1ee91844ae" target='_blank'>
              Unraveling the key role of chromatin structure in cancer development through epigenetic landscape characterization of oral cancer
              </a>
            </td>
          <td>
            Yue Xue, Lu Liu, Ye Zhang, Yueying He, Jingyao Wang, Zicheng Ma, Tie-jun Li, Jianyun Zhang, Yanyi Huang, Yi Qin Gao
          </td>
          <td>2024-09-06</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Recent advances in barcoding technologies have significantly enhanced the scalability of single-cell genomic experiments. However, large-scale experiments are still rare due to high costs, complex logistics, and laborintensive procedures. To facilitate the routine application of the largest scalability, it is critical to simplify the production and use of barcoding reagents. Here, we introduce AmpliDrop, a technology that initiates the barcoding process using a pool of inexpensive single-copy barcodes and integrates barcode multiplicity generation with tagging of cellular content into a single reaction driven by DNA polymerase during library preparation. The barcoding reactions are compartmentalized using an electronic pipette or a robotic or standalone liquid handling system. These innovations eliminate the need for barcoded beads and complex combinatorial indexing workflows and provide flexibility for a wide range of scales and tube formats, as well as compatibility with automation. We show that AmpliDrop is capable of capturing transcriptomes and chromatin accessibility, and it can also be adapted for user-customized applications, including antibody-based protein detection, bacterial or viral DNA detection, and CRISPR perturbations without dual guide RNA-expression vectors. We validated AmpliDrop by investigating the influence of short-term static culturing on cell composition in human forebrain organoids, revealing metabolic reprogramming in lineage progenitors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2a1929b40e5049e798f4ed9d63d9a91f9f10d52" target='_blank'>
              Single-Molecule Barcoding Technology for Single-Cell Genomics
              </a>
            </td>
          <td>
            Ivan Garcia-Bassets, Guoya Mo, Yun Xia, Tsai-Chin Wu, Immanuel Mekuria, Veronika Mikhaylova, Madison Rzepka, Tetsuya Kawamura, Peter L. Chang, Amber Paasch, Long Pham, Surya Shiv Venugopal, Sandra Sanchez, Janaína S. de Souza, Likun Yao, Sifeng Gu, Z. Bodai, Alexis C. Komor, A. Muotri, Joy Wang, Yong Wang, Ming Lei, A. Almenar-Queralt, Zhoutao Chen
          </td>
          <td>2024-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="Cholangiocarcinoma (CCA) is a diverse collection of malignant tumors that originate in the bile ducts. Mitochondria, the energy converters in eukaryotic cells, contain circular mitochondrial DNA (mtDNA) which has a greater mutation rate than nuclear DNA. Heteroplasmic variations in mtDNA may suggest an increased risk of cancer-related mortality, serving as a potential prognostic marker. In this study, we investigated the mtDNA variations of five CCA cell lines, including KKU- 023, KKU-055, KKU-100, KKU213A, and KKU-452 and compared them to the non-tumor cholangiocyte MMNK-1 cell line. We used Oxford Nanopore Technologies (ONT), a long-read sequencing technology capable of synthesizing the whole mitochondrial genome, which facilitates enhanced identification of complicated rearrangements in mitogenomics. The analysis revealed a high frequency of SNVs and INDELs, particularly in the D-loop, MT-RNR2, MT-CO1, MT-ND4, and MT-ND5 genes. Significant mutations were detected in all CCA cell lines, with particularly notable non-synonymous SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. The presence of these mutations suggests the potential for mitochondrial dysfunction and CCA progression. Analysis of mtDNA structural variants (SV) revealed significant variability among the cell lines. We identified 208 SVs in KKU-023, 185 SVs in KKU-055, 231 SVs in KKU-100, 69 SVs in KKU-213A, 172 SVs in KKU-452, and 217 SVs in MMNK-1. These SVs included deletions, duplications, and inversions, with the highest variability observed in KKU-100 and the lowest in KKU-213A. Our results underscore the diverse mtDNA mutation landscape in CCA cell lines, highlighting the potential impact of these mutations on mitochondrial function and CCA cell line progression. Future research is required to investigate the functional impacts of these variants, their interactions with nuclear DNA in CCA, and their potential as targets for therapeutic intervention. Author Summary Bile duct cancer has the highest incidence in northeastern Thailand. In this study, we explored the mtDNA of CCA cell lines, as mitochondria play a key role in cellular energy production. We aimed to identify mutations that could serve as therapeutic targets or biomarkers. Our analysis revealed that each CCA cell line has unique mtDNA profiles, including INDELs, SVs, and SNVs such as m.8462T>C in KKU-023, m.9493G>A in KKU-055, m.9172C>A in KKU-100, m.15024G>C in KKU-213A, m.12994G>A in KKU-452, and m.13406G>A in MMNK-1, which demonstrated high pathogenicity scores. yet they also shared some common mutations. Future research is needed to understand how these mutations affect cell function, interact with nuclear DNA, and their potential for therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4eafb79f26f32f1aeaf6d648b336c327747f051f" target='_blank'>
              Mitochondrial genomic alterations in cholangiocarcinoma cell lines
              </a>
            </td>
          <td>
            Athitaya Faipan, Sirinya Sitthirak, Arporn Wangwiwatsin, N. Namwat, P. Klanrit, Attapol Titapan, A. Jareanrat, V. Thanasukarn, Natcha Khuntikeo, Luke Boulter, W. Loilome
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="MeCP2 is a DNA-binding transcriptional regulator that is present at near-histone levels in mammalian cortical neurons. Originally identified as a DNA methylation reader, MeCP2 has been proposed to repress transcription by recruiting corepressors to methylated DNA. While some genome-wide occupancy studies support a preference for methylated DNA, others suggest that MeCP2 binding is more influenced by DNA sequence and accessibility than methylation status. Moreover, multiple studies also suggest a role for MeCP2 in gene activation. To clarify MeCP2 function we expressed MeCP2 in Saccharomyces cerevisiae, which lacks DNA methylation and known MeCP2 corepressors. We find that MeCP2 is toxic to yeast and globally inhibits transcription, indicating that MeCP2 can have significant functional impacts without DNA methylation or mammalian corepressors. A subset of MeCP2 mutations that cause the neurodevelopmental disorder Rett syndrome, particularly those that map to the DNA binding domain, alleviate the toxicity of MeCP2 in yeast. Consistent with the importance of DNA binding for toxicity in yeast, we show that MeCP2 binds to the yeast genome, with increased occupancy at GC-rich, nucleosome-depleted sequences. These findings present yeast as a useful tool for analyzing MeCP2 and reveal MeCP2 properties that are not strictly dependent on DNA methylation or mammalian corepressors. Summary MeCP2, a transcription regulator found in vertebrates, is proposed to repress transcription by recruiting corepressors to methylated DNA. In this work Brown et al. show that MeCP2 expressed in Saccharomyces cerevisiae, which lacks DNA methylation and mammalian co-repressors, binds promoters and inhibits transcription. A subset of MeCP2 mutations that cause the neurodevelopmental disorder, Rett syndrome, rescues MeCP2-induced phenotypes in yeast, supporting the relevance of these results to MeCP2 function in mammals.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d4ccb5c9508f4b93da72c0b40438841b862fc29" target='_blank'>
              Human MeCP2 binds to promoters and inhibits transcription in an unmethylated S. cerevisiae genome
              </a>
            </td>
          <td>
            Joshua A. R. Brown, Maggie Y. M. Ling, Juan Ausió, L. Howe
          </td>
          <td>2024-08-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c4d24999230a049c47c4ad63da565877c2b583c" target='_blank'>
              PPM1D activity promotes cellular transformation by preventing senescence and cell death.
              </a>
            </td>
          <td>
            Miroslav Stoyanov, Andra S. Martinikova, Kateřina Matějková, K. Horackova, P. Zemánková, Kamila Burdová, Zuzana Zemanova, P. Kleiblova, Z. Kleibl, L. Macurek
          </td>
          <td>2024-09-05</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/141df69604dde0dcf2c796a0436f024e9fb9a66b" target='_blank'>
              Two-polarized roles of transcription factor FOSB in lung cancer progression and prognosis: dependent on p53 status
              </a>
            </td>
          <td>
            Hongchao Zhang, Guopei Zhang, Mingyang Xiao, S. Cui, Cuihong Jin, Jinghua Yang, Sheng-wen Wu, Xiaobo Lu
          </td>
          <td>2024-08-21</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Abstract Pediatric classic Hodgkin lymphoma (cHL) patients have a high survival rate but suffer from severe long‐term side effects induced by chemo‐ and radiotherapy. cHL tumors are characterized by the low fraction (0.1%–10%) of malignant Hodgkin and Reed–Sternberg (HRS) cells in the tumor. The HRS cells depend on the surrounding immune cells for survival and growth. This dependence is leveraged by current treatments that target the PD‐1/PD‐L1 axis in cHL tumors. The development of more targeted therapies that are specific for the tumor and are therefore less toxic for healthy tissue compared with conventional chemotherapy could improve the quality of life of pediatric cHL survivors. Here, we applied single‐cell RNA sequencing (scRNA‐seq) on isolated HRS cells and the immune cells from the same cHL tumors. Besides TNFRSF8 (CD30), we identified other genes of cell surface proteins that are consistently overexpressed in HRS cells, such as NRXN3 and LRP8, which can potentially be used as alternative targets for antibody–drug conjugates or CAR T cells. Finally, we identified potential interactions by which HRS cells inhibit T cells, among which are the galectin‐1/CD69 and HLA‐II/LAG3 interactions. RNAscope was used to validate the enrichment of CD69 and LAG3 expression on T cells near HRS cells and indicated large variability of the interaction strength with the corresponding ligands between patients and between tumor tissue regions. In conclusion, this study identifies new potential therapeutic targets for cHL and highlights the importance of studying heterogeneity when identifying therapy targets, specifically those that target tumor‐immune cell interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/963141cd2c957924d6cb5b66200fd817e1bf6a44" target='_blank'>
              Single‐cell RNA sequencing of pediatric Hodgkin lymphoma to study the inhibition of T cell subtypes
              </a>
            </td>
          <td>
            Jurrian K de Kanter, A. Steemers, D. M. Gonzalez, Ravian L. van Ineveld, Catharina Blijleven, N. Groenen, L. Trabut, M. Scheijde-Vermeulen, Liset Westera, Auke Beishuizen, Anne C. Rios, F. C. Holstege, A. Brandsma, T. Margaritis, R. van Boxtel, F. Meyer-Wentrup
          </td>
          <td>2024-09-01</td>
          <td>HemaSphere</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="
 Ovarian cancer is one of the most lethal gynecological malignancies worldwide, posing immense challenges in early detection, displaying grim prognoses, and having high rates of recurrence and mortality. This malignancy exhibits a range of ethnic diversity-mortality rates, each with distinct sub-categories characterized by variations in both molecular attributes and clinical behavior. The most prevalent and also most aggressive among the various ovarian cancer sub-types is high-grade serous ovarian carcinoma (HGSOC). This carcinoma is marked by genomic instability, resulting in the malfunction of tumor suppressor genes and oncogenes, caused by homologous recombination deficiency, copy number variations, chromosomal abnormalities, and epigenetic changes. The pursuit of understanding and targeting genomic instability has significantly guided approaches to screening, prognosis, and therapeutic strategies in ovarian cancer. Further insights from molecular investigations have unveiled the regulatory role of microRNAs (short non-coding RNAs) in modulating the expression of target tumor suppressors and oncogenes by interacting with messenger RNA (mRNA). Studies have shown that miRNAs are dysregulated in ovarian cancer, however, no data has shown significant relationships between these miRNAs to the patient demographics, including race/ethnicity. miRNAs can alter oncogenes, and tumor suppressor gene expression levels, serving as early biomarkers. We utilized the TCGA database from the University of Alabama at Birmingham (UALCAN) software to analyze the expressions of targeted genes and miRNAs. We determined various oncogenes and tumor suppressor genes with higher statistical significance on survival rates, race, cancer stages, tumor grades, and miRNAs linked to them. Additionally, genomic instability was assessed using aneuploidy score, fraction of genome altered, mutation counts, mutation burden, and prognosis (i.e., overall survival and progression-free survival), downloaded from cBioPortal, and the genomic instability parameters were calculated by Spearman correlation, and represented by a heatmap. Further, DNA and RNA from 22 HGSOC samples of paraffin-embedded tissues of ovarian cancer patients were collected for RT-qPCR assays for several miRNAs. Of the genes analyzed, BRCA1, BRAF, CCNE1, and miRNAs linked to these genes showed statistically significant data on survival rates. BRAF/CCNE1 is upregulated while BRCA 1 is downregulated in ovarian high-grade serous carcinoma. We also found quantitative differential expressions of some miRNAs analyzed in African Americans versus White Americans. Successful identification of early molecular markers in ovarian cancer will potentially allow for early detection and decreased mortality rates with HGSOC. Analysis of the appearance and abundance of miRNAs by qPCR assay in early-stage ovarian cancer can aid as an early diagnostic and prognostic tool. In a further study, we hope to verify our results by testing a larger cohort of samples from African Ancestry.
 Citation Format: Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga. miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr C075.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55f1e0052202e2129f599047f6e77f66413139b6" target='_blank'>
              Abstract C075: miRNAs linked to genome instability in ovarian high-grade serous carcinoma in patients with African ancestry
              </a>
            </td>
          <td>
            Jane M. Muinde, Cody S Carter, Salma Khan, Hang Chang, Innocent O. Maranga
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a51aa320dccf7a4322f2753df2fe86ac6b9b3a8" target='_blank'>
              Unraveling the three-dimensional (3D) genome architecture in Neurodevelopmental Disorders (NDDs).
              </a>
            </td>
          <td>
            P. Carballo-Pacoret, A. Carracedo, C. Rodriguez-Fontenla
          </td>
          <td>2024-08-27</td>
          <td>Neurogenetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Epigenetic alterations are implicated in the early stages of tumorigenesis and are widely recognized as a ubiquitous phenomenon in cancer development. Aberrant epigenetic modifications can alter the expression of target genes, induce heterochromatin formation, and gradually drive normal cells towards immortalized tumor cells with significant consequences. SETDB1 (SET domain bifurcated histone lysine methyltransferase 1), a typical histone me-thyltransferase, promotes the formation of heterochromatin and inhibits the transcription of genes by modifying the methylation of lysine 9 of histone 3. SETDB1 is usually highly ex-pressed in tumors with high copy numbers, accompanied by poor prognosis and low patient survival rates, which is a typical case of abnormal epigenetic modification. We discuss the mechanism of SETDB1 in a variety of cancers and review the epigenetic inhibitors that have been reported in recent years, along with their anti-tumor effects. In addition, we summarize the role of SETDB1 in a variety of diseases and cell functions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d8068b1c29211fa575e01e9ddbf87ca49f2785f" target='_blank'>
              Targeting the Tumor Epigenetic Regulator SETDB1 for Tumor Therapy.
              </a>
            </td>
          <td>
            Cheng Peng, Nini Zhou, Tengjiang Chen, Jie Lei, Changwen Chen, Shun-qin Zhu
          </td>
          <td>2024-08-27</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/416e85b3682e5ce745f93530334b6600c5e45ed0" target='_blank'>
              Joint single-cell genetic and transcriptomic analysis reveal pre-malignant SCP-like subclones in human neuroblastoma
              </a>
            </td>
          <td>
            T. Olsen, Jörg Otte, S. Mei, Bethel Tesfai Embaie, P. Kameneva, Huaitao Cheng, Teng Gao, V. Zachariadis, Ioanna Tsea, Åsa Björklund, Emil Kryukov, Ziyi Hou, Anna Johansson, Erik Sundström, Tommy Martinsson, S. Fransson, Jakob Stenman, S. S. Fard, J. Johnsen, P. Kogner, I. Adameyko, Martin Enge, P. V. Kharchenko, Ninib Baryawno
          </td>
          <td>2024-08-31</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="The nucleus, a highly organized and dynamic organelle, plays a crucial role in regulating cellular processes. During cell differentiation, profound changes occur in gene expression, chromatin organization, and nuclear morphology. This review explores the intricate relationship between nuclear architecture and cellular function, focusing on the roles of the nuclear lamina, nuclear pore complexes (NPCs), sub-nuclear bodies, and the nuclear scaffold. These components collectively maintain nuclear integrity, organize chromatin, and interact with key regulatory factors. The dynamic remodeling of chromatin, its interactions with nuclear structures, and epigenetic modifications work in concert to modulate gene accessibility and ensure precise spatiotemporal control of gene expression. The nuclear lamina stabilizes nuclear shape and is associated with inactive chromatin regions, while NPCs facilitate selective transport. Sub-nuclear bodies contribute to genome organization and gene regulation, often by influencing RNA processing. The nuclear scaffold provides structural support, impacting 3D genome organization, which is crucial for proper gene expression during differentiation. This review underscores the significance of nuclear architecture in regulating gene expression and guiding cell differentiation. Further investigation into nuclear structure and 3D genome organization will deepen our understanding of the mechanisms governing cell fate determination.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e3a4a1b9e55e69b48e38215e900d744d3c0775b" target='_blank'>
              Nuclear Structures and Their Emerging Roles in Cell Differentiation and Development.
              </a>
            </td>
          <td>
            Hye Ji Cha
          </td>
          <td>2024-08-22</td>
          <td>BMB reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The p53 tumor suppressor is an indispensable regulator of DNA damage responses that accelerates carcinogenesis when mutated. In this report, we uncover a new mechanism by which p53 maintains genomic integrity in the absence of canonical DNA damage response activation. Specifically, loss of p53 dramatically alters chromatin structure at the nuclear periphery, allowing increased transmission of an environmental carcinogen, ultraviolet (UV) radiation, into the nucleus. Genome-wide mapping of UV-induced DNA lesions in p53-deficient primary cells reveals elevated lesion abundance in regions corresponding to locations of high mutation burden in malignant melanomas. These findings uncover a novel role of p53 in the suppression of mutations that contribute to cancer and highlight the critical influence of nuclear architecture in regulating sensitivity to carcinogens. One-Sentence Summary The p53 tumor suppressor reduces carcinogen sensitivity and mutagenic potential by maintaining nuclear architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ce95350c5fb375408fd587e0eea1b652005b9ec" target='_blank'>
              p53 Regulates Nuclear Architecture to Reduce Carcinogen Sensitivity and Mutagenic Potential
              </a>
            </td>
          <td>
            Devin A. King, Dakota E. McCoy, Adrian Perdyan, Jakub Mieczkowski, Thierry Douki, Jennifer A. Dionne, Rafael E. Herrera, Ashby J. Morrison
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Transposable elements (TEs) are DNA sequences able to create copies of themselves within the genome. Despite their limited expression due to silencing, TEs still manage to impact the host genome. For instance, some TEs have been shown to act as cis-regulatory elements and be co-opted in the human genome. This highlights that the contributions of TEs to the host might come from their relationship with the epigenome rather than their expression. However, a systematic analysis that relates TEs in the human genome directly with chromatin histone marks across distinct cell types remains lacking. Here we leverage a new dataset from the International Human Epigenome Consortium with 4867 uniformly processed ChIP-seq experiments for 6 histone marks across 175 annotated cell labels and show that TEs have drastically different enrichments levels across marks. Overall, we find that TEs are generally depleted in H3K9me3 histone modification, except for L1s, while MIRs were highly enriched in H3K4me1, H3K27ac and H3K27me3 and Alus were enriched in H3K36me3. Furthermore, we present a generalised profile of the relationship between TEs enrichment and TE age which reveals a few TE families (Alu, MIR, L2) as diverging from expected dynamics. We also find some significant differences in TE enrichment between cell types and that in 20% of the cases, these enrichments were cell-type specific. We report that at least 4% of cell types with healthy and cancer samples featured significant differences. Notably, we identify 456 TE-Cell Type-histone triplet candidates with the strongest cell-type specific enrichments. We show that many of these candidates are associated with relevant biological processes and genes expressed in the relevant cell type. These results further support a role for TE in genome regulation and highlight novel associations between TEs and histone marks across cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa3b0974577b9fd96a44fb8b012fd715ee99dc99" target='_blank'>
              Transposable elements impact the regulatory landscape through cell type specific epigenomic associations
              </a>
            </td>
          <td>
            Jeffrey Hyacinthe, Guillaume Bourque
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Diffuse large B cell lymphomas and follicular lymphomas show recurrent mutations in epigenetic regulators; among these are loss-of-function mutations in KMT2D and gain-of-function mutations in EZH2. To systematically explore the effects of these mutations on the wiring of the epigenetic network, we applied a single-cell approach to probe a wide array of histone modifications. We show that mutant-EZH2 elicits extensive effects on the epigenome of lymphomas, beyond alterations to H3K27 methylations, and is dominant over KMT2D mutations. Utilizing the single-cell data, we present computational methods to measure epigenetic heterogeneity. We identify an unexpected characteristic of mutant-EZH2, but not KMT2D, in increasing heterogeneity, shedding light on a novel oncogenic mechanism mediated by this mutation. Finally, we present tools to reconstruct known interactions within the epigenetic network, as well as reveal potential novel cross talk between various modifications, validated by functional perturbations. Our work highlights novel roles for mutant-EZH2 in lymphomagenesis and establishes new concepts for measuring epigenetic heterogeneity and intra-chromatin connectivity in cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8eb55432a81ae9f11fc653e3722684f5e8a7ed1" target='_blank'>
              Mutant EZH2 alters the epigenetic network and increases epigenetic heterogeneity in B cell lymphoma
              </a>
            </td>
          <td>
            Ofir Griess, N. Furth, Nofar Harpaz, Nicoletta Di Bernardo, T. Salame, B. Dassa, Ioannis Karagiannidis, Yusuke Isshiki, Menachem Gross, Ari M Melnick, W. Béguelin, Guy Ron, E. Shema
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Long INterspersed Element-1 (LINE-1; L1) and Alu are two families of transposable elements (TEs) occupying ∼17% and ∼11% of the human genome, respectively. Though only a small fraction of L1 copies is able to produce the machinery to mobilize autonomously, Alu elements and degenerate L1 copies can hijack their functional machinery and mobilize in trans. The expression and subsequent copy number expansion of L1 and Alu can exert pathological effects on their hosts, promoting genome instability, inflammation, and cell cycle alterations. These features have made L1 and Alu promising focus subjects in studies of aging and aging diseases where they can become active. However, the mechanisms regulating variation in their expression and copy number remain incompletely characterized. Moreover, the relevance of known mechanisms to diverse human populations remains unclear, as mechanisms are often characterized in isogenic cell culture models. To address these gaps, we leveraged genomic data from the 1000 Genomes Project to carry out a trans-ethnic GWAS of L1 and Alu insertion global singletons. These singletons are rare insertions observed only once in a population, potentially reflecting recently acquired L1 and Alu integrants or structural variants, and which we used as proxies for L1/Alu-associated copy number variation. Our computational approach identified single nucleotide variants in genomic regions containing genes with potential and known TE regulatory properties, and it enriched for single nucleotide variants in regions containing known regulators of L1 expression. Moreover, we identified many reference TE copies and polymorphic structural variants that were associated with L1/Alu singletons, suggesting their potential contribution to TE copy number variation through transposition-dependent or transposition-independent mechanisms. Finally, a transcriptional analysis of lymphoblastoid cells highlighted potential cell cycle alterations in a subset of samples harboring L1/Alu singletons. Collectively, our results (i) suggest that known TE regulatory mechanisms may also play regulatory roles in diverse human populations, (ii) expand the list of genic and repetitive genomic loci implicated in TE copy number variation, and (iii) reinforce the links between TEs and disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c547e837e626947d2c017f057b521d189f2c4fd3" target='_blank'>
              Multi-ancestry GWAS reveals loci linked to human variation in LINE-1- and Alu-copy numbers
              </a>
            </td>
          <td>
            Juan I. Bravo, Lucia Zhang, BA Benayoun
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The yeast mating type switching process is used as a biological model system to study eukaryotic DNA double strand break (DSB) repair, a process of fundamental importance in health and disease. However, tools for tracking the mating type switching process in real-time in live cells are limited. We developed methods to follow and quantify mating type switching occurrence in real-time, at the single live cell level, applicable for detection of rare events and population heterogeneity. First, we show, using a fluorescent reporter in live cells, that switching probability in strains tagged by a fluorescent repressor-operator system (FROS) is reduced, not only due to the integration of operator repeats, but also due to the overexpressed fluorescent molecules in the nucleus. Next, by applying wavefront shaping to imaging flow cytometry, we collected 3D data in high throughput, and were able to differentiate between MATa and MATα conformation and to detect DNA dynamics during switching with unprecedented spatial and temporal resolutions. Using a strain with the MAT-stk mutation, which gives rise to inefficient HO endonuclease DSB activity, we discriminated between two consecutive distances between the HMLα and the MAT loci upon DNA damage. We suggest that initial nearing between the loci occurs for both single and double strand breaks, and further and closer proximity occurs for DSB only, leading to mating type switching. Our approach can be adapted to other biological systems to study DSB repair with high precision, throughput and sensitivity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8a495c60aa12fd6c68881614741a82c18cfc68a" target='_blank'>
              Monitoring DNA double strand break repair in Saccharomyces cerevisiae at high throughput and high resolution
              </a>
            </td>
          <td>
            Yael Shalev Ezra, Alon Saguy, Gaia Levin, Lucien E. Weiss, Onit Alalouf, Y. Shechtman
          </td>
          <td>2024-08-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Background Building on our prior work that RNA alternative splicing modulates the druggability of kinase fusions, this study probes the clinical significance of sole reciprocal fusions. These rare genomic arrangements, despite lacking kinase domains at the DNA level, demonstrated potential RNA‐level druggability in sporadic cases from our prior research. Methods Utilizing the large‐scale multicenter approach, we performed RNA sequencing and clinical follow‐up to evaluate a broad spectrum of kinase fusions, including ALK, ROS1, RET, BRAF, NTRK, MET, NRG1, and EGFR, in 1943 patients. Results Our findings revealed 51 instances (2.57%) of sole reciprocal fusions, predominantly in lung (57%), colorectal (14%), and glioma (10%) cancers. Comparative analysis with an MSKCC cohort confirmed the prevalence in diverse cancer types and identified unique fusion partners and chromosomal locales. Cross‐validation through RNA‐NGS and FISH authenticated the existence of functional kinase domains in subsets including ALK, ROS1, RET, and BRAF, which correlated with positive clinical responses to targeted kinase inhibitors (KIs). Conversely, fusions involving EGFR, NRG1, and NTRK1/2/3 generated nonfunctional transcripts, suggesting the need for alternative therapeutic interventions. Conclusion This inaugural multicenter study introduces a novel algorithm for detecting and treating sole reciprocal fusions in advanced cancers, expanding the patient population potentially amenable to KIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6cc03ed5b4cf3438881405d76d4a3c612f2c0230" target='_blank'>
              Clinical relevance and druggability of sole reciprocal kinase fusions: A large‐scale study
              </a>
            </td>
          <td>
            Jiao Feng, Tonghui Ma, Chunyang Wang, Baoming Wang, Qian Liu, Zhengchuang Liu, H. Tao, Zaiyuan Ye
          </td>
          <td>2024-09-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Simple Summary Human cells are typically diploid, that is, they contain two sets of chromosomes. Some cells, however, contain three or more sets of chromosomes and are called polyploid. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of undergoing asymmetric division via nuclear budding and bursting (amitosis), giving rise to daughter cells that proliferate rapidly and can contribute to tumorigenesis (transformation of normal cells to a neoplastic state). This commentary provides an update on these intriguing discoveries. Abstract Cell division is crucial for the survival of living organisms. Human cells undergo three types of cell division: mitosis, meiosis, and amitosis. The former two types occur in somatic cells and germ cells, respectively. Amitosis involves nuclear budding and occurs in cells that exhibit abnormal nuclear morphology (e.g., polyploidy) with increased cell size. In the early 2000s, Kirsten Walen and Rengaswami Rajaraman and his associates independently reported that polyploid human cells are capable of producing progeny via amitotic cell division, and that a subset of emerging daughter cells proliferate rapidly, exhibit stem cell-like properties, and can contribute to tumorigenesis. Polyploid cells that arise in solid tumors/tumor-derived cell lines are referred to as polyploid giant cancer cells (PGCCs) and are known to contribute to therapy resistance and disease recurrence following anticancer treatment. This commentary provides an update on some of these intriguing discoveries as a tribute to Drs. Walen and Rajaraman.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/149407ed82c685a5b413633998b54affc7e6d4b2" target='_blank'>
              Amitotic Cell Division, Malignancy, and Resistance to Anticancer Agents: A Tribute to Drs. Walen and Rajaraman
              </a>
            </td>
          <td>
            R. Mirzayans, David Murray
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Herpesviruses are ubiquitous pathogens that cause a wide range of disease. Upon nuclear entry, their genomes associate with histones and chromatin modifying enzymes that regulate the progression of viral transcription and outcome of infection. While the composition and modification of viral chromatin has been extensively studied on bulk populations of infected cells by chromatin immunoprecipitation, this key regulatory process remains poorly defined at single-genome resolution. Here we use high-resolution quantitative imaging to investigate the spatial proximity of canonical and variant histones at individual Herpes Simplex Virus 1 (HSV-1) genomes within the first 90 minutes of infection. We identify significant population heterogeneity in the stable enrichment and spatial proximity of canonical histones (H2A, H2B, H3.1) at viral DNA (vDNA) relative to established promyelocytic leukaemia nuclear body (PML-NB) host factors that are actively recruited to viral genomes upon nuclear entry. We show the replication-independent histone H3.3/H4 chaperone Daxx to cooperate with PML to mediate the enrichment and spatial localization of variant histone H3.3 at vDNA that limits the rate of HSV-1 genome decompaction to restrict the progress of immediate-early (IE) transcription. This host response is counteracted by the viral ubiquitin ligase ICP0, which degrades PML to disperse Daxx and variant histone H3.3 from vDNA to stimulate the progression of viral genome expansion, IE transcription, and onset of HSV-1 replication. Our data support a model of intermediate and sequential histone assembly initiated by Daxx that limits the rate of HSV-1 genome decompaction independently of the stable enrichment of histones H2A and H2B at vDNA required to facilitate canonical nucleosome assembly. We identify HSV-1 genome decompaction upon nuclear infection to play a key role in the initiation and functional outcome of HSV-1 lytic infection, findings pertinent to the transcriptional regulation of many nuclear replicating herpesvirus pathogens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd10788fc22834b17fdae65c2c26cce49f7d5c7d" target='_blank'>
              Daxx mediated histone H3.3 deposition on HSV-1 DNA restricts genome decompaction and the progression of immediate-early transcription
              </a>
            </td>
          <td>
            Ashley P.E. Roberts, Anne Orr, V. Iliev, Lauren Orr, S. McFarlane, Zhousiyu Yang, I. Epifano, C. Loney, Milagros Collados Rodriguez, Anna R. Cliffe, Kristen L Conn, Chris Boutell
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background: Cervical cancer (CC) stands as a significant health threat to women globally, with high-risk human papillomaviruses as major etiologic agents. The DNA damage repair (DDR) protein topoisomerase I (TOP1) has been linked to various cancers, yet its distinct roles and mechanisms in CC are not fully elucidated. Methods: We investigated TOP1 expression in cervical intraepithelial neoplasia (CIN) and CC tissues utilizing qRT-PCR and IHC, correlating findings with patient prognosis. Subsequent knockdown studies were performed in vitro and in vivo to evaluate the influence of TOP1 on tumor growth, DNA repair, and inflammatory responses. Results: TOP1 was highly expressed in CIN and CC, negatively correlating with patient prognosis. Inhibition of TOP1 impeded CC cell growth and disrupted DNA repair. TOP1 was shown to regulate tumor-promoting inflammation and programmed death-ligand 1 (PD-L1) production in a cGAS-dependent manner. HPV oncoproteins E6 and E7 upregulated TOP1 and activated the cGAS-PD-L1 pathway. Conclusions: TOP1 acts as a DNA repair mediator, promoting CC development and immune evasion. Targeting the TOP1-cGAS-PD-L1 axis could be a potential therapeutic strategy for CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f71748837721c3a12ecd5a0bb258b1048a43b9" target='_blank'>
              Oncoproteins E6 and E7 upregulate topoisomerase I to activate the cGAS-PD-L1 pathway in cervical cancer development
              </a>
            </td>
          <td>
            Ying Luo, Mengda Niu, Yanfei Liu, Miaochang Zhang, Yuanyuan Deng, Dan Mu, Junfen Xu, Shiyuan Hong
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Ductal carcinoma in situ and invasive ductal carcinoma represent two stages of breast cancer progression. A multitude of studies have shown that genomic instability increases during tumor development, as manifested by higher mutation and copy number variation rates. The advent of single-cell and spatial transcriptomics has enabled the investigation of the subtle differences in cellular states during the tumor progression at single-cell level, thereby providing more nuanced understanding of the intercellular interactions within the solid tumor. However, the evolutionary trajectory of tumor cells and the establishment of the immunosuppressive microenvironment during breast cancer progression remain unclear. In this study, we performed an exploratory analysis of the single-cell sequencing dataset of 13 ductal carcinoma in situ and invasive ductal carcinoma samples. We revealed that tumor cells became more malignant and aggressive during their progression, and T cells transited to an exhausted state. The tumor cells expressed various coinhibitory ligands that interacted with the receptors of immune cells to create an immunosuppressive tumor microenvironment. Furthermore, spatial transcriptomics data confirmed the spatial colocalization of tumor and immune cells, as well as the expression of the coinhibitory ligand-receptor pairs. Our analysis provides insights into the cellular and molecular mechanism underlying the formation of the immunosuppressive landscape during two typical stages of breast cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e0688a86cba179d2efd47f09e2fca2a5bb1e682" target='_blank'>
              Single-cell and Spatial Transcriptomic Analyses Implicate Formation of the Immunosuppressive Microenvironment during Breast Tumor Progression.
              </a>
            </td>
          <td>
            Fengfeng Cai, YuanYuan Li, Hui Liu, Judong Luo
          </td>
          <td>2024-09-16</td>
          <td>Journal of immunology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="TBC1D3 is a primate-specific gene family that has expanded in the human lineage and has been implicated in neuronal progenitor proliferation and expansion of the frontal cortex. The gene family and its expression have been challenging to investigate because it is embedded in high-identity and highly variable segmental duplications. We sequenced and assembled the gene family using long-read sequencing data from 34 humans and 11 non-human primate species. Our analysis shows that this particular gene family has independently duplicated in at least five primate lineages, and the duplicated loci are enriched at sites of large-scale chromosomal rearrangements on Chromosome 17. We find that all human copy number variation maps to two distinct clusters located at Chromosome 17q12 and that humans are highly structurally variable at this locus, differing by as many as 20 copies and ~1 Mbp in length depending on haplotypes. We also show evidence of positive selection, as well as a significant change in the predicted human TBC1D3 protein sequence. Lastly, we find that, despite multiple duplications, human TBC1D3 expression is limited to a subset of copies and, most notably, from a single paralog group: TBC1D3-CDKL These observations may help explain why a gene potentially important in cortical development can be so variable in the human population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872db936ea87de98bc9c479ae7fb31da39d2f13a" target='_blank'>
              Independent expansion, selection and hypervariability of the TBC1D3 gene family in humans.
              </a>
            </td>
          <td>
            Xavi Guitart, David Porubsky, DongAhn Yoo, Max L. Dougherty, Philip C. Dishuck, Katherine M. Munson, Alexandra P. Lewis, Kendra Hoekzema, Jordan Knuth, Stephen Chang, Tomi Pastinen, Evan E. Eichler
          </td>
          <td>2024-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Non-viral gene editing via CRISPR-Cas12a offers an alternative to Cas9-based methods, providing better targeting of AT-rich regions, simplified guide RNA manufacturing, and high specificity. However, the efficacy of editing outcomes is subject to various factors, with template format playing a crucial role. Currently, the predominant non-viral template format for inducing homology-directed repair (HDR) after nuclease-induced DNA breaks is double-stranded DNA (dsDNA), which is toxic when transfected at high doses. Previous studies have demonstrated that using single-stranded DNA (ssDNA) with flanking double-stranded Cas-target-sequences (CTS) as a repair template for Cas9-mediated gene editing can mitigate this toxicity and increase knock-in efficiency. Here, we investigate CTS design for AsCas12a Ultra by exploring PAM orientation and binding requirements of the Cas12a-crRNA complex. Additionally, we rule out in vitro ssDNase activity of AsCas12a Ultra under cell-physiological Mg2+ conditions. Finally, we showcase the advantage of using ssDNA with double-stranded CTS end modifications (ssCTS) at high doses for delivering clinically relevant transgenes of varying sizes into three T-cell receptor-CD3 complex genes (TRAC, CD3ζ, CD3ε), achieving up to 90% knock-in rates for a 0.8kb insert at the CD3ε locus. Overall, AsCas12a Ultra and ssCTS donors represent a platform for highly efficient knock-in in primary human T cells with minimal toxicity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fa53cac4bc809cec9ab1898c0c41bd74b95519e" target='_blank'>
              Single-stranded HDR templates with truncated Cas12a binding sequences improve knock-in efficiencies in primary human T cells
              </a>
            </td>
          <td>
            Ana-Maria Nitulescu, Weijie Du, Viktor Glaser, Jonas Kath, Robert Greensmith, N. S. Mikkelsen, Maik Stein, R. Bak, Michael M Kaminski, D. Wagner
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdd7f6016d3ca163a2a60df3216dfee270ad74c8" target='_blank'>
              The plant early recombinosome: a high security complex to break DNA during meiosis.
              </a>
            </td>
          <td>
            N. Rafiei, Arnaud Ronceret
          </td>
          <td>2024-09-27</td>
          <td>Plant reproduction</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10bd05b394b6759a7d6959abc8009eb518a19281" target='_blank'>
              Multifaceted role of the DNA replication protein MCM10 in maintaining genome stability and its implication in human diseases.
              </a>
            </td>
          <td>
            Sumayyah M. Q. Ahmed, Jayaprakash Sasikumar, Suparna Laha, Shankar Prasad Das
          </td>
          <td>2024-09-06</td>
          <td>Cancer metastasis reviews</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background Single nucleotide polymorphisms (SNPs) in DNA repair genes can impair protein function and hinder DNA repair, leading to genetic instability and increased cancer risk. The Excision Repair Cross-Complementation (ERCC) family plays a crucial role in nucleotide excision repair, yet their comprehensive multi-omics characterization and roles in tumor prognosis and immune microenvironment remain unexplored. Methods and materials We performed bioinformatics analysis using publicly available data from 33 cancer types to investigate associations between ERCC gene expression, patient prognosis, and clinical features. We also validated the role of ERCC2 in bladder cancer through in vitro assays, including CCK-8, colony formation, wound healing, and Transwell assays. Results By utilizing the most recent database, we have conducted an analysis that reveals associations between variations in ERCC expression across multiple cancer types and both patient prognosis and the tumor microenvironment. To ensure the reliability of our findings, we applied the Benjamini-Hochberg procedure to adjust for multiple testing. After correction, we identified that ERCC expression levels remained significantly correlated with patient prognosis in various cancer types (p < 0.05). In addition, according to the results of drug sensitivity studies of anticancer drugs, there is a large correlation between ERCC expression and the sensitivity of different anticancer drugs. Finally, in vitro cell behavioral assays determined that knockdown of ERCC2 gene expression significantly inhibited the proliferation, migration and invasion of bladder cancer cells. Conclusion Through in-depth exploration of ERCC differential expression and its correlation with immune-related indicators, the unique microenvironment of tumors, and patient prognosis, we verified the potential role of ERCC2 in the process of bladder cancer genesis and progression. Therefore, we believe that the ERCC family of genes is expected to be a new option for cancer treatment and deserves to be further explored in the future.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7956b050d97190f79bdb4b65156dfa093429bf2" target='_blank'>
              Bioinformatics analysis of ERCC family in pan-cancer and ERCC2 in bladder cancer
              </a>
            </td>
          <td>
            Siyang Zhang, Zhenghui Guan, Qiangqiang Xia, Chong Shen, Hailong Hu, Jiangping Wang
          </td>
          <td>2024-08-13</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdf251e653d85e2d3fa3eb31d13a78d3467c5985" target='_blank'>
              Bioinformatics advances in eccDNA identification and analysis.
              </a>
            </td>
          <td>
            Fuyu Li, Wenlong Ming, Wenxiang Lu, Ying Wang, Xianjun Dong, Yunfei Bai
          </td>
          <td>2024-08-29</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="BACKGROUND
Many plant secondary metabolites (PSMs) were shown to intercalate into DNA helix or interact with DNA grooves. This may influence histone-DNA interactions changeing chromatin structure and genome functioning.


METHODS
Nucleosome stability and linker histone H1.2, H1.4 and H1.5 localizations were studied in HeLa cells after the treatment with 15 PSMs, which are DNA-binders and possess anticancer activity according to published data. Chromatin remodeler CBL0137 was used as a control. Effects of PSMs were studied using fluorescent microscopy, flowcytometry, quantitative reverse transcriptase-polymerase chain reaction (RT-qPCR), western-blotting.


RESULTS
We showed that 1-hour treatment with CBL0137 strongly inhibited DNA synthesis and caused intensive linker histone depletion consistent with nucleosome destabilization. None of PSMs caused nucleosome destabilization, while most of them demonstrated significant influence on linker histone localizations. In particular, cell treatment with 11 PSMs at non-toxic concentrations induced significant translocation of the histone H1.5 to nucleoli and most of PSMs caused depletion of the histones H1.2 and H1.4 from chromatin fraction. Curcumin, resveratrol, berberine, naringenin, and quercetin caused significant redistribution of all three variants of the studied linker histones showing some overlap of PSM effects on linker histone DNA-binding. We demonstrated that PSMs, which induced the most significant redistribution of the histone H1.5 (berberine, curcumin and naringenin), influence the proportion of cells synthesizing DNA, expressing or non-expressing cyclin B and influence cell cycle distribution. Berberine induction of H1.5 translocations to nucleoli was shown to occur independently on the phases of cell cycle (metaphase was not analyzed).


CONCLUSIONS
For the first time we revealed PSM influence on linker histone location in cell nuclei that opens a new direction of PSM research as anticancer agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31259052a1bbdcbe810f26e91f0f54bf14b49fe4" target='_blank'>
              Anticancer Plant Secondary Metabolites Induce Linker Histone Depletion from Chromatin.
              </a>
            </td>
          <td>
            O. Vlasova, Irina Antonova, Roman Zenkov, Denis Naberezhnov, Gennady Belitsky, Anna Borunova, Tatiana Zabotina, Daniel García-Gomis, A. Safina, Katerina Gurova, A. Gudkov, Kirill Kirsanov, Albert Jordan, Marianna Yakubovskaya
          </td>
          <td>2024-08-05</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Transcription of actively expressed genes is dampened for kilobases around DNA lesions via chromatin modifications. This is believed to favour repair and prevent genome instability. Nonetheless, mounting evidence suggests that transcription may be induced by DNA breakage, resulting in the local de novo synthesis of non-coding RNAs (ncRNAs). Such transcripts have been proposed to play important functions in both DNA damage signalling and repair. Here, we review the recently identified mechanistic details of transcriptional silencing at damaged chromatin, highlighting how post-translational histone modifications can also be modulated by the local synthesis of DNA damage-induced ncRNAs. Finally, we envision that these entangled transcriptional events at DNA breakages can be targeted to modulate DNA repair, with potential implications for locus-specific therapeutic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8045eca3080065650673ed1f8657736a081879a" target='_blank'>
              Traffic light at DSB-transit regulation between gene transcription and DNA repair.
              </a>
            </td>
          <td>
            Stefania Modafferi, Francesca Esposito, Sara Tavella, Ubaldo Gioia, S. Francia
          </td>
          <td>2024-09-27</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 SMARCB1 is a tumor suppressor gene and encodes a core subunit of the SWI-SNF chromatin remodeling complex. Loss of SMARCB1 leads to the development of highly aggressive pediatric malignancies, termed “SMARCB1-deficient cancers.” This includes malignant rhabdoid tumor, atypical teratoid rhabdoid tumor (ATRT), and renal medullary carcinoma (RMC). Survival rates in these patients remain low, and resistance to chemotherapeutic agents used to treat these cancers are common. One such chemotherapy is doxorubicin. Therefore, this project aims to reveal and understand the mechanisms of doxorubicin resistance in SMARCB1-deficient cancers. To address this question, we generated doxorubicin resistant (DR) cells from one ATRT and two RMC primary cell lines (BT16, PEDS005T, and RCRF1009T respectively) utilizing a dose escalation approach. This resulted in DR cell lines that are 10-fold to 30-fold resistant compared to the parental cell line. Upon generation of these stably resistant cell lines, we profiled their transcriptomes by RNA-sequencing and found an upregulation of the multidrug transporter ABCB1, also known as MDR1. To further probe these cell lines, we performed whole-genome sequencing (WGS) on two of our DR cell lines (BT16 and PEDS005T DR). Here, we found a chromosome 7 aneuploidy in BT16 DR cells, but no aneuploidy or mutations were seen in PEDS005T DR cells. ABCB1 is located on chromosome 7. We confirmed this aneuploidy via karyotyping. The mechanism of ABCB1 upregulation in the RMC DR cell lines remains unknown, suggesting multiple modalities leading to ABCB1 upregulation. We then overexpressed ABCB1 in two ATRT parental cell lines (BT16 and CHLA-06-ATRT) and one RMC cell line (PEDS005T) which led to enhanced short-term resistance. This contrasted to JMSU, a SMARCB1 wild-type bladder cancer cell line, where there was stable resistance. Together, this suggests that ABCB1 may be necessary but not sufficient to confer doxorubicin resistance in these SMARCB1-deficient cancer cells. These preliminary findings suggest that despite a common genetic driver (loss of SMARCB1), there may be different pathways to developing resistance to the DNA-damaging agent, doxorubicin. Furthermore, despite the well documented role of ABCB1 in drug resistance, there may be other factors involved in conferring stable resistance in the setting of SMARCB1-deficient cancers.
 Citation Format: Katie T. Skinner, Benjamin P. Lee, Jessica S Yoon, Andrew L. Hong. The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A069.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdc2732137e2a0d6d27507359d94e24f1e66d2f" target='_blank'>
              Abstract A069 The role of ABCB1 in doxorubicin resistant SMARCB1-deficient cancers
              </a>
            </td>
          <td>
            Katie T. Skinner, Benjamin P Lee, Jessica S Yoon, Andrew L. Hong
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Rationale: Tumor cells remodel transcriptome to construct an ecosystem with stemness features, which maintains tumor growth and highly malignant characteristics. However, the core regulatory factors involved in this process still need to be further discovered. Methods: Single cell RNA-sequncing (scRNA-seq) and bulk RNA-sequencing profiles derived from fetal liver, normal liver, liver tumors, and their adjacent samples were collected to analyze the ecosystem of liver cancer. Mouse models were established to identify molecular functions of oncofetal-related oncogenes using hydrodynamic tail vein injection. Results: We found that liver cancer rebuilt oncofetal ecosystem to maintain malignant features. Interestingly, we identified a group of RNA-binding proteins (RBPs) that were highly overexpressed with oncofetal features. Among them, TRIM71 was specifically expressed in liver cancers and was associated with poor outcomes. TRIM71 drove the carcinogenesis of hepatocellular carcinoma (HCC), and knockdown of TRIM71 significantly abolished liver cancer cell proliferation. Mechanistically, TRIM71 formed a protein complex with IGF2BP1, bound to and stabilized the mRNA of CEBPA in an m6A-dependent manner, enhance the serine/glycine metabolic pathway, and ultimately promoted liver cancer progression. Furthermore, we identified that all-trans-retinoic acid (ATRA) combined with e1A binding protein p300 (EP300) inhibitor A-485 repressed TRIM71, attenuated glycine/serine metabolism, and inhibited liver cancer cell proliferation with high TRIM71 levels. Conclusions: We demonstrated the oncofetal status in liver cancer and highlighted the crucial role of TRIM71 and provided potential therapeutic strategies and liver cancer-specific biomarker for liver cancer patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9ce6c5051059c8c5192cce1ee450847798b9ec45" target='_blank'>
              Oncofetal TRIM71 drives liver cancer carcinogenesis through remodeling CEBPA-mediated serine/glycine metabolism
              </a>
            </td>
          <td>
            Ying Su, Ziteng Li, Qin Li, Xinyi Guo, He-Yun Zhang, Yan Li, Zhiqiang Meng, Shenglin Huang, Zhixiang Hu
          </td>
          <td>2024-08-12</td>
          <td>Theranostics</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Aflatoxin B1 (AFB1), a potent mycotoxin, is one of the environmental risk factors that cause liver cancer. In the liver, the bioactivated AFB1 intercalates into the DNA double helix to form a bulky DNA adduct which will lead to mutation if left unrepaired. Here, we adapted the tXR-seq method to measure the nucleotide excision repair of AFB1-induced DNA adducts at single-nucleotide resolution on a genome-wide scale, and compared it with repair data obtained from conventional UV-damage XR-seq. Our results showed that transcription-coupled repair plays a major role in the damage removal process. We further analyzed the distribution of nucleotide excision repair sites for AFB1-induced DNA adducts within the 3D human genome organization. Our analysis revealed a heterogeneous AFB1-dG repair across four different organization levels, including chromosome territories, A/B compartments, TADs, and chromatin loops. We found that chromosomes positioned closer to the nuclear center and regions within A compartments have higher levels of nucleotide excision repair. Notably, we observed high repair activity around both TAD boundaries and loop anchors. These findings provide insights into the complex interplay between AFB1-induced DNA damage repair, transcription, and 3D genome organization, shedding light on the mechanisms underlying AFB1-induced mutagenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d6fed0c3af4e712a403c0af2af9bb447bf31d09" target='_blank'>
              Nucleotide excision repair of aflatoxin-induced DNA damage within the 3D human genome organization.
              </a>
            </td>
          <td>
            Yiran Wu, Muhammad Muzammal Adeel, Dian Xia, Aziz Sancar, Wentao Li
          </td>
          <td>2024-09-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Glioma represents the most common primary central nervous system (CNS) neoplasm in adults. IDH1 mutation is recognized as a hallmark alteration with important diagnostic and prognostic implications. There is considerable evidence for global DNA hypermethylation induced secondary to IDH mutation. However, there is limited understanding of the RNA methylation patterns and its role in glioma biology. In this study, we performed transcriptome wide profiling of N6-methyladenosine (m6A) modifications across IDH mutant (n = 8) and wild-type (n = 7) gliomas using Oxford Nanopore Technologies direct RNA sequencing platform. Our approach enabled high-depth coverage of native transcripts, revealing nearly twice as many full-length transcripts in IDH mutant gliomas compared to wild-type. Notable differences in alternative splicing were observed across glioma subtypes, with truncated and non-coding isoforms more prevalent in glioblastoma (GBM). We further identified significant changes in isoform usage within key metabolic (NAMPT, PKM) and immune (CD63, CD151, CD81) pathways. Chromosomal distribution of m6A sites showed a higher prevalence of m6A modifications in IDH mutant gliomas, with the most pronounced differences on chromosomes 19 and 16. Further stratification by TERT, MGMT, and TP53 mutations revealed similar patterns of increased m6A site numbers in mutant groups, highlighting the importance of integrating epigenomic and epitranscriptomic data in glioma research. These findings highlight the role of m6A modifications in the metabolic reprogramming unique to IDH mutant gliomas, providing insights into potential mechanisms of tumorigenesis and therapeutic resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a097c511887c9900da059382e760aacd6b331c36" target='_blank'>
              Epitranscriptome Mapping of m6A RNA Modifications in Glioma Tumor Tissue
              </a>
            </td>
          <td>
            S. Batool, Saad Murtaza Khan, K. Muralidharan, Ana K Escobedo, Hanna Lee, Emil Ekanyake, Tiffaney Hsia, Bob S Carter, L. Balaj, Cambridge St, MA Boston
          </td>
          <td>2024-09-25</td>
          <td>None</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Genome rearrangement is a hallmark of cancer and ageing. DNA double-strand breaks (DSBs) are highly toxic lesions that can generate genome rearrangements. Several pathways compete for DSB repair, and selection of the appropriate repair process is decisive for genetic stability. DSB repair act according to two steps: 1-the choice between nonhomologous end-joining (NHEJ) versus resection of DNA ends, generating single-stranded DNA (ssDNA); 2-on ssDNA, homologous recombination (HR) and sub-processes. Here we show that 53BP1, which plays a prime role at the first step by protecting against resection and fostering NHEJ, physically interacts with the catalytic subunits of the cAMP dependant protein kinase A (PKAcs). PKA favors the recruitment of 53BP1 at DNA damages sites, and consistently, prevents resection, favoring NHEJ. Inhibition of PKA stimulated resection and coherently reduces NHEJ. Reversely, activation of PKA with 8-Bromo-cAMP stimulate NHEJ and reduces HR. These data open new avenues for potential anticancer strategies. More generally, they underline the high complexity of the regulation of DSB repair, identifying PKA as a novel actor of the DNA damage response, acting at the choice of the DSB repair process, which might be essential for genome integrity maintenance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fcb615918d35bbed71f2c3e0b64876e126043052" target='_blank'>
              PKA prevents resection of DNA double strand breaks and favor non-homologous end-joining
              </a>
            </td>
          <td>
            Houaria Khater-Sellou, I. Magdalou, Vasily Ogryzko, Bernard S. Lopez
          </td>
          <td>2024-08-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Humanized rodent models, especially humanization of genetic/genomic components involved in immunity have significantly advanced our understanding of human immune system. Here, we utilized trans-chromosomic (Tc) technology to generate a TC-mAb rat model that stably harbors a mouse artificial chromosome carrying full-length human immunoglobulin (Ig) heavy and kappa light chain genes (IGHK-NAC) in a rat Ig knockout background. In contrast with TC-mAb mice, serum human IgG concentration was found higher than IgM. Number of lymphocytes was recovered, and B cell population in the spleen was normal. Remarkably, repertoire analysis revealed similarities between the model and human PBMCs; somatic hypermutation and class-switch recombination also more closely resembled humans. Furthermore, immunization resulted in generation of antigen-specific human antibodies. Collectively, our strategy to generate both rat and mouse models through introduction of the identical IGHK-NAC offers unprecedented opportunities to comprehensively evaluate genomic regulation and its outcomes associated with genomic sequences and host-derived protein factors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aade4b17ea2ad65e9633a9f7cb9607cbc453286" target='_blank'>
              Comparative analysis of trans-chromosomic rodent models reveals improved somatic hypermutation and class-switch recombination in rats
              </a>
            </td>
          <td>
            H. Satofuka, S. Abe, Takashi Moriwaki, Akane Okada, K. Kazuki, S. Hamamichi, M. Hiratsuka, Masumi Hirabayashi, Kazuomi Nakamura, Tetsushi Sakuma, Takashi Yamamoto, Yoshihiro Baba, Kazuma Tomizuka, Yasuhiro Kazuki
          </td>
          <td>2024-08-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="A variety of newly developed next-generation sequencing technologies are making their way rapidly into the research and clinical applications, for which accuracy and cross-lab reproducibility are critical, and reference standards are much needed. Our previous multicenter studies under the SEQC-2 umbrella using a breast cancer cell line with paired B-cell line have produced a large amount of different genomic data including whole genome sequencing (Illumina, PacBio, Nanopore), HiC, and scRNA-seq with detailed analyses on somatic mutations, single-nucleotide variations (SNVs), and structural variations (SVs). However, there is still a lack of well-characterized reference materials which include epigenomic and proteomic data. Here we further performed ATAC-seq, Methyl-seq, RNA-seq, and proteomic analyses and provided a comprehensive catalog of the epigenomic landscape, which overlapped with the transcriptomes and proteomes for the two cell lines. We identified >7,700 peptide isoforms, where the majority (95%) of the genes had a single peptide isoform. Protein expression of the transcripts overlapping CGIs were much higher than the protein expression of the non-CGI transcripts in both cell lines. We further demonstrated the evidence that certain SNVs were incorporated into mutated peptides. We observed that open chromatin regions had low methylation which were largely regulated by CG density, where CG-rich regions had more accessible chromatin, low methylation, and higher gene and protein expression. The CG-poor regions had higher repressive epigenetic regulations (higher DNA methylation) and less open chromatin, resulting in a cell line specific methylation and gene expression patterns. Our studies provide well-defined reference materials consisting of two cell lines with genomic, epigenomic, transcriptomic, scRNA-seq and proteomic characterizations which can serve as standards for validating and benchmarking not only on various omics assays, but also on bioinformatics methods. It will be a valuable resource for both research and clinical communities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5405253ebdcaa518e6236a2ec5d5b3c004462522" target='_blank'>
              Epigenomic, transcriptomic and proteomic characterizations of reference samples
              </a>
            </td>
          <td>
            Chirag Nepal, Wanqiu Chen, Zhong Chen, John Wrobel, Ling Xie, Wenjing Liao, Chunlin Xiao, Adrew Farmer, Malcolm Moos, W. Jones, Xian Chen, Charles Wang
          </td>
          <td>2024-09-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bca0012968a9023f1a5b2833b62edce18fdae052" target='_blank'>
              LncRNA DNAH17-AS1 promotes gastric cancer proliferation and radioresistance by sponging miR-202-3p to upregulate ONECUT2
              </a>
            </td>
          <td>
            Yugang Ge, Hui Cang, J-Q Xiao, Hongshuai Wu, Biao Wang, Qing Shao
          </td>
          <td>2024-09-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="CRISPR-Associated Transposases (CASTs) hold tremendous potential for microbial genome editing due to their ability to integrate large DNA cargos in a programmable and site-specific manner. However, the widespread application of CASTs has been hindered by their low efficiency in diverse, non-model bacteria. In an effort to address this shortcoming, we conducted the first genome-wide screen for host factors impacting Vibrio cholerae CAST (VchCAST) activity and used the findings to increase VchCAST editing efficiency. A genome-wide loss-of-function mutant library in E. coli was screened to identify 15 genes that impact type VchCAST transposition. Of these, seven factors were validated to improve VchCAST activity and two were found to be inhibitory. Informed by homologous recombination involved effectors, RecD and RecA, we tested the λ-Red recombineering system in our VchCAST editing vectors, which increased its insertion meditated-editing efficiency by 25.7-fold in E. coli while maintaining high target specificity and similar insertion arrangements. Furthermore, λ-Red-enhanced VchCAST achieved increased editing efficiency in the industrially important bacteria Pseudomonas putida and the emerging pathogen Klebsiella michiganensis. This study improves understanding of factors impacting VchCAST activity and enhances its efficiency as a bacterial genome editor. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc2483d77579dd4ab8c5f977694f46b22d04f0a6" target='_blank'>
              Identification of Proteins Influencing CRISPR-Associated Transposases for Enhanced Genome Editing
              </a>
            </td>
          <td>
            Leo Song, Amanda T.P. Alker, Agnès Oromí-Bosch, Sophia E. Swartz, Jonathan N.V. Martinson, Jigyasa Arora, Abby M. Wang, Rachel Rovinsky, Sara J. Smith, Emily C. Pierce, A. Deutschbauer, Jennifer A. Doudna, Brady F. Cress, Benjamin E. Rubin
          </td>
          <td>2024-09-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="
 Medulloblastoma (MB) is the most prevalent malignant brain tumor in children that demonstrates clinical and genomic heterogeneity. Of the four major subgroups, Group 3 tumors (MYC-MB), and particularly the 3gsubtype, exhibit high levels of MYC, high rates of metastasis, and have the worst prognosis. Despite treatment with surgery, radiation and chemotherapy, Group 3 medulloblastoma patients are more likely to develop recurrent tumors which are far more aggressive resulting in a 5-year survival rate after recurrence of 12%. Thus, there is an imperative need to examine the underlying mechanisms mediating treatment resistance. Recent evidence has identified cellular plasticity, as a major driver of therapy resistance. To investigate whether epigenetic mediated cell plasticity contributes to Myc-MB therapy response we performed Single Nuclei Multiome ATAC + Gene Expression analysis of paired primary and relapsed tumors. We identified the emergence of novel progenitor populations in resistant tumors with stem-like gene expression networks and altered chromatin accessibility. These emergent popuoations demostrated new enhancer-promoer interactions and exhibited increased metabolic reprogramming with potential for theraputic targeting. Because radiation is one of the key treatment modalities for Myc-MB, we chose to model radiation resistance in further studies. Single-cell RNA sequencing of outgrown in vivo irradiated orthotopic xenograft MB tumors showed an overall upregulation of metabolic pathways with the enhancement of genes that mediate oxidative phosphorylation and glutamate metabolism centering on wild-type IDH1 (wtIDH1), consistent with our patient data. IR-resistant cells showed altered chromatin occupancy of H3K3me3 and H3K27Ac. Disruption of IDH1 activity in IR resistant cells in vitro altered the epigenomic landscape, facilitating reversion to the pre-irradiated state. Interestingly, we observed that pluripotent genes with bivalent regions (H3K4me3 and H3K27me3) were enhanced in radiation-resistant cells and suppressed by IDH1 inhibition. Metabolic mass spectrometry reveled that metabolites from the citric acid cycle and synthesis of unsaturated fatty acids were among the main metabolically elevated molecules in radiation-resistant cells consistnat with the observed epigenomic reprograming. IDH1 inhibition reversed the metabolic reprograming and resensitized MB cells to radiation. In total our data identify a novel mechanism of radiation resistance in medulloblastoma with potential theraputic implications.
 Citation Format: Bethany Veo, Dong Wang, John Desisto, Rajeev Vibhakar. Mechanisms of radiation resistance in medulloblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A071.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaed4654cb7c14fb325db4bb243c211a30ba0383" target='_blank'>
              Abstract A071 Mechanisms of radiation resistance in medulloblastoma
              </a>
            </td>
          <td>
            Bethany L. Veo, Dong Wang, J. DeSisto, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5a77337f38c8a116c6d75b10e0eda38d81a91a" target='_blank'>
              ΔNp63 bookmarks and creates an accessible epigenetic environment for TGFβ-induced cancer cell stemness and invasiveness
              </a>
            </td>
          <td>
            Eleftheria Vasilaki, Yu Bai, Mohamad Moustafa Ali, Anders Sundqvist, A. Moustakas, C. Heldin
          </td>
          <td>2024-08-23</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>155</td>
        </tr>

        <tr id="Studies of bacterial adaptation and evolution are hampered by the difficulty of measuring traits such as virulence, drug resistance, and transmissibility in large populations. In contrast, it is now feasible to obtain high-quality complete assemblies of many bacterial genomes thanks to scalable high-accuracy long-read sequencing technologies. To exploit this opportunity, we introduce a phenotype- and alignment-free method for discovering coselected and epistatically interacting genomic variation from genome assemblies covering both core and accessory parts of genomes. Our approach uses a compact colored de Bruijn graph to approximate the intragenome distances between pairs of loci for a collection of bacterial genomes to account for the impacts of linkage disequilibrium (LD). We demonstrate the versatility of our approach to efficiently identify associations between loci linked with drug resistance and adaptation to the hospital niche in the major human bacterial pathogens Streptococcus pneumoniae and Enterococcus faecalis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ba999d01a93235712556c85e81e52a6ac50e10" target='_blank'>
              Pangenome-spanning epistasis and coselection analysis via de Bruijn graphs
              </a>
            </td>
          <td>
            Juri Kuronen, Samuel T. Horsfield, A. K. Pöntinen, S. Mallawaarachchi, S. Arredondo-Alonso, Harry A. Thorpe, R. Gladstone, Rob J. L. Willems, Stephen D. Bentley, N. Croucher, J. Pensar, John A. Lees, G. Tonkin-Hill, Jukka Corander
          </td>
          <td>2024-07-01</td>
          <td>Genome Research</td>
          <td>0</td>
          <td>51</td>
        </tr>

        <tr id="DNA double-strand breaks (DSBs) are highly detrimental to cells, as improper repair can result in inheritable genetic rearrangements or cell death. The role of RNA:DNA hybrids (RDHs) in DSB repair remains poorly understood, but their transient accumulation and subsequent resolution are crucial for accurate repair. The absence of the end-joining factor Nej1 at DSBs significantly reduced RDH levels, which was linked to increased activity of the Dna2 nuclease. Dna2 limits the accumulation of hybrids at DSBs, with levels rising in the presence of a nuclease-dead Dna2 and RNH201 deletion. Dna2 has a heightened preference for resolving RDHs with 5’ RNA overhangs compared to duplex substrates with 5’ DNA overhangs. This selective resolution by Dna2 helps restrict hybrid accumulation at DSBs and promotes resection, a function not shared by Exo1. This study underscores the multifunctional roles of canonical repair factors in ensuring efficient homology-directed repair. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5b2cacdb3d323609fc9f634d2ea46c9da9e5cdc" target='_blank'>
              Dna2 nuclease resolves RNA:DNA hybrids at double-strand breaks
              </a>
            </td>
          <td>
            A. Mojumdar, Courtney Granger, Jennifer A. Cobb
          </td>
          <td>2024-08-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Over the years the spread of HIV-1 across the globe resulted in the creation of multiple subtypes and new recombinant forms (RFs). While the pol gene region of the HIV-1 genome is used for resistance mutations analysis and initial detection of recombinants, whole genome sequencing analysis is required to determine recombination events across the viral genome. Here, we present a novel robust near-full length genome (NFLG) sequencing approach for the sequencing of HIV-1 genomes out of clinical whole blood samples. This method has been successfully tested for various HIV-1 subtypes and RFs. The method is based on an in-house developed set of 32 pan-genotypic primer pairs, divided into two pools, each containing 16 primer pairs covering the entire HIV-1 genome. Two parallel multiplex PCR reactions were used to generate 32 overlapping DNA fragments spanning the HIV-1 genome. Nextera XT protocol was used to obtain barcoded libraries which were run on the Illumina Miseq platform using a V3 kit. A consensus sequence was determined for each sample and was used to define recombination events across the genome. For that aim, a combined analysis of several computational tools including HIV BLAST, phylogenetic analysis, RIP, SimPlot++ and jpHMM was employed. Overall, plasma samples from 33 patients suspected to carry RFs and 2 different, known pure subtypes controls, were included in this study. Overall coverage varied between different RFs. While the gag and pol genes were highly covered in all samples analyzed, the highly variable env gene region was not fully covered. Yet, these NFLG analyses enabled the identification of recombination events across the viral genome, with the identification of CRFs and RFs such as CRF01_AE, CRF02_AG, BF1, BC, A6B, CA1 and some potential URFs. In summary, we describe a methodology for HIV-1 NFLG sequencing, which is based on multiple and partially overlapping PCR fragments spanning the genome of HIV-1. This method enables sequencing of 20 different patient samples in a single MiSeq run and was used for the characterization of different HIV-1 RFs and pure subtypes circulating in Israel. Additionally, this newly refined method was compared to HIV-1 NFLG results of PCR-free metagenomic sequencing approach and proved to obtain higher number of covered bases of the HXB2 reference genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/569f10e9f73098e9324aa6f1b16181f512f570d2" target='_blank'>
              Novel Amplicons based Method for Near-Full-Length Genome (NFLG) Sequencing of HIV-1 Group M Recombinant Forms
              </a>
            </td>
          <td>
            R. Moscona, T. Wagner, Miranda Geva, E. Bucris, O. Erster, N. Zuckerman, Orna Mor
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4568f027a9bf73e7d22e0523c166497f5edf7122" target='_blank'>
              Perturbing TET2 condensation promotes aberrant genome-wide DNA methylation and curtails leukaemia cell growth.
              </a>
            </td>
          <td>
            Lei Guo, Tingting Hong, Yi-Tsang Lee, Xue Hu, Guokai Pan, Rongjie Zhao, Yuhan Yang, Jingwen Yang, Xiaoli Cai, Logan Rivera, Jie Liang, Rui Wang, Yaling Dou, Srikanth Kodali, Wenbo Li, Leng Han, Bruno Di Stefano, Yubin Zhou, Jia Li, Yun Huang
          </td>
          <td>2024-09-09</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="DNA methylation at the 5 position of cytosine (5mC) is an ancient epigenetic mark in eukaryotes. The levels of total 5mC vary enormously between different species, and the DNA methyltransferases that introduce 5mC have been repeatedly lost in several independent lineages. DNA methyltransferases are a threat to genomic stability due to the increased mutagenicity of 5mC bases and the propensity of DNA methyltransferases themselves to introduce DNA alkylation damage as an off-target effect. However, whether alkylation damage explains why 5mC is frequently lost in evolution is unclear. Here we tested the fitness consequences of DNA methyltransferase-induced alkylation damage by introducing a eukaryotic-like 5mC system into E. coli. We showed that introducing 5mC genome-wide leads to increased sensitivity to alkylating agents, which is strongly enhanced by removal of the 3mC repair enzyme AlkB. Unexpectedly, we discovered that 5mC introduction led to increased sensitivity to oxidative stress. We showed that this is due to increased formation of reactive oxygen in the presence of 5mC. We determined that reactive oxygen species led to non-enzymatic oxidation of 5mC, producing modified cytosines such as 5fC that are recognised as DNA base damage in E. coli. Overall, our work identifies increased sensitivity to oxidative stress, as well as alkylating agents, as a negative consequence of genome-wide 5mC. Oxidative stress is frequently encountered by organisms in their environment, thus offering a plausible reason for total loss of 5mC in some species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe9e3ad69751a1fed99ef603d00526bf9e51599" target='_blank'>
              Introduction of cytosine-5 DNA methylation sensitizes cells to oxidative damage
              </a>
            </td>
          <td>
            J. Krwawicz, C.J. Sheeba, K. Hains, T. McMahon, Y. Zhang, S. Kriaučionis, P. Sarkies
          </td>
          <td>2024-09-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Genome instability is a characteristic trait of tumours and includes changes in DNA and in chromosomes. The aim of the study was to identify chromosome damage using the sister chromatid exchange assay and DNA fragmentation by the comet assay in dogs with cancer, as well as to determine the suitability of these techniques for the assessment of chromatin stability in healthy and sick dogs. The assays identified genomic instabilities in dogs with cancer (squamous cell carcinoma) and in healthy dogs. The genetic assays are very sensitive and can be used as biomarkers of normal DNA replication and repair potential and the maintenance of control over the entire cell cycle. The use of the cytogenetic tests will enable the more precise assessment of genome stability and integrity in animals and make it possible to determine the number of chromosomal instabilities generated in a given individual, which can be indicative of its health status. The identification of instabilities can be used in routine diagnostic examination in dogs with cancer for more accurate diagnosis and prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a7548d405e861a202a9ad5c3f6e2cc2fac0cc66" target='_blank'>
              Genomic Instability in the Lymphocytes of Dogs with Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Ewa Wójcik, Emilia Kot, Iga Wójcik, A. Wysokińska, P. Matusevičius
          </td>
          <td>2024-09-24</td>
          <td>Animals</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407d56e1f37d6b7ede8b27130702ccb149beda6e" target='_blank'>
              Multiomics and machine learning-based analysis of pancancer pseudouridine modifications
              </a>
            </td>
          <td>
            Jiheng Zhang, Lei Xu, Xiuwei Yan, Jiahe Hu, Xin Gao, Hongtao Zhao, Mo Geng, Nan Wang, Shaoshan Hu
          </td>
          <td>2024-08-20</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Cytogenetic information about the product of conception (POC) is important to determine the presence of recurrent chromosomal abnormalities that are an indication for preimplantation genetic testing for aneuploidy or structural rearrangements. Although microscopic examination by G-staining has long been used for such an evaluation, detection failures are relatively common with this method, due to cell-culture-related issues. The utility of low-coverage whole-genome sequencing (lcWGS) using short-read next-generation sequencing (NGS) has been highlighted recently as an alternative cytogenomic approach for POC analysis. We, here, performed comparative analysis of two NGS-based protocols for this purpose based on different short-read sequencers (the Illumina VeriSeq system using a MiSeq sequencer and the Thermo Fisher ReproSeq system using an Ion S5 sequencer). The cytogenomic diagnosis obtained with each NGS method was equivalent in each of 20 POC samples analyzed. Notably, X chromosome sequence reads were reduced in some female samples with both systems. The possibility of low-level mosaicism for monosomy X as an explanation for this was excluded by FISH analysis. Additional data from samples with various degrees of X chromosome aneuploidy suggested that it was a technical artifact related to X chromosome inactivation. Indeed, subsequent nanopore sequencing indicated that the DNA in the samples showing the artifact was predominantly unmethylated. Our current findings indicate that although X chromosome data must be interpreted with caution, both the systems we tested for NGS-based lcWGS are useful alternatives for the karyotyping of POC samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/244f7b9f408d5f90077c203078c04f62d033674c" target='_blank'>
              Comparative Analysis of Two NGS-Based Platforms for Product-of-Conception Karyotyping
              </a>
            </td>
          <td>
            Yuri Murase, Yui Shichiri, H. Inagaki, T. Nakano, Y. Nakaoka, Y. Morimoto, Tomoko Ichikawa, H. Nishizawa, Eiji Sugihara, Hiroki Kurahashi
          </td>
          <td>2024-08-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Mobilization of transposable elements (TEs) can generate large effect mutations. However, because new TE insertions are challenging to detect and transposition is typically rare, the actual rate and landscape of new insertions remains unexplored for most TEs. Results Here, we introduce a TE display sequencing approach that leverages target amplification of TE extremities to detect non-reference TE insertions with high sensitivity and specificity. By implementing this approach on serial dilutions of genomic DNA from A. thaliana lines carrying different repertoires of new TE insertions, we show that the method can detect TE insertions that are present at frequencies as low as 1:250 000 within a DNA sample. In addition, TE display sequencing can be multiplexed to simultaneously detect insertions for distinct TE families, including both retrotransposons and DNA transposons, increasing its versatility and cost-effectiveness to investigate complex “mobilomes”. Importantly, when combined with nanopore sequencing, this approach enables the identification of insertions using long-reads and achieves a turn around time from DNA extraction to insertion identification of less than 24h, significantly reducing the time-to-answer. Analysis of TE insertions in large populations of A. thaliana plants undergoing a transposition burst demonstrate the power of the multiplex TE display sequencing to assess the rates and allele frequencies of heritable insertions, enabling its implementation to study large-scale ‘evolve and resequence’ experiments. Furthermore, we found that ∼6% of de novo TE insertions show recurrent allele frequency changes consistent with either positive or negative selection. Conclusions TE display sequencing is an ultra-sensitive, specific, quick, and cost-effective approach to investigate the rate and landscape of new insertions for multiple TEs in large scale population experiments. We provide a step-by-step experimental protocol as well as ready-to-use bioinformatic pipelines, ensuring straightforward implementation of the method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d225ed0cb2ff38da801717215b18fc7567f3912a" target='_blank'>
              Ultra-Sensitive Detection of Transposon Insertions Across Multiple Families by Transposable Element Display Sequencing
              </a>
            </td>
          <td>
            Pol Vendrell-Mir, Basile Leduque, Leandro Quadrana
          </td>
          <td>2024-08-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genetic variations and DNA modification are two common dominant factors ubiquitous across the entire human genome and induce human disease, especially through static genetic variations in DNA or RNA that cause human genetic diseases. DNA N6-methyladenosine (6mA) methylation, as a new epigenetic modification mark, has been widely studied for regulatory biological processes in humans. However, the effect of DNA modification on dynamic transcriptional genetic variations from DNA to RNA has rarely been reported. Here, we identified DNA, RNA and transcriptional genetic variations from Illumina short-read sequencing data in East Asian samples (HX1 and AK1) and detected global DNA 6mA modification using single-molecule, real-time sequencing (SMRT) data. We decoded the effects of DNA 6mA modification on transcriptional genetic variations in East Asian samples and the results were extensively verified in the HeLa cell line. DNA 6mA modification had a stabilized distribution in the East Asian samples and the methylated genes were less likely to mutate than the non-methylated genes. For methylated genes, the 6mA density was positively correlated with the number of variations. DNA 6mA modification had a selective effect on transcriptional genetic variations from DNA to RNA, in which the dynamic transcriptional variations of heterozygous (0/1 to 0/1) and homozygous (1/1 to 1/1) were significantly affected by 6mA modification. The effect of DNA methylation on transcriptional genetic variations provides new insights into the influencing factors of DNA to RNA transcriptional regulation in the central doctrine of molecular biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0dd1403e82b438ee3e0a09ce9c885d1dfcf61ad" target='_blank'>
              Selective Effect of DNA N6-Methyladenosine Modification on Transcriptional Genetic Variations in East Asian Samples
              </a>
            </td>
          <td>
            Meiwei Luan, Kaining Chen, Wenwen Zhao, Minqiang Tang, Lingxia Wang, Shoubai Liu, Linan Zhu, Shangqian Xie
          </td>
          <td>2024-09-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667549573815f8b84f3de00bdda0de7e5af3d623" target='_blank'>
              Emerin mislocalization during chromatin bridge resolution can drive prostate cancer cell invasiveness in a collagen-rich microenvironment.
              </a>
            </td>
          <td>
            M. Popęda, Kamil Kowalski, Tomasz Wenta, Galina V Beznoussenko, Michał Rychłowski, Alexander A. Mironov, Zeno Lavagnino, Sara Barozzi, Julia Richert, Rebecca Bertolio, Kamil Myszczyński, J. Szade, Michał Bieńkowski, Kevin Miszewski, Marcin Matuszewski, A. Żaczek, Luca Braga, Giannino Del Sal, N. Bednarz-Knoll, P. Maiuri, P. Nastały
          </td>
          <td>2024-09-02</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Breast cancer (BC) is a prevailing malignancy among women, and its inconspicuous development contributes significantly to mortality. The RNA N6-methyladenosine (m6A) modification represents an emerging mechanism for gene expression regulation, with the active involvement of the YTH N6-methyladenosine RNA binding protein 3 (YTHDF3) in tumor progression across multiple cancer types. Nonetheless, its precise function in breast cancer necessitates further investigation.The expression of YTHDF3 in both cell lines and patient tissues was examined using Western blotting, reverse transcription quantitative PCR (RT-qPCR), and immunohistochemistry (IHC) techniques. Bioinformatics analysis of methylated RNA immunoprecipitation sequencing (MeRIP-seq) and transcriptome RNA sequencing (RNA-seq) data was employed to screen for the target genes of YTHDF3. The main focus of this study was to investigate the in vitro biological functions of YTHDF3. The specific binding of YTHDF3 to its target genes and its correlation with m6A methylation were studied through RNA pull-down, RNA immunoprecipitation, and co-immunoprecipitation experiments. The protein regulatory mechanisms of downstream genes of YTHDF3 were assessed using protein stability analysis. Furthermore, the biological functions of YTHDF3 and its target genes in breast cancer cells were validated through CRISPR-Cas9 technology and rescue experiments.By constructing a risk model using the TCGA database, YTHDF3 was identified as a high-risk factor among m6A methylation factors. Subsequent investigations revealed its elevated expression in various subtypes of breast cancer, accompanied by poor prognosis. MeRIP-seq analysis further revealed fibroblast growth factor 2 (FGF2) as a downstream gene of YTHDF3. Knockdown of YTHDF3 in breast cancer cells led to significant inhibition of cell self-renewal, migration, and invasion abilities in vitro. Mechanistically, YTHDF3 specifically recognized the methylated transcript of FGF2 within its coding sequence (CDS) region, leading to the inhibition of FGF2 protein degradation. Moreover, depletion of FGF2 markedly suppressed the biological functions of breast cancer cells, while reducing FGF2 expression in YTHDF3-overexpressing breast cancer cell lines substantially alleviated the malignant progression.In summary, our study elucidates the role of YTHDF3 as an oncogene in maintaining FGF2 expression in BC cells through an m6A-dependent mechanism. Additionally, we provide a potential biomarker panel for prognostic prediction in BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4a36ec9c5649a0b1a66054fa0cef4c652b513ff" target='_blank'>
              YTHDF3 modulates the progression of breast cancer cells by regulating FGF2 through m6A methylation
              </a>
            </td>
          <td>
            R. F. Gong, Z. H. Zhang, T. T. Sun, Y. X. Zhao, Wen Fang
          </td>
          <td>2024-09-20</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84febdf4c77dd510d0a0709248848e1311db0643" target='_blank'>
              Biological and clinical relevance of correlated expression levels of coding and long noncoding RNAs in HPV16 positive cervical cancers
              </a>
            </td>
          <td>
            Abhisikta Ghosh, Abarna Sinha, Arnab Ghosh, Somrita Roy, Sumana Mallick, Vinoth Kumar, S. Mathai, Jaydip Bhaumik, Asima Mukhopadhyay, Saugata Sen, Aditi Chandra, A. Maitra, N. Biswas, Partha P. Majumder, Sharmila Sengupta
          </td>
          <td>2024-08-29</td>
          <td>Human Genomics</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Satellite DNAs (satDNAs) are tandemly repeated sequences that make up a significant portion of almost all eukaryotic genomes. Although satDNAs have been shown to play a very important role in genome organization and evolution, they are relatively poorly analysed even in model. One of the main reasons for the current lack of in-depth studies on satDNAs is their underrepresentation in genome assemblies. The complexity and highly repetitive nature of satDNAs make their analysis challenging, and there is a need for efficient tools that can ensure accurate annotation and analysis of satDNAs. We present a novel pipeline, named Satellite DNA Exploration (SatXplor), designed to robustly characterize satDNA elements and analyse their arrays and flanking regions. SatXplor is benchmarked against curated satDNA datasets from diverse species, showcasing its versatility across genomes with varying complexities and different satDNA profile. Component algorithms excel in the identification of tandemly repeated sequences and for the first time enable evaluation of satDNA variation and array annotation with the addition of information about surrounding genomic landscape. SatXplor is an innovative pipeline for satDNA analysis that can be paired with any tool used for satDNA detection, offering insights into the structural characteristics, array determination and genomic context of satDNA elements. By integrating various computational techniques, from sequence analysis and homology investigation to advanced clustering and graph-based methods, it provides a versatile and comprehensive approach to explore the complexity of satDNA organization and to understand the underlying mechanisms and evolutionary aspects. It is open-source and freely accessible at https://github.com/mvolar/SatXplor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93bcbb026ce597eb6926d4d48baa8708101680" target='_blank'>
              SatXplor – A comprehensive pipeline for satellite DNA analyses in complex genome assemblies
              </a>
            </td>
          <td>
            Marin Volarić, N. Meštrović, Evelin Despot-Slade
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba675eb14cd995fdc082910df7aeef7f5f1a429b" target='_blank'>
              Immune landscape of oncohistone-mutant gliomas reveals diverse myeloid populations and tumor-promoting function
              </a>
            </td>
          <td>
            Augusto Faria Andrade, Alva Annett, Elham Karimi, D. G. Topouza, Morteza Rezanejad, Yitong Liu, Michael McNicholas, Eduardo G Gonzalez Santiago, Dhana Llivichuzhca-Loja, Arne Gehlhaar, Selin Jessa, Antonella De Cola, Bhavyaa Chandarana, Caterina Russo, D. Faury, Geoffroy Danieau, Evan Puligandla, Yuhong Wei, M. Zeinieh, Qing Wu, S. Hébert, N. Juretic, Emily M. Nakada, Brian Krug, Valérie Larouche, Alexander G Weil, Roy W. R. Dudley, Jason Karamchandani, S. Agnihotri, D. Quail, B. Ellezam, Liza Konnikova, Logan A Walsh, Manav Pathania, Claudia L Kleinman, Nada Jabado
          </td>
          <td>2024-09-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Simple Summary Forkhead-box M1 (FOXM1) is a proliferation-associated transcription factor, overexpressed in almost all the cancers, making its regulation worth investigating. FOXM1 is known to bind to the promoters of certain microRNAs, a class of biomolecules involved in post-transcriptional gene regulation. Database mining followed by Venn analysis led us to the shortlisting of seven microRNAs, among which microRNA-532-3p showed the strongest interaction with FOXM1 3’UTR. The ectopic overexpression of miR-532-3p in colorectal cancer (CRC) cells led to decreased FOXM1 levels which resulted in the alteration of CRC functionalities. These altered phenotypes were supported by a change in the expression of proliferation and EMT markers. In conclusion, the present study suggested that the regulation of FOXM1 by microRNA-532-3p via its interaction with FOXM1 3’UTR inhibited CRC proliferation, migration, and invasion. Abstract Colorectal cancer (CRC) is a heterogeneous disease and classified into various subtypes, among which transcriptional alterations result in CRC progression, metastasis, and drug resistance. Forkhead-box M1 (FOXM1) is a proliferation-associated transcription factor which is overexpressed in CRC and the mechanisms of FOXM1 regulation have been under investigation. Previously, we showed that FOXM1 binds to promoters of certain microRNAs. Database mining led to several microRNAs that might interact with FOXM1 3’UTR. The interactions between shortlisted microRNAs and FOXM1 3’UTR were quantitated by a dual-luciferase reporter assay. MicroRNA-532-3p interacted with the 3’UTR of the FOXM1 mRNA transcript most efficiently. MicroRNA-532-3p was ectopically overexpressed in colorectal cancer (CRC) cell lines, leading to reduced transcript and protein levels of FOXM1 and cyclin B1, a direct transcriptional target of FOXM1. Further, a clonogenic assay was conducted in overexpressed miR-532-3p CRC cells that revealed a decline in the ability of cells to form colonies and a reduction in migratory and invading potential. These alterations were reinforced at molecular levels by the altered transcript and protein levels of the conventional EMT markers E-cadherin and vimentin. Overall, this study identifies the regulation of FOXM1 by microRNA-532-3p via its interaction with FOXM1 3’UTR, resulting in the suppression of proliferation, migration, and invasion, suggesting its role as a tumor suppressor in CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/affbc7e03338ebd318b510b5965cce7c6dfa9311" target='_blank'>
              MicroRNA-532-3p Modulates Colorectal Cancer Cell Proliferation and Invasion via Suppression of FOXM1
              </a>
            </td>
          <td>
            Nita Ahuja, Akira Yamauchi, David Wong, Ketakee Mahajan, Ani V. Das, S. Alahari, Ramesh Pothuraju, S. A. Nair
          </td>
          <td>2024-09-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="DNA damage tolerance pathways that allow for the completion of replication following fork arrest are critical in maintaining genome stability during cell division. The main DNA damage tolerance pathways include strand switching, replication fork reversal and translesion synthesis (TLS). The TLS pathway is mediated by specialised DNA polymerases that can accommodate altered DNA structures during DNA synthesis, and are important in allowing replication to proceed after fork arrest, preventing fork collapse that can generate more deleterious double-strand breaks in the genome. TLS may occur directly at the fork, or at gaps remaining behind the fork, in the process of post-replication repair. Inactivating mutations in the human POLH gene encoding the Y-family DNA polymerase Pol η causes the skin cancer-prone genetic disease xeroderma pigmentosum variant (XPV). Pol η also contributes to chemoresistance during cancer treatment by bypassing DNA lesions induced by anti-cancer drugs including cisplatin. We review the current understanding of the canonical role of Pol η in translesion synthesis following replication arrest, as well as a number of emerging non-canonical roles of the protein in other aspects of DNA metabolism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/247363b13f865e5ca1882a279d7c40f1f96c114f" target='_blank'>
              Canonical and Non-Canonical Roles of Human DNA Polymerase η
              </a>
            </td>
          <td>
            Salma Bedaiwi, Anam Usmani, Michael P. Carty
          </td>
          <td>2024-09-27</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Myxofibrosarcoma (MFS) and undifferentiated pleomorphic sarcoma (UPS) are two common and aggressive subtypes of soft tissue sarcoma. The aim of this study was to assess potential transcriptomic differences between MFS and UPS tumours and to evaluate the extent to which differences in gene expression profiles were associated with genomic and clinical features. The study included 162 patients with tumours diagnosed as MFS (N = 62) or UPS (N = 100). The patients had been diagnosed and treated at two Swedish sarcoma centres during a 30-year period. For gene expression profiling and gene fusion detection all tumours were analysed using RNA-sequencing and could be compared with data on clinical outcome (N = 155), global copy number profiles (N = 145), and gene mutations (N = 128). Gene expression profiling revealed three transcriptomic clusters (TCs) without any clear separation of MFS and UPS. One TC was associated with longer metastasis-free survival. These tumours had lower tumour mutation burden (TMB), were enriched for a copy number signature representative of focal LOH and chromosomal instability on a diploid background, and were relatively immune-depleted. MFS and UPS showed extensive genomic overlap, with whole genome doubling occurring more frequently among the latter. The results support the idea that MFS and UPS tumours have largely overlapping genomic and transcriptomic features, with UPS tumours showing more aggressive behaviour and more complex genomes. Independently of the tumour type, clinically relevant subgroups were revealed by gene expression analysis, and the finding of multiple genomic subgroups strongly suggest the existence of subgroups of relevance to treatment stratification. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cdad99fbec5c71439ba17d6c812cff6bcce80b" target='_blank'>
              Transcriptomic profiles of myxofibrosarcoma and undifferentiated pleomorphic sarcoma correlate with clinical and genomic features.
              </a>
            </td>
          <td>
            Shamik Mitra, Akanksha Farswan, Paul Piccinelli, Saskia Sydow, A. Hesla, P. Tsagkozis, F. Vult von Steyern, Martin Almqvist, Mikael Eriksson, L. Magnusson, J. Nilsson, N. Pillay, Fredrik Mertens
          </td>
          <td>2024-09-11</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Characterizing DNA methylation patterns is important for addressing key questions in evolutionary biology, geroscience, and medical genomics. While costs are decreasing, whole-genome DNA methylation profiling remains prohibitively expensive for most population-scale studies, creating a need for cost-effective, reduced representation approaches (i.e., assays that rely on microarrays, enzyme digests, or sequence capture to target a subset of the genome). Most common whole genome and reduced representation techniques rely on bisulfite conversion, which can damage DNA resulting in DNA loss and sequencing biases. Enzymatic methyl sequencing (EM-seq) was recently proposed to overcome these issues, but thorough benchmarking of EM-seq combined with cost-effective, reduced representation strategies has not yet been performed. To do so, we optimized Targeted Methylation Sequencing protocol (TMS)—which profiles ∼4 million CpG sites—for miniaturization, flexibility, and multispecies use at a cost of ∼$80. First, we tested modifications to increase throughput and reduce cost, including increasing multiplexing, decreasing DNA input, and using enzymatic rather than mechanical fragmentation to prepare DNA. Second, we compared our optimized TMS protocol to commonly used techniques, specifically the Infinium MethylationEPIC BeadChip (n=55 paired samples) and whole genome bisulfite sequencing (n=6 paired samples). In both cases, we found strong agreement between technologies (R² = 0.97 and 0.99, respectively). Third, we tested the optimized TMS protocol in three non-human primate species (rhesus macaques, geladas, and capuchins). We captured a high percentage (mean=77.1%) of targeted CpG sites and produced methylation level estimates that agreed with those generated from reduced representation bisulfite sequencing (R² = 0.98). Finally, we applied our protocol to profile age-associated DNA methylation variation in two subsistence-level populations—the Tsimane of lowland Bolivia and the Orang Asli of Peninsular Malaysia—and found age-methylation patterns that were strikingly similar to those reported in high income cohorts, despite known differences in age-health relationships between lifestyle contexts. Altogether, our optimized TMS protocol will enable cost-effective, population-scale studies of genome-wide DNA methylation levels across human and non-human primate species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8ea49a3598e0f07df9c655927729fa7dc601194" target='_blank'>
              Cost-effective solutions for high-throughput enzymatic DNA methylation sequencing
              </a>
            </td>
          <td>
            Amy Longtin, Marina M. Watowich, Baptiste Sadoughi, Rachel M. Petersen, Sarah F. Brosnan, Kenneth Buetow, Qiuyin Cai, M. Gurven, Heather M. Highland, Yi-Ting Huang, Hillard Kaplan, Thomas S Kraft, Yvonne A L Lim, J. Long, Amanda D. Melin, Jamie Roberson, Kee-Seong Ng, J. Stieglitz, B. Trumble, Vivek V Venkataraman, Ian J Wallace, Jie Wu, N. Snyder‐Mackler, Angela Jones, Alexander G. Bick, Amanda J. Lea
          </td>
          <td>2024-09-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Chromatin organization is important for gene transcription in pig genome. However, its three-dimensional (3D) structure and dynamics are much less investigated than those in human. Here we applied the long-reads chromatin interaction analysis by paired-end tag sequencing (ChIA-PET) method to map the whole-genome chromatin interactions mediated by CCCTC-binding factor (CTCF) and RNA polymerase Ⅱ (RNAPⅡ or POLⅡ) in porcine macrophage cells before and after polyinosinic-polycytidylic acid [Poly(I:C)] induction. Our results revealed that Poly(I:C) induction impacts the 3D genome organization in the 3D4/21 cells at the fine-scale chromatin loop level rather than at the large-scale domain level. Furthermore, our findings underscored the pivotal role of CTCF anchored chromatin interactions in reshaping chromatin architecture during immune responses. Knock-out of the CTCF locus further confirmed that the CTCF anchored enhancers are associated with the activation of immune genes via long-range interactions. Notably, ChIA-PET data also supported the spatial relationship between single nucleotide polymorphisms (SNPs) and the related gene transcription in 3D genome aspect. Our findings in this study provide new clues and potential targets to explore key elements related to diseases in swine and are also likely to shed light on elucidating chromatin organization and dynamics underlying the process of mammalian infectious diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97442a763cf352589738944e6bb5a4d860334082" target='_blank'>
              Virus Infection Induces Immune Gene Activation with CTCF Anchored Enhancers and Chromatin Interactions in Pig Genome.
              </a>
            </td>
          <td>
            Jianhua Cao, Ruimin Ren, Xiaolong Li, Xiaoqian Zhang, Yan Sun, Xiaohuan Tian, Ru Liu, Xiangdong Liu, Yijun Ruan, Guoliang Li, Shuhong Zhao
          </td>
          <td>2024-09-23</td>
          <td>Genomics, proteomics & bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bba6936c2ac2d76bcee0a653b32c7b9ef4038fa" target='_blank'>
              Yeast EndoG prevents genome instability by degrading extranuclear DNA species
              </a>
            </td>
          <td>
            Yang Yu, Xin Wang, Jordan Fox, Ruofan Yu, P. Thakre, Brenna S. McCauley, Nic Nikoloutsos, Yang Yu, Qian Li, P. Hastings, Weiwei Dang, Kaifu Chen, Grzegorz Ira
          </td>
          <td>2024-09-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Phenotypic variability in isogenic bacterial populations helps them to cope with external stress. This variability can stem from gene expression noise and/or unequal partitioning of low-copy-number freely diffusing proteins during cell division. Some high-copy-number components are transiently associated with almost immobile large assemblies (hyperstructures) and thereby may not be distributed equally. We hypothesize that the hyperstructure organization of bacterial cells can contribute to bacterial phenotypic variability. We focus on nucleoid hyperstructure containing many DNA-associated proteins. Our previous findings suggested that replication initiation timing may be the main source of variability for nucleoid segregation events. To uncover the role of the initiator DnaA in phenotypic variability in E. coli, we followed its intra- and intercellular distribution using chromosomally encoded fluorescently tagged DnaA and histone-like HU under their native promoters. The mobility of DnaA is low, corresponding to a diffusion-binding mode, but still one order faster than that of HU. The intracellular distribution of DnaA concentration is homogeneous in contrast to the significant asymmetry in the distribution of HU; the latter leading to unequal DNA content of nucleoids in future daughter compartments. The intercellular variability of HU (CV=26%), DnaA, and their ratio (both with CV=18%) is high. The variable DnaA/DNA may cause a variable replication initiation time (initiation noise). Asynchronous initiation at different oriCs may be, in turn, a plausible origin of the asymmetric DNA distribution. Our findings indicate that the DnaA/DNA ratio is the feature determining the variability of initiation time in E. coli rather than each of them separately. Importance Phenotypic variability is an important phenomenon of bacterial physiology related to adaptation to environmental stresses and antibiotic resistance, not yet completely understood. This work further examines our hypothesis regarding the role of hyperstructure organization of bacterial cells in this phenomenon, focusing on the nucleoid hyperstructure and the DNA-associated replication initiator DnaA. Exploiting fluorescently tagged proteins, we analyze intra- and intercellular distributions, mobility, and concentration variabilities of DnaA and DNA in the population of Escherichia coli. The data obtained indicate that the DnaA/DNA ratio may define the timing of replication initiation event and the ensuing chromosome segregation. As a result, newborn cells with different DnaA/DNA ratios will display variability in the main cell cycle characteristics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2d2069c6e7f009db748eca4a82f0ae5bcf52f6c" target='_blank'>
              Relative Distribution of DnaA and DNA in Escherichia coli Cells as a Factor of Their Phenotypic Variability
              </a>
            </td>
          <td>
            Sharanya K. Namboodiri, A. Aranovich, U. Hadad, Levi Geber, Mario Feingold, I. Fishov
          </td>
          <td>2024-09-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48137767fd26c2f7051bdcf540a74692bdb8a20b" target='_blank'>
              The role of hexokinases in epigenetic regulation: altered hexokinase expression and chromatin stability in yeast
              </a>
            </td>
          <td>
            Srinivasu Karri, Quinn Dickinson, Jing Jia, Yi Yang, Haiyun Gan, Zhiquan Wang, Yibin Deng, Chuanhe Yu
          </td>
          <td>2024-08-27</td>
          <td>Epigenetics & Chromatin</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Genome integration technology has markedly expedited the construction of cell factories. However, its application is currently limited by the inefficient integration of large DNA fragments. Here, we report a CRISPR/Cas9 based integration of large DNA fragments (CILF) method to efficiently integrate large DNA fragments in Saccharomyces cerevisiae. In this approach, a fusion protein, Cas9-Brex27-FadR, was employed for the targeted delivery of donor plasmid to double-strand breaks (DSBs), while simultaneously recruiting Rad51 to enhance the efficiency of homologous recombination (HR). Our findings demonstrate that this method can achieve an integration efficiency of 98% for 10 kb DNA fragments and nearly 80% for 40 kb DNA fragments at a single site, using donor plasmids with 1000 bp homology arms (HAs) and 12 FadR binding sites (BSs). The CILF technique significantly enriches the synthetic biology toolbox of S. cerevisiae, offering significant potential to propel advancements in both synthetic biology and metabolic engineering.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0e29b2dc3cc477a76f6b6e9a8ba280b35bb76869" target='_blank'>
              CILF: CRISPR/Cas9 based integration of large DNA fragments in Saccharomyces cerevisiae.
              </a>
            </td>
          <td>
            Shijie Xu, Jie Meng, Qi Zhang, Baisong Tong, Zihe Liu, Jinyu Fu, S. Shi
          </td>
          <td>2024-08-18</td>
          <td>Biotechnology and bioengineering</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7c1e4b7568509ca6d857531ee73ea8a0c9b0585" target='_blank'>
              Decoding the genomic landscape of chromatin-associated biomolecular condensates
              </a>
            </td>
          <td>
            Zhaowei Yu, Qi Wang, Qichen Zhang, Yawen Tian, Guo Yan, Jidong Zhu, Guangya Zhu, Yong Zhang
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7aed930352660788bbfcd1cc9c93c4e4f947acaa" target='_blank'>
              PD-L1 deglycosylation promotes its nuclear translocation and accelerates DNA double-strand-break repair in cancer
              </a>
            </td>
          <td>
            Zhen Shu, Bhakti Dwivedi, Jeffrey M. Switchenko, David S Yu, Xingming Deng
          </td>
          <td>2024-08-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction Accurate reference genomes are fundamental to understanding biological evolution, biodiversity, hereditary phenomena and diseases. However, many assembled nuclear chromosomes are often contaminated by organelle genomes, which will mislead bioinformatic analysis, and genomic and transcriptomic data interpretation. Methods To address this issue, we developed a tool named Chlomito, aiming at precise identification and elimination of organelle genome contamination from nuclear genome assembly. Compared to conventional approaches, Chlomito utilized new metrics, alignment length coverage ratio (ALCR) and sequencing depth ratio (SDR), thereby effectively distinguishing true organelle genome sequences from those transferred into nuclear genomes via horizontal gene transfer (HGT). Results The accuracy of Chlomito was tested using sequencing data from Plum, Mango and Arabidopsis. The results confirmed that Chlomito can accurately detect contigs originating from the organelle genomes, and the identified contigs covered most regions of the organelle reference genomes, demonstrating efficiency and precision of Chlomito. Considering user convenience, we further packaged this method into a Docker image, simplified the data processing workflow. Discussion Overall, Chlomito provides an efficient, accurate and convenient method for identifying and removing contigs derived from organelle genomes in genomic assembly data, contributing to the improvement of genome assembly quality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2949fcd41e909221ac38423a6ad3cf2f0aef961d" target='_blank'>
              Chlomito: a novel tool for precise elimination of organelle genome contamination from nuclear genome assembly
              </a>
            </td>
          <td>
            Wei Song, Chong Li, Yanming Lu, Dawei Shen, Yunxiao Jia, Yixin Huo, Weilan Piao, Hua Jin
          </td>
          <td>2024-08-27</td>
          <td>Frontiers in Plant Science</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64123a29f4e4d91a84682b8e0733680d1726f975" target='_blank'>
              Phase separation of phospho-HDAC6 drives aberrant chromatin architecture in triple-negative breast cancer.
              </a>
            </td>
          <td>
            Bing Lu, Ru Qiu, Jiatian Wei, Li Wang, Qinkai Zhang, Mingsen Li, Xiudan Zhan, Jian Chen, I-yun Hsieh, Ciqiu Yang, Jing Zhang, Zicheng Sun, Yifan Zhu, Tao Jiang, Han Zhu, Jie Li, Wei Zhao
          </td>
          <td>2024-08-28</td>
          <td>Nature cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/311bc754350204df7366f1cc87322a6aee4bc5a6" target='_blank'>
              Assessment of SWI/SNF chromatin remodeling complex related genes as potential biomarkers and therapeutic targets in pan-cancer
              </a>
            </td>
          <td>
            Kai Zhuang, Lishan Wang, Chengyu Lu, Zhiping Liu, Dongli Yang, Hao Zhong, Jiami Zou, Aamir Fahira, Jiaojiao Wang, Zunnan Huang
          </td>
          <td>2024-08-27</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b02e26ddc26802806307ff2a8acf516e9dc75bb" target='_blank'>
              Exosomal transcript cargo and functional correlation with HNSCC patients’ survival
              </a>
            </td>
          <td>
            Joni Yadav, A. Chaudhary, Tanya Tripathi, Divya Janjua, Udit Joshi, N. Aggarwal, Arun Chhokar, Chetkar Chandra Keshavam, Anna Senrung, A. Bharti
          </td>
          <td>2024-09-13</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Somatic mitochondrial DNA (mtDNA) mutations are prevalent in tumors, yet defining their biological significance remains challenging due to the intricate interplay between selective pressure, heteroplasmy, and cell state. Utilizing bulk whole-genome sequencing data from matched tumor and normal samples from two cohorts of pediatric cancer patients, we uncover differences in the accumulation of synonymous and nonsynonymous mtDNA mutations in pediatric leukemias, indicating distinct selective pressures. By integrating single-cell sequencing (SCS) with mathematical modeling and network-based systems biology approaches, we identify a correlation between the extent of cell-state changes associated with tumor-enriched mtDNA mutations and the selective pressures shaping their distribution among individual leukemic cells. Our findings also reveal an association between specific heteroplasmic mtDNA mutations and cellular responses that may contribute to functional heterogeneity among leukemic cells and influence their fitness. This study highlights the potential of SCS strategies for distinguishing between pathogenic and passenger somatic mtDNA mutations in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a627b6a95aa1e7df9a6c11d1a1fe81399b864950" target='_blank'>
              Somatic mitochondrial DNA mutations are a source of heterogeneity among primary leukemic cells
              </a>
            </td>
          <td>
            K. McCastlain, C. Welsh, Y. Ni, L. Ding, M. Franco, R. Autry, B. Sejdiu, T.-C. Chang, W. Chen, H. Wu, Q. Pan, V. Gonzalez-Pena, P. Schreiner, S. Arunachalam, J. Joo, B. Li, , S. Brady, J. Zhang, C. Gawad, W. Evans, M. M. Babu, K. Khrapko, G. Wu, J. Yu, S. Pounds, M. Kundu
          </td>
          <td>2024-09-27</td>
          <td>None</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Glioblastoma cell lines derived from different patients are widely used in tumor biology research and drug screening. A key feature of glioblastoma is the high level of inter- and intratumor heterogeneity that accounts for treatment resistance. Our aim was to investigate whether intratumor heterogeneity is maintained in cell models. Single-cell RNA sequencing was used to investigate the cellular composition of a tumor sample and six patient-derived glioblastoma cell lines. Three cell lines preserved the mutational profile of the original tumor, whereas three others differed from their precursors. Copy-number variation analysis showed significantly rearranged genomes in all the cell lines and in the tumor sample. The tumor had the most complex cell composition, including cancer cells and microenvironmental cells. Cell lines with a conserved genome had less diverse cellularity, and during cultivation, a relative increase in the stem-cell-derived progenitors was noticed. Cell lines with genomes different from those of the primary tumors mainly contained neural progenitor cells and microenvironmental cells. The establishment of cell lines without the driver mutations that are intrinsic to the original tumors may be related to the selection of clones or cell populations during cultivation. Thus, patient-derived glioblastoma cell lines differ substantially in their cellular profile, which should be taken into account in translational studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/777ddb1f8f83bf7d55d6fc7f2f07b867d23bc980" target='_blank'>
              Intratumoral Cell Heterogeneity in Patient-Derived Glioblastoma Cell Lines Revealed by Single-Cell RNA-Sequencing
              </a>
            </td>
          <td>
            Mikhail Arbatskiy, Dmitriy Balandin, A. Churov, V. Varachev, Eugenia Nikolaeva, Alexei Mitrofanov, Ali Bekyashev, O. Tkacheva, O. Susova, T. Nasedkina
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>1</td>
          <td>8</td>
        </tr>

        <tr id="Noncoding single nucleotide variants are the predominant class of genetic variation in whole genome sequencing and are key drivers of phenotypic variation. However, their functional annotation remains challenging. To address this, we develop a hypothesis-driven functional annotation scheme for CTCF binding sites given CTCF’s critical roles in gene regulation and extensive profiling in regulatory datasets. We synthesize CTCF’s binding patterns at 1,063,879 genomic loci across 214 biological contexts into a summary metric, which we refer to as binding activity. We find that binding activity is significantly enriched for both conserved nucleotides (Pearson R = 0.31, p < 2.2 x 10-16) and sequences that contain high-quality CTCF binding motifs (Pearson R = 0.63, p = 2.9 x 10-12). We then integrate binding activity with high confidence change in precision weight matrix scores. By applying this framework to 1,253,330 SNVs in gnomAD, we explore signatures of selection acting against the disruption of CTCF binding. We find a strong, positive relationship between the mutability adjusted proportion of singletons (MAPS) metric and the loss of CTCF binding at loci with high in vitro activity (Pearson R = 0.67, p = 1.5 x 10-14). To contextualize these findings, we apply MAPS to other functional classes of variation and find that a subset of 198,149 loss of CTCF binding variants are observed as infrequently as missense variants. This work implicates these thousands of rare, noncoding variants that disrupt CTCF binding for further functional studies while providing a blueprint for the interpretable annotation of noncoding variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea605606df9f3fb36737c3abacedd277ab50869b" target='_blank'>
              Identifying deleterious noncoding variation through gain and loss of CTCF binding activity
              </a>
            </td>
          <td>
            Colby Tubbs, Mary Lauren Benton, Evonne McArthur, John A. Capra, Douglas M. Ruderfer
          </td>
          <td>2024-09-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Single-cell genomic analysis enables researchers to gain novel insights across diverse research areas, including developmental biology, tumor heterogeneity, and disease pathogenesis. Conducting single-cell genomic analysis using next-generation sequencing (NGS) methods has traditionally been challenging as the amount of genomic DNA present in a single cell is limited. Advancements in multiple displacement amplification (MDA) technologies allow the unbiased amplification of limited quantities of DNA under conditions that maintain its integrity. This method of amplification results in high yield and facilitates the generation of high-complexity NGS libraries that ensure the highest coverage to effectively allow variant calling. With the introduction of new sequencing platforms and chemistry, whole genome sequencing became a more cost-effective application, but enrichment of specific regions of interest further reduces the amount of required sequencing output and associated costs. There are two enrichment methods, polymerase chain reaction (PCR)-based and hybrid-capture-based methods. PCR-based methods are very flexible and highly effective but focus on specific loci, typically known to be associated with disease. Inherited diseases of unknown genetic origin require a more comprehensive approach to capture the genetic variation that is not yet associated with a specific disease. Hybrid capture enrichment methods require considerable amounts of DNA such that exome enrichment from single cells is only possible after preamplification of this limited material. This article describes the complete workflow from single cells and small quantities of DNA to exome-NGS libraries for Illumina sequencing instruments and includes the following protocols: © 2024 The Author(s). Current Protocols published by Wiley Periodicals LLC. Basic Protocol 1: Whole genome amplification from single cells or small amounts of gDNA Basic Protocol 2: NGS library generation of MDA-amplified material Basic Protocol 3: Exome enrichment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891307a5d438a6375e34496a127912e0af6f10c1" target='_blank'>
              Exome Sequencing Starting from Single Cells.
              </a>
            </td>
          <td>
            Ioanna Andreou, Markus Storbeck, Peter Hahn, Samuel J. Rulli, Eric Lader
          </td>
          <td>2024-09-01</td>
          <td>Current protocols</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a0102937166e631177aafa289ad76311941f3d10" target='_blank'>
              Genomic and immune heterogeneity of multiple synchronous lung adenocarcinoma at different developmental stages
              </a>
            </td>
          <td>
            Yue Zhao, Jian Gao, Jun Wang, F. Fan, Chao Cheng, Danwen Qian, Ran Guo, Yang Zhang, Ting Ye, Marcellus Augustine, Yicong Lin, Jun Shang, Hang Li, Yunjian Pan, Qingyuan Huang, Haiqing Chen, Han Han, Zhendong Gao, Qiming Wang, Shiyue Zhang, Mou Zhang, F. Fu, Yueren Yan, Shanila Fernandez Patel, Roberto Vendramin, Hui Yuan, Yawei Zhang, Jiaqing Xiang, Hong Hu, Yihua Sun, Yuan Li, Kevin Litchfield, Zhiwei Cao, Haiquan Chen
          </td>
          <td>2024-09-10</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9824378cb72ff7a6ac9e3450ad8fb3f2d8abaa3a" target='_blank'>
              Human genomic DNA is widely interspersed with i-motif structures.
              </a>
            </td>
          <td>
            C. D. Peña Martinez, Mahdi Zeraati, R. Rouet, Ohan Mazigi, Jake Y Henry, Brian Gloss, J. Kretzmann, Cameron W Evans, E. Ruggiero, Irene Zanin, Maja Marušič, J. Plavec, Sara N. Richter, Tracy M. Bryan, Nicole M Smith, Marcel E. Dinger, Sarah Kummerfeld, Daniel Christ
          </td>
          <td>2024-08-29</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="
 Sequential changes in epigenomically specified transcriptional cell states are essential for normal developmental progression. These normal processes are co-opted in cancers, allowing genetically identical cells in different states to cooperatively form heterogeneous tumors. Specific cell states may display distinct malignant properties, including variability in sensitivity to conventional therapies, resulting in heterogeneous responses to therapy and a potential mechanism of patient relapse. As a result, deriving mechanistic approaches to control cell state transitions for therapeutic benefit has been limited by the lack of tractable live-cell tools able to reflect complex and multifaceted cell states. To address this problem, here we demonstrate the design, conception, synthesis and interrogation of a new method termed “Transcriptional Reporter Elements of Cell State” (TRECS). TRECS integrates epigenomics and transcriptomics to identify endogenous genomic elements that can label and separate cell states within heterogenous populations. Using integrative three-dimentional chromatin conformation data and CRISPR deletion studies, we link the mechanism of TRECS reporter activity to control by master transcriptional regulators of distinct cell states. Implementing this system in the high-risk pediatric solid tumor neuroblastoma, we quantitate intratumoral transcriptional and epigenetic heterogeneity and observe real-time cell state plasticity. We demonstrate that neuroblastoma cell line models generally contain plastic cell populations, including one cell state with intrinsic broad chemoresistance. Capitalizing on this intrinsic cell state plasticity, we perform a high-throughput imaging-based small molecule screen to identify chemical controllers of cell state. We identify the coactivator proteins EP300/CBP as primary regulators of the neuroblastoma chemoresistant cell state. Transient disruption of EP300/CBP activity induces sustained epigenetic and transcriptional reprogramming, which fundamentally alters the chemoresistant neuroblastoma cell state, resulting in enhanced chemosensitivity. These findings demonstrate a new, scalable approach to visualize and dissect high-risk cell states with distinct malignant properties in heterogenous tumors. This platform also guides chemical-genetic strategies to control especially challenging cell states, for example, by targeting EP300/CBP maintenance of chemoresistance.
 Citation Format: Noha AM Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad AM Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, Adam D. Durbin. A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A054.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a3089b52a90788a8d540ab999521babb47d6723" target='_blank'>
              Abstract A054 A genome-derived cell state reporter permits dissection and control of intratumoral heterogeneity and chemoresistance
              </a>
            </td>
          <td>
            N. Shendy, Yang Zhang, Ha-Won Lee, Stephanie Nance, Yousef Khashana, Mohammad A.M. Nezhady, Elaine Ritter, Shivendra Singh, Qi Liu, Yiping Fan, Jun Yang, Anand G. Patel, Jun Qi, Taosheng Chen, Brian J. Abraham, A. D. Durbin
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e48d863d20b27ee2206ff6d80ccbbfa69c29c0c2" target='_blank'>
              3D genome contributes to MHC-II neoantigen prediction
              </a>
            </td>
          <td>
            Mofan Feng, Liangjie Liu, K. Su, Xianbin Su, Luming Meng, Zehua Guo, Dan Cao, Jiayi Wang, Guang He, Yi Shi
          </td>
          <td>2024-09-26</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="We report the development and performance of a novel genomics platform, TempO-LINC, for conducting high-throughput transcriptomic analysis on single cells and nuclei. TempO-LINC works by adding cell-identifying molecular barcodes onto highly selective and high-sensitivity gene expression probes within fixed cells, without having to first generate cDNA. Using an instrument-free combinatorial-indexing approach, all probes within the same fixed cell receive an identical barcode, enabling the reconstruction of single-cell gene expression profiles across as few as several hundred cells and up to 100,000+ cells per run. The TempO-LINC approach is easily scalable based on the number of barcodes and rounds of barcoding performed; however, for the experiments reported in this study, the assay utilized over 5.3 million unique barcodes. TempO-LINC has a robust protocol for fixing and banking cells and displays high-sensitivity gene detection from multiple diverse sample types. We show that TempO-LINC has an observed multiplet rate of less than 1.1% and a cell capture rate of ∼50%. Although the assay can accurately profile the whole transcriptome (19,683 human or 21,400 mouse genes), it can be targeted to measure only actionable/informative genes and molecular pathways of interest – thereby reducing sequencing requirements. In this study, we applied TempO-LINC to profile the transcriptomes of 89,722 cells across multiple sample types, including nuclei from mouse lung, kidney and brain tissues. The data demonstrated the ability to identify and annotate at least 50 unique cell populations and positively correlate expression of cell type-specific molecular markers within them. TempO-LINC is a robust new single-cell technology that is ideal for large-scale applications/studies across thousands of samples with high data quality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b9ada174b98957ed877dc22bea85346576005cf9" target='_blank'>
              High-throughput gene expression analysis with TempO-LINC sensitively resolves complex brain, lung and kidney heterogeneity at single-cell resolution
              </a>
            </td>
          <td>
            Dennis J. Eastburn, Kevin White, Nathan D. Jayne, Salvatore Camiolo, Gioele Montis, Seungeun Ha, Kendall G. Watson, Joanne M. Yeakley, J. Mccomb, Bruce Seligmann
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62cbbfa74d4b379f3ac75ff31b1584f4b3b436bc" target='_blank'>
              Heterochromatin formation and remodeling by IRTKS condensates counteract cellular senescence.
              </a>
            </td>
          <td>
            Jia Xie, Zhao-Ning Lu, Shi-Hao Bai, Xiao-Fang Cui, He-Yuan Lian, Chen-Yi Xie, Na Wang, Lan Wang, Ze-Guang Han
          </td>
          <td>2024-08-27</td>
          <td>The EMBO journal</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="
 Atypical Teratoid Rhabdoid Tumor (ATRT) is a highly aggressive pediatric brain tumor and characterized by SMARCB1 deletion. SMARCB1 is a member of the SWI/SNF chromatin remodeling complex that is responsible for determining cellular pluripotency and lineage commitment. We performed an RNAi-based functional genomic screen to identify key epigenetic regulators that co-operate with SMARCB1 loss and contribute to tumorigenesis in ATRT. 406 genes associated with epigenetic regulation in patient derived ATRT cell lines were targeted. This screen identified BMI1, a component of the Polycomb Repressive Complex 1 (PRC1), as one of the top targets essential for ATRT cell growth. Using RNA-seq and ChIP-Seq analysis we demonstrated that BMI1 co-operates with SMARCB1 deletion to suppress transcription of pro-differentiation pathways and promote self-renewal of tumor stem cells. In SMARCB1 deficient cells BMI1 forms a partial PRC1 complex devoid of DNA binding components. We found that genetic inactivation of BMI1 suppresses ATRT cell clonogenicity in vitro and decreases S-phase length. By Doxycycline-inducible SMARCB1 re-expression system we revealed activation of proapoptotic proteins p16 and p21. With immunoprecipitation for BMI1, followed by mass spectrometry analysis we demonstrated that re-expression of SMARCB1 activates two PRC1 chromatin localizing components CBX4 and CBX8. Moreover, inhibition of BMI1 significantly prolongs mice survival and decreases tumor size as evaluated by IVIS imaging system and T2-MRI. Thus, we demonstrated that SMARCB1 deletion results in reprograming of BMI1 chromatin occupancy away from lineage specification by altering the components of the PRC1 complex and that PRC1/BMI1 inhibition is a novel therapeutic approach in ATRT.
 Citation Format: Irina Irina Alimova, Gillian Murdock, Angela M. Pierce, Dong Wang, Nathan Dahl, Sujatha Venkataraman, Rajeev Vibhakar. PRC1/BMI1 activity alters chromatin accessibility to regulate differentiation in atypical teratoid rhabdoid tumors [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A080.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d643026696b6226900916830deb821d92b3f76f" target='_blank'>
              Abstract A080 PRC1/BMI1 activity alters chromatin accessibility to regulate differentiation in atypical teratoid rhabdoid tumors
              </a>
            </td>
          <td>
            Irina Irina Alimova, Gillian Murdock, Angela M Pierce, Dong Wang, N. Dahl, S. Venkataraman, Rajeev Vibhakar
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Extramedullary disease (EMD) is a high-risk feature of multiple myeloma (MM) and remains a poor prognostic factor even in the era of novel immunotherapies. Here we applied spatial transcriptomics (tomo-seq [n=2] and 10X Visium [n=12]), and single-cell RNA sequencing (scRNAseq [n=3]) to a set of 14 EMD biopsies to dissect the three-dimensional architecture of tumor cells and their microenvironment. Overall, the infiltrating immune and stromal cells showed both intra- and inter-patient variation with no uniform distribution over the lesion. We observed substantial heterogeneity at the copy number level within plasma cells, including the emergence of new subclones in circumscribed areas of the tumor, consistent with genomic instability. We further identified spatial expression differences of GPRC5D and TNFRSF17, two important antigens for bispecific antibody therapy. EMD masses were infiltrated by various immune cells, including T-cells. Notably, exhausted TIM3+/PD-1+ T-cells diffusely co-localized with MM cells, whereas functional and activated CD8+ T-cells showed a focal infiltration pattern along with M1 macrophages in otherwise tumor-free regions. This segregation of fit and exhausted T-cells was resolved in the case of response to T-cell engaging bispecific antibodies. MM cells and microenvironment cells were embedded in a complex network that influenced immune activation and angiogenesis, and oxidative phosphorylation represented the major metabolic program within EMD lesions. In summary, spatial transcriptomics has revealed a multicellular ecosystem in EMD with checkpoint inhibition and dual targeting as potential new therapeutic avenues.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/44c7b18e1e67d3b7bc8c1e36ac521009addbf9e8" target='_blank'>
              Spatial transcriptomics reveals profound subclonal heterogeneity and T-cell dysfunction in extramedullary myeloma.
              </a>
            </td>
          <td>
            Maria John, Moutaz Helal, Johannes Duell, Greta Mattavelli, E. Stanojkovska, Nazia Afrin, A. Leipold, M. Steinhardt, Xiang Zhou, D. Žihala, A. Anilkumar Sithara, J. Mersi, J. Waldschmidt, C. Riedhammer, S. Kadel, M. Truger, Rudolf A Werner, C. Haferlach, H. Einsele, Kai Kretzschmar, T. Jelínek, Andreas Rosenwald, K. M. Kortüm, Angela Riedel, L. Rasche
          </td>
          <td>2024-08-20</td>
          <td>Blood</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="The genome of living cells is constantly challenged by DNA lesions that interfere with cellular processes such as transcription and replication. A manifold of mechanisms act in concert to ensure adequate DNA repair, gene expression, and genome stability. Bulky DNA lesions, such as those induced by UV light or the DNA-damaging agent 4-nitroquinoline oxide, act as transcriptional and replicational roadblocks and thus represent a major threat to cell metabolism. When located on the transcribed strand of active genes, these lesions are handled by transcription-coupled nucleotide excision repair (TC-NER), a yet incompletely understood NER sub-pathway. Here, using a genetic screen in the yeast Saccharomyces cerevisiae, we identified histone variant H2A.Z as an important component to safeguard transcription and DNA integrity following UV irradiation. In the absence of H2A.Z, repair by TC-NER is severely impaired and RNA polymerase II clearance reduced, leading to an increase in double-strand breaks. Thus, H2A.Z is needed for proficient TC-NER and plays a major role in the maintenance of genome stability upon UV irradiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dfdbec941bcb8e8c471b2aca881333dab9e8da3d" target='_blank'>
              Histone variant H2A.Z is needed for efficient transcription-coupled NER and genome integrity in UV challenged yeast cells
              </a>
            </td>
          <td>
            H. Gaillard, Toni Ciudad, A. Aguilera, R. Wellinger
          </td>
          <td>2024-09-01</td>
          <td>PLOS Genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="
 Despite recent advances in the treatment of pancreatic adenocarcinoma (PDAC), the median survival remains <12 months. Patients typically present with late-stage disease and are often unable to tolerate drug combination regimens. Large-scale genomic approaches categorise PDAC into two distinct molecular subtypes, based on transcriptomic signatures, and these are termed the classical and squamous (or basal) subtypes. Although it is well-described that PDAC harbours gene mutations (i.e. KRAS G12D, p53 R172H) that initiate the disease, little is known about the influence of lineage-defining Transcription Factors (TF) on PDAC subtypes and their contribution to tumor progression. We took an unbiased approach to discover driving TFs in PDAC, we performed ChIP-seq of the active enhancer mark H3K27Ac from surplus tissue following resection of primary PDAC tumors (n=6) and normal adjacent tissue (n=4). Analysis of the H3K27Ac sites showed that the most enriched motifs within all experimentally mapped enhancer elements were HNF motifs (HNF4G & HNF4A) and Forkhead (i.e., FOXA1), confirming that these two classes of TFs constitute the lineage-defining factors in these clinical samples. Investigation of publicly available clinical datasets revealed HNF4G to be amplified in 9% of PDAC patients. Using Rapid Immunoprecipitation and Mass Spectrometry of Endogenous protein complexes (RIME), our method for unbiased discovery of endogenous protein complexes, we investigated the fundamental differences in the FOXA1 interactome between models of classical and squamous subtypes. HNF4A and HNF4G were only seen as FOXA1-interactors in models of the classical subtype. HNF4G Knockout (KO) clones injected orthotopically into the pancreas of mice, showed a significant increase (p-value<0.0006) in median survival and slower tumor growth compared to Control (Cas-9) (p-value <0.0001). HNF4G-KO dependent genes were specifically expressed in primary tumors (with decreased EMT signalling) versus metastatic samples (PanCRux dataset), implying that the HNF4G target genes were mostly limited to primary tumor contexts. We found Protein Arginine Methylate Transferase (PRMT1) to be a significant and reproducible interactor of FOXA1 and HNF4G. We observed a significant global reduction in PRMT1 chromatin binding in both the HNF4G-KO and FOXA1 depleted cells compared to the control. This data suggests a dependency on both HNF4G and FOXA1 for recruitment of PRMT1 to chromatin. We find HNF4G to be the key driver of classical subtype primary disease, in part by recruiting the methyltransferase PRMT1 to regulatory regions revealing a therapeutically exploitable opportunity. A molecular switch occurs in advanced disease, whereby HNF4G expression decreases, unmasking FOXA1’s transcriptional activity. De-repressed FOXA1 drives late-stage disease, by orchestrating metastasis-specific enhancer-promoter loops to direct metastatic gene regulation. We suggest the existence of stage-dependent TF activity, triggered by molecular compartmentalisation that mediates multistage progression of PDAC.
 Citation Format: Shalini Vasudev Rao, Lisa V Young, Danya Cheeseman, Stephanie V Mack, Dominique-Laurent Couturier, Sean V Flynn, Rebecca Brias, Amy Smith, Jill Temple, Duncan Jodrell, Alasdair Russell V Russell, Igor Chernukhin, Jason Carroll. Transcription factor switching drives progression of the classical subtype of pancreatic cancer [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C071.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e663bb2d35a48fcc777219947b4d5e9554b7ef19" target='_blank'>
              Abstract C071: Transcription factor switching drives progression of the classical subtype of pancreatic cancer
              </a>
            </td>
          <td>
            Shalini V Rao, Lisa Young, D. Cheeseman, Stephanie V Mack, Dominique-Laurent Couturier, Sean V Flynn, Rebecca Brias, Amy Smith, Jill Temple, Duncan Jodrell, Alasdair Russell V Russell, Igor Chernukhin, Jason Carroll
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61a1e72cd7656def115fe3bbce4a02021bb17ead" target='_blank'>
              Involvement of the splicing factor SART1 in the BRCA1-dependent homologous recombination repair of DNA double-strand breaks
              </a>
            </td>
          <td>
            Kie Ozaki, Reona Kato, Takaaki Yasuhara, Yuki Uchihara, Miyako Hirakawa, Yu Abe, Hiroki Shibata, Reika Kawabata-Iwakawa, Aizhan Shakayeva, Palina Kot, Kiyoshi Miyagawa, Keiji Suzuki, Naoki Matsuda, Atsushi Shibata, M. Yamauchi
          </td>
          <td>2024-08-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966f20b4115e34215d01f56c4a256d96a9d46af1" target='_blank'>
              Mosaic variegated aneuploidy in development, ageing and cancer.
              </a>
            </td>
          <td>
            Marcos Malumbres, Carolina Villarroya-Beltri
          </td>
          <td>2024-08-21</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3734fdaa3ed3777431c0f723757629a9b7e4aa2" target='_blank'>
              High-complexity of DNA double-strand breaks is key for alternative end-joining choice
              </a>
            </td>
          <td>
            Zhiyang Hou, Tianxiang Yu, Qiyi Yi, Yan Du, Libin Zhou, Ye Zhao, Yuejin Wu, Lijun Wu, Ting Wang, Po Bian
          </td>
          <td>2024-08-03</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Tracing cell lineages has become a valuable tool for studying biological processes. Among the available tools for human data, mitochondria DNA (mtDNA) has a high potential due to its ability to be used in conjunction with single-cell chromatin accessibility data, giving access to the cell phenotype. Nonetheless, the existing mutation calling tools are ill-equipped to deal with the polyploid nature of the mtDNA and lack a robust statistical framework. Here we introduce scMitoMut, an innovative R package that leverages statistical methodologies to accurately identify mitochondrial lineage related mutations at the single-cell level. scMitoMut assigns a mutation quality q-value based on beta-binomial distribution to each mutation at each locus within individual cells, ensuring higher sensitivity and precision of lineage related mutation calling in comparison to current methodologies. We tested scMitoMut using single-cell DNA sequencing, scATAC sequencing and 10× Genomics single cell multiome datasets. Using a single-cell DNA sequencing dataset from a mixed population of cell lines, scMitoMut demonstrated superior sensitivity in identifying small proportion of cancer cell lines compared to existing methods. In a human colorectal cancer scATAC dataset, scMitoMut identified more mutations than state-of-the-art methods. Applied to 10× Genomics multiome datasets, scMitoMut effectively measured the lineage distance in cells from blood or brain tissues. Thus, the scMitoMut is a free available (https://www.bioconductor.org/packages/devel/bioc/html/scMitoMut.html.), well-engineered toolkit for mtDNA mutation calling with high memory and CPU efficiency. Consequently, it will significantly advance the application of single-cell sequencing, facilitating the precise delineation of mitochondrial mutations for lineage tracing purposes in development, tumor and stem cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a057cdbd6d7399eed5dec23b92e83e85fe7d0c9" target='_blank'>
              scMitoMut for calling mitochondrial lineage–related mutations in single cells
              </a>
            </td>
          <td>
            Wenjie Sun, Daphne van Ginneken, Leïla Perié
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1f5936217eac5b4043bb1665424b2339a9f20d" target='_blank'>
              MYC and KRAS cooperation: from historical challenges to therapeutic opportunities in cancer
              </a>
            </td>
          <td>
            S. Casacuberta-Serra, Íñigo González-Larreategui, Daniel Capitán-Leo, Laura Soucek
          </td>
          <td>2024-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c063409361f89f55a497cfd8fcbb23f7e584a467" target='_blank'>
              Deciphering genetic and nongenetic factors underlying tumour dormancy: insights from multiomics analysis of two syngeneic MRD models of melanoma and leukemia
              </a>
            </td>
          <td>
            Marie-Océane Laguillaumie, Sofia Titah, Aurel Guillemette, Bernadette Neve, Frédéric Leprêtre, Pascaline Ségard, F. Shaik, Dominique Collard, Jean-Claude Gerbedoen, L. Fléchon, L. Hasan Bou Issa, Audrey Vincent, M. Figeac, Shéhérazade Sebda, C. Villenet, Jérôme Kluza, William Laine, Isabelle Fournier, Jean-Pascal Gimeno, M. Wisztorski, Salomon Manier, M. Tarhan, Bruno Quesnel, Thierry Idziorek, Yasmine Touil
          </td>
          <td>2024-09-03</td>
          <td>Biological Research</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Childhood neuroblastoma with MYCN-amplification is classified as high-risk and often relapses after intensive treatments. Immune checkpoint blockade therapy against the PD-1/L1 axis shows limited efficacy in neuroblastoma patients and the cancer intrinsic immune regulatory network is poorly understood. Here, we leverage genome-wide CRISPR/Cas9 screens and identify H2AFY as a resistance gene to the clinically approved PD-1 blocking antibody, nivolumab. Analysis of single-cell RNA sequencing datasets reveals that H2AFY mRNA is enriched in adrenergic cancer cells and is associated with worse patient survival. Genetic deletion of H2afy in MYCN-driven neuroblastoma cells reverts in vivo resistance to PD-1 blockade by eliciting activation of the adaptive and innate immunity. Mapping of the epigenetic and translational landscape demonstrates that H2afy deletion promotes cell transition to a mesenchymal-like state. With a multi-omics approach, we uncover H2AFY-associated genes that are functionally relevant and prognostic in patients. Altogether, our study elucidates the role of H2AFY as an epigenetic gatekeeper for cell states and immunogenicity in high-risk neuroblastoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dae04d78ec0c131d4f8ff3da9b40c7edb7f93956" target='_blank'>
              Epigenetic regulation of cell state by H2AFY governs immunogenicity in high-risk neuroblastoma.
              </a>
            </td>
          <td>
            Divya Nagarajan, Rebeca T Parracho, David Corujo, Minglu Xie, G. Kutkaite, T. Olsen, Marta Rúbies Bedós, Maede Salehi, Ninib Baryawno, M. Menden, Xingqi Chen, M. Buschbeck, Y. Mao
          </td>
          <td>2024-09-10</td>
          <td>The Journal of clinical investigation</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="The role of focal amplifications and extrachromosomal circular DNA (eccDNA) is still uncertain in prostate adenocarcinoma (PRAD). Here, we first mapped the global characterizations of eccDNA and then investigate the characterization of eccDNA-amplified key differentially expressed encoded genes (eKDEGs) in the progression, immune response and immunotherapy of PRAD.Circular_seq was used in conjunction with the TCGA-PRAD transcriptome dataset to sequence, annotate, and filter for eccDNA-amplified differentially expressed coding genes (eDEGs) in PRAD and para-cancerous normal prostate tissues. Afterwards, risk models were created and eKDEGs linked to the PRAD prognosis were identified using Cox and Lasso regression analysis. The immune microenvironment of the risk model was quantified using a variety of immunological algorithms, which also identified its characteristics with regard to immunotherapy, immune response, and immune infiltration.In this research, there was no significant difference in the size, type, and chromosomal distribution of eccDNA in PRAD and para-cancerous normal prostate tissues. However, 4,290 differentially expressed eccDNAs were identified and 1,981 coding genes were amplified. Following that, 499 eDEGs were tested in conjunction with the transcriptome dataset from TCGA-PRAD. By using Cox and Lasso regression techniques, ZNF330 and PITPNM3 were identified as eKDEGs of PRAD, and a new PRAD risk model was conducted based on this. Survival analysis showed that the high-risk group of this model was associated with poor prognosis and validated in external data. Immune infiltration analysis showed that the model risks affected immune cell infiltration in PRAD, not only mediating changes in immune cell function, but also correlating with immunophenotyping. Furthermore, the high-risk group was negatively associated with anti-CTLA-4/anti-PD-1 response and mutational burden. In addition, Tumor Immune Dysfunction and Exclusion analyses showed that high-risk group was more prone to immune escape. Drug sensitivity analyses identified 10 drugs, which were instructive for PRAD treatment.ZNF330 and PITPNM are the eKDEGs for PRAD, which can be used as potential new prognostic markers. The two-factor combined risk model can effectively assess the survival and prognosis of PRAD patients, but also can predict the different responses of immunotherapy to PRAD patients, which may provide new ideas for PRAD immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/189d5f00b52f7644f54771a980ef61f81d31dae9" target='_blank'>
              Extrachromosomal circular DNAs in prostate adenocarcinoma: global characterizations and a novel prediction model
              </a>
            </td>
          <td>
            Qingliu He, Qingfu Su, Chengcheng Wei, Pu Zhang, Weihui Liu, Junyi Chen, Xiaoping Su, Zhuang Wei
          </td>
          <td>2024-09-17</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Whole exon sequencing (WES) data are frequently used for cancer diagnosis and genome-wide association studies (GWAS), hinging upon high-coverage read mapping, informative variant calling, and high-quality reference genomes. The center position of the currently used genome assembly, GRCh38, is now challenged by two newly publicized telomere-to-telomere or T2T genomes, T2T-CHM13 and T2T-YAO, and it becomes urgent to have a comparative study to test population specificity using the three reference genomes based on real case WES data. We here report our analysis along this line for 19 tumor samples collected from Chinese patients. The primary comparison of the exon regions among the three references reveals that the sequences in up to ∼1% target regions in YAO are widely diversified from GRCh38 and may lead to off-target in sequence capture. However, YAO still outperforms GRCh38 genomes by obtaining 7.41% more mapped reads. Due to more reliable read-mapping and closer phylogenetic relationship with the samples than GRCh38, YAO reduces half of variant calls of clinical significance which are mostly benign while keeping sensitivity in identifying pathogenic variants. YAO also outperforms CHM13 in reducing calls of Chinese-specific variants. Our findings highlight the critical need for employing population-specific reference genomes in genomic analysis to ensure accurate variant analysis and the significant benefits of tailoring these approaches to the unique genetic backgrounds of each ethnic group.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/125110dee90b36e3730eebfc5dcf7afd43e85aa6" target='_blank'>
              Enhancing Variant Calling in Whole Exome Sequencing (WES) Data Using Population-Matched Reference Genomes
              </a>
            </td>
          <td>
            Shuming Guo, Zhuo Huang, Yanming Zhang, Yukun He, Xiangju Chen, Wenjuan Wang, Lansheng Li, Yu Kang, Zhancheng Gao, Jun Yu, Zhenglin Du, Yanan Chu
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="CRISPR-Cas12a, an RNA-guided nuclease, has been repurposed for genome editing and molecular diagnostics due to its capability of cis-cleavage on target DNA and trans-cleavage on non-target single-strand DNA (ssDNA). However, the mechanisms underlying the activation of trans-cleavage activity of Cas12a, particularly in the context of split DNA activators, remain poorly understood. We elucidate the synergistic effect of these activators and introduce the concepts of induced targeting effect and exon-unwinding to describe the phenomenon. We demonstrate that upon binding of split DNA activators adjacent to the Protospacer Adjacent Motif (PAM) to the Cas12a ribonucleoprotein (Cas12a-RNP), a ternary complex form that can capture and interact with distal split DNA activators to achieve synergistic effects. Notably, if the distal activator is double-strand DNA (dsDNA), the complex initiates exon-unwinding, facilitating the RNA-guide sequence's access. Our findings provide a mechanistic insight into action of Cas12a and propose a model that could significantly advance our understanding of its function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21bfca41537975350cb4978fbebd5605ae4be926" target='_blank'>
              Synergistic effect of split DNA activators of Cas12a with exon-unwinding and induced targeting effect.
              </a>
            </td>
          <td>
            Shen Huang, Yongliang Lou, Laibao Zheng
          </td>
          <td>2024-09-11</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7300fe63a85fac8fe37bf10af42231e58f921f5" target='_blank'>
              Analysis of the role of CHPF in colorectal cancer tumorigenesis and immunotherapy based on bioinformatics and experiments
              </a>
            </td>
          <td>
            Qingyu Song, Pengchao Wang, Jingyu Wu, Ming Lu, Qingcheng Xia, Yexin Shi, Zijun Wang, Xiang Ma, Qinghong Zhao
          </td>
          <td>2024-09-18</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="
 Study of early pancreas neoplasia in humans had previously been limited by lack of available tissue. Recent work by our group on deceased donor pancreata identified pancreatic intraepithelial neoplasia (PanIN) lesions in non-diseased organs and defined a transcriptomic signature for the microenvironment of these pre-cancerous lesions. Prevalence of PanINs in healthy human tissue was higher than expected and established a novel model to expand understanding of the complex biology of these precursor lesions. Epigenetic changes in chromatin structure and the subsequent effects on gene expression are essential processes in neoplasia that have yet to be described at the single cell level in the human pancreas. We hypothesized that previously defined gene signatures could identify acinar, ductal, and PanIN cells in donor pancreata based on chromatin accessibility profiling by single nuclear transposase-accessible chromatin preparation followed by high-throughput sequencing (snATAC-seq). Comparison between the accessibility patterns, transcription factor motifs, and differences in accessibility versus expression profiles can then be used to expand prior knowledge of early pancreatic neoplasia and identify novel targets for further study and therapy selection. To test this hypothesis, single nuclei from seventeen donor pancreata (twenty total samples) were isolated for snATAC-seq using the 10x Genomics platform. Data analysis was conducted using CellRanger, Seurat, Signac, AUCell, and other programs to identify and label chromatin peaks in individual nuclei corresponding to acinar, ductal, and PanIN origin according to gene signatures identified by scRNA-seq and special transcriptomics on matched donor pancreata. Using this method, cells corresponding to these gene signatures, including PanIN- like cells, were indeed identifiable with differentially accessible regions of the genome then analyzed for transcription factor motif enrichment between these cell types. Motifs enriched in cells identified as PanINs compared to either ductal or acinar cells included several transcription factors implicated in tumorigenesis. These results are similar to previously demonstrated essential factors in pancreatic neoplasia, suggesting this strategy is a relevant method for data analysis and discovery of novel factors implicated in tumorigenesis. Ongoing studies include validation of these findings in vitro using human pancreatic normal epithelial cells, human tumor cell lines, and organoid models derived from human normal and tumor samples. Together, this study demonstrates that PanINs can be identified in human pancreas tissue by snATAC-seq and that differentially expressed motifs can be identified, providing opportunity for further integration with transcriptomic information to elucidate detailed understanding of early neoplasia in the pancreas for future drug development and targeted therapeutics studies.
 Citation Format: Jamie N Mills, Aaron Dendekker, Joyce K Thompson, Hannah Watkoske, Simone Benitz, Padma Kadiyala, Ahmed Elhossiny, Howard Crawford, Eileen Carpenter, Marina Pasca di Magliano, Filip Bednar. Characterization of chromatin accessibility patterns in the human pancreas and early pancreatic neoplastic lesions using snATAC-seq [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr C056.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eee7faf599c3d04827a3bdb6a5acb4e73be0ca2f" target='_blank'>
              Abstract C056: Characterization of chromatin accessibility patterns in the human pancreas and early pancreatic neoplastic lesions using snATAC-seq
              </a>
            </td>
          <td>
            Jamie N. Mills, Aaron Dendekker, Joyce K Thompson, Hannah R Watkoske, S. Benitz, Padma Kadiyala, Ahmed M. Elhossiny, Howard C. Crawford, Eileen S Carpenter, M. Pasca di Magliano, Filip Bednar
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>35</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54ad6a53d090caa0cff0b86e7a380a1302888db4" target='_blank'>
              Circulating tumor cells: from new biological insights to clinical practice
              </a>
            </td>
          <td>
            Xuyu Gu, Shiyou Wei, Xin Lv
          </td>
          <td>2024-09-02</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c35f97c112d09b7e983874a9efe5c2fc2f69d6c" target='_blank'>
              Enhancers in T Cell development and malignant lesions
              </a>
            </td>
          <td>
            Tong Zhang, Lin Zou
          </td>
          <td>2024-09-17</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/028d055f673c0c4d806112c2a9d63a57da73eaef" target='_blank'>
              A brief review of noncoding RNA
              </a>
            </td>
          <td>
            Tina P. George, Suja Subramanian, M. H. Supriya
          </td>
          <td>2024-09-02</td>
          <td>Egyptian Journal of Medical Human Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Accumulating evidence indicates that G‐quadruplexes (G4s) are involved in transcriptional regulation. Previous studies have demonstrated that DHX36 preferentially resolves G4s, suggesting its potential impact on gene transcription mediated by these structures. However, systematic validation is required to establish a link between DHX36 activity and its roles in transcriptional regulation. In this study, we investigate the role of DHX36 in transcription. First, we employ the cleavage under targets and tagmentation (CUT&Tag), an efficient method for mapping protein–DNA interactions, to identify the binding sites in the chromatin of MCF‐7 cells. Subsequently, we use the auxin‐inducible degron (AID) protein degradation system and improved nascent RNA sequencing method acrylonitrile‐mediated uridine‐to‐cytidine conversion sequencing (AMUC‐seq) to pinpoint genes directly regulated by DHX36. Our results reveal a significant enrichment of G4 structures at DHX36 target sites, predominantly located in active genomic regions. In vitro assays further demonstrate DHX36's interaction with G4 sequences from three specific oncogenes. These findings underscore the potential role of DHX36 in modulating gene transcription through G4 structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/07f9bb5fca73acfa9f0b936e9e6d763af42a8898" target='_blank'>
              Rapid degradation of DHX36 revealing its transcriptional role by interacting with G‐quadruplex
              </a>
            </td>
          <td>
            Ziang Lu, Jinglei Xu, Yuqi Chen, Yuanyuan Zhou, Xiaolu Zhou, Qi Wang, Qi Wei, Shaoqing Han, Ruiqi Zhao, X. Weng, Xiaolian Zhang, Xiang Zhou
          </td>
          <td>2024-08-19</td>
          <td>Aggregate</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Precise regulation of gene expression is essential for proper development and the maintenance of homeostasis in organisms. Studies have shown that some transcriptional regulatory proteins influence gene expression through the formation of dynamic, locally concentrated assemblies known as condensates, while dysregulation of transcriptional condensates was associated with several cancers, such as Ewing sarcoma and AML [Wang Y et al. (2023) Nat Chem Biol 19, 1223-1234; Chandra B et al. (2022) Cancer Discov 12, 1152-1169]. Mutations in the histone acetylation "reader" eleven-nineteen-leukemia (ENL) have been shown to form discrete condensates at endogenous genomic targets, but it remains unclear how ENL mutations drive tumorigenesis and whether it is correlated with their condensate formation property. Liu et al. now show, using a conditional knock-in mouse model, that ENL YEATS domain mutation is a bona fide oncogenic driver for AML. This mutant ENL forms condensates in hematopoietic stem/progenitor cells at the genomic loci of key leukemogenic genes, including Meis1 and Hoxa cluster genes, and disrupting condensate formation via mutagenesis impairs its chromatin and oncogenic function. Furthermore, they show that small-molecule inhibition of the acetyl-binding activity displaces ENL mutant condensates from oncogenic target loci, and this inhibitor significantly impairs the onset and progression of AML driven by mutant ENL in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe9e0f2605685a3812a479056f634ccfed5efb36" target='_blank'>
              ENL mutation and AML: a new model that reveals oncogenic condensate's function in leukemogenesis.
              </a>
            </td>
          <td>
            Zhong Fan, Yanan Jiang, Xiaotian Zhang
          </td>
          <td>2024-09-26</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
 Objective: Structural variations (SVs) constitute the greatest source of genomic alterations leading to the oncogenesis of many cancers, including the deadliest pancreatic cancer (PC). However, SVs in PC remain largely undefined due to technological limitations of the traditional short-read sequencing. This study aims to establish a comprehensive signature of genomic SVs in PC. Methods: PacBio whole genome long-read sequencing was employed to investigate the genomic landscape of SVs in eight tissues from four pancreatic cancer patients. A variety of SVs callers, including pbsv, Sniffles, cuteSV, and svim, were comprehensively adopted to improve the accuracy of SVs detection. Many undefined characteristics of germline and somatic SVs were revealed by comparing with multiple previously published databases. Besides, the impacts of structural variations on 3D chromatin organization were analyzed by integrating multi-omics data including Hi-C, RNA seq and ChIP seq. Results: A total of 26,844 and 27,028 non-redundant SVs were identified in human pancreatic cancer and matched para-cancerous tissues, respectively. Notably, among these SVs, 23.97% were novel and had not been previously reported. Germline SVs were further characterized and found enriched in 25 well-known susceptible genes, such as TP63, PVT1, and ABO. Additionally, we proposed a panel of germline SVs with predicted highest pathogenicity, involving STOX1, DSPP, and WRN. Totally, 616 somatic SVs were identified and presented high burdens in patients with lymph node metastasis. We observed that somatic DELs and INSs were significantly more prevalent in repetitive regions than in non-repetitive regions. Interestingly, DELs showed a decrease in the proportion occurring in simple repeat regions but an increase in the proportion occurring in LINE and SINE regions compared to INSs. Moreover, it was observed that deletions within topologically associated domains were associated with local enhanced interactions and disrupted chromatin loops, which might influence H3K27ac modification of histone contributing to enhancer or silencer hijacking. Conclusion: This study provides insights into the genomic signature of SVs in human pancreatic cancer. These findings might shed new light on the complexity of SVs and their pathogenesis by interplaying with chromatin organization.
 Citation Format: Yongxing Du, Xiaohao Zheng, Yunjie Duan, Chengfeng Wang. Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pancreatic Cancer Research; 2024 Sep 15-18; Boston, MA. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl_2):Abstract nr A083.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d40a9fb18df1c1c8a73e8c29f87767e5466baa02" target='_blank'>
              Abstract A083: Genomic Analysis of Structural Variations in Pancreatic Cancer Using Long-Read Sequencing
              </a>
            </td>
          <td>
            Yongxing Du, Xiaohao Zheng, Yunjie Duan, Cheng-Guang Wang
          </td>
          <td>2024-09-15</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f0222f474962325ec3943ada79fb1a6a8a5016a" target='_blank'>
              Spatial resolved transcriptomics reveals distinct cross-talk between cancer cells and tumor-associated macrophages in intrahepatic cholangiocarcinoma
              </a>
            </td>
          <td>
            Zhao-Ru Dong, Meng-Ya Zhang, Ling-Xin Qu, Jie Zou, Yong-Heng Yang, Yun-Long Ma, Chuncheng Yang, Xue-Lei Cao, Li-Yuan Wang, Xiao-Lu Zhang, Tao Li
          </td>
          <td>2024-09-11</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Centrosome amplification is a feature of cancer cells associated with chromosome instability and invasiveness. Enhancing chromosome instability and subsequent cancer cell death via centrosome unclustering and multipolar divisions is an aimed-for therapeutic approach. Here, we show that centrosome amplification potentiates responses to conventional chemotherapy in addition to its effect on multipolar divisions and chromosome instability. We perform single-cell live imaging of chemotherapy responses in epithelial ovarian cancer cell lines and observe increased cell death when centrosome amplification is induced. By correlating cell fate with mitotic behaviors, we show that enhanced cell death can occur independently of chromosome instability. We identify that cells with centrosome amplification are primed for apoptosis. We show they are dependent on the apoptotic inhibitor BCL-XL and that this is not a consequence of mitotic stresses associated with centrosome amplification. Given the multiple mechanisms that promote chemotherapy responses in cells with centrosome amplification, we assess such a relationship in an epithelial ovarian cancer patient cohort. We show that high centrosome numbers associate with improved treatment responses and longer overall survival. Our work identifies apoptotic priming as a clinically relevant consequence of centrosome amplification, expanding our understanding of this pleiotropic cancer cell feature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a9ce2e6325eaf9b5466f08fa222bbaf8fdd0b70" target='_blank'>
              Centrosome amplification primes ovarian cancer cells for apoptosis and potentiates the response to chemotherapy.
              </a>
            </td>
          <td>
            Frances Edwards, Giulia Fantozzi, Anthony Y Simon, Jean-Philippe Morretton, A. Herbette, Andrea E. Tijhuis, René Wardenaar, Stacy Foulane, Simon Gemble, D. Spierings, F. Foijer, Odette Mariani, A. Vincent-Salomon, S. Roman-Roman, X. Sastre-Garau, O. Goundiam, R. Basto
          </td>
          <td>2024-09-05</td>
          <td>PLoS biology</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="R3HDM1, an RNA binding protein with one R3H domain, remains uncharacterized in terms of its association with tumor progression, malignant cell regulation, and the tumor immune microenvironment. This paper aims to fill this gap by analyzing the potential of R3HDM1 in diagnosis, prognosis, chemotherapy, and immune function across various cancers.Data was collected from the Firehost database (http://gdac.broadinstitute.org) to obtain the TCGA pan-cancer queue containing tumor and normal samples. Additional data on miRNA, TCPA, mutations, and clinical information were gathered from the UCSC Xena database (https://xenabrowser.net/datapages/). The mutation frequency and locus of R3HDM1 in the TCGA database were examined using the cBioPortal. External validation through GEO data was conducted to assess the differential expression of R3HDM1 in different cancers. Protein expression levels were evaluated using the Clinical Proteomics Tumor Analysis Alliance (CPTAC). The differential expression of R3HDM1 was verified in lung adenocarcinoma cell lines and normal lung glandular epithelial cells via RT-qPCR. Cell migration and proliferation experiments were conducted by knocking down the expression of R3HDM1 in two lung adenocarcinoma cell lines using small interfering RNA. The biological role of R3HDM1 in pan-cancer was explored using the GSEA method. Multiple immune infiltration algorithms from the TIMER2.0 database was employed to investigate the correlation between R3HDM1 expression and the tumor immune microenvironment. Validation of transcriptome immune infiltration was based on 140 single-cell datasets from the TISCH database. The study also characterized a pan-cancer survival profile and analyzed the differential expression of R3HDM1 in different molecular subtypes. The relationship between R3HDM1 and drug resistance was investigated using four chemotherapy data sources: CellMiner, GDSC, CTRP and PRISM. The impact of chemicals on the expression of R3HDM1 was explored through the CTD database.The study revealed differential expression of R3HDM1 in various tumors, indicating its potential as an early diagnostic marker. Changes in somatic copy number (SCNA) and DNA methylation were identified as factors contributing to abnormal expression levels. Additionally, the study found that R3HDM1 expression is associated with clinical features, metabolic pathways, and important pathways related to metastasis and the immune system. High expression of R3HDM1 was linked to poor prognosis across different tumors and altered drug sensitivity. Furthermore, the expression of R3HDM1 showed significant correlations with immune modulatory molecules and biomarkers of lymphocyte subpopulation infiltration. Finally, the study highlighted four chemicals that could influence the expression of R3HDM1.Overall, this study proposes that R3HDM1 expression is a promising biomarker for predicting the prognosis of cancer, especially lung adenocarcinoma, and the efficacy of immunotherapy, demonstrating the rationale for further exploration in the development of anti-tumor therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/379d6a67141459985f554b1a56ebc613e9c22b34" target='_blank'>
              Comprehensive pan-cancer analysis and experiments revealed R3HDM1 as a novel predictive biomarker for prognosis and immune therapy response
              </a>
            </td>
          <td>
            Jiawei Liu, Zhitong Bing, Junling Wang
          </td>
          <td>2024-09-23</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/feda6005d00312b29429cd211524a37b760d02e2" target='_blank'>
              Pan-cancer analysis of the immunological and oncogenic roles of ATAD2 with verification in papillary thyroid carcinoma
              </a>
            </td>
          <td>
            Zhecheng Li, Yaxin Zhao, Peng Huang, Zhaoyi Wu, D. Ouyang, AdolphusOsei Nyarko, Lei Ai, Zhejia Zhang, Shi Chang
          </td>
          <td>2024-09-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/962a15086155c288de21a400a171d7f9e76960cc" target='_blank'>
              Specific multivalent molecules boost CRISPR-mediated transcriptional activation
              </a>
            </td>
          <td>
            Rui Chen, Xinyao Shi, Xiangrui Yao, Tong Gao, Guangyu Huang, Duo Ning, Zemin Cao, Youxin Xu, Weizheng Liang, S. Tian, Qionghua Zhu, Liang Fang, Meizhen Zheng, Yuhui Hu, Huanhuan Cui, Wei Chen
          </td>
          <td>2024-08-22</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="R-loops, structures that play a crucial role in various biological processes, are integral to gene expression, the maintenance of genome stability, and the formation of epigenomic signatures. When these R-loops are deregulated, they can contribute to the development of serious health conditions, including cancer and neurodegenerative diseases. The detection of R-loops is a complex process that involves several approaches. These include S9.6 antibody- or RNAse H-based immunoprecipitation, non-denaturing bisulfite footprinting, gel electrophoresis, and electron microscopy. Each of these methods offers unique insights into the nature and behavior of R-loops. In our study, we introduce a novel protocol that has been developed based on a single-molecule DNA combing assay. This innovative approach allows for the direct and simultaneous visualization of RNA:DNA hybrids and replication forks, providing a more comprehensive understanding of these structures. Our findings confirm the transcriptional origin of the hybrids, adding to the body of knowledge about their formation. Furthermore, we demonstrate that these hybrids have an inhibitory effect on the progression of replication forks, highlighting their potential impact on DNA replication and cellular function.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3b0d0ec1a89d3fa05ea164f315ebe78596d3a2da" target='_blank'>
              Simultaneous Visualization of R-Loops/RNA:DNA Hybrids and Replication Forks in a DNA Combing Assay
              </a>
            </td>
          <td>
            Miroslav Penchev Ivanov, Heather Zecchini, Petra Hamerlik
          </td>
          <td>2024-09-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b69c52796ad88ee763ab51cb8f0791d90f0f5161" target='_blank'>
              Antagonism between H3K27me3 and genome-lamina association drives atypical spatial genome organization in the totipotent embryo.
              </a>
            </td>
          <td>
            Isabel Guerreiro, Franka J. Rang, Yumiko K Kawamura, Carla Kroon-Veenboer, Jeroen Korving, Femke C Groenveld, Ramada E van Beek, Silke J. A. Lochs, Ellen Boele, A. H. Peters, Jop Kind
          </td>
          <td>2024-09-16</td>
          <td>Nature genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cell biology and genetic analysis of intracellular, intercellular and inter-organ interaction studies in animal models are key for understanding development, physiology, and disease. The MARCM technique can emulate tumor development by simultaneous clonal tumor suppressor loss-of-function generation coupled with GAL4-UAS-driven oncogene and marker expression, but the utility is limited for studying tumor-host interactions due to genetic constraints. To overcome this, we introduce EyaHOST, a novel system that replaces MARCM with the QF2-QUAS binary gene expression system under the eya promoter control, unleashing the fly community genome-wide GAL4-UAS driven tools to manipulate any host cells or tissue at scale. EyaHOST generates epithelial clones in the eye epithelium similar to MARCM. EyaHOST-driven RasV12 oncogene overexpression coupled with scribble tumor suppressor knockdown recapitulates key cancer features, including systemic catabolic switching and organ wasting. We demonstrate effective tissue-specific manipulation of host compartments such as neighbouring epithelial cells, immune cells, fat body, and muscle using fly avatars with tissue-specific GAL4 drivers. Organ-specific inhibition of autophagy or stimulation of growth-signaling through PTEN knockdown in fat body or muscle prevents cachexia-like wasting. Additionally, we show that RasV12, scribRNAi tumors induce caspase-driven apoptosis in the epithelial microenvironment. Inhibition of apoptosis by p35 expression in the microenvironment promotes tumor growth. EyaHOST offers a versatile modular platform for dissecting tumor-host interactions and other mechanisms involving intercellular and inter-organ communication in Drosophila. Highlights *eyes absent, eye disc-specific enhancer drives clonal KD recombinase flip-out activated QF2 expression in the larval eye epithelium for simultaneous QUAS-driven gain and loss-of-function analysis of gene function. *Clones are visualized by QUAS-tagBFP or QUAS-eGFP facilitating analysis of existing fluorescent reporters. *The GAL4-UAS system and existing genome-wide genetic tools are released to independently manipulate any cell population in the animal for cell biology, intercellular or inter-organ analysis for developmental, physiological, or disease model analysis. *Fly avatars for tumor-host interaction studies with multiple organs allow live monitoring and manipulation of tumors and organs in translucent larva.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f6d5a397edb7ccb05aea0ec451c7e0881eefa9e" target='_blank'>
              EyaHOST, a modular genetic system for investigation of intercellular and tumor-host interactions in Drosophila melanogaster
              </a>
            </td>
          <td>
            José Teles-Reis, Ashish Jain, Dan Liu, Rojyar Khezri, Sofia Micheli, Alicia Alfonso Gomez, Caroline Dillard, T. Rusten
          </td>
          <td>2024-09-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3260115731c81c0bb245a4f0f48eb9f7d32bce58" target='_blank'>
              Single-strand DNA-binding protein suppresses illegitimate recombination in Escherichia coli, acting in synergy with RecQ helicase
              </a>
            </td>
          <td>
            Isidoro Feliciello, Sven Ljubić, E. Đermić, S. Ivanković, D. Zahradka, Damir Đermić
          </td>
          <td>2024-09-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bc92a939542e292edaa1bc766f527a0552465d51" target='_blank'>
              PTIP epigenetically regulates DNA damage-induced cell cycle arrest by upregulating PRDM1
              </a>
            </td>
          <td>
            Y. Nakata, Shion Nagasawa, Y. Sera, N. Yamasaki, A. Kanai, K. Kobatake, Takeshi Ueda, M. Koizumi, Ichiro Manabe, Osamu Kaminuma, Hiroaki Honda
          </td>
          <td>2024-08-03</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The cell cycle-regulated DNA methyltransferase CcrM is conserved in most Alphaproteobacteria, but its role in bacteria with complex or multicentric genomes remains unexplored. Here, we compare the methylome, the transcriptome and the phenotypes of wild-type and CcrM-depleted Agrobacterium tumefaciens cells with a dicentric chromosome with two essential replication origins. We find that DNA methylation has a pleiotropic impact on motility, biofilm formation and viability. Remarkably, CcrM promotes the expression of the repABCCh2 operon, encoding proteins required for replication initiation/partitioning at ori2, and represses gcrA, encoding a conserved global cell cycle regulator. Imaging ori1 and ori2 in live cells, we show that replication from ori2 is often delayed in cells with a hypo-methylated genome, while ori2 over-initiates in cells with a hyper-methylated genome. Further analyses show that GcrA promotes the expression of the RepCCh2 initiator, most likely through the repression of a RepECh2 anti-sense RNA. Altogether, we propose that replication at ori1 leads to a transient hemi-methylation and activation of the gcrA promoter, allowing repCCh2 activation by GcrA and contributing to initiation at ori2. This study then uncovers a novel and original connection between CcrM-dependent DNA methylation, a conserved epigenetic regulator and genome maintenance in an Alphaproteobacterial pathogen.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad339078add660717cbec9308e097c1844ce59b8" target='_blank'>
              DNA methylation by CcrM contributes to genome maintenance in the Agrobacterium tumefaciens plant pathogen.
              </a>
            </td>
          <td>
            Sandra Martin, Florian Fournes, Giovanna Ambrosini, Christian Iseli, Karolina Bojkowska, Julien Marquis, Nicolas Guex, Justine Collier
          </td>
          <td>2024-09-04</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Summary DNA sequencing is becoming more affordable and faster through advances in high-throughput technologies. This rise in data availability has contributed to the development of novel algorithms to elucidate previously obscure features and led to an increased reliance on complex workflows to integrate such tools into analyses pipelines. To facilitate the analysis of DNA sequencing data, we created metapipeline-DNA, a highly configurable and extensible pipeline. It encompasses a broad range of processing including raw sequencing read alignment and recalibration, variant calling, quality control and subclonal reconstruction. Metapipeline-DNA also contains configuration options to select and tune analyses while being robust to failures. This standardizes and simplifies the ability to analyze large DNA sequencing in both clinical and research settings. Availability Metapipeline-DNA is an open-source Nextflow pipeline under the GPLv2 license and is freely available at https://github.com/uclahs-cds/metapipeline-DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bce629f819b8f5832564713fdef250ffa1edd564" target='_blank'>
              Metapipeline-DNA: A Comprehensive Germline & Somatic Genomics Nextflow Pipeline
              </a>
            </td>
          <td>
            Yash Patel, Chenghao Zhu, Takafumi N. Yamaguchi, Nicholas K. Wang, Nicholas Wiltsie, Alfredo Gonzalez, Helena Winata, Nicole Zeltser, Yu Pan, M. F. E. Mootor, Timothy Sanders, C. Kandoth, Sorel T. Fitz-Gibbon, J. Livingstone, Lydia Y. Liu, Benjamin Carlin, Aaron Holmes, Jieun Oh, John Sahrmann, Shu Tao, Stefan E. Eng, Rupert Hugh-White, Kiarod Pashminehazar, Andrew Park, Arpi Beshlikyan, Madison Jordan, Selina Wu, Mao Tian, J. Arbet, B. Neilsen, Yuan Zhe Bugh, Gina Kim, Joseph Salmingo, Wenshu Zhang, Roni Haas, Aakarsh Anand, Edward Hwang, Anna Neiman-Golden, Philippa L. Steinberg, Wenyan Zhao, Prateek Anand, Brandon L. Tsai, P. Boutros
          </td>
          <td>2024-09-07</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>39</td>
        </tr>

        <tr id="Clustered regularly interspaced short palindromic repeats (CRISPR)-Cas (CRISPR-associated proteins) systems provide acquired heritable protection to bacteria and archaea against selfish DNA elements, such as viruses. These systems must be tightly regulated because they can capture DNA fragments from foreign selfish elements, and also occasionally from self-chromosomes, resulting in autoimmunity. Most known species from the halophilic archaeal genus Haloferax contain type I-B CRISPR-Cas systems, and the strongest hotspot for self-spacer acquisition by H. mediterranei was a locus that contained a putative transposable element, as well as the gene HFX_2341, which was a very frequent target for self-targeting spacers. To test whether this gene is CRISPR-associated, we investigated it using bioinformatics, deletion, over-expression, and comparative transcriptomics. We show that HFX_2341 is a global transcriptional regulator that can repress diverse genes, since its deletion results in significantly higher expression of multiple genes, especially those involved in nutrient transport. When over-expressed, HFX_2341 strongly repressed the transcript production of all cas genes tested, both those involved in spacer acquisition (cas1, 2 and 4) and those required for destroying selfish genetic elements (cas3 and 5–8). Considering that HFX_2341 is highly conserved in haloarchaea, with homologs that are present in species that do not encode the CRISPR-Cas system, we conclude that it is a global regulator that is also involved in cas gene regulation, either directly or indirectly.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00459123b00f7b1a9dd02b717cf9a4770d4090a0" target='_blank'>
              A Haloarchaeal Transcriptional Regulator That Represses the Expression of CRISPR-Associated Genes
              </a>
            </td>
          <td>
            Israela Turgeman-Grott, Yarden Shalev, Netta Shemesh, Rachel Levy, Inbar Eini, M. Pasmanik-Chor, U. Gophna
          </td>
          <td>2024-08-27</td>
          <td>Microorganisms</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="The cell cycle is a fundamental process essential for cell proliferation, differentiation, and development. It consists of four major phases: G1, S, G2, and M. These phases collectively drive the reproductive cycle and are meticulously regulated by various proteins that play critical roles in both the prevention and progression of cancer. Traditional methods for studying these functions, such as flow cytometry, require a substantial number of cells to ensure accuracy. In this study, we have developed a user-friendly, immunofluorescence-based method for identifying cell cycle stages, providing single-cell resolution and precise identification of G1, early S, late S, early G2, late G2, and each sub-stage of the M phase using fluorescence microscopy. This method provides high-precision cell cycle identification and can serve as an alternative to, or in combination with, traditional flow cytometry to dissect detailed substages of the cell cycle in a variety of cell lines.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c6a0cb1f79ef1a3760e18e969fa1e9d5248adeb4" target='_blank'>
              ImmunoCellCycle-ID: A high-precision immunofluorescence-based method for cell cycle identification
              </a>
            </td>
          <td>
            Yu-Lin Chen, Yu-Chia Chen, Aussie Suzuki
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The fidelity of replication, especially in the presence of DNA damage, is essential for the proper function of cells. Mutations that inactivate genes involved in DNA damage repair or bypass are enriched in several types of cancer cells. Thus, it is important to further our understanding of the mechanisms governing replication fidelity. PCNA is a ring-shaped complex that encircles DNA at the front of the replication fork, at the double-stranded/single-stranded DNA junction. It serves as a processivity factor for the different DNA replication polymerases, allowing them to replicate longer stretches of DNA by physically tethering them to the DNA and preventing their detachment. In addition, PCNA also regulates and coordinates different DNA damage bypass pathways meant to allow DNA replication in the presence of DNA damage. Due to its essentiality and the numerous functions it has in the cell, much is still unclear about PCNA. Here, we utilize PCNA mutants that lower the stability of the PCNA complex on the chromatin, and thus tend to disassociate and fall from the DNA. Using these mutants, we show that PCNA’s physical presence on the DNA can prevent DNA misalignment at repetitive sequences, leading to increased mutation formation. We also show that PCNA-interacting proteins play an important role in strengthening the ring’s stability on the chromatin. Such repetitive sequence-induced mutations are common in several human diseases and it is important to study their formation and the mechanisms guarding against them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4d4a21eb48fc762d33559ed0c2e7196cd781bc5" target='_blank'>
              Effects of PCNA Stability on the Formation of Mutations
              </a>
            </td>
          <td>
            Matan Arbel-Groissman, Batia Liefshitz, M. Kupiec
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Chromosomal instability (CIN), characterized by fluctuations in chromosome number or structure within cells, stands out as a hallmark of cancer, enabling tumors to thrive in hostile conditions. CIN serves as a driver of genetic diversity, giving rise to clonal heterogeneity (CH). Emerging evidence points to a potential correlation between CIN, CH, and the prognosis of breast cancer (BC) patients, especially in tumors exhibiting overexpression of the human epidermal growth factor receptor 2 (HER2+). However, our understanding of the role of CIN in other subtypes of BC is limited. Furthermore, it remains unclear whether CIN levels above a certain threshold in BC tumors could adversely affect tumor growth, or if lower to moderate levels of CIN might be associated with a more favorable prognosis for BC patients compared to elevated levels. Elucidating these relationships could significantly influence risk assessment and the formulation of future therapeutic approaches targeting CIN in BC. This study aimed to assess CIN and CH in tumor tissue samples obtained from Colombian patients diagnosed with luminal A, luminal B, HER2+, or triple-negative BC, and compare them with established clinicopathological parameters. The findings of this study indicate that BC patients, exhibit intermediate CIN, high CH, and stable aneuploidy. All these characteristics were found to be related with clinicopathological features. Our results suggest that the identification of CIN, CH and aneuploidy could improve cancer risk stratification, which could help to clarify the prediction of clinical outcomes and guiding personalized therapeutic strategies for patients with different BC subtypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f23f4c53715c6bdbb1cc8b85a27c36e138c73e29" target='_blank'>
              Exploring chromosomal instability and clonal heterogeneity in breast cancer.
              </a>
            </td>
          <td>
            Giovanny Castellanos, Laura Valentina Camargo-Herrera, N. Rangel, G. A. Jiménez-Tobón, María Martínez-Agüero, Milena Rondón-Lagos
          </td>
          <td>2024-09-01</td>
          <td>Endocrine-related cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Mycosis fungoides (MF) is the most common subtype of cutaneous T-cell lymphoma (CTCL). Comprehensive analysis of MF cells in situ and ex vivo is complicated by the fact that is challenging to distinguish malignant from reactive T cells with certainty. Methods To overcome this limitation, we performed combined single-cell RNA (scRNAseq) and T-cell receptor TCR sequencing (scTCRseq) of skin lesions of cutaneous MF lesions from 12 patients. A sufficient quantity of living T cells was obtained from 9 patients, but 2 had to be excluded due to unclear diagnoses (coexisting CLL or revision to a fixed toxic drug eruption). Results From the remaining patients we established single-cell mRNA expression profiles and the corresponding TCR repertoire of 18,630 T cells. TCR clonality unequivocally identified 13,592 malignant T cells. Reactive T cells of all patients clustered together, while malignant cells of each patient formed a unique cluster expressing genes typical of naive/memory, such as CD27, CCR7 and IL7R, or cytotoxic T cells, e.g., GZMA, NKG7 and GNLY. Genes encoding classic CTCL markers were not detected in all clusters, consistent with the fact that mRNA expression does not correlate linearly with protein expression. Nevertheless, we successfully pinpointed distinctive gene signatures differentiating reactive malignant from malignant T cells: keratins (KRT81, KRT86), galectins (LGALS1, LGALS3) and S100 genes (S100A4, S100A6) being overexpressed in malignant cells. Conclusions Combined scRNAseq and scTCRseq not only allows unambiguous identification of MF cells, but also revealed marked heterogeneity between and within patients with unexpected functional phenotypes. While the correlation between mRNA and protein abundance was limited with respect to established MF markers, we were able to identify a single-cell gene expression signature that distinguishes malignant from reactive T cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/212bc68159cf864c0e23d5af31516d6565474ee3" target='_blank'>
              Single-cell RNA and T-cell receptor sequencing unveil mycosis fungoides heterogeneity and a possible gene signature
              </a>
            </td>
          <td>
            Nalini Srinivas, Lukas Peiffer, K. Horny, Kuan Cheok Lei, T. B. Buus, Linda Kubat, Meng Luo, Menghong Yin, Ivelina Spassova, A. Sucker, Farnoush Farahpour, Jan Kehrmann, Selma Ugurel, E. Livingstone, T. Gambichler, Niels Ødum, Jürgen C. Becker
          </td>
          <td>2024-08-07</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de9b939e6665a85e15c06d7919e4ac22bd18c3f" target='_blank'>
              Cellular origin and clonal evolution of human dedifferentiated liposarcoma
              </a>
            </td>
          <td>
            N. Gruel, C. Quignot, Laëtitia Lesage, Sophie El Zein, S. Bonvalot, Dimitri Tzanis, Khadija Ait Rais, Fabien Quinquis, Bastien Manciot, J. Vibert, Nadine El Tannir, A. Dahmani, Héloise Derrien, Didier Decaudin, Ivan Bièche, L. Courtois, O. Mariani, Laetitia K Linares, L. Gayte, Sylvain Baulande, J. Waterfall, O. Delattre, G. Pierron, Sarah Watson
          </td>
          <td>2024-09-12</td>
          <td>Nature Communications</td>
          <td>2</td>
          <td>40</td>
        </tr>

        <tr id="KMT2A (alias: mixed-lineage leukemia [MLL]) gene mapping on chromosome 11q23 encodes the lysine-specific histone N-methyltransferase 2A and promotes transcription by inducing an open chromatin conformation. Numerous genomic breakpoints within the KMT2A gene have been reported in young children and adults with hematologic disorders and are present in up to 10% of acute leukemias. These rearrangements describe distinct features and worse prognosis depending on the fusion partner, characterized by chemotherapy resistance and high rates of relapse, with a progression-free survival of 30–40% and overall survival below 25%. Less intensive regimens are used in pediatric patients, while new combination therapies and targeted immunotherapeutic agents are being explored in adults. Beneficial therapeutic effects, and even cure, can be reached with hematopoietic stem cell transplantation, mainly in young children with dismal molecular lesions; however, delayed related toxicities represent a concern. Herein, we summarize the translocation partner genes and partial tandem duplications of the KMT2A gene, their molecular impact, clinical aspects, and novel targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fd79253a028be52b0b23fd93ce3fdeb9b8b1453" target='_blank'>
              KMT2A Rearrangements in Leukemias: Molecular Aspects and Therapeutic Perspectives
              </a>
            </td>
          <td>
            Luca Guarnera, Matteo D’Addona, Carlos Bravo-Perez, V. Visconte
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7d183f3062975c60ce5ba3ada1020c80a06fee4" target='_blank'>
              Prion-like domain mediated phase separation of ARID1A promotes oncogenic potential of Ewing’s sarcoma
              </a>
            </td>
          <td>
            Yong Ryoul Kim, Jaegeon Joo, Hee Jung Lee, Chaelim Kim, Ju-Chan Park, Young Suk Yu, Chang Rok Kim, Do Hui Lee, Joowon Cha, Hyemin Kwon, Kimberley M. Hanssen, T. G. Grünewald, Murim Choi, Ilkyu Han, Sangsu Bae, Inkyung Jung, Yongdae Shin, Sung Hee Baek
          </td>
          <td>2024-08-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The DNA damage repair (DDR) pathway is a complex signaling cascade that can sense DNA damage and trigger cellular responses to DNA damage to maintain genome stability and integrity. A typical hallmark of cancer is genomic instability or nonintegrity, which is closely related to the accumulation of DNA damage within cancer cells. The treatment principles of radiotherapy and chemotherapy for cancer are based on their cytotoxic effects on DNA damage, which are accompanied by severe and unnecessary side effects on normal tissues, including dysregulation of the DDR and induced therapeutic tolerance. As a driving factor for oncogenes or tumor suppressor genes, noncoding RNA (ncRNA) have been shown to play an important role in cancer cell resistance to radiotherapy and chemotherapy. Recently, it has been found that ncRNA can regulate tumor treatment tolerance by altering the DDR induced by radiotherapy or chemotherapy in cancer cells, indicating that ncRNA are potential regulatory factors targeting the DDR to reverse tumor treatment tolerance. This review provides an overview of the basic information and functions of the DDR and ncRNAs in the tolerance or sensitivity of tumors to chemotherapy and radiation therapy. We focused on the impact of ncRNA (mainly microRNA [miRNA], long noncoding RNA [lncRNA], and circular RNA [circRNA]) on cancer treatment by regulating the DDR and the underlying molecular mechanisms of their effects. These findings provide a theoretical basis and new insights for tumor-targeted therapy and the development of novel drugs targeting the DDR or ncRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63af0c8bea30d510199d8c53e8bb1fd81c1002b8" target='_blank'>
              DNA damage response-related ncRNAs as regulators of therapy resistance in cancer
              </a>
            </td>
          <td>
            Ziru Gao, Xinchi Luan, Xuezhe Wang, Tianyue Han, Xiaoyuan Li, Zeyang Li, Peifeng Li, Zhixia Zhou
          </td>
          <td>2024-08-26</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract A major function of the DNA damage responses (DDRs) that act during the replicative phase of the cell cycle is to inhibit initiation and elongation of DNA replication. It has been shown that DNA replication of the polyomavirus, SV40, is inhibited and its replication fork is slowed by cellular DDR responses. The inhibition of SV40 DNA replication is associated with enhanced DDR kinase phosphorylation of SV40 Large T-antigen (LT), the viral DNA helicase. Mass spectroscopy was used to identify a novel highly conserved DDR kinase site, T518, on LT. In cell-based assays expression of a phosphomimetic form of LT at T518 (T518D) resulted in dramatically decreased levels of SV40 DNA replication, but LT-dependent transcriptional activation was unaffected. Purified WT and LT T518D were analyzed in vitro. In concordance with the cell-based data, reactions using SV40 LT-T518D, but not T518A, showed dramatic inhibition of SV40 DNA replication. A myriad of LT protein-protein interactions and LT’s biochemical functions were unaffected by the LT T518D mutation; however, LT’s DNA helicase activity was dramatically decreased on long, but not very short, DNA templates. These results suggest that DDR phosphorylation at T518 inhibits SV40 DNA replication by suppressing LT helicase activity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6419b03aac39bf7fba65ee38aeb527e0b46ff9c3" target='_blank'>
              DNA damage-induced phosphorylation of a replicative DNA helicase results in inhibition of DNA replication through attenuation of helicase function
              </a>
            </td>
          <td>
            Caleb Homiski, Rama Dey-Rao, Shichen Shen, Jun Qu, T. Melendy
          </td>
          <td>2024-08-10</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="DNAJB6, a major member of the DNAJ/HSP40 family, plays an important role in tumor development. We explored the effect of DNAJB6 expression on the prognosis of patients and its biological role in lung adenocarcinoma (LUAD). mRNA and clinical data were obtained from The Cancer Genome Atlas (TCGA). Enriched pathways were determined by the Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses. A nomogram incorporating DNAJB6 and three clinical features was constructed to predict the survival rate. DNAJB6 expression and function in LUAD were explored using immunohistochemistry, Western blotting, proliferation, cell cycle analysis, RNA sequencing, and xenograft tumor assays. DNAJB6 mRNA levels were elevated in the LUAD-TCGA dataset. DNAJB6 protein levels were higher in LUAD tumor tissues than in normal tissues. A high DNAJB6 level was an independent risk factor for poor prognosis in patients with LUAD. The proportion of tumor-infiltrating immune cells significantly differed between high and low DNAJB6 expression. DNAJB6 was associated with cell cycle pathways; therefore, its knockdown induced G2/M cell cycle arrest and inhibited LUAD cell proliferation. This is the first report of the DNAJB6 requirement for LUAD cell proliferation and its potentially crucial role in LUAD prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd99e6105741818b2b8a1ce73bd38d2fcdc17926" target='_blank'>
              Abnormally High Expression of DNAJB6 Accelerates Malignant Progression of Lung Adenocarcinoma
              </a>
            </td>
          <td>
            Di Wang, Jiayu Xiao, Yang Du, Li Zhang, Xuzhen Qin
          </td>
          <td>2024-09-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261ea82053aaf73b2c3fb5ebe769c19bae07d5fa" target='_blank'>
              53BP1 loss elicits cGAS-STING-dependent antitumor immunity in ovarian and pancreatic cancer
              </a>
            </td>
          <td>
            Yajie Sun, Jeffrey Patterson-Fortin, Sen Han, Zhe Li, Zuzanna Nowicka, Yuna Hirohashi, Susan Kilgas, Jae Kyo Yi, A. Spektor, Wojciech Fendler, Panagiotis A Konstantinopoulos, Dipanjan Chowdhury
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Radiotherapy (RT) is the primary treatment for diffuse midline glioma (DMG), a lethal pediatric malignancy defined by histone H3 lysine 27-to-methionine (H3K27M) mutation. Based on the loss of H3K27 trimethylation producing broad epigenomic alterations, we hypothesized that H3K27M causes a functional double-strand break (DSB) repair defect that could be leveraged therapeutically with PARP inhibitor and RT for selective radiosensitization and antitumor immune responses. Methods H3K27M isogenic DMG cells and orthotopic brainstem DMG tumors in immune deficient and syngeneic, immune competent mice were used to evaluate the efficacy and mechanisms of PARP1/2 inhibition by olaparib or PARP1 inhibition by AZD9574 with concurrent RT. Results H3K27M mutation caused an HRR defect characterized by impaired RT-induced K63-linked polyubiquitination of histone H1 and inhibition of HRR protein recruitment. H3K27M DMG cells were selectively radiosensitized by olaparib in comparison to isogenic controls, and this effect translated to efficacy in H3K27M orthotopic brainstem tumors. Olaparib and RT induced an innate immune response and induction of NK cell (NKG2D) activating ligands leading to increased NK cell-mediated lysis of DMG tumor cells. In immunocompetent syngeneic orthotopic DMG tumors, either olaparib or AZD9574 in combination with RT enhanced intratumoral NK cell infiltration and activity in association with NK cell-mediated therapeutic responses and favorable activity of AZD9574. Conclusions The HRR deficiency in H3K27M DMG can be therapeutically leveraged with PARP inhibitors to radiosensitize and induce an NK cell-mediated antitumor immune response selectively in H3K27M DMG, supporting the clinical investigation of best-in-class PARP inhibitors with RT in DMG patients. Key points H3K27M DMG are HRR defective and selectively radiosensitized by PARP inhibitor. PARP inhibitor with RT enhances NKG2D ligand expression and NK cell-mediated lysis. NK cells are required for the therapeutic efficacy of PARP inhibitor and RT. Importance of the Study Radiotherapy is the cornerstone of H3K27M-mutant diffuse midline glioma treatment, but almost all patients succumb to tumor recurrence with poor overall survival, underscoring the need for RT-based precision combination therapy. Here, we reveal HRR deficiency as an H3K27M-mediated vulnerability and identify a novel mechanism linking impaired RT-induced histone H1 polyubiquitination and the subsequent RNF168/BRCA1/RAD51 recruitment in H3K27M DMG. This model is supported by selective radiosensitization of H3K27M DMG by PARP inhibitor. Notably, the combination treatment results in NKG2D ligand expression that confers susceptibility to NK cell killing in H3K27M DMG. We also show that the novel brain penetrant, PARP1-selective inhibitor AZD9574 compares favorably to olaparib when combined with RT, prolonging survival in a syngeneic orthotopic model of H3K27M DMG. This study highlights the ability of PARP1 inhibition to radiosensitize and induce an NK cell-mediated antitumor immunity in H3K27M DMG and supports future clinical investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa409894637aca4cc204fe5ba28199f9da5c66db" target='_blank'>
              H3K27M diffuse midline glioma is homologous recombination defective and sensitized to radiotherapy and NK cell-mediated antitumor immunity by PARP inhibition
              </a>
            </td>
          <td>
            Yupei Guo, Zian Li, L. A. Parsels, Zhuwen Wang, J. Parsels, Anushka Dalvi, Stephanie The, Nan Hu, Victoria M. Valvo, Robert Doherty, Erik Peterson, Xinjun Wang, S. Venkataraman, S. Agnihotri, S. Venneti, Daniel R. Wahl, Michael D. Green, Theodore S. Lawrence, C. Koschmann, Meredith A. Morgan, Qiang Zhang
          </td>
          <td>2024-08-27</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Histones are hyper modified proteins that regulate chromatin accessibility and DNA readout. Co-existing post-translational modifications (PTMs) on histones affect interaction affinities of chromatin-associated proteins in ways that are still mostly unexplored. Here, we focus on the biological role of a specific histone code made of two PTMs with supposedly opposing biological functions, i.e. H3K9me3 marker of constitutive heterochromatin and H3K14ac benchmarking accessible chromatin. By applying multi-dimensional mass spectrometry, we demonstrated that EXOSC4 interacts with H3K9me3 + acetyls and affects post-transcriptional regulation. Specifically, EXOSC4 depletion leads to down-regulation of the RNA surveillance machinery and increased expression of non-coding transcripts, including anti-sense RNAs. Together, we elucidate the role of a co-modified histone tail and demonstrate its involvement in the RNA machinery and spurious transcription surveillance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/414a923cc5fe1402e6de4d18710208900c79f29c" target='_blank'>
              EXOSC4 is recruited by histone H3 co-modified with K9me3 and acetylations to surveil non-coding transcription
              </a>
            </td>
          <td>
            Stephanie Stransky, Ronald Cutler, Jennifer T. Aguilan, Joseph D. DeAngelo, D. Shechter, Simone Sidoli
          </td>
          <td>2024-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Deleterious accumulation of R-loops, a DNA-RNA hybrid structure, contributes to genome instability. They are associated with BRCA1 mutation-related breast cancer, an estrogen receptor α negative (ERα-) tumor type originating from luminal progenitor cells. However, a presumed causality of R-loops in tumorigenesis has not been established in vivo. Here, we overexpress mouse Rnaseh1 (Rh1-OE) in vivo to remove accumulated R-loops in Brca1-deficient mouse mammary epithelium (BKO). R-loop removal exacerbates DNA replication stress in proliferating BKO mammary epithelial cells, with little effect on homology-directed repair of double-strand breaks following ionizing radiation. Compared to their BKO counterparts, BKO-Rh1-OE mammary glands contain fewer luminal progenitor cells but more mature luminal cells. Despite a similar incidence of spontaneous mammary tumors in BKO and BKO-Rh1-OE mice, a significant percentage of BKO-Rh1-OE tumors express ERα and progesterone receptor. Our results suggest that rather than directly elevating the overall tumor incidence, R-loops influence the mammary tumor subtype by shaping the cell of origin for Brca1 tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd59964db49fa8717ee1e81f17faaff9786150ba" target='_blank'>
              R-loop functions in Brca1-associated mammary tumorigenesis.
              </a>
            </td>
          <td>
            Huai-Chin Chiang, Leilei Qi, Payal Mitra, Yimeng Huang, Yanfen Hu, Rong Li
          </td>
          <td>2024-08-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="TAD boundaries are genomic elements that separate biological processes in neighboring domains by blocking DNA loops that are formed through Cohesin-mediated loop extrusion. Most TAD boundaries consist of arrays of binding sites for the CTCF protein, whose interaction with the Cohesin complex blocks loop extrusion. TAD boundaries are not fully impermeable though and allow a limited amount of inter-TAD loop formation. Based on the reanalysis of Nano-C data, a multicontact Chromosome Conformation Capture assay, we propose a model whereby clustered CTCF binding sites promote the successive stalling of Cohesin and subsequent dissociation from the chromatin. A fraction of Cohesin nonetheless achieves boundary read-through. Due to a constant rate of Cohesin dissociation elsewhere in the genome, the maximum length of inter-TAD loops is restricted though. We speculate that the DNA-encoded organization of stalling sites regulates TAD boundary permeability and discuss implications for enhancer-promoter loop formation and other genomic processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7707b0397caa81eae89d125f02c4b85e10532d65" target='_blank'>
              Permeable TAD boundaries and their impact on genome-associated functions.
              </a>
            </td>
          <td>
            Li-Hsin Chang, D. Noordermeer
          </td>
          <td>2024-08-02</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c115fcb82a4ae08aa16b7db64204e01871cc9b8b" target='_blank'>
              Prognostic value of nucleotide excision repair and translesion DNA synthesis proteins in muscle-infiltrating bladder carcinoma
              </a>
            </td>
          <td>
            P. Palacka, Andrea Holíčková, J. Roška, Peter Makovický, M. Vallová, Csaba Biró, Eveline Órásová, J. Obertová, J. Mardiak, Thomas A Ward, Karol Kajo, M. Chovanec
          </td>
          <td>2024-09-05</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="During tumor development, promoter CpG islands that are normally silenced by Polycomb repressive complexes (PRCs) become DNA-hypermethylated. The molecular mechanism by which de novo DNA methyltransferase(s) [DNMT(s)] catalyze CpG methylation at PRC-regulated regions remains unclear. Here, we report a cryo–electron microscopy structure of the DNMT3A long isoform (DNMT3A1) amino-terminal region in complex with a nucleosome carrying PRC1-mediated histone H2A lysine-119 monoubiquitination (H2AK119Ub). We identify regions within the DNMT3A1 amino terminus that bind H2AK119Ub and the nucleosome acidic patch. This bidentate interaction is required for effective DNMT3A1 engagement with H2AK119Ub-modified chromatin in cells. Further, aberrant redistribution of DNMT3A1 to Polycomb target genes recapitulates the cancer-associated DNA hypermethylation signature and inhibits their transcriptional activation during cell differentiation. This effect is rescued by disruption of the DNMT3A1–acidic patch interaction. Together, our analyses reveal a binding interface critical for mediating promoter CpG island DNA hypermethylation, a major molecular hallmark of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63f64e00b184c0e797697cd862ef32ddd61be317" target='_blank'>
              Cancer-associated DNA hypermethylation of Polycomb targets requires DNMT3A dual recognition of histone H2AK119 ubiquitination and the nucleosome acidic patch
              </a>
            </td>
          <td>
            Kristjan H. Gretarsson, S. Abini-Agbomson, Susan L Gloor, Daniel N. Weinberg, Jamie L McCuiston, V. S. Kumary, Allison Hickman, Varun Sahu, Rachel Lee, Xinjing Xu, Natalie Lipieta, Samuel Flashner, Oluwatobi A Adeleke, Irina K Popova, Hailey F Taylor, Kelsey Noll, C. L. Windham, Danielle N Maryanski, Bryan J Venters, Hiroshi Nakagawa, Michael-Christopher Keogh, K. Armache, Chao Lu
          </td>
          <td>2024-08-28</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="ABSTRACT Background Prostate cancer (PCa) is among the three main types of cancer. Although prostate-specific antigen (PSA) is routinely tested, it has disadvantages, such as poor prognostic ability. Therefore, finding more PCa markers and therapeutic targets remains a subject of study. CircRNAs have been found to have regulatory roles in various diseases, such as diabetes, Central Nervous System (CNS) neuropathy, etc. where their application in cancer is even more valuable. Therefore, this paper aims to search for differentially expressed circRNAs in PCa and find downstream targeting pathways related to autophagy. Method By detecting the expression of circRNA in the samples, hsa_circ_0119816 was finally identified as the research target. The properties of circRNA were verified by RNase R, actinomycin D, and fluorescence in situ hybridization (FISH). The downstream target miRNAs and target proteins were predicted by an online database, and the targeting relationship was verified using dual luciferase and RNA Immunoprecipitation. The effects of circRNAs and their downstream signalling pathways on prostate cancer cell proliferation, migration, EMT and autophagy were examined by CCK-8, Transwell, immunofluorescence and Western blotting. Results CircBIRC6 is highly expressed in prostate cancer samples. Knockdown of its expression inhibits cell proliferation, invasion, EMT and autophagy and promotes apoptosis. CircBIRC6/miRNA-574-5p/DNAJB1 is a molecular axis that regulates prostate cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f9d1bad6dbf3ace4247e9abe87ef37ab3c3b685" target='_blank'>
              CircBIRC6 affects prostate cancer progression by regulating miR-574-5p and DNAJB1
              </a>
            </td>
          <td>
            Bin Zhao, Jinye Yang, Fengming Ran, Yuanlong Shi, Libo Yang, Yuanpeng Duan, Zhiyu Shi, Xin Li, Jianpeng Zhang, Zhiyao Li, Jiansong Wang
          </td>
          <td>2024-09-11</td>
          <td>Cancer Biology & Therapy</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54604fdc43654c5fa63c4063c1f8c16948e89144" target='_blank'>
              DNA repair-dependent immunogenic liabilities in colorectal cancer: opportunities from errors.
              </a>
            </td>
          <td>
            V. Amodio, P. P. Vitiello, A. Bardelli, G. Germano
          </td>
          <td>2024-09-13</td>
          <td>British journal of cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Human DNA methylation profiling offers great promises in various biomedical applications, including ageing, cancer and even forensics. So far, most DNA methylation techniques are based on a chemical process called sodium bisulfite conversion, which specifically converts non-methylated cytosines into uracils. However, despite the popularity of this approach, it is known to cause DNA fragmentation and loss affecting standardization, while incomplete conversion may result in potential misinterpretation of methylation-based outcomes. To offer the community a solution, we developed qBiCo - a novel quality-control method to address the quantity and quality of bisulfite-converted DNA. qBiCo is a 5-plex, TaqMan® probe-based, quantitative (q)PCR assay that amplifies single- and multi-copy DNA fragments of converted and non-converted nature. It estimates four parameters: converted DNA concentration, fragmentation, global conversion efficiency, and potential PCR inhibition. We optimized qBiCo using synthetic DNA standards and assessed it using standard developmental validation criteria, showcasing that qBiCo is reliable, robust and sensitive down to picogram level. We also evaluated its performance by testing decreasing DNA amounts using several commercial bisulfite conversion kits. Depending on the starting DNA quantity, bisulfite-converted DNA recoveries ranged from 8.5 – 100 %, conversion efficiencies from 78 - 99.9 %, while certain kits highly fragment DNA, demonstrating large variability in their performance. Towards building a prototype tool, we further optimized key functionalities, for example, by replacing the poorest performing single-plex assay and creating a more representative DNA standard. Aiming to scale-up and move towards implementation, we successfully transferred and validated our novel method in six different qPCR platforms from different major manufacturers. Overall, with the present study, we offer researchers in the epigenetic field a novel long-awaited QC tool that for the first time allows them to measure key quality and quantity parameters of the most popular DNA conversion process. The tool also enables standardization to prevent inconsistent data and false outcomes in the future, regardless of the downstream experimental analysis of DNA methylation-based research and applications across different fields of biology and biomedicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea5f2d2f1ff7d9e8b8418047a193bf39b8fe5d70" target='_blank'>
              qBiCo: A method to assess global DNA conversion performance in epigenetics via single-copy genes and repetitive elements
              </a>
            </td>
          <td>
            Faidra Karkala, Roy B. Simons, Floor Claessens, V. Kalamara, M. Kayser, A. Vidaki
          </td>
          <td>2024-08-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal cancer (CRC) is one of the major causes of morbidity and mortality worldwide, resulting from the accumulation of genetic and epigenetic alterations in several oncogenes and tumor suppressor genes. Recent studies have identified germline and somatic mutations in the exonuclease domain regions of both epsilon polymerase (POLE) and delta polymerase (POLD1) genes in CRCs. We sought to examine the mutation of these genes in a series of sporadic CRCs. To do this, we extracted DNA from 100 primary CRC samples and 40 corresponding normal tissue samples, which had been previously characterized for clinicopathological and molecular features. We employed a combination of quick-multiplex consensus (QMC)-polymerase chain reaction (PCR) and co-amplification at lower denaturation temperature (COLD)-PCR, followed by high-resolution melting (HRM) analysis and Sanger sequencing, to investigate the exonuclease domain regions of POLE and POLD1 genes for somatic mutations that may potentially alter the proofreading activities of these genes. In silico predictions of the functional significance of the identified genetic alterations were performed using the protein variation effect analyzer, sorting intolerant from tolerant, and PON-P2 algorithms. We identified a total of eight new and non-recurrent somatic variants in the endonuclease domains of POLE and two in POLD1. Nine out of ten variants caused amino acid substitutions, whereas one resulted in a stop codon. Although no significant associations or correlations were found between the POLE/POLD1 mutations and the clinicopathological or molecular features of the CRC cases, most of the POLE/POLD1-mutated cases were microsatellite stable (90%) and aneuploid (80%). Furthermore, in silico analyses showed that nine of the ten variants would likely cause some adverse effect on protein function. Ten somatic variants with predicted proofreading activity-altering effects have been identified in the endonuclease domains of POLE and POLD1 in CRC using a combination of QMC-PCR, COLD-PCR, HRM analyses, and Sanger sequencing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22443d76e404b9ff0c03738771a51c34f113cb72" target='_blank'>
              Exploring somatic mutations in POLE and POLD1: Their role in colorectal cancer pathogenesis and potential therapeutic strategies
              </a>
            </td>
          <td>
            Hersh A. Ham-Karim, Narmeen Ahmad, Mohammad Ilyas
          </td>
          <td>2024-09-06</td>
          <td>Tumor Discovery</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="
 Forkhead box A1 (FOXA1) acts as a pioneer factor for Estrogen Receptor alpha (ERα), playing a crucial role in determining where ERα binds to DNA and regulates gene expression in breast cancer cells. A pioneer factor is a class of transcription factors (TF) that is responsible for opening the chromatin to allow other TFs to attach and activate transcription of that gene. FOXA1 acts as a facilitator for ERα-DNA interactions. It binds to specific DNA regions, opening the chromatin for ERα to bind. It “pre-marks” these sites in the genome even in the absence of estrogen. In fact, FOXA1 is frequently overexpressed in ER+ breast cancers, suggesting it has a crucial role in driving tumor growth. As such, FOXA1 is an attractive therapeutic target that already serves as a great biomarker for many types of breast cancers. Inhibiting FOXA1 can potentially block ER activity and inhibit growth of ER+ breast cancer, especially in those resistant to anti-estrogen therapies, such as tamoxifen. In this project, we identified sites that FOXA1 opens and correlated them with SNP mutations commonly found in the Hispanic population. The presence of these SNPs portends a worse outcome in Hispanic patients. To assess which open chromatin regions are crucial for breast cancer pathogenesis, a ChIP-seq analysis was done on vehicle-treated and estrogen-treated samples to determine FOXA1 binding sites. After identifying gained, lost, and differentially expressed peaks set, these were compared to genome-wide association studies (GWAS) of Hispanic patients to develop an understanding of how much overlap is common between sites of mutations and open sections of chromatin due to FOXA1. Through our analysis, we uncovered a subset of FOXA1 binding sites that overlap with sites of inherited mutation in Hispanic patients. This informs on the subset of genes that are linked to breast cancer and potential therapeutic targets.
 Citation Format: Hemakshat Sharma, Alyssa Maria Arreola, Frances L. Heredia, Hector L. Franco, Julie Dutil. FOXA1 as a pioneer factor in ERα-DNA interactions and its therapeutic potential in ER+ breast cancer: Insights from ChIP-seq and GWAS in Hispanic populations [abstract]. In: Proceedings of the 17th AACR Conference on the Science of Cancer Health Disparities in Racial/Ethnic Minorities and the Medically Underserved; 2024 Sep 21-24; Los Angeles, CA. Philadelphia (PA): AACR; Cancer Epidemiol Biomarkers Prev 2024;33(9 Suppl):Abstract nr B011.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e221203c9b5991c2cb9afa79cdfe37b8baf68cb3" target='_blank'>
              Abstract B011: FOXA1 as a pioneer factor in ERα-DNA interactions and its therapeutic potential in ER+ breast cancer: Insights from ChIP-seq and GWAS in Hispanic populations
              </a>
            </td>
          <td>
            Hemakshat Sharma, Alyssa Maria Arreola, Frances L. Heredia, Hector L. Franco, Julie Dutil
          </td>
          <td>2024-09-21</td>
          <td>Cancer Epidemiology, Biomarkers &amp; Prevention</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="
 Cancers evolve not only through the acquisition and clonal transmission of somatic mutations but also by non-genetic mechanisms that modify cell phenotype. Such epigenetic causes of intra-tumoral heterogeneity may be particularly important in pediatric cancers with low rates of genomic mutations. Hepatoblastoma, the most common primary liver malignancy in children, has one of the lowest mutational burdens of all cancers, providing an ideal system to elucidate non-genetic mechanisms of intra-tumoral heterogeneity. Up to 80-90% of hepatoblastomas have activating mutations in CTNNB1 and the Wnt pathway. Yet, despite their simple genomics, these tumors exhibit significant histologic heterogeneity, including epithelial (embryonal and fetal histology) and mesenchymal components. Using transcriptomics and patient-derived tumoroids, we find that hepatobiliary lineage programs modulate the transcriptional outcome of mutated CTNNB1. A subset of cholangiocytic-like tumor cells express the growth factor FGF19, which promotes proliferation of neighboring cells in conjunction with activated Wnt signaling. To characterize the transcriptional profiles of the distinct histologic components of hepatoblastoma, we used laser capture microdissection to isolate tumor regions identified as embryonal, fetal, or mesenchymal histology, as well as adjacent normal liver. While all tumor regions showed increased expression of Wnt target genes as compared to normal liver, the embryonal components showed even higher expression of Wnt target genes as compared to the fetal components, as well as increased proliferation markers and higher expression of the embryonic biliary transcription factor SOX4. RNA in situ hybridization in primary hepatoblastomas revealed distinctive foci of embryonal hepatoblastoma cells that express the growth factor FGF19. These FGF19-expressing cells were surrounded by FGF19-negative cells that expressed KLB, the co-receptor for FGF19 signaling, and showed an increased proliferation. We therefore hypothesized that FGF19 acts in a paracrine fashion to drive cell cycle progression of hepatoblastoma cells in cooperation with constitutively active Wnt signaling. We further generated patient-derived tumoroids to systematically test the requirements for exogenous growth factors. We found that despite activated Wnt signaling, FGF19-negative tumor cells required additional exogenous growth factors to proliferate. In some tumoroids, subsets of cells expressed FGF19 endogenously, downstream of Wnt/beta-catenin and SOX4, acting as a paracrine signal. We conclude that the embryonic biliary transcription factor SOX4 cooperates with mutated beta-catenin, inducing FGF19 as a paracrine growth signal that can promote proliferation of FGF19-negative tumor cells. Thus, in this pediatric cancer presumed to originate from a multipotent hepatobiliary progenitor, lineage-driven heterogeneity results in a functional growth advantage, a non-genetic mechanism whereby developmental lineage programs influence tumor evolution, with implications for treatment.
 Citation Format: Peng V. Wu, Matt Fish, Florette K. Hazard, Chunfang Zhu, Sujay Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse. Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr B024.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da0809899c0e27593ad23e793e639268c8de6597" target='_blank'>
              Abstract B024: Single cell transcriptomics and patient-derived tumoroids reveal that hepatobiliary lineage programs cooperate with Wnt pathway mutations to drive cell proliferation in hepatoblastoma
              </a>
            </td>
          <td>
            Peng V. Wu, Matt Fish, F. Hazard, Chunfang Zhu, S. Vennam, Hannah Walton, Joanna Przybyl, Maurizio Morri, Norma Neff, Robert B. West, Roel Nusse
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Abstract Purine and pyrimidine nucleotides are crucial building blocks for the survival of cells, and there are layers of pathways to make sure a stable supply of them including de novo nucleotide biosynthesis. Fast-growing cells including cancer cells have high demand for nucleotide, and they highly utilize the nucleotide biosynthesis pathways. Due to the nature of the fast-growing cells, they tend to make more errors in replication compared with the normal cells. Naturally, DNA repair and DNA lesion bypass are heavily employed in cancer cells to ensure fidelity and completion of the replication without stalling. There have been a lot of drugs targeting cancer that mimic the chemical structures of the nucleobase, nucleoside, and nucleotides, and the resistance toward those drugs is a serious problem. Herein, we have reviewed some of the representative nucleotide analog anticancer agents such as 5-fluorouracil, specifically their mechanism of action and resistance is discussed. Also, we have chosen several enzymes in nucleotide biosynthesis, DNA repair, and DNA lesion bypass, and we have discussed the known and potential roles of these enzymes in maintaining genomic fidelity and cancer chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7c452f57e681001ec1bbb61831e85fc6edf3296" target='_blank'>
              Connecting dots between nucleotide biosynthesis and DNA lesion repair/bypass in cancer
              </a>
            </td>
          <td>
            Jackson C Lin, Ayobami Oludare, Hunmin Jung
          </td>
          <td>2024-08-27</td>
          <td>Bioscience Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Chronic lymphocytic leukemia (CLL) and multiple myeloma (MM) are hematological disorders affecting B cells. The clonal relationship between CLL and MM has not always been clarified, although this information is critical to understanding its pathogenesis. Here, we present a rare clinical case of synchronous CLL and MM. Whole-genome sequencing (WGS) was performed using malignant lymph node (LN) and bone marrow (BM) tissues. Based on the high consistency of single nucleotide variants (SNVs), significantly mutated genes (SMGs), copy number variations (CNVs), different B cell receptor (BCR) IGH rearrangement features in LN and BM, and the different light-chain expression patterns in CLL and MM cells, we concluded that CLL and MM cells from this patient originated from the same hematopoietic stem cell/progenitors, different pro-B cells and suffered oncogenic mutations at different B cell differentiation stages. Depth analysis of genome features using WGS provides a new method to explore the process of malignant B cell genesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df7ddfd6fb847af7298f2ad4a41e1b29f8a44b77" target='_blank'>
              Whole-genome sequencing reveals cellular origin of concomitant chronic lymphocytic leukemia and multiple myeloma: a case report.
              </a>
            </td>
          <td>
            Jianing Zhang, Ji Ma, Ying Li, Xiao Lv, Lili Feng
          </td>
          <td>2024-09-18</td>
          <td>Cancer biology & therapy</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Glioblastoma is the most common primary malignant brain tumour. Despite decades of intensive research in the disease, its prognosis remains poor, with an average survival of only 14 months after diagnosis. The remarkable level of intra- and interpatient heterogeneity is certainly contributing to the lack of progress in tackling this tumour. Epigenetic dysregulation plays an important role in glioblastoma biology and significantly contributes to intratumour heterogeneity. However, it is becoming increasingly clear that it also contributes to intertumour heterogeneity, which historically had mainly been linked to diverse genetic events occurring in different patients. In this review, we explore how DNA methylation, chromatin remodelling, microRNA (miRNA) dysregulation, and long noncoding RNA (lncRNA) alterations contribute to intertumour heterogeneity in glioblastoma, including its implications for advanced tumour stratification, which is the essential first step for developing more effective patient-specific therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8a905205728cf89ec2231b745c2dd76f674ecb4" target='_blank'>
              Unravelling the mosaic: Epigenetic diversity in glioblastoma.
              </a>
            </td>
          <td>
            Sara Lucchini, Myrianni Constantinou, Silvia Marino
          </td>
          <td>2024-08-15</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Breast cancer (BC) is the most frequently diagnosed cancer and the primary cause of cancer-related mortality in women. Treatment of triple-negative breast cancer (TNBC) remains particularly challenging due to its resistance to chemotherapy and poor prognosis. Extensive research efforts in BC screening and therapy have improved clinical outcomes for BC patients. Therefore, identifying reliable biomarkers for TNBC is of great clinical importance. Here, we found that tyrosine aminotransferase (TAT) expression was significantly reduced in BC and strongly correlated with the poor prognosis of BC patients, which distinguished BC patients from normal individuals, indicating that TAT is a valuable biomarker for early BC diagnosis. Mechanistically, we uncovered that methylation of the TAT promoter was significantly increased by DNA methyltransferase 3 (DNMT3A/3B). In addition, reduced TAT contributes to DNA replication and cell cycle activation by regulating homologous recombination repair and mismatch repair to ensure genomic stability, which may be one of the reasons for TNBC resistance to chemotherapy. Furthermore, we demonstrated that Diazinon increases TAT expression as an inhibitor of DNMT3A/3B and inhibits the growth of BC by blocking downstream pathways. Taken together, we revealed that TAT is silenced by DNMT3A/3B in BC, especially in TNBC, which promotes the proliferation of tumor cells by supporting DNA replication, activating cell cycle, and enhancing DNA damage repair. These results provide fresh insights and a theoretical foundation for the clinical diagnosis and treatment of BC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98a77f352d33a7a2b9ba7ce61fe82a2e23ebd12e" target='_blank'>
              Identification of TAT as a Biomarker Involved in Cell Cycle and DNA Repair in Breast Cancer
              </a>
            </td>
          <td>
            Fei Xie, Saiwei Hua, Yajuan Guo, Taoyuan Wang, Changliang Shan, Lianwen Zhang, Tao He
          </td>
          <td>2024-08-30</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Pediatric acute lymphoblastic leukemia (ALL) is a highly heterogeneous disease at the molecular level, with over 20 subtypes of B-cell ALL (B-ALL) identified to date. These subtypes are critical for guiding risk-adapted therapy and precision medicine, as they are defined by specific chromosomal rearrangements, gene expression profiles, aneuploidies and point mutations. However, the ability to accurately classify these subtypes is limited by the technical and economic challenges faced by many centers, particularly in low- and middle-income countries. In this study, we employed transcriptome sequencing to perform molecular classification of B-ALL at diagnosis in pediatric patients enrolled in the multicentric ALLIC-GATLA-2010 clinical protocol in Argentina. Using a combination of bioinformatic tools, we successfully identified single nucleotide variants, fusion transcripts and gene expression profiles, achieving molecular classification in over 90% of patients. Our analysis also revealed high-risk molecular features and novel genetic alterations. These findings hold potential clinical value for improving risk stratification and identifying therapeutic targets, particularly for patients who remain unclassified by conventional diagnostic methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/59f39fa251d291498250d854c37a9028d4e85f79" target='_blank'>
              Molecular Characterization of Pediatric Acute Lymphoblastic Leukemia via Integrative Transcriptomics: A Multicenter Study in Argentina
              </a>
            </td>
          <td>
            Sol Ruiz, María Mercedes Abbate, Ezequiel Sosa, Daniel Avendaño, Ignacio Gomez Mercado, María Laura Lacreu, Mariano Riccheri, Virginia Schuttenberg, Luis Aversa, E. Vazquez, G. Gueron, J. Cotignola
          </td>
          <td>2024-09-22</td>
          <td>None</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cbd49dc4f182de3dedf2bd9170f18d407483728" target='_blank'>
              VolcanoSV enables accurate and robust structural variant calling in diploid genomes from single-molecule long read sequencing
              </a>
            </td>
          <td>
            Can Luo, Y. Liu, X. Zhou
          </td>
          <td>2024-08-13</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Simple Summary Childhood cancer survivors treated with radiotherapy face the likelihood of long-term complications, including mutations. In order to assess the long-term effect of radiotherapy on the capacity of cells to repair their DNA double-strand breaks (DSBs), five lambs received medulloblastoma radiotherapy to the thecal sac, with three lambs serving as controls. Four years later, lung biopsies were taken and fibroblast cells were amplified and re-irradiated. The cells from the previously treated sheep showed a significant impairment of their DNA DSB repair mechanism, highlighting a potential increase in radiosensitivity. Our results show that previous irradiation can impair the DNA DSB repair mechanism of ovine lung fibroblasts. Abstract Children with cancer previously treated with radiotherapy face the likelihood of side effects that can be debilitating or fatal. This study aimed to assess the long-term effect of medulloblastoma radiotherapy on the DNA double-strand break (DSB) repair capability of primary fibroblasts derived from lung biopsies of previously irradiated young sheep. This study included biopsies from three control and five irradiated sheep. The treated sheep had previously received spinal radiotherapy at a total dose of 28 Gy, which is equivalent to pediatric medulloblastoma treatment. Lung biopsies were taken 4 years post-irradiation from high-dose (HD, >18 Gy) and low-dose (LD, <2 Gy) regions. Fifteen cell lines were extracted (six control, four LD and five HD). The cells were irradiated, and DNA DSB repair was analyzed by immunofluorescence. Clonogenic, trypan blue and micronuclei assays were performed. Both the HD and LD cell lines had a significantly higher number of residual γH2AX foci 24 h and a significant decrease in pATM activity post-irradiation compared to the control. There was no statistically significant difference in the clonogenic assay, trypan blue and micronuclei results. Our study showed that a previous irradiation can impair the DNA DSB repair mechanism of ovine lung fibroblasts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5ea88bd1bedea4941e4772289fe7a7d48fca439" target='_blank'>
              Impaired DNA Double-Strand Break Repair in Irradiated Sheep Lung Fibroblasts: Late Effects of Previous Irradiation of the Spinal Thecal Sac
              </a>
            </td>
          <td>
            B. Youssef, C. Feghaly, Joelle Al Choboq, J. Bou-Gharios, R. Challita, J. Azzi, H. Bou Hadir, F. Abi Antoun, T. Araji, Phillip J. Taddei, F. Geara, Pierre Sfeir, Abdo Jurjus, W. Abou-Kheir, Larry Bodgi
          </td>
          <td>2024-08-26</td>
          <td>Cancers</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Several hundred billion cells die daily in the human body, releasing cell-free chromatin particles (cfChPs) in the process, which then enter the blood circulation and get taken up by healthy cells. We have previously reported that, these internalized cfChPs damage genomic DNA as well cause physical damage to mitochondria, resulting in increased mitochondrial ROS production. In the current study, we evaluated the potential damaging effects of the cfChP-induced increase in ROS production on genomic DNA. NIH3T3 mouse fibroblast cells were treated with cfChPs isolated from the sera of healthy individuals (H-cfChPs) or patients with cancer (C-cfChPs) in the presence or absence of the ROS scavenger Mito-TEMPO. The pre-incubation of cfChP-treated cells with Mito-TEMPO abolished ROS production, but did not prevent genomic DNA damage induced by H-cfChPs and C-cfChPs. Our results suggest that cfChP-induced genomic DNA damage occurs in an ROS-independent manner. These findings align with emerging evidence suggesting that mitochondrial ROS may not be a direct cause of genomic DNA damage and suggest that DNA damage attributed to ROS may in-fact be induced by cfChPs. This possibility opens up new therapeutic approaches involving deactivation of cfChPs to retard ageing and other degenerative conditions traditionally attributed to oxidative stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c2f89cdf5be913b593ba8f17e11056fcb407617" target='_blank'>
              Cell-free chromatin particles damage genomic DNA of healthy cells via an ROS-independent mechanism
              </a>
            </td>
          <td>
            Karishma Jain, G. V. Raghuram, Relestina Lopes, Naveen Kumar Khare, S. Shabrish, Indraneel Mittra
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Colorectal adenocarcinoma (COAD) has a poor prognosis. Cyclin-dependent kinase inhibitor 2A (CDKN2A) significantly affects the development and progression of various human tumors. However, the significance and pathological mechanisms of CDKN2A in COAD remain to be elucidated. We assessed expression levels, clinical significance, biological function, co-expressed genes, and enrichment of related pathways of CDKN2A in COAD using various databases, including The University of Alabama at Birmingham Cancer Data Analysis Portal, Gene Expression Profiling Interactive Analysis, Tumor Immune Estimation Resource, Human Protein Atlas, STRING, GeneMANIA, cBioPortal, and Linked Omics. Our investigation showed that CDKN2A was highly expressed in colon adenocarcinomas (P < .001). It is weakly expressed or not expressed in normal tissues. The survival time of patients with colon adenocarcinoma with high CDKN2A expression is significantly shorter than that of patients with low expression levels (P = .011). There was a significant positive correlation between the expression level of CDKN2A in colon adenocarcinoma tissues and the infiltration of CD4+ T cells, macrophages, and neutrophils. Moreover, there was a significant negative association between the expression level of CDKN2A in colon adenocarcinoma tissues and B cell infiltration. The ten hub genes included tumor protein 53, V-myc Avian Myelocytomatosis Viral Oncogene Homolog, AKT serine/threonine kinase 1, cyclin-dependent kinase 2, phosphatase and tensin homolog deleted on chromosome ten, cyclin D1, cyclin dependent kinase 4, cyclin dependent kinase inhibitor 1A, catenin beta 1, and B-Raf proto-oncogene, serine/threonine kinase. Mutations in the CDKN2A genome in colon adenocarcinoma reduce survival. Gene ontology and Kyoto Encyclopedia of Genes and Genomes analyses showed that the differentially expressed genes were enriched in apoptotic signaling pathways and multiple pathways related to metabolic progression. Our results indicate that CDKN2A can be used as a marker of poor prognosis in patients with colon adenocarcinoma. CDKN2A may regulate the occurrence and development of colon adenocarcinomas by influencing immune cell infiltration and metabolic pathways.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae51b20c2695c36facac2469a11d7ddc7c4c48d6" target='_blank'>
              Impact of CDKN2A gene expression on colon adenocarcinoma via biosignature analysis.
              </a>
            </td>
          <td>
            Chen Xu
          </td>
          <td>2024-09-06</td>
          <td>Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcaa5e009963f12c7b3cfac1a62813bb1ec1e2e7" target='_blank'>
              Removal of TREX1 activity enhances CRISPR-Cas9-mediated homologous recombination.
              </a>
            </td>
          <td>
            Mehmet E. Karasu, Eléonore Toufektchan, Yanyang Chen, Alessandra Albertelli, G. Cullot, J. Maciejowski, Jacob E. Corn
          </td>
          <td>2024-08-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37216ded337bd372b65a91cb1670af84243d9871" target='_blank'>
              Single cell whole-genome sequencing of brain cells: age- and cell-type specific mutational profiles
              </a>
            </td>
          <td>
            Melania Capasso, N. A. Aziz
          </td>
          <td>2024-08-09</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5cd2329352934bc5310fa61340f1798aa2a300f6" target='_blank'>
              Super-silencer perturbation by EZH2 and REST inhibition leads to large loss of chromatin interactions and reduction in cancer growth.
              </a>
            </td>
          <td>
            Ying Zhang, Kaijing Chen, Seng Chuan Tang, Yichao Cai, A. Nambu, Y. See, Chaoyu Fu, Anandhkumar Raju, Benjamin Lebeau, Zixun Ling, J. J. Chan, Yvonne Tay, Marek Mutwil, Manikandan Lakshmanan, G. Tucker-Kellogg, W. Chng, D. Tenen, Motomi Osato, Vinay Tergaonkar, M. Fullwood
          </td>
          <td>2024-09-20</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>110</td>
        </tr>

        <tr id="INTRODUCTION AND PURPOSE: Long non-coding RNAs are key regulators of gene expression and tumorigenesis. Deleted in leukemia 1 (DLEU1) is significant in cancer development. This article reviews DLEU1's role across cancers, assessing its diagnostic and therapeutic potential. 
METHODS: A review of scientific articles from PubMed and Google Scholar was conducted. 
DESCRIPTION OF THE STATE OF KNOWLEDGE: DLEU1 drives tumor growth in glioblastoma, esophageal, colorectal, and oral cancers by interacting with key miRNAs and genes, promoting malignancy. In breast and ovarian cancers, DLEU1 modulates migration and invasion via specific pathways. In cervical cancer, DLEU1 is linked to immune modulation and serves as a prognostic marker. 
SUMMARY: DLEU1 is a vital regulator in various cancers, offering potential as a diagnostic and therapeutic target.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4207b2d625dc476a8faac2c73c2417005cce5a42" target='_blank'>
              The multifaceted role of long non-coding RNA DLEU1 in cancer progression: implications for diagnosis and therapy
              </a>
            </td>
          <td>
            Paweł Stanicki, Julita Szarpak, Aldona Pażyra, Natalia Kusak, Natalia Żak, Barbara Jaworska
          </td>
          <td>2024-09-21</td>
          <td>Quality in Sport</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7401c81aea35658fe72a5961ba5c3e31cae1d2fa" target='_blank'>
              The genomic basis of childhood T-lineage acute lymphoblastic leukaemia.
              </a>
            </td>
          <td>
            P. Pölönen, Danika Di Giacomo, A. Seffernick, Abdelrahman A. Elsayed, Shunsuke Kimura, Francesca Benini, Lindsey E Montefiori, Brent L Wood, Jason Xu, Changya Chen, Zhong-Yi Cheng, H. Newman, Jason R Myers, Ilaria Iacobucci, Elizabeth Li, Jonathan H. Sussman, Dale Hedges, Yawei Hui, Caroline Diorio, Lahari Uppuluri, David Frank, Yiping Fan, Yunchao Chang, S. Meshinchi, R. Ries, R. Shraim, Alexander Li, Kathrin M Bernt, M. Devidas, S. Winter, K. Dunsmore, H. Inaba, William L Carroll, Nilsa C Ramirez, Aaron H. Phillips, Richard W. Kriwacki, Jun J Yang, T. Vincent, Yaqi Zhao, P. S. Ghate, Jian Wang, Colleen Reilly, Xin Zhou, Mathijs A. Sanders, Junko Takita, Motohiro Kato, Nao Takasugi, Bill H Chang, Richard D Press, M. Loh, E. Rampersaud, E. Raetz, S P Hunger, Kai Tan, Ti-Cheng Chang, Gang Wu, S. B. Pounds, C. Mullighan, D. Teachey
          </td>
          <td>2024-08-14</td>
          <td>Nature</td>
          <td>2</td>
          <td>102</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c2f544fdc28c5e62e6f82a4539edb8e492a2dce" target='_blank'>
              Integration of scHi-C and scRNA-seq data defines distinct 3D-regulated and biological-context dependent cell subpopulations
              </a>
            </td>
          <td>
            Yufan Zhou, Tian Li, Lavanya Choppavarapu, Kun Fang, Shili Lin, Victor X Jin
          </td>
          <td>2024-09-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Here, we present a method for enrichment of double-stranded cfDNA with an average length of ∼40 bp from cfDNA for high-throughput DNA sequencing. This class of cfDNA is enriched at gene promoters and binding sites of transcription factors or structural DNA-binding proteins, so that a genome-wide DNA footprint is directly captured from liquid biopsies. In short double-stranded cfDNA from healthy individuals, we find significant enrichment of 203 transcription factor motifs. Additionally, short double-stranded cfDNA signals at specific genomic regions correlate negatively with DNA methylation, positively with H3K4me3 histone modifications and gene transcription. The diagnostic potential of short double-stranded cell-free DNA (cfDNA) in blood plasma has not yet been recognized. When comparing short double-stranded cfDNA from patient samples of pancreatic ductal adenocarcinoma with colorectal carcinoma or septic with postoperative controls, we identify 136 and 241 differentially enriched loci, respectively. Using these differentially enriched loci, the disease types can be clearly distinguished by principal component analysis, demonstrating the diagnostic potential of short double-stranded cfDNA signals as a new class of biomarkers for liquid biopsies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/003bad913ef23b451e8afe9e333721e6d528290e" target='_blank'>
              A novel approach for in vivo DNA footprinting using short double-stranded cell-free DNA from plasma.
              </a>
            </td>
          <td>
            Jan Müller, Christina Hartwig, Mirko Sonntag, Lisa Bitzer, Christopher Adelmann, Y. Vainshtein, Karolina Glanz, S. Decker, Thorsten Brenner, Georg F. Weber, A. von Haeseler, Kai Sohn
          </td>
          <td>2024-09-13</td>
          <td>Genome research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="TP53 is a tumor suppressor gene that is mutated in most cancer types and has been extensively studied in cancer research. p53 plays a critical role in regulating the expression of target genes and is involved in key processes such as apoptosis, cell cycle regulation, and genomic stability, earning it the title “guardian of the genome.” Numerous studies have demonstrated p53’s influence on and regulation of autophagy, ferroptosis, the tumor microenvironment, and cell metabolism, all of which contribute to tumor suppression. Alterations in p53, specifically mutant p53 (mutp53), not only impair its tumor-suppressing functions but also enhance oncogenic characteristics. Recent data indicate that mutp53 is strongly associated with poor prognosis and advanced cancers, making it an ideal target for the development of novel cancer therapies. This review summarizes the post-translational modifications of p53, the mechanisms of mutp53 accumulation, and its gain-of-function, based on previous findings. Additionally, this review discusses its impact on metabolic homeostasis, ferroptosis, genomic instability, the tumor microenvironment, and cancer stem cells, and highlights recent advancements in mutp53 research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5ea389a27a85074a44af76e077628c890d00a2c8" target='_blank'>
              Roles of Mutant TP53 Gene in Cancer Development and Progression
              </a>
            </td>
          <td>
            Muhammad Abubakar, Baqaur Rehman
          </td>
          <td>2024-09-25</td>
          <td>Proceedings of Anticancer Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c54370344e70d12f9e093a0f96e506dd839c6df0" target='_blank'>
              Deciphering breast cancer dynamics: insights from single-cell and spatial profiling in the multi-omics era
              </a>
            </td>
          <td>
            Xin Xiong, Xin Wang, Cui-Cui Liu, Zhi-Ming Shao, Ke-Da Yu
          </td>
          <td>2024-09-18</td>
          <td>Biomarker Research</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Fibrolamellar carcinoma (FLC) is a liver cancer of adolescents and young adults characterized by fusions of the genes encoding the protein kinase A catalytic subunit, PRKACA, and heat shock protein, DNAJB1. The chimeric DNAJB1-PRKACA protein has increased kinase activity and is essential for FLC xenograft growth. Here, we explore the critical oncogenic pathways controlled by DNAJB1-PRKACA using patient-derived FLC models, engineered systems, and patient samples. We show that a core function of DNAJB1-PRKACA is the phosphorylation and inactivation of Salt-inducible kinases (SIKs). This leads to deregulation of the CRTC2 transcriptional co-activator and p300 acetyltransferase, resulting in transcriptional reprogramming and increased global histone acetylation, driving malignant growth. Our studies establish a central oncogenic mechanism of DNAJB1-PRKACA and suggest the potential of targeting CRTC2/p300 in FLC. Notably, these findings link this rare cancer's signature fusion oncoprotein to more common cancer gene alterations involving STK11 and GNAS, which also function via SIK suppression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a77fde5ecc4bc9340e646816c229c18dfe2fdb5e" target='_blank'>
              DNAJB1-PRKACA fusion drives fibrolamellar liver cancer through impaired SIK signaling and CRTC2/p300-mediated transcriptional reprogramming.
              </a>
            </td>
          <td>
            Ilaria Gritti, Jinkai Wan, Vajira Weeresekara, Joel M Vaz, Giuseppe Tarantino, Tenna Holgersen Bryde, Vindhya Vijay, Ashwin V. Kammula, Prabhat Kattel, Songli Zhu, Phuong Vu, Marina Chan, Meng-Ju Wu, John D Gordan, K. Patra, Vanessa S Silveira, R. Manguso, Marc N. Wein, Christopher J Ott, Jun Qi, David Liu, Kei Sakamoto, Taranjit S. Gujral, N. Bardeesy
          </td>
          <td>2024-09-27</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>101</td>
        </tr>

        <tr id="Patients with microsatellite instability-high (MSI-H) colorectal cancer (CRC) have high tumor mutation burden and tumor immunogenicity, exhibiting a higher response rate to immunotherapy and better survival. However, a portion of MSI-H CRC patients still experience adverse disease outcomes. We aimed to identify the tumor-autonomous regulators determining these heterogeneous clinical outcomes. The Cancer Genome Atlas (TCGA) dataset was used to identify regulators in MSI-H CRC patients with unfavorable outcomes. Stable CRC tumor clones expressing targeted regulators were established to evaluate migratory and stemness properties, immune cell vulnerability, and cell-in-cell (CIC) structure formation. RNA-sequencing (RNA-seq) was used to identify enriched biological pathways in stable CRC tumor clones. Clinicopathological characterization of formalin-fixed paraffin-embedded (FFPE) MSI-H CRC specimens was performed to explore the underlying mechanisms involved. We showed that cancer/testis antigen family 45 member A1 (CT45A1) expression was upregulated in MSI-H CRC patients with poor survival outcomes. CT45A1-expressing microsatellite stable (MSS) CRC cells showed enhanced migratory ability. However, CT45A1-expressing MSI-H CRC cells, but not MSS CRC cells, showed higher resistance to natural killer (NK) cell cytotoxicity and served as outer cells in homotypic CIC structures, preventing exogenous or therapeutic antibody access to inner CRC cells. Inactivating RHO-ROCK/MLCK-MLC2 signaling with small-molecule inhibitors or short-hairpin RNAs (shRNAs) targeting myosin light chain kinase (MYLK) abolished NK cell resistance and reduced the outer cell fate of CT45A1-expressing MSI-H CRC cells. In MSI-H CRC patients, CT45A1-positive tumors exhibited increased MLC2 phosphorylation, increased outer cell fate, and decreased survival. We demonstrated that CT45A1 potentiates the advanced progression of MSI-H CRC, and targeting MLC2 phosphorylation may enhance immunotherapy efficacy in CT45A1-positive MSI-H CRC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a08f29a07f3e7ccbababc06da74550a73f5f37c3" target='_blank'>
              CT45A1-mediated MLC2 (MYL9) phosphorylation promotes natural killer cell resistance and outer cell fate in a cell-in-cell structure, potentiating the progression of microsatellite instability-high colorectal cancer.
              </a>
            </td>
          <td>
            H. Teng, Hsiang-Yueh Huang, Chun-Chi Lin, Y. Twu, Wen-Hao Yang, Wen-Chun Lin, Hsin-Yi Lan, Yen-Yu Lin, Wei-Lun Hwang
          </td>
          <td>2024-09-25</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Retrotransposons are invasive genetic elements, which replicate by copying and pasting themselves throughout the genome in a process called retrotransposition. The most abundant retrotransposons by number in the human genome are Alu and LINE-1 elements, which comprise approximately 40% of the human genome. The ability of retrotransposons to expand and colonize eukaryotic genomes has rendered them evolutionarily successful and is responsible for creating genetic alterations leading to significant impacts on their hosts. Previous research suggested that hypomethylation of Alu and LINE-1 elements is associated with global hypomethylation and genomic instability in several types of cancer and diseases, such as neurodegenerative diseases, obesity, osteoporosis, and diabetes mellitus (DM). With the advancement of sequencing technologies and computational tools, the study of the retrotransposon’s association with physiology and diseases is becoming a hot topic among researchers. Quantifying Alu and LINE-1 methylation is thought to serve as a surrogate measurement of global DNA methylation level. Although Alu and LINE-1 hypomethylation appears to serve as a cellular senescence biomarker promoting genomic instability, there is sparse information available regarding their potential functional and biological significance in DM. This review article summarizes the current knowledge on the involvement of the main epigenetic alterations in the methylation status of Alu and LINE-1 retrotransposons and their potential role as epigenetic markers of global DNA methylation in the pathogenesis of DM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a07f55c13ed59ce933c2fb2995ef34b1c1fa76b5" target='_blank'>
              Retrotransposons and Diabetes Mellitus
              </a>
            </td>
          <td>
            Andromachi Katsanou, Charilaos Kostoulas, E. Liberopoulos, A. Tsatsoulis, I. Georgiou, S. Tigas
          </td>
          <td>2024-09-01</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>48</td>
        </tr>

        <tr id="
 Humans are exposed to a variety of environmental toxicants which have been implicated in contributing to cancer development. A possible explanation for this link is that adaptive epigenetic changes occur in stem cell populations upon toxicant exposure. These cells are particularly vulnerable during developmental stages, leading to abnormal gene expression later in life. Environmental exposures have been associated with the initiation of sarcoma development. This study will focus on Ewing Sarcoma (ES), as this malignancy has a low mutational burden relative to other cancer types and is thought to be linked to agricultural chemicals. We are also focusing on stromal antigen 2 (STAG2), which is found to be frequently mutated in ES in addition to multiple other tumor types. Patients with STAG2-mutated ES have higher rates of metastatic disease and worse outcomes, but the reason for this remains poorly understood. STAG2 plays a role in the regulation of genomic organization, DNA repair, and modulation of replication stress (RS). It has also been shown that STAG2 loss may lead to dysregulation of the cGAS-STING pathway, which is critical for tumor immunosurveillance. We are testing the effects of environmental toxicants in mesenchymal stem cells (MSCs), the putative cell of origin for ES and many other sarcomas. We have successfully used CRISPR/Cas9 to generate STAG2 knockout MSCs. With this model, we are testing the effects of STAG2 loss and toxicant exposure on levels of RS, epigenomic instability, and innate immune suppression in MSCs. Our results indicate that these toxicants alter the cell cycle dynamics of MSCs, and that these alterations change depending on STAG2 status. We have also observed changes in the DNA damage response of STAG2-mutant MSCs when exposed to environmental toxicants. Interestingly, effects were seen even at sub-lethal doses that may reflect common, low-level exposures. We believe that our data will contribute to the understanding of the mechanism by which environmental toxicants facilitate malignant transformation of Ewing sarcoma and other pediatric sarcomas.
 Citation Format: Rachael Kohrn, Joyce Ohm. Investigating the role of environmental toxicant exposures and STAG2 loss in sarcomagenesis [abstract]. In: Proceedings of the AACR Special Conference in Cancer Research: Advances in Pediatric Cancer Research; 2024 Sep 5-8; Toronto, Ontario, Canada. Philadelphia (PA): AACR; Cancer Res 2024;84(17 Suppl):Abstract nr A029.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fed252aa2969d787536854f4f2e71a66da32b809" target='_blank'>
              Abstract A029 Investigating the role of environmental toxicant exposures and STAG2 loss in sarcomagenesis
              </a>
            </td>
          <td>
            Rachael Kohrn, Joyce Ohm
          </td>
          <td>2024-09-05</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) technology has emerged as a powerful tool for dissecting cellular heterogeneity and understanding the intricate biology of diseases, including cancer. Endometrial cancer (EC) stands out as the most prevalent gynecological malignancy in Europe and the second most diagnosed worldwide, yet its cellular complexity remains poorly understood. In this review, we explore the contributions of scRNA-seq studies to shed light on the tumor cells and cellular landscape of EC. We discuss the diverse tumoral and microenvironmental populations identified through scRNA-seq, highlighting the implications for understanding disease progression. Furthermore, we address potential limitations inherent in scRNA-seq studies, such as technical biases and sample size constraints, emphasizing the need for larger-scale research encompassing a broader spectrum of EC histological subtypes. Notably, a significant proportion of scRNA-seq analyses have focused on primary endometrioid carcinoma tumors, underscoring the need to incorporate additional histological and aggressive types to comprehensively capture the heterogeneity of EC. By critically evaluating the current state of scRNA-seq research in EC, this review underscores the importance of advancing towards more comprehensive studies to accelerate our understanding of this complex disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f2842e25bccda324e64e3cb50c3333788f2e594" target='_blank'>
              Single-cell RNA sequencing in endometrial cancer: exploring the epithelial cells and the microenvironment landscape
              </a>
            </td>
          <td>
            Silvia González-Martínez, B. Pérez-Mies, Javier Cortés, José Palacios
          </td>
          <td>2024-08-20</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Leukemia is a malignant tumor with high heterogeneity and a complex evolutionary process. It is difficult to resolve the heterogeneity and clonal evolution of leukemia cells by applying traditional bulk sequencing techniques, thus preventing a deep understanding of the mechanisms of leukemia development and the identification of potential therapeutic targets. However, with the development and application of single-cell sequencing technology, it is now possible to investigate the gene expression profile, mutations, and epigenetic features of leukemia at the single-cell level, thus providing a new perspective for leukemia research. In this article, we review the recent applications and advances of single-cell sequencing technology in leukemia research, discuss its potential for enhancing our understanding of the mechanisms of leukemia development, discovering therapeutic targets and personalized treatment, and provide reference guidelines for the significance of this technology in clinical research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/059e4fa52cbb0003a82e3f417ae0958def767ace" target='_blank'>
              Application and research progress of single cell sequencing technology in leukemia
              </a>
            </td>
          <td>
            Dan Xie, Bangquan An, Mingyue Yang, Lei Wang, Min Guo, Heng Luo, Shengwen Huang, Fa Sun
          </td>
          <td>2024-08-29</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df4be94d1533f2155dbf3532c23eeba54389f11" target='_blank'>
              Cohesin mutations in acute myeloid leukemia.
              </a>
            </td>
          <td>
            Austin Boucher, J. Murray, Sridhar Rao
          </td>
          <td>2024-09-09</td>
          <td>Leukemia</td>
          <td>0</td>
          <td>3</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [13],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>