<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-26 06:29:45 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>20</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>50</td>
          <td>130</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>98</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, Jens Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>272</td>
          <td>130</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>234</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a form of oncogene amplification found across cancer types and associated with poor outcome in patients. ecDNA can be structurally complex and contain rearranged DNA sequences derived from multiple chromosome locations. As the structure of ecDNA can impact oncogene regulation and may indicate mechanisms of its formation, disentangling it at high resolution from sequencing data is essential. Even though methods have been developed to identify and reconstruct ecDNA in cancer genome sequencing, it remains challenging to resolve complex ecDNA structures, in particular amplicons with shared genomic footprints. We here introduce Decoil, a computational method which combines a breakpoint-graph approach with regression to reconstruct complex ecDNA and deconvolve co-occurring ecDNA elements with overlapping genomic footprints from long-read nanopore sequencing. Decoil outperforms de novo assembly and alignment-based methods in simulated long-read sequencing data for both simple and complex ecDNAs. Applying Decoil on whole genome sequencing data uncovered different ecDNA topologies and explored ecDNA structure heterogeneity in neuroblastoma tumors and cell lines, indicating that this method may improve ecDNA structural analyzes in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c914721ddd777cd78df6632d6860e72cff344be" target='_blank'>
              Reconstructing extrachromosomal DNA structural heterogeneity from long-read sequencing data using Decoil.
              </a>
            </td>
          <td>
            M. Giurgiu, Nadine Wittstruck, Elias Rodriguez-Fos, R. Chamorro González, Lotte Brueckner, Annabell Krienelke-Szymansky, K. Helmsauer, Anne Hartebrodt, P. Euskirchen, R. Koche, Kerstin Haase, Knut Reinert, A. Henssen
          </td>
          <td>2024-08-07</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5988602f52499713b4d1d42d555664a971cb3e9" target='_blank'>
              3D genomic analysis reveals novel enhancer-hijacking caused by complex structural alterations that drive oncogene overexpression
              </a>
            </td>
          <td>
            Katelyn L. Mortenson, Courtney Dawes, Emily R. Wilson, Nathan E. Patchen, Hailey E Johnson, Jason Gertz, Swneke D. Bailey, Yang Liu, K. Varley, Xiaoyang Zhang
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA), an extrachromosomal circular structured DNA, is extensively found in eukaryotes. Exploring eccDNA at the single-cell level contributes to understanding heterogeneity, evolution, development, and specific functions within cells. Nevertheless, the high-throughput identification methods for single-cell eccDNA are complex, and currently mature and widely applicable technologies are lacking. Those factors have led to a scarcity of resources for studying eccDNA at the single-cell level. Therefore, using available single-cell whole-genome sequencing (WGS) data, we have constructed a comprehensive eccDNA database named scEccDNAdb (https://lcbb.swjtu.edu.cn/scEccDNAdb/). Presently, scEccDNAdb comprises 3,195,464 single-cell eccDNA entries from both disease/health human and mouse samples, which provides comprehensive annotations including oncogenes, typical enhancers, super-enhancers, CTCF binding sites, SNPs, chromatin accessibility, eQTLs, transcription factor binding sites, motifs, and SVs. Additionally, it provides nine online analysis and visualization tools, facilitating the generation of publication-quality figures for eccDNA analysis through the upload of customized files. Overall, scEccDNAdb represents the first comprehensive database known to us for exploring and analyzing single-cell eccDNA data in diverse cell types, tissues, and species. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322c06c5550a609619dd96de2cba0b3494554b2a" target='_blank'>
              scEccDNAdb: an integrated single-cell eccDNA resource for human and mouse
              </a>
            </td>
          <td>
            Wenqing Wang, Xinyu Zhao, Tianyu Ma, Tengwei Zhong, Junnuo Zheng, Yi Yang, Yuanbiao Guo, Zhiyun Guo
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="DNA rearrangements are thought to arise from two classes of processes. The first class involves DNA breakage and fusion (“cut-and-paste”) without net DNA gain or loss. The second class involves aberrant DNA replication (“copy-and-paste”) and can produce either net DNA gain or loss. We previously demonstrated that the partitioning of chromosomes into aberrant structures of the nucleus, micronuclei or chromosome bridges, can generate cut-and-paste rearrangements by chromosome fragmentation and ligation. Surprisingly, in the progeny clones of single cells that have undergone chromosome bridge breakage, we identified large segmental duplications and short sequence insertions that are commonly attributed to copy-and-paste processes. Here, we demonstrate that both large duplications and short insertions are inherent outcomes of the replication and fusion of unligated DNA ends, a process we term breakage-replication/fusion (B-R/F). We propose that B-R/F provides a unifying explanation for complex rearrangement patterns including chromothripsis and chromoanasynthesis and enables rapid DNA amplification after chromosome fragmentation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c002d451b4a62962b702db7317ff655026459317" target='_blank'>
              Chromosome breakage-replication/fusion enables rapid DNA amplification
              </a>
            </td>
          <td>
            Cheng-Zhong Zhang, David Pellman
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is a pervasive driver event in cancer, yet our understanding of how ecDNA forms is limited. Here, we couple a CRISPR-based method for ecDNA induction with extensive characterization of newly formed ecDNA to examine their biogenesis. We find that DNA circularization is efficient, irrespective of 3D genome context, with formation of 800kb, 1 Mb, and 1.8 Mb ecDNAs reaching or exceeding 15%. We show non-homologous end joining and microhomology-mediated end joining both contribute to ecDNA formation, while inhibition of DNA-PKcs and ATM have opposing impacts on ecDNA formation. EcDNA and the corresponding chromosomal excision scar can form at significantly different rates and respond differently to DNA-PKcs and ATM inhibition. Taken together, our results support a model of ecDNA formation in which double strand break ends dissociate from their legitimate ligation partners prior to joining of illegitimate ends to form the ecDNA and excision scar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e488ec3b1ac40e7e2986571eab100d1986b6e3f" target='_blank'>
              Disparate pathways for extrachromosomal DNA biogenesis and genomic DNA repair.
              </a>
            </td>
          <td>
            John C. Rose, J. Belk, I. Wong, Jens Luebeck, Hudson T Horn, Bence Daniel, Matthew G Jones, Kathryn E Yost, King L. Hung, Kevin S Kolahi, Ellis J Curtis, Calvin J. Kuo, V. Bafna, Paul S. Mischel, Howard Y. Chang
          </td>
          <td>2024-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Sequencing-based mapping of ensemble pairwise interactions among regulatory elements support the existence of topological assemblies known as promoter-enhancer hubs or cliques in cancer. Yet, prevalence, regulators, and functions of promoter-enhancer hubs in individual cancer cells remain unclear. Here, we systematically integrated functional genomics, transcription factor screening, and optical mapping of promoter-enhancer interactions to identify key promoter-enhancer hubs, examine heterogeneity of their assembly, determine their regulators, and elucidate their role in gene expression control in individual triple negative breast cancer (TNBC) cells. Optical mapping of individual SOX9 and MYC alleles revealed the existence of frequent multiway interactions among promoters and enhancers within spatial hubs. Our single-allele studies further demonstrated that lineage-determining SOX9 and signaling-dependent NOTCH1 transcription factors compact MYC and SOX9 hubs. Together, our findings suggest that promoter-enhancer hubs are dynamic and heterogeneous topological assemblies, which are controlled by oncogenic transcription factors and facilitate subtype-restricted gene expression in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab8df926269367eb0273d95bc28b249a4b8bb66d" target='_blank'>
              Oncogenic transcription factors instruct promoter-enhancer hubs in individual triple negative breast cancer cells
              </a>
            </td>
          <td>
            Jingru Zhao, Yeqiao Zhou, Ilias Tzelepis, Noah G Burget, Junwei Shi, R. Faryabi
          </td>
          <td>2024-08-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Recent advances in high-resolution mapping of spatial interactions among regulatory elements support the existence of complex topological assemblies of enhancers and promoters known as enhancer-promoter hubs or cliques. Yet, organization principles of these multi-interacting enhancer-promoter hubs and their potential role in regulating gene expression in cancer remains unclear. Here, we systematically identified enhancer-promoter hubs in breast cancer, lymphoma, and leukemia. We found that highly interacting enhancer-promoter hubs form at key oncogenes and lineage-associated transcription factors potentially promoting oncogenesis of these diverse cancer types. Genomic and optical mapping of interactions among enhancer and promoter elements further showed that topological alterations in hubs coincide with transcriptional changes underlying acquired resistance to targeted therapy in T cell leukemia and B cell lymphoma. Together, our findings suggest that enhancer-promoter hubs are dynamic and heterogeneous topological assemblies with the potential to control gene expression circuits promoting oncogenesis and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9831a80afed1f71511cdd9f1e5ce45eefd2262c2" target='_blank'>
              Enhancer-promoter hubs organize transcriptional networks promoting oncogenesis and drug resistance
              </a>
            </td>
          <td>
            Brent S. Perlman, Noah G Burget, Yeqiao Zhou, Gregory W. Schwartz, Jelena Petrovic, Z. Modrusan, R. Faryabi
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Chromosomal instability results in widespread structural and numerical chromosomal abnormalities (CAs) during cancer evolution1–3. While CAs have been linked to mitotic errors resulting in the emergence of nuclear atypias4–7, the underlying processes and basal rates of spontaneous CA formation in human cells remain under-explored. Here we introduce machine learning-assisted genomics-and-imaging convergence (MAGIC), an autonomously operated platform that integrates automated live-cell imaging of micronucleated cells, machine learning in real-time, and single-cell genomics to investigate de novo CA formation at scale. Applying MAGIC to near-diploid, non-transformed cell lines, we track CA events over successive cell cycles, highlighting the common role of dicentric chromosomes as an initiating event. We determine the baseline CA rate, which approximately doubles in TP53-deficient cells, and show that chromosome losses arise more rapidly than gains. The targeted induction of DNA double-strand breaks along chromosomes triggers distinct CA processes, revealing stable isochromosomes, amplification and coordinated segregation of isoacentric segments in multiples of two, and complex CA outcomes, depending on the break location. Our data contrast de novo CA spectra from somatic mutational landscapes after selection occurred. The large-scale experimentation enabled by MAGIC provides insights into de novo CA formation, paving the way to unravel fundamental determinants of chromosome instability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a12d3e51de5c9c53670001797534816b7eeca397" target='_blank'>
              Origins of de novo chromosome rearrangements unveiled by coupled imaging and genomics
              </a>
            </td>
          <td>
            M. R. Cosenza, Alice Gaiatto, Büşra Erarslan Uysal, Álvaro Andrades Delgado, Nina Luisa Sautter, Michael Adrian Jendrusch, Sonia Zumalave Duro, Tobias Rausch, A. Halavatyi, Eva-Maria Geissen, Patrick Hasenfeld, I. Cortés-Ciriano, Andreas Kulozik, Rainer Pepperkok, J. Korbel
          </td>
          <td>2024-08-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="Somatic structural variations (SVs) in cancer can shuffle DNA content in the genome, relocate regulatory elements, and alter genome organization. Enhancer hijacking occurs when SVs relocate distal enhancers to activate proto-oncogenes. However, most enhancer hijacking studies have only focused on protein-coding genes. Here, we develop a computational algorithm 'HYENA' to identify candidate oncogenes (both protein-coding and non-coding) activated by enhancer hijacking based on tumor whole-genome and transcriptome sequencing data. HYENA detects genes whose elevated expression is associated with somatic SVs by using a rank-based regression model. We systematically analyze 1146 tumors across 25 types of adult tumors and identify a total of 108 candidate oncogenes including many non-coding genes. A long non-coding RNA TOB1-AS1 is activated by various types of SVs in 10% of pancreatic cancers through altered 3-dimensional genome structure. We find that high expression of TOB1-AS1 can promote cell invasion and metastasis. Our study highlights the contribution of genetic alterations in non-coding regions to tumorigenesis and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fb0cf8831bea2891dbca4714836cde381f2ca0" target='_blank'>
              HYENA detects oncogenes activated by distal enhancers in cancer.
              </a>
            </td>
          <td>
            Anqi Yu, Ali E. Yesilkanal, Ashish Thakur, Fan Wang, Yang Yang, William Phillips, Xiaoyang Wu, Alexander Muir, Xin He, Francois Spitz, Lixing Yang
          </td>
          <td>2024-07-25</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dec3a33e2b0139c6a02f10afa4c9e80abbaad7a" target='_blank'>
              Extrachromosomal circular DNA promotes prostate cancer progression through the FAM84B/CDKN1B/MYC/WWP1 axis
              </a>
            </td>
          <td>
            Wei Jin, Zhenqun Xu, Yan Song, Fangjie Chen
          </td>
          <td>2024-07-12</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="INTRODUCTION
Isodicentric Y chromosomes are relatively common structural variants of the human genome. The underlying mechanism of isodicentric Y chromosomes with short arm breakpoints [idic(Yq)] remains to be clarified.


CASE PRESENTATION
We encountered a Japanese man with azoospermia and mild short stature. G-banding and array-based comparative genomic hybridization indicated that his karyotype was 45,X/46,X,idic(Y)(qter→p11.32::p11.32→qter) with a ~1.8 Mb terminal deletion. Whole genome sequencing suggested that the Y chromosome had four breakpoints in a ~7 kb region of the pseudoautosomal region 1 (PAR1).


CONCLUSION
This case was assumed to have an idic(Yq) resulting from multiple DNA double-strand breaks in PAR1. This rearrangement may have been facilitated by the PAR1-specific chromatin architecture. The clinical features of the patient can be ascribed to SHOX haploinsufficiency and the presence of a 45,X cell line, although copy-number gains of some Yq genes and the size reduction of PAR1 may also contribute to his spermatogenic failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6ffa4553f26785a976ed64f6e1aecf9e157203" target='_blank'>
              Isodicentric Y chromosome with multiple breakpoints in the pseudoautosomal region 1.
              </a>
            </td>
          <td>
            Yasuko Ogiwara, Y. Kobori, E. Suzuki, Atsushi Hattori, Kanako Tanase-Nakao, Akiyoshi Osaka, Toshiyuki Iwahata, Hiroshi Okada, Y. Kuroki, M. Fukami
          </td>
          <td>2024-07-29</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b93e43a1982115416b67edb36296e1d7dd7f71fe" target='_blank'>
              The DNA methylome of pediatric brain tumors appears shaped by structural variation and predicts survival
              </a>
            </td>
          <td>
            Fengju Chen, Yiqun Zhang, Lanlan Shen, Chad J. Creighton
          </td>
          <td>2024-08-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A. Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Introduction Type B1 and B2 thymomas are lymphocyte-rich malignant tumors with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, non-coding regions remain uncharacterized. Here, we developed a rigorous tumor isolation method from lymphocyte-rich thymoma tissues and identified somatic mutations in non-coding and mitochondrial DNA. Methods CD205+CD45- pure tumor cells were isolated from fresh-frozen tissues using DEPArray system. Deep whole-genome sequencing was performed, and recurrent somatic alterations in coding, non-coding, and mitochondria regions were systemically identified by computational framework. The mutations were classified according to gene function, cis-regulatory element, and mutational signature. Results The total number of somatic mutations was approximately 80 times higher in non-coding regions than in coding regions in type B1-2 thymomas (1,671.3 vs. 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of cases exhibited fewer than four mutations in coding regions. A systematic non-coding analysis identified a total of 405.0 mutations per case on cis-regulatory elements, and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Mitochondrial heteroplasmy was observed in 90% of cases, with a significant proportion of mutations located in D-loop region. The single-base substitution pattern was signature 12. Conclusions Numerous non-coding mutations and mitochondrial heteroplasmy were detected in type B1 and B2 thymomas. Given the paucity of coding mutations observed in this disease entity, disruption of the non-coding landscape and mitochondrial heteroplasmic shift may be the primary cause of thymoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/559b9056aa5e8bd421441f43ad9a55ae5b38dbc9" target='_blank'>
              Genome-wide analysis of somatic non-coding mutation patterns and mitochondrial heteroplasmy in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Juliet King, Emma McLean, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Single nucleotide variants (SNVs) contribute to cancer by altering the coding and the non-coding regions of the genome. Connecting SNVs to transcriptomic and epigenetic changes at the single-cell level remains challenging. To enable studies of rare cell populations harboring specific point mutations, we developed STAR-FACS, Specific-To-Allele PCR-FACS, to sort cells based on genomic allele alterations. We show that STAR-FACS can separate cells based on TERT promoter mutation status and is compatible with bulk and single-cell transcriptomic and epigenetic profiling. We demonstrate that glioblastoma cell lines derived from the same tumor but harboring distinct TERT promoter SNVs have different transcriptional programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91f2ef09f49e055964d96a3768d156e4cf968931" target='_blank'>
              Cell sorting based on single nucleotide variation enables characterization of mutation-dependent transcriptome and chromatin states
              </a>
            </td>
          <td>
            Roberto Salatino, Marianna Franco, Arantxa Romero-Toledo, Yi Wang, Shanel Tsuda, Oszkar Szentirmai, M. Janiszewska
          </td>
          <td>2024-07-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Biallelic pathogenic variants in the essential DNA repair gene BRCA2 causes Fanconi anemia, complementation group FA-D1. Patients in this group are highly prone to develop embryonal tumors, most commonly medulloblastoma arising from the cerebellar granule cell progenitors (GCPs). GCPs undergo high proliferation in the postnatal cerebellum under SHH activation, but the type of DNA lesions that require the function of the BRCA2 to prevent tumorigenesis remains unknown. To identify such lesions, we assessed both GCP neurodevelopment and tumor formation using a mouse model with deletion of exons three and four of Brca2 in the central nervous system, coupled with global Trp53 loss. Brca2Δex3-4;Trp53-/- animals developed SHH subgroup medulloblastomas with complete penetrance. Whole-genome sequencing of the tumors identified structural variants with breakpoints enriched in areas overlapping G-quadruplexes (G4s). Brca2-deficient GCPs exhibited decreased replication speed in the presence of the G4-stabilizer pyridostatin. Pif1 helicase, which resolves G4s during replication, was highly upregulated in tumors, and Pif1 knockout in primary MB tumor cells resulted in increased genome instability upon pyridostatin treatment. These data suggest that G4s may represent sites prone to replication stalling in highly proliferative GCPs and without BRCA2, G4s become a source of genome instability. Tumor cells upregulate G4-resolving helicases to facilitate rapid proliferation through G4s highlighting PIF1 helicase as a potential therapeutic target for treatment of BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d543ffa68779aadc1c04b11411614442736e5f08" target='_blank'>
              G-quadruplexes are a source of vulnerability in BRCA2 deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, Carolina Sandoval-Garcia, Margaret L. MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="DNA modifications drive aging, neurodegeneration, carcinogenesis, and chemotherapy drug action. To understand the functional genomic roles of DNA modifications, it is critical to accurately map their diverse chemical forms with single-nucleotide precision in complex genomes, but it remains challenging. Click-code-seq is a click-chemistry-aided single-nucleotide-resolution strategy for guanine-oxidation mapping, used in yeast DNA but having poor applicability to human genomes. Here, we upgraded click-code-seq to enable its first application for sequencing DNA oxidation and depurination in human genomes. For this, we developed a companion fluorescence assay, click-fluoro-quant, to rapidly quantify different common DNA modifications, and devised novel adapters to minimize false modification detection and assess modification frequency in cell populations. We uncovered that endogenous DNA oxidation in a human cell line has a highly similar pattern to cancer mutational signatures associated with reactive oxygen species. We established that the DNA-alkylating chemotherapy drug irofulven preferentially induces depurination in ApA dimers and promoter regions. Intriguingly, we revealed that oxidized guanines and apurinic sites, both irofulven-induced and endogenous, are depleted in gene transcribed strands, and the strand bias widens with increasing gene expression. This work substantially advances click-code-seq for deciphering the impacts of key modifications in human DNA on cellular physiology and toxicological responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6b6bbbe3beae3b0b23fbadc72da494f15786faa" target='_blank'>
              Click-chemistry-aided quantitation and sequencing of oxidized guanines and apurinic sites uncovers their transcription-linked strand bias in human cells
              </a>
            </td>
          <td>
            V. Takhaveev, N. Püllen, Navnit K. Singh, Sabrina M. Huber, Stefan Schauer, Hailey L. Gahlon, A. Poetsch, S. Sturla
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="Epigenetic dysregulation is widespread in cancer. However, the specific epigenetic regulators and the processes they control to drive cancer phenotypes are poorly understood. Here, we employed a novel, scalable and high-throughput in vivo method to perform iterative functional screens of over 250 epigenetic regulatory genes within autochthonous oncogenic KRAS-driven lung tumors. We identified multiple novel epigenetic tumor suppressor and tumor dependency genes. We show that a specific HBO1 complex and the MLL1 complex are among the most impactful tumor suppressive epigenetic regulators in lung. The histone modifications generated by the HBO1 complex are frequently absent or reduced in human lung adenocarcinomas. The HBO1 and MLL1 complexes regulate chromatin accessibility of shared genomic regions, lineage fidelity and the expression of canonical tumor suppressor genes. The HBO1 and MLL1 complexes are epistatic during lung tumorigenesis, and their functional correlation is conserved in human cancer cell lines. Together, these results demonstrate the value of quantitative methods to generate a phenotypic roadmap of epigenetic regulatory genes in tumorigenesis in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/426bd5afd0c2737de358e33f8340d2bf45546851" target='_blank'>
              Functional mapping of epigenetic regulators uncovers coordinated tumor suppression by the HBO1 and MLL1 complexes
              </a>
            </td>
          <td>
            Yuning J. Tang, Haiqing Xu, Nicholas W. Hughes, Samuel H. Kim, Paloma Ruiz, Emily G. Shuldiner, Steven S. Lopez, Jess D. Hebert, S. Karmakar, Laura Andrejka, D. N. Dolcen, Gábor Boross, Pauline Chu, Colin R. Detrick, Sarah E. Pierce, Emily L. Ashkin, W.J. Greenleaf, Anne K. Voss, Tim Thomas, M. van de Rijn, D. Petrov, M. Winslow
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="There are two key signatures of pediatric cancers: (a) higher prevalence of germline alterations and (b) heterogeneity in alteration types. Recent population-based assessments have demonstrated that children with birth defects (BDs) are more likely to develop cancer even without chromosomal anomalies; therefore, explorations of genetic alterations in children with BDs and cancers could provide new insights into the underlying mechanisms for pediatric tumor development. We performed whole-genome sequencing (WGS) on blood-derived DNA for 1556 individuals without chromosomal anomalies, including 454 BD probands with at least one type of malignant tumor, 757 cancer-free children with BDs, and 345 healthy individuals, focusing on copy number variation (CNV) analysis. Roughly half of the children with BD-cancer have CNVs that are not identified in BD-only/healthy individuals, and CNVs are not evenly distributed among these patients. Strong heterogeneity was observed, with a limited number of cancer predisposition genes containing CNVs in more than three patients. Moreover, functional enrichments of genes with CNVs showed that dozens of patients have variations related to the same biological pathways, such as deletions of genes with neurological functions and duplications of immune response genes. Phenotype clustering uncovered recurrences of patients with sarcoma: A notable enrichment was observed involving non-coding RNA regulators, showing strong signals related to growth and cancer regulations in functional analysis. In conclusion, we conducted one of the first genomic studies exploring the impact of CNVs on cancer development in children with BDs, unveiling new insights into the underlying biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0421f07c60f2733c310940e72c92010ab69f33fb" target='_blank'>
              Copy number variations contribute to malignant tumor development in children with serious birth defects.
              </a>
            </td>
          <td>
            Yichuan Liu, J. Glessner, Hui-Qi Qu, X. Chang, H. Qiu, Tiancheng Wang, F. Mentch, H. Hakonarson
          </td>
          <td>2024-08-14</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="It has been observed that men have a shorter lifespan and higher incidence of certain cancers than women. We postulate here that this phenomenon may be attributed to the “toxic Y chromosome”, owing to the expression of normally silent HSATII repeats in cancer, abundantly present on chromosome Y. Investigating the contribution of all repetitive elements to chromatin interactions using a genome-wide HiC set derived from a colon cancer cell line (HCT116), we found several anomalies regarding satellite HSATII elements when compared to other genomic repeat types: (i) HSATII repeats tend to form HiC pairs mostly with their own kind, and depleted in most heterologous repeat pairs, whereas other types of repeats readily form heterologous HiC pairs; (ii) when treated with 5aza, a chemotherapeutic agent, the number of long-distance HSATII-overlapping HiC pairs significantly decreased when compared to the untreated samples; (iii) and most importantly, in the treated samples interchromosomal HiC read pairs involving a chromosome Y-located repeat and an HSATII repeat on either end almost completely disappear. (iv) We also found that HSATII repeats lack overlap with other types of repeats. To account for these anomalies we queried non-B DNA structure forming such as G-quadruplexes and transcription-related triplex forming. We found that Y chromosome-coded HSATII repeats form significantly stronger triplexes than HSATII repeats on other chromosomes, their strength significantly correlating with the number of HiC reads they overlap with. We surmise that the above phenomena may cause the higher incidence of cancer in men and may lead to the disappearance of the Y chromosome observed in cancerous cells in men.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a53990a3a3a681b314979b859a2d2941c439f782" target='_blank'>
              Y chromosome-coded HSATII repeats may contribute to higher incidence of cancer in men
              </a>
            </td>
          <td>
            Hedi Hegyi, Matej Lexa
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The mammalian mitochondrial genome (mtDNA) is multicopy and its copy number (mtCN) varies widely across tissues, in development and in disease. Here, we systematically catalog this variation by assaying mtCN in 52 human tissues across 952 donors (10,499 samples from the Genotype-Tissue Expression project) and 20 murine tissues using qPCR, capturing 50- and 200-fold variation, respectively. We also estimate per cell mtCN across 173 human cell lines from the Cancer Cell Line Encyclopedia using whole-genome sequencing data and observe >50-fold variation. We then leverage the vast amount of genomics data available for these repositories to credential our resource and uncover mtDNA-related biology. Using already existing proteomics data, we show that variation in mtCN can be predicted by variation in TFAM, histone, and mitochondrial ribosome protein abundance. We also integrate mtCN estimates with the CRISPR gene dependency measurements to find that cell lines with high mtCN are resistant to loss of GPX4, a glutathione phospholipid hydroperoxidase. Our resource captures variation in mtCN across mammalian tissues and should be broadly useful to the research community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/407c1ab023792825943a400486c262ae1672c036" target='_blank'>
              Mitochondrial genome copy number variation across tissues in mice and humans
              </a>
            </td>
          <td>
            S. Rath, Rahul Gupta, E. Todres, Hong Wang, Alexis A. Jourdain, K. Ardlie, Sarah E. Calvo, V. Mootha
          </td>
          <td>2024-08-06</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>99</td>
        </tr>

        <tr id="Fusion-positive rhabdomyosarcoma is an aggressive pediatric cancer molecularly characterized by arrested myogenesis. The defining genetic driver, PAX3::FOXO1, functions as a chimeric gain-of-function transcription factor. An incomplete understanding of PAX3::FOXO1’s in vivo epigenetic mechanisms has hindered therapeutic development. Here, we establish a PAX3::FOXO1 zebrafish injection model and semi-automated ChIP-seq normalization strategy to evaluate how PAX3::FOXO1 initially interfaces with chromatin in a developmental context. We investigated PAX3::FOXO1’s recognition of chromatin and subsequent transcriptional consequences. We find that PAX3::FOXO1 interacts with inaccessible chromatin through partial/homeobox motif recognition consistent with pioneering activity. However, PAX3::FOXO1-genome binding through a composite paired-box/homeobox motif alters chromatin accessibility and redistributes H3K27ac to activate neural transcriptional programs. We uncover neural signatures that are highly representative of clinical rhabdomyosarcoma gene expression programs that are enriched following chemotherapy. Overall, we identify partial/homeobox motif recognition as a new mode for PAX3::FOXO1 pioneer function and identify neural signatures as a potentially critical PAX3::FOXO1 tumor initiation event.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c34ac92eb278acb6b0ddaf9def7a0af29156e864" target='_blank'>
              Rhabdomyosarcoma fusion oncoprotein initially pioneers a neural signature in vivo
              </a>
            </td>
          <td>
            Jack Kucinski, Alexi Tallan, C. Taslim, Meng Wang, Matthew V. Cannon, Katherine M. Silvius, Benjamin Z. Stanton, Genevieve C. Kendall
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction. It has been established that the presence of homologous recombination deficiency in a breast tumor is associated with the effectiveness of treatment. But despite the high chemosensitivity of the tumor to DNA-damaging agents, complete pathological responses to treatment are very rare. And this process may be based on a change in the somatic status of BRCA1, that is, a reversion and return of the wild-type allele occurs and the DNA repair function is restored.Aim. To evaluate changes in the presence of chromosomal aberrations and the expression profile of the main genes of homologous recombination in cell models of breast cancer under the influence of cisplatin and docetaxel.Materials and methods. The study was conducted on breast cancer tumor cell cultures: MCF-7, MDA-MB-231 and MDA-MB-468. A cell model of drug resistance was obtained for two drugs: cisplatin and docetaxel. RNA and DNA were isolated from cell suspension using the RNeasy Plus Mini Kit and QIAamp DNA Mini Kit (Qiagen, Germany), respectively. The expression level of homologous recombination genes was assessed using reverse transcription polymerase chain reaction. To assess the presence of chromosomal aberrations, microarray analysis was performed on DNA chips.Results. Restoration of normal copy number for the BRCA1, CDK12, CHEK1 and RAD51D genes in MCF-7 under the influence of cisplatin was shown. For BRCA2 and PALB2, amplifications were detected. A statistically significant increase in the expression of the BRCA1 (p = 0.04), BRCA2 (p = 0.02), PALB2 (p = 0.01) and RAD51D (p = 0.05) genes was also shown. MDAMB-231 shows that all identified loci with deletions, where the BRCA2, BARD1, CHEK2, PALB2 and RAD54L genes are localized, are restored to normal copy number by cisplatin. The appearance of amplifications was registered for BRCA1, BRIP1, FANCL, RAD51B, PARP1. A similar result was shown for docetaxel. An increase in the expression level is typical for the genes BRCA1 (p = 0.02), BRCA2 (p = 0.02), CHEK2 (p = 0.05), FANCL (p = 0.04), PALB2 (p = 0.05), RAD51C (p = 0.02), PARP1 (p = 0.02), which corresponds to the appearance of amplifications. In the MDA-MB-468 cell culture, an increase in the copy number of only the BRCA1 gene is observed. The effect of docetaxel has no effect on this cell culture. The level of BRCA1 expression increases in direct proportion to the duration of drug action.Conclusion. Thus, the study showed that under the influence of cisplatin, reversion of not only homologous recombination gene mutations, but also other disorders can occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/767d30042e565f8e61dafca9671915ee6d03ecf7" target='_blank'>
              Modulation of homologous recombination gene activity in breast tumor cells in an in vitro model
              </a>
            </td>
          <td>
            M. Tsyganov, A. Frolova, E. A. Kravtsova, I. A. Tsydenova, M. Ibragimova
          </td>
          <td>2024-06-28</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Facioscapulohumeral muscular dystrophy (FSHD) is a hereditary myopathy linked to deletions of the tandemly arrayed D4Z4 macrosatellite repeats at human chromosome 4q35. These deletions accompany local chromatin changes and the anomalous expression of nearby transcripts FRG2A, DBET, and D4Z4. We discovered that FRG2A is one member of a family of long non-coding RNAs (lncRNAs) expressed at elevated levels in skeletal muscle cells with distinct amounts detected in individual patients. We found that FRG2A lncRNA preferentially associates with rDNA sequences and centromeres and promotes the three-dimensional association of centromeres with the nucleolar periphery in FSHD cells. Furthermore, we demonstrate that the elevated FRG2A expression in cells from FSHD patients reduces rDNA transcription and global protein synthesis. Our results frame an entirely unanticipated new disease model in which elevated lncRNAs levels mediated by deletions of D4Z4 macrosatellite repeats leads to a diminished protein synthesis capacity, thereby contributing to muscle wasting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3bbf00c4536b6dafb9187c1ed22578e5765757d7" target='_blank'>
              A novel family of lncRNAs relate facioscapulohumeral muscular dystrophy to nucleolar architecture and protein synthesis
              </a>
            </td>
          <td>
            Valentina Salsi, Francesca Losi, B. Fosso, Marco Ferrarini, Sara Pini, Marcello Manfredi, Gaetano Vattemi, T. Mongini, Lorenzo Maggi, G. Pesole, Anthony K. Henras, Paul D. Kaufman, Brian McStay, Rossella Tupler
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>77</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, Andre Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c780b28cafd38289d7c874a5207cf0360a5c59d4" target='_blank'>
              Co-inheritance of recombined chromatids maintains heterozygosity in a parthenogenetic ant
              </a>
            </td>
          <td>
            Kip D. Lacy, Taylor Hart, D. Kronauer
          </td>
          <td>2024-07-16</td>
          <td>Nature Ecology & Evolution</td>
          <td>1</td>
          <td>30</td>
        </tr>

        <tr id="The protozoan Trypanosoma cruzi, the causative agent of Chagas disease, exhibits polycistronic transcription and unidimensional genome compartmentalization of core (conserved) and disruptive (virulence factors from multigenic families) genes. Approximately 50% of its genome is repetitive, mainly virulence factor genes. Genomic sequences, including repeats, motifs of architectural proteins, and noncoding RNA loci are crucial for genome folding. Here, we evaluated the genomic features associated with higher-order chromatin organization in T. cruzi through extensive computational processing of high-throughput chromosome conformation capture (Hi-C) data, accounting for repetitive regions and improvements in genome annotation. Our study revealed that repetitive DNA (multimapped reads) influences 3D chromatin folding, particularly in determining the boundaries of topologically associated domains (TAD)-like structures. Virulence factor genes, unlike core genes, form shorter and more compact TAD-like structures enriched in loops, suggesting a gene expression regulatory mechanism. We found nonprotein-coding RNA loci (e.g., tRNAs) and transcription termination sites preferentially located at the boundaries of the TAD-like structures, while pseudogenes and multigenic family genes located in unstructured genomic regions. Our data indicate 3D clustering of tRNA loci, likely optimizing transcription by RNA polymerase III, and a complex interaction between spliced-leader RNA and 18S rRNA loci. Our findings provide insights into 3D genome organization in T. cruzi, contributing to the understanding of supranucleosome-level chromatin organization and suggesting possible links between 3D architecture and gene expression. We draw an analogy to the art of origami (e.g., papers folded into various shapes) resembling the DNA packed in chromatin fibers assuming distinct folds within the nucleus. Importance Despite the knowledge about the linear genome sequence and the identification of numerous virulence factors in the protozoan parasite Trypanosoma cruzi, there has been a limited understanding of how these genomic features are spatially organized within the nucleus and how this organization impacts gene regulation and pathogenicity. By providing a detailed analysis of the three-dimensional chromatin architecture in T. cruzi, our study contributed to filling this gap. We deciphered part of the origami structure hidden in the T. cruzi nucleus, showing the unidimensional genomic features are nonrandomly organized in the nuclear 3D landscape. We revealed the possible role of non-protein-coding RNA loci (e.g., tRNAs, SL-RNA, and 18S RNA) in shaping the genomic architecture. These findings provide insights into an additional epigenetic layer that may influence gene expression. Graphical abstract The spatial organization of chromatin within the nuclei of T. cruzi and its resemblance to origami art. A. Identification of the 3D nuclear architectures within T. cruzi nuclei: topologically associating domains (TADs) and their boundaries; chromatin loops; and 3D networks. Inter- and intrachromosomal interactions reflect DNA‒DNA contacts on the same (cis) and between different (trans) chromosomes. B. Resemblance between origami art and chromatin folding. Steps “a” to “l” show the process of folding a flat piece of paper from its unidimensional view up to its 3D boat form.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e7ca9b9ac7a3e68f2ea6b25af2d09f1d131e151" target='_blank'>
              Hidden origami in Trypanosoma cruzi nuclei highlights its nonrandom 3D genomic organization
              </a>
            </td>
          <td>
            N. K. Bellini, Pedro Leonardo Carvalho de Lima, D. Pires, J. D. da Cunha
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Near-haploidization, that is, loss of one copy of most chromosomes, is a relatively rare phenomenon in most tumors, but is enriched among certain soft tissue sarcomas, including undifferentiated pleomorphic sarcoma (UPS). Presumably, near-haploidization can arise through many mechanisms. This study aimed to identify gene rearrangements that could cause near-haploidization. We here present two UPS in which near-haploidization was an early event, identified through single nucleotide polymorphism (SNP) array analysis. One of the cases was studied further using whole genome and transcriptome sequencing, as well as cytogenetic and molecular cytogenetic methods. Both tumors had chromosomal rearrangements in the form of copy number shifts/structural variants affecting the SMC1A gene. These findings suggest that cohesin defects could contribute to mitotic errors resulting in massive loss of chromosomes. SMC1A encodes one of the components of the cohesin multiprotein complex, which is critical for proper alignment of the sister chromatids during S-phase and separation to opposite spindle poles. Further studies should explore the role of cohesin defects in near-haploidization in other sarcomas and to clarify its role in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac46389944e17adf0a38d1c92f41322a72ad7ff" target='_blank'>
              Structural Variants in the SMC1A Gene Associated With Near-Haploidy in Undifferentiated Pleomorphic Sarcomas.
              </a>
            </td>
          <td>
            Sebastian Ibstedt, Paul Piccinelli, Saskia Sydow, Jan Köster, Fredrik Mertens
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="While genotoxic chemotherapeutic agents are among the most effective tools to combat cancer, they are often associated with severe adverse effects caused by indiscriminate DNA damage in non-tumor tissue as well as increased risk of secondary carcinogenesis. This study builds on our previous work demonstrating that the RNA Polymerase I (Pol I) transcription inhibitor CX-5461 elicits a non-canonical DNA damage response and our discovery of a critical role for Topoisomerase 2α (Top2α) in the initiation of Pol I-dependent transcription. Here, we identify Top2α as a mediator of CX-5461 response in the murine Eµ-Myc B lymphoma model whereby sensitivity to CX-5461 is dependent on cellular Top2α expression/activity. Most strikingly, and in contrast to canonical Top2α poisons, we found that the Top2α-dependent DNA damage induced by CX-5461 is preferentially localized at the ribosomal DNA (rDNA) promoter region, thereby highlighting CX-5461 as a loci-specific DNA damaging agent. This mechanism underpins the efficacy of CX-5461 against certain types of cancer and can be used to develop effective non-genotoxic anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/913f673912030e76f53faa13fb8d57cb178fa48f" target='_blank'>
              CX-5461 Preferentially Induces Top2α-Dependent DNA Breaks at Ribosomal DNA Loci
              </a>
            </td>
          <td>
            Donald P. Cameron, J. Sornkom, Sameerh Alsahafi, D. Drygin, G. Poortinga, Grant A. McArthur, Nadine Hein, Ross D Hannan, Konstantin I. Panov
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract In ovarian and breast cancer, promoter methylation of BRCA1 or RAD51C is a promising biomarker for PARP inhibitor response, as high levels lead to homologous recombination deficiency (HRD). Yet the extent and role of such methylation in other cancers is not clear. This study comprehensively investigated promoter methylation of eight homologous recombination repair genes across 23 solid cancer types. Here, we showed that BRCA1 methylated cancers were associated with reduced gene expression, loss of heterozygosity (LOH), TP53 mutations and genomic features of HRD. We identified BRCA1 methylation in 3% of the copy-number high subtype of endometrial cancer, and as a rare event in six other cancer types, including lung squamous cell, pancreatic, bladder and stomach cancer. RAD51C promoter methylation was widespread across multiple cancer types, but HRD features were only observed for cases which contained high-level tumour methylation and LOH of RAD51C. While RAD51C methylation was frequent in stomach adenocarcinoma (6%) and low-grade glioma (2.5%), it was mostly detected at a low tumour level, suggestive of heterozygous methylation, and was associated with CpG island methylator phenotype. Our findings indicate that high-level tumour methylation of BRCA1 and RAD51C should be explored as a PARP inhibitor biomarker across multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/374a7a0b66d69ba1f5dc2c4d8219381d1c2a8cfc" target='_blank'>
              High-level tumour methylation of BRCA1 and RAD51C is required for homologous recombination deficiency in solid cancers
              </a>
            </td>
          <td>
            Lijun Xu, Brett Liddell, K. Nešić, Franziska Geissler, Lauren M. Ashwood, Matthew J. Wakefield, Clare L Scott, Nicola Waddell, O. Kondrashova
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9b28bde62e7c6ec79bde107b1d678f6c8aab03e" target='_blank'>
              MethNet: a robust approach to identify regulatory hubs and their distal targets from cancer data
              </a>
            </td>
          <td>
            Theodore Sakellaropoulos, Catherine Do, Guimei Jiang, Giulia Cova, Peter Meyn, D. Dimartino, Sitharam Ramaswami, Adriana Heguy, A. Tsirigos, Jane A Skok
          </td>
          <td>2024-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="RNA modification pathways are mis-regulated in multiple types of human cancer. To comprehensively identify cancer-relevant RNA modifications and their regulators, we screened all 150 annotated human RNA modifying proteins across 18 different normal and cancer cell lines using a CRISPR-based genetic knockout system. Fifty RNA modifying proteins were essential for survival of at least one cell type. A third of these essential genes were amplified in 38 different human primary cancer types and potentially drive cancer growth. Unexpectedly, the number of essential genes per cell line varied considerably, and this variation was not due to tissue of origin. Instead, we found that cancer cell-specific mitochondrial metabolic plasticity was responsible for the unique requirement of certain RNA modifications. For example, leukemia cells with high intrinsic drug tolerance required mitochondrial flexibility to survive treatment with the anti-leukemic drugs cytarabine and venetoclax. Synthetic lethality screens revealed that drug-resistance is abolished by deleting the mitochondrial methyltransferase TRMT5, which is responsible for the formation of N1-methylguanosine (m1G) in the tRNA anticodon loop. In summary, our study identifies cancer-relevant RNA modifying enzymes, and reveals a novel promising drug target for therapy-resistant acute myeloid leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7426bf54469ae857a96fb69b5f2913c6be2bfd40" target='_blank'>
              Unbiased functional genetic screens reveal essential RNA modifications in human cancer and drug resistance
              </a>
            </td>
          <td>
            C. Pauli, M. Kienhöfer, M. Blank, Oguzhan Begik, Christian Rohde, D. Heid, Fu Xu, Sarah Miriam Naomi Zimmermann, Katharina Weidenauer, Sylvain Delaunay, Nadja Krall, Katrin Trunk, Duoduo Zhao, Fengbiao Zhou, Anke Heit-Mondrzyk, U. Platzbecker, Claudia Baldus, H. Serve, M. Bornhäuser, Cathrine Broberg Vågbø, Salvador Aznar Benitah, Jeroen Krijgsveld, E. Novoa, Carsten Müller-Tidow, Michaela Frye
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c429a219f0b035a7a915d9f1d9d1a5ce65fcffe" target='_blank'>
              DKK1-SE recruits AP1 to activate the target gene DKK1 thereby promoting pancreatic cancer progression
              </a>
            </td>
          <td>
            Lan Shao, Haoran Yu, Mengyun Wang, Lu Chen, Boshu Ji, Tong Wu, Xiangqi Teng, Mu Su, Xiao Han, Weikai Shi, Xin Hu, Ziwen Wang, Hongjuan He, Guiping Han, Yan Zhang, Qiong Wu
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/724cc2cf5494857c9a50f7bbbea10f5d098c777c" target='_blank'>
              The genomic landscape of 2,023 colorectal cancers.
              </a>
            </td>
          <td>
            A. Cornish, Andreas J. Gruber, B. Kinnersley, D. Chubb, A. Frangou, G. Caravagna, Boris Noyvert, E. Lakatos, H.M. Wood, Steve Thorn, R. Culliford, Claudia Arnedo-Pac, J. Househam, William Cross, Amit Sud, P. Law, Máire Ní Leathlobhair, A. Hawari, C. Woolley, Kitty Sherwood, Nathalie Feeley, Güler Gül, Juan Fernandez-Tajes, L. Zapata, Ludmil B. Alexandrov, N. Murugaesu, A. Sosinsky, Jonathan Mitchell, Núria López-Bigas, P. Quirke, David N Church, Ian P. M. Tomlinson, A. Sottoriva, Trevor A Graham, David C Wedge, R. Houlston
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The relevant role of LINE-1 (L1) retrotransposition in cancer has been recurrently demonstrated in recent years. However, their repetitive nature hampers their identification and detection, hence remaining inaccessible for clinical practice. Also, its clinical relevance for cancer patients is still limited. Here, we develop a new method to quantify L1 activation, called RetroTest, based on targeted sequencing and a sophisticated bioinformatic pipeline, allowing its application in tumor biopsies. First, we performed the benchmarking of the method and confirmed its high specificity and reliability. Then, we unravel the L1 activation in HNSCC according to a more extensive cohort including all the HNSCC tumor stages. Our results confirm that RetroTest is remarkably efficient for L1 detection in tumor biopsies, reaching a high sensitivity and specificity. In addition, L1 retrotransposition estimation reveals a surprisingly early activation in HNSCC progression, contrary to its classical association with advanced tumor stages. This early activation together with the genomic mutational profiling of normal adjacent tissues supports field cancerization process in this tumor. These results underline the importance of estimating L1 retrotransposition in clinical practice towards an earlier and more efficient diagnosis in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ae21a6395bf2fa53698eb0f53f5dc6c7132bfe" target='_blank'>
              RetroTest unravels LINE-1 retrotransposition in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, Ana Oitaben, Sonia Zumalave, Bernardo Rodriguez-4 Martin, María Gallardo-Gómez, Martín Santamarina, Ana Pequeño-Valtierra, L. Juaneda-Magdalena, Ramón García-Escudero, J. López-Cedrún, Máximo Fraga, J. Tubío, Mónica Martínez-Fernández, Monica Martínez
          </td>
          <td>2024-07-24</td>
          <td>None</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Genomic, epigenomic, and transcriptomic alterations are hallmarks of cancer cells, and are closely connected. Especially, epigenetic regulation plays a critical role in tumorigenesis and progression. The growing single-cell epigenome data in cancer research provide new opportunities for data mining from a more comprehensive perspective. However, there is still a lack of databases designed for interactively exploring the single-cell multi-omics data of human pan-cancer, especially for the single-cell epigenome data. To fill in the gap, we developed scCancerExplorer, a comprehensive and user-friendly database to facilitate the exploration of the single-cell genome, epigenome (chromatin accessibility and DNA methylation), and transcriptome data of 50 cancer types. Five major modules were provided to explore those data interactively, including "integrated multi-omics analysis", "single-cell transcriptome", "single-cell epigenome", "single-cell genome", and "TCGA analysis". By simple clicking, users can easily investigate gene expression features, chromatin accessibility patterns, transcription factor activities, DNA methylation states, copy number variations, and TCGA survival analysis results. Taken together, scCancerExplorer is distinguished from previous databases with rich and interactive functions for exploring the single-cell multi-omics data of human pan-cancer. It bridges the gap between single-cell multi-omics data and the end-users, and will facilitate progress in the field of cancer research. scCancerExplorer is freely accessible via https://bianlab.cn/scCancerExplorer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f0bfe81970da09d476152acc53e069683a992ff" target='_blank'>
              scCancerExplorer: a comprehensive database for interactively exploring single-cell multi-omics data of human pan-cancer
              </a>
            </td>
          <td>
            Changzhi Huang, Zekai Liu, Yunlei Guo, Wanchu Wang, Zhen Yuan, Yusheng Guan, Deng Pan, Zhibin Hu, Linhua Sun, Zan Fu, Shuhui Bian
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Purpose: Complex chromosomal rearrangements (CCRs) are rare structural variants involving three or more chromosomal break- points. Most de novo reported CCRs pose challenges for diagnosis and management. They often require karyotyping, fluorescence in situ hybridization (FISH), and chromosomal microarray analysis (CMA) for clinical diagnosis because of the limitations of each method. Here we report an inherited exceptionally complex CCR involving 4 chromosomes and 11 breakpoints in a family with multisystem anomalies. Methods: We evaluated the CCRs using karyotyping, FISH, CMA, and two emerging genomic technologies: high-throughput chromosome conformation capture sequencing (Hi-C; aka genomic proximity mapping, GPM) and optical genome mapping (OGM). We also performed functional studies using transcriptome and methylome analyses. Results: The proband, who had intellectual disability and immune deficiency, shared CCRs with her unaffected mother involving chromosomes 1, 7, and 11 by karyotyping. However, CMA revealed a duplication and three deletions in the proband in contrast to her mother's balanced genome. Hi-C (GPM) and OGM detected the CCRs and copy number alterations but also uncovered additional breakpoints at high resolution, including an insertion in 4p and two cryptic rearrangements at 7p. Transcriptome and methylome analyses identified likely biological pathways associated with the proband's phenotypes. Conclusion: Combining cytogenetic and genomic methods provided comprehensive characterization and defined the breakpoints at high resolution in both proband and mother. This underscores the value of novel cytogenetic and genomic techniques in deciphering complex genome rearrangements and the significance of integrative genomic analysis and functional characterization in understanding clinical phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d8dc703c96a52057ab6829a72726f61eb74bd56" target='_blank'>
              Genetic and functional characterization of inherited complex chromosomal rearrangements in a family with multisystem anomalies
              </a>
            </td>
          <td>
            H. Fang, S. M. Eacker, Y. Wu, W. Neufeld-Kaiser, M. Laurino, S. Keel, M. S. Horwitz, Y. Liu
          </td>
          <td>2024-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Esophageal adenocarcinoma (EAC) is a highly lethal cancer of the upper gastrointestinal tract with rising incidence in western populations. To decipher EAC disease progression and therapeutic response, we performed multiomic analyses of a cohort of primary and metastatic EAC tumors, incorporating single-nuclei transcriptomic and chromatin accessibility sequencing, along with spatial profiling. We identified tumor microenvironmental features previously described to associate with therapy response. We identified five malignant cell programs, including undifferentiated, intermediate, differentiated, epithelial-to-mesenchymal transition, and cycling programs, which were associated with differential epigenetic plasticity and clinical outcomes, and for which we inferred candidate transcription factor regulons. Furthermore, we revealed diverse spatial localizations of malignant cells expressing their associated transcriptional programs and predicted their significant interactions with microenvironmental cell types. We validated our findings in three external single-cell RNA-seq and three bulk RNA-seq studies. Altogether, our findings advance the understanding of EAC heterogeneity, disease progression, and therapeutic response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50fba5605b21a8d390e36bfc5266450e36cd5998" target='_blank'>
              Cell states and neighborhoods in distinct clinical stages of primary and metastatic esophageal adenocarcinoma
              </a>
            </td>
          <td>
            Josephine Yates, Camille Mathey-Andrews, Jihye Park, Amanda Garza, Andréanne Gagné, Samantha Hoffman, Kevin Bi, Breanna M. Titchen, Connor J Hennessey, Joshua Remland, Erin Shannon, Sabrina Camp, Siddhi Balamurali, Shweta Kiran Cavale, Zhixin Li, Akhouri Kishore Raghawan, Agnieszka Kraft, Genevieve Boland, Andrew J. Aguirre, Nilay S. Sethi, Valentina Boeva, E. V. Van Allen
          </td>
          <td>2024-08-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Of the more than 100 types of brain cancer, glioblastoma (GBM) is the deadliest. As GBM stem cells (GSCs) are considered to be responsible for therapeutic resistance and tumor recurrence, effective targeting and elimination of GSCs could hold promise for preventing GBM recurrence and achieving potential cures. We show here that SUV39H1, which encodes a histone-3, lysine-9 methyltransferase, plays a critical role in GSC maintenance and GBM progression. Upregulation of SUV39H1 was observed in GBM samples compared to normal brain tissues, and knockdown of SUV39H1 in patient-derived GSCs impaired their proliferation and stemness. Single-cell RNA-seq analysis demonstrated restricted expression of SUV39H1 is in GSCs relative to non-stem GBM cells, likely due to super-enhancer-mediated transcriptional activation, while whole cell RNA-seq analysis revealed that SUV39H1 regulates G2/M cell cycle progression, stem cell maintenance, and cell death pathways in GSCs. By integrating the RNA-seq data with ATAC-seq (assay for transposase-accessible chromatin followed by sequencing), we further demonstrated altered chromatin accessibility in key genes associated with these pathways following SUV39H1 knockdown. Treatment with chaetocin, a SUV39H1 inhibitor, mimicked the functional effects of SUV39H1 knockdown in GSCs and sensitized GSCs to the GBM chemotherapy drug temozolomide. Furthermore, targeting SUV39H1 in vivo using a patient-derived xenograft model for GBM inhibited GSC-driven tumor formation. This is the first report demonstrating a critical role for SUV39H1 in GSC maintenance. SUV39H1-mediated targeting of GSCs could enhance the efficacy of existing chemotherapy, presenting a promising strategy for improving GBM treatment and patient outcomes. Highlights SUV39H1 is upregulated in GBM, especially GSCs Targeting SUV39H1 disrupts GSC maintenance and sensitizes GSCs to TMZ Targeting SUV39H1 alters chromatin accessibility at cell cycle and stemness genes Targeting SUV39H1 suppresses GSC-driven tumors in a patient-derived xenograft model">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3f51a1ca26253028a59eb7d526ac8f76a7e04a6" target='_blank'>
              SUV39H1 Preserves Cancer Stem Cell Chromatin State and Properties in Glioblastoma
              </a>
            </td>
          <td>
            Chunying Li, Qiqi Xie, Sugata Ghosh, Bihui Cao, Yuanning Du, G. V. Vo, Timothy Y. Huang, Charles Spruck, Y. A. Wang, K. Nephew, Jia Shen
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>69</td>
        </tr>

        <tr id="Animal chromosomes can persist with recognizable homology over hundreds of millions of years, in spite of homology-obfuscating processes such as chromosomal fusion and translocation. The frequency and pace of these major genome structural changes are unknown, and it remains unclear whether or how they impact long-term genome evolution. Here, we compare whole chromosomal sequences of 3,631 genomes from 2,291 species spanning all major animal clades and show that animal karyotypes evolve primarily via karyotype contraction, associated with increased rates of chromosomal fusion-with-mixing and dispersion that largely obey chromosomal algebra1, or karyotype expansion, which breaks up ancestral linkage groups and forms new chromosomal elements via non-algebraic changes. We show that chromosomal changes can be associated with major extinction events. Using a multi-scale encoding of pan-animal genome homology and a manifold representation of genomic changes, we find that genome evolution is not only driven by changes at the chromosomal level, but that subchromosomal mixing and irreversibility define clade-specific evolution. Using this ‘evolutionary genome topology’ approach, we calculate extrema of irreversible genomic configurations and identify species that occupy intermediate manifold positions, providing evidence for distinct macro-evolutionary trajectories. We propose that investigation of mixed state accumulation around important gene loci (such as Hox) will be crucial in capturing and further study of clade-specific regulatory innovations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bcfa759f5e30b92ec5cb430b3076d461c210e1e" target='_blank'>
              Topological mixing and irreversibility in animal chromosome evolution
              </a>
            </td>
          <td>
            D. Schultz, Arno Blümel, Dalila Destanović, Fatih Sarigol, Oleg Simakov
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The myeloma overexpressed gene (MYEOV) has been proposed to be a proto-oncogene due to high RNA transcript levels found in multiple cancers, including myeloma, breast, lung, pancreas and esophageal cancer. The presence of an open reading frame (ORF) in humans and other primates suggests protein-coding potential. Yet, we still lack evidence of a functional MYEOV protein. It remains undetermined how MYEOV overexpression affects cancerous tissues. In this work, we show that MYEOV has likely originated and may still function as an enhancer, regulating CCND1 and LTO1. Firstly, MYEOV 3′ enhancer activity was confirmed in humans using publicly available ATAC-STARR-seq data, performed on B-cell-derived GM12878 cells. We detected enhancer histone marks H3K4me1 and H3K27ac overlapping MYEOV in multiple healthy human tissues, which include B cells, liver and lung tissue. The analysis of 3D genome datasets revealed chromatin interactions between a MYEOV-3′-putative enhancer and the proto-oncogene CCND1. BLAST searches and multi-sequence alignment results showed that DNA sequence from this human enhancer element is conserved from the amphibians/amniotes divergence, with a 273 bp conserved region also found in all mammals, and even in chickens, where it is consistently located near the corresponding CCND1 orthologues. Furthermore, we observed conservation of an active enhancer state in the MYEOV orthologues of four non-human primates, dogs, rats, and mice. When studying this homologous region in mice, where the ORF of MYEOV is absent, we not only observed an enhancer chromatin state but also found interactions between the mouse enhancer homolog and Ccnd1 using 3D-genome interaction data. This is similar to the interaction observed in humans and, interestingly, coincides with CTCF binding sites in both species. Taken together, this suggests that MYEOV is a primate-specific gene with a de novo ORF that originated at an evolutionarily older enhancer region. This deeply conserved putative enhancer element could regulate CCND1 in both humans and mice, opening the possibility of studying MYEOV regulatory functions in cancer using non-primate animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f17ce82b4897c471d6bebe571f550d964a45b5c" target='_blank'>
              Evolutionarily conserved enhancer-associated features within the MYEOV locus suggest a regulatory role for this non-coding DNA region in cancer
              </a>
            </td>
          <td>
            Brigid S. A. Davidson, Juliana Estefanía Arcila-Galvis, Marco Trevisan-Herraz, Aneta Mikulasova, C. Brackley, Lisa J Russell, Daniel Rico
          </td>
          <td>2024-07-30</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Melanoma showcases a complex interplay of genetic alterations and intra- and inter-cellular morphological changes during metastatic transformation. While pivotal, the role of specific mutations in dictating these changes still needs to be fully elucidated. Telomerase promoter mutations (TERTp mutations) significantly influence melanoma’s progression, invasiveness, and resistance to various emerging treatments, including chemical inhibitors, telomerase inhibitors, targeted therapy, and immunotherapies. We aim to understand the morphological and phenotypic implications of the two dominant monoallelic TERTp mutations, C228T and C250T, enriched in melanoma metastasis. We developed isogenic clonal cell lines containing the TERTp mutations and utilized dual-color expression reporters steered by the endogenous Telomerase promoter, giving us allelic resolution. This approach allowed us to monitor morpholomic variations induced by these mutations. TERTp mutation-bearing cells exhibited significant morpholome differences from their wild-type counterparts, with increased allele expression patterns, augmented wound-healing rates, and unique spatiotemporal dynamics. Notably, the C250T mutation exerted more pronounced changes in the morpholome than C228T, suggesting a differential role in metastatic potential. Our findings underscore the distinct influence of TERTp mutations on melanoma’s cellular architecture and behavior. The C250T mutation may offer a unique morpholomic and systems-driven advantage for metastasis. These insights provide a foundational understanding of how a non-coding mutation in melanoma metastasis affects the system, manifesting in cellular morpholome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b66856ef0863e2a38d3da5fdaad88a280ff968" target='_blank'>
              Using deep learning to decipher the impact of telomerase promoter mutations on the dynamic metastatic morpholome
              </a>
            </td>
          <td>
            Andres J. Nevarez, Anusorn Mudla, Sabrina A. Diaz, Nan Hao
          </td>
          <td>2024-07-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Background and Aim: Canine transmissible venereal tumor (CTVT), a unique transmissible cancer in dogs, affects the external genitalia and potentially spreads to other parts of the body. While somatic mutations in oncogenic and tumor-suppressing genes are linked to CTVT development, the impact of DNA methylation, which affects gene expression, remains unclear. This study explored whether DNA methylation in the promoter regions of the MYC oncogene and CDKN2B tumor suppressor genes in CTVTs is associated with their expression, both at the gene and protein levels. Materials and Methods: To investigate promoter DNA methylation of MYC and CDKN2B in CTVTs, we analyzed frozen tissue samples from genital CTVT (GTVTs) and extragenital CTVT (ETVTs). Genomic DNA was extracted, bisulfite-treated, and analyzed using bisulfite polymerase chain reaction (PCR) and sequencing. The messenger RNA and protein of MYC and CDKN2B were also extracted and assessed by real-time PCR and Western blotting. Matching formalin-fixed, paraffin-embedded blocks were used for immunohistochemical staining to visualize protein distribution in GTVT and ETVT tissues. Results: Although both GTVT and ETVT samples showed MYC promoter methylation, the extent of methylation differed significantly. GTVTs displayed a much higher degree of methylation, potentially explaining the more pronounced downregulation of MYC gene expression and reduction in c-MYC protein levels observed in GTVTs compared with ETVTs. Our data revealed a prevalent hypermethylation pattern in the CDKN2B promoter across both sample types. However, DNA methylation, which was expected to have a suppressive effect, did not correlate with gene/protein expression. GTVTs displayed high protein levels despite significantly reduced CDKN2B expression. Conversely, ETVTs maintained regular CDKN2B expression but exhibited reduced protein production, suggesting a complex interplay between methylation and expression in these tumors. Conclusion: MYC demonstrated a clear association between its promoter methylation status, gene expression, and protein levels; however, CDKN2B lacked this correlation, implying the involvement of methylation-independent regulatory mechanisms and highlighting the need for further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26c5019910fb12afd2000e4151f192b0d24fad38" target='_blank'>
              Divergent DNA methylation patterns and gene expression in MYC and CDKN2B in canine transmissible venereal tumors
              </a>
            </td>
          <td>
            Soukkangna Keopaseuth, K. Pringproa, P. Patchanee, C. Setthawongsin, S. Techangamsuwan, Phongsakorn Chuammitri
          </td>
          <td>2024-07-01</td>
          <td>Veterinary World</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The expression of eukaryotic genes relies on the precise 3’-terminal cleavage and polyadenylation of newly synthesized pre-mRNA transcripts. Defects in these processes have been associated with various diseases, including cancer. While cancer-focused sequencing studies have identified numerous driver mutations in protein-coding sequences, noncoding drivers – particularly those affecting the cis-elements required for pre-mRNA cleavage and polyadenylation – have received less attention. Here, we systematically analysed cancer somatic mutations affecting 3’UTR polyadenylation signals using the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset. We found a striking enrichment of cancer-specific somatic mutations that disrupt strong and evolutionarily conserved cleavage and polyadenylation signals within tumour suppressor genes. Further bioinformatics and experimental analyses conducted as a part of our study suggest that these mutations have a profound capacity to downregulate the expression of tumour suppressor genes. Thus, this work uncovers a novel class of noncoding somatic mutations with significant potential to drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4b3938c7f84b24956b4223cbd287669eff1caf" target='_blank'>
              Recurrent disruption of tumour suppressor genes in cancer by somatic mutations in cleavage and polyadenylation signals
              </a>
            </td>
          <td>
            Yaroslav A. Kainov, F. Hamid, E. Makeyev
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Mutually exclusive loss-of-function alterations in gene pairs are those that occur together less frequently than may be expected and may denote a synthetically lethal relationship (SSL) between the genes. SSLs can be exploited therapeutically to selectively kill cancer cells. Here, we analysed mutation, copy number variation, and methylation levels in samples from The Cancer Genome Atlas, using the hypergeometric and the Poisson binomial tests to identify mutually exclusive inactivated genes. We focused on gene pairs where one is an inactivated tumour suppressor and the other a gene whose protein product can be inhibited by known drugs. This provided an abundance of potential targeted therapeutics and repositioning opportunities for several cancers. These data are available on the MexDrugs website, https://bioinformaticslab.sussex.ac.uk/mexdrugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7250b34d6949747e0dee284668cf7df5c1a39253" target='_blank'>
              Identifying actionable synthetically lethal cancer gene pairs using mutual exclusivity.
              </a>
            </td>
          <td>
            Sarah K. Wooller, Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2024-07-08</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1082aba026dea72fee57aef4a45e7856e9efcf8" target='_blank'>
              Topoisomerase-modulated genome-wide DNA supercoiling domains colocalize with nuclear compartments and regulate human gene expression.
              </a>
            </td>
          <td>
            Qian Yao, Linying Zhu, Zhen Shi, Subhadra Banerjee, Chongyi Chen
          </td>
          <td>2024-08-16</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0807e64db184649a4b0df3bd08502c4d34be0a53" target='_blank'>
              Programming tumor evolution with selection gene drives to proactively combat drug resistance.
              </a>
            </td>
          <td>
            Scott M. Leighow, Josh Reynolds, Ivan Sokirniy, Shun Yao, Zeyu Yang, Haider Inam, Dominik Wodarz, Marco Archetti, Justin Pritchard
          </td>
          <td>2024-07-04</td>
          <td>Nature biotechnology</td>
          <td>2</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Neuroendocrine prostate cancer (NEPC) presents a formidable clinical challenge owing to its aggressive progression and resistance to conventional therapies. A key driver of NEPC is the overexpression of MYCN, a well-established oncogene associated with neuroendocrine tumors. However, efforts to directly inhibit the N-Myc protein encoded by this gene have resulted in limited success, thereby hindering therapeutic advancements. To overcome this obstacle, we conducted unbiased genome-wide screening using isogenic prostate cancer cell lines to identify the synthetic vulnerabilities of MYCN. Among the identified candidates, NEUROG2 emerged as a significant candidate. Neurog2 is a proneural transcription factor (PTF) known for its role in developmental processes and trans-differentiation of adult cells. Our findings demonstrate that Neurog2 depletion does not affect non-malignant cells, but significantly suppresses the growth of MYCN-overexpressing cells and tumors in orthotopic NEPC models. Furthermore, our observations indicate that the Neurog2-mediated regulation of PTFs can facilitate NEPC development. Thus, targeting Neurog2 holds promise as an effective therapeutic strategy for MYCN-overexpressing NEPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6178b90a3b078d38756c518ec592d0bd4f1191a1" target='_blank'>
              A novel role for Neurog2 in MYCN driven neuroendocrine plasticity of prostate cancer
              </a>
            </td>
          <td>
            Prachi Walke, Jared D.W. Price, Frederick S. Vizeacoumar, Nickson Joseph, Vincent Maranda, Bari Chowdhury, Jay Patel, Yue Zhang, He Dong, Lara New, Ashtalakshmi Ganapathysamy, Lihui Gong, Hussain Elhasasna, Kalpana K Bhanumathy, Yuliang Wu, A. Freywald, Anand Krishnan, Franco J Vizeacoumar
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21a24ea5db444de6c4cb2a6e44207610c8118487" target='_blank'>
              RHNO1: at the crossroads of DNA replication stress, DNA repair, and cancer.
              </a>
            </td>
          <td>
            Niphat Jirapongwattana, S. Bunting, Donald R Ronning, Gargi Ghosal, A. Karpf
          </td>
          <td>2024-08-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic regulatory mechanism that plays a critical role in a variety of cellular processes, including the regulation of cell fate during development, maintenance of cell identity, and genome stability. DNA methylation is tightly regulated by enzymatic reactions and its deregulation plays an important role in the development of cancer. Specific DNA methylation alterations have been found in pediatric solid tumors, providing new insights into the development of these tumors. In addition, DNA methylation profiles have greatly contributed to tune the diagnosis of pediatric solid tumors and to define subgroups of patients with different risks of progression, leading to the reduction in unwanted toxicity and the improvement of treatment efficacy. This review highlights the dysregulated DNA methylome in pediatric solid tumors and how this information provides promising targets for epigenetic therapies, particularly inhibitors of DNMT enzymes (DNMTis). Opportunities and limitations are considered, including the ability of DNMTis to induce viral mimicry and immune signaling by tumors. Besides intrinsic action against cancer cells, DNMTis have the potential to sensitize immune-cold tumors to immunotherapies and may represent a remarkable option to improve the treatment of challenging pediatric solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1eac857dfcc5a5e3b672ddb17d6097e48bc70f" target='_blank'>
              Targeting DNA Methylation Machinery in Pediatric Solid Tumors
              </a>
            </td>
          <td>
            C. Cristalli, Katia Scotlandi
          </td>
          <td>2024-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Sex-chromosome dosage represents a challenge for heterogametic species to maintain correct proportion of gene products across chromosomes in each sex. While therian mammals (XX/XY system) achieve near-perfect balance of X-chromosome mRNAs through X-upregulation and X-inactivation, birds (ZW/ZZ system) have been found to lack efficient compensation at RNA level, challenging the necessity of resolving major gene-dosage discrepancies in avian cells. Through allele-resolved multiome analyses, we comprehensively examined dosage compensation in female (ZW), male (ZZ), and rare intersex (ZZW) chicken. Remarkably, this revealed that females exhibit upregulation of their single Z through increased transcriptional burst frequency similar to mammalian X-upregulation, and that Z-protein levels are further balanced via enhanced translation efficiency in females. Global analyses of transcriptional kinetics elements in birds demonstrate remarkable conservation of the genomic encoding of burst kinetics between mammals and birds. Our study uncovers new mechanisms for achieving sex-chromosome dosage compensation and highlights the importance of gene-dosage balance across diverse species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05f4b40e70c2988331617fc51e819b31adbb3451" target='_blank'>
              Multi-layered dosage compensation of the avian Z chromosome
              </a>
            </td>
          <td>
            Natali Papanicolaou, Antonio Lentini, Sebastian Wettersten, Michael Hagemann-Jensen, Annika Krüger, Jilin Zhang, Christos Coucoravas, Ioannis Petrosian, Xian Xin, Ilhan Ceyhan, Joanna Rorbach, D. Wright, B. Reinius
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Transcribed enhancer maps can reveal nuclear interactions underpinning each cell type and connect specific cell types to diseases. Using a 5′ single-cell RNA sequencing approach, we defined transcription start sites of enhancer RNAs and other classes of coding and noncoding RNAs in human CD4+ T cells, revealing cellular heterogeneity and differentiation trajectories. Integration of these datasets with single-cell chromatin profiles showed that active enhancers with bidirectional RNA transcription are highly cell type–specific and that disease heritability is strongly enriched in these enhancers. The resulting cell type–resolved multimodal atlas of bidirectionally transcribed enhancers, which we linked with promoters using fine-scale chromatin contact maps, enabled us to systematically interpret genetic variants associated with a range of immune-mediated diseases. Editor’s summary Currently, 3′ single-cell RNA sequencing is widely used as a tool for single-cell gene expression analysis. By leveraging a distinctive “cap signature” derived from the 5′-end cap, Oguchi et al. introduced a technique to investigate the precise position of the RNA 5′-ends in single cells, enabling simultaneous profiling of gene expression and enhancer activity. The authors constructed a CD4+ T cell–type resolved multimodal atlas to integrate and elucidate CD4+ T cell diversity and genetic risk of immune-mediated diseases. They also identified rare, uncharacterized CD4+ T cell types that might contribute to the pathology of immune-mediated diseases. —Di Jiang INTRODUCTION Enhancers are distal cis-regulatory elements that increase the expression of target genes. Active enhancers produce bidirectional enhancer RNAs (eRNAs). Transcription of eRNAs can be used to measure enhancer activity and to delineate enhancer architecture at nucleotide resolution. Profiling of transcribed enhancers in single human cells can provide insights into cell type–specific gene regulation and genetic predisposition to diseases. However, comprehensive maps of eRNAs in cell types relevant to diseases are still lacking. RATIONALE The 5′ cap is added to the first nucleotide of an RNA molecule. By leveraging a specific “cap signature,” we developed read-level prefiltering and transcribed enhancer call (ReapTEC), a method for simultaneously profiling gene expression and enhancer activity by using 5′-end single-cell RNA sequencing (5′ scRNA-seq). We applied ReapTEC and single-nucleus assay for transposase-accessible chromatin using sequencing (snATAC-seq) to a total of ~1 million single human CD4+ T cells and constructed a CD4+ T cell type–resolved multimodal atlas to pinpoint genetic and cellular causes of immune-mediated diseases. RESULTS By using ReapTEC, we accurately mapped the transcription start sites of mRNAs, long noncoding RNAs, and eRNAs in single cells, which allowed us to study the expression of not only genes but also bidirectionally transcribed candidate enhancers (btcEnhs). We characterized rare and uncharacterized cell types, including GPR25-expressing regulatory T cells, along with the expression profiles of ~130 surface proteins. We identified 62,803 btcEnhs and 218,508 accessible chromatin regions active across diverse CD4+ T cells. We found higher cell type specificity for RNA transcription than for chromatin accessibility both at promoters and enhancers. We also showed that the heritability of immune-mediated diseases was enriched more in btcEnhs than in open chromatin regions. Using our CD4+ T cell type–resolved atlas of btcEnhs, we systematically interpreted genetic variants associated with a range of immune-mediated diseases. We found that btcEnhs harboring disease SNPs were often active in specific subsets of CD4+ T cells; generally, they were conserved across mammals but in some cases were primate specific. We investigated the target genes of disease-relevant btcEnhs by generating fine-scale chromatin contact maps derived from ultradeep Micro-C data; we further performed CRISPR activation of selected disease-relevant btcEnhs to validate their target genes. CONCLUSION Our study establishes a platform for integrating and elucidating CD4+ T cell diversity and genetic risk of immune-mediated diseases. Although 3′ scRNA-seq is widely used as a major tool for single-cell gene expression analysis, our study demonstrates the strength of 5′ scRNA-seq. We anticipate the utility of our multimodal atlas in addressing variant-to-function challenges. CD4+ T cell type–resolved atlas of btcEnhs for systematic interpretation of the genetic basis of immune-mediated diseases. (A) ReapTEC allows us to detect transcription start sites (TSSs) and active enhancers using 5′ scRNA-seq. (B) Disease-relevant btcEnhs containing genetic variants associated with immune-mediated diseases were linked to target genes by using fine-scale chromatin contact maps. CITE-seq, cellular indexing of transcriptomes and epitopes by sequencing; Ref, reference allele; Alt, alternative allele.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a4b6cb952b21d2e76916c18021dfddb89e4c76" target='_blank'>
              An atlas of transcribed enhancers across helper T cell diversity for decoding human diseases
              </a>
            </td>
          <td>
            Akiko Oguchi, A. Suzuki, Shuichiro Komatsu, Hiroyuki Yoshitomi, S. Bhagat, Raku Son, R. Bonnal, Shohei Kojima, Masaru Koido, Kazuhiro Takeuchi, K. Myouzen, Gyo Inoue, Tomoya Hirai, Hiromi Sano, Y. Takegami, A. Kanemaru, Itaru Yamaguchi, Y. Ishikawa, Nao Tanaka, Shigeki Hirabayashi, Riyo Konishi, Sho Sekito, Takahiro Inoue, Juha Kere, Shunichi Takeda, A. Takaori-Kondo, Itaru Endo, S. Kawaoka, Hideya Kawaji, K. Ishigaki, H. Ueno, Y. Hayashizaki, Massimiliano Pagani, P. Carninci, M. Yanagita, Nicholas Parrish, Chikashi Terao, Kazuhiko Yamamoto, Yasuhiro Murakawa
          </td>
          <td>2024-07-05</td>
          <td>Science</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Pancreatic cancer is a malignancy arising from the endocrine or exocrine compartment of this organ. Tumors from exocrine origin comprise over 90% of all pancreatic cancers diagnosed. Of these, pancreatic ductal adenocarcinoma (PDAC) is the most common histological subtype. The five-year survival rate for PDAC ranged between 5 and 9% for over four decades, and only recently saw a modest increase to ∼12-13%, making this a severe and lethal disease. Like other cancers, PDAC initiation stems from genetic changes. However, therapeutic targeting of PDAC genetic drivers has remained relatively unsuccessful, thus the focus in recent years has expanded to the non-genetic factors underlying the disease pathogenesis. Specifically, it has been proposed that dynamic changes in the epigenetic landscape promote tumor growth and metastasis. Emphasis has been given to the re-organization of enhancers, essential regulatory elements controlling oncogenic gene expression, commonly marked my histone 3 lysine 4 monomethylation (H3K4me1). H3K4me1 is typically deposited by histone lysine methyltransferases (KMTs). While well characterized as oncogenes in other cancer types, recent work has expanded the role of KMTs as tumor suppressor in pancreatic cancer. Here, we review the role and translational significance for PDAC development and therapeutics of KMTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a7417d79919ae9f2eab24051f1e99fe379fd6b" target='_blank'>
              Insights into the mechanisms driven by H3K4 KMTs in pancreatic cancer.
              </a>
            </td>
          <td>
            Kayla C LaRue-Nolan, G. L. Raja Arul, Ashley N Sigafoos, Jiaqi Shi, Martin E Fernandez-Zapico
          </td>
          <td>2024-07-30</td>
          <td>The Biochemical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Endothelial cells (ECs) maintain vessel tone and barrier integrity, regulate blood homeostasis, and prevent the extravasation of leukocytes under normal physiological conditions. Because of the limited lifespans and batch-to-batch differences with respect to the genetic make-up of primary ECs, established immortal EC lines are extensively used for studying endothelial biology. To address this issue, the immortal endothelial cell line EA.hy926 was developed by fusing primary human umbilical vein endothelial cells (HUVECs) with human lung carcinoma A549 cells. EA.hy926 cells share a number of similar endothelial properties with HUVECs and are considered the immortal counterpart to primary HUVECs. However, the cytogenetic integrity of EA.hy926 cells is not fully elucidated. We characterized EA.hy926 cells with conventional G-banding and molecular cytogenetic techniques such as spectral karyotyping and subtelomeric fluorescence in situ hybridization. Cytogenetic analysis revealed an array of numerical and stable structural chromosomal rearrangements including one deletion, one duplication, one isochromosome, seven simple translocations, and five complex translocations in Ea.hy926 cells. These findings will advance comprehension of EA.hy926 cell biology and augment future endothelial studies, specifically in comparison studies between HUVECs and EA.hy926 cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ad1b12e69b1e161748ea2c28f86f896e19f2f6" target='_blank'>
              G-Banding and Molecular Cytogenetics Detect Novel Translocations and Cryptic Aberrations in Human Immortal Endothelial Cells
              </a>
            </td>
          <td>
            Regina Lichti Binz, R. Pathak
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Disruption of the epigenome is a hallmark of human disease including liver cirrhosis and hepatocellular carcinoma (HCC). While genetic heterogeneity is an established effector of pathologic phenotypes, epigenetic heterogeneity is less well understood. Environmental exposures alter the liver-specific DNA methylation landscape and influence the onset of liver cancer. Given that currently available treatments are unable to target frequently mutated genes in HCC, there is an unmet need for novel therapeutics to prevent or reverse liver damage leading to hepatic tumorigenesis, which the epigenome may provide. We performed genome-wide profiling of DNA methylation, copy number, and gene expression from multiple liver regions from 31 patients with liver disease to examine their crosstalk and define the individual and combinatorial contribution of these processes to liver disease progression. We identify epigenetic heterogeneity hotspots that are conserved across patients. Elevated epigenetic heterogeneity associates with increased gene expression heterogeneity. Cirrhotic regions comprise two distinct cohorts, one exclusively epigenetic, and a second where epigenetic and copy number variations collaborate. Epigenetic heterogeneity hotspots are enriched for genes central to liver function (e.g., HNF1A) and known tumor suppressors (e.g., RASSF1A). These hotspots encompass genes including ACSL1, ACSL5, MAT1A, and ELFN1, which have phenotypic effects in functional screens, supporting their relevance to hepatocarcinogenesis. Moreover, epigenetic heterogeneity hotspots are linked to clinical measures of outcome.


CONCLUSION
Substantial epigenetic heterogeneity arises early in liver disease development, targeting key pathways in the progression and initiation of both cirrhosis and HCC. Integration of epigenetic and transcriptional heterogeneity unveils putative epigenetic regulators of hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46b475bb15d42990ae95e5c459fd1ea63a3f93cd" target='_blank'>
              Epigenetic heterogeneity hotspots in human liver disease progression.
              </a>
            </td>
          <td>
            R. Hlady, Xia Zhao, Louis El Khoury, Ryan T. Wagner, Aesis M. Luna, Kien Pham, Nikolaos Pyrosopoulos, Dhanpat Jain, Liguo Wang, Chen Liu, Keith Robertson
          </td>
          <td>2024-07-19</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="DNA base lesions, such as incorporation of uracil into DNA or base mismatches, can be mutagenic and toxic to replicating cells. To discover factors in repair of genomic uracil, we performed a CRISPR knockout screen in the presence of floxuridine, a chemotherapeutic agent that incorporates uracil and fluoro-uracil into DNA. We identified known factors, such as uracil DNA N-glycosylase (UNG), but also unknown factors, such as the N6-adenosine methyltransferase, METTL3, as required to overcome floxuridine-driven cytotoxicity. Visualized with immunofluorescence, the product of METTL3 activity, N6-methyladenosine, formed nuclear foci in cells treated with floxuridine. The observed N6-methyladenosine was embedded in DNA, called 6mA, which was confirmed using mass spectrometry. METTL3 and 6mA were required for repair of lesions driven by additional base damaging agents, including raltitrexed, gemcitabine, and hydroxyurea. Our results establish a role for METTL3 and 6mA to promote genome stability in mammalian cells, specially in response to base damage.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7db989576812beb4e3ed7a27aef984122d9c71b2" target='_blank'>
              N6-methyladenosine in DNA promotes genome stability
              </a>
            </td>
          <td>
            Brooke A. Conti, Leo Novikov, Deyan Tong, Qing Xiang, Savon Vigil, Thomas J. McLellan, Chuong Nguyen, Nancy De La Cruz, Reshma T. Veettil, Prashant Pradhan, Parag Sahasrabudhe, Jason D. Arroyo, Lei Shang, Benjamin R. Sabari, David J. Shields, Mariano Oppikofer
          </td>
          <td>2024-07-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Survival of Medulloblastoma (MB) depends on various factors, including the gene expression profiles of MB tumor tissues. In this study, we identified 967 MB survival-related genes (SRGs) using a gene expression dataset and the Cox proportional hazards regression model. Notably, the SRGs were over-represented on chromosomes 6 and 17, known for the abnormalities monosomy 6 and isochromosome 17 in MB. The most significant SRG was HMGA1 (high mobility group AT-hook 1) on chromosome 6, which is a known oncogene and a histone H1 competitor. High expression of HMGA1 was associated with worse survival, primarily in the Group 3γ subtype. The high expression of HMGA1 was unrelated to any known somatic copy number alteration. Most SRGs on chromosome 17p were associated with low expression in Group 4β, the MB subtype, with 93% deletion of 17p and 98% copy gain of 17q. GO enrichment analysis showed that both chromosomes 6 and 17 included SRGs related to telomere maintenance and provided a rationale for testing telomerase inhibitors in Group 3 MBs. We conclude that HMGA1, along with other SRGs on chromosomes 6 and 17, warrant further investigation as potential therapeutic targets in selected subgroups or subtypes of MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bb63da6d24c38cbfdc6fe840507e9aa6581f878" target='_blank'>
              Survival-Related Genes on Chromosomes 6 and 17 in Medulloblastoma
              </a>
            </td>
          <td>
            Jerry Vriend, Xiao-Qing Liu
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers.
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a2685872da8abf8d89c45e2e00bd648c1daaeb" target='_blank'>
              DNA methylation and gene expression as determinants of genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Michaël Noë, Dimitrios Mathios, A. Annapragada, Shashikant Koul, Zachariah H. Foda, J. Medina, S. Cristiano, Christopher Cherry, D. C. Bruhm, N. Niknafs, V. Adleff, Leonardo Ferreira, Hari Easwaran, Stephen Baylin, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Gene expression regulation is one of the most fundamental cellular processes, enabling the activation of a gene to produce either the translatable protein-coding transcript (mRNA) or a functional non-coding RNA with gene regulatory functions, ultimately determining cell identity and function. Although gene expression regulation can occur at transcriptional, translational, and post-translational levels, transcription initiation is the first and the most important step in gene expression, facilitating the transfer of biological information from DNA to protein. Enhancers and super-enhancers are among the master regulators of tissue- and cell-specific transcription regulation involved in cell differentiation and tumor formation. Despite four decades passing since the first discovery of enhancers in eukaryotes and extensive efforts undertaken to identify enhancers on a genomic scale during the last decade, the discovery of enhancers still faces certain limitations and needs further investigation. The perturbation of enhancer function due to genetic or epigenetic changes is closely linked to a range of human disorders, including the development and progression of cancers. Thus, the detection of early cancer-related enhancer activity and the subsequent normalization of expression abnormalities using enhancer-targeting CRISPR epigenetic editing, as well as enhancer-targeting pharmaceuticals, are regarded as groundbreaking therapeutic tactics in preclinical stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782ddfe76cab2a16620844e3e84b112d8c90071b" target='_blank'>
              Enhancers and super-enhancers as master regulators in cancer
              </a>
            </td>
          <td>
            P. Sarvari, Pourya Sarvari, Ivonne Ramírez-Díaz, Karla Rubio
          </td>
          <td>2024-07-24</td>
          <td>INNOSC Theranostics and Pharmacological Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="OBJECTIVE
Ionizing radiation generates genomic instability by promoting the accumulation of chromosomal rearrangements. The oncogenic translocation RET/PTC1 is present in more than 70% of radiation-induced thyroid cancers. Both RET and CCDC6, the genes implicated in RET/PTC1, are found within common fragile sites - chromosomal regions prone to DNA breakage during slight replication stress. Given that irradiated cells become more susceptible to genomic destabilization due to the accumulation of replication-stress-related double-strand breaks (DSBs), we explored whether RET and CCDC6 exhibit DNA breakage under replicative stress several days post-irradiation of thyroid cells.


METHODS
We analyzed the dynamic of DNA replication in human thyroid epithelial cells (HThy-ori-3.1) 4 days post a 5-Gy exposure using molecular DNA combing. The DNA replication schedule was evaluated through replication-timing experiments. We implemented a ChIP-qPCR assay to determine whether the RET and CCDC6 genes break following irradiation.


RESULTS
Our study indicates that replicative stress, occurring several days post-irradiation in thyroid cells, primarily causes DSBs in the RET gene. We discovered that both the RET and CCDC6 genes undergo late replication in thyroid cells. However, only RET's replication rate is notably delayed after irradiation.


CONCLUSION
The findings suggest that post-irradiation in the RET gene causes a breakage in the replication fork, which could potentially invade another genomic area, including CCDC6. As a result, this could greatly contribute to the high prevalence of chromosomal RET/PTC rearrangements seen in patients exposed to external radiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eea49490c4342b6b738c402ce79c72ea23e5f552" target='_blank'>
              A post-irradiation-induced replication stress promotes RET proto-oncogene breakage.
              </a>
            </td>
          <td>
            Fabio Hecht, Laura Valerio, Carlos Frederico Lima Gonçalves, Marylin Harinquet, Rabii Ameziane El Hassani, Denise P. Carvalho, Stephane Koundrioukoff, Jean-Charles Cadoret, Corinne Dupuy
          </td>
          <td>2024-07-01</td>
          <td>European thyroid journal</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Interphase mammalian genomes are folded in 3D with complex locus-specific patterns that impact gene regulation. CTCF (CCCTC-binding factor) is a key architectural protein that binds specific DNA sites, halts cohesin-mediated loop extrusion, and enables long-range chromatin interactions. There are hundreds of thousands of annotated CTCF-binding sites in mammalian genomes; disruptions of some result in distinct phenotypes, while others have no visible effect. Despite their importance, the determinants of which CTCF sites are necessary for genome folding and gene regulation remain unclear. Here, we update and utilize Akita, a convolutional neural network model, to extract the sequence preferences and grammar of CTCF contributing to genome folding. Our analyses of individual CTCF sites reveal four predictions: (i) only a small fraction of genomic sites are impactful, (ii) insulation strength is highly dependent on sequences flanking the core CTCF binding motif, (iii) core and flanking sequences are broadly compatible, and (iv) core and flanking nucleotides contribute largely additively to overall strength. Our analysis of collections of CTCF sites make two predictions for multi-motif grammar: (i) insulation strength depends on the number of CTCF sites within a cluster, and (ii) pattern formation is governed by the orientation and spacing of these sites, rather than any inherent specialization of the CTCF motifs themselves. In sum, we present a framework for using neural network models to probe the sequences instructing genome folding and provide a number of predictions to guide future experimental inquiries. Author Summary Mammalian genomes are spatially organized in 3D with profound consequences for all processes involving DNA. CTCF is a key genome organizer, recognizing numerous sites and creating a variety of contact patterns across the genome. Despite the importance of CTCF, the sequence determinants and grammar of how individual sites collectively instruct genome folding remain unclear. This work leverages the ability of Akita, a deep neural network, to make high-throughput predictions for genome folding after DNA sequence perturbations. Using Akita, we make several experimentally testable predictions. First, only a minority of annotated sites individually impact folding, and flanking DNA sequences greatly modulate their impact. Second, multiple sites together influence folding based on their number, orientation, and spacing. In sum, we provide a roadmap for interpreting neural networks to better understand genome folding and important considerations for the design of experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a8a450d11a02a4e12bb1863b2d5f19340a89cc" target='_blank'>
              Interpreting the CTCF-mediated sequence grammar of genome folding with AkitaV2
              </a>
            </td>
          <td>
            Paulina N. Smaruj, Fahad Kamulegeya, David R. Kelley, G. Fudenberg
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The mammalian non-homologous end joining (NHEJ) is required for V(D)J recombination as well as coping with exogenously induced DNA double strand breaks (DSBs). Initiated by the binding of KU70/KU80 (KU) dimer to DNA ends and the subsequent recruitment of the DNA- dependent protein kinase catalytic subunit (DNA-PKcs), NHEJ plays a key role in DNA repair. While there has been significant structural understandings of how KU70 participates in NHEJ, the specific function of its highly conserved C-terminal SAP domain remains elusive. In this study, we developed a novel mouse model by deleting the SAP domain but preserving the KU70 nuclear localization and its dimerization ability with KU80. We found that the KU70 SAP deletion did not affect the V(D)J recombination or animal development but significantly impaired the animals and cells in repairing exogenously induced DSBs. We further showed an inability of KU70-ΔSAP cells to retain the DNA Ligase IV (LIG4) and other NHEJ co-factors on chromatin, and a spreading pattern of DSB marker γH2AX in KU70-ΔSAP cells after DNA damage. Our findings suggest that a specific inhibition of the SAP function may offer an opportunity to modulate cell sensitivity to therapeutic DSB-inducing agents without interfering with the developmental function of KU70. KeyPoints Generation of a novel transgenic mouse line lacking the C-terminal conserved KU70-SAP domain KU70-SAP defends against exogenous DSBs, but unessential for development and V(D)J recombination KU70-SAP aids in recruiting and retaining NHEJ components, such as LIG4, to DSB sites">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5b4f65f1fff981360d615dcaa36c9fb6c898d37" target='_blank'>
              The Mammalian KU70 C-terminus SAP Domain Is Required to Repair Exogenous DNA Damage
              </a>
            </td>
          <td>
            Yuan Wang, Michael S. Czap, Hailey Kim, Huimei Lu, Jingmei Liu, Yokechen Chang, Peter J. Romanienko, Cristina Montagna, Zhiyuan Shen
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Purpose: Homologous recombination deficiency (HRD) impacts cancer treatment strategies, particularly the effectiveness of PARP inhibitors. However, the variability different HRD assays has hampered the selection of oncology patients who may benefit from these therapies. Our study aims to assess the whole genome landscape to better define HRD in a pan-cancer cohort and to contribute to harmonization of HRD detection. Methods: We employed a whole-genome sequencing WGS HRD classifier that included genome-wide features associated with HRD to analyze 580 tumor/normal paired pan-cancer samples. The HRD results were correlated retrospectively with treatment responses and were compared with commercial HRD tests in a subset of cases. Results: HRD phenotype was identified in 62 samples across various cancers including breast (19%), pancreaticobiliary (17%), gynecological (15%), prostate (8%), upper gastrointestinal (GI) (2%), and other cancers (1%). HRD cases were not confined to BRCA1/2 mutations; 24% of HRD cases were BRCA1/2 wild-type. A diverse range of HRR pathway gene alterations involved in HRD were elucidated, including biallelic mutations in FANCF, XRCC2, and FANCC, and deleterious structural variants. Comparison with results from commercial HRD assays suggests a better performance of WGS to detect HRD, based on treatment response. Conclusion: HRD is a biomarker used to determine which cancer patients would benefit from PARPi and platinum-based chemotherapy. However, a lack of harmonization of tests to determine HRD status makes it challenging to interpret their results. Our study highlights the use of comprehensive WGS analysis to predict HRD in a pan-cancer cohort, elucidates new genomic mechanisms associated with HRD, and enables an accurate identification of this phenotype, paving the way for improved outcomes in oncology care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80d347a4838be16d195accf2ef681565a4b6929" target='_blank'>
              Whole Genome Landscape Analysis of Homologous Recombination Deficiency in a Pan-Cancer Cohort
              </a>
            </td>
          <td>
            MD Majd Al Assaad, PhD Kevin Hadi, MS Max F. Levine, MD Daniela Guevara, Mph Minal Patel, MS Marvel Tranquille, BS Abigail King, BS John Otilano, MS Alissa Semaan, PhD Gunes Gundem, MS Juan S. Medina-Martínez, BS Michael Sigouros, Mph Jyothi Manohar, BS Hui-Hsuan Kuo, PhD David C. Wilkes, MD Eleni Andreopoulou, MD Eloise Chapman-Davis, MD Scott T. Tagawa, PhD Andrea Sboner, MD Allyson J. Ocean, MD Manish Shah, PhD Elli Papaemmanuil, Cora N. Sternberg, MD Kevin Holcomb, MD David M. Nanus, PhD Olivier Elemento, Juan Miguel Mosquera, J. Medina-Martinez
          </td>
          <td>2024-06-30</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bb0c25603cf78f65def294ef7f4472ac1c4808b" target='_blank'>
              Simultaneous de novo calling and phasing of genetic variants at chromosome-scale using NanoStrand-seq
              </a>
            </td>
          <td>
            Xiuzhen Bai, Zonggui Chen, Kexuan Chen, Zixin Wu, Rui Wang, Jun'e Liu, Liang Chang, Lu Wen, Fuchou Tang
          </td>
          <td>2024-07-09</td>
          <td>Cell Discovery</td>
          <td>0</td>
          <td>81</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae4759ce6db8b9a6ef9019909e5c3c340419abf2" target='_blank'>
              UBE2M forms a positive feedback loop with estrogen receptor to drive breast cancer progression and drug resistance
              </a>
            </td>
          <td>
            Xiongzhi Lin, Dongsheng Sun, Shuhan Yang, Kai Cheng, XingYi Wang, Weijia Meng, Haowei Wu, Wenlin Liu, Xiaoyu Wu, Hui Yang, Xiaojun Wang, Lisha Zhou
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0b0e09e628f8fcf9bb1ad3d19ff2d5112c171387" target='_blank'>
              Unmutated RRAS2 emerges as a key oncogene in post-partum-associated triple negative breast cancer
              </a>
            </td>
          <td>
            C. Cifuentes, Clara L. Oeste, Isabel Fernández-Pisonero, Alejandro Hortal, C. García-Macías, Jeanne Hochart, Regina Rubira, L. Horndler, Carlos Horndler, Xosé R. Bustelo, Balbino Alarcón
          </td>
          <td>2024-07-10</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="During the sexual cycle, programmed genome rearrangement (PGR) in Paramecium tetraurelia involves the non-homologous end joining (NHEJ) DNA repair pathway to eliminate specific germinal Internal Eliminated Sequences (IESs) from the newly developing somatic nucleus. In addition to the core NHEJ factors Ku70/80 and Xrcc4/Lig4, additional enzymes are required to process the 4-base 5’-protruding ends generated following DNA cleavage at IES boundaries, prior to their ligation. Here, we report that PolX (a,b,c,d), four P. tetraurelia distant orthologs of the human Polλ DNA polymerase, are involved in repair of IES excision junctions. During rearrangements, PolX-depleted cells accumulate genome-wide errors, such as unrepaired double-strand breaks, 1-nucleotide deletions and IES retention. Although all PolX paralogs can process DNA ends, two of them (PolXa&b) are induced during PGR and have acquired tight nuclear anchoring properties through their N-terminal region, which contains a predicted BRCT domain. Finally, we show that PolXa accumulates in nuclear foci together with other NHEJ proteins and the Dicer-like enzyme Dcl5, which is involved in the biogenesis of IES-specific small RNAs. We propose that these “DNA repair foci” correspond to the sites where IES concatemers, a by-product of IES excision, are ligated together to produce the precursors of iesRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8a893b216f167d132f42d3e505df3cd9a940a5" target='_blank'>
              A development-specific POLX essential for programmed genome rearrangement in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Baptiste Verron, O. Arnaiz, Coralie Zangarelli, Nathalie Mathy, M. Bétermier, J. Bischerour
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="A portion of the genetic basis for many common autoimmune disorders has been uncovered by genome-wide association studies (GWAS), but GWAS do not reveal causal variants, effector genes, or the cell types impacted by disease-associated variation. We have generated 3D genomic datasets consisting of promoter-focused Capture-C, Hi-C, ATAC-seq, and RNA-seq and integrated these data with GWAS of 16 autoimmune traits to physically map disease-associated variants to the effector genes they likely regulate in 57 human cell types. These 3D maps of gene cis-regulatory architecture are highly powered to identify the cell types most likely impacted by disease-associated genetic variation compared to 1D genomic features, and tend to implicate different effector genes than eQTL approaches in the same cell types. Most of the variants implicated by these cis-regulatory architectures are highly trait-specific, but nearly half of the target genes connected to these variants are shared across multiple autoimmune disorders in multiple cell types, suggesting a high level of genetic diversity and complexity among autoimmune diseases that nonetheless converge at the level of target gene and cell type. Substantial effector gene sharing led to the common enrichment of similar biological networks across disease and cell types. However, trait-specific pathways representing potential areas for disease-specific intervention were identified. To test this, we pharmacologically validated squalene synthase, a cholesterol biosynthetic enzyme encoded by the FDFT1 gene implicated by our approach in MS and SLE, as a novel immunomodulatory drug target controlling inflammatory cytokine production by human T cells. These data represent a comprehensive resource for basic discovery of gene cis-regulatory mechanisms, and the analyses reported reveal mechanisms by which autoimmune-associated variants act to regulate gene expression, function, and pathology across multiple, distinct tissues and cell types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bcb0132fe51fa781b26a6bbd4ec64ea24cccc76" target='_blank'>
              3D chromatin-based variant-to-gene maps across 57 human cell types reveal the cellular and genetic architecture of autoimmune disease susceptibility
              </a>
            </td>
          <td>
            K. Trang, P. Sharma, L. Cook, Z. Mount, R. M. Thomas, N. N. Kulkarni, M. Pahl, J. Pippin, C. Su, K. H. Kaestner, J. M. O'Brien, Y. Wagley, K. Hankenson, A. Jermusyk, J. W. Hoskins, L. Amundadottir, M. Xu, K. M. Brown, S. A. Anderson, W. Yang, P. Titchenell, P. Seale, B. S. Zemel, A. Chesi, N. Romberg, M. Levings, S. F. Grant, A. Wells
          </td>
          <td>2024-08-12</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic highly polyploid macronucleus is lost. The formation of a new macronucleus involves Programmed Genome Rearrangements, introducing DNA DSBs at approximately 45,000 loci. P. tetraurelia employs a Non-Homologous End Joining (NHEJ)-related mechanism for the systematic repair of these DSBs. Four genes encoding DNA polymerases of family X are present in the genome, one of which was found recently to colocalize with other proteins of NHEJ. The question arises as to how they make almost no error. Here we show that these enzymes are most similar to metazoan DNA polymerase λ and exhibit high fidelity through two different molecular mechanisms. Using X-ray structure determination of polymerase lambda mutants recapitulating sequence determinants of P. tetraurelia PolXs, we find both a local conformational change that involves exchanging partners in a crucial salt bridge in the active site upon binding of correct dNTPs, and a larger conformational change involving the closure of Loop3. This stabilizes the template DNA in the active site, only in the presence of the correct incoming dNTP. Differences with human pol λ and pol β are discussed. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a2c8e3978ae53a388430b008c681d13e31a282" target='_blank'>
              Mechanisms Ensuring Fidelity of Family X DNA Polymerases in Programmed DNA rearrangements in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Antonin Nourisson, Sophia Missoury, A. Haouz, Marc Delarue
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Spermatogenesis is a complex process that can be disrupted by genetic and epigenetic changes, potentially leading to male infertility. Recent research has rapidly increased the number of protein coding mutations causally linked to impaired spermatogenesis in humans and mice. However, the role of non-coding mutations remains largely unexplored. As a case study to evaluate the effects of non-coding mutations on spermatogenesis, we first identified an evolutionarily conserved topologically associated domain (TAD) boundary near two genes with important roles in mammalian testis function: Dmrtb1 and Lrp8. We then used CRISPR-Cas9 to generate a mouse line where 26kb of the boundary was removed including a strong and evolutionarily conserved CTCF binding site. ChIP-seq and Hi-C experiments confirmed the removal of the CTCF site and a resulting increase in the DNA-DNA interactions across the domain boundary. Mutant mice displayed significant changes in testis gene expression, abnormal testis histology, a 35% drop in the estimated efficiency of spermatogenesis and a 28% decrease in daily sperm production compared to littermate controls. Despite these quantitative changes in testis function, mutant mice show no significant changes in fertility. This suggests that non-coding deletions affecting testis gene regulation may have smaller effects on fertility compared to coding mutations of the same genes. Our results demonstrate that disruption of a TAD boundary can have a negative impact on sperm production and highlight the importance of considering non-coding mutations in the analysis of patients with male infertility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b89d71ce0de2a2d1a214068f2ba66cfb212c33d" target='_blank'>
              Deletion of an evolutionarily conserved TAD boundary compromises spermatogenesis in mice
              </a>
            </td>
          <td>
            Ana C. Lima, M. Okhovat, Alexandra M. Stendahl, Jake VanCampen, K. Nevonen, Jarod Herrera, Weiyu Li, L. Harshman, Ran Yang, Lev M. Fedorov, Katinka A. Vigh-Conrad, N. Ahituv, Donald F. Conrad, Lucia Carbone
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a3fa3038c0b72f301278947a57024534e90cee18" target='_blank'>
              DiPRO1 distinctly reprograms muscle and mesenchymal cancer cells
              </a>
            </td>
          <td>
            Jeremy Rich, Melanie Bennaroch, Laura Notel, Polina Patalakh, Julien Alberola, Fayez Issa, P. Opolon, O. Bawa, Windy Rondof, A. Marchais, Philippe Dessen, Guillaume Meurice, Morgane Le-Gall, Melanie Polrot, Karine Ser-Le Roux, K. Mamchaoui, N. Droin, H. Raslova, Pascal Maire, Birgit Geoerger, I. Pirozhkova
          </td>
          <td>2024-07-15</td>
          <td>EMBO Molecular Medicine</td>
          <td>0</td>
          <td>59</td>
        </tr>

        <tr id="Xist, a pivotal player in X chromosome inactivation (XCI), has long been perceived as a cis-acting long noncoding RNA that binds exclusively to the inactive X chromosome (Xi). However, Xist’s ability to diffuse under select circumstances has also been documented, leading us to suspect that Xist RNA may have targets and functions beyond the Xi. Here, using female mouse embryonic stem cells (ES) and mouse embryonic fibroblasts (MEF) as models, we demonstrate that Xist RNA indeed can localize beyond the Xi. However, its binding is limited to ∼100 genes in cells undergoing XCI (ES cells) and in post-XCI cells (MEFs). The target genes are diverse in function but are unified by their active chromatin status. Xist binds discretely to promoters of target genes in neighborhoods relatively depleted for Polycomb marks, contrasting with the broad, Polycomb-enriched domains reported for human XIST RNA. We find that Xist binding is associated with down-modulation of autosomal gene expression. However, unlike on the Xi, Xist binding does not lead to full silencing and also does not spread beyond the target gene. Over-expressing Xist in transgenic ES cells similarly lead to autosomal gene suppression, while deleting Xist’s Repeat B motif reduces autosomal binding and perturbs autosomal down-regulation. Furthermore, treating female ES cells with the Xist inhibitor, X1, leads to loss of autosomal suppression. Altogether, our findings reveal Xist targets ∼100 genes beyond the Xi, identify Repeat B as a crucial domain for its in-trans function in mice, and indicate that autosomal targeting can be disrupted by the X1 small molecule inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccb2a52596b1d4a0361228a895e9d27e5c7cb55" target='_blank'>
              Xist RNA binds select autosomal genes and depends on Repeat B to regulate their expression
              </a>
            </td>
          <td>
            Shengze Yao, Yesu Jeon, Barry A. Kesner, Jeannie T Lee
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Chromatin bridges are DNA strings that link anaphase poles or daughter nuclei and are associated with carcinogenesis. In response to chromatin bridges, mammalian cells delay abscission by activating the abscission checkpoint which delays completion of cytokinesis in order to prevent chromatin breakage or tetraploidization by furrow regression. We recently showed that in cytokinesis with chromatin bridges a biochemical pathway including MRN complex, ATM and Chk2 promotes INCENP localization to the midbody to delay abscission and prevent chromatin breakage. In this study we used site directed mutagenesis, RNA silencing, time lapse and confocal microscopy, FISH, BrdU labeling, TUNEL assay and differential retention assay. Here, we show that spontaneous or replication stress-induced chromatin bridges derived from catenated DNA exhibit “knots” of tangled, overtwisted DNA next to the midbody. Topoisomerase IIα (Top2a), an enzyme that relaxes supercoils and untangles catenated DNA forms irreversible Top2-DNA cleavage complexes (Top2ccs) on DNA knots. Inhibition of Top2a results in diminished localization of Rad17, MRN, ATM, Chk2 and CPC complex to the DNA bridges and induces chromatin bridge breakage. Furthermore, proteolytic degradation of Top2ccs is required for localization of Rad17 to the bridge DNA. In turn, Rad17 promotes recruitment of the MRN complex to DNA knots and downstream abscission checkpoint signaling to delay abscission and prevent chromatin bridge breakage in cytokinesis. In contrast, chromatin bridges generated by dicentric chromosomes do not exhibit DNA knots or Top2ccs next to the midbody, and fail to recruit Top2a, Rad17 and other downstream proteins and are unable to induce an abscission delay. Our results describe a novel mechanism by which the abscission checkpoint detects chromatin bridges in human cells, through generation of irreversible Top2ccs on DNA knots. Because chromosomal instability is a major cause of cancer, identifying new genes that protect genome integrity is very important for cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/34285c04c5509bb4dc8d0ef9ffcf10003592c2a2" target='_blank'>
              A Novel Mechanism for Chromatin Bridge Sensing by the Abscission Checkpoint in Human Cells
              </a>
            </td>
          <td>
            Eleni Petsalaki, Sofia Balafouti, G. Zachos
          </td>
          <td>2024-07-01</td>
          <td>JCO Global Oncology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="We present the first chromosome-level assembled genome of an individual from Karnataka, India (KIn1), using de novo assembly of 12x coverage of PacBio HiFi reads generated in-house and 30x Hi-C data from the most ethnically related individual to KIn1 from Andhra Pradesh (HG04712), publicly available under International Genome Sample Resource (IGSR). We achieved an N50 of 137 Mb and an L50 of 9—very close to the maximum achievable N50 of 147 Mb and minimum achievable L50 of 8 respectively for human genomes. A comparison of the KIn1 assembly with assemblies of four other individual genomes from a Puerto Rican (PR1), Ashkenazi (Ash1), Hans Chinese (Hans1), and Northern European (T2T) reported to date from resources like IGSR and/or Genome-in-a-Bottle (GIBR) collections shows that the longest scaffolds from KIn1 cover 80-90% of the reference chromosomes and exhibit near collinearity. We find a unique inversion in the chromosome 7 euchromatic region and other major translocations in chromosomes 17 and 20 in KIn1. We also produced a chromosome-level assembly for another individual from Punjab, Lahore (HG03492), for use in our comparison. The analysis of variants among select IGSR individuals using the KIn1 assembly as a reference reveals that the Indian Telugu from the UK (ITU) population in IGSR is ethnically closest to KIn1, followed by the Sri Lankan Tamil from the UK (STU) as representatives of South Asian ancestry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/275aa75a83406231c0d873b0f4dbb0f2fa0f847b" target='_blank'>
              The first assembled chromosome-level genome of an individual from Karnataka, India
              </a>
            </td>
          <td>
            Anisha Mhatre, Yash Chindarkar, Apoorva Ganesh, Aditi Thakare, Prakruti Mishra, Aditya Sharma, Febina Ravindran, S. Srinivasan, Bibha Choudhary
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Background Genomic variations contribute to the phenotypic diversity of individuals. A number of polymorphisms in protein-coding regions that alter drug efficacy or lead to adverse reactions have been characterized; however, noncoding regions that affect drug responses are largely overlooked, except for a limited number of well-studied enhancers. Results We conducted a quantitative assessment of cis-regulatory elements (CREs) based on transcription initiation profiling of mRNAs and noncoding RNAs, including enhancer RNAs, by using CAGE (Cap Analysis of Gene Expression). Candidate CREs identified in a hepatocellular carcinoma HepG2 cell line with stable expression of drug-responsive transcription factor pregnane X receptor (PXR) were further narrowed down by integrating data of PXR-binding sites in human primary hepatocytes and genome-wide association studies. We found more than 100-fold enrichments of the candidates to genetically associated loci with circulating levels of bilirubin and vitamin D, which implicated a link to adverse reactions of PXR ligands. We uncovered novel enhancers of UGT1A1 and TSKU through CRISPR/Cas9 knockout experiments. We identified alleles altering regulatory activities of UGT1A1 and CYP24A1enhancers by using luciferase reporter assay. Furthermore, our siRNA experiments revealed an unexpected impact of TSKU on the expression of vitamin D-metabolizing enzymes. Conclusions Our transcriptome-based assessment of CREs expanded the list of drug-inducible and PXR-mediated enhancers and super-enhancers. We identified regulatory alleles that alter drug-induced gene expressions, and discovered a novel molecular cascade associated with an adverse reaction. Our results contribute a precise understanding of the noncoding elements of the human genome underlying drug responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f12494b9c7c48c33bdacd730ac5e83409d9983a" target='_blank'>
              Drug-induced cis-regulatory elements in human hepatocytes affect molecular phenotypes associated with adverse reactions
              </a>
            </td>
          <td>
            Saki Gotoh-Saito, Ryoko Wada, Hideya Kawaji
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cancer is a heterogeneous disease that results from genetic alteration of cell cycle and proliferation controls. Identifying mutations that drive cancer, understanding cancer type specificities, and delineating how driver mutations interact with each other to establish disease is vital for identifying therapeutic vulnerabilities. Such cancer specific patterns and gene co-occurrences can be identified by studying tumor genome sequences, and networks have proven effective in uncovering relationships between sequences. We present two network-based approaches to identify driver gene patterns among tumor samples. The first approach relies on analysis using the Directed Weighted All Nearest Neighbors (DiWANN) model, which is a variant of sequence similarity network, and the second approach uses bipartite network analysis. A data reduction framework was implemented to extract the minimal relevant information for the sequence similarity network analysis, where a transformed reference sequence is generated for constructing the driver gene network. This data reduction process combined with the efficiency of the DiWANN network model, greatly lowered the computational cost (in terms of execution time and memory usage) of generating the networks enabling us to work at a much larger scale than previously possible. The DiWANN network helped us identify cancer types in which samples were more closely connected to each other suggesting they are less heterogeneous and potentially susceptible to a common drug. The bipartite network analysis provided insight into gene associations and co-occurrences. We identified genes that were broadly mutated in multiple cancer types and mutations exclusive to only a few. Additionally, weighted one-mode gene projections of the bipartite networks revealed a pattern of occurrence of driver genes in different cancers. Our study demonstrates that network-based approaches can be an effective tool in cancer genomics. The analysis identifies co-occurring and exclusive driver genes and mutations for specific cancer types, providing a better understanding of the driver genes that lead to tumor initiation and evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26d7d425039cbe06da7fb2e443336941bd0e0ae5" target='_blank'>
              Network analysis of driver genes in human cancers
              </a>
            </td>
          <td>
            S. S. Patil, Steven A. Roberts, A. Gebremedhin
          </td>
          <td>2024-07-08</td>
          <td>Frontiers in Bioinformatics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="PURPOSE
Metastatic castration-resistant prostate cancer (mCRPC) resistant to androgen receptor signaling inhibitors (ARSIs) is often lethal. Liquid biopsy biomarkers for this deadly form of disease remain under investigation, and underpinning mechanisms remain ill-understood.


EXPERIMENTAL DESIGN
We applied targeted cell-free DNA sequencing to 126 mCRPC patients from three academic cancer centers, and separately performed genome-wide cell-free DNA methylation sequencing on 43 plasma samples collected prior to the initiation of first-line ARSI treatment. To analyze the genome-wide sequencing data, we performed nucleosome-positioning and differential methylated region analysis. We additionally analyzed single-cell and bulk RNA sequencing data from 14 and 80 mCRPC patients, respectively, to develop and validate a stem-like signature, which we inferred from cell-free DNA.


RESULTS
Targeted cell-free DNA sequencing detected AR/enhancer alterations prior to first-line ARSIs which correlated with significantly worse PFS (p = 0.01; HR = 2.12) and OS (p = 0.02; HR = 2.48). Plasma methylome analysis revealed that AR/enhancer lethal mCRPC patients have significantly higher promoter-level hypomethylation than AR/enhancer wild-type mCRPC patients (p < 0.0001). Moreover, gene ontology and CytoTRACE analysis of nucleosomally more accessible transcription factors in cell-free DNA revealed enrichment for stemness-associated transcription factors in lethal mCRPC patients. The resulting stemness signature was then validated in a completely held-out cohort of 80 mCRPC patients profiled by tumor RNA sequencing.


CONCLUSIONS
We analyzed a total of 220 mCRPC patients, validated the importance of cell-free AR/enhancer alterations as a prognostic biomarker in lethal mCRPC and showed that the underlying mechanism for lethality involves reprogramming developmental states toward increased stemness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f83f524d639c8a9ee2df4427ed663a4ce58cc377" target='_blank'>
              Genomic and epigenomic analysis of plasma cell-free DNA identifies stemness features associated with worse survival in lethal prostate cancer.
              </a>
            </td>
          <td>
            P. Chauhan, I. Alahi, S. Sinha, E. Ledet, R. Mueller, Jessica Linford, A. Shiang, J. Webster, L. Greiner, Breanna Yang, Gabris Ni, Ha X Dang, Debanjan Saha, R. Babbra, W. Feng, P. Harris, F. Qaium, D. Duose, Alexander Sanchez-Espitia, Alexander D Sherry, E. Jaeger, P. Miller, S. A. Caputo, J. Orme, Fabrice Lucien, Sean S Park, Chad Tang, R. Pachynski, Oliver Sartor, C. Maher, A. Chaudhuri
          </td>
          <td>2024-08-23</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Cycling cells replicate their DNA during the S phase through a defined temporal program known as replication timing. Mutation frequencies, epigenetic chromatin states, and transcriptional activities are different for genomic regions that are replicated early and late in the S phase. Here, we found from ChIP-Seq analysis that DNA polymerase (Pol) κ is enriched in early-replicating genomic regions in HEK293T cells. In addition, by feeding cells with N 2-heptynyl-2′-deoxyguanosine followed by click chemistry–based enrichment and high-throughput sequencing, we observed elevated Pol κ activities in genomic regions that are replicated early in the S phase. On the basis of the established functions of Pol κ in accurate and efficient nucleotide insertion opposite endogenously induced N 2-modified dG lesions, our work suggests that active engagement of Pol κ may contribute to diminished mutation rates observed in early-replicating regions of the human genome, including cancer genomes. Together, our work expands the functions of Pol κ and offered a plausible mechanism underlying replication timing–dependent mutation accrual in the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417d907a79911389824959d3e561a3c8846017ac" target='_blank'>
              DNA polymerase κ participates in early S-phase DNA replication in human cells
              </a>
            </td>
          <td>
            Feng Tang, Yinan Wang, Ting Zhao, Jun Yuan, Andrew H. Kellum, Yinsheng Wang
          </td>
          <td>2024-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/358937b5c51be3f8de419eabd20cda30e0675361" target='_blank'>
              Emerging toolkits for decoding the co-occurrence of modified histones and chromatin proteins
              </a>
            </td>
          <td>
            Anne-Sophie Pépin, Robert Schneider
          </td>
          <td>2024-08-01</td>
          <td>EMBO Reports</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec914b4189eafe0cce08165c6a7c2a53bfc159e" target='_blank'>
              Unveiling FRG1’s DNA repair role in breast cancer
              </a>
            </td>
          <td>
            Shubhanjali Shubhanjali, Talina Mohapatra, Rehan Khan, Manjusha Dixit
          </td>
          <td>2024-08-21</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Neuro-Oncological Ventral Antigen 1 (NOVA1) is best known for its role in mediating an alternative splicing (AS) program in neurons, yet was first discovered as an antigen expressed in breast tumors, causing rare autoimmune reactions and paraneoplastic neurological disorders (PNDs). The PND model suggests a plausible role of the tumor antigen expression in tumor suppression, whereas it has emerged that NOVA may function as an oncogene in a variety of cancers. In addition, whether NOVA mediates AS in breast cancer remains unanswered. Here we examine the AS profiles of breast invasive carcinoma (BRCA) tumor samples and demonstrate that ectopic NOVA1 expression led to the activation of neuron-like splicing patterns in many genes, including exons targeted by NOVA in the brain. The splicing dysregulation is especially prevalent in cell periphery and cytoskeleton genes related to cell-cell communication, actin-based movement, and neuronal functions. We find that NOVA1-mediated AS is most prominent in Luminal A tumors and high NOVA1 expression in this subtype is associated with poorer prognosis. Our results suggest that ectopic NOVA1 in tumors has regulatory activity affecting pathways with high relevance to tumor progression and that this might be a more general mechanism for PND antigens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e50ff50fd94e0098cbe1b6b46456df93e0c5100" target='_blank'>
              Oncogenic NOVA1 expression dysregulates alternative splicing in breast cancer
              </a>
            </td>
          <td>
            Daniel F. Moakley, Chaolin Zhang
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Cancer stemness plays an important role in cancer initiation and progression, and is the major cause of tumor invasion, metastasis, recurrence, and poor prognosis. Non-coding RNAs (ncRNAs) are a class of RNA transcripts that generally cannot encode proteins and have been demonstrated to play a critical role in regulating cancer stemness. Here, we developed the ncStem database to record manually curated and predicted ncRNAs associated with cancer stemness. In total, ncStem contains 645 experimentally verified entries, including 159 long non-coding RNAs (lncRNAs), 254 microRNAs (miRNAs), 39 circular RNAs (circRNAs), and 5 other ncRNAs. The detailed information of each entry includes the ncRNA name, ncRNA identifier, disease, reference, expression direction, tissue, species, and so on. In addition, ncStem also provides computationally predicted cancer stemness-associated ncRNAs for 33 TCGA cancers, which were prioritized using the random walk with restart (RWR) algorithm based on regulatory and co-expression networks. The total predicted cancer stemness-associated ncRNAs included 11 132 lncRNAs and 972 miRNAs. Moreover, ncStem provides tools for functional enrichment analysis, survival analysis, and cell location interrogation for cancer stemness-associated ncRNAs. In summary, ncStem provides a platform to retrieve cancer stemness-associated ncRNAs, which may facilitate research on cancer stemness and offer potential targets for cancer treatment. Database URL: http://www.nidmarker-db.cn/ncStem/index.html.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb97313e3402ad277fe895b46bce91bdb6c21f55" target='_blank'>
              ncStem: a comprehensive resource of curated and predicted ncRNAs in cancer stemness
              </a>
            </td>
          <td>
            Hui Liu, Nan Zhang, Yijie Jia, Jun Wang, Aokun Ye, Siru Yang, Honghan Zhou, Yingli Lv, Chaohan Xu, Shuyuan Wang
          </td>
          <td>2024-08-13</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Each piece of cell-free DNA (cfDNA) has a length determined by the exact metabolic conditions in the cell it belonged to at the time of cell death. The changes in cellular regulation leading to a variety of patterns, which are based on the different number of fragments with lengths up to several hundred base pairs at each of the almost three billion genomic positions, allow for the detection of disease and also the precise identification of the tissue of their origin. A Kullback-Leibler (KL) divergence computation identifies different fragment lengths and areas of the human genome, depending on the stage, for which disease samples, starting from pre-clinical disease stages, diverge from healthy donor samples. We provide examples of genes related to colorectal cancer (CRC), which our algorithm detected to belong to divergent genomic bins. The staging of CRC can be viewed as a Markov Chain and that provides a framework for studying disease progression and the types of epigenetic changes occurring longitudinally at each stage, which might aid the correct classification of a new hospital sample. In a new look to treat such data as grayscale value images, pattern recognition using artificial intelligence (AI) could be one approach to classification. In CRC, Stage I disease does not, for the most part, shed any tumor circulation, making detection difficult for established machine learning (ML) methods. This leads to the deduction that early detection, where we can only rely on changes in the metabolic patterns, can be accomplished when the information is considered in its entirety, for example by applying computer vision methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769c868029bf18fe644fcae9a2141c21f2b5a763" target='_blank'>
              Channel Capacity of Genome-Wide Cell-Free DNA Fragment Length Distribution in Colorectal Cancer
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-07-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="In humans, the HS1.2 enhancer in the Ig heavy-chain locus is modular, with length polymorphism. Previous studies have shown the following features for this variation: (i) strong population structuring; (ii) association with autoimmune diseases; and (iii) association with developmental changes in Ig expression. The HS1.2 region could then be considered as a contributor to inter-individual diversity in humoral response in adaptive immunity. We experimentally determined the HS1.2-length class genotype in 72 of the 1000 Genomes CEU cell lines and assigned the HS1.2 alleles to haplotypes defined by 18 landmark SNPs. We also sequenced the variable portion and ~200 bp of the flanking DNA of 34 HS1.2 alleles. Furthermore, we computationally explored the ability of different allelic arrangements to bind transcription factors. Non-random association between HS1.2 and Gm allotypes in the European population clearly emerged. We show a wealth of variation in the modular composition of HS1.2, with five SNPs further contributing to diversity. Longer alleles offer more potential sites for binding but, for same-length alleles, SNP variation creates/destroys potential binding sites. Altogether, the arrangements of modules and SNP alleles both inside and outside HS1.2 denote an organization of diversity far from randomness. In the context of the strong divergence of human populations for this genomic region and the reported disease associations, our results suggest that selective forces shaped the pattern of its diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98510c8b2eb332dede446f7bc9c0188abf831f75" target='_blank'>
              Variation of the 3’RR1 HS1.2 Enhancer and Its Genomic Context
              </a>
            </td>
          <td>
            C. Jodice, P. Malaspina, B. Ciminelli, Cristina Martinez-Labarga, M. Biancolella, Giuseppe Novelli, Andrea Novelletto
          </td>
          <td>2024-06-29</td>
          <td>Genes</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f891979797e3688f35366887366668ec188a50ae" target='_blank'>
              Microsatellite instability at U2AF-binding polypyrimidic tract sites perturbs alternative splicing during colorectal cancer initiation
              </a>
            </td>
          <td>
            V. Jonchère, Hugo Montémont, Enora Le Scanf, A. Siret, Q. Letourneur, Emmanuel Tubacher, Christophe Battail, Assane Fall, Karim Labreche, Victor Renault, T. Ratovomanana, O. Buhard, A. Jolly, Philippe Le Rouzic, Cody Feys, Emmanuelle Despras, Habib Zouali, R. Nicolle, Pascale Cervera, M. Svrcek, P. Bourgoin, Hélène Blanché, A. Boland, Jérémie Lefèvre, Y. Parc, M. Touat, F. Bielle, D. Arzur, Gwennina Cueff, C. Le Jossic-Corcos, Gaël Quéré, Gwendal Dujardin, Marc Blondel, Cédric Le Maréchal, R. Cohen, Thierry André, Florence Coulet, P. de la Grange, A. de Reyniès, Jean-françois Flejou, Florence Renaud, A. Alentorn, Laurent Corcos, J. Deleuze, A. Collura, A. Duval
          </td>
          <td>2024-08-06</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b0e84c70be39b55c5ab372a108ddda6ab970765" target='_blank'>
              Polθ: emerging synthetic lethal partner in homologous recombination-deficient tumors.
              </a>
            </td>
          <td>
            T. B. Bazan Russo, Clarissa Mujacic, Emilia Di Giovanni, Maria Concetta Vitale, Carla Ferrante Bannera, U. Randazzo, Silvia Contino, M. Bono, V. Gristina, A. Galvano, A. Perez, G. Badalamenti, A. Russo, V. Bazan, L. Incorvaia
          </td>
          <td>2024-08-09</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facf4dda707b67b3b56d646f95496db5b6cc7df8" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N. Anderson
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) is increasingly studied for its diverse applications in non-invasive detection. Non-randomly cleaved by nucleases and released into the bloodstream, cfDNA exhibits a variety of intrinsic fragmentation patterns indicative of cell status. Particularly, these fragmentation patterns have recently been demonstrated to be effective in predicting cancer and its tissue-of-origin, owing to increased variation of fragmentation features observed in tumor patients. However, there remains a lack of detailed exploration of altered cfDNA fragmentation profiles in tumors, which consist of a mixture of both non-tumor cfDNA and circulating tumor DNA (ctDNA). Hence, we leveraged the human tumor cell line-derived xenograft (CDX) mouse model, where different tumor cell lines were implanted into different anatomical sites, to isolate pure ctDNA and separately investigate the fragment properties of CDX-induced cfDNA and ctDNA. We found an enrichment of short cfDNA fragments in both CDX-induced cfDNA and ctDNA compared to normal plasma cfDNA, with more elevated short fragments in ctDNA. Moreover, the CDX-induced cfDNA fragmentation features distinguished between CDX models of different tumor cell lines, while the ctDNA fragmentation features conversely discriminate between CDX models of different anatomical sites. The results suggested that both non-tumor cfDNA and ctDNA contribute to the increased variation observed in tumors, and that cfDNA fragmentation may be highly variable and susceptible to regulations by both original cells and cells within the local niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6138f9d167b66612f618b5e26b115af024b7013f" target='_blank'>
              Dissecting cell-free DNA fragmentation variation in tumors using cell line-derived xenograft mouse
              </a>
            </td>
          <td>
            Ruiqing Fu, He Amy Su, Yafei Tian, Hongyan Chen, Daru Lu
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b40bf07806bd0a97226f528a2d387e1d1cd79723" target='_blank'>
              Enhancers associated with unstable RNAs are rare in plants
              </a>
            </td>
          <td>
            Bayley R McDonald, Colette L. Picard, Ian M. Brabb, Marina I. Savenkova, Robert J. Schmitz, Steve Jacobsen, Sascha H. C. Duttke
          </td>
          <td>2024-07-30</td>
          <td>Nature Plants</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa8cd8dcee3abb97837fb9cb766c024fc2216aaa" target='_blank'>
              Canine hyper-sociability structural variants associated with altered three-dimensional chromatin state
              </a>
            </td>
          <td>
            Dhriti Tandon, E. Kubinyi, Sára Sándor, Hannah Faughnan, Ádám Miklósi, B. Vonholdt
          </td>
          <td>2024-08-07</td>
          <td>BMC Genomics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Transcription and splicing errors in cancer cells generate erroneous transcripts. Since erroneous transcripts are degraded by the nonsense-mediated mRNA decay (NMD) pathway, whether they are junk or could be functional has been overlooked and understudied. We addressed this question by first performing a pan-cancer analysis and identified substantial erroneous noncoding transcripts (ENT) in cancers. Given that RNA/DNA binding domains (DBD) were predicted in ENTs, we deleted predicted DBDs in multiple ENTs in multiple cell lines, with RNA-sequencing and cell experiments before and after DBD deletion. DBD deletion caused significantly changed expression of ENTs’ target genes (whose promoter regions contain ENTs’ DNA binding sites, DBS) and changed cell migration and proliferation ability, indicating that many ENTs can transcriptionally regulate genes. Tightly coupled data analysis and experiments reveal that ENTs’ functions are highly cancer- and cellular-context specific, making ENTs a new class of safe and specific targets for noncoding RNA-based cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e45af8d1c765a560ff555893aa2ed200e47f3c" target='_blank'>
              Many erroneous noncoding transcripts in cancer cells can highly specifically regulate cancer-related genes and pathways
              </a>
            </td>
          <td>
            Sha He, Wei Xiong, Jianping Huo, Jie Lin, Jianmin Li, Hao Zhu
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799e1ddffbac6e1b1f9a23cb6190cda5aacba309" target='_blank'>
              Spatial multiomics reveals a subpopulation of fibroblasts associated with cancer stemness in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si Jing, Dan Liu, Na Feng, Hui Dong, He-Qi Wang, Xi Yan, Xu-Feng Chen, Mincheng Qu, Ping Lin, Bin Yi, Feiling Feng, Lei Chen, Hong-Yang Wang, Hong Li, Yufei He
          </td>
          <td>2024-08-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, Bert van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Non-SMC condensin I complex subunit D2 (NCAPD2) is a newly identified oncogene; however, the specific biological function and molecular mechanism of NCAPD2 in liver cancer progression remain unknown. In the present study, the aberrant expression of NCAPD2 in liver cancer was investigated using public tumor databases, including TNMplot, The Cancer Genome Atlas and the International Cancer Genome Consortium based on bioinformatics analyses, and it was validated using a clinical cohort. It was revealed that NCAPD2 was significantly upregulated in liver cancer tissues compared with in control liver tissues, and NCAPD2 served as an independent prognostic factor and predicted poor prognosis in liver cancer. In addition, the expression of NCAPD2 was positively correlated with the percentage of Ki67+ cells. Finally, single-cell sequencing data, gene-set enrichment analyses and in vitro investigations, including cell proliferation assay, Transwell assay, wound healing assay, cell cycle experiments, cell apoptosis assay and western blotting, were carried out in human liver cancer cell lines to assess the biological mechanisms of NCAPD2 in patients with liver cancer. The results revealed that the upregulation of NCAPD2 enhanced tumor cell proliferation, invasion and cell cycle progression at the G2/M-phase transition, and inhibited apoptosis in liver cancer cells. Furthermore, NCAPD2 overexpression was closely associated with the phosphatidylinositol 3-kinase (PI3K)-Akt-mammalian target of rapamycin (mTOR)/c-Myc signaling pathway and epithelial-mesenchymal transition (EMT) progression in HepG2 and Huh7 cells. In addition, upregulated NCAPD2 was shown to have adverse effects on overall survival and disease-specific survival in liver cancer. In conclusion, the overexpression of NCAPD2 was shown to lead to cell cycle progression at the G2/M-phase transition, activation of the PI3K-Akt-mTOR/c-Myc signaling pathway and EMT progression in human liver cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6873a90e35d5a8f62be3a51b8f0dd6c3ab4c961" target='_blank'>
              NCAPD2 augments the tumorigenesis and progression of human liver cancer via the PI3K-Akt-mTOR signaling pathway
              </a>
            </td>
          <td>
            Jiangxue Gu, Ke Huang, Wei-Lin Zhao, Xiao-Ming Zheng, Yu-Qin Wu, Shirong Yan, Yu-Gang Huang, Pei Hu
          </td>
          <td>2024-07-31</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9e6bae3255c905d58e9f2d6fd8dca01970a1953f" target='_blank'>
              Chromatin conformation and histone modification profiling across human kidney anatomic regions
              </a>
            </td>
          <td>
            Haikuo Li, Dian Li, Benjamin D. Humphreys
          </td>
          <td>2024-07-18</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a190a83f30593fab67bd2e0b946b6e5dcca9e7" target='_blank'>
              STAU1-mediated CNBP mRNA degradation by LINC00665 alters stem cell characteristics in ovarian cancer
              </a>
            </td>
          <td>
            Xiaofang Liu, Yang Chen, Ying Li, Jinling Bai, Zhi Zeng, Min Wang, Yaodong Dong, Yingying Zhou
          </td>
          <td>2024-07-30</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="TBC1D3 is a primate-specific gene family that has expanded in the human lineage and has been implicated in neuronal progenitor proliferation and expansion of the frontal cortex. The gene family and its expression have been challenging to investigate because it is embedded in high-identity and highly variable segmental duplications. We sequenced and assembled the gene family using long-read sequencing data from 34 humans and 11 non-human primate species. Our analysis shows that this particular gene family has independently duplicated in at least five primate lineages, and the duplicated loci are enriched at sites of large-scale chromosomal rearrangements on Chromosome 17. We find that all human copy number variation maps to two distinct clusters located at Chromosome 17q12 and that humans are highly structurally variable at this locus, differing by as many as 20 copies and ~1 Mbp in length depending on haplotypes. We also show evidence of positive selection, as well as a significant change in the predicted human TBC1D3 protein sequence. Lastly, we find that, despite multiple duplications, human TBC1D3 expression is limited to a subset of copies and, most notably, from a single paralog group: TBC1D3-CDKL These observations may help explain why a gene potentially important in cortical development can be so variable in the human population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872db936ea87de98bc9c479ae7fb31da39d2f13a" target='_blank'>
              Independent expansion, selection and hypervariability of the TBC1D3 gene family in humans.
              </a>
            </td>
          <td>
            Xavi Guitart, David Porubsky, DongAhn Yoo, Max L. Dougherty, Philip C. Dishuck, Katherine M. Munson, Alexandra P. Lewis, Kendra Hoekzema, Jordan Knuth, Stephen Chang, Tomi Pastinen, Evan E. Eichler
          </td>
          <td>2024-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="MYC has been implicated in the pathogenesis of a wide range of human tumors and has been described for many years as a transcription factor that regulates genes with pleiotropic functions to promote tumorigenic growth. However, despite extensive efforts to identify specific target genes of MYC that alone could be responsible for promoting tumorigenesis, the field is yet to reach a consensus whether this is the crucial function of MYC. Recent work shifts the view on MYC's function from being a gene-specific transcription factor to an essential stress resilience factor. In highly proliferating cells, MYC preserves cell integrity by promoting DNA repair at core promoters, protecting stalled replication forks, and/or preventing transcription-replication conflicts. Furthermore, an increasing body of evidence demonstrates that MYC not only promotes tumorigenesis by driving cell-autonomous growth, but also enables tumors to evade the host's immune system. In this review, we summarize our current understanding of how MYC impairs antitumor immunity and why this function is evolutionarily hard-wired to the biology of the MYC protein family. We show why the cell-autonomous and immune evasive functions of MYC are mutually dependent and discuss ways to target MYC proteins in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877f0298df11cd9063e84174a42a5eb6afca2c71" target='_blank'>
              Immune evasion: An imperative and consequence of MYC deregulation.
              </a>
            </td>
          <td>
            Bastian Krenz, Jongkuen Lee, Toshitha Kannan, Martin Eilers
          </td>
          <td>2024-07-02</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7fb32bc7591905c2f53c6bd62cdcf10e8e244e19" target='_blank'>
              Chromosomal copy number and mutational status are required to authenticate ovarian cancer cell lines as appropriate cell models
              </a>
            </td>
          <td>
            B. Stordal, A. Farrelly, Bryan T. Hennessy
          </td>
          <td>2024-06-28</td>
          <td>Molecular Biology Reports</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/63c7fbf4c7ba6436f534afc2646382ace9105446" target='_blank'>
              Genomic characterization of AML with aberrations of chromosome 7: a multinational cohort of 519 patients
              </a>
            </td>
          <td>
            Adriane Halik, Marlon Tilgner, Patricia Silva, Natalia Estrada, Robert Altwasser, E. Jahn, M. Heuser, Hsin-An Hou, M. Pratcorona, R. Hills, K. Metzeler, L. Fenwarth, A. Dolnik, Christine Terré, Klara Kopp, Olga Blau, Martin Szyska, F. Christen, J. Krönke, Loic Vasseur, Bob Löwenberg, Jordi Esteve, Peter J. Valk, M. Duchmann, W. Chou, D. Linch, H. Döhner, R. Gale, Konstanze Döhner, Lars Bullinger, Kenichi Yoshida, Frederik Damm
          </td>
          <td>2024-08-19</td>
          <td>Journal of Hematology & Oncology</td>
          <td>0</td>
          <td>87</td>
        </tr>

        <tr id="The genomic architecture of 22q11.2 Deletion Syndrome (22q11.2DS) has focused on analysis of white genomes. However, Black individuals appear to have a lower prevalence of 22q11.2DS compared to whites. To improve the understanding of different populations in relation to 22q11.2DS, optical mapping data from 106 genomes across various Black and white genomes were used to determine the organization of 22q11.2 genomic structures. This revealed extensive variability between the groups regarding copy number and orientation changes of the elements comprising the 22q11.2 low copy repeats (LCR22s). Several novel CNVs and whole haplotype configurations, private and of different prevalence to each group were detected. The diversity of CNVs within Black genomes compared to white genomes was especially striking. To determine the impact of this variability, Black families with de novo 22q11.2DS probands were compared to white families. The highly variable configurations of Black and white haplotypes led to several unique non-allelic homologous recombination (NAHR) scenarios with recombinations at different loci. In particular, Black families had unique recombinations yet to be observed. Thus, the unique and highly variable haplotype configurations of LCR22s in Black individuals may play a role in their decreased incidence of 22q11.2DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/654ffdc07a99700bb41c69f30252e8dc1b2d3481" target='_blank'>
              Optically Mapped Black Genomes: Distinct Structures and 22q11.2 Deletion Syndrome Mechanisms
              </a>
            </td>
          <td>
            Steven Pastor, Oanh Tran, Ryan Lapointe, Arnold Z. Olali, Douglas C. Wallace, Bernice Morrow, E. Zackai, Donna M. McDonald-McGinn, B. Emanuel
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/153ab2ed142f22e69713eea228d0914f5b69386b" target='_blank'>
              Telomere-to-telomere genome assembly of sorghum
              </a>
            </td>
          <td>
            Meng Li, Chunhai Chen, Haigang Wang, Huibin Qin, Sen Hou, Xukui Yang, Jianbo Jian, Peng Gao, Minxuan Liu, Z. Mu
          </td>
          <td>2024-08-02</td>
          <td>Scientific Data</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="New high-quality human genome assemblies derived from lymphoblastoid cell lines (LCLs) provide reference genomes and pangenomes for genomics studies. However, the characteristics of LCLs pose technical challenges to profiling immunoglobulin (IG) genes. IG loci in LCLs contain a mixture of germline and somatically recombined haplotypes, making them difficult to genotype or assemble accurately. To address these challenges, we introduce IGLoo, a software tool that implements novel methods for analyzing sequence data and genome assemblies derived from LCLs. IGLoo characterizes somatic V(D)J recombination events in the sequence data and identifies the breakpoints and missing IG genes in the LCL-based assemblies. Furthermore, IGLoo implements a novel reassembly framework to improve germline assembly quality by integrating information about somatic events and population structural variantions in the IG loci. We applied IGLoo to study the assemblies from the Human Pangenome Reference Consortium, providing new insights into the mechanisms, gene usage, and patterns of V(D)J recombination, causes of assembly fragmentation in the IG heavy chain (IGH) locus, and improved representation of the IGH assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8f48cf5eb22832827a1cb138fb7f409be2963b7" target='_blank'>
              IGLoo: Profiling the Immunoglobulin Heavy chain locus in Lymphoblastoid Cell Lines with PacBio High-Fidelity Sequencing reads
              </a>
            </td>
          <td>
            Mao-Jan Lin, Ben Langmead, Yana Safonova
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The mechanisms that work alongside BRAFV600E oncogene in melanoma development, in addition to ultraviolet (UV) radiation (UVR), are of great interest. Analysis of human melanoma tumors [data from The Cancer Genome Atlas (TCGA)] revealed that 50% or more of the samples expressed no or low amounts of serine/threonine protein kinase STK11 (also known as LKB1) protein. Here, we report that, in a mouse model, concomitant neonatal BrafV600E activation and Lkb1 tumor suppressor ablation in melanocytes led to full melanoma development. A single postnatal dose of UVB radiation had no effect on melanoma onset in Lkb1-depleted mice compared with BrafV600E-irradiated mice, but increased tumor multiplicity. In concordance with these findings and previous reports, Lkb1-null irradiated mice exhibited deficient DNA damage repair (DDR). Histologically, tumors lacking Lkb1 were enriched in neural-like tumor morphology. Genetic profiling and gene set enrichment analyses of tumor sample mutated genes indicated that loss of Lkb1 promoted the selection of altered genes associated with neural differentiation processes. Thus, these results suggest that the loss of Lkb1 cooperates with BrafV600E and UVR, impairing the DDR and increasing melanoma multiplicity and neural-like dedifferentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/468890566d37157a91439461401c20d4b0665a15" target='_blank'>
              Loss of Lkb1 cooperates with BrafV600E and ultraviolet radiation, increasing melanoma multiplicity and neural-like dedifferentiation.
              </a>
            </td>
          <td>
            K. McGrail, Elena González-Sánchez, Paula Granado-Martínez, Roberto Orsenigo, Yuxin Ding, Berta Ferrer, Javier Hernández-Losa, Iván Ortega, Juan Martín-Caballero, Eva Muñoz-Couselo, Vicente García-Patos, Juan A. Recio
          </td>
          <td>2024-08-08</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Polyploidization plays a crucial role in plant evolution and is becoming increasingly important in breeding. Structural variations and epigenomic repatterning have been observed in synthetic polyploidizations. However, the mechanisms underlying the occurrence and their effects on gene expression and phenotype remain unknown. Here, we investigated genome-wide large deletion/duplication regions (DelDups) and genomic methylation dynamics in leaf organs of progeny from the first eight generations of synthetic tetraploids derived from Chinese cabbage (Brassica rapa L. ssp. pekinensis) and cabbage (Brassica oleracea L. var. capitata). One- or two-copy DelDups, with a mean size of 5.70 Mb (400 kb - 65.85 Mb), occurred from the first generation of selfing and thereafter. The duplication of a fragment in one subgenome consistently coincided with the deletion of its syntenic fragment in the other subgenome, and vice versa, indicating that these DelDups were generated by homoeologous exchanges (HEs). Interestingly, the larger the genomic syntenic region, the higher the frequency of DelDups, further suggesting that the pairing of large homoeologous fragments is crucial for HEs. Moreover, we found that the active transcription of continuously distributed genes in local regions is positively associated with the occurrence of HE breakpoints. In addition, the expression of genes within DelDups exhibited a dosage effect, and plants with extra parental genomic fragments generally displayed phenotypes biased towards the corresponding parent. Genome-wide methylation fluctuated remarkably, which did not clearly affect gene expression on a large scale. Our findings provide insights into the early evolution of polyploid genomes, offering valuable knowledge for polyploidization-based breeding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc22507c27b064911f132b6bdaa4330162e63d6f" target='_blank'>
              Regional Active Transcription Associates with Homoeologous Exchange Breakpoints in Synthetic Brassica Tetraploids.
              </a>
            </td>
          <td>
            Chengcheng Cai, Yinqing Yang, Lei Zhang, Yinan Cui, Jian Wu, Jianli Liang, Xing Li, Lingkui Zhang, Xin Zhang, Yiyue Zhang, Zhongwei Guo, Shumin Chen, Kang Zhang, M. Freeling, Xiaowu Wang, Feng Cheng
          </td>
          <td>2024-08-20</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>73</td>
        </tr>

        <tr id="Abstract CCCTC-binding factor (CTCF) is an insulator protein that binds to a highly conserved DNA motif and facilitates regulation of three-dimensional (3D) nuclear architecture and transcription. CTCF binding sites (CTCF-BSs) reside in non-coding DNA and are frequently mutated in cancer. Our previous study identified a small subclass of CTCF-BSs that are resistant to CTCF knock down, termed persistent CTCF binding sites (P-CTCF-BSs). P-CTCF-BSs show high binding conservation and potentially regulate cell-type constitutive 3D chromatin architecture. Here, using ICGC sequencing data we made the striking observation that P-CTCF-BSs display a highly elevated mutation rate in breast and prostate cancer when compared to all CTCF-BSs. To address whether P-CTCF-BS mutations are also enriched in other cell-types, we developed CTCF-INSITE—a tool utilising machine learning to predict persistence based on genetic and epigenetic features of experimentally-determined P-CTCF-BSs. Notably, predicted P-CTCF-BSs also show a significantly elevated mutational burden in all 12 cancer-types tested. Enrichment was even stronger for P-CTCF-BS mutations with predicted functional impact to CTCF binding and chromatin looping. Using in vitro binding assays we validated that P-CTCF-BS cancer mutations, predicted to be disruptive, indeed reduced CTCF binding. Together this study reveals a new subclass of cancer specific CTCF-BS DNA mutations and provides insights into their importance in genome organization in a pan-cancer setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994d6f59097fc218409b42c901b1ae72776752f9" target='_blank'>
              Machine learning enables pan-cancer identification of mutational hotspots at persistent CTCF binding sites
              </a>
            </td>
          <td>
            Wenhan Chen, Y. Zeng, Joanna Achinger-Kawecka, Elyssa Campbell, Alicia K Jones, A. Stewart, Amanda Khoury, Susan J Clark
          </td>
          <td>2024-07-02</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Combined CDK4/6 inhibitor (CDK4/6i) and endocrine therapy significantly improves outcome for patients with estrogen receptor-positive (ER+) metastatic breast cancer, but drug resistance and thus disease progression inevitably occur. Herein, we aimed to identify genomic alterations associated with combined CDK4/6i and endocrine therapy resistance, and follow the levels of specific mutations in longitudinal circulating tumor DNA (ctDNA) for early detection of progression. From a cohort of 86 patients with ER+ metastatic breast cancer we performed whole exome sequencing or targeted sequencing of paired tumor (N = 8) or blood samples (N = 5) obtained before initiation of combined CDK4/6i and endocrine therapy and at disease progression. Mutations in oncogenic genes at progression were rare, while amplifications of growth-regulating genes were more frequent. The most frequently acquired alterations observed were PIK3CA and TP53 mutations and PDK1 amplification. Longitudinal ctDNA dynamics of mutant PIK3CA or private mutations revealed increased mutation levels at progression in 8 of 10 patients (80%). Impressively, rising levels of PIK3CA-mutated ctDNA were detected 4-17 months before imaging. Our data add to the growing evidence supporting longitudinal ctDNA analysis for real-time monitoring of CDK4/6i response and early detection of progression in advanced breast cancer. Further, our analysis suggests that amplification of growth-related genes may contribute to combined CDK4/6i and endocrine therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4f8f900b692158e74287adf0a693a5a4d08296" target='_blank'>
              Genomic alterations associated with resistance and circulating tumor DNA dynamics for early detection of progression on CDK4/6 inhibitor in advanced breast cancer.
              </a>
            </td>
          <td>
            Charlotte K Kindt, Carla L. Alves, S. Ehmsen, Amalie Kragh, T. Reinert, Marianne Vogsen, A. R. Kodahl, Jeanette D Rønlev, Dilan Ardik, Anna L Sørensen, Kirstine Evald, Mia L Clemmensen, J. Staaf, Henrik J Ditzel
          </td>
          <td>2024-08-11</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb5a77337f38c8a116c6d75b10e0eda38d81a91a" target='_blank'>
              ΔNp63 bookmarks and creates an accessible epigenetic environment for TGFβ-induced cancer cell stemness and invasiveness
              </a>
            </td>
          <td>
            Eleftheria Vasilaki, Yu Bai, Mohamad Moustafa Ali, Anders Sundqvist, A. Moustakas, C. Heldin
          </td>
          <td>2024-08-23</td>
          <td>Cell Communication and Signaling : CCS</td>
          <td>0</td>
          <td>154</td>
        </tr>

        <tr id="Most DNA double-strand breaks (DSBs) are harmful to genome integrity. However, some forms of DSBs are essential to biological processes, such as meiotic recombination and V(D)J recombination. DSBs are also required for programmed DNA elimination (PDE) in ciliates and nematodes. In nematodes, the DSBs are healed with telomere addition. While telomere addition sites have been well characterized, little is known regarding the DSBs that fragment nematode chromosomes. Here, we used embryos from the human and pig parasitic nematode Ascaris to characterize the DSBs. Using END-seq, we demonstrate that DSBs are introduced before mitosis, followed by extensive end resection. The resection profile is unique for each break site, and the resection generates 3'-overhangs before the addition of neotelomeres. Interestingly, telomere healing occurs much more frequently on retained DSB ends than on eliminated ends. This biased repair of the DSB ends may be due to the sequestration of the eliminated DNA into micronuclei, preventing neotelomere formation at their ends. Additional DNA breaks occur within the eliminated DNA in both Ascaris and Parascaris, ensuring chromosomal breakage and providing a fail-safe mechanism for PDE. Overall, our data indicate that telomere healing of DSBs is specific to the break sites responsible for nematode PDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/104afc2d31453d804bc1298637d05cb0465a32ee" target='_blank'>
              End resection and telomere healing of DNA double-strand breaks during nematode programmed DNA elimination.
              </a>
            </td>
          <td>
            Brandon Estrem, Richard E Davis, Jianbin Wang
          </td>
          <td>2024-07-02</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8e9f04f74a6159b9e49f1e14a0705e366ea67c" target='_blank'>
              Epigenetic alterations affecting hematopoietic regulatory networks as drivers of mixed myeloid/lymphoid leukemia
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Stanley van Herk, Margit Nuetzel, Aniko Sijs-Szabo, Noelia Díaz, Katherine Kelly, Claudia A. J. Erpelinck-Verschueren, Lucia Schwarzfischer-Pfeilschifter, Hanna Stanewsky, Ute Ackermann, Dagmar Glatz, Johanna Raithel, A. Fischer, Sandra Pohl, Anita Rijneveld, Juan M. Vaquerizas, Christian Thiede, C. Plass, Bas J. Wouters, Ruud Delwel, Michael Rehli, C. Gebhard
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="With age, hematopoietic stem cells can acquire somatic mutations in leukemogenic genes that confer a proliferative advantage in a phenomenon termed “clonal hematopoiesis of indeterminate potential” (CHIP). How these mutations confer a proliferative advantage and result in increased risk for numerous age-related diseases remains poorly understood. We conducted a multiracial meta-analysis of epigenome-wide association studies (EWAS) of CHIP and its subtypes in four cohorts (N=8196) to elucidate the molecular mechanisms underlying CHIP and illuminate how these changes influence cardiovascular disease risk. The EWAS findings were functionally validated using human hematopoietic stem cell (HSC) models of CHIP. A total of 9615 CpGs were associated with any CHIP, 5990 with DNMT3A CHIP, 5633 with TET2 CHIP, and 6078 with ASXL1 CHIP (P <1×10−7). CpGs associated with CHIP subtypes overlapped moderately, and the genome-wide DNA methylation directions of effect were opposite for TET2 and DNMT3A CHIP, consistent with their opposing effects on global DNA methylation. There was high directional concordance between the CpGs shared from the meta-EWAS and human edited CHIP HSCs. Expression quantitative trait methylation analysis further identified transcriptomic changes associated with CHIP-associated CpGs. Causal inference analyses revealed 261 CHIP-associated CpGs associated with cardiovascular traits and all-cause mortality (FDR adjusted p-value <0.05). Taken together, our study sheds light on the epigenetic changes impacted by CHIP and their associations with age-related disease outcomes. The novel genes and pathways linked to the epigenetic features of CHIP may serve as therapeutic targets for preventing or treating CHIP-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb071f9470db1b0d43e08fc0664a4d81582bdd97" target='_blank'>
              Epigenome-wide DNA Methylation Association Study of CHIP Provides Insight into Perturbed Gene Regulation
              </a>
            </td>
          <td>
            Daniel Levy, Sara Kirmani, T. Huan, Joseph Van Amburg, R. Joehanes, M. Uddin, N. Nguyen, Bing Yu, Jennifer A. Brody, M. Fornage, Jan Bressler, N. Sotoodehnia, David Ong, Fabio Puddu, James Floyd, Christie M. Ballantyne, B. Psaty, L. Raffield, Pradeep Natarajan, Karen Conneely, April P. Carson, Leslie Lange, Kendra Ferrier, N. HEARD-COSTA, Joanne M Murabito, A. Bick
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="The transcription factor STAT3 serves as an immunoregulator by playing a crucial role in cytokine receptor signaling. However, in various cancer cell types, STAT3 is involved in the molecular mechanisms of oncogenesis. Specifically, in glioblastoma, the STAT3 immunoregulator has been linked to resistance to temozolomide, the most commonly used chemical agent for treating this type of cancer. Furthermore, literature suggests that activation of this oncogene in glioblastoma cells can significantly impact the tolerogenic tumor microenvironment, weakening the antitumor immune response and contributing to the aggressive course of the disease. Therefore, suppressing STAT3 may not only affect cell growth and resistance to chemotherapy but also enhance the immune response by improving the tumor microenvironment.The development of sequencing technologies has revealed that most of the transcribed material in the cell is noncoding. Long non-coding RNAs are gaining popularity in the study of oncogenesis due to their functional role in the development of various diseases, including oncology. A subtype of long non-coding RNAs transcribed from enhancer elements, known as enhancer RNAs, has garnered attention due to their high specificity in various cells and tissues. Gene co-expression analysis in glioblastoma tumors showed a correlation between STAT3 expression and the enhancer RNA LINC00910, which is located in the same chromosomal domain as the Stat3 gene. Previous literature has shown that LINC00910 is associated with both colorectal and gastric cancer. Additionally, data from the GeneHancer database suggests that the enhancer RNA LINC00910 may be involved in regulating the STAT3 immunoregulator. RNA interference was used to effectively knockdown the enhancer RNA LINC00910, resulting in an 8- to 10-fold reduction in its expression in glioblastoma cell lines. The reduction of LINC00910 expression did not significantly affect Stat3 gene expression in glioblastoma cell lines DBTRG-05MG and U251. This suggests that the correlation between LINC00910 RNA expression and STAT3 gene expression is not due to LINC00910’s direct involvement in STAT3 gene regulation in these cells. Further studies using the selected interfering RNA will help to clarify the role of the enhancer RNA LINC00910 in other signallingsignaling pathways, as well as its potential relationship with cancer development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3328c229f12115a982fc9f6e5b3d51533340c9b" target='_blank'>
              Analysis of enhancer RNA LINC00910 expression correlating with the immunoregulator STAT3 in glioblastoma cells
              </a>
            </td>
          <td>
            E. M. Stasevich, A. V. Simonova, A. Uvarova, E. A. Zheremyan, K. Korneev, E. A. Bogomolova, D. Demin
          </td>
          <td>2024-07-27</td>
          <td>Medical Immunology (Russia)</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="

 In this article, Antoinette van Ouwerkerk and Salvatore Spicuglia from INSERM highlight the significance of regulatory variants within the non-coding genome – often referred to as the ‘dark genome’ – in influencing gene expression and disease. The major genetic cause of human disease has historically been thought to be mutations within coding genes. However, there has been a shift in perspective over recent years, with more attention now being paid to the role of the non-coding regions in the genome. Intense research in this area has revealed that the non-coding genome contributes to gene-regulatory networks that serve to control where and when sets of genes are expressed. In turn, dysregulation of gene expression due to changes or alterations in the non-coding genome drastically impacts the aetiology of different diseases.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1b69d242a16126cd54a0cc28fd1836e10bb273d" target='_blank'>
              How the dark genome enlightens the molecular mechanisms of diseases
              </a>
            </td>
          <td>
            Salvatore Spicuglia
          </td>
          <td>2024-07-09</td>
          <td>Open Access Government</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4935f35210bf40b045d154a339912e6ddb86073" target='_blank'>
              Decoding high mobility group A2 protein expression regulation and implications in human cancers
              </a>
            </td>
          <td>
            Farah Khazem, A. B. Zetoune
          </td>
          <td>2024-07-31</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The genome is partitioned into distinct chromatin compartments with at least two main classes, a transcriptionally active A and an inactive B compartment, corresponding mostly to the segregation of euchromatin and heterochromatin. Chromatin within the same compartment has a higher tendency to interact with itself than with regions in opposing compartments. A/B compartments are traditionally derived from ensemble Hi-C contact matrices through principal component analysis of their covariance matrices. However, defining compartments in single cells from single-cell Hi-C maps is non trivial due to sparsity of the data and the fact that homologous copies are typically not resolved. Here we present an unsupervised approach, named MaxComp, to determine single-cell A/B compartments from geometric considerations in 3D chromosome structures, either from multiplexed FISH imaging or from models derived from Hi-C data. By representing each single-cell structure as an undirected graph with edge-weights encoding structural information, the problem of predicting chromosome compartments can be transformed to an alternative form of the Max-cut problem, a semidefinite graph programming method (SPD) to determine an optimal division of a chromosome structure graph into two structural compartments. Our results show that compartment annotations from principal component analysis of ensemble Hi-C data can be perfectly reproduced as population averages of our single-cell compartment predictions. We therefore prove that compartment predictions can be achieved from geometric considerations alone using 3D coordinates of chromatin regions together with information about their nuclear microenvironment. Our results reveal substantial cell-to-cell heterogeneity of compartments in a cell population, which substantially differs between individual genomic regions. Moreover, by applying our approach to multiplexed FISH tracing experiments, our method sheds light on the relationship between single-cell compartment annotations and gene transcriptional activity in single cells. Overall our approach provides new insights into single-cell chromatin condensation, relationship between population and single-cell chromatin compartmentalization, the cell-to-cell variations of chromatin compartments and its impact on gene transcription. Author Summary Chromosome conformation capture and imaging techniques revealed the segregation of genomic chromatin into at least two functional compartments. Hi-C contact frequency matrices show checkerboard-like patterns indicating that chromatin regions are divided into at least two states, possibly a result of phase separation. Chromatin regions in the same state have preferential interactions with each other, often over extended sequence distances, while interactions to regions in the opposing state are minimized. Principal component analysis (PCA) on ensemble Hi-C contact frequency matrices can identify these compartment states. However, because the compartment annotations are derived from a cell population, this method cannot provide information about compartments in single cells. Here in this study, we introduce an unsupervised method to predict single-cell compartments using graph-based programming, which utilizes only structural information in single cells. Our results demonstrate that PCA-based ensemble compartment annotations can be reproduced as population averages of our single-cell compartment predictions. Moreover, our results reveal the cell-to-cell heterogeneity of compartments in a cell population, which shows significant disparities among different chromatin regions. Moreover, by applying our approach to multiplexed FISH tracing experiments, our method reveals the relationship between single-cell compartment annotations and gene transcriptional activity in single cells. Finally, our approach also allows us to relate chromatin structural features in single cells with compartment properties. Comparison with other existing approaches showed that our method produces overall better compartmentalization scores in single cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5153cd191eb013027c48e1d56d079848585fef6" target='_blank'>
              Prediction of single-cell chromatin compartments from single-cell chromosome structures by MaxComp
              </a>
            </td>
          <td>
            Yuxiang Zhan, Francesco Musella, Frank Alber
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Despite significant advances in our understanding of genetic cancer susceptibility, known inherited cancer predisposition syndromes explain at most 20% of early-onset cancers. As early-onset cancer prevalence continues to increase, the need to assess previously inaccessible areas of the human genome, harnessing a trio or quad family-based architecture for variant filtration, may reveal further insights into cancer susceptibility. To assess a broader spectrum of variation than can be ascertained by multi-gene panel sequencing, or even whole genome sequencing with short reads, we employed long read whole genome sequencing using an Oxford Nanopore Technology (ONT) PromethION of 3 families containing an early-onset cancer proband using a trio or quad family architecture. Analysis included 2 early-onset colorectal cancer family trios and one quad consisting of two siblings with testicular cancer, all with unaffected parents. Structural variants (SVs), epigenetic profiles and single nucleotide variants (SNVs) were determined for each individual, and a filtering strategy was employed to refine and prioritize candidate variants based on the family architecture. The family architecture enabled us to focus on inapposite variants while filtering variants shared with the unaffected parents, significantly decreasing background variation that can hamper identification of potentially disease causing differences. Candidate de novo and compound heterozygous variants were identified in this way. Gene expression, in matched neoplastic and pre-neoplastic lesions, was assessed for one trio. Our study demonstrates the feasibility of a streamlined analysis of genomic variants from long read ONT whole genome sequencing and a way to prioritize key variants for further evaluation of pathogenicity, while revealing what may be missing from panel based analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579a5b24c1657cfe092825293758535eef92a0f6" target='_blank'>
              Exploring the genetic and epigenetic underpinnings of early-onset cancers: Variant prioritization for long read whole genome sequencing from family cancer pedigrees
              </a>
            </td>
          <td>
            M. Kramer, Sara Goodwin, R. Wappel, Matilde Borio, K. Offit, D. Feldman, Z. Stadler, W. R. McCombie
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>128</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84ee9b26190f6915876b45d7c5c57408aac14f02" target='_blank'>
              Loss of histone deubiquitinase Bap1 triggers anti-tumor immunity.
              </a>
            </td>
          <td>
            Hong Chang, Mingxia Li, Linlin Zhang, Meng Li, S. Ong, Zhiwei Zhang, Jie Zheng, Xiang Xu, Yu Zhang, Jing Wang, Xingjie Liu, Kairui Li, Yao Luo, Haiyun Wang, Zhichao Miao, Xi Chen, Jie Zha, Yong Yu
          </td>
          <td>2024-08-14</td>
          <td>Cellular oncology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Elucidating the genetic architecture of DNA methylation (DNAm) is crucial for decoding the etiology of complex diseases. However, current epigenomic studies often suffer from incomplete coverage of methylation sites and the use of tissues containing heterogeneous cell populations. To address these challenges, we present a comprehensive human methylome atlas based on deep whole-genome bisulfite sequencing (WGBS) and whole-genome sequencing (WGS) of purified monocytes from 298 European Americans (EA) and 160 African Americans (AA) in the Louisiana Osteoporosis Study. Our atlas enables the analysis of over 25 million DNAm sites. We identified 1,383,250 and 1,721,167 methylation quantitative trait loci (meQTLs) in cis-regions for EA and AA populations, respectively, with 880,108 sites shared between ancestries. While cis-meQTLs exhibited population-specific patterns, primarily due to differences in minor allele frequencies, shared cis-meQTLs showed high concordance across ancestries. Notably, cis-heritability estimates revealed significantly higher mean values in the AA population (0.09) compared to the EA population (0.04). Furthermore, we developed population-specific DNAm imputation models using Elastic Net, enabling methylome-wide association studies (MWAS) for 1,976,046 and 2,657,581 methylation sites in EA and AA, respectively. The performance of our MWAS models was validated through a systematic multi-ancestry analysis of 41 complex traits from the Million Veteran Program. Our findings bridge the gap between genomics and the monocyte methylome, uncovering novel methylation-phenotype associations and their transferability across diverse ancestries. The identified meQTLs, MWAS models, and data resources are freely available at www.gcbhub.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02561d4c021777b32cc89ad698157b2f333e71dc" target='_blank'>
              An atlas of genetic effects on the monocyte methylome across European and African populations
              </a>
            </td>
          <td>
            W. Zhang, X. Zhang, C. Qiu, Z. Zhang, K.-J. Su, Z. Luo, Q. Tian, M. Liu, B. Zhao, L. Wu, H. Shen, C. Wu, H.-W. Deng
          </td>
          <td>2024-08-13</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cfb136406d6e0103812e5102a4ed1670edf2da" target='_blank'>
              METTL3-mediated m6A modification of circGLIS3 promotes prostate cancer progression and represents a potential target for ARSI therapy
              </a>
            </td>
          <td>
            Xiaofeng Cheng, Heng Yang, Yujun Chen, Zhenhao Zeng, Yifu Liu, Xiaochen Zhou, Cheng Zhang, An Xie, Gongxian Wang
          </td>
          <td>2024-08-14</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 What’s the pattern of genome-wide chromatin occupancy of SCP3 during meiotic prophase, and how its interaction with chromatin is regulated?



 We document SCP3 enriched at active chromatin regions with largely inherited from leptotene to pachytene stage, which facilitated by transcription and condensate assembly in spermatocytes.



 Meiosis is a critical step during the process of gametogenesis to form haploid cells which finally develop into functional gametes. The synaptonemal complex (SC) is a meiosis-specific protein assembly which bridges homologous chromosomes to establish the architecture of meiotic chromosome. Genetic ablation of genes encoding SC components impedes meiosis and gametogenesis in mice and mutation of these genes is frequently found in infertile patients. SCP3 localization changes with chromosome morphology in autosomes and is involved in maintaining the association of sex chromosomes. Male mice with null mutation of SCP3 failed to form SCs, resulting in developmental arrest during meiotic prophase.



 This study included male mice at post-natal day 10 (P10), 4 weeks or adult (8 weeks or older) were maintained on C57BL/6J background. Spermatocytes were performed modified CUT&Tag using antibodies. ChIP-seq data was analyzed together with WGBS and ATAC-seq data.



 Spermatocytes were obtained by a discontinuous BSA density gradient. Cell pellet was resuspended in culture medium (MEM with 10% FBS) at a concentration of 105 cells/ml and cultured at 37 °C in a 5% CO2 incubator. Spermatocytes were cultured with or without 1,6-hexanediol or α-amanitin. Spermatocytes with or without drug treatment were collected at indicated time points for further examination and then modified CUT&Tag and co-IP using anti-SCP3 and anti-POL II antibodies.



 Chromatin occupancy of SCP3 in leptotene and pachytene spermatocytes, SCP3-Core sites retained upon treatment of 1,6-hexanediol which dissolves SCP3 condensate, and SCP1 occupancy at meiotic chromatin in pachytene spermatocytes. When we determined inter-peak distances and width of peaks, we observe that: (1) SCP3 occupancy in leptotene stage had greater inter-peak distances and narrower peak width than SCP3 occupancy in pachytene stage, demonstrating gradual accumulated SCP3 at chromatin during meiotic progression; (2) SCP3-Core sites have broader peak width than SCP3-NonCore sites, in agreement with stability of SCP3 occupancy at SCP3-Core sites; (3) SCP1 occupancy had greater inter-peak distances and narrower peak width than SCP3 occupancy in pachytene stage, supporting that SCP1 peaks are a subset of SCP3 peaks and SCP1’s major function in connecting LE and CE respectively. Generally, at pachytene-stage meiosis, SCP3 mainly occupies chromatin regions including transcriptionally active sites, Deu and Alu elements, while SCP1 mainly co-localizes with SCP3 at a subset of regions with open chromatin and high cohesin enrichment to connect meiotic chromosome axes with CE. In this way, a local active chromatin environment is created at meiotic chromosome axes, facilitating homologous chromosome recombination.



 Our test was not successful to examine SCP3 occupancy with as few as less than 1000 spermatocytes using our protocol. It is improvement of efficiency of tagmentation or development of more efficient antibodies against SCP3 will be helpful for clinical sample detection.



 SCP3 occupancy was largely inherited from leptotene to pachytene stage, facilitated by transcription and condensate assembly, and enriched at specific SINE repeats. Further exploration using knockout mouse models of meiotic axial proteins and epigenetic modifiers will provide valuable information on meiotic chromatin architecture and crosstalk among meiotic chromatin events.



 N/A
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd5370d474a768836582b1cb861c8c3d70c98bd" target='_blank'>
              P-045 Deciphering meiotic chromatin organization by SCP3
              </a>
            </td>
          <td>
            S. Guo
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1f187326818c3bc4b9634f53afc541e906548d2" target='_blank'>
              Targeting PRMT5 enhances the radiosensitivity of tumor cells grown in vitro and in vivo
              </a>
            </td>
          <td>
            Charlotte M. Degorre, Steven Lohard, Christina N. Bobrek, Komal N. Rawal, Skyler Kuhn, P. Tofilon
          </td>
          <td>2024-07-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Background Tandem duplication (TD) is a common and important type of structural variation in the human genome. TDs have been shown to play an essential role in many diseases, including cancer. However, it is difficult to accurately detect TDs due to the uneven distribution of reads and the inherent complexity of next-generation sequencing (NGS) data. Methods This article proposes a method called DTDHM (detection of tandem duplications based on hybrid methods), which utilizes NGS data to detect TDs in a single sample. DTDHM builds a pipeline that integrates read depth (RD), split read (SR), and paired-end mapping (PEM) signals. To solve the problem of uneven distribution of normal and abnormal samples, DTDHM uses the K-nearest neighbor (KNN) algorithm for multi-feature classification prediction. Then, the qualified split reads and discordant reads are extracted and analyzed to achieve accurate localization of variation sites. This article compares DTDHM with three other methods on 450 simulated datasets and five real datasets. Results In 450 simulated data samples, DTDHM consistently maintained the highest F1-score. The average F1-score of DTDHM, SVIM, TARDIS, and TIDDIT were 80.0%, 56.2%, 43.4%, and 67.1%, respectively. The F1-score of DTDHM had a small variation range and its detection effect was the most stable and 1.2 times that of the suboptimal method. Most of the boundary biases of DTDHM fluctuated around 20 bp, and its boundary deviation detection ability was better than TARDIS and TIDDIT. In real data experiments, five real sequencing samples (NA19238, NA19239, NA19240, HG00266, and NA12891) were used to test DTDHM. The results showed that DTDHM had the highest overlap density score (ODS) and F1-score of the four methods. Conclusions Compared with the other three methods, DTDHM achieved excellent results in terms of sensitivity, precision, F1-score, and boundary bias. These results indicate that DTDHM can be used as a reliable tool for detecting TDs from NGS data, especially in the case of low coverage depth and tumor purity samples.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/332aeecb63c38630dbdee5179707b4c99d4bb7e6" target='_blank'>
              DTDHM: detection of tandem duplications based on hybrid methods using next-generation sequencing data
              </a>
            </td>
          <td>
            Tianting Yuan, Jinxin Dong, Baoxian Jia, Hua Jiang, Zuyao Zhao, Mengjiao Zhou
          </td>
          <td>2024-07-26</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Satellite DNAs (satDNAs) are tandemly repeated sequences that make up a significant portion of almost all eukaryotic genomes. Although satDNAs have been shown to play a very important role in genome organization and evolution, they are relatively poorly analysed even in model. One of the main reasons for the current lack of in-depth studies on satDNAs is their underrepresentation in genome assemblies. The complexity and highly repetitive nature of satDNAs make their analysis challenging, and there is a need for efficient tools that can ensure accurate annotation and analysis of satDNAs. We present a novel pipeline, named Satellite DNA Exploration (SatXplor), designed to robustly characterize satDNA elements and analyse their arrays and flanking regions. SatXplor is benchmarked against curated satDNA datasets from diverse species, showcasing its versatility across genomes with varying complexities and different satDNA profile. Component algorithms excel in the identification of tandemly repeated sequences and for the first time enable evaluation of satDNA variation and array annotation with the addition of information about surrounding genomic landscape. SatXplor is an innovative pipeline for satDNA analysis that can be paired with any tool used for satDNA detection, offering insights into the structural characteristics, array determination and genomic context of satDNA elements. By integrating various computational techniques, from sequence analysis and homology investigation to advanced clustering and graph-based methods, it provides a versatile and comprehensive approach to explore the complexity of satDNA organization and to understand the underlying mechanisms and evolutionary aspects. It is open-source and freely accessible at https://github.com/mvolar/SatXplor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93bcbb026ce597eb6926d4d48baa8708101680" target='_blank'>
              SatXplor – A comprehensive pipeline for satellite DNA analyses in complex genome assemblies
              </a>
            </td>
          <td>
            Marin Volarić, N. Meštrović, Evelin Despot-Slade
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4ee6e732452c0ae079ae842747b6a8d042906b7" target='_blank'>
              The contribution of silencer variants to human diseases
              </a>
            </td>
          <td>
            D. Huang, I. Ovcharenko
          </td>
          <td>2024-07-08</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) stands as one of the most prevalent and aggressive forms of renal malignancies, accounting for a substantial portion of kidney cancer-related mortality worldwide. The heterogeneity within ccRCC poses a significant challenge to understanding its molecular mechanisms and devising effective therapeutic strategies. [1] Previous studies utilizing single nucleus RNA sequencing (snRNA-seq) have revealed transcriptional contributions to cell-type specificity in both mature human and mouse kidneys. Moreover, recent advancements have expanded this approach to single-cell profiling of chromatin accessibility. Single-nuclei assay for transposase-accessible chromatin using sequencing (snATAC-seq) has enabled the measurement of chromatin accessibility in thousands of individual cells, providing insights into the dynamic process that drives nephron development and differentiation. Integration and analysis of multimodal single-cell datasets, such as snRNA-seq and snATAC-seq, allows for the prediction of cell-type-specific cis-regulatory DNA interactions and transcription factor activity, complementing the transcriptional information obtained by snRNA-seq.[2] [3] [1] Yang, Wang, and Yang, “Treatment Strategies for Clear Cell Renal Cell Carcinoma.” [2] Muto et al., “Single Cell Transcriptional and Chromatin Accessibility Profiling Redefine Cellular Heterogeneity in the Adult Human Kidney.” [3] Monteagudo-Sánchez, Noordermeer, and Greenberg, “The Impact of DNA Methylation on CTCF-Mediated 3D Genome Organization.” Methods This study leveraged paired snRNA/ATAC-seq in tumor and normal-adjacent samples from the Renal Tumor Biobank at Dartmouth (n= 63) to explore the ccRCC landscape at single-cell resolution using 10x multiome technology. The snRNA-seq dataset was processed with Seurat v5.0 to remove low-quality nuclei. Doublets were removed with DoubletFinder v2.02, and ambient DNA was removed using decontX from celda. Normalization was conducted via SCTransform, and missing data was imputed with ALRA. Clustering was performed by constructing a KNN graph and running the Louvain algorithm. Cell type assignments were predicted using Azimuth with the human kidney reference dataset. The snATAC-seq data was processed with ArchR v1.0.1, which performed dimensionality reduction and clustering. Cluster identities defined in the Seurat workflow were cross-annotated to the ATAT-seq data, and open peaks were identified within these cell-type clusters using MACS3 v3.0.1. Identification of marker peaks, motif enrichment, and motif enrichment deviations were inferred utilizing ArchR’s getmarkerFeatures, peakAnnoEnrichment, and addMotifAnnotations functions, respectively. Results Motif enrichment analysis revealed canonical cancer-associated transcription factor (TF) dysregulation in tumor cells (nephron loop-like), including enrichment of HNF1B, SMARCC1, FOSL1, and JUND. Interestingly, HNF and SMARCC were also enriched in the lymphoid compartment, indicating possible crosstalk between these two cell types. Previous work has investigated the involvement of the HNF1 in ccRCC and in chromophobe RCC for prognosis,and we plan on investigating the relationship of these enrichments to tumor grade, stage and outcome.[1] Additional TF enrichments included BCL families and ELF within the myeloid compartment, and vascularization related TF families in endothelial cell types including enrichment of various ETS TFs. Investigation of cell-cell interactions between immune and endothelial cell populations as well as between tumor and normal adjacent tissues will be performed to further disentangle the role differentially accessable chromatin plays in ccRCC. [1] Buchner et al., “Downregulation of HNF-1B in Renal Cell Carcinoma Is Associated With Tumor Progression and Poor Prognosis.” Conclusions Our multi-omic approach has several advantages. First, it enables the characterization of cellular heterogeneity within the tumor microenvironment, identifying cell populations and their contributions to disease progression. Secondly, by integrating transcriptomic and epigenomic data, we gain insights into the dynamic interplay between gene expression and chromatin accessibility, uncovering potential therapeutic vulnerabilities and biomarkers for personalized treatment strategies. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e88d9a75048a69eb2afe5150d509757fff83b4" target='_blank'>
              60 Dissecting the Cellular and Epigenetic Landscape of Clear Cell Renal Cell Carcinoma Through Paired snRNA/ATAC-Seq Analysis
              </a>
            </td>
          <td>
            Ian Gingerich
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Circular RNAs (circRNAs) are involved in the pathogenesis of esophageal squamous cell carcinoma (ESCC). This study aimed to explore the mechanisms of aberrant expression and functions of circ_0006168 in ESCC. In this study, real-time qPCR and fluorescence in situ hybridization (FISH) are adopted to estimate the expression and localization of circ_0006168 in cancer tissues and cells. Methylated RNA immunoprecipitation (MeRIP) was performed to detect the N6-methyladenosine (m6A) modification of circ_0006168. Gain- and loss-of-functions of circ_0006168 were performed to identify its role in ESCC progression. RNA-binding protein immunoprecipitation (RIP) was used to detect the interaction of circ_0006168 with IGF2BP2. Luciferase reporter assay and RIP are used to confirm the circ_0006168/miR-384/STAT3 ceRNA network. Our results showed that the expression of circ_0006168 was upregulated in ESCC tissues and cells. METTL3-mediated m6A modification increased the expression of circ_0006168 via IGF2BP2-dependent way in TE-1 cells. Circ_0006168 promoted cell proliferation, migration, invasion, cell cycle progression and inhibited cell apoptosis, while knockdown of circ_0006168 had the reverse effects. Mechanistically, circ_0006168 acted its functions via miR-384/STAT3/Snail axis in TE-1 cells. In conclusion, circ_0006168 is upregulated in ESCC and m6A methylation increased its expression via IGF2BP2. CircRNA_0006168 promotes cell migration, invasion by regulating EMT via miR-384/STAT3/Snail axis in ESCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5633a4be443d2222f80bed6a025bc3273be4d5d6" target='_blank'>
              N6-methyladenosine-modified circ_0006168 promotes epithelial mesenchymal transition via miR-384/STAT3/Snail axis in esophageal squamous cell carcinoma
              </a>
            </td>
          <td>
            Guandi Wu, Qin Hou, Zhe Liu, Zejin Pu, Lingfei Wu
          </td>
          <td>2024-07-16</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The Microphthalmia associated transcription factor (Mitf) is a critical regulator of the melanocyte lineage and also plays an important role in eye development. Mitf activity in different cell types is controlled in part by ten alternative promoters and their resulting isoforms. A useful tool for melanocyte-based research, the Mitf-cre transgene was designed to express Cre recombinase from the Mitf-M promoter, which is melanocyte specific. However, Mitf-cre mice are also microphthalmic, perhaps because of insertional mutagenesis or disrupted gene expression. Here, we investigated these possibilities. We determined that the eye phenotype arises early, with Mitf-cre embryos at E13.5 exhibiting variable ocular sizes and abnormalities, but all with coloboma. Targeted locus amplification and next generation sequencing indicated that multiple copies of the transgene integrated into an intergenic region on chromosome 2, in between Spred1 and Meis2. The BAC transgene used to make Mitf-cre was larger than expected, carrying three upstream alternative promoters, Mitf-H, Mitf-D, and Mitf-B, which could express their isoforms intact off the transgene. RT-qPCR using eye tissue demonstrated a 5-fold increase in Mitf transcripts containing exon 1B1b, which is shared by Mitf-H, Mitf-D, and Mitf-B, while Spred1 and Meis2 did not differ in their expression. These findings clarify and support the usage of Mitf-cre in conditional mutagenesis in melanocytes. The specific over-expression of the Mitf-H and Mitf-D isoforms, which are preferentially expressed in the RPE, presents a unique resource for those interested in eye development and coloboma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f416be172f324512ebeadaba72cc51d72c665bd9" target='_blank'>
              Mitf over-expression leads to microphthalmia and coloboma in Mitf-cre mice
              </a>
            </td>
          <td>
            A. Longakit, Hannah Bourget, C. D. Van Raamsdonk
          </td>
          <td>2024-07-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86376d37d11acefead8c66a577f19b5d0ad22d4f" target='_blank'>
              Circular RNA circ_ARHGEF28 inhibits MST1/2 dimerization to suppress Hippo pathway to induce cisplatin resistance in ovarian cancer
              </a>
            </td>
          <td>
            R. Lei, Yun Long, Qingjian Li, Qingsheng Xie, Xiaoting Ling, Meiqing Xie, Hui Zhou, Bingzhong Zhang
          </td>
          <td>2024-07-21</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Despite the established role of DNA methylation in transcriptional silencing, most of the genes remain unchanged upon its global erasing from the genome, while expression of other genes is actually raised. Since its discovery a few decades ago, this unsolved enigma has hampered true understanding of DNA methylation role. By applying an updated mathematical model to the results of the original studies, we showed that this paradoxical effect is due to between-gene differences in the balance between methylation-guided enhancers and silencers, which protect particular disease-causing genes from the otherwise harsh effects of global methylation perturbation. Nevertheless, local methylation alternation may still affect these genes. The results a settle a long-standing wondering and suggest that DNA-methylation regulatory role is significantly greater than previously thought.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f326a9727e723fecbefe9526db8fc37ada53d66" target='_blank'>
              On the enigmatic null effect of global methylation perturbation
              </a>
            </td>
          <td>
            Y. Edrei, A. Hellman
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Radiation cytogenetics has a rich history seldom appreciated by those outside the field. Early radiobiology was dominated by physics and biophysical concepts that borrowed heavily from the study of radiation-induced chromosome aberrations. From such studies, quantitative relationships between biological effect and changes in absorbed dose, dose rate and ionization density were codified into key concepts of radiobiological theory that have persisted for nearly a century. This review aims to provide a historical perspective of some of these concepts, including evidence supporting the contention that chromosome aberrations underlie development of many, if not most, of the biological effects of concern for humans exposed to ionizing radiations including cancer induction, on the one hand, and tumor eradication on the other. The significance of discoveries originating from these studies has widened and extended far beyond their original scope. Chromosome structural rearrangements viewed in mitotic cells were first attributed to the production of breaks by the radiations during interphase, followed by the rejoining or mis-rejoining among ends of other nearby breaks. These relatively modest beginnings eventually led to the discovery and characterization of DNA repair of double-strand breaks by non-homologous end joining, whose importance to various biological processes is now widely appreciated. Two examples, among many, are V(D)J recombination and speciation. Rapid technological advancements in cytogenetics, the burgeoning fields of molecular radiobiology and third-generation sequencing served as a point of confluence between the old and new. As a result, the emergent field of "cytogenomics" now becomes uniquely positioned for the purpose of more fully understanding mechanisms underlying the biological effects of ionizing radiation exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c57db9d0b65c8dcb0db634d392394a245d1aa2a" target='_blank'>
              The Central Role of Cytogenetics in Radiation Biology.
              </a>
            </td>
          <td>
            Susan M. Bailey, Stephen R. Kunkel, Joel S. Bedford, Michael N. Cornforth
          </td>
          <td>2024-07-10</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Introduction Glioblastomas utilize malignant gene expression pathways to drive growth. Many of these gene pathways are not directly accessible with molecularly targeted pharmacological agents. Chromatin-modifying compounds can alter gene expression and target glioblastoma growth pathways. In this study, we utilize a systematic screen of chromatin-modifying compounds on a panel of patient-derived glioblastoma lines to identify promising compounds and their associated gene targets. Methods Five glioblastoma cell lines were subjected to a drug screen of 106 chromatin-modifying compounds representing 36 unique drug classes to determine the twelve most promising drug classes and the best candidate inhibitors in each class. These twelve drugs were then tested with a panel of twelve patient-derived gliomasphere lines to identify growth inhibition and corresponding gene expression patterns. Overlap analysis and weighted co-expression network analysis (WCGNA) were utilized to determine potential target genes and gene pathways. Results The initial drug screen identified twelve candidate pharmacologic agents for further testing. Drug sensitivity testing indicated an overall high degree of variability between gliomasphere lines. However, CPI203 was the most consistently effective compound, and the BET inhibitor class was the most consistently effective class of compounds across the gliomasphere panel. Correspondingly, most of the compounds tested had highly variable effects on gene expression between gliomasphere lines. CPI203 stood out as the only compound to induce a consistent effect on gene expression across different gliomasphere lines, specifically down-regulation of DNA-synthesis genes. Amongst the twelve tested cell lines, high expression of CDKN2A and CDKN2B distinguished more drug sensitive from more drug resistant lines. WCGNA identified two oncogenic gene modules (FBXO5 and MELK) that were effectively downregulated by CPI203 (FBXO5) and ML228 (FBXO5 and MELK). Conclusions The bromodomain inhibitor CPI203 induced relatively consistent effects on gene expression and growth across a variety of glioblastoma lines, specifically down-regulating genes associated with DNA replication. We propose that clinically effective BET inhibitors have the potential to induce consistent beneficial effects across a spectrum of glioblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1561fa27def3a17181508712df5ccd39c21a9f63" target='_blank'>
              CPI203, a BET inhibitor, down-regulates a consistent set of DNA synthesis genes across a wide array of glioblastoma lines
              </a>
            </td>
          <td>
            Matthew C. Garrett, Troy P Carnwath, Rebecca R Albano, Yonghua Zhuang, Catherine A Behrmann, Merissa Pemberton, Farah Barakat, Robert Lober, Mark Hoeprich, Anthony Paravati, Marilyn Reed, Hailey Spry, Daniel Woo, Eric O’Brien, Brett VanCauwenbergh, John Perentesis, Rani Nasser, Mario Medvedovic, D. Plas
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic mechanism orchestrating gene expression networks in many biological processes. Nonetheless, studying the role of specific gene methylation events in fish faces challenges. In this study, we validate the regulation of DNA methylation on empty spiracles homeobox 2 (emx2) expression with decitabine treatment in Chinese tongue sole testis cells. We used the emx2 gene as the target gene and developed a new DNA methylation editing system by fusing dnmt3a with catalytic dead Cas9 (dCas9) and demonstrated its ability for sequence-specific DNA methylation editing. Results revealed that utilizing dCas9-dnmt3a to target emx2 promoter region led to increased DNA methylation levels and decreased emx2 expression in Chinese tongue sole testis cells. More importantly, the DNA methylation editing significantly suppressed the expression of MYC proto-oncogene, bHLH transcription factor (myc), one target gene of emx2. Furthermore, we assessed the off-target effects of dCas9-dnmt3a and confirmed no significant impact on the predicted off-target gene expression. Taken together, we developed the first DNA methylation editing system in marine species and demonstrated its effective editing ability in Chinese tongue sole cells. This provides a new strategy for both epigenetic research and molecular breeding of marine species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/418776c51af85059269724bfd4c08f2823970cf0" target='_blank'>
              CRISPR/dCas9-Mediated DNA Methylation Editing on emx2 in Chinese Tongue Sole (Cynoglossus semilaevis) Testis Cells
              </a>
            </td>
          <td>
            Yanxu Sun, Hongyan Wang, Binghua Liu, Bowen Yue, Qian Liu, Yuyan Liu, Ivana F. Rosa, L. Doretto, Shenglei Han, Lei Lin, Xiaoling Gong, Changwei Shao
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="R-loops are trimeric nucleic acid structures that form when an RNA molecule hybridizes with its complementary DNA strand, displacing the opposite strand. These structures regulate transcription as well as replication, but aberrant R-loops can form, leading to DNA breaks and genomic instability if unresolved. R-loop levels are elevated in many cancers as well as cells that maintain high-risk human papillomaviruses. We investigated how the distribution as well as function of R-loops changed between normal keratinocytes and HPV positive cells derived from a precancerous lesion of the cervix (CIN I). The levels of R-loops associated with cellular genes were found to be up to 10-fold higher in HPV positive cells than in normal keratinocytes while increases at ALU1 elements increased by up to 500-fold. The presence of enhanced R-loops resulted in altered levels of gene transcription, with equal numbers increased as decreased. While no uniform global effects on transcription due to the enhanced levels of R-loops were detected, genes in several pathways were coordinately increased or decreased in expression only in the HPV positive cells. This included the downregulation of genes in the innate immune pathway, such as DDX58, IL-6, STAT1, IFN-β, and NLRP3. All differentially expressed innate immune genes dependent on R-loops were also associated with H3K36me3 modified histones. Genes that were upregulated by the presence of R-loops in HPV positive cells included those in the DNA damage repair such as ATM, ATRX, and members of the Fanconi Anemia pathway. These genes exhibited a linkage between R-loops and H3K36me3 as well as γH2AX histone marks only in HPV positive cells. These studies identify a potential link in HPV positive cells between DNA damage repair as well as innate immune regulatory pathways with R-loops and γH2AX/H3K36me3 histone marks that may contribute to regulating important functions for HPV pathogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d30cc12d40eed32c558ede6628c4c8dfd335ab28" target='_blank'>
              HPV induced R-loop formation represses innate immune gene expression while activating DNA damage repair pathways.
              </a>
            </td>
          <td>
            Conor W Templeton, Lou Laimins
          </td>
          <td>2024-08-23</td>
          <td>PLoS pathogens</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7688d245ed2ef09eafd9e57de454a0d8db59f71a" target='_blank'>
              Transposable elements contribute to tissue-specific gene regulation in humans.
              </a>
            </td>
          <td>
            Arsala Ali, Ping Liang
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Trinucleotide repeat (TNR) expansion is the cause of over 40 neurodegenerative diseases, including Huntington’s disease and Friedreich’s ataxia (FRDA). There are no effective treatments for these diseases due to the poor understanding of molecular mechanisms underlying somatic TNR expansion and contraction in neural systems. We and others have found that DNA base excision repair (BER) actively modulates TNR instability, shedding light on the development of effective treatments for the diseases by contracting expanded repeats through DNA repair. In this study, temozolomide (TMZ) was employed as a model DNA base damaging agent to reveal the mechanisms of the BER pathway in modulating GAA repeat instability at the frataxin (FXN) gene in FRDA neural cells and transgenic mouse mice. We found that TMZ induced large GAA repeat contraction in FRDA mouse brain tissue, neurons, and FRDA iPSC-differentiated neural cells, increasing frataxin protein levels in FRDA mouse brain and neural cells. Surprisingly, we found that TMZ could also inhibit H3K9 methyltransferases, leading to open chromatin and increasing ssDNA breaks and recruitment of the key BER enzyme, pol β, on the repeats in FRDA neural cells. We further demonstrated that the H3K9 methyltransferase inhibitor BIX01294 also induced the contraction of the expanded repeats and increased frataxin protein in FRDA neural cells by opening the chromatin and increasing the endogenous ssDNA breaks and recruitment of pol β on the repeats. Our study provides new mechanistic insight illustrating that inhibition of H3K9 methylation can crosstalk with BER to induce GAA repeat contraction in FRDA. Our results will open a new avenue for developing novel gene therapy by targeting histone methylation and the BER pathway for repeat expansion diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/88103a73ada31d035f661593eaffbaace8d21383" target='_blank'>
              DNA Base Damage Repair Crosstalks with Chromatin Structures to Contract Expanded GAA Repeats in Friedreich’s Ataxia
              </a>
            </td>
          <td>
            Yanhao Lai, Nicole Diaz, Rhyisa Armbrister, Irina Agoulnik, Yuan Liu
          </td>
          <td>2024-07-01</td>
          <td>Biomolecules</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="There remains a large need for a greater understanding of the metastatic process within the prostate cancer field. Our research aims to understand the adaptive – ergo potentially metastatic – responses of cancer to changing microenvironments. Emerging evidence has implicated a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study of prostate cancer patients revealed that PACC presence in the prostate at the time of radical prostatectomy was predictive of future metastatic progression. To test for a causative relationship between PACC state biology and metastasis, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) in subcutaneous, caudal artery, and intracardiac mouse models of metastasis. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. In vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites following dormancy. Additional direct and indirect mechanistic in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence that PACCs are mechanistically linked to metastasis. Statement of Significance We provide the first evidence that cells in the polyaneuploid cancer cell state contribute to increased metastatic competency in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2da05a6c448f502761bb367a94125d4543338" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T.A. Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, SR Amend
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="RNA-guided endonucleases are involved in processes ranging from adaptive immunity to site-specific transposition and have revolutionized genome editing. CRISPR-Cas9, -Cas12 and related proteins use guide RNAs to recognize ∼20-nucleotide target sites within genomic DNA by mechanisms that are not yet fully understood. We used structural and biochemical methods to assess early steps in DNA recognition by Cas12a protein-guide RNA complexes. We show here that Cas12a initiates DNA target recognition by bending DNA to induce transient nucleotide flipping that exposes nucleobases for DNA-RNA hybridization. Cryo-EM structural analysis of a trapped Cas12a-RNA-DNA surveillance complex and fluorescence-based conformational probing show that Cas12a-induced DNA helix destabilization enables target discovery and engagement. This mechanism of initial DNA interrogation resembles that of CRISPR-Cas9 despite distinct evolutionary origins and different RNA-DNA hybridization directionality of these enzyme families. Our findings support a model in which RNA-mediated DNA engineering begins with local helix distortion by transient CRISPR-Cas protein binding.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ecca8177ea3bf5a7611f5cd9ffa09c83dca74a49" target='_blank'>
              CRISPR-Cas12a bends DNA to destabilize base pairs during target interrogation
              </a>
            </td>
          <td>
            K. Soczek, Joshua C. Cofsky, Owen T. Tuck, Honglue Shi, Jennifer A. Doudna
          </td>
          <td>2024-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b3ca92a03051caca152f854fcb5c2f6b12d56f76" target='_blank'>
              Integrated genomic/epigenomic analysis stratifies subtypes of clear cell ovarian carcinoma, highlighting their cellular origin
              </a>
            </td>
          <td>
            Akira Nishijima, K. Oda, Kosei Hasegawa, Takahiro Koso, K. Asada, Yuji Ikeda, Ayumi Taguchi, Daichi Maeda, G. Nagae, S. Tsuji, K. Tatsuno, Y. Uehara, A. Kurosaki, S. Sato, M. Tanikawa, K. Sone, M. Mori, M. Ikemura, Keiichi Fujiwara, Tetsuo Ushiku, Yutaka Osuga, Hiroyuki Aburatani
          </td>
          <td>2024-08-13</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="The DNA modifications, 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), represent powerful epigenetic regulators of temporal and spatial gene expression. Yet, how the cooperation of these genome-wide, epigenetic marks determine unique transcriptional signatures across different brain cell populations is unclear. Here we applied Nanopore sequencing of native DNA to obtain a complete, genome-wide, single-base resolution atlas of 5mC and 5hmC modifications in neurons, astrocytes and microglia in the mouse cortex (99% genome coverage, 40 million CpG sites). In tandem with RNA sequencing, analysis of 5mC and 5hmC patterns across cell types reveals astrocytes drive uniquely high brain 5hmC levels and support two decades of research regarding methylation patterns, gene expression and alternative splicing, benchmarking this resource. As such, we provide the most comprehensive DNA methylation data in mouse brain as an interactive, online tool (NAM-Me, https://olsenlab.shinyapps.io/NAMME/) to serve as a resource dataset for those interested in the methylome landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60076a55629be60f22fe1afccd00156f5bc051e7" target='_blank'>
              Decoding the Epigenetic Landscape: Insights into 5mC and 5hmC Patterns in Mouse Cortical Cell Types
              </a>
            </td>
          <td>
            Xiaoran Wei, Jiangtao Li, Zuolin Cheng, Songtao Wei, Guoqiang Yu, Michelle L. Olsen
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Clonal hematopoiesis (CH) is defined by the expansion of a lineage of genetic identical cells in blood. Genetic lesions that confer a fitness advantage, such as point mutations or mosaic chromosomal alterations (mCAs) in genes associated with hematologic malignancy, are frequent mediators of CH. However, recent analyses of both single cell-derived colonies of hematopoietic cells and population sequencing cohorts have revealed CH frequently occurs in the absence of previously implicated driver genetic lesions. To characterize CH without known driver genetic lesions, we used 51,399 deeply sequenced whole genomes from the NHLBI TOPMed sequencing initiative to perform simultaneous germline and somatic mutation analyses among individuals without leukemogenic point mutations (LPM), which we term CH-LPMneg. We quantified CH by estimating the total mutation burden. Because estimating somatic mutation burden without a paired-tissue sample is challenging, we developed a novel statistical method, the Genomic and Epigenomic informed Mutation (GEM) rate, that uses external genomic and epigenomic data sources to distinguish artifactual signals from true somatic mutations. We performed a genome-wide association study of GEM to discover the germline determinants of mutation burden in blood. After fine-mapping and variant-to-gene analyses, we identified seven genes associated with CH-LPMneg (TCL1A, TERT, SMC4, NRIP1, PRDM16, MSRA, SCARB1), and one locus associated with a sex-associated mutation pathway (SRGAP2C). We performed a secondary analysis excluding individuals with mCAs, finding that the genetic architecture was largely unaffected by their inclusion. Functional analyses of SMC4 and NRIP1 implicate altered HSC self-renewal and proliferation as the primary mediator of mutation burden in blood. We then performed comprehensive multi-tissue transcriptomic analyses, finding that the expression levels of 404 genes are associated with GEM. Finally, we performed phenotypic association analyses, finding that GEM is associated with increased white blood cell count and increased risk for incident peripheral artery disease, but is not significantly associated with incident stroke or coronary disease events. Overall, we develop GEM for quantifying mutation burden from WGS without a paired-tissue sample and use GEM to discover the genetic, genomic, and phenotypic correlates of CH-LPMneg.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cbb2b70bb8c9d2cf0cfc0c3f0db89c3a414b8c4" target='_blank'>
              The Genetic Determinants and Genomic Consequences of Non-Leukemogenic Somatic Point Mutations
              </a>
            </td>
          <td>
            J. Weinstock, Sharjeel A. Chaudhry, Maria Ioannou, Maria Viskadourou, Paula Reventun, Yasminka A. Jakubek, L. Liggett, C. Laurie, J. Broome, Alyna T Khan, Kent D. Taylor, Xiuqing Guo, P. Peyser, Eric Boerwinkle, N. Chami, Eimear E Kenny, R. Loos, B. Psaty, Tracy P. Russell, Jennifer A. Brody, Jeong H Yun, M. Cho, Ramachandran S. Vasan, S. Kardia, Jennifer A. Smith, L. Raffield, A. Bidulescu, Emily O'Brien, Mariza de Andrade, Jerome I. Rotter, Steve Rich, Russell P. Tracy, Yii Der, Ida Chen, C. Charles, Gu, Chao A. Hsiung, C. Kooperberg, Bernhard Haring, Rami Nassir, Rasika Mathias, Alexander Reiner, Vijay G. Sankaran, Charles J Lowenstein, T. Blackwell, Gonçalo R. Abecasis, A. Smith, Hyun M. Kang, P. Natarajan, S. Jaiswal, A. Bick, Wendy S. Post, P. Scheet, Paul L Auer, Theodoros Karantanos, Alexis Battle, Marios Arvanitis
          </td>
          <td>2024-08-22</td>
          <td>None</td>
          <td>0</td>
          <td>121</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891d9fb97d4fae14734f86d5adf86855004117f2" target='_blank'>
              Clonal inactivation of TERT impairs stem cell competition
              </a>
            </td>
          <td>
            Kazuteru Hasegawa, Yang Zhao, A. Garbuzov, M. Corces, P. Neuhöfer, Victoria M Gillespie, Peggie Cheung, J. Belk, Yung-Hsin Huang, Yuning Wei, Lu Chen, Howard Y. Chang, S. Artandi
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Cancer is a complex and heterogeneous disease, in which a number of genetic and epigenetic changes occur in tumor onset and progression. Recent studies indicate that changes at the RNA level are also involved in tumorigenesis, such as adenosine-to-inosine (A-to-I) RNA editing. Here, we systematically investigate transcriptome-wide A-to-I editing events in a large number of samples from Non-Hodgkin lymphomas (NHLs). Using a computational pipeline that determines significant differences in editing level between NHL and normal samples at known A-to-I editing sites, we identify a number of differentially edited editing sites between NHL subtypes and normal samples. Most of the differentially edited sites are located in non-coding regions, and many such sites show a strong correlation between gene expression level and editing efficiency, indicating that RNA editing might have direct consequences for the cancer cell’s aberrant gene regulation status in these cases. Moreover, we establish a strong link between RNA editing and NHL by demonstrating that NHL and normal samples and even NHL subtypes can be distinguished based on genome-wide RNA editing profiles alone. Our study establishes a strong link between RNA editing, cancer and aberrant gene regulation in NHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3709fa38f5f1119b1fcb4db3f2e4cd77244ecc08" target='_blank'>
              A-to-I Editing Is Subtype-Specific in Non-Hodgkin Lymphomas
              </a>
            </td>
          <td>
            Cai Chen, Ralf Bundschuh
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Acute myeloid leukemia (AML) has a poor survival rate for both pediatric and adult patients due to its frequent relapse. To elucidate the bioenergetic principle underlying AML relapse, we investigated the transcriptional regulation of mitochondrial–nuclear dual genomes responsible for metabolic plasticity in treatment-resistant blasts. Both the gain and loss of function results demonstrated that NFκB2, a noncanonical transcription factor (TF) of the NFκB (nuclear factor kappa-light-chain-enhancer of activated B cells) family, can control the expression of TFAM (mitochondrial transcription factor A), which is known to be essential for metabolic biogenesis. Furthermore, genetic tracking and promoter assays revealed that NFκB2 is in the mitochondria and can bind the specific “TTGGGGGGTG” region of the regulatory D-loop domain to activate the light-strand promoter (LSP) and heavy-strand promoter 1 (HSP1), promoters of the mitochondrial genome. Based on our discovery of NFκB2′s novel function of regulating mitochondrial–nuclear dual genomes, we explored a novel triplet therapy including inhibitors of NFκB2, tyrosine kinase, and mitochondrial ATP synthase that effectively eliminated primary AML blasts with mutations of the FMS-related receptor tyrosine kinase 3 (FLT3) and displayed minimum toxicity to control cells ex vivo. As such, effective treatments for AML must include strong inhibitory actions on the dual genomes mediating metabolic plasticity to improve leukemia prognosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cd91cb6b195c34903e27bb3147d1ba2db984023c" target='_blank'>
              Discovery of NFκB2-Coordinated Dual Regulation of Mitochondrial and Nuclear Genomes Leads to an Effective Therapy for Acute Myeloid Leukemia
              </a>
            </td>
          <td>
            Yi Xu, David J. Baylink, Jeffrey Xiao, Lily Tran, Vinh Nguyen, Brandon Park, Ismael Valladares, Scott Lee, Kevin Codorniz, Laren D Tan, Chien-Shing Chen, Hisham Abdel-Azim, Mark E. Reeves, Hamid Mirshahidi, Guido Marcucci, H. Cao
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe60321069cd8bdd4dcd7dd1f6cb9aee5e0996c" target='_blank'>
              Z-DNA formation in promoters conserved between human and mouse are associated with increased transcription reinitiation rates
              </a>
            </td>
          <td>
            N. Beknazarov, Dmitry Konovalov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-08-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background VPS72 is highly expressed in hepatocellular carcinoma and prostate cancer, participating in various cellular processes such as gene transcription, replication, DNA repair, maintenance of genome integrity, and cancer progression. However, its role in colorectal cancer remains unknown. Methods Bioinformatic methods were used to analyze gene expression, correlation and patient survival. Western blotting, colony formation assays and animal experiments were used to evaluate the function of VPS72 in colon cancer in vivo and in vitro. Results VPS72 was highly expressed in colon cancer tissues and correlated with poor overall survival (P<0.05) and relapse free survival (P<0.01). Furthermore, patients with III/IV clinical stage (P<0.001), N1 nodal metastasis (P<0.001) or N2 nodal metastasis (P<0.05) status had poor overall survival. Further analysis showed that VPS72 is correlated with proliferation and EMT biomarkers. Western blotting, colony formation assays and animal experiments showed that VPS72 overexpression promoted colon cancer proliferation and EMT progress. Conclusion Our study found that VPS72 was correlated with poor overall survival in colon cancer patients, and high expressed level of VPS72 promoted colon cancer progression, indicating its role as a potential prognosis biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f996c9ebc6898a960d942def154ad310992ed0de" target='_blank'>
              Expression Analysis of VPS72 and Associated Biological Behaviors in Colon Cancer
              </a>
            </td>
          <td>
            Junhong Cao, Hao Zhang, Xin Xie, Wei Wang
          </td>
          <td>2024-08-01</td>
          <td>International Journal of General Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background/Objectives The complex system of cancer has led to an emphasis on understanding the more general causal relationship within the disease. In this context, concepts of symmetry and symmetry-breaking in distinct biological cell features or components have been examined as an approach to cancer investigation. However, there can be possible limitations in directly interpreting the symmetry-based approach from a physical viewpoint due to the lack of understanding of physical laws governing symmetry in complex systems like cancer. Methods Fractal geometry and DNA walk representation were employed to investigate the geometric features i.e., self-similarity and heterogeneity in DNA nucleotide coding sequences of wild-type and mutated oncogenes, tumour-suppressor, and other unclassified genes. The mutation-facilitated self-similar and heterogenous features were quantified by the fractal dimension and lacunarity coefficient measures, respectively. Additionally, the geometrical orderedness and disorderedness in the analyzed sequences were interpreted from the combination of the fractal measures. Results The findings showed distinct fractal geometric features in the case of fusion mutations. It also highlights the possible interpretation of the observed fractal features as geometric analogues concerning explicit observations corresponding to specific cancer types. In addition, the two-dimensional multi-fractal analysis highlighted the presence of a single exponent in the scaling of mutation-mediated gene sequence self-similarity/complexity and heterogeneity. Conclusions The approach identified mutation-induced geometric features in gene sequences, demonstrating the potential of DNA walks and fractal analysis in translational research regarding cancer. The findings suggest that investigating fractal parameters can capture unique geometric features in nucleotide sequences, contributing to the understanding of cancer’s molecular complexity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a73efc783fc74b512f9c22058780b5d74879423" target='_blank'>
              DNA walk of specific fused oncogenes exhibit distinct fractal geometric characteristics in nucleotide patterns
              </a>
            </td>
          <td>
            Abhijeet Das, Manas Sehgal, A. Singh, Rishabh Goyal, Mallika Prabhakar, Jeremy Fricke, Isa Mambetsariev, Prakash Kulkarni, M. Jolly, Ravi Salgia
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="In this study, we investigated recurrent copy number variations (CNVs) in the 19p12 locus, which are associated with neurodevelopmental disorders. The two genes in this locus, ZNF675 and ZNF681, arose via gene duplication in primates, and their presence in several pathological CNVs in the human population suggests that either or both of these genes are required for normal human brain development. ZNF675 and ZNF681 are members of the Krüppel-associated box zinc finger (KZNF) protein family, a class of transcriptional repressors important for epigenetic silencing of specific genomic regions. About 170 primate-specific KZNFs are present in the human genome. Although KZNFs are primarily associated with repressing retrotransposon-derived DNA, evidence is emerging that they can be co-opted for other gene regulatory processes. We show that genetic deletion of ZNF675 causes developmental defects in cortical organoids, and our data suggest that part of the observed neurodevelopmental phenotype is mediated by a gene regulatory role of ZNF675 on the promoter of the neurodevelopmental gene Hes family BHLH transcription factor 1 (HES1). We also find evidence for the recently evolved regulation of genes involved in neurological disorders, microcephalin 1 and sestrin 3. We show that ZNF675 interferes with HES1 auto-inhibition, a process essential for the maintenance of neural progenitors. As a striking example of how some KZNFs have integrated into preexisting gene expression networks, these findings suggest the emergence of ZNF675 has caused a change in the balance of HES1 autoregulation. The association of ZNF675 CNV with human developmental disorders and ZNF675-mediated regulation of neurodevelopmental genes suggests that it evolved into an important factor for human brain development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2122cac7536c55e0373736baf9954110c9f4179" target='_blank'>
              Emergence of the ZNF675 Gene During Primate Evolution-Influenced Human Neurodevelopment Through Changing HES1 Autoregulation.
              </a>
            </td>
          <td>
            G. A. Lodewijk, Matthijs de Geus, Rita L F P Guimarães, F. Jacobs
          </td>
          <td>2024-07-01</td>
          <td>The Journal of comparative neurology</td>
          <td>1</td>
          <td>5</td>
        </tr>

        <tr id="Dysregulation of the tumour suppressor gene Adenomatous Polyposis Coli (APC) is a canonical step in colorectal cancer development. Curiously, most colorectal tumours carry biallelic mutations that result in only partial loss of APC function, suggesting that a ‘just-right’ level of APC inactivation, and hence Wnt signalling, provides the optimal conditions for tumorigenesis. Mutational processes act variably across the APC gene, which could contribute to the bias against complete APC inactivation. Thus the selective consequences of partial APC loss are unclear. Here we propose a mathematical model to quantify the tumorigenic effect of biallelic APC genotypes, controlling for somatic mutational processes. Analysing sequence data from >2500 colorectal cancers, we find that APC genotypes resulting in partial protein function confer about 50 times higher probability of progressing to cancer compared to complete APC inactivation. The optimal inactivation level varies with anatomical location and additional mutations of Wnt pathway regulators. We use this context dependency to assess the regulatory effect of secondary Wnt drivers in combination with APC in vivo, and provide evidence that mutant AMER1 combines with APC genotypes that lead to relatively low Wnt. The fitness landscape of APC inactivation is consistent across microsatellite unstable and POLE-deficient colorectal cancers and tumours in patients with Familial Adenomatous Polyposis suggesting a general ‘just-right’ optimum, and pointing to Wnt hyperactivation as a potential cancer vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5f474ac3dba3c99ab4d1fd2c61a492c95822b9" target='_blank'>
              Quantifying ‘just-right’ APC inactivation for colorectal cancer initiation
              </a>
            </td>
          <td>
            Meritxell Brunet Guasch, Nathalie Feeley, Ignacio Soriano, Steve Thorn, Ian Tomlinson, Michael D. Nicholson, Tibor Antal
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="DNA methylation-based indices predict mortality and provide insights into aging mechanisms. Even though sex chromosomes are thought to play an important role in sex-specific aging, they have been largely ignored in epigenetic aging studies. We identified age-related CpG (AR-CpG) sites in the full methylomes of two avian species (zebra finch and jackdaw), including for the first time the avian sex chromosomes (Z and the female-specific, haploid W). In both species, we find a high concentration of AR-CpG sites on the W chromosome, with DNA methylation levels predominantly decreasing with age. Our results highlight distinct age-related changes in sex chromosome DNA methylation compared to the rest of the genome, suggesting that this feature of sex chromosomes may contribute to sex-dependent aging.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d31690344c13aafc58b5419447fd6edd4d52dcc4" target='_blank'>
              Sex-chromosome-dependent aging in female heterogametic methylomes
              </a>
            </td>
          <td>
            Marianthi Tangili, Joanna Sudyka, Fabrício Furni, P. Palsbøll, Simon Verhulst
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7966c5016d9df57d57b899f6bbc5b0375d66e21b" target='_blank'>
              Comprehensive network modeling approaches unravel dynamic enhancer-promoter interactions across neural differentiation
              </a>
            </td>
          <td>
            William DeGroat, Fumitaka Inoue, Tal Ashuach, N. Yosef, N. Ahituv, Anat Kreimer
          </td>
          <td>2024-08-14</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>60</td>
        </tr>

        <tr id="Phenotypic variability in isogenic bacterial populations helps them to cope with external stress. This variability can stem from gene expression noise and/or unequal partitioning of low-copy-number freely diffusing proteins during cell division. Some high-copy-number components are transiently associated with almost immobile large assemblies (hyperstructures) and thereby may not be distributed equally. We hypothesize that hyperstructure organization of bacterial cells can contribute to bacterial phenotypic variability. We focus on nucleoid hyperstructure containing many DNA-associated proteins. Our previous findings suggested that replication initiation timing may be the main source of variability for nucleoid segregation events. To uncover the role of the initiator DnaA in phenotypic variability in E. coli, we followed its intra- and intercellular distribution using chromosomally encoded fluorescently tagged DnaA and histone-like HU under their native promoters. The mobility of DnaA is low, corresponding to a diffusion-binding mode, but still one order faster than that of HU. The intracellular distribution of DnaA concentration is homogeneous in contrast to the significant asymmetry in the distribution of HU; the latter leading to unequal DNA content of nucleoids in future daughter compartments. The intercellular variability of HU (CV=26%), DnaA, and their ratio (both with CV=18%) is high. The variable DnaA/DNA may cause a variable replication initiation time (initiation noise). Asynchronous initiation at different oriCs may be, in turn, a plausible origin of the asymmetric DNA distribution. We specify that the DnaA/DNA ratio may be a more rigorous feature in determining the variability of initiation time in E. coli than each of them separately. Importance Phenotypic variability is an important phenomenon of bacterial physiology related to adaptation to environmental stresses and antibiotic resistance, not yet completely understood. This work is a continued examination of our hypothesis of the role of hyperstructure organization of bacterial cells in this phenomenon, focusing on the nucleoid hyperstructure and the DNA-associated replication initiator DnaA. Exploiting fluorescently tagged proteins, we analyze intra- and intercellular distributions, mobility, and concentration variabilities of DnaA and DNA in the population of Escherichia coli. The data obtained indicate that the variable DnaA/DNA ratio may serve as an essential source of variability of the replication initiation event and the ensuing chromosome segregation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2d2069c6e7f009db748eca4a82f0ae5bcf52f6c" target='_blank'>
              Relative Distribution of DnaA and DNA in Escherichia coli Cells as a Factor of Their Phenotypic Variability
              </a>
            </td>
          <td>
            Sharanya K. Namboodiri, A. Aranovich, U. Hadad, Levi A. Gheber, Mario Feingold, I. Fishov
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Aneuploidy is nearly ubiquitous in tumor genomes, but the role of aneuploidy in the early stages of cancer evolution remains unclear. Here, by inducing heterogeneous aneuploidy in non-transformed human colon organoids (colonoids), we investigated how the effects of aneuploidy on cell growth and differentiation may promote malignant transformation. Previous work implicated p53 activation as a downstream response to aneuploidy induction. We found that simple aneuploidy, characterized by 1-3 gained or lost chromosomes, resulted in little or modest p53 activation and cell cycle arrest when compared with more complex aneuploid cells. Single-cell RNA sequencing analysis revealed that the degree of p53 activation was strongly correlated with karyotype complexity. Single-cell tracking showed that cells could continue to divide despite the observation of one to a few lagging chromosomes. Unexpectedly, colonoids with simple aneuploidy exhibited impaired differentiation after niche factor withdrawal. These findings demonstrate that simple aneuploid cells can escape p53 surveillance and may contribute to niche factor-independent growth of cancer-initiating colon stem cells. [Media: see text].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69141397c86397dd6c1f0121be2fa0ac82f1c38" target='_blank'>
              Simple aneuploidy evades p53 surveillance and promotes niche factor-independent growth in human intestinal organoids.
              </a>
            </td>
          <td>
            Blake A. Johnson, Albert Z Liu, Tianhao Bi, Yi Dong, Taibo Li, Dingjingyu Zhou, A. Narkar, Yufei Wu, Sean X. Sun, Tatianna C. Larman, Jin Zhu, Rong Li
          </td>
          <td>2024-07-10</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35393ed7857060735a43bccb7ec1795bc1e251a9" target='_blank'>
              Comprehensive molecular characterization of TFE3-rearranged renal cell carcinoma.
              </a>
            </td>
          <td>
            Cho-Rong Lee, J. Suh, Dongjun Jang, Bo-Yeong Jin, Jaeso Cho, Moses Lee, Hyungtai Sim, Minyong Kang, Jueun Lee, Ju Hyun Park, Kyoung-Hwa Lee, Geum-Sook Hwang, Kyung Chul Moon, Cheryn Song, J. Ku, C. Kwak, Hyeon-Hoe Kim, Sung-Yup Cho, Murim Choi, Chang Wook Jeong
          </td>
          <td>2024-08-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers. This phenomenon has been investigated since the late 1980s without resolution. Expanding beyond previous gene-centric studies, we investigated the co-occurrence in a genome-wide manner taking an evolutionary perspective. Mining of large-scale tumor aneuploidy data confirmed the previous finding of a small-scale longitudinal study that the most likely order is chromosome 10 loss followed by chromosome 7 gain. Extensive analysis of genomic and transcriptomic data from both patients and cell lines revealed that this co-occurrence can be explained by functional rescue interactions that are highly enriched on chromosome 7, which could potentially compensate for any detrimental consequences arising from the loss of chromosome 10. Transcriptomic data from various normal, non-cancerous human brain tissues were analyzed to assess which tissues may be most predisposed to tolerate compensation of chromosome 10 loss by chromosome 7 gain. The analysis indicated that the pre-existing transcriptomic states in the cortex and frontal cortex, where gliomas arise, are more favorable than other brain regions for compensation by rescuer genes that are active on chromosome 7. Collectively, these findings suggest that the phenomenon of chromosome 10 loss and chromosome 7 gain in gliomas is orchestrated by a complex interaction of many genes residing within these two chromosomes and provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff13af5a4c0be0182481407e6fc3feb597b938be" target='_blank'>
              Chromosome 7 gain compensates for chromosome 10 loss in glioma.
              </a>
            </td>
          <td>
            N. Nair, Alejandro A Schaffer, E. Gertz, Kuoyuan Cheng, Johanna Zerbib, A. Sahu, Gil Leor, E. Shulman, Kenneth D Aldape, Uri Ben-David, E. Ruppin
          </td>
          <td>2024-07-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="The p53 family remains a captivating focus of an extensive number of current studies. Accumulating evidence indicates that p53 abnormalities rank among the most prevalent in cancer. Given the numerous existing studies, which mostly focus on the mutations, expression profiles, and functional perturbations exhibited by members of the p53 family across diverse malignancies, this review will concentrate more on less explored facets regarding p53 activation and stabilization by the nuclear pore complex (NPC) in cancer, drawing on several studies. p53 integrates a broad spectrum of signals and is subject to diverse regulatory mechanisms to enact the necessary cellular response. It is widely acknowledged that each stage of p53 regulation, from synthesis to degradation, significantly influences its functionality in executing specific tasks. Over recent decades, a large body of data has established that mechanisms of regulation, closely linked with protein activation and stabilization, involve intricate interactions with various cellular components. These often transcend canonical regulatory pathways. This new knowledge has expanded from the regulation of genes themselves to epigenomics and proteomics, whereby interaction partners increase in number and complexity compared with earlier paradigms. Specifically, studies have recently shown the involvement of the NPC protein in such complex interactions, underscoring the further complexity of p53 regulation. Furthermore, we also discuss therapeutic strategies based on recent developments in this field in combination with established targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f4ce5cb3bd6bcbf7187f9f514ca001d925085c" target='_blank'>
              Unlocking the Gateway: The Spatio-Temporal Dynamics of the p53 Family Driven by the Nuclear Pores and Its Implication for the Therapeutic Approach in Cancer
              </a>
            </td>
          <td>
            Dini Kurnia Ikliptikawati, Keiichi Makiyama, M. Hazawa, Richard W. Wong
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/69e1db6874aa879e7cfcdc2ee3255c993cf70dae" target='_blank'>
              USP9X regulates the proliferation, survival, migration and invasion of gastric cancer cells by stabilizing MTH1
              </a>
            </td>
          <td>
            Wenji Xu, Yaping Zhang, Yingrui Su, Libin Li, Xinxia Yang, Lixing Wang, Hongzhi Gao
          </td>
          <td>2024-07-29</td>
          <td>BMC Gastroenterology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3e129666ae56c033aa207d599e42711f40eae8" target='_blank'>
              Gene body DNA hydroxymethylation restricts the magnitude of transcriptional changes during aging
              </a>
            </td>
          <td>
            James R. Occean, Na Yang, Yan Sun, Marshall S. Dawkins, R. Munk, Cedric Belair, Showkat A. Dar, C. Anerillas, Lin Wang, Changyou Shi, Christopher Dunn, Michel Bernier, Nathan L. Price, Julie S Kim, Chang-Yi Cui, Jinshui Fan, Moitrayee Bhattacharyya, Supriyo De, Manolis Maragkakis, R. de Cabo, Simone Sidoli, P. Sen
          </td>
          <td>2024-07-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/702c4dbd707857d617c2885be2e2c156f92086ef" target='_blank'>
              NXPH4 mediated by m5C contributes to the malignant characteristics of colorectal cancer via inhibiting HIF1A degradation
              </a>
            </td>
          <td>
            Lei Yang, Jiawen Shi, Mingyang Zhong, Pingping Sun, Xiaojing Zhang, Zhengyi Lian, Hang Yin, Lijun Xu, Guying He, Haiyan Xu, Han Wu, Ziheng Wang, Kai Miao, Jianfei Huang
          </td>
          <td>2024-08-20</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="CONTEXT
Hyperinsulinemic hypoglycemia (HI) can be the presenting feature of Kabuki syndrome (KS), which is caused by loss-of-function variants in KMT2D or KDM6A. As these genes play a critical role in maintaining methylation status in chromatin, individuals with pathogenic variants have a disease-specific epigenomic profile -an episignature.


OBJECTIVE
We evaluated the pathogenicity of three novel partial KDM6A duplications identified in three individuals presenting with neonatal-onset HI without typical features of KS at the time of genetic testing.


METHODS
Three different partial KDM6A duplications were identified by routine targeted next generation sequencing for HI and initially classified as variants of uncertain significance (VUS) as their location, and hence their impact on the gene, was not known. Whole genome sequencing (WGS) was undertaken to map the breakpoints of the duplications with DNA methylation profiling performed in two individuals to investigate the presence of a KS-specific episignature.


RESULTS
WGS confirmed the duplication in proband 1 as pathogenic as it caused a frameshift in the normal copy of the gene leading to a premature termination codon. The duplications identified in probands 2 and 3 did not alter the reading frame and therefore their significance remained uncertain after WGS. Subsequent DNA methylation profiling identified a KS-specific episignature in proband 2 but not in proband 3.


CONCLUSIONS
Our findings confirm a role for KDM6A partial gene duplications in the etiology of KS and highlight the importance of performing in-depth molecular genetic analysis to properly assess the clinical significance of VUS's in the KDM6A gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d408f1c2defab4001206066e972ac0d962f9c62" target='_blank'>
              Congenital hyperinsulinism and novel KDM6A duplications -resolving pathogenicity with genome and epigenetic analyses.
              </a>
            </td>
          <td>
            Jonna M E Männistö, J. J. Hopkins, Thomas I. Hewat, Fatima Nasser, Joseph Burrage, A. Dastamani, A. Mirante, Nuala Murphy, J. Rzasa, J. Kerkhof, R. Relator, Matthew B. Johnson, T. Laver, Luke Weymouth, J. Houghton, M. Wakeling, B. Sadikovic, Emma L Dempster, Sarah E. Flanagan
          </td>
          <td>2024-07-30</td>
          <td>The Journal of clinical endocrinology and metabolism</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcf3e13b55bc25cd806bf0acc955517dc375f3ab" target='_blank'>
              CircRNA as an Achilles heel of cancer: characterization, biomarker and therapeutic modalities
              </a>
            </td>
          <td>
            Jun Zhang, Zai Luo, Yang Zheng, Mingyu Duan, Zheng-Jun Qiu, Chen Huang
          </td>
          <td>2024-08-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/966f20b4115e34215d01f56c4a256d96a9d46af1" target='_blank'>
              Mosaic variegated aneuploidy in development, ageing and cancer.
              </a>
            </td>
          <td>
            Marcos Malumbres, Carolina Villarroya-Beltri
          </td>
          <td>2024-08-21</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d65d81f0ff12009e99b264502d06073751ad5efb" target='_blank'>
              The role of distinct APOBEC/ADAR mRNA levels in mutational signatures linked to aging and ultraviolet radiation
              </a>
            </td>
          <td>
            Ahmadreza Niavarani
          </td>
          <td>2024-07-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Tracing cell lineages has become a valuable tool for studying biological processes. Among the available tools for human data, mitochondria DNA (mtDNA) has a high potential due to its ability to be used in conjunction with single-cell chromatin accessibility data, giving access to the cell phenotype. Nonetheless, the existing mutation calling tools are ill-equipped to deal with the polyploid nature of the mtDNA and lack a robust statistical framework. Here we introduce scMitoMut, an innovative R package that leverages statistical methodologies to accurately identify mitochondrial lineage related mutations at the single-cell level. scMitoMut assigns a mutation quality q-value based on beta-binomial distribution to each mutation at each locus within individual cells, ensuring higher sensitivity and precision of lineage related mutation calling in comparison to current methodologies. We tested scMitoMut using single-cell DNA sequencing, scATAC sequencing and 10× Genomics single cell multiome datasets. Using a single-cell DNA sequencing dataset from a mixed population of cell lines, scMitoMut demonstrated superior sensitivity in identifying small proportion of cancer cell lines compared to existing methods. In a human colorectal cancer scATAC dataset, scMitoMut identified more mutations than state-of-the-art methods. Applied to 10× Genomics multiome datasets, scMitoMut effectively measured the lineage distance in cells from blood or brain tissues. Thus, the scMitoMut is a free available (https://www.bioconductor.org/packages/devel/bioc/html/scMitoMut.html.), well-engineered toolkit for mtDNA mutation calling with high memory and CPU efficiency. Consequently, it will significantly advance the application of single-cell sequencing, facilitating the precise delineation of mitochondrial mutations for lineage tracing purposes in development, tumor and stem cell biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a057cdbd6d7399eed5dec23b92e83e85fe7d0c9" target='_blank'>
              scMitoMut for calling mitochondrial lineage–related mutations in single cells
              </a>
            </td>
          <td>
            Wenjie Sun, Daphne van Ginneken, Leïla Perié
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a1f5936217eac5b4043bb1665424b2339a9f20d" target='_blank'>
              MYC and KRAS cooperation: from historical challenges to therapeutic opportunities in cancer
              </a>
            </td>
          <td>
            S. Casacuberta-Serra, Íñigo González-Larreategui, Daniel Capitán-Leo, Laura Soucek
          </td>
          <td>2024-08-21</td>
          <td>Signal Transduction and Targeted Therapy</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ad718715d0e23f67c54707d35f7ee63dc651a7" target='_blank'>
              Homeotic DUX4 Genes Shape Dynamic Inter-Chromosomal Contacts with Nucleoli in Human Cells.
              </a>
            </td>
          <td>
            E. S. Klushevskaya, I. R. Alembekov, Y. V. Kravatsky, N. A. Tchurikov
          </td>
          <td>2024-07-13</td>
          <td>Doklady. Biochemistry and biophysics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Background Cancer variability among patients underscores the need for personalized therapy based on genomic understanding. BRCAness, characterized by vulnerabilities similar to BRCA mutations, particularly in homologous recombination repair, shows potential sensitivity to DNA-damaging agents like PARP inhibitors, highlighting it’s clinical significance. Methods We employed Graph Convolutional Neural Networks (GCNNs) with Layer-wise Relevance Propagation (LRP) to analyze gene expression data from the TCGA Pan-Cancer dataset. The study compared the efficacy of GCNNs against traditional machine learning models and differential gene expression analysis, focusing on their ability to elucidate complex genomic interactions defining BRCAness. Results Differential Gene Expression (DGE) analysis proved limited in capturing the nuances of BRCAness. In contrast, GLRP significantly identified genes related to transcription regulation and cancer processes, emphasizing the phenotype’s complexity. Gene Set Enrichment Analysis (GSEA) highlighted crucial pathways like Nuclear Receptors signaling, Cellular Senescence, and ESR-mediated signaling, underscoring their roles in BRCAness and therapeutic potential. Conclusion GLRP outperformed traditional approaches in analyzing BRCAness, providing deep insights into transcriptional and oncogenic processes critical to the BRCAness phenotype. Our findings suggest new directions for developing targeted and personalized cancer treatments, leveraging intricate molecular interactions associated with BRCAness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f261eb31af18dda214096a2121806c4281467bcd" target='_blank'>
              Deciphering BRCAness Phenotype in Cancer: A Graph Convolutional Neural Network Approach with Layer-wise Relevance Propagation Analysis
              </a>
            </td>
          <td>
            Jingyu Yang, Hryhorii Chereda, Jürgen Dönitz, Annalen Bleckmann, Tim Beißbarth
          </td>
          <td>2024-08-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved DNA repair system, in which aberrations can lead to the accumulation of DNA damage and genomic scars known as homologous recombination deficiency (HRD). The identification of mutations in key genes (i.e., BRCA1, and BRCA2 (BRCA)) and the quantification of large-scale structural variants (e.g., loss of heterozygosity) are indicators of the HRD phenotype. HRD is a stable biomarker and remains unchanged during recurrence, but fails to reveal the molecular profile of tumor progression. Moreover, interpretation of the current HRD score lacks comprehensiveness, especially for the HR-proficient group. Poly (ADP-ribose) polymerase (PARP) enzymes play an important role in the repair of DNA single-strand breaks, the blockage of which using PARP inhibitors (PARPi) can generate synthetic lethality in cancer cells with HRD. Although numerous studies have demonstrated that the benefit of PARPi is substantial in ovarian cancer (OC) patients, the efficacy is limited by the development of resistance, and seems to be irrespective of HR and/or BRCA mutation status. Moreover, in addition to improving progression-free survival, long-term benefit as overall survival brought by PARPi for advanced, recurrent and refractory OC patients remains unclear. Therefore, further investigations are needed to uncover the role of HR genes beyond BRCA and their interactions with other oncogenic pathways, to determine the value of HRD in the recurrent setting, and to identify alternative strategies for the precise management of advanced, refractory OC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fe4c9f0bc408e017d25339a0121e8f087ab765" target='_blank'>
              Limitations of homologous recombination status testing and poly (ADP-ribose) polymerase inhibitor treatment in the current management of ovarian cancer
              </a>
            </td>
          <td>
            Qianying Zhao, Liping Bai, Yu Tan, Mingrong Qie
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3ef3973d270e32f48aaa19a00bef4e18bca5be" target='_blank'>
              Research Progress of Long Non-coding RNA-ZFAS1 in Malignant Tumors.
              </a>
            </td>
          <td>
            Xin Liu, Zhong Ma, Xianxu Zhang, Shicheng Li, Jiangdong An, Zhiqiang Luo
          </td>
          <td>2024-07-26</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87375be953b6b1e62c0930938d4b8c30692440ac" target='_blank'>
              Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants.
              </a>
            </td>
          <td>
            Max Frenkel, James E. Corban, Margaux L. A. Hujoel, Zachary Morris, Srivatsan Raman
          </td>
          <td>2024-07-24</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chromatin states, fundamental to gene regulation and cellular identity, are defined by a unique combination of histone post-translational modifications. Despite their importance, comprehensive patterns within chromatin state sequences, which could provide insights into key biological functions, remain largely unexplored. In this study, we introduce ChromBERT, a BERT-based model specifically designed to detect distinct patterns of chromatin state annotation data sequences. Notably, ChromBERT was pre-trained on promoter regions across a diverse range of epigenomes and subsequently fine-tuned using a dataset from multiple cell lines where RNA-seq data were available, highlighting the model’s ability to discern conserved chromatin state patterns within these regions. In addition to its predictive powers across tasks, evidenced by high AUC scores, ChromBERT provides further analysis through the incorporation of motif clustering using Dynamic Time Warping (DTW). This method enhances the model’s ability to dissect chromatin state sequence motifs, typically involving transcription and enhancer sites. The introduction of motif clustering with DTW into ChromBERT’s workflow is poised to facilitate the discovery of genomic regions linked to novel biological functions, deepening our understanding of chromatin state dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758a0c4bd29ca771e1573835ff87961f8849bdb7" target='_blank'>
              ChromBERT: Uncovering Chromatin State Motifs in the Human Genome Using a BERT-based Approach
              </a>
            </td>
          <td>
            Seohyun Lee, Che Lin, Chien-Yu Chen, Ryuichiro Nakato
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Telomeres are specialized structures at the ends of linear chromosomes that protect them from degradation and fusion. It’s replication is a complex process that involves both DNA polymerases and a specialized enzyme called telomerase which is a ribonucleoprotein complex that synthesizes telomeric DNA by using an internal RNA template. However, telomerase alone cannot fully replicate the telomeric DNA, and requires the cooperation of other factors, such as shelterin, CST, and DNA repair proteins. Telomere replication is tightly regulated by various mechanisms, such as cell cycle checkpoints, telomere length homeostasis, and telomere position effect and dysregulation of it can lead to genomic instability, cellular senescence, and cancer. Therefore, understanding the molecular details of telomere replication is crucial for elucidating the role of telomeres in aging and disease. ">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e590a2bd1420d5e3b8ed0e5debba5cfdd475b0f8" target='_blank'>
              Unravelling the Intricacies of Telomere Replication: A Molecular Conundrum
              </a>
            </td>
          <td>
            Neelabh Datta
          </td>
          <td>2024-07-16</td>
          <td>Canadian Journal for the Academic Mind</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Sequencing mitochondrial DNA (mtDNA) variants from single cells has resolved clonality and lineage in native human samples and clinical specimens. Prior work established that heteroplasmic mtDNA variants can be used to delineate clonality in hematopoiesis, but they have limited ability to reconstruct cellular phylogenies. However, a recent report by Weng et al. challenges the current paradigm by describing an unprecedented number of shared mtDNA variants between cells that reportedly resolve high-resolution phylogenetic trees. We re-examined the claims of Weng et al., and identified two major points of concern regarding this unprecedented connectedness. First, shared variants between cells are disproportionately detected in a single molecule per cell, and second, these variants are enriched 10–20-fold at the edges of mtDNA molecules, reminiscent of artifacts reported in other sequencing approaches. Further, our analyses show that pruning low support and likely artificial mtDNA variants removes nearly all of the reported phylogenetic structure. Thus, we strongly caution against using mtDNA variant calling workflows that rely on minimal evidence, including the computational pipeline introduced in Weng et al., as variants with high connectedness and low evidence are likely artifacts that lead to the construction of false phylogenies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae6d387965c56c26ee411bc842b91f11e043e8f3" target='_blank'>
              Artifacts in single-cell mitochondrial DNA mutation analyses misinform phylogenetic inference
              </a>
            </td>
          <td>
            Caleb A. Lareau, M. Chapman, Livius Penter, T. Nawy, D. Pe’er, Leif S. Ludwig
          </td>
          <td>2024-07-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Turner syndrome (TS) is caused by a complete or partial absence of an X or Y chromosome, including chromosomal mosaicism, affecting 1 in 2500 female live births. Sister chromatid exchange (SCE) is used as a sensitive indicator of spontaneous chromosome instability. Cells from mosaic patients constitute useful material for SCE evaluations as they grow under the influence of the same genetic background and endogenous and exogenous factors. We evaluated the proliferation dynamics and SCE frequencies of 45,X and 46,XN cells of 17 mosaic TS patients. In two participants, the 45,X cells exhibited a proliferative disadvantage in relation to 46,XN cells after 72 h of cultivation. The analysis of the mean proliferation index (PI) showed a trend for a significant difference between the 45,X and 46,X+der(X)/der(Y) cell lineages; however, there were no intra-individual differences. On the other hand, mean SCE frequencies showed that 46,X+der(X) had the highest mean value and 46,XX the lowest, with 45,X occupying an intermediate position among the lineages found in at least three participants; moreover, there were intra-individual differences in five patients. Although 46,X+der(X)/der(Y) cell lineages, found in more than 70% of participants, were the most unstable, they had a slightly higher mean PI than the 45,X cell lineages in younger (≤17 years) mosaic TS participants. This suggests that cells with a karyotype distinct from 45,X may increase with time in mosaic TS children and adolescents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8a35420c99ba97fe4f5acd7a759bcc6a732cb8d1" target='_blank'>
              Cell Cycle Kinetics and Sister Chromatid Exchange in Mosaic Turner Syndrome
              </a>
            </td>
          <td>
            M. B. Goulart, Eduardo Vieira Neto, Daniela R. Ney Garcia, Marília Martins Guimarães, Isaías Soares de Paiva, Karina de Ferran, Nathalia Correia Krause dos Santos, Luciana Santos Barbosa, Amanda F. de Figueiredo, M. Ribeiro, Márcia Gonçalves Ribeiro
          </td>
          <td>2024-07-01</td>
          <td>Life</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="All cell lineages accumulate mutations over time, increasing the probability that some lineages eventually become malignant. Many of the processes responsible for generating mutations are leaving a characteristic footprint in the genome that allows their presence to be detected. However, the mutational pattern in a tumor is usually the combined result of multiple mutational processes being at work simultaneously, and the problem of disentangling the different footprints and their relative impact then becomes a deconvolution problem. Several algorithms have been developed for this purpose, most of them involving a factorization of the mutation count matrix into two nonnegative matrices, representing respectively the underlying mutational signatures and the relative weighting of (or exposure to) these signatures. Here, we introduce the StarSignDNA algorithm for mutational signature analysis, which offers efficient refitting and de novo mutational signature extraction. StarSignDNA is capable of deciphering well-differentiated signatures linked to known mutagenic mechanisms and suggesting clinically relevant predictions for a single patient. The package offers a command line-based interface and data visualization routines. Author summary StarSignDNA is a novel algorithm for identifying mutational signatures from cancer sequencing data. It excels in low mutation count scenarios, balancing signature detection and true signature discovery. StarSignDNA improves prediction accuracy and biological validity by addressing challenges such as overfitting and underfitting, achieving optimal variable selection and shrinkage, and ensuring interpretability. In refitting, it reduces prediction variance and handle sparsity, resulting in more precise predictions. In de novo extraction, it improves the detection of challenging signatures and achieves better alignment between detected and known signatures through unsupervisedcross-validation. Its unique features, including prediction confidence and customizable reference signatures, offer valuable insights for clinical single-sample analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a5a94297c1c9cbe8ea0fed782c76348b449df01" target='_blank'>
              StarSignDNA: Signature tracing for accurate representation of mutational processes
              </a>
            </td>
          <td>
            C. Bope, Sumana Kalyanasundaram, Knut D. Rand, S. Nakken, O. Lingjærde, E. Hovig
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>52</td>
        </tr>

        <tr id="The endonuclease activity of Pms1 directs mismatch repair by generating a nick in the newly replicated DNA strand. Inactivating Pms2, the human homologue of yeast Pms1, increases the chances of colorectal and uterine cancers. Here we use whole genome sequencing to show that loss of this endonuclease activity, via the pms1-DE variant, results in strong mutator effects throughout the Saccharomyces cerevisiae genome. Mutation rates are strongly increased for mutations resulting from all types of single-base substitutions and for a wide variety of single- and multi-base indel mutations. Rates for these events are further increased in strains combining pms1-DE with mutator variants of each of the three major leading and lagging strand replicases. In all cases, mutation rates, spectra, biases, and context preferences are statistically indistinguishable from strains with equivalent polymerases but lacking initial mismatch recognition due to deletion of MSH2. This implies that, across the nuclear genome, strand discrimination via the Pms1 endonuclease is as important for MMR as is initial mismatch recognition by Msh2 heterodimers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93c81863bf8b4bcd0533ef4d09eddc5a93c0da4b" target='_blank'>
              Instability throughout the Saccharomyces cerevisiae genome resulting from Pms1 endonuclease deficiency.
              </a>
            </td>
          <td>
            S. Lujan, Marta A. Garbacz, S. E. Liberti, Adam B Burkholder, Thomas A. Kunkel
          </td>
          <td>2024-07-17</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Epigenetics are the changes due to DNA methylation, histone modifications, and noncoding RNA regulation, which play a crucial part in the expression of genes. Although these changes do not alter the sequence of the DNA, they are significant in determining the phenotype and inheritance of the genes. The impact of epigenetics on inherited diseases like Fragile X syndrome, Angelman syndrome, and Prader-Willi syndrome emphasizes the significance of studying epigenetic dysregulation in disease pathology. The current review aims to investigate the role of epigenetic mechanisms in inherited diseases and to evaluate the utility of molecular diagnostic tools for epigenetic analysis in disease diagnosis. Case studies of diseases such as Duchenne muscular dystrophy and Prader-Willi syndrome highlight the clinical relevance of epigenetic analysis in disease diagnosis and management. This research is conducted through an extensive analysis of recent literature to explore the epigenetic mechanisms underlying inherited diseases. Additionally, advanced molecular diagnostic techniques like droplet digital PCR (ddPCR), transposase-based bisulfite tagging, and data mining coupled with conventional experimental procedures. However, the intricate relationship between genetic and epigenetic factors and challenges related to inclined data need to be addressed. Future research endeavors to elucidate the role of epigenetic modifications in disease pathogenesis and develop strategies for personalized medicinal treatment and therapeutic intervention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9f24f0960efd210e8ee51cbad54070948fecc2ef" target='_blank'>
              A Review on Epigenetics of Human Inherited Diseases: Molecular Diagnosis
              </a>
            </td>
          <td>
            Bibi Shazia Sabir
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Innovative Science and Research Technology (IJISRT)</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3101f7048ced189804d5ffb2bbda06ceae417ac" target='_blank'>
              CRISPR-Cas9-mediated deletion enhancer of MECOM play a tumor suppressor role in ovarian cancer.
              </a>
            </td>
          <td>
            Yujie Chen, Qiuwen Jiang, Yingzhuo Xue, Weiguan Chen, Minhui Hua
          </td>
          <td>2024-07-12</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background: Prostate cancer, a common malignancy, is driven by androgen receptor (AR) signaling. Understanding the function of AR signaling is critical for prostate cancer research. Methods: We performed multi-omics data analysis for the AR+, androgen-sensitive LNCaP cell line, focusing on gene expression (RNAseq), chromatin accessibility (ATACseq), and transcription factor binding (ChIPseq). High-quality datasets were curated from public repositories and processed using state-of-the-art bioinformatics tools. Results: Our analysis identified 1004 up-regulated and 707 down-regulated genes in response to androgen deprivation therapy (ADT) which diminished AR signaling activity. Gene-set enrichment analysis revealed that AR signaling influences pathways related to neuron differentiation, cell adhesion, P53 signaling, and inflammation. ATACseq and ChIPseq data demonstrated that as a transcription factor, AR primarily binds to distal enhancers, influencing chromatin modifications without affecting proximal promoter regions. In addition, the AR-induced genes maintained higher active chromatin states than AR-inhibited genes, even under ADT conditions. Furthermore, ADT did not directly induce neuroendocrine differentiation in LNCaP cells, suggesting a complex mechanism behind neuroendocrine prostate cancer development. In addition, a publicly available online application LNCaP-ADT (https://pcatools.shinyapps.io/shinyADT/) was launched for users to visualize and browse data generated by this study. Conclusion: This study provides a comprehensive multi-omics dataset, elucidating the role of AR signaling in prostate cancer at the transcriptomic and epigenomic levels. The reprocessed data is publicly available, offering a valuable resource for future prostate cancer research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d7223e61e9e31ea8bc035f5350cb65220e6b63d2" target='_blank'>
              Systematic Multi-Omics Investigation of Androgen Receptor Driven Gene Expression and Epigenetics changes in Prostate Cancer
              </a>
            </td>
          <td>
            Lin Li, Kyung Hyun Cho, Xiuping Yu, Siyuan Cheng
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec9eb84ca8c610e2b08f8d4ce038f9c0401c702" target='_blank'>
              De novo variants in the RNU4-2 snRNA cause a frequent neurodevelopmental syndrome
              </a>
            </td>
          <td>
            Yuyang Chen, R. Dawes, Hyung Chul Kim, A. Ljungdahl, S. Stenton, Susan Walker, Jenny Lord, Gabrielle Lemire, Alexandra C Martin-Geary, Vijay S Ganesh, Jialan Ma, J. Ellingford, E. Delage, Elston N. D’Souza, Shan Dong, David R Adams, Kirsten Allan, Madhura Bakshi, Erin E Baldwin, Seth I Berger, Jonathan A Bernstein, Ishita Bhatnagar, Ed Blair, Natasha J Brown, Lindsay C Burrage, Kimberly Chapman, David J Coman, A. Compton, Chloe A Cunningham, Precilla D’Souza, P. Danecek, E. Délot, Kerith-Rae Dias, Ellen R Elias, F. Elmslie, Carey-Anne Evans, L. Ewans, Kimberly M Ezell, Jamie L Fraser, L. Gallacher, C. Genetti, A. Goriely, Christina L Grant, Tobias B Haack, Jenny E Higgs, A. Hinch, M. Hurles, Alma Kuechler, Katherine L Lachlan, S. Lalani, F. Lecoquierre, E. Leitão, A. L. Fevre, R. Leventer, J. Liebelt, S. Lindsay, Paul J Lockhart, Alan S Ma, Ellen F. Macnamara, Sahar Mansour, Taylor M Maurer, Hector R Mendez, Kay Metcalfe, Stephen B. Montgomery, Mariya Moosajee, M. Nassogne, Serena Neumann, Michael O'Donoghue, Melanie O'Leary, E. E. Palmer, N. Pattani, John Phillips, Georgia Pitsava, Ryan Pysar, Heidi L. Rehm, Chloe M. Reuter, N. Revencu, A. Riess, Rocio Rius, Lance Rodan, T. Roscioli, Jill A. Rosenfeld, R. Sachdev, Charles J Shaw-Smith, Cas Simons, Sanjay M. Sisodiya, Penny Snell, Laura St Clair, Zornitza Stark, Helen Stewart, T. Tan, Natalie B Tan, Suzanna E L Temple, D. Thorburn, C. Tifft, E. Uebergang, G. VanNoy, Pradeep Vasudevan, Eric Vilain, David Viskochil, Laura Wedd, M. Wheeler, Susan M. White, Monica Wojcik, Lynne A Wolfe, Zoe Wolfenson, Caroline F Wright, Changrui Xiao, David Zocche, J. L. Rubenstein, Eirene Markenscoff-Papadimitriou, S. Fica, Diana Baralle, C. Depienne, D. MacArthur, Joanna M M Howson, Stephan J. Sanders, A. O’Donnell-Luria, Nicola Whiffin
          </td>
          <td>2024-07-11</td>
          <td>Nature</td>
          <td>1</td>
          <td>107</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70f393647e59fb1238cccba92da569418e311d5e" target='_blank'>
              Deciphering the genetics and mechanisms of predisposition to multiple myeloma
              </a>
            </td>
          <td>
            M. Went, Laura Duran-Lozano, G. Halldorsson, Andrea Gunnell, Nerea Ugidos-Damboriena, P. Law, Ludvig Ekdahl, Amit Sud, G. Thorleifsson, Malte Thodberg, T. Olafsdottir, Antton Lamarca-Arrizabalaga, C. Cafaro, Abhishek Niroula, Ram Ajore, Aitzkoa Lopez de Lapuente Portilla, Zain Ali, M. Pertesi, Hartmut Goldschmidt, L. Stefánsdóttir, S. Kristinsson, S. Stacey, T. Love, Saemundur Rognvaldsson, R. Hájek, Pavel Vodicka, Ulrika Pettersson-Kymmer, Florentin Späth, C. Schinke, F. van Rhee, P. Sulem, Egil Ferkingstad, Grimur Hjorleifsson Eldjarn, U. Mellqvist, I. Jónsdóttir, Gareth Morgan, P. Sonneveld, Anders Waage, N. Weinhold, H. Thomsen, A. Försti, Markus Hansson, Annette Juul-Vangsted, U. Thorsteinsdóttir, Kari Hemminki, M. Kaiser, T. Rafnar, Kári Stefánsson, R. Houlston, Björn Nilsson
          </td>
          <td>2024-08-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>182</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec67b6cd2c5b516c2d3f922caad0a44789a5bdd" target='_blank'>
              TRF1 and TRF2: pioneering targets in telomere-based cancer therapy
              </a>
            </td>
          <td>
            Anoop Kallingal, Radosław Krzemieniecki, Natalia Maciejewska, Wioletta Brankiewicz-Kopcińska, Maciej Baginski
          </td>
          <td>2024-07-16</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A significant number of the genetic alterations observed in cancer patients lie within nonprotein-coding segments of the genome, including regions coding for long noncoding RNAs (lncRNAs). LncRNAs display aberrant expression in breast cancer (BrCa), but the functional implications of this altered expression remain to be elucidated. By performing transcriptome screen in a triple negative BrCa (TNBC) isogenic 2D and 3D spheroid model, we observed aberrant expression of >1000 lncRNAs during BrCa progression. The chromatin-associated lncRNA MANCR shows elevated expression in metastatic TNBC. MANCR is upregulated in response to cellular stress and modulates DNA repair and cell proliferation. MANCR promotes metastasis as MANCR-depleted cells show reduced cell migration, invasion, and wound healing in vitro, and reduced metastatic lung colonization in xenograft experiments in vivo. Transcriptome analyses reveal that MANCR modulates expression and pre-mRNA splicing of genes, controlling DNA repair and checkpoint response. MANCR promotes the transcription of NET1A, a Rho-GEF that regulates DNA damage checkpoint and metastatic processes in cis, by differential promoter usage. Experiments suggest that MANCR regulates the expression of cancer-associated genes by modulating the association of various transcription factors and RNA-binding proteins. Our results identified the metastasis-promoting activities of MANCR in TNBC by cis-regulation of gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917a434f9bc31bbdd4efde82c1a4158058876264" target='_blank'>
              MANCR lncRNA Modulates Cell-Cycle Progression and Metastasis by Cis-Regulation of Nuclear Rho-GEF.
              </a>
            </td>
          <td>
            D. K. Singh, Zhengmin Cong, Y. Song, Minxue Liu, Ritu Chaudhary, Dazhen Liu, Yu Wang, Rishabh Prasanth, Rajendra K C, Simon Lizarazo, Miriam Akhnoukh, Omid Gholamalamdari, Anurupa Moitra, Lisa M. Jenkins, Rohit Bhargava, Erik R Nelson, Kevin Van Bortle, S. Prasanth, K. Prasanth
          </td>
          <td>2024-08-12</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ed5339b18c8cb661ce34053233bc7343f92ee3a" target='_blank'>
              1q jumping translocation as a biomarker in myeloid malignancy: frequently mutated genes associated with bad prognosis and low survival
              </a>
            </td>
          <td>
            Eitan Halper-Stromberg, V. Stinnett, L. Morsberger, Aparna Pallavajjala, M. Levis, A. DeZern, Michelle Lei, Brian Phan, Rena R. Xian, Christopher D. Gocke, Guilin Tang, Ying S Zou
          </td>
          <td>2024-08-01</td>
          <td>Experimental Hematology & Oncology</td>
          <td>0</td>
          <td>76</td>
        </tr>

        <tr id="DNA G-quadruplex(G4) is a guanine-rich single-stranded DNA sequence that spontaneously folds into a spherical four-stranded DNA secondary structure in oncogene promoter sequences and telomeres. G4s are highly associated with the occurrence and development of cancer and have emerged as promising anticancer targets. Natural products have long been important sources of anticancer drug development. In recent years, significant progress has been made in the discovery of natural drugs targeting DNA G4s, with many DNA G4s have been confirmed as promising targets of natural products, including MYC-G4, KRAS-G4, PDGFR-β-G4, BCL-2-G4, VEGF-G4, and telomeric G4. This review summarizes the research progress in discovering natural small molecules that target DNA G4s and their binding mechanisms. It also discusses the opportunities of and challenges in developing drugs targeting DNA G4s. This review will serve as a valuable reference for the research on natural products, particularly in the development of novel antitumor medications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0615bcc8ba2b9aa59d7f6c8fe9659c90b7d6290" target='_blank'>
              [Research progress in targeting DNA G-quadruplexes using natural products].
              </a>
            </td>
          <td>
            Zhi-Yuan Chen, Rong-Shuang Cheng, Wubulikasimu Ayixia, Yu-Shuang Liu, Kai-Bo Wang
          </td>
          <td>2024-07-01</td>
          <td>Zhongguo Zhong yao za zhi = Zhongguo zhongyao zazhi = China journal of Chinese materia medica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Abstract Senataxin is an RNA:DNA helicase that plays an important role in the resolution of RNA:DNA hybrids (R-loops) formed during transcription. R-loops are involved in the regulation of biological processes such as immunoglobulin class switching, gene expression and DNA repair. Excessive accumulation of R-loops results in DNA damage and loss of genomic integrity. Senataxin is critical for maintaining optimal levels of R-loops to prevent DNA damage and acts as a genome guardian. Within the nucleus, senataxin interacts with various RNA processing factors and DNA damage response and repair proteins. Senataxin interactors include survival motor neuron and zinc finger protein 1, with whom it co-localizes in sub-nuclear bodies. Despite its ubiquitous expression, mutations in senataxin specifically affect neurons and result in distinct neurodegenerative diseases such as amyotrophic lateral sclerosis type 4 and ataxia with oculomotor apraxia type 2, which are attributed to the gain-of-function and the loss-of-function mutations in senataxin, respectively. In addition, low levels of senataxin (loss-of-function) in spinal muscular atrophy result in the accumulation of R-loops causing DNA damage and motor neuron degeneration. Senataxin may play multiple functions in diverse cellular processes; however, its emerging role in R-loop resolution and maintenance of genomic integrity is gaining attention in the field of neurodegenerative diseases. In this review, we highlight the role of senataxin in R-loop resolution and its potential as a therapeutic target to treat neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0918b5cad58e2e8aa658850e44f481a80ca9f6d7" target='_blank'>
              Role of senataxin in R-loop-mediated neurodegeneration
              </a>
            </td>
          <td>
            Annapoorna Kannan, Shyni Gangadharan Leela, Dana Branzei, L. Gangwani
          </td>
          <td>2024-07-15</td>
          <td>Brain Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="


 Can a simple, and scalable assay be developed to concurrently perform all forms of preimplantation genetic testing (PGT), effectively addressing the increasing demand?



 Whole genome sequencing-based haplarithmisis offers a single-assay solution for all forms of PGT, enhancing resolution and diagnosis in complex cases.



 State-of-the-art preimplantation genetic testing (PGT) methods, including genotyping-by-sequencing (GBS)-based haplotyping, enable PGT for monogenic disorders (PGT-M), structural rearrangements (PGT-SR), and aneuploidies (PGT-A) in a single assay. However, they are labour intensive, only partially cover the genome, require a close relative to determine meiotic and mitotic origin of aneuploidies, and are troublesome for consanguineous families and in complex genomic contexts. Furthermore, the prevailing PGT approach for mitochondrial DNA disorders (PGT-MT) involves a PCR-based method. This underscores the need to develop a universal PGT method that streamlines laboratory protocols and provides genome-wide coverage to address genetic disorders, even in complex genomic loci.



 In a retrospective pilot study, we compared whole genome sequencing (WGS) at 30-40X with GBS (6 embryos, 2 families, 3 indications). We carried out subsampling of WGS data and determined 10X as the optimal diagnostic depth-of-coverage. We clinically validated our approach by comparing WGS to GBS for PGT-M and PGT-SR (31 embryos, 16 families, 22 indications), to shallow sequencing for PGT-SR (10 embryos, 3 families) and to PCR-based PGT for PGT-MT (10 embryos, 2 families).



 DNA from parents and close relative(s) and whole genome amplified (WGAed) DNA from embryos, underwent WGS with subsequent haplarithmisis. Quality metrics, such as Mendelian inconsistency rates, number of informative SNPs, and haplotype concordance were assessed. Translocation breakpoints were identified utilizing aberrantly aligned paired sequencing reads. Additionally, single-nucleotide variants (SNVs), including mitochondrial DNA SNVs, were detected by allele counting. Finally, segregational origin was determined via a parents-only haplarithmisis approach that uses the embryo as phasing reference.



 WGS-PGT alleviates technical limitations by decreasing WGA artifacts by 68.4%, increasing breadth of coverage by 4-fold, and decreasing wet-lab turnaround time by 2.5-fold. Owing to its inherent higher resolution and improved genomic coverage, 5 out of 13 – previously inconclusive – embryos with genetic disorders in complex genomic regions could be assigned a diagnosis. Furthermore, we show the ability to directly detect single- and few-base pair genetic variants in any region of interest. Importantly, we show trio-based PGT-A for aneuploidy origin, an approach we term PGT-AO that – using the parents-only genotype calls – can detect the segregational origin of aneuploidies without the need of a close relative as reference for phasing. The segregational origin detected by parents-only haplarithmisis was concordant with the segregational origin determined from haplarithmisis when using a close relative. Unbalanced translocations could be detected indirectly via haplotyping or directly at single base resolution using a structural variant caller. Mitochondrial heteroplasmy level calculation from WGS yielded comparable results to PCR-based methods, both in blastomere and trophectoderm biopsy samples.



 Direct variant detection with an average coverage of 10X may occasionally be inadequate, as certain locations may lack sufficient coverage (<5 reads). Inherently to the limited sample size of PGT-MT families, WGS-PGT needs further validation. The need for invasive trophectoderm biopsies of preimplantation embryos remains the key limitation of WGS-PGT.



 Our approach enables all forms of PGT in a single assay. Direct variant detection complements haplarithmisis, addressing inconclusive findings and aiding families with a de novo variant or lacking a close relative for phasing. Finally, PGT-AO facilitates research on the origin of aneuploidies in preimplantation embryos and their clinical outcome.



 not applicable
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dad9bc31410d3990a8eae04b617df3a5988ac444" target='_blank'>
              O-270 Clinical-grade whole genome sequencing-based haplarithmisis enables all forms of preimplantation genetic testing
              </a>
            </td>
          <td>
            A. Janssen, R. Koeck, R. Essers, W. van Dijk, M. Drüsedau, J. Meekels, B. Yaldiz, M. van de Vorst, J. Dreesen, C. De Die-Smulders, E. Coonen, H. Brunner, A. Van den Wijngaard, A. Paulussen, M. Zamani Esteki
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b11c9fdbfd43bb000396d5cb58b5ead9a774473" target='_blank'>
              Hypermethylated RASAL1’s promotive role in chemoresistance and tumorigenesis of choriocarcinoma was regulated by TET2 but not DNMTs
              </a>
            </td>
          <td>
            Xianling Zeng, Rui An, Ruixia Guo, Han Li
          </td>
          <td>2024-08-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Colorectal cancer (CRC) presents a significant global health burden, with high rates of incidence and mortality, and an urgent need to improve prognosis. STM2457, a novel small molecule inhibitor specific for N6-methyladenosine (m6A) catalytic enzyme Methyltransferase-like 3 (METTL3) has implicated significant treatment potentials in a few of types of cancer. However, its impact and underlying mechanism are still unclear in CRC cells. Methods: We used CCK-8 and colony formation assay to observe cell growth, flow cytometry and TUNEL approaches to detect cell apoptosis under the treatment of STM2457 on CRC cells in vitro or in vivo. RNA-sequencing, qRT-PCR and western blotting were performed to explore downstream effectors of STM2457. Messenger RNA stability was evaluated by qRT-PCR after treatment with actinomycin D. The methylated RNA immunoprecipitation (MeRIP) qPCR, dual-luciferase reporter analyses and m6A dot blotting were carried out to measure the m6A modification. Associated gene expression pattern and clinical relevance in CRC clinical tissue samples were analyzed using online database. Results: STM2457 exhibited a strong influence on cell growth suppression and apoptosis of CRC cells in vitro and subcutaneous xenograft growth in vivo. Asparagine synthetase (ASNS) was markedly downregulated upon STM2457 treatment or METTL3 knockdown and exogenous overexpression of ASNS could rescue the biological defects induced by STM2457. Mechanistically, the downregulation of ASNS by STM2457 may be due to the decrease of m6A modification level in ASNS mRNA mediated by METTL3. Conclusions: Our findings suggest that STM2457 may serve as a potential therapeutic agent and ASNS may be a new promising therapeutic target for CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9d2ad6ae3e7c70f86f85ea0a48c17dd82345f44" target='_blank'>
              METTL3 Inhibition Suppresses Cell Growth and Survival in Colorectal Cancer via ASNS Downregulation
              </a>
            </td>
          <td>
            Yang Yu, Yanan Hai, Hongfeng Zhou, Wenfang Bao, Xiaowei Hu, Yong Gao, Jin Wu
          </td>
          <td>2024-07-16</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a62302568e13d5d4ea358c501360fb449540886f" target='_blank'>
              Epigenetic regulation of HERVs: Implications for cancer immunotherapy.
              </a>
            </td>
          <td>
            Joo Mi Yi
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01d8689b47879107902739db4fd362bf3064c23" target='_blank'>
              Spatially resolved epigenome sequencing via Tn5 transposition and deterministic DNA barcoding in tissue.
              </a>
            </td>
          <td>
            Negin Farzad, Archibald Enninful, Shuozhen Bao, Di Zhang, Yanxiang Deng, Rong Fan
          </td>
          <td>2024-06-28</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135134a62bac76cb46dae2f04090c849f2c01172" target='_blank'>
              Targeting chromosomal instability in patients with cancer.
              </a>
            </td>
          <td>
            Duaa H Al-Rawi, Emanuele Lettera, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-11</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Disease risk prediction based on genomic sequence and transcriptional profile can improve disease screening and prevention. Despite identifying many disease-associated DNA variants, distinguishing deleterious non-coding DNA variations remains poor for most common diseases. In this study, we designed in vitro experiments to uncover the significance of occupancy and competitive binding between P53 and cMYC on common target genes. Analyzing publicly available ChIP-seq data for P53 and cMYC in embryonic stem cells showed that ~344-366 regions are co-occupied, and on average, two cis-overlapping motifs (CisOMs) per region were identified, suggesting that co-occupancy is evolutionarily conserved. Using U2OS and Raji cells untreated and treated with doxorubicin to increase P53 protein level while potentially reducing cMYC level, ChIP-seq analysis illustrated that around 16 to 922 genomic regions were co-occupied by P53 and cMYC, and substitutions of cMYC signals by P53 were detected post doxorubicin treatment. Around 187 expressed genes near co-occupied regions were altered at mRNA level according to RNA-seq data analysis. We utilized a computational motif-matching approach to illustrate that changes in predicted P53 binding affinity in CisOMs of co-occupied elements significantly correlate with alterations in reporter gene expression. We performed a similar analysis using SNPs mapped in CisOMs for P53 and cMYC from ChIP-seq data, and expression of target genes from GTEx portal. We found significant correlation between change in cMYC-motif binding affinity in CisOMs and altered expression. Our study brings us closer to developing a generally applicable approach to filter etiological non-coding variations associated with common diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2175dd5f742d55fc20f67f497dba908aa719b17b" target='_blank'>
              Sequence-to-expression approach to identify etiological non-coding DNA variations in P53 and cMYC-driven diseases.
              </a>
            </td>
          <td>
            K. Kin, Shounak Bhogale, Lisha Zhu, Derrick Thomas, Jessica W Bertol, W. J. Zheng, Saurabh Sinha, W. Fakhouri
          </td>
          <td>2024-07-17</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8b5865a1769acd4899db5ef067b527a73bc2f7f" target='_blank'>
              Inferring clonal somatic mutations directed by X chromosome inactivation status in single cells
              </a>
            </td>
          <td>
            Ilke Demirci, Anton J. M. Larsson, Xinsong Chen, Johan Hartman, R. Sandberg, Jonas Frisén
          </td>
          <td>2024-08-09</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe961497d985de77bb27297cd1364379e55768f" target='_blank'>
              Circular RNA in cancer.
              </a>
            </td>
          <td>
            Vanessa M. Conn, A. Chinnaiyan, Simon J. Conn
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Transposable elements (TEs) have emerged as important factors in establishing the cell type-specific gene regulatory networks and evolutionary novelty of embryonic and placental development. Recently, studies on the role of TEs and their dysregulation in cancers have shed light on the transcriptional, transpositional, and regulatory activity of TEs, revealing that the activation of developmental transcriptional programs by TEs may have a role in the dedifferentiation of cancer cells to the progenitor-like cell states. This essay reviews the recent evidence of the cis-regulatory TEs (henceforth crTE) in normal development and malignancy as well as the key transcription factors and regulatory pathways that are implicated in both cell states, and presents existing gaps remaining to be studied, limitations of current technologies, and therapeutic possibilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9ae01d2549bee9fa618c02f57bcecdeddf683a" target='_blank'>
              Transposable elements as drivers of dedifferentiation: Connections between enhancers in embryonic stem cells, placenta, and cancer.
              </a>
            </td>
          <td>
            Konsta Karttunen, Divyesh Patel, Biswajyoti Sahu
          </td>
          <td>2024-07-29</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="The Chromodomain helicase DNA-binding protein 1-like (CHD1L) is a nucleosome remodeling enzyme, which plays a key role in chromatin relaxation during the DNA damage response. Genome editing has shown that deletion of CHD1L sensitizes cells to PARPi, but the effect of its pharmacological inhibition has not been defined. Triple-negative breast cancer SUM149PT, HCC1937, and MDA-MB-231 cells were used to assess the mechanism of action of the CHD1Li OTI-611. Cytotoxicity as a single agent or in combination with standard-of-care treatments was assessed in tumor organoids. Immunofluorescence was used to assess the translocation of PAR and AIF to the cytoplasm or the nucleus and to study markers of DNA damage or apoptosis. Trapping of PARP1/2 or CHD1L onto chromatin was also assessed by in situ subcellular fractionation and immunofluorescence and validated by Western blot. We show that the inhibition of CHD1L’s ATPase activity by OTI-611 is cytotoxic to triple-negative breast cancer tumor organoids and synergizes with PARPi and chemotherapy independently of the BRCA mutation status. The inhibition of the remodeling function blocks the phosphorylation of H2AX, traps CHD1L on chromatin, and leaves PAR chains on PARP1/2 open for hydrolysis. PAR hydrolysis traps PARP1/2 at DNA damage sites and mediates PAR translocation to the cytoplasm, release of AIF from the mitochondria, and induction of PARthanatos. The targeted inhibition of CHD1L’s oncogenic function by OTI-611 signifies an innovative therapeutic strategy for breast cancer and other cancers. This approach capitalizes on CHD1L-mediated DNA repair and cell survival vulnerabilities, thereby creating synergy with standard-of-care therapies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401b6daf1d2a20088ba1aa72a3182ab37cb9c8e4" target='_blank'>
              CHD1L Regulates Cell Survival in Breast Cancer and Its Inhibition by OTI-611 Impedes the DNA Damage Response and Induces PARthanatos
              </a>
            </td>
          <td>
            Rita Sala, Hector Esquer, Timothy Kellett, Jeffrey T. Kearns, Paul Awolade, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2024-08-06</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40fcdfbbc2df4270401ef8beed0b78736d658d46" target='_blank'>
              DNA methylation in mammalian development and disease.
              </a>
            </td>
          <td>
            Zachary D. Smith, Sara Hetzel, Alexander Meissner
          </td>
          <td>2024-08-12</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Mutations in the shelterin protein POT1 are associated with diverse cancers, but their role in cancer progression remains unclear. To resolve this, we performed deep scanning mutagenesis in POT1 locally haploid human stem cells to assess the impact of POT1 variants on cellular viability and cancer-associated telomeric phenotypes. Though POT1 is essential, frame-shift mutants are rescued by chemical ATR inhibition, indicating that POT1 is not required for telomere replication or lagging strand synthesis. In contrast, a substantial fraction of clinically-validated pathogenic mutations support normal cellular proliferation, but still drive ATR-dependent telomeric DNA damage signaling and ATR-independent telomere elongation. Moreover, this class of cancer-associated POT1 variants elongates telomeres more rapidly than POT1 frame-shifts, indicating they actively drive oncogenesis and are not simple loss-of-function mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1a8aab5db363e4286320d81195cfe1e75198f812" target='_blank'>
              Dissecting the oncogenic mechanisms of POT1 cancer mutations through deep scanning mutagenesis
              </a>
            </td>
          <td>
            Annika Martin, Johannes Schabort, Rebecca Bartke-Croughan, Stella Tran, Atul Preetham, Robert Lu, Richard Ho, Jianpu Gao, Shirin Jenkins, John M Boyle, G. Ghanim, Milind Jagota, Yun S. Song, Hanqin Li, D. Hockemeyer
          </td>
          <td>2024-08-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16db7fe8f3bc4329cf344c224d8e623bae436c6d" target='_blank'>
              Elucidating the pan-oncologic landscape of S100A9: prognostic and therapeutic corollaries from an integrative bioinformatics and Mendelian randomization analysis
              </a>
            </td>
          <td>
            Yingying Chen, Zixuan Wu, Xingxing Yi
          </td>
          <td>2024-08-17</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Progression from pre-cancers like ductal carcinoma in situ (DCIS) to invasive disease (cancer) is driven by somatic evolution and is altered by clinical interventions. We hypothesized that genetic and/or phenotypic intra-tumor heterogeneity would predict clinical outcomes for DCIS since it serves as the substrate for natural selection among cells. We profiled two samples from two geographically distinct foci from each DCIS in both cross-sectional (N = 119) and longitudinal cohorts (N = 224), with whole exome sequencing, low-pass whole genome sequencing, and a panel of immunohistochemical markers. In the longitudinal cohorts, the only statistically significant predictors of time to non-invasive DCIS recurrence were the combination of treatment (lumpectomy only vs mastectomy or lumpectomy with radiation, HR = 12.13, p = 0.003, Wald test with FDR correction), ER status (HR = 0.16 for ER+ compared to ER-, p = 0.0045), and divergence in SNVs between the two samples (HR = 1.33 per 10% divergence, p = 0.018). SNV divergence also distinguished between pure DCIS and DCIS synchronous with invasive disease in the cross-sectional cohort. In contrast, the only statistically significant predictors of time to progression to invasive disease were the combination of the width of the surgical margin (HR = 0.67 per mm, p = 0.043) and the number of mutations that were detectable at high allele frequencies (HR = 1.30 per 10 SNVs, p = 0.02). These results imply that recurrence with DCIS is a clinical and biological process different from invasive progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2027c796343907ccaa630775f7381619210caedc" target='_blank'>
              Evolutionary Measures Show that Recurrence of DCIS is Distinct from Progression to Breast Cancer
              </a>
            </td>
          <td>
            Angelo Fortunato, D. Mallo, Luis Cisneros, Lorraine M. King, Aziz Khan, Christina Curtis, M. Ryser, Joseph Y. Lo, Allison Hall, Jeffrey R. Marks, E. S. Hwang, Carlo C. Maley
          </td>
          <td>2024-08-16</td>
          <td>medRxiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="In transcription-coupled repair, stalled RNA polymerase II (Pol II) is recognized by CSB and CRL4CSA, which co-operate with UVSSSA and ELOF1 to recruit TFIIH for nucleotide excision repair (TC-NER). To explore the mechanism of TC-NER, we recapitulated this reaction in vitro. When a plasmid containing a site-specific lesion is transcribed in frog egg extract, error-free repair is observed that depends on CSB, CRL4CSA, UVSSA, and ELOF1. Repair also depends on STK19, a factor previously implicated in transcription recovery after UV exposure. A 1.9 Å cryo-electron microscopy structure shows that STK19 joins the TC-NER complex by binding CSA and the RPB1 subunit of Pol II. Furthermore, AlphaFold predicts that STK19 interacts with the XPD subunit of TFIIH, and disrupting this interface impairs cell-free repair. Molecular modeling suggests that STK19 positions TFIIH ahead of Pol II for lesion verification. In summary, our analysis of cell-free TC-NER suggests that STK19 couples RNA polymerase II stalling to downstream repair events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297011593f4e81e15ed2ffb10b29598b2dac1064" target='_blank'>
              STK19 positions TFIIH for cell-free transcription-coupled DNA repair
              </a>
            </td>
          <td>
            T. Mevissen, Maximilian Kümmecke, Ernst W. Schmid, L. Farnung, Johannes C. Walter
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Maintenance of DNA integrity is essential to all forms of life. DNA damage generated by reaction with genotoxic chemicals results in deleterious mutations, genome instability, and cell death. Pathogenic bacteria encounter several genotoxic agents during infection. In keeping with this, the loss of DNA repair networks results in virulence attenuation in several bacterial species. Interstrand DNA crosslinks (ICLs) are a type of DNA lesion formed by covalent linkage of opposing DNA strands and are particularly toxic as they interfere with replication and transcription. Bacteria have evolved specialized DNA glycosylases that unhook ICLs, thereby initiating their repair. In this study, we describe AlkX, a DNA glycosylase encoded by the multidrug resistant pathogen Acinetobacter baumannii. AlkX exhibits ICL unhooking activity similar to that of its Escherichia coli homolog YcaQ. Interrogation of the in vivo role of AlkX revealed that its loss sensitizes cells to DNA crosslinking and impairs A. baumannii colonization of the lungs and dissemination to distal tissues during pneumonia. These results suggest that AlkX participates in A. baumannii pathogenesis and protects the bacterium from stress conditions encountered in vivo. Consistent with this, we found that acidic pH, an environment encountered during host colonization, results in A. baumannii DNA damage and that alkX is induced by, and contributes to, defense against acidic conditions. Collectively, these studies reveal functions for a recently described class of proteins encoded in a broad range of pathogenic bacterial species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/52e8859ed92eefb483b7d273e5025bd69777d468" target='_blank'>
              An interstrand DNA crosslink glycosylase aids Acinetobacter baumannii pathogenesis.
              </a>
            </td>
          <td>
            D. E. Kunkle, Yujuan Cai, B. Eichman, Eric P Skaar
          </td>
          <td>2024-06-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Deleterious accumulation of R-loops, a DNA-RNA hybrid structure, contributes to genome instability. They are associated with BRCA1 mutation-related breast cancer, an estrogen receptor α negative (ERα-) tumor type originating from luminal progenitor cells. However, a presumed causality of R-loops in tumorigenesis has not been established in vivo. Here, we overexpress mouse Rnaseh1 (Rh1-OE) in vivo to remove accumulated R-loops in Brca1-deficient mouse mammary epithelium (BKO). R-loop removal exacerbates DNA replication stress in proliferating BKO mammary epithelial cells, with little effect on homology-directed repair of double-strand breaks following ionizing radiation. Compared to their BKO counterparts, BKO-Rh1-OE mammary glands contain fewer luminal progenitor cells but more mature luminal cells. Despite a similar incidence of spontaneous mammary tumors in BKO and BKO-Rh1-OE mice, a significant percentage of BKO-Rh1-OE tumors express ERα and progesterone receptor. Our results suggest that rather than directly elevating the overall tumor incidence, R-loops influence the mammary tumor subtype by shaping the cell of origin for Brca1 tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fd59964db49fa8717ee1e81f17faaff9786150ba" target='_blank'>
              R-loop functions in Brca1-associated mammary tumorigenesis.
              </a>
            </td>
          <td>
            Huai-Chin Chiang, Leilei Qi, Payal Mitra, Yimeng Huang, Yanfen Hu, Rong Li
          </td>
          <td>2024-08-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Knowledge of locations and activities of cis-regulatory elements (CREs) is needed to decipher basic mechanisms of gene regulation and to understand the impact of genetic variants on complex traits. Previous studies identified candidate CREs (cCREs) using epigenetic features in one species, making comparisons difficult between species. In contrast, we conducted an interspecies study defining epigenetic states and identifying cCREs in blood cell types to generate regulatory maps that are comparable between species, using integrative modeling of eight epigenetic features jointly in human and mouse in our Validated Systematic Integration (VISION) Project. The resulting catalogs of cCREs are useful resources for further studies of gene regulation in blood cells, indicated by high overlap with known functional elements and strong enrichment for human genetic variants associated with blood cell phenotypes. The contribution of each epigenetic state in cCREs to gene regulation, inferred from a multivariate regression, was used to estimate epigenetic state Regulatory Potential (esRP) scores for each cCRE in each cell type, which were used to categorize dynamic changes in cCREs. Groups of cCREs displaying similar patterns of regulatory activity in human and mouse cell types, obtained by joint clustering on esRP scores, harbored distinctive transcription factor binding motifs that were similar between species. An interspecies comparison of cCREs revealed both conserved and species-specific patterns of epigenetic evolution. Finally, we showed that comparisons of the epigenetic landscape between species can reveal elements with similar roles in regulation, even in the absence of genomic sequence alignment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e8211dd3cbaad431cfef28f6397a18c06fd052" target='_blank'>
              Interspecies regulatory landscapes and elements revealed by novel joint systematic integration of human and mouse blood cell epigenomes.
              </a>
            </td>
          <td>
            Guanjue Xiang, Xi He, B. Giardine, Kathryn J. Isaac, Dylan J. Taylor, R. McCoy, Camden Jansen, C. Keller, A. Wixom, April Cockburn, Amber Miller, Qian Qi, Yanghua He, Yichao Li, Jens Lichtenberg, Elisabeth F. Heuston, Stacie M. Anderson, Jing Luan, M. Vermunt, Feng Yue, M. Sauria, Michael C. Schatz, James Taylor, B. Göttgens, Jim R. Hughes, D. Higgs, Mitchell J. Weiss, Yong Cheng, G. Blobel, D. Bodine, Yu Zhang, Qunhua Li, Shaun Mahony, R. Hardison
          </td>
          <td>2024-07-01</td>
          <td>Genome research</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="This article theoretically speculates that after replication of a pair of DNA they must be allocated to two daughter cells in a transverse division or erect division manner. Transverse division represents differentiation, while erect division represents transformation. The molecular structure determines that DNA has greater hydrophobicity than RNA. The hydrophobic modification of chromatin regulates gene activity and centromere similarity between new and old DNA. A difference in division between cancer cells and embryonic stem cells is discussed">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c6b78e19de2f6d2d4dda38ace7242257dc237cb" target='_blank'>
              On the Cancer
              </a>
            </td>
          <td>
            Xiao En Wang
          </td>
          <td>2024-08-31</td>
          <td>Journal of Cancer Research Reviews &amp; Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Ubiquitin modification and alternative polyadenylation play crucial roles in the onset and progression of cancer. Hence, this study aims to comprehensively and deeply understand gene regulation and associated biological processes in lung adenocarcinoma (LUAD) by integrating both mechanisms. Alternative polyadenylation (APA)-related E3 ubiquitin ligases in LUAD were identified through multiple databases, and the association between selected genetic loci influencing gene expression (apaQTL-SNPs) and LUAD risk were evaluated through the GWAS database of the Female Lung Cancer Consortium in Asia (FLCCA). Subsequently, the interaction between RNF213 and ZBTB20, as well as their functional mechanisms in LUAD, were investigated using bioinformatics analysis, Western blot, co-immunoprecipitation, and colony formation experiments. A total of five apaQTL-SNPs (rs41301932, rs4494603, rs9890400, rs56066320, and rs41301932), located on RNF213, were significantly associated with LUAD risk (p < 0.05), and they inhibit tumor growth through ubiquitin-mediated degradation of ZBTB20.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5c9426c96fac5af729404fd06cbe66dceed7c43" target='_blank'>
              Crosstalk among Alternative Polyadenylation, Genetic Variants and Ubiquitin Modification Contribute to Lung Adenocarcinoma Risk
              </a>
            </td>
          <td>
            Yutong Wu, Yanqiong Yuan, Huiwen Xu, Wendi Zhang, Anhui Ning, Siqi Li, Qiong Chen, Xiaobo Tao, Gongbu Pan, Tian Tian, Lei Zhang, Minjie Chu, Jiahua Cui
          </td>
          <td>2024-07-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="


 Can centromeres be labelled in live human oocytes using Cas9 system and is the labelling efficiency affected by advancing female age?



 Centromeres in human oocytes can be labelled using Cas9 system. Signal intensity decreases in oocytes from older patients, suggesting centromeric DNA unravelling as oocytes age.



 Aneuploidy in human oocytes is associated with infertility, pregnancy loss or genetic diseases in offspring. It arises due to abnormal chromosome segregation during meiosis, with the risk increasing with advancing female age. This is attributed to age-dependent loss of pericentromere-localised cohesion which holds sister chromatids together. Our work revealed that shugoshin-2 (Sgo2), a protein involved in protection of sister chromatid cohesion is also lost with age. This can exacerbate the effects of maternal age by rendering residual cohesin at pericentromeres vulnerable to loss in anaphase I. This hints at more extensive alterations to centromere organisation with advanced maternal age.



 This study uses human oocytes deselected from fertility treatment donated for research purposes with informed consent (HFEA RO155). The oocytes were collected fresh at germinal vesicle (GV), metaphase I (MI) or metaphase II (MII) stage, injected with Cas9 labelling system and live imaged or fixed.



 Centromeres were labelled in human oocytes using a dead (nuclease-deficient) (d)Cas9-EGFP protein and alpha satellite-specific small guide RNAs (sgRNA). Human oocytes were injected with dCas9-EGFP:sgAlphaSat and live-imaged using dual colour confocal microscopy with SPY555-DNA dye to also visualise chromosomes. High resolution imaging results were analysed with in-house automated kinetochore tracking software. Additional oocytes were fixed and processed for immunofluorescence to determine the high resolution localisation of dCas9-EGFP:sgAlphaSat and Sgo2 molecules.



 We found that centromeres in human oocytes are efficiently labelled when using Cas9 system. During both meiosis I and II the dCas9-EGFP:sgAlphaSat probe co-localised with pericentromeric bridge and subcentromeric pools of Sgo2. We observed that the dCas9-EGFP:sgAlphaSat signal fades with advancing female age. Quantification reveals that in meiosis II the loss is abrupt at around 35 years of age, resembling Sgo2 behaviour. Initial live imaging indicates that already compromised signal in older oocytes is completely lost about 4hrs after germinal vesicle breakdown, suggesting that advancing age renders centromeres more vulnerable as meiosis I progresses. We have successfully tracked the centromeres in high resolution imaging using the automated software, which allows for detailed investigation of human meiosis chromosome dynamics. This method has been previously validated and successfully used for other cell systems.



 Availability of human oocytes for research purposes is low, limiting our sample size. Moreover, most oocytes used for this study were deselected from treatment of patients diagnosed with infertility, potentially affecting their quality.



 Our results reveal a large-scale change in centromeric DNA structure in human oocytes with increasing age. Live centromere labelling is a first successful attempt to bivalent tracking in human meiosis with potential to provide insight into age-related loss of fertility in women.



 n/a
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/441db6fa05acf55025e3ab0193301dfa95dff58a" target='_blank'>
              O-070 Cas9-based chromosome labelling reveals large-scale age-dependent centromere reorganisation in human oocytes
              </a>
            </td>
          <td>
            A. Byrska, C. Currie, D. Taylor, M. Erent, A. Inchingolo, G. Hartshorne, A. McAinsh
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene conversions are broadly defined as the transfer of genetic material from a ‘donor’ to an ‘acceptor’ sequence and can happen both in meiosis and mitosis. They are a subset of non-crossover events and like crossover events, gene conversion can generate new combinations of alleles, erode linkage disequilibrium, and even counteract the mutation load by reverting germline mutations through GC-biased gene conversion. Estimating the rate of gene conversion and the distribution of gene conversion tract lengths remains challenging. Here, we present a new method for estimating tract length, rate and detection probability of non-crossover events directly in HiFi PacBio long read data. The method can be applied with data from a single individual, is unbiased even under low single nucleotide variant densities and does not necessitate any demographic or evolutionary assumptions. We apply the method to gene conversion events observed directly in Pacbio HiFI read data from a human sperm sample and find that human gene conversion tracts are shorter (mean of 50 base pairs) than estimates from yeast or Drosophila. We also estimate that typical human male gametes undergo on average 280 non-crossover events where approximately 7 are expected to become visible as gene conversions moving variants from one donor haplotype to an acceptor haplotype.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f463a46e1b5590cae1deadecdd92d1f8b1f8321" target='_blank'>
              Estimating gene conversion tract length and rate from PacBio HiFi data
              </a>
            </td>
          <td>
            A. P. Charmouh, Peter Porsborg Sørud, T. Bataillon, A. Hobolth, Lasse Thorup Hansen, Søren Besenbacher, S. B. Winge, K. Almstrup, M. Schierup
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf95f1dc0f1af0cb6e8bd77dab713b5fc609919" target='_blank'>
              CircRHBDD1 promotes immune escape via IGF2BP2/PD-L1 signaling and acts as a nanotherapeutic target in gastric cancer
              </a>
            </td>
          <td>
            Yanna Li, Zhixiong Wang, Peng Gao, Danping Cao, R. Dong, Menglin Zhu, Yao Fei, X. Zuo, Juan Cai
          </td>
          <td>2024-07-30</td>
          <td>Journal of Translational Medicine</td>
          <td>1</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cc5ea42f9ac2f6d502e99cce5e348de337e22eda" target='_blank'>
              Effects of HOX family regulator-mediated modification patterns and immunity characteristics on tumor-associated cell type in endometrial cancer
              </a>
            </td>
          <td>
            Jiaolin Yang, Jinpeng Li, Sufen Li, Yutong Yang, Huancheng Su, Hongrui Guo, Jing Lei, YaLin Wang, KaiTing Wen, Xia Li, Sanyuan Zhang, Zhe Wang
          </td>
          <td>2024-08-14</td>
          <td>Molecular Biomedicine</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract The use of k-mers to capture genetic variation in bacterial genome-wide association studies (bGWAS) has demonstrated its effectiveness in overcoming the plasticity of bacterial genomes by providing a comprehensive array of genetic variants in a genome set that is not confined to a single reference genome. However, little attempt has been made to interpret k-mers in the context of genome rearrangements, partly due to challenges in the exhaustive and high-throughput identification of genome structure and individual rearrangement events. Here, we present GWarrange, a pre- and post-bGWAS processing methodology that leverages the unique properties of k-mers to facilitate bGWAS for genome rearrangements. Repeat sequences are common instigators of genome rearrangements through intragenomic homologous recombination, and they are commonly found at rearrangement boundaries. Using whole-genome sequences, repeat sequences are replaced by short placeholder sequences, allowing the regions flanking repeats to be incorporated into relatively short k-mers. Then, locations of flanking regions in significant k-mers are mapped back to complete genome sequences to visualise genome rearrangements. Four case studies based on two bacterial species (Bordetella pertussis and Enterococcus faecium) and a simulated genome set are presented to demonstrate the ability to identify phenotype-associated rearrangements. GWarrange is available at https://github.com/DorothyTamYiLing/GWarrangehttps://github.com/DorothyTamYiLing/genome_rearrangement.git.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42162bc2b2f883e8a6834a9dc952dc10d3392ba" target='_blank'>
              GWarrange: a pre- and post- genome-wide association studies pipeline for detecting phenotype-associated genome rearrangement events
              </a>
            </td>
          <td>
            Yi Ling Tam, Sarah Cameron, Andrew Preston, Lauren A. Cowley
          </td>
          <td>2024-07-01</td>
          <td>Microbial Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 What role do active enhancers play in determining the fate of early embryonic cells? Are there differences in active enhancers between mouse and human blastocysts?



 Tissue-specific active enhancers in inner cell mass (ICM) and trophoblast (TE) are essential for blastocyst formation, regulating gene expression through interactive loops with promoters.



 At the two-cell stage of nuclear transfer embryos, the absence of interaction between Zscan4d promoter and distal active enhancer leads to reduced Zscan4d expression, resulting in lower developmental rates. Overexpressing the demethylase Kdm4d alleviates reprogramming-resistant H3K9me3 modification, restoring the structural loop between the Zscan4d promoter and active enhancer and enhancing developmental efficiency. These findings strongly suggest a close association between the formation of regulatory chromatin interactions in early embryonic development and histone modifications. Indicating active enhancers marked with H3K27ac may regulate differentiation gene expression through distal interactions in blastocyst.



 Analyze variations in histone H3K27ac modification, chromatin accessibility, higher-order structures, and transcriptomes during mouse embryo differentiation from 8-cell stage to ICM and TE formation. Identify tissue-specific active enhancers in ICM and TE, validate their gene expression regulation, explore their role in early differentiation-associated genes in mice. Compare and clarify similarities and differences in enhancer regulation mechanisms during early embryo development in humans and mice.



 The 8-cell, morula and blastocyst stage embryos were collected from the oviducts and uterus of super-ovulated female ICR mice mated with male mice. The clinical human blastocysts were collected and separated the ICM and TE. Low-input multi-omics sequencing, including Cut-Tag, ATAC-seq, Hi-C, and RNA-seq, was performed on the embryonic samples. After analyzing and identifying active enhancers, tissue-specific ones were selected and validated for their impact on gene expression via CRISPR-Cas9 knockout in embryos.



 Inhibiting ep300 expression in mouse embryos through siRNA interference reduces blastocyst formation from 90% to approximately 40%. Introducing the small molecule inhibitor A485 at the 8-cell stage, blocking the active center of CBP/p300 enzymes, completely arrests embryos at the morula stage, preventing their development into blastocysts. Emphasizing the crucial role of correctly establishing active enhancers marked by H3K27ac modification in embryonic differentiation.
 Multi-omics data demonstrates high repeatability and quality. PCA clustering analysis reveals a clear differentiation path in the distribution of H3K27ac-modified histones, chromatin accessibility sites, and the transcriptome from the 8-cell stage to the blastocyst. Differences in H3K27ac histone modification distribution between the ICM and TE indicate the presence of tissue-specific active enhancers in blastocyst.
 The disparities in H3K27ac modification distribution between blastocyst ICM and TE primarily occur in distal gene regions rather than proximal ones. Moreover, these distal enhancers exhibit a positive correlation with the expression of lineage-specific genes, suggesting a probable regulatory role of distal enhancers in gene modulation during embryonic differentiation.
 Tissue-specific enhancer-promoter (EP) interaction loops emerge during embryonic differentiation, allowing direct interaction between tissue-specific active enhancers and promoters, thereby activating the expression of tissue-specific genes.



 While DNA sequences with H3K27ac modification are currently considered markers for active enhancers, H3K4me modification is another histone marker for enhancers. However, enhancers associated with H3K4me modification may not necessarily be in an activated state. Future research can simultaneously incorporate appropriate detection methods for H3K4me modification.



 After elucidating the mechanisms through which active enhancers regulate mouse differentiation, comparing the differences in active enhancers during the first lineage differentiation process between mice and humans can help identify factors that enhance human blastocyst quality. This can potentially improve clinical outcomes in assisted reproduction.



 not applicable
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06a46c47cbfd5b64e709929a1eba256f72f28a5e" target='_blank'>
              P-548 An investigation into the mechanisms of active enhancers regulating the initial lineage differentiation in mammals
              </a>
            </td>
          <td>
            M. Chen
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Glioblastoma is the most common primary malignant brain tumour. Despite decades of intensive research in the disease, its prognosis remains poor, with an average survival of only 14 months after diagnosis. The remarkable level of intra- and interpatient heterogeneity is certainly contributing to the lack of progress in tackling this tumour. Epigenetic dysregulation plays an important role in glioblastoma biology and significantly contributes to intratumour heterogeneity. However, it is becoming increasingly clear that it also contributes to intertumour heterogeneity, which historically had mainly been linked to diverse genetic events occurring in different patients. In this review, we explore how DNA methylation, chromatin remodelling, microRNA (miRNA) dysregulation, and long noncoding RNA (lncRNA) alterations contribute to intertumour heterogeneity in glioblastoma, including its implications for advanced tumour stratification, which is the essential first step for developing more effective patient-specific therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d8a905205728cf89ec2231b745c2dd76f674ecb4" target='_blank'>
              Unravelling the mosaic: Epigenetic diversity in glioblastoma.
              </a>
            </td>
          <td>
            Sara Lucchini, Myrianni Constantinou, Silvia Marino
          </td>
          <td>2024-08-15</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25bc89ba9898a52d88ff5aa506e261a761286bfb" target='_blank'>
              piR-823 tale as emerging cancer-hallmark molecular marker in different cancer types: a step-toward ncRNA-precision.
              </a>
            </td>
          <td>
            Fatma H Shaker, E. Sanad, H. Elghazaly, Shih-Min Hsia, N. Hamdy
          </td>
          <td>2024-08-05</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Ewing sarcoma (EwS) is a cancer that arises in the bones and soft tissues, typically driven by the Ewing’s sarcoma breakpoint region 1-Friend leukemia virus integration 1 (EWS-FLI) oncogene. Implementation of genetically modified animal models of EwS has proved difficult largely owing to EWS-FLI’s high toxicity. The EWS-FLI1FS frameshift variant that circumvents toxicity but is still able to perform key oncogenic functions provided the first study model in Drosophila. However, the quest for Drosophila lines expressing full-length, unmodified EWS-FLI remained open. Here, we show that EWS-FLI1FS’s lower toxicity is owed to reduced protein levels caused by its frameshifted C-terminal peptide, and report new strategies through which we have generated Drosophila lines that express full-length, unmodified EWS-FLI. Using these lines, we have found that the upregulation of transcription from GGAA-microsatellites (GGAAμSats) presents a positive linear correlation within a wide range of EWS-FLI protein concentrations. In contrast, rather counterintuitively, GGAAμSats-independent transcriptomic dysregulation presents relatively minor differences across the same range, suggesting that GGAAμSat-dependent and -independent transcriptional upregulation present different kinetics of response with regards to changing EWS-FLI protein concentration. Our results underpin the functional relevance of varying EWS-FLI expression levels and provide experimental tools to investigate, in Drosophila, the effect of the EWS-FLI ‘high’ and ‘low’ states that have been reported and are suspected to be important for EwS in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae15bdf2a86e467bdb76a0d6c4c39f6d7b414f7" target='_blank'>
              Human EWS-FLI protein levels and neomorphic functions show a complex, function-specific dose–response relationship in Drosophila
              </a>
            </td>
          <td>
            Serena Mahnoor, Cristina Molnar, Diego Velázquez, Jose Reina, S. Llamazares, J. P. Heinen, Jaume Mora, Cayetano Gonzalez
          </td>
          <td>2024-07-01</td>
          <td>Open Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcaa5e009963f12c7b3cfac1a62813bb1ec1e2e7" target='_blank'>
              Removal of TREX1 activity enhances CRISPR-Cas9-mediated homologous recombination.
              </a>
            </td>
          <td>
            Mehmet E. Karasu, Eléonore Toufektchan, Yanyang Chen, Alessandra Albertelli, G. Cullot, J. Maciejowski, Jacob E. Corn
          </td>
          <td>2024-08-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7098328b990f26128e9d5dd53699039600c56625" target='_blank'>
              Noncanonical formation of SNX5 gene-derived circular RNA regulates cancer growth
              </a>
            </td>
          <td>
            Yi-Tung Chen, Hui-Ju Tsai, C. Kan, Chung-Pei Ma, Hui-Wen Chen, I. Chang, Hsuan Liu, Chih-Ching Wu, Wei-Yun Chu, Ya-Chun Wu, , Jau-Song Yu, B. Tan
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7401c81aea35658fe72a5961ba5c3e31cae1d2fa" target='_blank'>
              The genomic basis of childhood T-lineage acute lymphoblastic leukaemia.
              </a>
            </td>
          <td>
            P. Pölönen, Danika Di Giacomo, A. Seffernick, Abdelrahman A. Elsayed, Shunsuke Kimura, Francesca Benini, Lindsey E Montefiori, Brent L Wood, Jason Xu, Changya Chen, Zhong-Yi Cheng, H. Newman, Jason R Myers, Ilaria Iacobucci, Elizabeth Li, Jonathan H. Sussman, Dale Hedges, Yawei Hui, Caroline Diorio, Lahari Uppuluri, David Frank, Yiping Fan, Yunchao Chang, S. Meshinchi, R. Ries, R. Shraim, Alexander Li, Kathrin M Bernt, M. Devidas, S. Winter, K. Dunsmore, H. Inaba, William L Carroll, Nilsa C Ramirez, Aaron H. Phillips, Richard W. Kriwacki, Jun J Yang, T. Vincent, Yaqi Zhao, P. S. Ghate, Jian Wang, Colleen Reilly, Xin Zhou, Mathijs A. Sanders, Junko Takita, Motohiro Kato, Nao Takasugi, Bill H Chang, Richard D Press, M. Loh, E. Rampersaud, E. Raetz, S P Hunger, Kai Tan, Ti-Cheng Chang, Gang Wu, S. B. Pounds, C. Mullighan, D. Teachey
          </td>
          <td>2024-08-14</td>
          <td>Nature</td>
          <td>1</td>
          <td>102</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="The SAGA complex is an evolutionarily conserved histone acetyltransferase complex and transcription coactivator essential for development and disease. Dysregulation of SAGA is implicated in various human diseases, including cancer. In this issue of Genes & Development, Chen et al. (doi/10.1101/gad.351789.124) uncover a critical role for SAGA in multiple myeloma wherein SAGA's ADA2B component is required for the expression of mTORC1 pathway genes and targets of the MYC, E2F, and MAF (musculoaponeurotic fibrosarcoma) transcription factors. SAGA cooperates with MYC and MAF to sustain oncogenic gene expression programs vital for multiple myeloma survival and thus may serve as a therapeutic target for future cancer therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa0500e20ce37a63c1e23fe219d04a1b016c2f1c" target='_blank'>
              The MYC-MAF-SAGA axis drives oncogenic gene expression in multiple myeloma.
              </a>
            </td>
          <td>
            Hongkuan Wang, Hong Wen, Xiaobing Shi
          </td>
          <td>2024-08-21</td>
          <td>Genes & development</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c66330ddcae28e0daca5306a93cbc05a61a0ce25" target='_blank'>
              Circulating tumor DNA assisting lymphoma genetic feature profiling and identification.
              </a>
            </td>
          <td>
            Hongbiao Wang, Zhao Wang, Sujuan Zhu, Zhifeng Li, Hang Yang, Peng Sun, Minyi Zhu, Xiaotian Zhao, Lu Shen, Qiuxiang Ou, Hui Yang, Zhiming Li
          </td>
          <td>2024-07-16</td>
          <td>Annals of hematology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In recent years, the detection and analysis of circulating tumor DNA (ctDNA) have emerged as a new focus in the field of cancer research, particularly in the early diagnosis of hepatocellular carcinoma (HCC) and monitoring of therapeutic efficacy. ctDNA, which refers to cell-free DNA fragments released into the bloodstream from tumor cells upon cell death or shedding, carries tumor-specific genetic and epigenetic alterations, thereby providing a non-invasive approach for cancer diagnosis and prognosis. The concentration of ctDNA in the blood is higher compared to that in healthy individuals or other liquid biopsies from early-stage cancers, which is closely associated with the early diagnosis and comprehensive sequencing studies of HCC. Recent studies have indicated that sequential ctDNA analysis in patients receiving primary or adjuvant therapy for HCC can detect treatment resistance and recurrence before visible morphological changes in the tumor, making it a valuable basis for rapid adjustment of treatment strategies. However, this technology is continuously being optimized and improved. Challenges such as enhancing the accuracy of ctDNA sequencing tests, reducing the burden of high-throughput sequencing on a large number of samples, and controlling variables in the assessment of the relationship between ctDNA concentration and tumor burden, need to be addressed. Overall, despite the existing challenges, the examination and analysis of ctDNA have opened up new avenues for early diagnosis and therapeutic efficacy monitoring in hepatocellular carcinoma, expanding the horizons of this field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe795a1ff20add2c9eb3d87e4f8cefe083305b7e" target='_blank'>
              Circulating tumor DNA mutation analysis: advances in its application for early diagnosis of hepatocellular carcinoma and therapeutic efficacy monitoring
              </a>
            </td>
          <td>
            Jing Yang, Na Lin, Miaomiao Niu, Boshu Yin
          </td>
          <td>2024-07-19</td>
          <td>Aging (Albany NY)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSC) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosomes 20q or 1q, both known for conferring in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings show higher loads of both CNVs and SNVs at later passages which are due to the cumulative acquisition of mutations over a longer time in culture and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs follow the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid cancer-associated mutations. This suggests that the recurrent chromosomal abnormalities are the potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f552b60bd87e180c4d6a5dd522428893e4b81b" target='_blank'>
              De novo cancer mutations frequently associate with recurrent chromosomal abnormalities during long-term human pluripotent stem cell culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huygebaert, M. Regin, Chi Mai Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Chromosomal fusions play an integral role in genome remodeling and karyotype evolution. Fusions that join a sex chromosome to an autosome are particularly abundant across the tree of life. However, previous models on the establishment of such fusions have not accounted for the physical structure of the chromosomes. We predict a fusion joining an autosome to the pseudoautosomal region (PAR) of a sex chromosome will not remain stable, and the fusion will switch from the X to the Y chromosome each generation due to recombination. We have produced a forward-time population genetic simulation to explore the outcomes of fusions to both the PAR and non-PAR of sex chromosomes. The model can simulate the fusion of an autosome containing a sexually antagonistic locus to either the PAR or non-PAR end of a sex chromosome. Our model is diploid, two-locus, and biallelic. Our results show a clear pattern where fusions to the non-PAR are favored in the presence of sexual antagonism, whereas fusions to the PAR are disfavored in the presence of sexual antagonism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417dce982562c863a7e38c28a474ba15591636ef" target='_blank'>
              Worse than nothing at all: the inequality of fusions joining autosomes to the PAR and non-PAR portions of sex chromosomes
              </a>
            </td>
          <td>
            Kayla T. Wilhoit, Emmarie P. Alexander, Heath Blackmon
          </td>
          <td>2024-07-23</td>
          <td>PeerJ</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/798b264fbd39508ed9588f5472e886b8b6303884" target='_blank'>
              Navigating the complexity of p53-DNA binding: implications for cancer therapy
              </a>
            </td>
          <td>
            Kelly M. Thayer, Sean Stetson, Fernando Caballero, Christopher Chiu, I. Han
          </td>
          <td>2024-07-11</td>
          <td>Biophysical Reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The p53 genome is a gene that has truly taken the scientific world by storm. Once referred to as an oncogene, p53 has proved itself to be more complex and dubious time and time again. This paper will talk about the beginning doubts and queries revolving around p53 and the discoveries that led to it being referred to as a tumour suppressor gene, rather than being mislabeled as an oncogene. It will touch upon the importance that p53 brings to our daily lives such as repairing the DNA, maintaining the cell cycle and starting apoptosis, as well as highlight the differences between the gene’s two types: wild-type p53 and mutant p53. Furthermore, it will talk about how a slight mutation in p53, can cause it to form many protein isoforms, a leading contributing factor in cancers. It will also mention the effects post-translational modification will have on the two types of p53. In addition to this, it will delve into the negative and potentially harmful effects that p53 could have on our bodies. Finally, it will also talk about p53-mediated therapy and targeting mutant p53 with autophagy, which are 2 processes that have shown successful outcomes in reducing the complications caused by mutant p53.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1dc70ab96424bdf601f1a4ba7f9b8f2ff8a5b550" target='_blank'>
              P53: Our Saviour and Our Destroyer
              </a>
            </td>
          <td>
            Narayani Haldar
          </td>
          <td>2024-07-23</td>
          <td>International Journal For Multidisciplinary Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad1575162f33cdbe2adcebbaac6a20b693ec8530" target='_blank'>
              Single-cell EpiChem jointly measures drug-chromatin binding and multimodal epigenome.
              </a>
            </td>
          <td>
            Chao Dong, Xiaoxuan Meng, Tong Zhang, Zhifang Guo, Yaxi Liu, Peihuang Wu, Shiwei Chen, Fanqi Zhou, Yanni Ma, Haiqing Xiong, Shaokun Shu, Aibin He
          </td>
          <td>2024-07-18</td>
          <td>Nature methods</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Genomic instability is one of the main drivers of tumorigenesis and the development of hematological malignancies. Cancer cells can remedy chemotherapeutic-induced DNA damage by upregulating DNA-repair genes and ultimately inducing therapy resistance. Nevertheless, the association between the DNA-repair genes, drug resistance, and disease relapse has not been well characterized in acute lymphoblastic leukemia (ALL). This study aimed to explore the role of the DNA-repair machinery and the molecular mechanisms by which it is regulated in early- and late-relapsing pediatric ALL patients. We performed secondary data analysis on the Therapeutically Applicable Research to Generate Effective Treatments (TARGET)—ALL expansion phase II trial of 198 relapsed pediatric precursor B-cell ALL. Comprehensive genetic and epigenetic investigations of 147 DNA-repair genes were conducted in the study. Gene expression was assessed using Microarray and RNA-sequencing platforms. Genomic alternations, methylation status, and miRNA transcriptome were investigated for the candidate DNA-repair genes. We identified three DNA-repair genes, ALKBH3, NHEJ1, and PARP1, that were upregulated in early relapsers compared to late relapsers (p < 0.05). Such upregulation at diagnosis was significantly associated with disease-free survival and overall survival in precursor-B-ALL (p < 0.05). Moreover, PARP1 upregulation accompanied a significant downregulation of its targeting miRNA, miR-1301-3p (p = 0.0152), which was strongly linked with poorer disease-free and overall survivals. Upregulation of DNA-repair genes, PARP1 in particular, increases the likelihood of early relapse of precursor-B-ALL in children. The observation that PARP1 was upregulated in early relapsers relative to late relapsers might serve as a valid rationale for proposing alternative treatment approaches, such as using PARP inhibitors with chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/51e9e18af70423e8a01753d2f90a2a9bf475d2db" target='_blank'>
              Genetic and Epigenetic Biomarkers Associated with Early Relapse in Pediatric Acute Lymphoblastic Leukemia: A Focused Bioinformatics Study on DNA-Repair Genes
              </a>
            </td>
          <td>
            Walaa F Albaqami, Ali A. Alshamrani, Ali A. Almubarak, Faris E. Alotaibi, Basil Alotaibi, Abdulrahman M. Alanazi, M. Alotaibi, Ali R. Alhoshani, Homood M. As Sobeai
          </td>
          <td>2024-08-05</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="OBJECTIVES
The present study used single-cell RNA sequencing (scRNA-seq) to characterize the cellular composition of ovarian carcinosarcoma (OCS) and identify its molecular characteristics.


METHODS
scRNA-seq was performed in resected primary OCS for an in-depth analysis of tumor cells and the tumor microenvironment. Immunohistochemistry staining was used for validation. The scRNA-seq data of OCS were compared with those of high-grade serous ovarian carcinoma (HGSOC) tumors and other OCS tumors.


RESULTS
Both malignant epithelial and malignant mesenchymal cells were observed in the OCS patient of this study. We identified four epithelial cell subclusters with different biological roles. Among them, epithelial subcluster 4 presented high levels of breast cancer type 1 susceptibility protein homolog (BRCA1) and DNA topoisomerase 2-α (TOP2A) expression and was related to drug resistance and cell cycle. We analyzed the interaction between epithelial and mesenchymal cells and found that fibroblast growth factor (FGF) and pleiotrophin (PTN) signalings were the main pathways contributing to communication between these cells. Moreover, we compared the malignant epithelial and mesenchymal cells of this OCS tumor with our previous published HGSOC scRNA-seq data and OCS data. All the epithelial subclusters in the OCS tumor could be found in the HGSOC samples. Notably, the mesenchymal subcluster C14 exhibited specific expression patterns in the OCS tumor, characterized by elevated expression of cytochrome P450 family 24 subfamily A member 1 (CYP24A1), collagen type XXIII α1 chain (COL23A1), cholecystokinin (CCK), bone morphogenetic protein 7 (BMP7), PTN, Wnt inhibitory factor 1 (WIF1), and insulin-like growth factor 2 (IGF2). Moreover, this subcluster showed distinct characteristics when compared with both another previously published OCS tumor and normal ovarian tissue.


CONCLUSIONS
This study provides the single-cell transcriptomics signature of human OCS, which constitutes a new resource for elucidating OCS diversity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cbf6169030bf9196f7db5cdbd57ccc6a7a671e33" target='_blank'>
              Single-cell transcriptomics reveals tumor landscape in ovarian carcinosarcoma.
              </a>
            </td>
          <td>
            Junfen Xu, Mengyan Tu
          </td>
          <td>2024-08-15</td>
          <td>Journal of Zhejiang University. Science. B</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="BACKGROUND
Cellular angiofibroma, a rare benign mesenchymal neoplasm, is classified within the 13q/RB1 family of tumors due to morphological, immunohistochemical, and genetic similarities with spindle cell lipoma. Here, genetic data reveal pathogenetic heterogeneity in cellular angiofibroma.


METHODS
Three cellular angiofibromas were studied using G-banding/Karyotyping, array comparative genomic hybridization, RNA sequencing, and direct cycling sequencing.


RESULTS
The first tumor carried a del(13)(q12) together with heterozygous loss and minimal expression of the RB1 gene. Tumors two and three displayed chromosome 8 abnormalities associated with chimeras of the pleomorphic adenoma gene 1 (PLAG1). In tumor 2, the cathepsin B (CTSB) fused to PLAG1 (CTSB::PLAG1) while in tumor 3, the mir-99a-let-7c cluster host gene (MIR99AHG) fused to PLAG1 (MIR99AHG::PLAG1), both leading to elevated expression of PLAG1 and insulin growth factor 2.


CONCLUSION
This study uncovers two genetic pathways contributing to the pathogenetic heterogeneity within cellular angiofibromas. The first aligns with the 13q/RB1 family of tumors and the second involves PLAG1-chimeras. These findings highlight the diverse genetic landscape of cellular angiofibromas, providing insights into potential diagnostic strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/72129daa715b46d571770497bea5325bf92afaeb" target='_blank'>
              Genetic Heterogeneity in Cellular Angiofibromas.
              </a>
            </td>
          <td>
            Ioannis Panagopoulos, K. Andersen, I. Lobmaier, M. Lund-Iversen
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91fe3c701c57ad11bfba55ea29642c8877198c85" target='_blank'>
              Single-cell analysis identified PDIA3 as regulator of malignant characteristics and macrophage function in human cancers.
              </a>
            </td>
          <td>
            Wantao Wu, Gang Peng, Kaiyue Wang, Yijian Yang, Zhikun Liu, Gelei Xiao
          </td>
          <td>2024-08-13</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Deep learning offers new approaches to investigate the mechanisms underlying complex biological phenomena, such as subgenome dominance. Subgenome dominance refers to the dominant expression and/or biased fractionation of genes in one subgenome of allopolyploids, which has shaped the evolution of a large group of plants. However, the underlying cause of subgenome dominance remains elusive. Here, we adopt deep learning to construct two convolutional neural network (CNN) models, binary expression model (BEM) and homoeolog contrast model (HCM), to investigate the mechanism underlying subgenome dominance using DNA sequence and methylation sites. We apply these CNN models to analyze three representative polyploidization systems, Brassica, Gossypium, and Cucurbitaceae, each with available ancient and neo/synthetic polyploidized genomes. The BEM shows that DNA sequence of the promoter region can accurately predict whether a gene is expressed or not. More importantly, the HCM shows that the DNA sequence of the promoter region predicts dominant expression status between homoeologous gene pairs retained from ancient polyploidizations, thus predicting subgenome dominance associated with these events. However, HCM fails to predict gene expression dominance between new homoeologous gene pairs arising from the neo/synthetic polyploidizations. These results are consistent across the three plant polyploidization systems, indicating broad applicability of our models. Furthermore, the two models based on methylation sites produce similar results. These results show that subgenome dominance is associated with long-term sequence differentiation between the promoters of homoeologs, suggesting that subgenome expression dominance precedes and is the driving force or even the determining factor for sequence divergence between subgenomes following polyploidization.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5a177f119aaec38ae7469517fb1f1f336659b36" target='_blank'>
              Deep learning can predict subgenome dominance in ancient but not in neo/synthetic polyploidized genomes.
              </a>
            </td>
          <td>
            Zhongwei Guo, Kang Zhang, Chengcheng Cai, Xing Li, Lingkui Zhang, Yinqing Yang, Xiang Wang, Shumin Chen, Lei Zhang, Feng Cheng
          </td>
          <td>2024-08-12</td>
          <td>The Plant journal : for cell and molecular biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The deoxyribonucleic acid (DNA) carries hereditary codes which is translated by the cells to synthesize the ribonucleic acid (RNA) and polypeptides which can generate and perform vital function. The double helix structure is the most studied model of the DNA that was proposed by Watson and Crick. The capability of DNA to work as a genetic material can be stored and conducted during cell division to permit this information to be doubled and transmitted to the incoming generation. Any damage in the structure of DNA is an essential direct cause for the progression of cancer and other disorders. The factors for DNA damage can be classified as exogenous and endogenous factors. In this review article, we highlight the evidence-supported information about the structure, functions and clinical significance of DNA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23ff4de2b8e4130d71de29efe87be0afb81a98a9" target='_blank'>
              Structure, Functions And Clinical Significance Of DNA: A Review Article
              </a>
            </td>
          <td>
            Ahmed M. Amshawee, Maryam A. Hussain, Seenaa Taqi Mansour Al Muhtaser, Ali A. Al- Faham
          </td>
          <td>2024-07-22</td>
          <td>International Journal of Health &amp; Medical Research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Understanding the impact of clonal hematopoiesis of indeterminate potential (CHIP) and mosaic chromosomal alterations (mCAs) on solid tumor risk and mortality can shed light on novel cancer pathways.


METHODS
The authors analyzed whole genome sequencing data from the Trans-Omics for Precision Medicine Women's Health Initiative study (n = 10,866). They investigated the presence of CHIP and mCA and their association with the development and mortality of breast, lung, and colorectal cancers.


RESULTS
CHIP was associated with higher risk of breast (hazard ratio [HR], 1.30; 95% confidence interval [CI], 1.03-1.64; p = .02) but not colorectal (p = .77) or lung cancer (p = .32). CHIP carriers who developed colorectal cancer also had a greater risk for advanced-stage (p = .01), but this was not seen in breast or lung cancer. CHIP was associated with increased colorectal cancer mortality both with (HR, 3.99; 95% CI, 2.41-6.62; p < .001) and without adjustment (HR, 2.50; 95% CI, 1.32-4.72; p = .004) for advanced-stage and a borderline higher breast cancer mortality (HR, 1.53; 95% CI, 0.98-2.41; p = .06). Conversely, mCA (cell fraction [CF] >3%) did not correlate with cancer risk. With higher CFs (mCA >5%), autosomal mCA was associated with increased breast cancer risk (HR, 1.39; 95% CI, 1.06-1.83; p = .01). There was no association of mCA (>3%) with breast, colorectal, or lung mortality except higher colon cancer mortality (HR, 2.19; 95% CI, 1.11-4.3; p = .02) with mCA >5%.


CONCLUSIONS
CHIP and mCA (CF >5%) were associated with higher breast cancer risk and colorectal cancer mortality individually. These data could inform on novel pathways that impact cancer risk and lead to better risk stratification.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0fa8e5d699d9cb31c3abe6a09df01fb728c1626c" target='_blank'>
              Association of clonal hematopoiesis and mosaic chromosomal alterations with solid malignancy incidence and mortality.
              </a>
            </td>
          <td>
            Pinkal M. Desai, Ying Zhou, Justin Grenet, Samuel K Handelman, Cynthia Crispino, Laura N Tarbay, E. Whitsel, Gail Roboz, Ana Barac, Michael Honigberg, A. Bick, Garnet Anderson, Jean Wactawski-Wende, Yasminka A Jakubek Swartzlander, Jason Bacon, Justin Wong, Xiaolong Ma, P. Scheet, Zichan Li, P. Kasi, Ross Prentice, Paul L Auer, J. Manson, Alexander Reiner, Michael S. Simon
          </td>
          <td>2024-07-16</td>
          <td>Cancer</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Head and neck malignancies are a significant global health concern, with head and neck squamous cell carcinoma (HNSCC) being the sixth most common cancer worldwide accounting for > 90% of cases. In recent years, there has been growing recognition of the potential role of alternative splicing (AS) in the etiology of cancer. Increasing evidence suggests that AS is associated with various aspects of cancer progression, including tumor occurrence, invasion, metastasis, and drug resistance. Additionally, AS is involved in shaping the tumor microenvironment, which plays a crucial role in tumor development and response to therapy. AS can influence the expression of factors involved in angiogenesis, immune response, and extracellular matrix remodeling, all of which contribute to the formation of a supportive microenvironment for tumor growth. Exploring the mechanism of AS events in HNSCC could provide insights into the development and progression of this cancer, as well as its interaction with the tumor microenvironment. Understanding how AS contributes to the molecular changes in HNSCC cells and influences the tumor microenvironment could lead to the identification of new therapeutic targets. Targeted chemotherapy and immunotherapy strategies tailored to the specific AS patterns in HNSCC could potentially improve treatment outcomes and reduce side effects. This review explores the concept, types, processes, and technological advancements of AS, focusing on its role in the initiation, progression, treatment, and prognosis of HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/237a9433a011d162afee432657a141ab7fd1d189" target='_blank'>
              Alternative Splicing Landscape of Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Kehan Wu, Qianhui Sun, Dongxu Liu, Jiayi Lu, Deyu Wen, Xiyan Zang, Li Gao
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Accurate inference of granular cell states that co-occur within the tumour microenvironment (TME) is central to defining pro- and anti-tumour environments. However, to reliably identify recurrent coexisting cell populations it is fundamental to analyze datasets encompassing a substantial number of tumour samples with a resolution sufficient to capture granular cell states. Here, we leverage eight scRNA-seq datasets of pancreatic ductal adenocarcinoma (PDAC) in a unique discovery-validation setup and find reproducible cell states, gene programs, and cellular niches that are predictive of specific clinical outcomes. Across tumours, we show highly consistent co-occurrence of cell states within and between lineages, including those reflecting known and de novo cellular interactions alongside the formation of multi-cellular clusters such as tertiary lymphoid structures. In addition, we develop a novel probabilistic model to quantify multi-cellular communities directly from atlas-scale scRNA-seq datasets. This model identified cellular niches predictive of clinical outcomes including communities associated with response to therapy and with specific KRAS mutations. Together, this work lays the foundation for inferring reproducible multicellular niches directly from large nonspatial scRNA-seq atlases and linking their presence in individual patients to prognosis and therapy response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae43d3347571967285a6dfcd086345c41b6bceff" target='_blank'>
              Decoding multicellular niche formation in the tumour microenvironment from nonspatial single-cell expression data
              </a>
            </td>
          <td>
            Chengxin Yu, Michael Geuenich, Sabrina Ge, G. Jang, Tan Tiak Ju, Amy Zhang, G. O’Kane, F. Notta, Kieran R. Campbell
          </td>
          <td>2024-08-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="BACKGROUND
Colorectal cancer is a worldwide leading cause of death accounting for high-rate mortality. Mutations in the epidermal growth factor receptor and RAS/MAPK pathways, as well as altered methylation genes profiles, have been described as molecular mechanisms promoting and sustaining tumour development and progression. Aberrant methylation is a well-known epigenetic mechanism involved in gene regulation; particularly several genes were reported as hypermethylated in CRC. Recently, it was shown that epigenetic alterations in genes such as neuropeptide y, proenkephalin and Wnt inhibitory factor 1 can be used as promising disease biomarkers. Almost all methods developed for the DNA methylation analysis combined next generation sequencing, conventional qRT-PCR or ddPCR with the prior DNA modification with sodium bisulfite.


METHODS AND RESULTS
We implemented a ddPCR method to assess the methylation status of Wnt inhibitory factor 1 and neuropeptide y using the methylation sensitive restriction enzyme approach that does not impact on DNA quality and guarantees the discrimination of DNA methylation independent of bisulfite conversion.


CONCLUSIONS
We showed that this method is robust and sensitive also allowing the monitoring of CRC disease progression when applied to circulating free DNA samples from liquid biopsies, proving to be a fast and easy to implement assay to be used for the monitoring of the methylation pattern of clinically relevant target genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da77d34d6821e2dc47c1fef430ea3b1323e93726" target='_blank'>
              Implementation of a MSRE ddPCR method for the detection of methylated WIF1 and NPY genes in colorectal cancer patients.
              </a>
            </td>
          <td>
            Luca Trentin, Debora Basile, Elena Lazzari, Elena Fietta, Alice Rossi, Filomena Graziani, Alessandro Cappetta, Francesca Simionato, Emanuele D'Amore, Omar Perbellini, Giuseppe Aprile
          </td>
          <td>2024-08-05</td>
          <td>Tumori</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Kirsten Rat Sarcoma (KRAS) is the most commonly mutated oncogene in colorectal carcinoma (CRC). We have previously reported the interactions between microsatellite instability (MSI), DNA promoter methylation, and gene expression. In this study, we looked for associations between KRAS mutation, gene expression, and methylation that may help with precision medicine. Genome-wide gene expression and DNA methylation were done in paired CRC tumor and surrounding healthy tissues. The results suggested that (a) the magnitude of dysregulation of many major gene pathways in CRC was significantly greater in patients with the KRAS mutation, (b) the up- and down-regulation of these dysregulated gene pathways could be correlated with the corresponding hypo- and hyper-methylation, and (c) the up-regulation of CDKN2A was more pronounced in tumors with the KRAS mutation. A recent cell line study showed that there were higher CDKN2A levels in 5-FU-resistant CRC cells and that these could be down-regulated by Villosol. Our findings suggest the possibility of a better response to anti-CDKN2A therapy with Villosol in KRAS-mutant CRC. Also, the more marked up-regulation of genes in the proteasome pathway in CRC tissue, especially with the KRAS mutation and MSI, may suggest a potential role of a proteasome inhibitor (bortezomib, carfilzomib, or ixazomib) in selected CRC patients if necessary.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40b3e427194ea41aedea0dfcb0cf1b6c9f955b47" target='_blank'>
              Association of KRAS Mutation and Gene Pathways in Colorectal Carcinoma: A Transcriptome- and Methylome-Wide Study and Potential Implications for Therapy
              </a>
            </td>
          <td>
            F. Jasmine, Armando Almazan, Yuliia Khamkevych, Marc Bissonnette, Habibul Ahsan, M. Kibriya
          </td>
          <td>2024-07-25</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="BACKGROUND
Mitochondrial defects are thought to play a role in cancer initiation and progression for a long time. Because of the absence of protective histones and an inefficiency in the DNA repair process, mitochondrial DNA is known to be prone to mutations. The deletion of 4977bp is one of the most common mutations in human cancers. This study aimed to investigate the relationship between 4977bp common deletion and Esophageal Squamous Cell Carcinoma Disease (SCC) to provide prognostic information.


METHODS
By using a PCR protocol, this study identified the 4977bp deletion of mtDNA. A PCR method was used on tumor samples from 41 squamous cell carcinoma patients and blood samples from 50 healthy individuals to detect DNA.


RESULTS
Among the 41 tumor samples (80.5%), 33 were found to have the 4977bp deletion, while none of the blood samples from healthy individuals contained it.


CONCLUSIONS
It is shown that the deletion of 4977bp of mtDNA correlates significantly with SCC in this study. A 4977bp deletion could be used as an effective cancer screening indicator and biomarker for early diagnosis and prevention of cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/89d1cf2bd20b900530c920095208d877919594e9" target='_blank'>
              Evaluation of mtDNA common deletion in esophageal squamous cell carcinoma.
              </a>
            </td>
          <td>
            Fatemeh Ghadyani, Shahrbanoo Sharif, Saeid Morovvati
          </td>
          <td>2024-07-23</td>
          <td>Indian journal of cancer</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="INTRODUCTION
Endogenous DNA damage is a significant factor in the damage of hematopoietic cells. Megakaryopoiesis is one of the pathways of hematopoiesis that ends with the production of platelets and plays the most crucial role in hemostasis. Despite the presence of efficient DNA repair mechanisms, some endogenous lesions can lead to mutagenic alterations, disruption of pathways of hematopoiesis including megakaryopoiesis and potentially result in human diseases.


AREAS COVERED
The complex regulation of DNA repair mechanisms plays a central role in maintaining genomic integrity during megakaryopoiesis and influences platelet production efficiency and quality. Moreover, anomalies in DNA repair processes are involved in several diseases associated with megakaryopoiesis, including myeloproliferative disorders and thrombocytopenia.


EXPERT OPINION
In the era of personalized medicine, diagnosing diseases related to megakaryopoiesis can only be made with a complete assessment of their molecular aspects to provide physicians with critical molecular data for patient management and to identify the subset of patients who could benefit from targeted therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c877f3b120520ddb635e7e753f1569dbb2962a5f" target='_blank'>
              DNA damage repair in megakaryopoiesis: molecular and clinical aspects.
              </a>
            </td>
          <td>
            Zeinab Eftekhar, Mojtaba Aghaei, N. Saki
          </td>
          <td>2024-08-08</td>
          <td>Expert review of hematology</td>
          <td>0</td>
          <td>27</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [11],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>