<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="..">
      
      
        <link rel="next" href="../Drug-Resistance%20forecasting/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey for EDC" class="md-header__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey for EDC
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              ecDNA forecasting
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  ecDNA forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-tabs__link">
        
  
    
  
  Drug-Resistance forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-tabs__link">
        
  
    
  
  Spatial-Heterogeneity forecasting

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey for EDC" class="md-nav__button md-logo" aria-label="Literature Survey for EDC" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey for EDC
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    ecDNA forecasting
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Drug-Resistance%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Drug-Resistance forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Spatial-Heterogeneity%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Spatial-Heterogeneity forecasting
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>ecDNA forecasting</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-08-19 06:29:06 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on ecDNA forecasting</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on ecDNA forecasting</a><br>
      <a href="#recommended_articles">3. Recommended articles on ecDNA forecasting</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on ecDNA forecasting</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>ecDNA forecasting</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="In cancer, complex genome rearrangements and other structural alterations, including the amplification of oncogenes on circular extrachromosomal DNA (ecDNA) elements, drive the formation and progression of tumors. ecDNA is a particularly challenging structural alteration. By untethering oncogenes from chromosomal constraints, it elevates oncogene copy number, drives intratumoral genetic heterogeneity, promotes rapid tumor evolution, and results in treatment resistance. The profound changes in DNA shape and nuclear architecture generated by ecDNA alter the transcriptional landscape of tumors by catalyzing new types of regulatory interactions that do not occur on chromosomes. The current suite of tools for interrogating cancer genomes is well suited for deciphering sequence but has limited ability to resolve the complex changes in DNA structure and dynamics that ecDNA generates. Here, we review the challenges of resolving ecDNA form and function and discuss the emerging tool kit for deciphering ecDNA architecture and spatial organization, including what has been learned to date about how this dramatic change in shape alters tumor development, progression, and drug resistance. Expected final online publication date for the Annual Review of Genomics and Human Genetics, Volume 23 is October 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e" target='_blank'>
                Extrachromosomal DNA in Cancer.
                </a>
              </td>
          <td>
            V. Bafna, P. Mischel
          </td>
          <td>2022-05-24</td>
          <td>Annual review of genomics and human genetics</td>
          <td>20</td>
          <td>102</td>

            <td><a href='../recommendations/4fcae11b2d56fafcb92b191ddd9c75cf3d06473e' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Human genes are arranged on 23 pairs of chromosomes, but in cancer, tumor-promoting genes and regulatory elements can free themselves from chromosomes and relocate to circular, extrachromosomal pieces of DNA (ecDNA). ecDNA, because of its nonchromosomal inheritance, drives high-copy-number oncogene amplification and enables tumors to evolve their genomes rapidly. Furthermore, the circular ecDNA architecture fundamentally alters gene regulation and transcription, and the higher-order organization of ecDNA contributes to tumor pathogenesis. Consequently, patients whose cancers harbor ecDNA have significantly shorter survival. Although ecDNA was first observed more than 50 years ago, its critical importance has only recently come to light. In this review, we discuss the current state of understanding of how ecDNAs form and function as well as how they contribute to drug resistance and accelerated cancer evolution. Expected final online publication date for the Annual Review of Pathology: Mechanisms of Disease, Volume 17 is January 2022. Please see http://www.annualreviews.org/page/journal/pubdates for revised estimates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cf6b048483ed593ce03ea93be9c40bac14e4fb74" target='_blank'>
                Extrachromosomal DNA: An Emerging Hallmark in Human Cancer.
                </a>
              </td>
          <td>
            Sihan Wu, V. Bafna, Howard Y. Chang, P. Mischel
          </td>
          <td>2021-11-09</td>
          <td>Annual review of pathology</td>
          <td>50</td>
          <td>130</td>

            <td><a href='../recommendations/cf6b048483ed593ce03ea93be9c40bac14e4fb74' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Extrachromosomal DNAs (ecDNAs) are prevalent in human cancers and mediate high oncogene expression through elevated copy number and altered gene regulation1. Gene expression typically involves distal enhancer DNA elements that contact and activate genes on the same chromosome2,3. Here we show that ecDNA hubs, comprised of ~10-100 ecDNAs clustered in the nucleus of interphase cells, drive intermolecular enhancer input for amplified oncogene expression. Single-molecule sequencing, single-cell multiome, and 3D enhancer connectome reveal subspecies of MYC-PVT1 ecDNAs lacking enhancers that access intermolecular and ectopic enhancer-promoter interactions in ecDNA hubs. ecDNA hubs persist without transcription and are tethered by BET protein BRD4. BET inhibitor JQ1 disperses ecDNA hubs, preferentially inhibits ecDNA oncogene transcription, and kills ecDNA+ cancer cells. Two amplified oncogenes MYC and FGFR2 intermix in ecDNA hubs, engage in intermolecular enhancer-promoter interactions, and transcription is uniformly sensitive to JQ1. Thus, ecDNA hubs are nuclear bodies of many ecDNAs tethered by proteins and platforms for cooperative transcription, leveraging the power of oncogene diversification and combinatorial DNA interactions. We suggest ecDNA hubs, rather than individual ecDNAs, as units of oncogene function, cooperative evolution, and new targets for cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc" target='_blank'>
                EcDNA hubs drive cooperative intermolecular oncogene expression
                </a>
              </td>
          <td>
            King L. Hung, K. Yost, Liangqi Xie, Quanming Shi, K. Helmsauer, Jens Luebeck, R. Schöpflin, Joshua T. Lange, R. Chamorro González, Natasha E Weiser, Celine Y. Chen, Maria E. Valieva, I. Wong, Sihan Wu, S. R. Dehkordi, Connor V. Duffy, K. Kraft, Junfang Tang, J. Belk, John C. Rose, M. Corces, Jeffrey M. Granja, Rui Li, Utkrisht Rajkumar, Jordan Friedlein, A. Bagchi, Ansuman T. Satpathy, R. Tjian, S. Mundlos, V. Bafna, A. Henssen, P. Mischel, Zhe J. Liu, Howard Y. Chang
          </td>
          <td>2020-11-20</td>
          <td>bioRxiv</td>
          <td>98</td>
          <td>142</td>

            <td><a href='../recommendations/3a89d9a5a15ed5b317861a40b5eed1bef9ac8adc' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0aadcbbf8935c1dedaa1188ba380c871eb908e70" target='_blank'>
                Extrachromosomal DNA is associated with oncogene amplification and poor outcome across multiple cancers
                </a>
              </td>
          <td>
            Hoon Kim, Nam Nguyen, Kristen M. Turner, Sihan Wu, A. Gujar, Jens Luebeck, Jihe Liu, Viraj Deshpande, Utkrisht Rajkumar, Sandeep Namburi, S. Amin, E. Yi, F. Menghi, J. Schulte, A. Henssen, Howard Y. Chang, C. Beck, P. Mischel, V. Bafna, R. Verhaak
          </td>
          <td>2020-07-19</td>
          <td>Nature genetics</td>
          <td>272</td>
          <td>130</td>

            <td><a href='../recommendations/0aadcbbf8935c1dedaa1188ba380c871eb908e70' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82c7ff8550ce34adb59580b2989a550b6faded59" target='_blank'>
                Chromothripsis drives the evolution of gene amplification in cancer
                </a>
              </td>
          <td>
            Ofer Shoshani, Simon F. Brunner, R. Yaeger, P. Ly, Yael Nechemia-Arbely, D. H. Kim, Rongxin Fang, Guillaume A. Castillon, Miao Yu, Julia S Z Li, Ying Sun, Mark Ellisman, B. Ren, P. Campbell, D. Cleveland
          </td>
          <td>2020-12-23</td>
          <td>Nature</td>
          <td>233</td>
          <td>156</td>

            <td><a href='../recommendations/82c7ff8550ce34adb59580b2989a550b6faded59' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for ecDNA forecasting'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>ecDNA forecasting</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="Structural variants (SVs) represent a mechanism by which cancer cells activate oncogenes or disrupt the function of genes with tumor suppressor roles. A recent study by the PCAWG Consortium investigated structural variants in 30 tumor types, identifying focal rearrangements in the oncogenic BRD4 gene in ovarian, endometrial and breast cancers. These rearrangements resulted in decreased BRD4 expression despite increased copy number, suggesting a novel mechanism to finetune gene over-expression. In this study, we show that focal deletions of BRD4 disrupt genomic regulatory regions and impact gene isoform expression in breast and ovarian tumors when compared to their expression across normal tissues. To determine the functional impact of these concomitant amplification and focal deletion events, we first leveraged open-reading-frame (ORF) screen data from 16 cancer cell lines, where we observed that overexpression of BRD4-long and BRD4-short isoforms is toxic for cancer cell growth. We confirmed these results in OVSAHO ovarian cancer cells, where overexpression of both isoforms significantly reduced tumor growth. Next, we mimicked the focal deletions occurring in BRD4 regulatory regions by CRISPR-Cas9 technology, and observed that their depletion functionally ablates tumor cell growth. We finally show that these focal deletions rescue ovarian carcinoma cells from the toxicity effects associated with gene overexpression, suggesting that global BRD4 gene expression levels must be fine tuned to ensure proper cancer cell proliferation. Our study provides experimental evidence for BRD4 deletions constituting the first example of a driver SV alteration reducing toxicity in cancer, therefore expanding the landscape of cancer progression mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81e02489180577a24d50b9dd3dde1ae9ce016420" target='_blank'>
              Recurrent breakpoints in the BRD4 locus reduce toxicity associated with gene amplification
              </a>
            </td>
          <td>
            J. Wala, Simona Dalin, Sophie Webster, Ofer Shapira, John P. Busanovich, R. Beroukhim, P. Bandopadhayay, Verónica Rendo
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>144</td>
        </tr>

        <tr id="Circular extrachromosomal DNA (ecDNA) is a form of oncogene amplification found across cancer types and associated with poor outcome in patients. ecDNA can be structurally complex and contain rearranged DNA sequences derived from multiple chromosome locations. As the structure of ecDNA can impact oncogene regulation and may indicate mechanisms of its formation, disentangling it at high resolution from sequencing data is essential. Even though methods have been developed to identify and reconstruct ecDNA in cancer genome sequencing, it remains challenging to resolve complex ecDNA structures, in particular amplicons with shared genomic footprints. We here introduce Decoil, a computational method which combines a breakpoint-graph approach with regression to reconstruct complex ecDNA and deconvolve co-occurring ecDNA elements with overlapping genomic footprints from long-read nanopore sequencing. Decoil outperforms de novo assembly and alignment-based methods in simulated long-read sequencing data for both simple and complex ecDNAs. Applying Decoil on whole genome sequencing data uncovered different ecDNA topologies and explored ecDNA structure heterogeneity in neuroblastoma tumors and cell lines, indicating that this method may improve ecDNA structural analyzes in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c914721ddd777cd78df6632d6860e72cff344be" target='_blank'>
              Reconstructing extrachromosomal DNA structural heterogeneity from long-read sequencing data using Decoil.
              </a>
            </td>
          <td>
            M. Giurgiu, Nadine Wittstruck, Elias Rodriguez-Fos, R. Chamorro González, Lotte Brueckner, Annabell Krienelke-Szymansky, K. Helmsauer, Anne Hartebrodt, P. Euskirchen, R. Koche, Kerstin Haase, Knut Reinert, A. Henssen
          </td>
          <td>2024-08-07</td>
          <td>Genome research</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Abstract Recently, the cancer community has gained a heightened awareness of the roles of extrachromosomal DNA (ecDNA) in cancer proliferation, drug resistance and epigenetic remodeling. However, a hindrance to studying ecDNA is the lack of available cancer model systems that express ecDNA. Increasing our awareness of which model systems express ecDNA will advance our understanding of fundamental ecDNA biology and unlock a wealth of potential targeting strategies for ecDNA-driven cancers. To bridge this gap, we created CytoCellDB, a resource that provides karyotype annotations for cell lines within the Cancer Dependency Map (DepMap) and the Cancer Cell Line Encyclopedia (CCLE). We identify 139 cell lines that express ecDNA, a 200% increase from what is currently known. We expanded the total number of cancer cell lines with ecDNA annotations to 577, which is a 400% increase, covering 31% of cell lines in CCLE/DepMap. We experimentally validate several cell lines that we predict express ecDNA or homogeneous staining regions (HSRs). We demonstrate that CytoCellDB can be used to characterize aneuploidy alongside other molecular phenotypes, (gene essentialities, drug sensitivities, gene expression). We anticipate that CytoCellDB will advance cytogenomics research as well as provide insights into strategies for developing therapeutics that overcome ecDNA-driven drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e61e6842068f0934c45c7bca8373df3c0dbd9903" target='_blank'>
              CytoCellDB: a comprehensive resource for exploring extrachromosomal DNA in cancer cell lines
              </a>
            </td>
          <td>
            Jacob Fessler, Stephanie Ting, Hong Yi, Santiago Haase, Jingting Chen, Saygin Gulec, Yue Wang, Nathan Smyers, Kohen Goble, Danielle Cannon, Aarav Mehta, Christina Ford, Elizabeth Brunk
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5988602f52499713b4d1d42d555664a971cb3e9" target='_blank'>
              3D genomic analysis reveals novel enhancer-hijacking caused by complex structural alterations that drive oncogene overexpression
              </a>
            </td>
          <td>
            Katelyn L. Mortenson, Courtney Dawes, Emily R. Wilson, Nathan E. Patchen, Hailey E Johnson, Jason Gertz, Swneke D. Bailey, Yang Liu, K. Varley, Xiaoyang Zhang
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA), an extrachromosomal circular structured DNA, is extensively found in eukaryotes. Exploring eccDNA at the single-cell level contributes to understanding heterogeneity, evolution, development, and specific functions within cells. Nevertheless, the high-throughput identification methods for single-cell eccDNA are complex, and currently mature and widely applicable technologies are lacking. Those factors have led to a scarcity of resources for studying eccDNA at the single-cell level. Therefore, using available single-cell whole-genome sequencing (WGS) data, we have constructed a comprehensive eccDNA database named scEccDNAdb (https://lcbb.swjtu.edu.cn/scEccDNAdb/). Presently, scEccDNAdb comprises 3,195,464 single-cell eccDNA entries from both disease/health human and mouse samples, which provides comprehensive annotations including oncogenes, typical enhancers, super-enhancers, CTCF binding sites, SNPs, chromatin accessibility, eQTLs, transcription factor binding sites, motifs, and SVs. Additionally, it provides nine online analysis and visualization tools, facilitating the generation of publication-quality figures for eccDNA analysis through the upload of customized files. Overall, scEccDNAdb represents the first comprehensive database known to us for exploring and analyzing single-cell eccDNA data in diverse cell types, tissues, and species. Graphical abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/322c06c5550a609619dd96de2cba0b3494554b2a" target='_blank'>
              scEccDNAdb: an integrated single-cell eccDNA resource for human and mouse
              </a>
            </td>
          <td>
            Wenqing Wang, Xinyu Zhao, Tianyu Ma, Tengwei Zhong, Junnuo Zheng, Yi Yang, Yuanbiao Guo, Zhiyun Guo
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Extrachromosomal circular DNA (eccDNA) is a form of a circular double-stranded DNA that exists independently of conventional chromosomes. eccDNA exhibits a broad and random distribution across eukaryotic cells and has been associated with tumor-related properties due to its ability to harbor the complete gene information of oncogenes. The complex and multifaceted mechanisms underlying eccDNA formation include pathways such as DNA damage repair, breakage–fusion–bridge (BFB) mechanisms, chromothripsis, and cell apoptosis. Of note, eccDNA plays a pivotal role in tumor development, genetic heterogeneity, and therapeutic resistance. The high copy number and transcriptional activity of oncogenes carried by eccDNA contribute to the accelerated growth of tumors. Notably, the amplification of oncogenes on eccDNA is implicated in the malignant progression of cancer cells. The improvement of high-throughput sequencing techniques has greatly enhanced our knowledge of eccDNA by allowing for a detailed examination of its genetic structures and functions. However, we still lack a comprehensive and efficient annotation for eccDNA, while challenges persist in the study and understanding of the functional role of eccDNA, emphasizing the need for the development of robust methodologies. The potential clinical applications of eccDNA, such as its role as a measurable biomarker or therapeutic target in diseases, particularly within the spectrum of human malignancies, is a promising field for future research. In conclusion, eccDNA represents a quite dynamic and multifunctional genetic entity with far-reaching implications in cancer pathogenesis and beyond. Further research is essential to unravel the molecular pathways of eccDNA formation, elucidate its functional roles, and explore its clinical applications. Addressing these aspects is crucial for advancing our understanding of genomic instability and developing novel strategies for tailored therapeutics, especially in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eba9d45916b5521fe06e0dfbffaa25ccf1f330f0" target='_blank'>
              Beyond the Chromosome: Recent Developments in Decoding the Significance of Extrachromosomal Circular DNA (eccDNA) in Human Malignancies
              </a>
            </td>
          <td>
            P. Tsiakanikas, Konstantina Athanasopoulou, Ioanna A. Darioti, Vasiliki Taxiarchoula Agiassoti, Stamatis Theocharis, Andreas Scorilas, P. Adamopoulos
          </td>
          <td>2024-07-24</td>
          <td>Life</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Oncogene amplification on extrachromosomal DNA (ecDNA) is a pervasive driver event in cancer, yet our understanding of how ecDNA forms is limited. Here, we couple a CRISPR-based method for ecDNA induction with extensive characterization of newly formed ecDNA to examine their biogenesis. We find that DNA circularization is efficient, irrespective of 3D genome context, with formation of 800kb, 1 Mb, and 1.8 Mb ecDNAs reaching or exceeding 15%. We show non-homologous end joining and microhomology-mediated end joining both contribute to ecDNA formation, while inhibition of DNA-PKcs and ATM have opposing impacts on ecDNA formation. EcDNA and the corresponding chromosomal excision scar can form at significantly different rates and respond differently to DNA-PKcs and ATM inhibition. Taken together, our results support a model of ecDNA formation in which double strand break ends dissociate from their legitimate ligation partners prior to joining of illegitimate ends to form the ecDNA and excision scar.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e488ec3b1ac40e7e2986571eab100d1986b6e3f" target='_blank'>
              Disparate pathways for extrachromosomal DNA biogenesis and genomic DNA repair.
              </a>
            </td>
          <td>
            John C. Rose, J. Belk, I. Wong, Jens Luebeck, Hudson T Horn, Bence Daniel, Matthew G Jones, Kathryn E Yost, King L. Hung, Kevin S Kolahi, Ellis J Curtis, Calvin J. Kuo, V. Bafna, Paul S. Mischel, Howard Y. Chang
          </td>
          <td>2024-08-07</td>
          <td>Cancer discovery</td>
          <td>0</td>
          <td>71</td>
        </tr>

        <tr id="Sequencing-based mapping of ensemble pairwise interactions among regulatory elements support the existence of topological assemblies known as promoter-enhancer hubs or cliques in cancer. Yet, prevalence, regulators, and functions of promoter-enhancer hubs in individual cancer cells remain unclear. Here, we systematically integrated functional genomics, transcription factor screening, and optical mapping of promoter-enhancer interactions to identify key promoter-enhancer hubs, examine heterogeneity of their assembly, determine their regulators, and elucidate their role in gene expression control in individual triple negative breast cancer (TNBC) cells. Optical mapping of individual SOX9 and MYC alleles revealed the existence of frequent multiway interactions among promoters and enhancers within spatial hubs. Our single-allele studies further demonstrated that lineage-determining SOX9 and signaling-dependent NOTCH1 transcription factors compact MYC and SOX9 hubs. Together, our findings suggest that promoter-enhancer hubs are dynamic and heterogeneous topological assemblies, which are controlled by oncogenic transcription factors and facilitate subtype-restricted gene expression in cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab8df926269367eb0273d95bc28b249a4b8bb66d" target='_blank'>
              Oncogenic transcription factors instruct promoter-enhancer hubs in individual triple negative breast cancer cells
              </a>
            </td>
          <td>
            Jingru Zhao, Yeqiao Zhou, Ilias Tzelepis, Noah G Burget, Junwei Shi, R. Faryabi
          </td>
          <td>2024-08-07</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Whole-genome doubling (WGD) is a critical driver of tumor development and is linked to drug resistance and metastasis in solid malignancies. Here, we demonstrate that WGD is an ongoing mutational process in tumor evolution. Using single-cell whole-genome sequencing, we measured and modeled how WGD events are distributed across cellular populations within tumors and associated WGD dynamics with properties of genome diversification and phenotypic consequences of innate immunity. We studied WGD evolution in 65 high-grade serous ovarian cancer (HGSOC) tissue samples from 40 patients, yielding 29,481 tumor cell genomes. We found near-ubiquitous evidence of WGD as an ongoing mutational process promoting cell-cell diversity, high rates of chromosomal missegregation, and consequent micronucleation. Using a novel mutation-based WGD timing method, doubleTime, we delineated specific modes by which WGD can drive tumor evolution: (i) unitary evolutionary origin followed by significant diversification, (ii) independent WGD events on a pre-existing background of copy number diversity, and (iii) evolutionarily late clonal expansions of WGD populations. Additionally, through integrated single-cell RNA sequencing and high-resolution immunofluorescence microscopy, we found that inflammatory signaling and cGAS-STING pathway activation result from ongoing chromosomal instability and are restricted to tumors that remain predominantly diploid. This contrasted with predominantly WGD tumors, which exhibited significant quiescent and immunosuppressive phenotypic states. Together, these findings establish WGD as an evolutionarily ‘active’ mutational process that promotes evolvability and dysregulated immunity in late stage ovarian cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7492c9bf200e273b7b6b57e3677bc99bc004737e" target='_blank'>
              Ongoing genome doubling promotes evolvability and immune dysregulation in ovarian cancer
              </a>
            </td>
          <td>
            Andrew McPherson, I. Vázquez-García, Matthew A. Myers, Matthew Zatzman, Duaa H Al-Rawi, A. Weiner, Samuel S. Freeman, N. Mohibullah, Gryte Satas, Marc J. Williams, Nick Ceglia, Allen W. Zhang, Jun Li, J. Lim, Michelle Wu, Seongmin Choi, Eliyahu Havasov, Diljot S. Grewal, Hongyu Shi, Minsoo Kim, RF Schwarz, Tom L. Kaufmann, K. Dinh, F. Uhlitz, Julie Tran, Yushi Wu, Ruchi Patel, Satish Ramakrishnan, DooA Kim, Justin Clarke, Hunter Green, Emily Ali, Melody DiBona, Nancy Varice, Ritika Kundra, V. Broach, G. Gardner, K. Roche, Y. Sonoda, O. Zivanovic, Sarah H. Kim, Rachel Grisham, Ying L. Liu, A. Viale, Nicole Rusk, Y. Lakhman, Lora Hedrick Ellenson, Simon Tavaré, Samuel Aparicio, Dennis S Chi, Carol Aghajanian, N. Abu-Rustum, C. Friedman, Dmitriy Zamarin, Britta Weigelt, S. Bakhoum, Sohrab P. Shah
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Recent advances in high-resolution mapping of spatial interactions among regulatory elements support the existence of complex topological assemblies of enhancers and promoters known as enhancer-promoter hubs or cliques. Yet, organization principles of these multi-interacting enhancer-promoter hubs and their potential role in regulating gene expression in cancer remains unclear. Here, we systematically identified enhancer-promoter hubs in breast cancer, lymphoma, and leukemia. We found that highly interacting enhancer-promoter hubs form at key oncogenes and lineage-associated transcription factors potentially promoting oncogenesis of these diverse cancer types. Genomic and optical mapping of interactions among enhancer and promoter elements further showed that topological alterations in hubs coincide with transcriptional changes underlying acquired resistance to targeted therapy in T cell leukemia and B cell lymphoma. Together, our findings suggest that enhancer-promoter hubs are dynamic and heterogeneous topological assemblies with the potential to control gene expression circuits promoting oncogenesis and drug resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9831a80afed1f71511cdd9f1e5ce45eefd2262c2" target='_blank'>
              Enhancer-promoter hubs organize transcriptional networks promoting oncogenesis and drug resistance
              </a>
            </td>
          <td>
            Brent S. Perlman, Noah G Burget, Yeqiao Zhou, Gregory W. Schwartz, Jelena Petrovic, Z. Modrusan, R. Faryabi
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>90</td>
        </tr>

        <tr id="Somatic structural variations (SVs) in cancer can shuffle DNA content in the genome, relocate regulatory elements, and alter genome organization. Enhancer hijacking occurs when SVs relocate distal enhancers to activate proto-oncogenes. However, most enhancer hijacking studies have only focused on protein-coding genes. Here, we develop a computational algorithm 'HYENA' to identify candidate oncogenes (both protein-coding and non-coding) activated by enhancer hijacking based on tumor whole-genome and transcriptome sequencing data. HYENA detects genes whose elevated expression is associated with somatic SVs by using a rank-based regression model. We systematically analyze 1146 tumors across 25 types of adult tumors and identify a total of 108 candidate oncogenes including many non-coding genes. A long non-coding RNA TOB1-AS1 is activated by various types of SVs in 10% of pancreatic cancers through altered 3-dimensional genome structure. We find that high expression of TOB1-AS1 can promote cell invasion and metastasis. Our study highlights the contribution of genetic alterations in non-coding regions to tumorigenesis and tumor progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5fb0cf8831bea2891dbca4714836cde381f2ca0" target='_blank'>
              HYENA detects oncogenes activated by distal enhancers in cancer.
              </a>
            </td>
          <td>
            Anqi Yu, Ali E. Yesilkanal, Ashish Thakur, Fan Wang, Yang Yang, William Phillips, Xiaoyang Wu, Alexander Muir, Xin He, Francois Spitz, Lixing Yang
          </td>
          <td>2024-07-25</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2dec3a33e2b0139c6a02f10afa4c9e80abbaad7a" target='_blank'>
              Extrachromosomal circular DNA promotes prostate cancer progression through the FAM84B/CDKN1B/MYC/WWP1 axis
              </a>
            </td>
          <td>
            Wei Jin, Zhenqun Xu, Yan Song, Fangjie Chen
          </td>
          <td>2024-07-12</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="INTRODUCTION
Isodicentric Y chromosomes are relatively common structural variants of the human genome. The underlying mechanism of isodicentric Y chromosomes with short arm breakpoints [idic(Yq)] remains to be clarified.


CASE PRESENTATION
We encountered a Japanese man with azoospermia and mild short stature. G-banding and array-based comparative genomic hybridization indicated that his karyotype was 45,X/46,X,idic(Y)(qter→p11.32::p11.32→qter) with a ~1.8 Mb terminal deletion. Whole genome sequencing suggested that the Y chromosome had four breakpoints in a ~7 kb region of the pseudoautosomal region 1 (PAR1).


CONCLUSION
This case was assumed to have an idic(Yq) resulting from multiple DNA double-strand breaks in PAR1. This rearrangement may have been facilitated by the PAR1-specific chromatin architecture. The clinical features of the patient can be ascribed to SHOX haploinsufficiency and the presence of a 45,X cell line, although copy-number gains of some Yq genes and the size reduction of PAR1 may also contribute to his spermatogenic failure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ad6ffa4553f26785a976ed64f6e1aecf9e157203" target='_blank'>
              Isodicentric Y chromosome with multiple breakpoints in the pseudoautosomal region 1.
              </a>
            </td>
          <td>
            Yasuko Ogiwara, Y. Kobori, E. Suzuki, Atsushi Hattori, Kanako Tanase-Nakao, Akiyoshi Osaka, Toshiyuki Iwahata, Hiroshi Okada, Y. Kuroki, M. Fukami
          </td>
          <td>2024-07-29</td>
          <td>Cytogenetic and genome research</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="Although somatic cell genomes are usually entirely clonally inherited, nuclear DNA exchange between cells of an organism can occur sporadically by cell fusion, phagocytosis or other mechanisms1–3. This phenomenon has long been noted in the context of cancer, where it could be envisaged that DNA horizontal transfer plays a functional role in disease evolution4–13. However, an understanding of the frequency and significance of this process in naturally occurring tumours is lacking. The host-tumour genetic discordance of transmissible cancers, malignant clones which pass between animals as allogeneic grafts, provides an opportunity to investigate this. We screened for host-to-tumour horizontal transfer of nuclear DNA in 174 tumours from three transmissible cancers affecting dogs and Tasmanian devils, and detected a single instance in the canine transmissible venereal tumour (CTVT). This involved introduction of a 15-megabase dicentric genetic element, composed of 11 rearranged fragments of six chromosomes, to a CTVT sublineage occurring in Asia around 2,000 years ago. The element forms the short arm of a small submetacentric chromosome, and derives from a dog with ancestry associated with the ancient Middle East. The introduced DNA fragment is transcriptionally active and has adopted the expression profile of CTVT. Its 143 genes do not, however, confer any obvious advantage to its spatially restricted CTVT sublineage. Our findings indicate that nuclear DNA horizontal transfer, although likely a rare event in tumour evolution, provides a viable mechanism for the acquisition of genetic material in naturally occurring cancer genomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0a14fc4c5f037668099865f0d0c54dfd906b6685" target='_blank'>
              Horizontal transfer of nuclear DNA in transmissible cancer
              </a>
            </td>
          <td>
            Kevin Gori, Adrian Baez-Ortega, A. Strakova, M. Stammnitz, Jinhong Wang, Jonathan Chan, Katherine Hughes, Sophia Belkhir, Maurine Hammel, Daniela Moralli, James Bancroft, Edward Drydale, Karen M. Allum, María Verónica Brignone, Anne M Corrigan, Karina F. de Castro, Edward M. Donelan, I. Faramade, Alison Hayes, Nataliia Ignatenko, Rockson Karmacharya, Debbie Koenig, M. Lanza-Perea, A. L. Lopez Quintana, Michael Meyer, W. Neunzig, F. Pedraza-Ordóñez, Yoenten Phuentshok, K. Phuntsho, Juan C. Ramirez-Ante, J. Reece, Sheila K. Schmeling, Sanjay Singh, Lester J. Tapia Martínez, Marian Taulescu, Samir Thapa, Sunil Thapa, Mirjam G. van der Wel, Alvaro S. Wehrle-Martinez, Michael R Stratton, E. Murchison
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01c82722b4065aa54e731d4f283926374f9696da" target='_blank'>
              Chromosomal instability as a driver of cancer progression.
              </a>
            </td>
          <td>
            Xuelan Chen, Albert S. Agustinus, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2196d99b39ce23824b3c9030fb9688e8f81117c7" target='_blank'>
              Replication timing alterations are associated with mutation acquisition during breast and lung cancer evolution
              </a>
            </td>
          <td>
            M. Dietzen, Haoran Zhai, Olivia Lucas, O. Pich, Christopher Barrington, Wei-Ting Lu, S. Ward, Yanping Guo, R. Hynds, S. Zaccaria, C. Swanton, N. Mcgranahan, N. Kanu
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>66</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e9fb6e4e6751f59e5a7f630fe095f0e2f627ca4d" target='_blank'>
              Copy number losses of oncogenes and gains of tumor suppressor genes generate common driver mutations
              </a>
            </td>
          <td>
            Elizaveta Besedina, F. Supek
          </td>
          <td>2024-07-20</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Human tumors progress in part by accumulating epigenetic alterations, which include gains and losses of DNA methylation in different parts of the cancer cell genome. Recent work has revealed a link between these two opposite alterations by showing that DNA hypomethylation in tumors can induce the expression of transcripts that overlap downstream gene promoters and thereby induce their hypermethylation. Preliminary in silico evidence prompted us to investigate if this mechanism applies to the locus harboring AGO1, a gene that plays a central role in miRNA biogenesis and RNA interference. Inspection of public RNA-Seq datasets and RT-qPCR experiments show that an alternative transcript starting 13.4 kb upstream of AGO1 (AGO1-V2) is expressed specifically in testicular germ cells, and becomes aberrantly activated in different types of tumors, particularly in tumors of the esophagus, stomach, and lung. This expression pattern classifies AGO1-V2 into the group of “Cancer-Germline” (CG) genes. Analysis of transcriptomic and methylomic datasets provided evidence that transcriptional activation of AGO1-V2 depends on DNA demethylation of its promoter region. Western blot experiments revealed that AGO1-V2 encodes a shortened isoform of AGO1, corresponding to a truncation of 75 aa in the N-terminal domain, and which we therefore referred to as “∆NAGO1”. Interestingly, significant correlations between hypomethylation/activation of AGO1-V2 and hypermethylation/repression of AGO1 were observed upon examination of tumor cell lines and tissue datasets. Overall, our study reveals the existence of a process of interdependent epigenetic alterations in the AGO1 locus, which promotes swapping between two AGO1 protein-coding mRNA isoforms in tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27dc2f1e01e40a64b0f8910cba78bb0f22899826" target='_blank'>
              DNA Hypomethylation Underlies Epigenetic Swapping between AGO1 and AGO1-V2 Isoforms in Tumors
              </a>
            </td>
          <td>
            Jean S. Fain, Camille Wangermez, A. Loriot, Claudia Denoue, C. De Smet
          </td>
          <td>2024-06-22</td>
          <td>Epigenomes</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b93e43a1982115416b67edb36296e1d7dd7f71fe" target='_blank'>
              The DNA methylome of pediatric brain tumors appears shaped by structural variation and predicts survival
              </a>
            </td>
          <td>
            Fengju Chen, Yiqun Zhang, Lanlan Shen, Chad J. Creighton
          </td>
          <td>2024-08-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="ABSTRACT Aneuploidy, the presence of an aberrant number of chromosomes, has been associated with tumorigenesis for over a century. More recently, advances in karyotyping techniques have revealed its high prevalence in cancer: About 90% of solid tumors and 50–70% of hematopoietic cancers exhibit chromosome gains or losses. When analyzed at the level of specific chromosomes, there are strong patterns that are observed in cancer karyotypes both pan-cancer and for specific cancer types. These specific aneuploidy patterns correlate strongly with outcomes for tumor initiation, progression, metastasis formation, immune evasion and resistance to therapeutic treatment. Despite their prominence, understanding the basis underlying aneuploidy patterns in cancer has been challenging. Advances in genetic engineering and bioinformatic analyses now offer insights into the genetic determinants of aneuploidy pattern selection. Overall, there is substantial evidence that expression changes of particular genes can act as the positive selective forces for adaptation through aneuploidy. Recent findings suggest that multiple genes contribute to the selection of specific aneuploid chromosomes in cancer; however, further research is necessary to identify the most impactful driver genes. Determining the genetic basis and accompanying vulnerabilities of specific aneuploidy patterns is an essential step in selectively targeting these hallmarks of tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0123dd290f56b3193001c1283aea1071e6f411ed" target='_blank'>
              Selection forces underlying aneuploidy patterns in cancer
              </a>
            </td>
          <td>
            Tamara C. Klockner, Christopher S. Campbell
          </td>
          <td>2024-06-24</td>
          <td>Molecular & Cellular Oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Dysregulation of epigenetic processes leads to a plethora of abnormalities including disease states such as cancer. Therapies focused on epigenetic modulation alter gene expression to correct dysfunction, though the mechanisms and perpetuation of these states is unknown. Here, we use integrated epigenomics and three-dimensional chromatin structure-function analyses after acute histone deacetylase inhibitor cancer drug treatment (suberoylanilide hydroxamic acid in lung cancer cells). Treatment induced substantial (13%) genomic rearrangement that rebounds despite persistent gene expression changes and spreading of acetylation. The chromatin functional landscape (accessibility, active transcription modification, and gene expression) is controlled and locus-specific, while chromatin contacts are globally altered resulting in a moderate weakening of topologically associating domains. Chromatin states are more dynamic at transcriptionally active loci while genes with reduced expression are epigenetically stable suggesting chromatin architectural turnover and nucleosome remodeling is locus-specific and underlies the bidirectional expression changes. Thus, local 3D chromatin and genome structural dynamics is integral for loci regulation in response to epigenomic perturbation. The partial persistence of these altered features may have larger implications for efficacy of epigenetic drugs in amelioration of disease states.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/19da33c78f7874d1b2e6e72af7313f058dabfa1d" target='_blank'>
              Epigenomic manipulation reveals the relationship between locus specific chromatin dynamics and gene expression
              </a>
            </td>
          <td>
            Vrinda Venu, Eric M Small, Cullen Roth, Samantha H. Adikari, Anna Hendrika Cornelia Vlot, Kyle A. Sullivan, Chanaka Roshan Abeyratne, Daniel A Jacobson, Shawns Starkenburg, K. Sanbonmatsu, Christina R. Steadman
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Copy number alterations (CNAs), resulting from the gain or loss of genetic material from as little as 50 base pairs or as big as entire chromosome(s), have been associated with many congenital diseases, de novo syndromes and cancer. It is established that CNAs disturb the dosage of genomic regions including enhancers/promoters, long non-coding RNA and gene(s) among others, ultimately leading to an altered balance of key cellular functions. In cancer, CNAs have been associated with almost all steps of the disease: predisposition, initiation, development, maintenance, response to treatment, resistance, and relapse. Therefore, understanding how specific CNAs contribute to tumourigenesis may provide prognostic insight and ultimately lead to the development of new therapeutic approaches to improve patient outcomes. In this review, we provide a snapshot of what is currently known about CNAs and cancer, incorporating topics regarding their detection, clinical impact, origin, and nature, and discuss the integration of innovative genetic engineering strategies, to highlight the potential for targeting CNAs using novel, dosage-sensitive and less toxic therapies for CNA-driven cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b107cf5f7a3a29e8e0fe916d4102734572eefc3a" target='_blank'>
              Insights into the Clinical, Biological and Therapeutic Impact of Copy Number Alteration in Cancer
              </a>
            </td>
          <td>
            Shannon L. Carey-Smith, R. Kotecha, L. C. Cheung, Sébastien Malinge
          </td>
          <td>2024-06-21</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Introduction Type B1 and B2 thymomas are lymphocyte-rich malignant tumors with few somatic mutations in protein-coding regions of the nuclear genome; nonetheless, non-coding regions remain uncharacterized. Here, we developed a rigorous tumor isolation method from lymphocyte-rich thymoma tissues and identified somatic mutations in non-coding and mitochondrial DNA. Methods CD205+CD45- pure tumor cells were isolated from fresh-frozen tissues using DEPArray system. Deep whole-genome sequencing was performed, and recurrent somatic alterations in coding, non-coding, and mitochondria regions were systemically identified by computational framework. The mutations were classified according to gene function, cis-regulatory element, and mutational signature. Results The total number of somatic mutations was approximately 80 times higher in non-coding regions than in coding regions in type B1-2 thymomas (1,671.3 vs. 21.1 per case). Coding mutations were identified in epigenetic regulators, DNA repair genes, and some other genes. Nevertheless, 40% of cases exhibited fewer than four mutations in coding regions. A systematic non-coding analysis identified a total of 405.0 mutations per case on cis-regulatory elements, and detected six recurrent mutations: one interferon regulatory factor (IRF8), two E3 ubiquitin ligases (UBR2 and RNF213), and three intergenic regions. Mitochondrial heteroplasmy was observed in 90% of cases, with a significant proportion of mutations located in D-loop region. The single-base substitution pattern was signature 12. Conclusions Numerous non-coding mutations and mitochondrial heteroplasmy were detected in type B1 and B2 thymomas. Given the paucity of coding mutations observed in this disease entity, disruption of the non-coding landscape and mitochondrial heteroplasmic shift may be the primary cause of thymoma.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/559b9056aa5e8bd421441f43ad9a55ae5b38dbc9" target='_blank'>
              Genome-wide analysis of somatic non-coding mutation patterns and mitochondrial heteroplasmy in type B1 and B2 thymomas
              </a>
            </td>
          <td>
            Kohei Fujikura, Isabel Correa, Susanne Heck, Juliet King, Emma McLean, Andrea Billè, Daisuke Nonaka
          </td>
          <td>2024-08-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We adapted a previously developed targeted single-nucleus DNA sequencing (snDNA-seq) method and constructed a new suite of computational analysis tools to study 137,491 single-nucleus DNA libraries from 24 pancreatic cancers collected under a variety of clinical scenarios including early and late diagnoses, different metastatic sites and before- and after-treatment. We refined the mutational landscape of pancreatic cancer by capturing events missed by bulk sequencing, and validated the evolution pattern of early fixation of driver single-nucleotide variants (SNVs) followed by generation of intratumoral heterogeneity for copy number variations (CNVs). Intertumoral convergent evolution was common, including subclonal inactivation of TGF-β pathway by mutating various components of it; intratumoral convergence was rarely observed, likely due to strong selective force in pancreatic cancer development. Continuous evolution was frequently seen manifesting as CNVs. In the context of non-targeted treatments, no particular pattern was found across metastases or through treatment. In six pancreatic cancers with germline BRCA2 mutation, we discovered varied timing of biallelic inactivation of BRCA2, which sculpted different evolutionary trajectories and could presumably contribute to differential response to treatment. As the first large-scale application of targeted snDNA-seq on pancreatic cancer, this work provides a sample processing and computational analysis pipeline that warrants further clinical utility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcb8661871968f256a1ef9033377a1300e9ea64b" target='_blank'>
              Genomic evolution of pancreatic cancer at single-cell resolution
              </a>
            </td>
          <td>
            Haochen Zhang, P. Sashittal, Elias-Ramzey Karnoub, Akhil Jakatdar, Shigeaki Umeda, Jungeui Hong, Anne Marie Noronha, Agustin Cardenas, Amanda Erakky, Caitlin A. McIntyre, Akimasa Hayashi, Nicolas Lecomte, Wungki Park, Nan Pang, Eileen M. O'Reilly, Alice C. Wei, Benjamin J. Raphael, Christine A. Iacobuzio-Donahue
          </td>
          <td>2024-08-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="RNA polymerase II (Pol II) dysfunction is frequently implied in human disease. Understanding its functional mechanism is essential for designing innovative therapeutic strategies. To visualize its supra-molecular interactions with genes and nascent RNA, we generated a human cell line carrying ~335 consecutive copies of a recombinant β-globin gene. Confocal microscopy showed that Pol II was not homogeneously concentrated around these identical gene copies. Moreover, Pol II signals partially overlapped with the genes and their nascent RNA, revealing extensive compartmentalization. Using a cell line carrying a single copy of the β-globin gene, we also tested if the binding of catalytically dead CRISPR-associated system 9 (dCas9) to different gene regions affected Pol II transcriptional activity. We assessed Pol II localization and nascent RNA levels using chromatin immunoprecipitation and droplet digital reverse transcription PCR, respectively. Some enrichment of transcriptionally paused Pol II accumulated in the promoter region was detected in a strand-specific way of gRNA binding, and there was no decrease in nascent RNA levels. Pol II preserved its transcriptional activity in the presence of DNA-bound dCas9. Our findings contribute further insight into the complex mechanism of mRNA transcription in human cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5b25d997a1002918d7b323e2e8364e23e9d3a4ab" target='_blank'>
              Human RNA Polymerase II Segregates from Genes and Nascent RNA and Transcribes in the Presence of DNA-Bound dCas9
              </a>
            </td>
          <td>
            João Pessoa, C. Carvalho
          </td>
          <td>2024-08-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/28684de767f629655a5d7133bf4405708af9dfcf" target='_blank'>
              Integrated analyses highlight interactions between the three-dimensional genome and DNA, RNA and epigenomic alterations in metastatic prostate cancer
              </a>
            </td>
          <td>
            Shuang G Zhao, M. Bootsma, Stanley Zhou, Raunak Shrestha, Thaidy Moreno-Rodriguez, A. Lundberg, Chu Pan, C. Arlidge, James R. Hawley, A. Foye, Alana S Weinstein, M. Sjöström, Meng Zhang, Haolong Li, L. Chesner, N. Rydzewski, K. Helzer, Yue Shi, Molly Lynch, S. Dehm, Joshua M Lang, J. Alumkal, Hansen H He, Alexander W Wyatt, Rahul Aggarwal, W. Zwart, Eric J Small, David A Quigley, Mathieu Lupien, Felix Y Feng
          </td>
          <td>2024-07-17</td>
          <td>Nature Genetics</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33af20a7d0aac05ac83adef2c51ac9ffcee3995c" target='_blank'>
              Enhancer engagement sustains oncogenic transformation and progression of B-cell precursor acute lymphoblastic leukemia
              </a>
            </td>
          <td>
            G. Corleone, C. Sorino, M. Caforio, S. Di Giovenale, F. de Nicola, F. Goeman, V. Bertaina, A. Pitisci, C. Cortile, Franco Locatelli, V. Folgiero, Maurizio Fanciulli
          </td>
          <td>2024-06-27</td>
          <td>Journal of Experimental & Clinical Cancer Research : CR</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Accurate detection of homologous recombination deficiency (HRD) in cancer patients is paramount in clinical applications, as HRD confers sensitivity to poly (ADP-ribose) polymerase (PARP) inhibitors. With the advances in genome sequencing technology, mutational profiling on a genome-wide scale has become readily accessible, and our knowledge of the genomic consequences of HRD has been greatly expanded and refined. Here, we review the recent advances in HRD detection methods. We examine the copy number and structural alterations that often accompany the genome instability that results from HRD, describe the advantages of mutational signature-based methods that do not rely on specific gene mutations, and review some of the existing algorithms used for HRD detection. We also discuss the choice of sequencing platforms (panel, exome, or whole-genome) and catalog the HRD detection assays used in key PARP inhibitor trials.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b04df45a7add0404411b8f4a6e0dd524a6a72b61" target='_blank'>
              Recent Advances in Genomic Approaches for the Detection of Homologous Recombination Deficiency.
              </a>
            </td>
          <td>
            Yoo-Na Kim, D. Gulhan, Hu Jin, Dominik Glodzik, Peter J. Park
          </td>
          <td>2024-07-17</td>
          <td>Cancer research and treatment</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Abstract Aneuploidy, or a change in the number of whole chromosomes or chromosome arms, is a near-universal feature of cancer. Chromosomes affected by aneuploidy are not random, with observed cancer-specific and tissue-specific patterns. Recent advances in genome engineering methods have allowed the creation of models with targeted aneuploidy events. These models can be used to uncover the downstream effects of individual aneuploidies on cancer phenotypes including proliferation, apoptosis, metabolism, and immune signaling. Here, we review the current state of research into the patterns of aneuploidy in cancer and their impact on signaling pathways and biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/53976b63387a7558e4fe16be8478ae589398c516" target='_blank'>
              Patterns of Aneuploidy and Signaling Consequences in Cancer
              </a>
            </td>
          <td>
            Nadja Zhakula-Kostadinova, Alison M Taylor
          </td>
          <td>2024-06-26</td>
          <td>Cancer Research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Recurrent mutations in serine / arginine-rich splicing factor 2 (SRSF2), particularly the proline-to-histidine substitution at position 95 (P95H), have been proposed to drive neoplastic diseases. SRSF2 plays pivotal roles in pre-mRNA processing and gene transcription. However, the precise impact of these diverse functions of SRSF2 on cancer cell behaviour remains unclear. Here, we show that deletion or homozygous P95H mutation of SRSF2 both cause extensive DNA damage leading to cell cycle arrest in vitro and in vivo, regardless of cell genotype. We mechanistically demonstrate that SRSF2 is required for efficient bi-directional transcription of DNA replication and repair genes, independent of its function in splicing. In contrast, SRSF2 haploinsufficiency induces DNA damage without halting the cell cycle in cancer cells. Inducing the heterozygous Srsf2 P95H mutation leads to clonal expansion of epidermal cells in mouse skin, but tumor formation is inhibited following exposure to carcinogens. To survive carcinogen treatment, cells containing the Srsf2 P95H mutation undergo substantial transcription rewiring that restores bi-directional gene expression. Thus, our study reveals crucial roles of SRSF2 beyond splicing and underscores its importance in regulating transcription to orchestrate the cell cycle along with the DNA damage response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32907ead7270689c743bba801c421a14be049a66" target='_blank'>
              SRSF2 transcriptional function maintains genome integrity during cell division
              </a>
            </td>
          <td>
            Rebecca E. Wagner, Leonie Arnetzl, Thiago Britto-Borges, Anke Heit-Mondrzyk, Ali Bakr, Etienne Sollier, Jasper Panten, Sylvain Delaunay, Daniela Sohn, P. Schmezer, D. T. Odom, Karin Müller-Decker, C. Plass, Christoph Dieterich, P. Lutsik, Susanne Bornelöv, Michaela Frye
          </td>
          <td>2024-07-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b6dae7fa4630abea723eb2e5ea8093d08d91876" target='_blank'>
              Cancer, metastasis, and the epigenome
              </a>
            </td>
          <td>
            Saurav Kiri, T. Ryba
          </td>
          <td>2024-08-02</td>
          <td>Molecular Cancer</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Chromosomal translocations in non-Hodgkin lymphoma (NHL) result in activation of oncogenes by placing them under the regulation of immunoglobulin heavy chain (IGH) super-enhancers. Aberrant expression of translocated oncogenes induced by enhancer activity can contribute to lymphomagenesis. The role of the IGH enhancers in normal B-cell development is well established, but knowledge regarding the precise mechanisms of their involvement in control of the translocated oncogenes is limited. The goal of this project was to define the critical regions in the IGH regulatory elements and identify enhancer RNAs (eRNA). We designed a sgRNA library densely covering the IGH enhancers and performed tiling CRISPR interference screens in three NHL cell lines. This revealed three regions crucial for NHL cell growth. With chromatin-enriched RNA-Seq we showed transcription from the core enhancer regions and subsequently validated expression of the eRNAs in a panel of NHL cell lines and tissue samples. Inhibition of the essential IGH enhancer regions decreased expression of eRNAs and translocated oncogenes in several NHL cell lines. The observed expression and growth patterns were consistent with the breakpoints in the IGH locus. Moreover, targeting the Eμ enhancer resulted in loss of B-cell receptor expression. In a Burkitt lymphoma cell line, MYC overexpression partially rescued the phenotype induced by IGH enhancer inhibition. Our results indicated the most critical regions in the IGH enhancers and provided new insights into the current understanding of the role of IGH enhancers in B-cell NHL. As such, this study forms a basis for development of potential therapeutic approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/801fa2579edbf74a0b66ac42b4a2406cc3439a8b" target='_blank'>
              Core regions in immunoglobulin heavy chain enhancers essential for survival of non-Hodgkin lymphoma cells are identified by a CRISPR interference screen.
              </a>
            </td>
          <td>
            Marta elzbieta Kasprzyk, Weronika Sura, M. Podralska, Marta Kazimierska, Annika Seitz, Wojciech Łosiewski, Tomasz Woźniak, Jeroen E J Guikema, A. Diepstra, Joost L. Kluiver, Anke van den Berg, N. Rozwadowska, A. Dzikiewicz-Krawczyk
          </td>
          <td>2024-06-27</td>
          <td>Haematologica</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Biallelic pathogenic variants in the essential DNA repair gene BRCA2 causes Fanconi anemia, complementation group FA-D1. Patients in this group are highly prone to develop embryonal tumors, most commonly medulloblastoma arising from the cerebellar granule cell progenitors (GCPs). GCPs undergo high proliferation in the postnatal cerebellum under SHH activation, but the type of DNA lesions that require the function of the BRCA2 to prevent tumorigenesis remains unknown. To identify such lesions, we assessed both GCP neurodevelopment and tumor formation using a mouse model with deletion of exons three and four of Brca2 in the central nervous system, coupled with global Trp53 loss. Brca2Δex3-4;Trp53-/- animals developed SHH subgroup medulloblastomas with complete penetrance. Whole-genome sequencing of the tumors identified structural variants with breakpoints enriched in areas overlapping G-quadruplexes (G4s). Brca2-deficient GCPs exhibited decreased replication speed in the presence of the G4-stabilizer pyridostatin. Pif1 helicase, which resolves G4s during replication, was highly upregulated in tumors, and Pif1 knockout in primary MB tumor cells resulted in increased genome instability upon pyridostatin treatment. These data suggest that G4s may represent sites prone to replication stalling in highly proliferative GCPs and without BRCA2, G4s become a source of genome instability. Tumor cells upregulate G4-resolving helicases to facilitate rapid proliferation through G4s highlighting PIF1 helicase as a potential therapeutic target for treatment of BRCA2-deficient medulloblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d543ffa68779aadc1c04b11411614442736e5f08" target='_blank'>
              G-quadruplexes are a source of vulnerability in BRCA2 deficient granule cell progenitors and medulloblastoma
              </a>
            </td>
          <td>
            Danielle Keahi, Mathijs A. Sanders, M. Paul, A. L. Webster, Yin Fang, Tom Wiley, Samer Shalaby, Thomas S. Carroll, S. Chandrasekharappa, Carolina Sandoval-Garcia, Margaret L. MacMillan, John E. Wagner, M. Hatten, A. Smogorzewska
          </td>
          <td>2024-07-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Intra-tumor heterogeneity is an important driver of tumor evolution and therapy response. Advances in precision cancer treatment will require understanding of mutation clonality and subclonal architecture. Currently the slow computational speed of subclonal reconstruction hinders large cohort studies. To overcome this bottleneck, we developed Clonal structure identification through Pairwise Penalization, or CliPP, which clusters subclonal mutations using a regularized likelihood model. CliPP reliably processed whole-genome and whole-exome sequencing data from over 12,000 tumor samples within 24 hours, thus enabling large-scale downstream association analyses between subclonal structures and clinical outcomes. Through a pan-cancer investigation of 7,827 tumors from 32 cancer types, we found that high subclonal mutational load (sML), a measure of latency time in tumor evolution, was significantly associated with better patient outcomes in 16 cancer types with low to moderate tumor mutation burden (TMB). In a cohort of prostate cancer patients participating in an immunotherapy clinical trial, high sML was indicative of favorable response to immune checkpoint blockade. This comprehensive study using CliPP underscores sML as a key feature of cancer. sML may be essential for linking mutation dynamics with immunotherapy response in the large population of non-high TMB cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9eca4fd8b4bb1bf5ae8673767195fa7c9bf0f23c" target='_blank'>
              Pan-cancer subclonal mutation analysis of 7,827 tumors predicts clinical outcome
              </a>
            </td>
          <td>
            Yujie Jiang, Matthew D. Montierth, Kaixian Yu, Shuangxi Ji, Shuai Guo, Quang Tran, Xiaoqian Liu, Seung Jun Shin, S. Cao, Ruonan Li, Yuxin Tang, T. Lesluyes, S. Kopetz, Jaffer J. Ajani, P. Msaouel, S. Subudhi, Ana Aparicio, Padmanee Sharma, J. P. Shen, Anil K. Sood, M. Tarabichi, Jennifer R. Wang, Marek Kimmel, P. Van Loo, Hongtu Zhu, Wenyi Wang
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>89</td>
        </tr>

        <tr id="We developed a generally applicable method, CRISPR/Cas9-targeted long-read sequencing (CTLR-Seq), to resolve, haplotype-specifically, the large and complex regions in the human genome that had been previously impenetrable to sequencing analysis, such as large segmental duplications (SegDups) and their associated genome rearrangements. CTLR-Seq combines in vitro Cas9-mediated cutting of the genome and pulse-field gel electrophoresis to isolate intact large (i.e., up to 2,000 kb) genomic regions that encompass previously unresolvable genomic sequences. These targets are then sequenced (amplification-free) at high on-target coverage using long-read sequencing, allowing for their complete sequence assembly. We applied CTLR-Seq to the SegDup-mediated rearrangements that constitute the boundaries of, and give rise to, the 22q11.2 Deletion Syndrome (22q11DS), the most common human microdeletion disorder. We then performed de novo assembly to resolve, at base-pair resolution, the full sequence rearrangements and exact chromosomal breakpoints of 22q11.2DS (including all common subtypes). Across multiple patients, we found a high degree of variability for both the rearranged SegDup sequences and the exact chromosomal breakpoint locations, which coincide with various transposons within the 22q11.2 SegDups, suggesting that 22q11DS can be driven by transposon-mediated genome recombination. Guided by CTLR-Seq results from two 22q11DS patients, we performed three-dimensional chromosomal folding analysis for the 22q11.2 SegDups from patient-derived neurons and astrocytes and found chromosome interactions anchored within the SegDups to be both cell type-specific and patient-specific. Lastly, we demonstrated that CTLR-Seq enables cell-type specific analysis of DNA methylation patterns within the deletion haplotype of 22q11DS.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ce8a6eb0facea9feaf4d43b99611c152795c17c" target='_blank'>
              Resolving the 22q11.2 deletion using CTLR-Seq reveals chromosomal rearrangement mechanisms and individual variance in breakpoints.
              </a>
            </td>
          <td>
            Bo Zhou, C. Purmann, Hanmin Guo, G. Shin, Yiling Huang, Reenal Pattni, Qingxi Meng, S. Greer, Tanmoy Roychowdhury, Raegan N Wood, Marcus Ho, H. Dohna, A. Abyzov, Joachim F Hallmayer, W. H. Wong, Hanlee P. Ji, A. E. Urban
          </td>
          <td>2024-07-23</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="There are two key signatures of pediatric cancers: (a) higher prevalence of germline alterations and (b) heterogeneity in alteration types. Recent population-based assessments have demonstrated that children with birth defects (BDs) are more likely to develop cancer even without chromosomal anomalies; therefore, explorations of genetic alterations in children with BDs and cancers could provide new insights into the underlying mechanisms for pediatric tumor development. We performed whole-genome sequencing (WGS) on blood-derived DNA for 1556 individuals without chromosomal anomalies, including 454 BD probands with at least one type of malignant tumor, 757 cancer-free children with BDs, and 345 healthy individuals, focusing on copy number variation (CNV) analysis. Roughly half of the children with BD-cancer have CNVs that are not identified in BD-only/healthy individuals, and CNVs are not evenly distributed among these patients. Strong heterogeneity was observed, with a limited number of cancer predisposition genes containing CNVs in more than three patients. Moreover, functional enrichments of genes with CNVs showed that dozens of patients have variations related to the same biological pathways, such as deletions of genes with neurological functions and duplications of immune response genes. Phenotype clustering uncovered recurrences of patients with sarcoma: A notable enrichment was observed involving non-coding RNA regulators, showing strong signals related to growth and cancer regulations in functional analysis. In conclusion, we conducted one of the first genomic studies exploring the impact of CNVs on cancer development in children with BDs, unveiling new insights into the underlying biological processes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0421f07c60f2733c310940e72c92010ab69f33fb" target='_blank'>
              Copy number variations contribute to malignant tumor development in children with serious birth defects.
              </a>
            </td>
          <td>
            Yichuan Liu, J. Glessner, Hui-Qi Qu, X. Chang, H. Qiu, Tiancheng Wang, F. Mentch, H. Hakonarson
          </td>
          <td>2024-08-14</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="An obligate step in the life cycle of HIV-1 and other retroviruses is the establishment of the provirus in target cell chromosomes. Transcriptional regulation of proviruses is complex, and understanding the mechanisms underlying this regulation has ramifications for fundamental biology, human health, and gene therapy implementation. The three core components of the Human Silencing Hub (HUSH) complex, TASOR, MPHOSPH8 (MPP8), and PPHLN1 (Periphilin 1), were identified in forward genetic screens for host genes that repress provirus expression. Subsequent loss-of-function screens revealed accessory proteins that collaborate with the HUSH complex to silence proviruses in particular contexts. To identify proteins associated with a HUSH complex-repressed provirus in human cells, we developed a technique, Provirus Proximal Proteomics, based on proximity labeling with C-BERST (dCas9-APEX2 biotinylation at genomic elements by restricted spatial tagging). Our screen exploited a lentiviral reporter that is silenced by the HUSH complex in a manner that is independent of the integration site in chromatin. Our data reveal that proviruses silenced by the HUSH complex are associated with DNA repair, mRNA processing, and transcriptional silencing proteins, including L3MBTL2, a member of the non-canonical polycomb repressive complex 1.6 (PRC1.6). A forward genetic screen confirmed that PRC1.6 components L3MBTL2 and MGA contribute to HUSH complex-mediated silencing. PRC1.6 was then shown to silence HUSH-sensitive proviruses in a promoter-specific manner. Genome wide profiling showed striking colocalization of the PRC1.6 and HUSH complexes on chromatin, primarily at sites of active promoters. Finally, PRC1.6 binding at a subset of genes that are silenced by the HUSH complex was dependent on the core HUSH complex component MPP8. These studies offer new tools with great potential for studying the transcriptional regulation of proviruses and reveal crosstalk between the HUSH complex and PRC1.6.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e6151d660aa506b4ad3a4c66d40fa26e99798e8" target='_blank'>
              PRC1.6 localizes on chromatin with the human silencing hub (HUSH) complex for promoter-specific silencing
              </a>
            </td>
          <td>
            Tom S Rodriguez, Leonid A Yurkovetskiy, Karthika Nagalekshmi, Chin Hung Oscar Lam, Eva Jazbec, Stacy A. Maitland, S. Wolfe, E. Sontheimer, Jeremy Luban
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Introduction. It has been established that the presence of homologous recombination deficiency in a breast tumor is associated with the effectiveness of treatment. But despite the high chemosensitivity of the tumor to DNA-damaging agents, complete pathological responses to treatment are very rare. And this process may be based on a change in the somatic status of BRCA1, that is, a reversion and return of the wild-type allele occurs and the DNA repair function is restored.Aim. To evaluate changes in the presence of chromosomal aberrations and the expression profile of the main genes of homologous recombination in cell models of breast cancer under the influence of cisplatin and docetaxel.Materials and methods. The study was conducted on breast cancer tumor cell cultures: MCF-7, MDA-MB-231 and MDA-MB-468. A cell model of drug resistance was obtained for two drugs: cisplatin and docetaxel. RNA and DNA were isolated from cell suspension using the RNeasy Plus Mini Kit and QIAamp DNA Mini Kit (Qiagen, Germany), respectively. The expression level of homologous recombination genes was assessed using reverse transcription polymerase chain reaction. To assess the presence of chromosomal aberrations, microarray analysis was performed on DNA chips.Results. Restoration of normal copy number for the BRCA1, CDK12, CHEK1 and RAD51D genes in MCF-7 under the influence of cisplatin was shown. For BRCA2 and PALB2, amplifications were detected. A statistically significant increase in the expression of the BRCA1 (p = 0.04), BRCA2 (p = 0.02), PALB2 (p = 0.01) and RAD51D (p = 0.05) genes was also shown. MDAMB-231 shows that all identified loci with deletions, where the BRCA2, BARD1, CHEK2, PALB2 and RAD54L genes are localized, are restored to normal copy number by cisplatin. The appearance of amplifications was registered for BRCA1, BRIP1, FANCL, RAD51B, PARP1. A similar result was shown for docetaxel. An increase in the expression level is typical for the genes BRCA1 (p = 0.02), BRCA2 (p = 0.02), CHEK2 (p = 0.05), FANCL (p = 0.04), PALB2 (p = 0.05), RAD51C (p = 0.02), PARP1 (p = 0.02), which corresponds to the appearance of amplifications. In the MDA-MB-468 cell culture, an increase in the copy number of only the BRCA1 gene is observed. The effect of docetaxel has no effect on this cell culture. The level of BRCA1 expression increases in direct proportion to the duration of drug action.Conclusion. Thus, the study showed that under the influence of cisplatin, reversion of not only homologous recombination gene mutations, but also other disorders can occur.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/767d30042e565f8e61dafca9671915ee6d03ecf7" target='_blank'>
              Modulation of homologous recombination gene activity in breast tumor cells in an in vitro model
              </a>
            </td>
          <td>
            M. Tsyganov, A. Frolova, E. A. Kravtsova, I. A. Tsydenova, M. Ibragimova
          </td>
          <td>2024-06-28</td>
          <td>Advances in Molecular Oncology</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Accurately quantifying the functional consequences of non-coding mosaic variants requires the pairing of DNA sequence with both accessible and closed chromatin architectures along individual DNA molecules—a pairing that cannot be achieved using traditional fragmentation-based chromatin assays. We demonstrate that targeted single-molecule chromatin fiber sequencing (Fiber-seq) achieves this, permitting single-molecule, long-read genomic and epigenomic profiling across targeted >100 kilobase loci with ∼10-fold enrichment over untargeted sequencing. Targeted Fiber-seq reveals that pathogenic expansions of the DMPK CTG repeat that underlie Myotonic Dystrophy 1 are characterized by somatic instability and disruption of multiple nearby regulatory elements, both of which are repeat length-dependent. Furthermore, we reveal that therapeutic adenine base editing of the segmentally duplicated γ-globin (HBG1/HBG2) promoters in primary human hematopoietic cells induced towards an erythroblast lineage increases the accessibility of the HBG1 promoter as well as neighboring regulatory elements. Overall, we find that these non-protein coding mosaic variants can have complex impacts on chromatin architectures, including extending beyond the regulatory element harboring the variant.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9447591d5b0bc6ff15f592902f9149ea8d6bbf6" target='_blank'>
              Resolving the chromatin impact of mosaic variants with targeted Fiber-seq
              </a>
            </td>
          <td>
            Stephanie C. Bohaczuk, Zachary J. Amador, Chang Li, Ben Mallory, Elliott G. Swanson, Jane Ranchalis, Mitchell R. Vollger, Katherine M. Munson, Tom Walsh, Morgan O. Hamm, Yizi Mao, Andre Lieber, A. Stergachis
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="The analysis of DNA sequence outcomes provides molecular insights into double-strand break (DSB) repair mechanisms. By employing parallel in-pool profiling of Cas9-induced indels within a genome-wide knockout library, we present a comprehensive catalog detailing how virtually every human gene influences the DSB repair process. This REPAIRome resource is validated through the identification of novel mechanisms, pathways and factors involved in DSB repair, including unexpected opposing roles for XLF and PAXX in DNA end processing, a molecular explanation for Cas9-induced multi-nucleotide insertions, the identification of HLTF as a DSB-repair factor, the involvement of the SAGA complex in microhomology-mediated end joining, and importantly, an indel mutational signature linked to VHL loss, renal carcinoma and hypoxia. Collectively, these results exemplify the potential of REPAIRome to drive future discoveries in DSB repair, CRISPR-Cas gene editing and the etiology of cancer mutational signatures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/821f1075e49379e32d3660c5c28ffcf24b062dbf" target='_blank'>
              A comprehensive genetic catalog of human double-strand break repair
              </a>
            </td>
          <td>
            Ernesto López de Alba, Israel Salguero, D. Giménez-Llorente, Ángel Fernández-Sanromán, Ester Casajús-Pelegay, José Terrón-Bautista, Jonathan Barroso-González, J. Bernal, Geoffrey Macintyre, R. Fernández-Leiro, Ana Losada, Felipe Cortés-Ledesma
          </td>
          <td>2024-08-03</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Introduction Mesenchymal stem cells (MSCs) have long been postulated as an important source cell in regenerative medicine. During subculture expansion, mesenchymal stem cell (MSC) senescence diminishes their multi-differentiation capabilities, leading to a loss of therapeutic potential. Up to date, the extrachromosomal circular DNAs (eccDNAs) have been demonstrated to be involved in senescence but the roles of eccDNAs during MSC. Methods Here we explored eccDNA profiles in human bone marrow MSCs (BM-MSCs). EccDNA and mRNA was purified and sequenced, followed by quantification and functional annotation. Moreover, we mapped our datasets with the downloading enhancer and transcription factor-regulated genes to explore the potential role of eccDNAs. Results Sequentially, gene annotation analysis revealed that the majority of eccDNA were mapped in the intron regions with limited BM-MSC enhancer overlaps. We discovered that these eccDNA motifs in senescent BMSCs acted as motifs for binding transcription factors (TFs) of senescence-related genes. Discussion These findings are highly significant for identifying biomarkers of senescence and therapeutic targets in mesenchymal stem cells (MSCs) for future clinical applications. The potential of eccDNA as a stable therapeutic target for senescence-related disorders warrants further investigation, particularly exploring chemically synthesized eccDNAs as transcription factor regulatory elements to reverse cellular senescence.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0b0fc0e697c338e4e8ee6ebc06de54d45231d9a" target='_blank'>
              Genome-wide sequencing identified extrachromosomal circular DNA as a transcription factor-binding motif of the senescence genes that govern replicative senescence in human mesenchymal stem cells
              </a>
            </td>
          <td>
            Wei Yang, Wei Ji, Boyu Liao, Zhongbo Li, Jian Wang, Haishu Lin, Jingbo Wang, Qian He
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Cellular Neuroscience</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Near-haploidization, that is, loss of one copy of most chromosomes, is a relatively rare phenomenon in most tumors, but is enriched among certain soft tissue sarcomas, including undifferentiated pleomorphic sarcoma (UPS). Presumably, near-haploidization can arise through many mechanisms. This study aimed to identify gene rearrangements that could cause near-haploidization. We here present two UPS in which near-haploidization was an early event, identified through single nucleotide polymorphism (SNP) array analysis. One of the cases was studied further using whole genome and transcriptome sequencing, as well as cytogenetic and molecular cytogenetic methods. Both tumors had chromosomal rearrangements in the form of copy number shifts/structural variants affecting the SMC1A gene. These findings suggest that cohesin defects could contribute to mitotic errors resulting in massive loss of chromosomes. SMC1A encodes one of the components of the cohesin multiprotein complex, which is critical for proper alignment of the sister chromatids during S-phase and separation to opposite spindle poles. Further studies should explore the role of cohesin defects in near-haploidization in other sarcomas and to clarify its role in tumor development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ac46389944e17adf0a38d1c92f41322a72ad7ff" target='_blank'>
              Structural Variants in the SMC1A Gene Associated With Near-Haploidy in Undifferentiated Pleomorphic Sarcomas.
              </a>
            </td>
          <td>
            Sebastian Ibstedt, Paul Piccinelli, Saskia Sydow, Jan Köster, Fredrik Mertens
          </td>
          <td>2024-08-01</td>
          <td>Genes, chromosomes & cancer</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="While genotoxic chemotherapeutic agents are among the most effective tools to combat cancer, they are often associated with severe adverse effects caused by indiscriminate DNA damage in non-tumor tissue as well as increased risk of secondary carcinogenesis. This study builds on our previous work demonstrating that the RNA Polymerase I (Pol I) transcription inhibitor CX-5461 elicits a non-canonical DNA damage response and our discovery of a critical role for Topoisomerase 2α (Top2α) in the initiation of Pol I-dependent transcription. Here, we identify Top2α as a mediator of CX-5461 response in the murine Eµ-Myc B lymphoma model whereby sensitivity to CX-5461 is dependent on cellular Top2α expression/activity. Most strikingly, and in contrast to canonical Top2α poisons, we found that the Top2α-dependent DNA damage induced by CX-5461 is preferentially localized at the ribosomal DNA (rDNA) promoter region, thereby highlighting CX-5461 as a loci-specific DNA damaging agent. This mechanism underpins the efficacy of CX-5461 against certain types of cancer and can be used to develop effective non-genotoxic anticancer drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/913f673912030e76f53faa13fb8d57cb178fa48f" target='_blank'>
              CX-5461 Preferentially Induces Top2α-Dependent DNA Breaks at Ribosomal DNA Loci
              </a>
            </td>
          <td>
            Donald P. Cameron, J. Sornkom, Sameerh Alsahafi, D. Drygin, G. Poortinga, Grant A. McArthur, Nadine Hein, Ross D Hannan, Konstantin I. Panov
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Abnormal transcription initiation from alternative first exon has been reported to promote tumorigenesis. However, the prevalence and impact of gene expression regulation mediated by alternative tandem transcription initiation were mostly unknown in cancer. Here, we developed a robust computational method to analyze alternative tandem transcription start site (TSS) usage from standard RNA sequencing data. Applying this method to pan-cancer RNA sequencing datasets, we observed widespread dysregulation of tandem TSS usage in tumors, many of which were independent of changes in overall expression level or alternative first exon usage. We showed that the dynamics of tandem TSS usage was associated with epigenomic modulation. We found that significant 5′ untranslated region shortening of gene TIMM13 contributed to increased protein production, and up-regulation of TIMM13 by CRISPR-mediated transcriptional activation promoted proliferation and migration of lung cancer cells. Our findings suggest that dysregulated tandem TSS usage represents an addtional layer of cancer-associated transcriptome alterations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e10e26b2a90c7cf14b59fe18386ef2bb8242323d" target='_blank'>
              Pan-cancer transcriptome analysis reveals widespread regulation through alternative tandem transcription initiation
              </a>
            </td>
          <td>
            Zhaozhao Zhao, Yu Chen, Xudong Zou, Limin Lin, Xiaolan Zhou, Xiaomeng Cheng, Guangrui Yang, Qiushi Xu, Lihai Gong, Lei Li, Ting Ni
          </td>
          <td>2024-07-10</td>
          <td>Science Advances</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract In ovarian and breast cancer, promoter methylation of BRCA1 or RAD51C is a promising biomarker for PARP inhibitor response, as high levels lead to homologous recombination deficiency (HRD). Yet the extent and role of such methylation in other cancers is not clear. This study comprehensively investigated promoter methylation of eight homologous recombination repair genes across 23 solid cancer types. Here, we showed that BRCA1 methylated cancers were associated with reduced gene expression, loss of heterozygosity (LOH), TP53 mutations and genomic features of HRD. We identified BRCA1 methylation in 3% of the copy-number high subtype of endometrial cancer, and as a rare event in six other cancer types, including lung squamous cell, pancreatic, bladder and stomach cancer. RAD51C promoter methylation was widespread across multiple cancer types, but HRD features were only observed for cases which contained high-level tumour methylation and LOH of RAD51C. While RAD51C methylation was frequent in stomach adenocarcinoma (6%) and low-grade glioma (2.5%), it was mostly detected at a low tumour level, suggestive of heterozygous methylation, and was associated with CpG island methylator phenotype. Our findings indicate that high-level tumour methylation of BRCA1 and RAD51C should be explored as a PARP inhibitor biomarker across multiple cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/374a7a0b66d69ba1f5dc2c4d8219381d1c2a8cfc" target='_blank'>
              High-level tumour methylation of BRCA1 and RAD51C is required for homologous recombination deficiency in solid cancers
              </a>
            </td>
          <td>
            Lijun Xu, Brett Liddell, K. Nešić, Franziska Geissler, Lauren M. Ashwood, Matthew J. Wakefield, Clare L Scott, Nicola Waddell, O. Kondrashova
          </td>
          <td>2024-07-09</td>
          <td>NAR Cancer</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Male sex chromosome aneuploidies are frequent genetic aberrations in humans characterized by additional Y or X chromosome complements. Jacob (JS) and Klinefelter syndromes (KS), carrying 47,XYY and 47,XXY chromosomes, respectively, share several clinical features, including sterility, hormonal deficits, neurocognitive delay, and skeletal-muscle defects. Despite the high incidence, a complete understanding of the molecular mechanisms underlying the clinical manifestations in sex aneuploid patients is still elusive. In this study, we generated and characterized the inaugural cohort of 47,XYY human induced pluripotent stem cells (iPSCs). We performed a comprehensive transcriptional analysis, including JS primary fibroblasts, iPSCs, and neural stem cells (NSCs) alongside KS cells. Our study revealed a transcriptional feedback mechanism tuning non-PAR X chromosome genes (NPX) homologs in Y supernumerary cells, a phenomenon not detected in X aneuploid iPSCs. By ectopically modulating the expression of selected NPY genes, we demonstrated a direct transcriptional link between the UTY-UTX gene pair. Furthermore, our analyses identified a shared transcriptomic signature between JS and KS, discernible already at the iPSC stage, with a notable enrichment for processes related to neurological functions. This transcriptomic convergence underscores potential commonalities in the molecular pathways underpinning the pathophysiology of male sex chromosome aneuploidies. Finally, through genome-wide DNA methylation profiling of JS iPSCs, we demonstrated that a supernumerary Y chromosome only minimally impacts the methylation status of 47,XYY cells at the pluripotent stage. Our work reveals critical transcriptional feedback mechanisms and shared gene expression signatures in male sex chromosome aneuploidies, paving the way for a better understanding of their common phenotypic features.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a4a1e46b6ba1f6258afb3db5dff0a58811a485a6" target='_blank'>
              An iPSC-based model of Jacob Syndrome reveals a DNA methylation-independent transcriptional dysregulation shared with X aneuploid cells
              </a>
            </td>
          <td>
            V. Astro, K. Cardona-Londoño, L. V. Cortés-Medina, R. Alghamdi, G. Ramírez-Calderón, F. Kefalas, J. Dilmé-Capó, S. Radío, A. Adamo
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d9b28bde62e7c6ec79bde107b1d678f6c8aab03e" target='_blank'>
              MethNet: a robust approach to identify regulatory hubs and their distal targets from cancer data
              </a>
            </td>
          <td>
            Theodore Sakellaropoulos, Catherine Do, Guimei Jiang, Giulia Cova, Peter Meyn, D. Dimartino, Sitharam Ramaswami, Adriana Heguy, A. Tsirigos, Jane A Skok
          </td>
          <td>2024-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Although the study of leukemogenesis has traditionally focused on protein-coding genes, the role of enhancer dysregulation is becoming increasingly recognized. The advent of high-throughput sequencing, together with a better understanding of enhancer biology, has revealed how various genetic and epigenetic lesions produce oncogenic enhancers that drive transformation. These aberrations include translocations that lead to enhancer hijacking, point mutations that modulate enhancer activity, and copy number alterations that modify enhancer dosage. In this review, we describe these mechanisms in the context of leukemia and discuss potential therapeutic avenues to target these regulatory elements. Significance: Large-scale sequencing projects have uncovered recurrent gene mutations in leukemia, but the picture remains incomplete: some patients harbor no such aberrations, whereas others carry only a few that are insufficient to bring about transformation on their own. One of the missing pieces is enhancer dysfunction, which only recently has emerged as a critical driver of leukemogenesis. Knowledge of the various mechanisms of enhancer dysregulation is thus key for a complete understanding of leukemia and its causes, as well as the development of targeted therapies in the era of precision medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4963169247626a5146ad859dcbd326045f924327" target='_blank'>
              Oncogenic Enhancers in Leukemia.
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Ruud Delwel
          </td>
          <td>2024-08-02</td>
          <td>Blood cancer discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c429a219f0b035a7a915d9f1d9d1a5ce65fcffe" target='_blank'>
              DKK1-SE recruits AP1 to activate the target gene DKK1 thereby promoting pancreatic cancer progression
              </a>
            </td>
          <td>
            Lan Shao, Haoran Yu, Mengyun Wang, Lu Chen, Boshu Ji, Tong Wu, Xiangqi Teng, Mu Su, Xiao Han, Weikai Shi, Xin Hu, Ziwen Wang, Hongjuan He, Guiping Han, Yan Zhang, Qiong Wu
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/724cc2cf5494857c9a50f7bbbea10f5d098c777c" target='_blank'>
              The genomic landscape of 2,023 colorectal cancers.
              </a>
            </td>
          <td>
            A. Cornish, Andreas J. Gruber, B. Kinnersley, D. Chubb, A. Frangou, G. Caravagna, Boris Noyvert, E. Lakatos, H.M. Wood, Steve Thorn, R. Culliford, Claudia Arnedo-Pac, J. Househam, William Cross, Amit Sud, P. Law, Máire Ní Leathlobhair, A. Hawari, C. Woolley, Kitty Sherwood, Nathalie Feeley, Güler Gül, Juan Fernandez-Tajes, L. Zapata, Ludmil B. Alexandrov, N. Murugaesu, A. Sosinsky, Jonathan Mitchell, Núria López-Bigas, P. Quirke, David N Church, Ian P. M. Tomlinson, A. Sottoriva, Trevor A Graham, David C Wedge, R. Houlston
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="The relevant role of LINE-1 (L1) retrotransposition in cancer has been recurrently demonstrated in recent years. However, their repetitive nature hampers their identification and detection, hence remaining inaccessible for clinical practice. Also, its clinical relevance for cancer patients is still limited. Here, we develop a new method to quantify L1 activation, called RetroTest, based on targeted sequencing and a sophisticated bioinformatic pipeline, allowing its application in tumor biopsies. First, we performed the benchmarking of the method and confirmed its high specificity and reliability. Then, we unravel the L1 activation in HNSCC according to a more extensive cohort including all the HNSCC tumor stages. Our results confirm that RetroTest is remarkably efficient for L1 detection in tumor biopsies, reaching a high sensitivity and specificity. In addition, L1 retrotransposition estimation reveals a surprisingly early activation in HNSCC progression, contrary to its classical association with advanced tumor stages. This early activation together with the genomic mutational profiling of normal adjacent tissues supports field cancerization process in this tumor. These results underline the importance of estimating L1 retrotransposition in clinical practice towards an earlier and more efficient diagnosis in HNSCC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08ae21a6395bf2fa53698eb0f53f5dc6c7132bfe" target='_blank'>
              RetroTest unravels LINE-1 retrotransposition in Head and Neck Squamous Cell Carcinoma
              </a>
            </td>
          <td>
            Jenifer Brea-Iglesias, Ana Oitaben, Sonia Zumalave, Bernardo Rodriguez-4 Martin, María Gallardo-Gómez, Martín Santamarina, Ana Pequeño-Valtierra, L. Juaneda-Magdalena, Ramón García-Escudero, J. López-Cedrún, Máximo Fraga, J. Tubío, Mónica Martínez-Fernández, Monica Martínez
          </td>
          <td>2024-07-24</td>
          <td>None</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Genomic, epigenomic, and transcriptomic alterations are hallmarks of cancer cells, and are closely connected. Especially, epigenetic regulation plays a critical role in tumorigenesis and progression. The growing single-cell epigenome data in cancer research provide new opportunities for data mining from a more comprehensive perspective. However, there is still a lack of databases designed for interactively exploring the single-cell multi-omics data of human pan-cancer, especially for the single-cell epigenome data. To fill in the gap, we developed scCancerExplorer, a comprehensive and user-friendly database to facilitate the exploration of the single-cell genome, epigenome (chromatin accessibility and DNA methylation), and transcriptome data of 50 cancer types. Five major modules were provided to explore those data interactively, including "integrated multi-omics analysis", "single-cell transcriptome", "single-cell epigenome", "single-cell genome", and "TCGA analysis". By simple clicking, users can easily investigate gene expression features, chromatin accessibility patterns, transcription factor activities, DNA methylation states, copy number variations, and TCGA survival analysis results. Taken together, scCancerExplorer is distinguished from previous databases with rich and interactive functions for exploring the single-cell multi-omics data of human pan-cancer. It bridges the gap between single-cell multi-omics data and the end-users, and will facilitate progress in the field of cancer research. scCancerExplorer is freely accessible via https://bianlab.cn/scCancerExplorer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4f0bfe81970da09d476152acc53e069683a992ff" target='_blank'>
              scCancerExplorer: a comprehensive database for interactively exploring single-cell multi-omics data of human pan-cancer
              </a>
            </td>
          <td>
            Changzhi Huang, Zekai Liu, Yunlei Guo, Wanchu Wang, Zhen Yuan, Yusheng Guan, Deng Pan, Zhibin Hu, Linhua Sun, Zan Fu, Shuhui Bian
          </td>
          <td>2024-08-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Purpose: Complex chromosomal rearrangements (CCRs) are rare structural variants involving three or more chromosomal break- points. Most de novo reported CCRs pose challenges for diagnosis and management. They often require karyotyping, fluorescence in situ hybridization (FISH), and chromosomal microarray analysis (CMA) for clinical diagnosis because of the limitations of each method. Here we report an inherited exceptionally complex CCR involving 4 chromosomes and 11 breakpoints in a family with multisystem anomalies. Methods: We evaluated the CCRs using karyotyping, FISH, CMA, and two emerging genomic technologies: high-throughput chromosome conformation capture sequencing (Hi-C; aka genomic proximity mapping, GPM) and optical genome mapping (OGM). We also performed functional studies using transcriptome and methylome analyses. Results: The proband, who had intellectual disability and immune deficiency, shared CCRs with her unaffected mother involving chromosomes 1, 7, and 11 by karyotyping. However, CMA revealed a duplication and three deletions in the proband in contrast to her mother's balanced genome. Hi-C (GPM) and OGM detected the CCRs and copy number alterations but also uncovered additional breakpoints at high resolution, including an insertion in 4p and two cryptic rearrangements at 7p. Transcriptome and methylome analyses identified likely biological pathways associated with the proband's phenotypes. Conclusion: Combining cytogenetic and genomic methods provided comprehensive characterization and defined the breakpoints at high resolution in both proband and mother. This underscores the value of novel cytogenetic and genomic techniques in deciphering complex genome rearrangements and the significance of integrative genomic analysis and functional characterization in understanding clinical phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7d8dc703c96a52057ab6829a72726f61eb74bd56" target='_blank'>
              Genetic and functional characterization of inherited complex chromosomal rearrangements in a family with multisystem anomalies
              </a>
            </td>
          <td>
            H. Fang, S. M. Eacker, Y. Wu, W. Neufeld-Kaiser, M. Laurino, S. Keel, M. S. Horwitz, Y. Liu
          </td>
          <td>2024-07-21</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Animal chromosomes can persist with recognizable homology over hundreds of millions of years, in spite of homology-obfuscating processes such as chromosomal fusion and translocation. The frequency and pace of these major genome structural changes are unknown, and it remains unclear whether or how they impact long-term genome evolution. Here, we compare whole chromosomal sequences of 3,631 genomes from 2,291 species spanning all major animal clades and show that animal karyotypes evolve primarily via karyotype contraction, associated with increased rates of chromosomal fusion-with-mixing and dispersion that largely obey chromosomal algebra1, or karyotype expansion, which breaks up ancestral linkage groups and forms new chromosomal elements via non-algebraic changes. We show that chromosomal changes can be associated with major extinction events. Using a multi-scale encoding of pan-animal genome homology and a manifold representation of genomic changes, we find that genome evolution is not only driven by changes at the chromosomal level, but that subchromosomal mixing and irreversibility define clade-specific evolution. Using this ‘evolutionary genome topology’ approach, we calculate extrema of irreversible genomic configurations and identify species that occupy intermediate manifold positions, providing evidence for distinct macro-evolutionary trajectories. We propose that investigation of mixed state accumulation around important gene loci (such as Hox) will be crucial in capturing and further study of clade-specific regulatory innovations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1bcfa759f5e30b92ec5cb430b3076d461c210e1e" target='_blank'>
              Topological mixing and irreversibility in animal chromosome evolution
              </a>
            </td>
          <td>
            D. Schultz, Arno Blümel, Dalila Destanović, Fatih Sarigol, Oleg Simakov
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="The myeloma overexpressed gene (MYEOV) has been proposed to be a proto-oncogene due to high RNA transcript levels found in multiple cancers, including myeloma, breast, lung, pancreas and esophageal cancer. The presence of an open reading frame (ORF) in humans and other primates suggests protein-coding potential. Yet, we still lack evidence of a functional MYEOV protein. It remains undetermined how MYEOV overexpression affects cancerous tissues. In this work, we show that MYEOV has likely originated and may still function as an enhancer, regulating CCND1 and LTO1. Firstly, MYEOV 3′ enhancer activity was confirmed in humans using publicly available ATAC-STARR-seq data, performed on B-cell-derived GM12878 cells. We detected enhancer histone marks H3K4me1 and H3K27ac overlapping MYEOV in multiple healthy human tissues, which include B cells, liver and lung tissue. The analysis of 3D genome datasets revealed chromatin interactions between a MYEOV-3′-putative enhancer and the proto-oncogene CCND1. BLAST searches and multi-sequence alignment results showed that DNA sequence from this human enhancer element is conserved from the amphibians/amniotes divergence, with a 273 bp conserved region also found in all mammals, and even in chickens, where it is consistently located near the corresponding CCND1 orthologues. Furthermore, we observed conservation of an active enhancer state in the MYEOV orthologues of four non-human primates, dogs, rats, and mice. When studying this homologous region in mice, where the ORF of MYEOV is absent, we not only observed an enhancer chromatin state but also found interactions between the mouse enhancer homolog and Ccnd1 using 3D-genome interaction data. This is similar to the interaction observed in humans and, interestingly, coincides with CTCF binding sites in both species. Taken together, this suggests that MYEOV is a primate-specific gene with a de novo ORF that originated at an evolutionarily older enhancer region. This deeply conserved putative enhancer element could regulate CCND1 in both humans and mice, opening the possibility of studying MYEOV regulatory functions in cancer using non-primate animal models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f17ce82b4897c471d6bebe571f550d964a45b5c" target='_blank'>
              Evolutionarily conserved enhancer-associated features within the MYEOV locus suggest a regulatory role for this non-coding DNA region in cancer
              </a>
            </td>
          <td>
            Brigid S. A. Davidson, Juliana Estefanía Arcila-Galvis, Marco Trevisan-Herraz, Aneta Mikulasova, C. Brackley, Lisa J Russell, Daniel Rico
          </td>
          <td>2024-07-30</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Melanoma showcases a complex interplay of genetic alterations and intra- and inter-cellular morphological changes during metastatic transformation. While pivotal, the role of specific mutations in dictating these changes still needs to be fully elucidated. Telomerase promoter mutations (TERTp mutations) significantly influence melanoma’s progression, invasiveness, and resistance to various emerging treatments, including chemical inhibitors, telomerase inhibitors, targeted therapy, and immunotherapies. We aim to understand the morphological and phenotypic implications of the two dominant monoallelic TERTp mutations, C228T and C250T, enriched in melanoma metastasis. We developed isogenic clonal cell lines containing the TERTp mutations and utilized dual-color expression reporters steered by the endogenous Telomerase promoter, giving us allelic resolution. This approach allowed us to monitor morpholomic variations induced by these mutations. TERTp mutation-bearing cells exhibited significant morpholome differences from their wild-type counterparts, with increased allele expression patterns, augmented wound-healing rates, and unique spatiotemporal dynamics. Notably, the C250T mutation exerted more pronounced changes in the morpholome than C228T, suggesting a differential role in metastatic potential. Our findings underscore the distinct influence of TERTp mutations on melanoma’s cellular architecture and behavior. The C250T mutation may offer a unique morpholomic and systems-driven advantage for metastasis. These insights provide a foundational understanding of how a non-coding mutation in melanoma metastasis affects the system, manifesting in cellular morpholome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66b66856ef0863e2a38d3da5fdaad88a280ff968" target='_blank'>
              Using deep learning to decipher the impact of telomerase promoter mutations on the dynamic metastatic morpholome
              </a>
            </td>
          <td>
            Andres J. Nevarez, Anusorn Mudla, Sabrina A. Diaz, Nan Hao
          </td>
          <td>2024-07-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Alternative DNA conformation formed by sequences called flipons potentially alter the readout of genetic information by directing the shape-specific assembly of complexes on DNA The biological roles of G-quadruplexes formed by motifs rich in guanosine repeats have been investigated experimentally using many different methodologies including G4-seq, G4 ChIP-seq, permanganate nuclease footprinting (KEx), KAS-seq, CUT&Tag with varying degrees of overlap between the results. Here we trained large language model DNABERT on existing data generated by KEx, a rapid chemical footprinting technique performed on live, intact cells using potassium permanganate. The snapshot of flipon state when combined with results from other in vitro methods that are performed on permeabilized cells, allows a high confidence mapping of G-flipons to proximal enhancer and promoter sequences. Using G4-DNABERT predictions,with data from ENdb, Zoonomia cCREs and single cell G4 CUT&Tag experiments, we found support for a model where G4-quadruplexes regulate gene expression through chromatin loop formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ea25e2e4cfb927141977cb915b650e3384446d86" target='_blank'>
              Analysis of live cell data with G-DNABERT supports a role for G-quadruplexes in chromatin looping
              </a>
            </td>
          <td>
            Dmitry Konovalov, Dmitry Umerenkov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-06-25</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="While genome-wide association studies and expression quantitative trait loci (eQTL) analysis have made significant progress in identifying noncoding variants associated with prostate cancer risk and bulk tissue transcriptome changes, the regulatory effect of these genetic elements on gene expression remains largely unknown. Recent developments in single-cell sequencing have made it possible to perform ATAC-seq and RNA-seq profiling simultaneously to capture functional associations between chromatin accessibility and gene expression. In this study, we tested our hypothesis that this multiome single-cell approach allows for mapping regulatory elements and their target genes at prostate cancer risk loci. We applied a 10X Multiome ATAC + Gene Expression platform to encapsulate Tn5 transposase-tagged nuclei from multiple prostate cell lines for a total of 65,501 high quality single cells from RWPE1, RWPE2, PrEC, BPH1, DU145, PC3, 22Rv1 and LNCaP cell lines. To address data sparsity commonly seen in the single-cell sequencing, we performed targeted sequencing to enrich sequencing data at prostate cancer risk loci involving 2,730 candidate germline variants and 273 associated genes. Although not increasing the number of captured cells, the targeted multiome data did improve eQTL gene expression abundance by about 20% and chromatin accessibility abundance by about 5%. Based on this multiomic profiling, we further associated RNA expression alterations with chromatin accessibility of germline variants at single cell levels. Cross validation analysis showed high overlaps between the multiome associations and the bulk eQTL findings from GTEx prostate cohort. We found that about 20% of GTEx eQTLs were covered within the significant multiome associations (p-value ≤ 0.05, gene abundance percentage ≥ 5%), and roughly 10% of the multiome associations could be identified by significant GTEx eQTLs. We also analyzed accessible regions with available heterozygous SNP reads and observed more frequent association in genomic regions with allelically accessible variants (p = 0.0055). Among these findings were previously reported regulatory variants including rs60464856-RUVBL1 (multiome p-value = 0.0099 in BPH1) and rs7247241-SPINT2 (multiome p-value = 0.0002- 0.0004 in 22Rv1). We also functionally validated a new regulatory SNP and its target gene rs2474694-VPS53 (multiome p-value = 0.00956 in BPH1 and 0.00625 in DU145) by reporter assay and SILAC proteomics sequencing. Taken together, our data demonstrated the feasibility of the multiome single-cell approach for identifying regulatory SNPs and their regulated genes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/936580ce17da8d58425b498ecb0e8504faaa215a" target='_blank'>
              Identify Regulatory eQTLs by Multiome Sequencing in Prostate Single Cells
              </a>
            </td>
          <td>
            Yijun Tian, Lang Wu, Chang-Ching Huang, Liang Wang
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background and Aim: Canine transmissible venereal tumor (CTVT), a unique transmissible cancer in dogs, affects the external genitalia and potentially spreads to other parts of the body. While somatic mutations in oncogenic and tumor-suppressing genes are linked to CTVT development, the impact of DNA methylation, which affects gene expression, remains unclear. This study explored whether DNA methylation in the promoter regions of the MYC oncogene and CDKN2B tumor suppressor genes in CTVTs is associated with their expression, both at the gene and protein levels. 

Materials and Methods: To investigate promoter DNA methylation of MYC and CDKN2B in CTVTs, we analyzed frozen tissue samples from genital CTVT (GTVTs) and extragenital CTVT (ETVTs). Genomic DNA was extracted, bisulfite-treated, and analyzed using bisulfite polymerase chain reaction (PCR) and sequencing. The messenger RNA and protein of MYC and CDKN2B were also extracted and assessed by real-time PCR and Western blotting. Matching formalin-fixed, paraffin-embedded blocks were used for immunohistochemical staining to visualize protein distribution in GTVT and ETVT tissues. 

Results: Although both GTVT and ETVT samples showed MYC promoter methylation, the extent of methylation differed significantly. GTVTs displayed a much higher degree of methylation, potentially explaining the more pronounced downregulation of MYC gene expression and reduction in c-MYC protein levels observed in GTVTs compared with ETVTs. Our data revealed a prevalent hypermethylation pattern in the CDKN2B promoter across both sample types. However, DNA methylation, which was expected to have a suppressive effect, did not correlate with gene/protein expression. GTVTs displayed high protein levels despite significantly reduced CDKN2B expression. Conversely, ETVTs maintained regular CDKN2B expression but exhibited reduced protein production, suggesting a complex interplay between methylation and expression in these tumors. 

Conclusion: MYC demonstrated a clear association between its promoter methylation status, gene expression, and protein levels; however, CDKN2B lacked this correlation, implying the involvement of methylation-independent regulatory mechanisms and highlighting the need for further investigation. 

Keywords: canine transmissible venereal tumor, CDKN2B, DNA methylation, MYC, oncogene, tumor suppressor gene.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/26c5019910fb12afd2000e4151f192b0d24fad38" target='_blank'>
              Divergent DNA methylation patterns and gene expression in MYC and CDKN2B in canine transmissible venereal tumors
              </a>
            </td>
          <td>
            Soukkangna Keopaseuth, K. Pringproa, P. Patchanee, C. Setthawongsin, S. Techangamsuwan, Phongsakorn Chuammitri
          </td>
          <td>2024-07-01</td>
          <td>Veterinary World</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="The expression of eukaryotic genes relies on the precise 3’-terminal cleavage and polyadenylation of newly synthesized pre-mRNA transcripts. Defects in these processes have been associated with various diseases, including cancer. While cancer-focused sequencing studies have identified numerous driver mutations in protein-coding sequences, noncoding drivers – particularly those affecting the cis-elements required for pre-mRNA cleavage and polyadenylation – have received less attention. Here, we systematically analysed cancer somatic mutations affecting 3’UTR polyadenylation signals using the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset. We found a striking enrichment of cancer-specific somatic mutations that disrupt strong and evolutionarily conserved cleavage and polyadenylation signals within tumour suppressor genes. Further bioinformatics and experimental analyses conducted as a part of our study suggest that these mutations have a profound capacity to downregulate the expression of tumour suppressor genes. Thus, this work uncovers a novel class of noncoding somatic mutations with significant potential to drive cancer progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4b3938c7f84b24956b4223cbd287669eff1caf" target='_blank'>
              Recurrent disruption of tumour suppressor genes in cancer by somatic mutations in cleavage and polyadenylation signals
              </a>
            </td>
          <td>
            Yaroslav A. Kainov, F. Hamid, E. Makeyev
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="Mutually exclusive loss-of-function alterations in gene pairs are those that occur together less frequently than may be expected and may denote a synthetically lethal relationship (SSL) between the genes. SSLs can be exploited therapeutically to selectively kill cancer cells. Here, we analysed mutation, copy number variation, and methylation levels in samples from The Cancer Genome Atlas, using the hypergeometric and the Poisson binomial tests to identify mutually exclusive inactivated genes. We focused on gene pairs where one is an inactivated tumour suppressor and the other a gene whose protein product can be inhibited by known drugs. This provided an abundance of potential targeted therapeutics and repositioning opportunities for several cancers. These data are available on the MexDrugs website, https://bioinformaticslab.sussex.ac.uk/mexdrugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7250b34d6949747e0dee284668cf7df5c1a39253" target='_blank'>
              Identifying actionable synthetically lethal cancer gene pairs using mutual exclusivity.
              </a>
            </td>
          <td>
            Sarah K. Wooller, Laurence H. Pearl, Frances M. G. Pearl
          </td>
          <td>2024-07-08</td>
          <td>FEBS letters</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d417bff4628adb0291d6a29056ca242d46664396" target='_blank'>
              Molecular characteristics and oncogenic role of CHD family genes: a pan-cancer analysis based on bioinformatic and biological analysis
              </a>
            </td>
          <td>
            Yujia Lu, Jiebang Jiang, Zhihong He, Zhouzhou Bao, Xin Chen, Jie Cheng
          </td>
          <td>2024-08-15</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The centromeric protein-A (CENP-A) is an evolutionary conserved histone H3 variant that marks the identity of the centromeres. Several mechanisms regulate the centromeric deposition of CENP-A as its mislocalization causes erroneous chromosome segregation, leading to aneuploidy-based diseases, including cancers. The most crucial deposition factor is a CENP-A specific chaperone, HJURP (Scm3 in budding yeast), which specifically binds to CENP-A. However, the discovery of HJURP as a DDR (DNA damage repair) protein and evidence of its binding to Holliday junctions in vitro indicate a CENP-A-deposition-independent role of these chaperones. In this study, using budding yeast, we demonstrate that Scm3 is crucial for the DDR pathway as scm3 cells are sensitive to DNA damage. We further observe that the scm3 mutant interacts with the rad52 DDR mutant and is compromised in activating DDR-mediated arrest. We demonstrate that Scm3 associates with the DNA damage sites and undergoes posttranslational modifications upon DNA damage. Overall, from this report and earlier studies on HJURP, we conclude that DDR functions of CENP-A chaperones are conserved across eukaryotes. Thus, the revelation that these chaperones confer genome stability in more than one pathway has clinical significance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/20b8f002bffe33b7b0eebb870a6636324b784a0d" target='_blank'>
              Evidence of centromeric histone 3 chaperone involved in DNA damage repair pathway
              </a>
            </td>
          <td>
            Prakhar Agarwal, S. Ghosh
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Three-stranded DNA-RNA structures known as R-loops that form during papillomavirus transcription can cause transcription-replication conflicts and lead to DNA damage. We found that R-loops accumulated at the viral early promoter in human papillomavirus (HPV) episomal cells but were greatly reduced in cells with integrated HPV genomes. RNA-DNA helicases unwind R-loops and allow for transcription and replication to proceed. Depletion of the RNA-DNA helicase senataxin (SETX) using siRNAs increased the presence of R-loops at the viral early promoter in HPV-31 (CIN612) and HPV-16 (W12) episomal HPV cell lines. Depletion of SETX reduced viral transcripts in episomal HPV cell lines. The viral E2 protein, which binds with high affinity to specific palindromes near the promoter and origin, complexes with SETX, and both SETX and E2 are present at the viral p97 promoter in CIN612 and W12 cells. SETX overexpression increased E2 transcription activity on the p97 promoter. SETX depletion also significantly increased integration of viral genomes in CIN612 cells. Our results demonstrate that SETX resolves viral R-loops to proceed with HPV transcription and prevent genome integration.IMPORTANCEPapillomaviruses contain small circular genomes of approximately 8 kilobase pairs and undergo unidirectional transcription from the sense strand of the viral genome. Co-transcriptional R-loops were recently reported to be present at high levels in cells that maintain episomal HPV and were also detected at the early viral promoter. R-loops can inhibit transcription and DNA replication. The process that removes R-loops from the PV genome and the requisite enzymes are unknown. We propose a model in which the host RNA-DNA helicase senataxin assembles on the HPV genome to resolve R-loops in order to maintain the episomal status of the viral genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb3257970da042eeca9c08a9a174f0a9b3a46f14" target='_blank'>
              Senataxin mediates R-loop resolution on HPV episomes.
              </a>
            </td>
          <td>
            L. Jose, Keely Smith, Anaiya Crowner, EJ Androphy, M. DeSmet
          </td>
          <td>2024-07-24</td>
          <td>Journal of virology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4bde92262496f0de13dfb84458af04ef9c55e997" target='_blank'>
              NHEJ and HDR can occur simultaneously during gene integration into the genome of Aspergillus niger
              </a>
            </td>
          <td>
            Susanne Fritsche, Aline Reinfurt, Felix Fronek, Matthias G Steiger
          </td>
          <td>2024-08-05</td>
          <td>Fungal Biology and Biotechnology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/21a24ea5db444de6c4cb2a6e44207610c8118487" target='_blank'>
              RHNO1: at the crossroads of DNA replication stress, DNA repair, and cancer.
              </a>
            </td>
          <td>
            Niphat Jirapongwattana, Samuel F. Bunting, Donald R Ronning, Gargi Ghosal, A. Karpf
          </td>
          <td>2024-08-06</td>
          <td>Oncogene</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="DNA methylation is a key epigenetic regulatory mechanism that plays a critical role in a variety of cellular processes, including the regulation of cell fate during development, maintenance of cell identity, and genome stability. DNA methylation is tightly regulated by enzymatic reactions and its deregulation plays an important role in the development of cancer. Specific DNA methylation alterations have been found in pediatric solid tumors, providing new insights into the development of these tumors. In addition, DNA methylation profiles have greatly contributed to tune the diagnosis of pediatric solid tumors and to define subgroups of patients with different risks of progression, leading to the reduction in unwanted toxicity and the improvement of treatment efficacy. This review highlights the dysregulated DNA methylome in pediatric solid tumors and how this information provides promising targets for epigenetic therapies, particularly inhibitors of DNMT enzymes (DNMTis). Opportunities and limitations are considered, including the ability of DNMTis to induce viral mimicry and immune signaling by tumors. Besides intrinsic action against cancer cells, DNMTis have the potential to sensitize immune-cold tumors to immunotherapies and may represent a remarkable option to improve the treatment of challenging pediatric solid tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8b1eac857dfcc5a5e3b672ddb17d6097e48bc70f" target='_blank'>
              Targeting DNA Methylation Machinery in Pediatric Solid Tumors
              </a>
            </td>
          <td>
            C. Cristalli, Katia Scotlandi
          </td>
          <td>2024-07-01</td>
          <td>Cells</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Transcribed enhancer maps can reveal nuclear interactions underpinning each cell type and connect specific cell types to diseases. Using a 5′ single-cell RNA sequencing approach, we defined transcription start sites of enhancer RNAs and other classes of coding and noncoding RNAs in human CD4+ T cells, revealing cellular heterogeneity and differentiation trajectories. Integration of these datasets with single-cell chromatin profiles showed that active enhancers with bidirectional RNA transcription are highly cell type–specific and that disease heritability is strongly enriched in these enhancers. The resulting cell type–resolved multimodal atlas of bidirectionally transcribed enhancers, which we linked with promoters using fine-scale chromatin contact maps, enabled us to systematically interpret genetic variants associated with a range of immune-mediated diseases. Editor’s summary Currently, 3′ single-cell RNA sequencing is widely used as a tool for single-cell gene expression analysis. By leveraging a distinctive “cap signature” derived from the 5′-end cap, Oguchi et al. introduced a technique to investigate the precise position of the RNA 5′-ends in single cells, enabling simultaneous profiling of gene expression and enhancer activity. The authors constructed a CD4+ T cell–type resolved multimodal atlas to integrate and elucidate CD4+ T cell diversity and genetic risk of immune-mediated diseases. They also identified rare, uncharacterized CD4+ T cell types that might contribute to the pathology of immune-mediated diseases. —Di Jiang INTRODUCTION Enhancers are distal cis-regulatory elements that increase the expression of target genes. Active enhancers produce bidirectional enhancer RNAs (eRNAs). Transcription of eRNAs can be used to measure enhancer activity and to delineate enhancer architecture at nucleotide resolution. Profiling of transcribed enhancers in single human cells can provide insights into cell type–specific gene regulation and genetic predisposition to diseases. However, comprehensive maps of eRNAs in cell types relevant to diseases are still lacking. RATIONALE The 5′ cap is added to the first nucleotide of an RNA molecule. By leveraging a specific “cap signature,” we developed read-level prefiltering and transcribed enhancer call (ReapTEC), a method for simultaneously profiling gene expression and enhancer activity by using 5′-end single-cell RNA sequencing (5′ scRNA-seq). We applied ReapTEC and single-nucleus assay for transposase-accessible chromatin using sequencing (snATAC-seq) to a total of ~1 million single human CD4+ T cells and constructed a CD4+ T cell type–resolved multimodal atlas to pinpoint genetic and cellular causes of immune-mediated diseases. RESULTS By using ReapTEC, we accurately mapped the transcription start sites of mRNAs, long noncoding RNAs, and eRNAs in single cells, which allowed us to study the expression of not only genes but also bidirectionally transcribed candidate enhancers (btcEnhs). We characterized rare and uncharacterized cell types, including GPR25-expressing regulatory T cells, along with the expression profiles of ~130 surface proteins. We identified 62,803 btcEnhs and 218,508 accessible chromatin regions active across diverse CD4+ T cells. We found higher cell type specificity for RNA transcription than for chromatin accessibility both at promoters and enhancers. We also showed that the heritability of immune-mediated diseases was enriched more in btcEnhs than in open chromatin regions. Using our CD4+ T cell type–resolved atlas of btcEnhs, we systematically interpreted genetic variants associated with a range of immune-mediated diseases. We found that btcEnhs harboring disease SNPs were often active in specific subsets of CD4+ T cells; generally, they were conserved across mammals but in some cases were primate specific. We investigated the target genes of disease-relevant btcEnhs by generating fine-scale chromatin contact maps derived from ultradeep Micro-C data; we further performed CRISPR activation of selected disease-relevant btcEnhs to validate their target genes. CONCLUSION Our study establishes a platform for integrating and elucidating CD4+ T cell diversity and genetic risk of immune-mediated diseases. Although 3′ scRNA-seq is widely used as a major tool for single-cell gene expression analysis, our study demonstrates the strength of 5′ scRNA-seq. We anticipate the utility of our multimodal atlas in addressing variant-to-function challenges. CD4+ T cell type–resolved atlas of btcEnhs for systematic interpretation of the genetic basis of immune-mediated diseases. (A) ReapTEC allows us to detect transcription start sites (TSSs) and active enhancers using 5′ scRNA-seq. (B) Disease-relevant btcEnhs containing genetic variants associated with immune-mediated diseases were linked to target genes by using fine-scale chromatin contact maps. CITE-seq, cellular indexing of transcriptomes and epitopes by sequencing; Ref, reference allele; Alt, alternative allele.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/13a4b6cb952b21d2e76916c18021dfddb89e4c76" target='_blank'>
              An atlas of transcribed enhancers across helper T cell diversity for decoding human diseases
              </a>
            </td>
          <td>
            Akiko Oguchi, A. Suzuki, Shuichiro Komatsu, Hiroyuki Yoshitomi, S. Bhagat, Raku Son, R. Bonnal, Shohei Kojima, Masaru Koido, Kazuhiro Takeuchi, K. Myouzen, Gyo Inoue, Tomoya Hirai, Hiromi Sano, Y. Takegami, A. Kanemaru, Itaru Yamaguchi, Y. Ishikawa, Nao Tanaka, Shigeki Hirabayashi, Riyo Konishi, Sho Sekito, Takahiro Inoue, Juha Kere, Shunichi Takeda, A. Takaori-Kondo, Itaru Endo, S. Kawaoka, Hideya Kawaji, K. Ishigaki, H. Ueno, Y. Hayashizaki, Massimiliano Pagani, P. Carninci, M. Yanagita, Nicholas Parrish, Chikashi Terao, Kazuhiko Yamamoto, Yasuhiro Murakawa
          </td>
          <td>2024-07-05</td>
          <td>Science</td>
          <td>0</td>
          <td>118</td>
        </tr>

        <tr id="Pancreatic cancer is a malignancy arising from the endocrine or exocrine compartment of this organ. Tumors from exocrine origin comprise over 90% of all pancreatic cancers diagnosed. Of these, pancreatic ductal adenocarcinoma (PDAC) is the most common histological subtype. The five-year survival rate for PDAC ranged between 5 and 9% for over four decades, and only recently saw a modest increase to ∼12-13%, making this a severe and lethal disease. Like other cancers, PDAC initiation stems from genetic changes. However, therapeutic targeting of PDAC genetic drivers has remained relatively unsuccessful, thus the focus in recent years has expanded to the non-genetic factors underlying the disease pathogenesis. Specifically, it has been proposed that dynamic changes in the epigenetic landscape promote tumor growth and metastasis. Emphasis has been given to the re-organization of enhancers, essential regulatory elements controlling oncogenic gene expression, commonly marked my histone 3 lysine 4 monomethylation (H3K4me1). H3K4me1 is typically deposited by histone lysine methyltransferases (KMTs). While well characterized as oncogenes in other cancer types, recent work has expanded the role of KMTs as tumor suppressor in pancreatic cancer. Here, we review the role and translational significance for PDAC development and therapeutics of KMTs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0a7417d79919ae9f2eab24051f1e99fe379fd6b" target='_blank'>
              Insights into the mechanisms driven by H3K4 KMTs in pancreatic cancer.
              </a>
            </td>
          <td>
            Kayla C LaRue-Nolan, G. L. Raja Arul, Ashley N Sigafoos, Jiaqi Shi, Martin E Fernandez-Zapico
          </td>
          <td>2024-07-30</td>
          <td>The Biochemical journal</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Premature aging is a hallmark of Down syndrome, caused by trisomy of human chromosome 21, but the reason is unclear and difficult to study in humans. We used an aneuploid model in wild yeast to show that chromosome amplification disrupts nutrient-induced cell-cycle arrest, quiescence entry, and healthy aging, across genetic backgrounds and amplified chromosomes. We discovered that these defects are due in part to aneuploidy-induced dysfunction in Ribosome Quality Control (RQC). Compared to euploids, aneuploids entering quiescence display aberrant ribosome profiles, accumulate RQC intermediates, and harbor an increased load of protein aggregates. Although they have normal proteasome capacity, aneuploids show signs of ubiquitin dysregulation, which impacts cyclin abundance to disrupt arrest. Remarkably, inducing ribosome stalling in euploids produces similar aberrations, while up-regulating limiting RQC subunits or proteins in ubiquitin metabolism alleviates many of the aneuploid defects. Our results provide implications for other aneuploidy disorders including Down syndrome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c6d2e4dd1921e05fad92cf19e5a9d35d322983a" target='_blank'>
              Premature aging in aneuploid yeast is caused in part by aneuploidy-induced defects in Ribosome Quality Control
              </a>
            </td>
          <td>
            Leah E. Escalante, James Hose, Hollis Howe, Norah Paulsen, Michael Place, A. Gasch
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>43</td>
        </tr>

        <tr id="The bacterium Deinococcus radiodurans is known to efficiently and accurately reassemble its genome after hundreds of DNA double-strand breaks (DSBs). Only at very large amounts of radiation-induced DSBs is this accuracy affected in the wild-type D. radiodurans, causing rearrangements in its genome structure. However, changes in its genome structure may also be possible during the propagation and storage of cell cultures. We investigate this possibility by listing structural differences between three completely sequenced genomes of D. radiodurans strains with a recent common ancestor—the type strain stored and sequenced in two different laboratories (of the ATCC 13939 lineage) and the first sequenced strain historically used as the reference (ATCC BAA-816). We detected a number of structural differences and found the most likely mechanisms behind them: (i) transposition/copy number change in mobile interspersed repeats—insertion sequences and small non-coding repeats, (ii) variable number of monomers within tandem repeats, (iii) deletions between long direct DNA repeats, and (iv) deletions between short (4–10 bp) direct DNA repeats. The most surprising finding was the deletions between short repeats because it indicates the utilization of a less accurate DSB repair mechanism in conditions in which a more accurate one should be both available and preferred. The detected structural differences, as well as SNPs and short indels, while being important footprints of deinococcal DNA metabolism and repair, are also a valuable resource for researchers using these D. radiodurans strains.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55030650317fbed25fb2ab5464d15ce87378bc16" target='_blank'>
              Structural Differences between the Genomes of Deinococcus radiodurans Strains from Different Laboratories
              </a>
            </td>
          <td>
            K. Zahradka, D. Zahradka, J. Repar
          </td>
          <td>2024-06-27</td>
          <td>Genes</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d667ed72276b4514cb71b1b131d78ceeae693d2" target='_blank'>
              miR-876-3p is a tumor suppressor on 9p21 that is inactivated in melanoma and targets ERK
              </a>
            </td>
          <td>
            V. Bezrookove, Imran Khan, Anukana Bhattacharjee, Juifang Fan, Robyn Jones, Anima Sharma, M. Nosrati, P. Desprez, Nathan Salomonis, Yihui Shi, A. Dar, M. Kashani-Sabet
          </td>
          <td>2024-08-13</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c964458f9642633fba6003bd596ed63245c6ffb5" target='_blank'>
              PARP4 interacts with hnRNPM to regulate splicing during lung cancer progression
              </a>
            </td>
          <td>
            Y. Lee, C. Phua, Ju Yuan, Bin Zhang, May Yin Lee, S. Kannan, Yui Hei Jasper Chiu, Casslynn Wei Qian Koh, Choon Kong Yap, E. Lim, Jianbin Chen, Yuhua Lim, Jane Jia Hui Lee, A. Skanderup, Zhenxun Wang, Weiwei Zhai, Nguan Soon Tan, Chandra S Verma, Yvonne Tay, D. Tan, Wai Leong Tam
          </td>
          <td>2024-07-22</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Endothelial cells (ECs) maintain vessel tone and barrier integrity, regulate blood homeostasis, and prevent the extravasation of leukocytes under normal physiological conditions. Because of the limited lifespans and batch-to-batch differences with respect to the genetic make-up of primary ECs, established immortal EC lines are extensively used for studying endothelial biology. To address this issue, the immortal endothelial cell line EA.hy926 was developed by fusing primary human umbilical vein endothelial cells (HUVECs) with human lung carcinoma A549 cells. EA.hy926 cells share a number of similar endothelial properties with HUVECs and are considered the immortal counterpart to primary HUVECs. However, the cytogenetic integrity of EA.hy926 cells is not fully elucidated. We characterized EA.hy926 cells with conventional G-banding and molecular cytogenetic techniques such as spectral karyotyping and subtelomeric fluorescence in situ hybridization. Cytogenetic analysis revealed an array of numerical and stable structural chromosomal rearrangements including one deletion, one duplication, one isochromosome, seven simple translocations, and five complex translocations in Ea.hy926 cells. These findings will advance comprehension of EA.hy926 cell biology and augment future endothelial studies, specifically in comparison studies between HUVECs and EA.hy926 cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56ad1b12e69b1e161748ea2c28f86f896e19f2f6" target='_blank'>
              G-Banding and Molecular Cytogenetics Detect Novel Translocations and Cryptic Aberrations in Human Immortal Endothelial Cells
              </a>
            </td>
          <td>
            Regina Lichti Binz, R. Pathak
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Disruption of the epigenome is a hallmark of human disease including liver cirrhosis and hepatocellular carcinoma (HCC). While genetic heterogeneity is an established effector of pathologic phenotypes, epigenetic heterogeneity is less well understood. Environmental exposures alter the liver-specific DNA methylation landscape and influence the onset of liver cancer. Given that currently available treatments are unable to target frequently mutated genes in HCC, there is an unmet need for novel therapeutics to prevent or reverse liver damage leading to hepatic tumorigenesis, which the epigenome may provide. We performed genome-wide profiling of DNA methylation, copy number, and gene expression from multiple liver regions from 31 patients with liver disease to examine their crosstalk and define the individual and combinatorial contribution of these processes to liver disease progression. We identify epigenetic heterogeneity hotspots that are conserved across patients. Elevated epigenetic heterogeneity associates with increased gene expression heterogeneity. Cirrhotic regions comprise two distinct cohorts, one exclusively epigenetic, and a second where epigenetic and copy number variations collaborate. Epigenetic heterogeneity hotspots are enriched for genes central to liver function (e.g., HNF1A) and known tumor suppressors (e.g., RASSF1A). These hotspots encompass genes including ACSL1, ACSL5, MAT1A, and ELFN1, which have phenotypic effects in functional screens, supporting their relevance to hepatocarcinogenesis. Moreover, epigenetic heterogeneity hotspots are linked to clinical measures of outcome.


CONCLUSION
Substantial epigenetic heterogeneity arises early in liver disease development, targeting key pathways in the progression and initiation of both cirrhosis and HCC. Integration of epigenetic and transcriptional heterogeneity unveils putative epigenetic regulators of hepatocarcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/46b475bb15d42990ae95e5c459fd1ea63a3f93cd" target='_blank'>
              Epigenetic heterogeneity hotspots in human liver disease progression.
              </a>
            </td>
          <td>
            R. Hlady, Xia Zhao, Louis El Khoury, Ryan T. Wagner, Aesis M. Luna, Kien Pham, Nikolaos Pyrosopoulos, Dhanpat Jain, Liguo Wang, Chen Liu, Keith Robertson
          </td>
          <td>2024-07-19</td>
          <td>Hepatology</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Survival of Medulloblastoma (MB) depends on various factors, including the gene expression profiles of MB tumor tissues. In this study, we identified 967 MB survival-related genes (SRGs) using a gene expression dataset and the Cox proportional hazards regression model. Notably, the SRGs were over-represented on chromosomes 6 and 17, known for the abnormalities monosomy 6 and isochromosome 17 in MB. The most significant SRG was HMGA1 (high mobility group AT-hook 1) on chromosome 6, which is a known oncogene and a histone H1 competitor. High expression of HMGA1 was associated with worse survival, primarily in the Group 3γ subtype. The high expression of HMGA1 was unrelated to any known somatic copy number alteration. Most SRGs on chromosome 17p were associated with low expression in Group 4β, the MB subtype, with 93% deletion of 17p and 98% copy gain of 17q. GO enrichment analysis showed that both chromosomes 6 and 17 included SRGs related to telomere maintenance and provided a rationale for testing telomerase inhibitors in Group 3 MBs. We conclude that HMGA1, along with other SRGs on chromosomes 6 and 17, warrant further investigation as potential therapeutic targets in selected subgroups or subtypes of MB.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8bb63da6d24c38cbfdc6fe840507e9aa6581f878" target='_blank'>
              Survival-Related Genes on Chromosomes 6 and 17 in Medulloblastoma
              </a>
            </td>
          <td>
            Jerry Vriend, Xiao-Qing Liu
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bba674d0d4447d398ec9ef85794c34ef0bbb9043" target='_blank'>
              Prognostic genome and transcriptome signatures in colorectal cancers.
              </a>
            </td>
          <td>
            Luís Nunes, Fuqiang Li, Meizhen Wu, Tian Luo, K. Hammarström, Emma Torell, I. Ljuslinder, Artur Mezheyeuski, Per-Henrik D. Edqvist, Anna Löfgren-Burström, C. Zingmark, S. Edin, Chatarina Larsson, L. Mathot, E. Osterman, E. Osterlund, V. Ljungström, I. Neves, Nicole Yacoub, Unnur Guðnadóttir, H. Birgisson, M. Enblad, Fredrik Pontén, R. Palmqvist, Xun Xu, Mathias Uhlén, Kui Wu, B. Glimelius, Cong Lin, T. Sjöblom
          </td>
          <td>2024-08-07</td>
          <td>Nature</td>
          <td>0</td>
          <td>62</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e2a2685872da8abf8d89c45e2e00bd648c1daaeb" target='_blank'>
              DNA methylation and gene expression as determinants of genome-wide cell-free DNA fragmentation
              </a>
            </td>
          <td>
            Michaël Noë, Dimitrios Mathios, A. Annapragada, Shashikant Koul, Zachariah H. Foda, J. Medina, S. Cristiano, Christopher Cherry, D. C. Bruhm, N. Niknafs, V. Adleff, Leonardo Ferreira, Hari Easwaran, Stephen Baylin, J. Phallen, Robert B Scharpf, V. Velculescu
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>115</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fa98c0061b5c115d1ee9f59c1e39ed2f207c47e1" target='_blank'>
              Intertwining roles of R-loops and G-quadruplexes in DNA repair, transcription and genome organization.
              </a>
            </td>
          <td>
            P. Wulfridge, K. Sarma
          </td>
          <td>2024-06-24</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdb874383a628cf292c0913e4a06655cd292ef13" target='_blank'>
              Frequency-dependent selection of neoantigens fosters tumor immune escape and predicts immunotherapy response
              </a>
            </td>
          <td>
            Shaoqing Chen, Duo Xie, Zan Li, Jiguang Wang, Zheng Hu, Da Zhou
          </td>
          <td>2024-06-25</td>
          <td>Communications Biology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/729c3507158456cf0b6b747d89cf47634b551b4c" target='_blank'>
              BUB1B monoallelic germline variants contribute to prostate cancer predisposition by triggering chromosomal instability
              </a>
            </td>
          <td>
            Maria P. Silva, Luísa T Ferreira, Natércia F. Brás, L. Torres, Andreia Brandão, M. Pinheiro, Marta Cardoso, Adriana Resende, Joana Vieira, Carlos Palmeira, Gabriela Martins, Miguel Silva, C. Pinto, Ana Peixoto, João Silva, Rui Henrique, S. Maia, H. Maiato, Manuel R. Teixeira, P. Paulo
          </td>
          <td>2024-07-16</td>
          <td>Journal of Biomedical Science</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Gene expression regulation is one of the most fundamental cellular processes, enabling the activation of a gene to produce either the translatable protein-coding transcript (mRNA) or a functional non-coding RNA with gene regulatory functions, ultimately determining cell identity and function. Although gene expression regulation can occur at transcriptional, translational, and post-translational levels, transcription initiation is the first and the most important step in gene expression, facilitating the transfer of biological information from DNA to protein. Enhancers and super-enhancers are among the master regulators of tissue- and cell-specific transcription regulation involved in cell differentiation and tumor formation. Despite four decades passing since the first discovery of enhancers in eukaryotes and extensive efforts undertaken to identify enhancers on a genomic scale during the last decade, the discovery of enhancers still faces certain limitations and needs further investigation. The perturbation of enhancer function due to genetic or epigenetic changes is closely linked to a range of human disorders, including the development and progression of cancers. Thus, the detection of early cancer-related enhancer activity and the subsequent normalization of expression abnormalities using enhancer-targeting CRISPR epigenetic editing, as well as enhancer-targeting pharmaceuticals, are regarded as groundbreaking therapeutic tactics in preclinical stages.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/782ddfe76cab2a16620844e3e84b112d8c90071b" target='_blank'>
              Enhancers and super-enhancers as master regulators in cancer
              </a>
            </td>
          <td>
            P. Sarvari, Pourya Sarvari, Ivonne Ramírez-Díaz, Karla Rubio
          </td>
          <td>2024-07-24</td>
          <td>INNOSC Theranostics and Pharmacological Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/96ca99a18dd08dc5d59b7f900d05406ec6c8cd83" target='_blank'>
              Amplifications of EVX2 and HOXD9-HOXD13 on 2q31 in mature cystic teratomas of the ovary identified by array comparative genomic hybridization may explain teratoma characteristics in chondrogenesis and osteogenesis
              </a>
            </td>
          <td>
            Wen-Chung Wang, Tai-Cheng Hou, C. Kuo, Yen-Chein Lai
          </td>
          <td>2024-06-21</td>
          <td>Journal of Ovarian Research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="OBJECTIVE
Ionizing radiation generates genomic instability by promoting the accumulation of chromosomal rearrangements. The oncogenic translocation RET/PTC1 is present in more than 70% of radiation-induced thyroid cancers. Both RET and CCDC6, the genes implicated in RET/PTC1, are found within common fragile sites - chromosomal regions prone to DNA breakage during slight replication stress. Given that irradiated cells become more susceptible to genomic destabilization due to the accumulation of replication-stress-related double-strand breaks (DSBs), we explored whether RET and CCDC6 exhibit DNA breakage under replicative stress several days post-irradiation of thyroid cells.


METHODS
We analyzed the dynamic of DNA replication in human thyroid epithelial cells (HThy-ori-3.1) 4 days post a 5-Gy exposure using molecular DNA combing. The DNA replication schedule was evaluated through replication-timing experiments. We implemented a ChIP-qPCR assay to determine whether the RET and CCDC6 genes break following irradiation.


RESULTS
Our study indicates that replicative stress, occurring several days post-irradiation in thyroid cells, primarily causes DSBs in the RET gene. We discovered that both the RET and CCDC6 genes undergo late replication in thyroid cells. However, only RET's replication rate is notably delayed after irradiation.


CONCLUSION
The findings suggest that post-irradiation in the RET gene causes a breakage in the replication fork, which could potentially invade another genomic area, including CCDC6. As a result, this could greatly contribute to the high prevalence of chromosomal RET/PTC rearrangements seen in patients exposed to external radiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eea49490c4342b6b738c402ce79c72ea23e5f552" target='_blank'>
              A post-irradiation-induced replication stress promotes RET proto-oncogene breakage.
              </a>
            </td>
          <td>
            Fabio Hecht, Laura Valerio, Carlos Frederico Lima Gonçalves, Marylin Harinquet, Rabii Ameziane El Hassani, Denise P. Carvalho, Stephane Koundrioukoff, Jean-Charles Cadoret, Corinne Dupuy
          </td>
          <td>2024-07-01</td>
          <td>European thyroid journal</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Interphase mammalian genomes are folded in 3D with complex locus-specific patterns that impact gene regulation. CTCF (CCCTC-binding factor) is a key architectural protein that binds specific DNA sites, halts cohesin-mediated loop extrusion, and enables long-range chromatin interactions. There are hundreds of thousands of annotated CTCF-binding sites in mammalian genomes; disruptions of some result in distinct phenotypes, while others have no visible effect. Despite their importance, the determinants of which CTCF sites are necessary for genome folding and gene regulation remain unclear. Here, we update and utilize Akita, a convolutional neural network model, to extract the sequence preferences and grammar of CTCF contributing to genome folding. Our analyses of individual CTCF sites reveal four predictions: (i) only a small fraction of genomic sites are impactful, (ii) insulation strength is highly dependent on sequences flanking the core CTCF binding motif, (iii) core and flanking sequences are broadly compatible, and (iv) core and flanking nucleotides contribute largely additively to overall strength. Our analysis of collections of CTCF sites make two predictions for multi-motif grammar: (i) insulation strength depends on the number of CTCF sites within a cluster, and (ii) pattern formation is governed by the orientation and spacing of these sites, rather than any inherent specialization of the CTCF motifs themselves. In sum, we present a framework for using neural network models to probe the sequences instructing genome folding and provide a number of predictions to guide future experimental inquiries. Author Summary Mammalian genomes are spatially organized in 3D with profound consequences for all processes involving DNA. CTCF is a key genome organizer, recognizing numerous sites and creating a variety of contact patterns across the genome. Despite the importance of CTCF, the sequence determinants and grammar of how individual sites collectively instruct genome folding remain unclear. This work leverages the ability of Akita, a deep neural network, to make high-throughput predictions for genome folding after DNA sequence perturbations. Using Akita, we make several experimentally testable predictions. First, only a minority of annotated sites individually impact folding, and flanking DNA sequences greatly modulate their impact. Second, multiple sites together influence folding based on their number, orientation, and spacing. In sum, we provide a roadmap for interpreting neural networks to better understand genome folding and important considerations for the design of experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05a8a450d11a02a4e12bb1863b2d5f19340a89cc" target='_blank'>
              Interpreting the CTCF-mediated sequence grammar of genome folding with AkitaV2
              </a>
            </td>
          <td>
            Paulina N. Smaruj, Fahad Kamulegeya, David R. Kelley, G. Fudenberg
          </td>
          <td>2024-08-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="The mammalian non-homologous end joining (NHEJ) is required for V(D)J recombination as well as coping with exogenously induced DNA double strand breaks (DSBs). Initiated by the binding of KU70/KU80 (KU) dimer to DNA ends and the subsequent recruitment of the DNA- dependent protein kinase catalytic subunit (DNA-PKcs), NHEJ plays a key role in DNA repair. While there has been significant structural understandings of how KU70 participates in NHEJ, the specific function of its highly conserved C-terminal SAP domain remains elusive. In this study, we developed a novel mouse model by deleting the SAP domain but preserving the KU70 nuclear localization and its dimerization ability with KU80. We found that the KU70 SAP deletion did not affect the V(D)J recombination or animal development but significantly impaired the animals and cells in repairing exogenously induced DSBs. We further showed an inability of KU70-ΔSAP cells to retain the DNA Ligase IV (LIG4) and other NHEJ co-factors on chromatin, and a spreading pattern of DSB marker γH2AX in KU70-ΔSAP cells after DNA damage. Our findings suggest that a specific inhibition of the SAP function may offer an opportunity to modulate cell sensitivity to therapeutic DSB-inducing agents without interfering with the developmental function of KU70. KeyPoints Generation of a novel transgenic mouse line lacking the C-terminal conserved KU70-SAP domain KU70-SAP defends against exogenous DSBs, but unessential for development and V(D)J recombination KU70-SAP aids in recruiting and retaining NHEJ components, such as LIG4, to DSB sites">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e5b4f65f1fff981360d615dcaa36c9fb6c898d37" target='_blank'>
              The Mammalian KU70 C-terminus SAP Domain Is Required to Repair Exogenous DNA Damage
              </a>
            </td>
          <td>
            Yuan Wang, Michael S. Czap, Hailey Kim, Huimei Lu, Jingmei Liu, Yokechen Chang, Peter J. Romanienko, Cristina Montagna, Zhiyuan Shen
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Purpose: Homologous recombination deficiency (HRD) impacts cancer treatment strategies, particularly the effectiveness of PARP inhibitors. However, the variability different HRD assays has hampered the selection of oncology patients who may benefit from these therapies. Our study aims to assess the whole genome landscape to better define HRD in a pan-cancer cohort and to contribute to harmonization of HRD detection. Methods: We employed a whole-genome sequencing WGS HRD classifier that included genome-wide features associated with HRD to analyze 580 tumor/normal paired pan-cancer samples. The HRD results were correlated retrospectively with treatment responses and were compared with commercial HRD tests in a subset of cases. Results: HRD phenotype was identified in 62 samples across various cancers including breast (19%), pancreaticobiliary (17%), gynecological (15%), prostate (8%), upper gastrointestinal (GI) (2%), and other cancers (1%). HRD cases were not confined to BRCA1/2 mutations; 24% of HRD cases were BRCA1/2 wild-type. A diverse range of HRR pathway gene alterations involved in HRD were elucidated, including biallelic mutations in FANCF, XRCC2, and FANCC, and deleterious structural variants. Comparison with results from commercial HRD assays suggests a better performance of WGS to detect HRD, based on treatment response. Conclusion: HRD is a biomarker used to determine which cancer patients would benefit from PARPi and platinum-based chemotherapy. However, a lack of harmonization of tests to determine HRD status makes it challenging to interpret their results. Our study highlights the use of comprehensive WGS analysis to predict HRD in a pan-cancer cohort, elucidates new genomic mechanisms associated with HRD, and enables an accurate identification of this phenotype, paving the way for improved outcomes in oncology care.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b80d347a4838be16d195accf2ef681565a4b6929" target='_blank'>
              Whole Genome Landscape Analysis of Homologous Recombination Deficiency in a Pan-Cancer Cohort
              </a>
            </td>
          <td>
            MD Majd Al Assaad, PhD Kevin Hadi, MS Max F. Levine, MD Daniela Guevara, Mph Minal Patel, MS Marvel Tranquille, BS Abigail King, BS John Otilano, MS Alissa Semaan, PhD Gunes Gundem, MS Juan S. Medina-Martínez, BS Michael Sigouros, Mph Jyothi Manohar, BS Hui-Hsuan Kuo, PhD David C. Wilkes, MD Eleni Andreopoulou, MD Eloise Chapman-Davis, MD Scott T. Tagawa, PhD Andrea Sboner, MD Allyson J. Ocean, MD Manish Shah, PhD Elli Papaemmanuil, Cora N. Sternberg, MD Kevin Holcomb, MD David M. Nanus, PhD Olivier Elemento, Juan Miguel Mosquera, J. Medina-Martinez
          </td>
          <td>2024-06-30</td>
          <td>None</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="During the sexual cycle, programmed genome rearrangement (PGR) in Paramecium tetraurelia involves the non-homologous end joining (NHEJ) DNA repair pathway to eliminate specific germinal Internal Eliminated Sequences (IESs) from the newly developing somatic nucleus. In addition to the core NHEJ factors Ku70/80 and Xrcc4/Lig4, additional enzymes are required to process the 4-base 5’-protruding ends generated following DNA cleavage at IES boundaries, prior to their ligation. Here, we report that PolX (a,b,c,d), four P. tetraurelia distant orthologs of the human Polλ DNA polymerase, are involved in repair of IES excision junctions. During rearrangements, PolX-depleted cells accumulate genome-wide errors, such as unrepaired double-strand breaks, 1-nucleotide deletions and IES retention. Although all PolX paralogs can process DNA ends, two of them (PolXa&b) are induced during PGR and have acquired tight nuclear anchoring properties through their N-terminal region, which contains a predicted BRCT domain. Finally, we show that PolXa accumulates in nuclear foci together with other NHEJ proteins and the Dicer-like enzyme Dcl5, which is involved in the biogenesis of IES-specific small RNAs. We propose that these “DNA repair foci” correspond to the sites where IES concatemers, a by-product of IES excision, are ligated together to produce the precursors of iesRNAs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8a893b216f167d132f42d3e505df3cd9a940a5" target='_blank'>
              A development-specific POLX essential for programmed genome rearrangement in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Baptiste Verron, O. Arnaiz, Coralie Zangarelli, Nathalie Mathy, M. Bétermier, J. Bischerour
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Repairing programmed DNA double-strand breaks (DSBs) is crucial in the lifecycle of Paramecium tetraurelia, especially during its sexual reproduction phase when its somatic highly polyploid macronucleus is lost. The formation of a new macronucleus involves Programmed Genome Rearrangements, introducing DNA DSBs at approximately 45,000 loci. P. tetraurelia employs a Non-Homologous End Joining (NHEJ)-related mechanism for the systematic repair of these DSBs. Four genes encoding DNA polymerases of family X are present in the genome, one of which was found recently to colocalize with other proteins of NHEJ. The question arises as to how they make almost no error. Here we show that these enzymes are most similar to metazoan DNA polymerase λ and exhibit high fidelity through two different molecular mechanisms. Using X-ray structure determination of polymerase lambda mutants recapitulating sequence determinants of P. tetraurelia PolXs, we find both a local conformational change that involves exchanging partners in a crucial salt bridge in the active site upon binding of correct dNTPs, and a larger conformational change involving the closure of Loop3. This stabilizes the template DNA in the active site, only in the presence of the correct incoming dNTP. Differences with human pol λ and pol β are discussed. GRAPHICAL ABSTRACT">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08a2c8e3978ae53a388430b008c681d13e31a282" target='_blank'>
              Mechanisms Ensuring Fidelity of Family X DNA Polymerases in Programmed DNA rearrangements in Paramecium tetraurelia
              </a>
            </td>
          <td>
            Antonin Nourisson, Sophia Missoury, A. Haouz, Marc Delarue
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Spermatogenesis is a complex process that can be disrupted by genetic and epigenetic changes, potentially leading to male infertility. Recent research has rapidly increased the number of protein coding mutations causally linked to impaired spermatogenesis in humans and mice. However, the role of non-coding mutations remains largely unexplored. As a case study to evaluate the effects of non-coding mutations on spermatogenesis, we first identified an evolutionarily conserved topologically associated domain (TAD) boundary near two genes with important roles in mammalian testis function: Dmrtb1 and Lrp8. We then used CRISPR-Cas9 to generate a mouse line where 26kb of the boundary was removed including a strong and evolutionarily conserved CTCF binding site. ChIP-seq and Hi-C experiments confirmed the removal of the CTCF site and a resulting increase in the DNA-DNA interactions across the domain boundary. Mutant mice displayed significant changes in testis gene expression, abnormal testis histology, a 35% drop in the estimated efficiency of spermatogenesis and a 28% decrease in daily sperm production compared to littermate controls. Despite these quantitative changes in testis function, mutant mice show no significant changes in fertility. This suggests that non-coding deletions affecting testis gene regulation may have smaller effects on fertility compared to coding mutations of the same genes. Our results demonstrate that disruption of a TAD boundary can have a negative impact on sperm production and highlight the importance of considering non-coding mutations in the analysis of patients with male infertility.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b89d71ce0de2a2d1a214068f2ba66cfb212c33d" target='_blank'>
              Deletion of an evolutionarily conserved TAD boundary compromises spermatogenesis in mice
              </a>
            </td>
          <td>
            Ana C. Lima, M. Okhovat, Alexandra M. Stendahl, Jake VanCampen, K. Nevonen, Jarod Herrera, Weiyu Li, L. Harshman, Ran Yang, Lev M. Fedorov, Katinka A. Vigh-Conrad, N. Ahituv, Donald F. Conrad, Lucia Carbone
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="The genetic integrity of pluripotent stem cells (PSC) is integral to their applications in research and therapy, but it is compromised by frequent development of copy number variations. Little is known about the basis of the variable genomic integrity among different PSC lines. Here we identify aneuploidies using RNA-seq and proteomics data from a panel of mouse embryonic stem cell (mESC) lines derived from 170 Diversity Outbred mice. We identified 62 lines with detectable aneuploid subpopulations and a subset of originally XX lines that lost one Chromosome X (XO). Strikingly, a much lower proportion of XX lines were aneuploid, compared to XY or XO lines. Two single-cell RNA-seq data sets demonstrated that aneuploid XY DO mESC also show lower Chromosome X gene expression and that XY mESC accumulate higher aneuploid proportions in culture than isogenic XX lines. Possible mechanisms for this protective effect of X chromosome dosage include our discovery that the lines with two active X Chromosomes have a higher expression of 2-cell-like state genes, and differential expression of X-linked tumor suppressor genes associated with DNA damage response. Highlights First genetic analysis of predisposition to aneuploidy in pluripotent stem cell cultures Chromosomal regions duplicated in aneuploid mouse embryonic stem cells are syntenic with regions overrepresented in human pluripotent stem cell lines bearing recurrent genetic abnormalities X-Chromosome dosage strongly influences susceptibility to aneuploidy in mouse embryonic stem cells and to a lesser degree in human pluripotent stem cells XX mouse embryonic stem cell lines show a higher proportion of cells in 2 cell-like state and higher expression of tumor suppressor genes associated with DNA damage response">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f853ae6e95a31cb9ad3712a21b0838a6601fe33" target='_blank'>
              Chromosome X Dosage Modulates Development of Aneuploidy in Genetically Diverse Mouse Embryonic Stem Cells
              </a>
            </td>
          <td>
            Alexander Stanton, Selcan Aydin, Daniel A. Skelly, Dylan Stavish, Kim Leonhard, Seth M. Taapken, Erik M. McIntire, Matthew Pankratz, A. Czechanski, T. Ludwig, Ted Choi, Steve P. Gygi, Ivana Barbaric, Steven C. Munger, L. Reinholdt, Martin F. Pera
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Xist, a pivotal player in X chromosome inactivation (XCI), has long been perceived as a cis-acting long noncoding RNA that binds exclusively to the inactive X chromosome (Xi). However, Xist’s ability to diffuse under select circumstances has also been documented, leading us to suspect that Xist RNA may have targets and functions beyond the Xi. Here, using female mouse embryonic stem cells (ES) and mouse embryonic fibroblasts (MEF) as models, we demonstrate that Xist RNA indeed can localize beyond the Xi. However, its binding is limited to ∼100 genes in cells undergoing XCI (ES cells) and in post-XCI cells (MEFs). The target genes are diverse in function but are unified by their active chromatin status. Xist binds discretely to promoters of target genes in neighborhoods relatively depleted for Polycomb marks, contrasting with the broad, Polycomb-enriched domains reported for human XIST RNA. We find that Xist binding is associated with down-modulation of autosomal gene expression. However, unlike on the Xi, Xist binding does not lead to full silencing and also does not spread beyond the target gene. Over-expressing Xist in transgenic ES cells similarly lead to autosomal gene suppression, while deleting Xist’s Repeat B motif reduces autosomal binding and perturbs autosomal down-regulation. Furthermore, treating female ES cells with the Xist inhibitor, X1, leads to loss of autosomal suppression. Altogether, our findings reveal Xist targets ∼100 genes beyond the Xi, identify Repeat B as a crucial domain for its in-trans function in mice, and indicate that autosomal targeting can be disrupted by the X1 small molecule inhibitor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bccb2a52596b1d4a0361228a895e9d27e5c7cb55" target='_blank'>
              Xist RNA binds select autosomal genes and depends on Repeat B to regulate their expression
              </a>
            </td>
          <td>
            Shengze Yao, Yesu Jeon, Barry A. Kesner, Jeannie T Lee
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Background Genomic variations contribute to the phenotypic diversity of individuals. A number of polymorphisms in protein-coding regions that alter drug efficacy or lead to adverse reactions have been characterized; however, noncoding regions that affect drug responses are largely overlooked, except for a limited number of well-studied enhancers. Results We conducted a quantitative assessment of cis-regulatory elements (CREs) based on transcription initiation profiling of mRNAs and noncoding RNAs, including enhancer RNAs, by using CAGE (Cap Analysis of Gene Expression). Candidate CREs identified in a hepatocellular carcinoma HepG2 cell line with stable expression of drug-responsive transcription factor pregnane X receptor (PXR) were further narrowed down by integrating data of PXR-binding sites in human primary hepatocytes and genome-wide association studies. We found more than 100-fold enrichments of the candidates to genetically associated loci with circulating levels of bilirubin and vitamin D, which implicated a link to adverse reactions of PXR ligands. We uncovered novel enhancers of UGT1A1 and TSKU through CRISPR/Cas9 knockout experiments. We identified alleles altering regulatory activities of UGT1A1 and CYP24A1enhancers by using luciferase reporter assay. Furthermore, our siRNA experiments revealed an unexpected impact of TSKU on the expression of vitamin D-metabolizing enzymes. Conclusions Our transcriptome-based assessment of CREs expanded the list of drug-inducible and PXR-mediated enhancers and super-enhancers. We identified regulatory alleles that alter drug-induced gene expressions, and discovered a novel molecular cascade associated with an adverse reaction. Our results contribute a precise understanding of the noncoding elements of the human genome underlying drug responses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f12494b9c7c48c33bdacd730ac5e83409d9983a" target='_blank'>
              Drug-induced cis-regulatory elements in human hepatocytes affect molecular phenotypes associated with adverse reactions
              </a>
            </td>
          <td>
            Saki Gotoh-Saito, Ryoko Wada, Hideya Kawaji
          </td>
          <td>2024-07-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Cycling cells replicate their DNA during the S phase through a defined temporal program known as replication timing. Mutation frequencies, epigenetic chromatin states, and transcriptional activities are different for genomic regions that are replicated early and late in the S phase. Here, we found from ChIP-Seq analysis that DNA polymerase (Pol) κ is enriched in early-replicating genomic regions in HEK293T cells. In addition, by feeding cells with N 2-heptynyl-2′-deoxyguanosine followed by click chemistry–based enrichment and high-throughput sequencing, we observed elevated Pol κ activities in genomic regions that are replicated early in the S phase. On the basis of the established functions of Pol κ in accurate and efficient nucleotide insertion opposite endogenously induced N 2-modified dG lesions, our work suggests that active engagement of Pol κ may contribute to diminished mutation rates observed in early-replicating regions of the human genome, including cancer genomes. Together, our work expands the functions of Pol κ and offered a plausible mechanism underlying replication timing–dependent mutation accrual in the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/417d907a79911389824959d3e561a3c8846017ac" target='_blank'>
              DNA polymerase κ participates in early S-phase DNA replication in human cells
              </a>
            </td>
          <td>
            Feng Tang, Yinan Wang, Ting Zhao, Jun Yuan, Andrew H. Kellum, Yinsheng Wang
          </td>
          <td>2024-07-01</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Neuro-Oncological Ventral Antigen 1 (NOVA1) is best known for its role in mediating an alternative splicing (AS) program in neurons, yet was first discovered as an antigen expressed in breast tumors, causing rare autoimmune reactions and paraneoplastic neurological disorders (PNDs). The PND model suggests a plausible role of the tumor antigen expression in tumor suppression, whereas it has emerged that NOVA may function as an oncogene in a variety of cancers. In addition, whether NOVA mediates AS in breast cancer remains unanswered. Here we examine the AS profiles of breast invasive carcinoma (BRCA) tumor samples and demonstrate that ectopic NOVA1 expression led to the activation of neuron-like splicing patterns in many genes, including exons targeted by NOVA in the brain. The splicing dysregulation is especially prevalent in cell periphery and cytoskeleton genes related to cell-cell communication, actin-based movement, and neuronal functions. We find that NOVA1-mediated AS is most prominent in Luminal A tumors and high NOVA1 expression in this subtype is associated with poorer prognosis. Our results suggest that ectopic NOVA1 in tumors has regulatory activity affecting pathways with high relevance to tumor progression and that this might be a more general mechanism for PND antigens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e50ff50fd94e0098cbe1b6b46456df93e0c5100" target='_blank'>
              Oncogenic NOVA1 expression dysregulates alternative splicing in breast cancer
              </a>
            </td>
          <td>
            Daniel F. Moakley, Chaolin Zhang
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Cancer stemness plays an important role in cancer initiation and progression, and is the major cause of tumor invasion, metastasis, recurrence, and poor prognosis. Non-coding RNAs (ncRNAs) are a class of RNA transcripts that generally cannot encode proteins and have been demonstrated to play a critical role in regulating cancer stemness. Here, we developed the ncStem database to record manually curated and predicted ncRNAs associated with cancer stemness. In total, ncStem contains 645 experimentally verified entries, including 159 long non-coding RNAs (lncRNAs), 254 microRNAs (miRNAs), 39 circular RNAs (circRNAs), and 5 other ncRNAs. The detailed information of each entry includes the ncRNA name, ncRNA identifier, disease, reference, expression direction, tissue, species, and so on. In addition, ncStem also provides computationally predicted cancer stemness-associated ncRNAs for 33 TCGA cancers, which were prioritized using the random walk with restart (RWR) algorithm based on regulatory and co-expression networks. The total predicted cancer stemness-associated ncRNAs included 11 132 lncRNAs and 972 miRNAs. Moreover, ncStem provides tools for functional enrichment analysis, survival analysis, and cell location interrogation for cancer stemness-associated ncRNAs. In summary, ncStem provides a platform to retrieve cancer stemness-associated ncRNAs, which may facilitate research on cancer stemness and offer potential targets for cancer treatment. Database URL: http://www.nidmarker-db.cn/ncStem/index.html.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fb97313e3402ad277fe895b46bce91bdb6c21f55" target='_blank'>
              ncStem: a comprehensive resource of curated and predicted ncRNAs in cancer stemness
              </a>
            </td>
          <td>
            Hui Liu, Nan Zhang, Yijie Jia, Jun Wang, Aokun Ye, Siru Yang, Honghan Zhou, Yingli Lv, Chaohan Xu, Shuyuan Wang
          </td>
          <td>2024-08-13</td>
          <td>Database: The Journal of Biological Databases and Curation</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Each piece of cell-free DNA (cfDNA) has a length determined by the exact metabolic conditions in the cell it belonged to at the time of cell death. The changes in cellular regulation leading to a variety of patterns, which are based on the different number of fragments with lengths up to several hundred base pairs at each of the almost three billion genomic positions, allow for the detection of disease and also the precise identification of the tissue of their origin. A Kullback-Leibler (KL) divergence computation identifies different fragment lengths and areas of the human genome, depending on the stage, for which disease samples, starting from pre-clinical disease stages, diverge from healthy donor samples. We provide examples of genes related to colorectal cancer (CRC), which our algorithm detected to belong to divergent genomic bins. The staging of CRC can be viewed as a Markov Chain and that provides a framework for studying disease progression and the types of epigenetic changes occurring longitudinally at each stage, which might aid the correct classification of a new hospital sample. In a new look to treat such data as grayscale value images, pattern recognition using artificial intelligence (AI) could be one approach to classification. In CRC, Stage I disease does not, for the most part, shed any tumor circulation, making detection difficult for established machine learning (ML) methods. This leads to the deduction that early detection, where we can only rely on changes in the metabolic patterns, can be accomplished when the information is considered in its entirety, for example by applying computer vision methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/769c868029bf18fe644fcae9a2141c21f2b5a763" target='_blank'>
              Channel Capacity of Genome-Wide Cell-Free DNA Fragment Length Distribution in Colorectal Cancer
              </a>
            </td>
          <td>
            A. Matov
          </td>
          <td>2024-07-18</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Background Germline mutations have been identified in a small number of hereditary cancers, but the genetic predisposition for many familial cancers remains to be elucidated. Methods This study identified a Chinese pedigree that presented different cancers (breast cancer, BRCA; adenocarcinoma of the esophagogastric junction, AEG; and B‐cell acute lymphoblastic leukemia, B‐ALL) in each of the three generations. Whole‐genome sequencing and whole‐exome sequencing were performed on peripheral blood or bone marrow and cancer biopsy samples. Whole‐genome bisulfite sequencing was conducted on the monozygotic twin brothers, one of whom developed B‐ALL. Results According to the ACMG guidelines, bioinformatic analysis of the genome sequencing revealed 20 germline mutations, particularly mutations in the DNAH11 (c.9463G > A) and CFH (c.2314G > A) genes that were documented in the COSMIC database and validated by Sanger sequencing. Forty‐one common somatic mutated genes were identified in the cancer samples, displaying the same type of single nucleotide substitution Signature 5. Meanwhile, hypomethylation of PLEK2, MRAS, and RXRA as well as hypermethylation of CpG island associated with WT1 was shown in the twin with B‐ALL. Conclusions These findings reveal genomic alterations in a pedigree with multiple cancers. Mutations found in the DNAH11, CFH genes, and other genes predispose to malignancies in this family. Dysregulated methylation of WT1, PLEK2, MRAS, and RXRA in the twin with B‐ALL increases cancer susceptibility. The similarity of the somatic genetic changes among the three cancers indicates a hereditary impact on the pedigree. These familial cancers with germline and somatic mutations, as well as epigenomic alterations, represent a common molecular basis for many multiple cancer pedigrees.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d3f0ae27a463e75a2f599e816c3a044e8b7c954" target='_blank'>
              Impact of genomic and epigenomic alterations of multigene on a multicancer pedigree
              </a>
            </td>
          <td>
            Jinyu Gao, Yongzhang Wu, Jieming Yu, Yinbin Qiu, Tiantian Yi, Chaochao Luo, Junxiao Zhang, Gary Lu, Xu Li, Fu Xiong, Xuedong Wu, Xinghua Pan
          </td>
          <td>2024-07-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a2e505484ed8ee3ae85c30bb5570c5fc512848e2" target='_blank'>
              Alternative splicing of BAZ1A in colorectal cancer disrupts the DNA damage response and increases chemosensitization
              </a>
            </td>
          <td>
            Nivedhitha Mohan, G. Johnson, Jorge Enrique Tovar Perez, W. Dashwood, P. Rajendran, Roderick H. Dashwood
          </td>
          <td>2024-08-01</td>
          <td>Cell Death & Disease</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f019d23b5adc09f83ebfe644966cdb6ede88d903" target='_blank'>
              Immune evasion impacts the landscape of driver genes during cancer evolution
              </a>
            </td>
          <td>
            Lucie E. Gourmet, A. Sottoriva, Simon Walker-Samuel, M. Secrier, L. Zapata
          </td>
          <td>2024-06-26</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b0e84c70be39b55c5ab372a108ddda6ab970765" target='_blank'>
              Polθ: emerging synthetic lethal partner in homologous recombination-deficient tumors.
              </a>
            </td>
          <td>
            T. B. Bazan Russo, Clarissa Mujacic, Emilia Di Giovanni, Maria Concetta Vitale, Carla Ferrante Bannera, U. Randazzo, Silvia Contino, M. Bono, V. Gristina, A. Galvano, A. Perez, G. Badalamenti, A. Russo, V. Bazan, L. Incorvaia
          </td>
          <td>2024-08-09</td>
          <td>Cancer gene therapy</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1acdebafa769f861e73d2cd78d8c49c2d3b69320" target='_blank'>
              USP53 Affects the Proliferation and Apoptosis of Breast Cancer Cells by Regulating the Ubiquitination Level of ZMYND11
              </a>
            </td>
          <td>
            Xiangchao Meng, Hongye Chen, Zhihui Tan, Weitao Yan, Yinfeng Liu, Ji Lv, Meng Han
          </td>
          <td>2024-07-23</td>
          <td>Biological Procedures Online</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="INTRODUCTION
Advances in understanding of tumor biology shed light on hallmarks of cancer development and progression that include dysregulated DNA damage repair (DDR) machinery. Leveraging underlying tumor genomic instability and tumor specific defects in DDR, Poly (ADP-ribose) polymerases (PARP) inhibitors (PARPi) induced DNA damage emerges as a novel non-chemotherapy therapeutic opportunity. PARPis are currently approved in multiple tumor types, with the largest benefit seen in tumors with homologous recombination repair (HRR) deficiency, including germline and somatic mutations in BRCA1/2 genes (BRCA) and other pathway members such as PALB2 and Rad51c.


AREAS COVERED
This review article summarizes the current approval landscape and known and proposed mechanisms of resistance to PARPi. Further, therapeutic strategies to overcome PARPi resistance are discussed, including ongoing clinical trials.


EXPERT OPINION
PARPi have proven to be a safe and effective therapy and represent a cornerstone treatment across multiple solid tumor types. Elucidating innate and acquired mechanisms of resistance, coupled with the emergence of novel therapeutic options to capitalize on the activity of PARPi and prevent or reverse the acquisition of resistance, provides an opportunity to further expand the role of PARPi in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ec7393511b29f158c45a3ee7ca1c8ffe71c659b" target='_blank'>
              Strategies for the prevention or reversal of PARP inhibitor resistance.
              </a>
            </td>
          <td>
            Zahi I. Mitri, Shaun M Goodyear, Gordon B. Mills
          </td>
          <td>2024-08-15</td>
          <td>Expert review of anticancer therapy</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f9c5a9b1b536dcc4bfe3e239817233ef120675cf" target='_blank'>
              Werner helicase interacting protein 1 contributes to G-quadruplex processing in human cells
              </a>
            </td>
          <td>
            Lili Hegedűs, Á. Tóth, Gábor M. Harami, J. Pálinkás, Nargis Karatayeva, Enikő Sajben-Nagy, Szabolcs Bene, Sara Afzali Jaktajdinani, Mihaly Kovacs, Szilvia Juhász, Peter Burkovics
          </td>
          <td>2024-07-08</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Targeted demethylation by DNA glycosylases (DNGs) results in differential methylation between parental alleles in the endosperm, which drives imprinted expression. Here, we performed RNA sequencing on endosperm derived from DNG mutant mdr1 and wild-type endosperm. Consistent with the role of DNA methylation in gene silencing, we find 96 gene and 86 TE differentially expressed (DE) transcripts that lost expression in the hypermethylated mdr1 mutant. Compared with other endosperm transcripts, the mdr1 targets are enriched for TEs (particularly Helitrons), and DE genes are depleted for both core genes and GO term assignments, suggesting that the majority of DE transcripts are TEs and pseudo-genes. By comparing DE genes to imprinting calls from prior studies, we find that the majority of DE genes have maternally biased expression, and approximately half of all maternally expressed genes (MEGs) are DE in this study. In contrast, no paternally expressed genes (PEGs) are DE. DNG-dependent imprinted genes are distinguished by maternal demethylation and expression primarily in the endosperm, so we also performed EM-seq on hybrids to identify maternal demethylation and utilized a W22 gene expression atlas to identify genes expressed primarily in the endosperm. Overall, approximately ⅔ of all MEGs show evidence of regulation by DNA glycosylases. Taken together, this study solidifies the role of MDR1 in the regulation of maternally expressed, imprinted genes and TEs and identifies subsets of genes with DNG-independent imprinting regulation. Significance Statement This work investigates the transcriptome changes resulting from the loss of function of DNA glycosylase MDR1, revealing that, in wild-type endosperm, targets of MDR1 are expressed predominantly from the maternal allele and this expression is suppressed in mutants. Furthermore, by combining expression data, DNA methylation data, and developmental expression data, we are able to categorize all maternally expressed, imprinted genes based on DNA glycosylase dependent or independent regulatory methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/facf4dda707b67b3b56d646f95496db5b6cc7df8" target='_blank'>
              MDR1 DNA glycosylase regulates the expression of genomically imprinted genes and helitrons
              </a>
            </td>
          <td>
            Kaitlin M. Higgins, J. Gent, Sarah N. Anderson
          </td>
          <td>2024-08-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Cell-free DNA (cfDNA) is increasingly studied for its diverse applications in non-invasive detection. Non-randomly cleaved by nucleases and released into the bloodstream, cfDNA exhibits a variety of intrinsic fragmentation patterns indicative of cell status. Particularly, these fragmentation patterns have recently been demonstrated to be effective in predicting cancer and its tissue-of-origin, owing to increased variation of fragmentation features observed in tumor patients. However, there remains a lack of detailed exploration of altered cfDNA fragmentation profiles in tumors, which consist of a mixture of both non-tumor cfDNA and circulating tumor DNA (ctDNA). Hence, we leveraged the human tumor cell line-derived xenograft (CDX) mouse model, where different tumor cell lines were implanted into different anatomical sites, to isolate pure ctDNA and separately investigate the fragment properties of CDX-induced cfDNA and ctDNA. We found an enrichment of short cfDNA fragments in both CDX-induced cfDNA and ctDNA compared to normal plasma cfDNA, with more elevated short fragments in ctDNA. Moreover, the CDX-induced cfDNA fragmentation features distinguished between CDX models of different tumor cell lines, while the ctDNA fragmentation features conversely discriminate between CDX models of different anatomical sites. The results suggested that both non-tumor cfDNA and ctDNA contribute to the increased variation observed in tumors, and that cfDNA fragmentation may be highly variable and susceptible to regulations by both original cells and cells within the local niche.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6138f9d167b66612f618b5e26b115af024b7013f" target='_blank'>
              Dissecting cell-free DNA fragmentation variation in tumors using cell line-derived xenograft mouse
              </a>
            </td>
          <td>
            Ruiqing Fu, He Amy Su, Yafei Tian, Hongyan Chen, Daru Lu
          </td>
          <td>2024-08-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Functional copy-number alterations (fCNAs) are DNA copy-number changes with concordant differential gene expression. These are less likely to be bystander genetic lesions and could serve as robust and reproducible tumor biomarkers. To identify candidate fCNAs in neuroendocrine tumors (NETs), we integrated chromosomal microarray (CMA) and RNA-seq differential gene-expression data from 31 pancreatic (pNETs) and 33 small-bowel neuroendocrine tumors (sbNETs). Tumors were resected from 47 early-disease-progression (<24 months) and 17 late-disease-progression (>24 months) patients. Candidate fCNAs that accurately differentiated these groups in this discovery cohort were then replicated using fluorescence in situ hybridization (FISH) on formalin-fixed, paraffin-embedded (FFPE) tissues in a larger validation cohort of 60 pNETs and 82 sbNETs (52 early- and 65 late-disease-progression samples). Logistic regression analysis revealed the predictive ability of these biomarkers, as well as the assay-performance metrics of sensitivity, specificity, and area under the curve. Our results indicate that copy-number changes at chromosomal loci 4p16.3, 7q31.2, 9p21.3, 17q12, 18q21.2, and 19q12 may be used as diagnostic and prognostic NET biomarkers. This involves a rapid, cost-effective approach to determine the primary tumor site for patients with metastatic liver NETs and to guide risk-stratified therapeutic decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/976250a22dce56b41eaaf1761c3d5be047660920" target='_blank'>
              Functional Copy-Number Alterations as Diagnostic and Prognostic Biomarkers in Neuroendocrine Tumors
              </a>
            </td>
          <td>
            Hayley Vaughn, Heather Major, Evangeline Kadera, Kendall Keck, Timothy Dunham, Qining Qian, Bartley Brown, Aaron Scott, Andrew M. Bellizzi, Terry A. Braun, Patrick Breheny, Dawn E. Quelle, James R. Howe, Benjamin W Darbro
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b40bf07806bd0a97226f528a2d387e1d1cd79723" target='_blank'>
              Enhancers associated with unstable RNAs are rare in plants.
              </a>
            </td>
          <td>
            Bayley R McDonald, Colette L. Picard, Ian M. Brabb, Marina I. Savenkova, Robert J. Schmitz, Steve Jacobsen, Sascha H. C. Duttke
          </td>
          <td>2024-07-30</td>
          <td>Nature plants</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d52a6a8ffb7a2117b993b45f9223ba99b461abc" target='_blank'>
              Chromosomal structural rearrangements implicate long non-coding RNAs in rare germline disorders
              </a>
            </td>
          <td>
            Rebecca E Andersen, Ibrahim F Alkuraya, Abna Ajeesh, Tyler Sakamoto, E. L. Mena, S. Amr, Hila Romi, Margaret A Kenna, Caroline D Robson, Ellen S Wilch, Katarena Nalbandian, Raul Piña-Aguilar, Christopher A. Walsh, Cynthia C Morton
          </td>
          <td>2024-07-26</td>
          <td>Human Genetics</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Transcription and splicing errors in cancer cells generate erroneous transcripts. Since erroneous transcripts are degraded by the nonsense-mediated mRNA decay (NMD) pathway, whether they are junk or could be functional has been overlooked and understudied. We addressed this question by first performing a pan-cancer analysis and identified substantial erroneous noncoding transcripts (ENT) in cancers. Given that RNA/DNA binding domains (DBD) were predicted in ENTs, we deleted predicted DBDs in multiple ENTs in multiple cell lines, with RNA-sequencing and cell experiments before and after DBD deletion. DBD deletion caused significantly changed expression of ENTs’ target genes (whose promoter regions contain ENTs’ DNA binding sites, DBS) and changed cell migration and proliferation ability, indicating that many ENTs can transcriptionally regulate genes. Tightly coupled data analysis and experiments reveal that ENTs’ functions are highly cancer- and cellular-context specific, making ENTs a new class of safe and specific targets for noncoding RNA-based cancer therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c2e45af8d1c765a560ff555893aa2ed200e47f3c" target='_blank'>
              Many erroneous noncoding transcripts in cancer cells can highly specifically regulate cancer-related genes and pathways
              </a>
            </td>
          <td>
            Sha He, Wei Xiong, Jianping Huo, Jie Lin, Jianmin Li, Hao Zhu
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Background/Aim: BRCA1/2 mutations in breast cancer cells impair homologous recombination and promote alternative end joining (Alt-EJ) for DNA-damage repair. DNA polymerase theta, encoded by POLQ, plays a crucial role in Alt-EJ, making it a potential therapeutic target, particularly in BRCA1/2-mutant cancers. Methionine restriction is a promising approach to target cancer cells due to their addiction to this amino acid. The present study investigated the expression of POLQ in BRCA1/2 wild-type and BRCA1-mutant breast cancer cells under methionine restriction. Materials and Methods: POLQ mRNA expression was measured using qRT-PCR in BRCA1/2 wild-type (MDA-MB-231) and BRCA1- mutant (HCC1937 and MDA-MB-436) breast-cancer cells under normal, or serum-restricted, or serum- and methionine-restricted conditions. Results: Compared to BRCA1/2 wild-type cells, BRCA1-mutant cells displayed significantly higher basal POLQ expression in normal medium. Methionine restriction further increased POLQ expression in the BRCA1-mutant cells but decreased it in the BRCA1/2 wild-type cells. Conclusion: The present findings suggest that methionine restriction showed differential effects on POLQ expression, potentially impacting Alt-EJ activity, in BRCA1/2 wild-type and BRCA1-mutant breast-cancer cells. Further investigation is needed to explore the potential of combining methionine restriction with DNA-repair inhibitors, such as PARP inhibitors, to overcome drug resistance in BRCA1/2 mutant cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f0523a0d152aaab9d94d339ede11927c41a78c4b" target='_blank'>
              Induction of the DNA-Repair Gene POLQ only in BRCA1-mutant Breast-Cancer Cells by Methionine Restriction
              </a>
            </td>
          <td>
            T. Kunihisa, Sachiko Inubushi, Hirokazu Tanino, Robert M. Hoffman
          </td>
          <td>2024-06-29</td>
          <td>Cancer Genomics & Proteomics</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/799e1ddffbac6e1b1f9a23cb6190cda5aacba309" target='_blank'>
              Spatial multiomics reveals a subpopulation of fibroblasts associated with cancer stemness in human hepatocellular carcinoma
              </a>
            </td>
          <td>
            Si Jing, Dan Liu, Na Feng, Hui Dong, He-Qi Wang, Xi Yan, Xu-Feng Chen, Mincheng Qu, Ping Lin, Bin Yi, Feiling Feng, Lei Chen, Hong-Yang Wang, Hong Li, Yufei He
          </td>
          <td>2024-08-13</td>
          <td>Genome Medicine</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Replication stress (RS) is a key trait of cancer cells, and a potential actionable target in cancer treatment. Accurate methods to measure RS in tumour samples are currently lacking. DNA fibre analysis has been used as a common technique to measure RS in cell lines. Here, we investigated DNA fibre analysis on fresh breast cancer specimens and correlated DNA replication kinetics to known RS markers and genomic alterations. Fresh, treatment-naïve primary breast cancer samples (n = 74) were subjected to ex vivo DNA fibre analysis to measure DNA replication kinetics. Tumour cell proliferation was confirmed by EdU incorporation and cytokeratin AE1/AE3 (CK) staining. The RS markers phospho-S33-RPA and γH2AX and the RS-inducing proto-oncogenes Cyclin E1 and c-Myc were analysed by immunohistochemistry. Copy number variations (CNVs) were assessed from genome-wide single nucleotide polymorphism (SNP) arrays. We found that the majority of proliferating (EdU-positive) cells in each sample were CK-positive and therefore considered to be tumour cells. DNA fibre lengths varied largely in most tumour samples. The median DNA fibre length showed a significant inverse correlation with pRPA expression (r = -0.29, p = 0.033) but was not correlated with Cyclin E1 or c-Myc expression and global CNVs in this study. Nuclear Cyclin E1 expression showed a positive correlation with pRPA levels (r = 0.481, p < 0.0001), while cytoplasmic Cyclin E1 expression exhibited an inverse association with pRPA expression (r = -0.353, p = 0.002) and a positive association with global CNVs (r = 0.318, p = 0.016). In conclusion, DNA fibre analysis performed with fresh primary breast cancer samples is feasible. Fibre lengths were associated with pRPA expression. Cyclin E1 expression was associated with pRPA and the percentage of CNVs. © 2024 The Author(s). The Journal of Pathology published by John Wiley & Sons Ltd on behalf of The Pathological Society of Great Britain and Ireland.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ed27868f78a660923e19eff55fc51acf1aa4b35" target='_blank'>
              Functional ex vivo DNA fibre assay to measure replication dynamics in breast cancer tissue.
              </a>
            </td>
          <td>
            Mengting Chen, Nathalie van den Tempel, A. Bhattacharya, Shibo Yu, B. Rutgers, R. Fehrmann, Sander de Haas, Bert van der Vegt, M. V. van Vugt
          </td>
          <td>2024-07-18</td>
          <td>The Journal of pathology</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Non-SMC condensin I complex subunit D2 (NCAPD2) is a newly identified oncogene; however, the specific biological function and molecular mechanism of NCAPD2 in liver cancer progression remain unknown. In the present study, the aberrant expression of NCAPD2 in liver cancer was investigated using public tumor databases, including TNMplot, The Cancer Genome Atlas and the International Cancer Genome Consortium based on bioinformatics analyses, and it was validated using a clinical cohort. It was revealed that NCAPD2 was significantly upregulated in liver cancer tissues compared with in control liver tissues, and NCAPD2 served as an independent prognostic factor and predicted poor prognosis in liver cancer. In addition, the expression of NCAPD2 was positively correlated with the percentage of Ki67+ cells. Finally, single-cell sequencing data, gene-set enrichment analyses and in vitro investigations, including cell proliferation assay, Transwell assay, wound healing assay, cell cycle experiments, cell apoptosis assay and western blotting, were carried out in human liver cancer cell lines to assess the biological mechanisms of NCAPD2 in patients with liver cancer. The results revealed that the upregulation of NCAPD2 enhanced tumor cell proliferation, invasion and cell cycle progression at the G2/M-phase transition, and inhibited apoptosis in liver cancer cells. Furthermore, NCAPD2 overexpression was closely associated with the phosphatidylinositol 3-kinase (PI3K)-Akt-mammalian target of rapamycin (mTOR)/c-Myc signaling pathway and epithelial-mesenchymal transition (EMT) progression in HepG2 and Huh7 cells. In addition, upregulated NCAPD2 was shown to have adverse effects on overall survival and disease-specific survival in liver cancer. In conclusion, the overexpression of NCAPD2 was shown to lead to cell cycle progression at the G2/M-phase transition, activation of the PI3K-Akt-mTOR/c-Myc signaling pathway and EMT progression in human liver cancer cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6873a90e35d5a8f62be3a51b8f0dd6c3ab4c961" target='_blank'>
              NCAPD2 augments the tumorigenesis and progression of human liver cancer via the PI3K-Akt-mTOR signaling pathway
              </a>
            </td>
          <td>
            Jiangxue Gu, Ke Huang, Wei-Lin Zhao, Xiao-Ming Zheng, Yu-Qin Wu, Shirong Yan, Yu-Gang Huang, Pei Hu
          </td>
          <td>2024-07-31</td>
          <td>International Journal of Molecular Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Background: Cervical cancer (CC) stands as a significant health threat to women globally, with high-risk human papillomaviruses as major etiologic agents. The DNA damage repair (DDR) protein topoisomerase I (TOP1) has been linked to various cancers, yet its distinct roles and mechanisms in CC are not fully elucidated. Methods: We investigated TOP1 expression in cervical intraepithelial neoplasia (CIN) and CC tissues utilizing qRT-PCR and IHC, correlating findings with patient prognosis. Subsequent knockdown studies were performed in vitro and in vivo to evaluate the influence of TOP1 on tumor growth, DNA repair, and inflammatory responses. Results: TOP1 was highly expressed in CIN and CC, negatively correlating with patient prognosis. Inhibition of TOP1 impeded CC cell growth and disrupted DNA repair. TOP1 was shown to regulate tumor-promoting inflammation and programmed death-ligand 1 (PD-L1) production in a cGAS-dependent manner. HPV oncoproteins E6 and E7 upregulated TOP1 and activated the cGAS-PD-L1 pathway. Conclusions: TOP1 acts as a DNA repair mediator, promoting CC development and immune evasion. Targeting the TOP1-cGAS-PD-L1 axis could be a potential therapeutic strategy for CC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/11f71748837721c3a12ecd5a0bb258b1048a43b9" target='_blank'>
              Oncoproteins E6 and E7 upregulate topoisomerase I to activate the cGAS-PD-L1 pathway in cervical cancer development
              </a>
            </td>
          <td>
            Ying Luo, Mengda Niu, Yanfei Liu, Miaochang Zhang, Yuanyuan Deng, Dan Mu, Junfen Xu, Shiyuan Hong
          </td>
          <td>2024-08-02</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a190a83f30593fab67bd2e0b946b6e5dcca9e7" target='_blank'>
              STAU1-mediated CNBP mRNA degradation by LINC00665 alters stem cell characteristics in ovarian cancer
              </a>
            </td>
          <td>
            Xiaofang Liu, Yang Chen, Ying Li, Jinling Bai, Zhi Zeng, Min Wang, Yaodong Dong, Yingying Zhou
          </td>
          <td>2024-07-30</td>
          <td>Biology Direct</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Metastatic castration-resistant prostate cancer (mCRPC) is a lethal disease that resists therapy targeting androgen signaling, the primary driver of prostate cancer. mCRPC resists androgen receptor (AR) inhibitors by amplifying AR signaling or by evolving into therapy-resistant subtypes that do not depend on AR. Elucidation of the epigenetic underpinnings of these subtypes could provide important insights into the drivers of therapy resistance. In this study, we produced chromatin accessibility maps linked to the binding of lineage-specific transcription factors (TF) by performing ATAC sequencing on 70 mCRPC tissue biopsies integrated with transcriptome and whole genome sequencing. mCRPC had a distinct global chromatin accessibility profile linked to AR function. Analysis of TF occupancy across accessible chromatin revealed 203 TFs associated with mCRPC subtypes. Notably, ZNF263 was identified as a putative prostate cancer TF with a significant impact on gene activity in the double-negative (AR- neuroendocrine-) subtype, potentially activating MYC targets. Overall, this analysis of chromatin accessibility in mCRPC provides valuable insights into epigenetic changes that occur during progression to mCRPC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ac71bf183535cdb84fd367ef22969e3b56650628" target='_blank'>
              An Atlas of Accessible Chromatin in Advanced Prostate Cancer Reveals the Epigenetic Evolution during Tumor Progression.
              </a>
            </td>
          <td>
            Raunak Shrestha, L. Chesner, Meng Zhang, Stanley Zhou, A. Foye, A. Lundberg, Alana S Weinstein, M. Sjöström, Xiaolin Zhu, Thaidy Moreno-Rodriguez, Haolong Li, Su C/Pcf West Coast Prostate Cancer Dream Team, J. Alumkal, Rahul Aggarwal, Eric J Small, Mathieu Lupien, David A Quigley, Felix Y Feng
          </td>
          <td>2024-07-11</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="TBC1D3 is a primate-specific gene family that has expanded in the human lineage and has been implicated in neuronal progenitor proliferation and expansion of the frontal cortex. The gene family and its expression have been challenging to investigate because it is embedded in high-identity and highly variable segmental duplications. We sequenced and assembled the gene family using long-read sequencing data from 34 humans and 11 non-human primate species. Our analysis shows that this particular gene family has independently duplicated in at least five primate lineages, and the duplicated loci are enriched at sites of large-scale chromosomal rearrangements on Chromosome 17. We find that all human copy number variation maps to two distinct clusters located at Chromosome 17q12 and that humans are highly structurally variable at this locus, differing by as many as 20 copies and ~1 Mbp in length depending on haplotypes. We also show evidence of positive selection, as well as a significant change in the predicted human TBC1D3 protein sequence. Lastly, we find that, despite multiple duplications, human TBC1D3 expression is limited to a subset of copies and, most notably, from a single paralog group: TBC1D3-CDKL These observations may help explain why a gene potentially important in cortical development can be so variable in the human population.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/872db936ea87de98bc9c479ae7fb31da39d2f13a" target='_blank'>
              Independent expansion, selection and hypervariability of the TBC1D3 gene family in humans.
              </a>
            </td>
          <td>
            Xavi Guitart, David Porubsky, DongAhn Yoo, Max L. Dougherty, Philip C. Dishuck, Katherine M. Munson, Alexandra P. Lewis, Kendra Hoekzema, Jordan Knuth, Stephen Chang, Tomi Pastinen, Evan E. Eichler
          </td>
          <td>2024-08-06</td>
          <td>Genome research</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="MYC has been implicated in the pathogenesis of a wide range of human tumors and has been described for many years as a transcription factor that regulates genes with pleiotropic functions to promote tumorigenic growth. However, despite extensive efforts to identify specific target genes of MYC that alone could be responsible for promoting tumorigenesis, the field is yet to reach a consensus whether this is the crucial function of MYC. Recent work shifts the view on MYC's function from being a gene-specific transcription factor to an essential stress resilience factor. In highly proliferating cells, MYC preserves cell integrity by promoting DNA repair at core promoters, protecting stalled replication forks, and/or preventing transcription-replication conflicts. Furthermore, an increasing body of evidence demonstrates that MYC not only promotes tumorigenesis by driving cell-autonomous growth, but also enables tumors to evade the host's immune system. In this review, we summarize our current understanding of how MYC impairs antitumor immunity and why this function is evolutionarily hard-wired to the biology of the MYC protein family. We show why the cell-autonomous and immune evasive functions of MYC are mutually dependent and discuss ways to target MYC proteins in cancer therapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/877f0298df11cd9063e84174a42a5eb6afca2c71" target='_blank'>
              Immune evasion: An imperative and consequence of MYC deregulation.
              </a>
            </td>
          <td>
            Bastian Krenz, Jongkuen Lee, Toshitha Kannan, Martin Eilers
          </td>
          <td>2024-07-02</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="New high-quality human genome assemblies derived from lymphoblastoid cell lines (LCLs) provide reference genomes and pangenomes for genomics studies. However, the characteristics of LCLs pose technical challenges to profiling immunoglobulin (IG) genes. IG loci in LCLs contain a mixture of germline and somatically recombined haplotypes, making them difficult to genotype or assemble accurately. To address these challenges, we introduce IGLoo, a software tool that implements novel methods for analyzing sequence data and genome assemblies derived from LCLs. IGLoo characterizes somatic V(D)J recombination events in the sequence data and identifies the breakpoints and missing IG genes in the LCL-based assemblies. Furthermore, IGLoo implements a novel reassembly framework to improve germline assembly quality by integrating information about somatic events and population structural variantions in the IG loci. We applied IGLoo to study the assemblies from the Human Pangenome Reference Consortium, providing new insights into the mechanisms, gene usage, and patterns of V(D)J recombination, causes of assembly fragmentation in the IG heavy chain (IGH) locus, and improved representation of the IGH assemblies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8f48cf5eb22832827a1cb138fb7f409be2963b7" target='_blank'>
              IGLoo: Profiling the Immunoglobulin Heavy chain locus in Lymphoblastoid Cell Lines with PacBio High-Fidelity Sequencing reads
              </a>
            </td>
          <td>
            Mao-Jan Lin, Ben Langmead, Yana Safonova
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The mechanisms that work alongside BRAFV600E oncogene in melanoma development, in addition to ultraviolet (UV) radiation (UVR), are of great interest. Analysis of human melanoma tumors [data from The Cancer Genome Atlas (TCGA)] revealed that 50% or more of the samples expressed no or low amounts of serine/threonine protein kinase STK11 (also known as LKB1) protein. Here, we report that, in a mouse model, concomitant neonatal BrafV600E activation and Lkb1 tumor suppressor ablation in melanocytes led to full melanoma development. A single postnatal dose of UVB radiation had no effect on melanoma onset in Lkb1-depleted mice compared with BrafV600E-irradiated mice, but increased tumor multiplicity. In concordance with these findings and previous reports, Lkb1-null irradiated mice exhibited deficient DNA damage repair (DDR). Histologically, tumors lacking Lkb1 were enriched in neural-like tumor morphology. Genetic profiling and gene set enrichment analyses of tumor sample mutated genes indicated that loss of Lkb1 promoted the selection of altered genes associated with neural differentiation processes. Thus, these results suggest that the loss of Lkb1 cooperates with BrafV600E and UVR, impairing the DDR and increasing melanoma multiplicity and neural-like dedifferentiation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/468890566d37157a91439461401c20d4b0665a15" target='_blank'>
              Loss of Lkb1 cooperates with BrafV600E and ultraviolet radiation, increasing melanoma multiplicity and neural-like dedifferentiation.
              </a>
            </td>
          <td>
            K. McGrail, Elena González-Sánchez, Paula Granado-Martínez, Roberto Orsenigo, Yuxin Ding, Berta Ferrer, Javier Hernández-Losa, Iván Ortega, Juan Martín-Caballero, Eva Muñoz-Couselo, Vicente García-Patos, Juan A. Recio
          </td>
          <td>2024-08-08</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract CCCTC-binding factor (CTCF) is an insulator protein that binds to a highly conserved DNA motif and facilitates regulation of three-dimensional (3D) nuclear architecture and transcription. CTCF binding sites (CTCF-BSs) reside in non-coding DNA and are frequently mutated in cancer. Our previous study identified a small subclass of CTCF-BSs that are resistant to CTCF knock down, termed persistent CTCF binding sites (P-CTCF-BSs). P-CTCF-BSs show high binding conservation and potentially regulate cell-type constitutive 3D chromatin architecture. Here, using ICGC sequencing data we made the striking observation that P-CTCF-BSs display a highly elevated mutation rate in breast and prostate cancer when compared to all CTCF-BSs. To address whether P-CTCF-BS mutations are also enriched in other cell-types, we developed CTCF-INSITE—a tool utilising machine learning to predict persistence based on genetic and epigenetic features of experimentally-determined P-CTCF-BSs. Notably, predicted P-CTCF-BSs also show a significantly elevated mutational burden in all 12 cancer-types tested. Enrichment was even stronger for P-CTCF-BS mutations with predicted functional impact to CTCF binding and chromatin looping. Using in vitro binding assays we validated that P-CTCF-BS cancer mutations, predicted to be disruptive, indeed reduced CTCF binding. Together this study reveals a new subclass of cancer specific CTCF-BS DNA mutations and provides insights into their importance in genome organization in a pan-cancer setting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/994d6f59097fc218409b42c901b1ae72776752f9" target='_blank'>
              Machine learning enables pan-cancer identification of mutational hotspots at persistent CTCF binding sites
              </a>
            </td>
          <td>
            Wenhan Chen, Y. Zeng, Joanna Achinger-Kawecka, Elyssa Campbell, Alicia K Jones, A. Stewart, Amanda Khoury, Susan J Clark
          </td>
          <td>2024-07-02</td>
          <td>Nucleic Acids Research</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Combined CDK4/6 inhibitor (CDK4/6i) and endocrine therapy significantly improves outcome for patients with estrogen receptor-positive (ER+) metastatic breast cancer, but drug resistance and thus disease progression inevitably occur. Herein, we aimed to identify genomic alterations associated with combined CDK4/6i and endocrine therapy resistance, and follow the levels of specific mutations in longitudinal circulating tumor DNA (ctDNA) for early detection of progression. From a cohort of 86 patients with ER+ metastatic breast cancer we performed whole exome sequencing or targeted sequencing of paired tumor (N = 8) or blood samples (N = 5) obtained before initiation of combined CDK4/6i and endocrine therapy and at disease progression. Mutations in oncogenic genes at progression were rare, while amplifications of growth-regulating genes were more frequent. The most frequently acquired alterations observed were PIK3CA and TP53 mutations and PDK1 amplification. Longitudinal ctDNA dynamics of mutant PIK3CA or private mutations revealed increased mutation levels at progression in 8 of 10 patients (80%). Impressively, rising levels of PIK3CA-mutated ctDNA were detected 4-17 months before imaging. Our data add to the growing evidence supporting longitudinal ctDNA analysis for real-time monitoring of CDK4/6i response and early detection of progression in advanced breast cancer. Further, our analysis suggests that amplification of growth-related genes may contribute to combined CDK4/6i and endocrine therapy resistance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da4f8f900b692158e74287adf0a693a5a4d08296" target='_blank'>
              Genomic alterations associated with resistance and circulating tumor DNA dynamics for early detection of progression on CDK4/6 inhibitor in advanced breast cancer.
              </a>
            </td>
          <td>
            Charlotte K Kindt, Carla L. Alves, S. Ehmsen, Amalie Kragh, T. Reinert, Marianne Vogsen, A. R. Kodahl, Jeanette D Rønlev, Dilan Ardik, Anna L Sørensen, Kirstine Evald, Mia L Clemmensen, J. Staaf, Henrik J Ditzel
          </td>
          <td>2024-08-11</td>
          <td>International journal of cancer</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Super-enhancer-associated transcription factor networks define cell identity in neuroblastoma (NB). Dysregulation of these transcription factors contributes to the initiation and maintenance of NB by enforcing early developmental identity states. We report that the class I basic helix-loop-helix (bHLH) transcription factor 4 (TCF4; also known as E2-2) is a critical NB dependency gene that significantly contributes to these identity states through heterodimerization with cell-identity-specific bHLH transcription factors. Knockdown of TCF4 significantly induces apoptosis in vitro and inhibits tumorigenicity in vivo. We used genome-wide expression profiling, TCF4 chromatin immunoprecipitation sequencing (ChIP-seq) and TCF4 immunoprecipitation-mass spectrometry to determine the role of TCF4 in NB cells. Our results, along with recent findings in NB for the transcription factors T-box transcription factor TBX2, heart- and neural crest derivatives-expressed protein 2 (HAND2) and twist-related protein 1 (TWIST1), propose a role for TCF4 in regulating forkhead box protein M1 (FOXM1)/transcription factor E2F-driven gene regulatory networks that control cell cycle progression in cooperation with N-myc proto-oncogene protein (MYCN), TBX2, and the TCF4 dimerization partners HAND2 and TWIST1. Collectively, we showed that TCF4 promotes cell proliferation through direct transcriptional regulation of the c-MYC/MYCN oncogenic program that drives high-risk NB. Mechanistically, our data suggest the novel finding that TCF4 acts to support MYC activity by recruiting multiple factors known to regulate MYC function to sites of colocalization between critical NB transcription factors, TCF4 and MYC oncoproteins. Many of the TCF4-recruited factors are druggable, giving insight into potential therapies for high-risk NB. This study identifies a new function for class I bHLH transcription factors (e.g., TCF3, TCF4, and TCF12) that are important in cancer and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1fec99d385bab8935fc4d136d69d7a5564c71fa0" target='_blank'>
              Transcription factor 4 is a key mediator of oncogenesis in neuroblastoma by promoting MYC activity.
              </a>
            </td>
          <td>
            Nour Aljouda, Dewan Shrestha, Chelsea DeVaux, R. Olsen, Satyanarayana Alleboina, Megan Walker, Yong Cheng, Kevin W Freeman
          </td>
          <td>2024-08-09</td>
          <td>Molecular oncology</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Abstract Chimeric RNAs, distinct from DNA gene fusions, have emerged as promising therapeutic targets with diverse functions in cancer treatment. However, the functional significance and therapeutic potential of most chimeric RNAs remain unclear. Here we identify a novel fusion transcript of solute carrier family 2‐member 11 (SLC2A11) and macrophage migration inhibitory factor (MIF). In this study, we investigated the upregulation of SLC2A11–MIF in The Cancer Genome Atlas cohort and a cohort of patients from Sun Yat‐Sen Memorial Hospital. Subsequently, functional investigations demonstrated that SLC2A11–MIF enhanced the proliferation, antiapoptotic effects, and metastasis of bladder cancer cells in vitro and in vivo. Mechanistically, the fusion protein encoded by SLC2A11–MIF interacted with polypyrimidine tract binding protein 1 (PTBP1) and regulated the mRNA half‐lives of Polo Like Kinase 1, Roundabout guidance receptor 1, and phosphoinositide‐3‐kinase regulatory subunit 3 in BCa cells. Moreover, PTBP1 knockdown abolished the enhanced impact of SLC2A11–MIF on biological function and mRNA stability. Furthermore, the expression of SLC2A11–MIF mRNA is regulated by CCCTC‐binding factor and stabilized through RNA N4‐acetylcytidine modification facilitated by N‐acetyltransferase 10. Overall, our findings revealed a significant fusion protein orchestrated by the SLC2A11–MIF–PTBP1 axis that governs mRNA stability during the multistep progression of bladder cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/024d970fb32d3f01ae87aaf1382c1c0690e3d1f6" target='_blank'>
              Oncogenic SLC2A11–MIF fusion protein interacts with polypyrimidine tract binding protein 1 to facilitate bladder cancer proliferation and metastasis by regulating mRNA stability
              </a>
            </td>
          <td>
            Liang Cheng, Chenwei Yang, Junlin Lu, Ming Huang, Ruihui Xie, Sarah Lynch, Justin Elfman, Yuhang Huang, Sen Liu, Siting Chen, Baoqing He, Tianxin Lin, Hui Li, Xu Chen, Jian Huang
          </td>
          <td>2024-08-14</td>
          <td>MedComm</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Studies of bacterial adaptation and evolution are hampered by the difficulty of measuring traits such as virulence, drug resistance, and transmissibility in large populations. In contrast, it is now feasible to obtain high-quality complete assemblies of many bacterial genomes thanks to scalable high-accuracy long-read sequencing technologies. To exploit this opportunity, we introduce a phenotype- and alignment-free method for discovering coselected and epistatically interacting genomic variation from genome assemblies covering both core and accessory parts of genomes. Our approach uses a compact colored de Bruijn graph to approximate the intragenome distances between pairs of loci for a collection of bacterial genomes to account for the impacts of linkage disequilibrium (LD). We demonstrate the versatility of our approach to efficiently identify associations between loci linked with drug resistance and adaptation to the hospital niche in the major human bacterial pathogens Streptococcus pneumoniae and Enterococcus faecalis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c9ba999d01a93235712556c85e81e52a6ac50e10" target='_blank'>
              Pangenome-spanning epistasis and coselection analysis via de Bruijn graphs.
              </a>
            </td>
          <td>
            Juri Kuronen, Samuel T. Horsfield, A. K. Pöntinen, S. Mallawaarachchi, S. Arredondo-Alonso, Harry A. Thorpe, R. Gladstone, Rob J. L. Willems, Stephen D. Bentley, N. Croucher, J. Pensar, John A. Lees, G. Tonkin-Hill, Jukka Corander
          </td>
          <td>2024-08-12</td>
          <td>Genome research</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Most DNA double-strand breaks (DSBs) are harmful to genome integrity. However, some forms of DSBs are essential to biological processes, such as meiotic recombination and V(D)J recombination. DSBs are also required for programmed DNA elimination (PDE) in ciliates and nematodes. In nematodes, the DSBs are healed with telomere addition. While telomere addition sites have been well characterized, little is known regarding the DSBs that fragment nematode chromosomes. Here, we used embryos from the human and pig parasitic nematode Ascaris to characterize the DSBs. Using END-seq, we demonstrate that DSBs are introduced before mitosis, followed by extensive end resection. The resection profile is unique for each break site, and the resection generates 3'-overhangs before the addition of neotelomeres. Interestingly, telomere healing occurs much more frequently on retained DSB ends than on eliminated ends. This biased repair of the DSB ends may be due to the sequestration of the eliminated DNA into micronuclei, preventing neotelomere formation at their ends. Additional DNA breaks occur within the eliminated DNA in both Ascaris and Parascaris, ensuring chromosomal breakage and providing a fail-safe mechanism for PDE. Overall, our data indicate that telomere healing of DSBs is specific to the break sites responsible for nematode PDE.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/104afc2d31453d804bc1298637d05cb0465a32ee" target='_blank'>
              End resection and telomere healing of DNA double-strand breaks during nematode programmed DNA elimination.
              </a>
            </td>
          <td>
            Brandon Estrem, Richard E Davis, Jianbin Wang
          </td>
          <td>2024-07-02</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Homologous recombination rearranges genetic information during meiosis to generate new combinations of variants. Recombination also causes new mutations, affects the GC content of the genome and reduces selective interference. Here, we use HiFi long-read sequencing to directly detect crossover and gene conversion events from switches between the two haplotypes along single HiFi-reads from testis tissue of humans, chimpanzees and gorillas as well as human sperm samples. Furthermore, based on DNA methylation calls, we classify the cellular origin of reads to either somatic or germline cells in the testis tissue. We identify 1692 crossovers and 1032 gene conversions in nine samples and investigate their chromosomal distribution. Crossovers are more telomeric and correlate better with recombination maps than gene conversions. We show a strong concordance between a human double-strand break map and the human samples, but not for the other species, supporting different PRDM9-programmed double-strand break loci. We estimate the average gene conversion tract lengths to be similar and very short in all three species (means 40-100 bp, fitted well by a geometric distribution) and that 95-98% of non-crossover events do not involve tracts intersecting with polymorphism and are therefore not detectable. Finally, we detect a GC bias in the gene conversion of both single and multiple SNVs and show that the GC-biased gene conversion affects SNVs flanking crossover events. This implies that gene conversion events associated with crossover events are much longer (estimated above 500 bp) than those associated with non-crossover events. Highly accurate long-read sequencing combined with the classification of reads to specific cell types provides a new, powerful way to make individual, detailed maps of gene conversion and crossovers for any species.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5e343940af051e78f3a88674b4f23d5e3391f5fb" target='_blank'>
              Insights into gene conversion and crossing-over processes from long-read sequencing of human, chimpanzee and gorilla testes and sperm
              </a>
            </td>
          <td>
            Peter Soerud Porsborg, A. P. Charmouh, Vinod Kumar Singh, S. B. Winge, C. Hvilsom, Marta Pelizzola, S. Laurentino, N. Neuhaus, A. Hobolth, Thomas Bataillon, Kristian Almstrup, Søren Besenbacher, M. Schierup
          </td>
          <td>2024-07-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/eb8e9f04f74a6159b9e49f1e14a0705e366ea67c" target='_blank'>
              Epigenetic alterations affecting hematopoietic regulatory networks as drivers of mixed myeloid/lymphoid leukemia
              </a>
            </td>
          <td>
            Roger Mulet-Lazaro, Stanley van Herk, Margit Nuetzel, Aniko Sijs-Szabo, Noelia Díaz, Katherine Kelly, Claudia A. J. Erpelinck-Verschueren, Lucia Schwarzfischer-Pfeilschifter, Hanna Stanewsky, Ute Ackermann, Dagmar Glatz, Johanna Raithel, A. Fischer, Sandra Pohl, Anita Rijneveld, Juan M. Vaquerizas, Christian Thiede, C. Plass, Bas J. Wouters, Ruud Delwel, Michael Rehli, C. Gebhard
          </td>
          <td>2024-07-07</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="With age, hematopoietic stem cells can acquire somatic mutations in leukemogenic genes that confer a proliferative advantage in a phenomenon termed “clonal hematopoiesis of indeterminate potential” (CHIP). How these mutations confer a proliferative advantage and result in increased risk for numerous age-related diseases remains poorly understood. We conducted a multiracial meta-analysis of epigenome-wide association studies (EWAS) of CHIP and its subtypes in four cohorts (N=8196) to elucidate the molecular mechanisms underlying CHIP and illuminate how these changes influence cardiovascular disease risk. The EWAS findings were functionally validated using human hematopoietic stem cell (HSC) models of CHIP. A total of 9615 CpGs were associated with any CHIP, 5990 with DNMT3A CHIP, 5633 with TET2 CHIP, and 6078 with ASXL1 CHIP (P <1×10−7). CpGs associated with CHIP subtypes overlapped moderately, and the genome-wide DNA methylation directions of effect were opposite for TET2 and DNMT3A CHIP, consistent with their opposing effects on global DNA methylation. There was high directional concordance between the CpGs shared from the meta-EWAS and human edited CHIP HSCs. Expression quantitative trait methylation analysis further identified transcriptomic changes associated with CHIP-associated CpGs. Causal inference analyses revealed 261 CHIP-associated CpGs associated with cardiovascular traits and all-cause mortality (FDR adjusted p-value <0.05). Taken together, our study sheds light on the epigenetic changes impacted by CHIP and their associations with age-related disease outcomes. The novel genes and pathways linked to the epigenetic features of CHIP may serve as therapeutic targets for preventing or treating CHIP-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bb071f9470db1b0d43e08fc0664a4d81582bdd97" target='_blank'>
              Epigenome-wide DNA Methylation Association Study of CHIP Provides Insight into Perturbed Gene Regulation
              </a>
            </td>
          <td>
            Daniel Levy, Sara Kirmani, T. Huan, Joseph Van Amburg, R. Joehanes, M. Uddin, N. Nguyen, Bing Yu, Jennifer A. Brody, M. Fornage, Jan Bressler, N. Sotoodehnia, David Ong, Fabio Puddu, James Floyd, Christie M. Ballantyne, B. Psaty, L. Raffield, Pradeep Natarajan, Karen Conneely, April P. Carson, Leslie Lange, Kendra Ferrier, N. HEARD-COSTA, Joanne M Murabito, A. Bick
          </td>
          <td>2024-07-16</td>
          <td>Research Square</td>
          <td>0</td>
          <td>112</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6308353370607972c108245e832838518d5ac763" target='_blank'>
              KPNA2 promotes the progression of gastric cancer by regulating the alternative splicing of related genes
              </a>
            </td>
          <td>
            Xia Chen, Hui Wei, Ailin Yue, Huiyun Zhang, Ya Zheng, Weiming Sun, Yongning Zhou, Yuping Wang
          </td>
          <td>2024-07-25</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="DNA demethylases TET2 and TET3 play a fundamental role in thymic invariant natural killer T (iNKT) cell differentiation by mediating DNA demethylation of genes encoding for lineage specifying factors. Paradoxically, differential gene expression analysis revealed that significant number of genes were upregulated upon TET2 and TET3 loss in iNKT cells. This unexpected finding could be potentially explained if loss of TET proteins was reducing the expression of proteins that suppress gene expression. In this study, we discover that TET2 and TET3 synergistically regulate Drosha expression, by generating 5hmC across the gene body and by impacting chromatin accessibility. As DROSHA is involved in microRNA biogenesis, we proceed to investigate the impact of TET2/3 loss on microRNAs in iNKT cells. We report that among the downregulated microRNAs are members of the Let-7 family that downregulate in vivo the expression of the iNKT cell lineage specifying factor PLZF. Our data link TET proteins with microRNA expression and reveal an additional layer of TET mediated regulation of gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ddc6479019a43ee43ca5c9384965e96b131f891" target='_blank'>
              TET proteins regulate Drosha expression and impact microRNAs in iNKT cells
              </a>
            </td>
          <td>
            Marianthi Gioulbasani, Tarmo Äijö, Jair E. Valenzuela, Julia Buquera Bettes, Ageliki Tsagaratou
          </td>
          <td>2024-07-31</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c4935f35210bf40b045d154a339912e6ddb86073" target='_blank'>
              Decoding high mobility group A2 protein expression regulation and implications in human cancers
              </a>
            </td>
          <td>
            Farah Khazem, A. B. Zetoune
          </td>
          <td>2024-07-31</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aba56364331e9a8e54090b261a21c198f4de3054" target='_blank'>
              ARAP1-AS1: a novel long non-coding RNA with a vital regulatory role in human cancer development
              </a>
            </td>
          <td>
            Jialing Wang, Hongliang Luo, Lu Yang, Huazhao Yuan
          </td>
          <td>2024-08-01</td>
          <td>Cancer Cell International</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="The Ten-Eleven Translocation (TET) family genes are implicated in a wide array of biological functions across various human cancers. Nonetheless, there is a scarcity of studies that comprehensively analyze the correlation between TET family members and the molecular phenotypes and clinical characteristics of different cancers. Leveraging updated public databases and employing several bioinformatics analysis methods, we assessed the expression levels, somatic variations, methylation levels, and prognostic values of TET family genes. Additionally, we explored the association between the expression of TET family genes and pathway activity, tumor microenvironment (TME), stemness score, immune subtype, clinical staging, and drug sensitivity in pan-cancer. Molecular biology and cytology experiments were conducted to validate the potential role of TET3 in tumor progression. Each TET family gene displayed distinct expression patterns across at least ten detected tumors. The frequency of Single Nucleotide Variant (SNV) in TET genes was found to be 91.24%, primarily comprising missense mutation types, with the main types of copy number variant (CNV) being heterozygous amplifications and deletions. TET1 gene exhibited high methylation levels, whereas TET2 and TET3 genes displayed hypomethylation in most cancers, which correlated closely with patient prognosis. Pathway activity analysis revealed the involvement of TET family genes in multiple signaling pathways, including cell cycle, apoptosis, DNA damage response, hormone AR, PI3K/AKT, and RTK. Furthermore, the expression levels of TET family genes were shown to impact the clinical staging of tumor patients, modulate the sensitivity of chemotherapy drugs, and thereby influence patient prognosis by participating in the regulation of the tumor microenvironment, cellular stemness potential, and immune subtype. Notably, TET3 was identified to promote cancer progression across various tumors, and its silencing was found to inhibit tumor malignancy and enhance chemotherapy sensitivity. These findings shed light on the role of TET family genes in cancer progression and offer insights for further research on TET3 as a potential therapeutic target for pan-cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db4df11d8b0544aeb2264d4da3fba3e8f78e1111" target='_blank'>
              Pan-cancer analyses reveal the molecular and clinical characteristics of TET family members and suggests that TET3 maybe a potential therapeutic target
              </a>
            </td>
          <td>
            Chunyan Zhang, Jie Zheng, Jin Liu, Yanxia Li, G. Xing, Shupeng Zhang, Hekai Chen, Jian Wang, Z. Shao, Yongyuan Li, Zhongmin Jiang, Yingzi Pan, Xiaozhi Liu, Ping Xu, Wenhan Wu
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Pharmacology</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a3d53c60ef5486ebcbf381bfd90970c04135b42" target='_blank'>
              Low-input PacBio sequencing generates high-quality individual fly genomes and characterizes mutational processes
              </a>
            </td>
          <td>
            Hangxing Jia, Shengjun Tan, Yingao Cai, Ya-Dong Guo, Jieyu Shen, Yaqiong Zhang, Huijing Ma, Qingzhu Zhang, Jinfeng Chen, Gexia Qiao, Jue Ruan, Yong E Zhang
          </td>
          <td>2024-07-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4597b2c9168e5b3eb619c4010a4d29dad0a9c28b" target='_blank'>
              mTORC2-driven chromatin cGAS mediates chemoresistance through epigenetic reprogramming in colorectal cancer.
              </a>
            </td>
          <td>
            Guoqing Lv, Qian Wang, Lin Lin, Qiao Ye, Xi Li, Qian Zhou, Xiangzhen Kong, Hongxia Deng, Fuping You, Hebing Chen, Song Wu, Lin Yuan
          </td>
          <td>2024-07-30</td>
          <td>Nature cell biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Genome merging is a common phenomenon in many organisms, causing a wide range of consequences on phenotype, adaptation, and gene expression, among other effects, yet its broader implications are not well understood. Two consequences of genome merging on gene expression remain poorly understood: dosage effects and evolution of expression. In this study, we employed Chlamydomonas reinhardtii as a model to investigate the effects of asymmetric genome merging by crossing a diploid with a haploid strain to create a novel triploid line. Five independent clonal lineages derived from this triploid line were evolved for 425 asexual generations in a laboratory natural selection (LNS) experiment. Utilizing fitness assays, qPCR, and RNA-Seq, we assessed the immediate consequences of genome merging and subsequent evolution over time. Our findings reveal substantial alterations in gene expression, protein homeostasis (proteostasis) and cytonuclear stoichiometry. Notably, gene expression exhibited expression level dominance and transgressivity (i.e., expression level higher or lower than either parent). Ongoing expression level dominance and a pattern of “functional dominance” from the haploid parent was observed, alongside remarkable stability in expression patterns across generations. Despite major nucleo-cytoplasmic disruptions, enhanced fitness was detected in the triploid strain. By comparing gene expression across generations, our results indicate that proteostasis restoration is a critical component of rapid adaptation following genome merging in Chlamydomonas reinhardtii and possibly other systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a15e55d2c045a09190880662b2365054897839f5" target='_blank'>
              Asymmetric genome merging leads to gene expression novelty through nucleo-cytoplasmic disruptions and transcriptomic shock in Chlamydomonas triploids
              </a>
            </td>
          <td>
            Lucas Prost-Boxoen, Quinten Bafort, Antoine Van de Vloet, Fabrício Almeida-Silva, Yunn Thet Paing, G. Casteleyn, S. D’hondt, O. De Clerck, Yves Van de Peer
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Elucidating the genetic architecture of DNA methylation (DNAm) is crucial for decoding the etiology of complex diseases. However, current epigenomic studies often suffer from incomplete coverage of methylation sites and the use of tissues containing heterogeneous cell populations. To address these challenges, we present a comprehensive human methylome atlas based on deep whole-genome bisulfite sequencing (WGBS) and whole-genome sequencing (WGS) of purified monocytes from 298 European Americans (EA) and 160 African Americans (AA) in the Louisiana Osteoporosis Study. Our atlas enables the analysis of over 25 million DNAm sites. We identified 1,383,250 and 1,721,167 methylation quantitative trait loci (meQTLs) in cis-regions for EA and AA populations, respectively, with 880,108 sites shared between ancestries. While cis-meQTLs exhibited population-specific patterns, primarily due to differences in minor allele frequencies, shared cis-meQTLs showed high concordance across ancestries. Notably, cis-heritability estimates revealed significantly higher mean values in the AA population (0.09) compared to the EA population (0.04). Furthermore, we developed population-specific DNAm imputation models using Elastic Net, enabling methylome-wide association studies (MWAS) for 1,976,046 and 2,657,581 methylation sites in EA and AA, respectively. The performance of our MWAS models was validated through a systematic multi-ancestry analysis of 41 complex traits from the Million Veteran Program. Our findings bridge the gap between genomics and the monocyte methylome, uncovering novel methylation-phenotype associations and their transferability across diverse ancestries. The identified meQTLs, MWAS models, and data resources are freely available at www.gcbhub.org.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02561d4c021777b32cc89ad698157b2f333e71dc" target='_blank'>
              An atlas of genetic effects on the monocyte methylome across European and African populations
              </a>
            </td>
          <td>
            W. Zhang, X. Zhang, C. Qiu, Z. Zhang, K.-J. Su, Z. Luo, Q. Tian, M. Liu, B. Zhao, L. Wu, H. Shen, C. Wu, H.-W. Deng
          </td>
          <td>2024-08-13</td>
          <td>None</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06cfb136406d6e0103812e5102a4ed1670edf2da" target='_blank'>
              METTL3-mediated m6A modification of circGLIS3 promotes prostate cancer progression and represents a potential target for ARSI therapy
              </a>
            </td>
          <td>
            Xiaofeng Cheng, Heng Yang, Yujun Chen, Zhenhao Zeng, Yifu Liu, Xiaochen Zhou, Cheng Zhang, An Xie, Gongxian Wang
          </td>
          <td>2024-08-14</td>
          <td>Cellular & Molecular Biology Letters</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="


 What’s the pattern of genome-wide chromatin occupancy of SCP3 during meiotic prophase, and how its interaction with chromatin is regulated?



 We document SCP3 enriched at active chromatin regions with largely inherited from leptotene to pachytene stage, which facilitated by transcription and condensate assembly in spermatocytes.



 Meiosis is a critical step during the process of gametogenesis to form haploid cells which finally develop into functional gametes. The synaptonemal complex (SC) is a meiosis-specific protein assembly which bridges homologous chromosomes to establish the architecture of meiotic chromosome. Genetic ablation of genes encoding SC components impedes meiosis and gametogenesis in mice and mutation of these genes is frequently found in infertile patients. SCP3 localization changes with chromosome morphology in autosomes and is involved in maintaining the association of sex chromosomes. Male mice with null mutation of SCP3 failed to form SCs, resulting in developmental arrest during meiotic prophase.



 This study included male mice at post-natal day 10 (P10), 4 weeks or adult (8 weeks or older) were maintained on C57BL/6J background. Spermatocytes were performed modified CUT&Tag using antibodies. ChIP-seq data was analyzed together with WGBS and ATAC-seq data.



 Spermatocytes were obtained by a discontinuous BSA density gradient. Cell pellet was resuspended in culture medium (MEM with 10% FBS) at a concentration of 105 cells/ml and cultured at 37 °C in a 5% CO2 incubator. Spermatocytes were cultured with or without 1,6-hexanediol or α-amanitin. Spermatocytes with or without drug treatment were collected at indicated time points for further examination and then modified CUT&Tag and co-IP using anti-SCP3 and anti-POL II antibodies.



 Chromatin occupancy of SCP3 in leptotene and pachytene spermatocytes, SCP3-Core sites retained upon treatment of 1,6-hexanediol which dissolves SCP3 condensate, and SCP1 occupancy at meiotic chromatin in pachytene spermatocytes. When we determined inter-peak distances and width of peaks, we observe that: (1) SCP3 occupancy in leptotene stage had greater inter-peak distances and narrower peak width than SCP3 occupancy in pachytene stage, demonstrating gradual accumulated SCP3 at chromatin during meiotic progression; (2) SCP3-Core sites have broader peak width than SCP3-NonCore sites, in agreement with stability of SCP3 occupancy at SCP3-Core sites; (3) SCP1 occupancy had greater inter-peak distances and narrower peak width than SCP3 occupancy in pachytene stage, supporting that SCP1 peaks are a subset of SCP3 peaks and SCP1’s major function in connecting LE and CE respectively. Generally, at pachytene-stage meiosis, SCP3 mainly occupies chromatin regions including transcriptionally active sites, Deu and Alu elements, while SCP1 mainly co-localizes with SCP3 at a subset of regions with open chromatin and high cohesin enrichment to connect meiotic chromosome axes with CE. In this way, a local active chromatin environment is created at meiotic chromosome axes, facilitating homologous chromosome recombination.



 Our test was not successful to examine SCP3 occupancy with as few as less than 1000 spermatocytes using our protocol. It is improvement of efficiency of tagmentation or development of more efficient antibodies against SCP3 will be helpful for clinical sample detection.



 SCP3 occupancy was largely inherited from leptotene to pachytene stage, facilitated by transcription and condensate assembly, and enriched at specific SINE repeats. Further exploration using knockout mouse models of meiotic axial proteins and epigenetic modifiers will provide valuable information on meiotic chromatin architecture and crosstalk among meiotic chromatin events.



 N/A
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbd5370d474a768836582b1cb861c8c3d70c98bd" target='_blank'>
              P-045 Deciphering meiotic chromatin organization by SCP3
              </a>
            </td>
          <td>
            S. Guo
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1f187326818c3bc4b9634f53afc541e906548d2" target='_blank'>
              Targeting PRMT5 enhances the radiosensitivity of tumor cells grown in vitro and in vivo
              </a>
            </td>
          <td>
            Charlotte M. Degorre, Steven Lohard, Christina N. Bobrek, Komal N. Rawal, Skyler Kuhn, P. Tofilon
          </td>
          <td>2024-07-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>53</td>
        </tr>

        <tr id="Satellite DNAs (satDNAs) are tandemly repeated sequences that make up a significant portion of almost all eukaryotic genomes. Although satDNAs have been shown to play a very important role in genome organization and evolution, they are relatively poorly analysed even in model. One of the main reasons for the current lack of in-depth studies on satDNAs is their underrepresentation in genome assemblies. The complexity and highly repetitive nature of satDNAs make their analysis challenging, and there is a need for efficient tools that can ensure accurate annotation and analysis of satDNAs. We present a novel pipeline, named Satellite DNA Exploration (SatXplor), designed to robustly characterize satDNA elements and analyse their arrays and flanking regions. SatXplor is benchmarked against curated satDNA datasets from diverse species, showcasing its versatility across genomes with varying complexities and different satDNA profile. Component algorithms excel in the identification of tandemly repeated sequences and for the first time enable evaluation of satDNA variation and array annotation with the addition of information about surrounding genomic landscape. SatXplor is an innovative pipeline for satDNA analysis that can be paired with any tool used for satDNA detection, offering insights into the structural characteristics, array determination and genomic context of satDNA elements. By integrating various computational techniques, from sequence analysis and homology investigation to advanced clustering and graph-based methods, it provides a versatile and comprehensive approach to explore the complexity of satDNA organization and to understand the underlying mechanisms and evolutionary aspects. It is open-source and freely accessible at https://github.com/mvolar/SatXplor.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db93bcbb026ce597eb6926d4d48baa8708101680" target='_blank'>
              SatXplor – A comprehensive pipeline for satellite DNA analyses in complex genome assemblies
              </a>
            </td>
          <td>
            Marin Volarić, N. Meštrović, Evelin Despot-Slade
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f98b7ad020e16cb10102ff0aa9974285386fae8" target='_blank'>
              Spatial transcriptome and single-cell reveal the role of nucleotide metabolism in colorectal cancer progression and tumor microenvironment
              </a>
            </td>
          <td>
            Junzhi Liu, Huimin Li, Lantian Wang, Shurui Wang, Qiang Tang
          </td>
          <td>2024-07-29</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Detecting somatic mutations in normal cells and tissues is notoriously challenging due to their low abundance, orders of magnitude below the sequencing error rate. While several techniques, such as single-cell and single-molecule sequencing, have been developed to identify somatic mutations, they are insufficient for detecting genomic structural variants (SVs), which have a significantly greater impact than single-nucleotide variants (SNVs). We introduce Single-Molecule Mutation Sequencing for Structural Variants (SMM-SV-seq), a novel method combining Tn5-mediated, chimera-free library preparation with the precision of error-corrected next-generation sequencing (ecNGS). This approach enhances SV detection accuracy without relying on independent supporting sequencing reads. Our validation studies on human primary fibroblasts treated with varying concentrations of the clastogen bleomycin demonstrated a significant, up to tenfold and dose-dependent, increase in deletions and translocations 24 hours post-treatment. Evaluating SMM-SV-seq’s performance against established computational tools for SV detection, such as Manta and DELLY, using a well-characterized human cell line, SMM-SV-seq showed precision and recall rates of 61.9% and 85.8%, respectively, significantly outperforming Manta (10% precision, 23% recall) and DELLY (15% precision, 32% recall). Using SMM-SV-seq, we documented clear, direct evidence of negative selection against structural variants over time. After a single 2 Gy dose of ionizing radiation, SVs in normal human primary fibroblasts peaked at 24 hours post-intervention and then declined to nearly background levels by day six, highlighting the cellular mechanisms that selectively disadvantage cells harboring these mutations. Additionally, SMM-SV-seq revealed that BRCA1-deficient human breast epithelial cells are more susceptible to the mutagenic effects of ionizing radiation compared to BRCA1-proficient isogenic control cells, suggesting a potential molecular mechanism for increased breast cancer risk in BRCA1 mutation carriers. SMM-SV-seq represents a significant advancement in genomic analysis, enabling the accurate detection of somatic structural variants in normal cells and tissues for the first time. This method complements our previously published Single-Molecule Mutation sequencing (SMM-seq), effective for detecting single-nucleotide variants (SNVs) and small insertions and deletions (INDELs). By addressing challenges such as self-ligation in library preparation and leveraging a powerful ecNGS strategy, SMM-SV-seq enhances the robustness of our genomic analysis toolkit. This breakthrough paves the way for new research into genetic variability and mutation processes, offering deeper insights that could advance our understanding of aging, cancer, and other human diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a54aec0db6bdf317c2066ae223030ee9204ff3b0" target='_blank'>
              Detection of genome structural variation in normal cells and tissues by single molecule sequencing
              </a>
            </td>
          <td>
            Johanna Heid, Zhenqiu Huang, Moonsook Lee, Sergey Makhortov, Elizabeth Pan, Cristina Montagna, Shixiang Sun, Jan Vijg, A. Maslov
          </td>
          <td>2024-08-08</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="The growing accessibility of sequencing experiments has significantly accelerated the development of personalized immunotherapies based on the identification of cancer neoantigens. Still, the prediction of neoantigens involves lengthy and inefficient protocols, requiring simultaneous analysis of sequencing data from paired tumor/normal exomes and tumor transcriptome, often resulting in a low success rate. To date, the feasibility of adopting a more efficient strategy has not been fully evaluated. To this end, we developed ENEO, a computational approach to detect cancer neoantigens using solely the tumor RNA-seq data while addressing the lack of matched control through a Bayesian probabilistic model. ENEO was assessed on TESLA benchmark dataset, reporting efficient identification of DNA-alterations derived neoantigens and compelling results against state-of-art exome-based methods. We further validated the method on two independent cohorts, encompassing different tumor types and experimental procedures. Our work demonstrates that a tumor-only RNA-based approach, such as the one implemented in ENEO, maintains accuracy in identifying mutated peptides resulting from expressed genomic alterations, while also broadening the pool of potential pMHCs with RNAspecific mutations in a faster and cost-effective way. ENEO is freely available at https://github.com/ctglab/ENEO">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/09a616187ebc5880f69d8900824e6ea2e18ae9d7" target='_blank'>
              Efficient and effective identification of cancer neoantigens from tumor only RNA-seq
              </a>
            </td>
          <td>
            Danilo Tatoni, Mattia Dalsass, Giulia Brunelli, Guido Grandi, Mario Chiariello, Romina D’Aurizio
          </td>
          <td>2024-08-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4c87cc397593943df3c9c096299828a9dcf972f" target='_blank'>
              Obesity increases genomic instability at DNA repeat-mediated endogenous mutation hotspots
              </a>
            </td>
          <td>
            Pallavi Kompella, Guliang Wang, Russell E Durrett, Yanhao Lai, Celeste Marin, Yuan Liu, Samy L Habib, John DiGiovanni, Karen M. Vasquez
          </td>
          <td>2024-07-23</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/14e3eb9b781fd010e03617fa1a78add9e2d4a45e" target='_blank'>
              cBAF generates subnucleosomes that expand OCT4 binding and function beyond DNA motifs at enhancers.
              </a>
            </td>
          <td>
            M. C. Nocente, Anida Mesihovic Karamitsos, Emilie Drouineau, Manon Soleil, Waad Albawardi, C. Dulary, F. Ribiérre, H. Picaud, Olivier Alibert, Joël Acker, Marie Kervella, J. Aude, N. Gilbert, Françoise Ochsenbein, Sophie Chantalat, Matthieu Gérard
          </td>
          <td>2024-07-02</td>
          <td>Nature structural & molecular biology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Background Clear cell renal cell carcinoma (ccRCC) stands as one of the most prevalent and aggressive forms of renal malignancies, accounting for a substantial portion of kidney cancer-related mortality worldwide. The heterogeneity within ccRCC poses a significant challenge to understanding its molecular mechanisms and devising effective therapeutic strategies. [1] Previous studies utilizing single nucleus RNA sequencing (snRNA-seq) have revealed transcriptional contributions to cell-type specificity in both mature human and mouse kidneys. Moreover, recent advancements have expanded this approach to single-cell profiling of chromatin accessibility. Single-nuclei assay for transposase-accessible chromatin using sequencing (snATAC-seq) has enabled the measurement of chromatin accessibility in thousands of individual cells, providing insights into the dynamic process that drives nephron development and differentiation. Integration and analysis of multimodal single-cell datasets, such as snRNA-seq and snATAC-seq, allows for the prediction of cell-type-specific cis-regulatory DNA interactions and transcription factor activity, complementing the transcriptional information obtained by snRNA-seq.[2] [3] [1] Yang, Wang, and Yang, “Treatment Strategies for Clear Cell Renal Cell Carcinoma.” [2] Muto et al., “Single Cell Transcriptional and Chromatin Accessibility Profiling Redefine Cellular Heterogeneity in the Adult Human Kidney.” [3] Monteagudo-Sánchez, Noordermeer, and Greenberg, “The Impact of DNA Methylation on CTCF-Mediated 3D Genome Organization.” Methods This study leveraged paired snRNA/ATAC-seq in tumor and normal-adjacent samples from the Renal Tumor Biobank at Dartmouth (n= 63) to explore the ccRCC landscape at single-cell resolution using 10x multiome technology. The snRNA-seq dataset was processed with Seurat v5.0 to remove low-quality nuclei. Doublets were removed with DoubletFinder v2.02, and ambient DNA was removed using decontX from celda. Normalization was conducted via SCTransform, and missing data was imputed with ALRA. Clustering was performed by constructing a KNN graph and running the Louvain algorithm. Cell type assignments were predicted using Azimuth with the human kidney reference dataset. The snATAC-seq data was processed with ArchR v1.0.1, which performed dimensionality reduction and clustering. Cluster identities defined in the Seurat workflow were cross-annotated to the ATAT-seq data, and open peaks were identified within these cell-type clusters using MACS3 v3.0.1. Identification of marker peaks, motif enrichment, and motif enrichment deviations were inferred utilizing ArchR’s getmarkerFeatures, peakAnnoEnrichment, and addMotifAnnotations functions, respectively. Results Motif enrichment analysis revealed canonical cancer-associated transcription factor (TF) dysregulation in tumor cells (nephron loop-like), including enrichment of HNF1B, SMARCC1, FOSL1, and JUND. Interestingly, HNF and SMARCC were also enriched in the lymphoid compartment, indicating possible crosstalk between these two cell types. Previous work has investigated the involvement of the HNF1 in ccRCC and in chromophobe RCC for prognosis,and we plan on investigating the relationship of these enrichments to tumor grade, stage and outcome.[1] Additional TF enrichments included BCL families and ELF within the myeloid compartment, and vascularization related TF families in endothelial cell types including enrichment of various ETS TFs. Investigation of cell-cell interactions between immune and endothelial cell populations as well as between tumor and normal adjacent tissues will be performed to further disentangle the role differentially accessable chromatin plays in ccRCC. [1] Buchner et al., “Downregulation of HNF-1B in Renal Cell Carcinoma Is Associated With Tumor Progression and Poor Prognosis.” Conclusions Our multi-omic approach has several advantages. First, it enables the characterization of cellular heterogeneity within the tumor microenvironment, identifying cell populations and their contributions to disease progression. Secondly, by integrating transcriptomic and epigenomic data, we gain insights into the dynamic interplay between gene expression and chromatin accessibility, uncovering potential therapeutic vulnerabilities and biomarkers for personalized treatment strategies. DOD CDMRP Funding: yes">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e8e88d9a75048a69eb2afe5150d509757fff83b4" target='_blank'>
              60 Dissecting the Cellular and Epigenetic Landscape of Clear Cell Renal Cell Carcinoma Through Paired snRNA/ATAC-Seq Analysis
              </a>
            </td>
          <td>
            Ian Gingerich
          </td>
          <td>2024-08-05</td>
          <td>The Oncologist</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="CRNDE is an oncogene expressed as a long non-coding RNA. However, our team previously reported that the CRNDE gene also encodes a micropeptide, CRNDEP. The amino acid sequence of CRNDEP has recently been revealed by other researchers, too. This study aimed to investigate genetic alterations within the CRNDEP-coding region of the CRNDE gene, methylation profiling of this gene, and CRNDEP expression analysis. All investigations were performed on clinical material from patients with ovarian tumors of diverse aggressiveness. We found that CRNDEP levels were significantly elevated in highly aggressive tumors compared to benign neoplasms. Consistently, a high level of this micropeptide was a negative, independent, prognostic, and predictive factor in high-grade ovarian cancer (hgOvCa) patients. The cancer-promoting role of CRNDE(P), shown in our recent study, was also supported by genetic and epigenetic results obtained herein, revealing no CRNDEP-disrupting mutations in any clinical sample. Moreover, in borderline ovarian tumors (BOTS), but not in ovarian cancers, the presence of a single nucleotide polymorphism in CRNDE, rs115515594, significantly increased the risk of recurrence. Consistently, in BOTS only, the same genetic variant was highly overrepresented compared to healthy individuals. We also discovered that hypomethylation of CRNDE is associated with increased aggressiveness of ovarian tumors. Accordingly, hypomethylation of this gene’s promoter/first exon correlated with hgOvCa resistance to chemotherapy, but only in specimens with accumulation of the TP53 tumor suppressor protein. Taken together, these results contribute to a better understanding of the role of CRNDE(P) in tumorigenesis and potentially may lead to improvements in screening, diagnosis, and treatment of ovarian neoplasms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/41a5d81fb227910421858aeef5200b2eb49cb75b" target='_blank'>
              The Clinical Significance of CRNDE Gene Methylation, Polymorphisms, and CRNDEP Micropeptide Expression in Ovarian Tumors
              </a>
            </td>
          <td>
            L. A. Szafron, R. Iwanicka-Nowicka, A. Podgórska, Arkadiusz M Bonna, Piotr Sobiczewski, J. Kupryjańczyk, L. Szafron
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Abstract Summary Mitochondrial DNA sequences are used extensively in phylogeographic and phylogenetic studies for a wide range of organisms. With the advent of low-cost, high-throughput “next generation” DNA sequencing, and user-friendly bioinformatics pipelines for generating and annotating whole mitochondrial genome assemblies, the analysis of whole mitochondrial genomes has become an important component of phylogenomic studies for taxa with high species diversity but limited coverage for other genomic resources. An important step in characterizing de novo mitochondrial genome assemblies is to evaluate and describe structural rearrangements relative to reference taxa. Accessible tools are needed to help visualize gene and non-coding feature complement, their order, and strand orientation. However, there are few dedicated applications that generate high-quality genome diagrams. Here we present circularMT and circularMT-console that allow users to create highly customizable, publication-quality images, of linear and circular mitochondrial genome maps, either individually or integrated into an analysis pipeline. Availability and implementation Both applications are implemented in C#, with binaries, source code, and user guides available on GitHub (https://github.com/msjimc/circularMT). An archive of the published version is available on Zenodo (https://zenodo.org/records/10912319).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/45ad314515566c3f36d4dbf28b1d1c77340aa083" target='_blank'>
              Drawing mitochondrial genomes with circularMT
              </a>
            </td>
          <td>
            Simon J. Goodman, Ian M. Carr
          </td>
          <td>2024-07-01</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Despite the established role of DNA methylation in transcriptional silencing, most of the genes remain unchanged upon its global erasing from the genome, while expression of other genes is actually raised. Since its discovery a few decades ago, this unsolved enigma has hampered true understanding of DNA methylation role. By applying an updated mathematical model to the results of the original studies, we showed that this paradoxical effect is due to between-gene differences in the balance between methylation-guided enhancers and silencers, which protect particular disease-causing genes from the otherwise harsh effects of global methylation perturbation. Nevertheless, local methylation alternation may still affect these genes. The results a settle a long-standing wondering and suggest that DNA-methylation regulatory role is significantly greater than previously thought.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f326a9727e723fecbefe9526db8fc37ada53d66" target='_blank'>
              On the enigmatic null effect of global methylation perturbation
              </a>
            </td>
          <td>
            Y. Edrei, A. Hellman
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Radiation cytogenetics has a rich history seldom appreciated by those outside the field. Early radiobiology was dominated by physics and biophysical concepts that borrowed heavily from the study of radiation-induced chromosome aberrations. From such studies, quantitative relationships between biological effect and changes in absorbed dose, dose rate and ionization density were codified into key concepts of radiobiological theory that have persisted for nearly a century. This review aims to provide a historical perspective of some of these concepts, including evidence supporting the contention that chromosome aberrations underlie development of many, if not most, of the biological effects of concern for humans exposed to ionizing radiations including cancer induction, on the one hand, and tumor eradication on the other. The significance of discoveries originating from these studies has widened and extended far beyond their original scope. Chromosome structural rearrangements viewed in mitotic cells were first attributed to the production of breaks by the radiations during interphase, followed by the rejoining or mis-rejoining among ends of other nearby breaks. These relatively modest beginnings eventually led to the discovery and characterization of DNA repair of double-strand breaks by non-homologous end joining, whose importance to various biological processes is now widely appreciated. Two examples, among many, are V(D)J recombination and speciation. Rapid technological advancements in cytogenetics, the burgeoning fields of molecular radiobiology and third-generation sequencing served as a point of confluence between the old and new. As a result, the emergent field of "cytogenomics" now becomes uniquely positioned for the purpose of more fully understanding mechanisms underlying the biological effects of ionizing radiation exposure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c57db9d0b65c8dcb0db634d392394a245d1aa2a" target='_blank'>
              The Central Role of Cytogenetics in Radiation Biology.
              </a>
            </td>
          <td>
            Susan M. Bailey, Stephen R. Kunkel, Joel S. Bedford, Michael N. Cornforth
          </td>
          <td>2024-07-10</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Introduction Glioblastomas utilize malignant gene expression pathways to drive growth. Many of these gene pathways are not directly accessible with molecularly targeted pharmacological agents. Chromatin-modifying compounds can alter gene expression and target glioblastoma growth pathways. In this study, we utilize a systematic screen of chromatin-modifying compounds on a panel of patient-derived glioblastoma lines to identify promising compounds and their associated gene targets. Methods Five glioblastoma cell lines were subjected to a drug screen of 106 chromatin-modifying compounds representing 36 unique drug classes to determine the twelve most promising drug classes and the best candidate inhibitors in each class. These twelve drugs were then tested with a panel of twelve patient-derived gliomasphere lines to identify growth inhibition and corresponding gene expression patterns. Overlap analysis and weighted co-expression network analysis (WCGNA) were utilized to determine potential target genes and gene pathways. Results The initial drug screen identified twelve candidate pharmacologic agents for further testing. Drug sensitivity testing indicated an overall high degree of variability between gliomasphere lines. However, CPI203 was the most consistently effective compound, and the BET inhibitor class was the most consistently effective class of compounds across the gliomasphere panel. Correspondingly, most of the compounds tested had highly variable effects on gene expression between gliomasphere lines. CPI203 stood out as the only compound to induce a consistent effect on gene expression across different gliomasphere lines, specifically down-regulation of DNA-synthesis genes. Amongst the twelve tested cell lines, high expression of CDKN2A and CDKN2B distinguished more drug sensitive from more drug resistant lines. WCGNA identified two oncogenic gene modules (FBXO5 and MELK) that were effectively downregulated by CPI203 (FBXO5) and ML228 (FBXO5 and MELK). Conclusions The bromodomain inhibitor CPI203 induced relatively consistent effects on gene expression and growth across a variety of glioblastoma lines, specifically down-regulating genes associated with DNA replication. We propose that clinically effective BET inhibitors have the potential to induce consistent beneficial effects across a spectrum of glioblastomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1561fa27def3a17181508712df5ccd39c21a9f63" target='_blank'>
              CPI203, a BET inhibitor, down-regulates a consistent set of DNA synthesis genes across a wide array of glioblastoma lines
              </a>
            </td>
          <td>
            Matthew C. Garrett, Troy P Carnwath, Rebecca R Albano, Yonghua Zhuang, Catherine A Behrmann, Merissa Pemberton, Farah Barakat, Robert Lober, Mark Hoeprich, Anthony Paravati, Marilyn Reed, Hailey Spry, Daniel Woo, Eric O’Brien, Brett VanCauwenbergh, John Perentesis, Rani Nasser, Mario Medvedovic, D. Plas
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9d2a4f583e73c9388ee06b9414fc44f7c5cdf6bf" target='_blank'>
              Whole genome sequencing refines stratification and therapy of patients with clear cell renal cell carcinoma
              </a>
            </td>
          <td>
            R. Culliford, Sam Lawrence, Charlie Mills, Z. Tippu, D. Chubb, A. Cornish, Lisa Browning, B. Kinnersley, R. Bentham, Amit Sud, H. Pallikonda, A. Frangou, Andreas J. Gruber, K. Litchfield, David C Wedge, James Larkin, S. Turajlic, R. Houlston
          </td>
          <td>2024-07-15</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="In geminiviruses belonging to the genus Begomovirus, coat protein (CP) expression depends on viral AL2 protein, which derepresses and activates the CP promoter through sequence elements that lie within the viral intergenic region (IR). However, AL2 does not exhibit sequence-specific DNA binding activity but is instead directed to responsive promoters through interactions with host factors, most likely transcriptional activators and/or repressors. In this study, we describe a repressive plant-specific transcription factor, Arabidopsis thaliana TCP24 (AtTCP24), that interacts with AL2 and recognizes a class II TCP binding site in the CP promoter (GTGGTCCC). This motif corresponds to the previously identified conserved late element (CLE). We also report that histone 3 lysine 27 trimethylation (H3K27me3), an epigenetic mark associated with facultative repression, is enriched over the viral IR. H3K27me3 is deposited by Polycomb Repressive Complex 2 (PRC2), a critical regulator of gene expression and development in plants and animals. Remarkably, mutation of the TCP24 binding site (the CLE) in tomato golden mosaic virus (TGMV) and cabbage leaf curl virus (CaLCuV) CP promoters greatly diminishes H3K27me3 levels on viral chromatin and causes a dramatic delay and attenuation of disease symptoms in infected Arabidopsis and Nicotiana benthamiana plants. Symptom remission is accompanied by decreased viral DNA levels in systemically infected tissue. Nevertheless, in transient replication assays CLE mutation delays but does not limit the accumulation of viral double-stranded DNA, although single-stranded DNA and CP mRNA levels are decreased. These findings suggest that TCP24 binding to the CLE leads to CP promoter repression and H3K27me3 deposition, while TCP24-AL2 interaction may recruit AL2 to derepress and activate the promoter. Thus, a repressive host transcription factor may be repurposed to target a viral factor essential for promoter activity. The presence of the CLE in many begomoviruses suggests a common scheme for late promoter regulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57ddd53cac15774ed5f120c8687e7a74de7cf3b7" target='_blank'>
              Mutation of the conserved late element in geminivirus CP promoters abolishes Arabidopsis TCP24 transcription factor binding and decreases H3K27me3 levels on viral chromatin
              </a>
            </td>
          <td>
            Jacqueline Williams, Elizabeth Regedanz, Natalia Lucinda, Alba Ruth Nava Fereira, Gabriela Lacatus, Mary Berger, Nels O'Connell, Tami L. Coursey, Jianhua Ruan, D. Bisaro, G. Sunter
          </td>
          <td>2024-07-01</td>
          <td>PLOS Pathogens</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="Naïve pluripotent stem cells (nPSC) frequently undergo pathological and not readily reversible loss of DNA methylation marks at imprinted gene loci. This abnormality poses a hurdle for using pluripotent cell lines in biomedical applications and underscores the need to identify the causes of imprint instability in these cells. We show that nPSCs from inbred mouse strains exhibit pronounced strain-specific susceptibility to locus-specific deregulation of imprinting marks during reprogramming to pluripotency and upon culture with MAP kinase inhibitors, a common approach to maintain naïve pluripotency. Analysis of genetically highly diverse nPSCs from the Diversity Outbred (DO) stock confirms that genetic variation is a major determinant of epigenome stability in pluripotent cells. We leverage the variable DNA hypomethylation in DO lines to identify several trans-acting quantitative trait loci (QTLs) that determine epigenome stability at either specific target loci or genome-wide. Candidate factors encoded by two multi-target QTLs on chromosomes 4 and 17 suggest specific transcriptional regulators that contribute to DNA methylation maintenance in nPSCs. We propose that genetic variants represent candidate biomarkers to identify pluripotent cell lines with desirable properties and might serve as entry points for the targeted engineering of nPSCs with stable epigenomes. Highlights Naïve pluripotent stem cells from distinct inbred mouse strains exhibit variable DNA methylation levels at imprinted gene loci. The vulnerability of pluripotent stem cells to loss of genomic imprinting caused by MAP kinase inhibition strongly differs between inbred mouse strains. Genetically diverse pluripotent stem cell lines from Diversity Outbred mouse stock allow the identification of quantitative trait loci controlling DNA methylation stability. Genetic variants may serve as biomarkers to identify naïve pluripotent stem cell lines that are epigenetically stable in specific culture conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fcf4c9c2e254cef424889809e18c41add6362bb" target='_blank'>
              Genetic variation modulates susceptibility to aberrant DNA hypomethylation and imprint deregulation in naïve pluripotent stem cells
              </a>
            </td>
          <td>
            C. Parikh, RA Glenn, Y. Shi, K. Chatterjee, EE Swanzey, S. Singer, SC Do, Y. Zhan, Y. Furuta, M. Tahiliani, E. Apostolou, A. Polyzos, R. Koche, JG Mezey, T. Vierbuchen, M. Stadtfeld
          </td>
          <td>2024-06-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7688d245ed2ef09eafd9e57de454a0d8db59f71a" target='_blank'>
              Transposable elements contribute to tissue-specific gene regulation in humans.
              </a>
            </td>
          <td>
            Arsala Ali, Ping Liang
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Introduction Nucleoporin 98 (NUP98) fusion proteins are recurrently found in leukemia and are associated with unfavorable clinical outcomes. They are distributed to the nucleus and contribute to leukemogenesis via aberrant transcriptional regulation. We previously identified NUP98‐BPTF (NB) fusion in patients with T‐cell acute lymphoblastic leukemia (T‐ALL) using next‐generation sequencing. The FG‐repeat of NUP98 and the PHD finger and bromodomain of bromodomain PHD finger transcription factor (BPTF) are retained in the fusion. Like other NUP98 fusion proteins, NB is considered to regulate genes that are essential for leukemogenesis. However, its target genes or pathways remain unknown. Materials and Methods To investigate the potential oncogenic properties of the NB fusion protein, we lentivirally transduced a doxycycline‐inducible NB expression vector into mouse NIH3T3 fibroblasts and human Jurkat T‐ALL cells. Results NB promoted the transformation of mouse NIH3T3 fibroblasts by upregulating the proto‐oncogene Pim1, which encodes a serine/threonine kinase. NB transcriptionally regulated Pim1 expression by binding to its promoter and activated MYC and mTORC1 signaling. PIM1 knockdown or pharmacological inhibition of mTORC1 signaling suppressed NB‐induced NIH3T3 cell transformation. Furthermore, NB enhanced the survival of human Jurkat T‐ALL cells by inactivating the pro‐apoptotic protein BCL2‐associated agonist of cell death (BAD). Conclusion We demonstrated the pivotal role of NB in cell transformation and survival and identified PIM1as a key downstream target of NB. These findings propose a promising therapeutic strategy for patients with NB fusion‐positive leukemia.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c7672e605eea8b8da66257b0ff4b0e7a564addb" target='_blank'>
              NUP98‐BPTF promotes oncogenic transformation through PIM1 upregulation
              </a>
            </td>
          <td>
            M. Noura, Sakura Tomita, T. Yasuda, Shinobu Tsuzuki, Hitoshi Kiyoi, F. Hayakawa
          </td>
          <td>2024-06-28</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Loss-of-function mutations in methyl-CpG binding protein 2 (MECP2) cause Rett syndrome, a postnatal neurodevelopmental disorder that occurs in ∼1/10,000 live female births. MeCP2 binds to methylated cytosines across genomic DNA and recruits various partners to regulate gene expression. MeCP2 has been shown to repress transcription in vitro and interacts with co-repressors such as the Sin3A and NCoR complexes. Based on these observations, MeCP2 has been largely considered as a repressor of transcription. However, a mouse model of RTT displays many down-regulated genes, and those same genes are up-regulated in a MECP2 duplication mouse model. Furthermore, TCF20, which has been associated with transcriptional activation, have recently been identified as a protein interactor of MeCP2. These data broaden the potential functions of MeCP2 as a regulator of gene expression. Yet, the molecular mechanisms underlying MeCP2-dependent gene regulation remain largely unknown. Here, using a human MECP2 gain-of-function Drosophila model, we screened for genetic modifiers of MECP2-induced phenotypes. Our approach identified several subunits of the Drosophila super elongation complex, a P-TEFb containing RNA polymerase II (RNA pol II) elongation factor required for the release of promoter-proximally paused RNA pol II, as genetic interactors of MECP2. We discovered that MeCP2 physically interacts with the SEC in human cells and in the mouse brain. Furthermore, we found that MeCP2 directly binds AFF4, the scaffold of the SEC, via the transcriptional repression domain. Finally, loss of MeCP2 in the mouse cortex caused reduced binding of AFF4 specifically on a subset of genes involved in the regulation of synaptic function, which also displayed the strongest decrease in RNA pol II binding in the genebody. Taken together, our study reveals a previously unrecognized mechanism through which MeCP2 regulates transcription, providing a new dimension to its regulatory role in gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62efed55dc8b31895230549b72d5254a49569042" target='_blank'>
              MeCP2 Interacts with the Super Elongation Complex to Regulate Transcription
              </a>
            </td>
          <td>
            Jun Young Sonn, Wonho Kim, Marta Iwanaszko, Y. Aoi, Yan Li, Luke Parkitny, Janice L. Brissette, Lorin Weiner, I. Al-Ramahi, Juan Botas, A. Shilatifard, Huda Y. Zoghbi
          </td>
          <td>2024-07-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="There remains a large need for a greater understanding of the metastatic process within the prostate cancer field. Our research aims to understand the adaptive – ergo potentially metastatic – responses of cancer to changing microenvironments. Emerging evidence has implicated a role of the Polyaneuploid Cancer Cell (PACC) state in metastasis, positing the PACC state as capable of conferring metastatic competency. Mounting in vitro evidence supports increased metastatic potential of cells in the PACC state. Additionally, our recent retrospective study of prostate cancer patients revealed that PACC presence in the prostate at the time of radical prostatectomy was predictive of future metastatic progression. To test for a causative relationship between PACC state biology and metastasis, we leveraged a novel method designed for flow-cytometric detection of circulating tumor cells (CTCs) and disseminated tumor cells (DTCs) in subcutaneous, caudal artery, and intracardiac mouse models of metastasis. This approach provides both quantitative and qualitative information about the number and PACC-status of recovered CTCs and DTCs. Collating data from all models, we found that 74% of recovered CTCs and DTCs were in the PACC state. In vivo colonization assays proved PACC populations can regain proliferative capacity at metastatic sites following dormancy. Additional direct and indirect mechanistic in vitro analyses revealed a PACC-specific partial Epithelial-to-Mesenchymal-Transition phenotype and a pro-metastatic secretory profile, together providing preliminary evidence that PACCs are mechanistically linked to metastasis. Statement of Significance We provide the first evidence that cells in the polyaneuploid cancer cell state contribute to increased metastatic competency in vivo.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5d2da05a6c448f502761bb367a94125d4543338" target='_blank'>
              Cells in the Polyaneuploid Cancer Cell State are Pro-Metastatic
              </a>
            </td>
          <td>
            Mikaela M. Mallin, Louis T.A. Rolle, Michael J Schmidt, Shilpa Priyadarsini Nair, A. Zurita, Peter Kuhn, James B. Hicks, Kenneth J Pienta, SR Amend
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>32</td>
        </tr>

        <tr id="Background Ovarian carcinoma (OC) is a prevalent gynecological malignancy associated with high recurrence rates and mortality, often diagnosed at advanced stages. Despite advances in immunotherapy, immune exhaustion remains a significant challenge in achieving optimal tumor control. However, the exploration of intratumoral heterogeneity of malignant epithelial cells and the ovarian cancer tumor microenvironment is still limited, hindering our comprehensive understanding of the disease. Materials and methods Utilizing single-cell RNA sequencing (scRNA-seq), we comprehensively investigated the cellular composition across six ovarian cancer patients with omental metastasis. Our focus centered on analysis of the malignant epithelial cells. Employing CytoTRACE and slingshot pseudotime analyses, we identified critical subpopulations and explored associated transcription factors (TFs) influencing ovarian cancer progression. Furthermore, by integrating clinical factors from a large cohort of bulk RNA sequencing data, we have established a novel prognostic model to investigate the impact of the tumor immune microenvironment on ovarian cancer patients. Furthermore, we have investigated the condition of immunological exhaustion. Results Our study identified a distinct and highly proliferative subgroup of malignant epithelial cells, known as C2 TOP2A+ TCs. This subgroup primarily consisted of patients who hadn’t received neoadjuvant chemotherapy. Ovarian cancer patients with elevated TOP2A expression exhibited heightened sensitivity to neoadjuvant chemotherapy (NACT). Moreover, the transcription factor MYBL2 in this subgroup played a critical role in ovarian cancer development. Additionally, we developed an independent prognostic indicator, the TOP2A TCs Risk Score (TTRS), which revealed a correlation between the High TTRS Group and unfavorable outcomes. Furthermore, immune infiltration and drug sensitivity analyses demonstrated increased responsiveness to Paclitaxel, Cisplatin, and Gemcitabine in the Low TTRS Group. Conclusion This research deepens our understanding of malignant epithelial cells in ovarian cancer and enhances our knowledge of the ovarian cancer immune microenvironment and immune exhaustion. We have revealed the heightened susceptibility of the C2 TOP2A+ TCs subgroup to neoadjuvant chemotherapy and emphasized the role of MYBL2 within the C2 subgroup in promoting the occurrence and progression of ovarian cancer. These insights provide valuable guidance for the management of ovarian cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/198cae610c194371f85789bbe0373585a219e238" target='_blank'>
              Single-cell RNA sequencing reveals that MYBL2 in malignant epithelial cells is involved in the development and progression of ovarian cancer
              </a>
            </td>
          <td>
            Wenwen Shao, Zhiheng Lin, Zhikai Xiahou, Fu Zhao, Jue Xu, Xinqi Liu, Pingping Cai
          </td>
          <td>2024-07-29</td>
          <td>Frontiers in Immunology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Non-coding regions of the human genome are important for functional regulations, but their mechanisms remain elusive. We used machine learning to guide a CRISPR screening on hubs (i.e. non-coding loci forming many 3D contacts) and significantly increased the discovery rate of hubs essential for cell growth. We found no clear genetic or epigenetic differences between essential and nonessential hubs, but we observed that some neighboring hubs in the linear genome have distinct spatial contacts and opposite effects on cell growth. One such pair in an epigenetically quiescent region showed different impacts on gene expression, chromatin accessibility and chromatin organization. We also found that deleting the essential hub altered the genetic network activity and increased the entropy of chromatin accessibility, more severe than that caused by deletion of the nonessential hub, suggesting that they are critical for maintaining an ordered chromatin structure. Our study reveals new insights into the system-level roles of non-coding regions in the human genome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c37fecffd504fcf6b86c79eeec8f4d7ba8dc1c39" target='_blank'>
              Differential impact of quiescent non-coding loci on chromatin entropy.
              </a>
            </td>
          <td>
            Peiyao Wu, Mina Yao, Wei Wang
          </td>
          <td>2024-06-22</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Single-stranded DNA (ssDNA) is essential for various DNA-templated processes in both eukaryotes and prokaryotes. However, comprehensive characterizations of ssDNA still lag in plants compared to non-plant systems. Here, we conducted in situ S1-seq (ISS1-seq), with starting gDNA ranging from 5 µg to 250 ng, followed by comprehensive characterizations of ssDNA in rice (Oryza sativa L.). We found that ssDNA loci were substantially associated with a subset of non-B DNA structures and functional genomic loci. Subtypes of ssDNA loci had distinct epigenetic features. Importantly, ssDNA may act alone or partly coordinate with non-B DNA structures, functional genomic loci, or epigenetic marks to actively or repressively modulate gene transcription, which is genomic-region-dependent and associated with the distinct accumulation of RNA Pol II. Moreover, distinct types of ssDNA had differential impacts on the activities and evolution of TEs (especially common or conserved TEs) in the rice genome. Our study showcases an antibody-independent technique for characterizing non-B DNA structures or functional genomic loci in plants. It lays the groundwork and fills a crucial gap for further exploration of ssDNA, non-B DNA structures, or functional genomic loci, thereby advancing our understanding of their biology in plants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9a41f2ac1057a9cc0aff5ab06cd10c7413719a7" target='_blank'>
              Genome-wide characterization of single-stranded DNA in rice.
              </a>
            </td>
          <td>
            Yulian Peng, Pengtao Zhao, Zhaoguo Li, Ning Mu, Shentong Tao, Yilong Feng, Xuejiao Cheng, Wenli Zhang
          </td>
          <td>2024-06-25</td>
          <td>Plant physiology</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="PURPOSE
Sarcoma encompasses a diverse group of cancers that are typically resistant to current therapies, including immune checkpoint blockade (ICB), and underlying mechanisms are poorly understood. The contexture of sarcomas limits generation of high-quality data using cutting-edge molecular profiling methods, such as single-cell RNA-seq, thus hampering progress in understanding these understudied cancers.


EXPERIMENTAL DESIGN
Here, we demonstrate feasibility of producing multi-modal single-cell genomics and whole-genome sequencing data from frozen tissues, profiling 75,716 cell transcriptomes of five undifferentiated pleomorphic (UPS) and three intimal sarcomas (INS) samples, including paired specimens from two patients treated with ICB.


RESULTS
We find that genomic diversity decreases in patients with response to ICB, and, in unbiased analyses, identify cancer cell programs associated with therapy resistance. Although interactions of tumor-infiltrating T lymphocytes within the tumor ecosystem increase in ICB responders, clonal expansion of CD8+ T cells alone was insufficient to predict drug responses.


CONCLUSION
This study provides a framework for studying rare tumors and identifies salient and treatment-associated cancer cell intrinsic and tumor-microenvironmental features in sarcomas.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ece2adecc484d75c5dbd396e58a992a80003e58" target='_blank'>
              Single-cell profiling of sarcomas from archival tissue reveals programs associated with resistance to immune checkpoint blockade.
              </a>
            </td>
          <td>
            Karan Luthria, Parin Shah, Blake Caldwell, Johannes C. Melms, Sinan Abuzaid, Viktoria Jakubikova, D. Z. Brodtman, S. Bose, A. D. Amin, P. Ho, Jana Biermann, Somnath Tagore, M. Ingham, Gary K Schwartz, Benjamin Izar
          </td>
          <td>2024-07-31</td>
          <td>Clinical cancer research : an official journal of the American Association for Cancer Research</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Introduction A hallmark of small cell lung cancer (SCLC) is its recalcitrance to therapy. While most SCLCs respond to frontline therapy, resistance inevitably develops. Identifying phenotypes potentiating chemoresistance and immune evasion is a crucial unmet need. Previous reports have linked upregulation of the DNA damage response (DDR) machinery to chemoresistance and immune evasion across cancers. However, it is unknown if SCLCs exhibit distinct DDR phenotypes. Methods To study SCLC DDR phenotypes, we developed a new DDR gene analysis method and applied it to SCLC clinical samples, in vitro, and in vivo model systems. We then investigated how DDR regulation is associated with SCLC biology, chemotherapy response, and tumor evolution following therapy. Results Using multi-omic profiling, we demonstrate that SCLC tumors cluster into three DDR phenotypes with unique molecular features. Hallmarks of these DDR clusters include differential expression of DNA repair genes, increased replication stress, and heightened G2/M cell cycle arrest. SCLCs with elevated DDR phenotypes exhibit increased neuroendocrine features and decreased “inflamed” biomarkers, both within and across SCLC subtypes. Treatment naive DDR status identified SCLC patients with different responses to frontline chemotherapy. Tumors with initial DDR Intermediate and DDR High phenotypes demonstrated greater tendency for subtype switching and emergence of heterogeneous phenotypes following treatment. Conclusions We establish that SCLC can be classified into one of three distinct, clinically relevant DDR clusters. Our data demonstrates that DDR status plays a key role in shaping SCLC phenotypes, chemotherapy response, and patterns of tumor evolution. Future work targeting DDR specific phenotypes will be instrumental in improving patient outcomes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d740c08f76c9a3391148ca3278218472f317ff1" target='_blank'>
              DNA damage response signatures are associated with frontline chemotherapy response and routes of tumor evolution in extensive stage small cell lung cancer
              </a>
            </td>
          <td>
            Benjamin B. Morris, Simon Heeke, Yuanxin Xi, L. Diao, Qi Wang, Pedro Rocha, Edurne Arriola, Myung Chang Lee, Darren R. Tyson, K. Concannon, K. Ramkumar, C. Stewart, R. Cardnell, Runsheng Wang, Vito Quaranta, Jing Wang, J. Heymach, B. Nabet, D. Shames, C. Gay, L. Byers
          </td>
          <td>2024-07-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>104</td>
        </tr>

        <tr id="Budding Uninhibited by Benzimidazole-Related 1 (BubR1) or BUB1 Mitotic Checkpoint Serine/Threonine Kinase B (BUB1B) is an essential component of the spindle assembly checkpoint (SAC), which controls chromosome separation during mitosis. Overexpression of BubR1 has been associated with the progression of various cancers. This study demonstrated that high expression of BubR1 correlated with cholangiocarcinogenesis in a hamster cholangiocarcinoma (CCA) model and was associated with shorter survival in patients with CCA. Co-expression of BubR1 and MPS1, which is a SAC-related protein, indicated a shorter survival rate in patients with CCA. Knockdown of BubR1 expression by specific siRNA (siBubR1) significantly decreased cell proliferation and colony formation while inducing apoptosis in CCA cell lines. In addition, suppression of BubR1 inhibited migration and invasion abilities via epithelial–mesenchymal transition (EMT). A combination of siBubR1 and chemotherapeutic drugs showed synergistic effects in CCA cell lines. Taken together, this finding suggested that BubR1 had oncogenic functions, which influenced CCA progression. Suppression of BubR1 might be an alternative option for CCA treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fffe48083a43fe4b5ae04bca7dda6c2a1378f93" target='_blank'>
              Overexpression of BubR1 Mitotic Checkpoint Protein Predicts Short Survival and Influences the Progression of Cholangiocarcinoma
              </a>
            </td>
          <td>
            Nongnapas Pokaew, P. Prajumwongs, K. Vaeteewoottacharn, S. Wongkham, C. Pairojkul, K. Sawanyawisuth
          </td>
          <td>2024-07-01</td>
          <td>Biomedicines</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="The DNA modifications, 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC), represent powerful epigenetic regulators of temporal and spatial gene expression. Yet, how the cooperation of these genome-wide, epigenetic marks determine unique transcriptional signatures across different brain cell populations is unclear. Here we applied Nanopore sequencing of native DNA to obtain a complete, genome-wide, single-base resolution atlas of 5mC and 5hmC modifications in neurons, astrocytes and microglia in the mouse cortex (99% genome coverage, 40 million CpG sites). In tandem with RNA sequencing, analysis of 5mC and 5hmC patterns across cell types reveals astrocytes drive uniquely high brain 5hmC levels and support two decades of research regarding methylation patterns, gene expression and alternative splicing, benchmarking this resource. As such, we provide the most comprehensive DNA methylation data in mouse brain as an interactive, online tool (NAM-Me, https://olsenlab.shinyapps.io/NAMME/) to serve as a resource dataset for those interested in the methylome landscape.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60076a55629be60f22fe1afccd00156f5bc051e7" target='_blank'>
              Decoding the Epigenetic Landscape: Insights into 5mC and 5hmC Patterns in Mouse Cortical Cell Types
              </a>
            </td>
          <td>
            Xiaoran Wei, Jiangtao Li, Zuolin Cheng, Songtao Wei, Guoqiang Yu, Michelle L. Olsen
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/891d9fb97d4fae14734f86d5adf86855004117f2" target='_blank'>
              Clonal inactivation of TERT impairs stem cell competition
              </a>
            </td>
          <td>
            Kazuteru Hasegawa, Yang Zhao, A. Garbuzov, M. Corces, P. Neuhöfer, Victoria M Gillespie, Peggie Cheung, J. Belk, Yung-Hsin Huang, Yuning Wei, Lu Chen, Howard Y. Chang, S. Artandi
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>44</td>
        </tr>

        <tr id="Background Functionally redundant paralogs in the human genome are the most common source of synthetic lethality (i.e., loss of one paralog conveys dependency to another). However, most paralogs have yet to be experimentally tested and the human paralogome remains largely uncharacterized. Results We performed the first pairwise genetic screen of all human paralogs using a multiplexed CRISPR-Cas12 library, which revealed that digenic synthetic lethalities are relatively rare (<0.5% of all paralog pairs) and varied in penetrance across different cancer models. We hypothesized that the variable penetrance of digenic synthetic lethalities was a result of complex polygenic interactions with endogenous factors. A multivariable regression analysis of 1,278 pairs and endogenous cellular features across 30 cancer models revealed that perturbations of related pathways were frequently predictive of paralog synthetic lethality. A machine learning classifier was also built to predict synthetic lethalities using a weighted set of true positives, accounting for the variable penetrance of synthetic lethal interactions. Intuitively, the predictive scores revealed that the penetrance of synthetic lethal interactions was driven by the overlap and essentiality of the protein-protein interactions for each paralog pair. Conclusions This study provided a comprehensive analysis of all digenic interactions in the human paralogome, as well as the key features that underlie the heterogeneity in synthetic lethalities that have been reported here and elsewhere.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d55e188d70b44cfce388fd66288e98b0920388d1" target='_blank'>
              Mapping Genetic Interactions of All Paralogs in the Human Genome
              </a>
            </td>
          <td>
            Michael J. Flister, Daniel Verduzco, Sakina M. Petiwala, Christos Ghekas, Xu Shi, Charles Lu, Zoltan Dezso
          </td>
          <td>2024-07-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Cancer is a complex and heterogeneous disease, in which a number of genetic and epigenetic changes occur in tumor onset and progression. Recent studies indicate that changes at the RNA level are also involved in tumorigenesis, such as adenosine-to-inosine (A-to-I) RNA editing. Here, we systematically investigate transcriptome-wide A-to-I editing events in a large number of samples from Non-Hodgkin lymphomas (NHLs). Using a computational pipeline that determines significant differences in editing level between NHL and normal samples at known A-to-I editing sites, we identify a number of differentially edited editing sites between NHL subtypes and normal samples. Most of the differentially edited sites are located in non-coding regions, and many such sites show a strong correlation between gene expression level and editing efficiency, indicating that RNA editing might have direct consequences for the cancer cell’s aberrant gene regulation status in these cases. Moreover, we establish a strong link between RNA editing and NHL by demonstrating that NHL and normal samples and even NHL subtypes can be distinguished based on genome-wide RNA editing profiles alone. Our study establishes a strong link between RNA editing, cancer and aberrant gene regulation in NHL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3709fa38f5f1119b1fcb4db3f2e4cd77244ecc08" target='_blank'>
              A-to-I Editing Is Subtype-Specific in Non-Hodgkin Lymphomas
              </a>
            </td>
          <td>
            Cai Chen, Ralf Bundschuh
          </td>
          <td>2024-07-01</td>
          <td>Genes</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ffe60321069cd8bdd4dcd7dd1f6cb9aee5e0996c" target='_blank'>
              Z-DNA formation in promoters conserved between human and mouse are associated with increased transcription reinitiation rates
              </a>
            </td>
          <td>
            N. Beknazarov, Dmitry Konovalov, Alan Herbert, M. Poptsova
          </td>
          <td>2024-08-01</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Background VPS72 is highly expressed in hepatocellular carcinoma and prostate cancer, participating in various cellular processes such as gene transcription, replication, DNA repair, maintenance of genome integrity, and cancer progression. However, its role in colorectal cancer remains unknown. Methods Bioinformatic methods were used to analyze gene expression, correlation and patient survival. Western blotting, colony formation assays and animal experiments were used to evaluate the function of VPS72 in colon cancer in vivo and in vitro. Results VPS72 was highly expressed in colon cancer tissues and correlated with poor overall survival (P<0.05) and relapse free survival (P<0.01). Furthermore, patients with III/IV clinical stage (P<0.001), N1 nodal metastasis (P<0.001) or N2 nodal metastasis (P<0.05) status had poor overall survival. Further analysis showed that VPS72 is correlated with proliferation and EMT biomarkers. Western blotting, colony formation assays and animal experiments showed that VPS72 overexpression promoted colon cancer proliferation and EMT progress. Conclusion Our study found that VPS72 was correlated with poor overall survival in colon cancer patients, and high expressed level of VPS72 promoted colon cancer progression, indicating its role as a potential prognosis biomarker.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f996c9ebc6898a960d942def154ad310992ed0de" target='_blank'>
              Expression Analysis of VPS72 and Associated Biological Behaviors in Colon Cancer
              </a>
            </td>
          <td>
            Junhong Cao, Hao Zhang, Xin Xie, Wei Wang
          </td>
          <td>2024-08-01</td>
          <td>International Journal of General Medicine</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Background Colorectal cancer (CRC) remains a global health concern with persistently high incidence and mortality rates. However, the specific pathogenesis of CRC remains poorly understood. This study aims to investigate the role and pathogenesis of serine and arginine rich splicing factor 10 (SRSF10) in colorectal cancer. Methods Bioinformatics analysis was employed to predict SRSF10 gene expression in CRC patients. Functional experiments involving SRSF10 knockdown and overexpression were conducted using CCK8, transwell, scratch assay, and flow cytometry. Additionally, the PRIdictor website was utilized to predict the SRSF10 interaction site with RFC5. The identification of different transcripts of SRSF10-acting RFC5 pre-mRNA was achieved through agarose gel electrophoresis. Result The knockdown of SRSF10 inhibited the proliferation and migration ability of CRC cells, while promoting apoptosis and altering the DNA replication of CRC cells. Conversely, when SRSF10 was highly expressed, it enhanced the proliferation and migration ability of CRC cells and caused changes in the cell cycle of colorectal cancer cells. This study revealed a change in the replicating factor C subunit 5 (RFC5) gene in colorectal cancer cells following SRSF10 knockdown. Furthermore, it was confirmed that SRSF10 increased RFC5 exon2-AS1(S) transcription variants, thereby promoting the development of colorectal cancer through AS1 exclusion to exon 2 of RFC5. Conclusion In summary, this study demonstrates that SRSF10 promotes the progression of colorectal cancer by generating an aberrantly spliced exclusion isoform of AS1 within RFC5 exon 2. These findings suggest that SRSF10 could serve as a crucial target for the clinical diagnosis and treatment of CRC.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/183e863439be0cd3db11d90e783d10b2b9aa76e8" target='_blank'>
              High Expression of SRSF10 Promotes Colorectal Cancer Progression by Aberrant Alternative Splicing of RFC5
              </a>
            </td>
          <td>
            Shuai Xu, Fangmin Zhong, Junyao Jiang, Fangyi Yao, Meiyong Li, Mengxin Tang, Ying Cheng, Yulin Yang, Wen Wen, Xueru Zhang, Bo Huang, Xiao-Zhong Wang
          </td>
          <td>2024-01-01</td>
          <td>Technology in Cancer Research & Treatment</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="DNA-PKcs is a crucial protein target involved in DNA repair and response pathways, with its abnormal activity closely associated with the occurrence and progression of various cancers. In this study, we employed a deep learning-based screening and molecular dynamics (MD) simulation-based pipeline, identifying eight candidates for DNA-PKcs targets. Subsequent experiments revealed the effective inhibition of DNA-PKcs-mediated cell proliferation by three small molecules (5025-0002, M769-1095, and V008-1080). These molecules exhibited anticancer activity with IC50 (inhibitory concentration at 50%) values of 152.6 μM, 30.71 μM, and 74.84 μM, respectively. Notably, V008-1080 enhanced homology-directed repair (HDR) mediated by CRISPR/Cas9 while inhibiting non-homologous end joining (NHEJ) efficiency. Further investigations into the structure-activity relationships unveiled the binding sites and critical interactions between these small molecules and DNA-PKcs. This is the first application of DeepBindGCN_RG in a real drug screening task, and the successful discovery of a novel DNA-PKcs inhibitor demonstrates its efficiency as a core component in the screening pipeline. Moreover, this study provides important insights for exploring novel anticancer therapeutics and advancing the development of gene editing techniques by targeting DNA-PKcs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be9a29ce821456aedecdc9a3d9fda37a07d024f1" target='_blank'>
              Identification and Validation of New DNA-PKcs Inhibitors through High-Throughput Virtual Screening and Experimental Verification
              </a>
            </td>
          <td>
            Liujiang Dai, Pengfei Yu, Hongjie Fan, Wei Xia, Yaopeng Zhao, Pengfei Zhang, J. Z. Zhang, Haiping Zhang, Yang Chen
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Colorectal carcinoma (CRC) is a heterogeneous group of tumors with varying therapeutic response and prognosis, and evidence suggests the tumor immune microenvironment (TIME) plays a pivotal role. Using advanced molecular and spatial biology technologies, we aimed to evaluate the TIME in patients with CRC to determine whether specific alterations in the immune composition correlated with prognosis. We identified primary and metastatic tumor samples from 31 consented patients, which were profiled with whole-exome sequencing and bulk RNA-seq. Immune cell deconvolution followed by gene set enrichment analysis and unsupervised clustering was performed. A subset of tumors underwent in situ analysis of the TIME spatial composition at single-cell resolution through Imaging Mass Mass Cytometry. Gene set enrichment analysis revealed two distinct groups of advanced CRC, one with an immune activated phenotype and the other with a suppressed immune microenvironment. The activated TIME phenotype contained increased Th1 cells, activated dendritic cells, tertiary lymphoid structures, and higher counts of CD8+ T cells whereas the inactive or suppressed TIME contained increased macrophages and a higher M2/M1 ratio. Our findings were further supported by RNA-seq data analysis from the TCGA CRC database, in which unsupervised clustering also identified two separate groups. The immunosuppressed CRC TIME had a lower overall survival probability (HR 1.66, p=0.007). This study supports the pertinent role of the CRC immune microenvironment in tumor progression and patient prognosis. We characterized the immune cell composition to better understand the complexity and vital role that immune activity states of the TIME play in determining patient outcome.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c69179f505eff3fb4390c601ab14e625f208dae3" target='_blank'>
              Integrative Transcriptomic and Single-Cell Protein Characterization of Colorectal Carcinoma Delineates Distinct Tumor Immune Microenvironments Associated with Overall Survival
              </a>
            </td>
          <td>
            Erika Hissong, B. Bhinder, Junbum Kim, Kentaro Ohara, H. Ravichandran, M. Assaad, Sarah Elsoukkary, Michael Shusterman, Uqba Khan, K. Eng, R. Bareja, J. Manohar, M. Sigouros, A. Rendeiro, Jose Jessurun, Allyson J. Ocean, A. Sboner, Olivier Elemento, Juan Miguel Mosquera, Manish A. Shah
          </td>
          <td>2024-07-25</td>
          <td>Research Square</td>
          <td>0</td>
          <td>65</td>
        </tr>

        <tr id="Breast cancer is the most common type of cancer globally and presents an escalating problem and a huge burden on societies. Several strategies are implemented in clinics to treat patients and prevent disease incidence. Efforts to understand the underlying causes of disease emergence are pivotal, and the latest examination of human transcriptomic studies showed the involvement of the noncoding RNA regulatory molecules in influencing both pathological and physiological conditions. Several molecular mechanisms are involved in the process and collaborate to develop tumor plasticity and drug resistance. In this review, we highlight for the first time the interplay between long noncoding RNAs and Aurora kinases in breast cancer and review the latest advances in the field in an attempt to pave the way for a better understanding of the course of the disease and to delineate the targets for treatment strategies in the clinic.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e548fe1625d3f8fbf15ae302f7d6d3f128c4eaf7" target='_blank'>
              Deciphering the Complex Interplay of Long Noncoding RNAs and Aurora Kinases: Novel Insights into Breast Cancer Development and Therapeutic Strategies
              </a>
            </td>
          <td>
            M. K. Saadeldin, G. Curigliano, A. K. Abdel-Aziz
          </td>
          <td>2024-07-30</td>
          <td>Future Pharmacology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Whole-genome sequencing efforts has during the past decade unveiled the central role of genomic rearrangements—such as chromosomal inversions—in evolutionary processes, including local adaptation in a wide range of taxa. However, employment of reference genomes from distantly or even closely related species for mapping and the subsequent variant calling, can lead to errors and/or biases in the datasets generated for downstream analyses. Here, we capitalize on the recently generated chromosome-anchored genome assemblies for Arctic cod (Arctogadus glacialis), polar cod (Boreogadus saida), and Atlantic cod (Gadus morhua) to evaluate the extent and consequences of reference bias on population sequencing datasets (approx. 15-20x coverage) for both Arctic cod and polar cod. Our findings demonstrate that the choice of reference genome impacts population genetic statistics, including individual mapping depth, heterozygosity levels, and cross-species comparisons of nucleotide diversity (π) and genetic divergence (DXY). Further, it became evident that using a more distantly related reference genome can lead to inaccurate detection and characterization of chromosomal inversions, i.e., in terms of size (length) and location (position), due to inter-chromosomal reorganizations between species. Additionally, we observe that several of the detected species-specific inversions were split into multiple genomic regions when mapped towards a heterospecific reference. Inaccurate identification of chromosomal rearrangements as well as biased population genetic measures could potentially lead to erroneous interpretation of species-specific genomic diversity, impede the resolution of local adaptation, and thus, impact predictions of their genomic potential to respond to climatic and other environmental perturbations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e083b1a379647c0ff911211cf65b6e9e0db94c8" target='_blank'>
              Reference genome bias in light of species-specific chromosomal reorganization and translocations
              </a>
            </td>
          <td>
            Marius F. Maurstad, S. Hoff, José Cerca, M. Ravinet, Ian R. Bradbury, K. Jakobsen, K. Præbel, S. Jentoft
          </td>
          <td>2024-06-28</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>68</td>
        </tr>

        <tr id="Abstract Obesity, characterized by the accumulation of excess fat, is a complex condition resulting from the combination of genetic and epigenetic factors. Recent studies have found correspondence between DNA methylation and cell differentiation, suggesting a role of the former in cell fate determination. There is a lack of comprehensive understanding concerning the underpinnings of preadipocyte differentiation, specifically when cells are undergoing terminal differentiation (TD). To gain insight into dynamic genome-wide methylation, 3T3 L1 preadipocyte cells were differentiated by a hormone cocktail. The genomic DNA was isolated from undifferentiated cells and 4 hours, 2 days postdifferentiated cells, and 15 days TD cells. We employed whole-genome bisulfite sequencing (WGBS) to ascertain global genomic DNA methylation alterations at single base resolution as preadipocyte cells differentiate. The genome-wide distribution of DNA methylation showed similar overall patterns in pre-, post-, and terminally differentiated adipocytes, according to WGBS analysis. DNA methylation decreases at 4 hours after differentiation initiation, followed by methylation gain as cells approach TD. Studies revealed novel differentially methylated regions (DMRs) associated with adipogenesis. DMR analysis suggested that though DNA methylation is global, noticeable changes are observed at specific sites known as “hotspots.” Hotspots are genomic regions rich in transcription factor (TF) binding sites and exhibit methylation-dependent TF binding. Subsequent analysis indicated hotspots as part of DMRs. The gene expression profile of key adipogenic genes in differentiating adipocytes is context-dependent, as we found a direct and inverse relationship between promoter DNA methylation and gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8fe23442ed2a23d5dd7f6594deddfb3e2cbb93c9" target='_blank'>
              Genome-wide Methylation Dynamics and Context-dependent Gene Expression Variability in Differentiating Preadipocytes
              </a>
            </td>
          <td>
            Binduma Yadav, Dalwinder Singh, Shrikant S. Mantri, Vikas Rishi
          </td>
          <td>2024-06-27</td>
          <td>Journal of the Endocrine Society</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Dysregulation of the tumour suppressor gene Adenomatous Polyposis Coli (APC) is a canonical step in colorectal cancer development. Curiously, most colorectal tumours carry biallelic mutations that result in only partial loss of APC function, suggesting that a ‘just-right’ level of APC inactivation, and hence Wnt signalling, provides the optimal conditions for tumorigenesis. Mutational processes act variably across the APC gene, which could contribute to the bias against complete APC inactivation. Thus the selective consequences of partial APC loss are unclear. Here we propose a mathematical model to quantify the tumorigenic effect of biallelic APC genotypes, controlling for somatic mutational processes. Analysing sequence data from >2500 colorectal cancers, we find that APC genotypes resulting in partial protein function confer about 50 times higher probability of progressing to cancer compared to complete APC inactivation. The optimal inactivation level varies with anatomical location and additional mutations of Wnt pathway regulators. We use this context dependency to assess the regulatory effect of secondary Wnt drivers in combination with APC in vivo, and provide evidence that mutant AMER1 combines with APC genotypes that lead to relatively low Wnt. The fitness landscape of APC inactivation is consistent across microsatellite unstable and POLE-deficient colorectal cancers and tumours in patients with Familial Adenomatous Polyposis suggesting a general ‘just-right’ optimum, and pointing to Wnt hyperactivation as a potential cancer vulnerability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2f5f474ac3dba3c99ab4d1fd2c61a492c95822b9" target='_blank'>
              Quantifying ‘just-right’ APC inactivation for colorectal cancer initiation
              </a>
            </td>
          <td>
            Meritxell Brunet Guasch, Nathalie Feeley, Ignacio Soriano, Steve Thorn, Ian Tomlinson, Michael D. Nicholson, Tibor Antal
          </td>
          <td>2024-07-09</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7966c5016d9df57d57b899f6bbc5b0375d66e21b" target='_blank'>
              Comprehensive network modeling approaches unravel dynamic enhancer-promoter interactions across neural differentiation
              </a>
            </td>
          <td>
            William DeGroat, Fumitaka Inoue, Tal Ashuach, Nir Yosef, N. Ahituv, Anat Kreimer
          </td>
          <td>2024-08-14</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>49</td>
        </tr>

        <tr id="Aneuploidy is nearly ubiquitous in tumor genomes, but the role of aneuploidy in the early stages of cancer evolution remains unclear. Here, by inducing heterogeneous aneuploidy in non-transformed human colon organoids (colonoids), we investigated how the effects of aneuploidy on cell growth and differentiation may promote malignant transformation. Previous work implicated p53 activation as a downstream response to aneuploidy induction. We found that simple aneuploidy, characterized by 1-3 gained or lost chromosomes, resulted in little or modest p53 activation and cell cycle arrest when compared with more complex aneuploid cells. Single-cell RNA sequencing analysis revealed that the degree of p53 activation was strongly correlated with karyotype complexity. Single-cell tracking showed that cells could continue to divide despite the observation of one to a few lagging chromosomes. Unexpectedly, colonoids with simple aneuploidy exhibited impaired differentiation after niche factor withdrawal. These findings demonstrate that simple aneuploid cells can escape p53 surveillance and may contribute to niche factor-independent growth of cancer-initiating colon stem cells. [Media: see text].">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d69141397c86397dd6c1f0121be2fa0ac82f1c38" target='_blank'>
              Simple aneuploidy evades p53 surveillance and promotes niche factor-independent growth in human intestinal organoids.
              </a>
            </td>
          <td>
            Blake A. Johnson, Albert Z Liu, Tianhao Bi, Yi Dong, Taibo Li, Dingjingyu Zhou, A. Narkar, Yufei Wu, Sean X. Sun, Tatianna C. Larman, Jin Zhu, Rong Li
          </td>
          <td>2024-07-10</td>
          <td>Molecular biology of the cell</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4258bdff3f4b35f27d94279fb805fe639ac20d7" target='_blank'>
              CYB561 is a potential therapeutic target for breast cancer and is associated with immune cell infiltration
              </a>
            </td>
          <td>
            Jiang Zhuo, Yanchun Zhao, Ruiying Hao, He Li, Zilin Zheng, Luxian Dai, Ankang Sheng, Hanyu Yao, Yubao Tang, Rao Wang, Xiaohong Yang, Wei-Gang Liu
          </td>
          <td>2024-08-12</td>
          <td>European Journal of Medical Research</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35393ed7857060735a43bccb7ec1795bc1e251a9" target='_blank'>
              Comprehensive molecular characterization of TFE3-rearranged renal cell carcinoma.
              </a>
            </td>
          <td>
            Cho-Rong Lee, J. Suh, Dongjun Jang, Bo-Yeong Jin, Jaeso Cho, Moses Lee, Hyungtai Sim, Minyong Kang, Jueun Lee, Ju Hyun Park, Kyoung-Hwa Lee, Geum-Sook Hwang, Kyung Chul Moon, Cheryn Song, J. Ku, C. Kwak, Hyeon-Hoe Kim, Sung-Yup Cho, Murim Choi, Chang Wook Jeong
          </td>
          <td>2024-08-01</td>
          <td>Experimental & molecular medicine</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="The co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers. This phenomenon has been investigated since the late 1980s without resolution. Expanding beyond previous gene-centric studies, we investigated the co-occurrence in a genome-wide manner taking an evolutionary perspective. Mining of large-scale tumor aneuploidy data confirmed the previous finding of a small-scale longitudinal study that the most likely order is chromosome 10 loss followed by chromosome 7 gain. Extensive analysis of genomic and transcriptomic data from both patients and cell lines revealed that this co-occurrence can be explained by functional rescue interactions that are highly enriched on chromosome 7, which could potentially compensate for any detrimental consequences arising from the loss of chromosome 10. Transcriptomic data from various normal, non-cancerous human brain tissues were analyzed to assess which tissues may be most predisposed to tolerate compensation of chromosome 10 loss by chromosome 7 gain. The analysis indicated that the pre-existing transcriptomic states in the cortex and frontal cortex, where gliomas arise, are more favorable than other brain regions for compensation by rescuer genes that are active on chromosome 7. Collectively, these findings suggest that the phenomenon of chromosome 10 loss and chromosome 7 gain in gliomas is orchestrated by a complex interaction of many genes residing within these two chromosomes and provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ff13af5a4c0be0182481407e6fc3feb597b938be" target='_blank'>
              Chromosome 7 gain compensates for chromosome 10 loss in glioma.
              </a>
            </td>
          <td>
            N. Nair, Alejandro A Schaffer, E. Gertz, Kuoyuan Cheng, Johanna Zerbib, A. Sahu, Gil Leor, E. Shulman, Kenneth D Aldape, Uri Ben-David, E. Ruppin
          </td>
          <td>2024-07-30</td>
          <td>Cancer research</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64a4c5df664b8da868e003295eaab61a4ab903a5" target='_blank'>
              CHD4 R975H mutant activates tumorigenic pathways and promotes stemness and M2-like macrophage polarization in endometrial cancer
              </a>
            </td>
          <td>
            Qinglin Zhang, Fengzhi Zhu, Yin Tong, Diwen Shi, Jiangwen Zhang
          </td>
          <td>2024-08-10</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The p53 family remains a captivating focus of an extensive number of current studies. Accumulating evidence indicates that p53 abnormalities rank among the most prevalent in cancer. Given the numerous existing studies, which mostly focus on the mutations, expression profiles, and functional perturbations exhibited by members of the p53 family across diverse malignancies, this review will concentrate more on less explored facets regarding p53 activation and stabilization by the nuclear pore complex (NPC) in cancer, drawing on several studies. p53 integrates a broad spectrum of signals and is subject to diverse regulatory mechanisms to enact the necessary cellular response. It is widely acknowledged that each stage of p53 regulation, from synthesis to degradation, significantly influences its functionality in executing specific tasks. Over recent decades, a large body of data has established that mechanisms of regulation, closely linked with protein activation and stabilization, involve intricate interactions with various cellular components. These often transcend canonical regulatory pathways. This new knowledge has expanded from the regulation of genes themselves to epigenomics and proteomics, whereby interaction partners increase in number and complexity compared with earlier paradigms. Specifically, studies have recently shown the involvement of the NPC protein in such complex interactions, underscoring the further complexity of p53 regulation. Furthermore, we also discuss therapeutic strategies based on recent developments in this field in combination with established targeted therapies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30f4ce5cb3bd6bcbf7187f9f514ca001d925085c" target='_blank'>
              Unlocking the Gateway: The Spatio-Temporal Dynamics of the p53 Family Driven by the Nuclear Pores and Its Implication for the Therapeutic Approach in Cancer
              </a>
            </td>
          <td>
            Dini Kurnia Ikliptikawati, Keiichi Makiyama, M. Hazawa, Richard W. Wong
          </td>
          <td>2024-07-01</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Single-cell RNA sequencing (scRNA-seq) is a widely used method for classifying cell types and states and revealing disease mechanisms. However, most contemporary scRNA-seq platforms fail to explore the multi-landscape of RNA. Here, we designed a microfluidic chip combined oligo-dT primers and Random Bridging Co-labeling (RBCL) RNA sequencing to develop an innovative Chigene scRNA-seq technology that can identify gene expression, mutations, and RNA splicing landscapes at the single-cell level. The Chigene scRNA-seq platform demonstrated exceptional performance, with minimal doublet rates of 0.94% (Chigene V1) and 1.93% (Chigene V2). Both versions exhibit high sensitivity, with Chigene V2 achieving nearly 100% RNA coverage and detecting over 1800 genes per cell on average. Targeted capture of single-cell gene mutations enhances mutation detection sensitivity. Moreover, this Chigene V2 platform has been validated in clinical samples for its ability to detect mutations, gene fusions and alternative splicing. The reliability of the platform was further corroborated using known functional gene mutation (CDKN1A) and fusion (FGFR3-TACC). To validate this method’s potential for discovering novel gene mutations in clinical samples, our investigation revealed an intriguing cell subpopulation carrying an ARHGAP5 mutation in urothelial carcinoma. These cells exhibited high-frequency mRNA splicing and exhibited specific crosstalk with T cells, distinguishing them from the subpopulation with the ARHGAP5 wild-type phenotype. Overall, this method provides a robust scRNA-seq platform suitable for comprehensive analyses of clinical specimens at different genetic information levels, thereby offering significant potential in the discovery of novel genes and interactions at the single-cell level.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be600ce14b879188e7c67b73a02d6fde03c11a8c" target='_blank'>
              Identification of Multi-landscape and Cell Interactions in the Tumor Microenvironment through High-Coverage Single-Cell Sequencing
              </a>
            </td>
          <td>
            Wenlong Zhong, Ligang Wang, Tonglei Guo, Lihua Zhao, Daqin Wu, Fei Xie, Xiao Wang, Xiuxin Li, FangXiao Wang, Weiyue Gu, Tianxin Lin, Xu Chen
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/933aeab529c88805bcf4a9c969afbb698b11a342" target='_blank'>
              DNA methylation-mediated suppression of TUSC1 expression regulates the malignant progression of esophagogastric junction cancer
              </a>
            </td>
          <td>
            Zhiqiang Liu, Ganshu Xia, Xiaolong Liang, Baozhong Li, Jingyu Deng
          </td>
          <td>2024-07-23</td>
          <td>Clinical Epigenetics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Skin cutaneous melanoma (SKCM), a malignant melanocyte-derived skin cancer, potentially leads to fatal outcomes without effective treatment. The variability in immunotherapy responses among melanoma patients is significantly influenced by the intricate immune microenvironment, particularly due to the status of tumor T cells, encompassing their activity, exhaustion levels, and antigen recognition capabilities. This study utilized single-cell RNA sequencing (scRNA-seq) to analyze 34 melanoma samples from two public datasets (GSE215120 and GSE115978). Herein, we extracted 706 marker genes associated with immune checkpoint (ICP) therapy from these T cells, 509 markers of T cells from 11 melanoma tissues, and eventually identified 33 candidate genes. These genes underwent LASSO and COX regression analyses to identify the signature genes. Of the initial 33 candidate genes, we successfully isolated six distinct T cell-associated immunotherapy-related genes (IRTGs). Additionally, the computation of each patient risk score proved beneficial in evaluating the immune cell infiltration level and functions as an independent prognostic factor for melanoma patient survival. The risk score results revealed promising predictive outcomes in determining the response of melanoma patients to immunotherapy. Notably, our study is the first to reveal the potential correlation between signature gene PEB4B and the immune microenvironment in melaoma, which was explored with multiple immunofluorescence (IF) and Immune Infiltration Assessment. In a conclusion, our findings demonstrate the potential utility of a risk score dependent on signature genes as a predictive tool for assessing the prognosis and response to immunotherapeutic interventions in melanoma patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4480d7b0dac6fe343faaffd214e0f795e255eda" target='_blank'>
              The combination of single-cell and RNA sequencing analysis decodes the melanoma tumor microenvironment and identifies novel T cell-associated signature genes
              </a>
            </td>
          <td>
            Sihan Luo, Daiyue Wang, Jiajie Chen, Shaocheng Hong, Yuanyuan Fang, Lu Cao, Liang Yong, Shengxiu Liu
          </td>
          <td>2024-08-06</td>
          <td>Journal of Cancer</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb3e129666ae56c033aa207d599e42711f40eae8" target='_blank'>
              Gene body DNA hydroxymethylation restricts the magnitude of transcriptional changes during aging
              </a>
            </td>
          <td>
            James R. Occean, Na Yang, Yan Sun, Marshall S. Dawkins, R. Munk, Cedric Belair, Showkat A. Dar, C. Anerillas, Lin Wang, Changyou Shi, Christopher Dunn, Michel Bernier, Nathan L. Price, Julie S Kim, Chang-Yi Cui, Jinshui Fan, Moitrayee Bhattacharyya, Supriyo De, Manolis Maragkakis, R. de Cabo, Simone Sidoli, P. Sen
          </td>
          <td>2024-07-28</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>88</td>
        </tr>

        <tr id="For many years, scientific research has dedicated significant attention to unraveling the complexities of inheritance, the process by which traits are passed from one generation to the next. At the heart of this exploration lies DNA methylation, a pivotal mechanism that exerts profound influence over gene expression, cellular identity, and the overarching development of organisms. This in-depth review meticulously probes the intricate dynamics of DNA methylation, meticulously dissecting its multifaceted impact on inheritance patterns and the spectrum of phenotypic variations observed. The review scrutinizes the underlying mechanisms governing the establishment and maintenance of DNA methylation, offering a nuanced understanding of its regulatory roles. Through a synthesis of insights gleaned from the fields of molecular biology, epigenetics, and genomics, it illuminates the far-reaching implications of DNA methylation for both health and disease. By peering into the molecular machinery that orchestrates these epigenetic modifications, the review provides a comprehensive framework for comprehending the interplay between DNA methylation and the transmission of hereditary traits. Moreover, by contextualizing these findings within the broader landscape of genetics and inheritance, the review offers valuable perspectives for guiding future research endeavors. These insights not only deepen our understanding of the intricate processes underlying heredity but also hold potential implications for the development of novel therapeutic interventions aimed at modulating DNA methylation patterns. In essence, this review serves as a cornerstone in the ongoing quest to unravel the mysteries of inheritance, paving the way for innovative approaches to address complex biological phenomena.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36f158b12af137fd58de0918eb48149c92baa1bb" target='_blank'>
              Dynamics of DNA Methylation During Gametogenesis: Insights and Detection Methods
              </a>
            </td>
          <td>
            Gangadhara Doggalli, D. S. Shailaja, Keerthana S, Manojkumar H G, Santhoshinii E, Mitali Srivastava, Ritama Kundu, Arfa Ameen, Varun Kumar K
          </td>
          <td>2024-06-25</td>
          <td>Journal of Advances in Biology &amp; Biotechnology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d670592e2fca2071f917bdd4a3307abf46562a44" target='_blank'>
              Chromatin remodellers as therapeutic targets.
              </a>
            </td>
          <td>
            Hayden A Malone, Charles W M Roberts
          </td>
          <td>2024-07-16</td>
          <td>Nature reviews. Drug discovery</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcf3e13b55bc25cd806bf0acc955517dc375f3ab" target='_blank'>
              CircRNA as an Achilles heel of cancer: characterization, biomarker and therapeutic modalities
              </a>
            </td>
          <td>
            Jun Zhang, Zai Luo, Yang Zheng, Mingyu Duan, Zheng-Jun Qiu, Chen Huang
          </td>
          <td>2024-08-10</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82ad718715d0e23f67c54707d35f7ee63dc651a7" target='_blank'>
              Homeotic DUX4 Genes Shape Dynamic Inter-Chromosomal Contacts with Nucleoli in Human Cells.
              </a>
            </td>
          <td>
            E. S. Klushevskaya, I. R. Alembekov, Y. V. Kravatsky, N. A. Tchurikov
          </td>
          <td>2024-07-13</td>
          <td>Doklady. Biochemistry and biophysics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="The stability of the genome relies on Phosphatidyl Inositol 3-Kinase-related Kinases (PIKKs) that sense DNA damage and trigger elaborate downstream signaling responses. In S. cerevisiae, the Tel1 kinase (ortholog of human ATM) is activated at DNA double strand breaks (DSBs) and short telomeres. Despite the well-established roles of Tel1 in the control of telomere maintenance, suppression of chromosomal rearrangements, activation of cell cycle checkpoints, and repair of DSBs, the substrates through which Tel1 controls these processes remain incompletely understood. Here we performed an in depth phosphoproteomic screen for Tel1-dependent phosphorylation events. To achieve maximal coverage of the phosphoproteome, we developed a scaled-up approach that accommodates large amounts of protein extracts and chromatographic fractions. Compared to previous reports, we expanded the number of detected Tel1-dependent phosphorylation events by over 10-fold. Surprisingly, in addition to the identification of phosphorylation sites featuring the canonical motif for Tel1 phosphorylation (S/T-Q), the results revealed a novel motif (D/E-S/T) highly prevalent and enriched in the set of Tel1-dependent events. This motif is unique to Tel1 signaling and not shared with the Mec1 kinase, providing clues to how Tel1 plays specialized roles in DNA repair and telomere length control. Overall, these findings define a Tel1-signaling network targeting numerous proteins involved in DNA repair, chromatin regulation, and telomere maintenance that represents a framework for dissecting the molecular mechanisms of Tel1 action.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c52189157f674b0e90c368d11189e2abe058e0ae" target='_blank'>
              Profiling Tel1 Signaling Reveals a Non-Canonical Motif Targeting DNA Repair and Telomere Control Machineries
              </a>
            </td>
          <td>
            W. Comstock, E. Sanford, M. Navarro, Marcus B Smolka
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Homologous recombination (HR) is a highly conserved DNA repair system, in which aberrations can lead to the accumulation of DNA damage and genomic scars known as homologous recombination deficiency (HRD). The identification of mutations in key genes (i.e., BRCA1, and BRCA2 (BRCA)) and the quantification of large-scale structural variants (e.g., loss of heterozygosity) are indicators of the HRD phenotype. HRD is a stable biomarker and remains unchanged during recurrence, but fails to reveal the molecular profile of tumor progression. Moreover, interpretation of the current HRD score lacks comprehensiveness, especially for the HR-proficient group. Poly (ADP-ribose) polymerase (PARP) enzymes play an important role in the repair of DNA single-strand breaks, the blockage of which using PARP inhibitors (PARPi) can generate synthetic lethality in cancer cells with HRD. Although numerous studies have demonstrated that the benefit of PARPi is substantial in ovarian cancer (OC) patients, the efficacy is limited by the development of resistance, and seems to be irrespective of HR and/or BRCA mutation status. Moreover, in addition to improving progression-free survival, long-term benefit as overall survival brought by PARPi for advanced, recurrent and refractory OC patients remains unclear. Therefore, further investigations are needed to uncover the role of HR genes beyond BRCA and their interactions with other oncogenic pathways, to determine the value of HRD in the recurrent setting, and to identify alternative strategies for the precise management of advanced, refractory OC patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81fe4c9f0bc408e017d25339a0121e8f087ab765" target='_blank'>
              Limitations of homologous recombination status testing and poly (ADP-ribose) polymerase inhibitor treatment in the current management of ovarian cancer
              </a>
            </td>
          <td>
            Qianying Zhao, Liping Bai, Yu Tan, Mingrong Qie
          </td>
          <td>2024-07-22</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ae3ef3973d270e32f48aaa19a00bef4e18bca5be" target='_blank'>
              Research Progress of Long Non-coding RNA-ZFAS1 in Malignant Tumors.
              </a>
            </td>
          <td>
            Xin Liu, Zhong Ma, Xianxu Zhang, Shicheng Li, Jiangdong An, Zhiqiang Luo
          </td>
          <td>2024-07-26</td>
          <td>Cell biochemistry and biophysics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Transcription factors (TFs) are essential proteins regulating gene expression by binding to specific nucleotide sequences upstream of genes. Among TF families, the forkhead box (FOX) proteins, characterized by a conserved DNA-binding domain, play vital roles in various cellular processes, including cancer. The FOXA subfamily, encompassing FOXA1, FOXA2, and FOXA3, stands out for its pivotal role in mammalian development. FOXA1, initially identified in the liver, exhibits diverse expression across multiple organ tissues and plays a critical role in cell proliferation, differentiation, and tumor development. Its structural composition includes transactivation domains and a DNA-binding domain, facilitating its function as a pioneer factor, which is crucial for chromatin interaction and the recruitment of other transcriptional regulators. The involvement of FOXA1 in sex hormone-related tumors underscores its significance in cancer biology. This review provides an overview of multifaceted roles of FOXA1 in normal development and its implications in the pathogenesis of hormone-related cancers, particularly breast cancer and prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d64081e444f0350370e17639a169ee9d126d4248" target='_blank'>
              The Role of FOXA1 in Human Normal Development and Its Functions in Sex Hormone-Related Cancers.
              </a>
            </td>
          <td>
            Juan Zhu, Yunjian Wei, Fangnan Deng, Ying Zhou, Zaijia Yang, Yanlin Ma
          </td>
          <td>2024-06-21</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/87375be953b6b1e62c0930938d4b8c30692440ac" target='_blank'>
              Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants.
              </a>
            </td>
          <td>
            Max Frenkel, James E. Corban, Margaux L. A. Hujoel, Zachary Morris, Srivatsan Raman
          </td>
          <td>2024-07-24</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Chromatin states, fundamental to gene regulation and cellular identity, are defined by a unique combination of histone post-translational modifications. Despite their importance, comprehensive patterns within chromatin state sequences, which could provide insights into key biological functions, remain largely unexplored. In this study, we introduce ChromBERT, a BERT-based model specifically designed to detect distinct patterns of chromatin state annotation data sequences. Notably, ChromBERT was pre-trained on promoter regions across a diverse range of epigenomes and subsequently fine-tuned using a dataset from multiple cell lines where RNA-seq data were available, highlighting the model’s ability to discern conserved chromatin state patterns within these regions. In addition to its predictive powers across tasks, evidenced by high AUC scores, ChromBERT provides further analysis through the incorporation of motif clustering using Dynamic Time Warping (DTW). This method enhances the model’s ability to dissect chromatin state sequence motifs, typically involving transcription and enhancer sites. The introduction of motif clustering with DTW into ChromBERT’s workflow is poised to facilitate the discovery of genomic regions linked to novel biological functions, deepening our understanding of chromatin state dynamics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/758a0c4bd29ca771e1573835ff87961f8849bdb7" target='_blank'>
              ChromBERT: Uncovering Chromatin State Motifs in the Human Genome Using a BERT-based Approach
              </a>
            </td>
          <td>
            Seohyun Lee, Che Lin, Chien-Yu Chen, Ryuichiro Nakato
          </td>
          <td>2024-07-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Carcinomas are common in humans but rare among closely related "great apes". Plausible explanations, including human-specific genomic alterations affecting the biology of sialic acids are proposed, but causality remains unproven. Here, an integrated evolutionary genetics-phenome-transcriptome approach studied the role of SIGLEC12 gene (encodes Siglec-XII) on epithelial transformation and cancer. Exogenous expression of the protein in cell lines and genetically engineered mice recapitulated ~30% of the human population in whom the protein is expressed in a form that cannot bind ligand due to a fixed, homozygous, human-universal missense mutation. Siglec-XII null cells/mice recapitulated the remaining ~70% of the human population in whom an additional polymorphic frameshift mutation eliminates the entire protein. Siglec-XII expression drove several pro-oncogenic phenotypes in cell lines, and increased tumor burden in mice challenged with chemical carcinogen and inflammation. Transcriptomic studies yielded a 29-gene signature of Siglec-XII-positive disease and when used as a computational tool for navigating human datasets, pinpointed with surprising precision that SIGLEC12 expression (model) recapitulates a very specific type of colorectal carcinomas (disease) that is associated with mismatch-repair defects and inflammation, disproportionately affects European-Americans, and carries a better prognosis. They revealed a hitherto unknown evolutionary genetic mechanism for an ethnic/environmental predisposition of carcinogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3a026fde004925d82bafb73d8c39f7bda40aeeb" target='_blank'>
              Human-specific elimination of epithelial Siglec-XII suppresses the risk of inflammation driven colorectal cancers.
              </a>
            </td>
          <td>
            Hector A Cuello, Saptarshi Sinha, Andrea L Verhagen, N. Varki, Ajit Varki, Pradipta Ghosh
          </td>
          <td>2024-07-11</td>
          <td>JCI insight</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ba856bced2e48ff7a94743b3236b182aa83cc934" target='_blank'>
              Epigenome editing technologies for discovery and medicine.
              </a>
            </td>
          <td>
            Sean R. McCutcheon, Dahlia Rohm, Nahid Iglesias, C. Gersbach
          </td>
          <td>2024-07-29</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="In eukaryotic cells, primases are the key polymerase during DNA replication and DNA damage repair, which includes primase subunit 1 (PRIM1) and primase subunit 2 (PRIM2). Recent studies reported that the aberrant expression and activity of PRIM enzymes are closely associated with the carcinogenesis and development of various cancers. PRIM1 is overexpressed in hepatocellular carcinoma, breast cancer, and other cancers, while PRIM2 is highly expressed in lung cancer, gastrointestinal cancer, and other cancers. Further studies revealed that the knockdown of PRIM1 promoted the apoptosis of liver cancer cells, while Dihydroartemisinin (DHA) can inhibit PRIM2 expression, suppress lung cancer cell proliferation, and result in ferroptosis. The present review summarized the recent advancements in the research of the aberrant expression of PRIM1 and PRIM2 and their activity in DNA replication, DNA damage repair, and carcinogenesis. Furthermore, the strategies targeting PRIM1 or/and PRIM2 become potential therapeutic approaches in cancer treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de305a5eb143a82c984f51a4722fafc0bc430b50" target='_blank'>
              New Progress on DNA Primase Subunit Enzymes Research and Link to Cancer Development and Treatment Approaches.
              </a>
            </td>
          <td>
            Bingbing Liu, Jingjing Yan, Zhanfei Guo, Xiaodan Chen, Zhiwei Zhang
          </td>
          <td>2024-07-05</td>
          <td>Current cancer drug targets</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="All cell lineages accumulate mutations over time, increasing the probability that some lineages eventually become malignant. Many of the processes responsible for generating mutations are leaving a characteristic footprint in the genome that allows their presence to be detected. However, the mutational pattern in a tumor is usually the combined result of multiple mutational processes being at work simultaneously, and the problem of disentangling the different footprints and their relative impact then becomes a deconvolution problem. Several algorithms have been developed for this purpose, most of them involving a factorization of the mutation count matrix into two nonnegative matrices, representing respectively the underlying mutational signatures and the relative weighting of (or exposure to) these signatures. Here, we introduce the StarSignDNA algorithm for mutational signature analysis, which offers efficient refitting and de novo mutational signature extraction. StarSignDNA is capable of deciphering well-differentiated signatures linked to known mutagenic mechanisms and suggesting clinically relevant predictions for a single patient. The package offers a command line-based interface and data visualization routines. Author summary StarSignDNA is a novel algorithm for identifying mutational signatures from cancer sequencing data. It excels in low mutation count scenarios, balancing signature detection and true signature discovery. StarSignDNA improves prediction accuracy and biological validity by addressing challenges such as overfitting and underfitting, achieving optimal variable selection and shrinkage, and ensuring interpretability. In refitting, it reduces prediction variance and handle sparsity, resulting in more precise predictions. In de novo extraction, it improves the detection of challenging signatures and achieves better alignment between detected and known signatures through unsupervisedcross-validation. Its unique features, including prediction confidence and customizable reference signatures, offer valuable insights for clinical single-sample analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a5a94297c1c9cbe8ea0fed782c76348b449df01" target='_blank'>
              StarSignDNA: Signature tracing for accurate representation of mutational processes
              </a>
            </td>
          <td>
            C. Bope, Sumana Kalyanasundaram, Knut D. Rand, S. Nakken, O. Lingjærde, E. Hovig
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d3101f7048ced189804d5ffb2bbda06ceae417ac" target='_blank'>
              CRISPR-Cas9-mediated deletion enhancer of MECOM play a tumor suppressor role in ovarian cancer.
              </a>
            </td>
          <td>
            Yujie Chen, Qiuwen Jiang, Yingzhuo Xue, Weiguan Chen, Minhui Hua
          </td>
          <td>2024-07-12</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Polyploid giant cancer cells (PGCCs) are characterized by the presence of either a single enlarged nucleus or multiple nuclei and are closely associated with tumor progression and treatment resistance. These cells contribute significantly to cellular heterogeneity and can arise from various stressors, including radiation, chemotherapy, hypoxia, and environmental factors. The formation of PGCCs can occur through mechanisms such as endoreplication, cell fusion, cytokinesis failure, mitotic slippage, or cell cannibalism. Notably, PGCCs exhibit traits similar to cancer stem cells (CSCs) and generate highly invasive progeny through asymmetric division. The presence of PGCCs and their progeny is pivotal in conferring resistance to chemotherapy and radiation, as well as facilitating tumor recurrence and metastasis. This review provides a comprehensive analysis of the origins, potential formation mechanisms, stressors, unique characteristics, and regulatory pathways of PGCCs, alongside therapeutic strategies targeting these cells. The objective is to enhance the understanding of PGCC initiation and progression, offering novel insights into tumor biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/66163bd868b4436fb4d269ccebdc926754441914" target='_blank'>
              Polyploid giant cancer cells: origin, possible pathways of formation, characteristics, and mechanisms of regulation
              </a>
            </td>
          <td>
            Pan Liu, Lili Wang, Huiying Yu
          </td>
          <td>2024-07-11</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/70f393647e59fb1238cccba92da569418e311d5e" target='_blank'>
              Deciphering the genetics and mechanisms of predisposition to multiple myeloma
              </a>
            </td>
          <td>
            M. Went, Laura Duran-Lozano, G. Halldorsson, Andrea Gunnell, Nerea Ugidos-Damboriena, P. Law, Ludvig Ekdahl, Amit Sud, G. Thorleifsson, Malte Thodberg, T. Olafsdottir, Antton Lamarca-Arrizabalaga, C. Cafaro, Abhishek Niroula, Ram Ajore, Aitzkoa Lopez de Lapuente Portilla, Zain Ali, M. Pertesi, Hartmut Goldschmidt, L. Stefánsdóttir, S. Kristinsson, S. Stacey, T. Love, Saemundur Rognvaldsson, R. Hájek, Pavel Vodicka, Ulrika Pettersson-Kymmer, Florentin Späth, C. Schinke, F. van Rhee, P. Sulem, Egil Ferkingstad, Grimur Hjorleifsson Eldjarn, U. Mellqvist, I. Jónsdóttir, Gareth Morgan, P. Sonneveld, Anders Waage, N. Weinhold, H. Thomsen, A. Försti, Markus Hansson, Annette Juul-Vangsted, U. Thorsteinsdóttir, Kari Hemminki, M. Kaiser, T. Rafnar, Kári Stefánsson, R. Houlston, Björn Nilsson
          </td>
          <td>2024-08-05</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>182</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ec67b6cd2c5b516c2d3f922caad0a44789a5bdd" target='_blank'>
              TRF1 and TRF2: pioneering targets in telomere-based cancer therapy
              </a>
            </td>
          <td>
            Anoop Kallingal, Radosław Krzemieniecki, Natalia Maciejewska, Wioletta Brankiewicz-Kopcińska, Maciej Baginski
          </td>
          <td>2024-07-16</td>
          <td>Journal of Cancer Research and Clinical Oncology</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="A significant number of the genetic alterations observed in cancer patients lie within nonprotein-coding segments of the genome, including regions coding for long noncoding RNAs (lncRNAs). LncRNAs display aberrant expression in breast cancer (BrCa), but the functional implications of this altered expression remain to be elucidated. By performing transcriptome screen in a triple negative BrCa (TNBC) isogenic 2D and 3D spheroid model, we observed aberrant expression of >1000 lncRNAs during BrCa progression. The chromatin-associated lncRNA MANCR shows elevated expression in metastatic TNBC. MANCR is upregulated in response to cellular stress and modulates DNA repair and cell proliferation. MANCR promotes metastasis as MANCR-depleted cells show reduced cell migration, invasion, and wound healing in vitro, and reduced metastatic lung colonization in xenograft experiments in vivo. Transcriptome analyses reveal that MANCR modulates expression and pre-mRNA splicing of genes, controlling DNA repair and checkpoint response. MANCR promotes the transcription of NET1A, a Rho-GEF that regulates DNA damage checkpoint and metastatic processes in cis, by differential promoter usage. Experiments suggest that MANCR regulates the expression of cancer-associated genes by modulating the association of various transcription factors and RNA-binding proteins. Our results identified the metastasis-promoting activities of MANCR in TNBC by cis-regulation of gene expression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/917a434f9bc31bbdd4efde82c1a4158058876264" target='_blank'>
              MANCR lncRNA Modulates Cell-Cycle Progression and Metastasis by Cis-Regulation of Nuclear Rho-GEF.
              </a>
            </td>
          <td>
            D. K. Singh, Zhengmin Cong, Y. Song, Minxue Liu, Ritu Chaudhary, Dazhen Liu, Yu Wang, Rishabh Prasanth, Rajendra K C, Simon Lizarazo, Miriam Akhnoukh, Omid Gholamalamdari, Anurupa Moitra, Lisa M. Jenkins, Rohit Bhargava, Erik R Nelson, Kevin Van Bortle, S. Prasanth, K. Prasanth
          </td>
          <td>2024-08-12</td>
          <td>Molecular and cellular biology</td>
          <td>0</td>
          <td>39</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ddfe88dbc57ad9cb3e53a1a082129fee82af037" target='_blank'>
              Determinants of p53 DNA binding, gene regulation, and cell fate decisions
              </a>
            </td>
          <td>
            Martin Fischer, Morgan A. Sammons
          </td>
          <td>2024-06-29</td>
          <td>Cell Death and Differentiation</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="DNA G-quadruplex(G4) is a guanine-rich single-stranded DNA sequence that spontaneously folds into a spherical four-stranded DNA secondary structure in oncogene promoter sequences and telomeres. G4s are highly associated with the occurrence and development of cancer and have emerged as promising anticancer targets. Natural products have long been important sources of anticancer drug development. In recent years, significant progress has been made in the discovery of natural drugs targeting DNA G4s, with many DNA G4s have been confirmed as promising targets of natural products, including MYC-G4, KRAS-G4, PDGFR-β-G4, BCL-2-G4, VEGF-G4, and telomeric G4. This review summarizes the research progress in discovering natural small molecules that target DNA G4s and their binding mechanisms. It also discusses the opportunities of and challenges in developing drugs targeting DNA G4s. This review will serve as a valuable reference for the research on natural products, particularly in the development of novel antitumor medications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c0615bcc8ba2b9aa59d7f6c8fe9659c90b7d6290" target='_blank'>
              [Research progress in targeting DNA G-quadruplexes using natural products].
              </a>
            </td>
          <td>
            Zhi-Yuan Chen, Rong-Shuang Cheng, Wubulikasimu Ayixia, Yu-Shuang Liu, Kai-Bo Wang
          </td>
          <td>2024-07-01</td>
          <td>Zhongguo Zhong yao za zhi = Zhongguo zhongyao zazhi = China journal of Chinese materia medica</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb4b36c7afe762fcfe4d2e605f3f7060a7a118b" target='_blank'>
              ATM inhibition exploits checkpoint defects and ATM-dependent double strand break repair in TP53-mutant glioblastoma
              </a>
            </td>
          <td>
            D. Laverty, Shiv K Gupta, Gary A. Bradshaw, Alexander S Hunter, B. Carlson, Nery Matias Calmo, Jiajia Chen, Shulan Tian, J. Sarkaria, Zachary D. Nagel
          </td>
          <td>2024-06-21</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>74</td>
        </tr>

        <tr id="Horizontal gene transfer is a key driver of bacterial evolution, but it also presents severe risks to bacteria by introducing invasive mobile genetic elements. To counter these threats, bacteria have developed various defense systems, including prokaryotic Argonautes (pAgo) and the DNA Defense Module DdmDE system. Through biochemical analysis, structural determination, and in vivo plasmid clearance assays, we elucidate the assembly and activation mechanisms of DdmDE, which eliminates small, multicopy plasmids. We demonstrate that DdmE, a pAgo-like protein, acts as a catalytically inactive, DNA-guided, DNA-targeting defense module. In the presence of guide DNA, DdmE targets plasmids and recruits a dimeric DdmD, which contains nuclease and helicase domains. Upon binding to DNA substrates, DdmD transitions from an autoinhibited dimer to an active monomer, which then translocates along and cleaves the plasmids. Together, our findings reveal the intricate mechanisms underlying DdmDE-mediated plasmid clearance, offering fundamental insights into bacterial defense systems against plasmid invasions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fac230afd0ac05cd0569717ab193d8b0059c7437" target='_blank'>
              DdmDE eliminates plasmid invasion by DNA-guided DNA targeting
              </a>
            </td>
          <td>
            Xiao-Yuan Yang, Zhangfei Shen, Chen Wang, Kotaro Nakanishi, Tian-Min Fu
          </td>
          <td>2024-07-20</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Abstract Senataxin is an RNA:DNA helicase that plays an important role in the resolution of RNA:DNA hybrids (R-loops) formed during transcription. R-loops are involved in the regulation of biological processes such as immunoglobulin class switching, gene expression and DNA repair. Excessive accumulation of R-loops results in DNA damage and loss of genomic integrity. Senataxin is critical for maintaining optimal levels of R-loops to prevent DNA damage and acts as a genome guardian. Within the nucleus, senataxin interacts with various RNA processing factors and DNA damage response and repair proteins. Senataxin interactors include survival motor neuron and zinc finger protein 1, with whom it co-localizes in sub-nuclear bodies. Despite its ubiquitous expression, mutations in senataxin specifically affect neurons and result in distinct neurodegenerative diseases such as amyotrophic lateral sclerosis type 4 and ataxia with oculomotor apraxia type 2, which are attributed to the gain-of-function and the loss-of-function mutations in senataxin, respectively. In addition, low levels of senataxin (loss-of-function) in spinal muscular atrophy result in the accumulation of R-loops causing DNA damage and motor neuron degeneration. Senataxin may play multiple functions in diverse cellular processes; however, its emerging role in R-loop resolution and maintenance of genomic integrity is gaining attention in the field of neurodegenerative diseases. In this review, we highlight the role of senataxin in R-loop resolution and its potential as a therapeutic target to treat neurodegenerative diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0918b5cad58e2e8aa658850e44f481a80ca9f6d7" target='_blank'>
              Role of senataxin in R-loop-mediated neurodegeneration
              </a>
            </td>
          <td>
            Annapoorna Kannan, Shyni Gangadharan Leela, Dana Branzei, L. Gangwani
          </td>
          <td>2024-07-15</td>
          <td>Brain Communications</td>
          <td>0</td>
          <td>41</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8ee3ef93f75d18cca805d005d653d62f6f76b88e" target='_blank'>
              Comprehensive pan-cancer analysis reveals that C5orf34 regulates the proliferation and mortality of lung cancer.
              </a>
            </td>
          <td>
            Mengmeng Yang, Yuhan Deng, Yu Ma, Chunli Song, Zhenhua Wu, Xiayimaierdan Yibulayin, Xiaohong Sun, Yunquan Guo, Dan He
          </td>
          <td>2024-06-29</td>
          <td>Functional & integrative genomics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4b11c9fdbfd43bb000396d5cb58b5ead9a774473" target='_blank'>
              Hypermethylated RASAL1’s promotive role in chemoresistance and tumorigenesis of choriocarcinoma was regulated by TET2 but not DNMTs
              </a>
            </td>
          <td>
            Xianling Zeng, Rui An, Ruixia Guo, Han Li
          </td>
          <td>2024-08-08</td>
          <td>BMC Cancer</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a62302568e13d5d4ea358c501360fb449540886f" target='_blank'>
              Epigenetic regulation of HERVs: Implications for cancer immunotherapy.
              </a>
            </td>
          <td>
            Joo Mi Yi
          </td>
          <td>2024-08-01</td>
          <td>Genes & genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c01d8689b47879107902739db4fd362bf3064c23" target='_blank'>
              Spatially resolved epigenome sequencing via Tn5 transposition and deterministic DNA barcoding in tissue.
              </a>
            </td>
          <td>
            Negin Farzad, Archibald Enninful, Shuozhen Bao, Di Zhang, Yanxiang Deng, Rong Fan
          </td>
          <td>2024-06-28</td>
          <td>Nature protocols</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/135134a62bac76cb46dae2f04090c849f2c01172" target='_blank'>
              Targeting chromosomal instability in patients with cancer.
              </a>
            </td>
          <td>
            Duaa H Al-Rawi, Emanuele Lettera, Jun Li, Melody DiBona, S. Bakhoum
          </td>
          <td>2024-07-11</td>
          <td>Nature reviews. Clinical oncology</td>
          <td>1</td>
          <td>36</td>
        </tr>

        <tr id="Disease risk prediction based on genomic sequence and transcriptional profile can improve disease screening and prevention. Despite identifying many disease-associated DNA variants, distinguishing deleterious non-coding DNA variations remains poor for most common diseases. In this study, we designed in vitro experiments to uncover the significance of occupancy and competitive binding between P53 and cMYC on common target genes. Analyzing publicly available ChIP-seq data for P53 and cMYC in embryonic stem cells showed that ~344-366 regions are co-occupied, and on average, two cis-overlapping motifs (CisOMs) per region were identified, suggesting that co-occupancy is evolutionarily conserved. Using U2OS and Raji cells untreated and treated with doxorubicin to increase P53 protein level while potentially reducing cMYC level, ChIP-seq analysis illustrated that around 16 to 922 genomic regions were co-occupied by P53 and cMYC, and substitutions of cMYC signals by P53 were detected post doxorubicin treatment. Around 187 expressed genes near co-occupied regions were altered at mRNA level according to RNA-seq data analysis. We utilized a computational motif-matching approach to illustrate that changes in predicted P53 binding affinity in CisOMs of co-occupied elements significantly correlate with alterations in reporter gene expression. We performed a similar analysis using SNPs mapped in CisOMs for P53 and cMYC from ChIP-seq data, and expression of target genes from GTEx portal. We found significant correlation between change in cMYC-motif binding affinity in CisOMs and altered expression. Our study brings us closer to developing a generally applicable approach to filter etiological non-coding variations associated with common diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2175dd5f742d55fc20f67f497dba908aa719b17b" target='_blank'>
              Sequence-to-expression approach to identify etiological non-coding DNA variations in P53 and cMYC-driven diseases.
              </a>
            </td>
          <td>
            K. Kin, Shounak Bhogale, Lisha Zhu, Derrick Thomas, Jessica W Bertol, W. J. Zheng, Saurabh Sinha, W. Fakhouri
          </td>
          <td>2024-07-17</td>
          <td>Human molecular genetics</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fbe961497d985de77bb27297cd1364379e55768f" target='_blank'>
              Circular RNA in cancer.
              </a>
            </td>
          <td>
            Vanessa M. Conn, A. Chinnaiyan, Simon J. Conn
          </td>
          <td>2024-07-29</td>
          <td>Nature reviews. Cancer</td>
          <td>0</td>
          <td>164</td>
        </tr>

        <tr id="Transposable elements (TEs) have emerged as important factors in establishing the cell type-specific gene regulatory networks and evolutionary novelty of embryonic and placental development. Recently, studies on the role of TEs and their dysregulation in cancers have shed light on the transcriptional, transpositional, and regulatory activity of TEs, revealing that the activation of developmental transcriptional programs by TEs may have a role in the dedifferentiation of cancer cells to the progenitor-like cell states. This essay reviews the recent evidence of the cis-regulatory TEs (henceforth crTE) in normal development and malignancy as well as the key transcription factors and regulatory pathways that are implicated in both cell states, and presents existing gaps remaining to be studied, limitations of current technologies, and therapeutic possibilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d9ae01d2549bee9fa618c02f57bcecdeddf683a" target='_blank'>
              Transposable elements as drivers of dedifferentiation: Connections between enhancers in embryonic stem cells, placenta, and cancer.
              </a>
            </td>
          <td>
            Konsta Karttunen, Divyesh Patel, Biswajyoti Sahu
          </td>
          <td>2024-07-29</td>
          <td>BioEssays : news and reviews in molecular, cellular and developmental biology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Transposable elements (TEs) comprise a substantial portion of the mammalian genome, with potential implications for both embryonic development and cancer. This study aimed to characterize the expression profiles of TEs in embryonic stem cells (ESCs), cancer cell lines, tumor tissues, and the tumor microenvironment (TME). We observed similarities in TE expression profiles between cancer cells and ESCs, suggesting potential parallels in regulatory mechanisms. Notably, four TE RNAs (HERVH, LTR7, HERV-Fc1, HERV-Fc2) exhibited significant downregulation across cancer cell lines and tumor tissues compared to ESCs, highlighting potential roles in pluripotency regulation. The strong up-regulation of the latter two TEs (HERV-Fc1, HERV-Fc2) in ESCs has not been previously demonstrated and may be a first indication of their role in the regulation of pluripotency. Conversely, tandemly repeated sequences (MSR1, CER, ALR) showed up-regulation in cancer contexts. Moreover, a difference in TE expression was observed between the TME and the tumor bulk transcriptome, with distinct dysregulated TE profiles. Some TME-specific TEs were absent in normal tissues, predominantly belonging to LTR and L1 retrotransposon families. These findings not only shed light on the regulatory roles of TEs in both embryonic development and cancer but also suggest novel targets for anti-cancer therapy. Understanding the interplay between cancer cells and the TME at the TE level may pave the way for further research into therapeutic interventions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/00a20858efbeb57c8dadb0f8fcbf11554339edb7" target='_blank'>
              Dysregulation of Transposon Transcription Profiles in Cancer Cells Resembles That of Embryonic Stem Cells
              </a>
            </td>
          <td>
            A. Solovyeva, Roman V. Afanasev, Marina A. Popova, N. Enukashvily
          </td>
          <td>2024-08-05</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The Chromodomain helicase DNA-binding protein 1-like (CHD1L) is a nucleosome remodeling enzyme, which plays a key role in chromatin relaxation during the DNA damage response. Genome editing has shown that deletion of CHD1L sensitizes cells to PARPi, but the effect of its pharmacological inhibition has not been defined. Triple-negative breast cancer SUM149PT, HCC1937, and MDA-MB-231 cells were used to assess the mechanism of action of the CHD1Li OTI-611. Cytotoxicity as a single agent or in combination with standard-of-care treatments was assessed in tumor organoids. Immunofluorescence was used to assess the translocation of PAR and AIF to the cytoplasm or the nucleus and to study markers of DNA damage or apoptosis. Trapping of PARP1/2 or CHD1L onto chromatin was also assessed by in situ subcellular fractionation and immunofluorescence and validated by Western blot. We show that the inhibition of CHD1L’s ATPase activity by OTI-611 is cytotoxic to triple-negative breast cancer tumor organoids and synergizes with PARPi and chemotherapy independently of the BRCA mutation status. The inhibition of the remodeling function blocks the phosphorylation of H2AX, traps CHD1L on chromatin, and leaves PAR chains on PARP1/2 open for hydrolysis. PAR hydrolysis traps PARP1/2 at DNA damage sites and mediates PAR translocation to the cytoplasm, release of AIF from the mitochondria, and induction of PARthanatos. The targeted inhibition of CHD1L’s oncogenic function by OTI-611 signifies an innovative therapeutic strategy for breast cancer and other cancers. This approach capitalizes on CHD1L-mediated DNA repair and cell survival vulnerabilities, thereby creating synergy with standard-of-care therapies">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/401b6daf1d2a20088ba1aa72a3182ab37cb9c8e4" target='_blank'>
              CHD1L Regulates Cell Survival in Breast Cancer and Its Inhibition by OTI-611 Impedes the DNA Damage Response and Induces PARthanatos
              </a>
            </td>
          <td>
            Rita Sala, Hector Esquer, Timothy Kellett, Jeffrey T. Kearns, Paul Awolade, Qiong Zhou, Daniel V. LaBarbera
          </td>
          <td>2024-08-06</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40fcdfbbc2df4270401ef8beed0b78736d658d46" target='_blank'>
              DNA methylation in mammalian development and disease.
              </a>
            </td>
          <td>
            Zachary D. Smith, Sara Hetzel, Alexander Meissner
          </td>
          <td>2024-08-12</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/df8402eef6e3423a478e5702cc35b37ae6d3bc0c" target='_blank'>
              CTHRC1 modulates cell proliferation and invasion in hepatocellular carcinoma by DNA methylation
              </a>
            </td>
          <td>
            Xiangjun Sun, Ye Liu, Changdong Cheng, Haoyu Sun, Liqiang Tian
          </td>
          <td>2024-08-12</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Breast cancer (BC) is the most prevalent malignancy affecting women worldwide, including Portugal. While the majority of BC cases are sporadic, hereditary forms account for 5-10% of cases. The most common inherited mutations associated with BC are germline mutations in the BReast CAncer (BRCA) 1/2 gene (gBRCA1/2). They are found in approximately 5-6% of BC patients and are inherited in an autosomal dominant manner, primarily affecting younger women. Pathogenic variants within BRCA1/2 genes elevate the risk of both breast and ovarian cancers and give rise to distinct clinical phenotypes. BRCA proteins play a key role in maintaining genome integrity by facilitating the repair of double-strand breaks through the homologous recombination (HR) pathway. Therefore, any mutation that impairs the function of BRCA proteins can result in the accumulation of DNA damage, genomic instability, and potentially contribute to cancer development and progression. Testing for gBRCA1/2 status is relevant for treatment planning, as it can provide insights into the likely response to therapy involving platinum-based chemotherapy and poly[adenosine diphosphate (ADP)-ribose] polymerase inhibitors (PARPi). The aim of this review was to investigate the impact of HR deficiency in BC, focusing on BRCA mutations and their impact on the modulation of responses to platinum and PARPi therapy, and to share the experience of Unidade Local de Saúde Santa Maria in the management of metastatic BC patients with DNA damage targeted therapy, including those with the Portuguese c.156_157insAlu BRCA2 founder mutation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/886964a622b1b6bd32136b243a3eaa0a8803cae8" target='_blank'>
              DNA damage targeted therapy for advanced breast cancer
              </a>
            </td>
          <td>
            V. Patel, S. Casimiro, C. Abreu, Tiago Barroso, R. T. de Sousa, Sofia Torres, L. Ribeiro, Gonçalo Nogueira-Costa, H. Pais, Conceição Pinto, Leila Costa, Luís Costa
          </td>
          <td>2024-06-25</td>
          <td>Exploration of Targeted Anti-tumor Therapy</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e7d183f3062975c60ce5ba3ada1020c80a06fee4" target='_blank'>
              Prion-like domain mediated phase separation of ARID1A promotes oncogenic potential of Ewing’s sarcoma
              </a>
            </td>
          <td>
            Yong Ryoul Kim, Jaegeon Joo, Hee Jung Lee, Chaelim Kim, Ju-Chan Park, Young Suk Yu, Chang Rok Kim, Do Hui Lee, Joowon Cha, Hyemin Kwon, Kimberley M. Hanssen, T. G. Grünewald, Murim Choi, Ilkyu Han, Sangsu Bae, Inkyung Jung, Yongdae Shin, Sung Hee Baek
          </td>
          <td>2024-08-03</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b0bff8174988ff460b1ed14cde5b076ac3673528" target='_blank'>
              Methods and applications of genome-wide profiling of DNA damage and rare mutations.
              </a>
            </td>
          <td>
            Gerd P Pfeifer, Seung-Gi Jin
          </td>
          <td>2024-06-25</td>
          <td>Nature reviews. Genetics</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is the leading cause of cancer-related death worldwide, but the underlying molecular mechanisms remain largely unclear. The transcription factor (TF) specificity protein 1 (SP1) plays a crucial role in the development of various cancers, including LUAD. Recent studies have indicated that master TFs may form phase-separated macromolecular condensates to promote super-enhancer (SE) assembly and oncogene expression. In this study, we demonstrated that SP1 undergoes phase separation and that its zinc finger 3 in the DNA-binding domain is essential for this process. Through Cleavage Under Targets & Release Using Nuclease (CUT&RUN) using antibodies against SP1 and H3K27ac, we found a significant correlation between SP1 enrichment and SE elements, identified the regulator of the G protein signaling 20 (RGS20) gene as the most likely target regulated by SP1 through SE mechanisms, and verified this finding using different approaches. The oncogenic activity of SP1 relies on its phase separation ability and RGS20 gene activation, which can be abolished by glycogen synthase kinase J4 (GSK-J4), a demethylase inhibitor. Together, our findings provide evidence that SP1 regulates its target oncogene expression through phase separation and SE mechanisms, thereby promoting LUAD cell progression. This study also revealed an innovative target for LUAD therapies through intervening in SP1-mediated SE formation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62bc7989f17a367d0edf46ad13008c32a2a0160d" target='_blank'>
              SP1 undergoes phase separation and activates RGS20 expression through super-enhancers to promote lung adenocarcinoma progression.
              </a>
            </td>
          <td>
            L. Shan, Wenmeng Wang, Lijuan Du, Dangdang Li, Yunxuan Wang, Yu Xie, Hongyan Li, Jiale Wang, Zhihao Shi, Yang Zhou, Daling Zhu, Guangchao Sui, Fang Liu
          </td>
          <td>2024-07-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e1731e1b468345dc4e9d5ee9c2101e06de967b49" target='_blank'>
              Timing of whole genome duplication is associated with tumor-specific MHC-II depletion in serous ovarian cancer
              </a>
            </td>
          <td>
            Nikki L Burdett, Madelynne O. Willis, A. Pandey, Laura Twomey, Sara Alaei, D. Bowtell, E. Christie
          </td>
          <td>2024-07-18</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/261ea82053aaf73b2c3fb5ebe769c19bae07d5fa" target='_blank'>
              53BP1 loss elicits cGAS-STING-dependent antitumor immunity in ovarian and pancreatic cancer
              </a>
            </td>
          <td>
            Yajie Sun, Jeffrey Patterson-Fortin, Sen Han, Zhe Li, Zuzanna Nowicka, Yuna Hirohashi, Susan Kilgas, Jae Kyo Yi, A. Spektor, Wojciech Fendler, Panagiotis A Konstantinopoulos, Dipanjan Chowdhury
          </td>
          <td>2024-08-06</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Progression from pre-cancers like ductal carcinoma in situ (DCIS) to invasive disease (cancer) is driven by somatic evolution and is altered by clinical interventions. We hypothesized that genetic and/or phenotypic intra-tumor heterogeneity would predict clinical outcomes for DCIS since it serves as the substrate for natural selection among cells. We profiled two samples from two geographically distinct foci from each DCIS in both cross-sectional (N = 119) and longitudinal cohorts (N = 224), with whole exome sequencing, low-pass whole genome sequencing, and a panel of immunohistochemical markers. In the longitudinal cohorts, the only statistically significant predictors of time to non-invasive DCIS recurrence were the combination of treatment (lumpectomy only vs mastectomy or lumpectomy with radiation, HR = 12.13, p = 0.003, Wald test with FDR correction), ER status (HR = 0.16 for ER+ compared to ER-, p = 0.0045), and divergence in SNVs between the two samples (HR = 1.33 per 10% divergence, p = 0.018). SNV divergence also distinguished between pure DCIS and DCIS synchronous with invasive disease in the cross-sectional cohort. In contrast, the only statistically significant predictors of time to progression to invasive disease were the combination of the width of the surgical margin (HR = 0.67 per mm, p = 0.043) and the number of mutations that were detectable at high allele frequencies (HR = 1.30 per 10 SNVs, p = 0.02). These results imply that recurrence with DCIS is a clinical and biological process different from invasive progression.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2027c796343907ccaa630775f7381619210caedc" target='_blank'>
              Evolutionary Measures Show that Recurrence of DCIS is Distinct from Progression to Breast Cancer
              </a>
            </td>
          <td>
            Angelo Fortunato, D. Mallo, Luis Cisneros, Lorraine M. King, Aziz Khan, Christina Curtis, M. Ryser, Joseph Y. Lo, Allison Hall, Jeffrey R. Marks, E. S. Hwang, Carlo C. Maley, E. Tyler
          </td>
          <td>2024-08-16</td>
          <td>None</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e8868db406f4d297083b3336bdacf7e70f944e3" target='_blank'>
              Characterize molecular signatures and establish a prognostic signature of gastric cancer by integrating single-cell RNA sequencing and bulk RNA sequencing
              </a>
            </td>
          <td>
            Zhiwei Wang, Zhiyan Weng, Luping Lin, Xianyi Wu, Wenju Liu, Yong Zhuang, J. Jian, Changhua Zhuo
          </td>
          <td>2024-07-24</td>
          <td>Discover Oncology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Background Recent research indicates a positive correlation between DEP structural domain‐containing 1B (DEPDC1B) and the cell cycle in various tumors. However, the role of DEPDC1B in the infiltration of the tumor immune microenvironment (TIME) remains unexplored. Methods We analyzed the differential expression and prognostic significance of DEPDC1B in colon adenocarcinoma (COAD) using the R package “limma” and the Gene Expression Profiling Interactive Analysis (GEPIA) website. Gene set enrichment analysis (GSEA) was employed to investigate the functions and interactions of DEPDC1B expression in COAD. Cell Counting Kit‐8 (CCK‐8) assays and colony formation assays were utilized to assess the proliferative function of DEPDC1B. Correlations between DEPDC1B expression and tumor‐infiltrating immune cells, immune checkpoints, tumor mutational burden (TMB), and microsatellite instability (MSI) status were examined using Spearman correlation analysis and CIBERSORT. Results DEPDC1B was highly expressed in COAD. Elevated DEPDC1B expression was associated with lower epithelial‐to‐mesenchymal transition (EMT) and TNM stages, leading to a favorable prognosis. DEPDC1B mRNA was prominently expressed in COAD cell lines. CCK‐8 and colony formation assays demonstrated that DEPDC1B inhibited the proliferation of COAD cells. Analysis using the CIBERSORT database and Spearman correlation revealed that DEPDC1B correlated with four types of tumor‐infiltrating immune cells. Furthermore, high DEPDC1B expression was linked to the expression of PD‐L1, CTLA4, SIGLEC15, PD‐L2, TMB, and MSI‐H. High DEPDC1B expression also indicated responsiveness to anti‐PD‐L1 immunotherapy. Conclusions DEPDC1B inhibits the proliferation of COAD cells and positively regulates the cell cycle, showing a positive correlation with CCNB1 and PBK expression. DEPDC1B expression in COAD is associated with tumor‐infiltrating immune cells, immune checkpoints, TMB, and MSI‐H in the tumor immune microenvironment. This suggests that DEPDC1B may serve as a novel prognostic marker and a potential target for immunotherapy in COAD.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/293cb0f525ffa84f15dbabefe7162aab0692a337" target='_blank'>
              DEPDC1B: A novel tumor suppressor gene associated with immune infiltration in colon adenocarcinoma
              </a>
            </td>
          <td>
            Dandan Zhu, Huolun Feng, Zhixiong Zhang, Jiaqi Li, Yong Li, Tieying Hou
          </td>
          <td>2024-08-01</td>
          <td>Cancer Medicine</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In transcription-coupled repair, stalled RNA polymerase II (Pol II) is recognized by CSB and CRL4CSA, which co-operate with UVSSSA and ELOF1 to recruit TFIIH for nucleotide excision repair (TC-NER). To explore the mechanism of TC-NER, we recapitulated this reaction in vitro. When a plasmid containing a site-specific lesion is transcribed in frog egg extract, error-free repair is observed that depends on CSB, CRL4CSA, UVSSA, and ELOF1. Repair also depends on STK19, a factor previously implicated in transcription recovery after UV exposure. A 1.9 Å cryo-electron microscopy structure shows that STK19 joins the TC-NER complex by binding CSA and the RPB1 subunit of Pol II. Furthermore, AlphaFold predicts that STK19 interacts with the XPD subunit of TFIIH, and disrupting this interface impairs cell-free repair. Molecular modeling suggests that STK19 positions TFIIH ahead of Pol II for lesion verification. In summary, our analysis of cell-free TC-NER suggests that STK19 couples RNA polymerase II stalling to downstream repair events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/297011593f4e81e15ed2ffb10b29598b2dac1064" target='_blank'>
              STK19 positions TFIIH for cell-free transcription-coupled DNA repair
              </a>
            </td>
          <td>
            T. Mevissen, Maximilian Kümmecke, Ernst W. Schmid, L. Farnung, Johannes C. Walter
          </td>
          <td>2024-07-23</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f5fd8111e35edd2eb7d5b85919fabd765c8e2e1" target='_blank'>
              Subcellular location of L1 retrotransposon-encoded ORF1p, reverse transcription products, and DNA sensors in lupus granulocytes
              </a>
            </td>
          <td>
            Fatemeh Moadab, Sepideh Sohrabi, Xiaoxing Wang, Rayan Najjar, Justina C Wolters, Hua Jiang, Wenyan Miao, Donna L Romero, Dennis M Zaller, Megan Tran, Alison Bays, Martin S Taylor, Rosana Kapeller, J. LaCava, Tomas Mustelin
          </td>
          <td>2024-06-27</td>
          <td>Mobile DNA</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/16eda9843eb0037407162ede7ade3db9a52371a7" target='_blank'>
              SNM1A is crucial for efficient repair of complex DNA breaks in human cells
              </a>
            </td>
          <td>
            L. P. Swift, B. C. Lagerholm, Lucy R. Henderson, Malitha Ratnaweera, H. T. Baddock, B. Sengerova, Sook Y. Lee, A. Cruz-Migoni, Dominic Waithe, Christian Renz, Helle D. Ulrich, Joseph A. Newman, Christopher J. Schofield, P. McHugh
          </td>
          <td>2024-06-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="Knowledge of locations and activities of cis-regulatory elements (CREs) is needed to decipher basic mechanisms of gene regulation and to understand the impact of genetic variants on complex traits. Previous studies identified candidate CREs (cCREs) using epigenetic features in one species, making comparisons difficult between species. In contrast, we conducted an interspecies study defining epigenetic states and identifying cCREs in blood cell types to generate regulatory maps that are comparable between species, using integrative modeling of eight epigenetic features jointly in human and mouse in our Validated Systematic Integration (VISION) Project. The resulting catalogs of cCREs are useful resources for further studies of gene regulation in blood cells, indicated by high overlap with known functional elements and strong enrichment for human genetic variants associated with blood cell phenotypes. The contribution of each epigenetic state in cCREs to gene regulation, inferred from a multivariate regression, was used to estimate epigenetic state Regulatory Potential (esRP) scores for each cCRE in each cell type, which were used to categorize dynamic changes in cCREs. Groups of cCREs displaying similar patterns of regulatory activity in human and mouse cell types, obtained by joint clustering on esRP scores, harbored distinctive transcription factor binding motifs that were similar between species. An interspecies comparison of cCREs revealed both conserved and species-specific patterns of epigenetic evolution. Finally, we showed that comparisons of the epigenetic landscape between species can reveal elements with similar roles in regulation, even in the absence of genomic sequence alignment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62e8211dd3cbaad431cfef28f6397a18c06fd052" target='_blank'>
              Interspecies regulatory landscapes and elements revealed by novel joint systematic integration of human and mouse blood cell epigenomes.
              </a>
            </td>
          <td>
            Guanjue Xiang, Xi He, B. Giardine, Kathryn J. Isaac, Dylan J. Taylor, R. McCoy, Camden Jansen, C. Keller, A. Wixom, April Cockburn, Amber Miller, Qian Qi, Yanghua He, Yichao Li, Jens Lichtenberg, Elisabeth F. Heuston, Stacie M. Anderson, Jing Luan, M. Vermunt, Feng Yue, M. Sauria, Michael C. Schatz, James Taylor, B. Göttgens, Jim R. Hughes, D. Higgs, Mitchell J. Weiss, Yong Cheng, G. Blobel, D. Bodine, Yu Zhang, Qunhua Li, Shaun Mahony, R. Hardison
          </td>
          <td>2024-07-01</td>
          <td>Genome research</td>
          <td>0</td>
          <td>78</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/740b43d1d740fa1f91da9a11435ead179fc56531" target='_blank'>
              Epigenetic silencing of BEND4, a novel DNA damage repair gene, is a synthetic lethal marker for ATM inhibitor in pancreatic cancer.
              </a>
            </td>
          <td>
            Yuanxin Yao, Honghui Lv, Meiying Zhang, Yuan Li, James G Herman, Malcolm V. Brock, Aiai Gao, Qian Wang, Francois Fuks, Lirong Zhang, Mingzhou Guo
          </td>
          <td>2024-06-27</td>
          <td>Frontiers of medicine</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="This article theoretically speculates that after replication of a pair of DNA they must be allocated to two daughter cells in a transverse division or erect division manner. Transverse division represents differentiation, while erect division represents transformation. The molecular structure determines that DNA has greater hydrophobicity than RNA. The hydrophobic modification of chromatin regulates gene activity and centromere similarity between new and old DNA. A difference in division between cancer cells and embryonic stem cells is discussed">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5c6b78e19de2f6d2d4dda38ace7242257dc237cb" target='_blank'>
              On the Cancer
              </a>
            </td>
          <td>
            Xiao En Wang
          </td>
          <td>2024-08-31</td>
          <td>Journal of Cancer Research Reviews &amp; Reports</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 Can centromeres be labelled in live human oocytes using Cas9 system and is the labelling efficiency affected by advancing female age?



 Centromeres in human oocytes can be labelled using Cas9 system. Signal intensity decreases in oocytes from older patients, suggesting centromeric DNA unravelling as oocytes age.



 Aneuploidy in human oocytes is associated with infertility, pregnancy loss or genetic diseases in offspring. It arises due to abnormal chromosome segregation during meiosis, with the risk increasing with advancing female age. This is attributed to age-dependent loss of pericentromere-localised cohesion which holds sister chromatids together. Our work revealed that shugoshin-2 (Sgo2), a protein involved in protection of sister chromatid cohesion is also lost with age. This can exacerbate the effects of maternal age by rendering residual cohesin at pericentromeres vulnerable to loss in anaphase I. This hints at more extensive alterations to centromere organisation with advanced maternal age.



 This study uses human oocytes deselected from fertility treatment donated for research purposes with informed consent (HFEA RO155). The oocytes were collected fresh at germinal vesicle (GV), metaphase I (MI) or metaphase II (MII) stage, injected with Cas9 labelling system and live imaged or fixed.



 Centromeres were labelled in human oocytes using a dead (nuclease-deficient) (d)Cas9-EGFP protein and alpha satellite-specific small guide RNAs (sgRNA). Human oocytes were injected with dCas9-EGFP:sgAlphaSat and live-imaged using dual colour confocal microscopy with SPY555-DNA dye to also visualise chromosomes. High resolution imaging results were analysed with in-house automated kinetochore tracking software. Additional oocytes were fixed and processed for immunofluorescence to determine the high resolution localisation of dCas9-EGFP:sgAlphaSat and Sgo2 molecules.



 We found that centromeres in human oocytes are efficiently labelled when using Cas9 system. During both meiosis I and II the dCas9-EGFP:sgAlphaSat probe co-localised with pericentromeric bridge and subcentromeric pools of Sgo2. We observed that the dCas9-EGFP:sgAlphaSat signal fades with advancing female age. Quantification reveals that in meiosis II the loss is abrupt at around 35 years of age, resembling Sgo2 behaviour. Initial live imaging indicates that already compromised signal in older oocytes is completely lost about 4hrs after germinal vesicle breakdown, suggesting that advancing age renders centromeres more vulnerable as meiosis I progresses. We have successfully tracked the centromeres in high resolution imaging using the automated software, which allows for detailed investigation of human meiosis chromosome dynamics. This method has been previously validated and successfully used for other cell systems.



 Availability of human oocytes for research purposes is low, limiting our sample size. Moreover, most oocytes used for this study were deselected from treatment of patients diagnosed with infertility, potentially affecting their quality.



 Our results reveal a large-scale change in centromeric DNA structure in human oocytes with increasing age. Live centromere labelling is a first successful attempt to bivalent tracking in human meiosis with potential to provide insight into age-related loss of fertility in women.



 n/a
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/441db6fa05acf55025e3ab0193301dfa95dff58a" target='_blank'>
              O-070 Cas9-based chromosome labelling reveals large-scale age-dependent centromere reorganisation in human oocytes
              </a>
            </td>
          <td>
            A. Byrska, C. Currie, D. Taylor, M. Erent, A. Inchingolo, G. Hartshorne, A. McAinsh
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9cf95f1dc0f1af0cb6e8bd77dab713b5fc609919" target='_blank'>
              CircRHBDD1 promotes immune escape via IGF2BP2/PD-L1 signaling and acts as a nanotherapeutic target in gastric cancer
              </a>
            </td>
          <td>
            Yanna Li, Zhixiong Wang, Peng Gao, Danping Cao, R. Dong, Menglin Zhu, Yao Fei, X. Zuo, Juan Cai
          </td>
          <td>2024-07-30</td>
          <td>Journal of Translational Medicine</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Solid tumors as well as leukemias and lymphomas show striking changes in nuclear structure including nuclear size and shape, the number and size of nucleoli, and chromatin texture. These alterations have been used in cancer diagnosis and might be related to the altered functional properties of cancer cells. The nuclear matrix (NM) represents the structural composition of the nucleus and consists of nuclear lamins and pore complexes, an internal ribonucleic protein network, and residual nucleoli. In the nuclear microenvironment, the NM is associated with multi-protein complexes, such as basal transcription factors, signaling proteins, histone-modifying factors, and chromatin remodeling machinery directly or indirectly through scaffolding proteins. Therefore, alterations in the composition of NM could result in altered DNA topology and changes in the interaction of various genes, which could then participate in a cascade of the cancer process. Using an androgen-sensitive prostate cancer cell line, LNCaP, and its androgen-independent derivative, LN96, conventional 2D-proteomic analysis of the NM proteins revealed that purine-rich element binding protein alpha (PURα) was detected in the NM proteins and differentially expressed between the cell lines. In this article, we will review the potential role of the molecule in prostate cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6e4cf98a2631cb0f5a99ef7fdb453a505020fcc1" target='_blank'>
              Purine-Rich Element Binding Protein Alpha, a Nuclear Matrix Protein, Has a Role in Prostate Cancer Progression
              </a>
            </td>
          <td>
            Takahiro Inoue, Xin Bao, Takumi Kageyama, Y. Sugino, S. Sekito, Shiori Miyachi, Takeshi Sasaki, Robert Getzenberg
          </td>
          <td>2024-06-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="DNA integrity and stability are vital for proper cellular activity. Nevertheless, to treat cancer patients, DNA is the main target for inducing tumoral cell death. Nowadays, cancer treatment is improving by the development of new technologies, protocols and strategies. Amongst them, the introduction of charged particles as a form of radiotherapy is underway. However, tumor-neighboring healthy tissues are still exposed to ionizing radiations (IR) and subject to late side effects. Skeletal muscle is one of those tissues most likely to be affected. To decipher the DNA damage response (DDR) of skeletal muscle cells, myogenic cells, we irradiated them with microbeams of protons or α-particles and followed the accumulation of DDR proteins at irradiation sites. Thereby, we showed that myoblasts, proliferating myogenic cells, repair IR-induced DNA damage through both non-homologous end-joining and homologous recombination with different recruitment dynamics depending on the characteristics of ionizing particles (type, energy deposition and time after irradiation), whereas myotubes, post-mitotic myogenic cells, display reduced DNA damage response. Summary Mouse skeletal muscle cells and their precursor cells irradiated by ion microbeam show DNA damage response and DNA repair dynamics that vary according to the characteristics of ionizing particles delivered. Graphical Abstract">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58e611cf27e87af80de9e630a6803a7011f97552" target='_blank'>
              The DNA damage response relies on the characteristics of ionizing particles in myogenic cells
              </a>
            </td>
          <td>
            H. Sutcu, Arthur Thomas--Joyeux, M. Cardot-Martin, François Vianna, M. Benadjaoud, Marc Benderitter, Céline Baldeyron
          </td>
          <td>2024-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Somatic evolution in ageing tissues underlies many cancers. However, our quantitive understanding of the rules governing this pre-cancerous evolution remains incomplete. Here we exploit a unique collection of serial blood samples collected annually up to 15 years prior to diagnosis of acute myeloid leukaemia (AML) to provide a quantitative description of pre-cancerous evolutionary dynamics. Using deep duplex sequencing and evolutionary theory, we quantify the acquisition ages and fitness effects of the key driver events in AML development. The first driver mutations are typically acquired in the first few decades of life when the blood remains highly polyclonal. These early slow-growing clones subsequently acquire multiple further driver mutations which confer selective advantages up to 100-fold larger than the early drivers. These faster-growing clones harbouring multiple driver mutations can cause complete somatic sweeps of the blood decades before diagnosis, a feature strongly associated with future AML. Once established in the blood, the dynamics of driver mutations are highly predictable. Trajectories are shaped by strong clonal competition between lineages with limited evidence of other extrinsic factors playing a major role. Our data show that the clonal dynamics of blood are consistent with a set of remarkably simple evolutionary rules which strike a balance between chance and determinism.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7701c59da513a576289502ef6c9a6a503d909b39" target='_blank'>
              Evolutionary dynamics in the decades preceding acute myeloid leukaemia
              </a>
            </td>
          <td>
            Caroline J. Watson, Gladys Y. P. Poon, Hamish A.J. MacGregor, Adriana V.A. Fonseca, S. Apostolidou, A. Gentry-Maharaj, Usha Menon, J. Blundell
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Abstract The use of k-mers to capture genetic variation in bacterial genome-wide association studies (bGWAS) has demonstrated its effectiveness in overcoming the plasticity of bacterial genomes by providing a comprehensive array of genetic variants in a genome set that is not confined to a single reference genome. However, little attempt has been made to interpret k-mers in the context of genome rearrangements, partly due to challenges in the exhaustive and high-throughput identification of genome structure and individual rearrangement events. Here, we present GWarrange, a pre- and post-bGWAS processing methodology that leverages the unique properties of k-mers to facilitate bGWAS for genome rearrangements. Repeat sequences are common instigators of genome rearrangements through intragenomic homologous recombination, and they are commonly found at rearrangement boundaries. Using whole-genome sequences, repeat sequences are replaced by short placeholder sequences, allowing the regions flanking repeats to be incorporated into relatively short k-mers. Then, locations of flanking regions in significant k-mers are mapped back to complete genome sequences to visualise genome rearrangements. Four case studies based on two bacterial species (Bordetella pertussis and Enterococcus faecium) and a simulated genome set are presented to demonstrate the ability to identify phenotype-associated rearrangements. GWarrange is available at https://github.com/DorothyTamYiLing/GWarrangehttps://github.com/DorothyTamYiLing/genome_rearrangement.git.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c42162bc2b2f883e8a6834a9dc952dc10d3392ba" target='_blank'>
              GWarrange: a pre- and post- genome-wide association studies pipeline for detecting phenotype-associated genome rearrangement events
              </a>
            </td>
          <td>
            Yi Ling Tam, Sarah Cameron, Andrew Preston, Lauren A. Cowley
          </td>
          <td>2024-07-01</td>
          <td>Microbial Genomics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="


 What role do active enhancers play in determining the fate of early embryonic cells? Are there differences in active enhancers between mouse and human blastocysts?



 Tissue-specific active enhancers in inner cell mass (ICM) and trophoblast (TE) are essential for blastocyst formation, regulating gene expression through interactive loops with promoters.



 At the two-cell stage of nuclear transfer embryos, the absence of interaction between Zscan4d promoter and distal active enhancer leads to reduced Zscan4d expression, resulting in lower developmental rates. Overexpressing the demethylase Kdm4d alleviates reprogramming-resistant H3K9me3 modification, restoring the structural loop between the Zscan4d promoter and active enhancer and enhancing developmental efficiency. These findings strongly suggest a close association between the formation of regulatory chromatin interactions in early embryonic development and histone modifications. Indicating active enhancers marked with H3K27ac may regulate differentiation gene expression through distal interactions in blastocyst.



 Analyze variations in histone H3K27ac modification, chromatin accessibility, higher-order structures, and transcriptomes during mouse embryo differentiation from 8-cell stage to ICM and TE formation. Identify tissue-specific active enhancers in ICM and TE, validate their gene expression regulation, explore their role in early differentiation-associated genes in mice. Compare and clarify similarities and differences in enhancer regulation mechanisms during early embryo development in humans and mice.



 The 8-cell, morula and blastocyst stage embryos were collected from the oviducts and uterus of super-ovulated female ICR mice mated with male mice. The clinical human blastocysts were collected and separated the ICM and TE. Low-input multi-omics sequencing, including Cut-Tag, ATAC-seq, Hi-C, and RNA-seq, was performed on the embryonic samples. After analyzing and identifying active enhancers, tissue-specific ones were selected and validated for their impact on gene expression via CRISPR-Cas9 knockout in embryos.



 Inhibiting ep300 expression in mouse embryos through siRNA interference reduces blastocyst formation from 90% to approximately 40%. Introducing the small molecule inhibitor A485 at the 8-cell stage, blocking the active center of CBP/p300 enzymes, completely arrests embryos at the morula stage, preventing their development into blastocysts. Emphasizing the crucial role of correctly establishing active enhancers marked by H3K27ac modification in embryonic differentiation.
 Multi-omics data demonstrates high repeatability and quality. PCA clustering analysis reveals a clear differentiation path in the distribution of H3K27ac-modified histones, chromatin accessibility sites, and the transcriptome from the 8-cell stage to the blastocyst. Differences in H3K27ac histone modification distribution between the ICM and TE indicate the presence of tissue-specific active enhancers in blastocyst.
 The disparities in H3K27ac modification distribution between blastocyst ICM and TE primarily occur in distal gene regions rather than proximal ones. Moreover, these distal enhancers exhibit a positive correlation with the expression of lineage-specific genes, suggesting a probable regulatory role of distal enhancers in gene modulation during embryonic differentiation.
 Tissue-specific enhancer-promoter (EP) interaction loops emerge during embryonic differentiation, allowing direct interaction between tissue-specific active enhancers and promoters, thereby activating the expression of tissue-specific genes.



 While DNA sequences with H3K27ac modification are currently considered markers for active enhancers, H3K4me modification is another histone marker for enhancers. However, enhancers associated with H3K4me modification may not necessarily be in an activated state. Future research can simultaneously incorporate appropriate detection methods for H3K4me modification.



 After elucidating the mechanisms through which active enhancers regulate mouse differentiation, comparing the differences in active enhancers during the first lineage differentiation process between mice and humans can help identify factors that enhance human blastocyst quality. This can potentially improve clinical outcomes in assisted reproduction.



 not applicable
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06a46c47cbfd5b64e709929a1eba256f72f28a5e" target='_blank'>
              P-548 An investigation into the mechanisms of active enhancers regulating the initial lineage differentiation in mammals
              </a>
            </td>
          <td>
            M. Chen
          </td>
          <td>2024-07-01</td>
          <td>Human Reproduction</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The burden of cancer is on a rapid rise globally. Deregulated gene expression profiles may lead to the development of cancer. Master regulators are the regulatory checkpoints that define and control the intricate networks of gene and protein interactions that make up cell physiology. The molecular programs that generate particular cellular phenotypes depend on master regulators. SOX4 gene is a master regulator that controls the expression of other genes responsible for tumorigenesis and plays a crucial role in multiple signaling pathways. The expression of SOX4 is upregulated in various malignancies. Increased proliferation of cancer cells, survival, apoptosis, and epithelial-to-mesenchymal transition leading to metastasis have all been linked to SOX4 expression in cancer. Elevated levels of SOX4 also possess a correlation with poor prognosis in various cancer types. Recently, SOX4 has surfaced as a possible target for cancer therapeutics. Furthermore, it has been shown that targeting SOX4 could inhibit tumor growth and enhance the efficacy of conventional cancer therapies. The present review summarizes the current status of SOX4 in the initiation and progression of various human cancers.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47cd13b5840ecb03dba39c075ed71c34056378ac" target='_blank'>
              SOX4 expression in cancer: Insights from developmental regulation and deregulation in tumorigenesis
              </a>
            </td>
          <td>
            Surbhi Arora, Meena Godha, N. K. Lohiya, Nirmala Jagadish
          </td>
          <td>2024-07-29</td>
          <td>Journal of Reproductive Healthcare and Medicine</td>
          <td>0</td>
          <td>28</td>
        </tr>

        <tr id="Breast cancer (BC) is a highly heterogeneous disease, and the presence of germline breast cancer gene mutation (gBRCAm) is associated with a poor prognosis. Triple-negative breast cancer (TNBC) is a BC subtype, characterized by the absence of hormone and growth factor receptor expression, making therapeutic decisions difficult. Defects in the DNA damage response pathway due to mutation in breast cancer genes (BRCA 1/2) lead to homologous recombination deficiency (HRD). However, in HRD conditions, poly (adenosine diphosphate-ribose) polymerase (PARP) proteins repair DNA damage and lead to tumor cell survival. Biological understanding of HRD leads to the development of PARP inhibitors (PARPi), which trap PARP proteins and cause genomic instability and tumor cell lysis. HRD assessment can be an important biomarker in identifying gBRCAm patients with BC who could benefit from PARPi therapy. HRD can be identified by homologous recombination repair (HRR) gene-based assays, genomic-scarring assays and mutational signatures, transcription and protein expression profiles, and functional assays. However, gold standard methodologies that are robust and reliable to assess HRD are not available currently. Hence, there is a pressing need to develop accurate biomarkers identifying HRD tumors to guide targeted therapies such as PARPi in patients with BC. HRD assessment has shown fruitful outcomes in chemotherapy studies and preliminary evidence on PARPi intervention as monotherapy and combination therapy in HRD-stratified patients. Furthermore, ongoing trials are exploring the potential of PARPi in BC and clinically complex TNBC settings, where HRD testing is used as an adjunct to stratify patients based on BRCA mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da5590d00c940b501f01d6658b06c89cdb6ff89d" target='_blank'>
              Poly (adenosine diphosphate-ribose) polymerase inhibitors in the treatment of triple-negative breast cancer with homologous repair deficiency.
              </a>
            </td>
          <td>
            P. Yuan, Nan Ma, Binghe Xu
          </td>
          <td>2024-06-24</td>
          <td>Medicinal research reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25bc89ba9898a52d88ff5aa506e261a761286bfb" target='_blank'>
              piR-823 tale as emerging cancer-hallmark molecular marker in different cancer types: a step-toward ncRNA-precision.
              </a>
            </td>
          <td>
            Fatma H Shaker, E. Sanad, H. Elghazaly, Shih-Min Hsia, N. Hamdy
          </td>
          <td>2024-08-05</td>
          <td>Naunyn-Schmiedeberg's archives of pharmacology</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Ewing sarcoma (EwS) is a cancer that arises in the bones and soft tissues, typically driven by the Ewing’s sarcoma breakpoint region 1-Friend leukemia virus integration 1 (EWS-FLI) oncogene. Implementation of genetically modified animal models of EwS has proved difficult largely owing to EWS-FLI’s high toxicity. The EWS-FLI1FS frameshift variant that circumvents toxicity but is still able to perform key oncogenic functions provided the first study model in Drosophila. However, the quest for Drosophila lines expressing full-length, unmodified EWS-FLI remained open. Here, we show that EWS-FLI1FS’s lower toxicity is owed to reduced protein levels caused by its frameshifted C-terminal peptide, and report new strategies through which we have generated Drosophila lines that express full-length, unmodified EWS-FLI. Using these lines, we have found that the upregulation of transcription from GGAA-microsatellites (GGAAμSats) presents a positive linear correlation within a wide range of EWS-FLI protein concentrations. In contrast, rather counterintuitively, GGAAμSats-independent transcriptomic dysregulation presents relatively minor differences across the same range, suggesting that GGAAμSat-dependent and -independent transcriptional upregulation present different kinetics of response with regards to changing EWS-FLI protein concentration. Our results underpin the functional relevance of varying EWS-FLI expression levels and provide experimental tools to investigate, in Drosophila, the effect of the EWS-FLI ‘high’ and ‘low’ states that have been reported and are suspected to be important for EwS in humans.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aae15bdf2a86e467bdb76a0d6c4c39f6d7b414f7" target='_blank'>
              Human EWS-FLI protein levels and neomorphic functions show a complex, function-specific dose–response relationship in Drosophila
              </a>
            </td>
          <td>
            Serena Mahnoor, Cristina Molnar, Diego Velázquez, Jose Reina, S. Llamazares, J. P. Heinen, Jaume Mora, Cayetano Gonzalez
          </td>
          <td>2024-07-01</td>
          <td>Open Biology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dcaa5e009963f12c7b3cfac1a62813bb1ec1e2e7" target='_blank'>
              Removal of TREX1 activity enhances CRISPR-Cas9-mediated homologous recombination.
              </a>
            </td>
          <td>
            Mehmet E. Karasu, Eléonore Toufektchan, Yanyang Chen, Alessandra Albertelli, G. Cullot, J. Maciejowski, Jacob E. Corn
          </td>
          <td>2024-08-12</td>
          <td>Nature biotechnology</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cfa9b85d031e2f46dc36f65fc9e3a327afe81e85" target='_blank'>
              DNA hypo-methylation and expression of GBP4 induces T cell exhaustion in pancreatic cancer
              </a>
            </td>
          <td>
            Yesiboli Tasiheng, Xuan Lin, Xu Wang, Xuan Zou, Yusheng Chen, Yu Yan, Ming-zhe Ma, Zhen-bo Dai, Xu Wang, Xianjun Yu, He Cheng, Chen Liu
          </td>
          <td>2024-08-07</td>
          <td>Cancer Immunology, Immunotherapy : CII</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The genome of a cell is continuously battered by a plethora of exogenous and endogenous processes that can lead to damaged DNA. Repair mechanisms correct this damage most of the time, but failure to do so leaves mutations. Mutations do not occur in random manner, but rather typically follow a more or less specific pattern due to known or imputed mutational processes. Mutational signature analysis is the process by which the predominant mutational process can be inferred for a cancer and can be used in several contexts to study both the genesis of cancer and its response to therapy. Recent pan-cancer genomic efforts such as "The Cancer Genome Atlas" have identified numerous mutational signatures that can be categorized into single base substitutions, doublet base substitutions, or small insertions/deletions. Understanding these mutational signatures as they occur in non-small lung cancer could improve efforts at prevention, predict treatment response to personalized treatments, and guide the development of therapies targeting tumor evolution. For non-small cell lung cancer, several mutational signatures have been identified that correlate with exposures such as tobacco smoking and radon and can also reflect endogenous processes such as aging, APOBEC activity, and loss of mismatch repair. Herein, we provide an overview of the current knowledge of mutational signatures in non-small lung cancer.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/37777d22dfc5a2bd99dc42f300aad4d0a01e922f" target='_blank'>
              Topography of mutational signatures in non-small cell lung cancer: emerging concepts, clinical applications, and limitations.
              </a>
            </td>
          <td>
            Ritujith Jayakrishnan, David J. Kwiatkowski, Michal G Rose, Amin H. Nassar
          </td>
          <td>2024-06-22</td>
          <td>The oncologist</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7401c81aea35658fe72a5961ba5c3e31cae1d2fa" target='_blank'>
              The genomic basis of childhood T-lineage acute lymphoblastic leukaemia.
              </a>
            </td>
          <td>
            P. Pölönen, Danika Di Giacomo, Anna Eames Seffernick, Abdelrahman A. Elsayed, Shunsuke Kimura, Francesca Benini, Lindsey E Montefiori, Brent L Wood, Jason Xu, Changya Chen, Zhong-Yi Cheng, H. Newman, Jason R Myers, Ilaria Iacobucci, Elizabeth Li, Jonathan H. Sussman, Dale Hedges, Yawei Hui, Caroline Diorio, L. Uppuluri, David Frank, Yiping Fan, Yunchao Chang, S. Meshinchi, R. Ries, R. Shraim, Alexander Li, Kathrin M Bernt, M. Devidas, S. Winter, K. Dunsmore, H. Inaba, William L Carroll, Nilsa C Ramirez, Aaron H. Phillips, Richard W. Kriwacki, Jun J Yang, T. Vincent, Yaqi Zhao, P. S. Ghate, Jian Wang, Colleen Reilly, Xin Zhou, Mathijs A. Sanders, Junko Takita, Motohiro Kato, Nao Takasugi, Bill H Chang, Richard D Press, M. Loh, E. Rampersaud, E. Raetz, S P Hunger, Kai Tan, Ti-Cheng Chang, Gang Wu, S. B. Pounds, C. Mullighan, D. Teachey
          </td>
          <td>2024-08-14</td>
          <td>Nature</td>
          <td>0</td>
          <td>102</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ab7292394d6cef623d3eb21cd9965f9e6d3c70cc" target='_blank'>
              Unraveling the metastatic niche in breast cancer bone metastasis through single-cell RNA sequencing.
              </a>
            </td>
          <td>
            Xiangyu Li, Ziyu Gao, Meiling Yang, Ciqiu Yang, Dongyang Yang, Wenhui Cui, Dandan Wu, Jie-Qiong Zhou
          </td>
          <td>2024-07-27</td>
          <td>Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Our conception of gene regulation specificity has undergone profound changes over the last 20 years. Previously, regulators were considered to control few genes, recognized with exquisite specificity by a 'lock and key' mechanism. However, recently genome-wide exploration of regulator binding site occupancy (whether on DNA or RNA targets) revealed extensive lists of molecular targets for every studied regulator. Such poor biochemical specificity suggested that each regulator controls many genes, collectively contributing to biological phenotypes. Here, I propose a third model, whereby regulators' biological specificity is only partially due to 'lock and key' biochemistry. Rather, regulators affect many genes at the microscopic scale, but biological consequences for most interactions are attenuated at the mesoscopic scale: only a few regulatory events propagate from microscopic to macroscopic scale; others are made inconsequential by homeostatic mechanisms. This model is well supported by the microRNA literature, and data suggest that it extends to other regulators. It reconciles contradicting observations from biochemistry and comparative genomics on one hand and in vivo genetics on the other hand, but this conceptual unification is obscured by common misconceptions and counter-intuitive modes of graphical display. Profound understanding of gene regulation requires conceptual clarification, and better suited statistical analyses and graphical representation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/817f534202a0cedb44557f652834c36649dcd999" target='_blank'>
              A new perspective on microRNA-guided gene regulation specificity, and its potential generalization to transcription factors and RNA-binding proteins.
              </a>
            </td>
          <td>
            Hervé Seitz
          </td>
          <td>2024-08-16</td>
          <td>Nucleic acids research</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/263cda8d76b200b36443f700782153fb43cb23ea" target='_blank'>
              Increased AID results in mutations at the CRLF2 locus implicated in Latin American ALL health disparities
              </a>
            </td>
          <td>
            Valeria Rangel, Jason N. Sterrenberg, Aya Garawi, Vyanka Mezcord, Melissa L. Folkerts, Sabrina E. Calderon, Yadhira E. Garcia, Jinglong Wang, Eli M. Soyfer, Oliver S Eng, Jennifer B. Valerin, S. Tanjasiri, Fabiola Quintero-Rivera, M. Seldin, Selma Masri, Richard L. Frock, Angela G. Fleischman, Nicholas R Pannunzio
          </td>
          <td>2024-07-27</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Simple Summary This study has provided new insights into the role of E2F transcription factors, a group of proteins, in glioblastoma (GBM), a type of brain cancer. These proteins play a part in various processes that can lead to cell death, growth, and the formation of new blood vessels, all of which are crucial in the development of GBM. To understand how these proteins impact a specific type of GBM that responds to the drug Bevacizumab (BVZ), researchers utilized advanced computer algorithms on datasets from multiple large databases. By doing so, they were able to predict the potential lifespan of GBM patients based on the levels of these proteins found in their tumors. The study revealed that an increase in a protein called E2F8 and its regulatory networks following BVZ treatment might be associated with the recurrence of cancer. This suggests that E2F8 could be a key factor in the reemergence of GBM after treatment. Furthermore, the study indicated that BVZ could be detrimental to GBM patients whose tumors do not respond to the drug, potentially worsening the disease. These findings emphasize the significance of E2F proteins in the aggressiveness of GBM and its response to BVZ, suggesting their potential as markers for predicting patient outcomes and as targets for more personalized treatment approaches. Abstract The early 2-factor (E2F) family of transcription factors, including E2F1 through 8, plays a critical role in apoptosis, metabolism, proliferation, and angiogenesis within glioblastoma (GBM). However, the specific functions of E2F transcription factors (E2Fs) and their impact on the malignancy of Bevacizumab (BVZ)-responsive GBM subtypes remain unclear. This study used data from The Cancer Genome Atlas (TCGA), Chinese Glioma Genome Atlas (CGGA), European Molecular Biology Laboratory’s European Bioinformatics Institute (EMBL-EBI), and Gene Expression Omnibus (GEO) to explore the impact of eight E2F family members on the clinical characteristics of BVZ-responsive GBM subtypes and possible mechanisms of recurrence after BVZ treatment. Using machine learning algorithms, including TreeBagger and deep neural networks, we systematically predicted and validated GBM patient survival terms based on the expression profiles of E2Fs across BVZ-responsive GBM subtypes. Our bioinformatics analyses suggested that a significant increase in E2F8 post-BVZ treatment may enhance the function of angiogenesis and stem cell proliferation, implicating this factor as a candidate mechanism of GBM recurrence after treatment. In addition, BVZ treatment in unresponsive GBM patients may potentially worsen disease progression. These insights underscore that E2F family members play important roles in GBM malignancy and BVZ treatment response, highlighting their potential as prognostic biomarkers, therapeutic targets, and recommending precision BVZ treatment to individual GBM patients.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e894aab96af453781daef1e7295f1ea3fa78e9fa" target='_blank'>
              Early 2-Factor Transcription Factors Associated with Progression and Recurrence in Bevacizumab-Responsive Subtypes of Glioblastoma
              </a>
            </td>
          <td>
            Jian Shi
          </td>
          <td>2024-07-01</td>
          <td>Cancers</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f5dc719f0ae90529603e70fa22df6d2a71e97dd8" target='_blank'>
              In vivo single-cell CRISPR uncovers distinct TNF programmes in tumour evolution
              </a>
            </td>
          <td>
            Peter F Renz, Umesh Ghoshdastider, Simona Baghai Sain, Fabiola Valdivia-Francia, Ameya Khandekar, Mark Ormiston, Martino Bernasconi, Clara Duré, Jonas A. Kretz, Minkyoung Lee, Katie Hyams, Merima Forny, Marcel Pohly, Xenia Ficht, Stephanie J Ellis, A. Moor, Ataman Sendoel
          </td>
          <td>2024-07-17</td>
          <td>Nature</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Understanding the genetic regulation, for example, gene expressions (GEs) by copy number variations and methylations, is crucial to uncover the development and progression of complex diseases. Advancing from early studies that are mostly focused on homogeneous groups of patients, some recent studies have shifted their focus toward different patient groups, explored their commonalities and differences, and led to insightful findings. However, the analysis can be very challenging with one GE possibly regulated by multiple regulators and one regulator potentially regulating the expressions of multiple genes, leading to two distinct types of commonalities/differences in the patterns of genetic regulation. In addition, the high dimensionality of both sides of regulation poses challenges to computation. In this study, we develop a two-way fusion integrative analysis approach, which innovatively applies two fusion penalties to simultaneously identify commonalities/differences in the regulated pattern of GEs and regulating pattern of regulators, and adopt a Huber loss function to accommodate the possible data contamination. Moreover, a simple yet efficient iterative optimization algorithm is developed, which does not need to introduce any auxiliary variables and extra tuning parameters and is guaranteed to converge to a globally optimal solution. The advantages of the proposed approach are demonstrated in extensive simulations. The analysis of The Cancer Genome Atlas data on melanoma and lung cancer leads to interesting findings and satisfactory prediction performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c30f4bc0f5f2a42888c972380cc183c3c5a04c6" target='_blank'>
              Unveiling Commonalities and Differences in Genetic Regulations via Two-Way Fusion.
              </a>
            </td>
          <td>
            Biao Mei, Yu Jiang, Yifan Sun
          </td>
          <td>2024-08-12</td>
          <td>Journal of computational biology : a journal of computational molecular cell biology</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Richter’s transformation (RT) is a progression of chronic lymphocytic leukemia (CLL) to aggressive lymphoma. MGA (Max gene associated), a functional MYC suppressor, is mutated at 3% in CLL and 36% in RT. However, genetic models and molecular mechanisms of MGA deletion that drive CLL to RT remain elusive. We established an RT mouse model by knockout of Mga in the Sf3b1/Mdr CLL model using CRISPR-Cas9 to determine the role of Mga in RT. Murine RT cells exhibited mitochondrial aberrations with elevated oxidative phosphorylation (OXPHOS). Through RNA sequencing and functional characterization, we identified Nme1 (nucleoside diphosphate kinase) as an Mga target, which drives RT by modulating OXPHOS. Given that NME1 is also a known MYC target without targetable compounds, we found that concurrent inhibition of MYC and electron transport chain complex II substantially prolongs the survival of RT mice in vivo. Our results suggest that the Mga-Nme1 axis drives murine CLL-to-RT transition via modulating OXPHOS, highlighting a potential therapeutic avenue for RT. A MGA/NME1 regulatory axis can promote CLL-to-RT transition by modulating OXPHOS, highlighting this pathway as a target for RT treatment. Editor’s summary Richter’s transformation (RT) involves the progression of chronic lymphocytic leukemia (CLL) to a more aggressive lymphoma with poor outcomes. Understanding the transition into RT is key to more targeted therapeutic strategies. Iyer et al. examined the role of the MYC suppressor MGA, which is differentially expressed in RT, in an in vivo CLL model with MGA dysfunction. MGA loss resulted in increased mitochondrial function and reactive oxygen species and led to aberrant cell morphology and up-regulated MYC as commonly observed in patients. Combined inhibition of MYC and mitochondrial respiration prolonged in vivo survival in this model, highlighting potential therapeutic strategies for RT with this genetic profile. —Catherine Charneski">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75d1c8fe0890ca3dad978b633b25c5e7b5178f59" target='_blank'>
              MGA deletion leads to Richter’s transformation by modulating mitochondrial OXPHOS
              </a>
            </td>
          <td>
            Prajish Iyer, Bo Zhang, Tingting Liu, Meiling Jin, Kevyn Hart, Jibin Zhang, Viola Siegert, Marianne Remke, Xuesong Wang, Lei Yu, Joo Y Song, Girish Venkataraman, Wing C. Chan, Zhenyu Jia, Maike Buchner, Tanya Siddiqi, Steven T. Rosen, Alexey Danilov, Lili Wang
          </td>
          <td>2024-07-31</td>
          <td>Science Translational Medicine</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="The p53, a pivotal tumor suppressor, regulates various cellular responses, including DNA repair and apoptosis. Normally, p53 levels are low due to murine double minute clone 2 (MDM2) mediated polyubiquitination. However, stress signals disrupt p53-MDM2 interaction, stabilizing p53 and activating target genes. Dysfunctional p53 is common in cancers, especially colorectal cancer (CRC), with TP53 mutations in 43% of tumors. These mutations impair wild-type p53 function or confer novel activities, promoting cancer progression. Despite drugs targeting p53 entering trials, understanding wild-type and mutant p53 functions is crucial for novel CRC therapies. P53 mutations not only impact DNA repair and apoptosis but also play a crucial role in tumor immunotherapy. While rendering tumors resistant to chemotherapy, p53 mutations provide opportunities for immunotherapy due to neoantigen-rich tumors. Additionally, p53 mutations influence tumor microenvironment cells, such as fibroblasts and immunosuppressive cells, through p53-mediated signaling pathways. Investigating p53 mutations in tumor therapy is vital for personalized medicine and immunotherapy. In cancer treatment research, scientists explore drugs and strategies to restore or enhance p53 function. Targeting wild-type p53 aims to restore DNA repair and cell cycle control, while targeting mutant p53 seeks new drugs to inhibit its detrimental effects, advancing tumor treatment. Understanding p53 drugs and strategies is crucial for cancer therapy progress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4492a2019f4f8e179322826eb5b657b70923676" target='_blank'>
              Unraveling the Role of TP53 in Colorectal Cancer Therapy: From Wild-Type Regulation to Mutant.
              </a>
            </td>
          <td>
            Wenshu Li, Longyuan Li, Huan Yang, Chenxi Shi, Zhe Lei, Lingchuan Guo, Yuhong Wang
          </td>
          <td>2024-07-25</td>
          <td>Frontiers in bioscience</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Human pluripotent stem cells (hPSC) are pivotal in regenerative medicine, yet their in vitro expansion often leads to genetic abnormalities, raising concerns about their safety in clinical applications. This study analyzed ten human embryonic stem cell lines across multiple passages to elucidate the dynamics of chromosomal abnormalities and single nucleotide variants (SNVs) in 380 cancer-related genes. Prolonged in vitro culture resulted in 80% of the lines acquiring gains of chromosomes 20q or 1q, both known for conferring in vitro growth advantage. 70% of lines also acquired other copy number variants (CNVs) outside the recurrent set. Additionally, we detected 122 SNVs in 88 genes, with all lines acquiring at least one de novo SNV during culture. Our findings show higher loads of both CNVs and SNVs at later passages which are due to the cumulative acquisition of mutations over a longer time in culture and not to an increased rate of mutagenesis over time. Importantly, we observed that SNVs and rare CNVs follow the acquisition of chromosomal gains in 1q and 20q, while most of the low-passage and genetically balanced samples were devoid cancer-associated mutations. This suggests that the recurrent chromosomal abnormalities are the potential drivers for the acquisition of other mutations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/97f552b60bd87e180c4d6a5dd522428893e4b81b" target='_blank'>
              De novo cancer mutations frequently associate with recurrent chromosomal abnormalities during long-term human pluripotent stem cell culture
              </a>
            </td>
          <td>
            Diana Al Delbany, Manjusha S Ghosh, Nuša Krivec, Anfien Huygebaert, M. Regin, Chi Mai Duong, Yingnan Lei, K. Sermon, Catharina Olsen, C. Spits
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="Missense mutations can have diverse effects on proteins, depending on their location within the protein and the specific amino acid substitution. Mutations in the DNA mismatch repair gene MLH1 are associated with Lynch syndrome, yet the underlying mechanism of most disease-causing mutations remain elusive. To address this gap, we aim to disentangle the mutational effects on two essential properties for MLH1 function: protein stability and protein-protein interaction. We systematically examine the cellular abundance and interaction with PMS2 of 4839 (94%) MLH1 variants in the C-terminal domain. Our combined data shows that most MLH1 variants lose interaction with PMS2 due to reduced cellular abundance. However, substitutions to charged residues in the canonical interface lead to reduced interaction with PMS2. Unexpectedly, we also identify a distal region in the C-terminal domain of MLH1 where substitutions cause both decreased and increased binding with PMS2. Our data successfully distinguish benign from pathogenic MLH1 variants and correlate with thermodynamic stability predictions and evolutionary conservation. This work provides mechanistic insights into variant consequences and may help interpret MLH1 variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3c8c730b0b678584c2417f3109307d9b766e48e0" target='_blank'>
              Disentangling the mutational effects on protein stability and interaction of human MLH1
              </a>
            </td>
          <td>
            Sven Larsen-Ledet, Amelie Stein
          </td>
          <td>2024-08-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Retinoblastoma, a pediatric ocular malignancy, presents significant challenges in comprehending its molecular underpinnings and targeted therapeutic approaches. The dysregulated activity of histone deacetylases (HDACs) has been associated with retinoblastoma pathogenesis, influencing critical cellular processes like cell cycle regulation or retinal ganglion cell apoptosis. Through their deacetylase activity, HDACs exert control over key tumor suppressors and oncogenes, influencing the delicate equilibrium between proliferation and cell death. Furthermore, the interplay between HDACs and the retinoblastoma protein pathway, a pivotal aspect of retinoblastoma etiology, reveals a complex network of interactions influencing the tumor microenvironment. The examination of HDAC inhibitors, encompassing both established and novel compounds, offers insights into potential approaches to restore acetylation balance and impede retinoblastoma progression. Moreover, the identification of specific HDAC isoforms exhibiting varying expression in retinoblastoma provides avenues for personalized therapeutic strategies, allowing for interventions tailored to individual patient profiles. This review focuses on the intricate interrelationship between HDACs and retinoblastoma, shedding light on epigenetic mechanisms that control tumor development and progression. The exploration of HDAC-targeted therapies underscores the potential for innovative treatment modalities in the pursuit of more efficacious and personalized management strategies for this disease.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5855b4c3961d5048496222d3549d192782680eaf" target='_blank'>
              Histone Deacetylases in Retinoblastoma
              </a>
            </td>
          <td>
            Malwina Lisek, Julia Tomczak, Julia Swiatek, Aleksandra Kaluza, T. Boczek
          </td>
          <td>2024-06-24</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Phylogenetic relationship of cells within tumours can help us to understand how cancer develops in space and time, iden-tify driver mutations and other evolutionary events that enable can-cer growth and spread. Numerous studies have reconstructed phylo-genies from single-cell DNA-seq data. Here we are looking into the problem of phylogenetic analysis of spatially resolved near single-cell RNA-seq data, which is a cost-efficient alternative (or complemen-tary) data source that integrates multiple sources of evolutionary information including point mutations, copy-number changes, and epimutations. Recent attempts to use such data, although promis-ing, raised many methodological challenges. Here, we explored data-preprocessing and modelling approaches for evolutionary analyses of Visium spatial transcriptomics data. We conclude that using only highly variable genes and accounting for heterogeneous RNA capture across tissue-covered spots improves the reconstructed topological relationships and influences estimated branch lengths.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d567ed01155056fa70bf3fe42ae8fbf32987b2be" target='_blank'>
              Filtering for highly variable genes and high quality spots improves phylogenetic analysis of cancer spatial transcriptomics Visium data
              </a>
            </td>
          <td>
            Alexandra “Sasha” Gavryushkina, H. R. Pinkney, Sarah D. Diermeier, A. Gavryushkin
          </td>
          <td>2024-07-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fce76a61aebd891702bfcda234d051c833ee17c5" target='_blank'>
              The role of declining ataxia-telangiectasia-mutated (ATM) function in oocyte aging
              </a>
            </td>
          <td>
            Reiko Suzuki, Xiujuan Tan, Katarzyna J Szymanska, N. Kubikova, Columba Avila Perez, D. Wells, K. H. Oktay
          </td>
          <td>2024-06-25</td>
          <td>Cell Death Discovery</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Abstract Clonal cell population dynamics play a critical role in both disease and development. Due to high mitochondrial mutation rates under both healthy and diseased conditions, mitochondrial genomic variability is a particularly useful resource in facilitating the identification of clonal population structure. Here we present mitoClone2, an all-inclusive R package allowing for the identification of clonal populations through integration of mitochondrial heteroplasmic variants discovered from single-cell sequencing experiments. Our package streamlines the investigation of this phenomenon by providing: built-in compatibility with commonly used tools for the delineation of clonal structure, the ability to directly use multiplexed BAM files as input, annotations for both human and mouse mitochondrial genomes, and helper functions for calling, filtering, clustering, and visualizing variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/214acf148b721235cd5b2a8bca71ee2774084a79" target='_blank'>
              Mitoclone2: an R package for elucidating clonal structure in single-cell RNA-sequencing data using mitochondrial variants
              </a>
            </td>
          <td>
            Benjamin Story, Lars Velten, Gregor Mönke, Ahrmad Annan, Lars M Steinmetz
          </td>
          <td>2024-07-02</td>
          <td>NAR Genomics and Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="TRIM28 (tripartite motif protein 28) was initially believed to be a transcription inhibitor that plays an important role in DNA damage repair (DDR) and in maintaining cancer cellular stemness. As research has continued to deepen, several studies have found that TRIM28 not only has ubiquitin E3 ligase activity to promote degradation of substrates, but also can promote SUMOylation of substrates. Although TRIM28 is highly expressed in various cancer tissues and has oncogenic effects, there are still a few studies indicating that TRIM28 has certain anticancer effects. Additionally, TRIM28 is subject to complex upstream regulation. In this review, we have elaborated on the structure and regulation of TRIM28. At the same time, highlighting the functional role of TRIM28 in tumor development and emphasizing its impact on cancer treatment provides a new direction for future clinical antitumor treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/29d32f8c1b00487c33689a79eb7f6b394b29878a" target='_blank'>
              TRIM28 in cancer and cancer therapy
              </a>
            </td>
          <td>
            Kailang Li, Haifeng Wang, Bitao Jiang, Xiaofeng Jin
          </td>
          <td>2024-07-19</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT Mycobacterium smegmatis Nei2 is a monomeric enzyme with AP β-lyase activity on single-stranded DNA. Expression of Nei2, and its operonic neighbor Lhr (a tetrameric 3′-to-5′ helicase), is induced in mycobacteria exposed to DNA damaging agents. Here, we find that nei2 deletion sensitizes M. smegmatis to killing by DNA inter-strand crosslinker trimethylpsoralen but not to crosslinkers mitomycin C and cisplatin. By contrast, deletion of lhr sensitizes to killing by all three crosslinking agents. We report a 1.45 Å crystal structure of recombinant Nei2, which is composed of N and C terminal lobes flanking a central groove suitable for DNA binding. The C lobe includes a tetracysteine zinc complex. Mutational analysis identifies the N-terminal proline residue (Pro2 of the ORF) and Lys51, but not Glu3, as essential for AP lyase activity. We find that Nei2 has 5-hydroxyuracil glycosylase activity on single-stranded DNA that is effaced by alanine mutations of Glu3 and Lys51 but not Pro2. Testing complementation of psoralen sensitivity by expression of wild-type and mutant nei2 alleles in ∆nei2 cells established that AP lyase activity is neither sufficient nor essential for crosslink repair. By contrast, complementation of psoralen sensitivity of ∆lhr cells by mutant lhr alleles depended on Lhr’s ATPase/helicase activities and its tetrameric quaternary structure. The lhr–nei2 operon comprises a unique bacterial system to rectify inter-strand crosslinks. IMPORTANCE The DNA inter-strand crosslinking agents mitomycin C, cisplatin, and psoralen–UVA are used clinically for the treatment of cancers and skin diseases; they have been invaluable in elucidating the pathways of inter-strand crosslink repair in eukaryal systems. Whereas DNA crosslinkers are known to trigger a DNA damage response in bacteria, the roster of bacterial crosslink repair factors is incomplete and likely to vary among taxa. This study implicates the DNA damage-inducible mycobacterial lhr–nei2 gene operon in protecting Mycobacterium smegmatis from killing by inter-strand crosslinkers. Whereas interdicting the activity of the Lhr helicase sensitizes mycobacteria to mitomycin C, cisplatin, and psoralen–UVA, the Nei2 glycosylase functions uniquely in evasion of damage caused by psoralen–UVA.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/471e890636196ecfd4be94e1a14ead4277ca532d" target='_blank'>
              Structure and in vivo psoralen DNA crosslink repair activity of mycobacterial Nei2
              </a>
            </td>
          <td>
            G. M. Warren, Stewart Shuman
          </td>
          <td>2024-07-16</td>
          <td>mBio</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/798b264fbd39508ed9588f5472e886b8b6303884" target='_blank'>
              Navigating the complexity of p53-DNA binding: implications for cancer therapy
              </a>
            </td>
          <td>
            Kelly M. Thayer, Sean Stetson, Fernando Caballero, Christopher Chiu, I. Han
          </td>
          <td>2024-07-11</td>
          <td>Biophysical Reviews</td>
          <td>0</td>
          <td>2</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2024'],
    y: [2],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>